
<html lang="en"     class="pb-page"  data-request-id="103c4d94-65c6-4758-aabe-e28cca8eca1a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2004.47.issue-19;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/jm031139y;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Fuzzy Pharmacophore Models from Molecular Alignments for Correlation-Vector-Based Virtual Screening" /></meta><meta name="dc.Creator" content="Steffen  Renner and" /></meta><meta name="dc.Creator" content="Gisbert  Schneider*" /></meta><meta name="dc.Description" content="A pharmacophore-based approach for compiling focused screening libraries is presented. It integrates information from three-dimensional molecular alignments into correlation vector-based database s..." /></meta><meta name="Description" content="A pharmacophore-based approach for compiling focused screening libraries is presented. It integrates information from three-dimensional molecular alignments into correlation vector-based database s..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="August 11, 2004" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm031139y" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-47-19-4653" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(03)01139-7" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2004 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm031139y" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm031139y" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm031139y" /></link>
        
    
    

<title>Fuzzy Pharmacophore Models from Molecular Alignments for Correlation-Vector-Based Virtual Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm031139y" /></meta><meta property="og:title" content="Fuzzy Pharmacophore Models from Molecular Alignments for Correlation-Vector-Based Virtual Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yn00001.jpeg" /></meta><meta property="og:description" content=" A pharmacophore-based approach for compiling focused screening libraries is presented. It integrates information from three-dimensional molecular alignments into correlation vector-based database screening. The pharmacophore model is represented by a number of spheres of Gaussian-distributed feature densities. Different degrees of “fuzziness” can be introduced to influence the model's resolution. Transformation of this pharmacophore representation into a correlation vector results in a vector of feature probabilities which can be utilized for rapid virtual screening of compound databases or virtual libraries. The approach was validated by retrospective screening for cyclooxygenase 2 (COX-2) and thrombin ligands. A variety of models with different degrees of fuzziness were calculated and tested for both classes of molecules. Best performance was obtained with pharmacophore models reflecting an intermediate degree of fuzziness, yielding an enrichment factor of up to 39 for the first 1% of the ranked database. Appropriately weighted fuzzy pharmacophore models performed better in retrospective screening than similarity searching using only a single query molecule. The new pharmacophore method was shown to complement existing approaches.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm031139y"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm031139y">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm031139y&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm031139y&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm031139y&amp;href=/doi/10.1021/jm031139y" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2004</span><span class="cit-fg-volume">, 47</span><span class="cit-fg-issue">, 19</span><span class="cit-fg-pageRange">, 4653-4664</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/47/19" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm0495374" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm0498001" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fuzzy Pharmacophore Models from Molecular Alignments for Correlation-Vector-Based Virtual Screening</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steffen++Renner">Steffen Renner</a></span></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gisbert++Schneider">Gisbert Schneider</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Johann Wolfgang Goethe-Universität, Institut für Organische Chemie und Chemische Biologie, Marie-Curie-Strasse 11, D-60439 Frankfurt, Germany </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm031139y&amp;href=/doi/10.1021%2Fjm031139y" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2004</span></span><span class="cit-volume">, 47</span><span class="cit-issue">, 19</span><span class="cit-pageRange">, 4653–4664</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 11, 2004</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 December 2003</li><li><span class="item_label"><b>Published</b> online</span>11 August 2004</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 September 2004</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm031139y" title="DOI URL">https://doi.org/10.1021/jm031139y</a></div><div class="article_header-article-copyright"><strong>Copyright © 2004 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4653%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSteffen%2BRenner%252C%2BGisbert%2BSchneider%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D47%26issueNum%3D19%26contentID%3Djm031139y%26title%3DFuzzy%2BPharmacophore%2BModels%2Bfrom%2BMolecular%2BAlignments%2Bfor%2BCorrelation-Vector-Based%2BVirtual%2BScreening%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4664%26publicationDate%3DSeptember%2B2004">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm031139y"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">484</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">40</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm031139y" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fuzzy Pharmacophore Models from Molecular Alignments for Correlation-Vector-Based Virtual Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Steffen&quot;,&quot;last_name&quot;:&quot;Renner&quot;},{&quot;first_name&quot;:&quot;Gisbert&quot;,&quot;last_name&quot;:&quot;Schneider&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2004&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;19&quot;,&quot;volume&quot;:&quot;47&quot;,&quot;pages&quot;:&quot;4653-4664&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm031139y&quot;},&quot;abstract&quot;:&quot;A pharmacophore-based approach for compiling focused screening libraries is presented. It integrates information from three-dimensional molecular alignments into correlation vector-based database screening. The pharmacophore model is represented by a number of spheres of Gaussian-distributed feature densities. Different degrees of “fuzziness” can be introduced to influence the model's resolution. Transformation of this pharmacophore representation into a correlation vector results in a vector of feature probabilities which can be utilized for rapid virtual screening of compound databases or virtual libraries. The approach was validated by retrospective screening for cyclooxygenase 2 (COX-2) and thrombin ligands. A variety of models with different degrees of fuzziness were calculated and tested for both classes of molecules. Best performance was obtained with pharmacophore models reflecting an intermediate degree of fuzziness, yielding an enrichment factor of up to 39 for the first 1% of the ranked database. A&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm031139y&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm031139y" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm031139y&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm031139y" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm031139y&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm031139y" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm031139y&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm031139y&amp;href=/doi/10.1021/jm031139y" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm031139y" /></input><a href="/doi/pdf/10.1021/jm031139y" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (474 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm031139y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm031139y%26sid%3Dliteratum%253Aachs%26pmid%3D15341481%26genre%3Darticle%26aulast%3DRenner%26date%3D2004%26atitle%3DFuzzy%2BPharmacophore%2BModels%2Bfrom%2BMolecular%2BAlignments%2Bfor%2BCorrelation-Vector-Based%2BVirtual%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D47%26issue%3D19%26spage%3D4653%26epage%3D4664%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290663" title="Pharmacophores">Pharmacophores</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290957" title="Molecular modeling">Molecular modeling</a>,</li><li><a href="/action/doSearch?ConceptID=291967" title="Cluster chemistry">Cluster chemistry</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/47/19" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A pharmacophore-based approach for compiling focused screening libraries is presented. It integrates information from three-dimensional molecular alignments into correlation vector-based database screening. The pharmacophore model is represented by a number of spheres of Gaussian-distributed feature densities. Different degrees of “fuzziness” can be introduced to influence the model's resolution. Transformation of this pharmacophore representation into a correlation vector results in a vector of feature probabilities which can be utilized for rapid virtual screening of compound databases or virtual libraries. The approach was validated by retrospective screening for cyclooxygenase 2 (COX-2) and thrombin ligands. A variety of models with different degrees of fuzziness were calculated and tested for both classes of molecules. Best performance was obtained with pharmacophore models reflecting an intermediate degree of fuzziness, yielding an enrichment factor of up to 39 for the first 1% of the ranked database. Appropriately weighted fuzzy pharmacophore models performed better in retrospective screening than similarity searching using only a single query molecule. The new pharmacophore method was shown to complement existing approaches. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm031139yAF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  To whom correspondence should be addressed. Phone:  +49 69 798 29821. Fax:  +49 69 798 29826. E-mail:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="3d5a544e5f584f49134e5e5553585459584f7d505259515c5f13595813">[email protected]</a> </p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25113" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25113" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Virtual screening has proven to be a fast and efficient tool for hit discovery and optimization.<sup><a onclick="showRef(event, 'jm031139yb00001'); return false;" href="javascript:void(0);" class="ref jm031139yb00001">1</a></sup> Virtual compound classification prior to experimental testing is a very fast and cost saving way of finding new screening candidates with desired biological activity. Classification methods range from simple comparison of physicochemical descriptors to complex estimations of binding constants by docking calculations or quantitative structure−activity relationship (QSAR) models.<sup><a onclick="showRef(event, 'jm031139yb00002'); return false;" href="javascript:void(0);" class="ref jm031139yb00002">2</a></sup> Among a plethora of different approaches, pharmacophore-based applications are very popular and have shown to be successful in the capture of new active molecules and lead candidates.<sup><a onclick="showRef(event, 'jm031139yb00003'); return false;" href="javascript:void(0);" class="ref jm031139yb00003">3</a></sup> Pharmacophore models represent the location of generalized receptor−ligand interaction sites in three-dimensional (3D) space which are considered to form a specific binding pattern. Two basically different approaches for the representation of the pharmacophore can be distinguished:  3D pharmacophore models and pharmacophore fingerprints.<sup><a onclick="showRef(event, 'jm031139yb00004'); return false;" href="javascript:void(0);" class="ref jm031139yb00004">4</a></sup></div><div class="NLM_p">The traditional 3D-approach, implemented in program packages such as Catalyst,<sup><a onclick="showRef(event, 'jm031139yb00005'); return false;" href="javascript:void(0);" class="ref jm031139yb00005">5</a></sup> DISCO,<sup><a onclick="showRef(event, 'jm031139yb00006'); return false;" href="javascript:void(0);" class="ref jm031139yb00006">6</a></sup> GASP,<sup><a onclick="showRef(event, 'jm031139yb00007'); return false;" href="javascript:void(0);" class="ref jm031139yb00007">7</a></sup> or MOE,<sup><a onclick="showRef(event, 'jm031139yb00008'); return false;" href="javascript:void(0);" class="ref jm031139yb00008">8</a></sup> usually determines the most conserved features of a set of structurally aligned known active ligands. The spatial configuration of generalized interaction sites provides the basis for subsequent virtual screening for new molecules with the same biological activity. Those molecules which comprise all or a user-defined minimum number of the features are presumed to be active. A general drawback of this approach is the necessity to align a molecule to the pharmacophore query before it can be classified as potentially active or inactive.<sup><a onclick="showRef(event, 'jm031139yb00009'); return false;" href="javascript:void(0);" class="ref jm031139yb00009">9</a></sup> This step can limit the screening of very large databases or virtual libraries. Another drawback can be the lack of information about less conserved regions. This is not easily possible to take into consideration since a minimum number of features have to be satisfied by a molecule to be classified as active.<sup>4</sup> “Excluded volumes” can compensate for a part of this problem by preventing the selection of molecules that are too large for the binding pocket.<sup>4</sup></div><div class="NLM_p">Pharmacophore fingerprints describe the spatial arrangement of pharmacophore features typically as a bitstring where each bit corresponds to a certain feature, or in the form of a correlation vector (CV).<sup><a onclick="showRef(event, 'jm031139yb00010 jm031139yb00011'); return false;" href="javascript:void(0);" class="ref jm031139yb00010 jm031139yb00011">10,11</a></sup> The latter has the advantage that no explicit alignment is needed to estimate the activity of a given molecule, which allows for rapid screening of large compound databases and makes the method resistant to alignment errors. Still, the time-limiting factor is the calculation of 3D conformers. Usually, for each molecule a separate CV is stored. The common application is to take one or more known active query molecules and calculate the similarity of the CVs of each candidate molecule to the CV of the query (or “reference”) molecule. The most similar compounds are then presumed to be active. </div><div class="NLM_p">While there is information retained about all features present in a molecule, a limitation of many pharmacophore methods is that they usually do not contain <a name="jm004701904654" id="jm004701904654"></a> information about the conservation and tolerance of individual pharmacophore features. First successful applications have been described that go into this direction,<sup><a onclick="showRef(event, 'jm031139yb00012'); return false;" href="javascript:void(0);" class="ref jm031139yb00012">12</a></sup> but still information about conserved features should be more reliable. A recent attempt to introduce fuzziness into CV-based retrospective screening showed no significant improvement.<sup><a onclick="showRef(event, 'jm031139yb00013'); return false;" href="javascript:void(0);" class="ref jm031139yb00013">13</a></sup> A shortcoming of this approach might have been that a uniform degree of fuzziness was used for all pairs of features of the molecules. </div><div class="NLM_p">Another property of pharmacophore fingerprints that can either be considered as benefit or as drawback is that the descriptor of a molecule is not necessarily unique for that molecule. On one hand this can lead us to molecules that are structurally different from the reference molecules and do not exhibit any desired biological activity. On the other hand it can help discover compounds with similar biological activity but significantly different scaffolds, which is often a desired result and is sometimes referred to as “scaffold hopping”.<sup>11</sup> Examples of pharmacophore fingerprints are our own CATS pharmacophore descriptors<sup><a onclick="showRef(event, 'jm031139yb00014'); return false;" href="javascript:void(0);" class="ref jm031139yb00014">11,14</a></sup> and the well-established three- and four-point pharmacophore descriptors.<sup><a onclick="showRef(event, 'jm031139yb00015 jm031139yb00016'); return false;" href="javascript:void(0);" class="ref jm031139yb00015 jm031139yb00016">15,16</a></sup></div><div class="NLM_p last">The present study presents a pharmacophore approach combining some advantages of both methods, which we termed SQUID (<i>S</i>ophisticated <i>Qu</i>antification of <i>I</i>nteraction <i>D</i>istributions). The aim was to use the information about the conservation and tolerance of features from the alignment of known active ligands and to transform this information into a CV for fast database screening. The information about feature conservation is stored in the form of feature probabilities. A further aim of this work was to explore the effect of different degrees of structural generalization from the original molecular architecture. In SQUID pharmacophore models the distribution of features from the structural alignment is described by a number of Gaussian feature densities with different positions, conservation weights, and tolerances, which are referred to as potential pharmacophore points (PPPs) herein. Thus, the pharmacophore model may contain large numbers of small PPPs or small numbers of large PPPs. The second situation results in a more generalized or “fuzzy” representation of the alignment. Virtual screening with fuzzy pharmacophore representations might allow for the capture of more structural dissimilar molecules with similar biological activity. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57477" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57477" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Pharmacophore Model Perception.</b> An overview of the calculation of a SQUID pharmacophore model and the derivation of a CV is given in Figure 1<a data-tab="pane-pcw-Figures" href="#jm031139yf00001" id="" class=" internalNav"></a>. The starting point of the calculation is a 3D alignment of reference molecules. This alignment is transformed into a three-dimensional distribution of generalized pharmacophoric interaction points (Figure 1a). For SQUID pharmacophore models this field of features is approximated by a set of Gaussian PPPs for a generalized description of the molecules. Dependent on a defined cluster radius, the local density maxima of the pharmacophore feature distribution are determined for the clustering of features into PPPs (Figure 1b). As a result, the distribution of features is approximated by Gaussian PPPs with appropriate tolerances, which are placed into the center of the clustered features. Then the PPPs are weighted according to the conservation of their underlying pharmacophore features among the reference molecules (Figure 1c). In the last step the spatial distribution of PPP densities is transformed into a CV of feature densities describing the distribution of pairs of PPPs within the CV. For this purpose the feature density distributions between all pairs of PPPs is summed up in distance bins of the CV (Figure 1d). <div id="_i5"><figure id="jm031139yf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00001.jpeg" id="_i6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Schematic overview of the calculation of a SQUID pharmacophore model and a corresponding CV descriptor. (a) An alignment of active reference molecules is transformed into a 3D distribution of generalized pharmacophore interaction points. Yellow points represent hydrophobic interacting atoms, green points represent atoms which can interact as hydrogen bond acceptor and donor. (b) The interaction points are clustered to form PPPs based on the feature density distribution of the respective features and based on the defined resolution of the resulting model. The resolution of the model is affected by the cluster radius. Each PPP should represent a local maximum in the local feature density distribution. The local feature density LFD for each interaction point is calculated with eq 1. In this example a cluster radius of 1.5 Å was used. The circles exemplary show the cluster radii around the features which are marked by dots. The two maxima are highlighted in red. As can be seen in Figure 1b, the hydrophobic maximum is the only feature with five neighboring hydrophobic features within its cluster radius. All other hydrophobic features with lower LFD values have fewer hydrophobic features within their cluster radii. After determination of the maxima, all remaining interaction points are assigned to the nearest maximum of their respective feature-type. (c) PPPs are defined at the position of the geometric center of the atoms which are assigned to a local maximum. Two PPPs (yellow = hydrophobic, green = polar = hydrogen bond acceptor and donor) have been calculated in this example. The standard deviations σ of the PPPs are defined by the median distance of the atoms of the corresponding maximum to the position of the PPP, with a minimum value of 0.5 Å like for the polar PPP. The polar PPP has a conservation weight <i>W</i><sub>c</sub> of 0.5 since it consists only of atoms of one of the two molecules. The <i>W</i><sub>c</sub> of 0.96 for the hydrophobic PPP results from the slightly different number of hydrophobic interactions in the two molecules. (d) For virtual screening the pharmacophore model is transformed into a CV with eq 3. This CV represents a vector of probabilities for the presence of pairs of interactions within certain distances. A section of the resulting CV representing polar−hydrophobic pairs of PPPs within a distance range of 0 to 20 Å is shown. The highest probability of finding such a pair is such within a distance range of 2 to 4 Å. With more pairs of PPPs the probability distributions of these pairs of the bins are added. For virtual screening the whole CV is scaled to a maximum of 1. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">Alignments of reference compounds were obtained either with the flexible alignment tool or with the homology alignment tool of MOE.<sup>8</sup> Flexible alignments were generated with the features “hydrogen-bond donor”, “hydrogen-bond acceptor”, “aromaticity”, “hydrophobe”, and “volume” with default weights. Alignments of protein structures were generated with the homology alignment tool in MOE and default parameters. Transformation of the ligands was included in the transformation of protein structures. </div><div class="NLM_p">For the assignment of SQUID pharmacophore features, six different generalized interaction types were considered:  “cationic”, “anionic”, “polar”, “hydrogen-bond donor”, “hydrogen-bond acceptor”, and “hydrophobic”. One of these features was assigned to each of the atoms of the alignment of known active ligands with the ph4_aType function available in MOE.<sup>8</sup> Atoms for which no match was found were not further considered. In this way the 3D alignment of reference molecules was transferred into a three-dimensional distribution of pharmacophoric features, defined by generalized interaction types. This distribution could be considered as a field of the features of the molecules (Figure 1a). </div><div class="NLM_p">For a more generalized or fuzzy approximation of this field by Gaussian PPPs, the single features had to be clustered into PPPs. Each PPP should represent a local maximum of the distribution of the pharmacophoric features. To define the degree of “fuzziness” of the pharmacophore model, the “cluster radius” was introduced as a variable, which defines the spatial area within which feature maxima were determined. The cluster radius determines the total number of PPPs that are present in a pharmacophore model and thus the degree of structural generalization from an atomistic view. To determine the local maxima of the feature distribution, a cluster radius dependent local feature density LFD was calculated for each position of an atom of the molecular ensemble. For an atom k of the pharmacophore feature-type T, the local feature density was calculated by<a class="ref internalNav" href="#jm031139ye00001" aria-label=""></a><span class="NLM_disp-formula" id="jm031139ye00001"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye00001.gif" alt="" id="_i7" /></img></span> where <i>n</i> is the number of atoms of type T in the molecular ensemble, <i>D</i><sub>2</sub> is the Euclidean distance between two atoms, and <i>r</i><sub>c</sub> is the cluster radius. Only atoms within <i>r</i><sub>c</sub> around an atom contribute to the LFD of this atom. The shorter the distance of an atom to the atom under consideration the higher is the impact of this atom on the LFD. A local maximum of the distribution of a pharmacophore feature <i>T</i> was defined as the <a name="jm004701904655" id="jm004701904655"></a> position of an atom of feature-type <i>T</i> for which there was no other atom of type <i>T</i> present within <i>r</i><sub>c</sub> yielding a higher LFD value (Figure 1b). If there were two atoms with the same LFD value, one of the atoms was selected randomly. </div><div class="NLM_p">For clustering of the atoms into PPPs, all atoms of the system were then assigned to the nearest maximum of their type. The geometric center of the atoms of a PPP was taken as the position of the PPP. The median distance of all atoms contributing to a PPP to the center of this PPP was taken as the value of the standard deviation σ of the PPP. For this value a minimum of 0.5 Å was defined (Figure 1c). The standard deviation of a PPP was assumed to be roughly proportional to the number of atoms, which are represented by a PPP. </div><div class="NLM_p">A basic assumption of our approach is that conserved features of a set of similarly acting molecules should have a stronger impact on the biological function than features which are not very well conserved. The premise for this assumption is that the molecules have comparable activities on their receptor. Thus the PPPs should not be weighted by the number of atoms they represent; instead they were weighted by the conservation of the features they represent within the reference molecules. The information about feature conservation was expressed by the conservation weight:<a class="ref internalNav" href="#jm031139ye00002" aria-label=""></a><span class="NLM_disp-formula" id="jm031139ye00002"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye00002.gif" alt="" id="_i8" /></img></span> where <i>n</i> is the number of molecules in the model. This function returns a maximum value of 1 for PPPs representing the same number of atoms from all molecules of the ensemble, and a minimum value of <i>n</i><sup>-1</sup> for PPPs which consist only of atoms of one molecule (Figure 1c). The conservation weight of a PPP of type T can be considered as the probability of finding a feature of type T at the position of the PPP. The resulting SQUID pharmacophore model can thus be considered as the three-dimensional probability distribution of the features found in a 3D molecular alignment. </div><div class="NLM_p"><b>Database Screening.</b> For the sake of very rapid database screening, we did not explicitly align molecules to a 3D pharmacophore model. Instead the 3D model was transformed into a CV which can be utilized for efficient screening of databases containing CV-encoded <a name="jm004701904656" id="jm004701904656"></a> molecules. In the way the three-dimensional pharmacophore model accounts for the probabilities for the presence of features at positions in the explicit 3D space, the CV descriptor should account for the same probability distribution within the CV. For this purpose, the pharmacophore model and the molecules of a screening database were both constructed as a pairwise distance and feature-type dependent autocorrelation vector. The pharmacophore CV contains the probabilities for the presence of pairs of atoms within certain distances (Figure 1d) instead of the probabilities for the presence of single atoms within 3D space in the underlying pharmacophore model. </div><div class="NLM_p">All entries in the screening database were encoded by the CATS3D descriptor,<sup>14</sup> which is a 3D extension of the topological CV-based CATS descriptor.<sup>11</sup> As described for the calculations of the pharmacophore model (vide supra), each atom of a molecule was assigned to one of the six generalized atom types of the ph4_aType function of MOE. For the 21 possible pairs of these generalized atom types {cationic−cationic, cationic−anionic, cationic-polar, etc., hydrogen-bond acceptor−hydrophobic, hydrophobic−hydrophobic}, the Euclidean distances of all atoms were measured, and the pair frequencies were partitioned into 20 equal distance bins in [0,20] Å, which resulted in a CV of 420 dimensions. The value stored in each bin was scaled by the added incidences of the two respective features. Thus each dimension of the CATS3D CV was calculated according to<a class="ref internalNav" href="#jm031139ye00003" aria-label=""></a><span class="NLM_disp-formula" id="jm031139ye00003"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye00003.gif" alt="" id="_i9" /></img></span> where <i>i</i> and <i>j</i> are atoms, <i>d</i> is a distance range, TP are the pharmacophoric feature-types T of the pair of atoms <i>i</i> and <i>j</i>, <i>A</i> and <i>B</i> are the number of atoms of the feature-types of atom <i>i</i> and atom <i>j</i>, respectively, and<span class="NLM_inline-graphic"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye10001.gif" alt="" /></img></span> (Kronecker delta) evaluates to 1 for all pairs of atoms of types TP within the distance range <i>d</i>. The factor of 0.5 in the sum avoids double counting of pairs. Pairs of atoms with themselves were not considered. In contrast to the original CATS3D version used for similarity searching,<sup>14</sup> the CV was finally scaled to a maximum value of 1. </div><div class="NLM_p">For comparison of database compounds with the SQUID pharmacophore model, both CV representations must have the same number of dimensions and the same arrangement of distance bins and pairs of generalized atoms. Using the conservation weights and the standard deviations of the PPPs, the Gaussian probability density for the presence of each of the PPP pairs within the distance ranges of each bin of the CV was calculated (eq 4). The resulting SQUID CV contains the normalized sum over the probability densities of all pairs of PPPs:<a class="ref internalNav" href="#jm031139ye00004" aria-label=""></a><span class="NLM_disp-formula" id="jm031139ye00004"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye00004.gif" alt="" id="_i10" /></img></span> where <i>i</i> and <i>j</i> are PPPs, <i>d</i> is a distance range, TP are the feature-types T of the pair of PPPs <i>i</i> and <i>j</i>, <i>A</i> and <i>B</i> are the number of PPPs of the feature-types of <i>i</i> and <i>j</i>, <i>W</i><sub>c</sub> is the conservation weight of a PPP, σ is the standard deviation of a PPP, center<i><sub>d</sub></i> is the center of the distance range <i>d</i>, and δ<sup>TP</sup> evaluates to 1 for all pairs of PPPs of types TP<i>.</i> The factor of 0.5 in the sum avoids double counting of pairs. Pairs of PPPs with themselves were not considered. The values of each dimension were scaled by the total number of possible pairs of PPPs of the two features considered. Finally the CV was scaled to a maximum value of 1, which is a premise for the interpretation of the bin-values as probabilities. Equation 4 transforms the two Gaussian probability distributions of two PPPs found in a certain distance into one Gaussian probability distribution for the presence of the pair of PPPs within the distance ranges given by the CV (Figure 1d). The pairwise Gaussian distributions were summed over all possible pairs of PPPs of the system. </div><div class="NLM_p">In addition to the conservation weights we introduced “feature-type weights” for the mutual weighting of the importance of the generalized feature-types within the CV, e.g. by supervised training using sets of reference molecules. These weights were introduced since we observed that the conservation weights alone were not always adequate to result in successful retrieval of active molecules by retrospective screening. Weight values were specified for each of the pharmacophore feature-types present in the model. The sums of single feature-type weights were used as feature-type weights for each pair of feature-types present in the CV. Subsequently the sum of the probabilities in the CV was scaled individually for each pair of feature-types over all distance bins. It was scaled to the value of the feature-type weight for the respective pair. Finally the CV was again scaled to a maximum of 1. </div><div class="NLM_p">A score for the similarity between the SQUID model and an individual molecule was calculated according to eq 5.<a class="ref internalNav" href="#jm031139ye00005" aria-label=""></a><span class="NLM_disp-formula" id="jm031139ye00005"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye00005.gif" alt="" id="_i11" /></img></span> where <i>a</i><i><sub>i</sub></i> is the value of the <i>i</i>-th element of the SQUID descriptor, <i>b</i><i><sub>i</sub></i> is the value of the <i>i</i>-th element of a molecule descriptor, and <i>n</i> is the total number of dimensions. The value <i>a</i><i><sub>i</sub></i> may be considered as the idealized probability of the presence of atom features in <i>b</i><i><sub>i</sub></i>. This results in high scores for molecules with many features in regions of the query descriptor which have a high probability. To penalize the presence of such atom pairs in regions with a low probability, the denominator weights the presence of atom pairs with the inverted probabilities of the descriptor of the pharmacophore model. A value of 1 was added to the denominator to avoid division by zero and high scores resulting from a very low value in the denominator of the term. </div><div class="NLM_p">All methods for the calculation of the pharmacophore model, the calculation of the CVs of the pharmacophore <a name="jm004701904657" id="jm004701904657"></a> model and the calculation of the CATS3D descriptors were implemented in the SVL programming language in MOE.<sup>8</sup> For all calculations in MOE we used version 2003.02 of the software. </div><div class="NLM_p"><b>Retrospective Screening.</b> The quality of a SQUID model was evaluated by its capability to produce high scoring values for molecules with the desired biological activity and low scoring values for molecules without this activity. This capability was expressed by the enrichment factor ef (eq 6). ef represents the ratio between the percentage of active molecules in a top x% fraction of the sorted database to the percentage of active molecules in the whole database.<a class="ref internalNav" href="#jm031139ye00006" aria-label=""></a><span class="NLM_disp-formula" id="jm031139ye00006"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye00006.gif" alt="" id="_i12" /></img></span> where <i>F</i><sub>act</sub> and <i>D</i><sub>act</sub> are the numbers of known active molecules in the subset and the whole database, and <i>F</i><sub>all</sub> and <i>D</i><sub>all</sub> are the total numbers of molecules in the subset and the whole database, respectively. An enrichment factor of 1 corresponds to a random distribution of active molecules in the ranked database; thus an effective pharmacophore model should result in an ef above 1. </div><div class="NLM_p">In virtual screening, one is usually interested in the enrichment of the first few percentages of a database. In cases where the focus was a more general assessment of the enrichment capabilities of pharmacophore models over the whole set of reference molecules, we used the following enrichment value <i>ev</i> for the comparison of the pharmacophore models.<a class="ref internalNav" href="#jm031139ye00007" aria-label=""></a><span class="NLM_disp-formula" id="jm031139ye00007"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139ye00007.gif" alt="" id="_i13" /></img></span> where ef(<i>i</i>%) is the enrichment factor for the first <i>i</i>% of the ranked database. This returns the weighted sum of the enrichment factors of the whole database. The smaller the fraction of the database, the higher is the weight for the ef. </div><div class="NLM_p"><b>Data Sets.</b> The COBRA data collection was used for retrospective screening experiments.<sup><a onclick="showRef(event, 'jm031139yb00017'); return false;" href="javascript:void(0);" class="ref jm031139yb00017">17</a></sup> The database version consisted of 4705 annotated ligands of biological targets compiled from scientific literature. All ligands were present in neutralized form. A database with up to 50 conformations per molecule was calculated with MOE using default parameter settings. Each resulting conformation was energy minimized with the MMFF94 force field with default parameters in MOE. For 73 molecules the algorithm failed to calculate conformations. For these molecules, single 3D conformations were calculated with CORINA.<sup><a onclick="showRef(event, 'jm031139yb00018'); return false;" href="javascript:void(0);" class="ref jm031139yb00018">18</a></sup> The resulting database consisted of 124 910 conformations. For retrospective screening, we removed those molecules from the database, which were used for pharmacophore model generation. The resulting databases for retrospective screening consisted of 92 active molecules and 4611 inactive molecules for cyclooxygenase 2 (COX-2), and 188 actives and 4517 inactive compounds for thrombin. </div><div class="NLM_p last">For calculation of a COX-2 pharmacophore model, the crystal structures of COX-2 with the specific inhibitor SC-558 (1CX2) and the structures of COX-2 with the unspecific inhibitors flurbiprofen (3PGH) and indomethacin (4COX) were used to model a template alignment for the flexible alignment of the specific COX-2 inhibitors rofecoxib and molecule 5 (M5) from Palomer et al.<sup><a onclick="showRef(event, 'jm031139yb00019'); return false;" href="javascript:void(0);" class="ref jm031139yb00019">19</a></sup> For calculation of the thrombin pharmacophore model we used the pdb structures 1C4V, 1D4P, 1D6W, 1D9I, 1DWD, 1FPC, and 1TOM.<sup><a onclick="showRef(event, 'jm031139yb00020'); return false;" href="javascript:void(0);" class="ref jm031139yb00020">20</a></sup></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70468" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70468" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The SQUID approach was tested for two classes of biological targets:  COX-2 and thrombin. Both targets are well characterized in the literature, and crystal structures of the receptors with bound inhibitors are available. This was important since our method depends on a meaningful alignment of ligands. For both targets, pharmacophore models had already been developed by other groups,<sup>19,20</sup> and diverse sets of ligands are known, which provided a good starting point for retrospective screening. SQUID pharmacophore models were calculated using different PPP cluster radii and tested for performance by retrospective screening. </div><div class="NLM_p"><b>Pharmacophore Model of COX-2 Ligands.</b> Palomer et al.<sup>19</sup> derived a pharmacophore model for COX-2 inhibitors on the basis of five specific inhibitors SC-558 (<b>1</b>), rofecoxib (<b>2</b>), DFU, celecoxib, and a molecule which they termed “molecule 5” (M5, <b>3</b>). For calculation of a 3D structural alignment of these ligands, they used a template alignment of all COX-2 ligands, for which there was a crystal structure of the ligand−receptor complex available. Crystal structures were at hand for SC-558 (1CX2) and the two unspecific inhibitors flurbiprofen (3PGH) and indomethacin (4COX). The alignment of these molecules was performed by superposition of their protein structures. The remaining ligands were aligned to the template alignment with the program Catalyst.<sup>5</sup> This approach was taken as a reference for the development of a pharmacophore model with our own program SQUID. The molecules DFU and celecoxib were not included in the SQUID pharmacophore model, because they are close analogues of rofecoxib and SC-558. The 2D structures of the remaining molecules are shown in Scheme 1<a data-tab="pane-pcw-Figures" href="#jm031139yh00001" id="" class=" internalNav"></a>. Crystal structures 1CX2, 3PGH, and 4COX were aligned with the homology alignment tool of MOE.<sup>5</sup> Rofecoxib and M5 were aligned to this template alignment with the flexible alignment tool of MOE. First, rofecoxib was aligned to the fixed template alignment. Then, M5 was aligned to the fixed alignment resulted from the previous step. For the final alignment, the unspecific inhibitors were removed. The resulting alignment of COX-2 inhibitors is shown in Figure 2<a data-tab="pane-pcw-Figures" href="#jm031139yf00002" id="" class=" internalNav"></a>. In accordance with the model of Palomer et al. the crucial pharmacophore features of these molecules are the sulfonyl group and the two aromatic six-membered rings.<sup>19</sup> The aromatic rings close to the sulfonyl group, <a name="jm004701904658" id="jm004701904658"></a> further referred to as “ring A”, are nearly parallel to each other in the model. The angles between the planes of the distant aromatic rings, further referred to as “ring B”, seem to be less constrained. The least conserved region of the model is the linker region between the two aromatic ring centers. <div id="_i15"><figure id="jm031139yh00001" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yh00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00001.jpeg" id="_i16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Scheme 1.   2D Structures of the Known Active COX-2 Inhibitors Used for the Calculation of the SQUID Pharmacophore Model </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yh00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm031139yf00002" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00002.jpeg" id="_i17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Three-dimensional alignment of the COX-2 inhibitors. Rofecoxib and M5 were aligned to the crystal structure conformation of SC-558 bound to COX-2. According to Palomer et al., essential interactions for specific COX-2 inhibitors are the aromatic rings A and B and the sulfonyl group.<sup>19</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">SQUID pharmacophore models were calculated with cluster radii from 0.5 Å to 3.5 Å in steps of 0.1 Å. A sample set of these pharmacophore models is shown in Figure 3<a data-tab="pane-pcw-Figures" href="#jm031139yf00003" id="" class=" internalNav"></a>. The models consisted of only three generalized interaction types:  hydrogen-bond donors, hydrogen-bond acceptors, and hydrophobic interactions. The model resulting from 1 Å cluster radius is the most detailed one. Here atoms in close proximity are combined to PPPs, which results in a low abstraction from the chemical scaffolds. In contrast to all other models shown, the preferred angle between the two aromatic rings A and B are preserved in this model. The models resulting from 1.5 and 2.0 Å exhibit a higher degree of generalization from molecular structure. Many atoms, especially in the regions of the aromatic rings A and B, were combined to form large PPPs, covering several atoms from each of the molecules. Up to 2.0 Å only hydrophobic atoms were combined. The models from the cluster radii 2.5 Å and 3.0 Å still represent the overall shape of the molecular alignment with three hydrophobic PPPs, but in the 3.5 Å model the shape of the alignment is only marginally visible. In all models with a cluster radius up to 2.0 Å the sulfonyl group is represented by two highly conserved hydrogen-bond acceptor PPPs, one hydrogen-bond donor PPP, and one hydrophobic PPP. In the models resulting from cluster radii greater than 2.0 Å all oxygen atoms of the sulfonyl group are represented by a single large PPP. Moreover, the hydrophobic PPP vanished since the methyl group was assigned to the PPP of ring A. <div id="_i18"><figure id="jm031139yf00003" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00003.jpeg" id="_i19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Pharmacophore models calculated with different cluster radii from the aligned COX-2 inhibitors. The cluster radii of the six shown models are (a) 1.0 Å, (b) 1.5 Å, (c) 2.0 Å, (d) 2.5 Å, (e) 3.0 Å, and (f) 3.5 Å. The colored spheres are representing the PPPs of the models. The radii of the spheres denote the standard deviations of the spatial distributions of the atoms of each PPP. Yellow PPPs represent hydrophobic interactions, magenta PPPs represent hydrogen-bond donors, and cyan PPPs represent hydrogen-bond acceptors. The intensity of the color of a PPP denotes the conservation of the PPP among the aligned molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>Retrospective Screening for COX-2 Inhibitors.</b> As the results of retrospective screening were very sensitive to the feature-type weights (data not shown), we decided to perform a restrained exhaustive search for the optimization of these weights. For every calculated model, each of the feature-type weights for features present in the pharmacophore model was varied from 0.1 to 0.5 in steps of 0.1, which resulted in 125 different weighting schemes for the COX-2 pharmacophore models. Each of the resulting descriptors was evaluated by retrospective screening. To obtain statistically more significant results, five different subsets of the COBRA database were created. For each of the subsets 50% of actives and 50% of inactives were randomly chosen from the original database for retrospective screening. </div><div class="NLM_p">The results of the optimization procedure are shown in Figure 4<a data-tab="pane-pcw-Figures" href="#jm031139yf00004" id="" class=" internalNav"></a>. For each model calculated with a different cluster radius, the average enrichment factors for the first 1%, 5%, and 10% of the five ranked databases obtained with the best found weighting scheme are shown. The highest average enrichment factor of 39 for the first 1% of the database was obtained with the model calculated with a cluster radius of 1.4 Å and feature-<a name="jm004701904659" id="jm004701904659"></a> type weights of 0.1 for hydrogen-bond donors, 0.4 for hydrogen-bond acceptors, and 0.3 for hydrophobic interactions. <div id="_i20"><figure id="jm031139yf00004" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00004.jpeg" id="_i21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Results of the retrospective screenings for COX-2 inhibitors with the pharmacophore models calculated with cluster radii from 0.5 Å to 3.5 Å with the optimized feature-type weights of each model. The shown enrichment factors are average values from screening of five randomly selected subsets of the COBRA database. The enrichment factors are calculated from the first 1% in part a, for the first 5% in part b and for the first 10% in part c. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">As it could be anticipated, the standard deviations of the enrichment factors were the highest for the first 1% and decreased for the first 5% and 10%. Nevertheless according to their standard deviations the enrichment factors for the first 1% of the database still seem to be appropriate for an evaluation of our pharmacophore models. All three curves exhibit the same general characteristics for different cluster radii, although the differences between the models vanish more and more considering the enrichment of the first 5% and 10% of the database (Figure 4). </div><div class="NLM_p">Considering the performance of the models for the enrichment in the first 1% of the database, large enrichment factors could be obtained for all models with a cluster radius from 0.5 Å to 2.4 Å. As can be seen in Figure 3, these models only differ in the description of the hydrophobic interactions, while models with 2.5 Å and greater cluster radii differ from the other models in the description of the oxygen atoms of the sulfonyl group. The models with a large cluster radius use a single PPP for the description of these atoms while the models with small cluster radii use two PPPs. It seems that a single PPP for the description of these oxygen atoms is not sufficient for a reasonably performing pharmacophore model. The models from 0.5 Å to 2.4 Å can be divided into four groups. The pharmacophore models of the first group from 0.5 Å to 0.9 Å with enrichment factors of roughly 27 consist only of PPPs merging atoms from different molecules within close spatial proximity, e.g., all aromatic rings are described by six PPPs. From 1.0 Å to 1.2 Å, a minimum in the performance of the models was observed. In these models, ring A is represented by six PPPs, and ring B is represented by four or five PPPs, which might not be an adequate number for the description of an aromatic six-ring. </div><div class="NLM_p">The three best performing models were obtained with cluster radii of 1.3 Å to 1.5 Å. Both models from 1.4 Å and 1.5 Å describe ring A with a single PPP and ring B with three and two PPPs, respectively. Like within the poorly performing models employing cluster radii from 1.0 Å to 1.2 Å, in the model obtained with a cluster radius of 1.3 Å, ring B is represented by four PPPs, but ring A is represented by three PPPs. The larger tolerances of the three PPPs of ring A might have compensated the unfavorable description of ring B. Within the models from 1.6 Å to 2.4 Å, the hydrophobic interactions are represented by a decreasing number of five to three hydrophobic PPPs. </div><div class="NLM_p">For comparison, a pharmacophore model was calculated including the two additional COX-2 inhibitors DFU and celecoxib from the model of Palomer et al.<sup>19</sup> A slightly better ef for the first 1% of the database (ef = 40) was obtained with a model calculated with a cluster radius of 1.5 Å and feature-type weights of 0.2 for hydrogen-bond donors, 0.5 for hydrogen-bond acceptors, and 0.5 for hydrophobic interactions (data not shown). </div><div class="NLM_p">To test if our approach for the optimization of feature-type weights is also valid in situations with significantly fewer reference molecules, we repeated the optimization procedure with only the molecules from the pharmacophore model as reference molecules for assessment of the enrichment capabilities of the SQUID models. For all models with cluster radii from 0.5 Å to 2.4 Å, several weighting schemes were found that ranked two of the three reference molecules into the first percent of the database. In no case were found all three molecules in the first percent. Ranking of all models according to eq 7 resulted in four similarly top-scoring 1.4 Å models with different weighting schemes. Among these models, the previously found best working model was found, with feature-type weights of 0.1 for hydrogen-bond donors, 0.4 for hydrogen-bond acceptors, and 0.3 for hydrophobic interactions. The worst of the other three models still resulted in an ef of 34, screening the database with the 92 COX-2 inhibitors. </div><div class="NLM_p">To compare our method with another established method, we performed retrospective screenings with the molecules from which the pharmacophore models were calculated. For this approach, we encoded these molecules with the CATS3D descriptor, but without scaling the descriptor to a maximum of 1. The database molecules were scored by the Euclidean distance to the query molecule, and the database was sorted according to the calculated distances to the query molecule. A comparison of the results of the similarity search with the results obtained from the best SQUID model is shown in Figure 5<a data-tab="pane-pcw-Figures" href="#jm031139yf00005" id="" class=" internalNav"></a>. Rofecoxib performed best in comparison to the other two COX-2 inhibitors. This might be a consequence of its comparably small size. The <a name="jm004701904660" id="jm004701904660"></a> pharmacophore model performed better than rofecoxib for the first 15% of the database. With the SQUID approach 75% of the active COX-2 inhibitors were ranked into the first 6% of the database. In comparison, rofecoxib retrieved 75% of the actives among the top 16% of the ranked database. Interestingly, the performance of the pharmacophore model decreased significantly for the last 25% of the active molecules in comparison to the COX-2 inhibitors. <div id="_i22"><figure id="jm031139yf00005" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00005.jpeg" id="_i23" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 Comparison of the enrichment curves of the best found pharmacophore model resulting from 1.4 Å cluster radius with the results of the retrospective screening with the CATS3D descriptors of the COX-2 inhibitors used for the model calculation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>Pharmacophore Model of Thrombin Ligands.</b> A diverse set of seven noncovalent, nonpeptidic thrombin inhibitors was adopted from Patel et al<i>.</i><sup>20</sup> The 2D structures of these molecules are shown in Scheme 2<a data-tab="pane-pcw-Figures" href="#jm031139yh00002" id="" class=" internalNav"></a>. All ligands were aligned by superposition of the protein structures with the homology alignment tool of MOE. The resulting alignment of the thrombin inhibitors is shown in Figure 6<a data-tab="pane-pcw-Figures" href="#jm031139yf00006" id="" class=" internalNav"></a>. According to Patel and co-workers the major interactions are B, H1, H2, and H3, where B is a basic interaction which interacts with the carboxylic group of Asp189. H1, H2, and H3 are hydrophobic interactions. Less conserved interactions are D1 and A1, where D1 is a hydrogen-bond donor and A1 is a hydrogen-bond acceptor. SQUID pharmacophore models were calculated from the 3D alignment with cluster radii from 0.5 Å to 3.5 Å within steps of 0.1 Å. <div id="_i24"><figure id="jm031139yh00002" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yh00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00002.jpeg" id="_i25" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 2.   2D Structures of the Known Active Thrombin Inhibitors Used for the Calculation of the SQUID Pharmacophore Model<i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> The names beneath the molecules are the pdb identifiers of the protein structures from which the conformations of these molecules were extracted.</p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yh00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm031139yf00006" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00006.jpeg" id="_i26" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Three-dimensional alignment of the thrombin inhibitors. The molecules were aligned by superposition of their appropriate protein structures. All essential interactions with the receptor, according to Patel et al., are indicated.<sup>20</sup> B is a basic interaction, H1, H2, and H3 are hydrophobic interactions, A1 is a hydrogen-bond acceptor, and D1 is a hydrogen-bond donor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">A sample set of the resulting models is shown in Figure 7<a data-tab="pane-pcw-Figures" href="#jm031139yf00007" id="" class=" internalNav"></a>. Four generalized interaction types were found in the ligands based on the ph4_aType function of MOE:  hydrogen-bond acceptor, hydrogen-bond donor, polar, and hydrophobic. Since all ligands were presented in neutralized state, interaction B was not identified as cationic feature; instead it was represented by hydrogen-bond donor and polar interactions and an additional hydrogen-bond acceptor. In the 1.0 Å and 1.5 Å models the description of the three hydrophobic interactions H1, H2, and H3 is very detailed using a large number of PPPs. With a cluster radius of 2.0 Å, only four PPPs are left. In the models with cluster radii of 2.5 Å, 3.0 Å, and 3.5 Å, these hydrophobic interactions are represented by only three PPPs. Both A1 and D1 are structurally conserved features in the alignment. All appropriate atoms from the different molecules lie in near proximity to each other. A1 is represented by a small conserved PPP in all models except for the 3.5 Å model, where it is represented by a large PPP, including other hydrogen-bond acceptors. D1 is also represented by a small conserved PPP except for the models with 3.0 Å and 3.5 Å cluster radius. <div id="_i27"><figure id="jm031139yf00007" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00007.jpeg" id="_i28" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Pharmacophore models calculated with different cluster radii from the aligned thrombin inhibitors. The cluster radii of the six shown models are (a) 1.0 Å, (b) 1.5 Å, (c) 2.0 Å, (d) 2.5 Å, (e) 3.0 Å, and (f) 3.5 Å. Yellow PPPs represent hydrophobic interactions, magenta PPPs represent hydrogen-bond donors, cyan PPPs represent hydrogen-bond acceptors, and green PPPs represent polar interaction. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>Retrospective Screening for Thrombin Inhibitors.</b> For retrospective screening with the SQUID pharmacophore models obtained from the alignment of thrombin inhibitors, the same procedure for feature-type weight optimization was applied as for the screening for COX-2 inhibitors. For the thrombin optimization, 625 weighting schemes had to be evaluated per model. </div><div class="NLM_p">The results of the optimization procedure are shown in Figure 8<a data-tab="pane-pcw-Figures" href="#jm031139yf00008" id="" class=" internalNav"></a>. The best average enrichment factor of 18 for the first 1% of the database was obtained with the model calculated with a cluster radius of 2.0 Å and feature-type weights of 0.4 for polar, 0.5 for hydrogen-bond donors, 0.3 for hydrogen-bond acceptors, and 0.5 for hydrophobic interactions. <div id="_i29"><figure id="jm031139yf00008" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00008.jpeg" id="_i30" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 Results of the retrospective screenings for thrombin inhibitors with the pharmacophore models calculated with cluster radii from 0.5 Å to 3.5 Å with the optimized feature-type weights of each model. The shown enrichment factors are average values from screening of five randomly selected subsets of the COBRA database. The enrichment factors are calculated from the first 1% in a, for the first 5% in b and for the first 10% in c. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">We detected two peaks for each of the enrichment factors, one for models with a high degree of generalization with cluster radii from 2.0 Å to 2.2 Å, and one for <a name="jm004701904661" id="jm004701904661"></a> models with a lower degree of generalization with cluster radii of 1.0 Å and 1.1 Å. Interestingly, models with cluster radii greater than 2.8 Å performed very well, too. As can be seen in Figure 7, the model with a cluster radius of 1.0 Å from the first peak mainly clustered atoms within near proximity into PPPs, while already favoring conserved atoms. The model with a cluster radius of 2.0 Å from the second peak represents the features with a drastically diminished overall number of PPPs. In particular the three hydrophobic interactions are represented by four PPPs, in contrast to all other models with a smaller cluster radius. The models resulting from cluster radii larger than 2.8 Å consist mostly of PPPs with large tolerances, but unlike for COX-2, these PPPs represent the shape of the molecular alignment very well. </div><div class="NLM_p">Like for COX-2, the optimization procedure was repeated with only the molecules from the pharmacophore model as reference molecules. For many models, weighting schemes were found which ranked two of the seven reference molecules into the first 1% of the database. In no case more molecules were found in the first 1%. Ranking of all models according to eq 7 resulted in the previously found best working 2.0 Å model with feature-type weights of 0.4 for polar interactions, 0.5 for hydrogen-bond donors, 0.4 for hydrogen-bond acceptors, and 0.5 for hydrophobic interactions. </div><div class="NLM_p">The result of the 2.0 Å SQUID model was compared with results from retrospective screening with CATS3D descriptors calculated from the molecules used for the calculation of the pharmacophore model. Enrichment curves are shown in Figure 9<a data-tab="pane-pcw-Figures" href="#jm031139yf00009" id="" class=" internalNav"></a>. Major differences were observed in the performance of the individual thrombin inhibitors. The inhibitors from the crystal structures 1FPC and 1DWD performed best. The three inhibitors from structures 1D4P, 1D9I, and 1TOM performed even worse than a random distribution of active molecules within some regions of the ranked database. The SQUID pharmacophore model performed better than the most successful similarity search for the first 40% of the database. Fifty percent of the active molecules were ranked into the first 6% of the database by the <a name="jm004701904662" id="jm004701904662"></a> pharmacophore model in comparison to the best thrombin inhibitor from 1DWD, which ranked 50% of the active molecules into the first 13% of the database. <div id="_i31"><figure id="jm031139yf00009" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00009.jpeg" id="_i32" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 Comparison of the enrichment curves of the best found pharmacophore model resulting from 2.0 Å cluster radius with the results of the retrospective screening with the CATS3D descriptors of the thrombin inhibitors used for the model calculation. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>Method Performance.</b> For an additional comparison of the SQUID pharmacophore model with an established method, we calculated pharmacophore models from the two alignments of COX-2 and thrombin reference compounds with the pharmacophore tool of MOE.<sup>8</sup> For both models, we used the atom type scheme PCH_ALL which consists of atom types for cationic, anionic, hydrogen-bond donor, hydrogen-bond acceptor, aromatic ring centers, and hydrophobic interactions. In contrast to SQUID, one PPP in MOE can describe multiple atom types, which can be combined by logic operators. As a starting point for the alignments, pharmacophore models were calculated automatically with the consensus pharmacophore function using MOE default parameters. This function clusters features into PPPs which are more conserved than a threshold value. For the threshold, 50% conservation was used. Retrospective screening with these first pharmacophore models was very slow, and the program failed to screen the whole database. As a consequence, we modified the models manually by removing PPPs which were not among the key features of the pharmacophore models published by Palomer et al<i>.</i><sup>19</sup> or Patel et al<i>.</i><sup>20</sup> respectively (Figure 2, Figure 6). For the thrombin model the radii and the positions of the PPPs for H1, H2, and D1 were manually adjusted for a more accurate representation of the underlying ring structures and the cluster of hydrogen-bond donors. Additional multiple features of the PPPs were also removed. The resulting MOE pharmacophore models are shown in Figure 10<a data-tab="pane-pcw-Figures" href="#jm031139yf00010" id="" class=" internalNav"></a>. Both models were evaluated by retrospective screening of the COBRA database. <div id="_i33"><figure id="jm031139yf00010" data-index="12" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00010.jpeg" id="_i34" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 MOE Pharmacophore models for COX-2 (a) and thrombin (b). In the COX-2 pharmacophore model the rings A and B are represented by two aromatic ring center PPPs, and the sulfonyl group is represented by two acceptor PPPs and one PPP for a donor or hydrophobic interaction. In the thrombin pharmacophore model the hydrophobic interactions H1 and H2 are represented by hydrophobic PPPs while H3 is represented by an aromatic PPP. For A1 a hydrogen-bond acceptor PPP and for D1 a hydrogen-bond donor PPP was found. The basic interaction B was represented by two PPPs, one for hydrogen-bond acceptor or hydrogen-bond donor, and one for hydrogen-bond acceptor and hydrogen-bond donor. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">With the MOE COX-2 model (Figure 10a), we retrieved 84 matching molecules among which we found 49 (58%) of the known COX-2 inhibitors. In comparison, the COX-2 SQUID model found 47 (56%) active molecules in the first 84 compounds from the ranked database. Reinsertion of a PPP from the first MOE model, which represents the central five-ring of the COX-2 inhibitors by an acceptor, aromatic, or hydrophobic interaction, resulted in 48 actives (91%) out of 53 matches. Within the first 53 molecules of the ranked database, the SQUID pharmacophore model retrieved only 38 (72%) active compounds. A comparison of the actives found by MOE and SQUID showed that the overlap was only 25 molecules, i.e., that both methods complement each other very well. SQUID retrieved an additional 13 actives which were missed by the refined MOE model. </div><div class="NLM_p">With the MOE thrombin model (Figure 10b), we retrieved five actives (31%) among 16 matches, in comparison to the SQUID model which retrieved 13 actives (81%) among the first 16 molecules of the sorted database. Retrospective screening with the partial match option of the MOE pharmacophore search function requiring only six of the seven PPPs as matching criterion resulted in 489 matches including 87 (18%) thrombin inhibitors. With SQUID 119 actives (24%) were found among the first 489 molecules of the ranked database. The two sets of actives have 64 molecules in common. Again, we conclude that the two pharmacophore searching approaches complement each other. </div><div class="NLM_p">To gain further confidence in our approach, we took a look at the two top-scoring nonactive molecules from each of the best pharmacophore models for COX-2 and thrombin (Scheme 3<a data-tab="pane-pcw-Figures" href="#jm031139yh00003" id="" class=" internalNav"></a>). Molecules <b>11</b><sup><a onclick="showRef(event, 'jm031139yb00021'); return false;" href="javascript:void(0);" class="ref jm031139yb00021">21</a></sup> and <b>12</b><sup><a onclick="showRef(event, 'jm031139yb00022'); return false;" href="javascript:void(0);" class="ref jm031139yb00022">22</a></sup> were found with the COX-2 pharmacophore model with 1.4 Å cluster radius, and molecules <b>13</b><sup><a onclick="showRef(event, 'jm031139yb00023'); return false;" href="javascript:void(0);" class="ref jm031139yb00023">23</a></sup> and <b>14</b><sup><a onclick="showRef(event, 'jm031139yb00024'); return false;" href="javascript:void(0);" class="ref jm031139yb00024">24</a></sup> were found with the thrombin pharmacophore model with 2.0 Å cluster radius. Ethoxzolamide (<b>12</b>) is an inhibitor of carbonic anhydrase. Also, it has been shown recently <a name="jm004701904663" id="jm004701904663"></a> that celecoxib is a nanomolar inhibitor of carbonic anhydrase.<sup><a onclick="showRef(event, 'jm031139yb00025'); return false;" href="javascript:void(0);" class="ref jm031139yb00025">25</a></sup> EMATE (<b>11</b>) is an inhibitor of estrone sulfatase, and a nanomolar inhibitory effect of EMATE on carbonic anhydrase activity has been reported.<sup><a onclick="showRef(event, 'jm031139yb00026'); return false;" href="javascript:void(0);" class="ref jm031139yb00026">26</a></sup> This indicates that both “nonactive” molecules share common features with the COX-2 inhibitors from the pharmacophore model. According to our knowledge, no COX-2 activity has been reported for the molecules <b>11</b> and <b>12</b>, and we think it would be worthwhile testing them. <div id="_i35"><figure id="jm031139yh00003" data-index="13" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yh00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00003.jpeg" id="_i36" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 3.   2D Structures of the Best Scored Nonactive Molecules Found with the Best Pharmacophore Models<i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> Compounds <b>11</b> and <b>12</b> were found with the COX-2 pharmacophore model with 1.4 Å cluster radius. Compounds <b>13</b> and <b>14</b> were found with the thrombin model derived from 2.0 Å cluster radius. </p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yh00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">Molecule <b>13</b> (BOC-<span class="smallcaps smallerCapital">d</span>-Arg-Pro-Arg) is an inhibitor of Factor Xa for which nanomolar inhibition of thrombin has been reported.<sup>23</sup> It thus represents a real hit. BIBP3226 (<b>14</b>) is an antagonist of the neuropeptide Y<sub>1</sub> receptor. To our knowledge, thrombin activity has not been tested for this molecule. Nevertheless obvious similarities to the thrombin inhibitors are present in BIBP3226. </div><div class="NLM_p last">Summarizing, we demonstrated that the SQUID approach can complement existing pharmacophore methods by retrieving additional active compounds. The SQUID technique contains many parameters that must be tuned appropriately, which certainly is a disadvantage of the method. In future implementations we intend to address this issue. In particular, the use of “feature-type weights” in addition to “conservation weights” adds complexity to the models which we intend to eliminate. Still, it should be kept in mind that the feature-type weights allow for an optimization of a pharmacophore model in situations where many reference molecules are at hand. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66543" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66543" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We developed a fuzzy pharmacophore model approach for the compilation of focused screening libraries (“SQUID”). The method is characterized by the following points: </div><div class="NLM_p">• Information from molecular alignments of known active compounds is included in the pharmacophore models. </div><div class="NLM_p">• Features of the aligned molecules are represented by a set of spheres of Gaussian densities, weighted by the conservation of the respective feature at the location of the sphere. </div><div class="NLM_p">• The degree of fuzziness of the pharmacophore model can be varied, thereby affecting the number and size of PPP spheres. </div><div class="NLM_p">• Transformation of the pharmacophore model into a correlation vector of probabilities allows for a rapid database screening that does not depend on aligning database molecules to the pharmacophore model. In this way we obtained a quantitative approach for the ranking of compound databases with respect to putative biological activity. </div><div class="NLM_p last">We challenged our approach using inhibitors of COX-2 and thrombin. For both classes of molecules, pharmacophore models were calculated with cluster radii from 0.5 Å to 3.5 Å and evaluated by retrospective screening. Although optimization of the relative weights of the feature-types in the correlation vectors was essential in some cases to obtain a satisfying enrichment, we showed that optimization on the basis of some few reference molecules can be sufficient. The best retrospective screening results for COX-2 were obtained with the model resulting from a cluster radius of 1.4 Å, yielding an enrichment factor of 39 for the first 1% of the ranked database. For thrombin, the best results for the enrichment in the first 1% of the database were obtained with the model resulting from a cluster radius of 2.0 Å, yielding ef = 18. For both targets, the best models outperformed retrospective screening by CATS3D similarity searching. This showed that, independent from the overall enrichment and thus independent of the explicit selection of active molecules, the pharmacophore model outperformed conventional similarity searching. In comparison to conventional pharmacophore searching with MOE, SQUID identified additional actives. We demonstrated that the SQUID pharmacophore model approach provides a potentially useful new method for virtual screening. The inherent fuzzy description of the molecules should support the goal of ‘scaffold hopping', especially with higher degrees of fuzziness. </div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74084" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74084" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steffen Renner</span> - <span class="hlFld-Affiliation affiliation">Johann Wolfgang Goethe-Universität, Institut für Organische Chemie und Chemische Biologie, Marie-Curie-Strasse 11, D-60439 Frankfurt, Germany </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gisbert Schneider</span> - <span class="hlFld-Affiliation affiliation">Johann Wolfgang Goethe-Universität, Institut für Organische Chemie und Chemische Biologie, Marie-Curie-Strasse 11, D-60439 Frankfurt, Germany </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm031139y" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the modlab-team for help and stimulating discussions, especially Jürgen Paetz for carefully reading the manuscript and Norbert Dichter for maintenance of the computer cluster. This work was supported by the Beilstein-Institut zur Förderung der Chemischen Wissenschaften. </p></div><div class="refs-header-label"><a name="jm004701904664" id="jm004701904664"></a></div><p class="references-count">
            This article references 26 other publications.
        </p><ol id="references" class="useLabel"><li id="jm031139yb00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm031139yb00001_1"><span><span class="NLM_string-ref">Böhm, H.-J., Schneider, G., Eds. <i>  Virtual Screening for Bioactive Molecules</i>; Wiley-VCH:  Weinheim, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm031139yb00001/jm031139yb00001_1&amp;dbid=16&amp;doi=10.1021%2Fjm031139y&amp;key=10.1002%2F9783527613083" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=H.-J.+B%C3%B6hm&author=G.+Schneider&title=Virtual+Screening+for+Bioactive+Molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00001_1&amp;dbid=16384&amp;doi=10.1002%2F9783527613083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527613083%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26btitle%3DVirtual%2520Screening%2520for%2520Bioactive%2520Molecules%26pub%3DWiley-VCH%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm031139yb00002_1"><span><span class="NLM_string-ref">Xue, L.; Bajorath, J. Molecular descriptors in chemoinformatics, computational combinatorial chemistry, and virtual screening, <i>Comb. Chem. High </i><i>Throughput Screen</i>. <b>2000</b>, <i>3</i>, 363−372.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2000&pages=372&author=L.+Xue&author=J+Bajorath&title=Molecular+descriptors+in+chemoinformatics%2C+computational+combinatorial+chemistry%2C+and+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DL.%26atitle%3DMolecular%2520descriptors%2520in%2520chemoinformatics%252C%2520computational%2520combinatorial%2520chemistry%252C%2520and%2520virtual%2520screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screen.%26date%3D2000%26volume%3D3%26spage%3D372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm031139yb00003_1"><span><span class="NLM_string-ref">Guner, O., Ed. <i>  Pharmacophore Perception, Development and Use in Drug </i><i>Design</i>; International University Line:  La Jolla CA, 2000.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=O.+Guner&title=Pharmacophore+Perception%2C+Development+and+Use+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGuner%26aufirst%3DO.%26btitle%3DPharmacophore%2520Perception%252C%2520Development%2520and%2520Use%2520in%2520Drug%2520Design%26pub%3DInternational%2520University%2520Line%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm031139yb00004_1"><span><span class="NLM_string-ref">Pickett, S. The biophore concept. In <i>Protein</i><i>−</i><i>Ligand Interactions</i>; Böhm, H.-J., Schneider, G., Eds.; Wiley-VCH:  Weinheim, New York, 2003; pp 73−105.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&pages=105&author=S.+Pickett&title=Protein%E2%88%92Ligand+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPickett%26aufirst%3DS.%26atitle%3DThe%2520biophore%2520concept%26btitle%3DProtein%25E2%2588%2592Ligand%2520Interactions%26pub%3DWiley-VCH%26date%3D2003%26spage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm031139yb00005_1"><span><span class="NLM_string-ref">Greene, J.; Kahn, S.; Savoj, H.; Sprague, P.; Teig, S. Chemical function queries for 3D database search. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>1994</b>, <i>34</i>, 1297−1308.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1994&pages=1308&author=J.+Greene&author=S.+Kahn&author=H.+Savoj&author=P.+Sprague&author=S+Teig&title=Chemical+function+queries+for+3D+database+search"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreene%26aufirst%3DJ.%26atitle%3DChemical%2520function%2520queries%2520for%25203D%2520database%2520search%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1994%26volume%3D34%26spage%3D1308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm031139yb00006_1"><span><span class="NLM_string-ref">Martin, Y. C.; Bures, M. G.; Danaher, E. A.; DeLazzer, J.; Lico, I.; Pavlik, P. A. A fast approach to pharmacophore mapping and its application to dopaminergic and benzodiazepine agonists. <i>  J. Comput:-Aided Mol. Des.</i><b>1993</b>, <i>7</i>, 83−102.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1993&pages=102&author=Y.+C.+Martin&author=M.+G.+Bures&author=E.+A.+Danaher&author=J.+DeLazzer&author=I.+Lico&author=P.+A+Pavlik&title=A+fast+approach+to+pharmacophore+mapping+and+its+application+to+dopaminergic+and+benzodiazepine+agonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DA%2520fast%2520approach%2520to%2520pharmacophore%2520mapping%2520and%2520its%2520application%2520to%2520dopaminergic%2520and%2520benzodiazepine%2520agonists%26jtitle%3DJ.%2520Comput%253A-Aided%2520Mol.%2520Des.%26date%3D1993%26volume%3D7%26spage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm031139yb00007_1"><span><span class="NLM_string-ref">Jones, G.; Willett, P.; Glen, R. C. A genetic algorithm for flexible molecular overlay and pharmacophore elucidation. <i>  J. Comput:-Aided Mol. Des.</i><b>1995</b>, <i>9</i>, 532−549.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=549&author=G.+Jones&author=P.+Willett&author=R.+C+Glen&title=A+genetic+algorithm+for+flexible+molecular+overlay+and+pharmacophore+elucidation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26atitle%3DA%2520genetic%2520algorithm%2520for%2520flexible%2520molecular%2520overlay%2520and%2520pharmacophore%2520elucidation%26jtitle%3DJ.%2520Comput%253A-Aided%2520Mol.%2520Des.%26date%3D1995%26volume%3D9%26spage%3D549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm031139yb00008_1"><span><span class="NLM_string-ref">MOE, Molecular Operating Environment. Distributor:  Chemical Computing Group, 1010 Sherbrooke St. West, #910, Montreal, Canada H3A.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MOE%2C+Molecular+Operating+Environment.+Distributor%3A%E2%80%89+Chemical+Computing+Group%2C+1010+Sherbrooke+St.+West%2C+%23910%2C+Montreal%2C+Canada+H3A.+MOE%2C+Molecular+Operating+Environment.+Distributor%3A%E2%80%89+Chemical+Computing+Group%2C+1010+Sherbrooke+St.+West%2C+%23910%2C+Montreal%2C+Canada+H3A."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3D%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm031139yb00009_1"><span><span class="NLM_string-ref">Vedani, A.; Dobler, M. Multidimensional QSAR in drug research. Predicting binding affinities, toxicity and pharmacokinetic parameters. <i>  Prog. Drug </i><i>Res</i>. <b>2000</b>, <i>55</i>, 105−135.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2000&pages=135&author=A.+Vedani&author=M+Dobler&title=Multidimensional+QSAR+in+drug+research.+Predicting+binding+affinities%2C+toxicity+and+pharmacokinetic+parameters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVedani%26aufirst%3DA.%26atitle%3DMultidimensional%2520QSAR%2520in%2520drug%2520research.%2520Predicting%2520binding%2520affinities%252C%2520toxicity%2520and%2520pharmacokinetic%2520parameters%26jtitle%3DProg.%2520Drug%2520Res.%26date%3D2000%26volume%3D55%26spage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00010"><div class="reference"><div class="NLM_citation" id="jm031139yb00010_1"><span><span class="NLM_string-ref">Mason, J. S.; Cheney, D. L. Library design and virtual screening using multiple 4-point pharmacophore fingerprints. <i>  Pac. Symp. Biocomput</i>. <b>2000</b>, 576−587.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&conference=Pac.+Symp.+Biocomput.&author=J.+S.+Mason&author=D.+L.+Cheney&title=Library+design+and+virtual+screening+using+multiple+4-point+pharmacophore+fingerprints"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DLibrary%2520design%2520and%2520virtual%2520screening%2520using%2520multiple%25204-point%2520pharmacophore%2520fingerprints%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm031139yb00011_1"><span><span class="NLM_string-ref">Schneider, G.; Neidhart, W.; Giller, T.; Schmid, G. “Scaffold-Hopping” by topological pharmacophore search:  a contribution to virtual screening. <i>  Angew. Chem., Int. Ed. </i><b>1999</b>, <i>38</i>, 2894−2896</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=2896&author=G.+Schneider&author=W.+Neidhart&author=T.+Giller&author=G+Schmid&title=Scaffold-Hopping%E2%80%9D+by+topological+pharmacophore+search%3A%E2%80%89+a+contribution+to+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DScaffold-Hopping%25E2%2580%259D%2520by%2520topological%2520pharmacophore%2520search%253A%25E2%2580%2589%2520a%2520contribution%2520to%2520virtual%2520screening%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D2896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm031139yb00012_1"><span><span class="NLM_string-ref">Xue, L.; Stahura, F. L.; Godden, J. W.; Bajorath, J. Fingerprint scaling increases the probability of identifying molecules with similar activity in virtual screening calculations. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci.</i><b>2001</b>, <i>41</i>, 746−753.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=753&author=L.+Xue&author=F.+L.+Stahura&author=J.+W.+Godden&author=J+Bajorath&title=Fingerprint+scaling+increases+the+probability+of+identifying+molecules+with+similar+activity+in+virtual+screening+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DL.%26atitle%3DFingerprint%2520scaling%2520increases%2520the%2520probability%2520of%2520identifying%2520molecules%2520with%2520similar%2520activity%2520in%2520virtual%2520screening%2520calculations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm031139yb00013_1"><span><span class="NLM_string-ref">Horvath, D.; Mao, B. Neighbourhood behaviour. Fuzzy molecular descriptors and their influence on the relationship between structural similarity and property similarity. <i>  QSAR Comb. Sci.</i><b>2003</b>, <i>22</i>, 498−509.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=509&author=D.+Horvath&author=B+Mao&title=Neighbourhood+behaviour.+Fuzzy+molecular+descriptors+and+their+influence+on+the+relationship+between+structural+similarity+and+property+similarity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHorvath%26aufirst%3DD.%26atitle%3DNeighbourhood%2520behaviour.%2520Fuzzy%2520molecular%2520descriptors%2520and%2520their%2520influence%2520on%2520the%2520relationship%2520between%2520structural%2520similarity%2520and%2520property%2520similarity%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2003%26volume%3D22%26spage%3D509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm031139yb00014_1"><span><span class="NLM_string-ref">Fechner, U.; Franke, L.; Renner, S.; Schneider, P.; Schneider, P.; Schneider, G. Comparison of correlation vector methods for ligand-based similarity searching. <i>  J. Comput.-Aided Mol. Des</i>. <b>2003</b>, <i>17</i>, 687−698.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=698&author=U.+Fechner&author=L.+Franke&author=S.+Renner&author=P.+Schneider&author=P.+Schneider&author=G+Schneider&title=Comparison+of+correlation+vector+methods+for+ligand-based+similarity+searching"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFechner%26aufirst%3DU.%26atitle%3DComparison%2520of%2520correlation%2520vector%2520methods%2520for%2520ligand-based%2520similarity%2520searching%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2003%26volume%3D17%26spage%3D698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm031139yb00015_1"><span><span class="NLM_string-ref">Good, A. C.; Kuntz, I. D. Investigating the extension of pairwise distance pharmacophore measures to triplet-based descriptors. <i>  J. Comput.-Aided </i><i>Mol. Des.</i><b>1995</b>, <i>9</i>, 373−379.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=379&author=A.+C.+Good&author=I.+D+Kuntz&title=Investigating+the+extension+of+pairwise+distance+pharmacophore+measures+to+triplet-based+descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGood%26aufirst%3DA.%2BC.%26atitle%3DInvestigating%2520the%2520extension%2520of%2520pairwise%2520distance%2520pharmacophore%2520measures%2520to%2520triplet-based%2520descriptors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1995%26volume%3D9%26spage%3D379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm031139yb00016_1"><span><span class="NLM_string-ref">Mason, J. S.; Morize, I.; Menard, P. R.; Cheney, D. L.; Hulme, C.; Labaudiniere, R. F. New 4-point pharmacophore method for molecular similarity and diversity applications:  Overview of the method and applications, including a novel approach to the design of combinatorial libraries containing privileged substructures. <i>  J. Med. Chem.</i><b>1999</b>, <i>42</i>, 3251−3264.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=3264&author=J.+S.+Mason&author=I.+Morize&author=P.+R.+Menard&author=D.+L.+Cheney&author=C.+Hulme&author=R.+F+Labaudiniere&title=New+4-point+pharmacophore+method+for+molecular+similarity+and+diversity+applications%3A%E2%80%89+Overview+of+the+method+and+applications%2C+including+a+novel+approach+to+the+design+of+combinatorial+libraries+containing+privileged+substructures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DNew%25204-point%2520pharmacophore%2520method%2520for%2520molecular%2520similarity%2520and%2520diversity%2520applications%253A%25E2%2580%2589%2520Overview%2520of%2520the%2520method%2520and%2520applications%252C%2520including%2520a%2520novel%2520approach%2520to%2520the%2520design%2520of%2520combinatorial%2520libraries%2520containing%2520privileged%2520substructures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D3264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm031139yb00017_1"><span><span class="NLM_string-ref">Schneider, P.; Schneider, G. Collection of bioactive reference compounds for focused library design. <i>  QSAR Comb. Sci</i>. <b>2003</b>, <i>22</i>, 713−718.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=718&author=P.+Schneider&author=G+Schneider&title=Collection+of+bioactive+reference+compounds+for+focused+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DP.%26atitle%3DCollection%2520of%2520bioactive%2520reference%2520compounds%2520for%2520focused%2520library%2520design%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2003%26volume%3D22%26spage%3D718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm031139yb00018_1"><span><span class="NLM_string-ref">CORINA. Distributor:  Molecular Networks GmbH, Computerchemie, Nägelsbachstrasse 25, D-91052 Erlangen, Germany, <a class="ext-link" href="http://www.mol-net.de/" target="_blank">http://www.mol-net.de/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CORINA.+Distributor%3A%E2%80%89+Molecular+Networks+GmbH%2C+Computerchemie%2C+N%C3%A4gelsbachstrasse+25%2C+D-91052+Erlangen%2C+Germany%2C+http%3A%2F%2Fwww.mol-net.de%2F.+CORINA.+Distributor%3A%E2%80%89+Molecular+Networks+GmbH%2C+Computerchemie%2C+N%C3%A4gelsbachstrasse+25%2C+D-91052+Erlangen%2C+Germany%2C+http%3A%2F%2Fwww.mol-net.de%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.mol-net.de%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm031139yb00019_1"><span><span class="NLM_string-ref">Palomer, A.; Cabre, F.; Pascual, J.; Campos, J.; Trujillo, M.; Entrena, A.; Gallo, M.; Garcia, L.; Mauleon, D.; Espinosa, A. Identification of novel cyclooxygenase-2 selective inhibitors using pharmacophore models. <i>  J. </i><i>Med. Chem. </i><b>2002</b>, <i>45</i>, 1402−1411.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1411&author=A.+Palomer&author=F.+Cabre&author=J.+Pascual&author=J.+Campos&author=M.+Trujillo&author=A.+Entrena&author=M.+Gallo&author=L.+Garcia&author=D.+Mauleon&author=A+Espinosa&title=Identification+of+novel+cyclooxygenase-2+selective+inhibitors+using+pharmacophore+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPalomer%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520novel%2520cyclooxygenase-2%2520selective%2520inhibitors%2520using%2520pharmacophore%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00020"><div class="reference"><div class="NLM_citation" id="jm031139yb00020_1"><span><span class="NLM_string-ref">Patel, Y.; Gillet, V. J.; Bravi, G.; Leach, A. R. A Comparison of the pharmacophore identification programs:  Catalyst, DISCO and GASP. <i>  J. </i><i>Comput.-Aided Mol. Des. </i><b>2002</b>, <i>16</i>, 653−681.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=681&author=Y.+Patel&author=V.+J.+Gillet&author=G.+Bravi&author=A.+R+Leach&title=A+Comparison+of+the+pharmacophore+identification+programs%3A%E2%80%89+Catalyst%2C+DISCO+and+GASP"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26atitle%3DA%2520Comparison%2520of%2520the%2520pharmacophore%2520identification%2520programs%253A%25E2%2580%2589%2520Catalyst%252C%2520DISCO%2520and%2520GASP%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm031139yb00021_1"><span><span class="NLM_string-ref">Woo, L. W.; Howarth, N. M.; Purohit, A.; Hejaz, H. A.; Reed, M. J.; Potter, B. V. Steroidal and nonsteroidal sulfamates as potent inhibitors of steroid sulfatase. <i>  J. Med. Chem.</i><b>1998</b>, <i>41</i>, 1068−1083.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1083&author=L.+W.+Woo&author=N.+M.+Howarth&author=A.+Purohit&author=H.+A.+Hejaz&author=M.+J.+Reed&author=B.+V+Potter&title=Steroidal+and+nonsteroidal+sulfamates+as+potent+inhibitors+of+steroid+sulfatase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWoo%26aufirst%3DL.%2BW.%26atitle%3DSteroidal%2520and%2520nonsteroidal%2520sulfamates%2520as%2520potent%2520inhibitors%2520of%2520steroid%2520sulfatase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm031139yb00022_1"><span><span class="NLM_string-ref">Supuran, C. T.; Scozzafava, A.; Casini, A. Carbonic anhydrase inhibitors. <i>  Med. </i><i>Res. Rev.</i><b>2003</b>, <i>23</i>, 146−189.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=189&author=C.+T.+Supuran&author=A.+Scozzafava&author=A+Casini&title=Carbonic+anhydrase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26atitle%3DCarbonic%2520anhydrase%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2003%26volume%3D23%26spage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm031139yb00023_1"><span><span class="NLM_string-ref">Marlowe, C. K.; Sinha, U.; Gunn, A. C.; Scarborough, R. M. Design, synthesis and structure−activity relationship of a series of arginine aldehyde factor Xa inhibitors. Part 1:  structures based on the (D)-Arg-Gly-Arg tripeptide sequence. <i>  Bioorg. Med. Chem. Lett.</i><b>2000</b>, <i>10</i>, 13−16.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=16&author=C.+K.+Marlowe&author=U.+Sinha&author=A.+C.+Gunn&author=R.+M+Scarborough&title=Design%2C+synthesis+and+structure%E2%88%92activity+relationship+of+a+series+of+arginine+aldehyde+factor+Xa+inhibitors.+Part+1%3A%E2%80%89+structures+based+on+the+%28D%29-Arg-Gly-Arg+tripeptide+sequence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarlowe%26aufirst%3DC.%2BK.%26atitle%3DDesign%252C%2520synthesis%2520and%2520structure%25E2%2588%2592activity%2520relationship%2520of%2520a%2520series%2520of%2520arginine%2520aldehyde%2520factor%2520Xa%2520inhibitors.%2520Part%25201%253A%25E2%2580%2589%2520structures%2520based%2520on%2520the%2520%2528D%2529-Arg-Gly-Arg%2520tripeptide%2520sequence%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00024"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00024'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="jm031139yb00024_1"><span><span class="NLM_string-ref">Rudolf, K.; Eberlein, W.; Engel, W.; Wieland, H. A.; Willim, K. D.; Entzeroth, M.; Wienen, W.; Beck-Sickinger, A. G.; Doods, H. N. The first highly potent and selective non-peptide neuropeptide Y Y1 receptor antagonist:  BIBP3226. <i>  Eur. J. Pharmacol.</i><b>1994</b>, <i>271</i>, R11−R13.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1994&pages=R13&author=K.+Rudolf&author=W.+Eberlein&author=W.+Engel&author=H.+A.+Wieland&author=K.+D.+Willim&author=M.+Entzeroth&author=W.+Wienen&author=A.+G.+Beck-Sickinger&author=H.+N+Doods&title=The+first+highly+potent+and+selective+non-peptide+neuropeptide+Y+Y1+receptor+antagonist%3A%E2%80%89+BIBP3226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DK.%26atitle%3DThe%2520first%2520highly%2520potent%2520and%2520selective%2520non-peptide%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%253A%25E2%2580%2589%2520BIBP3226%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1994%26volume%3D271%26spage%3DR13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00025"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00025'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="jm031139yb00025_1"><span><span class="NLM_string-ref">Weber, A.; Casini, A.; Heine, A.; Kuhn, D.; Supuran, C. T.; Scozzafava, A.; Klebe, G. Unexpected nanomolar inhibition of carbonic anhydrase by COX-2-selective celecoxib:  new pharmacological opportunities due to related binding site recognition. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 550−557.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=557&author=A.+Weber&author=A.+Casini&author=A.+Heine&author=D.+Kuhn&author=C.+T.+Supuran&author=A.+Scozzafava&author=G+Klebe&title=Unexpected+nanomolar+inhibition+of+carbonic+anhydrase+by+COX-2-selective+celecoxib%3A%E2%80%89+new+pharmacological+opportunities+due+to+related+binding+site+recognition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DA.%26atitle%3DUnexpected%2520nanomolar%2520inhibition%2520of%2520carbonic%2520anhydrase%2520by%2520COX-2-selective%2520celecoxib%253A%25E2%2580%2589%2520new%2520pharmacological%2520opportunities%2520due%2520to%2520related%2520binding%2520site%2520recognition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm031139yb00026"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm031139yb00026'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="jm031139yb00026_1"><span><span class="NLM_string-ref">Ho, Y. T.; Purohit, A.; Vicker, N.; Newman, S. P.; Robinson, J. J.; Leese, M. P.; Ganeshapillai, D.; Woo, L. W.; Potter, B. V.; Reed, M. J. Inhibition of carbonic anhydrase II by steroidal and nonsteroidal sulphamates. <i>  Biochem. </i><i>Biophys. Res. Commun.</i><b>2003</b>, <i>305</i>, 909−914.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2003&pages=914&author=Y.+T.+Ho&author=A.+Purohit&author=N.+Vicker&author=S.+P.+Newman&author=J.+J.+Robinson&author=M.+P.+Leese&author=D.+Ganeshapillai&author=L.+W.+Woo&author=B.+V.+Potter&author=M.+J+Reed&title=Inhibition+of+carbonic+anhydrase+II+by+steroidal+and+nonsteroidal+sulphamates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DY.%2BT.%26atitle%3DInhibition%2520of%2520carbonic%2520anhydrase%2520II%2520by%2520steroidal%2520and%2520nonsteroidal%2520sulphamates%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2003%26volume%3D305%26spage%3D914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm031139yb00001':['jm031139yb00001_1'],'jm031139yb00002':['jm031139yb00002_1'],'jm031139yb00003':['jm031139yb00003_1'],'jm031139yb00004':['jm031139yb00004_1'],'jm031139yb00005':['jm031139yb00005_1'],'jm031139yb00006':['jm031139yb00006_1'],'jm031139yb00007':['jm031139yb00007_1'],'jm031139yb00008':['jm031139yb00008_1'],'jm031139yb00009':['jm031139yb00009_1'],'jm031139yb00010':['jm031139yb00010_1'],'jm031139yb00011':['jm031139yb00011_1'],'jm031139yb00012':['jm031139yb00012_1'],'jm031139yb00013':['jm031139yb00013_1'],'jm031139yb00014':['jm031139yb00014_1'],'jm031139yb00015':['jm031139yb00015_1'],'jm031139yb00016':['jm031139yb00016_1'],'jm031139yb00017':['jm031139yb00017_1'],'jm031139yb00018':['jm031139yb00018_1'],'jm031139yb00019':['jm031139yb00019_1'],'jm031139yb00020':['jm031139yb00020_1'],'jm031139yb00021':['jm031139yb00021_1'],'jm031139yb00022':['jm031139yb00022_1'],'jm031139yb00023':['jm031139yb00023_1'],'jm031139yb00024':['jm031139yb00024_1'],'jm031139yb00025':['jm031139yb00025_1'],'jm031139yb00026':['jm031139yb00026_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 40 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Tobias Fehlmann, <span class="NLM_string-name hlFld-ContribAuthor">Michael C. Hutter</span>. </span><span class="cited-content_cbyCitation_article-title">Conservation and Relevance of Pharmacophore Point Types. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (4)
                                     , 1314-1323. <a href="https://doi.org/10.1021/acs.jcim.8b00757" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00757</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00757%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DConservation%252Band%252BRelevance%252Bof%252BPharmacophore%252BPoint%252BTypes%26aulast%3DFehlmann%26aufirst%3DTobias%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D29102018%26date%3D12032019%26date%3D26022019%26volume%3D59%26issue%3D4%26spage%3D1314%26epage%3D1323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jan  Dreher</span>, <span class="hlFld-ContribAuthor ">Josef  Scheiber</span>, <span class="hlFld-ContribAuthor ">Nikolaus  Stiefl</span>, and <span class="hlFld-ContribAuthor ">Knut  Baumann</span>  . </span><span class="cited-content_cbyCitation_article-title">xMaP—An Interpretable Alignment-Free Four-Dimensional Quantitative Structure–Activity Relationship Technique Based on Molecular Surface Properties and Conformer Ensembles. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (1)
                                     , 165-181. <a href="https://doi.org/10.1021/acs.jcim.7b00419" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00419%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DxMaP%2525E2%252580%252594An%252BInterpretable%252BAlignment-Free%252BFour-Dimensional%252BQuantitative%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BTechnique%252BBased%252Bon%252BMolecular%252BSurface%252BProperties%252Band%252BConformer%252BEnsembles%26aulast%3DDreher%26aufirst%3DJan%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D13072017%26date%3D02012018%26date%3D22012018%26date%3D27112017%26volume%3D58%26issue%3D1%26spage%3D165%26epage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marijn P. A.  Sanders</span>, <span class="hlFld-ContribAuthor ">Arménio J. M.  Barbosa</span>, <span class="hlFld-ContribAuthor ">Barbara  Zarzycka</span>, <span class="hlFld-ContribAuthor ">Gerry A.F.  Nicolaes</span>, <span class="hlFld-ContribAuthor ">Jan P.G.  Klomp</span>, <span class="hlFld-ContribAuthor ">Jacob  de Vlieg</span>, and <span class="hlFld-ContribAuthor ">Alberto  Del Rio</span>  . </span><span class="cited-content_cbyCitation_article-title">Comparative Analysis of Pharmacophore Screening Tools. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (6)
                                     , 1607-1620. <a href="https://doi.org/10.1021/ci2005274" title="DOI URL">https://doi.org/10.1021/ci2005274</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci2005274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci2005274%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComparative%252BAnalysis%252Bof%252BPharmacophore%252BScreening%252BTools%26aulast%3DSanders%26aufirst%3DMarijn%2BP.%2BA.%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D04112011%26date%3D13062012%26date%3D25062012%26date%3D30052012%26volume%3D52%26issue%3D6%26spage%3D1607%26epage%3D1620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marijn P. A.  Sanders</span>, <span class="hlFld-ContribAuthor ">Stefan  Verhoeven</span>, <span class="hlFld-ContribAuthor ">Chris   de Graaf</span>, <span class="hlFld-ContribAuthor ">Luc  Roumen</span>, <span class="hlFld-ContribAuthor ">Bas  Vroling</span>, <span class="hlFld-ContribAuthor ">Sander B.  Nabuurs</span>, <span class="hlFld-ContribAuthor ">Jacob   de Vlieg</span>, and <span class="hlFld-ContribAuthor ">Jan P. G.  Klomp</span>  . </span><span class="cited-content_cbyCitation_article-title">Snooker: A Structure-Based Pharmacophore Generation Tool Applied to Class A GPCRs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (9)
                                     , 2277-2292. <a href="https://doi.org/10.1021/ci200088d" title="DOI URL">https://doi.org/10.1021/ci200088d</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200088d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200088d%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSnooker%25253A%252BA%252BStructure-Based%252BPharmacophore%252BGeneration%252BTool%252BApplied%252Bto%252BClass%252BA%252BGPCRs%26aulast%3DSanders%26aufirst%3DMarijn%2BP.%2BA.%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D21022011%26date%3D06092011%26date%3D26092011%26date%3D25082011%26volume%3D51%26issue%3D9%26spage%3D2277%26epage%3D2292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caterina  Barillari</span>, <span class="hlFld-ContribAuthor ">Gilles  Marcou</span> and <span class="hlFld-ContribAuthor ">Didier  Rognan</span>  . </span><span class="cited-content_cbyCitation_article-title">Hot-Spots-Guided Receptor-Based Pharmacophores (HS-Pharm): A Knowledge-Based Approach to Identify Ligand-Anchoring Atoms in Protein Cavities and Prioritize Structure-Based Pharmacophores. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (7)
                                     , 1396-1410. <a href="https://doi.org/10.1021/ci800064z" title="DOI URL">https://doi.org/10.1021/ci800064z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci800064z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci800064z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHot-Spots-Guided%252BReceptor-Based%252BPharmacophores%252B%252528HS-Pharm%252529%25253A%252BA%252BKnowledge-Based%252BApproach%252Bto%252BIdentify%252BLigand-Anchoring%252BAtoms%252Bin%252BProtein%252BCavities%252Band%252BPrioritize%252BStructure-Based%252BPharmacophores%26aulast%3DBarillari%26aufirst%3DCaterina%26date%3D2008%26date%3D2008%26date%3D2008%26date%3D28%252007%25202008%26date%3D21%252006%25202008%26date%3D21%252002%25202008%26volume%3D48%26issue%3D7%26spage%3D1396%26epage%3D1410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lukas  Friedrich</span>, <span class="hlFld-ContribAuthor ">Gino  Cingolani</span>, <span class="hlFld-ContribAuthor ">Ying‐Hui  Ko</span>, <span class="hlFld-ContribAuthor ">Mariaclara  Iaselli</span>, <span class="hlFld-ContribAuthor ">Morena  Miciaccia</span>, <span class="hlFld-ContribAuthor ">Maria Grazia  Perrone</span>, <span class="hlFld-ContribAuthor ">Konstantin  Neukirch</span>, <span class="hlFld-ContribAuthor ">Veronika  Bobinger</span>, <span class="hlFld-ContribAuthor ">Daniel  Merk</span>, <span class="hlFld-ContribAuthor ">Robert Klaus  Hofstetter</span>, <span class="hlFld-ContribAuthor ">Oliver  Werz</span>, <span class="hlFld-ContribAuthor ">Andreas  Koeberle</span>, <span class="hlFld-ContribAuthor ">Antonio  Scilimati</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Learning from Nature: From a Marine Natural Product to Synthetic Cyclooxygenase‐1 Inhibitors by Automated De Novo Design. </span><span class="cited-content_cbyCitation_journal-name">Advanced Science</span><span> <strong>2021,</strong> <em>21 </em>, 2100832. <a href="https://doi.org/10.1002/advs.202100832" title="DOI URL">https://doi.org/10.1002/advs.202100832</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/advs.202100832&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadvs.202100832%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Science%26atitle%3DLearning%252Bfrom%252BNature%25253A%252BFrom%252Ba%252BMarine%252BNatural%252BProduct%252Bto%252BSynthetic%252BCyclooxygenase%2525E2%252580%2525901%252BInhibitors%252Bby%252BAutomated%252BDe%252BNovo%252BDesign%26aulast%3DFriedrich%26aufirst%3DLukas%26date%3D2021%26date%3D2021%26volume%3D21%26spage%3D2100832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Javier  Pérez-Sianes</span>, <span class="hlFld-ContribAuthor ">Horacio  Pérez-Sánchez</span>, <span class="hlFld-ContribAuthor ">Fernando  Díaz</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening Meets Deep Learning. </span><span class="cited-content_cbyCitation_journal-name">Current Computer-Aided Drug Design</span><span> <strong>2018,</strong> <em>15 </em>
                                    (1)
                                     , 6-28. <a href="https://doi.org/10.2174/1573409914666181018141602" title="DOI URL">https://doi.org/10.2174/1573409914666181018141602</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573409914666181018141602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573409914666181018141602%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Computer-Aided%2520Drug%2520Design%26atitle%3DVirtual%252BScreening%252BMeets%252BDeep%252BLearning%26aulast%3DP%25C3%25A9rez-Sianes%26aufirst%3DJavier%26date%3D2018%26volume%3D15%26issue%3D1%26spage%3D6%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sivakumar Prasanth  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Receptor pharmacophore ensemble (REPHARMBLE): a probabilistic pharmacophore modeling approach using multiple protein-ligand complexes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2018,</strong> <em>24 </em>
                                    (10)
                                     <a href="https://doi.org/10.1007/s00894-018-3820-7" title="DOI URL">https://doi.org/10.1007/s00894-018-3820-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-018-3820-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-018-3820-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DReceptor%252Bpharmacophore%252Bensemble%252B%252528REPHARMBLE%252529%25253A%252Ba%252Bprobabilistic%252Bpharmacophore%252Bmodeling%252Bapproach%252Busing%252Bmultiple%252Bprotein-ligand%252Bcomplexes%26aulast%3DKumar%26aufirst%3DSivakumar%2BPrasanth%26date%3D2018%26date%3D2018%26volume%3D24%26issue%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian P.  Koch</span>, <span class="hlFld-ContribAuthor ">Michael  Reutlinger</span>, <span class="hlFld-ContribAuthor ">Nickolay  Todoroff</span>, <span class="hlFld-ContribAuthor ">Petra  Schneider</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">CATS for Scaffold Hopping in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 119-130. <a href="https://doi.org/10.1002/9783527665143.ch07" title="DOI URL">https://doi.org/10.1002/9783527665143.ch07</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527665143.ch07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527665143.ch07%26sid%3Dliteratum%253Aachs%26atitle%3DCATS%252Bfor%252BScaffold%252BHopping%252Bin%252BMedicinal%252BChemistry%26aulast%3DKoch%26aufirst%3DChristian%2BP.%26date%3D2013%26date%3D2013%26spage%3D119%26epage%3D130%26pub%3DWiley%26atitle%3DScaffold%252BHopping%252Bin%252BMedicinal%252BChemistry%26aulast%3DBrown%26aufirst%3DNathan%26date%3D2013%26date%3D2013%26volume%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">De novo design – hop(p)ing against hope. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2013,</strong> <em>10 </em>
                                    (4)
                                     , e453-e460. <a href="https://doi.org/10.1016/j.ddtec.2012.06.001" title="DOI URL">https://doi.org/10.1016/j.ddtec.2012.06.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2012.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2012.06.001%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DDe%252Bnovo%252Bdesign%252B%2525E2%252580%252593%252Bhop%252528p%252529ing%252Bagainst%252Bhope%26aulast%3DSchneider%26aufirst%3DGisbert%26date%3D2013%26volume%3D10%26issue%3D4%26spage%3De453%26epage%3De460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marijn P. A.  Sanders</span>, <span class="hlFld-ContribAuthor ">Ross  McGuire</span>, <span class="hlFld-ContribAuthor ">Luc  Roumen</span>, <span class="hlFld-ContribAuthor ">Iwan J. P.  de Esch</span>, <span class="hlFld-ContribAuthor ">Jacob  de Vlieg</span>, <span class="hlFld-ContribAuthor ">Jan P. G.  Klomp</span>, <span class="hlFld-ContribAuthor ">Chris  de Graaf</span>. </span><span class="cited-content_cbyCitation_article-title">From the protein's perspective: the benefits and challenges of protein structure-based pharmacophore modeling. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2012,</strong> <em>3 </em>
                                    (1)
                                     , 28-38. <a href="https://doi.org/10.1039/C1MD00210D" title="DOI URL">https://doi.org/10.1039/C1MD00210D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C1MD00210D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC1MD00210D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DFrom%252Bthe%252Bprotein%252527s%252Bperspective%25253A%252Bthe%252Bbenefits%252Band%252Bchallenges%252Bof%252Bprotein%252Bstructure-based%252Bpharmacophore%252Bmodeling%26aulast%3DSanders%26aufirst%3DMarijn%2BP.%2BA.%26date%3D2012%26date%3D2012%26volume%3D3%26issue%3D1%26spage%3D28%26epage%3D38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Similarity searching. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2011,</strong> <em>1 </em>
                                    (2)
                                     , 260-282. <a href="https://doi.org/10.1002/wcms.23" title="DOI URL">https://doi.org/10.1002/wcms.23</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.23%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DSimilarity%252Bsearching%26aulast%3DStumpfe%26aufirst%3DDagmar%26date%3D2011%26date%3D2011%26volume%3D1%26issue%3D2%26spage%3D260%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dorota A.  Urbanek</span>, <span class="hlFld-ContribAuthor ">Ewgenij  Proschak</span>, <span class="hlFld-ContribAuthor ">Yusuf  Tanrikulu</span>, <span class="hlFld-ContribAuthor ">Steffi  Becker</span>, <span class="hlFld-ContribAuthor ">Michael  Karas</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Scaffold-hopping from aminoglycosides to small synthetic inhibitors of bacterial protein biosynthesis using a pseudoreceptor model. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2011,</strong> <em>2 </em>
                                    (3)
                                     , 181. <a href="https://doi.org/10.1039/c0md00207k" title="DOI URL">https://doi.org/10.1039/c0md00207k</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/c0md00207k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2Fc0md00207k%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DScaffold-hopping%252Bfrom%252Baminoglycosides%252Bto%252Bsmall%252Bsynthetic%252Binhibitors%252Bof%252Bbacterial%252Bprotein%252Bbiosynthesis%252Busing%252Ba%252Bpseudoreceptor%252Bmodel%26aulast%3DUrbanek%26aufirst%3DDorota%2BA.%26date%3D2011%26volume%3D2%26issue%3D3%26spage%3D181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander  Klenner</span>, <span class="hlFld-ContribAuthor ">Markus  Hartenfeller</span>, <span class="hlFld-ContribAuthor ">Petra  Schneider</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">‘Fuzziness’ in pharmacophore-based virtual screening and de novo design. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2010,</strong> <em>7 </em>
                                    (4)
                                     , e237-e244. <a href="https://doi.org/10.1016/j.ddtec.2010.10.004" title="DOI URL">https://doi.org/10.1016/j.ddtec.2010.10.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2010.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2010.10.004%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3D%2525E2%252580%252598Fuzziness%2525E2%252580%252599%252Bin%252Bpharmacophore-based%252Bvirtual%252Bscreening%252Band%252Bde%252Bnovo%252Bdesign%26aulast%3DKlenner%26aufirst%3DAlexander%26date%3D2010%26volume%3D7%26issue%3D4%26spage%3De237%26epage%3De244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haitao  Ji</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Design: Considerations for Good ADME Properties. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2010,</strong>,, 417-485. <a href="https://doi.org/10.1002/9780470915110.ch11" title="DOI URL">https://doi.org/10.1002/9780470915110.ch11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9780470915110.ch11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9780470915110.ch11%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BDrug%252BDesign%25253A%252BConsiderations%252Bfor%252BGood%252BADME%252BProperties%26aulast%3DJi%26aufirst%3DHaitao%26date%3D2010%26date%3D2010%26spage%3D417%26epage%3D485%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26atitle%3DADMET%252Bfor%252BMedicinal%252BChemists%26aulast%3DTsaioun%26aufirst%3DKatya%26date%3D2011%26volume%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah R.  Langdon</span>, <span class="hlFld-ContribAuthor ">Peter  Ertl</span>, <span class="hlFld-ContribAuthor ">Nathan  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Bioisosteric Replacement and Scaffold Hopping in Lead Generation and Optimization. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2010,</strong> <em>29 </em>
                                    (5)
                                     , 366-385. <a href="https://doi.org/10.1002/minf.201000019" title="DOI URL">https://doi.org/10.1002/minf.201000019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201000019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201000019%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DBioisosteric%252BReplacement%252Band%252BScaffold%252BHopping%252Bin%252BLead%252BGeneration%252Band%252BOptimization%26aulast%3DLangdon%26aufirst%3DSarah%2BR.%26date%3D2010%26date%3D2010%26volume%3D29%26issue%3D5%26spage%3D366%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Weisel</span>, <span class="hlFld-ContribAuthor ">Jan M.  Kriegl</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Architectural Repertoire of Ligand-Binding Pockets on Protein Surfaces. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2010,</strong> <em>11 </em>
                                    (4)
                                     , 556-563. <a href="https://doi.org/10.1002/cbic.200900604" title="DOI URL">https://doi.org/10.1002/cbic.200900604</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.200900604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.200900604%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DArchitectural%252BRepertoire%252Bof%252BLigand-Binding%252BPockets%252Bon%252BProtein%252BSurfaces%26aulast%3DWeisel%26aufirst%3DMartin%26date%3D2010%26volume%3D11%26issue%3D4%26spage%3D556%26epage%3D563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lisa  Peltason</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic computational analysis of structure–activity relationships: concepts, challenges and recent advances. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2009,</strong> <em>1 </em>
                                    (3)
                                     , 451-466. <a href="https://doi.org/10.4155/fmc.09.41" title="DOI URL">https://doi.org/10.4155/fmc.09.41</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.09.41&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.09.41%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSystematic%252Bcomputational%252Banalysis%252Bof%252Bstructure%2525E2%252580%252593activity%252Brelationships%25253A%252Bconcepts%25252C%252Bchallenges%252Band%252Brecent%252Badvances%26aulast%3DPeltason%26aufirst%3DLisa%26date%3D2009%26volume%3D1%26issue%3D3%26spage%3D451%26epage%3D466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yusuf  Tanrikulu</span>, <span class="hlFld-ContribAuthor ">Ewgenij  Proschak</span>, <span class="hlFld-ContribAuthor ">Tim  Werner</span>, <span class="hlFld-ContribAuthor ">Tim  Geppert</span>, <span class="hlFld-ContribAuthor ">Nickolay  Todoroff</span>, <span class="hlFld-ContribAuthor ">Alexander  Klenner</span>, <span class="hlFld-ContribAuthor ">Tim  Kottke</span>, <span class="hlFld-ContribAuthor ">Kerstin  Sander</span>, <span class="hlFld-ContribAuthor ">Erich  Schneider</span>, <span class="hlFld-ContribAuthor ">Roland  Seifert</span>, <span class="hlFld-ContribAuthor ">Holger  Stark</span>, <span class="hlFld-ContribAuthor ">Timothy  Clark</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Homology Model Adjustment and Ligand Screening with a Pseudoreceptor of the Human Histamine H
              4
              Receptor. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2009,</strong> <em>4 </em>
                                    (5)
                                     , 820-827. <a href="https://doi.org/10.1002/cmdc.200800443" title="DOI URL">https://doi.org/10.1002/cmdc.200800443</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.200800443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.200800443%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DHomology%252BModel%252BAdjustment%252Band%252BLigand%252BScreening%252Bwith%252Ba%252BPseudoreceptor%252Bof%252Bthe%252BHuman%252BHistamine%252BH%252B4%252BReceptor%26aulast%3DTanrikulu%26aufirst%3DYusuf%26date%3D2009%26volume%3D4%26issue%3D5%26spage%3D820%26epage%3D827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Madhu  Chopra</span>, <span class="hlFld-ContribAuthor ">Ruby  Gupta</span>, <span class="hlFld-ContribAuthor ">Swati  Gupta</span>, <span class="hlFld-ContribAuthor ">Daman  Saluja</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular modeling study on chemically diverse series of cyclooxygenase-2 selective inhibitors: generation of predictive pharmacophore model using Catalyst. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2008,</strong> <em>14 </em>
                                    (11)
                                     , 1087-1099. <a href="https://doi.org/10.1007/s00894-008-0350-8" title="DOI URL">https://doi.org/10.1007/s00894-008-0350-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-008-0350-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-008-0350-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DMolecular%252Bmodeling%252Bstudy%252Bon%252Bchemically%252Bdiverse%252Bseries%252Bof%252Bcyclooxygenase-2%252Bselective%252Binhibitors%25253A%252Bgeneration%252Bof%252Bpredictive%252Bpharmacophore%252Bmodel%252Busing%252BCatalyst%26aulast%3DChopra%26aufirst%3DMadhu%26date%3D2008%26date%3D2008%26volume%3D14%26issue%3D11%26spage%3D1087%26epage%3D1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deepak  Bandyopadhyay</span>, <span class="hlFld-ContribAuthor ">Dimitris K.  Agrafiotis</span>. </span><span class="cited-content_cbyCitation_article-title">A self-organizing algorithm for molecular alignment and pharmacophore development. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2008,</strong> <em>29 </em>
                                    (6)
                                     , 965-982. <a href="https://doi.org/10.1002/jcc.20854" title="DOI URL">https://doi.org/10.1002/jcc.20854</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.20854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.20854%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DA%252Bself-organizing%252Balgorithm%252Bfor%252Bmolecular%252Balignment%252Band%252Bpharmacophore%252Bdevelopment%26aulast%3DBandyopadhyay%26aufirst%3DDeepak%26date%3D2008%26date%3D2008%26volume%3D29%26issue%3D6%26spage%3D965%26epage%3D982" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sune  Askjaer</span>, <span class="hlFld-ContribAuthor ">Morten  Langgård</span>. </span><span class="cited-content_cbyCitation_article-title">Combining Pharmacophore Fingerprints and PLS-Discriminant Analysis for Virtual Screening and SAR Elucidation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (3)
                                     , 476-488. <a href="https://doi.org/10.1021/ci700356w" title="DOI URL">https://doi.org/10.1021/ci700356w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci700356w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci700356w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCombining%252BPharmacophore%252BFingerprints%252Band%252BPLS-Discriminant%252BAnalysis%252Bfor%252BVirtual%252BScreening%252Band%252BSAR%252BElucidation%26aulast%3DAskjaer%26aufirst%3DSune%26date%3D2008%26date%3D2008%26volume%3D48%26issue%3D3%26spage%3D476%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steffen  Renner</span>, <span class="hlFld-ContribAuthor ">Swetlana  Derksen</span>, <span class="hlFld-ContribAuthor ">Sebastian  Radestock</span>, <span class="hlFld-ContribAuthor ">Fabian  Mörchen</span>. </span><span class="cited-content_cbyCitation_article-title">Maximum Common Binding Modes (MCBM):  Consensus Docking Scoring Using Multiple Ligand Information and Interaction Fingerprints. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (2)
                                     , 319-332. <a href="https://doi.org/10.1021/ci7003626" title="DOI URL">https://doi.org/10.1021/ci7003626</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci7003626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci7003626%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMaximum%252BCommon%252BBinding%252BModes%252B%252528MCBM%252529%25253A%2525E2%252580%252589%252BConsensus%252BDocking%252BScoring%252BUsing%252BMultiple%252BLigand%252BInformation%252Band%252BInteraction%252BFingerprints%26aulast%3DRenner%26aufirst%3DSteffen%26date%3D2008%26date%3D2008%26volume%3D48%26issue%3D2%26spage%3D319%26epage%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yusuf  Tanrikulu</span>, <span class="hlFld-ContribAuthor ">Manuel  Nietert</span>, <span class="hlFld-ContribAuthor ">Ute  Scheffer</span>, <span class="hlFld-ContribAuthor ">Ewgenij  Proschak</span>, <span class="hlFld-ContribAuthor ">Kristina  Grabowski</span>, <span class="hlFld-ContribAuthor ">Petra  Schneider</span>, <span class="hlFld-ContribAuthor ">Markus  Weidlich</span>, <span class="hlFld-ContribAuthor ">Michael  Karas</span>, <span class="hlFld-ContribAuthor ">Michael  Göbel</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Scaffold Hopping by “Fuzzy” Pharmacophores and its Application to RNA Targets. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2007,</strong> <em>8 </em>
                                    (16)
                                     , 1932-1936. <a href="https://doi.org/10.1002/cbic.200700195" title="DOI URL">https://doi.org/10.1002/cbic.200700195</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.200700195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.200700195%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DScaffold%252BHopping%252Bby%252B%2525E2%252580%25259CFuzzy%2525E2%252580%25259D%252BPharmacophores%252Band%252Bits%252BApplication%252Bto%252BRNA%252BTargets%26aulast%3DTanrikulu%26aufirst%3DYusuf%26date%3D2007%26volume%3D8%26issue%3D16%26spage%3D1932%26epage%3D1936" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew C.  Good</span>. </span><span class="cited-content_cbyCitation_article-title">Novel DOCK clique driven 3D similarity database search tools for molecule shape matching and beyond: Adding flexibility to the search for ligand kin. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2007,</strong> <em>26 </em>
                                    (3)
                                     , 656-666. <a href="https://doi.org/10.1016/j.jmgm.2007.03.016" title="DOI URL">https://doi.org/10.1016/j.jmgm.2007.03.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2007.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2007.03.016%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DNovel%252BDOCK%252Bclique%252Bdriven%252B3D%252Bsimilarity%252Bdatabase%252Bsearch%252Btools%252Bfor%252Bmolecule%252Bshape%252Bmatching%252Band%252Bbeyond%25253A%252BAdding%252Bflexibility%252Bto%252Bthe%252Bsearch%252Bfor%252Bligand%252Bkin%26aulast%3DGood%26aufirst%3DAndrew%2BC.%26date%3D2007%26volume%3D26%26issue%3D3%26spage%3D656%26epage%3D666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">L  Fischer</span>, <span class="hlFld-ContribAuthor ">M  Hornig</span>, <span class="hlFld-ContribAuthor ">C  Pergola</span>, <span class="hlFld-ContribAuthor ">N  Meindl</span>, <span class="hlFld-ContribAuthor ">L  Franke</span>, <span class="hlFld-ContribAuthor ">Y  Tanrikulu</span>, <span class="hlFld-ContribAuthor ">G  Dodt</span>, <span class="hlFld-ContribAuthor ">G  Schneider</span>, <span class="hlFld-ContribAuthor ">D  Steinhilber</span>, <span class="hlFld-ContribAuthor ">O  Werz</span>. </span><span class="cited-content_cbyCitation_article-title">The molecular mechanism of the inhibition by licofelone of the biosynthesis of 5-lipoxygenase products. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Pharmacology</span><span> <strong>2007,</strong> <em>152 </em>
                                    (4)
                                     , 471-480. <a href="https://doi.org/10.1038/sj.bjp.0707416" title="DOI URL">https://doi.org/10.1038/sj.bjp.0707416</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/sj.bjp.0707416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fsj.bjp.0707416%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Pharmacology%26atitle%3DThe%252Bmolecular%252Bmechanism%252Bof%252Bthe%252Binhibition%252Bby%252Blicofelone%252Bof%252Bthe%252Bbiosynthesis%252Bof%252B5-lipoxygenase%252Bproducts%26aulast%3DFischer%26aufirst%3DL%26date%3D2007%26date%3D2009%26volume%3D152%26issue%3D4%26spage%3D471%26epage%3D480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ingo  Schellhammer</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>. </span><span class="cited-content_cbyCitation_article-title">TrixX: structure-based molecule indexing for large-scale virtual screening in sublinear time. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2007,</strong> <em>21 </em>
                                    (5)
                                     , 223-238. <a href="https://doi.org/10.1007/s10822-007-9103-5" title="DOI URL">https://doi.org/10.1007/s10822-007-9103-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-007-9103-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-007-9103-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DTrixX%25253A%252Bstructure-based%252Bmolecule%252Bindexing%252Bfor%252Blarge-scale%252Bvirtual%252Bscreening%252Bin%252Bsublinear%252Btime%26aulast%3DSchellhammer%26aufirst%3DIngo%26date%3D2007%26date%3D2007%26volume%3D21%26issue%3D5%26spage%3D223%26epage%3D238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregory A.  Landrum</span>, <span class="hlFld-ContribAuthor ">Julie E.  Penzotti</span>, <span class="hlFld-ContribAuthor ">Santosh  Putta</span>. </span><span class="cited-content_cbyCitation_article-title">Feature-map vectors: a new class of informative descriptors for computational drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2007,</strong> <em>20 </em>
                                    (12)
                                     , 751-762. <a href="https://doi.org/10.1007/s10822-006-9085-8" title="DOI URL">https://doi.org/10.1007/s10822-006-9085-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-006-9085-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-006-9085-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DFeature-map%252Bvectors%25253A%252Ba%252Bnew%252Bclass%252Bof%252Binformative%252Bdescriptors%252Bfor%252Bcomputational%252Bdrug%252Bdiscovery%26aulast%3DLandrum%26aufirst%3DGregory%2BA.%26date%3D2007%26date%3D2007%26volume%3D20%26issue%3D12%26spage%3D751%26epage%3D762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Good</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2007,</strong>,, 459-494. <a href="https://doi.org/10.1016/B0-08-045044-X/00262-5" title="DOI URL">https://doi.org/10.1016/B0-08-045044-X/00262-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B0-08-045044-X/00262-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB0-08-045044-X%2F00262-5%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%26aulast%3DGood%26aufirst%3DA.%26date%3D2007%26spage%3D459%26epage%3D494%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BII%26date%3D2007%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian N.  Parker</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Towards Unified Compound Screening Strategies: A Critical Evaluation of Error Sources in Experimental and Virtual High-Throughput Screening. </span><span class="cited-content_cbyCitation_journal-name">QSAR & Combinatorial Science</span><span> <strong>2006,</strong> <em>25 </em>
                                    (12)
                                     , 1153-1161. <a href="https://doi.org/10.1002/qsar.200610069" title="DOI URL">https://doi.org/10.1002/qsar.200610069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/qsar.200610069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fqsar.200610069%26sid%3Dliteratum%253Aachs%26jtitle%3DQSAR%2520%2526%2520Combinatorial%2520Science%26atitle%3DTowards%252BUnified%252BCompound%252BScreening%252BStrategies%25253A%252BA%252BCritical%252BEvaluation%252Bof%252BError%252BSources%252Bin%252BExperimental%252Band%252BVirtual%252BHigh-Throughput%252BScreening%26aulast%3DParker%26aufirst%3DChristian%25E2%2580%2585N.%26date%3D2006%26volume%3D25%26issue%3D12%26spage%3D1153%26epage%3D1161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>, <span class="hlFld-ContribAuthor ">Petra  Schneider</span>, <span class="hlFld-ContribAuthor ">Steffen  Renner</span>. </span><span class="cited-content_cbyCitation_article-title">Scaffold-Hopping: How Far Can You Jump?. </span><span class="cited-content_cbyCitation_journal-name">QSAR & Combinatorial Science</span><span> <strong>2006,</strong> <em>25 </em>
                                    (12)
                                     , 1162-1171. <a href="https://doi.org/10.1002/qsar.200610091" title="DOI URL">https://doi.org/10.1002/qsar.200610091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/qsar.200610091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fqsar.200610091%26sid%3Dliteratum%253Aachs%26jtitle%3DQSAR%2520%2526%2520Combinatorial%2520Science%26atitle%3DScaffold-Hopping%25253A%252BHow%252BFar%252BCan%252BYou%252BJump%25253F%26aulast%3DSchneider%26aufirst%3DGisbert%26date%3D2006%26volume%3D25%26issue%3D12%26spage%3D1162%26epage%3D1171" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine  Michaux</span>, <span class="hlFld-ContribAuthor ">Xavier  de Leval</span>, <span class="hlFld-ContribAuthor ">Fabien  Julémont</span>, <span class="hlFld-ContribAuthor ">Jean-Michel  Dogné</span>, <span class="hlFld-ContribAuthor ">Bernard  Pirotte</span>, <span class="hlFld-ContribAuthor ">François  Durant</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based pharmacophore of COX-2 selective inhibitors and identification of original lead compounds from 3D database searching method. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2006,</strong> <em>41 </em>
                                    (12)
                                     , 1446-1455. <a href="https://doi.org/10.1016/j.ejmech.2006.07.017" title="DOI URL">https://doi.org/10.1016/j.ejmech.2006.07.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2006.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2006.07.017%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bpharmacophore%252Bof%252BCOX-2%252Bselective%252Binhibitors%252Band%252Bidentification%252Bof%252Boriginal%252Blead%252Bcompounds%252Bfrom%252B3D%252Bdatabase%252Bsearching%252Bmethod%26aulast%3DMichaux%26aufirst%3DCatherine%26date%3D2006%26volume%3D41%26issue%3D12%26spage%3D1446%26epage%3D1455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander  Böcker</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>, <span class="hlFld-ContribAuthor ">Andreas  Teckentrup</span>. </span><span class="cited-content_cbyCitation_article-title">NIPALSTREE:  A New Hierarchical Clustering Approach for Large Compound Libraries and Its Application to Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2006,</strong> <em>46 </em>
                                    (6)
                                     , 2220-2229. <a href="https://doi.org/10.1021/ci050541d" title="DOI URL">https://doi.org/10.1021/ci050541d</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci050541d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci050541d%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNIPALSTREE%25253A%2525E2%252580%252589%252BA%252BNew%252BHierarchical%252BClustering%252BApproach%252Bfor%252BLarge%252BCompound%252BLibraries%252Band%252BIts%252BApplication%252Bto%252BVirtual%252BScreening%26aulast%3DB%25C3%25B6cker%26aufirst%3DAlexander%26date%3D2006%26date%3D2006%26volume%3D46%26issue%3D6%26spage%3D2220%26epage%3D2229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steffen  Renner</span>, <span class="hlFld-ContribAuthor ">Christof H.  Schwab</span>, <span class="hlFld-ContribAuthor ">Johann  Gasteiger</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of Conformational Flexibility on Three-Dimensional Similarity Searching Using Correlation Vectors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2006,</strong> <em>46 </em>
                                    (6)
                                     , 2324-2332. <a href="https://doi.org/10.1021/ci050075s" title="DOI URL">https://doi.org/10.1021/ci050075s</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci050075s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci050075s%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImpact%252Bof%252BConformational%252BFlexibility%252Bon%252BThree-Dimensional%252BSimilarity%252BSearching%252BUsing%252BCorrelation%252BVectors%26aulast%3DRenner%26aufirst%3DSteffen%26date%3D2006%26date%3D2006%26volume%3D46%26issue%3D6%26spage%3D2324%26epage%3D2332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John W  Davies</span>, <span class="hlFld-ContribAuthor ">Meir  Glick</span>, <span class="hlFld-ContribAuthor ">Jeremy L  Jenkins</span>. </span><span class="cited-content_cbyCitation_article-title">Streamlining lead discovery by aligning in silico and high-throughput screening. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2006,</strong> <em>10 </em>
                                    (4)
                                     , 343-351. <a href="https://doi.org/10.1016/j.cbpa.2006.06.022" title="DOI URL">https://doi.org/10.1016/j.cbpa.2006.06.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2006.06.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2006.06.022%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DStreamlining%252Blead%252Bdiscovery%252Bby%252Baligning%252Bin%252Bsilico%252Band%252Bhigh-throughput%252Bscreening%26aulast%3DDavies%26aufirst%3DJohn%2BW%26date%3D2006%26volume%3D10%26issue%3D4%26spage%3D343%26epage%3D351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ana G.  Maldonado</span>, <span class="hlFld-ContribAuthor ">J. P.  Doucet</span>, <span class="hlFld-ContribAuthor ">Michel  Petitjean</span>, <span class="hlFld-ContribAuthor ">Bo-Tao  Fan</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular similarity and diversity in chemoinformatics: From theory to applications. </span><span class="cited-content_cbyCitation_journal-name">Molecular Diversity</span><span> <strong>2006,</strong> <em>10 </em>
                                    (1)
                                     , 39-79. <a href="https://doi.org/10.1007/s11030-006-8697-1" title="DOI URL">https://doi.org/10.1007/s11030-006-8697-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11030-006-8697-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11030-006-8697-1%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Diversity%26atitle%3DMolecular%252Bsimilarity%252Band%252Bdiversity%252Bin%252Bchemoinformatics%25253A%252BFrom%252Btheory%252Bto%252Bapplications%26aulast%3DMaldonado%26aufirst%3DAna%2BG.%26date%3D2006%26volume%3D10%26issue%3D1%26spage%3D39%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andreas  Bender</span>, <span class="hlFld-ContribAuthor ">Jeremy L.  Jenkins</span>, <span class="hlFld-ContribAuthor ">Qingliang  Li</span>, <span class="hlFld-ContribAuthor ">Sam E.  Adams</span>, <span class="hlFld-ContribAuthor ">Edward O.  Cannon</span>, <span class="hlFld-ContribAuthor ">Robert C.  Glen</span>. </span><span class="cited-content_cbyCitation_article-title">Chapter 9 Molecular Similarity: Advances in Methods, Applications and Validations in Virtual Screening and QSAR. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2006,</strong>,, 141-168. <a href="https://doi.org/10.1016/S1574-1400(06)02009-3" title="DOI URL">https://doi.org/10.1016/S1574-1400(06)02009-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1574-1400(06)02009-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1574-1400%2806%2902009-3%26sid%3Dliteratum%253Aachs%26atitle%3DChapter%252B9%252BMolecular%252BSimilarity%25253A%252BAdvances%252Bin%252BMethods%25252C%252BApplications%252Band%252BValidations%252Bin%252BVirtual%252BScreening%252Band%252BQSAR%26aulast%3DBender%26aufirst%3DAndreas%26date%3D2006%26spage%3D141%26epage%3D168%26pub%3DElsevier%26date%3D2006%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lutz  Franke</span>, <span class="hlFld-ContribAuthor ">Evgeny  Byvatov</span>, <span class="hlFld-ContribAuthor ">Oliver  Werz</span>, <span class="hlFld-ContribAuthor ">Dieter  Steinhilber</span>, <span class="hlFld-ContribAuthor ">Petra  Schneider</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">Extraction and Visualization of Potential Pharmacophore Points Using Support Vector Machines:  Application to Ligand-Based Virtual Screening for COX-2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2005,</strong> <em>48 </em>
                                    (22)
                                     , 6997-7004. <a href="https://doi.org/10.1021/jm050619h" title="DOI URL">https://doi.org/10.1021/jm050619h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm050619h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm050619h%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExtraction%252Band%252BVisualization%252Bof%252BPotential%252BPharmacophore%252BPoints%252BUsing%252BSupport%252BVector%252BMachines%25253A%2525E2%252580%252589%252BApplication%252Bto%252BLigand-Based%252BVirtual%252BScreening%252Bfor%252BCOX-2%252BInhibitors%26aulast%3DFranke%26aufirst%3DLutz%26date%3D2005%26date%3D2005%26volume%3D48%26issue%3D22%26spage%3D6997%26epage%3D7004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lutz  Weber</span>. </span><span class="cited-content_cbyCitation_article-title">Current Status of Virtual Combinatorial Library Design. </span><span class="cited-content_cbyCitation_journal-name">QSAR & Combinatorial Science</span><span> <strong>2005,</strong> <em>24 </em>
                                    (7)
                                     , 809-823. <a href="https://doi.org/10.1002/qsar.200510120" title="DOI URL">https://doi.org/10.1002/qsar.200510120</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/qsar.200510120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fqsar.200510120%26sid%3Dliteratum%253Aachs%26jtitle%3DQSAR%2520%2526%2520Combinatorial%2520Science%26atitle%3DCurrent%252BStatus%252Bof%252BVirtual%252BCombinatorial%252BLibrary%252BDesign%26aulast%3DWeber%26aufirst%3DLutz%26date%3D2005%26volume%3D24%26issue%3D7%26spage%3D809%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steffen  Renner</span>, <span class="hlFld-ContribAuthor ">Verena  Ludwig</span>, <span class="hlFld-ContribAuthor ">Oliver  Boden</span>, <span class="hlFld-ContribAuthor ">Ute  Scheffer</span>, <span class="hlFld-ContribAuthor ">Michael  Göbel</span>, <span class="hlFld-ContribAuthor ">Gisbert  Schneider</span>. </span><span class="cited-content_cbyCitation_article-title">New Inhibitors of the Tat-TAR RNA Interaction Found with a “Fuzzy” Pharmacophore Model. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2005,</strong> <em>6 </em>
                                    (6)
                                     , 1119-1125. <a href="https://doi.org/10.1002/cbic.200400376" title="DOI URL">https://doi.org/10.1002/cbic.200400376</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.200400376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.200400376%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DNew%252BInhibitors%252Bof%252Bthe%252BTat-TAR%252BRNA%252BInteraction%252BFound%252Bwith%252Ba%252B%2525E2%252580%25259CFuzzy%2525E2%252580%25259D%252BPharmacophore%252BModel%26aulast%3DRenner%26aufirst%3DSteffen%26date%3D2005%26date%3D2005%26volume%3D6%26issue%3D6%26spage%3D1119%26epage%3D1125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00001.jpeg" id="rightTab-_i6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Schematic overview of the calculation of a SQUID pharmacophore model and a corresponding CV descriptor. (a) An alignment of active reference molecules is transformed into a 3D distribution of generalized pharmacophore interaction points. Yellow points represent hydrophobic interacting atoms, green points represent atoms which can interact as hydrogen bond acceptor and donor. (b) The interaction points are clustered to form PPPs based on the feature density distribution of the respective features and based on the defined resolution of the resulting model. The resolution of the model is affected by the cluster radius. Each PPP should represent a local maximum in the local feature density distribution. The local feature density LFD for each interaction point is calculated with eq 1. In this example a cluster radius of 1.5 Å was used. The circles exemplary show the cluster radii around the features which are marked by dots. The two maxima are highlighted in red. As can be seen in Figure 1b, the hydrophobic maximum is the only feature with five neighboring hydrophobic features within its cluster radius. All other hydrophobic features with lower LFD values have fewer hydrophobic features within their cluster radii. After determination of the maxima, all remaining interaction points are assigned to the nearest maximum of their respective feature-type. (c) PPPs are defined at the position of the geometric center of the atoms which are assigned to a local maximum. Two PPPs (yellow = hydrophobic, green = polar = hydrogen bond acceptor and donor) have been calculated in this example. The standard deviations σ of the PPPs are defined by the median distance of the atoms of the corresponding maximum to the position of the PPP, with a minimum value of 0.5 Å like for the polar PPP. The polar PPP has a conservation weight <i>W</i><sub>c</sub> of 0.5 since it consists only of atoms of one of the two molecules. The <i>W</i><sub>c</sub> of 0.96 for the hydrophobic PPP results from the slightly different number of hydrophobic interactions in the two molecules. (d) For virtual screening the pharmacophore model is transformed into a CV with eq 3. This CV represents a vector of probabilities for the presence of pairs of interactions within certain distances. A section of the resulting CV representing polar−hydrophobic pairs of PPPs within a distance range of 0 to 20 Å is shown. The highest probability of finding such a pair is such within a distance range of 2 to 4 Å. With more pairs of PPPs the probability distributions of these pairs of the bins are added. For virtual screening the whole CV is scaled to a maximum of 1. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yh00001" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yh00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00001.jpeg" id="rightTab-_i16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Scheme 1.   2D Structures of the Known Active COX-2 Inhibitors Used for the Calculation of the SQUID Pharmacophore Model </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yh00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00002" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00002.jpeg" id="rightTab-_i17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Three-dimensional alignment of the COX-2 inhibitors. Rofecoxib and M5 were aligned to the crystal structure conformation of SC-558 bound to COX-2. According to Palomer et al., essential interactions for specific COX-2 inhibitors are the aromatic rings A and B and the sulfonyl group.<sup>19</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00003" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00003.jpeg" id="rightTab-_i19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Pharmacophore models calculated with different cluster radii from the aligned COX-2 inhibitors. The cluster radii of the six shown models are (a) 1.0 Å, (b) 1.5 Å, (c) 2.0 Å, (d) 2.5 Å, (e) 3.0 Å, and (f) 3.5 Å. The colored spheres are representing the PPPs of the models. The radii of the spheres denote the standard deviations of the spatial distributions of the atoms of each PPP. Yellow PPPs represent hydrophobic interactions, magenta PPPs represent hydrogen-bond donors, and cyan PPPs represent hydrogen-bond acceptors. The intensity of the color of a PPP denotes the conservation of the PPP among the aligned molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00004" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00004.jpeg" id="rightTab-_i21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Results of the retrospective screenings for COX-2 inhibitors with the pharmacophore models calculated with cluster radii from 0.5 Å to 3.5 Å with the optimized feature-type weights of each model. The shown enrichment factors are average values from screening of five randomly selected subsets of the COBRA database. The enrichment factors are calculated from the first 1% in part a, for the first 5% in part b and for the first 10% in part c. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00005" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00005.jpeg" id="rightTab-_i23" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 Comparison of the enrichment curves of the best found pharmacophore model resulting from 1.4 Å cluster radius with the results of the retrospective screening with the CATS3D descriptors of the COX-2 inhibitors used for the model calculation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yh00002" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yh00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00002.jpeg" id="rightTab-_i25" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 2.   2D Structures of the Known Active Thrombin Inhibitors Used for the Calculation of the SQUID Pharmacophore Model<i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> The names beneath the molecules are the pdb identifiers of the protein structures from which the conformations of these molecules were extracted.</p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yh00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00006" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00006.jpeg" id="rightTab-_i26" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Three-dimensional alignment of the thrombin inhibitors. The molecules were aligned by superposition of their appropriate protein structures. All essential interactions with the receptor, according to Patel et al., are indicated.<sup>20</sup> B is a basic interaction, H1, H2, and H3 are hydrophobic interactions, A1 is a hydrogen-bond acceptor, and D1 is a hydrogen-bond donor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00007" data-index="9" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00007.jpeg" id="rightTab-_i28" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Pharmacophore models calculated with different cluster radii from the aligned thrombin inhibitors. The cluster radii of the six shown models are (a) 1.0 Å, (b) 1.5 Å, (c) 2.0 Å, (d) 2.5 Å, (e) 3.0 Å, and (f) 3.5 Å. Yellow PPPs represent hydrophobic interactions, magenta PPPs represent hydrogen-bond donors, cyan PPPs represent hydrogen-bond acceptors, and green PPPs represent polar interaction. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00008" data-index="10" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00008.jpeg" id="rightTab-_i30" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 Results of the retrospective screenings for thrombin inhibitors with the pharmacophore models calculated with cluster radii from 0.5 Å to 3.5 Å with the optimized feature-type weights of each model. The shown enrichment factors are average values from screening of five randomly selected subsets of the COBRA database. The enrichment factors are calculated from the first 1% in a, for the first 5% in b and for the first 10% in c. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00009" data-index="11" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00009.jpeg" id="rightTab-_i32" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 9 Comparison of the enrichment curves of the best found pharmacophore model resulting from 2.0 Å cluster radius with the results of the retrospective screening with the CATS3D descriptors of the thrombin inhibitors used for the model calculation. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00009"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yf00010" data-index="12" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yf00010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00010.jpeg" id="rightTab-_i34" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 10 MOE Pharmacophore models for COX-2 (a) and thrombin (b). In the COX-2 pharmacophore model the rings A and B are represented by two aromatic ring center PPPs, and the sulfonyl group is represented by two acceptor PPPs and one PPP for a donor or hydrophobic interaction. In the thrombin pharmacophore model the hydrophobic interactions H1 and H2 are represented by hydrophobic PPPs while H3 is represented by an aromatic PPP. For A1 a hydrogen-bond acceptor PPP and for D1 a hydrogen-bond donor PPP was found. The basic interaction B was represented by two PPPs, one for hydrogen-bond acceptor or hydrogen-bond donor, and one for hydrogen-bond acceptor and hydrogen-bond donor. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yf00010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yf00010"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm031139yh00003" data-index="13" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/medium/jm031139yh00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00003.jpeg" id="rightTab-_i36" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first"> Scheme 3.   2D Structures of the Best Scored Nonactive Molecules Found with the Best Pharmacophore Models<i><sup>a</sup></i><sup></sup></p><p class="last"><p class="last"><i><sup>a</sup></i><sup></sup> Compounds <b>11</b> and <b>12</b> were found with the COX-2 pharmacophore model with 1.4 Å cluster radius. Compounds <b>13</b> and <b>14</b> were found with the thrombin model derived from 2.0 Å cluster radius. </p></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2004/jmcmar.2004.47.issue-19/jm031139y/production/images/large/jm031139yh00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm031139y&amp;id=jm031139yh00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><a name="jm004701904664" id="jm004701904664"></a></div><p class="rightTab-references-count">
            This article references 26 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm031139yb00001"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00001_1"><span><span class="NLM_string-ref">Böhm, H.-J., Schneider, G., Eds. <i>  Virtual Screening for Bioactive Molecules</i>; Wiley-VCH:  Weinheim, 2000.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm031139yb00001/jm031139yb00001_1&amp;dbid=16&amp;doi=10.1021%2Fjm031139y&amp;key=10.1002%2F9783527613083" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=H.-J.+B%C3%B6hm&author=G.+Schneider&title=Virtual+Screening+for+Bioactive+Molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00001_1&amp;dbid=16384&amp;doi=10.1002%2F9783527613083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527613083%26sid%3Dliteratum%253Aachs%26aulast%3DB%25C3%25B6hm%26aufirst%3DH.-J.%26btitle%3DVirtual%2520Screening%2520for%2520Bioactive%2520Molecules%26pub%3DWiley-VCH%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00002"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00002_1"><span><span class="NLM_string-ref">Xue, L.; Bajorath, J. Molecular descriptors in chemoinformatics, computational combinatorial chemistry, and virtual screening, <i>Comb. Chem. High </i><i>Throughput Screen</i>. <b>2000</b>, <i>3</i>, 363−372.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2000&pages=372&author=L.+Xue&author=J+Bajorath&title=Molecular+descriptors+in+chemoinformatics%2C+computational+combinatorial+chemistry%2C+and+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DL.%26atitle%3DMolecular%2520descriptors%2520in%2520chemoinformatics%252C%2520computational%2520combinatorial%2520chemistry%252C%2520and%2520virtual%2520screening%26jtitle%3DComb.%2520Chem.%2520High%2520Throughput%2520Screen.%26date%3D2000%26volume%3D3%26spage%3D372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00003"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00003_1"><span><span class="NLM_string-ref">Guner, O., Ed. <i>  Pharmacophore Perception, Development and Use in Drug </i><i>Design</i>; International University Line:  La Jolla CA, 2000.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=O.+Guner&title=Pharmacophore+Perception%2C+Development+and+Use+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGuner%26aufirst%3DO.%26btitle%3DPharmacophore%2520Perception%252C%2520Development%2520and%2520Use%2520in%2520Drug%2520Design%26pub%3DInternational%2520University%2520Line%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00004"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00004_1"><span><span class="NLM_string-ref">Pickett, S. The biophore concept. In <i>Protein</i><i>−</i><i>Ligand Interactions</i>; Böhm, H.-J., Schneider, G., Eds.; Wiley-VCH:  Weinheim, New York, 2003; pp 73−105.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&pages=105&author=S.+Pickett&title=Protein%E2%88%92Ligand+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPickett%26aufirst%3DS.%26atitle%3DThe%2520biophore%2520concept%26btitle%3DProtein%25E2%2588%2592Ligand%2520Interactions%26pub%3DWiley-VCH%26date%3D2003%26spage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00005"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00005_1"><span><span class="NLM_string-ref">Greene, J.; Kahn, S.; Savoj, H.; Sprague, P.; Teig, S. Chemical function queries for 3D database search. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci</i>. <b>1994</b>, <i>34</i>, 1297−1308.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1994&pages=1308&author=J.+Greene&author=S.+Kahn&author=H.+Savoj&author=P.+Sprague&author=S+Teig&title=Chemical+function+queries+for+3D+database+search"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreene%26aufirst%3DJ.%26atitle%3DChemical%2520function%2520queries%2520for%25203D%2520database%2520search%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1994%26volume%3D34%26spage%3D1308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00006"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00006_1"><span><span class="NLM_string-ref">Martin, Y. C.; Bures, M. G.; Danaher, E. A.; DeLazzer, J.; Lico, I.; Pavlik, P. A. A fast approach to pharmacophore mapping and its application to dopaminergic and benzodiazepine agonists. <i>  J. Comput:-Aided Mol. Des.</i><b>1993</b>, <i>7</i>, 83−102.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1993&pages=102&author=Y.+C.+Martin&author=M.+G.+Bures&author=E.+A.+Danaher&author=J.+DeLazzer&author=I.+Lico&author=P.+A+Pavlik&title=A+fast+approach+to+pharmacophore+mapping+and+its+application+to+dopaminergic+and+benzodiazepine+agonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DY.%2BC.%26atitle%3DA%2520fast%2520approach%2520to%2520pharmacophore%2520mapping%2520and%2520its%2520application%2520to%2520dopaminergic%2520and%2520benzodiazepine%2520agonists%26jtitle%3DJ.%2520Comput%253A-Aided%2520Mol.%2520Des.%26date%3D1993%26volume%3D7%26spage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00007"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00007_1"><span><span class="NLM_string-ref">Jones, G.; Willett, P.; Glen, R. C. A genetic algorithm for flexible molecular overlay and pharmacophore elucidation. <i>  J. Comput:-Aided Mol. Des.</i><b>1995</b>, <i>9</i>, 532−549.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=549&author=G.+Jones&author=P.+Willett&author=R.+C+Glen&title=A+genetic+algorithm+for+flexible+molecular+overlay+and+pharmacophore+elucidation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26atitle%3DA%2520genetic%2520algorithm%2520for%2520flexible%2520molecular%2520overlay%2520and%2520pharmacophore%2520elucidation%26jtitle%3DJ.%2520Comput%253A-Aided%2520Mol.%2520Des.%26date%3D1995%26volume%3D9%26spage%3D549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00008"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00008_1"><span><span class="NLM_string-ref">MOE, Molecular Operating Environment. Distributor:  Chemical Computing Group, 1010 Sherbrooke St. West, #910, Montreal, Canada H3A.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MOE%2C+Molecular+Operating+Environment.+Distributor%3A%E2%80%89+Chemical+Computing+Group%2C+1010+Sherbrooke+St.+West%2C+%23910%2C+Montreal%2C+Canada+H3A.+MOE%2C+Molecular+Operating+Environment.+Distributor%3A%E2%80%89+Chemical+Computing+Group%2C+1010+Sherbrooke+St.+West%2C+%23910%2C+Montreal%2C+Canada+H3A."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3D%26aulast%3D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00009"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00009_1"><span><span class="NLM_string-ref">Vedani, A.; Dobler, M. Multidimensional QSAR in drug research. Predicting binding affinities, toxicity and pharmacokinetic parameters. <i>  Prog. Drug </i><i>Res</i>. <b>2000</b>, <i>55</i>, 105−135.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2000&pages=135&author=A.+Vedani&author=M+Dobler&title=Multidimensional+QSAR+in+drug+research.+Predicting+binding+affinities%2C+toxicity+and+pharmacokinetic+parameters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVedani%26aufirst%3DA.%26atitle%3DMultidimensional%2520QSAR%2520in%2520drug%2520research.%2520Predicting%2520binding%2520affinities%252C%2520toxicity%2520and%2520pharmacokinetic%2520parameters%26jtitle%3DProg.%2520Drug%2520Res.%26date%3D2000%26volume%3D55%26spage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm031139yb00010_1"><span><span class="NLM_string-ref">Mason, J. S.; Cheney, D. L. Library design and virtual screening using multiple 4-point pharmacophore fingerprints. <i>  Pac. Symp. Biocomput</i>. <b>2000</b>, 576−587.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&conference=Pac.+Symp.+Biocomput.&author=J.+S.+Mason&author=D.+L.+Cheney&title=Library+design+and+virtual+screening+using+multiple+4-point+pharmacophore+fingerprints"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DLibrary%2520design%2520and%2520virtual%2520screening%2520using%2520multiple%25204-point%2520pharmacophore%2520fingerprints%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00011"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00011_1"><span><span class="NLM_string-ref">Schneider, G.; Neidhart, W.; Giller, T.; Schmid, G. “Scaffold-Hopping” by topological pharmacophore search:  a contribution to virtual screening. <i>  Angew. Chem., Int. Ed. </i><b>1999</b>, <i>38</i>, 2894−2896</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=2896&author=G.+Schneider&author=W.+Neidhart&author=T.+Giller&author=G+Schmid&title=Scaffold-Hopping%E2%80%9D+by+topological+pharmacophore+search%3A%E2%80%89+a+contribution+to+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DScaffold-Hopping%25E2%2580%259D%2520by%2520topological%2520pharmacophore%2520search%253A%25E2%2580%2589%2520a%2520contribution%2520to%2520virtual%2520screening%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D2896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00012"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00012_1"><span><span class="NLM_string-ref">Xue, L.; Stahura, F. L.; Godden, J. W.; Bajorath, J. Fingerprint scaling increases the probability of identifying molecules with similar activity in virtual screening calculations. <i>  J. Chem. </i><i>Inf.</i><i> Comput. Sci.</i><b>2001</b>, <i>41</i>, 746−753.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=753&author=L.+Xue&author=F.+L.+Stahura&author=J.+W.+Godden&author=J+Bajorath&title=Fingerprint+scaling+increases+the+probability+of+identifying+molecules+with+similar+activity+in+virtual+screening+calculations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DL.%26atitle%3DFingerprint%2520scaling%2520increases%2520the%2520probability%2520of%2520identifying%2520molecules%2520with%2520similar%2520activity%2520in%2520virtual%2520screening%2520calculations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2001%26volume%3D41%26spage%3D753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00013"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00013_1"><span><span class="NLM_string-ref">Horvath, D.; Mao, B. Neighbourhood behaviour. Fuzzy molecular descriptors and their influence on the relationship between structural similarity and property similarity. <i>  QSAR Comb. Sci.</i><b>2003</b>, <i>22</i>, 498−509.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=509&author=D.+Horvath&author=B+Mao&title=Neighbourhood+behaviour.+Fuzzy+molecular+descriptors+and+their+influence+on+the+relationship+between+structural+similarity+and+property+similarity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHorvath%26aufirst%3DD.%26atitle%3DNeighbourhood%2520behaviour.%2520Fuzzy%2520molecular%2520descriptors%2520and%2520their%2520influence%2520on%2520the%2520relationship%2520between%2520structural%2520similarity%2520and%2520property%2520similarity%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2003%26volume%3D22%26spage%3D509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00014"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00014_1"><span><span class="NLM_string-ref">Fechner, U.; Franke, L.; Renner, S.; Schneider, P.; Schneider, P.; Schneider, G. Comparison of correlation vector methods for ligand-based similarity searching. <i>  J. Comput.-Aided Mol. Des</i>. <b>2003</b>, <i>17</i>, 687−698.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=698&author=U.+Fechner&author=L.+Franke&author=S.+Renner&author=P.+Schneider&author=P.+Schneider&author=G+Schneider&title=Comparison+of+correlation+vector+methods+for+ligand-based+similarity+searching"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFechner%26aufirst%3DU.%26atitle%3DComparison%2520of%2520correlation%2520vector%2520methods%2520for%2520ligand-based%2520similarity%2520searching%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2003%26volume%3D17%26spage%3D698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00015"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00015_1"><span><span class="NLM_string-ref">Good, A. C.; Kuntz, I. D. Investigating the extension of pairwise distance pharmacophore measures to triplet-based descriptors. <i>  J. Comput.-Aided </i><i>Mol. Des.</i><b>1995</b>, <i>9</i>, 373−379.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=379&author=A.+C.+Good&author=I.+D+Kuntz&title=Investigating+the+extension+of+pairwise+distance+pharmacophore+measures+to+triplet-based+descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGood%26aufirst%3DA.%2BC.%26atitle%3DInvestigating%2520the%2520extension%2520of%2520pairwise%2520distance%2520pharmacophore%2520measures%2520to%2520triplet-based%2520descriptors%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1995%26volume%3D9%26spage%3D379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00016"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00016_1"><span><span class="NLM_string-ref">Mason, J. S.; Morize, I.; Menard, P. R.; Cheney, D. L.; Hulme, C.; Labaudiniere, R. F. New 4-point pharmacophore method for molecular similarity and diversity applications:  Overview of the method and applications, including a novel approach to the design of combinatorial libraries containing privileged substructures. <i>  J. Med. Chem.</i><b>1999</b>, <i>42</i>, 3251−3264.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=3264&author=J.+S.+Mason&author=I.+Morize&author=P.+R.+Menard&author=D.+L.+Cheney&author=C.+Hulme&author=R.+F+Labaudiniere&title=New+4-point+pharmacophore+method+for+molecular+similarity+and+diversity+applications%3A%E2%80%89+Overview+of+the+method+and+applications%2C+including+a+novel+approach+to+the+design+of+combinatorial+libraries+containing+privileged+substructures"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMason%26aufirst%3DJ.%2BS.%26atitle%3DNew%25204-point%2520pharmacophore%2520method%2520for%2520molecular%2520similarity%2520and%2520diversity%2520applications%253A%25E2%2580%2589%2520Overview%2520of%2520the%2520method%2520and%2520applications%252C%2520including%2520a%2520novel%2520approach%2520to%2520the%2520design%2520of%2520combinatorial%2520libraries%2520containing%2520privileged%2520substructures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D3264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00017"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00017_1"><span><span class="NLM_string-ref">Schneider, P.; Schneider, G. Collection of bioactive reference compounds for focused library design. <i>  QSAR Comb. Sci</i>. <b>2003</b>, <i>22</i>, 713−718.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=718&author=P.+Schneider&author=G+Schneider&title=Collection+of+bioactive+reference+compounds+for+focused+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DP.%26atitle%3DCollection%2520of%2520bioactive%2520reference%2520compounds%2520for%2520focused%2520library%2520design%26jtitle%3DQSAR%2520Comb.%2520Sci.%26date%3D2003%26volume%3D22%26spage%3D718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00018"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00018_1"><span><span class="NLM_string-ref">CORINA. Distributor:  Molecular Networks GmbH, Computerchemie, Nägelsbachstrasse 25, D-91052 Erlangen, Germany, <a class="ext-link" href="http://www.mol-net.de/" target="_blank">http://www.mol-net.de/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CORINA.+Distributor%3A%E2%80%89+Molecular+Networks+GmbH%2C+Computerchemie%2C+N%C3%A4gelsbachstrasse+25%2C+D-91052+Erlangen%2C+Germany%2C+http%3A%2F%2Fwww.mol-net.de%2F.+CORINA.+Distributor%3A%E2%80%89+Molecular+Networks+GmbH%2C+Computerchemie%2C+N%C3%A4gelsbachstrasse+25%2C+D-91052+Erlangen%2C+Germany%2C+http%3A%2F%2Fwww.mol-net.de%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.mol-net.de%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00019"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00019_1"><span><span class="NLM_string-ref">Palomer, A.; Cabre, F.; Pascual, J.; Campos, J.; Trujillo, M.; Entrena, A.; Gallo, M.; Garcia, L.; Mauleon, D.; Espinosa, A. Identification of novel cyclooxygenase-2 selective inhibitors using pharmacophore models. <i>  J. </i><i>Med. Chem. </i><b>2002</b>, <i>45</i>, 1402−1411.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1411&author=A.+Palomer&author=F.+Cabre&author=J.+Pascual&author=J.+Campos&author=M.+Trujillo&author=A.+Entrena&author=M.+Gallo&author=L.+Garcia&author=D.+Mauleon&author=A+Espinosa&title=Identification+of+novel+cyclooxygenase-2+selective+inhibitors+using+pharmacophore+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPalomer%26aufirst%3DA.%26atitle%3DIdentification%2520of%2520novel%2520cyclooxygenase-2%2520selective%2520inhibitors%2520using%2520pharmacophore%2520models%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm031139yb00020_1"><span><span class="NLM_string-ref">Patel, Y.; Gillet, V. J.; Bravi, G.; Leach, A. R. A Comparison of the pharmacophore identification programs:  Catalyst, DISCO and GASP. <i>  J. </i><i>Comput.-Aided Mol. Des. </i><b>2002</b>, <i>16</i>, 653−681.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2002&pages=681&author=Y.+Patel&author=V.+J.+Gillet&author=G.+Bravi&author=A.+R+Leach&title=A+Comparison+of+the+pharmacophore+identification+programs%3A%E2%80%89+Catalyst%2C+DISCO+and+GASP"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DY.%26atitle%3DA%2520Comparison%2520of%2520the%2520pharmacophore%2520identification%2520programs%253A%25E2%2580%2589%2520Catalyst%252C%2520DISCO%2520and%2520GASP%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2002%26volume%3D16%26spage%3D681" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00021"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00021_1"><span><span class="NLM_string-ref">Woo, L. W.; Howarth, N. M.; Purohit, A.; Hejaz, H. A.; Reed, M. J.; Potter, B. V. Steroidal and nonsteroidal sulfamates as potent inhibitors of steroid sulfatase. <i>  J. Med. Chem.</i><b>1998</b>, <i>41</i>, 1068−1083.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1083&author=L.+W.+Woo&author=N.+M.+Howarth&author=A.+Purohit&author=H.+A.+Hejaz&author=M.+J.+Reed&author=B.+V+Potter&title=Steroidal+and+nonsteroidal+sulfamates+as+potent+inhibitors+of+steroid+sulfatase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWoo%26aufirst%3DL.%2BW.%26atitle%3DSteroidal%2520and%2520nonsteroidal%2520sulfamates%2520as%2520potent%2520inhibitors%2520of%2520steroid%2520sulfatase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00022"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00022_1"><span><span class="NLM_string-ref">Supuran, C. T.; Scozzafava, A.; Casini, A. Carbonic anhydrase inhibitors. <i>  Med. </i><i>Res. Rev.</i><b>2003</b>, <i>23</i>, 146−189.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2003&pages=189&author=C.+T.+Supuran&author=A.+Scozzafava&author=A+Casini&title=Carbonic+anhydrase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26atitle%3DCarbonic%2520anhydrase%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2003%26volume%3D23%26spage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00023"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00023_1"><span><span class="NLM_string-ref">Marlowe, C. K.; Sinha, U.; Gunn, A. C.; Scarborough, R. M. Design, synthesis and structure−activity relationship of a series of arginine aldehyde factor Xa inhibitors. Part 1:  structures based on the (D)-Arg-Gly-Arg tripeptide sequence. <i>  Bioorg. Med. Chem. Lett.</i><b>2000</b>, <i>10</i>, 13−16.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=16&author=C.+K.+Marlowe&author=U.+Sinha&author=A.+C.+Gunn&author=R.+M+Scarborough&title=Design%2C+synthesis+and+structure%E2%88%92activity+relationship+of+a+series+of+arginine+aldehyde+factor+Xa+inhibitors.+Part+1%3A%E2%80%89+structures+based+on+the+%28D%29-Arg-Gly-Arg+tripeptide+sequence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMarlowe%26aufirst%3DC.%2BK.%26atitle%3DDesign%252C%2520synthesis%2520and%2520structure%25E2%2588%2592activity%2520relationship%2520of%2520a%2520series%2520of%2520arginine%2520aldehyde%2520factor%2520Xa%2520inhibitors.%2520Part%25201%253A%25E2%2580%2589%2520structures%2520based%2520on%2520the%2520%2528D%2529-Arg-Gly-Arg%2520tripeptide%2520sequence%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2000%26volume%3D10%26spage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00024"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00024" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00024_1"><span><span class="NLM_string-ref">Rudolf, K.; Eberlein, W.; Engel, W.; Wieland, H. A.; Willim, K. D.; Entzeroth, M.; Wienen, W.; Beck-Sickinger, A. G.; Doods, H. N. The first highly potent and selective non-peptide neuropeptide Y Y1 receptor antagonist:  BIBP3226. <i>  Eur. J. Pharmacol.</i><b>1994</b>, <i>271</i>, R11−R13.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1994&pages=R13&author=K.+Rudolf&author=W.+Eberlein&author=W.+Engel&author=H.+A.+Wieland&author=K.+D.+Willim&author=M.+Entzeroth&author=W.+Wienen&author=A.+G.+Beck-Sickinger&author=H.+N+Doods&title=The+first+highly+potent+and+selective+non-peptide+neuropeptide+Y+Y1+receptor+antagonist%3A%E2%80%89+BIBP3226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRudolf%26aufirst%3DK.%26atitle%3DThe%2520first%2520highly%2520potent%2520and%2520selective%2520non-peptide%2520neuropeptide%2520Y%2520Y1%2520receptor%2520antagonist%253A%25E2%2580%2589%2520BIBP3226%26jtitle%3DEur.%2520J.%2520Pharmacol.%26date%3D1994%26volume%3D271%26spage%3DR13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00025"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00025" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00025_1"><span><span class="NLM_string-ref">Weber, A.; Casini, A.; Heine, A.; Kuhn, D.; Supuran, C. T.; Scozzafava, A.; Klebe, G. Unexpected nanomolar inhibition of carbonic anhydrase by COX-2-selective celecoxib:  new pharmacological opportunities due to related binding site recognition. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 550−557.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=557&author=A.+Weber&author=A.+Casini&author=A.+Heine&author=D.+Kuhn&author=C.+T.+Supuran&author=A.+Scozzafava&author=G+Klebe&title=Unexpected+nanomolar+inhibition+of+carbonic+anhydrase+by+COX-2-selective+celecoxib%3A%E2%80%89+new+pharmacological+opportunities+due+to+related+binding+site+recognition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWeber%26aufirst%3DA.%26atitle%3DUnexpected%2520nanomolar%2520inhibition%2520of%2520carbonic%2520anhydrase%2520by%2520COX-2-selective%2520celecoxib%253A%25E2%2580%2589%2520new%2520pharmacological%2520opportunities%2520due%2520to%2520related%2520binding%2520site%2520recognition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm031139yb00026"><div class="reference"><strong class="refLabel"><a href="#jm031139yb00026" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-jm031139yb00026_1"><span><span class="NLM_string-ref">Ho, Y. T.; Purohit, A.; Vicker, N.; Newman, S. P.; Robinson, J. J.; Leese, M. P.; Ganeshapillai, D.; Woo, L. W.; Potter, B. V.; Reed, M. J. Inhibition of carbonic anhydrase II by steroidal and nonsteroidal sulphamates. <i>  Biochem. </i><i>Biophys. Res. Commun.</i><b>2003</b>, <i>305</i>, 909−914.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2003&pages=914&author=Y.+T.+Ho&author=A.+Purohit&author=N.+Vicker&author=S.+P.+Newman&author=J.+J.+Robinson&author=M.+P.+Leese&author=D.+Ganeshapillai&author=L.+W.+Woo&author=B.+V.+Potter&author=M.+J+Reed&title=Inhibition+of+carbonic+anhydrase+II+by+steroidal+and+nonsteroidal+sulphamates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm031139yb00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DY.%2BT.%26atitle%3DInhibition%2520of%2520carbonic%2520anhydrase%2520II%2520by%2520steroidal%2520and%2520nonsteroidal%2520sulphamates%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2003%26volume%3D305%26spage%3D914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm031139y&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2004.47.issue-19%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Fjm031139y%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm031139y" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                474KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67997ed8ef873d06","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
