
<html lang="en"     class="pb-page"  data-request-id="32b3b39c-30b3-4e21-a46b-e238d511375e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.0c00485;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-15;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A Venomics Approach Coupled to High-Throughput Toxin Production Strategies Identifies the First Venom-Derived Melanocortin Receptor Agonists" /></meta><meta name="dc.Creator" content="Steve  Reynaud" /></meta><meta name="dc.Creator" content="Justyna  Ciolek" /></meta><meta name="dc.Creator" content="Michel  Degueldre" /></meta><meta name="dc.Creator" content="Natalie J.  Saez" /></meta><meta name="dc.Creator" content="Ana Filipa  Sequeira" /></meta><meta name="dc.Creator" content="Yoan  Duhoo" /></meta><meta name="dc.Creator" content="Joana L. A.  Brás" /></meta><meta name="dc.Creator" content="Hervé  Meudal" /></meta><meta name="dc.Creator" content="Miguel  Cabo Díez" /></meta><meta name="dc.Creator" content="Victoria  Fernández Pedrosa" /></meta><meta name="dc.Creator" content="Marion  Verdenaud" /></meta><meta name="dc.Creator" content="Julia  Boeri" /></meta><meta name="dc.Creator" content="Oscar  Pereira Ramos" /></meta><meta name="dc.Creator" content="Frédéric  Ducancel" /></meta><meta name="dc.Creator" content="Margot  Vanden Driessche" /></meta><meta name="dc.Creator" content="Rudy  Fourmy" /></meta><meta name="dc.Creator" content="Aude  Violette" /></meta><meta name="dc.Creator" content="Grégory  Upert" /></meta><meta name="dc.Creator" content="Gilles  Mourier" /></meta><meta name="dc.Creator" content="Annette G.  Beck-Sickinger" /></meta><meta name="dc.Creator" content="Karin  Mörl" /></meta><meta name="dc.Creator" content="Céline  Landon" /></meta><meta name="dc.Creator" content="Carlos M. G. A.  Fontes" /></meta><meta name="dc.Creator" content="Rebeca  Miñambres Herráiz" /></meta><meta name="dc.Creator" content="Ricardo C.  Rodríguez de la Vega" /></meta><meta name="dc.Creator" content="Steve  Peigneur" /></meta><meta name="dc.Creator" content="Jan  Tytgat" /></meta><meta name="dc.Creator" content="Loïc  Quinton" /></meta><meta name="dc.Creator" content="Edwin  De Pauw" /></meta><meta name="dc.Creator" content="Renaud  Vincentelli" /></meta><meta name="dc.Creator" content="Denis  Servent" /></meta><meta name="dc.Creator" content="Nicolas  Gilles" /></meta><meta name="dc.Description" content="Animal venoms are rich in hundreds of toxins with extraordinary biological activities. Their exploitation is difficult due to their complexity and the small quantities of venom available from most ..." /></meta><meta name="Description" content="Animal venoms are rich in hundreds of toxins with extraordinary biological activities. Their exploitation is difficult due to their complexity and the small quantities of venom available from most ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 30, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00485" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00485" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00485" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00485" /></link>
        
    
    

<title>A Venomics Approach Coupled to High-Throughput Toxin Production Strategies Identifies the First Venom-Derived Melanocortin Receptor Agonists | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00485" /></meta><meta property="og:title" content="A Venomics Approach Coupled to High-Throughput Toxin Production Strategies Identifies the First Venom-Derived Melanocortin Receptor Agonists" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0007.jpeg" /></meta><meta property="og:description" content="Animal venoms are rich in hundreds of toxins with extraordinary biological activities. Their exploitation is difficult due to their complexity and the small quantities of venom available from most venomous species. We developed a Venomics approach combining transcriptomic and proteomic characterization of 191 species and identified 20,206 venom toxin sequences. Two complementary production strategies based on solid-phase synthesis and recombinant expression in Escherichia coli generated a physical bank of 3597 toxins. Screened on hMC4R, this bank gave an incredible hit rate of 8%. Here, we focus on two novel toxins: N-TRTX-Preg1a, exhibiting an inhibitory cystine knot (ICK) motif, and N-BUTX-Ptr1a, a short scorpion-CSαβ structure. Neither N-TRTX-Preg1a nor N-BUTX-Ptr1a affects ion channels, the known targets of their toxin scaffolds, but binds to four melanocortin receptors with low micromolar affinities and activates the hMC1R/Gs pathway. Phylogenetically, these two toxins form new groups within their respective families and represent novel hMC1R agonists, structurally unrelated to the natural agonists." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00485"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00485">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00485&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00485&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00485&amp;href=/doi/10.1021/acs.jmedchem.0c00485" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 8250-8264</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02119" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00494" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A Venomics Approach Coupled to High-Throughput Toxin Production Strategies Identifies the First Venom-Derived Melanocortin Receptor Agonists</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Steve Reynaud</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steve Reynaud</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris-Sud, 15 Rue Georges Clemenceau, Orsay 91405 France</div><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steve++Reynaud">Steve Reynaud</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Justyna Ciolek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Justyna Ciolek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Justyna++Ciolek">Justyna Ciolek</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michel Degueldre</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michel Degueldre</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mass Spectrometry Laboratory, Université de Liège, Allée du six Aout 11, Quartier Agora, Liege 4000 Belgium</div><div class="loa-info-affiliations-info">Department of Analytical Science Biologicals, UCB, Chemin du Foriest, Braine L’Alleud 1420 Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michel++Degueldre">Michel Degueldre</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Natalie J. Saez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Natalie J. Saez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre National de la Recherche Scientifique, Architecture et Fonction des Macromolécules Biologiques, Campus de Luminy, Marseille 13288 France</div><div class="loa-info-affiliations-info">Institute for Molecular Bioscience, The University of Queensland, St Lucia 4072, QLD, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Natalie+J.++Saez">Natalie J. Saez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ana Filipa Sequeira</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ana Filipa Sequeira</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Universidade de Lisboa, CIISA - Faculdade de Medicina Veterinária, Avenida da Universidade Técnica, Lisboa 1300-477 Portugal</div><div class="loa-info-affiliations-info">NZYTech Lda, Genes & Enzymes, Estrada do Paço do Lumiar, Campus do Lumiar, Edifício E - R/C, Lisboa 1649-038 Portugal</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ana+Filipa++Sequeira">Ana Filipa Sequeira</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yoan Duhoo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yoan Duhoo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre National de la Recherche Scientifique, Architecture et Fonction des Macromolécules Biologiques, Campus de Luminy, Marseille 13288 France</div><div class="loa-info-affiliations-info">Institute for Molecular Bioscience, The University of Queensland, St Lucia 4072, QLD, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yoan++Duhoo">Yoan Duhoo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joana L. A. Brás</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joana L. A. Brás</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Universidade de Lisboa, CIISA - Faculdade de Medicina Veterinária, Avenida da Universidade Técnica, Lisboa 1300-477 Portugal</div><div class="loa-info-affiliations-info">NZYTech Lda, Genes & Enzymes, Estrada do Paço do Lumiar, Campus do Lumiar, Edifício E - R/C, Lisboa 1649-038 Portugal</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joana+L.+A.++Br%C3%A1s">Joana L. A. Brás</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hervé Meudal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hervé Meudal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre National de la Recherche Scientifique, Centre de Biophysique Moléculaire, rue Charles Sadron, Orléans 45071 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Herv%C3%A9++Meudal">Hervé Meudal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Miguel Cabo Díez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Miguel Cabo Díez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Next-Generation Sequencing Laboratory, Sistemas Genómicos Ltd., Ronda de Guglielmo Marconi, 6, Paterna 46980 Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Miguel++Cabo+D%C3%ADez">Miguel Cabo Díez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Victoria Fernández Pedrosa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Victoria Fernández Pedrosa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Next-Generation Sequencing Laboratory, Sistemas Genómicos Ltd., Ronda de Guglielmo Marconi, 6, Paterna 46980 Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Victoria++Fern%C3%A1ndez+Pedrosa">Victoria Fernández Pedrosa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marion Verdenaud</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marion Verdenaud</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marion++Verdenaud">Marion Verdenaud</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julia Boeri</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julia Boeri</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julia++Boeri">Julia Boeri</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oscar Pereira Ramos</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oscar Pereira Ramos</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oscar++Pereira+Ramos">Oscar Pereira Ramos</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Frédéric Ducancel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Frédéric Ducancel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département IDMIT, 18 route du Panorama, 92265 Fontenay-aux-Roses, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fr%C3%A9d%C3%A9ric++Ducancel">Frédéric Ducancel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Margot Vanden Driessche</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Margot Vanden Driessche</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Margot++Vanden+Driessche">Margot Vanden Driessche</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rudy Fourmy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rudy Fourmy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alphabiotoxine Laboratory sprl, Barberie 15, Montroeul-au-bois 7911 Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rudy++Fourmy">Rudy Fourmy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aude Violette</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aude Violette</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alphabiotoxine Laboratory sprl, Barberie 15, Montroeul-au-bois 7911 Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aude++Violette">Aude Violette</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Grégory Upert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Grégory Upert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gr%C3%A9gory++Upert">Grégory Upert</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gilles Mourier</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gilles Mourier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gilles++Mourier">Gilles Mourier</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Annette G. Beck-Sickinger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Annette G. Beck-Sickinger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemistry, Universitat Leipzig, Leipzig 04103 Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Annette+G.++Beck-Sickinger">Annette G. Beck-Sickinger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4560-8020" title="Orcid link">http://orcid.org/0000-0003-4560-8020</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Karin Mörl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Karin Mörl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Biochemistry, Universitat Leipzig, Leipzig 04103 Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Karin++M%C3%B6rl">Karin Mörl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Céline Landon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Céline Landon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre National de la Recherche Scientifique, Centre de Biophysique Moléculaire, rue Charles Sadron, Orléans 45071 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=C%C3%A9line++Landon">Céline Landon</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0248-1990" title="Orcid link">http://orcid.org/0000-0002-0248-1990</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlos M. G. A. Fontes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlos M. G. A. Fontes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Universidade de Lisboa, CIISA - Faculdade de Medicina Veterinária, Avenida da Universidade Técnica, Lisboa 1300-477 Portugal</div><div class="loa-info-affiliations-info">NZYTech Lda, Genes & Enzymes, Estrada do Paço do Lumiar, Campus do Lumiar, Edifício E - R/C, Lisboa 1649-038 Portugal</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlos+M.+G.+A.++Fontes">Carlos M. G. A. Fontes</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rebeca Miñambres Herráiz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rebeca Miñambres Herráiz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Next-Generation Sequencing Laboratory, Sistemas Genómicos Ltd., Ronda de Guglielmo Marconi, 6, Paterna 46980 Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rebeca++Mi%C3%B1ambres+Herr%C3%A1iz">Rebeca Miñambres Herráiz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ricardo C. Rodríguez de la Vega</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ricardo C. Rodríguez de la Vega</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Universite Paris-Sud, Ecologie, Systematique et Evolution, 15 rue Georges Clémenceau, Orsay 91405 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ricardo+C.++Rodr%C3%ADguez+de+la+Vega">Ricardo C. Rodríguez de la Vega</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Steve Peigneur</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Steve Peigneur</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Toxicology and Pharmacology, University of Leuven (KU Leuven), Herestraat 49, Leuven 3000 Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Steve++Peigneur">Steve Peigneur</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jan Tytgat</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jan Tytgat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Toxicology and Pharmacology, University of Leuven (KU Leuven), Herestraat 49, Leuven 3000 Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Tytgat">Jan Tytgat</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Loïc Quinton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Loïc Quinton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mass Spectrometry Laboratory, Université de Liège, Allée du six Aout 11, Quartier Agora, Liege 4000 Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lo%C3%AFc++Quinton">Loïc Quinton</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8153-9590" title="Orcid link">http://orcid.org/0000-0001-8153-9590</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Edwin De Pauw</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edwin De Pauw</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Mass Spectrometry Laboratory, Université de Liège, Allée du six Aout 11, Quartier Agora, Liege 4000 Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edwin++De+Pauw">Edwin De Pauw</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Renaud Vincentelli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renaud Vincentelli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre National de la Recherche Scientifique, Architecture et Fonction des Macromolécules Biologiques, Campus de Luminy, Marseille 13288 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renaud++Vincentelli">Renaud Vincentelli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Denis Servent</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Denis Servent</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Denis++Servent">Denis Servent</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Nicolas Gilles</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicolas Gilles</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Université Paris Saclay, CEA, Département Médicaments et Technologies pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#7a34131915161b09541d1316161f093a191f1b541c08"><span class="__cf_email__" data-cfemail="e0ae89838f8c8193ce87898c8c8593a0838581ce8692">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Gilles">Nicolas Gilles</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6853-9764" title="Orcid link">http://orcid.org/0000-0001-6853-9764</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00485&amp;href=/doi/10.1021%2Facs.jmedchem.0c00485" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 8250–8264</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 30, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 March 2020</li><li><span class="item_label"><b>Published</b> online</span>30 June 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00485" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00485</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8250%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSteve%2BReynaud%252C%2BJustyna%2BCiolek%252C%2BMichel%2BDegueldre%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D15%26contentID%3Dacs.jmedchem.0c00485%26title%3DA%2BVenomics%2BApproach%2BCoupled%2Bto%2BHigh-Throughput%2BToxin%2BProduction%2BStrategies%2BIdentifies%2Bthe%2BFirst%2BVenom-Derived%2BMelanocortin%2BReceptor%2BAgonists%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8264%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00485"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">753</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00485" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A Venomics Approach Coupled to High-Throughput Toxin Production Strategies Identifies the First Venom-Derived Melanocortin Receptor Agonists&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Steve&quot;,&quot;last_name&quot;:&quot;Reynaud&quot;},{&quot;first_name&quot;:&quot;Justyna&quot;,&quot;last_name&quot;:&quot;Ciolek&quot;},{&quot;first_name&quot;:&quot;Michel&quot;,&quot;last_name&quot;:&quot;Degueldre&quot;},{&quot;first_name&quot;:&quot;Natalie&quot;,&quot;last_name&quot;:&quot;J. Saez&quot;},{&quot;first_name&quot;:&quot;Ana&quot;,&quot;last_name&quot;:&quot;Filipa Sequeira&quot;},{&quot;first_name&quot;:&quot;Yoan&quot;,&quot;last_name&quot;:&quot;Duhoo&quot;},{&quot;first_name&quot;:&quot;Joana&quot;,&quot;last_name&quot;:&quot;L. A. Brás&quot;},{&quot;first_name&quot;:&quot;Hervé&quot;,&quot;last_name&quot;:&quot;Meudal&quot;},{&quot;first_name&quot;:&quot;Miguel&quot;,&quot;last_name&quot;:&quot;Cabo Díez&quot;},{&quot;first_name&quot;:&quot;Victoria&quot;,&quot;last_name&quot;:&quot;Fernández Pedrosa&quot;},{&quot;first_name&quot;:&quot;Marion&quot;,&quot;last_name&quot;:&quot;Verdenaud&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Boeri&quot;},{&quot;first_name&quot;:&quot;Oscar&quot;,&quot;last_name&quot;:&quot;Pereira Ramos&quot;},{&quot;first_name&quot;:&quot;Frédéric&quot;,&quot;last_name&quot;:&quot;Ducancel&quot;},{&quot;first_name&quot;:&quot;Margot&quot;,&quot;last_name&quot;:&quot;Vanden Driessche&quot;},{&quot;first_name&quot;:&quot;Rudy&quot;,&quot;last_name&quot;:&quot;Fourmy&quot;},{&quot;first_name&quot;:&quot;Aude&quot;,&quot;last_name&quot;:&quot;Violette&quot;},{&quot;first_name&quot;:&quot;Grégory&quot;,&quot;last_name&quot;:&quot;Upert&quot;},{&quot;first_name&quot;:&quot;Gilles&quot;,&quot;last_name&quot;:&quot;Mourier&quot;},{&quot;first_name&quot;:&quot;Annette&quot;,&quot;last_name&quot;:&quot;G. Beck-Sickinger&quot;},{&quot;first_name&quot;:&quot;Karin&quot;,&quot;last_name&quot;:&quot;Mörl&quot;},{&quot;first_name&quot;:&quot;Céline&quot;,&quot;last_name&quot;:&quot;Landon&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;M. G. A. Fontes&quot;},{&quot;first_name&quot;:&quot;Rebeca&quot;,&quot;last_name&quot;:&quot;Miñambres Herráiz&quot;},{&quot;first_name&quot;:&quot;Ricardo&quot;,&quot;last_name&quot;:&quot;C. Rodríguez de la Vega&quot;},{&quot;first_name&quot;:&quot;Steve&quot;,&quot;last_name&quot;:&quot;Peigneur&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Tytgat&quot;},{&quot;first_name&quot;:&quot;Loïc&quot;,&quot;last_name&quot;:&quot;Quinton&quot;},{&quot;first_name&quot;:&quot;Edwin&quot;,&quot;last_name&quot;:&quot;De Pauw&quot;},{&quot;first_name&quot;:&quot;Renaud&quot;,&quot;last_name&quot;:&quot;Vincentelli&quot;},{&quot;first_name&quot;:&quot;Denis&quot;,&quot;last_name&quot;:&quot;Servent&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Gilles&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8250-8264&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00485&quot;},&quot;abstract&quot;:&quot;Animal venoms are rich in hundreds of toxins with extraordinary biological activities. Their exploitation is difficult due to their complexity and the small quantities of venom available from most venomous species. We developed a Venomics approach combining transcriptomic and proteomic characterization of 191 species and identified 20,206 venom toxin sequences. Two complementary production strategies based on solid-phase synthesis and recombinant expression in Escherichia coli generated a physical bank of 3597 toxins. Screened on hMC4R, this bank gave an incredible hit rate of 8%. Here, we focus on two novel toxins: N-TRTX-Preg1a, exhibiting an inhibitory cystine knot (ICK) motif, and N-BUTX-Ptr1a, a short scorpion-CSαβ structure. Neither N-TRTX-Preg1a nor N-BUTX-Ptr1a affects ion channels, the known targets of their toxin scaffolds, but binds to four melanocortin receptors with low micromolar affinities and activates the hMC1R/Gs pathway. Phylogenetically, these two toxins form new groups within their respec&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00485&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00485" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00485&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00485" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00485&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00485" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00485&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00485&amp;href=/doi/10.1021/acs.jmedchem.0c00485" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00485" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00485" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00485%26sid%3Dliteratum%253Aachs%26pmid%3D32602722%26genre%3Darticle%26aulast%3DReynaud%26date%3D2020%26atitle%3DA%2BVenomics%2BApproach%2BCoupled%2Bto%2BHigh-Throughput%2BToxin%2BProduction%2BStrategies%2BIdentifies%2Bthe%2BFirst%2BVenom-Derived%2BMelanocortin%2BReceptor%2BAgonists%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D15%26spage%3D8250%26epage%3D8264%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291206" title="Disulfides">Disulfides</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291735" title="Toxins">Toxins</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/jmcmar.2020.63.issue-15/20200813/jmcmar.2020.63.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Animal venoms are rich in hundreds of toxins with extraordinary biological activities. Their exploitation is difficult due to their complexity and the small quantities of venom available from most venomous species. We developed a Venomics approach combining transcriptomic and proteomic characterization of 191 species and identified 20,206 venom toxin sequences. Two complementary production strategies based on solid-phase synthesis and recombinant expression in <i>Escherichia coli</i> generated a physical bank of 3597 toxins. Screened on hMC4R, this bank gave an incredible hit rate of 8%. Here, we focus on two novel toxins: N-TRTX-Preg1a, exhibiting an inhibitory cystine knot (ICK) motif, and N-BUTX-Ptr1a, a short scorpion-CSαβ structure. Neither N-TRTX-Preg1a nor N-BUTX-Ptr1a affects ion channels, the known targets of their toxin scaffolds, but binds to four melanocortin receptors with low micromolar affinities and activates the hMC1R/Gs pathway. Phylogenetically, these two toxins form new groups within their respective families and represent novel hMC1R agonists, structurally unrelated to the natural agonists.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51442" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51442" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">It is estimated that more than 200,000 venomous species exist on Earth, found in almost all animal subgroups across a range of diverse habitats.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Many are terrestrial, such as some vertebrates (e.g., snakes, platypus, lizards) or arthropods (e.g., scorpions, spiders, centipedes, ticks, ants), some marine, such as cnidarians (e.g., sea anemones and jellyfish) or some mollusks (e.g., cuttlefish, octopus, marine cone snails, terebrids, squids), and some aerial including insects (e.g., mosquitoes, bugs, wasps, bees) and vampire bats. Venoms are used for defense and/or predation and are composed, among other components, of peptide toxins. Toxins are small proteins up to 100 residues in size, reticulated by disulfide bridges. The number of toxins present in venoms depends on the species and on the method used to identify them. In the 1990s, publications mentioned a few tens of toxins in <i>Conus</i> venom. Recently, almost 6000 toxins were reported in the species <i>Conus lividus</i>.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> All in all, we can estimate that at least 40 million toxins exist on the planet, and of these, less than 5000 have been pharmacologically characterized. Despite this very small number, a plethora of activities have been reported. Toxins interacting with membrane receptors like voltage- and ligand-gated ion channels<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> and G-protein-coupled receptors (GPCRs)<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10">(5−10)</a> have been frequently described, whereas other toxins inhibit enzymes or bind membranes to exert hemolytic and/or antimicrobial actions.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Toxins are very often selective (with the exception of some lytic toxins) and make excellent pharmacological tools to better understand the physiological or pathophysiological roles of their targets. In some cases, the toxins themselves or their derivatives have applied clinical uses, such as exenatide from the beaded lizard, a treatment against type-1 diabetes, ziconotide from a cone snail, a treatment for chronic pain, captopril derived from a snake toxin and widely used to treat hypertension, but also some other snake toxins, e.g., batroxobin with clotting properties used to stop bleeding.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> In addition, more than 20 other toxins are currently investigated in clinical trials as biopharmaceutical and therapeutic tools.</div><div class="NLM_p">Venoms are a very promising source of selective bioactive molecules, and considerable effort is being undertaken to discover new toxins with exploitable activities.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Classically, toxins are identified by bioguidage, i.e., successive cycles of venom deconvolution and screening tests against a given target. After defining its sequence, the novel toxin of interest must be produced to allow in-depth characterization. This strategy is long, costly, and risky and requires large amounts of venoms, unavailable for most venomous animals which are very small in size. This strategy is not adapted for the vast majority of this fantastic biodiversity.</div><div class="NLM_p">New sequencing strategies that appeared in the 21st century are revolutionizing the toxinological field. These technologies have been commonly applied to whole genome sequencing; however, eukaryote genomes are typically large with tandem repeats, and only a few genomes from venomous species were reported (several snakes, two scorpions, a handful of spiders, and dozens of Hymenopteras) but none from cone snails. By contrast, many studies have taken advantage of these same strategies for the sequencing of various venom gland transcriptomes. Toxins are produced as precursor proteins by venom-apparatus specific cells and undergo several maturation steps before being secreted.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The formation of disulfide bonds is the most important post-translational modification (PTM), and their arrangement is commonly used to classify toxins.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Sequenced transcripts are translated into their cognate amino acid sequences allowing identification of the endoplasmic reticulum signal peptide, the propeptide and mature peptide regions. While tools like SignalP often accurately predict the endoplasmic reticulum signal peptide boundary, relying on transcriptomics alone, it is far more complicated to obtain robust information regarding further processing to the mature peptide due to the lack of information on most pro-protein convertase specific cleavage sites. Moreover, it is important to note that using a purely transcriptomic strategy, the physical presence in the venoms of most of the sequences described in databases is almost never demonstrated. Alternatively, proteomic strategies study only secreted toxins and provide the opportunity to characterize post-translational modifications (PTMs). Thus, only the combination of these complementary proteomic and transcriptomic strategies leads to a higher level of confidence for mature peptide sequences. Consequently, it is expected that the toxin sequences present in these types of databases will become more and more accurate.</div><div class="NLM_p last">The last and most complicated challenge remains the biological characterization of these natural peptides. This step cannot be done by a simple sequence analysis, especially for peptides with a low sequence similarity with previously characterized toxins. It is necessary to build a physical bank of peptides ready to be screened. The FP7 Venomics Project is part of the biotechnology program “Small-to-Medium Enterprise (SME)-targeted research for developing tools and technologies for high-throughput research”. The total project budget was close to 10 million euros (equivalent to 11 million USD) shared by eight partners from five countries. The vision associated with the Venomics Project is not only to investigate the diversity of venom peptides through the development of “omics” technologies but also to replicate toxin diversity in vitro to generate novel peptide banks to be used in drug discovery programs.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> This unique toxin bank was generated and screened on G-protein-coupled receptor (GPCR) families for which no toxin modulators have previously been described.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50354" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50354" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> A Large Venomous Animal Collection</h3><div class="NLM_p">In total, 323 animal samples from 191 different species were gathered (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Almost every terrestrial and marine venomous clade was represented by at least one species such as the ray (<i>Potamotrygon motoro</i>) or venomous lizard (<i>Heloderma horridum exasperatum</i>). Small terrestrial arthropods represented 95 species, i.e., half of the sourcing. 55 species (29% of the total sourcing) were studied for the first time, including 44 new species from the most infamous venomous clades (Araneae, Conus, Scorpiones, and Serpentes). For instance, 12 previously unexplored species of Neogastropods were sourced including <i>C. canonicus</i> from the Conidae family, the most studied group of Neogastropoda. The Turridae and Terebridae groups also belong to the Neogastropoda order but have been largely neglected compared to their counterpart.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> For instance, among over 4000 described species of Turridae, venom secretions of only 12 species have been studied so far. One of them, <i>Unedogemmula bisaya</i> was included in the sourcing. Concerning the Terebridae family, only seven of around 350 described species have been subject of venom analysis. In Venomics sourcing, two of them, <i>Terebra subulata</i> and <i>T. guttata</i>, were chosen. A third species, <i>T. maculata</i>, never explored before, completed the Terebridae species of the sourcing. Sourcing also included totally unexplored groups such as Mitridae. Two species considered nonvenomous, the <i>Silurus glanis</i> fish and the <i>Trachycephalus resinifictrix</i> frog, but known to produce toxic skin secretions, were also part of the collection. The diverse collection of source animals encompassed various habitats, morphologies, and animal sizes ranging from millimeters (<i>Manica rubida</i> ant) to several meters (<i>Ophiophagus hannah</i> snake).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Selected Taxa for Sourcing</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">taxa</th><th class="colsep0 rowsep0" align="center">main families</th><th class="colsep0 rowsep0" align="center">total no of species</th><th class="colsep0 rowsep0" align="center">never studied species</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">Neogastropoda</td><td class="colsep0 rowsep0" align="left">Conidae</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">others (Turridae, Terebridae, and Mitridae)</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Octopoda</td><td class="colsep0 rowsep0" align="left">Octopodidae</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Chilopoda</td><td class="colsep0 rowsep0" align="left">Scolopendridae</td><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Scorpiones</td><td class="colsep0 rowsep0" align="left">Buthidae, Scorpionidae, Vaejovidae, Scorpionidae, Hemiscorpiidae, and Euscorpiidae</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Araneae</td><td class="colsep0 rowsep0" align="left">Theraphosidae, Ctenidae, Agelenidae, Araneidae, Dipluridae, Lycosidae, Nephilidae, Sparassidae, and Theridiidae</td><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left">23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Hymenoptera</td><td class="colsep0 rowsep0" align="left">Apidae, Vespida, Formicidae, and Myrmicinae</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Chondrichthyes</td><td class="colsep0 rowsep0" align="left">Potamotrygonidae</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Teleostei</td><td class="colsep0 rowsep0" align="left">Synanceiidae, Plotosidae, and Siluridae</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">Squamata</td><td class="colsep0 rowsep0" align="left">Serpentes (Elapidae, Viperidae, and Atractaspididae)</td><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Anguimorpha (Helodermatidae)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Anura</td><td class="colsep0 rowsep0" align="left">Hylidae</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td></tr></tbody></table></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> A Huge Sequence Toxin Diversity in Animal Venoms</h3><div class="NLM_p">De novo transcriptomics opens the door to the study of nature without any previous knowledge of it and with a very high resolution, allowing the unbiased identification and quantification of transcripts in a broad spectrum, even for those of low expression. Total RNA was extracted from venom glands. In order to include species diversity, sample requirements were decreased from 2 μg to 100 ng of RNA, being even as low as 60 ng in specific samples. Illumina specific 2 × 100 pair-end libraries were constructed from each RNA following the manufacturer′s protocols. Over 60 million reads were obtained per sample using a Hiseq 2000 sequencing platform with an average length of 500 nucleotides.</div><div class="NLM_p">Two proteomic strategies were used according to the size of the toxins.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The first one, for venoms presenting most of the toxins with less than 25 amino acid residues (like Neogastropoda, Hymenoptera, Anura), analyzed the peptide post-disulfide-reduction by liquid chromatography coupled to tandem mass spectrometry. The second strategy, dedicated to the sequencing of long venom toxins (>25 amino acid residues (like Chilopoda, Scorpiones, Araneae, Squamata)), required additional alkylation and trypsin digestion (short time and low enzyme concentration) steps after disulfide treatment leading to the generation of overlapping peptides (±15 amino acids).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> These resulting peptides were then analyzed with the same mass spectrometry platform. The number of sequences varied between fifty to several hundreds, depending on the venom. A high number of sequences are observed in Conidae venoms by proteomics. This diversity relies on the presence of post-translational modifications at different sites in a single toxin and has been discussed in previous publications.<a onclick="showRef(event, 'ref2 ref20'); return false;" href="javascript:void(0);" class="ref ref2 ref20">(2,20)</a> A total of 21,300,000 sequence tags were obtained.</div><div class="NLM_p">Proteomic and transcriptomic data generated from the same animal were cross-validated to extract only sequences present in both the venom and venom gland transcripts, resulting in 20,206 validated toxin sequences (defined here as a peptide up to 110 residues with at least one disulfide bridge). <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> describes these sequences in term of origins, disulfide bridges (1 to 9), sizes (6 to 110), and PTMs. The quasi-totality of Conus sequences (98%) bears at least one PTM, in addition to disulfide bridges. Amidation is found in 2889 sequences (14%) from all groups studied except Terebridae, Turridae, Mitridae, and Teleostei. Hydroxylation on Pro, Met, Lys, or Val residues was found in 2630 sequences (13%) from various groups with a great majority from Conus (70%). In our study, 4-carboxyglutamic acid, pyroglutamic acid, and bromo-tryptophan appeared to be Conus-specific. Strangely, we did not find pyroglutamic acid in Arachnid venoms (Scorpiones and Araneae), although its presence has been described previously in many other studies.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> The 20,206 toxin sequences were divided into 137 different cysteine patterns, 97 with an even number of cysteines. Among these 97 patterns, 38 present new cysteine patterns never described in any venomous species. Regarding the 59 known patterns, the C-C-CC-C-C framework is the most represented comprising 19% of the bank (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Table S1</a>). It is related to the inhibitory cystine knot toxin (ICK), which is known to be particularly abundant in spider, snail, and scorpion venoms.<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> For the first time, ICK peptides were identified at the protein level from Mitridae, Turridae, <i>M. rubida</i> ant, Chilopoda, and the octopus <i>Hapalochlaena lunulata</i><i>.</i></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Richness of animal origin and structural features (number of disulfide bridges, length, and post-translational modification) of the Venomics bank (lower panel) compared to the original Omics bank (upper panel). The Omics bank represents 20,206 sequences identified both in transcriptomic analysis of the venom glands and proteomic analysis of the venoms. From it, 3597 peptides were produced to build the Venomics bank, which closely replicates the origin and structural characteristics of the primary Omics bank.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Interestingly, 874 sequences covering 40 cysteine patterns display an odd number of cysteine residues (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Table S2</a>). Only 13 were previously described. Two patterns (both lacking vicinal cysteine residues) are particularly abundant: C-C-C-C-C-C-C with 130 sequences (Scorpiones, Serpentes, Octopoda, Conus) and C-C-C-C-C specific to the Araneae taxon with 179 sequences. We did not identify any sequences from Conus sharing the latter pattern despite it being previously described.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><div class="NLM_p last">Among these 20,206 sequences, 782 (4%) display a new cysteine pattern and 4393 (22%) share less than 50% sequence identity with any described peptide or protein in the UniprotKB/SwissProt database using default settings (e-value max = 1). It must be noted that the cysteine connectivity of the native toxins cannot be explicitly determined using this pipeline.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> From Sequences to a High-Throughput Screening Compatible Toxin Bank</h3><div class="NLM_p">In order to allow screening and characterization of peptide activity, we developed two high-throughput production protocols. 5923 sequences representing the broad diversity of the sequence database were selected and produced in the dedicated time of 18 months. Coming from 64 different species, they cover 91 different cysteine patterns, including 14 with an odd number of cysteine residues (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Tables S1 and S2</a>).</div><div class="NLM_p">We selected 931 peptides shorter than 35 residues, with or without PTMs, to be produced by solid-phase chemical synthesis.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The crude synthetic product was cleaved from the resin, desalted, and subjected to an optimized oxidation process, whatever the number or pattern of cysteines.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> After a final desalting step, liquid chromatography and mass analysis were performed to validate the presence of the expected oxidized mass (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Figure S1</a>). 893 toxins from 9 to 34 residues with 1 to 3 disulfide bridges were successfully produced, representing a success rate of 96%.</div><div class="NLM_p">The remaining 4992 sequences containing more than 34 residues were selected for heterologous expression in <i>Escherichia coli</i>.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29−31)</a> All synthetic genes encoding toxin sequences were subcloned into an optimized periplasmic expression plasmid containing an N-terminal <i>E. coli</i> disulfide bond isomerase C (DsbC) chaperone fusion tag, a 6xHIS tag for purification and a tobacco etch virus (TEV) protease cleavage site to release the native peptide from the fusion protein. Expressed fusion proteins are purified on a nickel resin, subjected to overnight TEV cleavage and finally desalting. The correct mass was confirmed by mass analysis (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Figure S2</a>). 2704 toxins from 35 to 110 residues with 1 to 9 disulfide bridges were successfully produced representing a success rate of 54%.</div><div class="NLM_p">Determining disulfide connectivity in native toxins is a challenge, and the Venomics project did not have the resources to carry out this task. In the absence of this information, cysteine residues were randomly oxidized and the vast majority of toxins presented a unique peak in LC, suggesting that they adopted only one conformation, hopefully the natural one. In the case where LC profiles presented two or three peaks with the good expected oxidized mass, we assumed that it could be due to the generation of peptide isomers<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Figures S1 and S2</a>).</div><div class="NLM_p last">Together, these two strategies resulted in a bank of 3597 peptides composed of 71 different cysteine frameworks (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Tables S1 and S2</a>). In addition, 158 toxins (4%) display a novel cysteine pattern, and 1026 (29%) share less than 50% sequence identity with any previously described peptide or protein in the UniprotKB/SwissProt database. All together, we have successfully translated the high complexity of the Omics bank into a physical toxin bank via high-throughput methods.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> First Animal Toxins on Melanocortin Receptors</h3><div class="NLM_p">Animal toxins are mostly known for their bioactivity on ion channels. Few but very interesting toxins active on GPCRs have also been identified.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> For example, the commercially available type 2 diabetes treatment, exenatide derived from a venomous lizard, targets a class A GPCR. Belonging to the same class, the human melanocortin 4 receptor (hMC4R) regulates food intake, and its genetic invalidation is responsible for morbid obesity. Screening of the Venomics bank on hMC4R gave an amazing high hit rate of 8%, 163 coming from chemical synthesis and 123 from recombinant expression. We decided to further investigate two hits displaying cysteine patterns related to two of the largest structural families in animal toxins, the ICK-related pattern (-C-C-CC-C-C-) and the CSαβ-related cysteine patterns (-C-C-C-C-C-K-C-), which are primarily renowned for their activities on ion channels. The ICK-related N-TRTX-Preg1a comes from the spider <i>Poecilotheria regalis</i>, while the CSαβ-related N-BUTX-Ptr1a comes from the scorpion <i>Parabuthus transvaalicus</i>. The N letter (uppercase nu Greek letter) has been chosen as the activity descriptor for modulators of hMCRs. In primary screening on hMC4R, N-TRTX-Preg1a and N-BUTX-Ptr1a both showed strong inhibition of <sup>125</sup>I-NDP-α-MSH, 97 and 86%, respectively (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Further pharmacological investigations were then done with newly synthetically produced toxins of more than 95% purity (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Figure S3</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Screening results on hMC4R from the two plates containing N-TRTX-Preg1a (A) and N-BUTX-Ptr1a (B) peptides in the Venomics bank. Screening on hMC4R, stably expressed in CHO cells, was based on competitive binding on the receptor of the tested product against the radiolabeled ligand, <sup>125</sup>I-NDP-α-MSH. The full red line is the median, and the red dashed lines are median ± 3 median absolute deviation. Dots above or under the dashed lines are hits. Each dot corresponds to a tested Venomics peptide (80 peptides by plate). The colored dot represents either N-TRTX-Preg1a (in green) or N-BUTX-Ptr1a (in purple). Preg1a: N-TRTX-Preg1a. Ptr1a: N-BUTX-Ptr1a.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analytical HPLC chromatogram of the final purified products of N-TRTX-Preg1a and N-BUTX-Ptr1a. Between 10 and 15 μg of each toxin were analyzed on a X Bridge BEH 300 Å 5 μm 4.6 × 250 mm with a gradient of 10–50% CH<sub>3</sub>CN/TFA 0.1% in 40 min at a flow rate of 1 mL/min. A spontaneous transformation of the N-terminal glutamine (Q form) residue of the peptide N-BUTX-Ptr1a into a pyroglutamic acid (Z form) can be detected after crude peptide deprotection. According to LC–MS analysis performed using an Elute Sp Pump HPG 700 (Bruker) coupled with an AmaZon SL LC/MS (Bruker), the peak at 18.077 min corresponds to the Q form (average mass of 3795.70 <i>m</i>/<i>z</i>) and represents 96.5% of the total area. The peak at 18.802 min with an average mass of 3778.70 <i>m</i>/<i>z</i> seems to correspond to the Z form of the peptide and represent 3.5% of the total area.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">N-TRTX-Preg1a and N-BUTX-Ptr1a display low micromolar affinities on membrane preparations of every studied subtype of melanocortin receptor stably expressed in CHO-K1 cells for hMC1R, hMC4R, hMC5R, and HEK293 cells for hMC3R (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). A final check was performed by measuring the affinities of the linear toxins with cysteines alkylated by iodoacetamide treatment. Up to 100 μM (1, 10, and 100 μM concentrations were tested), unstructured N-TRTX-Preg1a and N-BUTX-Ptr1a did not inhibit the binding of <sup>125</sup>I-NDP-α-MSH on hMC1R and hMC4R, demonstrating that the three-dimensional structures of the toxins are essential to their activities (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Their agonistic and antagonistic activities on the signaling pathways hMC1R/Gs and hMC4R/Gs were then assessed on HEK293 cells. As control, it was demonstrated that neither toxin induced any effect on control hMCR-free HEK293. Both toxins tested at 100 μM induced cAMP production on hMC1R, similar to 1 nM of α-MSH, while no effect was observed on hMC4R (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). In the presence of 1 nM α-MSH, the toxins slightly enhanced hMC1R-related cAMP production but did not reduce the cAMP production related to hMC4R (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). Both toxins were also tested at 100 μM on several human sodium channels (Na<sub>V</sub>1.1, Na<sub>V</sub>1.2, Na<sub>V</sub>1.3, Na<sub>V</sub>1.4, Na<sub>V</sub>1.5, Na<sub>V</sub>1.6, Na<sub>V</sub>1.7, and Na<sub>V</sub>1.8), potassium channels (K<sub>V</sub>1.1, K<sub>V</sub>1.2, K<sub>V</sub>1.3, K<sub>V</sub>1.4, K<sub>V</sub>1.5, K<sub>V</sub>1.6, K<sub>V</sub>2.1, K<sub>V</sub>3.1, K<sub>V</sub>4.3, K<sub>V</sub>7.1, K<sub>V</sub>7.3, K<sub>V</sub>10.1, K<sub>V</sub>11.1(hERG), GIRK1), a voltage-gated calcium channel (Ca<sub>V</sub>3.3), and ligand-gated ion channels nAChR α1<sub>2</sub>β1γδ and nAChR α7. No activity could be detected (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Figures S4 and S5</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacological characterization of N-TRTX-Preg1a and N-BUTX-Ptr1a. Representative curves of binding of <sup>125</sup>I-NDP-α-MSH on membrane preparations of hMC1R, hMC4R, and hMC5R (stably expressed in CHO cells) and on hMC3R (stably expressing in HEK293 cells) in the presence of N-TRTX-Preg1a (left panel) or N-BUTX-Ptr1a (right panel). Mean (%) ± SD (<i>n</i> = 2–3). Open squares represent peptides with alkylated cysteines by iodoacetamide treatment, tested on hMC1R and hMC4R (A). Functional assays on HEK293 (gray), HEK293 stably expressing hMC1R (blue), or HEK293 transitory expressing hMC4R (red). cAMP production expressed as % of the positive control (1 μM forskolin) (B). Preg1a: N-TRTX-Preg1a. Ptr1a: N-BUTX-Ptr1a. Mean (%) ± SEM (<i>n</i> = 3), ***: <i>p</i> < 0.001; one-way analysis of variance (ANOVA) followed by Dunnett’s multiple comparison test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibition Constant (<i>K</i><sub>i</sub>) Values of N-TRTX-Preg1a and N-BUTX-Ptr1a on Melanocortin Receptors, Mean (μM) ± SEM (<i>N</i> = 3–10)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="±">hMC1R</th><th class="colsep0 rowsep0" align="center" char="±">hMC3R</th><th class="colsep0 rowsep0" align="center" char="±">hMC4R</th><th class="colsep0 rowsep0" align="center" char="±">hMC5R</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">N-TRTX-Preg1a</td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">19.8 ± 3.5</td><td class="colsep0 rowsep0" align="char" char="±">7.1 ± 1.5</td><td class="colsep0 rowsep0" align="char" char="±">10.0 ± 4.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">N-BUTX-Ptr1a</td><td class="colsep0 rowsep0" align="char" char="±">2.9 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">3.9 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">2.6 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">2.2 ± 0.8</td></tr></tbody></table></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Are N-TRTX-Preg1a and N-BUTX-Ptr1a Real ICK-Related and CSαβ-Related Scaffold Toxins?</h3><div class="NLM_p">The two N-TRTX-Preg1a and N-BUTX-Ptr1a toxins have a cysteine pattern corresponding to the ICK-related and the CSαβ-related structures with the following pairing: C1-C4, C2-C5, C3-C6. As both toxins display original pharmacological activities, we characterized both structures by NMR and determined experimentally their cysteine connectivity. Both peptides are well-folded, illustrated by their good dispersion of resonances. All the <sup>1</sup>H (except for the methyl groups of Met2 and Met28 of N-BUTX-Ptr1a), the <sup>15</sup>N chemical shifts, and the <sup>13</sup>C resonances of Cα and Cβ were assigned. Depositions in the BioMagResBank were done with the entry code 34117 for N-TRTX-Preg1a and 34,118 for N-BUTX-Ptr1a.</div><div class="NLM_p">A set of 546 NOEs for N-TRTX-Preg1a were used in the final ARIA run, which includes 454 unambiguous restraints (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Table S3</a>). Additional constraints were introduced during the iterative ARIA procedure: 38 dihedral angle restraints, 10 H-bond constraints observed within the β-strands, and 3 covalent bonds between sulfur atoms (C2-C19, C9-C22, C18-C27). Indeed, when enough experimental data have been introduced into the first runs of the iterative ARIA process, the C18-C27 disulfide bond is formed in two-thirds of the N-TRTX-Preg1a structures. The fine analysis of NOE constraints and the evaluation of distance between sulfur atoms allowed discrimination between the two remaining possibilities: C2-C19, C9-C22, C18-C27 (favorable) or C2-C22, C9-C19, C18-C27 (excluded). Therefore, the corresponding covalent bonds were added between sulfur atoms to refine the N-TRTX-Preg1a structure in the last runs of the calculations. The 10 models representative of the solution structure (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A) display a very low RMSD, small potential energy values, respect all the experimental data, and are in perfect agreement with standard covalent geometry with 64.8% of the residues in the most favored regions of the Ramachandran plot and 35.2% in additional allowed regions (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Table S3</a>). The N-TRTX-Preg1a structure is very well defined, organized into the ICK motif without ambiguities regarding cysteine pairing. It is composed of a short antiparallel β-sheet made up of two short strands (22–23 and 26–27). The very short turn between these two strands, containing two arginine residues (R24 and R25), is highly constrained. Note the presence of an extended region (7–9), which faces the two-stranded β-sheet, resembling a third quasi-β-strand. Proline residue 12 is in a majority <i>trans</i> configuration (about 80%). The N-TRTX-Preg1a sequence produced only a handful of significant hits in public databases and shares only 61% identity with the closest toxin with a known activity, Hs1a from <i>Cyriopagopus schmidti</i> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C), a calcium channel blocker.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structural characterization of N-TRTX-Preg1a and N-BUTX-Ptr1a. Primary structures and overlay of the 10 NMR models representative of the solution structures of N-TRTX-Preg1a (A) and N-BUTX-Ptr1a (B). N and C termini are indicated. Comparison of amino acid sequences between N-TRTX-Preg1a (C) or N-BUTX-Ptr1a (D) and the most similar toxins obtained with the NCBI Protein Blast program (default settings).<a onclick="showRef(event, 'ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44">(35−44)</a> Cysteine residues are in bold, and conserved residues from N-TRTX-Preg1a or N-BUTX-Ptr1a are in green. Sequence alignment was performed with Clustal Omega (version 1.2.4 from Emboss programs, EBlosum62 matrix for two-pair alignment).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Regarding structural determination of N-BUTX-Ptr1a, 839 NOEs (including 558 unambiguous distance restraints) and 4 hydrogen bonds present within the β-sheet were used in the final ARIA run (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Table S4</a>). 92% of the 250 final structures display a C1-C4, C2-C5, C3-C6 disulfide bridge array, without adding any constraint on the bonds. The 10 models representative of the solution structure (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) display low root-mean-square deviations (RMSD), small potential energy values, and respect all the experimental data, as attested by the absence of residual violations (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Table S4</a>). The Ramachandran plot exhibits 53% of the (ϕ,ψ) angles in the most favored regions and 40% in the additional allowed regions. The N-BUTX-Ptr1a structure scaffold is a well-known motif within scorpion toxins,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> plant defensins,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and insect defensins<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and was designated as a cysteine-stabilized α-helix β-sheet motif (CSαβ motif).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The structure is composed of an α-helix (11–19) and a two-stranded short β-sheet (26–28; 31–33), stabilized by three disulfide bridges (C6-C27, C12-C32, C16-C34) unambiguously assigned. N-BUTX-Ptr1a shares the same sequence pattern as the majority of α-KTx toxins known to block potassium channels: C-x(4,5)-C-x(3)-C-x(6,9)-[GAS]-K-C-[IMQT]-x(3)-C-x-C (Prosite PS01138). Yet, N-BUTX-Ptr1a shares only 59% identity with the closest toxin with a known activity, Parabutoxin-3 from the same species <i>P. transvaalicus</i> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D), a potassium channel blocker.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The coordinates of structures have been deposited in the Protein Data Bank (<a href="http://www.pdb.org" class="extLink">http://www.pdb.org</a>) with the entry codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAA">6SAA</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAB">6SAB</a> for N-TRTX-Preg1a and N-BUTX-Ptr1a, respectively.</div><div class="NLM_p last">While N-TRTX-Preg1a and N-BUTX-Ptr1a display the canonical ICK and CSαβ structures, respectively, their sequences diverged so much from most described toxins that homology searches or phylogenetic reconstruction failed to assign them into known toxin groups. On the one hand, N-TRTX-Preg1a matched only a handful of spider toxins that could not be linked to the main group of spider ICK toxins. On the other hand, phylogenetic reconstruction with all known alpha-KTx (UniProt) placed N-BUTX-Ptr1a as a sister to a clade consisting of several monophyletic alpha-KTx subfamilies (alpha-KTx 2, 4, and 12).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05008" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05008" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To our knowledge, this is the first time that a thorough “-omic” analysis of venoms has been carried out on such a wide number and diversity of venomous species from various habitats (128 terrestrial, 55 aquatic, and 8 aerial) including two European species (<i>M. rubida</i> ant and <i>Buthus occitanus</i> scorpion). The <i>M. rubida</i> ant was the smallest species of the project, studied for the first time, and 35 sequences were validated. The number of validated sequences ranges from zero, due to experimental failures (some collecting conditions were not compatible with proper mRNA extraction), to 1163, in the venom of the Mediterranean scorpion <i>B. occitanus</i>. Finally, no validated toxin sequences could be detected in the skin secretions of <i>S. glanis</i> fish and the <i>T. resinifictrix</i> frog. An average of 106 toxin sequences per venom was validated, much less than what could be expected according to the assumed richness of venoms. This difference highlights the difficulties in obtaining full and unambiguous toxin sequences. The gotten sequence diversity also highlights our very limited state of knowledge of the toxin world. Indeed, almost one-quarter of the toxins share less than 50% sequence identity with known toxin sequences, suggesting unpredictable and original biological activities. Furthermore, 65 new naturally occurring cysteine patterns have been found, suggesting 65 new structural organizations. This is an opportunity to discover peptides with attractive biophysical properties in terms of engineering and thermodynamic stability for optimal drugability. Another level of complexity, not investigated here, is the dimerization (homo- and/or hetero-) of toxins containing an odd number of cysteines. A specific proteomic analysis should be designed to address this point.</div><div class="NLM_p">Converting a sequence database into physical peptides is challenging, in particular for the oxidation of disulfide bridges. Our recombinant expression strategy was able to directly produce oxidized peptide and has been proven to be effective in terms of high-throughput production. The two-step chemical synthesis strategy was also extremely efficient. This is mainly due to the use of a universal folding strategy applied on all toxins regardless of size or the number of cysteines. These production strategies provided peptides with impurities as no high-efficiency purification step was performed. While the presence of these contaminations could have caused numerous false positive results during the screening tests,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> we have shown that this was not the case as 90% of the tested hits were confirmed.</div><div class="NLM_p">GPCRs constitute among the largest and most versatile families of receptors. They are involved in a wide spectrum of physiological functions. Since the late 20th century, pharmaceutical companies have significantly diminished their natural product programs, partly because of the advances in both HTS and combinatorial synthesis, thereby the creation of large synthetic libraries of small but artificial molecules. Natural product libraries consist of extracts and partially purified fractions and are perceived to be incompatible with HTS. Unfortunately, this loss of interest in natural products correlated with the overall reduction in new leads in the drug development pipeline and the substantial decline in new drug approval.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The Venomics bank reconciles venoms, a particularly difficult exploitable resource of natural peptides, with HTS. The screening of the Venomics bank provided an extraordinary hit rate of 3%, much higher than the one reported in the literature concerning artificial molecules and estimated at around 0.05%.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> To our knowledge, this is the first large-scale demonstration of the power of natural peptide toxins to provide GPCR ligands. This strategy opens up an unexplored avenue for the discovery of new ligands on receptors linked to diseases with unmet medical needs such as obesity.</div><div class="NLM_p">MC4R is mainly expressed in the paraventricular nucleus of the hypothalamus. Its activity is finely regulated by both anorexigenic agonists, the melanocyte stimulating hormones (MSH), and the orexigenic antagonist, Agouti-related protein (AgRP).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Loss-of-function mutations on hMC4R are the primary cause of monogenic obesity (<a href="http://www.orpha.net/" class="extLink">http://www.orpha.net/</a>) making it a therapeutic target in eating disorders. Currently, only one MC4R ligand is in clinical trials (Setmelanotide, <a href="https://clinicaltrials.gov/" class="extLink">https://clinicaltrials.gov/</a>). Subtype 1 (MC1R) is also of therapeutic interest with its protective role against melanoma. It elicits eumelanin production, a dark, UV radiation-absorbing, photoprotective pigment,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and promotes nucleotide excision repair of DNA damaged by UV radiation.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> There is an urgent need for development of innovative ligands to better understand MCR functions and to generate new medicines. N-TRTX-Preg1a and N-BUTX-Ptr1a are the first MCR venom-peptide ligands, structurally unrelated to natural endogenous MCR agonists, while similarities and differences exist with other natural ligands such as human Agouti-related protein (hAgRP) or human β-defensin 3 (HBD3). As N-TRTX-Preg1a, hAgRP displays an ICK fold in its C-terminal part (hAgRP 83–132) representing its hypothesized mature form<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> with two additional disulfide bridges. As presented in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, cysteine residues involved in the ICK fold in N-TRTX-Preg1a and hAgRP 83−132 are well aligned. In addition, a similarly β-sheets, stabilized by the last cysteine residues of the ICK pattern, are present in the two peptides. Composition and loop sizes are quite divergent between the two peptides especially for the third loop, which is almost two times longer in hAgRP 83–132 (C3-C9). This loop contains the Arg-Phe-Phe pharmacophore of AgRP<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> that is lacking in N-TRTX-Ptr1a. This difference suggests that despite sharing the same structural ICK family, N-TRTX-Preg1a and hAgRP 83–132 have a different mode of action on MCRs. Regarding N-BUTX-Ptr1a, we noticed that it shared with HBD3 interesting features. They both display similar secondary structures composed of a short α-helix and several antiparallel β-sheets reticulated by three disulfide bridges (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). In addition, they are both rich in cationic residues with high basic pIs of 10 for HBD3 and 9.3 for N-BUTX-Ptr1a. This last remark was of particular interest as, in a previous study, Nix et al. demonstrated the role of large patches of positively charged residues in the binding of HBD3 on melanocortin receptors.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> In the same study, the authors described nanomolar affinities on melanocortin receptors (42 nM on hMC1R and 110 nM on hMC4R), but more recently, Ericson et al. published micromolar affinities, 7 μM on hMC1R and 100 μM on hMC4R.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> According to Ericson et al., this difference may be explained by a discrepancy between protocols. We used herein the same methodology as in Ericson et al. but with a longer incubation time (3 versus 1 h). Without testing HBD3 in our conditions, we cannot be sure that HBD3 and N-BUTX-Ptr1a share or not similar affinities on melanocortin receptors. Nevertheless, the micromolar affinity of HBD3 on hMC1R was supported by functional results that described the peptide as a micromolar agonist of the Gs pathway on hMC1R (EC<sub>50</sub> = 0.40 μM),<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> a feature shared with N-BUTX-Ptr1a. Despite these similarities, important differences between the two peptides have also been noticed. First, they do not share global three-dimensional structures, surely linked to the difference in the cysteine pattern (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Second, Nix et al. showed that the absence of disulfide bridges, by cysteine replacement in serine in the HBD3 sequence, decreased HBD3 affinities by less than 10 times<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> contrary to the linear N-BUTX-Ptr1, which lost at least 100 times in affinity. Therefore, the structural constraints induced by disulfide bridges may be more important in N-BUTX-Ptr1a binding than in HBD3. These two structural differences may be linked in distinct interaction profiles of the two peptides with melanocortin receptors.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structure comparison of the spider and scorpion toxins with melanocortin receptor structured peptide ligands. N-TRTX-Preg1a versus human Agouti-related protein 83–132 (hAgRP 83–132). Upper panel: primary (Needl, version 6.6.0 from Emboss programs, EBlosum62 matrix) and secondary structure (arrow: β-sheet) alignment. Cysteine residues and associated disulfide connectivity are in bold and orange when implied in the ICK motif or in bold and black otherwise. The hAgRP Arg-Phe-Phe is underlined. Lower panel: overlay of N-TRTX-Preg1a and hAgRP 87–132 (gray, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HYK">1HYK</a><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a>) NMR structures. The aligned ICK part of hAgRP is represented in colors. The hAgRP pharmacophore residues (Arg-Phe-Phe) have developed side chains (A). N-BUTX-Ptr1a (Ptr1a) versus human β-defensin 3 (HBD3). Upper panel: primary and secondary structures (arrow: β-sheet, tube: α-helix). Cysteine residues and associated disulfide connectivity are in orange. Basic residues are in blue. Lower panel: tertiary NMR structure comparison of N-BUTX-Ptr1a and HBD3 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KJ6">1KJ6</a><a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a>). Structures display the solvent-exposed surface area with basic residues in blue. “C” and “N” correspond to the N- and C-termini of the peptides, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">N-TRTX-Preg1a and N-BUTX-Ptr1a also lack the HFRW pharmacophore, described in endogenous agonists as the minimal active sequence,<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62,63)</a> suggesting that they activate hMC1R in a different way, which would be interesting to deeply study. In addition, both toxins promote a synergistic effect with endogenous agonist α-MSH. This activity deserves to be finely studied as both ligands may activate hMC1R by a new molecular mechanism. N-TRTX-Preg1a and N-BUTX-Ptr1a inhibit <sup>125</sup>I-NDP-α-MSH binding on hMC4R but are not able to modulate, activate, or antagonize the hMC4R/Gs pathway, under our experimental conditions. This discrepancy can come from our experimental procedure. We perhaps did not incubate enough time as the toxins display low affinities and may need a longer time to bind the receptor. Indeed, functional tests are designed according to the pharmacological characteristic of the agonist, which is nM. Another possibility is that these toxins display biased functions, with activities on other pathways than the Gs one. A deeper exploration of the functions of the two toxins on various MC4R signaling pathways (Gq, β-arrestin, kinases, etc.) will be performed to better understand their modes of action.</div><div class="NLM_p">N-TRTX-Preg1a exhibits a common ICK fold only described so far for activity on ion channels. Its sequence is close to calcium channel toxins but does not possess the tyrosine residue in position 16 (N-TRTX-Preg1a numbering), described to be essential for blocking Ca<sub>V</sub>2.2 channel conductivity.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> N-BUTX-Ptr1a, close in sequence to potassium channel toxins, possesses a dyad composed of a basic and an aromatic residue<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65,66)</a> (F24 and K26, N-BUTX-Ptr1a numbering). This dyad is stated as essential for potassium activity,<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65,66)</a> but we showed that this is clearly not sufficient for N-BUTX-Ptr1a to bind potassium channels. N-TRTX-Preg1a and N-BUTX-Ptr1a sequences diverge so much within their own group that it was even impossible to phylogenetically anchor them into known toxin families. Thus, they constitute new clades for ICK and CSαβ families. Several other toxins of the produced bank sharing more than 80% of sequence identity with these two toxins are present in the Venomics database, suggesting that N-TRTX-Preg1a and N-BUTX-Ptr1a are not the only members of these new families. Production and pharmacological characterization of these natural variants should provide interesting insights into the mode of action of these toxins on hMCRs. This may aid the generation of ligands with improved affinity and selectivity for development of new hMC1R agonists with applications in the treatment and/or prevention of melanoma.</div><div class="NLM_p last">To our knowledge, the Venomics bank is the largest one ever built. We demonstrated that it represents a real opportunity to identify ligands with unusual structures and pharmacological properties. Its high hit rate shows that quality can beat quantity. Two toxins, with classical toxin structures but original pharmacological activities, highlight the diversity of toxin properties and open the door to the development of new hMCR modulators.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67602" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67602" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Sourcing</h3><div class="NLM_p last">We paid important caution to respect the natural resources to which we had access. Thus, we limited the number of collected specimens to the strict number necessary to ensure that the amounts and volumes of biological samples were compatible with the experiments to be carried out within the Venomics consortium. For instance, during the Mayotte and Polynesia expeditions, supernumerary cone snails collected during the day were systematically dropped back to water. In the case of the venomous lizard specimen (<i>H. horridum exasperatum</i>), the lizard was anesthetized while a veterinary surgeon was performing surgery to remove only one of the venomous salivary glands of the animal. The animal fully recovered. This demonstrates the applicability of such a strategy of tissue extraction that could certainly be applied with success to rare snakes, too.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Transcriptomic Analysis</h3><div class="NLM_p last">The Illumina paired-end NGS platform was selected to assure sequence and data quality. 203 RNA samples from different venomous species were sequenced through a low-input method. Quality and quantity of total RNA was determined using a Bioanalyzer 2100 (Agilent, California) and Qubit 2.0 (Thermo Fisher, Massachusetts). The poly(A) + mRNA fraction was isolated from total RNA, and cDNA libraries were obtained following Illumina′s recommendations. Briefly, poly(A) + RNA was isolated on poly-T oligo-attached magnetic beads and chemically fragmented prior to reverse transcription and cDNA generation. cDNA fragments then went through an end repair process, the addition of a single “A” base to the 3′ end and then ligation of the adapters. Finally, the products were purified and enriched by PCR to create the indexed final double-stranded cDNA library. Library quality was analyzed via a Bioanalyzer 2100 high-sensitivity assay, and quantity was determined by real-time PCR using a LightCycler 480 (Roche, Basilea, Switzerland). The pool of libraries was sequenced by paired-end sequencing (100 × 2) using an Illumina HiSeq 2000 sequencer (Illumina, California). Bioinformatic analysis was a crucial step of the process to identify peptide sequences. This analysis started with NGS raw data quality control followed by exhaustive cleaning of primers, adapters, indexes, and low-quality bases. The next step consisted of joining the paired-end reads and dereplicating all reads before entering the assembly step itself. To assure the quality of the assembly, two different assemblers were used: Velvet (v1.2.10)<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> and Trinity (v2.1.1),<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> creating a robust and comprehensive de novo assembly pipeline.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Proteomics Analysis</h3><div class="NLM_p last">Venoms predominantly containing toxins with less than 25 amino acid residues such as cone snails and wasps were treated with dithiothreitol (10 mM final concentration, 1 h at 56 °C) and alkylated with iodoacetamide (20 mM final concentration, 1 h at room temperature). Reduced and alkylated peptidic toxins were separated by reverse phase chromatography (nanoLC-UPLC Acquity, Waters, Milford, Massachusetts, United States) and analyzed by tandem mass spectrometry (Q-Exactive, Thermo Fisher, Waltham, Massachusetts, United States). This “top-down” proteomics approach has the advantage of measuring the mass of the intact full-size toxins but also allows for sequence determination using collision-based fragmentation mass spectrometry experiments. Venoms containing large toxins, such as snake, spider, and scorpion venoms, were, after reduction and alkylation, enzymatically cleaved (by using a ratio of 1:50 protein/trypsin, with incubation at 37 °C over 1 h). These cleavage conditions yielded truncated peptides cleaved at different positions, with several overlapping peptides covering the full toxin sequence. The peptides were then analyzed with the same nanoUPLC–MS/MS platform, in the same way as the smaller toxins permitting the full toxin sequences to be determined. As large toxins have been transformed into a collection of smaller peptides, the fragmentation yield is then much better and MS/MS spectra can provide useful sequence information.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Sequence Validation</h3><div class="NLM_p last">For the transcriptomics data, different databases were generated: (1) the first one including genes coding for peptidic toxins by using predictive bioinformatics tools and (2) the rest of the venom gland transcriptome translated into the six possible open reading frames. The bioinformatics data analysis has been supported by Peaks software (Bioinformatics Solutions Inc., Waterloo, Canada). In addition to classical data analysis, Peaks was also used to de novo sequence peptides not identified from the databases and to predict mutations with Spider algorithms. All the peptides discovered were classified, and only those with a false discovery rate of less than 0.1% were considered for inclusion in the Venomics database.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> High-Throughput Peptide Production by Solid-Phase Synthesis</h3><div class="NLM_p last">The high-throughput chemical production of toxins up to 34 residues with or without post-translational modifications (PTMs) was undertaken by solid-phase peptide synthesis using an automated Prelude synthesizer (Protein Technologies, USA) equipped with six reaction vessels using the Fmoc strategy. Amino acids were coupled twice for 10 min using five or twenty equivalents for toxins up to 25 residues or between 26 and 34 residues, respectively, using a ChemMatrix or preloaded Wang resins using HCTU as a coupling reagent. Both strategies led to toxins with excellent purities higher than 50%. After a desalting step on homemade C18 columns and a drying step, the refolding of the different toxins was performed directly on the crude material. A universal folding protocol was applied for each toxin as already described<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> using 50 mM HEPES buffer pH 7.5, 20% acetronitrile, 1 mM cysteine, 0.1 mM cystine, over 48 h at 4 °C. After a desalting step and analytical characterizations by analytical HPLC using a Waters e2695 separation module equipped with an XBridge BEH300 C18 column (5 μm, 4.6 × 250 mm, Waters) and by mass spectrometry using a MALDI-TOF spectrometer (4800 MALDI TOF-TOF Analyzer from Applied Biosystems, USA), the toxins were distributed in 96-well plates at concentrations of 2–120 μM.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> High-Throughput Peptide Production by Recombinant Expression</h3><div class="NLM_p last">Optimization of gene synthesis and expression clone constructs for venom-peptide production were published previously,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> in addition to the final protocols used in this project.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Briefly, genes encoding animal venom peptides were designed with codon optimization for expression in <i>E. coli</i> using the ATGenium codon optimization algorithm (NZYTech genes & enzymes, Portugal). Synthetic genes were produced in a high-throughput pipeline, and purified PCR products were subsequently cloned into the pHTP4 expression vector using the NZYEasy Cloning kit (NZYTech genes & enzymes, Portugal), according to ligation independent cloning (LIC) technology protocols. In pHTP4, peptide genes were placed under the control of a T7 promoter and were produced as fusion proteins in the <i>E. coli</i> periplasmic space. Fusions contain an N-terminal disulfide bond isomerase C (DsbC) fusion, an internal hexa-histidine tag for purification and a TEV protease recognition sequence to allow cleavage and isolation of the native peptides. Optimization of expression and purification conditions<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> and the final recombinant production protocols used herein<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> were described previously. All steps were carried out in 24-deep-well plates or in 96-deep-well plates on a Tecan Freedom EVO 200 robot (Switzerland).<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> pHTP4 expression plasmids were transformed into BL21 (DE3) pLysS <i>E. coli</i> and cultured overnight at 25 °C before harvesting. Cell pellets were lysed enzymatically and by sonication. DsbC-His-peptide fusions were isolated from the soluble cell lysates by nickel affinity purification. Concentrations of the 96 purified DsbC-His-fusion proteins were estimated by UV absorbance at 280 nm in a microtiter plate reader (GENios Plus, Tecan, Switzerland). TEVsh protease was added (1/10 w/w) directly into the eluted fusion proteins. The mixture was supplemented with a final concentration of 0.1 mM DTT and then incubated overnight at room temperature in a Microtron shaking incubator (INFORS-HT, Switzerland) (200 rpm). Finally, samples were acidified for 1 h with a final concentration of 5% acetonitrile and 0.1% formic acid (FA) at room temperature with mild agitation. TEV proteases were precipitated during this acidification process, and the 96 recombinant toxins per plate were subjected to a final purification step on C18 chromatographic resin by solid-phase extraction (SPE). The target peptides were eluted from the C18 resin and analyzed by UHPLC and MS to confirm the identity and the quantity of the toxin (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">Figure S2</a>). The toxins were distributed in 96-well plates at concentrations of 1–10 μM.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Production of N-TRTX-Preg1a and N-BUTX-Ptr1a Toxins</h3><div class="NLM_p last">The linear peptide was obtained by solid-phase chemical synthesis following the same protocol described above using Fmoc-Thr(tBu)-Wang resin LL (Novabiochem) for N-TRTX-Preg1a and H-Cys(Trt)-Trityl NovaPEG resin (Novabiochem) for N-BUTX-Ptr1a. After resin cleavage and ether precipitation, the linear peptide was then purified using a Waters Preparative HPLC 600 with an XBridge BEH 300 Å Prep C18 column (5 μm, 19 × 250 mm, Waters). If disulfide bridges were to be prevented from forming, the linear peptide was treated with guanidinium chloride 6 M with iodoacetamide (10 times in excess) for 2 h under agitation at room temperature. The oxidation step was performed under Venomics oxidation conditions.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The resulting oxidized peptide was finally purified using a Waters 600 controller using an XBridge BEH 300 Å Prep C18 column (5 μm, 10 × 250 mm, Waters). Purity of the final product was above 95% and was assessed separately by MALDI-TOF spectrometry (4800 MALDI TOF-TOF Analyzer (Applied Biosystems, USA)) and analytical HPLC using a Waters e2695 separation module with an XBridge BEH 300 Å C18 column (5 μm, 4.6 × 250 mm, Waters). Final quantity was measured by amino-acid quantification using the PicoTag kit (Waters Associates, Milford, MA).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Sequence Analysis and Phylogenetic Reconstruction</h3><div class="NLM_p last">Sequences were queried against UniProt (release 2019_02) and refseq_protein (release-96) with blastp 2.29.1. Searching parameters were adjusted for short-length sequences. Spider and scorpion venom proteins in UniProt were annotated as in VenomZone (<a href="https://venomzone.expasy.org/" class="extLink">https://venomzone.expasy.org/</a> release May 2019). Multiple sequence alignments were performed with mafft 7.310 --auto option. Phylogenetic trees were obtained with RAxML 8.2.11 --PROTGAMMAAUTO --# 100 options and MrBayes v3.2.0 ngen = 1,000,000 nchains = 4 nruns = 2.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> NMR Structures</h3><div class="NLM_p last">Solutions of 0.5 mM N-TRTX-Preg1a and 1.2 mM N-BUTX-Ptr1a were prepared in 3 mm tubes (315 μg of N-TRTX-Preg1a and about 1 mg of N-BUTX-Ptr1a were dissolved in 220 μL of H<sub>2</sub>O/D<sub>2</sub>O (95:5), at pH 4.5). All NMR spectra were acquired using a Bruker 700 MHz NMR spectrometer equipped with a 5 mm TCI cryoprobe, at 298 K. The standard set of 2D <sup>1</sup>H-NMR experiments (COSY, 80 ms TOCSY, and 160 ms NOESY) were completed with natural abundance <sup>15</sup>N- and <sup>13</sup>C-HSQC spectra. Processing, analyses, and calculations were successively performed with Bruker’s TopSpin3.2, CcpNMR,<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> and Aria2.3<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> software. Chemical shift assignments were carried out following the classical procedure, and distance constraints were deduced from the volume integration of NOE correlations and introduced into the standard iterative calculation process of the Aria (ambiguous restraints for iterative assignment) software. For each Aria run, 500 structure calculations were performed per iteration, and 250 structures were calculated in the final iteration in water. For both samples, the first runs were performed allowing all combinations of disulfide pairing (ambiguous intersulfur constraints option). This option was kept for the N-BUTX-Ptr1a, for which the calculations converged toward a unique array of disulfide bridges, and the only additional constraints were the H-bonds observed in the β-sheet already formed in the first runs. For the N-TRTX-Preg1a sample, (φ,ψ) angular restraints deduced by the TALOS software<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> from <sup>13</sup>C and <sup>1</sup>H chemical shifts were added to NOE constraints. Covalent bonds between sulfur atoms involved in each defined bridge were introduced in the last runs of the calculation. Finally, for each peptide, 10 of the structures refined in water in the final iteration were selected according to their agreement with experimental data. The final sets of structures were displayed and analyzed using PyMol (The PyMOL Molecular Graphics System, Version 2.0 Schródinger, LLC), and their quality were evaluated using Promotif<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> and PROCHECK3.5<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> software.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Screening</h3><div class="NLM_p last">The <sup>125</sup>I-NDP-α-MSH and membranes expressing receptors were purchased from PerkinElmer (Courtaboeuf, France). Screening conditions were optimized in order to get a <i>Z</i>′-factor value superior to 0.5. <i>Z</i>′-factor is defined as <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_m001.gif" alt="" id="_i23" /></img></span>, where MAD equals median absolute deviation. On each 96-well plate, eight wells were dedicated to negative controls and correspond to total binding of the radioligand in the absence of any other ligand. Nonspecific binding was measured in the presence of 1 μM NDP-α-MSH (Tocris Bioscience) in eight other wells per plate. Following screening, peptides with an associated binding value bigger or lower than median – 3MAD<sub>median</sub><a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> were classified as hits. Peptides were screened at concentrations of 0.1 to 100 μM according to their production yields.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Radioligand Binding Assays</h3><div class="NLM_p last">The <sup>125</sup>I-NDP-α-MSH and melanocortin receptors were purchased from PerkinElmer (Courtaboeuf, France). Binding experiments were performed using a radioligand at 50 pM in a 100 μL reaction mixture at room temperature in a buffer composed of 25 mM HEPES pH 7.4, 1 mM MgCl<sub>2</sub>, 1 mM CaCl<sub>2</sub>, 0.2 g/L BSA with various concentrations of competitors. Controls were the same as for the primary screening on hMC4R. After 3 h of incubation time, samples were filtered through 96-well GF/C filter plates preincubated with 0.5% polyethylenimine. 25 μL of MicroScint 0 was added onto each dry filter, and the radioactivity was quantified using a TopCount beta counter (PerkinElmer, Courtaboeuf, France) with 45% yield. Competition binding data were fitted to a one-site/state inhibition mass action curve using Prism software, v5.0.0 (GraphPad, La Jolla, CA, USA). IC<sub>50</sub> values were converted to <i>K</i><sub>i</sub>, using the Cheng–Prusoff equation.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a><i>K</i><sub>d</sub> values were 0.34, 0.47, 0.25, and 0.88 nM, respectively, for hMC1R, hMC3R, hMC4R, and hMC5R. Data represent at least three independent experiments performed in duplicate and are presented as mean ± SEM.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Functional Assays</h3><div class="NLM_p last">Human embryonic kidney cells (HEK293, DSMZ, ACC305, authentication: full-matching STR reference profile of 293 in the database, authentic) and HEK293-MC1R (human embryonic kidney cells expressing the human melanocortin 1 receptor with an N-terminal HA-tag and C-terminal FLAG-tag due to stable transfection, selection marker: hygromycin, AG Beck-Sickinger, University Leipzig, Leipzig, Germany) were grown in Dulbecco’s modified Eagle’s medium (DMEM) with 4.5 g/L glucose and <span class="smallcaps smallerCapital">l</span>-glutamine and Ham’s F12 (1:1, v/v; Lonza) supplied with 15% (v/v) heat-inactivated fetal calf serum (FCS; Lonza) in a humidified atmosphere at 37 °C and 5% CO<sub>2</sub>. HEK293 or stably transfected HEK293-HA_MC1R_FLAG cells were transfected in solution with 0.15 μL of Lipofectamine 2000 (Invitrogen) per well and 30 ng of Plasmid pGL4.29[luc2P/CRE/Hygro] (Promega) per well and additionally 30 ng of pcDNA3-hMC4R per well when needed. Lipofectamine and plasmid solutions were prepared in 6 μL of Opti-MEM medium (Gibco) per well. After 5 min of incubation at RT, the Lipofectamine solution was slowly added into the DNA solution and incubated for 30 min at RT. Transfection of 20,000 cells/well in 30 μL of medium (DMEM/Ham’s F12/ 15% FCS) without antibiotics was performed in solution by slowly adding the DNA/Lipofectamine solution to the cells. After careful mixing, 42 μL per well was plated on a 384-well microplate (Greiner bio-one, F-Bottom, white) and incubated overnight. The day after transfection, the medium was removed and cells were stimulated with 15 μL of peptide solution in a serum-free medium (1 nM α-MSH (AG Beck-Sickinger, University Leipzig, Leipzig, Germany) corresponding to ∼EC<sub>85</sub> at hMC1R or ∼EC<sub>75</sub> at hMC4R (100 μM N-TRTX-Preg1a or 100 μM N-BUTX-Ptr1a) for 3 h at 37 °C, 5% CO<sub>2</sub>. As a positive control, 1 μM forskolin (Sigma-Aldrich) was used, while a serum-free medium was used as the negative control (baseline). For luciferase detection, 15 μL of OneGlo Substrate (Promega) was added at RT to each well. The 384-well plate was transferred directly to the plate reader (Tecan Infinite 200 PRO), and luminescence was measured after shaking for 5 s (double orbital, amplitude: 3.5 mm) and 5 min of incubation with an integration time of 1000 ms at room temperature. Data analysis was performed with GraphPad Prism. After baseline subtraction, data were normalized to stimulation with 1 μM forskolin. Data represent the mean ± SEM from three independent experiments performed in triplicate. Statistical analysis was performed with GraphPad Prism 5.03 (GraphPad Software, Inc., San Diego, California, USA). Statistical significance was determined by one-way analysis of variance (ANOVA) followed by Dunnett’s multiple comparison test. *** <i>p</i> < 0.001.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Electrophysiology</h3><div class="NLM_p">For the expression of Na<sub>V</sub> channels (human (h)Na<sub>V</sub>1.1, rat (r)Na<sub>V</sub>1.2, rNa<sub>V</sub>1.3, rNa<sub>V</sub>1.4, hNa<sub>V</sub>1.5, mouse (m)Na<sub>V</sub>1.6, hNa<sub>V</sub>1.7, rNa<sub>V</sub>1.8 channels, and the auxiliary subunits rβ1, hβ1), K<sub>V</sub> channels (rK<sub>V</sub>1.1, rK<sub>V</sub>1.2, hK<sub>V</sub>1.3, rK<sub>V</sub>1.4, rK<sub>V</sub>1.5, rK<sub>V</sub>1.6, rK<sub>V</sub>2.1, hK<sub>V</sub>3.1, rK<sub>V</sub>4.3, hK<sub>V</sub>7.2, hK<sub>V</sub>7.3, hK<sub>V</sub>10.1, hK<sub>V</sub>11.1), Ca<sub>V</sub> channels (rCa<sub>V</sub>3.3), hGIRK1, and nAChRs (hα1β1γδ and hα7) in <i>Xenopus laevis</i> oocytes, the linearized plasmids were transcribed using the T7 or SP6 mMESSAGE-mMACHINE transcription kit (Ambion, Carlsbad, CA). The harvesting of stage V–VI oocytes from anesthetized female <i>X. laevis</i> frogs was previously described.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> Oocytes were injected with 50 nL of cRNA at a concentration of 1 ng/nL using a microinjector (Drummond Scientific, Broomall, PA). The oocytes were incubated in a solution containing 96 mM NaCl, 2 mM KCl, 1.8 mM CaCl<sub>2</sub>, 2 mM MgCl<sub>2</sub>, and 5 mM HEPES pH 7.4, supplemented with 50 mg/L gentamycin sulfate.</div><div class="NLM_p last">Two-electrode voltage-clamp recordings were performed at room temperature (18–22 °C) using a Geneclamp 500 amplifier (Molecular Devices, Downingtown, PA) controlled by a pClamp data acquisition system (Axon Instruments, Union City, CA). Whole cell currents from oocytes were recorded 1–4 days after injection. Bath solution composition was 96 mM NaCl, 2 mM KCl, 1.8 mM CaCl<sub>2</sub>, 2 mM MgCl<sub>2</sub>, and 5 mM HEPES pH 7.4. Toxins were applied directly to the bath. Resistances of both electrodes were kept between 0.8 and 1.5 MΩ. The elicited currents were filtered at 2 (Na<sub>V</sub>), 0.5 (K<sub>V</sub> amd Ca<sub>V</sub>), and 0.05 kHz (GIRK and nAChR) and sampled at 20 (Na<sub>V</sub>), 2 (K<sub>V</sub> and Ca<sub>V</sub>), and 0.1 kHz (GIRK and nAChR) using a four-pole low-pass Bessel filter. Leak subtraction was performed using a -P/4 protocol. For the electrophysiological analysis, a number of protocols were applied from a holding potential of −90 mV. K<sub>V</sub>1.1–K<sub>V</sub>1.6 currents were evoked by 250 ms depolarizations to 0 mV followed by a 250 ms pulse to −50 mV. K<sub>V</sub>2.1, K<sub>V</sub>3.1, and K<sub>V</sub>4.3 currents were elicited by 250 ms pulses to +20 mV from a holding potential of −90 mV. K<sub>V</sub>7 and K<sub>V</sub>10.1 currents were evoked by 2 s depolarizing pulses to 0 mV from a holding potential of −90 mV. Current traces of K<sub>V</sub>11.1 channels were elicited by applying a +40 mV prepulse for 2 s followed by a step to −120 mV for 2 s. Sodium current traces were evoked by 100 ms depolarization to the voltage corresponding to maximal sodium current in control conditions. For GIRK measurements, the oocytes were clamped at −70 mV and superfused with a high K<sup>+</sup> solution (96 mM KCl, 2 mM NaCl, 1 mM MgCl<sub>2</sub>, 1.8 mM CaCl<sub>2</sub>, 5 mM HEPES pH 7.5). Ca<sub>V</sub> channel currents, carried by 10 mM Ba<sup>2+</sup>, were elicited by 200 ms test depolarizations applied from −90 mV to voltages ranging from −50 or +30 mV. For nAChRs, oocytes were clamped at a holding potential of −70 mV throughout all recordings. To minimize the interference of endogenous Ca<sup>2+</sup>-activated Cl<sup>–</sup> channels during the experiments, a Ca<sup>2+</sup>-free bath perfusion solution was used, containing 82.5 mM NaCl, 2.5 mM KCl, 1 mM MgCl<sub>2</sub>, 2.5 mM BaCl<sub>2</sub>, and 5 mM HEPES pH 7.4. Current responses were evoked by 10 s applications of increasing concentrations of acetylcholine (starting at 200 μM). All data were analyzed using pClamp Clampfit 10.1 (Molecular Devices) and Origin 7.5 software (OriginLab, Northampton, MA).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Toxin Nomenclature</h3><div class="NLM_p last">Newly characterized toxins were named according to the rational nomenclature proposed by King et al.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> with a Greek letter associated with a biological function followed by the generic name indicating from which taxon the toxin is derived, genus and species indicated by uppercase and lowercase letters, respectively, a number to distinguish different toxins with similar activity, and a lowercase letter for isoforms. In this study, the uppercase nu Greek letter (N) was chosen as the new descriptor of toxins active on melanocortin receptors.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00485" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00485?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00485</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">LC–MS analysis of five representative toxins with their cystein patterns, produced by solid-phase chemical synthesis; analysis of four representative toxins with their cystein patterns, produced by recombinant expression; quality assessment at each stage of N-TRTX-Preg1a and N-BUTX-Ptr1a production in Fmoc solid-phase chemical synthesis; activity of 100 μM N-TRTX-Preg1a on ion channels; activity of 100 μM N-BUTX-Ptr1a on ion channels; cysteine pattern diversity for toxins containing an even number of cysteines identified in the Venomics bank; cysteine pattern diversity for toxins containing an odd number of cysteine residues identified in the Venomics bank; experimental NMR data used for the calculations and final structural statistics for the 10 models representative of the solution structure of N-TRTX-Preg1a; experimental NMR data used for the calculations and final structural statistics for the 10 models representative of the solution structure of N-BUTX-Ptr1a (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf">jm0c00485_si_001.pdf (806.64 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00485" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06524" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06524" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Gilles</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6853-9764" title="Orcid link">http://orcid.org/0000-0001-6853-9764</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4907202a2625283a672e2025252c3a092a2c28672f3b"><span class="__cf_email__" data-cfemail="8bc5e2e8e4e7eaf8a5ece2e7e7eef8cbe8eeeaa5edf9">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steve Reynaud</span> - <span class="hlFld-Affiliation affiliation">Université
Paris-Sud, 15 Rue Georges Clemenceau, Orsay 91405 France</span>; 
    <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justyna Ciolek</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michel Degueldre</span> - <span class="hlFld-Affiliation affiliation">Mass
Spectrometry Laboratory, Université
de Liège, Allée du six Aout 11, Quartier Agora, Liege 4000 Belgium</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Analytical Science Biologicals, UCB, Chemin du Foriest, Braine L’Alleud 1420 Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalie J. Saez</span> - <span class="hlFld-Affiliation affiliation">Centre
National de la Recherche Scientifique, Architecture et Fonction des
Macromolécules Biologiques, Campus de Luminy, Marseille 13288 France</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Molecular Bioscience, The University
of Queensland, St Lucia 4072, QLD, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ana Filipa Sequeira</span> - <span class="hlFld-Affiliation affiliation">Universidade
de Lisboa, CIISA - Faculdade de Medicina Veterinária, Avenida da Universidade Técnica, Lisboa 1300-477 Portugal</span>; 
    <span class="hlFld-Affiliation affiliation">NZYTech
Lda, Genes & Enzymes, Estrada do Paço do Lumiar, Campus do Lumiar,
Edifício E - R/C, Lisboa 1649-038 Portugal</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yoan Duhoo</span> - <span class="hlFld-Affiliation affiliation">Centre
National de la Recherche Scientifique, Architecture et Fonction des
Macromolécules Biologiques, Campus de Luminy, Marseille 13288 France</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Molecular Bioscience, The University
of Queensland, St Lucia 4072, QLD, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joana L. A. Brás</span> - <span class="hlFld-Affiliation affiliation">Universidade
de Lisboa, CIISA - Faculdade de Medicina Veterinária, Avenida da Universidade Técnica, Lisboa 1300-477 Portugal</span>; 
    <span class="hlFld-Affiliation affiliation">NZYTech
Lda, Genes & Enzymes, Estrada do Paço do Lumiar, Campus do Lumiar,
Edifício E - R/C, Lisboa 1649-038 Portugal</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hervé Meudal</span> - <span class="hlFld-Affiliation affiliation">Centre National de la Recherche Scientifique,
Centre de Biophysique
Moléculaire, rue
Charles Sadron, Orléans 45071 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Miguel Cabo Díez</span> - <span class="hlFld-Affiliation affiliation">Next-Generation
Sequencing Laboratory, Sistemas Genómicos
Ltd., Ronda de Guglielmo
Marconi, 6, Paterna 46980 Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Victoria Fernández Pedrosa</span> - <span class="hlFld-Affiliation affiliation">Next-Generation
Sequencing Laboratory, Sistemas Genómicos
Ltd., Ronda de Guglielmo
Marconi, 6, Paterna 46980 Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marion Verdenaud</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Boeri</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oscar Pereira Ramos</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frédéric Ducancel</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département IDMIT, 18 route du Panorama, 92265 Fontenay-aux-Roses, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Margot Vanden Driessche</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rudy Fourmy</span> - <span class="hlFld-Affiliation affiliation">Alphabiotoxine
Laboratory sprl, Barberie
15, Montroeul-au-bois 7911 Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aude Violette</span> - <span class="hlFld-Affiliation affiliation">Alphabiotoxine
Laboratory sprl, Barberie
15, Montroeul-au-bois 7911 Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Grégory Upert</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gilles Mourier</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annette G. Beck-Sickinger</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, Universitat Leipzig, Leipzig 04103 Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4560-8020" title="Orcid link">http://orcid.org/0000-0003-4560-8020</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karin Mörl</span> - <span class="hlFld-Affiliation affiliation">Institute
of Biochemistry, Universitat Leipzig, Leipzig 04103 Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Céline Landon</span> - <span class="hlFld-Affiliation affiliation">Centre National de la Recherche Scientifique,
Centre de Biophysique
Moléculaire, rue
Charles Sadron, Orléans 45071 France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0248-1990" title="Orcid link">http://orcid.org/0000-0002-0248-1990</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos M. G. A. Fontes</span> - <span class="hlFld-Affiliation affiliation">Universidade
de Lisboa, CIISA - Faculdade de Medicina Veterinária, Avenida da Universidade Técnica, Lisboa 1300-477 Portugal</span>; 
    <span class="hlFld-Affiliation affiliation">NZYTech
Lda, Genes & Enzymes, Estrada do Paço do Lumiar, Campus do Lumiar,
Edifício E - R/C, Lisboa 1649-038 Portugal</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rebeca Miñambres Herráiz</span> - <span class="hlFld-Affiliation affiliation">Next-Generation
Sequencing Laboratory, Sistemas Genómicos
Ltd., Ronda de Guglielmo
Marconi, 6, Paterna 46980 Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ricardo C. Rodríguez de la Vega</span> - <span class="hlFld-Affiliation affiliation">Universite Paris-Sud, Ecologie, Systematique et Evolution, 15 rue Georges Clémenceau, Orsay 91405 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steve Peigneur</span> - <span class="hlFld-Affiliation affiliation">Toxicology
and Pharmacology, University of Leuven (KU
Leuven), Herestraat 49, Leuven 3000 Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Tytgat</span> - <span class="hlFld-Affiliation affiliation">Toxicology
and Pharmacology, University of Leuven (KU
Leuven), Herestraat 49, Leuven 3000 Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Loïc Quinton</span> - <span class="hlFld-Affiliation affiliation">Mass
Spectrometry Laboratory, Université
de Liège, Allée du six Aout 11, Quartier Agora, Liege 4000 Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8153-9590" title="Orcid link">http://orcid.org/0000-0001-8153-9590</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edwin De Pauw</span> - <span class="hlFld-Affiliation affiliation">Mass
Spectrometry Laboratory, Université
de Liège, Allée du six Aout 11, Quartier Agora, Liege 4000 Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renaud Vincentelli</span> - <span class="hlFld-Affiliation affiliation">Centre
National de la Recherche Scientifique, Architecture et Fonction des
Macromolécules Biologiques, Campus de Luminy, Marseille 13288 France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Denis Servent</span> - <span class="hlFld-Affiliation affiliation">Université
Paris Saclay, CEA, Département Médicaments et Technologies
pour la Santé, SIMoS, Gif-sur-Yvette 91191 France</span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by the European FP7 project 278346 2011–2015 and by the French Infrastructure for Integrated Structural Biology (FRISBI) ANR-10-INSB-05-01. A Ph.D. grant (S.R.) was supported by the University of Paris-Sud. J.T. was funded by GOC2319N and GOA4919N (F.W.O. Vlaanderen) and CELSA/17/047 (BOF, KU Leuven). S.P. is a postdoctoral fellow supported by KU Leuven funding (PDM/19/164).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07650" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07650" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We specifically thank Dr. Philipe Robin, lecturer at the Université Paris-Sud, and Mrs. Marion Chaigneau for technical support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ANOVA</td><td class="NLM_def"><p class="first last">one-way analysis of variance</p></td></tr><tr><td class="NLM_term">BUTX</td><td class="NLM_def"><p class="first last">butantoxin</p></td></tr><tr><td class="NLM_term">Ca<sub>V</sub></td><td class="NLM_def"><p class="first last">voltage-gated calcium channel</p></td></tr><tr><td class="NLM_term">CHO</td><td class="NLM_def"><p class="first last">Chinese hamster ovary cell</p></td></tr><tr><td class="NLM_term">COSY</td><td class="NLM_def"><p class="first last">correlation spectroscopy</p></td></tr><tr><td class="NLM_term">CSαβ</td><td class="NLM_def"><p class="first last">cysteine-stabilized α-helix β-sheet</p></td></tr><tr><td class="NLM_term">DsbC</td><td class="NLM_def"><p class="first last">disulfide bond isomerase C</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">Fmoc</td><td class="NLM_def"><p class="first last">fluorenylmethoxycarbonyl</p></td></tr><tr><td class="NLM_term">GIRK</td><td class="NLM_def"><p class="first last">G-protein-coupled inwardly rectifying potassium channel</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">HCTU</td><td class="NLM_def"><p class="first last">H-(6-chlorobenzotriazole-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HEK</td><td class="NLM_def"><p class="first last">human embryonic kidney cell</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">hydroxyethyl piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HT</td><td class="NLM_def"><p class="first last">high throughput</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">ICK</td><td class="NLM_def"><p class="first last">inhibitory cystine knot</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>i</sub></td><td class="NLM_def"><p class="first last">inhibitory constant</p></td></tr><tr><td class="NLM_term">K<sub>V</sub></td><td class="NLM_def"><p class="first last">voltage-gated potassium channel</p></td></tr><tr><td class="NLM_term">LC</td><td class="NLM_def"><p class="first last">liquid chromatography</p></td></tr><tr><td class="NLM_term">hMC1R</td><td class="NLM_def"><p class="first last">human melanocortin 1 receptor</p></td></tr><tr><td class="NLM_term">hMC3R</td><td class="NLM_def"><p class="first last">human melanocortin 3 receptor</p></td></tr><tr><td class="NLM_term">hMC4R</td><td class="NLM_def"><p class="first last">human melanocortin 4 receptor</p></td></tr><tr><td class="NLM_term">hMC5R</td><td class="NLM_def"><p class="first last">human melanocortin 5 receptor</p></td></tr><tr><td class="NLM_term">MALDI-TOF</td><td class="NLM_def"><p class="first last">matrix-assisted laser desorption ionization - time of flight</p></td></tr><tr><td class="NLM_term">MAD</td><td class="NLM_def"><p class="first last">median absolute deviation</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrometry</p></td></tr><tr><td class="NLM_term">MSH</td><td class="NLM_def"><p class="first last">melanocyte-stimulating hormone</p></td></tr><tr><td class="NLM_term">no</td><td class="NLM_def"><p class="first last">number</p></td></tr><tr><td class="NLM_term">Na<sub>V</sub></td><td class="NLM_def"><p class="first last">voltage-gated sodium channel</p></td></tr><tr><td class="NLM_term">NDP-α-MSH</td><td class="NLM_def"><p class="first last">[Nle4,D-phe7]-alpha-melanocyte-stimulating hormone</p></td></tr><tr><td class="NLM_term">NGS</td><td class="NLM_def"><p class="first last">next-generation sequencing</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">NOE</td><td class="NLM_def"><p class="first last">nuclear Overhauser effect</p></td></tr><tr><td class="NLM_term">NOESY</td><td class="NLM_def"><p class="first last">nuclear Overhauser effect spectroscopy</p></td></tr><tr><td class="NLM_term">PTM</td><td class="NLM_def"><p class="first last">post-translational modification</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">Seq</td><td class="NLM_def"><p class="first last">sequence</p></td></tr><tr><td class="NLM_term">TOCSY</td><td class="NLM_def"><p class="first last">total correlation spectroscopy</p></td></tr><tr><td class="NLM_term">TRTX</td><td class="NLM_def"><p class="first last">theraphotoxin</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultra-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">UV</td><td class="NLM_def"><p class="first last">ultraviolet</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 78 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenner, R. A.</span>; <span class="NLM_string-name">Undheim, E.</span></span> <i>Venom: The Secrets of Nature’s Deadliest Weapon</i>; <span class="NLM_publisher-name">Smithsonian Books</span>: <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=R.+A.+Jenner&author=E.+Undheim&title=Venom%3A+The+Secrets+of+Nature%E2%80%99s+Deadliest+Weapon"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJenner%26aufirst%3DR.%2BA.%26btitle%3DVenom%253A%2520The%2520Secrets%2520of%2520Nature%25E2%2580%2599s%2520Deadliest%2520Weapon%26pub%3DSmithsonian%2520Books%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degueldre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdenaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legarda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minambres, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuniga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducancel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span> <span> </span><span class="NLM_article-title">Diversity in Sequences, Post-Translational Modifications and Expected Pharmacological Activities of Toxins from Four <i>Conus</i> Species Revealed by the Combination of Cutting-Edge Proteomics, Transcriptomics and Bioinformatics</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2017.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2017.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28223047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktVKlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2017&pages=116-125&author=M.+Degueldreauthor=M.+Verdenaudauthor=G.+Legardaauthor=R.+Minambresauthor=S.+Zunigaauthor=M.+Leblancauthor=N.+Gillesauthor=F.+Ducancelauthor=E.+De+Pauwauthor=L.+Quinton&title=Diversity+in+Sequences%2C+Post-Translational+Modifications+and+Expected+Pharmacological+Activities+of+Toxins+from+Four+Conus+Species+Revealed+by+the+Combination+of+Cutting-Edge+Proteomics%2C+Transcriptomics+and+Bioinformatics&doi=10.1016%2Fj.toxicon.2017.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity in sequences, post-translational modifications and expected pharmacological activities of toxins from four Conus species revealed by the combination of cutting-edge proteomics, transcriptomics and bioinformatics</span></div><div class="casAuthors">Degueldre, Michel; Verdenaud, Marion; Legarda, Garikoitz; Minambres, Rebeca; Zuniga, Sheila; Leblanc, Mathieu; Gilles, Nicolas; Ducancel, Frederic; De Pauw, Edwin; Quinton, Loic</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">116-125</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Venomous animals have developed a huge arsenal of reticulated peptides for defense and predation.  Based on various scaffolds, they represent a colossal pharmacol. diversity, making them top candidates for the development of innovative drugs.  Instead of relying on the classical, low-throughput bioassay-guided approach to identify innovative bioactive peptides, this work exploits a recent paradigm to access to venom diversity.  This strategy bypasses the classical approach by combining high-throughput transcriptomics, proteomics and bioinformatics cutting-edge technologies to generate reliable peptide sequences.  The strategy employed to generate hundreds of reliable sequences from Conus venoms is deeply described.  The study led to the discovery of (i) conotoxins that belong to known pharmacol. families targeting various GPCRs or ion-gated channels, and (ii) new families of conotoxins, never described to date.  It also focusses on the diversity of genes, sequences, folds, and PTM's provided by such species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvTuilVaXFwrVg90H21EOLACvtfcHk0lh8XSfs-rrECw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktVKlsrg%253D&md5=d2e962030a24f30e49487abe6f11c910</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2017.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2017.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DDegueldre%26aufirst%3DM.%26aulast%3DVerdenaud%26aufirst%3DM.%26aulast%3DLegarda%26aufirst%3DG.%26aulast%3DMinambres%26aufirst%3DR.%26aulast%3DZuniga%26aufirst%3DS.%26aulast%3DLeblanc%26aufirst%3DM.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDucancel%26aufirst%3DF.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DQuinton%26aufirst%3DL.%26atitle%3DDiversity%2520in%2520Sequences%252C%2520Post-Translational%2520Modifications%2520and%2520Expected%2520Pharmacological%2520Activities%2520of%2520Toxins%2520from%2520Four%2520Conus%2520Species%2520Revealed%2520by%2520the%2520Combination%2520of%2520Cutting-Edge%2520Proteomics%252C%2520Transcriptomics%2520and%2520Bioinformatics%26jtitle%3DToxicon%26date%3D2017%26volume%3D130%26spage%3D116%26epage%3D125%26doi%3D10.1016%2Fj.toxicon.2017.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beraud, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandy, K. G.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Potential of Peptide Toxins That Target Ion Channels</span>. <i>Inflamm. Allergy Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.2174/187152811797200696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.2174%2F187152811797200696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=21745182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=322-342&author=E.+Beraudauthor=K.+G.+Chandy&title=Therapeutic+Potential+of+Peptide+Toxins+That+Target+Ion+Channels&doi=10.2174%2F187152811797200696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of peptide toxins that target ion channels</span></div><div class="casAuthors">Beraud, Evelyne; Chandy, K. George</div><div class="citationInfo"><span class="NLM_cas:title">Inflammation & Allergy: Drug Targets</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">322-342</span>CODEN:
                <span class="NLM_cas:coden">IADTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1871-5281</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Traditional healthcare systems in China, India, Greece and the Middle East have for centuries exploited venomous creatures as a resource for medicines.  This review focuses on one class of pharmacol. active compds. from venom, namely peptide toxins that target ion channels.  We highlight their therapeutic potential and the specific channels they target.  The field of therapeutic application is vast, including pain, inflammation, cancer, neurol. disorders, cardioprotection, and autoimmune diseases.  One of these peptides is in clin. use, and many others are in various stages of pre-clin. and clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD27jVncwDZbVg90H21EOLACvtfcHk0lh8XSfs-rrECw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrL&md5=a04d7eface3437562719693d3be1d75d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F187152811797200696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152811797200696%26sid%3Dliteratum%253Aachs%26aulast%3DBeraud%26aufirst%3DE.%26aulast%3DChandy%26aufirst%3DK.%2BG.%26atitle%3DTherapeutic%2520Potential%2520of%2520Peptide%2520Toxins%2520That%2520Target%2520Ion%2520Channels%26jtitle%3DInflamm.%2520Allergy%2520Drug%2520Targets%26date%3D2011%26volume%3D10%26spage%3D322%26epage%3D342%26doi%3D10.2174%2F187152811797200696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, M. J.</span></span> <span> </span><span class="NLM_article-title">Conus Venom Peptide Pharmacology</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1124/pr.111.005322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1124%2Fpr.111.005322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=22407615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC38XmslKksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=259-298&author=R.+J.+Lewisauthor=S.+Dutertreauthor=I.+Vetterauthor=M.+J.+Christie&title=Conus+Venom+Peptide+Pharmacology&doi=10.1124%2Fpr.111.005322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Conus venom peptide pharmacology</span></div><div class="casAuthors">Lewis, Richard J.; Dutertre, Sebastien; Vetter, Irina; Christie, MacDonald J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">259-298</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Conopeptides are a diverse group of recently evolved venom peptides used for prey capture and/or defense.  Each species of cone snails produces in excess of 1000 conopeptides, with those pharmacol. characterized (∼0.1%) targeting a diverse range of membrane proteins typically with high potency and specificity.  The majority of conopeptides inhibit voltage- or ligand-gated ion channels, providing valuable research tools for the dissection of the role played by specific ion channels in excitable cells.  It is noteworthy that many of these targets are found to be expressed in pain pathways, with several conopeptides having entered the clinic as potential treatments for pain [e.g., pyroglutamate1-MrIA (Xen2174)] and one now marketed for intrathecal treatment of severe pain [ziconotide (Prialt)].  This review discusses the diversity, pharmacol., structure-activity relationships, and therapeutic potential of cone snail venom peptide families acting at voltage-gated ion channels (ω-, μ-, μO-, δ-, ι-, and κ-conotoxins), ligand-gated ion channels (α-conotoxins, σ-conotoxin, ikot-ikot, and conantokins), G-protein-coupled receptors (ρ-conopeptides, conopressins, and contulakins), and neurotransmitter transporters (χ-conopeptides), with expanded discussion on the clin. potential of sodium and calcium channel inhibitors and α-conotoxins.  Expanding the discovery of new bioactives using proteomic/transcriptomic approaches combined with high-throughput platforms and better defining conopeptide structure-activity relationships using relevant membrane protein crystal structures are expected to grow the already significant impact conopeptides have had as both research probes and leads to new therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxAcubmbcLt7Vg90H21EOLACvtfcHk0lg2I9bBKzeBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmslKksbc%253D&md5=fb1f04e0e7b00241326c9be7a8f06859</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005322%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DChristie%26aufirst%3DM.%2BJ.%26atitle%3DConus%2520Venom%2520Peptide%2520Pharmacology%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26spage%3D259%26epage%3D298%26doi%3D10.1124%2Fpr.111.005322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fruchart-Gaillard, C.</span></span> <span> </span><span class="NLM_article-title">Muscarinic Toxins: Tools for the Study of the Pharmacological and Functional Properties of Muscarinic Receptors</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2009.06092.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1111%2Fj.1471-4159.2009.06092.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19457160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntVCju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2009&pages=1193-1202&author=D.+Serventauthor=C.+Fruchart-Gaillard&title=Muscarinic+Toxins%3A+Tools+for+the+Study+of+the+Pharmacological+and+Functional+Properties+of+Muscarinic+Receptors&doi=10.1111%2Fj.1471-4159.2009.06092.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic toxins: tools for the study of the pharmacological and functional properties of muscarinic receptors</span></div><div class="casAuthors">Servent, Denis; Fruchart-Gaillard, Carole</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1193-1202</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Muscarinic receptors mediate metabotropic actions of acetylcholine in the CNS and PNS and autocrine functions of acetylcholine in non-neuronal systems.  Because of the lack of highly selective muscarinic ligands, the precise location, functional role, and roles in various diseases of the five muscarinic receptor subtypes remain unclear.  Muscarinic toxins isolated from the venom of Dendroaspis snakes have a natural high affinity and selectivity, assocd. with roles as competitive antagonists, allosteric modulators, and potential agonists.  These toxins may therefore be invaluable tools for studying muscarinic receptors.  We review data on the structural and pharmacol. characterization of the muscarinic toxins, focusing on recent structure-function studies on toxin-receptor interactions.  We discuss the potential benefits of using these toxins for investigating muscarinic function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNb6V-N4f51bVg90H21EOLACvtfcHk0lg2I9bBKzeBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntVCju7w%253D&md5=b542ee6ce11c11168de633e805262187</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2009.06092.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2009.06092.x%26sid%3Dliteratum%253Aachs%26aulast%3DServent%26aufirst%3DD.%26aulast%3DFruchart-Gaillard%26aufirst%3DC.%26atitle%3DMuscarinic%2520Toxins%253A%2520Tools%2520for%2520the%2520Study%2520of%2520the%2520Pharmacological%2520and%2520Functional%2520Properties%2520of%2520Muscarinic%2520Receptors%26jtitle%3DJ.%2520Neurochem.%26date%3D2009%26volume%3D109%26spage%3D1193%26epage%3D1202%26doi%3D10.1111%2Fj.1471-4159.2009.06092.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciolek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinfrank, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viengchareun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stura, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigismeau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouillac, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orcel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droctové, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevoux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombès, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzgall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">Green Mamba Peptide Targets Type-2 Vasopressin Receptor against Polycystic Kidney Disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">7154</span>– <span class="NLM_lpage">7159</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620454114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.1620454114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28630289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=7154-7159&author=J.+Ciolekauthor=H.+Reinfrankauthor=L.+Quintonauthor=S.+Viengchareunauthor=E.+A.+Sturaauthor=L.+Veraauthor=S.+Sigismeauauthor=B.+Mouillacauthor=H.+Orcelauthor=S.+Peigneurauthor=J.+Tytgatauthor=L.+Droctov%C3%A9author=F.+Beauauthor=J.+Nevouxauthor=M.+Lomb%C3%A8sauthor=G.+Mourierauthor=E.+De+Pauwauthor=D.+Serventauthor=C.+Mendreauthor=R.+Witzgallauthor=N.+Gilles&title=Green+Mamba+Peptide+Targets+Type-2+Vasopressin+Receptor+against+Polycystic+Kidney+Disease&doi=10.1073%2Fpnas.1620454114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Green mamba peptide targets type-2 vasopressin receptor against polycystic kidney disease</span></div><div class="casAuthors">Ciolek, Justyna; Reinfrank, Helen; Quinton, Loic; Viengchareun, Say; Stura, Enrico A.; Vera, Laura; Sigismeau, Sabrina; Mouillac, Bernard; Orcel, Helene; Peigneur, Steve; Tytgat, Jan; Droctove, Laura; Beau, Fabrice; Nevoux, Jerome; Lombes, Marc; Mourier, Gilles; De Pauw, Edwin; Servent, Denis; Mendre, Christiane; Witzgall, Ralph; Gilles, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">7154-7159</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Polycystic kidney diseases (PKDs) are genetic disorders that can cause renal failure and death in children and adults.  Lowering cAMP in cystic tissues through the inhibition of the type-2 vasopressin receptor (V2R) constitutes a validated strategy to reduce disease progression.  The authors identified a peptide from green mamba venom that exhibits nanomolar affinity for the V2R without any activity on 155 other G-protein-coupled receptors or on 15 ionic channels.  Mambaquaretin-1 is a full antagonist of the V2R activation pathways studied: cAMP prodn., beta-arrestin interaction, and MAP kinase activity.  This peptide adopts the Kunitz fold known to mostly act on potassium channels and serine proteases.  Mambaquaretin-1 interacts selectively with the V2R through its first loop, in the same manner that aprotinin inhibits trypsin.  Injected in mice, mambaquaretin-1 increases in a dose-dependent manner urine outflow with concomitant redn. of urine osmolality, indicating a purely aquaretic effect assocd. with the in vivo blockade of V2R.  CD1-pcy/pcy mice, a juvenile model of PKD, daily treated with 13 μg of mambaquaretin-1 for 99 d, developed less abundant (by 33%) and smaller (by 47%) cysts than control mice.  Neither tachyphylaxis nor apparent toxicity has been noted.  Mambaquaretin-1 represents a promising therapeutic agent against PKDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI-J7A636tYrVg90H21EOLACvtfcHk0lg2I9bBKzeBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N&md5=2cad6950c107df021baf5a66c6b086b0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620454114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620454114%26sid%3Dliteratum%253Aachs%26aulast%3DCiolek%26aufirst%3DJ.%26aulast%3DReinfrank%26aufirst%3DH.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DViengchareun%26aufirst%3DS.%26aulast%3DStura%26aufirst%3DE.%2BA.%26aulast%3DVera%26aufirst%3DL.%26aulast%3DSigismeau%26aufirst%3DS.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DOrcel%26aufirst%3DH.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DDroctov%25C3%25A9%26aufirst%3DL.%26aulast%3DBeau%26aufirst%3DF.%26aulast%3DNevoux%26aufirst%3DJ.%26aulast%3DLomb%25C3%25A8s%26aufirst%3DM.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DMendre%26aufirst%3DC.%26aulast%3DWitzgall%26aufirst%3DR.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DGreen%2520Mamba%2520Peptide%2520Targets%2520Type-2%2520Vasopressin%2520Receptor%2520against%2520Polycystic%2520Kidney%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D7154%26epage%3D7159%26doi%3D10.1073%2Fpnas.1620454114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocking, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biass, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolini-Bertrand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boelens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocklin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span> <span> </span><span class="NLM_article-title">Identification, Structural and Pharmacological Characterization of τ-CnVA, a Conopeptide That Selectively Interacts with Somatostatin Sst<sub>3</sub> Receptor</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">1663</span>– <span class="NLM_lpage">1671</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2013.03.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.bcp.2013.03.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23567999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsleru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2013&pages=1663-1671&author=C.+Petrelauthor=H.+G.+Hockingauthor=M.+Reynaudauthor=G.+Upertauthor=P.+Favreauauthor=D.+Biassauthor=M.+Paolini-Bertrandauthor=S.+Peigneurauthor=J.+Tytgatauthor=N.+Gillesauthor=O.+Hartleyauthor=R.+Boelensauthor=R.+Stocklinauthor=D.+Servent&title=Identification%2C+Structural+and+Pharmacological+Characterization+of+%CF%84-CnVA%2C+a+Conopeptide+That+Selectively+Interacts+with+Somatostatin+Sst3+Receptor&doi=10.1016%2Fj.bcp.2013.03.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification, structural and pharmacological characterization of τ-CnVA, a conopeptide that selectively interacts with somatostatin sst3 receptor</span></div><div class="casAuthors">Petrel, C.; Hocking, H. G.; Reynaud, M.; Upert, G.; Favreau, Ph.; Biass, D.; Paolini-Bertrand, M.; Peigneur, S.; Tytgat, J.; Gilles, N.; Hartley, O.; Boelens, R.; Stocklin, R.; Servent, D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1663-1671</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Conopeptides are a diverse array of small linear and reticulated peptides that interact with high potency and selectivity with a large diversity of receptors and ion channels.  They are used by cone snails for prey capture or defense.  Recent advances in venom gland transcriptomic and venom peptidomic/proteomic technologies combined with bioactivity screening approaches lead to the identification of new toxins with original pharmacol. profiles.  Here, from transcriptomic/proteomic analyses of the Conus consors cone snail, we identified a new conopeptide called τ-CnVA, which displays the typical cysteine framework V of the T1-conotoxin superfamily.  This peptide was chem. synthesized and its three-dimensional structure was solved by NMR anal. and compared to that of TxVA belonging to the same family, revealing very few common structural features apart a common orientation of the intercysteine loop.  Because of the lack of a clear biol. function assocd. with the T-conotoxin family, τ-CnVA was screened against more than fifty different ion channels and receptors, highlighting its capacity to interact selectively with the somatostatine sst3 receptor.  Pharmacol. and functional studies show that τ-CnVA displays a micromolar (Ki of 1.5 μM) antagonist property for the sst3 receptor, being currently the only known toxin to interact with this GPCR subfamily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdXgulOm8sYLVg90H21EOLACvtfcHk0ljtfFIHAdgAeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsleru7g%253D&md5=d10f6a983313c057a3b4cf3213417bd5</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2013.03.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2013.03.019%26sid%3Dliteratum%253Aachs%26aulast%3DPetrel%26aufirst%3DC.%26aulast%3DHocking%26aufirst%3DH.%2BG.%26aulast%3DReynaud%26aufirst%3DM.%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DBiass%26aufirst%3DD.%26aulast%3DPaolini-Bertrand%26aufirst%3DM.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DHartley%26aufirst%3DO.%26aulast%3DBoelens%26aufirst%3DR.%26aulast%3DStocklin%26aufirst%3DR.%26aulast%3DServent%26aufirst%3DD.%26atitle%3DIdentification%252C%2520Structural%2520and%2520Pharmacological%2520Characterization%2520of%2520%25CF%2584-CnVA%252C%2520a%2520Conopeptide%2520That%2520Selectively%2520Interacts%2520with%2520Somatostatin%2520Sst3%2520Receptor%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2013%26volume%3D85%26spage%3D1663%26epage%3D1671%26doi%3D10.1016%2Fj.bcp.2013.03.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouget, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maïga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamot-Rooke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span> <span> </span><span class="NLM_article-title">Identification of a Novel Snake Peptide Toxin Displaying High Affinity and Antagonist Behaviour for the Α2-Adrenoceptors</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">1361</span>– <span class="NLM_lpage">1374</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.00966.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1111%2Fj.1476-5381.2010.00966.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20659106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFWhsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2010&pages=1361-1374&author=C.+Rougetauthor=L.+Quintonauthor=A.+Ma%C3%AFgaauthor=C.+Galesauthor=G.+Masuyerauthor=C.+Malosseauthor=J.+Chamot-Rookeauthor=R.+Thaiauthor=G.+Mourierauthor=E.+De+Pauwauthor=N.+Gillesauthor=D.+Servent&title=Identification+of+a+Novel+Snake+Peptide+Toxin+Displaying+High+Affinity+and+Antagonist+Behaviour+for+the+%CE%912-Adrenoceptors&doi=10.1111%2Fj.1476-5381.2010.00966.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel snake peptide toxin displaying high affinity and antagonist behaviour for the α2-adrenoceptors</span></div><div class="casAuthors">Rouget, Celine; Quinton, Loic; Maiga, Arhamatoulaye; Gales, Celine; Masuyer, Geoffrey; Malosse, Christian; Chamot-Rooke, Julia; Thai, Robert; Mourier, Gilles; de Pauw, Edwin; Gilles, Nicolas; Servent, Denis</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1361-1374</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose Muscarinic and adrenergic G protein-coupled receptors (GPCRs) are the targets of rare peptide toxins isolated from snake or cone snail venoms.  We used a screen to identify novel toxins from Dendroaspis angusticeps targeting aminergic GPCRs.  These toxins may offer new candidates for the development of new tools and drugs.  Exptl. Approach In binding expts. with 3H-rauwolscine, we studied the interactions of green mamba venom fractions with α2-adrenoceptors from rat brain synaptosomes.  We isolated, sequenced and chem. synthesized a novel peptide, ρ-Da1b.  This peptide was pharmacol. characterized using binding expts. and functional tests on human α2-adrenoceptors expressed in mammalian cells.  Key Results ρ-Da1b, a 66-amino acid peptide stabilized by four disulfide bridges, belongs to the three-finger-fold peptide family.  Its synthetic homolog inhibited 80% of 3H-rauwolscine binding to the three α2-adrenoceptor subtypes, with an affinity between 14 and 73 nM and Hill slopes close to unity.  Functional expts. on α2A-adrenoceptor demonstrated that ρ-Da1b is an antagonist, shifting adrenaline activation curves to the right.  Schild regression revealed slopes of 0.97 and 0.67and pA2 values of 5.93 and 5.32 for yohimbine and ρ-Da1b, resp.  Conclusions and Implications ρ-Da1b is the first toxin identified to specifically interact with α2-adrenoceptors, extending the list of class A GPCRs sensitive to toxins.  Addnl., its affinity and atypical mode of interaction open up the possibility of its use as a new pharmacol. tool, in the study of the physiol. roles of α2-adrenoceptor subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphELeKwRIMpbVg90H21EOLACvtfcHk0ljtfFIHAdgAeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFWhsrrJ&md5=4fe4691ccc8a09ad202933269e49b893</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.00966.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.00966.x%26sid%3Dliteratum%253Aachs%26aulast%3DRouget%26aufirst%3DC.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DMa%25C3%25AFga%26aufirst%3DA.%26aulast%3DGales%26aufirst%3DC.%26aulast%3DMasuyer%26aufirst%3DG.%26aulast%3DMalosse%26aufirst%3DC.%26aulast%3DChamot-Rooke%26aufirst%3DJ.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DServent%26aufirst%3DD.%26atitle%3DIdentification%2520of%2520a%2520Novel%2520Snake%2520Peptide%2520Toxin%2520Displaying%2520High%2520Affinity%2520and%2520Antagonist%2520Behaviour%2520for%2520the%2520%25CE%25912-Adrenoceptors%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2010%26volume%3D161%26spage%3D1361%26epage%3D1374%26doi%3D10.1111%2Fj.1476-5381.2010.00966.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rekik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lluel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larregola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciolek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molgó, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fruchart-Gaillard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamot-Rooke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">Isolation and Pharmacological Characterization of AdTx1, a Natural Peptide Displaying Specific Insurmountable Antagonism of the α 1A- Adrenoceptor</span>. <i>Br. J.Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00532.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1111%2Fj.1476-5381.2009.00532.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20015090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhslGju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2010&pages=316-325&author=L.+Quintonauthor=E.+Girardauthor=A.+Maigaauthor=M.+Rekikauthor=P.+Lluelauthor=G.+Masuyerauthor=M.+Larregolaauthor=C.+Marquerauthor=J.+Ciolekauthor=T.+Magninauthor=R.+Wagnerauthor=J.+Molg%C3%B3author=R.+Thaiauthor=C.+Fruchart-Gaillardauthor=G.+Mourierauthor=J.+Chamot-Rookeauthor=A.+M%C3%A9nezauthor=S.+Paleaauthor=D.+Serventauthor=N.+Gilles&title=Isolation+and+Pharmacological+Characterization+of+AdTx1%2C+a+Natural+Peptide+Displaying+Specific+Insurmountable+Antagonism+of+the+%CE%B1+1A-+Adrenoceptor&doi=10.1111%2Fj.1476-5381.2009.00532.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and pharmacological characterization of AdTx1, a natural peptide displaying specific insurmountable antagonism of the α1A-adrenoceptor</span></div><div class="casAuthors">Quinton, L.; Girard, E.; Maiga, A.; Rekik, M.; Lluel, P.; Masuyer, G.; Larregola, M.; Marquer, C.; Ciolek, J.; Magnin, T.; Wagner, R.; Molgo, J.; Thai, R.; Fruchart-Gaillard, C.; Mourier, G.; Chamot-Rooke, J.; Menez, A.; Palea, S.; Servent, D.; Gilles, N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">316-325</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Venoms are a rich source of ligands for ion channels, but very little is known about their capacity to modulate G-protein coupled receptor (GPCR) activity.  We developed a strategy to identify novel toxins targeting GPCRs.  We studied the interactions of mamba venom fractions with α1-adrenoceptors in binding expts. with 3H-prazosin.  The active peptide (AdTx1) was sequenced by Edman degrdn. and mass spectrometry fragmentation.  Its synthetic homolog was pharmacol. characterized by binding expts. using cloned receptors and by functional expts. on rabbit isolated prostatic smooth muscle.  AdTx1, a 65 amino-acid peptide stabilized by four disulfide bridges, belongs to the three-finger-fold peptide family.  It has subnanomolar affinity (Ki = 0.35 nM) and high specificity for the human α1A-adrenoceptor subtype.  We showed high selectivity and affinity (Kd = 0.6 nM) of radio-labeled AdTx1 in direct binding expts. and revealed a slow assocn. const. (kon = 6 × 106 M-1/min-1) with an unusually stable α1A-adrenoceptor/AdTx1 complex (t1/2diss = 3.6 h).  AdTx1 displayed potent insurmountable antagonism of phenylephrine's actions in vitro (rabbit isolated prostatic muscle) at concns. of 10 to 100 nM.  AdTx1 is the most specific and selective peptide inhibitor for the α1A-adrenoceptor identified to date.  It displays insurmountable antagonism, acting as a potent relaxant of smooth muscle.  Its peptidic nature can be exploited to develop new tools, as a radio-labeled-AdTx1 or a fluoro-labeled-AdTx1.  Identification of AdTx1 thus offers new perspectives for developing new drugs for treating benign prostatic hyperplasia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2V3_7is37W7Vg90H21EOLACvtfcHk0ljtfFIHAdgAeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhslGju70%253D&md5=c5a528b4755efe3ffdba1bf00e1612ca</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00532.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00532.x%26sid%3Dliteratum%253Aachs%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DGirard%26aufirst%3DE.%26aulast%3DMaiga%26aufirst%3DA.%26aulast%3DRekik%26aufirst%3DM.%26aulast%3DLluel%26aufirst%3DP.%26aulast%3DMasuyer%26aufirst%3DG.%26aulast%3DLarregola%26aufirst%3DM.%26aulast%3DMarquer%26aufirst%3DC.%26aulast%3DCiolek%26aufirst%3DJ.%26aulast%3DMagnin%26aufirst%3DT.%26aulast%3DWagner%26aufirst%3DR.%26aulast%3DMolg%25C3%25B3%26aufirst%3DJ.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DFruchart-Gaillard%26aufirst%3DC.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DChamot-Rooke%26aufirst%3DJ.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DPalea%26aufirst%3DS.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DIsolation%2520and%2520Pharmacological%2520Characterization%2520of%2520AdTx1%252C%2520a%2520Natural%2520Peptide%2520Displaying%2520Specific%2520Insurmountable%2520Antagonism%2520of%2520the%2520%25CE%25B1%25201A-%2520Adrenoceptor%26jtitle%3DBr.%2520J.Pharmacol.%26date%3D2010%26volume%3D159%26spage%3D316%26epage%3D325%26doi%3D10.1111%2Fj.1476-5381.2009.00532.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jolkkonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellman, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernstedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, E.</span></span> <span> </span><span class="NLM_article-title">A Snake Toxin against Muscarinic Acetylcholine Receptors: Amino Acid Sequence, Subtype Specificity and Effect on Guinea-Pig Ileum</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">399</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/0041-0101(94)00102-e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2F0041-0101%2894%2900102-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=7570626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2MXmtFentrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1995&pages=399-410&author=M.+Jolkkonenauthor=A.+Ademauthor=U.+Hellmanauthor=C.+Wernstedtauthor=E.+Karlsson&title=A+Snake+Toxin+against+Muscarinic+Acetylcholine+Receptors%3A+Amino+Acid+Sequence%2C+Subtype+Specificity+and+Effect+on+Guinea-Pig+Ileum&doi=10.1016%2F0041-0101%2894%2900102-e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A snake toxin against muscarinic acetylcholine receptors: amino acid sequence, subtype specificity and effect on guinea pig ileum</span></div><div class="casAuthors">Jolkkonen, Mikael; Adem, Abdu; Hellman, Ulf; Wernstedt, Christer; Karlsson, Evert</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">399-40</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The sequence of muscarinic toxin 1 (MT1) from Dendroaspis angusticeps (green mamba) was detd. (66 amino acids, Mr 7509).  The central part, peptide 25-40, is rich in hydrophobic amino acids, which is a characteristic of muscarinic toxins.  MT1 started to inhibit [3H]-NMS (N-methylscopolamine) binding to synaptosomal membranes of porcine brain (contains all five receptor subtypes) at about 1 nM and to membranes from pig heart muscle (only subtype m2) at about 1 μM.  Binding of [3H]-AF-DX 384 to heart was inhibited with an IC50 of 14 μM and to brain in two steps.  In the first step (IC50 = 32 nM) binding decreased by 37%, indicating that the toxin acted on m1 or m4 receptors, each accounting for about 40%, of total receptor content.  The second step was similar to the effect on heart.  Pirenzepine inhibited binding of [125I]-MT1 to brain receptors with an IC50 of 6.5 nM, corresponding to a Ki of about 6 nM.  Literature values of Ki for pirenzepine are 16-18 nM for m1 and ≥120 mM for other subtypes.  This indicates binding of m1 receptors.  [125I]-MT1 bound to brain with a Kd of 20 nM and a Hill coeff. of 1.0, i.e. one toxin mol. per receptor.  In guinea-pig ileum, MT1 (670 nM) produced a rapid contraction, reversible by atropine.  The toxin may be an agonist, but might also cause contraction by inducing acetylcholine release by a different mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGollZAclD-2bLVg90H21EOLACvtfcHk0lhTcCH9nPDeyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtFentrc%253D&md5=fdf75fd4b8d479f9f2ddba6ef05327c6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2F0041-0101%2894%2900102-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0041-0101%252894%252900102-E%26sid%3Dliteratum%253Aachs%26aulast%3DJolkkonen%26aufirst%3DM.%26aulast%3DAdem%26aufirst%3DA.%26aulast%3DHellman%26aufirst%3DU.%26aulast%3DWernstedt%26aufirst%3DC.%26aulast%3DKarlsson%26aufirst%3DE.%26atitle%3DA%2520Snake%2520Toxin%2520against%2520Muscarinic%2520Acetylcholine%2520Receptors%253A%2520Amino%2520Acid%2520Sequence%252C%2520Subtype%2520Specificity%2520and%2520Effect%2520on%2520Guinea-Pig%2520Ileum%26jtitle%3DToxicon%26date%3D1995%26volume%3D33%26spage%3D399%26epage%3D410%26doi%3D10.1016%2F0041-0101%2894%2900102-e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cologna, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dos Santos Cardoso, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jourdan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degueldre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uetanabaro, A. P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa Neto, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thonart, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span> <span> </span><span class="NLM_article-title">Peptidomic Comparison and Characterization of the Major Components of the Venom of the Giant Ant <i>Dinoponera Quadriceps</i> Collected in Four Different Areas of Brazil</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2013.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.jprot.2013.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=24157790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2013&pages=413-422&author=C.+T.+Colognaauthor=J.+dos+Santos+Cardosoauthor=E.+Jourdanauthor=M.+Degueldreauthor=G.+Upertauthor=N.+Gillesauthor=A.+P.+T.+Uetanabaroauthor=E.+M.+Costa+Netoauthor=P.+Thonartauthor=E.+de+Pauwauthor=L.+Quinton&title=Peptidomic+Comparison+and+Characterization+of+the+Major+Components+of+the+Venom+of+the+Giant+Ant+Dinoponera+Quadriceps+Collected+in+Four+Different+Areas+of+Brazil&doi=10.1016%2Fj.jprot.2013.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidomic comparison and characterization of the major components of the venom of the giant ant Dinoponera quadriceps collected in four different areas of Brazil</span></div><div class="casAuthors">Cologna, Camila Takeno; Cardoso, Jaqueline dos Santos; Jourdan, Emmanuel; Degueldre, Michel; Upert, Gregory; Gilles, Nicolas; Uetanabaro, Ana Paula Trovatti; Costa Neto, Eraldo Medeiro; Thonart, Philippe; de Pauw, Edwin; Quinton, Loic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">413-422</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We study the venom of the giant Dinoponera quadriceps ant collected in 4 different regions of Brazil.  By using a combination of complementary mass spectrometric approaches, we aim at: (1) characterizing the venom compn. of these ants; (2) establishing a comparative anal. of the venom from 4 geog. different regions in Brazil.  This approach demonstrates that ant venom is a copious source of new compds.  Several peptides were identified and selected for de novo sequencing.  Since most of the new peptides showed similarities with antimicrobial peptides (AMPs), antimicrobial assays were performed with the purpose of evaluating their activity.  In regard to the comparative study of the 4 regions, we obsd. not only major differences in the venom compns., but also that the venoms collected in closest areas are more similar than the ones collected in distant regions.  These observations seem to highlight an adaptation of the ant venoms to the local environment.  Concerning the biol. assays, the peptides called Dq-3162 and Da-3177 showed a wide-ranging antimicrobial activity.  The characterization of new AMPs with a broad spectrum of activity and different scaffolds may aid scientists to design new therapeutic agents and understand the mechanisms of those peptides to interact with microbial membranes.  The results obtained betoken the biotechnol. potential of ant's venom.  For the 1st time this manuscript describes an extensive proteomics characterization of the D. quadriceps venom.  In addn. this study reports the variation in venom compn. of primitive ants from 4 geog. different areas of Brazil.  The results reveal the presence of ∼335 compds. for each venom/area and inter-colony variations were obsd. 16 new peptides were characterized and 2 of them were synthesized and biol. assayed.  These findings highlight the considerable and still unexplored diversity of ant's venom which could be used as valuable research tools in different areas of knowledge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonkEgVy8wrTrVg90H21EOLACvtfcHk0lhTcCH9nPDeyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsrbJ&md5=f6cca8bc8d0fd0d454a050a2f4b994ab</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2013.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2013.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DCologna%26aufirst%3DC.%2BT.%26aulast%3Ddos%2BSantos%2BCardoso%26aufirst%3DJ.%26aulast%3DJourdan%26aufirst%3DE.%26aulast%3DDegueldre%26aufirst%3DM.%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DUetanabaro%26aufirst%3DA.%2BP.%2BT.%26aulast%3DCosta%2BNeto%26aufirst%3DE.%2BM.%26aulast%3DThonart%26aufirst%3DP.%26aulast%3Dde%2BPauw%26aufirst%3DE.%26aulast%3DQuinton%26aufirst%3DL.%26atitle%3DPeptidomic%2520Comparison%2520and%2520Characterization%2520of%2520the%2520Major%2520Components%2520of%2520the%2520Venom%2520of%2520the%2520Giant%2520Ant%2520Dinoponera%2520Quadriceps%2520Collected%2520in%2520Four%2520Different%2520Areas%2520of%2520Brazil%26jtitle%3DJ.%2520Proteomics%26date%3D2013%26volume%3D94%26spage%3D413%26epage%3D422%26doi%3D10.1016%2Fj.jprot.2013.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickremasinghe, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartgerink, J. D.</span></span> <span> </span><span class="NLM_article-title">Nanofibrous Snake Venom Hemostat</span>. <i>ACS Biomater. Sci. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1305</span>, <span class="refDoi"> DOI: 10.1021/acsbiomaterials.5b00356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsbiomaterials.5b00356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12lsrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=1300-1305&author=V.+A.+Kumarauthor=N.+C.+Wickremasingheauthor=S.+Shiauthor=J.+D.+Hartgerink&title=Nanofibrous+Snake+Venom+Hemostat&doi=10.1021%2Facsbiomaterials.5b00356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Nanofibrous snake venom hemostat</span></div><div class="casAuthors">Kumar, Vivek A.; Wickremasinghe, Navindee C.; Shi, Siyu; Hartgerink, Jeffrey D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Biomaterials Science & Engineering</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1300-1305</span>CODEN:
                <span class="NLM_cas:coden">ABSEBA</span>;
        ISSN:<span class="NLM_cas:issn">2373-9878</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Controlling perioperative bleeding is of crit. importance to minimize hemorrhaging and fatality.  Patients on anticoagulant therapy such as heparin have diminished clotting potential and are at risk for hemorrhaging.  Here we describe a self-assembling nanofibrous peptide hydrogel (termed SLac) that on its own can act as a phys. barrier to blood loss.  SLac was loaded with snake-venom derived Batroxobin (50 μg/mL) yielding a drug-loaded hydrogel (SB50).  SB50 was potentiated to enhance clotting even in the presence of heparin.  In vitro evaluation of fibrin and whole blood clotting helped identify appropriate concns. for hemostasis in vivo.  Batroxobin-loaded hydrogels rapidly (within 20s) stop bleeding in both normal and heparin-treated rats in a lateral liver incision model.  Compared to std. of care, Gelfoam, and investigational hemostats such as Puramatrix, only SB50 showed rapid liver incision hemostasis post surgical application.  This snake venom-loaded peptide hydrogel can be applied via syringe and conforms to the wound site resulting in hemostasis.  This demonstrates a facile method for surgical hemostasis even in the presence of anticoagulant therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRPUTMb5YJsbVg90H21EOLACvtfcHk0ljycfW0KbQy5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12lsrzK&md5=5d23f7835a81de4649709140d1072b72</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsbiomaterials.5b00356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsbiomaterials.5b00356%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DV.%2BA.%26aulast%3DWickremasinghe%26aufirst%3DN.%2BC.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DHartgerink%26aufirst%3DJ.%2BD.%26atitle%3DNanofibrous%2520Snake%2520Venom%2520Hemostat%26jtitle%3DACS%2520Biomater.%2520Sci.%2520Eng.%26date%3D2015%26volume%3D1%26spage%3D1300%26epage%3D1305%26doi%3D10.1021%2Facsbiomaterials.5b00356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics from Venom: Current Status and Potential</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2738</span>– <span class="NLM_lpage">2758</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.bmc.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28988749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2738-2758&author=M.+W.+Penningtonauthor=A.+Czerwinskiauthor=R.+S.+Norton&title=Peptide+Therapeutics+from+Venom%3A+Current+Status+and+Potential&doi=10.1016%2Fj.bmc.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics from venom: Current status and potential</span></div><div class="casAuthors">Pennington, Michael W.; Czerwinski, Andrzej; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2738-2758</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized as being highly selective, potent and relatively safe as potential therapeutics.  Peptides isolated from the venom of different animals satisfy most of these criteria with the possible exception of safety, but when isolated as single compds. and used at appropriate concns., venom-derived peptides can become useful drugs.  Although the no. of venom-derived peptides that have successfully progressed to the clinic is currently limited, the prospects for venom-derived peptides look very optimistic.  As proteomic and transcriptomic approaches continue to identify new sequences, the potential of venom-derived peptides to find applications as therapeutics, cosmetics and insecticides grows accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8haCt_E4Tu7Vg90H21EOLACvtfcHk0ljycfW0KbQy5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI&md5=253b75540d1a2117338259c087b119f7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DCzerwinski%26aufirst%3DA.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520Therapeutics%2520from%2520Venom%253A%2520Current%2520Status%2520and%2520Potential%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2738%26epage%3D2758%26doi%3D10.1016%2Fj.bmc.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span> <span> </span><span class="NLM_article-title">Versatile Spider Venom Peptides and Their Medical and Agricultural Applications</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2018.11.298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2018.11.298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=30543821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFOmsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2019&pages=109-126&author=N.+J.+Saezauthor=V.+Herzig&title=Versatile+Spider+Venom+Peptides+and+Their+Medical+and+Agricultural+Applications&doi=10.1016%2Fj.toxicon.2018.11.298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Versatile spider venom peptides and their medical and agricultural applications</span></div><div class="casAuthors">Saez, Natalie J.; Herzig, Volker</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">109-126</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Spiders have been evolving complex and diverse repertoires of peptides in their venoms with vast pharmacol. activities for more than 300 million years.  Spiders use their venoms for prey capture and defense, hence they contain peptides that target both prey (mainly arthropods) and predators (other arthropods or vertebrates).  This includes peptides that potently and selectively modulate a range of targets such as ion channels, receptors and signaling pathways involved in physiol. processes.  The contribution of these targets in particular disease pathophysiologies makes spider venoms a valuable source of peptides with potential therapeutic use.  In addn., peptides with insecticidal activities, used for prey capture, can be exploited for the development of novel bioinsecticides for agricultural use.  Although we have already reviewed potential applications of spider venom peptides as therapeutics (in 2010) and as bioinsecticides (in 2012), a considerable no. of research articles on both topics have been published since, warranting an updated review.  Here we explore the most recent research on the use of spider venom peptides for both medical and agricultural applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZSoXvfe9NjrVg90H21EOLACvtfcHk0ljycfW0KbQy5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFOmsbzK&md5=73378135ef93499af7accc6b2f2280f6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2018.11.298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2018.11.298%26sid%3Dliteratum%253Aachs%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DHerzig%26aufirst%3DV.%26atitle%3DVersatile%2520Spider%2520Venom%2520Peptides%2520and%2520Their%2520Medical%2520and%2520Agricultural%2520Applications%26jtitle%3DToxicon%26date%3D2019%26volume%3D158%26spage%3D109%26epage%3D126%26doi%3D10.1016%2Fj.toxicon.2018.11.298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mebs, D.</span></span> <span> </span><span class="NLM_article-title">Toxicity in Animals. Trends in Evolution?</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1016/S0041-0101(00)00155-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0041-0101%2800%2900155-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10936625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A280%3ADC%252BD3M7htVKquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2001&pages=87-96&author=D.+Mebs&title=Toxicity+in+Animals.+Trends+in+Evolution%3F&doi=10.1016%2FS0041-0101%2800%2900155-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicity in animals. Trends in evolution?</span></div><div class="casAuthors">Mebs D</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon : official journal of the International Society on Toxinology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">87-96</span>
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    </div><div class="casAbstract">Animals acquire toxicity either by metabolic synthesis of toxins (secondary metabolites), by expression of toxin genes or by the uptake, storage and sequestration of toxins produced by other organisms, i.e., microbes, plants or other animals.  Variability of toxin structure and function is high.  Peptide toxins in particular, although relying on a limited number of structural frameworks, often exhibit considerable structural hypervariability.  An accelerated rate of evolution in the toxin gene structure (conserved introns, but high substitution rates in the exons) leads to the functional diversity of these peptides or proteins.  The selective forces which may drive toxin evolution are unknown.  Venomousness or the possession of toxins can be essential for survival, but the advantage of toxin biosynthesis may also be of minor importance or has been lost during evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxzMviJQQtcJSWdIdmEou5fW6udTcc2eYRbG8Rx-BY57ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3M7htVKquw%253D%253D&md5=f2701ded72deb5c568433b893a0ce0a3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0041-0101%2800%2900155-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0041-0101%252800%252900155-0%26sid%3Dliteratum%253Aachs%26aulast%3DMebs%26aufirst%3DD.%26atitle%3DToxicity%2520in%2520Animals.%2520Trends%2520in%2520Evolution%253F%26jtitle%3DToxicon%26date%3D2001%26volume%3D39%26spage%3D87%26epage%3D96%26doi%3D10.1016%2FS0041-0101%2800%2900155-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westermann, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Conopeptide Characterization and Classifications: An Analysis Using ConoServer</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2010.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2010.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20211197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFGgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2010&pages=1491-1509&author=Q.+Kaasauthor=J.-C.+Westermannauthor=D.+J.+Craik&title=Conopeptide+Characterization+and+Classifications%3A+An+Analysis+Using+ConoServer&doi=10.1016%2Fj.toxicon.2010.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conopeptide characterization and classifications: An analysis using ConoServer</span></div><div class="casAuthors">Kaas, Quentin; Westermann, Jan-Christoph; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1491-1509</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cone snails are carnivorous marine gastropods that have evolved potent venoms to capture their prey.  These venoms comprise a rich and diverse cocktail of peptide toxins, or conopeptides, whose high diversity has arisen from an efficient hypermutation mechanism, combined with a high frequency of post-translational modifications.  Conopeptides bind with high specificity to distinct membrane receptors, ion channels, and transporters of the central and muscular nervous system.  As well as serving their natural function in prey capture, conopeptides have been utilized as versatile tools in neuroscience and have proven valuable as drug leads that target the nervous system in humans.  This paper examines current knowledge on conopeptide sequences based on an anal. of gene and peptide sequences in ConoServer, a specialized database of conopeptide sequences and three-dimensional structures.  The authors describe updates to the content and organization of ConoServer and discuss correlations between gene superfamilies, cysteine frameworks, pharmacol. families targeted by conopeptides, and the phylogeny, habitat, and diet of cone snails.  The study identifies gaps in current knowledge of conopeptides and points to potential directions for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSrDCcmmrOPbVg90H21EOLACvtfcHk0lhzyedLfQ7RQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFGgtb4%253D&md5=700eedb2cffcd75936337703494cfdae</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2010.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2010.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DWestermann%26aufirst%3DJ.-C.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DConopeptide%2520Characterization%2520and%2520Classifications%253A%2520An%2520Analysis%2520Using%2520ConoServer%26jtitle%3DToxicon%26date%3D2010%26volume%3D55%26spage%3D1491%26epage%3D1509%26doi%3D10.1016%2Fj.toxicon.2010.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span> <span> </span><span class="NLM_article-title">The European FP7 Venomics Project</span>. <i>Futur. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1611</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.4155/FMC.14.85</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.4155%2Ffmc.14.85" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=25406001" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1611-1612&author=N.+Gillesauthor=D.+Servent&title=The+European+FP7+Venomics+Project&doi=10.4155%2FFMC.14.85"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.85&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.85%26sid%3Dliteratum%253Aachs%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DServent%26aufirst%3DD.%26atitle%3DThe%2520European%2520FP7%2520Venomics%2520Project%26jtitle%3DFutur.%2520Med.%2520Chem%26date%3D2014%26volume%3D6%26spage%3D1611%26epage%3D1612%26doi%3D10.4155%2FFMC.14.85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span> <span> </span><span class="NLM_article-title">Bioactive Compounds Isolated from Neglected Predatory Marine Gastropods</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">118</span>, <span class="refDoi"> DOI: 10.3390/md16040118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Fmd16040118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlOgtr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=118&author=A.+H.+Turnerauthor=D.+J.+Craikauthor=Q.+Kaasauthor=C.+I.+Schroeder&title=Bioactive+Compounds+Isolated+from+Neglected+Predatory+Marine+Gastropods&doi=10.3390%2Fmd16040118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive compounds isolated from neglected predatory marine gastropods</span></div><div class="casAuthors">Turner, Ashlin H.; Craik, David J.; Kaas, Quentin; Schroeder, Christina I.</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">118/1-118/20</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A diverse range of predatory marine gastropods produce toxins, yet most of these mols. remain uncharacterized.  Conus species have received the most attention from researchers, leading to several conopeptides reaching clin. trials.  This review aims to summarize what is known about bioactive compds. isolated from species of neglected marine gastropods, esp. in the Turridae, Terebridae, Babyloniidae, Muricidae, Buccinidae, Colubrariidae, Nassariidae, Cassidae, and Ranellidae families.  Multiple species have been reported to contain bioactive compds. with potential toxic activity, but most of these compds. have not been characterized or even clearly identified.  The bioactive properties and potential applications of echotoxins and related porins from the Ranellidae family are discussed in more detail.  Finally, the review concludes with a call for research on understudied species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQy90TNWblmrVg90H21EOLACvtfcHk0lhzyedLfQ7RQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlOgtr3N&md5=ade043d4918d08ebf8570a7b6b2a8942</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fmd16040118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd16040118%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DA.%2BH.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26atitle%3DBioactive%2520Compounds%2520Isolated%2520from%2520Neglected%2520Predatory%2520Marine%2520Gastropods%26jtitle%3DMar.%2520Drugs%26date%3D2018%26volume%3D16%26spage%3D118%26doi%3D10.3390%2Fmd16040118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morsa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiwir, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Rocca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanozin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grifnée, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longuespée, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuwis, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smargiasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzucchelli, G.</span></span> <span> </span><span class="NLM_article-title">Multi-Enzymatic Limited Digestion: The Next-Generation Sequencing for Proteomics?</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2501</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.9b00044</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.9b00044" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1yitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=2501-2513&author=D.+Morsaauthor=D.+Baiwirauthor=R.+La+Roccaauthor=T.+A.+Zimmermanauthor=E.+Hanozinauthor=E.+Grifn%C3%A9eauthor=R.+Longuesp%C3%A9eauthor=M.-A.+Meuwisauthor=N.+Smargiassoauthor=E.+De+Pauwauthor=G.+Mazzucchelli&title=Multi-Enzymatic+Limited+Digestion%3A+The+Next-Generation+Sequencing+for+Proteomics%3F&doi=10.1021%2Facs.jproteome.9b00044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Enzymatic Limited Digestion: The Next-Generation Sequencing for Proteomics?</span></div><div class="casAuthors">Morsa, Denis; Baiwir, Dominique; La Rocca, Raphael; Zimmerman, Tyler A.; Hanozin, Emeline; Grifnee, Elodie; Longuespee, Remi; Meuwis, Marie-Alice; Smargiasso, Nicolas; Pauw, Edwin De; Mazzucchelli, Gabriel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2501-2513</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Over the past 40 years, proteomics, generically defined as the field dedicated to the identification and anal. of proteins, has tremendously gained in popularity and potency through advancements in genome sequencing, separative techniques, mass spectrometry and bioinformatics algorithms.  As a consequence, its scope of application has gradually enlarged and diversified to meet specialized topical biomedical subjects nowadays.  Although the tryptic bottom-up approach is widely regarded as the gold std. for rapid screening of complex samples, its application for precise and confident mapping of protein modifications is often hindered due to partial sequence coverage, poor redundancy in indicative peptides and lack of method flexibility.  The authors here show how the synergic and time-limited action of a properly dild. mix of multiple enzymes can be exploited in a versatile yet straightforward protocol to alleviate present-day drawbacks.  Merging bottom-up and middle-down ideologies, the results highlight broad assemblies of overlapping peptides that enable refined and reliable characterizations of proteins, including variants identification, and of their carried modifications, including posttranslational modifications, truncations and cleavages.  Beyond this boost in performance, the methodol. also offers efficient de novo sequencing capabilities, in view of which the authors here present a dedicated custom assembly algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLdDzX1izqM7Vg90H21EOLACvtfcHk0lilJv4u_d_6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1yitb0%253D&md5=5be9e66709e46adf9206017919044bf9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.9b00044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.9b00044%26sid%3Dliteratum%253Aachs%26aulast%3DMorsa%26aufirst%3DD.%26aulast%3DBaiwir%26aufirst%3DD.%26aulast%3DLa%2BRocca%26aufirst%3DR.%26aulast%3DZimmerman%26aufirst%3DT.%2BA.%26aulast%3DHanozin%26aufirst%3DE.%26aulast%3DGrifn%25C3%25A9e%26aufirst%3DE.%26aulast%3DLonguesp%25C3%25A9e%26aufirst%3DR.%26aulast%3DMeuwis%26aufirst%3DM.-A.%26aulast%3DSmargiasso%26aufirst%3DN.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DMazzucchelli%26aufirst%3DG.%26atitle%3DMulti-Enzymatic%2520Limited%2520Digestion%253A%2520The%2520Next-Generation%2520Sequencing%2520for%2520Proteomics%253F%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2019%26volume%3D18%26spage%3D2501%26epage%3D2513%26doi%3D10.1021%2Facs.jproteome.9b00044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, A.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span> <span> </span><span class="NLM_article-title">Deep Venomics Reveals the Mechanism for Expanded Peptide Diversity in Cone Snail Venom</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">312</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1074/mcp.M112.021469</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fmcp.M112.021469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23152539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFerurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=312-329&author=S.+Dutertreauthor=A.-h.+Jinauthor=Q.+Kaasauthor=A.+Jonesauthor=P.+F.+Alewoodauthor=R.+J.+Lewis&title=Deep+Venomics+Reveals+the+Mechanism+for+Expanded+Peptide+Diversity+in+Cone+Snail+Venom&doi=10.1074%2Fmcp.M112.021469"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Deep Venomics Reveals the Mechanism for Expanded Peptide Diversity in Cone Snail Venom</span></div><div class="casAuthors">Dutertre, Sebastien; Jin, Ai-hua; Kaas, Quentin; Jones, Alun; Alewood, Paul F.; Lewis, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">312-329</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cone snails produce highly complex venom comprising mostly small biol. active peptides known as conotoxins or conopeptides.  Early ests. that suggested 50-200 venom peptides are produced per species have been recently increased at least 10-fold using advanced mass spectrometry.  To uncover the mechanism(s) responsible for generating this impressive diversity, we used an integrated approach combining second-generation transcriptome sequencing with high sensitivity proteomics.  From the venom gland transcriptome of Conus marmoreus, a total of 105 conopeptide precursor sequences from 13 gene superfamilies were identified.  Over 60% of these precursors belonged to the three gene superfamilies O1, T, and M, consistent with their high levels of expression, which suggests these conotoxins play an important role in prey capture and/or defense.  Seven gene superfamilies not previously identified in C. marmoreus, including five novel superfamilies, were also discovered.  To confirm the expression of toxins identified at the transcript level, the injected venom of C. marmoreus was comprehensively analyzed by mass spectrometry, revealing 2710 and 3172 peptides using MALDI and ESI-MS, resp., and 6254 peptides using an ESI-MS TripleTOF 5600 instrument.  All conopeptides derived from transcriptomic sequences could be matched to masses obtained on the TripleTOF within 100 ppm accuracy, with 66 (63%) providing MS/MS coverage that unambiguously confirmed these matches.  Comprehensive integration of transcriptomic and proteomic data revealed for the first time that the vast majority of the conopeptide diversity arises from a more limited set of genes through a process of variable peptide processing, which generates conopeptides with alternative cleavage sites, heterogeneous post-translational modifications, and highly variable N- and C-terminal truncations.  Variable peptide processing is expected to contribute to the evolution of venoms, and explains how a limited set of ∼ 100 gene transcripts can generate thousands of conopeptides in a single species of cone snail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG3WIrFRD1XbVg90H21EOLACvtfcHk0lilJv4u_d_6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFerurs%253D&md5=2cb31d94877f9c33d1b559542796f512</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M112.021469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M112.021469%26sid%3Dliteratum%253Aachs%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DA.-h.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26atitle%3DDeep%2520Venomics%2520Reveals%2520the%2520Mechanism%2520for%2520Expanded%2520Peptide%2520Diversity%2520in%2520Cone%2520Snail%2520Venom%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2013%26volume%3D12%26spage%3D312%26epage%3D329%26doi%3D10.1074%2Fmcp.M112.021469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span> <span> </span><span class="NLM_article-title">Native Pyroglutamation of Huwentoxin-IV: A Post-Translational Modification That Increases the Trapping Ability to the Sodium Channel</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e65984</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0065984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1371%2Fjournal.pone.0065984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23826086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVyhtrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=M.+Rongauthor=Z.+Duanauthor=J.+Chenauthor=J.+Liauthor=Y.+Xiaoauthor=S.+Liang&title=Native+Pyroglutamation+of+Huwentoxin-IV%3A+A+Post-Translational+Modification+That+Increases+the+Trapping+Ability+to+the+Sodium+Channel&doi=10.1371%2Fjournal.pone.0065984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Native pyroglutamation of huwentoxin-IV: a post-translational modification that increases the trapping ability to the sodium channel</span></div><div class="casAuthors">Rong, Mingqiang; Duan, Zhigui; Chen, Juliang; Li, Jianglin; Xiao, Yuchen; Liang, Songping</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e65984</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Huwentoxin-IV (HWTX-IV), a tetrodotoxin-sensitive (TTX-s) sodium channel antagonist, is found in the venom of the Chinese spider Ornithoctonus huwena.  A naturally modified HWTX-IV (mHWTX-IV), having a mol. mass 18 Da lower than HWTX-IV, has also been isolated from the venom of the same spider.  By a combination of enzymic fragmentation and MS/MS de novo sequencing, mHWTX-IV has been shown to have the same amino acid sequence as that of HWTX-IV, except that the N-terminal glutamic acid replaced by pyroglutamic acid.  MHWTX-IV inhibited tetrodotoxin-sensitive voltage-gated sodium channels of dorsal root ganglion neurons with an IC50 nearly equal to native HWTX-IV.  MHWTX-IV showed the same activation and inactivation kinetics seen for native HWTX-IV.  In contrast with HWTX-IV, which dissocs. at moderate voltage depolarization voltages (+50 mV, 180000 ms), mHWTX-IV inhibition of TTX-sensitive sodium channels is not reversed by strong depolarization voltages (+200 mV, 500 ms).  Recovery of Nav1.7current was voltage-dependent and was induced by extreme depolarization in the presence of HWTX-IV, but no obvious current was elicited after application of mHWTX-IV.  Our data indicate that the N-terminal modification of HWTX-IV gives the peptide toxin a greater ability to trap the voltage sensor in the sodium channel.  Loss of a neg. charge, caused by cyclization at the N-terminus, is a possible reason why the modified toxin binds much stronger.  To our knowledge, this is the first report of a pyroglutamic acid residue in a spider toxin; this modification seems to increase the trapping ability of the voltage sensor in the sodium channel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogEk5MciNqr7Vg90H21EOLACvtfcHk0lilJv4u_d_6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVyhtrjK&md5=2a408cf4a206b555856f438263f13635</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0065984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0065984%26sid%3Dliteratum%253Aachs%26aulast%3DRong%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DS.%26atitle%3DNative%2520Pyroglutamation%2520of%2520Huwentoxin-IV%253A%2520A%2520Post-Translational%2520Modification%2520That%2520Increases%2520the%2520Trapping%2520Ability%2520to%2520the%2520Sodium%2520Channel%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0065984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravindran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moczydlowski, E.</span></span> <span> </span><span class="NLM_article-title">Analysis of the Blocking Activity of Charybdotoxin Homologs and Iodinated Derivatives against Ca<sup>2+</sup>-Activated K<sup>+</sup> Channels</span>. <i>J. Membr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1007/BF01870284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2FBF01870284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=2477548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaL1MXlvV2hsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=1989&pages=269-281&author=K.+Lucchesiauthor=A.+Ravindranauthor=H.+Youngauthor=E.+Moczydlowski&title=Analysis+of+the+Blocking+Activity+of+Charybdotoxin+Homologs+and+Iodinated+Derivatives+against+Ca2%2B-Activated+K%2B+Channels&doi=10.1007%2FBF01870284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the blocking activity of charybdotoxin homologs and iodinated derivatives against calcium-activated potassium channels</span></div><div class="casAuthors">Lucchesi, Kathryn; Ravindran, Arippa; Young, Howard; Moczydlowski, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Membrane Biology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-81</span>CODEN:
                <span class="NLM_cas:coden">JMBBBO</span>;
        ISSN:<span class="NLM_cas:issn">0022-2631</span>.
    </div><div class="casAbstract">Two charybdotoxin peptides were purified from venom of the Israeli scorpion, Leiurus quinquestriatus hebraeus.  Microsequencing of the most abundant toxin, ChTX-Lq1, revealed identity with the 37-residue peptide.  Sequence data on the minor peptide, ChTX-Lq2, showed substantial homol. to ChTX-Lq1 with differences obsd. at 8 positions.  These 2 charybdotoxin sequences, along with that of noxiustoxin, define a distinct family of scorpion peptide toxins with activity against K+ channels.  Both charybdotoxin homologs inhibited Ca2+-dependent K+ efflux from human erythrocytes with similar potency, K0.5 ≃ 40 nM.  In planar bilayer assays of single K(Ca) channels from rat muscle, ChTX-Lq1 and ChTX-Lq2 blocked with intrinsic Kd's of 1.3 and 43 nM, resp., in the presence of 50 mM external KCl.  A new application of dwell-time histogram anal. of single-channel blocking events was used to characterize the kinetic homogeneity of toxin samples and the blocking kinetics of ChTX derivs.  The lower blocking affinity of ChTX-Lq2 was the combined result of a faster dissocn. rate and a slower assocn. rate as compared to ChTX-Lq1.  The blocking activity of 2 monoiodinated derivs. of ChTX-Lq1 was also analyzed.  Blocked dwell-time histograms of the iodinated peptides were characterized by predominately brief (0.2-2 s) blocking events in comparison to the native toxin (20 s).  Histogram anal. revealed that monoiodination of ChTX-Lq1 impairs blocking activity by adverse effects on both dissocn. and assocn. rate consts.  Frequency d. histograms of single channel blocking events provide a sensitive assay of toxin purity suitable for quantitating structure-activity relationships of charybdotoxin derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbd84u1dutE7Vg90H21EOLACvtfcHk0lilJv4u_d_6xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlvV2hsLY%253D&md5=156df9680d3aee8bea904ec57872a723</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2FBF01870284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01870284%26sid%3Dliteratum%253Aachs%26aulast%3DLucchesi%26aufirst%3DK.%26aulast%3DRavindran%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DH.%26aulast%3DMoczydlowski%26aufirst%3DE.%26atitle%3DAnalysis%2520of%2520the%2520Blocking%2520Activity%2520of%2520Charybdotoxin%2520Homologs%2520and%2520Iodinated%2520Derivatives%2520against%2520Ca2%252B-Activated%2520K%252B%2520Channels%26jtitle%3DJ.%2520Membr.%2520Biol.%26date%3D1989%26volume%3D109%26spage%3D269%26epage%3D281%26doi%3D10.1007%2FBF01870284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Er, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span> <span> </span><span class="NLM_article-title">Spider-Venom Peptides as Therapeutics</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2851</span>– <span class="NLM_lpage">2871</span>, <span class="refDoi"> DOI: 10.3390/toxins2122851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Ftoxins2122851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=22069579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFWgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=2851-2871&author=N.+J.+Saezauthor=S.+Senffauthor=J.+E.+Jensenauthor=S.+Y.+Erauthor=V.+Herzigauthor=L.+D.+Rashauthor=G.+F.+King&title=Spider-Venom+Peptides+as+Therapeutics&doi=10.3390%2Ftoxins2122851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Spider-venom peptides as therapeutics</span></div><div class="casAuthors">Saez, Natalie J.; Senff, Sebastian; Jensen, Jonas E.; Er, Sing Yan; Herzig, Volker; Rash, Lachlan D.; King, Glenn F.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2851-2871</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  Spiders are the most successful venomous animals and the most abundant terrestrial predators.  Their remarkable success is due in large part to their ingenious exploitation of silk and the evolution of pharmacol. complex venoms that ensure rapid subjugation of prey.  Most spider venoms are dominated by disulfide-rich peptides that typically have high affinity and specificity for particular subtypes of ion channels and receptors.  Spider venoms are conservatively predicted to contain more than 10 million bioactive peptides, making them a valuable resource for drug discovery.  Here we review the structure and pharmacol. of spider-venom peptides that are being used as leads for the development of therapeutics against a wide range of pathophysiol. conditions including cardiovascular disorders, chronic pain, inflammation, and erectile dysfunction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA11F1oYodH7Vg90H21EOLACvtfcHk0lgHWMC0in6fPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFWgsQ%253D%253D&md5=8b4bcd142d15507b22d66a45eb943bfa</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Ftoxins2122851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins2122851%26sid%3Dliteratum%253Aachs%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DSenff%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DJ.%2BE.%26aulast%3DEr%26aufirst%3DS.%2BY.%26aulast%3DHerzig%26aufirst%3DV.%26aulast%3DRash%26aufirst%3DL.%2BD.%26aulast%3DKing%26aufirst%3DG.%2BF.%26atitle%3DSpider-Venom%2520Peptides%2520as%2520Therapeutics%26jtitle%3DToxins%26date%3D2010%26volume%3D2%26spage%3D2851%26epage%3D2871%26doi%3D10.3390%2Ftoxins2122851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronjat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Waard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span> <span> </span><span class="NLM_article-title">Functional Evolution of Scorpion Venom Peptides with an Inhibitor Cystine Knot Fold</span>. <i>Biosci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1042/BSR20130052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1042%2FBSR20130052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVSmtL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=513-527&author=B.+Gaoauthor=P.+J.+Harveyauthor=D.+J.+Craikauthor=M.+Ronjatauthor=M.+De+Waardauthor=S.+Zhu&title=Functional+Evolution+of+Scorpion+Venom+Peptides+with+an+Inhibitor+Cystine+Knot+Fold&doi=10.1042%2FBSR20130052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Functional evolution of scorpion venom peptides with an inhibitor cystine knot fold</span></div><div class="casAuthors">Gao, Bin; Harvey, Peta J.; Craik, David J.; Ronjat, Michel; De Waard, Michel; Zhu, Shunyi</div><div class="citationInfo"><span class="NLM_cas:title">Bioscience Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">513-527, S513/1</span>CODEN:
                <span class="NLM_cas:coden">BRPTDT</span>;
        ISSN:<span class="NLM_cas:issn">0144-8463</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The ICK (inhibitor cystine knot) defines a large superfamily of polypeptides with high structural stability and functional diversity.  Here, we describe a new scorpion venom-derived K+ channel toxin (named λ-MeuKTx-1) with an ICK fold through gene cloning, chem. synthesis, NMR spectroscopy, Ca2+ release measurements and electrophysiol. recordings.  λ-MeuKTx-1 was found to adopt an ICK fold that contains a three-strand anti-parallel β-sheet and a 310-helix.  Functionally, this peptide selectively inhibits the Drosophila Shaker K+ channel but is not capable of activating skeletal-type Ca2+ release channels/ryanodine receptors, which is remarkably different from the previously known scorpion venom ICK peptides.  The removal of two C-terminal residues of λ-MeuKTx-1 led to the loss of the inhibitory activity on the channel, whereas the C-terminal amidation resulted in the emergence of activity on four mammalian K+ channels accompanied by the loss of activity on the Shaker channel.  A combination of structural and pharmacol. data allows the recognition of three putative functional sites involved in channel blockade of λ-MeuKTx-1.  The presence of a functional dyad in λ-MeuKTx-1 supports functional convergence among scorpion venom peptides with different folds.  Furthermore, similarities in precursor organization, exon-intron structure, 3D-fold and function suggest that scorpion venom ICK-type K+ channel inhibitors and Ca2+ release channel activators share a common ancestor and their divergence occurs after speciation between buthidae and non-buthids.  The structural and functional characterizations of the first scorpion venom ICK toxin with K+ channel-blocking activity sheds light on functionally divergent and convergent evolution of this conserved scaffold of ancient origin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwiFGTa9ug97Vg90H21EOLACvtfcHk0lgHWMC0in6fPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVSmtL%252FN&md5=580122c3695ef7934fbf520bda938616</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1042%2FBSR20130052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20130052%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DB.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DRonjat%26aufirst%3DM.%26aulast%3DDe%2BWaard%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.%26atitle%3DFunctional%2520Evolution%2520of%2520Scorpion%2520Venom%2520Peptides%2520with%2520an%2520Inhibitor%2520Cystine%2520Knot%2520Fold%26jtitle%3DBiosci.%2520Rep.%26date%3D2013%26volume%3D33%26spage%3D513%26epage%3D527%26doi%3D10.1042%2FBSR20130052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Conotoxin Gene Superfamilies</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">6058</span>– <span class="NLM_lpage">6101</span>, <span class="refDoi"> DOI: 10.3390/md12126058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Fmd12126058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=25522317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1Klsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=6058-6101&author=S.+D.+Robinsonauthor=R.+S.+Norton&title=Conotoxin+Gene+Superfamilies&doi=10.3390%2Fmd12126058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Conotoxin gene superfamilies</span></div><div class="casAuthors">Robinson, Samuel D.; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6058-6101, 44 pp.</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Conotoxins are the peptidic components of the venoms of marine cone snails (genus Conus).  They are remarkably diverse in terms of structure and function.  Unique potency and selectivity profiles for a range of neuronal targets have made several conotoxins valuable as research tools, drug leads and even therapeutics, and has resulted in a concerted and increasing drive to identify and characterize new conotoxins.  Conotoxins are translated from mRNA as peptide precursors, and cDNA sequencing is now the primary method for identification of new conotoxin sequences.  As a result, gene superfamily, a classification based on precursor signal peptide identity, has become the most convenient method of conotoxin classification.  Here we review each of the described conotoxin gene superfamilies, with a focus on the structural and functional diversity present in each.  This review is intended to serve as a practical guide to conotoxin superfamilies and to facilitate interpretation of the increasing no. of conotoxin precursor sequences being identified by targeted-cDNA sequencing and more recently high-throughput transcriptome sequencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0yTywrt35WLVg90H21EOLACvtfcHk0lhxFZUAbyn7UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1Klsbw%253D&md5=7a11520b65e7031b9656b41a1052f26e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3390%2Fmd12126058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd12126058%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DS.%2BD.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DConotoxin%2520Gene%2520Superfamilies%26jtitle%3DMar.%2520Drugs%26date%3D2014%26volume%3D12%26spage%3D6058%26epage%3D6101%26doi%3D10.3390%2Fmd12126058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imperial, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillyard, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivera, B. M.</span></span> <span> </span><span class="NLM_article-title">The Augertoxins: Biochemical Characterization of Venom Components from the Toxoglossate Gastropod <i>Terebra Subulata</i></span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1016/S0041-0101(03)00169-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0041-0101%2803%2900169-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=14505939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntl2jt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=391-398&author=J.+S.+Imperialauthor=M.+Watkinsauthor=P.+Chenauthor=D.+R.+Hillyardauthor=L.+J.+Cruzauthor=B.+M.+Olivera&title=The+Augertoxins%3A+Biochemical+Characterization+of+Venom+Components+from+the+Toxoglossate+Gastropod+Terebra+Subulata&doi=10.1016%2FS0041-0101%2803%2900169-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The augertoxins: biochemical characterization of venom components from the toxoglossate gastropod Terebra subulata</span></div><div class="casAuthors">Imperial, Julita S.; Watkins, Maren; Chen, Ping; Hillyard, David R.; Cruz, Lourdes J.; Olivera, Baldomero M.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">391-398</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We describe the purifn. and biochem. characterization of three components from the venom of the toxoglossate gastropod Terebra subulata.  The three polypeptide venom components, augertoxins s6a, s7a and s11a, are 40-41 AA in length with 3-4 disulfide linkages.  The arrangement of Cys residues is reminiscent of certain conopeptide superfamilies, but mol. cloning failed to show the highly conserved sequence features diagnostic of the conopeptide gene superfamily with a similar arrangement of Cys residues.  One of the purified peptides, s7a, elicited an uncoordinated twisting syndrome when injected into the nematode Caenorhabditis elegans, but had no effect on mice.  T. subulata belongs to the family Terebridae, one of four major groups of toxoglossate gastropods in the superfamily Conacea.  The results reveal that some features of the augertoxins and conotoxins are generally similar, such as the organization of prepropeptide precursors and their proteolytic processing into mature toxins; however, Terebra may have evolved generally larger venom components that are less highly post-translationally modified.  The results suggest that Conus peptide gene superfamilies probably do not extend to the Terebridae, suggesting that distinctive venom gene superfamilies may be expressed in each major division of Conacean gastropods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruKqhz-aXpPrVg90H21EOLACvtfcHk0lhxFZUAbyn7UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntl2jt7k%253D&md5=377146964be8d14418df6096112d35bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0041-0101%2803%2900169-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0041-0101%252803%252900169-7%26sid%3Dliteratum%253Aachs%26aulast%3DImperial%26aufirst%3DJ.%2BS.%26aulast%3DWatkins%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DHillyard%26aufirst%3DD.%2BR.%26aulast%3DCruz%26aufirst%3DL.%2BJ.%26aulast%3DOlivera%26aufirst%3DB.%2BM.%26atitle%3DThe%2520Augertoxins%253A%2520Biochemical%2520Characterization%2520of%2520Venom%2520Components%2520from%2520the%2520Toxoglossate%2520Gastropod%2520Terebra%2520Subulata%26jtitle%3DToxicon%26date%3D2003%26volume%3D42%26spage%3D391%26epage%3D398%26doi%3D10.1016%2FS0041-0101%2803%2900169-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavergne, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harliwong, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span> <span> </span><span class="NLM_article-title">Optimized Deep-Targeted Proteotranscriptomic Profiling Reveals Unexplored Conus Toxin Diversity and Novel Cysteine Frameworks</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E3782</span>– <span class="NLM_lpage">E3791</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501334112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.1501334112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=26150494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFajsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E3782-E3791&author=V.+Lavergneauthor=I.+Harliwongauthor=A.+Jonesauthor=D.+Millerauthor=R.+J.+Taftauthor=P.+F.+Alewood&title=Optimized+Deep-Targeted+Proteotranscriptomic+Profiling+Reveals+Unexplored+Conus+Toxin+Diversity+and+Novel+Cysteine+Frameworks&doi=10.1073%2Fpnas.1501334112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Optimized deep-targeted proteotranscriptomic profiling reveals unexplored Conus toxin diversity and novel cysteine frameworks</span></div><div class="casAuthors">Lavergne, Vincent; Harliwong, Ivon; Jones, Alun; Miller, David; Taft, Ryan J.; Alewood, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">E3782-E3791</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cone snails are predatory marine gastropods characterized by a sophisticated venom app. responsible for the biosynthesis and delivery of complex mixts. of cysteine-rich toxin peptides.  These conotoxins fold into small highly structured frameworks, allowing them to potently and selectively interact with heterologous ion channels and receptors.  Approx. 2,000 toxins from an estd. no. of >70,000 bioactive peptides have been identified in the genus Conus to date.  Here, we describe a high-resoln. interrogation of the transcriptomes (available at www.ddbj.nig.ac.jp) and proteomes of the diverse compartments of the Conus episcopatus venom app.  Using biochem. and bioinformatic tools, we found the highest no. of conopeptides yet discovered in a single Conus specimen, with 3,305 novel precursor toxin sequences classified into 9 known superfamilies (A, I1, I2, M, O1, O2, S, T, Z), and identified 16 new superfamilies showing unique signal peptide signatures.  We were also able to depict the largest population of venom peptides contg. the pharmacol. active C-C-CC-C-C inhibitor cystine knot and CC-C-C motifs (168 and 44 toxins, resp.), as well as 208 new conotoxins displaying odd nos. of cysteine residues derived from known conotoxin motifs.  Importantly, six novel cysteine-rich frameworks were revealed which may have novel pharmacol.  Finally, analyses of codon usage bias and RNA-editing processes of the conotoxin transcripts demonstrate a specific conservation of the cysteine skeleton at the nucleic acid level and provide new insights about the origin of sequence hypervariablity in mature toxin regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGvczG7RzuLVg90H21EOLACvtfcHk0lhxFZUAbyn7UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFajsr%252FM&md5=4a45d5932749b5814d76c7f93cd2b848</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501334112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501334112%26sid%3Dliteratum%253Aachs%26aulast%3DLavergne%26aufirst%3DV.%26aulast%3DHarliwong%26aufirst%3DI.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DTaft%26aufirst%3DR.%2BJ.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26atitle%3DOptimized%2520Deep-Targeted%2520Proteotranscriptomic%2520Profiling%2520Reveals%2520Unexplored%2520Conus%2520Toxin%2520Diversity%2520and%2520Novel%2520Cysteine%2520Frameworks%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE3782%26epage%3DE3791%26doi%3D10.1073%2Fpnas.1501334112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdenaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Production of Two Disulphide-Bridge Toxins</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">8408</span>– <span class="NLM_lpage">8411</span>, <span class="refDoi"> DOI: 10.1039/c4cc02679a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1039%2Fc4cc02679a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=24947561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFWntrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=8408-8411&author=G.+Upertauthor=G.+Mourierauthor=A.+Pastorauthor=M.+Verdenaudauthor=D.+Aliliauthor=D.+Serventauthor=N.+Gilles&title=High-Throughput+Production+of+Two+Disulphide-Bridge+Toxins&doi=10.1039%2Fc4cc02679a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput production of two disulphide-bridge toxins</span></div><div class="casAuthors">Upert, Gregory; Mourier, Gilles; Pastor, Alexandra; Verdenaud, Marion; Alili, Doria; Servent, Denis; Gilles, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">61</span>),
    <span class="NLM_cas:pages">8408-8411</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A quick and efficient prodn. compatible with high-throughput screening was developed using 36 toxins belonging to four different families of two disulfide-bridge toxins.  Final toxins were characterized using HPLC co-elution, CD and pharmacol. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzZqdjcWb5k7Vg90H21EOLACvtfcHk0lhxFZUAbyn7UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFWntrzP&md5=f71ace2e088f676b46a2d586cfb3c8ba</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc4cc02679a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4cc02679a%26sid%3Dliteratum%253Aachs%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DPastor%26aufirst%3DA.%26aulast%3DVerdenaud%26aufirst%3DM.%26aulast%3DAlili%26aufirst%3DD.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DHigh-Throughput%2520Production%2520of%2520Two%2520Disulphide-Bridge%2520Toxins%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D8408%26epage%3D8411%26doi%3D10.1039%2Fc4cc02679a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duhoo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequeira, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turchetto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peysson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brás, J. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darbon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, C. M. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Production of Oxidized Animal Toxins in Escherichia Coli</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2025</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-9624-7_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2F978-1-4939-9624-7_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=31267452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtF2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2025&publication_year=2019&pages=165-190&author=Y.+Duhooauthor=A.+F.+Sequeiraauthor=N.+J.+Saezauthor=J.+Turchettoauthor=L.+Ramondauthor=F.+Peyssonauthor=J.+L.+A.+Br%C3%A1sauthor=N.+Gillesauthor=H.+Darbonauthor=C.+M.+G.+A.+Fontesauthor=R.+Vincentelli&title=High-Throughput+Production+of+Oxidized+Animal+Toxins+in+Escherichia+Coli&doi=10.1007%2F978-1-4939-9624-7_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Production of Oxidized Animal Toxins in Escherichia coli</span></div><div class="casAuthors">Duhoo, Yoan; Sequeira, Ana Filipa; Saez, Natalie J.; Turchetto, Jeremy; Ramond, Laurie; Peysson, Fanny; Bras, Joana L. A.; Gilles, Nicolas; Darbon, Herve; Fontes, Carlos M. G. A.; Vincentelli, Renaud</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">2025</span>
        (<span class="NLM_cas:issue">High-Throughput Protein Production and Purification</span>),
    <span class="NLM_cas:pages">165-190</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  High-throughput prodn. (HTP) of synthetic genes is becoming an important tool to explore the biol. function of the extensive genomic and meta-genomic information currently available from various sources.  One such source is animal venom, which contains thousands of novel bioactive peptides with potential uses as novel therapeutics to treat a plethora of diseases as well as in environmentally benign bioinsecticide formulations.  Here, we describe a HTP platform for recombinant bacterial prodn. of oxidized disulfide-rich proteins and peptides from animal venoms.  High-throughput, host-optimized, gene synthesis and subcloning, combined with robust HTP expression and purifn. protocols, generate a semiautomated pipeline for the accelerated prodn. of proteins and peptides identified from genomic or transcriptomic libraries.  The platform has been applied to the prodn. of thousands of animal venom peptide toxins for the purposes of drug discovery, but has the power to be universally applicable for high-level prodn. of various and diverse target proteins in sol. form.  This chapter details the HTP protocol for gene synthesis and prodn., which supported high levels of peptide expression in the E. coli periplasm using a cleavable DsbC fusion.  Finally, target proteins and peptides are purified using automated HTP methods, before undergoing quality control and screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPARfxMXjPTrVg90H21EOLACvtfcHk0lhFnX-ZxBnPKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtF2lsLY%253D&md5=ccaa62b457c99f2c7552ebbd2e4c259b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-9624-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-9624-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DDuhoo%26aufirst%3DY.%26aulast%3DSequeira%26aufirst%3DA.%2BF.%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DTurchetto%26aufirst%3DJ.%26aulast%3DRamond%26aufirst%3DL.%26aulast%3DPeysson%26aufirst%3DF.%26aulast%3DBr%25C3%25A1s%26aufirst%3DJ.%2BL.%2BA.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDarbon%26aufirst%3DH.%26aulast%3DFontes%26aufirst%3DC.%2BM.%2BG.%2BA.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DHigh-Throughput%2520Production%2520of%2520Oxidized%2520Animal%2520Toxins%2520in%2520Escherichia%2520Coli%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2019%26volume%3D2025%26spage%3D165%26epage%3D190%26doi%3D10.1007%2F978-1-4939-9624-7_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sequeira, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turchetto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peysson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhoo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blémont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gama, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, L. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerreiro, C. I. P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darbon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, C. M. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">Gene Design, Fusion Technology and TEV Cleavage Conditions Influence the Purification of Oxidized Disulphide-Rich Venom Peptides in <i>Escherichia Coli</i></span>. <i>Microb. Cell Fact.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1186/s12934-016-0618-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1186%2Fs12934-016-0618-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28049473" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1-16&author=A.+F.+Sequeiraauthor=J.+Turchettoauthor=N.+J.+Saezauthor=F.+Peyssonauthor=L.+Ramondauthor=Y.+Duhooauthor=M.+Bl%C3%A9montauthor=V.+O.+Fernandesauthor=L.+T.+Gamaauthor=L.+M.+A.+Ferreiraauthor=C.+I.+P.+I.+Guerreiroauthor=N.+Gillesauthor=H.+Darbonauthor=C.+M.+G.+A.+Fontesauthor=R.+Vincentelli&title=Gene+Design%2C+Fusion+Technology+and+TEV+Cleavage+Conditions+Influence+the+Purification+of+Oxidized+Disulphide-Rich+Venom+Peptides+in+Escherichia+Coli&doi=10.1186%2Fs12934-016-0618-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1186%2Fs12934-016-0618-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12934-016-0618-0%26sid%3Dliteratum%253Aachs%26aulast%3DSequeira%26aufirst%3DA.%2BF.%26aulast%3DTurchetto%26aufirst%3DJ.%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DPeysson%26aufirst%3DF.%26aulast%3DRamond%26aufirst%3DL.%26aulast%3DDuhoo%26aufirst%3DY.%26aulast%3DBl%25C3%25A9mont%26aufirst%3DM.%26aulast%3DFernandes%26aufirst%3DV.%2BO.%26aulast%3DGama%26aufirst%3DL.%2BT.%26aulast%3DFerreira%26aufirst%3DL.%2BM.%2BA.%26aulast%3DGuerreiro%26aufirst%3DC.%2BI.%2BP.%2BI.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDarbon%26aufirst%3DH.%26aulast%3DFontes%26aufirst%3DC.%2BM.%2BG.%2BA.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DGene%2520Design%252C%2520Fusion%2520Technology%2520and%2520TEV%2520Cleavage%2520Conditions%2520Influence%2520the%2520Purification%2520of%2520Oxidized%2520Disulphide-Rich%2520Venom%2520Peptides%2520in%2520Escherichia%2520Coli%26jtitle%3DMicrob.%2520Cell%2520Fact.%26date%3D2017%26volume%3D16%26spage%3D1%26epage%3D16%26doi%3D10.1186%2Fs12934-016-0618-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turchetto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequeira, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peysson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brás, J. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhoo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blémont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerreiro, C. I. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darbon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, C. M. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Expression of Animal Venom Toxins in <i>Escherichia Coli</i> to Generate a Large Library of Oxidized Disulphide-Reticulated Peptides for Drug Discovery</span>. <i>Microb. Cell Fact.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1186/s12934-016-0617-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1186%2Fs12934-016-0617-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28049473" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1-15&author=J.+Turchettoauthor=A.+F.+Sequeiraauthor=L.+Ramondauthor=F.+Peyssonauthor=J.+L.+A.+Br%C3%A1sauthor=N.+J.+Saezauthor=Y.+Duhooauthor=M.+Bl%C3%A9montauthor=C.+I.+P.+D.+Guerreiroauthor=L.+Quintonauthor=E.+Pauwauthor=N.+Gillesauthor=H.+Darbonauthor=C.+M.+G.+A.+Fontesauthor=R.+Vincentelli&title=High-Throughput+Expression+of+Animal+Venom+Toxins+in+Escherichia+Coli+to+Generate+a+Large+Library+of+Oxidized+Disulphide-Reticulated+Peptides+for+Drug+Discovery&doi=10.1186%2Fs12934-016-0617-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2Fs12934-016-0617-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12934-016-0617-1%26sid%3Dliteratum%253Aachs%26aulast%3DTurchetto%26aufirst%3DJ.%26aulast%3DSequeira%26aufirst%3DA.%2BF.%26aulast%3DRamond%26aufirst%3DL.%26aulast%3DPeysson%26aufirst%3DF.%26aulast%3DBr%25C3%25A1s%26aufirst%3DJ.%2BL.%2BA.%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DDuhoo%26aufirst%3DY.%26aulast%3DBl%25C3%25A9mont%26aufirst%3DM.%26aulast%3DGuerreiro%26aufirst%3DC.%2BI.%2BP.%2BD.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DPauw%26aufirst%3DE.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDarbon%26aufirst%3DH.%26aulast%3DFontes%26aufirst%3DC.%2BM.%2BG.%2BA.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DHigh-Throughput%2520Expression%2520of%2520Animal%2520Venom%2520Toxins%2520in%2520Escherichia%2520Coli%2520to%2520Generate%2520a%2520Large%2520Library%2520of%2520Oxidized%2520Disulphide-Reticulated%2520Peptides%2520for%2520Drug%2520Discovery%26jtitle%3DMicrob.%2520Cell%2520Fact.%26date%3D2017%26volume%3D16%26spage%3D1%26epage%3D15%26doi%3D10.1186%2Fs12934-016-0617-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haler, J. R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smargiasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">Disulfide Connectivity Analysis of Peptides Bearing Two Intramolecular Disulfide Bonds Using MALDI In-Source Decay</span>. <i>J. Am. Soc. Mass Spectrom.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1995</span>– <span class="NLM_lpage">2002</span>, <span class="refDoi"> DOI: 10.1007/s13361-018-2022-y</span> </span><div class="citationLinks">[<a href="/doi/10.1007/s13361-018-2022-y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWgu7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=1995-2002&author=P.+Massonnetauthor=J.+R.+N.+Halerauthor=G.+Upertauthor=N.+Smargiassoauthor=G.+Mourierauthor=N.+Gillesauthor=L.+Quintonauthor=E.+De+Pauw&title=Disulfide+Connectivity+Analysis+of+Peptides+Bearing+Two+Intramolecular+Disulfide+Bonds+Using+MALDI+In-Source+Decay&doi=10.1007%2Fs13361-018-2022-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide Connectivity Analysis of Peptides Bearing Two Intramolecular Disulfide Bonds Using MALDI In-Source Decay</span></div><div class="casAuthors">Massonnet, Philippe; Haler, Jean R. N.; Upert, Gregory; Smargiasso, Nicolas; Mourier, Gilles; Gilles, Nicolas; Quinton, Loic; De Pauw, Edwin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1995-2002</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Disulfide connectivity in peptides bearing at least two intramol. disulfide bonds is highly important for the structure and the biol. activity of the peptides.  In that context, anal. strategies allowing a characterization of the cysteine pairing are of prime interest for chemists, biochemists, and biologists.  For that purpose, this study evaluates the potential of MALDI in-source decay (ISD) for characterizing cysteine pairs through the systematic anal. of identical peptides bearing two disulfide bonds, but not the same cysteine connectivity.  Three different matrixes have been tested in pos. and/or in neg. mode (1,5-DAN, 2-AB and 2-AA).  As MALDI-ISD is known to partially reduce disulfide bonds, the data anal. of this study rests firstly on the deconvolution of the isotope pattern of the parent ions.  Moreover, data anal. is also based on the formed fragment ions and their signal intensities.  Results from MS/MS-expts. (MALDI-ISD-MS/MS) constitute the last ref. for data interpretation.  Owing to the combined use of different ISD-promoting matrixes, cysteine connectivity identification could be performed on the considered peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKmXEE2uTWfbVg90H21EOLACvtfcHk0ljbnZ9AiPUNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWgu7bI&md5=b29930e096f4668a5b28c8d5b61d9056</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2Fs13361-018-2022-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13361-018-2022-y%26sid%3Dliteratum%253Aachs%26aulast%3DMassonnet%26aufirst%3DP.%26aulast%3DHaler%26aufirst%3DJ.%2BR.%2BN.%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DSmargiasso%26aufirst%3DN.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DDisulfide%2520Connectivity%2520Analysis%2520of%2520Peptides%2520Bearing%2520Two%2520Intramolecular%2520Disulfide%2520Bonds%2520Using%2520MALDI%2520In-Source%2520Decay%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D2018%26volume%3D29%26spage%3D1995%26epage%3D2002%26doi%3D10.1007%2Fs13361-018-2022-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muratspahić, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freissmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, C. W.</span></span> <span> </span><span class="NLM_article-title">Nature-Derived Peptides: A Growing Niche for GPCR Ligand Discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">309</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2019.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.tips.2019.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=30955896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVWnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=309-326&author=E.+Muratspahi%C4%87author=M.+Freissmuthauthor=C.+W.+Gruber&title=Nature-Derived+Peptides%3A+A+Growing+Niche+for+GPCR+Ligand+Discovery&doi=10.1016%2Fj.tips.2019.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Nature-Derived Peptides: A Growing Niche for GPCR Ligand Discovery</span></div><div class="casAuthors">Muratspahic, Edin; Freissmuth, Michael; Gruber, Christian W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">309-326</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) represent important drug targets, as they regulate pivotal physiol. processes and they have proved to be readily druggable.  Natural products have been and continue to be amongst the most valuable sources for drug discovery and development.  Here, we surveyed small mols. and (poly-)peptides derived from plants, animals, fungi, and bacteria, which modulate GPCR signaling.  Among naturally occurring compds., peptides from plants, cone-snails, snakes, spiders, scorpions, fungi, and bacteria are of particular interest as lead compds. for the development of GPCR ligands, since they cover a chem. space, which differs from that of synthetic small mols.  Peptides, however, face challenges, some of which can be overcome by studying plant-derived compds.  We argue here that the opportunities outweigh the challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCENhph4se5bVg90H21EOLACvtfcHk0ljbnZ9AiPUNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVWnsr8%253D&md5=19bb90ea2b7e181a036f3ebc8668c419</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2019.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2019.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DMuratspahi%25C4%2587%26aufirst%3DE.%26aulast%3DFreissmuth%26aufirst%3DM.%26aulast%3DGruber%26aufirst%3DC.%2BW.%26atitle%3DNature-Derived%2520Peptides%253A%2520A%2520Growing%2520Niche%2520for%2520GPCR%2520Ligand%2520Discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2019%26volume%3D40%26spage%3D309%26epage%3D326%26doi%3D10.1016%2Fj.tips.2019.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span> <span> </span><span class="NLM_article-title">Function and Solution Structure of Huwentoxin-X, a Specific Blocker of N-Type Calcium Channels, from the Chinese Bird Spider <i>Ornithoctonus Huwena</i></span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">8628</span>– <span class="NLM_lpage">8635</span>, <span class="refDoi"> DOI: 10.1074/jbc.M513542200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.M513542200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=16439354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVCnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=8628-8635&author=Z.+Liuauthor=J.+Daiauthor=L.+Daiauthor=M.+Dengauthor=Z.+Huauthor=W.+Huauthor=S.+Liang&title=Function+and+Solution+Structure+of+Huwentoxin-X%2C+a+Specific+Blocker+of+N-Type+Calcium+Channels%2C+from+the+Chinese+Bird+Spider+Ornithoctonus+Huwena&doi=10.1074%2Fjbc.M513542200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Function and Solution Structure of Huwentoxin-X, a Specific Blocker of N-type Calcium Channels, from the Chinese Bird Spider Ornithoctonus huwena</span></div><div class="casAuthors">Liu, Zhonghua; Dai, Jie; Dai, Longjun; Deng, Meichun; Hu, Zhe; Hu, Weijun; Liang, Songping</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8628-8635</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Huwentoxin-X (HWTX-X) is a novel peptide toxin, purified from the venom of the spider Ornithoctonus huwena.  It comprises 28 amino acid residues including six cysteine residues as disulfide bridges linked in the pattern of I-IV, II-V, and III-VI.  Its cDNA, detd. by rapid amplification of 3' and 5' cDNA ends, encodes a 68-residue prepropeptide.  HWTX-X shares low sequence homol. with ω-conotoxins GVIA and MVIIA, two well known blockers of N-type Ca2+ channels.  Nevertheless, whole cell studies indicate that it can block N-type Ca2+ channels in rat dorsal root ganglion cells (IC50 40 nM) and the blockage by HWTX-X is completely reversible.  The rank order of specificity for N-type Ca2+ channels is GVIA ≈ HWTX-X > MVIIA.  In contrast to GVIA and MVIIA, HWTX-X had no detectable effect on the twitch response of rat vas deferens to low frequency elec. stimulation, indicating that HWTX-X has different selectivity for isoforms of N-type Ca2+ channels, compared with GVIA or MVIIA.  A comparison of the structures of HWTX-X and GVIA reveals that they not only adopt a common structural motif (inhibitor cystine knot), but also have a similar functional motif, a binding surface formed by the crit. residue Tyr, and several basic residues.  However, the dissimilarities of their binding surfaces provide some insights into their different selectivities for isoforms of N-type Ca2+ channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbvZ_5V2kK7Vg90H21EOLACvtfcHk0ljbnZ9AiPUNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVCnsrw%253D&md5=2fd8b3f315d5ee0a4602d8627eb0db93</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M513542200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M513542200%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DS.%26atitle%3DFunction%2520and%2520Solution%2520Structure%2520of%2520Huwentoxin-X%252C%2520a%2520Specific%2520Blocker%2520of%2520N-Type%2520Calcium%2520Channels%252C%2520from%2520the%2520Chinese%2520Bird%2520Spider%2520Ornithoctonus%2520Huwena%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D8628%26epage%3D8635%26doi%3D10.1074%2Fjbc.M513542200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamthanh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bournaud, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimahara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laboute, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letourneux, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molgó, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Gall, F.</span></span> <span> </span><span class="NLM_article-title">A New ω-Conotoxin That Targets N-Type Voltage-Sensitive Calcium Channels with Unusual Specificity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">14567</span>– <span class="NLM_lpage">14575</span>, <span class="refDoi"> DOI: 10.1021/bi002871r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi002871r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD3MXotV2ksrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=14567-14575&author=P.+Favreauauthor=N.+Gillesauthor=H.+Lamthanhauthor=R.+Bournaudauthor=T.+Shimaharaauthor=F.+Bouetauthor=P.+Labouteauthor=Y.+Letourneuxauthor=A.+M%C3%A9nezauthor=J.+Molg%C3%B3author=F.+Le+Gall&title=A+New+%CF%89-Conotoxin+That+Targets+N-Type+Voltage-Sensitive+Calcium+Channels+with+Unusual+Specificity&doi=10.1021%2Fbi002871r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A New ω-Conotoxin That Targets N-Type Voltage-Sensitive Calcium Channels with Unusual Specificity</span></div><div class="casAuthors">Favreau, Philippe; Gilles, Nicolas; Lamthanh, Hung; Bournaud, Roland; Shimahara, Takeshi; Bouet, Francoise; Laboute, Pierre; Letourneux, Yves; Menez, Andre; Molgo, Jordi; Le Gall, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">14567-14575</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new specific voltage-sensitive calcium channel (VSCC) blocker has been isolated from the venom of the fish-hunting cone snail Conus consors.  This peptide, named ω-Ctx CNVIIA, consists of 27 amino acid residues folded by 3 disulfide bridges.  Interestingly, loop 4, which is supposed to be crucial for selectivity, shows an unusual sequence (SSSKGR).  The synthesis of the linear peptide was performed using the Fmoc strategy, and the correct folding was achieved in the presence of guanidinium chloride, potassium buffer, and reduced/oxidized glutathione at 4° for 3 days.  Both synthetic and native toxin caused an intense shaking activity, characteristic of ω-conotoxins targeting N-type VSCC when injected intracerebroventricularly to mice.  Binding studies on rat brain synaptosomes revealed that the radioiodinated ω-Ctx CNVIIA specifically and reversibly binds to high-affinity sites with a Kd of 36.3 pM.  Its binding is competitive with ω-Ctx MVIIA at low concn. (Ki = 2 pM).  Moreover, ω-Ctx CNVIIA exhibits a clear selectivity for N-type VSCCs vs. P/Q-type VSCCs targeted resp. by radioiodinated ω-Ctx GVIA and ω-Ctx MVIIC.  Although ω-Ctx CNVIIA clearly blocked N-type Ca2+ current in chromaffin cells, this toxin did not inhibit acetylcholine release evoked by nerve stimuli at the frog neuromuscular junction, in marked contrast to ω-Ctx GVIA.  ω-Ctx CNVIIA thus represents a new selective tool for blocking N-type VSCC that displays a unique pharmacol. profile and highlights the diversity of voltage-sensitive Ca channels in the animal kingdom.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPGQafn7SVfLVg90H21EOLACvtfcHk0ljbnZ9AiPUNCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXotV2ksrs%253D&md5=45455a19156ab15af2db65c64834e2f8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi002871r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi002871r%26sid%3Dliteratum%253Aachs%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DLamthanh%26aufirst%3DH.%26aulast%3DBournaud%26aufirst%3DR.%26aulast%3DShimahara%26aufirst%3DT.%26aulast%3DBouet%26aufirst%3DF.%26aulast%3DLaboute%26aufirst%3DP.%26aulast%3DLetourneux%26aufirst%3DY.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DMolg%25C3%25B3%26aufirst%3DJ.%26aulast%3DLe%2BGall%26aufirst%3DF.%26atitle%3DA%2520New%2520%25CF%2589-Conotoxin%2520That%2520Targets%2520N-Type%2520Voltage-Sensitive%2520Calcium%2520Channels%2520with%2520Unusual%2520Specificity%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D14567%26epage%3D14575%26doi%3D10.1021%2Fbi002871r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis of Toxicity of N-Type Calcium Channel Inhibitor MVIIA</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2015.08.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.neuropharm.2015.08.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=26344359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGhtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=137-145&author=F.+Wangauthor=Z.+Yanauthor=Z.+Liuauthor=S.+Wangauthor=Q.+Wuauthor=S.+Yuauthor=J.+Dingauthor=Q.+Dai&title=Molecular+Basis+of+Toxicity+of+N-Type+Calcium+Channel+Inhibitor+MVIIA&doi=10.1016%2Fj.neuropharm.2015.08.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of toxicity of N-type calcium channel inhibitor MVIIA</span></div><div class="casAuthors">Wang, Fei; Yan, Zhenzhen; Liu, Zhuguo; Wang, Sheng; Wu, Qiaoling; Yu, Shuo; Ding, Jiuping; Dai, Qiuyun</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137-145</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MVIIA (ziconotide) is a specific inhibitor of N-type calcium channel, Cav2.2.  It is derived from Cone snail and currently used for the treatment of severe chronic pains in patients unresponsive to opioid therapy.  However, MVIIA produces severe side-effects, including dizziness, nystagmus, somnolence, abnormal gait, and ataxia, that limit its wider application.  We previously identified a novel inhibitor of Cav2.2, ω-conopeptide SO-3, which possesses similar structure and analgesic activity to MVIIA's.  To investigate the key residues for MVIIA toxicity, MVIIA/SO-3 hybrids and MVIIA variants carrying mutations in its loop 2 were synthesized.  The substitution of MVIIA's loop 1 with the loop 1 of SO-3 resulted in significantly reduced Cav2.2 binding activity in vitro; the replacement of MVIIA loop 2 by the loop 2 of SO-3 not only enhanced the peptide/Cav2.2 binding but also decreased its toxicity on goldfish, attenuated mouse tremor symptom, spontaneous locomotor activity, and coordinated locomotion function.  Further mutation anal. and mol. calcn. revealed that the toxicity of MVIIA mainly arose from Met12 in the loop 2, and this residue inserts into a hydrophobic hole (Ile300, Phe302 and Leu305) located between repeats II and III of Cav2.2.  The combinative mutations of the loop 2 of MVIIA or other ω-conopeptides may be used for future development of more effective Cav2.2 inhibitors with lower side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn3CuU5x-TB7Vg90H21EOLACvtfcHk0lglvJmXrCg8ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGhtbrJ&md5=0123de2b1d8af2ddd90b977940f12f88</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2015.08.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2015.08.047%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DQ.%26atitle%3DMolecular%2520Basis%2520of%2520Toxicity%2520of%2520N-Type%2520Calcium%2520Channel%2520Inhibitor%2520MVIIA%26jtitle%3DNeuropharmacology%26date%3D2016%26volume%3D101%26spage%3D137%26epage%3D145%26doi%3D10.1016%2Fj.neuropharm.2015.08.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huys, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyason, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waelkens, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Zyl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">du Plessis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Walt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clynen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoofs, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span> <span> </span><span class="NLM_article-title">Purification, Characterization and Biosynthesis of Parabutoxin 3, a Component of <i>Parabuthus Transvaalicus</i> Venom</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">1854</span>– <span class="NLM_lpage">1865</span>, <span class="refDoi"> DOI: 10.1046/j.1432-1033.2002.02833.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1046%2Fj.1432-1033.2002.02833.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=11952787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtVOns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=2002&pages=1854-1865&author=I.+Huysauthor=K.+Dyasonauthor=E.+Waelkensauthor=F.+Verdonckauthor=J.+van+Zylauthor=J.+du+Plessisauthor=G.+J.+M%C3%BCllerauthor=J.+van+der+Waltauthor=E.+Clynenauthor=L.+Schoofsauthor=J.+Tytgat&title=Purification%2C+Characterization+and+Biosynthesis+of+Parabutoxin+3%2C+a+Component+of+Parabuthus+Transvaalicus+Venom&doi=10.1046%2Fj.1432-1033.2002.02833.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Purification, characterization and biosynthesis of parabutoxin 3, a component of Parabuthus transvaalicus venom</span></div><div class="casAuthors">Huys, Isabelle; Dyason, Karin; Waelkens, Etienne; Verdonck, Fons; Van Zyl, Johann; du Plessis, Johan; Muller, Gert J.; Van der Walt, Jurg; Clynen, Elke; Schoofs, Liliane; Tytgat, Jan</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1854-1865</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A novel peptidyl inhibitor of voltage-gated K+ channels, named parabutoxin 3 (PBTx3), has been purified to homogeneity from the venom of Parabuthus transvaalicus.  This scorpion toxin contains 37 residues, has a mass of 4274 Da and displays 41% identity with charybdotoxin (ChTx, also called "α-KTx1.1").  PBTx3 is the tenth member (called "α-KTx1.10") of subfamily 1 of K+ channel-blocking peptides known thus far.  Electrophysiol. expts. using Xenopus laevis oocytes indicate that PBTx3 is an inhibitor of Kv1 channels (Kv1.1, Kv1.2, Kv1.3), but has no detectable effects on Kir-type and ERG-type channels.  The dissocn. consts. (Kd) for Kv1.1, Kv1.2 and Kv1.3 channels are, resp., 79 μM, 547 nM and 492 nM.  A synthetic gene encoding a PBTx3 homolog was designed and expressed as a fusion protein with the maltose-binding protein (MBP) in Escherichia coli.  The recombinant protein was purified from the bacterial periplasm compartment using an amylose affinity resin column, followed by a gel filtration purifn. step and cleavage by factor Xa (fXa) to release the recombinant toxin peptide (rPBTx3).  After final purifn. and refolding, rPBTx3 was shown to be identical to the native PBTx3 with respect to HPLC retention time, mass spectrometric anal. and functional properties.  The three-dimensional structure of PBTx3 is proposed by homol. modeling to contain a double-stranded antiparallel β sheet and a single α-helix, connected by three disulfide bridges.  The scaffold of PBTx3 is homologous to most other α-KTx scorpion toxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUvdZ9pBYMmrVg90H21EOLACvtfcHk0lglvJmXrCg8ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtVOns7o%253D&md5=f6a41f19d6dec99566b01173369fc386</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1033.2002.02833.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1033.2002.02833.x%26sid%3Dliteratum%253Aachs%26aulast%3DHuys%26aufirst%3DI.%26aulast%3DDyason%26aufirst%3DK.%26aulast%3DWaelkens%26aufirst%3DE.%26aulast%3DVerdonck%26aufirst%3DF.%26aulast%3Dvan%2BZyl%26aufirst%3DJ.%26aulast%3Ddu%2BPlessis%26aufirst%3DJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DG.%2BJ.%26aulast%3Dvan%2Bder%2BWalt%26aufirst%3DJ.%26aulast%3DClynen%26aufirst%3DE.%26aulast%3DSchoofs%26aufirst%3DL.%26aulast%3DTytgat%26aufirst%3DJ.%26atitle%3DPurification%252C%2520Characterization%2520and%2520Biosynthesis%2520of%2520Parabutoxin%25203%252C%2520a%2520Component%2520of%2520Parabuthus%2520Transvaalicus%2520Venom%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D2002%26volume%3D269%26spage%3D1854%26epage%3D1865%26doi%3D10.1046%2Fj.1432-1033.2002.02833.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varga, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herceg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Román-González, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panyi, G.</span></span> <span> </span><span class="NLM_article-title">Tst26, a Novel Peptide Blocker of Kv1.2 and Kv1.3 Channels from the Venom of <i>Tityus Stigmurus</i></span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2009.05.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2009.05.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19500613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1OktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2009&pages=379-389&author=F.+Pappauthor=C.+V.+F.+Batistaauthor=Z.+Vargaauthor=M.+Hercegauthor=S.+A.+Rom%C3%A1n-Gonz%C3%A1lezauthor=R.+Gasparauthor=L.+D.+Possaniauthor=G.+Panyi&title=Tst26%2C+a+Novel+Peptide+Blocker+of+Kv1.2+and+Kv1.3+Channels+from+the+Venom+of+Tityus+Stigmurus&doi=10.1016%2Fj.toxicon.2009.05.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Tst26, a novel peptide blocker of Kv1.2 and Kv1.3 channels from the venom of Tityus stigmurus</span></div><div class="casAuthors">Papp, Ferenc; Batista, Cesar V. F.; Varga, Zoltan; Herceg, Monika; Roman-Gonzalez, Sergio A.; Gaspar, Rezso; Possani, Lourival D.; Panyi, Gyorgy</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-389</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Using high-performance liq. chromatog. Tst26, a novel potassium channel blocker peptide, was purified from the venom of the Brazilian scorpion Tityus stigmurus.  Its primary structure was detd. by means of automatic Edman degrdn. and mass spectrometry anal.  The peptide is composed of 37 amino acid residues and tightly folded through three disulfide bridges, similar to other K+ channel blocking peptides purified from scorpion venoms.  It contains the "essential dyad" for K+ channel recognition comprised of a lysine at position 27 and a tyrosine at position 36.  Electrophysiol. assays revealed that Tst26 blocked hKv1.2 and hKv1.3 channels with high affinity (Kd = 1.9 nM and 10.7 nM, resp.) while it did not affect several other ion channels (mKv1.1, hKv1.4, hKv1.5, hERG, hIKCa1, hBK, hNav1.5) tested at 10 nM concn.  The voltage-dependent steady-state parameters of K+ channel gating were unaffected by the toxin in both channels, but due to the fast assocn. and dissocn. kinetics Tst26 slowed the rate of inactivation of Kv1.3 channels.  Based on the primary structure, the systematic nomenclature proposed for this peptide is α-KTx 4.6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5q0vBfePfq7Vg90H21EOLACvtfcHk0lglvJmXrCg8ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1OktLg%253D&md5=3c87f27791af485ddeed6a2825f3b04f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2009.05.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2009.05.023%26sid%3Dliteratum%253Aachs%26aulast%3DPapp%26aufirst%3DF.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DVarga%26aufirst%3DZ.%26aulast%3DHerceg%26aufirst%3DM.%26aulast%3DRom%25C3%25A1n-Gonz%25C3%25A1lez%26aufirst%3DS.%2BA.%26aulast%3DGaspar%26aufirst%3DR.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26aulast%3DPanyi%26aufirst%3DG.%26atitle%3DTst26%252C%2520a%2520Novel%2520Peptide%2520Blocker%2520of%2520Kv1.2%2520and%2520Kv1.3%2520Channels%2520from%2520the%2520Venom%2520of%2520Tityus%2520Stigmurus%26jtitle%3DToxicon%26date%3D2009%26volume%3D54%26spage%3D379%26epage%3D389%26doi%3D10.1016%2Fj.toxicon.2009.05.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogowski, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, M. P.</span></span> <span> </span><span class="NLM_article-title">Tityustoxin K Alpha Blocks Voltage-Gated Noninactivating K+ Channels and Unblocks Inactivating K+ Channels Blocked by Alpha-Dendrotoxin in Synaptosomes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1479</span>, <span class="refDoi"> DOI: 10.1073/pnas.91.4.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.91.4.1475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=7509073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2cXis1entro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=1475-1479&author=R.+S.+Rogowskiauthor=B.+K.+Kruegerauthor=J.+H.+Collinsauthor=M.+P.+Blaustein&title=Tityustoxin+K+Alpha+Blocks+Voltage-Gated+Noninactivating+K%2B+Channels+and+Unblocks+Inactivating+K%2B+Channels+Blocked+by+Alpha-Dendrotoxin+in+Synaptosomes&doi=10.1073%2Fpnas.91.4.1475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Tityustoxin Kα blocks voltage-gated noninactivating K+ channels and unblocks inactivating K+ channels blocked by α-dendrotoxin in synaptosomes</span></div><div class="casAuthors">Rogowski, Robert S.; Krueger, Bruce K.; Collins, John H.; Blaustein, Mordecai P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1475-9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Two nonhomologs polypeptide toxins, tityustoxin Kα (TsTX-Kα) and tityustoxin Kβ (TsTX-Kβ), purified from the venom of the Brazilian scorpion Tityus serrulatus, selectively block voltage-gated noninactivating K+ channels in synaptosomes (IC50 values of 8 nM and 30 nM, resp.).  In contrast, α-dendrotoxin (α-DTX) and charybdotoxin (ChTX) block voltage-gated inactivating K+ channels in synaptosomes (IC50 values of 90 nM and 40 nM, resp.).  The authors studied interactions among these toxins in 125I-α-DTX binding and 86Rb efflux expts.  Both TsTX-Kα and ChTX completely displaced specifically bound 125I-α-DTX from synaptic membranes, but TsTX-Kβ had no effect on bound α-DTX.  TsTX-Kα and TsTX-Kβ blocked the same noninactivating component of 100 mM K+-stimulated 86Rb efflux in synaptosomes.  Both α-DTX and ChTX blocked the same inactivating component of the K+-stimulated 86Rb efflux in synaptosomes.  Both the inactivating and the noninactivating components of the 100 mM K+-stimulated 86Rb efflux were completely blocked when 200 mM TsTX-Kβ and either 600 nM α-DTX or 200 nM ChTX were present.  The effects of TsTX-Kα was added in the presence of α-DTX, however, only the noninactivating component of the K+-stimulated efflux was blocked.  The inactivating component could then be blocked by ChTX, which is structurally homologs to TsTX-Kα.  The authors conclude that TsTX-Kα unblocks the voltage-gated inactivating K+ channels in synaptosomes when they are blocked by α-DTX, but not when they are blocked by ChTX.  TsTX-Kα binds to a site on the inactivating K+ channel that does not occlude the pore; its binding apparently prevents α-DTX (7054 Da), but not ChTX (4300 Da), from blocking the pore.  The effects of TsTX-Kα on 125I-α-DTX binding and 86Rb efflux are mimicked by noxiustoxin, which is homologs to TsTX-Kα and ChTX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOAe4ArbxDzLVg90H21EOLACvtfcHk0liuxvJoKqiaIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXis1entro%253D&md5=2885b62e5cdb7d1fdd9e82ff354b2fd1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.4.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.4.1475%26sid%3Dliteratum%253Aachs%26aulast%3DRogowski%26aufirst%3DR.%2BS.%26aulast%3DKrueger%26aufirst%3DB.%2BK.%26aulast%3DCollins%26aufirst%3DJ.%2BH.%26aulast%3DBlaustein%26aufirst%3DM.%2BP.%26atitle%3DTityustoxin%2520K%2520Alpha%2520Blocks%2520Voltage-Gated%2520Noninactivating%2520K%252B%2520Channels%2520and%2520Unblocks%2520Inactivating%2520K%252B%2520Channels%2520Blocked%2520by%2520Alpha-Dendrotoxin%2520in%2520Synaptosomes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1994%26volume%3D91%26spage%3D1475%26epage%3D1479%26doi%3D10.1073%2Fpnas.91.4.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerni, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucca, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cremonez, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordon, K. C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arantes, E. C.</span></span> <span> </span><span class="NLM_article-title">Electrophysiological Characterization of Ts6 and Ts7, K+ Channel Toxins Isolated through an Improved Tityus Serrulatus Venom Purification Procedure</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.3390/toxins6030892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Ftoxins6030892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=24590385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=892-913&author=F.+A.+Cerniauthor=M.+B.+Puccaauthor=S.+Peigneurauthor=C.+M.+Cremonezauthor=K.+C.+F.+Bordonauthor=J.+Tytgatauthor=E.+C.+Arantes&title=Electrophysiological+Characterization+of+Ts6+and+Ts7%2C+K%2B+Channel+Toxins+Isolated+through+an+Improved+Tityus+Serrulatus+Venom+Purification+Procedure&doi=10.3390%2Ftoxins6030892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophysiological characterization of Ts6 and Ts7, K+ channel toxins isolated through an improved Tityus serrulatus venom purification procedure</span></div><div class="casAuthors">Cerni, Felipe A.; Pucca, Manuela B.; Peigneur, Steve; Cremonez, Caroline M.; Bordon, Karla C. F.; Tytgat, Jan; Arantes, Eliane C.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-913, 22 pp.</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In Brazil, Tityus serrulatus (Ts) is the species responsible for most of the scorpion related accidents.  Among the Ts toxins, the neurotoxins with action on potassium channels (α-KTx) present high interest, due to their effect in the envenoming process and the ion channel specificity they display.  The α-KTx toxins family is the most relevant because its toxins can be used as therapeutic tools for specific target cells.  The improved isolation method provided toxins with high resoln., obtaining pure Ts6 and Ts7 in two chromatog. steps.  The effects of Ts6 and Ts7 toxins were evaluated in 14 different types of potassium channels using the voltage-clamp technique with two-microelectrodes.  Ts6 toxin shows high affinity for Kv1.2, Kv1.3 and Shaker IR, blocking these channels in low concns.  Moreover, Ts6 blocks the Kv1.3 channel in picomolar concns. with an IC50 of 0.55 nM and therefore could be of valuable assistance to further designing immunosuppressive therapeutics.  Ts7 toxin blocks multiple subtypes channels, showing low selectivity among the channels analyzed.  This work also stands out in its attempt to elucidate the residues important for interacting with each channel and, in the near future, to model a desired drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbFM-3lmyazbVg90H21EOLACvtfcHk0liuxvJoKqiaIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7Y%253D&md5=ee62c4fee705205b8dbc24131693e39d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.3390%2Ftoxins6030892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins6030892%26sid%3Dliteratum%253Aachs%26aulast%3DCerni%26aufirst%3DF.%2BA.%26aulast%3DPucca%26aufirst%3DM.%2BB.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DCremonez%26aufirst%3DC.%2BM.%26aulast%3DBordon%26aufirst%3DK.%2BC.%2BF.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DArantes%26aufirst%3DE.%2BC.%26atitle%3DElectrophysiological%2520Characterization%2520of%2520Ts6%2520and%2520Ts7%252C%2520K%252B%2520Channel%2520Toxins%2520Isolated%2520through%2520an%2520Improved%2520Tityus%2520Serrulatus%2520Venom%2520Purification%2520Procedure%26jtitle%3DToxins%26date%3D2014%26volume%3D6%26spage%3D892%26epage%3D913%26doi%3D10.3390%2Ftoxins6030892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, A. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arantes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monje, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stühmer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varanda, W. A.</span></span> <span> </span><span class="NLM_article-title">Tityustoxin-K(Alpha) Blockade of the Voltage-Gated Potassium Channel Kv1.3</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">1180</span>– <span class="NLM_lpage">1186</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0705343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1038%2Fsj.bjp.0705343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=12871837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmtFemtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2003&pages=1180-1186&author=A.+R.+A.+Rodriguesauthor=E.+C.+Arantesauthor=F.+Monjeauthor=W.+St%C3%BChmerauthor=W.+A.+Varanda&title=Tityustoxin-K%28Alpha%29+Blockade+of+the+Voltage-Gated+Potassium+Channel+Kv1.3&doi=10.1038%2Fsj.bjp.0705343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Tityustoxin-K(alpha) blockade of the voltage-gated potassium channel Kv1.3</span></div><div class="casAuthors">Rodrigues, Aldo Rogelis A.; Arantes, Eliane C.; Monje, Francisco; Stuehmer, Walter; Varanda, Wamberto Antonio</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1180-1186</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We investigated the action of TsTX-Kα on cloned Kv1.3 channels of the Shaker subfamily of voltage-gated potassium channels, using the voltage-clamp technique.  Highly purified TsTX-Kα was obtained from the venom of the Brazilian scorpion Tityus serrulatus using a new purifn. protocol.  Our results show that TsTX-Kα blocks Kv1.3 with high affinity in two expression systems.  TsTX-Kα blockade of Kv1.3 channels expressed in Xenopus oocytes was found to be completely reversible and to exhibit a pH dependence.  The KD was 3.9 nM at pH 7.5, 9.5 nM at pH 7.0 and 94.5 nM at pH 6.5.  The blocking properties of TsTX-Kα in a mammalian cell line (L929), stably transfected to express Kv1.3, were studied using the patch-clamp technique.  In this prepn., the toxin had a KD of 19.8 nM at pH 7.4.  TsTX-Kα was found to affect neither the voltage-dependence of activation, nor the activation and deactivation time consts.  The block appeared to be independent of the transmembrane voltage and the toxin did not interfere with the C-type inactivation process.  Taken as a whole, our findings indicate that TsTX-Kα acts as a simple blocker of Kv1.3 channels.  It is concluded that this toxin is a useful tool for probing not only the physiol. roles of Kv1.2, but also those mediated by Kv1.3 channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB96JwGIv6VLVg90H21EOLACvtfcHk0liuxvJoKqiaIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmtFemtLo%253D&md5=1d5582adac27db363d214a7be30be1eb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705343%26sid%3Dliteratum%253Aachs%26aulast%3DRodrigues%26aufirst%3DA.%2BR.%2BA.%26aulast%3DArantes%26aufirst%3DE.%2BC.%26aulast%3DMonje%26aufirst%3DF.%26aulast%3DSt%25C3%25BChmer%26aufirst%3DW.%26aulast%3DVaranda%26aufirst%3DW.%2BA.%26atitle%3DTityustoxin-K%2528Alpha%2529%2520Blockade%2520of%2520the%2520Voltage-Gated%2520Potassium%2520Channel%2520Kv1.3%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D139%26spage%3D1180%26epage%3D1186%26doi%3D10.1038%2Fsj.bjp.0705343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diego-García, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Gómez, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Candido, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Lagunas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">The Brazilian Scorpion <i>Tityus Costatus</i> Karsch: Genes, Peptides and Function</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2004.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2004.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=15683865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpt1altQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=273-283&author=E.+Diego-Garc%C3%ADaauthor=C.+V.+F.+Batistaauthor=B.+I.+Garc%C3%ADa-G%C3%B3mezauthor=S.+Lucasauthor=D.+M.+Candidoauthor=F.+G%C3%B3mez-Lagunasauthor=L.+D.+Possani&title=The+Brazilian+Scorpion+Tityus+Costatus+Karsch%3A+Genes%2C+Peptides+and+Function&doi=10.1016%2Fj.toxicon.2004.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Brazilian scorpion Tityus costatus Karsch: genes, peptides and function</span></div><div class="casAuthors">Diego-Garcia, Elia; Batista, Cesar V. F.; Garcia-Gomez, Blanca I.; Lucas, Sylvia; Candido, Denise M.; Gomez-Lagunas, Froylan; Possani, Lourival D.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">273-283</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The venom of the scorpion Tityus costatus contains peptides toxic to humans but scarce information on their structure and function is available.  Here, we report the sepn. of 50 different components by high performance liq. chromatog. and the identification of approx. 90 distinct components by mass spectrometry anal., with mol. wts. varying from 413 to 45482 amu.  Four peptides were fully sequenced: (i) a butantoxin-like peptide that blocks Shaker K+ channel; (ii) an insect toxin-like peptide; (iii) a scorpine-like peptide, and a short heptapeptide of unknown function.  Fifteen peptides were directly sequenced at the N-terminal region, among which are components toxic to mice.  A cDNA library was constructed and 13 clones were isolated and sequenced.  Some of these peptides and genes are similar to other known scorpion toxins.  Based on these results, stings by scorpions of the species Tityus costatus should be taken with caution by medical doctors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTT1RQ9IGqprVg90H21EOLACvtfcHk0liKhM952cVR8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpt1altQ%253D%253D&md5=dcc648d2e72dcffa0a4444de103c718e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2004.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2004.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DDiego-Garc%25C3%25ADa%26aufirst%3DE.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DGarc%25C3%25ADa-G%25C3%25B3mez%26aufirst%3DB.%2BI.%26aulast%3DLucas%26aufirst%3DS.%26aulast%3DCandido%26aufirst%3DD.%2BM.%26aulast%3DG%25C3%25B3mez-Lagunas%26aufirst%3DF.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DThe%2520Brazilian%2520Scorpion%2520Tityus%2520Costatus%2520Karsch%253A%2520Genes%252C%2520Peptides%2520and%2520Function%26jtitle%3DToxicon%26date%3D2005%26volume%3D45%26spage%3D273%26epage%3D283%26doi%3D10.1016%2Fj.toxicon.2004.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Calvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKinnon, R.</span></span> <span> </span><span class="NLM_article-title">Purification and Characterization of Three Inhibitors of Voltage-Dependent K+ Channels from Leiurus Quinquestriatus Var. Hebraeus Venom</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">6834</span>– <span class="NLM_lpage">6839</span>, <span class="refDoi"> DOI: 10.1021/bi00188a012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00188a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2cXktVeju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=6834-6839&author=M.+L.+Garciaauthor=M.+Garcia-Calvoauthor=P.+Hidalgoauthor=A.+Leeauthor=R.+MacKinnon&title=Purification+and+Characterization+of+Three+Inhibitors+of+Voltage-Dependent+K%2B+Channels+from+Leiurus+Quinquestriatus+Var.+Hebraeus+Venom&doi=10.1021%2Fbi00188a012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and Characterization of Three Inhibitors of Voltage-Dependent K+ Channels from Leiurus Quinquestriatus var. Hebraeus Venom</span></div><div class="casAuthors">Garcia, Maria L.; Garcia-Calvo, Margarita; Hidalgo, Patricia; Lee, Alice; MacKinnon, Roderick</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6834-9</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Three new toxins from the venom of the scorpion Leiurus quinquestriatus var. hebraeus have been identified on the basis of their ability to block the Shaker K+ channel.  These toxins have been purified using HPLC techniques and characterized as 38 amino acid peptides by mass spectroscopy, amino acid anal., and sequence detn.  Their chem. identity was confirmed by producing fully functional synthetic toxins using recombinant methods.  These peptides are potent inhibitors of the Shaker K+ channel (Kd < 1 nM) as well as the mammalian homologues of Shaker.  They are related to other previously described K+ channel toxins, but form a new subclass within the larger family of K+ channel inhibitors derived from scorpion venom.  The authors have named these toxins agitoxin 1, 2, and 3, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3vGX6FDTOfrVg90H21EOLACvtfcHk0liKhM952cVR8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVeju74%253D&md5=ceb54a549da2b1b1b2b30a56a0cd78d1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi00188a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00188a012%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DM.%2BL.%26aulast%3DGarcia-Calvo%26aufirst%3DM.%26aulast%3DHidalgo%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DMacKinnon%26aufirst%3DR.%26atitle%3DPurification%2520and%2520Characterization%2520of%2520Three%2520Inhibitors%2520of%2520Voltage-Dependent%2520K%252B%2520Channels%2520from%2520Leiurus%2520Quinquestriatus%2520Var.%2520Hebraeus%2520Venom%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26spage%3D6834%26epage%3D6839%26doi%3D10.1021%2Fbi00188a012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takacs, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toups, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollewe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuello, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driessens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biancalana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponte, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perozo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajewski, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Kurtz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, S. A. N.</span></span> <span> </span><span class="NLM_article-title">A Designer Ligand Specific for Kv1.3 Channels from a Scorpion Neurotoxin-Based Library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">22211</span>– <span class="NLM_lpage">22216</span>, <span class="refDoi"> DOI: 10.1073/pnas.0910123106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.0910123106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20007782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtlWqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=22211-22216&author=Z.+Takacsauthor=M.+Toupsauthor=A.+Kolleweauthor=E.+Johnsonauthor=L.+G.+Cuelloauthor=G.+Driessensauthor=M.+Biancalanaauthor=A.+Koideauthor=C.+G.+Ponteauthor=E.+Perozoauthor=T.+F.+Gajewskiauthor=G.+Suarez-Kurtzauthor=S.+Koideauthor=S.+A.+N.+Goldstein&title=A+Designer+Ligand+Specific+for+Kv1.3+Channels+from+a+Scorpion+Neurotoxin-Based+Library&doi=10.1073%2Fpnas.0910123106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A designer ligand specific for Kv1.3 channels from a scorpion neurotoxin-based library</span></div><div class="casAuthors">Takacs, Zoltan; Toups, Megan; Kollewe, Astrid; Johnson, Erik; Cuello, Luis G.; Driessens, Gregory; Biancalana, Matthew; Koide, Akiko; Ponte, Cristiano G.; Perozo, Eduardo; Gajewski, Thomas F.; Suarez-Kurtz, Guilherme; Koide, Shohei; Goldstein, Steve A. N.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">22211-22216, S22211/1-S22211/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Venomous animals immobilize prey using protein toxins that act on ion channels and other targets of biol. importance.  Broad use of toxins for biomedical research, diagnosis, and therapy has been limited by inadequate target discrimination, for example, among ion channel subtypes.  Here, a synthetic toxin is produced by a new strategy to be specific for human Kv1.3 channels, crit. regulators of immune T cells.  A phage display library of 11,200 de novo proteins is designed using the α-KTx scaffold of 31 scorpion toxin sequences known or predicted to bind to potassium channels.  Mokatoxin-1 (moka1) is isolated by affinity selection on purified target.  Moka1 blocks Kv1.3 at nanomolar levels that do not inhibit Kv1.1, Kv1.2, or KCa1.1.  As a result, moka1 suppresses CD3/28-induced cytokine secretion by T cells without cross-reactive gastrointestinal hyperactivity.  Mokatoxin-1 (moka1) is isolated by affinity selection on purified target.  The 3D structure of moka1 rationalizes its specificity and validates the engineering approach, revealing a unique interaction surface supported on an α-KTx scaffold.  This scaffold-based/target-biased strategy overcomes many obstacles to prodn. of selective toxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosX_RmtjhXh7Vg90H21EOLACvtfcHk0liKhM952cVR8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtlWqsw%253D%253D&md5=da8f9ebf9282d762e30a912cd74592f2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0910123106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0910123106%26sid%3Dliteratum%253Aachs%26aulast%3DTakacs%26aufirst%3DZ.%26aulast%3DToups%26aufirst%3DM.%26aulast%3DKollewe%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DCuello%26aufirst%3DL.%2BG.%26aulast%3DDriessens%26aufirst%3DG.%26aulast%3DBiancalana%26aufirst%3DM.%26aulast%3DKoide%26aufirst%3DA.%26aulast%3DPonte%26aufirst%3DC.%2BG.%26aulast%3DPerozo%26aufirst%3DE.%26aulast%3DGajewski%26aufirst%3DT.%2BF.%26aulast%3DSuarez-Kurtz%26aufirst%3DG.%26aulast%3DKoide%26aufirst%3DS.%26aulast%3DGoldstein%26aufirst%3DS.%2BA.%2BN.%26atitle%3DA%2520Designer%2520Ligand%2520Specific%2520for%2520Kv1.3%2520Channels%2520from%2520a%2520Scorpion%2520Neurotoxin-Based%2520Library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D22211%26epage%3D22216%26doi%3D10.1073%2Fpnas.0910123106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmenkov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grishin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilevski, A. A.</span></span> <span> </span><span class="NLM_article-title">Diversity of Potassium Channel Ligands: Focus on Scorpion Toxins</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">1764</span>– <span class="NLM_lpage">1799</span>, <span class="refDoi"> DOI: 10.1134/s0006297915130118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1134%2Fs0006297915130118" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2016&pages=1764-1799&author=A.+I.+Kuzmenkovauthor=E.+V.+Grishinauthor=A.+A.+Vassilevski&title=Diversity+of+Potassium+Channel+Ligands%3A+Focus+on+Scorpion+Toxins&doi=10.1134%2Fs0006297915130118"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1134%2Fs0006297915130118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1134%252Fs0006297915130118%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmenkov%26aufirst%3DA.%2BI.%26aulast%3DGrishin%26aufirst%3DE.%2BV.%26aulast%3DVassilevski%26aufirst%3DA.%2BA.%26atitle%3DDiversity%2520of%2520Potassium%2520Channel%2520Ligands%253A%2520Focus%2520on%2520Scorpion%2520Toxins%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D80%26spage%3D1764%26epage%3D1799%26doi%3D10.1134%2Fs0006297915130118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Oliveira Carvalho, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, V. M.</span></span> <span> </span><span class="NLM_article-title">Plant Defensins--Prospects for the Biological Functions and Biotechnological Properties</span>. <i>Peptides</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1020</span>, <span class="refDoi"> DOI: 10.1016/j.peptides.2009.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.peptides.2009.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19428780" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1007-1020&author=A.+de%0AOliveira+Carvalhoauthor=V.+M.+Gomes&title=Plant+Defensins%2D%2DProspects+for+the+Biological+Functions+and+Biotechnological+Properties&doi=10.1016%2Fj.peptides.2009.01.018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.peptides.2009.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.peptides.2009.01.018%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BOliveira%2BCarvalho%26aufirst%3DA.%26aulast%3DGomes%26aufirst%3DV.%2BM.%26atitle%3DPlant%2520Defensins--Prospects%2520for%2520the%2520Biological%2520Functions%2520and%2520Biotechnological%2520Properties%26jtitle%3DPeptides%26date%3D2009%26volume%3D30%26spage%3D1007%26epage%3D1020%26doi%3D10.1016%2Fj.peptides.2009.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koehbach, J.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships of Insect Defensins</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">45</span>, <span class="refDoi"> DOI: 10.3389/fchem.2017.00045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3389%2Ffchem.2017.00045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28748179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlWksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=45&author=J.+Koehbach&title=Structure-Activity+Relationships+of+Insect+Defensins&doi=10.3389%2Ffchem.2017.00045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of insect defensins</span></div><div class="casAuthors">Koehbach, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45/1-45/10</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Insects make up the largest and most diverse group of organisms on earth with several million species to exist in total.  Considering the sheer no. of insect species and the vast variety of ways they interact with their environment through chem., it is clear that they have significant potential as a source of new lead mols.  They have adapted to a range of ecol. habitats and exhibit a symbiotic lifestyle with various microbes such as bacteria and fungi.  Accordingly, numerous antimicrobial compds. have been identified including for example defensin peptides.  Insect defensins were found to have broad-spectrum activity against various gram-pos./neg. bacteria as well as fungi.  They exhibit a unique structural topol. involving the complex arrangement of three disulfide bonds as well as an alpha helix and beta sheets, which is known as cysteine-stabilized αβ motif.  Their stability and amenability to peptide engineering make them promising candidates for the development of novel antibiotics lead mols.  This review highlights the current knowledge regarding the structure-activity relationships of insect defensin peptides and provides basis for future studies focusing on the rational design of novel cysteine-rich antimicrobial peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBzN1x06PTOrVg90H21EOLACvtfcHk0lhNnDqqArZTNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlWksbg%253D&md5=a1e4ffffc6013c64cdcf93db611621bf</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2017.00045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2017.00045%26sid%3Dliteratum%253Aachs%26aulast%3DKoehbach%26aufirst%3DJ.%26atitle%3DStructure-Activity%2520Relationships%2520of%2520Insect%2520Defensins%26jtitle%3DFront.%2520Chem.%26date%3D2017%26volume%3D5%26spage%3D45%26doi%3D10.3389%2Ffchem.2017.00045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonmatin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hetru, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vovelle, F.</span></span> <span> </span><span class="NLM_article-title">Refined Three-Dimensional Solution Structure of Insect Defensin A</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">448</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(01)00177-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0969-2126%2801%2900177-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=7663941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2MXlvFyhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1995&pages=435-448&author=B.+Cornetauthor=J.+M.+Bonmatinauthor=C.+Hetruauthor=J.+A.+Hoffmannauthor=M.+Ptakauthor=F.+Vovelle&title=Refined+Three-Dimensional+Solution+Structure+of+Insect+Defensin+A&doi=10.1016%2FS0969-2126%2801%2900177-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Refined three-dimensional solution structure of insect defensin A</span></div><div class="casAuthors">Cornet, Bruno; Bonmatin, Jena-Marc; Hetru, Charles; Hoffmann, Jules A.; Ptak, Marius; Volvelle, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">435-48</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">Insect defensin A is a basic 4 kDa protein secreted by Phormia terranovae larvae in response to bacteria challenges or injuries.  Previous biol. test suggest that the bacterial cytoplasmic membrane is the target of defensin A.  The structural study of this protein in the first step towards establishing a structure-activity relation and forms the basis for understanding its antibiotic activity at the mol. level.  The authors describe a refined model of the three-dimensional structure of defensin A derived from an extensive anal. of 786 inter-proton nuclear Overhauser effects.  The backbone fold involves an N-terminal loop and an α-helical fragment followed by an antiparallel β-structure.  The helix and β-structure are connected by two of the three disulfide bridges present in defensin A, forming a so-called 'cysteine-stabilized αβ' (CSαβ) motif.  The N-terminal loop, which forms the core of the defensin A structure, appears to be a common organization for several families of small proteins with toxic properties.  The distribution of amino acid side chain in the proteins structure creates several hydrophobic or hydrophilic patches.  This lead the authors' to propose that the initial step in the action of pos. charged defensin A mols. with cytoplasmic membranes may involve interaction with acidic phospholipids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPmfGsl4P16LVg90H21EOLACvtfcHk0lhNnDqqArZTNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlvFyhsLo%253D&md5=b79764c3487806f2785f1ee3f51da4d3</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2801%2900177-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252801%252900177-0%26sid%3Dliteratum%253Aachs%26aulast%3DCornet%26aufirst%3DB.%26aulast%3DBonmatin%26aufirst%3DJ.%2BM.%26aulast%3DHetru%26aufirst%3DC.%26aulast%3DHoffmann%26aufirst%3DJ.%2BA.%26aulast%3DPtak%26aufirst%3DM.%26aulast%3DVovelle%26aufirst%3DF.%26atitle%3DRefined%2520Three-Dimensional%2520Solution%2520Structure%2520of%2520Insect%2520Defensin%2520A%26jtitle%3DStructure%26date%3D1995%26volume%3D3%26spage%3D435%26epage%3D448%26doi%3D10.1016%2FS0969-2126%2801%2900177-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Synergy and Antagonism of Promiscuous Inhibition in Multiple-Compound Mixtures</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2151</span>– <span class="NLM_lpage">2154</span>, <span class="refDoi"> DOI: 10.1021/jm060029z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060029z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVygtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2151-2154&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=Synergy+and+Antagonism+of+Promiscuous+Inhibition+in+Multiple-Compound+Mixtures&doi=10.1021%2Fjm060029z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy and Antagonism of Promiscuous Inhibition in Multiple-Compound Mixtures</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2151-2154</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening in mixts. is a common approach for increasing the efficiency of high-throughput screening.  Here we investigate how the "compd. load" of mixts. influences promiscuous aggregate-based inhibition.  We screened 764 mols. individually and in mixts. of 10 at 5 μM each, comparing the obsd. inhibition of the mixts. to that predicted from single-compd. results.  Synergistic effects on aggregation predominated, although antagonism was also obsd.  These results suggest that screening mixts. can increase aggregation-based inhibition in a nonadditive manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5WCPXJ_ed2bVg90H21EOLACvtfcHk0lgxvINPomVjOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitVygtLs%253D&md5=b62e1b87ca617b154f6c64b46476fa08</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm060029z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060029z%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DSynergy%2520and%2520Antagonism%2520of%2520Promiscuous%2520Inhibition%2520in%2520Multiple-Compound%2520Mixtures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2151%26epage%3D2154%26doi%3D10.1021%2Fjm060029z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. W.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery and Natural Products: End of an Era or an Endless Frontier?</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>325</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1126/science.1168243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1126%2Fscience.1168243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19589993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovVGmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=161-165&author=J.+W.-H.+Liauthor=J.+C.+Vederas&title=Drug+Discovery+and+Natural+Products%3A+End+of+an+Era+or+an+Endless+Frontier%3F&doi=10.1126%2Fscience.1168243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery and natural products end of an era or an endless frontier?</span></div><div class="casAuthors">Li, Jess W.-H.; Vederas, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5937</span>),
    <span class="NLM_cas:pages">161-165</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90S_IHvC-8bVg90H21EOLACvtfcHk0lgxvINPomVjOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovVGmsrc%253D&md5=92cbb0ae2947501b031a99b774d2adfb</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1126%2Fscience.1168243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1168243%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%2BW.-H.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26atitle%3DDrug%2520Discovery%2520and%2520Natural%2520Products%253A%2520End%2520of%2520an%2520Era%2520or%2520an%2520Endless%2520Frontier%253F%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D161%26epage%3D165%26doi%3D10.1126%2Fscience.1168243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisman, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, W. A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Which Aspects of HTS Are Empirically Correlated with Downstream Success?</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">337</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=18428086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=327-337&author=A.+Benderauthor=D.+Bojanicauthor=J.+W.+Daviesauthor=T.+J.+Crismanauthor=D.+Mikhailovauthor=J.+Scheiberauthor=J.+L.+Jenkinsauthor=Z.+Dengauthor=W.+A.+G.+Hillauthor=M.+Popovauthor=E.+Jacobyauthor=M.+Glick&title=Which+Aspects+of+HTS+Are+Empirically+Correlated+with+Downstream+Success%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Which aspects of HTS are empirically correlated with downstream success?</span></div><div class="casAuthors">Bender, Andreas; Bojanic, Dejan; Davies, John W.; Crisman, Thomas J.; Mikhailov, Dmitri; Scheiber, Josef; Jenkins, Jeremy L.; Deng, Zhan; Hill, W. Adam G.; Popov, Maxim; Jacoby, Edgar; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-337</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established hit-finding approach used in the pharmaceutical industry.  In this article, recent experience at Novartis with respect to factors influencing the success of HTS campaigns is discussed.  An inherent measure of HTS quality could be defined by the assay Z and Z' factors, the no. of hits and their biol. potencies; however, such measures of quality do not always correlate with the advancement of hits to the later stages of drug discovery.  Also, for many target classes, such as kinases, it is easy to identify hits, but, as a result of selectivity, intellectual property and other issues, the projects do not result in lead declarations.  In this article, HTS success is defined as the fraction of HTS campaigns that advance into the later stages of drug discovery, and the major influencing factors are examd.  Interestingly, screening compds. in individual wells or in mixts. did not have a major impact on the HTS success and, equally interesting, there was no difference in the progression rates of biochem. and cell-based assays.  Particular target types, assay technologies, structure-activity relationships and powder availability had a much greater impact on success as defined above.  In addn., significant mutual dependencies can be obsd. - while one assay format works well with one target type, this situation might be completely reversed for a combination of the same readout technol. with a different target type.  The results and opinions presented here should be regarded as groundwork, and a plethora of factors that influence the fate of a project, such as biophys. measurements, chem. attractiveness of the hits, strategic reasons and safety pharmacol., are not covered here.  Nonetheless, it is hoped that this information will be used industry-wide to improve success rates in terms of hits progressing into exploratory chem. and beyond.  The support that can be obtained from new in silico approaches to phase transitions are also described, along with the gaps they are designed to fill.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqddDbeDZXfNbVg90H21EOLACvtfcHk0lgxvINPomVjOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D&md5=c65c4142d449fe752b37c397cd2a941e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DCrisman%26aufirst%3DT.%2BJ.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DHill%26aufirst%3DW.%2BA.%2BG.%26aulast%3DPopov%26aufirst%3DM.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DWhich%2520Aspects%2520of%2520HTS%2520Are%2520Empirically%2520Correlated%2520with%2520Downstream%2520Success%253F%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26spage%3D327%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">18428086</p></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Current Trends in Lead Discovery: Are We Looking for the Appropriate Properties?</span>. <i>Mol. Diversity</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1023/a:1021368007777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1023%2FA%3A1021368007777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=12549672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFOjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2002&pages=199-208&author=T.+I.+Oprea&title=Current+Trends+in+Lead+Discovery%3A+Are+We+Looking+for+the+Appropriate+Properties%3F&doi=10.1023%2Fa%3A1021368007777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Current trends in lead discovery: Are we looking for the appropriate properties?</span></div><div class="casAuthors">Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">199-208</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  The new drug discovery paradigm is based on high-throughput technologies, both with respect to synthesis and screening.  The progression HTS hits lead series candidate drug marketed drug appears to indicate that the probability of reaching launched status is one in a million.  This has shifted the focus from good quality candidate drugs to good quality leads.  We examd. the current trends in lead discovery by comparing MW (mol. wt.), LogP (octanol/water partition coeff., estd. by Kowwin [17]) and LogSw (intrinsic water soly., estd. by Wskowwin [18]) for the following categories: 62 leads and 75 drugs [11]; compds. in the development phase (I, II, III and launched), as indexed in MDDR; and compds. indexed in medicinal chem. journals [ref. 20], categorized according to their biol. activity.  Comparing the distribution of the above properties, the 62 lead structures show the lowest median with respect to MW (smaller) and LogP (less hydrophobic), and the highest median with respect to LogSw (more sol.).  By contrast, over 50% of the medicinal chem. compds. with activities above 1 nanomolar have MW > 425, LogP > 4.25 and LogSw < -4.75, indicating that the reported active compds. are larger, more hydrophobic and less sol. when compared to time-tested quality leads.  In the MDDR set, a progressive constraint to reduce MW and LogP, and to increase LogSw, can be obsd. when examg. trends in the developmental sequence: phase I, II, III and launched drugs.  These trends indicate that other properties besides binding affinity, e.g., soly. and hydrophobicity, need to be considered when choosing the appropriate leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHbdVqIWk9UrVg90H21EOLACvtfcHk0lgxvINPomVjOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFOjurg%253D&md5=1feef5374888c429af331a2620cf7e73</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1023%2FA%3A1021368007777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1021368007777%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DCurrent%2520Trends%2520in%2520Lead%2520Discovery%253A%2520Are%2520We%2520Looking%2520for%2520the%2520Appropriate%2520Properties%253F%26jtitle%3DMol.%2520Diversity%26date%3D2002%26volume%3D5%26spage%3D199%26epage%3D208%26doi%3D10.1023%2Fa%3A1021368007777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andermann, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowell, B. B.</span></span> <span> </span><span class="NLM_article-title">Toward a Wiring Diagram Understanding of Appetite Control</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">757</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2017.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.neuron.2017.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28817798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlClur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2017&pages=757-778&author=M.+L.+Andermannauthor=B.+B.+Lowell&title=Toward+a+Wiring+Diagram+Understanding+of+Appetite+Control&doi=10.1016%2Fj.neuron.2017.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Toward a Wiring Diagram Understanding of Appetite Control</span></div><div class="casAuthors">Andermann, Mark L.; Lowell, Bradford B.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">757-778</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Prior mouse genetic research has set the stage for a deep understanding of appetite regulation.  This goal is now being realized through the use of recent technol. advances, such as the ability to map connectivity between neurons, manipulate neural activity in real time, and measure neural activity during behavior.  Indeed, major progress has been made with regard to meal-related gut control of appetite, arcuate nucleus-based hypothalamic circuits linking energy state to the motivational drive, hunger, and, finally, limbic and cognitive processes that bring about hunger-mediated increases in reward value and perception of food.  Unexpected findings are also being made; for example, the rapid regulation of homeostatic neurons by cues that predict future food consumption.  The aim of this review is to cover the major underpinnings of appetite regulation, describe recent advances resulting from new technologies, and synthesize these findings into an updated view of appetite regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDBDn95oahuLVg90H21EOLACvtfcHk0lhF5y8tocDPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlClur3M&md5=5c00631d3124358c5328b079ece02bc8</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2017.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2017.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DAndermann%26aufirst%3DM.%2BL.%26aulast%3DLowell%26aufirst%3DB.%2BB.%26atitle%3DToward%2520a%2520Wiring%2520Diagram%2520Understanding%2520of%2520Appetite%2520Control%26jtitle%3DNeuron%26date%3D2017%26volume%3D95%26spage%3D757%26epage%3D778%26doi%3D10.1016%2Fj.neuron.2017.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishigaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span> <span> </span><span class="NLM_article-title">Supranuclear Melanin Caps Reduce Ultraviolet Induced DNA Photoproducts in Human Epidermis</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">806</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1046/j.1523-1747.1998.00178.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1046%2Fj.1523-1747.1998.00178.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9579550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1cXjtVGnt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1998&pages=806-810&author=N.+Kobayashiauthor=A.+Nakagawaauthor=T.+Muramatsuauthor=Y.+Yamashinaauthor=T.+Shiraiauthor=M.+W.+Hashimotoauthor=Y.+Ishigakiauthor=T.+Ohnishiauthor=T.+Mori&title=Supranuclear+Melanin+Caps+Reduce+Ultraviolet+Induced+DNA+Photoproducts+in+Human+Epidermis&doi=10.1046%2Fj.1523-1747.1998.00178.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Supranuclear melanin caps reduce ultraviolet-induced DNA photoproducts in human epidermis</span></div><div class="casAuthors">Kobayashi, Nobuhiko; Nakagawa, Akemi; Muramatsu, Tsutomu; Yamashina, Yukio; Shirai, Toshihiko; Hashimoto, Mitsumasa W.; Ishigaki, Yasuhito; Ohnishi, Takeo; Mori, Toshio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">806-810</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science, Inc.</span>)
        </div><div class="casAbstract">Melanin can form supranuclear caps in human epidermis, suggesting that intracellular melanin reduces UV transmission to underlying cell nuclei and inhibits the formation of UV-induced DNA photoproducts.  The purpose of this study was to det. the photoprotective effect of epidermal melanin.  We irradiated normal human skin explants with UV B and detd. the formation of cyclobutane pyrimidine dimers and (6-4)photoproducts in individual epidermal cells by indirect immunofluorescence and by laser cytometry using monoclonal antibodies specific for cyclobutane dimers or for (6-4)photoproducts.  We found that epidermal cells with supranuclear melanin caps had significantly less DNA photoproducts (both types) than epidermal cells without supranuclear melanin caps.  Moreover, the protection factor against both types of photolesions correlated with melanin concn. in epidermal cells.  These results indicate that melanin reduces UV induced DNA photoproducts in human epidermis in a concn. dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJzwMcI7wq3LVg90H21EOLACvtfcHk0lhF5y8tocDPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtVGnt7o%253D&md5=2100eeb68caf08c222228d357ff1e1ed</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1046%2Fj.1523-1747.1998.00178.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1523-1747.1998.00178.x%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DN.%26aulast%3DNakagawa%26aufirst%3DA.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DYamashina%26aufirst%3DY.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DM.%2BW.%26aulast%3DIshigaki%26aufirst%3DY.%26aulast%3DOhnishi%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26atitle%3DSupranuclear%2520Melanin%2520Caps%2520Reduce%2520Ultraviolet%2520Induced%2520DNA%2520Photoproducts%2520in%2520Human%2520Epidermis%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D1998%26volume%3D110%26spage%3D806%26epage%3D810%26doi%3D10.1046%2Fj.1523-1747.1998.00178.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadekaro, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leachman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavanagh, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swope, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwemberger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babcock, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakamatsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koshoffer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boissy, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manga, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Malek, Z. A.</span></span> <span> </span><span class="NLM_article-title"><i>Melanocortin 1 Receptor Genotype</i>: An Important Determinant of the Damage Response of Melanocytes to Ultraviolet Radiation</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3850</span>– <span class="NLM_lpage">3860</span>, <span class="refDoi"> DOI: 10.1096/fj.10-158485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1096%2Ffj.10-158485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20519635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12isLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=3850-3860&author=A.+L.+Kadekaroauthor=S.+Leachmanauthor=R.+J.+Kavanaghauthor=V.+Swopeauthor=P.+Cassidyauthor=D.+Suppauthor=M.+Sartorauthor=S.+Schwembergerauthor=G.+Babcockauthor=K.+Wakamatsuauthor=S.+Itoauthor=A.+Koshofferauthor=R.+E.+Boissyauthor=P.+Mangaauthor=R.+A.+Sturmauthor=Z.+A.+Abdel-Malek&title=Melanocortin+1+Receptor+Genotype%3A+An+Important+Determinant+of+the+Damage+Response+of+Melanocytes+to+Ultraviolet+Radiation&doi=10.1096%2Ffj.10-158485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Melanocortin 1 receptor genotype: an important determinant of the damage response of melanocytes to ultraviolet radiation</span></div><div class="casAuthors">Kadekaro, Ana Luisa; Leachman, Sancy; Kavanagh, Renny J.; Swope, Viki; Cassidy, Pamela; Supp, Dorothy; Sartor, Maureen; Schwemberger, Sandy; Babcock, George; Wakamatsu, Kazumasa; Ito, Shosuke; Koshoffer, Amy; Boissy, Raymond E.; Manga, Prashiela; Sturm, Richard A.; Abdel-Malek, Zalfa A.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3850-3860, 10.1096/fj.10-158485</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">The melanocortin 1 receptor gene is a main determinant of human pigmentation, and a melanoma susceptibility gene, because its variants that are strongly assocd. with red hair color increase melanoma risk.  To test exptl. the assocn. between melanocortin 1 receptor genotype and melanoma susceptibility, we compared the responses of primary human melanocyte cultures naturally expressing different melanocortin 1 receptor variants to α-melanocortin and UV radiation.  We found that expression of 2 red hair variants abolished the response to α-melanocortin and its photoprotective effects, evidenced by lack of functional coupling of the receptor, and absence of redn. in UV radiation-induced hydrogen peroxide generation or enhancement of repair of DNA photoproducts, resp.  These variants had different heterozygous effects on receptor function.  Microarray data confirmed the obsd. differences in responses of melanocytes with functional vs. non-functional receptor to α-melanocortin and UV radiation, and identified DNA repair and antioxidant genes that are modulated by α-melanocortin.  Our findings highlight the mol. mechanisms by which the melanocortin 1 receptor genotype controls genomic stability of and the mutagenic effect of UV radiation on human melanocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofwk7RqEhWCrVg90H21EOLACvtfcHk0lhF5y8tocDPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12isLfF&md5=c52c8aef97605bc44d98152e6e95121b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1096%2Ffj.10-158485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.10-158485%26sid%3Dliteratum%253Aachs%26aulast%3DKadekaro%26aufirst%3DA.%2BL.%26aulast%3DLeachman%26aufirst%3DS.%26aulast%3DKavanagh%26aufirst%3DR.%2BJ.%26aulast%3DSwope%26aufirst%3DV.%26aulast%3DCassidy%26aufirst%3DP.%26aulast%3DSupp%26aufirst%3DD.%26aulast%3DSartor%26aufirst%3DM.%26aulast%3DSchwemberger%26aufirst%3DS.%26aulast%3DBabcock%26aufirst%3DG.%26aulast%3DWakamatsu%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DS.%26aulast%3DKoshoffer%26aufirst%3DA.%26aulast%3DBoissy%26aufirst%3DR.%2BE.%26aulast%3DManga%26aufirst%3DP.%26aulast%3DSturm%26aufirst%3DR.%2BA.%26aulast%3DAbdel-Malek%26aufirst%3DZ.%2BA.%26atitle%3DMelanocortin%25201%2520Receptor%2520Genotype%253A%2520An%2520Important%2520Determinant%2520of%2520the%2520Damage%2520Response%2520of%2520Melanocytes%2520to%2520Ultraviolet%2520Radiation%26jtitle%3DFASEB%2520J.%26date%3D2010%26volume%3D24%26spage%3D3850%26epage%3D3860%26doi%3D10.1096%2Ffj.10-158485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Creemers, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Rouzic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meulemans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardlaw, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luckman, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennand, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span> <span> </span><span class="NLM_article-title">Agouti-Related Protein Is Posttranslationally Cleaved by Proprotein Convertase 1 to Generate Agouti-Related Protein (AGRP)<sub>83-132</sub>: Interaction between AGRP<sub>83-132</sub> and Melanocortin Receptors Cannot Be Influenced by Syndecan-3</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">1621</span>– <span class="NLM_lpage">1631</span>, <span class="refDoi"> DOI: 10.1210/en.2005-1373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1210%2Fen.2005-1373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=16384863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt1Ohsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2006&pages=1621-1631&author=J.+W.+M.+Creemersauthor=L.+E.+Pritchardauthor=A.+Gyteauthor=P.+Le+Rouzicauthor=S.+Meulemansauthor=S.+L.+Wardlawauthor=X.+Zhuauthor=D.+F.+Steinerauthor=N.+Daviesauthor=D.+Armstrongauthor=C.+B.+Lawrenceauthor=S.+M.+Luckmanauthor=C.+A.+Schmitzauthor=R.+A.+Daviesauthor=J.+C.+Brennandauthor=A.+White&title=Agouti-Related+Protein+Is+Posttranslationally+Cleaved+by+Proprotein+Convertase+1+to+Generate+Agouti-Related+Protein+%28AGRP%2983-132%3A+Interaction+between+AGRP83-132+and+Melanocortin+Receptors+Cannot+Be+Influenced+by+Syndecan-3&doi=10.1210%2Fen.2005-1373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Agouti-related protein is posttranslationally cleaved by proprotein convertase 1 to generate agouti-related protein (AGRP)83-132: interaction between AGRP83-132 and melanocortin receptors cannot be influenced by syndecan-3</span></div><div class="casAuthors">Creemers, John W. M.; Pritchard, Lynn E.; Gyte, Amy; Le Rouzic, Philippe; Meulemans, Sandra; Wardlaw, Sharon L.; Zhu, Xiaorong; Steiner, Donald F.; Davies, Nicola; Armstrong, Duncan; Lawrence, Catherine B.; Luckman, Simon M.; Schmitz, Catherine A.; Davies, Rick A.; Brennand, John C.; White, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1621-1631</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Agouti-related protein (AGRP) plays a key role in energy homeostasis.  The C-terminal domain of AGRP acts as an endogenous antagonist of the melanocortin-4 receptor (MC4-R).  It has been suggested that the N-terminal domain of AGRP binds to syndecan-3, thereby modulating the effects of C-terminal AGRP at the MC4-R.  This model assumes that AGRP is secreted as a full-length peptide.  In this study the authors found that AGRP is processed intercellularly after Arg79-Glu80-Pro81-Arg82.  The processing site suggests cleavage by proprotein convertases (PCs).  RNA interference and overexpression expts. showed that PC1/3 is primarily responsible for cleavage in vitro, although both PC2 and PC5/6A can also process AGRP.  Dual in situ hybridization demonstrated that PC1/3 is expressed in AGRP neurons in the rat hypothalamus.  Moreover, hypothalamic exts. from PC1-null mice contained 3.3-fold more unprocessed full-length AGRP, compared with wild-type mice, based on combined HPLC and RIA anal., demonstrating that PC1/3 plays a role in AGRP cleavage in vivo.  The authors also found that AGRP83-132 is more potent an antagonist than full-length AGRP, based on cAMP reporter assays, suggesting that posttranslational cleavage is required to potentiate the effect of AGRP at the MC4-R.  Because AGRP is cleaved into distinct N-terminal and C-terminal peptides, the authors tested whether N-terminal peptides modulate food intake.  However, intracerebroventricular injection of rat AGRP25-47 and AGRP50-80 had no effect on body wt., food intake, or core body temp.  Because AGRP is cleaved before secretion, syndecan-3 must influence food intake independently of the MC4-R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyGX8-BXRj87Vg90H21EOLACvtfcHk0lg0dRVU4XWAuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt1Ohsbk%253D&md5=01d541e583842f4eb81d65685ab6639f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1210%2Fen.2005-1373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2005-1373%26sid%3Dliteratum%253Aachs%26aulast%3DCreemers%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DPritchard%26aufirst%3DL.%2BE.%26aulast%3DGyte%26aufirst%3DA.%26aulast%3DLe%2BRouzic%26aufirst%3DP.%26aulast%3DMeulemans%26aufirst%3DS.%26aulast%3DWardlaw%26aufirst%3DS.%2BL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DSteiner%26aufirst%3DD.%2BF.%26aulast%3DDavies%26aufirst%3DN.%26aulast%3DArmstrong%26aufirst%3DD.%26aulast%3DLawrence%26aufirst%3DC.%2BB.%26aulast%3DLuckman%26aufirst%3DS.%2BM.%26aulast%3DSchmitz%26aufirst%3DC.%2BA.%26aulast%3DDavies%26aufirst%3DR.%2BA.%26aulast%3DBrennand%26aufirst%3DJ.%2BC.%26aulast%3DWhite%26aufirst%3DA.%26atitle%3DAgouti-Related%2520Protein%2520Is%2520Posttranslationally%2520Cleaved%2520by%2520Proprotein%2520Convertase%25201%2520to%2520Generate%2520Agouti-Related%2520Protein%2520%2528AGRP%252983-132%253A%2520Interaction%2520between%2520AGRP83-132%2520and%2520Melanocortin%2520Receptors%2520Cannot%2520Be%2520Influenced%2520by%2520Syndecan-3%26jtitle%3DEndocrinology%26date%3D2006%26volume%3D147%26spage%3D1621%26epage%3D1631%26doi%3D10.1210%2Fen.2005-1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tota, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacNeil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Ploeg, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, T. M.</span></span> <span> </span><span class="NLM_article-title">Molecular Interaction of Agouti Protein and Agouti-Related Protein with Human Melanocortin Receptors</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">904</span>, <span class="refDoi"> DOI: 10.1021/bi9815602</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9815602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1cXotFajtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=897-904&author=M.+R.+Totaauthor=T.+S.+Smithauthor=C.+Maoauthor=T.+MacNeilauthor=R.+T.+Mosleyauthor=L.+H.+Van+der+Ploegauthor=T.+M.+Fong&title=Molecular+Interaction+of+Agouti+Protein+and+Agouti-Related+Protein+with+Human+Melanocortin+Receptors&doi=10.1021%2Fbi9815602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Interaction of Agouti Protein and Agouti-Related Protein with Human Melanocortin Receptors</span></div><div class="casAuthors">Tota, M. R.; Smith, T. S.; Mao, C.; MacNeil, T.; Mosley, R. T.; Van der Ploeg, L. H. T.; Fong, T. M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">897-904</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Agouti protein and the Agouti-related protein (AGRP) are antagonists of the melanocortin-3 receptor and melanocortin-4 receptor.  Both proteins contain 10 cysteines in the C-terminal domain arranged in five disulfide bonds.  One possible arrangement of the disulfide bonds predicts an octapeptide loop, and the chem. properties of four residues within this loop (residues 111-114 in human AGRP) bear striking resemblance to those of several melanocortin peptides, including α-MSH, MT-II, and SHU-9119.  We showed that cyclic synthetic octapeptides based on the sequence of this loop from Agouti protein or human AGRP are functional antagonists of the human melanocortin-4 receptor.  All peptides had a lower affinity for the melanocortin-3 receptor than for the melanocortin-4 receptor.  Substitution of serines for cysteines resulted in linear peptides which had reduced binding affinities for both receptors.  Mutational anal. of human AGRP indicated that its C-terminal domain is functionally equiv. to the intact human AGRP.  The RFF111-113 triplet appears to be the most crit. portion of AGRP in detg. the binding affinity for both melanocortin-3 and melanocortin-4 receptors.  These data strongly suggest that the loop defined by Cys-110 and Cys-117 is crit. in detg. the antagonist activity of human AGRP.  Our data provide indirect evidence for the suggestion that the Cys-110 to Cys-117 octapeptide loop of human AGRP mimics the conformation of α-MSH, MT-II, and SHU-9119.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-2vz5sSpD8rVg90H21EOLACvtfcHk0lg0dRVU4XWAuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotFajtbg%253D&md5=58feaee4b8438ff9742c8d11af8f7c24</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fbi9815602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9815602%26sid%3Dliteratum%253Aachs%26aulast%3DTota%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DT.%2BS.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DMacNeil%26aufirst%3DT.%26aulast%3DMosley%26aufirst%3DR.%2BT.%26aulast%3DVan%2Bder%2BPloeg%26aufirst%3DL.%2BH.%26aulast%3DFong%26aufirst%3DT.%2BM.%26atitle%3DMolecular%2520Interaction%2520of%2520Agouti%2520Protein%2520and%2520Agouti-Related%2520Protein%2520with%2520Human%2520Melanocortin%2520Receptors%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D897%26epage%3D904%26doi%3D10.1021%2Fbi9815602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nix, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsh, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span> <span> </span><span class="NLM_article-title">Molecular and Functional Analysis of Human β-Defensin 3 Action at Melanocortin Receptors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.chembiol.2013.04.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23790489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVWgtLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=784-795&author=M.+A.+Nixauthor=C.+B.+Kaelinauthor=T.+Taauthor=A.+Weisauthor=G.+J.+Mortonauthor=G.+S.+Barshauthor=G.+L.+Millhauser&title=Molecular+and+Functional+Analysis+of+Human+%CE%B2-Defensin+3+Action+at+Melanocortin+Receptors&doi=10.1016%2Fj.chembiol.2013.04.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and Functional Analysis of Human β-Defensin 3 Action at Melanocortin Receptors</span></div><div class="casAuthors">Nix, Matthew A.; Kaelin, Christopher B.; Ta, Tina; Weis, Allison; Morton, Gregory J.; Barsh, Gregory S.; Millhauser, Glenn L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">784-795</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The β-defensins are a class of small, cationic proteins first recognized as antimicrobial components of the innate and adaptive immune system.  More recently, one of the major β-defensins produced in skin, β-defensin 3, has been discovered to function as a melanocortin receptor ligand in vivo and in vitro, but its biophys. and pharmacol. basis of action has been enigmatic.  Here, we report functional and biochem. studies focused on human β-defensin 3 (HBD3) and melanocortin receptors 1 and 4.  Genetic and pharmacol. studies indicate that HBD3 acts as a neutral melanocortin receptor antagonist capable of blocking the action of either stimulatory agonists such as α-MSH or inhibitory inverse agonists such as Agouti signaling protein (ASIP) and Agouti-related protein (AGRP).  A comprehensive structure-function anal. demonstrates that two patches of pos. charged residues, located on opposite poles of HBD3 and spatially organized by the compact β-defensin fold, are primarily responsible for high-affinity binding to melanocortin receptors.  These findings identify a distinct mode of melanocortin receptor-ligand interactions based primarily on electrostatic complementarity, with implications for designing ligands that target melanocortin and potentially other seven transmembrane receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG6EBWHno3DLVg90H21EOLACvtfcHk0lg0dRVU4XWAuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVWgtLzK&md5=1d500a5e57f1d6918f182f6d17adebfb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DNix%26aufirst%3DM.%2BA.%26aulast%3DKaelin%26aufirst%3DC.%2BB.%26aulast%3DTa%26aufirst%3DT.%26aulast%3DWeis%26aufirst%3DA.%26aulast%3DMorton%26aufirst%3DG.%2BJ.%26aulast%3DBarsh%26aufirst%3DG.%2BS.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26atitle%3DMolecular%2520and%2520Functional%2520Analysis%2520of%2520Human%2520%25CE%25B2-Defensin%25203%2520Action%2520at%2520Melanocortin%2520Receptors%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D784%26epage%3D795%26doi%3D10.1016%2Fj.chembiol.2013.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ericson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tala, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslach, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirain, M. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaub, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eick, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lensing, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smeester, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adank, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskell-Luevano, C.</span></span> <span> </span><span class="NLM_article-title">Human β-Defensin 1 and β-Defensin 3 (Mouse Ortholog MBD14) Function as Full Endogenous Agonists at Select Melanocortin Receptors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3738</span>– <span class="NLM_lpage">3744</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00251</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00251" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVKnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3738-3744&author=M.+D.+Ericsonauthor=A.+Singhauthor=S.+R.+Talaauthor=E.+M.+Haslachauthor=M.+L.+S.+Dirainauthor=J.+W.+Schaubauthor=V.+Floresauthor=N.+Eickauthor=C.+J.+Lensingauthor=K.+T.+Freemanauthor=B.+A.+Smeesterauthor=D.+N.+Adankauthor=S.+L.+Wilberauthor=R.+Spethauthor=C.+Haskell-Luevano&title=Human+%CE%B2-Defensin+1+and+%CE%B2-Defensin+3+%28Mouse+Ortholog+MBD14%29+Function+as+Full+Endogenous+Agonists+at+Select+Melanocortin+Receptors&doi=10.1021%2Facs.jmedchem.8b00251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Human β-defensin 1 and β-defensin 3 (mouse ortholog mBD14) function as full endogenous agonists at select melanocortin receptors</span></div><div class="casAuthors">Ericson, Mark D.; Singh, Anamika; Tala, Srinivasa R.; Haslach, Erica M.; Dirain, Marvin L. S.; Schaub, Jay W.; Flores, Viktor; Eick, Natalie; Lensing, Cody J.; Freeman, Katie T.; Smeester, Branden A.; Adank, Danielle N.; Wilber, Stacey L.; Speth, Robert; Haskell-Luevano, Carrie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3738-3744</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">β-Defensin 3 (BD3) was identified as a ligand for the melanocortin receptors (MCRs) in 2007, although the pharmacol. activity of BD3 has not been clearly elucidated.  Herein, it is demonstrated that human BD3 and mouse BD3 are full micromolar agonists at the MCRs.  Furthermore, mouse β-defensin 1 (BD1) and human BD1 are also MCR micromolar agonists.  This work identifies BD1 as an endogenous MCR ligand and clarifies the controversial role of BD3 as a micromolar agonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogj7J3XNbylrVg90H21EOLACvtfcHk0lhWq8IYolyHdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVKnurY%253D&md5=de44e85a43566ca94cd136c8f3123f87</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00251%26sid%3Dliteratum%253Aachs%26aulast%3DEricson%26aufirst%3DM.%2BD.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DTala%26aufirst%3DS.%2BR.%26aulast%3DHaslach%26aufirst%3DE.%2BM.%26aulast%3DDirain%26aufirst%3DM.%2BL.%2BS.%26aulast%3DSchaub%26aufirst%3DJ.%2BW.%26aulast%3DFlores%26aufirst%3DV.%26aulast%3DEick%26aufirst%3DN.%26aulast%3DLensing%26aufirst%3DC.%2BJ.%26aulast%3DFreeman%26aufirst%3DK.%2BT.%26aulast%3DSmeester%26aufirst%3DB.%2BA.%26aulast%3DAdank%26aufirst%3DD.%2BN.%26aulast%3DWilber%26aufirst%3DS.%2BL.%26aulast%3DSpeth%26aufirst%3DR.%26aulast%3DHaskell-Luevano%26aufirst%3DC.%26atitle%3DHuman%2520%25CE%25B2-Defensin%25201%2520and%2520%25CE%25B2-Defensin%25203%2520%2528Mouse%2520Ortholog%2520MBD14%2529%2520Function%2520as%2520Full%2520Endogenous%2520Agonists%2520at%2520Select%2520Melanocortin%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3738%26epage%3D3744%26doi%3D10.1021%2Facs.jmedchem.8b00251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trulson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gantz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span> <span> </span><span class="NLM_article-title">NMR Structure of a Minimized Human Agouti Related Protein Prepared by Total Chemical Synthesis</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1016/s0014-5793(99)00553-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0014-5793%2899%2900553-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10371151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1MXjsVSjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=1999&pages=125-131&author=K.+A.+Bolinauthor=D.+J.+Andersonauthor=J.+A.+Trulsonauthor=D.+A.+Thompsonauthor=J.+Wilkenauthor=S.+B.+H.+Kentauthor=I.+Gantzauthor=G.+L.+Millhauser&title=NMR+Structure+of+a+Minimized+Human+Agouti+Related+Protein+Prepared+by+Total+Chemical+Synthesis&doi=10.1016%2Fs0014-5793%2899%2900553-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">NMR structure of a minimized human agouti related protein prepared by total chemical synthesis</span></div><div class="casAuthors">Bolin, Kimberly A.; Anderson, D. Joe; Trulson, Julie A.; Thompson, Darren A.; Wilken, Jill; Kent, Stephen B. H.; Gantz, Ira; Millhauser, Glenn L.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-131</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The structure of the chem. synthesized C-terminal region of the human agouti related protein (AGRP) was detd. by 2D 1H NMR.  Referred to as minimized agouti related protein, MARP is a 46 residue polypeptide contg. 10 Cys residues involved in five disulfide bonds that retains the biol. activity of full length AGRP.  AGRP is a mammalian signaling mol., involved in wt. homeostasis, that causes adult onset obesity when overexpressed in mice.  AGRP was originally identified by homol. to the agouti protein, another potent signaling mol. involved in obesity disorders in mice.  While AGRP's exact mechanism of action is unknown, it has been identified as a competitive antagonist of melanocortin receptors 3 and 4 (MC3r, MC4r), and MC4r in particular is implicated in the hypothalamic control of feeding behavior.  Full length agouti and AGRP are only 25% homologous, however, their active C-terminal regions are ∼40% homologous, with nine out of the 10 Cys residues spatially conserved.  Until now, 3D structures have not been available for either agouti, AGRP or their C-terminal regions.  The NMR structure of MARP reported here can be characterized as three major loops, with four of the five disulfide bridges at the base of the structure.  Though its fold is well defined, no canonical secondary structure is identified.  While previously reported structural models of the C-terminal region of AGRP were attempted based on Cys homol. between AGRP and certain toxin proteins, we find that Cys spacing is not sufficient to correctly det. the 3D fold of the mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN7u4VlD35BbVg90H21EOLACvtfcHk0lhWq8IYolyHdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjsVSjtLg%253D&md5=7c74e63ec620183f63eb0069a9ee5b42</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2899%2900553-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252899%252900553-0%26sid%3Dliteratum%253Aachs%26aulast%3DBolin%26aufirst%3DK.%2BA.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DTrulson%26aufirst%3DJ.%2BA.%26aulast%3DThompson%26aufirst%3DD.%2BA.%26aulast%3DWilken%26aufirst%3DJ.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26aulast%3DGantz%26aufirst%3DI.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26atitle%3DNMR%2520Structure%2520of%2520a%2520Minimized%2520Human%2520Agouti%2520Related%2520Protein%2520Prepared%2520by%2520Total%2520Chemical%2520Synthesis%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D451%26spage%3D125%26epage%3D131%26doi%3D10.1016%2Fs0014-5793%2899%2900553-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schibli, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aseyev, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starner, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiencek, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCray, P. B.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tack, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H. J.</span></span> <span> </span><span class="NLM_article-title">The Solution Structures of the Human Beta-Defensins Lead to a Better Understanding of the Potent Bactericidal Activity of HBD3 against <i>Staphylococcus Aureus</i></span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">8279</span>– <span class="NLM_lpage">8289</span>, <span class="refDoi"> DOI: 10.1074/jbc.M108830200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.M108830200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=11741980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFymt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=8279-8289&author=D.+J.+Schibliauthor=H.+N.+Hunterauthor=V.+Aseyevauthor=T.+D.+Starnerauthor=J.+M.+Wiencekauthor=P.+B.+McCrayauthor=B.+F.+Tackauthor=H.+J.+Vogel&title=The+Solution+Structures+of+the+Human+Beta-Defensins+Lead+to+a+Better+Understanding+of+the+Potent+Bactericidal+Activity+of+HBD3+against+Staphylococcus+Aureus&doi=10.1074%2Fjbc.M108830200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The solution structures of the human β-defensins lead to a better understanding of the potent bactericidal activity of HBD3 against Staphylococcus aureus</span></div><div class="casAuthors">Schibli, David J.; Hunter, Howard N.; Aseyev, Vladimir; Starner, Timothy D.; Wiencek, John M.; McCray, Paul B., Jr.; Tack, Brian F.; Vogel, Hans J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">8279-8289</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The three human β-defensins, HBD1-3, are 33-47-residue, cationic antimicrobial proteins expressed by epithelial cells.  All three proteins have broad-spectrum antimicrobial activity, with HBD3 consistently being the most potent.  Addnl., HBD3 has significant bactericidal activity against Gram-pos. Staphylococcus aureus at physiol. salt concns.  We have compared the multimeric state of the three β-defensins using NMR diffusion spectroscopy, dynamic and static light scattering, and anal. of the migration of the three β-defensins on a native gel.  All three techniques are in agreement, suggesting that HBD-3 is a dimer, while HBD-1 and HBD-2 are monomeric.  Subsequently, the NMR soln. structures of HBD1 and HBD3 were detd. using std. homonuclear techniques and compared with the previously detd. soln. structure of HBD2.  Both HBD1 and HBD3 form well defined structures with backbone root mean square deviations of 0.451 and 0.616 Å, resp.  The tertiary structures of all three β-defensins are similar, with a short helical segment preceding a three-stranded antiparallel β-sheet.  The surface charge d. of each of the defensins is markedly different, with the surface of HBD3 significantly more basic.  Anal. of the NMR data and structures led us to suggest that HBD3 forms a sym. dimer through strand β2 of the β-sheet.  The increased anti-Staphylococcal activity of HBD3 may be explained by the capacity of the protein to form dimers in soln. at low concns., an amphipathic dimer structure, and the increased pos. surface charge compared with HBD1 and HBD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqItwZ3MBW107Vg90H21EOLACvtfcHk0lhWq8IYolyHdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFymt78%253D&md5=e7f4fb1e06497eb88c69140e6511eec0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M108830200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M108830200%26sid%3Dliteratum%253Aachs%26aulast%3DSchibli%26aufirst%3DD.%2BJ.%26aulast%3DHunter%26aufirst%3DH.%2BN.%26aulast%3DAseyev%26aufirst%3DV.%26aulast%3DStarner%26aufirst%3DT.%2BD.%26aulast%3DWiencek%26aufirst%3DJ.%2BM.%26aulast%3DMcCray%26aufirst%3DP.%2BB.%26aulast%3DTack%26aufirst%3DB.%2BF.%26aulast%3DVogel%26aufirst%3DH.%2BJ.%26atitle%3DThe%2520Solution%2520Structures%2520of%2520the%2520Human%2520Beta-Defensins%2520Lead%2520to%2520a%2520Better%2520Understanding%2520of%2520the%2520Potent%2520Bactericidal%2520Activity%2520of%2520HBD3%2520against%2520Staphylococcus%2520Aureus%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D8279%26epage%3D8289%26doi%3D10.1074%2Fjbc.M108830200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkes, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Obeidi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staples, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vaux, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dym, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castrucci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintz, M. F.</span></span> <span> </span><span class="NLM_article-title">.alpha.-Melanotropin: The Minimal Active Sequence in the Frog Skin Bioassay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2130</span>, <span class="refDoi"> DOI: 10.1021/jm00394a033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00394a033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaL2sXlslKntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1987&pages=2126-2130&author=V.+J.+Hrubyauthor=B.+C.+Wilkesauthor=M.+E.+Hadleyauthor=F.+Al-Obeidiauthor=T.+K.+Sawyerauthor=D.+J.+Staplesauthor=A.+E.+de+Vauxauthor=O.+Dymauthor=A.+M.+Castrucciauthor=M.+F.+Hintz&title=.alpha.-Melanotropin%3A+The+Minimal+Active+Sequence+in+the+Frog+Skin+Bioassay&doi=10.1021%2Fjm00394a033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">α-Melanotropin:  the minimal active sequence in the frog skin bioassay</span></div><div class="casAuthors">Hruby, Victor J.; Wilkes, Brian C.; Hadley, Mac E.; Al-Obeidi, Fahad; Sawyer, Tomi K.; Staples, Douglas J.; DeVaux, Ann E.; Dym, Orin; Castrucci, Ana Maria de L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2126-30</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A series of fragment analogs of α-MSH (I) were prepd. in order to det. the contribution of each individual amino acid to the biol. activity of the native hormone.  The minimal potency of Ac-α-MSH6-9-NH2 could be enhanced about a factor of 16 by the addn. of glycine to the C-terminus, yielding Ac-α-MSH6-10-NH2.  Addn. of glutamic acid to the N-terminus provided Ac-α-MSH5-10-NH2, which was only slightly more potent than Ac-α-MSH6-10-NH2, indicating that position 5 contributes little to the biol. potency of α-MSH in this assay.  Addn. of methionine to the N-terminus of Ac-α-MSH5-10-NH2 resulted in Ac-α-MSH4-10-NH2, which was only about 4-fold more potent than Ac-α-MSH5-10-NH2.  Addn. of lysine and proline to the C-terminal of the Ac-α-MSH4-10-NH2 sequence yielded Ac-α-MSH4-12-NH2 with a 360-fold increase in potency relative to Ac-α-MSH4-10-NH2.  This peptide was only about 6-fold less potent than α-MSH.  Nle-4-substituted analogs were also prepd.  Ac-[Nle4]-α-MSH4-10-NH2 and Ac-[Nle4]-α-MSH4-11-NH2 were ∼4 times more potent than Ac-α-MSH4-10-NH2, demonstrating that lysine-11 contributes somewhat to the biol. activity of α-MSH on the frog skin melanocyte receptor.  However, addn. of proline-12 to this fragment, yielding Ac-[Nle4]-α-MSH4-12-NH2, resulted in about a 90-fold increase in relative potency of the melanotropin.  Addn. of the final C-terminal valine-13 provided Ac-[Nle4]-α-MSH4-13-NH2, which showed only a small further increase in potency.  This analog was, however, only about 2 to 3-fold less active than α-MSH.  Addn. of the N-terminal tripeptide Ac-Ser-Tyr-Ser to yield [Nle4]-α-MSH resulted in an analog that was 3 times more potent than α-MSH.  The central tetrapeptide sequence, Ac-His-Phe-Arg-Trp-NH2, represents the min. chain length for observable biol. activity.  The active sequence of α-MSH is contiguous in that no two structurally noncontiguous fragment sequences were found to have biol. activity.  Met-4, Gly-10, and Pro-12 are important potentiating amino acids and contribute significantly to the biopotency of α-MSH, and Ser-1 and -3, Tyr-2, Glu-5, Lys-11, and Val-13 apparently contribute only minimally to the biol. potency of α-MSH at the frog melanocyte skin receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqNbLyfhM07Vg90H21EOLACvtfcHk0liZh1ewG-6Wjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXlslKntLk%253D&md5=453a1b54b73e07414b096237b3bdcbdc</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm00394a033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00394a033%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26aulast%3DWilkes%26aufirst%3DB.%2BC.%26aulast%3DHadley%26aufirst%3DM.%2BE.%26aulast%3DAl-Obeidi%26aufirst%3DF.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DStaples%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BVaux%26aufirst%3DA.%2BE.%26aulast%3DDym%26aufirst%3DO.%26aulast%3DCastrucci%26aufirst%3DA.%2BM.%26aulast%3DHintz%26aufirst%3DM.%2BF.%26atitle%3D.alpha.-Melanotropin%253A%2520The%2520Minimal%2520Active%2520Sequence%2520in%2520the%2520Frog%2520Skin%2520Bioassay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1987%26volume%3D30%26spage%3D2126%26epage%3D2130%26doi%3D10.1021%2Fjm00394a033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslach, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskell-Luevano, C.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships (SAR) of Melanocortin and Agouti-Related (AGRP) Peptides</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>681</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1007/978-1-4419-6354-3_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2F978-1-4419-6354-3_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=21222256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WrsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=681&publication_year=2010&pages=1-18&author=A.+Singhauthor=E.+M.+Haslachauthor=C.+Haskell-Luevano&title=Structure-Activity+Relationships+%28SAR%29+of+Melanocortin+and+Agouti-Related+%28AGRP%29+Peptides&doi=10.1007%2F978-1-4419-6354-3_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relatinships (SAR) of melanocortin and agouti-related (AGRP) peptides</span></div><div class="casAuthors">Singh, Anamika; Haslach, Erica M.; Haskell-Luevano, Carrie</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">681</span>
        (<span class="NLM_cas:issue">Melanocortins</span>),
    <span class="NLM_cas:pages">1-18</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Structure-activity relationship (SAR) studies are a key feature of peptide and peptidomimetic research to improve the biol. properties of native peptides and convert them into more drug-like compds.  Peptide SAR studies involve the systematic modification of a lead peptide to provide insight into the mol. determinants of the ligand-receptor interactions that result in either receptor stimulation or inhibition.  This chapter will discuss structure-activity relationships of the endogenous and synthetic agonists and the antagonists of the melanocortin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ4R5zOYjvFLVg90H21EOLACvtfcHk0liZh1ewG-6Wjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WrsrnL&md5=936828fae7e953678d05fd26a712d31e</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-6354-3_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-6354-3_1%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DHaslach%26aufirst%3DE.%2BM.%26aulast%3DHaskell-Luevano%26aufirst%3DC.%26atitle%3DStructure-Activity%2520Relationships%2520%2528SAR%2529%2520of%2520Melanocortin%2520and%2520Agouti-Related%2520%2528AGRP%2529%2520Peptides%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2010%26volume%3D681%26spage%3D1%26epage%3D18%26doi%3D10.1007%2F978-1-4419-6354-3_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span> <span> </span><span class="NLM_article-title">Venom Peptides as a Rich Source of Cav2.2 Channel Blockers</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.3390/toxins5020286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Ftoxins5020286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23381143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVWisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=286-314&author=S.+R.+Sousaauthor=I.+Vetterauthor=R.+J.+Lewis&title=Venom+Peptides+as+a+Rich+Source+of+Cav2.2+Channel+Blockers&doi=10.3390%2Ftoxins5020286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Venom peptides as a rich source of Cav2.2 channel blockers</span></div><div class="casAuthors">Sousa, Silmara R.; Vetter, Irina; Lewis, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">286-314</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Cav2.2 is a calcium channel subtype localized at nerve terminals, including nociceptive fibers, where it initiates neurotransmitter release.  Cav2.2 is an important contributor to synaptic transmission in ascending pain pathways, and is up-regulated in the spinal cord in chronic pain states along with the auxiliary α2δ1 subunit.  It is therefore not surprising that toxins that inhibit Cav2.2 are analgesic.  Venomous animals, such as cone snails, spiders, snakes, assassin bugs, centipedes and scorpions are rich sources of remarkably potent and selective Cav2.2 inhibitors.  However, side effects in humans currently limit their clin. use.  Here we review Cav2.2 inhibitors from venoms and their potential as drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP3nbJeRZPY7Vg90H21EOLACvtfcHk0liZh1ewG-6Wjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVWisb8%253D&md5=2f700d274eeab82e73ea7a9a72126ff0</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.3390%2Ftoxins5020286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins5020286%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DS.%2BR.%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26atitle%3DVenom%2520Peptides%2520as%2520a%2520Rich%2520Source%2520of%2520Cav2.2%2520Channel%2520Blockers%26jtitle%3DToxins%26date%3D2013%26volume%3D5%26spage%3D286%26epage%3D314%26doi%3D10.3390%2Ftoxins5020286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasparini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danse, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecoq, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkasfeld, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn-Justin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Medeiros, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span> <span> </span><span class="NLM_article-title">Delineation of the Functional Site of α-Dendrotoxin. The Functional Topographies of Dendrotoxins Are Different but Share a Conserved Core with Those of Other Kv1 Potassium Channel-Blocking Toxins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">25393</span>– <span class="NLM_lpage">25403</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.39.25393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.273.39.25393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9738007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1cXmsVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=25393-25403&author=S.+Gaspariniauthor=J.+M.+Danseauthor=A.+Lecoqauthor=S.+Pinkasfeldauthor=S.+Zinn-Justinauthor=L.+C.+Youngauthor=C.+C.+de+Medeirosauthor=E.+G.+Rowanauthor=A.+L.+Harveyauthor=A.+M%C3%A9nez&title=Delineation+of+the+Functional+Site+of+%CE%B1-Dendrotoxin.+The+Functional+Topographies+of+Dendrotoxins+Are+Different+but+Share+a+Conserved+Core+with+Those+of+Other+Kv1+Potassium+Channel-Blocking+Toxins&doi=10.1074%2Fjbc.273.39.25393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Delineation of the functional site of α-dendrotoxin. The functional topographies of dendrotoxins are different but share a conserved core with those of other Kv1 potassium channel-blocking toxins</span></div><div class="casAuthors">Gasparini, Sylvaine; Danse, Jean-Marc; Lecoq, Alain; Pinkasfeld, Suzanne; Zinn-Justin, Sophie; Young, Louise C.; De Medeiros, Cleane C. L.; Rowan, Edward G.; Harvey, Alan L.; Menez, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">25393-25403</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Residues important for the binding of α-dendrotoxin (αDTX) to Kv1 potassium channels on rat brain synaptosomal membranes were identified using a mutational approach based on site-directed mutagenesis and chem. synthesis.  Twenty-six of its 59 residues were individually substituted by alanine.  Substitutions of Lys5 and Leu9 decreased affinity more than 1000-fold, and substitutions of Arg3, Arg4, Leu6, and Ile8 by 5-30-fold.  Substitution of Lys5 by norleucine or ornithine also greatly altered the binding properties of αDTX.  All of these analogs displayed similar CD spectra as compared with the wild-type αDTX, indicating that none of these substitutions affect the overall conformation of the toxin.  Substitutions of Ser38 and Arg46 also reduced the affinity of the toxin but, in addn., modified its dichroic properties, suggesting that these two residues play a structural role.  The other residues were excluded from the recognition site because their substitutions caused no significant affinity change.  Thus, the functional site of αDTX includes six major binding residues, all located in its N-terminal region, with Lys5 and Leu9 being the most important.  Comparison of the functional site of αDTX with that of DTX-K, another dendrotoxin (Smith, L. A., Reid, P. F., Wang, F. C., Parcej, D. N., Schmidt, J. J., Olson, M. A., and Dolly, J. O. (1997) Biochem. 36, 7690-7696), reveals that they only share the predominant lysine and probably a leucine residue; the addnl. functional residues differ from one toxin to the other.  Comparison of the functional site of αDTX with those of structurally unrelated potassium channel-blocking toxins from venomous invertebrates revealed the common presence of a protruding key lysine with a close important hydrophobic residue (Leu, Tyr, or Phe) and few addnl. residues.  Therefore, irresp. of their phylogenetic origin, all of these toxins may have undergone a functional convergence.  The functional site of αDTX is topog. unrelated to the "antiprotease site" of the structurally analogous bovine pancreatic trypsin inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB9cTJoyCsdbVg90H21EOLACvtfcHk0lhIoHaQ_z8mZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsVGmu7g%253D&md5=1ce3da14c231c642ebaae4e42bdcc05d</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.39.25393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.39.25393%26sid%3Dliteratum%253Aachs%26aulast%3DGasparini%26aufirst%3DS.%26aulast%3DDanse%26aufirst%3DJ.%2BM.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DPinkasfeld%26aufirst%3DS.%26aulast%3DZinn-Justin%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DL.%2BC.%26aulast%3Dde%2BMedeiros%26aufirst%3DC.%2BC.%26aulast%3DRowan%26aufirst%3DE.%2BG.%26aulast%3DHarvey%26aufirst%3DA.%2BL.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26atitle%3DDelineation%2520of%2520the%2520Functional%2520Site%2520of%2520%25CE%25B1-Dendrotoxin.%2520The%2520Functional%2520Topographies%2520of%2520Dendrotoxins%2520Are%2520Different%2520but%2520Share%2520a%2520Conserved%2520Core%2520with%2520Those%2520of%2520Other%2520Kv1%2520Potassium%2520Channel-Blocking%2520Toxins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D25393%26epage%3D25403%26doi%3D10.1074%2Fjbc.273.39.25393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dauplais, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecoq, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilquin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roumestand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vita, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Medeiros, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span> <span> </span><span class="NLM_article-title">On the Convergent Evolution of Animal Toxins. Conservation of a Diad of Functional Residues in Potassium Channel-Blocking Toxins with Unrelated Structures</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">4302</span>– <span class="NLM_lpage">4309</span>, <span class="refDoi"> DOI: 10.1074/jbc.272.7.4302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.272.7.4302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9020148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2sXht1Oqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=4302-4309&author=M.+Dauplaisauthor=A.+Lecoqauthor=J.+Songauthor=J.+Cottonauthor=N.+Jaminauthor=B.+Gilquinauthor=C.+Roumestandauthor=C.+Vitaauthor=C.+L.+de+Medeirosauthor=E.+G.+Rowanauthor=A.+L.+Harveyauthor=A.+M%C3%A9nez&title=On+the+Convergent+Evolution+of+Animal+Toxins.+Conservation+of+a+Diad+of+Functional+Residues+in+Potassium+Channel-Blocking+Toxins+with+Unrelated+Structures&doi=10.1074%2Fjbc.272.7.4302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">On the convergent evolution of animal toxins. Conservation of a diad of functional residues in potassium channel-blocking toxins with unrelated structures</span></div><div class="casAuthors">Dauplais, Marc; Lecoq, Alain; Song, Jianxing; Cotton, Joel; Jamin, Nadege; Gilquin, Bernard; Roumestand, Christian; Vita, Claudio; de Medeiros, Cleane L. C.; Rowan, Edward G.; Harvey, Alan L.; Menez, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4302-4309</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">BgK is a K+ channel-blocking toxin from the sea anemone Bunodosoma granulifera.  It is a 37-residue protein that adopts a novel fold, as detd. by NMR and modeling.  An alanine-scanning-based anal. revealed the functional importance of five residues, which include a crit. lysine and an arom. residue sepd. by 6.6±1.0 Å.  The same diad is found in the three known homologous toxins from sea anemones.  More strikingly, a similar functional diad is present in all K+ channel-blocking toxins from scorpions, although these toxins adopt a distinct scaffold.  Moreover, the functional diads of potassium channel-blocking toxins from sea anemone and scorpions superimpose in the three-dimensional structures.  Therefore, toxins that have unrelated structures but similar functions possess conserved key functional residues, organized in an identical topol., suggesting a convergent functional evolution for these small proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh-osMmt309bVg90H21EOLACvtfcHk0lhIoHaQ_z8mZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1Oqsbs%253D&md5=151c3834341c30ba3c901c69854ab5f8</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.7.4302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.7.4302%26sid%3Dliteratum%253Aachs%26aulast%3DDauplais%26aufirst%3DM.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DCotton%26aufirst%3DJ.%26aulast%3DJamin%26aufirst%3DN.%26aulast%3DGilquin%26aufirst%3DB.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DVita%26aufirst%3DC.%26aulast%3Dde%2BMedeiros%26aufirst%3DC.%2BL.%26aulast%3DRowan%26aufirst%3DE.%2BG.%26aulast%3DHarvey%26aufirst%3DA.%2BL.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26atitle%3DOn%2520the%2520Convergent%2520Evolution%2520of%2520Animal%2520Toxins.%2520Conservation%2520of%2520a%2520Diad%2520of%2520Functional%2520Residues%2520in%2520Potassium%2520Channel-Blocking%2520Toxins%2520with%2520Unrelated%2520Structures%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D4302%26epage%3D4309%26doi%3D10.1074%2Fjbc.272.7.4302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zerbino, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birney, E.</span></span> <span> </span><span class="NLM_article-title">Velvet: Algorithms for de Novo Short Read Assembly Using de Bruijn Graphs</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">829</span>, <span class="refDoi"> DOI: 10.1101/gr.074492.107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1101%2Fgr.074492.107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=18349386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslChsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=821-829&author=D.+R.+Zerbinoauthor=E.+Birney&title=Velvet%3A+Algorithms+for+de+Novo+Short+Read+Assembly+Using+de+Bruijn+Graphs&doi=10.1101%2Fgr.074492.107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</span></div><div class="casAuthors">Zerbino, Daniel R.; Birney, Ewan</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">821-829</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We have developed a new set of algorithms, collectively called "Velvet," to manipulate de Bruijn graphs for genomic sequence assembly.  A de Bruijn graph is a compact representation based on short words (k-mers) that is ideal for high coverage, very short read (25-50 bp) data sets.  Applying Velvet to very short reads and paired-ends information only, one can produce contigs of significant length, up to 50-kb N50 length in simulations of prokaryotic data and 3-kb N50 on simulated mammalian BACs.  When applied to real Solexa data sets without read pairs, Velvet generated contigs of ∼8 kb in a prokaryote and 2 kb in a mammalian BAC, in close agreement with our simulated results without read-pair information.  Velvet represents a new approach to assembly that can leverage very short reads in combination with read pairs to produce useful assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7wYfw5TD8frVg90H21EOLACvtfcHk0lhIoHaQ_z8mZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslChsLg%253D&md5=2378b6c6efad2611e0eadbd86f77e01f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1101%2Fgr.074492.107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.074492.107%26sid%3Dliteratum%253Aachs%26aulast%3DZerbino%26aufirst%3DD.%2BR.%26aulast%3DBirney%26aufirst%3DE.%26atitle%3DVelvet%253A%2520Algorithms%2520for%2520de%2520Novo%2520Short%2520Read%2520Assembly%2520Using%2520de%2520Bruijn%2520Graphs%26jtitle%3DGenome%2520Res.%26date%3D2008%26volume%3D18%26spage%3D821%26epage%3D829%26doi%3D10.1101%2Fgr.074492.107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grabherr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yassour, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amit, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adiconis, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raychowdhury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauceli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hacohen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnirke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhind, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Palma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birren, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusbaum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindblad-Toh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span> <span> </span><span class="NLM_article-title">Full-Length Transcriptome Assembly from RNA-Seq Data without a Reference Genome</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">644</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1038/nbt.1883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1038%2Fnbt.1883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=21572440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtV2hsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=644-652&author=M.+G.+Grabherrauthor=B.+J.+Haasauthor=M.+Yassourauthor=J.+Z.+Levinauthor=D.+A.+Thompsonauthor=I.+Amitauthor=X.+Adiconisauthor=L.+Fanauthor=R.+Raychowdhuryauthor=Q.+Zengauthor=Z.+Chenauthor=E.+Mauceliauthor=N.+Hacohenauthor=A.+Gnirkeauthor=N.+Rhindauthor=F.+di+Palmaauthor=B.+W.+Birrenauthor=C.+Nusbaumauthor=K.+Lindblad-Tohauthor=N.+Friedmanauthor=A.+Regev&title=Full-Length+Transcriptome+Assembly+from+RNA-Seq+Data+without+a+Reference+Genome&doi=10.1038%2Fnbt.1883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span></div><div class="casAuthors">Grabherr, Manfred G.; Haas, Brian J.; Yassour, Moran; Levin, Joshua Z.; Thompson, Dawn A.; Amit, Ido; Adiconis, Xian; Fan, Lin; Raychowdhury, Raktima; Zeng, Qiandong; Chen, Zehua; Mauceli, Evan; Hacohen, Nir; Gnirke, Andreas; Rhind, Nicholas; di Palma, Federica; Birren, Bruce W.; Nusbaum, Chad; Lindblad-Toh, Kerstin; Friedman, Nir; Regev, Aviv</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">644-652</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Massively parallel sequencing of cDNA has enabled deep and efficient probing of transcriptomes.  Current approaches for transcript reconstruction from such data often rely on aligning reads to a ref. genome, and are thus unsuitable for samples with a partial or missing ref. genome.  Here we present the Trinity method for de novo assembly of full-length transcripts and evaluate it on samples from fission yeast, mouse and whitefly, whose ref. genome is not yet available.  By efficiently constructing and analyzing sets of de Bruijn graphs, Trinity fully reconstructs a large fraction of transcripts, including alternatively spliced isoforms and transcripts from recently duplicated genes.  Compared with other de novo transcriptome assemblers, Trinity recovers more full-length transcripts across a broad range of expression levels, with a sensitivity similar to methods that rely on genome alignments.  Our approach provides a unified soln. for transcriptome reconstruction in any sample, esp. in the absence of a ref. genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprN3mUm_p2wbVg90H21EOLACvtfcHk0lg8KuKgjL18tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtV2hsbc%253D&md5=975f291debee3f027d32c13688b57f1b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1883%26sid%3Dliteratum%253Aachs%26aulast%3DGrabherr%26aufirst%3DM.%2BG.%26aulast%3DHaas%26aufirst%3DB.%2BJ.%26aulast%3DYassour%26aufirst%3DM.%26aulast%3DLevin%26aufirst%3DJ.%2BZ.%26aulast%3DThompson%26aufirst%3DD.%2BA.%26aulast%3DAmit%26aufirst%3DI.%26aulast%3DAdiconis%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DRaychowdhury%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMauceli%26aufirst%3DE.%26aulast%3DHacohen%26aufirst%3DN.%26aulast%3DGnirke%26aufirst%3DA.%26aulast%3DRhind%26aufirst%3DN.%26aulast%3Ddi%2BPalma%26aufirst%3DF.%26aulast%3DBirren%26aufirst%3DB.%2BW.%26aulast%3DNusbaum%26aufirst%3DC.%26aulast%3DLindblad-Toh%26aufirst%3DK.%26aulast%3DFriedman%26aufirst%3DN.%26aulast%3DRegev%26aufirst%3DA.%26atitle%3DFull-Length%2520Transcriptome%2520Assembly%2520from%2520RNA-Seq%2520Data%2520without%2520a%2520Reference%2520Genome%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D644%26epage%3D652%26doi%3D10.1038%2Fnbt.1883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozach, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blemont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">High Throughput Quantitative Expression Screening and Purification Applied to Recombinant Disulfide-Rich Venom Proteins Produced in <i>E. Coli</i></span>. <i>J. Visualized Exp.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>89</i></span>, <span class="NLM_elocation-id">e51464</span> <span class="refDoi"> DOI: 10.3791/51464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3791%2F51464" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2014&author=N.+J.+Saezauthor=H.+Nozachauthor=M.+Blemontauthor=R.+Vincentelli&title=High+Throughput+Quantitative+Expression+Screening+and+Purification+Applied+to+Recombinant+Disulfide-Rich+Venom+Proteins+Produced+in+E.+Coli&doi=10.3791%2F51464"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.3791%2F51464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F51464%26sid%3Dliteratum%253Aachs%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DNozach%26aufirst%3DH.%26aulast%3DBlemont%26aufirst%3DM.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DHigh%2520Throughput%2520Quantitative%2520Expression%2520Screening%2520and%2520Purification%2520Applied%2520to%2520Recombinant%2520Disulfide-Rich%2520Venom%2520Proteins%2520Produced%2520in%2520E.%2520Coli%26jtitle%3DJ.%2520Visualized%2520Exp.%26date%3D2014%26volume%3D89%26doi%3D10.3791%2F51464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vranken, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fogh, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pajon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llinas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markley, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ionides, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laue, E. D.</span></span> <span> </span><span class="NLM_article-title">The CCPN Data Model for NMR Spectroscopy: Development of a Software Pipeline</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">696</span>, <span class="refDoi"> DOI: 10.1002/prot.20449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1002%2Fprot.20449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=15815974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2005&pages=687-696&author=W.+F.+Vrankenauthor=W.+Boucherauthor=T.+J.+Stevensauthor=R.+H.+Foghauthor=A.+Pajonauthor=M.+Llinasauthor=E.+L.+Ulrichauthor=J.+L.+Markleyauthor=J.+Ionidesauthor=E.+D.+Laue&title=The+CCPN+Data+Model+for+NMR+Spectroscopy%3A+Development+of+a+Software+Pipeline&doi=10.1002%2Fprot.20449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The CCPN data model for NMR spectroscopy: development of a software pipeline</span></div><div class="casAuthors">Vranken, Wim F.; Boucher, Wayne; Stevens, Tim J.; Fogh, Rasmus H.; Pajon, Anne; Llinas, Miguel; Ulrich, Eldon L.; Markley, John L.; Ionides, John; Laue, Ernest D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-696</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">To address data management and data exchange problems in the NMR community, the Collaborative Computing Project for the NMR community (CCPN) created a "Data Model" that describes all the different types of information needed in an NMR structural study, from mol. structure and NMR parameters to coordinates.  This paper describes the development of a set of software applications that use the Data Model and its assocd. libraries, thus validating the approach.  These applications are freely available and provide a pipeline for high-throughput anal. of NMR data.  Three programs work directly with the Data Model: CcpNmr Anal., an entirely new anal. and interactive display program, the CcpNmr FormatConverter, which allows transfer of data from programs commonly used in NMR to and from the Data Model, and the CLOUDS software for automated structure calcn. and assignment (Carnegie Mellon University), which was rewritten to interact directly with the Data Model.  The ARIA 2.0 software for structure calcn. (Institut Pasteur) and the QUEEN program for validation of restraints (University of Nijmegen) were extended to provide conversion of their data to the Data Model.  During these developments the Data Model has been thoroughly tested and used, demonstrating that applications can successfully exchange data via the Data Model.  The software architecture developed by CCPN is now ready for new developments, such as integration with addnl. software applications and extensions of the Data Model into other areas of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpl6jPIIOYWbVg90H21EOLACvtfcHk0lg8KuKgjL18tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D&md5=b07b9701f64f281d58d178f9c2033596</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fprot.20449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20449%26sid%3Dliteratum%253Aachs%26aulast%3DVranken%26aufirst%3DW.%2BF.%26aulast%3DBoucher%26aufirst%3DW.%26aulast%3DStevens%26aufirst%3DT.%2BJ.%26aulast%3DFogh%26aufirst%3DR.%2BH.%26aulast%3DPajon%26aufirst%3DA.%26aulast%3DLlinas%26aufirst%3DM.%26aulast%3DUlrich%26aufirst%3DE.%2BL.%26aulast%3DMarkley%26aufirst%3DJ.%2BL.%26aulast%3DIonides%26aufirst%3DJ.%26aulast%3DLaue%26aufirst%3DE.%2BD.%26atitle%3DThe%2520CCPN%2520Data%2520Model%2520for%2520NMR%2520Spectroscopy%253A%2520Development%2520of%2520a%2520Software%2520Pipeline%26jtitle%3DProteins%26date%3D2005%26volume%3D59%26spage%3D687%26epage%3D696%26doi%3D10.1002%2Fprot.20449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rieping, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardiaux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malliavin, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span> <span> </span><span class="NLM_article-title">ARIA2: Automated NOE Assignment and Data Integration in NMR Structure Calculation</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">382</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btl589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1093%2Fbioinformatics%2Fbtl589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=17121777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=381-382&author=W.+Riepingauthor=M.+Habeckauthor=B.+Bardiauxauthor=A.+Bernardauthor=T.+E.+Malliavinauthor=M.+Nilges&title=ARIA2%3A+Automated+NOE+Assignment+and+Data+Integration+in+NMR+Structure+Calculation&doi=10.1093%2Fbioinformatics%2Fbtl589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">ARIA2: Automated NOE assignment and data integration in NMR structure calculation</span></div><div class="casAuthors">Rieping, Wolfgang; Habeck, Michael; Bardiaux, Benjamin; Bernard, Aymeric; Malliavin, Therese E.; Nilges, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-382</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: Modern structural genomics projects demand for integrated methods for the interpretation and storage of NMR data.  Here the authors present version 2.1 of the authors' program ARIA (Ambiguous Restraints for Iterative Assignment) for automated assignment of nuclear Overhauser enhancement (NOE) data and NMR structure calcn.  The authors report on recent developments, most notably a graphical user interface, and the incorporation of the object-oriented data model of the Collaborative Computing Project for NMR (CCPN).  The CCPN data model defines a storage model for NMR data, which greatly facilitates the transfer of data between different NMR software packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2HL9x73wPfrVg90H21EOLACvtfcHk0lg8KuKgjL18tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOltb8%253D&md5=75e604b2faf4375ace7ea455365737f1</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtl589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtl589%26sid%3Dliteratum%253Aachs%26aulast%3DRieping%26aufirst%3DW.%26aulast%3DHabeck%26aufirst%3DM.%26aulast%3DBardiaux%26aufirst%3DB.%26aulast%3DBernard%26aufirst%3DA.%26aulast%3DMalliavin%26aufirst%3DT.%2BE.%26aulast%3DNilges%26aufirst%3DM.%26atitle%3DARIA2%253A%2520Automated%2520NOE%2520Assignment%2520and%2520Data%2520Integration%2520in%2520NMR%2520Structure%2520Calculation%26jtitle%3DBioinformatics%26date%3D2007%26volume%3D23%26spage%3D381%26epage%3D382%26doi%3D10.1093%2Fbioinformatics%2Fbtl589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornilescu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">Protein Backbone Angle Restraints from Searching a Database for Chemical Shift and Sequence Homology</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1023/A:1008392405740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1023%2FA%3A1008392405740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10212987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1MXitlejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=289-302&author=G.+Cornilescuauthor=F.+Delaglioauthor=A.+Bax&title=Protein+Backbone+Angle+Restraints+from+Searching+a+Database+for+Chemical+Shift+and+Sequence+Homology&doi=10.1023%2FA%3A1008392405740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Protein backbone angle restraints from searching a database for chemical shift and sequence homology</span></div><div class="casAuthors">Cornilescu, Gabriel; Delaglio, Frank; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">289-302</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Chem. shifts of backbone atoms in proteins are exquisitely sensitive to local conformation, and homologous proteins show quite similar patterns of secondary chem. shifts.  The inverse of this relation is used to search a database for triplets of adjacent residues with secondary chem. shifts and sequence similarity which provide the best match to the query triplet of interest.  The database contains 13Cα, 13Cβ, 13C', 1Hα and 15N chem. shifts for 20 proteins for which a high resoln. x-ray structure is available.  The computer program TALOS was developed to search this database for strings of residues with chem. shift and residue type homol.  The relative importance of the weighting factors attached to the secondary chem. shifts of the five types of resonances relative to that of sequence similarity was optimized empirically.  TALOS yields the 10 triplets which have the closest similarity in secondary chem. shift and amino acid sequence to those of the query sequence.  If the central residues in these 10 triplets exhibit similar φ and ψ backbone angles, their avs. can reliably be used as angular restraints for the protein whose structure is being studied.  Tests carried out for proteins of known structure indicate that the root-mean-square difference (rmsd) between the output of TALOS and the x-ray derived backbone angles is about 15°.  Approx. 3% of the predictions made by TALOS are found to be in error.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcjgFEXHFQV7Vg90H21EOLACvtfcHk0limhCB7HlUOfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitlejsL8%253D&md5=1083ff7ba22757c8eaa78e9cc507d1db</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008392405740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008392405740%26sid%3Dliteratum%253Aachs%26aulast%3DCornilescu%26aufirst%3DG.%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DProtein%2520Backbone%2520Angle%2520Restraints%2520from%2520Searching%2520a%2520Database%2520for%2520Chemical%2520Shift%2520and%2520Sequence%2520Homology%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1999%26volume%3D13%26spage%3D289%26epage%3D302%26doi%3D10.1023%2FA%3A1008392405740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hutchinson, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">PROMOTIF--a Program to Identify and Analyze Structural Motifs in Proteins</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">212</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1002/pro.5560050204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1002%2Fpro.5560050204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=8745398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK28XhtVagtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1996&pages=212-220&author=E.+G.+Hutchinsonauthor=J.+M.+Thornton&title=PROMOTIF%2D%2Da+Program+to+Identify+and+Analyze+Structural+Motifs+in+Proteins&doi=10.1002%2Fpro.5560050204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">PROMOTIF - a program to identify and analyze structural motifs in proteins</span></div><div class="casAuthors">Hutchinson, E. Gail; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">212-20</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The authors described a suite of programs, PROMOTIF, that analyzes a protein coordinate file and provides details about the structural motifs in the protein.  The program currently analyzes the following structural features: secondary structure; β- and γ-turns; helical geometry and interactions; β-strands and β-sheet topol.; β-bulges; β-hairpins; β-α-β units and ψ-loops; disulfide bridges; and main-chain hydrogen bonding patterns.  PROMOTIF creates postscripts files showing the examples of each type of motif in the protein, and a summary page.  The program can also be used to compare motifs in a group of related structures, such as an ensemble of NMR structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR_hvCDMO8mLVg90H21EOLACvtfcHk0limhCB7HlUOfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtVagtL8%253D&md5=72e3118995037f7b28ca1421e2990585</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560050204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560050204%26sid%3Dliteratum%253Aachs%26aulast%3DHutchinson%26aufirst%3DE.%2BG.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROMOTIF--a%2520Program%2520to%2520Identify%2520and%2520Analyze%2520Structural%2520Motifs%2520in%2520Proteins%26jtitle%3DProtein%2520Sci.%26date%3D1996%26volume%3D5%26spage%3D212%26epage%3D220%26doi%3D10.1002%2Fpro.5560050204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rullmannn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">AQUA and PROCHECK-NMR: Programs for Checking the Quality of Protein Structures Solved by NMR</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1007/BF00228148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2FBF00228148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9008363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1996&pages=477-486&author=R.+A.+Laskowskiauthor=J.+A.+Rullmannnauthor=M.+W.+MacArthurauthor=R.+Kapteinauthor=J.+M.+Thornton&title=AQUA+and+PROCHECK-NMR%3A+Programs+for+Checking+the+Quality+of+Protein+Structures+Solved+by+NMR&doi=10.1007%2FBF00228148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR</span></div><div class="casAuthors">Laskowski, Roman A.; Antoon, J.; Rullmann, C.; Macarthur, Malcolm W.; Kaptein, Robert; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-486</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The AQUA and PROCHECK-NMR programs provide a means of validating the geometry and restraint violations of an ensemble of protein structures solved by soln. NMR.  The outputs include a detailed breakdown of the restraint violations, a no. of plots in PostScript format and summary statistics.  These various analyses indicate both the degree of agreement of the model structures with the exptl. data, and the quality of their geometrical properties.  They are intended to be of use both to support ongoing NMR structure detn. and in the validation of the final results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsafj3zezC3rVg90H21EOLACvtfcHk0liLwfGQUZxsMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D&md5=030956fb35cbfdaab78801c61ba5a3e1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2FBF00228148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00228148%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DRullmannn%26aufirst%3DJ.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DKaptein%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DAQUA%2520and%2520PROCHECK-NMR%253A%2520Programs%2520for%2520Checking%2520the%2520Quality%2520of%2520Protein%2520Structures%2520Solved%2520by%2520NMR%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1996%26volume%3D8%26spage%3D477%26epage%3D486%26doi%3D10.1007%2FBF00228148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldenburg, K. R.</span></span> <span> </span><span class="NLM_article-title">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span>. <i>J. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1177/108705719900400206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=67-73&author=J.+H.+Zhangauthor=T.+D.+Chungauthor=K.+R.+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays&doi=10.1177%2F108705719900400206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2eYCUni3DJD0drntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DChung%26aufirst%3DT.%2BD.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screen.%26date%3D1999%26volume%3D4%26spage%3D67%26epage%3D73%26doi%3D10.1177%2F108705719900400206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the Inhibition Constant (<i>K<sub>I</sub></i>) and the Concentration of Inhibitor Which Causes 50 per Cent Inhibition (<i>I</i><sub>50</sub>) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+Inhibition+Constant+%28KI%29+and+the+Concentration+of+Inhibitor+Which+Causes+50+per+Cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0liLwfGQUZxsMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520Inhibition%2520Constant%2520%2528KI%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520Which%2520Causes%252050%2520per%2520Cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Lima, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span> <span> </span><span class="NLM_article-title">Where Cone Snails and Spiders Meet: Design of Small Cyclic Sodium-Channel Inhibitors</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">3693</span>– <span class="NLM_lpage">3703</span>, <span class="refDoi"> DOI: 10.1096/fj.201801909R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1096%2Ffj.201801909R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=30509130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptl2isbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=3693-3703&author=S.+Peigneurauthor=O.+Chenevalauthor=M.+Maitiauthor=E.+Leipoldauthor=S.+H.+Heinemannauthor=E.+Lescrinierauthor=P.+Herdewijnauthor=M.+E.+De+Limaauthor=D.+J.+Craikauthor=C.+I.+Schroederauthor=J.+Tytgat&title=Where+Cone+Snails+and+Spiders+Meet%3A+Design+of+Small+Cyclic+Sodium-Channel+Inhibitors&doi=10.1096%2Ffj.201801909R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Where cone snails and spiders meet: design of small cyclic sodium-channel inhibitors</span></div><div class="casAuthors">Peigneur, Steve; Cheneval, Olivier; Maiti, Mohitosh; Leipold, Enrico; Heinemann, Stefan H.; Lescrinier, Eveline; Herdewijn, Piet; De Lima, Maria Elena; Craik, David J.; Schroeder, Christina I.; Tytgat, Jan</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">3693-3703</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A13 aa residue voltage-gated sodium (NaV) channel inhibitor peptide, Pn, contg. 2 disulfide bridges was designed by using a chimeric approach.  This approach was based on a common pharmacophore deduced from sequence and secondary structural homol. of 2NaV inhibitors: Conus kinoshitai toxin IIIA, a 14 residue cone snail peptide with 3 disulfide bonds, and Phoneutria nigriventer toxin 1, a 78 residue spider toxin with 7 disulfide bonds.  As with the parent peptides, this novel NaV channel inhibitor was active on NaV1.2.  Through the generation of 3 series of peptide mutants, we investigated the role of key residues and cyclization and their influence on NaV inhibition and subtype selectivity.  Cyclic PnCS1, a 10 residue peptide cyclized via a disulfide bond, exhibited increased inhibitory activity toward therapeutically relevant NaV channel subtypes, including NaV1.7 and NaV1.9, while displaying remarkable serum stability.  These peptides represent the first and the smallest cyclic peptide NaV modulators to date and are promising templates for the development of toxin-based therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcVQf9N4HK07Vg90H21EOLACvtfcHk0lid_T-ehfMmJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptl2isbc%253D&md5=62d7184451dfdaff85c47db8c427f013</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1096%2Ffj.201801909R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201801909R%26sid%3Dliteratum%253Aachs%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DMaiti%26aufirst%3DM.%26aulast%3DLeipold%26aufirst%3DE.%26aulast%3DHeinemann%26aufirst%3DS.%2BH.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DHerdewijn%26aufirst%3DP.%26aulast%3DDe%2BLima%26aufirst%3DM.%2BE.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26aulast%3DTytgat%26aufirst%3DJ.%26atitle%3DWhere%2520Cone%2520Snails%2520and%2520Spiders%2520Meet%253A%2520Design%2520of%2520Small%2520Cyclic%2520Sodium-Channel%2520Inhibitors%26jtitle%3DFASEB%2520J.%26date%3D2019%26volume%3D33%26spage%3D3693%26epage%3D3703%26doi%3D10.1096%2Ffj.201801909R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentz, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoubas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, G. M.</span></span> <span> </span><span class="NLM_article-title">A Rational Nomenclature for Naming Peptide Toxins from Spiders and Other Venomous Animals</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2008.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2008.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=18619481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVaisrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=264-276&author=G.+F.+Kingauthor=M.+C.+Gentzauthor=P.+Escoubasauthor=G.+M.+Nicholson&title=A+Rational+Nomenclature+for+Naming+Peptide+Toxins+from+Spiders+and+Other+Venomous+Animals&doi=10.1016%2Fj.toxicon.2008.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">A rational nomenclature for naming peptide toxins from spiders and other venomous animals</span></div><div class="casAuthors">King, Glenn F.; Gentz, Margaret C.; Escoubas, Pierre; Nicholson, Graham M.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-276</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mol. toxinol. research was initially driven by an interest in the small subset of animal toxins that are lethal to humans.  However, the realization that many venomous creatures possess a complex repertoire of bioactive peptide toxins with potential pharmaceutical and agrochem. applications has led to an explosion in the no. of new peptide toxins being discovered and characterized.  Unfortunately, this increased awareness of peptide-toxin diversity has not been matched by the development of a generic nomenclature that enables these toxins to be rationally classified, catalogued, and compared.  In this article, we introduce a rational nomenclature that can be applied to the naming of peptide toxins from spiders and other venomous animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvrDZuDtMdNbVg90H21EOLACvtfcHk0lid_T-ehfMmJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVaisrjE&md5=643b380308cdc16f9369e180fa787837</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2008.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2008.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DG.%2BF.%26aulast%3DGentz%26aufirst%3DM.%2BC.%26aulast%3DEscoubas%26aufirst%3DP.%26aulast%3DNicholson%26aufirst%3DG.%2BM.%26atitle%3DA%2520Rational%2520Nomenclature%2520for%2520Naming%2520Peptide%2520Toxins%2520from%2520Spiders%2520and%2520Other%2520Venomous%2520Animals%26jtitle%3DToxicon%26date%3D2008%26volume%3D52%26spage%3D264%26epage%3D276%26doi%3D10.1016%2Fj.toxicon.2008.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carol M.  Trim</span>, <span class="hlFld-ContribAuthor ">Lee J.  Byrne</span>, <span class="hlFld-ContribAuthor ">Steven A.  Trim</span>. </span><span class="cited-content_cbyCitation_article-title">Utilisation of compounds from venoms in drug discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-66. <a href="https://doi.org/10.1016/bs.pmch.2021.01.001" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.001%26sid%3Dliteratum%253Aachs%26atitle%3DUtilisation%252Bof%252Bcompounds%252Bfrom%252Bvenoms%252Bin%252Bdrug%252Bdiscovery%26aulast%3DTrim%26aufirst%3DCarol%2BM.%26date%3D2021%26spage%3D1%26epage%3D66%26pub%3DElsevier%26date%3D2021%26volume%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Richness of animal origin and structural features (number of disulfide bridges, length, and post-translational modification) of the Venomics bank (lower panel) compared to the original Omics bank (upper panel). The Omics bank represents 20,206 sequences identified both in transcriptomic analysis of the venom glands and proteomic analysis of the venoms. From it, 3597 peptides were produced to build the Venomics bank, which closely replicates the origin and structural characteristics of the primary Omics bank.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Screening results on hMC4R from the two plates containing N-TRTX-Preg1a (A) and N-BUTX-Ptr1a (B) peptides in the Venomics bank. Screening on hMC4R, stably expressed in CHO cells, was based on competitive binding on the receptor of the tested product against the radiolabeled ligand, <sup>125</sup>I-NDP-α-MSH. The full red line is the median, and the red dashed lines are median ± 3 median absolute deviation. Dots above or under the dashed lines are hits. Each dot corresponds to a tested Venomics peptide (80 peptides by plate). The colored dot represents either N-TRTX-Preg1a (in green) or N-BUTX-Ptr1a (in purple). Preg1a: N-TRTX-Preg1a. Ptr1a: N-BUTX-Ptr1a.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analytical HPLC chromatogram of the final purified products of N-TRTX-Preg1a and N-BUTX-Ptr1a. Between 10 and 15 μg of each toxin were analyzed on a X Bridge BEH 300 Å 5 μm 4.6 × 250 mm with a gradient of 10–50% CH<sub>3</sub>CN/TFA 0.1% in 40 min at a flow rate of 1 mL/min. A spontaneous transformation of the N-terminal glutamine (Q form) residue of the peptide N-BUTX-Ptr1a into a pyroglutamic acid (Z form) can be detected after crude peptide deprotection. According to LC–MS analysis performed using an Elute Sp Pump HPG 700 (Bruker) coupled with an AmaZon SL LC/MS (Bruker), the peak at 18.077 min corresponds to the Q form (average mass of 3795.70 <i>m</i>/<i>z</i>) and represents 96.5% of the total area. The peak at 18.802 min with an average mass of 3778.70 <i>m</i>/<i>z</i> seems to correspond to the Z form of the peptide and represent 3.5% of the total area.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacological characterization of N-TRTX-Preg1a and N-BUTX-Ptr1a. Representative curves of binding of <sup>125</sup>I-NDP-α-MSH on membrane preparations of hMC1R, hMC4R, and hMC5R (stably expressed in CHO cells) and on hMC3R (stably expressing in HEK293 cells) in the presence of N-TRTX-Preg1a (left panel) or N-BUTX-Ptr1a (right panel). Mean (%) ± SD (<i>n</i> = 2–3). Open squares represent peptides with alkylated cysteines by iodoacetamide treatment, tested on hMC1R and hMC4R (A). Functional assays on HEK293 (gray), HEK293 stably expressing hMC1R (blue), or HEK293 transitory expressing hMC4R (red). cAMP production expressed as % of the positive control (1 μM forskolin) (B). Preg1a: N-TRTX-Preg1a. Ptr1a: N-BUTX-Ptr1a. Mean (%) ± SEM (<i>n</i> = 3), ***: <i>p</i> < 0.001; one-way analysis of variance (ANOVA) followed by Dunnett’s multiple comparison test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structural characterization of N-TRTX-Preg1a and N-BUTX-Ptr1a. Primary structures and overlay of the 10 NMR models representative of the solution structures of N-TRTX-Preg1a (A) and N-BUTX-Ptr1a (B). N and C termini are indicated. Comparison of amino acid sequences between N-TRTX-Preg1a (C) or N-BUTX-Ptr1a (D) and the most similar toxins obtained with the NCBI Protein Blast program (default settings).<a onclick="showRef(event, 'ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43 ref44">(35−44)</a> Cysteine residues are in bold, and conserved residues from N-TRTX-Preg1a or N-BUTX-Ptr1a are in green. Sequence alignment was performed with Clustal Omega (version 1.2.4 from Emboss programs, EBlosum62 matrix for two-pair alignment).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/medium/jm0c00485_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Structure comparison of the spider and scorpion toxins with melanocortin receptor structured peptide ligands. N-TRTX-Preg1a versus human Agouti-related protein 83–132 (hAgRP 83–132). Upper panel: primary (Needl, version 6.6.0 from Emboss programs, EBlosum62 matrix) and secondary structure (arrow: β-sheet) alignment. Cysteine residues and associated disulfide connectivity are in bold and orange when implied in the ICK motif or in bold and black otherwise. The hAgRP Arg-Phe-Phe is underlined. Lower panel: overlay of N-TRTX-Preg1a and hAgRP 87–132 (gray, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HYK">1HYK</a><a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a>) NMR structures. The aligned ICK part of hAgRP is represented in colors. The hAgRP pharmacophore residues (Arg-Phe-Phe) have developed side chains (A). N-BUTX-Ptr1a (Ptr1a) versus human β-defensin 3 (HBD3). Upper panel: primary and secondary structures (arrow: β-sheet, tube: α-helix). Cysteine residues and associated disulfide connectivity are in orange. Basic residues are in blue. Lower panel: tertiary NMR structure comparison of N-BUTX-Ptr1a and HBD3 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KJ6">1KJ6</a><a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a>). Structures display the solvent-exposed surface area with basic residues in blue. “C” and “N” correspond to the N- and C-termini of the peptides, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-15/acs.jmedchem.0c00485/20200806/images/large/jm0c00485_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00485&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88248" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88248" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 78 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenner, R. A.</span>; <span class="NLM_string-name">Undheim, E.</span></span> <i>Venom: The Secrets of Nature’s Deadliest Weapon</i>; <span class="NLM_publisher-name">Smithsonian Books</span>: <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=R.+A.+Jenner&author=E.+Undheim&title=Venom%3A+The+Secrets+of+Nature%E2%80%99s+Deadliest+Weapon"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJenner%26aufirst%3DR.%2BA.%26btitle%3DVenom%253A%2520The%2520Secrets%2520of%2520Nature%25E2%2580%2599s%2520Deadliest%2520Weapon%26pub%3DSmithsonian%2520Books%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Degueldre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdenaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legarda, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minambres, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuniga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducancel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span> <span> </span><span class="NLM_article-title">Diversity in Sequences, Post-Translational Modifications and Expected Pharmacological Activities of Toxins from Four <i>Conus</i> Species Revealed by the Combination of Cutting-Edge Proteomics, Transcriptomics and Bioinformatics</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2017.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2017.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28223047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktVKlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2017&pages=116-125&author=M.+Degueldreauthor=M.+Verdenaudauthor=G.+Legardaauthor=R.+Minambresauthor=S.+Zunigaauthor=M.+Leblancauthor=N.+Gillesauthor=F.+Ducancelauthor=E.+De+Pauwauthor=L.+Quinton&title=Diversity+in+Sequences%2C+Post-Translational+Modifications+and+Expected+Pharmacological+Activities+of+Toxins+from+Four+Conus+Species+Revealed+by+the+Combination+of+Cutting-Edge+Proteomics%2C+Transcriptomics+and+Bioinformatics&doi=10.1016%2Fj.toxicon.2017.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity in sequences, post-translational modifications and expected pharmacological activities of toxins from four Conus species revealed by the combination of cutting-edge proteomics, transcriptomics and bioinformatics</span></div><div class="casAuthors">Degueldre, Michel; Verdenaud, Marion; Legarda, Garikoitz; Minambres, Rebeca; Zuniga, Sheila; Leblanc, Mathieu; Gilles, Nicolas; Ducancel, Frederic; De Pauw, Edwin; Quinton, Loic</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">116-125</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Venomous animals have developed a huge arsenal of reticulated peptides for defense and predation.  Based on various scaffolds, they represent a colossal pharmacol. diversity, making them top candidates for the development of innovative drugs.  Instead of relying on the classical, low-throughput bioassay-guided approach to identify innovative bioactive peptides, this work exploits a recent paradigm to access to venom diversity.  This strategy bypasses the classical approach by combining high-throughput transcriptomics, proteomics and bioinformatics cutting-edge technologies to generate reliable peptide sequences.  The strategy employed to generate hundreds of reliable sequences from Conus venoms is deeply described.  The study led to the discovery of (i) conotoxins that belong to known pharmacol. families targeting various GPCRs or ion-gated channels, and (ii) new families of conotoxins, never described to date.  It also focusses on the diversity of genes, sequences, folds, and PTM's provided by such species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvTuilVaXFwrVg90H21EOLACvtfcHk0lh5znMWhvfrug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktVKlsrg%253D&md5=d2e962030a24f30e49487abe6f11c910</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2017.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2017.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DDegueldre%26aufirst%3DM.%26aulast%3DVerdenaud%26aufirst%3DM.%26aulast%3DLegarda%26aufirst%3DG.%26aulast%3DMinambres%26aufirst%3DR.%26aulast%3DZuniga%26aufirst%3DS.%26aulast%3DLeblanc%26aufirst%3DM.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDucancel%26aufirst%3DF.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DQuinton%26aufirst%3DL.%26atitle%3DDiversity%2520in%2520Sequences%252C%2520Post-Translational%2520Modifications%2520and%2520Expected%2520Pharmacological%2520Activities%2520of%2520Toxins%2520from%2520Four%2520Conus%2520Species%2520Revealed%2520by%2520the%2520Combination%2520of%2520Cutting-Edge%2520Proteomics%252C%2520Transcriptomics%2520and%2520Bioinformatics%26jtitle%3DToxicon%26date%3D2017%26volume%3D130%26spage%3D116%26epage%3D125%26doi%3D10.1016%2Fj.toxicon.2017.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beraud, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandy, K. G.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Potential of Peptide Toxins That Target Ion Channels</span>. <i>Inflamm. Allergy Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">322</span>– <span class="NLM_lpage">342</span>, <span class="refDoi"> DOI: 10.2174/187152811797200696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.2174%2F187152811797200696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=21745182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=322-342&author=E.+Beraudauthor=K.+G.+Chandy&title=Therapeutic+Potential+of+Peptide+Toxins+That+Target+Ion+Channels&doi=10.2174%2F187152811797200696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic potential of peptide toxins that target ion channels</span></div><div class="casAuthors">Beraud, Evelyne; Chandy, K. George</div><div class="citationInfo"><span class="NLM_cas:title">Inflammation & Allergy: Drug Targets</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">322-342</span>CODEN:
                <span class="NLM_cas:coden">IADTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1871-5281</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Traditional healthcare systems in China, India, Greece and the Middle East have for centuries exploited venomous creatures as a resource for medicines.  This review focuses on one class of pharmacol. active compds. from venom, namely peptide toxins that target ion channels.  We highlight their therapeutic potential and the specific channels they target.  The field of therapeutic application is vast, including pain, inflammation, cancer, neurol. disorders, cardioprotection, and autoimmune diseases.  One of these peptides is in clin. use, and many others are in various stages of pre-clin. and clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD27jVncwDZbVg90H21EOLACvtfcHk0lh5znMWhvfrug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlSqsrrL&md5=a04d7eface3437562719693d3be1d75d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F187152811797200696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152811797200696%26sid%3Dliteratum%253Aachs%26aulast%3DBeraud%26aufirst%3DE.%26aulast%3DChandy%26aufirst%3DK.%2BG.%26atitle%3DTherapeutic%2520Potential%2520of%2520Peptide%2520Toxins%2520That%2520Target%2520Ion%2520Channels%26jtitle%3DInflamm.%2520Allergy%2520Drug%2520Targets%26date%3D2011%26volume%3D10%26spage%3D322%26epage%3D342%26doi%3D10.2174%2F187152811797200696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, M. J.</span></span> <span> </span><span class="NLM_article-title">Conus Venom Peptide Pharmacology</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">298</span>, <span class="refDoi"> DOI: 10.1124/pr.111.005322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1124%2Fpr.111.005322" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=22407615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC38XmslKksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2012&pages=259-298&author=R.+J.+Lewisauthor=S.+Dutertreauthor=I.+Vetterauthor=M.+J.+Christie&title=Conus+Venom+Peptide+Pharmacology&doi=10.1124%2Fpr.111.005322"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Conus venom peptide pharmacology</span></div><div class="casAuthors">Lewis, Richard J.; Dutertre, Sebastien; Vetter, Irina; Christie, MacDonald J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">259-298</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">1521-0081</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  Conopeptides are a diverse group of recently evolved venom peptides used for prey capture and/or defense.  Each species of cone snails produces in excess of 1000 conopeptides, with those pharmacol. characterized (∼0.1%) targeting a diverse range of membrane proteins typically with high potency and specificity.  The majority of conopeptides inhibit voltage- or ligand-gated ion channels, providing valuable research tools for the dissection of the role played by specific ion channels in excitable cells.  It is noteworthy that many of these targets are found to be expressed in pain pathways, with several conopeptides having entered the clinic as potential treatments for pain [e.g., pyroglutamate1-MrIA (Xen2174)] and one now marketed for intrathecal treatment of severe pain [ziconotide (Prialt)].  This review discusses the diversity, pharmacol., structure-activity relationships, and therapeutic potential of cone snail venom peptide families acting at voltage-gated ion channels (ω-, μ-, μO-, δ-, ι-, and κ-conotoxins), ligand-gated ion channels (α-conotoxins, σ-conotoxin, ikot-ikot, and conantokins), G-protein-coupled receptors (ρ-conopeptides, conopressins, and contulakins), and neurotransmitter transporters (χ-conopeptides), with expanded discussion on the clin. potential of sodium and calcium channel inhibitors and α-conotoxins.  Expanding the discovery of new bioactives using proteomic/transcriptomic approaches combined with high-throughput platforms and better defining conopeptide structure-activity relationships using relevant membrane protein crystal structures are expected to grow the already significant impact conopeptides have had as both research probes and leads to new therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxAcubmbcLt7Vg90H21EOLACvtfcHk0lh5znMWhvfrug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmslKksbc%253D&md5=fb1f04e0e7b00241326c9be7a8f06859</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1124%2Fpr.111.005322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.111.005322%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DChristie%26aufirst%3DM.%2BJ.%26atitle%3DConus%2520Venom%2520Peptide%2520Pharmacology%26jtitle%3DPharmacol.%2520Rev.%26date%3D2012%26volume%3D64%26spage%3D259%26epage%3D298%26doi%3D10.1124%2Fpr.111.005322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fruchart-Gaillard, C.</span></span> <span> </span><span class="NLM_article-title">Muscarinic Toxins: Tools for the Study of the Pharmacological and Functional Properties of Muscarinic Receptors</span>. <i>J. Neurochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1193</span>– <span class="NLM_lpage">1202</span>, <span class="refDoi"> DOI: 10.1111/j.1471-4159.2009.06092.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1111%2Fj.1471-4159.2009.06092.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19457160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntVCju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2009&pages=1193-1202&author=D.+Serventauthor=C.+Fruchart-Gaillard&title=Muscarinic+Toxins%3A+Tools+for+the+Study+of+the+Pharmacological+and+Functional+Properties+of+Muscarinic+Receptors&doi=10.1111%2Fj.1471-4159.2009.06092.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Muscarinic toxins: tools for the study of the pharmacological and functional properties of muscarinic receptors</span></div><div class="casAuthors">Servent, Denis; Fruchart-Gaillard, Carole</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neurochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1193-1202</span>CODEN:
                <span class="NLM_cas:coden">JONRA9</span>;
        ISSN:<span class="NLM_cas:issn">0022-3042</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Muscarinic receptors mediate metabotropic actions of acetylcholine in the CNS and PNS and autocrine functions of acetylcholine in non-neuronal systems.  Because of the lack of highly selective muscarinic ligands, the precise location, functional role, and roles in various diseases of the five muscarinic receptor subtypes remain unclear.  Muscarinic toxins isolated from the venom of Dendroaspis snakes have a natural high affinity and selectivity, assocd. with roles as competitive antagonists, allosteric modulators, and potential agonists.  These toxins may therefore be invaluable tools for studying muscarinic receptors.  We review data on the structural and pharmacol. characterization of the muscarinic toxins, focusing on recent structure-function studies on toxin-receptor interactions.  We discuss the potential benefits of using these toxins for investigating muscarinic function in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNb6V-N4f51bVg90H21EOLACvtfcHk0liLXFSiNsf_eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntVCju7w%253D&md5=b542ee6ce11c11168de633e805262187</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1471-4159.2009.06092.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1471-4159.2009.06092.x%26sid%3Dliteratum%253Aachs%26aulast%3DServent%26aufirst%3DD.%26aulast%3DFruchart-Gaillard%26aufirst%3DC.%26atitle%3DMuscarinic%2520Toxins%253A%2520Tools%2520for%2520the%2520Study%2520of%2520the%2520Pharmacological%2520and%2520Functional%2520Properties%2520of%2520Muscarinic%2520Receptors%26jtitle%3DJ.%2520Neurochem.%26date%3D2009%26volume%3D109%26spage%3D1193%26epage%3D1202%26doi%3D10.1111%2Fj.1471-4159.2009.06092.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciolek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinfrank, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viengchareun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stura, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigismeau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouillac, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orcel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droctové, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevoux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombès, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzgall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">Green Mamba Peptide Targets Type-2 Vasopressin Receptor against Polycystic Kidney Disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">7154</span>– <span class="NLM_lpage">7159</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620454114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.1620454114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28630289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=7154-7159&author=J.+Ciolekauthor=H.+Reinfrankauthor=L.+Quintonauthor=S.+Viengchareunauthor=E.+A.+Sturaauthor=L.+Veraauthor=S.+Sigismeauauthor=B.+Mouillacauthor=H.+Orcelauthor=S.+Peigneurauthor=J.+Tytgatauthor=L.+Droctov%C3%A9author=F.+Beauauthor=J.+Nevouxauthor=M.+Lomb%C3%A8sauthor=G.+Mourierauthor=E.+De+Pauwauthor=D.+Serventauthor=C.+Mendreauthor=R.+Witzgallauthor=N.+Gilles&title=Green+Mamba+Peptide+Targets+Type-2+Vasopressin+Receptor+against+Polycystic+Kidney+Disease&doi=10.1073%2Fpnas.1620454114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Green mamba peptide targets type-2 vasopressin receptor against polycystic kidney disease</span></div><div class="casAuthors">Ciolek, Justyna; Reinfrank, Helen; Quinton, Loic; Viengchareun, Say; Stura, Enrico A.; Vera, Laura; Sigismeau, Sabrina; Mouillac, Bernard; Orcel, Helene; Peigneur, Steve; Tytgat, Jan; Droctove, Laura; Beau, Fabrice; Nevoux, Jerome; Lombes, Marc; Mourier, Gilles; De Pauw, Edwin; Servent, Denis; Mendre, Christiane; Witzgall, Ralph; Gilles, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">7154-7159</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Polycystic kidney diseases (PKDs) are genetic disorders that can cause renal failure and death in children and adults.  Lowering cAMP in cystic tissues through the inhibition of the type-2 vasopressin receptor (V2R) constitutes a validated strategy to reduce disease progression.  The authors identified a peptide from green mamba venom that exhibits nanomolar affinity for the V2R without any activity on 155 other G-protein-coupled receptors or on 15 ionic channels.  Mambaquaretin-1 is a full antagonist of the V2R activation pathways studied: cAMP prodn., beta-arrestin interaction, and MAP kinase activity.  This peptide adopts the Kunitz fold known to mostly act on potassium channels and serine proteases.  Mambaquaretin-1 interacts selectively with the V2R through its first loop, in the same manner that aprotinin inhibits trypsin.  Injected in mice, mambaquaretin-1 increases in a dose-dependent manner urine outflow with concomitant redn. of urine osmolality, indicating a purely aquaretic effect assocd. with the in vivo blockade of V2R.  CD1-pcy/pcy mice, a juvenile model of PKD, daily treated with 13 μg of mambaquaretin-1 for 99 d, developed less abundant (by 33%) and smaller (by 47%) cysts than control mice.  Neither tachyphylaxis nor apparent toxicity has been noted.  Mambaquaretin-1 represents a promising therapeutic agent against PKDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI-J7A636tYrVg90H21EOLACvtfcHk0liLXFSiNsf_eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N&md5=2cad6950c107df021baf5a66c6b086b0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620454114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620454114%26sid%3Dliteratum%253Aachs%26aulast%3DCiolek%26aufirst%3DJ.%26aulast%3DReinfrank%26aufirst%3DH.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DViengchareun%26aufirst%3DS.%26aulast%3DStura%26aufirst%3DE.%2BA.%26aulast%3DVera%26aufirst%3DL.%26aulast%3DSigismeau%26aufirst%3DS.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DOrcel%26aufirst%3DH.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DDroctov%25C3%25A9%26aufirst%3DL.%26aulast%3DBeau%26aufirst%3DF.%26aulast%3DNevoux%26aufirst%3DJ.%26aulast%3DLomb%25C3%25A8s%26aufirst%3DM.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DMendre%26aufirst%3DC.%26aulast%3DWitzgall%26aufirst%3DR.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DGreen%2520Mamba%2520Peptide%2520Targets%2520Type-2%2520Vasopressin%2520Receptor%2520against%2520Polycystic%2520Kidney%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D7154%26epage%3D7159%26doi%3D10.1073%2Fpnas.1620454114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocking, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biass, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolini-Bertrand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boelens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocklin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span> <span> </span><span class="NLM_article-title">Identification, Structural and Pharmacological Characterization of τ-CnVA, a Conopeptide That Selectively Interacts with Somatostatin Sst<sub>3</sub> Receptor</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">1663</span>– <span class="NLM_lpage">1671</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2013.03.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.bcp.2013.03.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23567999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmsleru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2013&pages=1663-1671&author=C.+Petrelauthor=H.+G.+Hockingauthor=M.+Reynaudauthor=G.+Upertauthor=P.+Favreauauthor=D.+Biassauthor=M.+Paolini-Bertrandauthor=S.+Peigneurauthor=J.+Tytgatauthor=N.+Gillesauthor=O.+Hartleyauthor=R.+Boelensauthor=R.+Stocklinauthor=D.+Servent&title=Identification%2C+Structural+and+Pharmacological+Characterization+of+%CF%84-CnVA%2C+a+Conopeptide+That+Selectively+Interacts+with+Somatostatin+Sst3+Receptor&doi=10.1016%2Fj.bcp.2013.03.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification, structural and pharmacological characterization of τ-CnVA, a conopeptide that selectively interacts with somatostatin sst3 receptor</span></div><div class="casAuthors">Petrel, C.; Hocking, H. G.; Reynaud, M.; Upert, G.; Favreau, Ph.; Biass, D.; Paolini-Bertrand, M.; Peigneur, S.; Tytgat, J.; Gilles, N.; Hartley, O.; Boelens, R.; Stocklin, R.; Servent, D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1663-1671</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Conopeptides are a diverse array of small linear and reticulated peptides that interact with high potency and selectivity with a large diversity of receptors and ion channels.  They are used by cone snails for prey capture or defense.  Recent advances in venom gland transcriptomic and venom peptidomic/proteomic technologies combined with bioactivity screening approaches lead to the identification of new toxins with original pharmacol. profiles.  Here, from transcriptomic/proteomic analyses of the Conus consors cone snail, we identified a new conopeptide called τ-CnVA, which displays the typical cysteine framework V of the T1-conotoxin superfamily.  This peptide was chem. synthesized and its three-dimensional structure was solved by NMR anal. and compared to that of TxVA belonging to the same family, revealing very few common structural features apart a common orientation of the intercysteine loop.  Because of the lack of a clear biol. function assocd. with the T-conotoxin family, τ-CnVA was screened against more than fifty different ion channels and receptors, highlighting its capacity to interact selectively with the somatostatine sst3 receptor.  Pharmacol. and functional studies show that τ-CnVA displays a micromolar (Ki of 1.5 μM) antagonist property for the sst3 receptor, being currently the only known toxin to interact with this GPCR subfamily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdXgulOm8sYLVg90H21EOLACvtfcHk0lgNDw6SEA6ybw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmsleru7g%253D&md5=d10f6a983313c057a3b4cf3213417bd5</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2013.03.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2013.03.019%26sid%3Dliteratum%253Aachs%26aulast%3DPetrel%26aufirst%3DC.%26aulast%3DHocking%26aufirst%3DH.%2BG.%26aulast%3DReynaud%26aufirst%3DM.%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DBiass%26aufirst%3DD.%26aulast%3DPaolini-Bertrand%26aufirst%3DM.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DHartley%26aufirst%3DO.%26aulast%3DBoelens%26aufirst%3DR.%26aulast%3DStocklin%26aufirst%3DR.%26aulast%3DServent%26aufirst%3DD.%26atitle%3DIdentification%252C%2520Structural%2520and%2520Pharmacological%2520Characterization%2520of%2520%25CF%2584-CnVA%252C%2520a%2520Conopeptide%2520That%2520Selectively%2520Interacts%2520with%2520Somatostatin%2520Sst3%2520Receptor%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2013%26volume%3D85%26spage%3D1663%26epage%3D1671%26doi%3D10.1016%2Fj.bcp.2013.03.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rouget, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maïga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamot-Rooke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span> <span> </span><span class="NLM_article-title">Identification of a Novel Snake Peptide Toxin Displaying High Affinity and Antagonist Behaviour for the Α2-Adrenoceptors</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>161</i></span>,  <span class="NLM_fpage">1361</span>– <span class="NLM_lpage">1374</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2010.00966.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1111%2Fj.1476-5381.2010.00966.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20659106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFWhsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=2010&pages=1361-1374&author=C.+Rougetauthor=L.+Quintonauthor=A.+Ma%C3%AFgaauthor=C.+Galesauthor=G.+Masuyerauthor=C.+Malosseauthor=J.+Chamot-Rookeauthor=R.+Thaiauthor=G.+Mourierauthor=E.+De+Pauwauthor=N.+Gillesauthor=D.+Servent&title=Identification+of+a+Novel+Snake+Peptide+Toxin+Displaying+High+Affinity+and+Antagonist+Behaviour+for+the+%CE%912-Adrenoceptors&doi=10.1111%2Fj.1476-5381.2010.00966.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a novel snake peptide toxin displaying high affinity and antagonist behaviour for the α2-adrenoceptors</span></div><div class="casAuthors">Rouget, Celine; Quinton, Loic; Maiga, Arhamatoulaye; Gales, Celine; Masuyer, Geoffrey; Malosse, Christian; Chamot-Rooke, Julia; Thai, Robert; Mourier, Gilles; de Pauw, Edwin; Gilles, Nicolas; Servent, Denis</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1361-1374</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background and Purpose Muscarinic and adrenergic G protein-coupled receptors (GPCRs) are the targets of rare peptide toxins isolated from snake or cone snail venoms.  We used a screen to identify novel toxins from Dendroaspis angusticeps targeting aminergic GPCRs.  These toxins may offer new candidates for the development of new tools and drugs.  Exptl. Approach In binding expts. with 3H-rauwolscine, we studied the interactions of green mamba venom fractions with α2-adrenoceptors from rat brain synaptosomes.  We isolated, sequenced and chem. synthesized a novel peptide, ρ-Da1b.  This peptide was pharmacol. characterized using binding expts. and functional tests on human α2-adrenoceptors expressed in mammalian cells.  Key Results ρ-Da1b, a 66-amino acid peptide stabilized by four disulfide bridges, belongs to the three-finger-fold peptide family.  Its synthetic homolog inhibited 80% of 3H-rauwolscine binding to the three α2-adrenoceptor subtypes, with an affinity between 14 and 73 nM and Hill slopes close to unity.  Functional expts. on α2A-adrenoceptor demonstrated that ρ-Da1b is an antagonist, shifting adrenaline activation curves to the right.  Schild regression revealed slopes of 0.97 and 0.67and pA2 values of 5.93 and 5.32 for yohimbine and ρ-Da1b, resp.  Conclusions and Implications ρ-Da1b is the first toxin identified to specifically interact with α2-adrenoceptors, extending the list of class A GPCRs sensitive to toxins.  Addnl., its affinity and atypical mode of interaction open up the possibility of its use as a new pharmacol. tool, in the study of the physiol. roles of α2-adrenoceptor subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphELeKwRIMpbVg90H21EOLACvtfcHk0lgNDw6SEA6ybw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFWhsrrJ&md5=4fe4691ccc8a09ad202933269e49b893</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2010.00966.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2010.00966.x%26sid%3Dliteratum%253Aachs%26aulast%3DRouget%26aufirst%3DC.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DMa%25C3%25AFga%26aufirst%3DA.%26aulast%3DGales%26aufirst%3DC.%26aulast%3DMasuyer%26aufirst%3DG.%26aulast%3DMalosse%26aufirst%3DC.%26aulast%3DChamot-Rooke%26aufirst%3DJ.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DServent%26aufirst%3DD.%26atitle%3DIdentification%2520of%2520a%2520Novel%2520Snake%2520Peptide%2520Toxin%2520Displaying%2520High%2520Affinity%2520and%2520Antagonist%2520Behaviour%2520for%2520the%2520%25CE%25912-Adrenoceptors%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2010%26volume%3D161%26spage%3D1361%26epage%3D1374%26doi%3D10.1111%2Fj.1476-5381.2010.00966.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiga, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rekik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lluel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuyer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larregola, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciolek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molgó, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fruchart-Gaillard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamot-Rooke, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palea, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">Isolation and Pharmacological Characterization of AdTx1, a Natural Peptide Displaying Specific Insurmountable Antagonism of the α 1A- Adrenoceptor</span>. <i>Br. J.Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">325</span>, <span class="refDoi"> DOI: 10.1111/j.1476-5381.2009.00532.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1111%2Fj.1476-5381.2009.00532.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20015090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhslGju70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2010&pages=316-325&author=L.+Quintonauthor=E.+Girardauthor=A.+Maigaauthor=M.+Rekikauthor=P.+Lluelauthor=G.+Masuyerauthor=M.+Larregolaauthor=C.+Marquerauthor=J.+Ciolekauthor=T.+Magninauthor=R.+Wagnerauthor=J.+Molg%C3%B3author=R.+Thaiauthor=C.+Fruchart-Gaillardauthor=G.+Mourierauthor=J.+Chamot-Rookeauthor=A.+M%C3%A9nezauthor=S.+Paleaauthor=D.+Serventauthor=N.+Gilles&title=Isolation+and+Pharmacological+Characterization+of+AdTx1%2C+a+Natural+Peptide+Displaying+Specific+Insurmountable+Antagonism+of+the+%CE%B1+1A-+Adrenoceptor&doi=10.1111%2Fj.1476-5381.2009.00532.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation and pharmacological characterization of AdTx1, a natural peptide displaying specific insurmountable antagonism of the α1A-adrenoceptor</span></div><div class="casAuthors">Quinton, L.; Girard, E.; Maiga, A.; Rekik, M.; Lluel, P.; Masuyer, G.; Larregola, M.; Marquer, C.; Ciolek, J.; Magnin, T.; Wagner, R.; Molgo, J.; Thai, R.; Fruchart-Gaillard, C.; Mourier, G.; Chamot-Rooke, J.; Menez, A.; Palea, S.; Servent, D.; Gilles, N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">316-325</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Venoms are a rich source of ligands for ion channels, but very little is known about their capacity to modulate G-protein coupled receptor (GPCR) activity.  We developed a strategy to identify novel toxins targeting GPCRs.  We studied the interactions of mamba venom fractions with α1-adrenoceptors in binding expts. with 3H-prazosin.  The active peptide (AdTx1) was sequenced by Edman degrdn. and mass spectrometry fragmentation.  Its synthetic homolog was pharmacol. characterized by binding expts. using cloned receptors and by functional expts. on rabbit isolated prostatic smooth muscle.  AdTx1, a 65 amino-acid peptide stabilized by four disulfide bridges, belongs to the three-finger-fold peptide family.  It has subnanomolar affinity (Ki = 0.35 nM) and high specificity for the human α1A-adrenoceptor subtype.  We showed high selectivity and affinity (Kd = 0.6 nM) of radio-labeled AdTx1 in direct binding expts. and revealed a slow assocn. const. (kon = 6 × 106 M-1/min-1) with an unusually stable α1A-adrenoceptor/AdTx1 complex (t1/2diss = 3.6 h).  AdTx1 displayed potent insurmountable antagonism of phenylephrine's actions in vitro (rabbit isolated prostatic muscle) at concns. of 10 to 100 nM.  AdTx1 is the most specific and selective peptide inhibitor for the α1A-adrenoceptor identified to date.  It displays insurmountable antagonism, acting as a potent relaxant of smooth muscle.  Its peptidic nature can be exploited to develop new tools, as a radio-labeled-AdTx1 or a fluoro-labeled-AdTx1.  Identification of AdTx1 thus offers new perspectives for developing new drugs for treating benign prostatic hyperplasia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2V3_7is37W7Vg90H21EOLACvtfcHk0lgNDw6SEA6ybw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhslGju70%253D&md5=c5a528b4755efe3ffdba1bf00e1612ca</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1476-5381.2009.00532.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1476-5381.2009.00532.x%26sid%3Dliteratum%253Aachs%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DGirard%26aufirst%3DE.%26aulast%3DMaiga%26aufirst%3DA.%26aulast%3DRekik%26aufirst%3DM.%26aulast%3DLluel%26aufirst%3DP.%26aulast%3DMasuyer%26aufirst%3DG.%26aulast%3DLarregola%26aufirst%3DM.%26aulast%3DMarquer%26aufirst%3DC.%26aulast%3DCiolek%26aufirst%3DJ.%26aulast%3DMagnin%26aufirst%3DT.%26aulast%3DWagner%26aufirst%3DR.%26aulast%3DMolg%25C3%25B3%26aufirst%3DJ.%26aulast%3DThai%26aufirst%3DR.%26aulast%3DFruchart-Gaillard%26aufirst%3DC.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DChamot-Rooke%26aufirst%3DJ.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DPalea%26aufirst%3DS.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DIsolation%2520and%2520Pharmacological%2520Characterization%2520of%2520AdTx1%252C%2520a%2520Natural%2520Peptide%2520Displaying%2520Specific%2520Insurmountable%2520Antagonism%2520of%2520the%2520%25CE%25B1%25201A-%2520Adrenoceptor%26jtitle%3DBr.%2520J.Pharmacol.%26date%3D2010%26volume%3D159%26spage%3D316%26epage%3D325%26doi%3D10.1111%2Fj.1476-5381.2009.00532.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jolkkonen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellman, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wernstedt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, E.</span></span> <span> </span><span class="NLM_article-title">A Snake Toxin against Muscarinic Acetylcholine Receptors: Amino Acid Sequence, Subtype Specificity and Effect on Guinea-Pig Ileum</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">399</span>– <span class="NLM_lpage">410</span>, <span class="refDoi"> DOI: 10.1016/0041-0101(94)00102-e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2F0041-0101%2894%2900102-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=7570626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2MXmtFentrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1995&pages=399-410&author=M.+Jolkkonenauthor=A.+Ademauthor=U.+Hellmanauthor=C.+Wernstedtauthor=E.+Karlsson&title=A+Snake+Toxin+against+Muscarinic+Acetylcholine+Receptors%3A+Amino+Acid+Sequence%2C+Subtype+Specificity+and+Effect+on+Guinea-Pig+Ileum&doi=10.1016%2F0041-0101%2894%2900102-e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">A snake toxin against muscarinic acetylcholine receptors: amino acid sequence, subtype specificity and effect on guinea pig ileum</span></div><div class="casAuthors">Jolkkonen, Mikael; Adem, Abdu; Hellman, Ulf; Wernstedt, Christer; Karlsson, Evert</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">399-40</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The sequence of muscarinic toxin 1 (MT1) from Dendroaspis angusticeps (green mamba) was detd. (66 amino acids, Mr 7509).  The central part, peptide 25-40, is rich in hydrophobic amino acids, which is a characteristic of muscarinic toxins.  MT1 started to inhibit [3H]-NMS (N-methylscopolamine) binding to synaptosomal membranes of porcine brain (contains all five receptor subtypes) at about 1 nM and to membranes from pig heart muscle (only subtype m2) at about 1 μM.  Binding of [3H]-AF-DX 384 to heart was inhibited with an IC50 of 14 μM and to brain in two steps.  In the first step (IC50 = 32 nM) binding decreased by 37%, indicating that the toxin acted on m1 or m4 receptors, each accounting for about 40%, of total receptor content.  The second step was similar to the effect on heart.  Pirenzepine inhibited binding of [125I]-MT1 to brain receptors with an IC50 of 6.5 nM, corresponding to a Ki of about 6 nM.  Literature values of Ki for pirenzepine are 16-18 nM for m1 and ≥120 mM for other subtypes.  This indicates binding of m1 receptors.  [125I]-MT1 bound to brain with a Kd of 20 nM and a Hill coeff. of 1.0, i.e. one toxin mol. per receptor.  In guinea-pig ileum, MT1 (670 nM) produced a rapid contraction, reversible by atropine.  The toxin may be an agonist, but might also cause contraction by inducing acetylcholine release by a different mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGollZAclD-2bLVg90H21EOLACvtfcHk0lgp7MFMl6P2Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtFentrc%253D&md5=fdf75fd4b8d479f9f2ddba6ef05327c6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2F0041-0101%2894%2900102-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0041-0101%252894%252900102-E%26sid%3Dliteratum%253Aachs%26aulast%3DJolkkonen%26aufirst%3DM.%26aulast%3DAdem%26aufirst%3DA.%26aulast%3DHellman%26aufirst%3DU.%26aulast%3DWernstedt%26aufirst%3DC.%26aulast%3DKarlsson%26aufirst%3DE.%26atitle%3DA%2520Snake%2520Toxin%2520against%2520Muscarinic%2520Acetylcholine%2520Receptors%253A%2520Amino%2520Acid%2520Sequence%252C%2520Subtype%2520Specificity%2520and%2520Effect%2520on%2520Guinea-Pig%2520Ileum%26jtitle%3DToxicon%26date%3D1995%26volume%3D33%26spage%3D399%26epage%3D410%26doi%3D10.1016%2F0041-0101%2894%2900102-e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cologna, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dos Santos Cardoso, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jourdan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degueldre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uetanabaro, A. P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa Neto, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thonart, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span> <span> </span><span class="NLM_article-title">Peptidomic Comparison and Characterization of the Major Components of the Venom of the Giant Ant <i>Dinoponera Quadriceps</i> Collected in Four Different Areas of Brazil</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2013.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.jprot.2013.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=24157790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2013&pages=413-422&author=C.+T.+Colognaauthor=J.+dos+Santos+Cardosoauthor=E.+Jourdanauthor=M.+Degueldreauthor=G.+Upertauthor=N.+Gillesauthor=A.+P.+T.+Uetanabaroauthor=E.+M.+Costa+Netoauthor=P.+Thonartauthor=E.+de+Pauwauthor=L.+Quinton&title=Peptidomic+Comparison+and+Characterization+of+the+Major+Components+of+the+Venom+of+the+Giant+Ant+Dinoponera+Quadriceps+Collected+in+Four+Different+Areas+of+Brazil&doi=10.1016%2Fj.jprot.2013.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidomic comparison and characterization of the major components of the venom of the giant ant Dinoponera quadriceps collected in four different areas of Brazil</span></div><div class="casAuthors">Cologna, Camila Takeno; Cardoso, Jaqueline dos Santos; Jourdan, Emmanuel; Degueldre, Michel; Upert, Gregory; Gilles, Nicolas; Uetanabaro, Ana Paula Trovatti; Costa Neto, Eraldo Medeiro; Thonart, Philippe; de Pauw, Edwin; Quinton, Loic</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">413-422</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We study the venom of the giant Dinoponera quadriceps ant collected in 4 different regions of Brazil.  By using a combination of complementary mass spectrometric approaches, we aim at: (1) characterizing the venom compn. of these ants; (2) establishing a comparative anal. of the venom from 4 geog. different regions in Brazil.  This approach demonstrates that ant venom is a copious source of new compds.  Several peptides were identified and selected for de novo sequencing.  Since most of the new peptides showed similarities with antimicrobial peptides (AMPs), antimicrobial assays were performed with the purpose of evaluating their activity.  In regard to the comparative study of the 4 regions, we obsd. not only major differences in the venom compns., but also that the venoms collected in closest areas are more similar than the ones collected in distant regions.  These observations seem to highlight an adaptation of the ant venoms to the local environment.  Concerning the biol. assays, the peptides called Dq-3162 and Da-3177 showed a wide-ranging antimicrobial activity.  The characterization of new AMPs with a broad spectrum of activity and different scaffolds may aid scientists to design new therapeutic agents and understand the mechanisms of those peptides to interact with microbial membranes.  The results obtained betoken the biotechnol. potential of ant's venom.  For the 1st time this manuscript describes an extensive proteomics characterization of the D. quadriceps venom.  In addn. this study reports the variation in venom compn. of primitive ants from 4 geog. different areas of Brazil.  The results reveal the presence of ∼335 compds. for each venom/area and inter-colony variations were obsd. 16 new peptides were characterized and 2 of them were synthesized and biol. assayed.  These findings highlight the considerable and still unexplored diversity of ant's venom which could be used as valuable research tools in different areas of knowledge.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonkEgVy8wrTrVg90H21EOLACvtfcHk0lgp7MFMl6P2Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOlsrbJ&md5=f6cca8bc8d0fd0d454a050a2f4b994ab</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2013.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2013.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DCologna%26aufirst%3DC.%2BT.%26aulast%3Ddos%2BSantos%2BCardoso%26aufirst%3DJ.%26aulast%3DJourdan%26aufirst%3DE.%26aulast%3DDegueldre%26aufirst%3DM.%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DUetanabaro%26aufirst%3DA.%2BP.%2BT.%26aulast%3DCosta%2BNeto%26aufirst%3DE.%2BM.%26aulast%3DThonart%26aufirst%3DP.%26aulast%3Dde%2BPauw%26aufirst%3DE.%26aulast%3DQuinton%26aufirst%3DL.%26atitle%3DPeptidomic%2520Comparison%2520and%2520Characterization%2520of%2520the%2520Major%2520Components%2520of%2520the%2520Venom%2520of%2520the%2520Giant%2520Ant%2520Dinoponera%2520Quadriceps%2520Collected%2520in%2520Four%2520Different%2520Areas%2520of%2520Brazil%26jtitle%3DJ.%2520Proteomics%26date%3D2013%26volume%3D94%26spage%3D413%26epage%3D422%26doi%3D10.1016%2Fj.jprot.2013.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wickremasinghe, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartgerink, J. D.</span></span> <span> </span><span class="NLM_article-title">Nanofibrous Snake Venom Hemostat</span>. <i>ACS Biomater. Sci. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1305</span>, <span class="refDoi"> DOI: 10.1021/acsbiomaterials.5b00356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsbiomaterials.5b00356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12lsrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=1300-1305&author=V.+A.+Kumarauthor=N.+C.+Wickremasingheauthor=S.+Shiauthor=J.+D.+Hartgerink&title=Nanofibrous+Snake+Venom+Hemostat&doi=10.1021%2Facsbiomaterials.5b00356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Nanofibrous snake venom hemostat</span></div><div class="casAuthors">Kumar, Vivek A.; Wickremasinghe, Navindee C.; Shi, Siyu; Hartgerink, Jeffrey D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Biomaterials Science & Engineering</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1300-1305</span>CODEN:
                <span class="NLM_cas:coden">ABSEBA</span>;
        ISSN:<span class="NLM_cas:issn">2373-9878</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Controlling perioperative bleeding is of crit. importance to minimize hemorrhaging and fatality.  Patients on anticoagulant therapy such as heparin have diminished clotting potential and are at risk for hemorrhaging.  Here we describe a self-assembling nanofibrous peptide hydrogel (termed SLac) that on its own can act as a phys. barrier to blood loss.  SLac was loaded with snake-venom derived Batroxobin (50 μg/mL) yielding a drug-loaded hydrogel (SB50).  SB50 was potentiated to enhance clotting even in the presence of heparin.  In vitro evaluation of fibrin and whole blood clotting helped identify appropriate concns. for hemostasis in vivo.  Batroxobin-loaded hydrogels rapidly (within 20s) stop bleeding in both normal and heparin-treated rats in a lateral liver incision model.  Compared to std. of care, Gelfoam, and investigational hemostats such as Puramatrix, only SB50 showed rapid liver incision hemostasis post surgical application.  This snake venom-loaded peptide hydrogel can be applied via syringe and conforms to the wound site resulting in hemostasis.  This demonstrates a facile method for surgical hemostasis even in the presence of anticoagulant therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRPUTMb5YJsbVg90H21EOLACvtfcHk0ljK4uo6NLyDqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12lsrzK&md5=5d23f7835a81de4649709140d1072b72</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsbiomaterials.5b00356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsbiomaterials.5b00356%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DV.%2BA.%26aulast%3DWickremasinghe%26aufirst%3DN.%2BC.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DHartgerink%26aufirst%3DJ.%2BD.%26atitle%3DNanofibrous%2520Snake%2520Venom%2520Hemostat%26jtitle%3DACS%2520Biomater.%2520Sci.%2520Eng.%26date%3D2015%26volume%3D1%26spage%3D1300%26epage%3D1305%26doi%3D10.1021%2Facsbiomaterials.5b00356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics from Venom: Current Status and Potential</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2738</span>– <span class="NLM_lpage">2758</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.bmc.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28988749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2738-2758&author=M.+W.+Penningtonauthor=A.+Czerwinskiauthor=R.+S.+Norton&title=Peptide+Therapeutics+from+Venom%3A+Current+Status+and+Potential&doi=10.1016%2Fj.bmc.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics from venom: Current status and potential</span></div><div class="casAuthors">Pennington, Michael W.; Czerwinski, Andrzej; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2738-2758</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized as being highly selective, potent and relatively safe as potential therapeutics.  Peptides isolated from the venom of different animals satisfy most of these criteria with the possible exception of safety, but when isolated as single compds. and used at appropriate concns., venom-derived peptides can become useful drugs.  Although the no. of venom-derived peptides that have successfully progressed to the clinic is currently limited, the prospects for venom-derived peptides look very optimistic.  As proteomic and transcriptomic approaches continue to identify new sequences, the potential of venom-derived peptides to find applications as therapeutics, cosmetics and insecticides grows accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8haCt_E4Tu7Vg90H21EOLACvtfcHk0ljK4uo6NLyDqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI&md5=253b75540d1a2117338259c087b119f7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DCzerwinski%26aufirst%3DA.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520Therapeutics%2520from%2520Venom%253A%2520Current%2520Status%2520and%2520Potential%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2738%26epage%3D2758%26doi%3D10.1016%2Fj.bmc.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span> <span> </span><span class="NLM_article-title">Versatile Spider Venom Peptides and Their Medical and Agricultural Applications</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>158</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2018.11.298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2018.11.298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=30543821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFOmsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=158&publication_year=2019&pages=109-126&author=N.+J.+Saezauthor=V.+Herzig&title=Versatile+Spider+Venom+Peptides+and+Their+Medical+and+Agricultural+Applications&doi=10.1016%2Fj.toxicon.2018.11.298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Versatile spider venom peptides and their medical and agricultural applications</span></div><div class="casAuthors">Saez, Natalie J.; Herzig, Volker</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">158</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">109-126</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Spiders have been evolving complex and diverse repertoires of peptides in their venoms with vast pharmacol. activities for more than 300 million years.  Spiders use their venoms for prey capture and defense, hence they contain peptides that target both prey (mainly arthropods) and predators (other arthropods or vertebrates).  This includes peptides that potently and selectively modulate a range of targets such as ion channels, receptors and signaling pathways involved in physiol. processes.  The contribution of these targets in particular disease pathophysiologies makes spider venoms a valuable source of peptides with potential therapeutic use.  In addn., peptides with insecticidal activities, used for prey capture, can be exploited for the development of novel bioinsecticides for agricultural use.  Although we have already reviewed potential applications of spider venom peptides as therapeutics (in 2010) and as bioinsecticides (in 2012), a considerable no. of research articles on both topics have been published since, warranting an updated review.  Here we explore the most recent research on the use of spider venom peptides for both medical and agricultural applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZSoXvfe9NjrVg90H21EOLACvtfcHk0ljK4uo6NLyDqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFOmsbzK&md5=73378135ef93499af7accc6b2f2280f6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2018.11.298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2018.11.298%26sid%3Dliteratum%253Aachs%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DHerzig%26aufirst%3DV.%26atitle%3DVersatile%2520Spider%2520Venom%2520Peptides%2520and%2520Their%2520Medical%2520and%2520Agricultural%2520Applications%26jtitle%3DToxicon%26date%3D2019%26volume%3D158%26spage%3D109%26epage%3D126%26doi%3D10.1016%2Fj.toxicon.2018.11.298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mebs, D.</span></span> <span> </span><span class="NLM_article-title">Toxicity in Animals. Trends in Evolution?</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1016/S0041-0101(00)00155-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0041-0101%2800%2900155-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10936625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A280%3ADC%252BD3M7htVKquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2001&pages=87-96&author=D.+Mebs&title=Toxicity+in+Animals.+Trends+in+Evolution%3F&doi=10.1016%2FS0041-0101%2800%2900155-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicity in animals. Trends in evolution?</span></div><div class="casAuthors">Mebs D</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon : official journal of the International Society on Toxinology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">87-96</span>
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    </div><div class="casAbstract">Animals acquire toxicity either by metabolic synthesis of toxins (secondary metabolites), by expression of toxin genes or by the uptake, storage and sequestration of toxins produced by other organisms, i.e., microbes, plants or other animals.  Variability of toxin structure and function is high.  Peptide toxins in particular, although relying on a limited number of structural frameworks, often exhibit considerable structural hypervariability.  An accelerated rate of evolution in the toxin gene structure (conserved introns, but high substitution rates in the exons) leads to the functional diversity of these peptides or proteins.  The selective forces which may drive toxin evolution are unknown.  Venomousness or the possession of toxins can be essential for survival, but the advantage of toxin biosynthesis may also be of minor importance or has been lost during evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQxzMviJQQtcJSWdIdmEou5fW6udTcc2ebKEIFydxO56Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3M7htVKquw%253D%253D&md5=f2701ded72deb5c568433b893a0ce0a3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0041-0101%2800%2900155-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0041-0101%252800%252900155-0%26sid%3Dliteratum%253Aachs%26aulast%3DMebs%26aufirst%3DD.%26atitle%3DToxicity%2520in%2520Animals.%2520Trends%2520in%2520Evolution%253F%26jtitle%3DToxicon%26date%3D2001%26volume%3D39%26spage%3D87%26epage%3D96%26doi%3D10.1016%2FS0041-0101%2800%2900155-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westermann, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Conopeptide Characterization and Classifications: An Analysis Using ConoServer</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1509</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2010.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2010.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20211197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFGgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2010&pages=1491-1509&author=Q.+Kaasauthor=J.-C.+Westermannauthor=D.+J.+Craik&title=Conopeptide+Characterization+and+Classifications%3A+An+Analysis+Using+ConoServer&doi=10.1016%2Fj.toxicon.2010.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conopeptide characterization and classifications: An analysis using ConoServer</span></div><div class="casAuthors">Kaas, Quentin; Westermann, Jan-Christoph; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1491-1509</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Cone snails are carnivorous marine gastropods that have evolved potent venoms to capture their prey.  These venoms comprise a rich and diverse cocktail of peptide toxins, or conopeptides, whose high diversity has arisen from an efficient hypermutation mechanism, combined with a high frequency of post-translational modifications.  Conopeptides bind with high specificity to distinct membrane receptors, ion channels, and transporters of the central and muscular nervous system.  As well as serving their natural function in prey capture, conopeptides have been utilized as versatile tools in neuroscience and have proven valuable as drug leads that target the nervous system in humans.  This paper examines current knowledge on conopeptide sequences based on an anal. of gene and peptide sequences in ConoServer, a specialized database of conopeptide sequences and three-dimensional structures.  The authors describe updates to the content and organization of ConoServer and discuss correlations between gene superfamilies, cysteine frameworks, pharmacol. families targeted by conopeptides, and the phylogeny, habitat, and diet of cone snails.  The study identifies gaps in current knowledge of conopeptides and points to potential directions for future research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSrDCcmmrOPbVg90H21EOLACvtfcHk0lihDp8fwR6MsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFGgtb4%253D&md5=700eedb2cffcd75936337703494cfdae</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2010.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2010.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DWestermann%26aufirst%3DJ.-C.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DConopeptide%2520Characterization%2520and%2520Classifications%253A%2520An%2520Analysis%2520Using%2520ConoServer%26jtitle%3DToxicon%26date%3D2010%26volume%3D55%26spage%3D1491%26epage%3D1509%26doi%3D10.1016%2Fj.toxicon.2010.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span> <span> </span><span class="NLM_article-title">The European FP7 Venomics Project</span>. <i>Futur. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1611</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.4155/FMC.14.85</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.4155%2Ffmc.14.85" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=25406001" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1611-1612&author=N.+Gillesauthor=D.+Servent&title=The+European+FP7+Venomics+Project&doi=10.4155%2FFMC.14.85"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.85&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.85%26sid%3Dliteratum%253Aachs%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DServent%26aufirst%3DD.%26atitle%3DThe%2520European%2520FP7%2520Venomics%2520Project%26jtitle%3DFutur.%2520Med.%2520Chem%26date%3D2014%26volume%3D6%26spage%3D1611%26epage%3D1612%26doi%3D10.4155%2FFMC.14.85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turner, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span> <span> </span><span class="NLM_article-title">Bioactive Compounds Isolated from Neglected Predatory Marine Gastropods</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">118</span>, <span class="refDoi"> DOI: 10.3390/md16040118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Fmd16040118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlOgtr3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=118&author=A.+H.+Turnerauthor=D.+J.+Craikauthor=Q.+Kaasauthor=C.+I.+Schroeder&title=Bioactive+Compounds+Isolated+from+Neglected+Predatory+Marine+Gastropods&doi=10.3390%2Fmd16040118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive compounds isolated from neglected predatory marine gastropods</span></div><div class="casAuthors">Turner, Ashlin H.; Craik, David J.; Kaas, Quentin; Schroeder, Christina I.</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">118/1-118/20</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A diverse range of predatory marine gastropods produce toxins, yet most of these mols. remain uncharacterized.  Conus species have received the most attention from researchers, leading to several conopeptides reaching clin. trials.  This review aims to summarize what is known about bioactive compds. isolated from species of neglected marine gastropods, esp. in the Turridae, Terebridae, Babyloniidae, Muricidae, Buccinidae, Colubrariidae, Nassariidae, Cassidae, and Ranellidae families.  Multiple species have been reported to contain bioactive compds. with potential toxic activity, but most of these compds. have not been characterized or even clearly identified.  The bioactive properties and potential applications of echotoxins and related porins from the Ranellidae family are discussed in more detail.  Finally, the review concludes with a call for research on understudied species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQy90TNWblmrVg90H21EOLACvtfcHk0lihDp8fwR6MsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlOgtr3N&md5=ade043d4918d08ebf8570a7b6b2a8942</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.3390%2Fmd16040118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd16040118%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DA.%2BH.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26atitle%3DBioactive%2520Compounds%2520Isolated%2520from%2520Neglected%2520Predatory%2520Marine%2520Gastropods%26jtitle%3DMar.%2520Drugs%26date%3D2018%26volume%3D16%26spage%3D118%26doi%3D10.3390%2Fmd16040118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morsa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiwir, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Rocca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanozin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grifnée, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longuespée, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meuwis, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smargiasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzucchelli, G.</span></span> <span> </span><span class="NLM_article-title">Multi-Enzymatic Limited Digestion: The Next-Generation Sequencing for Proteomics?</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2501</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.9b00044</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.9b00044" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos1yitb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=2501-2513&author=D.+Morsaauthor=D.+Baiwirauthor=R.+La+Roccaauthor=T.+A.+Zimmermanauthor=E.+Hanozinauthor=E.+Grifn%C3%A9eauthor=R.+Longuesp%C3%A9eauthor=M.-A.+Meuwisauthor=N.+Smargiassoauthor=E.+De+Pauwauthor=G.+Mazzucchelli&title=Multi-Enzymatic+Limited+Digestion%3A+The+Next-Generation+Sequencing+for+Proteomics%3F&doi=10.1021%2Facs.jproteome.9b00044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-Enzymatic Limited Digestion: The Next-Generation Sequencing for Proteomics?</span></div><div class="casAuthors">Morsa, Denis; Baiwir, Dominique; La Rocca, Raphael; Zimmerman, Tyler A.; Hanozin, Emeline; Grifnee, Elodie; Longuespee, Remi; Meuwis, Marie-Alice; Smargiasso, Nicolas; Pauw, Edwin De; Mazzucchelli, Gabriel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2501-2513</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Over the past 40 years, proteomics, generically defined as the field dedicated to the identification and anal. of proteins, has tremendously gained in popularity and potency through advancements in genome sequencing, separative techniques, mass spectrometry and bioinformatics algorithms.  As a consequence, its scope of application has gradually enlarged and diversified to meet specialized topical biomedical subjects nowadays.  Although the tryptic bottom-up approach is widely regarded as the gold std. for rapid screening of complex samples, its application for precise and confident mapping of protein modifications is often hindered due to partial sequence coverage, poor redundancy in indicative peptides and lack of method flexibility.  The authors here show how the synergic and time-limited action of a properly dild. mix of multiple enzymes can be exploited in a versatile yet straightforward protocol to alleviate present-day drawbacks.  Merging bottom-up and middle-down ideologies, the results highlight broad assemblies of overlapping peptides that enable refined and reliable characterizations of proteins, including variants identification, and of their carried modifications, including posttranslational modifications, truncations and cleavages.  Beyond this boost in performance, the methodol. also offers efficient de novo sequencing capabilities, in view of which the authors here present a dedicated custom assembly algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLdDzX1izqM7Vg90H21EOLACvtfcHk0lhPn0cIQEnZ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos1yitb0%253D&md5=5be9e66709e46adf9206017919044bf9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.9b00044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.9b00044%26sid%3Dliteratum%253Aachs%26aulast%3DMorsa%26aufirst%3DD.%26aulast%3DBaiwir%26aufirst%3DD.%26aulast%3DLa%2BRocca%26aufirst%3DR.%26aulast%3DZimmerman%26aufirst%3DT.%2BA.%26aulast%3DHanozin%26aufirst%3DE.%26aulast%3DGrifn%25C3%25A9e%26aufirst%3DE.%26aulast%3DLonguesp%25C3%25A9e%26aufirst%3DR.%26aulast%3DMeuwis%26aufirst%3DM.-A.%26aulast%3DSmargiasso%26aufirst%3DN.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DMazzucchelli%26aufirst%3DG.%26atitle%3DMulti-Enzymatic%2520Limited%2520Digestion%253A%2520The%2520Next-Generation%2520Sequencing%2520for%2520Proteomics%253F%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2019%26volume%3D18%26spage%3D2501%26epage%3D2513%26doi%3D10.1021%2Facs.jproteome.9b00044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, A.-h.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span> <span> </span><span class="NLM_article-title">Deep Venomics Reveals the Mechanism for Expanded Peptide Diversity in Cone Snail Venom</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">312</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1074/mcp.M112.021469</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fmcp.M112.021469" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23152539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFerurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=312-329&author=S.+Dutertreauthor=A.-h.+Jinauthor=Q.+Kaasauthor=A.+Jonesauthor=P.+F.+Alewoodauthor=R.+J.+Lewis&title=Deep+Venomics+Reveals+the+Mechanism+for+Expanded+Peptide+Diversity+in+Cone+Snail+Venom&doi=10.1074%2Fmcp.M112.021469"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Deep Venomics Reveals the Mechanism for Expanded Peptide Diversity in Cone Snail Venom</span></div><div class="casAuthors">Dutertre, Sebastien; Jin, Ai-hua; Kaas, Quentin; Jones, Alun; Alewood, Paul F.; Lewis, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">312-329</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9484</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Cone snails produce highly complex venom comprising mostly small biol. active peptides known as conotoxins or conopeptides.  Early ests. that suggested 50-200 venom peptides are produced per species have been recently increased at least 10-fold using advanced mass spectrometry.  To uncover the mechanism(s) responsible for generating this impressive diversity, we used an integrated approach combining second-generation transcriptome sequencing with high sensitivity proteomics.  From the venom gland transcriptome of Conus marmoreus, a total of 105 conopeptide precursor sequences from 13 gene superfamilies were identified.  Over 60% of these precursors belonged to the three gene superfamilies O1, T, and M, consistent with their high levels of expression, which suggests these conotoxins play an important role in prey capture and/or defense.  Seven gene superfamilies not previously identified in C. marmoreus, including five novel superfamilies, were also discovered.  To confirm the expression of toxins identified at the transcript level, the injected venom of C. marmoreus was comprehensively analyzed by mass spectrometry, revealing 2710 and 3172 peptides using MALDI and ESI-MS, resp., and 6254 peptides using an ESI-MS TripleTOF 5600 instrument.  All conopeptides derived from transcriptomic sequences could be matched to masses obtained on the TripleTOF within 100 ppm accuracy, with 66 (63%) providing MS/MS coverage that unambiguously confirmed these matches.  Comprehensive integration of transcriptomic and proteomic data revealed for the first time that the vast majority of the conopeptide diversity arises from a more limited set of genes through a process of variable peptide processing, which generates conopeptides with alternative cleavage sites, heterogeneous post-translational modifications, and highly variable N- and C-terminal truncations.  Variable peptide processing is expected to contribute to the evolution of venoms, and explains how a limited set of ∼ 100 gene transcripts can generate thousands of conopeptides in a single species of cone snail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG3WIrFRD1XbVg90H21EOLACvtfcHk0lhPn0cIQEnZ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFerurs%253D&md5=2cb31d94877f9c33d1b559542796f512</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M112.021469&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M112.021469%26sid%3Dliteratum%253Aachs%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DJin%26aufirst%3DA.-h.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26atitle%3DDeep%2520Venomics%2520Reveals%2520the%2520Mechanism%2520for%2520Expanded%2520Peptide%2520Diversity%2520in%2520Cone%2520Snail%2520Venom%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2013%26volume%3D12%26spage%3D312%26epage%3D329%26doi%3D10.1074%2Fmcp.M112.021469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span> <span> </span><span class="NLM_article-title">Native Pyroglutamation of Huwentoxin-IV: A Post-Translational Modification That Increases the Trapping Ability to the Sodium Channel</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e65984</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0065984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1371%2Fjournal.pone.0065984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23826086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVyhtrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=M.+Rongauthor=Z.+Duanauthor=J.+Chenauthor=J.+Liauthor=Y.+Xiaoauthor=S.+Liang&title=Native+Pyroglutamation+of+Huwentoxin-IV%3A+A+Post-Translational+Modification+That+Increases+the+Trapping+Ability+to+the+Sodium+Channel&doi=10.1371%2Fjournal.pone.0065984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Native pyroglutamation of huwentoxin-IV: a post-translational modification that increases the trapping ability to the sodium channel</span></div><div class="casAuthors">Rong, Mingqiang; Duan, Zhigui; Chen, Juliang; Li, Jianglin; Xiao, Yuchen; Liang, Songping</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e65984</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Huwentoxin-IV (HWTX-IV), a tetrodotoxin-sensitive (TTX-s) sodium channel antagonist, is found in the venom of the Chinese spider Ornithoctonus huwena.  A naturally modified HWTX-IV (mHWTX-IV), having a mol. mass 18 Da lower than HWTX-IV, has also been isolated from the venom of the same spider.  By a combination of enzymic fragmentation and MS/MS de novo sequencing, mHWTX-IV has been shown to have the same amino acid sequence as that of HWTX-IV, except that the N-terminal glutamic acid replaced by pyroglutamic acid.  MHWTX-IV inhibited tetrodotoxin-sensitive voltage-gated sodium channels of dorsal root ganglion neurons with an IC50 nearly equal to native HWTX-IV.  MHWTX-IV showed the same activation and inactivation kinetics seen for native HWTX-IV.  In contrast with HWTX-IV, which dissocs. at moderate voltage depolarization voltages (+50 mV, 180000 ms), mHWTX-IV inhibition of TTX-sensitive sodium channels is not reversed by strong depolarization voltages (+200 mV, 500 ms).  Recovery of Nav1.7current was voltage-dependent and was induced by extreme depolarization in the presence of HWTX-IV, but no obvious current was elicited after application of mHWTX-IV.  Our data indicate that the N-terminal modification of HWTX-IV gives the peptide toxin a greater ability to trap the voltage sensor in the sodium channel.  Loss of a neg. charge, caused by cyclization at the N-terminus, is a possible reason why the modified toxin binds much stronger.  To our knowledge, this is the first report of a pyroglutamic acid residue in a spider toxin; this modification seems to increase the trapping ability of the voltage sensor in the sodium channel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogEk5MciNqr7Vg90H21EOLACvtfcHk0lhPn0cIQEnZ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVyhtrjK&md5=2a408cf4a206b555856f438263f13635</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0065984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0065984%26sid%3Dliteratum%253Aachs%26aulast%3DRong%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DS.%26atitle%3DNative%2520Pyroglutamation%2520of%2520Huwentoxin-IV%253A%2520A%2520Post-Translational%2520Modification%2520That%2520Increases%2520the%2520Trapping%2520Ability%2520to%2520the%2520Sodium%2520Channel%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0065984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravindran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moczydlowski, E.</span></span> <span> </span><span class="NLM_article-title">Analysis of the Blocking Activity of Charybdotoxin Homologs and Iodinated Derivatives against Ca<sup>2+</sup>-Activated K<sup>+</sup> Channels</span>. <i>J. Membr. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1007/BF01870284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2FBF01870284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=2477548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaL1MXlvV2hsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=1989&pages=269-281&author=K.+Lucchesiauthor=A.+Ravindranauthor=H.+Youngauthor=E.+Moczydlowski&title=Analysis+of+the+Blocking+Activity+of+Charybdotoxin+Homologs+and+Iodinated+Derivatives+against+Ca2%2B-Activated+K%2B+Channels&doi=10.1007%2FBF01870284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the blocking activity of charybdotoxin homologs and iodinated derivatives against calcium-activated potassium channels</span></div><div class="casAuthors">Lucchesi, Kathryn; Ravindran, Arippa; Young, Howard; Moczydlowski, Edward</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Membrane Biology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-81</span>CODEN:
                <span class="NLM_cas:coden">JMBBBO</span>;
        ISSN:<span class="NLM_cas:issn">0022-2631</span>.
    </div><div class="casAbstract">Two charybdotoxin peptides were purified from venom of the Israeli scorpion, Leiurus quinquestriatus hebraeus.  Microsequencing of the most abundant toxin, ChTX-Lq1, revealed identity with the 37-residue peptide.  Sequence data on the minor peptide, ChTX-Lq2, showed substantial homol. to ChTX-Lq1 with differences obsd. at 8 positions.  These 2 charybdotoxin sequences, along with that of noxiustoxin, define a distinct family of scorpion peptide toxins with activity against K+ channels.  Both charybdotoxin homologs inhibited Ca2+-dependent K+ efflux from human erythrocytes with similar potency, K0.5 ≃ 40 nM.  In planar bilayer assays of single K(Ca) channels from rat muscle, ChTX-Lq1 and ChTX-Lq2 blocked with intrinsic Kd's of 1.3 and 43 nM, resp., in the presence of 50 mM external KCl.  A new application of dwell-time histogram anal. of single-channel blocking events was used to characterize the kinetic homogeneity of toxin samples and the blocking kinetics of ChTX derivs.  The lower blocking affinity of ChTX-Lq2 was the combined result of a faster dissocn. rate and a slower assocn. rate as compared to ChTX-Lq1.  The blocking activity of 2 monoiodinated derivs. of ChTX-Lq1 was also analyzed.  Blocked dwell-time histograms of the iodinated peptides were characterized by predominately brief (0.2-2 s) blocking events in comparison to the native toxin (20 s).  Histogram anal. revealed that monoiodination of ChTX-Lq1 impairs blocking activity by adverse effects on both dissocn. and assocn. rate consts.  Frequency d. histograms of single channel blocking events provide a sensitive assay of toxin purity suitable for quantitating structure-activity relationships of charybdotoxin derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbd84u1dutE7Vg90H21EOLACvtfcHk0lh2pkr7ZZ7A_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXlvV2hsLY%253D&md5=156df9680d3aee8bea904ec57872a723</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2FBF01870284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01870284%26sid%3Dliteratum%253Aachs%26aulast%3DLucchesi%26aufirst%3DK.%26aulast%3DRavindran%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DH.%26aulast%3DMoczydlowski%26aufirst%3DE.%26atitle%3DAnalysis%2520of%2520the%2520Blocking%2520Activity%2520of%2520Charybdotoxin%2520Homologs%2520and%2520Iodinated%2520Derivatives%2520against%2520Ca2%252B-Activated%2520K%252B%2520Channels%26jtitle%3DJ.%2520Membr.%2520Biol.%26date%3D1989%26volume%3D109%26spage%3D269%26epage%3D281%26doi%3D10.1007%2FBF01870284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Er, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span> <span> </span><span class="NLM_article-title">Spider-Venom Peptides as Therapeutics</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2851</span>– <span class="NLM_lpage">2871</span>, <span class="refDoi"> DOI: 10.3390/toxins2122851</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Ftoxins2122851" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=22069579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFWgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=2851-2871&author=N.+J.+Saezauthor=S.+Senffauthor=J.+E.+Jensenauthor=S.+Y.+Erauthor=V.+Herzigauthor=L.+D.+Rashauthor=G.+F.+King&title=Spider-Venom+Peptides+as+Therapeutics&doi=10.3390%2Ftoxins2122851"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Spider-venom peptides as therapeutics</span></div><div class="casAuthors">Saez, Natalie J.; Senff, Sebastian; Jensen, Jonas E.; Er, Sing Yan; Herzig, Volker; Rash, Lachlan D.; King, Glenn F.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2851-2871</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  Spiders are the most successful venomous animals and the most abundant terrestrial predators.  Their remarkable success is due in large part to their ingenious exploitation of silk and the evolution of pharmacol. complex venoms that ensure rapid subjugation of prey.  Most spider venoms are dominated by disulfide-rich peptides that typically have high affinity and specificity for particular subtypes of ion channels and receptors.  Spider venoms are conservatively predicted to contain more than 10 million bioactive peptides, making them a valuable resource for drug discovery.  Here we review the structure and pharmacol. of spider-venom peptides that are being used as leads for the development of therapeutics against a wide range of pathophysiol. conditions including cardiovascular disorders, chronic pain, inflammation, and erectile dysfunction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpA11F1oYodH7Vg90H21EOLACvtfcHk0lh2pkr7ZZ7A_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFWgsQ%253D%253D&md5=8b4bcd142d15507b22d66a45eb943bfa</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3390%2Ftoxins2122851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins2122851%26sid%3Dliteratum%253Aachs%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DSenff%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DJ.%2BE.%26aulast%3DEr%26aufirst%3DS.%2BY.%26aulast%3DHerzig%26aufirst%3DV.%26aulast%3DRash%26aufirst%3DL.%2BD.%26aulast%3DKing%26aufirst%3DG.%2BF.%26atitle%3DSpider-Venom%2520Peptides%2520as%2520Therapeutics%26jtitle%3DToxins%26date%3D2010%26volume%3D2%26spage%3D2851%26epage%3D2871%26doi%3D10.3390%2Ftoxins2122851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronjat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Waard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span> <span> </span><span class="NLM_article-title">Functional Evolution of Scorpion Venom Peptides with an Inhibitor Cystine Knot Fold</span>. <i>Biosci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1042/BSR20130052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1042%2FBSR20130052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVSmtL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=513-527&author=B.+Gaoauthor=P.+J.+Harveyauthor=D.+J.+Craikauthor=M.+Ronjatauthor=M.+De+Waardauthor=S.+Zhu&title=Functional+Evolution+of+Scorpion+Venom+Peptides+with+an+Inhibitor+Cystine+Knot+Fold&doi=10.1042%2FBSR20130052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Functional evolution of scorpion venom peptides with an inhibitor cystine knot fold</span></div><div class="casAuthors">Gao, Bin; Harvey, Peta J.; Craik, David J.; Ronjat, Michel; De Waard, Michel; Zhu, Shunyi</div><div class="citationInfo"><span class="NLM_cas:title">Bioscience Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">513-527, S513/1</span>CODEN:
                <span class="NLM_cas:coden">BRPTDT</span>;
        ISSN:<span class="NLM_cas:issn">0144-8463</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The ICK (inhibitor cystine knot) defines a large superfamily of polypeptides with high structural stability and functional diversity.  Here, we describe a new scorpion venom-derived K+ channel toxin (named λ-MeuKTx-1) with an ICK fold through gene cloning, chem. synthesis, NMR spectroscopy, Ca2+ release measurements and electrophysiol. recordings.  λ-MeuKTx-1 was found to adopt an ICK fold that contains a three-strand anti-parallel β-sheet and a 310-helix.  Functionally, this peptide selectively inhibits the Drosophila Shaker K+ channel but is not capable of activating skeletal-type Ca2+ release channels/ryanodine receptors, which is remarkably different from the previously known scorpion venom ICK peptides.  The removal of two C-terminal residues of λ-MeuKTx-1 led to the loss of the inhibitory activity on the channel, whereas the C-terminal amidation resulted in the emergence of activity on four mammalian K+ channels accompanied by the loss of activity on the Shaker channel.  A combination of structural and pharmacol. data allows the recognition of three putative functional sites involved in channel blockade of λ-MeuKTx-1.  The presence of a functional dyad in λ-MeuKTx-1 supports functional convergence among scorpion venom peptides with different folds.  Furthermore, similarities in precursor organization, exon-intron structure, 3D-fold and function suggest that scorpion venom ICK-type K+ channel inhibitors and Ca2+ release channel activators share a common ancestor and their divergence occurs after speciation between buthidae and non-buthids.  The structural and functional characterizations of the first scorpion venom ICK toxin with K+ channel-blocking activity sheds light on functionally divergent and convergent evolution of this conserved scaffold of ancient origin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwiFGTa9ug97Vg90H21EOLACvtfcHk0lh2pkr7ZZ7A_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVSmtL%252FN&md5=580122c3695ef7934fbf520bda938616</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1042%2FBSR20130052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBSR20130052%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DB.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DRonjat%26aufirst%3DM.%26aulast%3DDe%2BWaard%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.%26atitle%3DFunctional%2520Evolution%2520of%2520Scorpion%2520Venom%2520Peptides%2520with%2520an%2520Inhibitor%2520Cystine%2520Knot%2520Fold%26jtitle%3DBiosci.%2520Rep.%26date%3D2013%26volume%3D33%26spage%3D513%26epage%3D527%26doi%3D10.1042%2FBSR20130052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Conotoxin Gene Superfamilies</span>. <i>Mar. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">6058</span>– <span class="NLM_lpage">6101</span>, <span class="refDoi"> DOI: 10.3390/md12126058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Fmd12126058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=25522317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1Klsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=6058-6101&author=S.+D.+Robinsonauthor=R.+S.+Norton&title=Conotoxin+Gene+Superfamilies&doi=10.3390%2Fmd12126058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Conotoxin gene superfamilies</span></div><div class="casAuthors">Robinson, Samuel D.; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Marine Drugs</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6058-6101, 44 pp.</span>CODEN:
                <span class="NLM_cas:coden">MDARE6</span>;
        ISSN:<span class="NLM_cas:issn">1660-3397</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Conotoxins are the peptidic components of the venoms of marine cone snails (genus Conus).  They are remarkably diverse in terms of structure and function.  Unique potency and selectivity profiles for a range of neuronal targets have made several conotoxins valuable as research tools, drug leads and even therapeutics, and has resulted in a concerted and increasing drive to identify and characterize new conotoxins.  Conotoxins are translated from mRNA as peptide precursors, and cDNA sequencing is now the primary method for identification of new conotoxin sequences.  As a result, gene superfamily, a classification based on precursor signal peptide identity, has become the most convenient method of conotoxin classification.  Here we review each of the described conotoxin gene superfamilies, with a focus on the structural and functional diversity present in each.  This review is intended to serve as a practical guide to conotoxin superfamilies and to facilitate interpretation of the increasing no. of conotoxin precursor sequences being identified by targeted-cDNA sequencing and more recently high-throughput transcriptome sequencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0yTywrt35WLVg90H21EOLACvtfcHk0lh2pkr7ZZ7A_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1Klsbw%253D&md5=7a11520b65e7031b9656b41a1052f26e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3390%2Fmd12126058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmd12126058%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DS.%2BD.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DConotoxin%2520Gene%2520Superfamilies%26jtitle%3DMar.%2520Drugs%26date%3D2014%26volume%3D12%26spage%3D6058%26epage%3D6101%26doi%3D10.3390%2Fmd12126058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Imperial, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillyard, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivera, B. M.</span></span> <span> </span><span class="NLM_article-title">The Augertoxins: Biochemical Characterization of Venom Components from the Toxoglossate Gastropod <i>Terebra Subulata</i></span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">391</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1016/S0041-0101(03)00169-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0041-0101%2803%2900169-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=14505939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntl2jt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=391-398&author=J.+S.+Imperialauthor=M.+Watkinsauthor=P.+Chenauthor=D.+R.+Hillyardauthor=L.+J.+Cruzauthor=B.+M.+Olivera&title=The+Augertoxins%3A+Biochemical+Characterization+of+Venom+Components+from+the+Toxoglossate+Gastropod+Terebra+Subulata&doi=10.1016%2FS0041-0101%2803%2900169-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The augertoxins: biochemical characterization of venom components from the toxoglossate gastropod Terebra subulata</span></div><div class="casAuthors">Imperial, Julita S.; Watkins, Maren; Chen, Ping; Hillyard, David R.; Cruz, Lourdes J.; Olivera, Baldomero M.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">391-398</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We describe the purifn. and biochem. characterization of three components from the venom of the toxoglossate gastropod Terebra subulata.  The three polypeptide venom components, augertoxins s6a, s7a and s11a, are 40-41 AA in length with 3-4 disulfide linkages.  The arrangement of Cys residues is reminiscent of certain conopeptide superfamilies, but mol. cloning failed to show the highly conserved sequence features diagnostic of the conopeptide gene superfamily with a similar arrangement of Cys residues.  One of the purified peptides, s7a, elicited an uncoordinated twisting syndrome when injected into the nematode Caenorhabditis elegans, but had no effect on mice.  T. subulata belongs to the family Terebridae, one of four major groups of toxoglossate gastropods in the superfamily Conacea.  The results reveal that some features of the augertoxins and conotoxins are generally similar, such as the organization of prepropeptide precursors and their proteolytic processing into mature toxins; however, Terebra may have evolved generally larger venom components that are less highly post-translationally modified.  The results suggest that Conus peptide gene superfamilies probably do not extend to the Terebridae, suggesting that distinctive venom gene superfamilies may be expressed in each major division of Conacean gastropods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruKqhz-aXpPrVg90H21EOLACvtfcHk0lgwmLY-jh7yqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntl2jt7k%253D&md5=377146964be8d14418df6096112d35bc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0041-0101%2803%2900169-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0041-0101%252803%252900169-7%26sid%3Dliteratum%253Aachs%26aulast%3DImperial%26aufirst%3DJ.%2BS.%26aulast%3DWatkins%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DHillyard%26aufirst%3DD.%2BR.%26aulast%3DCruz%26aufirst%3DL.%2BJ.%26aulast%3DOlivera%26aufirst%3DB.%2BM.%26atitle%3DThe%2520Augertoxins%253A%2520Biochemical%2520Characterization%2520of%2520Venom%2520Components%2520from%2520the%2520Toxoglossate%2520Gastropod%2520Terebra%2520Subulata%26jtitle%3DToxicon%26date%3D2003%26volume%3D42%26spage%3D391%26epage%3D398%26doi%3D10.1016%2FS0041-0101%2803%2900169-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavergne, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harliwong, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span> <span> </span><span class="NLM_article-title">Optimized Deep-Targeted Proteotranscriptomic Profiling Reveals Unexplored Conus Toxin Diversity and Novel Cysteine Frameworks</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E3782</span>– <span class="NLM_lpage">E3791</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501334112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.1501334112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=26150494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFajsr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E3782-E3791&author=V.+Lavergneauthor=I.+Harliwongauthor=A.+Jonesauthor=D.+Millerauthor=R.+J.+Taftauthor=P.+F.+Alewood&title=Optimized+Deep-Targeted+Proteotranscriptomic+Profiling+Reveals+Unexplored+Conus+Toxin+Diversity+and+Novel+Cysteine+Frameworks&doi=10.1073%2Fpnas.1501334112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Optimized deep-targeted proteotranscriptomic profiling reveals unexplored Conus toxin diversity and novel cysteine frameworks</span></div><div class="casAuthors">Lavergne, Vincent; Harliwong, Ivon; Jones, Alun; Miller, David; Taft, Ryan J.; Alewood, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">E3782-E3791</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cone snails are predatory marine gastropods characterized by a sophisticated venom app. responsible for the biosynthesis and delivery of complex mixts. of cysteine-rich toxin peptides.  These conotoxins fold into small highly structured frameworks, allowing them to potently and selectively interact with heterologous ion channels and receptors.  Approx. 2,000 toxins from an estd. no. of >70,000 bioactive peptides have been identified in the genus Conus to date.  Here, we describe a high-resoln. interrogation of the transcriptomes (available at www.ddbj.nig.ac.jp) and proteomes of the diverse compartments of the Conus episcopatus venom app.  Using biochem. and bioinformatic tools, we found the highest no. of conopeptides yet discovered in a single Conus specimen, with 3,305 novel precursor toxin sequences classified into 9 known superfamilies (A, I1, I2, M, O1, O2, S, T, Z), and identified 16 new superfamilies showing unique signal peptide signatures.  We were also able to depict the largest population of venom peptides contg. the pharmacol. active C-C-CC-C-C inhibitor cystine knot and CC-C-C motifs (168 and 44 toxins, resp.), as well as 208 new conotoxins displaying odd nos. of cysteine residues derived from known conotoxin motifs.  Importantly, six novel cysteine-rich frameworks were revealed which may have novel pharmacol.  Finally, analyses of codon usage bias and RNA-editing processes of the conotoxin transcripts demonstrate a specific conservation of the cysteine skeleton at the nucleic acid level and provide new insights about the origin of sequence hypervariablity in mature toxin regions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFGvczG7RzuLVg90H21EOLACvtfcHk0lgwmLY-jh7yqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFajsr%252FM&md5=4a45d5932749b5814d76c7f93cd2b848</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501334112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501334112%26sid%3Dliteratum%253Aachs%26aulast%3DLavergne%26aufirst%3DV.%26aulast%3DHarliwong%26aufirst%3DI.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DMiller%26aufirst%3DD.%26aulast%3DTaft%26aufirst%3DR.%2BJ.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26atitle%3DOptimized%2520Deep-Targeted%2520Proteotranscriptomic%2520Profiling%2520Reveals%2520Unexplored%2520Conus%2520Toxin%2520Diversity%2520and%2520Novel%2520Cysteine%2520Frameworks%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3DE3782%26epage%3DE3791%26doi%3D10.1073%2Fpnas.1501334112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdenaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alili, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Production of Two Disulphide-Bridge Toxins</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">8408</span>– <span class="NLM_lpage">8411</span>, <span class="refDoi"> DOI: 10.1039/c4cc02679a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1039%2Fc4cc02679a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=24947561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFWntrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2014&pages=8408-8411&author=G.+Upertauthor=G.+Mourierauthor=A.+Pastorauthor=M.+Verdenaudauthor=D.+Aliliauthor=D.+Serventauthor=N.+Gilles&title=High-Throughput+Production+of+Two+Disulphide-Bridge+Toxins&doi=10.1039%2Fc4cc02679a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput production of two disulphide-bridge toxins</span></div><div class="casAuthors">Upert, Gregory; Mourier, Gilles; Pastor, Alexandra; Verdenaud, Marion; Alili, Doria; Servent, Denis; Gilles, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">61</span>),
    <span class="NLM_cas:pages">8408-8411</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A quick and efficient prodn. compatible with high-throughput screening was developed using 36 toxins belonging to four different families of two disulfide-bridge toxins.  Final toxins were characterized using HPLC co-elution, CD and pharmacol. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzZqdjcWb5k7Vg90H21EOLACvtfcHk0lgwmLY-jh7yqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFWntrzP&md5=f71ace2e088f676b46a2d586cfb3c8ba</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc4cc02679a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc4cc02679a%26sid%3Dliteratum%253Aachs%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DPastor%26aufirst%3DA.%26aulast%3DVerdenaud%26aufirst%3DM.%26aulast%3DAlili%26aufirst%3DD.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DHigh-Throughput%2520Production%2520of%2520Two%2520Disulphide-Bridge%2520Toxins%26jtitle%3DChem.%2520Commun.%26date%3D2014%26volume%3D50%26spage%3D8408%26epage%3D8411%26doi%3D10.1039%2Fc4cc02679a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duhoo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequeira, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turchetto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peysson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brás, J. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darbon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, C. M. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Production of Oxidized Animal Toxins in Escherichia Coli</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2025</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-9624-7_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2F978-1-4939-9624-7_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=31267452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtF2lsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2025&publication_year=2019&pages=165-190&author=Y.+Duhooauthor=A.+F.+Sequeiraauthor=N.+J.+Saezauthor=J.+Turchettoauthor=L.+Ramondauthor=F.+Peyssonauthor=J.+L.+A.+Br%C3%A1sauthor=N.+Gillesauthor=H.+Darbonauthor=C.+M.+G.+A.+Fontesauthor=R.+Vincentelli&title=High-Throughput+Production+of+Oxidized+Animal+Toxins+in+Escherichia+Coli&doi=10.1007%2F978-1-4939-9624-7_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Production of Oxidized Animal Toxins in Escherichia coli</span></div><div class="casAuthors">Duhoo, Yoan; Sequeira, Ana Filipa; Saez, Natalie J.; Turchetto, Jeremy; Ramond, Laurie; Peysson, Fanny; Bras, Joana L. A.; Gilles, Nicolas; Darbon, Herve; Fontes, Carlos M. G. A.; Vincentelli, Renaud</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">2025</span>
        (<span class="NLM_cas:issue">High-Throughput Protein Production and Purification</span>),
    <span class="NLM_cas:pages">165-190</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  High-throughput prodn. (HTP) of synthetic genes is becoming an important tool to explore the biol. function of the extensive genomic and meta-genomic information currently available from various sources.  One such source is animal venom, which contains thousands of novel bioactive peptides with potential uses as novel therapeutics to treat a plethora of diseases as well as in environmentally benign bioinsecticide formulations.  Here, we describe a HTP platform for recombinant bacterial prodn. of oxidized disulfide-rich proteins and peptides from animal venoms.  High-throughput, host-optimized, gene synthesis and subcloning, combined with robust HTP expression and purifn. protocols, generate a semiautomated pipeline for the accelerated prodn. of proteins and peptides identified from genomic or transcriptomic libraries.  The platform has been applied to the prodn. of thousands of animal venom peptide toxins for the purposes of drug discovery, but has the power to be universally applicable for high-level prodn. of various and diverse target proteins in sol. form.  This chapter details the HTP protocol for gene synthesis and prodn., which supported high levels of peptide expression in the E. coli periplasm using a cleavable DsbC fusion.  Finally, target proteins and peptides are purified using automated HTP methods, before undergoing quality control and screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPARfxMXjPTrVg90H21EOLACvtfcHk0ljeu89yfBJbaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtF2lsLY%253D&md5=ccaa62b457c99f2c7552ebbd2e4c259b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-9624-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-9624-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DDuhoo%26aufirst%3DY.%26aulast%3DSequeira%26aufirst%3DA.%2BF.%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DTurchetto%26aufirst%3DJ.%26aulast%3DRamond%26aufirst%3DL.%26aulast%3DPeysson%26aufirst%3DF.%26aulast%3DBr%25C3%25A1s%26aufirst%3DJ.%2BL.%2BA.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDarbon%26aufirst%3DH.%26aulast%3DFontes%26aufirst%3DC.%2BM.%2BG.%2BA.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DHigh-Throughput%2520Production%2520of%2520Oxidized%2520Animal%2520Toxins%2520in%2520Escherichia%2520Coli%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2019%26volume%3D2025%26spage%3D165%26epage%3D190%26doi%3D10.1007%2F978-1-4939-9624-7_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sequeira, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turchetto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peysson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhoo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blémont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gama, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, L. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerreiro, C. I. P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darbon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, C. M. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">Gene Design, Fusion Technology and TEV Cleavage Conditions Influence the Purification of Oxidized Disulphide-Rich Venom Peptides in <i>Escherichia Coli</i></span>. <i>Microb. Cell Fact.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1186/s12934-016-0618-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1186%2Fs12934-016-0618-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28049473" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1-16&author=A.+F.+Sequeiraauthor=J.+Turchettoauthor=N.+J.+Saezauthor=F.+Peyssonauthor=L.+Ramondauthor=Y.+Duhooauthor=M.+Bl%C3%A9montauthor=V.+O.+Fernandesauthor=L.+T.+Gamaauthor=L.+M.+A.+Ferreiraauthor=C.+I.+P.+I.+Guerreiroauthor=N.+Gillesauthor=H.+Darbonauthor=C.+M.+G.+A.+Fontesauthor=R.+Vincentelli&title=Gene+Design%2C+Fusion+Technology+and+TEV+Cleavage+Conditions+Influence+the+Purification+of+Oxidized+Disulphide-Rich+Venom+Peptides+in+Escherichia+Coli&doi=10.1186%2Fs12934-016-0618-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1186%2Fs12934-016-0618-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12934-016-0618-0%26sid%3Dliteratum%253Aachs%26aulast%3DSequeira%26aufirst%3DA.%2BF.%26aulast%3DTurchetto%26aufirst%3DJ.%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DPeysson%26aufirst%3DF.%26aulast%3DRamond%26aufirst%3DL.%26aulast%3DDuhoo%26aufirst%3DY.%26aulast%3DBl%25C3%25A9mont%26aufirst%3DM.%26aulast%3DFernandes%26aufirst%3DV.%2BO.%26aulast%3DGama%26aufirst%3DL.%2BT.%26aulast%3DFerreira%26aufirst%3DL.%2BM.%2BA.%26aulast%3DGuerreiro%26aufirst%3DC.%2BI.%2BP.%2BI.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDarbon%26aufirst%3DH.%26aulast%3DFontes%26aufirst%3DC.%2BM.%2BG.%2BA.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DGene%2520Design%252C%2520Fusion%2520Technology%2520and%2520TEV%2520Cleavage%2520Conditions%2520Influence%2520the%2520Purification%2520of%2520Oxidized%2520Disulphide-Rich%2520Venom%2520Peptides%2520in%2520Escherichia%2520Coli%26jtitle%3DMicrob.%2520Cell%2520Fact.%26date%3D2017%26volume%3D16%26spage%3D1%26epage%3D16%26doi%3D10.1186%2Fs12934-016-0618-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turchetto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequeira, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramond, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peysson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brás, J. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhoo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blémont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerreiro, C. I. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darbon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontes, C. M. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">High-Throughput Expression of Animal Venom Toxins in <i>Escherichia Coli</i> to Generate a Large Library of Oxidized Disulphide-Reticulated Peptides for Drug Discovery</span>. <i>Microb. Cell Fact.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1186/s12934-016-0617-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1186%2Fs12934-016-0617-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28049473" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=1-15&author=J.+Turchettoauthor=A.+F.+Sequeiraauthor=L.+Ramondauthor=F.+Peyssonauthor=J.+L.+A.+Br%C3%A1sauthor=N.+J.+Saezauthor=Y.+Duhooauthor=M.+Bl%C3%A9montauthor=C.+I.+P.+D.+Guerreiroauthor=L.+Quintonauthor=E.+Pauwauthor=N.+Gillesauthor=H.+Darbonauthor=C.+M.+G.+A.+Fontesauthor=R.+Vincentelli&title=High-Throughput+Expression+of+Animal+Venom+Toxins+in+Escherichia+Coli+to+Generate+a+Large+Library+of+Oxidized+Disulphide-Reticulated+Peptides+for+Drug+Discovery&doi=10.1186%2Fs12934-016-0617-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1186%2Fs12934-016-0617-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12934-016-0617-1%26sid%3Dliteratum%253Aachs%26aulast%3DTurchetto%26aufirst%3DJ.%26aulast%3DSequeira%26aufirst%3DA.%2BF.%26aulast%3DRamond%26aufirst%3DL.%26aulast%3DPeysson%26aufirst%3DF.%26aulast%3DBr%25C3%25A1s%26aufirst%3DJ.%2BL.%2BA.%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DDuhoo%26aufirst%3DY.%26aulast%3DBl%25C3%25A9mont%26aufirst%3DM.%26aulast%3DGuerreiro%26aufirst%3DC.%2BI.%2BP.%2BD.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DPauw%26aufirst%3DE.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DDarbon%26aufirst%3DH.%26aulast%3DFontes%26aufirst%3DC.%2BM.%2BG.%2BA.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DHigh-Throughput%2520Expression%2520of%2520Animal%2520Venom%2520Toxins%2520in%2520Escherichia%2520Coli%2520to%2520Generate%2520a%2520Large%2520Library%2520of%2520Oxidized%2520Disulphide-Reticulated%2520Peptides%2520for%2520Drug%2520Discovery%26jtitle%3DMicrob.%2520Cell%2520Fact.%26date%3D2017%26volume%3D16%26spage%3D1%26epage%3D15%26doi%3D10.1186%2Fs12934-016-0617-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haler, J. R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upert, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smargiasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">Disulfide Connectivity Analysis of Peptides Bearing Two Intramolecular Disulfide Bonds Using MALDI In-Source Decay</span>. <i>J. Am. Soc. Mass Spectrom.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1995</span>– <span class="NLM_lpage">2002</span>, <span class="refDoi"> DOI: 10.1007/s13361-018-2022-y</span> </span><div class="citationLinks">[<a href="/doi/10.1007/s13361-018-2022-y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWgu7bI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=1995-2002&author=P.+Massonnetauthor=J.+R.+N.+Halerauthor=G.+Upertauthor=N.+Smargiassoauthor=G.+Mourierauthor=N.+Gillesauthor=L.+Quintonauthor=E.+De+Pauw&title=Disulfide+Connectivity+Analysis+of+Peptides+Bearing+Two+Intramolecular+Disulfide+Bonds+Using+MALDI+In-Source+Decay&doi=10.1007%2Fs13361-018-2022-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide Connectivity Analysis of Peptides Bearing Two Intramolecular Disulfide Bonds Using MALDI In-Source Decay</span></div><div class="casAuthors">Massonnet, Philippe; Haler, Jean R. N.; Upert, Gregory; Smargiasso, Nicolas; Mourier, Gilles; Gilles, Nicolas; Quinton, Loic; De Pauw, Edwin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Society for Mass Spectrometry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1995-2002</span>CODEN:
                <span class="NLM_cas:coden">JAMSEF</span>;
        ISSN:<span class="NLM_cas:issn">1044-0305</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Disulfide connectivity in peptides bearing at least two intramol. disulfide bonds is highly important for the structure and the biol. activity of the peptides.  In that context, anal. strategies allowing a characterization of the cysteine pairing are of prime interest for chemists, biochemists, and biologists.  For that purpose, this study evaluates the potential of MALDI in-source decay (ISD) for characterizing cysteine pairs through the systematic anal. of identical peptides bearing two disulfide bonds, but not the same cysteine connectivity.  Three different matrixes have been tested in pos. and/or in neg. mode (1,5-DAN, 2-AB and 2-AA).  As MALDI-ISD is known to partially reduce disulfide bonds, the data anal. of this study rests firstly on the deconvolution of the isotope pattern of the parent ions.  Moreover, data anal. is also based on the formed fragment ions and their signal intensities.  Results from MS/MS-expts. (MALDI-ISD-MS/MS) constitute the last ref. for data interpretation.  Owing to the combined use of different ISD-promoting matrixes, cysteine connectivity identification could be performed on the considered peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKmXEE2uTWfbVg90H21EOLACvtfcHk0ljeu89yfBJbaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWgu7bI&md5=b29930e096f4668a5b28c8d5b61d9056</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1007%2Fs13361-018-2022-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13361-018-2022-y%26sid%3Dliteratum%253Aachs%26aulast%3DMassonnet%26aufirst%3DP.%26aulast%3DHaler%26aufirst%3DJ.%2BR.%2BN.%26aulast%3DUpert%26aufirst%3DG.%26aulast%3DSmargiasso%26aufirst%3DN.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DDisulfide%2520Connectivity%2520Analysis%2520of%2520Peptides%2520Bearing%2520Two%2520Intramolecular%2520Disulfide%2520Bonds%2520Using%2520MALDI%2520In-Source%2520Decay%26jtitle%3DJ.%2520Am.%2520Soc.%2520Mass%2520Spectrom.%26date%3D2018%26volume%3D29%26spage%3D1995%26epage%3D2002%26doi%3D10.1007%2Fs13361-018-2022-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muratspahić, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freissmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, C. W.</span></span> <span> </span><span class="NLM_article-title">Nature-Derived Peptides: A Growing Niche for GPCR Ligand Discovery</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">309</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2019.03.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.tips.2019.03.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=30955896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVWnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=309-326&author=E.+Muratspahi%C4%87author=M.+Freissmuthauthor=C.+W.+Gruber&title=Nature-Derived+Peptides%3A+A+Growing+Niche+for+GPCR+Ligand+Discovery&doi=10.1016%2Fj.tips.2019.03.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Nature-Derived Peptides: A Growing Niche for GPCR Ligand Discovery</span></div><div class="casAuthors">Muratspahic, Edin; Freissmuth, Michael; Gruber, Christian W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">309-326</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) represent important drug targets, as they regulate pivotal physiol. processes and they have proved to be readily druggable.  Natural products have been and continue to be amongst the most valuable sources for drug discovery and development.  Here, we surveyed small mols. and (poly-)peptides derived from plants, animals, fungi, and bacteria, which modulate GPCR signaling.  Among naturally occurring compds., peptides from plants, cone-snails, snakes, spiders, scorpions, fungi, and bacteria are of particular interest as lead compds. for the development of GPCR ligands, since they cover a chem. space, which differs from that of synthetic small mols.  Peptides, however, face challenges, some of which can be overcome by studying plant-derived compds.  We argue here that the opportunities outweigh the challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCENhph4se5bVg90H21EOLACvtfcHk0lgQ9CHFrYNVcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVWnsr8%253D&md5=19bb90ea2b7e181a036f3ebc8668c419</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2019.03.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2019.03.004%26sid%3Dliteratum%253Aachs%26aulast%3DMuratspahi%25C4%2587%26aufirst%3DE.%26aulast%3DFreissmuth%26aufirst%3DM.%26aulast%3DGruber%26aufirst%3DC.%2BW.%26atitle%3DNature-Derived%2520Peptides%253A%2520A%2520Growing%2520Niche%2520for%2520GPCR%2520Ligand%2520Discovery%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2019%26volume%3D40%26spage%3D309%26epage%3D326%26doi%3D10.1016%2Fj.tips.2019.03.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span> <span> </span><span class="NLM_article-title">Function and Solution Structure of Huwentoxin-X, a Specific Blocker of N-Type Calcium Channels, from the Chinese Bird Spider <i>Ornithoctonus Huwena</i></span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">8628</span>– <span class="NLM_lpage">8635</span>, <span class="refDoi"> DOI: 10.1074/jbc.M513542200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.M513542200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=16439354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD28XivVCnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=8628-8635&author=Z.+Liuauthor=J.+Daiauthor=L.+Daiauthor=M.+Dengauthor=Z.+Huauthor=W.+Huauthor=S.+Liang&title=Function+and+Solution+Structure+of+Huwentoxin-X%2C+a+Specific+Blocker+of+N-Type+Calcium+Channels%2C+from+the+Chinese+Bird+Spider+Ornithoctonus+Huwena&doi=10.1074%2Fjbc.M513542200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Function and Solution Structure of Huwentoxin-X, a Specific Blocker of N-type Calcium Channels, from the Chinese Bird Spider Ornithoctonus huwena</span></div><div class="casAuthors">Liu, Zhonghua; Dai, Jie; Dai, Longjun; Deng, Meichun; Hu, Zhe; Hu, Weijun; Liang, Songping</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8628-8635</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Huwentoxin-X (HWTX-X) is a novel peptide toxin, purified from the venom of the spider Ornithoctonus huwena.  It comprises 28 amino acid residues including six cysteine residues as disulfide bridges linked in the pattern of I-IV, II-V, and III-VI.  Its cDNA, detd. by rapid amplification of 3' and 5' cDNA ends, encodes a 68-residue prepropeptide.  HWTX-X shares low sequence homol. with ω-conotoxins GVIA and MVIIA, two well known blockers of N-type Ca2+ channels.  Nevertheless, whole cell studies indicate that it can block N-type Ca2+ channels in rat dorsal root ganglion cells (IC50 40 nM) and the blockage by HWTX-X is completely reversible.  The rank order of specificity for N-type Ca2+ channels is GVIA ≈ HWTX-X > MVIIA.  In contrast to GVIA and MVIIA, HWTX-X had no detectable effect on the twitch response of rat vas deferens to low frequency elec. stimulation, indicating that HWTX-X has different selectivity for isoforms of N-type Ca2+ channels, compared with GVIA or MVIIA.  A comparison of the structures of HWTX-X and GVIA reveals that they not only adopt a common structural motif (inhibitor cystine knot), but also have a similar functional motif, a binding surface formed by the crit. residue Tyr, and several basic residues.  However, the dissimilarities of their binding surfaces provide some insights into their different selectivities for isoforms of N-type Ca2+ channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVbvZ_5V2kK7Vg90H21EOLACvtfcHk0lgQ9CHFrYNVcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivVCnsrw%253D&md5=2fd8b3f315d5ee0a4602d8627eb0db93</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M513542200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M513542200%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DLiang%26aufirst%3DS.%26atitle%3DFunction%2520and%2520Solution%2520Structure%2520of%2520Huwentoxin-X%252C%2520a%2520Specific%2520Blocker%2520of%2520N-Type%2520Calcium%2520Channels%252C%2520from%2520the%2520Chinese%2520Bird%2520Spider%2520Ornithoctonus%2520Huwena%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D8628%26epage%3D8635%26doi%3D10.1074%2Fjbc.M513542200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Favreau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamthanh, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bournaud, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimahara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laboute, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letourneux, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molgó, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Gall, F.</span></span> <span> </span><span class="NLM_article-title">A New ω-Conotoxin That Targets N-Type Voltage-Sensitive Calcium Channels with Unusual Specificity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">14567</span>– <span class="NLM_lpage">14575</span>, <span class="refDoi"> DOI: 10.1021/bi002871r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi002871r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD3MXotV2ksrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=14567-14575&author=P.+Favreauauthor=N.+Gillesauthor=H.+Lamthanhauthor=R.+Bournaudauthor=T.+Shimaharaauthor=F.+Bouetauthor=P.+Labouteauthor=Y.+Letourneuxauthor=A.+M%C3%A9nezauthor=J.+Molg%C3%B3author=F.+Le+Gall&title=A+New+%CF%89-Conotoxin+That+Targets+N-Type+Voltage-Sensitive+Calcium+Channels+with+Unusual+Specificity&doi=10.1021%2Fbi002871r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A New ω-Conotoxin That Targets N-Type Voltage-Sensitive Calcium Channels with Unusual Specificity</span></div><div class="casAuthors">Favreau, Philippe; Gilles, Nicolas; Lamthanh, Hung; Bournaud, Roland; Shimahara, Takeshi; Bouet, Francoise; Laboute, Pierre; Letourneux, Yves; Menez, Andre; Molgo, Jordi; Le Gall, Frederic</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">14567-14575</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new specific voltage-sensitive calcium channel (VSCC) blocker has been isolated from the venom of the fish-hunting cone snail Conus consors.  This peptide, named ω-Ctx CNVIIA, consists of 27 amino acid residues folded by 3 disulfide bridges.  Interestingly, loop 4, which is supposed to be crucial for selectivity, shows an unusual sequence (SSSKGR).  The synthesis of the linear peptide was performed using the Fmoc strategy, and the correct folding was achieved in the presence of guanidinium chloride, potassium buffer, and reduced/oxidized glutathione at 4° for 3 days.  Both synthetic and native toxin caused an intense shaking activity, characteristic of ω-conotoxins targeting N-type VSCC when injected intracerebroventricularly to mice.  Binding studies on rat brain synaptosomes revealed that the radioiodinated ω-Ctx CNVIIA specifically and reversibly binds to high-affinity sites with a Kd of 36.3 pM.  Its binding is competitive with ω-Ctx MVIIA at low concn. (Ki = 2 pM).  Moreover, ω-Ctx CNVIIA exhibits a clear selectivity for N-type VSCCs vs. P/Q-type VSCCs targeted resp. by radioiodinated ω-Ctx GVIA and ω-Ctx MVIIC.  Although ω-Ctx CNVIIA clearly blocked N-type Ca2+ current in chromaffin cells, this toxin did not inhibit acetylcholine release evoked by nerve stimuli at the frog neuromuscular junction, in marked contrast to ω-Ctx GVIA.  ω-Ctx CNVIIA thus represents a new selective tool for blocking N-type VSCC that displays a unique pharmacol. profile and highlights the diversity of voltage-sensitive Ca channels in the animal kingdom.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPGQafn7SVfLVg90H21EOLACvtfcHk0lgQ9CHFrYNVcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXotV2ksrs%253D&md5=45455a19156ab15af2db65c64834e2f8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi002871r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi002871r%26sid%3Dliteratum%253Aachs%26aulast%3DFavreau%26aufirst%3DP.%26aulast%3DGilles%26aufirst%3DN.%26aulast%3DLamthanh%26aufirst%3DH.%26aulast%3DBournaud%26aufirst%3DR.%26aulast%3DShimahara%26aufirst%3DT.%26aulast%3DBouet%26aufirst%3DF.%26aulast%3DLaboute%26aufirst%3DP.%26aulast%3DLetourneux%26aufirst%3DY.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26aulast%3DMolg%25C3%25B3%26aufirst%3DJ.%26aulast%3DLe%2BGall%26aufirst%3DF.%26atitle%3DA%2520New%2520%25CF%2589-Conotoxin%2520That%2520Targets%2520N-Type%2520Voltage-Sensitive%2520Calcium%2520Channels%2520with%2520Unusual%2520Specificity%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D14567%26epage%3D14575%26doi%3D10.1021%2Fbi002871r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Q.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis of Toxicity of N-Type Calcium Channel Inhibitor MVIIA</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2015.08.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.neuropharm.2015.08.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=26344359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFGhtbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2016&pages=137-145&author=F.+Wangauthor=Z.+Yanauthor=Z.+Liuauthor=S.+Wangauthor=Q.+Wuauthor=S.+Yuauthor=J.+Dingauthor=Q.+Dai&title=Molecular+Basis+of+Toxicity+of+N-Type+Calcium+Channel+Inhibitor+MVIIA&doi=10.1016%2Fj.neuropharm.2015.08.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of toxicity of N-type calcium channel inhibitor MVIIA</span></div><div class="casAuthors">Wang, Fei; Yan, Zhenzhen; Liu, Zhuguo; Wang, Sheng; Wu, Qiaoling; Yu, Shuo; Ding, Jiuping; Dai, Qiuyun</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137-145</span>CODEN:
                <span class="NLM_cas:coden">NEPHBW</span>;
        ISSN:<span class="NLM_cas:issn">0028-3908</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">MVIIA (ziconotide) is a specific inhibitor of N-type calcium channel, Cav2.2.  It is derived from Cone snail and currently used for the treatment of severe chronic pains in patients unresponsive to opioid therapy.  However, MVIIA produces severe side-effects, including dizziness, nystagmus, somnolence, abnormal gait, and ataxia, that limit its wider application.  We previously identified a novel inhibitor of Cav2.2, ω-conopeptide SO-3, which possesses similar structure and analgesic activity to MVIIA's.  To investigate the key residues for MVIIA toxicity, MVIIA/SO-3 hybrids and MVIIA variants carrying mutations in its loop 2 were synthesized.  The substitution of MVIIA's loop 1 with the loop 1 of SO-3 resulted in significantly reduced Cav2.2 binding activity in vitro; the replacement of MVIIA loop 2 by the loop 2 of SO-3 not only enhanced the peptide/Cav2.2 binding but also decreased its toxicity on goldfish, attenuated mouse tremor symptom, spontaneous locomotor activity, and coordinated locomotion function.  Further mutation anal. and mol. calcn. revealed that the toxicity of MVIIA mainly arose from Met12 in the loop 2, and this residue inserts into a hydrophobic hole (Ile300, Phe302 and Leu305) located between repeats II and III of Cav2.2.  The combinative mutations of the loop 2 of MVIIA or other ω-conopeptides may be used for future development of more effective Cav2.2 inhibitors with lower side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn3CuU5x-TB7Vg90H21EOLACvtfcHk0lgQEQzXG36mVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFGhtbrJ&md5=0123de2b1d8af2ddd90b977940f12f88</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2015.08.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2015.08.047%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DQ.%26atitle%3DMolecular%2520Basis%2520of%2520Toxicity%2520of%2520N-Type%2520Calcium%2520Channel%2520Inhibitor%2520MVIIA%26jtitle%3DNeuropharmacology%26date%3D2016%26volume%3D101%26spage%3D137%26epage%3D145%26doi%3D10.1016%2Fj.neuropharm.2015.08.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huys, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyason, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waelkens, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Zyl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">du Plessis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Walt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clynen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoofs, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span> <span> </span><span class="NLM_article-title">Purification, Characterization and Biosynthesis of Parabutoxin 3, a Component of <i>Parabuthus Transvaalicus</i> Venom</span>. <i>Eur. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>269</i></span>,  <span class="NLM_fpage">1854</span>– <span class="NLM_lpage">1865</span>, <span class="refDoi"> DOI: 10.1046/j.1432-1033.2002.02833.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1046%2Fj.1432-1033.2002.02833.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=11952787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD38XjtVOns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=2002&pages=1854-1865&author=I.+Huysauthor=K.+Dyasonauthor=E.+Waelkensauthor=F.+Verdonckauthor=J.+van+Zylauthor=J.+du+Plessisauthor=G.+J.+M%C3%BCllerauthor=J.+van+der+Waltauthor=E.+Clynenauthor=L.+Schoofsauthor=J.+Tytgat&title=Purification%2C+Characterization+and+Biosynthesis+of+Parabutoxin+3%2C+a+Component+of+Parabuthus+Transvaalicus+Venom&doi=10.1046%2Fj.1432-1033.2002.02833.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Purification, characterization and biosynthesis of parabutoxin 3, a component of Parabuthus transvaalicus venom</span></div><div class="casAuthors">Huys, Isabelle; Dyason, Karin; Waelkens, Etienne; Verdonck, Fons; Van Zyl, Johann; du Plessis, Johan; Muller, Gert J.; Van der Walt, Jurg; Clynen, Elke; Schoofs, Liliane; Tytgat, Jan</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">269</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1854-1865</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A novel peptidyl inhibitor of voltage-gated K+ channels, named parabutoxin 3 (PBTx3), has been purified to homogeneity from the venom of Parabuthus transvaalicus.  This scorpion toxin contains 37 residues, has a mass of 4274 Da and displays 41% identity with charybdotoxin (ChTx, also called "α-KTx1.1").  PBTx3 is the tenth member (called "α-KTx1.10") of subfamily 1 of K+ channel-blocking peptides known thus far.  Electrophysiol. expts. using Xenopus laevis oocytes indicate that PBTx3 is an inhibitor of Kv1 channels (Kv1.1, Kv1.2, Kv1.3), but has no detectable effects on Kir-type and ERG-type channels.  The dissocn. consts. (Kd) for Kv1.1, Kv1.2 and Kv1.3 channels are, resp., 79 μM, 547 nM and 492 nM.  A synthetic gene encoding a PBTx3 homolog was designed and expressed as a fusion protein with the maltose-binding protein (MBP) in Escherichia coli.  The recombinant protein was purified from the bacterial periplasm compartment using an amylose affinity resin column, followed by a gel filtration purifn. step and cleavage by factor Xa (fXa) to release the recombinant toxin peptide (rPBTx3).  After final purifn. and refolding, rPBTx3 was shown to be identical to the native PBTx3 with respect to HPLC retention time, mass spectrometric anal. and functional properties.  The three-dimensional structure of PBTx3 is proposed by homol. modeling to contain a double-stranded antiparallel β sheet and a single α-helix, connected by three disulfide bridges.  The scaffold of PBTx3 is homologous to most other α-KTx scorpion toxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUvdZ9pBYMmrVg90H21EOLACvtfcHk0lgQEQzXG36mVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjtVOns7o%253D&md5=f6a41f19d6dec99566b01173369fc386</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1033.2002.02833.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1033.2002.02833.x%26sid%3Dliteratum%253Aachs%26aulast%3DHuys%26aufirst%3DI.%26aulast%3DDyason%26aufirst%3DK.%26aulast%3DWaelkens%26aufirst%3DE.%26aulast%3DVerdonck%26aufirst%3DF.%26aulast%3Dvan%2BZyl%26aufirst%3DJ.%26aulast%3Ddu%2BPlessis%26aufirst%3DJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DG.%2BJ.%26aulast%3Dvan%2Bder%2BWalt%26aufirst%3DJ.%26aulast%3DClynen%26aufirst%3DE.%26aulast%3DSchoofs%26aufirst%3DL.%26aulast%3DTytgat%26aufirst%3DJ.%26atitle%3DPurification%252C%2520Characterization%2520and%2520Biosynthesis%2520of%2520Parabutoxin%25203%252C%2520a%2520Component%2520of%2520Parabuthus%2520Transvaalicus%2520Venom%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D2002%26volume%3D269%26spage%3D1854%26epage%3D1865%26doi%3D10.1046%2Fj.1432-1033.2002.02833.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Papp, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varga, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herceg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Román-González, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panyi, G.</span></span> <span> </span><span class="NLM_article-title">Tst26, a Novel Peptide Blocker of Kv1.2 and Kv1.3 Channels from the Venom of <i>Tityus Stigmurus</i></span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">379</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2009.05.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2009.05.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19500613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1OktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2009&pages=379-389&author=F.+Pappauthor=C.+V.+F.+Batistaauthor=Z.+Vargaauthor=M.+Hercegauthor=S.+A.+Rom%C3%A1n-Gonz%C3%A1lezauthor=R.+Gasparauthor=L.+D.+Possaniauthor=G.+Panyi&title=Tst26%2C+a+Novel+Peptide+Blocker+of+Kv1.2+and+Kv1.3+Channels+from+the+Venom+of+Tityus+Stigmurus&doi=10.1016%2Fj.toxicon.2009.05.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Tst26, a novel peptide blocker of Kv1.2 and Kv1.3 channels from the venom of Tityus stigmurus</span></div><div class="casAuthors">Papp, Ferenc; Batista, Cesar V. F.; Varga, Zoltan; Herceg, Monika; Roman-Gonzalez, Sergio A.; Gaspar, Rezso; Possani, Lourival D.; Panyi, Gyorgy</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">379-389</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Using high-performance liq. chromatog. Tst26, a novel potassium channel blocker peptide, was purified from the venom of the Brazilian scorpion Tityus stigmurus.  Its primary structure was detd. by means of automatic Edman degrdn. and mass spectrometry anal.  The peptide is composed of 37 amino acid residues and tightly folded through three disulfide bridges, similar to other K+ channel blocking peptides purified from scorpion venoms.  It contains the "essential dyad" for K+ channel recognition comprised of a lysine at position 27 and a tyrosine at position 36.  Electrophysiol. assays revealed that Tst26 blocked hKv1.2 and hKv1.3 channels with high affinity (Kd = 1.9 nM and 10.7 nM, resp.) while it did not affect several other ion channels (mKv1.1, hKv1.4, hKv1.5, hERG, hIKCa1, hBK, hNav1.5) tested at 10 nM concn.  The voltage-dependent steady-state parameters of K+ channel gating were unaffected by the toxin in both channels, but due to the fast assocn. and dissocn. kinetics Tst26 slowed the rate of inactivation of Kv1.3 channels.  Based on the primary structure, the systematic nomenclature proposed for this peptide is α-KTx 4.6.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5q0vBfePfq7Vg90H21EOLACvtfcHk0lgQEQzXG36mVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1OktLg%253D&md5=3c87f27791af485ddeed6a2825f3b04f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2009.05.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2009.05.023%26sid%3Dliteratum%253Aachs%26aulast%3DPapp%26aufirst%3DF.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DVarga%26aufirst%3DZ.%26aulast%3DHerceg%26aufirst%3DM.%26aulast%3DRom%25C3%25A1n-Gonz%25C3%25A1lez%26aufirst%3DS.%2BA.%26aulast%3DGaspar%26aufirst%3DR.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26aulast%3DPanyi%26aufirst%3DG.%26atitle%3DTst26%252C%2520a%2520Novel%2520Peptide%2520Blocker%2520of%2520Kv1.2%2520and%2520Kv1.3%2520Channels%2520from%2520the%2520Venom%2520of%2520Tityus%2520Stigmurus%26jtitle%3DToxicon%26date%3D2009%26volume%3D54%26spage%3D379%26epage%3D389%26doi%3D10.1016%2Fj.toxicon.2009.05.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogowski, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, M. P.</span></span> <span> </span><span class="NLM_article-title">Tityustoxin K Alpha Blocks Voltage-Gated Noninactivating K+ Channels and Unblocks Inactivating K+ Channels Blocked by Alpha-Dendrotoxin in Synaptosomes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1479</span>, <span class="refDoi"> DOI: 10.1073/pnas.91.4.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.91.4.1475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=7509073" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2cXis1entro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=1475-1479&author=R.+S.+Rogowskiauthor=B.+K.+Kruegerauthor=J.+H.+Collinsauthor=M.+P.+Blaustein&title=Tityustoxin+K+Alpha+Blocks+Voltage-Gated+Noninactivating+K%2B+Channels+and+Unblocks+Inactivating+K%2B+Channels+Blocked+by+Alpha-Dendrotoxin+in+Synaptosomes&doi=10.1073%2Fpnas.91.4.1475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Tityustoxin Kα blocks voltage-gated noninactivating K+ channels and unblocks inactivating K+ channels blocked by α-dendrotoxin in synaptosomes</span></div><div class="casAuthors">Rogowski, Robert S.; Krueger, Bruce K.; Collins, John H.; Blaustein, Mordecai P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1475-9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Two nonhomologs polypeptide toxins, tityustoxin Kα (TsTX-Kα) and tityustoxin Kβ (TsTX-Kβ), purified from the venom of the Brazilian scorpion Tityus serrulatus, selectively block voltage-gated noninactivating K+ channels in synaptosomes (IC50 values of 8 nM and 30 nM, resp.).  In contrast, α-dendrotoxin (α-DTX) and charybdotoxin (ChTX) block voltage-gated inactivating K+ channels in synaptosomes (IC50 values of 90 nM and 40 nM, resp.).  The authors studied interactions among these toxins in 125I-α-DTX binding and 86Rb efflux expts.  Both TsTX-Kα and ChTX completely displaced specifically bound 125I-α-DTX from synaptic membranes, but TsTX-Kβ had no effect on bound α-DTX.  TsTX-Kα and TsTX-Kβ blocked the same noninactivating component of 100 mM K+-stimulated 86Rb efflux in synaptosomes.  Both α-DTX and ChTX blocked the same inactivating component of the K+-stimulated 86Rb efflux in synaptosomes.  Both the inactivating and the noninactivating components of the 100 mM K+-stimulated 86Rb efflux were completely blocked when 200 mM TsTX-Kβ and either 600 nM α-DTX or 200 nM ChTX were present.  The effects of TsTX-Kα was added in the presence of α-DTX, however, only the noninactivating component of the K+-stimulated efflux was blocked.  The inactivating component could then be blocked by ChTX, which is structurally homologs to TsTX-Kα.  The authors conclude that TsTX-Kα unblocks the voltage-gated inactivating K+ channels in synaptosomes when they are blocked by α-DTX, but not when they are blocked by ChTX.  TsTX-Kα binds to a site on the inactivating K+ channel that does not occlude the pore; its binding apparently prevents α-DTX (7054 Da), but not ChTX (4300 Da), from blocking the pore.  The effects of TsTX-Kα on 125I-α-DTX binding and 86Rb efflux are mimicked by noxiustoxin, which is homologs to TsTX-Kα and ChTX.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOAe4ArbxDzLVg90H21EOLACvtfcHk0lhOODtpX9IocQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXis1entro%253D&md5=2885b62e5cdb7d1fdd9e82ff354b2fd1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.4.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.4.1475%26sid%3Dliteratum%253Aachs%26aulast%3DRogowski%26aufirst%3DR.%2BS.%26aulast%3DKrueger%26aufirst%3DB.%2BK.%26aulast%3DCollins%26aufirst%3DJ.%2BH.%26aulast%3DBlaustein%26aufirst%3DM.%2BP.%26atitle%3DTityustoxin%2520K%2520Alpha%2520Blocks%2520Voltage-Gated%2520Noninactivating%2520K%252B%2520Channels%2520and%2520Unblocks%2520Inactivating%2520K%252B%2520Channels%2520Blocked%2520by%2520Alpha-Dendrotoxin%2520in%2520Synaptosomes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1994%26volume%3D91%26spage%3D1475%26epage%3D1479%26doi%3D10.1073%2Fpnas.91.4.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerni, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucca, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cremonez, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordon, K. C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arantes, E. C.</span></span> <span> </span><span class="NLM_article-title">Electrophysiological Characterization of Ts6 and Ts7, K+ Channel Toxins Isolated through an Improved Tityus Serrulatus Venom Purification Procedure</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">913</span>, <span class="refDoi"> DOI: 10.3390/toxins6030892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Ftoxins6030892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=24590385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=892-913&author=F.+A.+Cerniauthor=M.+B.+Puccaauthor=S.+Peigneurauthor=C.+M.+Cremonezauthor=K.+C.+F.+Bordonauthor=J.+Tytgatauthor=E.+C.+Arantes&title=Electrophysiological+Characterization+of+Ts6+and+Ts7%2C+K%2B+Channel+Toxins+Isolated+through+an+Improved+Tityus+Serrulatus+Venom+Purification+Procedure&doi=10.3390%2Ftoxins6030892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophysiological characterization of Ts6 and Ts7, K+ channel toxins isolated through an improved Tityus serrulatus venom purification procedure</span></div><div class="casAuthors">Cerni, Felipe A.; Pucca, Manuela B.; Peigneur, Steve; Cremonez, Caroline M.; Bordon, Karla C. F.; Tytgat, Jan; Arantes, Eliane C.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">892-913, 22 pp.</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In Brazil, Tityus serrulatus (Ts) is the species responsible for most of the scorpion related accidents.  Among the Ts toxins, the neurotoxins with action on potassium channels (α-KTx) present high interest, due to their effect in the envenoming process and the ion channel specificity they display.  The α-KTx toxins family is the most relevant because its toxins can be used as therapeutic tools for specific target cells.  The improved isolation method provided toxins with high resoln., obtaining pure Ts6 and Ts7 in two chromatog. steps.  The effects of Ts6 and Ts7 toxins were evaluated in 14 different types of potassium channels using the voltage-clamp technique with two-microelectrodes.  Ts6 toxin shows high affinity for Kv1.2, Kv1.3 and Shaker IR, blocking these channels in low concns.  Moreover, Ts6 blocks the Kv1.3 channel in picomolar concns. with an IC50 of 0.55 nM and therefore could be of valuable assistance to further designing immunosuppressive therapeutics.  Ts7 toxin blocks multiple subtypes channels, showing low selectivity among the channels analyzed.  This work also stands out in its attempt to elucidate the residues important for interacting with each channel and, in the near future, to model a desired drug.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbFM-3lmyazbVg90H21EOLACvtfcHk0lhOODtpX9IocQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtlyju7Y%253D&md5=ee62c4fee705205b8dbc24131693e39d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.3390%2Ftoxins6030892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins6030892%26sid%3Dliteratum%253Aachs%26aulast%3DCerni%26aufirst%3DF.%2BA.%26aulast%3DPucca%26aufirst%3DM.%2BB.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DCremonez%26aufirst%3DC.%2BM.%26aulast%3DBordon%26aufirst%3DK.%2BC.%2BF.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DArantes%26aufirst%3DE.%2BC.%26atitle%3DElectrophysiological%2520Characterization%2520of%2520Ts6%2520and%2520Ts7%252C%2520K%252B%2520Channel%2520Toxins%2520Isolated%2520through%2520an%2520Improved%2520Tityus%2520Serrulatus%2520Venom%2520Purification%2520Procedure%26jtitle%3DToxins%26date%3D2014%26volume%3D6%26spage%3D892%26epage%3D913%26doi%3D10.3390%2Ftoxins6030892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, A. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arantes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monje, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stühmer, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varanda, W. A.</span></span> <span> </span><span class="NLM_article-title">Tityustoxin-K(Alpha) Blockade of the Voltage-Gated Potassium Channel Kv1.3</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">1180</span>– <span class="NLM_lpage">1186</span>, <span class="refDoi"> DOI: 10.1038/sj.bjp.0705343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1038%2Fsj.bjp.0705343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=12871837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmtFemtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2003&pages=1180-1186&author=A.+R.+A.+Rodriguesauthor=E.+C.+Arantesauthor=F.+Monjeauthor=W.+St%C3%BChmerauthor=W.+A.+Varanda&title=Tityustoxin-K%28Alpha%29+Blockade+of+the+Voltage-Gated+Potassium+Channel+Kv1.3&doi=10.1038%2Fsj.bjp.0705343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Tityustoxin-K(alpha) blockade of the voltage-gated potassium channel Kv1.3</span></div><div class="casAuthors">Rodrigues, Aldo Rogelis A.; Arantes, Eliane C.; Monje, Francisco; Stuehmer, Walter; Varanda, Wamberto Antonio</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1180-1186</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We investigated the action of TsTX-Kα on cloned Kv1.3 channels of the Shaker subfamily of voltage-gated potassium channels, using the voltage-clamp technique.  Highly purified TsTX-Kα was obtained from the venom of the Brazilian scorpion Tityus serrulatus using a new purifn. protocol.  Our results show that TsTX-Kα blocks Kv1.3 with high affinity in two expression systems.  TsTX-Kα blockade of Kv1.3 channels expressed in Xenopus oocytes was found to be completely reversible and to exhibit a pH dependence.  The KD was 3.9 nM at pH 7.5, 9.5 nM at pH 7.0 and 94.5 nM at pH 6.5.  The blocking properties of TsTX-Kα in a mammalian cell line (L929), stably transfected to express Kv1.3, were studied using the patch-clamp technique.  In this prepn., the toxin had a KD of 19.8 nM at pH 7.4.  TsTX-Kα was found to affect neither the voltage-dependence of activation, nor the activation and deactivation time consts.  The block appeared to be independent of the transmembrane voltage and the toxin did not interfere with the C-type inactivation process.  Taken as a whole, our findings indicate that TsTX-Kα acts as a simple blocker of Kv1.3 channels.  It is concluded that this toxin is a useful tool for probing not only the physiol. roles of Kv1.2, but also those mediated by Kv1.3 channels.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrB96JwGIv6VLVg90H21EOLACvtfcHk0lhOODtpX9IocQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmtFemtLo%253D&md5=1d5582adac27db363d214a7be30be1eb</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705343%26sid%3Dliteratum%253Aachs%26aulast%3DRodrigues%26aufirst%3DA.%2BR.%2BA.%26aulast%3DArantes%26aufirst%3DE.%2BC.%26aulast%3DMonje%26aufirst%3DF.%26aulast%3DSt%25C3%25BChmer%26aufirst%3DW.%26aulast%3DVaranda%26aufirst%3DW.%2BA.%26atitle%3DTityustoxin-K%2528Alpha%2529%2520Blockade%2520of%2520the%2520Voltage-Gated%2520Potassium%2520Channel%2520Kv1.3%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D139%26spage%3D1180%26epage%3D1186%26doi%3D10.1038%2Fsj.bjp.0705343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diego-García, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista, C. V. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Gómez, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Candido, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-Lagunas, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possani, L. D.</span></span> <span> </span><span class="NLM_article-title">The Brazilian Scorpion <i>Tityus Costatus</i> Karsch: Genes, Peptides and Function</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2004.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2004.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=15683865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpt1altQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=273-283&author=E.+Diego-Garc%C3%ADaauthor=C.+V.+F.+Batistaauthor=B.+I.+Garc%C3%ADa-G%C3%B3mezauthor=S.+Lucasauthor=D.+M.+Candidoauthor=F.+G%C3%B3mez-Lagunasauthor=L.+D.+Possani&title=The+Brazilian+Scorpion+Tityus+Costatus+Karsch%3A+Genes%2C+Peptides+and+Function&doi=10.1016%2Fj.toxicon.2004.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">The Brazilian scorpion Tityus costatus Karsch: genes, peptides and function</span></div><div class="casAuthors">Diego-Garcia, Elia; Batista, Cesar V. F.; Garcia-Gomez, Blanca I.; Lucas, Sylvia; Candido, Denise M.; Gomez-Lagunas, Froylan; Possani, Lourival D.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">273-283</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The venom of the scorpion Tityus costatus contains peptides toxic to humans but scarce information on their structure and function is available.  Here, we report the sepn. of 50 different components by high performance liq. chromatog. and the identification of approx. 90 distinct components by mass spectrometry anal., with mol. wts. varying from 413 to 45482 amu.  Four peptides were fully sequenced: (i) a butantoxin-like peptide that blocks Shaker K+ channel; (ii) an insect toxin-like peptide; (iii) a scorpine-like peptide, and a short heptapeptide of unknown function.  Fifteen peptides were directly sequenced at the N-terminal region, among which are components toxic to mice.  A cDNA library was constructed and 13 clones were isolated and sequenced.  Some of these peptides and genes are similar to other known scorpion toxins.  Based on these results, stings by scorpions of the species Tityus costatus should be taken with caution by medical doctors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTT1RQ9IGqprVg90H21EOLACvtfcHk0li1MESGz4is_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpt1altQ%253D%253D&md5=dcc648d2e72dcffa0a4444de103c718e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2004.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2004.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DDiego-Garc%25C3%25ADa%26aufirst%3DE.%26aulast%3DBatista%26aufirst%3DC.%2BV.%2BF.%26aulast%3DGarc%25C3%25ADa-G%25C3%25B3mez%26aufirst%3DB.%2BI.%26aulast%3DLucas%26aufirst%3DS.%26aulast%3DCandido%26aufirst%3DD.%2BM.%26aulast%3DG%25C3%25B3mez-Lagunas%26aufirst%3DF.%26aulast%3DPossani%26aufirst%3DL.%2BD.%26atitle%3DThe%2520Brazilian%2520Scorpion%2520Tityus%2520Costatus%2520Karsch%253A%2520Genes%252C%2520Peptides%2520and%2520Function%26jtitle%3DToxicon%26date%3D2005%26volume%3D45%26spage%3D273%26epage%3D283%26doi%3D10.1016%2Fj.toxicon.2004.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Calvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKinnon, R.</span></span> <span> </span><span class="NLM_article-title">Purification and Characterization of Three Inhibitors of Voltage-Dependent K+ Channels from Leiurus Quinquestriatus Var. Hebraeus Venom</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">6834</span>– <span class="NLM_lpage">6839</span>, <span class="refDoi"> DOI: 10.1021/bi00188a012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00188a012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2cXktVeju74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1994&pages=6834-6839&author=M.+L.+Garciaauthor=M.+Garcia-Calvoauthor=P.+Hidalgoauthor=A.+Leeauthor=R.+MacKinnon&title=Purification+and+Characterization+of+Three+Inhibitors+of+Voltage-Dependent+K%2B+Channels+from+Leiurus+Quinquestriatus+Var.+Hebraeus+Venom&doi=10.1021%2Fbi00188a012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and Characterization of Three Inhibitors of Voltage-Dependent K+ Channels from Leiurus Quinquestriatus var. Hebraeus Venom</span></div><div class="casAuthors">Garcia, Maria L.; Garcia-Calvo, Margarita; Hidalgo, Patricia; Lee, Alice; MacKinnon, Roderick</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6834-9</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Three new toxins from the venom of the scorpion Leiurus quinquestriatus var. hebraeus have been identified on the basis of their ability to block the Shaker K+ channel.  These toxins have been purified using HPLC techniques and characterized as 38 amino acid peptides by mass spectroscopy, amino acid anal., and sequence detn.  Their chem. identity was confirmed by producing fully functional synthetic toxins using recombinant methods.  These peptides are potent inhibitors of the Shaker K+ channel (Kd < 1 nM) as well as the mammalian homologues of Shaker.  They are related to other previously described K+ channel toxins, but form a new subclass within the larger family of K+ channel inhibitors derived from scorpion venom.  The authors have named these toxins agitoxin 1, 2, and 3, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3vGX6FDTOfrVg90H21EOLACvtfcHk0li1MESGz4is_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVeju74%253D&md5=ceb54a549da2b1b1b2b30a56a0cd78d1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi00188a012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00188a012%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia%26aufirst%3DM.%2BL.%26aulast%3DGarcia-Calvo%26aufirst%3DM.%26aulast%3DHidalgo%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DMacKinnon%26aufirst%3DR.%26atitle%3DPurification%2520and%2520Characterization%2520of%2520Three%2520Inhibitors%2520of%2520Voltage-Dependent%2520K%252B%2520Channels%2520from%2520Leiurus%2520Quinquestriatus%2520Var.%2520Hebraeus%2520Venom%26jtitle%3DBiochemistry%26date%3D1994%26volume%3D33%26spage%3D6834%26epage%3D6839%26doi%3D10.1021%2Fbi00188a012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takacs, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toups, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollewe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuello, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driessens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biancalana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponte, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perozo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajewski, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suarez-Kurtz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, S. A. N.</span></span> <span> </span><span class="NLM_article-title">A Designer Ligand Specific for Kv1.3 Channels from a Scorpion Neurotoxin-Based Library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">22211</span>– <span class="NLM_lpage">22216</span>, <span class="refDoi"> DOI: 10.1073/pnas.0910123106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1073%2Fpnas.0910123106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20007782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmtlWqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=22211-22216&author=Z.+Takacsauthor=M.+Toupsauthor=A.+Kolleweauthor=E.+Johnsonauthor=L.+G.+Cuelloauthor=G.+Driessensauthor=M.+Biancalanaauthor=A.+Koideauthor=C.+G.+Ponteauthor=E.+Perozoauthor=T.+F.+Gajewskiauthor=G.+Suarez-Kurtzauthor=S.+Koideauthor=S.+A.+N.+Goldstein&title=A+Designer+Ligand+Specific+for+Kv1.3+Channels+from+a+Scorpion+Neurotoxin-Based+Library&doi=10.1073%2Fpnas.0910123106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A designer ligand specific for Kv1.3 channels from a scorpion neurotoxin-based library</span></div><div class="casAuthors">Takacs, Zoltan; Toups, Megan; Kollewe, Astrid; Johnson, Erik; Cuello, Luis G.; Driessens, Gregory; Biancalana, Matthew; Koide, Akiko; Ponte, Cristiano G.; Perozo, Eduardo; Gajewski, Thomas F.; Suarez-Kurtz, Guilherme; Koide, Shohei; Goldstein, Steve A. N.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">22211-22216, S22211/1-S22211/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Venomous animals immobilize prey using protein toxins that act on ion channels and other targets of biol. importance.  Broad use of toxins for biomedical research, diagnosis, and therapy has been limited by inadequate target discrimination, for example, among ion channel subtypes.  Here, a synthetic toxin is produced by a new strategy to be specific for human Kv1.3 channels, crit. regulators of immune T cells.  A phage display library of 11,200 de novo proteins is designed using the α-KTx scaffold of 31 scorpion toxin sequences known or predicted to bind to potassium channels.  Mokatoxin-1 (moka1) is isolated by affinity selection on purified target.  Moka1 blocks Kv1.3 at nanomolar levels that do not inhibit Kv1.1, Kv1.2, or KCa1.1.  As a result, moka1 suppresses CD3/28-induced cytokine secretion by T cells without cross-reactive gastrointestinal hyperactivity.  Mokatoxin-1 (moka1) is isolated by affinity selection on purified target.  The 3D structure of moka1 rationalizes its specificity and validates the engineering approach, revealing a unique interaction surface supported on an α-KTx scaffold.  This scaffold-based/target-biased strategy overcomes many obstacles to prodn. of selective toxins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosX_RmtjhXh7Vg90H21EOLACvtfcHk0li1MESGz4is_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmtlWqsw%253D%253D&md5=da8f9ebf9282d762e30a912cd74592f2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0910123106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0910123106%26sid%3Dliteratum%253Aachs%26aulast%3DTakacs%26aufirst%3DZ.%26aulast%3DToups%26aufirst%3DM.%26aulast%3DKollewe%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DCuello%26aufirst%3DL.%2BG.%26aulast%3DDriessens%26aufirst%3DG.%26aulast%3DBiancalana%26aufirst%3DM.%26aulast%3DKoide%26aufirst%3DA.%26aulast%3DPonte%26aufirst%3DC.%2BG.%26aulast%3DPerozo%26aufirst%3DE.%26aulast%3DGajewski%26aufirst%3DT.%2BF.%26aulast%3DSuarez-Kurtz%26aufirst%3DG.%26aulast%3DKoide%26aufirst%3DS.%26aulast%3DGoldstein%26aufirst%3DS.%2BA.%2BN.%26atitle%3DA%2520Designer%2520Ligand%2520Specific%2520for%2520Kv1.3%2520Channels%2520from%2520a%2520Scorpion%2520Neurotoxin-Based%2520Library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D22211%26epage%3D22216%26doi%3D10.1073%2Fpnas.0910123106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmenkov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grishin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassilevski, A. A.</span></span> <span> </span><span class="NLM_article-title">Diversity of Potassium Channel Ligands: Focus on Scorpion Toxins</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">1764</span>– <span class="NLM_lpage">1799</span>, <span class="refDoi"> DOI: 10.1134/s0006297915130118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1134%2Fs0006297915130118" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2016&pages=1764-1799&author=A.+I.+Kuzmenkovauthor=E.+V.+Grishinauthor=A.+A.+Vassilevski&title=Diversity+of+Potassium+Channel+Ligands%3A+Focus+on+Scorpion+Toxins&doi=10.1134%2Fs0006297915130118"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1134%2Fs0006297915130118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1134%252Fs0006297915130118%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmenkov%26aufirst%3DA.%2BI.%26aulast%3DGrishin%26aufirst%3DE.%2BV.%26aulast%3DVassilevski%26aufirst%3DA.%2BA.%26atitle%3DDiversity%2520of%2520Potassium%2520Channel%2520Ligands%253A%2520Focus%2520on%2520Scorpion%2520Toxins%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D80%26spage%3D1764%26epage%3D1799%26doi%3D10.1134%2Fs0006297915130118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Oliveira Carvalho, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, V. M.</span></span> <span> </span><span class="NLM_article-title">Plant Defensins--Prospects for the Biological Functions and Biotechnological Properties</span>. <i>Peptides</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1020</span>, <span class="refDoi"> DOI: 10.1016/j.peptides.2009.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.peptides.2009.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19428780" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=1007-1020&author=A.+de%0AOliveira+Carvalhoauthor=V.+M.+Gomes&title=Plant+Defensins%2D%2DProspects+for+the+Biological+Functions+and+Biotechnological+Properties&doi=10.1016%2Fj.peptides.2009.01.018"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.peptides.2009.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.peptides.2009.01.018%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BOliveira%2BCarvalho%26aufirst%3DA.%26aulast%3DGomes%26aufirst%3DV.%2BM.%26atitle%3DPlant%2520Defensins--Prospects%2520for%2520the%2520Biological%2520Functions%2520and%2520Biotechnological%2520Properties%26jtitle%3DPeptides%26date%3D2009%26volume%3D30%26spage%3D1007%26epage%3D1020%26doi%3D10.1016%2Fj.peptides.2009.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koehbach, J.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships of Insect Defensins</span>. <i>Front. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">45</span>, <span class="refDoi"> DOI: 10.3389/fchem.2017.00045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3389%2Ffchem.2017.00045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28748179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlWksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2017&pages=45&author=J.+Koehbach&title=Structure-Activity+Relationships+of+Insect+Defensins&doi=10.3389%2Ffchem.2017.00045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of insect defensins</span></div><div class="casAuthors">Koehbach, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Chemistry (Lausanne, Switzerland)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45/1-45/10</span>CODEN:
                <span class="NLM_cas:coden">FCLSAA</span>;
        ISSN:<span class="NLM_cas:issn">2296-2646</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Insects make up the largest and most diverse group of organisms on earth with several million species to exist in total.  Considering the sheer no. of insect species and the vast variety of ways they interact with their environment through chem., it is clear that they have significant potential as a source of new lead mols.  They have adapted to a range of ecol. habitats and exhibit a symbiotic lifestyle with various microbes such as bacteria and fungi.  Accordingly, numerous antimicrobial compds. have been identified including for example defensin peptides.  Insect defensins were found to have broad-spectrum activity against various gram-pos./neg. bacteria as well as fungi.  They exhibit a unique structural topol. involving the complex arrangement of three disulfide bonds as well as an alpha helix and beta sheets, which is known as cysteine-stabilized αβ motif.  Their stability and amenability to peptide engineering make them promising candidates for the development of novel antibiotics lead mols.  This review highlights the current knowledge regarding the structure-activity relationships of insect defensin peptides and provides basis for future studies focusing on the rational design of novel cysteine-rich antimicrobial peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBzN1x06PTOrVg90H21EOLACvtfcHk0lgOtgWUpQtO2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlWksbg%253D&md5=a1e4ffffc6013c64cdcf93db611621bf</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.3389%2Ffchem.2017.00045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffchem.2017.00045%26sid%3Dliteratum%253Aachs%26aulast%3DKoehbach%26aufirst%3DJ.%26atitle%3DStructure-Activity%2520Relationships%2520of%2520Insect%2520Defensins%26jtitle%3DFront.%2520Chem.%26date%3D2017%26volume%3D5%26spage%3D45%26doi%3D10.3389%2Ffchem.2017.00045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornet, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonmatin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hetru, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vovelle, F.</span></span> <span> </span><span class="NLM_article-title">Refined Three-Dimensional Solution Structure of Insect Defensin A</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">448</span>, <span class="refDoi"> DOI: 10.1016/S0969-2126(01)00177-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0969-2126%2801%2900177-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=7663941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2MXlvFyhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1995&pages=435-448&author=B.+Cornetauthor=J.+M.+Bonmatinauthor=C.+Hetruauthor=J.+A.+Hoffmannauthor=M.+Ptakauthor=F.+Vovelle&title=Refined+Three-Dimensional+Solution+Structure+of+Insect+Defensin+A&doi=10.1016%2FS0969-2126%2801%2900177-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Refined three-dimensional solution structure of insect defensin A</span></div><div class="casAuthors">Cornet, Bruno; Bonmatin, Jena-Marc; Hetru, Charles; Hoffmann, Jules A.; Ptak, Marius; Volvelle, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">435-48</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">Insect defensin A is a basic 4 kDa protein secreted by Phormia terranovae larvae in response to bacteria challenges or injuries.  Previous biol. test suggest that the bacterial cytoplasmic membrane is the target of defensin A.  The structural study of this protein in the first step towards establishing a structure-activity relation and forms the basis for understanding its antibiotic activity at the mol. level.  The authors describe a refined model of the three-dimensional structure of defensin A derived from an extensive anal. of 786 inter-proton nuclear Overhauser effects.  The backbone fold involves an N-terminal loop and an α-helical fragment followed by an antiparallel β-structure.  The helix and β-structure are connected by two of the three disulfide bridges present in defensin A, forming a so-called 'cysteine-stabilized αβ' (CSαβ) motif.  The N-terminal loop, which forms the core of the defensin A structure, appears to be a common organization for several families of small proteins with toxic properties.  The distribution of amino acid side chain in the proteins structure creates several hydrophobic or hydrophilic patches.  This lead the authors' to propose that the initial step in the action of pos. charged defensin A mols. with cytoplasmic membranes may involve interaction with acidic phospholipids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPmfGsl4P16LVg90H21EOLACvtfcHk0lgOtgWUpQtO2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXlvFyhsLo%253D&md5=b79764c3487806f2785f1ee3f51da4d3</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2801%2900177-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252801%252900177-0%26sid%3Dliteratum%253Aachs%26aulast%3DCornet%26aufirst%3DB.%26aulast%3DBonmatin%26aufirst%3DJ.%2BM.%26aulast%3DHetru%26aufirst%3DC.%26aulast%3DHoffmann%26aufirst%3DJ.%2BA.%26aulast%3DPtak%26aufirst%3DM.%26aulast%3DVovelle%26aufirst%3DF.%26atitle%3DRefined%2520Three-Dimensional%2520Solution%2520Structure%2520of%2520Insect%2520Defensin%2520A%26jtitle%3DStructure%26date%3D1995%26volume%3D3%26spage%3D435%26epage%3D448%26doi%3D10.1016%2FS0969-2126%2801%2900177-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Synergy and Antagonism of Promiscuous Inhibition in Multiple-Compound Mixtures</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2151</span>– <span class="NLM_lpage">2154</span>, <span class="refDoi"> DOI: 10.1021/jm060029z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060029z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVygtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=2151-2154&author=B.+Y.+Fengauthor=B.+K.+Shoichet&title=Synergy+and+Antagonism+of+Promiscuous+Inhibition+in+Multiple-Compound+Mixtures&doi=10.1021%2Fjm060029z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Synergy and Antagonism of Promiscuous Inhibition in Multiple-Compound Mixtures</span></div><div class="casAuthors">Feng, Brian Y.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2151-2154</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening in mixts. is a common approach for increasing the efficiency of high-throughput screening.  Here we investigate how the "compd. load" of mixts. influences promiscuous aggregate-based inhibition.  We screened 764 mols. individually and in mixts. of 10 at 5 μM each, comparing the obsd. inhibition of the mixts. to that predicted from single-compd. results.  Synergistic effects on aggregation predominated, although antagonism was also obsd.  These results suggest that screening mixts. can increase aggregation-based inhibition in a nonadditive manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5WCPXJ_ed2bVg90H21EOLACvtfcHk0lgOtgWUpQtO2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XitVygtLs%253D&md5=b62e1b87ca617b154f6c64b46476fa08</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fjm060029z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060029z%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DSynergy%2520and%2520Antagonism%2520of%2520Promiscuous%2520Inhibition%2520in%2520Multiple-Compound%2520Mixtures%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D2151%26epage%3D2154%26doi%3D10.1021%2Fjm060029z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. W.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery and Natural Products: End of an Era or an Endless Frontier?</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>325</i></span>,  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">165</span>, <span class="refDoi"> DOI: 10.1126/science.1168243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1126%2Fscience.1168243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=19589993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1MXovVGmsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=161-165&author=J.+W.-H.+Liauthor=J.+C.+Vederas&title=Drug+Discovery+and+Natural+Products%3A+End+of+an+Era+or+an+Endless+Frontier%3F&doi=10.1126%2Fscience.1168243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery and natural products end of an era or an endless frontier?</span></div><div class="casAuthors">Li, Jess W.-H.; Vederas, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5937</span>),
    <span class="NLM_cas:pages">161-165</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90S_IHvC-8bVg90H21EOLACvtfcHk0lg59_wRNJnb8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXovVGmsrc%253D&md5=92cbb0ae2947501b031a99b774d2adfb</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1126%2Fscience.1168243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1168243%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%2BW.-H.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26atitle%3DDrug%2520Discovery%2520and%2520Natural%2520Products%253A%2520End%2520of%2520an%2520Era%2520or%2520an%2520Endless%2520Frontier%253F%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D161%26epage%3D165%26doi%3D10.1126%2Fscience.1168243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bender, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisman, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikhailov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, W. A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacoby, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span> <span> </span><span class="NLM_article-title">Which Aspects of HTS Are Empirically Correlated with Downstream Success?</span>. <i>Curr. Opin. Drug Discovery Dev.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">337</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=18428086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2008&pages=327-337&author=A.+Benderauthor=D.+Bojanicauthor=J.+W.+Daviesauthor=T.+J.+Crismanauthor=D.+Mikhailovauthor=J.+Scheiberauthor=J.+L.+Jenkinsauthor=Z.+Dengauthor=W.+A.+G.+Hillauthor=M.+Popovauthor=E.+Jacobyauthor=M.+Glick&title=Which+Aspects+of+HTS+Are+Empirically+Correlated+with+Downstream+Success%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Which aspects of HTS are empirically correlated with downstream success?</span></div><div class="casAuthors">Bender, Andreas; Bojanic, Dejan; Davies, John W.; Crisman, Thomas J.; Mikhailov, Dmitri; Scheiber, Josef; Jenkins, Jeremy L.; Deng, Zhan; Hill, W. Adam G.; Popov, Maxim; Jacoby, Edgar; Glick, Meir</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-337</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) is a well-established hit-finding approach used in the pharmaceutical industry.  In this article, recent experience at Novartis with respect to factors influencing the success of HTS campaigns is discussed.  An inherent measure of HTS quality could be defined by the assay Z and Z' factors, the no. of hits and their biol. potencies; however, such measures of quality do not always correlate with the advancement of hits to the later stages of drug discovery.  Also, for many target classes, such as kinases, it is easy to identify hits, but, as a result of selectivity, intellectual property and other issues, the projects do not result in lead declarations.  In this article, HTS success is defined as the fraction of HTS campaigns that advance into the later stages of drug discovery, and the major influencing factors are examd.  Interestingly, screening compds. in individual wells or in mixts. did not have a major impact on the HTS success and, equally interesting, there was no difference in the progression rates of biochem. and cell-based assays.  Particular target types, assay technologies, structure-activity relationships and powder availability had a much greater impact on success as defined above.  In addn., significant mutual dependencies can be obsd. - while one assay format works well with one target type, this situation might be completely reversed for a combination of the same readout technol. with a different target type.  The results and opinions presented here should be regarded as groundwork, and a plethora of factors that influence the fate of a project, such as biophys. measurements, chem. attractiveness of the hits, strategic reasons and safety pharmacol., are not covered here.  Nonetheless, it is hoped that this information will be used industry-wide to improve success rates in terms of hits progressing into exploratory chem. and beyond.  The support that can be obtained from new in silico approaches to phase transitions are also described, along with the gaps they are designed to fill.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqddDbeDZXfNbVg90H21EOLACvtfcHk0lg59_wRNJnb8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1Wkur4%253D&md5=c65c4142d449fe752b37c397cd2a941e</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBender%26aufirst%3DA.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BW.%26aulast%3DCrisman%26aufirst%3DT.%2BJ.%26aulast%3DMikhailov%26aufirst%3DD.%26aulast%3DScheiber%26aufirst%3DJ.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DHill%26aufirst%3DW.%2BA.%2BG.%26aulast%3DPopov%26aufirst%3DM.%26aulast%3DJacoby%26aufirst%3DE.%26aulast%3DGlick%26aufirst%3DM.%26atitle%3DWhich%2520Aspects%2520of%2520HTS%2520Are%2520Empirically%2520Correlated%2520with%2520Downstream%2520Success%253F%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2008%26volume%3D11%26spage%3D327%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">18428086</p></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span> <span> </span><span class="NLM_article-title">Current Trends in Lead Discovery: Are We Looking for the Appropriate Properties?</span>. <i>Mol. Diversity</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1023/a:1021368007777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1023%2FA%3A1021368007777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=12549672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFOjurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2002&pages=199-208&author=T.+I.+Oprea&title=Current+Trends+in+Lead+Discovery%3A+Are+We+Looking+for+the+Appropriate+Properties%3F&doi=10.1023%2Fa%3A1021368007777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Current trends in lead discovery: Are we looking for the appropriate properties?</span></div><div class="casAuthors">Oprea, Tudor I.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Diversity</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">199-208</span>CODEN:
                <span class="NLM_cas:coden">MODIF4</span>;
        ISSN:<span class="NLM_cas:issn">1381-1991</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  The new drug discovery paradigm is based on high-throughput technologies, both with respect to synthesis and screening.  The progression HTS hits lead series candidate drug marketed drug appears to indicate that the probability of reaching launched status is one in a million.  This has shifted the focus from good quality candidate drugs to good quality leads.  We examd. the current trends in lead discovery by comparing MW (mol. wt.), LogP (octanol/water partition coeff., estd. by Kowwin [17]) and LogSw (intrinsic water soly., estd. by Wskowwin [18]) for the following categories: 62 leads and 75 drugs [11]; compds. in the development phase (I, II, III and launched), as indexed in MDDR; and compds. indexed in medicinal chem. journals [ref. 20], categorized according to their biol. activity.  Comparing the distribution of the above properties, the 62 lead structures show the lowest median with respect to MW (smaller) and LogP (less hydrophobic), and the highest median with respect to LogSw (more sol.).  By contrast, over 50% of the medicinal chem. compds. with activities above 1 nanomolar have MW > 425, LogP > 4.25 and LogSw < -4.75, indicating that the reported active compds. are larger, more hydrophobic and less sol. when compared to time-tested quality leads.  In the MDDR set, a progressive constraint to reduce MW and LogP, and to increase LogSw, can be obsd. when examg. trends in the developmental sequence: phase I, II, III and launched drugs.  These trends indicate that other properties besides binding affinity, e.g., soly. and hydrophobicity, need to be considered when choosing the appropriate leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHbdVqIWk9UrVg90H21EOLACvtfcHk0lg59_wRNJnb8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFOjurg%253D&md5=1feef5374888c429af331a2620cf7e73</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1023%2FA%3A1021368007777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1021368007777%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DCurrent%2520Trends%2520in%2520Lead%2520Discovery%253A%2520Are%2520We%2520Looking%2520for%2520the%2520Appropriate%2520Properties%253F%26jtitle%3DMol.%2520Diversity%26date%3D2002%26volume%3D5%26spage%3D199%26epage%3D208%26doi%3D10.1023%2Fa%3A1021368007777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andermann, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowell, B. B.</span></span> <span> </span><span class="NLM_article-title">Toward a Wiring Diagram Understanding of Appetite Control</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">757</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2017.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.neuron.2017.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=28817798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlClur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2017&pages=757-778&author=M.+L.+Andermannauthor=B.+B.+Lowell&title=Toward+a+Wiring+Diagram+Understanding+of+Appetite+Control&doi=10.1016%2Fj.neuron.2017.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Toward a Wiring Diagram Understanding of Appetite Control</span></div><div class="casAuthors">Andermann, Mark L.; Lowell, Bradford B.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">757-778</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Prior mouse genetic research has set the stage for a deep understanding of appetite regulation.  This goal is now being realized through the use of recent technol. advances, such as the ability to map connectivity between neurons, manipulate neural activity in real time, and measure neural activity during behavior.  Indeed, major progress has been made with regard to meal-related gut control of appetite, arcuate nucleus-based hypothalamic circuits linking energy state to the motivational drive, hunger, and, finally, limbic and cognitive processes that bring about hunger-mediated increases in reward value and perception of food.  Unexpected findings are also being made; for example, the rapid regulation of homeostatic neurons by cues that predict future food consumption.  The aim of this review is to cover the major underpinnings of appetite regulation, describe recent advances resulting from new technologies, and synthesize these findings into an updated view of appetite regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDBDn95oahuLVg90H21EOLACvtfcHk0lg59_wRNJnb8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlClur3M&md5=5c00631d3124358c5328b079ece02bc8</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2017.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2017.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DAndermann%26aufirst%3DM.%2BL.%26aulast%3DLowell%26aufirst%3DB.%2BB.%26atitle%3DToward%2520a%2520Wiring%2520Diagram%2520Understanding%2520of%2520Appetite%2520Control%26jtitle%3DNeuron%26date%3D2017%26volume%3D95%26spage%3D757%26epage%3D778%26doi%3D10.1016%2Fj.neuron.2017.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishigaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohnishi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span> <span> </span><span class="NLM_article-title">Supranuclear Melanin Caps Reduce Ultraviolet Induced DNA Photoproducts in Human Epidermis</span>. <i>J. Invest. Dermatol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">806</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1046/j.1523-1747.1998.00178.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1046%2Fj.1523-1747.1998.00178.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9579550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1cXjtVGnt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1998&pages=806-810&author=N.+Kobayashiauthor=A.+Nakagawaauthor=T.+Muramatsuauthor=Y.+Yamashinaauthor=T.+Shiraiauthor=M.+W.+Hashimotoauthor=Y.+Ishigakiauthor=T.+Ohnishiauthor=T.+Mori&title=Supranuclear+Melanin+Caps+Reduce+Ultraviolet+Induced+DNA+Photoproducts+in+Human+Epidermis&doi=10.1046%2Fj.1523-1747.1998.00178.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Supranuclear melanin caps reduce ultraviolet-induced DNA photoproducts in human epidermis</span></div><div class="casAuthors">Kobayashi, Nobuhiko; Nakagawa, Akemi; Muramatsu, Tsutomu; Yamashina, Yukio; Shirai, Toshihiko; Hashimoto, Mitsumasa W.; Ishigaki, Yasuhito; Ohnishi, Takeo; Mori, Toshio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Investigative Dermatology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">806-810</span>CODEN:
                <span class="NLM_cas:coden">JIDEAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-202X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Science, Inc.</span>)
        </div><div class="casAbstract">Melanin can form supranuclear caps in human epidermis, suggesting that intracellular melanin reduces UV transmission to underlying cell nuclei and inhibits the formation of UV-induced DNA photoproducts.  The purpose of this study was to det. the photoprotective effect of epidermal melanin.  We irradiated normal human skin explants with UV B and detd. the formation of cyclobutane pyrimidine dimers and (6-4)photoproducts in individual epidermal cells by indirect immunofluorescence and by laser cytometry using monoclonal antibodies specific for cyclobutane dimers or for (6-4)photoproducts.  We found that epidermal cells with supranuclear melanin caps had significantly less DNA photoproducts (both types) than epidermal cells without supranuclear melanin caps.  Moreover, the protection factor against both types of photolesions correlated with melanin concn. in epidermal cells.  These results indicate that melanin reduces UV induced DNA photoproducts in human epidermis in a concn. dependent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJzwMcI7wq3LVg90H21EOLACvtfcHk0lg9GKIFHGubXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtVGnt7o%253D&md5=2100eeb68caf08c222228d357ff1e1ed</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1046%2Fj.1523-1747.1998.00178.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1523-1747.1998.00178.x%26sid%3Dliteratum%253Aachs%26aulast%3DKobayashi%26aufirst%3DN.%26aulast%3DNakagawa%26aufirst%3DA.%26aulast%3DMuramatsu%26aufirst%3DT.%26aulast%3DYamashina%26aufirst%3DY.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DM.%2BW.%26aulast%3DIshigaki%26aufirst%3DY.%26aulast%3DOhnishi%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26atitle%3DSupranuclear%2520Melanin%2520Caps%2520Reduce%2520Ultraviolet%2520Induced%2520DNA%2520Photoproducts%2520in%2520Human%2520Epidermis%26jtitle%3DJ.%2520Invest.%2520Dermatol.%26date%3D1998%26volume%3D110%26spage%3D806%26epage%3D810%26doi%3D10.1046%2Fj.1523-1747.1998.00178.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadekaro, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leachman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavanagh, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swope, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwemberger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babcock, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakamatsu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koshoffer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boissy, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manga, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturm, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Malek, Z. A.</span></span> <span> </span><span class="NLM_article-title"><i>Melanocortin 1 Receptor Genotype</i>: An Important Determinant of the Damage Response of Melanocytes to Ultraviolet Radiation</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3850</span>– <span class="NLM_lpage">3860</span>, <span class="refDoi"> DOI: 10.1096/fj.10-158485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1096%2Ffj.10-158485" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=20519635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12isLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=3850-3860&author=A.+L.+Kadekaroauthor=S.+Leachmanauthor=R.+J.+Kavanaghauthor=V.+Swopeauthor=P.+Cassidyauthor=D.+Suppauthor=M.+Sartorauthor=S.+Schwembergerauthor=G.+Babcockauthor=K.+Wakamatsuauthor=S.+Itoauthor=A.+Koshofferauthor=R.+E.+Boissyauthor=P.+Mangaauthor=R.+A.+Sturmauthor=Z.+A.+Abdel-Malek&title=Melanocortin+1+Receptor+Genotype%3A+An+Important+Determinant+of+the+Damage+Response+of+Melanocytes+to+Ultraviolet+Radiation&doi=10.1096%2Ffj.10-158485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Melanocortin 1 receptor genotype: an important determinant of the damage response of melanocytes to ultraviolet radiation</span></div><div class="casAuthors">Kadekaro, Ana Luisa; Leachman, Sancy; Kavanagh, Renny J.; Swope, Viki; Cassidy, Pamela; Supp, Dorothy; Sartor, Maureen; Schwemberger, Sandy; Babcock, George; Wakamatsu, Kazumasa; Ito, Shosuke; Koshoffer, Amy; Boissy, Raymond E.; Manga, Prashiela; Sturm, Richard A.; Abdel-Malek, Zalfa A.</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3850-3860, 10.1096/fj.10-158485</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">The melanocortin 1 receptor gene is a main determinant of human pigmentation, and a melanoma susceptibility gene, because its variants that are strongly assocd. with red hair color increase melanoma risk.  To test exptl. the assocn. between melanocortin 1 receptor genotype and melanoma susceptibility, we compared the responses of primary human melanocyte cultures naturally expressing different melanocortin 1 receptor variants to α-melanocortin and UV radiation.  We found that expression of 2 red hair variants abolished the response to α-melanocortin and its photoprotective effects, evidenced by lack of functional coupling of the receptor, and absence of redn. in UV radiation-induced hydrogen peroxide generation or enhancement of repair of DNA photoproducts, resp.  These variants had different heterozygous effects on receptor function.  Microarray data confirmed the obsd. differences in responses of melanocytes with functional vs. non-functional receptor to α-melanocortin and UV radiation, and identified DNA repair and antioxidant genes that are modulated by α-melanocortin.  Our findings highlight the mol. mechanisms by which the melanocortin 1 receptor genotype controls genomic stability of and the mutagenic effect of UV radiation on human melanocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofwk7RqEhWCrVg90H21EOLACvtfcHk0lg9GKIFHGubXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12isLfF&md5=c52c8aef97605bc44d98152e6e95121b</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1096%2Ffj.10-158485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.10-158485%26sid%3Dliteratum%253Aachs%26aulast%3DKadekaro%26aufirst%3DA.%2BL.%26aulast%3DLeachman%26aufirst%3DS.%26aulast%3DKavanagh%26aufirst%3DR.%2BJ.%26aulast%3DSwope%26aufirst%3DV.%26aulast%3DCassidy%26aufirst%3DP.%26aulast%3DSupp%26aufirst%3DD.%26aulast%3DSartor%26aufirst%3DM.%26aulast%3DSchwemberger%26aufirst%3DS.%26aulast%3DBabcock%26aufirst%3DG.%26aulast%3DWakamatsu%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DS.%26aulast%3DKoshoffer%26aufirst%3DA.%26aulast%3DBoissy%26aufirst%3DR.%2BE.%26aulast%3DManga%26aufirst%3DP.%26aulast%3DSturm%26aufirst%3DR.%2BA.%26aulast%3DAbdel-Malek%26aufirst%3DZ.%2BA.%26atitle%3DMelanocortin%25201%2520Receptor%2520Genotype%253A%2520An%2520Important%2520Determinant%2520of%2520the%2520Damage%2520Response%2520of%2520Melanocytes%2520to%2520Ultraviolet%2520Radiation%26jtitle%3DFASEB%2520J.%26date%3D2010%26volume%3D24%26spage%3D3850%26epage%3D3860%26doi%3D10.1096%2Ffj.10-158485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Creemers, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Rouzic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meulemans, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardlaw, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luckman, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennand, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, A.</span></span> <span> </span><span class="NLM_article-title">Agouti-Related Protein Is Posttranslationally Cleaved by Proprotein Convertase 1 to Generate Agouti-Related Protein (AGRP)<sub>83-132</sub>: Interaction between AGRP<sub>83-132</sub> and Melanocortin Receptors Cannot Be Influenced by Syndecan-3</span>. <i>Endocrinology</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">1621</span>– <span class="NLM_lpage">1631</span>, <span class="refDoi"> DOI: 10.1210/en.2005-1373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1210%2Fen.2005-1373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=16384863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt1Ohsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2006&pages=1621-1631&author=J.+W.+M.+Creemersauthor=L.+E.+Pritchardauthor=A.+Gyteauthor=P.+Le+Rouzicauthor=S.+Meulemansauthor=S.+L.+Wardlawauthor=X.+Zhuauthor=D.+F.+Steinerauthor=N.+Daviesauthor=D.+Armstrongauthor=C.+B.+Lawrenceauthor=S.+M.+Luckmanauthor=C.+A.+Schmitzauthor=R.+A.+Daviesauthor=J.+C.+Brennandauthor=A.+White&title=Agouti-Related+Protein+Is+Posttranslationally+Cleaved+by+Proprotein+Convertase+1+to+Generate+Agouti-Related+Protein+%28AGRP%2983-132%3A+Interaction+between+AGRP83-132+and+Melanocortin+Receptors+Cannot+Be+Influenced+by+Syndecan-3&doi=10.1210%2Fen.2005-1373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Agouti-related protein is posttranslationally cleaved by proprotein convertase 1 to generate agouti-related protein (AGRP)83-132: interaction between AGRP83-132 and melanocortin receptors cannot be influenced by syndecan-3</span></div><div class="casAuthors">Creemers, John W. M.; Pritchard, Lynn E.; Gyte, Amy; Le Rouzic, Philippe; Meulemans, Sandra; Wardlaw, Sharon L.; Zhu, Xiaorong; Steiner, Donald F.; Davies, Nicola; Armstrong, Duncan; Lawrence, Catherine B.; Luckman, Simon M.; Schmitz, Catherine A.; Davies, Rick A.; Brennand, John C.; White, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Endocrinology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1621-1631</span>CODEN:
                <span class="NLM_cas:coden">ENDOAO</span>;
        ISSN:<span class="NLM_cas:issn">0013-7227</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Agouti-related protein (AGRP) plays a key role in energy homeostasis.  The C-terminal domain of AGRP acts as an endogenous antagonist of the melanocortin-4 receptor (MC4-R).  It has been suggested that the N-terminal domain of AGRP binds to syndecan-3, thereby modulating the effects of C-terminal AGRP at the MC4-R.  This model assumes that AGRP is secreted as a full-length peptide.  In this study the authors found that AGRP is processed intercellularly after Arg79-Glu80-Pro81-Arg82.  The processing site suggests cleavage by proprotein convertases (PCs).  RNA interference and overexpression expts. showed that PC1/3 is primarily responsible for cleavage in vitro, although both PC2 and PC5/6A can also process AGRP.  Dual in situ hybridization demonstrated that PC1/3 is expressed in AGRP neurons in the rat hypothalamus.  Moreover, hypothalamic exts. from PC1-null mice contained 3.3-fold more unprocessed full-length AGRP, compared with wild-type mice, based on combined HPLC and RIA anal., demonstrating that PC1/3 plays a role in AGRP cleavage in vivo.  The authors also found that AGRP83-132 is more potent an antagonist than full-length AGRP, based on cAMP reporter assays, suggesting that posttranslational cleavage is required to potentiate the effect of AGRP at the MC4-R.  Because AGRP is cleaved into distinct N-terminal and C-terminal peptides, the authors tested whether N-terminal peptides modulate food intake.  However, intracerebroventricular injection of rat AGRP25-47 and AGRP50-80 had no effect on body wt., food intake, or core body temp.  Because AGRP is cleaved before secretion, syndecan-3 must influence food intake independently of the MC4-R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyGX8-BXRj87Vg90H21EOLACvtfcHk0lg9GKIFHGubXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt1Ohsbk%253D&md5=01d541e583842f4eb81d65685ab6639f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1210%2Fen.2005-1373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fen.2005-1373%26sid%3Dliteratum%253Aachs%26aulast%3DCreemers%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DPritchard%26aufirst%3DL.%2BE.%26aulast%3DGyte%26aufirst%3DA.%26aulast%3DLe%2BRouzic%26aufirst%3DP.%26aulast%3DMeulemans%26aufirst%3DS.%26aulast%3DWardlaw%26aufirst%3DS.%2BL.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DSteiner%26aufirst%3DD.%2BF.%26aulast%3DDavies%26aufirst%3DN.%26aulast%3DArmstrong%26aufirst%3DD.%26aulast%3DLawrence%26aufirst%3DC.%2BB.%26aulast%3DLuckman%26aufirst%3DS.%2BM.%26aulast%3DSchmitz%26aufirst%3DC.%2BA.%26aulast%3DDavies%26aufirst%3DR.%2BA.%26aulast%3DBrennand%26aufirst%3DJ.%2BC.%26aulast%3DWhite%26aufirst%3DA.%26atitle%3DAgouti-Related%2520Protein%2520Is%2520Posttranslationally%2520Cleaved%2520by%2520Proprotein%2520Convertase%25201%2520to%2520Generate%2520Agouti-Related%2520Protein%2520%2528AGRP%252983-132%253A%2520Interaction%2520between%2520AGRP83-132%2520and%2520Melanocortin%2520Receptors%2520Cannot%2520Be%2520Influenced%2520by%2520Syndecan-3%26jtitle%3DEndocrinology%26date%3D2006%26volume%3D147%26spage%3D1621%26epage%3D1631%26doi%3D10.1210%2Fen.2005-1373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tota, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacNeil, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Ploeg, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, T. M.</span></span> <span> </span><span class="NLM_article-title">Molecular Interaction of Agouti Protein and Agouti-Related Protein with Human Melanocortin Receptors</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">904</span>, <span class="refDoi"> DOI: 10.1021/bi9815602</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9815602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1cXotFajtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=897-904&author=M.+R.+Totaauthor=T.+S.+Smithauthor=C.+Maoauthor=T.+MacNeilauthor=R.+T.+Mosleyauthor=L.+H.+Van+der+Ploegauthor=T.+M.+Fong&title=Molecular+Interaction+of+Agouti+Protein+and+Agouti-Related+Protein+with+Human+Melanocortin+Receptors&doi=10.1021%2Fbi9815602"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Interaction of Agouti Protein and Agouti-Related Protein with Human Melanocortin Receptors</span></div><div class="casAuthors">Tota, M. R.; Smith, T. S.; Mao, C.; MacNeil, T.; Mosley, R. T.; Van der Ploeg, L. H. T.; Fong, T. M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">897-904</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Agouti protein and the Agouti-related protein (AGRP) are antagonists of the melanocortin-3 receptor and melanocortin-4 receptor.  Both proteins contain 10 cysteines in the C-terminal domain arranged in five disulfide bonds.  One possible arrangement of the disulfide bonds predicts an octapeptide loop, and the chem. properties of four residues within this loop (residues 111-114 in human AGRP) bear striking resemblance to those of several melanocortin peptides, including α-MSH, MT-II, and SHU-9119.  We showed that cyclic synthetic octapeptides based on the sequence of this loop from Agouti protein or human AGRP are functional antagonists of the human melanocortin-4 receptor.  All peptides had a lower affinity for the melanocortin-3 receptor than for the melanocortin-4 receptor.  Substitution of serines for cysteines resulted in linear peptides which had reduced binding affinities for both receptors.  Mutational anal. of human AGRP indicated that its C-terminal domain is functionally equiv. to the intact human AGRP.  The RFF111-113 triplet appears to be the most crit. portion of AGRP in detg. the binding affinity for both melanocortin-3 and melanocortin-4 receptors.  These data strongly suggest that the loop defined by Cys-110 and Cys-117 is crit. in detg. the antagonist activity of human AGRP.  Our data provide indirect evidence for the suggestion that the Cys-110 to Cys-117 octapeptide loop of human AGRP mimics the conformation of α-MSH, MT-II, and SHU-9119.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-2vz5sSpD8rVg90H21EOLACvtfcHk0lgpIAAOEnF5DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotFajtbg%253D&md5=58feaee4b8438ff9742c8d11af8f7c24</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fbi9815602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9815602%26sid%3Dliteratum%253Aachs%26aulast%3DTota%26aufirst%3DM.%2BR.%26aulast%3DSmith%26aufirst%3DT.%2BS.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DMacNeil%26aufirst%3DT.%26aulast%3DMosley%26aufirst%3DR.%2BT.%26aulast%3DVan%2Bder%2BPloeg%26aufirst%3DL.%2BH.%26aulast%3DFong%26aufirst%3DT.%2BM.%26atitle%3DMolecular%2520Interaction%2520of%2520Agouti%2520Protein%2520and%2520Agouti-Related%2520Protein%2520with%2520Human%2520Melanocortin%2520Receptors%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D897%26epage%3D904%26doi%3D10.1021%2Fbi9815602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nix, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsh, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span> <span> </span><span class="NLM_article-title">Molecular and Functional Analysis of Human β-Defensin 3 Action at Melanocortin Receptors</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">795</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2013.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.chembiol.2013.04.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23790489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVWgtLzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=784-795&author=M.+A.+Nixauthor=C.+B.+Kaelinauthor=T.+Taauthor=A.+Weisauthor=G.+J.+Mortonauthor=G.+S.+Barshauthor=G.+L.+Millhauser&title=Molecular+and+Functional+Analysis+of+Human+%CE%B2-Defensin+3+Action+at+Melanocortin+Receptors&doi=10.1016%2Fj.chembiol.2013.04.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and Functional Analysis of Human β-Defensin 3 Action at Melanocortin Receptors</span></div><div class="casAuthors">Nix, Matthew A.; Kaelin, Christopher B.; Ta, Tina; Weis, Allison; Morton, Gregory J.; Barsh, Gregory S.; Millhauser, Glenn L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">784-795</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The β-defensins are a class of small, cationic proteins first recognized as antimicrobial components of the innate and adaptive immune system.  More recently, one of the major β-defensins produced in skin, β-defensin 3, has been discovered to function as a melanocortin receptor ligand in vivo and in vitro, but its biophys. and pharmacol. basis of action has been enigmatic.  Here, we report functional and biochem. studies focused on human β-defensin 3 (HBD3) and melanocortin receptors 1 and 4.  Genetic and pharmacol. studies indicate that HBD3 acts as a neutral melanocortin receptor antagonist capable of blocking the action of either stimulatory agonists such as α-MSH or inhibitory inverse agonists such as Agouti signaling protein (ASIP) and Agouti-related protein (AGRP).  A comprehensive structure-function anal. demonstrates that two patches of pos. charged residues, located on opposite poles of HBD3 and spatially organized by the compact β-defensin fold, are primarily responsible for high-affinity binding to melanocortin receptors.  These findings identify a distinct mode of melanocortin receptor-ligand interactions based primarily on electrostatic complementarity, with implications for designing ligands that target melanocortin and potentially other seven transmembrane receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG6EBWHno3DLVg90H21EOLACvtfcHk0lgpIAAOEnF5DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVWgtLzK&md5=1d500a5e57f1d6918f182f6d17adebfb</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2013.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2013.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DNix%26aufirst%3DM.%2BA.%26aulast%3DKaelin%26aufirst%3DC.%2BB.%26aulast%3DTa%26aufirst%3DT.%26aulast%3DWeis%26aufirst%3DA.%26aulast%3DMorton%26aufirst%3DG.%2BJ.%26aulast%3DBarsh%26aufirst%3DG.%2BS.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26atitle%3DMolecular%2520and%2520Functional%2520Analysis%2520of%2520Human%2520%25CE%25B2-Defensin%25203%2520Action%2520at%2520Melanocortin%2520Receptors%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D784%26epage%3D795%26doi%3D10.1016%2Fj.chembiol.2013.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ericson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tala, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslach, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirain, M. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaub, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eick, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lensing, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smeester, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adank, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskell-Luevano, C.</span></span> <span> </span><span class="NLM_article-title">Human β-Defensin 1 and β-Defensin 3 (Mouse Ortholog MBD14) Function as Full Endogenous Agonists at Select Melanocortin Receptors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3738</span>– <span class="NLM_lpage">3744</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00251</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00251" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVKnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3738-3744&author=M.+D.+Ericsonauthor=A.+Singhauthor=S.+R.+Talaauthor=E.+M.+Haslachauthor=M.+L.+S.+Dirainauthor=J.+W.+Schaubauthor=V.+Floresauthor=N.+Eickauthor=C.+J.+Lensingauthor=K.+T.+Freemanauthor=B.+A.+Smeesterauthor=D.+N.+Adankauthor=S.+L.+Wilberauthor=R.+Spethauthor=C.+Haskell-Luevano&title=Human+%CE%B2-Defensin+1+and+%CE%B2-Defensin+3+%28Mouse+Ortholog+MBD14%29+Function+as+Full+Endogenous+Agonists+at+Select+Melanocortin+Receptors&doi=10.1021%2Facs.jmedchem.8b00251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Human β-defensin 1 and β-defensin 3 (mouse ortholog mBD14) function as full endogenous agonists at select melanocortin receptors</span></div><div class="casAuthors">Ericson, Mark D.; Singh, Anamika; Tala, Srinivasa R.; Haslach, Erica M.; Dirain, Marvin L. S.; Schaub, Jay W.; Flores, Viktor; Eick, Natalie; Lensing, Cody J.; Freeman, Katie T.; Smeester, Branden A.; Adank, Danielle N.; Wilber, Stacey L.; Speth, Robert; Haskell-Luevano, Carrie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3738-3744</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">β-Defensin 3 (BD3) was identified as a ligand for the melanocortin receptors (MCRs) in 2007, although the pharmacol. activity of BD3 has not been clearly elucidated.  Herein, it is demonstrated that human BD3 and mouse BD3 are full micromolar agonists at the MCRs.  Furthermore, mouse β-defensin 1 (BD1) and human BD1 are also MCR micromolar agonists.  This work identifies BD1 as an endogenous MCR ligand and clarifies the controversial role of BD3 as a micromolar agonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogj7J3XNbylrVg90H21EOLACvtfcHk0lgpIAAOEnF5DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVKnurY%253D&md5=de44e85a43566ca94cd136c8f3123f87</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00251%26sid%3Dliteratum%253Aachs%26aulast%3DEricson%26aufirst%3DM.%2BD.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DTala%26aufirst%3DS.%2BR.%26aulast%3DHaslach%26aufirst%3DE.%2BM.%26aulast%3DDirain%26aufirst%3DM.%2BL.%2BS.%26aulast%3DSchaub%26aufirst%3DJ.%2BW.%26aulast%3DFlores%26aufirst%3DV.%26aulast%3DEick%26aufirst%3DN.%26aulast%3DLensing%26aufirst%3DC.%2BJ.%26aulast%3DFreeman%26aufirst%3DK.%2BT.%26aulast%3DSmeester%26aufirst%3DB.%2BA.%26aulast%3DAdank%26aufirst%3DD.%2BN.%26aulast%3DWilber%26aufirst%3DS.%2BL.%26aulast%3DSpeth%26aufirst%3DR.%26aulast%3DHaskell-Luevano%26aufirst%3DC.%26atitle%3DHuman%2520%25CE%25B2-Defensin%25201%2520and%2520%25CE%25B2-Defensin%25203%2520%2528Mouse%2520Ortholog%2520MBD14%2529%2520Function%2520as%2520Full%2520Endogenous%2520Agonists%2520at%2520Select%2520Melanocortin%2520Receptors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3738%26epage%3D3744%26doi%3D10.1021%2Facs.jmedchem.8b00251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trulson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent, S. B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gantz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span> <span> </span><span class="NLM_article-title">NMR Structure of a Minimized Human Agouti Related Protein Prepared by Total Chemical Synthesis</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>451</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1016/s0014-5793(99)00553-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2FS0014-5793%2899%2900553-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10371151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1MXjsVSjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=1999&pages=125-131&author=K.+A.+Bolinauthor=D.+J.+Andersonauthor=J.+A.+Trulsonauthor=D.+A.+Thompsonauthor=J.+Wilkenauthor=S.+B.+H.+Kentauthor=I.+Gantzauthor=G.+L.+Millhauser&title=NMR+Structure+of+a+Minimized+Human+Agouti+Related+Protein+Prepared+by+Total+Chemical+Synthesis&doi=10.1016%2Fs0014-5793%2899%2900553-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">NMR structure of a minimized human agouti related protein prepared by total chemical synthesis</span></div><div class="casAuthors">Bolin, Kimberly A.; Anderson, D. Joe; Trulson, Julie A.; Thompson, Darren A.; Wilken, Jill; Kent, Stephen B. H.; Gantz, Ira; Millhauser, Glenn L.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">451</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-131</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The structure of the chem. synthesized C-terminal region of the human agouti related protein (AGRP) was detd. by 2D 1H NMR.  Referred to as minimized agouti related protein, MARP is a 46 residue polypeptide contg. 10 Cys residues involved in five disulfide bonds that retains the biol. activity of full length AGRP.  AGRP is a mammalian signaling mol., involved in wt. homeostasis, that causes adult onset obesity when overexpressed in mice.  AGRP was originally identified by homol. to the agouti protein, another potent signaling mol. involved in obesity disorders in mice.  While AGRP's exact mechanism of action is unknown, it has been identified as a competitive antagonist of melanocortin receptors 3 and 4 (MC3r, MC4r), and MC4r in particular is implicated in the hypothalamic control of feeding behavior.  Full length agouti and AGRP are only 25% homologous, however, their active C-terminal regions are ∼40% homologous, with nine out of the 10 Cys residues spatially conserved.  Until now, 3D structures have not been available for either agouti, AGRP or their C-terminal regions.  The NMR structure of MARP reported here can be characterized as three major loops, with four of the five disulfide bridges at the base of the structure.  Though its fold is well defined, no canonical secondary structure is identified.  While previously reported structural models of the C-terminal region of AGRP were attempted based on Cys homol. between AGRP and certain toxin proteins, we find that Cys spacing is not sufficient to correctly det. the 3D fold of the mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN7u4VlD35BbVg90H21EOLACvtfcHk0liJJ6RtX_WtCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjsVSjtLg%253D&md5=7c74e63ec620183f63eb0069a9ee5b42</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2899%2900553-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252899%252900553-0%26sid%3Dliteratum%253Aachs%26aulast%3DBolin%26aufirst%3DK.%2BA.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DTrulson%26aufirst%3DJ.%2BA.%26aulast%3DThompson%26aufirst%3DD.%2BA.%26aulast%3DWilken%26aufirst%3DJ.%26aulast%3DKent%26aufirst%3DS.%2BB.%2BH.%26aulast%3DGantz%26aufirst%3DI.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26atitle%3DNMR%2520Structure%2520of%2520a%2520Minimized%2520Human%2520Agouti%2520Related%2520Protein%2520Prepared%2520by%2520Total%2520Chemical%2520Synthesis%26jtitle%3DFEBS%2520Lett.%26date%3D1999%26volume%3D451%26spage%3D125%26epage%3D131%26doi%3D10.1016%2Fs0014-5793%2899%2900553-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schibli, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aseyev, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starner, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiencek, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCray, P. B.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tack, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, H. J.</span></span> <span> </span><span class="NLM_article-title">The Solution Structures of the Human Beta-Defensins Lead to a Better Understanding of the Potent Bactericidal Activity of HBD3 against <i>Staphylococcus Aureus</i></span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">8279</span>– <span class="NLM_lpage">8289</span>, <span class="refDoi"> DOI: 10.1074/jbc.M108830200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.M108830200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=11741980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFymt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=8279-8289&author=D.+J.+Schibliauthor=H.+N.+Hunterauthor=V.+Aseyevauthor=T.+D.+Starnerauthor=J.+M.+Wiencekauthor=P.+B.+McCrayauthor=B.+F.+Tackauthor=H.+J.+Vogel&title=The+Solution+Structures+of+the+Human+Beta-Defensins+Lead+to+a+Better+Understanding+of+the+Potent+Bactericidal+Activity+of+HBD3+against+Staphylococcus+Aureus&doi=10.1074%2Fjbc.M108830200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The solution structures of the human β-defensins lead to a better understanding of the potent bactericidal activity of HBD3 against Staphylococcus aureus</span></div><div class="casAuthors">Schibli, David J.; Hunter, Howard N.; Aseyev, Vladimir; Starner, Timothy D.; Wiencek, John M.; McCray, Paul B., Jr.; Tack, Brian F.; Vogel, Hans J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">8279-8289</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The three human β-defensins, HBD1-3, are 33-47-residue, cationic antimicrobial proteins expressed by epithelial cells.  All three proteins have broad-spectrum antimicrobial activity, with HBD3 consistently being the most potent.  Addnl., HBD3 has significant bactericidal activity against Gram-pos. Staphylococcus aureus at physiol. salt concns.  We have compared the multimeric state of the three β-defensins using NMR diffusion spectroscopy, dynamic and static light scattering, and anal. of the migration of the three β-defensins on a native gel.  All three techniques are in agreement, suggesting that HBD-3 is a dimer, while HBD-1 and HBD-2 are monomeric.  Subsequently, the NMR soln. structures of HBD1 and HBD3 were detd. using std. homonuclear techniques and compared with the previously detd. soln. structure of HBD2.  Both HBD1 and HBD3 form well defined structures with backbone root mean square deviations of 0.451 and 0.616 Å, resp.  The tertiary structures of all three β-defensins are similar, with a short helical segment preceding a three-stranded antiparallel β-sheet.  The surface charge d. of each of the defensins is markedly different, with the surface of HBD3 significantly more basic.  Anal. of the NMR data and structures led us to suggest that HBD3 forms a sym. dimer through strand β2 of the β-sheet.  The increased anti-Staphylococcal activity of HBD3 may be explained by the capacity of the protein to form dimers in soln. at low concns., an amphipathic dimer structure, and the increased pos. surface charge compared with HBD1 and HBD2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqItwZ3MBW107Vg90H21EOLACvtfcHk0liJJ6RtX_WtCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFymt78%253D&md5=e7f4fb1e06497eb88c69140e6511eec0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M108830200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M108830200%26sid%3Dliteratum%253Aachs%26aulast%3DSchibli%26aufirst%3DD.%2BJ.%26aulast%3DHunter%26aufirst%3DH.%2BN.%26aulast%3DAseyev%26aufirst%3DV.%26aulast%3DStarner%26aufirst%3DT.%2BD.%26aulast%3DWiencek%26aufirst%3DJ.%2BM.%26aulast%3DMcCray%26aufirst%3DP.%2BB.%26aulast%3DTack%26aufirst%3DB.%2BF.%26aulast%3DVogel%26aufirst%3DH.%2BJ.%26atitle%3DThe%2520Solution%2520Structures%2520of%2520the%2520Human%2520Beta-Defensins%2520Lead%2520to%2520a%2520Better%2520Understanding%2520of%2520the%2520Potent%2520Bactericidal%2520Activity%2520of%2520HBD3%2520against%2520Staphylococcus%2520Aureus%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D8279%26epage%3D8289%26doi%3D10.1074%2Fjbc.M108830200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkes, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Obeidi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staples, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Vaux, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dym, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castrucci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintz, M. F.</span></span> <span> </span><span class="NLM_article-title">.alpha.-Melanotropin: The Minimal Active Sequence in the Frog Skin Bioassay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2130</span>, <span class="refDoi"> DOI: 10.1021/jm00394a033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00394a033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaL2sXlslKntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1987&pages=2126-2130&author=V.+J.+Hrubyauthor=B.+C.+Wilkesauthor=M.+E.+Hadleyauthor=F.+Al-Obeidiauthor=T.+K.+Sawyerauthor=D.+J.+Staplesauthor=A.+E.+de+Vauxauthor=O.+Dymauthor=A.+M.+Castrucciauthor=M.+F.+Hintz&title=.alpha.-Melanotropin%3A+The+Minimal+Active+Sequence+in+the+Frog+Skin+Bioassay&doi=10.1021%2Fjm00394a033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">α-Melanotropin:  the minimal active sequence in the frog skin bioassay</span></div><div class="casAuthors">Hruby, Victor J.; Wilkes, Brian C.; Hadley, Mac E.; Al-Obeidi, Fahad; Sawyer, Tomi K.; Staples, Douglas J.; DeVaux, Ann E.; Dym, Orin; Castrucci, Ana Maria de L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2126-30</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A series of fragment analogs of α-MSH (I) were prepd. in order to det. the contribution of each individual amino acid to the biol. activity of the native hormone.  The minimal potency of Ac-α-MSH6-9-NH2 could be enhanced about a factor of 16 by the addn. of glycine to the C-terminus, yielding Ac-α-MSH6-10-NH2.  Addn. of glutamic acid to the N-terminus provided Ac-α-MSH5-10-NH2, which was only slightly more potent than Ac-α-MSH6-10-NH2, indicating that position 5 contributes little to the biol. potency of α-MSH in this assay.  Addn. of methionine to the N-terminus of Ac-α-MSH5-10-NH2 resulted in Ac-α-MSH4-10-NH2, which was only about 4-fold more potent than Ac-α-MSH5-10-NH2.  Addn. of lysine and proline to the C-terminal of the Ac-α-MSH4-10-NH2 sequence yielded Ac-α-MSH4-12-NH2 with a 360-fold increase in potency relative to Ac-α-MSH4-10-NH2.  This peptide was only about 6-fold less potent than α-MSH.  Nle-4-substituted analogs were also prepd.  Ac-[Nle4]-α-MSH4-10-NH2 and Ac-[Nle4]-α-MSH4-11-NH2 were ∼4 times more potent than Ac-α-MSH4-10-NH2, demonstrating that lysine-11 contributes somewhat to the biol. activity of α-MSH on the frog skin melanocyte receptor.  However, addn. of proline-12 to this fragment, yielding Ac-[Nle4]-α-MSH4-12-NH2, resulted in about a 90-fold increase in relative potency of the melanotropin.  Addn. of the final C-terminal valine-13 provided Ac-[Nle4]-α-MSH4-13-NH2, which showed only a small further increase in potency.  This analog was, however, only about 2 to 3-fold less active than α-MSH.  Addn. of the N-terminal tripeptide Ac-Ser-Tyr-Ser to yield [Nle4]-α-MSH resulted in an analog that was 3 times more potent than α-MSH.  The central tetrapeptide sequence, Ac-His-Phe-Arg-Trp-NH2, represents the min. chain length for observable biol. activity.  The active sequence of α-MSH is contiguous in that no two structurally noncontiguous fragment sequences were found to have biol. activity.  Met-4, Gly-10, and Pro-12 are important potentiating amino acids and contribute significantly to the biopotency of α-MSH, and Ser-1 and -3, Tyr-2, Glu-5, Lys-11, and Val-13 apparently contribute only minimally to the biol. potency of α-MSH at the frog melanocyte skin receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLqNbLyfhM07Vg90H21EOLACvtfcHk0liJJ6RtX_WtCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXlslKntLk%253D&md5=453a1b54b73e07414b096237b3bdcbdc</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm00394a033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00394a033%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26aulast%3DWilkes%26aufirst%3DB.%2BC.%26aulast%3DHadley%26aufirst%3DM.%2BE.%26aulast%3DAl-Obeidi%26aufirst%3DF.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DStaples%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BVaux%26aufirst%3DA.%2BE.%26aulast%3DDym%26aufirst%3DO.%26aulast%3DCastrucci%26aufirst%3DA.%2BM.%26aulast%3DHintz%26aufirst%3DM.%2BF.%26atitle%3D.alpha.-Melanotropin%253A%2520The%2520Minimal%2520Active%2520Sequence%2520in%2520the%2520Frog%2520Skin%2520Bioassay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1987%26volume%3D30%26spage%3D2126%26epage%3D2130%26doi%3D10.1021%2Fjm00394a033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslach, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haskell-Luevano, C.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships (SAR) of Melanocortin and Agouti-Related (AGRP) Peptides</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>681</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1007/978-1-4419-6354-3_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2F978-1-4419-6354-3_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=21222256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WrsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=681&publication_year=2010&pages=1-18&author=A.+Singhauthor=E.+M.+Haslachauthor=C.+Haskell-Luevano&title=Structure-Activity+Relationships+%28SAR%29+of+Melanocortin+and+Agouti-Related+%28AGRP%29+Peptides&doi=10.1007%2F978-1-4419-6354-3_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relatinships (SAR) of melanocortin and agouti-related (AGRP) peptides</span></div><div class="casAuthors">Singh, Anamika; Haslach, Erica M.; Haskell-Luevano, Carrie</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">681</span>
        (<span class="NLM_cas:issue">Melanocortins</span>),
    <span class="NLM_cas:pages">1-18</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Structure-activity relationship (SAR) studies are a key feature of peptide and peptidomimetic research to improve the biol. properties of native peptides and convert them into more drug-like compds.  Peptide SAR studies involve the systematic modification of a lead peptide to provide insight into the mol. determinants of the ligand-receptor interactions that result in either receptor stimulation or inhibition.  This chapter will discuss structure-activity relationships of the endogenous and synthetic agonists and the antagonists of the melanocortin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ4R5zOYjvFLVg90H21EOLACvtfcHk0lgROVtLk6c3xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WrsrnL&md5=936828fae7e953678d05fd26a712d31e</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2F978-1-4419-6354-3_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4419-6354-3_1%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DHaslach%26aufirst%3DE.%2BM.%26aulast%3DHaskell-Luevano%26aufirst%3DC.%26atitle%3DStructure-Activity%2520Relationships%2520%2528SAR%2529%2520of%2520Melanocortin%2520and%2520Agouti-Related%2520%2528AGRP%2529%2520Peptides%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2010%26volume%3D681%26spage%3D1%26epage%3D18%26doi%3D10.1007%2F978-1-4419-6354-3_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span> <span> </span><span class="NLM_article-title">Venom Peptides as a Rich Source of Cav2.2 Channel Blockers</span>. <i>Toxins</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">286</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.3390/toxins5020286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3390%2Ftoxins5020286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=23381143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtVWisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=286-314&author=S.+R.+Sousaauthor=I.+Vetterauthor=R.+J.+Lewis&title=Venom+Peptides+as+a+Rich+Source+of+Cav2.2+Channel+Blockers&doi=10.3390%2Ftoxins5020286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Venom peptides as a rich source of Cav2.2 channel blockers</span></div><div class="casAuthors">Sousa, Silmara R.; Vetter, Irina; Lewis, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Toxins</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">286-314</span>CODEN:
                <span class="NLM_cas:coden">TOXIB7</span>;
        ISSN:<span class="NLM_cas:issn">2072-6651</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Cav2.2 is a calcium channel subtype localized at nerve terminals, including nociceptive fibers, where it initiates neurotransmitter release.  Cav2.2 is an important contributor to synaptic transmission in ascending pain pathways, and is up-regulated in the spinal cord in chronic pain states along with the auxiliary α2δ1 subunit.  It is therefore not surprising that toxins that inhibit Cav2.2 are analgesic.  Venomous animals, such as cone snails, spiders, snakes, assassin bugs, centipedes and scorpions are rich sources of remarkably potent and selective Cav2.2 inhibitors.  However, side effects in humans currently limit their clin. use.  Here we review Cav2.2 inhibitors from venoms and their potential as drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP3nbJeRZPY7Vg90H21EOLACvtfcHk0lgROVtLk6c3xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtVWisb8%253D&md5=2f700d274eeab82e73ea7a9a72126ff0</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.3390%2Ftoxins5020286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Ftoxins5020286%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DS.%2BR.%26aulast%3DVetter%26aufirst%3DI.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26atitle%3DVenom%2520Peptides%2520as%2520a%2520Rich%2520Source%2520of%2520Cav2.2%2520Channel%2520Blockers%26jtitle%3DToxins%26date%3D2013%26volume%3D5%26spage%3D286%26epage%3D314%26doi%3D10.3390%2Ftoxins5020286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasparini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danse, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecoq, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkasfeld, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn-Justin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Medeiros, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span> <span> </span><span class="NLM_article-title">Delineation of the Functional Site of α-Dendrotoxin. The Functional Topographies of Dendrotoxins Are Different but Share a Conserved Core with Those of Other Kv1 Potassium Channel-Blocking Toxins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">25393</span>– <span class="NLM_lpage">25403</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.39.25393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.273.39.25393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9738007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1cXmsVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=25393-25403&author=S.+Gaspariniauthor=J.+M.+Danseauthor=A.+Lecoqauthor=S.+Pinkasfeldauthor=S.+Zinn-Justinauthor=L.+C.+Youngauthor=C.+C.+de+Medeirosauthor=E.+G.+Rowanauthor=A.+L.+Harveyauthor=A.+M%C3%A9nez&title=Delineation+of+the+Functional+Site+of+%CE%B1-Dendrotoxin.+The+Functional+Topographies+of+Dendrotoxins+Are+Different+but+Share+a+Conserved+Core+with+Those+of+Other+Kv1+Potassium+Channel-Blocking+Toxins&doi=10.1074%2Fjbc.273.39.25393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Delineation of the functional site of α-dendrotoxin. The functional topographies of dendrotoxins are different but share a conserved core with those of other Kv1 potassium channel-blocking toxins</span></div><div class="casAuthors">Gasparini, Sylvaine; Danse, Jean-Marc; Lecoq, Alain; Pinkasfeld, Suzanne; Zinn-Justin, Sophie; Young, Louise C.; De Medeiros, Cleane C. L.; Rowan, Edward G.; Harvey, Alan L.; Menez, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">25393-25403</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Residues important for the binding of α-dendrotoxin (αDTX) to Kv1 potassium channels on rat brain synaptosomal membranes were identified using a mutational approach based on site-directed mutagenesis and chem. synthesis.  Twenty-six of its 59 residues were individually substituted by alanine.  Substitutions of Lys5 and Leu9 decreased affinity more than 1000-fold, and substitutions of Arg3, Arg4, Leu6, and Ile8 by 5-30-fold.  Substitution of Lys5 by norleucine or ornithine also greatly altered the binding properties of αDTX.  All of these analogs displayed similar CD spectra as compared with the wild-type αDTX, indicating that none of these substitutions affect the overall conformation of the toxin.  Substitutions of Ser38 and Arg46 also reduced the affinity of the toxin but, in addn., modified its dichroic properties, suggesting that these two residues play a structural role.  The other residues were excluded from the recognition site because their substitutions caused no significant affinity change.  Thus, the functional site of αDTX includes six major binding residues, all located in its N-terminal region, with Lys5 and Leu9 being the most important.  Comparison of the functional site of αDTX with that of DTX-K, another dendrotoxin (Smith, L. A., Reid, P. F., Wang, F. C., Parcej, D. N., Schmidt, J. J., Olson, M. A., and Dolly, J. O. (1997) Biochem. 36, 7690-7696), reveals that they only share the predominant lysine and probably a leucine residue; the addnl. functional residues differ from one toxin to the other.  Comparison of the functional site of αDTX with those of structurally unrelated potassium channel-blocking toxins from venomous invertebrates revealed the common presence of a protruding key lysine with a close important hydrophobic residue (Leu, Tyr, or Phe) and few addnl. residues.  Therefore, irresp. of their phylogenetic origin, all of these toxins may have undergone a functional convergence.  The functional site of αDTX is topog. unrelated to the "antiprotease site" of the structurally analogous bovine pancreatic trypsin inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB9cTJoyCsdbVg90H21EOLACvtfcHk0lgROVtLk6c3xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsVGmu7g%253D&md5=1ce3da14c231c642ebaae4e42bdcc05d</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.39.25393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.39.25393%26sid%3Dliteratum%253Aachs%26aulast%3DGasparini%26aufirst%3DS.%26aulast%3DDanse%26aufirst%3DJ.%2BM.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DPinkasfeld%26aufirst%3DS.%26aulast%3DZinn-Justin%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DL.%2BC.%26aulast%3Dde%2BMedeiros%26aufirst%3DC.%2BC.%26aulast%3DRowan%26aufirst%3DE.%2BG.%26aulast%3DHarvey%26aufirst%3DA.%2BL.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26atitle%3DDelineation%2520of%2520the%2520Functional%2520Site%2520of%2520%25CE%25B1-Dendrotoxin.%2520The%2520Functional%2520Topographies%2520of%2520Dendrotoxins%2520Are%2520Different%2520but%2520Share%2520a%2520Conserved%2520Core%2520with%2520Those%2520of%2520Other%2520Kv1%2520Potassium%2520Channel-Blocking%2520Toxins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D25393%26epage%3D25403%26doi%3D10.1074%2Fjbc.273.39.25393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dauplais, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lecoq, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilquin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roumestand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vita, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Medeiros, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ménez, A.</span></span> <span> </span><span class="NLM_article-title">On the Convergent Evolution of Animal Toxins. Conservation of a Diad of Functional Residues in Potassium Channel-Blocking Toxins with Unrelated Structures</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">4302</span>– <span class="NLM_lpage">4309</span>, <span class="refDoi"> DOI: 10.1074/jbc.272.7.4302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1074%2Fjbc.272.7.4302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9020148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2sXht1Oqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=4302-4309&author=M.+Dauplaisauthor=A.+Lecoqauthor=J.+Songauthor=J.+Cottonauthor=N.+Jaminauthor=B.+Gilquinauthor=C.+Roumestandauthor=C.+Vitaauthor=C.+L.+de+Medeirosauthor=E.+G.+Rowanauthor=A.+L.+Harveyauthor=A.+M%C3%A9nez&title=On+the+Convergent+Evolution+of+Animal+Toxins.+Conservation+of+a+Diad+of+Functional+Residues+in+Potassium+Channel-Blocking+Toxins+with+Unrelated+Structures&doi=10.1074%2Fjbc.272.7.4302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">On the convergent evolution of animal toxins. Conservation of a diad of functional residues in potassium channel-blocking toxins with unrelated structures</span></div><div class="casAuthors">Dauplais, Marc; Lecoq, Alain; Song, Jianxing; Cotton, Joel; Jamin, Nadege; Gilquin, Bernard; Roumestand, Christian; Vita, Claudio; de Medeiros, Cleane L. C.; Rowan, Edward G.; Harvey, Alan L.; Menez, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4302-4309</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">BgK is a K+ channel-blocking toxin from the sea anemone Bunodosoma granulifera.  It is a 37-residue protein that adopts a novel fold, as detd. by NMR and modeling.  An alanine-scanning-based anal. revealed the functional importance of five residues, which include a crit. lysine and an arom. residue sepd. by 6.6±1.0 Å.  The same diad is found in the three known homologous toxins from sea anemones.  More strikingly, a similar functional diad is present in all K+ channel-blocking toxins from scorpions, although these toxins adopt a distinct scaffold.  Moreover, the functional diads of potassium channel-blocking toxins from sea anemone and scorpions superimpose in the three-dimensional structures.  Therefore, toxins that have unrelated structures but similar functions possess conserved key functional residues, organized in an identical topol., suggesting a convergent functional evolution for these small proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh-osMmt309bVg90H21EOLACvtfcHk0lh-G16ad6EVSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1Oqsbs%253D&md5=151c3834341c30ba3c901c69854ab5f8</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.7.4302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.7.4302%26sid%3Dliteratum%253Aachs%26aulast%3DDauplais%26aufirst%3DM.%26aulast%3DLecoq%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DCotton%26aufirst%3DJ.%26aulast%3DJamin%26aufirst%3DN.%26aulast%3DGilquin%26aufirst%3DB.%26aulast%3DRoumestand%26aufirst%3DC.%26aulast%3DVita%26aufirst%3DC.%26aulast%3Dde%2BMedeiros%26aufirst%3DC.%2BL.%26aulast%3DRowan%26aufirst%3DE.%2BG.%26aulast%3DHarvey%26aufirst%3DA.%2BL.%26aulast%3DM%25C3%25A9nez%26aufirst%3DA.%26atitle%3DOn%2520the%2520Convergent%2520Evolution%2520of%2520Animal%2520Toxins.%2520Conservation%2520of%2520a%2520Diad%2520of%2520Functional%2520Residues%2520in%2520Potassium%2520Channel-Blocking%2520Toxins%2520with%2520Unrelated%2520Structures%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D4302%26epage%3D4309%26doi%3D10.1074%2Fjbc.272.7.4302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zerbino, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birney, E.</span></span> <span> </span><span class="NLM_article-title">Velvet: Algorithms for de Novo Short Read Assembly Using de Bruijn Graphs</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">829</span>, <span class="refDoi"> DOI: 10.1101/gr.074492.107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1101%2Fgr.074492.107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=18349386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslChsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=821-829&author=D.+R.+Zerbinoauthor=E.+Birney&title=Velvet%3A+Algorithms+for+de+Novo+Short+Read+Assembly+Using+de+Bruijn+Graphs&doi=10.1101%2Fgr.074492.107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</span></div><div class="casAuthors">Zerbino, Daniel R.; Birney, Ewan</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">821-829</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">We have developed a new set of algorithms, collectively called "Velvet," to manipulate de Bruijn graphs for genomic sequence assembly.  A de Bruijn graph is a compact representation based on short words (k-mers) that is ideal for high coverage, very short read (25-50 bp) data sets.  Applying Velvet to very short reads and paired-ends information only, one can produce contigs of significant length, up to 50-kb N50 length in simulations of prokaryotic data and 3-kb N50 on simulated mammalian BACs.  When applied to real Solexa data sets without read pairs, Velvet generated contigs of ∼8 kb in a prokaryote and 2 kb in a mammalian BAC, in close agreement with our simulated results without read-pair information.  Velvet represents a new approach to assembly that can leverage very short reads in combination with read pairs to produce useful assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7wYfw5TD8frVg90H21EOLACvtfcHk0lh-G16ad6EVSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslChsLg%253D&md5=2378b6c6efad2611e0eadbd86f77e01f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1101%2Fgr.074492.107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.074492.107%26sid%3Dliteratum%253Aachs%26aulast%3DZerbino%26aufirst%3DD.%2BR.%26aulast%3DBirney%26aufirst%3DE.%26atitle%3DVelvet%253A%2520Algorithms%2520for%2520de%2520Novo%2520Short%2520Read%2520Assembly%2520Using%2520de%2520Bruijn%2520Graphs%26jtitle%3DGenome%2520Res.%26date%3D2008%26volume%3D18%26spage%3D821%26epage%3D829%26doi%3D10.1101%2Fgr.074492.107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grabherr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yassour, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levin, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amit, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adiconis, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raychowdhury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauceli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hacohen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnirke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhind, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">di Palma, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birren, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nusbaum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindblad-Toh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span> <span> </span><span class="NLM_article-title">Full-Length Transcriptome Assembly from RNA-Seq Data without a Reference Genome</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">644</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1038/nbt.1883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1038%2Fnbt.1883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=21572440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtV2hsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=644-652&author=M.+G.+Grabherrauthor=B.+J.+Haasauthor=M.+Yassourauthor=J.+Z.+Levinauthor=D.+A.+Thompsonauthor=I.+Amitauthor=X.+Adiconisauthor=L.+Fanauthor=R.+Raychowdhuryauthor=Q.+Zengauthor=Z.+Chenauthor=E.+Mauceliauthor=N.+Hacohenauthor=A.+Gnirkeauthor=N.+Rhindauthor=F.+di+Palmaauthor=B.+W.+Birrenauthor=C.+Nusbaumauthor=K.+Lindblad-Tohauthor=N.+Friedmanauthor=A.+Regev&title=Full-Length+Transcriptome+Assembly+from+RNA-Seq+Data+without+a+Reference+Genome&doi=10.1038%2Fnbt.1883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Full-length transcriptome assembly from RNA-Seq data without a reference genome</span></div><div class="casAuthors">Grabherr, Manfred G.; Haas, Brian J.; Yassour, Moran; Levin, Joshua Z.; Thompson, Dawn A.; Amit, Ido; Adiconis, Xian; Fan, Lin; Raychowdhury, Raktima; Zeng, Qiandong; Chen, Zehua; Mauceli, Evan; Hacohen, Nir; Gnirke, Andreas; Rhind, Nicholas; di Palma, Federica; Birren, Bruce W.; Nusbaum, Chad; Lindblad-Toh, Kerstin; Friedman, Nir; Regev, Aviv</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">644-652</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Massively parallel sequencing of cDNA has enabled deep and efficient probing of transcriptomes.  Current approaches for transcript reconstruction from such data often rely on aligning reads to a ref. genome, and are thus unsuitable for samples with a partial or missing ref. genome.  Here we present the Trinity method for de novo assembly of full-length transcripts and evaluate it on samples from fission yeast, mouse and whitefly, whose ref. genome is not yet available.  By efficiently constructing and analyzing sets of de Bruijn graphs, Trinity fully reconstructs a large fraction of transcripts, including alternatively spliced isoforms and transcripts from recently duplicated genes.  Compared with other de novo transcriptome assemblers, Trinity recovers more full-length transcripts across a broad range of expression levels, with a sensitivity similar to methods that rely on genome alignments.  Our approach provides a unified soln. for transcriptome reconstruction in any sample, esp. in the absence of a ref. genome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprN3mUm_p2wbVg90H21EOLACvtfcHk0lh-G16ad6EVSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtV2hsbc%253D&md5=975f291debee3f027d32c13688b57f1b</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1883%26sid%3Dliteratum%253Aachs%26aulast%3DGrabherr%26aufirst%3DM.%2BG.%26aulast%3DHaas%26aufirst%3DB.%2BJ.%26aulast%3DYassour%26aufirst%3DM.%26aulast%3DLevin%26aufirst%3DJ.%2BZ.%26aulast%3DThompson%26aufirst%3DD.%2BA.%26aulast%3DAmit%26aufirst%3DI.%26aulast%3DAdiconis%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DRaychowdhury%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DMauceli%26aufirst%3DE.%26aulast%3DHacohen%26aufirst%3DN.%26aulast%3DGnirke%26aufirst%3DA.%26aulast%3DRhind%26aufirst%3DN.%26aulast%3Ddi%2BPalma%26aufirst%3DF.%26aulast%3DBirren%26aufirst%3DB.%2BW.%26aulast%3DNusbaum%26aufirst%3DC.%26aulast%3DLindblad-Toh%26aufirst%3DK.%26aulast%3DFriedman%26aufirst%3DN.%26aulast%3DRegev%26aufirst%3DA.%26atitle%3DFull-Length%2520Transcriptome%2520Assembly%2520from%2520RNA-Seq%2520Data%2520without%2520a%2520Reference%2520Genome%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D644%26epage%3D652%26doi%3D10.1038%2Fnbt.1883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozach, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blemont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincentelli, R.</span></span> <span> </span><span class="NLM_article-title">High Throughput Quantitative Expression Screening and Purification Applied to Recombinant Disulfide-Rich Venom Proteins Produced in <i>E. Coli</i></span>. <i>J. Visualized Exp.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>89</i></span>, <span class="NLM_elocation-id">e51464</span> <span class="refDoi"> DOI: 10.3791/51464</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.3791%2F51464" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2014&author=N.+J.+Saezauthor=H.+Nozachauthor=M.+Blemontauthor=R.+Vincentelli&title=High+Throughput+Quantitative+Expression+Screening+and+Purification+Applied+to+Recombinant+Disulfide-Rich+Venom+Proteins+Produced+in+E.+Coli&doi=10.3791%2F51464"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.3791%2F51464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3791%252F51464%26sid%3Dliteratum%253Aachs%26aulast%3DSaez%26aufirst%3DN.%2BJ.%26aulast%3DNozach%26aufirst%3DH.%26aulast%3DBlemont%26aufirst%3DM.%26aulast%3DVincentelli%26aufirst%3DR.%26atitle%3DHigh%2520Throughput%2520Quantitative%2520Expression%2520Screening%2520and%2520Purification%2520Applied%2520to%2520Recombinant%2520Disulfide-Rich%2520Venom%2520Proteins%2520Produced%2520in%2520E.%2520Coli%26jtitle%3DJ.%2520Visualized%2520Exp.%26date%3D2014%26volume%3D89%26doi%3D10.3791%2F51464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vranken, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fogh, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pajon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llinas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulrich, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markley, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ionides, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laue, E. D.</span></span> <span> </span><span class="NLM_article-title">The CCPN Data Model for NMR Spectroscopy: Development of a Software Pipeline</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">696</span>, <span class="refDoi"> DOI: 10.1002/prot.20449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1002%2Fprot.20449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=15815974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2005&pages=687-696&author=W.+F.+Vrankenauthor=W.+Boucherauthor=T.+J.+Stevensauthor=R.+H.+Foghauthor=A.+Pajonauthor=M.+Llinasauthor=E.+L.+Ulrichauthor=J.+L.+Markleyauthor=J.+Ionidesauthor=E.+D.+Laue&title=The+CCPN+Data+Model+for+NMR+Spectroscopy%3A+Development+of+a+Software+Pipeline&doi=10.1002%2Fprot.20449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">The CCPN data model for NMR spectroscopy: development of a software pipeline</span></div><div class="casAuthors">Vranken, Wim F.; Boucher, Wayne; Stevens, Tim J.; Fogh, Rasmus H.; Pajon, Anne; Llinas, Miguel; Ulrich, Eldon L.; Markley, John L.; Ionides, John; Laue, Ernest D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-696</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">To address data management and data exchange problems in the NMR community, the Collaborative Computing Project for the NMR community (CCPN) created a "Data Model" that describes all the different types of information needed in an NMR structural study, from mol. structure and NMR parameters to coordinates.  This paper describes the development of a set of software applications that use the Data Model and its assocd. libraries, thus validating the approach.  These applications are freely available and provide a pipeline for high-throughput anal. of NMR data.  Three programs work directly with the Data Model: CcpNmr Anal., an entirely new anal. and interactive display program, the CcpNmr FormatConverter, which allows transfer of data from programs commonly used in NMR to and from the Data Model, and the CLOUDS software for automated structure calcn. and assignment (Carnegie Mellon University), which was rewritten to interact directly with the Data Model.  The ARIA 2.0 software for structure calcn. (Institut Pasteur) and the QUEEN program for validation of restraints (University of Nijmegen) were extended to provide conversion of their data to the Data Model.  During these developments the Data Model has been thoroughly tested and used, demonstrating that applications can successfully exchange data via the Data Model.  The software architecture developed by CCPN is now ready for new developments, such as integration with addnl. software applications and extensions of the Data Model into other areas of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpl6jPIIOYWbVg90H21EOLACvtfcHk0lhkkkY75VszQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D&md5=b07b9701f64f281d58d178f9c2033596</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fprot.20449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20449%26sid%3Dliteratum%253Aachs%26aulast%3DVranken%26aufirst%3DW.%2BF.%26aulast%3DBoucher%26aufirst%3DW.%26aulast%3DStevens%26aufirst%3DT.%2BJ.%26aulast%3DFogh%26aufirst%3DR.%2BH.%26aulast%3DPajon%26aufirst%3DA.%26aulast%3DLlinas%26aufirst%3DM.%26aulast%3DUlrich%26aufirst%3DE.%2BL.%26aulast%3DMarkley%26aufirst%3DJ.%2BL.%26aulast%3DIonides%26aufirst%3DJ.%26aulast%3DLaue%26aufirst%3DE.%2BD.%26atitle%3DThe%2520CCPN%2520Data%2520Model%2520for%2520NMR%2520Spectroscopy%253A%2520Development%2520of%2520a%2520Software%2520Pipeline%26jtitle%3DProteins%26date%3D2005%26volume%3D59%26spage%3D687%26epage%3D696%26doi%3D10.1002%2Fprot.20449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rieping, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeck, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardiaux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malliavin, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span> <span> </span><span class="NLM_article-title">ARIA2: Automated NOE Assignment and Data Integration in NMR Structure Calculation</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">382</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btl589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1093%2Fbioinformatics%2Fbtl589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=17121777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=381-382&author=W.+Riepingauthor=M.+Habeckauthor=B.+Bardiauxauthor=A.+Bernardauthor=T.+E.+Malliavinauthor=M.+Nilges&title=ARIA2%3A+Automated+NOE+Assignment+and+Data+Integration+in+NMR+Structure+Calculation&doi=10.1093%2Fbioinformatics%2Fbtl589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">ARIA2: Automated NOE assignment and data integration in NMR structure calculation</span></div><div class="casAuthors">Rieping, Wolfgang; Habeck, Michael; Bardiaux, Benjamin; Bernard, Aymeric; Malliavin, Therese E.; Nilges, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-382</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: Modern structural genomics projects demand for integrated methods for the interpretation and storage of NMR data.  Here the authors present version 2.1 of the authors' program ARIA (Ambiguous Restraints for Iterative Assignment) for automated assignment of nuclear Overhauser enhancement (NOE) data and NMR structure calcn.  The authors report on recent developments, most notably a graphical user interface, and the incorporation of the object-oriented data model of the Collaborative Computing Project for NMR (CCPN).  The CCPN data model defines a storage model for NMR data, which greatly facilitates the transfer of data between different NMR software packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2HL9x73wPfrVg90H21EOLACvtfcHk0lhkkkY75VszQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOltb8%253D&md5=75e604b2faf4375ace7ea455365737f1</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtl589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtl589%26sid%3Dliteratum%253Aachs%26aulast%3DRieping%26aufirst%3DW.%26aulast%3DHabeck%26aufirst%3DM.%26aulast%3DBardiaux%26aufirst%3DB.%26aulast%3DBernard%26aufirst%3DA.%26aulast%3DMalliavin%26aufirst%3DT.%2BE.%26aulast%3DNilges%26aufirst%3DM.%26atitle%3DARIA2%253A%2520Automated%2520NOE%2520Assignment%2520and%2520Data%2520Integration%2520in%2520NMR%2520Structure%2520Calculation%26jtitle%3DBioinformatics%26date%3D2007%26volume%3D23%26spage%3D381%26epage%3D382%26doi%3D10.1093%2Fbioinformatics%2Fbtl589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cornilescu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaglio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bax, A.</span></span> <span> </span><span class="NLM_article-title">Protein Backbone Angle Restraints from Searching a Database for Chemical Shift and Sequence Homology</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1023/A:1008392405740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1023%2FA%3A1008392405740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10212987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK1MXitlejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1999&pages=289-302&author=G.+Cornilescuauthor=F.+Delaglioauthor=A.+Bax&title=Protein+Backbone+Angle+Restraints+from+Searching+a+Database+for+Chemical+Shift+and+Sequence+Homology&doi=10.1023%2FA%3A1008392405740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Protein backbone angle restraints from searching a database for chemical shift and sequence homology</span></div><div class="casAuthors">Cornilescu, Gabriel; Delaglio, Frank; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">289-302</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">Chem. shifts of backbone atoms in proteins are exquisitely sensitive to local conformation, and homologous proteins show quite similar patterns of secondary chem. shifts.  The inverse of this relation is used to search a database for triplets of adjacent residues with secondary chem. shifts and sequence similarity which provide the best match to the query triplet of interest.  The database contains 13Cα, 13Cβ, 13C', 1Hα and 15N chem. shifts for 20 proteins for which a high resoln. x-ray structure is available.  The computer program TALOS was developed to search this database for strings of residues with chem. shift and residue type homol.  The relative importance of the weighting factors attached to the secondary chem. shifts of the five types of resonances relative to that of sequence similarity was optimized empirically.  TALOS yields the 10 triplets which have the closest similarity in secondary chem. shift and amino acid sequence to those of the query sequence.  If the central residues in these 10 triplets exhibit similar φ and ψ backbone angles, their avs. can reliably be used as angular restraints for the protein whose structure is being studied.  Tests carried out for proteins of known structure indicate that the root-mean-square difference (rmsd) between the output of TALOS and the x-ray derived backbone angles is about 15°.  Approx. 3% of the predictions made by TALOS are found to be in error.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcjgFEXHFQV7Vg90H21EOLACvtfcHk0lg76susdGYAKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitlejsL8%253D&md5=1083ff7ba22757c8eaa78e9cc507d1db</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008392405740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008392405740%26sid%3Dliteratum%253Aachs%26aulast%3DCornilescu%26aufirst%3DG.%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DProtein%2520Backbone%2520Angle%2520Restraints%2520from%2520Searching%2520a%2520Database%2520for%2520Chemical%2520Shift%2520and%2520Sequence%2520Homology%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1999%26volume%3D13%26spage%3D289%26epage%3D302%26doi%3D10.1023%2FA%3A1008392405740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hutchinson, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">PROMOTIF--a Program to Identify and Analyze Structural Motifs in Proteins</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">212</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1002/pro.5560050204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1002%2Fpro.5560050204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=8745398" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK28XhtVagtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1996&pages=212-220&author=E.+G.+Hutchinsonauthor=J.+M.+Thornton&title=PROMOTIF%2D%2Da+Program+to+Identify+and+Analyze+Structural+Motifs+in+Proteins&doi=10.1002%2Fpro.5560050204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">PROMOTIF - a program to identify and analyze structural motifs in proteins</span></div><div class="casAuthors">Hutchinson, E. Gail; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">212-20</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The authors described a suite of programs, PROMOTIF, that analyzes a protein coordinate file and provides details about the structural motifs in the protein.  The program currently analyzes the following structural features: secondary structure; β- and γ-turns; helical geometry and interactions; β-strands and β-sheet topol.; β-bulges; β-hairpins; β-α-β units and ψ-loops; disulfide bridges; and main-chain hydrogen bonding patterns.  PROMOTIF creates postscripts files showing the examples of each type of motif in the protein, and a summary page.  The program can also be used to compare motifs in a group of related structures, such as an ensemble of NMR structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqR_hvCDMO8mLVg90H21EOLACvtfcHk0lg76susdGYAKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtVagtL8%253D&md5=72e3118995037f7b28ca1421e2990585</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560050204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560050204%26sid%3Dliteratum%253Aachs%26aulast%3DHutchinson%26aufirst%3DE.%2BG.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROMOTIF--a%2520Program%2520to%2520Identify%2520and%2520Analyze%2520Structural%2520Motifs%2520in%2520Proteins%26jtitle%3DProtein%2520Sci.%26date%3D1996%26volume%3D5%26spage%3D212%26epage%3D220%26doi%3D10.1002%2Fpro.5560050204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rullmannn, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title">AQUA and PROCHECK-NMR: Programs for Checking the Quality of Protein Structures Solved by NMR</span>. <i>J. Biomol. NMR</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">477</span>– <span class="NLM_lpage">486</span>, <span class="refDoi"> DOI: 10.1007/BF00228148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1007%2FBF00228148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=9008363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1996&pages=477-486&author=R.+A.+Laskowskiauthor=J.+A.+Rullmannnauthor=M.+W.+MacArthurauthor=R.+Kapteinauthor=J.+M.+Thornton&title=AQUA+and+PROCHECK-NMR%3A+Programs+for+Checking+the+Quality+of+Protein+Structures+Solved+by+NMR&doi=10.1007%2FBF00228148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">AQUA and PROCHECK-NMR: programs for checking the quality of protein structures solved by NMR</span></div><div class="casAuthors">Laskowski, Roman A.; Antoon, J.; Rullmann, C.; Macarthur, Malcolm W.; Kaptein, Robert; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">477-486</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The AQUA and PROCHECK-NMR programs provide a means of validating the geometry and restraint violations of an ensemble of protein structures solved by soln. NMR.  The outputs include a detailed breakdown of the restraint violations, a no. of plots in PostScript format and summary statistics.  These various analyses indicate both the degree of agreement of the model structures with the exptl. data, and the quality of their geometrical properties.  They are intended to be of use both to support ongoing NMR structure detn. and in the validation of the final results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsafj3zezC3rVg90H21EOLACvtfcHk0lg76susdGYAKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvVSnsw%253D%253D&md5=030956fb35cbfdaab78801c61ba5a3e1</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2FBF00228148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00228148%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DRullmannn%26aufirst%3DJ.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DKaptein%26aufirst%3DR.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DAQUA%2520and%2520PROCHECK-NMR%253A%2520Programs%2520for%2520Checking%2520the%2520Quality%2520of%2520Protein%2520Structures%2520Solved%2520by%2520NMR%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1996%26volume%3D8%26spage%3D477%26epage%3D486%26doi%3D10.1007%2FBF00228148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldenburg, K. R.</span></span> <span> </span><span class="NLM_article-title">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span>. <i>J. Biomol. Screen.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1177/108705719900400206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1177%2F108705719900400206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10838414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1999&pages=67-73&author=J.+H.+Zhangauthor=T.+D.+Chungauthor=K.+R.+Oldenburg&title=A+Simple+Statistical+Parameter+for+Use+in+Evaluation+and+Validation+of+High+Throughput+Screening+Assays&doi=10.1177%2F108705719900400206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Statistical Parameter for Use in Evaluation and Validation of High Throughput Screening Assays</span></div><div class="casAuthors">Zhang; Chung; Oldenburg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of biomolecular screening</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">67-73</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The ability to identify active compounds (3hits2) from large chemical libraries accurately and rapidly has been the ultimate goal in developing high-throughput screening (HTS) assays.  The ability to identify hits from a particular HTS assay depends largely on the suitability or quality of the assay used in the screening.  The criteria or parameters for evaluating the 3suitability2 of an HTS assay for hit identification are not well defined and hence it still remains difficult to compare the quality of assays directly.  In this report, a screening window coefficient, called 3Z-factor,2 is defined.  This coefficient is reflective of both the assay signal dynamic range and the data variation associated with the signal measurements, and therefore is suitable for assay quality assessment.  The Z-factor is a dimensionless, simple statistical characteristic for each HTS assay.  The Z-factor provides a useful tool for comparison and evaluation of the quality of assays, and can be utilized in assay optimization and validation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkwW5JDdXnbsNIRw0GKUoEfW6udTcc2ebfrncY8c7Tirntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sbit12juw%253D%253D&md5=0ec4ed2ccdd3b4a16521567a0975d31f</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1177%2F108705719900400206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F108705719900400206%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%2BH.%26aulast%3DChung%26aufirst%3DT.%2BD.%26aulast%3DOldenburg%26aufirst%3DK.%2BR.%26atitle%3DA%2520Simple%2520Statistical%2520Parameter%2520for%2520Use%2520in%2520Evaluation%2520and%2520Validation%2520of%2520High%2520Throughput%2520Screening%2520Assays%26jtitle%3DJ.%2520Biomol.%2520Screen.%26date%3D1999%26volume%3D4%26spage%3D67%26epage%3D73%26doi%3D10.1177%2F108705719900400206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the Inhibition Constant (<i>K<sub>I</sub></i>) and the Concentration of Inhibitor Which Causes 50 per Cent Inhibition (<i>I</i><sub>50</sub>) of an Enzymatic Reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=4202581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=Y.-C.+Chengauthor=W.+H.+Prusoff&title=Relationship+between+the+Inhibition+Constant+%28KI%29+and+the+Concentration+of+Inhibitor+Which+Causes+50+per+Cent+Inhibition+%28I50%29+of+an+Enzymatic+Reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Relation between the inhibition constant K1) and the concentration of inhibitor which causes fifty per cent inhibition (I50) of an enzymic reaction</span></div><div class="casAuthors">Cheng, Yung-Chi; Prusoff, William H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3099-108</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    </div><div class="casAbstract">The Ki and I50 values are equal where the kinetics are non- or uncompetitive, but not where they are competitive.  The relation between the 2 values was analyzed for non- and uncompetitive bi- and monosubstrate reactions and for competitive bisubstrate reactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqodFY_8msz57Vg90H21EOLACvtfcHk0ljusyFSVgp6Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXhtVGgs7c%253D&md5=fe7175ca300d8f4085df87af37f8656c</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DY.-C.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520Inhibition%2520Constant%2520%2528KI%2529%2520and%2520the%2520Concentration%2520of%2520Inhibitor%2520Which%2520Causes%252050%2520per%2520Cent%2520Inhibition%2520%2528I50%2529%2520of%2520an%2520Enzymatic%2520Reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leipold, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescrinier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Lima, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span> <span> </span><span class="NLM_article-title">Where Cone Snails and Spiders Meet: Design of Small Cyclic Sodium-Channel Inhibitors</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">3693</span>– <span class="NLM_lpage">3703</span>, <span class="refDoi"> DOI: 10.1096/fj.201801909R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1096%2Ffj.201801909R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=30509130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptl2isbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=3693-3703&author=S.+Peigneurauthor=O.+Chenevalauthor=M.+Maitiauthor=E.+Leipoldauthor=S.+H.+Heinemannauthor=E.+Lescrinierauthor=P.+Herdewijnauthor=M.+E.+De+Limaauthor=D.+J.+Craikauthor=C.+I.+Schroederauthor=J.+Tytgat&title=Where+Cone+Snails+and+Spiders+Meet%3A+Design+of+Small+Cyclic+Sodium-Channel+Inhibitors&doi=10.1096%2Ffj.201801909R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Where cone snails and spiders meet: design of small cyclic sodium-channel inhibitors</span></div><div class="casAuthors">Peigneur, Steve; Cheneval, Olivier; Maiti, Mohitosh; Leipold, Enrico; Heinemann, Stefan H.; Lescrinier, Eveline; Herdewijn, Piet; De Lima, Maria Elena; Craik, David J.; Schroeder, Christina I.; Tytgat, Jan</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">3693-3703</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A13 aa residue voltage-gated sodium (NaV) channel inhibitor peptide, Pn, contg. 2 disulfide bridges was designed by using a chimeric approach.  This approach was based on a common pharmacophore deduced from sequence and secondary structural homol. of 2NaV inhibitors: Conus kinoshitai toxin IIIA, a 14 residue cone snail peptide with 3 disulfide bonds, and Phoneutria nigriventer toxin 1, a 78 residue spider toxin with 7 disulfide bonds.  As with the parent peptides, this novel NaV channel inhibitor was active on NaV1.2.  Through the generation of 3 series of peptide mutants, we investigated the role of key residues and cyclization and their influence on NaV inhibition and subtype selectivity.  Cyclic PnCS1, a 10 residue peptide cyclized via a disulfide bond, exhibited increased inhibitory activity toward therapeutically relevant NaV channel subtypes, including NaV1.7 and NaV1.9, while displaying remarkable serum stability.  These peptides represent the first and the smallest cyclic peptide NaV modulators to date and are promising templates for the development of toxin-based therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcVQf9N4HK07Vg90H21EOLACvtfcHk0ljusyFSVgp6Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptl2isbc%253D&md5=62d7184451dfdaff85c47db8c427f013</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1096%2Ffj.201801909R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201801909R%26sid%3Dliteratum%253Aachs%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DMaiti%26aufirst%3DM.%26aulast%3DLeipold%26aufirst%3DE.%26aulast%3DHeinemann%26aufirst%3DS.%2BH.%26aulast%3DLescrinier%26aufirst%3DE.%26aulast%3DHerdewijn%26aufirst%3DP.%26aulast%3DDe%2BLima%26aufirst%3DM.%2BE.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26aulast%3DTytgat%26aufirst%3DJ.%26atitle%3DWhere%2520Cone%2520Snails%2520and%2520Spiders%2520Meet%253A%2520Design%2520of%2520Small%2520Cyclic%2520Sodium-Channel%2520Inhibitors%26jtitle%3DFASEB%2520J.%26date%3D2019%26volume%3D33%26spage%3D3693%26epage%3D3703%26doi%3D10.1096%2Ffj.201801909R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentz, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoubas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, G. M.</span></span> <span> </span><span class="NLM_article-title">A Rational Nomenclature for Naming Peptide Toxins from Spiders and Other Venomous Animals</span>. <i>Toxicon</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1016/j.toxicon.2008.05.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=10.1016%2Fj.toxicon.2008.05.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=18619481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVaisrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2008&pages=264-276&author=G.+F.+Kingauthor=M.+C.+Gentzauthor=P.+Escoubasauthor=G.+M.+Nicholson&title=A+Rational+Nomenclature+for+Naming+Peptide+Toxins+from+Spiders+and+Other+Venomous+Animals&doi=10.1016%2Fj.toxicon.2008.05.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">A rational nomenclature for naming peptide toxins from spiders and other venomous animals</span></div><div class="casAuthors">King, Glenn F.; Gentz, Margaret C.; Escoubas, Pierre; Nicholson, Graham M.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicon</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-276</span>CODEN:
                <span class="NLM_cas:coden">TOXIA6</span>;
        ISSN:<span class="NLM_cas:issn">0041-0101</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mol. toxinol. research was initially driven by an interest in the small subset of animal toxins that are lethal to humans.  However, the realization that many venomous creatures possess a complex repertoire of bioactive peptide toxins with potential pharmaceutical and agrochem. applications has led to an explosion in the no. of new peptide toxins being discovered and characterized.  Unfortunately, this increased awareness of peptide-toxin diversity has not been matched by the development of a generic nomenclature that enables these toxins to be rationally classified, catalogued, and compared.  In this article, we introduce a rational nomenclature that can be applied to the naming of peptide toxins from spiders and other venomous animals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvrDZuDtMdNbVg90H21EOLACvtfcHk0ljusyFSVgp6Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVaisrjE&md5=643b380308cdc16f9369e180fa787837</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.toxicon.2008.05.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.toxicon.2008.05.020%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DG.%2BF.%26aulast%3DGentz%26aufirst%3DM.%2BC.%26aulast%3DEscoubas%26aufirst%3DP.%26aulast%3DNicholson%26aufirst%3DG.%2BM.%26atitle%3DA%2520Rational%2520Nomenclature%2520for%2520Naming%2520Peptide%2520Toxins%2520from%2520Spiders%2520and%2520Other%2520Venomous%2520Animals%26jtitle%3DToxicon%26date%3D2008%26volume%3D52%26spage%3D264%26epage%3D276%26doi%3D10.1016%2Fj.toxicon.2008.05.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAA','PDB','6SAA'); return false;">PDB: 6SAA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SAB','PDB','6SAB'); return false;">PDB: 6SAB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HYK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1HYK','PDB','1HYK'); return false;">PDB: 1HYK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KJ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KJ6','PDB','1KJ6'); return false;">PDB: 1KJ6</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00485">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34388"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00485?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00485</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">LC–MS analysis of five representative toxins with their cystein patterns, produced by solid-phase chemical synthesis; analysis of four representative toxins with their cystein patterns, produced by recombinant expression; quality assessment at each stage of N-TRTX-Preg1a and N-BUTX-Ptr1a production in Fmoc solid-phase chemical synthesis; activity of 100 μM N-TRTX-Preg1a on ion channels; activity of 100 μM N-BUTX-Ptr1a on ion channels; cysteine pattern diversity for toxins containing an even number of cysteines identified in the Venomics bank; cysteine pattern diversity for toxins containing an odd number of cysteine residues identified in the Venomics bank; experimental NMR data used for the calculations and final structural statistics for the 10 models representative of the solution structure of N-TRTX-Preg1a; experimental NMR data used for the calculations and final structural statistics for the 10 models representative of the solution structure of N-BUTX-Ptr1a (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00485/suppl_file/jm0c00485_si_001.pdf">jm0c00485_si_001.pdf (806.64 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00485&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00485%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-15%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00485" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a06df8d663dc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
