
<html lang="en"     class="pb-page"  data-request-id="3c2422e2-3b48-4822-a6cf-b84783010168"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b02061;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-6"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Protein-Induced Change in Ligand Protonation during Trypsin and Thrombin Binding: Hint on Differences in Selectivity Determinants of Both Proteins?" /></meta><meta name="dc.Creator" content="Khang  Ngo" /></meta><meta name="dc.Creator" content="Chelsey  Collins-Kautz" /></meta><meta name="dc.Creator" content="Stefan  Gerstenecker" /></meta><meta name="dc.Creator" content="Björn  Wagner" /></meta><meta name="dc.Creator" content="Andreas  Heine" /></meta><meta name="dc.Creator" content="Gerhard  Klebe" /></meta><meta name="dc.Description" content="Trypsin and thrombin, structurally similar serine proteases, recognize different substrates; thrombin cleaves after Arg, whereas trypsin cleaves after Lys/Arg. Both recognize basic substrate headgr..." /></meta><meta name="Description" content="Trypsin and thrombin, structurally similar serine proteases, recognize different substrates; thrombin cleaves after Arg, whereas trypsin cleaves after Lys/Arg. Both recognize basic substrate headgr..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 3, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b02061" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02061" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b02061" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02061" /></link>
        
    
    

<title>Protein-Induced Change in Ligand Protonation during Trypsin and Thrombin Binding: Hint on Differences in Selectivity Determinants of Both Proteins? | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b02061" /></meta><meta property="og:title" content="Protein-Induced Change in Ligand Protonation during Trypsin and Thrombin Binding: Hint on Differences in Selectivity Determinants of Both Proteins?" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0005.jpeg" /></meta><meta property="og:description" content="Trypsin and thrombin, structurally similar serine proteases, recognize different substrates; thrombin cleaves after Arg, whereas trypsin cleaves after Lys/Arg. Both recognize basic substrate headgroups via Asp189 at the bottom of the S1 pocket. By crystallography and isothermal titration calorimetry (ITC), we studied a series of d-Phe/d-DiPhe-Pro-(amino)pyridines. Identical ligand pairs show the same binding poses. Surprisingly, one ligand binds to trypsin in protonated state and to thrombin in unprotonated state at P1 along with differences in the residual solvation pattern. While trypsin binding is mediated by an ordered water molecule, in thrombin, water is scattered over three hydration sites. Although having highly similar S1 pockets, our results suggest different electrostatic properties of Asp189 possibly contributing to the selectivity determinant. Thrombin binds a specific Na+ ion next to Asp189, which is absent in trypsin. The electrostatic properties across the S1 pocket are further attenuated by charged Glu192 at the rim of S1 in thrombin, which is replaced by uncharged Gln192 in trypsin." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b02061"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02061">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b02061&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b02061&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b02061&amp;href=/doi/10.1021/acs.jmedchem.9b02061" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3274-3289</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02022" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b02063" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Protein-Induced Change in Ligand Protonation during Trypsin and Thrombin Binding: Hint on Differences in Selectivity Determinants of Both Proteins?</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Khang Ngo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Khang Ngo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Khang++Ngo">Khang Ngo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chelsey Collins-Kautz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chelsey Collins-Kautz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chelsey++Collins-Kautz">Chelsey Collins-Kautz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Gerstenecker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Gerstenecker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Gerstenecker">Stefan Gerstenecker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Björn Wagner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Björn Wagner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pharma Research Non-Clinical Safety, F. Hoffmann-La Roche AG, 4070 Basel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bj%C3%B6rn++Wagner">Björn Wagner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Heine</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Heine</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Heine">Andreas Heine</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Gerhard Klebe</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gerhard Klebe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Philipps-University Marburg, Marbacher Weg 6, 35032 Marburg, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#6c0700090e092c1f180d0a0a4219020541010d1e0e191e0b420809"><span class="__cf_email__" data-cfemail="5b30373e393e1b282f3a3d3d752e353276363a29392e293c753f3e">[email protected]</span></a>. Tel.: +49 6421 28 21313.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gerhard++Klebe">Gerhard Klebe</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4913-390X" title="Orcid link">http://orcid.org/0000-0002-4913-390X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02061&amp;href=/doi/10.1021%2Facs.jmedchem.9b02061" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3274–3289</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 3, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 December 2019</li><li><span class="item_label"><b>Published</b> online</span>3 February 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b02061" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02061</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3274%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKhang%2BNgo%252C%2BChelsey%2BCollins-Kautz%252C%2BStefan%2BGerstenecker%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D6%26contentID%3Dacs.jmedchem.9b02061%26title%3DProtein-Induced%2BChange%2Bin%2BLigand%2BProtonation%2Bduring%2BTrypsin%2Band%2BThrombin%2BBinding%253A%2BHint%2Bon%2BDifferences%2Bin%2BSelectivity%2BDeterminants%2Bof%2BBoth%2BProteins%253F%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3289%26publicationDate%3DMarch%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b02061"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">523</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b02061" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Protein-Induced Change in Ligand Protonation during Trypsin and Thrombin Binding: Hint on Differences in Selectivity Determinants of Both Proteins?&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Khang&quot;,&quot;last_name&quot;:&quot;Ngo&quot;},{&quot;first_name&quot;:&quot;Chelsey&quot;,&quot;last_name&quot;:&quot;Collins-Kautz&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Gerstenecker&quot;},{&quot;first_name&quot;:&quot;Björn&quot;,&quot;last_name&quot;:&quot;Wagner&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Heine&quot;},{&quot;first_name&quot;:&quot;Gerhard&quot;,&quot;last_name&quot;:&quot;Klebe&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;3274-3289&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b02061&quot;},&quot;abstract&quot;:&quot;Trypsin and thrombin, structurally similar serine proteases, recognize different substrates; thrombin cleaves after Arg, whereas trypsin cleaves after Lys/Arg. Both recognize basic substrate headgroups via Asp189 at the bottom of the S1 pocket. By crystallography and isothermal titration calorimetry (ITC), we studied a series of d-Phe/d-DiPhe-Pro-(amino)pyridines. Identical ligand pairs show the same binding poses. Surprisingly, one ligand binds to trypsin in protonated state and to thrombin in unprotonated state at P1 along with differences in the residual solvation pattern. While trypsin binding is mediated by an ordered water molecule, in thrombin, water is scattered over three hydration sites. Although having highly similar S1 pockets, our results suggest different electrostatic properties of Asp189 possibly contributing to the selectivity determinant. Thrombin binds a specific Na+ ion next to Asp189, which is absent in trypsin. The electrostatic properties across the S1 pocket are further attenuated by c&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02061&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02061" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02061&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02061" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02061&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02061" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02061&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02061&amp;href=/doi/10.1021/acs.jmedchem.9b02061" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b02061" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b02061" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b02061%26sid%3Dliteratum%253Aachs%26pmid%3D32011145%26genre%3Darticle%26aulast%3DNgo%26date%3D2020%26atitle%3DProtein-Induced%2BChange%2Bin%2BLigand%2BProtonation%2Bduring%2BTrypsin%2Band%2BThrombin%2BBinding%253A%2BHint%2Bon%2BDifferences%2Bin%2BSelectivity%2BDeterminants%2Bof%2BBoth%2BProteins%253F%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D6%26spage%3D3274%26epage%3D3289%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291309" title="Reaction mechanisms">Reaction mechanisms</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/jmcmar.2020.63.issue-6/20200326/jmcmar.2020.63.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Trypsin and thrombin, structurally similar serine proteases, recognize different substrates; thrombin cleaves after Arg, whereas trypsin cleaves after Lys/Arg. Both recognize basic substrate headgroups via Asp189 at the bottom of the S1 pocket. By crystallography and isothermal titration calorimetry (ITC), we studied a series of <span class="smallcaps smallerCapital">d</span>-Phe/<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-(amino)pyridines. Identical ligand pairs show the same binding poses. Surprisingly, one ligand binds to trypsin in protonated state and to thrombin in unprotonated state at P1 along with differences in the residual solvation pattern. While trypsin binding is mediated by an ordered water molecule, in thrombin, water is scattered over three hydration sites. Although having highly similar S1 pockets, our results suggest different electrostatic properties of Asp189 possibly contributing to the selectivity determinant. Thrombin binds a specific Na<sup>+</sup> ion next to Asp189, which is absent in trypsin. The electrostatic properties across the S1 pocket are further attenuated by charged Glu192 at the rim of S1 in thrombin, which is replaced by uncharged Gln192 in trypsin.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10038" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10038" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The success of a drug molecule in therapy strongly depends on its selectivity profile toward a given protein target. Experience shows that in some cases, high selectivity is the prerequisite for an effective therapy without unwanted side effects, whereas in other cases, a mixed profile toward several closely related isoforms of a target family is desired.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Nevertheless, as long as highly selective tool compounds are not available, this intriguing question about the best-suited profile can hardly be answered. Therefore, the issue of sufficient drug selectivity is increasingly moving into the focus of contemporary drug research.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a></div><div class="NLM_p">First of all, appropriate steric fit, optimal shape complementarity, and high-affinity binding are usually defined as prime strategies in the development of drug molecules with sufficient selectivity. Such structural and energetic aspects of mutual protein–ligand recognition have to be accomplished to guarantee potent binding. They are established by noncovalent interactions, e.g., by forming hydrogen bonds, ionic interactions, hydrophobic contacts, or van der Waals interactions.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The resulting affinity of the formed complex is then characterized by the dissociation constant <i>K</i><sub>d</sub>, which is related to thermodynamic parameters such as Gibb’s free energy Δ<i>G</i>°, binding enthalpy Δ<i>H</i>°, and entropy −<i>T</i>Δ<i>S</i>°.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Furthermore, before a complex assembles, both binding partners are solvated with a solvation shell of water molecules. Some of these are in loose contact with the protein or ligand, while others establish strong hydrogen bonds in one or the other states. In all cases, the inventory of available degrees of freedom of the binding partners involving water molecules will change.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Furthermore, water molecules can actively participate in binding and mediate contacts between protein and ligand. Changes in the solvation pattern during the entire association/dissociation process will also play an essential role in the protein–ligand recognition and have an impact on affinity and particularly on binding kinetics.<a onclick="showRef(event, 'ref8 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref10 ref11">(8,10,11)</a> Thus, the sole consideration of the formed end-state complexes is usually not sufficient to explain affinity.</div><div class="NLM_p">However, are these criteria to improve affinity toward a given isoform of a target family over another family member sufficient to achieve sufficient selectivity? Definitely, affinity-enhancing features are a necessary criterion, but are they also sufficient for target selectivity discrimination? Likely, more factors have to be regarded to finally achieve the desired selectivity profile. To equip drug molecules by rational concepts with an appropriate selectivity profile, we must first elucidate which features the proteins have at disposal to evolve their selectivity determinants.</div><div class="NLM_p last">In the present investigation, we selected the well-studied and highly similar serine proteases trypsin and thrombin for a case study. The former is an important enzyme in digestion, and the latter displays the key enzyme in blood clotting.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Particularly, their S1 specificity pockets, known to be most important for selectivity discrimination, are, at first sight, highly similar in architecture.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In the literature, the only obvious residue exchange of Ser190 by Ala in the pocket is made responsible for discrimination.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> At first glance, the Ser/Ala exchange sacrifices the option to form an additional hydrogen bond either directly to a bound substrate or ligand molecule or mediated via active-site water molecules. This might modulate the residual solvation structure in the S1 pocket. In the present study, we investigated the difference in binding of a congeneric series of <span class="smallcaps smallerCapital">d</span>-Phe-Pro-(amino)pyridine inhibitors with a slightly deviating basic character of their C-terminal P1 residue toward thrombin and trypsin by X-ray crystallography and by isothermal titration calorimetry (ITC). Both proteases recognize substrates with basic P1 residues via binding to Asp189 at the bottom of their S1 pocket. Surprisingly, thrombin is highly selective to recognize natural P1-Arg substrates, whereas trypsin possesses a much broader substrate specificity and cleaves unselectively after P1-Arg or P1-Lys residues.<a onclick="showRef(event, 'ref14 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref17 ref18">(14,17,18)</a> One obvious difference between these two basic amino acids is a significant difference in their p<i>K</i><sub>a</sub> values. Nevertheless, such values can be strongly modulated depending on the local environment in their binding pockets. For example, in a hydrophobic pocket, either acidic or basic ligand headgroups will less likely adopt a charged state, whereas a basic ligand group becomes easily positively charged by protonation, when bound to a negatively charged residue in a more polar binding site. The opposite is the case if the ligand bears an acidic group and the protein provides basic residues such as Arg or Lys in the binding pocket.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Consequently, the protein environment is designed in a way to induce significant p<i>K</i><sub>a</sub> shifts of titratable groups of the bound ligand. However, is there any difference caused by the Asp189 residue in trypsin and thrombin that might have a differing effect on the observed p<i>K</i><sub>a</sub> shift of titratable groups of a bound substrate or inhibitor molecule? Since ITC can determine overall changes in the protonation inventory of a binding reaction, we studied our compound series with a deviating basic character in parallel against trypsin and thrombin. Remarkably, a difference is found, indicating that one ligand of the series binds to trypsin in the protonated state at its basic anchor group, whereas the same ligand is accommodated in the S1 pocket of thrombin in the neutral state. This experimental observation suggests that the two Asp residues in both proteases show different electrostatic properties and supposedly contribute to features of the two proteases responsible to achieve selectivity discrimination in the otherwise sterically nearly identical S1 pockets.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63014" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63014" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">For our comparative study, we designed the following congeneric series of substrate analogue inhibitors <b>1</b>–<b>4</b> with structurally different P1 (amino)pyridine residues possessing a slightly deviating basic character (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In the case of <b>4</b>, we attached a <span class="smallcaps smallerCapital">d</span>-DiPhe substituent at P3 position to enhance potency, which made characterization by ITC easier (see below). The listed binding constants and p<i>K</i><sub>a</sub> values of the basic headgroups were determined by an enzyme kinetic assay and potentiometric p<i>K</i><sub>a</sub> measurements (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibition towards Thrombin and Trypsin by Compounds <b>1</b>–<b>4</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0006.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>i</sub> values were determined using an enzyme kinetic fluorescence assay and are given as arithmetic mean from three measurements (±SD).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">p<i>K</i><sub>a</sub> values were obtained by potentiometric titration.</p></div></div><div></div></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis</h3><div class="NLM_p">The general synthesis route for <span class="smallcaps smallerCapital">d</span>-Phe/<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-based inhibitors was already described previously by us and other groups.<a onclick="showRef(event, 'ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22">(19−22)</a> For the present study, we focused on the same <span class="smallcaps smallerCapital">d</span>-Phe/<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro scaffold by introducing different aminopyridines for inhibitors <b>1</b>–<b>3</b> and unsubstituted pyridine for compound <b>4</b> as P1 building blocks. In the following, we will describe the synthesis of precursor <b>11</b>, which was used for the amide coupling reaction with Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro-OH leading to the target inhibitor <b>1</b>. Inhibitors <b>2–4</b> were synthesized from commercially available precursors <b>12</b>–<b>14</b>. Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro-OH was afforded according to the protocol of Steinmetzer et al.,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and analogously, Boc-<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-OH was obtained by adapting a similar procedure outlined by Baum et al.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The P1 building block <b>11</b> is a known fragment that was synthesized according to a patented procedure published by Hoffmann-La Roche Inc. in 2003. Therein, the synthesis was performed using gaseous reagents which were reacted under a high pressure (up to 20 bar).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> To avoid such drastic conditions, the synthesis of <b>11</b> was developed under milder conditions also suited on a smaller scale (see <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Precursor <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) POBr<sub>3</sub>, 130–150 °C, 2 h then MeOH, 0 °C to rt, 15 h, 74%; (b) NaBH<sub>4</sub>, MeOH, 65 °C, 3 h, 91%; (c) PBr<sub>3</sub>, 1,4-dioxane, 40 °C, 3 h; (d) NaN<sub>3</sub>, dimethylformamide (DMF), rt, 24 h, 66% (over two steps); (e) PMe<sub>3</sub>, tetrahydrofuran (THF), 0 °C to rt, 3 h then Boc<sub>2</sub>O, NEt<sub>3</sub>, THF, 0 °C to rt, 3 h, 50%; (f) 25% (w/v) NH<sub>3(aq)</sub>, Cu<sub>2</sub>O, dimethylethylenediamine (DMEDA), K<sub>2</sub>CO<sub>3</sub>, ethylene glycol, 60 °C, 16 h, 63%; (g) 4 <span class="smallcaps smallerCapital">M</span> HCl in 1,4-dioxane, rt, 3 h, 95%.</p></p></figure><div class="NLM_p">Precursor <b>11</b> was obtained from citrazinic acid <b>5</b>, which was converted into <b>6</b> via bromination and esterification.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Methyl ester <b>6</b> was reduced to the corresponding alcohol <b>7</b>,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> followed by bromination and azidation to <b>8</b>. Compound <b>8</b> was used for a one-pot Staudinger reaction and Boc-protection to afford the Boc-amine <b>9</b>.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> In the next step, the two amino groups could be introduced via an Ullmann reaction to provide intermediate <b>10</b> according to a procedure described by Elmkaddem et al.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Cleavage of the protecting group was performed under acidic conditions to yield the P1 building block <b>11</b> as double HCl salt for the subsequent synthesis of inhibitor <b>1</b>. Due to the minor nucleophilicity of the aromatic amino groups in compound <b>11</b>, the primary aliphatic amino function reacted exclusively by forming the wanted amide.</div><div class="NLM_p">In <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, the amide coupling is described for each precursor <b>11</b>–<b>14</b> (<b>12</b> and <b>13</b> were commercially available from ChemPUR, <b>14</b> from Sigma-Aldrich) to the respective Boc-<span class="smallcaps smallerCapital">d</span>-Phe/<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro derivatives <b>15</b>–<b>18</b>. In the final step, intermediates <b>15</b>–<b>18</b> were deprotected using trifluoroacetic acid (TFA) to the desired inhibitors <b>1</b>–<b>4</b> as double TFA salts.</div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Inhibitors <b>1</b>–<b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro-OH (for <b>15</b>–<b>17</b>), Boc-<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-OH (for <b>18</b>), HOB<i>t</i>, EDC·HCl, DMF, DIPEA, 0 °C to rt, 18–24 h, 65–78%; (b) TFA, rt, 2 h, 78–92%. R<sup>3</sup> exists only for <b>3</b>, <b>13</b>, and <b>17</b>.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> X-ray Crystallography</h3><div class="NLM_p">The four <span class="smallcaps smallerCapital">d</span>-Phe/<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-based thrombin and trypsin inhibitors were analyzed by X-ray crystallography. Eight crystal structures were obtained with a resolution of 0.95–1.57 Å.</div><div class="NLM_p">The diaminopyridine structure of inhibitor <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) uses its polar head group to interact with Asp189 at the bottom of the S1 pocket of either thrombin or trypsin. The ortho amino groups make contacts with Gly219 and the water molecule W2, whereas the pyridine-<i>N</i> undergoes a water-mediated interaction with Asp189 via W1. In the case of thrombin, additional interactions are experienced by the “left” exocyclic amino group with the carboxylate group of Glu192. Furthermore, this residue adopts a back-folded conformation to participate via the interstitial water molecule W3 in the binding of the P3 group and via W4 in the formation of a contact to the P2 group. In trypsin, this contact is not found and residue 192 is here replaced by a corresponding Gln, which is disordered in this structure (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). Due to the exchange of Ala190 to Ser in the trypsin complex, the Ser190 side chain interacts directly with the water molecule W2 and the “right” amino group. This amino group performs additionally an interaction with the backbone carbonyl group of Ser190 and with the carboxylate group of Asp189. Structurally, the back-folding of Glu192 in thrombin to undergo a water-mediated contact with <b>1</b> seems to be replaced in trypsin by the unique Ser190Oγ H-bond to the ligand, which is further supported by a contact with the more firmly fixed water molecule W2 in the latter protease.</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding modes of inhibitors <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSX">6HSX</a>, A), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q">6T3Q</a>, B), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T4A">6T4A</a>, C), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT">6TDT</a>, D) in the S1 pocket of thrombin (left) compared to the structures of trypsin (right) by inhibitors <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0M">6T0M</a>, E), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0P">6T0P</a>, F), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5W">6T5W</a>, G), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SY3">6SY3</a>, H). The inhibitors are represented in light brown, whereas the protein residues are displayed in green. Heteroatoms are color-coded by atom type. Water molecules are colored in red, and favorable hydrogen bonds are shown as dashed lines. The distances in the complex of thrombin between the water molecules (W1, W5, W6) at the bottom of the S1 site are given in Å and are colored in red. In the trypsin complexes with <b>1</b> (E) and <b>3</b> (G), Gln192 is not visible due to the weakly defined <i>mF</i><sub>o</sub><i>–DF</i><sub>c</sub> electron density of this residue after refinement. It is likely scattered over multiple orientations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The P1 residue of inhibitor <b>2</b> is bound in two conformations (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B, 76% for I and 24% for II) in the S1 pocket of thrombin. Thus, it formally mimics the binding mode of <b>1</b>. In orientation I, the amino group is orientated toward Tyr228 by forming a hydrogen bond with the water molecule W2; in the other orientation, its amino group points in the direction of Gly219, while the pyridine-<i>N</i> interacts with Asp189 via the water molecule W1 in both conformations. A related binding mode can be assigned to the trypsin structure (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>F); however, here, ligand <b>2</b> adopts only one conformation by addressing W2 and Ser190Oγ on the right-hand side via its amino group, and it is involved in an additional direct H-bond to the carboxylate of Asp189. A very similar interaction pattern is found for <b>1</b> and <b>2</b> toward Tyr228, as observed in thrombin forming water-mediated interactions of the amino group or the pyridine-type <i>N</i> to the carboxylic function of Asp189 and the water molecule W2 hosted on top of the phenyl moiety of Tyr228. Also in the thrombin–inhibitor complex with <b>2</b>, the residue Glu192 is in a back-folded conformation to interact via the water molecule W3 to the P3 group and via W4 to the P2 group. In trypsin, such contacts are missing and the corresponding Gln192 does not adopt a folded geometry to allow contacting the P1 ligand head group. Possibly, the charged Glu192 in thrombin is responsible for the observed dual orientation of the head group, even so the conformer hydrogen-bonding this residue is the lower populated one. Analogously to the trypsin complex of <b>1</b>, Ser190 is able to form the same interactions with W2 and the amino group of the ligand, which is furthermore, in both proteases, in contact with Asp189 and the backbone carbonyl group of the residue in position 190.</div><div class="NLM_p">Inhibitor <b>3</b> is a regioisomeric aminopyridine derivative of compound <b>2</b>, in which the P1 moiety is para-substituted by the amino group while the pyridine-<i>N</i> is in meta position with respect to the <span class="smallcaps smallerCapital">d</span>-Phe-Pro scaffold. The thrombin complex (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C) indicates a direct hydrogen bond to Asp189 complemented by further interactions to the backbone carbonyl oxygens of Ala190 and Gly219, while the pyridine-<i>N</i> interacts with W2. Glu192 establishes the same interaction geometry as found for the complexes with inhibitor <b>1</b> and <b>2</b> by water-mediated contacts with the P2 and P3 moiety via W3 and W4. For the P1 group, a similar binding pose can also be observed in the trypsin complex (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G), in which either direct or water-mediated (W1) interactions are formed with both oxygens of the carboxylate of Asp189. Furthermore, in the trypsin complex with inhibitor <b>3</b>, the Ser190 side chain interacts directly with the pyridine-<i>N</i>. Due to this direct contact with Ser190, W2 is shifted to the rear of the pocket but still interacts with the pyridine-<i>N</i> and with the hydroxy group of Ser190. The side chain of Gln192 is disordered and not visible in the trypsin complex with ligand <b>3</b>. Surprisingly, there is a second conformation observed for the aromatic ring of the terminal phenylalanine attached at the P3 position, which, however, has no influence on the binding mode of the P1 group.</div><div class="NLM_p last">The refined interactions of inhibitor <b>4</b> are comparable in both cases and do not suggest any structural differences in the S1 pocket among the complexes with the two enzymes. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D shows the binding mode of inhibitor <b>4</b> in thrombin. This ligand is the analogue of a compound studied in a previous contribution by us that possesses a <span class="smallcaps smallerCapital">d</span>-diphenylalanine as P3 group, but with respect to P1 and P2, it describes the same binding mode as the <span class="smallcaps smallerCapital">d</span>-monophenylalanine derivative.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> As reported earlier, the unsubstituted pyridine ring of the P1 head group forms an indirect interaction with Asp189 mediated by two water molecules (W1 and W5). They are found at sites which are not fully occupied (each with 50% occupancy) and show a close mutual distance of 2.0 Å along with reasonable <i>B</i>-factors (23–24 Å<sup>2</sup>). An additional water molecule (W6) is found at a site which was set to 50% occupancy with a distance of 2.1 Å to W1 and an acceptable <i>B</i>-factor (23 Å<sup>2</sup>). A similar disordered water pattern between the pyridine-type nitrogen and Asp189 has been described in our previously determined structure of the <span class="smallcaps smallerCapital">d</span>-Phe-Pro analogue with thrombin.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Remarkably, we observed for <b>4</b> a very similar binding mode to that for the aminopyridine inhibitors <b>1</b> and <b>2</b>; however, in all analogous trypsin complexes, pyridine-<i>N</i> interacts with Asp189 via a single and fully occupied water molecule found in one distinct spatial position. Residue Ser190 indicates two conformations which are possibly only resolved in this complex due to the achieved high resolution of 0.95 Å. Conformation I (occupancy of 79%) orientates to water molecule W2, while the other (II, occupancy of 21%) is in contact with the side chain of Tyr228. We can therefore summarize that the major difference between the thrombin and trypsin complexes with <b>4</b> are the following: In thrombin, the contacts between <b>4</b> and Asp189 are formed via a network of hydrogen bonds; however, refinement suggests that water is here distributed over several hydration sites, whereas in trypsin, the carboxylate of Asp189 reinforces an ordered contact mediated by a single water molecule.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> ITC Data Collection</h3><div class="NLM_p">For all inhibitors of the series, we can assume that the polar headgroups become at least partly protonated at the pyridine-<i>N</i> upon binding to the S1 pocket. This is in agreement with our previous studies, which suggested significant p<i>K</i><sub>a</sub> shifts of the basic anchor groups once bound next to the charged protein residues.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Accordingly, we studied the entire series by ITC titrations in different buffers to record putatively overlaid changes in protonation states. The values obtained for the different buffers are listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Thermodynamic Data of Δ<i>H</i>° from Direct ITC Titrations in Thrombin and Trypsin</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char="±">thrombin</th><th class="rowsep1 colsep0" colspan="3" align="center">trypsin</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Inhibitor</th><th class="colsep0 rowsep0" align="center" char="±">Δ<i>H</i>° (kJ mol<sup>–1</sup>) Phosphate</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i>° (kJ mol<sup>–1</sup>) HEPES</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i>° (kJ mol<sup>–1</sup>) Tricine</th><th class="colsep0 rowsep0" align="center" char="±">Δ<i>H</i>° (kJ mol<sup>–1</sup>) Tris</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i>° (kJ mol<sup>–1</sup>) HEPES</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i>° (kJ mol<sup>–1</sup>) Tricine</th><th class="colsep0 rowsep0" align="center">Δ<i>H</i>° (kJ mol<sup>–1</sup>) Tris</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">–107.0 ± 1.0</td><td class="colsep0 rowsep0" align="left">–87.5 ± 0.7</td><td class="colsep0 rowsep0" align="left">–80.6 ± 0.8</td><td class="colsep0 rowsep0" align="char" char="±">–64.7 ± 1.1</td><td class="colsep0 rowsep0" align="left">–39.1 ± 1.7</td><td class="colsep0 rowsep0" align="left">–30.5 ± 0.3</td><td class="colsep0 rowsep0" align="left">–17.7 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char="±">–83.7 ± 0.5</td><td class="colsep0 rowsep0" align="left">–64.9 ± 0.1</td><td class="colsep0 rowsep0" align="left">–54.5 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">–42.3 ± 0.2</td><td class="colsep0 rowsep0" align="left">–36.9 ± 0.9</td><td class="colsep0 rowsep0" align="left">–27.2 ± 1.8</td><td class="colsep0 rowsep0" align="left">–13.0 ± 1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char="±">–97.6 ± 0.3</td><td class="colsep0 rowsep0" align="left">–76.7 ± 0.4</td><td class="colsep0 rowsep0" align="left">–32.0 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">–22.9 ± 0.9</td><td class="colsep0 rowsep0" align="left">–36.8 ± 0.5</td><td class="colsep0 rowsep0" align="left">–29.5 ± 0.2</td><td class="colsep0 rowsep0" align="left">–16.4 ± 1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char="±">–30.8 ± 1.1</td><td class="colsep0 rowsep0" align="left">–26.8 ± 1.3</td><td class="colsep0 rowsep0" align="left">–24.8 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">–24.3 ± 0.9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="char" char="±">–23.2 ± 1.2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">–27.1 ± 0.6</td><td class="colsep0 rowsep0" align="left">–35.7 ± 1.9</td><td class="colsep0 rowsep0" align="left">–18.5 ± 0.3</td><td class="colsep0 rowsep0" align="left">–1.0 ± 1.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Standard deviations were determined from triplicate measurements. Δ<i>H</i>° is given as the observed enthalpy by a control experiment using <b>19</b> as a displacement ligand (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_001.pdf" class="ext-link">Table S2</a>), which confirms the buffer independence of inhibitor <b>4</b> in thrombin. For <b>4</b>, both direct titration and displacement titration (<b>4</b><sup>a</sup>) are listed. The displacement titration was performed in phosphate and tris(hydroxymethyl)aminomethane (tris) buffer since their ionization enthalpies have the largest difference of the chosen buffers.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> Due to the low binding affinity of <b>4</b> in trypsin, the values of <b>4</b> were obtained by displacement titration.</p></div></div></div><div class="NLM_p">As a pronounced buffer dependence has been observed, we can definitely conclude that changes in protonation state are given for the individual ligands (see <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and the discussion below). To relate the observed quantities to well-defined reference values in aqueous solution, we determined the p<i>K</i><sub>a</sub> properties of our four ligands experimentally in aqueous solution (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). As suggested by the crystal structures and the order/disorder phenomena observed next to Asp189, the unsubstituted derivative <b>4</b> indicates different binding properties in thrombin and trypsin. We therefore investigated particularly this ligand in more detail by ITC experiments (see below).</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Diagrams (A: thrombin, B: trypsin) displaying the experimentally determined enthalpies Δ<i>H</i>°<sub>obs</sub> of the inhibitors <b>1</b>–<b>4</b> under different buffer conditions. They are plotted against the heat of ionization Δ<i>H</i>°<sub>ion</sub> (phosphate:<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> 3.60 kJ mol<sup>–1</sup>, 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES):<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> 21.01 kJ mol<sup>–1</sup>, tricine:<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> 31.97 kJ mol<sup>–1</sup>, Tris:<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> 47.53 kJ mol<sup>–1</sup>) of the respective buffers. The slopes of the linear fit give the molar amounts of protons that are transferred between buffer, protein, and ligand upon binding (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The intercept of these plots is reported as “buffer-corrected” value (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Protonation Effects of Inhibitors <b>1</b>–<b>4</b> in Thrombin and Trypsin</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitor<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>n</i><sub>H</sub><sup>+</sup> (thrombin)</th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>n</i><sub>H</sub><sup>+</sup> (trypsin)</th><th class="colsep0 rowsep0" align="center" char=".">p<i>K</i><sub>a</sub>1<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">p<i>K</i><sub>a</sub>2<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">0.94</td><td class="colsep0 rowsep0" align="char" char=".">0.81</td><td class="colsep0 rowsep0" align="char" char=".">6.4</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">0.95</td><td class="colsep0 rowsep0" align="char" char=".">0.90</td><td class="colsep0 rowsep0" align="char" char=".">6.7</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">0.87</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td><td class="colsep0 rowsep0" align="char" char=".">6.3</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">0.15</td><td class="colsep0 rowsep0" align="char" char=".">1.30</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The protonation states of all four inhibitors were determined by direct titrations except for <b>4</b> in trypsin. The latter measurement was performed by a displacement titration.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">The p<i>K</i><sub>a</sub> values were determined by potentiometric titration and describe the acidity of the P1 head group (p<i>K</i><sub>a</sub>1) and the N-terminal amino group (p<i>K</i><sub>a</sub>2) in the protonated state.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Protonation Effects Recorded by ITC</h3><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> lists the overall changes of the protonation inventory of the respective inhibitors, which describes the exchange of protons (Δ<i>n</i><sub>H</sub><sup>+</sup>) during the binding reaction between ligand, protein, and buffer. The measured heat signals were plotted against the heat of ionization of the respective buffer, and a linear correlation is indicated. The slope of the linear fit (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) gives the molar amount of protons transferred during the entire binding process. It can be concluded that the aminopyridine derivatives <b>1</b>, <b>2</b>, and <b>3</b> receive between 0.78 and 0.94 mol of protons upon accommodation to the S1 pocket of both proteins.</div><div class="NLM_p">We studied the protonation inventory of this type of inhibitors already in detail in previous contributions.<a onclick="showRef(event, 'ref19 ref21 ref33 ref34 ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref19 ref21 ref33 ref34 ref35 ref36 ref37 ref38">(19,21,33−38)</a> The investigated <span class="smallcaps smallerCapital">d</span>-Phe-Pro derivatives contained a P1 benzamidino group, which is permanently charged both in the buffer and in the protein-bound state. Additionally, the <i>N</i>-terminal group is partially charged in the buffer and reveals full protonation in the complex. Furthermore, in all complexes, His57 releases 0.50–0.60 mol of protons upon ligand accommodation.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Thus, overall, the proton release from His57 and the entrapment by the ligand’s terminal P3 amino group is virtually compensated, leaving no net protonation effect for this type of ligands.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> We assume that the same inventory applies for the latter groups in <b>1</b>–<b>4</b>.</div><div class="NLM_p">However, as a difference to the permanently charged P1 benzamidino derivatives, we studied here the weakly basic (amino)pyridine derivatives <b>1</b>–<b>4</b>, and their P1 moieties possess p<i>K</i><sub>a</sub> values of 5.0–6.7 in aqueous solution. According to the Henderson–Hasselbalch equation, all four inhibitors <b>1</b>–<b>4</b> will be largely deprotonated under the applied buffer conditions at pH 7.8 prior to protein binding (maximally ca. 5–10% may be protonated in the buffer). Based on the results listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, we assume that the ligands experience a p<i>K</i><sub>a</sub> shift upon binding, reinforced by the spatial neighborhood to the negatively charged Asp189 residue. This leads to full protonation of the pyridine-<i>N</i> in <b>1</b>–<b>3</b> for both trypsin and thrombin. Obviously, ligand <b>4</b> behaves differently and shows protonation only in trypsin (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Schiebel et al. could prove full protonation of aniline in trypsin, despite its rather low p<i>K</i><sub>a</sub> of 4.6, by a neutron diffraction study.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Obviously, Asp189 has such a strong influence on the protonation state of the bound ligand and probably polarizes its neighborhood so strongly that a p<i>K</i><sub>a</sub> shift of several orders of magnitude is induced, triggering full protonation of the ligand, even in the case of a weakly basic aniline or pyridine residue.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> For this reason, we conclude that the inhibitors <b>1</b>–<b>3</b> entrap 0.78–0.95 mol of protons upon binding in both serine proteases besides the mutual compensation effect of the terminal amino function (p<i>K</i><sub>a</sub> = 7.5 for <b>1</b>–<b>3</b>,) which also becomes fully protonated upon binding along with the release of 0.5–0.6 mol of protons from His57.</div><div class="NLM_p">Surprisingly, binding of inhibitor <b>4</b> reveals a deviating protonation inventory for both enzymes. Even though both complexes show almost identical binding poses in the S1 pocket, the unsubstituted pyridine derivative <b>4</b> binds, as expected, in the protonated state at its pyridine-<i>N</i> to trypsin. Since the pyridine-<i>N</i> in <b>4</b> (p<i>K</i><sub>a</sub> = 5.0) is less basic than the headgroups of <b>1</b>–<b>3</b> (p<i>K</i><sub>a</sub> = 6.3–6.7), a slightly larger molar amount of protons must be entrapped. As mentioned above, we decided to study the ligand with a second phenyl group at P3 to increase the binding affinity toward the enzymes. By this, also the p<i>K</i><sub>a</sub> value of the <i>N</i>-terminal amino group is slightly shifted compared to <b>1</b>–<b>3</b>. This may also give rise to a slightly altered protonation inventory. Finally, due to lower affinity, the protonation effect of <b>4</b> during binding to trypsin could not be determined by direct ITC titration. Instead, we had to apply displacement titrations for <b>4</b> in three different buffers to trace the changes in protonation state. Possibly, the displacement titrations increase the estimated inaccuracy of this experiment due to error propagation. Nevertheless, the recorded value of 1.30 mol of protons clearly indicates full protonation of the pyridine-<i>N</i> head group of <b>4</b> in trypsin.</div><div class="NLM_p last">Unexpectedly, <b>4</b> shows a different protonation effect upon binding to thrombin, as no entrapment of a proton was registered at the pyridine-<i>N</i> head group. In a former study by Biela et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> we could investigate the analogous <span class="smallcaps smallerCapital">d</span>-Phe-Pro ligand, and it showed no obvious structural influence on the binding pose in the S1 pocket; thus, we believe that the binding mode and protonation inventory of the <span class="smallcaps smallerCapital">d</span>-Phe-Pro analogue and <b>4</b> can be directly compared to the aminopyridine derivatives <b>1</b>–<b>3</b>. Overall, only 0.15 mol of protons are transferred upon binding of <b>4</b> to thrombin (note that a minor difference can result from the slightly shifted p<i>K</i><sub>a</sub> value observed for the <span class="smallcaps smallerCapital">d</span>-DiPhe derivative), an amount which, as it falls close to the experimental accuracy, speaks for no protonation of <b>4</b> at the pyridine-<i>N</i> head group in the complex with thrombin. This observation agrees with the data of Biela et al., who also did not observe any protonation of the P1 pyridine head group at their <span class="smallcaps smallerCapital">d</span>-Phe-Pro analogue. However, they had to study the protonation inventory by displacement titration. Similarly to a protocol by Biela et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> we titrated the strong binding ligand <b>19</b> to displace <b>4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_001.pdf" class="ext-link">Table S2</a>) as a control experiment, and also by this displacement protocol, no change in the protonation state could be observed (Δ<i>n</i><sub>H</sub><sup>+</sup> = −0.09 mol).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Comparison of the Thermodynamic Signatures</h3><div class="NLM_p">In the following, the buffer-corrected values, which result from a linear extrapolation to hypothetical buffer conditions with zero heat of ionization (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), will be discussed. Based on the ITC data, we computed <i>K</i><sub>d</sub> values and they match within the 3× standard deviation with the <i>K</i><sub>i</sub> values from our enzyme kinetic assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Apart from the fact that very different signals are evaluated to obtain the inhibitory potency, the meanings of <i>K</i><sub>d</sub> as ITC dissociation constant and <i>K</i><sub>i</sub> as inhibitory constant from an enzyme assay are not identical.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Thermodynamic signatures of the inhibitors <b>1</b>–<b>4</b> in complex with thrombin (A) and trypsin (B). The results are represented as buffer-corrected values listed in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. They were determined by direct titration in thrombin. Notably, due to the relatively weak affinity of the studied ligands toward trypsin (<i>K</i><sub>i</sub> values between 40 and 500 μM), all ITC measurements with trypsin have been performed as displacement titrations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Thermodynamic Data of Δ<i>H</i>°, Δ<i>G</i>°<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a>, and −<i>T</i>Δ<i>S</i>°<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> from Direct ITC Titrations in Thrombin and Trypsin</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">thrombin</th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">trypsin</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">inhibitor</th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>H</i>° /kJ mol<sup>–1</sup></th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>G</i>°/kJ mol<sup>–1</sup></th><th class="colsep0 rowsep0" align="center" char=".">–<i>T</i>Δ<i>S</i>° /kJ mol<sup>–1</sup></th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>d</sub> /nM</th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>H</i>° /kJ mol<sup>–1</sup></th><th class="colsep0 rowsep0" align="center" char=".">Δ<i>G</i>° /kJ mol<sup>–1</sup></th><th class="colsep0 rowsep0" align="center" char=".">–<i>T</i>Δ<i>S</i>° /kJ mol<sup>–1</sup></th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>d</sub> /μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">–109.4</td><td class="colsep0 rowsep0" align="char" char=".">–39.8</td><td class="colsep0 rowsep0" align="char" char=".">69.6</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">–56.2</td><td class="colsep0 rowsep0" align="char" char=".">–23.8</td><td class="colsep0 rowsep0" align="char" char=".">32.4</td><td class="colsep0 rowsep0" align="left">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">–86.0</td><td class="colsep0 rowsep0" align="char" char=".">–34.4</td><td class="colsep0 rowsep0" align="char" char=".">51.6</td><td class="colsep0 rowsep0" align="char" char=".">916</td><td class="colsep0 rowsep0" align="char" char=".">–56.1</td><td class="colsep0 rowsep0" align="char" char=".">–25.3</td><td class="colsep0 rowsep0" align="char" char=".">30.8</td><td class="colsep0 rowsep0" align="left">38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">–98.2</td><td class="colsep0 rowsep0" align="char" char=".">–36.5</td><td class="colsep0 rowsep0" align="char" char=".">61.7</td><td class="colsep0 rowsep0" align="char" char=".">400</td><td class="colsep0 rowsep0" align="char" char=".">–53.6</td><td class="colsep0 rowsep0" align="char" char=".">–24.9</td><td class="colsep0 rowsep0" align="char" char=".">28.7</td><td class="colsep0 rowsep0" align="left">43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">–30.6</td><td class="colsep0 rowsep0" align="char" char=".">–29.4</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">3681</td><td class="colsep0 rowsep0" align="char" char=".">–61.9<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">–17.7<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char=".">44.3<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">757<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">All results are presented as buffer-corrected values. Δ<i>G</i>° is given as the mean over the results by titrations in different buffers (phosphate, HEPES, tricine and Tris; in the case of trypsin without phosphate).</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">–<i>T</i>Δ<i>S</i>° has been calculated as the difference between Δ<i>G</i>° and the buffer-corrected Δ<i>H</i>° values.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">These values were obtained by displacement titrations.</p></div></div></div><div class="NLM_p">The three aminopyridine inhibitors <b>1</b>–<b>3</b> show similar signatures in their thermodynamic profiles toward thrombin. Gibb’s free energy Δ<i>G</i>° of all ligands falls in the range of −34 to −40 kJ mol<sup>–1</sup>, resulting from an enthalpically more favored binding profile. This can be explained by the charge-assisted interactions to Asp189, sometimes mediated by interstitial water molecules. Further interactions are formed by the amino groups of <b>1</b> with Gly219 and Glu192. Furthermore, contacts with the water molecules W1 and W2 are found. In summary, these interactions give rise to a very strong enthalpic signature (Δ<i>H</i>° = −109.4 kJ mol<sup>–1</sup>) of <b>1</b>, closely followed by the other aminopyridine derivatives (Δ<i>H</i>° = −86.0 kJ mol<sup>–1</sup> for <b>2</b> and Δ<i>H</i>° = −98.2 kJ mol<sup>–1</sup> for <b>3</b>), which exhibit only a single amino group for interactions. The ligands are the most enthalpic binders found so far with this scaffold.<a onclick="showRef(event, 'ref21 ref35 ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref21 ref35 ref36 ref37 ref38">(21,35−38)</a> The stronger enthalpic signature compared to the analogous benzamidine derivative results from the enthalpically less expensive desolvation of the P1 headgroups in <b>1</b>–<b>3</b>, which are uncharged in solution prior to protein binding and only become charged in the binding pocket. This is definitely a desirable scenario, as enthalpically binding ligands have been favored as development candidates<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and the mere change of the protonation state at the protein binding site helps to design molecules with improved bioavailability as charged species exhibit unfavorable pharmacokinetic (PK) properties. The large enthalpic signature is compensated by an unfavorable entropic contribution. Ligands <b>1</b> and <b>3</b> are separated by a −<i>T</i>ΔΔ<i>S</i>°<sub><b>3→1</b></sub> difference of −7.9 kJ mol<sup>–1</sup>, whereas <b>2</b> differs even more in entropy (−<i>T</i>ΔΔ<i>S</i>°<sub><b>2→1</b></sub> = −18.0 kJ mol<sup>–1</sup> and −<i>T</i>ΔΔ<i>S</i>°<sub><b>2→3</b></sub> = −10.1 kJ mol<sup>–1</sup>). Its slightly increased entropic advantage over <b>1</b> and <b>3</b> results from the fact that <b>2</b> is able to bind with two alternative conformations (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B), which is, with respect to the inventory of degrees of freedom, entropically beneficial.</div><div class="NLM_p">The unsubstituted pyridine derivative <b>4</b> shows weaker binding to both proteins, therefore, to enable direct ITC titrations at least with thrombin, we selected the analogous diphenyl derivative with has a factor of 10 enhanced affinity (Baum et al.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a>). Overall, <b>4</b> binds with reduced enthalpic signature, likely as it does not exhibit a P1 group that establishes a charge-assisted contact with Asp189. Instead, the carboxylate group is connected via a network of disordered water molecules, which likely also contributes to an entropically more favored binding signature (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A).</div><div class="NLM_p last">If we compare the thrombin results to those for trypsin of the same ligands, it is remarkable to see by how much the binding affinity decreases for <b>1</b> (ΔΔ<i>G</i>°<sub>thrombin→trypsin</sub> = −16.0 kJ mol<sup>–1</sup>), <b>2</b> (ΔΔ<i>G</i>°<sub>thrombin→trypsin</sub> = −9.1 kJ mol<sup>–1</sup>), or <b>3</b> (ΔΔ<i>G</i>°<sub>thrombin→trypsin</sub> = −11.6 kJ mol<sup>–1</sup>) although very similar interactions are established in trypsin’s S1 pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E–H). The reduced binding affinities toward trypsin in the two-digit micromolar range made displacement titrations necessary in these cases (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> Since the S1 pockets are very similar for both proteins, the S2 and S3/4 pockets play an important role in gaining affinity. Thrombin is able to form strong hydrophobic contacts with the P2 proline moiety via its 60’s-loop, while the P3 group is well conserved in the S3/4 pocket also by hydrophobic interactions.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Additionally, the thrombin complexes show that Glu192 adopts a back-folded geometry stabilizing the amide groups of the <span class="smallcaps smallerCapital">d</span>-Phe-Pro scaffold and establishing polar contacts to the P1 head group (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A–D). In trypsin, Glu192 is replaced by Gln, which cannot form a charge-assisted contact. This all contributes to an enthalpic signature, which is not similarly possible in the case of trypsin. Consequently, affinity drops in the trypsin case compared to thrombin. Due to the stronger directional interactions with thrombin, the ligands most likely also entail a loss of degrees of freedom as they become more firmly fixed compared to the trypsin complexes. This explains to some extent the entropically more beneficial binding signature of <b>1</b>–<b>3</b> to trypsin. Facing the trypsin and thrombin data for ligand <b>4</b> shows different behavior, as it binds to the former protease with a charged head group, thus improving the enthalpic binding contribution. Furthermore, since a network of disordered water found in thrombin to mediate the contact to Asp189 becomes ordered in trypsin, the binding of <b>4</b> to trypsin becomes entropically less favored. Overall however, as the affinity-enhancing effects contributed by the interactions experienced in the S2 and S3/4 pockets of both proteases are quite different, a mutual comparison of the binding and thus the selectivity features of the four ligands toward thrombin and trypsin is difficult. Nevertheless, the main purpose of this study was to trace the selectivity determinants provided by trypsin and thrombin in their S1 pockets.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Different Charge Density of Asp189 Influences the Protonation Effect</h3><div class="NLM_p">We already observed in our previous neutron diffraction study on trypsin that Asp189 can induce strong p<i>K</i><sub>a</sub> shifts of the adjacent ligand functional groups once bound to the S1 pocket.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Compounds <b>1</b>–<b>3</b> experience a change in protonation either in thrombin or in trypsin, leading to a charge-assisted interaction partly mediated via W1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A–C, and E–G). In addition, the change of the protonation state will be stabilized by mesomeric effects assisted by the adjacent amino groups to delocalize the positive charge. The ITC results recorded for <b>4</b> show that this ligand binds unprotonated to thrombin but protonated to trypsin. This difference is also reflected by the local network of disordered vs. ordered water molecules formed between the P1 head group and Asp189. We believe that the local environment of Asp189 and the involved charge distribution provide an explanation for these differences. In total, three differences are observed between the S1 pockets of trypsin and thrombin. First of all, a Ser190Ala replacement is given, which, however, has no obvious impact on the binding poses in the current cases. Second, the Gln192Glu replacement adds another charged residue in the case of thrombin, which fairly adaptively adopts in the studied complexes a back-folded geometry bringing its likely charged carboxylate group in the neighborhood of the P1 head group. Finally, the main difference between the two serine proteases is the presence of a sodium ion in thrombin, which is about 5.3 Å remote from Asp189. It shows octahedral coordination geometry formed by four water molecules and the backbone of carbonyl groups of Arg221A and Lys224 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). In general, Na<sup>+</sup> is responsible for regulating the protein activity<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> as the Na<sup>+</sup>-bound form cleaves procoagulant substrates at faster rates, whereas the Na<sup>+</sup>-free form is less efficient and called the “slow form”.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> With respect to our studied ligand series, we assume that the presence of this cation has an electrostatic influence on the charge distribution on Asp189 and might attenuate its negative charge. Due to a reduced charge localization on the carboxylate oxygens of this residue, the interaction with the adjacent ligand functional groups is weakened. Supposedly, for the less basic pyridine moiety of <b>4</b>, the polarization effect is no longer sufficient to reinforce protonation of the pyridine-<i>N</i> in the case of thrombin. In the trypsin complex, where no comparable Na<sup>+</sup> is located next to Asp189, the charge attenuation of Asp189 is not in operation so that the highly negatively charged carboxylic group exerts stronger polarization effects, apparently still sufficient to induce a protonation of the ligand’s head group. Simultaneously, it establishes a well-ordered water-mediated charge–charge interaction with the ligand. This hypothesis has already been proposed in our previous study, where we speculated about a different charge distribution on Asp189 between thrombin and trypsin.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The hypothesis was based on the deviating geometries found for <i>N</i>-amidinopiperidine in the S1 pocket of both proteases. It is found with planar geometry at the <i>ipso</i> nitrogen in thrombin, whereas planar and pyramidal hybridization states were observed side by side in trypsin. Also here, this effect was related to a putative charge attenuation of Asp189 in thrombin.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Transferred to the present study, in thrombin, the adjacent pyridine nitrogen of <b>4</b> remains uncharged and the fixation of the interstitial solvation shell is relaxed, resulting in a network with water molecules scattered over several hydration sites. It remains speculative whether the proposed attenuated charge on Asp189 in thrombin takes also influence on the well-known substrate discrimination, as thrombin only recognizes peptide chains with the more basic P1-Arg, whereas trypsin accepts in addition to P1-Arg also the less basic P1-Lys at this position. Possibly, the charge attenuation on the carboxylate oxygens of Asp189 in thrombin results in weaker charge-assisted hydrogen bonds to the terminal amino anchor of a Lys compared to the slightly longer and deeper penetrating Arg head group, which usually establishes two parallel H-bonds using its guanidinium nitrogens.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode of ligand <b>4</b> in the S1 pocket of thrombin (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT">6TDT</a>) in the presence of Na<sup>+</sup> (purple ball) closely adjacent to Asp189. The octahedral coordination of Na<sup>+</sup> is represented as the dashed lines in purple. The ligand is shown in light brown, whereas the protein residues are displayed in green. Heteroatoms are color-coded by atom type. Water molecules are displayed as red spheres, and favorable hydrogen bonds are indicated as dashed lines with the corresponding distances in Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48777" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48777" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this comparative study between thrombin and trypsin, selectivity discriminating features, particularly in the S1 pocket between both proteins were investigated by enzyme kinetics, thermodynamics, and crystallography using a small series of <span class="smallcaps smallerCapital">d</span>-Phe/<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro derivatives with deviating (amino)pyridine headgroups. Thrombin and trypsin are structurally related proteins and possess closely similar S1 pockets, which are responsible for the substrate specificity.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Crystal structure analyses show that all four ligands adopt nearly identical binding modes in both proteins. Differences are presented by the enzyme kinetic and thermodynamic data. While the inhibitors are mainly nanomolar binders in thrombin, the binding affinity is reduced by at least 2 orders of magnitudes in trypsin (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). We relate these differences to contacts formed in the S2 and S3/4 pockets of the proteases. The reduced binding affinity toward trypsin is also notable in the thermodynamic profiles. ITC experiments show that the ligands have a much stronger enthalpic contribution to binding in the case of thrombin than trypsin. As a matter of fact, this effect is entropically largely compensated so that the free Gibb’s energy differs by only about 2 orders of magnitude in <i>K</i><sub>D</sub>. By investigating putatively overlaid protonation effects, it can be seen that the slightly more basic aminopyridine derivatives (p<i>K</i><sub>a</sub> = 6.3–6.7) all become fully protonated in both enzymes at the pyridine-<i>N</i>, whereas protonation in the buffered solution at pH 7.8 prior to binding is hardly experienced. In addition, His57 of the catalytic triad releases about 0.5–0.6 mol of protons, which, in turn, is compensated by the terminal amino group. It virtually entraps the same amount of protons upon protein binding. Thus, the proton exchange is compensated and the overall observed protonation effect is generated nearly exclusively by the protonation of the P1 moiety. In nearly all examples, the weakly basic P1 headgroups become fully protonated in the S1 pocket as they possess p<i>K</i><sub>a</sub> values of 6.3–6.7. The observed p<i>K</i><sub>a</sub> shift is determined by the polarization of the negatively charged Asp189, which triggers a p<i>K</i><sub>a</sub> shift in the S1 pocket and thus induces protonation at the pyridine-<i>N</i>. An astonishing difference is found for the complexes of <b>4</b> with both proteases (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D,H). The basicity of the pyridine derivative <b>4</b> is with p<i>K</i><sub>a</sub> = 5.0 about 1.5 units lower than those of <b>1</b>–<b>3</b>. Surprisingly, <b>4</b> remains unprotonated in the case of thrombin, but it entraps a proton upon binding to trypsin. We suggest to explain this difference by the presence of a sodium ion in thrombin about 5.3 Å remote from Asp189 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Likely, this cation has an electrostatic influence on the charge distribution on Asp189. We suppose that Na<sup>+</sup> attenuates the negative charge on Asp189 and thus modulates the binding properties in the S1 pocket. As a consequence, the carboxylate group cannot sufficiently polarize the pyridine-<i>N</i> of <b>4</b> to induce protonation. Because of the attenuated negative charge on the carboxylate oxygens, the water-mediated charge–charge interactions between the P1 head group of <b>4</b> and Asp189 is insufficient to induce protonation at the pyridine-<i>N</i>. Contrarily, in trypsin, where Na<sup>+</sup> is absent, the polarizing effect is strong enough to succeed in full protonation of the pyridine-type head group.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Enzyme Kinetic Fluorescence Assay</h3><div class="NLM_p">The inhibition of human α-thrombin (Beriplast, CSL Behring, Marburg, Germany) was investigated by a fluorescence assay using Tos-Gly-Pro-Arg-AMC·TFA as fluorogenic substrate in black 96-well plates with a Fluoroskan Ascent fluorometer (Thermo Fisher Scientific, Vantaa, Finland; λ<sub>ex</sub> = 380 nm, λ<sub>ex</sub> = 460 nm) according to Stürzebecher et al.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The measurements were performed at 25 °C in a buffer composed of 50 mM Tris, 154 mM NaCl, and 0.01% (w/v) PEG8000 at pH 7.8. The final thrombin concentration was 80 pM, while the inhibitors were dissolved in dimethyl sulfoxide (DMSO) and diluted in buffer to reach a final DMSO concentration below 1% (v/v) in the assay. For the determination of the inhibition constant <i>K<sub>i</sub></i>, substrate concentrations of 2.5, 5.0, and 10 μM were used at four different inhibitor concentrations together with control measurement in the absence of inhibitor in a total volume of 140 μL. Analogically, the inhibition of bovine trypsin (Sigma-Aldrich, product number T8003) was performed with Mes-<span class="smallcaps smallerCapital">d</span>-Arg-Gly-Arg-AMC·2 TFA as fluorogenic substrate at a protein concentration of 65 pM. Here, substrate concentrations of 6.3, 12.5, and 25 μM were used. The buffer contained 43 mM Tris, 132 mM NaCl, and 0.0086% (w/v) PEG8000 at pH 7.8.</div><div class="NLM_p last">The velocities were obtained from the slopes of the progress curves during a time frame of 5 min, and all <i>K<sub>i</sub></i> values were determined by a nonlinear fit to the Michaelis–Menten equation. All measurements were performed at least in triplicate.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthesis</h3><div class="NLM_p"><sup>1</sup>H and <sup>13</sup>C NMR spectra were measured on a JEOL ECX-400 or JEOL ECA-500 instrument. Chemical shifts are reported in ppm using residual peaks for the deuterated solvents as internal standard:<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> 7.26 ppm (<sup>1</sup>H NMR), 77.2 ppm (<sup>13</sup>C NMR), CDCl<sub>3;</sub> 2.50 ppm (<sup>1</sup>H NMR), 39.5 ppm (<sup>13</sup>C NMR), DMSO-<i>d</i><sub>6</sub>. The multiplicity of the signals is described with the following abbreviations: s = singlet, d = doublet, t = triplet, q = quartet, dd = doublet of doublet, m = multiplet, br = broad signal. The coupling constants <i>J</i> are given in Hz. The NMR spectra of the inhibitors <b>1</b>–<b>4</b> represent the signals of the main conformer. MS spectra were measured on a Q-Trap 2000 system with an electrospray interface (ESI). Flash column chromatography was performed on silica gel (0.04–0.063 mm, Macherey-Nagel, Düren, Germany) and was monitored by thin-layer chromatography (TLC) on aluminum sheets (silica gel 60 F254, Merck, Darmstadt, Germany) with UV light at 254 or 366 nm. Preparative high-performance liquid chromatography (HPLC) was performed with a Varian HPLC system gradient system (reverse-phase column: Nucleodur C<sub>18</sub>, 5 μm, 100 Å, 32 mm × 250 mm, Macherey-Nagel, Düren, Germany). All solvents were of HPLC grade, and in a gradient run, the percentage of acetonitrile was increased by 0.5% solvent min<sup>–1</sup> at a flow rate of 20 mL min<sup>–1</sup>. The purity of all inhibitors used for this study was at least 95% as determined by analytical HPLC with a Shimadzu LC-10A system (reverse-phase column: Nucleodur C<sub>18</sub>, 5 μm, 100 Å, 4.6 mm × 250 mm, Macherey-Nagel, Düren, Germany). All solvents were of HPLC grade, and in a gradient run, the percentage of acetonitrile was increased by 1% solvent min<sup>–1</sup> at a flow rate of 1 mL min<sup>–1</sup>. The detection was recorded at a wavelength of 220 nm. The inhibitors <b>1</b>–<b>4</b> were obtained as double TFA salts after lyophilization.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Methyl 2,6-dibromoisonicotinate (<b>6</b>)</h4><div class="NLM_p last">Citrazinic acid <b>5</b> (4.50 g, 29.0 mmol) and POBr<sub>3</sub> (24.9 g, 87.0 mmol) were heated to 130 °C for 0.5 h and for an additional 1.5 h at 150 °C. The mixture was cooled to 0 °C and MeOH (40 mL) was added carefully. The mixture was stirred for 15 h at room temperature, and the solvent was removed under reduced pressure. A saturated NaHCO<sub>3</sub> solution (200 mL) was added, and the mixture was neutralized with solid NaHCO<sub>3</sub>. The solution was extracted with dichloromethane (DCM, 3 × 200 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by flash column chromatography (cyclohexane/EtOAc, 11:1). <b>6</b> (6.32 g, 21.4 mmol, 74%) was obtained as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ = 7.99 (s, 2H), 3.97 (s, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ = 163.1, 141.6, 141.5, 126.8, 53.4. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>7</sub>H<sub>9</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>2</sub> [M+NH<sub>4</sub>]<sup>+</sup>: 310.90; found: 310.95.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (2,6-Dibromopyridin-4-yl)methanol (<b>7</b>)</h4><div class="NLM_p last"><b>6</b> (5.90 g, 20.0 mmol) was dissolved in MeOH (90 mL), and NaBH<sub>4</sub> (3.03 g, 80 mmol) was slowly added. The mixture was stirred for 3 h at 65 °C and cooled afterward to room temperature. 1 M HCl solution (40 mL) was added, and the solvent was concentrated to 10 mL volume. The solution was neutralized with solid Na<sub>2</sub>CO<sub>3</sub>, and the mixture was stirred for an additional 2 h at room temperature. The resulting precipitate was dissolved in DCM (50 mL) and diluted with H<sub>2</sub>O (50 mL). The aqueous layer was extracted with DCM (3 × 50 mL), and the combined organic layer was dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by flash column chromatography (cyclohexan/EtOAc, 3:1). <b>7</b> (4.86 g, 18.2 mmol, 91%) was obtained as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 7.61 (m, 2H), 5.62 (t, <i>J</i> = 5.8 Hz, 2H), 4.54 (d, <i>J</i> = 5.8 Hz, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 158.9, 139.7, 124.6, 60.5. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>6</sub>H<sub>9</sub>Br<sub>2</sub>N<sub>2</sub>O [M+NH<sub>4</sub>]<sup>+</sup>: 282.90; found: 282.96.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 4-(Azidomethyl)-2,6-dibromopyridine (<b>8</b>)</h4><div class="NLM_p last"><b>7</b> (4.40 g, 16.6 mmol) was dissolved in 1,4-dioxane (40 mL), and PBr<sub>3</sub> (1.9 mL, 19.7 mmol) was added. The mixture was stirred for 3 h at 40 °C and cooled afterward to room temperature. A saturated NaHCO<sub>3</sub> solution (100 mL) was added, and the aqueous layer was extracted with DCM (3 × 100 mL). The combined organic layer was dried over MgSO<sub>4</sub>, filtered, and the solvent was removed under reduced pressure. The crude product was dissolved in DMF (40 mL), and NaN<sub>3</sub> (2.14 g, 33.0 mmol) was added. The mixture was stirred for 18 h at room temperature and diluted with H<sub>2</sub>O (80 mL). The solution was extracted with EtOAc (3 × 80 mL), and the combined organic layer was washed with brine. The solution was dried over MgSO<sub>4</sub>, filtered, and the solvent was removed under reduced pressure. The residue was purified by flash column chromatography (cyclohexan/EtOAc, 10:1). <b>8</b> (3.19 g, 10.9 mmol, 66% over two steps) was obtained as a light-yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 7.72 (s, 2H), 4.60 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 151.5, 140.1, 126.2, 50.8. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>6</sub>H<sub>8</sub>Br<sub>2</sub>N<sub>5</sub> [M+NH<sub>4</sub>]<sup>+</sup>: 307.91; found: 307.95.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>tert</i>-Butyl((2,6-dibromopyridin-4-yl)methyl)carbamate (<b>9</b>)</h4><div class="NLM_p last">Azide <b>8</b> (2.73 g, 9.35 mmol) was dissolved in THF (30 mL), and a 1 M solution of PMe<sub>3</sub> in THF (10.3 mL, 10.3 mmol) was added at 0 °C. After 3 h stirring at room temperature, a solution of Boc<sub>2</sub>O (2.45 g, 11.2 mmol) in THF (9 mL) was dropwise added at 0 °C, and the mixture was stirred for an additional 2 h. 1 <span class="smallcaps smallerCapital">M</span> phosphate buffer (pH 7, 50 mL) was added to the mixture, and the aqueous layer was extracted with DCM (3 × 70 mL). The combined organic layer was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The residue was purified by flash column chromatography (cyclohexane/EtOAc, 6:1). <b>9</b> (1.72 g, 4.67 mmol, 50%) was obtained as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 7.54 (s, 2H), 4.28 (d, <i>J</i> = 6.1 Hz, 2H), 1.40 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 156.2, 155.7, 139.9, 125.5, 78.5, 41.7, 28.1. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>15</sub>Br<sub>2</sub>N<sub>2</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 364.95; found: 365.05.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>tert</i>-Butyl((2,6-diaminopyridin-4-yl)methyl)carbamate (<b>10</b>)</h4><div class="NLM_p last"><b>9</b> (1.00 g, 2.73 mmol), 25% (w/v) aqueous NH<sub>3</sub>-solution (8.2 mL, 109 mmol), Cu<sub>2</sub>O (39 mg, 0.273 mmol), K<sub>2</sub>CO<sub>3</sub> (76 mg, 0.546 mmol), and DMEDA (29.4 μL, 0.273 mmol) were dissolved in ethylene glycol (6 mL) and stirred for 16 h at 60 °C. The mixture was extracted with EtOAc (3 × 25 mL) and washed with brine. The combined organic layer was dried over MgSO<sub>4</sub>, filtered, and the solvent was removed under reduced pressure. The residue was purified by flash column chromatography (DCM/MeOH, 9:1 + 1% (v/v) NEt<sub>3</sub>). <b>10</b> (0.41 g, 1.72 mmol, 63%) was obtained as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 7.16 (t, <i>J</i> = 6.0 Hz, 1H), 5.52 (s, 2H), 5.29 (s, 4H), 3.82 (d, <i>J</i> = 6.2 Hz, 2H), 1.39 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 158.6, 155.6, 151.2, 93.5, 77.6, 42.8, 28.2. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>19</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 239.15; found: 239.16.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-(Aminomethyl)pyridine-2,6-diamine·2 HCl (<b>11</b>)</h4><div class="NLM_p last"><b>10</b> (330 mg, 1.39 mmol) was dissolved in 1,4-dioxane (5 mL), and a 4 M solution of HCl in THF (7 mL) was added. The mixture was stirred for 3 h at room temperature. The solvent was removed under reduced pressure. <b>10</b> (279 mg, 1.32 mmol, 95%) was obtained as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 13.1 (s, 1H), 8.75 (s, br, 3H), 7.53 (s, 4H), 5.95 (s, 2H), 3.82 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 151.8, 151.7, 94.2, 41.1. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>6</sub>H<sub>11</sub>N<sub>4</sub> [M + H]<sup>+</sup>: 139.10; found: 139.16.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Procedure for the Synthesis of the Intermediates <b>15</b>–<b>18</b></h3><div class="NLM_p">Boc-<span class="smallcaps smallerCapital">d</span>-Phe<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>/<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-OH<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> (1.0 eq, Boc-<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-OH only for <b>14</b>) and amine <b>11</b>–<b>14</b> (1.1 eq) were dissolved in DMF (5 mL), followed by the addition of HOB<i>t</i>·H<sub>2</sub>O (1.1 eq), EDC·HCl (1.1 eq), and <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA, 2.5–3.5 eq) at 0 °C. The mixture was stirred for 16–24 h at 0 °C to rt and extracted with EtOAc (3 × 15 mL). The resulting organic solution was washed with a 0.2 <span class="smallcaps smallerCapital">M</span> HCl solution (3 × 15 mL), saturated NaHCO<sub>3</sub> solution (3 × 15 mL), and brine (1 × 15 mL). The solvent was removed under reduced pressure, and the residue was purified by flash column chromatography.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro Derivative <b>15</b></h4><div class="NLM_p last">According to the general procedure A, compound <b>15</b> was prepared using Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro-OH (250 mg, 0.690 mmol), amine <b>11</b> (160 mg, 0.759 mmol), HOB<i>t</i>·H<sub>2</sub>O (116 mg, 0.759 mmol), EDC·HCl (146 mg, 0.759 mmol), and DIPEA (0.42 mL, 2.41 mmol). The crude product was purified by flash column chromatography (DCM/MeOH, 8:1). <b>15</b> (259 mg, 0.537 mmol, 78%) was obtained as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 7.81 (t, <i>J</i> = 6.0 Hz, 1H), 7.29–7.14 (m, 6H), 5.59–5.51 (m, 6H), 4.39–4.24 (m, 2H), 4.06–3.84 (m, 2H), 3.61–3.44 (m, 1H), 3.03–2.67 (m, 3H), 1.91–1.74 (m, 3H), 1.67–1.59 (m, 1H), 1.31 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.0, 170.3, 157.7, 155.4, 151.1, 138.4, 137.1, 129.3, 128.0, 126.4, 93.4, 78.3, 59.8, 53.8, 46.3, 41.4, 36.9, 29.0, 28.1, 23.9. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>35</sub>N<sub>6</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 483.27; found: 483.35.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro Derivative <b>16</b></h4><div class="NLM_p last">According to the general procedure A, compound <b>16</b> was prepared using Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro-OH (200 mg, 0.552 mmol), amine <b>12</b> (commercially available from ChemPur, 136 mg, 0.607 mmol), HOB<i>t</i>·H<sub>2</sub>O (93 mg, 0.607 mmol), EDC·HCl (116 mg, 0.607 mmol), and DIPEA (0.24 mL, 1.38 mmol). The crude product was purified by flash column chromatography (cyclohexane/EtOAc, 1:3). <b>16</b> (229 mg, 0.403 mmol, 73%) was obtained as a colorless solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 9.65 (s, 1H), 8.13–8.10 (m, 1H), 8.00 (t, <i>J</i> = 6.1 Hz, 1H), 7.67 (s, 1H), 7.29–7.12 (m, 6H), 6.91–6.88 (m, 1H), 4.43–4.25 (m, 3H), 3.64–3.44 (m, 1H), 3.03–2.92 (m, 1H), 2.92–2.81 (m, 2H), 2.81–2.73 (m, 1H), 1.92–1.80 (m, 3H), 1.69–1.60 (m, 1H), 1.46 (s, 9H), 1.25 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.3, 170.5, 155.4, 152.6, 152.4, 150.2, 147.5, 137.2, 129.3, 128.0, 126.4, 116.3, 109.9, 79.4, 78.3, 59.9, 53.8, 46.4, 41.3, 36.9, 28.9, 28.0, 27.9, 23.8. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>42</sub>N<sub>5</sub>O<sub>6</sub> [M + H]<sup>+</sup>: 568.32; found: 568.37.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro Derivative <b>17</b></h4><div class="NLM_p last">According to the general procedure A, compound <b>17</b> was prepared using Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro-OH (200 mg, 0.552 mmol), amine <b>13</b> (commercially available from ChemPur, 136 mg, 0.607 mmol), HOB<i>t</i>·H<sub>2</sub>O (93 mg, 0.607 mmol), EDC·HCl (116 mg, 0.607 mmol), and DIPEA (0.24 mL, 1.38 mmol). The crude product was purified by flash column chromatography (cyclohexane/EtOAc, 1:4). <b>17</b> (205 mg, 0.361 mmol, 65%) was obtained as a colorless solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 9.63 (s, 1H), 8.17–8.09 (m, 1H), 7.94 (t, <i>J</i> = 5.9 Hz, 1H), 7.70 (s, 1H), 7.29–7.19 (m, 6H), 7.09–7.07 (m, 1H), 4.43–4.22 (m, 3H), 3.61–3.43 (m, 1H), 3.03–2.97 (m, 1H), 2.92–2.79 (m, 2H), 2.73–2.72 (m, 1H), 1.85–1.71 (m, 3H), 1.65–1.68 (m, 1H), 1.47 (s, 9H), 1.27 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.2, 170.5, 155.4, 152.6, 151.1, 146.8, 146.1, 137.2, 136.5, 129.3, 128.0, 126.4, 111.8, 79.4, 78.3, 59.9, 53.8, 46.4, 36.9, 29.0, 27.9, 23.9. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>42</sub>N<sub>5</sub>O<sub>6</sub> [M + H]<sup>+</sup>: 568.32; found: 568.46.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Boc-<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro Derivative <b>18</b></h4><div class="NLM_p last">According to the general procedure A, compound <b>18</b> was prepared using Boc-<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-OH (270 mg, 0.616 mmol), amine <b>14</b> (commercially available from Sigma-Aldrich, 47 μL, 0.677 mmol), HOB<i>t</i>·H<sub>2</sub>O (104 mg, 0.677 mmol), EDC·HCl (130 mg, 0.677 mmol), and DIPEA (0.26 mL, 1.54 mmol). The crude product was purified by flash column chromatography (DCM/MeOH, 11:1). <b>18</b> (246 mg, 0.465 mmol, 76%) was obtained as a colorless solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.47–8.46 (m, 2H), 8.04 (t, <i>J</i> = 6.1 Hz, 1H), 7.41–7.39 (m, 2H), 7.31–7.16 (m, 11H), 7.09–7.07 (m, 1H), 5.13–5.08 (m, 1H), 4.39 (d, <i>J</i> = 11.6 Hz, 1H), 4.27 (d, <i>J</i> = 6.1 Hz, 2H), 3.96–3.93 (m, 1H), 3.74–2.68 (m, 1H), 3.13–3.07 (m, 1H), 1.80–1.69 (m, 2H), 1.56–1.42 (m, 2H), 1.18 (s, 9H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 172.0, 170.5, 156.1, 149.9, 148.8, 141.3, 141.2, 129.3, 128.9, 128.6, 127.2, 126.8, 122.2, 78.8, 59.9, 55.0, 53.2, 47.2, 41.5, 29.4, 28.5, 24.4. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>31</sub>H<sub>37</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 529.28; found: 529.46.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> General Procedure for the Synthesis of Inhibitors <b>1</b>–<b>4</b></h3><div class="NLM_p">Compounds <b>15</b>–<b>18</b> (1.0 eq) were dissolved in TFA (10 eq.). The mixture was stirred for 2 h and concentrated in vacuo. The residue was purified by preparative HPLC.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <span class="smallcaps smallerCapital">d</span>-Phe-Pro Inhibitor <b>1</b></h4><div class="NLM_p last">According to the general procedure B, inhibitor <b>1</b> was prepared using compound <b>15</b> (97.0 mg, 0.200 mmol) and TFA (2 mL). The crude product was purified by preparative HPLC. <b>1</b> (96.0 mg, 0.157 mmol, 78%) was obtained as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 12.8 (s, 1H), 8.42–8.40 (m, 3H), 7.37–7.23 (m, 6H), 7.19 (s, 4H), 5.76 (s, 2H), 4.39–4.30 (m, 1H), 4.20–4.18 (m, 1H), 4.09–3.99 (m, 2H), 3.54–3.50 (m, 1H), 3.12 (dd, <i>J</i> = 13.1, <i>J</i> = 5.8 Hz, 1H), 2.97 (dd, <i>J</i> = 13.1, <i>J</i> = 9.0 Hz, 1H), 2.65–2.60 (m, 1H), 1.83–1.70 (m, 3H), 1.44 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.2, 167.0, 158.7 (q, <i>J</i> = 33.3 Hz), 158.2, 151.7, 134.5, 129.4, 128.5, 127.4, 116.6 (q, <i>J</i> = 296.5 Hz), 92.9, 60.0, 52.0, 46.6, 41.3, 36.7, 29.2, 23.7. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>27</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 383.22; found: 383.22.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <span class="smallcaps smallerCapital">d</span>-Phe-Pro Inhibitor <b>2</b></h4><div class="NLM_p last">According to the general procedure B, inhibitor <b>2</b> was prepared using compound <b>16</b> (171 mg, 0.301 mmol) and TFA (3 mL). The crude product was purified by preparative HPLC. <b>2</b> (165 mg, 0.280 mmol, 92%) was obtained as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 13.9 (s, br, 1H), 8.52 (t, <i>J</i> = 6.0 Hz, 1H), 8.36 (s, br, 3H), 8.13 (s, br, 2H), 7.86–7.85 (m, 1H), 7.37–7.28 (m, 3H), 7.25–7.23 (m, 2H), 6.78–6.77 (m, 1H), 6.73 (dd, <i>J</i> = 6.6 Hz, <i>J</i> = 1.6 Hz, 1H), 4.40–4.34 (m, 1H), 4.26 (d, <i>J</i> = 5.9 Hz, 2H), 4.22–4.20 (m, 1H), 3.53–3.49 (m, 1H), 3.12 (dd, <i>J</i> = 13.2, <i>J</i> = 5.9 Hz, 1H), 2.98 (dd, <i>J</i> = 13.1, <i>J</i> = 8.7 Hz, 1H), 2.69–2.64 (m, 1H), 1.88–1.71 (m, 3H), 1.49–1.42 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.4, 167.0, 158.6 (q, <i>J</i> = 33.6 Hz), 158.2, 157.2, 154.0, 135.5, 134.4, 129.4, 128.5, 127.4, 116.4 (q, <i>J</i> = 296.0 Hz), 110.9, 109.5, 60.0, 52.0, 46.6, 41.3, 36.7, 29.2, 23.8. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 368.21; found: 368.23.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <span class="smallcaps smallerCapital">d</span>-Phe-Pro Inhibitor <b>3</b></h4><div class="NLM_p last">According to the general procedure B, inhibitor <b>3</b> was prepared using compound <b>17</b> (135 mg, 0.238 mmol) and TFA (2.4 mL). The crude product was purified by preparative HPLC. <b>3</b> (109 mg, 0.109 mmol, 82%) was obtained as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.40 (t, <i>J</i> = 5.9 Hz, 1H), 8.27 (s, br, 3H), 8.03 (s, br, 2H), 7.79–7.77 (m, 2H), 7.37–7.27 (m, 3H), 7.24–7.22 (m, 2H), 6.94–6.92 (m, 1H), 4.39–4.32 (m, 1H), 4.17–4.13 (m, 3H), 3.54–3.48 (m, 1H), 3.08 (dd, <i>J</i> = 13.3, <i>J</i> = 6.1 Hz, 1H), 2.97 (dd, <i>J</i> = 13.2, <i>J</i> = 8.5 Hz, 1H), 2.74–2.67 (m, 1H), 1.84–1.70 (m, 3H), 1.47–1.40 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.2, 167.0, 158.6 (q, <i>J</i> = 34.4 Hz), 153.4, 143.7, 134.4, 133.2, 129.4, 128.5, 127.4, 123.5, 116.2 (q, <i>J</i> = 294.3 Hz), 113.4, 60.0, 52.0, 46.6, 38.3, 36.6, 29.1, 23.7. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>26</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 368.21; found: 368.25.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <span class="smallcaps smallerCapital">d</span>-DiPhe-Pro Inhibitor <b>4</b></h4><div class="NLM_p last">According to the general procedure B, inhibitor <b>4</b> was prepared using compound <b>18</b> (159 mg, 0.301 mmol) and TFA (3 mL). The crude product was purified by preparative HPLC. <b>4</b> (182 mg, 0.277 mmol, 92%) was obtained as a colorless solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 8.77 (d, <i>J</i> = 6.3 Hz, 2H), 8.54 (t, <i>J</i> = 6.0 Hz, 1H), 8.35 (s, br, 3H), 7.79 (d, <i>J</i> = 6.4 Hz, 2H), 7.66–7.65 (m, 2H), 7.45–7.42 (m, 2H), 7.36–7.24 (m, 6H), 5.02 (d, <i>J</i> = 11.2 Hz, 1H), 4.53 (dd, <i>J</i> = 17.6, <i>J</i> = 6.3 Hz, 1H), 4.45 (dd, <i>J</i> = 17.6, <i>J</i> = 5.7 Hz, 1H), 4.40 (d, <i>J</i> = 11.2 Hz, 1H), 3.61–3.57 (m, 1H), 2.93–2.88 (m, 1H), 1.77–1.68 (m, 2H), 1.57–1.48 (m, 1H), 1.29–1.22 (m, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ = 171.3, 167.1, 158.4 (q, <i>J</i> = 35.0 Hz), 157.6, 143.0, 138.4, 137.5, 129.0, 128.9, 128.3, 127.6, 127.5, 123.9, 116.2 (q, <i>J</i> = 293.8 Hz), 60.0, 53.5, 53.3, 47.1, 41.5, 28.9, 23.6. MS (ESI+) <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>29</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 429.23; found: 429.29.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> p<i>K</i><sub>a</sub> Determination</h3><div class="NLM_p last">The ionization constants p<i>K</i><sub>a</sub> of the compounds of interest were determined by the SiriusT3 pH-metric p<i>K</i><sub>a</sub> method, which is based on the potentiometric titration method reported in ref <a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a>. For the titration, approximately 1.0 mg of solid sample was weighed into a vial and dissolved in 1.5 mL of water containing 0.15 M KCl as the background electrolyte. The electrolyte serves to improve the measurement precision and to mimic the physiological state. The pH of the dilute sample solution was adjusted to 2.0 by addition of 0.5 M HCl, and then it was titrated with a standardized base (0.5 M KOH) to pH 12.0 at 25 °C under argon atmosphere. During the titration, more than 25 pH readings (volume of titrant vs. pH) were collected and compared to a blank titration (without sample). The resulted shape of the curve indicates the amount of substance present and its characteristic acid–base ionization properties. p<i>K</i><sub>a</sub> results were calculated by SIRIUS T3 Refine Version 1.1.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> ITC Data Collection</h3><div class="NLM_p last">All ITC experiments were performed on a Microcal ITC<sub>200</sub> (Malvern) device at 25 °C with a reference power of 5 μcal s<sup>–1</sup>. To determine the thermodynamic profiles including protonation effects in thrombin, a direct titration was used for all four ligands. Ligand solutions of 0.4 mM <b>1</b>, 0.6 mM <b>2</b>, 0.3 mM <b>3</b>, and 1.5 mM <b>4</b> were titrated into 40, 60, 30, and 150 μM thrombin, respectively. Thrombin samples were prepared by a solution of human α-thrombin (Beriplast, CSL Behring, Marburg, Germany) in a buffer of 100 mM NaCl, 0.1% (w/v) PEG8000, and 50 mM phosphate, HEPES, <i>N</i>-(tri(hydroxymethyl)methyl)glycine (tricine) or Tris at pH 7.8, which was dialyzed at 4 °C overnight. Ligands were dissolved in the same buffer with 3% (v/v) DMSO to assure a complete solubility. The titration protocol consisted of an initial volume of 0.3 μL and 12–22 injections between 1.3 and 1.5 μL separated by an interval of 180 s. To collect the ITC data for trypsin, two titration protocols were used. The direct titration was performed for <b>1–3</b> with a ligand concentration of 2.5 mM, using 125 μM trypsin to determine the protonation states, whereas a displacement titration was used for ligand <b>4</b> due to its lower affinity toward trypsin. Trypsin samples were prepared by a solution of bovine-β-trypsin (Sigma-Aldrich, product number T8003) in a buffer of 80 mM NaCl, 10 mM CaCl<sub>2</sub>, 0.1% (w/v) PEG8000, and 100 mM HEPES, Tricine or Tris, at pH 7.8, which was dialyzed at 4 °C overnight. Ligands were dissolved in the same buffer with 3% (v/v) DMSO to assure a complete solubility as for the ligand preparation in thrombin. An initial volume of 0.3 μL was followed by 28–30 injections with a volume of 1.1 μL using 180 s as the interval time for the direct titration, while 19 injections were chosen with 1.9 μL volume and the same interval time for the displacement titration. According to this, 40 μM trypsin was preincubated with 3.4 mM of <b>4</b> using compound <b>19</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_001.pdf" class="ext-link">Table S3</a>) as a displacement ligand with a concentration of 0.4 mM. The displacement ligand <b>19</b> was analyzed in a direct measurement with a ligand concentration of 0.3 mM, which was titrated into 30 μM trypsin. To confirm the dissociation constants <i>K</i><sub>d</sub> of <b>1</b>–<b>3</b> from the direct titrations, the displacement titration was additionally performed in 40 μM trypsin (preincubated with 0.22 mM <b>1</b>, 0.45 mM <b>2</b>, or 0.35 mM <b>3</b>) after the same procedure as above. The ligand concentrations for the preincubation of the protein were calculated according to a previous study to reach a saturation goal of 87%, which is based on the <i>K</i><sub>i</sub> values from the fluorescence assay. All ITC titrations were performed at least in triplicate, and the raw thermograms were analyzed with NITPIC<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> (version 1.1.8) and SEDPHAT<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> (version 10.58d).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Crystallization</h3><div class="NLM_p">Human α-thrombin (Enzyme Research Laboratories, South Bend, IN) was dissolved in a crystallization buffer consisting of 20 mM NaH<sub>2</sub>PO<sub>4</sub> (pH 7.5), 350 mM NaCl, and 2 mM benzamidine at a concentration of 10 mg mL<sup>–1</sup>. A hirudin fragment (residues 54–65, Bachem, Bubendorf, Switzerland) was dissolved in the crystallization buffer at 2.5 mg mL<sup>–1</sup>. The two solutions were mixed in a ratio of 80% thrombin solution and 20% hirudin solution. The mixture was incubated for 1 h at 4 °C and centrifuged for 5 min. Thrombin crystals were grown at 4 °C by the hanging-drop vapor diffusion method in 2 μL crystallization drops. The thrombin–hirudin mixture (1 μL) was placed on a cover slide with 1 μL of reservoir buffer composed of 20 mM NaH<sub>2</sub>PO<sub>4</sub> (pH 7.5), 350 mM NaCl, and 27% (w/v) PEG8000 followed by microseeding. After 3 weeks, thrombin crystals were obtained in a good diffracting quality for soaking. The soaking solution (50% crystallization buffer and 50% reservoir buffer) was combined with the ligand solution in a final concentration of 5 mM ligand and 10% DMSO. The crystals were soaked for 24 h remaining at 4 °C and flash-frozen in liquid nitrogen in a cryoprotectant solution of reservoir buffer, 20% glycerol, and 5 mM of the respective ligand.</div><div class="NLM_p last">Bovine-β-trypsin was dissolved at a concentration of 20 mg mL<sup>–1</sup> in 10 mM CaCl<sub>2</sub>, 1 mM HCl, 3 mM <b>1</b>–<b>3</b> or 6 mM <b>4</b>, and 6% (v/v) DMSO. Trypsin crystals were grown at 18 °C on sitting-drop plates in 2 μL crystallization drops. Each crystallization drop consisted of 1 μL of the trypsin–ligand solution and 1 μL of the reservoir buffer, which contained 0.1 M imidazole, 0.1 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.1% (w/v) NaN<sub>3</sub>, and 25–27% (w/v) PEG8000 at pH 7–8. The crystals finished growing after 7 days and were flash-frozen in liquid nitrogen in a cryoprotectant solution composed of reservoir buffer, 20% glycerol, and 3–6 mM of the respective ligand.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> X-ray Data Collection, Structure Determination, and Refinement</h3><div class="NLM_p">The data collection of all crystals was performed on beamline 14.1 or 14.2 at BESSY II (Berlin, Germany) or on EMBL beamline P13 at DESY (Hamburg, Germany), which are both equipped with a Dectris Pilatus 6 M detector or Marmosaic 225. The data were processed using XDS<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and XDSAPP2.0.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The starting model was generated by molecular replacement using Phaser (version 2.5.7–2.7.17)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> with the PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZGB">2ZGB</a> or <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8D">1H8D</a> for thrombin and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MNK">5MNK</a> for trypsin as search models. The structures were refined using Phenix.refine (version 1.10.1–1.15.2)<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> against structure factor amplitudes in five cycles for each refinement. Coot (version 0.7.2–0.8.9.1)<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> was used to generate and to build a model into σ-weighted maps (2<i>mF</i><sub>o</sub><i>– DF</i><sub>c</sub> and <i>mF</i><sub>o</sub> – <i>DF</i><sub>c</sub>). Simulated annealing was performed in the first step to remove potential bias from the search model. Thrombin structures with inhibitors <b>2</b> and <b>3</b> were refined anisotropically, whereas the complexes with inhibitors <b>1</b> and <b>4</b> were refined isotropically. Translation/Libration/Screw (TLS) parameters were used subsequently to these isotropic refinements, whereupon the definition of the TLS groups was made with Phenix. For the trypsin complex structure with inhibitor <b>1</b>, the <i>B</i>-factors were refined isotropically with TLS groups. In all remaining trypsin structures, anisotropic <i>B</i>-factor refinement was applied. Multiple side-chain conformations, water molecules, ions, solvents, and ligands were built into the model and maintained during the refinement if the minor populated side chain displayed at least 20% occupancy. Further data and refinement statistics are described in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Tables <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. X-ray Data Collection and Refinement Statistics for Thrombin Structures of <b>1</b>–<b>4</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="rowsep1 colsep0" colspan="4" align="center">ligand (PDB code)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center">1 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSX">6HSX</a>)</th><th class="colsep0 rowsep0" align="center">2 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q">6T3Q</a>)</th><th class="colsep0 rowsep0" align="center">3 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T4A">6T4A</a>)</th><th class="colsep0 rowsep0" align="center">4 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT">6TDT</a>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="center">Data Collection and Processing</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Å)</td><td class="colsep0 rowsep0" align="left">0.9184</td><td class="colsep0 rowsep0" align="left">0.9184</td><td class="colsep0 rowsep0" align="left">0.9184</td><td class="colsep0 rowsep0" align="left">0.9184</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">beamline</td><td class="colsep0 rowsep0" align="left">BESSY, 14.1</td><td class="colsep0 rowsep0" align="left">BESSY, 14.2</td><td class="colsep0 rowsep0" align="left">BESSY, 14.2</td><td class="colsep0 rowsep0" align="left">BESSY, 14.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">detector</td><td class="colsep0 rowsep0" align="left">PILATUS 6 M</td><td class="colsep0 rowsep0" align="left">Marmosaic 225 mm</td><td class="colsep0 rowsep0" align="left">Marmosaic 225 mm</td><td class="colsep0 rowsep0" align="left">PILATUS 6 M</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left">C121</td><td class="colsep0 rowsep0" align="left">C121</td><td class="colsep0 rowsep0" align="left">C121</td><td class="colsep0 rowsep0" align="left">C121</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell parameters</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">70.0, 71.7, 72.6</td><td class="colsep0 rowsep0" align="left">70.0, 71.3, 72.6</td><td class="colsep0 rowsep0" align="left">70.2, 71.3, 72.5</td><td class="colsep0 rowsep0" align="left">70.0, 71.7, 72.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (°)</td><td class="colsep0 rowsep0" align="left">90.0, 100.4, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 100.6, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 100.6, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 100.2, 90.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Matthews coefficient (Å<sup>3</sup>/Da)</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent content (%)</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">51</td><td class="colsep0 rowsep0" align="left">51</td><td class="colsep0 rowsep0" align="left">51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="center">Diffraction Data<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">43.51–1.56 (1.59–1.56)</td><td class="colsep0 rowsep0" align="left">50.00–1.33 (1.35–1.33)</td><td class="colsep0 rowsep0" align="left">50.00–1.31 (1.33–1.31)</td><td class="colsep0 rowsep0" align="left">43.36–1.53 (1.62–1.53)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">49365 (7727)</td><td class="colsep0 rowsep0" align="left">80331 (4048)</td><td class="colsep0 rowsep0" align="left">84043 (4191)</td><td class="colsep0 rowsep0" align="left">52277 (8431)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">3.4 (3.5)</td><td class="colsep0 rowsep0" align="left">3.8 (3.5)</td><td class="colsep0 rowsep0" align="left">3.3 (3.2)</td><td class="colsep0 rowsep0" align="left">3.4 (3.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i>(<i>I</i>)<sub>sym</sub> (%)</td><td class="colsep0 rowsep0" align="left">4.0 (49.0)</td><td class="colsep0 rowsep0" align="left">4.6 (49.5)</td><td class="colsep0 rowsep0" align="left">4.3 (43.2)</td><td class="colsep0 rowsep0" align="left">8.5 (47.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wilson B-factor (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">19.6</td><td class="colsep0 rowsep0" align="left">11.8</td><td class="colsep0 rowsep0" align="left">11.1</td><td class="colsep0 rowsep0" align="left">15.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">97.6 (95.2)</td><td class="colsep0 rowsep0" align="left">100 (100)</td><td class="colsep0 rowsep0" align="left">99.9 (99.9)</td><td class="colsep0 rowsep0" align="left">98.7 (98.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">⟨<i>I</i>/σ(<i>I</i>)⟩</td><td class="colsep0 rowsep0" align="left">17.4 (2.5)</td><td class="colsep0 rowsep0" align="left">26.8 (2.6)</td><td class="colsep0 rowsep0" align="left">26.7 (2.7)</td><td class="colsep0 rowsep0" align="left">8.6 (2.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="center">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">43.51–1.56</td><td class="colsep0 rowsep0" align="left">18.51–1.33</td><td class="colsep0 rowsep0" align="left">35.63–1.31</td><td class="colsep0 rowsep0" align="left">43.36–1.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections (work/free)</td><td class="colsep0 rowsep0" align="left">46896/2469</td><td class="colsep0 rowsep0" align="left">76296/4035</td><td class="colsep0 rowsep0" align="left">79852/4189</td><td class="colsep0 rowsep0" align="left">49662/2611</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">final R value for all reflections (work/free) (%)</td><td class="colsep0 rowsep0" align="left">16.2/18.4</td><td class="colsep0 rowsep0" align="left">12.5/14.6</td><td class="colsep0 rowsep0" align="left">12.7/14.8</td><td class="colsep0 rowsep0" align="left">16.1/17.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein residues (L chain/H chain)</td><td class="colsep0 rowsep0" align="left">28/250</td><td class="colsep0 rowsep0" align="left">29/251</td><td class="colsep0 rowsep0" align="left">28/251</td><td class="colsep0 rowsep0" align="left">31/251</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">residues of hirudin (I chain)</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sodium atoms</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">inhibitor atoms</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="left">229</td><td class="colsep0 rowsep0" align="left">257</td><td class="colsep0 rowsep0" align="left">262</td><td class="colsep0 rowsep0" align="left">253</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSD from ideality</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.008</td><td class="colsep0 rowsep0" align="left">0.007</td><td class="colsep0 rowsep0" align="left">0.006</td><td class="colsep0 rowsep0" align="left">0.008</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (°)</td><td class="colsep0 rowsep0" align="left">0.993</td><td class="colsep0 rowsep0" align="left">1.046</td><td class="colsep0 rowsep0" align="left">0.994</td><td class="colsep0 rowsep0" align="left">1.030</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran plot (%)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">most favored regions</td><td class="colsep0 rowsep0" align="left">86.1</td><td class="colsep0 rowsep0" align="left">87.0</td><td class="colsep0 rowsep0" align="left">85.7</td><td class="colsep0 rowsep0" align="left">87.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">additionally allowed regions</td><td class="colsep0 rowsep0" align="left">13.5</td><td class="colsep0 rowsep0" align="left">12.6</td><td class="colsep0 rowsep0" align="left">14.3</td><td class="colsep0 rowsep0" align="left">11.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">generously allowed regions</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">disallowed regions</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean B-factors (Å<sup>2</sup>)<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein (L chain + H chain)</td><td class="colsep0 rowsep0" align="left">26.6</td><td class="colsep0 rowsep0" align="left">17.0</td><td class="colsep0 rowsep0" align="left">15.9</td><td class="colsep0 rowsep0" align="left">19.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand</td><td class="colsep0 rowsep0" align="left">20.1</td><td class="colsep0 rowsep0" align="left">12.1</td><td class="colsep0 rowsep0" align="left">11.4</td><td class="colsep0 rowsep0" align="left">19.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="left">33.7</td><td class="colsep0 rowsep0" align="left">28.4</td><td class="colsep0 rowsep0" align="left">26.8</td><td class="colsep0 rowsep0" align="left">27.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Values in parenthesis refer to the highest resolution shell.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Calculated using PROCHECK.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">Calculated using MOLEMAN excluding hydrogen atoms.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a></p></div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. X-ray Data Collection and Refinement Statistics for Trypsin Structures of <b>1</b>–<b>4</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="rowsep1 colsep0" colspan="4" align="center"><b>ligand</b> (PDB code)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="center"><b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0M">6T0M</a>)</th><th class="colsep0 rowsep0" align="center"><b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q">6T3Q</a>)</th><th class="colsep0 rowsep0" align="center"><b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5W">6T5W</a>)</th><th class="colsep0 rowsep0" align="center"><b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SY3">6SY3</a>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="center">Data Collection and Processing</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Å)</td><td class="colsep0 rowsep0" align="left">0.9763</td><td class="colsep0 rowsep0" align="left">0.9763</td><td class="colsep0 rowsep0" align="left">0.9184</td><td class="colsep0 rowsep0" align="left">0.9184</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">beamline</td><td class="colsep0 rowsep0" align="left">DESY, P13</td><td class="colsep0 rowsep0" align="left">DESY, P13</td><td class="colsep0 rowsep0" align="left">BESSY, 14.1</td><td class="colsep0 rowsep0" align="left">BESSY, 14.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">detector</td><td class="colsep0 rowsep0" align="left">PILATUS 6M-F</td><td class="colsep0 rowsep0" align="left">PILATUS 6M-F</td><td class="colsep0 rowsep0" align="left">PILATUS 6 M</td><td class="colsep0 rowsep0" align="left">PILATUS 6 M</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td><td class="colsep0 rowsep0" align="left">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td><td class="colsep0 rowsep0" align="left">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td><td class="colsep0 rowsep0" align="left">P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Unit cell parameters</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">54.5, 56.8, 66.7</td><td class="colsep0 rowsep0" align="left">54.6, 56.7, 66.7</td><td class="colsep0 rowsep0" align="left">54.2, 56.8, 66.2</td><td class="colsep0 rowsep0" align="left">54.5, 56.5, 66.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (°)</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Matthews coefficient (Å<sup>3</sup>/Da)</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">2.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent content (%)</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left">44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="center">Diffraction Data<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">43.22–1.51 (1.60–1.51)</td><td class="colsep0 rowsep0" align="left">43.18–1.19 (1.26–1.19)</td><td class="colsep0 rowsep0" align="left">43.12–1.13 (1.20–1.13)</td><td class="colsep0 rowsep0" align="left">43.10–0.95 (1.01–0.95)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">33113 (5172)</td><td class="colsep0 rowsep0" align="left">66940 (10620)</td><td class="colsep0 rowsep0" align="left">76852 (12065)</td><td class="colsep0 rowsep0" align="left">127398 (19451)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">5.8 (5.6)</td><td class="colsep0 rowsep0" align="left">5.7 (5.7)</td><td class="colsep0 rowsep0" align="left">5.0 (4.8)</td><td class="colsep0 rowsep0" align="left">4.1 (3.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i>(<i>I</i>)<sub>sym</sub> (%)</td><td class="colsep0 rowsep0" align="left">5.5 (48.7)</td><td class="colsep0 rowsep0" align="left">5.3 (49.2)</td><td class="colsep0 rowsep0" align="left">5.2 (56.5)</td><td class="colsep0 rowsep0" align="left">3.8 (49.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wilson B-factor (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">16.0</td><td class="colsep0 rowsep0" align="left">12.6</td><td class="colsep0 rowsep0" align="left">11.3</td><td class="colsep0 rowsep0" align="left">8.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.4 (97.5)</td><td class="colsep0 rowsep0" align="left">99.6 (98.9)</td><td class="colsep0 rowsep0" align="left">99.5 (97.8)</td><td class="colsep0 rowsep0" align="left">98.0 (93.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">⟨<i>I</i>/σ(<i>I</i>)⟩</td><td class="colsep0 rowsep0" align="left">17.5 (3.1)</td><td class="colsep0 rowsep0" align="left">15.0 (2.8)</td><td class="colsep0 rowsep0" align="left">14.6 (2.6)</td><td class="colsep0 rowsep0" align="left">16.0 (1.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="center">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">43.22–1.51</td><td class="colsep0 rowsep0" align="left">42.22–1.19</td><td class="colsep0 rowsep0" align="left">43.11–1.13</td><td class="colsep0 rowsep0" align="left">33.80–0.95</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections (work/free)</td><td class="colsep0 rowsep0" align="left">31457/1655</td><td class="colsep0 rowsep0" align="left">63593/3347</td><td class="colsep0 rowsep0" align="left">73009/3843</td><td class="colsep0 rowsep0" align="left">125174/2224</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">final <i>R</i> value for all reflections (work/free) (%)</td><td class="colsep0 rowsep0" align="left">14.8/17.1</td><td class="colsep0 rowsep0" align="left">12.3/13.9</td><td class="colsep0 rowsep0" align="left">12.8/14.5</td><td class="colsep0 rowsep0" align="left">12.4/13.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein residues</td><td class="colsep0 rowsep0" align="left">223</td><td class="colsep0 rowsep0" align="left">223</td><td class="colsep0 rowsep0" align="left">223</td><td class="colsep0 rowsep0" align="left">223</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">calcium atoms</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">inhibitor atoms</td><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="left">173</td><td class="colsep0 rowsep0" align="left">202</td><td class="colsep0 rowsep0" align="left">201</td><td class="colsep0 rowsep0" align="left">233</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSD from ideality</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.007</td><td class="colsep0 rowsep0" align="left">0.006</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (°)</td><td class="colsep0 rowsep0" align="left">0.935</td><td class="colsep0 rowsep0" align="left">0.906</td><td class="colsep0 rowsep0" align="left">0.903</td><td class="colsep0 rowsep0" align="left">0.944</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran plot (%)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">most favored regions</td><td class="colsep0 rowsep0" align="left">86.2</td><td class="colsep0 rowsep0" align="left">87.2</td><td class="colsep0 rowsep0" align="left">85.6</td><td class="colsep0 rowsep0" align="left">86.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">additionally allowed regions</td><td class="colsep0 rowsep0" align="left">13.8</td><td class="colsep0 rowsep0" align="left">12.8</td><td class="colsep0 rowsep0" align="left">14.4</td><td class="colsep0 rowsep0" align="left">13.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">generously allowed regions</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">disallowed regions</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean B-factors (Å<sup>2</sup>)<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">17.4</td><td class="colsep0 rowsep0" align="left">14.0</td><td class="colsep0 rowsep0" align="left">12.8</td><td class="colsep0 rowsep0" align="left">10.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand</td><td class="colsep0 rowsep0" align="left">21.4</td><td class="colsep0 rowsep0" align="left">15.2</td><td class="colsep0 rowsep0" align="left">11.4</td><td class="colsep0 rowsep0" align="left">10.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="left">28.5</td><td class="colsep0 rowsep0" align="left">27.4</td><td class="colsep0 rowsep0" align="left">26.1</td><td class="colsep0 rowsep0" align="left">23.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Values in parenthesis refer to the highest resolution shell.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Calculated using PROCHECK.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Calculated using MOLEMAN excluding hydrogen atoms.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a></p></div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42"><a href="/doi/suppl/10.1021/acs.jmedchem.9b02061" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46631" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46631" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b02061?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02061</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Figures of <i>mF</i><sub>o</sub> – <i>DF</i><sub>c</sub> omit maps of the investigated ligands; reference to the numbering of water molecules in the pdb files and the manuscript; ITC results from individual titrations; thermograms and binding isotherms from ITC measurements; and HPLC traces (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_001.pdf" class="ext-link">PDF</a>). Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_001.pdf">jm9b02061_si_001.pdf (1.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_002.csv">jm9b02061_si_002.csv (0.35 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank and will be released upon article publication: <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSX">6HSX</a>), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q">6T3Q</a>), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T4A">6T4A</a>), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT">6TDT</a>) for thrombin; <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0M">6T0M</a>), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0P">6T0P</a>), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5W">6T5W</a>), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SY3">6SY3</a>) for trypsin.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b02061" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38862" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38862" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gerhard Klebe</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Philipps-University
Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4913-390X" title="Orcid link">http://orcid.org/0000-0002-4913-390X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ea81868f888faa999e8b8c8cc49f8483c7878b98889f988dc48e8f"><span class="__cf_email__" data-cfemail="a0cbccc5c2c5e0d3d4c1c6c68ed5cec98dcdc1d2c2d5d2c78ec4c5">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Khang Ngo</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Philipps-University
Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chelsey Collins-Kautz</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Philipps-University
Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Gerstenecker</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Philipps-University
Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Björn Wagner</span> - <span class="hlFld-Affiliation affiliation">Pharma
Research Non-Clinical Safety, F. Hoffmann-La
Roche AG, 4070 Basel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Heine</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Philipps-University
Marburg, Marbacher Weg 6, 35032 Marburg, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e5388-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16205" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16205" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank CSL Behring (Marburg, Germany) for supplying human thrombin from the production of Beriplast. They are also thankful to the beamline staff at BESSY II (Helmholtz-Zentrum Berlin, Germany) and PETRA III (EMBL Hamburg, Germany) for their support during data collection. They acknowledge the receipt of a travel grant from the Helmholtz-Zentrum Berlin. They would also like to thank Namir Abazi (Univ. of Marburg, Germany) for his support in the crystallization of thrombin. The authors kindly thank Prof. Dr. Torsten Steinmetzer (University of Marburg, Germany) for his thorough proofreading and many valuable comments to improve the manuscript and his group for support in the enzyme kinetic fluorescence assay.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMC</td><td class="NLM_def"><p class="first last">7-amido-4-methylcoumarin</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electron spray ionization</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectroscopy</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance spectroscopy</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">poly(ethylene glycol)</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">Tricine</td><td class="NLM_def"><p class="first last"><i>N</i>-(tri(hydroxymethyl)methyl)glycine</p></td></tr><tr><td class="NLM_term">Tris</td><td class="NLM_def"><p class="first last">tris(hydroxymethyl)aminomethane</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34735" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34735" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 59 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Can We Rationally Design Promiscuous Drugs?</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.sbi.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16442279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=127-136&author=A.+L.+Hopkinsauthor=J.+S.+Masonauthor=J.+P.+Overington&title=Can+We+Rationally+Design+Promiscuous+Drugs%3F&doi=10.1016%2Fj.sbi.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Can we rationally design promiscuous drugs?</span></div><div class="casAuthors">Hopkins, Andrew L.; Mason, Jonathan S.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-136</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design is now used widely in modern medicinal chem.  The application of structural biol. to medicinal chem. has heralded the 'rational drug design' vision of discovering exquisitely selective ligands.  However, recent advances in post-genomic biol. are indicating that polypharmacol. may be a necessary trait for the efficacy of many drugs, therefore questioning the 'one drug, one target' assumption of current rational drug design.  By combining advances in chemoinformatics and structural biol., it might be possible to rationally design the next generation of promiscuous drugs with polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQWdh60x9AHrVg90H21EOLACvtfcHk0liGJKWXzzY1uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D&md5=8f9e95a41976b8e3ac790f2fff2c406f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DCan%2520We%2520Rationally%2520Design%2520Promiscuous%2520Drugs%253F%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D127%26epage%3D136%26doi%3D10.1016%2Fj.sbi.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huggins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span> <span> </span><span class="NLM_article-title">Rational Approaches to Improving Selectivity in Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1424</span>– <span class="NLM_lpage">1444</span>, <span class="refDoi"> DOI: 10.1021/jm2010332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2010332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1424-1444&author=D.+J.+Hugginsauthor=W.+Shermanauthor=B.+Tidor&title=Rational+Approaches+to+Improving+Selectivity+in+Drug+Design&doi=10.1021%2Fjm2010332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Approaches to Improving Selectivity in Drug Design</span></div><div class="casAuthors">Huggins, David J.; Sherman, Woody; Tidor, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1424-1444</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Rational Approaches to improving selectivity in Drug Design are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJv9AbW-c3TbVg90H21EOLACvtfcHk0lgYZRWvVBs1IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFWktg%253D%253D&md5=5b7bf49f9f96a05adad4d42f63739252</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm2010332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2010332%26sid%3Dliteratum%253Aachs%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DTidor%26aufirst%3DB.%26atitle%3DRational%2520Approaches%2520to%2520Improving%2520Selectivity%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1424%26epage%3D1444%26doi%3D10.1021%2Fjm2010332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alterio, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Fiore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ambrosio, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supuran, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Simone, G.</span></span> <span> </span><span class="NLM_article-title">Multiple Binding Modes of Inhibitors to Carbonic Anhydrases: How to Design Specific Drugs Targeting 15 Different Isoforms?</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">4421</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/cr200176r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr200176r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2012&pages=4421-4468&author=V.+Alterioauthor=A.+Di+Fioreauthor=K.+D%E2%80%99Ambrosioauthor=C.+T.+Supuranauthor=G.+De+Simone&title=Multiple+Binding+Modes+of+Inhibitors+to+Carbonic+Anhydrases%3A+How+to+Design+Specific+Drugs+Targeting+15+Different+Isoforms%3F&doi=10.1021%2Fcr200176r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Binding Modes of Inhibitors to Carbonic Anhydrases: How to Design Specific Drugs Targeting 15 Different Isoforms?</span></div><div class="casAuthors">Alterio, Vincenzo; Di Fiore, Anna; D'Ambrosio, Katia; Supuran, Claudiu T.; De Simone, Giuseppina</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4421-4468</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The aim of this review is to provide an exhaustive description of the structural studies on α-CAs (carbonic anhydrases) so far reported.  In particular, the main structural features of the catalytically active α-CA isoenzymes characterized to date will be described and the current state of the art on complexes of hCA II with the principal classes of inhibitors will be summarized.  A comparison with the corresponding adducts of other isoforms, when available, will be also performed, in order to find for each isoform the key residues responsible for enzyme/inhibitor interaction.  Finally, recent advances in the field of drug design of selective CAIs will be illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA0GdtMHwlZ7Vg90H21EOLACvtfcHk0lgYZRWvVBs1IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrsbg%253D&md5=eb45ffb5efe07980f1388d99bf51406a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fcr200176r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr200176r%26sid%3Dliteratum%253Aachs%26aulast%3DAlterio%26aufirst%3DV.%26aulast%3DDi%2BFiore%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Ambrosio%26aufirst%3DK.%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26aulast%3DDe%2BSimone%26aufirst%3DG.%26atitle%3DMultiple%2520Binding%2520Modes%2520of%2520Inhibitors%2520to%2520Carbonic%2520Anhydrases%253A%2520How%2520to%2520Design%2520Specific%2520Drugs%2520Targeting%252015%2520Different%2520Isoforms%253F%26jtitle%3DChem.%2520Rev.%26date%3D2012%26volume%3D112%26spage%3D4421%26epage%3D4468%26doi%3D10.1021%2Fcr200176r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitesides, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, V. M.</span></span> <span> </span><span class="NLM_article-title">Designing Ligands to Bind Proteins</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1017/S0033583506004240</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1017%2FS0033583506004240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16817982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XoslSgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2005&pages=385-395&author=G.+M.+Whitesidesauthor=V.+M.+Krishnamurthy&title=Designing+Ligands+to+Bind+Proteins&doi=10.1017%2FS0033583506004240"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Designing ligands to bind proteins</span></div><div class="casAuthors">Whitesides, George M.; Krishnamurthy, Vijay M.</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">385-395</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  The ability to design drugs (so-called 'rational drug design') was one of the long-term objectives of chem. for 50 years.  It is an exceptionally difficult problem, and many of its parts lie outside the expertise of chem.  The much more limited problem - how to design tight-binding ligands (rational ligand design) - would seem to be one that chem. could solve, but has also proved remarkably recalcitrant.  The question is Why is it so difficult' and the answer is 'We still don't entirely know'.  This perspective discusses some of the tech. issues - potential functions, protein plasticity, enthalpy/entropy compensation, and others - that contribute, and suggests areas where fundamental understanding of protein-ligand interactions falls short of what is needed.  It surveys recent technol. developments (in particular, isothermal titrn. calorimetry) that will, hopefully, make now the time for serious progress in this area.  It concludes with the calorimetric examn. of the assocn. of a series of systematically varied ligands with a model protein.  The counterintuitive thermodn. results obsd. serve to illustrate that, even in relatively simple systems, understanding protein-ligand assocn. is challenging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrImeG0nXP2w7Vg90H21EOLACvtfcHk0lgYZRWvVBs1IQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XoslSgt7g%253D&md5=24d27dce3770c6fd712cc47d6af9d4ad</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1017%2FS0033583506004240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583506004240%26sid%3Dliteratum%253Aachs%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26aulast%3DKrishnamurthy%26aufirst%3DV.%2BM.%26atitle%3DDesigning%2520Ligands%2520to%2520Bind%2520Proteins%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2005%26volume%3D38%26spage%3D385%26epage%3D395%26doi%3D10.1017%2FS0033583506004240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Puertas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo-Macias, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Romero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Vinas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morreale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span> <span> </span><span class="NLM_article-title">Computational Approaches to Model Ligand Selectivity in Drug Design</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.2174/156802606775193338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.2174%2F156802606775193338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16454757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVKmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=41-55&author=A.+Ortizauthor=P.+Gomez-Puertasauthor=A.+Leo-Maciasauthor=P.+Lopez-Romeroauthor=E.+Lopez-Vinasauthor=A.+Morrealeauthor=M.+Murciaauthor=K.+Wang&title=Computational+Approaches+to+Model+Ligand+Selectivity+in+Drug+Design&doi=10.2174%2F156802606775193338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches to model ligand selectivity in drug design</span></div><div class="casAuthors">Ortiz, Angel R.; Gomez-Puertas, Paulino; Leo-Macias, Alejandra; Lopez-Romero, Pedro; Lopez-Vinas, Eduardo; Morreale, Antonio; Murcia, Marta; Wang, Kun</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  To be effective, a designed drug must discriminate successfully the macromol. target from alternative structures present in the organism.  The last few years have witnessed the emergence of different computational tools aimed to the understanding and modeling of this process at mol. level.  Although still rudimentary, these methods are shaping a coherent approach to help in the design of mols. with high affinity and specificity, both in lead discovery and in lead optimization.  It is the purpose of this review to illustrate the array of computational tools available to consider selectivity in the design process, to summarize the most relevant applications, and to sketch the challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptjiqa0g9gK7Vg90H21EOLACvtfcHk0lhiwz4a1jat6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVKmtLo%253D&md5=582a6728d8614705a81e5af1dd3a6ed2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F156802606775193338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802606775193338%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%26aufirst%3DA.%26aulast%3DGomez-Puertas%26aufirst%3DP.%26aulast%3DLeo-Macias%26aufirst%3DA.%26aulast%3DLopez-Romero%26aufirst%3DP.%26aulast%3DLopez-Vinas%26aufirst%3DE.%26aulast%3DMorreale%26aufirst%3DA.%26aulast%3DMurcia%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DK.%26atitle%3DComputational%2520Approaches%2520to%2520Model%2520Ligand%2520Selectivity%2520in%2520Drug%2520Design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2006%26volume%3D6%26spage%3D41%26epage%3D55%26doi%3D10.2174%2F156802606775193338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Finding a Better Path to Drug Selectivity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">985</span>– <span class="NLM_lpage">990</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.drudis.2011.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=21839183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWktbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=985-990&author=Y.+Kawasakiauthor=E.+Freire&title=Finding+a+Better+Path+to+Drug+Selectivity&doi=10.1016%2Fj.drudis.2011.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Finding a better path to drug selectivity</span></div><div class="casAuthors">Kawasaki, Yuko; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">21/22</span>),
    <span class="NLM_cas:pages">985-990</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Extremely high affinity and selectivity are two of the most sought-after properties of drug mols.  Selectivity has been difficult to achieve, esp. for targets that belong to large families of structurally and functionally related proteins.  There are essentially two ways by which selectivity can be improved during lead optimization: a chem. modification of the lead compd. that improves the affinity towards the target to a higher extent than to off-target mols.; and a chem. modification that lowers the affinity of the lead compd. towards off-target mols.  Maximal selectivity is achieved when both mechanisms can be combined synergistically.  As we discuss here, anal. of several protease inhibitors that vary in a single functionality indicates that nonpolar functionalities preferentially follow the first mechanism, whereas polar functionalities follow the second, and that those features are imprinted in their thermodn. signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGj1IsjIk4bbVg90H21EOLACvtfcHk0lhiwz4a1jat6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWktbjF&md5=68ab56fd05adcdc61d12ee955b0fddc5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DFinding%2520a%2520Better%2520Path%2520to%2520Drug%2520Selectivity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D985%26epage%3D990%26doi%3D10.1016%2Fj.drudis.2011.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <i>Drug Design</i>; <span class="NLM_publisher-name">Spektrum</span>: <span class="NLM_publisher-loc">Berlin Heidelberg</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1007%2F978-3-642-17907-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=G.+Klebe&title=Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-17907-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-17907-5%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26btitle%3DDrug%2520Design%26pub%3DSpektrum%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Thermodynamics of Protein-Ligand Interactions as a Reference for Computational Analysis: How to Assess Accuracy, Reliability and Relevance of Experimental Data</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">867</span>– <span class="NLM_lpage">883</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9867-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1007%2Fs10822-015-9867-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=26376645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=867-883&author=S.+G.+Krimmerauthor=G.+Klebe&title=Thermodynamics+of+Protein-Ligand+Interactions+as+a+Reference+for+Computational+Analysis%3A+How+to+Assess+Accuracy%2C+Reliability+and+Relevance+of+Experimental+Data&doi=10.1007%2Fs10822-015-9867-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamics of protein-ligand interactions as a reference for computational analysis: how to assess accuracy, reliability and relevance of experimental data</span></div><div class="casAuthors">Krimmer, Stefan G.; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">867-883</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">For a conscientious interpretation of thermodn. parameters (Gibbs free energy, enthalpy and entropy) obtained by isothermal titrn. calorimetry (ITC), it is necessary to first evaluate the exptl. setup and conditions at which the data were measured.  The data quality must be assessed and the precision and accuracy of the measured parameters must be estd.  This information provides the basis at which level discussion of the data is appropriate, and allows insight into the significance of comparisons with other data.  The aim of this article is to provide the reader with basic understanding of the ITC technique and the exptl. practices commonly applied, in order to foster an appreciation for how much measured thermodn. parameters can deviate from ideal, error-free values.  Particular attention is paid to the shape of the recorded isotherm (c-value), the influence of the applied buffer used for the reaction (protonation reactions, pH), the chosen exptl. settings (temp.), impurities of protein and ligand, sources of systematic errors (soln. concn., soln. activity, and device calibration) and to the applied anal. software.  Furthermore, we comment on enthalpy-entropy compensation, heat capacities and van't Hoff enthalpies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0ngUaTMg_hbVg90H21EOLACvtfcHk0lhiwz4a1jat6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtr3O&md5=39a35e4588c4571925aa4af25f3254f9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9867-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9867-y%26sid%3Dliteratum%253Aachs%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DThermodynamics%2520of%2520Protein-Ligand%2520Interactions%2520as%2520a%2520Reference%2520for%2520Computational%2520Analysis%253A%2520How%2520to%2520Assess%2520Accuracy%252C%2520Reliability%2520and%2520Relevance%2520of%2520Experimental%2520Data%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D867%26epage%3D883%26doi%3D10.1007%2Fs10822-015-9867-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Betz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulsdorf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Impact of Surface Water Layers on Protein-Ligand Binding: How Well Are Experimental Data Reproduced by Molecular Dynamics Simulations in a Thermolysin Test Case?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGiur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=223-233&author=M.+Betzauthor=T.+Wulsdorfauthor=S.+G.+Krimmerauthor=G.+Klebe&title=Impact+of+Surface+Water+Layers+on+Protein-Ligand+Binding%3A+How+Well+Are+Experimental+Data+Reproduced+by+Molecular+Dynamics+Simulations+in+a+Thermolysin+Test+Case%3F&doi=10.1021%2Facs.jcim.5b00621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Surface Water Layers on Protein-Ligand Binding: How Well Are Experimental Data Reproduced by Molecular Dynamics Simulations in a Thermolysin Test Case?</span></div><div class="casAuthors">Betz, Michael; Wulsdorf, Tobias; Krimmer, Stefan G.; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">223-233</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug binding involves changes of the local water structure around proteins including water rearrangements across surface-solvation layers around protein and ligand portions exposed to the newly formed complex surface.  For a series of thermolysin-binding phosphonamidates, we discovered that variations of the partly exposed P2'-substituents modulate binding affinity up to 10 kJ mol-1 with even larger enthalpy/entropy partitioning of the binding signature.  The obsd. profiles cannot be completely explained by desolvation effects.  Instead, the quality and completeness of the surface water network wrapping around the formed complexes provide an explanation for the obsd. structure-activity relationship.  We used mol. dynamics to compute surface water networks and predict solvation sites around the complexes.  A fairly good correspondence with exptl. difference electron densities in high-resoln. crystal structures is achieved; in detail some problems with the potentials were discovered.  Charge-assisted contacts to waters appeared as exaggerated by AMBER, and stabilizing contributions of water-to-Me contacts were underestimated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-XbUxbQlrB7Vg90H21EOLACvtfcHk0lhiwz4a1jat6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGiur3F&md5=cf47671297d1fe1fa8b31ef170397ce6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00621%26sid%3Dliteratum%253Aachs%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DWulsdorf%26aufirst%3DT.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DImpact%2520of%2520Surface%2520Water%2520Layers%2520on%2520Protein-Ligand%2520Binding%253A%2520How%2520Well%2520Are%2520Experimental%2520Data%2520Reproduced%2520by%2520Molecular%2520Dynamics%2520Simulations%2520in%2520a%2520Thermolysin%2520Test%2520Case%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D223%26epage%3D233%26doi%3D10.1021%2Facs.jcim.5b00621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span> <span> </span><span class="NLM_article-title">Just Add Water! The Effect of Water on the Specificity of Protein- Ligand Binding Sites and Its Potential Application to Drug Design</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">973</span>– <span class="NLM_lpage">980</span>, <span class="refDoi"> DOI: 10.1016/S1074-5521(96)90164-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS1074-5521%2896%2990164-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=9000013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=973-980&author=J.+E.+Ladbury&title=Just+Add+Water%21+The+Effect+of+Water+on+the+Specificity+of+Protein-+Ligand+Binding+Sites+and+Its+Potential+Application+to+Drug+Design&doi=10.1016%2FS1074-5521%2896%2990164-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Just add water! The effect of water on the specificity of protein-ligand binding sites and its potential application to drug design</span></div><div class="casAuthors">Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">973-980</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">A review with 39 refs.  Recent data have highlighted the enigmatic role that water plays in biomol. complexes.  Water at the interface of a complex can increase the promiscuity of an interaction, yet it can also provide increased specificity and affinity.  The ability to engineer water-binding sites into the interface between a drug and its target might prove useful in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJqyhDhRviwbVg90H21EOLACvtfcHk0lhavEs9L11UYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D&md5=8eb3242b84897de4bb61e20ec6bc8d54</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2896%2990164-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252896%252990164-7%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DJust%2520Add%2520Water%2521%2520The%2520Effect%2520of%2520Water%2520on%2520the%2520Specificity%2520of%2520Protein-%2520Ligand%2520Binding%2520Sites%2520and%2520Its%2520Potential%2520Application%2520to%2520Drug%2520Design%26jtitle%3DChem.%2520Biol.%26date%3D1996%26volume%3D3%26spage%3D973%26epage%3D980%26doi%3D10.1016%2FS1074-5521%2896%2990164-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levy, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onuchic, J. N.</span></span> <span> </span><span class="NLM_article-title">Water Mediation in Protein Folding and Molecular Recognition</span>. <i>Annu. Rev. Biophys. Biomol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">415</span>, <span class="refDoi"> DOI: 10.1146/annurev.biophys.35.040405.102134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1146%2Fannurev.biophys.35.040405.102134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16689642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFSlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2006&pages=389-415&author=Y.+Levyauthor=J.+N.+Onuchic&title=Water+Mediation+in+Protein+Folding+and+Molecular+Recognition&doi=10.1146%2Fannurev.biophys.35.040405.102134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Water mediation in protein folding and molecular recognition</span></div><div class="casAuthors">Levy, Yaakov; Onuchic, Jose N.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">389-415</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Water is essential for life in many ways, and without it biomols. might no longer truly be biomols.  In particular, water is important to the structure, stability, dynamics, and function of biol. macromols.  In protein folding, water mediates the collapse of the chain and the search for the native topol. through a funneled energy landscape.  Water actively participates in mol. recognition by mediating the interactions between binding partners and contributes to either enthalpic or entropic stabilization.  Accordingly, water must be included in recognition and structure prediction codes to capture specificity.  Thus, water should not be treated as an inert environment, but rather as an integral and active component of biomol. systems, where it has both dynamic and structural roles.  Focusing on water sheds light on the physics and function of biol. machinery and self-assembly and may advance the understanding of the natural design of proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV8LpRDAZZZLVg90H21EOLACvtfcHk0lhavEs9L11UYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFSlu7c%253D&md5=3c2402d22d7c2fae5d1bccd9fb60a5b7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.35.040405.102134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.35.040405.102134%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DY.%26aulast%3DOnuchic%26aufirst%3DJ.%2BN.%26atitle%3DWater%2520Mediation%2520in%2520Protein%2520Folding%2520and%2520Molecular%2520Recognition%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2006%26volume%3D35%26spage%3D389%26epage%3D415%26doi%3D10.1146%2Fannurev.biophys.35.040405.102134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guvench, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, C. L.</span></span> <span> </span><span class="NLM_article-title">Receptor Rigidity and Ligand Mobility in Trypsin-Ligand Complexes</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1002/prot.20326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2Fprot.20326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=15578663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1Omtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2005&pages=407-417&author=O.+Guvenchauthor=D.+J.+Priceauthor=C.+L.+Brooks&title=Receptor+Rigidity+and+Ligand+Mobility+in+Trypsin-Ligand+Complexes&doi=10.1002%2Fprot.20326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor rigidity and ligand mobility in trypsin-ligand complexes</span></div><div class="casAuthors">Guvench, Olgun; Price, Daniel J.; Brooks, Charles L., III</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">407-417</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The trypsin-like serine proteases comprise a structurally similar family of proteins with a wide diversity of biol. functions.  Members of this family play roles in digestion, hemostasis, immune responses, and cancer metastasis.  Bovine trypsin is an archetypical member of this family that has been extensively characterized both functionally and structurally, and that preferentially hydrolyzes Arg/Lys-Xaa peptide bonds.  The authors have used mol. dynamics (MD) simulations to study bovine trypsin complexed with the two noncovalent small-mol. ligands, benzamidine and tranylcypromine, that have the same hydrogen-bond donating moieties as Arg and Lys side-chains, resp.  Multiple (10) simulations ranging from 1 ns to 2.2 ns, with explicit water mols. and periodic boundary conditions, were performed.  The simulations reveal that the trypsin binding pocket residues are relatively rigid regardless of whether there is no ligand, a high-affinity ligand (benzamidine), or a low-affinity ligand (tranylcypromine).  The thermal av. of the conformations sampled by benzamidine bound to trypsin is planar and consistent with the planar internal geometry of the benzamidine crystallog. model coordinates.  However, the most probable bound benzamidine conformations are ± 25° out of plane, implying that the obsd. x-ray electron d. represents an av. of densities from two mirror sym., nonplanar conformations.  Solvated benzamidine has free energy min. at ± 45°, and the induction of a more planar geometry upon binding is assocd. with ∼1 kcal/mol of intramol. strain.  Tranylcypromine's hydrogen-bonding pattern in the MD differs substantially from that inferred from the x-ray electron d.  Early in simulations of this system, tranylcypromine adopts an alternative binding conformation, changing from the crystallog. conformation, with a direct hydrogen bond between its amino moiety and the backbone oxygen of Gly-219, to one having a bridging water mol.  This result is consistently seen with the CHARMM22, Amber, or OPLS-AA force fields.  The trypsin-tranylcypromine hydrogen-bonding pattern obsd. in the simulations also occurs as the crystallog. binding mode of the Lys-15 side-chain of bovine pancreatic trypsin inhibitor bound to trypsin.  In this latter cocrystal, a bridging crystallog. water does reside between the side-chain's amino group and the trypsin Gly-219 backbone oxygen.  Furthermore, the trypsin-tranylcypromine simulations sample two different stable noncrystallog. binding poses.  These data suggest that some of the electron d. ascribed to tranylcypromine in the x-ray model is rather due to a bound water mol., and that multiple tranylcypromine binding conformations (crystallog. disorder) may be the cause of ambiguous electron d.  The combined trypsin-benzamidine and trypsin-tranylcypromine results highlight the ability of simulations to augment protein-ligand complex structural data by deconvoluting the effects of thermal and structural averaging, and by finding energetically optimal ligand and bound water positions for weakly bound ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh054GN8do9rVg90H21EOLACvtfcHk0lhavEs9L11UYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1Omtw%253D%253D&md5=0e5fff622c2c3b02d39ad70606e0bf52</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fprot.20326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20326%26sid%3Dliteratum%253Aachs%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26atitle%3DReceptor%2520Rigidity%2520and%2520Ligand%2520Mobility%2520in%2520Trypsin-Ligand%2520Complexes%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2005%26volume%3D58%26spage%3D407%26epage%3D417%26doi%3D10.1002%2Fprot.20326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Z. H.</span></span> <span> </span><span class="NLM_article-title">Selectivity of Neutral/Weakly Basic P1 Group Inhibitors of Thrombin and Trypsin by a Molecular Dynamics Study</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">8704</span>– <span class="NLM_lpage">8714</span>, <span class="refDoi"> DOI: 10.1002/chem.200800277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2Fchem.200800277" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=18680136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1OltrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=8704-8714&author=E.+L.+Wuauthor=K.+L.+Hanauthor=J.+Z.+H.+Zhang&title=Selectivity+of+Neutral%2FWeakly+Basic+P1+Group+Inhibitors+of+Thrombin+and+Trypsin+by+a+Molecular+Dynamics+Study&doi=10.1002%2Fchem.200800277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity of neutral/weakly basic P1 group inhibitors of thrombin and trypsin by a molecular dynamics study</span></div><div class="casAuthors">Wu, Emilia L.; Han, KeLi; Zhang, John Z. H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8704-8714</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations followed by mol. mechanics generalized Born surface area (MM-GBSA) analyses have been carried out to study the selectivity of two neutral and weakly basic P1 group inhibitors (177 and CDA) to thrombin and trypsin.  Detailed binding free energies between these inhibitors and individual protein residues are calcd. by using a per-residue basis decompn. method.  The anal. of the detailed interaction energies provides insight on the protein-inhibitor-binding mechanism and helps to elucidate the basis for achieving selectivity through interpretation of the structural and energetic results from the simulations.  The study shows that the dominant factor of selectivity for both inhibitors is van der Waals energy, which suggests better shape complementarity and packing with thrombin.  Nonpolar solvation free energy and total entropy contribution are also in favor of selectivity, but the contributions are much smaller.  Binding mode and structural anal. show that 177 binds to thrombin and trypsin in a similar binding mode.  In contrast, the CDA binds to thrombin and trypsin in very different modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoUaN5f4UltbVg90H21EOLACvtfcHk0lg9w0iOatwkkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1OltrnP&md5=dfb0aac1de205082e166f2d02ccbdf58</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fchem.200800277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200800277%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DE.%2BL.%26aulast%3DHan%26aufirst%3DK.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%2BZ.%2BH.%26atitle%3DSelectivity%2520of%2520Neutral%252FWeakly%2520Basic%2520P1%2520Group%2520Inhibitors%2520of%2520Thrombin%2520and%2520Trypsin%2520by%2520a%2520Molecular%2520Dynamics%2520Study%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2008%26volume%3D14%26spage%3D8704%26epage%3D8714%26doi%3D10.1002%2Fchem.200800277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilpert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banner, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gast, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadváry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Waterbeemd, H.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Potent and Highly Selective Thrombin Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3889</span>– <span class="NLM_lpage">3901</span>, <span class="refDoi"> DOI: 10.1021/jm00049a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00049a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2MXhs12hsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=3889-3901&author=K.+Hilpertauthor=J.+Ackermannauthor=D.+W.+Bannerauthor=A.+Gastauthor=K.+Gubernatorauthor=P.+Hadv%C3%A1ryauthor=L.+Lablerauthor=K.+M%C3%BCllerauthor=G.+Schmidauthor=T.+B.+Tschoppauthor=H.+van+de+Waterbeemd&title=Design+and+Synthesis+of+Potent+and+Highly+Selective+Thrombin+Inhibitors&doi=10.1021%2Fjm00049a008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Potent and Highly Selective Thrombin Inhibitors</span></div><div class="casAuthors">Hilpert, Kurt; Ackermann, Jean; Banner, David W.; Gast, Alain; Gubernator, Klaus; Hadvary, Paul; Labler, Ludvik; Mueller, Klaus; Schmid, Gerard; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3889-901</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Thrombin, a serine protease, plays a central role in the initiation and propagation of thrombotic events.  An extensive search for new thrombin inhibitors was performed, using an unconventional approach.  Screening of small basic mols. for binding in the recognition pocket of thrombin led to the discovery of (aminoiminomethyl)piperidine (amidinopiperidine) as a weak, but intrinsically selective, thrombin inhibitor.  Elaboration of this mol. provided compds. which inhibit thrombin with Ki's in the range of 20-50 nM and with selectivities of 1000-4000 against trypsin.  These inhibitor compds. show a new and unexpected binding mode to thrombin.  Modification of the central building block and then of one of the hydrophobic substituents led to the discovery of a new family of thrombin inhibitors which has reverted to the former binding mode to thrombin.  This last class of compds. shows inhibitory activities in the picomolar range, low toxicity, and a short plasma half life which favors its use for an i.v. application.  From this series of thrombin inhibitors, Ro 46-6240 (I) was selected for clin. development as an antithrombotic agent for i.v. administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcpB1l-ixNgbVg90H21EOLACvtfcHk0lg9w0iOatwkkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhs12hsbc%253D&md5=d1c8819926099edb6e538b8e32060e3f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm00049a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00049a008%26sid%3Dliteratum%253Aachs%26aulast%3DHilpert%26aufirst%3DK.%26aulast%3DAckermann%26aufirst%3DJ.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DGast%26aufirst%3DA.%26aulast%3DGubernator%26aufirst%3DK.%26aulast%3DHadv%25C3%25A1ry%26aufirst%3DP.%26aulast%3DLabler%26aufirst%3DL.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DSchmid%26aufirst%3DG.%26aulast%3DTschopp%26aufirst%3DT.%2BB.%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Potent%2520and%2520Highly%2520Selective%2520Thrombin%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D3889%26epage%3D3901%26doi%3D10.1021%2Fjm00049a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprengeler, P. A.</span></span> <span> </span><span class="NLM_article-title">Exploiting Subsite S1 of Trypsin-like Serine Proteases for Selectivity: Potent and Selective Inhibitors of Urokinase-Type Plasminogen Activator</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">3856</span>– <span class="NLM_lpage">3871</span>, <span class="refDoi"> DOI: 10.1021/jm010244+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010244%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlWrt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3856-3871&author=R.+L.+Mackmanauthor=B.+A.+Katzauthor=J.+G.+Breitenbucherauthor=H.+C.+Huiauthor=E.+Vernerauthor=C.+Luongauthor=L.+Liuauthor=P.+A.+Sprengeler&title=Exploiting+Subsite+S1+of+Trypsin-like+Serine+Proteases+for+Selectivity%3A+Potent+and+Selective+Inhibitors+of+Urokinase-Type+Plasminogen+Activator&doi=10.1021%2Fjm010244%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Subsite S1 of Trypsin-Like Serine Proteases for Selectivity: Potent and Selective Inhibitors of Urokinase-Type Plasminogen Activator</span></div><div class="casAuthors">Mackman, Richard L.; Katz, Bradley A.; Breitenbucher, J. Guy; Hui, Hon C.; Verner, Erik; Luong, Christine; Liu, Liang; Sprengeler, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3856-3871</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A nonselective inhibitor of trypsin-like serine proteases, 2-(2-hydroxybiphenyl-3-yl)-1H-indole-5-carboxamidine (I) (Verner, E.; Katz, B. A.; Spencer, J.; Allen, D.; Hataye, J.; Hruzewicz, W.; Hui, H. C.; Kolesnikov, A.; Li, Y.; Luong, C.; Martelli, A.; Radika. K.; Rai, R.; She, M.; Shrader, W.; Sprengeler, P. A.; Trapp, S.; Wang, J.; Young, W. B.; Mackman, R. L. J. Med. Chem. 2001, 44, 2753-2771) has been optimized through minor structural changes on the S1 binding group to afford remarkably selective and potent inhibitors of urokinase-type plasminogen activator (uPA).  The trypsin-like serine proteases1 that comprise drug targets can be broadly categorized into two subfamilies, those with Ser190 and those with Ala190.  A single-atom modification, for example, replacement of hydrogen for chlorine at the 6-position of the 5-amidinoindole P1 group on I, generated ≤6700-fold selectivity toward the Ser190 enzymes and against the Ala190 enzymes.  The larger chlorine atom displaces a water mol. (H2O1S1) that binds near residue 190 in all the complexes of I, and related inhibitors, in uPA, thrombin, and trypsin.  The water mol., H2O1S1, in both the Ser190 or Ala190 enzymes, hydrogen bonds with the amidine N1 nitrogen of the inhibitor.  When it is displaced, a redn. in affinity toward the Ala190 enzymes is obsd. due to the amidine N1 nitrogen of the bound inhibitor being deprived of a key hydrogen-bonding partner.  In the Ser190 enzymes the affinity is maintained since the serine hydroxyl oxygen OγSer190 compensates for the displaced water mol.  High-resoln. crystallog. provided evidence for the displacement of the water mol. and validated the design rationale.  In summation, a novel and powerful method for engineering selectivity toward Ser190 proteases and against Ala190 proteases without substantially increasing mol. wt. is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZsTVtV-cArVg90H21EOLACvtfcHk0lg9w0iOatwkkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlWrt7Y%253D&md5=d431091b463a9168c4e7223b7f96f460</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm010244%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010244%252B%26sid%3Dliteratum%253Aachs%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DKatz%26aufirst%3DB.%2BA.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26aulast%3DHui%26aufirst%3DH.%2BC.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DLuong%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26atitle%3DExploiting%2520Subsite%2520S1%2520of%2520Trypsin-like%2520Serine%2520Proteases%2520for%2520Selectivity%253A%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Urokinase-Type%2520Plasminogen%2520Activator%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3856%26epage%3D3871%26doi%3D10.1021%2Fjm010244%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sichler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfner, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopetzki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, H.</span></span> <span> </span><span class="NLM_article-title">The Influence of Residue 190 in the S1 Site of Trypsin-like Serine Proteases on Substrate Selectivity Is Universally Conserved</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>530</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(02)03495-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0014-5793%2802%2903495-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=12387896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvVOrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=530&publication_year=2002&pages=220-224&author=K.+Sichlerauthor=K.+P.+Hopfnerauthor=E.+Kopetzkiauthor=R.+Huberauthor=W.+Bodeauthor=H.+Brandstetter&title=The+Influence+of+Residue+190+in+the+S1+Site+of+Trypsin-like+Serine+Proteases+on+Substrate+Selectivity+Is+Universally+Conserved&doi=10.1016%2FS0014-5793%2802%2903495-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of residue 190 in the S1 site of trypsin-like serine proteases on substrate selectivity is universally conserved</span></div><div class="casAuthors">Sichler, Katrin; Hopfner, Karl-Peter; Kopetzki, Erhard; Huber, Robert; Bode, Wolfram; Brandstetter, Hans</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">530</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">220-224</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We examd. the influence of Ser/Ala190 in the S1 site on P1 substrate selectivity in several serine proteases.  The impact of residue 190 on the selectivity was const., regardless of differences in original selectivity or reactivity.  Substrate binding in S1 was optimized in all wild-type enzymes, while the effects on kcat depended on the combination of residue 190 and substrate.  Mutagenesis of residue 190 did not affect the S2-S4 sites.  Pronounced selectivity for arginine residues was coupled with low enzymic activity, in particular in recombinant factor IXa.  This is due to the dominance of the S1-P1 interaction over substrate binding in the S2-S4 sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZMSRLAdkTjbVg90H21EOLACvtfcHk0liiuk5wc-RT1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvVOrsL4%253D&md5=c18d833220d626ad03e2abed9efd00db</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2802%2903495-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252802%252903495-6%26sid%3Dliteratum%253Aachs%26aulast%3DSichler%26aufirst%3DK.%26aulast%3DHopfner%26aufirst%3DK.%2BP.%26aulast%3DKopetzki%26aufirst%3DE.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DBrandstetter%26aufirst%3DH.%26atitle%3DThe%2520Influence%2520of%2520Residue%2520190%2520in%2520the%2520S1%2520Site%2520of%2520Trypsin-like%2520Serine%2520Proteases%2520on%2520Substrate%2520Selectivity%2520Is%2520Universally%2520Conserved%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D530%26spage%3D220%26epage%3D224%26doi%3D10.1016%2FS0014-5793%2802%2903495-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lobo, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wos, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, W. B.</span></span> <span> </span><span class="NLM_article-title">Active Site Studies of Human Thrombin and Bovine Trypsin: Peptide Substrates</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1016/0003-9861(76)90433-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2F0003-9861%2876%2990433-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1033732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaE28XlvVyhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=1976&pages=235-244&author=A.+P.+Loboauthor=J.+D.+Wosauthor=S.+M.+Yuauthor=W.+B.+Lawson&title=Active+Site+Studies+of+Human+Thrombin+and+Bovine+Trypsin%3A+Peptide+Substrates&doi=10.1016%2F0003-9861%2876%2990433-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Active site studies of human thrombin and bovine trypsin:  peptide substrates</span></div><div class="casAuthors">Lobo, Angelo P.; Wos, J. David; Yu, Sherman M.; Lawson, William B.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-44</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">Twenty-four oligopeptides modeled after the N-terminal portion of the α(A)-chain of human fibrinogen were synthesized and tested as substrates for human thrombin and bovine trypsin.  The peptides contained either an Arg-Gly bond, or an Arg-Val bond, or both.  Glycine and glutamic acid were substituted at various positions within the peptides, and from the kcat/Km for each peptide with thrombin and with trypsin, the importance of residues on both sides of the Arg-Gly bond was evaluated.  The trypsin-catalyzed hydrolysis was faster than the thrombin-catalyzed hydrolysis of the peptides with glutamic acid at various distances N-terminal to the Arg-Gly bond.  The ratio of the thrombic rate to the tryptic rate was fastest when glutamic acid was 6 residues away.  Thrombin selectively cleaved the Arg-Gly bond in Ac-Val-Arg-Gly-Pro-Arg-Val-OH but cleaved both arginyl bonds rapidly in Ac-Val-Arg-Gly-Pro-Arg-Val-OMe.  Trypsin selectively cleaved an Arg-Val bond in Ac-Arg-Gly-Pro-Arg-Val-OMe.  The results are discussed in the light of thrombic cleavages of proteins.  Most of these are seen to occur at highly polar sequences that frequently contain a proline residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGZnePSk20jLVg90H21EOLACvtfcHk0liiuk5wc-RT1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XlvVyhu70%253D&md5=abd825a4ea3a9cc76a90d79065eff73e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2F0003-9861%2876%2990433-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-9861%252876%252990433-1%26sid%3Dliteratum%253Aachs%26aulast%3DLobo%26aufirst%3DA.%2BP.%26aulast%3DWos%26aufirst%3DJ.%2BD.%26aulast%3DYu%26aufirst%3DS.%2BM.%26aulast%3DLawson%26aufirst%3DW.%2BB.%26atitle%3DActive%2520Site%2520Studies%2520of%2520Human%2520Thrombin%2520and%2520Bovine%2520Trypsin%253A%2520Peptide%2520Substrates%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1976%26volume%3D177%26spage%3D235%26epage%3D244%26doi%3D10.1016%2F0003-9861%2876%2990433-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallwitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enoksson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellman, L.</span></span> <span> </span><span class="NLM_article-title">The Extended Cleavage Specificity of Human Thrombin</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e31756</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0031756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1371%2Fjournal.pone.0031756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=22384068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFylsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=M.+Gallwitzauthor=M.+Enokssonauthor=M.+Thorpeauthor=L.+Hellman&title=The+Extended+Cleavage+Specificity+of+Human+Thrombin&doi=10.1371%2Fjournal.pone.0031756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The extended cleavage specificity of human thrombin</span></div><div class="casAuthors">Gallwitz, Maike; Enoksson, Mattias; Thorpe, Michael; Hellman, Lars</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e31756</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Thrombin is one of the most extensively studied of all proteases.  Its central role in the coagulation cascade as well as several other areas has been thoroughly documented.  Despite this, its consensus cleavage site has never been detd. in detail.  Here we have detd. its extended substrate recognition profile using phage-display technol.  The consensus recognition sequence was identified as, P2-Pro, P1-Arg, P1'-Ser/Ala/Gly/Thr, P2'-not acidic and P3'-Arg.  Our anal. also identifies an important role for a P3'-arginine in thrombin substrates lacking a P2-proline.  In order to study kinetics of this cooperative or additive effect we developed a system for insertion of various pre-selected cleavable sequences in a linker region between two thioredoxin mols.  Using this system we show that mutations of P2-Pro and P3'-Arg lead to an approx. 20-fold and 14-fold redn., resp. in the rate of cleavage.  Mutating both Pro and Arg results in a drop in cleavage of 200-400 times, which highlights the importance of these two positions for maximal substrate cleavage.  Interestingly, no natural substrates display the obtained consensus sequence but represent sequences that show only 1-30 % of the optimal cleavage rate for thrombin.  This clearly indicates that maximal cleavage, excluding the help of exosite interactions, is not always desired, which may instead cause problems with dysregulated coagulation.  It is likely exosite cooperativity has a central role in detg. the specificity and rate of cleavage of many of these in vivo substrates.  Major effects on cleavage efficiency were also obsd. for residues as far away as 4 amino acids from the cleavage site.  Insertion of an aspartic acid in position P4 resulted in a drop in cleavage by a factor of almost 20 times.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLokncNWnNKbVg90H21EOLACvtfcHk0liiuk5wc-RT1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsFylsLc%253D&md5=ed3355e1e0762bf9618781f4a0ed0b7c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0031756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0031756%26sid%3Dliteratum%253Aachs%26aulast%3DGallwitz%26aufirst%3DM.%26aulast%3DEnoksson%26aufirst%3DM.%26aulast%3DThorpe%26aufirst%3DM.%26aulast%3DHellman%26aufirst%3DL.%26atitle%3DThe%2520Extended%2520Cleavage%2520Specificity%2520of%2520Human%2520Thrombin%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0031756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zayed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">More than a Simple Lipophilic Contact: A Detailed Thermodynamic Analysis of Nonbasic Residues in the S1 Pocket of Thrombin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.04.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2009.04.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19409395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Glu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=56-69&author=B.+Baumauthor=M.+Mohamedauthor=M.+Zayedauthor=C.+Gerlachauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=More+than+a+Simple+Lipophilic+Contact%3A+A+Detailed+Thermodynamic+Analysis+of+Nonbasic+Residues+in+the+S1+Pocket+of+Thrombin&doi=10.1016%2Fj.jmb.2009.04.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">More than a Simple Lipophilic Contact: A Detailed Thermodynamic Analysis of Nonbasic Residues in the S1 Pocket of Thrombin</span></div><div class="casAuthors">Baum, Bernhard; Mohamed, Menshawy; Zayed, Mohamed; Gerlach, Christof; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-69</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The field of medicinal chem. aims to design and optimize small mol. leads into drug candidates that may pos. interfere with pathol. disease situations in humans or combat the growth of infective pathogens.  From the plethora of crystal structures of protein-inhibitor complexes we have learned how mols. recognize each other geometrically, but we still have rather superficial understanding of why they bind to each other.  This contribution surveys a series of 26 thrombin inhibitors with small systematic structural differences to elucidate the rationale for their widely deviating binding affinity from 185 μM to 4 nM as recorded by enzyme kinetic measurements.  Five well-resolved (resoln. 2.30-1.47 Å) crystal structures of thrombin-inhibitor complexes and an apo-structure of the uncomplexed enzyme (1.50 Å) are correlated with thermodn. data recorded by isothermal titrn. calorimetry with 12 selected inhibitors from the series.  Taking soly. data into account, the variation in physicochem. properties allows conclusions to be reached about the relative importance of the enthalpic binding features as well as to est. the importance of the parameters more difficult to capture, such as residual ligand entropy and desolvation properties.  The collected data reveal a comprehensive picture of the thermodn. signature that explains the so far poorly understood attractive force experienced by m-chloro-benzylamides to thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvKms7QZtnUrVg90H21EOLACvtfcHk0lhKA1rAgXODYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Glu7c%253D&md5=ce3cf23e3a9246fc5084643b9a5a7ab4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.04.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.04.051%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMohamed%26aufirst%3DM.%26aulast%3DZayed%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DC.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DMore%2520than%2520a%2520Simple%2520Lipophilic%2520Contact%253A%2520A%2520Detailed%2520Thermodynamic%2520Analysis%2520of%2520Nonbasic%2520Residues%2520in%2520the%2520S1%2520Pocket%2520of%2520Thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D390%26spage%3D56%26epage%3D69%26doi%3D10.1016%2Fj.jmb.2009.04.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freindorf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D. G.</span></span> <span> </span><span class="NLM_article-title">Enhancement of Hydrophobic Interactions and Hydrogen Bond Strength by Cooperativity: Synthesis, Modeling and Molecular Dynamics Simulations of a Congeneric Series of Thrombin Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2135</span>, <span class="refDoi"> DOI: 10.1021/jm9016416</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9016416" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2126-2135&author=L.+Muleyauthor=B.+Baumauthor=M.+Smolinskiauthor=M.+Freindorfauthor=A.+Heineauthor=G.+Klebeauthor=D.+G.+Hangauer&title=Enhancement+of+Hydrophobic+Interactions+and+Hydrogen+Bond+Strength+by+Cooperativity%3A+Synthesis%2C+Modeling+and+Molecular+Dynamics+Simulations+of+a+Congeneric+Series+of+Thrombin+Inhibitors&doi=10.1021%2Fjm9016416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancement of Hydrophobic Interactions and Hydrogen Bond Strength by Cooperativity: Synthesis, Modeling, and Molecular Dynamics Simulations of a Congeneric Series of Thrombin Inhibitors</span></div><div class="casAuthors">Muley, Laveena; Baum, Bernhard; Smolinski, Michael; Freindorf, Marek; Heine, Andreas; Klebe, Gerhard; Hangauer, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2126-2135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurately predicting the binding affinity of ligands to their receptors by computational methods is one of the major challenges in structure-based drug design.  One of the potentially significant errors in these predictions is the common assumption that the ligand binding affinity contributions of noncovalent interactions are additive.  Herein we present data obtained from two sep. series of thrombin inhibitors contg. hydrophobic side chains of increasing size that bind in the S3 pocket and with, or without, an adjacent amine that engages in a hydrogen bond with Gly 216.  The first series of inhibitors has a m-chlorobenzyl moiety binding in the S1 pocket, and the second has a benzamidine moiety.  When the adjacent hydrogen bond is present, the enhanced binding affinity per Å2 of hydrophobic contact surface in the S3 pocket improves by 75% and 59%, resp., over the inhibitors lacking this hydrogen bond.  This improvement of the binding affinity per Å2 demonstrates cooperativity between the hydrophobic interaction and the hydrogen bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqytMqjMe1i9bVg90H21EOLACvtfcHk0lhKA1rAgXODYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFKjs7o%253D&md5=62e4474f7d6d5fa11f884c5b1e17fef9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm9016416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9016416%26sid%3Dliteratum%253Aachs%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DFreindorf%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DHangauer%26aufirst%3DD.%2BG.%26atitle%3DEnhancement%2520of%2520Hydrophobic%2520Interactions%2520and%2520Hydrogen%2520Bond%2520Strength%2520by%2520Cooperativity%253A%2520Synthesis%252C%2520Modeling%2520and%2520Molecular%2520Dynamics%2520Simulations%2520of%2520a%2520Congeneric%2520Series%2520of%2520Thrombin%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2126%26epage%3D2135%26doi%3D10.1021%2Fjm9016416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khayat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Impact of Ligand and Protein Desolvation on Ligand Binding to the S1 Pocket of Thrombin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>418</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.01.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2012.01.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=22366545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=418&publication_year=2012&pages=350-366&author=A.+Bielaauthor=M.+Khayatauthor=H.+Tanauthor=J.+Kongauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=Impact+of+Ligand+and+Protein+Desolvation+on+Ligand+Binding+to+the+S1+Pocket+of+Thrombin&doi=10.1016%2Fj.jmb.2012.01.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Ligand and Protein Desolvation on Ligand Binding to the S1 Pocket of Thrombin</span></div><div class="casAuthors">Biela, A.; Khayat, M.; Tan, H.; Kong, J.; Heine, A.; Hangauer, D.; Klebe, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">418</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">350-366</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In the present study, we investigate the impact of a tightly bound water mol. on ligand binding in the S1 pocket of thrombin.  The S1 pocket contains a deeply buried deprotonated aspartate residue (Asp189) which due to its charged state, is well hydrated in the uncomplexed state.  We systematically studied the importance of this water mol. by evaluating a series of ligands that contains pyridine-type P1 side chains that could potentially alter the binding properties of this water mol.  All of the pyridine derivs. retain the original hydration state albeit sometimes with a slight perturbance.  In order to prevent a direct H-bond formation with Asp189, and to create a permanent pos. charge on the P1 side chain that is positioned adjacent to the Asp189 carboxylate anion, we methylated the pyridine nitrogen.  This methylation resulted in displacement of water but was accompanied by a loss in binding affinity.  Quantum chem. calcns. of the ligand solvation free energy showed that the pos. charged methylpyridinium derivs. suffer a large penalty of desolvation upon binding.  Consequently, they have a substantially less favorable enthalpy of binding.  In addn. to the ligand desolvation penalty, the hydration shell around Asp189 has to be overcome, which is achieved in nearly all pyridinium derivs.  Only for the ortho deriv. is a partial population of a water next to Asp189 found.  Possibly, the gain of electrostatic interactions between the charged P1 side chain and Asp189 helps to compensate for the desolvation penalty.  In all uncharged pyridine derivs., the solvation shell remains next to Asp189, partly mediating interactions between ligand and protein.  In the case of the para-pyridine deriv., a strongly disordered cluster of water sites is obsd. between ligand and Asp189.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq7O_AkRSsXbVg90H21EOLACvtfcHk0ljCulEoIJkXEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFSmsL8%253D&md5=f52d76d0ef544bede3145c0ea9026d06</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.01.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.01.054%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DKhayat%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DImpact%2520of%2520Ligand%2520and%2520Protein%2520Desolvation%2520on%2520Ligand%2520Binding%2520to%2520the%2520S1%2520Pocket%2520of%2520Thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D418%26spage%3D350%26epage%3D366%26doi%3D10.1016%2Fj.jmb.2012.01.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardell, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freidinger, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor-Olsen, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramjit, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woltmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dancheck, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hare, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulichak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J. P.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel, Selective, and Orally Bioavailable Class of Thrombin Inhibitors Incorporating Aminopyridyl Moieties at the P1 Position</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">3726</span>– <span class="NLM_lpage">3733</span>, <span class="refDoi"> DOI: 10.1021/jm970493r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970493r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXmslWhs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=3726-3733&author=D.+M.+Fengauthor=S.+J.+Gardellauthor=S.+D.+Lewisauthor=M.+G.+Bockauthor=Z.+Chenauthor=R.+M.+Freidingerauthor=A.+M.+Naylor-Olsenauthor=H.+G.+Ramjitauthor=R.+Woltmannauthor=E.+P.+Baskinauthor=J.+J.+Lynchauthor=R.+Lucasauthor=J.+A.+Shaferauthor=K.+B.+Dancheckauthor=I-W.+Chenauthor=S.-S.+Maoauthor=J.+A.+Kruegerauthor=T.+R.+Hareauthor=A.+M.+Mulichakauthor=J.+P.+Vacca&title=Discovery+of+a+Novel%2C+Selective%2C+and+Orally+Bioavailable+Class+of+Thrombin+Inhibitors+Incorporating+Aminopyridyl+Moieties+at+the+P1+Position&doi=10.1021%2Fjm970493r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel, Selective, and Orally Bioavailable Class of Thrombin Inhibitors Incorporating Aminopyridyl Moieties at the P1 Position</span></div><div class="casAuthors">Feng, Dong-Mei; Gardell, Stephen J.; Lewis, S. Dale; Bock, Mark G.; Chen, Zhongguo; Freidinger, Roger M.; Naylor-Olsen, Adel; Ramjit, Harri G.; Woltmann, Richard; Baskin, Elizabeth P.; Lynch, Joseph J.; Lucas, Robert; Shafer, Jules A.; Dancheck, Kimberley B.; Chen, I-Wu; Mao, Shi-Shan; Krueger, Julie A.; Hare, Timothy R.; Mulichak, Anne M.; Vacca, Joseph P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3726-3733</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel class of thrombin inhibitors incorporating aminopyridyl moieties at the P1 position has been discovered.  Four of these thrombin inhibitors showed nanomolar potency (Ki 0.8-12 nM), 300-1500-fold selectivity for thrombin compared with trypsin, and good oral bioavailability (F = 40-76%) in rats or dogs.  The neutral P1 was expected to increase metabolic stability and oral absorption.  Identification of this novel aminopyridyl group at P1 was a key step in our search for a clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHTbN5pkVq9bVg90H21EOLACvtfcHk0ljCulEoIJkXEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslWhs7c%253D&md5=1ceb0ac6aa7e59570d4707c295c5ffcc</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm970493r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970493r%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DD.%2BM.%26aulast%3DGardell%26aufirst%3DS.%2BJ.%26aulast%3DLewis%26aufirst%3DS.%2BD.%26aulast%3DBock%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DFreidinger%26aufirst%3DR.%2BM.%26aulast%3DNaylor-Olsen%26aufirst%3DA.%2BM.%26aulast%3DRamjit%26aufirst%3DH.%2BG.%26aulast%3DWoltmann%26aufirst%3DR.%26aulast%3DBaskin%26aufirst%3DE.%2BP.%26aulast%3DLynch%26aufirst%3DJ.%2BJ.%26aulast%3DLucas%26aufirst%3DR.%26aulast%3DShafer%26aufirst%3DJ.%2BA.%26aulast%3DDancheck%26aufirst%3DK.%2BB.%26aulast%3DChen%26aufirst%3DI-W.%26aulast%3DMao%26aufirst%3DS.-S.%26aulast%3DKrueger%26aufirst%3DJ.%2BA.%26aulast%3DHare%26aufirst%3DT.%2BR.%26aulast%3DMulichak%26aufirst%3DA.%2BM.%26aulast%3DVacca%26aufirst%3DJ.%2BP.%26atitle%3DDiscovery%2520of%2520a%2520Novel%252C%2520Selective%252C%2520and%2520Orally%2520Bioavailable%2520Class%2520of%2520Thrombin%2520Inhibitors%2520Incorporating%2520Aminopyridyl%2520Moieties%2520at%2520the%2520P1%2520Position%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D3726%26epage%3D3733%26doi%3D10.1021%2Fjm970493r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konishi, Y.</span></span> <span> </span><span class="NLM_article-title">Potent Bivalent Thrombin Inhibitors: Replacement of the Scissile Peptide Bond at P1-P1′ with Arginyl Ketomethylene Isosteres</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">3109</span>– <span class="NLM_lpage">3115</span>, <span class="refDoi"> DOI: 10.1021/jm9807297</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9807297" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1MXktl2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=3109-3115&author=T.+Steinmetzerauthor=B.+Y.+Zhuauthor=Y.+Konishi&title=Potent+Bivalent+Thrombin+Inhibitors%3A+Replacement+of+the+Scissile+Peptide+Bond+at+P1-P1%E2%80%B2+with+Arginyl+Ketomethylene+Isosteres&doi=10.1021%2Fjm9807297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Bivalent Thrombin Inhibitors: Replacement of the Scissile Peptide Bond at P1-P1' with Arginyl Ketomethylene Isosteres</span></div><div class="casAuthors">Steinmetzer, Torsten; Zhu, Bing Yan; Konishi, Yasuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3109-3115</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have designed highly potent synthetic bivalent thrombin inhibitors, which consist of an active site blocking segment, a fibrinogen recognition exosite blocking segment, and a linker connecting these segments.  The bivalent inhibitors bind to the active site and the fibrinogen recognition exosite simultaneously.  As a result, the inhibitors showed much higher affinity for thrombin than the individual blocking segments.  Various arginyl ketomethylene isosteres ArgΨ[CO-CH2-X]P1' were incorporated into the bivalent inhibitors as P1-P1' segment to eliminate the scissile bond.  The P1' residue is a natural or unnatural amino acid; specifically, the incorporation of mercaptoacetic acid exhibited superiority in synthesis and affinity for thrombin.  Inhibitor (D-cyclohexylalanine)-Pro-ArgΨ[CO-CH2-S]Gly-(Gly)4-Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-cyclohexylalanine-(D-Glu)-OH showed the lowest Ki value of 3.5×10-13 M, which is comparable to that (Ki = 2.3×10-13 M) of recombinant hirudin.  Consequently the authors successfully reduced the size of the inhibitor from ∼7 kDa of recombinant hirudin to ∼2 kDa without losing the affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdArz5KnXR4rVg90H21EOLACvtfcHk0ljCulEoIJkXEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktl2iuro%253D&md5=40f938bfe8dd7a1196a73c988111d607</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm9807297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9807297%26sid%3Dliteratum%253Aachs%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DB.%2BY.%26aulast%3DKonishi%26aufirst%3DY.%26atitle%3DPotent%2520Bivalent%2520Thrombin%2520Inhibitors%253A%2520Replacement%2520of%2520the%2520Scissile%2520Peptide%2520Bond%2520at%2520P1-P1%25E2%2580%25B2%2520with%2520Arginyl%2520Ketomethylene%2520Isosteres%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D3109%26epage%3D3115%26doi%3D10.1021%2Fjm9807297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brodbeck, B.</span>; <span class="NLM_string-name">Nettekoven, M. H.</span></span> <span> </span><span class="NLM_article-title">Substituted[1,2,4]Triazolo[1,5A]Pyridine Derivatives with Activity as Adenosine Receptor Ligands</span>. U.S. Patent <span class="NLM_patent">US6,506,772 B1</span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=B.+Brodbeck&author=M.+H.+Nettekoven&title=Substituted%5B1%2C2%2C4%5DTriazolo%5B1%2C5A%5DPyridine+Derivatives+with+Activity+as+Adenosine+Receptor+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBrodbeck%26aufirst%3DB.%26atitle%3DSubstituted%255B1%252C2%252C4%255DTriazolo%255B1%252C5A%255DPyridine%2520Derivatives%2520with%2520Activity%2520as%2520Adenosine%2520Receptor%2520Ligands%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meredith, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beattie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capparelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPietro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosagrahara, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jewell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemon, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinsey, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagratis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plato, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozhitskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soldermann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Eis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monovich, L. G.</span></span> <span> </span><span class="NLM_article-title">Identification of Potent and Selective Amidobipyridyl Inhibitors of Protein Kinase D</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5422</span>– <span class="NLM_lpage">5438</span>, <span class="refDoi"> DOI: 10.1021/jm100076w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100076w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5422-5438&author=E.+L.+Meredithauthor=K.+Beattieauthor=R.+Burgisauthor=M.+Capparelliauthor=J.+Chapoauthor=L.+DiPietroauthor=G.+Gamberauthor=I.+Enyedyauthor=D.+B.+Hoodauthor=V.+Hosagraharaauthor=C.+Jewellauthor=K.+A.+Kochauthor=W.+Leeauthor=D.+D.+Lemonauthor=T.+A.+McKinseyauthor=K.+Mirandaauthor=N.+Pagratisauthor=D.+Phanauthor=C.+Platoauthor=C.+Raoauthor=O.+Rozhitskayaauthor=N.+Soldermannauthor=C.+Springerauthor=M.+van+Eisauthor=R.+B.+Vegaauthor=W.+Yanauthor=Q.+Zhuauthor=L.+G.+Monovich&title=Identification+of+Potent+and+Selective+Amidobipyridyl+Inhibitors+of+Protein+Kinase+D&doi=10.1021%2Fjm100076w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent and Selective Amidobipyridyl Inhibitors of Protein Kinase D</span></div><div class="casAuthors">Meredith, Erik L.; Beattie, Kimberly; Burgis, Robin; Capparelli, Michael; Chapo, Joseph; Di Pietro, Lucian; Gamber, Gabriel; Enyedy, Istvan; Hood, David B.; Hosagrahara, Vinayak; Jewell, Charles; Koch, Keith A.; Lee, Wendy; Lemon, Douglas D.; McKinsey, Timothy A.; Miranda, Karl; Pagratis, Nikos; Phan, Dillon; Plato, Craig; Rao, Chang; Rozhitskaya, Olga; Soldermann, Nicolas; Springer, Clayton; van Eis, Maurice; Vega, Richard B.; Yan, Wanlin; Zhu, Qingming; Monovich, Lauren G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5422-5438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and biol. evaluation of potent and selective PKD (protein kinase D) inhibitors, e.g. I, are described herein.  The compds. described in the present study selectively inhibit PKD among other putative HDAC (Histone deacetylase) kinases.  The PKD inhibitors of the present study blunt phosphorylation and subsequent nuclear export of HDAC4/5 in response to diverse agonists.  These compds. further establish the central role of PKD as an HDAC4/5 kinase and enhance the current understanding of cardiac myocyte signal transduction.  The in vivo efficacy of a representative example compd. on heart morphol. is reported herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1cpMoyvA7arVg90H21EOLACvtfcHk0ljiYtvp-uiHOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCiurw%253D&md5=ac3ffbd2c87b9a952858de20ddd10a02</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm100076w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100076w%26sid%3Dliteratum%253Aachs%26aulast%3DMeredith%26aufirst%3DE.%2BL.%26aulast%3DBeattie%26aufirst%3DK.%26aulast%3DBurgis%26aufirst%3DR.%26aulast%3DCapparelli%26aufirst%3DM.%26aulast%3DChapo%26aufirst%3DJ.%26aulast%3DDiPietro%26aufirst%3DL.%26aulast%3DGamber%26aufirst%3DG.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DHood%26aufirst%3DD.%2BB.%26aulast%3DHosagrahara%26aufirst%3DV.%26aulast%3DJewell%26aufirst%3DC.%26aulast%3DKoch%26aufirst%3DK.%2BA.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DLemon%26aufirst%3DD.%2BD.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26aulast%3DMiranda%26aufirst%3DK.%26aulast%3DPagratis%26aufirst%3DN.%26aulast%3DPhan%26aufirst%3DD.%26aulast%3DPlato%26aufirst%3DC.%26aulast%3DRao%26aufirst%3DC.%26aulast%3DRozhitskaya%26aufirst%3DO.%26aulast%3DSoldermann%26aufirst%3DN.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3Dvan%2BEis%26aufirst%3DM.%26aulast%3DVega%26aufirst%3DR.%2BB.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DQ.%26aulast%3DMonovich%26aufirst%3DL.%2BG.%26atitle%3DIdentification%2520of%2520Potent%2520and%2520Selective%2520Amidobipyridyl%2520Inhibitors%2520of%2520Protein%2520Kinase%2520D%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5422%26epage%3D5438%26doi%3D10.1021%2Fjm100076w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, S. C.</span></span> <span> </span><span class="NLM_article-title">6,6′-Dibromo-4,4′-Di(Hexoxymethyl)-2,2′-Bipyridine: A New Solubilizing Building Block for Macromolecular and Supramolecular Applications</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">4696</span>– <span class="NLM_lpage">4699</span>, <span class="refDoi"> DOI: 10.1021/jo060557i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo060557i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFKks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=4696-4699&author=C.+M.+Ambauthor=S.+C.+Rasmussen&title=6%2C6%E2%80%B2-Dibromo-4%2C4%E2%80%B2-Di%28Hexoxymethyl%29-2%2C2%E2%80%B2-Bipyridine%3A+A+New+Solubilizing+Building+Block+for+Macromolecular+and+Supramolecular+Applications&doi=10.1021%2Fjo060557i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">6,6'-Dibromo-4,4'-di(hexoxymethyl)-2,2'- bipyridine: A New Solubilizing Building Block for Macromolecular and Supramolecular Applications</span></div><div class="casAuthors">Amb, Chad M.; Rasmussen, Seth C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4696-4699</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although brominated bipyridines and terpyridines are highly desirable synthetic building blocks for both ligand design and macro- or supramol. applications, few such synthetic precursors have been reported that include much-needed solubilizing groups.  Reported here is an inexpensive route to 2,6-dibromo-4-(hexoxymethyl)pyridine from citrazinic acid with an overall yield of 44% and its efficient conversion (60%) to 6,6'-dibromo-4,4'-di(hexoxymethyl)-2,2'-bipyridine via oxidative coupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4FlTfLDwparVg90H21EOLACvtfcHk0ljiYtvp-uiHOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFKks7c%253D&md5=fe25ff28da15705d0171a43c7eea3cf4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjo060557i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo060557i%26sid%3Dliteratum%253Aachs%26aulast%3DAmb%26aufirst%3DC.%2BM.%26aulast%3DRasmussen%26aufirst%3DS.%2BC.%26atitle%3D6%252C6%25E2%2580%25B2-Dibromo-4%252C4%25E2%2580%25B2-Di%2528Hexoxymethyl%2529-2%252C2%25E2%2580%25B2-Bipyridine%253A%2520A%2520New%2520Solubilizing%2520Building%2520Block%2520for%2520Macromolecular%2520and%2520Supramolecular%2520Applications%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2006%26volume%3D71%26spage%3D4696%26epage%3D4699%26doi%3D10.1021%2Fjo060557i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariza, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urpí, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viladomat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilarrasa, J.</span></span> <span> </span><span class="NLM_article-title">One-Pot Conversion of Azides to Boc-Protected Amines with Trimethylphosphine and Boc-ON</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">9101</span>– <span class="NLM_lpage">9102</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(98)02006-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0040-4039%2898%2902006-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1cXnsFOrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1998&pages=9101-9102&author=X.+Arizaauthor=F.+Urp%C3%ADauthor=C.+Viladomatauthor=J.+Vilarrasa&title=One-Pot+Conversion+of+Azides+to+Boc-Protected+Amines+with+Trimethylphosphine+and+Boc-ON&doi=10.1016%2FS0040-4039%2898%2902006-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">One-pot conversion of azides to Boc-protected amines with trimethylphosphine and Boc-ON</span></div><div class="casAuthors">Ariza, Zavier; Urpi, Felix; Viladomat, Carles; Vilarrasa, Jaume</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">9101-9102</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Reaction of azides with trimethylphosphine followed by addn. of 2-(tert-butoxycarbonyloxyimino)-2-phenylacetonitrile (Boc-ON) at -20°C and stirring at r.t. for 5 h affords the desired Boc-amines in 87-100% yields.  Similar yields are obtained by mixing all components together (azide, Me3P, and Boc-ON) in toluene or THF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWPk82CKPvxrVg90H21EOLACvtfcHk0ljiYtvp-uiHOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnsFOrtbY%253D&md5=400516373fe743f7077fe96a8212b542</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2898%2902006-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252898%252902006-1%26sid%3Dliteratum%253Aachs%26aulast%3DAriza%26aufirst%3DX.%26aulast%3DUrp%25C3%25AD%26aufirst%3DF.%26aulast%3DViladomat%26aufirst%3DC.%26aulast%3DVilarrasa%26aufirst%3DJ.%26atitle%3DOne-Pot%2520Conversion%2520of%2520Azides%2520to%2520Boc-Protected%2520Amines%2520with%2520Trimethylphosphine%2520and%2520Boc-ON%26jtitle%3DTetrahedron%2520Lett.%26date%3D1998%26volume%3D39%26spage%3D9101%26epage%3D9102%26doi%3D10.1016%2FS0040-4039%2898%2902006-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariza, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urpí, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilarrasa, J.</span></span> <span> </span><span class="NLM_article-title">A Practical Procedure for the Preparation of Carbamates from Azides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">7515</span>– <span class="NLM_lpage">7517</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(99)01449-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0040-4039%2899%2901449-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1MXmslemtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1999&pages=7515-7517&author=X.+Arizaauthor=F.+Urp%C3%ADauthor=J.+Vilarrasa&title=A+Practical+Procedure+for+the+Preparation+of+Carbamates+from+Azides&doi=10.1016%2FS0040-4039%2899%2901449-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A practical procedure for the preparation of carbamates from azides</span></div><div class="casAuthors">Ariza, Xavier; Urpi, Felix; Vilarrasa, Jaume</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">7515-7517</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A practical procedure for the direct conversion of azides to carbamates has been found.  High yields are obtained when primary and secondary aliph. azides, as well as α-azido esters, are treated in THF with Me3P and several chloroformates (ClCO2Bn, ClCO2Me, ClCO2Et, ClCO2CH2CCl3, ClCO2CH2CH:CH2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodReSdQbslq7Vg90H21EOLACvtfcHk0ljvLy7IMipjoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmslemtb8%253D&md5=6def48e9ec2d6fd54ba0add21786311b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2899%2901449-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252899%252901449-5%26sid%3Dliteratum%253Aachs%26aulast%3DAriza%26aufirst%3DX.%26aulast%3DUrp%25C3%25AD%26aufirst%3DF.%26aulast%3DVilarrasa%26aufirst%3DJ.%26atitle%3DA%2520Practical%2520Procedure%2520for%2520the%2520Preparation%2520of%2520Carbamates%2520from%2520Azides%26jtitle%3DTetrahedron%2520Lett.%26date%3D1999%26volume%3D40%26spage%3D7515%26epage%3D7517%26doi%3D10.1016%2FS0040-4039%2899%2901449-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elmkaddem, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmeister, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J.-L.</span></span> <span> </span><span class="NLM_article-title">Efficient Synthesis of Aminopyridine Derivatives by Copper Catalyzed Amination Reactions</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1039/B916569J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1039%2FB916569J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20107652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFOisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2010&pages=925-927&author=M.+K.+Elmkaddemauthor=C.+Fischmeisterauthor=C.+M.+Thomasauthor=J.-L.+Renaud&title=Efficient+Synthesis+of+Aminopyridine+Derivatives+by+Copper+Catalyzed+Amination+Reactions&doi=10.1039%2FB916569J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient synthesis of aminopyridine derivatives by copper catalyzed amination reactions</span></div><div class="casAuthors">Elmkaddem, Mohammed K.; Fischmeister, Cedric; Thomas, Christophe M.; Renaud, Jean-Luc</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">925-927</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A copper(I) catalyzed amination reaction utilizing aq. ammonia and operating under mild conditions is presented.  This method was employed for the efficient synthesis of various aminopyridine derivs. bearing electron withdrawing and electron donating groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSFanPaj2mzrVg90H21EOLACvtfcHk0ljvLy7IMipjoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFOisrg%253D&md5=4a2759fea5464db1ae392168bf0993c7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FB916569J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB916569J%26sid%3Dliteratum%253Aachs%26aulast%3DElmkaddem%26aufirst%3DM.%2BK.%26aulast%3DFischmeister%26aufirst%3DC.%26aulast%3DThomas%26aufirst%3DC.%2BM.%26aulast%3DRenaud%26aufirst%3DJ.-L.%26atitle%3DEfficient%2520Synthesis%2520of%2520Aminopyridine%2520Derivatives%2520by%2520Copper%2520Catalyzed%2520Amination%2520Reactions%26jtitle%3DChem.%2520Commun.%26date%3D2010%26volume%3D46%26spage%3D925%26epage%3D927%26doi%3D10.1039%2FB916569J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Charges Shift Protonation: Neutron Diffraction Reveals That Aniline and 2-Aminopyridine Become Protonated Upon Binding to Trypsin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4887</span>– <span class="NLM_lpage">4890</span>, <span class="refDoi"> DOI: 10.1002/anie.201701038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2Fanie.201701038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=4887-4890&author=J.+Schiebelauthor=R.+Gaspariauthor=A.+Sandnerauthor=K.+Ngoauthor=H.+D.+Gerberauthor=A.+Cavalliauthor=A.+Ostermannauthor=A.+Heineauthor=G.+Klebe&title=Charges+Shift+Protonation%3A+Neutron+Diffraction+Reveals+That+Aniline+and+2-Aminopyridine+Become+Protonated+Upon+Binding+to+Trypsin&doi=10.1002%2Fanie.201701038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Charges shift protonation: Neutron diffraction reveals that aniline and 2-aminopyridine become protonated upon binding to trypsin</span></div><div class="casAuthors">Schiebel, Johannes; Gaspari, Roberto; Sandner, Anna; Ngo, Khang; Gerber, Hans-Dieter; Cavalli, Andrea; Ostermann, Andreas; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4887-4890</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">H atoms play a key role in protein-ligand recognition.  They det. the quality of established H-bonding networks and define the protonation of bound ligands.  Structural visualization of H atoms by x-ray crystallog. is rarely possible.  Here, we used neutron diffraction to det. the positions of the H atoms in the ligands, aniline and 2-aminopyridine, bound to the archetypical serine protease, trypsin.  The resulting structures showed the best resoln. so far achieved for proteins larger than 100 residues and allowed an accurate description of the protonation states and interactions with nearby water mols.  Despite its low pKa of 4.6 and a large distance of 3.6 Å to the charged Asp-189 residue at the bottom of the trypsin S1 pocket, the amino group of aniline became protonated, whereas in 2-aminopyridine, the pyridine N atom picked up the proton although its NH2 group was 1.6 Å closer to Asp-189.  Therefore, apart from charge-charge distances, tautomer stability was decisive for the resulting binding poses, an aspect that is pivotal for predicting correct binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg449p0i3o-7Vg90H21EOLACvtfcHk0ljvLy7IMipjoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D&md5=a48df0f5959e03a0fdfc528de58bbb6f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201701038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201701038%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DSandner%26aufirst%3DA.%26aulast%3DNgo%26aufirst%3DK.%26aulast%3DGerber%26aufirst%3DH.%2BD.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DCharges%2520Shift%2520Protonation%253A%2520Neutron%2520Diffraction%2520Reveals%2520That%2520Aniline%2520and%25202-Aminopyridine%2520Become%2520Protonated%2520Upon%2520Binding%2520to%2520Trypsin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D4887%26epage%3D4890%26doi%3D10.1002%2Fanie.201701038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span> <span> </span><span class="NLM_article-title">Enthalpy and Heat Capacity Changes for the Proton Dissociation of Various Buffer Components in 0.1 M Potassium Chloride</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19981101)33:2<159::AID-PROT2>3.0.CO;2-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=9779785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1cXmsFOnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1998&pages=159-166&author=H.+Fukadaauthor=K.+Takahashi&title=Enthalpy+and+Heat+Capacity+Changes+for+the+Proton+Dissociation+of+Various+Buffer+Components+in+0.1+M+Potassium+Chloride&doi=10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Enthalpy and heat capacity changes for the proton dissociation of various buffer components in 0.1 M potassium chloride</span></div><div class="casAuthors">Fukada, Harumi; Takahashi, Katsutada</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-166</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Enthalpy and heat capacity changes for the deprotonation of 18 buffers were calorimetrically detd. in 0.1 M potassium chloride at temps. ranging from 5 to 45°.  The values of the dissocn. const. were also detd. by means of potentiometric titrn.  The enthalpy changes for the deprotonation of buffers, except for the phosphate and glycerol 2-phosphate buffers, were found to be characterized by a linear function of temp.  The enthalpy changes for the second dissocn. of phosphate and glycerol 2-phosphate where divalent anion is formed on dissocn. were fitted with the second order function of temp. rather than the first order.  Temp. dependence of buffer pH calcd. by using the enthalpy and heat capacity changes obtained was in good agreement with the temp. variation of the pH values actually measured in the temp. range between 0 and 50° for all the buffers studied.  On the basis of the results obtained, a numeric table showing the temp. dependence of pK values for the 18 buffers is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjY9pGcfJG_rVg90H21EOLACvtfcHk0ljhiB-yDfu99g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsFOnsr4%253D&md5=4ee32c30437c670b02595822767697c0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819981101%252933%253A2%253C159%253A%253AAID-PROT2%253E3.0.CO%253B2-E%26sid%3Dliteratum%253Aachs%26aulast%3DFukada%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DK.%26atitle%3DEnthalpy%2520and%2520Heat%2520Capacity%2520Changes%2520for%2520the%2520Proton%2520Dissociation%2520of%2520Various%2520Buffer%2520Components%2520in%25200.1%2520M%2520Potassium%2520Chloride%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D1998%26volume%3D33%26spage%3D159%26epage%3D166%26doi%3D10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennen, R. M.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic Quantities for the Ionization Reactions of Buffers</span>. <i>J. Phys. Chem. Ref. Data</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1063/1.1416902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1063%2F1.1416902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Oisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2002&pages=231-370&author=R.+N.+Goldbergauthor=N.+Kishoreauthor=R.+M.+Lennen&title=Thermodynamic+Quantities+for+the+Ionization+Reactions+of+Buffers&doi=10.1063%2F1.1416902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic quantities for the ionization reactions of buffers</span></div><div class="casAuthors">Goldberg, Robert N.; Kishore, Nand; Lennen, Rebecca M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical and Chemical Reference Data</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-370</span>CODEN:
                <span class="NLM_cas:coden">JPCRBU</span>;
        ISSN:<span class="NLM_cas:issn">0047-2689</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">This review contains selected values of thermodn. quantities for the aq. ionization reactions of 64 buffers, many of which are used in biol. research.  Since the aim is to be able to predict values of the ionization const. at temps. not too far from ambient, the thermodn. quantities which are tabulated are the pK, std. molar Gibbs energy ΔrG°, std. molar enthalpy ΔrH°, and std. molar heat capacity change ΔrCp° for each of the ionization reactions at the temp. T = 298.15 K and the pressure p = 0.1 MPa.  The std. state is the hypothetical ideal soln. of unit molality.  The chem. name(s) and CAS registry no., structure, empirical formula, and mol. wt. are given for each buffer considered herein.  The selection of the values of the thermodn. quantities for each buffer is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCt3UuqJMfxbVg90H21EOLACvtfcHk0ljhiB-yDfu99g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Oisrg%253D&md5=9c95e655c39245ac228ddbca075a1ca3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1063%2F1.1416902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1416902%26sid%3Dliteratum%253Aachs%26aulast%3DGoldberg%26aufirst%3DR.%2BN.%26aulast%3DKishore%26aufirst%3DN.%26aulast%3DLennen%26aufirst%3DR.%2BM.%26atitle%3DThermodynamic%2520Quantities%2520for%2520the%2520Ionization%2520Reactions%2520of%2520Buffers%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Ref.%2520Data%26date%3D2002%26volume%3D31%26spage%3D231%26epage%3D370%26doi%3D10.1063%2F1.1416902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dullweber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürzebecher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Factorising Ligand Affinity: A Combined Thermodynamic and Crystallographic Study of Trypsin and Thrombin Inhibition</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1006/jmbi.2001.5062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1006%2Fjmbi.2001.5062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=11676542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnslCrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2001&pages=593-614&author=F.+Dullweberauthor=M.+T.+Stubbsauthor=D.+Musilauthor=J.+St%C3%BCrzebecherauthor=G.+Klebe&title=Factorising+Ligand+Affinity%3A+A+Combined+Thermodynamic+and+Crystallographic+Study+of+Trypsin+and+Thrombin+Inhibition&doi=10.1006%2Fjmbi.2001.5062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Factorising Ligand Affinity: A Combined Thermodynamic and Crystallographic Study of Trypsin and Thrombin Inhibition</span></div><div class="casAuthors">Dullweber, Frank; Stubbs, Milton T.; Musil, Dorde; Stuerzebecher, Joerg; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">593-614</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The binding of a series of low mol. wt. ligands towards trypsin and thrombin has been studied by isothermal titrn. calorimetry and protein crystallog.  In a series of congeneric ligands, surprising changes of protonation states occur and are overlaid on the binding process.  They result from induced pKa shifts depending on the local environment experienced by the ligand and protein functional groups in the complex (induced dielec. fit).  They involve addnl. heat effects that must be cor. before any conclusion on the binding enthalpy (ΔH) and entropy (ΔS) can be drawn.  After correction, trends in both contributions can be interpreted in structural terms with respect to the hydrogen bond inventory or residual ligand motions.  For all inhibitors studied, a strong neg. heat capacity change (ΔCp) is detected, thus binding becomes more exothermic and entropically less favorable with increasing temp.  Due to a mutual compensation, Gibbs free energy remains virtually unchanged.  The strong neg. ΔCp value cannot solely be explained by the removal of hydrophobic surface portions of the protein or ligand from water exposure.  Addnl. contributions must be considered, presumably arising from modulations of the local water structure, changes in vibrational modes or other ordering parameters.  For thrombin, smaller neg. ΔCp values are obsd. for ligand binding in the presence of sodium ions compared to the other alkali ions, probably due to stabilizing effects on the protein or changes in the bound water structure.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGortayuL9mRLbVg90H21EOLACvtfcHk0ljhiB-yDfu99g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnslCrsbk%253D&md5=59a21189828b9dcdcf90f8c8eb906621</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.5062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.5062%26sid%3Dliteratum%253Aachs%26aulast%3DDullweber%26aufirst%3DF.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DSt%25C3%25BCrzebecher%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFactorising%2520Ligand%2520Affinity%253A%2520A%2520Combined%2520Thermodynamic%2520and%2520Crystallographic%2520Study%2520of%2520Trypsin%2520and%2520Thrombin%2520Inhibition%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D313%26spage%3D593%26epage%3D614%26doi%3D10.1006%2Fjmbi.2001.5062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotriffer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Protonation Changes upon Ligand Binding to Trypsin and Thrombin: Structural Interpretation Based on p<i>K</i><sub>a</sub> Calculations and ITC Experiments</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>367</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1356</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2007.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=17316681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2007&pages=1347-1356&author=P.+Czodrowskiauthor=C.+A.+Sotrifferauthor=G.+Klebe&title=Protonation+Changes+upon+Ligand+Binding+to+Trypsin+and+Thrombin%3A+Structural+Interpretation+Based+on+pKa+Calculations+and+ITC+Experiments&doi=10.1016%2Fj.jmb.2007.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protonation Changes upon Ligand Binding to Trypsin and Thrombin: Structural Interpretation Based on pKa Calculations and ITC Experiments</span></div><div class="casAuthors">Czodrowski, Paul; Sotriffer, Christoph A.; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1356</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The protonation states of a protein and a ligand can be altered upon complex formation.  Such changes can be detected exptl. by isothermal titrn. calorimetry (ITC).  For a series of ligands binding to the serine proteases trypsin and thrombin, we previously performed an extensive ITC and crystallog. study and were able to identify protonation changes for four complexes.  However, since ITC measures only the overall proton exchange, it does not provide structural insights into the functional groups involved in the proton transfer.  Using Poisson-Boltzmann calcns. based on our recently developed PEOE_PB charges, we compute pKa values for all complexes of our former study in order to reveal the residues with altered protonation states.  The results indicate that His57, a member of the catalytic triad, is responsible for the most relevant pKa shifts leading to the exptl. detected protonation changes.  This finding is in contrast to our previous assumption that the obsd. protonation changes occur at the carboxylic group of the ligands.  The newly detected proton acceptor is used for a revised factorization of the ITC data, which is necessary whenever the protonation inventory changes upon complexation.  The pK a values of complexes showing no protonation change in the ITC expt. are reliably predicted in most cases, whereas predictions of strongly coupled systems remain problematic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzQ9tO1GIWD7Vg90H21EOLACvtfcHk0lgELZL5Lrv3uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKitLk%253D&md5=a451f9ac6fcd4a0efcdbc3616ec7849f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DSotriffer%26aufirst%3DC.%2BA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DProtonation%2520Changes%2520upon%2520Ligand%2520Binding%2520to%2520Trypsin%2520and%2520Thrombin%253A%2520Structural%2520Interpretation%2520Based%2520on%2520pKa%2520Calculations%2520and%2520ITC%2520Experiments%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D367%26spage%3D1347%26epage%3D1356%26doi%3D10.1016%2Fj.jmb.2007.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Think Twice: Understanding the High Potency of Bis(Phenyl)Methane Inhibitors of Thrombin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2009.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19520086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlWrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=2009&pages=552-564&author=B.+Baumauthor=L.+Muleyauthor=A.+Heineauthor=M.+Smolinskiauthor=D.+Hangauerauthor=G.+Klebe&title=Think+Twice%3A+Understanding+the+High+Potency+of+Bis%28Phenyl%29Methane+Inhibitors+of+Thrombin&doi=10.1016%2Fj.jmb.2009.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Think Twice: Understanding the High Potency of Bis(phenyl)methane Inhibitors of Thrombin</span></div><div class="casAuthors">Baum, Bernhard; Muley, Laveena; Heine, Andreas; Smolinski, Michael; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">552-564</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Successful design of potent and selective protein inhibitors, in terms of structure-based drug design, strongly relies on the correct understanding of the mol. features detg. the ligand binding to the target protein.  We present a case study of serine protease inhibitors with a bis(phenyl)methane moiety binding into the S3 pocket.  These inhibitors bind with remarkable potency to the active site of thrombin, the blood coagulation factor IIa.  A combination of x-ray crystallog. and isothermal titrn. calorimetry provides conclusive insights into the driving forces responsible for the surprisingly high potency of these inhibitors.  Anal. of six well-resolved crystal structures (resoln. 1.58-2.25 Å) along with the thermodn. data allows an explanation of the tight binding of the bis(phenyl)methane inhibitors.  Interestingly, the two Ph rings contribute to binding affinity for very different reasons - a fact that can only be elucidated by a structure-based approach.  The first Ph moiety occupies the hydrophobic S3 pocket, resulting in a mainly entropic advantage of binding.  This observation is based on the displacement of structural water mols. from the S3 pocket that are obsd. in complexes with inhibitors that do not bind in the S3 pocket.  The same classic hydrophobic effect cannot explain the enhanced binding affinity resulting from the attachment of the second, more solvent-exposed Ph ring.  For the bis(phenyl)methane inhibitors, an obsd. adaptive rotation of a glutamate residue adjacent to the S3 binding pocket attracted our attention.  The rotation of this glutamate into salt-bridging distance with a lysine moiety correlates with an enhanced enthalpic contribution to binding for these highly potent thrombin binders.  This explanation for the magnitude of the attractive force is confirmed by data retrieved by a Relibase search of several thrombin-inhibitor complexes deposited in the Protein Data Bank exhibiting similar mol. features.  Special attention was attributed to putative changes in the protonation states of the interaction partners.  For this purpose, two analogous inhibitors differing mainly in their potential to change the protonation state of a hydrogen-bond donor functionality were compared.  Buffer dependencies of the binding enthalpy assocd. with complex formation could be traced by isothermal titrn. calorimetry, which revealed, along with anal. of the crystal structures (resoln. 1.60 and 1.75 Å), that a virtually compensating proton interchange between enzyme, inhibitor and buffer is responsible for the obsd. buffer-independent thermodn. signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrayaqPbLFKe7Vg90H21EOLACvtfcHk0lgELZL5Lrv3uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlWrs7Y%253D&md5=2b6a2483a0a54b2dc30c622cf603cc55</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DThink%2520Twice%253A%2520Understanding%2520the%2520High%2520Potency%2520of%2520Bis%2528Phenyl%2529Methane%2520Inhibitors%2520of%2520Thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D391%26spage%3D552%26epage%3D564%26doi%3D10.1016%2Fj.jmb.2009.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Non-Additivity of Functional Group Contributions in Protein-Ligand Binding: A Comprehensive Study by Crystallography and Isothermal Titration Calorimetry</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>397</i></span>,  <span class="NLM_fpage">1042</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2010.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20156458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=2010&pages=1042-1054&author=B.+Baumauthor=L.+Muleyauthor=M.+Smolinskiauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=Non-Additivity+of+Functional+Group+Contributions+in+Protein-Ligand+Binding%3A+A+Comprehensive+Study+by+Crystallography+and+Isothermal+Titration+Calorimetry&doi=10.1016%2Fj.jmb.2010.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Non-additivity of Functional Group Contributions in Protein-Ligand Binding: A Comprehensive Study by Crystallography and Isothermal Titration Calorimetry</span></div><div class="casAuthors">Baum, Bernhard; Muley, Laveena; Smolinski, Michael; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">397</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1042-1054</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Additivity of functional group contributions to protein-ligand binding is a very popular concept in medicinal chem. as the basis of rational design and optimized lead structures.  Most of the currently applied scoring functions for docking build on such additivity models.  Even though the limitation of this concept is well known, case studies examg. in detail why additivity fails at the mol. level are still very scarce.  The present study shows, by use of crystal structure anal. and isothermal titrn. calorimetry for a congeneric series of thrombin inhibitors, that extensive cooperative effects between hydrophobic contacts and hydrogen bond formation are intimately coupled via dynamic properties of the formed complexes.  The formation of optimal lipophilic contacts with the surface of the thrombin S3 pocket and the full desolvation of this pocket can conflict with the formation of an optimal hydrogen bond between ligand and protein.  The mutual contributions of the competing interactions depend on the size of the ligand hydrophobic substituent and influence the residual mobility of ligand portions at the binding site.  Anal. of the individual crystal structures and factorizing the free energy into enthalpy and entropy demonstrates that binding affinity of the ligands results from a mixt. of enthalpic contributions from hydrogen bonding and hydrophobic contacts, and entropic considerations involving an increasing loss of residual mobility of the bound ligands.  This complex picture of mutually competing and partially compensating enthalpic and entropic effects dets. the non-additivity of free energy contributions to ligand binding at the mol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGousWs7tLpU3LVg90H21EOLACvtfcHk0lgELZL5Lrv3uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D&md5=2f61dde175a6ee27551179644eee3f00</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DNon-Additivity%2520of%2520Functional%2520Group%2520Contributions%2520in%2520Protein-Ligand%2520Binding%253A%2520A%2520Comprehensive%2520Study%2520by%2520Crystallography%2520and%2520Isothermal%2520Titration%2520Calorimetry%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D397%26spage%3D1042%26epage%3D1054%26doi%3D10.1016%2Fj.jmb.2010.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sielaff, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwesten, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding Stepwise Disrupts Water Network in Thrombin: Enthalpic and Entropic Changes Reveal Classical Hydrophobic Effect</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6094</span>– <span class="NLM_lpage">6110</span>, <span class="refDoi"> DOI: 10.1021/jm300337q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300337q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6094-6110&author=A.+Bielaauthor=F.+Sielaffauthor=F.+Terwestenauthor=A.+Heineauthor=T.+Steinmetzerauthor=G.+Klebe&title=Ligand+Binding+Stepwise+Disrupts+Water+Network+in+Thrombin%3A+Enthalpic+and+Entropic+Changes+Reveal+Classical+Hydrophobic+Effect&doi=10.1021%2Fjm300337q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Binding Stepwise Disrupts Water Network in Thrombin: Enthalpic and Entropic Changes Reveal Classical Hydrophobic Effect</span></div><div class="casAuthors">Biela, Adam; Sielaff, Frank; Terwesten, Felix; Heine, Andreas; Steinmetzer, Torsten; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6094-6110</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Well-ordered water mols. are displaced from thrombin's hydrophobic S3/4-pocket by P3-varied ligands (Gly, D-Ala, D-Val, D-Leu to D-Cha with increased hydrophobicity and steric requirement).  Two series with 2-(aminomethyl)-5-chlorobenzylamide and 4-amidinobenzylamide at P1 were examd. by ITC and crystallog.  Although experiencing different interactions in S1, they display almost equal potency.  For both scaffolds the terminal benzylsulfonyl substituent differs in binding, whereas the increasingly bulky P3-groups address S3/4 pocket similarly.  Small substituents leave the solvation pattern unperturbed as found in the uncomplexed enzyme while increasingly larger ones stepwise displace the waters.  Medium-sized groups show patterns with partially occupied waters.  The overall 40-fold affinity enhancement correlates with water displacement and growing no. of van der Waals contacts and is mainly attributed to favorable entropy.  Both Gly derivs. deviate from the series and adopt different binding modes.  Nonetheless, their thermodn. signatures are virtually identical with the homologous D-Ala derivs.  Accordingly, unchanged thermodn. profiles are no reliable indicator for conserved binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjEkdauKdcCrVg90H21EOLACvtfcHk0lhooqhthaR4uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt7g%253D&md5=7488723d7299d9b5233993436679ddac</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm300337q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300337q%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DSielaff%26aufirst%3DF.%26aulast%3DTerwesten%26aufirst%3DF.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DLigand%2520Binding%2520Stepwise%2520Disrupts%2520Water%2520Network%2520in%2520Thrombin%253A%2520Enthalpic%2520and%2520Entropic%2520Changes%2520Reveal%2520Classical%2520Hydrophobic%2520Effect%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6094%26epage%3D6110%26doi%3D10.1021%2Fjm300337q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sandner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüfner-Wulsdorf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Strategies for Late-Stage Optimization: Profiling Thermodynamics by Preorganization and Salt Bridge Shielding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9753</span>– <span class="NLM_lpage">9771</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2hurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9753-9771&author=A.+Sandnerauthor=T.+H%C3%BCfner-Wulsdorfauthor=A.+Heineauthor=T.+Steinmetzerauthor=G.+Klebe&title=Strategies+for+Late-Stage+Optimization%3A+Profiling+Thermodynamics+by+Preorganization+and+Salt+Bridge+Shielding&doi=10.1021%2Facs.jmedchem.9b01196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for Late-Stage Optimization: Profiling Thermodynamics by Preorganization and Salt Bridge Shielding</span></div><div class="casAuthors">Sandner, Anna; Huefner-Wulsdorf, Tobias; Heine, Andreas; Steinmetzer, Torsten; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9753-9771</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural fixation of a ligand in its bioactive conformation may, due to entropic reasons, improve affinity.  The authors present a congeneric series of thrombin ligands with a variety of functional groups triggering preorganization prior to binding.  Fixation in soln. and complex formation have been characterized by crystallog., ITC and MD simulations.  First, the authors show why these preorganizing modifications do not affect the overall binding mode and how key interactions are preserved.  Next, the authors demonstrate how preorganization thermodn. can be largely dominated by enthalpy, rather than entropy due to the significant population of low-energy conformations.  Furthermore, a salt bridge is shielded by actively reducing its surface exposure and thus, leading to an enhanced enthalpic binding profile.  The results suggest that the consideration of the ligand soln. ensemble by mol. dynamics simulation is necessary to predict preorganizing modifications that enhance the binding behavior of already promising binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQZ8FrLYcmBLVg90H21EOLACvtfcHk0lhooqhthaR4uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2hurrL&md5=211810b5c43718767916e3b1fcec6401</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01196%26sid%3Dliteratum%253Aachs%26aulast%3DSandner%26aufirst%3DA.%26aulast%3DH%25C3%25BCfner-Wulsdorf%26aufirst%3DT.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DStrategies%2520for%2520Late-Stage%2520Optimization%253A%2520Profiling%2520Thermodynamics%2520by%2520Preorganization%2520and%2520Salt%2520Bridge%2520Shielding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9753%26epage%3D9771%26doi%3D10.1021%2Facs.jmedchem.9b01196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Adding Calorimetric Data to Decision Making in Drug Development: A Hot Tip!</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1038/nrd3054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1038%2Fnrd3054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19960014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=23-27&author=J.+E.+Ladburyauthor=G.+Klebeauthor=E.+Freire&title=Adding+Calorimetric+Data+to+Decision+Making+in+Drug+Development%3A+A+Hot+Tip%21&doi=10.1038%2Fnrd3054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Adding calorimetric data to decision making in lead discovery: A hot tip</span></div><div class="casAuthors">Ladbury, John E.; Klebe, Gerhard; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-27</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recognition of the limitations of high-throughput screening approaches in the discovery of candidate drugs has reawakened interest in structure-based and other rational design methods.  Here, the authors describe how isothermal titrn. calorimetry can be used to obtain thermodn. data on the binding of compds. to protein targets.  The authors propose that these data - particularly the change in enthalpy - could provide a valuable, complementary addn. to established tools for selecting compds. in lead discovery and for aiding lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph3qvp2E8fbLVg90H21EOLACvtfcHk0lhooqhthaR4uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM&md5=e8446b6db983c35580e9010899eb40eb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd3054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3054%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DAdding%2520Calorimetric%2520Data%2520to%2520Decision%2520Making%2520in%2520Drug%2520Development%253A%2520A%2520Hot%2520Tip%2521%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D23%26epage%3D27%26doi%3D10.1038%2Fnrd3054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sigurskjold, B. W.</span></span> <span> </span><span class="NLM_article-title">Exact Analysis of Competition Ligand Binding by Displacement Isothermal Titration Calorimetry</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1006/abio.1999.4402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1006%2Fabio.1999.4402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10625516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhs1WmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2000&pages=260-266&author=B.+W.+Sigurskjold&title=Exact+Analysis+of+Competition+Ligand+Binding+by+Displacement+Isothermal+Titration+Calorimetry&doi=10.1006%2Fabio.1999.4402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Exact Analysis of Competition Ligand Binding by Displacement Isothermal Titration Calorimetry</span></div><div class="casAuthors">Sigurskjold, Bent W.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">260-266</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A rigorous method for the least-squares nonlinear regression anal. of displacement isothermal titrn. calorimetric data is presented.  The method can fit the binding isotherm of a ligand which is competitively inhibited in its binding by another bound ligand to a mol. with n identical and independent binding sites.  There are no other assumptions for the method and no approxns.  Anal. of previously published data of the strong binding of acarbose to glucoamylase is presented as an example.  The regression equations have been programmed for the Origin software supplied with the widely used titrn. calorimeters from Microcal, Inc., and an Origin Function Definition File with instructions is freely available from the author upon e-mail request.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsNy3023MELVg90H21EOLACvtfcHk0lg670DInr3nqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhs1WmsA%253D%253D&md5=3cce6711ca574bf075a56c297d996282</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fabio.1999.4402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1999.4402%26sid%3Dliteratum%253Aachs%26aulast%3DSigurskjold%26aufirst%3DB.%2BW.%26atitle%3DExact%2520Analysis%2520of%2520Competition%2520Ligand%2520Binding%2520by%2520Displacement%2520Isothermal%2520Titration%2520Calorimetry%26jtitle%3DAnal.%2520Biochem.%26date%3D2000%26volume%3D277%26spage%3D260%26epage%3D266%26doi%3D10.1006%2Fabio.1999.4402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turnbull, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daranas, A. H.</span></span> <span> </span><span class="NLM_article-title">On the Value of c: Can Low Affinity Systems Be Studied by Isothermal Titration Calorimetry?</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">14859</span>– <span class="NLM_lpage">14866</span>, <span class="refDoi"> DOI: 10.1021/ja036166s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja036166s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVSiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=14859-14866&author=W.+B.+Turnbullauthor=A.+H.+Daranas&title=On+the+Value+of+c%3A+Can+Low+Affinity+Systems+Be+Studied+by+Isothermal+Titration+Calorimetry%3F&doi=10.1021%2Fja036166s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">On the Value of c: Can Low Affinity Systems Be Studied by Isothermal Titration Calorimetry?</span></div><div class="casAuthors">Turnbull, W. Bruce; Daranas, Antonio H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">14859-14866</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) allows the detn. of ΔG°, ΔH°, and ΔS° from a single expt. and is thus widely used for studying binding thermodn. in both biol. and synthetic supramol. systems.  However, it is widely believed that it is not possible to derive accurate thermodn. information from ITC expts. in which the Wiseman "c" parameter (which is the product of the receptor concn. and the binding const., Ka) is less than ca. 10, constraining its use to high affinity systems.  Herein, exptl. titrns. and simulated data are used to demonstrate that this dogma is false, esp. for low affinity systems, assuming that (1) a sufficient portion of the binding isotherm is used for anal., (2) the binding stoichiometry is known, (3) the concns. of both ligand and receptor are known with accuracy, and (4) there is an adequate level of signal-to-noise in the data.  This study supports the validity of ITC for detg. the value of Ka and, hence, ΔG° from expts. conducted under low c conditions but advocates greater caution in the interpretation of values for ΔH°.  Therefore, isothermal titrn. calorimetry is a valid and useful technique for studying biol. and synthetically important low affinity systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6KxnjW2sZN7Vg90H21EOLACvtfcHk0lg670DInr3nqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVSiu7w%253D&md5=088d351b0836433e55b46e16189271e7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja036166s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja036166s%26sid%3Dliteratum%253Aachs%26aulast%3DTurnbull%26aufirst%3DW.%2BB.%26aulast%3DDaranas%26aufirst%3DA.%2BH.%26atitle%3DOn%2520the%2520Value%2520of%2520c%253A%2520Can%2520Low%2520Affinity%2520Systems%2520Be%2520Studied%2520by%2520Isothermal%2520Titration%2520Calorimetry%253F%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D14859%26epage%3D14866%26doi%3D10.1021%2Fja036166s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macgillivray, R. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Cera, E.</span></span> <span> </span><span class="NLM_article-title">Determinants of Specificity in Coagulation Proteases</span>. <i>J. Thromb. Haemostasis</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2401</span>– <span class="NLM_lpage">2408</span>, <span class="refDoi"> DOI: 10.1111/j.1538-7836.2005.01456.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1111%2Fj.1538-7836.2005.01456.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16241939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1KktLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=2401-2408&author=M.+J.+Pageauthor=R.+T.+A.+Macgillivrayauthor=E.+Di+Cera&title=Determinants+of+Specificity+in+Coagulation+Proteases&doi=10.1111%2Fj.1538-7836.2005.01456.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Determinants of specificity in coagulation proteases</span></div><div class="casAuthors">Page, M. J.; MacGillivray, R. T. A.; Di Cera, E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Thrombosis and Haemostasis</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2401-2408</span>CODEN:
                <span class="NLM_cas:coden">JTHOA5</span>;
        ISSN:<span class="NLM_cas:issn">1538-7933</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing, Inc.</span>)
        </div><div class="casAbstract">A review.  Proteases play diverse roles in a variety of essential biol. processes, both as nonspecific catalysts of protein degrdn. and as highly specific agents that control physiol. events.  Here, the authors review the mechanisms of substrate specificity employed by serine proteases and focus their discussion on blood-coagulation proteases.  The authors dissect the interplay between active site and exosite specificity and how substrate recognition is regulated allosterically by Na+ binding.  The authors also draw attention to a functional polarity that exists in the serine protease fold, which sheds light on the structural linkages between the active site and exosites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokX5M6DgS4WLVg90H21EOLACvtfcHk0lg670DInr3nqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1KktLzN&md5=a1a05ad79e24ceebdafdebf2a0638215</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1111%2Fj.1538-7836.2005.01456.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1538-7836.2005.01456.x%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BJ.%26aulast%3DMacgillivray%26aufirst%3DR.%2BT.%2BA.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DDeterminants%2520of%2520Specificity%2520in%2520Coagulation%2520Proteases%26jtitle%3DJ.%2520Thromb.%2520Haemostasis%26date%3D2005%26volume%3D3%26spage%3D2401%26epage%3D2408%26doi%3D10.1111%2Fj.1538-7836.2005.01456.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Cera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinto, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vindigni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayala, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulinsky, A.</span></span> <span> </span><span class="NLM_article-title">The Na<sup>+</sup> Binding Site of Thrombin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">22089</span>– <span class="NLM_lpage">22092</span>, <span class="refDoi"> DOI: 10.1074/jbc.270.38.22089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1074%2Fjbc.270.38.22089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=7673182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2MXot1Giu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=22089-22092&author=E.+Di%0ACeraauthor=E.+R.+Guintoauthor=A.+Vindigniauthor=Q.+D.+Dangauthor=Y.+M.+Ayalaauthor=M.+Wuyiauthor=A.+Tulinsky&title=The+Na%2B+Binding+Site+of+Thrombin&doi=10.1074%2Fjbc.270.38.22089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The Na+ binding site of thrombin</span></div><div class="casAuthors">Di Cera, Enrico; Guinto, Enriqueta R.; Vindigni, Alessandro; Dang, Quoc D.; Ayala, Youhna M.; Wuyi, Meng; Tulinsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">22089-92</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Thrombin is an allosteric serine protease existing in two forms, slow and fast, targeted toward anticoagulant and procoagulant activities.  The slow → fast transition is induced by Na+ binding to a site contained within a cylindrical cavity formed by three antiparallel β-strands of the B-chain (Met180-Tyr184a, Lys224-Tyr228, and Val213-Gly219) diagonally crossed by the Glu188-Glu192 strand.  The site is shaped further by the loop connecting the last two β-strands and is located more than 15 Å away from the catalytic triad.  The cavity traverses through thrombin from the active site to the opposite surface and contains Asp189 of the primary specificity site near its midpoint.  The bound Na+ is coordinated octahedrally by the carbonyl oxygen atoms of Tyr184a, Arg221a, and Lys224, and by three highly conserved water mols. in the D-Phe-Pro-Arg chloromethylketone thrombin.  The sequence in the Na+ binding loop is highly conserved in thrombin from 11 different species and is homologous to that found in other serine proteases involved in blood coagulation.  Mutation of two Asp residues flanking Arg221a (D221A/D222K) almost abolishes the allosteric properties of thrombin and shows that the Na+ binding loop is also involved in direct recognition of protein C and antithrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqffZELRpwnk7Vg90H21EOLACvtfcHk0lg670DInr3nqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXot1Giu7c%253D&md5=eeb9c69f2551480f4508a2feca1b0249</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.38.22089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.38.22089%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BCera%26aufirst%3DE.%26aulast%3DGuinto%26aufirst%3DE.%2BR.%26aulast%3DVindigni%26aufirst%3DA.%26aulast%3DDang%26aufirst%3DQ.%2BD.%26aulast%3DAyala%26aufirst%3DY.%2BM.%26aulast%3DWuyi%26aufirst%3DM.%26aulast%3DTulinsky%26aufirst%3DA.%26atitle%3DThe%2520Na%252B%2520Binding%2520Site%2520of%2520Thrombin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D22089%26epage%3D22092%26doi%3D10.1074%2Fjbc.270.38.22089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulinsky, A.</span></span> <span> </span><span class="NLM_article-title">The Molecular Environment of the Na<sup>+</sup> Binding Site of Thrombin</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1016/S0301-4622(96)02227-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0301-4622%2896%2902227-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=9108691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXhvFWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1997&pages=185-200&author=E.+Zhangauthor=A.+Tulinsky&title=The+Molecular+Environment+of+the+Na%2B+Binding+Site+of+Thrombin&doi=10.1016%2FS0301-4622%2896%2902227-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular environment of the Na+ binding site of thrombin</span></div><div class="casAuthors">Zhang, Erli; Tulinsky, A.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">185-200</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">When Na+ binds to thrombin, a conformational change is induced that renders the enzyme kinetically faster and more specific in the activation of fibrinogen.  Two Na+ binding sites have here been identified crystallog. by exchanging Na+ with Rb+.  One is intermol., found on the surface between two symmetry-related thrombin mols.  Since it is not present in thrombin crystal structures having different crystal systems, the other Na+ site is the functionally relevant one.  The second site has octahedral coordination with the carbonyl oxygen atoms of Arg221A and Lys224 and four conserved water mols.  It is located near Asp189 of the S1 specificity site in an elongated solvent channel (8×18 Å) formed by four antiparallel β-strands between Cys182-Cys191 and Val213-Tyr228.  This channel, extending from the active site to the opposite surface of the enzyme, was first noted in the hirudin-thrombin structure and contains about 20 conserved water mols. linked together by a hydrogen bonding network that connects to the main chain of thrombin.  Although the antiparallel β-strand interactions of the functional Na+ binding site are the same in prethrombin2, the loops between the strands are very different, so that Asp189 and Arg221A are not positioned properly for either substrate or Na+ binding in prethrombin2.  A water mol. with octahedral coordination has also been identified in factor Xa at the topol. equiv. Na+ site position of thrombin.  Since activated protein C shows enhanced activity with monovalent cation binding, the same position is probably utilized by Na+.  Since thrombin crystals could not be grown in the absence of Na+, the cation was leached from Na+-bound thrombin crystals by diffusion/exchange.  Although both Na+ and their coordinating water mols. were removed from the Na+ binding sites, the remainder of the thrombin structure was, unexpectedly, the same.  The lack of an allosteric change is most likely attributable to crystal packing effects.  Thus, the structure of the slow form remains to be established crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXh6_qcjrtd7Vg90H21EOLACvtfcHk0ljRHjTvQNoQUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvFWhsL8%253D&md5=9abb5ab1519bef7f16541c89138cc91a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0301-4622%2896%2902227-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0301-4622%252896%252902227-2%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DTulinsky%26aufirst%3DA.%26atitle%3DThe%2520Molecular%2520Environment%2520of%2520the%2520Na%252B%2520Binding%2520Site%2520of%2520Thrombin%26jtitle%3DBiophys.%2520Chem.%26date%3D1997%26volume%3D63%26spage%3D185%26epage%3D200%26doi%3D10.1016%2FS0301-4622%2896%2902227-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huntington, J. A.</span></span> <span> </span><span class="NLM_article-title">How Na<sup>+</sup> Activates Thrombin - A Review of the Functional and Structural Data</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1515/BC.2008.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1515%2FBC.2008.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=18979627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1Ohur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2008&pages=1025-1035&author=J.+A.+Huntington&title=How+Na%2B+Activates+Thrombin+-+A+Review+of+the+Functional+and+Structural+Data&doi=10.1515%2FBC.2008.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How Na+ activates thrombin - a review of the functional and structural data</span></div><div class="casAuthors">Huntington, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1025-1035</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Thrombin is the ultimate coagulation factor; it is the final protease generated in the blood coagulation cascade and is the effector of clot formation.  Regulation of thrombin activity is thus of great relevance to detg. the correct hemostatic balance, with dysregulation leading to bleeding or thrombosis.  One of the most enigmatic and controversial regulators of thrombin activity is the monovalent cation, Na+.  When bound to Na+, thrombin adopts a 'fast' conformation which cleaves all procoagulant substrates more rapidly, and when free of Na+, thrombin reverts to a 'slow' state which preferentially activates the protein C anticoagulant pathway.  Thus, Na+-binding allosterically modulates the activity of thrombin and helps det. the hemostatic balance.  Over the last 30 yr, there has been much research investigating the structural basis of thrombin allostery.  Biochem. and mutagenesis studies have established which regions and residues are involved in the slow → fast conformational change, and recently several crystal structures of the putative slow form have been solved.  Here, the biochem. and crystallog. data are reviewed to see if investigators are any closer to understanding the conformational basis of the Na+ activation of thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoItIkRWHB6WrVg90H21EOLACvtfcHk0ljRHjTvQNoQUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1Ohur0%253D&md5=c830271129b66beb2bd232c19c5472f1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1515%2FBC.2008.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2008.113%26sid%3Dliteratum%253Aachs%26aulast%3DHuntington%26aufirst%3DJ.%2BA.%26atitle%3DHow%2520Na%252B%2520Activates%2520Thrombin%2520-%2520A%2520Review%2520of%2520the%2520Functional%2520and%2520Structural%2520Data%26jtitle%3DBiol.%2520Chem.%26date%3D2008%26volume%3D389%26spage%3D1025%26epage%3D1035%26doi%3D10.1515%2FBC.2008.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulsdorf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Intriguing Role of Water in Protein-Ligand Binding Studied by Neutron Crystallography on Trypsin Complexes</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">3559</span> <span class="refDoi"> DOI: 10.1038/s41467-018-05769-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1038%2Fs41467-018-05769-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=30177695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ltVOhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&author=J.+Schiebelauthor=R.+Gaspariauthor=T.+Wulsdorfauthor=K.+Ngoauthor=C.+Sohnauthor=T.+E.+Schraderauthor=A.+Cavalliauthor=A.+Ostermannauthor=A.+Heineauthor=G.+Klebe&title=Intriguing+Role+of+Water+in+Protein-Ligand+Binding+Studied+by+Neutron+Crystallography+on+Trypsin+Complexes&doi=10.1038%2Fs41467-018-05769-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Intriguing role of water in protein-ligand binding studied by neutron crystallography on trypsin complexes</span></div><div class="casAuthors">Schiebel Johannes; Wulsdorf Tobias; Ngo Khang; Sohn Christian; Heine Andreas; Klebe Gerhard; Schiebel Johannes; Gaspari Roberto; Cavalli Andrea; Schrader Tobias E; Ostermann Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3559</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hydrogen bonds are key interactions determining protein-ligand binding affinity and therefore fundamental to any biological process.  Unfortunately, explicit structural information about hydrogen positions and thus H-bonds in protein-ligand complexes is extremely rare and similarly the important role of water during binding remains poorly understood.  Here, we report on neutron structures of trypsin determined at very high resolutions ≤1.5 ÅA in uncomplexed and inhibited state complemented by X-ray and thermodynamic data and computer simulations.  Our structures show the precise geometry of H-bonds between protein and the inhibitors N-amidinopiperidine and benzamidine along with the dynamics of the residual solvation pattern.  Prior to binding, the ligand-free binding pocket is occupied by water molecules characterized by a paucity of H-bonds and high mobility resulting in an imperfect hydration of the critical residue Asp189.  This phenomenon likely constitutes a key factor fueling ligand binding via water displacement and helps improving our current view on water influencing protein-ligand recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT2ZT4TkrW9o0h5eUSa2oe_fW6udTcc2eZO7rIiai3nkLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ltVOhtw%253D%253D&md5=56cfa8b99517f0da0967e3c9fbc09ac9</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05769-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05769-2%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DWulsdorf%26aufirst%3DT.%26aulast%3DNgo%26aufirst%3DK.%26aulast%3DSohn%26aufirst%3DC.%26aulast%3DSchrader%26aufirst%3DT.%2BE.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DIntriguing%2520Role%2520of%2520Water%2520in%2520Protein-Ligand%2520Binding%2520Studied%2520by%2520Neutron%2520Crystallography%2520on%2520Trypsin%2520Complexes%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26doi%3D10.1038%2Fs41467-018-05769-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stürzebecher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürzebecher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieweg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauptmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwardt, F.</span></span> <span> </span><span class="NLM_article-title">Synthetic Inhibitors of Bovine Factor Xa and Thrombin Comparison of Their Anticoagulant Efficiency</span>. <i>Thromb. Res.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/0049-3848(89)90232-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2F0049-3848%2889%2990232-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=2749616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaL1MXkslSgs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1989&pages=245-252&author=J.+St%C3%BCrzebecherauthor=U.+St%C3%BCrzebecherauthor=H.+Viewegauthor=G.+Wagnerauthor=J.+Hauptmannauthor=F.+Markwardt&title=Synthetic+Inhibitors+of+Bovine+Factor+Xa+and+Thrombin+Comparison+of+Their+Anticoagulant+Efficiency&doi=10.1016%2F0049-3848%2889%2990232-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic inhibitors of bovine factor Xa and thrombin: Comparison of their anticoagulant efficiency</span></div><div class="casAuthors">Stuerzebecher, J.; Stuerzebecher, Uta; Vieweg, H.; Wagner, G.; Hauptmann, J.; Markwardt, F.</div><div class="citationInfo"><span class="NLM_cas:title">Thrombosis Research</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-52</span>CODEN:
                <span class="NLM_cas:coden">THBRAA</span>;
        ISSN:<span class="NLM_cas:issn">0049-3848</span>.
    </div><div class="casAbstract">New derivs. of benzamidine (Nα-arylsulfonylaminoacetylated derivs. of 3-amidinophenylalanine) were potent inhibitors of bovine factor Xa (FXa).  The Me and Et esters of Nα-Tos-Gly- and Nα-βNas-Gly-substituted 3-amidinophenylalanine, inhibited F Xa more effectively (Ki values near 0.5 μmol/L) than thrombin (Ki about 50 μmol/L).  Reinvestigation of inhibition of F Xa by bis-benzamidines showed that compds. contg. a cycloheptanone linking bridge are tight-binding inhibitors of F Xa (Ki in the 10 nmol/L range).  The use of these inhibitors enabled us to clarify whether inhibition of F Xa or inhibition of thrombin is more efficient in anticoagulation.  The thrombin inhibitors and not the potent F Xa inhibitors proved to be particularly effective in anticoagulation in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGxZUXZRCYbbVg90H21EOLACvtfcHk0lgXjzwPnIlK2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkslSgs7Y%253D&md5=8cc9a02fb634fa0c9c441e63312cf2cb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2F0049-3848%2889%2990232-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0049-3848%252889%252990232-6%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25BCrzebecher%26aufirst%3DJ.%26aulast%3DSt%25C3%25BCrzebecher%26aufirst%3DU.%26aulast%3DVieweg%26aufirst%3DH.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DHauptmann%26aufirst%3DJ.%26aulast%3DMarkwardt%26aufirst%3DF.%26atitle%3DSynthetic%2520Inhibitors%2520of%2520Bovine%2520Factor%2520Xa%2520and%2520Thrombin%2520Comparison%2520of%2520Their%2520Anticoagulant%2520Efficiency%26jtitle%3DThromb.%2520Res.%26date%3D1989%26volume%3D54%26spage%3D245%26epage%3D252%26doi%3D10.1016%2F0049-3848%2889%2990232-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotlyar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nudelman, A.</span></span> <span> </span><span class="NLM_article-title">NMR Chemical Shifts of Common Laboratory Solvents as Trace Impurities</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7512</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jo971176v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo971176v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=1997&pages=7512-7515&author=H.+E.+Gottliebauthor=V.+Kotlyarauthor=A.+Nudelman&title=NMR+Chemical+Shifts+of+Common+Laboratory+Solvents+as+Trace+Impurities&doi=10.1021%2Fjo971176v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">NMR chemical shifts of common laboratory solvents as trace impurities</span></div><div class="casAuthors">Gottlieb, Hugo E.; Kotlyar, Vadim; Nudelman, Abraham</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7512-7515</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 1H (300 MHz) and 13C (75 MHz) NMR chem. shifts for >30 common impurities, such as lab. solvents, stabilizers, greases and water, are reported in 7 deuterated NMR solvents.  The temp. dependence of the chem. shift of the proton residual peak (HDO) in D2O is established for the entire liq. range of this sample.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocnoOemocS9LVg90H21EOLACvtfcHk0lgXjzwPnIlK2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D&md5=c297adda99f8c10e9f9b0ab9fe5f55be</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjo971176v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo971176v%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DH.%2BE.%26aulast%3DKotlyar%26aufirst%3DV.%26aulast%3DNudelman%26aufirst%3DA.%26atitle%3DNMR%2520Chemical%2520Shifts%2520of%2520Common%2520Laboratory%2520Solvents%2520as%2520Trace%2520Impurities%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1997%26volume%3D62%26spage%3D7512%26epage%3D7515%26doi%3D10.1021%2Fjo971176v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wollatz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haznar-Garbacz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitschies, W.</span></span> <span> </span><span class="NLM_article-title">Characterisation of Selected Active Agents Regarding p<i>K</i><sub>a</sub> Values, Solubility Concentrations and pH Profiles by SiriusT3</span>. <i>Eur. J. Pharm. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1016/j.ejpb.2015.02.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.ejpb.2015.02.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=25758123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A280%3ADC%252BC2MnivVKlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2015&pages=155-170&author=D.+Sch%C3%B6nherrauthor=U.+Wollatzauthor=D.+Haznar-Garbaczauthor=U.+Hankeauthor=K.+J.+Boxauthor=R.+Taylorauthor=R.+Ruizauthor=S.+Beatoauthor=D.+Beckerauthor=W.+Weitschies&title=Characterisation+of+Selected+Active+Agents+Regarding+pKa+Values%2C+Solubility+Concentrations+and+pH+Profiles+by+SiriusT3&doi=10.1016%2Fj.ejpb.2015.02.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Characterisation of selected active agents regarding pKa values, solubility concentrations and pH profiles by SiriusT3</span></div><div class="casAuthors">Schonherr D; Wollatz U; Haznar-Garbacz D; Hanke U; Box K J; Taylor R; Ruiz R; Beato S; Becker D; Weitschies W</div><div class="citationInfo"><span class="NLM_cas:title">European journal of pharmaceutics and biopharmaceutics : official journal of Arbeitsgemeinschaft fur Pharmazeutische Verfahrenstechnik e.V</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The aim of this work was to determine pKa values and solubility properties of 34active agents using the SiriusT3 apparatus.  The selected drug substances belong to the groups of ACE-inhibitors, β-blockers, antidiabetics and lipid lowering substances.  Experimentally obtained pKa and intrinsic solubility values were compared to calculated values (program ACD/ChemSketch) and pKa values to published data as well.  Solubility-pH profiles were generated to visualise the substance solubility over the gastrointestinal pH range.  The relationship between the solubility characteristic of a substance, its bioavailability and categorisation according to the Biopharmaceutics Classification System (BCS) was examined as well.  The results showed a good agreement between experimentally obtained, calculated and published pKa values.  The measured and calculated intrinsic solubility values indicated several major deviations.  All solubility-pH profiles showed the expected shape and appearance for acids, bases or zwitterionic substances.  The obtained results for the pKa and solubility measurements of the examined active agents may help to predict their physicochemical behaviour in vivo, and to understand the bioavailability of the substances according to their BCS categorisation.  The easy and reproducible determination of pKa and solubility values makes the SiriusT3 apparatus a useful tool in early stages of drug and formulation development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0tGrDkvlZJ-2qxNLjzXe8fW6udTcc2ebZ2h4Cn7LL_Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnivVKlsw%253D%253D&md5=02100c72ca191e1039faa3b20d5714d8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.ejpb.2015.02.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejpb.2015.02.028%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DD.%26aulast%3DWollatz%26aufirst%3DU.%26aulast%3DHaznar-Garbacz%26aufirst%3DD.%26aulast%3DHanke%26aufirst%3DU.%26aulast%3DBox%26aufirst%3DK.%2BJ.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DRuiz%26aufirst%3DR.%26aulast%3DBeato%26aufirst%3DS.%26aulast%3DBecker%26aufirst%3DD.%26aulast%3DWeitschies%26aufirst%3DW.%26atitle%3DCharacterisation%2520of%2520Selected%2520Active%2520Agents%2520Regarding%2520pKa%2520Values%252C%2520Solubility%2520Concentrations%2520and%2520pH%2520Profiles%2520by%2520SiriusT3%26jtitle%3DEur.%2520J.%2520Pharm.%2520Biopharm.%26date%3D2015%26volume%3D92%26spage%3D155%26epage%3D170%26doi%3D10.1016%2Fj.ejpb.2015.02.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszczek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span> <span> </span><span class="NLM_article-title">High-Precision Isothermal Titration Calorimetry with Automated Peak-Shape Analysis</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">5066</span>– <span class="NLM_lpage">5073</span>, <span class="refDoi"> DOI: 10.1021/ac3007522</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac3007522" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=5066-5073&author=S.+Kellerauthor=C.+Vargasauthor=H.+Zhaoauthor=G.+Piszczekauthor=C.+A.+Brautigamauthor=P.+Schuck&title=High-Precision+Isothermal+Titration+Calorimetry+with+Automated+Peak-Shape+Analysis&doi=10.1021%2Fac3007522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">High-Precision Isothermal Titration Calorimetry with Automated Peak-Shape Analysis</span></div><div class="casAuthors">Keller, Sandro; Vargas, Carolyn; Zhao, Huaying; Piszczek, Grzegorz; Brautigam, Chad A.; Schuck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5066-5073</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) is a powerful classical method that enables researchers in many fields to study the thermodn. of mol. interactions.  Primary ITC data comprise the temporal evolution of differential power reporting the heat of reaction during a series of injections of aliquots of a reactant into a sample cell.  By integration of each injection peak, an isotherm can be constructed of total changes in enthalpy as a function of changes in soln. compn., which is rich in thermodn. information on the reaction.  However, the signals from the injection peaks are superimposed by the stochastically varying time-course of the instrumental baseline power, limiting the precision of ITC isotherms.  The authors describe a method for automated peak assignment based on peak-shape anal. via singular value decompn. in combination with detailed least-squares modeling of local pre- and post- injection baselines.  This approach can effectively filter out contributions of short-term noise and adventitious events in the power trace.  This method also provides, for the first time, statistical error ests. for the individual isotherm data points.  In turn, this results in improved detection limits for high-affinity or low-enthalpy binding reactions and significantly higher precision of the derived thermodn. parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJcKwO_fHPFbVg90H21EOLACvtfcHk0lhYFdSWSk7cmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D&md5=bb157e956fa9cef94a88974d733cad1b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fac3007522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac3007522%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DS.%26aulast%3DVargas%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DPiszczek%26aufirst%3DG.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DSchuck%26aufirst%3DP.%26atitle%3DHigh-Precision%2520Isothermal%2520Titration%2520Calorimetry%2520with%2520Automated%2520Peak-Shape%2520Analysis%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D5066%26epage%3D5073%26doi%3D10.1021%2Fac3007522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span> <span> </span><span class="NLM_article-title">High-Precision, Automated Integration of Multiple Isothermal Titration Calorimetric Thermograms: New Features of NITPIC</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2014.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.ymeth.2014.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=25524420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVOrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2015&pages=87-98&author=T.+H.+Scheuermannauthor=C.+A.+Brautigam&title=High-Precision%2C+Automated+Integration+of+Multiple+Isothermal+Titration+Calorimetric+Thermograms%3A+New+Features+of+NITPIC&doi=10.1016%2Fj.ymeth.2014.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">High-precision, automated integration of multiple isothermal titration calorimetric thermograms: New features of NITPIC</span></div><div class="casAuthors">Scheuermann, Thomas H.; Brautigam, Chad A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87-98</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) has become a std. and widely available tool to measure the thermodn. parameters of macromol. assocns.  Modern applications of the method, including global anal. and drug screening, require the acquisition of multiple sets of data; sometimes these data sets no. in the hundreds.  Therefore, there is a need for quick, precise, and automated means to process the data, particularly at the first step of data anal., which is commonly the integration of the raw data to yield an interpretable isotherm.  Herein, we describe enhancements to an algorithm that previously has been shown to provide an automated, unbiased, and high-precision means to integrate ITC data.  These improvements allow for the speedy and precise serial integration of an unlimited no. of ITC data sets, and they have been implemented in the freeware program NITPIC, version 1.1.0.  We present a comprehensive comparison of the performance of this software against an older version of NITPIC and a current version of Origin, which is commonly used for integration.  The new methods recapitulate the excellent performance of the previous versions of NITPIC while speeding it up substantially, and their precision is significantly better than that of Origin.  This new version of NITPIC is therefore well suited to the serial integration of many ITC data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyqQqBGpeedbVg90H21EOLACvtfcHk0lhYFdSWSk7cmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVOrug%253D%253D&md5=eb4e3e0603a9dbd330d3dbc0302b731c</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26atitle%3DHigh-Precision%252C%2520Automated%2520Integration%2520of%2520Multiple%2520Isothermal%2520Titration%2520Calorimetric%2520Thermograms%253A%2520New%2520Features%2520of%2520NITPIC%26jtitle%3DMethods%26date%3D2015%26volume%3D76%26spage%3D87%26epage%3D98%26doi%3D10.1016%2Fj.ymeth.2014.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, Scaling, Space-Group Assignment and Post-Refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+Scaling%2C+Space-Group+Assignment+and+Post-Refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lhYFdSWSk7cmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520Scaling%252C%2520Space-Group%2520Assignment%2520and%2520Post-Refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sparta, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krug, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">XDSAPP2.0</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">1085</span>– <span class="NLM_lpage">1092</span>, <span class="refDoi"> DOI: 10.1107/S1600576716004416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS1600576716004416" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1085-1092&author=K.+M.+Spartaauthor=M.+Krugauthor=U.+Heinemannauthor=U.+Muellerauthor=M.+S.+Weiss&title=XDSAPP2.0&doi=10.1107%2FS1600576716004416"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS1600576716004416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600576716004416%26sid%3Dliteratum%253Aachs%26aulast%3DSparta%26aufirst%3DK.%2BM.%26aulast%3DKrug%26aufirst%3DM.%26aulast%3DHeinemann%26aufirst%3DU.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DXDSAPP2.0%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2016%26spage%3D1085%26epage%3D1092%26doi%3D10.1107%2FS1600576716004416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lh-_t8npQRDKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title"><i>PHENIX</i>: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lh-_t8npQRDKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of <i>Coot</i></span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lh-_t8npQRDKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title"><i>PROCHECK</i>: A Program to Check the Stereochemical Quality of Protein Structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+A+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lhScWgX9jDLKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520A%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleywegt, G. J.</span></span> <i>MOLEMAN</i> - Unpublished Program; <span class="NLM_publisher-name">Uppsala University</span>: <span class="NLM_publisher-loc">Uppsala, Sweden</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kleywegt%2C+G.+J.+MOLEMAN+-+Unpublished+Program%3B+Uppsala+University%3A+Uppsala%2C+Sweden."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKleywegt%26aufirst%3DG.%2BJ.%26btitle%3DMOLEMAN%26pub%3DUppsala%2520University" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleywegt, G. J.</span>; <span class="NLM_string-name">Zou, J. Y.</span>; <span class="NLM_string-name">Kjeldgaard, M.</span>; <span class="NLM_string-name">Jones, T. A.</span></span> In  <i>International Tables for Crystallography: Crystallography of Biological Macromolecules</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M. G.</span>; <span class="NLM_string-name">Arnold, E.</span></span>, Eds.; <span class="NLM_publisher-name">Kluwer Academic Publishers</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>, <span class="NLM_year">2001</span>; Vol.  <span class="NLM_volume">F</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=G.+J.+Kleywegt&author=J.+Y.+Zou&author=M.+Kjeldgaard&author=T.+A.+Jonesauthor=M.+G.+Rossmann&author=E.+Arnold&title=International+Tables+for+Crystallography%3A+Crystallography+of+Biological+Macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKleywegt%26aufirst%3DG.%2BJ.%26btitle%3DInternational%2520Tables%2520for%2520Crystallography%253A%2520Crystallography%2520of%2520Biological%2520Macromolecules%26aulast%3DRossmann%26aufirst%3DM.%2BG.%26pub%3DKluwer%2520Academic%2520Publishers%26date%3D2001%26volume%3DF" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Anna Sandner, Khang Ngo, Johannes Schiebel, Angela Ilse Marca Pizarroso, Linda Schmidt, Benjamin Wenzel, Torsten Steinmetzer, Andreas Ostermann, Andreas Heine, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Klebe</span>. </span><span class="cited-content_cbyCitation_article-title">How a Fragment Draws Attention to Selectivity Discriminating Features between the Related Proteases Trypsin and Thrombin. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1611-1625. <a href="https://doi.org/10.1021/acs.jmedchem.0c01809" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01809</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01809%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHow%252Ba%252BFragment%252BDraws%252BAttention%252Bto%252BSelectivity%252BDiscriminating%252BFeatures%252Bbetween%252Bthe%252BRelated%252BProteases%252BTrypsin%252Band%252BThrombin%26aulast%3DSandner%26aufirst%3DAnna%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16102020%26date%3D20012021%26volume%3D64%26issue%3D3%26spage%3D1611%26epage%3D1625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barbara  Wienen‐Schmidt</span>, <span class="hlFld-ContribAuthor ">Matthias  Oebbeke</span>, <span class="hlFld-ContribAuthor ">Khang  Ngo</span>, <span class="hlFld-ContribAuthor ">Andreas  Heine</span>, <span class="hlFld-ContribAuthor ">Gerhard  Klebe</span>. </span><span class="cited-content_cbyCitation_article-title">Two Methods, One Goal: Structural Differences between Cocrystallization and Crystal Soaking to Discover Ligand Binding Poses. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (1)
                                     , 292-300. <a href="https://doi.org/10.1002/cmdc.202000565" title="DOI URL">https://doi.org/10.1002/cmdc.202000565</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000565%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DTwo%252BMethods%25252C%252BOne%252BGoal%25253A%252BStructural%252BDifferences%252Bbetween%252BCocrystallization%252Band%252BCrystal%252BSoaking%252Bto%252BDiscover%252BLigand%252BBinding%252BPoses%26aulast%3DWienen%25E2%2580%2590Schmidt%26aufirst%3DBarbara%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D292%26epage%3D300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Precursor <b>11</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) POBr<sub>3</sub>, 130–150 °C, 2 h then MeOH, 0 °C to rt, 15 h, 74%; (b) NaBH<sub>4</sub>, MeOH, 65 °C, 3 h, 91%; (c) PBr<sub>3</sub>, 1,4-dioxane, 40 °C, 3 h; (d) NaN<sub>3</sub>, dimethylformamide (DMF), rt, 24 h, 66% (over two steps); (e) PMe<sub>3</sub>, tetrahydrofuran (THF), 0 °C to rt, 3 h then Boc<sub>2</sub>O, NEt<sub>3</sub>, THF, 0 °C to rt, 3 h, 50%; (f) 25% (w/v) NH<sub>3(aq)</sub>, Cu<sub>2</sub>O, dimethylethylenediamine (DMEDA), K<sub>2</sub>CO<sub>3</sub>, ethylene glycol, 60 °C, 16 h, 63%; (g) 4 <span class="smallcaps smallerCapital">M</span> HCl in 1,4-dioxane, rt, 3 h, 95%.</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Inhibitors <b>1</b>–<b>4</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc-<span class="smallcaps smallerCapital">d</span>-Phe-Pro-OH (for <b>15</b>–<b>17</b>), Boc-<span class="smallcaps smallerCapital">d</span>-DiPhe-Pro-OH (for <b>18</b>), HOB<i>t</i>, EDC·HCl, DMF, DIPEA, 0 °C to rt, 18–24 h, 65–78%; (b) TFA, rt, 2 h, 78–92%. R<sup>3</sup> exists only for <b>3</b>, <b>13</b>, and <b>17</b>.</p></p></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding modes of inhibitors <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSX">6HSX</a>, A), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q">6T3Q</a>, B), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T4A">6T4A</a>, C), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT">6TDT</a>, D) in the S1 pocket of thrombin (left) compared to the structures of trypsin (right) by inhibitors <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0M">6T0M</a>, E), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0P">6T0P</a>, F), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5W">6T5W</a>, G), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SY3">6SY3</a>, H). The inhibitors are represented in light brown, whereas the protein residues are displayed in green. Heteroatoms are color-coded by atom type. Water molecules are colored in red, and favorable hydrogen bonds are shown as dashed lines. The distances in the complex of thrombin between the water molecules (W1, W5, W6) at the bottom of the S1 site are given in Å and are colored in red. In the trypsin complexes with <b>1</b> (E) and <b>3</b> (G), Gln192 is not visible due to the weakly defined <i>mF</i><sub>o</sub><i>–DF</i><sub>c</sub> electron density of this residue after refinement. It is likely scattered over multiple orientations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Diagrams (A: thrombin, B: trypsin) displaying the experimentally determined enthalpies Δ<i>H</i>°<sub>obs</sub> of the inhibitors <b>1</b>–<b>4</b> under different buffer conditions. They are plotted against the heat of ionization Δ<i>H</i>°<sub>ion</sub> (phosphate:<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> 3.60 kJ mol<sup>–1</sup>, 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES):<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> 21.01 kJ mol<sup>–1</sup>, tricine:<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> 31.97 kJ mol<sup>–1</sup>, Tris:<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> 47.53 kJ mol<sup>–1</sup>) of the respective buffers. The slopes of the linear fit give the molar amounts of protons that are transferred between buffer, protein, and ligand upon binding (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The intercept of these plots is reported as “buffer-corrected” value (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Thermodynamic signatures of the inhibitors <b>1</b>–<b>4</b> in complex with thrombin (A) and trypsin (B). The results are represented as buffer-corrected values listed in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. They were determined by direct titration in thrombin. Notably, due to the relatively weak affinity of the studied ligands toward trypsin (<i>K</i><sub>i</sub> values between 40 and 500 μM), all ITC measurements with trypsin have been performed as displacement titrations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/medium/jm9b02061_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode of ligand <b>4</b> in the S1 pocket of thrombin (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT">6TDT</a>) in the presence of Na<sup>+</sup> (purple ball) closely adjacent to Asp189. The octahedral coordination of Na<sup>+</sup> is represented as the dashed lines in purple. The ligand is shown in light brown, whereas the protein residues are displayed in green. Heteroatoms are color-coded by atom type. Water molecules are displayed as red spheres, and favorable hydrogen bonds are indicated as dashed lines with the corresponding distances in Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02061/20200323/images/large/jm9b02061_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02061&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43161" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43161" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 59 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Can We Rationally Design Promiscuous Drugs?</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2006.01.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.sbi.2006.01.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16442279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=127-136&author=A.+L.+Hopkinsauthor=J.+S.+Masonauthor=J.+P.+Overington&title=Can+We+Rationally+Design+Promiscuous+Drugs%3F&doi=10.1016%2Fj.sbi.2006.01.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Can we rationally design promiscuous drugs?</span></div><div class="casAuthors">Hopkins, Andrew L.; Mason, Jonathan S.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-136</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Structure-based drug design is now used widely in modern medicinal chem.  The application of structural biol. to medicinal chem. has heralded the 'rational drug design' vision of discovering exquisitely selective ligands.  However, recent advances in post-genomic biol. are indicating that polypharmacol. may be a necessary trait for the efficacy of many drugs, therefore questioning the 'one drug, one target' assumption of current rational drug design.  By combining advances in chemoinformatics and structural biol., it might be possible to rationally design the next generation of promiscuous drugs with polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQWdh60x9AHrVg90H21EOLACvtfcHk0lgfCU2pXQj1aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsFOjsrc%253D&md5=8f9e95a41976b8e3ac790f2fff2c406f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2006.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2006.01.013%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DCan%2520We%2520Rationally%2520Design%2520Promiscuous%2520Drugs%253F%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D16%26spage%3D127%26epage%3D136%26doi%3D10.1016%2Fj.sbi.2006.01.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huggins, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span> <span> </span><span class="NLM_article-title">Rational Approaches to Improving Selectivity in Drug Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1424</span>– <span class="NLM_lpage">1444</span>, <span class="refDoi"> DOI: 10.1021/jm2010332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2010332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1424-1444&author=D.+J.+Hugginsauthor=W.+Shermanauthor=B.+Tidor&title=Rational+Approaches+to+Improving+Selectivity+in+Drug+Design&doi=10.1021%2Fjm2010332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Approaches to Improving Selectivity in Drug Design</span></div><div class="casAuthors">Huggins, David J.; Sherman, Woody; Tidor, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1424-1444</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Rational Approaches to improving selectivity in Drug Design are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJv9AbW-c3TbVg90H21EOLACvtfcHk0lgfCU2pXQj1aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFWktg%253D%253D&md5=5b7bf49f9f96a05adad4d42f63739252</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm2010332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2010332%26sid%3Dliteratum%253Aachs%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DTidor%26aufirst%3DB.%26atitle%3DRational%2520Approaches%2520to%2520Improving%2520Selectivity%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1424%26epage%3D1444%26doi%3D10.1021%2Fjm2010332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alterio, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Fiore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ambrosio, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supuran, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Simone, G.</span></span> <span> </span><span class="NLM_article-title">Multiple Binding Modes of Inhibitors to Carbonic Anhydrases: How to Design Specific Drugs Targeting 15 Different Isoforms?</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">4421</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/cr200176r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr200176r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2012&pages=4421-4468&author=V.+Alterioauthor=A.+Di+Fioreauthor=K.+D%E2%80%99Ambrosioauthor=C.+T.+Supuranauthor=G.+De+Simone&title=Multiple+Binding+Modes+of+Inhibitors+to+Carbonic+Anhydrases%3A+How+to+Design+Specific+Drugs+Targeting+15+Different+Isoforms%3F&doi=10.1021%2Fcr200176r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Multiple Binding Modes of Inhibitors to Carbonic Anhydrases: How to Design Specific Drugs Targeting 15 Different Isoforms?</span></div><div class="casAuthors">Alterio, Vincenzo; Di Fiore, Anna; D'Ambrosio, Katia; Supuran, Claudiu T.; De Simone, Giuseppina</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4421-4468</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The aim of this review is to provide an exhaustive description of the structural studies on α-CAs (carbonic anhydrases) so far reported.  In particular, the main structural features of the catalytically active α-CA isoenzymes characterized to date will be described and the current state of the art on complexes of hCA II with the principal classes of inhibitors will be summarized.  A comparison with the corresponding adducts of other isoforms, when available, will be also performed, in order to find for each isoform the key residues responsible for enzyme/inhibitor interaction.  Finally, recent advances in the field of drug design of selective CAIs will be illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA0GdtMHwlZ7Vg90H21EOLACvtfcHk0lgfCU2pXQj1aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrsbg%253D&md5=eb45ffb5efe07980f1388d99bf51406a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fcr200176r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr200176r%26sid%3Dliteratum%253Aachs%26aulast%3DAlterio%26aufirst%3DV.%26aulast%3DDi%2BFiore%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Ambrosio%26aufirst%3DK.%26aulast%3DSupuran%26aufirst%3DC.%2BT.%26aulast%3DDe%2BSimone%26aufirst%3DG.%26atitle%3DMultiple%2520Binding%2520Modes%2520of%2520Inhibitors%2520to%2520Carbonic%2520Anhydrases%253A%2520How%2520to%2520Design%2520Specific%2520Drugs%2520Targeting%252015%2520Different%2520Isoforms%253F%26jtitle%3DChem.%2520Rev.%26date%3D2012%26volume%3D112%26spage%3D4421%26epage%3D4468%26doi%3D10.1021%2Fcr200176r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitesides, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, V. M.</span></span> <span> </span><span class="NLM_article-title">Designing Ligands to Bind Proteins</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">395</span>, <span class="refDoi"> DOI: 10.1017/S0033583506004240</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1017%2FS0033583506004240" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16817982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XoslSgt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2005&pages=385-395&author=G.+M.+Whitesidesauthor=V.+M.+Krishnamurthy&title=Designing+Ligands+to+Bind+Proteins&doi=10.1017%2FS0033583506004240"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Designing ligands to bind proteins</span></div><div class="casAuthors">Whitesides, George M.; Krishnamurthy, Vijay M.</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">385-395</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  The ability to design drugs (so-called 'rational drug design') was one of the long-term objectives of chem. for 50 years.  It is an exceptionally difficult problem, and many of its parts lie outside the expertise of chem.  The much more limited problem - how to design tight-binding ligands (rational ligand design) - would seem to be one that chem. could solve, but has also proved remarkably recalcitrant.  The question is Why is it so difficult' and the answer is 'We still don't entirely know'.  This perspective discusses some of the tech. issues - potential functions, protein plasticity, enthalpy/entropy compensation, and others - that contribute, and suggests areas where fundamental understanding of protein-ligand interactions falls short of what is needed.  It surveys recent technol. developments (in particular, isothermal titrn. calorimetry) that will, hopefully, make now the time for serious progress in this area.  It concludes with the calorimetric examn. of the assocn. of a series of systematically varied ligands with a model protein.  The counterintuitive thermodn. results obsd. serve to illustrate that, even in relatively simple systems, understanding protein-ligand assocn. is challenging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrImeG0nXP2w7Vg90H21EOLACvtfcHk0liWzS3s7LSkYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XoslSgt7g%253D&md5=24d27dce3770c6fd712cc47d6af9d4ad</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1017%2FS0033583506004240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583506004240%26sid%3Dliteratum%253Aachs%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26aulast%3DKrishnamurthy%26aufirst%3DV.%2BM.%26atitle%3DDesigning%2520Ligands%2520to%2520Bind%2520Proteins%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2005%26volume%3D38%26spage%3D385%26epage%3D395%26doi%3D10.1017%2FS0033583506004240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Puertas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leo-Macias, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Romero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Vinas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morreale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span> <span> </span><span class="NLM_article-title">Computational Approaches to Model Ligand Selectivity in Drug Design</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">55</span>, <span class="refDoi"> DOI: 10.2174/156802606775193338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.2174%2F156802606775193338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16454757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVKmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=41-55&author=A.+Ortizauthor=P.+Gomez-Puertasauthor=A.+Leo-Maciasauthor=P.+Lopez-Romeroauthor=E.+Lopez-Vinasauthor=A.+Morrealeauthor=M.+Murciaauthor=K.+Wang&title=Computational+Approaches+to+Model+Ligand+Selectivity+in+Drug+Design&doi=10.2174%2F156802606775193338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches to model ligand selectivity in drug design</span></div><div class="casAuthors">Ortiz, Angel R.; Gomez-Puertas, Paulino; Leo-Macias, Alejandra; Lopez-Romero, Pedro; Lopez-Vinas, Eduardo; Morreale, Antonio; Murcia, Marta; Wang, Kun</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-55</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  To be effective, a designed drug must discriminate successfully the macromol. target from alternative structures present in the organism.  The last few years have witnessed the emergence of different computational tools aimed to the understanding and modeling of this process at mol. level.  Although still rudimentary, these methods are shaping a coherent approach to help in the design of mols. with high affinity and specificity, both in lead discovery and in lead optimization.  It is the purpose of this review to illustrate the array of computational tools available to consider selectivity in the design process, to summarize the most relevant applications, and to sketch the challenges ahead.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptjiqa0g9gK7Vg90H21EOLACvtfcHk0liWzS3s7LSkYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVKmtLo%253D&md5=582a6728d8614705a81e5af1dd3a6ed2</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2174%2F156802606775193338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802606775193338%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%26aufirst%3DA.%26aulast%3DGomez-Puertas%26aufirst%3DP.%26aulast%3DLeo-Macias%26aufirst%3DA.%26aulast%3DLopez-Romero%26aufirst%3DP.%26aulast%3DLopez-Vinas%26aufirst%3DE.%26aulast%3DMorreale%26aufirst%3DA.%26aulast%3DMurcia%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DK.%26atitle%3DComputational%2520Approaches%2520to%2520Model%2520Ligand%2520Selectivity%2520in%2520Drug%2520Design%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2006%26volume%3D6%26spage%3D41%26epage%3D55%26doi%3D10.2174%2F156802606775193338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Finding a Better Path to Drug Selectivity</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">985</span>– <span class="NLM_lpage">990</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2011.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.drudis.2011.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=21839183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVWktbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=985-990&author=Y.+Kawasakiauthor=E.+Freire&title=Finding+a+Better+Path+to+Drug+Selectivity&doi=10.1016%2Fj.drudis.2011.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Finding a better path to drug selectivity</span></div><div class="casAuthors">Kawasaki, Yuko; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">21/22</span>),
    <span class="NLM_cas:pages">985-990</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Extremely high affinity and selectivity are two of the most sought-after properties of drug mols.  Selectivity has been difficult to achieve, esp. for targets that belong to large families of structurally and functionally related proteins.  There are essentially two ways by which selectivity can be improved during lead optimization: a chem. modification of the lead compd. that improves the affinity towards the target to a higher extent than to off-target mols.; and a chem. modification that lowers the affinity of the lead compd. towards off-target mols.  Maximal selectivity is achieved when both mechanisms can be combined synergistically.  As we discuss here, anal. of several protease inhibitors that vary in a single functionality indicates that nonpolar functionalities preferentially follow the first mechanism, whereas polar functionalities follow the second, and that those features are imprinted in their thermodn. signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGj1IsjIk4bbVg90H21EOLACvtfcHk0liWzS3s7LSkYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVWktbjF&md5=68ab56fd05adcdc61d12ee955b0fddc5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DKawasaki%26aufirst%3DY.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DFinding%2520a%2520Better%2520Path%2520to%2520Drug%2520Selectivity%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2011%26volume%3D16%26spage%3D985%26epage%3D990%26doi%3D10.1016%2Fj.drudis.2011.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <i>Drug Design</i>; <span class="NLM_publisher-name">Spektrum</span>: <span class="NLM_publisher-loc">Berlin Heidelberg</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1007%2F978-3-642-17907-5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=G.+Klebe&title=Drug+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-17907-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-17907-5%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26btitle%3DDrug%2520Design%26pub%3DSpektrum%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Thermodynamics of Protein-Ligand Interactions as a Reference for Computational Analysis: How to Assess Accuracy, Reliability and Relevance of Experimental Data</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">867</span>– <span class="NLM_lpage">883</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9867-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1007%2Fs10822-015-9867-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=26376645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=867-883&author=S.+G.+Krimmerauthor=G.+Klebe&title=Thermodynamics+of+Protein-Ligand+Interactions+as+a+Reference+for+Computational+Analysis%3A+How+to+Assess+Accuracy%2C+Reliability+and+Relevance+of+Experimental+Data&doi=10.1007%2Fs10822-015-9867-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamics of protein-ligand interactions as a reference for computational analysis: how to assess accuracy, reliability and relevance of experimental data</span></div><div class="casAuthors">Krimmer, Stefan G.; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">867-883</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">For a conscientious interpretation of thermodn. parameters (Gibbs free energy, enthalpy and entropy) obtained by isothermal titrn. calorimetry (ITC), it is necessary to first evaluate the exptl. setup and conditions at which the data were measured.  The data quality must be assessed and the precision and accuracy of the measured parameters must be estd.  This information provides the basis at which level discussion of the data is appropriate, and allows insight into the significance of comparisons with other data.  The aim of this article is to provide the reader with basic understanding of the ITC technique and the exptl. practices commonly applied, in order to foster an appreciation for how much measured thermodn. parameters can deviate from ideal, error-free values.  Particular attention is paid to the shape of the recorded isotherm (c-value), the influence of the applied buffer used for the reaction (protonation reactions, pH), the chosen exptl. settings (temp.), impurities of protein and ligand, sources of systematic errors (soln. concn., soln. activity, and device calibration) and to the applied anal. software.  Furthermore, we comment on enthalpy-entropy compensation, heat capacities and van't Hoff enthalpies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0ngUaTMg_hbVg90H21EOLACvtfcHk0liC3VX72lU8xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtr3O&md5=39a35e4588c4571925aa4af25f3254f9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9867-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9867-y%26sid%3Dliteratum%253Aachs%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DThermodynamics%2520of%2520Protein-Ligand%2520Interactions%2520as%2520a%2520Reference%2520for%2520Computational%2520Analysis%253A%2520How%2520to%2520Assess%2520Accuracy%252C%2520Reliability%2520and%2520Relevance%2520of%2520Experimental%2520Data%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D867%26epage%3D883%26doi%3D10.1007%2Fs10822-015-9867-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Betz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulsdorf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krimmer, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Impact of Surface Water Layers on Protein-Ligand Binding: How Well Are Experimental Data Reproduced by Molecular Dynamics Simulations in a Thermolysin Test Case?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVGiur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=223-233&author=M.+Betzauthor=T.+Wulsdorfauthor=S.+G.+Krimmerauthor=G.+Klebe&title=Impact+of+Surface+Water+Layers+on+Protein-Ligand+Binding%3A+How+Well+Are+Experimental+Data+Reproduced+by+Molecular+Dynamics+Simulations+in+a+Thermolysin+Test+Case%3F&doi=10.1021%2Facs.jcim.5b00621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Surface Water Layers on Protein-Ligand Binding: How Well Are Experimental Data Reproduced by Molecular Dynamics Simulations in a Thermolysin Test Case?</span></div><div class="casAuthors">Betz, Michael; Wulsdorf, Tobias; Krimmer, Stefan G.; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">223-233</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug binding involves changes of the local water structure around proteins including water rearrangements across surface-solvation layers around protein and ligand portions exposed to the newly formed complex surface.  For a series of thermolysin-binding phosphonamidates, we discovered that variations of the partly exposed P2'-substituents modulate binding affinity up to 10 kJ mol-1 with even larger enthalpy/entropy partitioning of the binding signature.  The obsd. profiles cannot be completely explained by desolvation effects.  Instead, the quality and completeness of the surface water network wrapping around the formed complexes provide an explanation for the obsd. structure-activity relationship.  We used mol. dynamics to compute surface water networks and predict solvation sites around the complexes.  A fairly good correspondence with exptl. difference electron densities in high-resoln. crystal structures is achieved; in detail some problems with the potentials were discovered.  Charge-assisted contacts to waters appeared as exaggerated by AMBER, and stabilizing contributions of water-to-Me contacts were underestimated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-XbUxbQlrB7Vg90H21EOLACvtfcHk0liC3VX72lU8xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVGiur3F&md5=cf47671297d1fe1fa8b31ef170397ce6</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00621%26sid%3Dliteratum%253Aachs%26aulast%3DBetz%26aufirst%3DM.%26aulast%3DWulsdorf%26aufirst%3DT.%26aulast%3DKrimmer%26aufirst%3DS.%2BG.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DImpact%2520of%2520Surface%2520Water%2520Layers%2520on%2520Protein-Ligand%2520Binding%253A%2520How%2520Well%2520Are%2520Experimental%2520Data%2520Reproduced%2520by%2520Molecular%2520Dynamics%2520Simulations%2520in%2520a%2520Thermolysin%2520Test%2520Case%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D223%26epage%3D233%26doi%3D10.1021%2Facs.jcim.5b00621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span> <span> </span><span class="NLM_article-title">Just Add Water! The Effect of Water on the Specificity of Protein- Ligand Binding Sites and Its Potential Application to Drug Design</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">973</span>– <span class="NLM_lpage">980</span>, <span class="refDoi"> DOI: 10.1016/S1074-5521(96)90164-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS1074-5521%2896%2990164-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=9000013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=973-980&author=J.+E.+Ladbury&title=Just+Add+Water%21+The+Effect+of+Water+on+the+Specificity+of+Protein-+Ligand+Binding+Sites+and+Its+Potential+Application+to+Drug+Design&doi=10.1016%2FS1074-5521%2896%2990164-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Just add water! The effect of water on the specificity of protein-ligand binding sites and its potential application to drug design</span></div><div class="casAuthors">Ladbury, John E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">973-980</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology</span>)
        </div><div class="casAbstract">A review with 39 refs.  Recent data have highlighted the enigmatic role that water plays in biomol. complexes.  Water at the interface of a complex can increase the promiscuity of an interaction, yet it can also provide increased specificity and affinity.  The ability to engineer water-binding sites into the interface between a drug and its target might prove useful in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJqyhDhRviwbVg90H21EOLACvtfcHk0liC3VX72lU8xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslKrtw%253D%253D&md5=8eb3242b84897de4bb61e20ec6bc8d54</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2896%2990164-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252896%252990164-7%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26atitle%3DJust%2520Add%2520Water%2521%2520The%2520Effect%2520of%2520Water%2520on%2520the%2520Specificity%2520of%2520Protein-%2520Ligand%2520Binding%2520Sites%2520and%2520Its%2520Potential%2520Application%2520to%2520Drug%2520Design%26jtitle%3DChem.%2520Biol.%26date%3D1996%26volume%3D3%26spage%3D973%26epage%3D980%26doi%3D10.1016%2FS1074-5521%2896%2990164-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Levy, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onuchic, J. N.</span></span> <span> </span><span class="NLM_article-title">Water Mediation in Protein Folding and Molecular Recognition</span>. <i>Annu. Rev. Biophys. Biomol. Struct.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">415</span>, <span class="refDoi"> DOI: 10.1146/annurev.biophys.35.040405.102134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1146%2Fannurev.biophys.35.040405.102134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16689642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFSlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2006&pages=389-415&author=Y.+Levyauthor=J.+N.+Onuchic&title=Water+Mediation+in+Protein+Folding+and+Molecular+Recognition&doi=10.1146%2Fannurev.biophys.35.040405.102134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Water mediation in protein folding and molecular recognition</span></div><div class="casAuthors">Levy, Yaakov; Onuchic, Jose N.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics and Biomolecular Structure</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">389-415</span>CODEN:
                <span class="NLM_cas:coden">ABBSE4</span>;
        ISSN:<span class="NLM_cas:issn">1056-8700</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Water is essential for life in many ways, and without it biomols. might no longer truly be biomols.  In particular, water is important to the structure, stability, dynamics, and function of biol. macromols.  In protein folding, water mediates the collapse of the chain and the search for the native topol. through a funneled energy landscape.  Water actively participates in mol. recognition by mediating the interactions between binding partners and contributes to either enthalpic or entropic stabilization.  Accordingly, water must be included in recognition and structure prediction codes to capture specificity.  Thus, water should not be treated as an inert environment, but rather as an integral and active component of biomol. systems, where it has both dynamic and structural roles.  Focusing on water sheds light on the physics and function of biol. machinery and self-assembly and may advance the understanding of the natural design of proteins and nucleic acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpV8LpRDAZZZLVg90H21EOLACvtfcHk0lhEOtBTgyOqXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFSlu7c%253D&md5=3c2402d22d7c2fae5d1bccd9fb60a5b7</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.35.040405.102134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.35.040405.102134%26sid%3Dliteratum%253Aachs%26aulast%3DLevy%26aufirst%3DY.%26aulast%3DOnuchic%26aufirst%3DJ.%2BN.%26atitle%3DWater%2520Mediation%2520in%2520Protein%2520Folding%2520and%2520Molecular%2520Recognition%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2006%26volume%3D35%26spage%3D389%26epage%3D415%26doi%3D10.1146%2Fannurev.biophys.35.040405.102134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guvench, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, C. L.</span></span> <span> </span><span class="NLM_article-title">Receptor Rigidity and Ligand Mobility in Trypsin-Ligand Complexes</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1002/prot.20326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2Fprot.20326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=15578663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1Omtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2005&pages=407-417&author=O.+Guvenchauthor=D.+J.+Priceauthor=C.+L.+Brooks&title=Receptor+Rigidity+and+Ligand+Mobility+in+Trypsin-Ligand+Complexes&doi=10.1002%2Fprot.20326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor rigidity and ligand mobility in trypsin-ligand complexes</span></div><div class="casAuthors">Guvench, Olgun; Price, Daniel J.; Brooks, Charles L., III</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">407-417</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The trypsin-like serine proteases comprise a structurally similar family of proteins with a wide diversity of biol. functions.  Members of this family play roles in digestion, hemostasis, immune responses, and cancer metastasis.  Bovine trypsin is an archetypical member of this family that has been extensively characterized both functionally and structurally, and that preferentially hydrolyzes Arg/Lys-Xaa peptide bonds.  The authors have used mol. dynamics (MD) simulations to study bovine trypsin complexed with the two noncovalent small-mol. ligands, benzamidine and tranylcypromine, that have the same hydrogen-bond donating moieties as Arg and Lys side-chains, resp.  Multiple (10) simulations ranging from 1 ns to 2.2 ns, with explicit water mols. and periodic boundary conditions, were performed.  The simulations reveal that the trypsin binding pocket residues are relatively rigid regardless of whether there is no ligand, a high-affinity ligand (benzamidine), or a low-affinity ligand (tranylcypromine).  The thermal av. of the conformations sampled by benzamidine bound to trypsin is planar and consistent with the planar internal geometry of the benzamidine crystallog. model coordinates.  However, the most probable bound benzamidine conformations are ± 25° out of plane, implying that the obsd. x-ray electron d. represents an av. of densities from two mirror sym., nonplanar conformations.  Solvated benzamidine has free energy min. at ± 45°, and the induction of a more planar geometry upon binding is assocd. with ∼1 kcal/mol of intramol. strain.  Tranylcypromine's hydrogen-bonding pattern in the MD differs substantially from that inferred from the x-ray electron d.  Early in simulations of this system, tranylcypromine adopts an alternative binding conformation, changing from the crystallog. conformation, with a direct hydrogen bond between its amino moiety and the backbone oxygen of Gly-219, to one having a bridging water mol.  This result is consistently seen with the CHARMM22, Amber, or OPLS-AA force fields.  The trypsin-tranylcypromine hydrogen-bonding pattern obsd. in the simulations also occurs as the crystallog. binding mode of the Lys-15 side-chain of bovine pancreatic trypsin inhibitor bound to trypsin.  In this latter cocrystal, a bridging crystallog. water does reside between the side-chain's amino group and the trypsin Gly-219 backbone oxygen.  Furthermore, the trypsin-tranylcypromine simulations sample two different stable noncrystallog. binding poses.  These data suggest that some of the electron d. ascribed to tranylcypromine in the x-ray model is rather due to a bound water mol., and that multiple tranylcypromine binding conformations (crystallog. disorder) may be the cause of ambiguous electron d.  The combined trypsin-benzamidine and trypsin-tranylcypromine results highlight the ability of simulations to augment protein-ligand complex structural data by deconvoluting the effects of thermal and structural averaging, and by finding energetically optimal ligand and bound water positions for weakly bound ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh054GN8do9rVg90H21EOLACvtfcHk0lhEOtBTgyOqXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1Omtw%253D%253D&md5=0e5fff622c2c3b02d39ad70606e0bf52</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fprot.20326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20326%26sid%3Dliteratum%253Aachs%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26atitle%3DReceptor%2520Rigidity%2520and%2520Ligand%2520Mobility%2520in%2520Trypsin-Ligand%2520Complexes%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2005%26volume%3D58%26spage%3D407%26epage%3D417%26doi%3D10.1002%2Fprot.20326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Z. H.</span></span> <span> </span><span class="NLM_article-title">Selectivity of Neutral/Weakly Basic P1 Group Inhibitors of Thrombin and Trypsin by a Molecular Dynamics Study</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">8704</span>– <span class="NLM_lpage">8714</span>, <span class="refDoi"> DOI: 10.1002/chem.200800277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2Fchem.200800277" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=18680136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1OltrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=8704-8714&author=E.+L.+Wuauthor=K.+L.+Hanauthor=J.+Z.+H.+Zhang&title=Selectivity+of+Neutral%2FWeakly+Basic+P1+Group+Inhibitors+of+Thrombin+and+Trypsin+by+a+Molecular+Dynamics+Study&doi=10.1002%2Fchem.200800277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity of neutral/weakly basic P1 group inhibitors of thrombin and trypsin by a molecular dynamics study</span></div><div class="casAuthors">Wu, Emilia L.; Han, KeLi; Zhang, John Z. H.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8704-8714</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations followed by mol. mechanics generalized Born surface area (MM-GBSA) analyses have been carried out to study the selectivity of two neutral and weakly basic P1 group inhibitors (177 and CDA) to thrombin and trypsin.  Detailed binding free energies between these inhibitors and individual protein residues are calcd. by using a per-residue basis decompn. method.  The anal. of the detailed interaction energies provides insight on the protein-inhibitor-binding mechanism and helps to elucidate the basis for achieving selectivity through interpretation of the structural and energetic results from the simulations.  The study shows that the dominant factor of selectivity for both inhibitors is van der Waals energy, which suggests better shape complementarity and packing with thrombin.  Nonpolar solvation free energy and total entropy contribution are also in favor of selectivity, but the contributions are much smaller.  Binding mode and structural anal. show that 177 binds to thrombin and trypsin in a similar binding mode.  In contrast, the CDA binds to thrombin and trypsin in very different modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoUaN5f4UltbVg90H21EOLACvtfcHk0lj-OP9fPzu0Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1OltrnP&md5=dfb0aac1de205082e166f2d02ccbdf58</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fchem.200800277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200800277%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DE.%2BL.%26aulast%3DHan%26aufirst%3DK.%2BL.%26aulast%3DZhang%26aufirst%3DJ.%2BZ.%2BH.%26atitle%3DSelectivity%2520of%2520Neutral%252FWeakly%2520Basic%2520P1%2520Group%2520Inhibitors%2520of%2520Thrombin%2520and%2520Trypsin%2520by%2520a%2520Molecular%2520Dynamics%2520Study%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2008%26volume%3D14%26spage%3D8704%26epage%3D8714%26doi%3D10.1002%2Fchem.200800277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilpert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banner, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gast, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadváry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Waterbeemd, H.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Potent and Highly Selective Thrombin Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3889</span>– <span class="NLM_lpage">3901</span>, <span class="refDoi"> DOI: 10.1021/jm00049a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00049a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2MXhs12hsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=3889-3901&author=K.+Hilpertauthor=J.+Ackermannauthor=D.+W.+Bannerauthor=A.+Gastauthor=K.+Gubernatorauthor=P.+Hadv%C3%A1ryauthor=L.+Lablerauthor=K.+M%C3%BCllerauthor=G.+Schmidauthor=T.+B.+Tschoppauthor=H.+van+de+Waterbeemd&title=Design+and+Synthesis+of+Potent+and+Highly+Selective+Thrombin+Inhibitors&doi=10.1021%2Fjm00049a008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Potent and Highly Selective Thrombin Inhibitors</span></div><div class="casAuthors">Hilpert, Kurt; Ackermann, Jean; Banner, David W.; Gast, Alain; Gubernator, Klaus; Hadvary, Paul; Labler, Ludvik; Mueller, Klaus; Schmid, Gerard; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3889-901</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Thrombin, a serine protease, plays a central role in the initiation and propagation of thrombotic events.  An extensive search for new thrombin inhibitors was performed, using an unconventional approach.  Screening of small basic mols. for binding in the recognition pocket of thrombin led to the discovery of (aminoiminomethyl)piperidine (amidinopiperidine) as a weak, but intrinsically selective, thrombin inhibitor.  Elaboration of this mol. provided compds. which inhibit thrombin with Ki's in the range of 20-50 nM and with selectivities of 1000-4000 against trypsin.  These inhibitor compds. show a new and unexpected binding mode to thrombin.  Modification of the central building block and then of one of the hydrophobic substituents led to the discovery of a new family of thrombin inhibitors which has reverted to the former binding mode to thrombin.  This last class of compds. shows inhibitory activities in the picomolar range, low toxicity, and a short plasma half life which favors its use for an i.v. application.  From this series of thrombin inhibitors, Ro 46-6240 (I) was selected for clin. development as an antithrombotic agent for i.v. administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcpB1l-ixNgbVg90H21EOLACvtfcHk0lj-OP9fPzu0Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhs12hsbc%253D&md5=d1c8819926099edb6e538b8e32060e3f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm00049a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00049a008%26sid%3Dliteratum%253Aachs%26aulast%3DHilpert%26aufirst%3DK.%26aulast%3DAckermann%26aufirst%3DJ.%26aulast%3DBanner%26aufirst%3DD.%2BW.%26aulast%3DGast%26aufirst%3DA.%26aulast%3DGubernator%26aufirst%3DK.%26aulast%3DHadv%25C3%25A1ry%26aufirst%3DP.%26aulast%3DLabler%26aufirst%3DL.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26aulast%3DSchmid%26aufirst%3DG.%26aulast%3DTschopp%26aufirst%3DT.%2BB.%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Potent%2520and%2520Highly%2520Selective%2520Thrombin%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D3889%26epage%3D3901%26doi%3D10.1021%2Fjm00049a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenbucher, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprengeler, P. A.</span></span> <span> </span><span class="NLM_article-title">Exploiting Subsite S1 of Trypsin-like Serine Proteases for Selectivity: Potent and Selective Inhibitors of Urokinase-Type Plasminogen Activator</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">3856</span>– <span class="NLM_lpage">3871</span>, <span class="refDoi"> DOI: 10.1021/jm010244+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm010244%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlWrt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=3856-3871&author=R.+L.+Mackmanauthor=B.+A.+Katzauthor=J.+G.+Breitenbucherauthor=H.+C.+Huiauthor=E.+Vernerauthor=C.+Luongauthor=L.+Liuauthor=P.+A.+Sprengeler&title=Exploiting+Subsite+S1+of+Trypsin-like+Serine+Proteases+for+Selectivity%3A+Potent+and+Selective+Inhibitors+of+Urokinase-Type+Plasminogen+Activator&doi=10.1021%2Fjm010244%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting Subsite S1 of Trypsin-Like Serine Proteases for Selectivity: Potent and Selective Inhibitors of Urokinase-Type Plasminogen Activator</span></div><div class="casAuthors">Mackman, Richard L.; Katz, Bradley A.; Breitenbucher, J. Guy; Hui, Hon C.; Verner, Erik; Luong, Christine; Liu, Liang; Sprengeler, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3856-3871</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A nonselective inhibitor of trypsin-like serine proteases, 2-(2-hydroxybiphenyl-3-yl)-1H-indole-5-carboxamidine (I) (Verner, E.; Katz, B. A.; Spencer, J.; Allen, D.; Hataye, J.; Hruzewicz, W.; Hui, H. C.; Kolesnikov, A.; Li, Y.; Luong, C.; Martelli, A.; Radika. K.; Rai, R.; She, M.; Shrader, W.; Sprengeler, P. A.; Trapp, S.; Wang, J.; Young, W. B.; Mackman, R. L. J. Med. Chem. 2001, 44, 2753-2771) has been optimized through minor structural changes on the S1 binding group to afford remarkably selective and potent inhibitors of urokinase-type plasminogen activator (uPA).  The trypsin-like serine proteases1 that comprise drug targets can be broadly categorized into two subfamilies, those with Ser190 and those with Ala190.  A single-atom modification, for example, replacement of hydrogen for chlorine at the 6-position of the 5-amidinoindole P1 group on I, generated ≤6700-fold selectivity toward the Ser190 enzymes and against the Ala190 enzymes.  The larger chlorine atom displaces a water mol. (H2O1S1) that binds near residue 190 in all the complexes of I, and related inhibitors, in uPA, thrombin, and trypsin.  The water mol., H2O1S1, in both the Ser190 or Ala190 enzymes, hydrogen bonds with the amidine N1 nitrogen of the inhibitor.  When it is displaced, a redn. in affinity toward the Ala190 enzymes is obsd. due to the amidine N1 nitrogen of the bound inhibitor being deprived of a key hydrogen-bonding partner.  In the Ser190 enzymes the affinity is maintained since the serine hydroxyl oxygen OγSer190 compensates for the displaced water mol.  High-resoln. crystallog. provided evidence for the displacement of the water mol. and validated the design rationale.  In summation, a novel and powerful method for engineering selectivity toward Ser190 proteases and against Ala190 proteases without substantially increasing mol. wt. is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZsTVtV-cArVg90H21EOLACvtfcHk0lhKEP4Flt_shA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlWrt7Y%253D&md5=d431091b463a9168c4e7223b7f96f460</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm010244%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm010244%252B%26sid%3Dliteratum%253Aachs%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DKatz%26aufirst%3DB.%2BA.%26aulast%3DBreitenbucher%26aufirst%3DJ.%2BG.%26aulast%3DHui%26aufirst%3DH.%2BC.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DLuong%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26atitle%3DExploiting%2520Subsite%2520S1%2520of%2520Trypsin-like%2520Serine%2520Proteases%2520for%2520Selectivity%253A%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520Urokinase-Type%2520Plasminogen%2520Activator%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D3856%26epage%3D3871%26doi%3D10.1021%2Fjm010244%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sichler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopfner, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopetzki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, H.</span></span> <span> </span><span class="NLM_article-title">The Influence of Residue 190 in the S1 Site of Trypsin-like Serine Proteases on Substrate Selectivity Is Universally Conserved</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>530</i></span>,  <span class="NLM_fpage">220</span>– <span class="NLM_lpage">224</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(02)03495-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0014-5793%2802%2903495-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=12387896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD38XnvVOrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=530&publication_year=2002&pages=220-224&author=K.+Sichlerauthor=K.+P.+Hopfnerauthor=E.+Kopetzkiauthor=R.+Huberauthor=W.+Bodeauthor=H.+Brandstetter&title=The+Influence+of+Residue+190+in+the+S1+Site+of+Trypsin-like+Serine+Proteases+on+Substrate+Selectivity+Is+Universally+Conserved&doi=10.1016%2FS0014-5793%2802%2903495-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of residue 190 in the S1 site of trypsin-like serine proteases on substrate selectivity is universally conserved</span></div><div class="casAuthors">Sichler, Katrin; Hopfner, Karl-Peter; Kopetzki, Erhard; Huber, Robert; Bode, Wolfram; Brandstetter, Hans</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">530</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">220-224</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We examd. the influence of Ser/Ala190 in the S1 site on P1 substrate selectivity in several serine proteases.  The impact of residue 190 on the selectivity was const., regardless of differences in original selectivity or reactivity.  Substrate binding in S1 was optimized in all wild-type enzymes, while the effects on kcat depended on the combination of residue 190 and substrate.  Mutagenesis of residue 190 did not affect the S2-S4 sites.  Pronounced selectivity for arginine residues was coupled with low enzymic activity, in particular in recombinant factor IXa.  This is due to the dominance of the S1-P1 interaction over substrate binding in the S2-S4 sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZMSRLAdkTjbVg90H21EOLACvtfcHk0lhKEP4Flt_shA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnvVOrsL4%253D&md5=c18d833220d626ad03e2abed9efd00db</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2802%2903495-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252802%252903495-6%26sid%3Dliteratum%253Aachs%26aulast%3DSichler%26aufirst%3DK.%26aulast%3DHopfner%26aufirst%3DK.%2BP.%26aulast%3DKopetzki%26aufirst%3DE.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DBrandstetter%26aufirst%3DH.%26atitle%3DThe%2520Influence%2520of%2520Residue%2520190%2520in%2520the%2520S1%2520Site%2520of%2520Trypsin-like%2520Serine%2520Proteases%2520on%2520Substrate%2520Selectivity%2520Is%2520Universally%2520Conserved%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D530%26spage%3D220%26epage%3D224%26doi%3D10.1016%2FS0014-5793%2802%2903495-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lobo, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wos, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, W. B.</span></span> <span> </span><span class="NLM_article-title">Active Site Studies of Human Thrombin and Bovine Trypsin: Peptide Substrates</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>177</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1016/0003-9861(76)90433-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2F0003-9861%2876%2990433-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1033732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaE28XlvVyhu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=177&publication_year=1976&pages=235-244&author=A.+P.+Loboauthor=J.+D.+Wosauthor=S.+M.+Yuauthor=W.+B.+Lawson&title=Active+Site+Studies+of+Human+Thrombin+and+Bovine+Trypsin%3A+Peptide+Substrates&doi=10.1016%2F0003-9861%2876%2990433-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Active site studies of human thrombin and bovine trypsin:  peptide substrates</span></div><div class="casAuthors">Lobo, Angelo P.; Wos, J. David; Yu, Sherman M.; Lawson, William B.</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">177</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-44</span>CODEN:
                <span class="NLM_cas:coden">ABBIA4</span>;
        ISSN:<span class="NLM_cas:issn">0003-9861</span>.
    </div><div class="casAbstract">Twenty-four oligopeptides modeled after the N-terminal portion of the α(A)-chain of human fibrinogen were synthesized and tested as substrates for human thrombin and bovine trypsin.  The peptides contained either an Arg-Gly bond, or an Arg-Val bond, or both.  Glycine and glutamic acid were substituted at various positions within the peptides, and from the kcat/Km for each peptide with thrombin and with trypsin, the importance of residues on both sides of the Arg-Gly bond was evaluated.  The trypsin-catalyzed hydrolysis was faster than the thrombin-catalyzed hydrolysis of the peptides with glutamic acid at various distances N-terminal to the Arg-Gly bond.  The ratio of the thrombic rate to the tryptic rate was fastest when glutamic acid was 6 residues away.  Thrombin selectively cleaved the Arg-Gly bond in Ac-Val-Arg-Gly-Pro-Arg-Val-OH but cleaved both arginyl bonds rapidly in Ac-Val-Arg-Gly-Pro-Arg-Val-OMe.  Trypsin selectively cleaved an Arg-Val bond in Ac-Arg-Gly-Pro-Arg-Val-OMe.  The results are discussed in the light of thrombic cleavages of proteins.  Most of these are seen to occur at highly polar sequences that frequently contain a proline residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGZnePSk20jLVg90H21EOLACvtfcHk0lhKEP4Flt_shA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XlvVyhu70%253D&md5=abd825a4ea3a9cc76a90d79065eff73e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2F0003-9861%2876%2990433-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-9861%252876%252990433-1%26sid%3Dliteratum%253Aachs%26aulast%3DLobo%26aufirst%3DA.%2BP.%26aulast%3DWos%26aufirst%3DJ.%2BD.%26aulast%3DYu%26aufirst%3DS.%2BM.%26aulast%3DLawson%26aufirst%3DW.%2BB.%26atitle%3DActive%2520Site%2520Studies%2520of%2520Human%2520Thrombin%2520and%2520Bovine%2520Trypsin%253A%2520Peptide%2520Substrates%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D1976%26volume%3D177%26spage%3D235%26epage%3D244%26doi%3D10.1016%2F0003-9861%2876%2990433-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallwitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enoksson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorpe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellman, L.</span></span> <span> </span><span class="NLM_article-title">The Extended Cleavage Specificity of Human Thrombin</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e31756</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0031756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1371%2Fjournal.pone.0031756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=22384068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFylsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=M.+Gallwitzauthor=M.+Enokssonauthor=M.+Thorpeauthor=L.+Hellman&title=The+Extended+Cleavage+Specificity+of+Human+Thrombin&doi=10.1371%2Fjournal.pone.0031756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The extended cleavage specificity of human thrombin</span></div><div class="casAuthors">Gallwitz, Maike; Enoksson, Mattias; Thorpe, Michael; Hellman, Lars</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e31756</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Thrombin is one of the most extensively studied of all proteases.  Its central role in the coagulation cascade as well as several other areas has been thoroughly documented.  Despite this, its consensus cleavage site has never been detd. in detail.  Here we have detd. its extended substrate recognition profile using phage-display technol.  The consensus recognition sequence was identified as, P2-Pro, P1-Arg, P1'-Ser/Ala/Gly/Thr, P2'-not acidic and P3'-Arg.  Our anal. also identifies an important role for a P3'-arginine in thrombin substrates lacking a P2-proline.  In order to study kinetics of this cooperative or additive effect we developed a system for insertion of various pre-selected cleavable sequences in a linker region between two thioredoxin mols.  Using this system we show that mutations of P2-Pro and P3'-Arg lead to an approx. 20-fold and 14-fold redn., resp. in the rate of cleavage.  Mutating both Pro and Arg results in a drop in cleavage of 200-400 times, which highlights the importance of these two positions for maximal substrate cleavage.  Interestingly, no natural substrates display the obtained consensus sequence but represent sequences that show only 1-30 % of the optimal cleavage rate for thrombin.  This clearly indicates that maximal cleavage, excluding the help of exosite interactions, is not always desired, which may instead cause problems with dysregulated coagulation.  It is likely exosite cooperativity has a central role in detg. the specificity and rate of cleavage of many of these in vivo substrates.  Major effects on cleavage efficiency were also obsd. for residues as far away as 4 amino acids from the cleavage site.  Insertion of an aspartic acid in position P4 resulted in a drop in cleavage by a factor of almost 20 times.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLokncNWnNKbVg90H21EOLACvtfcHk0ljwUVuM77-4PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsFylsLc%253D&md5=ed3355e1e0762bf9618781f4a0ed0b7c</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0031756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0031756%26sid%3Dliteratum%253Aachs%26aulast%3DGallwitz%26aufirst%3DM.%26aulast%3DEnoksson%26aufirst%3DM.%26aulast%3DThorpe%26aufirst%3DM.%26aulast%3DHellman%26aufirst%3DL.%26atitle%3DThe%2520Extended%2520Cleavage%2520Specificity%2520of%2520Human%2520Thrombin%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0031756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zayed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">More than a Simple Lipophilic Contact: A Detailed Thermodynamic Analysis of Nonbasic Residues in the S1 Pocket of Thrombin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.04.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2009.04.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19409395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Glu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=56-69&author=B.+Baumauthor=M.+Mohamedauthor=M.+Zayedauthor=C.+Gerlachauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=More+than+a+Simple+Lipophilic+Contact%3A+A+Detailed+Thermodynamic+Analysis+of+Nonbasic+Residues+in+the+S1+Pocket+of+Thrombin&doi=10.1016%2Fj.jmb.2009.04.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">More than a Simple Lipophilic Contact: A Detailed Thermodynamic Analysis of Nonbasic Residues in the S1 Pocket of Thrombin</span></div><div class="casAuthors">Baum, Bernhard; Mohamed, Menshawy; Zayed, Mohamed; Gerlach, Christof; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-69</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The field of medicinal chem. aims to design and optimize small mol. leads into drug candidates that may pos. interfere with pathol. disease situations in humans or combat the growth of infective pathogens.  From the plethora of crystal structures of protein-inhibitor complexes we have learned how mols. recognize each other geometrically, but we still have rather superficial understanding of why they bind to each other.  This contribution surveys a series of 26 thrombin inhibitors with small systematic structural differences to elucidate the rationale for their widely deviating binding affinity from 185 μM to 4 nM as recorded by enzyme kinetic measurements.  Five well-resolved (resoln. 2.30-1.47 Å) crystal structures of thrombin-inhibitor complexes and an apo-structure of the uncomplexed enzyme (1.50 Å) are correlated with thermodn. data recorded by isothermal titrn. calorimetry with 12 selected inhibitors from the series.  Taking soly. data into account, the variation in physicochem. properties allows conclusions to be reached about the relative importance of the enthalpic binding features as well as to est. the importance of the parameters more difficult to capture, such as residual ligand entropy and desolvation properties.  The collected data reveal a comprehensive picture of the thermodn. signature that explains the so far poorly understood attractive force experienced by m-chloro-benzylamides to thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvKms7QZtnUrVg90H21EOLACvtfcHk0ljwUVuM77-4PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Glu7c%253D&md5=ce3cf23e3a9246fc5084643b9a5a7ab4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.04.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.04.051%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMohamed%26aufirst%3DM.%26aulast%3DZayed%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DC.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DMore%2520than%2520a%2520Simple%2520Lipophilic%2520Contact%253A%2520A%2520Detailed%2520Thermodynamic%2520Analysis%2520of%2520Nonbasic%2520Residues%2520in%2520the%2520S1%2520Pocket%2520of%2520Thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D390%26spage%3D56%26epage%3D69%26doi%3D10.1016%2Fj.jmb.2009.04.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freindorf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D. G.</span></span> <span> </span><span class="NLM_article-title">Enhancement of Hydrophobic Interactions and Hydrogen Bond Strength by Cooperativity: Synthesis, Modeling and Molecular Dynamics Simulations of a Congeneric Series of Thrombin Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2135</span>, <span class="refDoi"> DOI: 10.1021/jm9016416</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9016416" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2126-2135&author=L.+Muleyauthor=B.+Baumauthor=M.+Smolinskiauthor=M.+Freindorfauthor=A.+Heineauthor=G.+Klebeauthor=D.+G.+Hangauer&title=Enhancement+of+Hydrophobic+Interactions+and+Hydrogen+Bond+Strength+by+Cooperativity%3A+Synthesis%2C+Modeling+and+Molecular+Dynamics+Simulations+of+a+Congeneric+Series+of+Thrombin+Inhibitors&doi=10.1021%2Fjm9016416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancement of Hydrophobic Interactions and Hydrogen Bond Strength by Cooperativity: Synthesis, Modeling, and Molecular Dynamics Simulations of a Congeneric Series of Thrombin Inhibitors</span></div><div class="casAuthors">Muley, Laveena; Baum, Bernhard; Smolinski, Michael; Freindorf, Marek; Heine, Andreas; Klebe, Gerhard; Hangauer, David G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2126-2135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Accurately predicting the binding affinity of ligands to their receptors by computational methods is one of the major challenges in structure-based drug design.  One of the potentially significant errors in these predictions is the common assumption that the ligand binding affinity contributions of noncovalent interactions are additive.  Herein we present data obtained from two sep. series of thrombin inhibitors contg. hydrophobic side chains of increasing size that bind in the S3 pocket and with, or without, an adjacent amine that engages in a hydrogen bond with Gly 216.  The first series of inhibitors has a m-chlorobenzyl moiety binding in the S1 pocket, and the second has a benzamidine moiety.  When the adjacent hydrogen bond is present, the enhanced binding affinity per Å2 of hydrophobic contact surface in the S3 pocket improves by 75% and 59%, resp., over the inhibitors lacking this hydrogen bond.  This improvement of the binding affinity per Å2 demonstrates cooperativity between the hydrophobic interaction and the hydrogen bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqytMqjMe1i9bVg90H21EOLACvtfcHk0ljwUVuM77-4PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFKjs7o%253D&md5=62e4474f7d6d5fa11f884c5b1e17fef9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm9016416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9016416%26sid%3Dliteratum%253Aachs%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DFreindorf%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DHangauer%26aufirst%3DD.%2BG.%26atitle%3DEnhancement%2520of%2520Hydrophobic%2520Interactions%2520and%2520Hydrogen%2520Bond%2520Strength%2520by%2520Cooperativity%253A%2520Synthesis%252C%2520Modeling%2520and%2520Molecular%2520Dynamics%2520Simulations%2520of%2520a%2520Congeneric%2520Series%2520of%2520Thrombin%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2126%26epage%3D2135%26doi%3D10.1021%2Fjm9016416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khayat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Impact of Ligand and Protein Desolvation on Ligand Binding to the S1 Pocket of Thrombin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>418</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">366</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2012.01.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2012.01.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=22366545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFSmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=418&publication_year=2012&pages=350-366&author=A.+Bielaauthor=M.+Khayatauthor=H.+Tanauthor=J.+Kongauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=Impact+of+Ligand+and+Protein+Desolvation+on+Ligand+Binding+to+the+S1+Pocket+of+Thrombin&doi=10.1016%2Fj.jmb.2012.01.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Ligand and Protein Desolvation on Ligand Binding to the S1 Pocket of Thrombin</span></div><div class="casAuthors">Biela, A.; Khayat, M.; Tan, H.; Kong, J.; Heine, A.; Hangauer, D.; Klebe, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">418</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">350-366</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In the present study, we investigate the impact of a tightly bound water mol. on ligand binding in the S1 pocket of thrombin.  The S1 pocket contains a deeply buried deprotonated aspartate residue (Asp189) which due to its charged state, is well hydrated in the uncomplexed state.  We systematically studied the importance of this water mol. by evaluating a series of ligands that contains pyridine-type P1 side chains that could potentially alter the binding properties of this water mol.  All of the pyridine derivs. retain the original hydration state albeit sometimes with a slight perturbance.  In order to prevent a direct H-bond formation with Asp189, and to create a permanent pos. charge on the P1 side chain that is positioned adjacent to the Asp189 carboxylate anion, we methylated the pyridine nitrogen.  This methylation resulted in displacement of water but was accompanied by a loss in binding affinity.  Quantum chem. calcns. of the ligand solvation free energy showed that the pos. charged methylpyridinium derivs. suffer a large penalty of desolvation upon binding.  Consequently, they have a substantially less favorable enthalpy of binding.  In addn. to the ligand desolvation penalty, the hydration shell around Asp189 has to be overcome, which is achieved in nearly all pyridinium derivs.  Only for the ortho deriv. is a partial population of a water next to Asp189 found.  Possibly, the gain of electrostatic interactions between the charged P1 side chain and Asp189 helps to compensate for the desolvation penalty.  In all uncharged pyridine derivs., the solvation shell remains next to Asp189, partly mediating interactions between ligand and protein.  In the case of the para-pyridine deriv., a strongly disordered cluster of water sites is obsd. between ligand and Asp189.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq7O_AkRSsXbVg90H21EOLACvtfcHk0lgYYNaygVIH1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFSmsL8%253D&md5=f52d76d0ef544bede3145c0ea9026d06</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.01.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.01.054%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DKhayat%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DJ.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DImpact%2520of%2520Ligand%2520and%2520Protein%2520Desolvation%2520on%2520Ligand%2520Binding%2520to%2520the%2520S1%2520Pocket%2520of%2520Thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D418%26spage%3D350%26epage%3D366%26doi%3D10.1016%2Fj.jmb.2012.01.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardell, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bock, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freidinger, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor-Olsen, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramjit, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woltmann, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shafer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dancheck, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hare, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulichak, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J. P.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel, Selective, and Orally Bioavailable Class of Thrombin Inhibitors Incorporating Aminopyridyl Moieties at the P1 Position</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">3726</span>– <span class="NLM_lpage">3733</span>, <span class="refDoi"> DOI: 10.1021/jm970493r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970493r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXmslWhs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=3726-3733&author=D.+M.+Fengauthor=S.+J.+Gardellauthor=S.+D.+Lewisauthor=M.+G.+Bockauthor=Z.+Chenauthor=R.+M.+Freidingerauthor=A.+M.+Naylor-Olsenauthor=H.+G.+Ramjitauthor=R.+Woltmannauthor=E.+P.+Baskinauthor=J.+J.+Lynchauthor=R.+Lucasauthor=J.+A.+Shaferauthor=K.+B.+Dancheckauthor=I-W.+Chenauthor=S.-S.+Maoauthor=J.+A.+Kruegerauthor=T.+R.+Hareauthor=A.+M.+Mulichakauthor=J.+P.+Vacca&title=Discovery+of+a+Novel%2C+Selective%2C+and+Orally+Bioavailable+Class+of+Thrombin+Inhibitors+Incorporating+Aminopyridyl+Moieties+at+the+P1+Position&doi=10.1021%2Fjm970493r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel, Selective, and Orally Bioavailable Class of Thrombin Inhibitors Incorporating Aminopyridyl Moieties at the P1 Position</span></div><div class="casAuthors">Feng, Dong-Mei; Gardell, Stephen J.; Lewis, S. Dale; Bock, Mark G.; Chen, Zhongguo; Freidinger, Roger M.; Naylor-Olsen, Adel; Ramjit, Harri G.; Woltmann, Richard; Baskin, Elizabeth P.; Lynch, Joseph J.; Lucas, Robert; Shafer, Jules A.; Dancheck, Kimberley B.; Chen, I-Wu; Mao, Shi-Shan; Krueger, Julie A.; Hare, Timothy R.; Mulichak, Anne M.; Vacca, Joseph P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3726-3733</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel class of thrombin inhibitors incorporating aminopyridyl moieties at the P1 position has been discovered.  Four of these thrombin inhibitors showed nanomolar potency (Ki 0.8-12 nM), 300-1500-fold selectivity for thrombin compared with trypsin, and good oral bioavailability (F = 40-76%) in rats or dogs.  The neutral P1 was expected to increase metabolic stability and oral absorption.  Identification of this novel aminopyridyl group at P1 was a key step in our search for a clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHTbN5pkVq9bVg90H21EOLACvtfcHk0lgYYNaygVIH1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmslWhs7c%253D&md5=1ceb0ac6aa7e59570d4707c295c5ffcc</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm970493r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970493r%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DD.%2BM.%26aulast%3DGardell%26aufirst%3DS.%2BJ.%26aulast%3DLewis%26aufirst%3DS.%2BD.%26aulast%3DBock%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DFreidinger%26aufirst%3DR.%2BM.%26aulast%3DNaylor-Olsen%26aufirst%3DA.%2BM.%26aulast%3DRamjit%26aufirst%3DH.%2BG.%26aulast%3DWoltmann%26aufirst%3DR.%26aulast%3DBaskin%26aufirst%3DE.%2BP.%26aulast%3DLynch%26aufirst%3DJ.%2BJ.%26aulast%3DLucas%26aufirst%3DR.%26aulast%3DShafer%26aufirst%3DJ.%2BA.%26aulast%3DDancheck%26aufirst%3DK.%2BB.%26aulast%3DChen%26aufirst%3DI-W.%26aulast%3DMao%26aufirst%3DS.-S.%26aulast%3DKrueger%26aufirst%3DJ.%2BA.%26aulast%3DHare%26aufirst%3DT.%2BR.%26aulast%3DMulichak%26aufirst%3DA.%2BM.%26aulast%3DVacca%26aufirst%3DJ.%2BP.%26atitle%3DDiscovery%2520of%2520a%2520Novel%252C%2520Selective%252C%2520and%2520Orally%2520Bioavailable%2520Class%2520of%2520Thrombin%2520Inhibitors%2520Incorporating%2520Aminopyridyl%2520Moieties%2520at%2520the%2520P1%2520Position%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D3726%26epage%3D3733%26doi%3D10.1021%2Fjm970493r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konishi, Y.</span></span> <span> </span><span class="NLM_article-title">Potent Bivalent Thrombin Inhibitors: Replacement of the Scissile Peptide Bond at P1-P1′ with Arginyl Ketomethylene Isosteres</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">3109</span>– <span class="NLM_lpage">3115</span>, <span class="refDoi"> DOI: 10.1021/jm9807297</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9807297" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1MXktl2iuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=3109-3115&author=T.+Steinmetzerauthor=B.+Y.+Zhuauthor=Y.+Konishi&title=Potent+Bivalent+Thrombin+Inhibitors%3A+Replacement+of+the+Scissile+Peptide+Bond+at+P1-P1%E2%80%B2+with+Arginyl+Ketomethylene+Isosteres&doi=10.1021%2Fjm9807297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Bivalent Thrombin Inhibitors: Replacement of the Scissile Peptide Bond at P1-P1' with Arginyl Ketomethylene Isosteres</span></div><div class="casAuthors">Steinmetzer, Torsten; Zhu, Bing Yan; Konishi, Yasuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3109-3115</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have designed highly potent synthetic bivalent thrombin inhibitors, which consist of an active site blocking segment, a fibrinogen recognition exosite blocking segment, and a linker connecting these segments.  The bivalent inhibitors bind to the active site and the fibrinogen recognition exosite simultaneously.  As a result, the inhibitors showed much higher affinity for thrombin than the individual blocking segments.  Various arginyl ketomethylene isosteres ArgΨ[CO-CH2-X]P1' were incorporated into the bivalent inhibitors as P1-P1' segment to eliminate the scissile bond.  The P1' residue is a natural or unnatural amino acid; specifically, the incorporation of mercaptoacetic acid exhibited superiority in synthesis and affinity for thrombin.  Inhibitor (D-cyclohexylalanine)-Pro-ArgΨ[CO-CH2-S]Gly-(Gly)4-Asp-Tyr-Glu-Pro-Ile-Pro-Glu-Glu-Tyr-cyclohexylalanine-(D-Glu)-OH showed the lowest Ki value of 3.5×10-13 M, which is comparable to that (Ki = 2.3×10-13 M) of recombinant hirudin.  Consequently the authors successfully reduced the size of the inhibitor from ∼7 kDa of recombinant hirudin to ∼2 kDa without losing the affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdArz5KnXR4rVg90H21EOLACvtfcHk0lgYYNaygVIH1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktl2iuro%253D&md5=40f938bfe8dd7a1196a73c988111d607</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm9807297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9807297%26sid%3Dliteratum%253Aachs%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DB.%2BY.%26aulast%3DKonishi%26aufirst%3DY.%26atitle%3DPotent%2520Bivalent%2520Thrombin%2520Inhibitors%253A%2520Replacement%2520of%2520the%2520Scissile%2520Peptide%2520Bond%2520at%2520P1-P1%25E2%2580%25B2%2520with%2520Arginyl%2520Ketomethylene%2520Isosteres%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D3109%26epage%3D3115%26doi%3D10.1021%2Fjm9807297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brodbeck, B.</span>; <span class="NLM_string-name">Nettekoven, M. H.</span></span> <span> </span><span class="NLM_article-title">Substituted[1,2,4]Triazolo[1,5A]Pyridine Derivatives with Activity as Adenosine Receptor Ligands</span>. U.S. Patent <span class="NLM_patent">US6,506,772 B1</span><span class="NLM_year">2003</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=B.+Brodbeck&author=M.+H.+Nettekoven&title=Substituted%5B1%2C2%2C4%5DTriazolo%5B1%2C5A%5DPyridine+Derivatives+with+Activity+as+Adenosine+Receptor+Ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBrodbeck%26aufirst%3DB.%26atitle%3DSubstituted%255B1%252C2%252C4%255DTriazolo%255B1%252C5A%255DPyridine%2520Derivatives%2520with%2520Activity%2520as%2520Adenosine%2520Receptor%2520Ligands%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meredith, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beattie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capparelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPietro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enyedy, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosagrahara, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jewell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemon, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinsey, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagratis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plato, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozhitskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soldermann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Eis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monovich, L. G.</span></span> <span> </span><span class="NLM_article-title">Identification of Potent and Selective Amidobipyridyl Inhibitors of Protein Kinase D</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5422</span>– <span class="NLM_lpage">5438</span>, <span class="refDoi"> DOI: 10.1021/jm100076w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100076w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5422-5438&author=E.+L.+Meredithauthor=K.+Beattieauthor=R.+Burgisauthor=M.+Capparelliauthor=J.+Chapoauthor=L.+DiPietroauthor=G.+Gamberauthor=I.+Enyedyauthor=D.+B.+Hoodauthor=V.+Hosagraharaauthor=C.+Jewellauthor=K.+A.+Kochauthor=W.+Leeauthor=D.+D.+Lemonauthor=T.+A.+McKinseyauthor=K.+Mirandaauthor=N.+Pagratisauthor=D.+Phanauthor=C.+Platoauthor=C.+Raoauthor=O.+Rozhitskayaauthor=N.+Soldermannauthor=C.+Springerauthor=M.+van+Eisauthor=R.+B.+Vegaauthor=W.+Yanauthor=Q.+Zhuauthor=L.+G.+Monovich&title=Identification+of+Potent+and+Selective+Amidobipyridyl+Inhibitors+of+Protein+Kinase+D&doi=10.1021%2Fjm100076w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent and Selective Amidobipyridyl Inhibitors of Protein Kinase D</span></div><div class="casAuthors">Meredith, Erik L.; Beattie, Kimberly; Burgis, Robin; Capparelli, Michael; Chapo, Joseph; Di Pietro, Lucian; Gamber, Gabriel; Enyedy, Istvan; Hood, David B.; Hosagrahara, Vinayak; Jewell, Charles; Koch, Keith A.; Lee, Wendy; Lemon, Douglas D.; McKinsey, Timothy A.; Miranda, Karl; Pagratis, Nikos; Phan, Dillon; Plato, Craig; Rao, Chang; Rozhitskaya, Olga; Soldermann, Nicolas; Springer, Clayton; van Eis, Maurice; Vega, Richard B.; Yan, Wanlin; Zhu, Qingming; Monovich, Lauren G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5422-5438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The synthesis and biol. evaluation of potent and selective PKD (protein kinase D) inhibitors, e.g. I, are described herein.  The compds. described in the present study selectively inhibit PKD among other putative HDAC (Histone deacetylase) kinases.  The PKD inhibitors of the present study blunt phosphorylation and subsequent nuclear export of HDAC4/5 in response to diverse agonists.  These compds. further establish the central role of PKD as an HDAC4/5 kinase and enhance the current understanding of cardiac myocyte signal transduction.  The in vivo efficacy of a representative example compd. on heart morphol. is reported herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1cpMoyvA7arVg90H21EOLACvtfcHk0ljkr4v3cbxchQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCiurw%253D&md5=ac3ffbd2c87b9a952858de20ddd10a02</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm100076w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100076w%26sid%3Dliteratum%253Aachs%26aulast%3DMeredith%26aufirst%3DE.%2BL.%26aulast%3DBeattie%26aufirst%3DK.%26aulast%3DBurgis%26aufirst%3DR.%26aulast%3DCapparelli%26aufirst%3DM.%26aulast%3DChapo%26aufirst%3DJ.%26aulast%3DDiPietro%26aufirst%3DL.%26aulast%3DGamber%26aufirst%3DG.%26aulast%3DEnyedy%26aufirst%3DI.%26aulast%3DHood%26aufirst%3DD.%2BB.%26aulast%3DHosagrahara%26aufirst%3DV.%26aulast%3DJewell%26aufirst%3DC.%26aulast%3DKoch%26aufirst%3DK.%2BA.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DLemon%26aufirst%3DD.%2BD.%26aulast%3DMcKinsey%26aufirst%3DT.%2BA.%26aulast%3DMiranda%26aufirst%3DK.%26aulast%3DPagratis%26aufirst%3DN.%26aulast%3DPhan%26aufirst%3DD.%26aulast%3DPlato%26aufirst%3DC.%26aulast%3DRao%26aufirst%3DC.%26aulast%3DRozhitskaya%26aufirst%3DO.%26aulast%3DSoldermann%26aufirst%3DN.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3Dvan%2BEis%26aufirst%3DM.%26aulast%3DVega%26aufirst%3DR.%2BB.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DQ.%26aulast%3DMonovich%26aufirst%3DL.%2BG.%26atitle%3DIdentification%2520of%2520Potent%2520and%2520Selective%2520Amidobipyridyl%2520Inhibitors%2520of%2520Protein%2520Kinase%2520D%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5422%26epage%3D5438%26doi%3D10.1021%2Fjm100076w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, S. C.</span></span> <span> </span><span class="NLM_article-title">6,6′-Dibromo-4,4′-Di(Hexoxymethyl)-2,2′-Bipyridine: A New Solubilizing Building Block for Macromolecular and Supramolecular Applications</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">4696</span>– <span class="NLM_lpage">4699</span>, <span class="refDoi"> DOI: 10.1021/jo060557i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo060557i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD28XksFKks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=4696-4699&author=C.+M.+Ambauthor=S.+C.+Rasmussen&title=6%2C6%E2%80%B2-Dibromo-4%2C4%E2%80%B2-Di%28Hexoxymethyl%29-2%2C2%E2%80%B2-Bipyridine%3A+A+New+Solubilizing+Building+Block+for+Macromolecular+and+Supramolecular+Applications&doi=10.1021%2Fjo060557i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">6,6'-Dibromo-4,4'-di(hexoxymethyl)-2,2'- bipyridine: A New Solubilizing Building Block for Macromolecular and Supramolecular Applications</span></div><div class="casAuthors">Amb, Chad M.; Rasmussen, Seth C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4696-4699</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although brominated bipyridines and terpyridines are highly desirable synthetic building blocks for both ligand design and macro- or supramol. applications, few such synthetic precursors have been reported that include much-needed solubilizing groups.  Reported here is an inexpensive route to 2,6-dibromo-4-(hexoxymethyl)pyridine from citrazinic acid with an overall yield of 44% and its efficient conversion (60%) to 6,6'-dibromo-4,4'-di(hexoxymethyl)-2,2'-bipyridine via oxidative coupling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4FlTfLDwparVg90H21EOLACvtfcHk0ljkr4v3cbxchQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XksFKks7c%253D&md5=fe25ff28da15705d0171a43c7eea3cf4</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjo060557i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo060557i%26sid%3Dliteratum%253Aachs%26aulast%3DAmb%26aufirst%3DC.%2BM.%26aulast%3DRasmussen%26aufirst%3DS.%2BC.%26atitle%3D6%252C6%25E2%2580%25B2-Dibromo-4%252C4%25E2%2580%25B2-Di%2528Hexoxymethyl%2529-2%252C2%25E2%2580%25B2-Bipyridine%253A%2520A%2520New%2520Solubilizing%2520Building%2520Block%2520for%2520Macromolecular%2520and%2520Supramolecular%2520Applications%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2006%26volume%3D71%26spage%3D4696%26epage%3D4699%26doi%3D10.1021%2Fjo060557i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariza, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urpí, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viladomat, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilarrasa, J.</span></span> <span> </span><span class="NLM_article-title">One-Pot Conversion of Azides to Boc-Protected Amines with Trimethylphosphine and Boc-ON</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">9101</span>– <span class="NLM_lpage">9102</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(98)02006-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0040-4039%2898%2902006-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1cXnsFOrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1998&pages=9101-9102&author=X.+Arizaauthor=F.+Urp%C3%ADauthor=C.+Viladomatauthor=J.+Vilarrasa&title=One-Pot+Conversion+of+Azides+to+Boc-Protected+Amines+with+Trimethylphosphine+and+Boc-ON&doi=10.1016%2FS0040-4039%2898%2902006-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">One-pot conversion of azides to Boc-protected amines with trimethylphosphine and Boc-ON</span></div><div class="casAuthors">Ariza, Zavier; Urpi, Felix; Viladomat, Carles; Vilarrasa, Jaume</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">9101-9102</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Reaction of azides with trimethylphosphine followed by addn. of 2-(tert-butoxycarbonyloxyimino)-2-phenylacetonitrile (Boc-ON) at -20°C and stirring at r.t. for 5 h affords the desired Boc-amines in 87-100% yields.  Similar yields are obtained by mixing all components together (azide, Me3P, and Boc-ON) in toluene or THF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWPk82CKPvxrVg90H21EOLACvtfcHk0ljkr4v3cbxchQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnsFOrtbY%253D&md5=400516373fe743f7077fe96a8212b542</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2898%2902006-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252898%252902006-1%26sid%3Dliteratum%253Aachs%26aulast%3DAriza%26aufirst%3DX.%26aulast%3DUrp%25C3%25AD%26aufirst%3DF.%26aulast%3DViladomat%26aufirst%3DC.%26aulast%3DVilarrasa%26aufirst%3DJ.%26atitle%3DOne-Pot%2520Conversion%2520of%2520Azides%2520to%2520Boc-Protected%2520Amines%2520with%2520Trimethylphosphine%2520and%2520Boc-ON%26jtitle%3DTetrahedron%2520Lett.%26date%3D1998%26volume%3D39%26spage%3D9101%26epage%3D9102%26doi%3D10.1016%2FS0040-4039%2898%2902006-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariza, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urpí, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vilarrasa, J.</span></span> <span> </span><span class="NLM_article-title">A Practical Procedure for the Preparation of Carbamates from Azides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">7515</span>– <span class="NLM_lpage">7517</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(99)01449-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0040-4039%2899%2901449-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1MXmslemtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1999&pages=7515-7517&author=X.+Arizaauthor=F.+Urp%C3%ADauthor=J.+Vilarrasa&title=A+Practical+Procedure+for+the+Preparation+of+Carbamates+from+Azides&doi=10.1016%2FS0040-4039%2899%2901449-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A practical procedure for the preparation of carbamates from azides</span></div><div class="casAuthors">Ariza, Xavier; Urpi, Felix; Vilarrasa, Jaume</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">7515-7517</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A practical procedure for the direct conversion of azides to carbamates has been found.  High yields are obtained when primary and secondary aliph. azides, as well as α-azido esters, are treated in THF with Me3P and several chloroformates (ClCO2Bn, ClCO2Me, ClCO2Et, ClCO2CH2CCl3, ClCO2CH2CH:CH2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodReSdQbslq7Vg90H21EOLACvtfcHk0lgcDGcE3juGgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmslemtb8%253D&md5=6def48e9ec2d6fd54ba0add21786311b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2899%2901449-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252899%252901449-5%26sid%3Dliteratum%253Aachs%26aulast%3DAriza%26aufirst%3DX.%26aulast%3DUrp%25C3%25AD%26aufirst%3DF.%26aulast%3DVilarrasa%26aufirst%3DJ.%26atitle%3DA%2520Practical%2520Procedure%2520for%2520the%2520Preparation%2520of%2520Carbamates%2520from%2520Azides%26jtitle%3DTetrahedron%2520Lett.%26date%3D1999%26volume%3D40%26spage%3D7515%26epage%3D7517%26doi%3D10.1016%2FS0040-4039%2899%2901449-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elmkaddem, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmeister, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J.-L.</span></span> <span> </span><span class="NLM_article-title">Efficient Synthesis of Aminopyridine Derivatives by Copper Catalyzed Amination Reactions</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">927</span>, <span class="refDoi"> DOI: 10.1039/B916569J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1039%2FB916569J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20107652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFOisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2010&pages=925-927&author=M.+K.+Elmkaddemauthor=C.+Fischmeisterauthor=C.+M.+Thomasauthor=J.-L.+Renaud&title=Efficient+Synthesis+of+Aminopyridine+Derivatives+by+Copper+Catalyzed+Amination+Reactions&doi=10.1039%2FB916569J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient synthesis of aminopyridine derivatives by copper catalyzed amination reactions</span></div><div class="casAuthors">Elmkaddem, Mohammed K.; Fischmeister, Cedric; Thomas, Christophe M.; Renaud, Jean-Luc</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">925-927</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A copper(I) catalyzed amination reaction utilizing aq. ammonia and operating under mild conditions is presented.  This method was employed for the efficient synthesis of various aminopyridine derivs. bearing electron withdrawing and electron donating groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSFanPaj2mzrVg90H21EOLACvtfcHk0lgcDGcE3juGgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFOisrg%253D&md5=4a2759fea5464db1ae392168bf0993c7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FB916569J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FB916569J%26sid%3Dliteratum%253Aachs%26aulast%3DElmkaddem%26aufirst%3DM.%2BK.%26aulast%3DFischmeister%26aufirst%3DC.%26aulast%3DThomas%26aufirst%3DC.%2BM.%26aulast%3DRenaud%26aufirst%3DJ.-L.%26atitle%3DEfficient%2520Synthesis%2520of%2520Aminopyridine%2520Derivatives%2520by%2520Copper%2520Catalyzed%2520Amination%2520Reactions%26jtitle%3DChem.%2520Commun.%26date%3D2010%26volume%3D46%26spage%3D925%26epage%3D927%26doi%3D10.1039%2FB916569J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerber, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Charges Shift Protonation: Neutron Diffraction Reveals That Aniline and 2-Aminopyridine Become Protonated Upon Binding to Trypsin</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">4887</span>– <span class="NLM_lpage">4890</span>, <span class="refDoi"> DOI: 10.1002/anie.201701038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2Fanie.201701038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=4887-4890&author=J.+Schiebelauthor=R.+Gaspariauthor=A.+Sandnerauthor=K.+Ngoauthor=H.+D.+Gerberauthor=A.+Cavalliauthor=A.+Ostermannauthor=A.+Heineauthor=G.+Klebe&title=Charges+Shift+Protonation%3A+Neutron+Diffraction+Reveals+That+Aniline+and+2-Aminopyridine+Become+Protonated+Upon+Binding+to+Trypsin&doi=10.1002%2Fanie.201701038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Charges shift protonation: Neutron diffraction reveals that aniline and 2-aminopyridine become protonated upon binding to trypsin</span></div><div class="casAuthors">Schiebel, Johannes; Gaspari, Roberto; Sandner, Anna; Ngo, Khang; Gerber, Hans-Dieter; Cavalli, Andrea; Ostermann, Andreas; Heine, Andreas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4887-4890</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">H atoms play a key role in protein-ligand recognition.  They det. the quality of established H-bonding networks and define the protonation of bound ligands.  Structural visualization of H atoms by x-ray crystallog. is rarely possible.  Here, we used neutron diffraction to det. the positions of the H atoms in the ligands, aniline and 2-aminopyridine, bound to the archetypical serine protease, trypsin.  The resulting structures showed the best resoln. so far achieved for proteins larger than 100 residues and allowed an accurate description of the protonation states and interactions with nearby water mols.  Despite its low pKa of 4.6 and a large distance of 3.6 Å to the charged Asp-189 residue at the bottom of the trypsin S1 pocket, the amino group of aniline became protonated, whereas in 2-aminopyridine, the pyridine N atom picked up the proton although its NH2 group was 1.6 Å closer to Asp-189.  Therefore, apart from charge-charge distances, tautomer stability was decisive for the resulting binding poses, an aspect that is pivotal for predicting correct binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg449p0i3o-7Vg90H21EOLACvtfcHk0lgcDGcE3juGgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvF2ltr8%253D&md5=a48df0f5959e03a0fdfc528de58bbb6f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201701038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201701038%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DSandner%26aufirst%3DA.%26aulast%3DNgo%26aufirst%3DK.%26aulast%3DGerber%26aufirst%3DH.%2BD.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DCharges%2520Shift%2520Protonation%253A%2520Neutron%2520Diffraction%2520Reveals%2520That%2520Aniline%2520and%25202-Aminopyridine%2520Become%2520Protonated%2520Upon%2520Binding%2520to%2520Trypsin%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D4887%26epage%3D4890%26doi%3D10.1002%2Fanie.201701038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span> <span> </span><span class="NLM_article-title">Enthalpy and Heat Capacity Changes for the Proton Dissociation of Various Buffer Components in 0.1 M Potassium Chloride</span>. <i>Proteins: Struct., Funct., Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19981101)33:2<159::AID-PROT2>3.0.CO;2-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=9779785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK1cXmsFOnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1998&pages=159-166&author=H.+Fukadaauthor=K.+Takahashi&title=Enthalpy+and+Heat+Capacity+Changes+for+the+Proton+Dissociation+of+Various+Buffer+Components+in+0.1+M+Potassium+Chloride&doi=10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Enthalpy and heat capacity changes for the proton dissociation of various buffer components in 0.1 M potassium chloride</span></div><div class="casAuthors">Fukada, Harumi; Takahashi, Katsutada</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-166</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Enthalpy and heat capacity changes for the deprotonation of 18 buffers were calorimetrically detd. in 0.1 M potassium chloride at temps. ranging from 5 to 45°.  The values of the dissocn. const. were also detd. by means of potentiometric titrn.  The enthalpy changes for the deprotonation of buffers, except for the phosphate and glycerol 2-phosphate buffers, were found to be characterized by a linear function of temp.  The enthalpy changes for the second dissocn. of phosphate and glycerol 2-phosphate where divalent anion is formed on dissocn. were fitted with the second order function of temp. rather than the first order.  Temp. dependence of buffer pH calcd. by using the enthalpy and heat capacity changes obtained was in good agreement with the temp. variation of the pH values actually measured in the temp. range between 0 and 50° for all the buffers studied.  On the basis of the results obtained, a numeric table showing the temp. dependence of pK values for the 18 buffers is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjY9pGcfJG_rVg90H21EOLACvtfcHk0lh6hT_BTjuZjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsFOnsr4%253D&md5=4ee32c30437c670b02595822767697c0</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819981101%252933%253A2%253C159%253A%253AAID-PROT2%253E3.0.CO%253B2-E%26sid%3Dliteratum%253Aachs%26aulast%3DFukada%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DK.%26atitle%3DEnthalpy%2520and%2520Heat%2520Capacity%2520Changes%2520for%2520the%2520Proton%2520Dissociation%2520of%2520Various%2520Buffer%2520Components%2520in%25200.1%2520M%2520Potassium%2520Chloride%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D1998%26volume%3D33%26spage%3D159%26epage%3D166%26doi%3D10.1002%2F%28SICI%291097-0134%2819981101%2933%3A2%3C159%3A%3AAID-PROT2%3E3.0.CO%3B2-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kishore, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennen, R. M.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic Quantities for the Ionization Reactions of Buffers</span>. <i>J. Phys. Chem. Ref. Data</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1063/1.1416902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1063%2F1.1416902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Oisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2002&pages=231-370&author=R.+N.+Goldbergauthor=N.+Kishoreauthor=R.+M.+Lennen&title=Thermodynamic+Quantities+for+the+Ionization+Reactions+of+Buffers&doi=10.1063%2F1.1416902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic quantities for the ionization reactions of buffers</span></div><div class="casAuthors">Goldberg, Robert N.; Kishore, Nand; Lennen, Rebecca M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical and Chemical Reference Data</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-370</span>CODEN:
                <span class="NLM_cas:coden">JPCRBU</span>;
        ISSN:<span class="NLM_cas:issn">0047-2689</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">This review contains selected values of thermodn. quantities for the aq. ionization reactions of 64 buffers, many of which are used in biol. research.  Since the aim is to be able to predict values of the ionization const. at temps. not too far from ambient, the thermodn. quantities which are tabulated are the pK, std. molar Gibbs energy ΔrG°, std. molar enthalpy ΔrH°, and std. molar heat capacity change ΔrCp° for each of the ionization reactions at the temp. T = 298.15 K and the pressure p = 0.1 MPa.  The std. state is the hypothetical ideal soln. of unit molality.  The chem. name(s) and CAS registry no., structure, empirical formula, and mol. wt. are given for each buffer considered herein.  The selection of the values of the thermodn. quantities for each buffer is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCt3UuqJMfxbVg90H21EOLACvtfcHk0lh6hT_BTjuZjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Oisrg%253D&md5=9c95e655c39245ac228ddbca075a1ca3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1063%2F1.1416902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1416902%26sid%3Dliteratum%253Aachs%26aulast%3DGoldberg%26aufirst%3DR.%2BN.%26aulast%3DKishore%26aufirst%3DN.%26aulast%3DLennen%26aufirst%3DR.%2BM.%26atitle%3DThermodynamic%2520Quantities%2520for%2520the%2520Ionization%2520Reactions%2520of%2520Buffers%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Ref.%2520Data%26date%3D2002%26volume%3D31%26spage%3D231%26epage%3D370%26doi%3D10.1063%2F1.1416902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dullweber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürzebecher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Factorising Ligand Affinity: A Combined Thermodynamic and Crystallographic Study of Trypsin and Thrombin Inhibition</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>313</i></span>,  <span class="NLM_fpage">593</span>– <span class="NLM_lpage">614</span>, <span class="refDoi"> DOI: 10.1006/jmbi.2001.5062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1006%2Fjmbi.2001.5062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=11676542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnslCrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=313&publication_year=2001&pages=593-614&author=F.+Dullweberauthor=M.+T.+Stubbsauthor=D.+Musilauthor=J.+St%C3%BCrzebecherauthor=G.+Klebe&title=Factorising+Ligand+Affinity%3A+A+Combined+Thermodynamic+and+Crystallographic+Study+of+Trypsin+and+Thrombin+Inhibition&doi=10.1006%2Fjmbi.2001.5062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Factorising Ligand Affinity: A Combined Thermodynamic and Crystallographic Study of Trypsin and Thrombin Inhibition</span></div><div class="casAuthors">Dullweber, Frank; Stubbs, Milton T.; Musil, Dorde; Stuerzebecher, Joerg; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">313</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">593-614</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">The binding of a series of low mol. wt. ligands towards trypsin and thrombin has been studied by isothermal titrn. calorimetry and protein crystallog.  In a series of congeneric ligands, surprising changes of protonation states occur and are overlaid on the binding process.  They result from induced pKa shifts depending on the local environment experienced by the ligand and protein functional groups in the complex (induced dielec. fit).  They involve addnl. heat effects that must be cor. before any conclusion on the binding enthalpy (ΔH) and entropy (ΔS) can be drawn.  After correction, trends in both contributions can be interpreted in structural terms with respect to the hydrogen bond inventory or residual ligand motions.  For all inhibitors studied, a strong neg. heat capacity change (ΔCp) is detected, thus binding becomes more exothermic and entropically less favorable with increasing temp.  Due to a mutual compensation, Gibbs free energy remains virtually unchanged.  The strong neg. ΔCp value cannot solely be explained by the removal of hydrophobic surface portions of the protein or ligand from water exposure.  Addnl. contributions must be considered, presumably arising from modulations of the local water structure, changes in vibrational modes or other ordering parameters.  For thrombin, smaller neg. ΔCp values are obsd. for ligand binding in the presence of sodium ions compared to the other alkali ions, probably due to stabilizing effects on the protein or changes in the bound water structure.  (c) 2001 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGortayuL9mRLbVg90H21EOLACvtfcHk0lh6hT_BTjuZjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnslCrsbk%253D&md5=59a21189828b9dcdcf90f8c8eb906621</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2001.5062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2001.5062%26sid%3Dliteratum%253Aachs%26aulast%3DDullweber%26aufirst%3DF.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26aulast%3DMusil%26aufirst%3DD.%26aulast%3DSt%25C3%25BCrzebecher%26aufirst%3DJ.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DFactorising%2520Ligand%2520Affinity%253A%2520A%2520Combined%2520Thermodynamic%2520and%2520Crystallographic%2520Study%2520of%2520Trypsin%2520and%2520Thrombin%2520Inhibition%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2001%26volume%3D313%26spage%3D593%26epage%3D614%26doi%3D10.1006%2Fjmbi.2001.5062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotriffer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Protonation Changes upon Ligand Binding to Trypsin and Thrombin: Structural Interpretation Based on p<i>K</i><sub>a</sub> Calculations and ITC Experiments</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>367</i></span>,  <span class="NLM_fpage">1347</span>– <span class="NLM_lpage">1356</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2007.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2007.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=17316681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=367&publication_year=2007&pages=1347-1356&author=P.+Czodrowskiauthor=C.+A.+Sotrifferauthor=G.+Klebe&title=Protonation+Changes+upon+Ligand+Binding+to+Trypsin+and+Thrombin%3A+Structural+Interpretation+Based+on+pKa+Calculations+and+ITC+Experiments&doi=10.1016%2Fj.jmb.2007.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protonation Changes upon Ligand Binding to Trypsin and Thrombin: Structural Interpretation Based on pKa Calculations and ITC Experiments</span></div><div class="casAuthors">Czodrowski, Paul; Sotriffer, Christoph A.; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">367</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1347-1356</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The protonation states of a protein and a ligand can be altered upon complex formation.  Such changes can be detected exptl. by isothermal titrn. calorimetry (ITC).  For a series of ligands binding to the serine proteases trypsin and thrombin, we previously performed an extensive ITC and crystallog. study and were able to identify protonation changes for four complexes.  However, since ITC measures only the overall proton exchange, it does not provide structural insights into the functional groups involved in the proton transfer.  Using Poisson-Boltzmann calcns. based on our recently developed PEOE_PB charges, we compute pKa values for all complexes of our former study in order to reveal the residues with altered protonation states.  The results indicate that His57, a member of the catalytic triad, is responsible for the most relevant pKa shifts leading to the exptl. detected protonation changes.  This finding is in contrast to our previous assumption that the obsd. protonation changes occur at the carboxylic group of the ligands.  The newly detected proton acceptor is used for a revised factorization of the ITC data, which is necessary whenever the protonation inventory changes upon complexation.  The pK a values of complexes showing no protonation change in the ITC expt. are reliably predicted in most cases, whereas predictions of strongly coupled systems remain problematic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzQ9tO1GIWD7Vg90H21EOLACvtfcHk0li54t8KCpYuew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKitLk%253D&md5=a451f9ac6fcd4a0efcdbc3616ec7849f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2007.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2007.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DSotriffer%26aufirst%3DC.%2BA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DProtonation%2520Changes%2520upon%2520Ligand%2520Binding%2520to%2520Trypsin%2520and%2520Thrombin%253A%2520Structural%2520Interpretation%2520Based%2520on%2520pKa%2520Calculations%2520and%2520ITC%2520Experiments%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D367%26spage%3D1347%26epage%3D1356%26doi%3D10.1016%2Fj.jmb.2007.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Think Twice: Understanding the High Potency of Bis(Phenyl)Methane Inhibitors of Thrombin</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2009.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2009.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19520086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlWrs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=2009&pages=552-564&author=B.+Baumauthor=L.+Muleyauthor=A.+Heineauthor=M.+Smolinskiauthor=D.+Hangauerauthor=G.+Klebe&title=Think+Twice%3A+Understanding+the+High+Potency+of+Bis%28Phenyl%29Methane+Inhibitors+of+Thrombin&doi=10.1016%2Fj.jmb.2009.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Think Twice: Understanding the High Potency of Bis(phenyl)methane Inhibitors of Thrombin</span></div><div class="casAuthors">Baum, Bernhard; Muley, Laveena; Heine, Andreas; Smolinski, Michael; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">552-564</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Successful design of potent and selective protein inhibitors, in terms of structure-based drug design, strongly relies on the correct understanding of the mol. features detg. the ligand binding to the target protein.  We present a case study of serine protease inhibitors with a bis(phenyl)methane moiety binding into the S3 pocket.  These inhibitors bind with remarkable potency to the active site of thrombin, the blood coagulation factor IIa.  A combination of x-ray crystallog. and isothermal titrn. calorimetry provides conclusive insights into the driving forces responsible for the surprisingly high potency of these inhibitors.  Anal. of six well-resolved crystal structures (resoln. 1.58-2.25 Å) along with the thermodn. data allows an explanation of the tight binding of the bis(phenyl)methane inhibitors.  Interestingly, the two Ph rings contribute to binding affinity for very different reasons - a fact that can only be elucidated by a structure-based approach.  The first Ph moiety occupies the hydrophobic S3 pocket, resulting in a mainly entropic advantage of binding.  This observation is based on the displacement of structural water mols. from the S3 pocket that are obsd. in complexes with inhibitors that do not bind in the S3 pocket.  The same classic hydrophobic effect cannot explain the enhanced binding affinity resulting from the attachment of the second, more solvent-exposed Ph ring.  For the bis(phenyl)methane inhibitors, an obsd. adaptive rotation of a glutamate residue adjacent to the S3 binding pocket attracted our attention.  The rotation of this glutamate into salt-bridging distance with a lysine moiety correlates with an enhanced enthalpic contribution to binding for these highly potent thrombin binders.  This explanation for the magnitude of the attractive force is confirmed by data retrieved by a Relibase search of several thrombin-inhibitor complexes deposited in the Protein Data Bank exhibiting similar mol. features.  Special attention was attributed to putative changes in the protonation states of the interaction partners.  For this purpose, two analogous inhibitors differing mainly in their potential to change the protonation state of a hydrogen-bond donor functionality were compared.  Buffer dependencies of the binding enthalpy assocd. with complex formation could be traced by isothermal titrn. calorimetry, which revealed, along with anal. of the crystal structures (resoln. 1.60 and 1.75 Å), that a virtually compensating proton interchange between enzyme, inhibitor and buffer is responsible for the obsd. buffer-independent thermodn. signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrayaqPbLFKe7Vg90H21EOLACvtfcHk0li54t8KCpYuew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlWrs7Y%253D&md5=2b6a2483a0a54b2dc30c622cf603cc55</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DThink%2520Twice%253A%2520Understanding%2520the%2520High%2520Potency%2520of%2520Bis%2528Phenyl%2529Methane%2520Inhibitors%2520of%2520Thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D391%26spage%3D552%26epage%3D564%26doi%3D10.1016%2Fj.jmb.2009.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baum, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hangauer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Non-Additivity of Functional Group Contributions in Protein-Ligand Binding: A Comprehensive Study by Crystallography and Isothermal Titration Calorimetry</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>397</i></span>,  <span class="NLM_fpage">1042</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.jmb.2010.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20156458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=397&publication_year=2010&pages=1042-1054&author=B.+Baumauthor=L.+Muleyauthor=M.+Smolinskiauthor=A.+Heineauthor=D.+Hangauerauthor=G.+Klebe&title=Non-Additivity+of+Functional+Group+Contributions+in+Protein-Ligand+Binding%3A+A+Comprehensive+Study+by+Crystallography+and+Isothermal+Titration+Calorimetry&doi=10.1016%2Fj.jmb.2010.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Non-additivity of Functional Group Contributions in Protein-Ligand Binding: A Comprehensive Study by Crystallography and Isothermal Titration Calorimetry</span></div><div class="casAuthors">Baum, Bernhard; Muley, Laveena; Smolinski, Michael; Heine, Andreas; Hangauer, David; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">397</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1042-1054</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Additivity of functional group contributions to protein-ligand binding is a very popular concept in medicinal chem. as the basis of rational design and optimized lead structures.  Most of the currently applied scoring functions for docking build on such additivity models.  Even though the limitation of this concept is well known, case studies examg. in detail why additivity fails at the mol. level are still very scarce.  The present study shows, by use of crystal structure anal. and isothermal titrn. calorimetry for a congeneric series of thrombin inhibitors, that extensive cooperative effects between hydrophobic contacts and hydrogen bond formation are intimately coupled via dynamic properties of the formed complexes.  The formation of optimal lipophilic contacts with the surface of the thrombin S3 pocket and the full desolvation of this pocket can conflict with the formation of an optimal hydrogen bond between ligand and protein.  The mutual contributions of the competing interactions depend on the size of the ligand hydrophobic substituent and influence the residual mobility of ligand portions at the binding site.  Anal. of the individual crystal structures and factorizing the free energy into enthalpy and entropy demonstrates that binding affinity of the ligands results from a mixt. of enthalpic contributions from hydrogen bonding and hydrophobic contacts, and entropic considerations involving an increasing loss of residual mobility of the bound ligands.  This complex picture of mutually competing and partially compensating enthalpic and entropic effects dets. the non-additivity of free energy contributions to ligand binding at the mol. level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGousWs7tLpU3LVg90H21EOLACvtfcHk0li54t8KCpYuew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKmurg%253D&md5=2f61dde175a6ee27551179644eee3f00</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DBaum%26aufirst%3DB.%26aulast%3DMuley%26aufirst%3DL.%26aulast%3DSmolinski%26aufirst%3DM.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DHangauer%26aufirst%3DD.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DNon-Additivity%2520of%2520Functional%2520Group%2520Contributions%2520in%2520Protein-Ligand%2520Binding%253A%2520A%2520Comprehensive%2520Study%2520by%2520Crystallography%2520and%2520Isothermal%2520Titration%2520Calorimetry%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D397%26spage%3D1042%26epage%3D1054%26doi%3D10.1016%2Fj.jmb.2010.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sielaff, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwesten, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Ligand Binding Stepwise Disrupts Water Network in Thrombin: Enthalpic and Entropic Changes Reveal Classical Hydrophobic Effect</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6094</span>– <span class="NLM_lpage">6110</span>, <span class="refDoi"> DOI: 10.1021/jm300337q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300337q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6094-6110&author=A.+Bielaauthor=F.+Sielaffauthor=F.+Terwestenauthor=A.+Heineauthor=T.+Steinmetzerauthor=G.+Klebe&title=Ligand+Binding+Stepwise+Disrupts+Water+Network+in+Thrombin%3A+Enthalpic+and+Entropic+Changes+Reveal+Classical+Hydrophobic+Effect&doi=10.1021%2Fjm300337q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Binding Stepwise Disrupts Water Network in Thrombin: Enthalpic and Entropic Changes Reveal Classical Hydrophobic Effect</span></div><div class="casAuthors">Biela, Adam; Sielaff, Frank; Terwesten, Felix; Heine, Andreas; Steinmetzer, Torsten; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6094-6110</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Well-ordered water mols. are displaced from thrombin's hydrophobic S3/4-pocket by P3-varied ligands (Gly, D-Ala, D-Val, D-Leu to D-Cha with increased hydrophobicity and steric requirement).  Two series with 2-(aminomethyl)-5-chlorobenzylamide and 4-amidinobenzylamide at P1 were examd. by ITC and crystallog.  Although experiencing different interactions in S1, they display almost equal potency.  For both scaffolds the terminal benzylsulfonyl substituent differs in binding, whereas the increasingly bulky P3-groups address S3/4 pocket similarly.  Small substituents leave the solvation pattern unperturbed as found in the uncomplexed enzyme while increasingly larger ones stepwise displace the waters.  Medium-sized groups show patterns with partially occupied waters.  The overall 40-fold affinity enhancement correlates with water displacement and growing no. of van der Waals contacts and is mainly attributed to favorable entropy.  Both Gly derivs. deviate from the series and adopt different binding modes.  Nonetheless, their thermodn. signatures are virtually identical with the homologous D-Ala derivs.  Accordingly, unchanged thermodn. profiles are no reliable indicator for conserved binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjEkdauKdcCrVg90H21EOLACvtfcHk0li54t8KCpYuew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt7g%253D&md5=7488723d7299d9b5233993436679ddac</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm300337q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300337q%26sid%3Dliteratum%253Aachs%26aulast%3DBiela%26aufirst%3DA.%26aulast%3DSielaff%26aufirst%3DF.%26aulast%3DTerwesten%26aufirst%3DF.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DLigand%2520Binding%2520Stepwise%2520Disrupts%2520Water%2520Network%2520in%2520Thrombin%253A%2520Enthalpic%2520and%2520Entropic%2520Changes%2520Reveal%2520Classical%2520Hydrophobic%2520Effect%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6094%26epage%3D6110%26doi%3D10.1021%2Fjm300337q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sandner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüfner-Wulsdorf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetzer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Strategies for Late-Stage Optimization: Profiling Thermodynamics by Preorganization and Salt Bridge Shielding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9753</span>– <span class="NLM_lpage">9771</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01196</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01196" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvF2hurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9753-9771&author=A.+Sandnerauthor=T.+H%C3%BCfner-Wulsdorfauthor=A.+Heineauthor=T.+Steinmetzerauthor=G.+Klebe&title=Strategies+for+Late-Stage+Optimization%3A+Profiling+Thermodynamics+by+Preorganization+and+Salt+Bridge+Shielding&doi=10.1021%2Facs.jmedchem.9b01196"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies for Late-Stage Optimization: Profiling Thermodynamics by Preorganization and Salt Bridge Shielding</span></div><div class="casAuthors">Sandner, Anna; Huefner-Wulsdorf, Tobias; Heine, Andreas; Steinmetzer, Torsten; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9753-9771</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural fixation of a ligand in its bioactive conformation may, due to entropic reasons, improve affinity.  The authors present a congeneric series of thrombin ligands with a variety of functional groups triggering preorganization prior to binding.  Fixation in soln. and complex formation have been characterized by crystallog., ITC and MD simulations.  First, the authors show why these preorganizing modifications do not affect the overall binding mode and how key interactions are preserved.  Next, the authors demonstrate how preorganization thermodn. can be largely dominated by enthalpy, rather than entropy due to the significant population of low-energy conformations.  Furthermore, a salt bridge is shielded by actively reducing its surface exposure and thus, leading to an enhanced enthalpic binding profile.  The results suggest that the consideration of the ligand soln. ensemble by mol. dynamics simulation is necessary to predict preorganizing modifications that enhance the binding behavior of already promising binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQZ8FrLYcmBLVg90H21EOLACvtfcHk0ljQE2gmjt09uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvF2hurrL&md5=211810b5c43718767916e3b1fcec6401</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01196%26sid%3Dliteratum%253Aachs%26aulast%3DSandner%26aufirst%3DA.%26aulast%3DH%25C3%25BCfner-Wulsdorf%26aufirst%3DT.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DSteinmetzer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DStrategies%2520for%2520Late-Stage%2520Optimization%253A%2520Profiling%2520Thermodynamics%2520by%2520Preorganization%2520and%2520Salt%2520Bridge%2520Shielding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9753%26epage%3D9771%26doi%3D10.1021%2Facs.jmedchem.9b01196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ladbury, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freire, E.</span></span> <span> </span><span class="NLM_article-title">Adding Calorimetric Data to Decision Making in Drug Development: A Hot Tip!</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1038/nrd3054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1038%2Fnrd3054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19960014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=23-27&author=J.+E.+Ladburyauthor=G.+Klebeauthor=E.+Freire&title=Adding+Calorimetric+Data+to+Decision+Making+in+Drug+Development%3A+A+Hot+Tip%21&doi=10.1038%2Fnrd3054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Adding calorimetric data to decision making in lead discovery: A hot tip</span></div><div class="casAuthors">Ladbury, John E.; Klebe, Gerhard; Freire, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-27</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recognition of the limitations of high-throughput screening approaches in the discovery of candidate drugs has reawakened interest in structure-based and other rational design methods.  Here, the authors describe how isothermal titrn. calorimetry can be used to obtain thermodn. data on the binding of compds. to protein targets.  The authors propose that these data - particularly the change in enthalpy - could provide a valuable, complementary addn. to established tools for selecting compds. in lead discovery and for aiding lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph3qvp2E8fbLVg90H21EOLACvtfcHk0ljQE2gmjt09uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWjt7bM&md5=e8446b6db983c35580e9010899eb40eb</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fnrd3054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3054%26sid%3Dliteratum%253Aachs%26aulast%3DLadbury%26aufirst%3DJ.%2BE.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DFreire%26aufirst%3DE.%26atitle%3DAdding%2520Calorimetric%2520Data%2520to%2520Decision%2520Making%2520in%2520Drug%2520Development%253A%2520A%2520Hot%2520Tip%2521%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2010%26volume%3D9%26spage%3D23%26epage%3D27%26doi%3D10.1038%2Fnrd3054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sigurskjold, B. W.</span></span> <span> </span><span class="NLM_article-title">Exact Analysis of Competition Ligand Binding by Displacement Isothermal Titration Calorimetry</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1006/abio.1999.4402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1006%2Fabio.1999.4402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10625516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhs1WmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2000&pages=260-266&author=B.+W.+Sigurskjold&title=Exact+Analysis+of+Competition+Ligand+Binding+by+Displacement+Isothermal+Titration+Calorimetry&doi=10.1006%2Fabio.1999.4402"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Exact Analysis of Competition Ligand Binding by Displacement Isothermal Titration Calorimetry</span></div><div class="casAuthors">Sigurskjold, Bent W.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">260-266</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A rigorous method for the least-squares nonlinear regression anal. of displacement isothermal titrn. calorimetric data is presented.  The method can fit the binding isotherm of a ligand which is competitively inhibited in its binding by another bound ligand to a mol. with n identical and independent binding sites.  There are no other assumptions for the method and no approxns.  Anal. of previously published data of the strong binding of acarbose to glucoamylase is presented as an example.  The regression equations have been programmed for the Origin software supplied with the widely used titrn. calorimeters from Microcal, Inc., and an Origin Function Definition File with instructions is freely available from the author upon e-mail request.  (c) 2000 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsNy3023MELVg90H21EOLACvtfcHk0ljPnL3o3mw9CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhs1WmsA%253D%253D&md5=3cce6711ca574bf075a56c297d996282</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fabio.1999.4402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1999.4402%26sid%3Dliteratum%253Aachs%26aulast%3DSigurskjold%26aufirst%3DB.%2BW.%26atitle%3DExact%2520Analysis%2520of%2520Competition%2520Ligand%2520Binding%2520by%2520Displacement%2520Isothermal%2520Titration%2520Calorimetry%26jtitle%3DAnal.%2520Biochem.%26date%3D2000%26volume%3D277%26spage%3D260%26epage%3D266%26doi%3D10.1006%2Fabio.1999.4402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turnbull, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daranas, A. H.</span></span> <span> </span><span class="NLM_article-title">On the Value of c: Can Low Affinity Systems Be Studied by Isothermal Titration Calorimetry?</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">14859</span>– <span class="NLM_lpage">14866</span>, <span class="refDoi"> DOI: 10.1021/ja036166s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja036166s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVSiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=14859-14866&author=W.+B.+Turnbullauthor=A.+H.+Daranas&title=On+the+Value+of+c%3A+Can+Low+Affinity+Systems+Be+Studied+by+Isothermal+Titration+Calorimetry%3F&doi=10.1021%2Fja036166s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">On the Value of c: Can Low Affinity Systems Be Studied by Isothermal Titration Calorimetry?</span></div><div class="casAuthors">Turnbull, W. Bruce; Daranas, Antonio H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">14859-14866</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) allows the detn. of ΔG°, ΔH°, and ΔS° from a single expt. and is thus widely used for studying binding thermodn. in both biol. and synthetic supramol. systems.  However, it is widely believed that it is not possible to derive accurate thermodn. information from ITC expts. in which the Wiseman "c" parameter (which is the product of the receptor concn. and the binding const., Ka) is less than ca. 10, constraining its use to high affinity systems.  Herein, exptl. titrns. and simulated data are used to demonstrate that this dogma is false, esp. for low affinity systems, assuming that (1) a sufficient portion of the binding isotherm is used for anal., (2) the binding stoichiometry is known, (3) the concns. of both ligand and receptor are known with accuracy, and (4) there is an adequate level of signal-to-noise in the data.  This study supports the validity of ITC for detg. the value of Ka and, hence, ΔG° from expts. conducted under low c conditions but advocates greater caution in the interpretation of values for ΔH°.  Therefore, isothermal titrn. calorimetry is a valid and useful technique for studying biol. and synthetically important low affinity systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6KxnjW2sZN7Vg90H21EOLACvtfcHk0ljPnL3o3mw9CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVSiu7w%253D&md5=088d351b0836433e55b46e16189271e7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja036166s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja036166s%26sid%3Dliteratum%253Aachs%26aulast%3DTurnbull%26aufirst%3DW.%2BB.%26aulast%3DDaranas%26aufirst%3DA.%2BH.%26atitle%3DOn%2520the%2520Value%2520of%2520c%253A%2520Can%2520Low%2520Affinity%2520Systems%2520Be%2520Studied%2520by%2520Isothermal%2520Titration%2520Calorimetry%253F%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D14859%26epage%3D14866%26doi%3D10.1021%2Fja036166s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macgillivray, R. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Cera, E.</span></span> <span> </span><span class="NLM_article-title">Determinants of Specificity in Coagulation Proteases</span>. <i>J. Thromb. Haemostasis</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2401</span>– <span class="NLM_lpage">2408</span>, <span class="refDoi"> DOI: 10.1111/j.1538-7836.2005.01456.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1111%2Fj.1538-7836.2005.01456.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=16241939" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1KktLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2005&pages=2401-2408&author=M.+J.+Pageauthor=R.+T.+A.+Macgillivrayauthor=E.+Di+Cera&title=Determinants+of+Specificity+in+Coagulation+Proteases&doi=10.1111%2Fj.1538-7836.2005.01456.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Determinants of specificity in coagulation proteases</span></div><div class="casAuthors">Page, M. J.; MacGillivray, R. T. A.; Di Cera, E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Thrombosis and Haemostasis</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2401-2408</span>CODEN:
                <span class="NLM_cas:coden">JTHOA5</span>;
        ISSN:<span class="NLM_cas:issn">1538-7933</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing, Inc.</span>)
        </div><div class="casAbstract">A review.  Proteases play diverse roles in a variety of essential biol. processes, both as nonspecific catalysts of protein degrdn. and as highly specific agents that control physiol. events.  Here, the authors review the mechanisms of substrate specificity employed by serine proteases and focus their discussion on blood-coagulation proteases.  The authors dissect the interplay between active site and exosite specificity and how substrate recognition is regulated allosterically by Na+ binding.  The authors also draw attention to a functional polarity that exists in the serine protease fold, which sheds light on the structural linkages between the active site and exosites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokX5M6DgS4WLVg90H21EOLACvtfcHk0ljPnL3o3mw9CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1KktLzN&md5=a1a05ad79e24ceebdafdebf2a0638215</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1111%2Fj.1538-7836.2005.01456.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1538-7836.2005.01456.x%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BJ.%26aulast%3DMacgillivray%26aufirst%3DR.%2BT.%2BA.%26aulast%3DDi%2BCera%26aufirst%3DE.%26atitle%3DDeterminants%2520of%2520Specificity%2520in%2520Coagulation%2520Proteases%26jtitle%3DJ.%2520Thromb.%2520Haemostasis%26date%3D2005%26volume%3D3%26spage%3D2401%26epage%3D2408%26doi%3D10.1111%2Fj.1538-7836.2005.01456.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Cera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinto, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vindigni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayala, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulinsky, A.</span></span> <span> </span><span class="NLM_article-title">The Na<sup>+</sup> Binding Site of Thrombin</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>270</i></span>,  <span class="NLM_fpage">22089</span>– <span class="NLM_lpage">22092</span>, <span class="refDoi"> DOI: 10.1074/jbc.270.38.22089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1074%2Fjbc.270.38.22089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=7673182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2MXot1Giu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=270&publication_year=1995&pages=22089-22092&author=E.+Di%0ACeraauthor=E.+R.+Guintoauthor=A.+Vindigniauthor=Q.+D.+Dangauthor=Y.+M.+Ayalaauthor=M.+Wuyiauthor=A.+Tulinsky&title=The+Na%2B+Binding+Site+of+Thrombin&doi=10.1074%2Fjbc.270.38.22089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The Na+ binding site of thrombin</span></div><div class="casAuthors">Di Cera, Enrico; Guinto, Enriqueta R.; Vindigni, Alessandro; Dang, Quoc D.; Ayala, Youhna M.; Wuyi, Meng; Tulinsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">270</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">22089-92</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Thrombin is an allosteric serine protease existing in two forms, slow and fast, targeted toward anticoagulant and procoagulant activities.  The slow → fast transition is induced by Na+ binding to a site contained within a cylindrical cavity formed by three antiparallel β-strands of the B-chain (Met180-Tyr184a, Lys224-Tyr228, and Val213-Gly219) diagonally crossed by the Glu188-Glu192 strand.  The site is shaped further by the loop connecting the last two β-strands and is located more than 15 Å away from the catalytic triad.  The cavity traverses through thrombin from the active site to the opposite surface and contains Asp189 of the primary specificity site near its midpoint.  The bound Na+ is coordinated octahedrally by the carbonyl oxygen atoms of Tyr184a, Arg221a, and Lys224, and by three highly conserved water mols. in the D-Phe-Pro-Arg chloromethylketone thrombin.  The sequence in the Na+ binding loop is highly conserved in thrombin from 11 different species and is homologous to that found in other serine proteases involved in blood coagulation.  Mutation of two Asp residues flanking Arg221a (D221A/D222K) almost abolishes the allosteric properties of thrombin and shows that the Na+ binding loop is also involved in direct recognition of protein C and antithrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqffZELRpwnk7Vg90H21EOLACvtfcHk0ljPnL3o3mw9CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXot1Giu7c%253D&md5=eeb9c69f2551480f4508a2feca1b0249</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.270.38.22089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.270.38.22089%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BCera%26aufirst%3DE.%26aulast%3DGuinto%26aufirst%3DE.%2BR.%26aulast%3DVindigni%26aufirst%3DA.%26aulast%3DDang%26aufirst%3DQ.%2BD.%26aulast%3DAyala%26aufirst%3DY.%2BM.%26aulast%3DWuyi%26aufirst%3DM.%26aulast%3DTulinsky%26aufirst%3DA.%26atitle%3DThe%2520Na%252B%2520Binding%2520Site%2520of%2520Thrombin%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1995%26volume%3D270%26spage%3D22089%26epage%3D22092%26doi%3D10.1074%2Fjbc.270.38.22089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tulinsky, A.</span></span> <span> </span><span class="NLM_article-title">The Molecular Environment of the Na<sup>+</sup> Binding Site of Thrombin</span>. <i>Biophys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1016/S0301-4622(96)02227-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2FS0301-4622%2896%2902227-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=9108691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXhvFWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1997&pages=185-200&author=E.+Zhangauthor=A.+Tulinsky&title=The+Molecular+Environment+of+the+Na%2B+Binding+Site+of+Thrombin&doi=10.1016%2FS0301-4622%2896%2902227-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular environment of the Na+ binding site of thrombin</span></div><div class="casAuthors">Zhang, Erli; Tulinsky, A.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">185-200</span>CODEN:
                <span class="NLM_cas:coden">BICIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0301-4622</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">When Na+ binds to thrombin, a conformational change is induced that renders the enzyme kinetically faster and more specific in the activation of fibrinogen.  Two Na+ binding sites have here been identified crystallog. by exchanging Na+ with Rb+.  One is intermol., found on the surface between two symmetry-related thrombin mols.  Since it is not present in thrombin crystal structures having different crystal systems, the other Na+ site is the functionally relevant one.  The second site has octahedral coordination with the carbonyl oxygen atoms of Arg221A and Lys224 and four conserved water mols.  It is located near Asp189 of the S1 specificity site in an elongated solvent channel (8×18 Å) formed by four antiparallel β-strands between Cys182-Cys191 and Val213-Tyr228.  This channel, extending from the active site to the opposite surface of the enzyme, was first noted in the hirudin-thrombin structure and contains about 20 conserved water mols. linked together by a hydrogen bonding network that connects to the main chain of thrombin.  Although the antiparallel β-strand interactions of the functional Na+ binding site are the same in prethrombin2, the loops between the strands are very different, so that Asp189 and Arg221A are not positioned properly for either substrate or Na+ binding in prethrombin2.  A water mol. with octahedral coordination has also been identified in factor Xa at the topol. equiv. Na+ site position of thrombin.  Since activated protein C shows enhanced activity with monovalent cation binding, the same position is probably utilized by Na+.  Since thrombin crystals could not be grown in the absence of Na+, the cation was leached from Na+-bound thrombin crystals by diffusion/exchange.  Although both Na+ and their coordinating water mols. were removed from the Na+ binding sites, the remainder of the thrombin structure was, unexpectedly, the same.  The lack of an allosteric change is most likely attributable to crystal packing effects.  Thus, the structure of the slow form remains to be established crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXh6_qcjrtd7Vg90H21EOLACvtfcHk0lhICavzdy0NqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhvFWhsL8%253D&md5=9abb5ab1519bef7f16541c89138cc91a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0301-4622%2896%2902227-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0301-4622%252896%252902227-2%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DE.%26aulast%3DTulinsky%26aufirst%3DA.%26atitle%3DThe%2520Molecular%2520Environment%2520of%2520the%2520Na%252B%2520Binding%2520Site%2520of%2520Thrombin%26jtitle%3DBiophys.%2520Chem.%26date%3D1997%26volume%3D63%26spage%3D185%26epage%3D200%26doi%3D10.1016%2FS0301-4622%2896%2902227-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huntington, J. A.</span></span> <span> </span><span class="NLM_article-title">How Na<sup>+</sup> Activates Thrombin - A Review of the Functional and Structural Data</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">1025</span>– <span class="NLM_lpage">1035</span>, <span class="refDoi"> DOI: 10.1515/BC.2008.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1515%2FBC.2008.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=18979627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD1cXps1Ohur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2008&pages=1025-1035&author=J.+A.+Huntington&title=How+Na%2B+Activates+Thrombin+-+A+Review+of+the+Functional+and+Structural+Data&doi=10.1515%2FBC.2008.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How Na+ activates thrombin - a review of the functional and structural data</span></div><div class="casAuthors">Huntington, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1025-1035</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Thrombin is the ultimate coagulation factor; it is the final protease generated in the blood coagulation cascade and is the effector of clot formation.  Regulation of thrombin activity is thus of great relevance to detg. the correct hemostatic balance, with dysregulation leading to bleeding or thrombosis.  One of the most enigmatic and controversial regulators of thrombin activity is the monovalent cation, Na+.  When bound to Na+, thrombin adopts a 'fast' conformation which cleaves all procoagulant substrates more rapidly, and when free of Na+, thrombin reverts to a 'slow' state which preferentially activates the protein C anticoagulant pathway.  Thus, Na+-binding allosterically modulates the activity of thrombin and helps det. the hemostatic balance.  Over the last 30 yr, there has been much research investigating the structural basis of thrombin allostery.  Biochem. and mutagenesis studies have established which regions and residues are involved in the slow → fast conformational change, and recently several crystal structures of the putative slow form have been solved.  Here, the biochem. and crystallog. data are reviewed to see if investigators are any closer to understanding the conformational basis of the Na+ activation of thrombin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoItIkRWHB6WrVg90H21EOLACvtfcHk0lhICavzdy0NqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXps1Ohur0%253D&md5=c830271129b66beb2bd232c19c5472f1</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1515%2FBC.2008.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2008.113%26sid%3Dliteratum%253Aachs%26aulast%3DHuntington%26aufirst%3DJ.%2BA.%26atitle%3DHow%2520Na%252B%2520Activates%2520Thrombin%2520-%2520A%2520Review%2520of%2520the%2520Functional%2520and%2520Structural%2520Data%26jtitle%3DBiol.%2520Chem.%26date%3D2008%26volume%3D389%26spage%3D1025%26epage%3D1035%26doi%3D10.1515%2FBC.2008.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiebel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaspari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulsdorf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrader, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klebe, G.</span></span> <span> </span><span class="NLM_article-title">Intriguing Role of Water in Protein-Ligand Binding Studied by Neutron Crystallography on Trypsin Complexes</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">3559</span> <span class="refDoi"> DOI: 10.1038/s41467-018-05769-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1038%2Fs41467-018-05769-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=30177695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A280%3ADC%252BB3c3ltVOhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&author=J.+Schiebelauthor=R.+Gaspariauthor=T.+Wulsdorfauthor=K.+Ngoauthor=C.+Sohnauthor=T.+E.+Schraderauthor=A.+Cavalliauthor=A.+Ostermannauthor=A.+Heineauthor=G.+Klebe&title=Intriguing+Role+of+Water+in+Protein-Ligand+Binding+Studied+by+Neutron+Crystallography+on+Trypsin+Complexes&doi=10.1038%2Fs41467-018-05769-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Intriguing role of water in protein-ligand binding studied by neutron crystallography on trypsin complexes</span></div><div class="casAuthors">Schiebel Johannes; Wulsdorf Tobias; Ngo Khang; Sohn Christian; Heine Andreas; Klebe Gerhard; Schiebel Johannes; Gaspari Roberto; Cavalli Andrea; Schrader Tobias E; Ostermann Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3559</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hydrogen bonds are key interactions determining protein-ligand binding affinity and therefore fundamental to any biological process.  Unfortunately, explicit structural information about hydrogen positions and thus H-bonds in protein-ligand complexes is extremely rare and similarly the important role of water during binding remains poorly understood.  Here, we report on neutron structures of trypsin determined at very high resolutions ≤1.5 ÅA in uncomplexed and inhibited state complemented by X-ray and thermodynamic data and computer simulations.  Our structures show the precise geometry of H-bonds between protein and the inhibitors N-amidinopiperidine and benzamidine along with the dynamics of the residual solvation pattern.  Prior to binding, the ligand-free binding pocket is occupied by water molecules characterized by a paucity of H-bonds and high mobility resulting in an imperfect hydration of the critical residue Asp189.  This phenomenon likely constitutes a key factor fueling ligand binding via water displacement and helps improving our current view on water influencing protein-ligand recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT2ZT4TkrW9o0h5eUSa2oe_fW6udTcc2ebHplmErAIvnrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3ltVOhtw%253D%253D&md5=56cfa8b99517f0da0967e3c9fbc09ac9</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05769-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05769-2%26sid%3Dliteratum%253Aachs%26aulast%3DSchiebel%26aufirst%3DJ.%26aulast%3DGaspari%26aufirst%3DR.%26aulast%3DWulsdorf%26aufirst%3DT.%26aulast%3DNgo%26aufirst%3DK.%26aulast%3DSohn%26aufirst%3DC.%26aulast%3DSchrader%26aufirst%3DT.%2BE.%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DOstermann%26aufirst%3DA.%26aulast%3DHeine%26aufirst%3DA.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DIntriguing%2520Role%2520of%2520Water%2520in%2520Protein-Ligand%2520Binding%2520Studied%2520by%2520Neutron%2520Crystallography%2520on%2520Trypsin%2520Complexes%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26doi%3D10.1038%2Fs41467-018-05769-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stürzebecher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stürzebecher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieweg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauptmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markwardt, F.</span></span> <span> </span><span class="NLM_article-title">Synthetic Inhibitors of Bovine Factor Xa and Thrombin Comparison of Their Anticoagulant Efficiency</span>. <i>Thromb. Res.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1016/0049-3848(89)90232-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2F0049-3848%2889%2990232-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=2749616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaL1MXkslSgs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1989&pages=245-252&author=J.+St%C3%BCrzebecherauthor=U.+St%C3%BCrzebecherauthor=H.+Viewegauthor=G.+Wagnerauthor=J.+Hauptmannauthor=F.+Markwardt&title=Synthetic+Inhibitors+of+Bovine+Factor+Xa+and+Thrombin+Comparison+of+Their+Anticoagulant+Efficiency&doi=10.1016%2F0049-3848%2889%2990232-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic inhibitors of bovine factor Xa and thrombin: Comparison of their anticoagulant efficiency</span></div><div class="casAuthors">Stuerzebecher, J.; Stuerzebecher, Uta; Vieweg, H.; Wagner, G.; Hauptmann, J.; Markwardt, F.</div><div class="citationInfo"><span class="NLM_cas:title">Thrombosis Research</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-52</span>CODEN:
                <span class="NLM_cas:coden">THBRAA</span>;
        ISSN:<span class="NLM_cas:issn">0049-3848</span>.
    </div><div class="casAbstract">New derivs. of benzamidine (Nα-arylsulfonylaminoacetylated derivs. of 3-amidinophenylalanine) were potent inhibitors of bovine factor Xa (FXa).  The Me and Et esters of Nα-Tos-Gly- and Nα-βNas-Gly-substituted 3-amidinophenylalanine, inhibited F Xa more effectively (Ki values near 0.5 μmol/L) than thrombin (Ki about 50 μmol/L).  Reinvestigation of inhibition of F Xa by bis-benzamidines showed that compds. contg. a cycloheptanone linking bridge are tight-binding inhibitors of F Xa (Ki in the 10 nmol/L range).  The use of these inhibitors enabled us to clarify whether inhibition of F Xa or inhibition of thrombin is more efficient in anticoagulation.  The thrombin inhibitors and not the potent F Xa inhibitors proved to be particularly effective in anticoagulation in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGxZUXZRCYbbVg90H21EOLACvtfcHk0liDOlpJUQyS2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkslSgs7Y%253D&md5=8cc9a02fb634fa0c9c441e63312cf2cb</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2F0049-3848%2889%2990232-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0049-3848%252889%252990232-6%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25BCrzebecher%26aufirst%3DJ.%26aulast%3DSt%25C3%25BCrzebecher%26aufirst%3DU.%26aulast%3DVieweg%26aufirst%3DH.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DHauptmann%26aufirst%3DJ.%26aulast%3DMarkwardt%26aufirst%3DF.%26atitle%3DSynthetic%2520Inhibitors%2520of%2520Bovine%2520Factor%2520Xa%2520and%2520Thrombin%2520Comparison%2520of%2520Their%2520Anticoagulant%2520Efficiency%26jtitle%3DThromb.%2520Res.%26date%3D1989%26volume%3D54%26spage%3D245%26epage%3D252%26doi%3D10.1016%2F0049-3848%2889%2990232-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotlyar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nudelman, A.</span></span> <span> </span><span class="NLM_article-title">NMR Chemical Shifts of Common Laboratory Solvents as Trace Impurities</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7512</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jo971176v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo971176v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=1997&pages=7512-7515&author=H.+E.+Gottliebauthor=V.+Kotlyarauthor=A.+Nudelman&title=NMR+Chemical+Shifts+of+Common+Laboratory+Solvents+as+Trace+Impurities&doi=10.1021%2Fjo971176v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">NMR chemical shifts of common laboratory solvents as trace impurities</span></div><div class="casAuthors">Gottlieb, Hugo E.; Kotlyar, Vadim; Nudelman, Abraham</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">7512-7515</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 1H (300 MHz) and 13C (75 MHz) NMR chem. shifts for >30 common impurities, such as lab. solvents, stabilizers, greases and water, are reported in 7 deuterated NMR solvents.  The temp. dependence of the chem. shift of the proton residual peak (HDO) in D2O is established for the entire liq. range of this sample.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocnoOemocS9LVg90H21EOLACvtfcHk0liDOlpJUQyS2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVClsbo%253D&md5=c297adda99f8c10e9f9b0ab9fe5f55be</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjo971176v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo971176v%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DH.%2BE.%26aulast%3DKotlyar%26aufirst%3DV.%26aulast%3DNudelman%26aufirst%3DA.%26atitle%3DNMR%2520Chemical%2520Shifts%2520of%2520Common%2520Laboratory%2520Solvents%2520as%2520Trace%2520Impurities%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1997%26volume%3D62%26spage%3D7512%26epage%3D7515%26doi%3D10.1021%2Fjo971176v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wollatz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haznar-Garbacz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weitschies, W.</span></span> <span> </span><span class="NLM_article-title">Characterisation of Selected Active Agents Regarding p<i>K</i><sub>a</sub> Values, Solubility Concentrations and pH Profiles by SiriusT3</span>. <i>Eur. J. Pharm. Biopharm.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1016/j.ejpb.2015.02.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.ejpb.2015.02.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=25758123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A280%3ADC%252BC2MnivVKlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2015&pages=155-170&author=D.+Sch%C3%B6nherrauthor=U.+Wollatzauthor=D.+Haznar-Garbaczauthor=U.+Hankeauthor=K.+J.+Boxauthor=R.+Taylorauthor=R.+Ruizauthor=S.+Beatoauthor=D.+Beckerauthor=W.+Weitschies&title=Characterisation+of+Selected+Active+Agents+Regarding+pKa+Values%2C+Solubility+Concentrations+and+pH+Profiles+by+SiriusT3&doi=10.1016%2Fj.ejpb.2015.02.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Characterisation of selected active agents regarding pKa values, solubility concentrations and pH profiles by SiriusT3</span></div><div class="casAuthors">Schonherr D; Wollatz U; Haznar-Garbacz D; Hanke U; Box K J; Taylor R; Ruiz R; Beato S; Becker D; Weitschies W</div><div class="citationInfo"><span class="NLM_cas:title">European journal of pharmaceutics and biopharmaceutics : official journal of Arbeitsgemeinschaft fur Pharmazeutische Verfahrenstechnik e.V</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-70</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The aim of this work was to determine pKa values and solubility properties of 34active agents using the SiriusT3 apparatus.  The selected drug substances belong to the groups of ACE-inhibitors, β-blockers, antidiabetics and lipid lowering substances.  Experimentally obtained pKa and intrinsic solubility values were compared to calculated values (program ACD/ChemSketch) and pKa values to published data as well.  Solubility-pH profiles were generated to visualise the substance solubility over the gastrointestinal pH range.  The relationship between the solubility characteristic of a substance, its bioavailability and categorisation according to the Biopharmaceutics Classification System (BCS) was examined as well.  The results showed a good agreement between experimentally obtained, calculated and published pKa values.  The measured and calculated intrinsic solubility values indicated several major deviations.  All solubility-pH profiles showed the expected shape and appearance for acids, bases or zwitterionic substances.  The obtained results for the pKa and solubility measurements of the examined active agents may help to predict their physicochemical behaviour in vivo, and to understand the bioavailability of the substances according to their BCS categorisation.  The easy and reproducible determination of pKa and solubility values makes the SiriusT3 apparatus a useful tool in early stages of drug and formulation development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR0tGrDkvlZJ-2qxNLjzXe8fW6udTcc2ebIg8tmB8nvLrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnivVKlsw%253D%253D&md5=02100c72ca191e1039faa3b20d5714d8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.ejpb.2015.02.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejpb.2015.02.028%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DD.%26aulast%3DWollatz%26aufirst%3DU.%26aulast%3DHaznar-Garbacz%26aufirst%3DD.%26aulast%3DHanke%26aufirst%3DU.%26aulast%3DBox%26aufirst%3DK.%2BJ.%26aulast%3DTaylor%26aufirst%3DR.%26aulast%3DRuiz%26aufirst%3DR.%26aulast%3DBeato%26aufirst%3DS.%26aulast%3DBecker%26aufirst%3DD.%26aulast%3DWeitschies%26aufirst%3DW.%26atitle%3DCharacterisation%2520of%2520Selected%2520Active%2520Agents%2520Regarding%2520pKa%2520Values%252C%2520Solubility%2520Concentrations%2520and%2520pH%2520Profiles%2520by%2520SiriusT3%26jtitle%3DEur.%2520J.%2520Pharm.%2520Biopharm.%26date%3D2015%26volume%3D92%26spage%3D155%26epage%3D170%26doi%3D10.1016%2Fj.ejpb.2015.02.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszczek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span> <span> </span><span class="NLM_article-title">High-Precision Isothermal Titration Calorimetry with Automated Peak-Shape Analysis</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">5066</span>– <span class="NLM_lpage">5073</span>, <span class="refDoi"> DOI: 10.1021/ac3007522</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac3007522" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=5066-5073&author=S.+Kellerauthor=C.+Vargasauthor=H.+Zhaoauthor=G.+Piszczekauthor=C.+A.+Brautigamauthor=P.+Schuck&title=High-Precision+Isothermal+Titration+Calorimetry+with+Automated+Peak-Shape+Analysis&doi=10.1021%2Fac3007522"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">High-Precision Isothermal Titration Calorimetry with Automated Peak-Shape Analysis</span></div><div class="casAuthors">Keller, Sandro; Vargas, Carolyn; Zhao, Huaying; Piszczek, Grzegorz; Brautigam, Chad A.; Schuck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5066-5073</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) is a powerful classical method that enables researchers in many fields to study the thermodn. of mol. interactions.  Primary ITC data comprise the temporal evolution of differential power reporting the heat of reaction during a series of injections of aliquots of a reactant into a sample cell.  By integration of each injection peak, an isotherm can be constructed of total changes in enthalpy as a function of changes in soln. compn., which is rich in thermodn. information on the reaction.  However, the signals from the injection peaks are superimposed by the stochastically varying time-course of the instrumental baseline power, limiting the precision of ITC isotherms.  The authors describe a method for automated peak assignment based on peak-shape anal. via singular value decompn. in combination with detailed least-squares modeling of local pre- and post- injection baselines.  This approach can effectively filter out contributions of short-term noise and adventitious events in the power trace.  This method also provides, for the first time, statistical error ests. for the individual isotherm data points.  In turn, this results in improved detection limits for high-affinity or low-enthalpy binding reactions and significantly higher precision of the derived thermodn. parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJcKwO_fHPFbVg90H21EOLACvtfcHk0ljKNCFaTW-TQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVGlsbs%253D&md5=bb157e956fa9cef94a88974d733cad1b</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fac3007522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac3007522%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DS.%26aulast%3DVargas%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DPiszczek%26aufirst%3DG.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26aulast%3DSchuck%26aufirst%3DP.%26atitle%3DHigh-Precision%2520Isothermal%2520Titration%2520Calorimetry%2520with%2520Automated%2520Peak-Shape%2520Analysis%26jtitle%3DAnal.%2520Chem.%26date%3D2012%26volume%3D84%26spage%3D5066%26epage%3D5073%26doi%3D10.1021%2Fac3007522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span> <span> </span><span class="NLM_article-title">High-Precision, Automated Integration of Multiple Isothermal Titration Calorimetric Thermograms: New Features of NITPIC</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2014.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1016%2Fj.ymeth.2014.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=25524420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC2MXltVOrug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2015&pages=87-98&author=T.+H.+Scheuermannauthor=C.+A.+Brautigam&title=High-Precision%2C+Automated+Integration+of+Multiple+Isothermal+Titration+Calorimetric+Thermograms%3A+New+Features+of+NITPIC&doi=10.1016%2Fj.ymeth.2014.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">High-precision, automated integration of multiple isothermal titration calorimetric thermograms: New features of NITPIC</span></div><div class="casAuthors">Scheuermann, Thomas H.; Brautigam, Chad A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">87-98</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Isothermal titrn. calorimetry (ITC) has become a std. and widely available tool to measure the thermodn. parameters of macromol. assocns.  Modern applications of the method, including global anal. and drug screening, require the acquisition of multiple sets of data; sometimes these data sets no. in the hundreds.  Therefore, there is a need for quick, precise, and automated means to process the data, particularly at the first step of data anal., which is commonly the integration of the raw data to yield an interpretable isotherm.  Herein, we describe enhancements to an algorithm that previously has been shown to provide an automated, unbiased, and high-precision means to integrate ITC data.  These improvements allow for the speedy and precise serial integration of an unlimited no. of ITC data sets, and they have been implemented in the freeware program NITPIC, version 1.1.0.  We present a comprehensive comparison of the performance of this software against an older version of NITPIC and a current version of Origin, which is commonly used for integration.  The new methods recapitulate the excellent performance of the previous versions of NITPIC while speeding it up substantially, and their precision is significantly better than that of Origin.  This new version of NITPIC is therefore well suited to the serial integration of many ITC data sets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyqQqBGpeedbVg90H21EOLACvtfcHk0ljKNCFaTW-TQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXltVOrug%253D%253D&md5=eb4e3e0603a9dbd330d3dbc0302b731c</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26atitle%3DHigh-Precision%252C%2520Automated%2520Integration%2520of%2520Multiple%2520Isothermal%2520Titration%2520Calorimetric%2520Thermograms%253A%2520New%2520Features%2520of%2520NITPIC%26jtitle%3DMethods%26date%3D2015%26volume%3D76%26spage%3D87%26epage%3D98%26doi%3D10.1016%2Fj.ymeth.2014.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, Scaling, Space-Group Assignment and Post-Refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+Scaling%2C+Space-Group+Assignment+and+Post-Refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0ljKNCFaTW-TQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520Scaling%252C%2520Space-Group%2520Assignment%2520and%2520Post-Refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sparta, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krug, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">XDSAPP2.0</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">1085</span>– <span class="NLM_lpage">1092</span>, <span class="refDoi"> DOI: 10.1107/S1600576716004416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS1600576716004416" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1085-1092&author=K.+M.+Spartaauthor=M.+Krugauthor=U.+Heinemannauthor=U.+Muellerauthor=M.+S.+Weiss&title=XDSAPP2.0&doi=10.1107%2FS1600576716004416"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS1600576716004416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600576716004416%26sid%3Dliteratum%253Aachs%26aulast%3DSparta%26aufirst%3DK.%2BM.%26aulast%3DKrug%26aufirst%3DM.%26aulast%3DHeinemann%26aufirst%3DU.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DXDSAPP2.0%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2016%26spage%3D1085%26epage%3D1092%26doi%3D10.1107%2FS1600576716004416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhLXGRfAsf3_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title"><i>PHENIX</i>: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhLXGRfAsf3_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of <i>Coot</i></span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhLXGRfAsf3_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laskowski, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacArthur, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. M.</span></span> <span> </span><span class="NLM_article-title"><i>PROCHECK</i>: A Program to Check the Stereochemical Quality of Protein Structures</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1107/S0021889892009944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-291&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+A+Program+to+Check+the+Stereochemical+Quality+of+Protein+Structures&doi=10.1107%2FS0021889892009944"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0lj0BNnCqwue6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520A%2520Program%2520to%2520Check%2520the%2520Stereochemical%2520Quality%2520of%2520Protein%2520Structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D291%26doi%3D10.1107%2FS0021889892009944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleywegt, G. J.</span></span> <i>MOLEMAN</i> - Unpublished Program; <span class="NLM_publisher-name">Uppsala University</span>: <span class="NLM_publisher-loc">Uppsala, Sweden</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kleywegt%2C+G.+J.+MOLEMAN+-+Unpublished+Program%3B+Uppsala+University%3A+Uppsala%2C+Sweden."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKleywegt%26aufirst%3DG.%2BJ.%26btitle%3DMOLEMAN%26pub%3DUppsala%2520University" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleywegt, G. J.</span>; <span class="NLM_string-name">Zou, J. Y.</span>; <span class="NLM_string-name">Kjeldgaard, M.</span>; <span class="NLM_string-name">Jones, T. A.</span></span> In  <i>International Tables for Crystallography: Crystallography of Biological Macromolecules</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M. G.</span>; <span class="NLM_string-name">Arnold, E.</span></span>, Eds.; <span class="NLM_publisher-name">Kluwer Academic Publishers</span>: <span class="NLM_publisher-loc">Dordrecht, The Netherlands</span>, <span class="NLM_year">2001</span>; Vol.  <span class="NLM_volume">F</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=G.+J.+Kleywegt&author=J.+Y.+Zou&author=M.+Kjeldgaard&author=T.+A.+Jonesauthor=M.+G.+Rossmann&author=E.+Arnold&title=International+Tables+for+Crystallography%3A+Crystallography+of+Biological+Macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKleywegt%26aufirst%3DG.%2BJ.%26btitle%3DInternational%2520Tables%2520for%2520Crystallography%253A%2520Crystallography%2520of%2520Biological%2520Macromolecules%26aulast%3DRossmann%26aufirst%3DM.%2BG.%26pub%3DKluwer%2520Academic%2520Publishers%26date%3D2001%26volume%3DF" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSX','PDB','6HSX'); return false;">PDB: 6HSX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q','PDB','6T3Q'); return false;">PDB: 6T3Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T4A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T4A','PDB','6T4A'); return false;">PDB: 6T4A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT','PDB','6TDT'); return false;">PDB: 6TDT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0M','PDB','6T0M'); return false;">PDB: 6T0M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0P','PDB','6T0P'); return false;">PDB: 6T0P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5W','PDB','6T5W'); return false;">PDB: 6T5W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SY3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SY3','PDB','6SY3'); return false;">PDB: 6SY3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZGB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZGB','PDB','2ZGB'); return false;">PDB: 2ZGB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1H8D','PDB','1H8D'); return false;">PDB: 1H8D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MNK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MNK','PDB','5MNK'); return false;">PDB: 5MNK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i42"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02061">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_74723"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b02061?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02061</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Figures of <i>mF</i><sub>o</sub> – <i>DF</i><sub>c</sub> omit maps of the investigated ligands; reference to the numbering of water molecules in the pdb files and the manuscript; ITC results from individual titrations; thermograms and binding isotherms from ITC measurements; and HPLC traces (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_001.pdf">PDF</a>). Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_001.pdf">jm9b02061_si_001.pdf (1.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02061/suppl_file/jm9b02061_si_002.csv">jm9b02061_si_002.csv (0.35 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors have been deposited in the Protein Data Bank and will be released upon article publication: <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6HSX">6HSX</a>), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T3Q">6T3Q</a>), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T4A">6T4A</a>), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TDT">6TDT</a>) for thrombin; <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0M">6T0M</a>), <b>2</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T0P">6T0P</a>), <b>3</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T5W">6T5W</a>), and <b>4</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SY3">6SY3</a>) for trypsin.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b02061&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b02061%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-6" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b02061" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799e8b288ba22c0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
