
<html lang="en"     class="pb-page"  data-request-id="426a80c9-3a12-4b06-af5e-e88db5c6b73c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2020.63.issue-9;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.9b01910"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Imidazo[1,2-a]pyridine Derivatives as Aldehyde Dehydrogenase Inhibitors: Novel Chemotypes to Target Glioblastoma Stem Cells" /></meta><meta name="dc.Creator" content="Luca  Quattrini" /></meta><meta name="dc.Creator" content="Edoardo Luigi Maria  Gelardi" /></meta><meta name="dc.Creator" content="Vito  Coviello" /></meta><meta name="dc.Creator" content="Stefania  Sartini" /></meta><meta name="dc.Creator" content="Davide Maria  Ferraris" /></meta><meta name="dc.Creator" content="Mattia  Mori" /></meta><meta name="dc.Creator" content="Ichiro  Nakano" /></meta><meta name="dc.Creator" content="Silvia  Garavaglia" /></meta><meta name="dc.Creator" content="Concettina  La Motta" /></meta><meta name="dc.Description" content="Glioblastoma multiforme (GBM) is the deadliest form of brain tumor. It is known for its ability to escape the therapeutic options available to date thanks to the presence of a subset of cells endow..." /></meta><meta name="Description" content="Glioblastoma multiforme (GBM) is the deadliest form of brain tumor. It is known for its ability to escape the therapeutic options available to date thanks to the presence of a subset of cells endow..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 30, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01910" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01910" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01910" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01910" /></link>
        
    
    

<title>Imidazo[1,2-a]pyridine Derivatives as Aldehyde Dehydrogenase Inhibitors: Novel Chemotypes to Target Glioblastoma Stem Cells | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01910" /></meta><meta property="og:title" content="Imidazo[1,2-a]pyridine Derivatives as Aldehyde Dehydrogenase Inhibitors: Novel Chemotypes to Target Glioblastoma Stem Cells" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0012.jpeg" /></meta><meta property="og:description" content="Glioblastoma multiforme (GBM) is the deadliest form of brain tumor. It is known for its ability to escape the therapeutic options available to date thanks to the presence of a subset of cells endowed with stem-like properties and ability to resist to cytotoxic treatments. As the cytosolic enzyme aldehyde dehydrogenase 1A3 turns out to be overexpressed in these kinds of cells, playing a key role for their vitality, treatments targeting this enzyme may represent a successful strategy to fight GBM. In this work, we describe a novel class of imidazo[1,2-a]pyridine derivatives as aldehyde dehydrogenase 1A3 inhibitors, reporting the evidence of their significance as novel drug candidates for the treatment of GBM. Besides showing an interesting functional profile, in terms of activity against the target enzyme and selectivity toward highly homologous isoenzymes, representative examples of the series also showed a nanomolar to picomolar efficacy against patient-derived GBM stem-like cells, thus proving the concept that targeting aldehyde dehydrogenase might represent a novel and promising way to combat GBM by striking its ability to divide immortally." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01910"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01910">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01910&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01910&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01910&amp;href=/doi/10.1021/acs.jmedchem.9b01910" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4603-4616</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01835" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01918" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Imidazo[1,2-<i>a</i>]pyridine Derivatives as Aldehyde Dehydrogenase Inhibitors: Novel Chemotypes to Target Glioblastoma Stem Cells</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Luca Quattrini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luca Quattrini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luca++Quattrini">Luca Quattrini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Edoardo Luigi Maria Gelardi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Edoardo Luigi Maria Gelardi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Piemonte Orientale, Via Bovio 6, 28100 Novara, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Edoardo+Luigi+Maria++Gelardi">Edoardo Luigi Maria Gelardi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vito Coviello</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vito Coviello</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vito++Coviello">Vito Coviello</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefania Sartini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefania Sartini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefania++Sartini">Stefania Sartini</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7671-5243" title="Orcid link">http://orcid.org/0000-0002-7671-5243</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Davide Maria Ferraris</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Davide Maria Ferraris</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Piemonte Orientale, Via Bovio 6, 28100 Novara, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Davide+Maria++Ferraris">Davide Maria Ferraris</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mattia Mori</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mattia Mori</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biotechnology, Chemistry and Pharmacy “Department of Excellence 2018-2022”, University of Siena, via Aldo Moro 2, 53100 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mattia++Mori">Mattia Mori</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2398-1254" title="Orcid link">http://orcid.org/0000-0003-2398-1254</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ichiro Nakano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ichiro Nakano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Neurosurgery, University of Alabama at Birmingham, WTI 401, 1824 Sixth Avenue South, Birmingham, Alabama 35233, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ichiro++Nakano">Ichiro Nakano</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Silvia Garavaglia</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Silvia Garavaglia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Piemonte Orientale, Via Bovio 6, 28100 Novara, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6f1c060319060e41080e1d0e190e0803060e2f1a01061a1f0041061b"><span class="__cf_email__" data-cfemail="fe8d979288979fd0999f8c9f889f9992979fbe8b90978b8e91d0978a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Silvia++Garavaglia">Silvia Garavaglia</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Concettina La Motta</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Concettina La Motta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f6959998959382829f9897d89a979b99828297b683989f869fd89f82"><span class="__cf_email__" data-cfemail="16757978757362627f7877387a777b796262775663787f667f387f62">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Concettina++La+Motta">Concettina La Motta</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0194-5431" title="Orcid link">http://orcid.org/0000-0002-0194-5431</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01910&amp;href=/doi/10.1021%2Facs.jmedchem.9b01910" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4603–4616</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 30, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 November 2019</li><li><span class="item_label"><b>Published</b> online</span>30 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 May 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01910" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01910</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4603%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLuca%2BQuattrini%252C%2BEdoardo%2BLuigi%2BMaria%2BGelardi%252C%2BVito%2BCoviello%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D9%26contentID%3Dacs.jmedchem.9b01910%26title%3DImidazo%255B1%252C2-a%255Dpyridine%2BDerivatives%2Bas%2BAldehyde%2BDehydrogenase%2BInhibitors%253A%2BNovel%2BChemotypes%2Bto%2BTarget%2BGlioblastoma%2BStem%2BCells%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4616%26publicationDate%3DMay%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01910"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1145</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01910" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Imidazo[1,2-a]pyridine Derivatives as Aldehyde Dehydrogenase Inhibitors: Novel Chemotypes to Target Glioblastoma Stem Cells&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Luca&quot;,&quot;last_name&quot;:&quot;Quattrini&quot;},{&quot;first_name&quot;:&quot;Edoardo&quot;,&quot;last_name&quot;:&quot;Luigi Maria Gelardi&quot;},{&quot;first_name&quot;:&quot;Vito&quot;,&quot;last_name&quot;:&quot;Coviello&quot;},{&quot;first_name&quot;:&quot;Stefania&quot;,&quot;last_name&quot;:&quot;Sartini&quot;},{&quot;first_name&quot;:&quot;Davide&quot;,&quot;last_name&quot;:&quot;Maria Ferraris&quot;},{&quot;first_name&quot;:&quot;Mattia&quot;,&quot;last_name&quot;:&quot;Mori&quot;},{&quot;first_name&quot;:&quot;Ichiro&quot;,&quot;last_name&quot;:&quot;Nakano&quot;},{&quot;first_name&quot;:&quot;Silvia&quot;,&quot;last_name&quot;:&quot;Garavaglia&quot;},{&quot;first_name&quot;:&quot;Concettina&quot;,&quot;last_name&quot;:&quot;La Motta&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4603-4616&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01910&quot;},&quot;abstract&quot;:&quot;Glioblastoma multiforme (GBM) is the deadliest form of brain tumor. It is known for its ability to escape the therapeutic options available to date thanks to the presence of a subset of cells endowed with stem-like properties and ability to resist to cytotoxic treatments. As the cytosolic enzyme aldehyde dehydrogenase 1A3 turns out to be overexpressed in these kinds of cells, playing a key role for their vitality, treatments targeting this enzyme may represent a successful strategy to fight GBM. In this work, we describe a novel class of imidazo[1,2-a]pyridine derivatives as aldehyde dehydrogenase 1A3 inhibitors, reporting the evidence of their significance as novel drug candidates for the treatment of GBM. Besides showing an interesting functional profile, in terms of activity against the target enzyme and selectivity toward highly homologous isoenzymes, representative examples of the series also showed a nanomolar to picomolar efficacy against patient-derived GBM stem-like cells, thus proving the concept th&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01910&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01910" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01910&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01910" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01910&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01910" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01910&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01910&amp;href=/doi/10.1021/acs.jmedchem.9b01910" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01910" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01910" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01910%26sid%3Dliteratum%253Aachs%26pmid%3D32223240%26genre%3Darticle%26aulast%3DQuattrini%26date%3D2020%26atitle%3DImidazo%255B1%252C2-a%255Dpyridine%2BDerivatives%2Bas%2BAldehyde%2BDehydrogenase%2BInhibitors%253A%2BNovel%2BChemotypes%2Bto%2BTarget%2BGlioblastoma%2BStem%2BCells%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D9%26spage%3D4603%26epage%3D4616%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292236" title="Aldehydes">Aldehydes</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/jmcmar.2020.63.issue-9/20200514/jmcmar.2020.63.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Glioblastoma multiforme (GBM) is the deadliest form of brain tumor. It is known for its ability to escape the therapeutic options available to date thanks to the presence of a subset of cells endowed with stem-like properties and ability to resist to cytotoxic treatments. As the cytosolic enzyme aldehyde dehydrogenase 1A3 turns out to be overexpressed in these kinds of cells, playing a key role for their vitality, treatments targeting this enzyme may represent a successful strategy to fight GBM. In this work, we describe a novel class of imidazo[1,2-<i>a</i>]pyridine derivatives as aldehyde dehydrogenase 1A3 inhibitors, reporting the evidence of their significance as novel drug candidates for the treatment of GBM. Besides showing an interesting functional profile, in terms of activity against the target enzyme and selectivity toward highly homologous isoenzymes, representative examples of the series also showed a nanomolar to picomolar efficacy against patient-derived GBM stem-like cells, thus proving the concept that targeting aldehyde dehydrogenase might represent a novel and promising way to combat GBM by striking its ability to divide immortally.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44371" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44371" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Glioblastoma multiforme (GBM) is the most aggressive and fast-growing astrocytic tumor, accounting for about 15% of all brain tumors in adults. Although rare at presentation, with a yearly incidence of about two to three per 100 000 adults in both North America and Europe, GBM represents a critical health issue due to its unavoidably poor prognosis.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">The treatment of choice for GMB relies on the maximal surgical resection of the tumor mass, associated with external beam radiation therapy and the adjuvant administration of Temozolomide (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>). However, the remarkable toxicity of the therapeutic intervention, combined with both its marginal efficacy and the emergence of relapse, generally limit the median survival of the treated patients to approximately 14 to 15 months from the diagnosis, with less than 5% surviving over 5 years.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Accordingly, a ground-breaking therapeutic solution for people affected by GBM is urgently needed.</div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0013.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Temozolomide and Aldehyde Dehydrogenase Inhibitors under Investigation</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As a malignancy, GBM is made of a heterogeneous community of different cells governed by cooperative interactions.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> A small percentage of them, bearing stem-like properties, plays a key role in tumor perpetuating and progression, being endowed with indefinite self-renewal ability and aberrant differentiation capacity.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a></div><div class="NLM_p">Differently from tumor bulk cells, glioma stem-like cells (GSCs) are able to resist cytotoxic treatments commonly exploited to target rapidly dividing cells, as they are characterized by a slow mitotic activity possibly associated with a quiescent G0/G1 state. Hence, rather than being eradicated, GSCs turn out to be selected over the cancer bulk, being free to drive tumor relapse and mediate metastasis. Therefore, they represent the reference subpopulation that should be brought into focus, as undermining self-renewal signaling pathways supporting their growth and residence in the stem state may represent an effective strategy to wipe out efficiently the whole tumor architecture. Indeed, experience gained in the management of different malignancies proves the concept that targeting cancer stem cells improves patient outcomes by preventing tumors recurrences,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> increasing chemotherapy efficacy,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and reversing chemotherapy resistance.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">To date, two distinct and mutually exclusive populations of GSCs have been differentiated by transcriptome array analyses, namely the proneural (PN-) and the mesenchymal (MES-) ones, strictly correlated to the clinically relevant proneural and mesenchymal subtypes of GBM.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">While the PN-GSC subtype tends to adopt a MES identity in response to radiation treatment, MES-GSCs display a well-characterized profile, turning out to be an aggressive phenotype markedly resistant to radiation and showing high expression of genes associated with DNA repair. MES-GSCs also exhibit high levels of the cytoplasmic enzyme aldehyde dehydrogenase, subtype 1A3 (ALDH1A3). Besides being acknowledged as the distinctive hallmark of this type of cancer cell subpopulation, ALDH1A3 also plays a crucial role in stem cells vitality, being involved in the differentiation of progenitor cells into mature entities. Its overexpression endorses high grade gliomas and is decisive for tumor invasion, thus having a prognostic value for poor clinical outcomes.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Accordingly, targeting this enzyme might represent a successful strategy to put in place a keystone molecular treatment of GBM.</div><div class="NLM_p">By exploiting a rational approach, we recently succeeded in turning the natural ALDH inhibitor daidzin into a novel effective derivative, 2,6-diphenylimidazo[1,2-<i>a</i>]pyridine, namely GA11 (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>, compound <b>3a</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), which proved to combine a significant <i>in vitro</i> inhibitory activity against the target enzyme with a potent <i>in vivo</i> antitumor efficacy when administered in a murine GSC-derived xenograft model of GBM.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Taken together, these results were extremely encouraging as they affirmed GA11 as a sound and viable candidate deserving of further development.</div><div class="NLM_p">Therefore, guided by a preliminary crystallization study, performed to highlight GA11 accommodation into the active site of ALDH1A3, we embarked in the structural optimization of the hit compound with the purpose to obtain a novel and effective class of ALDH1A3 inhibitors.</div><div class="NLM_p last">In this work, we present the synthesis of novel derivatives of GA11, whose phenyl rings in both positions 2 and 6 of the heterocyclic core were suitably modified in their electronic and steric features through the insertion suitable substituents. All the synthesized compounds were evaluated in vitro for their activity against the target ALDH1A3 and selectivity toward the parent ALDH1A1 and ALDH1A2 subtypes as well as for their antiproliferative activity toward selected GBM cell lines. Results obtained were rationalized through molecular docking reasoning.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34168" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34168" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Crystallization Study</h3><div class="NLM_p">The development of a compound able to target the human ALDH1A3 is a difficult challenge, especially when a selectivity profile for the 1A1 and 1A2 isoenzymes is also pursued. Therefore, an integrated approach that would use both structural studies and in silico drug design is advisable. All the ALDH1A isoenzymes are able to catalyze the conversion of the natural substrate retinal to retinoic acid, but they show preferences for a different isomer of retinal. ALDH1A3, in particular, has a major affinity for all-trans retinal. It implies some diversity in the features defining the catalytic site of the different isoforms, which should be taken into consideration to get to novel active and selective derivatives.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Indeed, the geometry of the substrate access tunnel has a central role in ALDHs specificity.</div><div class="NLM_p">We started our research project cocrystallizing the reference hit GA11 (<b>3a</b> from here on out), with the human recombinant ALDH1A3, to begin deciphering its binding mode to the target enzyme (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>). The obtained three-dimensional structure of human ALDH1A3 in complex with <b>3a</b> has been solved by molecular replacement using the human ALDH1A3 atomic coordinates as a search model (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FHZ">5FHZ</a><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a>) and refined at a resolution of 3.2 Å (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The final human ALDH1A3 model contains two identical chains per asymmetric unit, arranged as dimer, and a total of 11 solvent molecules (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Data Collection and Refinement Statistics of Three-Dimensional Structure of Human ALDH1A3 in Complex with Compounds <b>3a</b> (PDB Code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>) and <b>3q</b> (PDB Code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5">6TE5</a>)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="3" align="center">data collection</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">compound <b>3a</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">compound <b>3b</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>22<sub>1</sub></td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>22<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left">Cell Dimensions</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">81.6, 89.4, 159.0</td><td class="colsep0 rowsep0" align="left">81.2, 89.4, 159.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (°)</td><td class="colsep0 rowsep0" align="left">90, 90, 90</td><td class="colsep0 rowsep0" align="left">90, 90, 90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">48.02–3.2</td><td class="colsep0 rowsep0" align="left">48.02–3.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>pim</sub>/<i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="left">0.064 (0.399)/0.128 (0.787)</td><td class="colsep0 rowsep0" align="left">0.076 (0.399)/0.168 (0.772)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean(I)/sd(I)</td><td class="colsep0 rowsep0" align="left">9.8 (2.1)</td><td class="colsep0 rowsep0" align="left">8.9 (2.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.4 (99.3)</td><td class="colsep0 rowsep0" align="left">99.8 (99.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">4.7 (4.6)</td><td class="colsep0 rowsep0" align="left">5.6 (5.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">3.2</td><td class="colsep0 rowsep0" align="left">3.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. reflections</td><td class="colsep0 rowsep0" align="left">18 833 (1840)</td><td class="colsep0 rowsep0" align="left">18 833 (1823)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub>/<i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">0.19/0.26</td><td class="colsep0 rowsep0" align="left">0.18/0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left">No. Atoms</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">7446</td><td class="colsep0 rowsep0" align="left">7446</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">121</td><td class="colsep0 rowsep0" align="left">148</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left">Mean B-Factors</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">77.6</td><td class="colsep0 rowsep0" align="left">56.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">52.4</td><td class="colsep0 rowsep0" align="left">48.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">94.8</td><td class="colsep0 rowsep0" align="left">82.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="left">R.m.s. Deviations</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.006</td><td class="colsep0 rowsep0" align="left">0.014</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (°)</td><td class="colsep0 rowsep0" align="left">1.10</td><td class="colsep0 rowsep0" align="left">1.63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran outliers (%)</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">PDB deposition name: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11">GA11</a>.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">PDB deposition name: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43">LQ43</a>.</p></div></div></div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overall monomer structure of human ALDH1A3 in complex with <b>3a</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>, PDB deposition name: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11">GA11</a>). (A) Ribbon representation of the overall structure of the monomer of ALDH1A3. Each monomer consists of three domains: the NAD<sup>+</sup> binding-domain, the catalytic-domain, and the oligomerization domain. The ligands <b>3a</b> and NAD<sup>+</sup> are shown as yellow and gray sticks, respectively. (B) Electron density map SA-omit (2mFo-DFc) of <b>3a</b>, in gray, contoured at 1 sigma. (C) Superimposition between electron density map of <b>3a</b> SA-omit (2mFo-DFc), in gray, and <b>3a</b> 2mFo-DFc, in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The structure analysis confirmed that the obtained <b>3a</b>-ALDH1A3 complex folds into 13 α-helices, 19 β-strands, and the connecting loops, arranged into three functional domains. Both the NAD<sup>+</sup> binding domain and the catalytic domain are built on a topologically related βαβ type polypeptide fold (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Our crystallographic data revealed that, in all monomers, <b>3a</b> binds to the enzyme at the entrance of the tunnel, which therefore turns out to be partially clogged by the compound. It is important to highlight that <b>3a</b> is a symmetric compound, and its phenyl rings at position 2 and 6 of the imidazo[1,2-<i>a</i>]pyridine scaffold can bind the enzyme site in two overturned, almost equivalent conformations Therefore, to fully unveil the binding mode of the compound, we decided to make use of docking simulations. Molecular modeling was carried out by GOLD docking program<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> toward the crystallographic structure of ALDH1A3 described above and solved in complex with <b>3a</b>. Results clearly show that two possible binding modes of <b>3a</b> are possible, depicted in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. They both share a comparable affinity, with a slightly higher theoretical affinity for the binding mode described in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, where the phenyl ring in position 6 of the imidazo[1,2-<i>a</i>]pyridine scaffold is projected toward the binding site flanked by residues G136, L185, W189, L471, and A473. It is worth noting that this is the same site occupied by the cyclohexene ring of the natural ligand retinoic acid, such as recently highlighted by X-ray crystallography.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> By adopting this pose, <b>3a</b> casts the 2-phenyl ring toward the side chain of the aromatic F308, thus allowing a π–π stacking interaction which makes the compound binding more effective. No poses were found within the hydrophobic pocket of the catalytic site, in proximity to the catalytic residues.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Stick representations of the possible docking poses of compound <b>3a</b> into the binding site of ALDH1A3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>). The binding site of ALDH1A3 is represented as green ribbons. Residues relevant for the inhibitor binding have been labeled, and their side chains are shown as green sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Piecing together the inspection of the electron density map of the <b>3a</b>-ALDH1A3 complex with the evidence achieved by the molecular modeling study, we resolved to ascribe to <b>3a</b> the binding pose depicted in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>. In this conformation, the N1 atom of the scaffold interacts with the carbonyl oxygen of F131, while the whole heterocyclic core lies in a conserved hydrophobic pocket located at the entrance of the tunnel that provides substrate access to and product release from, the catalytic cysteine. Moreover, the phenyl ring in position 6 of the nucleus establishes contact with the protein backbone, including π–π stacking interaction with W189 (3.8 Å) and Van der Waals contacts with G136 (3.6 Å), L185 (3.8 Å), and L471 (4.5 Å) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representation of the main interactions between <b>3a</b> and the ALDH1A3 binding site (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>). Buried surface areas of monomer A is colored in forest green and the ligand <b>3a</b> is shown as yellow stick. Side chains of key residues, involved in <b>3a</b> binding, are shown in forest green sticks. E135, F131, G136, L185, W189, A473, L471, and Y472 stabilize through π–π stacking and hydrophobic interactions the <b>3a</b> phenyl rings. The carbonyl oxygen of F131 interacts with N1 of <b>3a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The mentioned residues are conserved in each of the 1A1, 1A2, and 1A3 isozymes, being that this enzymatic area is the one stabilizing retinoic acid binding in the whole members of the 1A subfamily, as demonstrated in a previous work.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> ALDH1A3 shows a high homology degree compared to the other members of the 1A subfamily. Actually, the 1A3 structure can be optimally superimposed onto human ALDH1A1 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WB9">4WB9</a><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a>), with a r.m.s.d. of 0.96 Å, and onto human ALDH1A2 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5G">6B5G</a><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a>), with a r.m.s.d. of 0.92 Å, based on 485 equivalent Cα pairs (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of ALDH1A binding-sites conformations. Superposition ribbon representation of monomer overall structures of ALDH1A3 in forest green (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>), ALDH1A2 in orange (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5g">6B5g</a>), and ALDH1A1 in cyan (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WB9">4WB9</a>). The three structures maintain the same arrangement. The LigPlus representation of the complex <b>3a</b>-ALDH1A3 monomer A highlights all the eight amino acids involved in <b>3a</b> binding site. The ALDH1A3 amino acid residues circled in green are not conserved in ALDH1A1 and ALDH1A2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, ALDH1A3 shares 71% amino acid identity with human isozymes 1A1 and 72% amino acid identity with 1A2. However, as detailed in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, a few differences in the amino acid sequences of the three isozymes do exist. Indeed, the superposition of 1A3, 1A1, and 1A2 forms highlights that three amino acids are not conserved. The hydrophilic F131 of ALDH1A3 corresponds to L120 in ALDH1A1 and Y137 in ALDH1A2, the E135 residue of ALDH1A3 corresponds to A124 in ALDH1A1 and Q141 in ALDH1A2, while the Y472 residue of ALDH1A3 corresponds to S461 in ALDH1A1 and to N478 in ALDH1A2. The three mentioned residues affect the binding pocket hosting <b>3a</b> in particular, thus giving reason to the inhibitory selectivity displayed by the compound against the three isozymes (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Superposition overview of different amino acids involved in <b>3a</b>-ALDH1A3 binding. Side chains of key residues involved in the binding (F131, E135, Y472, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>) are shown in forest green sticks. The correspondent residues not conserved of ALDH1A2 (Y137, Q141, N478, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5G">6B5G</a>) and of ALDH1A1 (L120, A124, S461, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WB9">4WB9</a>) are represented by orange sticks and cyan sticks, respectively. In the sequence overlapping of the three isoenzymes, the residues not conserved are circled in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. ALDH Inhibitory Activity of Selected Imidazo[1,2-<i>a</i>]pyridine Derivatives <b>3a–v</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0011.gif" alt="" id="GRAPHIC-d7e893-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">IC<sub>50</sub> (μM<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a>) or % inhibition<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">N</th><th class="colsep0 rowsep0" align="center">R<sub>1</sub></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>3</sub></th><th class="colsep0 rowsep0" align="center">ALDH1A1</th><th class="colsep0 rowsep0" align="center">ALDH1A2</th><th class="colsep0 rowsep0" align="center">ALDH1A3</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">19.6 ± 1.6</td><td class="colsep0 rowsep0" align="left">54.6 ± 1.9</td><td class="colsep0 rowsep0" align="left">4.7 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-F</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">22.8 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="left">68.8 ± 1.5</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">45.8 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3d</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-SCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">27.1 ± 1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3e</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">17.8 ± 1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3f</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-COOCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">96.9 ± 2.2</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">27.4 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3h</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3-CN</td><td class="colsep0 rowsep0" align="left">11.3%</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">5.3 ± 1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3i</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-CN</td><td class="colsep0 rowsep0" align="left">15.9%</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">5.2 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3j</b></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">22.4%</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">11.3 ± 1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3m</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="left">17.3 ± 1.6</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3o</b></td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">22.9%</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">11.4 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3p</b></td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3,4-diOCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">17.5%</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">8.3 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3q</b></td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3,5-diOCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">197.7 ± 3.0</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">3.5 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3r</b></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">4-F</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">92.9 ± 1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3u</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">4-F</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">21.2 ± 1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3v</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">n.a.<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">6.4 ± 1.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values represent the concentration required to produce 50% enzyme inhibition.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Percentage of enzyme inhibition. Standard errors of the mean (SEMs) ≤ 10%.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Not active. No significant enzyme inhibition was observed at 25 μM.</p></div></div></div><div class="NLM_p">Overall, both crystallographic studies and molecular modeling investigations evidence that <b>3a</b> is able to bind the entrance of the catalytic site of ALDH1A3 without protruding inside the catalytic pocket. Clear docking sites for phenyl rings in positions 2 and 6 of the nucleus were identified, thus offering interesting clues for the rational optimization of the hit. In particular, they lead to speculate that the insertion of suitable substituents on the two pendant rings may help to fill in further the binding pocket of the target proteins, thus getting to novel effective analogues. On the basis of these premises, we synthesized a number of 6-substituted-imidazo[1,2-<i>a</i>]pyridines, <b>3a–v</b> (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>) as well as few examples of 7-substituted-imidazo[1,2-<i>a</i>]pyridines, <b>7a–i</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>) to shed light on both electronic and geometrical requirements of the imidazo[1,2-<i>a</i>]pyridine class for the obtainment of ALDH1A3 inhibitors.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 6-Substituted-imidazo[1,2-<i>a</i>]pyridine Derivatives <b>3a–k,n–v</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 2-bromo-1-(4-(3)substitutedphenyl)ethan-1-one, K<sub>2</sub>CO<sub>3</sub> Δ; (ii) substitued-phenylboronic acid, Pd(OAc)<sub>2;</sub> PPh<sub>3</sub> K<sub>2</sub>CO<sub>3</sub> Δ.</p></p></figure><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 6-Substituted-imidazo[<b>1</b>,2-<i>a</i>]pyridine Derivatives <b>31,m</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 4-chlorophenylboronic acid, Pd(OAc)<sub>2,</sub> PPh<sub>3,</sub> K<sub>2</sub>CO<sub>3</sub> Δ 2-bromo-1-(4-substitutedphenyl)ethan-1-one, K<sub>2</sub>CO<sub>3</sub> Δ.</p></p></figure><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 7-Substituted-imidazo[1,2-<i>a</i>]pyridine Derivatives <b>7a–i</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 2-bromo-l-(4-(3)substitutedphenyl)ethan-l-one, K<sub>2</sub>CO<sub>3</sub> Δ; (ii) substitued-phenylboronic acid, Pd(OAc)<sub>2,</sub> PPh<sub>3,</sub> K<sub>2</sub>CO<sub>3</sub> Δ.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Chemistry</h3><div class="NLM_p last">The synthesis of the 6-substituted-imidazo[1,2-<i>a</i>]pyridines <b>3a–k,p–v</b> was performed as outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The commercially available 2-amino-5-bromo pyridine <b>1</b> was treated with the suitably substituted 2-bromo-1-phenylethan-1-one, in the presence of potassium carbonate, to achieve the corresponding 2-phenyl-imidazo[1,2-<i>a</i>]pyridines <b>2a–e</b>. Coupling the obtained heterocycles with differently substituted phenylboronic acids, performed in the presence of Pd(OAc)<sub>2</sub> and PPh<sub>3</sub> as the catalysts, provided the desired inhibitors. Compounds <b>3l,m</b> were synthesized following the alternative synthetic sequence described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, which allowed researchers to get the imidazo[1,2-<i>a</i>]pyridine derivatives with comparable yields. Coupling 2-amino-5-bromopyridine <b>1</b> with the appropriate phenylboronic acids, under the experimental conditions outlined above, gave the intermediate 5-(4-chlorophenyl)pyridin-2-amine <b>4</b>, which was cyclized to the desired 2-phenyl-imidazo[1,2-<i>a</i>]pyrimidines <b>3l,m</b> by reaction with 2-bromo-1-(4- substitutedphenyl)eth an-1-one. 7-Substituted-imidazo[1,2-<i>a</i>]pyridines, <b>7a–i</b>, represented in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, were obtained following similar reactions. The commercially available 2-amino-4-bromo pyridine <b>5</b> and the appropriate 2-bromo-1-phenylethan-1-ones gave the key intermediates 7-bromo-2-phenylimidazo[1,2-<i>a</i>]pyridines <b>6a–c</b>, which were converted into the target inhibitors <b>7a–i</b> by reaction with different phenyl boronic acids, under the customary coupling conditions in the presence of Pd(OAc)<sub>2</sub> and PPh<sub>3</sub> as the catalysts.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Functional Evaluation and Modeling Studies</h3><div class="NLM_p">The novel 6-phenyl-substituted-imidazo[1,2-<i>a</i>]pyridine derivatives were tested for both their inhibitory efficacy against the human recombinant ALDH1A3 and selectivity toward the ALDH1A1 and ALDH1A2 isoforms. The kinetic parameters were determined for all the compounds, following the experimental protocol described hereafter. Regarding the reference hit <b>3a</b> (GA11, IC<sub>50</sub> 4.7 ± 1.7 μM, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), an in depth analysis of its functional activity revealed a <i>K</i><sub>i</sub> value of 0.54 ± 0.11 μM and a partially competitive mode of action against the target enzyme ALDH1A3 (<a class="ref internalNav" href="#notes1" aria-label="Figure 1-SI">Figure 1-SI</a>, Supporting Information). These results are consistent with structural data, showing that the compound binds to the entrance of the catalytic tunnel, hindering partially the substrate flow without coordinating directly the catalytic residue C314. To provide atomistic insights into the mechanism of inhibition, the copresence of <b>3a</b> and the substrate acetaldehyde in the catalytic cavity of ALDH1A3 was investigated by molecular docking and molecular dynamic (MD) simulations (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). By referring to the crystallographic complex between ALDH1A3 and <b>3a</b>, two poses for acetaldehyde were found, that is, in proximity of F308 and close to C314 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). MD simulations on both docking poses clearly showed that, starting from F308, acetaldehyde quickly leaves the catalytic site and moves to the bulk solvent (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). In contrast, starting from C314, acetaldehyde persists within the catalytic tunnel for around 150 ns (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C) before moving to the bulk solvent, being <b>3a</b> firmly anchored at the entrance of the site. Overall, the structural analysis suggests that both acetaldehyde and <b>3a</b> can occupy simultaneously the ALDH1A3 catalytic site, thus giving rise to a partially competitive mechanism of action.</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Predicted binding mode and MD simulations of acetaldehyde in the <b>3a</b>-ALDH1A3 crystallographic complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>). (A) Obtained docking poses of acetaldehyde, represented as sticks and either cyan or yellow balls. (B, C) Plots of the distances between acetaldehyde and the catalytic C314 along MD trajectories starting from the docking poses, represented in cyan and yellow respectively, indicated by black arrows. MD simulations on the cyan pose were stopped after around 150 ns, as the small molecule detaches from the ALDH1A3 catalytic site and moves to the bulk solvent.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As for the novel imidazo[1,2-<i>a</i>]pyridine derivatives, most of them proved to inhibit the target ALDH1A3 isoform when tested at 25 μM concentration, displaying different degrees of efficacy depending on the substitution patterns on the pendant phenyl rings (<a class="ref internalNav" href="#notes1" aria-label="Figure 2-SI">Figure 2-SI</a>, Supporting Information). Compounds showing good inhibitory activity at 25 μM were investigated further to determine their IC<sub>50</sub> values (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Results obtained enable us to draft structure–activity relationships for the imidazo[1,2-<i>a</i>]pyridine class.</div><div class="NLM_p">In particular, starting from the unsubstituted hit <b>3a</b> (GA11, IC<sub>50</sub> 4.7 ± 1.7 μM), modifications of the 6-phenyl area alone, through the insertion of increasingly bulky substituents as in compounds <b>3b–i</b>, were well tolerated, regardless the electronic properties of the added groups. The best results were achieved with compounds <b>3h</b>,<b>i</b>, bearing the electro-withdrawing nitrile group in position <i>meta</i> and <i>para</i> of the ring, respectively, which turned out to be as potent as the parent hit <b>3a</b> (<b>3h</b>, IC<sub>50</sub> 5.3 ± 1.5 μM; <b>3i</b>, IC<sub>50</sub> 5.2 ± 1.7 μM). The inhibitory efficacy of the hit was kept unaltered also by inserting a methyl group in position <i>para</i> of the 2-phenyl ring, as in derivative <b>3j</b> (IC<sub>50</sub> 11.3 ± 1.5 μM).</div><div class="NLM_p">Double substitution patterns on both the pendant phenyl rings of the hit were explored as well, pursuing different combinations of both electron-withdrawing and electron-donating substituents. In this regard, the best results in terms of inhibitory efficacy were obtained by using the electron-donating methoxy group. Actually, by merging the 2-(4-methoxy)phenyl residue with a mono- or dimethoxy substitution on the 6-phenyl ring, a relevant activity was obtained and the resulting compounds, <b>3o–q</b>, displayed IC<sub>50</sub> values in the low micromolar range.</div><div class="NLM_p">By co-crystallizing <b>3q</b> (IC<sub>50</sub> 3.5 ± 1.2 μM) with the human ALDH1A3 enzyme (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5">6TE5</a>, PDB deposition name <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43">LQ43</a>, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), it has been possible to verify that the oxygen atom in position para on the 2-phenyl ring is involved in a hydrogen bond with Q304. This interaction is complemented by a further hydrogen bond between the methoxy group in position <i>meta</i> of the 6-phenyl ring (O2) and the NE1 residue of W189 (3.5 Å) and OG1 of T140 (3.39 Å), which ties firmly the compound into the ALDH1A3 binding site. The 6-phenyl ring of the compound also establishes hydrophobic contacts with the protein backbone, including π–π stacking interaction with E135 and Y472, crucial residues of ALDH1A3 not conserved in the parent isoenzymes. Actually, as highlighted in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, the key Y472 is replaced by S461 and N478 in the 1A1 and 1A2 isoenzymes, respectively, thus accounting for the inhibitory selectivity displayed by <b>3q</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overall monomer structure of human ALDH1A3 in complex with <b>3q</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5">6TE5</a>, PDB deposition name <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43">LQ43</a>). (A) Ribbon representation of the overall structure of the monomer of ALDH1A3. The LigPlus representation of the complex <b>3q</b>-ALDH1A3 monomer B highlights all the amino acids involved in the binding. The ALDH1A3 amino acid residues circled in green are not conserved in ALDH1A1 and ALDH1A2. (B) Compound <b>3q</b> into the enzyme binding site. The ligand is shown as cyan stick and the side chains of key residues, involved in the binding, are shown in forest green sticks. The electron density map (2mFo-dFc) countered at 1.0 sigma, used to determine the presence of <b>3q</b> into the enzyme binding site, is in blue. (C) Electron density map SA-omit (2mFo-dFc) of <b>3q</b>, in gray, contoured at 1 sigma. (D) Superimposition between electron density map of <b>3q</b> SA-omit (2mFo-dFc), in gray, and <b>3q</b> 2mFo-dFc, in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The same order of efficacy was also obtained with compounds <b>3u</b> (IC<sub>50</sub> 21.1 ± 1.7 μM) and <b>3v</b> (IC<sub>50</sub> 6.4 ± 1.3 μM), bearing a 2-(3-methoxy)phenyl fragment combined with a fluoro and a chloro atom, respectively, in the <i>para</i> position of the 6-phenyl ring (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">Tested against the related dehydrogenase isoforms ALDH1A1 and ALDH1A2, none of the synthesized compounds but the unsubstituted <b>3a</b> showed any relevant inhibitory efficacy (<a class="ref internalNav" href="#notes1" aria-label="Figure 2-SI">Figure 2-SI</a>, Supporting Information), thus proving that suitably amending the aromatic tails of the imidazo[1,2-<i>a</i>]pyridine core a complete selectivity between similar enzymes can be achieved. However, it is worth mentioning derivative <b>3m</b>, 2-(4-bromophenyl)-6-(4-chlorophenyl)imidazo[1,2-<i>a</i>]pyridine, whose double substitutions carried out with a 4-bromo and a 4-chloro on the 2- and 6-phenyl ring, respectively, turned the activity profile of the imidazopyridine class upside down by proving to be fully selective for the ALDH1A1 subtype (IC<sub>50</sub> 17.3 ± 1.6 μM).</div><div class="NLM_p last">To verify that the 2,6-diphenyl motif is the structural geometry that best comply with the structural requirements of the ALDH1A3 binding site, the series of <b>7a–i</b> analogues was synthesized and tested as well. Shifting the phenyl ring to position 7 of the imidazopyridine nucleus modified significantly the inhibitory activity of the resulting compounds, which was substantially downsized against the ALDH1A3 isoform and even nullified in the case of ALDH1A1 and ALDH1A2 enzymes (<a class="ref internalNav" href="#notes1" aria-label="Figure 3-SI">Figure 3-SI</a>, Supporting Information).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Antiproliferative Activity</h3><div class="NLM_p">Representative examples of the imidazopyridine derivatives, <b>3a</b>–<b>c</b>,<b>e</b>,<b>f</b>,<b>u</b>,<b>v</b>, chosen among those possessing the best profile of inhibitory activity and selectivity, were tested for their antiproliferative efficacy against three different patient-derived glioma sphere samples, both proneural (PN-157) and mesenchymal (MES-267 and MES-374) ones, already characterized for their ALDH1 expression patterns.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> The 72-h exposure to the test compounds significantly inhibited the growth of glioma spheres. All the tested imidazopyridines displayed IC<sub>50</sub> values in the nanomolar/subnanomolar range against the different cell subtypes, showing a more marked activity against the MES-374 line (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The 6-(4-fluoro)phenyl derivative <b>3b</b>, combining the best functional profile in terms of activity and selectivity against the ALDH1 proteins (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), confirmed its efficacy also in the cell-based test, showing IC<sub>50</sub> values of 25.2 nM, 63.4 nM, and 2.58 pM against the PN-157, MES-267, and MES-374 cell lines, respectively.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antiproliferative Activity of Selected Imidazo[1,2-<i>a</i>]pyridine Derivatives <b>3a–c</b>,<b>e,f,u,v</b> against 157-, 267-, and 374-GSC Lines</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="4" align="center">GSC lines</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">N</th><th class="colsep0 rowsep0" align="center" char=".">157 (IC<sub>50</sub>, nM<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a>)</th><th class="colsep0 rowsep0" align="center" char=".">267 (IC<sub>50</sub>, nM<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a>)</th><th class="colsep0 rowsep0" align="center">374 (IC<sub>50</sub>, nM<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="char" char=".">151.0</td><td class="colsep0 rowsep0" align="char" char=".">46.9</td><td class="colsep0 rowsep0" align="left">2.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="char" char=".">25.2</td><td class="colsep0 rowsep0" align="char" char=".">63.4</td><td class="colsep0 rowsep0" align="left">0.00258</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="char" char=".">53.1</td><td class="colsep0 rowsep0" align="char" char=".">21.5</td><td class="colsep0 rowsep0" align="left">0.0196</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3e</b></td><td class="colsep0 rowsep0" align="char" char=".">2660</td><td class="colsep0 rowsep0" align="char" char=".">660</td><td class="colsep0 rowsep0" align="left">n.t.<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3f</b></td><td class="colsep0 rowsep0" align="char" char=".">822.0</td><td class="colsep0 rowsep0" align="char" char=".">823.0</td><td class="colsep0 rowsep0" align="left">n.t<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3u</b></td><td class="colsep0 rowsep0" align="char" char=".">1510</td><td class="colsep0 rowsep0" align="char" char=".">350</td><td class="colsep0 rowsep0" align="left">n.t.<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3v</b></td><td class="colsep0 rowsep0" align="char" char=".">29.6</td><td class="colsep0 rowsep0" align="char" char=".">71.2</td><td class="colsep0 rowsep0" align="left">13.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values represent the concentration of drug that decreased cell count by 50%. Standard errors of the mean (SEMs) ≤ 10%.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">n.t.: not tested.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Concluding Remarks</h3><div class="NLM_p last">In this work, we described a novel series of 2,6-diphenyl-imidazo[1,2-<i>a</i>]pyridine derivatives as aldehyde dehydrogenase inhibitors, proving their significance not only as pharmacological tools but also as promising functional leads. Moving from the previously described hit GA11<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and taking advantage of both crystallographic studies and molecular modeling investigations, we succeeded in customizing the selectivity profile of the compound by varying its substitution pattern on both the pendant phenyl rings. A 2-(4-bromophenyl)-6-(4-chlorophenyl) decoration, as in derivative <b>3m</b>, gave a compound fully selective for the ALDH1A1 isoenzyme, while most of the other explored substituents led to compounds able to inhibit the key ALDH1A3 subtype, accountable for GBM stem cells vitality being involved in the differentiation of progenitor cells into mature entities. In this respect, derivative <b>3b</b> turned out to be the most promising one. Besides showing the most efficient combination of inhibitory activity against the target 1A3 isoenzyme and selectivity toward the parent 1A1 and 1A2, <b>3b</b> displayed a relevant antiproliferative efficacy when tested against well-characterized patient-derived glioma sphere samples, showing IC<sub>50</sub> values in the nanomolar/picomolar range. This compound therefore represents an original example of ALDH inhibitor whose functional profile may provide, in principle, a novel and viable chance to treat GBM by targeting stem-like cells. Clearly, thorough investigations in animal models of GBM are now necessary to prove its robustness as a novel prototype of drug candidate.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23108" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23108" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div class="NLM_p last">Melting points were determined using a Reichert Köfler hot-stage apparatus and are uncorrected. Routine <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded in DMSO-d<sub>6</sub> solution on a Bruker 400 spectrometer operating at 400 MHz. Evaporation was performed in vacuo (rotary evaporator). Analytical TLCs were carried out on Merck 0.2 mm precoated silica gel aluminum sheets (60 F-254). Purity of the target inhibitors was determined by HPLC analysis, using a Merck Hitachi D-7000 liquid chromatograph (PDA, 250–500 nm) and a Luna C18 column (250 mm × 4.6 mm, 5 μm, Phenomenex), with a gradient of 30% water and 70% acetonitrile and a flow rate of 1.0 mL/min. All the compounds showed percent purity values ≥ 95% (<a class="ref internalNav" href="#notes1" aria-label="Figures 4-SI–18-SI">Figures 4-SI–18-SI</a>, Supporting Information). 4-Bromopyridin-2-amine, 5-bromopyridin-2-amine, the suitably substituted 2-bromo-1-phenylethan-1-one, and the appropriate boronic acids, used to obtain the target inhibitors, were from Alfa Aesar, Aldrich, and Fluka. 6-Bromo-2-phenylimidazo[1,2-<i>a</i>]pyridine <b>2a</b>,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> 6-bromo-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine <b>2b</b>,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> 6-bromo-2-(4-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine <b>2c</b>,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> 5-(4-chlorophenyl)pyridin-2-amine <b>4</b><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> were synthesized following reported procedures.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure for Synthesis of 6-Bromo-2-(substituted)phenylimidazo[1,2-<i>a</i>]pyridines, <b>2d,e</b></h3><div class="NLM_p last">A solution of 5-bromopyridin-2-amine <b>1</b> (1.00 mmol) in ethanol was added with the suitably substituted 2-bromo-1-phenylethan-1-one (1.00 mmol) and potassium carbonate (138 mg, 1.00 mmol), and the resulting mixture was refluxed under stirring until the disappearance of the starting material (TLC analysis). After cooling, the crude obtained was evaporated to dryness; then water was added to the residue, and the desired heterocycle, <b>2d,e</b>, separated as a white solid, was collected by filtration, purified through recrystallization from the suitable solvent and characterized with physio-chemical and spectroscopic data.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 6-Bromo-2-(4-nitrophenyl)imidazo[1,2-<i>a</i>]pyridine, <b>2d</b></h3><div class="NLM_p last">White solid. Mp 258–260 °C. Cryst. solvent: Acetonitrile. Yield: 55%. <sup>1</sup>H NMR (δ, ppm): 8.953 (d, 1H, <i>J</i> = 1.20 Hz), 8.603 (s, 1H), 8.321 (d, 2H, <i>J</i> = 8.88 Hz), 8.242 (d, 2H, <i>J</i> = 8.88 Hz), 7.638 (d, 1H, <i>J</i> = 9.60 Hz), 7.445 (dd, 1H, <i>J</i> = 9.60 Hz, <i>J</i> = 1.71 Hz).</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 6-Bromo-2-(3-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>2e</b></h3><div class="NLM_p last">White solid. Mp 232–234 °C. Cryst. solvent: EtOH. Yield: 76%. <sup>1</sup>H NMR (δ, ppm): 9.189 (s, 1H), 8.678 (s, 1H), 7.928 (d, 1H, <i>J</i> = 9.28 Hz), 7.853 (d, 1H, <i>J</i> = 9.44 Hz), 7.557 (s, 1H), 7.550 (d, 1H, <i>J</i> = 7.95 Hz), 7.499 (t, 1H, <i>J</i> = 8.09 Hz), 7.091 (d, 1H, <i>J</i> = 7.92 Hz), 3.867 (s, 3H).</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Procedure for Synthesis of 2,6-(Substituted)diphenylimidazo[1,2-<i>a</i>]pyridines, <b>3a–k,n–v</b></h3><div class="NLM_p last">A solution of the appropriate imidazo[1,2-<i>a</i>]pyridine derivative <b>2a–e</b> (1.00 mmol), Pd(OAc)<sub>2</sub> (0.10 mmol), and PPh<sub>3</sub> (0.20 mmol) in toluene was added with the suitable phenyl boronic acid (1.50 mmol), dissolved in ethanol, and 2 mL of Na<sub>2</sub>CO<sub>3</sub> 2M. The resulting mixture was refluxed under stirring until the disappearance of the starting material (TLC analysis). After cooling, the crude obtained was evaporated to dryness, and the residue was purified by column chromatography (silica gel, ethyl acetate/petroleum ether). The pure product was then recrystallized from the suitable solvent and characterized by physio-chemical and spectroscopic data.</div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 2,6-Diphenylimidazo[1,2-<i>a</i>]pyridine, <b>3a</b></h3><div class="NLM_p last">White solid. Mp 179–181 °C. Cryst. solvent: MeOH. Yield: 78%. <sup>1</sup>H NMR (δ, ppm): 8.899 (s, 1H), 8.420 (s, 1H), 7.999 (d, 2H, <i>J</i> = 7.10 Hz), 7.755 (d, 2H, <i>J</i> = 7.20 Hz), 7.679 (d, 1H, <i>J</i> = 9.40 Hz), 7.615 (dd, 1H, <i>J</i> = 9.40 Hz, <i>J</i> = 1.80 Hz), 7.524 (t, 2H, <i>J</i> = 7.40 Hz), 7.466 (t, 3H, <i>J</i> = 7.44 Hz), 7.441 (t, 1H, <i>J</i> = 7.32 Hz), 7.343 (t, 1H, <i>J</i> = 7.30 Hz). <sup>13</sup>C NMR (δ, ppm): 145.45, 144.66, 137.14, 134.32, 129.57, 129.20, 128.24, 128.18, 126.99, 126.07, 125.70, 125.41, 124.47, 117.12, 110.02.</div></div><div id="sec3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 6-(4-Fluorophenyl)-2-phenylimidazo[1,2-<i>a</i>]pyridine, <b>3b</b></h3><div class="NLM_p last">White solid. Mp 173–175 °C. Cryst. solvent: AcOEt. Yield: 58%. <sup>1</sup>H NMR (δ, ppm): 8.879 (s, 1H), 8.405 (s, 1H), 8.002 (d, 2H, <i>J</i> = 7.70 Hz), 7.786 (dd, 2H, <i>J</i> = 7.10 Hz, <i>J</i> = 5.36 Hz), 7.679 (d, 1H, <i>J</i> = 9.16 Hz), 7.595 (dd, 1H, <i>J</i> = 9.38 Hz, <i>J</i> = 1.86 Hz), 7.463 (t, 2H, <i>J</i> = 6.78 Hz), 7.358 (t, 3H, <i>J</i> = 8.33 Hz). <sup>13</sup>C NMR (δ, ppm): 163.64, 161.21, 145.40, 144.55, 136.92, 134.17, 133.60, 133.57, 129.22, 129.09, 129.01, 128.31, 126.05, 125.46, 124.83, 124.45, 117.06, 116.51, 116.29, 114.84, 110.05.</div></div><div id="sec3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 6-(4-Chlorophenyl)-2-phenylimidazo[1,2-<i>a</i>]pyridine, <b>3c</b></h3><div class="NLM_p last">White solid. Mp 192–194 °C. Cryst. solvent: MeOH. Yield: 56%. <sup>1</sup>H NMR (δ, ppm): 8.934 (s, 1H), 8.413 (s, 1H), 8.004 (d, 2H, <i>J</i> = 7.08 Hz), 7.779 (d, 2H, <i>J</i> = 8.60 Hz), 7.690 (d, 1H, <i>J</i> = 9.36 Hz), 7.614 (dd, 1H, <i>J</i> = 9.44 Hz, <i>J</i> = 1.88 Hz), 7.582 (d, 2H, <i>J</i> = 8.61 Hz), 7.466 (t, 2H, <i>J</i> = 7.62 Hz), 7.345 (t, 1H, <i>J</i> = 7.37 Hz). <sup>13</sup>C NMR (δ, ppm): 145.54, 144.62, 136.03, 134.23, 129.52, 129.21, 128.74, 128.29, 126.08, 125.13, 124.73, 124.44, 117.19, 110.08.</div></div><div id="sec3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 6-(4-(Methylthio)phenyl)-2-phenylimidazo[1,2-<i>a</i>]pyridine, <b>3d</b></h3><div class="NLM_p last">White solid. Mp 158–160 °C. Cryst. solvent: Acetonitrile. Yield: 47%. <sup>1</sup>H NMR (δ, ppm): 8.873 (s, 1H), 8.394 (s, 1H), 7.990 (d, 2H, <i>J</i> = 7.32 Hz), 7.678 (d, 2H, <i>J</i> = 8.30 Hz), 7.590 (d, 2H, <i>J</i> = 7.60 Hz), 7.493–7.329 (m, 5H), 3. 603 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 145.39, 144.58, 138.37, 134.30, 133.56, 129.20, 128.24, 127.37, 126.95, 126.05, 125.18, 125.11, 124.09, 117.12, 109.99, 15.13.</div></div><div id="sec3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 6-(4-Methoxyphenyl)-2-phenylimidazo[1,2-<i>a</i>]pyridine, <b>3e</b></h3><div class="NLM_p last">White solid. Mp 173–175 °C. Cryst. solvent: EtOH. Yield: 35%. <sup>1</sup>H NMR (δ, ppm): 8.765 (s, 1H), 8.369 (s, 1H), 7.944 (d, 2H, <i>J</i> = 7.81 Hz), 7.631 (d, 4H, <i>J</i> = 8.78 Hz), 7.444 (t, 2H, <i>J</i> = 6.84 Hz), 7.327 (t, 1H, <i>J</i> = 6.35 Hz), 7.047 (d, 2H, <i>J</i> = 7.32 Hz), 3. 809 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 159.56, 145.14, 144.42, 134.27, 129.37, 129.20, 128.15, 126.03, 125.52, 123.63, 116.96, 115.03, 109.92, 55.71.</div></div><div id="sec3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Methyl 4-(2-phenylimidazo[1,2-<i>a</i>]pyridin-6-yl)benzoate, <b>3f</b></h3><div class="NLM_p last">White solid. Mp 165–168 °C. Cryst. solvent: EtOH. Yield: 53%. <sup>1</sup>H NMR (δ, ppm): 9.014 (s, 1H), 8.413 (s, 1H), 8.063 (d, 2H, <i>J</i> = 8.30 Hz), 7.994 (d, 2H, <i>J</i> = 8.31 Hz), 7.889 (d, 2H, <i>J</i> = 8.30 Hz), 7.454 (t, 2H, 6.83 Hz), 7.328 (t, 1H, <i>J</i> = 7.32 Hz), 3.875 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 166.52, 166.40, 145.16, 143.74, 139.29, 130.35, 130.31, 130.27. 130.23, 129.74, 129.57, 128.91, 128.88, 127.81, 127.78, 127.45, 127.43, 61.33, 52.71, 52.62, 14.64.</div></div><div id="sec3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 2-(2-Phenylimidazo[1,2-<i>a</i>]pyridin-6-yl)benzonitrile, <b>3g</b></h3><div class="NLM_p last">White solid. Mp 213–215 °C. Cryst. solvent: EtOH. Yield: 26%. <sup>1</sup>H NMR (δ, ppm): 8.857 (s, 1H), 8.855 (s, 1H), 8.852 (d, 3H, <i>J</i> = 7.76 Hz), 7.879 (t, 1H, <i>J</i> = 7.64 Hz), 7.775–7.740 (m, 2H), 7.655 (t, 1H, <i>J</i> = 7.68 Hz), 7.475 (t, 3H, <i>J</i> = 7.72 Hz), 7.355 (t, 1H, <i>J</i> = 7.37 Hz). <sup>13</sup>C NMR (δ, ppm): 145.85, 144.56, 141.31, 134.36, 134.15, 134.09, 130.81, 129.24, 129.16, 128.43, 126.88, 126.55, 126.16, 123.44, 118.80, 116.94, 111.19, 110.24.</div></div><div id="sec3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 3-(2-Phenylimidazo[1,2-<i>a</i>]pyridin-6-yl)benzonitrile, <b>3h</b></h3><div class="NLM_p last">White solid. Mp 170–171 °C. Cryst. solvent: EtOH. Yield: 30%. <sup>1</sup>H NMR (δ, ppm): 9.043 (s, 1H), 8.415 (s, 1H), 8.272 (s, 1H), 8.110 (d, 1H, <i>J</i> = 8.00 Hz), 8.022 (d, 2H, <i>J</i> = 7.30 Hz), 7.881 (d, 1H, <i>J</i> = 7.80 Hz), 7.737 (d, 2H, <i>J</i> = 7.77 Hz), 7.707 (s, 1H), 7.470 (t, 2H, <i>J</i> = 7.52 Hz), 7.352 (t, 1H, <i>J</i> = 7.28 Hz). <sup>13</sup>C NMR (δ, ppm): 145.71, 144.68, 138.36, 134.12, 131.69, 130.72, 130.54, 129.20, 128.36, 126.14, 125.44, 125.00, 123.67, 119.18, 117.23, 112.69, 110.12.</div></div><div id="sec3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 4-(2-Phenylimidazo[1,2-<i>a</i>]pyridin-6-yl)benzonitrile, <b>3i</b></h3><div class="NLM_p last">White solid. Mp 209–210 °C. Cryst. solvent: Acetonitrile. Yield: 39%. <sup>1</sup>H NMR (δ, ppm): 9.066 (s, 1H), 8.436 (s, 1H), 8.017 (d, 2H, <i>J</i> = 7.08 Hz), 7.979 (d, 4H, <i>J</i> = 10.20 Hz), 7.729 (d, 1H, <i>J</i> = 9.37 Hz), 7.690 (dd, 1H, <i>J</i> = 10.16 Hz, <i>J</i> = 1.60 Hz), 7.472 (t, 2H, <i>J</i> = 7.60 Hz), 7.354 (t, 1H, <i>J</i> = 7.40 Hz). <sup>13</sup>C NMR (δ, ppm): 145.81, 144.75, 141.83, 134.10, 133.46, 129.23, 128.40, 127.76, 125.84, 124.91, 119.26, 117.33, 110.67, 110.25.</div></div><div id="sec3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 6-Phenyl-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine, <b>3j</b></h3><div class="NLM_p last">White solid. Mp 165–167 °C. Cryst. solvent: Acetonitrile. Yield: 36%. <sup>1</sup>H NMR (δ, ppm): 8.879 (s, 1H), 8.359 (s, 1H), 7.887 (d, 2H, <i>J</i> = 8.01 Hz), 7.739 (d, 2H, <i>J</i> = 7.33 Hz), 7.663 (d, 1H, <i>J</i> = 9.29 Hz), 7.603 (dd, 1H, <i>J</i> = 9.36 Hz, <i>J</i> = 1.76 Hz), 7.520 (t, 2H, <i>J</i> = 7.58 Hz), 7.415 (t, 1H, <i>J</i> = 7.37 Hz), 7.272 (d, 2H, <i>J</i> = 8.09 Hz), 2.353 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 159.62, 145.38, 144.49, 137.19, 134.52, 130.44, 129.57, 128.14, 127.79, 127.41, 126.97, 126.82, 125.58, 125.21, 124.34, 116.79, 114.67, 108.97, 55.64.</div></div><div id="sec3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 2-(4-Methoxyphenyl)-6-phenylimidazo[1,2-<i>a</i>]pyridine, <b>3k</b></h3><div class="NLM_p last">White solid. Mp 176–179 °C. Cryst. solvent: EtOH. Yield: 68%. <sup>1</sup>H NMR (δ, ppm): 8.879 (s, 1H), 8.359 (s, 1H), 7.887 (d, 2H, <i>J</i> = 8.01 Hz), 7.739 (d, 2H, <i>J</i> = 7.33 Hz), 7.662 (d, 1H, <i>J</i> = 9.29 Hz), 7.603 (dd, 1H, <i>J</i> = 9.36 Hz, <i>J</i> = 1.76 Hz), 7.521 (t, 2H, <i>J</i> = 7.40 Hz), 7.415 (t, 1H, <i>J</i> = 7.37 Hz), 7.272 (d, 2H, <i>J</i> = 8.09 Hz). <sup>13</sup>C NMR (δ, ppm): 145.57, 144.57, 137.54, 137.17, 131.54, 129.78, 129.56, 128.15, 126.96, 126.00, 125.58, 125.23, 124.38, 117.00, 109.55, 21.34.</div></div><div id="sec3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 6-(4-Chlorophenyl)-2-(4-nitrophenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3n</b></h3><div class="NLM_p last">White solid. Mp 198–200 °C. Cryst. solvent: EtOH. Yield: 28%. <sup>1</sup>H NMR (δ, ppm): 8.957 (s, 1H), 8.644 (s, 1H), 8.324 (d, 2H, <i>J</i> = 8.88 Hz), 8.270 (d, 2H, <i>J</i> = 8.89 Hz), 7.789 (d, 2H, <i>J</i> = 8.52 Hz), 7.736 (d, 1H, <i>J</i> = 9.37 Hz), 7.677 (dd, 1H, <i>J</i> = 9.40 Hz, <i>J</i> = 1.56 Hz), 7.588 (d, 2H, <i>J</i> = 8.48 Hz). <sup>13</sup>C NMR (δ, ppm): 146.92, 144.96, 143.08, 140.72, 135.69, 133.17, 129.48, 128.74, 126.77, 126.03, 124.94, 124.54, 117.47, 112.54.</div></div><div id="sec3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 2,6-Bis(4-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3o</b></h3><div class="NLM_p last">White solid. Mp 202–203 °C. Cryst. solvent: EtOH. Yield: 42%. <sup>1</sup>H NMR (δ, ppm): 8.767 (s, 1H), 8.257 (s, 1H), 7.899 (d, 2H, <i>J</i> = 6.84 Hz), 7.644 (d, 2H, <i>J</i> = 6.84 Hz), 7.566 (d, 2H, <i>J</i> = 6.84 Hz), 7.052 (d, 2H, <i>J</i> = 7.08 Hz), 7.006 (d, 2H, <i>J</i> = 7.08 Hz), 3.794 (s, 6H). <sup>13</sup>C NMR (δ, ppm): 159.54, 145.19, 144.29, 133.76, 132.48, 131.91, 129.28, 128.13, 126.82, 125.24, 123.51, 116.66, 115.03, 114.66, 108.87, 55.72, 55.63.</div></div><div id="sec3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 6-(3,4-Dimethoxyphenyl)-2-(4-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3p</b></h3><div class="NLM_p last">White solid. Mp 245–248 °C. Cryst. solvent: EtOH. Yield: 39%. <sup>1</sup>H NMR (δ, ppm): 8.805 (s, 1H), 8.259 (s, 1H), 7.806 (d, 2H, <i>J</i> = 8.79 Hz), 7.580 (s, 1H) 7.567 (d, 1H, <i>J</i> = 7.37 Hz), 7.20 (d, 2H, <i>J</i> = 8.06 Hz), 7.039 (d, 1H, <i>J</i> = 7.56 Hz), 6.998 (d, 2H, <i>J</i> = 8.55 Hz), 3.865 (s, 3H), 3.796 (s, 6H). <sup>13</sup>C NMR (δ, ppm): 159.52, 149.67, 149.12, 145.37, 144.45, 129.83, 127.34, 126.99, 125.53, 125.26, 123.63, 119.16, 116.64, 114.62, 112.74, 110.74, 108.75, 56.12, 56.06, 55.60.</div></div><div id="sec3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 6-(3,5-Dimethoxyphenyl)-2-(4-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3q</b></h3><div class="NLM_p last">White solid. Mp 213–215 °C. Cryst. solvent: EtOH. Yield: 41%. <sup>1</sup>H NMR (δ, ppm): 9.176 (s, 1H), 8.553 (s, 1H), 8.136 (d, 1H, <i>J</i> = 9.45 Hz), 7.934 (d, 2H, <i>J</i> = 8.84 Hz), 7.903 (d, 1H, <i>J</i> = 9.56 Hz), 7.171 (d, 2H, <i>J</i> = 8.76 Hz), 6.945 (s, 1H), 6.939 (s, 1H), 6.637 (s, 1H), 3.858 (s, 9H). <sup>13</sup>C NMR (δ, ppm): 161.47, 159.57, 145.52, 144.66, 139.28, 127.38, 126.89, 125.38, 125.21, 124.59, 116.65, 114.64, 109.87, 105.11, 100.05, 55.84, 55.62.</div></div><div id="sec3_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 6-(4-Fluorophenyl)-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine, <b>3r</b></h3><div class="NLM_p last">White solid. Mp 156–158 °C. Cryst. solvent: Acetonitrile. Yield: 39%. <sup>1</sup>H NMR (δ, ppm): 8.862 (s, 1H), 8.348 (s, 1H), 7.888 (d, 2H, <i>J</i> = 8.09 Hz), 7.781 (dd, 2H, <i>J</i> = 7.76 Hz, <i>J</i> = 4.36 Hz), 7.656 (d, 1H, <i>J</i> = 9.41 Hz), 7.580 (dd, 1H, <i>J</i> = 9.37 Hz, <i>J</i> = 1.80 Hz), 7.356 (t, 2H, <i>J</i> = 8.87 Hz), 7.270 (d, 2H, <i>J</i> = 7.43 Hz), 2.351 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 163.62, 161.19, 145.60, 144.48, 137.54, 133.67, 133.64, 131.51, 129.76, 129.04, 128.96, 126.00, 125.18, 124.65, 124.36, 116.98, 116.46, 116.25, 109.54, 21.32.</div></div><div id="sec3_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 6-(4-Chlorophenyl)-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine, <b>3s</b></h3><div class="NLM_p last">White solid. Mp 188–190 °C. Cryst. solvent: Acetonitrile. Yield: 36%. <sup>1</sup>H NMR (δ, ppm): 8.914 (s, 1H), 8.353 (s, 1H), 7.889 (d, 2H, <i>J</i> = 8.09 Hz), 7.775 (d, 2H, <i>J</i> = 8.56 Hz), 7.660 (d, 1H, <i>J</i> = 9.32 Hz), 7.597 (dd, 1H, <i>J</i> = 9.12 Hz, <i>J</i> = 1.80 Hz), 7.577 (d, 2H, <i>J</i> = 8.53 Hz), 7.271 (d, 2H, <i>J</i> = 8.01 Hz), 2.352 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 145.67, 144.54, 137.60, 136.06, 132.97, 131.46, 129.78, 129.50, 128.72, 126.02, 124.95, 124.63, 124.32, 117.06, 109.61, 21.34.</div></div><div id="sec3_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 2-(3-Methoxyphenyl)-6-phenylimidazo[1,2-<i>a</i>]pyridine, <b>3t</b></h3><div class="NLM_p last">White solid. Mp 140–141 °C. Cryst. solvent: Acetonitrile. Yield: 39%. <sup>1</sup>H NMR (δ, ppm): 8.877 (s, 1H), 8.433 (s, 1H), 7.741 (d, 2H, <i>J</i> = 7.24 Hz), 7.690 (d, 1H, <i>J</i> = 9.36 Hz), 7.618 (dd, 1H, <i>J</i> = 9.36 Hz, <i>J</i> = 1.76 Hz), 7.574 (s, 1H), 7.560 (d, 1H, <i>J</i> = 7.42 Hz), 7.522 (t, 2H, <i>J</i> = 7.44 Hz), 7.418 (t, 1H, <i>J</i> = 7.37 Hz), 7.370 (t, 1H, <i>J</i> = 8.20 Hz), 6.912 (dd, 1H, <i>J</i> = 7.91 Hz, <i>J</i> = 1.76 Hz), 3.845 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 160.13, 145.38, 144.52, 136.02, 135.64, 133.02, 130.29, 129.52, 128.77, 125.18, 124.70, 124.48, 118.49, 117.20, 114.15, 111.15, 110.38, 55.57.</div></div><div id="sec3_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 6-(4-Fluorophenyl)-2-(3-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3u</b></h3><div class="NLM_p last">White solid. Mp 147–148 °C. Cryst. solvent: Acetonitrile. Yield: 27%. <sup>1</sup>H NMR (δ, ppm): 8.865 (s, 1H), 8.424 (s, 1H), 7.785 (dd, 2H, <i>J</i> = 9.00 Hz, <i>J</i> = 5.78 Hz), 7.686 (d, 1H, <i>J</i> = 9.24 Hz), 7.587–7.564 (m, 3H), 7.391–7.337 (m, 3H), 6.916 (d, 1H, <i>J</i> = 9.40 Hz), 3.845 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 163.65, 161.21, 160.13, 145.30, 144.45, 135.69, 133.63, 133.60, 130.27, 129.10, 129.01, 125.41, 124.80, 124.43, 118.47, 117.11, 116.49, 116.28, 114.10, 111.14, 110.31, 55.55.</div></div><div id="sec3_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 6-(4-Chlorophenyl)-2-(3-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3v</b></h3><div class="NLM_p last">White solid. Mp 161–163 °C. Cryst. solvent: MeOH. Yield: 24%. <sup>1</sup>H NMR (δ, ppm): 8.911 (s, 1H), 8.426 (s, 1H), 7.777 (d, 2H, <i>J</i> = 6.64 Hz), 7.693 (d, 1H, <i>J</i> = 9.36 Hz), 7.610 (dd, 1H, <i>J</i> = 9.44 Hz, <i>J</i> = 1.76 Hz), 7.371 (t, 1H, <i>J</i> = 8.10 Hz), 7.915 (dd, 1H, <i>J</i> = 6.00 Hz, <i>J</i> = 1.40 Hz), 3.844 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 160.14, 145.39, 144.52, 136.03, 135.65, 133.02, 130.29, 129.52, 128.76, 125.18, 124.70, 124.49, 118.50, 117.20, 114.15, 111.17, 110.37, 55.58.</div></div><div id="sec3_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Synthesis of 2,6-bis(4-Chlorophenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3l</b>, and 2-(4-Bromophenyl)-6-(4-chlorophenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3m</b></h3><div class="NLM_p last">A solution of 5-(4-chlorophenyl)pyridin-2-amine<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a><b>4</b> (1.00 mmol) in ethanol was added with the suitably substituted 2-bromo-1-phenylethan-1-one (1.00 mmol) and potassium carbonate (138 mg, 1.00 mmol), and the resulting mixture was refluxed under stirring until the disappearance of the starting material (TLC analysis). After cooling, the crude obtained was evaporated to dryness; then water was added to the residue, and the desired heterocycle <b>3l,m</b>, separated as a white solid, was collected by filtration, purified through recrystallization from the suitable solvent, and characterized with physio-chemical and spectroscopic data.</div></div><div id="sec3_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 2,6-bis(4-Chlorophenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3l</b></h3><div class="NLM_p last">White solid. Mp 278–280 °C. Cryst. solvent: EtOH. Yield: 81%. <sup>1</sup>H NMR (δ, ppm): 9.192 (s, 1H), 8.692 (s, 1H), 8.102 (s, 1H), 8.031 (d, 2H, <i>J</i> = 8.60 Hz), 7.945 (d, 1H, <i>J</i> = 8.90 Hz), 7.842 (d, 2H, <i>J</i> = 8.50 Hz), 7.686–7.638 (m, 4H). <sup>13</sup>C NMR (δ, ppm): 140.58, 136.55, 135.27, 134.26, 134.20, 132.39, 129.97, 129.81, 129.29, 128.61, 128.47, 126.70, 126.58, 113.28, 112.19.</div></div><div id="sec3_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 2-(4-Bromophenyl)-6-(4-chlorophenyl)imidazo[1,2-<i>a</i>]pyridine, <b>3m</b></h3><div class="NLM_p last">White solid. Mp 298–300 °C. Cryst. solvent: MeOH. Yield: 48%. <sup>1</sup>H NMR (δ, ppm): 9.183 (s, 1H), 8.693 (s, 1H), 8.096 (d, 1H, <i>J</i> = 8.16 Hz), 7.976–7.926 (m, 3H), 7.841 (d, 2H, <i>J</i> = 8.60 Hz), 7.807 (d, 2H, <i>J</i> = 8.50 Hz), 7.640 (d, 2H, <i>J</i> = 8.60 Hz). <sup>13</sup>C NMR (δ, ppm): 140.50, 136.45, 134.29, 134.14, 132.88, 132.52, 129.80, 129.27, 128.67, 126.91, 126.58, 124.05, 113.18, 112.21.</div></div><div id="sec3_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> General Procedure for Synthesis of 7-Bromo-2-(substituted)phenylimidazo[1,2-<i>a</i>]pyridines, <b>6a–c</b></h3><div class="NLM_p last">A solution of 4-bromopyridin-2-amine <b>5</b> (1.00 mmol) in ethanol was added with the suitably substituted 2-bromo-1-phenylethan-1-one (1.00 mmol) and potassium carbonate (138 mg, 1.00 mmol), and the resulting mixture was refluxed under stirring until the disappearance of the starting material (TLC analysis). After cooling, the crude obtained was evaporated to dryness; then water was added to the residue, and the desired heterocycle, <b>6a–c</b>, separated as a white solid, was collected by filtration, purified through recrystallization from the suitable solvent, and characterized with physio-chemical and spectroscopic data.</div></div><div id="sec3_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 7-Bromo-2-phenylimidazo[1,2-<i>a</i>]pyridine, <b>6a</b></h3><div class="NLM_p last">White solid. Mp 290 °C. Cryst. solvent: MeOH. Yield: 81%. <sup>1</sup>H NMR (δ, ppm): 8.732 (d, 1H, <i>J</i> = 7.25 Hz), 8.695 (s, 1H), 8.151 (s, 1H), 7.958 (d, 2H, <i>J</i> = 7.05 Hz), 7.565 (d, 1H, <i>J</i> = 7.22 Hz), 7.514 (t, 3H, <i>J</i> = 7.12 Hz).</div></div><div id="sec3_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 7-Bromo-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine, <b>6b</b></h3><div class="NLM_p last">White solid. Mp 280 °C. Cryst. solvent: EtOH. Yield: 83%. <sup>1</sup>H NMR (δ, ppm): 8.747 (d, 1H, <i>J</i> = 7.12 Hz), 8.675 (s, 1H), 8.156 (s, 1H), 7.848 (d, 2H, <i>J</i> = 8.16 Hz), 7.568 (d, 1H, <i>J</i> = 7.07 Hz), 7.399 (d, 2H, <i>J</i> = 8.00 Hz), 2.390 (s, 3H).</div></div><div id="sec3_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 7-Bromo-2-(3-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>6c</b></h3><div class="NLM_p last">White solid. Mp 261–263 °C. Cryst. solvent: MeOH. Yield: 70%. <sup>1</sup>H NMR (δ, ppm): 8.605 (d, 1H, <i>J</i> = 7.15 Hz), 8.581 (s, 1H), 8.049 (s, 1H), 7.538–7.521 (m, 2H), 7.480–7.429 (m, 2H), 7.027 (d, 1H, <i>J</i> = 7.09 Hz), 3.854 (s, 3H).</div></div><div id="sec3_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> General Procedure for Synthesis of 2,7-(Substituted)diphenylimidazo[1,2-<i>a</i>]pyridines, <b>7a–i</b></h3><div class="NLM_p last">A solution of the appropriate imidazo[1,2-<i>a</i>]pyridine derivative <b>6a–c</b> (1.00 mmol), Pd(OAc)<sub>2</sub> (0.10 mmol), and PPh<sub>3</sub> (0.20 mmol) in toluene was added with the suitable phenyl boronic acid (1.50 mmol), dissolved in ethanol, and 2 mL of Na<sub>2</sub>CO<sub>3</sub> 2M. The resulting mixture was refluxed under stirring until the disappearance of the starting material (TLC analysis). After cooling, the crude obtained was evaporated to dryness, and the residue was purified by column chromatography (silica gel, ethyl acetate/petroleum ether). The pure product was then recrystallized from the suitable solvent and characterized by physio-chemical and spectroscopic data.</div></div><div id="sec3_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 2,7-Diphenylimidazo[1,2-<i>a</i>]pyridine, <b>7a</b></h3><div class="NLM_p last">White solid. Mp 196–198 °C. Cryst. solvent: MeOH. Yield: 45%. <sup>1</sup>H NMR (δ, ppm): 8.611 (d, 1H, <i>J</i> = 7.10 Hz), 8.435 (s, 1H), 8.004 (d, 2H, <i>J</i> = 7.04 Hz), 7.924 (s, 1H), 7.851 (d, 2H, <i>J</i> = 7.20 Hz), 7.522 (t, 2H, <i>J</i> = 7.54 Hz,), 7.465 (t, 2H, <i>J</i> = 7.40 Hz), 7.431 (t, 1H, <i>J</i> = 7.58 Hz), 7.360 (t, 1H, <i>J</i> = 7.33 Hz,), 7.310 (dd, 1H, <i>J</i> = 7.12 Hz, <i>J</i> = 1.84 Hz,). <sup>13</sup>C NMR (δ, ppm): 145.45, 144.66, 137.14, 134.32, 129.57, 129.20, 128.24, 128.18, 126.99, 126.07, 125.70, 125.41, 124.47, 117.12, 110.02.</div></div><div id="sec3_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 7-(4-Fluorophenyl)-2-phenylimidazo[1,2-<i>a</i>]pyridine, <b>7b</b></h3><div class="NLM_p last">White solid. Mp 222–223 °C. Cryst. solvent: MeOH. Yield: 42%. <sup>1</sup>H NMR (δ, ppm): 8.606 (d, 1H, <i>J</i> = 7.24 Hz), 8.427 (s, 1H), 7.999 (d, 2H, <i>J</i> = 7.36 Hz,), 7.910 (s, 1H), 7.901 (t, 2H, <i>J</i> = 5.40 Hz), 7.464 (t, 2H, <i>J</i> = 7.66 Hz,), 7.360 (d, 2H, <i>J</i> = 7.56 Hz), 7.342 (t, 1H, <i>J</i> = 7.33 Hz), 7.294 (dd, 1H, <i>J</i> = 7.14 Hz, <i>J</i> = 1.78 Hz). <sup>13</sup>C NMR (δ, ppm): 163.65, 161.21, 145.45, 144.54, 134.26, 133.64, 133.61, 129.20, 129.09, 129.01, 128.26, 126.07, 125.39, 124.77, 124.48, 117.11, 116.50, 116.28, 110.01.</div></div><div id="sec3_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 7-(4-Chlorophenyl)-2-phenylimidazo[1,2-<i>a</i>]pyridine, <b>7c</b></h3><div class="NLM_p last">White solid. Mp 224–226 °C. Cryst. solvent: Acetonitrile. Yield: 35%. <sup>1</sup>H NMR (δ, ppm): 8.617 (d, 1H, <i>J</i> = 7.14 Hz,), 8.442 (s, 1H), 8.003 (d, 2H, <i>J</i> = 7.08 Hz), 7.959 (s, 1H), 7.888 (d, 2H, <i>J</i> = 6.65 Hz), 7.565 (d, 2H, <i>J</i> = 6.65 Hz), 7.466 (t, 2H, <i>J</i> = 7.60 Hz), 7.344 (t, 1H, <i>J</i> = 7.37 Hz), 7.307 (dd, 1H, <i>J</i> = 7.12 Hz, <i>J</i> = 1.88 Hz). <sup>13</sup>C NMR (δ, ppm): 145.84, 145.64, 137.22, 135.49, 134.29, 133.48, 129.49, 129.22, 128.78, 128.29, 127.48, 126.05, 113.59, 111.75, 109.57.</div></div><div id="sec3_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 7-Phenyl-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine, <b>7d</b></h3><div class="NLM_p last">White solid. Mp 251–252 °C. Cryst. solvent: EtOH. Yield: 30%. <sup>1</sup>H NMR (δ, ppm): 8.590 (d, 1H, <i>J</i> = 7.12 Hz), 8.374 (s, 1H), 7.886 (d, 3H, <i>J</i> = 8.09 Hz), 7.844 (d, 2H, <i>J</i> = 6.09 Hz), 7.518 (t, 2H, <i>J</i> = 7.55 Hz), 7.426 (t, 1H, <i>J</i> = 7.34 Hz), 7.293 (dd, 1H, <i>J</i> = 7.20 Hz, <i>J</i> = 1.88 Hz), 7.269 (d, 2H, <i>J</i> = 7.89 Hz), 2.351 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 145.83, 145.70, 138.41, 137.53, 136.71, 131.60, 129.78, 129.56, 128.66, 127.29, 126.96, 125.98, 113.25, 111.89, 108.96, 21.35.</div></div><div id="sec3_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 7-(4-Fluorophenyl)-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine, <b>7e</b></h3><div class="NLM_p last">White solid. Mp 266–268 °C. Cryst. solvent: Acetonitrile. Yield: 21%. <sup>1</sup>H NMR (δ, ppm): 8.588 (d, 1H, <i>J</i> = 7.12 Hz), 8.371 (s, 1H), 7.915–7.873 (m, 5H), 7.346 (t, 2H, <i>J</i> = 8.85 Hz), 7.291–7.259 (m, 3H), 2,350 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 146.04, 145.42, 137.54, 135.68, 131.58, 129.78, 129.13, 129.05, 127.31, 125.98, 116.48, 116.26, 113.25, 111.81, 108.95, 21.34.</div></div><div id="sec3_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 7-(4-Chlorophenyl)-2-(p-tolyl)imidazo[1,2-<i>a</i>]pyridine, <b>7f</b></h3><div class="NLM_p last">White solid. Mp 270–275 °C. Cryst. solvent: Acetonitrile. Yield: 27%. <sup>1</sup>H NMR (δ, ppm): 8.597 (d, 1H, <i>J</i> = 7.08 Hz), 8.384 (s, 1H), 7.935 (s, 1H), 7.882 (dd, 4H, <i>J</i> = 8.39 Hz, <i>J</i> = 1.90 Hz), 7.566 (d, 2H, <i>J</i> = 6.66 Hz), 7.299 (dd, 1H, <i>J</i> = 7.20 Hz, <i>J</i> = 1.84 Hz), 7.270 (d, 2H, <i>J</i> = 7.81 Hz), 2.350 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 137.69, 135.41, 133.50, 129.80, 129.49, 128.77, 127.95, 127.46, 125.99, 123.80, 117.94, 116.96, 113.57, 111.67, 109.11, 21.35.</div></div><div id="sec3_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 2-(3-Methoxyphenyl)-7-phenylimidazo[1,2-<i>a</i>]pyridine, <b>7g</b></h3><div class="NLM_p last">White solid. Mp 176–178 °C. Cryst. solvent: MeOH. Yield: 57%. <sup>1</sup>H NMR (δ, ppm): 8.597 (d, 1H, <i>J</i> = 7.10 Hz), 8.450 (s, 1H), 7.930 (s, 1H), 7.849 (d, 2H, <i>J</i> = 7.82 Hz), 7.568 (s, 1H), 7.538 (d, 1H, <i>J</i> = 7.20 Hz), 7.521 (t, 2H, <i>J</i> = 7.54 Hz), 7.429 (t, 1H, <i>J</i> = 7.31 Hz), 7.369 (t, 1H, <i>J</i> = 8.16 Hz), 7.309 (dd, 1H, <i>J</i> = 7.12 Hz, <i>J</i> = 1.88 Hz), 6.911 (d, 1H, <i>J</i> = 8.22 Hz), 3.845 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 160.13, 145.70, 145.55, 138.37, 136.89, 135.79, 130.28, 129.56, 128.69, 127.36, 126.98, 118.44, 114.11, 113.38, 112.04, 111.08, 109.71, 55.55.</div></div><div id="sec3_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 7-(4-Fluorophenyl)-2-(3-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>7h</b></h3><div class="NLM_p last">White solid. Mp 180–182 °C. Cryst. solvent: Acetonitrile. Yield: 60%. <sup>1</sup>H NMR (δ, ppm): 8.595 (d, 1H, <i>J</i> = 7.20 Hz), 8.446 (s, 1H), 7.930 (s, 1H), 7.916 (d, 1H, <i>J</i> = 8.96 Hz), 7.894 (d, 1H, <i>J</i> = 8.72 Hz), 7.575 (s, 1H), 7.565 (d, 1H, <i>J</i> = 7.84 Hz), 7.380 (d, 2H, <i>J</i> = 8.08 Hz), 7.359 (t, 1H, <i>J</i> = 8.32 Hz), 7.295 (dd, 1H, <i>J</i> = 7.12 Hz, <i>J</i> = 1.80 Hz), 6.911 (d, 1H, <i>J</i> = 8.16 Hz), 3.844 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 163.65, 161.22, 160.13, 145.30, 144.45, 135.69, 133.63, 133.60, 130.28, 129.10, 129.02, 125.42, 124.80, 124.44, 118.47, 117.12, 116.50, 116.28, 114.11, 111.13, 110.31, 55.55.</div></div><div id="sec3_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 7-(4-Chlorophenyl)-2-(3-methoxyphenyl)imidazo[1,2-<i>a</i>]pyridine, <b>7i</b></h3><div class="NLM_p last">White solid. Mp 182–183 °C. Cryst. solvent: MeOH. Yield: 60%. <sup>1</sup>H NMR (δ, ppm): 8.605 (d, 1H, <i>J</i> = 7.16 Hz), 8.460 (s, 1H), 7.969 (s, 1H), 7.887 (d, 2H, <i>J</i> = 8.61 Hz), 7.569 (d, 4H, <i>J</i> = 7.89 Hz,), 7.371 (t, 1H, <i>J</i> = 8.08 Hz), 7.307 (dd, 1H, <i>J</i> = 7.12 Hz, <i>J</i> = 1.80 Hz), 6.914 (d, 1H, <i>J</i> = 8.10 Hz), 3.844 (s, 3H). <sup>13</sup>C NMR (δ, ppm): 160.12, 145.69, 145.55, 137.20, 135.71, 135.50, 133.49, 130.29, 129.48, 128.77, 127.45, 118.44, 114,15, 113.61, 111.78, 111.10, 109.86, 55.55.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i59">Biology: Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37313" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37313" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Expression and Purification of Human ALDHs</h3><div class="NLM_p last">To obtain a large amount of pure ALDH1A1, ALDH1A2, and ALDH1A3 isoenzymes, a common experimental protocol has been optimized for the three proteins. The full-length ALDH1A1, ALDH1A2, and ALDH1A3 expression vectors were transformed in <i>E. coli</i> strain BL21(DE3) (Novagen) spread onto 2xTY agar plates with 50 μg/mL ampicillin for ALDH1A1 and ALDH1A3, and 50 μg/mL kanamicine for ALDH1A2, for overnight growth, at 37 °C. The next day, colonies were scraped and inoculated in 1 L of 2xTY medium supplemented with 50 μg/mL ampicillin for ALDH1A1 and ALDH1A3, and 50 μg/mL kanamicine for ALDH1A2. For the three enzymes, once OD<sub>600</sub> of 0.6–0.8 was achieved, the temperature was shifted to 20 °C to induce the recombinant protein production. Induced cells, collected by centrifugation, were resuspended in 1/25 original volume of lysis buffer (50 mM Na<sub>2</sub>HPO<sub>4</sub>, 300 mM NaCl, 1 mM β-mercaptoethanol, 20 mM imidazole, pH 7.5) and 1 μL per 80 mL of lysis buffer of benzonase nuclease (250 U/ μL). Plasma membranes of BL21(DE3) cells were fragmented by applying high pressure through the use of a French Press, three times at 1.5 KBar. Protease inhibitor cocktail from SIGMA (100 μL per 40 mL lysis buffer) was added to the crude extract that was clarified by 40 min centrifugation at 20 000 rpm. The supernatant was purified by a His-tag affinity chromatography followed by size-exclusion chromatography, using an AKTA FPLC system, at 4 °C. To evaluate the purity and homogeneity of the protein, after each purification step, eluted fractions were analyzed by SDS-PAGE and the protein quantification was always determined by Bradford protein assay. In the first purification step, the soluble fraction of recombinant human 6xHis-ALDH1A* was purified with a Qiagen Ni-NTA Superflow 5 mL cartridge. The supernatant was loaded on the Ni-NTA column, previously equilibrated with 10 column volumes of lysis buffer. The Ni-NTA cartridge was washed with 50 mM Na<sub>2</sub>HPO<sub>4</sub>, 300 mM NaCl, 1 mM β-mercaptoethanol, 50 mM imidazole, pH 7.5, until the absorbance at 280 nm return to the baseline (15 column volumes). The recombinant protein was eluted with 50 mM Na<sub>2</sub>HPO<sub>4</sub>, 300 mM NaCl, 1 mM β-mercaptoethanol, 250 mM imidazole, pH 8, by applying a linear gradient in 10 column volumes. Eluted fractions were pooled and concentrated to 5 mL with Merck Millipore Amicon Ultra-15 30 kDa and loaded on a HiLoad 16/600 Superdex 200 pg column on AKTA FPLC system. Elution buffer contained 20 mM TrisHCl pH 8.0, 150 mM KCl, 1 mM β-mercaptoethanol, and a flow rate of 1 mL/min was applied. This procedure allowed to obtain about 20 mg of pure and active ALDH1A3, ALDH1A2, and ALDH1A1 used for both crystallization trials and kinetic analyses.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> High-Throughput Screening to Determine Inhibitory Efficacy of Imidazo[1,2-<i>a</i>]pyridine Derivatives</h3><div class="NLM_p last">A high throughput screening campaign was performed for enzymes ALDH1A3, ALDH1A1, and ALDH1A2 using the previously reported continuous spectrometric assay for ALDH1A3 optimized to suit Greiner Bio-One 96-UV-Transparent Microplate.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Tests were carried out by using 200 μL of reaction mixture, containing 20 mM Tris HCl pH 8.0, 1 mM β-mercaptoethanol, 150 mM KCl, 15% DMSO 500 μM NAD<sup>+</sup>, and 2.6 μM pure recombinant isoenzymes. Reactions were started by the addition of 20 mM acetaldehyde. Change in absorbance at 340 nm (ε<sub>NADH</sub> = 6220 M<sup>–1</sup> cm<sup>–1</sup>) was monitored for 30 min in a Tecan Sunrise at 25 °C. The activity of each enzymes was tested in the presence of each synthesized compound at 25 μM concentration in triplicates.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Enzyme Kinetic Analysis to Calculate IC<sub>50</sub> Values of the Most Promising Compound</h3><div class="NLM_p last">Selected compounds showing the best inhibitory activity at 25 μM were investigated further for their inhibitory efficacy to calculate the IC<sub>50</sub> values. The enzymatic inhibition assays were performed in a total volume of 200 μL of 20 mM Tris HCl pH 8.0, 1 mM β-mercaptoethanol, 150 mM KCl, 15% DMSO, 500 μM NAD<sup>+</sup>, 2.6 μM of pure recombinant isoenzymes in the presence of different inhibitors concentrations from 100 μM to 0.78125 μM and preincubated for 5 min. The kinetic parameters were determined by fitting the measured data to a Michaelis–Menten curve<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> by using SigmaPlot.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Crystallization and Structure Determination</h3><div class="NLM_p last">Crystals of ALDH1A3 in complex with <b>3a</b> (PDB deposition name <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11">GA11</a>) and <b>3q</b> (PDB deposition name <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ45">LQ45</a>) were obtained by using the vapor-diffusion technique in sitting drop and applying a spare-matrix-based strategy with a crystallization robot (Oryx4, Douglas Instruments). The best crystals of <b>3a</b>-ALDH1A3 were grown by mixing 0.5 μL of protein solution at a concentration of 8 mg/mL, preincubated with 1 mM NAD<sup>+</sup> and 300 μM <b>3a</b>, with an equal volume of a reservoir solution containing 2.4 M sodium malonate, pH 7.0, and equilibrated against 50 μL of the reservoir solution, at 20 °C in about 30 days. For X-ray data collection, crystals were quickly equilibrated in a solution containing the crystallization buffer and 12.5% glycerol as cryo-protectant and flash frozen at 100 K in liquid nitrogen. Data up to 3.2 Å resolution were collected at the beamline ID29 European Synchrotron Radiation Facility (ESRF, Grenoble, France). Analysis of the diffraction data set allowed us to assign the crystal to the orthorhombic <i>P</i>2<sub>1</sub>22<sub>1</sub> space group with cell dimensions of <i>a</i> = 81.56 Å, <i>b</i> = 89.36 Å, <i>c</i> = 159.04 Å and α = β = γ = 90°, containing two molecules per asymmetric unit with a corresponding solvent content of 52.5%. The best crystals of <b>3q</b>-ALDH1A3 were grown by mixing 0.5 μL of protein solution at a concentration of 8 mg/mL, preincubated with 1 mM NAD<sup>+</sup> and 300 μM <b>3q</b>, with an equal volume of a reservoir solution containing 2.4 M sodium malonate, pH 7.0, and equilibrated against 50 μL of the reservoir solution, at 20 °C in about 30 days. For X-ray data collection, crystals were quickly equilibrated in a solution containing the crystallization buffer and 12.5% glycerol as cryo-protectant and flash frozen at 100 K in liquid nitrogen. Data up to 3.2 Å resolution were collected at the beamline ID29 European Synchrotron Radiation Facility (ESRF, Grenoble, France). Analysis of the diffraction data set allowed us to assign the crystal to the orthorhombic <i>P</i>2<sub>1</sub>22<sub>1</sub> space group with cell dimensions of <i>a</i> = 81.2 Å, <i>b</i> = 89.4 Å, <i>c</i> = 159.2 Å and α = β = γ = 90°, containing two molecules per asymmetric unit with a corresponding solvent content of 47%. Data were processed using the program package XDS<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and the CCP4 suite of programs<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> was used for scaling. Determinations of both structures, <b>3a</b>-ALDH1A3 and <b>3q</b>-ALDH1A3, were carried out by means of the molecular replacement technique using the coordinates of the tetramer of human ALDH1A3 as the search model (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FHZ">5FHZ</a><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a>). PHASER<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> was used to automatically determine the ALDH1A3 structure in complex with the two compounds. The initial model was subjected to iterative cycles of crystallographic refinement with the programs REFMAC5<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and PHENIX.REFINE,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> alternated with manual graphic session for model building using the program Coot.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Then 5% of randomly chosen reflections were excluded from refinement of the structure and used for the Free R factor calculation.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The program ARP/wARP<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was used for adding solvent molecules. Refinement was continued until convergence to R-factor and free R-factor values of 0.19 and 0.26 for <b>3a</b>-ALDH1A3 and 0.18 and 0.24 for <b>3q</b>-ALDH1A3, respectively, with ideal geometry. Data collection and refinement statistics are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The atomic coordinates and structure factors of human ALDH1A3 in complex with <b>3a</b> (PDB deposition name <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11">GA11</a>) and <b>3q</b> (PDB deposition name <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43">LQ43</a>) have been deposited with the Protein Data Bank (<a href="http://www.rcsb.org" class="extLink">www.rcsb.org</a>) with the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5">6TE5</a>, respectively.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Illustrations</h3><div class="NLM_p last">Figures were generated using the program PyMOL.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Molecular Modeling</h3><div class="NLM_p last">The crystallographic structure of ALDH1A3 solved in this work was used as rigid receptor in molecular docking simulations, which were performed with GOLD docking program version 5.0.1.<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> PLP scoring function was used for ranking ligand’s poses. The binding site was centered in the mass center of residues Gly136, Trp189, and Leu471 and had a radius of 18 Å. Twenty genetic algorithm runs were carried out and visually inspected.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Antiproliferative Assays on GSCs</h3><div class="NLM_p last">The glioma sphere cultures used to investigate the antiproliferative activity of the novel compounds were generated by I. N. at the University of Alabama at Birmingham, following previously reported procedures.<a onclick="showRef(event, 'ref11 ref12 ref33'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref33">(11,12,33)</a> They were cultured in DMEM/F-12 medium (Invitrogen) containing 2% (vol/vol) B27 supplement (Invitrogen), epidermal growth factor (EGF, 20 ng/ml, Peprotech), basic fibroblast growth factor (bFGF, 20 ng/ml, Peprotech), and heparin (2.5 μg/ml). Cells were dissociated into single cell solution with Accutase and seeded into 96-well plates at 2000 cells/well, as previously described,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> then treated for 72 h with the test compounds or their vehicles alone. Stock solutions of the test compounds were obtained in DMSO, whose concentration in the final test solutions never exceeded 0.5%. At the end of the experiment, medium was removed and cell viability was measured using the AlamarBlue test (ThermoFischer Scientific), according to manufacturer’s protocol. The concentration of compounds able to reduce cell proliferation by 50% (IC<sub>50</sub>) versus controls was calculated with the SPSS 19.0 software by analyzing the nonlinear regression fit of the mean values of the data obtained in triplicate experiments.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01910" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51201" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51201" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01910?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01910</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Biochemical characterization of derivative <b>3a</b>; residual ALDHs activities in presence of 2,6-disubstituted-imidazo[1,2-<i>a</i>]pyridine derivatives <b>3a–v</b>; residual ALDHs activities in presence of 2,7-disubstituted-imidazo[1,2-<i>a</i>]pyridine derivatives <b>7a–i</b>; HPLC traces of 2,6-disubstituted-imidazo[1,2-<i>a</i>]pyridine derivatives, <b>3a–v</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula string (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_001.pdf">jm9b01910_si_001.pdf (1.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_002.csv">jm9b01910_si_002.csv (1.44 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01910" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Silvia Garavaglia</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of
Piemonte Orientale, Via
Bovio 6, 28100 Novara, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#98ebf1f4eef1f9b6fff9eaf9eef9fff4f1f9d8edf6f1ede8f7b6f1ec"><span class="__cf_email__" data-cfemail="493a20253f2028672e283b283f282e252028093c27203c392667203d">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Concettina La Motta</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0194-5431" title="Orcid link">http://orcid.org/0000-0002-0194-5431</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2e4d41404d4b5a5a47404f00424f43415a5a4f6e5b40475e4700475a"><span class="__cf_email__" data-cfemail="56353938353322223f3837783a373b392222371623383f263f783f22">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luca Quattrini</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Edoardo Luigi Maria Gelardi</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of
Piemonte Orientale, Via
Bovio 6, 28100 Novara, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vito Coviello</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefania Sartini</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Pisa, Via Bonanno 6, 56126 Pisa, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7671-5243" title="Orcid link">http://orcid.org/0000-0002-7671-5243</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Davide Maria Ferraris</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of
Piemonte Orientale, Via
Bovio 6, 28100 Novara, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mattia Mori</span> - <span class="hlFld-Affiliation affiliation">Department
of Biotechnology, Chemistry and Pharmacy “Department of Excellence
2018-2022”, University of Siena, via Aldo Moro 2, 53100 Siena, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2398-1254" title="Orcid link">http://orcid.org/0000-0003-2398-1254</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ichiro Nakano</span> - <span class="hlFld-Affiliation affiliation">Department
of Neurosurgery, University of Alabama at
Birmingham, WTI 401, 1824 Sixth Avenue South, Birmingham, Alabama 35233, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>L.Q. and E.L.M.G. contributed equally to the work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a> for the structure of human ALDH1A3 in complex with compound <b>3a</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11">GA11</a> in the PDB repository); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5">6TE5</a> for the structure of human ALDH1A3 in complex with compound <b>3q</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43">LQ43</a> in the PDB repository). Authors will release the atomic coordinates and experimental data upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">C.L.M. acknowledges the University of Pisa for the financial support (PRA_2017_51 Project). S.G. and D.M.F. acknowledge the University of Piemonte Orientale (grant Ricerca Ateneo FAR_2016) and ROCHE per la ricerca 2017 for the financial support. M.M. acknowledges a MIUR (Ministero dell’Istruzione, dell’Università e della Ricerca) grant “Dipartimento di Eccellenza 2018-2022”. All the authors thank the ESRF for data collections at the beam lines ID29.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">GBM</td><td class="NLM_def"><p class="first last">glioblastoma multiforme</p></td></tr><tr><td class="NLM_term">GSCs</td><td class="NLM_def"><p class="first last">glioma stem-like cells</p></td></tr><tr><td class="NLM_term">PN-GSCs</td><td class="NLM_def"><p class="first last">proneural glioma stem-like cells</p></td></tr><tr><td class="NLM_term">MES-GSCs</td><td class="NLM_def"><p class="first last">mesenchymal glioma stem-like cells</p></td></tr><tr><td class="NLM_term">DNA</td><td class="NLM_def"><p class="first last">deoxyribonucleic acid</p></td></tr><tr><td class="NLM_term">ALDH1A3</td><td class="NLM_def"><p class="first last">aldehyde dehydrogenase subtype 1A3</p></td></tr><tr><td class="NLM_term">ALDH1A1</td><td class="NLM_def"><p class="first last">aldehyde dehydrogenase subtype 1A1</p></td></tr><tr><td class="NLM_term">ALDH1A2</td><td class="NLM_def"><p class="first last">aldehyde dehydrogenase subtype 1A2</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77114" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77114" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Batash, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asna, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, M.</span></span> <span> </span><span class="NLM_article-title">Glioblastoma Multiforme, Diagnosis and Treatment; Recent Literature Review</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3002</span>– <span class="NLM_lpage">3009</span>, <span class="refDoi"> DOI: 10.2174/0929867324666170516123206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.2174%2F0929867324666170516123206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=28521700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sru7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=3002-3009&author=R.+Batashauthor=N.+Asnaauthor=P.+Schafferauthor=N.+Francisauthor=M.+Schaffer&title=Glioblastoma+Multiforme%2C+Diagnosis+and+Treatment%3B+Recent+Literature+Review&doi=10.2174%2F0929867324666170516123206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Glioblastoma Multiforme, Diagnosis and Treatment; Recent Literature Review</span></div><div class="casAuthors">Batash, Ron; Asna, Noam; Schaffer, Pamela; Francis, Nicole; Schaffer, Moshe</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3002-3009</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Background: Glioblastoma multiforme (GBM) is the most common malignant primary brain tumor, with an incidence of 3.19 cases per 100,000 person years and remarkably poor prognosis showing a 5-yr survival rate of 4-5%, and only a 26-33% survival rate at 2 years in clin. trials. </P><P> Objective: In this paper, we review the different types of treatment modalities based on the relevant databases.  Methods of diagnosis will be described briefly. </P><P> Method: Systemic compilation of the relevant literature. </P><P> Results & Conclusion: Today's treatments cannot cure GBM patients but only extend their overall survival.  The use of chemoradiation, immunotherapy, and radio sensitizers as an adjuvant therapy cannot reduce the high rates of recurrence within a few months after treatment.  Radiotherapy will remain the backbone of the treatment but new treatment modalities must be developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7b7aMyW_Nc7Vg90H21EOLACvtfcHk0lgsv-amL6_uvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sru7fN&md5=65623173dc9b1aa8e7c8ca62d4372c12</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F0929867324666170516123206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867324666170516123206%26sid%3Dliteratum%253Aachs%26aulast%3DBatash%26aufirst%3DR.%26aulast%3DAsna%26aufirst%3DN.%26aulast%3DSchaffer%26aufirst%3DP.%26aulast%3DFrancis%26aufirst%3DN.%26aulast%3DSchaffer%26aufirst%3DM.%26atitle%3DGlioblastoma%2520Multiforme%252C%2520Diagnosis%2520and%2520Treatment%253B%2520Recent%2520Literature%2520Review%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2017%26volume%3D24%26spage%3D3002%26epage%3D3009%26doi%3D10.2174%2F0929867324666170516123206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delgado-López, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corrales-García, E. M.</span></span> <span> </span><span class="NLM_article-title">Survival in Glioblastoma: a Review on the Impact of Treatment Modalities</span>. <i>Clin. Transl. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1062</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1007/s12094-016-1497-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1007%2Fs12094-016-1497-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26960561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVygsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=1062-1071&author=P.+D.+Delgado-L%C3%B3pezauthor=E.+M.+Corrales-Garc%C3%ADa&title=Survival+in+Glioblastoma%3A+a+Review+on+the+Impact+of+Treatment+Modalities&doi=10.1007%2Fs12094-016-1497-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Survival in glioblastoma: a review on the impact of treatment modalities</span></div><div class="casAuthors">Delgado-Lopez, P. D.; Corrales-Garcia, E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Translational Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1062-1071</span>CODEN:
                <span class="NLM_cas:coden">CTOLAM</span>;
        ISSN:<span class="NLM_cas:issn">1699-048X</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag Italia S.r.l.</span>)
        </div><div class="casAbstract">Glioblastoma (GBM) is the most common and lethal tumor of the central nervous system.  The natural history of treated GBM remains very poor with 5-yr survival rates of 5 %.  Survival has not significantly improved over the last decades.  Currently, the best that can be offered is a modest 14-mo overall median survival in patients undergoing max. safe resection plus adjuvant chemoradiotherapy.  Prognostic factors involved in survival include age, performance status, grade, specific markers (MGMT methylation, mutation of IDH1, IDH2 or TERT, 1p19q codeletion, overexpression of EGFR, etc.) and, likely, the extent of resection.  Certain adjuncts to surgery, esp. cortical mapping and 5-ALA fluorescence, favor higher rates of gross total resection with apparent pos. impact on survival.  Recurrent tumors can be offered re-intervention, participation in clin. trials, anti-angiogenic agent or local elec. field therapy, without an evident impact on survival.  Mol.-targeted therapies, immunotherapy and gene therapy are promising tools currently under research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcVrZshktqKbVg90H21EOLACvtfcHk0lgsv-amL6_uvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVygsrc%253D&md5=5bcfb0aebc622d839a9a97122db846db</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs12094-016-1497-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12094-016-1497-x%26sid%3Dliteratum%253Aachs%26aulast%3DDelgado-L%25C3%25B3pez%26aufirst%3DP.%2BD.%26aulast%3DCorrales-Garc%25C3%25ADa%26aufirst%3DE.%2BM.%26atitle%3DSurvival%2520in%2520Glioblastoma%253A%2520a%2520Review%2520on%2520the%2520Impact%2520of%2520Treatment%2520Modalities%26jtitle%3DClin.%2520Transl.%2520Oncol.%26date%3D2016%26volume%3D18%26spage%3D1062%26epage%3D1071%26doi%3D10.1007%2Fs12094-016-1497-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meacham, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, S. J.</span></span> <span> </span><span class="NLM_article-title">Tumour Heterogeneity and Cancer Cell Plasticity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>501</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nature12624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnature12624" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=24048065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOnurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=501&publication_year=2013&pages=328-337&author=C.+E.+Meachamauthor=S.+J.+Morrison&title=Tumour+Heterogeneity+and+Cancer+Cell+Plasticity&doi=10.1038%2Fnature12624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor heterogeneity and cancer cell plasticity</span></div><div class="casAuthors">Meacham, Corbin E.; Morrison, Sean J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">501</span>
        (<span class="NLM_cas:issue">7467</span>),
    <span class="NLM_cas:pages">328-337</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Phenotypic and functional heterogeneity arise among cancer cells within the same tumor as a consequence of genetic change, environmental differences and reversible changes in cell properties.  Some cancers also contain a hierarchy in which tumorigenic cancer stem cells differentiate into non-tumorigenic progeny.  However, it remains unclear what fraction of cancers follow the stem-cell model and what clin. behaviors the model explains.  Studies using lineage tracing and deep sequencing could have implications for the cancer stem-cell model and may help to det. the extent to which it accounts for therapy resistance and disease progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdbWv3pYPK67Vg90H21EOLACvtfcHk0lgU2mKQg3297w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOnurjN&md5=0d4aa5a83725d8cc272d1c19a9eafba9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature12624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12624%26sid%3Dliteratum%253Aachs%26aulast%3DMeacham%26aufirst%3DC.%2BE.%26aulast%3DMorrison%26aufirst%3DS.%2BJ.%26atitle%3DTumour%2520Heterogeneity%2520and%2520Cancer%2520Cell%2520Plasticity%26jtitle%3DNature%26date%3D2013%26volume%3D501%26spage%3D328%26epage%3D337%26doi%3D10.1038%2Fnature12624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Tumorigenesis: it Takes a Village</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/nrc3971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnrc3971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26156638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyksr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=473-483&author=D.+P.+Tabassumauthor=K.+Polyak&title=Tumorigenesis%3A+it+Takes+a+Village&doi=10.1038%2Fnrc3971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tumorigenesis: it takes a village</span></div><div class="casAuthors">Tabassum, Doris P.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">473-483</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is widely accepted that most cancers exhibit some degree of intratumor heterogeneity, we are far from understanding the dynamics that operate among subpopulations within tumors.  There is growing evidence that cancer cells behave as communities, and increasing attention is now being directed towards the cooperative behavior of subclones that can influence disease progression.  As expected, these interactions can add a greater layer of complexity to therapeutic interventions in heterogeneous tumors, often leading to a poor prognosis.  In this Review, we highlight studies that demonstrate such interactions in cancer and postulate ways to overcome them with better-designed therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocd11y6T1krrVg90H21EOLACvtfcHk0lgU2mKQg3297w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyksr3F&md5=1871f88a05e0990d6193e4985fc5fe6e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc3971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3971%26sid%3Dliteratum%253Aachs%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DTumorigenesis%253A%2520it%2520Takes%2520a%2520Village%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D473%26epage%3D483%26doi%3D10.1038%2Fnrc3971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pattabiraman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Tackling the Cancer Stem Cells - What Challenges Do They Pose?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">512</span>, <span class="refDoi"> DOI: 10.1038/nrd4253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnrd4253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=24981363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOisbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=497-512&author=D.+R.+Pattabiramanauthor=R.+A.+Weinberg&title=Tackling+the+Cancer+Stem+Cells+-+What+Challenges+Do+They+Pose%3F&doi=10.1038%2Fnrd4253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the cancer stem cells - what challenges do they pose?</span></div><div class="casAuthors">Pattabiraman, Diwakar R.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">497-512</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since their identification in 1994, cancer stem cells (CSCs) have been objects of intensive study.  Their properties and mechanisms of formation have become a major focus of current cancer research, in part because of their enhanced ability to initiate and drive tumor growth and their intrinsic resistance to conventional therapeutics.  The discovery that activation of the epithelial-to-mesenchymal transition (EMT) program in carcinoma cells can give rise to cells with stem-like properties has provided one possible mechanism explaining how CSCs arise and presents a possible avenue for their therapeutic manipulation.  Here we address recent developments in CSC research, focusing on carcinomas that are able to undergo EMT.  We discuss the signalling pathways that create these cells, cell-intrinsic mechanisms that could be exploited for selective elimination or induction of their differentiation, and the role of the tumor microenvironment in sustaining them.  Finally, we propose ways to use our current knowledge of the complex biol. of CSCs to design novel therapies to eliminate them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHXcwOZJN_67Vg90H21EOLACvtfcHk0lgU2mKQg3297w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOisbzI&md5=db211f41b19198981cb37487b7a9abc3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4253%26sid%3Dliteratum%253Aachs%26aulast%3DPattabiraman%26aufirst%3DD.%2BR.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DTackling%2520the%2520Cancer%2520Stem%2520Cells%2520-%2520What%2520Challenges%2520Do%2520They%2520Pose%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D497%26epage%3D512%26doi%3D10.1038%2Fnrd4253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squire, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hide, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henkelman, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusimano, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirks, P. B.</span></span> <span> </span><span class="NLM_article-title">Identification of Human Brain Tumour Initiating Cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">396</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1038/nature03128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnature03128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=15549107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=396-401&author=S.+K.+Singhauthor=C.+Hawkinsauthor=I.+D.+Clarkeauthor=J.+A.+Squireauthor=J.+Bayaniauthor=T.+Hideauthor=R.+M.+Henkelmanauthor=M.+D.+Cusimanoauthor=P.+B.+Dirks&title=Identification+of+Human+Brain+Tumour+Initiating+Cells&doi=10.1038%2Fnature03128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of human brain tumor initiating cells</span></div><div class="casAuthors">Singh, Sheila K.; Hawkins, Cynthia; Clarke, Ian D.; Squire, Jeremy A.; Bayani, Jane; Hide, Takuichiro; Henkelman, R. Mark; Cusimano, Michael D.; Dirks, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">396-401</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cancer stem cell (CSC) hypothesis suggests that neoplastic clones are maintained exclusively by a rare fraction of cells with stem cell properties.  Although the existence of CSCs in human leukemia is established, little evidence exists for CSCs in solid tumors, except for breast cancer.  Recently, the authors prospectively isolated a CD133+ cell subpopulation from human brain tumors that exhibited stem cell properties in vitro.  However, the true measures of CSCs are their capacity for self renewal and exact recapitulation of the original tumor.  Here the authors report the development of a xenograft assay that identified human brain tumor initiating cells that initiate tumors in vivo.  Only the CD133+ brain tumor fraction contains cells that are capable of tumor initiation in NOD-SCID (non-obese diabetic, severe combined immunodeficient) mouse brains.  Injection of as few as 100 CD133+ cells produced a tumor that could be serially transplanted and was a phenocopy of the patient's original tumor, whereas injection of 105 CD133- cells engrafted but did not cause a tumor.  Thus, the identification of brain tumor initiating cells provides insights into human brain tumor pathogenesis, giving strong support for the CSC hypothesis as the basis for many solid tumors, and establishes a previously unidentified cellular target for more effective cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK8T42M28y7bVg90H21EOLACvtfcHk0lgU2mKQg3297w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbY%253D&md5=f74762c020a9304ee683b24fb6e35919</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature03128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03128%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DHawkins%26aufirst%3DC.%26aulast%3DClarke%26aufirst%3DI.%2BD.%26aulast%3DSquire%26aufirst%3DJ.%2BA.%26aulast%3DBayani%26aufirst%3DJ.%26aulast%3DHide%26aufirst%3DT.%26aulast%3DHenkelman%26aufirst%3DR.%2BM.%26aulast%3DCusimano%26aufirst%3DM.%2BD.%26aulast%3DDirks%26aufirst%3DP.%2BB.%26atitle%3DIdentification%2520of%2520Human%2520Brain%2520Tumour%2520Initiating%2520Cells%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D396%26epage%3D401%26doi%3D10.1038%2Fnature03128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ginestier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diebel, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korkaya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicinski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabaud, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charafe-Jauffret, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnbaum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dontu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span> <span> </span><span class="NLM_article-title">CXCR1 Blockade Selectively Targets Human Breast Cancer Stem Cells in Vitro and in Xenografts</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1172/JCI39397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1172%2FJCI39397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=20051626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2010&pages=485-497&author=C.+Ginestierauthor=S.+Liuauthor=M.+E.+Diebelauthor=H.+Korkayaauthor=M.+Luoauthor=M.+Brownauthor=J.+Wicinskiauthor=O.+Cabaudauthor=E.+Charafe-Jauffretauthor=D.+Birnbaumauthor=J.+L.+Guanauthor=G.+Dontuauthor=M.+S.+Wicha&title=CXCR1+Blockade+Selectively+Targets+Human+Breast+Cancer+Stem+Cells+in+Vitro+and+in+Xenografts&doi=10.1172%2FJCI39397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">CXCR1 blockade selectively targets human breast cancer stem cells in vitro and in xenografts</span></div><div class="casAuthors">Ginestier, Christophe; Liu, Suling; Diebel, Mark E.; Korkaya, Hasan; Luo, Ming; Brown, Marty; Wicinski, Julien; Cabaud, Olivier; Charafe-Jauffret, Emmanuelle; Birnbaum, Daniel; Guan, Jun-Lin; Dontu, Gabriela; Wicha, Max S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">485-497</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Recent evidence suggests that breast cancer and other solid tumors possess a rare population of cells capable of extensive self-renewal that contribute to metastasis and treatment resistance.  We report here the development of a strategy to target these breast cancer stem cells (CSCs) through blockade of the IL-8 receptor CXCR1.  CXCR1 blockade using either a CXCR1-specific blocking antibody or repertaxin, a small-mol. CXCR1 inhibitor, selectively depleted the CSC population in 2 human breast cancer cell lines in vitro.  Furthermore, this was followed by the induction of massive apoptosis in the bulk tumor population via FASL/FAS signaling.  The effects of CXCR1 blockade on CSC viability and on FASL prodn. were mediated by the FAK/AKT/FOXO3A pathway.  In addn., repertaxin was able to specifically target the CSC population in human breast cancer xenografts, retarding tumor growth and reducing metastasis.  Our data therefore suggest that CXCR1 blockade may provide a novel means of targeting and eliminating breast CSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiolSraN9TPrVg90H21EOLACvtfcHk0liftnWWNQYd2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOktLw%253D&md5=cd042dfcf0abbff26b56a8b91245ecaf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2FJCI39397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI39397%26sid%3Dliteratum%253Aachs%26aulast%3DGinestier%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDiebel%26aufirst%3DM.%2BE.%26aulast%3DKorkaya%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DWicinski%26aufirst%3DJ.%26aulast%3DCabaud%26aufirst%3DO.%26aulast%3DCharafe-Jauffret%26aufirst%3DE.%26aulast%3DBirnbaum%26aufirst%3DD.%26aulast%3DGuan%26aufirst%3DJ.%2BL.%26aulast%3DDontu%26aufirst%3DG.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26atitle%3DCXCR1%2520Blockade%2520Selectively%2520Targets%2520Human%2520Breast%2520Cancer%2520Stem%2520Cells%2520in%2520Vitro%2520and%2520in%2520Xenografts%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2010%26volume%3D120%26spage%3D485%26epage%3D497%26doi%3D10.1172%2FJCI39397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span> <span> </span><span class="NLM_article-title">Induced Cancer Stem Cells Generated by Radiochemotherapy and their Therapeutic Implications</span>. <i>Oncotarget.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">17301</span>– <span class="NLM_lpage">17312</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.14230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.18632%2Foncotarget.14230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=28038467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A280%3ADC%252BC1c7gvVGisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=17301-17312&author=X.+Chenauthor=R.+Liaoauthor=D.+Liauthor=J.+Sun&title=Induced+Cancer+Stem+Cells+Generated+by+Radiochemotherapy+and+their+Therapeutic+Implications&doi=10.18632%2Foncotarget.14230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Induced cancer stem cells generated by radiochemotherapy and their therapeutic implications</span></div><div class="casAuthors">Chen Xiewan; Liao Rongxia; Chen Xiewan; Li Dezhi; Sun Jianguo</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">17301-17312</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Local and distant recurrence of malignant tumors following radio- and/or chemotherapy correlates with poor prognosis of patients.  Among the reasons for cancer recurrence, preexisting cancer stem cells (CSCs) are considered the most likely cause due to their properties of self-renewal, pluripotency, plasticity and tumorigenicity.  It has been demonstrated that preexisting cancer stem cells derive from normal stem cells and differentiated somatic cells that undergo transformation and dedifferentiation respectively under certain conditions.  However, recent studies have revealed that cancer stem cells can also be induced from non-stem cancer cells by radiochemotherapy, constituting the subpopulation of induced cancer stem cells (iCSCs).  These findings suggest that radiochemotherapy has the side effect of directly transforming non-stem cancer cells into induced cancer stem cells, possibly contributing to tumor recurrence and metastasis.  Therefore, drugs targeting cancer stem cells or preventing dedifferentiation of non-stem cancer cells can be combined with radiochemotherapy to improve its antitumor efficacy.  The current review is to investigate the mechanisms by which induced cancer stem cells are generated by radiochemotherapy and hence provide new strategies for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT2zDP_L2KDg9-cVm4E2RlLfW6udTcc2eaaX7WjlZcfFLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7gvVGisA%253D%253D&md5=68256fcb48cc0c5ed0596a67062b60db</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.14230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.14230%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DJ.%26atitle%3DInduced%2520Cancer%2520Stem%2520Cells%2520Generated%2520by%2520Radiochemotherapy%2520and%2520their%2520Therapeutic%2520Implications%26jtitle%3DOncotarget.%26date%3D2017%26volume%3D8%26spage%3D17301%26epage%3D17312%26doi%3D10.18632%2Foncotarget.14230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamdan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leboeuf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Bouchtaoui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gapihan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meles, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratajczak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Benedetto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janin, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Cancer Stem Cells to Overcome Chemoresistance</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4060</span>, <span class="refDoi"> DOI: 10.3390/ijms19124036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.3390%2Fijms19124036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=4036-4060&author=T.+Nunesauthor=D.+Hamdanauthor=C.+Leboeufauthor=M.+El+Bouchtaouiauthor=G.+Gapihanauthor=T.+T.+Nguyenauthor=S.+Melesauthor=E.+Angeliauthor=P.+Ratajczakauthor=H.+Luauthor=M.+Di+Benedettoauthor=G.+Bousquetauthor=A.+Janin&title=Targeting+Cancer+Stem+Cells+to+Overcome+Chemoresistance&doi=10.3390%2Fijms19124036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fijms19124036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19124036%26sid%3Dliteratum%253Aachs%26aulast%3DNunes%26aufirst%3DT.%26aulast%3DHamdan%26aufirst%3DD.%26aulast%3DLeboeuf%26aufirst%3DC.%26aulast%3DEl%2BBouchtaoui%26aufirst%3DM.%26aulast%3DGapihan%26aufirst%3DG.%26aulast%3DNguyen%26aufirst%3DT.%2BT.%26aulast%3DMeles%26aufirst%3DS.%26aulast%3DAngeli%26aufirst%3DE.%26aulast%3DRatajczak%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DDi%2BBenedetto%26aufirst%3DM.%26aulast%3DBousquet%26aufirst%3DG.%26aulast%3DJanin%26aufirst%3DA.%26atitle%3DTargeting%2520Cancer%2520Stem%2520Cells%2520to%2520Overcome%2520Chemoresistance%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D4036%26epage%3D4060%26doi%3D10.3390%2Fijms19124036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">ALDH1A3: a Marker of Mesenchymal Phenotype in Gliomas Associated with Cell Invasion</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0142856</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0142856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1371%2Fjournal.pone.0142856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26575197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSjsL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=W.+Zhangauthor=Y.+Liuauthor=H.+Huauthor=H.+Huangauthor=Z.+Baoauthor=P.+Yangauthor=Y.+Wangauthor=G.+Youauthor=W.+Yanauthor=T.+Jiangauthor=J.+Wangauthor=W.+Zhang&title=ALDH1A3%3A+a+Marker+of+Mesenchymal+Phenotype+in+Gliomas+Associated+with+Cell+Invasion&doi=10.1371%2Fjournal.pone.0142856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH1A3: a marker of mesenchymal phenotype in gliomas associated with cell invasion</span></div><div class="casAuthors">Zhang, Wenlong; Liu, Yanwei; Hu, Huimin; Huang, Hua; Bao, Zhaoshi; Yang, Pei; Wang, Yinyan; You, Gan; Yan, Wei; Jiang, Tao; Wang, Jiangfei; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0142856/1-e0142856/12</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDH) is a family of enzymes including 19 members.  For now, ALDH activity had been wildly used as a marker of cancer stem cells (CSCs).  But biol. functions of relevant isoforms and their clin. applications are still controversial.  Here, we investigate the clin. significance and potential function of ALDH1A3 in gliomas.  By whole-genome transcriptome microarray and mRNA sequencing anal., we compared the expression of ALDH1A3 in high- and low- grade gliomas as well as different mol. subtypes.  Microarray anal. was performed to identify the correlated genes of ALDH1A3.  We further used Gene Ontol. (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways anal. to explore the biol. function of ALDH1A3.  Finally, by mRNA knockdown we revealed the relationship between ALDH1A3 and the ability of tumor invasion.  ALDH1A3 overexpression was significantly assocd. with high grade as well as the higher mortality of gliomas in survival anal.  ALDH1A3 was characteristically highly expressed in Mesenchymal (Mes) subtype gliomas.  Moreover, we found that ALDH1A3 was most relevant to extracellular matrix organization and cell adhesion biol. process, and the ability of tumor invasion was suppressed after ALDH1A3 knockdown in vitro.  In conclusion, ALDH1A3 can serve as a novel marker of Mes phenotype in gliomas with potential clin. prognostic value.  The expression of ALDH1A3 is assocd. with tumor cell invasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouH3jvA9lFdLVg90H21EOLACvtfcHk0lh-3SPOninHfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSjsL7K&md5=4ff850171ba86b69e7b2bd07227c386f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0142856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0142856%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DBao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DALDH1A3%253A%2520a%2520Marker%2520of%2520Mesenchymal%2520Phenotype%2520in%2520Gliomas%2520Associated%2520with%2520Cell%2520Invasion%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0142856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santana-Santos, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandran, U. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benos, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobol, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, I.</span></span> <span> </span><span class="NLM_article-title">Mesenchymal Glioma Stem Cells are Maintained by Activated Glycolytic Metabolism Involving Aldehyde Dehydrogenase 1A3</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">8644</span>– <span class="NLM_lpage">8649</span>, <span class="refDoi"> DOI: 10.1073/pnas.1221478110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1073%2Fpnas.1221478110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=23650391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSjsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8644-8649&author=P.+Maoauthor=K.+Joshiauthor=J.+Liauthor=S.+H.+Kimauthor=P.+Liauthor=L.+Santana-Santosauthor=S.+Luthraauthor=U.+R.+Chandranauthor=P.+V.+Benosauthor=L.+Smithauthor=M.+Wangauthor=B.+Huauthor=S.+Y.+Chengauthor=R.+W.+Sobolauthor=I.+Nakano&title=Mesenchymal+Glioma+Stem+Cells+are+Maintained+by+Activated+Glycolytic+Metabolism+Involving+Aldehyde+Dehydrogenase+1A3&doi=10.1073%2Fpnas.1221478110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mesenchymal glioma stem cells are maintained by activated glycolytic metabolism involving aldehyde dehydrogenase 1A3</span></div><div class="casAuthors">Mao, Ping; Joshi, Kaushal; Li, Jianfeng; Kim, Sung-Hak; Li, Peipei; Santana-Santos, Lucas; Luthra, Soumya; Chandran, Uma R.; Benos, Panayiotis V.; Smith, Luke; Wang, Maode; Hu, Bo; Cheng, Shi-Yuan; Sobol, Robert W.; Nakano, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8644-8649, S8644/1-S8644/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Tumor heterogeneity of high-grade glioma (HGG) is recognized by four clin. relevant subtypes based on core gene signatures.  However, mol. signaling in glioma stem cells (GSCs) in individual HGG subtypes is poorly characterized.  Here we identified and characterized two mutually exclusive GSC subtypes with distinct dysregulated signaling pathways.  Anal. of mRNA profiles distinguished proneural (PN) from mesenchymal (Mes) GSCs and revealed a pronounced correlation with the corresponding PN or Mes HGGs.  Mes GSCs displayed more aggressive phenotypes in vitro and as intracranial xenografts in mice.  Further, Mes GSCs were markedly resistant to radiation compared with PN GSCs.  The glycolytic pathway, comprising aldehyde dehydrogenase (ALDH) family genes and in particular ALDH1A3, were enriched in Mes GSCs.  Glycolytic activity and ALDH activity were significantly elevated in Mes GSCs but not in PN GSCs.  Expression of ALDH1A3 was also increased in clin. HGG compared with low-grade glioma or normal brain tissue.  Moreover, inhibition of ALDH1 A3 attenuated the growth of Mes but not PN GSCs.  Last, radiation treatment of PN GSCs up-regulated Mes-assocd. markers and down-regulated PN-assocd. markers, whereas inhibition of ALDH1A3 attenuated an irradn.-induced gain of Mes identity in PN GSCs.  Taken together, our data suggest that two subtypes of GSCs, harboring distinct metabolic signaling pathways, represent intertumoral glioma heterogeneity and highlight previously unidentified roles of ALDH1 A3-assocd. signaling that promotes aberrant proliferation of Mes HGGs and GSCs.  Inhibition of ALDH1A3-mediated pathways therefore might provide a promising therapeutic approach for a subset of HGGs with the Mes signature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaISMNS12fzbVg90H21EOLACvtfcHk0lh-3SPOninHfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSjsbzE&md5=8896664bcf95f99bba9a928752a47b4d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221478110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221478110%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DP.%26aulast%3DJoshi%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSantana-Santos%26aufirst%3DL.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DChandran%26aufirst%3DU.%2BR.%26aulast%3DBenos%26aufirst%3DP.%2BV.%26aulast%3DSmith%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DCheng%26aufirst%3DS.%2BY.%26aulast%3DSobol%26aufirst%3DR.%2BW.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DMesenchymal%2520Glioma%2520Stem%2520Cells%2520are%2520Maintained%2520by%2520Activated%2520Glycolytic%2520Metabolism%2520Involving%2520Aldehyde%2520Dehydrogenase%25201A3%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D8644%26epage%3D8649%26doi%3D10.1073%2Fpnas.1221478110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waghmare, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coviello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohyeldin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlyukov, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentim, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chhipa, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Motta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kango-Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, I.</span></span> <span> </span><span class="NLM_article-title">FOXD1-ALDH1A3 Signaling is a Determinant for the Self-renewal and Tumorigenicity of Mesenchymal Glioma Stem Cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">7219</span>– <span class="NLM_lpage">7230</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2860</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1158%2F0008-5472.CAN-15-2860" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=27569208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVylsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=7219-7230&author=P.+Chengauthor=J.+Wangauthor=I.+Waghmareauthor=S.+Sartiniauthor=V.+Covielloauthor=Z.+Zhangauthor=S.+H.+Kimauthor=A.+Mohyeldinauthor=M.+S.+Pavlyukovauthor=M.+Minataauthor=C.+L.+Valentimauthor=R.+R.+Chhipaauthor=K.+P.+Bhatauthor=B.+Dasguptaauthor=C.+La+Mottaauthor=M.+Kango-Singhauthor=I.+Nakano&title=FOXD1-ALDH1A3+Signaling+is+a+Determinant+for+the+Self-renewal+and+Tumorigenicity+of+Mesenchymal+Glioma+Stem+Cells&doi=10.1158%2F0008-5472.CAN-15-2860"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">FOXD1-ALDH1A3 Signaling Is a Determinant for the Self-Renewal and Tumorigenicity of Mesenchymal Glioma Stem Cells</span></div><div class="casAuthors">Cheng, Peng; Wang, Jia; Waghmare, Indrayani; Sartini, Stefania; Coviello, Vito; Zhang, Zhuo; Kim, Sung-Hak; Mohyeldin, Ahmed; Pavlyukov, Marat S.; Minata, Mutsuko; Valentim, Claudia L. L.; Chhipa, Rishi Raj; Bhat, Krishna P. L.; Dasgupta, Biplab; La Motta, Concettina; Kango-Singh, Madhuri; Nakano, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7219-7230</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Glioma stem-like cells (GSC) with tumor-initiating activity orchestrate the cellular hierarchy in glioblastoma and engender therapeutic resistance.  Recent work has divided GSC into two subtypes with a mesenchymal (MES) GSC population as the more malignant subtype.  In this study, we identify the FOXD1-ALDH1A3 signaling axis as a determinant of the MES GSC phenotype.  The transcription factor FOXD1 is expressed predominantly in patient-derived cultures enriched with MES, but not with the proneural GSC subtype. shRNA-mediated attenuation of FOXD1 in MES GSC ablates their clonogenicity in vitro and in vivo.  Mechanistically, FOXD1 regulates the transcriptional activity of ALDH1A3, an established functional marker for MES GSC.  Indeed, the functional roles of FOXD1 and ALDH1A3 are likely evolutionally conserved, insofar as RNAi-mediated attenuation of their orthologous genes in Drosophila blocks formation of brain tumors engineered in that species.  In clin. specimens of high-grade glioma, the levels of expression of both FOXD1 and ALDH1A3 are inversely correlated with patient prognosis.  Finally, a novel small-mol. inhibitor of ALDH we developed, termed GA11, displays potent in vivo efficacy when administered systemically in a murine GSC-derived xenograft model of glioblastoma.  Collectively, our findings define a FOXD1-ALDH1A3 pathway in controlling the clonogenic and tumorigenic potential of MES GSC in glioblastoma tumors.  Cancer Res; 76(24); 7219-30. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9_D8WWqs-MbVg90H21EOLACvtfcHk0lh-3SPOninHfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVylsrjE&md5=0c8c01e69e88084ec73e2bf7b0a06d3a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2860%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWaghmare%26aufirst%3DI.%26aulast%3DSartini%26aufirst%3DS.%26aulast%3DCoviello%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DMohyeldin%26aufirst%3DA.%26aulast%3DPavlyukov%26aufirst%3DM.%2BS.%26aulast%3DMinata%26aufirst%3DM.%26aulast%3DValentim%26aufirst%3DC.%2BL.%26aulast%3DChhipa%26aufirst%3DR.%2BR.%26aulast%3DBhat%26aufirst%3DK.%2BP.%26aulast%3DDasgupta%26aufirst%3DB.%26aulast%3DLa%2BMotta%26aufirst%3DC.%26aulast%3DKango-Singh%26aufirst%3DM.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DFOXD1-ALDH1A3%2520Signaling%2520is%2520a%2520Determinant%2520for%2520the%2520Self-renewal%2520and%2520Tumorigenicity%2520of%2520Mesenchymal%2520Glioma%2520Stem%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D7219%26epage%3D7230%26doi%3D10.1158%2F0008-5472.CAN-15-2860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sobreira, T. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlétaz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simões-Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schechtman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronowicz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulo de Oliveira, P. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier-Neto, J.</span></span> <span> </span><span class="NLM_article-title">Structural Shifts of Aldehyde Dehydrogenase Enzymes were Instrumental for the Early Evolution of Retinoid-dependent Axial Patterning in Metazoans</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">226</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1073/pnas.1011223108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1073%2Fpnas.1011223108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=21169504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3MXms1KisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=226-231&author=T.+J.+P.+Sobreiraauthor=F.+Marl%C3%A9tazauthor=M.+Sim%C3%B5es-Costaauthor=D.+Schechtmanauthor=A.+C.+Pereiraauthor=F.+Brunetauthor=S.+Sweeneyauthor=A.+Paniauthor=J.+Aronowiczauthor=C.+J.+Loweauthor=B.+Davidsonauthor=V.+Laudetauthor=M.+Bronnerauthor=P.+S.+L.+Paulo+de+Oliveiraauthor=M.+Schubertauthor=J.+Xavier-Neto&title=Structural+Shifts+of+Aldehyde+Dehydrogenase+Enzymes+were+Instrumental+for+the+Early+Evolution+of+Retinoid-dependent+Axial+Patterning+in+Metazoans&doi=10.1073%2Fpnas.1011223108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural shifts of aldehyde dehydrogenase enzymes were instrumental for the early evolution of retinoid-dependent axial patterning in metazoans</span></div><div class="casAuthors">Sobreira, Tiago J. P.; Marletaz, Ferdinand; Simoes-Costa, Marcos; Schechtman, Deborah; Pereira, Alexandre C.; Brunet, Frederic; Sweeney, Sarah; Pani, Ariel; Aronowicz, Jochanan; Lowe, Christopher J.; Davidson, Bradley; Laudet, Vincent; Bronner, Marianne; de Oliveiraa, Paulo S. L.; Schubert, Michael; Xavier-Neto, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">226-231, S226/1-S226/22</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDHs) catabolize toxic aldehydes and process the vitamin A-derived retinaldehyde into retinoic acid (RA), a small diffusible mol. and a pivotal chordate morphogen.  In this study, we combine phylogenetic, structural, genomic, and developmental gene expression analyses to examine the evolutionary origins of ALDH substrate preference.  Structural modeling reveals that processing of small aldehydes, such as acetaldehyde, by ALDH2, vs. large aldehydes, including retinaldehyde, by ALDH1A is assocd. with small vs. large substrate entry channels (SECs), resp.  Moreover, we show that metazoan ALDH1s and ALDH2s are members of a single ALDH1/2 clade and that during evolution, eukaryote ALDH1/2s often switched between large and small SECs after gene duplication, transforming constricted channels into wide opened ones and vice versa.  Ancestral sequence reconstructions suggest that during the evolutionary emergence of RA signaling, the ancestral, narrow-channeled metazoan ALDH1/2 gave rise to large ALDH1 channels capable of accommodating bulky aldehydes, such as retinaldehyde, supporting the view that retinoid-dependent signaling arose from ancestral cellular detoxification mechanisms.  Our analyses also indicate that, on a more restricted evolutionary scale, ALDH1 duplicates from invertebrate chordates (amphioxus and ascidian tunicates) underwent switches to smaller and narrower SECs.  When combined with alterations in gene expression, these switches led to neo-functionalization from ALDH1-like roles in embryonic patterning to systemic, ALDH2-like roles, suggesting functional shifts from signaling to detoxification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfJDaWsQicerVg90H21EOLACvtfcHk0lhTLZM2pl7ghw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXms1KisQ%253D%253D&md5=225e4c673b52bb162b4c96e509012a4d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1011223108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1011223108%26sid%3Dliteratum%253Aachs%26aulast%3DSobreira%26aufirst%3DT.%2BJ.%2BP.%26aulast%3DMarl%25C3%25A9taz%26aufirst%3DF.%26aulast%3DSim%25C3%25B5es-Costa%26aufirst%3DM.%26aulast%3DSchechtman%26aufirst%3DD.%26aulast%3DPereira%26aufirst%3DA.%2BC.%26aulast%3DBrunet%26aufirst%3DF.%26aulast%3DSweeney%26aufirst%3DS.%26aulast%3DPani%26aufirst%3DA.%26aulast%3DAronowicz%26aufirst%3DJ.%26aulast%3DLowe%26aufirst%3DC.%2BJ.%26aulast%3DDavidson%26aufirst%3DB.%26aulast%3DLaudet%26aufirst%3DV.%26aulast%3DBronner%26aufirst%3DM.%26aulast%3DPaulo%2Bde%2BOliveira%26aufirst%3DP.%2BS.%2BL.%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DXavier-Neto%26aufirst%3DJ.%26atitle%3DStructural%2520Shifts%2520of%2520Aldehyde%2520Dehydrogenase%2520Enzymes%2520were%2520Instrumental%2520for%2520the%2520Early%2520Evolution%2520of%2520Retinoid-dependent%2520Axial%2520Patterning%2520in%2520Metazoans%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D226%26epage%3D231%26doi%3D10.1073%2Fpnas.1011223108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobol, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garavaglia, S.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human Aldehyde Dehydrogenase 1A3 Complexed with NAD<sup>+</sup> and Retinoic Acid</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">35710</span>– <span class="NLM_lpage">35721</span>, <span class="refDoi"> DOI: 10.1038/srep35710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fsrep35710" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=35710-35721&author=A.+Morettiauthor=J.+Liauthor=S.+Doniniauthor=R.+W.+Sobolauthor=M.+Rizziauthor=S.+Garavaglia&title=Crystal+Structure+of+Human+Aldehyde+Dehydrogenase+1A3+Complexed+with+NAD%2B+and+Retinoic+Acid&doi=10.1038%2Fsrep35710"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsrep35710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep35710%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DDonini%26aufirst%3DS.%26aulast%3DSobol%26aufirst%3DR.%2BW.%26aulast%3DRizzi%26aufirst%3DM.%26aulast%3DGaravaglia%26aufirst%3DS.%26atitle%3DCrystal%2520Structure%2520of%2520Human%2520Aldehyde%2520Dehydrogenase%25201A3%2520Complexed%2520with%2520NAD%252B%2520and%2520Retinoic%2520Acid%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D19%26spage%3D35710%26epage%3D35721%26doi%3D10.1038%2Fsrep35710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved Protein-ligand Docking using GOLD</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+Protein-ligand+Docking+using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Å of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of ∼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0lhTLZM2pl7ghw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520Protein-ligand%2520Docking%2520using%2520GOLD%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Development of a High-throughput in Vitro Assay to Identify Selective Inhibitors for Human ALDH1A1</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1016/j.cbi.2014.10.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1016%2Fj.cbi.2014.10.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=25450233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2015&pages=29-37&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Development+of+a+High-throughput+in+Vitro+Assay+to+Identify+Selective+Inhibitors+for+Human+ALDH1A1&doi=10.1016%2Fj.cbi.2014.10.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a high-throughput in vitro assay to identify selective inhibitors for human ALDH1A1</span></div><div class="casAuthors">Morgan, Cynthia A.; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-37</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The human aldehyde dehydrogenase (ALDH) superfamily consists of at least 19 enzymes that metabolize endogenous and exogenous aldehydes.  Currently, there are no com. available inhibitors that target ALDH1A1 but have little to no effect on the structurally and functionally similar ALDH2.  Here we present the first human ALDH1A1 structure, as the apo-enzyme and in complex with its cofactor NADH to a resoln. of 1.75 and 2.1 Å, respectfully.  Structural comparisons of the cofactor binding sites in ALDH1A1 with other closely related ALDH enzymes illustrate a high degree of similarity.  In order to minimize discovery of compds. that inhibit both isoenzymes by interfering with their conserved cofactor binding sites, this study reports the use of an in vitro, NAD+-independent, esterase-based high-throughput screen (HTS) of 64,000 compds. to discover novel, selective inhibitors of ALDH1A1.  We describe 256 hits that alter the esterase activity of ALDH1A1.  The effects on aldehyde oxidn. of 67 compds. were further analyzed, with 30 selectively inhibiting ALDH1A1 compared to ALDH2 and ALDH3A1.  One compd. inhibited ALDH1A1 and ALDH2, while another inhibited ALDH1A1, ALDH2, and the more distantly related ALDH3A1.  The results presented here indicate that this in vitro enzyme activity screening protocol successfully identified ALDH1A1 inhibitors with a high degree of isoenzyme selectivity.  The compds. identified via this screen plus the screening methodol. itself represent a starting point for the development of highly potent and selective inhibitors of ALDH1A1 that may be utilized to better understand the role of this enzyme in both normal and disease states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0H8M4JN5aC7Vg90H21EOLACvtfcHk0lhTLZM2pl7ghw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP&md5=64ce33cd18ae04a4ef8796bd9c4474a7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.10.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.10.028%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520High-throughput%2520in%2520Vitro%2520Assay%2520to%2520Identify%2520Selective%2520Inhibitors%2520for%2520Human%2520ALDH1A1%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2015%26volume%3D234%26spage%3D29%26epage%3D37%26doi%3D10.1016%2Fj.cbi.2014.10.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikles, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amory, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">590</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00685</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00685" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=582-590&author=Y.+Chenauthor=J.+Y.+Zhuauthor=K.+H.+Hongauthor=D.+C.+Miklesauthor=G.+I.+Georgauthor=A.+S.+Goldsteinauthor=J.+K.+Amoryauthor=E.+Sch%C3%B6nbrunn&title=Structural+Basis+of+ALDH1A2+Inhibition+by+Irreversible+and+Reversible+Small+Molecule+Inhibitors&doi=10.1021%2Facschembio.7b00685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span></div><div class="casAuthors">Chen, Yan; Zhu, Jin-Yi; Hong, Kwon Ho; Mikles, David C.; Georg, Gunda I.; Goldstein, Alex S.; Amory, John K.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">582-590</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes of the ALDH1A subfamily of aldehyde dehydrogenases are crucial in regulating retinoic acid (RA) signaling and have received attention as potential drug targets.  ALDH1A2 is the primary RA-synthesizing enzyme in mammalian spermatogenesis and is therefore considered a viable drug target for male contraceptive development.  However, only a small no. of ALDH1A2 inhibitors have been reported, and information on the structure of ALDH1A2 was limited to the NAD-liganded enzyme void of substrate or inhibitors.  Herein, the authors describe the mechanism of action of structurally unrelated reversible and irreversible inhibitors of human ALDH1A2 using direct binding studies and x-ray crystallog.  All inhibitors bind to the active sites of tetrameric ALDH1A2.  Compd. WIN18,446 covalently reacts with the side chain of the catalytic residue Cys 320, resulting in a chiral adduct in (R) configuration.  The covalent adduct directly affects the neighboring NAD mol., which assumes a contracted conformation suboptimal for the dehydrogenase reaction.  The reversible inhibitors interact predominantly through direct hydrogen bonding interactions with residues in the vicinity of Cys 320 without affecting NAD.  Upon interaction with inhibitors, a large flexible loop assumes regular structure, thereby shielding the active site from solvent.  The precise knowledge of the binding modes provides a new framework for the rational design of novel inhibitors of ALDH1A2 with improved potency and selectivity profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprC4rddNQ7u7Vg90H21EOLACvtfcHk0li2KWBpo7Ydvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK&md5=dff6eef4a876a603048f3ccd53f60dee</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00685%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DHong%26aufirst%3DK.%2BH.%26aulast%3DMikles%26aufirst%3DD.%2BC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DGoldstein%26aufirst%3DA.%2BS.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DStructural%2520Basis%2520of%2520ALDH1A2%2520Inhibition%2520by%2520Irreversible%2520and%2520Reversible%2520Small%2520Molecule%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D582%26epage%3D590%26doi%3D10.1021%2Facschembio.7b00685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerencser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panka, G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egyed, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uerge, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvas, F.</span></span> <span> </span><span class="NLM_article-title">Procedure for the Parallel Preparation of 3-Imidazo[1,2-<i>a</i>]pyridin-3-yl Propionic Acid Derivatives Involving Meldrum’s acid</span>. <i>J. Comb. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">530</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1021/cc040101j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc040101j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVWktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=530-538&author=J.+Gerencserauthor=G+Pankaauthor=T.+Nagyauthor=O.+Egyedauthor=G.+Dormanauthor=L.+Uergeauthor=F.+Darvas&title=Procedure+for+the+Parallel+Preparation+of+3-Imidazo%5B1%2C2-a%5Dpyridin-3-yl+Propionic+Acid+Derivatives+Involving+Meldrum%E2%80%99s+acid&doi=10.1021%2Fcc040101j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Procedure for the Parallel Preparation of 3-Imidazo[1,2-a]pyridin-3-ylpropionic Acid Derivatives Involving Meldrum's Acid</span></div><div class="casAuthors">Gerencser, Janos; Panka, Gabor; Nagy, Tamas; Egyed, Orsolya; Dorman, Gyoergy; Uerge, Laszlo; Darvas, Ferenc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">530-538</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe here an efficient and versatile method for the prepn. of 3-imidazo[1,2-a]pyridin-3-ylpropionic acids involving, as a key step, a three-component Michael-type reaction.  The extended and validated procedure allowed prepn. of various acids with three diversity points.  E.g., a three-component reaction of imidazo[1,2-a]pyridine, Meldrum's acid, and RCHO [R = substituted Ph, 2-furyl, 2-thienyl, Me2CH, BnCH2] in the presence of a catalytic amt. of DL-proline gave imidazo[1,2-a]pyridines I.  The latter were converted to 3-imidazo[1,2-a]pyridin-3-ylpropionic acids II.  The method was easily adaptable for parallel synthesis and an ∼2000-membered 3-imidazo[1,2-a]pyridin-3-ylpropionic acid amide library was prepd. in a semiautomated manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptft8VWbhkC7Vg90H21EOLACvtfcHk0li2KWBpo7Ydvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVWktro%253D&md5=d20874eace950681314e5ab6740447c6</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fcc040101j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc040101j%26sid%3Dliteratum%253Aachs%26aulast%3DGerencser%26aufirst%3DJ.%26aulast%3DPanka%26aufirst%3DG%26aulast%3DNagy%26aufirst%3DT.%26aulast%3DEgyed%26aufirst%3DO.%26aulast%3DDorman%26aufirst%3DG.%26aulast%3DUerge%26aufirst%3DL.%26aulast%3DDarvas%26aufirst%3DF.%26atitle%3DProcedure%2520for%2520the%2520Parallel%2520Preparation%2520of%25203-Imidazo%255B1%252C2-a%255Dpyridin-3-yl%2520Propionic%2520Acid%2520Derivatives%2520Involving%2520Meldrum%25E2%2580%2599s%2520acid%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2005%26volume%3D7%26spage%3D530%26epage%3D538%26doi%3D10.1021%2Fcc040101j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bollenbach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvat, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daubeuf, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yalcin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letellier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bihel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourguignon, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frossard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M.</span></span> <span> </span><span class="NLM_article-title">Phenylpyridine-2-ylguanidines and Rigid Mimetics as Novel Inhibitors of TNFα Overproduction: Beneficial Action in Models of Neuropathic Pain and of Acute Lung Inflammation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1016%2Fj.ejmech.2018.01.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=29432948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCmurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=163-182&author=M.+Bollenbachauthor=E.+Salvatauthor=F.+Daubeufauthor=P.+Wagnerauthor=I.+Yalcinauthor=M.+Humoauthor=B.+Letellierauthor=L.+J.+Beckerauthor=F.+Bihelauthor=J.-J.+Bourguignonauthor=P.+Villaauthor=A.+Obrechtauthor=N.+Frossardauthor=M.+Barrotauthor=M.+Schmitt&title=Phenylpyridine-2-ylguanidines+and+Rigid+Mimetics+as+Novel+Inhibitors+of+TNF%CE%B1+Overproduction%3A+Beneficial+Action+in+Models+of+Neuropathic+Pain+and+of+Acute+Lung+Inflammation&doi=10.1016%2Fj.ejmech.2018.01.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylpyridine-2-ylguanidines and rigid mimetics as novel inhibitors of TNFα overproduction: Beneficial action in models of neuropathic pain and of acute lung inflammation</span></div><div class="casAuthors">Bollenbach, Maud; Salvat, Eric; Daubeuf, Francois; Wagner, Patrick; Yalcin, Ipek; Humo, Muris; Letellier, Baptiste; Becker, Lea J.; Bihel, Frederic; Bourguignon, Jean-Jacques; Villa, Pascal; Obrecht, Adeline; Frossard, Nelly; Barrot, Michel; Schmitt, Martine</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">163-182</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">4-Phenylpyridin-2-yl-guanidine (5b): a new inhibitor of the overprodn. of pro-inflammatory cytokines (TNFα and Il1β) was identified from a high-throughput screening of a chem. library on human peripheral blood mononuclear cells (PBMCs) after LPS stimulation.  Derivs., homologues and rigid mimetics of 5b were designed and synthesized, and their cytotoxicity and ability to inhibit TNFα overprodn. were evaluated.  Among them, compd. 5b and its mimetic 12 (2-aminodihydroquinazoline) showed similar inhibitory activities, and were evaluated in vivo in models of lung inflammation and neuropathic pain in mice.  In particular, compd. 12 proved to be active (5mg/kg, i.p.) in both models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYdDk_3kb9JbVg90H21EOLACvtfcHk0liPf6vY9lvZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCmurs%253D&md5=7514977ba42f66e83430feb834ff631a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.049%26sid%3Dliteratum%253Aachs%26aulast%3DBollenbach%26aufirst%3DM.%26aulast%3DSalvat%26aufirst%3DE.%26aulast%3DDaubeuf%26aufirst%3DF.%26aulast%3DWagner%26aufirst%3DP.%26aulast%3DYalcin%26aufirst%3DI.%26aulast%3DHumo%26aufirst%3DM.%26aulast%3DLetellier%26aufirst%3DB.%26aulast%3DBecker%26aufirst%3DL.%2BJ.%26aulast%3DBihel%26aufirst%3DF.%26aulast%3DBourguignon%26aufirst%3DJ.-J.%26aulast%3DVilla%26aufirst%3DP.%26aulast%3DObrecht%26aufirst%3DA.%26aulast%3DFrossard%26aufirst%3DN.%26aulast%3DBarrot%26aufirst%3DM.%26aulast%3DSchmitt%26aufirst%3DM.%26atitle%3DPhenylpyridine-2-ylguanidines%2520and%2520Rigid%2520Mimetics%2520as%2520Novel%2520Inhibitors%2520of%2520TNF%25CE%25B1%2520Overproduction%253A%2520Beneficial%2520Action%2520in%2520Models%2520of%2520Neuropathic%2520Pain%2520and%2520of%2520Acute%2520Lung%2520Inflammation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D163%26epage%3D182%26doi%3D10.1016%2Fj.ejmech.2018.01.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michaelis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menten, M. L.</span></span> <span> </span><span class="NLM_article-title">Die Kinetik der Invertinwirkung</span>. <i>Biochemische Zeitschrift.</i> <span class="NLM_year" style="font-weight: bold;">1913</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">369</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADyaC3sXhsFaqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1913&pages=333-369&author=L.+Michaelisauthor=M.+L.+Menten&title=Die+Kinetik+der+Invertinwirkung"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of Invertase Action</span></div><div class="casAuthors">Michaelis, L.; Menten, Maud L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemische Zeitschrift</span>
        (<span class="NLM_cas:date">1913</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">333-69</span>CODEN:
                <span class="NLM_cas:coden">BIZEA2</span>;
        ISSN:<span class="NLM_cas:issn">0366-0753</span>.
    </div><div class="casAbstract">The course of sugar inversion by invertase is consistent with the assumption that saccharose and enzyme unite to form a combination, of which the dissociation const. is 0.0167.  The comp. is labile and breaks up into 1 mol. each of glucose, fructose, and invertin.  Invertin has an affinity for glucose and fructose, as well as for other carbohydrates and higher alcs. (mannitol and glycerol), although in none of these cases is the affinity so great as for saccharose.  The compds. formed with these substs. are not labile and the substs. do not suffer decomp.  They show their combining capacity for the enzyme by the fact that their presence retards the inversion of saccharose by invertin.  The concs. of all the invertin-carbohydrate compds. were calc. according to the law of mass action and the dissociation consts. detd.  The decomp. of the saccharose-invertin compd. being a monomol. reaction, the inversion velocity at any time is directly proportional to the conc. of the compd.  From the above assumptions a differential equation is derived which agrees well with the observed inversion rates of saccharose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkv37PaE45mbVg90H21EOLACvtfcHk0liPf6vY9lvZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaC3sXhsFaqtw%253D%253D&md5=8ea41914dde2d77f646b002014a1a1dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMichaelis%26aufirst%3DL.%26aulast%3DMenten%26aufirst%3DM.%2BL.%26atitle%3DDie%2520Kinetik%2520der%2520Invertinwirkung%26jtitle%3DBiochemische%2520Zeitschrift.%26date%3D1913%26volume%3D49%26spage%3D333%26epage%3D369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span> <i>SigmaPlot Extract Graphs and Data Analysis (StatSys v12.3)</i>; <span class="NLM_publisher-name">Systat</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SigmaPlot+Extract+Graphs+and+Data+Analysis+%28StatSys+v12.3%29%3B+Systat%2C+2020."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSigmaPlot%2520Extract%2520Graphs%2520and%2520Data%2520Analysis%2520%2528StatSys%2520v12.3%2529%26pub%3DSystat%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-32&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liPf6vY9lvZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D32%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 suite: Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0AComputational+Project%2C+Number+4&title=The+CCP4+suite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liVxkgI1UP7LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>D53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-likelihood+Method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liVxkgI1UP7LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3DD53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-67&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0liVxkgI1UP7LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D67%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhlpJXGzrVOdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunger, A. T.</span></span> <span> </span><span class="NLM_article-title">Free R Value: a Novel Statistic Quantity for Assesing the Accuracy of Crystal Structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">475</span>, <span class="refDoi"> DOI: 10.1038/355472a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2F355472a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=18481394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A280%3ADC%252BD1czhvFCquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=1992&pages=472-475&author=A.+T.+Brunger&title=Free+R+Value%3A+a+Novel+Statistic+Quantity+for+Assesing+the+Accuracy+of+Crystal+Structures&doi=10.1038%2F355472a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Free R value: a novel statistical quantity for assessing the accuracy of crystal structures</span></div><div class="casAuthors">Brunger A T</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6359</span>),
    <span class="NLM_cas:pages">472-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The determination of macromolecular structure by crystallography involves fitting atomic models to the observed diffraction data.  The traditional measure of the quality of this fit, and presumably the accuracy of the model, is the R value.  Despite stereochemical restraints, it is possible to overfit or 'misfit' the diffraction data: an incorrect model can be refined to fairly good R values as several recent examples have shown.  Here I propose a reliable and unbiased indicator of the accuracy of such models.  By analogy with the cross-validation method of testing statistical models I define a statistical quantity (R(free) (T) that measures the agreement between observed and computed structure factor amplitudes for a 'test' set of reflections that is omitted in the modelling and refinement process.  As examples show, there is a high correlation between R(free) (T) and the accuracy of the atomic model phases.  This is useful because experimental phase information is usually inaccurate, incomplete or unavailable.  I expect that R(free) (T) will provide a measure of the information content of recently proposed models of thermal motion and disorder, time-averaging and bulk solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQY_IzgfwBLm_h3YQQtAhNufW6udTcc2ebDvo_KeQUJGLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czhvFCquw%253D%253D&md5=ddeaa021753f1208b405e03c1428078f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2F355472a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F355472a0%26sid%3Dliteratum%253Aachs%26aulast%3DBrunger%26aufirst%3DA.%2BT.%26atitle%3DFree%2520R%2520Value%253A%2520a%2520Novel%2520Statistic%2520Quantity%2520for%2520Assesing%2520the%2520Accuracy%2520of%2520Crystal%2520Structures%26jtitle%3DNature%26date%3D1992%26volume%3D355%26spage%3D472%26epage%3D475%26doi%3D10.1038%2F355472a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perrakis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harkiolaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span> <span> </span><span class="NLM_article-title">ARP/wARP and Molecular Replacement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1445</span>– <span class="NLM_lpage">1450</span>, <span class="refDoi"> DOI: 10.1107/S0907444901014007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444901014007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=11567158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFGltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=1445-1450&author=A.+Perrakisauthor=M.+Harkiolakiauthor=K.+S.+Wilsonauthor=V.+S.+Lamzin&title=ARP%2FwARP+and+Molecular+Replacement&doi=10.1107%2FS0907444901014007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">ARP/wARP and molecular replacement</span></div><div class="casAuthors">Perrakis, Anastassis; Harkiolaki, Maria; Wilson, Keith S.; Lamzin, Victor S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">D57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1445-1450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The aim of ARP/wARP is improved automation of model building and refinement in macromol. crystallog.  Once a mol.-replacement soln. has been obtained, it is often tedious to refine and rebuild the initial (search) model.  ARP/wARP offers three options to automate that task to varying extents: (i) autobuilding of a completely new model based on phases calcd. from the mol.-replacement soln., (ii) updating of the initial model by atom addn. and deletion to obtain an improved map and (iii) docking of a structure onto a new (or mutated) sequence, followed by rebuilding and refining the side chains in real space.  A few examples are presented where ARP/wARP made a considerable difference in the speed of structure soln. and/or made possible refinement of otherwise difficult or uninterpretable maps.  The resoln. range allowing complete autobuilding of protein structures is currently 2.0 Å, but for map improvement considerable advances over more conventional refinement techniques are evident even at 3.2 Å spacing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra62pGOo0oCbVg90H21EOLACvtfcHk0lhlpJXGzrVOdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFGltLc%253D&md5=7c885ae862051042e716ad0e995e131c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444901014007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444901014007%26sid%3Dliteratum%253Aachs%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DHarkiolaki%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26atitle%3DARP%252FwARP%2520and%2520Molecular%2520Replacement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2001%26volume%3D57%26spage%3D1445%26epage%3D1450%26doi%3D10.1107%2FS0907444901014007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span> <i>The
PyMOL Molecular Graphics System, version 1.3</i>; <span class="NLM_publisher-name">Schrödinger LLC</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The%0APyMOL+Molecular+Graphics+System%2C+version+1.3%3B+Schr%C3%B6dinger+LLC%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%250APyMOL%2520Molecular%2520Graphics%2520System%252C%2520version%25201.3%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved Protein-ligand Docking Using GOLD</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+Protein-ligand+Docking+Using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Å of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of ∼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0lgb3kww0IHe_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520Protein-ligand%2520Docking%2520Using%2520GOLD%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for Flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lgb3kww0IHe_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezhilarasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego-Perez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabit, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chhipa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohyeldin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurozumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroiwa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guttridge, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DasGupta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goidts, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, I.</span></span> <span> </span><span class="NLM_article-title">Serine/Threonine Kinase MLK4 Determines Mesenchymal Identity in Glioma Stem Cells in an NF-κB-dependent Manner</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.01.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1016%2Fj.ccell.2016.01.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26859459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC28XitlSgsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=201-213&author=S.+H.+Kimauthor=R.+Ezhilarasanauthor=E.+Phillipsauthor=D.+Gallego-Perezauthor=A.+Sparksauthor=D.+Taylorauthor=K.+Ladnerauthor=T.+Furutaauthor=H.+Sabitauthor=R.+Chhipaauthor=J.+H.+Choauthor=A.+Mohyeldinauthor=S.+Beckauthor=K.+Kurozumiauthor=T.+Kuroiwaauthor=R.+Iwataauthor=A.+Asaiauthor=J.+Kimauthor=E.+P.+Sulmanauthor=S.+Y.+Chengauthor=L.+J.+Leeauthor=M.+Nakadaauthor=D.+Guttridgeauthor=B.+DasGuptaauthor=V.+Goidtsauthor=K.+P.+Bhatauthor=I.+Nakano&title=Serine%2FThreonine+Kinase+MLK4+Determines+Mesenchymal+Identity+in+Glioma+Stem+Cells+in+an+NF-%CE%BAB-dependent+Manner&doi=10.1016%2Fj.ccell.2016.01.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Serine/Threonine Kinase MLK4 Determines Mesenchymal Identity in Glioma Stem Cells in an NF-κB-dependent Manner</span></div><div class="casAuthors">Kim, Sung-Hak; Ezhilarasan, Ravesanker; Phillips, Emma; Gallego-Perez, Daniel; Sparks, Amanda; Taylor, David; Ladner, Katherine; Furuta, Takuya; Sabit, Hemragul; Chhipa, Rishi; Cho, Ju Hwan; Mohyeldin, Ahmed; Beck, Samuel; Kurozumi, Kazuhiko; Kuroiwa, Toshihiko; Iwata, Ryoichi; Asai, Akio; Kim, Jonghwan; Sulman, Erik P.; Cheng, Shi-Yuan; Lee, L. James; Nakada, Mitsutoshi; Guttridge, Denis; DasGupta, Biplab; Goidts, Violaine; Bhat, Krishna P.; Nakano, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-213</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Activation of nuclear factor κB (NF-κB) induces mesenchymal (MES) transdifferentiation and radioresistance in glioma stem cells (GSCs), but mol. mechanisms for NF-κB activation in GSCs are currently unknown.  Here, we report that mixed lineage kinase 4 (MLK4) is overexpressed in MES but not proneural (PN) GSCs.  Silencing MLK4 suppresses self-renewal, motility, tumorigenesis, and radioresistance of MES GSCs via a loss of the MES signature.  MLK4 binds and phosphorylates the NF-κB regulator IKKα, leading to activation of NF-κB signaling in GSCs.  MLK4 expression is inversely correlated with patient prognosis in MES, but not PN high-grade gliomas.  Collectively, our results uncover MLK4 as an upstream regulator of NF-κB signaling and a potential mol. target for the MES subtype of glioblastomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHRhj7RTa7mbVg90H21EOLACvtfcHk0lgb3kww0IHe_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitlSgsr8%253D&md5=9e2255918a33b2abae6b3234655883eb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.01.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.01.005%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DEzhilarasan%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DE.%26aulast%3DGallego-Perez%26aufirst%3DD.%26aulast%3DSparks%26aufirst%3DA.%26aulast%3DTaylor%26aufirst%3DD.%26aulast%3DLadner%26aufirst%3DK.%26aulast%3DFuruta%26aufirst%3DT.%26aulast%3DSabit%26aufirst%3DH.%26aulast%3DChhipa%26aufirst%3DR.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DMohyeldin%26aufirst%3DA.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DKurozumi%26aufirst%3DK.%26aulast%3DKuroiwa%26aufirst%3DT.%26aulast%3DIwata%26aufirst%3DR.%26aulast%3DAsai%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DSulman%26aufirst%3DE.%2BP.%26aulast%3DCheng%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DL.%2BJ.%26aulast%3DNakada%26aufirst%3DM.%26aulast%3DGuttridge%26aufirst%3DD.%26aulast%3DDasGupta%26aufirst%3DB.%26aulast%3DGoidts%26aufirst%3DV.%26aulast%3DBhat%26aufirst%3DK.%2BP.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DSerine%252FThreonine%2520Kinase%2520MLK4%2520Determines%2520Mesenchymal%2520Identity%2520in%2520Glioma%2520Stem%2520Cells%2520in%2520an%2520NF-%25CE%25BAB-dependent%2520Manner%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D201%26epage%3D213%26doi%3D10.1016%2Fj.ccell.2016.01.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Prashant S. Kharkar</span>. </span><span class="cited-content_cbyCitation_article-title">Cancer Stem Cell (CSC) Inhibitors in Oncology—A Promise for a Better Therapeutic Outcome: State of the Art and Future Perspectives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15279-15307. <a href="https://doi.org/10.1021/acs.jmedchem.0c01336" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01336%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCancer%252BStem%252BCell%252B%252528CSC%252529%252BInhibitors%252Bin%252BOncology%2525E2%252580%252594A%252BPromise%252Bfor%252Ba%252BBetter%252BTherapeutic%252BOutcome%25253A%252BState%252Bof%252Bthe%252BArt%252Band%252BFuture%252BPerspectives%26aulast%3DKharkar%26aufirst%3DPrashant%2BS.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31072020%26date%3D16122020%26volume%3D63%26issue%3D24%26spage%3D15279%26epage%3D15307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yifan Li, Kaichen Shu, Ping Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peipei Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Selective C-5 Oxidative Radical Silylation of Imidazopyridines Promoted by Lewis Acid. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (16)
                                     , 6304-6307. <a href="https://doi.org/10.1021/acs.orglett.0c02131" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c02131</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c02131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c02131%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSelective%252BC-5%252BOxidative%252BRadical%252BSilylation%252Bof%252BImidazopyridines%252BPromoted%252Bby%252BLewis%252BAcid%26aulast%3DLi%26aufirst%3DYifan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28062020%26date%3D03082020%26volume%3D22%26issue%3D16%26spage%3D6304%26epage%3D6307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lorenzo  Flori</span>, <span class="hlFld-ContribAuthor ">Giovanni  Petrarolo</span>, <span class="hlFld-ContribAuthor ">Simone  Brogi</span>, <span class="hlFld-ContribAuthor ">Concettina  La Motta</span>, <span class="hlFld-ContribAuthor ">Lara  Testai</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Calderone</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel SIRT1 activators endowed with cardioprotective profile. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Pharmaceutical Sciences</span><span> <strong>2021,</strong> <em>165 </em>, 105930. <a href="https://doi.org/10.1016/j.ejps.2021.105930" title="DOI URL">https://doi.org/10.1016/j.ejps.2021.105930</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejps.2021.105930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejps.2021.105930%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Pharmaceutical%2520Sciences%26atitle%3DIdentification%252Bof%252Bnovel%252BSIRT1%252Bactivators%252Bendowed%252Bwith%252Bcardioprotective%252Bprofile%26aulast%3DFlori%26aufirst%3DLorenzo%26date%3D2021%26volume%3D165%26spage%3D105930" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pankaj  Sharma</span>, <span class="hlFld-ContribAuthor ">Chris  LaRosa</span>, <span class="hlFld-ContribAuthor ">Janet  Antwi</span>, <span class="hlFld-ContribAuthor ">Rajgopal  Govindarajan</span>, <span class="hlFld-ContribAuthor ">Karl A.  Werbovetz</span>. </span><span class="cited-content_cbyCitation_article-title">Imidazoles as Potential Anticancer Agents: An Update on Recent Studies. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (14)
                                     , 4213. <a href="https://doi.org/10.3390/molecules26144213" title="DOI URL">https://doi.org/10.3390/molecules26144213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26144213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26144213%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DImidazoles%252Bas%252BPotential%252BAnticancer%252BAgents%25253A%252BAn%252BUpdate%252Bon%252BRecent%252BStudies%26aulast%3DSharma%26aufirst%3DPankaj%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D14%26spage%3D4213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandt C.  Huddle</span>, <span class="hlFld-ContribAuthor ">Edward  Grimley</span>, <span class="hlFld-ContribAuthor ">Mikhail  Chtcherbinine</span>, <span class="hlFld-ContribAuthor ">Cameron D.  Buchman</span>, <span class="hlFld-ContribAuthor ">Cyrus  Takahashi</span>, <span class="hlFld-ContribAuthor ">Bikash  Debnath</span>, <span class="hlFld-ContribAuthor ">Stacy C.  McGonigal</span>, <span class="hlFld-ContribAuthor ">Shuai  Mao</span>, <span class="hlFld-ContribAuthor ">Siwei  Li</span>, <span class="hlFld-ContribAuthor ">Jeremy  Felton</span>, <span class="hlFld-ContribAuthor ">Shu  Pan</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Duxin  Sun</span>, <span class="hlFld-ContribAuthor ">Nouri  Neamati</span>, <span class="hlFld-ContribAuthor ">Ronald J.  Buckanovich</span>, <span class="hlFld-ContribAuthor ">Thomas D.  Hurley</span>, <span class="hlFld-ContribAuthor ">Scott D.  Larsen</span>. </span><span class="cited-content_cbyCitation_article-title">Development of 2,5-dihydro-4H-pyrazolo[3,4-d]pyrimidin-4-one inhibitors of aldehyde dehydrogenase 1A (ALDH1A) as potential adjuncts to ovarian cancer chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>211 </em>, 113060. <a href="https://doi.org/10.1016/j.ejmech.2020.113060" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.113060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.113060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.113060%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252B2%25252C5-dihydro-4H-pyrazolo%25255B3%25252C4-d%25255Dpyrimidin-4-one%252Binhibitors%252Bof%252Baldehyde%252Bdehydrogenase%252B1A%252B%252528ALDH1A%252529%252Bas%252Bpotential%252Badjuncts%252Bto%252Bovarian%252Bcancer%252Bchemotherapy%26aulast%3DHuddle%26aufirst%3DBrandt%2BC.%26date%3D2021%26volume%3D211%26spage%3D113060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bingyan  Li</span>, <span class="hlFld-ContribAuthor ">Kang  Yang</span>, <span class="hlFld-ContribAuthor ">Dailin  Liang</span>, <span class="hlFld-ContribAuthor ">Cheng  Jiang</span>, <span class="hlFld-ContribAuthor ">Zonghui  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and development of selective aldehyde dehydrogenase 1A1 (ALDH1A1) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112940. <a href="https://doi.org/10.1016/j.ejmech.2020.112940" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112940</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112940%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252Bdevelopment%252Bof%252Bselective%252Baldehyde%252Bdehydrogenase%252B1A1%252B%252528ALDH1A1%252529%252Binhibitors%26aulast%3DLi%26aufirst%3DBingyan%26date%3D2021%26volume%3D209%26spage%3D112940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edoardo L. M.  Gelardi</span>, <span class="hlFld-ContribAuthor ">Giorgia  Colombo</span>, <span class="hlFld-ContribAuthor ">Francesca  Picarazzi</span>, <span class="hlFld-ContribAuthor ">Davide M.  Ferraris</span>, <span class="hlFld-ContribAuthor ">Andrea  Mangione</span>, <span class="hlFld-ContribAuthor ">Giovanni  Petrarolo</span>, <span class="hlFld-ContribAuthor ">Eleonora  Aronica</span>, <span class="hlFld-ContribAuthor ">Menico  Rizzi</span>, <span class="hlFld-ContribAuthor ">Mattia  Mori</span>, <span class="hlFld-ContribAuthor ">Concettina  La Motta</span>, <span class="hlFld-ContribAuthor ">Silvia  Garavaglia</span>. </span><span class="cited-content_cbyCitation_article-title">A Selective Competitive Inhibitor of Aldehyde Dehydrogenase 1A3 Hinders Cancer Cell Growth, Invasiveness and Stemness In Vitro. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (2)
                                     , 356. <a href="https://doi.org/10.3390/cancers13020356" title="DOI URL">https://doi.org/10.3390/cancers13020356</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13020356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13020356%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DA%252BSelective%252BCompetitive%252BInhibitor%252Bof%252BAldehyde%252BDehydrogenase%252B1A3%252BHinders%252BCancer%252BCell%252BGrowth%25252C%252BInvasiveness%252Band%252BStemness%252BIn%252BVitro%26aulast%3DGelardi%26aufirst%3DEdoardo%2BL.%2BM.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D2%26spage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luca  Quattrini</span>, <span class="hlFld-ContribAuthor ">Maria  Sadiq</span>, <span class="hlFld-ContribAuthor ">Giovanni  Petrarolo</span>, <span class="hlFld-ContribAuthor ">Norman J.  Maitland</span>, <span class="hlFld-ContribAuthor ">Fiona M.  Frame</span>, <span class="hlFld-ContribAuthor ">Klaus  Pors</span>, <span class="hlFld-ContribAuthor ">Concettina  La Motta</span>. </span><span class="cited-content_cbyCitation_article-title">Aldehyde Dehydrogenases and Prostate Cancer: Shedding Light on Isoform Distribution to Reveal Druggable Target. </span><span class="cited-content_cbyCitation_journal-name">Biomedicines</span><span> <strong>2020,</strong> <em>8 </em>
                                    (12)
                                     , 569. <a href="https://doi.org/10.3390/biomedicines8120569" title="DOI URL">https://doi.org/10.3390/biomedicines8120569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biomedicines8120569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiomedicines8120569%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicines%26atitle%3DAldehyde%252BDehydrogenases%252Band%252BProstate%252BCancer%25253A%252BShedding%252BLight%252Bon%252BIsoform%252BDistribution%252Bto%252BReveal%252BDruggable%252BTarget%26aulast%3DQuattrini%26aufirst%3DLuca%26date%3D2020%26date%3D2020%26volume%3D8%26issue%3D12%26spage%3D569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0013.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Temozolomide and Aldehyde Dehydrogenase Inhibitors under Investigation</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overall monomer structure of human ALDH1A3 in complex with <b>3a</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>, PDB deposition name: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11">GA11</a>). (A) Ribbon representation of the overall structure of the monomer of ALDH1A3. Each monomer consists of three domains: the NAD<sup>+</sup> binding-domain, the catalytic-domain, and the oligomerization domain. The ligands <b>3a</b> and NAD<sup>+</sup> are shown as yellow and gray sticks, respectively. (B) Electron density map SA-omit (2mFo-DFc) of <b>3a</b>, in gray, contoured at 1 sigma. (C) Superimposition between electron density map of <b>3a</b> SA-omit (2mFo-DFc), in gray, and <b>3a</b> 2mFo-DFc, in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Stick representations of the possible docking poses of compound <b>3a</b> into the binding site of ALDH1A3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>). The binding site of ALDH1A3 is represented as green ribbons. Residues relevant for the inhibitor binding have been labeled, and their side chains are shown as green sticks.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Representation of the main interactions between <b>3a</b> and the ALDH1A3 binding site (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>). Buried surface areas of monomer A is colored in forest green and the ligand <b>3a</b> is shown as yellow stick. Side chains of key residues, involved in <b>3a</b> binding, are shown in forest green sticks. E135, F131, G136, L185, W189, A473, L471, and Y472 stabilize through π–π stacking and hydrophobic interactions the <b>3a</b> phenyl rings. The carbonyl oxygen of F131 interacts with N1 of <b>3a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of ALDH1A binding-sites conformations. Superposition ribbon representation of monomer overall structures of ALDH1A3 in forest green (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>), ALDH1A2 in orange (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5g">6B5g</a>), and ALDH1A1 in cyan (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WB9">4WB9</a>). The three structures maintain the same arrangement. The LigPlus representation of the complex <b>3a</b>-ALDH1A3 monomer A highlights all the eight amino acids involved in <b>3a</b> binding site. The ALDH1A3 amino acid residues circled in green are not conserved in ALDH1A1 and ALDH1A2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Superposition overview of different amino acids involved in <b>3a</b>-ALDH1A3 binding. Side chains of key residues involved in the binding (F131, E135, Y472, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>) are shown in forest green sticks. The correspondent residues not conserved of ALDH1A2 (Y137, Q141, N478, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5G">6B5G</a>) and of ALDH1A1 (L120, A124, S461, PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WB9">4WB9</a>) are represented by orange sticks and cyan sticks, respectively. In the sequence overlapping of the three isoenzymes, the residues not conserved are circled in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 6-Substituted-imidazo[1,2-<i>a</i>]pyridine Derivatives <b>3a–k,n–v</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 2-bromo-1-(4-(3)substitutedphenyl)ethan-1-one, K<sub>2</sub>CO<sub>3</sub> Δ; (ii) substitued-phenylboronic acid, Pd(OAc)<sub>2;</sub> PPh<sub>3</sub> K<sub>2</sub>CO<sub>3</sub> Δ.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 6-Substituted-imidazo[<b>1</b>,2-<i>a</i>]pyridine Derivatives <b>31,m</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 4-chlorophenylboronic acid, Pd(OAc)<sub>2,</sub> PPh<sub>3,</sub> K<sub>2</sub>CO<sub>3</sub> Δ 2-bromo-1-(4-substitutedphenyl)ethan-1-one, K<sub>2</sub>CO<sub>3</sub> Δ.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of 7-Substituted-imidazo[1,2-<i>a</i>]pyridine Derivatives <b>7a–i</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) 2-bromo-l-(4-(3)substitutedphenyl)ethan-l-one, K<sub>2</sub>CO<sub>3</sub> Δ; (ii) substitued-phenylboronic acid, Pd(OAc)<sub>2,</sub> PPh<sub>3,</sub> K<sub>2</sub>CO<sub>3</sub> Δ.</p></p></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Predicted binding mode and MD simulations of acetaldehyde in the <b>3a</b>-ALDH1A3 crystallographic complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W">6S6W</a>). (A) Obtained docking poses of acetaldehyde, represented as sticks and either cyan or yellow balls. (B, C) Plots of the distances between acetaldehyde and the catalytic C314 along MD trajectories starting from the docking poses, represented in cyan and yellow respectively, indicated by black arrows. MD simulations on the cyan pose were stopped after around 150 ns, as the small molecule detaches from the ALDH1A3 catalytic site and moves to the bulk solvent.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/medium/jm9b01910_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Overall monomer structure of human ALDH1A3 in complex with <b>3q</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5">6TE5</a>, PDB deposition name <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43">LQ43</a>). (A) Ribbon representation of the overall structure of the monomer of ALDH1A3. The LigPlus representation of the complex <b>3q</b>-ALDH1A3 monomer B highlights all the amino acids involved in the binding. The ALDH1A3 amino acid residues circled in green are not conserved in ALDH1A1 and ALDH1A2. (B) Compound <b>3q</b> into the enzyme binding site. The ligand is shown as cyan stick and the side chains of key residues, involved in the binding, are shown in forest green sticks. The electron density map (2mFo-dFc) countered at 1.0 sigma, used to determine the presence of <b>3q</b> into the enzyme binding site, is in blue. (C) Electron density map SA-omit (2mFo-dFc) of <b>3q</b>, in gray, contoured at 1 sigma. (D) Superimposition between electron density map of <b>3q</b> SA-omit (2mFo-dFc), in gray, and <b>3q</b> 2mFo-dFc, in blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-9/acs.jmedchem.9b01910/20200507/images/large/jm9b01910_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01910&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84353" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84353" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Batash, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asna, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaffer, M.</span></span> <span> </span><span class="NLM_article-title">Glioblastoma Multiforme, Diagnosis and Treatment; Recent Literature Review</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3002</span>– <span class="NLM_lpage">3009</span>, <span class="refDoi"> DOI: 10.2174/0929867324666170516123206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.2174%2F0929867324666170516123206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=28521700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sru7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=3002-3009&author=R.+Batashauthor=N.+Asnaauthor=P.+Schafferauthor=N.+Francisauthor=M.+Schaffer&title=Glioblastoma+Multiforme%2C+Diagnosis+and+Treatment%3B+Recent+Literature+Review&doi=10.2174%2F0929867324666170516123206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Glioblastoma Multiforme, Diagnosis and Treatment; Recent Literature Review</span></div><div class="casAuthors">Batash, Ron; Asna, Noam; Schaffer, Pamela; Francis, Nicole; Schaffer, Moshe</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">3002-3009</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Background: Glioblastoma multiforme (GBM) is the most common malignant primary brain tumor, with an incidence of 3.19 cases per 100,000 person years and remarkably poor prognosis showing a 5-yr survival rate of 4-5%, and only a 26-33% survival rate at 2 years in clin. trials. </P><P> Objective: In this paper, we review the different types of treatment modalities based on the relevant databases.  Methods of diagnosis will be described briefly. </P><P> Method: Systemic compilation of the relevant literature. </P><P> Results & Conclusion: Today's treatments cannot cure GBM patients but only extend their overall survival.  The use of chemoradiation, immunotherapy, and radio sensitizers as an adjuvant therapy cannot reduce the high rates of recurrence within a few months after treatment.  Radiotherapy will remain the backbone of the treatment but new treatment modalities must be developed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7b7aMyW_Nc7Vg90H21EOLACvtfcHk0ljx19mMJCVn3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sru7fN&md5=65623173dc9b1aa8e7c8ca62d4372c12</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F0929867324666170516123206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867324666170516123206%26sid%3Dliteratum%253Aachs%26aulast%3DBatash%26aufirst%3DR.%26aulast%3DAsna%26aufirst%3DN.%26aulast%3DSchaffer%26aufirst%3DP.%26aulast%3DFrancis%26aufirst%3DN.%26aulast%3DSchaffer%26aufirst%3DM.%26atitle%3DGlioblastoma%2520Multiforme%252C%2520Diagnosis%2520and%2520Treatment%253B%2520Recent%2520Literature%2520Review%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2017%26volume%3D24%26spage%3D3002%26epage%3D3009%26doi%3D10.2174%2F0929867324666170516123206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delgado-López, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corrales-García, E. M.</span></span> <span> </span><span class="NLM_article-title">Survival in Glioblastoma: a Review on the Impact of Treatment Modalities</span>. <i>Clin. Transl. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1062</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1007/s12094-016-1497-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1007%2Fs12094-016-1497-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26960561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVygsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=1062-1071&author=P.+D.+Delgado-L%C3%B3pezauthor=E.+M.+Corrales-Garc%C3%ADa&title=Survival+in+Glioblastoma%3A+a+Review+on+the+Impact+of+Treatment+Modalities&doi=10.1007%2Fs12094-016-1497-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Survival in glioblastoma: a review on the impact of treatment modalities</span></div><div class="casAuthors">Delgado-Lopez, P. D.; Corrales-Garcia, E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical and Translational Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1062-1071</span>CODEN:
                <span class="NLM_cas:coden">CTOLAM</span>;
        ISSN:<span class="NLM_cas:issn">1699-048X</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag Italia S.r.l.</span>)
        </div><div class="casAbstract">Glioblastoma (GBM) is the most common and lethal tumor of the central nervous system.  The natural history of treated GBM remains very poor with 5-yr survival rates of 5 %.  Survival has not significantly improved over the last decades.  Currently, the best that can be offered is a modest 14-mo overall median survival in patients undergoing max. safe resection plus adjuvant chemoradiotherapy.  Prognostic factors involved in survival include age, performance status, grade, specific markers (MGMT methylation, mutation of IDH1, IDH2 or TERT, 1p19q codeletion, overexpression of EGFR, etc.) and, likely, the extent of resection.  Certain adjuncts to surgery, esp. cortical mapping and 5-ALA fluorescence, favor higher rates of gross total resection with apparent pos. impact on survival.  Recurrent tumors can be offered re-intervention, participation in clin. trials, anti-angiogenic agent or local elec. field therapy, without an evident impact on survival.  Mol.-targeted therapies, immunotherapy and gene therapy are promising tools currently under research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcVrZshktqKbVg90H21EOLACvtfcHk0ljx19mMJCVn3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVygsrc%253D&md5=5bcfb0aebc622d839a9a97122db846db</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs12094-016-1497-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12094-016-1497-x%26sid%3Dliteratum%253Aachs%26aulast%3DDelgado-L%25C3%25B3pez%26aufirst%3DP.%2BD.%26aulast%3DCorrales-Garc%25C3%25ADa%26aufirst%3DE.%2BM.%26atitle%3DSurvival%2520in%2520Glioblastoma%253A%2520a%2520Review%2520on%2520the%2520Impact%2520of%2520Treatment%2520Modalities%26jtitle%3DClin.%2520Transl.%2520Oncol.%26date%3D2016%26volume%3D18%26spage%3D1062%26epage%3D1071%26doi%3D10.1007%2Fs12094-016-1497-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meacham, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, S. J.</span></span> <span> </span><span class="NLM_article-title">Tumour Heterogeneity and Cancer Cell Plasticity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>501</i></span>,  <span class="NLM_fpage">328</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nature12624</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnature12624" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=24048065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOnurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=501&publication_year=2013&pages=328-337&author=C.+E.+Meachamauthor=S.+J.+Morrison&title=Tumour+Heterogeneity+and+Cancer+Cell+Plasticity&doi=10.1038%2Fnature12624"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor heterogeneity and cancer cell plasticity</span></div><div class="casAuthors">Meacham, Corbin E.; Morrison, Sean J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">501</span>
        (<span class="NLM_cas:issue">7467</span>),
    <span class="NLM_cas:pages">328-337</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Phenotypic and functional heterogeneity arise among cancer cells within the same tumor as a consequence of genetic change, environmental differences and reversible changes in cell properties.  Some cancers also contain a hierarchy in which tumorigenic cancer stem cells differentiate into non-tumorigenic progeny.  However, it remains unclear what fraction of cancers follow the stem-cell model and what clin. behaviors the model explains.  Studies using lineage tracing and deep sequencing could have implications for the cancer stem-cell model and may help to det. the extent to which it accounts for therapy resistance and disease progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdbWv3pYPK67Vg90H21EOLACvtfcHk0lim5kMIn3OkzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOnurjN&md5=0d4aa5a83725d8cc272d1c19a9eafba9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature12624&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12624%26sid%3Dliteratum%253Aachs%26aulast%3DMeacham%26aufirst%3DC.%2BE.%26aulast%3DMorrison%26aufirst%3DS.%2BJ.%26atitle%3DTumour%2520Heterogeneity%2520and%2520Cancer%2520Cell%2520Plasticity%26jtitle%3DNature%26date%3D2013%26volume%3D501%26spage%3D328%26epage%3D337%26doi%3D10.1038%2Fnature12624" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Tumorigenesis: it Takes a Village</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1038/nrc3971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnrc3971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26156638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyksr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=473-483&author=D.+P.+Tabassumauthor=K.+Polyak&title=Tumorigenesis%3A+it+Takes+a+Village&doi=10.1038%2Fnrc3971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tumorigenesis: it takes a village</span></div><div class="casAuthors">Tabassum, Doris P.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">473-483</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is widely accepted that most cancers exhibit some degree of intratumor heterogeneity, we are far from understanding the dynamics that operate among subpopulations within tumors.  There is growing evidence that cancer cells behave as communities, and increasing attention is now being directed towards the cooperative behavior of subclones that can influence disease progression.  As expected, these interactions can add a greater layer of complexity to therapeutic interventions in heterogeneous tumors, often leading to a poor prognosis.  In this Review, we highlight studies that demonstrate such interactions in cancer and postulate ways to overcome them with better-designed therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocd11y6T1krrVg90H21EOLACvtfcHk0lim5kMIn3OkzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyksr3F&md5=1871f88a05e0990d6193e4985fc5fe6e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc3971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3971%26sid%3Dliteratum%253Aachs%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DTumorigenesis%253A%2520it%2520Takes%2520a%2520Village%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D473%26epage%3D483%26doi%3D10.1038%2Fnrc3971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pattabiraman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Tackling the Cancer Stem Cells - What Challenges Do They Pose?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">512</span>, <span class="refDoi"> DOI: 10.1038/nrd4253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnrd4253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=24981363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOisbzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=497-512&author=D.+R.+Pattabiramanauthor=R.+A.+Weinberg&title=Tackling+the+Cancer+Stem+Cells+-+What+Challenges+Do+They+Pose%3F&doi=10.1038%2Fnrd4253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the cancer stem cells - what challenges do they pose?</span></div><div class="casAuthors">Pattabiraman, Diwakar R.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">497-512</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since their identification in 1994, cancer stem cells (CSCs) have been objects of intensive study.  Their properties and mechanisms of formation have become a major focus of current cancer research, in part because of their enhanced ability to initiate and drive tumor growth and their intrinsic resistance to conventional therapeutics.  The discovery that activation of the epithelial-to-mesenchymal transition (EMT) program in carcinoma cells can give rise to cells with stem-like properties has provided one possible mechanism explaining how CSCs arise and presents a possible avenue for their therapeutic manipulation.  Here we address recent developments in CSC research, focusing on carcinomas that are able to undergo EMT.  We discuss the signalling pathways that create these cells, cell-intrinsic mechanisms that could be exploited for selective elimination or induction of their differentiation, and the role of the tumor microenvironment in sustaining them.  Finally, we propose ways to use our current knowledge of the complex biol. of CSCs to design novel therapies to eliminate them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHXcwOZJN_67Vg90H21EOLACvtfcHk0lim5kMIn3OkzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOisbzI&md5=db211f41b19198981cb37487b7a9abc3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4253%26sid%3Dliteratum%253Aachs%26aulast%3DPattabiraman%26aufirst%3DD.%2BR.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DTackling%2520the%2520Cancer%2520Stem%2520Cells%2520-%2520What%2520Challenges%2520Do%2520They%2520Pose%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D497%26epage%3D512%26doi%3D10.1038%2Fnrd4253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squire, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hide, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henkelman, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusimano, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirks, P. B.</span></span> <span> </span><span class="NLM_article-title">Identification of Human Brain Tumour Initiating Cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>432</i></span>,  <span class="NLM_fpage">396</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1038/nature03128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fnature03128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=15549107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=396-401&author=S.+K.+Singhauthor=C.+Hawkinsauthor=I.+D.+Clarkeauthor=J.+A.+Squireauthor=J.+Bayaniauthor=T.+Hideauthor=R.+M.+Henkelmanauthor=M.+D.+Cusimanoauthor=P.+B.+Dirks&title=Identification+of+Human+Brain+Tumour+Initiating+Cells&doi=10.1038%2Fnature03128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of human brain tumor initiating cells</span></div><div class="casAuthors">Singh, Sheila K.; Hawkins, Cynthia; Clarke, Ian D.; Squire, Jeremy A.; Bayani, Jane; Hide, Takuichiro; Henkelman, R. Mark; Cusimano, Michael D.; Dirks, Peter B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">396-401</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cancer stem cell (CSC) hypothesis suggests that neoplastic clones are maintained exclusively by a rare fraction of cells with stem cell properties.  Although the existence of CSCs in human leukemia is established, little evidence exists for CSCs in solid tumors, except for breast cancer.  Recently, the authors prospectively isolated a CD133+ cell subpopulation from human brain tumors that exhibited stem cell properties in vitro.  However, the true measures of CSCs are their capacity for self renewal and exact recapitulation of the original tumor.  Here the authors report the development of a xenograft assay that identified human brain tumor initiating cells that initiate tumors in vivo.  Only the CD133+ brain tumor fraction contains cells that are capable of tumor initiation in NOD-SCID (non-obese diabetic, severe combined immunodeficient) mouse brains.  Injection of as few as 100 CD133+ cells produced a tumor that could be serially transplanted and was a phenocopy of the patient's original tumor, whereas injection of 105 CD133- cells engrafted but did not cause a tumor.  Thus, the identification of brain tumor initiating cells provides insights into human brain tumor pathogenesis, giving strong support for the CSC hypothesis as the basis for many solid tumors, and establishes a previously unidentified cellular target for more effective cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK8T42M28y7bVg90H21EOLACvtfcHk0ljTvbCuMjMlIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbY%253D&md5=f74762c020a9304ee683b24fb6e35919</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature03128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03128%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DHawkins%26aufirst%3DC.%26aulast%3DClarke%26aufirst%3DI.%2BD.%26aulast%3DSquire%26aufirst%3DJ.%2BA.%26aulast%3DBayani%26aufirst%3DJ.%26aulast%3DHide%26aufirst%3DT.%26aulast%3DHenkelman%26aufirst%3DR.%2BM.%26aulast%3DCusimano%26aufirst%3DM.%2BD.%26aulast%3DDirks%26aufirst%3DP.%2BB.%26atitle%3DIdentification%2520of%2520Human%2520Brain%2520Tumour%2520Initiating%2520Cells%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D396%26epage%3D401%26doi%3D10.1038%2Fnature03128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ginestier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diebel, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korkaya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicinski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabaud, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charafe-Jauffret, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnbaum, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dontu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicha, M. S.</span></span> <span> </span><span class="NLM_article-title">CXCR1 Blockade Selectively Targets Human Breast Cancer Stem Cells in Vitro and in Xenografts</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1172/JCI39397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1172%2FJCI39397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=20051626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOktLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2010&pages=485-497&author=C.+Ginestierauthor=S.+Liuauthor=M.+E.+Diebelauthor=H.+Korkayaauthor=M.+Luoauthor=M.+Brownauthor=J.+Wicinskiauthor=O.+Cabaudauthor=E.+Charafe-Jauffretauthor=D.+Birnbaumauthor=J.+L.+Guanauthor=G.+Dontuauthor=M.+S.+Wicha&title=CXCR1+Blockade+Selectively+Targets+Human+Breast+Cancer+Stem+Cells+in+Vitro+and+in+Xenografts&doi=10.1172%2FJCI39397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">CXCR1 blockade selectively targets human breast cancer stem cells in vitro and in xenografts</span></div><div class="casAuthors">Ginestier, Christophe; Liu, Suling; Diebel, Mark E.; Korkaya, Hasan; Luo, Ming; Brown, Marty; Wicinski, Julien; Cabaud, Olivier; Charafe-Jauffret, Emmanuelle; Birnbaum, Daniel; Guan, Jun-Lin; Dontu, Gabriela; Wicha, Max S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">485-497</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Recent evidence suggests that breast cancer and other solid tumors possess a rare population of cells capable of extensive self-renewal that contribute to metastasis and treatment resistance.  We report here the development of a strategy to target these breast cancer stem cells (CSCs) through blockade of the IL-8 receptor CXCR1.  CXCR1 blockade using either a CXCR1-specific blocking antibody or repertaxin, a small-mol. CXCR1 inhibitor, selectively depleted the CSC population in 2 human breast cancer cell lines in vitro.  Furthermore, this was followed by the induction of massive apoptosis in the bulk tumor population via FASL/FAS signaling.  The effects of CXCR1 blockade on CSC viability and on FASL prodn. were mediated by the FAK/AKT/FOXO3A pathway.  In addn., repertaxin was able to specifically target the CSC population in human breast cancer xenografts, retarding tumor growth and reducing metastasis.  Our data therefore suggest that CXCR1 blockade may provide a novel means of targeting and eliminating breast CSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiolSraN9TPrVg90H21EOLACvtfcHk0ljTvbCuMjMlIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOktLw%253D&md5=cd042dfcf0abbff26b56a8b91245ecaf</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1172%2FJCI39397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI39397%26sid%3Dliteratum%253Aachs%26aulast%3DGinestier%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DDiebel%26aufirst%3DM.%2BE.%26aulast%3DKorkaya%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DWicinski%26aufirst%3DJ.%26aulast%3DCabaud%26aufirst%3DO.%26aulast%3DCharafe-Jauffret%26aufirst%3DE.%26aulast%3DBirnbaum%26aufirst%3DD.%26aulast%3DGuan%26aufirst%3DJ.%2BL.%26aulast%3DDontu%26aufirst%3DG.%26aulast%3DWicha%26aufirst%3DM.%2BS.%26atitle%3DCXCR1%2520Blockade%2520Selectively%2520Targets%2520Human%2520Breast%2520Cancer%2520Stem%2520Cells%2520in%2520Vitro%2520and%2520in%2520Xenografts%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2010%26volume%3D120%26spage%3D485%26epage%3D497%26doi%3D10.1172%2FJCI39397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span> <span> </span><span class="NLM_article-title">Induced Cancer Stem Cells Generated by Radiochemotherapy and their Therapeutic Implications</span>. <i>Oncotarget.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">17301</span>– <span class="NLM_lpage">17312</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.14230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.18632%2Foncotarget.14230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=28038467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A280%3ADC%252BC1c7gvVGisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=17301-17312&author=X.+Chenauthor=R.+Liaoauthor=D.+Liauthor=J.+Sun&title=Induced+Cancer+Stem+Cells+Generated+by+Radiochemotherapy+and+their+Therapeutic+Implications&doi=10.18632%2Foncotarget.14230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Induced cancer stem cells generated by radiochemotherapy and their therapeutic implications</span></div><div class="casAuthors">Chen Xiewan; Liao Rongxia; Chen Xiewan; Li Dezhi; Sun Jianguo</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">17301-17312</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Local and distant recurrence of malignant tumors following radio- and/or chemotherapy correlates with poor prognosis of patients.  Among the reasons for cancer recurrence, preexisting cancer stem cells (CSCs) are considered the most likely cause due to their properties of self-renewal, pluripotency, plasticity and tumorigenicity.  It has been demonstrated that preexisting cancer stem cells derive from normal stem cells and differentiated somatic cells that undergo transformation and dedifferentiation respectively under certain conditions.  However, recent studies have revealed that cancer stem cells can also be induced from non-stem cancer cells by radiochemotherapy, constituting the subpopulation of induced cancer stem cells (iCSCs).  These findings suggest that radiochemotherapy has the side effect of directly transforming non-stem cancer cells into induced cancer stem cells, possibly contributing to tumor recurrence and metastasis.  Therefore, drugs targeting cancer stem cells or preventing dedifferentiation of non-stem cancer cells can be combined with radiochemotherapy to improve its antitumor efficacy.  The current review is to investigate the mechanisms by which induced cancer stem cells are generated by radiochemotherapy and hence provide new strategies for cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT2zDP_L2KDg9-cVm4E2RlLfW6udTcc2eYbdgBN9GYjwbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7gvVGisA%253D%253D&md5=68256fcb48cc0c5ed0596a67062b60db</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.14230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.14230%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DJ.%26atitle%3DInduced%2520Cancer%2520Stem%2520Cells%2520Generated%2520by%2520Radiochemotherapy%2520and%2520their%2520Therapeutic%2520Implications%26jtitle%3DOncotarget.%26date%3D2017%26volume%3D8%26spage%3D17301%26epage%3D17312%26doi%3D10.18632%2Foncotarget.14230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamdan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leboeuf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Bouchtaoui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gapihan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meles, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratajczak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Benedetto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bousquet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janin, A.</span></span> <span> </span><span class="NLM_article-title">Targeting Cancer Stem Cells to Overcome Chemoresistance</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4060</span>, <span class="refDoi"> DOI: 10.3390/ijms19124036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.3390%2Fijms19124036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=4036-4060&author=T.+Nunesauthor=D.+Hamdanauthor=C.+Leboeufauthor=M.+El+Bouchtaouiauthor=G.+Gapihanauthor=T.+T.+Nguyenauthor=S.+Melesauthor=E.+Angeliauthor=P.+Ratajczakauthor=H.+Luauthor=M.+Di+Benedettoauthor=G.+Bousquetauthor=A.+Janin&title=Targeting+Cancer+Stem+Cells+to+Overcome+Chemoresistance&doi=10.3390%2Fijms19124036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fijms19124036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19124036%26sid%3Dliteratum%253Aachs%26aulast%3DNunes%26aufirst%3DT.%26aulast%3DHamdan%26aufirst%3DD.%26aulast%3DLeboeuf%26aufirst%3DC.%26aulast%3DEl%2BBouchtaoui%26aufirst%3DM.%26aulast%3DGapihan%26aufirst%3DG.%26aulast%3DNguyen%26aufirst%3DT.%2BT.%26aulast%3DMeles%26aufirst%3DS.%26aulast%3DAngeli%26aufirst%3DE.%26aulast%3DRatajczak%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DDi%2BBenedetto%26aufirst%3DM.%26aulast%3DBousquet%26aufirst%3DG.%26aulast%3DJanin%26aufirst%3DA.%26atitle%3DTargeting%2520Cancer%2520Stem%2520Cells%2520to%2520Overcome%2520Chemoresistance%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D4036%26epage%3D4060%26doi%3D10.3390%2Fijms19124036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">ALDH1A3: a Marker of Mesenchymal Phenotype in Gliomas Associated with Cell Invasion</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0142856</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0142856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1371%2Fjournal.pone.0142856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26575197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVSjsL7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=W.+Zhangauthor=Y.+Liuauthor=H.+Huauthor=H.+Huangauthor=Z.+Baoauthor=P.+Yangauthor=Y.+Wangauthor=G.+Youauthor=W.+Yanauthor=T.+Jiangauthor=J.+Wangauthor=W.+Zhang&title=ALDH1A3%3A+a+Marker+of+Mesenchymal+Phenotype+in+Gliomas+Associated+with+Cell+Invasion&doi=10.1371%2Fjournal.pone.0142856"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">ALDH1A3: a marker of mesenchymal phenotype in gliomas associated with cell invasion</span></div><div class="casAuthors">Zhang, Wenlong; Liu, Yanwei; Hu, Huimin; Huang, Hua; Bao, Zhaoshi; Yang, Pei; Wang, Yinyan; You, Gan; Yan, Wei; Jiang, Tao; Wang, Jiangfei; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e0142856/1-e0142856/12</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDH) is a family of enzymes including 19 members.  For now, ALDH activity had been wildly used as a marker of cancer stem cells (CSCs).  But biol. functions of relevant isoforms and their clin. applications are still controversial.  Here, we investigate the clin. significance and potential function of ALDH1A3 in gliomas.  By whole-genome transcriptome microarray and mRNA sequencing anal., we compared the expression of ALDH1A3 in high- and low- grade gliomas as well as different mol. subtypes.  Microarray anal. was performed to identify the correlated genes of ALDH1A3.  We further used Gene Ontol. (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways anal. to explore the biol. function of ALDH1A3.  Finally, by mRNA knockdown we revealed the relationship between ALDH1A3 and the ability of tumor invasion.  ALDH1A3 overexpression was significantly assocd. with high grade as well as the higher mortality of gliomas in survival anal.  ALDH1A3 was characteristically highly expressed in Mesenchymal (Mes) subtype gliomas.  Moreover, we found that ALDH1A3 was most relevant to extracellular matrix organization and cell adhesion biol. process, and the ability of tumor invasion was suppressed after ALDH1A3 knockdown in vitro.  In conclusion, ALDH1A3 can serve as a novel marker of Mes phenotype in gliomas with potential clin. prognostic value.  The expression of ALDH1A3 is assocd. with tumor cell invasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouH3jvA9lFdLVg90H21EOLACvtfcHk0lhXxIcDl4KP6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVSjsL7K&md5=4ff850171ba86b69e7b2bd07227c386f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0142856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0142856%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DBao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DG.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DALDH1A3%253A%2520a%2520Marker%2520of%2520Mesenchymal%2520Phenotype%2520in%2520Gliomas%2520Associated%2520with%2520Cell%2520Invasion%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0142856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santana-Santos, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandran, U. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benos, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobol, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, I.</span></span> <span> </span><span class="NLM_article-title">Mesenchymal Glioma Stem Cells are Maintained by Activated Glycolytic Metabolism Involving Aldehyde Dehydrogenase 1A3</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">8644</span>– <span class="NLM_lpage">8649</span>, <span class="refDoi"> DOI: 10.1073/pnas.1221478110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1073%2Fpnas.1221478110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=23650391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSjsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8644-8649&author=P.+Maoauthor=K.+Joshiauthor=J.+Liauthor=S.+H.+Kimauthor=P.+Liauthor=L.+Santana-Santosauthor=S.+Luthraauthor=U.+R.+Chandranauthor=P.+V.+Benosauthor=L.+Smithauthor=M.+Wangauthor=B.+Huauthor=S.+Y.+Chengauthor=R.+W.+Sobolauthor=I.+Nakano&title=Mesenchymal+Glioma+Stem+Cells+are+Maintained+by+Activated+Glycolytic+Metabolism+Involving+Aldehyde+Dehydrogenase+1A3&doi=10.1073%2Fpnas.1221478110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mesenchymal glioma stem cells are maintained by activated glycolytic metabolism involving aldehyde dehydrogenase 1A3</span></div><div class="casAuthors">Mao, Ping; Joshi, Kaushal; Li, Jianfeng; Kim, Sung-Hak; Li, Peipei; Santana-Santos, Lucas; Luthra, Soumya; Chandran, Uma R.; Benos, Panayiotis V.; Smith, Luke; Wang, Maode; Hu, Bo; Cheng, Shi-Yuan; Sobol, Robert W.; Nakano, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8644-8649, S8644/1-S8644/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Tumor heterogeneity of high-grade glioma (HGG) is recognized by four clin. relevant subtypes based on core gene signatures.  However, mol. signaling in glioma stem cells (GSCs) in individual HGG subtypes is poorly characterized.  Here we identified and characterized two mutually exclusive GSC subtypes with distinct dysregulated signaling pathways.  Anal. of mRNA profiles distinguished proneural (PN) from mesenchymal (Mes) GSCs and revealed a pronounced correlation with the corresponding PN or Mes HGGs.  Mes GSCs displayed more aggressive phenotypes in vitro and as intracranial xenografts in mice.  Further, Mes GSCs were markedly resistant to radiation compared with PN GSCs.  The glycolytic pathway, comprising aldehyde dehydrogenase (ALDH) family genes and in particular ALDH1A3, were enriched in Mes GSCs.  Glycolytic activity and ALDH activity were significantly elevated in Mes GSCs but not in PN GSCs.  Expression of ALDH1A3 was also increased in clin. HGG compared with low-grade glioma or normal brain tissue.  Moreover, inhibition of ALDH1 A3 attenuated the growth of Mes but not PN GSCs.  Last, radiation treatment of PN GSCs up-regulated Mes-assocd. markers and down-regulated PN-assocd. markers, whereas inhibition of ALDH1A3 attenuated an irradn.-induced gain of Mes identity in PN GSCs.  Taken together, our data suggest that two subtypes of GSCs, harboring distinct metabolic signaling pathways, represent intertumoral glioma heterogeneity and highlight previously unidentified roles of ALDH1 A3-assocd. signaling that promotes aberrant proliferation of Mes HGGs and GSCs.  Inhibition of ALDH1A3-mediated pathways therefore might provide a promising therapeutic approach for a subset of HGGs with the Mes signature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaISMNS12fzbVg90H21EOLACvtfcHk0lhXxIcDl4KP6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSjsbzE&md5=8896664bcf95f99bba9a928752a47b4d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221478110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221478110%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DP.%26aulast%3DJoshi%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSantana-Santos%26aufirst%3DL.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DChandran%26aufirst%3DU.%2BR.%26aulast%3DBenos%26aufirst%3DP.%2BV.%26aulast%3DSmith%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DCheng%26aufirst%3DS.%2BY.%26aulast%3DSobol%26aufirst%3DR.%2BW.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DMesenchymal%2520Glioma%2520Stem%2520Cells%2520are%2520Maintained%2520by%2520Activated%2520Glycolytic%2520Metabolism%2520Involving%2520Aldehyde%2520Dehydrogenase%25201A3%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D8644%26epage%3D8649%26doi%3D10.1073%2Fpnas.1221478110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waghmare, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coviello, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohyeldin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlyukov, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentim, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chhipa, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasgupta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Motta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kango-Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, I.</span></span> <span> </span><span class="NLM_article-title">FOXD1-ALDH1A3 Signaling is a Determinant for the Self-renewal and Tumorigenicity of Mesenchymal Glioma Stem Cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">7219</span>– <span class="NLM_lpage">7230</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2860</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1158%2F0008-5472.CAN-15-2860" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=27569208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVylsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=7219-7230&author=P.+Chengauthor=J.+Wangauthor=I.+Waghmareauthor=S.+Sartiniauthor=V.+Covielloauthor=Z.+Zhangauthor=S.+H.+Kimauthor=A.+Mohyeldinauthor=M.+S.+Pavlyukovauthor=M.+Minataauthor=C.+L.+Valentimauthor=R.+R.+Chhipaauthor=K.+P.+Bhatauthor=B.+Dasguptaauthor=C.+La+Mottaauthor=M.+Kango-Singhauthor=I.+Nakano&title=FOXD1-ALDH1A3+Signaling+is+a+Determinant+for+the+Self-renewal+and+Tumorigenicity+of+Mesenchymal+Glioma+Stem+Cells&doi=10.1158%2F0008-5472.CAN-15-2860"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">FOXD1-ALDH1A3 Signaling Is a Determinant for the Self-Renewal and Tumorigenicity of Mesenchymal Glioma Stem Cells</span></div><div class="casAuthors">Cheng, Peng; Wang, Jia; Waghmare, Indrayani; Sartini, Stefania; Coviello, Vito; Zhang, Zhuo; Kim, Sung-Hak; Mohyeldin, Ahmed; Pavlyukov, Marat S.; Minata, Mutsuko; Valentim, Claudia L. L.; Chhipa, Rishi Raj; Bhat, Krishna P. L.; Dasgupta, Biplab; La Motta, Concettina; Kango-Singh, Madhuri; Nakano, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7219-7230</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Glioma stem-like cells (GSC) with tumor-initiating activity orchestrate the cellular hierarchy in glioblastoma and engender therapeutic resistance.  Recent work has divided GSC into two subtypes with a mesenchymal (MES) GSC population as the more malignant subtype.  In this study, we identify the FOXD1-ALDH1A3 signaling axis as a determinant of the MES GSC phenotype.  The transcription factor FOXD1 is expressed predominantly in patient-derived cultures enriched with MES, but not with the proneural GSC subtype. shRNA-mediated attenuation of FOXD1 in MES GSC ablates their clonogenicity in vitro and in vivo.  Mechanistically, FOXD1 regulates the transcriptional activity of ALDH1A3, an established functional marker for MES GSC.  Indeed, the functional roles of FOXD1 and ALDH1A3 are likely evolutionally conserved, insofar as RNAi-mediated attenuation of their orthologous genes in Drosophila blocks formation of brain tumors engineered in that species.  In clin. specimens of high-grade glioma, the levels of expression of both FOXD1 and ALDH1A3 are inversely correlated with patient prognosis.  Finally, a novel small-mol. inhibitor of ALDH we developed, termed GA11, displays potent in vivo efficacy when administered systemically in a murine GSC-derived xenograft model of glioblastoma.  Collectively, our findings define a FOXD1-ALDH1A3 pathway in controlling the clonogenic and tumorigenic potential of MES GSC in glioblastoma tumors.  Cancer Res; 76(24); 7219-30. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9_D8WWqs-MbVg90H21EOLACvtfcHk0lhXxIcDl4KP6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVylsrjE&md5=0c8c01e69e88084ec73e2bf7b0a06d3a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2860%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWaghmare%26aufirst%3DI.%26aulast%3DSartini%26aufirst%3DS.%26aulast%3DCoviello%26aufirst%3DV.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DMohyeldin%26aufirst%3DA.%26aulast%3DPavlyukov%26aufirst%3DM.%2BS.%26aulast%3DMinata%26aufirst%3DM.%26aulast%3DValentim%26aufirst%3DC.%2BL.%26aulast%3DChhipa%26aufirst%3DR.%2BR.%26aulast%3DBhat%26aufirst%3DK.%2BP.%26aulast%3DDasgupta%26aufirst%3DB.%26aulast%3DLa%2BMotta%26aufirst%3DC.%26aulast%3DKango-Singh%26aufirst%3DM.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DFOXD1-ALDH1A3%2520Signaling%2520is%2520a%2520Determinant%2520for%2520the%2520Self-renewal%2520and%2520Tumorigenicity%2520of%2520Mesenchymal%2520Glioma%2520Stem%2520Cells%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D7219%26epage%3D7230%26doi%3D10.1158%2F0008-5472.CAN-15-2860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sobreira, T. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlétaz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simões-Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schechtman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweeney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronowicz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laudet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulo de Oliveira, P. S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier-Neto, J.</span></span> <span> </span><span class="NLM_article-title">Structural Shifts of Aldehyde Dehydrogenase Enzymes were Instrumental for the Early Evolution of Retinoid-dependent Axial Patterning in Metazoans</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">226</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1073/pnas.1011223108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1073%2Fpnas.1011223108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=21169504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3MXms1KisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=226-231&author=T.+J.+P.+Sobreiraauthor=F.+Marl%C3%A9tazauthor=M.+Sim%C3%B5es-Costaauthor=D.+Schechtmanauthor=A.+C.+Pereiraauthor=F.+Brunetauthor=S.+Sweeneyauthor=A.+Paniauthor=J.+Aronowiczauthor=C.+J.+Loweauthor=B.+Davidsonauthor=V.+Laudetauthor=M.+Bronnerauthor=P.+S.+L.+Paulo+de+Oliveiraauthor=M.+Schubertauthor=J.+Xavier-Neto&title=Structural+Shifts+of+Aldehyde+Dehydrogenase+Enzymes+were+Instrumental+for+the+Early+Evolution+of+Retinoid-dependent+Axial+Patterning+in+Metazoans&doi=10.1073%2Fpnas.1011223108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structural shifts of aldehyde dehydrogenase enzymes were instrumental for the early evolution of retinoid-dependent axial patterning in metazoans</span></div><div class="casAuthors">Sobreira, Tiago J. P.; Marletaz, Ferdinand; Simoes-Costa, Marcos; Schechtman, Deborah; Pereira, Alexandre C.; Brunet, Frederic; Sweeney, Sarah; Pani, Ariel; Aronowicz, Jochanan; Lowe, Christopher J.; Davidson, Bradley; Laudet, Vincent; Bronner, Marianne; de Oliveiraa, Paulo S. L.; Schubert, Michael; Xavier-Neto, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">226-231, S226/1-S226/22</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aldehyde dehydrogenases (ALDHs) catabolize toxic aldehydes and process the vitamin A-derived retinaldehyde into retinoic acid (RA), a small diffusible mol. and a pivotal chordate morphogen.  In this study, we combine phylogenetic, structural, genomic, and developmental gene expression analyses to examine the evolutionary origins of ALDH substrate preference.  Structural modeling reveals that processing of small aldehydes, such as acetaldehyde, by ALDH2, vs. large aldehydes, including retinaldehyde, by ALDH1A is assocd. with small vs. large substrate entry channels (SECs), resp.  Moreover, we show that metazoan ALDH1s and ALDH2s are members of a single ALDH1/2 clade and that during evolution, eukaryote ALDH1/2s often switched between large and small SECs after gene duplication, transforming constricted channels into wide opened ones and vice versa.  Ancestral sequence reconstructions suggest that during the evolutionary emergence of RA signaling, the ancestral, narrow-channeled metazoan ALDH1/2 gave rise to large ALDH1 channels capable of accommodating bulky aldehydes, such as retinaldehyde, supporting the view that retinoid-dependent signaling arose from ancestral cellular detoxification mechanisms.  Our analyses also indicate that, on a more restricted evolutionary scale, ALDH1 duplicates from invertebrate chordates (amphioxus and ascidian tunicates) underwent switches to smaller and narrower SECs.  When combined with alterations in gene expression, these switches led to neo-functionalization from ALDH1-like roles in embryonic patterning to systemic, ALDH2-like roles, suggesting functional shifts from signaling to detoxification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfJDaWsQicerVg90H21EOLACvtfcHk0ljHCW2ynXZSpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXms1KisQ%253D%253D&md5=225e4c673b52bb162b4c96e509012a4d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1011223108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1011223108%26sid%3Dliteratum%253Aachs%26aulast%3DSobreira%26aufirst%3DT.%2BJ.%2BP.%26aulast%3DMarl%25C3%25A9taz%26aufirst%3DF.%26aulast%3DSim%25C3%25B5es-Costa%26aufirst%3DM.%26aulast%3DSchechtman%26aufirst%3DD.%26aulast%3DPereira%26aufirst%3DA.%2BC.%26aulast%3DBrunet%26aufirst%3DF.%26aulast%3DSweeney%26aufirst%3DS.%26aulast%3DPani%26aufirst%3DA.%26aulast%3DAronowicz%26aufirst%3DJ.%26aulast%3DLowe%26aufirst%3DC.%2BJ.%26aulast%3DDavidson%26aufirst%3DB.%26aulast%3DLaudet%26aufirst%3DV.%26aulast%3DBronner%26aufirst%3DM.%26aulast%3DPaulo%2Bde%2BOliveira%26aufirst%3DP.%2BS.%2BL.%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DXavier-Neto%26aufirst%3DJ.%26atitle%3DStructural%2520Shifts%2520of%2520Aldehyde%2520Dehydrogenase%2520Enzymes%2520were%2520Instrumental%2520for%2520the%2520Early%2520Evolution%2520of%2520Retinoid-dependent%2520Axial%2520Patterning%2520in%2520Metazoans%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D226%26epage%3D231%26doi%3D10.1073%2Fpnas.1011223108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moretti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobol, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garavaglia, S.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human Aldehyde Dehydrogenase 1A3 Complexed with NAD<sup>+</sup> and Retinoic Acid</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">35710</span>– <span class="NLM_lpage">35721</span>, <span class="refDoi"> DOI: 10.1038/srep35710</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2Fsrep35710" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=35710-35721&author=A.+Morettiauthor=J.+Liauthor=S.+Doniniauthor=R.+W.+Sobolauthor=M.+Rizziauthor=S.+Garavaglia&title=Crystal+Structure+of+Human+Aldehyde+Dehydrogenase+1A3+Complexed+with+NAD%2B+and+Retinoic+Acid&doi=10.1038%2Fsrep35710"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsrep35710&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep35710%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DDonini%26aufirst%3DS.%26aulast%3DSobol%26aufirst%3DR.%2BW.%26aulast%3DRizzi%26aufirst%3DM.%26aulast%3DGaravaglia%26aufirst%3DS.%26atitle%3DCrystal%2520Structure%2520of%2520Human%2520Aldehyde%2520Dehydrogenase%25201A3%2520Complexed%2520with%2520NAD%252B%2520and%2520Retinoic%2520Acid%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D19%26spage%3D35710%26epage%3D35721%26doi%3D10.1038%2Fsrep35710" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved Protein-ligand Docking using GOLD</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+Protein-ligand+Docking+using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Å of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of ∼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0ljHCW2ynXZSpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520Protein-ligand%2520Docking%2520using%2520GOLD%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, T. D.</span></span> <span> </span><span class="NLM_article-title">Development of a High-throughput in Vitro Assay to Identify Selective Inhibitors for Human ALDH1A1</span>. <i>Chem.-Biol. Interact.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1016/j.cbi.2014.10.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1016%2Fj.cbi.2014.10.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=25450233" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=2015&pages=29-37&author=C.+A.+Morganauthor=T.+D.+Hurley&title=Development+of+a+High-throughput+in+Vitro+Assay+to+Identify+Selective+Inhibitors+for+Human+ALDH1A1&doi=10.1016%2Fj.cbi.2014.10.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a high-throughput in vitro assay to identify selective inhibitors for human ALDH1A1</span></div><div class="casAuthors">Morgan, Cynthia A.; Hurley, Thomas D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemico-Biological Interactions</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">29-37</span>CODEN:
                <span class="NLM_cas:coden">CBINA8</span>;
        ISSN:<span class="NLM_cas:issn">0009-2797</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">The human aldehyde dehydrogenase (ALDH) superfamily consists of at least 19 enzymes that metabolize endogenous and exogenous aldehydes.  Currently, there are no com. available inhibitors that target ALDH1A1 but have little to no effect on the structurally and functionally similar ALDH2.  Here we present the first human ALDH1A1 structure, as the apo-enzyme and in complex with its cofactor NADH to a resoln. of 1.75 and 2.1 Å, respectfully.  Structural comparisons of the cofactor binding sites in ALDH1A1 with other closely related ALDH enzymes illustrate a high degree of similarity.  In order to minimize discovery of compds. that inhibit both isoenzymes by interfering with their conserved cofactor binding sites, this study reports the use of an in vitro, NAD+-independent, esterase-based high-throughput screen (HTS) of 64,000 compds. to discover novel, selective inhibitors of ALDH1A1.  We describe 256 hits that alter the esterase activity of ALDH1A1.  The effects on aldehyde oxidn. of 67 compds. were further analyzed, with 30 selectively inhibiting ALDH1A1 compared to ALDH2 and ALDH3A1.  One compd. inhibited ALDH1A1 and ALDH2, while another inhibited ALDH1A1, ALDH2, and the more distantly related ALDH3A1.  The results presented here indicate that this in vitro enzyme activity screening protocol successfully identified ALDH1A1 inhibitors with a high degree of isoenzyme selectivity.  The compds. identified via this screen plus the screening methodol. itself represent a starting point for the development of highly potent and selective inhibitors of ALDH1A1 that may be utilized to better understand the role of this enzyme in both normal and disease states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0H8M4JN5aC7Vg90H21EOLACvtfcHk0lgM2IHsP6A4Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvV2gsbzP&md5=64ce33cd18ae04a4ef8796bd9c4474a7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cbi.2014.10.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbi.2014.10.028%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DC.%2BA.%26aulast%3DHurley%26aufirst%3DT.%2BD.%26atitle%3DDevelopment%2520of%2520a%2520High-throughput%2520in%2520Vitro%2520Assay%2520to%2520Identify%2520Selective%2520Inhibitors%2520for%2520Human%2520ALDH1A1%26jtitle%3DChem.-Biol.%2520Interact.%26date%3D2015%26volume%3D234%26spage%3D29%26epage%3D37%26doi%3D10.1016%2Fj.cbi.2014.10.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikles, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amory, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">582</span>– <span class="NLM_lpage">590</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00685</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00685" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=582-590&author=Y.+Chenauthor=J.+Y.+Zhuauthor=K.+H.+Hongauthor=D.+C.+Miklesauthor=G.+I.+Georgauthor=A.+S.+Goldsteinauthor=J.+K.+Amoryauthor=E.+Sch%C3%B6nbrunn&title=Structural+Basis+of+ALDH1A2+Inhibition+by+Irreversible+and+Reversible+Small+Molecule+Inhibitors&doi=10.1021%2Facschembio.7b00685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of ALDH1A2 Inhibition by Irreversible and Reversible Small Molecule Inhibitors</span></div><div class="casAuthors">Chen, Yan; Zhu, Jin-Yi; Hong, Kwon Ho; Mikles, David C.; Georg, Gunda I.; Goldstein, Alex S.; Amory, John K.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">582-590</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymes of the ALDH1A subfamily of aldehyde dehydrogenases are crucial in regulating retinoic acid (RA) signaling and have received attention as potential drug targets.  ALDH1A2 is the primary RA-synthesizing enzyme in mammalian spermatogenesis and is therefore considered a viable drug target for male contraceptive development.  However, only a small no. of ALDH1A2 inhibitors have been reported, and information on the structure of ALDH1A2 was limited to the NAD-liganded enzyme void of substrate or inhibitors.  Herein, the authors describe the mechanism of action of structurally unrelated reversible and irreversible inhibitors of human ALDH1A2 using direct binding studies and x-ray crystallog.  All inhibitors bind to the active sites of tetrameric ALDH1A2.  Compd. WIN18,446 covalently reacts with the side chain of the catalytic residue Cys 320, resulting in a chiral adduct in (R) configuration.  The covalent adduct directly affects the neighboring NAD mol., which assumes a contracted conformation suboptimal for the dehydrogenase reaction.  The reversible inhibitors interact predominantly through direct hydrogen bonding interactions with residues in the vicinity of Cys 320 without affecting NAD.  Upon interaction with inhibitors, a large flexible loop assumes regular structure, thereby shielding the active site from solvent.  The precise knowledge of the binding modes provides a new framework for the rational design of novel inhibitors of ALDH1A2 with improved potency and selectivity profiles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprC4rddNQ7u7Vg90H21EOLACvtfcHk0lgM2IHsP6A4Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOgt7bK&md5=dff6eef4a876a603048f3ccd53f60dee</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00685%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DHong%26aufirst%3DK.%2BH.%26aulast%3DMikles%26aufirst%3DD.%2BC.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DGoldstein%26aufirst%3DA.%2BS.%26aulast%3DAmory%26aufirst%3DJ.%2BK.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DStructural%2520Basis%2520of%2520ALDH1A2%2520Inhibition%2520by%2520Irreversible%2520and%2520Reversible%2520Small%2520Molecule%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D582%26epage%3D590%26doi%3D10.1021%2Facschembio.7b00685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerencser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panka, G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egyed, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uerge, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darvas, F.</span></span> <span> </span><span class="NLM_article-title">Procedure for the Parallel Preparation of 3-Imidazo[1,2-<i>a</i>]pyridin-3-yl Propionic Acid Derivatives Involving Meldrum’s acid</span>. <i>J. Comb. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">530</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1021/cc040101j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc040101j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVWktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=530-538&author=J.+Gerencserauthor=G+Pankaauthor=T.+Nagyauthor=O.+Egyedauthor=G.+Dormanauthor=L.+Uergeauthor=F.+Darvas&title=Procedure+for+the+Parallel+Preparation+of+3-Imidazo%5B1%2C2-a%5Dpyridin-3-yl+Propionic+Acid+Derivatives+Involving+Meldrum%E2%80%99s+acid&doi=10.1021%2Fcc040101j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Procedure for the Parallel Preparation of 3-Imidazo[1,2-a]pyridin-3-ylpropionic Acid Derivatives Involving Meldrum's Acid</span></div><div class="casAuthors">Gerencser, Janos; Panka, Gabor; Nagy, Tamas; Egyed, Orsolya; Dorman, Gyoergy; Uerge, Laszlo; Darvas, Ferenc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">530-538</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors describe here an efficient and versatile method for the prepn. of 3-imidazo[1,2-a]pyridin-3-ylpropionic acids involving, as a key step, a three-component Michael-type reaction.  The extended and validated procedure allowed prepn. of various acids with three diversity points.  E.g., a three-component reaction of imidazo[1,2-a]pyridine, Meldrum's acid, and RCHO [R = substituted Ph, 2-furyl, 2-thienyl, Me2CH, BnCH2] in the presence of a catalytic amt. of DL-proline gave imidazo[1,2-a]pyridines I.  The latter were converted to 3-imidazo[1,2-a]pyridin-3-ylpropionic acids II.  The method was easily adaptable for parallel synthesis and an ∼2000-membered 3-imidazo[1,2-a]pyridin-3-ylpropionic acid amide library was prepd. in a semiautomated manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptft8VWbhkC7Vg90H21EOLACvtfcHk0lgM2IHsP6A4Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVWktro%253D&md5=d20874eace950681314e5ab6740447c6</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fcc040101j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc040101j%26sid%3Dliteratum%253Aachs%26aulast%3DGerencser%26aufirst%3DJ.%26aulast%3DPanka%26aufirst%3DG%26aulast%3DNagy%26aufirst%3DT.%26aulast%3DEgyed%26aufirst%3DO.%26aulast%3DDorman%26aufirst%3DG.%26aulast%3DUerge%26aufirst%3DL.%26aulast%3DDarvas%26aufirst%3DF.%26atitle%3DProcedure%2520for%2520the%2520Parallel%2520Preparation%2520of%25203-Imidazo%255B1%252C2-a%255Dpyridin-3-yl%2520Propionic%2520Acid%2520Derivatives%2520Involving%2520Meldrum%25E2%2580%2599s%2520acid%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D2005%26volume%3D7%26spage%3D530%26epage%3D538%26doi%3D10.1021%2Fcc040101j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bollenbach, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvat, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daubeuf, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yalcin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letellier, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bihel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourguignon, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frossard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, M.</span></span> <span> </span><span class="NLM_article-title">Phenylpyridine-2-ylguanidines and Rigid Mimetics as Novel Inhibitors of TNFα Overproduction: Beneficial Action in Models of Neuropathic Pain and of Acute Lung Inflammation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1016%2Fj.ejmech.2018.01.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=29432948" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCmurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=163-182&author=M.+Bollenbachauthor=E.+Salvatauthor=F.+Daubeufauthor=P.+Wagnerauthor=I.+Yalcinauthor=M.+Humoauthor=B.+Letellierauthor=L.+J.+Beckerauthor=F.+Bihelauthor=J.-J.+Bourguignonauthor=P.+Villaauthor=A.+Obrechtauthor=N.+Frossardauthor=M.+Barrotauthor=M.+Schmitt&title=Phenylpyridine-2-ylguanidines+and+Rigid+Mimetics+as+Novel+Inhibitors+of+TNF%CE%B1+Overproduction%3A+Beneficial+Action+in+Models+of+Neuropathic+Pain+and+of+Acute+Lung+Inflammation&doi=10.1016%2Fj.ejmech.2018.01.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylpyridine-2-ylguanidines and rigid mimetics as novel inhibitors of TNFα overproduction: Beneficial action in models of neuropathic pain and of acute lung inflammation</span></div><div class="casAuthors">Bollenbach, Maud; Salvat, Eric; Daubeuf, Francois; Wagner, Patrick; Yalcin, Ipek; Humo, Muris; Letellier, Baptiste; Becker, Lea J.; Bihel, Frederic; Bourguignon, Jean-Jacques; Villa, Pascal; Obrecht, Adeline; Frossard, Nelly; Barrot, Michel; Schmitt, Martine</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">163-182</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">4-Phenylpyridin-2-yl-guanidine (5b): a new inhibitor of the overprodn. of pro-inflammatory cytokines (TNFα and Il1β) was identified from a high-throughput screening of a chem. library on human peripheral blood mononuclear cells (PBMCs) after LPS stimulation.  Derivs., homologues and rigid mimetics of 5b were designed and synthesized, and their cytotoxicity and ability to inhibit TNFα overprodn. were evaluated.  Among them, compd. 5b and its mimetic 12 (2-aminodihydroquinazoline) showed similar inhibitory activities, and were evaluated in vivo in models of lung inflammation and neuropathic pain in mice.  In particular, compd. 12 proved to be active (5mg/kg, i.p.) in both models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYdDk_3kb9JbVg90H21EOLACvtfcHk0lirx7hyOqxGlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCmurs%253D&md5=7514977ba42f66e83430feb834ff631a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.049%26sid%3Dliteratum%253Aachs%26aulast%3DBollenbach%26aufirst%3DM.%26aulast%3DSalvat%26aufirst%3DE.%26aulast%3DDaubeuf%26aufirst%3DF.%26aulast%3DWagner%26aufirst%3DP.%26aulast%3DYalcin%26aufirst%3DI.%26aulast%3DHumo%26aufirst%3DM.%26aulast%3DLetellier%26aufirst%3DB.%26aulast%3DBecker%26aufirst%3DL.%2BJ.%26aulast%3DBihel%26aufirst%3DF.%26aulast%3DBourguignon%26aufirst%3DJ.-J.%26aulast%3DVilla%26aufirst%3DP.%26aulast%3DObrecht%26aufirst%3DA.%26aulast%3DFrossard%26aufirst%3DN.%26aulast%3DBarrot%26aufirst%3DM.%26aulast%3DSchmitt%26aufirst%3DM.%26atitle%3DPhenylpyridine-2-ylguanidines%2520and%2520Rigid%2520Mimetics%2520as%2520Novel%2520Inhibitors%2520of%2520TNF%25CE%25B1%2520Overproduction%253A%2520Beneficial%2520Action%2520in%2520Models%2520of%2520Neuropathic%2520Pain%2520and%2520of%2520Acute%2520Lung%2520Inflammation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D163%26epage%3D182%26doi%3D10.1016%2Fj.ejmech.2018.01.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michaelis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menten, M. L.</span></span> <span> </span><span class="NLM_article-title">Die Kinetik der Invertinwirkung</span>. <i>Biochemische Zeitschrift.</i> <span class="NLM_year" style="font-weight: bold;">1913</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">369</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADyaC3sXhsFaqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1913&pages=333-369&author=L.+Michaelisauthor=M.+L.+Menten&title=Die+Kinetik+der+Invertinwirkung"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of Invertase Action</span></div><div class="casAuthors">Michaelis, L.; Menten, Maud L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemische Zeitschrift</span>
        (<span class="NLM_cas:date">1913</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">333-69</span>CODEN:
                <span class="NLM_cas:coden">BIZEA2</span>;
        ISSN:<span class="NLM_cas:issn">0366-0753</span>.
    </div><div class="casAbstract">The course of sugar inversion by invertase is consistent with the assumption that saccharose and enzyme unite to form a combination, of which the dissociation const. is 0.0167.  The comp. is labile and breaks up into 1 mol. each of glucose, fructose, and invertin.  Invertin has an affinity for glucose and fructose, as well as for other carbohydrates and higher alcs. (mannitol and glycerol), although in none of these cases is the affinity so great as for saccharose.  The compds. formed with these substs. are not labile and the substs. do not suffer decomp.  They show their combining capacity for the enzyme by the fact that their presence retards the inversion of saccharose by invertin.  The concs. of all the invertin-carbohydrate compds. were calc. according to the law of mass action and the dissociation consts. detd.  The decomp. of the saccharose-invertin compd. being a monomol. reaction, the inversion velocity at any time is directly proportional to the conc. of the compd.  From the above assumptions a differential equation is derived which agrees well with the observed inversion rates of saccharose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkv37PaE45mbVg90H21EOLACvtfcHk0lirx7hyOqxGlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaC3sXhsFaqtw%253D%253D&md5=8ea41914dde2d77f646b002014a1a1dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMichaelis%26aufirst%3DL.%26aulast%3DMenten%26aufirst%3DM.%2BL.%26atitle%3DDie%2520Kinetik%2520der%2520Invertinwirkung%26jtitle%3DBiochemische%2520Zeitschrift.%26date%3D1913%26volume%3D49%26spage%3D333%26epage%3D369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span> <i>SigmaPlot Extract Graphs and Data Analysis (StatSys v12.3)</i>; <span class="NLM_publisher-name">Systat</span>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SigmaPlot+Extract+Graphs+and+Data+Analysis+%28StatSys+v12.3%29%3B+Systat%2C+2020."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSigmaPlot%2520Extract%2520Graphs%2520and%2520Data%2520Analysis%2520%2528StatSys%2520v12.3%2529%26pub%3DSystat%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-32&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lirx7hyOqxGlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D32%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Collaborative
Computational Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 suite: Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0AComputational+Project%2C+Number+4&title=The+CCP4+suite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0li2X0nPrH1PvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>D53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=D53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-likelihood+Method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0li2X0nPrH1PvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3DD53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustyakimov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Towards Automated Crystallographic Structure Refinement with Phenix.refine</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-67&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+Automated+Crystallographic+Structure+Refinement+with+Phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0li2X0nPrH1PvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520Automated%2520Crystallographic%2520Structure%2520Refinement%2520with%2520Phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D67%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0li2X0nPrH1PvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brunger, A. T.</span></span> <span> </span><span class="NLM_article-title">Free R Value: a Novel Statistic Quantity for Assesing the Accuracy of Crystal Structures</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">472</span>– <span class="NLM_lpage">475</span>, <span class="refDoi"> DOI: 10.1038/355472a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1038%2F355472a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=18481394" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A280%3ADC%252BD1czhvFCquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=1992&pages=472-475&author=A.+T.+Brunger&title=Free+R+Value%3A+a+Novel+Statistic+Quantity+for+Assesing+the+Accuracy+of+Crystal+Structures&doi=10.1038%2F355472a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Free R value: a novel statistical quantity for assessing the accuracy of crystal structures</span></div><div class="casAuthors">Brunger A T</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6359</span>),
    <span class="NLM_cas:pages">472-5</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The determination of macromolecular structure by crystallography involves fitting atomic models to the observed diffraction data.  The traditional measure of the quality of this fit, and presumably the accuracy of the model, is the R value.  Despite stereochemical restraints, it is possible to overfit or 'misfit' the diffraction data: an incorrect model can be refined to fairly good R values as several recent examples have shown.  Here I propose a reliable and unbiased indicator of the accuracy of such models.  By analogy with the cross-validation method of testing statistical models I define a statistical quantity (R(free) (T) that measures the agreement between observed and computed structure factor amplitudes for a 'test' set of reflections that is omitted in the modelling and refinement process.  As examples show, there is a high correlation between R(free) (T) and the accuracy of the atomic model phases.  This is useful because experimental phase information is usually inaccurate, incomplete or unavailable.  I expect that R(free) (T) will provide a measure of the information content of recently proposed models of thermal motion and disorder, time-averaging and bulk solvent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQY_IzgfwBLm_h3YQQtAhNufW6udTcc2ebPVMK1ubXYerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1czhvFCquw%253D%253D&md5=ddeaa021753f1208b405e03c1428078f</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2F355472a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F355472a0%26sid%3Dliteratum%253Aachs%26aulast%3DBrunger%26aufirst%3DA.%2BT.%26atitle%3DFree%2520R%2520Value%253A%2520a%2520Novel%2520Statistic%2520Quantity%2520for%2520Assesing%2520the%2520Accuracy%2520of%2520Crystal%2520Structures%26jtitle%3DNature%26date%3D1992%26volume%3D355%26spage%3D472%26epage%3D475%26doi%3D10.1038%2F355472a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perrakis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harkiolaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span> <span> </span><span class="NLM_article-title">ARP/wARP and Molecular Replacement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1445</span>– <span class="NLM_lpage">1450</span>, <span class="refDoi"> DOI: 10.1107/S0907444901014007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1107%2FS0907444901014007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=11567158" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFGltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=1445-1450&author=A.+Perrakisauthor=M.+Harkiolakiauthor=K.+S.+Wilsonauthor=V.+S.+Lamzin&title=ARP%2FwARP+and+Molecular+Replacement&doi=10.1107%2FS0907444901014007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">ARP/wARP and molecular replacement</span></div><div class="casAuthors">Perrakis, Anastassis; Harkiolaki, Maria; Wilson, Keith S.; Lamzin, Victor S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">D57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1445-1450</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">The aim of ARP/wARP is improved automation of model building and refinement in macromol. crystallog.  Once a mol.-replacement soln. has been obtained, it is often tedious to refine and rebuild the initial (search) model.  ARP/wARP offers three options to automate that task to varying extents: (i) autobuilding of a completely new model based on phases calcd. from the mol.-replacement soln., (ii) updating of the initial model by atom addn. and deletion to obtain an improved map and (iii) docking of a structure onto a new (or mutated) sequence, followed by rebuilding and refining the side chains in real space.  A few examples are presented where ARP/wARP made a considerable difference in the speed of structure soln. and/or made possible refinement of otherwise difficult or uninterpretable maps.  The resoln. range allowing complete autobuilding of protein structures is currently 2.0 Å, but for map improvement considerable advances over more conventional refinement techniques are evident even at 3.2 Å spacing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra62pGOo0oCbVg90H21EOLACvtfcHk0lg9gqeaLV472Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFGltLc%253D&md5=7c885ae862051042e716ad0e995e131c</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444901014007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444901014007%26sid%3Dliteratum%253Aachs%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DHarkiolaki%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26atitle%3DARP%252FwARP%2520and%2520Molecular%2520Replacement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2001%26volume%3D57%26spage%3D1445%26epage%3D1450%26doi%3D10.1107%2FS0907444901014007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span> <i>The
PyMOL Molecular Graphics System, version 1.3</i>; <span class="NLM_publisher-name">Schrödinger LLC</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The%0APyMOL+Molecular+Graphics+System%2C+version+1.3%3B+Schr%C3%B6dinger+LLC%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%250APyMOL%2520Molecular%2520Graphics%2520System%252C%2520version%25201.3%26pub%3DSchr%25C3%25B6dinger%2520LLC%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved Protein-ligand Docking Using GOLD</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+Protein-ligand+Docking+Using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Å of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of ∼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0lg9gqeaLV472Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520Protein-ligand%2520Docking%2520Using%2520GOLD%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willett, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glen, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">Development and Validation of a Genetic Algorithm for Flexible Docking</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>267</i></span>,  <span class="NLM_fpage">727</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1996.0897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+Validation+of+a+Genetic+Algorithm+for+Flexible+Docking&doi=10.1006%2Fjmbi.1996.0897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0liXQxa2yVoW5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520Validation%2520of%2520a%2520Genetic%2520Algorithm%2520for%2520Flexible%2520Docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748%26doi%3D10.1006%2Fjmbi.1996.0897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezhilarasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego-Perez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabit, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chhipa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohyeldin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurozumi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroiwa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulman, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guttridge, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DasGupta, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goidts, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakano, I.</span></span> <span> </span><span class="NLM_article-title">Serine/Threonine Kinase MLK4 Determines Mesenchymal Identity in Glioma Stem Cells in an NF-κB-dependent Manner</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.01.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=10.1016%2Fj.ccell.2016.01.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=26859459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;key=1%3ACAS%3A528%3ADC%252BC28XitlSgsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=201-213&author=S.+H.+Kimauthor=R.+Ezhilarasanauthor=E.+Phillipsauthor=D.+Gallego-Perezauthor=A.+Sparksauthor=D.+Taylorauthor=K.+Ladnerauthor=T.+Furutaauthor=H.+Sabitauthor=R.+Chhipaauthor=J.+H.+Choauthor=A.+Mohyeldinauthor=S.+Beckauthor=K.+Kurozumiauthor=T.+Kuroiwaauthor=R.+Iwataauthor=A.+Asaiauthor=J.+Kimauthor=E.+P.+Sulmanauthor=S.+Y.+Chengauthor=L.+J.+Leeauthor=M.+Nakadaauthor=D.+Guttridgeauthor=B.+DasGuptaauthor=V.+Goidtsauthor=K.+P.+Bhatauthor=I.+Nakano&title=Serine%2FThreonine+Kinase+MLK4+Determines+Mesenchymal+Identity+in+Glioma+Stem+Cells+in+an+NF-%CE%BAB-dependent+Manner&doi=10.1016%2Fj.ccell.2016.01.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Serine/Threonine Kinase MLK4 Determines Mesenchymal Identity in Glioma Stem Cells in an NF-κB-dependent Manner</span></div><div class="casAuthors">Kim, Sung-Hak; Ezhilarasan, Ravesanker; Phillips, Emma; Gallego-Perez, Daniel; Sparks, Amanda; Taylor, David; Ladner, Katherine; Furuta, Takuya; Sabit, Hemragul; Chhipa, Rishi; Cho, Ju Hwan; Mohyeldin, Ahmed; Beck, Samuel; Kurozumi, Kazuhiko; Kuroiwa, Toshihiko; Iwata, Ryoichi; Asai, Akio; Kim, Jonghwan; Sulman, Erik P.; Cheng, Shi-Yuan; Lee, L. James; Nakada, Mitsutoshi; Guttridge, Denis; DasGupta, Biplab; Goidts, Violaine; Bhat, Krishna P.; Nakano, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-213</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Activation of nuclear factor κB (NF-κB) induces mesenchymal (MES) transdifferentiation and radioresistance in glioma stem cells (GSCs), but mol. mechanisms for NF-κB activation in GSCs are currently unknown.  Here, we report that mixed lineage kinase 4 (MLK4) is overexpressed in MES but not proneural (PN) GSCs.  Silencing MLK4 suppresses self-renewal, motility, tumorigenesis, and radioresistance of MES GSCs via a loss of the MES signature.  MLK4 binds and phosphorylates the NF-κB regulator IKKα, leading to activation of NF-κB signaling in GSCs.  MLK4 expression is inversely correlated with patient prognosis in MES, but not PN high-grade gliomas.  Collectively, our results uncover MLK4 as an upstream regulator of NF-κB signaling and a potential mol. target for the MES subtype of glioblastomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHRhj7RTa7mbVg90H21EOLACvtfcHk0liXQxa2yVoW5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitlSgsr8%253D&md5=9e2255918a33b2abae6b3234655883eb</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.01.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.01.005%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DEzhilarasan%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DE.%26aulast%3DGallego-Perez%26aufirst%3DD.%26aulast%3DSparks%26aufirst%3DA.%26aulast%3DTaylor%26aufirst%3DD.%26aulast%3DLadner%26aufirst%3DK.%26aulast%3DFuruta%26aufirst%3DT.%26aulast%3DSabit%26aufirst%3DH.%26aulast%3DChhipa%26aufirst%3DR.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DMohyeldin%26aufirst%3DA.%26aulast%3DBeck%26aufirst%3DS.%26aulast%3DKurozumi%26aufirst%3DK.%26aulast%3DKuroiwa%26aufirst%3DT.%26aulast%3DIwata%26aufirst%3DR.%26aulast%3DAsai%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DSulman%26aufirst%3DE.%2BP.%26aulast%3DCheng%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DL.%2BJ.%26aulast%3DNakada%26aufirst%3DM.%26aulast%3DGuttridge%26aufirst%3DD.%26aulast%3DDasGupta%26aufirst%3DB.%26aulast%3DGoidts%26aufirst%3DV.%26aulast%3DBhat%26aufirst%3DK.%2BP.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DSerine%252FThreonine%2520Kinase%2520MLK4%2520Determines%2520Mesenchymal%2520Identity%2520in%2520Glioma%2520Stem%2520Cells%2520in%2520an%2520NF-%25CE%25BAB-dependent%2520Manner%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D201%26epage%3D213%26doi%3D10.1016%2Fj.ccell.2016.01.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6S6W','PDB','6S6W'); return false;">PDB: 6S6W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FHZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5FHZ','PDB','5FHZ'); return false;">PDB: 5FHZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TE5','PDB','6TE5'); return false;">PDB: 6TE5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=GA11','PDB','GA11'); return false;">PDB: GA11</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ43','PDB','LQ43'); return false;">PDB: LQ43</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WB9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4WB9','PDB','4WB9'); return false;">PDB: 4WB9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5G','PDB','6B5G'); return false;">PDB: 6B5G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5g" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B5g','PDB','6B5g'); return false;">PDB: 6B5g</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ45" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=LQ45','PDB','LQ45'); return false;">PDB: LQ45</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i67"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01910">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_87048"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01910?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01910</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Biochemical characterization of derivative <b>3a</b>; residual ALDHs activities in presence of 2,6-disubstituted-imidazo[1,2-<i>a</i>]pyridine derivatives <b>3a–v</b>; residual ALDHs activities in presence of 2,7-disubstituted-imidazo[1,2-<i>a</i>]pyridine derivatives <b>7a–i</b>; HPLC traces of 2,6-disubstituted-imidazo[1,2-<i>a</i>]pyridine derivatives, <b>3a–v</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula string (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_001.pdf">jm9b01910_si_001.pdf (1.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01910/suppl_file/jm9b01910_si_002.csv">jm9b01910_si_002.csv (1.44 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01910&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-9%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01910" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01910" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679abb1048b521d6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
