
<html lang="en"     class="pb-page"  data-request-id="bf0e0c53-32e5-4424-bb4e-dae09f4dc8b2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c00689;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies" /></meta><meta name="dc.Creator" content="Brianna D.  Mackie" /></meta><meta name="dc.Creator" content="Dongxing  Chen" /></meta><meta name="dc.Creator" content="Guangping  Dong" /></meta><meta name="dc.Creator" content="Cheng  Dong" /></meta><meta name="dc.Creator" content="Haley  Parker" /></meta><meta name="dc.Creator" content="Christine E.  Schaner Tooley" /></meta><meta name="dc.Creator" content="Nicholas  Noinaj" /></meta><meta name="dc.Creator" content="Jinrong  Min" /></meta><meta name="dc.Creator" content="Rong  Huang" /></meta><meta name="dc.Description" content="Protein N-terminal methyltransferases (NTMTs) methylate the α-N-terminal amines of proteins starting with the canonical X-P-K/R motif. Genetic studies imply that NTMT1 regulates cell mitosis and DN..." /></meta><meta name="Description" content="Protein N-terminal methyltransferases (NTMTs) methylate the α-N-terminal amines of proteins starting with the canonical X-P-K/R motif. Genetic studies imply that NTMT1 regulates cell mitosis and DN..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 21, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00689" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00689" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00689" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00689" /></link>
        
    
    

<title>Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00689" /></meta><meta property="og:title" content="Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0011.jpeg" /></meta><meta property="og:description" content="Protein N-terminal methyltransferases (NTMTs) methylate the α-N-terminal amines of proteins starting with the canonical X-P-K/R motif. Genetic studies imply that NTMT1 regulates cell mitosis and DNA damage repair. Herein, we report the rational design and development of the first potent peptidomimetic inhibitor for NTMT1/2. Biochemical and cocrystallization studies manifest that BM30 (with a half-maximal inhibitory concentration of 0.89 ± 0.10 μM) is a competitive inhibitor to the peptide substrate and noncompetitive to the cofactor S-adenosylmethionine. BM30 exhibits over 100-fold selectivity to NTMT1/2 among a panel of 41 MTs, indicating its potential to achieve high selectivity when targeting the peptide substrate binding site of NTMT1/2. Its cell-permeable analogue DC432 (IC50 of 54 ± 4 nM) decreases the N-terminal methylation level of the regulator of chromosome condensation 1 and SET proteins in HCT116 cells. This proof-of principle study provides valuable probes for NTMT1/2 and highlights the opportunity to develop more cell-potent inhibitors to elucidate the function of NTMTs in the future." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00689"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00689">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00689&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00689&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00689&amp;href=/doi/10.1021/acs.jmedchem.0c00689" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9512-9522</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00686" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00705" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Brianna D. Mackie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brianna D. Mackie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Institute of Structural Biology, Drug Discovery and Development, School of Pharmacy, Virginia Commonwealth University, Richmond, Virginia 23298, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brianna+D.++Mackie">Brianna D. Mackie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dongxing Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dongxing Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dongxing++Chen">Dongxing Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guangping Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guangping Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guangping++Dong">Guangping Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheng Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheng Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheng++Dong">Cheng Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haley Parker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haley Parker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Jacobs School of Medicine and Biomedical Sciences, University at Buffalo, Buffalo, New York 14203, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haley++Parker">Haley Parker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christine E. Schaner Tooley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christine E. Schaner Tooley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Jacobs School of Medicine and Biomedical Sciences, University at Buffalo, Buffalo, New York 14203, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christine+E.++Schaner+Tooley">Christine E. Schaner Tooley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas Noinaj</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas Noinaj</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences, Markey Center for Structural Biology, and the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas++Noinaj">Nicholas Noinaj</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jinrong Min</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jinrong Min</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jinrong++Min">Jinrong Min</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rong Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rong Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry, Institute of Structural Biology, Drug Discovery and Development, School of Pharmacy, Virginia Commonwealth University, Richmond, Virginia 23298, United States</div><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#59312c38373e742b19292c2b3d2c3c773c3d2c"><span class="__cf_email__" data-cfemail="d0b8a5b1beb7fda290a0a5a2b4a5b5feb5b4a5">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rong++Huang">Rong Huang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00689&amp;href=/doi/10.1021%2Facs.jmedchem.0c00689" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9512–9522</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 20, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 April 2020</li><li><span class="item_label"><b>Published</b> online</span>21 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00689" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00689</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9512%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DBrianna%2BD.%2BMackie%252C%2BDongxing%2BChen%252C%2BGuangping%2BDong%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.0c00689%26title%3DSelective%2BPeptidomimetic%2BInhibitors%2Bof%2BNTMT1%252F2%253A%2BRational%2BDesign%252C%2BSynthesis%252C%2BCharacterization%252C%2Band%2BCrystallographic%2BStudies%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9522%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00689"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1002</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00689" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Brianna&quot;,&quot;last_name&quot;:&quot;D. Mackie&quot;},{&quot;first_name&quot;:&quot;Dongxing&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Guangping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Cheng&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Haley&quot;,&quot;last_name&quot;:&quot;Parker&quot;},{&quot;first_name&quot;:&quot;Christine&quot;,&quot;last_name&quot;:&quot;E. Schaner Tooley&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;Noinaj&quot;},{&quot;first_name&quot;:&quot;Jinrong&quot;,&quot;last_name&quot;:&quot;Min&quot;},{&quot;first_name&quot;:&quot;Rong&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;21&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9512-9522&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00689&quot;},&quot;abstract&quot;:&quot;Protein N-terminal methyltransferases (NTMTs) methylate the α-N-terminal amines of proteins starting with the canonical X-P-K/R motif. Genetic studies imply that NTMT1 regulates cell mitosis and DNA damage repair. Herein, we report the rational design and development of the first potent peptidomimetic inhibitor for NTMT1/2. Biochemical and cocrystallization studies manifest that BM30 (with a half-maximal inhibitory concentration of 0.89 ± 0.10 μM) is a competitive inhibitor to the peptide substrate and noncompetitive to the cofactor S-adenosylmethionine. BM30 exhibits over 100-fold selectivity to NTMT1/2 among a panel of 41 MTs, indicating its potential to achieve high selectivity when targeting the peptide substrate binding site of NTMT1/2. Its cell-permeable analogue DC432 (IC50 of 54 ± 4 nM) decreases the N-terminal methylation level of the regulator of chromosome condensation 1 and SET proteins in HCT116 cells. This proof-of principle study provides valuable probes for NTMT1/2 and highlights the opportuni&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00689&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00689" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00689&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00689" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00689&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00689" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00689&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00689&amp;href=/doi/10.1021/acs.jmedchem.0c00689" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00689" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00689" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00689%26sid%3Dliteratum%253Aachs%26pmid%3D32689795%26genre%3Darticle%26aulast%3DMackie%26date%3D2020%26atitle%3DSelective%2BPeptidomimetic%2BInhibitors%2Bof%2BNTMT1%252F2%253A%2BRational%2BDesign%252C%2BSynthesis%252C%2BCharacterization%252C%2Band%2BCrystallographic%2BStudies%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9512%26epage%3D9522%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291304" title="Organic reactions">Organic reactions</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein N-terminal methyltransferases (NTMTs) methylate the α-N-terminal amines of proteins starting with the canonical X-P-K/R motif. Genetic studies imply that NTMT1 regulates cell mitosis and DNA damage repair. Herein, we report the rational design and development of the first potent peptidomimetic inhibitor for NTMT1/2. Biochemical and cocrystallization studies manifest that <b>BM30</b> (with a half-maximal inhibitory concentration of 0.89 ± 0.10 μM) is a competitive inhibitor to the peptide substrate and noncompetitive to the cofactor S-adenosylmethionine. <b>BM30</b> exhibits over 100-fold selectivity to NTMT1/2 among a panel of 41 MTs, indicating its potential to achieve high selectivity when targeting the peptide substrate binding site of NTMT1/2. Its cell-permeable analogue <b>DC432</b> (IC<sub>50</sub> of 54 ± 4 nM) decreases the N-terminal methylation level of the regulator of chromosome condensation 1 and SET proteins in HCT116 cells. This proof-of principle study provides valuable probes for NTMT1/2 and highlights the opportunity to develop more cell-potent inhibitors to elucidate the function of NTMTs in the future.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein methylation is an important epigenetic modification that regulates dynamic chromatin states and transcription.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Therefore, potent and specific inhibitors of protein methyltransferases (MTs) can serve as valuable chemical probes to elucidate functions in physiological and pathological contexts. For example, the recently approved TAZVERIK (tazemetostat) for epithelioid sarcoma treatment is an inhibitor of protein lysine MT (PKMT) EZH2. Another PKMT inhibitor, pinometostat targeting DOT1L, is in phase I trial for combination use in patients with acute myeloid leukemia.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In addition, a few protein arginine MT 5 (PRMT5) inhibitors, including JNJ64619178, GSK3326595, PF06939999, and PRT811, are also in clinical trials for advanced solid tumors.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">Protein α-N-terminal methylation has been known for nearly 4 decades since it was first uncovered on bacteria ribosomal proteins L33.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> The identification of N-terminal MT 1 (NTMT1/NRMT1) unveiled the first methylation writer for human protein α-N-termini.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> In addition to NTMT1, its close homologue NTMT2/NRMT2 that shares over 50% sequence similarity has been identified as another writer for human protein α-N-terminal methylation.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Both NTMT1 and 2 share the same X-P-K/R canonical recognition motif, where X can be any amino acid although D and E are not preferred.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Recently, METTL13 has been discovered to solely methylate the elongation factor 1A (EF1A) on both the α-N-terminus and the lysine side chain.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> However, the N-terminal sequence of EF1A is different from the X-P-K/R motif that NTMT1/2 prefers. Structural studies on NTMT1/2 in complex with their substrate peptides reveal that the peptide substrates bind at a defined binding pocket that juxtapose with the cofactor S-adenosyl methionine (SAM) binding site.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> This is distinct from PKMTs and PRMTs, which normally have a narrow channel to accommodate only the side chain of lysine (Lys) or arginine (Arg). The structural information gained from the cocrystal structures substantiated the unique substrate selectivity of NTMT1/2.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a></div><div class="NLM_p last">Protein α-N-terminal methylation was originally proposed to regulate protein–protein interactions as the methylated proteins initially identified were involved in large macromolecular complexes including histone proteins, cytochrome <i>c</i>-557, and myosin light chain proteins.<a onclick="showRef(event, 'ref4 ref14'); return false;" href="javascript:void(0);" class="ref ref4 ref14">(4,14)</a> Recent discoveries have demonstrated its relevance in protein–DNA interactions, as shown in strengthening the interactions of chromatin with centromere protein A (CENPA) and regulator of chromosome condensation 1 (RCC1), as well as of DNA damage-binding protein 2 (DDB2) with DNA damage foci.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a> Meanwhile, the newly identified substrates of NTMT1 also include tumor suppressor retinoblastoma 1 (RB1), oncoprotein SET, and poly(ADP-ribose) polymerase 3.<a onclick="showRef(event, 'ref6 ref18'); return false;" href="javascript:void(0);" class="ref ref6 ref18">(6,18)</a> In addition, previous genetic studies have inferred the importance of NTMT1 in mitosis, DNA damage repair, aging, and a variety of cancers.<a onclick="showRef(event, 'ref6 ref16 ref17 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref6 ref16 ref17 ref19 ref20 ref21">(6,16,17,19−21)</a> Despite the aforementioned advancements, our understanding of the protein α-N-terminal methylation function is still in its infancy. Therefore, it is imperative to discover specific and potent inhibitors for NTMTs to probe their biological functions. Guided by the kinetic mechanism of NTMT1, bisubstrate analogues that covalently link both SAM and the peptide substrate were designed to mimic the transition state and displayed high selectivity and potency for NTMT1.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26">(22−26)</a> Nevertheless, intrinsic properties of those bisubstrate analogues impose a challenge to penetrate cell membranes, which limit their applications in cell-based studies. Therefore, there is a need for a new strategy to discover inhibitors for NTMTs in order to probe the downstream process of protein α-N-terminal methylation. Given that the NTMT family has a unique peptide substrate binding site, we hypothesize that targeting the peptide binding site should achieve higher selectivity for NTMTs. Herein, we report the design and synthesis of the first potent and selective peptidomimetic inhibitor for NTMT1/2.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64417" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64417" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design</h3><div class="NLM_p">According to the crystal structures of NTMT1 in complex with peptide substrates (XPKRIA), the backbone carbonyl group of the first amino acid (X1) interacts with a conserved Asn168 residue NTMT1 through hydrogen (H)-bonding.<a onclick="showRef(event, 'ref9 ref27'); return false;" href="javascript:void(0);" class="ref ref9 ref27">(9,27)</a> Mutation of Asn168 to Lys resulted in a ∼36-fold loss in <i>K</i><sub>m</sub>, indicating the importance of this interaction between NTMT1 and its peptide substrate.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> However, the tolerance at the first position of the substrate suggests a possible spacious binding pocket surrounding the first position. The N-terminal amine of the peptide substrate points toward the methyl group of the SAM and serves as a nucleophile to receive the methyl group from the SAM. The second residue P2 forms a stacking interaction with the indole of Trp136. The third residue K3 forms two key hydrogen bonds with side chains of two aspartic acids.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Either a Lys or Arg residue has been discovered at the third position of known protein substrates. The fourth amino acid is adjacent to a negatively charged substrate binding channel, where the fifth and sixth residues are primarily exposed to the solvent.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Hence, we speculate that the α-N-terminal amine is essential for methyl transfer by acting as a nucleophile to attack SAM, but it is not essential for binding. Therefore, we hypothesized that the removal of the N-terminal amine of NTMT1/2 peptide substrates would convert the substrate into an inhibitor. As the first four amino acids contribute significantly to the interaction with NTMT1/2, we initiated our efforts by incorporating different substitutions (<b>R</b><sub><b>1</b></sub>) on a tripeptide (PKR) through an amide bond (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) to retain as much interaction as the peptide substrates. PPKRIA and YPKRIA are two peptide substrates for NTMT1 with the dissociation constant (<i>K</i><sub>d</sub>) value at 0.7 ± 0.2 nM and 0.1 ± 0.08 μM, respectively.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Hence, compounds <b>1</b>–<b>5</b> were designed to mimic the proline (Pro) or tyrosine (Tyr) moiety by introducing a cyclopentane, cyclohexane, furan, phenyl, and benzyl moiety as the <b>R</b><sub><b>1</b></sub> group.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design inhibitors based on the structure of NTMT1 peptide substrates. H-bond interactions are shown in red dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis</h3><div class="NLM_p last">The peptides were synthesized following the standard Fmoc solid-phase synthesis on Rink amide resin using a CEM Liberty microwave peptide synthesizer. The peptidomimetics that have carboxylic acids at the first position were prepared through the standard amino acid coupling reaction. All compounds were cleaved from the solid support in a cleavage cocktail consisting of trifluoroacetic acid (TFA)/2,2′-(ethylenedioxy)diethanethiol/water (H<sub>2</sub>O)/triisopropylsilane (94:2.5:2.5:1), followed by purification through reverse-phase high-performance liquid chromatography (RP-HPLC) and characterized through mass spectrometry (MS) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Figure S1</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Biochemical Characterization</h3><div class="NLM_p">All peptidomimetics were first subject to a methylation progression matrix-assisted laser desorption ionization mass spectroscopy (MALDI-MS) study to examine if they were NTMT1 substrates.<a onclick="showRef(event, 'ref22 ref28'); return false;" href="javascript:void(0);" class="ref ref22 ref28">(22,28)</a> None of the peptidomimetics displayed any methylation, indicating that they were not NTMT1 substrates. Next, the synthesized peptidomimetics were evaluated in a SAH hydrolase (SAHH)-coupled fluorescence assay at 100 μM of SAM and at the Michaelis constant (<i>K</i><sub>m</sub>) value of the peptide substrate (SPKRIA or GPKRIA).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Initial screening was carried out at the concentrations of 5, 30, and 100 μM for each compound. Compound <b>5</b> exhibited more than 50% of inhibition at 30 μM and was then subjected to subsequent half-maximal inhibitory concentration (IC<sub>50</sub>) studies at concentrations ranging from 0 to 100 μM in a threefold dilution. Compound <b>5</b> showed an IC<sub>50</sub> of 6.85 ± 2.82 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Figure S2</a>). A summary of the IC<sub>50</sub> values for the peptidomimetics is shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the First Position of <b>R</b><sub><b>1</b></sub>-PKR<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0009.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were determined in triplicate (<i>n</i> = 3) and presented as mean ± standard deviation (SD).</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Relationship Studies</h3><div class="NLM_p">In order to improve the inhibitory activity of the peptidomimetics, structure–activity relationship (SAR) studies were conducted using <b>5</b> as the lead compound (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of the Second to Fourth Positions of <b>R</b><sub><b>1</b></sub>-PKR<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0010.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were determined in triplicate (<i>n</i> = 3) and presented as mean ± SD.</p></div></div><div></div></div><div id="sec2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> First (<b>R</b><sub><b>1</b></sub>) Position</h4><div class="NLM_p last">The benzoyl substitution of compound <b>5</b> resembled the side chain of Tyr; therefore, we focused our SAR investigation on introducing various para substitutions on the phenyl ring, including a hydroxy, an amino, and a methoxy group at the para position of the phenyl ring to yield compounds <b>6</b>–<b>8</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Among all synthesized peptidomimetics, compound <b>6</b> (<b>BM30</b>) which contained a hydroxyl group at the para position showed the greatest inhibitory activity with an IC<sub>50</sub> of 0.89 ± 0.10 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Figure S3</a>), approximately sevenfold increased activity compared to <b>5</b>. However, a methoxy substitution led to a loss of inhibition of <b>8</b> (IC<sub>50</sub> > 100 μM), while its constitutional isomer <b>9</b> rescued the inhibitory activity (IC<sub>50</sub> = 1.5 ± 0.1 μM) with a 4-hydroxymethyl substitution. This suggests the preference of a H-bond donor at the first position. Given that introducing a methylene linker between the phenyl ring and the carbonyl group significantly increased the inhibition of <b>5</b> compared to <b>4</b>, an ethylene linker was introduced in <b>10</b> and <b>11</b> to probe the optimal linker length (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Both <b>10</b> and <b>11</b> displayed a fivefold and threefold loss in inhibition compared to their respective parent compounds <b>6</b> and <b>7</b>, suggesting a methylene group as the optimal linker for the NTMT1 inhibitor. Finally, compound <b>12</b> was prepared to explore the importance of the carbonyl group at the first position. Nevertheless, <b>12</b> did not show any inhibitory activity at 100 μM, implying the significance of the carbonyl group at the first position.</div></div><div id="sec2_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Second Position</h4><div class="NLM_p last">All the available cocrystal structures of NTMT1/2 in complex with X-P-K/R substrates reveal that Pro2 only exhibits the trans conformation, suggesting that the trans conformation is favored to interact with NTMT1/2.<a onclick="showRef(event, 'ref9 ref11 ref27'); return false;" href="javascript:void(0);" class="ref ref9 ref11 ref27">(9,11,27)</a> To verify the preference of this trans conformation, an α-methyl-Pro was introduced at the second position as the predominant conformation of α-methyl-Pro is cis.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The resulting compound <b>13</b> completely lost the inhibitory activity for NTMT1 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). These results substantiate the previous findings that Pro2 prefers the trans conformation; however, the additional methyl group may also introduce certain steric hindrances. Next, ortho- and meta-amino benzoic acids were also introduced at Pro2 to generate <b>14</b> and <b>15</b>, respectively. Both compounds did not exhibit any inhibitory activity, denoting the importance of Pro2 in the canonical X-P-K/R motif.</div></div><div id="sec2_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Third and Fourth Positions</h4><div class="NLM_p last">Next, we explored the modifications at the third and fourth positions in an attempt to increase the inhibitory activity, as well as to enhance stability, as peptide-based inhibitors are susceptible to degradation (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Arg has been detected at the third position in the validated NTMT1 substrate CENP-A (GPRR),<a onclick="showRef(event, 'ref10 ref15'); return false;" href="javascript:void(0);" class="ref ref10 ref15">(10,15)</a> thus <b>16</b> was synthesized with Arg at the third position. However, <b>16</b> displayed over a 20-fold loss of inhibition compared to <b>6</b>. This result signified the preference for Lys over Arg at the third position, potentially because of its important role in interacting with two acidic residues (Asp177 and Asp180 in NTMT1; Asp232 and Asp 235 in NTMT2).<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9−11)</a> Although Arg and Lys are two prevalent amino acids at the fourth position of the NTMT1 substrate peptides, additional amino acids have been observed in reported NTMT1 substrates. Therefore, Lys, threonine (Thr), alanine (Ala), and glycine (Gly) were introduced to replace the Arg at the fourth position of <b>BM30</b> to generate <b>17</b>–<b>21</b> to examine the contribution of a side chain at this position because it is exposed to the solvent in the cocrystal structures.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Compound <b>17</b>, which included a Lys in the fourth position, exhibited negligible difference in inhibition compared to <b>BM30</b> which contains an Arg. Introducing Thr or Ala at the fourth position led to a fivefold loss as shown in <b>18</b>–<b>19</b>, while incorporating Gly produced almost a 14-fold loss in inhibition as shown in <b>20</b>. When the fourth position was eliminated to yield <b>21</b>, the inhibitory activity decreased 18-fold compared to <b>BM30</b>. The result indicated that the side chain of the fourth amino acid contributed to the NTMT1 interaction although it is exposed to the solvent in the cocrystal structures. Additionally, a non-natural amino acid was also made at either the third or fourth position to produce <b>22</b>–<b>24</b> in an attempt to increase the stability against proteases. However, loss of inhibition ranging from nearly 5- to 37-fold was observed in compounds <b>22</b>–<b>24</b> compared to <b>BM30</b>, although <b>22</b> was the most potent one among these three protease-stable analogues.</div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Selectivity Studies</h3><div class="NLM_p">Given that many MTs exist, selectivity of the inhibitor is a critical requirement for its potential as a valuable probe. We first investigated the selectivity of <b>BM30</b> and <b>22</b> against one representative member from the protein lysine MT PKMT (G9a) and protein arginine MT PRMT (PRMT1) families. Both enzymes share a cofactor SAM binding site despite having different substrate peptide binding sites from NTMT1. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Table S1</a>, <b>BM30</b> demonstrated >100 μM IC<sub>50</sub> values against both G9a and PRMT1, but <b>22</b> exhibited inhibition of G9a. Thus, we proceeded with <b>BM30</b> to evaluate its selectivity against a panel of 41 MTs (Reaction Biology) that include NTMT1/2, PKMTs, PRMTs, and DNMTs (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Table S2</a>). The results indicated that <b>BM30</b> displays comparable inhibition against both NTMT1 and NTMT2, which was not surprising as NTMT1/2 share the canonical X-P-K/R recognition motif. Strikingly, <b>BM30</b> exhibited very marginal inhibition against the remaining 39 MTs even at 100 μM. More than 100-fold selectivity that <b>BM30</b> displays for NTMT1/2 compared to PKMTs, PRMTs, and DNMTs evinces the unique substrate specificity of NTMT1/2.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selectivity of the NTMT inhibitor <b>BM30</b>. Selectivity of <b>BM30</b> against a panel of 41 PKMTs, PRMTs, DNMTs, and NTMT1/2 was determined at a concentration of 100 μM in duplicate (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Inhibition Mechanism</h3><div class="NLM_p">Kinetic characterization of <b>BM30</b> was performed to examine its mechanism of action using the SAHH-coupled fluorescence-based assay (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, <b>BM30</b> exhibited a competitive inhibition pattern with respect to the NTMT1 peptide substrate RCC1-6 (SPKRIA). This is demonstrated by an ascending and linear dependence of the IC<sub>50</sub> values relative to the increasing peptide substrate concentration of RCC1-6. On the other hand, <b>BM30</b> displayed a noncompetitive inhibition pattern with respect to the cofactor SAM, indicated by a line that is parallel to the <i>x</i>-axis. These results verify that <b>BM30</b> competitively binds to the peptide substrate binding site, supporting our design strategy.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition mechanism of <b>BM30</b>. All experiments were run in triplicate (<i>n</i> = 3). (A) IC<sub>50</sub> curves of <b>BM30</b> at varying concentrations of the NTMT1 peptide substrate, RCC1-6 (SPKRIA), with a fixed concentration of SAM; (B) linear regression plot of IC<sub>50</sub> values with corresponding concentrations of RCC1-6; (C) IC<sub>50</sub> curves of <b>BM30</b> at varying concentrations of SAM with a fixed concentration of RCC1-6; (D) plot of IC<sub>50</sub> values with corresponding concentrations of SAM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Crystal Structure of the NTMT1–<b>BM30</b>–SAH Complex</h3><div class="NLM_p">To further confirm that <b>BM30</b> binds to the peptide substrate binding site of NTMT1, we solved the X-ray cocrystal structure of NTMT1–<b>BM30</b>–SAH (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A–C). <b>BM30</b> occupies the peptide substrate binding site of NTMT1. Superimposition of our NTMT1–<b>BM30</b>–SAH structure with the published NTMT1–YPKRIA–SAH ternary complex (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1D">5E1D</a>) gave a root-mean-square deviation (rmsd) value of 0.12 Å (across all residues of chain A, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D,E). The <b>BM30</b>–NTMT1 complex retains the same interactions as previously observed with YPKRIA–NTMT1 in the ternary complex of the substrate peptide/SAH.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> For instance, the carbonyl oxygen of 4-hydroxyl phenyl acetic acid interacts with the side chain of Asn168 through H-bonding. The second Pro occupies a hydrophobic pocket that is formed by Leu31, Ile37, and Ile214. In addition, the ε-amine of the third Lys forms electronic interactions with carboxylate groups of Asp177 and Asp180. In addition, a direct H-bond exists between the carbonyl oxygen of the fourth Arg and Try215. The 4-hydroxyl group of the phenyl ring also forms a H-bond with Asp180 and one water molecule.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of NTMT1 (gray cartoon)–<b>BM30</b> (green stick)–SAH (green stick) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>). (A) Fo-Fc omit density map of ligands. (B) Zoomed view of interactions of <b>BM30</b> with NTMT1. H-bond interactions are shown as yellow dotted lines. (C) <b>BM30</b> interaction diagram (Schrödinger Maestro) with NTMT1. (D,E) Structural alignment of NTMT1 (gray)–<b>BM30</b> (green stick)–SAH (green stick) and NTMT1 (yellow cartoon)–YPKRIA (yellow stick)–SAH (yellow stick) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1D">5E1D</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Design and Characterization of <b>DC431</b> and <b>DC432</b></h3><div class="NLM_p">We tested the cell permeability of <b>BM30</b> through MS detection of its cellular levels.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> However, <b>BM30</b> was minimally detected at 100 μM inside the cells (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Figure S4</a>). In order to increase the cell permeability, we designed and synthesized <b>DC431</b> and <b>DC432</b> by attaching a cell-permeable peptide TAT (GRKKRRQRRR-NH<sub>2</sub>) and five Arg at the C-terminus of <b>BM30</b>, respectively. It is known that incorporating six Arg into compounds can increase the cell permeability.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Hence, five additional Arg were added to <b>BM30</b> to generate <b>DC432</b> because <b>BM30</b> already had one Arg at its C-terminus. Given that the first four amino acids contribute significantly to the binding of NTMT1/2, we hypothesized that <b>DC431</b> and <b>DC432</b> would have slightly increased inhibitory activities than <b>BM30</b> as they contained additional amino acids at the C-terminus of <b>BM30</b>. To confirm our hypothesis, IC<sub>50</sub> values of <b>DC431</b> and <b>DC432</b> were determined to be 0.23 ± 0.03 and 0.054 ± 0.004 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B). <b>DC431</b> demonstrated a fourfold increased inhibition for NTMT1 than for <b>BM30</b> as predicted. Surprisingly, <b>DC432</b> displayed a 4-fold higher activity than <b>DC431</b> and 16-fold inhibitory activity compared to <b>BM30</b>.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Biochemical characterization of <b>DC431</b> and <b>DC432</b>. (A,B) IC<sub>50</sub> curves of <b>DC431</b> and <b>DC432</b>. IC<sub>50</sub> values were determined in triplicate (<i>n</i> = 3) and presented as mean <u class="uu">±</u> SD. (C) Representative western blots showing N-terminal dimethylation (Me<sub>2</sub>RCC1) and trimethylation (Me<sub>3</sub>RCC1) of full-length recombinant human RCC1 by full-length recombinant human NTMT1 in the presence of increasing concentrations of <b>BM30</b>, <b>DC431</b>, and <b>DC432</b> (<i>n</i> = 3). NTMT1 and RCC1 blots are shown as loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition, we tested the levels of N-terminal methylation by directly incubating the inhibitors with NTMT1, SAM, and the full-length substrate protein RCC1. At 2 μM concentration of <b>BM30</b> or <b>DC432</b>, there is a slight increase of me2-RCC1 levels and a decrease of me3-RCC1 levels as compared to untreated samples (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). An increase in me2-RCC1 levels indicates inhibition of NTMT1 trimethylation, as NTMT1 is a distributive enzyme that normally reaches saturated trimethylation levels under these conditions.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The appearance and persistence of me2-RCC1 signal enzymatic inhibition. At 10 and 50 μM concentrations, there was a considerable decrease in me3-RCC1 levels and an increase in me2-RCC1 levels for all three inhibitors (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). To further validate their interactions with both NTMT1 and 2, we determined the binding affinities of both cell-permeable analogues and <b>BM30</b> via isothermal titration calorimetry (ITC) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Table S4</a>). <b>BM30</b> exhibited a comparable <i>K</i><sub>d</sub> value of 3.0 and 3.7 μM to NTMT1 and 2, respectively. However, <b>DC431</b> showed a fourfold increase in binding to NTMT1. Furthermore, <b>DC432</b> displayed an eightfold and fourfold tighter binding affinity to NTMT1 and NTMT2, respectively. Increased inhibitory activities and binding affinities of both <b>DC431</b> and <b>DC432</b> corroborated the contributing roles of C-terminal amino acids to XPKR, as demonstrated before in the NTMT1–SPKRIA–SAH and NTMT2–SPKRIA–SAH ternary complexes.<a onclick="showRef(event, 'ref9 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref11">(9,11)</a> In addition, there is three- to six-fold difference in <b>DC431</b> and <b>DC432</b> for NTMT1 over NTMT2 based on the data from ITC, but unmodified <b>BM30</b> showed similar <i>K</i><sub>d</sub> values in both NTMT1/2. This change of selectivity for NTMT1/2 demonstrated the contribution of the C-terminal sequence to the selectivity, suggesting its potential for the development of selective inhibitors for NTMT1 over its homologue NTMT2. Furthermore, enhanced cell penetration of <b>DC431</b> and <b>DC432</b> was also confirmed through a cellular MS assay. Both <b>DC431</b> and <b>DC432</b> exhibited cellular penetration at 1 μM, while <b>BM30</b> did not display any even at 100 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">Figure S4</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding affinity for NTMT1 (A) (<i>n</i> = 2) and NTMT2 (B) (<i>n</i> = 1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Cellular N-Terminal Methylation Level</h3><div class="NLM_p">NTMT1 can catalyze the trimethylation of its substrate starting with an SPK motif. Therefore, a cell permeable inhibitor of NTMT1 would be expected to decrease the trimethylation level. Then, we proceeded to assess the inhibitory effects of <b>DC432</b> on the N-terminal trimethylation level via Western blotting using a specific antibody to the N-terminal me3-SPK motif. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, treatment with <b>DC432</b> for 72 h in HCT116 cells resulted in a decrease in the me3-SPK level at 100 μM. Although the impact on mono- and dimethylation levels were not examined for this investigation, cellular inhibition was over 100-fold lower than the value obtained from the enzymatic assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The discrepancy between these results may be due to the low efficiency of <b>DC432</b> to reach the target as NTMT1 is predominantly in the nucleus, as reported before.<a onclick="showRef(event, 'ref6 ref8 ref27'); return false;" href="javascript:void(0);" class="ref ref6 ref8 ref27">(6,8,27)</a></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Cellular N-terminal methylation assay of compound DC432 in HCT116 cells. (A) Structure of DC432. (B) Representative western blot results of effects of DC432 (0–1 mM) on the cellular methylation level (<i>n</i> = 2). RCC1, SET, and Lamin B1 blots are shown as loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08445" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08445" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, this study describes the discovery of the first potent and selective peptidomimetic inhibitor <b>BM30</b> for NTMT1/2 reported to date. Furthermore, <b>BM30</b> exhibits over 100-fold selectively against NTMT1/2 among a panel of 41 MTs. The cell-permeable derivative <b>DC432</b> represses N-terminal methylation levels of NTMT1/2 substrates in HCT116 cells, demonstrating that <b>DC432</b> would be a valuable probe for mechanistic studies of NTMT1/2 in the cellular context. In addition, the crystal structure of <b>BM30</b> in complex with NTMT1 revealed the structural basis for its potency and selectivity, which paves the way for the future design and development of inhibitors with improved cell potency.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Chemistry: General Information</h3><div class="NLM_p last">All reagents and solvents were purchased from commercial sources (Fisher, Aldrich, and Chem-Impex) and used directly. Final compounds were purified by preparative RP-HPLC performed on either a Waters 1525 or an Agilent 1260 Series system. Systems were run with 0–20% methanol/water gradient with 0.1% TFA. MALDI-MS spectral data were acquired in positive-ion mode using a Sciex 4800 MALDI TOF/TOF MS. The peptides (PKR, PKRIA, and PRRRS) were synthesized on a CEM Liberty Blue automated microwave peptide synthesizer with the manufacturer’s standard coupling cycles at a 0.1 mmol scale. All compounds were cleaved from the resin in a cocktail of TFA/2,2′-(ethylenedioxy)diethanethiol/H<sub>2</sub>O/triisopropylsilane (94:2.5:2.5:1) and confirmed by MS. The purity of final compounds was confirmed through an Agilent 1260 Series HPLC system by running with 5–30% methanol/water gradient with 0.1% TFA. All the purity of target compounds showed >95%.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Cell Culture and Antibodies</h3><div class="NLM_p last">HCT116 cells were obtained from ATCC and were cultured in McCoy’s 5A (modified) medium (Gibco, #16600082), supplemented with 10% fetal bovine serum (FBS) (Gibco, #16000044) and 1% antibiotic–antimycotic (Gibco, #15240062). Antibodies against the following proteins were used: Me3-SPK (Schaner Tooley lab<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a>), RCC1 (Proteintech, #22142-1-AP), SET (Proteintech, #55201-1-AP), lamin B1 (Proteintech, # 12987-1-AP), antirabbit IgG, and horseradish peroxidase (HRP)-linked (cell signaling, #7074S).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> NTMT1 Biochemical Assays</h3><div class="NLM_p">Expression and purification of NTMT1 were performed as previously described.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Kinetic characterization of the peptide inhibitors was determined in a fluorescence-based SAHH-coupled assay as described before. The inhibitors were added at concentrations ranging from 0 to 100 μM following a threefold dilution. The methylation assay was performed under the following conditions: 25 mM Tris (pH = 7.5), 50 mM potassium chloride (KCl), 0.01% Triton X-100, 5 μM SAHH, 0.2 μM NTMT1, 100 μM SAM, and 15 μM ThioGlo1. After 10 min of incubation at 37 °C, the reaction was initiated with 10 μL of 50 μM RCC1-6 for a total volume of 100 μL. Fluorescence intensity was monitored using a BMG ClarioStar microplate reader (<i>E</i><sub>x</sub> = 370 nm, <i>E</i><sub>m</sub> = 500 nm) at 37 °C for 15 min. The rates were fit to the log[inhibitor] versus response–variable slope (four parameters) model using least squares nonlinear regression using GraphPad Prism software 7 or 8. All experiments were performed in triplicate.</div><div class="NLM_p last">The inhibition mechanism of <b>BM30</b> was investigated using the fluorescence-based assay described above. Six independent IC<sub>50</sub> studies of <b>BM30</b> were performed in triplicate with varying concentrations of the substrate peptide, RCC1-6, and SAM at its <i>K</i><sub>m</sub> value. The inhibitors ranging in concentration of 0–10 mM (1 μL), following a threefold dilution, were incubated in the well-solution that was added in the following order: ddH<sub>2</sub>O, buffer (10 μL of 250 mM Tris, pH 7.4, and 500 mM KCl), SAM (1 μL of 10 μM), SAH hydrolase (5 μL of 10 mg/mL), NTMT1 (0.5 μL of 40 μM), and ThioGlo1 (1 μL of 1.5 mM). After 10 min of incubation at 37 °C, the reaction was initiated with each concentration of RCC1-6 (10 μL of 5, 10, 15, 20, 40, and 80 μM) for a total volume of 100 μL. The final concentrations are buffer (25 mM Tris, pH 7.4, and 50 mM KCl), SAM (1 μM), SAH hydrolase (10 μM), NTMT1 (0.2 μM), inhibitors (0–100 μM), RCC1-6 (0.25<i>K</i><sub>m</sub>, 0.5<i>K</i><sub>m</sub>, 0.75<i>K</i><sub>m</sub>, 2<i>K</i><sub>m</sub>, and 4<i>K</i><sub>m</sub>; 0.5, 1, 1.5, 2, 4, and 8 μM, respectively), and ThioGlo1 (15 μM). Fluorescence intensity was monitored using a CLARIOstar microplate reader (<i>E</i><sub>x</sub> = 370 nm, <i>E</i><sub>m</sub> = 500 nm) at 37 °C for 15 min. The rates were fit to the log[inhibitor] versus response model using least squares nonlinear regression through GraphPad Prism 7 software. The average IC<sub>50</sub> value of each independent triplicate study was then plotted against the concentration of [RCC1-6]/<i>K</i><sub>m</sub>. Next, the same experiment was repeated with six IC<sub>50</sub> studies of <b>BM30</b> in triplicate at varying concentrations of SAM (1 μL of 31.25, 62.5, 125, 250, 500, and 1000 μM) and RCC1-6 at its <i>K</i><sub>m</sub> value. The final concentrations are buffer (25 mM Tris, pH 7.4, and 50 mM KCl), SAM (0.3125<i>K</i><sub>m</sub>, 0.625<i>K</i><sub>m</sub>, 1.25<i>K</i><sub>m</sub>, 2.5<i>K</i><sub>m</sub>, 5<i>K</i><sub>m</sub>, and 10<i>K</i><sub>m</sub>; 0.3125, 0.625, 1.25, 2.5, 5, and 10 μM, respectively), SAH hydrolase (10 μM), NTMT1 (0.2 μM), inhibitors (0–100 μM), RCC1-6 (2 μM), and ThioGlo1 (15 μM). The average IC<sub>50</sub> value of each independent triplicate study was plotted against the concentration of the [SAM]/<i>K</i><sub>m</sub>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Selectivity Studies</h3><div class="NLM_p last">To determine selectivity, kinetic analysis of the peptide inhibitors was initially carried out using the SAH hydrolase-coupled fluorescence assay against G9a and PRMT1, respectively. The inhibitors were added at concentrations ranging from 0 to 100 μM following a threefold dilution. For G9a, the assay was performed in 1× phosphate-buffered saline, pH 7.4, 100 μM SAM, 10 μM SAH hydrolase, 25 nM G9a, inhibitors (0–100 μM), 20 μM H3-15 peptide, and 15 μM ThioGlo1. For PRMT1, the assay was carried out in 2.5 mM HEPES (pH 7.0), 25 mM sodium chloride (NaCl), 25 μM ethylenediaminetetraacetic acid, 50 μM tris(2-carboxyethyl)phosphine (TCEP), 100 μM SAM, 10 μM SAH hydrolase, 200 nM PRMT1, 100 μM H4-12 peptide, and 15 μM ThioGlo1. Fluorescence intensity was monitored using a CLARIOstar microplate reader (<i>E</i><sub>x</sub> = 370 nm, <i>E</i><sub>m</sub> = 500 nm) at 37 °C for 15 min. The rates were fit to the log[inhibitor] versus response model using least squares nonlinear regression through GraphPad Prism 7 software. All experiments were performed in triplicate.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Cocrystallization and Structure Determination</h3><div class="NLM_p last">Purified NTMT1 at 30 mg/mL was mixed with <b>BM30</b> at a molar ratio of 1:2 and incubated for 1 h at 4 °C. Broad matrix crystallization screening was performed using a mosquito-LCP high-throughput crystallization robot (TTP LabTech) using the hanging drop vapor diffusion method at 20 °C. Crystals containing <b>BM30</b> were grown in 0.1 M Na-HEPES, pH 7.5, 1.6 M ammonium sulfate, and 2% PEG 1000; all crystals were harvested directly from the drop and flash-cooled into liquid nitrogen. Data were collected on single crystals at 12.0 keV on the GM/CA-CAT ID-D beamline at the Advanced Photon Source, Argonne National Laboratory. The data were processed using the on-site automated pipeline with Fast DP, and molecular replacement was performed with Phaser-MR (PHENIX).<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> All model building was performed using COOT, and subsequent refinement was done using phenix.refine (PHENIX).<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> The structure has been solved, refined, and deposited at the Protein Data Bank with ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>. Structure-related figures were made with PyMOL (Schrödinger) and annotated and finalized with Adobe Photoshop and Illustrator.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>In Vitro</i> Methylation Assays with a Full-Length Recombinant Protein</h3><div class="NLM_p last">All methylation assays were conducted at 30 °C for 1 h in a MT buffer (50 mM potassium acetate, 50 mM Tris, pH 8.0). Recombinant NTMT1 (1 μg) was mixed with 1.0 μg of full-length recombinant RCC1 and 100 μM SAM, and inhibitors were added to give the indicated concentrations (2, 10, and 50 μM). All His<sub>6</sub>-tagged recombinant proteins were expressed in <i>Escherichia Coli</i> and purified as previously described. The same batches of NTMT1 and RCC1 were used in all experiments. Positive controls contained no inhibitor, while negative controls contained no NTMT1. One-tenth of each reactant was mixed with 5× loading dye and loaded onto a 10% sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) gel. Gels were transferred to nitrocellulose membranes using a Bio-Rad Trans-Blot SD semi-dry transfer cell. Membranes were blocked for 1 h in 5% milk/Tris-buffered saline with Tween 20 (TBST) and then incubated with anti-me3-SPK (1:10,000), anti-me2-SPK (1:5000), or NTMT1 (1:1000) antibodies overnight at 4 °C.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> They were then washed 3 times with 1× TBST and incubated with rabbit IgG-HRP antibody (Jackson ImmunoResearch) for 1 h at room temperature. The membrane was washed 3 times with 1× TBST and detected using a Biorad ChemiDoc imaging system. Image quantification was done using ImageJ software (NIH). For Me3RCC1, all bands were compared to the untreated control, which was set to 1.0. For Me2RCC1, all bands were compared to the highest intensity band, which was set to 1.0. For NTMT1 and RCC1 loading controls, all bands were compared to the average band intensity, which was set to 1.0.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">ITC measurements were performed at 25 °C using a VP-ITC MicroCal calorimeter. Purified NTMT1 and NTMT2 were diluted with ITC buffer (20 mM Tris-HCl, pH 7.5, 150 mM NaCl, and 0.5 mM TCEP) to the final concentration of 30–50 μM, and the compounds were dissolved to 0.5–1 mM in the same buffer. The compound was titrated into the protein solution with 26 injections of 10 μL each. Injections were spaced 180 s with a reference power of 15 μcal/s. The ITC data were processed with Origin 7.0 software (Microcal).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Cell Permeability Evaluation by MALDI</h3><div class="NLM_p last">The colon cancer cell line HCT116 was cultured in McCoy’s media supplemented with 10% FBS and 1% antibiotic–antimycotic. The cells were cultured in a tissue culture dish (Falcon 353003). Cells were maintained in cell culture flasks until seeding into a 24-well tissue culture plate (Falcon 353047). The medium was removed, and the cells were washed with 1× PBS (1 mL) twice, followed by treating with TrypLE Express (1 mL) into a 100 × 20 mm culture flask. The reaction was quenched by the addition of 4 mL of medium and the cells were counted. Cells were seeded into a 24-well tissue culture plate (Falcon 353047) at a density of 0.1 × 10<sup>6</sup> cells/mL and incubated overnight at 37 °C, 5% CO<sub>2</sub>, and 95% humidity, with the lid on. They were then treated with inhibitors at different concentrations and incubation was continued for the specified time. The TAT peptide (GRKKRRQRRR-NH<sub>2</sub>) was also incubated as a control experiment. The medium was removed, and the cells were washed with 1× PBS 3 times to remove any residual compound or peptide attached to the cell surface. Then, 100 μL of 1× PBS was added and the cells were snap-freezed in liquid nitrogen twice. The cell lysate was then analyzed with MALDI using the 2,5-dihydroxybenzoic acid matrix to identify the presence of the compound inside the cell.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cellular N-Methylation Level</h3><div class="NLM_p last">HCT116 cells were seeded as 20,000 cells/well on 24-well plates in the presence of 1× PBS or DC432 at different concentrations for 3 days. Then, cells were lysed in 1× RIPA cell lysis buffer (25 mM Tris-HCl, pH 7.6, 150 mM NaCl, 1% NP-40, 1% sodium deoxycholate, 0.1% SDS, and protease inhibitors) and incubated for 30 min on ice. The cell lysates were centrifuged at 15,000 rpm for 10 min, and the precipitates were removed. The concentration of total protein was quantified using a bicinchoninic acid protein assay kit (Thermo Fisher, #23228). Equal amounts of total protein were mixed with 4× loading dye, loaded onto a 12.5% SDS-PAGE gel, and separated. The gel was transferred to a polyvinylidene difluoride membrane using a Bio-Rad Trans-Blot Turbo system. The membrane was then blocked for 1 h in 5% milk TBST solution and washed with 1× TBST solution 3 times. The membrane was incubated with the anti-me3-SPK antibody (1:5000), lamin B1 antibody (1:2000), SET (1:1000), and RCC1 antibody (1:1000) at 4 °C for 12 h, washed with 1× TBST solution 3 times, and then incubated with rabbit IgG-HRP antibody (1:1000) for 1 h at room temperature. The membrane was again washed with 1× TBST solution 3 times and detected using a Protein Simple FluorChem imaging system. The data were analyzed in GraphPad prism software.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00689" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41660" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41660" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00689?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00689</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">MALDI-MS and HPLC spectra of compounds <b>1</b>–<b>24</b>, <b>DC431</b>, and <b>DC432</b>; IC<sub>50</sub> curve of compound <b>5</b>; IC<sub>50</sub> curve of <b>BM30</b>; cell permeability evaluation; selectivity study of <b>BM30</b> and <b>22</b> against G9a and PRMT1; selectivity study of <b>BM30</b> against a panel of 41 MTs (Reaction Biology); crystallography data and refinement statistics (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>); and IC<sub>50</sub> values and binding affinities of <b>BM30</b> and its cell-permeable analogues (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf">jm0c00689_si_001.pdf (4.08 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_002.csv">jm0c00689_si_002.csv (2.74 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NTMT1 in complex with <b>BM30</b> have been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00689" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75844" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75844" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rong Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Institute of Structural Biology, Drug Discovery
and Development, School of Pharmacy, Virginia
Commonwealth University, Richmond, Virginia 23298, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute
for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a1c9d4c0cfc68cd3e1d1d4d3c5d4c48fc4c5d4"><span class="__cf_email__" data-cfemail="204855414e470d52605055524455450e454455">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brianna D. Mackie</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Institute of Structural Biology, Drug Discovery
and Development, School of Pharmacy, Virginia
Commonwealth University, Richmond, Virginia 23298, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongxing Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute
for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guangping Dong</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute
for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng Dong</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haley Parker</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Jacobs School of Medicine and Biomedical Sciences, University at Buffalo, Buffalo, New York 14203, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christine E. Schaner Tooley</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Jacobs School of Medicine and Biomedical Sciences, University at Buffalo, Buffalo, New York 14203, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas Noinaj</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences, Markey Center for Structural Biology, and
the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jinrong Min</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, Department of Physiology, University of Toronto, Toronto, Ontario M5G 1L7, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26403" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26403" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors acknowledge the support from NIH grants R01GM117275 (R.H.), 1R01GM127896 (N.N.), 1R01AI127793 (N.N.), R01GM112721 (CST), P30 CA023168 (Purdue University Center for Cancer Research), and Natural Sciences and Engineering Research Council Grant RGPIN-2016-06300 (J.M.). The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada through Ontario Genomics Institute [OGI-055], Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA, Darmstadt, Germany, MSD, Novartis Pharma AG, Ontario Ministry of Research, Innovation and Science (MRIS), Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome. GM/<a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="bcfffdfcfdecef">[email protected]</a> was funded in whole or in part by Federal funds from the National Cancer Institute (ACB-12002) and the National Institute of General Medical Sciences (AGM-12006). This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract no. DE-AC02-06CH11357. The authors also thank support from the Department of Medicinal Chemistry and Molecular Pharmacology (R.H.) and Department of Biological Sciences (N.N.) at Purdue University.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">NTMT</td><td class="NLM_def"><p class="first last">protein N-terminal methyltransferase</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last">S-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last">S-5′-adenosyl-L-homocysteine</p></td></tr><tr><td class="NLM_term">SAHH</td><td class="NLM_def"><p class="first last">SAH hydrolase</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36875" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36875" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 34 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery and Chemical Biology of Cancer Epigenetics</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1120</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.chembiol.2017.08.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=28938089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1120-1147&author=S.+Ribichauthor=D.+Harveyauthor=R.+A.+Copeland&title=Drug+Discovery+and+Chemical+Biology+of+Cancer+Epigenetics&doi=10.1016%2Fj.chembiol.2017.08.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery and Chemical Biology of Cancer Epigenetics</span></div><div class="casAuthors">Ribich, Scott; Harvey, Darren; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1120-1147</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Comprehensive whole-exome sequencing, DNA copy-no. detn., and transcriptomic analyses of diverse cancers have greatly expanded our understanding of the biol. of many tumor types.  In addn. to mutations in the common cell-of-origin specific driver mutations, these studies have also revealed a large no. of loss-of-function and gain-of-function mutations in chromatin-modifying proteins (CMPs).  This has revealed that epigenetic dysregulation is a common feature of most pediatric and adult cancers.  Many specific and potent inhibitors have been developed for multiple CMP classes, which have assisted in elucidating the role of epigenetics as well as epigenetic vulnerabilities in these cancer types.  Clin. trials with numerous CMP inhibitors are also currently in progress to evaluate the therapeutic potential of epigenetic inhibitors.  In this review, we aim to provide a summary of genetic mutations in epigenetic genes and a review of CMP inhibitors suitable for preclin. studies or currently in clin. trials.  Addnl., we highlight the CMPs for which potent inhibitors have not been developed and addnl. research focus should be dedicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbPZmoOQ0SgLVg90H21EOLACvtfcHk0lgnUK7oyW7MYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfN&md5=a8454bbe1efc8b386dce299a37f6865e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.020%26sid%3Dliteratum%253Aachs%26aulast%3DRibich%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DD.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DDrug%2520Discovery%2520and%2520Chemical%2520Biology%2520of%2520Cancer%2520Epigenetics%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1120%26epage%3D1147%26doi%3D10.1016%2Fj.chembiol.2017.08.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">A Chemical Biology Toolbox to Study Protein Methyltransferases and Epigenetic Signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30602773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsF2msLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1-14&author=S.+Scheerauthor=S.+Acklooauthor=T.+S.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+A.+Wardauthor=A.+M.+Lewisauthor=J.+P.+Northropauthor=P.+L.+Richardsonauthor=H.+%C3%9C.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+A.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+V.+M.+Huberauthor=D.+D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+J.+Brownauthor=C.+H.+Arrowsmith&title=A+Chemical+Biology+Toolbox+to+Study+Protein+Methyltransferases+and+Epigenetic+Signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide association analysis of self-reported daytime sleepiness identifies 42 loci that suggest biological subtypes</span></div><div class="casAuthors">Wang, Heming; Lane, Jacqueline M.; Jones, Samuel E.; Dashti, Hassan S.; Ollila, Hanna M.; Wood, Andrew R.; van Hees, Vincent T.; Brumpton, Ben; Winsvold, Bendik S.; Kantojarvi, Katri; Palviainen, Teemu; Cade, Brian E.; Sofer, Tamar; Song, Yanwei; Patel, Krunal; Anderson, Simon G.; Bechtold, David A.; Bowden, Jack; Emsley, Richard; Kyle, Simon D.; Little, Max A.; Loudon, Andrew S.; Scheer, Frank A. J. L.; Purcell, Shaun M.; Richmond, Rebecca C.; Spiegelhalder, Kai; Tyrrell, Jessica; Zhu, Xiaofeng; Hublin, Christer; Kaprio, Jaakko A.; Kristiansson, Kati; Sulkava, Sonja; Paunio, Tiina; Hveem, Kristian; Nielsen, Jonas B.; Willer, Cristen J.; Zwart, John-Anker; Strand, Linn B.; Frayling, Timothy M.; Ray, David; Lawlor, Deborah A.; Rutter, Martin K.; Weedon, Michael N.; Redline, Susan; Saxena, Richa</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Excessive daytime sleepiness (EDS) affects 10-20% of the population and is assocd. with substantial functional deficits.  Here, we identify 42 loci for self-reported daytime sleepiness in GWAS of 452,071 individuals from the UK Biobank, with enrichment for genes expressed in brain tissues and in neuronal transmission pathways.  We confirm the aggregate effect of a genetic risk score of 42 SNPs on daytime sleepiness in independent Scandinavian cohorts and on other sleep disorders (restless legs syndrome, insomnia) and sleep traits (duration, chronotype, accelerometer-derived sleep efficiency and daytime naps or inactivity).  However, individual daytime sleepiness signals vary in their assocns. with objective short vs long sleep, and with markers of sleep continuity.  The 42 sleepiness variants primarily cluster into two predominant composite biol. subtypes - sleep propensity and sleep fragmentation.  Shared genetic links are also seen with obesity, coronary heart disease, psychiatric diseases, cognitive traits and reproductive aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjgmeiXjV66rVg90H21EOLACvtfcHk0lgnUK7oyW7MYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsF2msLvK&md5=08a8f2824def2ec24c778e19e10a4e5a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%2BA.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DA%2520Chemical%2520Biology%2520Toolbox%2520to%2520Study%2520Protein%2520Methyltransferases%2520and%2520Epigenetic%2520Signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1%26epage%3D14%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A Selective Inhibitor of PRMT5 with in Vivo and in Vitro Potency in MCL Models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+Selective+Inhibitor+of+PRMT5+with+in+Vivo+and+in+Vitro+Potency+in+MCL+Models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0ljFOeSqIlX4Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520Selective%2520Inhibitor%2520of%2520PRMT5%2520with%2520in%2520Vivo%2520and%2520in%2520Vitro%2520Potency%2520in%2520MCL%2520Models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology of Protein N-Terminal Methyltransferases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">976</span>– <span class="NLM_lpage">984</span>, <span class="refDoi"> DOI: 10.1002/cbic.201800615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1002%2Fcbic.201800615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30479015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivFSksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=976-984&author=R.+Huang&title=Chemical+Biology+of+Protein+N-Terminal+Methyltransferases&doi=10.1002%2Fcbic.201800615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology of Protein N-Terminal Methyltransferases</span></div><div class="casAuthors">Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">976-984</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein α-N-terminal methylation is catalyzed by protein N-terminal methyltransferases.  The prevalent occurrence of this methylation in ribosomes, myosin, and histones implies its function in protein-protein interactions.  Although its full spectrum of function has not yet been outlined, recent discoveries have revealed the emerging roles of α-N-terminal methylation in protein-chromatin interactions, DNA damage repair, and chromosome segregation.  Herein, an overview of the discovery of protein N-terminal methyltransferases and functions of α-N-terminal methylation is presented.  In addn., substrate recognition, mechanisms, and inhibition of N-terminal methyltransferases are reviewed.  Opportunities and gaps in protein α-N-terminal methylation are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUsFs0KjFzkLVg90H21EOLACvtfcHk0ljFOeSqIlX4Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivFSksro%253D&md5=cd70425d136c2a99d95735e59f2a1489</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201800615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201800615%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DChemical%2520Biology%2520of%2520Protein%2520N-Terminal%2520Methyltransferases%26jtitle%3DChemBioChem%26date%3D2019%26volume%3D20%26spage%3D976%26epage%3D984%26doi%3D10.1002%2Fcbic.201800615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann-Liebold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, W.</span></span> <span> </span><span class="NLM_article-title">Occurrence of Methylated Amino Acids as N-Termini of Proteins from Escherichia Coli Ribosomes</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1016/s0022-2836(77)80121-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2FS0022-2836%2877%2980121-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=323502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADyaE2sXkvFOltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1977&pages=173-181&author=R.+Chenauthor=J.+Brosiusauthor=B.+Wittmann-Lieboldauthor=W.+Sch%C3%A4fer&title=Occurrence+of+Methylated+Amino+Acids+as+N-Termini+of+Proteins+from+Escherichia+Coli+Ribosomes&doi=10.1016%2Fs0022-2836%2877%2980121-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Occurrence of methylated amino acids as N-termini of proteins from Escherichia coli ribosomes</span></div><div class="casAuthors">Chen, Robert; Brosius, Juergen; Wittmann-Liebold, Brigitte; Schaefer, Wolfram</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-81</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Ribosomal proteins of E. coli were shown to be methylated at the α-amino group of the protein chain.  The N-termini detd. were N-monomethylalanine in S11 and L33 and N-monomethylmethionine in L16.  The amino acids with secondary amino groups gave low extinction values at 590 nm after reaction with ninhydrin.  Although the above proteins were monomethylated at their N-termini they were accessible to the Edman degrdn. process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT1QeRi3IfqbVg90H21EOLACvtfcHk0lhO47cm-GAjKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXkvFOltLY%253D&md5=c8b51efc765131738e94d5f1efc65a7d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2877%2980121-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252877%252980121-6%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DBrosius%26aufirst%3DJ.%26aulast%3DWittmann-Liebold%26aufirst%3DB.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DW.%26atitle%3DOccurrence%2520of%2520Methylated%2520Amino%2520Acids%2520as%2520N-Termini%2520of%2520Proteins%2520from%2520Escherichia%2520Coli%2520Ribosomes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D111%26spage%3D173%26epage%3D181%26doi%3D10.1016%2Fs0022-2836%2877%2980121-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore-Schroeder, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">NRMT is an α-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1038/nature09343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fnature09343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20668449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1125-1128&author=C.+E.+Schaner%0ATooleyauthor=J.+J.+Petkowskiauthor=T.+L.+Muratore-Schroederauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=M.+Sabatauthor=W.+Minorauthor=D.+F.+Huntauthor=I.+G.+Macara&title=NRMT+is+an+%CE%B1-N-methyltransferase+that+methylates+RCC1+and+retinoblastoma+protein&doi=10.1038%2Fnature09343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT is an α-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span></div><div class="casAuthors">Schaner Tooley, Christine E.; Petkowski, Janusz J.; Muratore-Schroeder, Tara L.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Sabat, Michal; Minor, Wladek; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1125-1128</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The post-translational methylation of α-amino groups was first discovered over 30 years ago on the bacterial ribosomal proteins L16 and L33 (refs 1, 2), but almost nothing is known about the function or enzymol. of this modification.  Several other bacterial and eukaryotic proteins have since been shown to be α-N-methylated.  However, the Ran guanine nucleotide-exchange factor, RCC1, is the only protein for which any biol. function of α-N-methylation has been identified.  Methylation-defective mutants of RCC1 have reduced affinity for DNA and cause mitotic defects, but further characterization of this modification has been hindered by ignorance of the responsible methyltransferase.  All fungal and animal N-terminally methylated proteins contain a unique N-terminal motif, Met-(Ala/Pro/Ser)-Pro-Lys, indicating that they may be targets of the same, unknown enzyme.  The initiating Met is cleaved, and the exposed α-amino group is mono-, di- or trimethylated.  Here, we report the discovery of the first α-N-methyltransferase, which we named N-terminal RCC1 methyltransferase (NRMT).  Substrate docking and mutational anal. of RCC1 defined the NRMT recognition sequence and enabled the identification of numerous new methylation targets, including SET (also known as TAF-I or PHAPII) and the retinoblastoma protein, RB.  Knockdown of NRMT recapitulates the multi-spindle phenotype seen with methylation-defective RCC1 mutants, demonstrating the importance of α-N-methylation for normal bipolar spindle formation and chromosome segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo90urCnl8qTrVg90H21EOLACvtfcHk0lhO47cm-GAjKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D&md5=185740cc031677ea3f6a20654546824e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature09343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09343%26sid%3Dliteratum%253Aachs%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DMuratore-Schroeder%26aufirst%3DT.%2BL.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DNRMT%2520is%2520an%2520%25CE%25B1-N-methyltransferase%2520that%2520methylates%2520RCC1%2520and%2520retinoblastoma%2520protein%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1125%26epage%3D1128%26doi%3D10.1038%2Fnature09343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipson, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hadid, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitelegge, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">5225</span>– <span class="NLM_lpage">5235</span>, <span class="refDoi"> DOI: 10.1021/bi100428x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100428x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=5225-5235&author=K.+J.+Webbauthor=R.+S.+Lipsonauthor=Q.+Al-Hadidauthor=J.+P.+Whiteleggeauthor=S.+G.+Clarke&title=Identification+of+Protein+N-Terminal+Methyltransferases+in+Yeast+and+Humans&doi=10.1021%2Fbi100428x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span></div><div class="casAuthors">Webb, Kristofor J.; Lipson, Rebecca S.; Al-Hadid, Qais; Whitelegge, Julian P.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5225-5235</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein modification by methylation is important in cellular function.  We show here that the Saccharomyces cerevisiae YBR261C/TAE1 gene encodes an N-terminal protein methyltransferase catalyzing the modification of two ribosomal protein substrates, Rpl12ab and Rps25a/Rps25b.  The YBR261C/Tae1 protein is conserved across eukaryotes; all of these proteins share sequence similarity with known seven β-strand class I methyltransferases.  Wild-type yeast cytosol and mouse heart cytosol catalyze the methylation of a synthetic peptide (PPKQQLSKY) that contains the first eight amino acids of the processed N-terminus of Rps25a/Rps25b.  However, no methylation of this peptide is seen in yeast cytosol from a ΔYBR261C/tae1 deletion strain.  Yeast YBR261C/TAE1 and the human orthologue METTL11A genes were expressed as fusion proteins in Escherichia coli and were shown to be capable of stoichiometrically dimethylating the N-terminus of the synthetic peptide.  Furthermore, the YBR261C/Tae1 and METTL11A recombinant proteins methylate variants of the synthetic peptide contg. N-terminal alanine and serine residues.  However, methyltransferase activity is largely abolished when the proline residue in position 2 or the lysine residue in position 3 is substituted.  Thus, the methyltransferases described here specifically recognize the N-terminal X-Pro-Lys sequence motif, and we suggest designating the yeast enzyme Ntm1 and the human enzyme NTMT1.  These enzymes may account for nearly all previously described eukaryotic protein N-terminal methylation reactions.  A no. of other yeast and human proteins also share the recognition motif and may be similarly modified.  We conclude that protein X-Pro-Lys N-terminal methylation reactions catalyzed by the enzymes described here may be widespread in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiYjjV6t89rVg90H21EOLACvtfcHk0lhO47cm-GAjKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D&md5=deb28aa90896b14013bdf60e8b1410c3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbi100428x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100428x%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DK.%2BJ.%26aulast%3DLipson%26aufirst%3DR.%2BS.%26aulast%3DAl-Hadid%26aufirst%3DQ.%26aulast%3DWhitelegge%26aufirst%3DJ.%2BP.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DIdentification%2520of%2520Protein%2520N-Terminal%2520Methyltransferases%2520in%2520Yeast%2520and%2520Humans%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D5225%26epage%3D5235%26doi%3D10.1021%2Fbi100428x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkey, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT2 Is an N-Terminal Monomethylase That Primes for Its Homologue NRMT1</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>456</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1042/bj20131163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1042%2FBJ20131163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=24090352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVartrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=456&publication_year=2013&pages=453-462&author=J.+J.+Petkowskiauthor=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=D.+W.+Wilkeyauthor=M.+L.+Merchantauthor=I.+G.+Macaraauthor=C.+E.+Schaner+Tooley&title=NRMT2+Is+an+N-Terminal+Monomethylase+That+Primes+for+Its+Homologue+NRMT1&doi=10.1042%2Fbj20131163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT2 is an N-terminal monomethylase that primes for its homologue NRMT1</span></div><div class="casAuthors">Petkowski, Janusz J.; Bonsignore, Lindsay A.; Tooley, John G.; Wilkey, Daniel W.; Merchant, Michael L.; Macara, Ian G.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">456</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">453-462</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">NRMT (N-terminal regulator of chromatin condensation 1 methyltransferase) was the first eukaryotic methyltransferase identified to specifically methylate the free α-amino group of proteins.  Since the discovery of this N-terminal methyltransferase, many new substrates have been identified and the modification itself has been shown to regulate DNA-protein interactions.  Sequence anal. predicts one close human homolog of NRMT, METTL11B (methyltransferase-like protein 11B, now renamed NRMT2).  We show in the present paper for the first time that NRMT2 also has N-terminal methylation activity and recognizes the same N-terminal consensus sequences as NRMT (now NRMT1).  Both enzymes have similar tissue expression and cellular localization patterns.  However, enzyme assays and MS expts. indicate that they differ in their specific catalytic functions.  Although NRMT1 is a distributive methyltransferase that can mono-, di- and tri-methylate its substrates, NRMT2 is primarily a monomethylase.  Concurrent expression of NRMT1 and NRMT2 accelerates the prodn. of trimethylation, and we propose that NRMT2 activates NRMT1 by priming its substrates for trimethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopynrYAOFFVrVg90H21EOLACvtfcHk0lgZgZxG9mNpvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVartrzK&md5=20aebdfed2ae0676ee918c77f829a68d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20131163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20131163%26sid%3Dliteratum%253Aachs%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DWilkey%26aufirst%3DD.%2BW.%26aulast%3DMerchant%26aufirst%3DM.%2BL.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT2%2520Is%2520an%2520N-Terminal%2520Monomethylase%2520That%2520Primes%2520for%2520Its%2520Homologue%2520NRMT1%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D456%26spage%3D453%26epage%3D462%26doi%3D10.1042%2Fbj20131163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0lgZgZxG9mNpvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Histone N-Terminal Methylation by NRMT1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1101/gad.270926.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1101%2Fgad.270926.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=26543159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2337-2342&author=R.+Wuauthor=Y.+Yueauthor=X.+Zhengauthor=H.+Li&title=Molecular+Basis+for+Histone+N-Terminal+Methylation+by+NRMT1&doi=10.1101%2Fgad.270926.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for histone N-terminal methylation by NRMT1</span></div><div class="casAuthors">Wu, Ruoxi; Yue, Yuan; Zheng, Xiangdong; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2337-2342</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">NRMT1 is an N-terminal methyltransferase that methylates histone CENP-A as well as nonhistone substrates.  Here, we report the crystal structure of human NRMT1 bound to CENP-A peptide at 1.3 Å.  NRMT1 adopts a core methyltransferase fold that resembles DOT1L and PRMT but not SET domain family histone methyltransferases.  Key substrate recognition and catalytic residues were identified by mutagenesis studies.  Histone peptide profiling revealed that human NRMT1 is highly selective to human CENP-A and fruit fly H2B, which share a common "Xaa-Pro-Lys/Arg" motif.  These results, along with a 1.5 Å costructure of human NRMT1 bound to the fruit fly H2B peptide, underscore the importance of the NRMT1 recognition motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGIuplVjQhULVg90H21EOLACvtfcHk0lgZgZxG9mNpvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J&md5=f3601b0966da6c89d0acf9aeae4799be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.270926.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270926.115%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DR.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520Basis%2520for%2520Histone%2520N-Terminal%2520Methylation%2520by%2520NRMT1%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2337%26epage%3D2342%26doi%3D10.1101%2Fgad.270926.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">An Asparagine/Glycine Switch Governs Product Specificity of Human N-Terminal Methyltransferase NTMT2</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">183</span>, <span class="refDoi"> DOI: 10.1038/s42003-018-0196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fs42003-018-0196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30417120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=183&author=C.+Dongauthor=G.+Dongauthor=L.+Liauthor=L.+Zhuauthor=W.+Tempelauthor=Y.+Liuauthor=R.+Huangauthor=J.+Min&title=An+Asparagine%2FGlycine+Switch+Governs+Product+Specificity+of+Human+N-Terminal+Methyltransferase+NTMT2&doi=10.1038%2Fs42003-018-0196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An asparagine/glycine switch governs product specificity of human N-terminal methyltransferase NTMT2</span></div><div class="casAuthors">Dong Cheng; Li Li; Zhu Licheng; Tempel Wolfram; Liu Yanli; Min Jinrong; Dong Guangping; Huang Rong; Zhu Licheng; Min Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">α-N-terminal methylation of proteins is an important post-translational modification that is catalyzed by two different N-terminal methyltransferases, namely NTMT1 and NTMT2.  Previous studies have suggested that NTMT1 is a tri-methyltransferase, whereas NTMT2 is a mono-methyltransferase.  Here, we report the first crystal structures, to our knowledge, of NTMT2 in binary complex with S-adenosyl-L-methionine as well as in ternary complex with S-adenosyl-L-homocysteine and a substrate peptide.  Our structural observations combined with biochemical studies reveal that NTMT2 is also able to di-/tri-methylate the GPKRIA peptide and di-methylate the PPKRIA peptide, otherwise it is predominantly a mono-methyltransferase.  The residue N89 of NTMT2 serves as a gatekeeper residue that regulates the binding of unmethylated versus monomethylated substrate peptide.  Structural comparison of NTMT1 and NTMT2 prompts us to design a N89G mutant of NTMT2 that can profoundly alter its catalytic activities and product specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKFvwVHUgF0kwlCcfExbvbfW6udTcc2eb-r4geVd325Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D&md5=edc8c87334f423ab98872442f81d7928</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0196-2%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DAn%2520Asparagine%252FGlycine%2520Switch%2520Governs%2520Product%2520Specificity%2520of%2520Human%2520N-Terminal%2520Methyltransferase%2520NTMT2%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26spage%3D183%26doi%3D10.1038%2Fs42003-018-0196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakobsson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Małecki, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabelian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudithipudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davydova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuhairi, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidel, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falnes, P. Ø.</span></span> <span> </span><span class="NLM_article-title">The Dual Methyltransferase METTL13 Targets N Terminus and Lys55 of EEF1A and Modulates Codon-Specific Translation Rates</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3411</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-05646-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fs41467-018-05646-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30143613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BB3c3hs1OjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3411&author=M.+E.+Jakobssonauthor=J.+M.+Ma%C5%82eckiauthor=L.+Halabelianauthor=B.+S.+Nilgesauthor=R.+Pintoauthor=S.+Kudithipudiauthor=S.+Munkauthor=E.+Davydovaauthor=F.+R.+Zuhairiauthor=C.+H.+Arrowsmithauthor=A.+Jeltschauthor=S.+A.+Leidelauthor=J.+V.+Olsenauthor=P.+%C3%98.+Falnes&title=The+Dual+Methyltransferase+METTL13+Targets+N+Terminus+and+Lys55+of+EEF1A+and+Modulates+Codon-Specific+Translation+Rates&doi=10.1038%2Fs41467-018-05646-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The dual methyltransferase METTL13 targets N terminus and Lys55 of eEF1A and modulates codon-specific translation rates</span></div><div class="casAuthors">Jakobsson Magnus E; Malecki Jedrzej M; Pinto Rita; Davydova Erna; Zuhairi Fawzi R; Falnes Pal O; Jakobsson Magnus E; Munk Stephanie; Olsen Jesper V; Halabelian Levon; Arrowsmith Cheryl H; Nilges Benedikt S; Leidel Sebastian A; Nilges Benedikt S; Leidel Sebastian A; Kudithipudi Srikanth; Jeltsch Albert</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3411</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Eukaryotic elongation factor 1 alpha (eEF1A) delivers aminoacyl-tRNA to the ribosome and thereby plays a key role in protein synthesis.  Human eEF1A is subject to extensive post-translational methylation, but several of the responsible enzymes remain unknown.  Using a wide range of experimental approaches, we here show that human methyltransferase (MTase)-like protein 13 (METTL13) contains two distinct MTase domains targeting the N terminus and Lys55 of eEF1A, respectively.  Our biochemical and structural analyses provide detailed mechanistic insights into recognition of the eEF1A N terminus by METTL13.  Moreover, through ribosome profiling, we demonstrate that loss of METTL13 function alters translation dynamics and results in changed translation rates of specific codons.  In summary, we here unravel the function of a human MTase, showing that it methylates eEF1A and modulates mRNA translation in a codon-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSnalPvS2TRnBLDWw4HIq2sfW6udTcc2eb-r4geVd325Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3hs1OjsQ%253D%253D&md5=b336056ca88a555d488f1f2ae2b48efd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05646-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05646-y%26sid%3Dliteratum%253Aachs%26aulast%3DJakobsson%26aufirst%3DM.%2BE.%26aulast%3DMa%25C5%2582ecki%26aufirst%3DJ.%2BM.%26aulast%3DHalabelian%26aufirst%3DL.%26aulast%3DNilges%26aufirst%3DB.%2BS.%26aulast%3DPinto%26aufirst%3DR.%26aulast%3DKudithipudi%26aufirst%3DS.%26aulast%3DMunk%26aufirst%3DS.%26aulast%3DDavydova%26aufirst%3DE.%26aulast%3DZuhairi%26aufirst%3DF.%2BR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJeltsch%26aufirst%3DA.%26aulast%3DLeidel%26aufirst%3DS.%2BA.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DFalnes%26aufirst%3DP.%2B%25C3%2598.%26atitle%3DThe%2520Dual%2520Methyltransferase%2520METTL13%2520Targets%2520N%2520Terminus%2520and%2520Lys55%2520of%2520EEF1A%2520and%2520Modulates%2520Codon-Specific%2520Translation%2520Rates%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D3411%26doi%3D10.1038%2Fs41467-018-05646-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lofgren, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulea, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tandoc, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporicci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topisirovic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Rechem, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">METTL13 Methylation of EEF1A Increases Translational Output to Promote Tumorigenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>176</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">504.e21</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.11.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.cell.2018.11.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30612740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1Wjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2019&pages=491-504.e21&author=S.+Liuauthor=S.+Hausmannauthor=S.+M.+Carlsonauthor=M.+E.+Fuentesauthor=J.+W.+Francisauthor=R.+Pillaiauthor=S.+M.+Lofgrenauthor=L.+Huleaauthor=K.+Tandocauthor=J.+Luauthor=A.+Liauthor=N.+D.+Nguyenauthor=M.+Caporicciauthor=M.+P.+Kimauthor=A.+Maitraauthor=H.+Wangauthor=I.+I.+Wistubaauthor=J.+A.+Porcoauthor=M.+C.+Bassikauthor=J.+E.+Eliasauthor=J.+Songauthor=I.+Topisirovicauthor=C.+Van+Rechemauthor=P.+K.+Mazurauthor=O.+Gozani&title=METTL13+Methylation+of+EEF1A+Increases+Translational+Output+to+Promote+Tumorigenesis&doi=10.1016%2Fj.cell.2018.11.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">METTL13 Methylation of eEF1A Increases Translational Output to Promote Tumorigenesis</span></div><div class="casAuthors">Liu, Shuo; Hausmann, Simone; Carlson, Scott Moore; Fuentes, Mary Esmeralda; Francis, Joel William; Pillai, Renjitha; Lofgren, Shane Michael; Hulea, Laura; Tandoc, Kristofferson; Lu, Jiuwei; Li, Ami; Nguyen, Nicholas Dang; Caporicci, Marcello; Kim, Michael Paul; Maitra, Anirban; Wang, Huamin; Wistuba, Ignacio Ivan; Porco, John Anthony, Jr.; Bassik, Michael Cory; Elias, Joshua Eric; Song, Jikui; Topisirovic, Ivan; Van Rechem, Capucine; Mazur, Pawel Karol; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">491-504.e21</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Increased protein synthesis plays an etiol. role in diverse cancers.  Here, we demonstrate that METTL13 (methyltransferase-like 13) dimethylation of eEF1A (eukaryotic elongation factor 1A) lysine 55 (eEF1AK55me2) is utilized by Ras-driven cancers to increase translational output and promote tumorigenesis in vivo.  METTL13-catalyzed eEF1A methylation increases eEF1A's intrinsic GTPase activity in vitro and protein prodn. in cells.  METTL13 and eEF1AK55me2 levels are upregulated in cancer and neg. correlate with pancreatic and lung cancer patient survival.  METTL13 deletion and eEF1AK55me2 loss dramatically reduce Ras-driven neoplastic growth in mouse models and in patient-derived xenografts (PDXs) from primary pancreatic and lung tumors.  Finally, METTL13 depletion renders PDX tumors hypersensitive to drugs that target growth-signaling pathways.  Together, our work uncovers a mechanism by which lethal cancers become dependent on the METTL13-eEF1AK55me2 axis to meet their elevated protein synthesis requirement and suggests that METTL13 inhibition may constitute a targetable vulnerability of tumors driven by aberrant Ras signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrETutXdxNsQbVg90H21EOLACvtfcHk0lhiBXhEEbcQAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1Wjsg%253D%253D&md5=a5d12ace96f50aacff361b625414627c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.11.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.11.038%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DHausmann%26aufirst%3DS.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DFrancis%26aufirst%3DJ.%2BW.%26aulast%3DPillai%26aufirst%3DR.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DHulea%26aufirst%3DL.%26aulast%3DTandoc%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DN.%2BD.%26aulast%3DCaporicci%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DM.%2BP.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DElias%26aufirst%3DJ.%2BE.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DTopisirovic%26aufirst%3DI.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DMETTL13%2520Methylation%2520of%2520EEF1A%2520Increases%2520Translational%2520Output%2520to%2520Promote%2520Tumorigenesis%26jtitle%3DCell%26date%3D2019%26volume%3D176%26spage%3D491%26epage%3D504.e21%26doi%3D10.1016%2Fj.cell.2018.11.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyogoku, Y.</span></span> <span> </span><span class="NLM_article-title">N-Trimethylalanine, a Novel Blocked N-Terminal Residue of Tetrahymena Histone H2B1</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1675</span>– <span class="NLM_lpage">1678</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.jbchem.a134096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1093%2Foxfordjournals.jbchem.a134096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=6818230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADyaL3sXhvV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1982&pages=1675-1678&author=M.+Nomotoauthor=Y.+Kyogoku&title=N-Trimethylalanine%2C+a+Novel+Blocked+N-Terminal+Residue+of+Tetrahymena+Histone+H2B1&doi=10.1093%2Foxfordjournals.jbchem.a134096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">N-trimethylalanine, a novel blocked N-terminal residue of Tetrahymena histone H2B</span></div><div class="casAuthors">Nomoto, Minoru; Kyogoku, Yoshimasa; Iwai, Koichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry </span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1675-8</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    </div><div class="casAbstract">A previous study showed that the amino acid sequence of T. pyriformis histone H2B has a blocked N-terminal residue followed by 119 amino acid residues (Nomoto, M., et al., 1982).  The blocking group has been found to be N-trimethylalanine by 1H NMR spectroscopy of the isolated N-terminal tripeptide X-Pro-Lys and X.  This identification was supported by mass spectrometry of X-Pro-Lys and by column and paper chromatog. of X.  This is the 1st time that N-terminal blocking by methylation has ever been found in histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcvCBH0SBWNLVg90H21EOLACvtfcHk0lhtT5R0isCGUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhvV2h&md5=506196f48eabffe7a38c6c86c5959b36</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.jbchem.a134096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.jbchem.a134096%26sid%3Dliteratum%253Aachs%26aulast%3DNomoto%26aufirst%3DM.%26aulast%3DKyogoku%26aufirst%3DY.%26atitle%3DN-Trimethylalanine%252C%2520a%2520Novel%2520Blocked%2520N-Terminal%2520Residue%2520of%2520Tetrahymena%2520Histone%2520H2B1%26jtitle%3DJ.%2520Biochem.%26date%3D1982%26volume%3D92%26spage%3D1675%26epage%3D1678%26doi%3D10.1093%2Foxfordjournals.jbchem.a134096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sathyan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fachinetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foltz, D. R.</span></span> <span> </span><span class="NLM_article-title">α-Amino Trimethylation of CENP-A by NRMT Is Required for Full Recruitment of the Centromere</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14678</span>, <span class="refDoi"> DOI: 10.1038/ncomms14678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fncomms14678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=28266506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14678&author=K.+M.+Sathyanauthor=D.+Fachinettiauthor=D.+R.+Foltz&title=%CE%B1-Amino+Trimethylation+of+CENP-A+by+NRMT+Is+Required+for+Full+Recruitment+of+the+Centromere&doi=10.1038%2Fncomms14678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">α-amino trimethylation of CENP-A by NRMT is required for full recruitment of the centromere</span></div><div class="casAuthors">Sathyan Kizhakke M; Foltz Daniel R; Fachinetti Daniele; Foltz Daniel R; Foltz Daniel R</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14678</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Centromeres are unique chromosomal domains that control chromosome segregation, and are epigenetically specified by the presence of the CENP-A containing nucleosomes.  CENP-A governs centromere function by recruiting the constitutive centromere associated network (CCAN) complex.  The features of the CENP-A nucleosome necessary to distinguish centromeric chromatin from general chromatin are not completely understood.  Here we show that CENP-A undergoes α-amino trimethylation by the enzyme NRMT in vivo.  We show that α-amino trimethylation of the CENP-A tail contributes to cell survival.  Loss of α-amino trimethylation causes a reduction in the CENP-T and CENP-I CCAN components at the centromere and leads to lagging chromosomes and spindle pole defects.  The function of p53 alters the response of cells to defects associated with decreased CENP-A methylation.  Altogether we show an important functional role for α-amino trimethylation of the CENP-A nucleosome in maintaining centromere function and faithful chromosomes segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_zBuwd5i4JcNIRw0GKUoEfW6udTcc2eZvHyTLJxdaELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D&md5=7fb75e88a08eb5d87b0c84aa6c62c4ab</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fncomms14678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14678%26sid%3Dliteratum%253Aachs%26aulast%3DSathyan%26aufirst%3DK.%2BM.%26aulast%3DFachinetti%26aufirst%3DD.%26aulast%3DFoltz%26aufirst%3DD.%2BR.%26atitle%3D%25CE%25B1-Amino%2520Trimethylation%2520of%2520CENP-A%2520by%2520NRMT%2520Is%2520Required%2520for%2520Full%2520Recruitment%2520of%2520the%2520Centromere%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14678%26doi%3D10.1038%2Fncomms14678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner-Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">N-terminal α-methylation of RCC1 is necessary for stable chromatin association and normal mitosis</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">596</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1038/ncb1572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fncb1572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=17435751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=596-603&author=T.+Chenauthor=T.+L.+Muratoreauthor=C.+E.+Schaner-Tooleyauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=I.+G.+Macara&title=N-terminal+%CE%B1-methylation+of+RCC1+is+necessary+for+stable+chromatin+association+and+normal+mitosis&doi=10.1038%2Fncb1572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">N-terminal α-methylation of RCC1 is necessary for stable chromatin association and normal mitosis</span></div><div class="casAuthors">Chen, Ting; Muratore, Tara L.; Schaner-Tooley, Christine E.; Shabanowitz, Jeffrey; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">596-603</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Regulator of chromatin condensation 1 (RCC1) is the only known guanine nucleotide-exchange factor for the Ran GTPase and has pivotal roles in nucleo-cytoplasmic transport, mitosis, and nuclear-envelope assembly.  RCC1 assocs. dynamically with chromatin through binding to histones H2A and/or H2B in a Ran-regulated manner.  Here, we report that, unexpectedly, the amino-terminal serine or proline residue of RCC1 is uniquely methylated on its α-amino group.  Methylation requires removal of the initiating methionine, and the presence of proline and lysine at positions 3 and 4, resp.  Methylation-defective mutants of RCC1 bind less effectively than wild-type protein to chromatin during mitosis, which causes spindle-pole defects.  We propose a bimodal attachment mechanism for RCC1 in which the tail promotes stable RCC1 assocn. with chromatin through DNA binding in an α-N-methylation-dependent manner.  These data provide the first known function for N-terminal protein methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Cy1xI_cXwrVg90H21EOLACvtfcHk0lhtT5R0isCGUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D&md5=91f3adb84da3958ec1bd66cb91e660a4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fncb1572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1572%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DMuratore%26aufirst%3DT.%2BL.%26aulast%3DSchaner-Tooley%26aufirst%3DC.%2BE.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DN-terminal%2520%25CE%25B1-methylation%2520of%2520RCC1%2520is%2520necessary%2520for%2520stable%2520chromatin%2520association%2520and%2520normal%2520mitosis%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D596%26epage%3D603%26doi%3D10.1038%2Fncb1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">α-N-Methylation of Damaged DNA-binding Protein 2 (DDB2) and Its Function in Nucleotide Excision Repair</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">16046</span>– <span class="NLM_lpage">16056</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.558510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1074%2Fjbc.M114.558510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=24753253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=16046-16056&author=Q.+Caiauthor=L.+Fuauthor=Z.+Wangauthor=N.+Ganauthor=X.+Daiauthor=Y.+Wang&title=%CE%B1-N-Methylation+of+Damaged+DNA-binding+Protein+2+%28DDB2%29+and+Its+Function+in+Nucleotide+Excision+Repair&doi=10.1074%2Fjbc.m114.558510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">α-N-Methylation of damaged DNA-binding protein 2 (DDB2) and its function in nucleotide excision repair</span></div><div class="casAuthors">Cai, Qian; Fu, Lijuan; Wang, Zi; Gan, Nanqin; Dai, Xiaoxia; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16046-16056</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">DDB2 exhibits a high affinity toward UV-damaged DNA, and it is involved in the initial steps of global genome nucleotide excision repair.  Mutations in the DDB2 gene cause the genetic complementation group E of xeroderma pigmentosum, an autosomal recessive disease manifested clin. by hypersensitivity to sunlight exposure and an increased predisposition to skin cancer.  Here the authors found that, in human cells, the initiating methionine residue in DDB2 was removed and that the N-terminal alanine could be methylated on its α-amino group in human cells, with trimethylation being the major form.  The α-N-methylation of DDB2 is catalyzed by the N-terminal RCC1 methyltransferase.  In addn., a methylation-defective mutant of DDB2 displayed diminished nuclear localization and was recruited at a reduced efficiency to UV-induced cyclobutane pyrimidine dimer foci.  Moreover, loss of this methylation conferred compromised ATM (ataxia telangiectasia mutated) activation, decreased efficiency in cyclobutane pyrimidine dimer repair, and elevated sensitivity of cells toward UV light exposure.  This study provides new knowledge about the posttranslational regulation of DDB2 and expands the biol. functions of protein α-N-methylation to DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc3yE8db8RO7Vg90H21EOLACvtfcHk0lhdi2S9x_vrRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D&md5=e52a0c303e3420df4722d42cb0478af1</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558510%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGan%26aufirst%3DN.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3D%25CE%25B1-N-Methylation%2520of%2520Damaged%2520DNA-binding%2520Protein%25202%2520%2528DDB2%2529%2520and%2520Its%2520Function%2520in%2520Nucleotide%2520Excision%2520Repair%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D16046%26epage%3D16056%26doi%3D10.1074%2Fjbc.m114.558510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulten, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2575-2582&author=X.+Daiauthor=S.+L.+Rultenauthor=C.+Youauthor=K.+W.+Caldecottauthor=Y.+Wang&title=Identification+and+Functional+Characterizations+of+N-Terminal+%CE%B1-N-Methylation+and+Phosphorylation+of+Serine+461+in+Human+Poly%28ADP-ribose%29+Polymerase+3&doi=10.1021%2Facs.jproteome.5b00126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span></div><div class="casAuthors">Dai, Xiaoxia; Rulten, Stuart L.; You, Changjun; Caldecott, Keith W.; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase 3 (PARP3) is a member of the PARP family enzymes which catalyze the ADP-ribosylation of proteins.  PARP3 plays an important role in DNA damage repair and mitotic progression.  In this study, we identified, using mass spectrometric (MS) techniques, two novel post-translational modification sites in PARP3, α-N-methylation and phosphorylation of serine 461 (S461).  We found that the N-terminal α-amino group of PARP3 is heavily methylated in human cells, and N-terminal RCC1 methyltransferase (NRMT) is a key enzyme required for this methylation.  We also obsd. that the phosphorylation level of S461 in PARP3 could be reduced in human cells upon treatment with flavopiridol, a cyclin-dependent kinase inhibitor.  Moreover, we demonstrated that S461 phosphorylation, but not α-N-methylation of PARP3, may be involved in the cellular response toward DNA double-strand breaks (DSBs).  These findings provide novel insights into the post-translational regulation of PARP3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0mFKyXPLnrVg90H21EOLACvtfcHk0lhdi2S9x_vrRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D&md5=6ff05a312e38b2105b71fbbd17388e64</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00126%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DRulten%26aufirst%3DS.%2BL.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520and%2520Functional%2520Characterizations%2520of%2520N-Terminal%2520%25CE%25B1-N-Methylation%2520and%2520Phosphorylation%2520of%2520Serine%2520461%2520in%2520Human%2520Poly%2528ADP-ribose%2529%2520Polymerase%25203%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Facs.jproteome.5b00126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shields, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkey, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbett, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">Select Human Cancer Mutants of NRMT1 Alter Its Catalytic Activity and Decrease N-Terminal Trimethylation</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1639</span>– <span class="NLM_lpage">1652</span>, <span class="refDoi"> DOI: 10.1002/pro.3202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1002%2Fpro.3202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=28556566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFSqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=1639-1652&author=K.+M.+Shieldsauthor=J.+G.+Tooleyauthor=J.+J.+Petkowskiauthor=D.+W.+Wilkeyauthor=N.+C.+Garbettauthor=M.+L.+Merchantauthor=A.+Chengauthor=C.+E.+Schaner+Tooley&title=Select+Human+Cancer+Mutants+of+NRMT1+Alter+Its+Catalytic+Activity+and+Decrease+N-Terminal+Trimethylation&doi=10.1002%2Fpro.3202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Select human cancer mutants of NRMT1 alter its catalytic activity and decrease N-terminal trimethylation</span></div><div class="casAuthors">Shields, Kaitlyn M.; Tooley, John G.; Petkowski, Janusz J.; Wilkey, Daniel W.; Garbett, Nichola C.; Merchant, Michael L.; Cheng, Alan; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1639-1652</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A subset of B-cell lymphoma patients have dominant mutations in the histone H3 lysine 27 (H3K27) methyltransferase EZH2, which change it from a monomethylase to a trimethylase.  These mutations occur in arom. resides surrounding the active site and increase growth and alter transcription.  We study the N-terminal trimethylase NRMT1 and the N-terminal monomethylase NRMT2.  They are 50% identical, but differ in key arom. residues in their active site.  Given how these residues affect EZH2 activity, we tested whether they are responsible for the distinct catalytic activities of NRMT1/2.  Addnl., NRMT1 acts as a tumor suppressor in breast cancer cells.  Its loss promotes oncogenic phenotypes but sensitizes cells to DNA damage.  Mutations of NRMT1 naturally occur in human cancers, and we tested a select group for altered activity.  While directed mutation of the arom. residues had minimal catalytic effect, NRMT1 mutants N209I (endometrial cancer) and P211S (lung cancer) displayed decreased trimethylase and increased monomethylase/dimethylase activity.  Both mutations are located in the peptide-binding channel and indicate a second structural region impacting enzyme specificity.  The NRMT1 mutants demonstrated a slower rate of trimethylation and a requirement for higher substrate concn.  Expression of the mutants in wild type NRMT backgrounds showed no change in N-terminal methylation levels or growth rates, demonstrating they are not acting as dominant negatives.  Expression of the mutants in cells lacking endogenous NRMT1 resulted in minimal accumulation of N-terminal trimethylation, indicating homozygosity could help drive oncogenesis or serve as a marker for sensitivity to DNA damaging chemotherapeutics or γ-irradn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxowEFm1MLl7Vg90H21EOLACvtfcHk0lhdi2S9x_vrRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsFSqtr0%253D&md5=3d2c8424f37b210daf5d689ef70ef464</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.3202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3202%26sid%3Dliteratum%253Aachs%26aulast%3DShields%26aufirst%3DK.%2BM.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DWilkey%26aufirst%3DD.%2BW.%26aulast%3DGarbett%26aufirst%3DN.%2BC.%26aulast%3DMerchant%26aufirst%3DM.%2BL.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DSelect%2520Human%2520Cancer%2520Mutants%2520of%2520NRMT1%2520Alter%2520Its%2520Catalytic%2520Activity%2520and%2520Decrease%2520N-Terminal%2520Trimethylation%26jtitle%3DProtein%2520Sci.%26date%3D2017%26volume%3D26%26spage%3D1639%26epage%3D1652%26doi%3D10.1002%2Fpro.3202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoose, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT1 Knockout Mice Exhibit Phenotypes Associated with Impaired DNA Repair and Premature Aging</span>. <i>Mech. Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>146–148</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2015.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.mad.2015.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25843235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146%E2%80%93148&publication_year=2015&pages=42-52&author=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=P.+M.+Van+Hooseauthor=E.+Wangauthor=A.+Chengauthor=M.+P.+Coleauthor=C.+E.+Schaner+Tooley&title=NRMT1+Knockout+Mice+Exhibit+Phenotypes+Associated+with+Impaired+DNA+Repair+and+Premature+Aging&doi=10.1016%2Fj.mad.2015.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT1 knockout mice exhibit phenotypes associated with impaired DNA repair and premature aging</span></div><div class="casAuthors">Bonsignore, Lindsay A.; Tooley, John G.; Van Hoose, Patrick M.; Wang, Eugenia; Cheng, Alan; Cole, Marsha P.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">146-148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-52</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Though defective genome maintenance and DNA repair have long been known to promote phenotypes of premature aging, the role protein methylation plays in these processes is only now emerging.  We have recently identified the first N-terminal methyltransferase, NRMT1, which regulates protein-DNA interactions and is necessary for both accurate mitotic division and nucleotide excision repair.  To demonstrate if complete loss of NRMT1 subsequently resulted in developmental or aging phenotypes, we constructed the first NRMT1 knockout (Nrmt1-/-) mouse.  The majority of these mice die shortly after birth.  However, the ones that survive, exhibit decreased body size, female-specific infertility, kyphosis, decreased mitochondrial function, and early-onset liver degeneration; phenotypes characteristic of other mouse models deficient in DNA repair.  The livers from Nrmt1-/- mice produce less reactive oxygen species (ROS) than wild type controls, and Nrmt1-/- mouse embryonic fibroblasts show a decreased capacity for handling oxidative damage.  This indicates that decreased mitochondrial function may benefit Nrmt1-/- mice and protect them from excess internal ROS and subsequent DNA damage.  These studies position the NRMT1 knockout mouse as a useful new system for studying the effects of genomic instability and defective DNA damage repair on organismal and tissue-specific aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWx010KT-fbrVg90H21EOLACvtfcHk0lhYgGerqwfckg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D&md5=501bd0e3dade5777c3965efb8fe10b93</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2015.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2015.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DVan%2BHoose%26aufirst%3DP.%2BM.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DM.%2BP.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT1%2520Knockout%2520Mice%2520Exhibit%2520Phenotypes%2520Associated%2520with%2520Impaired%2520DNA%2520Repair%2520and%2520Premature%2520Aging%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2015%26volume%3D146%25E2%2580%2593148%26spage%3D42%26epage%3D52%26doi%3D10.1016%2Fj.mad.2015.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitakomate, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. R.</span></span> <span> </span><span class="NLM_article-title">The Methylated N-Terminal Tail of RCC1 Is Required for Stabilisation of Its Interaction with Chromatin by Ran in Live Cells</span>. <i>BMC Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">43</span>, <span class="refDoi"> DOI: 10.1186/1471-2121-11-43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1186%2F1471-2121-11-43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20565941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=43&author=E.+Hitakomateauthor=F.+E.+Hoodauthor=H.+S.+Sandersonauthor=P.+R.+Clarke&title=The+Methylated+N-Terminal+Tail+of+RCC1+Is+Required+for+Stabilisation+of+Its+Interaction+with+Chromatin+by+Ran+in+Live+Cells&doi=10.1186%2F1471-2121-11-43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The methylated N-terminal tail of RCC1 is required for stabilisation of its interaction with chromatin by Ran in live cells</span></div><div class="casAuthors">Hitakomate Ekarat; Hood Fiona E; Sanderson Helen S; Clarke Paul R</div><div class="citationInfo"><span class="NLM_cas:title">BMC cell biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Regulator of chromosome condensation 1 (RCC1) is the guanine nucleotide exchange factor for Ran GTPase.  Localised generation of Ran-GTP by RCC1 on chromatin is critical for nucleocytoplasmic transport, mitotic spindle assembly and nuclear envelope formation.  Both the N-terminal tail of RCC1 and its association with Ran are important for its interaction with chromatin in cells.  In vitro, the association of Ran with RCC1 induces a conformational change in the N-terminal tail that promotes its interaction with DNA.  RESULTS:  We have investigated the mechanism of the dynamic interaction of the alpha isoform of human RCC1 (RCC1alpha) with chromatin in live cells using fluorescence recovery after photobleaching (FRAP) of green fluorescent protein (GFP) fusions.  We show that the N-terminal tail stabilises the interaction of RCC1alpha with chromatin and this function can be partially replaced by another lysine-rich nuclear localisation signal.  Removal of the tail prevents the interaction of RCC1alpha with chromatin from being stabilised by RanT24N, a mutant that binds stably to RCC1alpha.  The interaction of RCC1alpha with chromatin is destabilised by mutation of lysine 4 (K4Q), which abolishes alpha-N-terminal methylation, and this interaction is no longer stabilised by RanT24N.  However, alpha-N-terminal methylation of RCC1alpha is not regulated by the binding of RanT24N.  Conversely, the association of Ran with precipitated RCC1alpha does not require the N-terminal tail of RCC1alpha or its methylation.  The mobility of RCC1alpha on chromatin is increased by mutation of aspartate 182 (D182A), which inhibits guanine-nucleotide exchange activity, but RCC1alphaD182A can still bind nucleotide-free Ran and its interaction with chromatin is stabilised by RanT24N.  CONCLUSIONS:  These results show that the stabilisation of the dynamic interaction of RCC1alpha with chromatin by Ran in live cells requires the N-terminal tail of RCC1alpha. alpha-N-methylation is not regulated by formation of the binary complex with Ran, but it promotes chromatin binding through the tail.  This work supports a model in which the association of RCC1alpha with chromatin is promoted by a conformational change in the alpha-N-terminal methylated tail that is induced allosterically in the binary complex with Ran.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQSyjm2mWbK-0g6wdEZmT2rfW6udTcc2eZSlnR8T0Fq37ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D&md5=2bfb8e66cc72441bfd68eb8827caf717</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2F1471-2121-11-43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2121-11-43%26sid%3Dliteratum%253Aachs%26aulast%3DHitakomate%26aufirst%3DE.%26aulast%3DHood%26aufirst%3DF.%2BE.%26aulast%3DSanderson%26aufirst%3DH.%2BS.%26aulast%3DClarke%26aufirst%3DP.%2BR.%26atitle%3DThe%2520Methylated%2520N-Terminal%2520Tail%2520of%2520RCC1%2520Is%2520Required%2520for%2520Stabilisation%2520of%2520Its%2520Interaction%2520with%2520Chromatin%2520by%2520Ran%2520in%2520Live%2520Cells%26jtitle%3DBMC%2520Cell%2520Biol.%26date%3D2010%26volume%3D11%26spage%3D43%26doi%3D10.1186%2F1471-2121-11-43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1074%2Fjbc.M114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.m114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0lhYgGerqwfckg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.m114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/c5ob00120j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1039%2FC5OB00120J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2Fc5ob00120j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0lht0mS2KWiGvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC5OB00120J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00120J%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2Fc5ob00120j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Facile Synthesis of SAM-Peptide Conjugates through Alkyl Linkers Targeting Protein N-Terminal Methyltransferase 1</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6771</span>, <span class="refDoi"> DOI: 10.1039/c5ra20625a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1039%2FC5RA20625A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=27588169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=6768-6771&author=G.+Zhangauthor=R.+Huang&title=Facile+Synthesis+of+SAM-Peptide+Conjugates+through+Alkyl+Linkers+Targeting+Protein+N-Terminal+Methyltransferase+1&doi=10.1039%2Fc5ra20625a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of SAM-peptide conjugates through alkyl linkers targeting protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Zhang, Gang; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">6768-6771</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report the first chem. synthesis of SAM-peptide (SAM = S-adenosyl-L-methionine) conjugates through alkyl linkers to prep. bisubstrate analogs for protein methyltransferases.  The authors demonstrate its application by developing a series of bisubstrate inhibitors for protein N-terminal methyltransferase 1 (NTMT1) and the most potent one exhibits a Ki = 310 ± 55 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_MNtkSc3Ne7Vg90H21EOLACvtfcHk0lht0mS2KWiGvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D&md5=e267ad97193784986c32b86c2e1ae88a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC5RA20625A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5RA20625A%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DFacile%2520Synthesis%2520of%2520SAM-Peptide%2520Conjugates%2520through%2520Alkyl%2520Linkers%2520Targeting%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D6768%26epage%3D6771%26doi%3D10.1039%2Fc5ra20625a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0lht0mS2KWiGvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Probing the Plasticity in the Active Site of Protein N-Terminal Methyltransferase 1 Using Bisubstrate Analogs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00770</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00770" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=D.+Chenauthor=C.+Dongauthor=G.+Dongauthor=K.+Srinivasanauthor=J.+Minauthor=N.+Noinajauthor=R.+Huang&title=Probing+the+Plasticity+in+the+Active+Site+of+Protein+N-Terminal+Methyltransferase+1+Using+Bisubstrate+Analogs&doi=10.1021%2Facs.jmedchem.0c00770"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00770%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DSrinivasan%26aufirst%3DK.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DProbing%2520the%2520Plasticity%2520in%2520the%2520Active%2520Site%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Using%2520Bisubstrate%2520Analogs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shumilin, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">Substrate Specificity of Mammalian N-Terminal α-Amino Methyltransferase NRMT</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5942</span>– <span class="NLM_lpage">5950</span>, <span class="refDoi"> DOI: 10.1021/bi300278f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi300278f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5942-5950&author=J.+J.+Petkowskiauthor=C.+E.+Schaner+Tooleyauthor=L.+C.+Andersonauthor=I.+A.+Shumilinauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=W.+Minorauthor=I.+G.+Macara&title=Substrate+Specificity+of+Mammalian+N-Terminal+%CE%B1-Amino+Methyltransferase+NRMT&doi=10.1021%2Fbi300278f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Specificity of Mammalian N-Terminal α-Amino Methyltransferase NRMT</span></div><div class="casAuthors">Petkowski, Janusz J.; Schaner Tooley, Christine E.; Anderson, Lissa C.; Shumilin, Igor A.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Hunt, Donald F.; Minor, Wladek; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5942-5950</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-Terminal methylation of free α-amino groups is a post-translational modification of proteins that was first described 30 years ago but has been studied very little.  In this modification, the initiating M residue is cleaved and the exposed α-amino group is mono-, di-, or trimethylated by NRMT, a recently identified N-terminal methyltransferase.  Currently, all known eukaryotic α-amino-methylated proteins have a unique N-terminal motif, M-X-P-K, where X is A, P, or S.  NRMT can also methylate artificial substrates in vitro in which X is G, F, Y, C, M, K, R, N, Q, or H.  Methylation efficiencies of N-terminal amino acids are variable with respect to the identity of X.  Here we use in vitro peptide methylation assays and substrate immunopptns. to show that the canonical M-X-P-K methylation motif is not the only one recognized by NRMT.  We predict that N-terminal methylation is a widespread post-translational modification and that there is interplay between N-terminal acetylation and N-terminal methylation.  We also use isothermal calorimetry expts. to demonstrate that NRMT can efficiently recognize and bind to its fully methylated products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8dl8ByIopbVg90H21EOLACvtfcHk0liCWw5PhXaFLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D&md5=dbddebac4bc6f6045fd144604e157bdb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fbi300278f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi300278f%26sid%3Dliteratum%253Aachs%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DAnderson%26aufirst%3DL.%2BC.%26aulast%3DShumilin%26aufirst%3DI.%2BA.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DSubstrate%2520Specificity%2520of%2520Mammalian%2520N-Terminal%2520%25CE%25B1-Amino%2520Methyltransferase%2520NRMT%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5942%26epage%3D5950%26doi%3D10.1021%2Fbi300278f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackie, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">A Direct, Ratiometric, and Quantitative MALDI-MS Assay for Protein Methyltransferases and Acetyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.ab.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25778392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2015&pages=59-64&author=S.+L.+Richardsonauthor=P.+Hanjraauthor=G.+Zhangauthor=B.+D.+Mackieauthor=D.+L.+Petersonauthor=R.+Huang&title=A+Direct%2C+Ratiometric%2C+and+Quantitative+MALDI-MS+Assay+for+Protein+Methyltransferases+and+Acetyltransferases&doi=10.1016%2Fj.ab.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A direct, ratiometric, and quantitative MALDI-MS assay for protein methyltransferases and acetyltransferases</span></div><div class="casAuthors">Richardson, Stacie L.; Hanjra, Pahul; Zhang, Gang; Mackie, Brianna D.; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-64</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein methylation and acetylation play important roles in biol. processes, and misregulation of these modifications is involved in various diseases.  Therefore, it is crit. to understand the activities of the enzymes responsible for these modifications.  Herein we describe a sensitive method for ratiometric quantification of methylated and acetylated peptides via MALDI-MS by direct spotting of enzymic methylation and acetylation reaction mixts. without tedious purifn. procedures.  The quantifiable detection limit for peptides with our method is approx. 10 fmol.  This is achieved by increasing the signal-to-noise ratio through the addn. of NH4H2PO4 to the matrix soln. and redn. of the matrix α-cyanohydroxycinnamic acid concn. to 2 mg/mL.  We have demonstrated the application of this method in enzyme kinetic anal. and inhibition studies.  The unique feature of this method is the simultaneous quantification of multiple peptide species for investigation of processivity mechanisms.  Its wide buffer compatibility makes it possible to be adapted to investigate the activity of any protein methyltransferase or acetyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2PCzex5Lwq7Vg90H21EOLACvtfcHk0liCWw5PhXaFLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D&md5=15c0b43193742c5abddf9a4f0d9ba488</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DMackie%26aufirst%3DB.%2BD.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DA%2520Direct%252C%2520Ratiometric%252C%2520and%2520Quantitative%2520MALDI-MS%2520Assay%2520for%2520Protein%2520Methyltransferases%2520and%2520Acetyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2015%26volume%3D478%26spage%3D59%26epage%3D64%26doi%3D10.1016%2Fj.ab.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Poli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peggion, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Formaggio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broxterman, Q. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniolo, C.</span></span> <span> </span><span class="NLM_article-title">Is the Backbone Conformation of Cα-Methyl Proline Restricted to a Single Region?</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">8015</span>– <span class="NLM_lpage">8025</span>, <span class="refDoi"> DOI: 10.1002/chem.200900688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1002%2Fchem.200900688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=19579242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1yrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=8015-8025&author=M.+De%0APoliauthor=A.+Morettoauthor=M.+Crismaauthor=C.+Peggionauthor=F.+Formaggioauthor=B.+Kapteinauthor=Q.+B.+Broxtermanauthor=C.+Toniolo&title=Is+the+Backbone+Conformation+of+C%CE%B1-Methyl+Proline+Restricted+to+a+Single+Region%3F&doi=10.1002%2Fchem.200900688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Is the backbone conformation of Cα-methyl proline restricted to a single region?</span></div><div class="casAuthors">De Poli, Matteo; Moretto, Alessandro; Crisma, Marco; Peggion, Cristina; Formaggio, Fernando; Kaptein, Bernard; Broxterman, Quirinus B.; Toniolo, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">8015-8025, S8015/1-S8015/6</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cα-Methyl-L-proline, or L-(αMe)Pro, is probably the most conformationally constrained α-amino acid.  In particular, its ω and φ torsion angles are restricted to about 180 and -60°, resp., and only three ranges of values are theor. available for ψ in mono- or longer peptides, namely, about -30° (cis', 310/α-helical structure), 60° (inverse γ turn), or 140° (trans', poly(L-Pro)n II structure).  In this work, we examd. the tendency of a no. of Nα-acyl dipeptide N'-alkylamides of the type RCO-(αMe)Pro-Xxx-NHR' or RCO-Xxx-(αMe)Pro-NHR', in which Xxx is L (or D)-Ala, Aib (α-aminoisoburyric acid), or L (or D)-(αMe)Pro, long enough to fold into intramolecularly hydrogen-bonded γ or β turns.  The results are compared with those obtained for the corresponding dipeptides based on Pro, a well-known turn-forming residue.  For the crystal-state 3D-structural anal. we used X-ray diffraction, whereas our soln. conformational anal. was heavily based on the FTIR absorption and 1H and 13C NMR spectroscopy techniques.  We conclude that (αMe)Pro is able to explore both trans' and cis' ψ areas of the conformational space, but in (αMe)Pro the latter is overwhelmingly more populated, in marked contrast to the Pro preference.  This finding is a clear indication that in (αMe)Pro the major 3D-structural determinant is the Cα-Me group.  The CD signature of a peptide type III' β-turn conformation is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiFIxKnGqbxrVg90H21EOLACvtfcHk0liCWw5PhXaFLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1yrt7k%253D&md5=bd76d96642ed6ee9f9d6eba5f47ace4d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fchem.200900688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200900688%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BPoli%26aufirst%3DM.%26aulast%3DMoretto%26aufirst%3DA.%26aulast%3DCrisma%26aufirst%3DM.%26aulast%3DPeggion%26aufirst%3DC.%26aulast%3DFormaggio%26aufirst%3DF.%26aulast%3DKaptein%26aufirst%3DB.%26aulast%3DBroxterman%26aufirst%3DQ.%2BB.%26aulast%3DToniolo%26aufirst%3DC.%26atitle%3DIs%2520the%2520Backbone%2520Conformation%2520of%2520C%25CE%25B1-Methyl%2520Proline%2520Restricted%2520to%2520a%2520Single%2520Region%253F%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2009%26volume%3D15%26spage%3D8015%26epage%3D8025%26doi%3D10.1002%2Fchem.200900688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyamu, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10783</span>– <span class="NLM_lpage">10797</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10783-10797&author=D.+Chenauthor=L.+Liauthor=K.+Diazauthor=I.+D.+Iyamuauthor=R.+Yadavauthor=N.+Noinajauthor=R.+Huang&title=Novel+Propargyl-Linked+Bisubstrate+Analogues+as+Tight-Binding+Inhibitors+for+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.jmedchem.9b01255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Novel propargyl-linked bisubstrate analogues as tight-binding inhibitors for nicotinamide N-methyltransferase</span></div><div class="casAuthors">Chen, Dongxing; Li, Linjie; Diaz, Krystal; Iyamu, Iredia D.; Yadav, Ravi; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10783-10797</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the Me transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-contg. compds.  NNMT is an important regulator for nicotinamide metab. and methylation potential.  Aberrant expression levels of NNMT have been implicated in cancer, metabolic, and neurodegenerative diseases, which makes NNMT a potential therapeutic target.  Therefore, potent and selective NNMT inhibitors can serve as valuable tools to investigate the roles of NNMT in its mediated diseases.  Here, we applied a rational strategy to design and synthesize the tight-binding bisubstrate inhibitor LL320 through a novel propargyl linker.  LL320 demonstrates a Ki value of 1.6 ± 0.3 nM, which is the most potent inhibitor to date.  The cocrystal structure of LL320 confirms its interaction with both the substrate and cofactor binding sites on NNMT.  Importantly, this is the first example of using the propargyl linker to construct potent methyltransferase inhibitors, which will expand our understanding of the transition state of Me transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs2XyawAg9ebVg90H21EOLACvtfcHk0ljV9aQVRPJxDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN&md5=a6e3a5dc2044dc50ede1596c84804f61</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DDiaz%26aufirst%3DK.%26aulast%3DIyamu%26aufirst%3DI.%2BD.%26aulast%3DYadav%26aufirst%3DR.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DNovel%2520Propargyl-Linked%2520Bisubstrate%2520Analogues%2520as%2520Tight-Binding%2520Inhibitors%2520for%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10783%26epage%3D10797%26doi%3D10.1021%2Facs.jmedchem.9b01255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briesewitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhiang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Efficient Delivery of Cyclic Peptides into Mammalian Cells with Short Sequence Motifs</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1021/cb3005275</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3005275" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1akur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=423-431&author=Z.+Qianauthor=T.+Liuauthor=Y.-Y.+Liuauthor=R.+Briesewitzauthor=A.+M.+Barriosauthor=S.+M.+Jhiangauthor=D.+Pei&title=Efficient+Delivery+of+Cyclic+Peptides+into+Mammalian+Cells+with+Short+Sequence+Motifs&doi=10.1021%2Fcb3005275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Delivery of Cyclic Peptides into Mammalian Cells with Short Sequence Motifs</span></div><div class="casAuthors">Qian, Ziqing; Liu, Tao; Liu, Yu-Yu; Briesewitz, Roger; Barrios, Amy M.; Jhiang, Sissy M.; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">423-431</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptides hold great potential as therapeutic agents and research tools, but their broad application has been limited by poor membrane permeability.  Here, we report a potentially general approach for intracellular delivery of cyclic peptides.  Short peptide motifs rich in arginine and hydrophobic residues (e.g., FΦRRRR, where Φ is l-2-naphthylalanine), when embedded into small- to medium-sized cyclic peptides (7-13 amino acids), bound to the plasma membrane of mammalian cultured cells and were subsequently internalized by the cells.  Confocal microscopy and a newly developed peptide internalization assay demonstrated that cyclic peptides contg. these transporter motifs were translocated into the cytoplasm and nucleus at efficiencies 2-5-fold higher than that of nonaarginine (R9).  Furthermore, incorporation of the FΦRRRR motif into a cyclic peptide contg. a phosphocoumaryl aminopropionic acid (pCAP) residue generated a cell permeable, fluorogenic probe for detecting intracellular protein tyrosine phosphatase activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouUMFM_wJk9bVg90H21EOLACvtfcHk0ljV9aQVRPJxDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1akur%252FO&md5=80f723598b2afcf4503bbb026a7290e9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcb3005275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3005275%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.-Y.%26aulast%3DBriesewitz%26aufirst%3DR.%26aulast%3DBarrios%26aufirst%3DA.%2BM.%26aulast%3DJhiang%26aufirst%3DS.%2BM.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DEfficient%2520Delivery%2520of%2520Cyclic%2520Peptides%2520into%2520Mammalian%2520Cells%2520with%2520Short%2520Sequence%2520Motifs%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D423%26epage%3D431%26doi%3D10.1021%2Fcb3005275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span>; <span class="NLM_string-name">Ashton, A.</span></span> <i>Fast_dp. Methods</i>; <span class="NLM_publisher-name">Zenodo</span>, <span class="NLM_year">2014</span>; Vol.  <span class="NLM_volume">55</span>, pp  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Winter&author=A.+Ashton&title=Fast_dp.+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26jtitle%3DFast_dp.%2520Methods%26pub%3DZenodo%26date%3D2014%26volume%3D55%26spage%3D81%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhDlI9FauSSsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhDlI9FauSSsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Guangping Dong, Youchao Deng, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based Discovery of Cell-Potent Peptidomimetic Inhibitors for Protein N-Terminal Methyltransferase 1. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (3)
                                     , 485-493. <a href="https://doi.org/10.1021/acsmedchemlett.1c00012" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00012%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DStructure-based%252BDiscovery%252Bof%252BCell-Potent%252BPeptidomimetic%252BInhibitors%252Bfor%252BProtein%252BN-Terminal%252BMethyltransferase%252B1%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06012021%26date%3D24022021%26date%3D01032021%26volume%3D12%26issue%3D3%26spage%3D485%26epage%3D493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ion John  Campeanu</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Jiang</span>, <span class="hlFld-ContribAuthor ">Lanxin  Liu</span>, <span class="hlFld-ContribAuthor ">Maksymilian  Pilecki</span>, <span class="hlFld-ContribAuthor ">Alvina  Najor</span>, <span class="hlFld-ContribAuthor ">Era  Cobani</span>, <span class="hlFld-ContribAuthor ">Morenci  Manning</span>, <span class="hlFld-ContribAuthor ">Xiaohong Mary  Zhang</span>, <span class="hlFld-ContribAuthor ">Zeng-Quan  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-omics integration of methyltransferase-like protein family reveals clinical outcomes and functional signatures in human cancer. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-94019-5" title="DOI URL">https://doi.org/10.1038/s41598-021-94019-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-94019-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-94019-5%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DMulti-omics%252Bintegration%252Bof%252Bmethyltransferase-like%252Bprotein%252Bfamily%252Breveals%252Bclinical%252Boutcomes%252Band%252Bfunctional%252Bsignatures%252Bin%252Bhuman%252Bcancer%26aulast%3DCampeanu%26aufirst%3DIon%2BJohn%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krystal  Diaz</span>, <span class="hlFld-ContribAuthor ">Ying  Meng</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Past, present, and perspectives of protein N-terminal methylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>63 </em>, 115-122. <a href="https://doi.org/10.1016/j.cbpa.2021.02.017" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.02.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.02.017%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DPast%25252C%252Bpresent%25252C%252Band%252Bperspectives%252Bof%252Bprotein%252BN-terminal%252Bmethylation%26aulast%3DDiaz%26aufirst%3DKrystal%26date%3D2021%26volume%3D63%26spage%3D115%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design inhibitors based on the structure of NTMT1 peptide substrates. H-bond interactions are shown in red dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selectivity of the NTMT inhibitor <b>BM30</b>. Selectivity of <b>BM30</b> against a panel of 41 PKMTs, PRMTs, DNMTs, and NTMT1/2 was determined at a concentration of 100 μM in duplicate (<i>n</i> = 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Inhibition mechanism of <b>BM30</b>. All experiments were run in triplicate (<i>n</i> = 3). (A) IC<sub>50</sub> curves of <b>BM30</b> at varying concentrations of the NTMT1 peptide substrate, RCC1-6 (SPKRIA), with a fixed concentration of SAM; (B) linear regression plot of IC<sub>50</sub> values with corresponding concentrations of RCC1-6; (C) IC<sub>50</sub> curves of <b>BM30</b> at varying concentrations of SAM with a fixed concentration of RCC1-6; (D) plot of IC<sub>50</sub> values with corresponding concentrations of SAM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of NTMT1 (gray cartoon)–<b>BM30</b> (green stick)–SAH (green stick) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>). (A) Fo-Fc omit density map of ligands. (B) Zoomed view of interactions of <b>BM30</b> with NTMT1. H-bond interactions are shown as yellow dotted lines. (C) <b>BM30</b> interaction diagram (Schrödinger Maestro) with NTMT1. (D,E) Structural alignment of NTMT1 (gray)–<b>BM30</b> (green stick)–SAH (green stick) and NTMT1 (yellow cartoon)–YPKRIA (yellow stick)–SAH (yellow stick) (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1D">5E1D</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Biochemical characterization of <b>DC431</b> and <b>DC432</b>. (A,B) IC<sub>50</sub> curves of <b>DC431</b> and <b>DC432</b>. IC<sub>50</sub> values were determined in triplicate (<i>n</i> = 3) and presented as mean <u class="uu">±</u> SD. (C) Representative western blots showing N-terminal dimethylation (Me<sub>2</sub>RCC1) and trimethylation (Me<sub>3</sub>RCC1) of full-length recombinant human RCC1 by full-length recombinant human NTMT1 in the presence of increasing concentrations of <b>BM30</b>, <b>DC431</b>, and <b>DC432</b> (<i>n</i> = 3). NTMT1 and RCC1 blots are shown as loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding affinity for NTMT1 (A) (<i>n</i> = 2) and NTMT2 (B) (<i>n</i> = 1).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/medium/jm0c00689_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Cellular N-terminal methylation assay of compound DC432 in HCT116 cells. (A) Structure of DC432. (B) Representative western blot results of effects of DC432 (0–1 mM) on the cellular methylation level (<i>n</i> = 2). RCC1, SET, and Lamin B1 blots are shown as loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00689/20200903/images/large/jm0c00689_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00689&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i34">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57611" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57611" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 34 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery and Chemical Biology of Cancer Epigenetics</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1120</span>– <span class="NLM_lpage">1147</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.08.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.chembiol.2017.08.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=28938089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1120-1147&author=S.+Ribichauthor=D.+Harveyauthor=R.+A.+Copeland&title=Drug+Discovery+and+Chemical+Biology+of+Cancer+Epigenetics&doi=10.1016%2Fj.chembiol.2017.08.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery and Chemical Biology of Cancer Epigenetics</span></div><div class="casAuthors">Ribich, Scott; Harvey, Darren; Copeland, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1120-1147</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Comprehensive whole-exome sequencing, DNA copy-no. detn., and transcriptomic analyses of diverse cancers have greatly expanded our understanding of the biol. of many tumor types.  In addn. to mutations in the common cell-of-origin specific driver mutations, these studies have also revealed a large no. of loss-of-function and gain-of-function mutations in chromatin-modifying proteins (CMPs).  This has revealed that epigenetic dysregulation is a common feature of most pediatric and adult cancers.  Many specific and potent inhibitors have been developed for multiple CMP classes, which have assisted in elucidating the role of epigenetics as well as epigenetic vulnerabilities in these cancer types.  Clin. trials with numerous CMP inhibitors are also currently in progress to evaluate the therapeutic potential of epigenetic inhibitors.  In this review, we aim to provide a summary of genetic mutations in epigenetic genes and a review of CMP inhibitors suitable for preclin. studies or currently in clin. trials.  Addnl., we highlight the CMPs for which potent inhibitors have not been developed and addnl. research focus should be dedicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbPZmoOQ0SgLVg90H21EOLACvtfcHk0lgsmk5QFY-xTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFGgurfN&md5=a8454bbe1efc8b386dce299a37f6865e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.08.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.08.020%26sid%3Dliteratum%253Aachs%26aulast%3DRibich%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DD.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DDrug%2520Discovery%2520and%2520Chemical%2520Biology%2520of%2520Cancer%2520Epigenetics%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1120%26epage%3D1147%26doi%3D10.1016%2Fj.chembiol.2017.08.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ackloo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northrop, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zepeda-Velazquez, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Carvalho, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span> <span> </span><span class="NLM_article-title">A Chemical Biology Toolbox to Study Protein Methyltransferases and Epigenetic Signaling</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-07905-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fs41467-018-07905-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30602773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsF2msLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1-14&author=S.+Scheerauthor=S.+Acklooauthor=T.+S.+Medinaauthor=M.+Schapiraauthor=F.+Liauthor=J.+A.+Wardauthor=A.+M.+Lewisauthor=J.+P.+Northropauthor=P.+L.+Richardsonauthor=H.+%C3%9C.+Kaniskanauthor=Y.+Shenauthor=J.+Liuauthor=D.+Smilauthor=D.+McLeodauthor=C.+A.+Zepeda-Velazquezauthor=M.+Luoauthor=J.+Jinauthor=D.+Barsyte-Lovejoyauthor=K.+V.+M.+Huberauthor=D.+D.+De+Carvalhoauthor=M.+Vedadiauthor=C.+Zaphauthor=P.+J.+Brownauthor=C.+H.+Arrowsmith&title=A+Chemical+Biology+Toolbox+to+Study+Protein+Methyltransferases+and+Epigenetic+Signaling&doi=10.1038%2Fs41467-018-07905-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide association analysis of self-reported daytime sleepiness identifies 42 loci that suggest biological subtypes</span></div><div class="casAuthors">Wang, Heming; Lane, Jacqueline M.; Jones, Samuel E.; Dashti, Hassan S.; Ollila, Hanna M.; Wood, Andrew R.; van Hees, Vincent T.; Brumpton, Ben; Winsvold, Bendik S.; Kantojarvi, Katri; Palviainen, Teemu; Cade, Brian E.; Sofer, Tamar; Song, Yanwei; Patel, Krunal; Anderson, Simon G.; Bechtold, David A.; Bowden, Jack; Emsley, Richard; Kyle, Simon D.; Little, Max A.; Loudon, Andrew S.; Scheer, Frank A. J. L.; Purcell, Shaun M.; Richmond, Rebecca C.; Spiegelhalder, Kai; Tyrrell, Jessica; Zhu, Xiaofeng; Hublin, Christer; Kaprio, Jaakko A.; Kristiansson, Kati; Sulkava, Sonja; Paunio, Tiina; Hveem, Kristian; Nielsen, Jonas B.; Willer, Cristen J.; Zwart, John-Anker; Strand, Linn B.; Frayling, Timothy M.; Ray, David; Lawlor, Deborah A.; Rutter, Martin K.; Weedon, Michael N.; Redline, Susan; Saxena, Richa</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Excessive daytime sleepiness (EDS) affects 10-20% of the population and is assocd. with substantial functional deficits.  Here, we identify 42 loci for self-reported daytime sleepiness in GWAS of 452,071 individuals from the UK Biobank, with enrichment for genes expressed in brain tissues and in neuronal transmission pathways.  We confirm the aggregate effect of a genetic risk score of 42 SNPs on daytime sleepiness in independent Scandinavian cohorts and on other sleep disorders (restless legs syndrome, insomnia) and sleep traits (duration, chronotype, accelerometer-derived sleep efficiency and daytime naps or inactivity).  However, individual daytime sleepiness signals vary in their assocns. with objective short vs long sleep, and with markers of sleep continuity.  The 42 sleepiness variants primarily cluster into two predominant composite biol. subtypes - sleep propensity and sleep fragmentation.  Shared genetic links are also seen with obesity, coronary heart disease, psychiatric diseases, cognitive traits and reproductive aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjgmeiXjV66rVg90H21EOLACvtfcHk0lgsmk5QFY-xTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsF2msLvK&md5=08a8f2824def2ec24c778e19e10a4e5a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07905-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07905-4%26sid%3Dliteratum%253Aachs%26aulast%3DScheer%26aufirst%3DS.%26aulast%3DAckloo%26aufirst%3DS.%26aulast%3DMedina%26aufirst%3DT.%2BS.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DWard%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DA.%2BM.%26aulast%3DNorthrop%26aufirst%3DJ.%2BP.%26aulast%3DRichardson%26aufirst%3DP.%2BL.%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMcLeod%26aufirst%3DD.%26aulast%3DZepeda-Velazquez%26aufirst%3DC.%2BA.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DDe%2BCarvalho%26aufirst%3DD.%2BD.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3DA%2520Chemical%2520Biology%2520Toolbox%2520to%2520Study%2520Protein%2520Methyltransferases%2520and%2520Epigenetic%2520Signaling%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1%26epage%3D14%26doi%3D10.1038%2Fs41467-018-07905-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan-Penebre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuplast, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boriack-Sjodin, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioux, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munchhof, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraj, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stickland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribich, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pappalardi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurse, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oza, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duncan, K. W.</span></span> <span> </span><span class="NLM_article-title">A Selective Inhibitor of PRMT5 with in Vivo and in Vitro Potency in MCL Models</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">432</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+Selective+Inhibitor+of+PRMT5+with+in+Vivo+and+in+Vitro+Potency+in+MCL+Models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0ljeAW952GcqLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520Selective%2520Inhibitor%2520of%2520PRMT5%2520with%2520in%2520Vivo%2520and%2520in%2520Vitro%2520Potency%2520in%2520MCL%2520Models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology of Protein N-Terminal Methyltransferases</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">976</span>– <span class="NLM_lpage">984</span>, <span class="refDoi"> DOI: 10.1002/cbic.201800615</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1002%2Fcbic.201800615" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30479015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivFSksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=976-984&author=R.+Huang&title=Chemical+Biology+of+Protein+N-Terminal+Methyltransferases&doi=10.1002%2Fcbic.201800615"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology of Protein N-Terminal Methyltransferases</span></div><div class="casAuthors">Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">976-984</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein α-N-terminal methylation is catalyzed by protein N-terminal methyltransferases.  The prevalent occurrence of this methylation in ribosomes, myosin, and histones implies its function in protein-protein interactions.  Although its full spectrum of function has not yet been outlined, recent discoveries have revealed the emerging roles of α-N-terminal methylation in protein-chromatin interactions, DNA damage repair, and chromosome segregation.  Herein, an overview of the discovery of protein N-terminal methyltransferases and functions of α-N-terminal methylation is presented.  In addn., substrate recognition, mechanisms, and inhibition of N-terminal methyltransferases are reviewed.  Opportunities and gaps in protein α-N-terminal methylation are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUsFs0KjFzkLVg90H21EOLACvtfcHk0ljeAW952GcqLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivFSksro%253D&md5=cd70425d136c2a99d95735e59f2a1489</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201800615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201800615%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DChemical%2520Biology%2520of%2520Protein%2520N-Terminal%2520Methyltransferases%26jtitle%3DChemBioChem%26date%3D2019%26volume%3D20%26spage%3D976%26epage%3D984%26doi%3D10.1002%2Fcbic.201800615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann-Liebold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, W.</span></span> <span> </span><span class="NLM_article-title">Occurrence of Methylated Amino Acids as N-Termini of Proteins from Escherichia Coli Ribosomes</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1016/s0022-2836(77)80121-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2FS0022-2836%2877%2980121-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=323502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADyaE2sXkvFOltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1977&pages=173-181&author=R.+Chenauthor=J.+Brosiusauthor=B.+Wittmann-Lieboldauthor=W.+Sch%C3%A4fer&title=Occurrence+of+Methylated+Amino+Acids+as+N-Termini+of+Proteins+from+Escherichia+Coli+Ribosomes&doi=10.1016%2Fs0022-2836%2877%2980121-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Occurrence of methylated amino acids as N-termini of proteins from Escherichia coli ribosomes</span></div><div class="casAuthors">Chen, Robert; Brosius, Juergen; Wittmann-Liebold, Brigitte; Schaefer, Wolfram</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-81</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Ribosomal proteins of E. coli were shown to be methylated at the α-amino group of the protein chain.  The N-termini detd. were N-monomethylalanine in S11 and L33 and N-monomethylmethionine in L16.  The amino acids with secondary amino groups gave low extinction values at 590 nm after reaction with ninhydrin.  Although the above proteins were monomethylated at their N-termini they were accessible to the Edman degrdn. process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT1QeRi3IfqbVg90H21EOLACvtfcHk0lgJM19S1rqMHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXkvFOltLY%253D&md5=c8b51efc765131738e94d5f1efc65a7d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2877%2980121-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252877%252980121-6%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DBrosius%26aufirst%3DJ.%26aulast%3DWittmann-Liebold%26aufirst%3DB.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DW.%26atitle%3DOccurrence%2520of%2520Methylated%2520Amino%2520Acids%2520as%2520N-Termini%2520of%2520Proteins%2520from%2520Escherichia%2520Coli%2520Ribosomes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D111%26spage%3D173%26epage%3D181%26doi%3D10.1016%2Fs0022-2836%2877%2980121-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore-Schroeder, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">NRMT is an α-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1038/nature09343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fnature09343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20668449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1125-1128&author=C.+E.+Schaner%0ATooleyauthor=J.+J.+Petkowskiauthor=T.+L.+Muratore-Schroederauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=M.+Sabatauthor=W.+Minorauthor=D.+F.+Huntauthor=I.+G.+Macara&title=NRMT+is+an+%CE%B1-N-methyltransferase+that+methylates+RCC1+and+retinoblastoma+protein&doi=10.1038%2Fnature09343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT is an α-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span></div><div class="casAuthors">Schaner Tooley, Christine E.; Petkowski, Janusz J.; Muratore-Schroeder, Tara L.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Sabat, Michal; Minor, Wladek; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1125-1128</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The post-translational methylation of α-amino groups was first discovered over 30 years ago on the bacterial ribosomal proteins L16 and L33 (refs 1, 2), but almost nothing is known about the function or enzymol. of this modification.  Several other bacterial and eukaryotic proteins have since been shown to be α-N-methylated.  However, the Ran guanine nucleotide-exchange factor, RCC1, is the only protein for which any biol. function of α-N-methylation has been identified.  Methylation-defective mutants of RCC1 have reduced affinity for DNA and cause mitotic defects, but further characterization of this modification has been hindered by ignorance of the responsible methyltransferase.  All fungal and animal N-terminally methylated proteins contain a unique N-terminal motif, Met-(Ala/Pro/Ser)-Pro-Lys, indicating that they may be targets of the same, unknown enzyme.  The initiating Met is cleaved, and the exposed α-amino group is mono-, di- or trimethylated.  Here, we report the discovery of the first α-N-methyltransferase, which we named N-terminal RCC1 methyltransferase (NRMT).  Substrate docking and mutational anal. of RCC1 defined the NRMT recognition sequence and enabled the identification of numerous new methylation targets, including SET (also known as TAF-I or PHAPII) and the retinoblastoma protein, RB.  Knockdown of NRMT recapitulates the multi-spindle phenotype seen with methylation-defective RCC1 mutants, demonstrating the importance of α-N-methylation for normal bipolar spindle formation and chromosome segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo90urCnl8qTrVg90H21EOLACvtfcHk0lgJM19S1rqMHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D&md5=185740cc031677ea3f6a20654546824e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnature09343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09343%26sid%3Dliteratum%253Aachs%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DMuratore-Schroeder%26aufirst%3DT.%2BL.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DNRMT%2520is%2520an%2520%25CE%25B1-N-methyltransferase%2520that%2520methylates%2520RCC1%2520and%2520retinoblastoma%2520protein%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1125%26epage%3D1128%26doi%3D10.1038%2Fnature09343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipson, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hadid, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitelegge, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">5225</span>– <span class="NLM_lpage">5235</span>, <span class="refDoi"> DOI: 10.1021/bi100428x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100428x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=5225-5235&author=K.+J.+Webbauthor=R.+S.+Lipsonauthor=Q.+Al-Hadidauthor=J.+P.+Whiteleggeauthor=S.+G.+Clarke&title=Identification+of+Protein+N-Terminal+Methyltransferases+in+Yeast+and+Humans&doi=10.1021%2Fbi100428x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span></div><div class="casAuthors">Webb, Kristofor J.; Lipson, Rebecca S.; Al-Hadid, Qais; Whitelegge, Julian P.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5225-5235</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein modification by methylation is important in cellular function.  We show here that the Saccharomyces cerevisiae YBR261C/TAE1 gene encodes an N-terminal protein methyltransferase catalyzing the modification of two ribosomal protein substrates, Rpl12ab and Rps25a/Rps25b.  The YBR261C/Tae1 protein is conserved across eukaryotes; all of these proteins share sequence similarity with known seven β-strand class I methyltransferases.  Wild-type yeast cytosol and mouse heart cytosol catalyze the methylation of a synthetic peptide (PPKQQLSKY) that contains the first eight amino acids of the processed N-terminus of Rps25a/Rps25b.  However, no methylation of this peptide is seen in yeast cytosol from a ΔYBR261C/tae1 deletion strain.  Yeast YBR261C/TAE1 and the human orthologue METTL11A genes were expressed as fusion proteins in Escherichia coli and were shown to be capable of stoichiometrically dimethylating the N-terminus of the synthetic peptide.  Furthermore, the YBR261C/Tae1 and METTL11A recombinant proteins methylate variants of the synthetic peptide contg. N-terminal alanine and serine residues.  However, methyltransferase activity is largely abolished when the proline residue in position 2 or the lysine residue in position 3 is substituted.  Thus, the methyltransferases described here specifically recognize the N-terminal X-Pro-Lys sequence motif, and we suggest designating the yeast enzyme Ntm1 and the human enzyme NTMT1.  These enzymes may account for nearly all previously described eukaryotic protein N-terminal methylation reactions.  A no. of other yeast and human proteins also share the recognition motif and may be similarly modified.  We conclude that protein X-Pro-Lys N-terminal methylation reactions catalyzed by the enzymes described here may be widespread in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiYjjV6t89rVg90H21EOLACvtfcHk0lgJM19S1rqMHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D&md5=deb28aa90896b14013bdf60e8b1410c3</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fbi100428x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100428x%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DK.%2BJ.%26aulast%3DLipson%26aufirst%3DR.%2BS.%26aulast%3DAl-Hadid%26aufirst%3DQ.%26aulast%3DWhitelegge%26aufirst%3DJ.%2BP.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DIdentification%2520of%2520Protein%2520N-Terminal%2520Methyltransferases%2520in%2520Yeast%2520and%2520Humans%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D5225%26epage%3D5235%26doi%3D10.1021%2Fbi100428x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkey, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT2 Is an N-Terminal Monomethylase That Primes for Its Homologue NRMT1</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>456</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1042/bj20131163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1042%2FBJ20131163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=24090352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVartrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=456&publication_year=2013&pages=453-462&author=J.+J.+Petkowskiauthor=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=D.+W.+Wilkeyauthor=M.+L.+Merchantauthor=I.+G.+Macaraauthor=C.+E.+Schaner+Tooley&title=NRMT2+Is+an+N-Terminal+Monomethylase+That+Primes+for+Its+Homologue+NRMT1&doi=10.1042%2Fbj20131163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT2 is an N-terminal monomethylase that primes for its homologue NRMT1</span></div><div class="casAuthors">Petkowski, Janusz J.; Bonsignore, Lindsay A.; Tooley, John G.; Wilkey, Daniel W.; Merchant, Michael L.; Macara, Ian G.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">456</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">453-462</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">NRMT (N-terminal regulator of chromatin condensation 1 methyltransferase) was the first eukaryotic methyltransferase identified to specifically methylate the free α-amino group of proteins.  Since the discovery of this N-terminal methyltransferase, many new substrates have been identified and the modification itself has been shown to regulate DNA-protein interactions.  Sequence anal. predicts one close human homolog of NRMT, METTL11B (methyltransferase-like protein 11B, now renamed NRMT2).  We show in the present paper for the first time that NRMT2 also has N-terminal methylation activity and recognizes the same N-terminal consensus sequences as NRMT (now NRMT1).  Both enzymes have similar tissue expression and cellular localization patterns.  However, enzyme assays and MS expts. indicate that they differ in their specific catalytic functions.  Although NRMT1 is a distributive methyltransferase that can mono-, di- and tri-methylate its substrates, NRMT2 is primarily a monomethylase.  Concurrent expression of NRMT1 and NRMT2 accelerates the prodn. of trimethylation, and we propose that NRMT2 activates NRMT1 by priming its substrates for trimethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopynrYAOFFVrVg90H21EOLACvtfcHk0lhrhUUYxn7S_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVartrzK&md5=20aebdfed2ae0676ee918c77f829a68d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1042%2FBJ20131163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20131163%26sid%3Dliteratum%253Aachs%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DWilkey%26aufirst%3DD.%2BW.%26aulast%3DMerchant%26aufirst%3DM.%2BL.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT2%2520Is%2520an%2520N-Terminal%2520Monomethylase%2520That%2520Primes%2520for%2520Its%2520Homologue%2520NRMT1%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D456%26spage%3D453%26epage%3D462%26doi%3D10.1042%2Fbj20131163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0lhrhUUYxn7S_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Histone N-Terminal Methylation by NRMT1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1101/gad.270926.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1101%2Fgad.270926.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=26543159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2337-2342&author=R.+Wuauthor=Y.+Yueauthor=X.+Zhengauthor=H.+Li&title=Molecular+Basis+for+Histone+N-Terminal+Methylation+by+NRMT1&doi=10.1101%2Fgad.270926.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for histone N-terminal methylation by NRMT1</span></div><div class="casAuthors">Wu, Ruoxi; Yue, Yuan; Zheng, Xiangdong; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2337-2342</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">NRMT1 is an N-terminal methyltransferase that methylates histone CENP-A as well as nonhistone substrates.  Here, we report the crystal structure of human NRMT1 bound to CENP-A peptide at 1.3 Å.  NRMT1 adopts a core methyltransferase fold that resembles DOT1L and PRMT but not SET domain family histone methyltransferases.  Key substrate recognition and catalytic residues were identified by mutagenesis studies.  Histone peptide profiling revealed that human NRMT1 is highly selective to human CENP-A and fruit fly H2B, which share a common "Xaa-Pro-Lys/Arg" motif.  These results, along with a 1.5 Å costructure of human NRMT1 bound to the fruit fly H2B peptide, underscore the importance of the NRMT1 recognition motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGIuplVjQhULVg90H21EOLACvtfcHk0lhrhUUYxn7S_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J&md5=f3601b0966da6c89d0acf9aeae4799be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.270926.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270926.115%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DR.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520Basis%2520for%2520Histone%2520N-Terminal%2520Methylation%2520by%2520NRMT1%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2337%26epage%3D2342%26doi%3D10.1101%2Fgad.270926.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">An Asparagine/Glycine Switch Governs Product Specificity of Human N-Terminal Methyltransferase NTMT2</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">183</span>, <span class="refDoi"> DOI: 10.1038/s42003-018-0196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fs42003-018-0196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30417120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=183&author=C.+Dongauthor=G.+Dongauthor=L.+Liauthor=L.+Zhuauthor=W.+Tempelauthor=Y.+Liuauthor=R.+Huangauthor=J.+Min&title=An+Asparagine%2FGlycine+Switch+Governs+Product+Specificity+of+Human+N-Terminal+Methyltransferase+NTMT2&doi=10.1038%2Fs42003-018-0196-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An asparagine/glycine switch governs product specificity of human N-terminal methyltransferase NTMT2</span></div><div class="casAuthors">Dong Cheng; Li Li; Zhu Licheng; Tempel Wolfram; Liu Yanli; Min Jinrong; Dong Guangping; Huang Rong; Zhu Licheng; Min Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">183</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">α-N-terminal methylation of proteins is an important post-translational modification that is catalyzed by two different N-terminal methyltransferases, namely NTMT1 and NTMT2.  Previous studies have suggested that NTMT1 is a tri-methyltransferase, whereas NTMT2 is a mono-methyltransferase.  Here, we report the first crystal structures, to our knowledge, of NTMT2 in binary complex with S-adenosyl-L-methionine as well as in ternary complex with S-adenosyl-L-homocysteine and a substrate peptide.  Our structural observations combined with biochemical studies reveal that NTMT2 is also able to di-/tri-methylate the GPKRIA peptide and di-methylate the PPKRIA peptide, otherwise it is predominantly a mono-methyltransferase.  The residue N89 of NTMT2 serves as a gatekeeper residue that regulates the binding of unmethylated versus monomethylated substrate peptide.  Structural comparison of NTMT1 and NTMT2 prompts us to design a N89G mutant of NTMT2 that can profoundly alter its catalytic activities and product specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSKFvwVHUgF0kwlCcfExbvbfW6udTcc2eZ6j3TJ1va7Wbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvovV2qug%253D%253D&md5=edc8c87334f423ab98872442f81d7928</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0196-2%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DAn%2520Asparagine%252FGlycine%2520Switch%2520Governs%2520Product%2520Specificity%2520of%2520Human%2520N-Terminal%2520Methyltransferase%2520NTMT2%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26spage%3D183%26doi%3D10.1038%2Fs42003-018-0196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakobsson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Małecki, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabelian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinto, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudithipudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davydova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuhairi, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeltsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leidel, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falnes, P. Ø.</span></span> <span> </span><span class="NLM_article-title">The Dual Methyltransferase METTL13 Targets N Terminus and Lys55 of EEF1A and Modulates Codon-Specific Translation Rates</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">3411</span>, <span class="refDoi"> DOI: 10.1038/s41467-018-05646-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fs41467-018-05646-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30143613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BB3c3hs1OjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=3411&author=M.+E.+Jakobssonauthor=J.+M.+Ma%C5%82eckiauthor=L.+Halabelianauthor=B.+S.+Nilgesauthor=R.+Pintoauthor=S.+Kudithipudiauthor=S.+Munkauthor=E.+Davydovaauthor=F.+R.+Zuhairiauthor=C.+H.+Arrowsmithauthor=A.+Jeltschauthor=S.+A.+Leidelauthor=J.+V.+Olsenauthor=P.+%C3%98.+Falnes&title=The+Dual+Methyltransferase+METTL13+Targets+N+Terminus+and+Lys55+of+EEF1A+and+Modulates+Codon-Specific+Translation+Rates&doi=10.1038%2Fs41467-018-05646-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">The dual methyltransferase METTL13 targets N terminus and Lys55 of eEF1A and modulates codon-specific translation rates</span></div><div class="casAuthors">Jakobsson Magnus E; Malecki Jedrzej M; Pinto Rita; Davydova Erna; Zuhairi Fawzi R; Falnes Pal O; Jakobsson Magnus E; Munk Stephanie; Olsen Jesper V; Halabelian Levon; Arrowsmith Cheryl H; Nilges Benedikt S; Leidel Sebastian A; Nilges Benedikt S; Leidel Sebastian A; Kudithipudi Srikanth; Jeltsch Albert</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3411</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Eukaryotic elongation factor 1 alpha (eEF1A) delivers aminoacyl-tRNA to the ribosome and thereby plays a key role in protein synthesis.  Human eEF1A is subject to extensive post-translational methylation, but several of the responsible enzymes remain unknown.  Using a wide range of experimental approaches, we here show that human methyltransferase (MTase)-like protein 13 (METTL13) contains two distinct MTase domains targeting the N terminus and Lys55 of eEF1A, respectively.  Our biochemical and structural analyses provide detailed mechanistic insights into recognition of the eEF1A N terminus by METTL13.  Moreover, through ribosome profiling, we demonstrate that loss of METTL13 function alters translation dynamics and results in changed translation rates of specific codons.  In summary, we here unravel the function of a human MTase, showing that it methylates eEF1A and modulates mRNA translation in a codon-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSnalPvS2TRnBLDWw4HIq2sfW6udTcc2eZaQ1dX4fsVC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3hs1OjsQ%253D%253D&md5=b336056ca88a555d488f1f2ae2b48efd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-05646-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-05646-y%26sid%3Dliteratum%253Aachs%26aulast%3DJakobsson%26aufirst%3DM.%2BE.%26aulast%3DMa%25C5%2582ecki%26aufirst%3DJ.%2BM.%26aulast%3DHalabelian%26aufirst%3DL.%26aulast%3DNilges%26aufirst%3DB.%2BS.%26aulast%3DPinto%26aufirst%3DR.%26aulast%3DKudithipudi%26aufirst%3DS.%26aulast%3DMunk%26aufirst%3DS.%26aulast%3DDavydova%26aufirst%3DE.%26aulast%3DZuhairi%26aufirst%3DF.%2BR.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJeltsch%26aufirst%3DA.%26aulast%3DLeidel%26aufirst%3DS.%2BA.%26aulast%3DOlsen%26aufirst%3DJ.%2BV.%26aulast%3DFalnes%26aufirst%3DP.%2B%25C3%2598.%26atitle%3DThe%2520Dual%2520Methyltransferase%2520METTL13%2520Targets%2520N%2520Terminus%2520and%2520Lys55%2520of%2520EEF1A%2520and%2520Modulates%2520Codon-Specific%2520Translation%2520Rates%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26spage%3D3411%26doi%3D10.1038%2Fs41467-018-05646-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lofgren, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulea, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tandoc, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caporicci, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maitra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topisirovic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Rechem, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gozani, O.</span></span> <span> </span><span class="NLM_article-title">METTL13 Methylation of EEF1A Increases Translational Output to Promote Tumorigenesis</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>176</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">504.e21</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.11.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.cell.2018.11.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=30612740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1Wjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=176&publication_year=2019&pages=491-504.e21&author=S.+Liuauthor=S.+Hausmannauthor=S.+M.+Carlsonauthor=M.+E.+Fuentesauthor=J.+W.+Francisauthor=R.+Pillaiauthor=S.+M.+Lofgrenauthor=L.+Huleaauthor=K.+Tandocauthor=J.+Luauthor=A.+Liauthor=N.+D.+Nguyenauthor=M.+Caporicciauthor=M.+P.+Kimauthor=A.+Maitraauthor=H.+Wangauthor=I.+I.+Wistubaauthor=J.+A.+Porcoauthor=M.+C.+Bassikauthor=J.+E.+Eliasauthor=J.+Songauthor=I.+Topisirovicauthor=C.+Van+Rechemauthor=P.+K.+Mazurauthor=O.+Gozani&title=METTL13+Methylation+of+EEF1A+Increases+Translational+Output+to+Promote+Tumorigenesis&doi=10.1016%2Fj.cell.2018.11.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">METTL13 Methylation of eEF1A Increases Translational Output to Promote Tumorigenesis</span></div><div class="casAuthors">Liu, Shuo; Hausmann, Simone; Carlson, Scott Moore; Fuentes, Mary Esmeralda; Francis, Joel William; Pillai, Renjitha; Lofgren, Shane Michael; Hulea, Laura; Tandoc, Kristofferson; Lu, Jiuwei; Li, Ami; Nguyen, Nicholas Dang; Caporicci, Marcello; Kim, Michael Paul; Maitra, Anirban; Wang, Huamin; Wistuba, Ignacio Ivan; Porco, John Anthony, Jr.; Bassik, Michael Cory; Elias, Joshua Eric; Song, Jikui; Topisirovic, Ivan; Van Rechem, Capucine; Mazur, Pawel Karol; Gozani, Or</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">176</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">491-504.e21</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Increased protein synthesis plays an etiol. role in diverse cancers.  Here, we demonstrate that METTL13 (methyltransferase-like 13) dimethylation of eEF1A (eukaryotic elongation factor 1A) lysine 55 (eEF1AK55me2) is utilized by Ras-driven cancers to increase translational output and promote tumorigenesis in vivo.  METTL13-catalyzed eEF1A methylation increases eEF1A's intrinsic GTPase activity in vitro and protein prodn. in cells.  METTL13 and eEF1AK55me2 levels are upregulated in cancer and neg. correlate with pancreatic and lung cancer patient survival.  METTL13 deletion and eEF1AK55me2 loss dramatically reduce Ras-driven neoplastic growth in mouse models and in patient-derived xenografts (PDXs) from primary pancreatic and lung tumors.  Finally, METTL13 depletion renders PDX tumors hypersensitive to drugs that target growth-signaling pathways.  Together, our work uncovers a mechanism by which lethal cancers become dependent on the METTL13-eEF1AK55me2 axis to meet their elevated protein synthesis requirement and suggests that METTL13 inhibition may constitute a targetable vulnerability of tumors driven by aberrant Ras signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrETutXdxNsQbVg90H21EOLACvtfcHk0lhYDHOCJoXRNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1Wjsg%253D%253D&md5=a5d12ace96f50aacff361b625414627c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.11.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.11.038%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DHausmann%26aufirst%3DS.%26aulast%3DCarlson%26aufirst%3DS.%2BM.%26aulast%3DFuentes%26aufirst%3DM.%2BE.%26aulast%3DFrancis%26aufirst%3DJ.%2BW.%26aulast%3DPillai%26aufirst%3DR.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DHulea%26aufirst%3DL.%26aulast%3DTandoc%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DN.%2BD.%26aulast%3DCaporicci%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DM.%2BP.%26aulast%3DMaitra%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DBassik%26aufirst%3DM.%2BC.%26aulast%3DElias%26aufirst%3DJ.%2BE.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DTopisirovic%26aufirst%3DI.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DMETTL13%2520Methylation%2520of%2520EEF1A%2520Increases%2520Translational%2520Output%2520to%2520Promote%2520Tumorigenesis%26jtitle%3DCell%26date%3D2019%26volume%3D176%26spage%3D491%26epage%3D504.e21%26doi%3D10.1016%2Fj.cell.2018.11.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyogoku, Y.</span></span> <span> </span><span class="NLM_article-title">N-Trimethylalanine, a Novel Blocked N-Terminal Residue of Tetrahymena Histone H2B1</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1675</span>– <span class="NLM_lpage">1678</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.jbchem.a134096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1093%2Foxfordjournals.jbchem.a134096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=6818230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADyaL3sXhvV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1982&pages=1675-1678&author=M.+Nomotoauthor=Y.+Kyogoku&title=N-Trimethylalanine%2C+a+Novel+Blocked+N-Terminal+Residue+of+Tetrahymena+Histone+H2B1&doi=10.1093%2Foxfordjournals.jbchem.a134096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">N-trimethylalanine, a novel blocked N-terminal residue of Tetrahymena histone H2B</span></div><div class="casAuthors">Nomoto, Minoru; Kyogoku, Yoshimasa; Iwai, Koichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry </span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1675-8</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    </div><div class="casAbstract">A previous study showed that the amino acid sequence of T. pyriformis histone H2B has a blocked N-terminal residue followed by 119 amino acid residues (Nomoto, M., et al., 1982).  The blocking group has been found to be N-trimethylalanine by 1H NMR spectroscopy of the isolated N-terminal tripeptide X-Pro-Lys and X.  This identification was supported by mass spectrometry of X-Pro-Lys and by column and paper chromatog. of X.  This is the 1st time that N-terminal blocking by methylation has ever been found in histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcvCBH0SBWNLVg90H21EOLACvtfcHk0lgjNaTpRe0j5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhvV2h&md5=506196f48eabffe7a38c6c86c5959b36</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.jbchem.a134096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.jbchem.a134096%26sid%3Dliteratum%253Aachs%26aulast%3DNomoto%26aufirst%3DM.%26aulast%3DKyogoku%26aufirst%3DY.%26atitle%3DN-Trimethylalanine%252C%2520a%2520Novel%2520Blocked%2520N-Terminal%2520Residue%2520of%2520Tetrahymena%2520Histone%2520H2B1%26jtitle%3DJ.%2520Biochem.%26date%3D1982%26volume%3D92%26spage%3D1675%26epage%3D1678%26doi%3D10.1093%2Foxfordjournals.jbchem.a134096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sathyan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fachinetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foltz, D. R.</span></span> <span> </span><span class="NLM_article-title">α-Amino Trimethylation of CENP-A by NRMT Is Required for Full Recruitment of the Centromere</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14678</span>, <span class="refDoi"> DOI: 10.1038/ncomms14678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fncomms14678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=28266506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14678&author=K.+M.+Sathyanauthor=D.+Fachinettiauthor=D.+R.+Foltz&title=%CE%B1-Amino+Trimethylation+of+CENP-A+by+NRMT+Is+Required+for+Full+Recruitment+of+the+Centromere&doi=10.1038%2Fncomms14678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">α-amino trimethylation of CENP-A by NRMT is required for full recruitment of the centromere</span></div><div class="casAuthors">Sathyan Kizhakke M; Foltz Daniel R; Fachinetti Daniele; Foltz Daniel R; Foltz Daniel R</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14678</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Centromeres are unique chromosomal domains that control chromosome segregation, and are epigenetically specified by the presence of the CENP-A containing nucleosomes.  CENP-A governs centromere function by recruiting the constitutive centromere associated network (CCAN) complex.  The features of the CENP-A nucleosome necessary to distinguish centromeric chromatin from general chromatin are not completely understood.  Here we show that CENP-A undergoes α-amino trimethylation by the enzyme NRMT in vivo.  We show that α-amino trimethylation of the CENP-A tail contributes to cell survival.  Loss of α-amino trimethylation causes a reduction in the CENP-T and CENP-I CCAN components at the centromere and leads to lagging chromosomes and spindle pole defects.  The function of p53 alters the response of cells to defects associated with decreased CENP-A methylation.  Altogether we show an important functional role for α-amino trimethylation of the CENP-A nucleosome in maintaining centromere function and faithful chromosomes segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_zBuwd5i4JcNIRw0GKUoEfW6udTcc2eaKjCHt_ds06bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D&md5=7fb75e88a08eb5d87b0c84aa6c62c4ab</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fncomms14678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14678%26sid%3Dliteratum%253Aachs%26aulast%3DSathyan%26aufirst%3DK.%2BM.%26aulast%3DFachinetti%26aufirst%3DD.%26aulast%3DFoltz%26aufirst%3DD.%2BR.%26atitle%3D%25CE%25B1-Amino%2520Trimethylation%2520of%2520CENP-A%2520by%2520NRMT%2520Is%2520Required%2520for%2520Full%2520Recruitment%2520of%2520the%2520Centromere%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14678%26doi%3D10.1038%2Fncomms14678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner-Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">N-terminal α-methylation of RCC1 is necessary for stable chromatin association and normal mitosis</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">596</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1038/ncb1572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1038%2Fncb1572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=17435751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=596-603&author=T.+Chenauthor=T.+L.+Muratoreauthor=C.+E.+Schaner-Tooleyauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=I.+G.+Macara&title=N-terminal+%CE%B1-methylation+of+RCC1+is+necessary+for+stable+chromatin+association+and+normal+mitosis&doi=10.1038%2Fncb1572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">N-terminal α-methylation of RCC1 is necessary for stable chromatin association and normal mitosis</span></div><div class="casAuthors">Chen, Ting; Muratore, Tara L.; Schaner-Tooley, Christine E.; Shabanowitz, Jeffrey; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">596-603</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Regulator of chromatin condensation 1 (RCC1) is the only known guanine nucleotide-exchange factor for the Ran GTPase and has pivotal roles in nucleo-cytoplasmic transport, mitosis, and nuclear-envelope assembly.  RCC1 assocs. dynamically with chromatin through binding to histones H2A and/or H2B in a Ran-regulated manner.  Here, we report that, unexpectedly, the amino-terminal serine or proline residue of RCC1 is uniquely methylated on its α-amino group.  Methylation requires removal of the initiating methionine, and the presence of proline and lysine at positions 3 and 4, resp.  Methylation-defective mutants of RCC1 bind less effectively than wild-type protein to chromatin during mitosis, which causes spindle-pole defects.  We propose a bimodal attachment mechanism for RCC1 in which the tail promotes stable RCC1 assocn. with chromatin through DNA binding in an α-N-methylation-dependent manner.  These data provide the first known function for N-terminal protein methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Cy1xI_cXwrVg90H21EOLACvtfcHk0lgjNaTpRe0j5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D&md5=91f3adb84da3958ec1bd66cb91e660a4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fncb1572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1572%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DMuratore%26aufirst%3DT.%2BL.%26aulast%3DSchaner-Tooley%26aufirst%3DC.%2BE.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DN-terminal%2520%25CE%25B1-methylation%2520of%2520RCC1%2520is%2520necessary%2520for%2520stable%2520chromatin%2520association%2520and%2520normal%2520mitosis%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D596%26epage%3D603%26doi%3D10.1038%2Fncb1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">α-N-Methylation of Damaged DNA-binding Protein 2 (DDB2) and Its Function in Nucleotide Excision Repair</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">16046</span>– <span class="NLM_lpage">16056</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.558510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1074%2Fjbc.M114.558510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=24753253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=16046-16056&author=Q.+Caiauthor=L.+Fuauthor=Z.+Wangauthor=N.+Ganauthor=X.+Daiauthor=Y.+Wang&title=%CE%B1-N-Methylation+of+Damaged+DNA-binding+Protein+2+%28DDB2%29+and+Its+Function+in+Nucleotide+Excision+Repair&doi=10.1074%2Fjbc.m114.558510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">α-N-Methylation of damaged DNA-binding protein 2 (DDB2) and its function in nucleotide excision repair</span></div><div class="casAuthors">Cai, Qian; Fu, Lijuan; Wang, Zi; Gan, Nanqin; Dai, Xiaoxia; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16046-16056</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">DDB2 exhibits a high affinity toward UV-damaged DNA, and it is involved in the initial steps of global genome nucleotide excision repair.  Mutations in the DDB2 gene cause the genetic complementation group E of xeroderma pigmentosum, an autosomal recessive disease manifested clin. by hypersensitivity to sunlight exposure and an increased predisposition to skin cancer.  Here the authors found that, in human cells, the initiating methionine residue in DDB2 was removed and that the N-terminal alanine could be methylated on its α-amino group in human cells, with trimethylation being the major form.  The α-N-methylation of DDB2 is catalyzed by the N-terminal RCC1 methyltransferase.  In addn., a methylation-defective mutant of DDB2 displayed diminished nuclear localization and was recruited at a reduced efficiency to UV-induced cyclobutane pyrimidine dimer foci.  Moreover, loss of this methylation conferred compromised ATM (ataxia telangiectasia mutated) activation, decreased efficiency in cyclobutane pyrimidine dimer repair, and elevated sensitivity of cells toward UV light exposure.  This study provides new knowledge about the posttranslational regulation of DDB2 and expands the biol. functions of protein α-N-methylation to DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc3yE8db8RO7Vg90H21EOLACvtfcHk0lhST5RIukHItw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D&md5=e52a0c303e3420df4722d42cb0478af1</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558510%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGan%26aufirst%3DN.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3D%25CE%25B1-N-Methylation%2520of%2520Damaged%2520DNA-binding%2520Protein%25202%2520%2528DDB2%2529%2520and%2520Its%2520Function%2520in%2520Nucleotide%2520Excision%2520Repair%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D16046%26epage%3D16056%26doi%3D10.1074%2Fjbc.m114.558510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulten, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2575-2582&author=X.+Daiauthor=S.+L.+Rultenauthor=C.+Youauthor=K.+W.+Caldecottauthor=Y.+Wang&title=Identification+and+Functional+Characterizations+of+N-Terminal+%CE%B1-N-Methylation+and+Phosphorylation+of+Serine+461+in+Human+Poly%28ADP-ribose%29+Polymerase+3&doi=10.1021%2Facs.jproteome.5b00126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span></div><div class="casAuthors">Dai, Xiaoxia; Rulten, Stuart L.; You, Changjun; Caldecott, Keith W.; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase 3 (PARP3) is a member of the PARP family enzymes which catalyze the ADP-ribosylation of proteins.  PARP3 plays an important role in DNA damage repair and mitotic progression.  In this study, we identified, using mass spectrometric (MS) techniques, two novel post-translational modification sites in PARP3, α-N-methylation and phosphorylation of serine 461 (S461).  We found that the N-terminal α-amino group of PARP3 is heavily methylated in human cells, and N-terminal RCC1 methyltransferase (NRMT) is a key enzyme required for this methylation.  We also obsd. that the phosphorylation level of S461 in PARP3 could be reduced in human cells upon treatment with flavopiridol, a cyclin-dependent kinase inhibitor.  Moreover, we demonstrated that S461 phosphorylation, but not α-N-methylation of PARP3, may be involved in the cellular response toward DNA double-strand breaks (DSBs).  These findings provide novel insights into the post-translational regulation of PARP3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0mFKyXPLnrVg90H21EOLACvtfcHk0lhST5RIukHItw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D&md5=6ff05a312e38b2105b71fbbd17388e64</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00126%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DRulten%26aufirst%3DS.%2BL.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520and%2520Functional%2520Characterizations%2520of%2520N-Terminal%2520%25CE%25B1-N-Methylation%2520and%2520Phosphorylation%2520of%2520Serine%2520461%2520in%2520Human%2520Poly%2528ADP-ribose%2529%2520Polymerase%25203%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Facs.jproteome.5b00126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shields, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkey, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbett, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merchant, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">Select Human Cancer Mutants of NRMT1 Alter Its Catalytic Activity and Decrease N-Terminal Trimethylation</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1639</span>– <span class="NLM_lpage">1652</span>, <span class="refDoi"> DOI: 10.1002/pro.3202</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1002%2Fpro.3202" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=28556566" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFSqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=1639-1652&author=K.+M.+Shieldsauthor=J.+G.+Tooleyauthor=J.+J.+Petkowskiauthor=D.+W.+Wilkeyauthor=N.+C.+Garbettauthor=M.+L.+Merchantauthor=A.+Chengauthor=C.+E.+Schaner+Tooley&title=Select+Human+Cancer+Mutants+of+NRMT1+Alter+Its+Catalytic+Activity+and+Decrease+N-Terminal+Trimethylation&doi=10.1002%2Fpro.3202"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Select human cancer mutants of NRMT1 alter its catalytic activity and decrease N-terminal trimethylation</span></div><div class="casAuthors">Shields, Kaitlyn M.; Tooley, John G.; Petkowski, Janusz J.; Wilkey, Daniel W.; Garbett, Nichola C.; Merchant, Michael L.; Cheng, Alan; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1639-1652</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A subset of B-cell lymphoma patients have dominant mutations in the histone H3 lysine 27 (H3K27) methyltransferase EZH2, which change it from a monomethylase to a trimethylase.  These mutations occur in arom. resides surrounding the active site and increase growth and alter transcription.  We study the N-terminal trimethylase NRMT1 and the N-terminal monomethylase NRMT2.  They are 50% identical, but differ in key arom. residues in their active site.  Given how these residues affect EZH2 activity, we tested whether they are responsible for the distinct catalytic activities of NRMT1/2.  Addnl., NRMT1 acts as a tumor suppressor in breast cancer cells.  Its loss promotes oncogenic phenotypes but sensitizes cells to DNA damage.  Mutations of NRMT1 naturally occur in human cancers, and we tested a select group for altered activity.  While directed mutation of the arom. residues had minimal catalytic effect, NRMT1 mutants N209I (endometrial cancer) and P211S (lung cancer) displayed decreased trimethylase and increased monomethylase/dimethylase activity.  Both mutations are located in the peptide-binding channel and indicate a second structural region impacting enzyme specificity.  The NRMT1 mutants demonstrated a slower rate of trimethylation and a requirement for higher substrate concn.  Expression of the mutants in wild type NRMT backgrounds showed no change in N-terminal methylation levels or growth rates, demonstrating they are not acting as dominant negatives.  Expression of the mutants in cells lacking endogenous NRMT1 resulted in minimal accumulation of N-terminal trimethylation, indicating homozygosity could help drive oncogenesis or serve as a marker for sensitivity to DNA damaging chemotherapeutics or γ-irradn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxowEFm1MLl7Vg90H21EOLACvtfcHk0lhST5RIukHItw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsFSqtr0%253D&md5=3d2c8424f37b210daf5d689ef70ef464</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fpro.3202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3202%26sid%3Dliteratum%253Aachs%26aulast%3DShields%26aufirst%3DK.%2BM.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DWilkey%26aufirst%3DD.%2BW.%26aulast%3DGarbett%26aufirst%3DN.%2BC.%26aulast%3DMerchant%26aufirst%3DM.%2BL.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DSelect%2520Human%2520Cancer%2520Mutants%2520of%2520NRMT1%2520Alter%2520Its%2520Catalytic%2520Activity%2520and%2520Decrease%2520N-Terminal%2520Trimethylation%26jtitle%3DProtein%2520Sci.%26date%3D2017%26volume%3D26%26spage%3D1639%26epage%3D1652%26doi%3D10.1002%2Fpro.3202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoose, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT1 Knockout Mice Exhibit Phenotypes Associated with Impaired DNA Repair and Premature Aging</span>. <i>Mech. Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>146–148</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2015.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.mad.2015.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25843235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146%E2%80%93148&publication_year=2015&pages=42-52&author=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=P.+M.+Van+Hooseauthor=E.+Wangauthor=A.+Chengauthor=M.+P.+Coleauthor=C.+E.+Schaner+Tooley&title=NRMT1+Knockout+Mice+Exhibit+Phenotypes+Associated+with+Impaired+DNA+Repair+and+Premature+Aging&doi=10.1016%2Fj.mad.2015.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT1 knockout mice exhibit phenotypes associated with impaired DNA repair and premature aging</span></div><div class="casAuthors">Bonsignore, Lindsay A.; Tooley, John G.; Van Hoose, Patrick M.; Wang, Eugenia; Cheng, Alan; Cole, Marsha P.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">146-148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-52</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Though defective genome maintenance and DNA repair have long been known to promote phenotypes of premature aging, the role protein methylation plays in these processes is only now emerging.  We have recently identified the first N-terminal methyltransferase, NRMT1, which regulates protein-DNA interactions and is necessary for both accurate mitotic division and nucleotide excision repair.  To demonstrate if complete loss of NRMT1 subsequently resulted in developmental or aging phenotypes, we constructed the first NRMT1 knockout (Nrmt1-/-) mouse.  The majority of these mice die shortly after birth.  However, the ones that survive, exhibit decreased body size, female-specific infertility, kyphosis, decreased mitochondrial function, and early-onset liver degeneration; phenotypes characteristic of other mouse models deficient in DNA repair.  The livers from Nrmt1-/- mice produce less reactive oxygen species (ROS) than wild type controls, and Nrmt1-/- mouse embryonic fibroblasts show a decreased capacity for handling oxidative damage.  This indicates that decreased mitochondrial function may benefit Nrmt1-/- mice and protect them from excess internal ROS and subsequent DNA damage.  These studies position the NRMT1 knockout mouse as a useful new system for studying the effects of genomic instability and defective DNA damage repair on organismal and tissue-specific aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWx010KT-fbrVg90H21EOLACvtfcHk0lhST5RIukHItw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D&md5=501bd0e3dade5777c3965efb8fe10b93</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2015.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2015.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DVan%2BHoose%26aufirst%3DP.%2BM.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DM.%2BP.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT1%2520Knockout%2520Mice%2520Exhibit%2520Phenotypes%2520Associated%2520with%2520Impaired%2520DNA%2520Repair%2520and%2520Premature%2520Aging%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2015%26volume%3D146%25E2%2580%2593148%26spage%3D42%26epage%3D52%26doi%3D10.1016%2Fj.mad.2015.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitakomate, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. R.</span></span> <span> </span><span class="NLM_article-title">The Methylated N-Terminal Tail of RCC1 Is Required for Stabilisation of Its Interaction with Chromatin by Ran in Live Cells</span>. <i>BMC Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">43</span>, <span class="refDoi"> DOI: 10.1186/1471-2121-11-43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1186%2F1471-2121-11-43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20565941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=43&author=E.+Hitakomateauthor=F.+E.+Hoodauthor=H.+S.+Sandersonauthor=P.+R.+Clarke&title=The+Methylated+N-Terminal+Tail+of+RCC1+Is+Required+for+Stabilisation+of+Its+Interaction+with+Chromatin+by+Ran+in+Live+Cells&doi=10.1186%2F1471-2121-11-43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The methylated N-terminal tail of RCC1 is required for stabilisation of its interaction with chromatin by Ran in live cells</span></div><div class="casAuthors">Hitakomate Ekarat; Hood Fiona E; Sanderson Helen S; Clarke Paul R</div><div class="citationInfo"><span class="NLM_cas:title">BMC cell biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Regulator of chromosome condensation 1 (RCC1) is the guanine nucleotide exchange factor for Ran GTPase.  Localised generation of Ran-GTP by RCC1 on chromatin is critical for nucleocytoplasmic transport, mitotic spindle assembly and nuclear envelope formation.  Both the N-terminal tail of RCC1 and its association with Ran are important for its interaction with chromatin in cells.  In vitro, the association of Ran with RCC1 induces a conformational change in the N-terminal tail that promotes its interaction with DNA.  RESULTS:  We have investigated the mechanism of the dynamic interaction of the alpha isoform of human RCC1 (RCC1alpha) with chromatin in live cells using fluorescence recovery after photobleaching (FRAP) of green fluorescent protein (GFP) fusions.  We show that the N-terminal tail stabilises the interaction of RCC1alpha with chromatin and this function can be partially replaced by another lysine-rich nuclear localisation signal.  Removal of the tail prevents the interaction of RCC1alpha with chromatin from being stabilised by RanT24N, a mutant that binds stably to RCC1alpha.  The interaction of RCC1alpha with chromatin is destabilised by mutation of lysine 4 (K4Q), which abolishes alpha-N-terminal methylation, and this interaction is no longer stabilised by RanT24N.  However, alpha-N-terminal methylation of RCC1alpha is not regulated by the binding of RanT24N.  Conversely, the association of Ran with precipitated RCC1alpha does not require the N-terminal tail of RCC1alpha or its methylation.  The mobility of RCC1alpha on chromatin is increased by mutation of aspartate 182 (D182A), which inhibits guanine-nucleotide exchange activity, but RCC1alphaD182A can still bind nucleotide-free Ran and its interaction with chromatin is stabilised by RanT24N.  CONCLUSIONS:  These results show that the stabilisation of the dynamic interaction of RCC1alpha with chromatin by Ran in live cells requires the N-terminal tail of RCC1alpha. alpha-N-methylation is not regulated by formation of the binary complex with Ran, but it promotes chromatin binding through the tail.  This work supports a model in which the association of RCC1alpha with chromatin is promoted by a conformational change in the alpha-N-terminal methylated tail that is induced allosterically in the binary complex with Ran.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQSyjm2mWbK-0g6wdEZmT2rfW6udTcc2eYrou-0Q_ZqJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D&md5=2bfb8e66cc72441bfd68eb8827caf717</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2F1471-2121-11-43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2121-11-43%26sid%3Dliteratum%253Aachs%26aulast%3DHitakomate%26aufirst%3DE.%26aulast%3DHood%26aufirst%3DF.%2BE.%26aulast%3DSanderson%26aufirst%3DH.%2BS.%26aulast%3DClarke%26aufirst%3DP.%2BR.%26atitle%3DThe%2520Methylated%2520N-Terminal%2520Tail%2520of%2520RCC1%2520Is%2520Required%2520for%2520Stabilisation%2520of%2520Its%2520Interaction%2520with%2520Chromatin%2520by%2520Ran%2520in%2520Live%2520Cells%26jtitle%3DBMC%2520Cell%2520Biol.%26date%3D2010%26volume%3D11%26spage%3D43%26doi%3D10.1186%2F1471-2121-11-43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1074%2Fjbc.M114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.m114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0limGQirPicztg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.m114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/c5ob00120j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1039%2FC5OB00120J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2Fc5ob00120j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0limGQirPicztg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC5OB00120J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00120J%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2Fc5ob00120j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Facile Synthesis of SAM-Peptide Conjugates through Alkyl Linkers Targeting Protein N-Terminal Methyltransferase 1</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6771</span>, <span class="refDoi"> DOI: 10.1039/c5ra20625a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1039%2FC5RA20625A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=27588169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=6768-6771&author=G.+Zhangauthor=R.+Huang&title=Facile+Synthesis+of+SAM-Peptide+Conjugates+through+Alkyl+Linkers+Targeting+Protein+N-Terminal+Methyltransferase+1&doi=10.1039%2Fc5ra20625a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of SAM-peptide conjugates through alkyl linkers targeting protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Zhang, Gang; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">6768-6771</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report the first chem. synthesis of SAM-peptide (SAM = S-adenosyl-L-methionine) conjugates through alkyl linkers to prep. bisubstrate analogs for protein methyltransferases.  The authors demonstrate its application by developing a series of bisubstrate inhibitors for protein N-terminal methyltransferase 1 (NTMT1) and the most potent one exhibits a Ki = 310 ± 55 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_MNtkSc3Ne7Vg90H21EOLACvtfcHk0lgVxhKJoRQvoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D&md5=e267ad97193784986c32b86c2e1ae88a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2FC5RA20625A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5RA20625A%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DFacile%2520Synthesis%2520of%2520SAM-Peptide%2520Conjugates%2520through%2520Alkyl%2520Linkers%2520Targeting%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D6768%26epage%3D6771%26doi%3D10.1039%2Fc5ra20625a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0lgVxhKJoRQvoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Probing the Plasticity in the Active Site of Protein N-Terminal Methyltransferase 1 Using Bisubstrate Analogs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00770</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00770" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=D.+Chenauthor=C.+Dongauthor=G.+Dongauthor=K.+Srinivasanauthor=J.+Minauthor=N.+Noinajauthor=R.+Huang&title=Probing+the+Plasticity+in+the+Active+Site+of+Protein+N-Terminal+Methyltransferase+1+Using+Bisubstrate+Analogs&doi=10.1021%2Facs.jmedchem.0c00770"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00770%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DSrinivasan%26aufirst%3DK.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DProbing%2520the%2520Plasticity%2520in%2520the%2520Active%2520Site%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Using%2520Bisubstrate%2520Analogs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c00770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shumilin, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">Substrate Specificity of Mammalian N-Terminal α-Amino Methyltransferase NRMT</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5942</span>– <span class="NLM_lpage">5950</span>, <span class="refDoi"> DOI: 10.1021/bi300278f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi300278f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=5942-5950&author=J.+J.+Petkowskiauthor=C.+E.+Schaner+Tooleyauthor=L.+C.+Andersonauthor=I.+A.+Shumilinauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=W.+Minorauthor=I.+G.+Macara&title=Substrate+Specificity+of+Mammalian+N-Terminal+%CE%B1-Amino+Methyltransferase+NRMT&doi=10.1021%2Fbi300278f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Substrate Specificity of Mammalian N-Terminal α-Amino Methyltransferase NRMT</span></div><div class="casAuthors">Petkowski, Janusz J.; Schaner Tooley, Christine E.; Anderson, Lissa C.; Shumilin, Igor A.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Hunt, Donald F.; Minor, Wladek; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">5942-5950</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-Terminal methylation of free α-amino groups is a post-translational modification of proteins that was first described 30 years ago but has been studied very little.  In this modification, the initiating M residue is cleaved and the exposed α-amino group is mono-, di-, or trimethylated by NRMT, a recently identified N-terminal methyltransferase.  Currently, all known eukaryotic α-amino-methylated proteins have a unique N-terminal motif, M-X-P-K, where X is A, P, or S.  NRMT can also methylate artificial substrates in vitro in which X is G, F, Y, C, M, K, R, N, Q, or H.  Methylation efficiencies of N-terminal amino acids are variable with respect to the identity of X.  Here we use in vitro peptide methylation assays and substrate immunopptns. to show that the canonical M-X-P-K methylation motif is not the only one recognized by NRMT.  We predict that N-terminal methylation is a widespread post-translational modification and that there is interplay between N-terminal acetylation and N-terminal methylation.  We also use isothermal calorimetry expts. to demonstrate that NRMT can efficiently recognize and bind to its fully methylated products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR8dl8ByIopbVg90H21EOLACvtfcHk0lgVxhKJoRQvoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVars78%253D&md5=dbddebac4bc6f6045fd144604e157bdb</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fbi300278f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi300278f%26sid%3Dliteratum%253Aachs%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DAnderson%26aufirst%3DL.%2BC.%26aulast%3DShumilin%26aufirst%3DI.%2BA.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DSubstrate%2520Specificity%2520of%2520Mammalian%2520N-Terminal%2520%25CE%25B1-Amino%2520Methyltransferase%2520NRMT%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D5942%26epage%3D5950%26doi%3D10.1021%2Fbi300278f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackie, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">A Direct, Ratiometric, and Quantitative MALDI-MS Assay for Protein Methyltransferases and Acetyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1016%2Fj.ab.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=25778392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2015&pages=59-64&author=S.+L.+Richardsonauthor=P.+Hanjraauthor=G.+Zhangauthor=B.+D.+Mackieauthor=D.+L.+Petersonauthor=R.+Huang&title=A+Direct%2C+Ratiometric%2C+and+Quantitative+MALDI-MS+Assay+for+Protein+Methyltransferases+and+Acetyltransferases&doi=10.1016%2Fj.ab.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A direct, ratiometric, and quantitative MALDI-MS assay for protein methyltransferases and acetyltransferases</span></div><div class="casAuthors">Richardson, Stacie L.; Hanjra, Pahul; Zhang, Gang; Mackie, Brianna D.; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-64</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein methylation and acetylation play important roles in biol. processes, and misregulation of these modifications is involved in various diseases.  Therefore, it is crit. to understand the activities of the enzymes responsible for these modifications.  Herein we describe a sensitive method for ratiometric quantification of methylated and acetylated peptides via MALDI-MS by direct spotting of enzymic methylation and acetylation reaction mixts. without tedious purifn. procedures.  The quantifiable detection limit for peptides with our method is approx. 10 fmol.  This is achieved by increasing the signal-to-noise ratio through the addn. of NH4H2PO4 to the matrix soln. and redn. of the matrix α-cyanohydroxycinnamic acid concn. to 2 mg/mL.  We have demonstrated the application of this method in enzyme kinetic anal. and inhibition studies.  The unique feature of this method is the simultaneous quantification of multiple peptide species for investigation of processivity mechanisms.  Its wide buffer compatibility makes it possible to be adapted to investigate the activity of any protein methyltransferase or acetyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2PCzex5Lwq7Vg90H21EOLACvtfcHk0livAATHJ3p9cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D&md5=15c0b43193742c5abddf9a4f0d9ba488</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DMackie%26aufirst%3DB.%2BD.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DA%2520Direct%252C%2520Ratiometric%252C%2520and%2520Quantitative%2520MALDI-MS%2520Assay%2520for%2520Protein%2520Methyltransferases%2520and%2520Acetyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2015%26volume%3D478%26spage%3D59%26epage%3D64%26doi%3D10.1016%2Fj.ab.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Poli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moretto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crisma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peggion, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Formaggio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaptein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broxterman, Q. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toniolo, C.</span></span> <span> </span><span class="NLM_article-title">Is the Backbone Conformation of Cα-Methyl Proline Restricted to a Single Region?</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">8015</span>– <span class="NLM_lpage">8025</span>, <span class="refDoi"> DOI: 10.1002/chem.200900688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1002%2Fchem.200900688" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=19579242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1yrt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=8015-8025&author=M.+De%0APoliauthor=A.+Morettoauthor=M.+Crismaauthor=C.+Peggionauthor=F.+Formaggioauthor=B.+Kapteinauthor=Q.+B.+Broxtermanauthor=C.+Toniolo&title=Is+the+Backbone+Conformation+of+C%CE%B1-Methyl+Proline+Restricted+to+a+Single+Region%3F&doi=10.1002%2Fchem.200900688"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Is the backbone conformation of Cα-methyl proline restricted to a single region?</span></div><div class="casAuthors">De Poli, Matteo; Moretto, Alessandro; Crisma, Marco; Peggion, Cristina; Formaggio, Fernando; Kaptein, Bernard; Broxterman, Quirinus B.; Toniolo, Claudio</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">8015-8025, S8015/1-S8015/6</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cα-Methyl-L-proline, or L-(αMe)Pro, is probably the most conformationally constrained α-amino acid.  In particular, its ω and φ torsion angles are restricted to about 180 and -60°, resp., and only three ranges of values are theor. available for ψ in mono- or longer peptides, namely, about -30° (cis', 310/α-helical structure), 60° (inverse γ turn), or 140° (trans', poly(L-Pro)n II structure).  In this work, we examd. the tendency of a no. of Nα-acyl dipeptide N'-alkylamides of the type RCO-(αMe)Pro-Xxx-NHR' or RCO-Xxx-(αMe)Pro-NHR', in which Xxx is L (or D)-Ala, Aib (α-aminoisoburyric acid), or L (or D)-(αMe)Pro, long enough to fold into intramolecularly hydrogen-bonded γ or β turns.  The results are compared with those obtained for the corresponding dipeptides based on Pro, a well-known turn-forming residue.  For the crystal-state 3D-structural anal. we used X-ray diffraction, whereas our soln. conformational anal. was heavily based on the FTIR absorption and 1H and 13C NMR spectroscopy techniques.  We conclude that (αMe)Pro is able to explore both trans' and cis' ψ areas of the conformational space, but in (αMe)Pro the latter is overwhelmingly more populated, in marked contrast to the Pro preference.  This finding is a clear indication that in (αMe)Pro the major 3D-structural determinant is the Cα-Me group.  The CD signature of a peptide type III' β-turn conformation is also proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiFIxKnGqbxrVg90H21EOLACvtfcHk0livAATHJ3p9cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1yrt7k%253D&md5=bd76d96642ed6ee9f9d6eba5f47ace4d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fchem.200900688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200900688%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BPoli%26aufirst%3DM.%26aulast%3DMoretto%26aufirst%3DA.%26aulast%3DCrisma%26aufirst%3DM.%26aulast%3DPeggion%26aufirst%3DC.%26aulast%3DFormaggio%26aufirst%3DF.%26aulast%3DKaptein%26aufirst%3DB.%26aulast%3DBroxterman%26aufirst%3DQ.%2BB.%26aulast%3DToniolo%26aufirst%3DC.%26atitle%3DIs%2520the%2520Backbone%2520Conformation%2520of%2520C%25CE%25B1-Methyl%2520Proline%2520Restricted%2520to%2520a%2520Single%2520Region%253F%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2009%26volume%3D15%26spage%3D8015%26epage%3D8025%26doi%3D10.1002%2Fchem.200900688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyamu, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10783</span>– <span class="NLM_lpage">10797</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01255</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10783-10797&author=D.+Chenauthor=L.+Liauthor=K.+Diazauthor=I.+D.+Iyamuauthor=R.+Yadavauthor=N.+Noinajauthor=R.+Huang&title=Novel+Propargyl-Linked+Bisubstrate+Analogues+as+Tight-Binding+Inhibitors+for+Nicotinamide+N-Methyltransferase&doi=10.1021%2Facs.jmedchem.9b01255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Novel propargyl-linked bisubstrate analogues as tight-binding inhibitors for nicotinamide N-methyltransferase</span></div><div class="casAuthors">Chen, Dongxing; Li, Linjie; Diaz, Krystal; Iyamu, Iredia D.; Yadav, Ravi; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10783-10797</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the Me transfer from the cofactor S-adenosylmethionine to nicotinamide and other pyridine-contg. compds.  NNMT is an important regulator for nicotinamide metab. and methylation potential.  Aberrant expression levels of NNMT have been implicated in cancer, metabolic, and neurodegenerative diseases, which makes NNMT a potential therapeutic target.  Therefore, potent and selective NNMT inhibitors can serve as valuable tools to investigate the roles of NNMT in its mediated diseases.  Here, we applied a rational strategy to design and synthesize the tight-binding bisubstrate inhibitor LL320 through a novel propargyl linker.  LL320 demonstrates a Ki value of 1.6 ± 0.3 nM, which is the most potent inhibitor to date.  The cocrystal structure of LL320 confirms its interaction with both the substrate and cofactor binding sites on NNMT.  Importantly, this is the first example of using the propargyl linker to construct potent methyltransferase inhibitors, which will expand our understanding of the transition state of Me transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqs2XyawAg9ebVg90H21EOLACvtfcHk0livAATHJ3p9cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFKksLvN&md5=a6e3a5dc2044dc50ede1596c84804f61</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DDiaz%26aufirst%3DK.%26aulast%3DIyamu%26aufirst%3DI.%2BD.%26aulast%3DYadav%26aufirst%3DR.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DNovel%2520Propargyl-Linked%2520Bisubstrate%2520Analogues%2520as%2520Tight-Binding%2520Inhibitors%2520for%2520Nicotinamide%2520N-Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10783%26epage%3D10797%26doi%3D10.1021%2Facs.jmedchem.9b01255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briesewitz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrios, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhiang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Efficient Delivery of Cyclic Peptides into Mammalian Cells with Short Sequence Motifs</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1021/cb3005275</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb3005275" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1akur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=423-431&author=Z.+Qianauthor=T.+Liuauthor=Y.-Y.+Liuauthor=R.+Briesewitzauthor=A.+M.+Barriosauthor=S.+M.+Jhiangauthor=D.+Pei&title=Efficient+Delivery+of+Cyclic+Peptides+into+Mammalian+Cells+with+Short+Sequence+Motifs&doi=10.1021%2Fcb3005275"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Delivery of Cyclic Peptides into Mammalian Cells with Short Sequence Motifs</span></div><div class="casAuthors">Qian, Ziqing; Liu, Tao; Liu, Yu-Yu; Briesewitz, Roger; Barrios, Amy M.; Jhiang, Sissy M.; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">423-431</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptides hold great potential as therapeutic agents and research tools, but their broad application has been limited by poor membrane permeability.  Here, we report a potentially general approach for intracellular delivery of cyclic peptides.  Short peptide motifs rich in arginine and hydrophobic residues (e.g., FΦRRRR, where Φ is l-2-naphthylalanine), when embedded into small- to medium-sized cyclic peptides (7-13 amino acids), bound to the plasma membrane of mammalian cultured cells and were subsequently internalized by the cells.  Confocal microscopy and a newly developed peptide internalization assay demonstrated that cyclic peptides contg. these transporter motifs were translocated into the cytoplasm and nucleus at efficiencies 2-5-fold higher than that of nonaarginine (R9).  Furthermore, incorporation of the FΦRRRR motif into a cyclic peptide contg. a phosphocoumaryl aminopropionic acid (pCAP) residue generated a cell permeable, fluorogenic probe for detecting intracellular protein tyrosine phosphatase activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouUMFM_wJk9bVg90H21EOLACvtfcHk0lg-eGFhYhYMMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1akur%252FO&md5=80f723598b2afcf4503bbb026a7290e9</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fcb3005275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb3005275%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.-Y.%26aulast%3DBriesewitz%26aufirst%3DR.%26aulast%3DBarrios%26aufirst%3DA.%2BM.%26aulast%3DJhiang%26aufirst%3DS.%2BM.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DEfficient%2520Delivery%2520of%2520Cyclic%2520Peptides%2520into%2520Mammalian%2520Cells%2520with%2520Short%2520Sequence%2520Motifs%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D423%26epage%3D431%26doi%3D10.1021%2Fcb3005275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span>; <span class="NLM_string-name">Ashton, A.</span></span> <i>Fast_dp. Methods</i>; <span class="NLM_publisher-name">Zenodo</span>, <span class="NLM_year">2014</span>; Vol.  <span class="NLM_volume">55</span>, pp  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">93</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=G.+Winter&author=A.+Ashton&title=Fast_dp.+Methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26jtitle%3DFast_dp.%2520Methods%26pub%3DZenodo%26date%3D2014%26volume%3D55%26spage%3D81%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lg-eGFhYhYMMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lg-eGFhYhYMMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8','PDB','6WH8'); return false;">PDB: 6WH8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1D','PDB','5E1D'); return false;">PDB: 5E1D</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i29"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00689">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_43328"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00689?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00689</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">MALDI-MS and HPLC spectra of compounds <b>1</b>–<b>24</b>, <b>DC431</b>, and <b>DC432</b>; IC<sub>50</sub> curve of compound <b>5</b>; IC<sub>50</sub> curve of <b>BM30</b>; cell permeability evaluation; selectivity study of <b>BM30</b> and <b>22</b> against G9a and PRMT1; selectivity study of <b>BM30</b> against a panel of 41 MTs (Reaction Biology); crystallography data and refinement statistics (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>); and IC<sub>50</sub> values and binding affinities of <b>BM30</b> and its cell-permeable analogues (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_001.pdf">jm0c00689_si_001.pdf (4.08 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00689/suppl_file/jm0c00689_si_002.csv">jm0c00689_si_002.csv (2.74 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NTMT1 in complex with <b>BM30</b> have been deposited under PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6WH8">6WH8</a>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00689&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00689%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00689" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a330cde9f3c46","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
