
<html lang="en"     class="pb-page"  data-request-id="721bfbc3-85f6-4724-b2d1-091f1fc727de"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2021.64.issue-11;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.1c00649;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Rational Design and Synthesis of Novel Dual PROTACs for Simultaneous Degradation of EGFR and PARP" /></meta><meta name="dc.Creator" content="Mengzhu  Zheng" /></meta><meta name="dc.Creator" content="Junfeng  Huo" /></meta><meta name="dc.Creator" content="Xiaoxia  Gu" /></meta><meta name="dc.Creator" content="Yali  Wang" /></meta><meta name="dc.Creator" content="Canrong  Wu" /></meta><meta name="dc.Creator" content="Qingzhe  Zhang" /></meta><meta name="dc.Creator" content="Wang  Wang" /></meta><meta name="dc.Creator" content="Yang  Liu" /></meta><meta name="dc.Creator" content="Yu  Liu" /></meta><meta name="dc.Creator" content="Xuechen  Zhou" /></meta><meta name="dc.Creator" content="Lixia  Chen" /></meta><meta name="dc.Creator" content="Yirong  Zhou" /></meta><meta name="dc.Creator" content="Hua  Li" /></meta><meta name="dc.Description" content="Inspired by the success of dual-targeting drugs, especially bispecific antibodies, we propose to combine the concept of proteolysis targeting chimera (PROTAC) and dual targeting to design and synth..." /></meta><meta name="Description" content="Inspired by the success of dual-targeting drugs, especially bispecific antibodies, we propose to combine the concept of proteolysis targeting chimera (PROTAC) and dual targeting to design and synth..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 26, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00649" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00649" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00649" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00649" /></link>
        
    
    

<title>Rational Design and Synthesis of Novel Dual PROTACs for Simultaneous Degradation of EGFR and PARP | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00649" /></meta><meta property="og:title" content="Rational Design and Synthesis of Novel Dual PROTACs for Simultaneous Degradation of EGFR and PARP" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0008.jpeg" /></meta><meta property="og:description" content="Inspired by the success of dual-targeting drugs, especially bispecific antibodies, we propose to combine the concept of proteolysis targeting chimera (PROTAC) and dual targeting to design and synthesize dual PROTAC molecules with the function of degrading two completely different types of targets simultaneously. A library of novel dual-targeting PROTAC molecules has been rationally designed and prepared. A convergent synthetic strategy has been utilized to achieve high synthetic efficiency. These dual PROTAC structures are characterized using trifunctional natural amino acids as star-type core linkers to connect two independent inhibitors and E3 ligands together. In this study, gefitinib, olaparib, and CRBN or VHL E3 ligands were used as substrates to synthesize novel dual PROTACs. They successfully degraded both the epidermal growth factor receptor (EGFR) and poly(ADP-ribose) polymerase (PARP) simultaneously in cancer cells. Being the first successful example of dual PROTACs, this technique will greatly widen the range of application of the PROTAC method and open up a new field for drug discovery." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00649"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00649">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00649&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00649&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00649&amp;href=/doi/10.1021/acs.jmedchem.1c00649" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 7839-7852</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00644" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00651" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rational Design and Synthesis of Novel Dual PROTACs for Simultaneous Degradation of EGFR and PARP</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Mengzhu Zheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mengzhu Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mengzhu++Zheng">Mengzhu Zheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Junfeng Huo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Junfeng Huo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Junfeng++Huo">Junfeng Huo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoxia Gu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoxia Gu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoxia++Gu">Xiaoxia Gu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yali Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yali Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wuya College of Innovation, Key Laboratory of Structure-Based Drug Design & Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yali++Wang">Yali Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Canrong Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Canrong Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Canrong++Wu">Canrong Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qingzhe Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qingzhe Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qingzhe++Zhang">Qingzhe Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wuya College of Innovation, Key Laboratory of Structure-Based Drug Design & Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wang++Wang">Wang Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yang Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yang Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wuya College of Innovation, Key Laboratory of Structure-Based Drug Design & Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yang++Liu">Yang Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Liu">Yu Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xuechen Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xuechen Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xuechen++Zhou">Xuechen Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lixia Chen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lixia Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wuya College of Innovation, Key Laboratory of Structure-Based Drug Design & Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#14676d6e6d77786c542522273a777b79"><span class="__cf_email__" data-cfemail="c3b0bab9baa0afbb83f2f5f0eda0acae">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lixia++Chen">Lixia Chen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2196-1428" title="Orcid link">https://orcid.org/0000-0003-2196-1428</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yirong Zhou</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yirong Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#473d2f28323e2e35282920072f32343369222332692429"><span class="__cf_email__" data-cfemail="700a181f050919021f1e1730180503045e1514055e131e">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yirong++Zhou">Yirong Zhou</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3470-4562" title="Orcid link">https://orcid.org/0000-0002-3470-4562</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Hua Li</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hua Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hubei Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation, School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji Medical College, Huazhong University of Science and Technology, Wuhan 430030, China</div><div class="loa-info-affiliations-info">Wuya College of Innovation, Key Laboratory of Structure-Based Drug Design & Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#bfd3d6e0d7cadeffd7cacccb91dadbca91dcd1"><span class="__cf_email__" data-cfemail="a3cfcafccbd6c2e3cbd6d0d78dc6c7d68dc0cd">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hua++Li">Hua Li</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1903-836X" title="Orcid link">https://orcid.org/0000-0003-1903-836X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00649&amp;href=/doi/10.1021%2Facs.jmedchem.1c00649" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 7839–7852</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 26, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 April 2021</li><li><span class="item_label"><b>Published</b> online</span>26 May 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00649" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00649</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7839%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMengzhu%2BZheng%252C%2BJunfeng%2BHuo%252C%2BXiaoxia%2BGu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D11%26contentID%3Dacs.jmedchem.1c00649%26title%3DRational%2BDesign%2Band%2BSynthesis%2Bof%2BNovel%2BDual%2BPROTACs%2Bfor%2BSimultaneous%2BDegradation%2Bof%2BEGFR%2Band%2BPARP%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7852%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00649"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2868</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00649" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rational Design and Synthesis of Novel Dual PROTACs for Simultaneous Degradation of EGFR and PARP&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Mengzhu&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Junfeng&quot;,&quot;last_name&quot;:&quot;Huo&quot;},{&quot;first_name&quot;:&quot;Xiaoxia&quot;,&quot;last_name&quot;:&quot;Gu&quot;},{&quot;first_name&quot;:&quot;Yali&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Canrong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Qingzhe&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Wang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yang&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xuechen&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Lixia&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yirong&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Hua&quot;,&quot;last_name&quot;:&quot;Li&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;7839-7852&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00649&quot;},&quot;abstract&quot;:&quot;Inspired by the success of dual-targeting drugs, especially bispecific antibodies, we propose to combine the concept of proteolysis targeting chimera (PROTAC) and dual targeting to design and synthesize dual PROTAC molecules with the function of degrading two completely different types of targets simultaneously. A library of novel dual-targeting PROTAC molecules has been rationally designed and prepared. A convergent synthetic strategy has been utilized to achieve high synthetic efficiency. These dual PROTAC structures are characterized using trifunctional natural amino acids as star-type core linkers to connect two independent inhibitors and E3 ligands together. In this study, gefitinib, olaparib, and CRBN or VHL E3 ligands were used as substrates to synthesize novel dual PROTACs. They successfully degraded both the epidermal growth factor receptor (EGFR) and poly(ADP-ribose) polymerase (PARP) simultaneously in cancer cells. Being the first successful example of dual PROTACs, this technique will greatly wide&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00649&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00649" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00649&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00649" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00649&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00649" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00649&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00649&amp;href=/doi/10.1021/acs.jmedchem.1c00649" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00649" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00649" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00649%26sid%3Dliteratum%253Aachs%26pmid%3D34038131%26genre%3Darticle%26aulast%3DZheng%26date%3D2021%26atitle%3DRational%2BDesign%2Band%2BSynthesis%2Bof%2BNovel%2BDual%2BPROTACs%2Bfor%2BSimultaneous%2BDegradation%2Bof%2BEGFR%2Band%2BPARP%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D11%26spage%3D7839%26epage%3D7852%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=291650" title="Alcohols">Alcohols</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292017" title="Solvents">Solvents</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/jmcmar.2021.64.issue-11/20210610/jmcmar.2021.64.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Inspired by the success of dual-targeting drugs, especially bispecific antibodies, we propose to combine the concept of proteolysis targeting chimera (PROTAC) and dual targeting to design and synthesize dual PROTAC molecules with the function of degrading two completely different types of targets simultaneously. A library of novel dual-targeting PROTAC molecules has been rationally designed and prepared. A convergent synthetic strategy has been utilized to achieve high synthetic efficiency. These dual PROTAC structures are characterized using trifunctional natural amino acids as star-type core linkers to connect two independent inhibitors and E3 ligands together. In this study, gefitinib, olaparib, and CRBN or VHL E3 ligands were used as substrates to synthesize novel dual PROTACs. They successfully degraded both the epidermal growth factor receptor (EGFR) and poly(ADP-ribose) polymerase (PARP) simultaneously in cancer cells. Being the first successful example of dual PROTACs, this technique will greatly widen the range of application of the PROTAC method and open up a new field for drug discovery.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33824" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33824" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In cancer, tumor cells often upregulate different growth-promoting factors, which can act independently or interfere with each other through a signal network.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It is easy for tumor cells to acquire drug resistance by resorting to the compensatory factors or switching the signal pathway that promotes proliferation.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Therefore, the treatment with drugs targeting only one single target inevitably exhibits limitations. In addition to drug resistance, the side effects and the tissue toxicity of single-target drugs often lead to reduced efficacy and decreased life quality of patients.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> To overcome these shortcomings of single-target drugs, drug combination targeting two or more different signal pathways of tumor has become a well-recognized effective method.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Due to the synergistic effects of the drug combination, a smaller dose of each single drug is needed, thus reducing the side effects.</div><div class="NLM_p">Another approach to improve therapeutic efficacy is to design a single hybrid molecule merged by two or more pharmacophores to target two or more antitumor epitopes or targets simultaneously.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Regulating multiple targets or pathways at the same time, these hybrid molecules usually exhibit better efficacy while causing fewer side effects. In the past few decades, these hybrid molecules, including bispecific antibodies and other small molecule drugs with double or multiple targets,<a onclick="showRef(event, 'ref3 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref5">(3,5)</a> have attracted great interest and achieved considerable success due to their superiority in the treatment of complex diseases. They have gradually become an alternative to combination therapy or the use of mixtures.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a></div><div class="NLM_p last">Proteolysis targeting chimera (PROTAC) is a kind of bifunctional small molecule, in which the target protein ligand and the E3 ubiquitin ligase ligand are linked together through a linker to form a triplet compound.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Compared with traditional small molecule inhibitors, PROTAC has several advantages. It no more needs to bind to the active site of the target protein to exert effects, while it is capable of the degradation of “nondruggable” targets. There exists an event-driven mechanism rendering its catalytic properties, and it works at lower doses while providing great opportunities for the development of anticancer drugs.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> However, the majority of reported PROTAC molecules only connect one inhibitor with one E3 ligand, which often degrades only a single-target protein and not exceeding the limit of two or more similar proteins.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44163" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44163" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Inspired by the great success of dual-targeting drugs, especially bispecific antibodies, we envisage that by combining the concept of PROTAC and dual targeting, a dual PROTAC molecule with two independent inhibitor units and one E3 ligase ligand can be designed for degrading two targets simultaneously in completely different pathways (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Dual PROTAC can not only degrade two targets at the same time, but also imbibe merits of both PROTACs and double targeting drugs, achieving similar efficacy as bispecific antibodies and still relatively conserving the advantages of small molecule drugs. To the best of our knowledge, there has been no report yet on this audacious and innovative idea. Therefore, it will be very interesting to design and synthesize such a novel dual PROTAC molecule for proof of concept.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of novel star-shaped dual PROTACs and the schematic diagram of dual-targeted protein degradation. Dual PROTACs consist of two ligands binding to two different target proteins and a ligand binding to the E3 ubiquitin ligase. Three ligands are linked through a linker to form a dual-targeting PROTAC, then recruit the E3 ligase to induce the ubiquitination of both two targets, and subsequent degradation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To verify our hypothesis shown above, the rational design of the novel star-type linker should function as the most pivotal element for dual PROTACs. Taking biocompatibility into consideration, natural amino acids will be a good choice. More importantly, some amino acids, such as tyrosine and serine, bearing a third reactive site besides normal amine and acid functional groups, are ideal star-type linkers to connect three different small molecules (two inhibitors and one E3 ligase ligand) together (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In addition, the three potential reactive sites display different reactivities, which makes them very suitable for sequential organic synthesis manipulation according to our rational design.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route Design and the Structures of Dual PROTACs (<b>DP-C 1-4</b>, <b>DP-V 1-4</b>) and Mono PROTACs (<b>MP-GC</b>, <b>MP-GV</b>, <b>MP-OC</b>, <b>MP-OV</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Tyrosine and serine were used as star-type linkers to connect two inhibitors and one E3 ligase ligand together. Eight dual PROTACs and four mono PROTACs were constructed consequently.</p></p></figure><div class="NLM_p">Drug resistance in advanced cancers has been reported to be mediated by different factors, such as the epidermal growth factor receptor (EGFR) over-expression and DNA repair enzymes. Poly(ADP-ribose) polymerase (PARP) is one such protein that is known to be a key player in base excision repair (BER)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and cellular signaling pathways.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> The inhibition of EGFR leads to the down-regulation of key players in BER and sensitizes cell response to alkylating agents and ionizing radiation.<a onclick="showRef(event, 'ref6 ref13'); return false;" href="javascript:void(0);" class="ref ref6 ref13">(6,13)</a> Receptor tyrosine kinase inhibitors (TKIs), just like EGFR inhibitors (e.g., gefitinib), are widely used in clinics and have shown excellent therapeutic efficacy.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> However, the emergence of drug resistance, such as that due to T790M mutation, greatly reduces their efficacy.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> EGFR mutant cancer cells have been shown to be sensitive to olaparib both in vivo and in vitro.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Therefore, inhibition of both EGFR and PARP may produce a synergistic effect.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> To verify the concept of dual PROTAC shown above, we designed the first round of dual PROTAC molecules by merging EGFR inhibitors and PARP inhibitors with the E3 ligase ligand in one novel star-shaped molecule and evaluated their capability to degrade two independent targets at the cellular level.</div><div class="NLM_p">As illustrated in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, a convergent synthetic strategy was rationally designed based on our synthetic experience. The first etherification of the free hydroxyl group of tyrosine and serine with propargyl bromide provided the protected amino acid A. Subsequently, a sequential amide condensation reaction was utilized as a practical operation to induce Gefitinib and Olaparib to the star-type linker, generating the key intermediate B. Eventually, the classical copper-promoted click reaction of azide and alkene was used to connect E3 ligands, CRBN, and VHL, respectively. Based on this concise synthetic route, a small library of eight dual PROTACs, including four CRBD ligand-based PROTACs (<b>DP-C 1-4</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">Figure S1</a>), four VHL ligand-based PROTACs (<b>DP-V 1-4</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">Figure S2</a>), and the corresponding four mono PROTACs in which one single inhibitor was linked (<b>MP-GC</b>, <b>MP-GV</b>, <b>MP-OC</b>, <b>MP-OV</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">Figure S3</a>) were efficiently constructed. <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> illustrates the detailed synthetic routes for the four mono PROTACs and eight dual PROTACs.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes for Mono PROTACs (<b>MP-GC</b>, <b>MP-GV</b>, <b>MP-OC</b>, <b>MP-OV</b>) and Dual PROTACs (<b>DP-C 1-4</b>, <b>DP-V 1-4</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Compound <b>1</b> was treated with 1 equiv of compound <b>2</b> and 2 equiv of <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) in dry dimethylformamide (DMF) at 85 °C overnight; (b) 1 equiv of acid and amine was treated with 1.2 equiv of 1-ethyl-3(3-dimethylpropylamine) carbodiimide (EDCI), 1-hydroxybenzotriazole (HOBt), and DIPEA in dry dichloromethane (DCM) at room temperature overnight; (c) compound <b>7</b> was treated with 1.1 equiv of compound <b>8</b> and 2 equiv of K<sub>2</sub>CO<sub>3</sub> in acetone by refluxing overnight; (d) ester was treated with 5 equiv of NaOH in 1:1 mixture solvent of methanol and THF at room temperature overnight; (e) 1.0 equiv of alkyne and azide was treated with 1 equiv of CuSO<sub>4</sub> and 2 equiv of sodium ascorbate in 4:1 mixture solvent of THF and water at room temperature for a few hours; (f) phenol or alcohol was treated with 2 equiv of bromide <b>17</b> and 2 equiv of K<sub>2</sub>CO<sub>3</sub> or NaH in dry DMF for a few hours; and (g) the Boc-protecting group was removed by treatment with HCl ethyl acetate solution at room temperature overnight.</p></p></figure><div class="NLM_p">Due to drug resistance shown in pancreatic cancer and nonsmall cell lung cancer, as well as the high expression of EGFR and PARP,<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22 ref23">(18−23)</a> SW1990 and H1299 cell lines were used in the experiments. Intracellularly, the degradation of EGFR in the SW1990 cell line was induced by the CRBN-based mono PROTAC of Gefitinib (<b>MP-GC</b>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), and the CRBN-based mono PROTAC of Olaparib (<b>MP-OC</b>) also exhibited a PARP degradation effect in the SW1900 cell line (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). However, neither Gefitinib nor Olaparib itself could induce the degradation of two targets in SW1990 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">Figures S4 and S5</a>).</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <b>DP-C-1</b> degraded EGFR and PARP simultaneously in a dose- and time-dependent manner in SW1990 cells. (A) Changes in associated proteins 24 h after the addition of <b>MP-GC</b>. (B) Changes in associated proteins 24 h after the addition of <b>MP-OC</b>. (C) Effect of four dual PROTAC compounds (<b>DP-C 1-4</b>) on the related proteins after 24 h. (D) Effect of 5 μM <b>DP-C-1</b> on related proteins at different time points. (E) Effect of <b>DP-C-1</b> on related proteins after the introduction of 700 nM MG132. EGFR and PARP degradation by <b>DP-C-1</b> in 24 h could be abrogated by pretreating with MG132 for 4 h in 700 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Further experiments revealed that all four CRBN-based dual PROTACs (<b>DP-C 1-4</b>) had degradation effects on EGFR and PARP simultaneously in SW1990 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C, compound <b>DP-C-1</b> showed the best degradation effect on both EGFR and PARP. The remaining compounds showed lower degradation ability on both targets compared to <b>DP-C-1</b>. These results also indicate that the degradation of EGFR and PARP by <b>DP-C 1-4</b> was enhanced by increasing concentrations. The simultaneous degradation of EGFR and PARP by a single dual PROTAC molecule successfully proves the concept of dual-targeting degradation.</div><div class="NLM_p">To further demonstrate the degradation kinetics of <b>DP-C-1</b>, time-dependent experiments were carried out. The results showed that <b>DP-C-1</b> began to degrade EGFR and PARP at 6 h. At first, the degradation rate increased with increasing time, but it started to decrease at 48 h (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">S6</a>). Therefore, <b>DP-C-1</b> degrades EGFR and PARP in the SW1990 cell line in a time-dependent manner within 48 h and the optimal degradation rate was around 24 and 36 h.</div><div class="NLM_p">PROTACs exert their degradation effect on target proteins through the ubiquitin–proteasome system. Therefore, MG132, a proteasome inhibitor, is widely used as a tool to verify the mechanism of such an action.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24−26)</a> To further verify that the degradation of proteins by <b>DP-C-1</b> functions via the ubiquitination pathway, the effect of <b>DP-C-1</b> was assayed by introducing the proteasome inhibitor MG132. It can be seen from <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E that due to the inhibitory effect of MG132 itself on the SW1990 cells, the group with MG132 alone had a slightly lower expression level of both EGFR and PARP compared to that of the negative control (NC) group. However, the addition of <b>DP-C-1</b> in various concentrations ceases to further decrease the expression levels of both targets, and they maintain approximately the same level. This shows that the degradation of <b>DP-C-1</b> is inhibited by MG132. These results suggest that the degradation of the target protein induced by <b>DP-C-1</b> functions via the proteasome pathway.</div><div class="NLM_p">To further verify the concept of dual PROTAC, another E3 ligase ligand (VHL-L) was chosen to develop new dual PROTACs. First, the corresponding mono PROTACs, <b>MP-GV</b> and <b>MP-OV</b>, were synthesized based on Gefitinib, Olaparib, and VHL ligands. In the H1299 cell line, <b>MP-GV</b> and <b>MP-OV</b> could induce EGFR and PARP degradation, respectively, while exhibiting no effect on the other target proteins (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A,B). According to this result, four dual PROTACs based on the VHL ligand were further developed and designated as <b>DP-V 1-4</b>, respectively. To evaluate their degradation ability of EGFR and PARP, H1299 cells were treated with these four compounds under six different concentrations for 36 h. Among the different compounds, <b>DP-V-4</b> exhibited the best degradation ability for both EGFR and PARP (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">S7</a>) and was selected for further investigation. Meanwhile, further experiments revealed that <b>DP-V-4</b> could also degrade EGFR and PARP in human epidermal carcinoma A431 cells simultaneously (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">Figure S8</a>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <b>DP-V-4</b> degraded EGFR and PARP simultaneously in a dose- and time-dependent manner in H1299 cells. (A) Changes in associated proteins 36 h after the addition of <b>MP-GV</b>. (B) Changes in associated proteins 36 h after the addition of <b>MP-OV</b>. (C) Effect of three dual PROTAC compounds (<b>DP-V 1-3</b>) on related proteins after 36 h. (D) Effect of <b>DP-V-4</b> on related proteins with different concentrations at 36 h. (E) Effect of 3.75 μM <b>DP-V-4</b> on related proteins at different times. (F) Effect of <b>DP-V-4</b> on related proteins after the introduction of 1 μM MG132. EGFR and PARP degradation by <b>DP-V-4</b> in 6 h could be abrogated by pretreating with MG132 for 4 h in 1 μM. (G) The antiproliferative activity of Gefitinib, Olaparib, and <b>DP-V-4</b> at 72 h was identified by a cell counting kit-8 (CCK-8) assay. Half-maximal inhibitory concentration (IC<sub>50</sub>) was expressed as mean ± standard deviation (SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In H1299 cells, <b>DP-V-4</b> degraded EGFR and PARP simultaneously in a dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). At a concentration of 0.47 μM, <b>DP-V-4</b> could induce about 50% of PARP degradation, and as the concentration increased, it induced PARP degradation to a greater level. For EGFR, <b>DP-V-4</b> also induced degradation at higher levels when the compound concentration gradually increased. Moreover, <b>DP-V-4</b> could significantly induce EGFR and PARP degradation at 15 μM. Apart from EGFR and PARP, p-EGFR concentrations decreased visibly when treated with <b>DP-V-4</b>, which could be the result of its degradation by <b>DP-V-4</b>, or reduced supply from EGFR.</div><div class="NLM_p">To further explore the mode of action, different time points were set in H1299 cells after being treated with 4 μM of <b>DP-V-4</b>. At 6 h, <b>DP-V-4</b> showed the best degradation effect, and EGFR and PARP recovered as time went on (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E), which might be explained by the accumulation of newly synthesized proteins in cells.</div><div class="NLM_p">To further verify that the degradation of proteins in cells by <b>DP-V-4</b> functions via the ubiquitination pathway, the effect of <b>DP-V-4</b> was assayed by introducing the proteasome inhibitor MG132. In H1299 cells, when pretreated with 1 μM MG132 for 4 h, the degradation of EGFR and PARP by <b>DP-V-4</b> completely terminated (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). These results proved that <b>DP-V-4</b> degrades EGFR and PARP through the ubiquitin–proteasome system.</div><div class="NLM_p">Besides degradation ability, we attempted to identify the antiproliferative activity of dual PROTACs in H1299 tumor cells. The CCK-8 assay determined Gefitinib, Olaparib, and <b>DP-V-4</b> with IC<sub>50</sub>s of 6.56 ± 1.07, 35.93 ± 1.05, and 19.92 ± 1.08 μM, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G). The relatively weaker antiproliferative activity of <b>DP-V-4</b> might be explained by its poorer solubility and cell penetrability, owing to its higher molecular weight.</div><div class="NLM_p">To further confirm the target engagement of these dual PROTACs, microscale thermophoresis (MST) was employed to assay the binding affinity between the two targets and two representative dual PROTACs, <b>DP-C-1</b> (CRBN based) and <b>DP-V-4</b> (VHL based). As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, although a little weaker binding was observed for both <b>DP-C-1</b> and <b>DP-V-4</b> compared to Gefitinib with EGFR (<i>K</i><sub>d</sub> 0.47 ± 0.05 μM) and Olaparib with PARP (<i>K</i><sub>d</sub> 5.31 ± 1.11 μM), they still retained strong binding with EGFR and PARP, with the disassociation constants for <b>DP-C-1</b> at 2.74 ± 0.13 and 7.89 ± 0.96 μM with EGFR and PARP, respectively, and for <b>DP-V-4</b> at 5.47 ± 0.67 and 12.80 ± 2.16 μM with EGFR and PARP, respectively.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding affinity of <b>DP-C-1</b> and <b>DP-V-4</b> with EGFR and PARP. (A–C) Measurement of affinity of <b>DP-C-1</b>, <b>DP-V-4</b>, and Gefitinib to EGFR by MST with <i>K</i><sub>d</sub> values of 2.74 ± 0.13, 5.47 ± 0.67, and 0.47 ± 0.05 μM, respectively. (D–F) Measurement of affinity of <b>DP-C-1</b>, <b>DP-V-4</b>, and Olaparib to PARP by MST with <i>K</i><sub>d</sub> values of 7.89 ± 0.96, 12.80 ± 2.16, and 5.31 ± 1.11 μM, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this study, we proposed the new concept of dual-targeting PROTAC. Inspired by bispecific antibodies,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> we creatively proposed the idea of designing a dual-targeting PROTAC molecule with two different warheads, in which an E3 ligand was linked to two inhibitors with different types/pathways of targets, and was capable of simultaneously degrading two completely different target proteins in tumor cells. To verify this concept, we used Gefitinib (an existing EGFR inhibitor) and Olaparib (a PARP inhibitor)<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> as substrates to synthesize dual-targeting degradation chimeric molecules (dual PROTAC) of EGFR and PARP proteins with different linker lengths and different E3 ligands (CRBN or VHL), which successfully degraded EGFR and PARP simultaneously in cancer cells. This is the first successful example of dual PROTACs.</div><div class="NLM_p">In addition, we used biocompatible natural amino acids as a linker to achieve the rapid synthesis of star-type dual-targeting PROTACs. The technique and method established in this study can be easily adapted for the synthesis of more dual-targeting PROTAC molecules just like building blocks, and it will greatly expand the application of PROTAC technology while opening up a new avenue for drug discovery. In the present clinical practice, the combination therapy of two different inhibitors, or even the treatment of some bispecific antibodies, could be substituted by the strategy shown in this study through designing a corresponding single dual PROTAC molecule for double targeting degradation, with the same or even better therapeutic effects.</div><div class="NLM_p last">It is easy to recognize that in the field of combination therapy of cancer, dual PROTACs can be quickly applied to replace many current combinations. For example, we could design and synthesize the dual PROTACs targeting two kinases, which are synthetically lethal, or targeting a tumor immune target plus an adjuvant immune target, kinase, or energy metabolism target, or targeting an epigenetic target plus an antiapoptotic target, etc. Of course, in addition to the obvious advantages listed above, the increased molecular weight of dual PROTACs will give rise to some issues in the aspect of drug properties and pharmacokinetics. The approach to solving these problems lies in two directions. One is to employ a nanodrug delivery system to improve drug absorption, and the other is to simplify the inhibitor part while retaining the minimum pharmacophore. We are currently working on realizing these approaches.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37107" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37107" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, we have designed and synthesized the first dual-targeting PROTAC molecules with two different warheads, and realized the simultaneous degradation of two completely different types of target proteins by one small molecule in tumor cells. We believe that this technique will greatly expand the application of PROTAC technology and open up a new field of drug discovery.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06750" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06750" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Cell Culture</h3><div class="NLM_p last">Cells were purchased from the Chinese Typical Culture Collection Center. The cells were cultured in Dulbecco’s modified Eagle’s medium containing 25 mM glucose (high DMEM, GIBCO) supplemented with 10% (v/v) fetal bovine serum in a saturated humidified atmosphere of 5% CO<sub>2</sub> at 37 °C. All cell lines were authenticated by short tandem repeat (STR) analysis.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Immunoblotting</h3><div class="NLM_p last">The cells were seeded in six-well plates at an individual density for 24 h and then treated with dimethyl sulfoxide (DMSO) or compounds at the indicated concentrations for various times. After being treated with the indicated concentrations of compounds for the specific time, the cells were harvested and washed once with phosphate-buffered saline (PBS) and then lysed in radioimmunoprecipitation assay (RIPA) lysis buffer (Beyotime, #P0013C) according to the directions. The suspension was centrifuged at 13 000 rpm for 15 min to remove insoluble material, and a bicinchoninic acid (BCA) assay was employed to determine the concentration of soluble proteins (Beyotime, #P0011). The lysate protein (20–40 μg) was subjected to 10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and electrophoretically transferred to poly(vinylidene difluoride) (PVDF) membranes (Millipore, #IPVH00010). The membranes were blocked with 5% nonfat milk or 5% bovine serum albumin (BSA) for 1 h and then incubated with the specific primary antibodies with a 1:1000 dilution, respectively, at 4 °C overnight. The primary antibodies and secondary antibodies were purchased from Abclonal and Biosharp, respectively. Membranes were imaged by the Tanon-5200 Chemiluminescent Imaging System with ECL luminescence reagents (Meilunbio, #MA0186).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Cell Proliferation Inhibition Assay</h3><div class="NLM_p last">About 4000 cells per well were seeded in 96-well plates in 100 μL of medium and incubated overnight. After that, the cells were treated with a series of concentrations of specific compounds for a further 72 h. Cell counting kit-8 (CCK-8) (Targetmol, #C0005) was used to identify the antiproliferative activity of compounds, and plates were read at 450 nm using a Bio-Tek Synergy 2 microplate reader. Data were analyzed using GraphPad Prism 7.0 Software.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Clone, Expression, and Purification of PARP</h3><div class="NLM_p last">Human PARP-1 ART domain Asp788-Trp1014 (NCBI RefSeq: <a href="https://NP_001609.2" class="ext-link">NP_001609.2</a>) containing a GST tag followed by the HRV-3C cleavage site was cloned into a PGEX-6p-1 and heterologously expressed in <i>Escherichia coli</i> strain BL21 (DE3).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The cells were cultured at 37 °C for 3–4 h and supplemented with 0.4 mM isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside (IPTG) and 0.1 mM ZnSO<sub>4</sub> to boost protein expression. The cells were collected by centrifugation at 3000<i>g</i> for 10 min after overnight growth at 16 °C. The cell pellet was resuspended in lysis buffer (50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES), pH 8.0, 200 mM NaCl, 5 mM 2-mercaptoethanol, 1 mM phenylmethylsulfonyl fluoride (PMSF)) and homogenized using a homogenizer on ice, followed by centrifugation at 40 000<i>g</i> for 1 h. After centrifugation, the supernatant was collected and incubated with 3–4 mL of GST agarose (ABclonal) in a gravity column (Bio-Rad), washed with 30 mL of buffer A (50 mM HEPES, pH 8.0, 200 mM NaCl, 1 mM dithiothreitol (DTT)). The washed agarose was left on-column in buffer A and digested with HRV-3C protease overnight. After HRV-3C protease digestion, the flow-through containing an untagged PARP-1 ART domain was collected, concentrated, and purified by size-exclusion chromatography on a Superdex 200 10/300 GL column (GE Healthcare) pre-equilibrated with buffer C (20 mM HEPES, pH 8.0, 150 mM NaCl, 1 mM DTT). The protein peak was collected for further use.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Microscale Thermophoresis (MST) Assay</h3><div class="NLM_p last">The MST assay was performed according to the supplied labeling protocol. The EFGR kinase domain (purchased from Sino Biological: Cat 10001-H20B2) and the PARP-1 ART domain were diluted with assay buffer (20 mM HEPES, pH 7.5, 150 mM NaCl) to 10 μM and labeled with the Monolith NTTM Protein Labeling Kit RED (Cat # L001).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> After protein purification, the labeled EFGR and PARP concentration were adjusted to 200 nM. The high concentration of compounds was used as the initial concentration for subsequent gradient dilution. Mixed the different concentrations of compounds with EFGR or PARP and incubated for 10 min at room temperature. The samples were loaded into Monolith standard-treated capillaries, and the thermophoresis was measured at room temperature after 30 min incubation on a Monolith NT.115 instrument (Germany). Furthermore, the laser power was set to 40% using 15 s on-time. The light-emitting diode (LED) power was set to 100%. The dissociation constant <i>K</i><sub>d</sub> values were fitted using NT-Analysis software.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Synthesis of Compounds and Their Characterization</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> General Information</h4><div class="NLM_p last">Compound <b>7</b> (CAS: 184475-71-6) and compound <b>13</b> (CAS: 763111-47-3) were both purchased from Bide Pharmatech Ltd. Unless otherwise noted, reagents and solvents were obtained from commercial suppliers and were used directly without further purification. Reactions were monitored by thin-layer chromatography (TLC) analysis, which was performed on aluminum plates precoated with silica gel (GF-254), and visualized by UV fluorescence (λ<sub>max</sub> = 254 nm) and/or by staining with 1% w/v KMnO<sub>4</sub> in 0.5 M aqueous K<sub>2</sub>CO<sub>3</sub>. Products were purified by flash column chromatography, which was performed using silica gel (230-400 mesh). <sup>1</sup>H NMR spectra were recorded on a 400 MHz Bruker Avance III spectrometer. Chemical shifts for <sup>1</sup>H NMR spectra are reported in parts per million (ppm) from tetramethylsilane with the solvent resonance as the internal standard (CDCl<sub>3</sub>, 7.26 ppm; DMSO-<i>d</i><sub>6</sub>, 2.50 ppm). Data are reported as follows: chemical shift, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, p = quintet, m = multiplet, br = broad), coupling constants (Hz), and integration. A high-resolution mass spectrum (HRMS) was measured on a FLEX-PC (ultraflex TOF/TOF) instrument equipped with an atmospheric pressure chemical ionization (APCI) or electrospray ionization (ESI) source in the positive-ion mode. The purity of compounds was determined by high-performance liquid chromatography (HPLC) analyses, and the purity was >95% for all tested compounds.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> General Procedure for the Preparation of Compound <b>3</b></h4><div class="NLM_p">A 25 mL reaction tube was charged with compound <b>1</b> (0.27 g, 1.0 mmol, 1.0 equiv) and 1 equiv of compound <b>2</b>. Then, 5 mL of dry DMF and 2 equiv of DIPEA were added to the mixture and the resulting mixture was stirred at 85 °C overnight. The reaction was monitored by TLC. When the reaction was completed, 50 mL of ethyl acetate was added and the mixture was washed with brine several times and then dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed and the residue was purified with column chromatography on silica gel, eluting with petroleum ether and ethyl acetate (gradient elution from 2:1 to 1:1) to afford the corresponding product <b>3</b> as a yellow solid.</div><div id="sec4_6_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i16" class="anchor-spacer"></div><h5 class="article-section__title" id="_i16"> 4-((2-(2-Azidoethoxy)ethyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>3a</b>)</h5><div class="NLM_p last">0.11 g, 29% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.31 (s, 1H), 7.50 (t, <i>J</i> = 8.0 Hz, 1H), 7.11 (d, <i>J</i> = 7.2 Hz, 1H), 6.94 (d, <i>J</i> = 8.4 Hz, 1H), 6.50 (t, <i>J</i> = 5.6 Hz, 1H), 4.92 (dd, <i>J</i> = 12.0, 5.2 Hz, 1H), 3.73 (t, <i>J</i> = 5.2 Hz, 2H), 3.68 (t, <i>J</i> = 5.2 Hz, 2H), 3.50 (q, <i>J</i> = 5.6 Hz, 2H), 3.41 (t, <i>J</i> = 5.2 Hz, 2H), 2.87–2.72 (m, 3H), 2.14–2.10 (m, 1H).</div></div><div id="sec4_6_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> 4-((2-(2-(2-Azidoethoxy)ethoxy)ethyl)amino)-2-(2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>3b</b>)</h5><div class="NLM_p last">0.14 g, 33% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.06 (brs, 1H), 7.49 (t, <i>J</i> = 7.8 Hz, 1H), 7.09 (dd, <i>J</i> = 7.2, 2.0 Hz, 1H), 6.93 (d, <i>J</i> = 8.4 Hz, 1H), 6.50 (t, <i>J</i> = 5.6 Hz, 1H), 4.96–4.92 (m, 1H), 3.74 (t, <i>J</i> = 5.2 Hz, 2H), 3.70–3.67 (m, 6H), 3.48 (q, <i>J</i> = 5.6 Hz, 2H), 3.38 (t, <i>J</i> = 4.4 Hz, 2H), 2.80–2.73 (m, 3H), 2.13–2.10 (m, 1H).</div></div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> General Procedure for Amide Condensation</h4><div class="NLM_p">One equiv of acid and amine was added to 50 mL of dry dichloromethane at 0 °C, then 1.2 equiv of EDCI, HOBt, and DIPEA was added in a sequence. Then, the reaction mixture was warmed to room temperature and stirred overnight. The reaction was monitored by TLC. When the reaction was completed, 50 mL of dichloromethane and 50 mL of saturated aqueous NaHCO<sub>3</sub> were added. The organic phase was separated and washed with 50 mL of brine and then dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed with a rotary evaporator. The residue was purified with column chromatography on silica gel, eluting with dichloromethane and methanol to afford the corresponding amide product.</div><div id="sec4_6_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-Azidobutanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>6</b>)</h5><div class="NLM_p last"><b>6</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>4</b> (0.86 g, 2.0 mmol, 1.0 equiv) and 1 equiv of compound <b>5</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 50:1 to 20:1) to afford the corresponding product as a white solid, 0.71 g, 65% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.70 (s, 1H), 7.37–7.32 (m, 5H), 6.36 (d, <i>J</i> = 8.4 Hz, 1H), 4.70 (t, <i>J</i> = 8.0 Hz, 1H), 4.58–4.52 (m, 3H), 4.34 (dd, <i>J</i> = 15.2, 5.6 Hz, 1H), 4.06 (d, <i>J</i> = 11.6 Hz, 1H), 3.63 (dd, <i>J</i> = 11.2, 3.6 Hz, 1H), 3.32 (td, <i>J</i> = 6.4, 1.6 Hz, 2H), 2.50 (s, 3H), 2.48–2.45 (m, 1H), 2.36–2.34 (m, 2H), 2.17–2.11 (m, 1H), 1.87 (dtd, <i>J</i> = 14.2, 6.8, 3.2 Hz, 2H), 0.94 (s, 9H).</div></div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> General Procedure for the Preparation of Compound <b>9</b></h4><div class="NLM_p">A 50 mL reaction tube was charged with compound <b>7</b> (0.64 g, 2.0 mmol, 1.0 equiv), 1.1 equiv of compound <b>8</b>, and 2 equiv of K<sub>2</sub>CO<sub>3</sub>. Then, 10 mL of acetone was added. The reaction tube was sealed and the resulting mixture was heated to refluxing and stirred overnight. The reaction was monitored by TLC. When the reaction was completed, the mixture was cooled down to room temperature and filtered. The filtrate was removed using a rotary evaporator. The residue was purified with column chromatography on silica gel, eluting with petroleum ether and ethyl acetate (1:1) to afford the corresponding product <b>9</b> as a white solid.</div><div id="sec4_6_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Ethyl 2-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)acetate (<b>9a</b>)</h5><div class="NLM_p last">0.34 g, 42% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.66 (brs, 1H), 8.48 (d, <i>J</i> = 4.4 Hz, 1H), 8.10 (d, <i>J</i> = 6.8 Hz, 1H), 7.87 (s, 1H), 7.74 (d, <i>J</i> = 5.0 Hz, 1H), 7.15 (td, <i>J</i> = 13.2, 2.8 Hz, 1H), 7.22 (d, <i>J</i> = 3.2 Hz, 1H), 4.94 (s, 2H), 4.21 (q, <i>J</i> = 7.2 Hz, 2H), 3.96 (s, 3H), 1.24 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_6_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> Methyl 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)butanoate (<b>9b</b>)</h5><div class="NLM_p last">0.43 g, 51% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.68 (s, 1H), 8.17 (brs, 1H), 8.05–8.02 (m, 2H), 7.87 (s, 1H), 7.26 (s, 1H), 7.16 (t, <i>J</i> = 8.8 Hz, 1H), 4.31 (t, <i>J</i> = 8.0 Hz, 2H), 4.02 (s, 3H), 3.86 (s, 3H), 2.59 (t, <i>J</i> = 6.0 Hz, 2H), 2.29–2.22 (m, 2H).</div></div><div id="sec4_6_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> Methyl 6-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)hexanoate (<b>9c</b>)</h5><div class="NLM_p last">0.54 g, 60% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.64 (s, 1H), 7.90 (dd, <i>J</i> = 6.4, 2.4 Hz, 1H), 7.83 (brs, 1H), 7.63–7.58 (m, 1H), 7.24 (d, <i>J</i> = 5.2 Hz, 2H), 7.15 (t, <i>J</i> = 8.4 Hz, 1H), 4.11 (t, <i>J</i> = 8.4 Hz, 2H), 3.98 (s, 3H), 3.68 (s, 3H), 2.39 (t, <i>J</i> = 7.2 Hz, 2H), 1.91 (p, <i>J</i> = 7.2 Hz, 2H), 1.74 (p, <i>J</i> = 7.2 Hz, 2H), 1.55 (p, <i>J</i> = 7.2 Hz, 2H).</div></div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> General Procedure for Saponification</h4><div class="NLM_p">One equiv of ester and 5 equiv of NaOH were added to the 1:1 mixture solvent of methanol and THF. The resulting mixture was stirred at room temperature overnight. The reaction was monitored by TLC. When the reaction was completed, 2 M HCl ethyl acetate solution was added to the mixture until the pH reached 2. Then, the solvent was removed to afford the corresponding acid product, which was used directly for the next step.</div><div id="sec4_6_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> 2-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)acetic Acid (<b>10a</b>)</h5><div class="NLM_p last"><b>10a</b> was prepared according to the general procedure for saponification, starting from compound <b>9a</b> (0.41 g, 1.0 mmol, 1.0 equiv). 0.34 g, 90% yield, white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.15 (brs, 1H), 12.06 (s, 1H), 8.85 (s, 1H), 8.78 (s, 1H), 8.12 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 7.86 (ddd, <i>J</i> = 9.0, 4.4, 2.6 Hz, 1H), 7.60 (s, 1H), 7.53 (t, <i>J</i> = 13.2 Hz, 1H), 5.09 (s, 2H), 4.00 (s, 3H).</div></div><div id="sec4_6_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)butanoic Acid (<b>10b</b>)</h5><div class="NLM_p last"><b>10b</b> was prepared according to the general procedure for saponification, starting from compound <b>9b</b> (0.42 g, 1.0 mmol, 1.0 equiv). 0.37 g, 92% yield, white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.03 (s, 2H), 8.84 (s, 1H), 8.69 (s, 1H), 8.10 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 7.87–7.84 (m, 1H), 7.56 (s, 1H), 7.53 (t, <i>J</i> = 9.2 Hz, 1H), 4.32 (t, <i>J</i> = 6.4 Hz, 2H), 3.99 (s, 3H), 2.46 (t, <i>J</i> = 7.2 Hz, 2H), 2.03 (p, <i>J</i> = 6.8 Hz, 2H).</div></div><div id="sec4_6_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> 6-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)hexanoic Acid (<b>10c</b>)</h5><div class="NLM_p last"><b>10c</b> was prepared according to the general procedure for saponification, starting from compound <b>9c</b> (0.45 g, 1.0 mmol, 1.0 equiv). 0.42 g, 96% yield, white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.13 (s, 2H), 8.84 (s, 1H), 8.73 (s, 1H), 8.12 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 7.89–7.86 (m, 1H), 7.56 (s, 1H), 7.52 (t, <i>J</i> = 8.8 Hz, 1H), 4.29 (t, <i>J</i> = 6.4 Hz, 2H), 3.98 (s, 3H), 2.26 (t, <i>J</i> = 7.2 Hz, 2H), 1.81 (p, <i>J</i> = 7.2 Hz, 2H), 1.60 (p, <i>J</i> = 7.2 Hz, 2H), 1.48 (p, <i>J</i> = 7.2 Hz, 2H).</div></div><div id="sec4_6_5_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(prop-2-yn-1-yl)butanamide (<b>12</b>)</h5><div class="NLM_p last"><b>12</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>10b</b> (81 mg, 0.2 mmol, 1.0 equiv) and 1 equiv of compound <b>11</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 80:1 to 30:1) to afford the corresponding product as a white solid, 55 mg, 63% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.54 (brs, 1H), 8.51 (s, 1H), 8.39 (t, <i>J</i> = 5.2 Hz, 1H), 8.13 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 7.82–7.79 (m, 2H), 7.45 (t, <i>J</i> = 8.8 Hz, 1H), 7.21 (s, 1H), 4.15 (t, <i>J</i> = 6.0 Hz, 2H), 3.95 (s, 3H), 3.88 (dd, <i>J</i> = 5.2, 2.4 Hz, 2H), 3.11 (t, <i>J</i> = 2.4 Hz, 1H), 2.36 (t, <i>J</i> = 7.2 Hz, 2H), 2.07 (p, <i>J</i> = 6.8 Hz, 2H).</div></div><div id="sec4_6_5_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> 4-(4-Fluoro-3-(4-(pent-4-ynoyl)piperazine-1-carbonyl)benzyl)phthalazin-1(2<i>H</i>)-one (<b>15</b>)</h5><div class="NLM_p last"><b>15</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>13</b> (0.73 g, 2.0 mmol, 1.0 equiv) and 1 equiv of compound <b>14</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 100:1 to 50:1) to afford the corresponding product as a white solid, 0.69 g, 78% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.86 (brs, 1H), 8.49–8.47 (m, 1H), 7.79–7.72 (m, 3H), 7.38–7.27 (m, 2H), 7.05 (t, <i>J</i> = 9.2 Hz, 1H), 4.30 (s, 2H), 3.80 (brs, 3H), 3.59 (brs, 2H), 3.46 (brs, 1H), 3.30 (brs, 2H), 2.62–2.55 (m, 4H), 1.99–1.97 (m, 1H).</div></div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> General Procedure for the Alkylation of the Hydroxyl Group</h4><div class="NLM_p">Phenol or alcohol was treated with 2 equiv of bromide <b>17</b> and 2 equiv of K<sub>2</sub>CO<sub>3</sub> or NaH in dry DMF at room temperature or at 0 °C for a few hours. The reaction was monitored by TLC. When the reaction was completed, 200 mL of ethyl acetate and 200 mL of brine were added. The organic phase was separated and washed with 50 mL of brine, and then dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed using a rotary evaporator. The residue was purified with column chromatography on silica gel, eluting with petroleum ether and ethyl acetate to afford the corresponding product.</div><div id="sec4_6_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Methyl 2-((<i>tert</i>-Butoxycarbonyl)amino)-3-(4-(prop-2-yn-1-yloxy)phenyl)propanoate (<b>18</b>)</h5><div class="NLM_p last"><b>18</b> was prepared according to the general procedure for the alkylation of the hydroxyl group, starting from compound <b>16</b> (5.90 g, 20 mmol, 1.0 equiv), 2 equiv of compound <b>17</b>, and 2 equiv of K<sub>2</sub>CO<sub>3</sub> at room temperature for 6 h. Chromatography purification used petroleum ether and ethyl acetate as elution solvents (8:1) to afford the corresponding product as a yellow oil, 6.50 g, 98% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.06 (d, <i>J</i> = 8.4 Hz, 2H), 6.91 (d, <i>J</i> = 8.8 Hz, 2H), 4.98 (d, <i>J</i> = 8.0 Hz, 1H), 4.67 (d, <i>J</i> = 2.4 Hz, 2H), 4.55 (dd, <i>J</i> = 14.0, 6.0 Hz, 1H), 3.72 (s, 3H), 3.03 (qd, <i>J</i> = 14.0, 6.0 Hz, 2H), 2.53 (t, <i>J</i> = 2.4 Hz, 1H), 1.42 (s, 9H).</div></div><div id="sec4_6_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 2-((<i>tert</i>-Butoxycarbonyl)amino)-3-(4-(prop-2-yn-1-yloxy)phenyl)propanoic Acid (<b>19</b>)</h5><div class="NLM_p last"><b>19</b> was prepared according to the general procedure for saponification, starting from compound <b>18</b> (0.66 g, 2.0 mmol, 1.0 equiv). 0.54 g, 85% yield, yellow oil, which was used directly for the next step.</div></div><div id="sec4_6_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> <i>tert</i>-Butyl(1-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-1-oxo-3-(4-(prop-2-yn-1-yloxy)phenyl)propan-2-yl)carbamate (<b>20</b>)</h5><div class="NLM_p last"><b>20</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>13</b> (0.73 g, 2.0 mmol, 1.0 equiv) and 1 equiv of compound <b>19</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 80:1 to 30:1) to afford the corresponding product as a white solid, 0.60 g, 45% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.76 (brs, 1H), 8.49–8.46 (m, 1H), 7.80–7.69 (m, 3H), 7.30–7.28 (m, 2H), 7.13 (t, <i>J</i> = 8.8 Hz, 2H), 7.02 (dd, <i>J</i> = 15.2, 8.8 Hz, 1H), 6.99 (dd, <i>J</i> = 15.6, 8.4 Hz, 2H), 5.44 (t, <i>J</i> = 8.0 Hz, 1H), 4.84–4.71 (m, 1H), 4.67 (dd, <i>J</i> = 10.8, 6.0 Hz, 2H), 4.28 (s, 2H), 3.83 (brs, 1H), 3.60–3.35 (m, 3H), 3.27–3.21 (m, 1H), 3.08–3.01 (m, 2H), 2.97 (dd, <i>J</i> = 13.6, 5.6 Hz, 1H), 2.89 (dd, <i>J</i> = 12.8, 5.6 Hz, 1H), 2.53 (t, <i>J</i> = 2.4 Hz, 1H), 1.42 (s, 9H).</div></div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> General Procedure for the Deprotection of the Boc Group</h4><div class="NLM_p">Boc-protected amine was treated with HCl ethyl acetate solution at room temperature overnight to remove the Boc group. The reaction was monitored by TLC. When the reaction was completed, the solvent was removed to afford the corresponding product as a white solid, which was used directly for the next step.</div><div id="sec4_6_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 4-(3-(4-(2-Amino-3-(4-(prop-2-yn-1-yloxy)phenyl)propanoyl)piperazine-1-carbonyl)-4-fluorobenzyl)phthalazin-1(2<i>H</i>)-one (<b>21</b>)</h5><div class="NLM_p last"><b>21</b> was prepared according to the general procedure for the deprotection of the Boc group, starting from compound <b>20</b> (0.2 mmol, 1.0 equiv) to afford the corresponding product quantitatively as a white solid, which was used directly for the next step.</div></div><div id="sec4_6_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 2-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(1-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-1-oxo-3-(4-(prop-2-yn-1-yloxy)phenyl)propan-2-yl)acetamide (<b>22</b>)</h5><div class="NLM_p last"><b>22</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>10a</b> (75 mg, 0.2 mmol, 1.0 equiv) and 1 equiv of compound <b>21</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 40:1 to 10:1) to afford the corresponding product as a white solid, 81 mg, 44% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 12.58 (s, 1H), 9.50 (d, <i>J</i> = 2.4 Hz, 1H), 8.53 (s, 1H), 8.25 (d, <i>J</i> = 6.4 Hz, 1H), 8.22 (t, <i>J</i> = 6.8 Hz, 1H), 8.11 (dd, <i>J</i> = 5.2, 2.0 Hz, 1H), 7.96–7.93 (m, 1H), 7.90–7.86 (m, 1H), 7.84–7.77 (m, 3H), 7.47–7.42 (m, 2H), 7.33 (d, <i>J</i> = 4.0 Hz, 1H), 7.22 (t, <i>J</i> = 7.2 Hz, 1H), 7.11 (dd, <i>J</i> = 11.2, 6.4 Hz, 2H), 6.81 (dd, <i>J</i> = 11.2, 6.4 Hz, 2H), 5.01 (ddd, <i>J</i> = 44.8, 13.8, 6.6 Hz, 1H), 4.71–4.69 (m, 4H), 4.32 (s, 2H), 3.94 (s, 3H), 3.56–3.44 (m, 7H), 3.11–3.08 (m, 2H), 2.97–2.84 (m, 3H).</div></div><div id="sec4_6_7_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 2-((<i>tert</i>-Butoxycarbonyl)amino)-3-(prop-2-yn-1-yloxy)propanoic Acid (<b>24</b>)</h5><div class="NLM_p last"><b>24</b> was prepared according to the general procedure for the alkylation of the hydroxyl group, starting from compound <b>23</b> (4.10 g, 20 mmol, 1.0 equiv), 2 equiv of compound <b>17</b>, and 2 equiv of NaH at 0 °C, then the mixture was warmed to room temperature, and stirred overnight. Chromatography purification used petroleum ether and ethyl acetate as elution solvents (1:1) to afford the corresponding product as a yellow oil, 3.61 g, 74% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.51 (brs, 1H), 5.42 (d, <i>J</i> = 8.4 Hz, 1H), 4.50 (dd, <i>J</i> = 5.2, 3.2 Hz, 1H), 4.19–4.18 (m, 2H), 4.00 (dd, <i>J</i> = 9.2, 3.2 Hz, 1H), 3.81 (dd, <i>J</i> = 9.2, 3.2 Hz, 1H), 2.48 (t, <i>J</i> = 2.4 Hz, 1H), 1.46 (s, 9H).</div></div><div id="sec4_6_7_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> <i>tert</i>-Butyl(1-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)carbamate (<b>25</b>)</h5><div class="NLM_p last"><b>25</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>13</b> (0.73 g, 2.0 mmol, 1.0 equiv) and 1 equiv of compound <b>24</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 100:1 to 50:1) to afford the corresponding product as a white solid, 0.59 g, 50% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.86 (brs, 1H), 8.49–8.47 (m, 1H), 7.79–7.71 (m, 3H), 7.35–7.32 (m, 2H), 7.05 (t, <i>J</i> = 8.8 Hz, 2H), 5.51 (d, <i>J</i> = 8.0 Hz, 1H), 4.83 (d, <i>J</i> = 28.8 Hz, 1H), 4.30 (s, 2H), 4.15–4.11 (m, 2H), 3.98 (brs, 1H), 3.81–3.36 (m, 9H), 2.44 (d, <i>J</i> = 11.2 Hz, 1H), 1.43 (s, 9H).</div></div><div id="sec4_6_7_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 4-(3-(4-(2-Amino-3-(prop-2-yn-1-yloxy)propanoyl)piperazine-1-carbonyl)-4-fluorobenzyl)phthalazin-1(2<i>H</i>)-one (<b>26</b>)</h5><div class="NLM_p last"><b>26</b> was prepared according to the general procedure for the deprotection of the Boc group, starting from compound <b>25</b> (0.2 mmol, 1.0 equiv) to afford the corresponding product quantitatively as a white solid, which was used directly for the next step.</div></div><div id="sec4_6_7_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(1-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)butanamide (<b>27</b>)</h5><div class="NLM_p last"><b>27</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>10b</b> (81 mg, 0.2 mmol, 1.0 equiv) and 1 equiv of compound <b>26</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 40:1 to 20:1) to afford the corresponding product as a white solid, 110 mg, 64% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 12.60 (s, 1H), 9.57 (brs, 1H), 8.51 (s, 1H), 8.37 (d, <i>J</i> = 8.0 Hz, 1H), 8.28–8.25 (m, 2H), 8.13 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 7.98–7.91 (m, 3H), 7.87–7.78 (m, 3H), 7.40–7.36 (m, 1H), 7.24 (t, <i>J</i> = 9.2 Hz, 1H), 5.00–4.90 (m, 1H), 4.32 (s, 2H), 4.16 (s, 4H), 3.95 (s, 3H), 3.63–3.53 (m, 8H), 3.47 (s, 2H), 3.16–3.13 (m, 2H), 2.39 (q, <i>J</i> = 6.8 Hz, 2H), 2.05 (q, <i>J</i> = 6.0 Hz, 2H).</div></div><div id="sec4_6_7_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> <i>tert</i>-Butyl(2-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-2-oxoethyl)carbamate (<b>29a</b>)</h5><div class="NLM_p last"><b>29a</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>13</b> (0.73 g, 2.0 mmol, 1.0 equiv) and 1 equiv of compound <b>28a</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 40:1 to 20:1) to afford the corresponding product as a white solid, 0.86 g, 82% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.91 (brs, 1H), 8.49–8.47 (m, 1H), 7.79–7.73 (m, 3H), 7.36–7.33 (m, 2H), 7.05 (t, <i>J</i> = 8.8 Hz, 1H), 5.51 (d, <i>J</i> = 16.0 Hz, 1H), 4.30 (s, 2H), 4.03–3.94 (m, 2H), 3.79 (brs, 3H), 3.60 (brs, 1H), 3.50 (brs, 1H), 3.37–3.33 (m, 3H), 1.45 (s, 9H).</div></div><div id="sec4_6_7_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> <i>tert</i>-Butyl(4-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-4-oxobutyl)carbamate (<b>29b</b>)</h5><div class="NLM_p last"><b>29b</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>13</b> (0.73 g, 2.0 mmol, 1.0 equiv) and 1 equiv of compound <b>28b</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 40:1 to 20:1) to afford the corresponding product as a white solid, 0.84 g, 76% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.94 (brs, 1H), 8.49–8.46 (m, 1H), 7.78–7.73 (m, 3H), 7.34–7.27 (m, 2H), 7.05 (t, <i>J</i> = 8.8 Hz, 1H), 4.81 (brs, 1H), 4.30 (s, 2H), 4.38 (brs, 3H), 3.57–3.56 (m, 2H), 3.43 (brs, 1H), 3.30 (brs, 2H), 3.20–3.17 (m, 2H), 2.42 (t, <i>J</i> = 6.8 Hz, 1H), 2.35 (t, <i>J</i> = 6.8 Hz, 1H), 1.88–1.81 (m, 2H), 1.44 (s, 9H).</div></div><div id="sec4_6_7_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> <i>tert</i>-Butyl(6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)carbamate (<b>29c</b>)</h5><div class="NLM_p"><b>29c</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>13</b> (0.73 g, 2.0 mmol, 1.0 equiv) and 1 equiv of compound <b>28c</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 40:1 to 20:1) to afford the corresponding product as a white solid, 0.98 g, 85% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.80 (brs, 1H), 8.49–8.47 (m, 1H), 7.78–7.72 (m, 3H), 7.33 (brs, 2H), 7.05 (t, <i>J</i> = 8.8 Hz, 1H), 4.60 (brs, 1H), 4.30 (s, 2H), 3.78 (brs, 3H), 3.57 (brs, 2H), 3.43 (brs, 1H), 3.29 (brs, 2H), 3.11 (brs, 2H), 2.37–2.30 (m, 2H), 1.66–1.65 (m, 2H), 1.51–1.50 (m, 2H), 1.44 (s, 9H), 1.37–1.36 (m, 2H).</div><div class="NLM_p last"><b>30</b> was prepared according to the general procedure for the deprotection of the Boc group, starting from compound <b>29</b> (0.2 mmol, 1.0 equiv) to afford the corresponding product quantitatively as a white solid, which was used directly for the next step.</div></div><div id="sec4_6_7_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> <i>tert</i>-Butyl(1-((2-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-2-oxoethyl)amino)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)carbamate (<b>31a</b>)</h5><div class="NLM_p last"><b>31a</b> was prepared according to the general procedure for amide condensation, starting from compound <b>24</b> (0.24 g, 1.0 mmol, 1.0 equiv) and 1 equiv of compound <b>30a</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 30:1 to 15:1) to afford the corresponding product as a white solid, 0.27 g, 50% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.27 (d, <i>J</i> = 12.0 Hz, 1H), 8.49–8.46 (m, 1H), 7.78–7.77 (m, 2H), 7.75–7.73 (m, 1H), 7.44 (d, <i>J</i> = 12.8 Hz, 1H), 7.36–7.28 (m, 2H), 7.05 (t, <i>J</i> = 9.2 Hz, 1H), 5.50 (d, <i>J</i> = 6.8 Hz, 1H), 4.39 (brs, 1H), 4.31 (s, 2H), 4.17–4.08 (m, 4H), 3.98 (brs, 1H), 3.79 (brs, 2H), 3.72–3.68 (m, 1H), 3.61 (brs, 1H), 3.53 (brs, 1H), 3.41 (brs, 1H), 3.33 (brs, 2H), 2.46 (q, <i>J</i> = 2.4 Hz, 1H), 1.46 (s, 9H).</div></div><div id="sec4_6_7_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> <i>tert</i>-Butyl(1-((4-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-4-oxobutyl)amino)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)carbamate (<b>31b</b>)</h5><div class="NLM_p last"><b>31b</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>24</b> (0.24 g, 1.0 mmol, 1.0 equiv) and 1 equiv of compound <b>30b</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 30:1 to 15:1) to afford the corresponding product as a white solid, 0.35 g, 61% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 8.26 (d, <i>J</i> = 7.6 Hz, 1H), 7.97 (d, <i>J</i> = 7.6 Hz, 1H), 7.90 (t, <i>J</i> = 7.6 Hz, 2H), 7.84 (t, <i>J</i> = 7.6 Hz, 1H), 7.44 (brs, 1H), 7.37 (t, <i>J</i> = 6.4 Hz, 1H), 7.24 (t, <i>J</i> = 9.2 Hz, 1H), 6.81 (t, <i>J</i> = 6.4 Hz, 1H), 4.33 (s, 2H), 4.12 (d, <i>J</i> = 12.4 Hz, 2H), 4.06–4.02 (m, 1H), 3.64 (brs, 1H), 3.58–3.46 (m, 5H), 3.41–3.38 (m, 1H), 3.19 (brs, 1H), 3.14 (brs, 1H), 3.08 (brs, 2H), 2.33 (t, <i>J</i> = 6.8 Hz, 1H), 2.26 (t, <i>J</i> = 6.8 Hz, 1H), 1.63 (q, <i>J</i> = 5.6 Hz, 1H), 1.35 (d, <i>J</i> = 24.8 Hz, 9H), 1.24 (s, 2H).</div></div><div id="sec4_6_7_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> <i>tert</i>-Butyl(1-((6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)amino)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)carbamate (<b>31c</b>)</h5><div class="NLM_p"><b>31c</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>24</b> (0.24 g, 1.0 mmol, 1.0 equiv) and 1 equiv of compound <b>30c</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 30:1 to 15:1) to afford the corresponding product as a white solid, 0.46 g, 76% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.06 (d, <i>J</i> = 46.8 Hz, 1H), 8.49–8.46 (m, 1H), 7.79–7.70 (m, 3H), 7.34–7.32 (m, 2H), 7.05 (t, <i>J</i> = 9.2 Hz, 1H), 6.57 (brs, 1H), 5.51–5.46 (m, 1H), 4.30 (s, 2H), 4.28–4.23 (m, 1H), 4.19–4.11 (m, 2H), 3.93–3.89 (m, 1H), 3.70–3.66 (m, 4H), 3.55 (brs, 2H), 3.43 (brs, 1H), 3.28 (brs, 4H), 2.48 (brs, 1H), 2.37 (t, <i>J</i> = 7.2 Hz, 1H), 2.29 (t, <i>J</i> = 7.2 Hz, 1H), 1.66 (p, <i>J</i> = 7.2 Hz, 2H), 1.55 (p, <i>J</i> = 7.2 Hz, 2H), 1.45 (s, 9H), 1.38 (p, <i>J</i> = 7.2 Hz, 2H).</div><div class="NLM_p last"><b>32</b> was prepared according to the general procedure for the deprotection of the Boc group, starting from compound <b>31</b> (0.2 mmol, 1.0 equiv) to afford the corresponding product quantitatively as a white solid, which was used directly for the next step.</div></div><div id="sec4_6_7_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(1-((4-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-4-oxobutyl)amino)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)butanamide (<b>33a</b>)</h5><div class="NLM_p last"><b>33a</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>10b</b> (40 mg, 0.1 mmol, 1.0 equiv) and 1 equiv of compound <b>32b</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 30:1 to 10:1) to afford the corresponding product as a white solid, 37 mg, 38% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 9.56 (s, 1H), 8.51 (s, 1H), 8.26 (dd, <i>J</i> = 7.6, 1.2 Hz, 1H), 8.15 (t, <i>J</i> = 7.6 Hz, 1H), 8.13 (dd, <i>J</i> = 7.2, 2.4 Hz, 1H), 8.04–7.99 (m, 1H), 7.95 (t, <i>J</i> = 8.4 Hz, 1H), 7.88 (t, <i>J</i> = 7.6 Hz, 1H), 7.85–7.77 (m, 3H), 7.46–7.42 (m, 2H), 7.36 (brs, 1H), 7.25–7.21 (m, 2H), 4.43 (p, <i>J</i> = 6.8 Hz, 1H), 4.32 (s, 2H), 4.15–4.11 (m, 4H), 3.94 (s, 3H), 3.60–3.57 (m, 4H), 3.50–3.99 (m, 4H), 3.30 (brs, 1H), 3.19 (brs, 1H), 3.13 (brs, 1H), 3.09–3.03 (m, 2H), 2.43–2.41 (m, 2H), 2.31 (t, <i>J</i> = 7.2 Hz, 1H), 2.24 (t, <i>J</i> = 7.2 Hz, 1H), 2.05 (p, <i>J</i> = 7.2 Hz, 2H), 1.61 (brs, 2H).</div></div><div id="sec4_6_7_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 6-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(1-((2-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-2-oxoethyl)amino)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)hexanamide (<b>33b</b>)</h5><div class="NLM_p last"><b>33b</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>10c</b> (43 mg, 0.1 mmol, 1.0 equiv) and 1 equiv of compound <b>32a</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 30:1 to 10:1) to afford the corresponding product as a white solid, 43 mg, 45% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 9.77 (s, 1H), 8.54 (s, 1H), 8.26 (d, <i>J</i> = 7.6 Hz, 1H), 8.14–8.10 (m, 1H), 8.02 (t, <i>J</i> = 4.8 Hz, 1H), 7.97–7.95 (m, 1H), 7.90–7.78 (m, 3H), 7.45 (t, <i>J</i> = 8.8 Hz, 2H), 7.36 (brs, 1H), 7.24 (brs, 1H), 7.21 (s, 1H), 4.56 (brs, 1H), 4.33 (s, 2H), 4.16–4.13 (m, 3H), 3.99 (brs, 1H), 3.95 (s, 3H), 3.63–3.44 (m, 6H), 3.19 (brs, 1H), 3.13 (brs, 1H), 3.05–2.99 (m, 1H), 2.75 (d, <i>J</i> = 4.4 Hz, 1H), 2.22 (t, <i>J</i> = 6.0 Hz, 2H), 1.86–1.82 (m, 2H), 1.61 (t, <i>J</i> = 6.4 Hz, 2H), 1.49 (p, <i>J</i> = 7.6 Hz, 2H).</div></div><div id="sec4_6_7_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 6-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(1-((6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)amino)-1-oxo-3-(prop-2-yn-1-yloxy)propan-2-yl)hexanamide (<b>33c</b>)</h5><div class="NLM_p last"><b>33c</b> was prepared according to the general procedure for the amide condensation, starting from compound <b>10c</b> (43 mg, 0.1 mmol, 1.0 equiv) and 1 equiv of compound <b>32c</b>. Chromatography purification used dichloromethane and methanol as elution solvents (gradient elution from 30:1 to 10:1) to afford the corresponding product as a white solid, 53 mg, 52% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.91 (d, <i>J</i> = 53.2 Hz, 1H), 9.15 (s, 1H), 8.58 (s, 1H), 8.40 (d, <i>J</i> = 7.2, 1H), 7.88–7.86 (m, 1H), 7.74–7.67 (m, 4H), 7.64–7.62 (m, 2H), 7.30–7.28 (m, 2H), 7.23–7.19 (m, 2H), 7.09–6.99 (m, 2H), 4.70–4.64 (m, 1H), 4.25 (s, 2H), 4.15–4.00 (m, 2H), 3.99 (brs, 2H), 3.94 (s, 3H), 3.82–3.65 (m, 6H), 3.51 (brs, 3H), 3.38–3.21 (m, 6H), 2.45 (s, 1H), 2.35–2.23 (m, 4H), 1.81 (t, <i>J</i> = 6.8 Hz, 2H), 1.66 (t, <i>J</i> = 6.8 Hz, 2H), 1.57–1.43 (m, 5H).</div></div></div><div id="sec4_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> General Procedure for Copper-Promoted Click Reaction to Access the Corresponding Mono and Dual PROTACs</h4><div class="NLM_p">A 5 mL reaction tube was charged with 1.0 equiv of alkyne and 1 equiv of azide. Then, 2 mL of THF was added. The reaction mixture was stirred at room temperature. One equiv of CuSO<sub>4</sub> and 2 equiv of sodium ascorbate were dissolved in 0.5 mL of water and the solution color became brown. The brown aqueous solution was added dropwise to the THF solution. Then, the resulting mixture was stirred at room temperature for 0.5–3 h. The reaction was monitored by TLC. When the reaction was completed, 10 mL of ethyl acetate and 10 mL of brine were added. The organic phase was separated and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed using a rotary evaporator. The residue was purified with column chromatography on silica gel, eluting with dichloromethane and methanol (gradient elution from 30:1 to 8:1) to afford the corresponding PROTAC products as a yellow or white solid. In the case of Mono PROTACs <b>MP-GC</b>, <b>MP-GV</b>, <b>MP-PC</b>, <b>MP-PV</b>, the click reaction was performed on a 0.05 mmol scale, while the reaction for dual PROTACs <b>DP-C 1-4</b> and <b>DP-V 1-4</b> was performed on a 0.02 mmol scale.</div><div id="sec4_6_8_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-((1-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methyl)butanamide (<b>MP-GC</b>)</h5><div class="NLM_p last">Yellow solid, 13 mg, 32% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.10 (s, 1H), 9.59 (s, 1H), 8.51 (s, 1H), 8.42 (t, <i>J</i> = 5.6 Hz, 1H), 8.13 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 7.89 (s, 1H), 7.82 (s, 1H), 7.80–7.77 (m, 1H), 7.55 (t, <i>J</i> = 8.0 Hz, 1H), 7.44 (t, <i>J</i> = 9.2 Hz, 1H), 7.20 (s, 1H), 7.07 (d, <i>J</i> = 8.8 Hz, 1H), 7.02 (d, <i>J</i> = 7.2 Hz, 1H), 6.56 (t, <i>J</i> = 6.0 Hz, 1H), 5.06 (dd, <i>J</i> = 12.8, 5.6 Hz, 1H), 4.49 (t, <i>J</i> = 4.8 Hz, 2H), 4.31 (d, <i>J</i> = 5.6 Hz, 2H), 4.15 (t, <i>J</i> = 6.0 Hz, 2H), 3.94 (s, 3H), 3.80 (t, <i>J</i> = 5.2 Hz, 2H), 3.58 (t, <i>J</i> = 5.2 Hz, 2H), 3.41 (dd, <i>J</i> = 10.8, 5.2 Hz, 2H), 2.94–2.84 (m, 1H), 2.61–2.54 (m, 2H), 2.36 (t, <i>J</i> = 7.2 Hz, 2H), 2.10–2.02 (m, 3H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>39</sub>H<sub>38</sub>ClFN<sub>10</sub>O<sub>8</sub> calcd: 829.2619, found: 829.2631.</div></div><div id="sec4_6_8_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 2-(2,6-Dioxopiperidin-3-yl)-4-((2-(2-(4-(3-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-3-oxopropyl)-1<i>H</i>-1,2,3-triazol-1-yl)ethoxy)ethyl)amino)isoindoline-1,3-dione (<b>MP-OC</b>)</h5><div class="NLM_p last">Yellow solid, 16 mg, 39% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.61 (s, 1H), 11.11 (s, 1H), 8.26 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (d, <i>J</i> = 8.0 Hz, 1H), 7.89 (td, <i>J</i> = 8.0, 1.6 Hz, 1H), 7.85–7.80 (m, 2H), 7.56 (dd, <i>J</i> = 12.8, 7.6 Hz, 1H), 7.46–7.43 (m, 1H), 7.37 (d, <i>J</i> = 6.4 Hz, 1H), 7.24 (t, <i>J</i> = 8.8 Hz, 1H), 7.11 (dd, <i>J</i> = 8.4, 5.6 Hz, 1H), 7.03 (dd, <i>J</i> = 6.8, 4.4 Hz, 1H), 6.58 (d, <i>J</i> = 4.4 Hz, 1H), 5.07 (dd, <i>J</i> = 12.8, 5.6 Hz, 1H), 4.49 (t, <i>J</i> = 4.4 Hz, 2H), 4.34 (s, 2H), 3.82 (d, <i>J</i> = 3.2 Hz, 2H), 3.62–3.61 (m, 4H), 3.52 (brs, 2H), 3.45 (brs, 2H), 3.37 (brs, 1H), 3.30 (brs, 1H), 3.16 (brs, 2H), 2.90–2.82 (m, 3H), 2.72–2.54 (m, 4H), 2.06–2.03 (m, 1H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>42</sub>H<sub>42</sub>FN<sub>10</sub>O<sub>8</sub> calcd: 833.3166, found: 833.3161.</div></div><div id="sec4_6_8_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-(4-((4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)butanamido)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>MP-GV</b>)</h5><div class="NLM_p last">White solid, 17 mg, 35% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.95 (brs, 1H), 8.67 (s, 1H), 8.61 (s, 1H), 8.04 (d, <i>J</i> = 4.4 Hz, 1H), 7.96 (brs, 1H), 7.88 (s, 1H), 7.46 (s, 1H), 7.36–7.29 (m, 5H), 7.20 (s, 1H), 7.14 (t, <i>J</i> = 8.8 Hz, 1H), 7.07 (d, <i>J</i> = 5.6 Hz, 1H), 6.53 (d, <i>J</i> = 8.0 Hz, 1H), 4.74 (t, <i>J</i> = 4.8 Hz, 1H), 4.59–4.47 (m, 4H), 4.44 (d, <i>J</i> = 8.8 Hz, 1H), 4.38–4.34 (m, 2H), 4.25–4.22 (m, 1H), 4.19–4.15 (m, 2H), 4.07 (d, <i>J</i> = 12.0 Hz, 1H), 3.97 (s, 3H), 3.62 (dd, <i>J</i> = 11.2, 3.2 Hz, 1H), 2.50 (s, 3H), 2.45 (t, <i>J</i> = 4.8 Hz, 3H), 2.22–2.01 (m, 7H), 0.93 (s, 9H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>48</sub>H<sub>56</sub>ClFN<sub>11</sub>O<sub>7</sub>S calcd: 984.3752, found: 984.3759.</div></div><div id="sec4_6_8_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-(4-(3-(4-(2-Fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-3-oxopropyl)-1<i>H</i>-1,2,3-triazol-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>MP-OV</b>)</h5><div class="NLM_p last">White solid, 14 mg, 36% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.33 (brs, 1H), 8.69 (s, 1H), 8.43 (s, 1H), 7.78–7.71 (m, 4H), 7.44–7.29 (m, 6H), 7.24–7.19 (m, 2H), 7.07–7.00 (m, 1H), 4.78–4.72 (m, 1H), 4.64–4.55 (m, 2H), 4.34–4.22 (m, 5H), 4.15 (t, <i>J</i> = 11.2 Hz, 1H), 3.70–3.67 (m, 2H), 3.48 (brs, 2H), 3.31 (brs, 1H), 3.17 (brs, 1H), 3.02 (brs, 2H), 2.76 (brs, 2H), 2.48 (s, 3H), 2.43–2.07 (m, 10H), 0.99 (s, 9H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> for C<sub>51</sub>H<sub>58</sub>FN<sub>11</sub>NaO<sub>7</sub>S calcd: 1010.4118, found: 1010.4128.</div></div><div id="sec4_6_8_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 2-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(3-(4-((1-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)phenyl)-1-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-1-oxopropan-2-yl)acetamide (<b>DP-C-1</b>)</h5><div class="NLM_p last">Yellow solid, 11 mg, 42% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.58 (s, 1H), 11.08 (s, 1H), 9.49 (s, 1H), 8.53 (s, 1H), 8.25 (d, <i>J</i> = 5.2 Hz, 1H), 8.21 (t, <i>J</i> = 6.8 Hz, 1H), 8.11–8.10 (m, 2H), 7.94 (t, <i>J</i> = 7.2 Hz, 1H), 7.87 (t, <i>J</i> = 7.2 Hz, 1H), 7.84–7.82 (m, 1H), 7.80–7.76 (m, 1H), 7.54 (t, <i>J</i> = 6.0 Hz, 1H), 7.44–7.42 (m, 2H), 7.35 (brs, 1H), 7.22 (t, <i>J</i> = 6.8 Hz, 1H), 7.13–7.07 (m, 3H), 7.01 (d, <i>J</i> = 6.0 Hz, 1H), 6.84 (t, <i>J</i> = 7.6 Hz, 2H), 6.57 (t, <i>J</i> = 4.8 Hz, 1H), 5.04–4.99 (m, 4H), 4.70 (brs, 2H), 4.54 (brs, 2H), 4.32 (brs, 2H), 4.06–4.02 (m, 1H), 3.96 (s, 3H), 3.84 (t, <i>J</i> = 4.0 Hz, 2H), 3.60 (t, <i>J</i> = 4.0 Hz, 2H), 3.51 (brs, 2H), 3.43 (q, <i>J</i> = 4.0 Hz, 4H), 3.10 (brs, 2H), 2.94–2.81 (m, 4H), 1.99–1.97 (m, 2H), 1.76–1.71 (m, 1H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>66</sub>H<sub>60</sub>ClF<sub>2</sub>N<sub>14</sub>O<sub>12</sub> calcd: 1313.4166, found:1313.4152.</div></div><div id="sec4_6_8_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(3-((1-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)-1-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-1-oxopropan-2-yl)butanamide (<b>DP-C-2</b>)</h5><div class="NLM_p last">Yellow solid, 12 mg, 48% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.59 (s, 1H), 11.09 (s, 1H), 9.52 (s, 1H), 8.50 (s, 1H), 8.32 (d, <i>J</i> = 8.0 Hz, 1H), 8.26–8.24 (m, 1H), 8.11 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 8.01–7.99 (m, 1H), 7.94–7.92 (m, 1H), 7.88 (td, <i>J</i> = 7.2, 1.2 Hz, 1H), 7.82–7.76 (m, 2H), 7.53 (t, <i>J</i> = 8.0 Hz, 1H), 7.45–7.41 (m, 2H), 7.35 (brs, 1H), 7.20 (brs, 2H), 7.08–7.02 (m, 2H), 7.01 (d, <i>J</i> = 7.2 Hz, 1H), 6.56 (brs, 1H), 5.05 (dd, <i>J</i> = 12.8, 5.2 Hz, 1H), 4.97–4.92 (m, 1H), 4.50 (brs, 4H), 4.32 (brs, 2H), 4.14–4.13 (m, 2H), 3.94 (s, 3H), 3.83 (brs, 2H), 3.58–3.42 (m, 12H), 3.12 (brs, 2H), 2.93–2.85 (m, 1H), 2.51–2.50 (m, 2H), 2.49–2.36 (m, 2H), 2.04 (brs, 2H), 1.83–1.81 (m, 1H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>62</sub>H<sub>60</sub>ClF<sub>2</sub>N<sub>14</sub>O<sub>12</sub> calcd: 1265.4166, found: 1265.4172.</div></div><div id="sec4_6_8_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(3-((1-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)-1-((4-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-4-oxobutyl)amino)-1-oxopropan-2-yl)butanamide (<b>DP-C-3</b>)</h5><div class="NLM_p last">Yellow solid, 12 mg, 45% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.59 (s, 1H), 11.10 (s, 1H), 9.56 (s, 1H), 8.50 (s, 1H), 8.25 (d, <i>J</i> = 7.6 Hz, 1H), 8.12 (dd, <i>J</i> = 6.8, 2.4 Hz, 2H), 8.00–7.76 (m, 6H), 7.53 (t, <i>J</i> = 8.0 Hz, 1H), 7.43 (t, <i>J</i> = 8.8 Hz, 2H), 7.35 (brs, 1H), 7.24–7.19 (m, 2H), 7.07 (d, <i>J</i> = 8.8 Hz, 1H), 7.01 (d, <i>J</i> = 7.2 Hz, 1H), 6.56 (t, <i>J</i> = 5.6 Hz, 1H), 5.05 (dd, <i>J</i> = 12.8, 5.2 Hz, 1H), 4.50–4.42 (m, 4H), 4.31 (d, <i>J</i> = 7.6 Hz, 2H), 4.12 (t, <i>J</i> = 5.2 Hz, 2H), 3.94 (s, 3H), 3.83 (brs, 2H), 3.58–3.57 (m, 5H), 3.49–3.42 (m, 5H), 3.29 (brs, 1H), 3.17 (brs, 1H), 3.12 (brs, 1H), 3.05–3.04 (m, 2H), 2.93–2.85 (m, 1H), 2.51–2.50 (m, 2H), 2.42 (t, <i>J</i> = 8.0 Hz, 2H), 2.28 (t, <i>J</i> = 7.2 Hz, 1H), 2.21 (t, <i>J</i> = 7.2 Hz, 1H), 2.03 (brs, 3H), 1.59 (brs, 2H), 1.23 (brs, 2H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>66</sub>H<sub>67</sub>ClF<sub>2</sub>N<sub>15</sub>O<sub>13</sub> calcd: 1350.4694, found: 1350.4681.</div></div><div id="sec4_6_8_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 6-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)-<i>N</i>-(3-((1-(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethoxy)ethyl)-1<i>H</i>-1,2,3-triazol-4-yl)methoxy)-1-((6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)amino)-1-oxopropan-2-yl)hexanamide (<b>DP-C-4</b>)</h5><div class="NLM_p last">Yellow solid, 12 mg, 41% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.06 (d, <i>J</i> = 16.08 Hz, 1H), 10.01 (s, 1H), 8.99 (s, 1H), 8.59 (s, 1H), 8.41 (d, <i>J</i> = 8.4 Hz, 1H), 7.85 (d, <i>J</i> = 6.4 Hz, 1H), 7.74–7.70 (m, 4H), 7.59–7.57 (m, 2H), 7.46 (t, <i>J</i> = 8.0 Hz, 1H), 7.33–7.28 (m, 3H), 7.08–7.04 (m, 4H), 6.86 (dd, <i>J</i> = 8.8, 4.4 Hz, 1H), 6.50 (d, <i>J</i> = 4.4 Hz, 1H), 4.94–4.89 (m, 1H), 4.54–4.49 (m, 5H), 4.25 (s, 2H), 4.02 (t, <i>J</i> = 7.2 Hz, 2H), 3.98 (s, 3H), 3.84 (brs, 3H), 3.68–3.64 (m, 5H), 3.60 (s, 4H), 3.52 (brs, 2H), 3.36 (brs, 2H), 3.28 (brs, 2H), 3.21 (brs, 3H), 2.80–2.70 (m, 2H), 2.32–2.29 (m, 3H), 2.22–2.21 (m, 1H), 1.84 (brs, 2H), 1.69 (brs, 2H), 1.57–1.43 (m, 7H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>72</sub>H<sub>79</sub>ClF<sub>2</sub>N<sub>15</sub>O<sub>14</sub> calcd: 1450.5582, found: 1450.5536.</div></div><div id="sec4_6_8_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> (2<i>S</i>,4<i>R</i>)-1-((2<i>S</i>)-2-(4-(4-((4-(2-(2-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)acetamido)-3-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-3-oxopropyl)phenoxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>DP-V-1</b>)</h5><div class="NLM_p last">White solid, 14 mg, 48% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 9.54 (s, 1H), 8.98 (s, 1H), 8.57 (t, <i>J</i> = 6.0 Hz, 1H), 8.54 (s, 1H), 8.25 (d, <i>J</i> = 7.6 Hz, 2H), 8.18 (d, <i>J</i> = 7.6 Hz, 1H), 8.11 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 8.02 (d, <i>J</i> = 9.2 Hz, 1H), 7.94 (t, <i>J</i> = 7.2 Hz, 1H), 7.89–7.78 (m, 4H), 7.47–7.37 (m, 6H), 7.26 (brs, 1H), 7.23 (t, <i>J</i> = 8.8 Hz, 1H), 7.11 (t, <i>J</i> = 8.8 Hz, 2H), 6.86 (t, <i>J</i> = 8.4 Hz, 2H), 5.14 (d, <i>J</i> = 3.2 Hz, 1H), 5.06–4.98 (m, 3H), 4.71 (brs, 2H), 4.54 (d, <i>J</i> = 9.2 Hz, 1H), 4.46–4.41 (m, 2H), 4.35–4.33 (m, 5H), 4.21 (dd, <i>J</i> = 16.0, 5.2 Hz, 1H), 3.96 (s, 3H), 3.66–3.43 (m, 7H), 3.11 (brs, 2H), 2.93–2.85 (m, 3H), 2.44 (s, 3H), 2.30–2.15 (m, 2H), 2.04–2.02 (m, 3H), 0.94 (s, 9H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>75</sub>H<sub>77</sub>ClF<sub>2</sub>N<sub>15</sub>O<sub>11</sub>S calcd: 1468.5299, found: 1468.5287.</div></div><div id="sec4_6_8_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> (2<i>S</i>,4<i>R</i>)-1-((2<i>S</i>)-2-(4-(4-((2-(4-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)butanamido)-3-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-3-oxopropoxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>DP-V-2</b>)</h5><div class="NLM_p last">White solid, 10 mg, 36% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 9.57 (s, 1H), 8.98 (s, 1H), 8.57 (t, <i>J</i> = 6.0 Hz, 1H), 8.51 (s, 1H), 8.34 (d, <i>J</i> = 8.0 Hz, 1H), 8.25 (d, <i>J</i> = 7.6 Hz, 1H), 8.11 (dd, <i>J</i> = 6.8, 2.4 Hz, 1H), 8.05 (d, <i>J</i> = 13.6 Hz, 1H), 7.99 (d, <i>J</i> = 9.2 Hz, 1H), 7.97–7.93 (m, 1H), 7.88 (td, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.84–7.76 (m, 3H), 7.46–7.35 (m, 7H), 7.21 (s, 2H), 5.13 (d, <i>J</i> = 3.6 Hz, 1H), 4.97–4.92 (m, 1H), 4.55–4.50 (m, 3H), 4.46–4.41 (m, 2H), 4.32 (brs, 5H), 4.22 (dd, <i>J</i> = 16.0, 5.6 Hz, 1H), 4.14 (d, <i>J</i> = 5.6 Hz, 2H), 3.94 (s, 3H), 3.65–3.41 (m, 10H), 3.13 (brs, 2H), 2.44 (s, 3H), 2.38 (d, <i>J</i> = 6.4 Hz, 2H), 2.28–2.15 (m, 2H), 2.03–2.01 (m, 5H), 0.92 (s, 9H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>71</sub>H<sub>77</sub>ClF<sub>2</sub>N<sub>15</sub>O<sub>11</sub>S calcd: 1420.5299, found: 1420.5281.</div></div><div id="sec4_6_8_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> (2<i>S</i>,4<i>R</i>)-1-((2<i>S</i>)-2-(4-(4-((2-(6-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)hexanamido)-3-((2-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-2-oxoethyl)amino)-3-oxopropoxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>DP-V-3</b>)</h5><div class="NLM_p last">White solid, 12 mg, 41% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 10.05 (brs, 1H), 8.98 (s, 1H), 8.59 (brs, 2H), 8.25 (d, <i>J</i> = 7.6 Hz, 1H), 8.13 (d, <i>J</i> = 7.6 Hz, 1H), 8.11–8.08 (m, 2H), 8.03–7.99 (m, 2H), 7.97–7.93 (m, 2H), 7.89 (brs, 1H), 7.83 (d, <i>J</i> = 7.6 Hz, 1H), 7.79–7.76 (m, 1H), 7.45–7.36 (m, 6H), 7.22 (s, 2H), 5.15 (brs, 1H), 4.55–4.52 (m, 4H), 4.45–4.50 (m, 2H), 4.34–4.30 (m, 5H), 4.21 (dd, <i>J</i> = 16.0, 5.6 Hz, 1H), 4.14 (t, <i>J</i> = 6.4 Hz, 2H), 3.99 (brs, 2H), 3.95 (s, 3H), 3.64–3.57 (m, 6H), 3.51–3.46 (m, 2H), 3.19 (brs, 1H), 3.13 (brs, 1H), 2.43 (s, 3H), 2.28–2.15 (m, 5H), 2.06–2.01 (m, 3H), 1.83–1.81 (m, 3H), 1.60 (brs, 2H), 1.46 (brs, 2H), 0.92 (s, 9H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>75</sub>H<sub>84</sub>ClF<sub>2</sub>N<sub>16</sub>O<sub>12</sub>S calcd: 1505.5826, found: 1505.5829.</div></div><div id="sec4_6_8_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> (2<i>S</i>,4<i>R</i>)-1-((2<i>S</i>)-2-(4-(4-((2-(6-((4-((3-Chloro-4-fluorophenyl)amino)-7-methoxyquinazolin-6-yl)oxy)hexanamido)-3-((6-(4-(2-fluoro-5-((4-oxo-3,4-dihydrophthalazin-1-yl)methyl)benzoyl)piperazin-1-yl)-6-oxohexyl)amino)-3-oxopropoxy)methyl)-1<i>H</i>-1,2,3-triazol-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>DP-V-4</b>)</h5><div class="NLM_p last">White solid, 11 mg, 35% yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 10.29 (brs, 2H), 8.98 (s, 1H), 8.65 (s, 1H), 8.59 (t, <i>J</i> = 6.0 Hz, 1H), 8.25 (d, <i>J</i> = 7.6 Hz, 1H), 8.08 (dd, <i>J</i> = 6.8, 2.0 Hz, 1H), 8.05 (brs, 1H), 8.02–7.94 (m, 4H), 7.88 (t, <i>J</i> = 7.6 Hz, 1H), 7.82 (t, <i>J</i> = 7.6 Hz, 1H), 7.78–7.76 (m, 1H), 7.49 (t, <i>J</i> = 8,8 Hz, 1H), 7.42–7.36 (m, 5H), 7.21–7.21 (m, 2H), 5.15 (brs, 1H), 4.55–4.40 (m, 5H), 4.32 (brs, 5H), 4.21 (dd, <i>J</i> = 15.6, 5.6 Hz, 1H), 4.14 (t, <i>J</i> = 5.2 Hz, 2H), 3.96 (s, 3H), 3.65–3.62 (m, 3H), 3.55–3.49 (m, 5H), 3.15 (d, <i>J</i> = 16.4 Hz, 3H), 3.02 (brs, 3H), 2.43 (s, 3H), 2.28–2.19 (m, 7H), 2.06–2.00 (m, 3H), 1.89–1.83 (m, 1H), 1.82 (brs, 2H), 1.58 (brs, 2H), 1.44–1.34 (m, 7H), 0.92 (s, 9H). HRMS (pos. ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> for C<sub>79</sub>H<sub>92</sub>ClF<sub>2</sub>N<sub>16</sub>O<sub>12</sub>S calcd: 1561.6452, found: 1561.6462.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00649" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12877" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12877" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00649?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00649</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR spectra of compounds in the text, HRMS spectra of <b>MP-GC</b>, <b>MP-OC</b>, <b>MP-GV</b>, <b>MP-OV</b>, <b>DP-C 1-4</b> and <b>DP-V 1-4</b> and HPLC for purity determination of <b>DP-C 1-4</b>, <b>DP-V 1-4</b>, <b>MP-GC</b>, <b>MP-OC</b>, <b>MP-GV</b>, and <b>MP-OV</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf">jm1c00649_si_001.pdf (3.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_002.csv">jm1c00649_si_002.csv (4.85 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00649" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lixia Chen</span> - <span class="hlFld-Affiliation affiliation">Wuya
College of Innovation, Key Laboratory of Structure-Based Drug Design
& Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2196-1428" title="Orcid link">https://orcid.org/0000-0003-2196-1428</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1f6c6665667c73675f2e292c317c7072"><span class="__cf_email__" data-cfemail="07747e7d7e646b7f473631342964686a">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yirong Zhou</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3470-4562" title="Orcid link">https://orcid.org/0000-0002-3470-4562</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#23594b4c565a4a514c4d44634b5650570d4647560d404d"><span class="__cf_email__" data-cfemail="186270776d61716a77767f58706d6b6c367d7c6d367b76">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hua Li</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span>; 
    <span class="hlFld-Affiliation affiliation">Wuya
College of Innovation, Key Laboratory of Structure-Based Drug Design
& Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-1903-836X" title="Orcid link">https://orcid.org/0000-0003-1903-836X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1a767345726f7b5a726f696e347f7e6f347974"><span class="__cf_email__" data-cfemail="6f030630071a0e2f071a1c1b410a0b1a410c01">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mengzhu Zheng</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junfeng Huo</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoxia Gu</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yali Wang</span> - <span class="hlFld-Affiliation affiliation">Wuya
College of Innovation, Key Laboratory of Structure-Based Drug Design
& Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Canrong Wu</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qingzhe Zhang</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wang Wang</span> - <span class="hlFld-Affiliation affiliation">Wuya
College of Innovation, Key Laboratory of Structure-Based Drug Design
& Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yang Liu</span> - <span class="hlFld-Affiliation affiliation">Wuya
College of Innovation, Key Laboratory of Structure-Based Drug Design
& Discovery, Ministry of Education, Shenyang Pharmaceutical University, Shenyang 110016, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Liu</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xuechen Zhou</span> - <span class="hlFld-Affiliation affiliation">Hubei
Key Laboratory of Natural Medicinal Chemistry and Resource Evaluation,
School of Pharmacy, Tongji-Rongcheng Center for Biomedicine, Tongji
Medical College, Huazhong University of
Science and Technology, Wuhan 430030, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.Z, J.H., and X.G. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3244-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i65">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80525" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80525" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (NSFC) (grant numbers 81773637, U1803122, and 81903863), the National Mega-project for Innovative Drugs (grant number 2019ZX09721001-004-007, China), the Chunhui Program-Cooperative Research Project of the Ministry of Education, the Natural Science Foundation of Hubei Province (no. 2020CFB642), the 100 Talents Program of the Hubei Provincial Government, the Liaoning Revitalization Talents Program (no. XLYC1807182), and the Liaoning Province Natural Science Foundation (no. 2020-MZLH-31). The Center of Analysis and Testing of Huazhong University of Science and Technology is gratefully acknowledged for the characterization of the new compounds. We thank Kexing Li from the School of Life Sciences, Tsinghua University, for English editing help.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3(3-dimethylpropylamine)carbodiimide</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21524" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21524" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 29 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
der Veeken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffelers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storm, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bergen En Henegouwen, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roovers, R. C.</span></span> <span> </span><span class="NLM_article-title">Crosstalk between epidermal growth factor receptor and insulin-like growth factor-1 receptor signaling: implications for cancer therapy</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">760</span>, <span class="refDoi"> DOI: 10.2174/156800909789271495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.2174%2F156800909789271495" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=19754359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlKnurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=748-760&author=J.+van%0Ader+Veekenauthor=S.+Oliveiraauthor=R.+M.+Schiffelersauthor=G.+Stormauthor=P.+M.+van+Bergen+En+Henegouwenauthor=R.+C.+Roovers&title=Crosstalk+between+epidermal+growth+factor+receptor+and+insulin-like+growth+factor-1+receptor+signaling%3A+implications+for+cancer+therapy&doi=10.2174%2F156800909789271495"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Crosstalk between epidermal growth factor receptor- and insulin-like growth factor-1 receptor signaling: implications for cancer therapy</span></div><div class="casAuthors">van der Veeken, J.; Oliveira, S.; Schiffelers, R. M.; Storm, G.; van Bergen en Henegouwen, P. M. P.; Roovers, R. C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">748-760</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Both the epidermal growth factor receptor (EGFR) and the insulin-like growth factor-1 receptor (IGF-1R) can contribute to tumor development and -progression through their effects on cell proliferation, inhibition of apoptosis, angiogenesis, anchorage-independent growth and tumor-assocd. inflammation.  EGFR-targeting monoclonal antibodies and small mol. tyrosine kinase inhibitors are currently in clin. use for the treatment of several types of cancer.  However, primary and acquired resistance to these agents often occurs and thereby limits the clin. efficacy of monospecific targeted therapy.  Results from both in vitro and in vivo studies indicate that cross-talk between EGFR and IGF-1R can lead to acquired resistance against EGFR-targeted drugs.  This review describes the interface between the EGFR and IGF-1R signaling networks and the implications of the extensive cross-talk between these two receptor systems for cancer therapy.  EGFR and IGF-1R interact on multiple levels, either through a direct assocn. between the two receptors, by mediating the availability of each others ligands, or indirectly, via common interaction partners such as G protein coupled receptors (GPCR) or downstream signaling mols.  This multi-layered cross-talk and its involvement in the induction of resistance to targeted therapies provide a clear rationale for dual targeting of EGFR and IGF-1R.  We discuss several (potential) strategies to simultaneously inhibit EGFR and IGF-1R signaling as promising novel therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_LfAONZf17Vg90H21EOLACvtfcHk0lgZOwd0_pm01w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlKnurrI&md5=ccaef4bafc2ddce6a807bf3605091ada</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F156800909789271495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156800909789271495%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BVeeken%26aufirst%3DJ.%26aulast%3DOliveira%26aufirst%3DS.%26aulast%3DSchiffelers%26aufirst%3DR.%2BM.%26aulast%3DStorm%26aufirst%3DG.%26aulast%3Dvan%2BBergen%2BEn%2BHenegouwen%26aufirst%3DP.%2BM.%26aulast%3DRoovers%26aufirst%3DR.%2BC.%26atitle%3DCrosstalk%2520between%2520epidermal%2520growth%2520factor%2520receptor%2520and%2520insulin-like%2520growth%2520factor-1%2520receptor%2520signaling%253A%2520implications%2520for%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2009%26volume%3D9%26spage%3D748%26epage%3D760%26doi%3D10.2174%2F156800909789271495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgillo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Y.</span></span> <span> </span><span class="NLM_article-title">Resistance to epidermal growth factor receptor-targeted therapy</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1016/j.drup.2005.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.drup.2005.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=16172017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Oms7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2005&pages=298-310&author=F.+Morgilloauthor=H.+Y.+Lee&title=Resistance+to+epidermal+growth+factor+receptor-targeted+therapy&doi=10.1016%2Fj.drup.2005.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to epidermal growth factor receptor-targeted therapy</span></div><div class="casAuthors">Morgillo, Floriana; Lee, Ho-Young</div><div class="citationInfo"><span class="NLM_cas:title">Drug Resistance Updates</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">298-310</span>CODEN:
                <span class="NLM_cas:coden">DRUPFW</span>;
        ISSN:<span class="NLM_cas:issn">1368-7646</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The epidermal growth factor receptor (EGFR) has been a major target of mol. anticancer therapy.  Two approaches have been developed, involving monoclonal antibodies and receptor tyrosine kinase inhibitors, and both have demonstrated benefit in clin. trials.  However, evidence of resistance to these drugs has been described.  Cellular levels of EGFR do not always correlate with response to the EGFR tyrosine kinase inhibitors, indicating acquired resistance to these drugs.  Since EGFR antagonists interfere with the activation of several intracellular pathways that control cell proliferation, survival, apoptosis, angiogenesis, invasion and metastasis, acquired resistance can occur as a result of several different mol. mechanisms: autocrine/paracrine prodn. of ligand, receptor mutation, constitutive activation of the downstream pathway and activation of alternative pathways.  We will describe here potential mechanisms that can cause resistance to EGFR-targeted drugs.  Combinations of EGFR antagonists with inhibitors targeting different signaling mechanism(s) - such as insulin-like growth factor receptor and vascular endothelial growth factor receptor - that share the same downstream mediator (e.g., phosphatidylinositol 3-kinase/Akt, mitogen-activated protein kinase), may circumvent or delay the development of resistance to EGFR antagonists resulting in enhanced antitumor activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwZiu46foBdbVg90H21EOLACvtfcHk0lip-jgV1V4tIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Oms7%252FE&md5=3979200d41479c15ddf9f87276595c8c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2005.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2005.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DMorgillo%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DH.%2BY.%26atitle%3DResistance%2520to%2520epidermal%2520growth%2520factor%2520receptor-targeted%2520therapy%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2005%26volume%3D8%26spage%3D298%26epage%3D310%26doi%3D10.1016%2Fj.drup.2005.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0lip-jgV1V4tIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. J.</span></span> <span> </span><span class="NLM_article-title">Combination Therapy of Bortezomib with Novel Targeted Agents: An Emerging Treatment Strategy</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4094</span>– <span class="NLM_lpage">4104</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-2882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1158%2F1078-0432.CCR-09-2882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20682705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVajs7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=4094-4104&author=J.+J.+Wright&title=Combination+Therapy+of+Bortezomib+with+Novel+Targeted+Agents%3A+An+Emerging+Treatment+Strategy&doi=10.1158%2F1078-0432.CCR-09-2882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Combination Therapy of Bortezomib with Novel Targeted Agents: An Emerging Treatment Strategy</span></div><div class="casAuthors">Wright, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4094-4104</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Clin. trials evaluating combinations of targeted agents with bortezomib, the first-in-class proteasome inhibitor, have been initiated, with the objective of enhancing its single agent activity in hematol. malignancies (myeloma, mantle cell lymphoma), as well as expanding its efficacy in solid tumors.  In most cases, preclin. studies have provided a supportive rationale for designing these doublet combination studies.  Novel, small mol.-targeted agents being investigated with bortezomib in clin. trials include protein deacetylase inhibitors, kinase inhibitors, farnesyltransferase inhibitors, heat-shock protein 90 inhibitors, pan-Bcl-2 family inhibitors, and other classes of targeted inhibitors.  Preliminary clin. data, available from a no. of ongoing trials, suggest that most of these combinations are well tolerated and some have promising clin. efficacy that will require subsequent confirmation.  Translational studies, conducted as part of the trials, may provide important insights into the putative mechanism of action delineated by preclin. studies of the combinations.  The emergence of novel proteasome inhibitors may also expand the opportunities for optimizing these combination therapies.  There is potential for an increasingly broad clin. trials program to investigate this therapeutic approach in a range of tumor types, as well as to consider addnl. agents in sequence or in combination.  Clin Cancer Res; 16(16); 4094-104.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80JIFlg1k7bVg90H21EOLACvtfcHk0lip-jgV1V4tIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVajs7zK&md5=70350d7044ef8f26fe4867d353d682b7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-2882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-2882%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DJ.%2BJ.%26atitle%3DCombination%2520Therapy%2520of%2520Bortezomib%2520with%2520Novel%2520Targeted%2520Agents%253A%2520An%2520Emerging%2520Treatment%2520Strategy%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D4094%26epage%3D4104%26doi%3D10.1158%2F1078-0432.CCR-09-2882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kontermann, R. E.</span></span> <span> </span><span class="NLM_article-title">Dual targeting strategies with bispecific antibodies</span>. <i>mAbs</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.4161/mabs.4.2.19000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.4161%2Fmabs.4.2.19000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=22453100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC38rgtVCisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=182-197&author=R.+E.+Kontermann&title=Dual+targeting+strategies+with+bispecific+antibodies&doi=10.4161%2Fmabs.4.2.19000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting strategies with bispecific antibodies</span></div><div class="casAuthors">Kontermann Roland E</div><div class="citationInfo"><span class="NLM_cas:title">mAbs</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Monoclonal antibodies are widely used for the treatment of cancer, inflammatory and infectious diseases and other disorders.  Most of the marketed antibodies are monospecific and therefore capable of interacting and interfering with a single target.  However, complex diseases are often multifactorial in nature, and involve redundant or synergistic action of disease mediators or upregulation of different receptors, including crosstalk between their signaling networks.  Consequently, blockade of multiple, different pathological factors and pathways may result in improved therapeutic efficacy.  This result can be achieved by combining different drugs, or use of the dual targeting strategies applying bispecific antibodies that have emerged as an alternative to combination therapy.  This review discusses the various dual targeting strategies for which bispecific antibodies have been developed and provides an overview of the established bispecific antibody formats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTgBg_RJQ0LBrBItaL1WaVAfW6udTcc2eYwS6AN1bO_Vbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rgtVCisA%253D%253D&md5=b6e2812d15cfd6e3649053796b0ecd7b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2Fmabs.4.2.19000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fmabs.4.2.19000%26sid%3Dliteratum%253Aachs%26aulast%3DKontermann%26aufirst%3DR.%2BE.%26atitle%3DDual%2520targeting%2520strategies%2520with%2520bispecific%2520antibodies%26jtitle%3DmAbs%26date%3D2012%26volume%3D4%26spage%3D182%26epage%3D197%26doi%3D10.4161%2Fmabs.4.2.19000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodfellow, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean-Claude, B. J.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of a Trifunctional Molecular System “Programmed” to Block Epidermal Growth Factor Receptor Tyrosine Kinase, Induce High Levels of DNA Damage, and Inhibit the DNA Repair Enzyme (Poly(ADP-ribose) Polymerase) in Prostate Cancer Cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5752</span>– <span class="NLM_lpage">5762</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVCqtLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5752-5762&author=J.+Schmittauthor=S.+Huangauthor=E.+Goodfellowauthor=C.+Williamsauthor=B.+J.+Jean-Claude&title=Design+and+Synthesis+of+a+Trifunctional+Molecular+System+%E2%80%9CProgrammed%E2%80%9D+to+Block+Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase%2C+Induce+High+Levels+of+DNA+Damage%2C+and+Inhibit+the+DNA+Repair+Enzyme+%28Poly%28ADP-ribose%29+Polymerase%29+in+Prostate+Cancer+Cells&doi=10.1021%2Facs.jmedchem.9b02008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Trifunctional Molecular System "Programmed" to Block Epidermal Growth Factor Receptor Tyrosine Kinase, Induce High Levels of DNA Damage, and Inhibit the DNA Repair Enzyme (Poly(ADP-ribose) Polymerase) in Prostate Cancer Cells</span></div><div class="casAuthors">Schmitt, Julie; Huang, Shanlong; Goodfellow, Elliot; Williams, Christopher; Jean-Claude, Bertrand J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5752-5762</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resistance to chemotherapy in advanced cancers can be mediated by different factors such as epidermal growth factor receptor (EGFR) overexpression and DNA repair enzymes.  Therefore, current stds. of care usually involve combinations of multiple treatments.  Here, to reduce the adverse effects of multiple drug combinations and improve outcome, we proposed a single drug approach to block multiple overlapping effects that characterize chemoresistance.  Thus, we designed a new linker that allows assembly of multiple functions (e.g., inhibition of EGFR phosphorylation, induction of DNA lesions, and blockade of their repair) into a single mol.  This led to the successful synthesis of a novel and potent combi-mol. JS230(I).  Here, we demonstrated that in resistant prostate cancer cells overexpressing EGFR, it was capable of (a) inhibiting EGFR in a dose-dependent manner, (b) damaging DNA, and (c) sustaining the damage by inhibiting the DNA repair protein poly(ADP-ribose) polymerase (PARP).  The triple mechanism of action of JS230 cumulated into growth inhibitory potency superior to that of classical two- or three-drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8CLi7wKtzibVg90H21EOLACvtfcHk0ljRbJkHKEI1zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVCqtLnN&md5=97f4b2f923a8af8cb63bd1f8ce848cb5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02008%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DGoodfellow%26aufirst%3DE.%26aulast%3DWilliams%26aufirst%3DC.%26aulast%3DJean-Claude%26aufirst%3DB.%2BJ.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520a%2520Trifunctional%2520Molecular%2520System%2520%25E2%2580%259CProgrammed%25E2%2580%259D%2520to%2520Block%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%252C%2520Induce%2520High%2520Levels%2520of%2520DNA%2520Damage%252C%2520and%2520Inhibit%2520the%2520DNA%2520Repair%2520Enzyme%2520%2528Poly%2528ADP-ribose%2529%2520Polymerase%2529%2520in%2520Prostate%2520Cancer%2520Cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5752%26epage%3D5762%26doi%3D10.1021%2Facs.jmedchem.9b02008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Desidero, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basolo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerbel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danesi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocci, G.</span></span> <span> </span><span class="NLM_article-title">Antiproliferative and ProapoptoticActivity of Sunitinib on Endothelial and Anaplastic Thyroid Cancer Cells via Inhibition of Akt and ERK1/2 Phosphorylation and by Down-Regulation of Cyclin-D1</span>. <i>J. Clin. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">E1465</span>– <span class="NLM_lpage">E1473</span>, <span class="refDoi"> DOI: 10.1210/jc.2013-1364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1210%2Fjc.2013-1364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=23969186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsV2isbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2013&pages=E1465-E1473&author=T.+Di%0ADesideroauthor=A.+Fioravantiauthor=P.+Orlandiauthor=B.+Canuauthor=R.+Gianniniauthor=N.+Borrelliauthor=S.+Manauthor=P.+Xuauthor=G.+Fontaniniauthor=F.+Basoloauthor=R.+S.+Kerbelauthor=G.+Franciaauthor=R.+Danesiauthor=G.+Bocci&title=Antiproliferative+and+ProapoptoticActivity+of+Sunitinib+on+Endothelial+and+Anaplastic+Thyroid+Cancer+Cells+via+Inhibition+of+Akt+and+ERK1%2F2+Phosphorylation+and+by+Down-Regulation+of+Cyclin-D1&doi=10.1210%2Fjc.2013-1364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Antiproliferative and proapoptotic activity of sunitinib on endothelial and anaplastic thyroid cancer cells via inhibition of Akt and ERK1/2 phosphorylation and by down-regulation of cyclin-D1</span></div><div class="casAuthors">Di Desidero, Teresa; Fioravanti, Anna; Orlandi, Paola; Canu, Bastianina; Giannini, Riccardo; Borrelli, Nicla; Man, Shan; Xu, Ping; Fontanini, Gabriella; Basolo, Fulvio; Kerbel, Robert S.; Francia, Giulio; Danesi, Romano; Bocci, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">E1465-E1473</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Context: Recent exptl. evidence suggests a rationale for the use of multitarget tyrosine kinase inhibitors for the treatment of thyroid cancers.  Sunitinib showed promising preliminary results against anaplastic thyroid cancer (ATC), and it has been used for some patients who are ineligible for clin. trials.  Objectives: The aims of this study were to investigate the in vitro and in vivo activity of sunitinib on ATC and on microvascular endothelial cells and the mol. mechanism for the obsd. sunitinib activity.  Methods: Proliferation and apoptotic assays were performed on human dermal microvascular endothelial and on BRAF- or H-ras-mutated ATC cells (8305C and FB3, resp.) after in vitro exposure to sunitinib for 72 h.  Vascular endothelial growth factor receptor-2, epithelial growth factor receptor, ERK1/2, and Akt phosphorylation was quantified by ELISA and Western blot.  Cyclin-D1 mRNA expression was evaluated by real-time PCR, and cyclin-D1 intracellular concns. were measured by ELISA.  8305C tumor xenografts in nude mice were treated with sunitinib at 50 mg/kg/d (i.p.).  Results: Antiproliferative and proapoptotic activity of sunitinib was obsd. in both endothelial and ATC cells.  Phospho-vascular endothelial growth factor receptor-2 levels significantly decreased after sunitinib treatment in activated endothelial cells.  Phospho-epidermal growth factor receptor, ERK1/2, and Akt phosphorylation was significantly inhibited by sunitinib treatment in endothelial and cancer cells, and cyclin-D1 mRNA and protein expression was inhibited.  Sunitinib administration in vivo caused significant inhibition of tumor growth (P < .05).  Conclusions: Sunitinib is active in vitro and in vivo against activated endothelial and ATC cells via the inhibition of Akt and ERK1/2 phosphorylation and through the down-regulation of cyclin-D1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVHmdomTTpebVg90H21EOLACvtfcHk0ljRbJkHKEI1zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsV2isbnN&md5=32acb61c45541dc9d721f0429861d6dd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1210%2Fjc.2013-1364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2013-1364%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BDesidero%26aufirst%3DT.%26aulast%3DFioravanti%26aufirst%3DA.%26aulast%3DOrlandi%26aufirst%3DP.%26aulast%3DCanu%26aufirst%3DB.%26aulast%3DGiannini%26aufirst%3DR.%26aulast%3DBorrelli%26aufirst%3DN.%26aulast%3DMan%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DFontanini%26aufirst%3DG.%26aulast%3DBasolo%26aufirst%3DF.%26aulast%3DKerbel%26aufirst%3DR.%2BS.%26aulast%3DFrancia%26aufirst%3DG.%26aulast%3DDanesi%26aufirst%3DR.%26aulast%3DBocci%26aufirst%3DG.%26atitle%3DAntiproliferative%2520and%2520ProapoptoticActivity%2520of%2520Sunitinib%2520on%2520Endothelial%2520and%2520Anaplastic%2520Thyroid%2520Cancer%2520Cells%2520via%2520Inhibition%2520of%2520Akt%2520and%2520ERK1%252F2%2520Phosphorylation%2520and%2520by%2520Down-Regulation%2520of%2520Cyclin-D1%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2013%26volume%3D98%26spage%3DE1465%26epage%3DE1473%26doi%3D10.1210%2Fjc.2013-1364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhxy60Smc1juA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lhxy60Smc1juA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussenzweig, A.</span></span> <span> </span><span class="NLM_article-title">The multifaceted roles of PARP1 in DNA repair and chromatin remodelling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1038/nrm.2017.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrm.2017.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=28676700" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=610-621&author=A.+R.+Chaudhuriauthor=A.+Nussenzweig&title=The+multifaceted+roles+of+PARP1+in+DNA+repair+and+chromatin+remodelling&doi=10.1038%2Fnrm.2017.53"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2017.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2017.53%26sid%3Dliteratum%253Aachs%26aulast%3DChaudhuri%26aufirst%3DA.%2BR.%26aulast%3DNussenzweig%26aufirst%3DA.%26atitle%3DThe%2520multifaceted%2520roles%2520of%2520PARP1%2520in%2520DNA%2520repair%2520and%2520chromatin%2520remodelling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D610%26epage%3D621%26doi%3D10.1038%2Fnrm.2017.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, P.</span></span> <span> </span><span class="NLM_article-title">Insights into the biogenesis, function, and regulation of ADP-ribosylation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">236</span>– <span class="NLM_lpage">243</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnchembio.2568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=29443986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislSntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=236-243&author=M.+S.+Cohenauthor=P.+Chang&title=Insights+into+the+biogenesis%2C+function%2C+and+regulation+of+ADP-ribosylation&doi=10.1038%2Fnchembio.2568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the biogenesis, function, and regulation of ADP-ribosylation</span></div><div class="casAuthors">Cohen, Michael S.; Chang, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">236-243</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  ADP-ribosylation, the transfer of ADP-ribose (ADPr) from NAD onto target mols., is catalyzed by members of the ADP-ribosyltransferase (ART) superfamily of proteins, found in all kingdoms of life.  The modification of amino acids in protein targets by ADPr regulates crit. cellular pathways in eukaryotes and underlies the pathogenicity of certain bacteria.  Several members of the ART superfamily are highly relevant for disease; these include the poly(ADP-ribose) polymerases (PARPs), recently shown to be important cancer targets, and the bacterial toxins, diphtheria toxin and cholera toxin, long known to be responsible for the symptoms of diphtheria and cholera that result in morbidity.  Here, we discuss the functions of amino acid ADPr modifications and the ART proteins that make them, the nature of the chem. linkage between ADPr and its targets and how this impacts function and stability, and the way that ARTs select specific amino acids in targets to modify.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTovOQejfF27Vg90H21EOLACvtfcHk0lhxy60Smc1juA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislSntLg%253D&md5=f92bee47c944be2a5deb3e7440644ad3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2568%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DM.%2BS.%26aulast%3DChang%26aufirst%3DP.%26atitle%3DInsights%2520into%2520the%2520biogenesis%252C%2520function%252C%2520and%2520regulation%2520of%2520ADP-ribosylation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D236%26epage%3D243%26doi%3D10.1038%2Fnchembio.2568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeaupin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapuis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Augustin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkovics, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biertumpfel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timinszky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, S.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose)-dependent chromatin unfolding facilitates the association of DNA-binding proteins with DNA at sites of damage</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">11250</span>– <span class="NLM_lpage">11267</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1093%2Fnar%2Fgkz820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=31566235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=11250-11267&author=R.+Smithauthor=T.+Lebeaupinauthor=S.+Juhaszauthor=C.+Chapuisauthor=O.+D%E2%80%99Augustinauthor=S.+Dutertreauthor=P.+Burkovicsauthor=C.+Biertumpfelauthor=G.+Timinszkyauthor=S.+Huet&title=Poly%28ADP-ribose%29-dependent+chromatin+unfolding+facilitates+the+association+of+DNA-binding+proteins+with+DNA+at+sites+of+damage&doi=10.1093%2Fnar%2Fgkz820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose)-dependent chromatin unfolding facilitates the association of DNA-binding proteins with DNA at sites of damage</span></div><div class="casAuthors">Smith, Rebecca; Lebeaupin, Theo; Juhasz, Szilvia; Chapuis, Catherine; D'Augustin, Ostiane; Dutertre, Stephanie; Burkovics, Peter; Biertumpfel, Christian; Timinszky, Gyula; Huet, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11250-11267</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The addn. of poly(ADP-ribose) (PAR) chains along the chromatin fiber due to PARP1 activity regulates the recruitment of multiple factors to sites of DNA damage.  In this manuscript, we investigated how, besides direct binding to PAR, early chromatin unfolding events controlled by PAR signaling contribute to recruitment to DNA lesions.  We obsd. that different DNA-binding, but not histone-binding, domains accumulate at damaged chromatin in a PAR-dependent manner, and that this recruitment correlates with their affinity for DNA.  Our findings indicate that this recruitment is promoted by early PAR-dependent chromatin remodeling rather than direct interaction with PAR.  Moreover, recruitment is not the consequence of reduced mol. crowding at unfolded damaged chromatin but instead originates from facilitated binding to more exposed DNA.  These findings are further substantiated by the observation that PAR-dependent chromatin remodeling at DNA lesions underlies increased DNAse hypersensitivity.  Finally, the relevance of this new mode of PAR-dependent recruitment to DNA lesions is demonstrated by the observation that reducing the affinity for DNA of both CHD4 and HP1a, two proteins shown to be involved in the DNA-damage response, strongly impairs their recruitment to DNA lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp08SUlzHmXErVg90H21EOLACvtfcHk0lhxy60Smc1juA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsrzO&md5=3c3856829c1453fc62265bf553415aa6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz820%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DLebeaupin%26aufirst%3DT.%26aulast%3DJuhasz%26aufirst%3DS.%26aulast%3DChapuis%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Augustin%26aufirst%3DO.%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DBurkovics%26aufirst%3DP.%26aulast%3DBiertumpfel%26aufirst%3DC.%26aulast%3DTiminszky%26aufirst%3DG.%26aulast%3DHuet%26aufirst%3DS.%26atitle%3DPoly%2528ADP-ribose%2529-dependent%2520chromatin%2520unfolding%2520facilitates%2520the%2520association%2520of%2520DNA-binding%2520proteins%2520with%2520DNA%2520at%2520sites%2520of%2520damage%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3D11250%26epage%3D11267%26doi%3D10.1093%2Fnar%2Fgkz820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yacoub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinstry, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagan, M. P.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor and ionizing radiation upregulate the DNA repair genes XRCC1 and ERCC1 in DU145 and LNCaP prostate carcinoma through MAPK signaling</span>. <i>Radiat. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1667/0033-7587(2003)159[0439:EGFAIR]2.0.CO;2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=12643788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVersL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2003&pages=439-452&author=A.+Yacoubauthor=R.+McKinstryauthor=D.+Hinmanauthor=T.+Chungauthor=P.+Dentauthor=M.+P.+Hagan&title=Epidermal+growth+factor+and+ionizing+radiation+upregulate+the+DNA+repair+genes+XRCC1+and+ERCC1+in+DU145+and+LNCaP+prostate+carcinoma+through+MAPK+signaling&doi=10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor and ionizing radiation up-regulate the DNA repair genes XRCC1 and ERCC1 in DU145 and LNCaP prostate carcinoma through MAPK signaling</span></div><div class="casAuthors">Yacoub, Adly; McKinstry, Robert; Hinman, Darin; Chung, Theodore; Dent, Paul; Hagan, Michael P.</div><div class="citationInfo"><span class="NLM_cas:title">Radiation Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-452</span>CODEN:
                <span class="NLM_cas:coden">RAREAE</span>;
        ISSN:<span class="NLM_cas:issn">0033-7587</span>.
    
            (<span class="NLM_cas:orgname">Radiation Research Society</span>)
        </div><div class="casAbstract">This work examd. the importance of radiation-induced and ligand-induced EGFR-ERK signaling for the regulation of DNA repair proteins XRCC1 and ERCC1 in prostate carcinoma cells, DU145 (TP53mut), displaying EGFR-TGFA-dependent autocrine growth and high MAPK (ERK1/2) activity, and LNCaP (TP53wt) cells expressing low constitutive levels of ERK1/2 activity.  Using quant. RT-PCR and Western analyses, we detd. that ionizing radiation activated the DNA repair genes XRCC1 and ERCC1 in an ERK1/2-dependent fashion for each cell line.  After irradn., a rapid increase followed by a decrease in ERK1/2 activity preceded the increase in XRCC1/ERCC1 expression in DU145 cells, while only the rapid decrease in ERK1/2 preceded the increase in XRCC1/ERCC1 expression in LNCaP cells.  Administration of EGF, however, markedly increased the up-regulation of phospho-ERK, ERCC1 and XRCC1 in both cell lines.  Although the EGFR inhibitor tyrphostin (AG-1478) and the MEK inhibitor PD90859 both attenuated EGF-induced levels of the ERCC1 and XRCC1 protein, PD98059 blocked the induction of ERCC1 and XRCC1 by radiation more effectively in both cell lines.  Inhibition of ERK at a level that reduced the up-regulation of DNA repair led to the persistence of apurinic/apyrimidinic (AP) sites of DNA damage and increased cell killing.  Taken together, these data imply a complex control of DNA repair activation that may be more generally dependent on MAPK (ERK1/2) signaling than was previously noted.  These data provide novel insights into the capacity of the EGFR-ERK signaling to modulate DNA repair in cancer cells and into the functional significance of this signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH8DufM3lxR7Vg90H21EOLACvtfcHk0lgGEtHz6dTneQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVersL4%253D&md5=0042f1fce51c1c8ea88109cd7bd815de</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1667%252F0033-7587%25282003%2529159%255B0439%253AEGFAIR%255D2.0.CO%253B2%26sid%3Dliteratum%253Aachs%26aulast%3DYacoub%26aufirst%3DA.%26aulast%3DMcKinstry%26aufirst%3DR.%26aulast%3DHinman%26aufirst%3DD.%26aulast%3DChung%26aufirst%3DT.%26aulast%3DDent%26aufirst%3DP.%26aulast%3DHagan%26aufirst%3DM.%2BP.%26atitle%3DEpidermal%2520growth%2520factor%2520and%2520ionizing%2520radiation%2520upregulate%2520the%2520DNA%2520repair%2520genes%2520XRCC1%2520and%2520ERCC1%2520in%2520DU145%2520and%2520LNCaP%2520prostate%2520carcinoma%2520through%2520MAPK%2520signaling%26jtitle%3DRadiat.%2520Res.%26date%3D2003%26volume%3D159%26spage%3D439%26epage%3D452%26doi%3D10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chmielecki, J.</span></span> <span> </span><span class="NLM_article-title">Rational, biologically based treatment of EGFR-mutant non-small-cell lung cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">774</span>, <span class="refDoi"> DOI: 10.1038/nrc2947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrc2947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20966921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWlsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=760-774&author=W.+Paoauthor=J.+Chmielecki&title=Rational%2C+biologically+based+treatment+of+EGFR-mutant+non-small-cell+lung+cancer&doi=10.1038%2Fnrc2947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational, biologically based treatment of EGFR-mutant non-small-cell lung cancer</span></div><div class="casAuthors">Pao, William; Chmielecki, Juliann</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">760-774</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epidermal growth factor receptor (EGFR)-mutant non-small-cell lung cancer (NSCLC) was first recognized in 2004 as a distinct, clin. relevant mol. subset of lung cancer.  The disease has been the subject of intensive research at both the basic scientific and clin. levels, becoming a paradigm for how to understand and treat oncogene-driven carcinomas.  Although patients with EGFR-mutant tumors have increased sensitivity to tyrosine kinase inhibitors (TKIs), primary and acquired resistance to these agents remains a major clin. problem.  This Review summarizes recent developments aimed at treating and ultimately curing the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_b4ITVJ2ZnrVg90H21EOLACvtfcHk0lgGEtHz6dTneQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWlsrrN&md5=448b40d66a0a5f171106baa1dc709c29</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrc2947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2947%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DChmielecki%26aufirst%3DJ.%26atitle%3DRational%252C%2520biologically%2520based%2520treatment%2520of%2520EGFR-mutant%2520non-small-cell%2520lung%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D760%26epage%3D774%26doi%3D10.1038%2Fnrc2947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">1500</span>, <span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-1500&author=J.+G.+Paezauthor=P.+A.+Janneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0lgGEtHz6dTneQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D1500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsudomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongprasert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marschner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralla, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebski, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. C.</span></span> <span> </span><span class="NLM_article-title">Gefitinib or Erlotinib vs Chemotherapy for EGFR Mutation-Positive Lung Cancer: Individual Patient Data Meta-Analysis of Overall Survival</span>. <i>J. Natl. Cancer. Inst.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>109</i></span>, <span class="NLM_elocation-id">djw279</span> <span class="refDoi"> DOI: 10.1093/jnci/djw279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1093%2Fjnci%2Fdjw279" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2017&author=C.+K.+Leeauthor=L.+Daviesauthor=Y.+L.+Wuauthor=T.+Mitsudomiauthor=A.+Inoueauthor=R.+Rosellauthor=C.+Zhouauthor=K.+Nakagawaauthor=S.+Thongprasertauthor=M.+Fukuokaauthor=S.+Lordauthor=I.+Marschnerauthor=Y.+K.+Tuauthor=R.+J.+Grallaauthor=V.+Gebskiauthor=T.+Mokauthor=J.+C.+Yang&title=Gefitinib+or+Erlotinib+vs+Chemotherapy+for+EGFR+Mutation-Positive+Lung+Cancer%3A+Individual+Patient+Data+Meta-Analysis+of+Overall+Survival&doi=10.1093%2Fjnci%2Fdjw279"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjw279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjw279%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BK.%26aulast%3DDavies%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%2BL.%26aulast%3DMitsudomi%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DRosell%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DNakagawa%26aufirst%3DK.%26aulast%3DThongprasert%26aufirst%3DS.%26aulast%3DFukuoka%26aufirst%3DM.%26aulast%3DLord%26aufirst%3DS.%26aulast%3DMarschner%26aufirst%3DI.%26aulast%3DTu%26aufirst%3DY.%2BK.%26aulast%3DGralla%26aufirst%3DR.%2BJ.%26aulast%3DGebski%26aufirst%3DV.%26aulast%3DMok%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26atitle%3DGefitinib%2520or%2520Erlotinib%2520vs%2520Chemotherapy%2520for%2520EGFR%2520Mutation-Positive%2520Lung%2520Cancer%253A%2520Individual%2520Patient%2520Data%2520Meta-Analysis%2520of%2520Overall%2520Survival%26jtitle%3DJ.%2520Natl.%2520Cancer.%2520Inst.%26date%3D2017%26volume%3D109%26doi%3D10.1093%2Fjnci%2Fdjw279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfäffle, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheorghiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraiolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greninger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequist, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willers, H.</span></span> <span> </span><span class="NLM_article-title">EGFR-activating mutations correlate with a Fanconi anemia-like cellular phenotype that includes PARP inhibitor sensitivity</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6254</span>– <span class="NLM_lpage">6263</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1158%2F0008-5472.CAN-13-0044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=23966292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC3sbhvVOltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6254-6263&author=H.+N.+Pf%C3%A4ffleauthor=M.+Wangauthor=L.+Gheorghiuauthor=N.+Ferraioloauthor=P.+Greningerauthor=K.+Borgmannauthor=J.+Settlemanauthor=C.+H.+Benesauthor=L.+V.+Sequistauthor=L.+Zouauthor=H.+Willers&title=EGFR-activating+mutations+correlate+with+a+Fanconi+anemia-like+cellular+phenotype+that+includes+PARP+inhibitor+sensitivity&doi=10.1158%2F0008-5472.CAN-13-0044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR-activating mutations correlate with a Fanconi anemia-like cellular phenotype that includes PARP inhibitor sensitivity</span></div><div class="casAuthors">Pfaffle Heike N; Wang Meng; Gheorghiu Liliana; Ferraiolo Natalie; Greninger Patricia; Borgmann Kerstin; Settleman Jeffrey; Benes Cyril H; Sequist Lecia V; Zou Lee; Willers Henning</div><div class="citationInfo"><span class="NLM_cas:title">Cancer research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6254-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In patients with lung cancer whose tumors harbor activating mutations in the EGF receptor (EGFR), increased responses to platinum-based chemotherapies are seen compared with wild-type cancers.  However, the mechanisms underlying this association have remained elusive.  Here, we describe a cellular phenotype of cross-linker sensitivity in a subset of EGFR-mutant lung cancer cell lines that is reminiscent of the defects seen in cells impaired in the Fanconi anemia pathway, including a pronounced G2-M cell-cycle arrest and chromosomal radial formation.  We identified a defect downstream of FANCD2 at the level of recruitment of FAN1 nuclease and DNA interstrand cross-link (ICL) unhooking.  The effect of EGFR mutation was epistatic with FANCD2.  Consistent with the known role of FANCD2 in promoting RAD51 foci formation and homologous recombination repair (HRR), EGFR-mutant cells also exhibited an impaired RAD51 foci response to ICLs, but not to DNA double-strand breaks.  EGFR kinase inhibition affected RAD51 foci formation neither in EGFR-mutant nor wild-type cells.  In contrast, EGFR depletion or overexpression of mutant EGFR in wild-type cells suppressed RAD51 foci, suggesting an EGFR kinase-independent regulation of DNA repair.  Interestingly, EGFR-mutant cells treated with the PARP inhibitor olaparib also displayed decreased FAN1 foci induction, coupled with a putative block in a late HRR step.  As a result, EGFR-mutant lung cancer cells exhibited olaparib sensitivity in vitro and in vivo.  Our findings provide insight into the mechanisms of cisplatin and PARP inhibitor sensitivity of EGFR-mutant cells, yielding potential therapeutic opportunities for further treatment individualization in this genetically defined subset of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTeLzyiu8fY1WBAiYWy6n3SfW6udTcc2ebXtDYQMQ7oFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbhvVOltg%253D%253D&md5=a2c3fb53e1cc765302ffd6cd56e18b08</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0044%26sid%3Dliteratum%253Aachs%26aulast%3DPf%25C3%25A4ffle%26aufirst%3DH.%2BN.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DGheorghiu%26aufirst%3DL.%26aulast%3DFerraiolo%26aufirst%3DN.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DBorgmann%26aufirst%3DK.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DWillers%26aufirst%3DH.%26atitle%3DEGFR-activating%2520mutations%2520correlate%2520with%2520a%2520Fanconi%2520anemia-like%2520cellular%2520phenotype%2520that%2520includes%2520PARP%2520inhibitor%2520sensitivity%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6254%26epage%3D6263%26doi%3D10.1158%2F0008-5472.CAN-13-0044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scagliotti, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulshine, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curran, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz-Ares, L.</span></span> <span> </span><span class="NLM_article-title">Lung cancer: current therapies and new targeted treatments</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)30958-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2FS0140-6736%2816%2930958-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=27574741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=299-311&author=F.+R.+Hirschauthor=G.+V.+Scagliottiauthor=J.+L.+Mulshineauthor=R.+Kwonauthor=W.+J.+Curranauthor=Y.+L.+Wuauthor=L.+Paz-Ares&title=Lung+cancer%3A+current+therapies+and+new+targeted+treatments&doi=10.1016%2FS0140-6736%2816%2930958-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Lung cancer: current therapies and new targeted treatments</span></div><div class="casAuthors">Hirsch, Fred R.; Scagliotti, Giorgio V.; Mulshine, James L.; Kwon, Regina; Curran, Walter J., Jr.; Wu, Yi-Long; Paz-Ares, Luis</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10066</span>),
    <span class="NLM_cas:pages">299-311</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is the most frequent cause of cancer-related deaths worldwide.  Every year, 1·8 million people are diagnosed with lung cancer, and 1·6 million people die as a result of the disease. 5-yr survival rates vary from 4-17% depending on stage and regional differences.  In this Seminar, we discuss existing treatment for patients with lung cancer and the promise of precision medicine, with special emphasis on new targeted therapies.  Some subgroups, eg-patients with poor performance status and elderly patients-are not specifically addressed, because these groups require special treatment considerations and no frameworks have been established in terms of new targeted therapies.  We discuss prevention and early detection of lung cancer with an emphasis on lung cancer screening.  Although we acknowledge the importance of smoking prevention and cessation, this is a large topic beyond the scope of this Seminar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_NrglINv67Vg90H21EOLACvtfcHk0liYXY2KaH1R9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP&md5=36ace1a455a803dc6e7af83dbffb2e34</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2930958-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252930958-8%26sid%3Dliteratum%253Aachs%26aulast%3DHirsch%26aufirst%3DF.%2BR.%26aulast%3DScagliotti%26aufirst%3DG.%2BV.%26aulast%3DMulshine%26aufirst%3DJ.%2BL.%26aulast%3DKwon%26aufirst%3DR.%26aulast%3DCurran%26aufirst%3DW.%2BJ.%26aulast%3DWu%26aufirst%3DY.%2BL.%26aulast%3DPaz-Ares%26aufirst%3DL.%26atitle%3DLung%2520cancer%253A%2520current%2520therapies%2520and%2520new%2520targeted%2520treatments%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D299%26epage%3D311%26doi%3D10.1016%2FS0140-6736%2816%2930958-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span> <span> </span><span class="NLM_article-title">Gefitinib–a novel targeted approach to treating cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">965</span>, <span class="refDoi"> DOI: 10.1038/nrc1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrc1506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=15573117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=956-965&author=R.+S.+Herbstauthor=M.+Fukuokaauthor=J.+Baselga&title=Gefitinib%E2%80%93a+novel+targeted+approach+to+treating+cancer&doi=10.1038%2Fnrc1506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Gefitinib - a novel targeted approach to treating cancer</span></div><div class="casAuthors">Herbst, Roy S.; Fukuoka, Masahiro; Baselga, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">956-965</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Twenty years after the epidermal growth factor receptor (EGFR) was identified as a potential anticancer target, the EGFR inhibitor gefitinib (Iressa; AstraZeneca) has been approved for the treatment of patients with advanced non-small-cell lung cancer in many countries.  Studies have indicated its potential for treating patients with other types of solid tumors.  Investigation of gefitinib has not only increased our knowledge about the biol. of EGFR signaling, but is contributing to our evolving understanding of which tumors are EGFR dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8V2qzLg_-rrVg90H21EOLACvtfcHk0lgmxjXYCH8o-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ&md5=03a6c08b947bc3e0ca59e1b92fa8d214</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrc1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1506%26sid%3Dliteratum%253Aachs%26aulast%3DHerbst%26aufirst%3DR.%2BS.%26aulast%3DFukuoka%26aufirst%3DM.%26aulast%3DBaselga%26aufirst%3DJ.%26atitle%3DGefitinib%25E2%2580%2593a%2520novel%2520targeted%2520approach%2520to%2520treating%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D956%26epage%3D965%26doi%3D10.1038%2Fnrc1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappuzzo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciuleanu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stelmakh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicenas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczesna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteban, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugger, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melezínek, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klingelschmitt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klughammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span> <span> </span><span class="NLM_article-title">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">529</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(10)70112-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2FS1470-2045%2810%2970112-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20493771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=521-529&author=F.+Cappuzzoauthor=T.+Ciuleanuauthor=L.+Stelmakhauthor=S.+Cicenasauthor=A.+Szczesnaauthor=E.+Juhaszauthor=E.+Estebanauthor=O.+Molinierauthor=W.+Bruggerauthor=I.+Melez%C3%ADnekauthor=G.+Klingelschmittauthor=B.+Klughammerauthor=G.+Giaccone&title=Erlotinib+as+maintenance+treatment+in+advanced+non-small-cell+lung+cancer%3A+a+multicentre%2C+randomised%2C+placebo-controlled+phase+3+study&doi=10.1016%2FS1470-2045%2810%2970112-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span></div><div class="casAuthors">Cappuzzo, Federico; Ciuleanu, Tudor; Stelmakh, Lilia; Cicenas, Saulius; Szczesna, Aleksandra; Juhasz, Erzsebet; Esteban, Emilio; Molinier, Olivier; Brugger, Wolfram; Melezinek, Ivan; Klingelschmitt, Gaelle; Klughammer, Barbara; Giaccone, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-529</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">First-line chemotherapy for advanced non-small-cell lung cancer (NSCLC) is usually limited to four to six cycles.  Maintenance therapy can delay progression and prolong survival.  The oral epidermal growth factor receptor (EGFR) tyrosine-kinase inhibitor erlotinib has proven efficacy and tolerability in second-line NSCLC.  We designed the phase 3, placebo-controlled Sequential Tarceva in Unresectable NSCLC (SATURN; BO18192) study to assess use of erlotinib as maintenance therapy in patients with non-progressive disease following first-line platinum-doublet chemotherapy.  Between Dec., 2005, and May, 2008, 1949 patients were included in the run-in phase (four cycles of platinum-based chemotherapy).  At the end of the run-in phase, 889 patients who did not have progressive disease were entered into the main study, and were randomly allocated using a 1:1 adaptive randomisation method through a third-party interactive voice response system to receive erlotinib (150 mg/day; n=438) or placebo (n=451) until progression or unacceptable toxicity.  Patients were stratified by EGFR immunohistochem. status, stage, Eastern Cooperative Oncol. Group performance status, chemotherapy regimen, smoking history, and region.  Co-primary endpoints were progression-free survival (PFS) in all analysable patients irresp. of EGFR status, and PFS in patients whose tumors had EGFR protein overexpression, as detd. by immunohistochem.  This study is registered with www.ClinicalTrials.gov, no. NCT00556712.  884 patients were analysable for PFS; 437 in the erlotinib group and 447 in the placebo group.  After a median follow-up of 11.4 mo for the erlotinib group and 11.5 mo for the placebo group, median PFS was significantly longer with erlotinib than with placebo: 12.3 wk for patients in the erlotinib group vs. 11.1 wk for those in the placebo group (HR 0.71, 95% CI 0.62-0.82; p<0.0001).  PFS was also significantly longer in patients with EGFR-pos. immunohistochem. who were treated with erlotinib (n=307) compared with EGFR-pos. patients given placebo (n=311; median PFS 12.3 wk in the erlotinib group vs 11.1 wk in the placebo group; HR 0.69, 0.58-0.82; p<0.0001).  The most common grade 3 or higher adverse events were rash (37 [9%] of 443 patients in the erlotinib group vs none of 445 in the placebo group) and diarrhoea (seven [2%] of 443 patients vs none of 445).  Serious adverse events were reported in 47 patients (11%) on erlotinib compared with 34 patients (8%) on placebo.  The most common serious adverse event was pneumonia (seven cases [2%] with erlotinib and four [<1%] with placebo).  Maintenance therapy with erlotinib for patients with NSCLC is well tolerated and significantly prolongs PFS compared with placebo.  First-line maintenance with erlotinib could be considered in patients who do not progress after four cycles of chemotherapy.  Funding: F Hoffmann-La Roche Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ1stlOmTgyrVg90H21EOLACvtfcHk0lgmxjXYCH8o-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D&md5=384f65f7c8ad10ca34d2d0fed7183484</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970112-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970112-1%26sid%3Dliteratum%253Aachs%26aulast%3DCappuzzo%26aufirst%3DF.%26aulast%3DCiuleanu%26aufirst%3DT.%26aulast%3DStelmakh%26aufirst%3DL.%26aulast%3DCicenas%26aufirst%3DS.%26aulast%3DSzczesna%26aufirst%3DA.%26aulast%3DJuhasz%26aufirst%3DE.%26aulast%3DEsteban%26aufirst%3DE.%26aulast%3DMolinier%26aufirst%3DO.%26aulast%3DBrugger%26aufirst%3DW.%26aulast%3DMelez%25C3%25ADnek%26aufirst%3DI.%26aulast%3DKlingelschmitt%26aufirst%3DG.%26aulast%3DKlughammer%26aufirst%3DB.%26aulast%3DGiaccone%26aufirst%3DG.%26atitle%3DErlotinib%2520as%2520maintenance%2520treatment%2520in%2520advanced%2520non-small-cell%2520lung%2520cancer%253A%2520a%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%2520phase%25203%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2010%26volume%3D11%26spage%3D521%26epage%3D529%26doi%3D10.1016%2FS1470-2045%2810%2970112-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muzumdar, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorans, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noll, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprick, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trumpp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span> <span> </span><span class="NLM_article-title">Survival of pancreatic cancer cells lacking KRAS function</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">1090</span> <span class="refDoi"> DOI: 10.1038/s41467-017-00942-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fs41467-017-00942-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=29061961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC1M7js1ymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=M.+D.+Muzumdarauthor=P.+Y.+Chenauthor=K.+J.+Doransauthor=K.+M.+Chungauthor=A.+Bhutkarauthor=E.+Hongauthor=E.+M.+Nollauthor=M.+R.+Sprickauthor=A.+Trumppauthor=T.+Jacks&title=Survival+of+pancreatic+cancer+cells+lacking+KRAS+function&doi=10.1038%2Fs41467-017-00942-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Survival of pancreatic cancer cells lacking KRAS function</span></div><div class="casAuthors">Muzumdar Mandar Deepak; Chen Pan-Yu; Dorans Kimberly Judith; Chung Katherine Minjee; Bhutkar Arjun; Hong Erin; Jacks Tyler; Muzumdar Mandar Deepak; Muzumdar Mandar Deepak; Chen Pan-Yu; Jacks Tyler; Noll Elisa M; Sprick Martin R; Trumpp Andreas; Noll Elisa M; Sprick Martin R; Trumpp Andreas; Jacks Tyler</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1090</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Activating mutations in the proto-oncogene KRAS are a hallmark of pancreatic ductal adenocarcinoma (PDAC), an aggressive malignancy with few effective therapeutic options.  Despite efforts to develop KRAS-targeted drugs, the absolute dependence of PDAC cells on KRAS remains incompletely understood.  Here we model complete KRAS inhibition using CRISPR/Cas-mediated genome editing and demonstrate that KRAS is dispensable in a subset of human and mouse PDAC cells.  Remarkably, nearly all KRAS deficient cells exhibit phosphoinositide 3-kinase (PI3K)-dependent mitogen-activated protein kinase (MAPK) signaling and induced sensitivity to PI3K inhibitors.  Furthermore, comparison of gene expression profiles of PDAC cells retaining or lacking KRAS reveal a role of KRAS in the suppression of metastasis-related genes.  Collectively, these data underscore the potential for PDAC resistance to even the very best KRAS inhibitors and provide insights into mechanisms of response and resistance to KRAS inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9zxYb5ouxkAwxfk_AGB-EfW6udTcc2eacavSJ_lfIwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7js1ymtA%253D%253D&md5=c25ffa63b98124b5013f4cf8f8e65048</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00942-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00942-5%26sid%3Dliteratum%253Aachs%26aulast%3DMuzumdar%26aufirst%3DM.%2BD.%26aulast%3DChen%26aufirst%3DP.%2BY.%26aulast%3DDorans%26aufirst%3DK.%2BJ.%26aulast%3DChung%26aufirst%3DK.%2BM.%26aulast%3DBhutkar%26aufirst%3DA.%26aulast%3DHong%26aufirst%3DE.%26aulast%3DNoll%26aufirst%3DE.%2BM.%26aulast%3DSprick%26aufirst%3DM.%2BR.%26aulast%3DTrumpp%26aufirst%3DA.%26aulast%3DJacks%26aufirst%3DT.%26atitle%3DSurvival%2520of%2520pancreatic%2520cancer%2520cells%2520lacking%2520KRAS%2520function%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-00942-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebenaler, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chugh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dommeti, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelenka-Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apel, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waninger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyunni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mody, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodrum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vats, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitchiaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellison, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar-Sinha, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">An essential role for Argonaute 2 in EGFR-KRAS signaling in pancreatic cancer development</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">2817</span> <span class="refDoi"> DOI: 10.1038/s41467-020-16309-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fs41467-020-16309-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=32499547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWlsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=S.+Shankarauthor=J.+C.+Tienauthor=R.+F.+Siebenalerauthor=S.+Chughauthor=V.+L.+Dommetiauthor=S.+Zelenka-Wangauthor=X.+M.+Wangauthor=I.+J.+Apelauthor=J.+Waningerauthor=S.+Eyunniauthor=A.+Xuauthor=M.+Modyauthor=A.+Goodrumauthor=Y.+Zhangauthor=J.+J.+Tesmerauthor=R.+Mannanauthor=X.+Caoauthor=P.+Vatsauthor=S.+Pitchiayaauthor=S.+J.+Ellisonauthor=J.+Shiauthor=C.+Kumar-Sinhaauthor=H.+C.+Crawfordauthor=A.+M.+Chinnaiyan&title=An+essential+role+for+Argonaute+2+in+EGFR-KRAS+signaling+in+pancreatic+cancer+development&doi=10.1038%2Fs41467-020-16309-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for Argonaute 2 in EGFR-KRAS signaling in pancreatic cancer development</span></div><div class="casAuthors">Shankar, Sunita; Tien, Jean Ching-Yi; Siebenaler, Ronald F.; Chugh, Seema; Dommeti, Vijaya L.; Zelenka-Wang, Sylvia; Wang, Xiao-Ming; Apel, Ingrid J.; Waninger, Jessica; Eyunni, Sanjana; Xu, Alice; Mody, Malay; Goodrum, Andrew; Zhang, Yuping; Tesmer, John J.; Mannan, Rahul; Cao, Xuhong; Vats, Pankaj; Pitchiaya, Sethuramasundaram; Ellison, Stephanie J.; Shi, Jiaqi; Kumar-Sinha, Chandan; Crawford, Howard C.; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2817</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Both KRAS and EGFR are essential mediators of pancreatic cancer development and interact with Argonaute 2 (AGO2) to perturb its function.  Here, in a mouse model of mutant KRAS-driven pancreatic cancer, loss of AGO2 allows precursor lesion (PanIN) formation yet prevents progression to pancreatic ductal adenocarcinoma (PDAC).  Precursor lesions with AGO2 ablation undergo oncogene-induced senescence with altered microRNA expression and EGFR/RAS signaling, bypassed by loss of p53.  In mouse and human pancreatic tissues, PDAC progression is assocd. with increased plasma membrane localization of RAS/AGO2.  Furthermore, phosphorylation of AGO2Y393 disrupts both the wild-type and oncogenic KRAS-AGO2 interaction, albeit under different conditions.  ARS-1620 (G12C-specific inhibitor) disrupts the KRASG12C-AGO2 interaction, suggesting that the interaction is targetable.  Altogether, our study supports a biphasic model of pancreatic cancer development: an AGO2-independent early phase of PanIN formation reliant on EGFR-RAS signaling, and an AGO2-dependent phase wherein the mutant KRAS-AGO2 interaction is crit. for PDAC progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjEHXX-Kh0KLVg90H21EOLACvtfcHk0lhpGc180-D4Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWlsb7O&md5=38f738b80e139e28d50421127c3adcc8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16309-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16309-2%26sid%3Dliteratum%253Aachs%26aulast%3DShankar%26aufirst%3DS.%26aulast%3DTien%26aufirst%3DJ.%2BC.%26aulast%3DSiebenaler%26aufirst%3DR.%2BF.%26aulast%3DChugh%26aufirst%3DS.%26aulast%3DDommeti%26aufirst%3DV.%2BL.%26aulast%3DZelenka-Wang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BM.%26aulast%3DApel%26aufirst%3DI.%2BJ.%26aulast%3DWaninger%26aufirst%3DJ.%26aulast%3DEyunni%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DA.%26aulast%3DMody%26aufirst%3DM.%26aulast%3DGoodrum%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%26aulast%3DMannan%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DVats%26aufirst%3DP.%26aulast%3DPitchiaya%26aufirst%3DS.%26aulast%3DEllison%26aufirst%3DS.%2BJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DKumar-Sinha%26aufirst%3DC.%26aulast%3DCrawford%26aufirst%3DH.%2BC.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DAn%2520essential%2520role%2520for%2520Argonaute%25202%2520in%2520EGFR-KRAS%2520signaling%2520in%2520pancreatic%2520cancer%2520development%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-16309-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span> <span> </span><span class="NLM_article-title">Epidermal Growth Factor Receptor-Targeting Peptide Nanoparticles Simultaneously Deliver Gemcitabine and Olaparib To Treat Pancreatic Cancer with Breast Cancer 2 (BRCA2) Mutation</span>. <i>ACS Nano</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">10785</span>– <span class="NLM_lpage">10796</span>, <span class="refDoi"> DOI: 10.1021/acsnano.8b01573</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsnano.8b01573" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFeis7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=10785-10796&author=C.+Duauthor=Y.+Qiauthor=Y.+Zhangauthor=Y.+Wangauthor=X.+Zhaoauthor=H.+Minauthor=X.+Hanauthor=J.+Langauthor=H.+Qinauthor=Q.+Shiauthor=Z.+Zhangauthor=X.+Tianauthor=G.+J.+Andersonauthor=Y.+Zhaoauthor=G.+Nieauthor=Y.+Yang&title=Epidermal+Growth+Factor+Receptor-Targeting+Peptide+Nanoparticles+Simultaneously+Deliver+Gemcitabine+and+Olaparib+To+Treat+Pancreatic+Cancer+with+Breast+Cancer+2+%28BRCA2%29+Mutation&doi=10.1021%2Facsnano.8b01573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal Growth Factor Receptor-Targeting Peptide Nanoparticles Simultaneously Deliver Gemcitabine and Olaparib To Treat Pancreatic Cancer with Breast Cancer 2 (BRCA2) Mutation</span></div><div class="casAuthors">Du, Chong; Qi, Yingqiu; Zhang, Yinlong; Wang, Yazhou; Zhao, Xiao; Min, Huan; Han, Xuexiang; Lang, Jiayan; Qin, Hao; Shi, Quanwei; Zhang, Zhengkui; Tian, Xiaodong; Anderson, Greg J.; Zhao, Ying; Nie, Guangjun; Yang, Yinmo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Nano</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">10785-10796</span>CODEN:
                <span class="NLM_cas:coden">ANCAC3</span>;
        ISSN:<span class="NLM_cas:issn">1936-0851</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pancreatic cancer (PCa) is one of the most lethal malignancies, with a 5 yr survival rate of less than 8%.  Current treatment regiments have a low response rate in unselected patients.  However, the subgroup of PCa patients with BRCA mutations may benefit from poly-ADP-ribose polymerase inhibitors (PARPi) due to their biol. properties in DNA repair.  Dose-limiting toxicity in normal tissues is frequently obsd. when PARPi are combined with other chemotherapies, and the co-delivery of two drugs to tumor sites at an adequate concn. is challenging.  To address this issue, we have engineered an epidermal growth factor receptor (EGFR) targeting (with GE11 peptide) self-assembly amphiphilic peptide nanoparticle (GENP) to co-deliver gemcitabine and the PARPi olaparib to treat BRCA mutant PCa.  The GENP was relatively stable, exhibited high encapsulation efficiency, and could coordinately release the two drugs in tumor milieu.  Gemcitabine and olaparib showed strong synergistic actions in optimized conditions in vitro.  The nanoparticle prolonged the half-life of both drugs and resulted in their tumor accumulation at the optimal therapeutic ratio in vivo.  The drug-loaded nanoparticles were able to significantly suppress tumor growth in a murine PCa model with minimal side effects.  Drug co-delivery of DNA damaging agents and PARP inhibitors via the GENP represents a promising approach for treatment of pancreatic cancers with mol. defects in the DNA repair pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4-Q4EYVi_D7Vg90H21EOLACvtfcHk0lhpGc180-D4Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFeis7jI&md5=5319397ebeedfa5bc177f52c7076874e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsnano.8b01573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsnano.8b01573%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMin%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DLang%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DG.%2BJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DNie%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DEpidermal%2520Growth%2520Factor%2520Receptor-Targeting%2520Peptide%2520Nanoparticles%2520Simultaneously%2520Deliver%2520Gemcitabine%2520and%2520Olaparib%2520To%2520Treat%2520Pancreatic%2520Cancer%2520with%2520Breast%2520Cancer%25202%2520%2528BRCA2%2529%2520Mutation%26jtitle%3DACS%2520Nano%26date%3D2018%26volume%3D12%26spage%3D10785%26epage%3D10796%26doi%3D10.1021%2Facsnano.8b01573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Discovery and biological evaluation of proteolysis targeting chimeras (PROTACs) as an EGFR degrader based on osimertinib and lenalidomide</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">127167</span> <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.bmcl.2020.127167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=32317208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnsV2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&author=K.+Heauthor=Z.+Zhangauthor=W.+Wangauthor=X.+Zhengauthor=X.+Wangauthor=X.+Zhang&title=Discovery+and+biological+evaluation+of+proteolysis+targeting+chimeras+%28PROTACs%29+as+an+EGFR+degrader+based+on+osimertinib+and+lenalidomide&doi=10.1016%2Fj.bmcl.2020.127167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and biological evaluation of proteolysis targeting chimeras (PROTACs) as an EGFR degraders based on osimertinib and lenalidomide</span></div><div class="casAuthors">He, Kailun; Zhang, Zhuo; Wang, Wenbing; Zheng, Xiaoliang; Wang, Xiaoju; Zhang, Xingxian</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">127167</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) is one of the important and valuable drug targets.  Overexpression of EGFR is assocd. with the development of many types of cancer.  In this study, three PROTACs small mols. (16a-16c) were designed, synthesized and evaluated for their cytotoxicity against the growth in different NSCLC cell line and the degrdn. effect.  The bioassay results indicated that 16c has a good inhibition in PC9 cells and H1975 cells, and the corresponding IC50 value was 0.413 μM and 0.657 μM, resp.  Western blotting results demonstrated that compd. 16c could serve as an effective EGFRdel19-targeting degrader in PC9 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgut0gW1TkiLVg90H21EOLACvtfcHk0lhpGc180-D4Yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnsV2qt74%253D&md5=bf63e55c59176f262c8569824417b2ef</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127167%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DDiscovery%2520and%2520biological%2520evaluation%2520of%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520as%2520an%2520EGFR%2520degrader%2520based%2520on%2520osimertinib%2520and%2520lenalidomide%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26doi%3D10.1016%2Fj.bmcl.2020.127167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. Q.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent epidermal growth factor receptor (EGFR) degraders by proteolysis targeting chimera (PROTAC)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>189</i></span>, <span class="NLM_elocation-id">112061</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.ejmech.2020.112061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=31951960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFCltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2020&author=H.+Zhangauthor=H.+Y.+Zhaoauthor=X.+X.+Xiauthor=Y.+J.+Liuauthor=M.+Xinauthor=S.+Maoauthor=J.+J.+Zhangauthor=A.+X.+Luauthor=S.+Q.+Zhang&title=Discovery+of+potent+epidermal+growth+factor+receptor+%28EGFR%29+degraders+by+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1016%2Fj.ejmech.2020.112061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent epidermal growth factor receptor (EGFR) degraders by proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Zhang, Hao; Zhao, Hong-Yi; Xi, Xiao-Xiao; Liu, Yan-Jie; Xin, Minhang; Mao, Shuai; Zhang, Jun-Jie; Lu, A-Xin; Zhang, San-Qi</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112061</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Herein, the discovery of small mol. EGFR degraders based on the proteolysis targeting chimera (PROTAC) strategy was reported.  In the present study, 13 EGFR degraders contg. pyrido[3,4-d]pyrimidine moiety I [X = CH2C(O)NHCH2CH2C(O), CH2CH2CH2CH2C(O), CH2CH2CH2CH2CH2C(O), etc.], II [C = CH2C(O)NHCH2CH2C(O), CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2C(O), CH2CH2OCH2CH2OCH2CH2, CH2CH2CH2CH2CH2CH2C(O), etc.] were designed and synthesized.  Promising PROTACs I [X = CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2CH2CH2C(O)] induced degrdn. of EGFR in HCC827 cells with the DC50 values of 45.2 and 34.8 nM, resp.  Cellular protein-controlling machinery ubiquitin proteasome system (UPS) was involved in the degrdn. process.  Furthermore, the degraders I [X = CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2CH2CH2C(O)] could significantly induce the apoptosis of HCC827 cells and arrest the cells in G1 phase.  These findings demonstrated that compds. I [X = CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2CH2CH2C(O)] could serve as effective EGFRdel19-targeting degraders in HCC827 cells. v.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonJUw1b6N-77Vg90H21EOLACvtfcHk0ljw9tsggvnFqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFCltr8%253D&md5=8f8a1fbc8df721332a2a2b5d6bbc7875</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112061%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DH.%2BY.%26aulast%3DXi%26aufirst%3DX.%2BX.%26aulast%3DLiu%26aufirst%3DY.%2BJ.%26aulast%3DXin%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%2BJ.%26aulast%3DLu%26aufirst%3DA.%2BX.%26aulast%3DZhang%26aufirst%3DS.%2BQ.%26atitle%3DDiscovery%2520of%2520potent%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520degraders%2520by%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D189%26doi%3D10.1016%2Fj.ejmech.2020.112061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of selective degraders of EGFR(L858R/T790M) mutant</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>192</i></span>, <span class="NLM_elocation-id">112199</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.ejmech.2020.112199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=32171162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvFyltbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2020&author=X.+Zhangauthor=F.+Xuauthor=L.+Tongauthor=T.+Zhangauthor=H.+Xieauthor=X.+Luauthor=X.+Renauthor=K.+Ding&title=Design+and+synthesis+of+selective+degraders+of+EGFR%28L858R%2FT790M%29+mutant&doi=10.1016%2Fj.ejmech.2020.112199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of selective degraders of EGFRL858R/T790M mutant</span></div><div class="casAuthors">Zhang, Xin; Xu, Fang; Tong, Linjiang; Zhang, Tao; Xie, Hua; Lu, Xiaoyun; Ren, Xiaomei; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112199</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of PROTAC (proteolysis targeting chimera) based selective EGFRL858R/T790M (leucine 858 to arginine 858 mutation and threonine 790 to methionine 790) mutant degraders were designed and synthesized.  One of the most potent compds. (I) effectively and selectively degraded EGFRL858R/T790M with an DC50 value of 5.9 nM, while did not show obvious effect on the wild-type protein.  Further mechanism investigation revealed that the degrdn. was mediated by ubiquitin proteasome pathway.  Compd. I could be utilized as an initial lead mol. for development of new EGFRL858R/T790M degrader based therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoDW2mGkQNzbVg90H21EOLACvtfcHk0ljw9tsggvnFqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvFyltbk%253D&md5=e8fdd02842f6cfd94e362f3a53a39096</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112199%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520selective%2520degraders%2520of%2520EGFR%2528L858R%252FT790M%2529%2520mutant%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D192%26doi%3D10.1016%2Fj.ejmech.2020.112199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelier, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascal, J. M.</span></span> <span> </span><span class="NLM_article-title">Purification of human PARP-1 and PARP-1 domains from <i>Escherichia coli</i> for structural and biochemical analysis</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>780</i></span>,  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">226</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-270-0_13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1007%2F978-1-61779-270-0_13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=21870263" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=780&publication_year=2011&pages=209-226&author=M.+F.+Angelierauthor=J.+L.+Planckauthor=K.+M.+Serventauthor=J.+M.+Pascal&title=Purification+of+human+PARP-1+and+PARP-1+domains+from+Escherichia+coli+for+structural+and+biochemical+analysis&doi=10.1007%2F978-1-61779-270-0_13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-270-0_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-270-0_13%26sid%3Dliteratum%253Aachs%26aulast%3DAngelier%26aufirst%3DM.%2BF.%26aulast%3DPlanck%26aufirst%3DJ.%2BL.%26aulast%3DServent%26aufirst%3DK.%2BM.%26aulast%3DPascal%26aufirst%3DJ.%2BM.%26atitle%3DPurification%2520of%2520human%2520PARP-1%2520and%2520PARP-1%2520domains%2520from%2520Escherichia%2520coli%2520for%2520structural%2520and%2520biochemical%2520analysis%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D780%26spage%3D209%26epage%3D226%26doi%3D10.1007%2F978-1-61779-270-0_13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukimoto-Niino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terazawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semba, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the altered drug sensitivities of non-small cell lung cancer-associated mutants of human epidermal growth factor receptor</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1038/onc.2012.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fonc.2012.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=22349823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvFKl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=27-38&author=S.+Yoshikawaauthor=M.+Kukimoto-Niinoauthor=L.+Parkerauthor=N.+Handaauthor=T.+Teradaauthor=T.+Fujimotoauthor=Y.+Terazawaauthor=M.+Wakiyamaauthor=M.+Satoauthor=S.+Sanoauthor=T.+Kobayashiauthor=T.+Tanakaauthor=L.+Chenauthor=Z.+J.+Liuauthor=B.+C.+Wangauthor=M.+Shirouzuauthor=S.+Kawaauthor=K.+Sembaauthor=T.+Yamamotoauthor=S.+Yokoyama&title=Structural+basis+for+the+altered+drug+sensitivities+of+non-small+cell+lung+cancer-associated+mutants+of+human+epidermal+growth+factor+receptor&doi=10.1038%2Fonc.2012.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the altered drug sensitivities of non-small cell lung cancer-associated mutants of human epidermal growth factor receptor</span></div><div class="casAuthors">Yoshikawa, S.; Kukimoto-Niino, M.; Parker, L.; Handa, N.; Terada, T.; Fujimoto, T.; Terazawa, Y.; Wakiyama, M.; Sato, M.; Sano, S.; Kobayashi, T.; Tanaka, T.; Chen, L.; Liu, Z.-J.; Wang, B.-C.; Shirouzu, M.; Kawa, S.; Semba, K.; Yamamoto, T.; Yokoyama, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) has an essential role in multiple signaling pathways, including cell proliferation and migration, through extracellular ligand binding and subsequent activation of its intracellular tyrosine kinase (TK) domain.  The non-small cell lung cancer (NSCLC)-assocd. EGFR mutants, L858R and G719S, are constitutively active and oncogenic.  They display sensitivity to TK inhibitors, including gefitinib and erlotinib.  In contrast, the secondary mutation of the gatekeeper residue, T790M, reportedly confers inhibitor resistance on the oncogenic EGFR mutants.  In this study, our biochem. analyses revealed that the introduction of the T790M mutation confers gefitinib resistance on the G719S mutant.  The G719S/T790M double mutant has enhanced activity and retains high gefitinib-binding affinity.  The T790M mutation increases the ATP affinity of the G719S mutant, explaining the acquired drug resistance of the double mutant.  Structural analyses of the G719S/T790M double mutant, as well as the wild type and the G719S and L858R mutants, revealed that the T790M mutation stabilizes the hydrophobic spine of the active EGFR-TK conformation.  The Met790 side chain of the G719S/T790M double mutant, in the apo form and gefitinib- and AMPPNP-bound forms, adopts different conformations that explain the accommodation of these ligands.  In the L858R mutant structure, the active-site cleft is expanded by the repositioning of Phe723 within the P-loop.  Notably, the introduction of the F723A mutation greatly enhanced the gefitinib sensitivity of the wild-type EGFR in vivo, supporting our hypothesis that the expansion of the active-site cleft results in enhanced gefitinib sensitivity.  Taken together, our results provide a structural basis for the altered drug sensitivities caused by distinct NSCLC-assocd. EGFR mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr7hEqvdS5i7Vg90H21EOLACvtfcHk0ljdMAeO0NQtkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvFKl&md5=f9b29aac7bcb52cc0605f157f7f14809</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.21%26sid%3Dliteratum%253Aachs%26aulast%3DYoshikawa%26aufirst%3DS.%26aulast%3DKukimoto-Niino%26aufirst%3DM.%26aulast%3DParker%26aufirst%3DL.%26aulast%3DHanda%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DFujimoto%26aufirst%3DT.%26aulast%3DTerazawa%26aufirst%3DY.%26aulast%3DWakiyama%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DSano%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DZ.%2BJ.%26aulast%3DWang%26aufirst%3DB.%2BC.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DKawa%26aufirst%3DS.%26aulast%3DSemba%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520for%2520the%2520altered%2520drug%2520sensitivities%2520of%2520non-small%2520cell%2520lung%2520cancer-associated%2520mutants%2520of%2520human%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D27%26epage%3D38%26doi%3D10.1038%2Fonc.2012.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wienken, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaske, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbauer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhr, S.</span></span> <span> </span><span class="NLM_article-title">Protein-binding assays in biological liquids using microscale thermophoresis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">100</span> <span class="refDoi"> DOI: 10.1038/ncomms1093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fncomms1093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20981028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&author=C.+J.+Wienkenauthor=P.+Baaskeauthor=U.+Rothbauerauthor=D.+Braunauthor=S.+Duhr&title=Protein-binding+assays+in+biological+liquids+using+microscale+thermophoresis&doi=10.1038%2Fncomms1093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-binding assays in biological liquids using microscale thermophoresis</span></div><div class="casAuthors">Wienken Christoph J; Baaske Philipp; Rothbauer Ulrich; Braun Dieter; Duhr Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein interactions inside the human body are expected to differ from the situation in vitro.  This is crucial when investigating protein functions or developing new drugs.  In this study, we present a sample-efficient, free-solution method, termed microscale thermophoresis, that is capable of analysing interactions of proteins or small molecules in biological liquids such as blood serum or cell lysate.  The technique is based on the thermophoresis of molecules, which provides information about molecule size, charge and hydration shell.  We validated the method using immunologically relevant systems including human interferon gamma and the interaction of calmodulin with calcium.  The affinity of the small-molecule inhibitor quercetin to its kinase PKA was determined in buffer and human serum, revealing a 400-fold reduced affinity in serum.  This information about the influence of the biological matrix may allow to make more reliable conclusions on protein functionality, and may facilitate more efficient drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOCqf6rsXbPv6BottYdIAWfW6udTcc2eYYdSkoyzsdgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D&md5=5660d96475370e7f46ea6682c9398cb1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fncomms1093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1093%26sid%3Dliteratum%253Aachs%26aulast%3DWienken%26aufirst%3DC.%2BJ.%26aulast%3DBaaske%26aufirst%3DP.%26aulast%3DRothbauer%26aufirst%3DU.%26aulast%3DBraun%26aufirst%3DD.%26aulast%3DDuhr%26aufirst%3DS.%26atitle%3DProtein-binding%2520assays%2520in%2520biological%2520liquids%2520using%2520microscale%2520thermophoresis%26jtitle%3DNat.%2520Commun.%26date%3D2010%26volume%3D1%26doi%3D10.1038%2Fncomms1093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design of novel star-shaped dual PROTACs and the schematic diagram of dual-targeted protein degradation. Dual PROTACs consist of two ligands binding to two different target proteins and a ligand binding to the E3 ubiquitin ligase. Three ligands are linked through a linker to form a dual-targeting PROTAC, then recruit the E3 ligase to induce the ubiquitination of both two targets, and subsequent degradation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Route Design and the Structures of Dual PROTACs (<b>DP-C 1-4</b>, <b>DP-V 1-4</b>) and Mono PROTACs (<b>MP-GC</b>, <b>MP-GV</b>, <b>MP-OC</b>, <b>MP-OV</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Tyrosine and serine were used as star-type linkers to connect two inhibitors and one E3 ligase ligand together. Eight dual PROTACs and four mono PROTACs were constructed consequently.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Routes for Mono PROTACs (<b>MP-GC</b>, <b>MP-GV</b>, <b>MP-OC</b>, <b>MP-OV</b>) and Dual PROTACs (<b>DP-C 1-4</b>, <b>DP-V 1-4</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(a) Compound <b>1</b> was treated with 1 equiv of compound <b>2</b> and 2 equiv of <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) in dry dimethylformamide (DMF) at 85 °C overnight; (b) 1 equiv of acid and amine was treated with 1.2 equiv of 1-ethyl-3(3-dimethylpropylamine) carbodiimide (EDCI), 1-hydroxybenzotriazole (HOBt), and DIPEA in dry dichloromethane (DCM) at room temperature overnight; (c) compound <b>7</b> was treated with 1.1 equiv of compound <b>8</b> and 2 equiv of K<sub>2</sub>CO<sub>3</sub> in acetone by refluxing overnight; (d) ester was treated with 5 equiv of NaOH in 1:1 mixture solvent of methanol and THF at room temperature overnight; (e) 1.0 equiv of alkyne and azide was treated with 1 equiv of CuSO<sub>4</sub> and 2 equiv of sodium ascorbate in 4:1 mixture solvent of THF and water at room temperature for a few hours; (f) phenol or alcohol was treated with 2 equiv of bromide <b>17</b> and 2 equiv of K<sub>2</sub>CO<sub>3</sub> or NaH in dry DMF for a few hours; and (g) the Boc-protecting group was removed by treatment with HCl ethyl acetate solution at room temperature overnight.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <b>DP-C-1</b> degraded EGFR and PARP simultaneously in a dose- and time-dependent manner in SW1990 cells. (A) Changes in associated proteins 24 h after the addition of <b>MP-GC</b>. (B) Changes in associated proteins 24 h after the addition of <b>MP-OC</b>. (C) Effect of four dual PROTAC compounds (<b>DP-C 1-4</b>) on the related proteins after 24 h. (D) Effect of 5 μM <b>DP-C-1</b> on related proteins at different time points. (E) Effect of <b>DP-C-1</b> on related proteins after the introduction of 700 nM MG132. EGFR and PARP degradation by <b>DP-C-1</b> in 24 h could be abrogated by pretreating with MG132 for 4 h in 700 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. <b>DP-V-4</b> degraded EGFR and PARP simultaneously in a dose- and time-dependent manner in H1299 cells. (A) Changes in associated proteins 36 h after the addition of <b>MP-GV</b>. (B) Changes in associated proteins 36 h after the addition of <b>MP-OV</b>. (C) Effect of three dual PROTAC compounds (<b>DP-V 1-3</b>) on related proteins after 36 h. (D) Effect of <b>DP-V-4</b> on related proteins with different concentrations at 36 h. (E) Effect of 3.75 μM <b>DP-V-4</b> on related proteins at different times. (F) Effect of <b>DP-V-4</b> on related proteins after the introduction of 1 μM MG132. EGFR and PARP degradation by <b>DP-V-4</b> in 6 h could be abrogated by pretreating with MG132 for 4 h in 1 μM. (G) The antiproliferative activity of Gefitinib, Olaparib, and <b>DP-V-4</b> at 72 h was identified by a cell counting kit-8 (CCK-8) assay. Half-maximal inhibitory concentration (IC<sub>50</sub>) was expressed as mean ± standard deviation (SD).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/medium/jm1c00649_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding affinity of <b>DP-C-1</b> and <b>DP-V-4</b> with EGFR and PARP. (A–C) Measurement of affinity of <b>DP-C-1</b>, <b>DP-V-4</b>, and Gefitinib to EGFR by MST with <i>K</i><sub>d</sub> values of 2.74 ± 0.13, 5.47 ± 0.67, and 0.47 ± 0.05 μM, respectively. (D–F) Measurement of affinity of <b>DP-C-1</b>, <b>DP-V-4</b>, and Olaparib to PARP by MST with <i>K</i><sub>d</sub> values of 7.89 ± 0.96, 12.80 ± 2.16, and 5.31 ± 1.11 μM, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-11/acs.jmedchem.1c00649/20210602/images/large/jm1c00649_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00649&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i67">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10641" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10641" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 29 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
der Veeken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiffelers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storm, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Bergen En Henegouwen, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roovers, R. C.</span></span> <span> </span><span class="NLM_article-title">Crosstalk between epidermal growth factor receptor and insulin-like growth factor-1 receptor signaling: implications for cancer therapy</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">760</span>, <span class="refDoi"> DOI: 10.2174/156800909789271495</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.2174%2F156800909789271495" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=19754359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlKnurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=748-760&author=J.+van%0Ader+Veekenauthor=S.+Oliveiraauthor=R.+M.+Schiffelersauthor=G.+Stormauthor=P.+M.+van+Bergen+En+Henegouwenauthor=R.+C.+Roovers&title=Crosstalk+between+epidermal+growth+factor+receptor+and+insulin-like+growth+factor-1+receptor+signaling%3A+implications+for+cancer+therapy&doi=10.2174%2F156800909789271495"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Crosstalk between epidermal growth factor receptor- and insulin-like growth factor-1 receptor signaling: implications for cancer therapy</span></div><div class="casAuthors">van der Veeken, J.; Oliveira, S.; Schiffelers, R. M.; Storm, G.; van Bergen en Henegouwen, P. M. P.; Roovers, R. C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">748-760</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Both the epidermal growth factor receptor (EGFR) and the insulin-like growth factor-1 receptor (IGF-1R) can contribute to tumor development and -progression through their effects on cell proliferation, inhibition of apoptosis, angiogenesis, anchorage-independent growth and tumor-assocd. inflammation.  EGFR-targeting monoclonal antibodies and small mol. tyrosine kinase inhibitors are currently in clin. use for the treatment of several types of cancer.  However, primary and acquired resistance to these agents often occurs and thereby limits the clin. efficacy of monospecific targeted therapy.  Results from both in vitro and in vivo studies indicate that cross-talk between EGFR and IGF-1R can lead to acquired resistance against EGFR-targeted drugs.  This review describes the interface between the EGFR and IGF-1R signaling networks and the implications of the extensive cross-talk between these two receptor systems for cancer therapy.  EGFR and IGF-1R interact on multiple levels, either through a direct assocn. between the two receptors, by mediating the availability of each others ligands, or indirectly, via common interaction partners such as G protein coupled receptors (GPCR) or downstream signaling mols.  This multi-layered cross-talk and its involvement in the induction of resistance to targeted therapies provide a clear rationale for dual targeting of EGFR and IGF-1R.  We discuss several (potential) strategies to simultaneously inhibit EGFR and IGF-1R signaling as promising novel therapeutic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_LfAONZf17Vg90H21EOLACvtfcHk0ljApnVzFXE45Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlKnurrI&md5=ccaef4bafc2ddce6a807bf3605091ada</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F156800909789271495&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156800909789271495%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2Bder%2BVeeken%26aufirst%3DJ.%26aulast%3DOliveira%26aufirst%3DS.%26aulast%3DSchiffelers%26aufirst%3DR.%2BM.%26aulast%3DStorm%26aufirst%3DG.%26aulast%3Dvan%2BBergen%2BEn%2BHenegouwen%26aufirst%3DP.%2BM.%26aulast%3DRoovers%26aufirst%3DR.%2BC.%26atitle%3DCrosstalk%2520between%2520epidermal%2520growth%2520factor%2520receptor%2520and%2520insulin-like%2520growth%2520factor-1%2520receptor%2520signaling%253A%2520implications%2520for%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2009%26volume%3D9%26spage%3D748%26epage%3D760%26doi%3D10.2174%2F156800909789271495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgillo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Y.</span></span> <span> </span><span class="NLM_article-title">Resistance to epidermal growth factor receptor-targeted therapy</span>. <i>Drug Resist. Updates</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1016/j.drup.2005.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.drup.2005.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=16172017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1Oms7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2005&pages=298-310&author=F.+Morgilloauthor=H.+Y.+Lee&title=Resistance+to+epidermal+growth+factor+receptor-targeted+therapy&doi=10.1016%2Fj.drup.2005.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to epidermal growth factor receptor-targeted therapy</span></div><div class="casAuthors">Morgillo, Floriana; Lee, Ho-Young</div><div class="citationInfo"><span class="NLM_cas:title">Drug Resistance Updates</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">298-310</span>CODEN:
                <span class="NLM_cas:coden">DRUPFW</span>;
        ISSN:<span class="NLM_cas:issn">1368-7646</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The epidermal growth factor receptor (EGFR) has been a major target of mol. anticancer therapy.  Two approaches have been developed, involving monoclonal antibodies and receptor tyrosine kinase inhibitors, and both have demonstrated benefit in clin. trials.  However, evidence of resistance to these drugs has been described.  Cellular levels of EGFR do not always correlate with response to the EGFR tyrosine kinase inhibitors, indicating acquired resistance to these drugs.  Since EGFR antagonists interfere with the activation of several intracellular pathways that control cell proliferation, survival, apoptosis, angiogenesis, invasion and metastasis, acquired resistance can occur as a result of several different mol. mechanisms: autocrine/paracrine prodn. of ligand, receptor mutation, constitutive activation of the downstream pathway and activation of alternative pathways.  We will describe here potential mechanisms that can cause resistance to EGFR-targeted drugs.  Combinations of EGFR antagonists with inhibitors targeting different signaling mechanism(s) - such as insulin-like growth factor receptor and vascular endothelial growth factor receptor - that share the same downstream mediator (e.g., phosphatidylinositol 3-kinase/Akt, mitogen-activated protein kinase), may circumvent or delay the development of resistance to EGFR antagonists resulting in enhanced antitumor activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwZiu46foBdbVg90H21EOLACvtfcHk0ljApnVzFXE45Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1Oms7%252FE&md5=3979200d41479c15ddf9f87276595c8c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.drup.2005.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drup.2005.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DMorgillo%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DH.%2BY.%26atitle%3DResistance%2520to%2520epidermal%2520growth%2520factor%2520receptor-targeted%2520therapy%26jtitle%3DDrug%2520Resist.%2520Updates%26date%3D2005%26volume%3D8%26spage%3D298%26epage%3D310%26doi%3D10.1016%2Fj.drup.2005.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0ljApnVzFXE45Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. J.</span></span> <span> </span><span class="NLM_article-title">Combination Therapy of Bortezomib with Novel Targeted Agents: An Emerging Treatment Strategy</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4094</span>– <span class="NLM_lpage">4104</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-2882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1158%2F1078-0432.CCR-09-2882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20682705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVajs7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=4094-4104&author=J.+J.+Wright&title=Combination+Therapy+of+Bortezomib+with+Novel+Targeted+Agents%3A+An+Emerging+Treatment+Strategy&doi=10.1158%2F1078-0432.CCR-09-2882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Combination Therapy of Bortezomib with Novel Targeted Agents: An Emerging Treatment Strategy</span></div><div class="casAuthors">Wright, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4094-4104</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Clin. trials evaluating combinations of targeted agents with bortezomib, the first-in-class proteasome inhibitor, have been initiated, with the objective of enhancing its single agent activity in hematol. malignancies (myeloma, mantle cell lymphoma), as well as expanding its efficacy in solid tumors.  In most cases, preclin. studies have provided a supportive rationale for designing these doublet combination studies.  Novel, small mol.-targeted agents being investigated with bortezomib in clin. trials include protein deacetylase inhibitors, kinase inhibitors, farnesyltransferase inhibitors, heat-shock protein 90 inhibitors, pan-Bcl-2 family inhibitors, and other classes of targeted inhibitors.  Preliminary clin. data, available from a no. of ongoing trials, suggest that most of these combinations are well tolerated and some have promising clin. efficacy that will require subsequent confirmation.  Translational studies, conducted as part of the trials, may provide important insights into the putative mechanism of action delineated by preclin. studies of the combinations.  The emergence of novel proteasome inhibitors may also expand the opportunities for optimizing these combination therapies.  There is potential for an increasingly broad clin. trials program to investigate this therapeutic approach in a range of tumor types, as well as to consider addnl. agents in sequence or in combination.  Clin Cancer Res; 16(16); 4094-104.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq80JIFlg1k7bVg90H21EOLACvtfcHk0ljb46OWBL_mYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVajs7zK&md5=70350d7044ef8f26fe4867d353d682b7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-2882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-2882%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DJ.%2BJ.%26atitle%3DCombination%2520Therapy%2520of%2520Bortezomib%2520with%2520Novel%2520Targeted%2520Agents%253A%2520An%2520Emerging%2520Treatment%2520Strategy%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D4094%26epage%3D4104%26doi%3D10.1158%2F1078-0432.CCR-09-2882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kontermann, R. E.</span></span> <span> </span><span class="NLM_article-title">Dual targeting strategies with bispecific antibodies</span>. <i>mAbs</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.4161/mabs.4.2.19000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.4161%2Fmabs.4.2.19000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=22453100" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC38rgtVCisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=182-197&author=R.+E.+Kontermann&title=Dual+targeting+strategies+with+bispecific+antibodies&doi=10.4161%2Fmabs.4.2.19000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Dual targeting strategies with bispecific antibodies</span></div><div class="casAuthors">Kontermann Roland E</div><div class="citationInfo"><span class="NLM_cas:title">mAbs</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-97</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Monoclonal antibodies are widely used for the treatment of cancer, inflammatory and infectious diseases and other disorders.  Most of the marketed antibodies are monospecific and therefore capable of interacting and interfering with a single target.  However, complex diseases are often multifactorial in nature, and involve redundant or synergistic action of disease mediators or upregulation of different receptors, including crosstalk between their signaling networks.  Consequently, blockade of multiple, different pathological factors and pathways may result in improved therapeutic efficacy.  This result can be achieved by combining different drugs, or use of the dual targeting strategies applying bispecific antibodies that have emerged as an alternative to combination therapy.  This review discusses the various dual targeting strategies for which bispecific antibodies have been developed and provides an overview of the established bispecific antibody formats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTgBg_RJQ0LBrBItaL1WaVAfW6udTcc2eZCVi6kucew1Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rgtVCisA%253D%253D&md5=b6e2812d15cfd6e3649053796b0ecd7b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2Fmabs.4.2.19000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fmabs.4.2.19000%26sid%3Dliteratum%253Aachs%26aulast%3DKontermann%26aufirst%3DR.%2BE.%26atitle%3DDual%2520targeting%2520strategies%2520with%2520bispecific%2520antibodies%26jtitle%3DmAbs%26date%3D2012%26volume%3D4%26spage%3D182%26epage%3D197%26doi%3D10.4161%2Fmabs.4.2.19000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodfellow, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jean-Claude, B. J.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of a Trifunctional Molecular System “Programmed” to Block Epidermal Growth Factor Receptor Tyrosine Kinase, Induce High Levels of DNA Damage, and Inhibit the DNA Repair Enzyme (Poly(ADP-ribose) Polymerase) in Prostate Cancer Cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5752</span>– <span class="NLM_lpage">5762</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVCqtLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5752-5762&author=J.+Schmittauthor=S.+Huangauthor=E.+Goodfellowauthor=C.+Williamsauthor=B.+J.+Jean-Claude&title=Design+and+Synthesis+of+a+Trifunctional+Molecular+System+%E2%80%9CProgrammed%E2%80%9D+to+Block+Epidermal+Growth+Factor+Receptor+Tyrosine+Kinase%2C+Induce+High+Levels+of+DNA+Damage%2C+and+Inhibit+the+DNA+Repair+Enzyme+%28Poly%28ADP-ribose%29+Polymerase%29+in+Prostate+Cancer+Cells&doi=10.1021%2Facs.jmedchem.9b02008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Trifunctional Molecular System "Programmed" to Block Epidermal Growth Factor Receptor Tyrosine Kinase, Induce High Levels of DNA Damage, and Inhibit the DNA Repair Enzyme (Poly(ADP-ribose) Polymerase) in Prostate Cancer Cells</span></div><div class="casAuthors">Schmitt, Julie; Huang, Shanlong; Goodfellow, Elliot; Williams, Christopher; Jean-Claude, Bertrand J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5752-5762</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resistance to chemotherapy in advanced cancers can be mediated by different factors such as epidermal growth factor receptor (EGFR) overexpression and DNA repair enzymes.  Therefore, current stds. of care usually involve combinations of multiple treatments.  Here, to reduce the adverse effects of multiple drug combinations and improve outcome, we proposed a single drug approach to block multiple overlapping effects that characterize chemoresistance.  Thus, we designed a new linker that allows assembly of multiple functions (e.g., inhibition of EGFR phosphorylation, induction of DNA lesions, and blockade of their repair) into a single mol.  This led to the successful synthesis of a novel and potent combi-mol. JS230(I).  Here, we demonstrated that in resistant prostate cancer cells overexpressing EGFR, it was capable of (a) inhibiting EGFR in a dose-dependent manner, (b) damaging DNA, and (c) sustaining the damage by inhibiting the DNA repair protein poly(ADP-ribose) polymerase (PARP).  The triple mechanism of action of JS230 cumulated into growth inhibitory potency superior to that of classical two- or three-drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8CLi7wKtzibVg90H21EOLACvtfcHk0ljb46OWBL_mYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVCqtLnN&md5=97f4b2f923a8af8cb63bd1f8ce848cb5</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02008%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DGoodfellow%26aufirst%3DE.%26aulast%3DWilliams%26aufirst%3DC.%26aulast%3DJean-Claude%26aufirst%3DB.%2BJ.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520a%2520Trifunctional%2520Molecular%2520System%2520%25E2%2580%259CProgrammed%25E2%2580%259D%2520to%2520Block%2520Epidermal%2520Growth%2520Factor%2520Receptor%2520Tyrosine%2520Kinase%252C%2520Induce%2520High%2520Levels%2520of%2520DNA%2520Damage%252C%2520and%2520Inhibit%2520the%2520DNA%2520Repair%2520Enzyme%2520%2528Poly%2528ADP-ribose%2529%2520Polymerase%2529%2520in%2520Prostate%2520Cancer%2520Cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5752%26epage%3D5762%26doi%3D10.1021%2Facs.jmedchem.9b02008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Desidero, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fioravanti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giannini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basolo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerbel, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danesi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocci, G.</span></span> <span> </span><span class="NLM_article-title">Antiproliferative and ProapoptoticActivity of Sunitinib on Endothelial and Anaplastic Thyroid Cancer Cells via Inhibition of Akt and ERK1/2 Phosphorylation and by Down-Regulation of Cyclin-D1</span>. <i>J. Clin. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">E1465</span>– <span class="NLM_lpage">E1473</span>, <span class="refDoi"> DOI: 10.1210/jc.2013-1364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1210%2Fjc.2013-1364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=23969186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsV2isbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2013&pages=E1465-E1473&author=T.+Di%0ADesideroauthor=A.+Fioravantiauthor=P.+Orlandiauthor=B.+Canuauthor=R.+Gianniniauthor=N.+Borrelliauthor=S.+Manauthor=P.+Xuauthor=G.+Fontaniniauthor=F.+Basoloauthor=R.+S.+Kerbelauthor=G.+Franciaauthor=R.+Danesiauthor=G.+Bocci&title=Antiproliferative+and+ProapoptoticActivity+of+Sunitinib+on+Endothelial+and+Anaplastic+Thyroid+Cancer+Cells+via+Inhibition+of+Akt+and+ERK1%2F2+Phosphorylation+and+by+Down-Regulation+of+Cyclin-D1&doi=10.1210%2Fjc.2013-1364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Antiproliferative and proapoptotic activity of sunitinib on endothelial and anaplastic thyroid cancer cells via inhibition of Akt and ERK1/2 phosphorylation and by down-regulation of cyclin-D1</span></div><div class="casAuthors">Di Desidero, Teresa; Fioravanti, Anna; Orlandi, Paola; Canu, Bastianina; Giannini, Riccardo; Borrelli, Nicla; Man, Shan; Xu, Ping; Fontanini, Gabriella; Basolo, Fulvio; Kerbel, Robert S.; Francia, Giulio; Danesi, Romano; Bocci, Guido</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">E1465-E1473</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Context: Recent exptl. evidence suggests a rationale for the use of multitarget tyrosine kinase inhibitors for the treatment of thyroid cancers.  Sunitinib showed promising preliminary results against anaplastic thyroid cancer (ATC), and it has been used for some patients who are ineligible for clin. trials.  Objectives: The aims of this study were to investigate the in vitro and in vivo activity of sunitinib on ATC and on microvascular endothelial cells and the mol. mechanism for the obsd. sunitinib activity.  Methods: Proliferation and apoptotic assays were performed on human dermal microvascular endothelial and on BRAF- or H-ras-mutated ATC cells (8305C and FB3, resp.) after in vitro exposure to sunitinib for 72 h.  Vascular endothelial growth factor receptor-2, epithelial growth factor receptor, ERK1/2, and Akt phosphorylation was quantified by ELISA and Western blot.  Cyclin-D1 mRNA expression was evaluated by real-time PCR, and cyclin-D1 intracellular concns. were measured by ELISA.  8305C tumor xenografts in nude mice were treated with sunitinib at 50 mg/kg/d (i.p.).  Results: Antiproliferative and proapoptotic activity of sunitinib was obsd. in both endothelial and ATC cells.  Phospho-vascular endothelial growth factor receptor-2 levels significantly decreased after sunitinib treatment in activated endothelial cells.  Phospho-epidermal growth factor receptor, ERK1/2, and Akt phosphorylation was significantly inhibited by sunitinib treatment in endothelial and cancer cells, and cyclin-D1 mRNA and protein expression was inhibited.  Sunitinib administration in vivo caused significant inhibition of tumor growth (P < .05).  Conclusions: Sunitinib is active in vitro and in vivo against activated endothelial and ATC cells via the inhibition of Akt and ERK1/2 phosphorylation and through the down-regulation of cyclin-D1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVHmdomTTpebVg90H21EOLACvtfcHk0lgUrAb9c0Juug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsV2isbnN&md5=32acb61c45541dc9d721f0429861d6dd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1210%2Fjc.2013-1364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2013-1364%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BDesidero%26aufirst%3DT.%26aulast%3DFioravanti%26aufirst%3DA.%26aulast%3DOrlandi%26aufirst%3DP.%26aulast%3DCanu%26aufirst%3DB.%26aulast%3DGiannini%26aufirst%3DR.%26aulast%3DBorrelli%26aufirst%3DN.%26aulast%3DMan%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DP.%26aulast%3DFontanini%26aufirst%3DG.%26aulast%3DBasolo%26aufirst%3DF.%26aulast%3DKerbel%26aufirst%3DR.%2BS.%26aulast%3DFrancia%26aufirst%3DG.%26aulast%3DDanesi%26aufirst%3DR.%26aulast%3DBocci%26aufirst%3DG.%26atitle%3DAntiproliferative%2520and%2520ProapoptoticActivity%2520of%2520Sunitinib%2520on%2520Endothelial%2520and%2520Anaplastic%2520Thyroid%2520Cancer%2520Cells%2520via%2520Inhibition%2520of%2520Akt%2520and%2520ERK1%252F2%2520Phosphorylation%2520and%2520by%2520Down-Regulation%2520of%2520Cyclin-D1%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2013%26volume%3D98%26spage%3DE1465%26epage%3DE1473%26doi%3D10.1210%2Fjc.2013-1364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lgUrAb9c0Juug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lgUrAb9c0Juug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhuri, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussenzweig, A.</span></span> <span> </span><span class="NLM_article-title">The multifaceted roles of PARP1 in DNA repair and chromatin remodelling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1038/nrm.2017.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrm.2017.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=28676700" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=610-621&author=A.+R.+Chaudhuriauthor=A.+Nussenzweig&title=The+multifaceted+roles+of+PARP1+in+DNA+repair+and+chromatin+remodelling&doi=10.1038%2Fnrm.2017.53"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2017.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2017.53%26sid%3Dliteratum%253Aachs%26aulast%3DChaudhuri%26aufirst%3DA.%2BR.%26aulast%3DNussenzweig%26aufirst%3DA.%26atitle%3DThe%2520multifaceted%2520roles%2520of%2520PARP1%2520in%2520DNA%2520repair%2520and%2520chromatin%2520remodelling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D610%26epage%3D621%26doi%3D10.1038%2Fnrm.2017.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, P.</span></span> <span> </span><span class="NLM_article-title">Insights into the biogenesis, function, and regulation of ADP-ribosylation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">236</span>– <span class="NLM_lpage">243</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnchembio.2568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=29443986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislSntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=236-243&author=M.+S.+Cohenauthor=P.+Chang&title=Insights+into+the+biogenesis%2C+function%2C+and+regulation+of+ADP-ribosylation&doi=10.1038%2Fnchembio.2568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the biogenesis, function, and regulation of ADP-ribosylation</span></div><div class="casAuthors">Cohen, Michael S.; Chang, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">236-243</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  ADP-ribosylation, the transfer of ADP-ribose (ADPr) from NAD onto target mols., is catalyzed by members of the ADP-ribosyltransferase (ART) superfamily of proteins, found in all kingdoms of life.  The modification of amino acids in protein targets by ADPr regulates crit. cellular pathways in eukaryotes and underlies the pathogenicity of certain bacteria.  Several members of the ART superfamily are highly relevant for disease; these include the poly(ADP-ribose) polymerases (PARPs), recently shown to be important cancer targets, and the bacterial toxins, diphtheria toxin and cholera toxin, long known to be responsible for the symptoms of diphtheria and cholera that result in morbidity.  Here, we discuss the functions of amino acid ADPr modifications and the ART proteins that make them, the nature of the chem. linkage between ADPr and its targets and how this impacts function and stability, and the way that ARTs select specific amino acids in targets to modify.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTovOQejfF27Vg90H21EOLACvtfcHk0lgCeIS0SN_RIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislSntLg%253D&md5=f92bee47c944be2a5deb3e7440644ad3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2568%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DM.%2BS.%26aulast%3DChang%26aufirst%3DP.%26atitle%3DInsights%2520into%2520the%2520biogenesis%252C%2520function%252C%2520and%2520regulation%2520of%2520ADP-ribosylation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D236%26epage%3D243%26doi%3D10.1038%2Fnchembio.2568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeaupin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapuis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Augustin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkovics, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biertumpfel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timinszky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, S.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose)-dependent chromatin unfolding facilitates the association of DNA-binding proteins with DNA at sites of damage</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">11250</span>– <span class="NLM_lpage">11267</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1093%2Fnar%2Fgkz820" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=31566235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsrzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=11250-11267&author=R.+Smithauthor=T.+Lebeaupinauthor=S.+Juhaszauthor=C.+Chapuisauthor=O.+D%E2%80%99Augustinauthor=S.+Dutertreauthor=P.+Burkovicsauthor=C.+Biertumpfelauthor=G.+Timinszkyauthor=S.+Huet&title=Poly%28ADP-ribose%29-dependent+chromatin+unfolding+facilitates+the+association+of+DNA-binding+proteins+with+DNA+at+sites+of+damage&doi=10.1093%2Fnar%2Fgkz820"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose)-dependent chromatin unfolding facilitates the association of DNA-binding proteins with DNA at sites of damage</span></div><div class="casAuthors">Smith, Rebecca; Lebeaupin, Theo; Juhasz, Szilvia; Chapuis, Catherine; D'Augustin, Ostiane; Dutertre, Stephanie; Burkovics, Peter; Biertumpfel, Christian; Timinszky, Gyula; Huet, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11250-11267</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The addn. of poly(ADP-ribose) (PAR) chains along the chromatin fiber due to PARP1 activity regulates the recruitment of multiple factors to sites of DNA damage.  In this manuscript, we investigated how, besides direct binding to PAR, early chromatin unfolding events controlled by PAR signaling contribute to recruitment to DNA lesions.  We obsd. that different DNA-binding, but not histone-binding, domains accumulate at damaged chromatin in a PAR-dependent manner, and that this recruitment correlates with their affinity for DNA.  Our findings indicate that this recruitment is promoted by early PAR-dependent chromatin remodeling rather than direct interaction with PAR.  Moreover, recruitment is not the consequence of reduced mol. crowding at unfolded damaged chromatin but instead originates from facilitated binding to more exposed DNA.  These findings are further substantiated by the observation that PAR-dependent chromatin remodeling at DNA lesions underlies increased DNAse hypersensitivity.  Finally, the relevance of this new mode of PAR-dependent recruitment to DNA lesions is demonstrated by the observation that reducing the affinity for DNA of both CHD4 and HP1a, two proteins shown to be involved in the DNA-damage response, strongly impairs their recruitment to DNA lesions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp08SUlzHmXErVg90H21EOLACvtfcHk0lgCeIS0SN_RIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVGqsrzO&md5=3c3856829c1453fc62265bf553415aa6</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz820%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%26aulast%3DLebeaupin%26aufirst%3DT.%26aulast%3DJuhasz%26aufirst%3DS.%26aulast%3DChapuis%26aufirst%3DC.%26aulast%3DD%25E2%2580%2599Augustin%26aufirst%3DO.%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DBurkovics%26aufirst%3DP.%26aulast%3DBiertumpfel%26aufirst%3DC.%26aulast%3DTiminszky%26aufirst%3DG.%26aulast%3DHuet%26aufirst%3DS.%26atitle%3DPoly%2528ADP-ribose%2529-dependent%2520chromatin%2520unfolding%2520facilitates%2520the%2520association%2520of%2520DNA-binding%2520proteins%2520with%2520DNA%2520at%2520sites%2520of%2520damage%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3D11250%26epage%3D11267%26doi%3D10.1093%2Fnar%2Fgkz820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yacoub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinstry, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagan, M. P.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor and ionizing radiation upregulate the DNA repair genes XRCC1 and ERCC1 in DU145 and LNCaP prostate carcinoma through MAPK signaling</span>. <i>Radiat. Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">439</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1667/0033-7587(2003)159[0439:EGFAIR]2.0.CO;2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=12643788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVersL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2003&pages=439-452&author=A.+Yacoubauthor=R.+McKinstryauthor=D.+Hinmanauthor=T.+Chungauthor=P.+Dentauthor=M.+P.+Hagan&title=Epidermal+growth+factor+and+ionizing+radiation+upregulate+the+DNA+repair+genes+XRCC1+and+ERCC1+in+DU145+and+LNCaP+prostate+carcinoma+through+MAPK+signaling&doi=10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor and ionizing radiation up-regulate the DNA repair genes XRCC1 and ERCC1 in DU145 and LNCaP prostate carcinoma through MAPK signaling</span></div><div class="casAuthors">Yacoub, Adly; McKinstry, Robert; Hinman, Darin; Chung, Theodore; Dent, Paul; Hagan, Michael P.</div><div class="citationInfo"><span class="NLM_cas:title">Radiation Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">439-452</span>CODEN:
                <span class="NLM_cas:coden">RAREAE</span>;
        ISSN:<span class="NLM_cas:issn">0033-7587</span>.
    
            (<span class="NLM_cas:orgname">Radiation Research Society</span>)
        </div><div class="casAbstract">This work examd. the importance of radiation-induced and ligand-induced EGFR-ERK signaling for the regulation of DNA repair proteins XRCC1 and ERCC1 in prostate carcinoma cells, DU145 (TP53mut), displaying EGFR-TGFA-dependent autocrine growth and high MAPK (ERK1/2) activity, and LNCaP (TP53wt) cells expressing low constitutive levels of ERK1/2 activity.  Using quant. RT-PCR and Western analyses, we detd. that ionizing radiation activated the DNA repair genes XRCC1 and ERCC1 in an ERK1/2-dependent fashion for each cell line.  After irradn., a rapid increase followed by a decrease in ERK1/2 activity preceded the increase in XRCC1/ERCC1 expression in DU145 cells, while only the rapid decrease in ERK1/2 preceded the increase in XRCC1/ERCC1 expression in LNCaP cells.  Administration of EGF, however, markedly increased the up-regulation of phospho-ERK, ERCC1 and XRCC1 in both cell lines.  Although the EGFR inhibitor tyrphostin (AG-1478) and the MEK inhibitor PD90859 both attenuated EGF-induced levels of the ERCC1 and XRCC1 protein, PD98059 blocked the induction of ERCC1 and XRCC1 by radiation more effectively in both cell lines.  Inhibition of ERK at a level that reduced the up-regulation of DNA repair led to the persistence of apurinic/apyrimidinic (AP) sites of DNA damage and increased cell killing.  Taken together, these data imply a complex control of DNA repair activation that may be more generally dependent on MAPK (ERK1/2) signaling than was previously noted.  These data provide novel insights into the capacity of the EGFR-ERK signaling to modulate DNA repair in cancer cells and into the functional significance of this signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH8DufM3lxR7Vg90H21EOLACvtfcHk0lgCeIS0SN_RIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVersL4%253D&md5=0042f1fce51c1c8ea88109cd7bd815de</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1667%252F0033-7587%25282003%2529159%255B0439%253AEGFAIR%255D2.0.CO%253B2%26sid%3Dliteratum%253Aachs%26aulast%3DYacoub%26aufirst%3DA.%26aulast%3DMcKinstry%26aufirst%3DR.%26aulast%3DHinman%26aufirst%3DD.%26aulast%3DChung%26aufirst%3DT.%26aulast%3DDent%26aufirst%3DP.%26aulast%3DHagan%26aufirst%3DM.%2BP.%26atitle%3DEpidermal%2520growth%2520factor%2520and%2520ionizing%2520radiation%2520upregulate%2520the%2520DNA%2520repair%2520genes%2520XRCC1%2520and%2520ERCC1%2520in%2520DU145%2520and%2520LNCaP%2520prostate%2520carcinoma%2520through%2520MAPK%2520signaling%26jtitle%3DRadiat.%2520Res.%26date%3D2003%26volume%3D159%26spage%3D439%26epage%3D452%26doi%3D10.1667%2F0033-7587%282003%29159%5B0439%3AEGFAIR%5D2.0.CO%3B2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chmielecki, J.</span></span> <span> </span><span class="NLM_article-title">Rational, biologically based treatment of EGFR-mutant non-small-cell lung cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">774</span>, <span class="refDoi"> DOI: 10.1038/nrc2947</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrc2947" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20966921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlWlsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=760-774&author=W.+Paoauthor=J.+Chmielecki&title=Rational%2C+biologically+based+treatment+of+EGFR-mutant+non-small-cell+lung+cancer&doi=10.1038%2Fnrc2947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational, biologically based treatment of EGFR-mutant non-small-cell lung cancer</span></div><div class="casAuthors">Pao, William; Chmielecki, Juliann</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">760-774</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epidermal growth factor receptor (EGFR)-mutant non-small-cell lung cancer (NSCLC) was first recognized in 2004 as a distinct, clin. relevant mol. subset of lung cancer.  The disease has been the subject of intensive research at both the basic scientific and clin. levels, becoming a paradigm for how to understand and treat oncogene-driven carcinomas.  Although patients with EGFR-mutant tumors have increased sensitivity to tyrosine kinase inhibitors (TKIs), primary and acquired resistance to these agents remains a major clin. problem.  This Review summarizes recent developments aimed at treating and ultimately curing the disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_b4ITVJ2ZnrVg90H21EOLACvtfcHk0lgmP013iVLGbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlWlsrrN&md5=448b40d66a0a5f171106baa1dc709c29</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrc2947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2947%26sid%3Dliteratum%253Aachs%26aulast%3DPao%26aufirst%3DW.%26aulast%3DChmielecki%26aufirst%3DJ.%26atitle%3DRational%252C%2520biologically%2520based%2520treatment%2520of%2520EGFR-mutant%2520non-small-cell%2520lung%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D760%26epage%3D774%26doi%3D10.1038%2Fnrc2947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paez, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tracy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaye, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindeman, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boggon, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naoki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span> <span> </span><span class="NLM_article-title">EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>304</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">1500</span>, <span class="refDoi"> DOI: 10.1126/science.1099314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1126%2Fscience.1099314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=15118125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=304&publication_year=2004&pages=1497-1500&author=J.+G.+Paezauthor=P.+A.+Janneauthor=J.+C.+Leeauthor=S.+Tracyauthor=H.+Greulichauthor=S.+Gabrielauthor=P.+Hermanauthor=F.+J.+Kayeauthor=N.+Lindemanauthor=T.+J.+Boggonauthor=K.+Naokiauthor=H.+Sasakiauthor=Y.+Fujiiauthor=M.+J.+Eckauthor=W.+R.+Sellersauthor=B.+E.+Johnsonauthor=M.+Meyerson&title=EGFR+mutations+in+lung+cancer%3A+correlation+with+clinical+response+to+gefitinib+therapy&doi=10.1126%2Fscience.1099314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR Mutations in Lung Cancer: Correlation with Clinical Response to Gefitinib Therapy</span></div><div class="casAuthors">Paez, J. Guillermo; Jaenne, Pasi A.; Lee, Jeffrey C.; Tracy, Sean; Greulich, Heidi; Gabriel, Stacey; Herman, Paula; Kaye, Frederic J.; Lindeman, Neal; Boggon, Titus J.; Naoki, Katsuhiko; Sasaki, Hidefumi; Fujii, Yoshitaka; Eck, Michael J.; Sellers, William R.; Johnson, Bruce E.; Meyerson, Matthew</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">304</span>
        (<span class="NLM_cas:issue">5676</span>),
    <span class="NLM_cas:pages">1497-1500</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Receptor tyrosine kinase genes were sequenced in non-small cell lung cancer (NSCLC) and matched normal tissue.  Somatic mutations of the epidermal growth factor receptor gene EGFR were found in 15 of 58 unselected tumors from Japan and 1 of 61 from the United States.  Treatment with the EGFR kinase inhibitor gefitinib (Iressa) causes tumor regression in some patients with NSCLC, more frequently in Japan.  EGFR mutations were found in addnl. lung cancer samples from U.S. patients who responded to gefitinib therapy and in a lung adenocarcinoma cell line that was hypersensitive to growth inhibition by gefitinib, but not in gefitinib-insensitive tumors or cell lines.  These results suggest that EGFR mutations may predict sensitivity to gefitinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGxLJH5f3Kb7Vg90H21EOLACvtfcHk0lgmP013iVLGbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVGmsbs%253D&md5=6df1b11bf9b3c284fbe000e34c61ebac</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099314%26sid%3Dliteratum%253Aachs%26aulast%3DPaez%26aufirst%3DJ.%2BG.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DJ.%2BC.%26aulast%3DTracy%26aufirst%3DS.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DHerman%26aufirst%3DP.%26aulast%3DKaye%26aufirst%3DF.%2BJ.%26aulast%3DLindeman%26aufirst%3DN.%26aulast%3DBoggon%26aufirst%3DT.%2BJ.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DFujii%26aufirst%3DY.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DMeyerson%26aufirst%3DM.%26atitle%3DEGFR%2520mutations%2520in%2520lung%2520cancer%253A%2520correlation%2520with%2520clinical%2520response%2520to%2520gefitinib%2520therapy%26jtitle%3DScience%26date%3D2004%26volume%3D304%26spage%3D1497%26epage%3D1500%26doi%3D10.1126%2Fscience.1099314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsudomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thongprasert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marschner, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralla, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebski, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mok, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. C.</span></span> <span> </span><span class="NLM_article-title">Gefitinib or Erlotinib vs Chemotherapy for EGFR Mutation-Positive Lung Cancer: Individual Patient Data Meta-Analysis of Overall Survival</span>. <i>J. Natl. Cancer. Inst.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>109</i></span>, <span class="NLM_elocation-id">djw279</span> <span class="refDoi"> DOI: 10.1093/jnci/djw279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1093%2Fjnci%2Fdjw279" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2017&author=C.+K.+Leeauthor=L.+Daviesauthor=Y.+L.+Wuauthor=T.+Mitsudomiauthor=A.+Inoueauthor=R.+Rosellauthor=C.+Zhouauthor=K.+Nakagawaauthor=S.+Thongprasertauthor=M.+Fukuokaauthor=S.+Lordauthor=I.+Marschnerauthor=Y.+K.+Tuauthor=R.+J.+Grallaauthor=V.+Gebskiauthor=T.+Mokauthor=J.+C.+Yang&title=Gefitinib+or+Erlotinib+vs+Chemotherapy+for+EGFR+Mutation-Positive+Lung+Cancer%3A+Individual+Patient+Data+Meta-Analysis+of+Overall+Survival&doi=10.1093%2Fjnci%2Fdjw279"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjw279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjw279%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%2BK.%26aulast%3DDavies%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%2BL.%26aulast%3DMitsudomi%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DA.%26aulast%3DRosell%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DNakagawa%26aufirst%3DK.%26aulast%3DThongprasert%26aufirst%3DS.%26aulast%3DFukuoka%26aufirst%3DM.%26aulast%3DLord%26aufirst%3DS.%26aulast%3DMarschner%26aufirst%3DI.%26aulast%3DTu%26aufirst%3DY.%2BK.%26aulast%3DGralla%26aufirst%3DR.%2BJ.%26aulast%3DGebski%26aufirst%3DV.%26aulast%3DMok%26aufirst%3DT.%26aulast%3DYang%26aufirst%3DJ.%2BC.%26atitle%3DGefitinib%2520or%2520Erlotinib%2520vs%2520Chemotherapy%2520for%2520EGFR%2520Mutation-Positive%2520Lung%2520Cancer%253A%2520Individual%2520Patient%2520Data%2520Meta-Analysis%2520of%2520Overall%2520Survival%26jtitle%3DJ.%2520Natl.%2520Cancer.%2520Inst.%26date%3D2017%26volume%3D109%26doi%3D10.1093%2Fjnci%2Fdjw279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pfäffle, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gheorghiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraiolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greninger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sequist, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willers, H.</span></span> <span> </span><span class="NLM_article-title">EGFR-activating mutations correlate with a Fanconi anemia-like cellular phenotype that includes PARP inhibitor sensitivity</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6254</span>– <span class="NLM_lpage">6263</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1158%2F0008-5472.CAN-13-0044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=23966292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC3sbhvVOltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6254-6263&author=H.+N.+Pf%C3%A4ffleauthor=M.+Wangauthor=L.+Gheorghiuauthor=N.+Ferraioloauthor=P.+Greningerauthor=K.+Borgmannauthor=J.+Settlemanauthor=C.+H.+Benesauthor=L.+V.+Sequistauthor=L.+Zouauthor=H.+Willers&title=EGFR-activating+mutations+correlate+with+a+Fanconi+anemia-like+cellular+phenotype+that+includes+PARP+inhibitor+sensitivity&doi=10.1158%2F0008-5472.CAN-13-0044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR-activating mutations correlate with a Fanconi anemia-like cellular phenotype that includes PARP inhibitor sensitivity</span></div><div class="casAuthors">Pfaffle Heike N; Wang Meng; Gheorghiu Liliana; Ferraiolo Natalie; Greninger Patricia; Borgmann Kerstin; Settleman Jeffrey; Benes Cyril H; Sequist Lecia V; Zou Lee; Willers Henning</div><div class="citationInfo"><span class="NLM_cas:title">Cancer research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6254-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In patients with lung cancer whose tumors harbor activating mutations in the EGF receptor (EGFR), increased responses to platinum-based chemotherapies are seen compared with wild-type cancers.  However, the mechanisms underlying this association have remained elusive.  Here, we describe a cellular phenotype of cross-linker sensitivity in a subset of EGFR-mutant lung cancer cell lines that is reminiscent of the defects seen in cells impaired in the Fanconi anemia pathway, including a pronounced G2-M cell-cycle arrest and chromosomal radial formation.  We identified a defect downstream of FANCD2 at the level of recruitment of FAN1 nuclease and DNA interstrand cross-link (ICL) unhooking.  The effect of EGFR mutation was epistatic with FANCD2.  Consistent with the known role of FANCD2 in promoting RAD51 foci formation and homologous recombination repair (HRR), EGFR-mutant cells also exhibited an impaired RAD51 foci response to ICLs, but not to DNA double-strand breaks.  EGFR kinase inhibition affected RAD51 foci formation neither in EGFR-mutant nor wild-type cells.  In contrast, EGFR depletion or overexpression of mutant EGFR in wild-type cells suppressed RAD51 foci, suggesting an EGFR kinase-independent regulation of DNA repair.  Interestingly, EGFR-mutant cells treated with the PARP inhibitor olaparib also displayed decreased FAN1 foci induction, coupled with a putative block in a late HRR step.  As a result, EGFR-mutant lung cancer cells exhibited olaparib sensitivity in vitro and in vivo.  Our findings provide insight into the mechanisms of cisplatin and PARP inhibitor sensitivity of EGFR-mutant cells, yielding potential therapeutic opportunities for further treatment individualization in this genetically defined subset of lung cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTeLzyiu8fY1WBAiYWy6n3SfW6udTcc2eZMVrMMKHyUIrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbhvVOltg%253D%253D&md5=a2c3fb53e1cc765302ffd6cd56e18b08</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0044%26sid%3Dliteratum%253Aachs%26aulast%3DPf%25C3%25A4ffle%26aufirst%3DH.%2BN.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DGheorghiu%26aufirst%3DL.%26aulast%3DFerraiolo%26aufirst%3DN.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DBorgmann%26aufirst%3DK.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DSequist%26aufirst%3DL.%2BV.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DWillers%26aufirst%3DH.%26atitle%3DEGFR-activating%2520mutations%2520correlate%2520with%2520a%2520Fanconi%2520anemia-like%2520cellular%2520phenotype%2520that%2520includes%2520PARP%2520inhibitor%2520sensitivity%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6254%26epage%3D6263%26doi%3D10.1158%2F0008-5472.CAN-13-0044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hirsch, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scagliotti, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulshine, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curran, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paz-Ares, L.</span></span> <span> </span><span class="NLM_article-title">Lung cancer: current therapies and new targeted treatments</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)30958-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2FS0140-6736%2816%2930958-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=27574741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=299-311&author=F.+R.+Hirschauthor=G.+V.+Scagliottiauthor=J.+L.+Mulshineauthor=R.+Kwonauthor=W.+J.+Curranauthor=Y.+L.+Wuauthor=L.+Paz-Ares&title=Lung+cancer%3A+current+therapies+and+new+targeted+treatments&doi=10.1016%2FS0140-6736%2816%2930958-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Lung cancer: current therapies and new targeted treatments</span></div><div class="casAuthors">Hirsch, Fred R.; Scagliotti, Giorgio V.; Mulshine, James L.; Kwon, Regina; Curran, Walter J., Jr.; Wu, Yi-Long; Paz-Ares, Luis</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10066</span>),
    <span class="NLM_cas:pages">299-311</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is the most frequent cause of cancer-related deaths worldwide.  Every year, 1·8 million people are diagnosed with lung cancer, and 1·6 million people die as a result of the disease. 5-yr survival rates vary from 4-17% depending on stage and regional differences.  In this Seminar, we discuss existing treatment for patients with lung cancer and the promise of precision medicine, with special emphasis on new targeted therapies.  Some subgroups, eg-patients with poor performance status and elderly patients-are not specifically addressed, because these groups require special treatment considerations and no frameworks have been established in terms of new targeted therapies.  We discuss prevention and early detection of lung cancer with an emphasis on lung cancer screening.  Although we acknowledge the importance of smoking prevention and cessation, this is a large topic beyond the scope of this Seminar.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_NrglINv67Vg90H21EOLACvtfcHk0ljt7TfMlklwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVamsbzP&md5=36ace1a455a803dc6e7af83dbffb2e34</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2930958-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252930958-8%26sid%3Dliteratum%253Aachs%26aulast%3DHirsch%26aufirst%3DF.%2BR.%26aulast%3DScagliotti%26aufirst%3DG.%2BV.%26aulast%3DMulshine%26aufirst%3DJ.%2BL.%26aulast%3DKwon%26aufirst%3DR.%26aulast%3DCurran%26aufirst%3DW.%2BJ.%26aulast%3DWu%26aufirst%3DY.%2BL.%26aulast%3DPaz-Ares%26aufirst%3DL.%26atitle%3DLung%2520cancer%253A%2520current%2520therapies%2520and%2520new%2520targeted%2520treatments%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D299%26epage%3D311%26doi%3D10.1016%2FS0140-6736%2816%2930958-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herbst, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span> <span> </span><span class="NLM_article-title">Gefitinib–a novel targeted approach to treating cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">956</span>– <span class="NLM_lpage">965</span>, <span class="refDoi"> DOI: 10.1038/nrc1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fnrc1506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=15573117" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=956-965&author=R.+S.+Herbstauthor=M.+Fukuokaauthor=J.+Baselga&title=Gefitinib%E2%80%93a+novel+targeted+approach+to+treating+cancer&doi=10.1038%2Fnrc1506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: Gefitinib - a novel targeted approach to treating cancer</span></div><div class="casAuthors">Herbst, Roy S.; Fukuoka, Masahiro; Baselga, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">956-965</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Twenty years after the epidermal growth factor receptor (EGFR) was identified as a potential anticancer target, the EGFR inhibitor gefitinib (Iressa; AstraZeneca) has been approved for the treatment of patients with advanced non-small-cell lung cancer in many countries.  Studies have indicated its potential for treating patients with other types of solid tumors.  Investigation of gefitinib has not only increased our knowledge about the biol. of EGFR signaling, but is contributing to our evolving understanding of which tumors are EGFR dependent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8V2qzLg_-rrVg90H21EOLACvtfcHk0ljt7TfMlklwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsLjJ&md5=03a6c08b947bc3e0ca59e1b92fa8d214</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrc1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1506%26sid%3Dliteratum%253Aachs%26aulast%3DHerbst%26aufirst%3DR.%2BS.%26aulast%3DFukuoka%26aufirst%3DM.%26aulast%3DBaselga%26aufirst%3DJ.%26atitle%3DGefitinib%25E2%2580%2593a%2520novel%2520targeted%2520approach%2520to%2520treating%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D956%26epage%3D965%26doi%3D10.1038%2Fnrc1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappuzzo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciuleanu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stelmakh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicenas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szczesna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juhasz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteban, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugger, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melezínek, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klingelschmitt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klughammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span> <span> </span><span class="NLM_article-title">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">521</span>– <span class="NLM_lpage">529</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(10)70112-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2FS1470-2045%2810%2970112-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20493771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=521-529&author=F.+Cappuzzoauthor=T.+Ciuleanuauthor=L.+Stelmakhauthor=S.+Cicenasauthor=A.+Szczesnaauthor=E.+Juhaszauthor=E.+Estebanauthor=O.+Molinierauthor=W.+Bruggerauthor=I.+Melez%C3%ADnekauthor=G.+Klingelschmittauthor=B.+Klughammerauthor=G.+Giaccone&title=Erlotinib+as+maintenance+treatment+in+advanced+non-small-cell+lung+cancer%3A+a+multicentre%2C+randomised%2C+placebo-controlled+phase+3+study&doi=10.1016%2FS1470-2045%2810%2970112-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Erlotinib as maintenance treatment in advanced non-small-cell lung cancer: a multicentre, randomised, placebo-controlled phase 3 study</span></div><div class="casAuthors">Cappuzzo, Federico; Ciuleanu, Tudor; Stelmakh, Lilia; Cicenas, Saulius; Szczesna, Aleksandra; Juhasz, Erzsebet; Esteban, Emilio; Molinier, Olivier; Brugger, Wolfram; Melezinek, Ivan; Klingelschmitt, Gaelle; Klughammer, Barbara; Giaccone, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-529</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">First-line chemotherapy for advanced non-small-cell lung cancer (NSCLC) is usually limited to four to six cycles.  Maintenance therapy can delay progression and prolong survival.  The oral epidermal growth factor receptor (EGFR) tyrosine-kinase inhibitor erlotinib has proven efficacy and tolerability in second-line NSCLC.  We designed the phase 3, placebo-controlled Sequential Tarceva in Unresectable NSCLC (SATURN; BO18192) study to assess use of erlotinib as maintenance therapy in patients with non-progressive disease following first-line platinum-doublet chemotherapy.  Between Dec., 2005, and May, 2008, 1949 patients were included in the run-in phase (four cycles of platinum-based chemotherapy).  At the end of the run-in phase, 889 patients who did not have progressive disease were entered into the main study, and were randomly allocated using a 1:1 adaptive randomisation method through a third-party interactive voice response system to receive erlotinib (150 mg/day; n=438) or placebo (n=451) until progression or unacceptable toxicity.  Patients were stratified by EGFR immunohistochem. status, stage, Eastern Cooperative Oncol. Group performance status, chemotherapy regimen, smoking history, and region.  Co-primary endpoints were progression-free survival (PFS) in all analysable patients irresp. of EGFR status, and PFS in patients whose tumors had EGFR protein overexpression, as detd. by immunohistochem.  This study is registered with www.ClinicalTrials.gov, no. NCT00556712.  884 patients were analysable for PFS; 437 in the erlotinib group and 447 in the placebo group.  After a median follow-up of 11.4 mo for the erlotinib group and 11.5 mo for the placebo group, median PFS was significantly longer with erlotinib than with placebo: 12.3 wk for patients in the erlotinib group vs. 11.1 wk for those in the placebo group (HR 0.71, 95% CI 0.62-0.82; p<0.0001).  PFS was also significantly longer in patients with EGFR-pos. immunohistochem. who were treated with erlotinib (n=307) compared with EGFR-pos. patients given placebo (n=311; median PFS 12.3 wk in the erlotinib group vs 11.1 wk in the placebo group; HR 0.69, 0.58-0.82; p<0.0001).  The most common grade 3 or higher adverse events were rash (37 [9%] of 443 patients in the erlotinib group vs none of 445 in the placebo group) and diarrhoea (seven [2%] of 443 patients vs none of 445).  Serious adverse events were reported in 47 patients (11%) on erlotinib compared with 34 patients (8%) on placebo.  The most common serious adverse event was pneumonia (seven cases [2%] with erlotinib and four [<1%] with placebo).  Maintenance therapy with erlotinib for patients with NSCLC is well tolerated and significantly prolongs PFS compared with placebo.  First-line maintenance with erlotinib could be considered in patients who do not progress after four cycles of chemotherapy.  Funding: F Hoffmann-La Roche Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ1stlOmTgyrVg90H21EOLACvtfcHk0ljt7TfMlklwDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVCksb0%253D&md5=384f65f7c8ad10ca34d2d0fed7183484</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2810%2970112-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252810%252970112-1%26sid%3Dliteratum%253Aachs%26aulast%3DCappuzzo%26aufirst%3DF.%26aulast%3DCiuleanu%26aufirst%3DT.%26aulast%3DStelmakh%26aufirst%3DL.%26aulast%3DCicenas%26aufirst%3DS.%26aulast%3DSzczesna%26aufirst%3DA.%26aulast%3DJuhasz%26aufirst%3DE.%26aulast%3DEsteban%26aufirst%3DE.%26aulast%3DMolinier%26aufirst%3DO.%26aulast%3DBrugger%26aufirst%3DW.%26aulast%3DMelez%25C3%25ADnek%26aufirst%3DI.%26aulast%3DKlingelschmitt%26aufirst%3DG.%26aulast%3DKlughammer%26aufirst%3DB.%26aulast%3DGiaccone%26aufirst%3DG.%26atitle%3DErlotinib%2520as%2520maintenance%2520treatment%2520in%2520advanced%2520non-small-cell%2520lung%2520cancer%253A%2520a%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%2520phase%25203%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2010%26volume%3D11%26spage%3D521%26epage%3D529%26doi%3D10.1016%2FS1470-2045%2810%2970112-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muzumdar, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorans, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noll, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sprick, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trumpp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span> <span> </span><span class="NLM_article-title">Survival of pancreatic cancer cells lacking KRAS function</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">1090</span> <span class="refDoi"> DOI: 10.1038/s41467-017-00942-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fs41467-017-00942-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=29061961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC1M7js1ymtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=M.+D.+Muzumdarauthor=P.+Y.+Chenauthor=K.+J.+Doransauthor=K.+M.+Chungauthor=A.+Bhutkarauthor=E.+Hongauthor=E.+M.+Nollauthor=M.+R.+Sprickauthor=A.+Trumppauthor=T.+Jacks&title=Survival+of+pancreatic+cancer+cells+lacking+KRAS+function&doi=10.1038%2Fs41467-017-00942-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Survival of pancreatic cancer cells lacking KRAS function</span></div><div class="casAuthors">Muzumdar Mandar Deepak; Chen Pan-Yu; Dorans Kimberly Judith; Chung Katherine Minjee; Bhutkar Arjun; Hong Erin; Jacks Tyler; Muzumdar Mandar Deepak; Muzumdar Mandar Deepak; Chen Pan-Yu; Jacks Tyler; Noll Elisa M; Sprick Martin R; Trumpp Andreas; Noll Elisa M; Sprick Martin R; Trumpp Andreas; Jacks Tyler</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1090</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Activating mutations in the proto-oncogene KRAS are a hallmark of pancreatic ductal adenocarcinoma (PDAC), an aggressive malignancy with few effective therapeutic options.  Despite efforts to develop KRAS-targeted drugs, the absolute dependence of PDAC cells on KRAS remains incompletely understood.  Here we model complete KRAS inhibition using CRISPR/Cas-mediated genome editing and demonstrate that KRAS is dispensable in a subset of human and mouse PDAC cells.  Remarkably, nearly all KRAS deficient cells exhibit phosphoinositide 3-kinase (PI3K)-dependent mitogen-activated protein kinase (MAPK) signaling and induced sensitivity to PI3K inhibitors.  Furthermore, comparison of gene expression profiles of PDAC cells retaining or lacking KRAS reveal a role of KRAS in the suppression of metastasis-related genes.  Collectively, these data underscore the potential for PDAC resistance to even the very best KRAS inhibitors and provide insights into mechanisms of response and resistance to KRAS inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9zxYb5ouxkAwxfk_AGB-EfW6udTcc2eZk694pz5Q4gLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7js1ymtA%253D%253D&md5=c25ffa63b98124b5013f4cf8f8e65048</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00942-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00942-5%26sid%3Dliteratum%253Aachs%26aulast%3DMuzumdar%26aufirst%3DM.%2BD.%26aulast%3DChen%26aufirst%3DP.%2BY.%26aulast%3DDorans%26aufirst%3DK.%2BJ.%26aulast%3DChung%26aufirst%3DK.%2BM.%26aulast%3DBhutkar%26aufirst%3DA.%26aulast%3DHong%26aufirst%3DE.%26aulast%3DNoll%26aufirst%3DE.%2BM.%26aulast%3DSprick%26aufirst%3DM.%2BR.%26aulast%3DTrumpp%26aufirst%3DA.%26aulast%3DJacks%26aufirst%3DT.%26atitle%3DSurvival%2520of%2520pancreatic%2520cancer%2520cells%2520lacking%2520KRAS%2520function%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-00942-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shankar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tien, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siebenaler, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chugh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dommeti, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zelenka-Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apel, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waninger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eyunni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mody, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodrum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesmer, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vats, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitchiaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellison, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar-Sinha, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">An essential role for Argonaute 2 in EGFR-KRAS signaling in pancreatic cancer development</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">2817</span> <span class="refDoi"> DOI: 10.1038/s41467-020-16309-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fs41467-020-16309-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=32499547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWlsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&author=S.+Shankarauthor=J.+C.+Tienauthor=R.+F.+Siebenalerauthor=S.+Chughauthor=V.+L.+Dommetiauthor=S.+Zelenka-Wangauthor=X.+M.+Wangauthor=I.+J.+Apelauthor=J.+Waningerauthor=S.+Eyunniauthor=A.+Xuauthor=M.+Modyauthor=A.+Goodrumauthor=Y.+Zhangauthor=J.+J.+Tesmerauthor=R.+Mannanauthor=X.+Caoauthor=P.+Vatsauthor=S.+Pitchiayaauthor=S.+J.+Ellisonauthor=J.+Shiauthor=C.+Kumar-Sinhaauthor=H.+C.+Crawfordauthor=A.+M.+Chinnaiyan&title=An+essential+role+for+Argonaute+2+in+EGFR-KRAS+signaling+in+pancreatic+cancer+development&doi=10.1038%2Fs41467-020-16309-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">An essential role for Argonaute 2 in EGFR-KRAS signaling in pancreatic cancer development</span></div><div class="casAuthors">Shankar, Sunita; Tien, Jean Ching-Yi; Siebenaler, Ronald F.; Chugh, Seema; Dommeti, Vijaya L.; Zelenka-Wang, Sylvia; Wang, Xiao-Ming; Apel, Ingrid J.; Waninger, Jessica; Eyunni, Sanjana; Xu, Alice; Mody, Malay; Goodrum, Andrew; Zhang, Yuping; Tesmer, John J.; Mannan, Rahul; Cao, Xuhong; Vats, Pankaj; Pitchiaya, Sethuramasundaram; Ellison, Stephanie J.; Shi, Jiaqi; Kumar-Sinha, Chandan; Crawford, Howard C.; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2817</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Both KRAS and EGFR are essential mediators of pancreatic cancer development and interact with Argonaute 2 (AGO2) to perturb its function.  Here, in a mouse model of mutant KRAS-driven pancreatic cancer, loss of AGO2 allows precursor lesion (PanIN) formation yet prevents progression to pancreatic ductal adenocarcinoma (PDAC).  Precursor lesions with AGO2 ablation undergo oncogene-induced senescence with altered microRNA expression and EGFR/RAS signaling, bypassed by loss of p53.  In mouse and human pancreatic tissues, PDAC progression is assocd. with increased plasma membrane localization of RAS/AGO2.  Furthermore, phosphorylation of AGO2Y393 disrupts both the wild-type and oncogenic KRAS-AGO2 interaction, albeit under different conditions.  ARS-1620 (G12C-specific inhibitor) disrupts the KRASG12C-AGO2 interaction, suggesting that the interaction is targetable.  Altogether, our study supports a biphasic model of pancreatic cancer development: an AGO2-independent early phase of PanIN formation reliant on EGFR-RAS signaling, and an AGO2-dependent phase wherein the mutant KRAS-AGO2 interaction is crit. for PDAC progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjEHXX-Kh0KLVg90H21EOLACvtfcHk0ljDsbnKA4HtBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWlsb7O&md5=38f738b80e139e28d50421127c3adcc8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16309-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16309-2%26sid%3Dliteratum%253Aachs%26aulast%3DShankar%26aufirst%3DS.%26aulast%3DTien%26aufirst%3DJ.%2BC.%26aulast%3DSiebenaler%26aufirst%3DR.%2BF.%26aulast%3DChugh%26aufirst%3DS.%26aulast%3DDommeti%26aufirst%3DV.%2BL.%26aulast%3DZelenka-Wang%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DX.%2BM.%26aulast%3DApel%26aufirst%3DI.%2BJ.%26aulast%3DWaninger%26aufirst%3DJ.%26aulast%3DEyunni%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DA.%26aulast%3DMody%26aufirst%3DM.%26aulast%3DGoodrum%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTesmer%26aufirst%3DJ.%2BJ.%26aulast%3DMannan%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DVats%26aufirst%3DP.%26aulast%3DPitchiaya%26aufirst%3DS.%26aulast%3DEllison%26aufirst%3DS.%2BJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DKumar-Sinha%26aufirst%3DC.%26aulast%3DCrawford%26aufirst%3DH.%2BC.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DAn%2520essential%2520role%2520for%2520Argonaute%25202%2520in%2520EGFR-KRAS%2520signaling%2520in%2520pancreatic%2520cancer%2520development%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26doi%3D10.1038%2Fs41467-020-16309-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Du, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span> <span> </span><span class="NLM_article-title">Epidermal Growth Factor Receptor-Targeting Peptide Nanoparticles Simultaneously Deliver Gemcitabine and Olaparib To Treat Pancreatic Cancer with Breast Cancer 2 (BRCA2) Mutation</span>. <i>ACS Nano</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">10785</span>– <span class="NLM_lpage">10796</span>, <span class="refDoi"> DOI: 10.1021/acsnano.8b01573</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsnano.8b01573" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFeis7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2018&pages=10785-10796&author=C.+Duauthor=Y.+Qiauthor=Y.+Zhangauthor=Y.+Wangauthor=X.+Zhaoauthor=H.+Minauthor=X.+Hanauthor=J.+Langauthor=H.+Qinauthor=Q.+Shiauthor=Z.+Zhangauthor=X.+Tianauthor=G.+J.+Andersonauthor=Y.+Zhaoauthor=G.+Nieauthor=Y.+Yang&title=Epidermal+Growth+Factor+Receptor-Targeting+Peptide+Nanoparticles+Simultaneously+Deliver+Gemcitabine+and+Olaparib+To+Treat+Pancreatic+Cancer+with+Breast+Cancer+2+%28BRCA2%29+Mutation&doi=10.1021%2Facsnano.8b01573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal Growth Factor Receptor-Targeting Peptide Nanoparticles Simultaneously Deliver Gemcitabine and Olaparib To Treat Pancreatic Cancer with Breast Cancer 2 (BRCA2) Mutation</span></div><div class="casAuthors">Du, Chong; Qi, Yingqiu; Zhang, Yinlong; Wang, Yazhou; Zhao, Xiao; Min, Huan; Han, Xuexiang; Lang, Jiayan; Qin, Hao; Shi, Quanwei; Zhang, Zhengkui; Tian, Xiaodong; Anderson, Greg J.; Zhao, Ying; Nie, Guangjun; Yang, Yinmo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Nano</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">10785-10796</span>CODEN:
                <span class="NLM_cas:coden">ANCAC3</span>;
        ISSN:<span class="NLM_cas:issn">1936-0851</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pancreatic cancer (PCa) is one of the most lethal malignancies, with a 5 yr survival rate of less than 8%.  Current treatment regiments have a low response rate in unselected patients.  However, the subgroup of PCa patients with BRCA mutations may benefit from poly-ADP-ribose polymerase inhibitors (PARPi) due to their biol. properties in DNA repair.  Dose-limiting toxicity in normal tissues is frequently obsd. when PARPi are combined with other chemotherapies, and the co-delivery of two drugs to tumor sites at an adequate concn. is challenging.  To address this issue, we have engineered an epidermal growth factor receptor (EGFR) targeting (with GE11 peptide) self-assembly amphiphilic peptide nanoparticle (GENP) to co-deliver gemcitabine and the PARPi olaparib to treat BRCA mutant PCa.  The GENP was relatively stable, exhibited high encapsulation efficiency, and could coordinately release the two drugs in tumor milieu.  Gemcitabine and olaparib showed strong synergistic actions in optimized conditions in vitro.  The nanoparticle prolonged the half-life of both drugs and resulted in their tumor accumulation at the optimal therapeutic ratio in vivo.  The drug-loaded nanoparticles were able to significantly suppress tumor growth in a murine PCa model with minimal side effects.  Drug co-delivery of DNA damaging agents and PARP inhibitors via the GENP represents a promising approach for treatment of pancreatic cancers with mol. defects in the DNA repair pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4-Q4EYVi_D7Vg90H21EOLACvtfcHk0liNWFQpnLuaBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFeis7jI&md5=5319397ebeedfa5bc177f52c7076874e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsnano.8b01573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsnano.8b01573%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DMin%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DLang%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DAnderson%26aufirst%3DG.%2BJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DNie%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DEpidermal%2520Growth%2520Factor%2520Receptor-Targeting%2520Peptide%2520Nanoparticles%2520Simultaneously%2520Deliver%2520Gemcitabine%2520and%2520Olaparib%2520To%2520Treat%2520Pancreatic%2520Cancer%2520with%2520Breast%2520Cancer%25202%2520%2528BRCA2%2529%2520Mutation%26jtitle%3DACS%2520Nano%26date%3D2018%26volume%3D12%26spage%3D10785%26epage%3D10796%26doi%3D10.1021%2Facsnano.8b01573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Discovery and biological evaluation of proteolysis targeting chimeras (PROTACs) as an EGFR degrader based on osimertinib and lenalidomide</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">127167</span> <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.bmcl.2020.127167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=32317208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnsV2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&author=K.+Heauthor=Z.+Zhangauthor=W.+Wangauthor=X.+Zhengauthor=X.+Wangauthor=X.+Zhang&title=Discovery+and+biological+evaluation+of+proteolysis+targeting+chimeras+%28PROTACs%29+as+an+EGFR+degrader+based+on+osimertinib+and+lenalidomide&doi=10.1016%2Fj.bmcl.2020.127167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and biological evaluation of proteolysis targeting chimeras (PROTACs) as an EGFR degraders based on osimertinib and lenalidomide</span></div><div class="casAuthors">He, Kailun; Zhang, Zhuo; Wang, Wenbing; Zheng, Xiaoliang; Wang, Xiaoju; Zhang, Xingxian</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">127167</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) is one of the important and valuable drug targets.  Overexpression of EGFR is assocd. with the development of many types of cancer.  In this study, three PROTACs small mols. (16a-16c) were designed, synthesized and evaluated for their cytotoxicity against the growth in different NSCLC cell line and the degrdn. effect.  The bioassay results indicated that 16c has a good inhibition in PC9 cells and H1975 cells, and the corresponding IC50 value was 0.413 μM and 0.657 μM, resp.  Western blotting results demonstrated that compd. 16c could serve as an effective EGFRdel19-targeting degrader in PC9 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgut0gW1TkiLVg90H21EOLACvtfcHk0liNWFQpnLuaBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnsV2qt74%253D&md5=bf63e55c59176f262c8569824417b2ef</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127167%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DDiscovery%2520and%2520biological%2520evaluation%2520of%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520as%2520an%2520EGFR%2520degrader%2520based%2520on%2520osimertinib%2520and%2520lenalidomide%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26doi%3D10.1016%2Fj.bmcl.2020.127167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, X. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, A. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S. Q.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent epidermal growth factor receptor (EGFR) degraders by proteolysis targeting chimera (PROTAC)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>189</i></span>, <span class="NLM_elocation-id">112061</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.ejmech.2020.112061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=31951960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFCltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=189&publication_year=2020&author=H.+Zhangauthor=H.+Y.+Zhaoauthor=X.+X.+Xiauthor=Y.+J.+Liuauthor=M.+Xinauthor=S.+Maoauthor=J.+J.+Zhangauthor=A.+X.+Luauthor=S.+Q.+Zhang&title=Discovery+of+potent+epidermal+growth+factor+receptor+%28EGFR%29+degraders+by+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1016%2Fj.ejmech.2020.112061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent epidermal growth factor receptor (EGFR) degraders by proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Zhang, Hao; Zhao, Hong-Yi; Xi, Xiao-Xiao; Liu, Yan-Jie; Xin, Minhang; Mao, Shuai; Zhang, Jun-Jie; Lu, A-Xin; Zhang, San-Qi</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">189</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112061</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Herein, the discovery of small mol. EGFR degraders based on the proteolysis targeting chimera (PROTAC) strategy was reported.  In the present study, 13 EGFR degraders contg. pyrido[3,4-d]pyrimidine moiety I [X = CH2C(O)NHCH2CH2C(O), CH2CH2CH2CH2C(O), CH2CH2CH2CH2CH2C(O), etc.], II [C = CH2C(O)NHCH2CH2C(O), CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2C(O), CH2CH2OCH2CH2OCH2CH2, CH2CH2CH2CH2CH2CH2C(O), etc.] were designed and synthesized.  Promising PROTACs I [X = CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2CH2CH2C(O)] induced degrdn. of EGFR in HCC827 cells with the DC50 values of 45.2 and 34.8 nM, resp.  Cellular protein-controlling machinery ubiquitin proteasome system (UPS) was involved in the degrdn. process.  Furthermore, the degraders I [X = CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2CH2CH2C(O)] could significantly induce the apoptosis of HCC827 cells and arrest the cells in G1 phase.  These findings demonstrated that compds. I [X = CH2CH2CH2CH2CH2CH2CH2C(O)] and III [X = CH2CH2CH2CH2CH2CH2C(O)] could serve as effective EGFRdel19-targeting degraders in HCC827 cells. v.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonJUw1b6N-77Vg90H21EOLACvtfcHk0liNWFQpnLuaBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFCltr8%253D&md5=8f8a1fbc8df721332a2a2b5d6bbc7875</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112061%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DH.%2BY.%26aulast%3DXi%26aufirst%3DX.%2BX.%26aulast%3DLiu%26aufirst%3DY.%2BJ.%26aulast%3DXin%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%2BJ.%26aulast%3DLu%26aufirst%3DA.%2BX.%26aulast%3DZhang%26aufirst%3DS.%2BQ.%26atitle%3DDiscovery%2520of%2520potent%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520degraders%2520by%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D189%26doi%3D10.1016%2Fj.ejmech.2020.112061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of selective degraders of EGFR(L858R/T790M) mutant</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>192</i></span>, <span class="NLM_elocation-id">112199</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1016%2Fj.ejmech.2020.112199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=32171162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvFyltbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2020&author=X.+Zhangauthor=F.+Xuauthor=L.+Tongauthor=T.+Zhangauthor=H.+Xieauthor=X.+Luauthor=X.+Renauthor=K.+Ding&title=Design+and+synthesis+of+selective+degraders+of+EGFR%28L858R%2FT790M%29+mutant&doi=10.1016%2Fj.ejmech.2020.112199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of selective degraders of EGFRL858R/T790M mutant</span></div><div class="casAuthors">Zhang, Xin; Xu, Fang; Tong, Linjiang; Zhang, Tao; Xie, Hua; Lu, Xiaoyun; Ren, Xiaomei; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112199</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of PROTAC (proteolysis targeting chimera) based selective EGFRL858R/T790M (leucine 858 to arginine 858 mutation and threonine 790 to methionine 790) mutant degraders were designed and synthesized.  One of the most potent compds. (I) effectively and selectively degraded EGFRL858R/T790M with an DC50 value of 5.9 nM, while did not show obvious effect on the wild-type protein.  Further mechanism investigation revealed that the degrdn. was mediated by ubiquitin proteasome pathway.  Compd. I could be utilized as an initial lead mol. for development of new EGFRL858R/T790M degrader based therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoDW2mGkQNzbVg90H21EOLACvtfcHk0liLbchv6HYUwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvFyltbk%253D&md5=e8fdd02842f6cfd94e362f3a53a39096</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112199%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520selective%2520degraders%2520of%2520EGFR%2528L858R%252FT790M%2529%2520mutant%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D192%26doi%3D10.1016%2Fj.ejmech.2020.112199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angelier, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planck, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascal, J. M.</span></span> <span> </span><span class="NLM_article-title">Purification of human PARP-1 and PARP-1 domains from <i>Escherichia coli</i> for structural and biochemical analysis</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>780</i></span>,  <span class="NLM_fpage">209</span>– <span class="NLM_lpage">226</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-270-0_13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1007%2F978-1-61779-270-0_13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=21870263" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=780&publication_year=2011&pages=209-226&author=M.+F.+Angelierauthor=J.+L.+Planckauthor=K.+M.+Serventauthor=J.+M.+Pascal&title=Purification+of+human+PARP-1+and+PARP-1+domains+from+Escherichia+coli+for+structural+and+biochemical+analysis&doi=10.1007%2F978-1-61779-270-0_13"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-270-0_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-270-0_13%26sid%3Dliteratum%253Aachs%26aulast%3DAngelier%26aufirst%3DM.%2BF.%26aulast%3DPlanck%26aufirst%3DJ.%2BL.%26aulast%3DServent%26aufirst%3DK.%2BM.%26aulast%3DPascal%26aufirst%3DJ.%2BM.%26atitle%3DPurification%2520of%2520human%2520PARP-1%2520and%2520PARP-1%2520domains%2520from%2520Escherichia%2520coli%2520for%2520structural%2520and%2520biochemical%2520analysis%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D780%26spage%3D209%26epage%3D226%26doi%3D10.1007%2F978-1-61779-270-0_13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshikawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kukimoto-Niino, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terazawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semba, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the altered drug sensitivities of non-small cell lung cancer-associated mutants of human epidermal growth factor receptor</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1038/onc.2012.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fonc.2012.21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=22349823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvFKl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=27-38&author=S.+Yoshikawaauthor=M.+Kukimoto-Niinoauthor=L.+Parkerauthor=N.+Handaauthor=T.+Teradaauthor=T.+Fujimotoauthor=Y.+Terazawaauthor=M.+Wakiyamaauthor=M.+Satoauthor=S.+Sanoauthor=T.+Kobayashiauthor=T.+Tanakaauthor=L.+Chenauthor=Z.+J.+Liuauthor=B.+C.+Wangauthor=M.+Shirouzuauthor=S.+Kawaauthor=K.+Sembaauthor=T.+Yamamotoauthor=S.+Yokoyama&title=Structural+basis+for+the+altered+drug+sensitivities+of+non-small+cell+lung+cancer-associated+mutants+of+human+epidermal+growth+factor+receptor&doi=10.1038%2Fonc.2012.21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the altered drug sensitivities of non-small cell lung cancer-associated mutants of human epidermal growth factor receptor</span></div><div class="casAuthors">Yoshikawa, S.; Kukimoto-Niino, M.; Parker, L.; Handa, N.; Terada, T.; Fujimoto, T.; Terazawa, Y.; Wakiyama, M.; Sato, M.; Sano, S.; Kobayashi, T.; Tanaka, T.; Chen, L.; Liu, Z.-J.; Wang, B.-C.; Shirouzu, M.; Kawa, S.; Semba, K.; Yamamoto, T.; Yokoyama, S.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) has an essential role in multiple signaling pathways, including cell proliferation and migration, through extracellular ligand binding and subsequent activation of its intracellular tyrosine kinase (TK) domain.  The non-small cell lung cancer (NSCLC)-assocd. EGFR mutants, L858R and G719S, are constitutively active and oncogenic.  They display sensitivity to TK inhibitors, including gefitinib and erlotinib.  In contrast, the secondary mutation of the gatekeeper residue, T790M, reportedly confers inhibitor resistance on the oncogenic EGFR mutants.  In this study, our biochem. analyses revealed that the introduction of the T790M mutation confers gefitinib resistance on the G719S mutant.  The G719S/T790M double mutant has enhanced activity and retains high gefitinib-binding affinity.  The T790M mutation increases the ATP affinity of the G719S mutant, explaining the acquired drug resistance of the double mutant.  Structural analyses of the G719S/T790M double mutant, as well as the wild type and the G719S and L858R mutants, revealed that the T790M mutation stabilizes the hydrophobic spine of the active EGFR-TK conformation.  The Met790 side chain of the G719S/T790M double mutant, in the apo form and gefitinib- and AMPPNP-bound forms, adopts different conformations that explain the accommodation of these ligands.  In the L858R mutant structure, the active-site cleft is expanded by the repositioning of Phe723 within the P-loop.  Notably, the introduction of the F723A mutation greatly enhanced the gefitinib sensitivity of the wild-type EGFR in vivo, supporting our hypothesis that the expansion of the active-site cleft results in enhanced gefitinib sensitivity.  Taken together, our results provide a structural basis for the altered drug sensitivities caused by distinct NSCLC-assocd. EGFR mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr7hEqvdS5i7Vg90H21EOLACvtfcHk0liLbchv6HYUwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvFKl&md5=f9b29aac7bcb52cc0605f157f7f14809</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.21%26sid%3Dliteratum%253Aachs%26aulast%3DYoshikawa%26aufirst%3DS.%26aulast%3DKukimoto-Niino%26aufirst%3DM.%26aulast%3DParker%26aufirst%3DL.%26aulast%3DHanda%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DFujimoto%26aufirst%3DT.%26aulast%3DTerazawa%26aufirst%3DY.%26aulast%3DWakiyama%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DM.%26aulast%3DSano%26aufirst%3DS.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DZ.%2BJ.%26aulast%3DWang%26aufirst%3DB.%2BC.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DKawa%26aufirst%3DS.%26aulast%3DSemba%26aufirst%3DK.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructural%2520basis%2520for%2520the%2520altered%2520drug%2520sensitivities%2520of%2520non-small%2520cell%2520lung%2520cancer-associated%2520mutants%2520of%2520human%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D27%26epage%3D38%26doi%3D10.1038%2Fonc.2012.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wienken, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaske, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbauer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhr, S.</span></span> <span> </span><span class="NLM_article-title">Protein-binding assays in biological liquids using microscale thermophoresis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">100</span> <span class="refDoi"> DOI: 10.1038/ncomms1093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=10.1038%2Fncomms1093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=20981028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;key=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&author=C.+J.+Wienkenauthor=P.+Baaskeauthor=U.+Rothbauerauthor=D.+Braunauthor=S.+Duhr&title=Protein-binding+assays+in+biological+liquids+using+microscale+thermophoresis&doi=10.1038%2Fncomms1093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-binding assays in biological liquids using microscale thermophoresis</span></div><div class="casAuthors">Wienken Christoph J; Baaske Philipp; Rothbauer Ulrich; Braun Dieter; Duhr Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein interactions inside the human body are expected to differ from the situation in vitro.  This is crucial when investigating protein functions or developing new drugs.  In this study, we present a sample-efficient, free-solution method, termed microscale thermophoresis, that is capable of analysing interactions of proteins or small molecules in biological liquids such as blood serum or cell lysate.  The technique is based on the thermophoresis of molecules, which provides information about molecule size, charge and hydration shell.  We validated the method using immunologically relevant systems including human interferon gamma and the interaction of calmodulin with calcium.  The affinity of the small-molecule inhibitor quercetin to its kinase PKA was determined in buffer and human serum, revealing a 400-fold reduced affinity in serum.  This information about the influence of the biological matrix may allow to make more reliable conclusions on protein functionality, and may facilitate more efficient drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOCqf6rsXbPv6BottYdIAWfW6udTcc2eZgLjeJdwDvJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D&md5=5660d96475370e7f46ea6682c9398cb1</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fncomms1093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1093%26sid%3Dliteratum%253Aachs%26aulast%3DWienken%26aufirst%3DC.%2BJ.%26aulast%3DBaaske%26aufirst%3DP.%26aulast%3DRothbauer%26aufirst%3DU.%26aulast%3DBraun%26aufirst%3DD.%26aulast%3DDuhr%26aufirst%3DS.%26atitle%3DProtein-binding%2520assays%2520in%2520biological%2520liquids%2520using%2520microscale%2520thermophoresis%26jtitle%3DNat.%2520Commun.%26date%3D2010%26volume%3D1%26doi%3D10.1038%2Fncomms1093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i63"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00649">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_83724"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00649?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00649</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR spectra of compounds in the text, HRMS spectra of <b>MP-GC</b>, <b>MP-OC</b>, <b>MP-GV</b>, <b>MP-OV</b>, <b>DP-C 1-4</b> and <b>DP-V 1-4</b> and HPLC for purity determination of <b>DP-C 1-4</b>, <b>DP-V 1-4</b>, <b>MP-GC</b>, <b>MP-OC</b>, <b>MP-GV</b>, and <b>MP-OV</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_001.pdf">jm1c00649_si_001.pdf (3.86 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00649/suppl_file/jm1c00649_si_002.csv">jm1c00649_si_002.csv (4.85 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00649&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-11%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00649%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00649" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac0770dec3c5e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
