
<html lang="en"     class="pb-page"  data-request-id="b70c341e-122b-4c82-a056-e10dc3fc0275"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.6b01018;issue:issue:10.1021/jmcmar.2017.60.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration" /></meta><meta name="dc.Creator" content="Luca  Sartori" /></meta><meta name="dc.Creator" content="Ciro  Mercurio" /></meta><meta name="dc.Creator" content="Federica  Amigoni" /></meta><meta name="dc.Creator" content="Anna  Cappa" /></meta><meta name="dc.Creator" content="Giovanni  Fagá" /></meta><meta name="dc.Creator" content="Raimondo  Fattori" /></meta><meta name="dc.Creator" content="Elena  Legnaghi" /></meta><meta name="dc.Creator" content="Giuseppe  Ciossani" /></meta><meta name="dc.Creator" content="Andrea  Mattevi" /></meta><meta name="dc.Creator" content="Giuseppe  Meroni" /></meta><meta name="dc.Creator" content="Loris  Moretti" /></meta><meta name="dc.Creator" content="Valentina  Cecatiello" /></meta><meta name="dc.Creator" content="Sebastiano  Pasqualato" /></meta><meta name="dc.Creator" content="Alessia  Romussi" /></meta><meta name="dc.Creator" content="Florian  Thaler" /></meta><meta name="dc.Creator" content="Paolo  Trifiró" /></meta><meta name="dc.Creator" content="Manuela  Villa" /></meta><meta name="dc.Creator" content="Stefania  Vultaggio" /></meta><meta name="dc.Creator" content="Oronza A.  Botrugno" /></meta><meta name="dc.Creator" content="Paola  Dessanti" /></meta><meta name="dc.Creator" content="Saverio  Minucci" /></meta><meta name="dc.Creator" content="Elisa  Zagarrí" /></meta><meta name="dc.Creator" content="Daniele  Carettoni" /></meta><meta name="dc.Creator" content="Lucia  Iuzzolino" /></meta><meta name="dc.Creator" content="Mario  Varasi" /></meta><meta name="dc.Creator" content="Paola  Vianello" /></meta><meta name="dc.Description" content="Lysine specific demethylase 1 KDM1A (LSD1) regulates histone methylation and it is increasingly recognized as a potential therapeutic target in oncology. We report on a high-throughput screening ca..." /></meta><meta name="Description" content="Lysine specific demethylase 1 KDM1A (LSD1) regulates histone methylation and it is increasingly recognized as a potential therapeutic target in oncology. We report on a high-throughput screening ca..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 27, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01018" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01018" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01018" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01018" /></link>
        
    
    

<title>Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01018" /></meta><meta property="og:title" content="Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0017.jpeg" /></meta><meta property="og:description" content="Lysine specific demethylase 1 KDM1A (LSD1) regulates histone methylation and it is increasingly recognized as a potential therapeutic target in oncology. We report on a high-throughput screening campaign performed on KDM1A/CoREST, using a time-resolved fluorescence resonance energy transfer (TR-FRET) technology, to identify reversible inhibitors. The screening led to 115 hits for which we determined biochemical IC50, thus identifying four chemical series. After data analysis, we have prioritized the chemical series of N-phenyl-4H-thieno[3, 2-b]pyrrole-5-carboxamide for which we obtained X-ray structures of the most potent hit (compound 19, IC50 = 2.9 μM) in complex with the enzyme. Initial expansion of this chemical class, both modifying core structure and decorating benzamide moiety, was directed toward the definition of the moieties responsible for the interaction with the enzyme. Preliminary optimization led to compound 90, which inhibited the enzyme with a submicromolar IC50 (0.162 μM), capable of inhibiting the target in cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01018"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01018">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01018&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01018&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01018&amp;href=/doi/10.1021/acs.jmedchem.6b01018" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 1673-1692</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01483" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01019" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Thieno[3,2-<i>b</i>]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Luca++Sartori">Luca Sartori</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ciro++Mercurio">Ciro Mercurio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Federica++Amigoni">Federica Amigoni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anna++Cappa">Anna Cappa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giovanni++Fag%C3%A1">Giovanni Fagá</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Raimondo++Fattori">Raimondo Fattori</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elena++Legnaghi">Elena Legnaghi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Ciossani">Giuseppe Ciossani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Mattevi">Andrea Mattevi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9523-7128" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Meroni">Giuseppe Meroni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Loris++Moretti">Loris Moretti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>¶</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-9851-2965" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Valentina++Cecatiello">Valentina Cecatiello</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sebastiano++Pasqualato">Sebastiano Pasqualato</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alessia++Romussi">Alessia Romussi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Florian++Thaler">Florian Thaler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paolo++Trifir%C3%B3">Paolo Trifiró</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Manuela++Villa">Manuela Villa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefania++Vultaggio">Stefania Vultaggio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oronza+A.++Botrugno">Oronza A. Botrugno</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Dessanti">Paola Dessanti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Saverio++Minucci">Saverio Minucci</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elisa++Zagarr%C3%AD">Elisa Zagarrí</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniele++Carettoni">Daniele Carettoni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lucia++Iuzzolino">Lucia Iuzzolino</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mario++Varasi">Mario Varasi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Vianello">Paola Vianello</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>◆</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-9100-2711" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Biology and Biotechnology, University of Pavia, Via Ferrata 1, 27100 Pavia, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Crystallography Unit, Department of Experimental Oncology, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">IFOM- The FIRC Institute of Molecular Oncology Foundation, Via Adamello 16, 20139 Milano, Italy</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Department of Biosciences, University of Milan, Via Celoria, 26, 20133 Milano, Italy</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Axxam, via A. Meucci 3, 20091 Bresso, Milano, Italy</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">∇</span> <span class="aff-text">Genextra Group, DAC s.r.l., Via Adamello 16, 20139 Milano, Italy</span></div><div class="corresp-info"><strong>*</strong>For L.S.: phone, +39 02 9437 5121; E-mail, <a href="/cdn-cgi/l/email-protection#ec80998f8dc29f8d9e98839e85ac858a8381c28999"><span class="__cf_email__" data-cfemail="aec2dbcdcf80ddcfdcdac1dcc7eec7c8c1c380cbdb">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For P.V.: phone, +39 02 9437 5030; E-mail, <a href="/cdn-cgi/l/email-protection#f282939d9e93dc849b939c979e9e9db29b949d9fdc9787"><span class="__cf_email__" data-cfemail="0272636d6e632c746b636c676e6e6d426b646d6f2c6777">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01018&amp;href=/doi/10.1021%2Facs.jmedchem.6b01018" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 1673–1692</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 10, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 July 2016</li><li><span class="item_label"><b>Published</b> online</span>27 February 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 March 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01018" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01018</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1673%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLuca%2BSartori%252C%2BCiro%2BMercurio%252C%2BFederica%2BAmigoni%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D5%26contentID%3Dacs.jmedchem.6b01018%26title%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2Bas%2BNew%2BReversible%2BInhibitors%2Bof%2BHistone%2BLysine%2BDemethylase%2BKDM1A%252FLSD1.%2BPart%2B1%253A%2BHigh-Throughput%2BScreening%2Band%2BPreliminary%2BExploration%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1692%26publicationDate%3DMarch%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01018"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2859</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">39</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01018" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 1: High-Throughput Screening and Preliminary Exploration&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Luca&quot;,&quot;last_name&quot;:&quot;Sartori&quot;},{&quot;first_name&quot;:&quot;Ciro&quot;,&quot;last_name&quot;:&quot;Mercurio&quot;},{&quot;first_name&quot;:&quot;Federica&quot;,&quot;last_name&quot;:&quot;Amigoni&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Cappa&quot;},{&quot;first_name&quot;:&quot;Giovanni&quot;,&quot;last_name&quot;:&quot;Fagá&quot;},{&quot;first_name&quot;:&quot;Raimondo&quot;,&quot;last_name&quot;:&quot;Fattori&quot;},{&quot;first_name&quot;:&quot;Elena&quot;,&quot;last_name&quot;:&quot;Legnaghi&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Ciossani&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Mattevi&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Meroni&quot;},{&quot;first_name&quot;:&quot;Loris&quot;,&quot;last_name&quot;:&quot;Moretti&quot;},{&quot;first_name&quot;:&quot;Valentina&quot;,&quot;last_name&quot;:&quot;Cecatiello&quot;},{&quot;first_name&quot;:&quot;Sebastiano&quot;,&quot;last_name&quot;:&quot;Pasqualato&quot;},{&quot;first_name&quot;:&quot;Alessia&quot;,&quot;last_name&quot;:&quot;Romussi&quot;},{&quot;first_name&quot;:&quot;Florian&quot;,&quot;last_name&quot;:&quot;Thaler&quot;},{&quot;first_name&quot;:&quot;Paolo&quot;,&quot;last_name&quot;:&quot;Trifiró&quot;},{&quot;first_name&quot;:&quot;Manuela&quot;,&quot;last_name&quot;:&quot;Villa&quot;},{&quot;first_name&quot;:&quot;Stefania&quot;,&quot;last_name&quot;:&quot;Vultaggio&quot;},{&quot;first_name&quot;:&quot;Oronza&quot;,&quot;last_name&quot;:&quot;A. Botrugno&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Dessanti&quot;},{&quot;first_name&quot;:&quot;Saverio&quot;,&quot;last_name&quot;:&quot;Minucci&quot;},{&quot;first_name&quot;:&quot;Elisa&quot;,&quot;last_name&quot;:&quot;Zagarrí&quot;},{&quot;first_name&quot;:&quot;Daniele&quot;,&quot;last_name&quot;:&quot;Carettoni&quot;},{&quot;first_name&quot;:&quot;Lucia&quot;,&quot;last_name&quot;:&quot;Iuzzolino&quot;},{&quot;first_name&quot;:&quot;Mario&quot;,&quot;last_name&quot;:&quot;Varasi&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Vianello&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;1673-1692&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01018&quot;},&quot;abstract&quot;:&quot;Lysine specific demethylase 1 KDM1A (LSD1) regulates histone methylation and it is increasingly recognized as a potential therapeutic target in oncology. We report on a high-throughput screening campaign performed on KDM1A/CoREST, using a time-resolved fluorescence resonance energy transfer (TR-FRET) technology, to identify reversible inhibitors. The screening led to 115 hits for which we determined biochemical IC50, thus identifying four chemical series. After data analysis, we have prioritized the chemical series of N-phenyl-4H-thieno[3, 2-b]pyrrole-5-carboxamide for which we obtained X-ray structures of the most potent hit (compound 19, IC50 = 2.9 μM) in complex with the enzyme. Initial expansion of this chemical class, both modifying core structure and decorating benzamide moiety, was directed toward the definition of the moieties responsible for the interaction with the enzyme. Preliminary optimization led to compound 90, which inhibited the enzyme with a submicromolar IC50 (0.162 μM), capable of inhibit&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01018&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01018" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01018&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01018" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01018&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01018" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01018&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01018&amp;href=/doi/10.1021/acs.jmedchem.6b01018" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01018" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01018" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01018%26sid%3Dliteratum%253Aachs%26pmid%3D28186755%26genre%3Darticle%26aulast%3DSartori%26date%3D2017%26atitle%3DThieno%255B3%252C2-b%255Dpyrrole-5-carboxamides%2Bas%2BNew%2BReversible%2BInhibitors%2Bof%2BHistone%2BLysine%2BDemethylase%2BKDM1A%252FLSD1.%2BPart%2B1%253A%2BHigh-Throughput%2BScreening%2Band%2BPreliminary%2BExploration%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D5%26spage%3D1673%26epage%3D1692%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/jmcmar.2017.60.issue-5/20170309/jmcmar.2017.60.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Lysine specific demethylase 1 KDM1A (LSD1) regulates histone methylation and it is increasingly recognized as a potential therapeutic target in oncology. We report on a high-throughput screening campaign performed on KDM1A/CoREST, using a time-resolved fluorescence resonance energy transfer (TR-FRET) technology, to identify reversible inhibitors. The screening led to 115 hits for which we determined biochemical IC<sub>50</sub>, thus identifying four chemical series. After data analysis, we have prioritized the chemical series of <i>N</i>-phenyl-4<i>H</i>-thieno[3, 2-<i>b</i>]pyrrole-5-carboxamide for which we obtained X-ray structures of the most potent hit (compound <b>19</b>, IC<sub>50</sub> = 2.9 μM) in complex with the enzyme. Initial expansion of this chemical class, both modifying core structure and decorating benzamide moiety, was directed toward the definition of the moieties responsible for the interaction with the enzyme. Preliminary optimization led to compound <b>90</b>, which inhibited the enzyme with a submicromolar IC<sub>50</sub> (0.162 μM), capable of inhibiting the target in cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31254" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31254" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histone lysine demethylases are responsible for post-translational modifications of the histone tails,<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> representing some of the most important partners in eukaryotic chromatin regulation.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Among them, KDM1A (LSD1), homologous to polyamine oxidase (PAO) and monoamine oxidase (MAO) A and B,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> catalyzes mono- and dimethyl histone 3 lysine 4 (H3K4me1, H3K4me2) and H3K9 demethylation.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> High expression of KDM1A has been reported in acute myeloid leukemia,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> lymphoid neoplasm,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> prostate cancer,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> ER-negative breast cancer,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and neuroblastoma.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Moreover, its high expression correlates with poor prognosis in several tumor types, such as breast cancer,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> prostate cancer,<a onclick="showRef(event, 'ref8 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref10">(8, 10)</a> non-small cell lung cancer,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and hepatocellular carcinoma.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> A role of KDM1A into conferring stem cell-like characteristics in breast cancer has also been proposed.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">From the discovery of tranylcypromine (<b>1</b> in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) as an irreversible inhibitor of KDM1A,<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> a number of analogues bearing the cyclopropyl moiety has been described<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20">(16-20)</a> and three clinical candidates are currently under investigation, two of them being disclosed, ORY1001/RG-6016<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> as compound <b>6</b> and GSK2879552<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22, 23)</a> as compound <b>7</b>, whereas INCB059872 from Incyte is still structurally unknown.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> We also have contributed to the field with the recent identification of a potent oral selective inhibitor [(1<i>S</i>,2<i>R</i>)-<b>3</b>, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>]<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> able to significantly prolong survival time in a murine promyelocytic leukemia model.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KDM1A inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Several examples of reversible KDM1A inhibitors have been reported in the literature (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>),<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20 ref25 ref26 ref27 ref28">(16-20, 25-28)</a> the development of which appears to be in the preclinical stage. A high-throughput screening (HTS) using time-resolved fluorescence energy transfer assay (TR-FRET) on KDM1A has also been described, although no inhibitors were reported.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Here we describe the identification of four chemical series that are active on KDM1A through the screening of a representative subset of our chemical collection (ca. 34000 commercially available compounds) and the process used to select the thieno[3,2-<i>b</i>]pyrrole-5-carboxamide series for further development. A very preliminary exploration allowed us to obtain submicromolar inhibitors. A subsequent extensive medicinal chemistry effort has led to a further optimization of the series, up to the identification of highly selective nanomolar inhibitors, endowed with high potency in cellular assays. This in depth exploration is the subject of the <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01019">accompanying paper</a> (DOI <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01019">10.1021/acs.jmedchem.6b01019</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> The Chemical Collection</h3><div class="NLM_p">The starting point for any HTS activity is a proper compound collection. To perform HTS for our projects, we built a structurally diverse collection, starting from five million commercially available compounds, filtering out molecules on the basis of their physicochemical properties, removal of undesired features,<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a> diversity, and chemotype representativeness, and we obtained a subset of 196000 compounds, which were purchased.</div><div class="NLM_p">This library was designed to represent each selected chemotype (scaffold-based) with a sufficient number of close analogues to obtain preliminary SARs for each hit identified during HTS. Although the chemical diversity is limited, the hit identification process is reinforced with the reduction of false positive rate. The analogues would help in deciding if positive (or negative) results are true values simply by looking at the whole activity range across the cluster. At the same time, the higher the number of cluster members (close analogues), the lower the “chemical diversity”, independently from chemical diversity space definition.</div><div class="NLM_p last">To maximize signal over noise ratio<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and to minimize the total number of assayed compounds still maintaining high number of confirmed hits, the collection has been divided into five subsets. Four of them, containing 8500 chemical diversity representatives each for a total of 34000, have been selected to be assayed in the first step of HTS. Close analogues of hits were pooled from the fifth set (Full Diversity Set), represented by the remaining 162000 compounds. These analogues were used during the hit expansion step. All compounds were plated into 384-well plates and were grouped by similarity, thus generating a sort of “compound cartridges” ready for screening and allowing hit expansions with a minimal effort.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Quantitative High-Throughput Screening</h3><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> Step 1</h4><div class="NLM_p">The screening was conducted on 34000 compounds, the representative subset of our chemical collection obtained from commercially available generic libraries, at the fixed concentration of 10 μM (see <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The KDM1A inhibiting activity was determined using a TR-FRET(time-resolved fluorescence resonance energy transfer) assay. The assay was performed at room temperature (RT) in 384-well plates in a final volume of 30 μL of assay buffer (Tris HCl 50 mM, pH 8.8, NaCl 50 mM, DTT 1 mM, Tween-20 0.01%, protease-free BSA 0.01% w/v) at the enzyme (KDM1A/CoREST) and substrate (histone H3K4 monomethylated) concentrations of 0.079 and 51 nM, respectively. After 60 min at RT, the reaction was stopped with tranylcypromine at the concentration of 300 μM. The primary screening campaign was completed in 5 days and the assays provided an overall Robust <i>z</i> prime factor (RZ′) of 0.85, demonstrating the robustness and the quality of the obtained data. The threshold for hit selection was set at 15% of activity, which corresponded to mean plus three times the standard deviation computed on neutral controls (maximal signal in absence of inhibitor). Compounds with high autofluorescence and therefore interfering with the assay were discarded. Applying these criteria, 320 out of 34000 compounds were identified as hits, corresponding to 0.95% hit rate. The 320 primary hits were then tested at 30, 10, and 3 μM with interplate triplicates. Out of 320 retested compounds, 113 were confirmed with a confirmation rate of 35% and with an overall RZ′ of 0.7. This result was consistent with the threshold applied in primary screening for hit selection. Indeed, the majority of compounds that did not confirm their activity were close to the primary screening threshold, whereas most of the compounds showing high activity in primary screening belonged to the group of confirmed hits. Using these compounds as templates, substructure and similarity searches against the fifth set of the collection was performed with ad hoc developed Pipeline Pilot<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> protocols. The protocols performed iteratively substructure and similarity searches (See <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details) for each of the reconfirmed hits and then collected and merged all the results, thus producing a series of plates, giving us the opportunity to screen an enriched set of compounds similar to those obtained at 10 μM.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. HTS workflow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Step 2</h4><div class="NLM_p last">Nineteen mother plates were selected from the Full Diversity Set to be tested in triplicate at 10 μM to identify hits from analogous compounds present in the library. An overall RZ′ of 0.81 was obtained, and 70 compounds among the 6080 tested were found to be active according to the same threshold of the primary screening, thus producing a hit rate of 1.15% based on all compounds and of 8% based on close analogues.</div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Step 3</h4><div class="NLM_p">Finally, a dose–response experiment was conducted on 115 selected compounds. The compounds were tested in dose–response at eight concentrations ranging from 30 to 0.1 μM with intraplate quadruplicate data points. The overall RZ′ was of 0.76, and 73 active compounds were identified. Among them, 27 compounds showed an IC<sub>50</sub> higher than 30 μM, 32 compounds showed an IC<sub>50</sub> between 10 and 30 μM, and 14 compounds showed an IC<sub>50</sub> lower than 10 μM. Overall, four chemical series and 14 singletons were identified (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. HTS: Chemical Series with Inhibitory Activity</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0011.gif" alt="" id="fx1" /></img><div></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> HTS Results</h3><div class="NLM_p">The four main chemical series identified are reported in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. They were evaluated on the basis of novelty, preliminary SAR evidence, chemical feasibility, and structure-based drug design (SBDD) amenability. Our interest was thus focused on compounds <b>19</b> and <b>21</b>. Compound <b>19</b> (and its furo analogue) showed an interesting preliminary SAR, having 20 analogues with IC<sub>50</sub> ranging from 10 to 30 μM. Compound <b>21</b>, although belonging to a much smaller cluster, has a low molecular weight, comparable to a fragment, with a single-digit micromolar IC<sub>50</sub>, and was considered for further investigation, taking into account its high potential for derivatization.</div><div class="NLM_p last">The chemical series prioritized, thieno(furo)pyrroles <b>19</b> and spiro derivatives <b>21</b>, were selected for further studies to better define their biochemical mechanism of action and to verify their amenability for X-ray crystallography.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Orthogonal Assay, Reversibility, and Selectivity</h3><div class="NLM_p">Compounds <b>19</b>, <b>21</b>, and tranylcypromine <b>1</b> were assessed with an orthogonal KDM1A assay based on horseradish peroxidase (HRP) assay to reduce the probability to pursue false positives and to obtain an independent confirmation of their histone demethylases activity. As reported in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, they inhibited KDM1A/CoREST complex also in the HRP assay, confirming to be genuine KDM1A inhibitors. Different experimental assay conditions in terms of enzyme, substrate concentrations, and incubation time used in the two assays justify the differences between the TR-FRET and HRP reported IC<sub>50</sub>s. Similar differences were observed between HTS and reconfirmation data.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Orthogonal Assay<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">TR-FRET HTS IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">TR-FRET IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">HRP IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">2.9</td><td class="colsep0 rowsep0" align="char" char="±">13.4 ± 1.2</td><td class="colsep0 rowsep0" align="char" char="±">27.1 ± 2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char="±">8.1 ± 3.3</td><td class="colsep0 rowsep0" align="char" char="±">82.4 ± 8.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="±">8.8 ± 2.6</td><td class="colsep0 rowsep0" align="char" char="±">30.6 ± 1.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div></div></div><div class="NLM_p">Because HRP assay is a continuous assay well suited to perform enzymatic inhibition kinetic analysis, it was used to verify reversibility and biochemical mechanism of action of compounds <b>19</b> and <b>21</b>. The reversibility was assayed by measuring the recovery of enzymatic activity after a rapid and large dilution of the enzyme–inhibitor complex in the presence of the substrate. Consistently with its irreversible mechanism of action, incubation of <b>1</b> resulted in complete inactivation of the enzyme, which was not recovered after dilution with the assay buffer containing the substrate (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). On the contrary, when KDM1A/CoREST was incubated with <b>19</b> and <b>21</b>, the recovery of enzymatic activity after dilution was almost complete, confirming for both compounds the reversibility of inhibition.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Compounds <b>19</b> and <b>21</b> reversibly inhibited KDM1A/CoREST enzymatic activity. KDM1A/CoREST complex was incubated at a concentration 100-fold over the concentration required for the enzymatic activity (2000 nM), with concentrations of the inhibitors <b>19</b>, <b>21</b>, and <b>1</b> equivalent to 10-fold of their IC<sub>50</sub>s. After 15 min, the above solution is diluted 100-fold into reaction buffer containing the substrate to initiate the reaction. <b>19</b> and <b>21</b>, contrarily to the irreversible inhibitor <b>1</b>, result in recovery of enzymatic activity. (B) Plot of 1/(Substrate)/1/Rate(μmol/min) for a range of concentrations of <b>19</b>. (C) Plot of 1/(Substrate)/1/Rate(μmol/min)for a range of concentration of <b>21</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">KDM1A/CoREST enzymatic kinetic was assessed at different concentrations of substrate and inhibitor. Initial velocities were fitted to the equations for competitive, noncompetitive, and uncompetitive inhibition, using SigmaPlot (Systat Software, San Jose, CA). As reported in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B,C, Lineweaver–Burk plot indicates a noncompetitive mechanism for <b>19</b> and a competitive mechanism for <b>21</b>. In particular, <b>19</b> showed to be a noncompetitive type inhibitor (α = 1.1)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> with the same affinity for free KDM1A/CoREST complex and the KDM1A/CoREST enzyme substrate complex, with a <i>K</i><sub>i</sub> of 43 μM, while <b>21</b> showed to be a competitive inhibitor, with a <i>K</i><sub>i</sub> of 48.4 μM.</div><div class="NLM_p">Finally, we investigated enzyme selectivity of <b>19</b> and <b>21</b> against KDM1B, MAOA, and MAOB using tranylcypromine <b>1</b> as reference. Both compounds showed high selectivity in the secondary assay, as illustrated in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Selectivity Profile of Compounds <b>19</b> and <b>21</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">KDM1A<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">KDM1B<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">MAOA<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">MAOB<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char="±">13.4 ± 1.2</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">56.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char="±">8.1 ± 3.3</td><td class="colsep0 rowsep0" align="center">nd</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">42.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">8.8 ± 2.6</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">1.06 ± 0.4</td><td class="colsep0 rowsep0" align="center">0.8 ± 0.3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Structural Analysis</h3><div class="NLM_p">The <i>N</i>-[3-(methoxymethyl)phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>19</b>) was successfully crystallized in complex with KDM1A/CoREST construct, see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details, and the X-ray diffraction quality allowed us to solve the complex at a resolution of 3.2 Å, adequate for structural analysis (see <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A and Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Table 1S</a>). We also attempted to produce KDM1A/CoREST crystal in complex with compound <b>21</b>, but all the trials failed. To our knowledge, <b>19</b> is the first small molecule in close contact with FAD, reversibly inhibiting KDM1A, for which the experimentally determined structure of the complex has been reported.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray of <b>19</b>. (A) Inhibitor interactions with target. (B) Comparison of crystallographic structure of <b>19</b> with KDM1A in complex with histone H3 peptide (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>): the superposition of the two structures shows the partial overlap of the thienopyrrole molecule with the atoms of the peptide. Trp699, in cyan, has shifted with respect to its reported position.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Key Interactions</h4><div class="NLM_p">The ligand is located within the catalytic site of the KDM1A enzyme, as seen in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A, with a relatively planar conformation due the electronic delocalization between the two ring systems bridged by the amide group. The inhibitor interaction with the target is mainly hydrophobic with no visible polar contacts. Its core, the thienopyrrole moiety, is caged between residues Val333, Thr335, Leu663, Tyr765, and Thr814. The flavin ring represents the back wall of such cage and the geometry of its contact with the bicyclic system is compatible with an aromatic–aromatic interaction. The second ring system, an aryl group, is packed between residues Val333, Phe542, and Trp699. The exit vector of the <i>meta</i> substituent on the second ring system points toward Gln358, suggesting a possible interaction region.</div><div class="NLM_p">Stability of the complex and its interactions have been evaluated using the molecular dynamics (MD) technique, propaedeutic to structure base drug design setup (see next paragraph).</div><div class="NLM_p last">Molecular details of the noncompetitive inhibition mechanism for <b>19</b> are revealed by the comparison of this crystallographic structure with the one of KDM1A in complex with histone H3 peptide (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> depicted in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B. The superposition of the two structures shows a very partial overlap of the thienopyrrole molecule with the atoms of the peptide, only the oxygen of the methoxymethyl superimpose with the histone, in a noneffective manner. In particular, the ligand is proximal to the region where the dimethylated lysine interacts with the target to undergo the demethylation reaction.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Further comparison of the two KDM1A complexes (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a> vs internal) highlights subtle differences of binding-site residue conformations. The two binding sites are quite identical with the exception of Trp699, which is present with different rotamers. The different conformation of this amino acid is due to the induced fit effect of thienopyrrole ligand binding to the target.</div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Structure-Based Drug Design</h3><div class="NLM_p">To support the development of this class of inhibitors, a specific molecular docking protocol was developed to better reproduce the binding mode of the thienopyrrole chemotype. A procedure was developed by combining docking programs, clustering techniques, and energy estimates (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details). The KDM1A protein conformations used for the docking procedures were those obtained from internal X-ray trials, where the conformation of Trp699 has shifted with respect to its reported position. This effect is induced by the binding of the ligand and in particular is due to the phenyl ring position (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B). The use of these protein structures and the docking protocols gave satisfactory results to reproduce the X-ray crystallographic conformation of <b>19</b> and produced reasonable poses.</div><div class="NLM_p">The conformational space of the ligand inside the target was explored with more than one methodology: Autodock,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Plants,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and Glide<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> produced multiple docking poses. They were clustered with the ACIAP (Autonomus-hirerarchical clustering),<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> a method based on their distances as RMSD between conformations. Finally, one representative pose per family was submitted to binding estimation after refinement (BEAR)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> evaluation.</div><div class="NLM_p">To identify binding site characteristics, such as hydrophobic-interaction pattern and hydrogen bonding propensity, we chose to explore the target protein with a grid-based approach. Autogrid software<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> was used to build different types of energetics maps by applying various atomic probes and the visualization of grid points below a certain threshold allowed us to scan the binding site for favorable interaction regions. Of all obtained maps, the hydrophobic pattern was quite informative and, associated with the electronegative areas, gave useful suggestions for the development of the thienopyrrole series. As seen in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, the ligand is buried into the hydrophobic region.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Favorable interaction regions. Protein surface is displayed as well as the surfaces of the favorable interaction areas for hydrophobic character, in light-yellow, and for the electronegative ones, in red. (A) the inhibitor and cofactor molecules are in sticks representation and their carbon atoms are color-coded cyan (inhibitor) and magenta (cofactor). In light-green, two vectors are depicted from the position where the substituent emerges to the two electronegative regions. (B) The peptide substrate is depicted, and some residues are visible in sticks with carbons in violet.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The analysis of the electrostatics maps reveals two negatively charged regions, one in the upward direction with respect to the thienopyrrole, where the residues Asp379 and Glu383 are found, and one downward, corresponding to residues Asp559, Asp560, and Glu563. In particular, the former corresponds to the site where the guanidine group of the Arg8 of the histone substrate interacts with the enzyme, see <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B. Thus, we decided to grow the chemical class elongating the substituent in position <i>meta</i> of the anilide in search of a favorable interaction regions along the upward direction. In addition, the distance between the benzene carbon in <i>para</i> position and the electronegative region is approximately 12 Å. Ultimately, these modeling suggestions prompted us to elongate the dimethylene chain of compound <b>19</b> into a chain ending with a positively charged group.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Preliminary SAR from HTS</h3><div class="NLM_p">Preliminary SAR indications referred to compound <b>19</b> can be obtained directly from the HTS data (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>): the thienopyrrole bicyclic ring is favored to the furopyrrole analogue (<b>19</b> vs <b>25</b>, <b>22</b> vs <b>29</b>), <i>meta</i> substitution on phenyl ring results in more potent compounds than the <i>ortho</i> analogues (<b>24</b> vs <b>31</b>, <b>22</b> vs <b>33</b>), amido nitrogen substitution with methyl is detrimental for activity (<b>38</b> and <b>40</b>), electron donor groups are preferred to halogens or to electron withdrawing groups.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Preliminary SAR<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0012.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">All the compounds reported in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> have been purchased during chemical collection acquisition.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">Compounds have also been synthesized according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</p></div></div><div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Explorative SAR</h3><div class="NLM_p">A small chemical exploration to further investigate the series represented by compound <b>21</b> has been performed, but none of the synthesized compound reported any inhibitory activity (data not shown) and the explorative activity concluded. For thienopyrroles derivatives instead we initiated a more comprehensive SAR study.</div><div class="NLM_p">We approached the modification on four areas (see <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>):<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">A</td><td colspan="5" valign="top" class="paragraph"><p class="last">: Different heterocycles to replace thienopyrrole;</p></td></tr><tr class="li1"><td valign="top"></td><td colspan="5" valign="top" class="paragraph"><p class="inline">Small substituents on the thienopyrrole;</p></td></tr></table></div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Four areas modified during explorative SAR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">B</td><td colspan="5" valign="top" class="paragraph"><p class="last">: Different chains replacement of the <i>N</i>-methyl group of the thienopyrrole;</p></td></tr><tr class="li1"><td valign="top">C</td><td colspan="5" valign="top" class="paragraph"><p class="last">: Isosters of the amido function;</p></td></tr><tr class="li1"><td valign="top">D</td><td colspan="5" valign="top" class="paragraph"><p class="last">: Variations of the <i>meta</i> substituent of the phenyl group.</p></td></tr></table></div><div class="NLM_p">As demonstrated by X-ray structure, A and B regions are involved in binding/interactions with flavin adenine dinucleotide (FAD), thus they have been explored together.</div><div class="NLM_p">Region D is involved in induced fitting of Trp699 and carries the exit vector toward putative interactions with negatively charged region.</div><div class="NLM_p last">Region C has been explored as a linker replacement.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthetic Chemistry</h3><div class="NLM_p">The appropriate heterocyclic aldehydes (<b>102a</b>–<b>l</b>) were converted into the azido derivatives (<b>103a</b>–<b>l</b>) by reaction with ethyl 2-azidoacetate and potassium ethylate in ethanol. Subsequent cyclization occurred by refluxing in xylene, providing <b>104a</b>–<b>l</b>, which reacted with methyl iodide and sodium hydride in DMF to give <b>105a</b>–<b>l</b>. Ester function could either be directly transformed into amide by reaction with aniline and LiHMDS in THF or hydrolyzed to the acids as in <b>106c</b> in LiOH, which in turn reacted with aniline previous activation to acyl chloride with thionyl chloride (see <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0008.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthesis of 5–5 Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) EtOK, EtOH, 4 h, 0 °C to rt; (b) xylene, reflux, 30 min; (c) (1) NaH, DMF, rt, 30 min to 1 h, (2) MeI, rt; (d) PhNH<sub>2</sub> (up to 1.6 equiv), LiHMDS (up to 2.6 equiv), THF, 0 °C to rt, 2 h; (e) LiOH, H<sub>2</sub>O/EtOH, rt 3 h; (f) (1) SOCl<sub>2</sub>, DCM/THF, 2 h reflux, (2) PhNH<sub>2</sub>, pyridine, rt, overnight.</p></p></figure><div class="NLM_p">Halogenation of thienopyrrole <b>23</b> was performed according to <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Chlorination with <i>N</i>-chlorosuccinimide as reported<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> went exclusively in position 6 on the pyrrole ring, affording 6-chloro-4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>56</b>) with 26% yield. Bromination with <i>N</i>-bromo succinimide has been reported on a similar substrate,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> but in our case it gave an inseparable mixture of the major 2-bromo derivative <b>44</b> and the 6-bromo analogue <b>57</b>. Compound <b>44</b> was alternatively prepared according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, starting from 5-bromothiophene-2-carbaldehyde (<b>102i</b>).</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0009.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Halogenated Thienopyrroles<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NCS (1.05 equiv), DCM, 0 °C to rt overnight; (b) NBS (1.05 equiv), DCM, 0 °C to rt overnight; (c) NIS (1.05 equiv), DCM 0 °C to rt overnight, 18% <b>58</b>, 16% <b>59</b>, 2% <b>60</b>; (d) LDA (3 equiv), THF, −78 °C, 1 h; NFSI (3 equiv), −78 °C to rt 1 h.</p></p></figure><div class="NLM_p">NIS gave a mixture of 6-Iodo (<b>58</b>), 2-Iodo (<b>59</b>), and 2,6-diIodo (<b>60</b>). Fluorination of the thiophene in position 2 (<b>61</b>) was achieved with <i>N</i>-fluorobenzenesulfonimide (NFSI) using LDA as a base. Attempts with NFSI and butyllithium<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> failed to give the desired product.</div><div class="NLM_p">The synthesis of 6-methoxymethyl and 6-methyl thienopyrroles (<b>62</b> and <b>63</b>, respectively) and the synthesis of 2- and 6-(dimethylaminomethyl) thienopyrroles (<b>64</b> and <b>65</b>, respectively) are reported in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p">Thienopyrrole <b>104a</b> was either treated with alkyl halide, benzyl chloride, or phenylsulphonyl chloride, in the presence of NaH in DMF, to give the intermediates <b>105m</b>–<b>q</b>, or was treated with phenylboronic acid and Cu(OAc)<sub>2</sub> in pyridine, to give <b>105p</b>. Subsequent hydrolysis to <b>106m</b>–<b>r</b>, conversion into the corresponding acyl chloride by treatment with thionyl chloride, and reaction with the required amine gave final amides <b>66–91</b>.</div><div class="NLM_p">Compounds <b>92</b>–<b>94</b> and the anilines ArNH<sub>2</sub> in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, when not commercially available, in particular those used to generate <b>90</b> and <b>91</b>, were prepared according to the procedures reported in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a>.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0010.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions. (a) (i) NaH, DMF, rt, 30 min to 1 h, then R1I, rt, or (ii) phenylboronic acid Cu(OAc)<sub>2</sub>, TEA, PYR; (b) LiOH, H<sub>2</sub>O/EtOH, rt 3 h; (c) (1) SOCl<sub>2</sub> (1.3 equiv), THF, 67 °C 2 h, (2) Ar-NH<sub>2</sub> (<b>102a</b>–<b>z</b>, 0.6 equiv), pyridine, rt 90 min; (d) PhNH<sub>2</sub> (up to 1.6 equiv), LiHMDS (up to 2.6 equiv), THF, 0 °C to rt, 2 h.</p></p></figure><div id="sec2_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22">A.1 Variation of the Heterocycle</h4><div class="NLM_p">To verify if the interaction with FAD in the binding pocket could be mimicked by other heterocycles, we replaced the bicycle ring with several other hetero aromatic systems. As illustrated in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>, only the regioisomer <b>45</b> resulted almost equipotent to <b>23</b>, furopyrrole <b>28</b> was more than three times less potent, whereas pyrrolothiazoles (<b>46</b> and <b>47</b>), pyrroloimidazoles (<b>48</b> and <b>50</b>), pyrrolopyrrole (<b>49</b>), pyrrolopyrazole (<b>51</b>), thienothiophene, thienoimidazole (<b>52</b>, <b>93</b>), or benzothiophene completely abolished the inhibitory activity.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Heterocycle Variations</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0013.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t5fn2"><sup>Table b</sup><p class="last">All the compounds have been prepared according to <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, except for <b>92</b><a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and <b>93</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a> for synthetic procedures).</p></div></div><div></div></div></div><div id="sec2_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24">A.2 Ring Substitution</h4><div class="NLM_p">We approached a limited exploration on all the accessible positions of the bicyclic system (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). None of the substitutions demonstrated the opportunity to decorate the core scaffold. Among the residues selected to substitute the pyrrolic nitrogen, only ethyl in compound <b>66</b> improved potency of almost three times when compared to the methyl analogue <b>23</b>, while the other chains were all detrimental for the activity (see 2-aminoethyl, phenyl, phenylsulfonyl, and benzyl in compounds <b>68</b>–<b>71</b>). The reason for this inactivity can be explained by steric hindrance of substituents in position 4 of thienopyrrole moiety: anything larger than the ethyl group in fact goes toward residues Leu663, Tyr765, as suggested in comments to <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>A. Analogously, the proximity of the enzyme wall to the ligand does not allow any atom replacing hydrogens in position 2, 3, and 6 of the thienopyrrole, neither small alkyls nor halogens of various types (<b>44</b>, <b>53</b>–<b>56</b>, <b>58</b>–<b>65</b>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Ring Substitutions</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0014.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div></div><div></div></div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">C Amide Replacement</h4><div class="NLM_p last">In search of new functional groups substituting the amide spacer, we made some isosteric replacements (ester, reverse amide, oxime) without any activity improvement (data not shown).</div></div><div id="sec2_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27">D.1 Phenyl Replacement</h4><div class="NLM_p">The phenyl group was replaced by several nitrogen containing six member rings, as illustrated in <a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a> such as pyridine (<b>72</b>–<b>74</b>) or pyrimidine (<b>75</b>–<b>76</b>), but none of them resulted in being more potent than the parent compound.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Phenyl Replacement</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0015.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div></div><div></div></div></div><div id="sec2_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29">D.2 Benzamide <i>meta</i> Substituted</h4><div class="NLM_p">From the data obtained, we concluded that the benzamido moiety is the only portion allowing substitution, without substantially abolishing the inhibitory activity. Keeping in mind that <b>19</b> resulted among the most interesting derivatives, we investigated several replacements of the methoxy methyl in <i>meta</i> position (see <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>).</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. <i>Meta</i> Substituents</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0016.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup>Table a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div></div><div></div></div><div class="NLM_p">Halogens, methoxy groups, and tertiary amines (<b>22</b>–<b>24</b>, <b>26</b>, <b>77</b>, <b>78</b>), did not influence much the activity, being the chlorine substituted (<b>26</b>) the most active compound, whereas primary amides and bromine analogue were highly detrimental as in <b>79</b> and <b>80</b> (see <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>a). Variation of methoxymethyl brought to the methylthiomethyl derivative <b>32</b> and morpholinomethyl <b>82</b> being slightly more potent, removal of the terminal methyl as in <b>85</b> worsens the activity 4-fold, whereas phenoxymethyl <b>86</b> caused a complete abolishment of inhibitory potency, which was partially recovered with a <i>p</i>-OH substitution <b>84</b> (see <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>b). The introduction of a pyridine (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>c) linked to the phenyl by a −CH<sub>2</sub>O– group (as in <b>88</b>) or by −OCH<sub>2</sub>– (as in <b>87</b>) brought to a slight improvement of the activity, whereas the 4-pyridinol derivative (<b>89</b>) resulted in being less active.</div><div class="NLM_p">When observing the whole series of derivatives, a tendency in potency improvement in the presence of longer chains carrying a distal basic nitrogen emerges. In support of this hypothesis, we observed a clear improvement with derivatives <b>90</b> and <b>91</b>: in both derivatives <i>p</i>-piperidinoxyphenoxy substituent was able to increase the potency down to a submicromolar range (<b>90</b>, IC<sub>50</sub> = 0.162 μM; <b>91</b>, IC<sub>50</sub> = 0.442 μM; <a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>d). Docking experiments of <b>90</b> have been performed using internal produced protein structure, but the presence of two negatively charged regions allowed dual pose orientations. Although the uniqueness of the pose could not be established, in both cases the activity improvement is well justified by the interactions of the piperidine moiety with one aspartate residue (Asp555 or Asp375). In <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a> is reported one of the accessible orientations.</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0007.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. One of the accessible orientations for compound <b>90</b> in extended conformation with interactions with Asp375.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further characterize this chemical class, we analyzed the selectivity profile of the two most potent compounds: <b>90</b> and <b>91</b> using <b>6</b> as positive control. As reported for the initial hit <b>19</b>, they demonstrated a high selectivity for KDM1A.</div><div class="NLM_p">Compounds <b>90</b> and <b>91</b> were evaluated in terms of KDM1A inhibitory activity in acute myeloid leukemia cell line THP-1. This cell line is characterized by the MLL-AF9 translocation and was selected in consideration of the experimental evidence, demonstrating the relevance of KDM1A in sustaining the oncogenic potential of MLL-AF9 leukemia stem cells.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">The absence of evident effect on a global level of histone H3K4 methylation after genetic ablation of KDM1A, both in cancer cells<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and hematopoietic cells,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> let us to deprioritize this experimental read out for cellular KDM1A inhibition experiments.</div><div class="NLM_p">We evaluated instead the compound effects on the expression level of two differentiation markers: CD14 and CD11B (modulated by KDM1A down-regulation as reported in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a>), considering that KDM1A inhibition induces differentiation of human AML cells.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> As reported in <a class="ref internalNav" href="#tbl9" aria-label="Table 9">Table 9</a>, <b>90</b> and <b>91</b>, as well as <b>6</b>, induced an increase of mRNA expression of CD14 and CD11B selected genes, similarly to KDM1A down-regulation by short hairpin RNA(shRNA). These results demonstrate the compounds’ ability to block KDM1A activity in cells and to induce differentiation of THP-1 cells. To additionally characterize their effects on leukemia cells, cell cycle analysis and assessment of apoptosis induction were performed in THP-1 cells. As reported in Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Figure S2</a>, treatment for 72 h at the dose of 1 μM for <b>90</b> and <b>91</b> and at the dose of 0.5 μM for <b>6</b> did not induce apoptosis and cell death, consistentely with the expected cytostatic pro-differentiation mechanism, due to KDM1A inhibition in AML cells.<a onclick="showRef(event, 'ref25 ref43'); return false;" href="javascript:void(0);" class="ref ref25 ref43">(25, 43)</a> Finally, the anticlonogenic potential of <b>90</b> was determined in the human MLL-AF9 leukemia cell line THP-1. As reported in the <a class="ref internalNav" href="#tbl10" aria-label="Table 10">Table 10</a>, at a dose of 5 μM, <b>90</b> was able to inhibit the colony formation in THP-1 cells. Moreover, <b>90</b>, as well as <b>6</b>, induces differentiation in THP-1 cells as demonstrated by expression increases of CD86 in the cells, recovered at the end of semisolid culture experiment.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. Biochemical and Cellular Characterization of Most Potent Derivatives</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">KDM1A<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">KDM1B<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">MAOA<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">MAOB<a class="ref internalNav" href="#t9fn1" aria-label="a">a</a> IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">CD14<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a> fold increase</th><th class="colsep0 rowsep0" align="center" char="±">CD11b<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a> fold increase</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>90</b></td><td class="colsep0 rowsep0" align="char" char="±">0.16 ± 0.02</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">7.16 ± 3.5</td><td class="colsep0 rowsep0" align="char" char="±">4.79 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">4.61 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>91</b></td><td class="colsep0 rowsep0" align="char" char="±">0.44 ± 0.03</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">7.60 ± 6.1</td><td class="colsep0 rowsep0" align="char" char="±">2.76 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">4.76 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b><a class="ref internalNav" href="#t9fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">0.0006.0 ± 0.0001</td><td class="colsep0 rowsep0" align="center">43.4 ± 9.6</td><td class="colsep0 rowsep0" align="center">48.3 ± 7.9</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char="±">5 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">11.1 ± 1.45</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup>a</sup><p class="last">Data are expressed as the mean of at least two determinations ± standard deviation.</p></div><div class="footnote" id="t9fn2"><sup>b</sup><p class="last">Fold increase mRNA expression, reported as mean values of several replicates (≥2) at the fixed dose of 1 μM, measured after 24 h of compound treatment.</p></div><div class="footnote" id="t9fn3"><sup>c</sup><p class="last">Fold increase mRNA expression, reported as mean values of several replicates (≥2) at the fixed dose of 0.5 μM, measured after 24 h of compound treatment.</p></div></div></div><div class="NLM_table-wrap" id="tbl10"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 10. Anticlonogenic Potential of Most Potent Derivatives</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">THP_1<a class="ref internalNav" href="#t10fn1" aria-label="a">a</a> colony formation inhibition</th><th class="colsep0 rowsep0" align="center" char="±">CD86 expression<a class="ref internalNav" href="#t10fn2" aria-label="b">b</a> fold increase</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>90</b><a class="ref internalNav" href="#t10fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">29.5 ± 5</td><td class="colsep0 rowsep0" align="char" char="±">14.3 ± 2.49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b><a class="ref internalNav" href="#t10fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char="±">70 ± 2.8</td><td class="colsep0 rowsep0" align="char" char="±">55 ± 8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t10fn1"><div class="footnote" id="t10fn1"><sup>a</sup><p class="last">Colonies were counted after 13 days. Percentage of inhibition is referenced to the vehicle (DMSO) treated cells, and the data reported are the mean of several replicates plus the standard deviation (≥2).</p></div><div class="footnote" id="t10fn2"><sup>b</sup><p class="last">Fold increase mRNA expression, reported as mean values of several replicates (≥2) measured a the end of experiment.</p></div><div class="footnote" id="t10fn3"><sup>c</sup><p class="last">Compound was tested at the dose of 5 μM.</p></div><div class="footnote" id="t10fn4"><sup>d</sup><p class="last">Compound was tested at the dose of 0.5 μM.</p></div></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41555" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41555" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We ran an HTS (TR-FRET assay) on 34000 compounds in search of reversible inhibitors of the demethylase KDM1A, identifying four chemical classes inhibiting the enzyme at a low micromolar concentration. We prioritized thieno[3,2-<i>b</i>]pyrrole-5-carboxamides and 3,8-diazaspiro[4.5]decanes, demonstrating the biochemical mechanism of action for each hit selected in both the series.</div><div class="NLM_p last">The binding mode of a representative compound of thienopyrrole series was experimentally determined through X-ray crystallography and represents the first structural knowledge of KDM1A reversible inhibitors. With this notion, we expanded the class by modifying the scaffold, the amido linker, and the aryl on the benzamide moiety. We discovered that residues bearing a terminal basic moiety located at about 18 Å away from thienopyrrole centroid drastically improved the activity. Further SAR exploration of the chemical series is reported in <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01019">part 2</a> (DOI <a href="https://doi.org/10.1021/10.1021/acs.jmedchem.6b01019">10.1021/acs.jmedchem.6b01019</a>),<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> in which we approached the ambiguity of the binding poses.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42612" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42612" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> HTS Assay Conditions (KDM1A Enzyme Inhibition)</h3><div class="NLM_p">To identify small molecule reversible inhibitor of KDM1A enzyme, TR-FRET technology was used to follow KDM1A enzymatic activity. In our assay, a biotinylated H3K4me1 peptide substrate underwent demethylation and the demethylated products were then captured by the Eu-labeled antibody (Eu-Ab) and U Light streptavidin (U Light-SA), thus bringing the Eu donor and U Light acceptor dye molecules into close proximity.</div><div class="NLM_p">Upon irradiation at 340 nm, the energy from the Eu donor was transferred to the U Light acceptor dye which, in turn, generated light at 665 nm. The intensity of the light emission was proportional to the level of biotinylated reaction product. Once identified, the best configuration of the assay in 384 format, optimizing the reaction and the detection conditions, assay adaptation to the automated screening-compatible conditions was performed, including the analysis of intraday variability and the spatial uniformity assessment. The overall RZ′ mean for the day 1 and day 2 plates respectively of 0.79 and 0.89 indicated the robustness of the identified conditions and protocol. Inter- and intraplate variability analysis identified no issues, the variability observed being below the threshold of 20%. RZ′ is defined as<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/jm-2016-01018d_m001.gif" alt="" /></img></span>RSD = robust standard deviation; ⟨ ⟩ = median of the signal values for the indicate control wells of a plate (NC for neutral control, IC for inhibitor control).</div><div class="NLM_p last">The KDM1A inhibiting activity was determined using a TR-FRET assay (time-resolved fluorescence resonance energy transfer, Lance Ultra Demethylase Technology, PerkinElmer, Waltham, MA, USA), which comprises a Europium chelate donor dye (TRF0404, PerkinElmer, Waltham, MA, USA), together with U-<i>Light</i> (TR0102, PerkinElmer, Waltham, MA, USA), a small molecular weight acceptor dye with a red-shifted fluorescent emission, and a biotinylated 21 amino acids histone H3-derived monomethylated peptide (H3K4me) [Lys(Me1)4]-histone H3 (1–21)-GGK(biotin), (64355, Anaspec, Fremont, CA, USA) as substrate. The intensity of the light emission is proportional to the level of biotinylated reaction product. The complex of human recombinant KDM1A/CoREST protein was produced in <i>Escherichia coli</i> as separate proteins and copurified as previously described.<a onclick="showRef(event, 'ref34 ref46'); return false;" href="javascript:void(0);" class="ref ref34 ref46">(34, 46)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Demethylase Assay Conditions</h3><div class="NLM_p last">The enzymatic assay was performed in 384-well white plates (6007290 OptiPlate, PerkinElmer, Waltham, MA, USA) in a final volume of 30 μL of assay buffer (Tris HCl 50 mM, pH 8.8, NaCl 50 mM, DTT 1 mM, Tween-20 0.01%, protease-free BSA 0.01% w/v) as follows: 10 μL of 237 nM KDM1A/CoREST protein were first incubated 10 min at RT with 10 μL of 3-fold concentrated compounds (prediluted in assay buffer at 1.5% DMSO). The reaction was then started adding with 10 μL of 150 nM histone H3K4 monomethylated substrate. After 60 min at RT, the reaction was stopped with 10 μL of 1.2 mM tranylcypromine (P8511-1G, Sigma-Aldrich, St. Louis, MO 63103, USA).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Detection Step Conditions</h3><div class="NLM_p last">Ten μL of detection mix containing 1 nM Eu-antibody and 4 nM U-<i>Light</i>–streptavidin in 1× Lance detection buffer (TRF0404, TR0102, CR97100, PerkinElmer, Waltham, MA, USA) were dispensed on 40 μL stopped reaction. The resulting mixture was incubated in the dark for 1 h at RT. Then, TR-FRET signal was read by a fluorimeter (PHERAstar<sup>Plus</sup>, BMG LABTECH GmbH, Germany) (excitation 337 nm, emission 665 and 620 nm, delay time 100 μs, window time 200 μs). Compound activity and IC<sub>50</sub> values in dose–response experiments were calculated on TR-FRET signal at 665 nm using Genedata Screener 10 (Genedata, Basel, Switzerland).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Routine Assay Conditions (KDM1A Enzyme Inhibition)</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Demethylase Assay Conditions</h4><div class="NLM_p">KDM1A/CoREST (0.25 nM) protein and compound in 100% DMSO were added in a final volume of 48 μL of assay buffer (Tris HCl 50 mM, pH 8.8, NaCl 50 mM, DTT 1 mM, Tween-20 0.01%) to each well of a 96-well half-area flat-bottom white plate (3693 Costar, Sigma-Aldrich, St. Louis, M, USA).</div><div class="NLM_p last">Demethylase reaction was started by the addition of 50 nM histone H3K4 monomethylated. After 20 min at RT, 300 μM tranylcypromine (P8511-1G, Sigma-Aldrich, St. Louis, MO 63103) was added to stop the reaction.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Detection Step Conditions</h4><div class="NLM_p last">Ten μL of the assay mixture was transferred from the original plate into a 384-well white plate (6007290 OptiPlate, PerkinElmer, Waltham, MA, USA) and 10 μL of the detection mix containing 2 nM Eu-antibody and 10 nM U-<i>Light</i>–Streptavidin in 1× Lance detection buffer (TRF0404, TR0102, CR97100, PerkinElmer, Waltham, MA, USA). The resulting mixture was incubated in the dark for 1 h at RT. Then, TR-FRET signal was read by a fluorimeter (Infinite F200, Tecan, Männedorf, Swirzerland) (excitation 320 nm, emission 665 and 620 nm, delay time 50 μs, window time 100 μs).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Reversibility and Michaelis–Menten Analysis</h3><div class="NLM_p">Biochemical characterization of reversibility and <i>K</i><sub>i</sub> was performed using the KDM1A/CoREST assay horseradish peroxidase. The complex of human recombinant KDM1A/CoREST protein was produced in <i>Escherichia coli</i> as separate proteins and copurified. The experiments were performed in 96-well half-area white plates (catalogue 3693, Corning, Corning, NY) using a monomethylated H3-K4 peptide containing 21 amino acids (custom synthesis done by Thermo Scientific) as substrate and in a 40 μL volume of 50 mM TRIS-HCl, pH 8.0, and 0.05 mg/mL BSA buffer. The peptide purity was >95% as checked by analytical high-pressure liquid chromatography and mass spectrometry. The demethylase activity was estimated under aerobic conditions and at RT by measuring the release of H<sub>2</sub>O<sub>2</sub> produced during the catalytic process by the Amplex UltraRed detection system coupled with horseradish peroxidase (HRP). Briefly, 20 nM of KDM1A/CoREST complex was incubated at RT for 15 min in the absence and/or the presence of various concentrations of the inhibitors, 50 μM Amplex UltraRed (Life Technologies), and 0.023 μM HRP (Sigma) in 50 mM Tris–HCl, pH 8.0, and 0.05 mg/mL BSA. The inhibitors were tested twice in duplicates at each concentration. Tranylcypromine (Sigma) was used as control. After preincubation of the enzyme with the inhibitor, the reaction was initiated by addition of 4.5 μM of monomethylated H3-K4 peptide. The conversion of the Amplex Ultra Red reagent to Amplex UltroxRed was monitored by fluorescence (excitation at 510 nm, emission at 595 nm) for 12 min and by using a microplate reader (Infinite 200, Tecan Group, Switzerland). Arbitrary units were used to measure the level of H<sub>2</sub>O<sub>2</sub> produced in the absence and/or in the presence of inhibition. The maximum demethylase activity of KDM1A/CoREST was obtained in the absence of inhibitors and corrected for background fluorescence in the absence of KDM1A/CoREST. The IC<sub>50</sub> was calculated using GraphPad Prism version 4.0 (GraphPad Software, San Diego, CA).</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Reversibility</h4><div class="NLM_p last">Reversibility was determined using jump dilution. Specifically KDM1A/CoRest complex was incubated at a concentration 100-fold over the concentration required for the enzymatic activity with saturating concentrations of the inhibitor (10-fold of its biochemical IC<sub>50</sub>). After 15 min of incubation, a 100-fold dilution into reaction buffer containing the H3K4me substrate was then performed to initiate the reaction. The reaction progress curves were followed and compared to a positive control (KDM1A/CoREST enzyme complex without inhibitor). The irreversible inhibitor <b>1</b> was used as a control of irreversible inhibition.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Michealis–Menten Analysis</h3><div class="NLM_p last">The <i>K</i><sub>i</sub> of compound <b>19</b> and <b>21</b> was determined using a Michaelis–Menten kinetic analysis across multiple concentrations of both compounds. Initial velocities of biochemical assay were then plotted toward substrate enzyme concentrations and fitted to the equations for competitive, noncompetitive, and uncompetitive inhibition, using SigmaPlot (Systat Software, San Jose, CA)</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Bioluminescent-Coupled Assay for Monoamine Oxidases (MAO-Glo Assay)</h3><div class="NLM_p last">The MAO Glo Assay from Promega (catalogue V1402, Promega, Madison, WI) was used to measure the effect of inhibitors on MAO A and MAO B activity. Human recombinant MAO A and MAO B from SIGMA (M7316 and M7441) have been used for the screening The assay was performed at RT in 50 μL (25 μL reaction solution +25 μL detection reagent) in 96-well half-area white plates (catalogue 3693, Corning, Corning, NY). Luminescence was measured after 20 min incubation in the dark using a microplate reader (Infinite F200, Tecan Group, Switzerland) with an integration time of 0.25 s per well. Then 16 ng/μL MAO A or 20 ng/μL MAO B were incubated with different inhibitor concentrations (from 0.004 to 100 μM) for 15 min at RT in Promega MAO Buffer or Promega MAO B buffer (MAO Glo Assay kit, catalogue no. V1402, Promega, Madison, WI). After 30 min of incubation, the reaction was stopped with the Promega detection reagent. All compounds were tested twice and IC<sub>50</sub> values were calculated using GraphPad Prism version 4.0 (GraphPad Software, San Diego, CA).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Chemoinformatics</h3><div class="NLM_p">To perform all chemoinformatics activities, including substructure searches, similarity searches, clustering, and to control all docking activities, Pipeline Pilot (vesion 9.1) protocols were used; Clustering (as implemented in Pipeline Pilot component) and Similarity Searches (similarity threshold between 0.85 and 9.0) were performed using Tanimoto distance and FCEP6 descriptors as implemented in Pipeline Pilot (BIOVIA).</div><div class="NLM_p last">Pipeline Pilot has been used also to verify absence of PAINS inhibitor features<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> in the synthesized compounds by means of SubStructureSearches (SSS) extracted from the PAINS paper.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Cellular Assay</h3><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Cell Type</h4><div class="NLM_p last">Human leukemia THP-1 cells were obtained from the Deutsche Sammlung von Mikroorganismen and Zellkulturen and resulted in being mycoplasma free both by PCR and Mycoalert (Lonza kit). Cells were grown in RPMI supplemented with 10% fetal bovine serum, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, and antibiotics and maintained in a humidified tissue culture incubator at 37 °C in 5% CO<sub>2</sub>.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Retroviral Constructs and Production of Retroviruses</h4><div class="NLM_p">shRNA constructs were prepared in the MSCV-based pLMP retroviral vector. The hairpins used in the study were #12 and #13 (scramble ShRNAs) and 1A#3 and 1A#5 (shRNAs against KDM1A):</div><ul class="NLM_list-list_type-label"><li><p class="inline">#12, AGTACGCGAAGAATACTATCGA</p></li><li><p class="inline">#13, AGTACGTTCAGAATATCATCGAT</p></li><li><p class="inline">1A#3, AAGTGATACTGTGCTTGTCCAC</p></li><li><p class="inline">1A#5, ATCTCAGAAGATGAGTATTATT</p></li></ul><div class="NLM_p last">Supernatants from transfected Phoenix packaging cells were collected at 48 and 72 h after transfection and immediately used for infections.</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Retroviral Transduction of THP-1 Cells</h4><div class="NLM_p last">For THP-1 cells infection, cells were diluted in retroviral supernatants added with 8 μg/mL polybrene and seeded at 10<sup>6</sup> cells/well in 24-well plates. Spin infection was performed for two consecutive days (two infections per day) at 1800 rpm for 45 min. Puromycin selection (3 μg/mL) started 24 h after the last cycle of infection. Assays were carried out 8 days after beginning of infection.</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Immunoblots and Antibodies</h4><div class="NLM_p last">Whole cell extracts were obtained by lysis in SDS buffer (50 mM Tris-HCl, 10% glycerol, 2% SDS). Proteins were separated by SDS-PAGE, blotted onto PVDF membrane and probed with the indicated antibodies. The antibodies used in the study were anti-LSD1(KDM1A) (no. 2139, Cell Signaling), anti-KDM1B (Sigma HPA031269), and anti-HDAC1 (ab7028),</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Transcriptional Assay</h4><div class="NLM_p">Human acute myeloid leukemia THP-1 were grown in RPMI supplemented with 10% fetal bovine serum, 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine, and antibiotics and maintained in a humidified tissue culture incubator at 37 °C in 5% CO<sub>2</sub>. Cells were treated at the fixed dose of 1 μM or with vehicle (DMSO). After 24 h, the cells were collected for RNA analysis. Total RNA was purified using RNeasy Mini Kit (Qiagen, Valencia, CA), quantified, and reverse transcribed. mRNA levels were measured by quantitative RT-PCR (Fast SYBR Green Master mix, Applied Biosystems Foster City, CA) using specific primers and normalized against TBP mRNA. Results are presented as fold induction relative to vehicle treated cells (DMSO). Primers used in this study were:</div><ul class="NLM_list-list_type-label"><li><p class="inline"><b>CD14</b>, GTTCGGAAGACTTATCGACCA–ATCGTCCAGCTCACAAGGTT</p></li><li><p class="inline"><b>CD11B</b>, AACCCCTGGTTCACCTCCT–CATGACATAAGGTCAAGGCTGT</p></li><li><p class="inline"><b>TBP</b>, GCTGGCCCATAGTGATCTTT −CTTCACACGCCAAGAAACAGT</p></li></ul></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Assessment of Apoptosis by Annexin V and Propidiun Iodide Double Staining</h4><div class="NLM_p last">Vehicle (DMSO) or drug-treated cells were stained with annexin V and propidiun iodide, and the percentages of apoptotic cells were determined by flow cytometry. Briefly, cells were resuspended in annexin-V binding buffer (10 mM Hepes, pH 7.4, 150 mM NaCl, 1 mM MgCl<sub>2</sub>, 3.6 mM CaCl<sub>2</sub>, 5 mM KCl) containing 1/50 dilution of annexin-V-APC (eBioscence and 3 μL of PI (Sigma) solution 50 μg/mL in PBS. Cells were freshly analyzed using a FACSCanto II flow cytometer (BD Biosciences).</div></div><div id="sec4_9_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Cell Cycle Analysis</h4><div class="NLM_p last">Cells were treated with vehicle (DMSO) or the reported inhibitors for 72 h, then harvested and fixed dropwise with 70% ethanol and kept at 4 °C before analysis by flow cytometry. Nucleic acids from fixed cells were stained with 1 mL of propidium iodide (50 ug/mL) plus RNase A (Sigma) (250 μg/mL), overnight at 4 °C. DNA content was measured by using a FACSCanto II flow cytometer (BD Biosciences) analyzed using BD FACS Diva Software and ModFit LT3.0.</div></div><div id="sec4_9_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Cell Studies (Clonogenic Assay)</h4><div class="NLM_p last">For the clonogenic assay, THP-1 cells were plated at a density of 250 cells/plate and cultured with drugs at reported doses in MethoCult H4435 Enriched (StemCell Technologies, Vancouver, BC) according with the manufacturer instructions for 13 days. After this time, colonies were counted. Percentage of inhibition is referenced versus the vehicle (DMSO) treated cells.</div></div></div><div id="sec4_9_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Crystallography</h3><div class="NLM_p last">Crystals of KDM1A/CoREST were obtained as described.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> For structure determination of enzyme–inhibitor complexes, crystals were soaked for 3–8 h in a solution consisting of 1.3 M sodium/potassium tartrate, 100 mM <i>N</i>-(2-acetamido)-2-iminodiacetic acid, pH 6.5, 12% (v/v) glycerol, and 0.5e1 mM inhibitor. Diffraction data were measured at beamlines of the European Synchrotron Radiation Facility (Grenoble, France) and the Swiss Light Source (Villigen, Switzerland). Data processing and coordinate refinement were performed with programs of the CCP4 package.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Statistical Analysis</h3><div class="NLM_p">SigmaPlot v11<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> was used.</div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Protein Preparation for Docking</h4><div class="NLM_p last">The crystallographic coordinates of <b>19</b> were prepared as previously reported.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> In summary, the protein was added of hydrogen atoms in Chimera software, optimizing the H-bond networking and minimizing their position with 1000 steps of Steepest Descent. Then with Semiempirical Quantum Mechanics (SQM), first the hydrogen atoms were optimized until a gradient norm of 50 and then the whole protein was fully optimized SQM for 50 steps. The resulting structure was used to develop the inputs necessary for all modeling programs by means of the procedure presented (BASH scripted) in the work.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Favorable Interaction Regions</h4><div class="NLM_p">Favorable interaction areas were derived from the exploration of proteins binding site with various atoms as probes. This procedure maps the interaction energy between a probe, which characterizes the type of interaction and the target. Examples of the probes are aliphatic carbon for the hydrophobic pattern or amine nitrogen for H-bond donor, etc. The Autogrid facility (from Autodock software) was used for this grid-based energy evaluation. Thus, the interaction was recorded placing the probe at each node of the grid in the binding site. Finally, selecting the appropriate energy cutoff and displaying only the grid’s points with energy below such value, the favorable interactions were visualized.</div><div class="NLM_p last">The Autodock plugin<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> for PyMOL software (The PyMOL Molecular Graphics System, version 1.7.4; Schrödinger, LLC) read these Autogrid files and generated the surface representation on the graphical rendering of protein–ligand coordinates.</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Pose Production</h4><div class="NLM_p last">To predict the putative binding pose of ligands interacting with the target, a docking protocol was used. To explore a larger portion of the conformational space for these molecules, results from various docking programs were mixed together. The combination of various methodologies had been previously observed to increase the chances of finding ligand conformation close to the crystallographic structure and thus to enhance of docking performances.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In our study, three programs were used: Autodock,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Plants<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and Glide,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> with the settings described in our previous work.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Pose Clustering</h4><div class="NLM_p">The docking predictions were processed as one ensemble undergoing cluster analysis to reduce data set dimensionality and to identify binding conformational families. The methodology used for this step is Autonomous Hierarchical Agglomerative Cluster Analysis Based Protocol (ACIAP).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> In brief, this method consists on the construction of a full hierarchical tree of the predicted poses using the root mean square deviation (RMSD) between ligand conformations as metric and the average linkage as criterion to determine the distances. Finally, the clustering level to cut the tree was identified with the KGS penalty function, which was based on the average spread of clusters at each tree level. The global minimum of this function ensured the best compromise between the number of clusters and the mean distances between clusters.</div><div class="NLM_p last">This methodology was implemented as automatic computational scripts by means of R software.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Once the procedure was performed, the medoid conformation for each cluster was kept as the representative for the family of binding. In addition, the number of cluster members is also used to judge the quality of a binding mode. The more a solution is found the more probable it is.</div></div><div id="sec4_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Pose Scoring</h4><div class="NLM_p">In our protocol, the surviving poses were then optimized and scored by means of binding estimate after refinement (BEAR).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> With this method, each docking solution was minimized in complex with the target for 2000 steps, then a 100 ps molecular dynamics simulation of sole ligand was run, keeping the protein frozen. The resulting conformation of the complex was reminimized and scored (single point) with GBSA and PBSA. In addition to the interaction energy estimation, another value was used to judge the final pose: the RMSD value of the ligand between the initial conformation and the one after the MD. The higher the RMSD the less stable is the predicted conformation and thus the reliability of the binding pose.</div><div class="NLM_p last">BEAR was implemented in house by means of the Amber software package<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> and Chimera.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a></div></div><div id="sec4_10_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Modeling Protocols Implementation</h4><div class="NLM_p last">Pose production, clustering, and scoring were implemented within the architecture described in our recent work.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The protocol was constructed in Pipeline Pilot<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> with its usual components plus the “CADDD components” that we developed to use the modeling software on a powerful GNU/Linux environment. This consists of PP sub protocols that invoke BASH scripts via SSH calls. The scripts are a combination of command lines, to interface with the wanted program, and scripting variables to automatize the process.</div></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Chemistry</h3><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> General Procedures</h4><div class="NLM_p last">Melting points were determined on a BUCHI melting point B-540 and were uncorrected. Reagents and solvents used, unless stated otherwise, were of commercially available reagent grade quality and were used without further purification. Reaction solvents were used anhydrous, purchased in Sure/Seal bottles stored over molecular sieves. Silica gel flash chromatography purifications were performed on Merck silica gel 60 (0.04–0.063 mm) or Biotage Isolera One system using prepacked columns from Grace and Biotage. Nuclear magnetic resonance spectra (<sup>1</sup>H NMR) were recorded on a Agilent VNMRS 500 MHz spectrometer at 300 K and were referenced in ppm (δ) relative to TMS. Coupling constants (<i>J</i>) are expressed in hertz (Hz). HPLC-MS experiments were performed on an Acquity UPLC apparatus, equipped with a diode array and a Micromass SQD single quadrupole (Waters). The compounds used for biological tests were monitored at 220 and 254 nM, with two different methods, and their purity was found to be at least 95%. Semipreparative HPLC was performed on a Waters apparatus eluting the sample through a BEH C 18 column (5 μm, 19 mm × 100 mm) using decreasingly polar mixtures of water (containing 5% of MeCN and 0.1% of trifluoroacetic acid) and MeCN (containing 0.1% of trifluoroacetic acid).</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> General Procedure to Prepare Benzamido Derivatives by Direct Amidation of Esters</h4><div class="NLM_p last">First, 3 mL (3 mmol) of a 1 M solution of LiHMDS in THF was added dropwise at 0 °C to a stirred solution of 1 mmol of ester and 1.25 mmol of the required aniline. Then the mixture was allowed to spontaneously warm to rt and then was poured into satd NH<sub>4</sub>Cl and extracted with DCM. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated, and the oil residue was purified by column chromatography (eluent: <i>n</i>-hexane/EtOAc, from 2% to 20% of EtOAc), providing the desired compound.</div></div><div id="sec4_11_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Similarly Prepared</h4><div id="sec4_11_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> <i>N</i>-(3-Methoxyphenyl)-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>22</b>) (from <b>105a</b> and <b>3</b>-Methoxyaniline)</h5><div class="NLM_p last">Yield 40% (23 mg), pale-brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.96 (s, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.45–7.41 (m, 1 H), 7.34–7.30 (m, 2 H), 7.26–7.19 (m, 2 H), 6.68–6.61 (m, 1 H), 4.01 (s, 3 H), 3.75 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 287 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 4-Methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>23</b>) (from <b>105a</b> and Aniline)</h5><div class="NLM_p last">Yield 82% (100 mg), beige solid, pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm) 7.69 (br s, 1 H), 7.60 (m, 2 H), 7.38 (m, 2 H), 7.32 (d, <i>J</i> = 5.4 Hz, 1 H), 7.15 (m, 1 H), 6.99 (d, <i>J</i> = 5.4 Hz, 1 H), 6.95 (s, 1 H), 4.11 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 257 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 4-Methyl-<i>N</i>-phenyl-furo[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>28</b>) (from <b>105d</b> and Aniline)</h5><div class="NLM_p last">Yield 83% (65 mg), brown solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.61–7.53 (m, 3 H), 7.51 (d, <i>J</i> = 2.0 Hz, 1 H), 7.40–7.33 (m, 2 H), 7.16–7.09 (m, 1 H), 6.60 (s, 1 H), 6.50 (d, <i>J</i> = 2.0 Hz, 1 H), 4.04 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 241 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 2-Bromo-4-Methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>44</b>) (from <b>105i</b> and Aniline)</h5><div class="NLM_p last">Yield 78% (150 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.05 (s, 1 H), 7.76–7.66 (m, 2 H), 7.59–7.52 (m, 1 H), 7.37–7.29 (m, 2 H), 7.28–7.22 (m, 1 H), 7.11–6.99 (m, 1 H), 3.97 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 336 [M + H]<sup>+</sup>; mp 168–170 °C.</div></div><div id="sec4_11_2_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 6-Methyl-<i>N</i>-phenyl-thieno[2,3-<i>b</i>]pyrrole-5-carboxamide (<b>45</b>) (from <b>105b</b> and Aniline)</h5><div class="NLM_p last">Yield 38% (31 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.67 (br s, 1 H), 7.62–7.57 (m, 2 H), 7.42–7.35 (m, 2 H), 7.18–7.12 (m, 1 H), 7.04–7.00 (m, 1 H), 6.98–6.95 (m, 1 H), 6.93–6.91 (m, 1 H), 4.09 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 257 [M + H]<sup>+</sup>; mp 145–148 °C.</div></div><div id="sec4_11_2_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 4-Methyl-<i>N</i>-phenyl-pyrrolo[3,2-<i>d</i>]thiazole-5-carboxamide (<b>46</b>) (from <b>105d</b> and Aniline)</h5><div class="NLM_p last">Yield 8% (6 mg), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.56 (s, 1 H), 7.74 (br s, 1 H), 7.64–7.57 (m, 2 H), 7.44–7.34 (m, 2 H), 7.21–7.10 (m, 2 H), 4.15 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i>: 258 [M + H]<sup>+</sup>; mp 194–198 °C.</div></div><div id="sec4_11_2_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 4-Methyl-<i>N</i>-phenyl-pyrrolo[2,3-<i>d</i>]thiazole-5-carboxamide (<b>47</b>) (from <b>105e</b> and Aniline)</h5><div class="NLM_p last">Yield 25% (3 mg), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.71 (s, 1 H), 7.72 (br s, 1 H), 7.64–7.57 (m, 2 H), 7.44–7.35 (m, 2 H), 7.20–7.13 (m, 1 H), 6.94 (s, 1 H), 4.25 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i>: 258 [M + H]<sup>+</sup>; mp 150–154 °C dec.</div></div><div id="sec4_11_2_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 1,4-Dimethyl-<i>N</i>-phenyl-pyrrolo[2,3-<i>d</i>]imidazole-5-carboxamide (<b>48</b>) (from <b>105f</b> and Aniline)</h5><div class="NLM_p last">Yield 49% (6 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ = 7.64–7.55 (m, 3 H), 7.50–7.45 (m, 1 H), 7.40–7.34 (m, <i>J</i> = 7.8, 7.8 Hz, 2 H), 7.16–7.10 (m, 1 H), 6.53 (s, 1 H), 4.09 (s, 3 H), 3.79 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 255 [M + H]<sup>+</sup>; mp 176–180 °C (dec 151 °C).</div></div><div id="sec4_11_2_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 4-(Benzenesulfonyl)-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>70</b>) (from <b>105q</b> and Aniline)</h5><div class="NLM_p last">Yield 34% (35 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.12 (s, 1 H) 7.79–7.18 (m, 10 H) 8.13 (m, 2 H) 10.64 (s, 1 H). MS (ESI): <i>m</i>/<i>z</i> 383 [M + H]<sup>+</sup>; mp 187–190 °C.</div></div><div id="sec4_11_2_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 4-Methyl-<i>N</i>-(3-pyridyl)thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>72</b>) (from <b>105a</b> and Pyridin-<b>3</b>-amine)</h5><div class="NLM_p last">Yield 30% (19 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.24 (s, 1 H), 8.93 (m, 1 H), 8.32–8.16 (m, 2 H), 7.55 (d, <i>J</i> = 5.4 Hz, 1 H), 7.42 (m, 1 H), 7.37 (s, 1 H), 7.26 (d, <i>J</i> = 5.4 Hz, 1 H), 4.03 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 258 [M + H]<sup>+</sup>; mp 161–164 °C.</div></div><div id="sec4_11_2_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 4-Methyl-<i>N</i>-(2-pyridyl)thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>73</b>) (from <b>105a</b> and Pyridin-<b>2</b>-amine)</h5><div class="NLM_p last">Yield 32% (52 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.57 (br s, 1 H), 8.35–8.29 (m, 2 H), 7.77 (m, 1 H), 7.35 (d, <i>J</i> = 5.4 Hz, 1 H), 7.10–7.05 (m, 2 H), 6.99 (d, <i>J</i> = 5.4 Hz, 1 H), 4.13 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 258 [M + H]<sup>+</sup>; mp 139–142 °C.</div></div><div id="sec4_11_2_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 4-Methyl-<i>N</i>-(4-pyridyl)thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>74</b>) (from <b>105a</b> and Pyridin-<b>4</b>-amine)</h5><div class="NLM_p last">Yield 34% (55 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.54 (m, 2 H), 7.90 (br s, 1 H), 7.60 (m, 2 H), 7.03 (s, 1 H), 7.37 (d, <i>J</i> = 5.38 Hz, 1 H), 6.99 (d, <i>J</i> = 5.38 Hz, 1 H), 4.12 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 258 [M + H]<sup>+</sup>; mp 128–131 °C.</div></div><div id="sec4_11_2_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 4-Methyl-<i>N</i>-pyrimidin-2-yl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>75</b>) (from <b>105a</b> and Pyrimidin-<b>2</b>-amine)</h5><div class="NLM_p last">Yield 14% (8.9 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.68 (m, 2 H), 8.57 (br s, 1 H), 7.37 (d, <i>J</i> =5.38 Hz, 1 H), 7.08 (s, 1 H), 7.06 (m, 1 H), 6.99 (d, <i>J</i> = 5.38 Hz, 1 H), 4.15 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 259 [M + H]<sup>+</sup>; mp 175–177 °C.</div></div><div id="sec4_11_2_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 4-Methyl-<i>N</i>-pyrimidin-4-yl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>76</b>) (from <b>105a</b> and Pyrimidin-<b>4</b>-amine)</h5><div class="NLM_p last">Yield 18% (11 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.91 (m, 1 H), 8.66 (m, 1 H), 8.48 (br s, 1 H), 8.28 (dd, <i>J</i> = 5.87, 0.98 Hz, 1 H), 7.41 (d, <i>J</i> = 5.38 Hz, 1 H), 7.10 (s, 1 H), 6.99 (d, <i>J</i> = 5.38 Hz, 1 H), 4.13 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 259 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> <i>N</i>-(3-Carbamoylphenyl)-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>79</b>) (from <b>105a</b> and <b>3</b>-Aminobenzamide)</h5><div class="NLM_p last">Yield 50% (71 mg), off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.11 (s, 1 H), 8.27–8.18 (m, 1 H), 7.94 (br s, 1 H), 7.89 (m, 1 H), 7.56 (m, 1 H), 7.53 (d, <i>J</i> = 5.49 Hz, 1 H), 7.44–7.30 (m, 3 H),7.25 (d, <i>J</i> = 5.49 Hz, 1 H), 4.03 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 300 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> <i>N</i>-[3-(HydroxyMethyl)phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>85</b>)</h5><div class="NLM_p last"><b>105a</b> reacted with 3-[[<i>tert</i>-butyl(dimethyl)silyl]oxymethyl]aniline. The thus formed amide was purified by SCX cartridge dissolving the crude in MeOH:DCM 9:1 and eluting with 25 mL of MeOH:DCM 9:1. Purification procedure led to TBDMS protective group removal, yielding <b>85</b> (31%, 30 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.96 (s, 1 H), 7.74 (m, 1 H), 7.60 (m, 1 H), 7.51 (d, <i>J</i> = 5.4 Hz, 1 H), 7.34 (s, 1 H), 7.29–7.21 (m, 2 H), 7.01 (m, 1 H), 5.20 (t, <i>J</i> = 5.6 Hz, 1 H), 4.49 (d, <i>J</i> = 5.4 Hz, 2 H), 4.02 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 287 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> 4-Methyl-<i>N</i>-[3-(phenoxymethyl)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>86</b>) (from <b>105a</b> and 3-(Phenoxymethyl)aniline, <b>86e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 34% (25 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.05 (s, 1 H), 7.86 (s, 1 H), 7.73–7.64 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.37–7.27 (m, 4 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 7.17–7.13 (m, 1 H), 7.04–6.99 (m, 2 H), 6.96–6.91 (m, 1 H), 5.09 (s, 2 H), 4.01 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 363 [M + H]<sup>+</sup>; mp 180–184 °C.</div></div></div></div><div id="sec4_11_2_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> General Procedure to Prepare Benzamido Derivatives via Acyl Chloride</h3><div class="NLM_p">First, 1.5 mmol of SOCl<sub>2</sub> and 2 drops of dry DMF were added at rt to a solution of 1 mmol of carboxylic acid in 5 mL of dry DCM/THF. Then the mixture was stirred for 2 h under reflux. The reaction mixture was then cooled down to rt and added to a solution of 1–1.5 mmol of aniline in 7 mL of pyridine. The mixture was stirred for 30 min at rt and for further 45 min at 60 °C. Then the mixture was diluted with DCM, washed with a saturated NaHCO<sub>3</sub> solution and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give an oil which was purified by flash column chromatography.</div><div id="sec4_11_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> Similarly Prepared</h4><div id="sec4_11_2_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> 4-Methyl-<i>N</i>-phenyl-furo[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>28</b>) (from <b>106c</b> and Aniline)</h5><div class="NLM_p last">Yield 83% (65 mg), brown solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.61–7.53 (m, 3 H), 7.51 (d, <i>J</i> = 2.0 Hz, 1 H), 7.40–7.33 (m, 2 H), 7.16–7.09 (m, 1 H), 6.60 (s, 1 H), 6.50 (d, <i>J</i> = 2.0 Hz, 1 H), 4.04 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 241 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 1-Methyl-<i>N</i>-phenyl-4<i>H</i>-pyrrolo[2,3-<i>d</i>]imidazole-5-carboxamide (<b>50</b>) (from <b>106f</b> and Aniline)</h5><div class="NLM_p last">Yield 4% (7 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.47 (s, 1 H), 9.73 (s, 1 H), 7.80–7.72 (m, 2 H), 7.68–7.62 (m, 1 H), 7.36–7.28 (m, 1 H), 7.08–6.99 (m, 2 H), 3.75 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 241 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 4-Ethyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>66</b>) (from <b>106m</b> and Aniline)</h5><div class="NLM_p last">Yield 90% (62 mg), beige powder. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.00 (s, 1 H), 7.79–7.00 (m, 8 H), 4.58–4.46 (m, 2 H), 1.32 (t, <i>J</i> = 6.8 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 271 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> <i>N</i>-Phenyl-4-propyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>67</b>) (from <b>106n</b> and Aniline)</h5><div class="NLM_p last">Yield 85% (69 mg), beige powder. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.01 (s, 1 H), 7.82–6.97 (m, 8 H), 4.47 (t, <i>J</i> = 7.1 Hz, 2 H), 1.80–1.66 (m, 2 H), 0.81 (t, <i>J</i> = 7.6 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 285 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> <i>N</i>,4-Diphenylthieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>68</b>) (from <b>106p</b> and Aniline)</h5><div class="NLM_p last">Yield 72% (47 mg), beige powder. <sup>1</sup>HNMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.24 (s, 1 H), 7.69–6.82 (m, 13 H). MS (ESI): <i>m</i>/<i>z</i> 319 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> <i>N</i>-Phenyl-4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>69</b>) (from <b>106r</b> and Aniline)</h5><div class="NLM_p last">Yield 59% (43 mg), ochre solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.88 (s, 1 H), 9.94 (s, 1 H), 7.83–6.88 (m, 8 H). MS (ESI): <i>m</i>/<i>z</i> 243 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> 4-Benzyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>71</b>) (from <b>106o</b> and Aniline)</h5><div class="NLM_p last">Yield 58% (45 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.07 (s, 1 H), 7.75–6.95 (m, 13 H), 5.80 (s, 2 H). MS (ESI): <i>m</i>/<i>z</i> 355 [M + Na]<sup>+</sup>; mp 168–170 °C.</div></div><div id="sec4_11_2_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> 4-Methyl-<i>N</i>-[3-(4-methylpiperazin-1-yl)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>77</b>) (from <b>106a</b> and 3-(4-Methylpiperazin-<b>1</b>-yl)aniline)</h5><div class="NLM_p last">Yield 36% (38 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.81 (s, 1 H), 7.51 (d, <i>J</i> = 5.4 Hz, 1 H), 7.40–7.36 (m, 1 H), 7.30 (s, 1 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 7.21–7.18 (m, 1 H), 7.16–7.11 (m, 1 H), 6.68–6.64 (m, 1 H), 4.01 (s, 3 H), 3.18–3.03 (m, 4 H), 2.48–2.42 (m, 4 H), 2.22 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 355 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> <i>N</i>-[3-(Dimethylamino)phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>78</b>) (from <b>106a</b> and <i>N</i>1,<i>N</i>1-Dimethylbenzene-1,3-diamine)</h5><div class="NLM_p last">Yield 27% (24 mg), light-brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.78 (s, 1 H), 7.55–7.46 (m, 1 H), 7.30 (s, 1 H), 7.26–7.22 (m, 1 H), 7.17 (br s, 1 H), 7.10 (br s, 2 H), 6.50–6.42 (m, 1 H), 4.01 (s, 3 H), 2.89 (s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 300 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> <i>N</i>-(3-Bromophenyl)-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>80</b>) (from <b>106a</b> and 3-Bromoaniline)</h5><div class="NLM_p last">Yield 35% (130 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.12 (s, 1 H), 8.11–8.03 (m, 1 H), 7.75–7.66 (m, 1 H), 7.57–7.50 (m, 1 H), 7.38–7.20 (m, 4 H), 4.01 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 336 [M + H]<sup>+</sup>; mp 153–157 °C.</div></div><div id="sec4_11_2_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> 4-Methyl-<i>N</i>-[3-[(1-methyl-3-piperidyl)methoxy]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>81</b>) (from <b>106a</b> and 3-[(1-Methyl-3-piperidyl)methoxy]aniline)</h5><div class="NLM_p last">Yield 51% (58 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.93 (s, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.46–7.41 (m, 1 H), 7.33–7.28 (m, 2 H), 7.26–7.23 (m, 1 H), 7.23–7.18 (m, 1 H), 6.66–6.60 (m, 1 H), 4.01 (s, 3 H), 3.88–3.76 (m, 2 H), 2.86–2.75 (m, 1 H), 2.67–2.57 (m, 1 H), 2.15 (s, 3 H), 2.04–1.96 (m, 1 H), 1.95–1.86 (m, 1 H), 1.85–1.75 (m, 1 H), 1.75–1.59 (m, 2 H), 1.55–1.42 (m, 1 H), 1.14–0.99 (m, 1 H). MS (ESI): <i>m</i>/<i>z</i> 384 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> <i>N</i>-[3-[(Dimethylamino)methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>83</b>) (from <b>106a</b> and 3-[(Dimethylamino)methyl]aniline, <b>83e</b>, See <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 30% (31 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.98 (s, 1 H), 7.84–6.88 (m, 7 H), 4.02 (s, 3 H), 3.54–3.35 (m, 2 H), 2.21 (br s, 6 H). MS (ESI): <i>m</i>/<i>z</i> 312 [M – H].</div></div><div id="sec4_11_2_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> <i>N</i>-[3-[(4-Hydroxyphenoxy)methyl]phenyl]-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>84</b>) (from <b>106a</b> and 4-[(3-Aminophenyl)methoxy]phenol, <b>84e</b>, see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 52% (748 mg), pale-brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.04 (s, 1 H), 8.94 (s, 1 H), 7.83 (m, 1 H), 7.68 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.35–7.30 (m, 2 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 7.11 (m, 1 H), 6.85–6.80 (m, 2 H), 6.70–6.64 (m, 2 H), 4.98 (s, 2 H), 4.02 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 379 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> 4-Methyl-<i>N</i>-[3-(4-pyridylmethoxy)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>87</b>) (from <b>106a</b> and 3-(4-Pyridylmethoxy)aniline)</h5><div class="NLM_p last">Yield 14% (15 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 9.99 (s, 1 H), 8.62–8.54 (m, 2 H), 7.57–7.54 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.47–7.43 (m, 2 H), 7.35–7.30 (m, 2 H), 7.27–7.21 (m, 2 H), 6.75–6.70 (m, 1 H), 5.18 (s, 2 H), 4.01 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>; mp 126–131 °C.</div></div><div id="sec4_11_2_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> 4-Methyl-<i>N</i>-[3-(4-pyridyloxymethyl)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>88</b>) (from <b>106a</b> and 3-(4-Pyridyloxymethyl)aniline)</h5><div class="NLM_p last">Yield 16% (17 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.07 (s, 1 H), 7.76–7.71 (m, 2 H), 7.71–7.66 (m, 2 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.37–7.33 (m, 1 H), 7.31 (s, 1 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 7.01–6.97 (m, 1 H), 6.15–6.07 (m, 2 H), 5.08 (s, 2 H), 4.00 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>; mp 295–298 °C dec.</div></div><div id="sec4_11_2_3_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> 4-Methyl-<i>N</i>-[3-(4-pyridyloxy)phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>89</b>) (from <b>106a</b> and 3-(4-Pyridyloxy)aniline)</h5><div class="NLM_p last">Yield 21% (22 mg), light-brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.25 (s, 1 H), 8.02–7.94 (m, 3 H), 7.78–7.72 (m, 1 H), 7.57–7.48 (m, 2 H), 7.37 (s, 1 H), 7.28–7.22 (m, 2 H), 6.29–6.20 (m, 2 H), 4.03 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 350 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> 4-Methyl-<i>N</i>-[3-[[4-(4-piperidyloxy)phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>90</b>)</h5><div class="NLM_p last">The reaction product between <b>106a</b> and <i>tert</i>-butyl 4-[4-[(3-aminophenyl)methoxy]phenoxy]piperidine-1-carboxylate, <b>90e</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a> for experimental details), was deprotected by acidic treatment, The reaction mixture was concentrated in vacuo, and a saturated aqueous solution of NaHCO<sub>3</sub> was added to the residue and extracted with EtOAc. The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated to yield <b>90</b> (53% over two steps, 57 mg) as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.03 (s, 1 H), 7.90–7.79 (m, 1 H), 7.71–7.65 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.36–7.30 (m, 2 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 7.15–7.09 (m, 1 H), 6.95–6.83 (m, 4 H), 5.02 (s, 2 H), 4.28–4.17 (m, 1 H), 4.02 (s, 3 H), 2.97–2.87 (m, 2 H), 2.57–2.50 (m, 2 H), 1.92–1.79 (m, 2 H), 1.46–1.32 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 462 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> 4-Methyl-<i>N</i>-[3-[[4-[(1-methyl-4-piperidyl)oxy]phenoxy]methyl]phenyl]thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>91</b>) (from <b>106a</b> and 3-[[4-[(1-Methyl-4-piperidyl)oxy]phenoxy]methyl]aniline, <b>91e</b>, see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">Supporting Information</a>)</h5><div class="NLM_p last">Yield 50% (67 mg), pale-pink solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 10.03 (s, 1 H), 7.88–7.81 (m, 1 H), 7.72–7.65 (m, 1 H), 7.52 (d, <i>J</i> = 5.4 Hz, 1 H), 7.37–7.30 (m, 2 H), 7.24 (d, <i>J</i> = 5.4 Hz, 1 H), 7.15–7.10 (m, 1 H), 6.96–6.84 (m, 4 H), 5.02 (s, 2 H), 4.26–4.14 (m, 1 H), 4.01 (s, 3 H), 2.65–2.53 (m, 2 H), 2.21–2.06 (m, 5 H), 1.91–1.79 (m, 2 H), 1.65–1.50 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 476 [M + H]<sup>+</sup>; mp 152–154 °C.</div></div><div id="sec4_11_2_3_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 6-Chloro-4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>56</b>)</h5><div class="NLM_p last">To a stirred solution of 0.05 g (0.2 mmol) of 4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>23</b>) dissolved in 4 mL of dry DCM, 0.027 g (0.21 mmol) of <i>N</i>-chlorosuccinimide was added in three portions at 0 °C. The reaction mixture was allowed to reach rt and stirred for about 17 h. The mixture was diluted with DCM (up to 30 mL) and washed with saturated aqueous NH<sub>4</sub>Cl solution (20 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and the solvent evaporated. The residue was purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc from 1% to 3% of EtOAc) to <b>56</b> (26%, 15 mg) as a white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.47 (br s, 1 H), 7.65 (m, 2 H), 7.40 (m, 2 H), 7.34 (d, <i>J</i> = 5.38 Hz, 1 H), 7.17 m, 1 H), 6.98 (d, <i>J</i> = 5.38 Hz, 1 H), 4.11 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 291 [M + H]<sup>+</sup>; mp 161–165 °C.</div></div><div id="sec4_11_2_3_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> 6-Iodo-4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>58</b>), 2-Iodo-4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>59</b>), and 2,6-Diiodo-4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>60</b>)</h5><div class="NLM_p">To a stirred solution of 0.05 g (0.2 mmol) of 4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>23</b>) dissolved in 4 mL of dry DCM, 0.0274 g (0.20 mmol) of <i>N</i>-iodosuccinimmide was added in three portions at 0 °C. The reaction mixture was allowed to reach rt and stirred for about 17 h. The mixture was diluted with DCM (up to 30 mL) and washed with saturated aqueous NH<sub>4</sub>Cl solution (20 mL). The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and the solvent evaporated. The residue was purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc 8:2) and subsequently by preparative HPLC.</div><div class="NLM_p"><b>58</b>: Yield 18% (13 mg), off-white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.15 (br s, 1 H), 7.69 (m, 2 H), 7.41 (m, 2 H), 7.35 (d, <i>J</i> = 5.4 Hz, 1 H), 7.18 (m, 1 H), 7.09 (d, <i>J</i> = 5.4 Hz, 1 H), 4.11 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 383 [M + H]<sup>+</sup>; mp 195–199 °C dec.</div><div class="NLM_p"><b>59</b>: Yield 16% (12 mg), off-white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.65 (br s, 1 H), 7.58 (m, 2 H), 7.38 (m, 2 H), 7.21 (s, 1 H), 7.16 (m, 1 H), 6.85 (s, 1 H), 4.06 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 383 [M + H]<sup>+</sup>.</div><div class="NLM_p last"><b>60</b>: Yield 2% (2 mg), off-white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>), δ (ppm): 8.09 (br s, 1 H), 7.67 (m, 2 H), 7.41 (m, 2 H), 7.30 (s, 1 H), 7.19 (m, 1 H), 4.05 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 509 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_3_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> 2-Fluoro-4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>61</b>)</h5><div class="NLM_p last">First, 0.47 mL (0.94 mmol) of a 2 M solution of LDA in THF was added to a solution of 0.08 g (0.3121 mmol) of 4-methyl-<i>N</i>-phenyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxamide (<b>23</b>) in 2.3 mL of dry THF at −78 °C. Then the mixture was stirred for 1 h, and then a solution of 0.30 g (0.94 mmol) of <i>N</i>-fluorobenzenesulfonimide in 1 mL dry THF was added. Stirring was continued at −78 °C for 15 min and at rt for 1 h. Water was added and the mixture extracted with EtOAc; the organics were combined, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to afford a brown oily residue purified by flash chromatography (eluent <i>n</i>-hexane/EtOAc from 5% to 20% of EtOAc). A further purification by preparative HPLC gave <b>61</b> (9%, 7.9 mg) as a white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>), δ (ppm): 7.63 (br s, 1 H), 7.58 (m, 2 H), 7.37 (m, 2 H), 7.15 (m, 1 H), 6.85 (s, 1 H), 6.55 (s, 1 H), 4.04 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 275 [M + H]<sup>+</sup>; mp 164–166 °C dec.</div></div></div></div><div id="sec4_11_2_3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> Synthesis of Azido Derivatives (<b>103a</b>–<b>l</b>)</h3><div id="sec4_11_2_3_17_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> Ethyl (<i>Z</i>)-2-Azido-3-(2-thienyl)prop-2-enoate (<b>103a</b>)</h4><div class="NLM_p last">A solution of 0.3 g (2.7 mmol) of thiophene-2-carbaldehyde (<b>102a</b>) and 2.1 g (4.0 mmol) of ethyl azidoacetate (25% in EtOH) was added dropwise to a solution of 1.4 g (4.0 mmol) of KOEt in 6 mL of ethanol at 0 °C. The reaction mixture was stirred at 0 °C for 2 h and rt for additional 2 h. The reaction mixture was poured into a saturated aqueous solution of NH<sub>4</sub>Cl and extracted with diethyl ether. The combined organics were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo. The residue was purified by silica gel flash chromatography (eluent: <i>n</i>-hexane/EtOAc, from 0% to 5% of EtOAc) to provide <b>103a</b> (33%, 200 mg) as a yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.54–7.48 (m, 1 H), 7.37–7.31 (m, 1 H), 7.18 (s, 1 H), 7.12–7.05 (m, 1 H), 4.43–4.30 (m, 2 H), 1.46–1.33 (m, 3 H).</div></div><div id="sec4_11_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> Similarly Prepared</h4><div id="sec4_11_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> Ethyl (<i>Z</i>)-2-Azido-3-(3-thienyl)prop-2-enoate (<b>103b</b>) (from <b>102b</b>)</h5><div class="NLM_p last">Yield 25% (200 mg), yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.93–7.86 (m, 1 H), 7.53–7.48 (m, 1 H), 7.37–7.31 (m, 1 H), 6.99–6.95 (m, 1 H), 4.45–4.29 (m, 2 H), 1.45–1.35 (m, 3 H).</div></div><div id="sec4_11_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> Ethyl (<i>Z</i>)-2-Azido-3-(2-furyl)prop-2-enoate (<b>103c</b>) (from <b>102c</b>)</h5><div class="NLM_p last">Yield 39% (840 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.53–7.47 (m, 1 H), 7.15–7.08 (m, 1 H), 6.88 (s, 1 H), 6.57–6.50 (m, 1 H), 4.36 (q, <i>J</i> = 6.8 Hz, 2 H), 1.39 (t, <i>J</i> = 6.8 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 225 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> Ethyl (<i>Z</i>)-2-Azido-3-thiazol-4-yl-prop-2-enoate (<b>103d</b>) (from <b>102d</b>)</h5><div class="NLM_p last">Yield 29% (580 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.86–8.75 (m, 1 H), 8.30–8.18 (m, 1 H), 7.34–7.19 (m, 1 H), 4.38 (q, <i>J</i> = 6.8 Hz, 2 H), 1.40 (t, <i>J</i> = 6.8 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 225 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> Ethyl (<i>Z</i>)-2-Azido-3-thiazol-5-yl-prop-2-enoate (<b>103e</b>) (from <b>102e</b>)</h5><div class="NLM_p last">Yield 10% (200 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.94–8.84 (m, 1 H), 8.13–8.03 (m, 1 H), 7.20 (s, 1 H), 4.39 (q, <i>J</i> = 7.3 Hz, 2 H), 1.42 (t, <i>J</i> = 7.3 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 222 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> Ethyl (<i>Z</i>)-2-Azido-3-(3-methylimidazol-4-yl)prop-2-enoate (<b>103f</b>) (from <b>102f</b>)</h5><div class="NLM_p last">Yield 11% (211 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.00–7.86 (m, 1 H), 7.57–7.49 (m, 1 H), 6.73 (s, 1 H), 4.41–4.35 (m, 2 H), 3.69 (s, 3 H), 1.45–1.35 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 225 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_4_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> Ethyl (<i>Z</i>)-2-Azido-3-(1-methylpyrrol-2-yl)prop-2-enoate (<b>103g</b>) (from <b>102g</b>)</h5><div class="NLM_p last">The crude was used in the next step without purification.</div></div><div id="sec4_11_2_4_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> Ethyl (<i>Z</i>)-2-Azido-3-(2-methylpyrazol-3-yl)prop-2-enoate (<b>103h</b>) (from <b>102h</b>)</h5><div class="NLM_p last">Yield 15% (151 mg), pale-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.57–7.41 (m, 1 H), 6.96 (d, <i>J</i> = 2.0 Hz, 1 H), 6.86 (s, 1 H), 4.32 (q, <i>J</i> = 7.1 Hz, 2 H), 3.89 (s, 3 H), 1.32 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 222 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_4_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> Ethyl (<i>Z</i>)-2-Azido-3-(5-bromo-2-thienyl)prop-2-enoate (<b>103i</b>) (from <b>102i</b>)</h5><div class="NLM_p last">Yield 28% (900 mg), orange oil. The crude was used in the next step without purification.</div></div><div id="sec4_11_2_4_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> Ethyl (<i>Z</i>)-2-Azido-3-(5-methyl-2-thienyl)prop-2-enoate (<b>103j</b>) (from <b>102j</b>)</h5><div class="NLM_p last">Yield 31% (574 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.14 (d, <i>J</i> = 3.4 Hz, 1 H), 7.10 (s, 1 H), 6.74 (d, <i>J</i> = 3.4 Hz, 1 H), 4.36 (q, <i>J</i> = 7.0 Hz, 2 H), 2.54 (s, 3 H), 1.39 (t, <i>J</i> = 7.1 Hz, 3 H).</div></div><div id="sec4_11_2_4_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> Ethyl (<i>Z</i>)-2-Azido-3-(4-methyl-2-thienyl)prop-2-enoate (<b>103k</b>) (from <b>102k</b>)</h5><div class="NLM_p last">Yield 95% (630 mg), brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.39 (s, 1 H), 7.37 (s, 1 H), 7.19 (s, 1 H), 4.29 (q, <i>J</i> = 7.0 Hz, 2 H), 2.20 (s, 3 H), 1.30 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 210 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_4_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> Ethyl (<i>Z</i>)-2-Azido-3-(4,5-dimethyl-2-thienyl)prop-2-enoate (<b>103l</b>) (from <b>102l</b>)</h5><div class="NLM_p last">Yield 30% (530 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.26 (s, 1 H), 7.12 (s, 1 H), 4.27 (q, <i>J</i> = 7.3 Hz, 2 H), 2.34 (s, 3 H), 2.07 (s, 3 H), 1.32–1.27 (m, 3 H).</div></div></div></div><div id="sec4_11_2_4_11_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i121" class="anchor-spacer"></div><h3 class="article-section__title" id="_i121"> Synthesis of Bicyclic Rings (<b>104a</b>–<b>l</b>)</h3><div id="sec4_11_2_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> Ethyl 6<i>H</i>-Thieno[2,3-<i>b</i>]pyrrole-5-carboxylate (<b>104b</b>)</h4><div class="NLM_p last">A solution of 0.18 g (0.81 mmol) of ethyl (<i>Z</i>)-2-azido-3-(3-thienyl)prop-2-enoate (<b>103b</b>) in 4 mL of xylene was heated at reflux for 30 min, concentrated in vacuo, diluted with diethyl ether, and then evaporated to dryness to give <b>104b</b> (96%, 151 mg) as a yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.23 (br s, 1 H), 7.17–7.07 (m, 1 H), 7.05–6.98 (m, 1 H), 6.96–6.87 (m, 1 H), 4.44–4.27 (m, 2 H), 1.47–1.30 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 196 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> Similarly Prepared</h4><div id="sec4_11_2_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i124" class="anchor-spacer"></div><h5 class="article-section__title" id="_i124"> Ethyl 4<i>H</i>-Thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104a</b>) (from <b>103a</b>)</h5><div class="NLM_p last">Yield 91% (140 mg), red solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.07 (br s, 1 H), 7.37–7.30 (m, 1 H), 7.18–7.12 (m, 1 H), 7.00–6.94 (m, 1 H), 4.38 (q, <i>J</i> = 6.8 Hz, 2 H), 1.40 (t, <i>J</i> = 6.8 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 196 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i125" class="anchor-spacer"></div><h5 class="article-section__title" id="_i125"> Ethyl 4<i>H</i>-Furo[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104c</b>) (from <b>103c</b>)</h5><div class="NLM_p last">Yield 91% (655 mg), yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.65 (br s, 1 H), 7.83–7.71 (m, 1 H), 6.77–6.68 (m, 1 H), 6.63–6.55 (m, 1 H), 4.25 (q, <i>J</i> = 6.8 Hz, 2 H), 1.28 (t, <i>J</i> = 6.8 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 180 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i126" class="anchor-spacer"></div><h5 class="article-section__title" id="_i126"> Ethyl 4<i>H</i>-Pyrrolo[3,2-<i>d</i>]thiazole-5-carboxylate (<b>104d</b>) (from <b>103d</b>)</h5><div class="NLM_p last">Yield 99% (490 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.30 (br s, 1 H), 8.57 (s, 1 H), 7.37–7.30 (m, 1 H), 4.40 (q, <i>J</i> = 6.8 Hz, 2 H), 1.41 (t, <i>J</i> = 6.8 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 197 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i127" class="anchor-spacer"></div><h5 class="article-section__title" id="_i127"> Ethyl 4<i>H</i>-Pyrrolo[2,3-<i>d</i>]thiazole-5-carboxylate (<b>104e</b>) (from <b>103e</b>)</h5><div class="NLM_p last">Yield 95% (150 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 9.87 (br s, 1 H), 8.82–8.72 (m, 1 H), 7.20–7.10 (m, 1 H), 4.41 (q, <i>J</i> = 7.3 Hz, 2 H), 1.42 (t, <i>J</i> = 7.3 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 197 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i128" class="anchor-spacer"></div><h5 class="article-section__title" id="_i128"> Ethyl 1-Methyl-4<i>H</i>-pyrrolo[2,3-<i>d</i>]imidazole-5-carboxylate (<b>104f</b>) (from <b>103f</b>)</h5><div class="NLM_p last">Yield 97% (170 mg), brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.63 (s, 1 H), 7.78–7.59 (m, 1 H), 6.70 (d, <i>J</i> = 1.5 Hz, 1 H), 4.22 (q, <i>J</i> = 6.8 Hz, 2 H), 3.71 (s, 3 H), 1.27 (t, <i>J</i> = 6.9 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 194 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i129" class="anchor-spacer"></div><h5 class="article-section__title" id="_i129"> Ethyl 1-Methyl-4<i>H</i>-pyrrolo[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104g</b>) (from <b>103g</b>)</h5><div class="NLM_p last">Yield 54% (190 mg), white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.41 (br s, 1 H), 6.83–6.79 (m, 1 H), 6.77–6.72 (m, 1 H), 5.97–5.91 (m, 1 H), 4.35 (q, <i>J</i> = 7.2 Hz, 2 H), 3.70 (s, 3 H), 1.38 (t, <i>J</i> = 7.2 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 193 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i130" class="anchor-spacer"></div><h5 class="article-section__title" id="_i130"> Ethyl 1-Methyl-6<i>H</i>-pyrrolo[2,3-<i>c</i>]pyrazole-5-carboxylate (<b>104h</b>) (from <b>103h</b>)</h5><div class="NLM_p last">Yield 18% (45 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.73 (br s, 1 H), 7.44 (s, 1 H), 6.74 (s, 1 H), 4.26 (q, <i>J</i> = 7.3 Hz, 2 H), 3.84 (s, 3 H), 1.29 (t, <i>J</i> = 7.3 Hz, 3 H)</div></div><div id="sec4_11_2_5_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i131" class="anchor-spacer"></div><h5 class="article-section__title" id="_i131"> Ethyl 2-Bromo-4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104i</b>) (from <b>103i</b>)</h5><div class="NLM_p last">Yield 98% (800 mg), orange solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.27–12.10 (m, 1 H), 7.18 (s, 1 H), 7.05 (s, 1 H), 4.33–4.18 (m, 2 H), 1.38–1.20 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 272 [M −H].</div></div><div id="sec4_11_2_5_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i132" class="anchor-spacer"></div><h5 class="article-section__title" id="_i132"> Ethyl 2-Methyl-4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104j</b>) (from <b>103j</b>)</h5><div class="NLM_p last">Yield 100% (495 mg), orange solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.89 (br s, 1 H), 7.06 (d, <i>J</i> = 1.5 Hz, 1 H), 6.66 (s, 1 H), 4.36 (q, <i>J</i> = 7.0 Hz, 2 H), 2.56 (s, 3 H), 1.39 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 210 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i133" class="anchor-spacer"></div><h5 class="article-section__title" id="_i133"> Ethyl 3-Methyl-4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104k</b>) (from <b>103k</b>)</h5><div class="NLM_p last">Yield 96% (630 mg), brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.12 (br s, 1 H), 7.09 (d, <i>J</i> = 1.0 Hz, 1 H), 7.04 (d, <i>J</i> = 2.0 Hz, 1 H), 4.28 (q, <i>J</i> = 7.3 Hz, 2 H), 2.29 (d, <i>J</i> = 1.0 Hz, 3 H), 1.30 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 210 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_5_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i134" class="anchor-spacer"></div><h5 class="article-section__title" id="_i134"> Ethyl 2,3-Dimethyl-4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104l</b>) (from <b>103l</b>)</h5><div class="NLM_p last">Yield 96% (426 mg). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 11.97 (br s, 1 H), 6.97 (d, <i>J</i> = 2.0 Hz, 1 H), 4.26 (q, <i>J</i> = 7.3 Hz, 2 H), 2.35 (s, 3 H), 2.19 (s, 3 H), 1.29 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 224 [M + H]<sup>+</sup>.</div></div></div></div><div id="sec4_11_2_5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i135" class="anchor-spacer"></div><h3 class="article-section__title" id="_i135"> Synthesis of Substituted Pyrrolic Nitrogen <b>105a</b>–<b>o</b>, <b>105q</b></h3><div id="sec4_11_2_5_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> Ethyl 4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105a</b>)</h4><div class="NLM_p last">To a stirred suspension of 0.040 g (1.0 mmol) of NaH in 10 mL dry DMF, 0.130 g (0.67 mmol) of ethyl 4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104a</b>) was added portionwise over 5–10 min. The mixture was stirred 15 min at rt, and then 0.125 mL (2 mmol) of methyl iodide were added in one portion. After 1 h, a solution of 10% NH<sub>4</sub>Cl was added and the mixture extracted with diethyl ether. The organic phase was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and evaporated to give <b>105a</b> (99%; 138 mg) as a red oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.34 (d, <i>J</i> = 5.4 Hz, 1 H), 7.20 (s, 1 H), 6.95 (d, <i>J</i> = 5.4 Hz, 1 H), 4.34 (q, <i>J</i> = 7.3 Hz, 2 H), 4.07 (s, 3 H), 1.39 (t, <i>J</i> = 7.3 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 210 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> Similarly Prepared</h4><div id="sec4_11_2_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i138" class="anchor-spacer"></div><h5 class="article-section__title" id="_i138"> Ethyl 6-Methylthieno[2,3-<i>b</i>]pyrrole-5-carboxylate (<b>105b</b>) (from <b>104b</b>)</h5><div class="NLM_p last">Yield 98% (142 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.17 (s, 1 H), 7.00 (d, <i>J</i> = 5.4 Hz, 1 H), 6.93 (d, <i>J</i> = 5.4 Hz, 1 H), 4.34 (q, <i>J</i> = 7.0 Hz, 2 H), 4.04 (s, 3 H), 1.39 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 210 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i139" class="anchor-spacer"></div><h5 class="article-section__title" id="_i139"> Ethyl 4-Methylfuro[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105c</b>) (from <b>104c</b>)</h5><div class="NLM_p last">Yield 99% (450 mg), red oil. <sup>1</sup>H H NMR (CDCl<sub>3</sub>) δ (ppm): 7.51 (d, <i>J</i> = 2.4 Hz, 1 H), 6.81 (s, 1 H), 6.45 (d, <i>J</i> = 2.4 Hz, 1 H), 4.31 (q, <i>J</i> = 7.3 Hz, 2 H), 3.98 (s, 3 H), 1.38 (t, <i>J</i> = 7.3 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 194 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i140" class="anchor-spacer"></div><h5 class="article-section__title" id="_i140"> Ethyl 4-Methylpyrrolo[3,2-<i>d</i>]thiazole-5-carboxylate (<b>105d</b>) (from <b>104d</b>)</h5><div class="NLM_p last">Yield 98% (157 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.54 (s, 1 H), 7.36 (s, 1 H), 4.35 (q, <i>J</i> = 7.0 Hz, 2 H), 4.10 (s, 3 H), 1.40 (t, <i>J</i> = 7.0 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 211 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i141" class="anchor-spacer"></div><h5 class="article-section__title" id="_i141"> Ethyl 4-Methylpyrrolo[2,3-<i>d</i>]thiazole-5-carboxylate (<b>105e</b>) (from <b>104e</b>)</h5><div class="NLM_p last">Yield 61% (13 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 8.72 (s, 1 H), 7.19 (s, 1 H), 4.37 (q, <i>J</i> = 7.3 Hz, 2 H), 4.20 (s, 3 H), 1.44–1.35 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 211 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i142" class="anchor-spacer"></div><h5 class="article-section__title" id="_i142"> Ethyl 1,4-Dimethylpyrrolo[2,3-<i>d</i>]imidazole-5-carboxylate (<b>105f</b>) (from <b>104f</b>)</h5><div class="NLM_p last">Yield 51% (11 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.53–7.42 (m, 1 H), 6.84–6.72 (m, 1 H), 4.32 (q, <i>J</i> = 7.2 Hz, 2 H), 4.03 (s, 3 H), 3.75 (s, 3 H), 1.38 (t, <i>J</i> = 7.2 Hz, 3 H). MS (ESI) <i>m</i>/<i>z</i> 208 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i143" class="anchor-spacer"></div><h5 class="article-section__title" id="_i143"> Ethyl 1,4-Dimethylpyrrolo[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105g</b>) (from <b>104g</b>)</h5><div class="NLM_p last">Yield 100% (107 mg), yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 6.83–6.79 (m, 1 H), 6.77 (s, 1 H), 5.95–5.88 (m, 1 H), 4.31 (q, <i>J</i> = 7.0 Hz, 2 H), 3.96 (s, 3 H), 3.67 (s, 3 H), 1.38 (t, <i>J</i> = 7.0 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 207 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i144" class="anchor-spacer"></div><h5 class="article-section__title" id="_i144"> Ethyl 1,6-Dimethylpyrrolo [2,3-<i>c</i>]pyrazole-5-carboxylate (<b>105h</b>) (from <b>104h</b>)</h5><div class="NLM_p last">Yield 98% (45 mg), white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.44 (s, 1 H), 6.84 (s, 1 H), 4.23 (q, <i>J</i> = 7.3 Hz, 2 H), 4.03 (s, 3 H), 4.00 (s, 3 H), 1.28 (t, <i>J</i> = 7.3 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 208 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i145" class="anchor-spacer"></div><h5 class="article-section__title" id="_i145"> Ethyl 2-Bromo-4-methyl-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105i</b>) (from <b>104i</b>)</h5><div class="NLM_p last">Yield 81%, (340 mg), light-yellow solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 7.56 (s, 1 H), 7.12 (s, 1 H), 4.25 (q, <i>J</i> = 7.3 Hz, 2 H), 3.96 (s, 3 H), 1.29 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 288 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i146" class="anchor-spacer"></div><h5 class="article-section__title" id="_i146"> Ethyl 2,4-Dimethylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105j</b>) (from <b>104j</b>)</h5><div class="NLM_p last">Yield 70% (362 mg), pale-yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.11 (s, 1 H), 6.64 (s, 1 H), 4.32 (q, <i>J</i> = 7.3 Hz, 2 H), 4.01 (s, 3 H), 2.57 (d, <i>J</i> = 1.0 Hz, 3 H), 1.38 (t, <i>J</i> = 7.3 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 224 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i147" class="anchor-spacer"></div><h5 class="article-section__title" id="_i147"> Ethyl 3,4-Dimethylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105k</b>) (from <b>104k</b>)</h5><div class="NLM_p last">Yield 74% (325 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.15 (s, 1 H), 6.89 (s, 1 H), 4.33 (q, <i>J</i> = 6.8 Hz, 2 H), 4.21 (s, 3 H), 2.51 (s, 3 H), 1.38 (t, <i>J</i> = 7.3 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 224 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i148" class="anchor-spacer"></div><h5 class="article-section__title" id="_i148"> Ethyl 2,3,4-Trimethylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105l</b>) (from <b>104l</b>)</h5><div class="NLM_p last">Yield 49% (218 mg). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.08 (s, 1 H), 4.31 (q, <i>J</i> = 6.8 Hz, 2 H), 4.20 (s, 3 H), 2.42 (s, 3 H), 2.39 (s, 3 H), 1.37 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 238 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i149" class="anchor-spacer"></div><h5 class="article-section__title" id="_i149"> Ethyl 4-Ethylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105m</b>) (from <b>104a</b> and Ethyl Iodide)</h5><div class="NLM_p last">Yield 96% (1.1 g), brown oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.34 (d, <i>J</i> = 5.4 Hz, 1 H), 7.20 (s, 1 H), 6.96 (d, <i>J</i> = 5.4 Hz, 1 H), 4.56 (q, <i>J</i> = 7.3 Hz, 2 H), 4.34 (q, <i>J</i> = 7.3 Hz, 2 H), 1.46–1.34 (m, 6 H). MS (ESI): <i>m</i>/<i>z</i> 224 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i150" class="anchor-spacer"></div><h5 class="article-section__title" id="_i150"> Ethyl 4-Propylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105n</b>) (from <b>104a</b> and Propyl Bromide)</h5><div class="NLM_p last">Yield 79% (240 mg), yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.37–7.29 (m, 1 H), 7.24–7.15 (m, 1 H), 6.99–6.90 (m, 1 H), 4.51–4.41 (m, 2 H), 4.38–4.28 (m, 2 H), 1.90–1.79 (m, 2 H), 1.44–1.32 (m, 3 H), 0.96–0.87 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 238 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i151" class="anchor-spacer"></div><h5 class="article-section__title" id="_i151"> Ethyl 4-Benzylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105o</b>) (from <b>104a</b> and Benzyl Bromide)</h5><div class="NLM_p last">Yield 84% (222 mg), yellow oil. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.36–7.06 (m, 7 H), 6.90–6.78 (m, 1 H), 5.77 (s, 2 H), 4.30 (q, <i>J</i> = 7.0 Hz, 2 H), 1.34 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 286 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_6_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i152" class="anchor-spacer"></div><h5 class="article-section__title" id="_i152"> Ethyl 4-(Benzenesulfonyl)thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105q</b>) (from <b>104a</b> and Benzenesulfonyl chloride)</h5><div class="NLM_p last">Yield 93% (160 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 8.05–7.47 (m, 8 H), 4.21 (q, <i>J</i> = 7.3 Hz, 2 H), 1.21 (t, <i>J</i> = 7.1 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 358 [M + Na]<sup>+</sup>.</div></div><div id="sec4_11_2_6_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i153" class="anchor-spacer"></div><h5 class="article-section__title" id="_i153"> Ethyl 4-Phenylthieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105p</b>)</h5><div class="NLM_p last">Four mL of dry DCM, 0.143 mL (1.0 mmol) of triethylamine, 0.0825 mL (1.0 mmol) of pyridine, and 0.5 g of grounded 3A molecular sieves were added to 0.1 g (0.51 mmol) of ethyl 4<i>H</i>-thieno[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>104a</b>), 0.186 g (1.0 mmol) of Cu(OAc)<sub>2</sub>, and 0.125 g (1.0 mmol) of phenylboronic acid under nitrogen atm. The mixture was vigorously stirred at rt for about 30 h and filtered through Celite and washed with DCM and EtOAc. The crude was purified by flash chromatography (eluent <i>n</i>-hexane:EtOAc from 1% to 10% of EtOAc) to give <b>105p</b> (97%, 135 mg) as a beige solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ (ppm): 7.56–7.19 (m, 7 H), 6.84–6.60 (m, 1 H), 4.21 (q, <i>J</i> = 7.3 Hz, 2 H), 1.26–1.18 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 272 [M + H]<sup>+</sup>.</div></div></div></div><div id="sec4_11_2_6_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i154" class="anchor-spacer"></div><h3 class="article-section__title" id="_i154"> Synthesis of Carboxylic Acids (<b>106a</b>, <b>106c</b>, <b>106f</b>, <b>106m</b>–<b>p</b>, <b>106r</b>)</h3><div id="sec4_11_2_6_17_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i155" class="anchor-spacer"></div><h4 class="article-section__title" id="_i155"> 4-Methylfuro[3,2-<i>b</i>]pyrrole-5-carboxylic Acid (<b>106c</b>)</h4><div class="NLM_p last">First, 0.365 g (2.2 mmol) of LiOH in 5 mL of H<sub>2</sub>O was added at rt to a solution of 0.45 g (2.33 mmol) of ethyl 4-methylfuro[3,2-<i>b</i>]pyrrole-5-carboxylate (<b>105c</b>) in 5 mL of ethanol, and the mixture was heated to reflux for about 1 h. Then ethanol was evaporated, water was added, and the pH brought to about 2 with 2 M HCl. The formed solid was filtered, washed with water, and dried to give <b>106c</b> (95%, 365 mg) as beige solid. <sup>1</sup>H NMR (methanol-<i>d</i><sub>4</sub>) δ (ppm): 7.60 (d, <i>J</i> = 2.2 Hz, 1 H), 6.77 (s, 1 H), 6.60 (d, <i>J</i> = 2.2 Hz, 1 H), 3.96 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 166 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i156" class="anchor-spacer"></div><h4 class="article-section__title" id="_i156"> Similarly Prepared</h4><div id="sec4_11_2_7_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i157" class="anchor-spacer"></div><h5 class="article-section__title" id="_i157"> 4-Methylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic Acid (<b>106a</b>) (from <b>105a</b>)</h5><div class="NLM_p last">Yield 98% (117 mg), light-brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.48 (br s, 1 H), 7.62–7.47 (m, 1 H), 7.26–7.19 (m, 1 H), 7.12 (s, 1 H), 3.99 (s, 3 H). MS (ESI): <i>m</i>/<i>z</i> 180 [M – H].</div></div><div id="sec4_11_2_7_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i158" class="anchor-spacer"></div><h5 class="article-section__title" id="_i158"> 1-Methyl-4<i>H</i>-pyrrolo [2,3-<i>d</i>]imidazole-5-carboxylic Acid (<b>106f</b>) (from <b>104f</b>)</h5><div class="NLM_p last">Yield 99% (119 mg). MS (ESI): <i>m</i>/<i>z</i> 166 [M + H]<sup>+</sup>. MS (ESI): <i>m</i>/<i>z</i> 164 [M – H].</div></div><div id="sec4_11_2_7_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i159" class="anchor-spacer"></div><h5 class="article-section__title" id="_i159"> 4-Ethylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic Acid (<b>106m</b>) (from <b>105m</b>)</h5><div class="NLM_p last">Yield 97% (230 mg), brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.46 (br s, 1 H), 7.62–7.46 (m, 1 H), 7.29–7.20 (m, 1 H), 7.16–7.07 (m, 1 H), 4.51 (q, <i>J</i> = 7.3 Hz, 2 H), 1.28 (t, <i>J</i> = 6.8 Hz, 3 H). MS (ESI): <i>m</i>/<i>z</i> 194 [M – H].</div></div><div id="sec4_11_2_7_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i160" class="anchor-spacer"></div><h5 class="article-section__title" id="_i160"> 4-Propylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic Acid (<b>106n</b>) (from <b>105n</b>)</h5><div class="NLM_p last">Yield 99% (200 mg), colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.43 (br s, 1 H), 7.60–7.44 (m, 1 H), 7.26–7.20 (m, 1 H), 7.13 (s, 1 H), 4.50–4.33 (m, 2 H), 1.78–1.61 (m, 2 H), 0.85–0.72 (m, 3 H). MS (ESI): <i>m</i>/<i>z</i> 208 [M – H].</div></div><div id="sec4_11_2_7_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i161" class="anchor-spacer"></div><h5 class="article-section__title" id="_i161"> 4-Benzylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic Acid (<b>106o</b>) (from <b>105o</b>)</h5><div class="NLM_p last">Yield 100% (190 mg), colorless solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.54 (br s, 1 H), 7.62–6.95 (m, 8 H), 5.77 (s, 2 H). MS (ESI): <i>m</i>/<i>z</i> 258 [M + H]<sup>+</sup>.</div></div><div id="sec4_11_2_7_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i162" class="anchor-spacer"></div><h5 class="article-section__title" id="_i162"> 4-Phenylthieno[3,2-<i>b</i>]pyrrole-5-carboxylic Acid (<b>106p</b>) (from <b>105p</b>)</h5><div class="NLM_p last">Yield 95% (110 mg), beige solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.44 (br s, 1 H), 7.62–6.69 (m, 8 H). MS (ESI): <i>m</i>/<i>z</i> 266 [M + Na].</div></div><div id="sec4_11_2_7_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i163" class="anchor-spacer"></div><h5 class="article-section__title" id="_i163"> 4<i>H</i>-Thieno[3,2-<i>b</i>]pyrrole-5-carboxylic Acid (<b>106r</b>) (from <b>105r</b>)</h5><div class="NLM_p last">Yield quantitative (180 mg), brown solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ (ppm): 12.53 (br s, 1 H), 11.90 (br s, 1 H), 7.52–7.35 (m, 1 H), 7.09–6.85 (m, 2 H). MS (ESI): <i>m</i>/<i>z</i> 168 [M + H]<sup>+</sup>.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i164"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01018">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56576" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56576" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01018" class="ext-link">10.1021/acs.jmedchem.6b01018</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthetic procedures describing, <b>62</b>–<b>65</b>, <b>83e</b>–<b>86e</b>, <b>90e</b>, <b>91e</b>, and <b>92</b>–<b>94</b>, additional figure illustrating the effects of KDM1A down-regulation on mRNA expression of CD11b and CD14 in THP-1 cells, apoptotic induction assessment in THP-1 cells, data collection, and refinement statistics for compound <b>19</b>, electron density map for compound <b>19</b>, interaction map for compound <b>19</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Protein Target (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Compound <b>90</b> pose A (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Compound <b>90</b> pose B (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_005.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf">jm6b01018_si_001.pdf (829.39 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_002.csv">jm6b01018_si_002.csv (6.23 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_003.pdb">jm6b01018_si_003.pdb (510.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_004.pdb">jm6b01018_si_004.pdb (11.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_005.pdb">jm6b01018_si_005.pdb (11.84 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates for the crystal structures of KDM1A with compound <b>19</b> can be accessed using PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGN">5LGN</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01018" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luca Sartori</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f69a839597d88597848299849fb69f90999bd89383"><span class="__cf_email__" data-cfemail="422e3721236c312330362d302b022b242d2f6c2737">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Vianello</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9100-2711" title="Orcid link">http://orcid.org/0000-0001-9100-2711</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#8afaebe5e6eba4fce3ebe4efe6e6e5cae3ece5e7a4efff"><span class="__cf_email__" data-cfemail="90e0f1fffcf1bee6f9f1fef5fcfcffd0f9f6fffdbef5e5">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ciro Mercurio</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Genextra
Group, DAC s.r.l., Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federica Amigoni</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Cappa</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giovanni Fagá</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raimondo Fattori</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elena Legnaghi</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Ciossani</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
1, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Mattevi</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
1, 27100 Pavia, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9523-7128" title="Orcid link">http://orcid.org/0000-0002-9523-7128</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Meroni</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Loris Moretti</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9851-2965" title="Orcid link">http://orcid.org/0000-0002-9851-2965</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Valentina Cecatiello</span> - <span class="hlFld-Affiliation affiliation">IFOM-
The FIRC Institute of Molecular Oncology Foundation, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Crystallography
Unit, Department of Experimental Oncology, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastiano Pasqualato</span> - <span class="hlFld-Affiliation affiliation">Crystallography
Unit, Department of Experimental Oncology, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessia Romussi</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Florian Thaler</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paolo Trifiró</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Manuela Villa</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefania Vultaggio</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oronza A. Botrugno</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Dessanti</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Saverio Minucci</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biosciences, University of Milan, Via Celoria, 26, 20133 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisa Zagarrí</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniele Carettoni</span> - <span class="hlFld-Affiliation affiliation">Axxam, via A. Meucci 3, 20091 Bresso, Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lucia Iuzzolino</span> - <span class="hlFld-Affiliation affiliation">Axxam, via A. Meucci 3, 20091 Bresso, Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mario Varasi</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology, Academic Drug Discovery, European Institute of Oncology, Via Adamello 16, 20139 Milano, Italy</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d46e6357-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i169">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51687" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51687" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Fondazione Cariplo (2010.0778), AIRC (15208), MIUR (Progetto Epigen), FIRB RBFR10ZJQT and RF-2010-2318330 projects by MIUR, Sapienza Ateneo Project 2013, IIT-Sapienza Project, FP7 projects BLUEPRINT/282510 and A-PARADDISE/602080, and Regione Lombardia, Fondo per la Promozione di Accordi Istituzionale, Bando di Invito di cui al Decreto no. 4779 del 14/05/2009, Progetto DiVA, BioStruct-X (FP7/2007-2013, grant agreement 283570), and Rasna Theraupeutics Inc. We thank scientists at the X06DA/PXIII beamline at the Swiss Light Source and the ID30A-1/Massif-1 beamline at the European Synchrotron Radiation Facility for valuable help with data collection.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i170" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i170"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i171" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i171"> Abbreviations Used</h2><tr><td class="NLM_term">LSD</td><td class="NLM_def"><p class="first last">lysine-specific histone demethylase</p></td></tr><tr><td class="NLM_term">MAO</td><td class="NLM_def"><p class="first last">monoamine oxidase</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">APL</td><td class="NLM_def"><p class="first last">acute promyelocytic leukemia</p></td></tr><tr><td class="NLM_term">br s</td><td class="NLM_def"><p class="first last">broad signal</p></td></tr><tr><td class="NLM_term"><i>n</i>-BuLi</td><td class="NLM_def"><p class="first last"><i>n</i>-butyllithium</p></td></tr><tr><td class="NLM_term">CH<sub>2</sub>Cl<sub>2</sub></td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DPPA</td><td class="NLM_def"><p class="first last">diphenylphosphoryl azide</p></td></tr><tr><td class="NLM_term">DMEDA</td><td class="NLM_def"><p class="first last">2-(dimethylamino)ethylamine</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">mp</td><td class="NLM_def"><p class="first last">melting point</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">(benzotriazol-1-yloxy)tripyrrolidino- phosphonium hexa fluorophosphate</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuOH</td><td class="NLM_def"><p class="first last"><i>tert</i>-butanol</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i172">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07316" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07316" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 55 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+Lysine+Methylation+Dynamics%3A+Establishment%2C+Regulation%2C+and+Biological+Impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0lgw43-jiTA3zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520Lysine%2520Methylation%2520Dynamics%253A%2520Establishment%252C%2520Regulation%252C%2520and%2520Biological%2520Impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Methylation: A Dynamic Mark in Health, Disease and Inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+Methylation%3A+A+Dynamic+Mark+in+Health%2C+Disease+and+Inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0lgw43-jiTA3zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Methylation%253A%2520A%2520Dynamic%2520Mark%2520in%2520Health%252C%2520Disease%2520and%2520Inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabari, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span> </span><span class="NLM_article-title">SnapShot: Histone Modifications</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.cell.2014.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25303536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=458&author=H.+Huangauthor=B.+R.+Sabariauthor=B.+A.+Garciaauthor=C.+D.+Allisauthor=Y.+Zhao&title=SnapShot%3A+Histone+Modifications&doi=10.1016%2Fj.cell.2014.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">SnapShot: Histone modifications</span></div><div class="casAuthors">Huang, He; Sabari, Benjamin R.; Garcia, Benjamin A.; Allis, C. David; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">458-458.e1</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Histones are decorated by a variety of protein post-translational modifications called histone marks that modulate chromatin structure and function, contributing to the cellular gene expression program.  This SnapShot summarizes the reported human, mouse, and rat histone marks, including recently identified Lys residue acylation marks.  It has been proposed that histone acylations provide a link between cellular metab. and epigenetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQmsTna8jsxrVg90H21EOLACvtfcHk0liQSEbDoS5oKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO&md5=969e02ba1cbaa8f87bd5aae32857720f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DSabari%26aufirst%3DB.%2BR.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DSnapShot%253A%2520Histone%2520Modifications%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D458%26doi%3D10.1016%2Fj.cell.2014.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Demethylation Mediated by the Nuclear Amine Oxidase Homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+Demethylation+Mediated+by+the+Nuclear+Amine+Oxidase+Homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liQSEbDoS5oKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Demethylation%2520Mediated%2520by%2520the%2520Nuclear%2520Amine%2520Oxidase%2520Homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">LSD1 Inhibition: A Therapeutic Strategy in Cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+P.+Somervaille&title=LSD1+Inhibition%3A+A+Therapeutic+Strategy+in+Cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0liQSEbDoS5oKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DLSD1%2520Inhibition%253A%2520A%2520Therapeutic%2520Strategy%2520in%2520Cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Niebel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majores, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutgemann, I.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 (LSD1) in Hematopoietic and Lymphoid Neoplasms</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span><span class="refDoi"> DOI: 10.1182/blood-2014-04-569525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1182%2Fblood-2014-04-569525" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=151-152&author=D.+Niebelauthor=J.+Kirfelauthor=V.+Janzenauthor=T.+Hollerauthor=M.+Majoresauthor=I.+Gutgemann&title=Lysine-Specific+Demethylase+1+%28LSD1%29+in+Hematopoietic+and+Lymphoid+Neoplasms&doi=10.1182%2Fblood-2014-04-569525"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-04-569525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-04-569525%26sid%3Dliteratum%253Aachs%26aulast%3DNiebel%26aufirst%3DD.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DV.%26aulast%3DHoller%26aufirst%3DT.%26aulast%3DMajores%26aufirst%3DM.%26aulast%3DGutgemann%26aufirst%3DI.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520%2528LSD1%2529%2520in%2520Hematopoietic%2520and%2520Lymphoid%2520Neoplasms%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D151%26epage%3D152%26doi%3D10.1182%2Fblood-2014-04-569525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H. F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 Demethylates Repressive Histone Marks to Promote Androgen-Receptor-Dependent Transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+Demethylates+Repressive+Histone+Marks+to+Promote+Androgen-Receptor-Dependent+Transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0liQVheCVq0_Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520Demethylates%2520Repressive%2520Histone%2520Marks%2520to%2520Promote%2520Androgen-Receptor-Dependent%2520Transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 (LSD1) Is Highly Expressed in ER-Negative Breast Cancers and a Biomarker Predicting Aggressive Biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Sch%C3%BCleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+%28LSD1%29+Is+Highly+Expressed+in+ER-Negative+Breast+Cancers+and+a+Biomarker+Predicting+Aggressive+Biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0liQVheCVq0_Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520%2528LSD1%2529%2520Is%2520Highly%2520Expressed%2520in%2520ER-Negative%2520Breast%2520Cancers%2520and%2520a%2520Biomarker%2520Predicting%2520Aggressive%2520Biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 Is Strongly Expressed in Poorly Differentiated Neuroblastoma: Implications for Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Schuleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+Is+Strongly+Expressed+in+Poorly+Differentiated+Neuroblastoma%3A+Implications+for+Therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0ljU0_-KTsO7Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520Is%2520Strongly%2520Expressed%2520in%2520Poorly%2520Differentiated%2520Neuroblastoma%253A%2520Implications%2520for%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen Receptor Coactivators Lysine-Specific Histone Demethylase 1 and Four and a Half LIM Domain Protein 2 Predict Risk of Prostate Cancer Recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Androgen+Receptor+Coactivators+Lysine-Specific+Histone+Demethylase+1+and+Four+and+a+Half+LIM+Domain+Protein+2+Predict+Risk+of+Prostate+Cancer+Recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520Receptor%2520Coactivators%2520Lysine-Specific%2520Histone%2520Demethylase%25201%2520and%2520Four%2520and%2520a%2520Half%2520LIM%2520Domain%2520Protein%25202%2520Predict%2520Risk%2520of%2520Prostate%2520Cancer%2520Recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Lv, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Over-Expression of LSD1 Promotes Proliferation, Migration and Invasion in Non-Small Cell Lung Cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e35065</span><span class="refDoi"> DOI: 10.1371/journal.pone.0035065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1371%2Fjournal.pone.0035065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e35065&author=T.+Lvauthor=D.+Yuanauthor=X.+Miaoauthor=Y.+Lvauthor=P.+Zhanauthor=X.+Shenauthor=Y.+Song&title=Over-Expression+of+LSD1+Promotes+Proliferation%2C+Migration+and+Invasion+in+Non-Small+Cell+Lung+Cancer&doi=10.1371%2Fjournal.pone.0035065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0035065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0035065%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DT.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DOver-Expression%2520of%2520LSD1%2520Promotes%2520Proliferation%252C%2520Migration%2520and%2520Invasion%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De35065%26doi%3D10.1371%2Fjournal.pone.0035065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Zhao, Z. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span> </span><span class="NLM_article-title">Bin. Overexpression of Lysine Specific Demethylase 1 Predicts Worse Prognosis in Primary Hepatocellular Carcinoma Patients</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6651</span><span class="NLM_x">–</span> <span class="NLM_lpage">6656</span><span class="refDoi"> DOI: 10.3748/wjg.v18.i45.6651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.3748%2Fwjg.v18.i45.6651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=23236241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6651-6656&author=Z.+K.+Zhaoauthor=H.+F.+Yuauthor=D.+R.+Wangauthor=P.+Dongauthor=L.+Chenauthor=W.+G.+Wuauthor=W.+J.+Dingauthor=Y.+Liu&title=Bin.+Overexpression+of+Lysine+Specific+Demethylase+1+Predicts+Worse+Prognosis+in+Primary+Hepatocellular+Carcinoma+Patients&doi=10.3748%2Fwjg.v18.i45.6651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span></div><div class="casAuthors">Zhao, Ze-Kun; Yu, Hai-Feng; Wang, Dao-Rong; Dong, Ping; Chen, Lei; Wu, Wen-Guang; Ding, Wen-Jun; Liu, Ying-Bin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">6651-6656</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Aim: To investigate the clinicopathol. features and prognostic value of lysine specific demethylase 1 (LSD1) in hepatocellular carcinoma (HCC).  Methods: We examd. LSD1 expression in 60 paired liver cancer tissues and adjacent noncancerous tissues by quant. real time polymerase chain reaction (qRT-PCR) and Western blotting.  In addn., we analyzed LSD1 expression in 198 HCC samples by immunohistochem.  The relationship between LSD1 expression, clinicopathol. features and patient survival was investigated.  Results: Immunohistochem., Western blotting, and qRT-PCR consistently confirmed LSD1 overexpression in HCC tissues compared to adjacent non-neoplastic tissues (P < 0.01).  Addnl., immunostaining showed more LSD1-pos. cells in the higher tumor stage (T3-4) and tumor grade (G3) than in the lower tumor stage (T1-2, P < 0.001) and tumor grade (G1-2, P < 0.001), resp.  Moreover, HCC patients with high LSD1 expression had significantly lower 5-yr overall survival rates (P < 0.001) and lower 5-yr disease-free survival rates (P < 0.001), resp.  A Cox proportional hazards model further demonstrated that LSD1 over-expression was an independent predictor of poor prognosis for both 5-yr disease-free survival [hazards ratio (HR) = 1.426, 95%CI: 0.672-2.146, P < 0.001] and 5-yr overall survival (HR = 2.456, 95%CI: 1.234-3.932, P < 0.001) in HCC.  Conclusion: Our data suggest for the first time that the overexpression of LSD1 protein in HCC tissues indicates tumor progression and predicts poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqX4-UuO3NnLVg90H21EOLACvtfcHk0ljU0_-KTsO7Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ&md5=c657f6b1e0b5e183c28fefbda4cc8745</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v18.i45.6651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v18.i45.6651%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%2BK.%26aulast%3DYu%26aufirst%3DH.%2BF.%26aulast%3DWang%26aufirst%3DD.%2BR.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DW.%2BG.%26aulast%3DDing%26aufirst%3DW.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DBin.%2520Overexpression%2520of%2520Lysine%2520Specific%2520Demethylase%25201%2520Predicts%2520Worse%2520Prognosis%2520in%2520Primary%2520Hepatocellular%2520Carcinoma%2520Patients%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2012%26volume%3D18%26spage%3D6651%26epage%3D6656%26doi%3D10.3748%2Fwjg.v18.i45.6651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The Deubiquitinase USP28 Stabilizes LSD1 and Confers Stem-Cell-like Traits to Breast Cancer Cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1016/j.celrep.2013.08.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.celrep.2013.08.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=24075993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=224-236&author=Y.+Wuauthor=Y.+Wangauthor=X.+H.+Yangauthor=T.+Kangauthor=Y.+Zhaoauthor=C.+Wangauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+Deubiquitinase+USP28+Stabilizes+LSD1+and+Confers+Stem-Cell-like+Traits+to+Breast+Cancer+Cells&doi=10.1016%2Fj.celrep.2013.08.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Deubiquitinase USP28 Stabilizes LSD1 and Confers Stem-Cell-like Traits to Breast Cancer Cells</span></div><div class="casAuthors">Wu, Yadi; Wang, Yifan; Yang, Xiuwei H.; Kang, Tiebang; Zhao, Yongxiang; Wang, Chi; Evers, B. Mark; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">224-236</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a crit. chromatin modulator that controls cellular pluripotency and differentiation through the demethylation of H3K4me1/2.  Overexpression of LSD1 has been obsd. in many types of tumors and is correlated with its oncogenic effects in tumorigenesis.  However, the mechanism leading to LSD1 upregulation in tumors remains unclear.  Using an unbiased siRNA screening against all the human deubiquitinases, we identified USP28 as a bona fide deubiquitinase of LSD1.  USP28 interacted with and stabilized LSD1 via deubiquitination.  USP28 overexpression correlated with LSD1 upregulation in multiple cancer cell lines and breast tumor samples.  Knockdown of USP28 resulted in LSD1 destabilization, leading to the suppression of cancer stem cell (CSC)-like characteristics in vitro and inhibition of tumorigenicity in vivo, which can be rescued by ectopic LSD1 expression.  Our study reveals a crit. mechanism underlying the epigenetic regulation by USP28 and provides another treatment approach against breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDEa93VRia7Vg90H21EOLACvtfcHk0ljU0_-KTsO7Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM&md5=802c1b0561ed2a8ffa926aa4adb82405</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2013.08.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2013.08.030%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%2BH.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520Deubiquitinase%2520USP28%2520Stabilizes%2520LSD1%2520and%2520Confers%2520Stem-Cell-like%2520Traits%2520to%2520Breast%2520Cancer%2520Cells%26jtitle%3DCell%2520Rep.%26date%3D2013%26volume%3D5%26spage%3D224%26epage%3D236%26doi%3D10.1016%2Fj.celrep.2013.08.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+Lysine+4+Demethylation+Is+a+Target+of+Nonselective+Antidepressive+Medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 μM.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0lhRIW2mr5U0-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520Lysine%25204%2520Demethylation%2520Is%2520a%2520Target%2520of%2520Nonselective%2520Antidepressive%2520Medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Schmidt, D. M. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Trans-2-Phenylcyclopropylamine Is a Mechanism-Based Inactivator of the Histone Demethylase LSD1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">4408</span><span class="NLM_x">–</span> <span class="NLM_lpage">4416</span><span class="refDoi"> DOI: 10.1021/bi0618621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0618621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4408-4416&author=D.+M.+Z.+Schmidtauthor=D.+G.+McCafferty&title=Trans-2-Phenylcyclopropylamine+Is+a+Mechanism-Based+Inactivator+of+the+Histone+Demethylase+LSD1&doi=10.1021%2Fbi0618621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">trans-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span></div><div class="casAuthors">Schmidt, Dawn M. Z.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4408-4416</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic domain of the flavin-dependent human histone demethylase, lysine-specific demethylase 1 (LSD1), belongs to the family of amine oxidases including polyamine oxidase and monoamine oxidase (MAO).  The authors previously assessed MAO inhibitors (MAOIs) for their ability to inhibit the reaction catalyzed by LSD1, demonstrating that trans-2-phenylcyclopropylamine (2-PCPA; tranylcypromine, Parnate) was the most potent inhibitor with respect to LSD1.  Here, the authors showed that 2-PCPA was a time-dependent, mechanism-based irreversible inhibitor of LSD1 with a Ki of 242 μM and a kinact of 0.0106 s-1.  2-PCPA showed limited selectivity for human MAOs vs. LSD1, with kinact/Ki values only 16-fold and 2.4-fold higher for MAO B and MAO A, resp.  Profiles of LSD1 activity and inactivation by 2-PCPA as a function of pH were consistent with a mechanism of inactivation dependent upon enzyme catalysis.  Mass spectrometry supported a role for FAD as the site of covalent modification by 2-PCPA.  These results will provide a foundation for the design of cyclopropylamine-based inhibitors that are selective for LSD1 to probe its role in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVK59U7IsBvbVg90H21EOLACvtfcHk0lhRIW2mr5U0-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D&md5=02925342b86c7dbfb5f1a99fa5264adc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fbi0618621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0618621%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%2BZ.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DTrans-2-Phenylcyclopropylamine%2520Is%2520a%2520Mechanism-Based%2520Inactivator%2520of%2520the%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D4408%26epage%3D4416%26doi%3D10.1021%2Fbi0618621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span> </span><span class="NLM_article-title">Targeting Histone Lysine Demethylases - Progress, Challenges, and the Future</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1839</span><span class="NLM_x">, </span> <span class="NLM_fpage">1416</span><span class="NLM_x">–</span> <span class="NLM_lpage">1432</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.bbagrm.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=24859458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=1416-1432&author=C.+C.+Thinnesauthor=K.+S.+Englandauthor=A.+Kawamuraauthor=R.+Chowdhuryauthor=C.+J.+Schofieldauthor=R.+J.+Hopkinson&title=Targeting+Histone+Lysine+Demethylases+-+Progress%2C+Challenges%2C+and+the+Future&doi=10.1016%2Fj.bbagrm.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine demethylases - Progress, challenges, and the future</span></div><div class="casAuthors">Thinnes, Cyrille C.; England, Katherine S.; Kawamura, Akane; Chowdhury, Rasheduzzaman; Schofield, Christopher J.; Hopkinson, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1416-1432</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  N-Methylation of lysine and arginine residues has emerged as a major mechanism of transcriptional regulation in eukaryotes.  In humans, Nε-methyllysine residue demethylation is catalyzed by two distinct subfamilies of demethylases (KDMs), the flavin-dependent KDM1 subfamily and the 2-oxoglutarate- (2OG) dependent JmjC subfamily, which both employ oxidative mechanisms.  Modulation of histone methylation status is proposed to be important in epigenetic regulation and has substantial medicinal potential for the treatment of diseases including cancer and genetic disorders.  This article provides an introduction to the enzymol. of the KDMs and the therapeutic possibilities and challenges assocd. with targeting them, followed by a review of reported KDM inhibitors and their mechanisms of action from kinetic and structural perspectives.  This article is part of a Special Issue entitled: Methylation: A Multifaceted Modification - looking at transcription and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_ltK_pdkkCrVg90H21EOLACvtfcHk0liYT0su4uakNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE&md5=d5d70e8873451dbedb4f36c66defc040</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520Histone%2520Lysine%2520Demethylases%2520-%2520Progress%252C%2520Challenges%252C%2520and%2520the%2520Future%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D1416%26epage%3D1432%26doi%3D10.1016%2Fj.bbagrm.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Zheng, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. M.</span><span> </span><span class="NLM_article-title">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">1032</span><span class="NLM_x">–</span> <span class="NLM_lpage">1071</span><span class="refDoi"> DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+Systematic+Review+of+Histone+Lysine-Specific+Demethylase+1+and+Its+Inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0liYT0su4uakNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520Systematic%2520Review%2520of%2520Histone%2520Lysine-Specific%2520Demethylase%25201%2520and%2520Its%2520Inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">McGrath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span> </span><span class="NLM_article-title">Targeting Histone Lysine Methylation in Cancer</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.pharmthera.2015.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25578037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=1-22&author=J.+McGrathauthor=P.+Trojer&title=Targeting+Histone+Lysine+Methylation+in+Cancer&doi=10.1016%2Fj.pharmthera.2015.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine methylation in cancer</span></div><div class="casAuthors">McGrath, John; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-22</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Within the vast landscape of histone modifications lysine methylation has gained increasing attention because of its profound regulatory potential.  The methylation of lysine residues on histone proteins modulates chromatin structure and thereby contributes to the regulation of DNA-based nuclear processes such as transcription, replication and repair.  Protein families with opposing catalytic activities, lysine methyltransferases (KMTs) and demethylases (KDMs), dynamically control levels of histone lysine methylation and individual enzymes within these families have become candidate oncol. targets in recent years.  A no. of high quality small mol. inhibitors of these enzymes have been identified.  Several of these compds. elicit selective cancer cell killing in vitro and robust efficacy in vivo, suggesting that targeting 'histone lysine methylation pathways' may be a relevant, emerging cancer therapeutic strategy.  Here, we discuss individual histone lysine methylation pathway targets, the properties of currently available small mol. inhibitors and their application in the context of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogqKUpTpHisrVg90H21EOLACvtfcHk0liYT0su4uakNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D&md5=225ece90384f762b248d2d7981fa120e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcGrath%26aufirst%3DJ.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DTargeting%2520Histone%2520Lysine%2520Methylation%2520in%2520Cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D150%26spage%3D1%26epage%3D22%26doi%3D10.1016%2Fj.pharmthera.2015.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">McAllister, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Recent Progress in Histone Demethylase Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1308</span><span class="NLM_x">–</span> <span class="NLM_lpage">1329</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1308-1329&author=T.+E.+McAllisterauthor=K.+S.+Englandauthor=R.+J.+Hopkinsonauthor=P.+E.+Brennanauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Recent+Progress+in+Histone+Demethylase+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in Histone Demethylase Inhibitors</span></div><div class="casAuthors">McAllister, Tom E.; England, Katherine S.; Hopkinson, Richard J.; Brennan, Paul E.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1308-1329</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is increasing interest in targeting histone N-methyl-lysine demethylases (KDMs) with small mols. both for the generation of probes for target exploration and for therapeutic purposes.  Here we update on previous reviews on the inhibition of the lysine-specific demethylases (LSDs or KDM1s) and JmjC families of N-methyl-lysine demethylases (JmjC KDMs, KDM2-7), focusing on the academic and patent literature from 2014 to date.  We also highlight recent biochem., biol., and structural studies which are relevant to KDM inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWx0bnBT9cDrVg90H21EOLACvtfcHk0lhjWvqsp1793A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE&md5=f744b65af58dff765cdab8f9fea85310</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01758%26sid%3Dliteratum%253Aachs%26aulast%3DMcAllister%26aufirst%3DT.%2BE.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DRecent%2520Progress%2520in%2520Histone%2520Demethylase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1308%26epage%3D1329%26doi%3D10.1021%2Facs.jmedchem.5b01758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Talele, T. T.</span><span> </span><span class="NLM_article-title">The “Cyclopropyl Fragment” is a Versatile Player That Frequently Appears in Preclinical/Clinical Drug Molecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">8712</span><span class="NLM_x">–</span> <span class="NLM_lpage">8756</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8712-8756&author=T.+T.+Talele&title=The+%E2%80%9CCyclopropyl+Fragment%E2%80%9D+is+a+Versatile+Player+That+Frequently+Appears+in+Preclinical%2FClinical+Drug+Molecules&doi=10.1021%2Facs.jmedchem.6b00472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The "Cyclopropyl Fragment" is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span></div><div class="casAuthors">Talele, Tanaji T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8712-8756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing use of the cyclopropyl ring in drug development to transition drug candidates from the preclin. to clin. stage.  Important features of the cyclopropane ring are, the (1) coplanarity of the three carbon atoms, (2) relatively shorter (1.51 Å) C-C bonds, (3) enhanced π-character of C-C bonds, and (4) C-H bonds are shorter and stronger than those in alkanes.  The present review will focus on the contributions that a cyclopropyl ring makes to the properties of drugs contg. it.  Consequently, the cyclopropyl ring addresses multiple roadblocks that can occur during drug discovery such as (a) enhancing potency, (b) reducing off-target effects, (c) increasing metabolic stability, (d) increasing brain permeability, (e) decreasing plasma clearance, (f) contributing to an entropically more favorable binding to the receptor, (g) conformational restriction of peptides/peptidomimetics to prevent proteolytic hydrolysis, and (h) altering drug pKa to reduce its P-glycoprotein efflux ratio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVIJAG1XVRG7Vg90H21EOLACvtfcHk0lhjWvqsp1793A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D&md5=b5586458ec4ab5e2d7586f5739663d1f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00472%26sid%3Dliteratum%253Aachs%26aulast%3DTalele%26aufirst%3DT.%2BT.%26atitle%3DThe%2520%25E2%2580%259CCyclopropyl%2520Fragment%25E2%2580%259D%2520is%2520a%2520Versatile%2520Player%2520That%2520Frequently%2520Appears%2520in%2520Preclinical%252FClinical%2520Drug%2520Molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8712%26epage%3D8756%26doi%3D10.1021%2Facs.jmedchem.6b00472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Ortega Munoz, A.; Fyfe, M. C. T.; Estiarte Martinez, M. d. L. A.; Valls Vidal, N.; Kurz, G.; Castro Palomino, L. J. C.</span><span> </span><span class="NLM_article-title">(Hetero)Aryl Cyclopropylamine Compounds as LSD1 Inhibitors</span>. WO2013/057320,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Ortega+Munoz&author=M.+C.+T.+Fyfe&author=M.+d.+L.+A.+Estiarte+Martinez&author=N.+Valls+Vidal&author=G.+Kurz&author=L.+J.+C.+Castro+Palomino&title=%28Hetero%29Aryl+Cyclopropylamine+Compounds+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%2BMunoz%26aufirst%3DA.%26atitle%3D%2528Hetero%2529Aryl%2520Cyclopropylamine%2520Compounds%2520as%2520LSD1%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Mohammad, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smitheman, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamat, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federowicz, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneck, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butticello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnette, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degenhardt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasparec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNulty, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouthamel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span> </span><span class="NLM_article-title">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+Hypomethylation+Signature+Predicts+Antitumor+Activity+of+LSD1+Inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lhjWvqsp1793A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520Hypomethylation%2520Signature%2520Predicts%2520Antitumor%2520Activity%2520of%2520LSD1%2520Inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span> </span><span class="NLM_article-title">Investigation of GSK2879552 in Subjects With Relapsed/Refractory Small Cell Lung Carcinoma</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&rank=1</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Investigation+of+GSK2879552+in+Subjects+With+Relapsed%2FRefractory+Small+Cell+Lung+Carcinoma.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2013%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02034123%3Fterm%3DGSK2879552%26rank%3D1+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DInvestigation%2520of%2520GSK2879552%2520in%2520Subjects%2520With%2520Relapsed%252FRefractory%2520Small%2520Cell%2520Lung%2520Carcinoma%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span> </span><span class="NLM_article-title">An Open-Label, Dose-Escalation/Dose-Expansion Safety Study of INCB059872 in Subjects With Advanced Malignancies</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/study/NCT02712905" class="extLink">https://clinicaltrials.gov/ct2/show/study/NCT02712905</a> (accessed October 7,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=An+Open-Label%2C+Dose-Escalation%2FDose-Expansion+Safety+Study+of+INCB059872+in+Subjects+With+Advanced+Malignancies.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2016%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2Fstudy%2FNCT02712905+%28accessed+October+7%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DAn%2520Open-Label%252C%2520Dose-Escalation%252FDose-Expansion%2520Safety%2520Study%2520of%2520INCB059872%2520in%2520Subjects%2520With%2520Advanced%2520Malignancies%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2016%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Zuffo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrocco, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stazi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiró, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span> </span><span class="NLM_article-title">Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1501</span><span class="NLM_x">–</span> <span class="NLM_lpage">1517</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1501-1517&author=P.+Vianelloauthor=O.+A.+Botrugnoauthor=A.+Cappaauthor=R.+Dal+Zuffoauthor=P.+Dessantiauthor=A.+Maiauthor=B.+Marroccoauthor=A.+Matteviauthor=G.+Meroniauthor=S.+Minucciauthor=G.+Staziauthor=F.+Thalerauthor=P.+Trifir%C3%B3author=S.+Valenteauthor=M.+Villaauthor=M.+Varasiauthor=C.+Mercurio&title=Discovery+of+a+Novel+Inhibitor+of+Histone+Lysine-Specific+Demethylase+1A+%28KDM1A%2FLSD1%29+as+Orally+Active+Antitumor+Agent&doi=10.1021%2Facs.jmedchem.5b01209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent</span></div><div class="casAuthors">Vianello, Paola; Botrugno, Oronza A.; Cappa, Anna; Dal Zuffo, Roberto; Dessanti, Paola; Mai, Antonello; Marrocco, Biagina; Mattevi, Andrea; Meroni, Giuseppe; Minucci, Saverio; Stazi, Giulia; Thaler, Florian; Trifiro, Paolo; Valente, Sergio; Villa, Manuela; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1501-1517</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the stereoselective synthesis and biol. activity of a novel series of tranylcypromine (TCPA) derivs, potent inhibitors of KDM1A.  The new compds. strongly inhibit the clonogenic potential of acute leukemia cell lines.  In particular three mols. showing selectivity vs. MAO A and remarkably inhibiting colony formation in THP-1 human leukemia cells, were assessed in mouse for their preliminary pharmacokinetic.  Among them, compds. I and II were further tested in vivo in a murine acute promyelocytic leukemia model, resulting I the most effective.  Its two enantiomers were synthesized: the (1S,2R) enantiomer showed higher activity than its (1R,2S) analog, in both biochem. and cellular assays.  Compd. (1S,2R)-I exhibited in vivo efficacy after oral administration, detg. a 62% increased survival in mouse leukemia model with evidence of KDM1A inhibition.  The biol. profile of compd. (1S,2R)-I supports its further investigation as a cancer therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraSPzAcLQ0u7Vg90H21EOLACvtfcHk0ljRTuwcFMZI0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfM&md5=b7df567ef2cf939a1365874fccac9df0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01209%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DDal%2BZuffo%26aufirst%3DR.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DStazi%26aufirst%3DG.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTrifir%25C3%25B3%26aufirst%3DP.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Inhibitor%2520of%2520Histone%2520Lysine-Specific%2520Demethylase%25201A%2520%2528KDM1A%252FLSD1%2529%2520as%2520Orally%2520Active%2520Antitumor%2520Agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1501%26epage%3D1517%26doi%3D10.1021%2Facs.jmedchem.5b01209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+Inhibitors+of+LSD1+as+Therapeutic+Agents+in+Acute+Myeloid+Leukemia%3A+Clinical+Significance+and+Progress+to+Date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0ljRTuwcFMZI0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520Inhibitors%2520of%2520LSD1%2520as%2520Therapeutic%2520Agents%2520in%2520Acute%2520Myeloid%2520Leukemia%253A%2520Clinical%2520Significance%2520and%2520Progress%2520to%2520Date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Yu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hierl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yakowec, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwandner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emkey, R.</span><span> </span><span class="NLM_article-title">High-Throughput TR-FRET Assays for Identifying Inhibitors of LSD1 and JMJD2C Histone Lysine Demethylases</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1177/1087057111418228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1177%2F1087057111418228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=21859682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1OltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=27-38&author=V.+Yuauthor=T.+Fischauthor=A.+M.+Longauthor=J.+Tangauthor=J.+H.+Leeauthor=M.+Hierlauthor=H.+Chenauthor=P.+Yakowecauthor=R.+Schwandnerauthor=R.+Emkey&title=High-Throughput+TR-FRET+Assays+for+Identifying+Inhibitors+of+LSD1+and+JMJD2C+Histone+Lysine+Demethylases&doi=10.1177%2F1087057111418228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput TR-FRET assays for identifying inhibitors of LSD1 and JMJD2C histone lysine demethylases</span></div><div class="casAuthors">Yu, Violeta; Fisch, Tanja; Long, Alexander M.; Tang, Jin; Lee, Josie Han; Hierl, Markus; Chen, Hao; Yakowec, Peter; Schwandner, Ralf; Emkey, Renee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Lysine demethylase 1 (LSD1) and Jumonji C domain-contg. oxygenase D2C (JMJD2C) participate in regulating the methylation status of histone H3 lysine residues.  In some contexts, LSD1 and JMJD2C activity causes enhanced cellular proliferation, which may lead to tumorigenesis.  The authors explored the utility of time-resolved fluorescence resonance energy transfer (TR-FRET) immunoassays, which employed peptides consisting of the first 21 amino acids of histone H3 in which lysine 4 (H3K4) or lysine 9 (H3K9) was methylated (me) to quantify LSD1 and JMJD2C activity.  The LSD1 assay monitored demethylation of the H3K4me1 peptide using an antibody that recognizes H3K4me1 but not the unmethylated peptide product.  The JMJD2C assay measured demethylation of H3K9me3 with an antibody that selectively recognizes H3K9me2.  The optimized conditions resulted in robust assays (Z' > 0.7) that required only 3 to 6 nM of enzyme in a reaction vol. of 6 to 10 μL.  These assays were used to compare the activity of different LSD1 constructs and to det. the apparent Km of each JMJD2C substrate.  Finally, both assays were used in a high-throughput setting for identifying demethylase inhibitors.  Compds. discovered by these TR-FRET methods may lead to powerful tools for ascertaining the roles of demethylases in a cellular context and ultimately for potential cancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVw3snvTyahbVg90H21EOLACvtfcHk0liDzQcZ-Fj1Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1OltrY%253D&md5=fe4eb00acca43ce6633377e9720a5d83</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1177%2F1087057111418228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111418228%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DV.%26aulast%3DFisch%26aufirst%3DT.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DHierl%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DYakowec%26aufirst%3DP.%26aulast%3DSchwandner%26aufirst%3DR.%26aulast%3DEmkey%26aufirst%3DR.%26atitle%3DHigh-Throughput%2520TR-FRET%2520Assays%2520for%2520Identifying%2520Inhibitors%2520of%2520LSD1%2520and%2520JMJD2C%2520Histone%2520Lysine%2520Demethylases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D27%26epage%3D38%26doi%3D10.1177%2F1087057111418228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Vianello, P.; Sartori, L.; Amigoni, F.; Cappa, A.; Fagá, G.; Fattori, R.; Legnaghi, E.; Ciossani, G.; Mattevi, A.; Meroni, G.; Moretti, L.; Cecatiello, V.; Pasqualato, S.; Romussi, A.; Thaler, F.; Trifirò, P.; Vultaggio, S.; Varasi, M.; Mercurio, C.</span><span> </span><span class="NLM_article-title">Thieno[3,2-B]pyrrole-5-Carboxamides as Novel Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship</span>.  <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span>, DOI <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01019</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&author=P.+Vianello&author=L.+Sartori&author=F.+Amigoni&author=A.+Cappa&author=G.+Fag%C3%A1&author=R.+Fattori&author=E.+Legnaghi&author=G.+Ciossani&author=A.+Mattevi&author=G.+Meroni&author=L.+Moretti&author=V.+Cecatiello&author=S.+Pasqualato&author=A.+Romussi&author=F.+Thaler&author=P.+Trifir%C3%B2&author=S.+Vultaggio&author=M.+Varasi&author=C.+Mercurio&title=Thieno%5B3%2C2-B%5Dpyrrole-5-Carboxamides+as+Novel+Reversible+Inhibitors+of+Histone+Lysine+Demethylase+KDM1A%2FLSD1.+Part+2%3A+Structure-Based+Drug+Design+and+Structure%E2%80%93Activity+Relationship&doi=10.1021%2Facs.jmedchem.6b01019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26atitle%3DThieno%255B3%252C2-B%255Dpyrrole-5-Carboxamides%2520as%2520Novel%2520Reversible%2520Inhibitors%2520of%2520Histone%2520Lysine%2520Demethylase%2520KDM1A%252FLSD1.%2520Part%25202%253A%2520Structure-Based%2520Drug%2520Design%2520and%2520Structure%25E2%2580%2593Activity%2520Relationship%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26doi%3D10.1021%2Facs.jmedchem.6b01019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G.</span><span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+Baellauthor=G.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0liDzQcZ-Fj1Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DHolloway%26aufirst%3DG.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Pouliot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeanmart, S.</span><span> </span><span class="NLM_article-title">Pan Assay Interference Compounds (PAINS) and Other Promiscuous Compounds in Antifungal Research</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=497-503&author=M.+Pouliotauthor=S.+Jeanmart&title=Pan+Assay+Interference+Compounds+%28PAINS%29+and+Other+Promiscuous+Compounds+in+Antifungal+Research&doi=10.1021%2Facs.jmedchem.5b00361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Pan Assay Interference Compounds (PAINS) and Other Promiscuous Compounds in Antifungal Research</span></div><div class="casAuthors">Pouliot, Martin; Jeanmart, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">497-503</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Every week, articles disclosing new antifungal leads reported as promising starting points for optimization projects are published.  In many cases, the mechanism that accounts for their antifungal activity has not been fully elucidated.  More significantly, the detrimental impact that could result from certain embedded chem. features has been underestimated or even overlooked.  In the course of the authors' research in the agrochem. area, the authors have concluded that in many cases such leads are actually nonoptimizable because they either contain what are now recognized as pan assay interference compds. (PAINS) or other promiscuous groups.  This article is aimed at highlighting the pitfalls the authors have encountered and hopefully to steer other research groups away from them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqURFxdfKNQIbVg90H21EOLACvtfcHk0liDzQcZ-Fj1Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7rE&md5=af11fcafd47de435fcc5fdf05080023a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00361%26sid%3Dliteratum%253Aachs%26aulast%3DPouliot%26aufirst%3DM.%26aulast%3DJeanmart%26aufirst%3DS.%26atitle%3DPan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520and%2520Other%2520Promiscuous%2520Compounds%2520in%2520Antifungal%2520Research%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D497%26epage%3D503%26doi%3D10.1021%2Facs.jmedchem.5b00361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Malo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanley, J. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerquozzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadon, R.</span><span> </span><span class="NLM_article-title">Statistical Practice in High-Throughput Screening Data Analysis</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span><span class="refDoi"> DOI: 10.1038/nbt1186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1038%2Fnbt1186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=16465162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFGqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=167-175&author=N.+Maloauthor=J.+a+Hanleyauthor=S.+Cerquozziauthor=J.+Pelletierauthor=R.+Nadon&title=Statistical+Practice+in+High-Throughput+Screening+Data+Analysis&doi=10.1038%2Fnbt1186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical practice in high-throughput screening data analysis</span></div><div class="casAuthors">Malo, Nathalie; Hanley, James A.; Cerquozzi, Sonia; Pelletier, Jerry; Nadon, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-175</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening is an early crit. step in drug discovery.  Its aim is to screen a large no. of diverse chem. compds. to identify candidate 'hits' rapidly and accurately.  Few statistical tools are currently available, however, to detect quality hits with a high degree of confidence.  We examine statistical aspects of data preprocessing and hit identification for primary screens.  We focus on concerns related to positional effects of wells within plates, choice of hit threshold and the importance of minimizing false-pos. and false-neg. rates.  We argue that replicate measurements are needed to verify assumptions of current methods and to suggest data anal. strategies when assumptions are not met.  The integration of replicates with robust statistical methods in primary screens will facilitate the discovery of reliable hits, ultimately improving the sensitivity and specificity of the screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXXI5Qnc2Z9rVg90H21EOLACvtfcHk0lhE7qLZ8cDW0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFGqs70%253D&md5=170a0238341aba8add873d4f74196d18</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnbt1186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1186%26sid%3Dliteratum%253Aachs%26aulast%3DMalo%26aufirst%3DN.%26aulast%3DHanley%26aufirst%3DJ.%2Ba%26aulast%3DCerquozzi%26aufirst%3DS.%26aulast%3DPelletier%26aufirst%3DJ.%26aulast%3DNadon%26aufirst%3DR.%26atitle%3DStatistical%2520Practice%2520in%2520High-Throughput%2520Screening%2520Data%2520Analysis%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D167%26epage%3D175%26doi%3D10.1038%2Fnbt1186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span>BIOVIA Pipeline Pilot | Scientific Workflow Authoring Application for Data Analysis; <span class="NLM_publisher-name">Biovia</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">; </span><a href="http://accelrys.com/products/collaborative-science/biovia-pipeline-pilot/" class="extLink">http://accelrys.com/products/collaborative-science/biovia-pipeline-pilot/</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BIOVIA+Pipeline+Pilot+%7C+Scientific+Workflow+Authoring+Application+for+Data+Analysis%3B+Biovia%3A+San+Diego%2C+CA%2C+2017%3B+http%3A%2F%2Faccelrys.com%2Fproducts%2Fcollaborative-science%2Fbiovia-pipeline-pilot%2F+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DBIOVIA%2520Pipeline%2520Pilot%2520%257C%2520Scientific%2520Workflow%2520Authoring%2520Application%2520for%2520Data%2520Analysis%26pub%3DBiovia%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Copeland, R. A.</span> <span class="citation_source-book">Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</span>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>, <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">20070</span><span class="NLM_x">–</span> <span class="NLM_lpage">20074</span><span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+Basis+of+LSD1-CoREST+Selectivity+in+Histone+H3+Recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0lhE7qLZ8cDW0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520LSD1-CoREST%2520Selectivity%2520in%2520Histone%2520H3%2520Recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Karasulu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiel, W.</span><span> </span><span class="NLM_article-title">Amine Oxidation Mediated by Lysine-Specific Demethylase 1: Quantum Mechanics/molecular Mechanics Insights into Mechanism and Role of Lysine 661</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">13400</span><span class="NLM_x">–</span> <span class="NLM_lpage">13413</span><span class="refDoi"> DOI: 10.1021/ja403582u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja403582u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=13400-13413&author=B.+Karasuluauthor=M.+Patilauthor=W.+Thiel&title=Amine+Oxidation+Mediated+by+Lysine-Specific+Demethylase+1%3A+Quantum+Mechanics%2Fmolecular+Mechanics+Insights+into+Mechanism+and+Role+of+Lysine+661&doi=10.1021%2Fja403582u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja403582u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja403582u%26sid%3Dliteratum%253Aachs%26aulast%3DKarasulu%26aufirst%3DB.%26aulast%3DPatil%26aufirst%3DM.%26aulast%3DThiel%26aufirst%3DW.%26atitle%3DAmine%2520Oxidation%2520Mediated%2520by%2520Lysine-Specific%2520Demethylase%25201%253A%2520Quantum%2520Mechanics%252Fmolecular%2520Mechanics%2520Insights%2520into%2520Mechanism%2520and%2520Role%2520of%2520Lysine%2520661%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D13400%26epage%3D13413%26doi%3D10.1021%2Fja403582u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Morris, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated Docking with Selective Receptor Flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span><span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+Docking+with+Selective+Receptor+Flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0liCkbfldoRpuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520Docking%2520with%2520Selective%2520Receptor%2520Flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stützle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/ci800298z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800298z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=84-96&author=O.+Korbauthor=T.+St%C3%BCtzleauthor=T.+E.+Exner&title=Empirical+Scoring+Functions+for+Advanced+Protein-Ligand+Docking+with+PLANTS&doi=10.1021%2Fci800298z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span></div><div class="casAuthors">Korb, Oliver; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we present two empirical scoring functions, PLANTSCHEMPLP and PLANTSPLP, designed for our docking algorithm PLANTS (Protein-Ligand ANT System), which is based on ant colony optimization (ACO).  They are related, regarding their functional form, to parts of already published scoring functions and force fields.  The parametrization procedure described here was able to identify several parameter settings showing an excellent performance for the task of pose prediction on two test sets comprising 298 complexes in total.  Up to 87% of the complexes of the Astex diverse set and 77% of the CCDC/Astex clean listnc (noncovalently bound complexes of the clean list) could be reproduced with root-mean-square deviations of less than 2 Å with respect to the exptl. detd. structures.  A comparison with the state-of-the-art docking tool GOLD clearly shows that this is, esp. for the druglike Astex diverse set, an improvement in pose prediction performance.  Addnl., optimized parameter settings for the search algorithm were identified, which can be used to balance pose prediction reliability and search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_kyR7CGk9mbVg90H21EOLACvtfcHk0liCkbfldoRpuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D&md5=fce960b75b9f8906354b7c97ff84c092</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci800298z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800298z%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DSt%25C3%25BCtzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DEmpirical%2520Scoring%2520Functions%2520for%2520Advanced%2520Protein-Ligand%2520Docking%2520with%2520PLANTS%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D84%26epage%3D96%26doi%3D10.1021%2Fci800298z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0liCkbfldoRpuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">AClAP, Autonomous Hierarchical Agglomerative Cluster Analysis Based Protocol to Partition Conformational Datasets</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">e58</span><span class="NLM_x">–</span> <span class="NLM_lpage">e65</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btl212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1093%2Fbioinformatics%2Fbtl212" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=e58-e65&author=G.+Bottegoniauthor=W.+Rocchiaauthor=M.+Recanatiniauthor=A.+Cavalli&title=AClAP%2C+Autonomous+Hierarchical+Agglomerative+Cluster+Analysis+Based+Protocol+to+Partition+Conformational+Datasets&doi=10.1093%2Fbioinformatics%2Fbtl212"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtl212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtl212%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DAClAP%252C%2520Autonomous%2520Hierarchical%2520Agglomerative%2520Cluster%2520Analysis%2520Based%2520Protocol%2520to%2520Partition%2520Conformational%2520Datasets%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3De58%26epage%3De65%26doi%3D10.1093%2Fbioinformatics%2Fbtl212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Degliesposti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Portioli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parenti, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span> </span><span class="NLM_article-title">BEAR, a Novel Virtual Screening Methodology for Drug Discovery</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1177/1087057110388276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1177%2F1087057110388276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=21084717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsl2gsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=129-133&author=G.+Degliespostiauthor=C.+Portioliauthor=M.+D.+Parentiauthor=G.+Rastelli&title=BEAR%2C+a+Novel+Virtual+Screening+Methodology+for+Drug+Discovery&doi=10.1177%2F1087057110388276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">BEAR, a novel virtual screening methodology for drug discovery</span></div><div class="casAuthors">Degliesposti, Gianluca; Portioli, Corinne; Parenti, Marco Daniele; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-133</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">BEAR (binding estn. after refinement) is a new virtual screening technol. based on the conformational refinement of docking poses through mol. dynamics and prediction of binding free energies using accurate scoring functions.  Here, the authors report the results of an extensive benchmark of the BEAR performance in identifying a smaller subset of known inhibitors seeded in a large (1.5 million) database of compds.  BEAR performance proved strikingly better if compared with std. docking screening methods.  The validations performed so far showed that BEAR is a reliable tool for drug discovery.  It is fast, modular, and automated, and it can be applied to virtual screenings against any biol. target with known structure and any database of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanbHvF9oSELVg90H21EOLACvtfcHk0lipGx0N6drT4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsl2gsLk%253D&md5=d07d88fff7b821ad76c0557ae2460903</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1177%2F1087057110388276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057110388276%26sid%3Dliteratum%253Aachs%26aulast%3DDegliesposti%26aufirst%3DG.%26aulast%3DPortioli%26aufirst%3DC.%26aulast%3DParenti%26aufirst%3DM.%2BD.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DBEAR%252C%2520a%2520Novel%2520Virtual%2520Screening%2520Methodology%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2011%26volume%3D16%26spage%3D129%26epage%3D133%26doi%3D10.1177%2F1087057110388276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Heffernan, M. L. R.; Foglesong, R. J.; Hopkins, S. C.; Soukri, M.; Jones, S. W.; Spear, K. L.; Varney, M. A.</span><span> </span><span class="NLM_article-title">Fluoro-Substituted Inhibitors of D-Amino Acid Oxidase</span>. U.S. Patent 7,884,124 B2,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=M.+L.+R.+Heffernan&author=R.+J.+Foglesong&author=S.+C.+Hopkins&author=M.+Soukri&author=S.+W.+Jones&author=K.+L.+Spear&author=M.+A.+Varney&title=Fluoro-Substituted+Inhibitors+of+D-Amino+Acid+Oxidase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHeffernan%26aufirst%3DM.%2BL.%2BR.%26atitle%3DFluoro-Substituted%2520Inhibitors%2520of%2520D-Amino%2520Acid%2520Oxidase%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Ching, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kam, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merits, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, L. F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, C. L. L.</span><span> </span><span class="NLM_article-title">Trisubstituted thieno[3,2-B]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9196</span><span class="NLM_x">–</span> <span class="NLM_lpage">9213</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9196-9213&author=K.+C.+Chingauthor=Y.+W.+Kamauthor=A.+Meritsauthor=L.+F.+P.+Ngauthor=C.+L.+L.+Chai&title=Trisubstituted+thieno%5B3%2C2-B%5Dpyrrole+5-Carboxamides+as+Potent+Inhibitors+of+Alphaviruses&doi=10.1021%2Facs.jmedchem.5b01047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Trisubstituted Thieno[3,2-b]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span></div><div class="casAuthors">Ching, Kuan-Chieh; Kam, Yiu-Wing; Merits, Andres; Ng, Lisa F. P.; Chai, Christina L. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9196-9213</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chikungunya virus (CHIKV) is a re-emerging vector-borne alphavirus and is transmitted to humans by Aedes mosquitoes.  Despite the re-emergence of CHIKV as an epidemic threat, there is no approved effective antiviral treatment currently available for CHIKV.  Herein, we report the synthesis and structure-activity relationship studies of a class of thieno[3,2-b]pyrroles and the discovery of a trisubstituted thieno[3,2-b]pyrrole 5-carboxamide 15c (I) that exhibits potent inhibitory activity against in vitro CHIKV infection.  Compd. 15c displayed low micromolar activity (EC50 value of ca. 2 μM) and limited cytotoxic liability (CC50 > 100 μM) therefore furnishing a selectivity index of greater than 32.  Notably, 15c not only controlled viral RNA prodn., but efficiently inhibited the expression of CHIKV nsP1, nsP3, capsid, and E2 proteins at a concn. as low as 2.5 μM.  More importantly, 15c also demonstrated broad spectrum antiviral activity against other clin. important alphaviruses such as O'nyong-nyong virus and Sindbis virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC7wEIPTmwQ7Vg90H21EOLACvtfcHk0lipGx0N6drT4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI&md5=dc938296fd5fd7628b7de37590b6918a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01047%26sid%3Dliteratum%253Aachs%26aulast%3DChing%26aufirst%3DK.%2BC.%26aulast%3DKam%26aufirst%3DY.%2BW.%26aulast%3DMerits%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DL.%2BF.%2BP.%26aulast%3DChai%26aufirst%3DC.%2BL.%2BL.%26atitle%3DTrisubstituted%2520thieno%255B3%252C2-B%255Dpyrrole%25205-Carboxamides%2520as%2520Potent%2520Inhibitors%2520of%2520Alphaviruses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9196%26epage%3D9213%26doi%3D10.1021%2Facs.jmedchem.5b01047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+P.+Somervaille&title=The+Histone+Demethylase+KDM1A+Sustains+the+Oncogenic+Potential+of+MLL-AF9+Leukemia+Stem+Cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0lipGx0N6drT4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DThe%2520Histone%2520Demethylase%2520KDM1A%2520Sustains%2520the%2520Oncogenic%2520Potential%2520of%2520MLL-AF9%2520Leukemia%2520Stem%2520Cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanigan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Loss of LSD1 (Lysine-Specific Demethylase 1) Suppresses Growth and Alters Gene Expression of Human Colon Cancer Cells in a p53- and DNMT1(DNA Methyltransferase 1)-Independent Manner</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">459</span><span class="NLM_x">–</span> <span class="NLM_lpage">468</span><span class="refDoi"> DOI: 10.1042/BJ20121360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1042%2FBJ20121360" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=459-468&author=L.+Jinauthor=C.+L.+Haniganauthor=Y.+Wuauthor=W.+Wangauthor=B.+H.+Parkauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Loss+of+LSD1+%28Lysine-Specific+Demethylase+1%29+Suppresses+Growth+and+Alters+Gene+Expression+of+Human+Colon+Cancer+Cells+in+a+p53-+and+DNMT1%28DNA+Methyltransferase+1%29-Independent+Manner&doi=10.1042%2FBJ20121360"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2FBJ20121360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121360%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHanigan%26aufirst%3DC.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DLoss%2520of%2520LSD1%2520%2528Lysine-Specific%2520Demethylase%25201%2529%2520Suppresses%2520Growth%2520and%2520Alters%2520Gene%2520Expression%2520of%2520Human%2520Colon%2520Cancer%2520Cells%2520in%2520a%2520p53-%2520and%2520DNMT1%2528DNA%2520Methyltransferase%25201%2529-Independent%2520Manner%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D459%26epage%3D468%26doi%3D10.1042%2FBJ20121360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone Demethylase Lsd1 Represses Hematopoietic Stem and Progenitor Cell Signatures during Blood Cell Maturation</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00633</span><span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.-J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+Demethylase+Lsd1+Represses+Hematopoietic+Stem+and+Progenitor+Cell+Signatures+during+Blood+Cell+Maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0ljDC2JVBcyKCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.-J.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520Demethylase%2520Lsd1%2520Represses%2520Hematopoietic%2520Stem%2520and%2520Progenitor%2520Cell%2520Signatures%2520during%2520Blood%2520Cell%2520Maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">LSD1: Oxidative Chemistry for Multifaceted Functions in Chromatin Regulation</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">189</span><span class="refDoi"> DOI: 10.1016/j.tibs.2008.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.tibs.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=18343668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktlCkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=181-189&author=F.+Fornerisauthor=C.+Bindaauthor=E.+Battaglioliauthor=A.+Mattevi&title=LSD1%3A+Oxidative+Chemistry+for+Multifaceted+Functions+in+Chromatin+Regulation&doi=10.1016%2Fj.tibs.2008.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1: oxidative chemistry for multifaceted functions in chromatin regulation</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">181-189</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Three years after its discovery, lysine-specific demethylase 1 remains at the forefront of chromatin research.  Its demethylase activity on Lys4 of histone H3 supports its role in gene repression.  By contrast, the biochem. mechanisms underlying lysine-specific demethylase 1 involvement in transcriptional activation are not firmly established.  Structural studies highlight a specific binding site for the histone H3 N-terminal tail and a catalytic machinery that is closely related to that of other flavin-dependent amine oxidases.  These insights are crucial for the development of demethylation inhibitors.  Furthermore, the exploration of putative non-histone substrates and potential signaling roles of hydrogen peroxide produced by the demethylation reaction could lead to new paradigms in chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNay__PwncNLVg90H21EOLACvtfcHk0ljDC2JVBcyKCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktlCkurk%253D&md5=dc5b6d422b49b41f67b95ca812ea841f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DLSD1%253A%2520Oxidative%2520Chemistry%2520for%2520Multifaceted%2520Functions%2520in%2520Chromatin%2520Regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D181%26epage%3D189%26doi%3D10.1016%2Fj.tibs.2008.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+Structural%2C+and+Biological+Evaluation+of+Tranylcypromine+Derivatives+as+Inhibitors+of+Histone+Demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0lh92mkbPIuzCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520Structural%252C%2520and%2520Biological%2520Evaluation%2520of%2520Tranylcypromine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Histone%2520Demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lh92mkbPIuzCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span>SigmaPlot, <span class="NLM_edition">v11</span>; <span class="NLM_publisher-name">Systat Software Inc.</span>: <span class="NLM_publisher-loc">San Jose, CA</span>; <a href="https://systatsoftware.com/products/sigmaplot/" class="extLink">https://systatsoftware.com/products/sigmaplot/</a> (accessed July 20,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SigmaPlot%2C+v11%3B+Systat+Software+Inc.%3A+San+Jose%2C+CA%3B+https%3A%2F%2Fsystatsoftware.com%2Fproducts%2Fsigmaplot%2F+%28accessed+July+20%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSigmaPlot%26pub%3DSystat%2520Software%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Moretti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartori, L.</span><span> </span><span class="NLM_article-title">Software Infrastructure for Computer-Aided Drug Discovery and Development, a Practical Example with Guidelines</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">390</span><span class="refDoi"> DOI: 10.1002/minf.201501037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fminf.201501037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=27546042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=382-390&author=L.+Morettiauthor=L.+Sartori&title=Software+Infrastructure+for+Computer-Aided+Drug+Discovery+and+Development%2C+a+Practical+Example+with+Guidelines&doi=10.1002%2Fminf.201501037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Software Infrastructure for Computer-aided Drug Discovery and Development, a Practical Example with Guidelines</span></div><div class="casAuthors">Moretti, Loris; Sartori, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">382-390</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In the field of Computer-Aided Drug Discovery and Development (CADDD) the proper software infrastructure is essential for everyday investigations.  The creation of such an environment should be carefully planned and implemented with certain features in order to be productive and efficient.  Here, we describe a soln. to integrate std. computational services into a functional unit that empowers modeling applications for drug discovery.  This system allows users with various level of expertise to run in silico expts. automatically and without the burden of file formatting for different software, managing the actual computation, keeping track of the activities and graphical rendering of the structural outcomes.  To showcase the potential of this approach, performances of five different docking programs on an Hiv-1 protease test set are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUtVBT1Wj49LVg90H21EOLACvtfcHk0lh92mkbPIuzCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL&md5=c9db5ce783d44c6d15f889cbec5ca3f7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fminf.201501037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201501037%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DSartori%26aufirst%3DL.%26atitle%3DSoftware%2520Infrastructure%2520for%2520Computer-Aided%2520Drug%2520Discovery%2520and%2520Development%252C%2520a%2520Practical%2520Example%2520with%2520Guidelines%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D382%26epage%3D390%26doi%3D10.1002%2Fminf.201501037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Seeliger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Groot, B. L.</span><span> </span><span class="NLM_article-title">Ligand Docking and Binding Site Analysis with PyMOL and Autodock/Vina</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">417</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1007/s10822-010-9352-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1007%2Fs10822-010-9352-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=20401516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=417-422&author=D.+Seeligerauthor=B.+L.+De+Groot&title=Ligand+Docking+and+Binding+Site+Analysis+with+PyMOL+and+Autodock%2FVina&doi=10.1007%2Fs10822-010-9352-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand docking and binding site analysis with PyMOL and Autodock/Vina</span></div><div class="casAuthors">Seeliger, Daniel; Groot, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">417-422</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Docking of small mol. compds. into the binding site of a receptor and estg. the binding affinity of the complex is an important part of the structure-based drug design process.  For a thorough understanding of the structural principles that det. the strength of a protein/ligand complex both, an accurate and fast docking protocol and the ability to visualize binding geometries and interactions are mandatory.  Here we present an interface between the popular mol. graphics system PyMOL and the mol. docking suites Autodock and Vina and demonstrate how the combination of docking and visualization can aid structure-based drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTR4WZx4OqG7Vg90H21EOLACvtfcHk0lj3mQ4UXf-3hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D&md5=35f8b79d6c1d6582919878a53b4a5129</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9352-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9352-6%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3DDe%2BGroot%26aufirst%3DB.%2BL.%26atitle%3DLigand%2520Docking%2520and%2520Binding%2520Site%2520Analysis%2520with%2520PyMOL%2520and%2520Autodock%252FVina%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D417%26epage%3D422%26doi%3D10.1007%2Fs10822-010-9352-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span>R, A Language and Environment for Statistical Computing; <span class="NLM_publisher-name">R Foundation for Statistical Computing</span>: <span class="NLM_publisher-loc">Vienna, Austria</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.r-project.org/" class="extLink">http://www.r-project.org/</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R%2C+A+Language+and+Environment+for+Statistical+Computing%3B+R+Foundation+for+Statistical+Computing%3A+Vienna%2C+Austria%2C+2016%3B+http%3A%2F%2Fwww.r-project.org%2F+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DR%252C%2520A%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DR%2520Foundation%2520for%2520Statistical%2520Computing%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Case, D. A.; Berryman, J. T.; Betz, R. M.; Cerutti, D. S.; Cheatham, T. E.,  III; Darden, T. A.; Duke, R. E.; Giese, T. J.; Gohlke, H.; Goetz, A. W.; Homeyer, N.; Izadi, S.; Janowski, P.; Kaus, J.; Kovalenko, A.; Lee, T. S.; LeGrand, S.; Li, P.; Luchko, T.; Luo, R.; Madej, B.; Merz, K. M.; Monard, G.; Needham, P.; Nguyen, H.; Nguyen, H. T.; Omelyan, I.; Onufriev, A.; Roe, D. R.; Roitberg, A.; Salomon-Ferrer, R.; Simmerling, C.L.; Smith, W.; Swails, J.; Walker, R.C.; Wang, J.; Wolf, R. M.; Wu, X.; York, D.M.; Kollman, P.A.</span>AMBER 2015; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=D.+A.+Case&author=J.+T.+Berryman&author=R.+M.+Betz&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=T.+J.+Giese&author=H.+Gohlke&author=A.+W.+Goetz&author=N.+Homeyer&author=S.+Izadi&author=P.+Janowski&author=J.+Kaus&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=G.+Monard&author=P.+Needham&author=H.+Nguyen&author=H.+T.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=D.+R.+Roe&author=A.+Roitberg&author=R.+Salomon-Ferrer&author=C.L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=D.M.+York&author=P.A.+Kollman&title=AMBER+2015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%25202015%26pub%3DUniversity%2520of%2520California%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera--a Visualization System for Exploratory Research and Analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span><span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%2D%2Da+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lj3mQ4UXf-3hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera--a%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Antonow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrafa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawood, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haque, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thurston, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinzalla, G.</span><span> </span><span class="NLM_article-title">Facile Oxidation of Electron-Poor Benzo[b]thiophenes to the Corresponding Sulfones with an Aqueous Solution of H<sub>2</sub>O<sub>2</sub> and P<sub>2</sub>O<sub>5</sub></span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2289</span><span class="NLM_x">–</span> <span class="NLM_lpage">2291</span><span class="refDoi"> DOI: 10.1039/b924333j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1039%2Fb924333j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=20234935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1Khsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2010&pages=2289-2291&author=D.+Antonowauthor=T.+Marrafaauthor=I.+Dawoodauthor=T.+Ahmedauthor=M.+R.+Haqueauthor=D.+E.+Thurstonauthor=G.+Zinzalla&title=Facile+Oxidation+of+Electron-Poor+Benzo%5Bb%5Dthiophenes+to+the+Corresponding+Sulfones+with+an+Aqueous+Solution+of+H2O2+and+P2O5&doi=10.1039%2Fb924333j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Facile oxidation of electron-poor benzo[b]thiophenes to the corresponding sulfones with an aqueous solution of H2O2 and P2O5</span></div><div class="casAuthors">Antonow, Dyeison; Marrafa, Teresa; Dawood, Irfaan; Ahmed, Tauheed; Haque, Mohammad R.; Thurston, David E.; Zinzalla, Giovanna</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2289-2291</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A facile oxidn. for the clean conversion of benzo[b]thiophenes to their corresponding sulfones e. g., I is described employing an aq. soln. of H2O2 and P2O5; the soln. can be prepd. and stored on a multi-gram scale with a shelf-life of up to two weeks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf5ccivyBLS7Vg90H21EOLACvtfcHk0ljPcuRssMeMXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1Khsbs%253D&md5=aa121a500c4f905bbdaf42c68159a565</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1039%2Fb924333j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb924333j%26sid%3Dliteratum%253Aachs%26aulast%3DAntonow%26aufirst%3DD.%26aulast%3DMarrafa%26aufirst%3DT.%26aulast%3DDawood%26aufirst%3DI.%26aulast%3DAhmed%26aufirst%3DT.%26aulast%3DHaque%26aufirst%3DM.%2BR.%26aulast%3DThurston%26aufirst%3DD.%2BE.%26aulast%3DZinzalla%26aufirst%3DG.%26atitle%3DFacile%2520Oxidation%2520of%2520Electron-Poor%2520Benzo%255Bb%255Dthiophenes%2520to%2520the%2520Corresponding%2520Sulfones%2520with%2520an%2520Aqueous%2520Solution%2520of%2520H2O2%2520and%2520P2O5%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2010%26volume%3D46%26spage%3D2289%26epage%3D2291%26doi%3D10.1039%2Fb924333j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 39 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Lei-Peng Xue, Xiao-Peng Xiong, Ying Zhou, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2466-2488. <a href="https://doi.org/10.1021/acs.jmedchem.0c02176" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02176%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DReversible%252BLysine%252BSpecific%252BDemethylase%252B1%252B%252528LSD1%252529%252BInhibitors%25253A%252BA%252BPromising%252BWrench%252Bto%252BImpair%252BLSD1%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D23022021%26volume%3D64%26issue%3D5%26spage%3D2466%26epage%3D2488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alessia Romussi, Anna Cappa, Paola Vianello, Silvia Brambillasca, Maria Rosaria Cera, Roberto Dal Zuffo, Giovanni Fagà, Raimondo Fattori, Loris Moretti, Paolo Trifirò, Manuela Villa, Stefania Vultaggio, Valentina Cecatiello, Sebastiano Pasqualato, Giulio Dondio, Chi Wai Eric So, Saverio Minucci, Luca Sartori, Mario Varasi, <span class="NLM_string-name hlFld-ContribAuthor">Ciro Mercurio</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Reversible Inhibitors of KDM1A Efficacious in Acute Myeloid Leukemia Models. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 754-759. <a href="https://doi.org/10.1021/acsmedchemlett.9b00604" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00604</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00604%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BReversible%252BInhibitors%252Bof%252BKDM1A%252BEfficacious%252Bin%252BAcute%252BMyeloid%252BLeukemia%252BModels%26aulast%3DRomussi%26aufirst%3DAlessia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D16122019%26date%3D13022020%26date%3D20022020%26date%3D13022020%26volume%3D11%26issue%3D5%26spage%3D754%26epage%3D759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel P.  Mould</span>, <span class="hlFld-ContribAuthor ">Cristina  Alli</span>, <span class="hlFld-ContribAuthor ">Ulf  Bremberg</span>, <span class="hlFld-ContribAuthor ">Sharon  Cartic</span>, <span class="hlFld-ContribAuthor ">Allan M.  Jordan</span>, <span class="hlFld-ContribAuthor ">Matthis  Geitmann</span>, <span class="hlFld-ContribAuthor ">Alba  Maiques-Diaz</span>, <span class="hlFld-ContribAuthor ">Alison E.  McGonagle</span>, <span class="hlFld-ContribAuthor ">Tim C. P.  Somervaille</span>, <span class="hlFld-ContribAuthor ">Gary J.  Spencer</span>, <span class="hlFld-ContribAuthor ">Fabrice  Turlais</span>, and <span class="hlFld-ContribAuthor ">Donald  Ogilvie</span>  . </span><span class="cited-content_cbyCitation_article-title">Development of (4-Cyanophenyl)glycine Derivatives as Reversible Inhibitors of Lysine Specific Demethylase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (19)
                                     , 7984-7999. <a href="https://doi.org/10.1021/acs.jmedchem.7b00462" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00462</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00462%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252B%2525284-Cyanophenyl%252529glycine%252BDerivatives%252Bas%252BReversible%252BInhibitors%252Bof%252BLysine%252BSpecific%252BDemethylase%252B1%26aulast%3DMould%26aufirst%3DDaniel%2BP.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D28032017%26date%3D21092017%26date%3D12102017%26date%3D11092017%26volume%3D60%26issue%3D19%26spage%3D7984%26epage%3D7999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Luca  Sartori</span>, <span class="hlFld-ContribAuthor ">Federica  Amigoni</span>, <span class="hlFld-ContribAuthor ">Anna  Cappa</span>, <span class="hlFld-ContribAuthor ">Giovanni  Fagá</span>, <span class="hlFld-ContribAuthor ">Raimondo  Fattori</span>, <span class="hlFld-ContribAuthor ">Elena  Legnaghi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Meroni</span>, <span class="hlFld-ContribAuthor ">Loris  Moretti</span>, <span class="hlFld-ContribAuthor ">Valentina  Cecatiello</span>, <span class="hlFld-ContribAuthor ">Sebastiano  Pasqualato</span>, <span class="hlFld-ContribAuthor ">Alessia  Romussi</span>, <span class="hlFld-ContribAuthor ">Florian  Thaler</span>, <span class="hlFld-ContribAuthor ">Paolo  Trifiró</span>, <span class="hlFld-ContribAuthor ">Manuela  Villa</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Stefania  Vultaggio</span>, <span class="hlFld-ContribAuthor ">Elisa  Zagarrí</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, and <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>  . </span><span class="cited-content_cbyCitation_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1693-1715. <a href="https://doi.org/10.1021/acs.jmedchem.6b01019" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxamides%252Bas%252BNew%252BReversible%252BInhibitors%252Bof%252BHistone%252BLysine%252BDemethylase%252BKDM1A%25252FLSD1.%252BPart%252B2%25253A%252BStructure-Based%252BDrug%252BDesign%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%26aulast%3DVianello%26aufirst%3DPaola%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D26072016%26date%3D27022017%26date%3D09032017%26date%3D10022017%26volume%3D60%26issue%3D5%26spage%3D1693%26epage%3D1715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Hailan  Huang</span>, <span class="hlFld-ContribAuthor ">Ziheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Yixiang  Sun</span>, <span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Yanting  Gu</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel benzofuran derivatives as potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113501. <a href="https://doi.org/10.1016/j.ejmech.2021.113501" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113501</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113501%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bbenzofuran%252Bderivatives%252Bas%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26volume%3D220%26spage%3D113501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abdel Aziz Qasem  Jbarah</span>. </span><span class="cited-content_cbyCitation_article-title">The electrosorption of 3-bromo-2-nitrothiophene on gold as studied with surface-enhanced Raman spectroscopy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Sciences</span><span> <strong>2021,</strong> <em>133 </em>
                                    (2)
                                     <a href="https://doi.org/10.1007/s12039-021-01911-8" title="DOI URL">https://doi.org/10.1007/s12039-021-01911-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12039-021-01911-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12039-021-01911-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Sciences%26atitle%3DThe%252Belectrosorption%252Bof%252B3-bromo-2-nitrothiophene%252Bon%252Bgold%252Bas%252Bstudied%252Bwith%252Bsurface-enhanced%252BRaman%252Bspectroscopy%26aulast%3DJbarah%26aufirst%3DAbdel%2BAziz%2BQasem%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Fang</span>, <span class="hlFld-ContribAuthor ">Chunsheng  Hu</span>, <span class="hlFld-ContribAuthor ">Yong  Ding</span>, <span class="hlFld-ContribAuthor ">Hongxia  Qin</span>, <span class="hlFld-ContribAuthor ">Yafei  Luo</span>, <span class="hlFld-ContribAuthor ">Zhigang  Xu</span>, <span class="hlFld-ContribAuthor ">Jiangping  Meng</span>, <span class="hlFld-ContribAuthor ">Zhongzhu  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of
              
                4
                H
              
              ‐thieno[3,2‐
              b
              ]pyrrole derivatives as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2021,</strong> <em>15 </em><a href="https://doi.org/10.1002/jhet.4285" title="DOI URL">https://doi.org/10.1002/jhet.4285</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.4285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.4285%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DDiscovery%252Bof%252B4%252BH%252B%2525E2%252580%252590thieno%25255B3%25252C2%2525E2%252580%252590%252Bb%252B%25255Dpyrrole%252Bderivatives%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DFang%26aufirst%3DBo%26date%3D2021%26date%3D2021%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking, 3D-QSAR, and molecular dynamics simulations of thieno[3,2-b]pyrrole derivatives against anticancer targets of KDM1A/LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>39 </em>
                                    (4)
                                     , 1189-1202. <a href="https://doi.org/10.1080/07391102.2020.1726819" title="DOI URL">https://doi.org/10.1080/07391102.2020.1726819</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1726819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1726819%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DMolecular%252Bdocking%25252C%252B3D-QSAR%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bof%252Bthieno%25255B3%25252C2-b%25255Dpyrrole%252Bderivatives%252Bagainst%252Banticancer%252Btargets%252Bof%252BKDM1A%25252FLSD1%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1189%26epage%3D1202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Jun  Fu</span>, <span class="hlFld-ContribAuthor ">Jun  Li</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Annual review of LSD1/KDM1A inhibitors in 2020. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113254. <a href="https://doi.org/10.1016/j.ejmech.2021.113254" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113254</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113254%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAnnual%252Breview%252Bof%252BLSD1%25252FKDM1A%252Binhibitors%252Bin%252B2020%26aulast%3DFu%26aufirst%3DDong-Jun%26date%3D2021%26volume%3D214%26spage%3D113254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleni  Pitsillou</span>, <span class="hlFld-ContribAuthor ">Julia  Liang</span>, <span class="hlFld-ContribAuthor ">Andrew  Hung</span>, <span class="hlFld-ContribAuthor ">Tom C.  Karagiannis</span>. </span><span class="cited-content_cbyCitation_article-title">The circadian machinery links metabolic disorders and depression: A review of pathways, proteins and potential pharmacological interventions. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2021,</strong> <em>265 </em>, 118809. <a href="https://doi.org/10.1016/j.lfs.2020.118809" title="DOI URL">https://doi.org/10.1016/j.lfs.2020.118809</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2020.118809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2020.118809%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DThe%252Bcircadian%252Bmachinery%252Blinks%252Bmetabolic%252Bdisorders%252Band%252Bdepression%25253A%252BA%252Breview%252Bof%252Bpathways%25252C%252Bproteins%252Band%252Bpotential%252Bpharmacological%252Binterventions%26aulast%3DPitsillou%26aufirst%3DEleni%26date%3D2021%26volume%3D265%26spage%3D118809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samir  Mehndiratta</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine specific demethylase 1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 969-981. <a href="https://doi.org/10.1039/D0MD00141D" title="DOI URL">https://doi.org/10.1039/D0MD00141D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00141D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00141D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DHistone%252Blysine%252Bspecific%252Bdemethylase%252B1%252Binhibitors%26aulast%3DMehndiratta%26aufirst%3DSamir%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D9%26spage%3D969%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Z. F.  Nuriakhmetova</span>, <span class="hlFld-ContribAuthor ">F. A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">V. A.  Egorov</span>, <span class="hlFld-ContribAuthor ">M. S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">4H-Thieno[3,2-b]pyrrole-5-carbohydrazides and Their Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>56 </em>
                                    (9)
                                     , 1545-1549. <a href="https://doi.org/10.1134/S1070428020090079" title="DOI URL">https://doi.org/10.1134/S1070428020090079</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428020090079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428020090079%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3D4H-Thieno%25255B3%25252C2-b%25255Dpyrrole-5-carbohydrazides%252Band%252BTheir%252BDerivatives%26aulast%3DTorosyan%26aufirst%3DS.%2BA.%26date%3D2020%26date%3D2020%26volume%3D56%26issue%3D9%26spage%3D1545%26epage%3D1549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarder  Arifuzzaman</span>, <span class="hlFld-ContribAuthor ">Mst Reshma  Khatun</span>, <span class="hlFld-ContribAuthor ">Rabeya  Khatun</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging of lysine demethylases (KDMs): From pathophysiological insights to novel therapeutic opportunities. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2020,</strong> <em>129 </em>, 110392. <a href="https://doi.org/10.1016/j.biopha.2020.110392" title="DOI URL">https://doi.org/10.1016/j.biopha.2020.110392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2020.110392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2020.110392%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DEmerging%252Bof%252Blysine%252Bdemethylases%252B%252528KDMs%252529%25253A%252BFrom%252Bpathophysiological%252Binsights%252Bto%252Bnovel%252Btherapeutic%252Bopportunities%26aulast%3DArifuzzaman%26aufirst%3DSarder%26date%3D2020%26volume%3D129%26spage%3D110392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyun  Liang</span>, <span class="hlFld-ContribAuthor ">Haiwen  Wang</span>, <span class="hlFld-ContribAuthor ">Yongliang  Du</span>, <span class="hlFld-ContribAuthor ">Bingling  Luo</span>, <span class="hlFld-ContribAuthor ">Ning  Meng</span>, <span class="hlFld-ContribAuthor ">Meifeng  Cen</span>, <span class="hlFld-ContribAuthor ">Peng  Huang</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>, <span class="hlFld-ContribAuthor ">Shijun  Wen</span>. </span><span class="cited-content_cbyCitation_article-title">New tranylcypromine derivatives containing sulfonamide motif as potent LSD1 inhibitors to target acute myeloid leukemia: Design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103808. <a href="https://doi.org/10.1016/j.bioorg.2020.103808" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103808</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103808%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNew%252Btranylcypromine%252Bderivatives%252Bcontaining%252Bsulfonamide%252Bmotif%252Bas%252Bpotent%252BLSD1%252Binhibitors%252Bto%252Btarget%252Bacute%252Bmyeloid%252Bleukemia%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DLiang%26aufirst%3DLiyun%26date%3D2020%26volume%3D99%26spage%3D103808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleni  Pitsillou</span>, <span class="hlFld-ContribAuthor ">Julia  Liang</span>, <span class="hlFld-ContribAuthor ">Andrew  Hung</span>, <span class="hlFld-ContribAuthor ">Tom C.  Karagiannis</span>. </span><span class="cited-content_cbyCitation_article-title">Chromatin modification by olive phenolics: In silico molecular docking studies utilising the phenolic groups categorised in the OliveNet™ database against lysine specific demethylase enzymes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2020,</strong> <em>97 </em>, 107575. <a href="https://doi.org/10.1016/j.jmgm.2020.107575" title="DOI URL">https://doi.org/10.1016/j.jmgm.2020.107575</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2020.107575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2020.107575%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DChromatin%252Bmodification%252Bby%252Bolive%252Bphenolics%25253A%252BIn%252Bsilico%252Bmolecular%252Bdocking%252Bstudies%252Butilising%252Bthe%252Bphenolic%252Bgroups%252Bcategorised%252Bin%252Bthe%252BOliveNet%2525E2%252584%2525A2%252Bdatabase%252Bagainst%252Blysine%252Bspecific%252Bdemethylase%252Benzymes%26aulast%3DPitsillou%26aufirst%3DEleni%26date%3D2020%26volume%3D97%26spage%3D107575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Cai  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Jiming  Wang</span>, <span class="hlFld-ContribAuthor ">Qinwen  Jiang</span>, <span class="hlFld-ContribAuthor ">Liyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>194 </em>, 112243. <a href="https://doi.org/10.1016/j.ejmech.2020.112243" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112243</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112243%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Btetrahydroquinoline-based%252Breversible%252BLSD1%252Binhibitors%26aulast%3DWang%26aufirst%3DXinran%26date%3D2020%26volume%3D194%26spage%3D112243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongtao  Xu</span>, <span class="hlFld-ContribAuthor ">Zihao  He</span>, <span class="hlFld-ContribAuthor ">Hongyi  Liu</span>, <span class="hlFld-ContribAuthor ">Yifan  Chen</span>, <span class="hlFld-ContribAuthor ">Yunlong  Gao</span>, <span class="hlFld-ContribAuthor ">Songjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Meiting  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Lu</span>, <span class="hlFld-ContribAuthor ">Chang  Wang</span>, <span class="hlFld-ContribAuthor ">Zongya  Zhao</span>, <span class="hlFld-ContribAuthor ">Yan  Liu</span>, <span class="hlFld-ContribAuthor ">Junqiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yi  Yu</span>, <span class="hlFld-ContribAuthor ">Min  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR, molecular docking, and molecular dynamics simulation study of thieno[3,2-
              b
              ]pyrrole-5-carboxamide derivatives as LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 6927-6943. <a href="https://doi.org/10.1039/C9RA10085G" title="DOI URL">https://doi.org/10.1039/C9RA10085G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA10085G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA10085G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3D3D-QSAR%25252C%252Bmolecular%252Bdocking%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulation%252Bstudy%252Bof%252Bthieno%25255B3%25252C2-%252Bb%252B%25255Dpyrrole-5-carboxamide%252Bderivatives%252Bas%252BLSD1%252Binhibitors%26aulast%3DXu%26aufirst%3DYongtao%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D6927%26epage%3D6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony G.  Coyne</span>. </span><span class="cited-content_cbyCitation_article-title">Bicyclic 5-5 Systems: Three Heteroatoms 1:2. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.1016/B978-0-12-409547-2.14914-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.14914-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.14914-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.14914-5%26sid%3Dliteratum%253Aachs%26atitle%3DBicyclic%252B5-5%252BSystems%25253A%252BThree%252BHeteroatoms%252B1%25253A2%26aulast%3DCoyne%26aufirst%3DAnthony%2BG.%26date%3D2020%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BChemistry%25252C%252BMolecular%252BSciences%252Band%252BChemical%252BEngineering%26date%3D2020%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Soraya  Alnabulsi</span>, <span class="hlFld-ContribAuthor ">Enas A.  Al-Hurani</span>, <span class="hlFld-ContribAuthor ">Nizar A.  Al-shar'i</span>, <span class="hlFld-ContribAuthor ">Tamam  El-Elimat</span>. </span><span class="cited-content_cbyCitation_article-title">Amino-carboxamide benzothiazoles as potential LSD1 hit inhibitors. Part I: Computational fragment-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2019,</strong> <em>93 </em>, 107440. <a href="https://doi.org/10.1016/j.jmgm.2019.107440" title="DOI URL">https://doi.org/10.1016/j.jmgm.2019.107440</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2019.107440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2019.107440%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DAmino-carboxamide%252Bbenzothiazoles%252Bas%252Bpotential%252BLSD1%252Bhit%252Binhibitors.%252BPart%252BI%25253A%252BComputational%252Bfragment-based%252Bdrug%252Bdesign%26aulast%3DAlnabulsi%26aufirst%3DSoraya%26date%3D2019%26volume%3D93%26spage%3D107440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Z. F.  Nuriakhmetova</span>, <span class="hlFld-ContribAuthor ">F. A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">M. S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">4H-Thieno[3,2-b]pyrrole-5-carboxylate Conjugates with Taurine and Its Tetrabutylammonium Salt. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>55 </em>
                                    (12)
                                     , 1902-1906. <a href="https://doi.org/10.1134/S1070428019120157" title="DOI URL">https://doi.org/10.1134/S1070428019120157</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428019120157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428019120157%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3D4H-Thieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxylate%252BConjugates%252Bwith%252BTaurine%252Band%252BIts%252BTetrabutylammonium%252BSalt%26aulast%3DTorosyan%26aufirst%3DS.%2BA.%26date%3D2019%26date%3D2020%26volume%3D55%26issue%3D12%26spage%3D1902%26epage%3D1906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Z. F.  Nuriakhmetova</span>, <span class="hlFld-ContribAuthor ">F. A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">M. S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">New 4-Substituted 5-(1H-Pyrrol-2-ylmethyl)-4H-thieno[3,2-b]pyrroles and Their Reactions with N-Bromosuccinimide. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>55 </em>
                                    (12)
                                     , 1907-1911. <a href="https://doi.org/10.1134/S1070428019120169" title="DOI URL">https://doi.org/10.1134/S1070428019120169</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428019120169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428019120169%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DNew%252B4-Substituted%252B5-%2525281H-Pyrrol-2-ylmethyl%252529-4H-thieno%25255B3%25252C2-b%25255Dpyrroles%252Band%252BTheir%252BReactions%252Bwith%252BN-Bromosuccinimide%26aulast%3DTorosyan%26aufirst%3DS.%2BA.%26date%3D2019%26date%3D2020%26volume%3D55%26issue%3D12%26spage%3D1907%26epage%3D1911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Fang</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">LSD1/KDM1A inhibitors in clinical trials: advances and prospects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2019,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-019-0811-9" title="DOI URL">https://doi.org/10.1186/s13045-019-0811-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DLSD1%25252FKDM1A%252Binhibitors%252Bin%252Bclinical%252Btrials%25253A%252Badvances%252Band%252Bprospects%26aulast%3DFang%26aufirst%3DYuan%26date%3D2019%26date%3D2019%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alina N.  Grozav</span>, <span class="hlFld-ContribAuthor ">Mariana Z.  Fedoriv</span>, <span class="hlFld-ContribAuthor ">Vitaliy A.  Chornous</span>, <span class="hlFld-ContribAuthor ">Alina A.  Palamar</span>, <span class="hlFld-ContribAuthor ">Mykhailo K.  Bratenko</span>, <span class="hlFld-ContribAuthor ">Mykhailo V.  Vovk</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of thieno[2,3-b]pyrrole-2(4)-carboxylic and 2,4-dicarboxylic acids. </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2019,</strong> <em>55 </em>
                                    (4-5)
                                     , 435-441. <a href="https://doi.org/10.1007/s10593-019-02476-8" title="DOI URL">https://doi.org/10.1007/s10593-019-02476-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-019-02476-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-019-02476-8%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DSynthesis%252Bof%252Bthieno%25255B2%25252C3-b%25255Dpyrrole-2%2525284%252529-carboxylic%252Band%252B2%25252C4-dicarboxylic%252Bacids%26aulast%3DGrozav%26aufirst%3DAlina%2BN.%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D4-5%26spage%3D435%26epage%3D441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Zhong-Rui  Li</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 388-401. <a href="https://doi.org/10.1016/j.ejmech.2019.02.039" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.039%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bstructure-activity%252Brelationship%252Bstudies%252Band%252Bbiological%252Bcharacterization%252Bof%252Bnew%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine-based%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DWang%26aufirst%3DShuai%26date%3D2019%26volume%3D167%26spage%3D388%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">LSD
              (Lysine‐Specific Demethylase): A Decade‐Long Trip from Discovery to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-261. <a href="https://doi.org/10.1002/9783527809257.ch10" title="DOI URL">https://doi.org/10.1002/9783527809257.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DLSD%252B%252528Lysine%2525E2%252580%252590Specific%252BDemethylase%252529%25253A%252BA%252BDecade%2525E2%252580%252590Long%252BTrip%252Bfrom%252BDiscovery%252Bto%252BClinical%252BTrials%26aulast%3DLee%26aufirst%3DAdam%26date%3D2019%26date%3D2019%26spage%3D221%26epage%3D261%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Tino  Heimburg</span>, <span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Computer‐based Lead Identification for Epigenetic Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 45-77. <a href="https://doi.org/10.1002/9783527809257.ch3" title="DOI URL">https://doi.org/10.1002/9783527809257.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DComputer%2525E2%252580%252590based%252BLead%252BIdentification%252Bfor%252BEpigenetic%252BTargets%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2019%26date%3D2019%26spage%3D45%26epage%3D77%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine-Specific Histone Demethylases 1/2 (LSD1/2) and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 197-219. <a href="https://doi.org/10.1007/7355_2019_74" title="DOI URL">https://doi.org/10.1007/7355_2019_74</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_74%26sid%3Dliteratum%253Aachs%26atitle%3DLysine-Specific%252BHistone%252BDemethylases%252B1%25252F2%252B%252528LSD1%25252F2%252529%252Band%252BTheir%252BInhibitors%26aulast%3DSuzuki%26aufirst%3DTakayoshi%26date%3D2019%26date%3D2020%26spage%3D197%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacoepigenetics of LSD1 Inhibitors in Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 523-530. <a href="https://doi.org/10.1016/B978-0-12-813939-4.00014-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-813939-4.00014-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-813939-4.00014-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-813939-4.00014-0%26sid%3Dliteratum%253Aachs%26atitle%3DPharmacoepigenetics%252Bof%252BLSD1%252BInhibitors%252Bin%252BCancer%26aulast%3DYu%26aufirst%3DBin%26date%3D2019%26spage%3D523%26epage%3D530%26pub%3DElsevier%26atitle%3DPharmacoepigenetics%26date%3D2019%26volume%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingchao  Duan</span>, <span class="hlFld-ContribAuthor ">Wenping  Qin</span>, <span class="hlFld-ContribAuthor ">Fengzhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhai</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Guan</span>, <span class="hlFld-ContribAuthor ">Xiaojuan  Wang</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (23-24)
                                     , 6000-6014. <a href="https://doi.org/10.1016/j.bmc.2018.10.037" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.10.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.10.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.10.037%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bin%252Bvitro%252Bevaluation%252Bof%252Bstilbene%252Bderivatives%252Bas%252Bnovel%252BLSD1%252Binhibitors%252Bfor%252BAML%252Btherapy%26aulast%3DDuan%26aufirst%3DYingchao%26date%3D2018%26volume%3D26%26issue%3D23-24%26spage%3D6000%26epage%3D6014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Chen  Zhou</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Qihua  Zhu</span>, <span class="hlFld-ContribAuthor ">Yungen  Xu</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of 5-arylidene barbiturates as potent, selective, reversible LSD1 inhibitors for the treatment of acute promyelocytic leukemia. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (17)
                                     , 4871-4880. <a href="https://doi.org/10.1016/j.bmc.2018.08.026" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.08.026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.08.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.08.026%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252B5-arylidene%252Bbarbiturates%252Bas%252Bpotent%25252C%252Bselective%25252C%252Breversible%252BLSD1%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Bacute%252Bpromyelocytic%252Bleukemia%26aulast%3DXu%26aufirst%3DSiyuan%26date%3D2018%26volume%3D26%26issue%3D17%26spage%3D4871%26epage%3D4880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seda A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Zoya F.  Nuriakhmetova</span>, <span class="hlFld-ContribAuthor ">Vadim V.  Zagitov</span>, <span class="hlFld-ContribAuthor ">Fanusa A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">Mansur S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">New 4H-thieno[3,2-b]pyrrole-5-carboxamides. </span><span class="cited-content_cbyCitation_journal-name">Chemistry of Heterocyclic Compounds</span><span> <strong>2018,</strong> <em>54 </em>
                                    (8)
                                     , 819-822. <a href="https://doi.org/10.1007/s10593-018-2355-2" title="DOI URL">https://doi.org/10.1007/s10593-018-2355-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10593-018-2355-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10593-018-2355-2%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520of%2520Heterocyclic%2520Compounds%26atitle%3DNew%252B4H-thieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxamides%26aulast%3DTorosyan%26aufirst%3DSeda%2BA.%26date%3D2018%26date%3D2018%26volume%3D54%26issue%3D8%26spage%3D819%26epage%3D822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Lulu  Zhang</span>, <span class="hlFld-ContribAuthor ">Xueyuan  Bi</span>, <span class="hlFld-ContribAuthor ">Yanshen  Ren</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Yueqing  Gu</span>, <span class="hlFld-ContribAuthor ">Yungen  Xu</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2018,</strong> <em>78 </em>, 7-16. <a href="https://doi.org/10.1016/j.bioorg.2018.02.016" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.02.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.02.016%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bactivity%252Bof%252B4-%2525284-benzyloxy%252529phenoxypiperidines%252Bas%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%26aulast%3DXi%26aufirst%3DJiayue%26date%3D2018%26volume%3D78%26spage%3D7%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Shin  Sato</span>, <span class="hlFld-ContribAuthor ">Tomoko  Hashimoto</span>, <span class="hlFld-ContribAuthor ">Kenji  Matsuno</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structure of LSD1 in Complex with 4-[5-(Piperidin-4-ylmethoxy)-2-(p-tolyl)pyridin-3-yl]benzonitrile. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (7)
                                     , 1538. <a href="https://doi.org/10.3390/molecules23071538" title="DOI URL">https://doi.org/10.3390/molecules23071538</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23071538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23071538%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DCrystal%252BStructure%252Bof%252BLSD1%252Bin%252BComplex%252Bwith%252B4-%25255B5-%252528Piperidin-4-ylmethoxy%252529-2-%252528p-tolyl%252529pyridin-3-yl%25255Dbenzonitrile%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D7%26spage%3D1538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">F. A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">V. V.  Zagitov</span>, <span class="hlFld-ContribAuthor ">A. S.  Erastov</span>, <span class="hlFld-ContribAuthor ">M. S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of N-Substituted Methyl 4H-Thieno[3,2-b]pyrrole-5-carboxylates. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2018,</strong> <em>54 </em>
                                    (6)
                                     , 912-917. <a href="https://doi.org/10.1134/S1070428018060131" title="DOI URL">https://doi.org/10.1134/S1070428018060131</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428018060131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428018060131%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Bof%252BN-Substituted%252BMethyl%252B4H-Thieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxylates%26aulast%3DTorosyan%26aufirst%3DS.%2BA.%26date%3D2018%26date%3D2018%26volume%3D54%26issue%3D6%26spage%3D912%26epage%3D917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seda A.  Torosyan</span>, <span class="hlFld-ContribAuthor ">Vadim V.  Zagitov</span>, <span class="hlFld-ContribAuthor ">Fanuza A.  Gimalova</span>, <span class="hlFld-ContribAuthor ">Raisa Z.  Biglova</span>, <span class="hlFld-ContribAuthor ">Mansur S.  Miftakhov</span>. </span><span class="cited-content_cbyCitation_article-title">Self-condensation of N-substituted (4 H -thieno[3,2- b ]-pyrrol-5-yl)methanols into bis(thienopyrrolyl)methanes. </span><span class="cited-content_cbyCitation_journal-name">Mendeleev Communications</span><span> <strong>2018,</strong> <em>28 </em>
                                    (2)
                                     , 192-194. <a href="https://doi.org/10.1016/j.mencom.2018.03.028" title="DOI URL">https://doi.org/10.1016/j.mencom.2018.03.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mencom.2018.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mencom.2018.03.028%26sid%3Dliteratum%253Aachs%26jtitle%3DMendeleev%2520Communications%26atitle%3DSelf-condensation%252Bof%252BN-substituted%252B%2525284%252BH%252B-thieno%25255B3%25252C2-%252Bb%252B%25255D-pyrrol-5-yl%252529methanols%252Binto%252Bbis%252528thienopyrrolyl%252529methanes%26aulast%3DTorosyan%26aufirst%3DSeda%2BA.%26date%3D2018%26volume%3D28%26issue%3D2%26spage%3D192%26epage%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Yong-Cheng  Ma</span>, <span class="hlFld-ContribAuthor ">Wen-Ping  Qin</span>, <span class="hlFld-ContribAuthor ">Li-Na  Ding</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Ying-Li  Zhu</span>, <span class="hlFld-ContribAuthor ">Xiao-Yu  Zhai</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of tranylcypromine derivatives as novel LSD1/HDACs dual inhibitors for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>140 </em>, 392-402. <a href="https://doi.org/10.1016/j.ejmech.2017.09.038" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.09.038</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.09.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.09.038%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Btranylcypromine%252Bderivatives%252Bas%252Bnovel%252BLSD1%25252FHDACs%252Bdual%252Binhibitors%252Bfor%252Bcancer%252Btreatment%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2017%26volume%3D140%26spage%3D392%26epage%3D402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoli  Fu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Advances toward LSD1 inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (11)
                                     , 1227-1242. <a href="https://doi.org/10.4155/fmc-2017-0068" title="DOI URL">https://doi.org/10.4155/fmc-2017-0068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Btoward%252BLSD1%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DFu%26aufirst%3DXiaoli%26date%3D2017%26volume%3D9%26issue%3D11%26spage%3D1227%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KDM1A inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. HTS workflow.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Compounds <b>19</b> and <b>21</b> reversibly inhibited KDM1A/CoREST enzymatic activity. KDM1A/CoREST complex was incubated at a concentration 100-fold over the concentration required for the enzymatic activity (2000 nM), with concentrations of the inhibitors <b>19</b>, <b>21</b>, and <b>1</b> equivalent to 10-fold of their IC<sub>50</sub>s. After 15 min, the above solution is diluted 100-fold into reaction buffer containing the substrate to initiate the reaction. <b>19</b> and <b>21</b>, contrarily to the irreversible inhibitor <b>1</b>, result in recovery of enzymatic activity. (B) Plot of 1/(Substrate)/1/Rate(μmol/min) for a range of concentrations of <b>19</b>. (C) Plot of 1/(Substrate)/1/Rate(μmol/min)for a range of concentration of <b>21</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray of <b>19</b>. (A) Inhibitor interactions with target. (B) Comparison of crystallographic structure of <b>19</b> with KDM1A in complex with histone H3 peptide (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>): the superposition of the two structures shows the partial overlap of the thienopyrrole molecule with the atoms of the peptide. Trp699, in cyan, has shifted with respect to its reported position.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Favorable interaction regions. Protein surface is displayed as well as the surfaces of the favorable interaction areas for hydrophobic character, in light-yellow, and for the electronegative ones, in red. (A) the inhibitor and cofactor molecules are in sticks representation and their carbon atoms are color-coded cyan (inhibitor) and magenta (cofactor). In light-green, two vectors are depicted from the position where the substituent emerges to the two electronegative regions. (B) The peptide substrate is depicted, and some residues are visible in sticks with carbons in violet.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Four areas modified during explorative SAR.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0008.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthesis of 5–5 Derivatives<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) EtOK, EtOH, 4 h, 0 °C to rt; (b) xylene, reflux, 30 min; (c) (1) NaH, DMF, rt, 30 min to 1 h, (2) MeI, rt; (d) PhNH<sub>2</sub> (up to 1.6 equiv), LiHMDS (up to 2.6 equiv), THF, 0 °C to rt, 2 h; (e) LiOH, H<sub>2</sub>O/EtOH, rt 3 h; (f) (1) SOCl<sub>2</sub>, DCM/THF, 2 h reflux, (2) PhNH<sub>2</sub>, pyridine, rt, overnight.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0009.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Halogenated Thienopyrroles<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NCS (1.05 equiv), DCM, 0 °C to rt overnight; (b) NBS (1.05 equiv), DCM, 0 °C to rt overnight; (c) NIS (1.05 equiv), DCM 0 °C to rt overnight, 18% <b>58</b>, 16% <b>59</b>, 2% <b>60</b>; (d) LDA (3 equiv), THF, −78 °C, 1 h; NFSI (3 equiv), −78 °C to rt 1 h.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0010.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 3. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions. (a) (i) NaH, DMF, rt, 30 min to 1 h, then R1I, rt, or (ii) phenylboronic acid Cu(OAc)<sub>2</sub>, TEA, PYR; (b) LiOH, H<sub>2</sub>O/EtOH, rt 3 h; (c) (1) SOCl<sub>2</sub> (1.3 equiv), THF, 67 °C 2 h, (2) Ar-NH<sub>2</sub> (<b>102a</b>–<b>z</b>, 0.6 equiv), pyridine, rt 90 min; (d) PhNH<sub>2</sub> (up to 1.6 equiv), LiHMDS (up to 2.6 equiv), THF, 0 °C to rt, 2 h.</p></p></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/medium/jm-2016-01018d_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0007.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. One of the accessible orientations for compound <b>90</b> in extended conformation with interactions with Asp375.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-5/acs.jmedchem.6b01018/20170303/images/large/jm-2016-01018d_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01018&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i172">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81510" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81510" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 55 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+Lysine+Methylation+Dynamics%3A+Establishment%2C+Regulation%2C+and+Biological+Impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0liPPXWD4psAoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520Lysine%2520Methylation%2520Dynamics%253A%2520Establishment%252C%2520Regulation%252C%2520and%2520Biological%2520Impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Methylation: A Dynamic Mark in Health, Disease and Inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+Methylation%3A+A+Dynamic+Mark+in+Health%2C+Disease+and+Inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0liPPXWD4psAoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Methylation%253A%2520A%2520Dynamic%2520Mark%2520in%2520Health%252C%2520Disease%2520and%2520Inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabari, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span> </span><span class="NLM_article-title">SnapShot: Histone Modifications</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">159</span><span class="NLM_x">, </span> <span class="NLM_fpage">458</span><span class="refDoi"> DOI: 10.1016/j.cell.2014.09.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.cell.2014.09.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25303536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=458&author=H.+Huangauthor=B.+R.+Sabariauthor=B.+A.+Garciaauthor=C.+D.+Allisauthor=Y.+Zhao&title=SnapShot%3A+Histone+Modifications&doi=10.1016%2Fj.cell.2014.09.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">SnapShot: Histone modifications</span></div><div class="casAuthors">Huang, He; Sabari, Benjamin R.; Garcia, Benjamin A.; Allis, C. David; Zhao, Yingming</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">458-458.e1</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Histones are decorated by a variety of protein post-translational modifications called histone marks that modulate chromatin structure and function, contributing to the cellular gene expression program.  This SnapShot summarizes the reported human, mouse, and rat histone marks, including recently identified Lys residue acylation marks.  It has been proposed that histone acylations provide a link between cellular metab. and epigenetic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQmsTna8jsxrVg90H21EOLACvtfcHk0lhxDiGnQdqpBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslGjsbjO&md5=969e02ba1cbaa8f87bd5aae32857720f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.09.037%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DSabari%26aufirst%3DB.%2BR.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DSnapShot%253A%2520Histone%2520Modifications%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D458%26doi%3D10.1016%2Fj.cell.2014.09.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Demethylation Mediated by the Nuclear Amine Oxidase Homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+Demethylation+Mediated+by+the+Nuclear+Amine+Oxidase+Homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lhxDiGnQdqpBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Demethylation%2520Mediated%2520by%2520the%2520Nuclear%2520Amine%2520Oxidase%2520Homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">LSD1 Inhibition: A Therapeutic Strategy in Cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+P.+Somervaille&title=LSD1+Inhibition%3A+A+Therapeutic+Strategy+in+Cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0lhxDiGnQdqpBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DLSD1%2520Inhibition%253A%2520A%2520Therapeutic%2520Strategy%2520in%2520Cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Niebel, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holler, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majores, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutgemann, I.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 (LSD1) in Hematopoietic and Lymphoid Neoplasms</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">124</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">152</span><span class="refDoi"> DOI: 10.1182/blood-2014-04-569525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1182%2Fblood-2014-04-569525" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=151-152&author=D.+Niebelauthor=J.+Kirfelauthor=V.+Janzenauthor=T.+Hollerauthor=M.+Majoresauthor=I.+Gutgemann&title=Lysine-Specific+Demethylase+1+%28LSD1%29+in+Hematopoietic+and+Lymphoid+Neoplasms&doi=10.1182%2Fblood-2014-04-569525"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-04-569525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-04-569525%26sid%3Dliteratum%253Aachs%26aulast%3DNiebel%26aufirst%3DD.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DJanzen%26aufirst%3DV.%26aulast%3DHoller%26aufirst%3DT.%26aulast%3DMajores%26aufirst%3DM.%26aulast%3DGutgemann%26aufirst%3DI.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520%2528LSD1%2529%2520in%2520Hematopoietic%2520and%2520Lymphoid%2520Neoplasms%26jtitle%3DBlood%26date%3D2014%26volume%3D124%26spage%3D151%26epage%3D152%26doi%3D10.1182%2Fblood-2014-04-569525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H. F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 Demethylates Repressive Histone Marks to Promote Androgen-Receptor-Dependent Transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+Demethylates+Repressive+Histone+Marks+to+Promote+Androgen-Receptor-Dependent+Transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0ljB29PeofubQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520Demethylates%2520Repressive%2520Histone%2520Marks%2520to%2520Promote%2520Androgen-Receptor-Dependent%2520Transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 (LSD1) Is Highly Expressed in ER-Negative Breast Cancers and a Biomarker Predicting Aggressive Biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Sch%C3%BCleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+%28LSD1%29+Is+Highly+Expressed+in+ER-Negative+Breast+Cancers+and+a+Biomarker+Predicting+Aggressive+Biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0ljB29PeofubQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520%2528LSD1%2529%2520Is%2520Highly%2520Expressed%2520in%2520ER-Negative%2520Breast%2520Cancers%2520and%2520a%2520Biomarker%2520Predicting%2520Aggressive%2520Biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 Is Strongly Expressed in Poorly Differentiated Neuroblastoma: Implications for Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Schuleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+Is+Strongly+Expressed+in+Poorly+Differentiated+Neuroblastoma%3A+Implications+for+Therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0ljB29PeofubQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520Is%2520Strongly%2520Expressed%2520in%2520Poorly%2520Differentiated%2520Neuroblastoma%253A%2520Implications%2520for%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen Receptor Coactivators Lysine-Specific Histone Demethylase 1 and Four and a Half LIM Domain Protein 2 Predict Risk of Prostate Cancer Recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Androgen+Receptor+Coactivators+Lysine-Specific+Histone+Demethylase+1+and+Four+and+a+Half+LIM+Domain+Protein+2+Predict+Risk+of+Prostate+Cancer+Recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520Receptor%2520Coactivators%2520Lysine-Specific%2520Histone%2520Demethylase%25201%2520and%2520Four%2520and%2520a%2520Half%2520LIM%2520Domain%2520Protein%25202%2520Predict%2520Risk%2520of%2520Prostate%2520Cancer%2520Recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Lv, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Over-Expression of LSD1 Promotes Proliferation, Migration and Invasion in Non-Small Cell Lung Cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e35065</span><span class="refDoi"> DOI: 10.1371/journal.pone.0035065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1371%2Fjournal.pone.0035065" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e35065&author=T.+Lvauthor=D.+Yuanauthor=X.+Miaoauthor=Y.+Lvauthor=P.+Zhanauthor=X.+Shenauthor=Y.+Song&title=Over-Expression+of+LSD1+Promotes+Proliferation%2C+Migration+and+Invasion+in+Non-Small+Cell+Lung+Cancer&doi=10.1371%2Fjournal.pone.0035065"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0035065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0035065%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DT.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DMiao%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DY.%26aulast%3DZhan%26aufirst%3DP.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DOver-Expression%2520of%2520LSD1%2520Promotes%2520Proliferation%252C%2520Migration%2520and%2520Invasion%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De35065%26doi%3D10.1371%2Fjournal.pone.0035065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Zhao, Z. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span> </span><span class="NLM_article-title">Bin. Overexpression of Lysine Specific Demethylase 1 Predicts Worse Prognosis in Primary Hepatocellular Carcinoma Patients</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6651</span><span class="NLM_x">–</span> <span class="NLM_lpage">6656</span><span class="refDoi"> DOI: 10.3748/wjg.v18.i45.6651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.3748%2Fwjg.v18.i45.6651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=23236241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6651-6656&author=Z.+K.+Zhaoauthor=H.+F.+Yuauthor=D.+R.+Wangauthor=P.+Dongauthor=L.+Chenauthor=W.+G.+Wuauthor=W.+J.+Dingauthor=Y.+Liu&title=Bin.+Overexpression+of+Lysine+Specific+Demethylase+1+Predicts+Worse+Prognosis+in+Primary+Hepatocellular+Carcinoma+Patients&doi=10.3748%2Fwjg.v18.i45.6651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span></div><div class="casAuthors">Zhao, Ze-Kun; Yu, Hai-Feng; Wang, Dao-Rong; Dong, Ping; Chen, Lei; Wu, Wen-Guang; Ding, Wen-Jun; Liu, Ying-Bin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">6651-6656</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Aim: To investigate the clinicopathol. features and prognostic value of lysine specific demethylase 1 (LSD1) in hepatocellular carcinoma (HCC).  Methods: We examd. LSD1 expression in 60 paired liver cancer tissues and adjacent noncancerous tissues by quant. real time polymerase chain reaction (qRT-PCR) and Western blotting.  In addn., we analyzed LSD1 expression in 198 HCC samples by immunohistochem.  The relationship between LSD1 expression, clinicopathol. features and patient survival was investigated.  Results: Immunohistochem., Western blotting, and qRT-PCR consistently confirmed LSD1 overexpression in HCC tissues compared to adjacent non-neoplastic tissues (P < 0.01).  Addnl., immunostaining showed more LSD1-pos. cells in the higher tumor stage (T3-4) and tumor grade (G3) than in the lower tumor stage (T1-2, P < 0.001) and tumor grade (G1-2, P < 0.001), resp.  Moreover, HCC patients with high LSD1 expression had significantly lower 5-yr overall survival rates (P < 0.001) and lower 5-yr disease-free survival rates (P < 0.001), resp.  A Cox proportional hazards model further demonstrated that LSD1 over-expression was an independent predictor of poor prognosis for both 5-yr disease-free survival [hazards ratio (HR) = 1.426, 95%CI: 0.672-2.146, P < 0.001] and 5-yr overall survival (HR = 2.456, 95%CI: 1.234-3.932, P < 0.001) in HCC.  Conclusion: Our data suggest for the first time that the overexpression of LSD1 protein in HCC tissues indicates tumor progression and predicts poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqX4-UuO3NnLVg90H21EOLACvtfcHk0liz2VssuZpsZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ&md5=c657f6b1e0b5e183c28fefbda4cc8745</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v18.i45.6651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v18.i45.6651%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%2BK.%26aulast%3DYu%26aufirst%3DH.%2BF.%26aulast%3DWang%26aufirst%3DD.%2BR.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DW.%2BG.%26aulast%3DDing%26aufirst%3DW.%2BJ.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DBin.%2520Overexpression%2520of%2520Lysine%2520Specific%2520Demethylase%25201%2520Predicts%2520Worse%2520Prognosis%2520in%2520Primary%2520Hepatocellular%2520Carcinoma%2520Patients%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2012%26volume%3D18%26spage%3D6651%26epage%3D6656%26doi%3D10.3748%2Fwjg.v18.i45.6651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The Deubiquitinase USP28 Stabilizes LSD1 and Confers Stem-Cell-like Traits to Breast Cancer Cells</span> <span class="citation_source-journal">Cell Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">224</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1016/j.celrep.2013.08.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.celrep.2013.08.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=24075993" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=224-236&author=Y.+Wuauthor=Y.+Wangauthor=X.+H.+Yangauthor=T.+Kangauthor=Y.+Zhaoauthor=C.+Wangauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+Deubiquitinase+USP28+Stabilizes+LSD1+and+Confers+Stem-Cell-like+Traits+to+Breast+Cancer+Cells&doi=10.1016%2Fj.celrep.2013.08.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The Deubiquitinase USP28 Stabilizes LSD1 and Confers Stem-Cell-like Traits to Breast Cancer Cells</span></div><div class="casAuthors">Wu, Yadi; Wang, Yifan; Yang, Xiuwei H.; Kang, Tiebang; Zhao, Yongxiang; Wang, Chi; Evers, B. Mark; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">224-236</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a crit. chromatin modulator that controls cellular pluripotency and differentiation through the demethylation of H3K4me1/2.  Overexpression of LSD1 has been obsd. in many types of tumors and is correlated with its oncogenic effects in tumorigenesis.  However, the mechanism leading to LSD1 upregulation in tumors remains unclear.  Using an unbiased siRNA screening against all the human deubiquitinases, we identified USP28 as a bona fide deubiquitinase of LSD1.  USP28 interacted with and stabilized LSD1 via deubiquitination.  USP28 overexpression correlated with LSD1 upregulation in multiple cancer cell lines and breast tumor samples.  Knockdown of USP28 resulted in LSD1 destabilization, leading to the suppression of cancer stem cell (CSC)-like characteristics in vitro and inhibition of tumorigenicity in vivo, which can be rescued by ectopic LSD1 expression.  Our study reveals a crit. mechanism underlying the epigenetic regulation by USP28 and provides another treatment approach against breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvDEa93VRia7Vg90H21EOLACvtfcHk0lhzO_LfZHvh_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFemsLvM&md5=802c1b0561ed2a8ffa926aa4adb82405</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2013.08.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2013.08.030%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%2BH.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520Deubiquitinase%2520USP28%2520Stabilizes%2520LSD1%2520and%2520Confers%2520Stem-Cell-like%2520Traits%2520to%2520Breast%2520Cancer%2520Cells%26jtitle%3DCell%2520Rep.%26date%3D2013%26volume%3D5%26spage%3D224%26epage%3D236%26doi%3D10.1016%2Fj.celrep.2013.08.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">563</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2006.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.chembiol.2006.05.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=16793513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=563-567&author=M.+G.+Leeauthor=C.+Wynderauthor=D.+M.+Schmidtauthor=D.+G.+McCaffertyauthor=R.+Shiekhattar&title=Histone+H3+Lysine+4+Demethylation+Is+a+Target+of+Nonselective+Antidepressive+Medications&doi=10.1016%2Fj.chembiol.2006.05.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Histone H3 Lysine 4 Demethylation Is a Target of Nonselective Antidepressive Medications</span></div><div class="casAuthors">Lee, Min Gyu; Wynder, Christopher; Schmidt, Dawn M.; McCafferty, Dewey G.; Shiekhattar, Ramin</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Demethylation of histone H3 lysine 4 is carried out by BHC110/LSD1, an enzyme with close homol. to monoamine oxidases (MAO).  Monoamine oxidase A or B are frequent targets of selective and nonselective small mol. inhibitors used for treatment of depression.  Here the authors show that in contrast to selective monoamine oxidase inhibitors such as pargyline, nonselective monoamine oxidase inhibitors potently inhibit nucleosomal demethylation of histone H3 lysine 4.  Tranylcypromine (brand name Parnate) displayed the best inhibitory activity with an IC50 of less than 2 μM.  Treatment of P19 embryonal carcinoma cells with tranylcypromine resulted in global increase in H3K4 methylation as well as transcriptional derepression of two BHC110 target genes, Egr1 and the pluripotent stem cell marker Oct4.  These results attest to the effectiveness of tranylcypromine as a small mol. inhibitor of histone demethylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatbgTiqG1Y7Vg90H21EOLACvtfcHk0lhzO_LfZHvh_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqs7k%253D&md5=4a877937287bada8b6e885638926bff2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2006.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2006.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DHistone%2520H3%2520Lysine%25204%2520Demethylation%2520Is%2520a%2520Target%2520of%2520Nonselective%2520Antidepressive%2520Medications%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D563%26epage%3D567%26doi%3D10.1016%2Fj.chembiol.2006.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Schmidt, D. M. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Trans-2-Phenylcyclopropylamine Is a Mechanism-Based Inactivator of the Histone Demethylase LSD1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">4408</span><span class="NLM_x">–</span> <span class="NLM_lpage">4416</span><span class="refDoi"> DOI: 10.1021/bi0618621</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0618621" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=4408-4416&author=D.+M.+Z.+Schmidtauthor=D.+G.+McCafferty&title=Trans-2-Phenylcyclopropylamine+Is+a+Mechanism-Based+Inactivator+of+the+Histone+Demethylase+LSD1&doi=10.1021%2Fbi0618621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">trans-2-Phenylcyclopropylamine is a mechanism-based inactivator of the histone demethylase LSD1</span></div><div class="casAuthors">Schmidt, Dawn M. Z.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4408-4416</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic domain of the flavin-dependent human histone demethylase, lysine-specific demethylase 1 (LSD1), belongs to the family of amine oxidases including polyamine oxidase and monoamine oxidase (MAO).  The authors previously assessed MAO inhibitors (MAOIs) for their ability to inhibit the reaction catalyzed by LSD1, demonstrating that trans-2-phenylcyclopropylamine (2-PCPA; tranylcypromine, Parnate) was the most potent inhibitor with respect to LSD1.  Here, the authors showed that 2-PCPA was a time-dependent, mechanism-based irreversible inhibitor of LSD1 with a Ki of 242 μM and a kinact of 0.0106 s-1.  2-PCPA showed limited selectivity for human MAOs vs. LSD1, with kinact/Ki values only 16-fold and 2.4-fold higher for MAO B and MAO A, resp.  Profiles of LSD1 activity and inactivation by 2-PCPA as a function of pH were consistent with a mechanism of inactivation dependent upon enzyme catalysis.  Mass spectrometry supported a role for FAD as the site of covalent modification by 2-PCPA.  These results will provide a foundation for the design of cyclopropylamine-based inhibitors that are selective for LSD1 to probe its role in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVK59U7IsBvbVg90H21EOLACvtfcHk0lhzO_LfZHvh_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXivF2jsr0%253D&md5=02925342b86c7dbfb5f1a99fa5264adc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fbi0618621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0618621%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%2BZ.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DTrans-2-Phenylcyclopropylamine%2520Is%2520a%2520Mechanism-Based%2520Inactivator%2520of%2520the%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DBiochemistry%26date%3D2007%26volume%3D46%26spage%3D4408%26epage%3D4416%26doi%3D10.1021%2Fbi0618621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span> </span><span class="NLM_article-title">Targeting Histone Lysine Demethylases - Progress, Challenges, and the Future</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1839</span><span class="NLM_x">, </span> <span class="NLM_fpage">1416</span><span class="NLM_x">–</span> <span class="NLM_lpage">1432</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.bbagrm.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=24859458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=1416-1432&author=C.+C.+Thinnesauthor=K.+S.+Englandauthor=A.+Kawamuraauthor=R.+Chowdhuryauthor=C.+J.+Schofieldauthor=R.+J.+Hopkinson&title=Targeting+Histone+Lysine+Demethylases+-+Progress%2C+Challenges%2C+and+the+Future&doi=10.1016%2Fj.bbagrm.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine demethylases - Progress, challenges, and the future</span></div><div class="casAuthors">Thinnes, Cyrille C.; England, Katherine S.; Kawamura, Akane; Chowdhury, Rasheduzzaman; Schofield, Christopher J.; Hopkinson, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1416-1432</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  N-Methylation of lysine and arginine residues has emerged as a major mechanism of transcriptional regulation in eukaryotes.  In humans, Nε-methyllysine residue demethylation is catalyzed by two distinct subfamilies of demethylases (KDMs), the flavin-dependent KDM1 subfamily and the 2-oxoglutarate- (2OG) dependent JmjC subfamily, which both employ oxidative mechanisms.  Modulation of histone methylation status is proposed to be important in epigenetic regulation and has substantial medicinal potential for the treatment of diseases including cancer and genetic disorders.  This article provides an introduction to the enzymol. of the KDMs and the therapeutic possibilities and challenges assocd. with targeting them, followed by a review of reported KDM inhibitors and their mechanisms of action from kinetic and structural perspectives.  This article is part of a Special Issue entitled: Methylation: A Multifaceted Modification - looking at transcription and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_ltK_pdkkCrVg90H21EOLACvtfcHk0lidmSszGBCjaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE&md5=d5d70e8873451dbedb4f36c66defc040</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520Histone%2520Lysine%2520Demethylases%2520-%2520Progress%252C%2520Challenges%252C%2520and%2520the%2520Future%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D1416%26epage%3D1432%26doi%3D10.1016%2Fj.bbagrm.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Zheng, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. M.</span><span> </span><span class="NLM_article-title">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">1032</span><span class="NLM_x">–</span> <span class="NLM_lpage">1071</span><span class="refDoi"> DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+Systematic+Review+of+Histone+Lysine-Specific+Demethylase+1+and+Its+Inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0lidmSszGBCjaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520Systematic%2520Review%2520of%2520Histone%2520Lysine-Specific%2520Demethylase%25201%2520and%2520Its%2520Inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">McGrath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span> </span><span class="NLM_article-title">Targeting Histone Lysine Methylation in Cancer</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">150</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.01.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.pharmthera.2015.01.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25578037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=1-22&author=J.+McGrathauthor=P.+Trojer&title=Targeting+Histone+Lysine+Methylation+in+Cancer&doi=10.1016%2Fj.pharmthera.2015.01.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine methylation in cancer</span></div><div class="casAuthors">McGrath, John; Trojer, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-22</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Within the vast landscape of histone modifications lysine methylation has gained increasing attention because of its profound regulatory potential.  The methylation of lysine residues on histone proteins modulates chromatin structure and thereby contributes to the regulation of DNA-based nuclear processes such as transcription, replication and repair.  Protein families with opposing catalytic activities, lysine methyltransferases (KMTs) and demethylases (KDMs), dynamically control levels of histone lysine methylation and individual enzymes within these families have become candidate oncol. targets in recent years.  A no. of high quality small mol. inhibitors of these enzymes have been identified.  Several of these compds. elicit selective cancer cell killing in vitro and robust efficacy in vivo, suggesting that targeting 'histone lysine methylation pathways' may be a relevant, emerging cancer therapeutic strategy.  Here, we discuss individual histone lysine methylation pathway targets, the properties of currently available small mol. inhibitors and their application in the context of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogqKUpTpHisrVg90H21EOLACvtfcHk0lidmSszGBCjaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KrtL4%253D&md5=225ece90384f762b248d2d7981fa120e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.01.002%26sid%3Dliteratum%253Aachs%26aulast%3DMcGrath%26aufirst%3DJ.%26aulast%3DTrojer%26aufirst%3DP.%26atitle%3DTargeting%2520Histone%2520Lysine%2520Methylation%2520in%2520Cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D150%26spage%3D1%26epage%3D22%26doi%3D10.1016%2Fj.pharmthera.2015.01.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">McAllister, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Recent Progress in Histone Demethylase Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1308</span><span class="NLM_x">–</span> <span class="NLM_lpage">1329</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1308-1329&author=T.+E.+McAllisterauthor=K.+S.+Englandauthor=R.+J.+Hopkinsonauthor=P.+E.+Brennanauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Recent+Progress+in+Histone+Demethylase+Inhibitors&doi=10.1021%2Facs.jmedchem.5b01758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Progress in Histone Demethylase Inhibitors</span></div><div class="casAuthors">McAllister, Tom E.; England, Katherine S.; Hopkinson, Richard J.; Brennan, Paul E.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1308-1329</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is increasing interest in targeting histone N-methyl-lysine demethylases (KDMs) with small mols. both for the generation of probes for target exploration and for therapeutic purposes.  Here we update on previous reviews on the inhibition of the lysine-specific demethylases (LSDs or KDM1s) and JmjC families of N-methyl-lysine demethylases (JmjC KDMs, KDM2-7), focusing on the academic and patent literature from 2014 to date.  We also highlight recent biochem., biol., and structural studies which are relevant to KDM inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWx0bnBT9cDrVg90H21EOLACvtfcHk0lhP7-4Jc5cdPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfE&md5=f744b65af58dff765cdab8f9fea85310</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01758%26sid%3Dliteratum%253Aachs%26aulast%3DMcAllister%26aufirst%3DT.%2BE.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DRecent%2520Progress%2520in%2520Histone%2520Demethylase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1308%26epage%3D1329%26doi%3D10.1021%2Facs.jmedchem.5b01758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Talele, T. T.</span><span> </span><span class="NLM_article-title">The “Cyclopropyl Fragment” is a Versatile Player That Frequently Appears in Preclinical/Clinical Drug Molecules</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">8712</span><span class="NLM_x">–</span> <span class="NLM_lpage">8756</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00472</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00472" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8712-8756&author=T.+T.+Talele&title=The+%E2%80%9CCyclopropyl+Fragment%E2%80%9D+is+a+Versatile+Player+That+Frequently+Appears+in+Preclinical%2FClinical+Drug+Molecules&doi=10.1021%2Facs.jmedchem.6b00472"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The "Cyclopropyl Fragment" is a Versatile Player that Frequently Appears in Preclinical/Clinical Drug Molecules</span></div><div class="casAuthors">Talele, Tanaji T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8712-8756</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recently, there has been an increasing use of the cyclopropyl ring in drug development to transition drug candidates from the preclin. to clin. stage.  Important features of the cyclopropane ring are, the (1) coplanarity of the three carbon atoms, (2) relatively shorter (1.51 Å) C-C bonds, (3) enhanced π-character of C-C bonds, and (4) C-H bonds are shorter and stronger than those in alkanes.  The present review will focus on the contributions that a cyclopropyl ring makes to the properties of drugs contg. it.  Consequently, the cyclopropyl ring addresses multiple roadblocks that can occur during drug discovery such as (a) enhancing potency, (b) reducing off-target effects, (c) increasing metabolic stability, (d) increasing brain permeability, (e) decreasing plasma clearance, (f) contributing to an entropically more favorable binding to the receptor, (g) conformational restriction of peptides/peptidomimetics to prevent proteolytic hydrolysis, and (h) altering drug pKa to reduce its P-glycoprotein efflux ratio.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVIJAG1XVRG7Vg90H21EOLACvtfcHk0lhP7-4Jc5cdPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpslWguro%253D&md5=b5586458ec4ab5e2d7586f5739663d1f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00472%26sid%3Dliteratum%253Aachs%26aulast%3DTalele%26aufirst%3DT.%2BT.%26atitle%3DThe%2520%25E2%2580%259CCyclopropyl%2520Fragment%25E2%2580%259D%2520is%2520a%2520Versatile%2520Player%2520That%2520Frequently%2520Appears%2520in%2520Preclinical%252FClinical%2520Drug%2520Molecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8712%26epage%3D8756%26doi%3D10.1021%2Facs.jmedchem.6b00472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Ortega Munoz, A.; Fyfe, M. C. T.; Estiarte Martinez, M. d. L. A.; Valls Vidal, N.; Kurz, G.; Castro Palomino, L. J. C.</span><span> </span><span class="NLM_article-title">(Hetero)Aryl Cyclopropylamine Compounds as LSD1 Inhibitors</span>. WO2013/057320,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+Ortega+Munoz&author=M.+C.+T.+Fyfe&author=M.+d.+L.+A.+Estiarte+Martinez&author=N.+Valls+Vidal&author=G.+Kurz&author=L.+J.+C.+Castro+Palomino&title=%28Hetero%29Aryl+Cyclopropylamine+Compounds+as+LSD1+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%2BMunoz%26aufirst%3DA.%26atitle%3D%2528Hetero%2529Aryl%2520Cyclopropylamine%2520Compounds%2520as%2520LSD1%2520Inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Mohammad, H. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smitheman, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamat, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federowicz, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneck, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carson, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butticello, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnette, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Degenhardt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasparec, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNulty, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rouse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDevitt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouthamel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpenter, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hann, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span> </span><span class="NLM_article-title">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1016/j.ccell.2015.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.ccell.2015.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=26175415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=57-69&author=H.+P.+Mohammadauthor=K.+N.+Smithemanauthor=C.+D.+Kamatauthor=D.+Soongauthor=K.+E.+Federowiczauthor=G.+S.+Van+Allerauthor=J.+L.+Schneckauthor=J.+D.+Carsonauthor=Y.+Liuauthor=M.+Butticelloauthor=W.+G.+Bonnetteauthor=S.+A.+Gormanauthor=Y.+Degenhardtauthor=Y.+Baiauthor=M.+T.+McCabeauthor=M.+B.+Pappalardiauthor=J.+Kasparecauthor=X.+Tianauthor=K.+C.+McNultyauthor=M.+Rouseauthor=P.+McDevittauthor=T.+Hoauthor=M.+Crouthamelauthor=T.+K.+Hartauthor=N.+O.+Conchaauthor=C.+F.+McHughauthor=W.+H.+Millerauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Carpenterauthor=N.+W.+Johnsonauthor=C.+L.+Hannauthor=R.+G.+Kruger&title=A+DNA+Hypomethylation+Signature+Predicts+Antitumor+Activity+of+LSD1+Inhibitors+in+SCLC&doi=10.1016%2Fj.ccell.2015.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA Hypomethylation Signature Predicts Antitumor Activity of LSD1 Inhibitors in SCLC</span></div><div class="casAuthors">Mohammad, Helai P.; Smitheman, Kimberly N.; Kamat, Chandrashekhar D.; Soong, David; Federowicz, Kelly E.; Van Aller, Glenn S.; Schneck, Jess L.; Carson, Jeffrey D.; Liu, Yan; Butticello, Michael; Bonnette, William G.; Gorman, Shelby A.; Degenhardt, Yan; Bai, Yuchen; McCabe, Michael T.; Pappalardi, Melissa B.; Kasparec, Jiri; Tian, Xinrong; McNulty, Kenneth C.; Rouse, Meagan; McDevitt, Patrick; Ho, Thau; Crouthamel, Michelle; Hart, Timothy K.; Concha, Nestor O.; McHugh, Charles F.; Miller, William H.; Dhanak, Dashyant; Tummino, Peter J.; Carpenter, Christopher L.; Johnson, Neil W.; Hann, Christine L.; Kruger, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-69</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Epigenetic dysregulation has emerged as an important mechanism in cancer.  Alterations in epigenetic machinery have become a major focus for targeted therapies.  The current report describes the discovery and biol. activity of a cyclopropylamine contg. inhibitor of Lysine Demethylase 1 (LSD1), GSK2879552.  This small mol. is a potent, selective, orally bioavailable, mechanism-based irreversible inactivator of LSD1.  A proliferation screen of cell lines representing a no. of tumor types indicated that small cell lung carcinoma (SCLC) is sensitive to LSD1 inhibition.  The subset of SCLC lines and primary samples that undergo growth inhibition in response to GSK2879552 exhibit DNA hypomethylation of a signature set of probes, suggesting this may be used as a predictive biomarker of activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppluiM41UspbVg90H21EOLACvtfcHk0lhP7-4Jc5cdPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grt7fJ&md5=863924e99fa3576dc1c67824b45462f8</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2015.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2015.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%2BP.%26aulast%3DSmitheman%26aufirst%3DK.%2BN.%26aulast%3DKamat%26aufirst%3DC.%2BD.%26aulast%3DSoong%26aufirst%3DD.%26aulast%3DFederowicz%26aufirst%3DK.%2BE.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DSchneck%26aufirst%3DJ.%2BL.%26aulast%3DCarson%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DButticello%26aufirst%3DM.%26aulast%3DBonnette%26aufirst%3DW.%2BG.%26aulast%3DGorman%26aufirst%3DS.%2BA.%26aulast%3DDegenhardt%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DPappalardi%26aufirst%3DM.%2BB.%26aulast%3DKasparec%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DMcNulty%26aufirst%3DK.%2BC.%26aulast%3DRouse%26aufirst%3DM.%26aulast%3DMcDevitt%26aufirst%3DP.%26aulast%3DHo%26aufirst%3DT.%26aulast%3DCrouthamel%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCarpenter%26aufirst%3DC.%2BL.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DHann%26aufirst%3DC.%2BL.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26atitle%3DA%2520DNA%2520Hypomethylation%2520Signature%2520Predicts%2520Antitumor%2520Activity%2520of%2520LSD1%2520Inhibitors%2520in%2520SCLC%26jtitle%3DCancer%2520Cell%26date%3D2015%26volume%3D28%26spage%3D57%26epage%3D69%26doi%3D10.1016%2Fj.ccell.2015.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span> </span><span class="NLM_article-title">Investigation of GSK2879552 in Subjects With Relapsed/Refractory Small Cell Lung Carcinoma</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&amp;rank=1" class="extLink">https://clinicaltrials.gov/ct2/show/NCT02034123?term=GSK2879552&rank=1</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Investigation+of+GSK2879552+in+Subjects+With+Relapsed%2FRefractory+Small+Cell+Lung+Carcinoma.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2013%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2FNCT02034123%3Fterm%3DGSK2879552%26rank%3D1+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DInvestigation%2520of%2520GSK2879552%2520in%2520Subjects%2520With%2520Relapsed%252FRefractory%2520Small%2520Cell%2520Lung%2520Carcinoma%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span> </span><span class="NLM_article-title">An Open-Label, Dose-Escalation/Dose-Expansion Safety Study of INCB059872 in Subjects With Advanced Malignancies</span>. ClinicalTrials.gov; <span class="NLM_publisher-name">U.S. National Institutes of Health</span>: <span class="NLM_publisher-loc">Bethesda, MD</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="https://clinicaltrials.gov/ct2/show/study/NCT02712905" class="extLink">https://clinicaltrials.gov/ct2/show/study/NCT02712905</a> (accessed October 7,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=An+Open-Label%2C+Dose-Escalation%2FDose-Expansion+Safety+Study+of+INCB059872+in+Subjects+With+Advanced+Malignancies.+ClinicalTrials.gov%3B+U.S.+National+Institutes+of+Health%3A+Bethesda%2C+MD%2C+2016%3B+https%3A%2F%2Fclinicaltrials.gov%2Fct2%2Fshow%2Fstudy%2FNCT02712905+%28accessed+October+7%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26atitle%3DAn%2520Open-Label%252C%2520Dose-Escalation%252FDose-Expansion%2520Safety%2520Study%2520of%2520INCB059872%2520in%2520Subjects%2520With%2520Advanced%2520Malignancies%26jtitle%3DClinicalTrials.gov%26pub%3DU.S.%2520National%2520Institutes%2520of%2520Health%26date%3D2016%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Vianello, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cappa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dal Zuffo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dessanti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrocco, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meroni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stazi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thaler, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trifiró, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, C.</span><span> </span><span class="NLM_article-title">Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">1501</span><span class="NLM_x">–</span> <span class="NLM_lpage">1517</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1501-1517&author=P.+Vianelloauthor=O.+A.+Botrugnoauthor=A.+Cappaauthor=R.+Dal+Zuffoauthor=P.+Dessantiauthor=A.+Maiauthor=B.+Marroccoauthor=A.+Matteviauthor=G.+Meroniauthor=S.+Minucciauthor=G.+Staziauthor=F.+Thalerauthor=P.+Trifir%C3%B3author=S.+Valenteauthor=M.+Villaauthor=M.+Varasiauthor=C.+Mercurio&title=Discovery+of+a+Novel+Inhibitor+of+Histone+Lysine-Specific+Demethylase+1A+%28KDM1A%2FLSD1%29+as+Orally+Active+Antitumor+Agent&doi=10.1021%2Facs.jmedchem.5b01209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Novel Inhibitor of Histone Lysine-Specific Demethylase 1A (KDM1A/LSD1) as Orally Active Antitumor Agent</span></div><div class="casAuthors">Vianello, Paola; Botrugno, Oronza A.; Cappa, Anna; Dal Zuffo, Roberto; Dessanti, Paola; Mai, Antonello; Marrocco, Biagina; Mattevi, Andrea; Meroni, Giuseppe; Minucci, Saverio; Stazi, Giulia; Thaler, Florian; Trifiro, Paolo; Valente, Sergio; Villa, Manuela; Varasi, Mario; Mercurio, Ciro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1501-1517</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report the stereoselective synthesis and biol. activity of a novel series of tranylcypromine (TCPA) derivs, potent inhibitors of KDM1A.  The new compds. strongly inhibit the clonogenic potential of acute leukemia cell lines.  In particular three mols. showing selectivity vs. MAO A and remarkably inhibiting colony formation in THP-1 human leukemia cells, were assessed in mouse for their preliminary pharmacokinetic.  Among them, compds. I and II were further tested in vivo in a murine acute promyelocytic leukemia model, resulting I the most effective.  Its two enantiomers were synthesized: the (1S,2R) enantiomer showed higher activity than its (1R,2S) analog, in both biochem. and cellular assays.  Compd. (1S,2R)-I exhibited in vivo efficacy after oral administration, detg. a 62% increased survival in mouse leukemia model with evidence of KDM1A inhibition.  The biol. profile of compd. (1S,2R)-I supports its further investigation as a cancer therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraSPzAcLQ0u7Vg90H21EOLACvtfcHk0lhysCZX-g2XFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVyrtbfM&md5=b7df567ef2cf939a1365874fccac9df0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01209%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DCappa%26aufirst%3DA.%26aulast%3DDal%2BZuffo%26aufirst%3DR.%26aulast%3DDessanti%26aufirst%3DP.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMarrocco%26aufirst%3DB.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMeroni%26aufirst%3DG.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DStazi%26aufirst%3DG.%26aulast%3DThaler%26aufirst%3DF.%26aulast%3DTrifir%25C3%25B3%26aufirst%3DP.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DVilla%26aufirst%3DM.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DMercurio%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Inhibitor%2520of%2520Histone%2520Lysine-Specific%2520Demethylase%25201A%2520%2528KDM1A%252FLSD1%2529%2520as%2520Orally%2520Active%2520Antitumor%2520Agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1501%26epage%3D1517%26doi%3D10.1021%2Facs.jmedchem.5b01209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+Inhibitors+of+LSD1+as+Therapeutic+Agents+in+Acute+Myeloid+Leukemia%3A+Clinical+Significance+and+Progress+to+Date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0lhysCZX-g2XFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520Inhibitors%2520of%2520LSD1%2520as%2520Therapeutic%2520Agents%2520in%2520Acute%2520Myeloid%2520Leukemia%253A%2520Clinical%2520Significance%2520and%2520Progress%2520to%2520Date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Yu, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hierl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yakowec, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwandner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emkey, R.</span><span> </span><span class="NLM_article-title">High-Throughput TR-FRET Assays for Identifying Inhibitors of LSD1 and JMJD2C Histone Lysine Demethylases</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">27</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1177/1087057111418228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1177%2F1087057111418228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=21859682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38Xit1OltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=27-38&author=V.+Yuauthor=T.+Fischauthor=A.+M.+Longauthor=J.+Tangauthor=J.+H.+Leeauthor=M.+Hierlauthor=H.+Chenauthor=P.+Yakowecauthor=R.+Schwandnerauthor=R.+Emkey&title=High-Throughput+TR-FRET+Assays+for+Identifying+Inhibitors+of+LSD1+and+JMJD2C+Histone+Lysine+Demethylases&doi=10.1177%2F1087057111418228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput TR-FRET assays for identifying inhibitors of LSD1 and JMJD2C histone lysine demethylases</span></div><div class="casAuthors">Yu, Violeta; Fisch, Tanja; Long, Alexander M.; Tang, Jin; Lee, Josie Han; Hierl, Markus; Chen, Hao; Yakowec, Peter; Schwandner, Ralf; Emkey, Renee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Lysine demethylase 1 (LSD1) and Jumonji C domain-contg. oxygenase D2C (JMJD2C) participate in regulating the methylation status of histone H3 lysine residues.  In some contexts, LSD1 and JMJD2C activity causes enhanced cellular proliferation, which may lead to tumorigenesis.  The authors explored the utility of time-resolved fluorescence resonance energy transfer (TR-FRET) immunoassays, which employed peptides consisting of the first 21 amino acids of histone H3 in which lysine 4 (H3K4) or lysine 9 (H3K9) was methylated (me) to quantify LSD1 and JMJD2C activity.  The LSD1 assay monitored demethylation of the H3K4me1 peptide using an antibody that recognizes H3K4me1 but not the unmethylated peptide product.  The JMJD2C assay measured demethylation of H3K9me3 with an antibody that selectively recognizes H3K9me2.  The optimized conditions resulted in robust assays (Z' > 0.7) that required only 3 to 6 nM of enzyme in a reaction vol. of 6 to 10 μL.  These assays were used to compare the activity of different LSD1 constructs and to det. the apparent Km of each JMJD2C substrate.  Finally, both assays were used in a high-throughput setting for identifying demethylase inhibitors.  Compds. discovered by these TR-FRET methods may lead to powerful tools for ascertaining the roles of demethylases in a cellular context and ultimately for potential cancer treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVw3snvTyahbVg90H21EOLACvtfcHk0ljvBiLlYkOG9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xit1OltrY%253D&md5=fe4eb00acca43ce6633377e9720a5d83</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1177%2F1087057111418228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111418228%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DV.%26aulast%3DFisch%26aufirst%3DT.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DHierl%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DYakowec%26aufirst%3DP.%26aulast%3DSchwandner%26aufirst%3DR.%26aulast%3DEmkey%26aufirst%3DR.%26atitle%3DHigh-Throughput%2520TR-FRET%2520Assays%2520for%2520Identifying%2520Inhibitors%2520of%2520LSD1%2520and%2520JMJD2C%2520Histone%2520Lysine%2520Demethylases%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D27%26epage%3D38%26doi%3D10.1177%2F1087057111418228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Vianello, P.; Sartori, L.; Amigoni, F.; Cappa, A.; Fagá, G.; Fattori, R.; Legnaghi, E.; Ciossani, G.; Mattevi, A.; Meroni, G.; Moretti, L.; Cecatiello, V.; Pasqualato, S.; Romussi, A.; Thaler, F.; Trifirò, P.; Vultaggio, S.; Varasi, M.; Mercurio, C.</span><span> </span><span class="NLM_article-title">Thieno[3,2-B]pyrrole-5-Carboxamides as Novel Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship</span>.  <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span>, DOI <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01019</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&author=P.+Vianello&author=L.+Sartori&author=F.+Amigoni&author=A.+Cappa&author=G.+Fag%C3%A1&author=R.+Fattori&author=E.+Legnaghi&author=G.+Ciossani&author=A.+Mattevi&author=G.+Meroni&author=L.+Moretti&author=V.+Cecatiello&author=S.+Pasqualato&author=A.+Romussi&author=F.+Thaler&author=P.+Trifir%C3%B2&author=S.+Vultaggio&author=M.+Varasi&author=C.+Mercurio&title=Thieno%5B3%2C2-B%5Dpyrrole-5-Carboxamides+as+Novel+Reversible+Inhibitors+of+Histone+Lysine+Demethylase+KDM1A%2FLSD1.+Part+2%3A+Structure-Based+Drug+Design+and+Structure%E2%80%93Activity+Relationship&doi=10.1021%2Facs.jmedchem.6b01019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26aulast%3DVianello%26aufirst%3DP.%26atitle%3DThieno%255B3%252C2-B%255Dpyrrole-5-Carboxamides%2520as%2520Novel%2520Reversible%2520Inhibitors%2520of%2520Histone%2520Lysine%2520Demethylase%2520KDM1A%252FLSD1.%2520Part%25202%253A%2520Structure-Based%2520Drug%2520Design%2520and%2520Structure%25E2%2580%2593Activity%2520Relationship%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26doi%3D10.1021%2Facs.jmedchem.6b01019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G.</span><span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+Baellauthor=G.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0ljvBiLlYkOG9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DHolloway%26aufirst%3DG.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Pouliot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeanmart, S.</span><span> </span><span class="NLM_article-title">Pan Assay Interference Compounds (PAINS) and Other Promiscuous Compounds in Antifungal Research</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00361</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00361" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=497-503&author=M.+Pouliotauthor=S.+Jeanmart&title=Pan+Assay+Interference+Compounds+%28PAINS%29+and+Other+Promiscuous+Compounds+in+Antifungal+Research&doi=10.1021%2Facs.jmedchem.5b00361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Pan Assay Interference Compounds (PAINS) and Other Promiscuous Compounds in Antifungal Research</span></div><div class="casAuthors">Pouliot, Martin; Jeanmart, Stephane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">497-503</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Every week, articles disclosing new antifungal leads reported as promising starting points for optimization projects are published.  In many cases, the mechanism that accounts for their antifungal activity has not been fully elucidated.  More significantly, the detrimental impact that could result from certain embedded chem. features has been underestimated or even overlooked.  In the course of the authors' research in the agrochem. area, the authors have concluded that in many cases such leads are actually nonoptimizable because they either contain what are now recognized as pan assay interference compds. (PAINS) or other promiscuous groups.  This article is aimed at highlighting the pitfalls the authors have encountered and hopefully to steer other research groups away from them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqURFxdfKNQIbVg90H21EOLACvtfcHk0lgi3gFUKH171g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7rE&md5=af11fcafd47de435fcc5fdf05080023a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00361%26sid%3Dliteratum%253Aachs%26aulast%3DPouliot%26aufirst%3DM.%26aulast%3DJeanmart%26aufirst%3DS.%26atitle%3DPan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520and%2520Other%2520Promiscuous%2520Compounds%2520in%2520Antifungal%2520Research%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D497%26epage%3D503%26doi%3D10.1021%2Facs.jmedchem.5b00361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Malo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanley, J. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cerquozzi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadon, R.</span><span> </span><span class="NLM_article-title">Statistical Practice in High-Throughput Screening Data Analysis</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span><span class="refDoi"> DOI: 10.1038/nbt1186</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1038%2Fnbt1186" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=16465162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFGqs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=167-175&author=N.+Maloauthor=J.+a+Hanleyauthor=S.+Cerquozziauthor=J.+Pelletierauthor=R.+Nadon&title=Statistical+Practice+in+High-Throughput+Screening+Data+Analysis&doi=10.1038%2Fnbt1186"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Statistical practice in high-throughput screening data analysis</span></div><div class="casAuthors">Malo, Nathalie; Hanley, James A.; Cerquozzi, Sonia; Pelletier, Jerry; Nadon, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-175</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening is an early crit. step in drug discovery.  Its aim is to screen a large no. of diverse chem. compds. to identify candidate 'hits' rapidly and accurately.  Few statistical tools are currently available, however, to detect quality hits with a high degree of confidence.  We examine statistical aspects of data preprocessing and hit identification for primary screens.  We focus on concerns related to positional effects of wells within plates, choice of hit threshold and the importance of minimizing false-pos. and false-neg. rates.  We argue that replicate measurements are needed to verify assumptions of current methods and to suggest data anal. strategies when assumptions are not met.  The integration of replicates with robust statistical methods in primary screens will facilitate the discovery of reliable hits, ultimately improving the sensitivity and specificity of the screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXXI5Qnc2Z9rVg90H21EOLACvtfcHk0lgi3gFUKH171g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFGqs70%253D&md5=170a0238341aba8add873d4f74196d18</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnbt1186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1186%26sid%3Dliteratum%253Aachs%26aulast%3DMalo%26aufirst%3DN.%26aulast%3DHanley%26aufirst%3DJ.%2Ba%26aulast%3DCerquozzi%26aufirst%3DS.%26aulast%3DPelletier%26aufirst%3DJ.%26aulast%3DNadon%26aufirst%3DR.%26atitle%3DStatistical%2520Practice%2520in%2520High-Throughput%2520Screening%2520Data%2520Analysis%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D167%26epage%3D175%26doi%3D10.1038%2Fnbt1186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span>BIOVIA Pipeline Pilot | Scientific Workflow Authoring Application for Data Analysis; <span class="NLM_publisher-name">Biovia</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">; </span><a href="http://accelrys.com/products/collaborative-science/biovia-pipeline-pilot/" class="extLink">http://accelrys.com/products/collaborative-science/biovia-pipeline-pilot/</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=BIOVIA+Pipeline+Pilot+%7C+Scientific+Workflow+Authoring+Application+for+Data+Analysis%3B+Biovia%3A+San+Diego%2C+CA%2C+2017%3B+http%3A%2F%2Faccelrys.com%2Fproducts%2Fcollaborative-science%2Fbiovia-pipeline-pilot%2F+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DBIOVIA%2520Pipeline%2520Pilot%2520%257C%2520Scientific%2520Workflow%2520Authoring%2520Application%2520for%2520Data%2520Analysis%26pub%3DBiovia%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Copeland, R. A.</span> <span class="citation_source-book">Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</span>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>, <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">20070</span><span class="NLM_x">–</span> <span class="NLM_lpage">20074</span><span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+Basis+of+LSD1-CoREST+Selectivity+in+Histone+H3+Recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0lgi3gFUKH171g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520Basis%2520of%2520LSD1-CoREST%2520Selectivity%2520in%2520Histone%2520H3%2520Recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Karasulu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patil, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiel, W.</span><span> </span><span class="NLM_article-title">Amine Oxidation Mediated by Lysine-Specific Demethylase 1: Quantum Mechanics/molecular Mechanics Insights into Mechanism and Role of Lysine 661</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">13400</span><span class="NLM_x">–</span> <span class="NLM_lpage">13413</span><span class="refDoi"> DOI: 10.1021/ja403582u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja403582u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=13400-13413&author=B.+Karasuluauthor=M.+Patilauthor=W.+Thiel&title=Amine+Oxidation+Mediated+by+Lysine-Specific+Demethylase+1%3A+Quantum+Mechanics%2Fmolecular+Mechanics+Insights+into+Mechanism+and+Role+of+Lysine+661&doi=10.1021%2Fja403582u"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja403582u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja403582u%26sid%3Dliteratum%253Aachs%26aulast%3DKarasulu%26aufirst%3DB.%26aulast%3DPatil%26aufirst%3DM.%26aulast%3DThiel%26aufirst%3DW.%26atitle%3DAmine%2520Oxidation%2520Mediated%2520by%2520Lysine-Specific%2520Demethylase%25201%253A%2520Quantum%2520Mechanics%252Fmolecular%2520Mechanics%2520Insights%2520into%2520Mechanism%2520and%2520Role%2520of%2520Lysine%2520661%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D13400%26epage%3D13413%26doi%3D10.1021%2Fja403582u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Morris, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huey, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindstrom, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belew, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock4 and AutoDockTools4: Automated Docking with Selective Receptor Flexibility</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2785</span><span class="NLM_x">–</span> <span class="NLM_lpage">2791</span><span class="refDoi"> DOI: 10.1002/jcc.21256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fjcc.21256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=19399780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2009&pages=2785-2791&author=G.+Morrisauthor=R.+Hueyauthor=W.+Lindstromauthor=M.+F.+Sannerauthor=R.+K.+Belewauthor=D.+S.+Goodsellauthor=A.+J.+Olson&title=AutoDock4+and+AutoDockTools4%3A+Automated+Docking+with+Selective+Receptor+Flexibility&doi=10.1002%2Fjcc.21256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock and AutoDockTools: Automated docking with selective receptor flexibility</span></div><div class="casAuthors">Morris, Garrett M.; Huey, Ruth; Lindstrom, William; Sanner, Michel F.; Belew, Richard K.; Goodsell, David S.; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2785-2791</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe the testing and release of AutoDock4 and the accompanying graphical user interface AutoDockTools.  AutoDock4 incorporates limited flexibility in the receptor.  Several tests are reported here, including a redocking expt. with 188 diverse ligand-protein complexes and a cross-docking expt. using flexible sidechains in 87 HIV protease complexes.  We also report its utility in anal. of covalently bound ligands, using both a grid-based docking method and a modification of the flexible sidechain technique. © 2009 Wiley Periodicals, Inc.  J Comput Chem, 2009.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqktE6WAo4GEbVg90H21EOLACvtfcHk0ljxyPfdZbAgWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1GitrnK&md5=679ce22fc50e9291c9aa16e7a1855845</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21256%26sid%3Dliteratum%253Aachs%26aulast%3DMorris%26aufirst%3DG.%26aulast%3DHuey%26aufirst%3DR.%26aulast%3DLindstrom%26aufirst%3DW.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DBelew%26aufirst%3DR.%2BK.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock4%2520and%2520AutoDockTools4%253A%2520Automated%2520Docking%2520with%2520Selective%2520Receptor%2520Flexibility%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2009%26volume%3D30%26spage%3D2785%26epage%3D2791%26doi%3D10.1002%2Fjcc.21256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Korb, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stützle, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Exner, T. E.</span><span> </span><span class="NLM_article-title">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1021/ci800298z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800298z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=84-96&author=O.+Korbauthor=T.+St%C3%BCtzleauthor=T.+E.+Exner&title=Empirical+Scoring+Functions+for+Advanced+Protein-Ligand+Docking+with+PLANTS&doi=10.1021%2Fci800298z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Scoring Functions for Advanced Protein-Ligand Docking with PLANTS</span></div><div class="casAuthors">Korb, Oliver; Stuetzle, Thomas; Exner, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-96</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this paper we present two empirical scoring functions, PLANTSCHEMPLP and PLANTSPLP, designed for our docking algorithm PLANTS (Protein-Ligand ANT System), which is based on ant colony optimization (ACO).  They are related, regarding their functional form, to parts of already published scoring functions and force fields.  The parametrization procedure described here was able to identify several parameter settings showing an excellent performance for the task of pose prediction on two test sets comprising 298 complexes in total.  Up to 87% of the complexes of the Astex diverse set and 77% of the CCDC/Astex clean listnc (noncovalently bound complexes of the clean list) could be reproduced with root-mean-square deviations of less than 2 Å with respect to the exptl. detd. structures.  A comparison with the state-of-the-art docking tool GOLD clearly shows that this is, esp. for the druglike Astex diverse set, an improvement in pose prediction performance.  Addnl., optimized parameter settings for the search algorithm were identified, which can be used to balance pose prediction reliability and search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_kyR7CGk9mbVg90H21EOLACvtfcHk0ljxyPfdZbAgWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisVSquw%253D%253D&md5=fce960b75b9f8906354b7c97ff84c092</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci800298z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800298z%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DO.%26aulast%3DSt%25C3%25BCtzle%26aufirst%3DT.%26aulast%3DExner%26aufirst%3DT.%2BE.%26atitle%3DEmpirical%2520Scoring%2520Functions%2520for%2520Advanced%2520Protein-Ligand%2520Docking%2520with%2520PLANTS%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D84%26epage%3D96%26doi%3D10.1021%2Fci800298z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljxyPfdZbAgWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Bottegoni, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocchia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Recanatini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavalli, A.</span><span> </span><span class="NLM_article-title">AClAP, Autonomous Hierarchical Agglomerative Cluster Analysis Based Protocol to Partition Conformational Datasets</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">e58</span><span class="NLM_x">–</span> <span class="NLM_lpage">e65</span><span class="refDoi"> DOI: 10.1093/bioinformatics/btl212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1093%2Fbioinformatics%2Fbtl212" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2006&pages=e58-e65&author=G.+Bottegoniauthor=W.+Rocchiaauthor=M.+Recanatiniauthor=A.+Cavalli&title=AClAP%2C+Autonomous+Hierarchical+Agglomerative+Cluster+Analysis+Based+Protocol+to+Partition+Conformational+Datasets&doi=10.1093%2Fbioinformatics%2Fbtl212"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtl212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtl212%26sid%3Dliteratum%253Aachs%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DRocchia%26aufirst%3DW.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DAClAP%252C%2520Autonomous%2520Hierarchical%2520Agglomerative%2520Cluster%2520Analysis%2520Based%2520Protocol%2520to%2520Partition%2520Conformational%2520Datasets%26jtitle%3DBioinformatics%26date%3D2006%26volume%3D22%26spage%3De58%26epage%3De65%26doi%3D10.1093%2Fbioinformatics%2Fbtl212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Degliesposti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Portioli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parenti, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rastelli, G.</span><span> </span><span class="NLM_article-title">BEAR, a Novel Virtual Screening Methodology for Drug Discovery</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span><span class="refDoi"> DOI: 10.1177/1087057110388276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1177%2F1087057110388276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=21084717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsl2gsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2011&pages=129-133&author=G.+Degliespostiauthor=C.+Portioliauthor=M.+D.+Parentiauthor=G.+Rastelli&title=BEAR%2C+a+Novel+Virtual+Screening+Methodology+for+Drug+Discovery&doi=10.1177%2F1087057110388276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">BEAR, a novel virtual screening methodology for drug discovery</span></div><div class="casAuthors">Degliesposti, Gianluca; Portioli, Corinne; Parenti, Marco Daniele; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129-133</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">BEAR (binding estn. after refinement) is a new virtual screening technol. based on the conformational refinement of docking poses through mol. dynamics and prediction of binding free energies using accurate scoring functions.  Here, the authors report the results of an extensive benchmark of the BEAR performance in identifying a smaller subset of known inhibitors seeded in a large (1.5 million) database of compds.  BEAR performance proved strikingly better if compared with std. docking screening methods.  The validations performed so far showed that BEAR is a reliable tool for drug discovery.  It is fast, modular, and automated, and it can be applied to virtual screenings against any biol. target with known structure and any database of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanbHvF9oSELVg90H21EOLACvtfcHk0li2-OraDu0r0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsl2gsLk%253D&md5=d07d88fff7b821ad76c0557ae2460903</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1177%2F1087057110388276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057110388276%26sid%3Dliteratum%253Aachs%26aulast%3DDegliesposti%26aufirst%3DG.%26aulast%3DPortioli%26aufirst%3DC.%26aulast%3DParenti%26aufirst%3DM.%2BD.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DBEAR%252C%2520a%2520Novel%2520Virtual%2520Screening%2520Methodology%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2011%26volume%3D16%26spage%3D129%26epage%3D133%26doi%3D10.1177%2F1087057110388276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Heffernan, M. L. R.; Foglesong, R. J.; Hopkins, S. C.; Soukri, M.; Jones, S. W.; Spear, K. L.; Varney, M. A.</span><span> </span><span class="NLM_article-title">Fluoro-Substituted Inhibitors of D-Amino Acid Oxidase</span>. U.S. Patent 7,884,124 B2,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=M.+L.+R.+Heffernan&author=R.+J.+Foglesong&author=S.+C.+Hopkins&author=M.+Soukri&author=S.+W.+Jones&author=K.+L.+Spear&author=M.+A.+Varney&title=Fluoro-Substituted+Inhibitors+of+D-Amino+Acid+Oxidase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHeffernan%26aufirst%3DM.%2BL.%2BR.%26atitle%3DFluoro-Substituted%2520Inhibitors%2520of%2520D-Amino%2520Acid%2520Oxidase%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Ching, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kam, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merits, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, L. F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, C. L. L.</span><span> </span><span class="NLM_article-title">Trisubstituted thieno[3,2-B]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">9196</span><span class="NLM_x">–</span> <span class="NLM_lpage">9213</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9196-9213&author=K.+C.+Chingauthor=Y.+W.+Kamauthor=A.+Meritsauthor=L.+F.+P.+Ngauthor=C.+L.+L.+Chai&title=Trisubstituted+thieno%5B3%2C2-B%5Dpyrrole+5-Carboxamides+as+Potent+Inhibitors+of+Alphaviruses&doi=10.1021%2Facs.jmedchem.5b01047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Trisubstituted Thieno[3,2-b]pyrrole 5-Carboxamides as Potent Inhibitors of Alphaviruses</span></div><div class="casAuthors">Ching, Kuan-Chieh; Kam, Yiu-Wing; Merits, Andres; Ng, Lisa F. P.; Chai, Christina L. L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9196-9213</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chikungunya virus (CHIKV) is a re-emerging vector-borne alphavirus and is transmitted to humans by Aedes mosquitoes.  Despite the re-emergence of CHIKV as an epidemic threat, there is no approved effective antiviral treatment currently available for CHIKV.  Herein, we report the synthesis and structure-activity relationship studies of a class of thieno[3,2-b]pyrroles and the discovery of a trisubstituted thieno[3,2-b]pyrrole 5-carboxamide 15c (I) that exhibits potent inhibitory activity against in vitro CHIKV infection.  Compd. 15c displayed low micromolar activity (EC50 value of ca. 2 μM) and limited cytotoxic liability (CC50 > 100 μM) therefore furnishing a selectivity index of greater than 32.  Notably, 15c not only controlled viral RNA prodn., but efficiently inhibited the expression of CHIKV nsP1, nsP3, capsid, and E2 proteins at a concn. as low as 2.5 μM.  More importantly, 15c also demonstrated broad spectrum antiviral activity against other clin. important alphaviruses such as O'nyong-nyong virus and Sindbis virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC7wEIPTmwQ7Vg90H21EOLACvtfcHk0li2-OraDu0r0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKlt7fI&md5=dc938296fd5fd7628b7de37590b6918a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01047%26sid%3Dliteratum%253Aachs%26aulast%3DChing%26aufirst%3DK.%2BC.%26aulast%3DKam%26aufirst%3DY.%2BW.%26aulast%3DMerits%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DL.%2BF.%2BP.%26aulast%3DChai%26aufirst%3DC.%2BL.%2BL.%26atitle%3DTrisubstituted%2520thieno%255B3%252C2-B%255Dpyrrole%25205-Carboxamides%2520as%2520Potent%2520Inhibitors%2520of%2520Alphaviruses%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9196%26epage%3D9213%26doi%3D10.1021%2Facs.jmedchem.5b01047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C. P.</span><span> </span><span class="NLM_article-title">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+P.+Somervaille&title=The+Histone+Demethylase+KDM1A+Sustains+the+Oncogenic+Potential+of+MLL-AF9+Leukemia+Stem+Cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0lgn90eB-F3xcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%2BP.%26atitle%3DThe%2520Histone%2520Demethylase%2520KDM1A%2520Sustains%2520the%2520Oncogenic%2520Potential%2520of%2520MLL-AF9%2520Leukemia%2520Stem%2520Cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanigan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Loss of LSD1 (Lysine-Specific Demethylase 1) Suppresses Growth and Alters Gene Expression of Human Colon Cancer Cells in a p53- and DNMT1(DNA Methyltransferase 1)-Independent Manner</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">459</span><span class="NLM_x">–</span> <span class="NLM_lpage">468</span><span class="refDoi"> DOI: 10.1042/BJ20121360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1042%2FBJ20121360" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=459-468&author=L.+Jinauthor=C.+L.+Haniganauthor=Y.+Wuauthor=W.+Wangauthor=B.+H.+Parkauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Loss+of+LSD1+%28Lysine-Specific+Demethylase+1%29+Suppresses+Growth+and+Alters+Gene+Expression+of+Human+Colon+Cancer+Cells+in+a+p53-+and+DNMT1%28DNA+Methyltransferase+1%29-Independent+Manner&doi=10.1042%2FBJ20121360"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2FBJ20121360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121360%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHanigan%26aufirst%3DC.%2BL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DLoss%2520of%2520LSD1%2520%2528Lysine-Specific%2520Demethylase%25201%2529%2520Suppresses%2520Growth%2520and%2520Alters%2520Gene%2520Expression%2520of%2520Human%2520Colon%2520Cancer%2520Cells%2520in%2520a%2520p53-%2520and%2520DNMT1%2528DNA%2520Methyltransferase%25201%2529-Independent%2520Manner%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D459%26epage%3D468%26doi%3D10.1042%2FBJ20121360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone Demethylase Lsd1 Represses Hematopoietic Stem and Progenitor Cell Signatures during Blood Cell Maturation</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00633</span><span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.-J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+Demethylase+Lsd1+Represses+Hematopoietic+Stem+and+Progenitor+Cell+Signatures+during+Blood+Cell+Maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0lgn90eB-F3xcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.-J.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520Demethylase%2520Lsd1%2520Represses%2520Hematopoietic%2520Stem%2520and%2520Progenitor%2520Cell%2520Signatures%2520during%2520Blood%2520Cell%2520Maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">LSD1: Oxidative Chemistry for Multifaceted Functions in Chromatin Regulation</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">189</span><span class="refDoi"> DOI: 10.1016/j.tibs.2008.01.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1016%2Fj.tibs.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=18343668" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktlCkurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=181-189&author=F.+Fornerisauthor=C.+Bindaauthor=E.+Battaglioliauthor=A.+Mattevi&title=LSD1%3A+Oxidative+Chemistry+for+Multifaceted+Functions+in+Chromatin+Regulation&doi=10.1016%2Fj.tibs.2008.01.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1: oxidative chemistry for multifaceted functions in chromatin regulation</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">181-189</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Three years after its discovery, lysine-specific demethylase 1 remains at the forefront of chromatin research.  Its demethylase activity on Lys4 of histone H3 supports its role in gene repression.  By contrast, the biochem. mechanisms underlying lysine-specific demethylase 1 involvement in transcriptional activation are not firmly established.  Structural studies highlight a specific binding site for the histone H3 N-terminal tail and a catalytic machinery that is closely related to that of other flavin-dependent amine oxidases.  These insights are crucial for the development of demethylation inhibitors.  Furthermore, the exploration of putative non-histone substrates and potential signaling roles of hydrogen peroxide produced by the demethylation reaction could lead to new paradigms in chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNay__PwncNLVg90H21EOLACvtfcHk0lgn90eB-F3xcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktlCkurk%253D&md5=dc5b6d422b49b41f67b95ca812ea841f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DLSD1%253A%2520Oxidative%2520Chemistry%2520for%2520Multifaceted%2520Functions%2520in%2520Chromatin%2520Regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D181%26epage%3D189%26doi%3D10.1016%2Fj.tibs.2008.01.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+Structural%2C+and+Biological+Evaluation+of+Tranylcypromine+Derivatives+as+Inhibitors+of+Histone+Demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0lhp3l8746fsqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520Structural%252C%2520and%2520Biological%2520Evaluation%2520of%2520Tranylcypromine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Histone%2520Demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhp3l8746fsqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span>SigmaPlot, <span class="NLM_edition">v11</span>; <span class="NLM_publisher-name">Systat Software Inc.</span>: <span class="NLM_publisher-loc">San Jose, CA</span>; <a href="https://systatsoftware.com/products/sigmaplot/" class="extLink">https://systatsoftware.com/products/sigmaplot/</a> (accessed July 20,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=SigmaPlot%2C+v11%3B+Systat+Software+Inc.%3A+San+Jose%2C+CA%3B+https%3A%2F%2Fsystatsoftware.com%2Fproducts%2Fsigmaplot%2F+%28accessed+July+20%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSigmaPlot%26pub%3DSystat%2520Software%2520Inc%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Moretti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sartori, L.</span><span> </span><span class="NLM_article-title">Software Infrastructure for Computer-Aided Drug Discovery and Development, a Practical Example with Guidelines</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">390</span><span class="refDoi"> DOI: 10.1002/minf.201501037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fminf.201501037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=27546042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=382-390&author=L.+Morettiauthor=L.+Sartori&title=Software+Infrastructure+for+Computer-Aided+Drug+Discovery+and+Development%2C+a+Practical+Example+with+Guidelines&doi=10.1002%2Fminf.201501037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Software Infrastructure for Computer-aided Drug Discovery and Development, a Practical Example with Guidelines</span></div><div class="casAuthors">Moretti, Loris; Sartori, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">8-9</span>),
    <span class="NLM_cas:pages">382-390</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In the field of Computer-Aided Drug Discovery and Development (CADDD) the proper software infrastructure is essential for everyday investigations.  The creation of such an environment should be carefully planned and implemented with certain features in order to be productive and efficient.  Here, we describe a soln. to integrate std. computational services into a functional unit that empowers modeling applications for drug discovery.  This system allows users with various level of expertise to run in silico expts. automatically and without the burden of file formatting for different software, managing the actual computation, keeping track of the activities and graphical rendering of the structural outcomes.  To showcase the potential of this approach, performances of five different docking programs on an Hiv-1 protease test set are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUtVBT1Wj49LVg90H21EOLACvtfcHk0lhp3l8746fsqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSgt7rL&md5=c9db5ce783d44c6d15f889cbec5ca3f7</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fminf.201501037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201501037%26sid%3Dliteratum%253Aachs%26aulast%3DMoretti%26aufirst%3DL.%26aulast%3DSartori%26aufirst%3DL.%26atitle%3DSoftware%2520Infrastructure%2520for%2520Computer-Aided%2520Drug%2520Discovery%2520and%2520Development%252C%2520a%2520Practical%2520Example%2520with%2520Guidelines%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D382%26epage%3D390%26doi%3D10.1002%2Fminf.201501037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Seeliger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Groot, B. L.</span><span> </span><span class="NLM_article-title">Ligand Docking and Binding Site Analysis with PyMOL and Autodock/Vina</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">417</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1007/s10822-010-9352-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1007%2Fs10822-010-9352-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=20401516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=417-422&author=D.+Seeligerauthor=B.+L.+De+Groot&title=Ligand+Docking+and+Binding+Site+Analysis+with+PyMOL+and+Autodock%2FVina&doi=10.1007%2Fs10822-010-9352-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand docking and binding site analysis with PyMOL and Autodock/Vina</span></div><div class="casAuthors">Seeliger, Daniel; Groot, Bert L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">417-422</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Docking of small mol. compds. into the binding site of a receptor and estg. the binding affinity of the complex is an important part of the structure-based drug design process.  For a thorough understanding of the structural principles that det. the strength of a protein/ligand complex both, an accurate and fast docking protocol and the ability to visualize binding geometries and interactions are mandatory.  Here we present an interface between the popular mol. graphics system PyMOL and the mol. docking suites Autodock and Vina and demonstrate how the combination of docking and visualization can aid structure-based drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTR4WZx4OqG7Vg90H21EOLACvtfcHk0lj352k3RZk5rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahtLo%253D&md5=35f8b79d6c1d6582919878a53b4a5129</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9352-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9352-6%26sid%3Dliteratum%253Aachs%26aulast%3DSeeliger%26aufirst%3DD.%26aulast%3DDe%2BGroot%26aufirst%3DB.%2BL.%26atitle%3DLigand%2520Docking%2520and%2520Binding%2520Site%2520Analysis%2520with%2520PyMOL%2520and%2520Autodock%252FVina%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D417%26epage%3D422%26doi%3D10.1007%2Fs10822-010-9352-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span>R, A Language and Environment for Statistical Computing; <span class="NLM_publisher-name">R Foundation for Statistical Computing</span>: <span class="NLM_publisher-loc">Vienna, Austria</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">; </span><a href="http://www.r-project.org/" class="extLink">http://www.r-project.org/</a> (accessed July 20, 2016).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=R%2C+A+Language+and+Environment+for+Statistical+Computing%3B+R+Foundation+for+Statistical+Computing%3A+Vienna%2C+Austria%2C+2016%3B+http%3A%2F%2Fwww.r-project.org%2F+%28accessed+July+20%2C+2016%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DR%252C%2520A%2520Language%2520and%2520Environment%2520for%2520Statistical%2520Computing%26pub%3DR%2520Foundation%2520for%2520Statistical%2520Computing%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Case, D. A.; Berryman, J. T.; Betz, R. M.; Cerutti, D. S.; Cheatham, T. E.,  III; Darden, T. A.; Duke, R. E.; Giese, T. J.; Gohlke, H.; Goetz, A. W.; Homeyer, N.; Izadi, S.; Janowski, P.; Kaus, J.; Kovalenko, A.; Lee, T. S.; LeGrand, S.; Li, P.; Luchko, T.; Luo, R.; Madej, B.; Merz, K. M.; Monard, G.; Needham, P.; Nguyen, H.; Nguyen, H. T.; Omelyan, I.; Onufriev, A.; Roe, D. R.; Roitberg, A.; Salomon-Ferrer, R.; Simmerling, C.L.; Smith, W.; Swails, J.; Walker, R.C.; Wang, J.; Wolf, R. M.; Wu, X.; York, D.M.; Kollman, P.A.</span>AMBER 2015; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=D.+A.+Case&author=J.+T.+Berryman&author=R.+M.+Betz&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=T.+J.+Giese&author=H.+Gohlke&author=A.+W.+Goetz&author=N.+Homeyer&author=S.+Izadi&author=P.+Janowski&author=J.+Kaus&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=P.+Li&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=G.+Monard&author=P.+Needham&author=H.+Nguyen&author=H.+T.+Nguyen&author=I.+Omelyan&author=A.+Onufriev&author=D.+R.+Roe&author=A.+Roitberg&author=R.+Salomon-Ferrer&author=C.L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=D.M.+York&author=P.A.+Kollman&title=AMBER+2015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%25202015%26pub%3DUniversity%2520of%2520California%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera--a Visualization System for Exploratory Research and Analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span><span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%2D%2Da+Visualization+System+for+Exploratory+Research+and+Analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lj352k3RZk5rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera--a%2520Visualization%2520System%2520for%2520Exploratory%2520Research%2520and%2520Analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Antonow, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrafa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dawood, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahmed, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haque, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thurston, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinzalla, G.</span><span> </span><span class="NLM_article-title">Facile Oxidation of Electron-Poor Benzo[b]thiophenes to the Corresponding Sulfones with an Aqueous Solution of H<sub>2</sub>O<sub>2</sub> and P<sub>2</sub>O<sub>5</sub></span> <span class="citation_source-journal">Chem. Commun. (Cambridge, U. K.)</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">2289</span><span class="NLM_x">–</span> <span class="NLM_lpage">2291</span><span class="refDoi"> DOI: 10.1039/b924333j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=10.1039%2Fb924333j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=20234935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjt1Khsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2010&pages=2289-2291&author=D.+Antonowauthor=T.+Marrafaauthor=I.+Dawoodauthor=T.+Ahmedauthor=M.+R.+Haqueauthor=D.+E.+Thurstonauthor=G.+Zinzalla&title=Facile+Oxidation+of+Electron-Poor+Benzo%5Bb%5Dthiophenes+to+the+Corresponding+Sulfones+with+an+Aqueous+Solution+of+H2O2+and+P2O5&doi=10.1039%2Fb924333j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Facile oxidation of electron-poor benzo[b]thiophenes to the corresponding sulfones with an aqueous solution of H2O2 and P2O5</span></div><div class="casAuthors">Antonow, Dyeison; Marrafa, Teresa; Dawood, Irfaan; Ahmed, Tauheed; Haque, Mohammad R.; Thurston, David E.; Zinzalla, Giovanna</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2289-2291</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A facile oxidn. for the clean conversion of benzo[b]thiophenes to their corresponding sulfones e. g., I is described employing an aq. soln. of H2O2 and P2O5; the soln. can be prepd. and stored on a multi-gram scale with a shelf-life of up to two weeks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf5ccivyBLS7Vg90H21EOLACvtfcHk0ljkLMsNIoIMAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjt1Khsbs%253D&md5=aa121a500c4f905bbdaf42c68159a565</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1039%2Fb924333j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb924333j%26sid%3Dliteratum%253Aachs%26aulast%3DAntonow%26aufirst%3DD.%26aulast%3DMarrafa%26aufirst%3DT.%26aulast%3DDawood%26aufirst%3DI.%26aulast%3DAhmed%26aufirst%3DT.%26aulast%3DHaque%26aufirst%3DM.%2BR.%26aulast%3DThurston%26aufirst%3DD.%2BE.%26aulast%3DZinzalla%26aufirst%3DG.%26atitle%3DFacile%2520Oxidation%2520of%2520Electron-Poor%2520Benzo%255Bb%255Dthiophenes%2520to%2520the%2520Corresponding%2520Sulfones%2520with%2520an%2520Aqueous%2520Solution%2520of%2520H2O2%2520and%2520P2O5%26jtitle%3DChem.%2520Commun.%2520%2528Cambridge%252C%2520U.%2520K.%2529%26date%3D2010%26volume%3D46%26spage%3D2289%26epage%3D2291%26doi%3D10.1039%2Fb924333j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D','PDB','2V1D'); return false;">PDB: 2V1D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGN','PDB','5LGN'); return false;">PDB: 5LGN</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i164"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01018">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_86774"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01018">10.1021/acs.jmedchem.6b01018</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthetic procedures describing, <b>62</b>–<b>65</b>, <b>83e</b>–<b>86e</b>, <b>90e</b>, <b>91e</b>, and <b>92</b>–<b>94</b>, additional figure illustrating the effects of KDM1A down-regulation on mRNA expression of CD11b and CD14 in THP-1 cells, apoptotic induction assessment in THP-1 cells, data collection, and refinement statistics for compound <b>19</b>, electron density map for compound <b>19</b>, interaction map for compound <b>19</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_002.csv">CSV</a>)</p></li><li><p class="inline">Protein Target (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Compound <b>90</b> pose A (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Compound <b>90</b> pose B (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_005.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_001.pdf">jm6b01018_si_001.pdf (829.39 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_002.csv">jm6b01018_si_002.csv (6.23 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_003.pdb">jm6b01018_si_003.pdb (510.58 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_004.pdb">jm6b01018_si_004.pdb (11.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01018/suppl_file/jm6b01018_si_005.pdb">jm6b01018_si_005.pdb (11.84 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates for the crystal structures of KDM1A with compound <b>19</b> can be accessed using PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LGN">5LGN</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01018&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01018%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01018" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799438bc86b3c22","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
