
<html lang="en"     class="pb-page"  data-request-id="0ec42f9d-6f02-423b-9868-883acf78d78e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.0c02124;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Betulonic Acid Derivatives Interfering with Human Coronavirus 229E Replication via the nsp15 Endoribonuclease" /></meta><meta name="dc.Creator" content="Annelies  Stevaert" /></meta><meta name="dc.Creator" content="Besir  Krasniqi" /></meta><meta name="dc.Creator" content="Benjamin  Van Loy" /></meta><meta name="dc.Creator" content="Tien  Nguyen" /></meta><meta name="dc.Creator" content="Joice  Thomas" /></meta><meta name="dc.Creator" content="Julie  Vandeput" /></meta><meta name="dc.Creator" content="Dirk  Jochmans" /></meta><meta name="dc.Creator" content="Volker  Thiel" /></meta><meta name="dc.Creator" content="Ronald  Dijkman" /></meta><meta name="dc.Creator" content="Wim  Dehaen" /></meta><meta name="dc.Creator" content="Arnout  Voet" /></meta><meta name="dc.Creator" content="Lieve  Naesens" /></meta><meta name="dc.Description" content="To develop antiviral therapeutics against human coronavirus (HCoV) infections, suitable coronavirus drug targets and corresponding lead molecules must be urgently identified. Here, we describe the ..." /></meta><meta name="Description" content="To develop antiviral therapeutics against human coronavirus (HCoV) infections, suitable coronavirus drug targets and corresponding lead molecules must be urgently identified. Here, we describe the ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 20, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02124" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02124" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02124" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02124" /></link>
        
    
    

<title>Betulonic Acid Derivatives Interfering with Human Coronavirus 229E Replication via the nsp15 Endoribonuclease | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02124" /></meta><meta property="og:title" content="Betulonic Acid Derivatives Interfering with Human Coronavirus 229E Replication via the nsp15 Endoribonuclease" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0010.jpeg" /></meta><meta property="og:description" content="To develop antiviral therapeutics against human coronavirus (HCoV) infections, suitable coronavirus drug targets and corresponding lead molecules must be urgently identified. Here, we describe the discovery of a class of HCoV inhibitors acting on nsp15, a hexameric protein component of the viral replication-transcription complexes, endowed with immune evasion-associated endoribonuclease activity. Structure–activity relationship exploration of these 1,2,3-triazolo-fused betulonic acid derivatives yielded lead molecule 5h as a strong inhibitor (antiviral EC50: 0.6 μM) of HCoV-229E replication. An nsp15 endoribonuclease active site mutant virus was markedly less sensitive to 5h, and selected resistance to the compound mapped to mutations in the N-terminal part of HCoV-229E nsp15, at an interface between two nsp15 monomers. The biological findings were substantiated by the nsp15 binding mode for 5h, predicted by docking. Hence, besides delivering a distinct class of inhibitors, our study revealed a druggable pocket in the nsp15 hexamer with relevance for anti-coronavirus drug development." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02124"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02124">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02124&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02124&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02124&amp;href=/doi/10.1021/acs.jmedchem.0c02124" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 5632-5644</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02079" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02126" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Betulonic Acid Derivatives Interfering with Human Coronavirus 229E Replication via the nsp15 Endoribonuclease</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Annelies Stevaert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Annelies Stevaert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Virology and Chemotherapy, Rega Institute, KU Leuven, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Annelies++Stevaert">Annelies Stevaert</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3316-2826" title="Orcid link">http://orcid.org/0000-0003-3316-2826</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Besir Krasniqi</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Besir Krasniqi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Design and Synthesis, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#7012150319025e1b0211031e190119301b051c150506151e5e1215"><span class="__cf_email__" data-cfemail="7012150319025e1b0211031e190119301b051c150506151e5e1215">[email protected]</span></a>. Tel.: (+32)496798402.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Besir++Krasniqi">Besir Krasniqi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Benjamin Van Loy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin Van Loy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Virology and Chemotherapy, Rega Institute, KU Leuven, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Van+Loy">Benjamin Van Loy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tien Nguyen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tien Nguyen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biochemistry, Molecular and Structural Biology, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tien++Nguyen">Tien Nguyen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joice Thomas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joice Thomas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Design and Synthesis, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joice++Thomas">Joice Thomas</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5517-4663" title="Orcid link">http://orcid.org/0000-0001-5517-4663</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julie Vandeput</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julie Vandeput</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Virology and Chemotherapy, Rega Institute, KU Leuven, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julie++Vandeput">Julie Vandeput</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dirk Jochmans</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dirk Jochmans</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Virology and Chemotherapy, Rega Institute, KU Leuven, 3000 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dirk++Jochmans">Dirk Jochmans</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Volker Thiel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Volker Thiel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Virology and Immunology (IVI), 3012 Bern and 3012 Bern, Switzerland</div><div class="loa-info-affiliations-info">Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern, 3012 Bern, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Volker++Thiel">Volker Thiel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ronald Dijkman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ronald Dijkman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Virology and Immunology (IVI), 3012 Bern and 3012 Bern, Switzerland</div><div class="loa-info-affiliations-info">Department of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern, 3012 Bern, Switzerland</div><div class="loa-info-affiliations-info">Institute for Infectious Diseases (IFIK), University of Bern, 3012 Bern, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ronald++Dijkman">Ronald Dijkman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wim Dehaen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wim Dehaen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Molecular Design and Synthesis, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wim++Dehaen">Wim Dehaen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Arnout Voet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Arnout Voet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Biochemistry, Molecular and Structural Biology, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Arnout++Voet">Arnout Voet</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3329-2703" title="Orcid link">http://orcid.org/0000-0002-3329-2703</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Lieve Naesens</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lieve Naesens</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratory of Virology and Chemotherapy, Rega Institute, KU Leuven, 3000 Leuven, Belgium</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#dfb3b6baa9baf1b1bebaacbab1ac9fb4aab3baaaa9bab1f1bdba"><span class="__cf_email__" data-cfemail="14787d7162713a7a757167717a67547f6178716162717a3a7671">[email protected]</span></a>. Tel.: (+32)16322098.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lieve++Naesens">Lieve Naesens</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9742-9302" title="Orcid link">http://orcid.org/0000-0001-9742-9302</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02124&amp;href=/doi/10.1021%2Facs.jmedchem.0c02124" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 5632–5644</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 20, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 December 2020</li><li><span class="item_label"><b>Published</b> online</span>20 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02124" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02124</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5632%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAnnelies%2BStevaert%252C%2BBesir%2BKrasniqi%252C%2BBenjamin%2BVan%2BLoy%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c02124%26title%3DBetulonic%2BAcid%2BDerivatives%2BInterfering%2Bwith%2BHuman%2BCoronavirus%2B229E%2BReplication%2Bvia%2Bthe%2Bnsp15%2BEndoribonuclease%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5644%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02124"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1501</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02124" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Betulonic Acid Derivatives Interfering with Human Coronavirus 229E Replication via the nsp15 Endoribonuclease&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Annelies&quot;,&quot;last_name&quot;:&quot;Stevaert&quot;},{&quot;first_name&quot;:&quot;Besir&quot;,&quot;last_name&quot;:&quot;Krasniqi&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Van Loy&quot;},{&quot;first_name&quot;:&quot;Tien&quot;,&quot;last_name&quot;:&quot;Nguyen&quot;},{&quot;first_name&quot;:&quot;Joice&quot;,&quot;last_name&quot;:&quot;Thomas&quot;},{&quot;first_name&quot;:&quot;Julie&quot;,&quot;last_name&quot;:&quot;Vandeput&quot;},{&quot;first_name&quot;:&quot;Dirk&quot;,&quot;last_name&quot;:&quot;Jochmans&quot;},{&quot;first_name&quot;:&quot;Volker&quot;,&quot;last_name&quot;:&quot;Thiel&quot;},{&quot;first_name&quot;:&quot;Ronald&quot;,&quot;last_name&quot;:&quot;Dijkman&quot;},{&quot;first_name&quot;:&quot;Wim&quot;,&quot;last_name&quot;:&quot;Dehaen&quot;},{&quot;first_name&quot;:&quot;Arnout&quot;,&quot;last_name&quot;:&quot;Voet&quot;},{&quot;first_name&quot;:&quot;Lieve&quot;,&quot;last_name&quot;:&quot;Naesens&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;5632-5644&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02124&quot;},&quot;abstract&quot;:&quot;To develop antiviral therapeutics against human coronavirus (HCoV) infections, suitable coronavirus drug targets and corresponding lead molecules must be urgently identified. Here, we describe the discovery of a class of HCoV inhibitors acting on nsp15, a hexameric protein component of the viral replication-transcription complexes, endowed with immune evasion-associated endoribonuclease activity. Structure–activity relationship exploration of these 1,2,3-triazolo-fused betulonic acid derivatives yielded lead molecule 5h as a strong inhibitor (antiviral EC50: 0.6 μM) of HCoV-229E replication. An nsp15 endoribonuclease active site mutant virus was markedly less sensitive to 5h, and selected resistance to the compound mapped to mutations in the N-terminal part of HCoV-229E nsp15, at an interface between two nsp15 monomers. The biological findings were substantiated by the nsp15 binding mode for 5h, predicted by docking. Hence, besides delivering a distinct class of inhibitors, our study revealed a druggable pock&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02124&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02124" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02124&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02124" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02124&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02124" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02124&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02124&amp;href=/doi/10.1021/acs.jmedchem.0c02124" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02124" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02124" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02124%26sid%3Dliteratum%253Aachs%26pmid%3D33877845%26genre%3Darticle%26aulast%3DStevaert%26date%3D2021%26atitle%3DBetulonic%2BAcid%2BDerivatives%2BInterfering%2Bwith%2BHuman%2BCoronavirus%2B229E%2BReplication%2Bvia%2Bthe%2Bnsp15%2BEndoribonuclease%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D5632%26epage%3D5644%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292644" title="SARS-CoV-2">SARS-CoV-2</a>,</li><li><a href="/action/doSearch?ConceptID=292038" title="Azides">Azides</a>,</li><li><a href="/action/doSearch?ConceptID=291456" title="Hydrocarbons">Hydrocarbons</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291235" title="Column chromatography">Column chromatography</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">To develop antiviral therapeutics against human coronavirus (HCoV) infections, suitable coronavirus drug targets and corresponding lead molecules must be urgently identified. Here, we describe the discovery of a class of HCoV inhibitors acting on nsp15, a hexameric protein component of the viral replication-transcription complexes, endowed with immune evasion-associated endoribonuclease activity. Structure–activity relationship exploration of these 1,2,3-triazolo-fused betulonic acid derivatives yielded lead molecule <b>5h</b> as a strong inhibitor (antiviral EC<sub>50</sub>: 0.6 μM) of HCoV-229E replication. An nsp15 endoribonuclease active site mutant virus was markedly less sensitive to <b>5h</b>, and selected resistance to the compound mapped to mutations in the N-terminal part of HCoV-229E nsp15, at an interface between two nsp15 monomers. The biological findings were substantiated by the nsp15 binding mode for <b>5h</b>, predicted by docking. Hence, besides delivering a distinct class of inhibitors, our study revealed a druggable pocket in the nsp15 hexamer with relevance for anti-coronavirus drug development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span>  Note</h4><p class="first last">This article is made available via the <a href="/page/vi/chemistry_coronavirus_research" class="ext-link">ACS COVID-19 subset</a> for unrestricted RESEARCH re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42418" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42418" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Four human CoVs (i.e., HCoV-229E, HCoV-HKU1, HCoV-NL63, and HCoV-OC43) are endemic in the population and account each year for 15 to 30% of common colds.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> These can evolve into life-threatening lower respiratory tract infections in elderly, children, and persons at risk.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> In addition, the current SARS-CoV-2 pandemic is causing a major crisis in terms of human health and socio-economic losses. Within a period of ∼20 years, SARS-CoV-2 is the third zoonotic coronavirus (CoV) to enter the human species, coming after SARS (severe acute respiratory syndrome) and MERS (Middle East respiratory syndrome).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Finally, the <i>Coronaviridae</i> family contains several species causing serious disease in pets and livestock.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Somewhat similar to the respiratory illness caused by endemic HCoVs, SARS-CoV-2 produces no or relatively mild disease in most young persons.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In contrast, in individuals with comorbidities or higher age, the SARS-CoV-2 replication phase is typically followed by a second phase that is characterized by hyperinflammation, acute respiratory distress, and multi-organ failure.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Hence, management of COVID-19 most likely requires antiviral drugs to suppress initial virus replication, plus anti-inflammatory medication, like corticosteroids, to treat severe cases.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Several CoV proteins may be suitable drug targets,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> but, at the moment, only two drug classes have reached formal approval by the FDA, i.e., anti-spike antibodies<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and the nucleotide analogue remdesivir, which inhibits the viral polymerase. Based on its interaction with the highly conserved polymerase catalytic site, remdesivir exhibits pan-coronavirus activity covering also HCoV-229E.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14">(11−14)</a> This broad CoV coverage also applies to the clinical candidate GC376, a catalytic site inhibitor of the CoV main protease (M<sup>pro</sup>).<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15−17)</a> Though less explored, the CoV nsp15 endoribonuclease (EndoU) is a highly attractive drug target since it has no cellular counterpart, its catalytic site is conserved among CoVs, and it is amenable to structure-based design based on available protein structures.<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22">(18−22)</a> Nsp15 is one of the non-structural protein (nsp) components of the replication-transcription complexes (RTCs), the site where CoV RNA synthesis occurs.<a onclick="showRef(event, 'ref5 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref5 ref23 ref24">(5,23,24)</a> Although the functions of nsp15 are not entirely understood, its EndoU function is known to regulate viral RNA synthesis, limit the recognition of viral dsRNA by cellular sensors, and prevent the dsRNA-activated antiviral host cell response.<a onclick="showRef(event, 'ref25 ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27 ref28 ref29">(25−29)</a> The interferon type I evading activity of nsp15 is well elaborated for mouse coronavirus MHV-A59<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and HCoV-229E<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and was recently also demonstrated for SARS-CoV-2.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The concept to inhibit nsp15 is thus unique since it combines a direct antiviral effect with the potential to revert viral evasion from host cell immunity.</div><div class="NLM_p last">We here report identification of a class of HCoV-229E nsp15 inhibitors with a 1,2,3-triazolo-fused betulonic acid structure. We describe their synthesis, structure–activity relationship (SAR), and the mechanistic findings, in particular resistance data, which corroborate nsp15 as the antiviral target for HCoV-229E. These biological data accord with the binding model that we obtained by compound docking in the hexameric nsp15 protein structure. The model also explains why the current lead is active against HCoV-229E, but not other coronaviruses like SARS-CoV-2. In short, our study validates the nsp15 protein, and particularly the interface where the lead compound binds, as a druggable and pertinent target for developing CoV inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24861" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24861" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Compound Synthesis and Structure–Activity Relationship</h3><div class="NLM_p">Since the 1,2,3-triazole moiety has the unique property to both accept and donate hydrogen bonds, introducing this moiety can increase the potency of pharmacologically active molecules.<a onclick="showRef(event, 'ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34">(31−34)</a> We decided to fuse this group to betulonic acid, a pentacyclic triterpenoid compound that was the starting point for a wide variety of agents with potential pharmacological use.<a onclick="showRef(event, 'ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37">(35−37)</a> For instance, the betulinic acid core is present in bevirimat, an HIV maturation inhibitor that has undergone phase 2 clinical evaluation.<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38,39)</a> The 1,2,3-triazolo-fused betulonic acid derivatives (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) were synthesized by our recently developed and convenient “triazolization” method to prepare 1,2,3-triazoles from primary amines and ketones.<a onclick="showRef(event, 'ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref40 ref41 ref42">(40−42)</a> First, Jones oxidation was performed to convert betulin <b>1</b> into betulonic acid <b>2</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Betulin <b>1</b>, a natural compound isolated from the bark of <i>Betula</i> species, is commercially available.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> Next, the triazolization method was applied to betulonic acid <b>2</b> as the ketone source, using primary amines <b>3</b> and 4-nitrophenyl azide <b>4</b>, and the previously reported reaction conditions.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> This yielded a series of 16 1,2,3-triazolo-fused betulonic acids <b>5</b>, most of which were isolated with high yield (∼80%; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Diverse primary amines <b>3</b> were attached to the 1,2,3-triazole ring to introduce a variety of aromatic or aliphatic moieties.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 1,2,3-Triazole-Fused Betulonic Acid Derivatives Starting from Betulin<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Na<sub>2</sub>Cr<sub>2</sub>O<sub>7</sub>, H<sub>2</sub>SO<sub>4</sub>, acetone, H<sub>2</sub>O, 0 °C to r.t., 1 h; (ii) betulonic acid <b>2</b> (1.0 equiv), primary amine <b>3</b> (1.4 equiv), 4-nitrophenyl azide <b>4</b> (1.0 equiv), toluene (0.4 mL), 100 °C, 4 Å MS, 24 h.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Anti-CoV Activity and Selectivity in Human HEL<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> Cells Infected with HCoV-229E</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0009.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">HEL: human embryonic lung fibroblast cells.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Yield after chromatographic purification.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">EC<sub>50</sub>: 50% effective concentration, i.e., compound concentration producing 50% protection against viral cytopathic effect (CPE), as assessed by MTS cell viability assay or microscopic scoring of the CPE.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">CC<sub>50</sub>: 50% cytotoxic concentration determined by MTS cell viability assay.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Selectivity index or ratio of CC<sub>50</sub> to EC<sub>50</sub>, both determined by MTS assay. Values are the mean ± SEM (<i>N</i> = 3).</p></div></div><div></div></div><div class="NLM_p">To establish the antiviral activity of the synthesized compounds, the molecules were submitted to cell-based assays with a broad range of DNA and RNA viruses. This phenotypic screening indicated strong and selective activity in human embryonic lung (HEL) fibroblast cells infected with HCoV-229E. We used a viral cytopathic effect (CPE) reduction assay, in which protection against CPE (expressed as the antiviral EC<sub>50</sub> value) was monitored by the MTS cell viability assay and verified by microscopy. The MTS assay was also used to quantify compound cytotoxicity (expressed as the CC<sub>50</sub> value) in mock-infected cells. Whereas the starting compounds betulin <b>1</b> and betulonic acid <b>2</b> were virtually inactive, almost all 1,2,3-triazolo-fused betulonic acid derivatives proved to be highly effective CoV inhibitors (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Several compounds in the series had EC<sub>50</sub> values below 1 μM, which makes them superior to two known CoV inhibitors, which we used as reference compounds, i.e., K22<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and GS-441524, the nucleoside form of remdesivir.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Three analogues stood out for having superior selectivity, i.e., <b>5g</b>, <b>5h</b>, and <b>5n</b>, having a selectivity index (ratio of CC<sub>50</sub> to EC<sub>50</sub>) of 31, 76, and 27, respectively. On the other hand, far lower activity was noted for <b>5i</b> (EC<sub>50</sub> value: 13 μM), which bears a non-aromatic cyclohexanemethyl substituent. Apparently, introducing this bulky group caused a considerable reduction in antiviral activity and selectivity. The capacity of <b>5h</b> to fully suppress HCoV-229E replication at non-toxic concentrations is evident from the microscopic images in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and the dose–response curves in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B. Also, <b>5h</b> fully prevented the formation of dsRNA intermediates of CoV RNA synthesis, as demonstrated by immunofluorescence staining of dsRNA in HCoV-229E-infected human bronchial epithelial 16HBE cells (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Activity of <b>5h</b> against HCoV-229E. (A) Representative microscopic images showing protection against virus-induced cytopathic effect (CPE) in human embryonic lung (HEL) cells. (B) Dose–response curves for inhibition of virus-induced CPE (solid squares) and for cytotoxicity (open squares) in HEL cells, both determined by MTS cell viability assay. Data points are the mean ± SEM (<i>N</i> = 3). (C) Immunofluorescence detection of viral dsRNA in HCoV-229E-infected human bronchial epithelial 16HBE cells at 24 h post-infection (p.i.). Green: dsRNA and blue: nuclear DAPI staining. Compounds: 12 μM <b>5h</b> or 12 μM GS-441524. Scale bar: 50 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To conduct a SAR exploration around the lead compound <b>5h</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), we first investigated the contribution of the α-methyl-phenylene moiety. Compound <b>5q</b>, in which this entire moiety is missing, had ∼6-fold lower antiviral activity than <b>5h</b>. When only the α-methyl moiety was missing (<b>5r</b>), the activity was not affected. Compound <b>5s</b>, which is the epimer at the 1,2,3-triazole substituent, displayed almost the same EC<sub>50</sub> value as <b>5h</b>, indicating that isomerism does not alter the activity. Cytotoxicity was however slightly decreased, resulting in an even better selectivity index (≥90) than that of <b>5h</b>. In order to elucidate the role of the isopropenyl side chain, we reduced this moiety by hydrogenation, yielding compound <b>5t</b>, which was 10- to 20-fold less active. Replacement of the carboxylic acid moiety by a methyl group (<b>5u</b>) proved deleterious.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. SAR Study around Compound <b>5h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) betulonic acid, NH<sub>4</sub>OAc, 4-nitrophenyl azide, DMF, 80 °C, 18 h, 4 Å MS, isolated yield 86%; (ii) betulonic acid, benzyl amine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 92%; (iii) betulonic acid, (R)-(+)-α-methylbenzylamine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 69%; (iv) dihydrobetulonic acid, (<i>S</i>)-(−)-α-methylbenzylamine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 90%; (v) lupenone, (<i>S</i>)-(−)-α-methylbenzylamine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 83%.</p></p></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Activity of <b>5h</b> Analogues in Human HEL<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> Cells Infected with HCoV-229E</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">antiviral activity<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a> (μM)</th><th class="rowsep1 colsep0" align="center">cytotoxicity<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a> (μM)</th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Code</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (MTS)</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (CPE)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub></th><th class="colsep0 rowsep0" align="center" char=".">SI<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5q</b></td><td class="colsep0 rowsep0" align="left">4.3 ± 0.6</td><td class="colsep0 rowsep0" align="left">3.4 ± 0.4</td><td class="colsep0 rowsep0" align="left">8.3 ± 0.7</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5r</b></td><td class="colsep0 rowsep0" align="left">0.85 ± 0.05</td><td class="colsep0 rowsep0" align="left">0.71 ± 0.04</td><td class="colsep0 rowsep0" align="left">12 ± 0</td><td class="colsep0 rowsep0" align="char" char=".">14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5s</b></td><td class="colsep0 rowsep0" align="left">1.1 ± 0.2</td><td class="colsep0 rowsep0" align="left">0.67 ± 0.02</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char=".">91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5t</b></td><td class="colsep0 rowsep0" align="left">13 ± 5</td><td class="colsep0 rowsep0" align="left">6.1 ± 1.7</td><td class="colsep0 rowsep0" align="left">≥99</td><td class="colsep0 rowsep0" align="char" char=".">7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5u</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5h</b></td><td class="colsep0 rowsep0" align="left">0.65 ± 0.08</td><td class="colsep0 rowsep0" align="left">0.60 ± 0.18</td><td class="colsep0 rowsep0" align="left">49 ± 2</td><td class="colsep0 rowsep0" align="char" char=".">76</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">HEL: human embryonic lung fibroblast cells.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">EC<sub>50</sub>: 50% effective concentration, i.e., compound concentration producing 50% protection against viral cytopathic effect (CPE), as assessed by MTS cell viability assay or microscopic scoring of the CPE.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">CC<sub>50</sub>: 50% cytotoxic concentration determined by MTS cell viability assay.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Selectivity index or ratio of CC<sub>50</sub> to EC<sub>50</sub>, both determined by MTS assay. Values are the mean ± SEM (<i>N</i> = 3).</p></div></div></div><div class="NLM_p last">We next evaluated <b>5h</b> in cell culture assays with a panel of other CoVs. The compound had no inhibitory effect on mouse hepatitis virus A59 (MHV-A59) and feline infectious peritonitis virus (FIPV), in CPE reduction assays in, respectively, murine fibroblast L2 cells and Crandell–Rees Feline Kidney cells (data not shown). HCoV-229E and FIPV belong to the alpha genus, while MHV-A59 belongs to the beta genus comprising also the highly pathogenic species SARS-CoV, MERS-CoV, and SARS-CoV-2.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> In VeroE6-eGFP cells infected with SARS-CoV-2, <b>5h</b> and <b>5t</b> were inactive [see ref <a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> for assay description]. Hence, though nicely active against HCoV-229E, <b>5h</b> appeared, unfortunately, to be confined to this CoV species. In addition, when tested against a broad panel of DNA and RNA viruses, the 1,2,3-triazolo-fused betulonic acid derivatives proved inactive against HIV, herpes simplex virus, vaccinia virus, adenovirus, vesicular stomatitis virus, Coxsackie B4 virus, respiratory syncytial virus, parainfluenza-3 virus, reovirus-1, Sindbis virus, Punta Toro virus, yellow fever virus, and influenza virus (data not shown).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Mechanistic Studies Establishing nsp15 as the Target of <b>5h</b></h3><div class="NLM_p">Given the robust activity of the betulonic acid derivatives against HCoV-229E, we used this virus to reveal their mechanism of action and appreciate how their anti-CoV activity spectrum may be expanded. A time-of-addition experiment indicated that <b>5h</b> acts post-entry at an early stage in viral RNA synthesis since the molecule started to have reduced activity when added at 6 h p.i. (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). For comparison, the action point of the entry inhibitor bafilomycin, an inhibitor of endosomal acidification, was situated before 2 h p.i. K22 was still effective when added as late as 8 h p.i. This CoV inhibitor targets nsp6-dependent anchorage of the viral replication-transcription complexes (RTCs) to host cell-derived double-membrane vesicles.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a></div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <b>5h</b> acts post-entry at an early stage in viral RNA synthesis. Compound addition was delayed until different time points after infecting HEL cells with HCoV-229E, and viral RNA was quantified at 16 h p.i. Compound concentrations: bafilomycin 6.3 nM; <b>5h</b> and K22: 15 μM. The <i>Y</i> axis shows the viral RNA copy number relative to the virus control (mean ± SEM of two independent experiments).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we performed two independent virus passaging experiments to select <b>5h</b>-resistant viruses and identify the viral target. After three cell culture passages under increasing concentrations (up to 40 μM) of <b>5h</b>, HCoV-229E acquired resistance. Whole virus genome sequencing revealed that this was attributed to two substitutions in nsp15, K60R (first selection) and T66I (second selection), located in the N-terminal part of this protein. For both mutants, <b>5h</b> exhibited an antiviral EC<sub>99</sub> value (= concentration producing 100-fold reduction in virus yield) of >40 μM, which is at least 14-fold higher than the EC<sub>99</sub> value of 2.9 μM measured for wild-type (WT) virus (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Both mutant viruses remained fully sensitive to GS-441524. The conclusion that <b>5h</b> targets nsp15 was corroborated by determining its activity against a reverse-engineered EndoU-deficient H250A-mutant HCoV-229E, which lacks the catalytic His250 residue in the EndoU active site.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a><b>5h</b> proved dramatically less active against this mutant (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), producing a maximal reduction in virus yield of 23-fold compared to 1479-fold for WT virus. Again, GS-441524 proved equally active against nsp15-mutant and WT virus. To conclude, we established <b>5h</b> as an inhibitor of nsp15 and showed that its activity is linked to residues Lys60 and Thr66 in the N-terminal part, plus His250 in the EndoU catalytic site of nsp15. This inhibition of nsp15 accords with the time-of-addition profile of <b>5h</b> (see above), showing that the compound interferes with an early stage in viral RNA synthesis.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Mutations in nsp15 confer resistance of HCoV-229E to <b>5h</b> (left panels), but not to GS-441524 (right panels). The graphs show the effect of the compounds on virus yield. (A) HEL cells infected with <b>5h</b>-resistant mutants obtained by virus passaging under <b>5h</b> and carrying substitution K60R (first selection) or T66I (second selection) in nsp15. (B) 16HBE cells infected with EndoU-deficient mutant virus (H250A<sub>nsp15</sub>), obtained by reverse genetics.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Data points are the mean ± SEM (<i>N</i> = 3). An unpaired <i>t</i>-test (GraphPad Prism 8.4.3) was used to compare the mutant viruses to WT, and the resulting two-tailed <i>P</i>-values were adjusted for multiple comparisons using Holm–Šídák (α = 0.05). *, <i>P</i> < 0.05; **, <i>P</i> < 0.01; ***, <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Binding Model of <b>5h</b> in the Hexameric nsp15 Protein</h3><div class="NLM_p">To predict the possible binding site of <b>5h</b>, the compound was docked into the X-ray structure of the hexameric nsp15 protein from HCoV-229E (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4">4RS4</a>). This hexameric structure formed by two trimers is the functional form of nsp15.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> First, a few sites in this PDB sequence were changed to obtain the fully correct nsp15 sequence of HCoV-229E virus (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details). By using a pocket-detection protocol implemented on the Site Finder module of Molecular Operating Environment (MOE) software, we identified a druggable binding pocket at the nsp15 dimer interface, surrounded by the catalytic residue His250 in the EndoU active site of one nsp15 monomer and residues Lys60 and Thr66 in the N-terminal domain of the other monomer (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Next, ligand <b>5h</b> was placed inside the pocket and docked by using both MOE and GOLD software, and the common top scoring binding mode was further analyzed. This docked result indicates that the carboxylic acid moiety of <b>5h</b> forms hydrogen bonds with the backbone of residues Cys294 and Thr295 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). The importance of this interaction is supported by the observation that <b>5u</b>, the <b>5h</b> analogue bearing a methyl instead of carboxylic acid group, lacks antiviral activity. Furthermore, the 1,2,3-triazole group of <b>5h</b> engages in hydrogen-bonding interactions with Thr245, explaining why the parent compounds <b>1</b> and <b>2</b> are not active against HCoV-229E. At the other side of the pocket, the aromatic ring of <b>5h</b> makes hydrophobic contacts with Val63 and Leu65. This may explain why nearby mutations K60R and T66I yield resistance to the compound since these substitutions may negatively affect the interactions of Val63 and Leu65 or disturb the conformation of the loop flanked by both residues. The structure of the substituent attached to the 1,2,3-triazole ring seems less determining since, in terms of antiviral activity, both aromatic and smaller aliphatic substituents are tolerated (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The one exception is the negative influence from a bulky cyclohexanemethyl group (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compound <b>5i</b>). Hence, the shape requirements for this substituent appear not very strict and compatible with voluminous changes that are sterically not clashing. This analysis concurs with the shallow and surface-exposed nature of the binding pocket.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode of <b>5h</b> in the HCoV-229E nsp15 hexameric protein (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4">4RS4</a>), as predicted by docking. (A) Hydrophobic pocket lies adjacent to the EndoU catalytic center (catalytic triad consisting of His235, His250, and Lys291) and at an nsp15 dimer interface (monomers depicted in a differently colored surface). The pocket is surrounded by His250, Lys60, and Thr66, explaining why <b>5h</b> is inactive against HCoV-229E viruses carrying mutations at these sites. (B) <b>5h</b> occupies the pocket by making hydrophobic interactions with Val293 and side chain fragments of Lys291 and Thr292. The molecule further engages in hydrogen-bonding interactions with Cys294 and Thr295 via the carboxylic acid moiety and with Thr245 via the 1,2,3-triazole moiety. Additional hydrophobic interactions with Val63, Leu65, and Thr292 are made via the aromatic ring-substituted 1,2,3-triazole moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Analysis of the nsp15 sequence similarity between HCoV-229E and SARS-CoV-2 showed that a few residues in the pocket are not conserved (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_001.pdf" class="ext-link">Figure S1</a> in the Supporting Information). This explains why docking <b>5h</b> into the SARS-CoV-2 nsp15 hexamer was unable to identify a similar pose within the top solutions. The largest influence seems attributed to the residue at position 244/245 since substituting Thr245 (present in HCoV-229E) by Gln244 (the corresponding residue in SARS-CoV-2) abrogates a hydrogen-bond interaction with the 1,2,3-triazole group of <b>5h</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Additionally, the loop between Val/Ile63 and Leu/Pro65 has a slightly different orientation in these two CoV nsp15 proteins. Both factors may explain why <b>5h</b> is active against HCoV-229E, but not SARS-CoV-2. Hence, one strategy to improve the binding to SARS-CoV-2 nsp15 might be to substitute the 1,2,3-triazole moiety by another type of ring structure. At the other side of the molecule, the carboxylic acid moiety probably needs to be kept since it forms hydrogen bonds with both nsp15 proteins. The fact that most of the pocket residues are conserved (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_001.pdf" class="ext-link">Figure S1</a> in the Supporting Information) underscores the relevance of this nsp15 interface pocket for drug design. The role of this protein region in forming inter-monomer interactions is evident from reports that nsp15 exists as a monomer when key interactions in this region (i.e., Arg61-Glu266 in SARS-CoV nsp15 and Tyr58-Glu263 in MERS-CoV nsp15) are eliminated by mutation.<a onclick="showRef(event, 'ref22 ref51'); return false;" href="javascript:void(0);" class="ref ref22 ref51">(22,51)</a> When nsp15 is unable to hexamerize, the EndoU catalytic site undergoes important structural changes that abolish RNA binding and enzymatic activity.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the hydrophobic pocket, occupied by <b>5h</b>, in the nsp15 proteins of HCoV-229E (left; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4">4RS4</a>) and SARS-CoV-2 (right; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7K1O">7K1O</a>). The carboxylic acid moiety of <b>5h</b> forms hydrogen bonds with both nsp15 binding pockets. On the other hand, the 1,2,3-triazole group engages hydrogen-bonding interactions with the HCoV-229E nsp15 protein but is incompatible with the SARS-CoV-2 pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To conclude, the nsp15 binding mode of <b>5h</b>, predicted by docking, nicely accords with the biological findings. Namely, the binding model rationalizes the requirement of the 1,2,3-triazolo function and carboxylate substituent; <b>5h</b> resistance of the nsp15-K60R, nsp15-T66I, and nsp15-H250A mutant viruses; and lack of activity against SARS-CoV-2.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">To conclude, we present the first prototype of CoV nsp15 inhibitors, having a 1,2,3-triazolo-fused betulonic acid structure. The SAR analysis, resistance data, and docking model provide strong evidence that the lead molecule <b>5h</b> binds to an inter-monomer nsp15 interface lying adjacent to the EndoU catalytic core. This provides an excellent basis to modify the substituents or betulonic acid scaffold and expand the activity spectrum beyond HCoV-229E. Since <b>5h</b> appears to interact with the catalytic His250 residue in the EndoU domain of nsp15, the molecule plausibly interferes with the role of EndoU in regulating viral dsRNA synthesis.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> To complement the findings in this report, obtained in non-immune cells, we are currently evaluating the antiviral and immunomodulatory effects of <b>5h</b> in HCoV-229E-infected human macrophages. This may validate the intriguing concept that nsp15 inhibition could have a dual outcome, by inhibiting CoV replication and promoting host cell antiviral immunity. Since nsp15 is also considered to be an important immune evasion factor for SARS-CoV-2, further exploration of our betulonic acid skeleton and its nsp15 binding pocket is clearly warranted.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85305" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85305" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Chemistry</h3><div class="NLM_p"><sup>1</sup>H and <sup>13</sup>C NMR spectra were measured on commercial instruments (Bruker Avance 300 MHz and Bruker AMX 400 MHz and 600 MHz). Chemical shifts (δ) are reported in parts per million (ppm) referenced to tetramethylsilane (<sup>1</sup>H) or the internal (NMR) solvent signal (<sup>13</sup>C). Melting points were determined using a Reichert Thermovar apparatus. For column chromatography, 70–230 mesh silica 60 (E. M. Merck) was used as the stationary phase. Chemicals received from commercial sources were used without further purification. Reaction dry solvents (toluene, DMF, and THF) were used as received from commercial sources. TLC was carried out on Kieselgel 60 F254 plates (Merck).</div><div class="NLM_p">Exact mass was acquired on a quadrupole orthogonal acceleration time-of-flight mass spectrometer (Synapt G2 HDMS, Waters, Milford, MA). Samples were infused at 3 μL/min, and spectra were obtained in positive (or: negative) ionization mode with a resolution of 15,000 (FWHM) using leucine enkephalin as the lock mass.</div><div class="NLM_p">All HPLC data were acquired on an Agilent 1200 HPLC with a quaternary pump, autosampler, and UV<i>-</i>DAD detector (set at 220, 230, and 240 nm). The injection volume was 5 μL of a dilution of 100 μg/mL (sample in mobile phase). The column was a Zorbax XBD C18 column (5 μm; 4.6 mm x 150 mm), and the mobile phases were as follows: (A) 0.1% formic acid in water, and (B) methanol. The gradient consisted of: (1) 5 min from 50% A/50% B toward 25% A/75% D; (2) 15 min toward 0% A/100% B; (3) 20 min 0% A/100% B. Data collection and analysis were done with Agilent Chemstation software. All tested compounds showed a purity of >95%.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> 3-Oxo-lup-20(29)-en-28-oic Acid (Betulonic Acid, <b>2</b>)</h4><div class="NLM_p last">To a solution of betulin (50.0 g, 113 mmol; purchased from Eburon Organics BVBA) in acetone (1500 mL, use an ultra-sonic bath to dissolve) was added freshly prepared Jones reagent [Na<sub>2</sub>Cr<sub>2</sub>O<sub>7</sub>, (66.5 g, 226 mmol) and H<sub>2</sub>SO<sub>4</sub> (60 mL) in water (500 mL)] for 1 h in an ice bath. The reaction mixture was allowed to warm to room temperature, and stirring was continued for 6 h followed by TLC. First, MeOH was added and then water to the reaction mixture. The precipitate was filtered off and washed with water (500 mL). The crude product was dried in a vacuum oven, dissolved in Et<sub>2</sub>O (600 mL), and washed with water (300 mL), 7.5% hydrochloric acid (200 mL), water (200 mL), saturated aqueous NaHCO<sub>3</sub> solution (200 mL), and water (200 mL). The crude reaction mixture was purified by column chromatography (silica gel), whereas the eluent was used a mixture of heptane and ethyl acetate (70:30) to afford betulonic acid (23 g, 45% yield). Spectroscopic data for betulonic acid was consistent with previously reported data for this compound.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 3-Oxo-lupan-28-oic Acid (Dihydrobetulonic Acid)</h4><div class="NLM_p last">Betulonic acid (180 mg, 0.396 mmol, 1.0 equiv) was dissolved in a mixture of MeOH/THF (2/6 mL). 10% Pd(OH)<sub>2</sub> (30 mg) was added under a N<sub>2</sub> atmosphere. This atmosphere was replaced by a H<sub>2</sub> atmosphere. The reaction was stirred under a H<sub>2</sub> atmosphere for 78 h, then filtered through celite, and washed with CHCl<sub>3</sub> to afford a white solid. The residue obtained was purified by silica gel column chromatography using 100% chloroform as an eluent to afford dihydrobetulonic acid (quantitative yield) as a white amorphous powder. The spectra proved the identity of the compound by comparing the data with the literature.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Procedure</h3><div class="NLM_p">To a dried screw-capped reaction tube equipped with a magnetic stirring bar was added betulonic acid, amine, 4-nitrophenyl azide, and 4 Å molecular sieves (50 mg). The mixture was dissolved in the proper solvent (toluene and DMF) and stirred at 100 °C for 12–72 h. The reaction was monitored using TLC with the plate first developed with DCM, and then different ratios petroleum ether/ethyl acetate (7:3 and 6:4) were used depending on the substrates. For visualization of TLC plates, 5% H<sub>2</sub>SO<sub>4</sub> in ethanol was used, and for more sensitive detection, cerium-ammonium-molybdate after heating to 150–200 °C was used. The crude reaction mixture was then directly purified by column chromatography (silica gel), first with CH<sub>2</sub>Cl<sub>2</sub> as an eluent to remove all 4-nitroaniline formed during the reaction followed by using a mixture of petroleum ether and ethyl acetate as an eluent to afford the betulonic acid 1,2,3-triazole derivatives.</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 1′-(3,4,5-Trimethoxybenzyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5a</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 3,4,5-trimethoxybenzylamine (56 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5a</b> (122 mg, 84% yield) as off-white crystals. m.p. 152 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.25 (s, 2H), 5.57 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.81 (s, 3H), 3.77 (s, 6H), 3.04 (m, 1H), 2.96 (d, <i>J</i> = 15.4 Hz, 1H), 2.32–2.15 (m, 3H), 2.05–1.94 (m, 2H), 1.77–1.64 (m, 5H), 1.59–1.40 (m, 11H), 1.20 (s, 7H), 1.03 (s, 3H), 1.00 (s, 3H), 0.97 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 180.9, 153.5, 150.2, 141.9, 138.0, 137.4, 132.1, 109.8, 103.5, 60.8, 56.3, 56.1, 54.6, 52.8, 49.3, 49.1, 46.8, 42.4, 40.5, 38.9, 38.5, 37.0, 33.7, 33.3, 32.0, 30.5, 29.8, 29.7, 28.7, 25.5, 21.3, 21.3, 19.4, 18.9, 16.0, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>40</sub>H<sub>57</sub>N<sub>3</sub>O<sub>5</sub>H [M + H]<sup>+</sup>: 660.4370, found 660.4384.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 1′-(3,5-Dimethoxybenzyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5b</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 3,5-dimethoxybenzylamine (48 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5b</b> (126 mg, 92% yield) as off-white crystals. m.p. 159 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.34 (s, 1H), 6.16 (s, 2H), 5.57 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.71 (s, 6H), 3.08–2.99 (m, 1H), 2.95 (d, <i>J</i> = 15.4 Hz, 1H), 2.32–2.14 (m, 3H), 2.05–1.94 (m, 2H), 1.79–1.62 (m, 5H), 1.60–1.33 (m, 11H), 1.29–1.10 (m, 7H), 1.03 (s, 3H), 1.00 (s, 3H), 0.98 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.6, 161.1, 150.2, 141.8, 138.9, 138.0, 109.8, 104.4, 99.6, 56.4, 55.3, 54.5, 52.8, 49.2, 49.2, 46.9, 42.4, 40.5, 38.9, 38.5, 38.3, 37.0, 33.7, 33.3, 32.0, 30.6, 29.8, 28.7, 25.5, 21.3, 21.2, 19.4, 18.9, 16.0, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>39</sub>H<sub>55</sub>N<sub>3</sub>O<sub>4</sub>H [M + H]<sup>+</sup>: 630.4265, found 630.4274.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 1′-(Pyridin-4-ylmethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5c</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 4-(aminomethyl)pyridine (31 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5c</b> (97 mg, 78% yield) as off-white crystals. m.p 172 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.56 (d, <i>J</i> = 5.6 Hz, 2H), 6.93 (d, <i>J</i> = 5.6 Hz, 2H), 5.66 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.06 (m, 1H), 2.97 (d, <i>J</i> = 15.4 Hz, 1H), 2.36–2.15 (m, 3H), 2.06–1.95 (m, 2H), 1.83–1.64 (m, 5H), 1.60–1.37 (m, 11H), 1.32–1.21 (m, 7H), 1.16 (s, 3H), 1.01 (m, 3H), 1.00 (m, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 180.6, 150.4, 149.8, 146.1, 142.3, 138.3, 121.2, 109.7, 56.3, 54.4, 51.6, 49.2, 49.2, 46.9, 42.4, 40.5, 39.0, 38.4, 38.2, 37.0, 33.6, 33.2, 32.1, 30.6, 29.8, 29.6, 28.8, 25.4, 21.5, 21.4, 19.4, 18.8, 16.0, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>36</sub>H<sub>50</sub>N<sub>4</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 571.4006, found 571.4013.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 1′-(4-Methylbenzyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5d</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 4-methylbenzylamine (35 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5d</b> (115 mg, 90% yield) as off-white crystals. m.p. 310 °C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.09 (d, <i>J</i> = 7.9 Hz, 2H), 6.92 (d, <i>J</i> = 7.9 Hz, 2H), 5.59 (s, br, 2H), 4.75 (s, 1H), 4.63 (s, 1H), 3.03 (m, 1H), 2.95 (d, <i>J</i> = 15.4 Hz, 1H), 2.32–2.23 (m, 5H), 2.17 (d, <i>J</i> = 15.4 Hz, 1H), 2.05–1.94 (m, 2H), 1.80–1.62 (m, 5H), 1.50 (m 11H), 1.22–1.09 (m, 7H), 1.03 (s, 3H), 0.99 (s, 3H), 0.97 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.8, 150.2, 141.7, 138.0, 137.4, 133.3, 129.3, 126.3, 109.8, 56.4, 54.5, 52.6, 49.2, 49.1, 46.8, 42.4, 40.8, 40.5, 38.9, 38.4, 38.2, 37.0, 33.7, 33.3, 32.0, 30.5, 29.7, 28.7, 28.4, 25.4, 23.8, 21.3, 21.3, 21.0, 20.8, 19.4, 18.9, 17.5, 17.2, 16.0, 15.6, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>38</sub>H<sub>53</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 584.4210, found 584.4217.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 1′-(4-Fluorobenzyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5e</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 4-fluorobenzylamine (36 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5e</b> (110 mg, 85% yield) as off-white crystals. m.p. 309 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.00 (m, 4H), 5.60 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.08–2.99 (m, 1H), 2.95 (d, <i>J</i> = 15.3 Hz, 1H), 2.33–2.14 (m, 3H), 2.06–1.93 (m, 2H), 1.82–1.62 (m, 5H), 1.61–1.31 (m, 11H), 1.30–1.10 (m, 7H), 1.08 (s, 3H), 1.02 (s, 3H), 1.00 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.1, 163.5, 161.0, 150.2, 142.0, 137.9, 132.2, 132.2, 128.2, 128.1, 115.8, 115.5, 109.8, 56.3, 54.5, 52.1, 49.2, 49.2, 46.9, 42.4, 40.5, 38.9, 38.5, 38.3, 37.0, 33.7, 33.3, 32.0, 30.5, 29.8, 28.7, 25.4, 21.3, 21.3, 19.4, 18.8, 16.0, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>37</sub>H<sub>50</sub>FN<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 588.3959, found 588.3969.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 1′-(4-Trifluoromethylbenzyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5f</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 4-(trifluoromethyl)benzylamine (50 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5f</b> (112 mg, 80% yield) as off-white crystals. m.p. 315 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.56 (d, <i>J</i> = 8.0 Hz, 2H), 7.11 (d, <i>J</i> = 8.0 Hz, 2H), 5.69 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.02 (m, 2H), 2.34–2.16 (m, 3H), 2.07–1.94 (m, 2H), 1.82–1.64 (m, 5H), 1.59–1.38 (m, 11H), 1.18 (m, 7H), 1.02 (s, 3H), 1.00 (s, 3H), 0.97 (s, 3H), 0.78 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.4, 150.2, 142.1, 140.5, 138.2, 126.6, 125.7, 125.7, 109.8, 56.4, 54.5, 52.2, 49.3, 49.1, 46.9, 42.4, 40.8, 40.5, 38.9, 38.5, 38.2, 37.0, 33.8, 33.6, 33.3, 32.0, 30.5, 29.8, 29.7, 28.8, 28.4, 25.4, 23.8, 21.4, 21.3, 20.8, 20.5, 19.4, 18.8, 17.5, 17.3, 16.1, 15.7, 14.6, 7.9. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>38</sub>H<sub>50</sub>F<sub>3</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 638.3927, found 638.3939.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 1′-(4-Dimethylaminobenzyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5g</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 4-(dimethylamino)benzylamine (43 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5g</b> (112 mg, 80% yield) as off-white crystals. m.p. 190 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.97 (d, <i>J</i> = 8.3 Hz, 2H), 6.64 (d, <i>J</i> = 8.3 Hz, 2H), 5.52 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.10–2.84 (m, 9H), 2.33–2.11 (m, 3H), 1.99 (m, 2H), 1.81–1.62 (m, 5H), 1.56–1.34 (m, 11H), 1.25–1.15 (m, 7H), 1.06 (s, 3H), 0.99 (s, 3H), 0.97 (s, 3H), 0.76 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.4, 150.2, 150.1, 141.6, 137.7, 127.6, 123.9, 112.5, 109.8, 56.4, 54.6, 52.6, 49.2, 49.1, 46.8, 42.4, 40.5, 40.5, 38.9, 38.9, 38.4, 38.3, 37.0, 33.7, 33.3, 32.1, 30.5, 29.8, 28.7, 25.5, 23.8, 21.3, 21.3, 20.8, 19.4, 18.9, 16.0, 15.6, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>39</sub>H<sub>56</sub>N<sub>4</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 613.4475, found 613.4480.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 1′-((<i>S</i>)-1-Phenylethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5h</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), (<i>S</i>)-(−)-α-methylbenzylamine (35 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5h</b> (107 mg, 84% yield) as off-white crystals. m.p. 327 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.30–7.21 (m, 3H), 7.13 (d, <i>J</i> = 7.2 Hz, 2H), 5.73 (m, 1H), 4.76 (s, 1H), 4.64 (s, 1H), 3.00 (m, 2H), 2.33–2.12 (m, 3H), 2.07–1.92 (m, 5H), 1.81–1.62 (m, 5H), 1.58–1.37 (m, 11H), 1.14 (m, 7H), 1.00 (s, 6H), 0.96 (s, 3H), 0.72 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 180.6, 150.2, 141.8, 141.1, 137.6, 128.6, 127.5, 126.1, 109.8, 59.1, 56.3, 54.8, 49.3, 49.1, 46.8, 42.4, 40.5, 38.8, 38.4, 37.0, 33.6, 33.3, 32.0, 30.5, 29.7, 29.7, 28.6, 25.4, 23.3, 21.4, 21.3, 19.4, 19.0, 15.9, 15.6, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>38</sub>H<sub>53</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 584.4210, found 584.4218.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1′-(Cyclohexylmethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5i</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), cyclohexanemethylamine (33 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5i</b> (103 mg, 82% yield) as off-white crystals. m.p. 338 °C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 4.76 (s, 1H), 4.64 (s, 1H), 4.09 (m, 2H), 3.04 (m, 1H), 2.92 (m, 1H), 2.33–2.21 (m, 2H), 2.13 (d, <i>J</i> = 15.3 Hz, 1H), 2.07–1.95 (m, 2H), 1.79–1.64 (m, 10H), 1.61–1.34 (m, 11H), 1.32–1.19 (m, 10H), 1.18–1.08 (m, 5H), 1.00 (s, 3H), 0.99 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.7, 150.2, 140.9, 137.8, 109.8, 56.4, 55.6, 54.7, 49.2, 49.2, 46.9, 42.4, 40.5, 38.8, 38.6, 38.5, 38.2, 37.0, 33.7, 33.3, 32.1, 31.0, 30.6, 29.8, 28.9, 26.3, 25.7, 25.4, 21.5, 21.3, 19.4, 18.9, 16.0, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>37</sub>H<sub>57</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 576.4523, found 576.4529.</div></div><div id="sec4_4_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 1′-(Benzo[<i>d</i>][1,3]dioxol-5-ylmethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5j</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), piperonylamine (43 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5j</b> (105 mg, 78% yield) as off-white crystals. m.p. 313 °C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 6.72 (d, <i>J</i> = 7.9 Hz, 1H), 6.59–6.48 (m, 2H), 5.93 (d, <i>J</i> = 0.9 Hz, 2H), 5.53 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.10–2.89 (m, 2H), 2.35–2.11 (m, 3H), 1.99 (m, 2H), 1.83–1.62 (m, 5H), 1.61–1.33 (m, 11H), 1.33–1.08 (m, 7H), 1.05 (s, 3H), 1.00 (s, 3H), 0.97 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.3, 150.2, 148.1, 147.2, 141.8, 137.9, 130.1, 119.9, 109.8, 108.3, 107.2, 101.1, 56.4, 54.5, 52.6, 50.8, 49.2, 49.2, 46.9, 42.4, 40.5, 38.9, 38.5, 38.3, 37.0, 33.7, 33.3, 32.0, 30.5, 29.8, 28.7, 25.4, 21.3, 21.3, 19.4, 18.9, 16.0, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>38</sub>H<sub>51</sub>N<sub>3</sub>O<sub>4</sub>H [M + H]<sup>+</sup>: 614.3952, found 614.3951.</div></div><div id="sec4_4_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 1′-(Furan-2-ylmethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5k</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), furfurylamine (28 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5k</b> (66 mg, 53% yield) as a brown crystals. m.p. 227 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.35 (s, 1H), 6.34–6.31 (m, 1H), 6.23 (d, <i>J</i> = 3.1 Hz, 1H), 5.55 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.07–2.99 (m, 1H), 2.93 (d, <i>J</i> = 15.3 Hz, 1H), 2.30–2.11 (m, 3H), 1.99 (d, <i>J</i> = 7.5 Hz, 2H), 1.78–1.66 (m, 5H), 1.60–1.40 (m, 11H), 1.28 (t, <i>J</i> = 10.2 Hz, 7H), 1.15 (s, 3H), 1.00 (s, 3H), 0.98 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 179.9, 150.2, 148.8, 142.5, 141.6, 137.7, 110.7, 109.9, 109.0, 56.3, 54.6, 49.3, 49.2, 46.8, 46.4, 42.4, 40.6, 39.0, 38.4, 38.3, 37.0, 33.6, 33.3, 32.0, 30.5, 29.8, 29.7, 28.6, 19.4, 18.9, 16.1, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>35</sub>H<sub>49</sub>N<sub>3</sub>O<sub>3</sub>H [M + H]<sup>+</sup>: 560.3846, found 560.3857.</div></div><div id="sec4_4_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 1′-((1<i>H</i>-Indol-3-yl)methyl)-1<i>H</i>′-Lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5l</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), tryptamine (46 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5l</b> (115 mg, 84% yield) as off-white crystals. m.p. 196 °C. <sup>1</sup>H NMR (400 MHz, DMSO) δ 10.87 (s, 1H), 7.41 (d, <i>J</i> = 7.8 Hz, 1H), 7.33 (d, <i>J</i> = 8.1 Hz, 1H), 7.10–7.02 (m, 2H), 6.96 (m, 1H), 4.72 (s, 1H), 4.72 (s, 1H), 4.59 (s, 1H), 4.54 (s, 2H), 3.17 (s, 1H), 3.03–2.92 (m, 1H), 2.70 (d, <i>J</i> = 15.2 Hz, 1H), 2.29 (m, 1H), 2.14–2.06 (m, 2H), 1.81 (d, <i>J</i> = 6.9 Hz, 2H), 1.63 (d, <i>J</i> = 29.9 Hz, 5H), 1.57–1.24 (m, 11H), 1.25–1.01 (m, 7H), 0.96 (s, 3H), 0.93 (s, 3H), 0.90 (s, 3H), 0.63 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO) δ 177.7, 150.7, 140.3, 137.8, 136.5, 127.4, 123.7, 121.4, 118.9, 118.3, 111.8, 110.7, 110.1, 55.9, 54.4, 50.1, 49.0, 48.9, 47.0, 42.5, 38.8, 38.1, 36.7, 33.5, 33.3, 32.0, 31.1, 30.5, 29.7, 28.6, 26.9, 25.5, 21.4, 21.1, 19.4, 18.8, 16.1, 15.7, 14.8. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>40</sub>H<sub>54</sub>N<sub>4</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 623.4319, found 623.4317.</div></div><div id="sec4_4_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 1′-(2-(1<i>H</i>-Indol-3-yl)ethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5m</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 5-methoxytryptamine (54 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5m</b> (126 mg, 88% yield) as off-white crystals. m.p. 240 °C. <sup>1</sup>H NMR (400 MHz, DMSO) δ 10.69 (s, 1H), 7.19 (d, <i>J</i> = 8.7 Hz, 1H), 7.05 (s, 1H), 6.76 (s, 1H), 6.67 (d, <i>J</i> = 8.7 Hz, 1H), 4.72 (s, 1H), 4.59 (s, 1H), 4.57–4.45 (m, 2H), 3.71 (s, 2H), 3.03–2.92 (m, 1H), 2.69 (d, <i>J</i> = 15.3 Hz, 1H), 2.28 (m, 1H), 2.09 (m, 2H), 1.81 (d, <i>J</i> = 6.9 Hz, 2H), 1.73–1.55 (m, 5H), 1.32 (m, 11H), 1.19–1.02 (m, 7H), 0.96 (s, 3H), 0.89 (s, 3H), 0.86 (s, 3H), 0.58 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO) δ 177.7, 153.5, 150.7, 140.3, 137.9, 131.5, 127.8, 124.3, 112.4, 111.7, 110.7, 110.1, 99.9, 55.9, 55.5, 54.4, 50.2, 48.9, 47.0, 42.5, 38.8, 38.1, 36.7, 33.5, 33.3, 32.0, 30.5, 29.7, 28.5, 26.9, 25.5, 22.5, 21.4, 20.9, 19.4, 18.8, 16.0, 15.7, 14.8. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>41</sub>H<sub>56</sub>N<sub>4</sub>O<sub>3</sub>H [M + H]<sup>+</sup>: 653.4424, found 653.4418.</div></div><div id="sec4_4_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 1′-((1<i>H</i>-Indol-4-yl)methyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5n</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 4-(aminomethyl) indole (42 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5n</b> (98 mg, 73% yield) as off white crystals. m.p. 260 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.32 (s, 1H), 7.31 (d, <i>J</i> = 8.2 Hz, 1H), 7.22 (m, 1H), 7.07 (m, 1H), 6.52–6.40 (m, 2H), 5.93 (m, 2H), 4.76 (s, 1H), 4.65 (s, 1H), 3.07–2.93 (m, 2H), 2.31–2.16 (m, 3H), 2.05–1.93 (m, 2H), 1.77–1.64 (m, 5H), 1.59–1.35 (m, 11H), 1.34–1.15 (m, 7H), 1.12 (d, <i>J</i> = 14.6 Hz, 3H), 0.99 (s, 3H), 0.96 (s, 3H), 0.78 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 180.4, 150.2, 141.9, 138.3, 135.7, 128.2, 125.2, 124.5, 122.1, 117.5, 110.6, 109.8, 100.0, 56.3, 54.6, 51.4, 49.2, 49.2, 46.9, 42.4, 40.5, 38.9, 37.0, 33.8, 33.3, 32.0, 30.5, 29.7, 29.7, 28.4, 25.5, 22.7, 21.3, 20.9, 19.4, 18.9, 16.0, 15.6, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>39</sub>H<sub>52</sub>N<sub>4</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 609.4162, found 609.4174.</div></div><div id="sec4_4_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 1′-(2-Hydroxyethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5o</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), ethanolamine (17 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by CH<sub>2</sub>Cl<sub>2</sub>:MeOH 95:5) affording <b>5o</b> (70 mg, 62% yield) as off-white crystals. m.p. 236 °C. <sup>1</sup>H NMR (400 MHz, DMSO) δ 4.72 (s, 1H), 4.59 (s, 1H), 4.36 (m, 2H), 3.86 (m, 2H), 2.98 (m, 1H), 2.70 (d, <i>J</i> = 15.2 Hz, 1H), 2.30 (m, 1H), 2.11 (m, 2H), 1.82 (d, <i>J</i> = 6.9 Hz, 2H), 1.64 (s,br, 5H), 1.58–1.33 (m, 11H), 1.33–1.22 (m, 7H), 1.17 (s, 3H), 0.98 (s, 3H), 0.93 (s, 3H), 0.71 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO) δ 177.7, 150.7, 140.2, 138.0, 110.1, 79.6, 60.7, 55.9, 54.5, 51.4, 48.9, 47.0, 42.5, 38.9, 38.1, 36.7, 33.6, 33.3, 32.0, 30.5, 29.8, 28.7, 25.5, 21.4, 19.4, 18.8, 16.2, 15.8, 14.8. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>32</sub>H<sub>49</sub>N<sub>3</sub>O<sub>3</sub>H [M + H]<sup>+</sup>: 524.3846, found 524.3853.</div></div><div id="sec4_4_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 1′-Heptyl-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5p</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), 1-heptylamine (33 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5p</b> (72 mg, 57% yield) as off-white crystals. m.p. 273 °C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 4.76 (s, 1H), 4.64 (s, 1H), 4.33–4.23 (m, 2H), 3.04 (m, 1H), 2.91 (d, <i>J</i> = 15.3 Hz, 1H), 2.33–2.22 (m, 2H), 2.13 (d, <i>J</i> = 15.3 Hz, 1H), 2.00–1.96 (m, 2H), 1.79–1.64 (m, 5H), 1.61–1.35 (m, 13H), 1.35–1.19 (m, 14H), 1.17 (s, 3H), 1.00 (s, 3H), 0.99 (s, 3H), 0.88 (m, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.7, 150.2, 141.0, 137.3, 109.8, 56.4, 54.6, 49.6, 49.2, 49.2, 46.9, 42.4, 40.5, 38.9, 38.5, 38.2, 37.0, 33.6, 33.3, 32.1, 31.6, 30.8, 30.6, 29.8, 29.7, 28.8, 28.6, 26.9, 25.4, 22.5, 21.3, 19.4, 18.9, 16.0, 15.7, 14.6, 14.0. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>37</sub>H<sub>59</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 578.4679, found 578.4687.</div></div><div id="sec4_4_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 1<i>H</i>′-Lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5q</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), ammonium acetate (85 mg, 5 equiv, 1.100 mmol), 4-nitrophenyl azide (51 mg, 1.4 equiv, 0.308 mmol), 80 °C, 4 Å molecular sieves (50 mg), and DMF (0.8 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by CH<sub>2</sub>Cl<sub>2</sub>:MeOH 95:5) affording <b>5q</b> (90 mg, 86% yield) as off-white crystals. m.p. 158 °C. Spectroscopic data for compound <b>5q</b> was consistent with previously reported data for this compound.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.77 (s, 1H), 4.64 (s, 1H), 3.03 (d, <i>J</i> = 10.1 Hz, 1H), 2.90 (d, <i>J</i> = 15.5 Hz, 1H), 2.37–2.22 (m, 2H), 2.12 (d, <i>J</i> = 15.5 Hz, 1H), 2.00 (dd, <i>J</i> = 19.9, 11.7 Hz, 2H), 1.83–1.66 (m, 5H), 1.57 (dd, <i>J</i> = 38.5, 24.6 Hz, 12H), 1.36–1.24 (m, 7H), 1.21 (d, <i>J</i> = 7.0 Hz, 4H), 1.01 (s, 3H), 1.00 (s, 3H) 0.78 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.0, 150.3, 150.1, 140.5, 109.8, 56.3, 53.4, 49.2, 49.0, 46.9, 42.5, 40.7, 39.0, 38.4, 37.3, 37.0, 33.3, 33.3, 32.1, 31.0, 30.6, 29.8, 25.5, 23.7, 21.4, 19.4, 19.1, 16.2, 15.6, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>30</sub>H<sub>45</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 480.3584, found 480.3585.</div></div><div id="sec4_4_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 1′-Benzyl-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5r</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), benzylamine (31 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5r</b> (115 mg, 92% yield) as off-white crystals. m.p. 290 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33–7.24 (m, 3H), 7.02 (d, <i>J</i> = 7.2 Hz, 2H), 5.64 (s, 2H), 4.76 (s, 1H), 4.64 (s, 1H), 3.09–3.00 (m, 1H), 2.96 (d, <i>J</i> = 15.4 Hz, 1H), 2.33–2.13 (m, 3H), 2.06–1.93 (m, 2H), 1.82–1.63 (m, 5H), 1.61–1.37 (m, 11H), 1.27–1.15 (m, 7H), 1.04 (s, 3H), 0.99 (s, 3H), 0.97 (s, 3H) , 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.5, 150.2, 141.8, 138.0, 136.4, 128.7, 127.7, 126.3, 109.8, 56.4, 54.5, 52.8, 49.2, 49.1, 46.9, 42.4, 40.5, 38.9, 38.5, 38.3, 37.0, 33.7, 33.3, 32.0, 30.5, 29.7, 28.7, 25.4, 23.8, 21.3, 21.3, 19.4, 18.9, 16.0, 15.7, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>37</sub>H<sub>51</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 570.4053, found 570.4064.</div></div><div id="sec4_4_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 1′-((<i>R</i>)-1-Phenylethyl)-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5s</b>)</h4><div class="NLM_p last">Betulonic acid (100 mg, 1 equiv, 0.220 mmol), (<i>R</i>)-(+)-α-methylbenzylamine (35 mg, 1.3 equiv, 0.286 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.220 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5s</b> (88 mg, 69% yield) as off white crystals. m.p. 327 °C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.30–7.20 (m, 3H), 7.15–7.10 (m, 2H), 5.73 (m, 1H), 4.75 (s, br, 1H), 4.64 (s,br, 1H), 3.02 (m, 1H), 2.95 (d, <i>J</i> = 15.4 Hz, 1H), 2.32–2.22 (m, 2H), 2.17 (d, <i>J</i> = 15.4 Hz, 1H), 2.05–1.95 (m, 5H), 1.80–1.64 (m, 5H), 1.64–1.39 (m, 11H), 1.38–1.25 (m, 7H), 1.00 (s, 3H), 0.99 (s, 3H), 0.96 (s, 3H) 0.72 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.5, 150.2, 141.7, 141.1, 137.6, 128.6, 127.5, 126.1, 109.8, 59.1, 56.4, 54.8, 49.3, 49.1, 46.8, 42.4, 40.5, 38.8, 38.4, 38.3, 37.0, 33.6, 33.3, 32.0, 30.5, 29.7, 28.6, 25.4, 23.3, 21.4, 21.3, 19.4, 19.0, 15.9, 15.6, 14.6. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>38</sub>H<sub>53</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 584.4210, found 584.4214.</div></div><div id="sec4_4_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 1′-((<i>S</i>)-1-Phenylethyl)-1<i>H</i>′-lupano-[2,3-<i>d</i>]-[1,2,3]-triazole-28-oic Acid (<b>5t</b>)</h4><div class="NLM_p last">Dihydrobetulonic acid (100 mg, 1 equiv, 0.219 mmol), (<i>S</i>)-(−)-α-methylbenzylamine (34 mg, 1.3 equiv, 0.285 mmol), 4-nitrophenyl azide (36 mg, 1 equiv, 0.219 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5t</b> (115 mg, 90% yield) as off-white crystals. m.p. 173 °C. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33–7.27 (m, 2H), 7.22 (m, 3H), 5.73 (m, 1H), 2.98 (d, <i>J</i> = 15.2 Hz, 1H), 2.32–2.22 (m, 3H), 2.17 (d, <i>J</i> = 15.3 Hz, 1H), 2.03 (m, 3H), 1.96–1.63 (m, 5H), 1.45 (m, 11H), 1.29–1.22 (m, 7H), 1.10 (s, 3H), 0.97 (m, 6H), 0.87 (s, 3H), 0.82 (s, 3H), 0.77 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 181.5, 141.7, 141.0, 137.5, 128.6, 127.5, 126.2, 59.2, 56.8, 54.7, 49.1, 48.7, 44.1, 42.6, 40.6, 38.8, 38.3, 38.3, 37.4, 33.8, 33.4, 31.9, 29.8, 29.7, 28.8, 26.8, 23.7, 23.0, 22.7, 21.3, 21.3, 18.9, 16.2, 15.7, 14.7, 14.5. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>38</sub>H<sub>55</sub>N<sub>3</sub>O<sub>2</sub>H [M + H]<sup>+</sup>: 586.4366, found 586.4370.</div></div><div id="sec4_4_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 1′-((<i>S</i>)-1-Phenylethyl)-28-methyl-1<i>H</i>′-lup-2-eno-[2,3-<i>d</i>]-[1,2,3]-triazole (<b>5u</b>)</h4><div class="NLM_p last">Lupenone (100 mg, 1 equiv, 0.235 mmol; provided by Milan Urban), (<i>S</i>)-(−)-α-methylbenzylamine (37 mg, 1.3 equiv, 0.306 mmol), 4-nitrophenyl azide (39 mg, 1 equiv, 0.235 mmol), 4 Å molecular sieves (50 mg), and toluene (0.5 mL). Reaction time is overnight. The product was purified by flash column chromatography (first washed with CH<sub>2</sub>Cl<sub>2</sub> followed by petroleum ether:EtOAc 9:1 → 6:4) affording <b>5u</b> (107 mg, 83% yield) as an off-white solid. m.p. 283 °C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ 7.29 (t, <i>J</i> = 7.7 Hz, 2H), 7.23 (t, <i>J</i> = 7.3 Hz, 1H), 7.15 (d, <i>J</i> = 7.8 Hz, 2H), 5.75 (m, 1H), 4.71 (s, 1H), 4.60 (s, 1H), 2.95 (d, <i>J</i> = 15.3 Hz, 1H), 2.39 (m, 1H), 2.17 (d, <i>J</i> = 15.4 Hz, 1H), 2.03 (d, <i>J</i> = 7.0 Hz, 3H), 1.93 (m, 1H), 1.74–1.68 (m, 5H), 1.68–1.40 (m, 11H), 1.40–1.27 (m, 9H), 1.06 (s, 3H), 0.98 (s, 3H), 0.96 (s, 3H), 0.80 (s, 3H), 0.73 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ 150.7, 141.9, 141.2, 137.6, 128.6, 127.5, 126.1, 109.5, 77.2, 77.0, 76.8, 59.1, 54.9, 49.2, 48.2, 47.9, 43.0, 42.8, 40.7, 39.9, 38.8, 38.5, 38.2, 35.5, 33.6, 33.3, 29.8, 28.6, 27.5, 25.1, 23.4, 21.5, 19.3, 19.0, 18.0, 15.9, 15.6, 14.5. HRMS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> calculated for C<sub>38</sub>H<sub>55</sub>N<sub>3</sub>H [M + H]<sup>+</sup>: 554.4468, found 554.4462.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Biology</h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Anti-coronavirus Evaluation in Cell Culture</h4><div class="NLM_p last">HCoV-229E was purchased from ATCC (VR-740) and expanded in human embryonic lung fibroblast cells (HEL; ATCC CCL-137). The titers of virus stocks were determined in HEL cells and expressed as TCID<sub>50</sub> (50% tissue culture infective dose).<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The cytopathic effect (CPE) reduction assay was performed in 96-well plates containing confluent HEL cell cultures, as previously described.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> Serial compound dilutions were added together with HCoV-229E at an MOI of 100. In parallel, the compounds were added to a mock-infected plate to assess cytotoxicity. Besides the test compounds, two references were included, i.e., K22 [(<i>Z</i>)-<i>N</i>-[3-[4-(4-bromophenyl)-4-hydroxypiperidin-1-yl]-3-oxo-1-phenylprop-1-en-2-yl]benzamide;<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> from ChemDiv] and GS-441524 (the nucleoside form of remdesivir; from Carbosynth). After 5 days incubation at 35 °C, microscopy was performed to score virus-induced CPE. To next perform the colorimetric MTS cell viability assay, the reagent (CellTiter 96 AQ<sub>ueous</sub> MTS Reagent from Promega) was added to the wells, and 24 h later, absorbance at 490 nm was measured in a plate reader. Antiviral activity was calculated from three independent experiments and expressed as EC<sub>50</sub> or concentration showing 50% efficacy in the MTS or microscopic assay (see ref <a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> for calculation details). Cytotoxicity was expressed as 50% cytotoxic concentration (CC<sub>50</sub>) in the MTS assay.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Immunofluorescence Detection of Viral dsRNA</h4><div class="NLM_p last">Semiconfluent cultures of human bronchial epithelial 16HBE cells (a gift from P. Hoet, Leuven, Belgium) in 8-well chamber slides (Ibidi) were infected with HCoV-229E (MOI: 1000) in the presence of 12 μM <b>5h</b> or GS-441524. After 4 h incubation at 35 °C, the inoculum was removed, the compound was added again, and the slides were further incubated. At 24 h p.i., the cells were subjected to immunostaining for dsRNA (all incubations at room temperature). After cell fixation with 3.7% formaldehyde in PBS for 15 min and permeabilization with 0.2% Triton X-100 in PBS for 10 min, unspecific binding sites were blocked with 1% BSA in PBS for 30 min. Next, 1 h incubation was done with mouse monoclonal anti-dsRNA antibody (J2, SCICONS English & Scientific Consulting Kft; diluted 1:1000 in PBS with 1% BSA,) followed by 1 h incubation with goat anti-mouse AlexaFluor488 (A21131, Invitrogen; 1:1000 in PBS with 1% BSA). Cell nuclei were stained with DAPI (4′,6-diamidino-2-phenylindole, Invitrogen) in PBS for 20 min at RT. Microscopic images were acquired using the Leica TCS SP5 confocal microscope (Leica Microsystems) with a HCX PL APO 63x (NA 1.2) water immersion objective. DAPI and AlexaFluor488 were detected with excitation lines at 405 nm (blue) and 488 nm (green) and emission lines of 410–480 nm (blue) and 495–565 nm (green).</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Time-of-Addition Assay</h4><div class="NLM_p last">Confluent HEL cells were infected with HCoV-229E (MOI: 100) in a 96-well plate, and the compounds [<b>5h</b>, bafilomycin A<sub>1</sub> (from Cayman), or K22] were added at −0.5, 0.5, 2, 4, 6, or 8 h post-infection (p.i.). At 16 h p.i., the supernatant was discarded and each well was washed twice with ice-cold PBS. The cells were lysed on ice for 10 min with 22 μL of lysis mix, consisting of lysis enhancer and resuspension buffer at a 1:10 ratio (both from the CellsDirect One-Step RT-qPCR kit; Invitrogen). Next, the lysates were incubated for 10 min at 75 °C and treated with DNase (Invitrogen) to remove interfering cellular DNA. The number of viral RNA copies in each sample was determined by one-step RT-qPCR. Five microliters of lysate was transferred to a qPCR plate containing 9.75 μL of RT-qPCR mix (CellsDirect One-Step RT-qPCR) and 0.25 μL of Superscript III RT/Platinum Taq enzyme and HCoV-229E N-gene specific primers and probe.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The RT-qPCR protocol consisted of 15 min at 50 °C; 2 min at 95 °C; and 40 cycles of 15 s at 95 °C and 45 s at 60 °C. An N-gene plasmid standard was included to allow absolute quantification of viral RNA genome copies. The data from two independent experiments were expressed as the number of viral RNA copies at 16 h p.i. relative to the virus control receiving no compound.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Selection of Resistant Coronavirus Mutants</h4><div class="NLM_p last">HEL cells were infected with HCoV-229E virus (MOI: 25), and <b>5h</b> was added at different concentrations. After 5 days incubation at 35 °C, the CPE was scored microscopically to estimate the EC<sub>50</sub> value. From the highest compound concentration conditions showing virus-induced CPE, the supernatants plus cells were frozen at −80 °C. These harvests were further passaged in HEL cells under gradually increasing compound concentrations, until a manifest increase in EC<sub>50</sub> was observed. A no compound control condition was passaged in parallel. The final virus passages were submitted to RNA extraction; reverse transcription; high-fidelity PCR; and cycle sequencing on the entire viral genome using a set of 39 primers (sequences available upon request). After sequence assembly with CLC Main Workbench 7.9.1 (Qiagen), the sequences of the viruses passaged in the absence and presence of <b>5h</b> were aligned in order to identify the <b>5h</b> resistance sites in the HCoV-229E genome.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Virus Yield Assay</h4><div class="NLM_p last">The virus yield assay was performed in 96-well plates with semiconfluent cultures of 16HBE cells or confluent HEL cells. Serial dilutions of compound <b>5h</b> were added, and the cells were infected (MOI: 100) with wild-type HCoV-229E (229E-WT), EndoU-deficient HCoV-229E (229E-H250A<sub>nsp15</sub>),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> or the mutant viruses obtained by passaging under <b>5h</b> (229E-K60R<sub>nsp15</sub> or 229E-T66I<sub>nsp15</sub>). After 4 h incubation at 35 °C, the inoculum was removed, the compound dilutions were added again, and the plates were further incubated. At 3 days p.i., the supernatants were collected, and 2 μL of each supernatant was lysed on ice by adding 11 μL of lysis mix containing lysis enhancer and resuspension buffer at a 1:10 ratio. The lysates were incubated for 10 min at 75 °C, and the viral RNA copy number was determined by RT-qPCR as described above for the time-of-addition assay. The data were collected in three independent experiments and expressed as the fold reduction in viral RNA compared to the virus control receiving no compound.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Computational Work</h4><div class="NLM_p last">Starting from the published X-ray structure of the hexameric nsp15 protein from HCoV-229E (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4">4RS4</a>), we first changed a few amino acid residues, i.e., S1Q, S17G, A142T, M219I, and S252L, to obtain the fully correct nsp15 protein sequence of HCoV-229E virus. The structures of HCoV-229E nsp15 and SARS-CoV-2 nsp 15 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7K1O">7K1O</a>) were prepared using MOE (Chemical Computing Group, Montreal, Canada). Hydrogen addition and optimization of protonation state and rotamers of the mutations were conducted using the AMBER-EHT force field, and identification of the potential binding sites in the multimeric complex was performed using MOE. Docking of betulonic acid derivatives was carried out by means of both MOE and GOLD with default settings, where GBVI/WSA score and Goldscore functions were used, respectively. The common top scoring solution was selected for further research.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02124" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57478" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57478" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02124?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02124</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">(Figure S1) Alignment of nsp15 protein sequences, HPLC traces of key compounds, and <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of the synthesized compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">PDB coordinates for computational models: <b>5h</b> in complex with nsp15 from HCoV-229E (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">PDB coordinates for computational models: <b>5h</b> in complex with nsp15 from SARS-CoV-2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_004.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_001.pdf">jm0c02124_si_001.pdf (8.54 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_002.pdb">jm0c02124_si_002.pdb (880.46 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_003.pdb">jm0c02124_si_003.pdb (889.72 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_004.csv">jm0c02124_si_004.csv (3.84 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02124" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Besir Krasniqi</span> - <span class="hlFld-Affiliation affiliation">Molecular
Design and Synthesis, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#96f4f3e5ffe4b8fde4f7e5f8ffe7ffd6fde3faf3e3e0f3f8b8f4f3"><span class="__cf_email__" data-cfemail="cba9aeb8a2b9e5a0b9aab8a5a2baa28ba0bea7aebebdaea5e5a9ae">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lieve Naesens</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Virology and Chemotherapy, Rega Institute,
KU Leuven, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9742-9302" title="Orcid link">http://orcid.org/0000-0001-9742-9302</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#65090c0013004b0b040016000b16250e1009001013000b4b0700"><span class="__cf_email__" data-cfemail="ddb1b4b8abb8f3b3bcb8aeb8b3ae9db6a8b1b8a8abb8b3f3bfb8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annelies Stevaert</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Virology and Chemotherapy, Rega Institute,
KU Leuven, 3000 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3316-2826" title="Orcid link">http://orcid.org/0000-0003-3316-2826</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Van Loy</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Virology and Chemotherapy, Rega Institute,
KU Leuven, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tien Nguyen</span> - <span class="hlFld-Affiliation affiliation">Biochemistry,
Molecular and Structural Biology, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joice Thomas</span> - <span class="hlFld-Affiliation affiliation">Molecular
Design and Synthesis, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</span>; 
    <span>Present Address:
                        Present address: Arcus Biosciences, Hayward, California 94545, United States (J.T.)</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5517-4663" title="Orcid link">http://orcid.org/0000-0001-5517-4663</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julie Vandeput</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Virology and Chemotherapy, Rega Institute,
KU Leuven, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dirk Jochmans</span> - <span class="hlFld-Affiliation affiliation">Laboratory
of Virology and Chemotherapy, Rega Institute,
KU Leuven, 3000 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Volker Thiel</span> - <span class="hlFld-Affiliation affiliation">Institute
of Virology and Immunology (IVI), 3012
Bern and 3012 Bern, Switzerland</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern, 3012 Bern, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ronald Dijkman</span> - <span class="hlFld-Affiliation affiliation">Institute
of Virology and Immunology (IVI), 3012
Bern and 3012 Bern, Switzerland</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Infectious Diseases and Pathobiology, Vetsuisse Faculty, University of Bern, 3012 Bern, Switzerland</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Infectious Diseases (IFIK), University
of Bern, 3012 Bern, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wim Dehaen</span> - <span class="hlFld-Affiliation affiliation">Molecular
Design and Synthesis, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Arnout Voet</span> - <span class="hlFld-Affiliation affiliation">Biochemistry,
Molecular and Structural Biology, Department of Chemistry, KU Leuven, 3001 Leuven, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3329-2703" title="Orcid link">http://orcid.org/0000-0002-3329-2703</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>A.S. and B.K. contributed equally. A.S. and L.N. designed, performed, and interpreted the biological experiments. B.K. and J.T. performed compound synthesis and analysis. B.V.L., J.V., and D.J. performed antiviral experiments. T.N. and A.V. performed and interpreted the in silico study. V.T. and R.D. provided the materials. A.S., B.K., W.D., A.V., and L.N. co-wrote the manuscript. All authors gave approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87253" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87253" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">B.K. acknowledges receipt of an Erasmus Mundus Western Balkan Action 2 and doctoral fellowship from KU Leuven. B.V.L. holds an SB-PhD fellowship from the FWO Research Foundation Flanders (project: 1S66321N). Mass spectrometry was made possible by the support of the Hercules Foundation of the Flemish Government (grant: 20100225-7). We acknowledge the support of NVIDIA Corporation for donating the Titan Xp GPU used for this study. This research work was supported by funding from the KU Leuven (IDO/12/020 and project C14/19/78); the European Union’s Innovative Medicines Initiative (IMI) under grant agreement 101005077 [Corona Accelerated R&D in Europe (CARE) project]; and Fundació La Marató de TV3, Spain (project no. 201832-30). We thank Amy Swinnen and Leentje Persoons for dedicated technical assistance and Els Vanstreels for providing confocal microscopy training. We also thank Milan Urban for providing lupenone.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CC<sub>50</sub></td><td class="NLM_def"><p class="first last">50% cytotoxic concentration</p></td></tr><tr><td class="NLM_term">CoV</td><td class="NLM_def"><p class="first last">coronavirus</p></td></tr><tr><td class="NLM_term">COVID-19</td><td class="NLM_def"><p class="first last">coronavirus disease 2019</p></td></tr><tr><td class="NLM_term">CPE</td><td class="NLM_def"><p class="first last">cytopathic effect</p></td></tr><tr><td class="NLM_term">DAPI</td><td class="NLM_def"><p class="first last">4′,6-diamidino-2-phenylindole</p></td></tr><tr><td class="NLM_term">EndoU</td><td class="NLM_def"><p class="first last">endoribonuclease</p></td></tr><tr><td class="NLM_term">FIPV</td><td class="NLM_def"><p class="first last">feline infectious peritonitis virus</p></td></tr><tr><td class="NLM_term">HEL</td><td class="NLM_def"><p class="first last">human embryonic lung</p></td></tr><tr><td class="NLM_term">MERS</td><td class="NLM_def"><p class="first last">Middle East respiratory syndrome</p></td></tr><tr><td class="NLM_term">MHV-A59</td><td class="NLM_def"><p class="first last">mouse hepatitis virus A59</p></td></tr><tr><td class="NLM_term">MTS</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxyphenyl)-2-(4-sulfophenyl)-2<i>H</i>-tetrazolium</p></td></tr><tr><td class="NLM_term">nsp</td><td class="NLM_def"><p class="first last">non-structural protein</p></td></tr><tr><td class="NLM_term">p.i.</td><td class="NLM_def"><p class="first last">post-infection</p></td></tr><tr><td class="NLM_term">RTC</td><td class="NLM_def"><p class="first last">replication-transcription complex</p></td></tr><tr><td class="NLM_term">SARS</td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome</p></td></tr><tr><td class="NLM_term">SARS-CoV-2</td><td class="NLM_def"><p class="first last">severe acute respiratory syndrome coronavirus 2</p></td></tr><tr><td class="NLM_term">TCID<sub>50</sub></td><td class="NLM_def"><p class="first last">50% tissue culture infective dose</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2016.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.tim.2016.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=27012512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWmsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=490-502&author=S.+Suauthor=G.+Wongauthor=W.+Shiauthor=J.+Liuauthor=A.+C.+K.+Laiauthor=J.+Zhouauthor=W.+Liuauthor=Y.+Biauthor=G.+F.+Gao&title=Epidemiology%2C+genetic+recombination%2C+and+pathogenesis+of+coronaviruses&doi=10.1016%2Fj.tim.2016.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span></div><div class="casAuthors">Su, Shuo; Wong, Gary; Shi, Weifeng; Liu, Jun; Lai, Alexander C. K.; Zhou, Jiyong; Liu, Wenjun; Bi, Yuhai; Gao, George F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">490-502</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Human coronaviruses (HCoVs) were first described in the 1960s for patients with the common cold.  Since then, more HCoVs have been discovered, including those that cause severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), two pathogens that, upon infection, can cause fatal respiratory disease in humans.  It was recently discovered that dromedary camels in Saudi Arabia harbor three different HCoV species, including a dominant MERS HCoV lineage that was responsible for the outbreaks in the Middle East and South Korea during 2015.  In this review we aim to compare and contrast the different HCoVs with regard to epidemiol. and pathogenesis, in addn. to the virus evolution and recombination events which have, on occasion, resulted in outbreaks amongst humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAQICS-ipqM7Vg90H21EOLACvtfcHk0li9qLVbUVL7Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWmsbc%253D&md5=c9215c7687711bf8c2ff73a5cc2d23f4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2016.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2016.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DA.%2BC.%2BK.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DBi%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DEpidemiology%252C%2520genetic%2520recombination%252C%2520and%2520pathogenesis%2520of%2520coronaviruses%26jtitle%3DTrends%2520Microbiol.%26date%3D2016%26volume%3D24%26spage%3D490%26epage%3D502%26doi%3D10.1016%2Fj.tim.2016.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falsey, A. R.</span></span> <span> </span><span class="NLM_article-title">Clinical impact of human coronaviruses 229E and OC43 infection in diverse adult populations</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">1634</span>– <span class="NLM_lpage">1642</span>, <span class="refDoi"> DOI: 10.1093/infdis/jit393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1093%2Finfdis%2Fjit393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=23922367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A280%3ADC%252BC3sfntl2ksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2013&pages=1634-1642&author=E.+E.+Walshauthor=J.+H.+Shinauthor=A.+R.+Falsey&title=Clinical+impact+of+human+coronaviruses+229E+and+OC43+infection+in+diverse+adult+populations&doi=10.1093%2Finfdis%2Fjit393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical impact of human coronaviruses 229E and OC43 infection in diverse adult populations</span></div><div class="casAuthors">Walsh Edward E; Shin Jae Hyun; Falsey Ann R</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of infectious diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1634-42</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The incidence and clinical impact of coronavirus (CoV) infection in elderly persons and those with underlying cardiopulmonary disease over a long duration is not well described.  We determined the incidence and clinical impact of 229E and OC43 CoV in this population during 4 consecutive winters, and compared illnesses to influenza A, respiratory syncytial virus, and human metapneumovirus.  METHODS:  CoV 229E and OC43 were detected by reverse transcription polymerase chain reaction and serology in 4 adult populations under surveillance for acute respiratory illness during the winters of 1999-2003.  Cohorts included healthy young adults, healthy elderly adults, high-risk adults with underlying cardiopulmonary disease, and a hospitalized group.  RESULTS:  Three hundred ninety-eight CoV infections were identified, with annual infection rates ranging from 2.8% to 26% in prospective cohorts, and prevalence ranging from 3.3% to 11.1% in the hospitalized cohort.  The incidence of infections with each strain was similar, although asymptomatic infection and viral coinfection was significantly more common with 229E than OC43 infection.  Although the incidence and clinical manifestations were similar for each strain, OC43-infected subjects tended to seek more medical care, as OC43 was twice as common as 229E among the hospitalized cohort.  CONCLUSIONS:  CoV infections in the elderly are frequent, likely causing substantial medical disease burden.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRKc5dNmh-_Y2PgL4LrC-0VfW6udTcc2eZklDh6e0ey07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfntl2ksw%253D%253D&md5=e1d57d74aaf1d4e7a39870d9a27c6146</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjit393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjit393%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DE.%2BE.%26aulast%3DShin%26aufirst%3DJ.%2BH.%26aulast%3DFalsey%26aufirst%3DA.%2BR.%26atitle%3DClinical%2520impact%2520of%2520human%2520coronaviruses%2520229E%2520and%2520OC43%2520infection%2520in%2520diverse%2520adult%2520populations%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2013%26volume%3D208%26spage%3D1634%26epage%3D1642%26doi%3D10.1093%2Finfdis%2Fjit393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dijkman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Hoek, L.</span></span> <span> </span><span class="NLM_article-title">Human coronaviruses 229E and NL63: close yet still so far</span>. <i>J. Formos. Med. Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/S0929-6646(09)60066-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2FS0929-6646%2809%2960066-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=19369173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFCmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2009&pages=270-279&author=R.+Dijkmanauthor=L.+van+der+Hoek&title=Human+coronaviruses+229E+and+NL63%3A+close+yet+still+so+far&doi=10.1016%2FS0929-6646%2809%2960066-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Human coronaviruses 229E and NL63: close yet still so far</span></div><div class="casAuthors">Dijkman, Ronald; van der Hoek, Lia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Formosan Medical Association</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">270-279</span>CODEN:
                <span class="NLM_cas:coden">JFASEO</span>;
        ISSN:<span class="NLM_cas:issn">0929-6646</span>.
    
            (<span class="NLM_cas:orgname">Elsevier (Singapore) Pte Ltd.</span>)
        </div><div class="casAbstract">A review.  HCoV-NL63 and HCoV-229E are two of the four human coronaviruses that circulate worldwide.  These two viruses are unique in their relationship towards each other.  Phylogenetically, the viruses are more closely related to each other than to any other human coronavirus, yet they only share 65% sequence identity.  Moreover, the viruses use different receptors to enter their target cell.  HCoV-NL63 is assocd. with croup in children, whereas all signs suggest that the virus probably causes the common cold in healthy adults.  HCoV-229E is a proven common cold virus in healthy adults, so it is probable that both viruses induce comparable symptoms in adults, even though their mode of infection differs.  Here, we present an overview of the current knowledge on both human coronaviruses, focusing on similarities and differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7RjSmMw3WnrVg90H21EOLACvtfcHk0lipxbytpZH0Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFCmtLk%253D&md5=6749f66f8d499f73ec4def49c1282c09</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0929-6646%2809%2960066-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0929-6646%252809%252960066-8%26sid%3Dliteratum%253Aachs%26aulast%3DDijkman%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BHoek%26aufirst%3DL.%26atitle%3DHuman%2520coronaviruses%2520229E%2520and%2520NL63%253A%2520close%2520yet%2520still%2520so%2520far%26jtitle%3DJ.%2520Formos.%2520Med.%2520Assoc.%26date%3D2009%26volume%3D108%26spage%3D270%26epage%3D279%26doi%3D10.1016%2FS0929-6646%2809%2960066-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horby, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">A novel coronavirus outbreak of global health concern</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(20)30185-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2FS0140-6736%2820%2930185-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=31986257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnsVaisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2020&pages=470-473&author=C.+Wangauthor=P.+W.+Horbyauthor=F.+G.+Haydenauthor=G.+F.+Gao&title=A+novel+coronavirus+outbreak+of+global+health+concern&doi=10.1016%2FS0140-6736%2820%2930185-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A novel coronavirus outbreak of global health concern</span></div><div class="casAuthors">Wang, Chen; Horby, Peter W.; Hayden, Frederick G.; Gao, George F.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">10223</span>),
    <span class="NLM_cas:pages">470-473</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review on COVID-19 and global health concern.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRk9FBxgaVErVg90H21EOLACvtfcHk0lipxbytpZH0Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnsVaisLg%253D&md5=f16ec2beb810968cba634e1813888780</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2820%2930185-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252820%252930185-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHorby%26aufirst%3DP.%2BW.%26aulast%3DHayden%26aufirst%3DF.%2BG.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DA%2520novel%2520coronavirus%2520outbreak%2520of%2520global%2520health%2520concern%26jtitle%3DLancet%26date%3D2020%26volume%3D395%26spage%3D470%26epage%3D473%26doi%3D10.1016%2FS0140-6736%2820%2930185-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fehr, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span></span> <span> </span><span class="NLM_article-title">Coronaviruses: an overview of their replication and pathogenesis</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1282</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2438-7_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1007%2F978-1-4939-2438-7_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=25720466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls12ksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1282&publication_year=2015&pages=1-23&author=A.+R.+Fehrauthor=S.+Perlman&title=Coronaviruses%3A+an+overview+of+their+replication+and+pathogenesis&doi=10.1007%2F978-1-4939-2438-7_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Coronaviruses: an overview of their replication and pathogenesis</span></div><div class="casAuthors">Fehr, Anthony R.; Perlman, Stanley</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1282</span>
        (<span class="NLM_cas:issue">Coronaviruses</span>),
    <span class="NLM_cas:pages">1-23</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Coronaviruses (CoVs), enveloped pos.-sense RNA viruses, are characterized by club-like spikes that project from their surface, an unusually large RNA genome, and a unique replication strategy.  Coronaviruses cause a variety of diseases in mammals and birds ranging from enteritis in cows and pigs and upper respiratory disease in chickens to potentially lethal human respiratory infections.  Here we provide a brief introduction to coronaviruses discussing their replication and pathogenicity, and current prevention and treatment strategies.  We also discuss the outbreaks of the highly pathogenic Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the recently identified Middle Eastern Respiratory Syndrome Coronavirus (MERS-CoV).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSI4XoT-KaBrVg90H21EOLACvtfcHk0lipxbytpZH0Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls12ksL8%253D&md5=2e0fc18cd6b7312f0704f933a3614913</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2438-7_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2438-7_1%26sid%3Dliteratum%253Aachs%26aulast%3DFehr%26aufirst%3DA.%2BR.%26aulast%3DPerlman%26aufirst%3DS.%26atitle%3DCoronaviruses%253A%2520an%2520overview%2520of%2520their%2520replication%2520and%2520pathogenesis%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1282%26spage%3D1%26epage%3D23%26doi%3D10.1007%2F978-1-4939-2438-7_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felsenstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedrich, C. M.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 infections in children and young people</span>. <i>Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>220</i></span>,  <span class="NLM_fpage">108588</span>, <span class="refDoi"> DOI: 10.1016/j.clim.2020.108588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.clim.2020.108588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32905851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSksLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=220&publication_year=2020&pages=108588&author=S.+Felsensteinauthor=C.+M.+Hedrich&title=SARS-CoV-2+infections+in+children+and+young+people&doi=10.1016%2Fj.clim.2020.108588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 infections in children and young people</span></div><div class="casAuthors">Felsenstein, Susanna; Hedrich, Christian M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Immunology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">220</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108588</span>CODEN:
                <span class="NLM_cas:coden">CLIIFY</span>;
        ISSN:<span class="NLM_cas:issn">1521-6616</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Though recent reports link SARS-CoV-2 infections with hyper-inflammatory states in children, most children experience no/mild symptoms, and hospitalization and mortality rates are low in the age group.  As symptoms are usually mild and seroconversion occurs at low frequencies, it remains unclear whether children significantly contribute to community transmission.  Several hypotheses try to explain age-related differences in disease presentation and severity.  Possible reasons for milder presentations in children as compared to adults include frequent contact to seasonal coronaviruses, presence of cross-reactive antibodies, and/or co-clearance with other viruses.  Increased expression of ACE2 in young people may facilitate virus infection, while limiting inflammation and reducing the risk of severe disease.  Further potential factors include recent vaccinations and a more diverse memory T cell repertoire.  This manuscript reviews age-related host factors that may protect children from COVID-19 and complications assocd., and addresses the confusion around seropositivity and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorKCcwMLFke7Vg90H21EOLACvtfcHk0lgjp_3R4EWBQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSksLnF&md5=6de5a48079318b0d286b9b4f143dce81</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.clim.2020.108588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clim.2020.108588%26sid%3Dliteratum%253Aachs%26aulast%3DFelsenstein%26aufirst%3DS.%26aulast%3DHedrich%26aufirst%3DC.%2BM.%26atitle%3DSARS-CoV-2%2520infections%2520in%2520children%2520and%2520young%2520people%26jtitle%3DClin.%2520Immunol.%26date%3D2020%26volume%3D220%26spage%3D108588%26doi%3D10.1016%2Fj.clim.2020.108588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amigues, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youngstein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayatilleke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konig, M.</span></span> <span> </span><span class="NLM_article-title">Coronavirus disease 2019: investigational therapies in the prevention and treatment of hyperinflammation</span>. <i>Expert Rev. Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1204</span>, <span class="refDoi"> DOI: 10.1080/1744666X.2021.1847084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1080%2F1744666X.2021.1847084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33146561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVClsL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1185-1204&author=I.+Amiguesauthor=A.+H.+Pearlmanauthor=A.+Patelauthor=P.+Reidauthor=P.+C.+Robinsonauthor=R.+Sinhaauthor=A.+H.+Kimauthor=T.+Youngsteinauthor=A.+Jayatillekeauthor=M.+Konig&title=Coronavirus+disease+2019%3A+investigational+therapies+in+the+prevention+and+treatment+of+hyperinflammation&doi=10.1080%2F1744666X.2021.1847084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus disease 2019: investigational therapies in the prevention and treatment of hyperinflammation</span></div><div class="casAuthors">Amigues, Isabelle; Pearlman, Alexander H.; Patel, Aarat; Reid, Pankti; Robinson, Philip C.; Sinha, Rashmi; Kim, Alfred Hj; Youngstein, Taryn; Jayatilleke, Arundathi; Konig, Maximilian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Clinical Immunology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1185-1204</span>CODEN:
                <span class="NLM_cas:coden">ERCIBU</span>;
        ISSN:<span class="NLM_cas:issn">1744-666X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The mortality of coronavirus disease 2019 (COVID-19) is frequently driven by an injurious immune response characterized by the development of acute respiratory distress syndrome (ARDS), endotheliitis, coagulopathy, and multi-organ failure.  This spectrum of hyperinflammation in COVID-19 is commonly referred to as cytokine storm syndrome (CSS).: Medline and Google Scholar were searched up until 15th of August 2020 for relevant literature.  Evidence supports a role of dysregulated immune responses in the immunopathogenesis of severe COVID-19.  CSS assocd. with SARS-CoV-2 shows similarities to the exuberant cytokine prodn. in some patients with viral infection (e.g.SARS-CoV-1) and may be confused with other syndromes of hyperinflammation like the cytokine release syndrome (CRS) in CAR-T cell therapy.  Interleukin (IL)-6, IL-8, and tumor necrosis factor-alpha have emerged as predictors of COVID-19 severity and in-hospital mortality.  Despite similarities, COVID-19-CSS appears to be distinct from HLH, MAS, and CRS, and the application of HLH diagnostic scores and criteria to COVID-19 is not supported by emerging data.  While immunosuppressive therapy with glucocorticoids has shown a mortality benefit, cytokine inhibitors may hold promise as 'rescue therapies' in severe COVID-19.  Given the arguably limited benefit in advanced disease, strategies to prevent the development of COVID-19-CSS are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxw7l66l9AeLVg90H21EOLACvtfcHk0lgjp_3R4EWBQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVClsL7P&md5=6cd0d70b3c9d3f986e96ffba3248d03b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1080%2F1744666X.2021.1847084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F1744666X.2021.1847084%26sid%3Dliteratum%253Aachs%26aulast%3DAmigues%26aufirst%3DI.%26aulast%3DPearlman%26aufirst%3DA.%2BH.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DReid%26aufirst%3DP.%26aulast%3DRobinson%26aufirst%3DP.%2BC.%26aulast%3DSinha%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DA.%2BH.%26aulast%3DYoungstein%26aufirst%3DT.%26aulast%3DJayatilleke%26aufirst%3DA.%26aulast%3DKonig%26aufirst%3DM.%26atitle%3DCoronavirus%2520disease%25202019%253A%2520investigational%2520therapies%2520in%2520the%2520prevention%2520and%2520treatment%2520of%2520hyperinflammation%26jtitle%3DExpert%2520Rev.%2520Clin.%2520Immunol.%26date%3D2020%26volume%3D16%26spage%3D1185%26epage%3D1204%26doi%3D10.1080%2F1744666X.2021.1847084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Recovery
Collaborative Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horby, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emberson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mafham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staplin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brightling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ustianowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmahi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadwick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rege, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fegan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappell, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faust, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juszczak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landray, M. J.</span></span> <span> </span><span class="NLM_article-title">Dexamethasone in hospitalized patients with COVID-19 - Preliminary report</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">704</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1056%2FNEJMoa2021436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32678530" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2020&pages=693-704&author=Recovery%0ACollaborative+Groupauthor=P.+Horbyauthor=W.+S.+Limauthor=J.+R.+Embersonauthor=M.+Mafhamauthor=J.+L.+Bellauthor=L.+Linsellauthor=N.+Staplinauthor=C.+Brightlingauthor=A.+Ustianowskiauthor=E.+Elmahiauthor=B.+Prudonauthor=C.+Greenauthor=T.+Feltonauthor=D.+Chadwickauthor=K.+Regeauthor=C.+Feganauthor=L.+C.+Chappellauthor=S.+N.+Faustauthor=T.+Jakiauthor=K.+Jefferyauthor=A.+Montgomeryauthor=K.+Rowanauthor=E.+Juszczakauthor=J.+K.+Baillieauthor=R.+Haynesauthor=M.+J.+Landray&title=Dexamethasone+in+hospitalized+patients+with+COVID-19+-+Preliminary+report"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2021436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2021436%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3DHorby%26aufirst%3DP.%26aulast%3DLim%26aufirst%3DW.%2BS.%26aulast%3DEmberson%26aufirst%3DJ.%2BR.%26aulast%3DMafham%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DJ.%2BL.%26aulast%3DLinsell%26aufirst%3DL.%26aulast%3DStaplin%26aufirst%3DN.%26aulast%3DBrightling%26aufirst%3DC.%26aulast%3DUstianowski%26aufirst%3DA.%26aulast%3DElmahi%26aufirst%3DE.%26aulast%3DPrudon%26aufirst%3DB.%26aulast%3DGreen%26aufirst%3DC.%26aulast%3DFelton%26aufirst%3DT.%26aulast%3DChadwick%26aufirst%3DD.%26aulast%3DRege%26aufirst%3DK.%26aulast%3DFegan%26aufirst%3DC.%26aulast%3DChappell%26aufirst%3DL.%2BC.%26aulast%3DFaust%26aufirst%3DS.%2BN.%26aulast%3DJaki%26aufirst%3DT.%26aulast%3DJeffery%26aufirst%3DK.%26aulast%3DMontgomery%26aufirst%3DA.%26aulast%3DRowan%26aufirst%3DK.%26aulast%3DJuszczak%26aufirst%3DE.%26aulast%3DBaillie%26aufirst%3DJ.%2BK.%26aulast%3DHaynes%26aufirst%3DR.%26aulast%3DLandray%26aufirst%3DM.%2BJ.%26atitle%3DDexamethasone%2520in%2520hospitalized%2520patients%2520with%2520COVID-19%2520-%2520Preliminary%2520report%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D384%26spage%3D693%26epage%3D704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M. Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: drug targets and potential treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.+%C3%81.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ram%C3%ADrezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+drug+targets+and+potential+treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0likOknSu17SiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%2B%25C3%2581.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRam%25C3%25ADrez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520drug%2520targets%2520and%2520potential%2520treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeFrancesco, L.</span></span> <span> </span><span class="NLM_article-title">COVID-19 antibodies on trial</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi"> DOI: 10.1038/s41587-020-0732-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41587-020-0732-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33087898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFGqtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2020&pages=1242-1252&author=L.+DeFrancesco&title=COVID-19+antibodies+on+trial&doi=10.1038%2Fs41587-020-0732-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19 antibodies on trial</span></div><div class="casAuthors">DeFrancesco, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1242-1252</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  With the first readouts of trials of antibodies against COVID-19 appearing and others coming thick and fast, Nature Biotechnol. asked a group of experts to comment on the challenges and timelines for these products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjXGHF7tce4bVg90H21EOLACvtfcHk0likOknSu17SiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFGqtLfI&md5=6b2a4fcc68271e9a8931f004da197b69</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41587-020-0732-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41587-020-0732-8%26sid%3Dliteratum%253Aachs%26aulast%3DDeFrancesco%26aufirst%3DL.%26atitle%3DCOVID-19%2520antibodies%2520on%2520trial%26jtitle%3DNat.%2520Biotechnol.%26date%3D2020%26volume%3D38%26spage%3D1242%26epage%3D1252%26doi%3D10.1038%2Fs41587-020-0732-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lye, D. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montejano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spinner, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahass, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SenGupta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osinusi, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaggar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brainard, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towner, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullane, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tashima, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span> <span> </span><span class="NLM_article-title">GS-US-540-5773 Investigators, Remdesivir for 5 or 10 days in patients with severe Covid-19</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>383</i></span>,  <span class="NLM_fpage">1827</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2015301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1056%2FNEJMoa2015301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32459919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1ylt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=383&publication_year=2020&pages=1827-1837&author=J.+D.+Goldmanauthor=D.+C.+B.+Lyeauthor=D.+S.+Huiauthor=K.+M.+Marksauthor=R.+Brunoauthor=R.+Montejanoauthor=C.+D.+Spinnerauthor=M.+Galliauthor=M.+Y.+Ahnauthor=R.+G.+Nahassauthor=Y.+S.+Chenauthor=D.+SenGuptaauthor=R.+H.+Hylandauthor=A.+O.+Osinusiauthor=H.+Caoauthor=C.+Blairauthor=X.+Weiauthor=A.+Gaggarauthor=D.+M.+Brainardauthor=W.+J.+Townerauthor=J.+Munozauthor=K.+M.+Mullaneauthor=F.+M.+Martyauthor=K.+T.+Tashimaauthor=G.+Diazauthor=A.+Subramanian&title=GS-US-540-5773+Investigators%2C+Remdesivir+for+5+or+10+days+in+patients+with+severe+Covid-19&doi=10.1056%2FNEJMoa2015301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Remdesivir for 5 or 10 days in patients with severe Covid-19</span></div><div class="casAuthors">Goldman, Jason D.; Lye, David C. B.; Hui, David S.; Marks, Kristen M.; Bruno, Raffaele; Montejano, Rocio; Spinner, Christoph D.; Galli, Massimo; Ahn, Mi-Young; Nahass, Ronald G.; Chen, Yao-Shen; Gupta, Devi Sen; Hyland, Robert H.; Osinusi, Anu O.; Cao, Huyen; Blair, Christiana; Wei, Xuelian; Gaggar, Anuj; Brainard, Diana M.; Towner, William J.; Munoz, Jose; Mullane, Kathleen M.; Marty, Francisco M.; Tashima, Karen T.; Diaz, George; Subramanian, Aruna</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">383</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">1827-1837</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Remdesivir is an RNA polymerase inhibitor with potent antiviral activity in vitro and efficacy in animal models of coronavirus disease 2019 (Covid-19).  Methods: The authors conducted a randomized, open-label, phase 3 trial involving hospitalized patients with confirmed SARS-CoV-2 infection, oxygen satn. of 94% or less while they were breathing ambient air, and radiol. evidence of pneumonia.  Patients were randomly assigned in a 1:1 ratio to receive i.v. remdesivir for either 5 days or 10 days.  All patients received 200 mg of remdesivir on day 1 and 100 mg once daily on subsequent days.  The primary end point was clin. status on day 14, assessed on a 7-point ordinal scale.  Results: In total, 397 patients underwent randomization and began treatment (200 patients for 5 days and 197 for 10 days).  The median duration of treatment was 5 days (interquartile range, 5 to 5) in the 5-day group and 9 days (interquartile range, 5 to 10) in the 10-day group.  At baseline, patients randomly assigned to the 10-day group had significantly worse clin. status than those assigned to the 5-day group (P = 0.02).  By day 14, a clin. improvement of 2 points or more on the ordinal scale occurred in 64% of patients in the 5-day group and in 54% in the 10-day group.  After adjustment for baseline clin. status, patients in the 10-day group had a distribution in clin. status at day 14 that was similar to that among patients in the 5-day group (P = 0.14).  The most common adverse events were nausea (9% of patients), worsening respiratory failure (8%), elevated alanine aminotransferase level (7%), and constipation (7%).  Conclusions: In patients with severe Covid-19 not requiring mech. ventilation, the authors' trial did not show a significant difference between a 5-day course and a 10-day course of remdesivir.  With no placebo control, however, the magnitude of benefit cannot be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq5F3H7ZXqCrVg90H21EOLACvtfcHk0likOknSu17SiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1ylt7bK&md5=bac0eb1b32dd628266ed1412cb9a9e5d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2015301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2015301%26sid%3Dliteratum%253Aachs%26aulast%3DGoldman%26aufirst%3DJ.%2BD.%26aulast%3DLye%26aufirst%3DD.%2BC.%2BB.%26aulast%3DHui%26aufirst%3DD.%2BS.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DBruno%26aufirst%3DR.%26aulast%3DMontejano%26aufirst%3DR.%26aulast%3DSpinner%26aufirst%3DC.%2BD.%26aulast%3DGalli%26aufirst%3DM.%26aulast%3DAhn%26aufirst%3DM.%2BY.%26aulast%3DNahass%26aufirst%3DR.%2BG.%26aulast%3DChen%26aufirst%3DY.%2BS.%26aulast%3DSenGupta%26aufirst%3DD.%26aulast%3DHyland%26aufirst%3DR.%2BH.%26aulast%3DOsinusi%26aufirst%3DA.%2BO.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DBlair%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DGaggar%26aufirst%3DA.%26aulast%3DBrainard%26aufirst%3DD.%2BM.%26aulast%3DTowner%26aufirst%3DW.%2BJ.%26aulast%3DMunoz%26aufirst%3DJ.%26aulast%3DMullane%26aufirst%3DK.%2BM.%26aulast%3DMarty%26aufirst%3DF.%2BM.%26aulast%3DTashima%26aufirst%3DK.%2BT.%26aulast%3DDiaz%26aufirst%3DG.%26aulast%3DSubramanian%26aufirst%3DA.%26atitle%3DGS-US-540-5773%2520Investigators%252C%2520Remdesivir%2520for%25205%2520or%252010%2520days%2520in%2520patients%2520with%2520severe%2520Covid-19%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D383%26spage%3D1827%26epage%3D1837%26doi%3D10.1056%2FNEJMoa2015301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1499</span>– <span class="NLM_lpage">1504</span>, <span class="refDoi"> DOI: 10.1126/science.abc1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1126%2Fscience.abc1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32358203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1499-1504&author=W.+Yinauthor=C.+Maoauthor=X.+Luanauthor=D.+D.+Shenauthor=Q.+Shenauthor=H.+Suauthor=X.+Wangauthor=F.+Zhouauthor=W.+Zhaoauthor=M.+Gaoauthor=S.+Changauthor=Y.+C.+Xieauthor=G.+Tianauthor=H.+W.+Jiangauthor=S.+C.+Taoauthor=J.+Shenauthor=Y.+Jiangauthor=H.+Jiangauthor=Y.+Xuauthor=S.+Zhangauthor=Y.+Zhangauthor=H.+E.+Xu&title=Structural+basis+for+inhibition+of+the+RNA-dependent+RNA+polymerase+from+SARS-CoV-2+by+remdesivir&doi=10.1126%2Fscience.abc1560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span></div><div class="casAuthors">Yin, Wanchao; Mao, Chunyou; Luan, Xiaodong; Shen, Dan-Dan; Shen, Qingya; Su, Haixia; Wang, Xiaoxi; Zhou, Fulai; Zhao, Wenfeng; Gao, Minqi; Chang, Shenghai; Xie, Yuan-Chao; Tian, Guanghui; Jiang, He-Wei; Tao, Sheng-Ce; Shen, Jingshan; Jiang, Yi; Jiang, Hualiang; Xu, Yechun; Zhang, Shuyang; Zhang, Yan; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">1499-1504</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The pandemic of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has become a global crisis.  Replication of SARS-CoV-2 requires the viral RNA-dependent RNA polymerase (RdRp) enzyme, a target of the antiviral drug remdesivir.  Here, we report the cryo-electron microscopy structure of the SARS-CoV-2 RdRp, both in the apo form at 2.8-angstrom resoln. and in complex with a 50-base template-primer RNA and remdesivir at 2.5-angstrom resoln.  The complex structure reveals that the partial double-stranded RNA template is inserted into the central channel of the RdRp, where remdesivir is covalently incorporated into the primer strand at the first replicated base pair, and terminates chain elongation.  Our structures provide insights into the mechanism of viral RNA replication and a rational template for drug design to combat the viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlizem_f4e1LVg90H21EOLACvtfcHk0ljBXgyw9cDC8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK&md5=e5d7c8a1c27a02fdbfa9381482a0c3e8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc1560%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DLuan%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DY.%2BC.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%2BW.%26aulast%3DTao%26aufirst%3DS.%2BC.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520RNA-dependent%2520RNA%2520polymerase%2520from%2520SARS-CoV-2%2520by%2520remdesivir%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1499%26epage%3D1504%26doi%3D10.1126%2Fscience.abc1560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menachery, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralinski, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leist, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyrc, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trantcheva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babusis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spahn, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmiotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denison, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baric, R. S.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">eaal3653</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aal3653</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1126%2Fscitranslmed.aal3653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28659436" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&author=T.+P.+Sheahanauthor=A.+C.+Simsauthor=R.+L.+Grahamauthor=V.+D.+Menacheryauthor=L.+E.+Gralinskiauthor=J.+B.+Caseauthor=S.+R.+Leistauthor=K.+Pyrcauthor=J.+Y.+Fengauthor=I.+Trantchevaauthor=R.+Bannisterauthor=Y.+Parkauthor=D.+Babusisauthor=M.+O.+Clarkeauthor=R.+L.+Mackmanauthor=J.+E.+Spahnauthor=C.+A.+Palmiottiauthor=D.+Siegelauthor=A.+S.+Rayauthor=T.+Cihlarauthor=R.+Jordanauthor=M.+R.+Denisonauthor=R.+S.+Baric&title=Broad-spectrum+antiviral+GS-5734+inhibits+both+epidemic+and+zoonotic+coronaviruses&doi=10.1126%2Fscitranslmed.aal3653"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aal3653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aal3653%26sid%3Dliteratum%253Aachs%26aulast%3DSheahan%26aufirst%3DT.%2BP.%26aulast%3DSims%26aufirst%3DA.%2BC.%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DMenachery%26aufirst%3DV.%2BD.%26aulast%3DGralinski%26aufirst%3DL.%2BE.%26aulast%3DCase%26aufirst%3DJ.%2BB.%26aulast%3DLeist%26aufirst%3DS.%2BR.%26aulast%3DPyrc%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%2BY.%26aulast%3DTrantcheva%26aufirst%3DI.%26aulast%3DBannister%26aufirst%3DR.%26aulast%3DPark%26aufirst%3DY.%26aulast%3DBabusis%26aufirst%3DD.%26aulast%3DClarke%26aufirst%3DM.%2BO.%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSpahn%26aufirst%3DJ.%2BE.%26aulast%3DPalmiotti%26aufirst%3DC.%2BA.%26aulast%3DSiegel%26aufirst%3DD.%26aulast%3DRay%26aufirst%3DA.%2BS.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DJordan%26aufirst%3DR.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26atitle%3DBroad-spectrum%2520antiviral%2520GS-5734%2520inhibits%2520both%2520epidemic%2520and%2520zoonotic%2520coronaviruses%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2017%26volume%3D9%26doi%3D10.1126%2Fscitranslmed.aal3653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinnon, K. H.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denison, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baric, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, T. P.</span></span> <span> </span><span class="NLM_article-title">Broad spectrum antiviral remdesivir inhibits human endemic and zoonotic deltacoronaviruses with a highly divergent RNA dependent RNA polymerase</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">104541</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2019.104541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.antiviral.2019.104541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=31233808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Git7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2019&pages=104541&author=A.+J.+Brownauthor=J.+J.+Wonauthor=R.+L.+Grahamauthor=K.+H.+Dinnonauthor=A.+C.+Simsauthor=J.+Y.+Fengauthor=T.+Cihlarauthor=M.+R.+Denisonauthor=R.+S.+Baricauthor=T.+P.+Sheahan&title=Broad+spectrum+antiviral+remdesivir+inhibits+human+endemic+and+zoonotic+deltacoronaviruses+with+a+highly+divergent+RNA+dependent+RNA+polymerase&doi=10.1016%2Fj.antiviral.2019.104541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Broad spectrum antiviral remdesivir inhibits human endemic and zoonotic deltacoronaviruses with a highly divergent RNA dependent RNA polymerase</span></div><div class="casAuthors">Brown, Ariane J.; Won, John J.; Graham, Rachel L.; Dinnon, Kenneth H., III; Sims, Amy C.; Feng, Joy Y.; Cihlar, Tomas; Denison, Mark R.; Baric, Ralph S.; Sheahan, Timothy P.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104541</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The genetically diverse Orthocoronavirinae (CoV) family is prone to cross species transmission and disease emergence in both humans and livestock.  Viruses similar to known epidemic strains circulating in wild and domestic animals further increase the probability of emergence in the future.  Currently, there are no approved therapeutics for any human CoV presenting a clear unmet medical need.  Remdesivir (RDV, GS-5734) is a monophosphoramidate prodrug of an adenosine analog with potent activity against an array of RNA virus families including Filoviridae, Paramyxoviridae, Pneumoviridae, and Orthocoronavirinae, through the targeting of the viral RNA dependent RNA polymerase (RdRp).  We developed multiple assays to further define the breadth of RDV antiviral activity against the CoV family.  Here, we show potent antiviral activity of RDV against endemic human CoVs OC43 (HCoV-OC43) and 229E (HCoV-229E) with submicromolar EC50 values.  Of known CoVs, the members of the deltacoronavirus genus have the most divergent RdRp as compared to SARS- and MERS-CoV and both avian and porcine members harbor a native residue in the RdRp that confers resistance in beta-CoVs.  Nevertheless, RDV is highly efficacious against porcine deltacoronavirus (PDCoV).  These data further extend the known breadth and antiviral activity of RDV to include both contemporary human and highly divergent zoonotic CoV and potentially enhance our ability to fight future emerging CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZuP2J4qyxbVg90H21EOLACvtfcHk0ljBXgyw9cDC8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Git7zI&md5=83a831806f8e8520713283d35de2bd7e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2019.104541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2019.104541%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DWon%26aufirst%3DJ.%2BJ.%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DDinnon%26aufirst%3DK.%2BH.%26aulast%3DSims%26aufirst%3DA.%2BC.%26aulast%3DFeng%26aufirst%3DJ.%2BY.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26aulast%3DSheahan%26aufirst%3DT.%2BP.%26atitle%3DBroad%2520spectrum%2520antiviral%2520remdesivir%2520inhibits%2520human%2520endemic%2520and%2520zoonotic%2520deltacoronaviruses%2520with%2520a%2520highly%2520divergent%2520RNA%2520dependent%2520RNA%2520polymerase%26jtitle%3DAntiviral%2520Res.%26date%3D2019%26volume%3D169%26spage%3D104541%26doi%3D10.1016%2Fj.antiviral.2019.104541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiew, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">11754</span>– <span class="NLM_lpage">11762</span>, <span class="refDoi"> DOI: 10.1128/JVI.01348-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.01348-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=22915796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=11754-11762&author=Y.+Kimauthor=S.+Lovellauthor=K.+C.+Tiewauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=K.+P.+Battaileauthor=W.+C.+Groutasauthor=K.+O.+Chang&title=Broad-spectrum+antivirals+against+3C+or+3C-like+proteases+of+picornaviruses%2C+noroviruses%2C+and+coronaviruses&doi=10.1128%2FJVI.01348-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span></div><div class="casAuthors">Kim, Yunjeong; Lovell, Scott; Tiew, Kok-Chuan; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Battaile, Kevin P.; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11754-11762</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phylogenetic anal. has demonstrated that some pos.-sense RNA viruses can be classified into the picornavirus-like super-cluster, which includes picornaviruses, caliciviruses, and coronaviruses.  These viruses possess 3C or 3C-like proteases (3Cpro or 3CLpro, resp.), which contain a typical chymotrypsin-like fold and a catalytic triad (or dyad) with a Cys residue as a nucleophile.  The conserved key sites of 3Cpro or 3CLpro may serve as attractive targets for the design of broad-spectrum antivirals for multiple viruses in the supercluster.  We previously reported the structure-based design and synthesis of potent protease inhibitors of Norwalk virus (NV), a member of the Caliciviridae family.  We report herein the broad-spectrum antiviral activities of three compds. possessing a common dipeptidyl residue with different warheads, i.e., an aldehyde (GC373), a bisulfite adduct (GC376), and an α-ketoamide (GC375), against viruses that belong to the supercluster.  All compds. were highly effective against the majority of tested viruses, with half-maximal inhibitory concns. in the high nanomolar or low micromolar range in enzyme- and/or cell-based assays and with high therapeutic indexes.  We also report the high-resoln. X-ray cocrystal structures of NV 3CLpro-, poliovirus 3Cpro-, and transmissible gastroenteritis virus 3CLpro- GC376 inhibitor complexes, which show the compd. covalently bound to a nucleophilic Cys residue in the catalytic site of the corresponding protease.  We conclude that these compds. have the potential to be developed as antiviral therapeutics aimed at a single virus or multiple viruses in the picornavirus-like supercluster by targeting 3Cpro or 3CLpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDn4MUH7bIlLVg90H21EOLACvtfcHk0lhtZfaSp4cGuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M&md5=c7fe984b52490199a2c2ef333c22fb98</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.01348-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01348-12%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DTiew%26aufirst%3DK.%2BC.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3DBroad-spectrum%2520antivirals%2520against%25203C%2520or%25203C-like%2520proteases%2520of%2520picornaviruses%252C%2520noroviruses%252C%2520and%2520coronaviruses%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D11754%26epage%3D11762%26doi%3D10.1128%2FJVI.01348-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4282</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18096-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41467-020-18096-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32855413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4282&author=W.+Vuongauthor=M.+B.+Khanauthor=C.+Fischerauthor=E.+Arutyunovaauthor=T.+Lamerauthor=J.+Shieldsauthor=H.+A.+Saffranauthor=R.+T.+McKayauthor=M.+J.+van+Belkumauthor=M.+A.+Joyceauthor=H.+S.+Youngauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=M.+J.+Lemieux&title=Feline+coronavirus+drug+inhibits+the+main+protease+of+SARS-CoV-2+and+blocks+virus+replication&doi=10.1038%2Fs41467-020-18096-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Vuong, Wayne; Khan, Muhammad Bashir; Fischer, Conrad; Arutyunova, Elena; Lamer, Tess; Shields, Justin; Saffran, Holly A.; McKay, Ryan T.; van Belkum, Marco J.; Joyce, Michael A.; Young, Howard S.; Tyrrell, D. Lorne; Vederas, John C.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4282</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The main protease, Mpro (or 3CLpro) in SARS-CoV-2 is a viable drug target because of its essential role in the cleavage of the virus polypeptide.  Feline infectious peritonitis, a fatal coronavirus infection in cats, was successfully treated previously with a prodrug GC376, a dipeptide-based protease inhibitor.  We show the prodrug and its parent GC373, are effective inhibitors of the Mpro from both SARS-CoV and SARS-CoV-2 with IC50 values in the nanomolar range.  Crystal structures of SARS-CoV-2 Mpro with these inhibitors have a covalent modification of the nucleophilic Cys145.  NMR anal. reveals that inhibition proceeds via reversible formation of a hemithioacetal.  GC373 and GC376 are potent inhibitors of SARS-CoV-2 replication in cell culture.  They are strong drug candidates for the treatment of human coronavirus infections because they have already been successful in animals.  The work here lays the framework for their use in human trials for the treatment of COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMA7W4XtRQrbVg90H21EOLACvtfcHk0lhtZfaSp4cGuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN&md5=c06d2dc9a65ae289505fe120d41d8885</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18096-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18096-2%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DShields%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DFeline%2520coronavirus%2520drug%2520inhibits%2520the%2520main%2520protease%2520of%2520SARS-CoV-2%2520and%2520blocks%2520virus%2520replication%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4282%26doi%3D10.1038%2Fs41467-020-18096-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">Both Boceprevir and GC376 efficaciously inhibit SARS-CoV-2 by targeting its main protease</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4417</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18233-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41467-020-18233-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32887884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSmsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4417&author=L.+Fuauthor=F.+Yeauthor=Y.+Fengauthor=F.+Yuauthor=Q.+Wangauthor=Y.+Wuauthor=C.+Zhaoauthor=H.+Sunauthor=B.+Huangauthor=P.+Niuauthor=H.+Songauthor=Y.+Shiauthor=X.+Liauthor=W.+Tanauthor=J.+Qiauthor=G.+F.+Gao&title=Both+Boceprevir+and+GC376+efficaciously+inhibit+SARS-CoV-2+by+targeting+its+main+protease&doi=10.1038%2Fs41467-020-18233-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Both Boceprevir and GC376 efficaciously inhibit SARS-CoV-2 by targeting its main protease</span></div><div class="casAuthors">Fu, Lifeng; Ye, Fei; Feng, Yong; Yu, Feng; Wang, Qisheng; Wu, Yan; Zhao, Cheng; Sun, Huan; Huang, Baoying; Niu, Peihua; Song, Hao; Shi, Yi; Li, Xuebing; Tan, Wenjie; Qi, Jianxun; Gao, George Fu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4417</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: COVID-19 was declared a pandemic on March 11 by WHO, due to its great threat to global public health.  The coronavirus main protease (Mpro, also called 3CLpro) is essential for processing and maturation of the viral polyprotein, therefore recognized as an attractive drug target.  Here we show that a clin. approved anti-HCV drug, Boceprevir, and a pre-clin. inhibitor against feline infectious peritonitis (corona) virus (FIPV), GC376, both efficaciously inhibit SARS-CoV-2 in Vero cells by targeting Mpro.  Moreover, combined application of GC376 with Remdesivir, a nucleotide analog that inhibits viral RNA dependent RNA polymerase (RdRp), results in sterilizing additive effect.  Further structural anal. reveals binding of both inhibitors to the catalytically active side of SARS-CoV-2 protease Mpro as main mechanism of inhibition.  Our findings may provide crit. information for the optimization and design of more potent inhibitors against the emerging SARS-CoV-2 virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt43KcNqLbkLVg90H21EOLACvtfcHk0lhtZfaSp4cGuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSmsr3F&md5=273b11d16241499f15103885be398b23</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18233-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18233-x%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DNiu%26aufirst%3DP.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DBoth%2520Boceprevir%2520and%2520GC376%2520efficaciously%2520inhibit%2520SARS-CoV-2%2520by%2520targeting%2520its%2520main%2520protease%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4417%26doi%3D10.1038%2Fs41467-020-18233-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricagno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egloff, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulferts, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurizzo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campanacci, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cambillau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canard, B.</span></span> <span> </span><span class="NLM_article-title">Crystal structure and mechanistic determinants of SARS coronavirus nonstructural protein 15 define an endoribonuclease family</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">11892</span>– <span class="NLM_lpage">11897</span>, <span class="refDoi"> DOI: 10.1073/pnas.0601708103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1073%2Fpnas.0601708103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=16882730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVSgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=11892-11897&author=S.+Ricagnoauthor=M.+P.+Egloffauthor=R.+Ulfertsauthor=B.+Coutardauthor=D.+Nurizzoauthor=V.+Campanacciauthor=C.+Cambillauauthor=J.+Ziebuhrauthor=B.+Canard&title=Crystal+structure+and+mechanistic+determinants+of+SARS+coronavirus+nonstructural+protein+15+define+an+endoribonuclease+family&doi=10.1073%2Fpnas.0601708103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and mechanistic determinants of SARS coronavirus nonstructural protein 15 define an endoribonuclease family</span></div><div class="casAuthors">Ricagno, Stefano; Egloff, Marie-Pierre; Ulferts, Rachel; Coutard, Bruno; Nurizzo, Didier; Campanacci, Valerie; Cambillau, Christian; Ziebuhr, John; Canard, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">11892-11897</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The ≈30-kb coronavirus (+)RNA genome is replicated and transcribed by a membrane-bound replicase complex made up of 16 viral nonstructural proteins (nsp) with multiple enzymic activities.  The complex includes an RNA endonuclease that is conserved among nidoviruses but no other RNA virus, making it a genetic marker of this virus order.  Nonstructural protein 15 (nsp15) exhibits Mn2+-dependent nidovirus uridylate-specific endoribonuclease (NendoU) activity which cleaves 2'-3'-cyclic phosphates 5' to the cleaved bond.  Neither biochem. nor sequence homol. criteria allow a classification of nsp15 into existing endonuclease families.  Here, we report the crystal structure of the severe acute respiratory syndrome coronavirus nsp15 at 2.6-Å resoln.  Nsp15 exhibits a unique fold and assembles into a toric hexamer with six potentially active, peripheric catalytic sites.  The structure and the spatial arrangement of the catalytic residues into an RNase A-like active site define a sep. endonuclease family, endoU, and represent another spectacular example of convergent evolution toward an enzymic function that is critically involved in the coronavirus replication cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UQQC4Vhe4LVg90H21EOLACvtfcHk0li3i3Dln-J6JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVSgsLs%253D&md5=de25bd173a87e86e5bf7d175d6e12977</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0601708103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0601708103%26sid%3Dliteratum%253Aachs%26aulast%3DRicagno%26aufirst%3DS.%26aulast%3DEgloff%26aufirst%3DM.%2BP.%26aulast%3DUlferts%26aufirst%3DR.%26aulast%3DCoutard%26aufirst%3DB.%26aulast%3DNurizzo%26aufirst%3DD.%26aulast%3DCampanacci%26aufirst%3DV.%26aulast%3DCambillau%26aufirst%3DC.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DCanard%26aufirst%3DB.%26atitle%3DCrystal%2520structure%2520and%2520mechanistic%2520determinants%2520of%2520SARS%2520coronavirus%2520nonstructural%2520protein%252015%2520define%2520an%2520endoribonuclease%2520family%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D11892%26epage%3D11897%26doi%3D10.1073%2Fpnas.0601708103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Crystallization and preliminary X-ray crystallographic analysis of a nonstructural protein 15 mutant from Human coronavirus 229E</span>. <i>Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1156</span>– <span class="NLM_lpage">1160</span>, <span class="refDoi"> DOI: 10.1107/S2053230X15007359</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1107%2FS2053230X15007359" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVymt7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1156-1160&author=T.+Huoauthor=X.+Liu&title=Crystallization+and+preliminary+X-ray+crystallographic+analysis+of+a+nonstructural+protein+15+mutant+from+Human+coronavirus+229E&doi=10.1107%2FS2053230X15007359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallization and preliminary x-ray crystallographic analysis of a nonstructural protein 15 mutant from human coronavirus 229E</span></div><div class="casAuthors">Huo, Tong; Liu, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1156-1160</span>CODEN:
                <span class="NLM_cas:coden">ACSFEN</span>;
        ISSN:<span class="NLM_cas:issn">2053-230X</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Nonstructural protein 15 (nsp15), also called Nsp15 endoribonuclease, is a gene product of open reading frame 1b (ORF 1b) in coronaviruses.  It is an important enzyme in the transcription/replication process involved in discontinuous neg.-strand RNA synthesis.  Here, mutants of nsp15 from human coronavirus 229E (HCoV-229E) were made based on structural anal. of the homologous nsp15 proteins in severe acute respiratory syndrome coronavirus (SARS-CoV) and mouse hepatitis virus (MHV).  The I26A/N52A mutant of nsp15 was overexpressed, purified, and crystd., and this mutant led to a trimeric form rather than hexamers or monomers.  Crystals of trimeric nsp15 were obtained by the hanging-drop vapor-diffusion method using polyethylene glycol as a precipitant and diffracted to 2.5 Å resoln.  The crystals belonged to space group C2221, with unit-cell parameters a = 85.9, b = 137.5, c = 423.1 Å, and α = β = γ = 90°.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXjlzkzcNBbrVg90H21EOLACvtfcHk0li3i3Dln-J6JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVymt7fN&md5=1eb0673f34363cf2c0ccd24991ed4133</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1107%2FS2053230X15007359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053230X15007359%26sid%3Dliteratum%253Aachs%26aulast%3DHuo%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DCrystallization%2520and%2520preliminary%2520X-ray%2520crystallographic%2520analysis%2520of%2520a%2520nonstructural%2520protein%252015%2520mutant%2520from%2520Human%2520coronavirus%2520229E%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520F%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2015%26volume%3D71%26spage%3D1156%26epage%3D1160%26doi%3D10.1107%2FS2053230X15007359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrzejczak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maltseva, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilamowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/pro.3873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1002%2Fpro.3873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32304108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosVWrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2020&pages=1596-1605&author=Y.+Kimauthor=R.+Jedrzejczakauthor=N.+I.+Maltsevaauthor=M.+Wilamowskiauthor=M.+Endresauthor=A.+Godzikauthor=K.+Michalskaauthor=A.+Joachimiak&title=Crystal+structure+of+Nsp15+endoribonuclease+NendoU+from+SARS-CoV-2&doi=10.1002%2Fpro.3873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</span></div><div class="casAuthors">Kim, Youngchang; Jedrzejczak, Robert; Maltseva, Natalia I.; Wilamowski, Mateusz; Endres, Michael; Godzik, Adam; Michalska, Karolina; Joachimiak, Andrzej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1596-1605</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) is rapidly spreading around the world.  There is no existing vaccine or proven drug to prevent infections and stop virus proliferation.  Although this virus is similar to human and animal SARS-CoVs and Middle East Respiratory Syndrome coronavirus (MERS-CoVs), the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies, and antivirals.  We applied high-throughput protein prodn. and structure detn. pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures.  Here we report two high-resoln. crystal structures of endoribonuclease Nsp15/NendoU.  We compare these structures with previously reported homologs from SARS and MERS coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7HBZSCjMXGrVg90H21EOLACvtfcHk0li3i3Dln-J6JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosVWrsbo%253D&md5=28cfa012064ca5416af49d55d53b1b46</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fpro.3873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3873%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DJedrzejczak%26aufirst%3DR.%26aulast%3DMaltseva%26aufirst%3DN.%2BI.%26aulast%3DWilamowski%26aufirst%3DM.%26aulast%3DEndres%26aufirst%3DM.%26aulast%3DGodzik%26aufirst%3DA.%26aulast%3DMichalska%26aufirst%3DK.%26aulast%3DJoachimiak%26aufirst%3DA.%26atitle%3DCrystal%2520structure%2520of%2520Nsp15%2520endoribonuclease%2520NendoU%2520from%2520SARS-CoV-2%26jtitle%3DProtein%2520Sci.%26date%3D2020%26volume%3D29%26spage%3D1596%26epage%3D1605%26doi%3D10.1002%2Fpro.3873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">New antiviral target revealed by the hexameric structure of mouse hepatitis virus nonstructural protein nsp15</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">7909</span>– <span class="NLM_lpage">7917</span>, <span class="refDoi"> DOI: 10.1128/JVI.00525-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.00525-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=16873248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1yrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2006&pages=7909-7917&author=X.+Xuauthor=Y.+Zhaiauthor=F.+Sunauthor=Z.+Louauthor=D.+Suauthor=Y.+Xuauthor=R.+Zhangauthor=A.+Joachimiakauthor=X.+C.+Zhangauthor=M.+Bartlamauthor=Z.+Rao&title=New+antiviral+target+revealed+by+the+hexameric+structure+of+mouse+hepatitis+virus+nonstructural+protein+nsp15&doi=10.1128%2FJVI.00525-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">New antiviral target revealed by the hexameric structure of mouse hepatitis virus nonstructural protein nsp15</span></div><div class="casAuthors">Xu, Xiaoling; Zhai, Yujia; Sun, Fei; Lou, Zhiyong; Su, Dan; Xu, Yuanyuan; Zhang, Rongguang; Joachimiak, Andrzej; Zhang, Xuejun C.; Bartlam, Mark; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7909-7917</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The unique coronavirus transcription/replication machinery comprised of multiple virus-encoded nonstructural proteins (nsp) plays a vital role during initial and intermediate phases of the viral life cycle.  The crystal structure of mouse hepatitis virus strain A59 (MHV-A59) nsp15 is reported at 2.15-Å resoln.  Nsp15 is an XendoU endoribonuclease and is the first one from this family to have its structure unveiled.  The MHV-A59 nsp15 monomer structure has a novel protein fold.  Two nsp15 trimers form a back-to-back hexamer that is believed to be the functional unit.  The structure reveals the catalytic site including the highly conserved residues His262, His277, and Lys317, which is supported by mutagenesis anal.  Gel filtration and enzyme activity assays confirmed that the hexamer is the active form for nsp15 and demonstrate the specificity of nsp15 for uridylate.  The high sequence conservation of nsp15 in coronaviruses, including that of severe acute respiratory syndrome, suggests that this protein may provide a new target for the design of antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdhBRvFvNjg7Vg90H21EOLACvtfcHk0lhWMe5fHPMukg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1yrtrw%253D&md5=a955ca6de0725a654ed8a67ceb1cfbd2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FJVI.00525-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00525-06%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%2BC.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DNew%2520antiviral%2520target%2520revealed%2520by%2520the%2520hexameric%2520structure%2520of%2520mouse%2520hepatitis%2520virus%2520nonstructural%2520protein%2520nsp15%26jtitle%3DJ.%2520Virol.%26date%3D2006%26volume%3D80%26spage%3D7909%26epage%3D7917%26doi%3D10.1128%2FJVI.00525-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Structural and biochemical characterization of endoribonuclease Nsp15 encoded by Middle East Respiratory Syndrome Coronavirus</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">e00893</span>– <span class="NLM_lpage">e00918</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.00893-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=30135128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFemsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2018&pages=e00893-e00918&author=L.+Zhangauthor=L.+Liauthor=L.+Yanauthor=Z.+Mingauthor=Z.+Jiaauthor=Z.+Louauthor=Z.+Rao&title=Structural+and+biochemical+characterization+of+endoribonuclease+Nsp15+encoded+by+Middle+East+Respiratory+Syndrome+Coronavirus"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biochemical characterization of endoribonuclease nsp15 encoded by middle east respiratory syndrome coronavirus</span></div><div class="casAuthors">Zhang, Lianqi; Li, Lei; Yan, Liming; Ming, Zhenhua; Jia, Zhihui; Lou, Zhiyong; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">e00893-18/1-e00893-18/16</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nonstructural protein 15 (Nsp15) encoded by coronavirus (CoV) is a nidoviral uridylate-specific endoribonuclease (NendoU) that plays an essential role in the life cycle of the virus.  Structural information on this crucial protein from the Middle East respiratory syndrome CoV (MERS-CoV), which is lethally pathogenic and has caused severe respiratory diseases worldwide, is lacking.  Here, we detd. the crystal structure of MERS-CoV Nsp15 at a 2.7-Å resoln. and performed the relevant biochem. assays to study how NendoU activity is regulated.  Although the overall structure is conserved, MERS-CoV Nsp15 shows unique and novel features compared to its homologs.  Serine substitution of residue F285, which harbors an arom. side chain that disturbs RNA binding compared with that of other homologs, increases catalytic activity.  Mutations of residues residing on the oligomerization interfaces that distort hexamerization, namely, N38A, Y58A, and N157A, decrease thermostability, decrease affinity of binding with RNA, and reduce the NendoU activity of Nsp15.  In contrast, mutant D39A exhibits increased activity and a higher substrate binding capacity.  Importantly, Nsp8 was found to interact with both monomeric and hexameric Nsp15.  The Nsp7/Nsp8 complex displays a higher binding affinity for Nsp15.  Furthermore, Nsp8 and the Nsp7/Nsp8 complex also enhance the NendoU activity of hexameric Nsp15 in vitro.  Taking the findings together, this work first provides evidence on how the activity of Nsp15 may be functionally mediated by catalytic residues, oligomeric assembly, RNA binding efficiency, or the possible assocn. with other nonstructural proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdCS6GYJAivLVg90H21EOLACvtfcHk0lhWMe5fHPMukg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFemsbo%253D&md5=e1379df18eefcb703a34d2acb34eb7fc</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FJVI.00893-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00893-18%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DMing%26aufirst%3DZ.%26aulast%3DJia%26aufirst%3DZ.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DStructural%2520and%2520biochemical%2520characterization%2520of%2520endoribonuclease%2520Nsp15%2520encoded%2520by%2520Middle%2520East%2520Respiratory%2520Syndrome%2520Coronavirus%26jtitle%3DJ.%2520Virol.%26date%3D2018%26volume%3D92%26spage%3De00893%26epage%3De00918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sevajol, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subissi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decroly, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbert, I.</span></span> <span> </span><span class="NLM_article-title">Insights into RNA synthesis, capping, and proofreading mechanisms of SARS-coronavirus</span>. <i>Virus Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.virusres.2014.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.virusres.2014.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=25451065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ltrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2014&pages=90-99&author=M.+Sevajolauthor=L.+Subissiauthor=E.+Decrolyauthor=B.+Canardauthor=I.+Imbert&title=Insights+into+RNA+synthesis%2C+capping%2C+and+proofreading+mechanisms+of+SARS-coronavirus&doi=10.1016%2Fj.virusres.2014.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into RNA synthesis, capping, and proofreading mechanisms of SARS-coronavirus</span></div><div class="casAuthors">Sevajol, Marion; Subissi, Lorenzo; Decroly, Etienne; Canard, Bruno; Imbert, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">90-99</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The successive emergence of highly pathogenic coronaviruses (CoVs) such as the Severe Acute Respiratory Syndrome (SARS-CoV) in 2003 and the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) in 2012 has stimulated a no. of studies on the mol. biol.  This research has provided significant new insight into functions and activities of the replication/transcription multi-protein complex.  The latter directs both continuous and discontinuous RNA synthesis to replicate and transcribe the large coronavirus genome made of a single-stranded, pos.-sense RNA of ∼30 kb.  In this review, we summarize our current understanding of SARS-CoV enzymes involved in RNA biochem., such as the in vitro characterization of a highly active and processive RNA polymerase complex which can assoc. with methyltransferase and 3'-5' exoribonuclease activities involved in RNA capping, and RNA proofreading, resp.  The recent discoveries reveal fascinating RNA-synthesizing machinery, highlighting the unique position of coronaviruses in the RNA virus world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkfBObzAjd3LVg90H21EOLACvtfcHk0lhWMe5fHPMukg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ltrzK&md5=1774c59cdf5e3b207839f973feb2197c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.virusres.2014.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virusres.2014.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DSevajol%26aufirst%3DM.%26aulast%3DSubissi%26aufirst%3DL.%26aulast%3DDecroly%26aufirst%3DE.%26aulast%3DCanard%26aufirst%3DB.%26aulast%3DImbert%26aufirst%3DI.%26atitle%3DInsights%2520into%2520RNA%2520synthesis%252C%2520capping%252C%2520and%2520proofreading%2520mechanisms%2520of%2520SARS-coronavirus%26jtitle%3DVirus%2520Res.%26date%3D2014%26volume%3D194%26spage%3D90%26epage%3D99%26doi%3D10.1016%2Fj.virusres.2014.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">V’Kovski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratzel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span> <span> </span><span class="NLM_article-title">Coronavirus biology and replication: implications for SARS-CoV-2</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/s41579-020-00468-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41579-020-00468-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33116300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1GjsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2021&pages=155-170&author=P.+V%E2%80%99Kovskiauthor=A.+Kratzelauthor=S.+Steinerauthor=H.+Stalderauthor=V.+Thiel&title=Coronavirus+biology+and+replication%3A+implications+for+SARS-CoV-2&doi=10.1038%2Fs41579-020-00468-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus biology and replication: implications for SARS-CoV-2</span></div><div class="casAuthors">V'kovski, Philip; Kratzel, Annika; Steiner, Silvio; Stalder, Hanspeter; Thiel, Volker</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">155-170</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Abstr.: The SARS-CoV-2 pandemic and its unprecedented global societal and economic disruptive impact has marked the third zoonotic introduction of a highly pathogenic coronavirus into the human population.  Although the previous coronavirus SARS-CoV and MERS-CoV epidemics raised awareness of the need for clin. available therapeutic or preventive interventions, to date, no treatments with proven efficacy are available.  The development of effective intervention strategies relies on the knowledge of mol. and cellular mechanisms of coronavirus infections, which highlights the significance of studying virus-host interactions at the mol. level to identify targets for antiviral intervention and to elucidate crit. viral and host determinants that are decisive for the development of severe disease.  In this Review, we summarize the first discoveries that shape our current understanding of SARS-CoV-2 infection throughout the intracellular viral life cycle and relate that to our knowledge of coronavirus biol.  The elucidation of similarities and differences between SARS-CoV-2 and other coronaviruses will support future preparedness and strategies to combat coronavirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNkjU36gq1BLVg90H21EOLACvtfcHk0liHLcFJK3cn3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1GjsLjF&md5=14ddf54f3e569fe39eb4f3295cf603e6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41579-020-00468-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41579-020-00468-6%26sid%3Dliteratum%253Aachs%26aulast%3DV%25E2%2580%2599Kovski%26aufirst%3DP.%26aulast%3DKratzel%26aufirst%3DA.%26aulast%3DSteiner%26aufirst%3DS.%26aulast%3DStalder%26aufirst%3DH.%26aulast%3DThiel%26aufirst%3DV.%26atitle%3DCoronavirus%2520biology%2520and%2520replication%253A%2520implications%2520for%2520SARS-CoV-2%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2021%26volume%3D19%26spage%3D155%26epage%3D170%26doi%3D10.1038%2Fs41579-020-00468-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackbart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettelman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mielech, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">Coronavirus nonstructural protein 15 mediates evasion of dsRNA sensors and limits apoptosis in macrophages</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E4251</span>– <span class="NLM_lpage">E4260</span>, <span class="refDoi"> DOI: 10.1073/pnas.1618310114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1073%2Fpnas.1618310114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28484023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Clu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E4251-E4260&author=X.+Dengauthor=M.+Hackbartauthor=R.+C.+Mettelmanauthor=A.+O%E2%80%99Brienauthor=A.+M.+Mielechauthor=G.+Yiauthor=C.+C.+Kaoauthor=S.+C.+Baker&title=Coronavirus+nonstructural+protein+15+mediates+evasion+of+dsRNA+sensors+and+limits+apoptosis+in+macrophages&doi=10.1073%2Fpnas.1618310114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus nonstructural protein 15 mediates evasion of dsRNA sensors and limits apoptosis in macrophages</span></div><div class="casAuthors">Deng, Xufang; Hackbart, Matthew; Mettelman, Robert C.; O'Brien, Amornrat; Mielech, Anna M.; Yi, Guanghui; Kao, C. Cheng; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">E4251-E4260</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Coronaviruses are pos.-sense RNA viruses that generate double-stranded RNA (dsRNA) intermediates during replication, yet evade detection by host innate immune sensors.  Here we report that coronavirus nonstructural protein 15 (nsp15), an endoribonuclease, is required for evasion of dsRNA sensors.  We evaluated two independent nsp15 mutant mouse coronaviruses, designated N15m1 and N15m3, and found that these viruses replicated poorly and induced rapid cell death in mouse bone marrow-derived macrophages.  Infection of macrophages with N15m1, which expresses an unstable nsp15, or N15m3, which expresses a catalysis-deficient nsp15, activated MDA5, PKR, and the OAS/RNase L system, resulting in an early, robust induction of type I IFN, PKR-mediated apoptosis, and RNA degrdn.  Immunofluorescence imaging of nsp15 mutant virus-infected macrophages revealed significant dispersal of dsRNA early during infection, whereas in WT virus-infected cells, the majority of the dsRNA was assocd. with replication complexes.  The loss of nsp15 activity also resulted in greatly attenuated disease in mice and stimulated a protective immune response.  Taken together, our findings demonstrate that coronavirus nsp15 is crit. for evasion of host dsRNA sensors in macrophages and reveal that modulating nsp15 stability and activity is a strategy for generating live-attenuated vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXhXNthEjvxLVg90H21EOLACvtfcHk0liHLcFJK3cn3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Clu7s%253D&md5=56fae15b7e25d83df1f04c257f9c6ab2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1618310114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1618310114%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DHackbart%26aufirst%3DM.%26aulast%3DMettelman%26aufirst%3DR.%2BC.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DA.%26aulast%3DMielech%26aufirst%3DA.%2BM.%26aulast%3DYi%26aufirst%3DG.%26aulast%3DKao%26aufirst%3DC.%2BC.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DCoronavirus%2520nonstructural%2520protein%252015%2520mediates%2520evasion%2520of%2520dsRNA%2520sensors%2520and%2520limits%2520apoptosis%2520in%2520macrophages%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE4251%26epage%3DE4260%26doi%3D10.1073%2Fpnas.1618310114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabouw, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Züst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">V’Kovski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habjan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervantes-Barragan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaughan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Kuppeveld, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludewig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalinke, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span> <span> </span><span class="NLM_article-title">Early endonuclease-mediated evasion of RNA sensing ensures efficient coronavirus replication</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1006195</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1006195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.ppat.1006195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28158275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=E.+Kindlerauthor=C.+Gil-Cruzauthor=J.+Spanierauthor=Y.+Liauthor=J.+Wilhelmauthor=H.+H.+Rabouwauthor=R.+Z%C3%BCstauthor=M.+Hwangauthor=P.+V%E2%80%99Kovskiauthor=H.+Stalderauthor=S.+Martiauthor=M.+Habjanauthor=L.+Cervantes-Barraganauthor=R.+Elliotauthor=N.+Karlauthor=C.+Gaughanauthor=F.+J.+van+Kuppeveldauthor=R.+H.+Silvermanauthor=M.+Kellerauthor=B.+Ludewigauthor=C.+C.+Bergmannauthor=J.+Ziebuhrauthor=S.+R.+Weissauthor=U.+Kalinkeauthor=V.+Thiel&title=Early+endonuclease-mediated+evasion+of+RNA+sensing+ensures+efficient+coronavirus+replication&doi=10.1371%2Fjournal.ppat.1006195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Early endonuclease-mediated evasion of RNA sensing ensures efficient coronavirus replication</span></div><div class="casAuthors">Kindler, Eveline; Gil-Cruz, Cristina; Spanier, Julia; Li, Yize; Wilhelm, Jochen; Rabouw, Huib H.; Zust, Roland; Hwang, Mihyun; V'kovski, Philip; Stalder, Hanspeter; Marti, Sabrina; Habjan, Matthias; Cervantes-Barragan, Luisa; Elliot, Ruth; Karl, Nadja; Gaughan, Christina; van Kuppeveld, Frank J. M.; Silverman, Robert H.; Keller, Markus; Ludewig, Burkhard; Bergmann, Cornelia C.; Ziebuhr, John; Weiss, Susan R.; Kalinke, Ulrich; Thiel, Volker</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1006195/1-e1006195/26</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses are of veterinary and medical importance and include highly pathogenic zoonotic viruses, such as SARS-CoV and MERS-CoV.  They are known to efficiently evade early innate immune responses, manifesting in almost negligible expression of type-I interferons (IFN-I).  This evasion strategy suggests an evolutionary conserved viral function that has evolved to prevent RNA-based sensing of infection in vertebrate hosts.  Here we show that the coronavirus endonuclease (EndoU) activity is key to prevent early induction of double-stranded RNA (dsRNA) host cell responses.  Replication of EndoU-deficient coronaviruses is greatly attenuated in vivo and severely restricted in primary cells even during the early phase of the infection.  In macrophages we found immediate induction of IFN-I expression and RNase L-mediated breakdown of rRNA.  Accordingly, EndoU-deficient viruses can retain replication only in cells that are deficient in IFN-I expression or sensing, and in cells lacking both RNase L and PKR.  Collectively our results demonstrate that the coronavirus EndoU efficiently prevents simultaneous activation of host cell dsRNA sensors, such as Mda5, OAS and PKR.  The localization of the EndoU activity at the site of viral RNA synthesis-within the replicase complex-suggests that coronaviruses have evolved a viral RNA decay pathway to evade early innate and intrinsic antiviral host cell responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJihv7-Ahq6rVg90H21EOLACvtfcHk0lhDog001M3UnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCqtrvP&md5=12cd5c885b2b147fe10fe55fc5ccd19c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006195%26sid%3Dliteratum%253Aachs%26aulast%3DKindler%26aufirst%3DE.%26aulast%3DGil-Cruz%26aufirst%3DC.%26aulast%3DSpanier%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWilhelm%26aufirst%3DJ.%26aulast%3DRabouw%26aufirst%3DH.%2BH.%26aulast%3DZ%25C3%25BCst%26aufirst%3DR.%26aulast%3DHwang%26aufirst%3DM.%26aulast%3DV%25E2%2580%2599Kovski%26aufirst%3DP.%26aulast%3DStalder%26aufirst%3DH.%26aulast%3DMarti%26aufirst%3DS.%26aulast%3DHabjan%26aufirst%3DM.%26aulast%3DCervantes-Barragan%26aufirst%3DL.%26aulast%3DElliot%26aufirst%3DR.%26aulast%3DKarl%26aufirst%3DN.%26aulast%3DGaughan%26aufirst%3DC.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%26aulast%3DSilverman%26aufirst%3DR.%2BH.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DLudewig%26aufirst%3DB.%26aulast%3DBergmann%26aufirst%3DC.%2BC.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DS.%2BR.%26aulast%3DKalinke%26aufirst%3DU.%26aulast%3DThiel%26aufirst%3DV.%26atitle%3DEarly%2520endonuclease-mediated%2520evasion%2520of%2520RNA%2520sensing%2520ensures%2520efficient%2520coronavirus%2520replication%26jtitle%3DPLoS%2520Pathog.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.ppat.1006195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hackbart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">Coronavirus endoribonuclease targets viral polyuridine sequences to evade activating host sensors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">8094</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1073/pnas.1921485117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1073%2Fpnas.1921485117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32198201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXms1ygt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=8094-8103&author=M.+Hackbartauthor=X.+Dengauthor=S.+C.+Baker&title=Coronavirus+endoribonuclease+targets+viral+polyuridine+sequences+to+evade+activating+host+sensors&doi=10.1073%2Fpnas.1921485117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus endoribonuclease targets viral polyuridine sequences to evade activating host sensors</span></div><div class="casAuthors">Hackbart, Matthew; Deng, Xufang; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">8094-8103</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Coronaviruses (CoVs) are pos.-sense RNA viruses that can emerge from endemic reservoirs and infect zoonotically, causing significant morbidity and mortality.  CoVs encode an endoribonuclease designated EndoU that facilitates evasion of host pattern recognition receptor MDA5, but the target of EndoU activity was not known.  Here, we report that EndoU cleaves the 5'-polyuridines from neg.-sense viral RNA, termed PUN RNA, which is the product of polyA-templated RNA synthesis.  Using a virus contg. an EndoU catalytic-inactive mutation, we detected a higher abundance of PUN RNA in the cytoplasm compared to wild-type-infected cells.  Furthermore, we found that transfecting PUN RNA into cells stimulates a robust, MDA5-dependent interferon response, and that removal of the polyuridine extension on the RNA dampens the response.  Overall, the results of this study reveal the PUN RNA to be a CoV MDA5-dependent pathogen-assocd. mol. pattern (PAMP).  We also establish a mechanism for EndoU activity to cleave and limit the accumulation of this PAMP.  Since EndoU activity is highly conserved in all CoVs, inhibiting this activity may serve as an approach for therapeutic interventions against existing and emerging CoV infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCgKILpCAxeLVg90H21EOLACvtfcHk0lhDog001M3UnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXms1ygt74%253D&md5=0f94b3fbcae5750168155855eed494a7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1921485117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1921485117%26sid%3Dliteratum%253Aachs%26aulast%3DHackbart%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DCoronavirus%2520endoribonuclease%2520targets%2520viral%2520polyuridine%2520sequences%2520to%2520evade%2520activating%2520host%2520sensors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D8094%26epage%3D8103%26doi%3D10.1073%2Fpnas.1921485117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ancar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ransom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesselberth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, D. J.</span></span> <span> </span><span class="NLM_article-title">Physiologic RNA targets and refined sequence specificity of coronavirus EndoU</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1976</span>– <span class="NLM_lpage">1999</span>, <span class="refDoi"> DOI: 10.1261/rna.076604.120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1261%2Frna.076604.120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32989044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3MXktFart70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=1976-1999&author=R.+Ancarauthor=Y.+Liauthor=E.+Kindlerauthor=D.+A.+Cooperauthor=M.+Ransomauthor=V.+Thielauthor=S.+R.+Weissauthor=J.+R.+Hesselberthauthor=D.+J.+Barton&title=Physiologic+RNA+targets+and+refined+sequence+specificity+of+coronavirus+EndoU&doi=10.1261%2Frna.076604.120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Physiologic RNA targets and refined sequence specificity of coronavirus EndoU</span></div><div class="casAuthors">Ancar, Rachel; Li, Yize; Kindler, Eveline; Cooper, Daphne A.; Ransom, Monica; Thiel, Volker; Weiss, Susan R.; Hesselberth, Jay R.; Barton, David J.</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1976-1999</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Coronavirus EndoU inhibits dsRNA-activated antiviral responses; however, the physiol. RNA substrates of EndoU are unknown.  In this study, we used mouse hepatitis virus (MHV)-infected bone marrow-derived macrophage (BMM) and cyclic phosphate cDNA sequencing to identify the RNA targets of EndoU.  EndoU targeted viral RNA, cleaving the 3' side of pyrimidines with a strong preference for U↓A and C↓A sequences (endoY↓A).  EndoU-dependent cleavage was detected in every region of MHV RNA, from the 5' NTR to the 3' NTR, including transcriptional regulatory sequences (TRS).  Cleavage at two CA dinucleotides immediately adjacent to the MHV poly(A) tail suggests a mechanism to suppress neg.-strand RNA synthesis and the accumulation of viral dsRNA.  MHV with EndoU (EndoUmut) or 2'-5' phosphodiesterase (PDEmut) mutations provoked the activation of RNase L in BMM, with corresponding cleavage of RNAs by RNase L.  The physiol. targets of EndoU are viral RNA templates required for neg.-strand RNA synthesis and dsRNA accumulation.  Coronavirus EndoU cleaves U↓A and C↓A sequences (endoY↓A) within viral (+) strand RNA to evade dsRNA-activated host responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNE7gc4bUh4LVg90H21EOLACvtfcHk0lhDog001M3UnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXktFart70%253D&md5=3d8918d5c8544ad4967efd183a1a3f0b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1261%2Frna.076604.120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.076604.120%26sid%3Dliteratum%253Aachs%26aulast%3DAncar%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKindler%26aufirst%3DE.%26aulast%3DCooper%26aufirst%3DD.%2BA.%26aulast%3DRansom%26aufirst%3DM.%26aulast%3DThiel%26aufirst%3DV.%26aulast%3DWeiss%26aufirst%3DS.%2BR.%26aulast%3DHesselberth%26aufirst%3DJ.%2BR.%26aulast%3DBarton%26aufirst%3DD.%2BJ.%26atitle%3DPhysiologic%2520RNA%2520targets%2520and%2520refined%2520sequence%2520specificity%2520of%2520coronavirus%2520EndoU%26jtitle%3DRNA%26date%3D2020%26volume%3D26%26spage%3D1976%26epage%3D1999%26doi%3D10.1261%2Frna.076604.120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">An ″Old″ protein with a new story: Coronavirus endoribonuclease is important for evading host antiviral defenses</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>517</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">163</span>, <span class="refDoi"> DOI: 10.1016/j.virol.2017.12.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.virol.2017.12.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=29307596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvFGqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2018&pages=157-163&author=X.+Dengauthor=S.+C.+Baker&title=An+%E2%80%B3Old%E2%80%B3+protein+with+a+new+story%3A+Coronavirus+endoribonuclease+is+important+for+evading+host+antiviral+defenses&doi=10.1016%2Fj.virol.2017.12.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">An "Old" protein with a new story: Coronavirus endoribonuclease is important for evading host antiviral defenses</span></div><div class="casAuthors">Deng, Xufang; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">157-163</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Here we review the evolving story of the coronavirus endoribonuclease (EndoU).  Coronavirus EndoU is encoded within the sequence of nonstructural protein (nsp) 15, which was initially identified as a component of the viral replication complex.  Biochem. and structural studies revealed the enzymic nature of nsp15/EndoU, which was postulated to be essential for the unique replication cycle of viruses in the order Nidovirales.  However, the role of nsp15 in coronavirus replication was enigmatic as EndoU-deficient coronaviruses were viable and replicated to near wild-type virus levels in fibroblast cells.  A breakthrough in our understanding of the role of EndoU was revealed in recent studies, which showed that EndoU mediates the evasion of viral double-stranded RNA recognition by host sensors in macrophages.  This new discovery of nsp15/EndoU function leads to new opportunities for investigating how a viral EndoU contributes to pathogenesis and exploiting this enzyme for therapeutics and vaccine design against pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkT3Mo_-SEMrVg90H21EOLACvtfcHk0lhDog001M3UnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvFGqtA%253D%253D&md5=ae511601ecd3b7f592ef90f9e28799d7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2017.12.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2017.12.024%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DAn%2520%25E2%2580%25B3Old%25E2%2580%25B3%2520protein%2520with%2520a%2520new%2520story%253A%2520Coronavirus%2520endoribonuclease%2520is%2520important%2520for%2520evading%2520host%2520antiviral%2520defenses%26jtitle%3DVirology%26date%3D2018%26volume%3D517%26spage%3D157%26epage%3D163%26doi%3D10.1016%2Fj.virol.2017.12.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kok, K. H.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 nsp13, nsp14, nsp15 and orf6 function as potent interferon antagonists</span>. <i>Emerging Microbes Infect.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1418</span>– <span class="NLM_lpage">1428</span>, <span class="refDoi"> DOI: 10.1080/22221751.2020.1780953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1080%2F22221751.2020.1780953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32529952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Sls7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1418-1428&author=C.+K.+Yuenauthor=J.+Y.+Lamauthor=W.+M.+Wongauthor=L.+F.+Makauthor=X.+Wangauthor=H.+Chuauthor=J.+P.+Caiauthor=D.+Y.+Jinauthor=K.+K.+Toauthor=J.+F.+Chanauthor=K.+Y.+Yuenauthor=K.+H.+Kok&title=SARS-CoV-2+nsp13%2C+nsp14%2C+nsp15+and+orf6+function+as+potent+interferon+antagonists&doi=10.1080%2F22221751.2020.1780953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 nsp13, nsp14, nsp15 and orf6 function as potent interferon antagonists</span></div><div class="casAuthors">Yuen, Chun-Kit; Lam, Joy-Yan; Wong, Wan-Man; Mak, Long-Fung; Wang, Xiaohui; Chu, Hin; Cai, Jian-Piao; Jin, Dong-Yan; To, Kelvin Kai-Wang; Chan, Jasper Fuk-Woo; Yuen, Kwok-Yung; Kok, Kin-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Microbes & Infections</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1418-1428</span>CODEN:
                <span class="NLM_cas:coden">EMIMC4</span>;
        ISSN:<span class="NLM_cas:issn">2222-1751</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The Coronavirus disease 2019 (COVID-19), which is caused by the novel SARS-CoV-2 virus, is now causing a tremendous global health concern.  Since its first appearance in Dec. 2019, the outbreak has already caused over 5.8 million infections worldwide (till 29 May 2020), with more than 0.35 million deaths.  Early virus-mediated immune suppression is believed to be one of the unique characteristics of SARS-CoV-2 infection and contributes at least partially to the viral pathogenesis.  In this study, we identified the key viral interferon antagonists of SARS-CoV-2 and compared them with two well-characterized SARS-CoV interferon antagonists, PLpro and orf6.  Here we demonstrated that the SARS-CoV-2 nsp13, nsp14, nsp15 and orf6, but not the unique orf8, could potently suppress primary interferon prodn. and interferon signaling.  Although SARS-CoV PLpro has been well-characterized for its potent interferon-antagonizing, deubiquitinase and protease activities, SARS-CoV-2 PLpro, despite sharing high amino acid sequence similarity with SARS-CoV, loses both interferon-antagonising and deubiquitinase activities.  Among the 27 viral proteins, SARS-CoV-2 orf6 demonstrated the strongest suppression on both primary interferon prodn. and interferon signaling.  Orf6-deleted SARS-CoV-2 may be considered for the development of intranasal live-but-attenuated vaccine against COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsdfWiicWnMrVg90H21EOLACvtfcHk0liBO6_cPbsu0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Sls7bE&md5=51cecf3ed6cae2cca0316fc62b98ae6b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1080%2F22221751.2020.1780953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F22221751.2020.1780953%26sid%3Dliteratum%253Aachs%26aulast%3DYuen%26aufirst%3DC.%2BK.%26aulast%3DLam%26aufirst%3DJ.%2BY.%26aulast%3DWong%26aufirst%3DW.%2BM.%26aulast%3DMak%26aufirst%3DL.%2BF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%2BP.%26aulast%3DJin%26aufirst%3DD.%2BY.%26aulast%3DTo%26aufirst%3DK.%2BK.%26aulast%3DChan%26aufirst%3DJ.%2BF.%26aulast%3DYuen%26aufirst%3DK.%2BY.%26aulast%3DKok%26aufirst%3DK.%2BH.%26atitle%3DSARS-CoV-2%2520nsp13%252C%2520nsp14%252C%2520nsp15%2520and%2520orf6%2520function%2520as%2520potent%2520interferon%2520antagonists%26jtitle%3DEmerging%2520Microbes%2520Infect.%26date%3D2020%26volume%3D9%26spage%3D1418%26epage%3D1428%26doi%3D10.1080%2F22221751.2020.1780953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansbach, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgartner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ba, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieh, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span> <span> </span><span class="NLM_article-title">Modulation of polypeptide conformation through donor-acceptor transformation of side-chain hydrogen bonding ligands</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">92</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-00079-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41467-017-00079-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28733648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A280%3ADC%252BC1cfgtFeksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=92&author=Z.+Songauthor=R.+A.+Mansbachauthor=H.+Heauthor=K.+C.+Shihauthor=R.+Baumgartnerauthor=N.+Zhengauthor=X.+Baauthor=Y.+Huangauthor=D.+Maniauthor=Y.+Liuauthor=Y.+Linauthor=M.+P.+Niehauthor=A.+L.+Fergusonauthor=L.+Yinauthor=J.+Cheng&title=Modulation+of+polypeptide+conformation+through+donor-acceptor+transformation+of+side-chain+hydrogen+bonding+ligands&doi=10.1038%2Fs41467-017-00079-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of polypeptide conformation through donor-acceptor transformation of side-chain hydrogen bonding ligands</span></div><div class="casAuthors">Song Ziyuan; Zheng Nan; Mani Deepak; Ferguson Andrew L; Cheng Jianjun; Mansbach Rachael A; He Hua; Yin Lichen; Shih Kuo-Chih; Lin Yao; Nieh Mu-Ping; Shih Kuo-Chih; Baumgartner Ryan; Ba Xiaochu; Huang Yinzhao; Liu Yun; Liu Yun; Lin Yao; Nieh Mu-Ping; Ferguson Andrew L</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Synthetic polypeptides have received increasing attention due to their ability to form higher ordered structures similar to proteins.  The control over their secondary structures, which enables dynamic conformational changes, is primarily accomplished by tuning the side-chain hydrophobic or ionic interactions.  Herein we report a strategy to modulate the conformation of polypeptides utilizing donor-acceptor interactions emanating from side-chain H-bonding ligands.  Specifically, 1,2,3-triazole groups, when incorporated onto polypeptide side-chains, serve as both H-bond donors and acceptors at neutral pH and disrupt the α-helical conformation.  When protonated, the resulting 1,2,3-triazolium ions lose the ability to act as H-bond acceptors, and the polypeptides regain their α-helical structure.  The conformational change of triazole polypeptides in response to the donor-acceptor pattern was conclusively demonstrated using both experimental-based and simulation-based methods.  We further showed the utility of this transition by designing smart, cell-penetrating polymers that undergo acid-activated endosomal escape in living cells.Hydrogen bonding plays a major role in determining the tridimensional structure of biopolymers.  Here, the authors show that control over a polypeptide conformation can be achieved by altering the donor-acceptor properties of side-chain triazole units via protonation-deprotonation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpbYP86F-n6GxpATR2EQURfW6udTcc2eb2rhFKcN4nYLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfgtFeksw%253D%253D&md5=8e48dd81d76eb172da4b14b2d4372b34</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00079-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00079-5%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DMansbach%26aufirst%3DR.%2BA.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DShih%26aufirst%3DK.%2BC.%26aulast%3DBaumgartner%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DN.%26aulast%3DBa%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DMani%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DNieh%26aufirst%3DM.%2BP.%26aulast%3DFerguson%26aufirst%3DA.%2BL.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DJ.%26atitle%3DModulation%2520of%2520polypeptide%2520conformation%2520through%2520donor-acceptor%2520transformation%2520of%2520side-chain%2520hydrogen%2520bonding%2520ligands%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D92%26doi%3D10.1038%2Fs41467-017-00079-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonandi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christodoulou, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumagalli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdicchia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passarella, D.</span></span> <span> </span><span class="NLM_article-title">The 1,2,3-triazole ring as a bioisostere in medicinal chemistry</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1572</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.drudis.2017.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28676407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFCmurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1572-1581&author=E.+Bonandiauthor=M.+S.+Christodoulouauthor=G.+Fumagalliauthor=D.+Perdicchiaauthor=G.+Rastelliauthor=D.+Passarella&title=The+1%2C2%2C3-triazole+ring+as+a+bioisostere+in+medicinal+chemistry&doi=10.1016%2Fj.drudis.2017.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The 1,2,3-triazole ring as a bioisostere in medicinal chemistry</span></div><div class="casAuthors">Bonandi, Elisa; Christodoulou, Michael S.; Fumagalli, Gaia; Perdicchia, Dario; Rastelli, Giulio; Passarella, Daniele</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1572-1581</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">1,2,3-Triazole is a well-known scaffold that has a widespread occurrence in different compds. characterized by several bioactivities, such as antimicrobial, antiviral, and antitumor effects.  Moreover, the structural features of 1,2,3-triazole enable it to mimic different functional groups, justifying its wide use as a bioisostere for the synthesis of new active mols.  Here, we provide an overview of the 1,2,3-triazole ring as a bioisostere for the design of drug analogs, highlighting relevant recent examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyNRynDv4W27Vg90H21EOLACvtfcHk0liBO6_cPbsu0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFCmurrK&md5=0ea3588d22d266e5eed9f401c887b838</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DBonandi%26aufirst%3DE.%26aulast%3DChristodoulou%26aufirst%3DM.%2BS.%26aulast%3DFumagalli%26aufirst%3DG.%26aulast%3DPerdicchia%26aufirst%3DD.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DPassarella%26aufirst%3DD.%26atitle%3DThe%25201%252C2%252C3-triazole%2520ring%2520as%2520a%2520bioisostere%2520in%2520medicinal%2520chemistry%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D1572%26epage%3D1581%26doi%3D10.1016%2Fj.drudis.2017.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kummetha, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichinchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, T. M.</span></span> <span> </span><span class="NLM_article-title">1,2,3-Triazoles as amide bioisosteres: discovery of a new class of potent HIV-1 Vif antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7677</span>– <span class="NLM_lpage">7682</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSltb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7677-7682&author=I.+Mohammedauthor=I.+R.+Kummethaauthor=G.+Singhauthor=N.+Sharovaauthor=G.+Lichinchiauthor=J.+Dangauthor=M.+Stevensonauthor=T.+M.+Rana&title=1%2C2%2C3-Triazoles+as+amide+bioisosteres%3A+discovery+of+a+new+class+of+potent+HIV-1+Vif+antagonists&doi=10.1021%2Facs.jmedchem.6b00247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">1,2,3-Triazoles as Amide Bioisosteres: Discovery of a New Class of Potent HIV-1 Vif Antagonists</span></div><div class="casAuthors">Mohammed, Idrees; Kummetha, Indrasena Reddy; Singh, Gatikrushna; Sharova, Natalia; Lichinchi, Gianluigi; Dang, Jason; Stevenson, Mario; Rana, Tariq M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7677-7682</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RN-18 based viral infectivity factor (Vif), Vif antagonists reduce viral infectivity by rescuing APOBEC3G (A3G) expression and enhancing A3G-dependent Vif degrdn.  Replacement of amide functionality in RN-18 (IC50 = 6 μM) by isosteric heterocycles resulted in the discovery of a 1,2,3-trizole, 1d (IC50 = 1.2 μM).  We identified several potent HIV-1 inhibitors from a 1d based library including 5ax (IC50 = 0.01 μM), 5bx (0.2 μM), 2ey (0.4 μM), 5ey (0.6 μM), and 6bx (0.2 μM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuCHEWexGqlLVg90H21EOLACvtfcHk0lgEuUXHS16DyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSltb7F&md5=f75735900056b2f0975a13a8b6d7355a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00247%26sid%3Dliteratum%253Aachs%26aulast%3DMohammed%26aufirst%3DI.%26aulast%3DKummetha%26aufirst%3DI.%2BR.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DSharova%26aufirst%3DN.%26aulast%3DLichinchi%26aufirst%3DG.%26aulast%3DDang%26aufirst%3DJ.%26aulast%3DStevenson%26aufirst%3DM.%26aulast%3DRana%26aufirst%3DT.%2BM.%26atitle%3D1%252C2%252C3-Triazoles%2520as%2520amide%2520bioisosteres%253A%2520discovery%2520of%2520a%2520new%2520class%2520of%2520potent%2520HIV-1%2520Vif%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7677%26epage%3D7682%26doi%3D10.1021%2Facs.jmedchem.6b00247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maqbool, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span> <span> </span><span class="NLM_article-title">CuAAC-ensembled 1,2,3-triazole-linked isosteres as pharmacophores in drug discovery: review</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5610</span>– <span class="NLM_lpage">5635</span>, <span class="refDoi"> DOI: 10.1039/C9RA09510A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC9RA09510A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivVelsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=5610-5635&author=A.+Raniauthor=G.+Singhauthor=A.+Singhauthor=U.+Maqboolauthor=G.+Kaurauthor=J.+Singh&title=CuAAC-ensembled+1%2C2%2C3-triazole-linked+isosteres+as+pharmacophores+in+drug+discovery%3A+review&doi=10.1039%2FC9RA09510A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">CuAAC-ensembled 1,2,3-triazole-linked isosteres as pharmacophores in drug discovery: review</span></div><div class="casAuthors">Rani, Alisha; Singh, Gurjaspreet; Singh, Akshpreet; Maqbool, Ubair; Kaur, Gurpreet; Singh, Jandeep</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5610-5635</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The review lays emphasis on the significance of 1,2,3-triazoles synthesized via CuAAC reaction having potential to act as anti-microbial, anti-cancer, anti-viral, anti-inflammatory, anti-tuberculosis, anti-diabetic, and anti-Alzheimer drugs.  The importance of click chem. is due to its 'quicker' methodol. that has the capability to create complex and efficient drugs with high yield and purity from simple and cheap starting materials.  The activity of different triazolyl compds. was compiled considering MIC, IC50, and EC50 values against different species of microbes.  In addn. to this, the anti-oxidant property of triazolyl compds. have also been reviewed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1q8vAJ1PzLbVg90H21EOLACvtfcHk0lgEuUXHS16DyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivVelsLY%253D&md5=f2bfa24f21320d47cc1f7632dbe1ab48</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FC9RA09510A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA09510A%26sid%3Dliteratum%253Aachs%26aulast%3DRani%26aufirst%3DA.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DMaqbool%26aufirst%3DU.%26aulast%3DKaur%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DCuAAC-ensembled%25201%252C2%252C3-triazole-linked%2520isosteres%2520as%2520pharmacophores%2520in%2520drug%2520discovery%253A%2520review%26jtitle%3DRSC%2520Adv.%26date%3D2020%26volume%3D10%26spage%3D5610%26epage%3D5635%26doi%3D10.1039%2FC9RA09510A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csuk, R.</span></span> <span> </span><span class="NLM_article-title">Betulinic acid and its derivatives: a patent review (2008-2013)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.927441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1517%2F13543776.2014.927441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=24909232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOhu7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=913-923&author=R.+Csuk&title=Betulinic+acid+and+its+derivatives%3A+a+patent+review+%282008-2013%29&doi=10.1517%2F13543776.2014.927441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Betulinic acid and its derivatives: a patent review (2008 - 2013)</span></div><div class="casAuthors">Csuk, Rene</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">913-923</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Betulinic acid (BA) is a triterpenoid that can be obtained from renewable resources.  BA is cytotoxic to many human tumor cell lines mainly by apoptosis but cell death might also be triggered by nonapoptotic pathways.  Many derivs. have been synthesized to improve the very weak soly. of parent BA and to increase its cytotoxicity as well as its selectivity toward tumor cells.  Areas covered: A brief introduction into cancer is given reflecting the different pathways this disease might be treated using chemotherapy using natural product analogs esp. triterpenes.  The different ways of action of BA in cancer cells are discussed.  Finally, this review describes the main synthetic modifications that have been performed and discusses, in short, the structure-activity relationships of these analogs, investigated between 2008 and 2013 including some important publications from early 2014.  Expert opinion: A no. of patents on BA analogs for the chemotherapy of cancer have been reported between 2008 and 2013.  Most of these patents deal with modifications at positions C-3, C-20 and C-28.  There are only a few compds. meeting the needs of a sufficient hydrosoly., while retaining high cytotoxicity and selectivity toward tumor cells.  Thus, one might expect that there will be some efforts in developing mols. of improved soly. and to find new and more efficient forms of administration (liposomes, transdermal application and nanoemulsions).  An important sideline might be the treatment of the age-dependent degeneration of the macula, a possible caveat of which might be a certain degree of CNS toxicity assocd. with several derivs. of BA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIOWZQT30Pa7Vg90H21EOLACvtfcHk0lgEuUXHS16DyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOhu7zL&md5=6cf01023cff1d13897d660df6260e1d2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.927441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.927441%26sid%3Dliteratum%253Aachs%26aulast%3DCsuk%26aufirst%3DR.%26atitle%3DBetulinic%2520acid%2520and%2520its%2520derivatives%253A%2520a%2520patent%2520review%2520%25282008-2013%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D913%26epage%3D923%26doi%3D10.1517%2F13543776.2014.927441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoufaly, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pokorny, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, M.</span></span> <span> </span><span class="NLM_article-title">Cytotoxic conjugates of betulinic acid and substituted triazoles prepared by Huisgen Cycloaddition from 30-azidoderivatives</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0171621</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0171621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.pone.0171621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28158265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=V.+Sidovaauthor=P.+Zoufalyauthor=J.+Pokornyauthor=P.+Dzubakauthor=M.+Hajduchauthor=I.+Popaauthor=M.+Urban&title=Cytotoxic+conjugates+of+betulinic+acid+and+substituted+triazoles+prepared+by+Huisgen+Cycloaddition+from+30-azidoderivatives&doi=10.1371%2Fjournal.pone.0171621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Cytotoxic conjugates of betulinic acid and substituted triazoles prepared by Huisgen Cycloaddition from 30-azidoderivatives</span></div><div class="casAuthors">Sidova, Veronika; Zoufaly, Pavel; Pokorny, Jan; Dzubak, Petr; Hajduch, Marian; Popa, Igor; Urban, Milan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e0171621/1-e0171621/25</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Synthesis of conjugates of betulinic acid with substituted triazoles I [R1 = H, MeC(O), Ph3Si; R2 = Cyclopentyl, Ph, 2-pyridinyl, etc.] via Huisgen 1,3-cycloaddn. was described.  All the synthesized compds. contained free 28-COOH group.  Allylic bromination of protected betulinic acid by NBS gave corresponding 30-bromoderivatives, their substitution with sodium azides produced 30-azidoderivatives and these azides were subjected to CuI catalyzed Huisgen 1,3-cycloaddn. to give the final conjugates I.  All new compds. were tested for their in vitro cytotoxic activities on eight cancer and two non-cancer cell lines.  The most active compds. were conjugates of 3beta;-O-acetylbetulinic acid and among them, conjugate with triazole substituted by benzaldehyde I [R1 = MeC(O); R2 = 2-H(O)CC6H4] was the best with IC50 of 3.3 μM and therapeutic index of 9.1.  Five compds. in this study had IC50 below 10 μM and inhibited DNA and RNA synthesis and caused block in G0/G1 cell cycle phase which was highly similar to actinomycin D.  It was unusual that here prepd. 3β-O-acetates were more active than compds. with the free 3-OH group and this suggested that this set might have common mechanism of action that was different than mechanism of action of previously known 3β-O-acetoxybetulinic acid derivs.  Benzaldehyde type conjugate I [R1 = MeC(O); R2 = 2-H(O)CC6H4] was the best candidate for further drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIIgdWn-BbLVg90H21EOLACvtfcHk0lhWmC0LHzxGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyqtrk%253D&md5=8213f1966cdcef6cb78b1fd6ab04a5fc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0171621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0171621%26sid%3Dliteratum%253Aachs%26aulast%3DSidova%26aufirst%3DV.%26aulast%3DZoufaly%26aufirst%3DP.%26aulast%3DPokorny%26aufirst%3DJ.%26aulast%3DDzubak%26aufirst%3DP.%26aulast%3DHajduch%26aufirst%3DM.%26aulast%3DPopa%26aufirst%3DI.%26aulast%3DUrban%26aufirst%3DM.%26atitle%3DCytotoxic%2520conjugates%2520of%2520betulinic%2520acid%2520and%2520substituted%2520triazoles%2520prepared%2520by%2520Huisgen%2520Cycloaddition%2520from%252030-azidoderivatives%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0171621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urban, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korinkova, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis of A-seco derivatives of betulinic acid with cytotoxic activity</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">1100</span>– <span class="NLM_lpage">1105</span>, <span class="refDoi"> DOI: 10.1021/np049938m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np049938m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD2cXltVCntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2004&pages=1100-1105&author=M.+Urbanauthor=J.+Sarekauthor=J.+Klinotauthor=G.+Korinkovaauthor=M.+Hajduch&title=Synthesis+of+A-seco+derivatives+of+betulinic+acid+with+cytotoxic+activity&doi=10.1021%2Fnp049938m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of A-Seco Derivatives of Betulinic Acid with Cytotoxic Activity</span></div><div class="casAuthors">Urban, Milan; Sarek, Jan; Klinot, Jiri; Korinkova, Gabriela; Hajduch, Marian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1100-1105</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, the relationships between the chem. structure and cytotoxic activity of betulinic acid derivs. were investigated.  Eight lupane derivs., one of them new, five diosphenols, four of them new, two new nor derivs., five seco derivs., four of them new, and three new seco-anhydrides were synthesized from betulinic acid, and their activities were compared with the activities of known compds.  The effects of substitution on the A-ring and esterification of the carboxyl group in position 28 on cytotoxicity were of special interest.  Significant cytotoxic activity against the T-lymphoblastic leukemia cell line CEM was found in two diosphenols (TCS50 4 and 5 μmol/L) and two seco-anhydrides (TCS50 7 and 6 μmol/L).  All compds. were also tested on cancer cell lines HT 29, K562, K562 Tax, and PC-3.  Diosphenols, as the most promising group of derivs., were further tested on four more lines (A 549, DU 145, MCF 7, SK-Mel2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCecePizJvl7Vg90H21EOLACvtfcHk0lhWmC0LHzxGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXltVCntL4%253D&md5=27a9c35cb8df089621b53f187179c27a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fnp049938m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp049938m%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DM.%26aulast%3DSarek%26aufirst%3DJ.%26aulast%3DKlinot%26aufirst%3DJ.%26aulast%3DKorinkova%26aufirst%3DG.%26aulast%3DHajduch%26aufirst%3DM.%26atitle%3DSynthesis%2520of%2520A-seco%2520derivatives%2520of%2520betulinic%2520acid%2520with%2520cytotoxic%2520activity%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2004%26volume%3D67%26spage%3D1100%26epage%3D1105%26doi%3D10.1021%2Fnp049938m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogundele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrest, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzwedel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allaway, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, D. E.</span></span> <span> </span><span class="NLM_article-title">Phase I and II study of the safety, virologic effect, and pharmacokinetics/pharmacodynamics of single-dose 3-o-(3′,3′-dimethylsuccinyl)betulinic acid (bevirimat) against human immunodeficiency virus infection</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3574</span>– <span class="NLM_lpage">3581</span>, <span class="refDoi"> DOI: 10.1128/AAC.00152-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FAAC.00152-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=17638699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3574-3581&author=P.+F.+Smithauthor=A.+Ogundeleauthor=A.+Forrestauthor=J.+Wiltonauthor=K.+Salzwedelauthor=J.+Dotoauthor=G.+P.+Allawayauthor=D.+E.+Martin&title=Phase+I+and+II+study+of+the+safety%2C+virologic+effect%2C+and+pharmacokinetics%2Fpharmacodynamics+of+single-dose+3-o-%283%E2%80%B2%2C3%E2%80%B2-dimethylsuccinyl%29betulinic+acid+%28bevirimat%29+against+human+immunodeficiency+virus+infection&doi=10.1128%2FAAC.00152-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I and II study of the safety, virologic effect, and pharmacokinetics/pharmacodynamics of single-dose 3-O-(3',3'-dimethylsuccinyl)betulinic acid (bevirimat) against human immunodeficiency virus infection</span></div><div class="casAuthors">Smith, Patrick F.; Ogundele, Abayomi; Forrest, Alan; Wilton, John; Salzwedel, Karl; Doto, Judy; Allaway, Graham P.; Martin, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3574-3581</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Bevirimat [3-O-(3',3'-dimethylsuccinyl)betulinic acid] is the first in a new class of anti-human immunodeficiency virus (HIV) drugs that inhibit viral maturation by specifically blocking cleavage of the Gag capsid (CA) precursor, CA-SP1, to mature CA protein, resulting in defective core condensation and release of immature noninfectious virions.  Four cohorts of six HIV-infected adults, with CD4 counts of >200 and plasma viral loads of 5000 to 250,000 transcripts/mL and not currently receiving antiretroviral therapy, were randomized to receive a single oral dose of placebo, 75, 150, or 250 mg of bevirimat.  Thirty blood samples for drug concns. and 20 HIV RNA measures were collected from each subject over a 20-day period.  Candidate pharmacokinetic/pharmacodynamic models were fit to individual subjects by max. likelihood followed by Bayesian estn.; model discrimination was by cor. Akaike's Information Criterion.  The bevirimat pharmacokinetics was well described by an oral two-compartment linear model (r2, 0.98), with a mean (percent coeff. of variation) half-life of 60.3 (13.6) h and apparent oral clearance of bevirimat from the plasma compartment of 0.17 (18) liters/h.  HIV RNA was modeled as being produced in infected CD4 cells, with bevirimat inhibiting infection of new CD4 cells through a Hill-type function (r2, 0.87).  Single oral doses of bevirimat were well tolerated and demonstrated a dose-dependent redn. in viral load.  The av. max. redn. from baseline following the 150- and 250-mg doses was greater than 0.45 log10, with individual patients having redns. of greater than 0.7 log10.  No bevirimat resistance mutations were detected during the course of the study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0xx_ZTDkPSLVg90H21EOLACvtfcHk0lhWmC0LHzxGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbjM&md5=598e3191feb2ae7ac5f2d94f4a8a1c86</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FAAC.00152-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00152-07%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DP.%2BF.%26aulast%3DOgundele%26aufirst%3DA.%26aulast%3DForrest%26aufirst%3DA.%26aulast%3DWilton%26aufirst%3DJ.%26aulast%3DSalzwedel%26aufirst%3DK.%26aulast%3DDoto%26aufirst%3DJ.%26aulast%3DAllaway%26aufirst%3DG.%2BP.%26aulast%3DMartin%26aufirst%3DD.%2BE.%26atitle%3DPhase%2520I%2520and%2520II%2520study%2520of%2520the%2520safety%252C%2520virologic%2520effect%252C%2520and%2520pharmacokinetics%252Fpharmacodynamics%2520of%2520single-dose%25203-o-%25283%25E2%2580%25B2%252C3%25E2%2580%25B2-dimethylsuccinyl%2529betulinic%2520acid%2520%2528bevirimat%2529%2520against%2520human%2520immunodeficiency%2520virus%2520infection%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3574%26epage%3D3581%26doi%3D10.1128%2FAAC.00152-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Regueiro-Ren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dicker, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanumegowda, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Second generation inhibitors of HIV-1 maturation</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00656</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00656" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXis1WqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=287-294&author=A.+Regueiro-Renauthor=I.+B.+Dickerauthor=U.+Hanumegowdaauthor=N.+A.+Meanwell&title=Second+generation+inhibitors+of+HIV-1+maturation&doi=10.1021%2Facsmedchemlett.8b00656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Second Generation Inhibitors of HIV-1 Maturation</span></div><div class="casAuthors">Regueiro-Ren, Alicia; Dicker, Ira B.; Hanumegowda, Umesh; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">287-294</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The strategy and tactics subtending the discovery and development of the second generation HIV-1 maturation inhibitor GSK-3532795/BMS-955176, a compd. that exhibits a broader spectrum of antiviral effect in vitro and in clin. studies than the prototypical maturation inhibitor bevirimat, are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokN8Btl19fDLVg90H21EOLACvtfcHk0liVA3tDm6QEJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXis1WqtL4%253D&md5=82f8d64ac2b5c5fd54dde15b448a0f6e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00656%26sid%3Dliteratum%253Aachs%26aulast%3DRegueiro-Ren%26aufirst%3DA.%26aulast%3DDicker%26aufirst%3DI.%2BB.%26aulast%3DHanumegowda%26aufirst%3DU.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSecond%2520generation%2520inhibitors%2520of%2520HIV-1%2520maturation%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D287%26epage%3D294%26doi%3D10.1021%2Facsmedchemlett.8b00656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span> <span> </span><span class="NLM_article-title">A general metal-free route towards the synthesis of 1,2,3-triazoles from readily available primary amines and ketones</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2885</span>– <span class="NLM_lpage">2888</span>, <span class="refDoi"> DOI: 10.1039/C5CC08347H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC5CC08347H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=26744743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVynsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2016&pages=2885-2888&author=J.+Thomasauthor=S.+Janaauthor=J.+Johnauthor=S.+Liekensauthor=W.+Dehaen&title=A+general+metal-free+route+towards+the+synthesis+of+1%2C2%2C3-triazoles+from+readily+available+primary+amines+and+ketones&doi=10.1039%2FC5CC08347H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A general metal-free route towards the synthesis of 1,2,3-triazoles from readily available primary amines and ketones</span></div><div class="casAuthors">Thomas, Joice; Jana, Sampad; John, Jubi; Liekens, Sandra; Dehaen, Wim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2885-2888</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An unprecedented approach that enables the direct and selective prepn. of 1,5-disubstituted 1,2,3-triazoles from abundantly available building blocks such as primary amines, enolizable ketones and 4-nitrophenyl azide as a renewable source of dinitrogen via an organocascade process has been developed.  Furthermore, this efficient methodol. also enables the synthesis of fully functionalized and fused N-substituted heterocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqi4PDkmmSf7Vg90H21EOLACvtfcHk0liVA3tDm6QEJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVynsb3F&md5=2a2e61bc280bd33db0440774e0ad73a7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1039%2FC5CC08347H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC08347H%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DJ.%26aulast%3DJana%26aufirst%3DS.%26aulast%3DJohn%26aufirst%3DJ.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DDehaen%26aufirst%3DW.%26atitle%3DA%2520general%2520metal-free%2520route%2520towards%2520the%2520synthesis%2520of%25201%252C2%252C3-triazoles%2520from%2520readily%2520available%2520primary%2520amines%2520and%2520ketones%26jtitle%3DChem.%2520Commun.%26date%3D2016%26volume%3D52%26spage%3D2885%26epage%3D2888%26doi%3D10.1039%2FC5CC08347H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Opsomer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span> <span> </span><span class="NLM_article-title">Chemoselectivity in the synthesis of 1,2,3-triazoles from enolizable ketones, primary alkylamines, and 4-nitrophenyl azide</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4191</span>– <span class="NLM_lpage">4198</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1055%2Fs-0036-1588856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1Gqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=4191-4198&author=T.+Opsomerauthor=J.+Thomasauthor=W.+Dehaen&title=Chemoselectivity+in+the+synthesis+of+1%2C2%2C3-triazoles+from+enolizable+ketones%2C+primary+alkylamines%2C+and+4-nitrophenyl+azide"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoselectivity in the Synthesis of 1,2,3-Triazoles from Enolizable Ketones, Primary Alkylamines, and 4-Nitrophenyl Azide</span></div><div class="casAuthors">Opsomer, Tomas; Thomas, Joice; Dehaen, Wim</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4191-4198</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A general metal-free route towards the synthesis of 1,5-di- or fully-substituted 1,2,3-triazoles were designed and named as 'triazolization' reaction of ketones.  Limitations of this route were encountered in reactions with more activated ketones, where the corresponding 1-(4-nitrophenyl)-1,2,3-triazole was found back as the major product.  Interestingly, three different triazoles were formed when 1,3-diphenylacetone was used as the ketone.  1,3-Diphenylacetone was used as a model substrate to investigate the chemoselectivity of metal-free route under different reaction circumstances.  This led to the conclusion that the formation of the desired 1,2,3-triazole was favored in apolar solvents, at high temps., and in the presence of acetic acid.  By using one equiv. of acetic acid, previously inaccessible fully decorated 1,2,3-triazoles were synthesized from simple α-arylketones in moderate to high yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLUd0kHriOqbVg90H21EOLACvtfcHk0liVA3tDm6QEJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1Gqs7s%253D&md5=76c4e7b1dc0b1b226d8ff7d7dc70cffc</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1055%2Fs-0036-1588856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0036-1588856%26sid%3Dliteratum%253Aachs%26aulast%3DOpsomer%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DJ.%26aulast%3DDehaen%26aufirst%3DW.%26atitle%3DChemoselectivity%2520in%2520the%2520synthesis%2520of%25201%252C2%252C3-triazoles%2520from%2520enolizable%2520ketones%252C%2520primary%2520alkylamines%252C%2520and%25204-nitrophenyl%2520azide%26jtitle%3DSynthesis%26date%3D2017%26volume%3D49%26spage%3D4191%26epage%3D4198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silveira-Dorta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span> <span> </span><span class="NLM_article-title">Straightforward synthesis of enantiomerically pure 1,2,3-triazoles derived from amino esters</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3168</span>– <span class="NLM_lpage">3176</span>, <span class="refDoi"> DOI: 10.1039/C8OB00533H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC8OB00533H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=29645062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmvVequ7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=3168-3176&author=G.+Silveira-Dortaauthor=S.+Janaauthor=L.+Borkovaauthor=J.+Thomasauthor=W.+Dehaen&title=Straightforward+synthesis+of+enantiomerically+pure+1%2C2%2C3-triazoles+derived+from+amino+esters&doi=10.1039%2FC8OB00533H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Straightforward synthesis of enantiomerically pure 1,2,3-triazoles derived from amino esters</span></div><div class="casAuthors">Silveira-Dorta, Gaston; Jana, Sampad; Borkova, Lucie; Thomas, Joice; Dehaen, Wim</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3168-3176</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The synthesis of enantiomerically pure 1,4,5-trisubstituted e.g., I, 1,5-disubstituted e.g., II, and fused 1,2,3-triazole derivs. e.g., III has been developed.  The synthesis employs enantiomerically pure amino esters such as tert-Bu L-phenylalaninate, L-glutamic acid di-Et ester, tert-Bu L-isoleucinate, etc. derived from amino acids such as D-phenylalanine, L-tyrosine, L-isoleucine, etc. and com. available ketones such as acetophenone, nonan-5-one, 2-acetylfuran, acetylferrocene, etc. under metal-free conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodwtq4YfAehrVg90H21EOLACvtfcHk0liVA3tDm6QEJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmvVequ7g%253D&md5=e1b421c0c39f9eb37711cbe0660ea4e2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC8OB00533H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB00533H%26sid%3Dliteratum%253Aachs%26aulast%3DSilveira-Dorta%26aufirst%3DG.%26aulast%3DJana%26aufirst%3DS.%26aulast%3DBorkova%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DJ.%26aulast%3DDehaen%26aufirst%3DW.%26atitle%3DStraightforward%2520synthesis%2520of%2520enantiomerically%2520pure%25201%252C2%252C3-triazoles%2520derived%2520from%2520amino%2520esters%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D3168%26epage%3D3176%26doi%3D10.1039%2FC8OB00533H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flekhter, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigmatullina, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltina, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karachurina, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galin, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarudii, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolstikov, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boreko, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlova, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolaeva, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savinova, O. V.</span></span> <span> </span><span class="NLM_article-title">Synthesis of betulinic acid from betulin extract and study of the antiviral and antiulcer activity of some related terpenoids</span>. <i>Pharm. Chem. J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1023/A:1021844705853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1023%2FA%3A1021844705853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFegsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2002&pages=484-487&author=O.+B.+Flekhterauthor=L.+R.+Nigmatullinaauthor=L.+A.+Baltinaauthor=L.+T.+Karachurinaauthor=F.+Z.+Galinauthor=F.+S.+Zarudiiauthor=G.+A.+Tolstikovauthor=E.+I.+Borekoauthor=N.+I.+Pavlovaauthor=S.+N.+Nikolaevaauthor=O.+V.+Savinova&title=Synthesis+of+betulinic+acid+from+betulin+extract+and+study+of+the+antiviral+and+antiulcer+activity+of+some+related+terpenoids&doi=10.1023%2FA%3A1021844705853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of betulinic acid from betulin extract and study of the antiviral and antiulcer activity of some related terpenoids</span></div><div class="casAuthors">Flekhter, O. B.; Nigmatullina, L. R.; Baltina, L. A.; Karachurina, L. T.; Galin, F. Z.; Zarudii, F. S.; Tolstikov, G. A.; Boreko, E. I.; Pavlova, N. I.; Nikolaeva, S. N.; Savinova, O. V.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Chemistry Journal (Translation of Khimiko-Farmatsevticheskii Zhurnal)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">484-487</span>CODEN:
                <span class="NLM_cas:coden">PCJOAU</span>;
        ISSN:<span class="NLM_cas:issn">0091-150X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Consultants Bureau</span>)
        </div><div class="casAbstract">Betulinic acid was synthesized by redn. of betulonic acid obtained from betulin ext.  The reaction involving betulonic acid and trifluoroacetic acid led to 28-oxo-allobetulone.  Investigation of the antiviral properties of 28-oxo-allobetulone in a cell culture showed inhibition of the growth of influenza type A virus.  This compd. was inactive with respect to herpes simplex virus and ECHO-6 enterovirus.  Betulonic acid and 28-oxo-allobetulone exhibited antiinflammatory properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpClydOcjiqBrVg90H21EOLACvtfcHk0lh_yPC86T5yxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFegsA%253D%253D&md5=e9fcb00f011957d7eda1fb3320d6de9e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1023%2FA%3A1021844705853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1021844705853%26sid%3Dliteratum%253Aachs%26aulast%3DFlekhter%26aufirst%3DO.%2BB.%26aulast%3DNigmatullina%26aufirst%3DL.%2BR.%26aulast%3DBaltina%26aufirst%3DL.%2BA.%26aulast%3DKarachurina%26aufirst%3DL.%2BT.%26aulast%3DGalin%26aufirst%3DF.%2BZ.%26aulast%3DZarudii%26aufirst%3DF.%2BS.%26aulast%3DTolstikov%26aufirst%3DG.%2BA.%26aulast%3DBoreko%26aufirst%3DE.%2BI.%26aulast%3DPavlova%26aufirst%3DN.%2BI.%26aulast%3DNikolaeva%26aufirst%3DS.%2BN.%26aulast%3DSavinova%26aufirst%3DO.%2BV.%26atitle%3DSynthesis%2520of%2520betulinic%2520acid%2520from%2520betulin%2520extract%2520and%2520study%2520of%2520the%2520antiviral%2520and%2520antiulcer%2520activity%2520of%2520some%2520related%2520terpenoids%26jtitle%3DPharm.%2520Chem.%2520J.%26date%3D2002%26volume%3D36%26spage%3D484%26epage%3D487%26doi%3D10.1023%2FA%3A1021844705853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dehelean, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">S̨oica, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledeţi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aluaş, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zupko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Găluşcan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinta-Pinzaru, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munteanu, M.</span></span> <span> </span><span class="NLM_article-title">Study of the betulin enriched birch bark extracts effects on human carcinoma cells and ear inflammation</span>. <i>Chem. Cent. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">137</span>, <span class="refDoi"> DOI: 10.1186/1752-153X-6-137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1186%2F1752-153X-6-137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=23158079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVentg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=137&author=C.+A.+Deheleanauthor=C.+S%CC%A8oicaauthor=I.+Lede%C5%A3iauthor=M.+Alua%C5%9Fauthor=I.+Zupkoauthor=A.+G%C4%83lu%C5%9Fcanauthor=S.+Cinta-Pinzaruauthor=M.+Munteanu&title=Study+of+the+betulin+enriched+birch+bark+extracts+effects+on+human+carcinoma+cells+and+ear+inflammation&doi=10.1186%2F1752-153X-6-137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Study of the betulin enriched birch bark extracts effects on human carcinoma cells and ear inflammation</span></div><div class="casAuthors">Dehelean, Cristina A.; Soica, Codruta; Ledeti, Ionut; Aluas, Mihaela; Zupko, Istvan; Galuscan, Atena; Cinta-Pinzaru, Simona; Munteanu, Melania</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry Central Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137</span>CODEN:
                <span class="NLM_cas:coden">CCJHA2</span>;
        ISSN:<span class="NLM_cas:issn">1752-153X</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Pentacyclic triterpenes, mainly betulin and betulinic acid, are valuable anticancer agents found in the bark of birch tree.  This study evaluates birch bark exts. for the active principles compn.  Results: New improved extn. methods were applied on the bark of Betula pendula in order to reach the max. content in active principles.  Exts. were analyzed by HPLC-MS, Raman, SERS and 13C NMR spectroscopy which revealed a very high yield of betulin (over 90%).  Growth inhibiting effects were measured in vitro on four malignant human cell lines: A431 (skin epidermoid carcinoma), A2780 (ovarian carcinoma), HeLa (cervix adenocarcinoma) and MCF7 (breast adenocarcinoma), by means of MTT assay.  All of the prepd. bark exts. exerted a pronounced antiproliferative effect against human cancer cell lines.  In vivo studies involved the anti-inflammatory effect of birch exts. on TPA-induced model of inflammation in mice.  Conclusions: The research revealed the efficacy of the extn. procedures as well as the antiproliferative and anti-inflammatory effects of birch exts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCZndmmC6F0rVg90H21EOLACvtfcHk0lh_yPC86T5yxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVentg%253D%253D&md5=0bba4d02dbddec0407d885ec1a6c0c54</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2F1752-153X-6-137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1752-153X-6-137%26sid%3Dliteratum%253Aachs%26aulast%3DDehelean%26aufirst%3DC.%2BA.%26aulast%3DS%25CC%25A8oica%26aufirst%3DC.%26aulast%3DLede%25C5%25A3i%26aufirst%3DI.%26aulast%3DAlua%25C5%259F%26aufirst%3DM.%26aulast%3DZupko%26aufirst%3DI.%26aulast%3DG%25C4%2583lu%25C5%259Fcan%26aufirst%3DA.%26aulast%3DCinta-Pinzaru%26aufirst%3DS.%26aulast%3DMunteanu%26aufirst%3DM.%26atitle%3DStudy%2520of%2520the%2520betulin%2520enriched%2520birch%2520bark%2520extracts%2520effects%2520on%2520human%2520carcinoma%2520cells%2520and%2520ear%2520inflammation%26jtitle%3DChem.%2520Cent.%2520J.%26date%3D2012%26volume%3D6%26spage%3D137%26doi%3D10.1186%2F1752-153X-6-137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ressmann, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremsmayr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaertner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zirbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bica, K.</span></span> <span> </span><span class="NLM_article-title">Toward a benign strategy for the manufacturing of betulinic acid</span>. <i>Green Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1039/C6GC02641A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC6GC02641A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKku7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1014-1022&author=A.+K.+Ressmannauthor=T.+Kremsmayrauthor=P.+Gaertnerauthor=R.+Zirbsauthor=K.+Bica&title=Toward+a+benign+strategy+for+the+manufacturing+of+betulinic+acid&doi=10.1039%2FC6GC02641A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Toward a benign strategy for the manufacturing of betulinic acid</span></div><div class="casAuthors">Ressmann, Anna K.; Kremsmayr, Thomas; Gaertner, Peter; Zirbs, Ronald; Bica, Katharina</div><div class="citationInfo"><span class="NLM_cas:title">Green Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1014-1022</span>CODEN:
                <span class="NLM_cas:coden">GRCHFJ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9262</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report a novel and efficient strategy for the prepn. of the high-value triterpenoid betulinic acid based on extn. and streamlined oxidn. of betulin from the industrial byproduct birch bark.  The initial extn. of betulin relies on a biphasic system and allowed extg. betulin in short times at room temp.  The crude ext. could be directly oxidized, thereby providing a chromium-free, time- and energy saving strategy for the manufg. of betulinic acid in high yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6NRBU_wi0s7Vg90H21EOLACvtfcHk0lh_yPC86T5yxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKku7nN&md5=38acb050f7e2cd84c69003cde2ba31b0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2FC6GC02641A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6GC02641A%26sid%3Dliteratum%253Aachs%26aulast%3DRessmann%26aufirst%3DA.%2BK.%26aulast%3DKremsmayr%26aufirst%3DT.%26aulast%3DGaertner%26aufirst%3DP.%26aulast%3DZirbs%26aufirst%3DR.%26aulast%3DBica%26aufirst%3DK.%26atitle%3DToward%2520a%2520benign%2520strategy%2520for%2520the%2520manufacturing%2520of%2520betulinic%2520acid%26jtitle%3DGreen%2520Chem.%26date%3D2017%26volume%3D19%26spage%3D1014%26epage%3D1022%26doi%3D10.1039%2FC6GC02641A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Šiman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipová, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichá, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezrouk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havelek, R.</span></span> <span> </span><span class="NLM_article-title">Effective method of purification of betulin from birch bark: the importance of its purity for scientific and medicinal use</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0154933</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0154933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.pone.0154933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=27152419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyrtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=P.+%C5%A0imanauthor=A.+Filipov%C3%A1author=A.+Tich%C3%A1author=M.+Niangauthor=A.+Bezroukauthor=R.+Havelek&title=Effective+method+of+purification+of+betulin+from+birch+bark%3A+the+importance+of+its+purity+for+scientific+and+medicinal+use&doi=10.1371%2Fjournal.pone.0154933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Effective method of purification of betulin from birch bark: the importance of its purity for scientific and medicinal use</span></div><div class="casAuthors">Siman, Pavel; Filipova, Alzbeta; Ticha, Alena; Niang, Mohamed; Bezrouk, Ales; Havelek, Radim</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e0154933/1-e0154933/14</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A new and relatively simple method for purifn. of betulin from birch bark ext. was developed in this study.  Its five purifn. steps are based on the differential soly. of ext. components in various solvents and their crystn. and/or pptn., on their affinity for Ca(OH)2 in ethanol, and on the affinity of some impurities for silica gel in chloroform.  In addn., all used solvents can be simply recycled.  Betulin of more than 99% purity can be prepd. by this method with minimal costs.  Various observations including crystn. of betulin, changes in crystals during heating, and attempt of localization of betulin in outer birch bark are also described in this work.  The original ext., fraction without betulinic acid and lupeol, amorphous fraction of pure betulin, final cryst. fraction of pure betulin and com. betulin as a std. were employed to det. the antiproliferative/cytotoxic effect.We usedWST-1 tetrazolium-based assays with triple neg. breast cancer cell line BT-549.  The decrease in cell survival showed clear relationship with the purity of the samples, being most pronounced using our final product of pure cryst. betulin.  WST-1 proliferation/cytotoxicity test using triple neg. breast cancer cell line BT-549 clearly showed the importance of purity of betulin for biol. expts. and, apparently, for its medicinal use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIN5k3lcMI1bVg90H21EOLACvtfcHk0lhgBivLYHMR6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyrtb%252FM&md5=55d956cc92ce9f704516cc8ba7b9816b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0154933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0154933%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0iman%26aufirst%3DP.%26aulast%3DFilipov%25C3%25A1%26aufirst%3DA.%26aulast%3DTich%25C3%25A1%26aufirst%3DA.%26aulast%3DNiang%26aufirst%3DM.%26aulast%3DBezrouk%26aufirst%3DA.%26aulast%3DHavelek%26aufirst%3DR.%26atitle%3DEffective%2520method%2520of%2520purification%2520of%2520betulin%2520from%2520birch%2520bark%253A%2520the%2520importance%2520of%2520its%2520purity%2520for%2520scientific%2520and%2520medicinal%2520use%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0154933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dijkman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannoun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonsdottir, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kint, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forlenza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trybala, E.</span></span> <span> </span><span class="NLM_article-title">Targeting membrane-bound viral RNA synthesis reveals potent inhibition of diverse coronaviruses including the Middle East respiratory syndrome virus</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1004166</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1004166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.ppat.1004166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=24874215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGkur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=A.+Lundinauthor=R.+Dijkmanauthor=T.+Bergstr%C3%B6mauthor=N.+Kannauthor=B.+Adamiakauthor=C.+Hannounauthor=E.+Kindlerauthor=H.+R.+Jonsdottirauthor=D.+Muthauthor=J.+Kintauthor=M.+Forlenzaauthor=M.+A.+Mullerauthor=C.+Drostenauthor=V.+Thielauthor=E.+Trybala&title=Targeting+membrane-bound+viral+RNA+synthesis+reveals+potent+inhibition+of+diverse+coronaviruses+including+the+Middle+East+respiratory+syndrome+virus&doi=10.1371%2Fjournal.ppat.1004166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting membrane-bound viral RNA synthesis reveals potent inhibition of diverse coronaviruses including the Middle East respiratory syndrome virus</span></div><div class="casAuthors">Lundin, Anna; Dijkman, Ronald; Bergstroem, Tomas; Kann, Nina; Adamiak, Beata; Hannoun, Charles; Kindler, Eveline; Jonsdottir, Hulda R.; Muth, Doreen; Kint, Joeri; Forlenza, Maria; Mueller, Marcel A.; Drosten, Christian; Thiel, Volker; Trybala, Edward</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1004166/1-e1004166/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses raise serious concerns as emerging zoonotic viruses without specific antiviral drugs available.  Here we screened a collection of 16671 diverse compds. for anti-human coronavirus 229E activity and identified an inhibitor, designated K22, that specifically targets membrane-bound coronaviral RNA synthesis.  K22 exerts most potent antiviral activity after virus entry during an early step of the viral life cycle.  Specifically, the formation of double membrane vesicles (DMVs), a hallmark of coronavirus replication, was greatly impaired upon K22 treatment accompanied by near-complete inhibition of viral RNA synthesis.  K22-resistant viruses contained substitutions in non-structural protein 6 (nsp6), a membrane-spanning integral component of the viral replication complex implicated in DMV formation, corroborating that K22 targets membrane bound viral RNA synthesis.  Besides K22 resistance, the nsp6 mutants induced a reduced no. of DMVs, displayed decreased specific infectivity, while RNA synthesis was not affected.  Importantly, K22 inhibits a broad range of coronaviruses, including Middle East respiratory syndrome coronavirus (MERS-CoV), and efficient inhibition was achieved in primary human epithelia cultures representing the entry port of human coronavirus infection.  Collectively, this study proposes an evolutionary conserved step in the life cycle of pos.-stranded RNA viruses, the recruitment of cellular membranes for viral replication, as vulnerable and, most importantly, druggable target for antiviral intervention.  We expect this mode of action to serve as a paradigm for the development of potent antiviral drugs to combat many animal and human virus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZm0zkCr8NwLVg90H21EOLACvtfcHk0lhgBivLYHMR6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGkur7J&md5=26b2d4d8c3890db67a7bd5e16450ef46</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004166%26sid%3Dliteratum%253Aachs%26aulast%3DLundin%26aufirst%3DA.%26aulast%3DDijkman%26aufirst%3DR.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DT.%26aulast%3DKann%26aufirst%3DN.%26aulast%3DAdamiak%26aufirst%3DB.%26aulast%3DHannoun%26aufirst%3DC.%26aulast%3DKindler%26aufirst%3DE.%26aulast%3DJonsdottir%26aufirst%3DH.%2BR.%26aulast%3DMuth%26aufirst%3DD.%26aulast%3DKint%26aufirst%3DJ.%26aulast%3DForlenza%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DM.%2BA.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DThiel%26aufirst%3DV.%26aulast%3DTrybala%26aufirst%3DE.%26atitle%3DTargeting%2520membrane-bound%2520viral%2520RNA%2520synthesis%2520reveals%2520potent%2520inhibition%2520of%2520diverse%2520coronaviruses%2520including%2520the%2520Middle%2520East%2520respiratory%2520syndrome%2520virus%26jtitle%3DPLoS%2520Pathog.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.ppat.1004166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, S. K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, V. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, P. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, K.-Y.</span></span> <span> </span><span class="NLM_article-title">Middle East respiratory syndrome coronavirus: another zoonotic betacoronavirus causing SARS-like disease</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">522</span>, <span class="refDoi"> DOI: 10.1128/CMR.00102-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FCMR.00102-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=25810418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=465-522&author=J.+F.+W.+Chanauthor=S.+K.+P.+Lauauthor=K.+K.+W.+Toauthor=V.+C.+C.+Chengauthor=P.+C.+Y.+Wooauthor=K.-Y.+Yuen&title=Middle+East+respiratory+syndrome+coronavirus%3A+another+zoonotic+betacoronavirus+causing+SARS-like+disease&doi=10.1128%2FCMR.00102-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Middle east respiratory syndrome coronavirus: another zoonotic betacoronavirus causing SARS-like disease</span></div><div class="casAuthors">Chan, Jasper F. W.; Lau, Susanna K. P.; To, Kelvin K. W.; Cheng, Vincent C. C.; Woo, Patrick C. Y.; Yuen, Kwok-Yung</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">465-522</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The source of the severe acute respiratory syndrome (SARS) epidemic was traced to wildlife market civets and ultimately to bats.  Subsequent hunting for novel coronaviruses (CoVs) led to the discovery of two addnl. human and over 40 animal CoVs, including the prototype lineage C betacoronaviruses, Tylonycteris bat CoV HKU4 and Pipistrellus bat CoV HKU5; these are phylogenetically closely related to the Middle East respiratory syndrome (MERS) CoV, which has affected more than 1,000 patients with over 35% fatality since its emergence in 2012.  All primary cases of MERS are epidemiol. linked to the Middle East.  Some of these patients had contacted camels which shed virus and/or had pos. serol.  Most secondary cases are related to health care-assocd. clusters.  The disease is esp. severe in elderly men with comorbidities.  Clin. severity may be related to MERS-CoV's ability to infect a broad range of cells with DPP4 expression, evade the host innate immune response, and induce cytokine dysregulation.  Reverse transcription-PCR on respiratory and/or extrapulmonary specimens rapidly establishes diagnosis.  Supportive treatment with extracorporeal membrane oxygenation and dialysis is often required in patients with organ failure.  Antivirals with potent in vitro activities include neutralizing monoclonal antibodies, antiviral peptides, interferons, mycophenolic acid, and lopinavir.  They should be evaluated in suitable animal models before clin. trials.  Developing an effective camel MERS-CoV vaccine and implementing appropriate infection control measures may control the continuing epidemic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2NNE4QR663rVg90H21EOLACvtfcHk0lhgBivLYHMR6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltlyhs74%253D&md5=c94188fe0de72aeb7df66a1f2144dde4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1128%2FCMR.00102-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00102-14%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DJ.%2BF.%2BW.%26aulast%3DLau%26aufirst%3DS.%2BK.%2BP.%26aulast%3DTo%26aufirst%3DK.%2BK.%2BW.%26aulast%3DCheng%26aufirst%3DV.%2BC.%2BC.%26aulast%3DWoo%26aufirst%3DP.%2BC.%2BY.%26aulast%3DYuen%26aufirst%3DK.-Y.%26atitle%3DMiddle%2520East%2520respiratory%2520syndrome%2520coronavirus%253A%2520another%2520zoonotic%2520betacoronavirus%2520causing%2520SARS-like%2520disease%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2015%26volume%3D28%26spage%3D465%26epage%3D522%26doi%3D10.1128%2FCMR.00102-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak</span>. <i>Emerging Microbes Infect.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">319</span>, <span class="refDoi"> DOI: 10.1080/22221751.2020.1725399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1080%2F22221751.2020.1725399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32020836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFOktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=313-319&author=L.+Chenauthor=W.+Liuauthor=Q.+Zhangauthor=K.+Xuauthor=G.+Yeauthor=W.+Wuauthor=Z.+Sunauthor=F.+Liuauthor=K.+Wuauthor=B.+Zhongauthor=Y.+Meiauthor=W.+Zhangauthor=Y.+Chenauthor=Y.+Liauthor=M.+Shiauthor=K.+Lanauthor=Y.+Liu&title=RNA+based+mNGS+approach+identifies+a+novel+human+coronavirus+from+two+individual+pneumonia+cases+in+2019+Wuhan+outbreak&doi=10.1080%2F22221751.2020.1725399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak</span></div><div class="casAuthors">Chen, Liangjun; Liu, Weiyong; Zhang, Qi; Xu, Ke; Ye, Guangming; Wu, Weichen; Sun, Ziyong; Liu, Fang; Wu, Kailang; Zhong, Bo; Mei, Yi; Zhang, Wenxia; Chen, Yu; Li, Yirong; Shi, Mang; Lan, Ke; Liu, Yingle</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Microbes & Infections</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">313-319</span>CODEN:
                <span class="NLM_cas:coden">EMIMC4</span>;
        ISSN:<span class="NLM_cas:issn">2222-1751</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">From Dec. 2019, an outbreak of unusual pneumonia was reported in Wuhan with many cases linked to Huanan Seafood Market that sells seafood as well as live exotic animals.  We investigated two patients who developed acute respiratory syndromes after independent contact history with this market.  The two patients shared common clin. features including fever, cough, and multiple ground-glass opacities in the bilateral lung field with patchy infiltration.  Here, we highlight the use of a low-input metagenomic next-generation sequencing (mNGS) approach on RNA extd. from bronchoalveolar lavage fluid (BALF).  It rapidly identified a novel coronavirus (named 2019-nCoV according to World Health Organization announcement) which was the sole pathogens in the sample with very high abundance level (1.5% and 0.62% of total RNA sequenced).  The entire viral genome is 29,881 nt in length (GenBank MN988668 and MN988669, Sequence Read Archive database Bioproject accession PRJNA601736) and is classified into β-coronavirus genus.  Phylogenetic anal. indicates that 2019-nCoV is close to coronaviruses (CoVs) circulating in Rhinolophus (Horseshoe bats), such as 98.7% nucleotide identity to partial RdRp gene of bat coronavirus strain BtCoV/4991 (GenBank KP876546, 370 nt sequence of RdRp and lack of other genome sequence) and 87.9% nucleotide identity to bat coronavirus strain bat-SL-CoVZC45 and bat-SL-CoVZXC21.  Evolutionary anal. based on ORF1a/1b, S, and N genes also suggests 2019-nCoV is more likely a novel CoV independently introduced from animals to humans.  Read Archive database Bioproject accession PRJNA601736.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmGr3MxtQ3mrVg90H21EOLACvtfcHk0ljhcjeHnoHsBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFOktbo%253D&md5=87cb9016fc26ee2e4948a322c4155223</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1080%2F22221751.2020.1725399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F22221751.2020.1725399%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DZhong%26aufirst%3DB.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DM.%26aulast%3DLan%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DRNA%2520based%2520mNGS%2520approach%2520identifies%2520a%2520novel%2520human%2520coronavirus%2520from%2520two%2520individual%2520pneumonia%2520cases%2520in%25202019%2520Wuhan%2520outbreak%26jtitle%3DEmerging%2520Microbes%2520Infect.%26date%3D2020%26volume%3D9%26spage%3D313%26epage%3D319%26doi%3D10.1080%2F22221751.2020.1725399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persoons, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanstreels, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Looveren, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckx, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geudens, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Man, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wauters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wauters, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanaudenaerde, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrechts, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dallmeier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibaut, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquemyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daelemans, D.</span></span> <span> </span><span class="NLM_article-title">Genome-wide CRISPR screening identifies TMEM106B as a proviral host factor for SARS-CoV-2</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_fpage">435</span>, <span class="refDoi"> DOI: 10.1038/s41588-021-00805-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41588-021-00805-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33686287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3MXmtVSrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&pages=435&author=J.+Baggenauthor=L.+Persoonsauthor=E.+Vanstreelsauthor=S.+Jansenauthor=D.+Van+Looverenauthor=B.+Boeckxauthor=V.+Geudensauthor=J.+De+Manauthor=D.+Jochmansauthor=J.+Wautersauthor=E.+Wautersauthor=B.+M.+Vanaudenaerdeauthor=D.+Lambrechtsauthor=J.+Neytsauthor=K.+Dallmeierauthor=H.+J.+Thibautauthor=M.+Jacquemynauthor=P.+Maesauthor=D.+Daelemans&title=Genome-wide+CRISPR+screening+identifies+TMEM106B+as+a+proviral+host+factor+for+SARS-CoV-2&doi=10.1038%2Fs41588-021-00805-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide CRISPR screening identifies TMEM106B as a proviral host factor for SARS-CoV-2</span></div><div class="casAuthors">Baggen, Jim; Persoons, Leentje; Vanstreels, Els; Jansen, Sander; Van Looveren, Dominique; Boeckx, Bram; Geudens, Vincent; De Man, Julie; Jochmans, Dirk; Wauters, Joost; Wauters, Els; Vanaudenaerde, Bart M.; Lambrechts, Diether; Neyts, Johan; Dallmeier, Kai; Thibaut, Hendrik Jan; Jacquemyn, Maarten; Maes, Piet; Daelemans, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-444</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Portfolio</span>)
        </div><div class="casAbstract">Abstr.: The ongoing COVID-19 pandemic has caused a global economic and health crisis.  To identify host factors essential for coronavirus infection, we performed genome-wide functional genetic screens with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and human coronavirus 229E.  These screens uncovered virus-specific as well as shared host factors, including TMEM41B and PI3K type 3.  We discovered that SARS-CoV-2 requires the lysosomal protein TMEM106B to infect human cell lines and primary lung cells.  TMEM106B overexpression enhanced SARS-CoV-2 infection as well as pseudovirus infection, suggesting a role in viral entry.  Furthermore, single-cell RNA-sequencing of airway cells from patients with COVID-19 demonstrated that TMEM106B expression correlates with SARS-CoV-2 infection.  The present study uncovered a collection of coronavirus host factors that may be exploited to develop drugs against SARS-CoV-2 infection or future zoonotic coronavirus outbreaks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyc63uZRz4gbVg90H21EOLACvtfcHk0ljhcjeHnoHsBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXmtVSrt7s%253D&md5=29d1554b6afe63cf8b7f9c3553d9038f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41588-021-00805-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-021-00805-2%26sid%3Dliteratum%253Aachs%26aulast%3DBaggen%26aufirst%3DJ.%26aulast%3DPersoons%26aufirst%3DL.%26aulast%3DVanstreels%26aufirst%3DE.%26aulast%3DJansen%26aufirst%3DS.%26aulast%3DVan%2BLooveren%26aufirst%3DD.%26aulast%3DBoeckx%26aufirst%3DB.%26aulast%3DGeudens%26aufirst%3DV.%26aulast%3DDe%2BMan%26aufirst%3DJ.%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DWauters%26aufirst%3DJ.%26aulast%3DWauters%26aufirst%3DE.%26aulast%3DVanaudenaerde%26aufirst%3DB.%2BM.%26aulast%3DLambrechts%26aufirst%3DD.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DDallmeier%26aufirst%3DK.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3DJacquemyn%26aufirst%3DM.%26aulast%3DMaes%26aufirst%3DP.%26aulast%3DDaelemans%26aufirst%3DD.%26atitle%3DGenome-wide%2520CRISPR%2520screening%2520identifies%2520TMEM106B%2520as%2520a%2520proviral%2520host%2520factor%2520for%2520SARS-CoV-2%26jtitle%3DNat.%2520Genet.%26date%3D2021%26spage%3D435%26doi%3D10.1038%2Fs41588-021-00805-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palaninathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcantara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, C. C.</span></span> <span> </span><span class="NLM_article-title">Structural and functional analyses of the severe acute respiratory syndrome coronavirus endoribonuclease Nsp15</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">3655</span>– <span class="NLM_lpage">3664</span>, <span class="refDoi"> DOI: 10.1074/jbc.M708375200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1074%2Fjbc.M708375200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=18045871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlyitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=3655-3664&author=K.+Bhardwajauthor=S.+Palaninathanauthor=J.+M.+Alcantaraauthor=L.+L.+Yiauthor=L.+Guarinoauthor=J.+C.+Sacchettiniauthor=C.+C.+Kao&title=Structural+and+functional+analyses+of+the+severe+acute+respiratory+syndrome+coronavirus+endoribonuclease+Nsp15&doi=10.1074%2Fjbc.M708375200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Functional Analyses of the Severe Acute Respiratory Syndrome Coronavirus Endoribonuclease Nsp15</span></div><div class="casAuthors">Bhardwaj, Kanchan; Palaninathan, Satheesh; Alcantara, Joanna Maria Ortiz; Li Yi, Lillian; Guarino, Linda; Sacchettini, James C.; Kao, C. Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3655-3664</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The severe acute respiratory syndrome (SARS) coronavirus encodes several RNA-processing enzymes that are unusual for RNA viruses, including Nsp15 (nonstructural protein 15), a hexameric endoribonuclease that preferentially cleaves 3' of uridines.  We solved the structure of a catalytically inactive mutant version of Nsp15, which was crystd. as a hexamer.  The structure contains unreported flexibility in the active site of each subunit.  Substitutions in the active site residues serine 293 and proline 343 allowed Nsp15 to cleave at cytidylate, whereas mutation of leucine 345 rendered Nsp15 able to cleave at purines as well as pyrimidines.  Mutations that targeted the residues involved in subunit interactions generally resulted in the formation of catalytically inactive monomers.  The RNA-binding residues were mapped by a method linking reversible crosslinking, RNA affinity purifn., and peptide fingerprinting.  Alanine substitution of several residues in the RNA-contacting portion of Nsp15 did not affect hexamer formation but decreased the affinity of RNA binding and reduced endonuclease activity.  This suggests a model for Nsp15 hexamer interaction with RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz2zWq3VF2j7Vg90H21EOLACvtfcHk0ljhcjeHnoHsBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlyitLc%253D&md5=f4aa526b6a9384f1cee8158a9881da40</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M708375200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M708375200%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DK.%26aulast%3DPalaninathan%26aufirst%3DS.%26aulast%3DAlcantara%26aufirst%3DJ.%2BM.%26aulast%3DYi%26aufirst%3DL.%2BL.%26aulast%3DGuarino%26aufirst%3DL.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DKao%26aufirst%3DC.%2BC.%26atitle%3DStructural%2520and%2520functional%2520analyses%2520of%2520the%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520endoribonuclease%2520Nsp15%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D3655%26epage%3D3664%26doi%3D10.1074%2Fjbc.M708375200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuman, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchmeier, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, P.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of a monomeric form of severe acute respiratory syndrome coronavirus endonuclease nsp15 suggests a role for hexamerization as an allosteric switch</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">6700</span>– <span class="NLM_lpage">6708</span>, <span class="refDoi"> DOI: 10.1128/JVI.02817-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.02817-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=17409150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Ort7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=6700-6708&author=J.+S.+Josephauthor=K.+S.+Saikatenduauthor=V.+Subramanianauthor=B.+W.+Neumanauthor=M.+J.+Buchmeierauthor=R.+C.+Stevensauthor=P.+Kuhn&title=Crystal+structure+of+a+monomeric+form+of+severe+acute+respiratory+syndrome+coronavirus+endonuclease+nsp15+suggests+a+role+for+hexamerization+as+an+allosteric+switch&doi=10.1128%2FJVI.02817-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a monomeric form of severe acute respiratory syndrome coronavirus endonuclease nsp15 suggests a role for hexamerization as an allosteric switch</span></div><div class="casAuthors">Joseph, Jeremiah S.; Saikatendu, Kumar Singh; Subramanian, Vanitha; Neuman, Benjamin W.; Buchmeier, Michael J.; Stevens, Raymond C.; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6700-6708</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Mature nonstructural protein-15 (nsp15) from the severe acute respiratory syndrome coronavirus (SARS-CoV) contains a novel uridylate-specific Mn2+-dependent endoribonuclease (NendoU).  Structure studies of the full-length form of the obligate hexameric enzyme from two CoVs, SARS-CoV and murine hepatitis virus, and its monomeric homolog, XendoU from Xenopus laevis, combined with mutagenesis studies have implicated several residues in enzymic activity and the N-terminal domain as the major determinant of hexamerization.  However, the tight link between hexamerization and enzyme activity in NendoUs has remained an enigma.  Here, we report the structure of a trimmed, monomeric form of SARS-CoV nsp15 (residues 28 to 335) detd. to a resoln. of 2.9 Å.  The catalytic loop (residues 234 to 249) with its two reactive histidines (His 234 and His 249) is dramatically flipped by ∼120° into the active site cleft.  Furthermore, the catalytic nucleophile Lys 289 points in a diametrically opposite direction, a consequence of an outward displacement of the supporting loop (residues 276 to 295).  In the full-length hexameric forms, these two loops are packed against each other and are stabilized by intimate intersubunit interactions.  Our results support the hypothesis that absence of an adjacent monomer due to deletion of the hexamerization domain is the most likely cause for disruption of the active site, offering a structural basis for why only the hexameric form of this enzyme is active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ9KTQXgd2z7Vg90H21EOLACvtfcHk0lgrS2la-AzyCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Ort7w%253D&md5=6dc7bf273fe8bc104273e0a9b481a0ad</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FJVI.02817-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02817-06%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph%26aufirst%3DJ.%2BS.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DSubramanian%26aufirst%3DV.%26aulast%3DNeuman%26aufirst%3DB.%2BW.%26aulast%3DBuchmeier%26aufirst%3DM.%2BJ.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DCrystal%2520structure%2520of%2520a%2520monomeric%2520form%2520of%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520endonuclease%2520nsp15%2520suggests%2520a%2520role%2520for%2520hexamerization%2520as%2520an%2520allosteric%2520switch%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D6700%26epage%3D6708%26doi%3D10.1128%2FJVI.02817-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strehle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoonjans, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auwerx, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saladin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span> <span> </span><span class="NLM_article-title">Redefining the TGR5 triterpenoid binding pocket at the C-3 position</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1983</span>– <span class="NLM_lpage">1988</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201000329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1002%2Fcmdc.201000329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=20979081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1983-1988&author=C.+Genetauthor=C.+Schmidtauthor=A.+Strehleauthor=K.+Schoonjansauthor=J.+Auwerxauthor=R.+Saladinauthor=A.+Wagner&title=Redefining+the+TGR5+triterpenoid+binding+pocket+at+the+C-3+position&doi=10.1002%2Fcmdc.201000329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Redefining the TGR5 Triterpenoid Binding Pocket at the C-3 Position</span></div><div class="casAuthors">Genet, Cedric; Schmidt, Celine; Strehle, Axelle; Schoonjans, Kristina; Auwerx, Johan; Saladin, Regis; Wagner, Alain</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1983-1988</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The synthesis and biol. evaluation of new C-3 modified betulinic acid derivs., which can broaden the understanding of the putative C-3 binding pocket of TGR5 is undertaken.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx_nJBfdvJRbVg90H21EOLACvtfcHk0lgrS2la-AzyCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsr7M&md5=25047211b92571537e00f49fac66e80b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000329%26sid%3Dliteratum%253Aachs%26aulast%3DGenet%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DC.%26aulast%3DStrehle%26aufirst%3DA.%26aulast%3DSchoonjans%26aufirst%3DK.%26aulast%3DAuwerx%26aufirst%3DJ.%26aulast%3DSaladin%26aufirst%3DR.%26aulast%3DWagner%26aufirst%3DA.%26atitle%3DRedefining%2520the%2520TGR5%2520triterpenoid%2520binding%2520pocket%2520at%2520the%2520C-3%2520position%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1983%26epage%3D1988%26doi%3D10.1002%2Fcmdc.201000329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reed, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muench, H.</span></span> <span> </span><span class="NLM_article-title">A simple method of estimating fifty per cent endpoints</span>. <i>Am. J. Epidemiol.</i> <span class="NLM_year" style="font-weight: bold;">1938</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.aje.a118408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1093%2Foxfordjournals.aje.a118408" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1938&pages=493-497&author=L.+J.+Reedauthor=H.+Muench&title=A+simple+method+of+estimating+fifty+per+cent+endpoints&doi=10.1093%2Foxfordjournals.aje.a118408"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.aje.a118408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.aje.a118408%26sid%3Dliteratum%253Aachs%26aulast%3DReed%26aufirst%3DL.%2BJ.%26aulast%3DMuench%26aufirst%3DH.%26atitle%3DA%2520simple%2520method%2520of%2520estimating%2520fifty%2520per%2520cent%2520endpoints%26jtitle%3DAm.%2520J.%2520Epidemiol.%26date%3D1938%26volume%3D27%26spage%3D493%26epage%3D497%26doi%3D10.1093%2Foxfordjournals.aje.a118408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Apaydin, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesur, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesur, Z.</span></span> <span> </span><span class="NLM_article-title">Synthesis and anti-coronavirus activity of a series of 1-thia-4-azaspiro[4.5]decan-3-one derivatives</span>. <i>Arch. Pharm. (Weinheim)</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>352</i></span>, <span class="NLM_elocation-id">e1800330</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1002%2Fardp.201800330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=31073993" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2019&author=C.+B.+Apaydinauthor=N.+Cesurauthor=A.+Stevaertauthor=L.+Naesensauthor=Z.+Cesur&title=Synthesis+and+anti-coronavirus+activity+of+a+series+of+1-thia-4-azaspiro%5B4.5%5Ddecan-3-one+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fardp.201800330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fardp.201800330%26sid%3Dliteratum%253Aachs%26aulast%3DApaydin%26aufirst%3DC.%2BB.%26aulast%3DCesur%26aufirst%3DN.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DCesur%26aufirst%3DZ.%26atitle%3DSynthesis%2520and%2520anti-coronavirus%2520activity%2520of%2520a%2520series%2520of%25201-thia-4-azaspiro%255B4.5%255Ddecan-3-one%2520derivatives%26jtitle%3DArch.%2520Pharm.%2520%2528Weinheim%2529%26date%3D2019%26volume%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vrijens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noppen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boogaerts, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanstreels, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiodelli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laporte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderlinden, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span> <span> </span><span class="NLM_article-title">Influenza virus entry via the GM3 ganglioside-mediated platelet-derived growth factor receptor β signalling pathway</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1099/jgv.0.001235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1099%2Fjgv.0.001235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=30762518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVWrtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2019&pages=583-601&author=P.+Vrijensauthor=S.+Noppenauthor=T.+Boogaertsauthor=E.+Vanstreelsauthor=R.+Roncaauthor=P.+Chiodelliauthor=M.+Laporteauthor=E.+Vanderlindenauthor=S.+Liekensauthor=A.+Stevaertauthor=L.+Naesens&title=Influenza+virus+entry+via+the+GM3+ganglioside-mediated+platelet-derived+growth+factor+receptor+%CE%B2+signalling+pathway&doi=10.1099%2Fjgv.0.001235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Influenza virus entry via the GM3 ganglioside-mediated platelet-derived growth factor receptor β signalling pathway</span></div><div class="casAuthors">Vrijens, Pieter; Noppen, Sam; Boogaerts, Talitha; Vanstreels, Els; Ronca, Roberto; Chiodelli, Paola; Laporte, Manon; Vanderlinden, Evelien; Liekens, Sandra; Stevaert, Annelies; Naesens, Lieve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">583-601</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">1465-2099</span>.
    
            (<span class="NLM_cas:orgname">Microbiology Society</span>)
        </div><div class="casAbstract">This requires a better understanding of virus- host interactions during the entry process, including the role of receptor tyrosine kinases (RTKs).  To search for cellular targets, we evaluated a panel of 276 protein kinase inhibitors in a multicycle antiviral assay in Madin-Darby canine kidney cells.  The RTK inhibitor Ki8751 displayed robust anti-influenza A and B virus activity and was selected for mechanistic investigations.  Ki8751 efficiently disrupted the endocytic process of influenza virus in different cell lines carrying plateletderived growth factor receptor β (PDGFRb), an RTK that is known to act at GM3 ganglioside-pos. lipid rafts.  The more efficient virus entry in CHO-K1 cells compared to the wild-type ancestor (CHO-wt) cells indicated a pos. effect of GM3, which is abundant in CHO-K1 but not in CHO-wt cells.  Entering virus localized to GM3-pos. lipid rafts and the PDGFRbcontaining endosomal compartment.  PDGFRβ/GM3-dependent virus internalization involved PDGFRβ phosphorylation, which was potently inhibited by Ki8751, and desialylation of activated PDGFRb by the viral neuraminidase.  Virus uptake coincided with strong activation of the Raf/MEK/Erk cascade, but not of PI3K/Akt or phospholipase C-γ.  We conclude that influenza virus efficiently hijacks the GM3-enhanced PDGFR β signaling pathway for cell penetration, providing an opportunity for host cell-targeting antiviral intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq594Z0J-3b3rVg90H21EOLACvtfcHk0liRPcZdoQIyAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVWrtL%252FO&md5=a1ee95dd1afd4a15abd613de41e43358</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1099%2Fjgv.0.001235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fjgv.0.001235%26sid%3Dliteratum%253Aachs%26aulast%3DVrijens%26aufirst%3DP.%26aulast%3DNoppen%26aufirst%3DS.%26aulast%3DBoogaerts%26aufirst%3DT.%26aulast%3DVanstreels%26aufirst%3DE.%26aulast%3DRonca%26aufirst%3DR.%26aulast%3DChiodelli%26aufirst%3DP.%26aulast%3DLaporte%26aufirst%3DM.%26aulast%3DVanderlinden%26aufirst%3DE.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DNaesens%26aufirst%3DL.%26atitle%3DInfluenza%2520virus%2520entry%2520via%2520the%2520GM3%2520ganglioside-mediated%2520platelet-derived%2520growth%2520factor%2520receptor%2520%25CE%25B2%2520signalling%2520pathway%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2019%26volume%3D100%26spage%3D583%26epage%3D601%26doi%3D10.1099%2Fjgv.0.001235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szűcs, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostorházi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herczegh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borbás, A.</span></span> <span> </span><span class="NLM_article-title">Reprogramming of the antibacterial drug vancomycin results in potent antiviral agents devoid of antibacterial activity</span>. <i>Pharmaceuticals (Basel)</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">139</span>, <span class="refDoi"> DOI: 10.3390/ph13070139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.3390%2Fph13070139" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2020&pages=139&author=Z.+Sz%C5%B1csauthor=L.+Naesensauthor=A.+Stevaertauthor=E.+Ostorh%C3%A1ziauthor=G.+Battaauthor=P.+Herczeghauthor=A.+Borb%C3%A1s&title=Reprogramming+of+the+antibacterial+drug+vancomycin+results+in+potent+antiviral+agents+devoid+of+antibacterial+activity&doi=10.3390%2Fph13070139"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.3390%2Fph13070139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph13070139%26sid%3Dliteratum%253Aachs%26aulast%3DSz%25C5%25B1cs%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DOstorh%25C3%25A1zi%26aufirst%3DE.%26aulast%3DBatta%26aufirst%3DG.%26aulast%3DHerczegh%26aufirst%3DP.%26aulast%3DBorb%25C3%25A1s%26aufirst%3DA.%26atitle%3DReprogramming%2520of%2520the%2520antibacterial%2520drug%2520vancomycin%2520results%2520in%2520potent%2520antiviral%2520agents%2520devoid%2520of%2520antibacterial%2520activity%26jtitle%3DPharmaceuticals%2520%2528Basel%2529%26date%3D2020%26volume%3D13%26spage%3D139%26doi%3D10.3390%2Fph13070139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 1,2,3-Triazole-Fused Betulonic Acid Derivatives Starting from Betulin<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Na<sub>2</sub>Cr<sub>2</sub>O<sub>7</sub>, H<sub>2</sub>SO<sub>4</sub>, acetone, H<sub>2</sub>O, 0 °C to r.t., 1 h; (ii) betulonic acid <b>2</b> (1.0 equiv), primary amine <b>3</b> (1.4 equiv), 4-nitrophenyl azide <b>4</b> (1.0 equiv), toluene (0.4 mL), 100 °C, 4 Å MS, 24 h.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Activity of <b>5h</b> against HCoV-229E. (A) Representative microscopic images showing protection against virus-induced cytopathic effect (CPE) in human embryonic lung (HEL) cells. (B) Dose–response curves for inhibition of virus-induced CPE (solid squares) and for cytotoxicity (open squares) in HEL cells, both determined by MTS cell viability assay. Data points are the mean ± SEM (<i>N</i> = 3). (C) Immunofluorescence detection of viral dsRNA in HCoV-229E-infected human bronchial epithelial 16HBE cells at 24 h post-infection (p.i.). Green: dsRNA and blue: nuclear DAPI staining. Compounds: 12 μM <b>5h</b> or 12 μM GS-441524. Scale bar: 50 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. SAR Study around Compound <b>5h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) betulonic acid, NH<sub>4</sub>OAc, 4-nitrophenyl azide, DMF, 80 °C, 18 h, 4 Å MS, isolated yield 86%; (ii) betulonic acid, benzyl amine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 92%; (iii) betulonic acid, (R)-(+)-α-methylbenzylamine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 69%; (iv) dihydrobetulonic acid, (<i>S</i>)-(−)-α-methylbenzylamine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 90%; (v) lupenone, (<i>S</i>)-(−)-α-methylbenzylamine, 4-nitrophenyl azide, toluene, 100 °C, 18 h, 4 Å MS, isolated yield 83%.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. <b>5h</b> acts post-entry at an early stage in viral RNA synthesis. Compound addition was delayed until different time points after infecting HEL cells with HCoV-229E, and viral RNA was quantified at 16 h p.i. Compound concentrations: bafilomycin 6.3 nM; <b>5h</b> and K22: 15 μM. The <i>Y</i> axis shows the viral RNA copy number relative to the virus control (mean ± SEM of two independent experiments).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Mutations in nsp15 confer resistance of HCoV-229E to <b>5h</b> (left panels), but not to GS-441524 (right panels). The graphs show the effect of the compounds on virus yield. (A) HEL cells infected with <b>5h</b>-resistant mutants obtained by virus passaging under <b>5h</b> and carrying substitution K60R (first selection) or T66I (second selection) in nsp15. (B) 16HBE cells infected with EndoU-deficient mutant virus (H250A<sub>nsp15</sub>), obtained by reverse genetics.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Data points are the mean ± SEM (<i>N</i> = 3). An unpaired <i>t</i>-test (GraphPad Prism 8.4.3) was used to compare the mutant viruses to WT, and the resulting two-tailed <i>P</i>-values were adjusted for multiple comparisons using Holm–Šídák (α = 0.05). *, <i>P</i> < 0.05; **, <i>P</i> < 0.01; ***, <i>P</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode of <b>5h</b> in the HCoV-229E nsp15 hexameric protein (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4">4RS4</a>), as predicted by docking. (A) Hydrophobic pocket lies adjacent to the EndoU catalytic center (catalytic triad consisting of His235, His250, and Lys291) and at an nsp15 dimer interface (monomers depicted in a differently colored surface). The pocket is surrounded by His250, Lys60, and Thr66, explaining why <b>5h</b> is inactive against HCoV-229E viruses carrying mutations at these sites. (B) <b>5h</b> occupies the pocket by making hydrophobic interactions with Val293 and side chain fragments of Lys291 and Thr292. The molecule further engages in hydrogen-bonding interactions with Cys294 and Thr295 via the carboxylic acid moiety and with Thr245 via the 1,2,3-triazole moiety. Additional hydrophobic interactions with Val63, Leu65, and Thr292 are made via the aromatic ring-substituted 1,2,3-triazole moiety.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/medium/jm0c02124_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the hydrophobic pocket, occupied by <b>5h</b>, in the nsp15 proteins of HCoV-229E (left; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4">4RS4</a>) and SARS-CoV-2 (right; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7K1O">7K1O</a>). The carboxylic acid moiety of <b>5h</b> forms hydrogen bonds with both nsp15 binding pockets. On the other hand, the 1,2,3-triazole group engages hydrogen-bonding interactions with the HCoV-229E nsp15 protein but is incompatible with the SARS-CoV-2 pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02124/20210506/images/large/jm0c02124_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02124&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2016.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.tim.2016.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=27012512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWmsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=490-502&author=S.+Suauthor=G.+Wongauthor=W.+Shiauthor=J.+Liuauthor=A.+C.+K.+Laiauthor=J.+Zhouauthor=W.+Liuauthor=Y.+Biauthor=G.+F.+Gao&title=Epidemiology%2C+genetic+recombination%2C+and+pathogenesis+of+coronaviruses&doi=10.1016%2Fj.tim.2016.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology, genetic recombination, and pathogenesis of coronaviruses</span></div><div class="casAuthors">Su, Shuo; Wong, Gary; Shi, Weifeng; Liu, Jun; Lai, Alexander C. K.; Zhou, Jiyong; Liu, Wenjun; Bi, Yuhai; Gao, George F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">490-502</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Human coronaviruses (HCoVs) were first described in the 1960s for patients with the common cold.  Since then, more HCoVs have been discovered, including those that cause severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS), two pathogens that, upon infection, can cause fatal respiratory disease in humans.  It was recently discovered that dromedary camels in Saudi Arabia harbor three different HCoV species, including a dominant MERS HCoV lineage that was responsible for the outbreaks in the Middle East and South Korea during 2015.  In this review we aim to compare and contrast the different HCoVs with regard to epidemiol. and pathogenesis, in addn. to the virus evolution and recombination events which have, on occasion, resulted in outbreaks amongst humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAQICS-ipqM7Vg90H21EOLACvtfcHk0lgQI9KwyK8vIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWmsbc%253D&md5=c9215c7687711bf8c2ff73a5cc2d23f4</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2016.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2016.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DG.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DA.%2BC.%2BK.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DBi%26aufirst%3DY.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DEpidemiology%252C%2520genetic%2520recombination%252C%2520and%2520pathogenesis%2520of%2520coronaviruses%26jtitle%3DTrends%2520Microbiol.%26date%3D2016%26volume%3D24%26spage%3D490%26epage%3D502%26doi%3D10.1016%2Fj.tim.2016.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falsey, A. R.</span></span> <span> </span><span class="NLM_article-title">Clinical impact of human coronaviruses 229E and OC43 infection in diverse adult populations</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">1634</span>– <span class="NLM_lpage">1642</span>, <span class="refDoi"> DOI: 10.1093/infdis/jit393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1093%2Finfdis%2Fjit393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=23922367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A280%3ADC%252BC3sfntl2ksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2013&pages=1634-1642&author=E.+E.+Walshauthor=J.+H.+Shinauthor=A.+R.+Falsey&title=Clinical+impact+of+human+coronaviruses+229E+and+OC43+infection+in+diverse+adult+populations&doi=10.1093%2Finfdis%2Fjit393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical impact of human coronaviruses 229E and OC43 infection in diverse adult populations</span></div><div class="casAuthors">Walsh Edward E; Shin Jae Hyun; Falsey Ann R</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of infectious diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1634-42</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The incidence and clinical impact of coronavirus (CoV) infection in elderly persons and those with underlying cardiopulmonary disease over a long duration is not well described.  We determined the incidence and clinical impact of 229E and OC43 CoV in this population during 4 consecutive winters, and compared illnesses to influenza A, respiratory syncytial virus, and human metapneumovirus.  METHODS:  CoV 229E and OC43 were detected by reverse transcription polymerase chain reaction and serology in 4 adult populations under surveillance for acute respiratory illness during the winters of 1999-2003.  Cohorts included healthy young adults, healthy elderly adults, high-risk adults with underlying cardiopulmonary disease, and a hospitalized group.  RESULTS:  Three hundred ninety-eight CoV infections were identified, with annual infection rates ranging from 2.8% to 26% in prospective cohorts, and prevalence ranging from 3.3% to 11.1% in the hospitalized cohort.  The incidence of infections with each strain was similar, although asymptomatic infection and viral coinfection was significantly more common with 229E than OC43 infection.  Although the incidence and clinical manifestations were similar for each strain, OC43-infected subjects tended to seek more medical care, as OC43 was twice as common as 229E among the hospitalized cohort.  CONCLUSIONS:  CoV infections in the elderly are frequent, likely causing substantial medical disease burden.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRKc5dNmh-_Y2PgL4LrC-0VfW6udTcc2eY4JdaGRYin97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sfntl2ksw%253D%253D&md5=e1d57d74aaf1d4e7a39870d9a27c6146</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Finfdis%2Fjit393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Finfdis%252Fjit393%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DE.%2BE.%26aulast%3DShin%26aufirst%3DJ.%2BH.%26aulast%3DFalsey%26aufirst%3DA.%2BR.%26atitle%3DClinical%2520impact%2520of%2520human%2520coronaviruses%2520229E%2520and%2520OC43%2520infection%2520in%2520diverse%2520adult%2520populations%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2013%26volume%3D208%26spage%3D1634%26epage%3D1642%26doi%3D10.1093%2Finfdis%2Fjit393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dijkman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Hoek, L.</span></span> <span> </span><span class="NLM_article-title">Human coronaviruses 229E and NL63: close yet still so far</span>. <i>J. Formos. Med. Assoc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/S0929-6646(09)60066-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2FS0929-6646%2809%2960066-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=19369173" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFCmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2009&pages=270-279&author=R.+Dijkmanauthor=L.+van+der+Hoek&title=Human+coronaviruses+229E+and+NL63%3A+close+yet+still+so+far&doi=10.1016%2FS0929-6646%2809%2960066-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Human coronaviruses 229E and NL63: close yet still so far</span></div><div class="casAuthors">Dijkman, Ronald; van der Hoek, Lia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the Formosan Medical Association</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">270-279</span>CODEN:
                <span class="NLM_cas:coden">JFASEO</span>;
        ISSN:<span class="NLM_cas:issn">0929-6646</span>.
    
            (<span class="NLM_cas:orgname">Elsevier (Singapore) Pte Ltd.</span>)
        </div><div class="casAbstract">A review.  HCoV-NL63 and HCoV-229E are two of the four human coronaviruses that circulate worldwide.  These two viruses are unique in their relationship towards each other.  Phylogenetically, the viruses are more closely related to each other than to any other human coronavirus, yet they only share 65% sequence identity.  Moreover, the viruses use different receptors to enter their target cell.  HCoV-NL63 is assocd. with croup in children, whereas all signs suggest that the virus probably causes the common cold in healthy adults.  HCoV-229E is a proven common cold virus in healthy adults, so it is probable that both viruses induce comparable symptoms in adults, even though their mode of infection differs.  Here, we present an overview of the current knowledge on both human coronaviruses, focusing on similarities and differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7RjSmMw3WnrVg90H21EOLACvtfcHk0lgQI9KwyK8vIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFCmtLk%253D&md5=6749f66f8d499f73ec4def49c1282c09</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0929-6646%2809%2960066-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0929-6646%252809%252960066-8%26sid%3Dliteratum%253Aachs%26aulast%3DDijkman%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BHoek%26aufirst%3DL.%26atitle%3DHuman%2520coronaviruses%2520229E%2520and%2520NL63%253A%2520close%2520yet%2520still%2520so%2520far%26jtitle%3DJ.%2520Formos.%2520Med.%2520Assoc.%26date%3D2009%26volume%3D108%26spage%3D270%26epage%3D279%26doi%3D10.1016%2FS0929-6646%2809%2960066-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horby, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">A novel coronavirus outbreak of global health concern</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(20)30185-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2FS0140-6736%2820%2930185-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=31986257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnsVaisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2020&pages=470-473&author=C.+Wangauthor=P.+W.+Horbyauthor=F.+G.+Haydenauthor=G.+F.+Gao&title=A+novel+coronavirus+outbreak+of+global+health+concern&doi=10.1016%2FS0140-6736%2820%2930185-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A novel coronavirus outbreak of global health concern</span></div><div class="casAuthors">Wang, Chen; Horby, Peter W.; Hayden, Frederick G.; Gao, George F.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">10223</span>),
    <span class="NLM_cas:pages">470-473</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review on COVID-19 and global health concern.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRk9FBxgaVErVg90H21EOLACvtfcHk0lgkzs1OtqvTrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnsVaisLg%253D&md5=f16ec2beb810968cba634e1813888780</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2820%2930185-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252820%252930185-9%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHorby%26aufirst%3DP.%2BW.%26aulast%3DHayden%26aufirst%3DF.%2BG.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DA%2520novel%2520coronavirus%2520outbreak%2520of%2520global%2520health%2520concern%26jtitle%3DLancet%26date%3D2020%26volume%3D395%26spage%3D470%26epage%3D473%26doi%3D10.1016%2FS0140-6736%2820%2930185-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fehr, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span></span> <span> </span><span class="NLM_article-title">Coronaviruses: an overview of their replication and pathogenesis</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1282</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-2438-7_1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1007%2F978-1-4939-2438-7_1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=25720466" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28Xls12ksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1282&publication_year=2015&pages=1-23&author=A.+R.+Fehrauthor=S.+Perlman&title=Coronaviruses%3A+an+overview+of+their+replication+and+pathogenesis&doi=10.1007%2F978-1-4939-2438-7_1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Coronaviruses: an overview of their replication and pathogenesis</span></div><div class="casAuthors">Fehr, Anthony R.; Perlman, Stanley</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1282</span>
        (<span class="NLM_cas:issue">Coronaviruses</span>),
    <span class="NLM_cas:pages">1-23</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Coronaviruses (CoVs), enveloped pos.-sense RNA viruses, are characterized by club-like spikes that project from their surface, an unusually large RNA genome, and a unique replication strategy.  Coronaviruses cause a variety of diseases in mammals and birds ranging from enteritis in cows and pigs and upper respiratory disease in chickens to potentially lethal human respiratory infections.  Here we provide a brief introduction to coronaviruses discussing their replication and pathogenicity, and current prevention and treatment strategies.  We also discuss the outbreaks of the highly pathogenic Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) and the recently identified Middle Eastern Respiratory Syndrome Coronavirus (MERS-CoV).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSI4XoT-KaBrVg90H21EOLACvtfcHk0lgkzs1OtqvTrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xls12ksL8%253D&md5=2e0fc18cd6b7312f0704f933a3614913</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2438-7_1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-2438-7_1%26sid%3Dliteratum%253Aachs%26aulast%3DFehr%26aufirst%3DA.%2BR.%26aulast%3DPerlman%26aufirst%3DS.%26atitle%3DCoronaviruses%253A%2520an%2520overview%2520of%2520their%2520replication%2520and%2520pathogenesis%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2015%26volume%3D1282%26spage%3D1%26epage%3D23%26doi%3D10.1007%2F978-1-4939-2438-7_1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felsenstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hedrich, C. M.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 infections in children and young people</span>. <i>Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>220</i></span>,  <span class="NLM_fpage">108588</span>, <span class="refDoi"> DOI: 10.1016/j.clim.2020.108588</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.clim.2020.108588" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32905851" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSksLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=220&publication_year=2020&pages=108588&author=S.+Felsensteinauthor=C.+M.+Hedrich&title=SARS-CoV-2+infections+in+children+and+young+people&doi=10.1016%2Fj.clim.2020.108588"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 infections in children and young people</span></div><div class="casAuthors">Felsenstein, Susanna; Hedrich, Christian M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Immunology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">220</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">108588</span>CODEN:
                <span class="NLM_cas:coden">CLIIFY</span>;
        ISSN:<span class="NLM_cas:issn">1521-6616</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Though recent reports link SARS-CoV-2 infections with hyper-inflammatory states in children, most children experience no/mild symptoms, and hospitalization and mortality rates are low in the age group.  As symptoms are usually mild and seroconversion occurs at low frequencies, it remains unclear whether children significantly contribute to community transmission.  Several hypotheses try to explain age-related differences in disease presentation and severity.  Possible reasons for milder presentations in children as compared to adults include frequent contact to seasonal coronaviruses, presence of cross-reactive antibodies, and/or co-clearance with other viruses.  Increased expression of ACE2 in young people may facilitate virus infection, while limiting inflammation and reducing the risk of severe disease.  Further potential factors include recent vaccinations and a more diverse memory T cell repertoire.  This manuscript reviews age-related host factors that may protect children from COVID-19 and complications assocd., and addresses the confusion around seropositivity and immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorKCcwMLFke7Vg90H21EOLACvtfcHk0lgkzs1OtqvTrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSksLnF&md5=6de5a48079318b0d286b9b4f143dce81</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.clim.2020.108588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clim.2020.108588%26sid%3Dliteratum%253Aachs%26aulast%3DFelsenstein%26aufirst%3DS.%26aulast%3DHedrich%26aufirst%3DC.%2BM.%26atitle%3DSARS-CoV-2%2520infections%2520in%2520children%2520and%2520young%2520people%26jtitle%3DClin.%2520Immunol.%26date%3D2020%26volume%3D220%26spage%3D108588%26doi%3D10.1016%2Fj.clim.2020.108588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amigues, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearlman, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reid, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youngstein, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayatilleke, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konig, M.</span></span> <span> </span><span class="NLM_article-title">Coronavirus disease 2019: investigational therapies in the prevention and treatment of hyperinflammation</span>. <i>Expert Rev. Clin. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1204</span>, <span class="refDoi"> DOI: 10.1080/1744666X.2021.1847084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1080%2F1744666X.2021.1847084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33146561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisVClsL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=1185-1204&author=I.+Amiguesauthor=A.+H.+Pearlmanauthor=A.+Patelauthor=P.+Reidauthor=P.+C.+Robinsonauthor=R.+Sinhaauthor=A.+H.+Kimauthor=T.+Youngsteinauthor=A.+Jayatillekeauthor=M.+Konig&title=Coronavirus+disease+2019%3A+investigational+therapies+in+the+prevention+and+treatment+of+hyperinflammation&doi=10.1080%2F1744666X.2021.1847084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus disease 2019: investigational therapies in the prevention and treatment of hyperinflammation</span></div><div class="casAuthors">Amigues, Isabelle; Pearlman, Alexander H.; Patel, Aarat; Reid, Pankti; Robinson, Philip C.; Sinha, Rashmi; Kim, Alfred Hj; Youngstein, Taryn; Jayatilleke, Arundathi; Konig, Maximilian</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Clinical Immunology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1185-1204</span>CODEN:
                <span class="NLM_cas:coden">ERCIBU</span>;
        ISSN:<span class="NLM_cas:issn">1744-666X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The mortality of coronavirus disease 2019 (COVID-19) is frequently driven by an injurious immune response characterized by the development of acute respiratory distress syndrome (ARDS), endotheliitis, coagulopathy, and multi-organ failure.  This spectrum of hyperinflammation in COVID-19 is commonly referred to as cytokine storm syndrome (CSS).: Medline and Google Scholar were searched up until 15th of August 2020 for relevant literature.  Evidence supports a role of dysregulated immune responses in the immunopathogenesis of severe COVID-19.  CSS assocd. with SARS-CoV-2 shows similarities to the exuberant cytokine prodn. in some patients with viral infection (e.g.SARS-CoV-1) and may be confused with other syndromes of hyperinflammation like the cytokine release syndrome (CRS) in CAR-T cell therapy.  Interleukin (IL)-6, IL-8, and tumor necrosis factor-alpha have emerged as predictors of COVID-19 severity and in-hospital mortality.  Despite similarities, COVID-19-CSS appears to be distinct from HLH, MAS, and CRS, and the application of HLH diagnostic scores and criteria to COVID-19 is not supported by emerging data.  While immunosuppressive therapy with glucocorticoids has shown a mortality benefit, cytokine inhibitors may hold promise as 'rescue therapies' in severe COVID-19.  Given the arguably limited benefit in advanced disease, strategies to prevent the development of COVID-19-CSS are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxw7l66l9AeLVg90H21EOLACvtfcHk0lgkzs1OtqvTrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisVClsL7P&md5=6cd0d70b3c9d3f986e96ffba3248d03b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1080%2F1744666X.2021.1847084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F1744666X.2021.1847084%26sid%3Dliteratum%253Aachs%26aulast%3DAmigues%26aufirst%3DI.%26aulast%3DPearlman%26aufirst%3DA.%2BH.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DReid%26aufirst%3DP.%26aulast%3DRobinson%26aufirst%3DP.%2BC.%26aulast%3DSinha%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DA.%2BH.%26aulast%3DYoungstein%26aufirst%3DT.%26aulast%3DJayatilleke%26aufirst%3DA.%26aulast%3DKonig%26aufirst%3DM.%26atitle%3DCoronavirus%2520disease%25202019%253A%2520investigational%2520therapies%2520in%2520the%2520prevention%2520and%2520treatment%2520of%2520hyperinflammation%26jtitle%3DExpert%2520Rev.%2520Clin.%2520Immunol.%26date%3D2020%26volume%3D16%26spage%3D1185%26epage%3D1204%26doi%3D10.1080%2F1744666X.2021.1847084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Recovery
Collaborative Group</span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horby, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emberson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mafham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsell, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staplin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brightling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ustianowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmahi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadwick, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rege, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fegan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chappell, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faust, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffery, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juszczak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landray, M. J.</span></span> <span> </span><span class="NLM_article-title">Dexamethasone in hospitalized patients with COVID-19 - Preliminary report</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">693</span>– <span class="NLM_lpage">704</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1056%2FNEJMoa2021436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32678530" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2020&pages=693-704&author=Recovery%0ACollaborative+Groupauthor=P.+Horbyauthor=W.+S.+Limauthor=J.+R.+Embersonauthor=M.+Mafhamauthor=J.+L.+Bellauthor=L.+Linsellauthor=N.+Staplinauthor=C.+Brightlingauthor=A.+Ustianowskiauthor=E.+Elmahiauthor=B.+Prudonauthor=C.+Greenauthor=T.+Feltonauthor=D.+Chadwickauthor=K.+Regeauthor=C.+Feganauthor=L.+C.+Chappellauthor=S.+N.+Faustauthor=T.+Jakiauthor=K.+Jefferyauthor=A.+Montgomeryauthor=K.+Rowanauthor=E.+Juszczakauthor=J.+K.+Baillieauthor=R.+Haynesauthor=M.+J.+Landray&title=Dexamethasone+in+hospitalized+patients+with+COVID-19+-+Preliminary+report"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2021436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2021436%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3DHorby%26aufirst%3DP.%26aulast%3DLim%26aufirst%3DW.%2BS.%26aulast%3DEmberson%26aufirst%3DJ.%2BR.%26aulast%3DMafham%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DJ.%2BL.%26aulast%3DLinsell%26aufirst%3DL.%26aulast%3DStaplin%26aufirst%3DN.%26aulast%3DBrightling%26aufirst%3DC.%26aulast%3DUstianowski%26aufirst%3DA.%26aulast%3DElmahi%26aufirst%3DE.%26aulast%3DPrudon%26aufirst%3DB.%26aulast%3DGreen%26aufirst%3DC.%26aulast%3DFelton%26aufirst%3DT.%26aulast%3DChadwick%26aufirst%3DD.%26aulast%3DRege%26aufirst%3DK.%26aulast%3DFegan%26aufirst%3DC.%26aulast%3DChappell%26aufirst%3DL.%2BC.%26aulast%3DFaust%26aufirst%3DS.%2BN.%26aulast%3DJaki%26aufirst%3DT.%26aulast%3DJeffery%26aufirst%3DK.%26aulast%3DMontgomery%26aufirst%3DA.%26aulast%3DRowan%26aufirst%3DK.%26aulast%3DJuszczak%26aufirst%3DE.%26aulast%3DBaillie%26aufirst%3DJ.%2BK.%26aulast%3DHaynes%26aufirst%3DR.%26aulast%3DLandray%26aufirst%3DM.%2BJ.%26atitle%3DDexamethasone%2520in%2520hospitalized%2520patients%2520with%2520COVID-19%2520-%2520Preliminary%2520report%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D384%26spage%3D693%26epage%3D704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M. Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: drug targets and potential treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.+%C3%81.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ram%C3%ADrezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+drug+targets+and+potential+treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0lj-02-bM7HS1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%2B%25C3%2581.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRam%25C3%25ADrez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520drug%2520targets%2520and%2520potential%2520treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeFrancesco, L.</span></span> <span> </span><span class="NLM_article-title">COVID-19 antibodies on trial</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1252</span>, <span class="refDoi"> DOI: 10.1038/s41587-020-0732-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41587-020-0732-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33087898" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFGqtLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2020&pages=1242-1252&author=L.+DeFrancesco&title=COVID-19+antibodies+on+trial&doi=10.1038%2Fs41587-020-0732-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19 antibodies on trial</span></div><div class="casAuthors">DeFrancesco, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1242-1252</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  With the first readouts of trials of antibodies against COVID-19 appearing and others coming thick and fast, Nature Biotechnol. asked a group of experts to comment on the challenges and timelines for these products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjXGHF7tce4bVg90H21EOLACvtfcHk0lj-02-bM7HS1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFGqtLfI&md5=6b2a4fcc68271e9a8931f004da197b69</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41587-020-0732-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41587-020-0732-8%26sid%3Dliteratum%253Aachs%26aulast%3DDeFrancesco%26aufirst%3DL.%26atitle%3DCOVID-19%2520antibodies%2520on%2520trial%26jtitle%3DNat.%2520Biotechnol.%26date%3D2020%26volume%3D38%26spage%3D1242%26epage%3D1252%26doi%3D10.1038%2Fs41587-020-0732-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lye, D. C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hui, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montejano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spinner, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahass, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SenGupta, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyland, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osinusi, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaggar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brainard, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Towner, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullane, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marty, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tashima, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span> <span> </span><span class="NLM_article-title">GS-US-540-5773 Investigators, Remdesivir for 5 or 10 days in patients with severe Covid-19</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>383</i></span>,  <span class="NLM_fpage">1827</span>– <span class="NLM_lpage">1837</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa2015301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1056%2FNEJMoa2015301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32459919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1ylt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=383&publication_year=2020&pages=1827-1837&author=J.+D.+Goldmanauthor=D.+C.+B.+Lyeauthor=D.+S.+Huiauthor=K.+M.+Marksauthor=R.+Brunoauthor=R.+Montejanoauthor=C.+D.+Spinnerauthor=M.+Galliauthor=M.+Y.+Ahnauthor=R.+G.+Nahassauthor=Y.+S.+Chenauthor=D.+SenGuptaauthor=R.+H.+Hylandauthor=A.+O.+Osinusiauthor=H.+Caoauthor=C.+Blairauthor=X.+Weiauthor=A.+Gaggarauthor=D.+M.+Brainardauthor=W.+J.+Townerauthor=J.+Munozauthor=K.+M.+Mullaneauthor=F.+M.+Martyauthor=K.+T.+Tashimaauthor=G.+Diazauthor=A.+Subramanian&title=GS-US-540-5773+Investigators%2C+Remdesivir+for+5+or+10+days+in+patients+with+severe+Covid-19&doi=10.1056%2FNEJMoa2015301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Remdesivir for 5 or 10 days in patients with severe Covid-19</span></div><div class="casAuthors">Goldman, Jason D.; Lye, David C. B.; Hui, David S.; Marks, Kristen M.; Bruno, Raffaele; Montejano, Rocio; Spinner, Christoph D.; Galli, Massimo; Ahn, Mi-Young; Nahass, Ronald G.; Chen, Yao-Shen; Gupta, Devi Sen; Hyland, Robert H.; Osinusi, Anu O.; Cao, Huyen; Blair, Christiana; Wei, Xuelian; Gaggar, Anuj; Brainard, Diana M.; Towner, William J.; Munoz, Jose; Mullane, Kathleen M.; Marty, Francisco M.; Tashima, Karen T.; Diaz, George; Subramanian, Aruna</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">383</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">1827-1837</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Remdesivir is an RNA polymerase inhibitor with potent antiviral activity in vitro and efficacy in animal models of coronavirus disease 2019 (Covid-19).  Methods: The authors conducted a randomized, open-label, phase 3 trial involving hospitalized patients with confirmed SARS-CoV-2 infection, oxygen satn. of 94% or less while they were breathing ambient air, and radiol. evidence of pneumonia.  Patients were randomly assigned in a 1:1 ratio to receive i.v. remdesivir for either 5 days or 10 days.  All patients received 200 mg of remdesivir on day 1 and 100 mg once daily on subsequent days.  The primary end point was clin. status on day 14, assessed on a 7-point ordinal scale.  Results: In total, 397 patients underwent randomization and began treatment (200 patients for 5 days and 197 for 10 days).  The median duration of treatment was 5 days (interquartile range, 5 to 5) in the 5-day group and 9 days (interquartile range, 5 to 10) in the 10-day group.  At baseline, patients randomly assigned to the 10-day group had significantly worse clin. status than those assigned to the 5-day group (P = 0.02).  By day 14, a clin. improvement of 2 points or more on the ordinal scale occurred in 64% of patients in the 5-day group and in 54% in the 10-day group.  After adjustment for baseline clin. status, patients in the 10-day group had a distribution in clin. status at day 14 that was similar to that among patients in the 5-day group (P = 0.14).  The most common adverse events were nausea (9% of patients), worsening respiratory failure (8%), elevated alanine aminotransferase level (7%), and constipation (7%).  Conclusions: In patients with severe Covid-19 not requiring mech. ventilation, the authors' trial did not show a significant difference between a 5-day course and a 10-day course of remdesivir.  With no placebo control, however, the magnitude of benefit cannot be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq5F3H7ZXqCrVg90H21EOLACvtfcHk0lhkTSKwdtLSqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1ylt7bK&md5=bac0eb1b32dd628266ed1412cb9a9e5d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa2015301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa2015301%26sid%3Dliteratum%253Aachs%26aulast%3DGoldman%26aufirst%3DJ.%2BD.%26aulast%3DLye%26aufirst%3DD.%2BC.%2BB.%26aulast%3DHui%26aufirst%3DD.%2BS.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DBruno%26aufirst%3DR.%26aulast%3DMontejano%26aufirst%3DR.%26aulast%3DSpinner%26aufirst%3DC.%2BD.%26aulast%3DGalli%26aufirst%3DM.%26aulast%3DAhn%26aufirst%3DM.%2BY.%26aulast%3DNahass%26aufirst%3DR.%2BG.%26aulast%3DChen%26aufirst%3DY.%2BS.%26aulast%3DSenGupta%26aufirst%3DD.%26aulast%3DHyland%26aufirst%3DR.%2BH.%26aulast%3DOsinusi%26aufirst%3DA.%2BO.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DBlair%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DGaggar%26aufirst%3DA.%26aulast%3DBrainard%26aufirst%3DD.%2BM.%26aulast%3DTowner%26aufirst%3DW.%2BJ.%26aulast%3DMunoz%26aufirst%3DJ.%26aulast%3DMullane%26aufirst%3DK.%2BM.%26aulast%3DMarty%26aufirst%3DF.%2BM.%26aulast%3DTashima%26aufirst%3DK.%2BT.%26aulast%3DDiaz%26aufirst%3DG.%26aulast%3DSubramanian%26aufirst%3DA.%26atitle%3DGS-US-540-5773%2520Investigators%252C%2520Remdesivir%2520for%25205%2520or%252010%2520days%2520in%2520patients%2520with%2520severe%2520Covid-19%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D383%26spage%3D1827%26epage%3D1837%26doi%3D10.1056%2FNEJMoa2015301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H. E.</span></span> <span> </span><span class="NLM_article-title">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1499</span>– <span class="NLM_lpage">1504</span>, <span class="refDoi"> DOI: 10.1126/science.abc1560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1126%2Fscience.abc1560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32358203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1499-1504&author=W.+Yinauthor=C.+Maoauthor=X.+Luanauthor=D.+D.+Shenauthor=Q.+Shenauthor=H.+Suauthor=X.+Wangauthor=F.+Zhouauthor=W.+Zhaoauthor=M.+Gaoauthor=S.+Changauthor=Y.+C.+Xieauthor=G.+Tianauthor=H.+W.+Jiangauthor=S.+C.+Taoauthor=J.+Shenauthor=Y.+Jiangauthor=H.+Jiangauthor=Y.+Xuauthor=S.+Zhangauthor=Y.+Zhangauthor=H.+E.+Xu&title=Structural+basis+for+inhibition+of+the+RNA-dependent+RNA+polymerase+from+SARS-CoV-2+by+remdesivir&doi=10.1126%2Fscience.abc1560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir</span></div><div class="casAuthors">Yin, Wanchao; Mao, Chunyou; Luan, Xiaodong; Shen, Dan-Dan; Shen, Qingya; Su, Haixia; Wang, Xiaoxi; Zhou, Fulai; Zhao, Wenfeng; Gao, Minqi; Chang, Shenghai; Xie, Yuan-Chao; Tian, Guanghui; Jiang, He-Wei; Tao, Sheng-Ce; Shen, Jingshan; Jiang, Yi; Jiang, Hualiang; Xu, Yechun; Zhang, Shuyang; Zhang, Yan; Xu, H. Eric</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6498</span>),
    <span class="NLM_cas:pages">1499-1504</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The pandemic of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has become a global crisis.  Replication of SARS-CoV-2 requires the viral RNA-dependent RNA polymerase (RdRp) enzyme, a target of the antiviral drug remdesivir.  Here, we report the cryo-electron microscopy structure of the SARS-CoV-2 RdRp, both in the apo form at 2.8-angstrom resoln. and in complex with a 50-base template-primer RNA and remdesivir at 2.5-angstrom resoln.  The complex structure reveals that the partial double-stranded RNA template is inserted into the central channel of the RdRp, where remdesivir is covalently incorporated into the primer strand at the first replicated base pair, and terminates chain elongation.  Our structures provide insights into the mechanism of viral RNA replication and a rational template for drug design to combat the viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlizem_f4e1LVg90H21EOLACvtfcHk0lhkTSKwdtLSqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1CltLfK&md5=e5d7c8a1c27a02fdbfa9381482a0c3e8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.abc1560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abc1560%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DC.%26aulast%3DLuan%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DD.%2BD.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DY.%2BC.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%2BW.%26aulast%3DTao%26aufirst%3DS.%2BC.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DH.%2BE.%26atitle%3DStructural%2520basis%2520for%2520inhibition%2520of%2520the%2520RNA-dependent%2520RNA%2520polymerase%2520from%2520SARS-CoV-2%2520by%2520remdesivir%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1499%26epage%3D1504%26doi%3D10.1126%2Fscience.abc1560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menachery, V. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gralinski, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leist, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pyrc, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trantcheva, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannister, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babusis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spahn, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmiotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denison, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baric, R. S.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antiviral GS-5734 inhibits both epidemic and zoonotic coronaviruses</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">eaal3653</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.aal3653</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1126%2Fscitranslmed.aal3653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28659436" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&author=T.+P.+Sheahanauthor=A.+C.+Simsauthor=R.+L.+Grahamauthor=V.+D.+Menacheryauthor=L.+E.+Gralinskiauthor=J.+B.+Caseauthor=S.+R.+Leistauthor=K.+Pyrcauthor=J.+Y.+Fengauthor=I.+Trantchevaauthor=R.+Bannisterauthor=Y.+Parkauthor=D.+Babusisauthor=M.+O.+Clarkeauthor=R.+L.+Mackmanauthor=J.+E.+Spahnauthor=C.+A.+Palmiottiauthor=D.+Siegelauthor=A.+S.+Rayauthor=T.+Cihlarauthor=R.+Jordanauthor=M.+R.+Denisonauthor=R.+S.+Baric&title=Broad-spectrum+antiviral+GS-5734+inhibits+both+epidemic+and+zoonotic+coronaviruses&doi=10.1126%2Fscitranslmed.aal3653"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aal3653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aal3653%26sid%3Dliteratum%253Aachs%26aulast%3DSheahan%26aufirst%3DT.%2BP.%26aulast%3DSims%26aufirst%3DA.%2BC.%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DMenachery%26aufirst%3DV.%2BD.%26aulast%3DGralinski%26aufirst%3DL.%2BE.%26aulast%3DCase%26aufirst%3DJ.%2BB.%26aulast%3DLeist%26aufirst%3DS.%2BR.%26aulast%3DPyrc%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%2BY.%26aulast%3DTrantcheva%26aufirst%3DI.%26aulast%3DBannister%26aufirst%3DR.%26aulast%3DPark%26aufirst%3DY.%26aulast%3DBabusis%26aufirst%3DD.%26aulast%3DClarke%26aufirst%3DM.%2BO.%26aulast%3DMackman%26aufirst%3DR.%2BL.%26aulast%3DSpahn%26aufirst%3DJ.%2BE.%26aulast%3DPalmiotti%26aufirst%3DC.%2BA.%26aulast%3DSiegel%26aufirst%3DD.%26aulast%3DRay%26aufirst%3DA.%2BS.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DJordan%26aufirst%3DR.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26atitle%3DBroad-spectrum%2520antiviral%2520GS-5734%2520inhibits%2520both%2520epidemic%2520and%2520zoonotic%2520coronaviruses%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2017%26volume%3D9%26doi%3D10.1126%2Fscitranslmed.aal3653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinnon, K. H.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihlar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denison, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baric, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheahan, T. P.</span></span> <span> </span><span class="NLM_article-title">Broad spectrum antiviral remdesivir inhibits human endemic and zoonotic deltacoronaviruses with a highly divergent RNA dependent RNA polymerase</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">104541</span>, <span class="refDoi"> DOI: 10.1016/j.antiviral.2019.104541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.antiviral.2019.104541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=31233808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Git7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2019&pages=104541&author=A.+J.+Brownauthor=J.+J.+Wonauthor=R.+L.+Grahamauthor=K.+H.+Dinnonauthor=A.+C.+Simsauthor=J.+Y.+Fengauthor=T.+Cihlarauthor=M.+R.+Denisonauthor=R.+S.+Baricauthor=T.+P.+Sheahan&title=Broad+spectrum+antiviral+remdesivir+inhibits+human+endemic+and+zoonotic+deltacoronaviruses+with+a+highly+divergent+RNA+dependent+RNA+polymerase&doi=10.1016%2Fj.antiviral.2019.104541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Broad spectrum antiviral remdesivir inhibits human endemic and zoonotic deltacoronaviruses with a highly divergent RNA dependent RNA polymerase</span></div><div class="casAuthors">Brown, Ariane J.; Won, John J.; Graham, Rachel L.; Dinnon, Kenneth H., III; Sims, Amy C.; Feng, Joy Y.; Cihlar, Tomas; Denison, Mark R.; Baric, Ralph S.; Sheahan, Timothy P.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104541</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The genetically diverse Orthocoronavirinae (CoV) family is prone to cross species transmission and disease emergence in both humans and livestock.  Viruses similar to known epidemic strains circulating in wild and domestic animals further increase the probability of emergence in the future.  Currently, there are no approved therapeutics for any human CoV presenting a clear unmet medical need.  Remdesivir (RDV, GS-5734) is a monophosphoramidate prodrug of an adenosine analog with potent activity against an array of RNA virus families including Filoviridae, Paramyxoviridae, Pneumoviridae, and Orthocoronavirinae, through the targeting of the viral RNA dependent RNA polymerase (RdRp).  We developed multiple assays to further define the breadth of RDV antiviral activity against the CoV family.  Here, we show potent antiviral activity of RDV against endemic human CoVs OC43 (HCoV-OC43) and 229E (HCoV-229E) with submicromolar EC50 values.  Of known CoVs, the members of the deltacoronavirus genus have the most divergent RdRp as compared to SARS- and MERS-CoV and both avian and porcine members harbor a native residue in the RdRp that confers resistance in beta-CoVs.  Nevertheless, RDV is highly efficacious against porcine deltacoronavirus (PDCoV).  These data further extend the known breadth and antiviral activity of RDV to include both contemporary human and highly divergent zoonotic CoV and potentially enhance our ability to fight future emerging CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtZuP2J4qyxbVg90H21EOLACvtfcHk0lgZh7IoxnUIMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Git7zI&md5=83a831806f8e8520713283d35de2bd7e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2019.104541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2019.104541%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DWon%26aufirst%3DJ.%2BJ.%26aulast%3DGraham%26aufirst%3DR.%2BL.%26aulast%3DDinnon%26aufirst%3DK.%2BH.%26aulast%3DSims%26aufirst%3DA.%2BC.%26aulast%3DFeng%26aufirst%3DJ.%2BY.%26aulast%3DCihlar%26aufirst%3DT.%26aulast%3DDenison%26aufirst%3DM.%2BR.%26aulast%3DBaric%26aufirst%3DR.%2BS.%26aulast%3DSheahan%26aufirst%3DT.%2BP.%26atitle%3DBroad%2520spectrum%2520antiviral%2520remdesivir%2520inhibits%2520human%2520endemic%2520and%2520zoonotic%2520deltacoronaviruses%2520with%2520a%2520highly%2520divergent%2520RNA%2520dependent%2520RNA%2520polymerase%26jtitle%3DAntiviral%2520Res.%26date%3D2019%26volume%3D169%26spage%3D104541%26doi%3D10.1016%2Fj.antiviral.2019.104541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiew, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K. O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">11754</span>– <span class="NLM_lpage">11762</span>, <span class="refDoi"> DOI: 10.1128/JVI.01348-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.01348-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=22915796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=11754-11762&author=Y.+Kimauthor=S.+Lovellauthor=K.+C.+Tiewauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=K.+P.+Battaileauthor=W.+C.+Groutasauthor=K.+O.+Chang&title=Broad-spectrum+antivirals+against+3C+or+3C-like+proteases+of+picornaviruses%2C+noroviruses%2C+and+coronaviruses&doi=10.1128%2FJVI.01348-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span></div><div class="casAuthors">Kim, Yunjeong; Lovell, Scott; Tiew, Kok-Chuan; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Battaile, Kevin P.; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11754-11762</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phylogenetic anal. has demonstrated that some pos.-sense RNA viruses can be classified into the picornavirus-like super-cluster, which includes picornaviruses, caliciviruses, and coronaviruses.  These viruses possess 3C or 3C-like proteases (3Cpro or 3CLpro, resp.), which contain a typical chymotrypsin-like fold and a catalytic triad (or dyad) with a Cys residue as a nucleophile.  The conserved key sites of 3Cpro or 3CLpro may serve as attractive targets for the design of broad-spectrum antivirals for multiple viruses in the supercluster.  We previously reported the structure-based design and synthesis of potent protease inhibitors of Norwalk virus (NV), a member of the Caliciviridae family.  We report herein the broad-spectrum antiviral activities of three compds. possessing a common dipeptidyl residue with different warheads, i.e., an aldehyde (GC373), a bisulfite adduct (GC376), and an α-ketoamide (GC375), against viruses that belong to the supercluster.  All compds. were highly effective against the majority of tested viruses, with half-maximal inhibitory concns. in the high nanomolar or low micromolar range in enzyme- and/or cell-based assays and with high therapeutic indexes.  We also report the high-resoln. X-ray cocrystal structures of NV 3CLpro-, poliovirus 3Cpro-, and transmissible gastroenteritis virus 3CLpro- GC376 inhibitor complexes, which show the compd. covalently bound to a nucleophilic Cys residue in the catalytic site of the corresponding protease.  We conclude that these compds. have the potential to be developed as antiviral therapeutics aimed at a single virus or multiple viruses in the picornavirus-like supercluster by targeting 3Cpro or 3CLpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDn4MUH7bIlLVg90H21EOLACvtfcHk0ljRm38-GGTs6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M&md5=c7fe984b52490199a2c2ef333c22fb98</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.01348-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01348-12%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DTiew%26aufirst%3DK.%2BC.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.%2BO.%26atitle%3DBroad-spectrum%2520antivirals%2520against%25203C%2520or%25203C-like%2520proteases%2520of%2520picornaviruses%252C%2520noroviruses%252C%2520and%2520coronaviruses%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D11754%26epage%3D11762%26doi%3D10.1128%2FJVI.01348-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arutyunova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shields, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Belkum, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyce, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrrell, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. J.</span></span> <span> </span><span class="NLM_article-title">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4282</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18096-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41467-020-18096-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32855413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4282&author=W.+Vuongauthor=M.+B.+Khanauthor=C.+Fischerauthor=E.+Arutyunovaauthor=T.+Lamerauthor=J.+Shieldsauthor=H.+A.+Saffranauthor=R.+T.+McKayauthor=M.+J.+van+Belkumauthor=M.+A.+Joyceauthor=H.+S.+Youngauthor=D.+L.+Tyrrellauthor=J.+C.+Vederasauthor=M.+J.+Lemieux&title=Feline+coronavirus+drug+inhibits+the+main+protease+of+SARS-CoV-2+and+blocks+virus+replication&doi=10.1038%2Fs41467-020-18096-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Feline coronavirus drug inhibits the main protease of SARS-CoV-2 and blocks virus replication</span></div><div class="casAuthors">Vuong, Wayne; Khan, Muhammad Bashir; Fischer, Conrad; Arutyunova, Elena; Lamer, Tess; Shields, Justin; Saffran, Holly A.; McKay, Ryan T.; van Belkum, Marco J.; Joyce, Michael A.; Young, Howard S.; Tyrrell, D. Lorne; Vederas, John C.; Lemieux, M. Joanne</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4282</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The main protease, Mpro (or 3CLpro) in SARS-CoV-2 is a viable drug target because of its essential role in the cleavage of the virus polypeptide.  Feline infectious peritonitis, a fatal coronavirus infection in cats, was successfully treated previously with a prodrug GC376, a dipeptide-based protease inhibitor.  We show the prodrug and its parent GC373, are effective inhibitors of the Mpro from both SARS-CoV and SARS-CoV-2 with IC50 values in the nanomolar range.  Crystal structures of SARS-CoV-2 Mpro with these inhibitors have a covalent modification of the nucleophilic Cys145.  NMR anal. reveals that inhibition proceeds via reversible formation of a hemithioacetal.  GC373 and GC376 are potent inhibitors of SARS-CoV-2 replication in cell culture.  They are strong drug candidates for the treatment of human coronavirus infections because they have already been successful in animals.  The work here lays the framework for their use in human trials for the treatment of COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMA7W4XtRQrbVg90H21EOLACvtfcHk0ljRm38-GGTs6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhslWqtbzN&md5=c06d2dc9a65ae289505fe120d41d8885</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18096-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18096-2%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DW.%26aulast%3DKhan%26aufirst%3DM.%2BB.%26aulast%3DFischer%26aufirst%3DC.%26aulast%3DArutyunova%26aufirst%3DE.%26aulast%3DLamer%26aufirst%3DT.%26aulast%3DShields%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DH.%2BA.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3Dvan%2BBelkum%26aufirst%3DM.%2BJ.%26aulast%3DJoyce%26aufirst%3DM.%2BA.%26aulast%3DYoung%26aufirst%3DH.%2BS.%26aulast%3DTyrrell%26aufirst%3DD.%2BL.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DLemieux%26aufirst%3DM.%2BJ.%26atitle%3DFeline%2520coronavirus%2520drug%2520inhibits%2520the%2520main%2520protease%2520of%2520SARS-CoV-2%2520and%2520blocks%2520virus%2520replication%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4282%26doi%3D10.1038%2Fs41467-020-18096-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G. F.</span></span> <span> </span><span class="NLM_article-title">Both Boceprevir and GC376 efficaciously inhibit SARS-CoV-2 by targeting its main protease</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">4417</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-18233-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41467-020-18233-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32887884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVSmsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=4417&author=L.+Fuauthor=F.+Yeauthor=Y.+Fengauthor=F.+Yuauthor=Q.+Wangauthor=Y.+Wuauthor=C.+Zhaoauthor=H.+Sunauthor=B.+Huangauthor=P.+Niuauthor=H.+Songauthor=Y.+Shiauthor=X.+Liauthor=W.+Tanauthor=J.+Qiauthor=G.+F.+Gao&title=Both+Boceprevir+and+GC376+efficaciously+inhibit+SARS-CoV-2+by+targeting+its+main+protease&doi=10.1038%2Fs41467-020-18233-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Both Boceprevir and GC376 efficaciously inhibit SARS-CoV-2 by targeting its main protease</span></div><div class="casAuthors">Fu, Lifeng; Ye, Fei; Feng, Yong; Yu, Feng; Wang, Qisheng; Wu, Yan; Zhao, Cheng; Sun, Huan; Huang, Baoying; Niu, Peihua; Song, Hao; Shi, Yi; Li, Xuebing; Tan, Wenjie; Qi, Jianxun; Gao, George Fu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4417</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: COVID-19 was declared a pandemic on March 11 by WHO, due to its great threat to global public health.  The coronavirus main protease (Mpro, also called 3CLpro) is essential for processing and maturation of the viral polyprotein, therefore recognized as an attractive drug target.  Here we show that a clin. approved anti-HCV drug, Boceprevir, and a pre-clin. inhibitor against feline infectious peritonitis (corona) virus (FIPV), GC376, both efficaciously inhibit SARS-CoV-2 in Vero cells by targeting Mpro.  Moreover, combined application of GC376 with Remdesivir, a nucleotide analog that inhibits viral RNA dependent RNA polymerase (RdRp), results in sterilizing additive effect.  Further structural anal. reveals binding of both inhibitors to the catalytically active side of SARS-CoV-2 protease Mpro as main mechanism of inhibition.  Our findings may provide crit. information for the optimization and design of more potent inhibitors against the emerging SARS-CoV-2 virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpt43KcNqLbkLVg90H21EOLACvtfcHk0ljk3vOiM5wa-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVSmsr3F&md5=273b11d16241499f15103885be398b23</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-18233-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-18233-x%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DL.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DNiu%26aufirst%3DP.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DG.%2BF.%26atitle%3DBoth%2520Boceprevir%2520and%2520GC376%2520efficaciously%2520inhibit%2520SARS-CoV-2%2520by%2520targeting%2520its%2520main%2520protease%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D4417%26doi%3D10.1038%2Fs41467-020-18233-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ricagno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egloff, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulferts, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nurizzo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campanacci, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cambillau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canard, B.</span></span> <span> </span><span class="NLM_article-title">Crystal structure and mechanistic determinants of SARS coronavirus nonstructural protein 15 define an endoribonuclease family</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">11892</span>– <span class="NLM_lpage">11897</span>, <span class="refDoi"> DOI: 10.1073/pnas.0601708103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1073%2Fpnas.0601708103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=16882730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVSgsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2006&pages=11892-11897&author=S.+Ricagnoauthor=M.+P.+Egloffauthor=R.+Ulfertsauthor=B.+Coutardauthor=D.+Nurizzoauthor=V.+Campanacciauthor=C.+Cambillauauthor=J.+Ziebuhrauthor=B.+Canard&title=Crystal+structure+and+mechanistic+determinants+of+SARS+coronavirus+nonstructural+protein+15+define+an+endoribonuclease+family&doi=10.1073%2Fpnas.0601708103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and mechanistic determinants of SARS coronavirus nonstructural protein 15 define an endoribonuclease family</span></div><div class="casAuthors">Ricagno, Stefano; Egloff, Marie-Pierre; Ulferts, Rachel; Coutard, Bruno; Nurizzo, Didier; Campanacci, Valerie; Cambillau, Christian; Ziebuhr, John; Canard, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">11892-11897</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The ≈30-kb coronavirus (+)RNA genome is replicated and transcribed by a membrane-bound replicase complex made up of 16 viral nonstructural proteins (nsp) with multiple enzymic activities.  The complex includes an RNA endonuclease that is conserved among nidoviruses but no other RNA virus, making it a genetic marker of this virus order.  Nonstructural protein 15 (nsp15) exhibits Mn2+-dependent nidovirus uridylate-specific endoribonuclease (NendoU) activity which cleaves 2'-3'-cyclic phosphates 5' to the cleaved bond.  Neither biochem. nor sequence homol. criteria allow a classification of nsp15 into existing endonuclease families.  Here, we report the crystal structure of the severe acute respiratory syndrome coronavirus nsp15 at 2.6-Å resoln.  Nsp15 exhibits a unique fold and assembles into a toric hexamer with six potentially active, peripheric catalytic sites.  The structure and the spatial arrangement of the catalytic residues into an RNase A-like active site define a sep. endonuclease family, endoU, and represent another spectacular example of convergent evolution toward an enzymic function that is critically involved in the coronavirus replication cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UQQC4Vhe4LVg90H21EOLACvtfcHk0ljk3vOiM5wa-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVSgsLs%253D&md5=de25bd173a87e86e5bf7d175d6e12977</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0601708103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0601708103%26sid%3Dliteratum%253Aachs%26aulast%3DRicagno%26aufirst%3DS.%26aulast%3DEgloff%26aufirst%3DM.%2BP.%26aulast%3DUlferts%26aufirst%3DR.%26aulast%3DCoutard%26aufirst%3DB.%26aulast%3DNurizzo%26aufirst%3DD.%26aulast%3DCampanacci%26aufirst%3DV.%26aulast%3DCambillau%26aufirst%3DC.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DCanard%26aufirst%3DB.%26atitle%3DCrystal%2520structure%2520and%2520mechanistic%2520determinants%2520of%2520SARS%2520coronavirus%2520nonstructural%2520protein%252015%2520define%2520an%2520endoribonuclease%2520family%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2006%26volume%3D103%26spage%3D11892%26epage%3D11897%26doi%3D10.1073%2Fpnas.0601708103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Crystallization and preliminary X-ray crystallographic analysis of a nonstructural protein 15 mutant from Human coronavirus 229E</span>. <i>Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1156</span>– <span class="NLM_lpage">1160</span>, <span class="refDoi"> DOI: 10.1107/S2053230X15007359</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1107%2FS2053230X15007359" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVymt7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1156-1160&author=T.+Huoauthor=X.+Liu&title=Crystallization+and+preliminary+X-ray+crystallographic+analysis+of+a+nonstructural+protein+15+mutant+from+Human+coronavirus+229E&doi=10.1107%2FS2053230X15007359"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallization and preliminary x-ray crystallographic analysis of a nonstructural protein 15 mutant from human coronavirus 229E</span></div><div class="casAuthors">Huo, Tong; Liu, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1156-1160</span>CODEN:
                <span class="NLM_cas:coden">ACSFEN</span>;
        ISSN:<span class="NLM_cas:issn">2053-230X</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Nonstructural protein 15 (nsp15), also called Nsp15 endoribonuclease, is a gene product of open reading frame 1b (ORF 1b) in coronaviruses.  It is an important enzyme in the transcription/replication process involved in discontinuous neg.-strand RNA synthesis.  Here, mutants of nsp15 from human coronavirus 229E (HCoV-229E) were made based on structural anal. of the homologous nsp15 proteins in severe acute respiratory syndrome coronavirus (SARS-CoV) and mouse hepatitis virus (MHV).  The I26A/N52A mutant of nsp15 was overexpressed, purified, and crystd., and this mutant led to a trimeric form rather than hexamers or monomers.  Crystals of trimeric nsp15 were obtained by the hanging-drop vapor-diffusion method using polyethylene glycol as a precipitant and diffracted to 2.5 Å resoln.  The crystals belonged to space group C2221, with unit-cell parameters a = 85.9, b = 137.5, c = 423.1 Å, and α = β = γ = 90°.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXjlzkzcNBbrVg90H21EOLACvtfcHk0ljk3vOiM5wa-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVymt7fN&md5=1eb0673f34363cf2c0ccd24991ed4133</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1107%2FS2053230X15007359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053230X15007359%26sid%3Dliteratum%253Aachs%26aulast%3DHuo%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DCrystallization%2520and%2520preliminary%2520X-ray%2520crystallographic%2520analysis%2520of%2520a%2520nonstructural%2520protein%252015%2520mutant%2520from%2520Human%2520coronavirus%2520229E%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520F%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2015%26volume%3D71%26spage%3D1156%26epage%3D1160%26doi%3D10.1107%2FS2053230X15007359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrzejczak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maltseva, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilamowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endres, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godzik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalska, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1596</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/pro.3873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1002%2Fpro.3873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32304108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosVWrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2020&pages=1596-1605&author=Y.+Kimauthor=R.+Jedrzejczakauthor=N.+I.+Maltsevaauthor=M.+Wilamowskiauthor=M.+Endresauthor=A.+Godzikauthor=K.+Michalskaauthor=A.+Joachimiak&title=Crystal+structure+of+Nsp15+endoribonuclease+NendoU+from+SARS-CoV-2&doi=10.1002%2Fpro.3873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of Nsp15 endoribonuclease NendoU from SARS-CoV-2</span></div><div class="casAuthors">Kim, Youngchang; Jedrzejczak, Robert; Maltseva, Natalia I.; Wilamowski, Mateusz; Endres, Michael; Godzik, Adam; Michalska, Karolina; Joachimiak, Andrzej</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1596-1605</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) is rapidly spreading around the world.  There is no existing vaccine or proven drug to prevent infections and stop virus proliferation.  Although this virus is similar to human and animal SARS-CoVs and Middle East Respiratory Syndrome coronavirus (MERS-CoVs), the detailed information about SARS-CoV-2 proteins structures and functions is urgently needed to rapidly develop effective vaccines, antibodies, and antivirals.  We applied high-throughput protein prodn. and structure detn. pipeline at the Center for Structural Genomics of Infectious Diseases to produce SARS-CoV-2 proteins and structures.  Here we report two high-resoln. crystal structures of endoribonuclease Nsp15/NendoU.  We compare these structures with previously reported homologs from SARS and MERS coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7HBZSCjMXGrVg90H21EOLACvtfcHk0li387TMSdmTqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosVWrsbo%253D&md5=28cfa012064ca5416af49d55d53b1b46</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fpro.3873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.3873%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DJedrzejczak%26aufirst%3DR.%26aulast%3DMaltseva%26aufirst%3DN.%2BI.%26aulast%3DWilamowski%26aufirst%3DM.%26aulast%3DEndres%26aufirst%3DM.%26aulast%3DGodzik%26aufirst%3DA.%26aulast%3DMichalska%26aufirst%3DK.%26aulast%3DJoachimiak%26aufirst%3DA.%26atitle%3DCrystal%2520structure%2520of%2520Nsp15%2520endoribonuclease%2520NendoU%2520from%2520SARS-CoV-2%26jtitle%3DProtein%2520Sci.%26date%3D2020%26volume%3D29%26spage%3D1596%26epage%3D1605%26doi%3D10.1002%2Fpro.3873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joachimiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">New antiviral target revealed by the hexameric structure of mouse hepatitis virus nonstructural protein nsp15</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">7909</span>– <span class="NLM_lpage">7917</span>, <span class="refDoi"> DOI: 10.1128/JVI.00525-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.00525-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=16873248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD28Xot1yrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2006&pages=7909-7917&author=X.+Xuauthor=Y.+Zhaiauthor=F.+Sunauthor=Z.+Louauthor=D.+Suauthor=Y.+Xuauthor=R.+Zhangauthor=A.+Joachimiakauthor=X.+C.+Zhangauthor=M.+Bartlamauthor=Z.+Rao&title=New+antiviral+target+revealed+by+the+hexameric+structure+of+mouse+hepatitis+virus+nonstructural+protein+nsp15&doi=10.1128%2FJVI.00525-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">New antiviral target revealed by the hexameric structure of mouse hepatitis virus nonstructural protein nsp15</span></div><div class="casAuthors">Xu, Xiaoling; Zhai, Yujia; Sun, Fei; Lou, Zhiyong; Su, Dan; Xu, Yuanyuan; Zhang, Rongguang; Joachimiak, Andrzej; Zhang, Xuejun C.; Bartlam, Mark; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7909-7917</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The unique coronavirus transcription/replication machinery comprised of multiple virus-encoded nonstructural proteins (nsp) plays a vital role during initial and intermediate phases of the viral life cycle.  The crystal structure of mouse hepatitis virus strain A59 (MHV-A59) nsp15 is reported at 2.15-Å resoln.  Nsp15 is an XendoU endoribonuclease and is the first one from this family to have its structure unveiled.  The MHV-A59 nsp15 monomer structure has a novel protein fold.  Two nsp15 trimers form a back-to-back hexamer that is believed to be the functional unit.  The structure reveals the catalytic site including the highly conserved residues His262, His277, and Lys317, which is supported by mutagenesis anal.  Gel filtration and enzyme activity assays confirmed that the hexamer is the active form for nsp15 and demonstrate the specificity of nsp15 for uridylate.  The high sequence conservation of nsp15 in coronaviruses, including that of severe acute respiratory syndrome, suggests that this protein may provide a new target for the design of antiviral therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdhBRvFvNjg7Vg90H21EOLACvtfcHk0li387TMSdmTqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xot1yrtrw%253D&md5=a955ca6de0725a654ed8a67ceb1cfbd2</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1128%2FJVI.00525-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00525-06%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DZhai%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DSu%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DJoachimiak%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DX.%2BC.%26aulast%3DBartlam%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DNew%2520antiviral%2520target%2520revealed%2520by%2520the%2520hexameric%2520structure%2520of%2520mouse%2520hepatitis%2520virus%2520nonstructural%2520protein%2520nsp15%26jtitle%3DJ.%2520Virol.%26date%3D2006%26volume%3D80%26spage%3D7909%26epage%3D7917%26doi%3D10.1128%2FJVI.00525-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ming, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span> <span> </span><span class="NLM_article-title">Structural and biochemical characterization of endoribonuclease Nsp15 encoded by Middle East Respiratory Syndrome Coronavirus</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">e00893</span>– <span class="NLM_lpage">e00918</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.00893-18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=30135128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvFemsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2018&pages=e00893-e00918&author=L.+Zhangauthor=L.+Liauthor=L.+Yanauthor=Z.+Mingauthor=Z.+Jiaauthor=Z.+Louauthor=Z.+Rao&title=Structural+and+biochemical+characterization+of+endoribonuclease+Nsp15+encoded+by+Middle+East+Respiratory+Syndrome+Coronavirus"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and biochemical characterization of endoribonuclease nsp15 encoded by middle east respiratory syndrome coronavirus</span></div><div class="casAuthors">Zhang, Lianqi; Li, Lei; Yan, Liming; Ming, Zhenhua; Jia, Zhihui; Lou, Zhiyong; Rao, Zihe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">e00893-18/1-e00893-18/16</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">1098-5514</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Nonstructural protein 15 (Nsp15) encoded by coronavirus (CoV) is a nidoviral uridylate-specific endoribonuclease (NendoU) that plays an essential role in the life cycle of the virus.  Structural information on this crucial protein from the Middle East respiratory syndrome CoV (MERS-CoV), which is lethally pathogenic and has caused severe respiratory diseases worldwide, is lacking.  Here, we detd. the crystal structure of MERS-CoV Nsp15 at a 2.7-Å resoln. and performed the relevant biochem. assays to study how NendoU activity is regulated.  Although the overall structure is conserved, MERS-CoV Nsp15 shows unique and novel features compared to its homologs.  Serine substitution of residue F285, which harbors an arom. side chain that disturbs RNA binding compared with that of other homologs, increases catalytic activity.  Mutations of residues residing on the oligomerization interfaces that distort hexamerization, namely, N38A, Y58A, and N157A, decrease thermostability, decrease affinity of binding with RNA, and reduce the NendoU activity of Nsp15.  In contrast, mutant D39A exhibits increased activity and a higher substrate binding capacity.  Importantly, Nsp8 was found to interact with both monomeric and hexameric Nsp15.  The Nsp7/Nsp8 complex displays a higher binding affinity for Nsp15.  Furthermore, Nsp8 and the Nsp7/Nsp8 complex also enhance the NendoU activity of hexameric Nsp15 in vitro.  Taking the findings together, this work first provides evidence on how the activity of Nsp15 may be functionally mediated by catalytic residues, oligomeric assembly, RNA binding efficiency, or the possible assocn. with other nonstructural proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdCS6GYJAivLVg90H21EOLACvtfcHk0li387TMSdmTqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvFemsbo%253D&md5=e1379df18eefcb703a34d2acb34eb7fc</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FJVI.00893-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00893-18%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DMing%26aufirst%3DZ.%26aulast%3DJia%26aufirst%3DZ.%26aulast%3DLou%26aufirst%3DZ.%26aulast%3DRao%26aufirst%3DZ.%26atitle%3DStructural%2520and%2520biochemical%2520characterization%2520of%2520endoribonuclease%2520Nsp15%2520encoded%2520by%2520Middle%2520East%2520Respiratory%2520Syndrome%2520Coronavirus%26jtitle%3DJ.%2520Virol.%26date%3D2018%26volume%3D92%26spage%3De00893%26epage%3De00918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sevajol, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subissi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Decroly, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbert, I.</span></span> <span> </span><span class="NLM_article-title">Insights into RNA synthesis, capping, and proofreading mechanisms of SARS-coronavirus</span>. <i>Virus Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">90</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.1016/j.virusres.2014.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.virusres.2014.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=25451065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ltrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2014&pages=90-99&author=M.+Sevajolauthor=L.+Subissiauthor=E.+Decrolyauthor=B.+Canardauthor=I.+Imbert&title=Insights+into+RNA+synthesis%2C+capping%2C+and+proofreading+mechanisms+of+SARS-coronavirus&doi=10.1016%2Fj.virusres.2014.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into RNA synthesis, capping, and proofreading mechanisms of SARS-coronavirus</span></div><div class="casAuthors">Sevajol, Marion; Subissi, Lorenzo; Decroly, Etienne; Canard, Bruno; Imbert, Isabelle</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">90-99</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The successive emergence of highly pathogenic coronaviruses (CoVs) such as the Severe Acute Respiratory Syndrome (SARS-CoV) in 2003 and the Middle East Respiratory Syndrome Coronavirus (MERS-CoV) in 2012 has stimulated a no. of studies on the mol. biol.  This research has provided significant new insight into functions and activities of the replication/transcription multi-protein complex.  The latter directs both continuous and discontinuous RNA synthesis to replicate and transcribe the large coronavirus genome made of a single-stranded, pos.-sense RNA of ∼30 kb.  In this review, we summarize our current understanding of SARS-CoV enzymes involved in RNA biochem., such as the in vitro characterization of a highly active and processive RNA polymerase complex which can assoc. with methyltransferase and 3'-5' exoribonuclease activities involved in RNA capping, and RNA proofreading, resp.  The recent discoveries reveal fascinating RNA-synthesizing machinery, highlighting the unique position of coronaviruses in the RNA virus world.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkfBObzAjd3LVg90H21EOLACvtfcHk0li387TMSdmTqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ltrzK&md5=1774c59cdf5e3b207839f973feb2197c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.virusres.2014.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virusres.2014.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DSevajol%26aufirst%3DM.%26aulast%3DSubissi%26aufirst%3DL.%26aulast%3DDecroly%26aufirst%3DE.%26aulast%3DCanard%26aufirst%3DB.%26aulast%3DImbert%26aufirst%3DI.%26atitle%3DInsights%2520into%2520RNA%2520synthesis%252C%2520capping%252C%2520and%2520proofreading%2520mechanisms%2520of%2520SARS-coronavirus%26jtitle%3DVirus%2520Res.%26date%3D2014%26volume%3D194%26spage%3D90%26epage%3D99%26doi%3D10.1016%2Fj.virusres.2014.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">V’Kovski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratzel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span> <span> </span><span class="NLM_article-title">Coronavirus biology and replication: implications for SARS-CoV-2</span>. <i>Nat. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/s41579-020-00468-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41579-020-00468-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33116300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXit1GjsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2021&pages=155-170&author=P.+V%E2%80%99Kovskiauthor=A.+Kratzelauthor=S.+Steinerauthor=H.+Stalderauthor=V.+Thiel&title=Coronavirus+biology+and+replication%3A+implications+for+SARS-CoV-2&doi=10.1038%2Fs41579-020-00468-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus biology and replication: implications for SARS-CoV-2</span></div><div class="casAuthors">V'kovski, Philip; Kratzel, Annika; Steiner, Silvio; Stalder, Hanspeter; Thiel, Volker</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Microbiology</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">155-170</span>CODEN:
                <span class="NLM_cas:coden">NRMACK</span>;
        ISSN:<span class="NLM_cas:issn">1740-1526</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Abstr.: The SARS-CoV-2 pandemic and its unprecedented global societal and economic disruptive impact has marked the third zoonotic introduction of a highly pathogenic coronavirus into the human population.  Although the previous coronavirus SARS-CoV and MERS-CoV epidemics raised awareness of the need for clin. available therapeutic or preventive interventions, to date, no treatments with proven efficacy are available.  The development of effective intervention strategies relies on the knowledge of mol. and cellular mechanisms of coronavirus infections, which highlights the significance of studying virus-host interactions at the mol. level to identify targets for antiviral intervention and to elucidate crit. viral and host determinants that are decisive for the development of severe disease.  In this Review, we summarize the first discoveries that shape our current understanding of SARS-CoV-2 infection throughout the intracellular viral life cycle and relate that to our knowledge of coronavirus biol.  The elucidation of similarities and differences between SARS-CoV-2 and other coronaviruses will support future preparedness and strategies to combat coronavirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNkjU36gq1BLVg90H21EOLACvtfcHk0liOLf2b9gkKTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXit1GjsLjF&md5=14ddf54f3e569fe39eb4f3295cf603e6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41579-020-00468-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41579-020-00468-6%26sid%3Dliteratum%253Aachs%26aulast%3DV%25E2%2580%2599Kovski%26aufirst%3DP.%26aulast%3DKratzel%26aufirst%3DA.%26aulast%3DSteiner%26aufirst%3DS.%26aulast%3DStalder%26aufirst%3DH.%26aulast%3DThiel%26aufirst%3DV.%26atitle%3DCoronavirus%2520biology%2520and%2520replication%253A%2520implications%2520for%2520SARS-CoV-2%26jtitle%3DNat.%2520Rev.%2520Microbiol.%26date%3D2021%26volume%3D19%26spage%3D155%26epage%3D170%26doi%3D10.1038%2Fs41579-020-00468-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackbart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettelman, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mielech, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">Coronavirus nonstructural protein 15 mediates evasion of dsRNA sensors and limits apoptosis in macrophages</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E4251</span>– <span class="NLM_lpage">E4260</span>, <span class="refDoi"> DOI: 10.1073/pnas.1618310114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1073%2Fpnas.1618310114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28484023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnt1Clu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E4251-E4260&author=X.+Dengauthor=M.+Hackbartauthor=R.+C.+Mettelmanauthor=A.+O%E2%80%99Brienauthor=A.+M.+Mielechauthor=G.+Yiauthor=C.+C.+Kaoauthor=S.+C.+Baker&title=Coronavirus+nonstructural+protein+15+mediates+evasion+of+dsRNA+sensors+and+limits+apoptosis+in+macrophages&doi=10.1073%2Fpnas.1618310114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus nonstructural protein 15 mediates evasion of dsRNA sensors and limits apoptosis in macrophages</span></div><div class="casAuthors">Deng, Xufang; Hackbart, Matthew; Mettelman, Robert C.; O'Brien, Amornrat; Mielech, Anna M.; Yi, Guanghui; Kao, C. Cheng; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">E4251-E4260</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Coronaviruses are pos.-sense RNA viruses that generate double-stranded RNA (dsRNA) intermediates during replication, yet evade detection by host innate immune sensors.  Here we report that coronavirus nonstructural protein 15 (nsp15), an endoribonuclease, is required for evasion of dsRNA sensors.  We evaluated two independent nsp15 mutant mouse coronaviruses, designated N15m1 and N15m3, and found that these viruses replicated poorly and induced rapid cell death in mouse bone marrow-derived macrophages.  Infection of macrophages with N15m1, which expresses an unstable nsp15, or N15m3, which expresses a catalysis-deficient nsp15, activated MDA5, PKR, and the OAS/RNase L system, resulting in an early, robust induction of type I IFN, PKR-mediated apoptosis, and RNA degrdn.  Immunofluorescence imaging of nsp15 mutant virus-infected macrophages revealed significant dispersal of dsRNA early during infection, whereas in WT virus-infected cells, the majority of the dsRNA was assocd. with replication complexes.  The loss of nsp15 activity also resulted in greatly attenuated disease in mice and stimulated a protective immune response.  Taken together, our findings demonstrate that coronavirus nsp15 is crit. for evasion of host dsRNA sensors in macrophages and reveal that modulating nsp15 stability and activity is a strategy for generating live-attenuated vaccines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXhXNthEjvxLVg90H21EOLACvtfcHk0liOLf2b9gkKTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnt1Clu7s%253D&md5=56fae15b7e25d83df1f04c257f9c6ab2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1618310114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1618310114%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DHackbart%26aufirst%3DM.%26aulast%3DMettelman%26aufirst%3DR.%2BC.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DA.%26aulast%3DMielech%26aufirst%3DA.%2BM.%26aulast%3DYi%26aufirst%3DG.%26aulast%3DKao%26aufirst%3DC.%2BC.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DCoronavirus%2520nonstructural%2520protein%252015%2520mediates%2520evasion%2520of%2520dsRNA%2520sensors%2520and%2520limits%2520apoptosis%2520in%2520macrophages%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE4251%26epage%3DE4260%26doi%3D10.1073%2Fpnas.1618310114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabouw, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Züst, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">V’Kovski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habjan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervantes-Barragan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karl, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaughan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Kuppeveld, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludewig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergmann, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalinke, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span> <span> </span><span class="NLM_article-title">Early endonuclease-mediated evasion of RNA sensing ensures efficient coronavirus replication</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1006195</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1006195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.ppat.1006195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28158275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=E.+Kindlerauthor=C.+Gil-Cruzauthor=J.+Spanierauthor=Y.+Liauthor=J.+Wilhelmauthor=H.+H.+Rabouwauthor=R.+Z%C3%BCstauthor=M.+Hwangauthor=P.+V%E2%80%99Kovskiauthor=H.+Stalderauthor=S.+Martiauthor=M.+Habjanauthor=L.+Cervantes-Barraganauthor=R.+Elliotauthor=N.+Karlauthor=C.+Gaughanauthor=F.+J.+van+Kuppeveldauthor=R.+H.+Silvermanauthor=M.+Kellerauthor=B.+Ludewigauthor=C.+C.+Bergmannauthor=J.+Ziebuhrauthor=S.+R.+Weissauthor=U.+Kalinkeauthor=V.+Thiel&title=Early+endonuclease-mediated+evasion+of+RNA+sensing+ensures+efficient+coronavirus+replication&doi=10.1371%2Fjournal.ppat.1006195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Early endonuclease-mediated evasion of RNA sensing ensures efficient coronavirus replication</span></div><div class="casAuthors">Kindler, Eveline; Gil-Cruz, Cristina; Spanier, Julia; Li, Yize; Wilhelm, Jochen; Rabouw, Huib H.; Zust, Roland; Hwang, Mihyun; V'kovski, Philip; Stalder, Hanspeter; Marti, Sabrina; Habjan, Matthias; Cervantes-Barragan, Luisa; Elliot, Ruth; Karl, Nadja; Gaughan, Christina; van Kuppeveld, Frank J. M.; Silverman, Robert H.; Keller, Markus; Ludewig, Burkhard; Bergmann, Cornelia C.; Ziebuhr, John; Weiss, Susan R.; Kalinke, Ulrich; Thiel, Volker</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1006195/1-e1006195/26</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses are of veterinary and medical importance and include highly pathogenic zoonotic viruses, such as SARS-CoV and MERS-CoV.  They are known to efficiently evade early innate immune responses, manifesting in almost negligible expression of type-I interferons (IFN-I).  This evasion strategy suggests an evolutionary conserved viral function that has evolved to prevent RNA-based sensing of infection in vertebrate hosts.  Here we show that the coronavirus endonuclease (EndoU) activity is key to prevent early induction of double-stranded RNA (dsRNA) host cell responses.  Replication of EndoU-deficient coronaviruses is greatly attenuated in vivo and severely restricted in primary cells even during the early phase of the infection.  In macrophages we found immediate induction of IFN-I expression and RNase L-mediated breakdown of rRNA.  Accordingly, EndoU-deficient viruses can retain replication only in cells that are deficient in IFN-I expression or sensing, and in cells lacking both RNase L and PKR.  Collectively our results demonstrate that the coronavirus EndoU efficiently prevents simultaneous activation of host cell dsRNA sensors, such as Mda5, OAS and PKR.  The localization of the EndoU activity at the site of viral RNA synthesis-within the replicase complex-suggests that coronaviruses have evolved a viral RNA decay pathway to evade early innate and intrinsic antiviral host cell responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJihv7-Ahq6rVg90H21EOLACvtfcHk0liOLf2b9gkKTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCqtrvP&md5=12cd5c885b2b147fe10fe55fc5ccd19c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1006195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1006195%26sid%3Dliteratum%253Aachs%26aulast%3DKindler%26aufirst%3DE.%26aulast%3DGil-Cruz%26aufirst%3DC.%26aulast%3DSpanier%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWilhelm%26aufirst%3DJ.%26aulast%3DRabouw%26aufirst%3DH.%2BH.%26aulast%3DZ%25C3%25BCst%26aufirst%3DR.%26aulast%3DHwang%26aufirst%3DM.%26aulast%3DV%25E2%2580%2599Kovski%26aufirst%3DP.%26aulast%3DStalder%26aufirst%3DH.%26aulast%3DMarti%26aufirst%3DS.%26aulast%3DHabjan%26aufirst%3DM.%26aulast%3DCervantes-Barragan%26aufirst%3DL.%26aulast%3DElliot%26aufirst%3DR.%26aulast%3DKarl%26aufirst%3DN.%26aulast%3DGaughan%26aufirst%3DC.%26aulast%3Dvan%2BKuppeveld%26aufirst%3DF.%2BJ.%26aulast%3DSilverman%26aufirst%3DR.%2BH.%26aulast%3DKeller%26aufirst%3DM.%26aulast%3DLudewig%26aufirst%3DB.%26aulast%3DBergmann%26aufirst%3DC.%2BC.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DWeiss%26aufirst%3DS.%2BR.%26aulast%3DKalinke%26aufirst%3DU.%26aulast%3DThiel%26aufirst%3DV.%26atitle%3DEarly%2520endonuclease-mediated%2520evasion%2520of%2520RNA%2520sensing%2520ensures%2520efficient%2520coronavirus%2520replication%26jtitle%3DPLoS%2520Pathog.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.ppat.1006195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hackbart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">Coronavirus endoribonuclease targets viral polyuridine sequences to evade activating host sensors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">8094</span>– <span class="NLM_lpage">8103</span>, <span class="refDoi"> DOI: 10.1073/pnas.1921485117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1073%2Fpnas.1921485117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32198201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXms1ygt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=8094-8103&author=M.+Hackbartauthor=X.+Dengauthor=S.+C.+Baker&title=Coronavirus+endoribonuclease+targets+viral+polyuridine+sequences+to+evade+activating+host+sensors&doi=10.1073%2Fpnas.1921485117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus endoribonuclease targets viral polyuridine sequences to evade activating host sensors</span></div><div class="casAuthors">Hackbart, Matthew; Deng, Xufang; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">8094-8103</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Coronaviruses (CoVs) are pos.-sense RNA viruses that can emerge from endemic reservoirs and infect zoonotically, causing significant morbidity and mortality.  CoVs encode an endoribonuclease designated EndoU that facilitates evasion of host pattern recognition receptor MDA5, but the target of EndoU activity was not known.  Here, we report that EndoU cleaves the 5'-polyuridines from neg.-sense viral RNA, termed PUN RNA, which is the product of polyA-templated RNA synthesis.  Using a virus contg. an EndoU catalytic-inactive mutation, we detected a higher abundance of PUN RNA in the cytoplasm compared to wild-type-infected cells.  Furthermore, we found that transfecting PUN RNA into cells stimulates a robust, MDA5-dependent interferon response, and that removal of the polyuridine extension on the RNA dampens the response.  Overall, the results of this study reveal the PUN RNA to be a CoV MDA5-dependent pathogen-assocd. mol. pattern (PAMP).  We also establish a mechanism for EndoU activity to cleave and limit the accumulation of this PAMP.  Since EndoU activity is highly conserved in all CoVs, inhibiting this activity may serve as an approach for therapeutic interventions against existing and emerging CoV infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCgKILpCAxeLVg90H21EOLACvtfcHk0ljrb9xZAMW0-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXms1ygt74%253D&md5=0f94b3fbcae5750168155855eed494a7</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1921485117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1921485117%26sid%3Dliteratum%253Aachs%26aulast%3DHackbart%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DCoronavirus%2520endoribonuclease%2520targets%2520viral%2520polyuridine%2520sequences%2520to%2520evade%2520activating%2520host%2520sensors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D8094%26epage%3D8103%26doi%3D10.1073%2Fpnas.1921485117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ancar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ransom, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesselberth, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, D. J.</span></span> <span> </span><span class="NLM_article-title">Physiologic RNA targets and refined sequence specificity of coronavirus EndoU</span>. <i>RNA</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1976</span>– <span class="NLM_lpage">1999</span>, <span class="refDoi"> DOI: 10.1261/rna.076604.120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1261%2Frna.076604.120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32989044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3MXktFart70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=1976-1999&author=R.+Ancarauthor=Y.+Liauthor=E.+Kindlerauthor=D.+A.+Cooperauthor=M.+Ransomauthor=V.+Thielauthor=S.+R.+Weissauthor=J.+R.+Hesselberthauthor=D.+J.+Barton&title=Physiologic+RNA+targets+and+refined+sequence+specificity+of+coronavirus+EndoU&doi=10.1261%2Frna.076604.120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Physiologic RNA targets and refined sequence specificity of coronavirus EndoU</span></div><div class="casAuthors">Ancar, Rachel; Li, Yize; Kindler, Eveline; Cooper, Daphne A.; Ransom, Monica; Thiel, Volker; Weiss, Susan R.; Hesselberth, Jay R.; Barton, David J.</div><div class="citationInfo"><span class="NLM_cas:title">RNA</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1976-1999</span>CODEN:
                <span class="NLM_cas:coden">RNARFU</span>;
        ISSN:<span class="NLM_cas:issn">1355-8382</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Coronavirus EndoU inhibits dsRNA-activated antiviral responses; however, the physiol. RNA substrates of EndoU are unknown.  In this study, we used mouse hepatitis virus (MHV)-infected bone marrow-derived macrophage (BMM) and cyclic phosphate cDNA sequencing to identify the RNA targets of EndoU.  EndoU targeted viral RNA, cleaving the 3' side of pyrimidines with a strong preference for U↓A and C↓A sequences (endoY↓A).  EndoU-dependent cleavage was detected in every region of MHV RNA, from the 5' NTR to the 3' NTR, including transcriptional regulatory sequences (TRS).  Cleavage at two CA dinucleotides immediately adjacent to the MHV poly(A) tail suggests a mechanism to suppress neg.-strand RNA synthesis and the accumulation of viral dsRNA.  MHV with EndoU (EndoUmut) or 2'-5' phosphodiesterase (PDEmut) mutations provoked the activation of RNase L in BMM, with corresponding cleavage of RNAs by RNase L.  The physiol. targets of EndoU are viral RNA templates required for neg.-strand RNA synthesis and dsRNA accumulation.  Coronavirus EndoU cleaves U↓A and C↓A sequences (endoY↓A) within viral (+) strand RNA to evade dsRNA-activated host responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNE7gc4bUh4LVg90H21EOLACvtfcHk0ljrb9xZAMW0-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXktFart70%253D&md5=3d8918d5c8544ad4967efd183a1a3f0b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1261%2Frna.076604.120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.076604.120%26sid%3Dliteratum%253Aachs%26aulast%3DAncar%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKindler%26aufirst%3DE.%26aulast%3DCooper%26aufirst%3DD.%2BA.%26aulast%3DRansom%26aufirst%3DM.%26aulast%3DThiel%26aufirst%3DV.%26aulast%3DWeiss%26aufirst%3DS.%2BR.%26aulast%3DHesselberth%26aufirst%3DJ.%2BR.%26aulast%3DBarton%26aufirst%3DD.%2BJ.%26atitle%3DPhysiologic%2520RNA%2520targets%2520and%2520refined%2520sequence%2520specificity%2520of%2520coronavirus%2520EndoU%26jtitle%3DRNA%26date%3D2020%26volume%3D26%26spage%3D1976%26epage%3D1999%26doi%3D10.1261%2Frna.076604.120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. C.</span></span> <span> </span><span class="NLM_article-title">An ″Old″ protein with a new story: Coronavirus endoribonuclease is important for evading host antiviral defenses</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>517</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">163</span>, <span class="refDoi"> DOI: 10.1016/j.virol.2017.12.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.virol.2017.12.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=29307596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjvFGqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2018&pages=157-163&author=X.+Dengauthor=S.+C.+Baker&title=An+%E2%80%B3Old%E2%80%B3+protein+with+a+new+story%3A+Coronavirus+endoribonuclease+is+important+for+evading+host+antiviral+defenses&doi=10.1016%2Fj.virol.2017.12.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">An "Old" protein with a new story: Coronavirus endoribonuclease is important for evading host antiviral defenses</span></div><div class="casAuthors">Deng, Xufang; Baker, Susan C.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">157-163</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Here we review the evolving story of the coronavirus endoribonuclease (EndoU).  Coronavirus EndoU is encoded within the sequence of nonstructural protein (nsp) 15, which was initially identified as a component of the viral replication complex.  Biochem. and structural studies revealed the enzymic nature of nsp15/EndoU, which was postulated to be essential for the unique replication cycle of viruses in the order Nidovirales.  However, the role of nsp15 in coronavirus replication was enigmatic as EndoU-deficient coronaviruses were viable and replicated to near wild-type virus levels in fibroblast cells.  A breakthrough in our understanding of the role of EndoU was revealed in recent studies, which showed that EndoU mediates the evasion of viral double-stranded RNA recognition by host sensors in macrophages.  This new discovery of nsp15/EndoU function leads to new opportunities for investigating how a viral EndoU contributes to pathogenesis and exploiting this enzyme for therapeutics and vaccine design against pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkT3Mo_-SEMrVg90H21EOLACvtfcHk0ljrb9xZAMW0-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjvFGqtA%253D%253D&md5=ae511601ecd3b7f592ef90f9e28799d7</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2017.12.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2017.12.024%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DBaker%26aufirst%3DS.%2BC.%26atitle%3DAn%2520%25E2%2580%25B3Old%25E2%2580%25B3%2520protein%2520with%2520a%2520new%2520story%253A%2520Coronavirus%2520endoribonuclease%2520is%2520important%2520for%2520evading%2520host%2520antiviral%2520defenses%26jtitle%3DVirology%26date%3D2018%26volume%3D517%26spage%3D157%26epage%3D163%26doi%3D10.1016%2Fj.virol.2017.12.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mak, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kok, K. H.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 nsp13, nsp14, nsp15 and orf6 function as potent interferon antagonists</span>. <i>Emerging Microbes Infect.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1418</span>– <span class="NLM_lpage">1428</span>, <span class="refDoi"> DOI: 10.1080/22221751.2020.1780953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1080%2F22221751.2020.1780953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32529952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Sls7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=1418-1428&author=C.+K.+Yuenauthor=J.+Y.+Lamauthor=W.+M.+Wongauthor=L.+F.+Makauthor=X.+Wangauthor=H.+Chuauthor=J.+P.+Caiauthor=D.+Y.+Jinauthor=K.+K.+Toauthor=J.+F.+Chanauthor=K.+Y.+Yuenauthor=K.+H.+Kok&title=SARS-CoV-2+nsp13%2C+nsp14%2C+nsp15+and+orf6+function+as+potent+interferon+antagonists&doi=10.1080%2F22221751.2020.1780953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 nsp13, nsp14, nsp15 and orf6 function as potent interferon antagonists</span></div><div class="casAuthors">Yuen, Chun-Kit; Lam, Joy-Yan; Wong, Wan-Man; Mak, Long-Fung; Wang, Xiaohui; Chu, Hin; Cai, Jian-Piao; Jin, Dong-Yan; To, Kelvin Kai-Wang; Chan, Jasper Fuk-Woo; Yuen, Kwok-Yung; Kok, Kin-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Microbes & Infections</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1418-1428</span>CODEN:
                <span class="NLM_cas:coden">EMIMC4</span>;
        ISSN:<span class="NLM_cas:issn">2222-1751</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The Coronavirus disease 2019 (COVID-19), which is caused by the novel SARS-CoV-2 virus, is now causing a tremendous global health concern.  Since its first appearance in Dec. 2019, the outbreak has already caused over 5.8 million infections worldwide (till 29 May 2020), with more than 0.35 million deaths.  Early virus-mediated immune suppression is believed to be one of the unique characteristics of SARS-CoV-2 infection and contributes at least partially to the viral pathogenesis.  In this study, we identified the key viral interferon antagonists of SARS-CoV-2 and compared them with two well-characterized SARS-CoV interferon antagonists, PLpro and orf6.  Here we demonstrated that the SARS-CoV-2 nsp13, nsp14, nsp15 and orf6, but not the unique orf8, could potently suppress primary interferon prodn. and interferon signaling.  Although SARS-CoV PLpro has been well-characterized for its potent interferon-antagonizing, deubiquitinase and protease activities, SARS-CoV-2 PLpro, despite sharing high amino acid sequence similarity with SARS-CoV, loses both interferon-antagonising and deubiquitinase activities.  Among the 27 viral proteins, SARS-CoV-2 orf6 demonstrated the strongest suppression on both primary interferon prodn. and interferon signaling.  Orf6-deleted SARS-CoV-2 may be considered for the development of intranasal live-but-attenuated vaccine against COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsdfWiicWnMrVg90H21EOLACvtfcHk0ljY70NBG7dalA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Sls7bE&md5=51cecf3ed6cae2cca0316fc62b98ae6b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1080%2F22221751.2020.1780953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F22221751.2020.1780953%26sid%3Dliteratum%253Aachs%26aulast%3DYuen%26aufirst%3DC.%2BK.%26aulast%3DLam%26aufirst%3DJ.%2BY.%26aulast%3DWong%26aufirst%3DW.%2BM.%26aulast%3DMak%26aufirst%3DL.%2BF.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChu%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%2BP.%26aulast%3DJin%26aufirst%3DD.%2BY.%26aulast%3DTo%26aufirst%3DK.%2BK.%26aulast%3DChan%26aufirst%3DJ.%2BF.%26aulast%3DYuen%26aufirst%3DK.%2BY.%26aulast%3DKok%26aufirst%3DK.%2BH.%26atitle%3DSARS-CoV-2%2520nsp13%252C%2520nsp14%252C%2520nsp15%2520and%2520orf6%2520function%2520as%2520potent%2520interferon%2520antagonists%26jtitle%3DEmerging%2520Microbes%2520Infect.%26date%3D2020%26volume%3D9%26spage%3D1418%26epage%3D1428%26doi%3D10.1080%2F22221751.2020.1780953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mansbach, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumgartner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ba, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieh, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span> <span> </span><span class="NLM_article-title">Modulation of polypeptide conformation through donor-acceptor transformation of side-chain hydrogen bonding ligands</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">92</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-00079-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41467-017-00079-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28733648" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A280%3ADC%252BC1cfgtFeksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=92&author=Z.+Songauthor=R.+A.+Mansbachauthor=H.+Heauthor=K.+C.+Shihauthor=R.+Baumgartnerauthor=N.+Zhengauthor=X.+Baauthor=Y.+Huangauthor=D.+Maniauthor=Y.+Liuauthor=Y.+Linauthor=M.+P.+Niehauthor=A.+L.+Fergusonauthor=L.+Yinauthor=J.+Cheng&title=Modulation+of+polypeptide+conformation+through+donor-acceptor+transformation+of+side-chain+hydrogen+bonding+ligands&doi=10.1038%2Fs41467-017-00079-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of polypeptide conformation through donor-acceptor transformation of side-chain hydrogen bonding ligands</span></div><div class="casAuthors">Song Ziyuan; Zheng Nan; Mani Deepak; Ferguson Andrew L; Cheng Jianjun; Mansbach Rachael A; He Hua; Yin Lichen; Shih Kuo-Chih; Lin Yao; Nieh Mu-Ping; Shih Kuo-Chih; Baumgartner Ryan; Ba Xiaochu; Huang Yinzhao; Liu Yun; Liu Yun; Lin Yao; Nieh Mu-Ping; Ferguson Andrew L</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Synthetic polypeptides have received increasing attention due to their ability to form higher ordered structures similar to proteins.  The control over their secondary structures, which enables dynamic conformational changes, is primarily accomplished by tuning the side-chain hydrophobic or ionic interactions.  Herein we report a strategy to modulate the conformation of polypeptides utilizing donor-acceptor interactions emanating from side-chain H-bonding ligands.  Specifically, 1,2,3-triazole groups, when incorporated onto polypeptide side-chains, serve as both H-bond donors and acceptors at neutral pH and disrupt the α-helical conformation.  When protonated, the resulting 1,2,3-triazolium ions lose the ability to act as H-bond acceptors, and the polypeptides regain their α-helical structure.  The conformational change of triazole polypeptides in response to the donor-acceptor pattern was conclusively demonstrated using both experimental-based and simulation-based methods.  We further showed the utility of this transition by designing smart, cell-penetrating polymers that undergo acid-activated endosomal escape in living cells.Hydrogen bonding plays a major role in determining the tridimensional structure of biopolymers.  Here, the authors show that control over a polypeptide conformation can be achieved by altering the donor-acceptor properties of side-chain triazole units via protonation-deprotonation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSpbYP86F-n6GxpATR2EQURfW6udTcc2eYog638eBTt5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfgtFeksw%253D%253D&md5=8e48dd81d76eb172da4b14b2d4372b34</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00079-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00079-5%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DMansbach%26aufirst%3DR.%2BA.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DShih%26aufirst%3DK.%2BC.%26aulast%3DBaumgartner%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DN.%26aulast%3DBa%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DMani%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DNieh%26aufirst%3DM.%2BP.%26aulast%3DFerguson%26aufirst%3DA.%2BL.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DJ.%26atitle%3DModulation%2520of%2520polypeptide%2520conformation%2520through%2520donor-acceptor%2520transformation%2520of%2520side-chain%2520hydrogen%2520bonding%2520ligands%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D92%26doi%3D10.1038%2Fs41467-017-00079-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonandi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christodoulou, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fumagalli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perdicchia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passarella, D.</span></span> <span> </span><span class="NLM_article-title">The 1,2,3-triazole ring as a bioisostere in medicinal chemistry</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1572</span>– <span class="NLM_lpage">1581</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1016%2Fj.drudis.2017.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28676407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFCmurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=1572-1581&author=E.+Bonandiauthor=M.+S.+Christodoulouauthor=G.+Fumagalliauthor=D.+Perdicchiaauthor=G.+Rastelliauthor=D.+Passarella&title=The+1%2C2%2C3-triazole+ring+as+a+bioisostere+in+medicinal+chemistry&doi=10.1016%2Fj.drudis.2017.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The 1,2,3-triazole ring as a bioisostere in medicinal chemistry</span></div><div class="casAuthors">Bonandi, Elisa; Christodoulou, Michael S.; Fumagalli, Gaia; Perdicchia, Dario; Rastelli, Giulio; Passarella, Daniele</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1572-1581</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">1,2,3-Triazole is a well-known scaffold that has a widespread occurrence in different compds. characterized by several bioactivities, such as antimicrobial, antiviral, and antitumor effects.  Moreover, the structural features of 1,2,3-triazole enable it to mimic different functional groups, justifying its wide use as a bioisostere for the synthesis of new active mols.  Here, we provide an overview of the 1,2,3-triazole ring as a bioisostere for the design of drug analogs, highlighting relevant recent examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyNRynDv4W27Vg90H21EOLACvtfcHk0ljY70NBG7dalA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFCmurrK&md5=0ea3588d22d266e5eed9f401c887b838</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DBonandi%26aufirst%3DE.%26aulast%3DChristodoulou%26aufirst%3DM.%2BS.%26aulast%3DFumagalli%26aufirst%3DG.%26aulast%3DPerdicchia%26aufirst%3DD.%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DPassarella%26aufirst%3DD.%26atitle%3DThe%25201%252C2%252C3-triazole%2520ring%2520as%2520a%2520bioisostere%2520in%2520medicinal%2520chemistry%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D1572%26epage%3D1581%26doi%3D10.1016%2Fj.drudis.2017.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kummetha, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lichinchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, T. M.</span></span> <span> </span><span class="NLM_article-title">1,2,3-Triazoles as amide bioisosteres: discovery of a new class of potent HIV-1 Vif antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7677</span>– <span class="NLM_lpage">7682</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlSltb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7677-7682&author=I.+Mohammedauthor=I.+R.+Kummethaauthor=G.+Singhauthor=N.+Sharovaauthor=G.+Lichinchiauthor=J.+Dangauthor=M.+Stevensonauthor=T.+M.+Rana&title=1%2C2%2C3-Triazoles+as+amide+bioisosteres%3A+discovery+of+a+new+class+of+potent+HIV-1+Vif+antagonists&doi=10.1021%2Facs.jmedchem.6b00247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">1,2,3-Triazoles as Amide Bioisosteres: Discovery of a New Class of Potent HIV-1 Vif Antagonists</span></div><div class="casAuthors">Mohammed, Idrees; Kummetha, Indrasena Reddy; Singh, Gatikrushna; Sharova, Natalia; Lichinchi, Gianluigi; Dang, Jason; Stevenson, Mario; Rana, Tariq M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7677-7682</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RN-18 based viral infectivity factor (Vif), Vif antagonists reduce viral infectivity by rescuing APOBEC3G (A3G) expression and enhancing A3G-dependent Vif degrdn.  Replacement of amide functionality in RN-18 (IC50 = 6 μM) by isosteric heterocycles resulted in the discovery of a 1,2,3-trizole, 1d (IC50 = 1.2 μM).  We identified several potent HIV-1 inhibitors from a 1d based library including 5ax (IC50 = 0.01 μM), 5bx (0.2 μM), 2ey (0.4 μM), 5ey (0.6 μM), and 6bx (0.2 μM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuCHEWexGqlLVg90H21EOLACvtfcHk0ljDoNC2pgfF5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlSltb7F&md5=f75735900056b2f0975a13a8b6d7355a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00247%26sid%3Dliteratum%253Aachs%26aulast%3DMohammed%26aufirst%3DI.%26aulast%3DKummetha%26aufirst%3DI.%2BR.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DSharova%26aufirst%3DN.%26aulast%3DLichinchi%26aufirst%3DG.%26aulast%3DDang%26aufirst%3DJ.%26aulast%3DStevenson%26aufirst%3DM.%26aulast%3DRana%26aufirst%3DT.%2BM.%26atitle%3D1%252C2%252C3-Triazoles%2520as%2520amide%2520bioisosteres%253A%2520discovery%2520of%2520a%2520new%2520class%2520of%2520potent%2520HIV-1%2520Vif%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7677%26epage%3D7682%26doi%3D10.1021%2Facs.jmedchem.6b00247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maqbool, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span> <span> </span><span class="NLM_article-title">CuAAC-ensembled 1,2,3-triazole-linked isosteres as pharmacophores in drug discovery: review</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">5610</span>– <span class="NLM_lpage">5635</span>, <span class="refDoi"> DOI: 10.1039/C9RA09510A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC9RA09510A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivVelsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2020&pages=5610-5635&author=A.+Raniauthor=G.+Singhauthor=A.+Singhauthor=U.+Maqboolauthor=G.+Kaurauthor=J.+Singh&title=CuAAC-ensembled+1%2C2%2C3-triazole-linked+isosteres+as+pharmacophores+in+drug+discovery%3A+review&doi=10.1039%2FC9RA09510A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">CuAAC-ensembled 1,2,3-triazole-linked isosteres as pharmacophores in drug discovery: review</span></div><div class="casAuthors">Rani, Alisha; Singh, Gurjaspreet; Singh, Akshpreet; Maqbool, Ubair; Kaur, Gurpreet; Singh, Jandeep</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5610-5635</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The review lays emphasis on the significance of 1,2,3-triazoles synthesized via CuAAC reaction having potential to act as anti-microbial, anti-cancer, anti-viral, anti-inflammatory, anti-tuberculosis, anti-diabetic, and anti-Alzheimer drugs.  The importance of click chem. is due to its 'quicker' methodol. that has the capability to create complex and efficient drugs with high yield and purity from simple and cheap starting materials.  The activity of different triazolyl compds. was compiled considering MIC, IC50, and EC50 values against different species of microbes.  In addn. to this, the anti-oxidant property of triazolyl compds. have also been reviewed and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1q8vAJ1PzLbVg90H21EOLACvtfcHk0ljDoNC2pgfF5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivVelsLY%253D&md5=f2bfa24f21320d47cc1f7632dbe1ab48</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1039%2FC9RA09510A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9RA09510A%26sid%3Dliteratum%253Aachs%26aulast%3DRani%26aufirst%3DA.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DMaqbool%26aufirst%3DU.%26aulast%3DKaur%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DCuAAC-ensembled%25201%252C2%252C3-triazole-linked%2520isosteres%2520as%2520pharmacophores%2520in%2520drug%2520discovery%253A%2520review%26jtitle%3DRSC%2520Adv.%26date%3D2020%26volume%3D10%26spage%3D5610%26epage%3D5635%26doi%3D10.1039%2FC9RA09510A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csuk, R.</span></span> <span> </span><span class="NLM_article-title">Betulinic acid and its derivatives: a patent review (2008-2013)</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">913</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1517/13543776.2014.927441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1517%2F13543776.2014.927441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=24909232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOhu7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=913-923&author=R.+Csuk&title=Betulinic+acid+and+its+derivatives%3A+a+patent+review+%282008-2013%29&doi=10.1517%2F13543776.2014.927441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Betulinic acid and its derivatives: a patent review (2008 - 2013)</span></div><div class="casAuthors">Csuk, Rene</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">913-923</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Betulinic acid (BA) is a triterpenoid that can be obtained from renewable resources.  BA is cytotoxic to many human tumor cell lines mainly by apoptosis but cell death might also be triggered by nonapoptotic pathways.  Many derivs. have been synthesized to improve the very weak soly. of parent BA and to increase its cytotoxicity as well as its selectivity toward tumor cells.  Areas covered: A brief introduction into cancer is given reflecting the different pathways this disease might be treated using chemotherapy using natural product analogs esp. triterpenes.  The different ways of action of BA in cancer cells are discussed.  Finally, this review describes the main synthetic modifications that have been performed and discusses, in short, the structure-activity relationships of these analogs, investigated between 2008 and 2013 including some important publications from early 2014.  Expert opinion: A no. of patents on BA analogs for the chemotherapy of cancer have been reported between 2008 and 2013.  Most of these patents deal with modifications at positions C-3, C-20 and C-28.  There are only a few compds. meeting the needs of a sufficient hydrosoly., while retaining high cytotoxicity and selectivity toward tumor cells.  Thus, one might expect that there will be some efforts in developing mols. of improved soly. and to find new and more efficient forms of administration (liposomes, transdermal application and nanoemulsions).  An important sideline might be the treatment of the age-dependent degeneration of the macula, a possible caveat of which might be a certain degree of CNS toxicity assocd. with several derivs. of BA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIOWZQT30Pa7Vg90H21EOLACvtfcHk0ljDoNC2pgfF5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOhu7zL&md5=6cf01023cff1d13897d660df6260e1d2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1517%2F13543776.2014.927441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543776.2014.927441%26sid%3Dliteratum%253Aachs%26aulast%3DCsuk%26aufirst%3DR.%26atitle%3DBetulinic%2520acid%2520and%2520its%2520derivatives%253A%2520a%2520patent%2520review%2520%25282008-2013%2529%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2014%26volume%3D24%26spage%3D913%26epage%3D923%26doi%3D10.1517%2F13543776.2014.927441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sidova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoufaly, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pokorny, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popa, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, M.</span></span> <span> </span><span class="NLM_article-title">Cytotoxic conjugates of betulinic acid and substituted triazoles prepared by Huisgen Cycloaddition from 30-azidoderivatives</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0171621</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0171621</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.pone.0171621" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=28158265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=V.+Sidovaauthor=P.+Zoufalyauthor=J.+Pokornyauthor=P.+Dzubakauthor=M.+Hajduchauthor=I.+Popaauthor=M.+Urban&title=Cytotoxic+conjugates+of+betulinic+acid+and+substituted+triazoles+prepared+by+Huisgen+Cycloaddition+from+30-azidoderivatives&doi=10.1371%2Fjournal.pone.0171621"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Cytotoxic conjugates of betulinic acid and substituted triazoles prepared by Huisgen Cycloaddition from 30-azidoderivatives</span></div><div class="casAuthors">Sidova, Veronika; Zoufaly, Pavel; Pokorny, Jan; Dzubak, Petr; Hajduch, Marian; Popa, Igor; Urban, Milan</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e0171621/1-e0171621/25</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Synthesis of conjugates of betulinic acid with substituted triazoles I [R1 = H, MeC(O), Ph3Si; R2 = Cyclopentyl, Ph, 2-pyridinyl, etc.] via Huisgen 1,3-cycloaddn. was described.  All the synthesized compds. contained free 28-COOH group.  Allylic bromination of protected betulinic acid by NBS gave corresponding 30-bromoderivatives, their substitution with sodium azides produced 30-azidoderivatives and these azides were subjected to CuI catalyzed Huisgen 1,3-cycloaddn. to give the final conjugates I.  All new compds. were tested for their in vitro cytotoxic activities on eight cancer and two non-cancer cell lines.  The most active compds. were conjugates of 3beta;-O-acetylbetulinic acid and among them, conjugate with triazole substituted by benzaldehyde I [R1 = MeC(O); R2 = 2-H(O)CC6H4] was the best with IC50 of 3.3 μM and therapeutic index of 9.1.  Five compds. in this study had IC50 below 10 μM and inhibited DNA and RNA synthesis and caused block in G0/G1 cell cycle phase which was highly similar to actinomycin D.  It was unusual that here prepd. 3β-O-acetates were more active than compds. with the free 3-OH group and this suggested that this set might have common mechanism of action that was different than mechanism of action of previously known 3β-O-acetoxybetulinic acid derivs.  Benzaldehyde type conjugate I [R1 = MeC(O); R2 = 2-H(O)CC6H4] was the best candidate for further drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkIIgdWn-BbLVg90H21EOLACvtfcHk0lhAwIyv1oPK0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyqtrk%253D&md5=8213f1966cdcef6cb78b1fd6ab04a5fc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0171621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0171621%26sid%3Dliteratum%253Aachs%26aulast%3DSidova%26aufirst%3DV.%26aulast%3DZoufaly%26aufirst%3DP.%26aulast%3DPokorny%26aufirst%3DJ.%26aulast%3DDzubak%26aufirst%3DP.%26aulast%3DHajduch%26aufirst%3DM.%26aulast%3DPopa%26aufirst%3DI.%26aulast%3DUrban%26aufirst%3DM.%26atitle%3DCytotoxic%2520conjugates%2520of%2520betulinic%2520acid%2520and%2520substituted%2520triazoles%2520prepared%2520by%2520Huisgen%2520Cycloaddition%2520from%252030-azidoderivatives%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0171621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urban, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinot, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korinkova, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis of A-seco derivatives of betulinic acid with cytotoxic activity</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">1100</span>– <span class="NLM_lpage">1105</span>, <span class="refDoi"> DOI: 10.1021/np049938m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/np049938m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD2cXltVCntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2004&pages=1100-1105&author=M.+Urbanauthor=J.+Sarekauthor=J.+Klinotauthor=G.+Korinkovaauthor=M.+Hajduch&title=Synthesis+of+A-seco+derivatives+of+betulinic+acid+with+cytotoxic+activity&doi=10.1021%2Fnp049938m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of A-Seco Derivatives of Betulinic Acid with Cytotoxic Activity</span></div><div class="casAuthors">Urban, Milan; Sarek, Jan; Klinot, Jiri; Korinkova, Gabriela; Hajduch, Marian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1100-1105</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study, the relationships between the chem. structure and cytotoxic activity of betulinic acid derivs. were investigated.  Eight lupane derivs., one of them new, five diosphenols, four of them new, two new nor derivs., five seco derivs., four of them new, and three new seco-anhydrides were synthesized from betulinic acid, and their activities were compared with the activities of known compds.  The effects of substitution on the A-ring and esterification of the carboxyl group in position 28 on cytotoxicity were of special interest.  Significant cytotoxic activity against the T-lymphoblastic leukemia cell line CEM was found in two diosphenols (TCS50 4 and 5 μmol/L) and two seco-anhydrides (TCS50 7 and 6 μmol/L).  All compds. were also tested on cancer cell lines HT 29, K562, K562 Tax, and PC-3.  Diosphenols, as the most promising group of derivs., were further tested on four more lines (A 549, DU 145, MCF 7, SK-Mel2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCecePizJvl7Vg90H21EOLACvtfcHk0lhAwIyv1oPK0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXltVCntL4%253D&md5=27a9c35cb8df089621b53f187179c27a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fnp049938m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fnp049938m%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DM.%26aulast%3DSarek%26aufirst%3DJ.%26aulast%3DKlinot%26aufirst%3DJ.%26aulast%3DKorinkova%26aufirst%3DG.%26aulast%3DHajduch%26aufirst%3DM.%26atitle%3DSynthesis%2520of%2520A-seco%2520derivatives%2520of%2520betulinic%2520acid%2520with%2520cytotoxic%2520activity%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2004%26volume%3D67%26spage%3D1100%26epage%3D1105%26doi%3D10.1021%2Fnp049938m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogundele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrest, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salzwedel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allaway, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, D. E.</span></span> <span> </span><span class="NLM_article-title">Phase I and II study of the safety, virologic effect, and pharmacokinetics/pharmacodynamics of single-dose 3-o-(3′,3′-dimethylsuccinyl)betulinic acid (bevirimat) against human immunodeficiency virus infection</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3574</span>– <span class="NLM_lpage">3581</span>, <span class="refDoi"> DOI: 10.1128/AAC.00152-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FAAC.00152-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=17638699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2007&pages=3574-3581&author=P.+F.+Smithauthor=A.+Ogundeleauthor=A.+Forrestauthor=J.+Wiltonauthor=K.+Salzwedelauthor=J.+Dotoauthor=G.+P.+Allawayauthor=D.+E.+Martin&title=Phase+I+and+II+study+of+the+safety%2C+virologic+effect%2C+and+pharmacokinetics%2Fpharmacodynamics+of+single-dose+3-o-%283%E2%80%B2%2C3%E2%80%B2-dimethylsuccinyl%29betulinic+acid+%28bevirimat%29+against+human+immunodeficiency+virus+infection&doi=10.1128%2FAAC.00152-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I and II study of the safety, virologic effect, and pharmacokinetics/pharmacodynamics of single-dose 3-O-(3',3'-dimethylsuccinyl)betulinic acid (bevirimat) against human immunodeficiency virus infection</span></div><div class="casAuthors">Smith, Patrick F.; Ogundele, Abayomi; Forrest, Alan; Wilton, John; Salzwedel, Karl; Doto, Judy; Allaway, Graham P.; Martin, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3574-3581</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Bevirimat [3-O-(3',3'-dimethylsuccinyl)betulinic acid] is the first in a new class of anti-human immunodeficiency virus (HIV) drugs that inhibit viral maturation by specifically blocking cleavage of the Gag capsid (CA) precursor, CA-SP1, to mature CA protein, resulting in defective core condensation and release of immature noninfectious virions.  Four cohorts of six HIV-infected adults, with CD4 counts of >200 and plasma viral loads of 5000 to 250,000 transcripts/mL and not currently receiving antiretroviral therapy, were randomized to receive a single oral dose of placebo, 75, 150, or 250 mg of bevirimat.  Thirty blood samples for drug concns. and 20 HIV RNA measures were collected from each subject over a 20-day period.  Candidate pharmacokinetic/pharmacodynamic models were fit to individual subjects by max. likelihood followed by Bayesian estn.; model discrimination was by cor. Akaike's Information Criterion.  The bevirimat pharmacokinetics was well described by an oral two-compartment linear model (r2, 0.98), with a mean (percent coeff. of variation) half-life of 60.3 (13.6) h and apparent oral clearance of bevirimat from the plasma compartment of 0.17 (18) liters/h.  HIV RNA was modeled as being produced in infected CD4 cells, with bevirimat inhibiting infection of new CD4 cells through a Hill-type function (r2, 0.87).  Single oral doses of bevirimat were well tolerated and demonstrated a dose-dependent redn. in viral load.  The av. max. redn. from baseline following the 150- and 250-mg doses was greater than 0.45 log10, with individual patients having redns. of greater than 0.7 log10.  No bevirimat resistance mutations were detected during the course of the study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0xx_ZTDkPSLVg90H21EOLACvtfcHk0lhAwIyv1oPK0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGhtbjM&md5=598e3191feb2ae7ac5f2d94f4a8a1c86</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FAAC.00152-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00152-07%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DP.%2BF.%26aulast%3DOgundele%26aufirst%3DA.%26aulast%3DForrest%26aufirst%3DA.%26aulast%3DWilton%26aufirst%3DJ.%26aulast%3DSalzwedel%26aufirst%3DK.%26aulast%3DDoto%26aufirst%3DJ.%26aulast%3DAllaway%26aufirst%3DG.%2BP.%26aulast%3DMartin%26aufirst%3DD.%2BE.%26atitle%3DPhase%2520I%2520and%2520II%2520study%2520of%2520the%2520safety%252C%2520virologic%2520effect%252C%2520and%2520pharmacokinetics%252Fpharmacodynamics%2520of%2520single-dose%25203-o-%25283%25E2%2580%25B2%252C3%25E2%2580%25B2-dimethylsuccinyl%2529betulinic%2520acid%2520%2528bevirimat%2529%2520against%2520human%2520immunodeficiency%2520virus%2520infection%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2007%26volume%3D51%26spage%3D3574%26epage%3D3581%26doi%3D10.1128%2FAAC.00152-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Regueiro-Ren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dicker, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanumegowda, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Second generation inhibitors of HIV-1 maturation</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">294</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.8b00656</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00656" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXis1WqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=287-294&author=A.+Regueiro-Renauthor=I.+B.+Dickerauthor=U.+Hanumegowdaauthor=N.+A.+Meanwell&title=Second+generation+inhibitors+of+HIV-1+maturation&doi=10.1021%2Facsmedchemlett.8b00656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Second Generation Inhibitors of HIV-1 Maturation</span></div><div class="casAuthors">Regueiro-Ren, Alicia; Dicker, Ira B.; Hanumegowda, Umesh; Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">287-294</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The strategy and tactics subtending the discovery and development of the second generation HIV-1 maturation inhibitor GSK-3532795/BMS-955176, a compd. that exhibits a broader spectrum of antiviral effect in vitro and in clin. studies than the prototypical maturation inhibitor bevirimat, are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokN8Btl19fDLVg90H21EOLACvtfcHk0lgaFQWtbqvl4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXis1WqtL4%253D&md5=82f8d64ac2b5c5fd54dde15b448a0f6e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00656%26sid%3Dliteratum%253Aachs%26aulast%3DRegueiro-Ren%26aufirst%3DA.%26aulast%3DDicker%26aufirst%3DI.%2BB.%26aulast%3DHanumegowda%26aufirst%3DU.%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSecond%2520generation%2520inhibitors%2520of%2520HIV-1%2520maturation%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D287%26epage%3D294%26doi%3D10.1021%2Facsmedchemlett.8b00656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">John, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span> <span> </span><span class="NLM_article-title">A general metal-free route towards the synthesis of 1,2,3-triazoles from readily available primary amines and ketones</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2885</span>– <span class="NLM_lpage">2888</span>, <span class="refDoi"> DOI: 10.1039/C5CC08347H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC5CC08347H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=26744743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVynsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2016&pages=2885-2888&author=J.+Thomasauthor=S.+Janaauthor=J.+Johnauthor=S.+Liekensauthor=W.+Dehaen&title=A+general+metal-free+route+towards+the+synthesis+of+1%2C2%2C3-triazoles+from+readily+available+primary+amines+and+ketones&doi=10.1039%2FC5CC08347H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A general metal-free route towards the synthesis of 1,2,3-triazoles from readily available primary amines and ketones</span></div><div class="casAuthors">Thomas, Joice; Jana, Sampad; John, Jubi; Liekens, Sandra; Dehaen, Wim</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2885-2888</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">An unprecedented approach that enables the direct and selective prepn. of 1,5-disubstituted 1,2,3-triazoles from abundantly available building blocks such as primary amines, enolizable ketones and 4-nitrophenyl azide as a renewable source of dinitrogen via an organocascade process has been developed.  Furthermore, this efficient methodol. also enables the synthesis of fully functionalized and fused N-substituted heterocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqi4PDkmmSf7Vg90H21EOLACvtfcHk0lgaFQWtbqvl4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVynsb3F&md5=2a2e61bc280bd33db0440774e0ad73a7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1039%2FC5CC08347H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC08347H%26sid%3Dliteratum%253Aachs%26aulast%3DThomas%26aufirst%3DJ.%26aulast%3DJana%26aufirst%3DS.%26aulast%3DJohn%26aufirst%3DJ.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DDehaen%26aufirst%3DW.%26atitle%3DA%2520general%2520metal-free%2520route%2520towards%2520the%2520synthesis%2520of%25201%252C2%252C3-triazoles%2520from%2520readily%2520available%2520primary%2520amines%2520and%2520ketones%26jtitle%3DChem.%2520Commun.%26date%3D2016%26volume%3D52%26spage%3D2885%26epage%3D2888%26doi%3D10.1039%2FC5CC08347H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Opsomer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span> <span> </span><span class="NLM_article-title">Chemoselectivity in the synthesis of 1,2,3-triazoles from enolizable ketones, primary alkylamines, and 4-nitrophenyl azide</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">4191</span>– <span class="NLM_lpage">4198</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1055%2Fs-0036-1588856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1Gqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=4191-4198&author=T.+Opsomerauthor=J.+Thomasauthor=W.+Dehaen&title=Chemoselectivity+in+the+synthesis+of+1%2C2%2C3-triazoles+from+enolizable+ketones%2C+primary+alkylamines%2C+and+4-nitrophenyl+azide"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoselectivity in the Synthesis of 1,2,3-Triazoles from Enolizable Ketones, Primary Alkylamines, and 4-Nitrophenyl Azide</span></div><div class="casAuthors">Opsomer, Tomas; Thomas, Joice; Dehaen, Wim</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4191-4198</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">1437-210X</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A general metal-free route towards the synthesis of 1,5-di- or fully-substituted 1,2,3-triazoles were designed and named as 'triazolization' reaction of ketones.  Limitations of this route were encountered in reactions with more activated ketones, where the corresponding 1-(4-nitrophenyl)-1,2,3-triazole was found back as the major product.  Interestingly, three different triazoles were formed when 1,3-diphenylacetone was used as the ketone.  1,3-Diphenylacetone was used as a model substrate to investigate the chemoselectivity of metal-free route under different reaction circumstances.  This led to the conclusion that the formation of the desired 1,2,3-triazole was favored in apolar solvents, at high temps., and in the presence of acetic acid.  By using one equiv. of acetic acid, previously inaccessible fully decorated 1,2,3-triazoles were synthesized from simple α-arylketones in moderate to high yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLUd0kHriOqbVg90H21EOLACvtfcHk0lgaFQWtbqvl4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1Gqs7s%253D&md5=76c4e7b1dc0b1b226d8ff7d7dc70cffc</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1055%2Fs-0036-1588856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0036-1588856%26sid%3Dliteratum%253Aachs%26aulast%3DOpsomer%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DJ.%26aulast%3DDehaen%26aufirst%3DW.%26atitle%3DChemoselectivity%2520in%2520the%2520synthesis%2520of%25201%252C2%252C3-triazoles%2520from%2520enolizable%2520ketones%252C%2520primary%2520alkylamines%252C%2520and%25204-nitrophenyl%2520azide%26jtitle%3DSynthesis%26date%3D2017%26volume%3D49%26spage%3D4191%26epage%3D4198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silveira-Dorta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jana, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehaen, W.</span></span> <span> </span><span class="NLM_article-title">Straightforward synthesis of enantiomerically pure 1,2,3-triazoles derived from amino esters</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3168</span>– <span class="NLM_lpage">3176</span>, <span class="refDoi"> DOI: 10.1039/C8OB00533H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC8OB00533H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=29645062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmvVequ7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=3168-3176&author=G.+Silveira-Dortaauthor=S.+Janaauthor=L.+Borkovaauthor=J.+Thomasauthor=W.+Dehaen&title=Straightforward+synthesis+of+enantiomerically+pure+1%2C2%2C3-triazoles+derived+from+amino+esters&doi=10.1039%2FC8OB00533H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Straightforward synthesis of enantiomerically pure 1,2,3-triazoles derived from amino esters</span></div><div class="casAuthors">Silveira-Dorta, Gaston; Jana, Sampad; Borkova, Lucie; Thomas, Joice; Dehaen, Wim</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3168-3176</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The synthesis of enantiomerically pure 1,4,5-trisubstituted e.g., I, 1,5-disubstituted e.g., II, and fused 1,2,3-triazole derivs. e.g., III has been developed.  The synthesis employs enantiomerically pure amino esters such as tert-Bu L-phenylalaninate, L-glutamic acid di-Et ester, tert-Bu L-isoleucinate, etc. derived from amino acids such as D-phenylalanine, L-tyrosine, L-isoleucine, etc. and com. available ketones such as acetophenone, nonan-5-one, 2-acetylfuran, acetylferrocene, etc. under metal-free conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodwtq4YfAehrVg90H21EOLACvtfcHk0lgaFQWtbqvl4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmvVequ7g%253D&md5=e1b421c0c39f9eb37711cbe0660ea4e2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC8OB00533H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8OB00533H%26sid%3Dliteratum%253Aachs%26aulast%3DSilveira-Dorta%26aufirst%3DG.%26aulast%3DJana%26aufirst%3DS.%26aulast%3DBorkova%26aufirst%3DL.%26aulast%3DThomas%26aufirst%3DJ.%26aulast%3DDehaen%26aufirst%3DW.%26atitle%3DStraightforward%2520synthesis%2520of%2520enantiomerically%2520pure%25201%252C2%252C3-triazoles%2520derived%2520from%2520amino%2520esters%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26spage%3D3168%26epage%3D3176%26doi%3D10.1039%2FC8OB00533H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Flekhter, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigmatullina, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baltina, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karachurina, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galin, F. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarudii, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolstikov, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boreko, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlova, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolaeva, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savinova, O. V.</span></span> <span> </span><span class="NLM_article-title">Synthesis of betulinic acid from betulin extract and study of the antiviral and antiulcer activity of some related terpenoids</span>. <i>Pharm. Chem. J.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">484</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1023/A:1021844705853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1023%2FA%3A1021844705853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFegsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2002&pages=484-487&author=O.+B.+Flekhterauthor=L.+R.+Nigmatullinaauthor=L.+A.+Baltinaauthor=L.+T.+Karachurinaauthor=F.+Z.+Galinauthor=F.+S.+Zarudiiauthor=G.+A.+Tolstikovauthor=E.+I.+Borekoauthor=N.+I.+Pavlovaauthor=S.+N.+Nikolaevaauthor=O.+V.+Savinova&title=Synthesis+of+betulinic+acid+from+betulin+extract+and+study+of+the+antiviral+and+antiulcer+activity+of+some+related+terpenoids&doi=10.1023%2FA%3A1021844705853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of betulinic acid from betulin extract and study of the antiviral and antiulcer activity of some related terpenoids</span></div><div class="casAuthors">Flekhter, O. B.; Nigmatullina, L. R.; Baltina, L. A.; Karachurina, L. T.; Galin, F. Z.; Zarudii, F. S.; Tolstikov, G. A.; Boreko, E. I.; Pavlova, N. I.; Nikolaeva, S. N.; Savinova, O. V.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Chemistry Journal (Translation of Khimiko-Farmatsevticheskii Zhurnal)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">484-487</span>CODEN:
                <span class="NLM_cas:coden">PCJOAU</span>;
        ISSN:<span class="NLM_cas:issn">0091-150X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Consultants Bureau</span>)
        </div><div class="casAbstract">Betulinic acid was synthesized by redn. of betulonic acid obtained from betulin ext.  The reaction involving betulonic acid and trifluoroacetic acid led to 28-oxo-allobetulone.  Investigation of the antiviral properties of 28-oxo-allobetulone in a cell culture showed inhibition of the growth of influenza type A virus.  This compd. was inactive with respect to herpes simplex virus and ECHO-6 enterovirus.  Betulonic acid and 28-oxo-allobetulone exhibited antiinflammatory properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpClydOcjiqBrVg90H21EOLACvtfcHk0lgc2bG5AWnWTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFegsA%253D%253D&md5=e9fcb00f011957d7eda1fb3320d6de9e</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1023%2FA%3A1021844705853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1021844705853%26sid%3Dliteratum%253Aachs%26aulast%3DFlekhter%26aufirst%3DO.%2BB.%26aulast%3DNigmatullina%26aufirst%3DL.%2BR.%26aulast%3DBaltina%26aufirst%3DL.%2BA.%26aulast%3DKarachurina%26aufirst%3DL.%2BT.%26aulast%3DGalin%26aufirst%3DF.%2BZ.%26aulast%3DZarudii%26aufirst%3DF.%2BS.%26aulast%3DTolstikov%26aufirst%3DG.%2BA.%26aulast%3DBoreko%26aufirst%3DE.%2BI.%26aulast%3DPavlova%26aufirst%3DN.%2BI.%26aulast%3DNikolaeva%26aufirst%3DS.%2BN.%26aulast%3DSavinova%26aufirst%3DO.%2BV.%26atitle%3DSynthesis%2520of%2520betulinic%2520acid%2520from%2520betulin%2520extract%2520and%2520study%2520of%2520the%2520antiviral%2520and%2520antiulcer%2520activity%2520of%2520some%2520related%2520terpenoids%26jtitle%3DPharm.%2520Chem.%2520J.%26date%3D2002%26volume%3D36%26spage%3D484%26epage%3D487%26doi%3D10.1023%2FA%3A1021844705853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dehelean, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">S̨oica, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ledeţi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aluaş, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zupko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Găluşcan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cinta-Pinzaru, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munteanu, M.</span></span> <span> </span><span class="NLM_article-title">Study of the betulin enriched birch bark extracts effects on human carcinoma cells and ear inflammation</span>. <i>Chem. Cent. J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">137</span>, <span class="refDoi"> DOI: 10.1186/1752-153X-6-137</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1186%2F1752-153X-6-137" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=23158079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVentg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2012&pages=137&author=C.+A.+Deheleanauthor=C.+S%CC%A8oicaauthor=I.+Lede%C5%A3iauthor=M.+Alua%C5%9Fauthor=I.+Zupkoauthor=A.+G%C4%83lu%C5%9Fcanauthor=S.+Cinta-Pinzaruauthor=M.+Munteanu&title=Study+of+the+betulin+enriched+birch+bark+extracts+effects+on+human+carcinoma+cells+and+ear+inflammation&doi=10.1186%2F1752-153X-6-137"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Study of the betulin enriched birch bark extracts effects on human carcinoma cells and ear inflammation</span></div><div class="casAuthors">Dehelean, Cristina A.; Soica, Codruta; Ledeti, Ionut; Aluas, Mihaela; Zupko, Istvan; Galuscan, Atena; Cinta-Pinzaru, Simona; Munteanu, Melania</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry Central Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">137</span>CODEN:
                <span class="NLM_cas:coden">CCJHA2</span>;
        ISSN:<span class="NLM_cas:issn">1752-153X</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Pentacyclic triterpenes, mainly betulin and betulinic acid, are valuable anticancer agents found in the bark of birch tree.  This study evaluates birch bark exts. for the active principles compn.  Results: New improved extn. methods were applied on the bark of Betula pendula in order to reach the max. content in active principles.  Exts. were analyzed by HPLC-MS, Raman, SERS and 13C NMR spectroscopy which revealed a very high yield of betulin (over 90%).  Growth inhibiting effects were measured in vitro on four malignant human cell lines: A431 (skin epidermoid carcinoma), A2780 (ovarian carcinoma), HeLa (cervix adenocarcinoma) and MCF7 (breast adenocarcinoma), by means of MTT assay.  All of the prepd. bark exts. exerted a pronounced antiproliferative effect against human cancer cell lines.  In vivo studies involved the anti-inflammatory effect of birch exts. on TPA-induced model of inflammation in mice.  Conclusions: The research revealed the efficacy of the extn. procedures as well as the antiproliferative and anti-inflammatory effects of birch exts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCZndmmC6F0rVg90H21EOLACvtfcHk0lgc2bG5AWnWTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVentg%253D%253D&md5=0bba4d02dbddec0407d885ec1a6c0c54</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1186%2F1752-153X-6-137&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1752-153X-6-137%26sid%3Dliteratum%253Aachs%26aulast%3DDehelean%26aufirst%3DC.%2BA.%26aulast%3DS%25CC%25A8oica%26aufirst%3DC.%26aulast%3DLede%25C5%25A3i%26aufirst%3DI.%26aulast%3DAlua%25C5%259F%26aufirst%3DM.%26aulast%3DZupko%26aufirst%3DI.%26aulast%3DG%25C4%2583lu%25C5%259Fcan%26aufirst%3DA.%26aulast%3DCinta-Pinzaru%26aufirst%3DS.%26aulast%3DMunteanu%26aufirst%3DM.%26atitle%3DStudy%2520of%2520the%2520betulin%2520enriched%2520birch%2520bark%2520extracts%2520effects%2520on%2520human%2520carcinoma%2520cells%2520and%2520ear%2520inflammation%26jtitle%3DChem.%2520Cent.%2520J.%26date%3D2012%26volume%3D6%26spage%3D137%26doi%3D10.1186%2F1752-153X-6-137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ressmann, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kremsmayr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaertner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zirbs, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bica, K.</span></span> <span> </span><span class="NLM_article-title">Toward a benign strategy for the manufacturing of betulinic acid</span>. <i>Green Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1022</span>, <span class="refDoi"> DOI: 10.1039/C6GC02641A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1039%2FC6GC02641A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKku7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1014-1022&author=A.+K.+Ressmannauthor=T.+Kremsmayrauthor=P.+Gaertnerauthor=R.+Zirbsauthor=K.+Bica&title=Toward+a+benign+strategy+for+the+manufacturing+of+betulinic+acid&doi=10.1039%2FC6GC02641A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Toward a benign strategy for the manufacturing of betulinic acid</span></div><div class="casAuthors">Ressmann, Anna K.; Kremsmayr, Thomas; Gaertner, Peter; Zirbs, Ronald; Bica, Katharina</div><div class="citationInfo"><span class="NLM_cas:title">Green Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1014-1022</span>CODEN:
                <span class="NLM_cas:coden">GRCHFJ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9262</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">We report a novel and efficient strategy for the prepn. of the high-value triterpenoid betulinic acid based on extn. and streamlined oxidn. of betulin from the industrial byproduct birch bark.  The initial extn. of betulin relies on a biphasic system and allowed extg. betulin in short times at room temp.  The crude ext. could be directly oxidized, thereby providing a chromium-free, time- and energy saving strategy for the manufg. of betulinic acid in high yield.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6NRBU_wi0s7Vg90H21EOLACvtfcHk0lgc2bG5AWnWTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKku7nN&md5=38acb050f7e2cd84c69003cde2ba31b0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2FC6GC02641A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6GC02641A%26sid%3Dliteratum%253Aachs%26aulast%3DRessmann%26aufirst%3DA.%2BK.%26aulast%3DKremsmayr%26aufirst%3DT.%26aulast%3DGaertner%26aufirst%3DP.%26aulast%3DZirbs%26aufirst%3DR.%26aulast%3DBica%26aufirst%3DK.%26atitle%3DToward%2520a%2520benign%2520strategy%2520for%2520the%2520manufacturing%2520of%2520betulinic%2520acid%26jtitle%3DGreen%2520Chem.%26date%3D2017%26volume%3D19%26spage%3D1014%26epage%3D1022%26doi%3D10.1039%2FC6GC02641A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Šiman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filipová, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichá, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bezrouk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Havelek, R.</span></span> <span> </span><span class="NLM_article-title">Effective method of purification of betulin from birch bark: the importance of its purity for scientific and medicinal use</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0154933</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0154933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.pone.0154933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=27152419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyrtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=P.+%C5%A0imanauthor=A.+Filipov%C3%A1author=A.+Tich%C3%A1author=M.+Niangauthor=A.+Bezroukauthor=R.+Havelek&title=Effective+method+of+purification+of+betulin+from+birch+bark%3A+the+importance+of+its+purity+for+scientific+and+medicinal+use&doi=10.1371%2Fjournal.pone.0154933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Effective method of purification of betulin from birch bark: the importance of its purity for scientific and medicinal use</span></div><div class="casAuthors">Siman, Pavel; Filipova, Alzbeta; Ticha, Alena; Niang, Mohamed; Bezrouk, Ales; Havelek, Radim</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e0154933/1-e0154933/14</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A new and relatively simple method for purifn. of betulin from birch bark ext. was developed in this study.  Its five purifn. steps are based on the differential soly. of ext. components in various solvents and their crystn. and/or pptn., on their affinity for Ca(OH)2 in ethanol, and on the affinity of some impurities for silica gel in chloroform.  In addn., all used solvents can be simply recycled.  Betulin of more than 99% purity can be prepd. by this method with minimal costs.  Various observations including crystn. of betulin, changes in crystals during heating, and attempt of localization of betulin in outer birch bark are also described in this work.  The original ext., fraction without betulinic acid and lupeol, amorphous fraction of pure betulin, final cryst. fraction of pure betulin and com. betulin as a std. were employed to det. the antiproliferative/cytotoxic effect.We usedWST-1 tetrazolium-based assays with triple neg. breast cancer cell line BT-549.  The decrease in cell survival showed clear relationship with the purity of the samples, being most pronounced using our final product of pure cryst. betulin.  WST-1 proliferation/cytotoxicity test using triple neg. breast cancer cell line BT-549 clearly showed the importance of purity of betulin for biol. expts. and, apparently, for its medicinal use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIN5k3lcMI1bVg90H21EOLACvtfcHk0ljPefEtbOcIvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyrtb%252FM&md5=55d956cc92ce9f704516cc8ba7b9816b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0154933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0154933%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0iman%26aufirst%3DP.%26aulast%3DFilipov%25C3%25A1%26aufirst%3DA.%26aulast%3DTich%25C3%25A1%26aufirst%3DA.%26aulast%3DNiang%26aufirst%3DM.%26aulast%3DBezrouk%26aufirst%3DA.%26aulast%3DHavelek%26aufirst%3DR.%26atitle%3DEffective%2520method%2520of%2520purification%2520of%2520betulin%2520from%2520birch%2520bark%253A%2520the%2520importance%2520of%2520its%2520purity%2520for%2520scientific%2520and%2520medicinal%2520use%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0154933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dijkman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kann, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamiak, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannoun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonsdottir, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kint, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forlenza, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trybala, E.</span></span> <span> </span><span class="NLM_article-title">Targeting membrane-bound viral RNA synthesis reveals potent inhibition of diverse coronaviruses including the Middle East respiratory syndrome virus</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1004166</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1004166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1371%2Fjournal.ppat.1004166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=24874215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGkur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=A.+Lundinauthor=R.+Dijkmanauthor=T.+Bergstr%C3%B6mauthor=N.+Kannauthor=B.+Adamiakauthor=C.+Hannounauthor=E.+Kindlerauthor=H.+R.+Jonsdottirauthor=D.+Muthauthor=J.+Kintauthor=M.+Forlenzaauthor=M.+A.+Mullerauthor=C.+Drostenauthor=V.+Thielauthor=E.+Trybala&title=Targeting+membrane-bound+viral+RNA+synthesis+reveals+potent+inhibition+of+diverse+coronaviruses+including+the+Middle+East+respiratory+syndrome+virus&doi=10.1371%2Fjournal.ppat.1004166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting membrane-bound viral RNA synthesis reveals potent inhibition of diverse coronaviruses including the Middle East respiratory syndrome virus</span></div><div class="casAuthors">Lundin, Anna; Dijkman, Ronald; Bergstroem, Tomas; Kann, Nina; Adamiak, Beata; Hannoun, Charles; Kindler, Eveline; Jonsdottir, Hulda R.; Muth, Doreen; Kint, Joeri; Forlenza, Maria; Mueller, Marcel A.; Drosten, Christian; Thiel, Volker; Trybala, Edward</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1004166/1-e1004166/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses raise serious concerns as emerging zoonotic viruses without specific antiviral drugs available.  Here we screened a collection of 16671 diverse compds. for anti-human coronavirus 229E activity and identified an inhibitor, designated K22, that specifically targets membrane-bound coronaviral RNA synthesis.  K22 exerts most potent antiviral activity after virus entry during an early step of the viral life cycle.  Specifically, the formation of double membrane vesicles (DMVs), a hallmark of coronavirus replication, was greatly impaired upon K22 treatment accompanied by near-complete inhibition of viral RNA synthesis.  K22-resistant viruses contained substitutions in non-structural protein 6 (nsp6), a membrane-spanning integral component of the viral replication complex implicated in DMV formation, corroborating that K22 targets membrane bound viral RNA synthesis.  Besides K22 resistance, the nsp6 mutants induced a reduced no. of DMVs, displayed decreased specific infectivity, while RNA synthesis was not affected.  Importantly, K22 inhibits a broad range of coronaviruses, including Middle East respiratory syndrome coronavirus (MERS-CoV), and efficient inhibition was achieved in primary human epithelia cultures representing the entry port of human coronavirus infection.  Collectively, this study proposes an evolutionary conserved step in the life cycle of pos.-stranded RNA viruses, the recruitment of cellular membranes for viral replication, as vulnerable and, most importantly, druggable target for antiviral intervention.  We expect this mode of action to serve as a paradigm for the development of potent antiviral drugs to combat many animal and human virus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZm0zkCr8NwLVg90H21EOLACvtfcHk0ljPefEtbOcIvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGkur7J&md5=26b2d4d8c3890db67a7bd5e16450ef46</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1004166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1004166%26sid%3Dliteratum%253Aachs%26aulast%3DLundin%26aufirst%3DA.%26aulast%3DDijkman%26aufirst%3DR.%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DT.%26aulast%3DKann%26aufirst%3DN.%26aulast%3DAdamiak%26aufirst%3DB.%26aulast%3DHannoun%26aufirst%3DC.%26aulast%3DKindler%26aufirst%3DE.%26aulast%3DJonsdottir%26aufirst%3DH.%2BR.%26aulast%3DMuth%26aufirst%3DD.%26aulast%3DKint%26aufirst%3DJ.%26aulast%3DForlenza%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DM.%2BA.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DThiel%26aufirst%3DV.%26aulast%3DTrybala%26aufirst%3DE.%26atitle%3DTargeting%2520membrane-bound%2520viral%2520RNA%2520synthesis%2520reveals%2520potent%2520inhibition%2520of%2520diverse%2520coronaviruses%2520including%2520the%2520Middle%2520East%2520respiratory%2520syndrome%2520virus%26jtitle%3DPLoS%2520Pathog.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.ppat.1004166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J. F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, S. K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, V. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, P. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuen, K.-Y.</span></span> <span> </span><span class="NLM_article-title">Middle East respiratory syndrome coronavirus: another zoonotic betacoronavirus causing SARS-like disease</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">522</span>, <span class="refDoi"> DOI: 10.1128/CMR.00102-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FCMR.00102-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=25810418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=465-522&author=J.+F.+W.+Chanauthor=S.+K.+P.+Lauauthor=K.+K.+W.+Toauthor=V.+C.+C.+Chengauthor=P.+C.+Y.+Wooauthor=K.-Y.+Yuen&title=Middle+East+respiratory+syndrome+coronavirus%3A+another+zoonotic+betacoronavirus+causing+SARS-like+disease&doi=10.1128%2FCMR.00102-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Middle east respiratory syndrome coronavirus: another zoonotic betacoronavirus causing SARS-like disease</span></div><div class="casAuthors">Chan, Jasper F. W.; Lau, Susanna K. P.; To, Kelvin K. W.; Cheng, Vincent C. C.; Woo, Patrick C. Y.; Yuen, Kwok-Yung</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">465-522</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The source of the severe acute respiratory syndrome (SARS) epidemic was traced to wildlife market civets and ultimately to bats.  Subsequent hunting for novel coronaviruses (CoVs) led to the discovery of two addnl. human and over 40 animal CoVs, including the prototype lineage C betacoronaviruses, Tylonycteris bat CoV HKU4 and Pipistrellus bat CoV HKU5; these are phylogenetically closely related to the Middle East respiratory syndrome (MERS) CoV, which has affected more than 1,000 patients with over 35% fatality since its emergence in 2012.  All primary cases of MERS are epidemiol. linked to the Middle East.  Some of these patients had contacted camels which shed virus and/or had pos. serol.  Most secondary cases are related to health care-assocd. clusters.  The disease is esp. severe in elderly men with comorbidities.  Clin. severity may be related to MERS-CoV's ability to infect a broad range of cells with DPP4 expression, evade the host innate immune response, and induce cytokine dysregulation.  Reverse transcription-PCR on respiratory and/or extrapulmonary specimens rapidly establishes diagnosis.  Supportive treatment with extracorporeal membrane oxygenation and dialysis is often required in patients with organ failure.  Antivirals with potent in vitro activities include neutralizing monoclonal antibodies, antiviral peptides, interferons, mycophenolic acid, and lopinavir.  They should be evaluated in suitable animal models before clin. trials.  Developing an effective camel MERS-CoV vaccine and implementing appropriate infection control measures may control the continuing epidemic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2NNE4QR663rVg90H21EOLACvtfcHk0ljPefEtbOcIvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltlyhs74%253D&md5=c94188fe0de72aeb7df66a1f2144dde4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1128%2FCMR.00102-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00102-14%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DJ.%2BF.%2BW.%26aulast%3DLau%26aufirst%3DS.%2BK.%2BP.%26aulast%3DTo%26aufirst%3DK.%2BK.%2BW.%26aulast%3DCheng%26aufirst%3DV.%2BC.%2BC.%26aulast%3DWoo%26aufirst%3DP.%2BC.%2BY.%26aulast%3DYuen%26aufirst%3DK.-Y.%26atitle%3DMiddle%2520East%2520respiratory%2520syndrome%2520coronavirus%253A%2520another%2520zoonotic%2520betacoronavirus%2520causing%2520SARS-like%2520disease%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2015%26volume%3D28%26spage%3D465%26epage%3D522%26doi%3D10.1128%2FCMR.00102-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak</span>. <i>Emerging Microbes Infect.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">319</span>, <span class="refDoi"> DOI: 10.1080/22221751.2020.1725399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1080%2F22221751.2020.1725399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=32020836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFOktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&pages=313-319&author=L.+Chenauthor=W.+Liuauthor=Q.+Zhangauthor=K.+Xuauthor=G.+Yeauthor=W.+Wuauthor=Z.+Sunauthor=F.+Liuauthor=K.+Wuauthor=B.+Zhongauthor=Y.+Meiauthor=W.+Zhangauthor=Y.+Chenauthor=Y.+Liauthor=M.+Shiauthor=K.+Lanauthor=Y.+Liu&title=RNA+based+mNGS+approach+identifies+a+novel+human+coronavirus+from+two+individual+pneumonia+cases+in+2019+Wuhan+outbreak&doi=10.1080%2F22221751.2020.1725399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">RNA based mNGS approach identifies a novel human coronavirus from two individual pneumonia cases in 2019 Wuhan outbreak</span></div><div class="casAuthors">Chen, Liangjun; Liu, Weiyong; Zhang, Qi; Xu, Ke; Ye, Guangming; Wu, Weichen; Sun, Ziyong; Liu, Fang; Wu, Kailang; Zhong, Bo; Mei, Yi; Zhang, Wenxia; Chen, Yu; Li, Yirong; Shi, Mang; Lan, Ke; Liu, Yingle</div><div class="citationInfo"><span class="NLM_cas:title">Emerging Microbes & Infections</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">313-319</span>CODEN:
                <span class="NLM_cas:coden">EMIMC4</span>;
        ISSN:<span class="NLM_cas:issn">2222-1751</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">From Dec. 2019, an outbreak of unusual pneumonia was reported in Wuhan with many cases linked to Huanan Seafood Market that sells seafood as well as live exotic animals.  We investigated two patients who developed acute respiratory syndromes after independent contact history with this market.  The two patients shared common clin. features including fever, cough, and multiple ground-glass opacities in the bilateral lung field with patchy infiltration.  Here, we highlight the use of a low-input metagenomic next-generation sequencing (mNGS) approach on RNA extd. from bronchoalveolar lavage fluid (BALF).  It rapidly identified a novel coronavirus (named 2019-nCoV according to World Health Organization announcement) which was the sole pathogens in the sample with very high abundance level (1.5% and 0.62% of total RNA sequenced).  The entire viral genome is 29,881 nt in length (GenBank MN988668 and MN988669, Sequence Read Archive database Bioproject accession PRJNA601736) and is classified into β-coronavirus genus.  Phylogenetic anal. indicates that 2019-nCoV is close to coronaviruses (CoVs) circulating in Rhinolophus (Horseshoe bats), such as 98.7% nucleotide identity to partial RdRp gene of bat coronavirus strain BtCoV/4991 (GenBank KP876546, 370 nt sequence of RdRp and lack of other genome sequence) and 87.9% nucleotide identity to bat coronavirus strain bat-SL-CoVZC45 and bat-SL-CoVZXC21.  Evolutionary anal. based on ORF1a/1b, S, and N genes also suggests 2019-nCoV is more likely a novel CoV independently introduced from animals to humans.  Read Archive database Bioproject accession PRJNA601736.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmGr3MxtQ3mrVg90H21EOLACvtfcHk0ljLP2jt_aefCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFOktbo%253D&md5=87cb9016fc26ee2e4948a322c4155223</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1080%2F22221751.2020.1725399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F22221751.2020.1725399%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DZhong%26aufirst%3DB.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DM.%26aulast%3DLan%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DRNA%2520based%2520mNGS%2520approach%2520identifies%2520a%2520novel%2520human%2520coronavirus%2520from%2520two%2520individual%2520pneumonia%2520cases%2520in%25202019%2520Wuhan%2520outbreak%26jtitle%3DEmerging%2520Microbes%2520Infect.%26date%3D2020%26volume%3D9%26spage%3D313%26epage%3D319%26doi%3D10.1080%2F22221751.2020.1725399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persoons, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanstreels, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Looveren, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boeckx, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geudens, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Man, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochmans, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wauters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wauters, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanaudenaerde, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambrechts, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dallmeier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibaut, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquemyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daelemans, D.</span></span> <span> </span><span class="NLM_article-title">Genome-wide CRISPR screening identifies TMEM106B as a proviral host factor for SARS-CoV-2</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_fpage">435</span>, <span class="refDoi"> DOI: 10.1038/s41588-021-00805-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1038%2Fs41588-021-00805-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=33686287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BB3MXmtVSrt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2021&pages=435&author=J.+Baggenauthor=L.+Persoonsauthor=E.+Vanstreelsauthor=S.+Jansenauthor=D.+Van+Looverenauthor=B.+Boeckxauthor=V.+Geudensauthor=J.+De+Manauthor=D.+Jochmansauthor=J.+Wautersauthor=E.+Wautersauthor=B.+M.+Vanaudenaerdeauthor=D.+Lambrechtsauthor=J.+Neytsauthor=K.+Dallmeierauthor=H.+J.+Thibautauthor=M.+Jacquemynauthor=P.+Maesauthor=D.+Daelemans&title=Genome-wide+CRISPR+screening+identifies+TMEM106B+as+a+proviral+host+factor+for+SARS-CoV-2&doi=10.1038%2Fs41588-021-00805-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Genome-wide CRISPR screening identifies TMEM106B as a proviral host factor for SARS-CoV-2</span></div><div class="casAuthors">Baggen, Jim; Persoons, Leentje; Vanstreels, Els; Jansen, Sander; Van Looveren, Dominique; Boeckx, Bram; Geudens, Vincent; De Man, Julie; Jochmans, Dirk; Wauters, Joost; Wauters, Els; Vanaudenaerde, Bart M.; Lambrechts, Diether; Neyts, Johan; Dallmeier, Kai; Thibaut, Hendrik Jan; Jacquemyn, Maarten; Maes, Piet; Daelemans, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">435-444</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Portfolio</span>)
        </div><div class="casAbstract">Abstr.: The ongoing COVID-19 pandemic has caused a global economic and health crisis.  To identify host factors essential for coronavirus infection, we performed genome-wide functional genetic screens with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and human coronavirus 229E.  These screens uncovered virus-specific as well as shared host factors, including TMEM41B and PI3K type 3.  We discovered that SARS-CoV-2 requires the lysosomal protein TMEM106B to infect human cell lines and primary lung cells.  TMEM106B overexpression enhanced SARS-CoV-2 infection as well as pseudovirus infection, suggesting a role in viral entry.  Furthermore, single-cell RNA-sequencing of airway cells from patients with COVID-19 demonstrated that TMEM106B expression correlates with SARS-CoV-2 infection.  The present study uncovered a collection of coronavirus host factors that may be exploited to develop drugs against SARS-CoV-2 infection or future zoonotic coronavirus outbreaks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyc63uZRz4gbVg90H21EOLACvtfcHk0ljLP2jt_aefCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXmtVSrt7s%253D&md5=29d1554b6afe63cf8b7f9c3553d9038f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fs41588-021-00805-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-021-00805-2%26sid%3Dliteratum%253Aachs%26aulast%3DBaggen%26aufirst%3DJ.%26aulast%3DPersoons%26aufirst%3DL.%26aulast%3DVanstreels%26aufirst%3DE.%26aulast%3DJansen%26aufirst%3DS.%26aulast%3DVan%2BLooveren%26aufirst%3DD.%26aulast%3DBoeckx%26aufirst%3DB.%26aulast%3DGeudens%26aufirst%3DV.%26aulast%3DDe%2BMan%26aufirst%3DJ.%26aulast%3DJochmans%26aufirst%3DD.%26aulast%3DWauters%26aufirst%3DJ.%26aulast%3DWauters%26aufirst%3DE.%26aulast%3DVanaudenaerde%26aufirst%3DB.%2BM.%26aulast%3DLambrechts%26aufirst%3DD.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3DDallmeier%26aufirst%3DK.%26aulast%3DThibaut%26aufirst%3DH.%2BJ.%26aulast%3DJacquemyn%26aufirst%3DM.%26aulast%3DMaes%26aufirst%3DP.%26aulast%3DDaelemans%26aufirst%3DD.%26atitle%3DGenome-wide%2520CRISPR%2520screening%2520identifies%2520TMEM106B%2520as%2520a%2520proviral%2520host%2520factor%2520for%2520SARS-CoV-2%26jtitle%3DNat.%2520Genet.%26date%3D2021%26spage%3D435%26doi%3D10.1038%2Fs41588-021-00805-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palaninathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcantara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, C. C.</span></span> <span> </span><span class="NLM_article-title">Structural and functional analyses of the severe acute respiratory syndrome coronavirus endoribonuclease Nsp15</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">3655</span>– <span class="NLM_lpage">3664</span>, <span class="refDoi"> DOI: 10.1074/jbc.M708375200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1074%2Fjbc.M708375200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=18045871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlyitLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=3655-3664&author=K.+Bhardwajauthor=S.+Palaninathanauthor=J.+M.+Alcantaraauthor=L.+L.+Yiauthor=L.+Guarinoauthor=J.+C.+Sacchettiniauthor=C.+C.+Kao&title=Structural+and+functional+analyses+of+the+severe+acute+respiratory+syndrome+coronavirus+endoribonuclease+Nsp15&doi=10.1074%2Fjbc.M708375200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Functional Analyses of the Severe Acute Respiratory Syndrome Coronavirus Endoribonuclease Nsp15</span></div><div class="casAuthors">Bhardwaj, Kanchan; Palaninathan, Satheesh; Alcantara, Joanna Maria Ortiz; Li Yi, Lillian; Guarino, Linda; Sacchettini, James C.; Kao, C. Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3655-3664</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The severe acute respiratory syndrome (SARS) coronavirus encodes several RNA-processing enzymes that are unusual for RNA viruses, including Nsp15 (nonstructural protein 15), a hexameric endoribonuclease that preferentially cleaves 3' of uridines.  We solved the structure of a catalytically inactive mutant version of Nsp15, which was crystd. as a hexamer.  The structure contains unreported flexibility in the active site of each subunit.  Substitutions in the active site residues serine 293 and proline 343 allowed Nsp15 to cleave at cytidylate, whereas mutation of leucine 345 rendered Nsp15 able to cleave at purines as well as pyrimidines.  Mutations that targeted the residues involved in subunit interactions generally resulted in the formation of catalytically inactive monomers.  The RNA-binding residues were mapped by a method linking reversible crosslinking, RNA affinity purifn., and peptide fingerprinting.  Alanine substitution of several residues in the RNA-contacting portion of Nsp15 did not affect hexamer formation but decreased the affinity of RNA binding and reduced endonuclease activity.  This suggests a model for Nsp15 hexamer interaction with RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz2zWq3VF2j7Vg90H21EOLACvtfcHk0ljLP2jt_aefCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlyitLc%253D&md5=f4aa526b6a9384f1cee8158a9881da40</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M708375200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M708375200%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DK.%26aulast%3DPalaninathan%26aufirst%3DS.%26aulast%3DAlcantara%26aufirst%3DJ.%2BM.%26aulast%3DYi%26aufirst%3DL.%2BL.%26aulast%3DGuarino%26aufirst%3DL.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DKao%26aufirst%3DC.%2BC.%26atitle%3DStructural%2520and%2520functional%2520analyses%2520of%2520the%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520endoribonuclease%2520Nsp15%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D3655%26epage%3D3664%26doi%3D10.1074%2Fjbc.M708375200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuman, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchmeier, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, P.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of a monomeric form of severe acute respiratory syndrome coronavirus endonuclease nsp15 suggests a role for hexamerization as an allosteric switch</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">6700</span>– <span class="NLM_lpage">6708</span>, <span class="refDoi"> DOI: 10.1128/JVI.02817-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1128%2FJVI.02817-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=17409150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Ort7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=6700-6708&author=J.+S.+Josephauthor=K.+S.+Saikatenduauthor=V.+Subramanianauthor=B.+W.+Neumanauthor=M.+J.+Buchmeierauthor=R.+C.+Stevensauthor=P.+Kuhn&title=Crystal+structure+of+a+monomeric+form+of+severe+acute+respiratory+syndrome+coronavirus+endonuclease+nsp15+suggests+a+role+for+hexamerization+as+an+allosteric+switch&doi=10.1128%2FJVI.02817-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a monomeric form of severe acute respiratory syndrome coronavirus endonuclease nsp15 suggests a role for hexamerization as an allosteric switch</span></div><div class="casAuthors">Joseph, Jeremiah S.; Saikatendu, Kumar Singh; Subramanian, Vanitha; Neuman, Benjamin W.; Buchmeier, Michael J.; Stevens, Raymond C.; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6700-6708</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Mature nonstructural protein-15 (nsp15) from the severe acute respiratory syndrome coronavirus (SARS-CoV) contains a novel uridylate-specific Mn2+-dependent endoribonuclease (NendoU).  Structure studies of the full-length form of the obligate hexameric enzyme from two CoVs, SARS-CoV and murine hepatitis virus, and its monomeric homolog, XendoU from Xenopus laevis, combined with mutagenesis studies have implicated several residues in enzymic activity and the N-terminal domain as the major determinant of hexamerization.  However, the tight link between hexamerization and enzyme activity in NendoUs has remained an enigma.  Here, we report the structure of a trimmed, monomeric form of SARS-CoV nsp15 (residues 28 to 335) detd. to a resoln. of 2.9 Å.  The catalytic loop (residues 234 to 249) with its two reactive histidines (His 234 and His 249) is dramatically flipped by ∼120° into the active site cleft.  Furthermore, the catalytic nucleophile Lys 289 points in a diametrically opposite direction, a consequence of an outward displacement of the supporting loop (residues 276 to 295).  In the full-length hexameric forms, these two loops are packed against each other and are stabilized by intimate intersubunit interactions.  Our results support the hypothesis that absence of an adjacent monomer due to deletion of the hexamerization domain is the most likely cause for disruption of the active site, offering a structural basis for why only the hexameric form of this enzyme is active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ9KTQXgd2z7Vg90H21EOLACvtfcHk0ljtIygGF2DO_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Ort7w%253D&md5=6dc7bf273fe8bc104273e0a9b481a0ad</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1128%2FJVI.02817-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.02817-06%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph%26aufirst%3DJ.%2BS.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DSubramanian%26aufirst%3DV.%26aulast%3DNeuman%26aufirst%3DB.%2BW.%26aulast%3DBuchmeier%26aufirst%3DM.%2BJ.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DCrystal%2520structure%2520of%2520a%2520monomeric%2520form%2520of%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520endonuclease%2520nsp15%2520suggests%2520a%2520role%2520for%2520hexamerization%2520as%2520an%2520allosteric%2520switch%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D6700%26epage%3D6708%26doi%3D10.1128%2FJVI.02817-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Genet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strehle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoonjans, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auwerx, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saladin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span> <span> </span><span class="NLM_article-title">Redefining the TGR5 triterpenoid binding pocket at the C-3 position</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1983</span>– <span class="NLM_lpage">1988</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201000329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1002%2Fcmdc.201000329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=20979081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsr7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1983-1988&author=C.+Genetauthor=C.+Schmidtauthor=A.+Strehleauthor=K.+Schoonjansauthor=J.+Auwerxauthor=R.+Saladinauthor=A.+Wagner&title=Redefining+the+TGR5+triterpenoid+binding+pocket+at+the+C-3+position&doi=10.1002%2Fcmdc.201000329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Redefining the TGR5 Triterpenoid Binding Pocket at the C-3 Position</span></div><div class="casAuthors">Genet, Cedric; Schmidt, Celine; Strehle, Axelle; Schoonjans, Kristina; Auwerx, Johan; Saladin, Regis; Wagner, Alain</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1983-1988</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The synthesis and biol. evaluation of new C-3 modified betulinic acid derivs., which can broaden the understanding of the putative C-3 binding pocket of TGR5 is undertaken.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx_nJBfdvJRbVg90H21EOLACvtfcHk0ljtIygGF2DO_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVOrsr7M&md5=25047211b92571537e00f49fac66e80b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000329%26sid%3Dliteratum%253Aachs%26aulast%3DGenet%26aufirst%3DC.%26aulast%3DSchmidt%26aufirst%3DC.%26aulast%3DStrehle%26aufirst%3DA.%26aulast%3DSchoonjans%26aufirst%3DK.%26aulast%3DAuwerx%26aufirst%3DJ.%26aulast%3DSaladin%26aufirst%3DR.%26aulast%3DWagner%26aufirst%3DA.%26atitle%3DRedefining%2520the%2520TGR5%2520triterpenoid%2520binding%2520pocket%2520at%2520the%2520C-3%2520position%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1983%26epage%3D1988%26doi%3D10.1002%2Fcmdc.201000329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reed, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muench, H.</span></span> <span> </span><span class="NLM_article-title">A simple method of estimating fifty per cent endpoints</span>. <i>Am. J. Epidemiol.</i> <span class="NLM_year" style="font-weight: bold;">1938</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">493</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.aje.a118408</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1093%2Foxfordjournals.aje.a118408" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1938&pages=493-497&author=L.+J.+Reedauthor=H.+Muench&title=A+simple+method+of+estimating+fifty+per+cent+endpoints&doi=10.1093%2Foxfordjournals.aje.a118408"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.aje.a118408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.aje.a118408%26sid%3Dliteratum%253Aachs%26aulast%3DReed%26aufirst%3DL.%2BJ.%26aulast%3DMuench%26aufirst%3DH.%26atitle%3DA%2520simple%2520method%2520of%2520estimating%2520fifty%2520per%2520cent%2520endpoints%26jtitle%3DAm.%2520J.%2520Epidemiol.%26date%3D1938%26volume%3D27%26spage%3D493%26epage%3D497%26doi%3D10.1093%2Foxfordjournals.aje.a118408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Apaydin, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesur, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cesur, Z.</span></span> <span> </span><span class="NLM_article-title">Synthesis and anti-coronavirus activity of a series of 1-thia-4-azaspiro[4.5]decan-3-one derivatives</span>. <i>Arch. Pharm. (Weinheim)</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>352</i></span>, <span class="NLM_elocation-id">e1800330</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1002%2Fardp.201800330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=31073993" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2019&author=C.+B.+Apaydinauthor=N.+Cesurauthor=A.+Stevaertauthor=L.+Naesensauthor=Z.+Cesur&title=Synthesis+and+anti-coronavirus+activity+of+a+series+of+1-thia-4-azaspiro%5B4.5%5Ddecan-3-one+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1002%2Fardp.201800330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fardp.201800330%26sid%3Dliteratum%253Aachs%26aulast%3DApaydin%26aufirst%3DC.%2BB.%26aulast%3DCesur%26aufirst%3DN.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DCesur%26aufirst%3DZ.%26atitle%3DSynthesis%2520and%2520anti-coronavirus%2520activity%2520of%2520a%2520series%2520of%25201-thia-4-azaspiro%255B4.5%255Ddecan-3-one%2520derivatives%26jtitle%3DArch.%2520Pharm.%2520%2528Weinheim%2529%26date%3D2019%26volume%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vrijens, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noppen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boogaerts, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanstreels, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronca, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiodelli, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laporte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanderlinden, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liekens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span> <span> </span><span class="NLM_article-title">Influenza virus entry via the GM3 ganglioside-mediated platelet-derived growth factor receptor β signalling pathway</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">601</span>, <span class="refDoi"> DOI: 10.1099/jgv.0.001235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.1099%2Fjgv.0.001235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=30762518" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVWrtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2019&pages=583-601&author=P.+Vrijensauthor=S.+Noppenauthor=T.+Boogaertsauthor=E.+Vanstreelsauthor=R.+Roncaauthor=P.+Chiodelliauthor=M.+Laporteauthor=E.+Vanderlindenauthor=S.+Liekensauthor=A.+Stevaertauthor=L.+Naesens&title=Influenza+virus+entry+via+the+GM3+ganglioside-mediated+platelet-derived+growth+factor+receptor+%CE%B2+signalling+pathway&doi=10.1099%2Fjgv.0.001235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Influenza virus entry via the GM3 ganglioside-mediated platelet-derived growth factor receptor β signalling pathway</span></div><div class="casAuthors">Vrijens, Pieter; Noppen, Sam; Boogaerts, Talitha; Vanstreels, Els; Ronca, Roberto; Chiodelli, Paola; Laporte, Manon; Vanderlinden, Evelien; Liekens, Sandra; Stevaert, Annelies; Naesens, Lieve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">583-601</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">1465-2099</span>.
    
            (<span class="NLM_cas:orgname">Microbiology Society</span>)
        </div><div class="casAbstract">This requires a better understanding of virus- host interactions during the entry process, including the role of receptor tyrosine kinases (RTKs).  To search for cellular targets, we evaluated a panel of 276 protein kinase inhibitors in a multicycle antiviral assay in Madin-Darby canine kidney cells.  The RTK inhibitor Ki8751 displayed robust anti-influenza A and B virus activity and was selected for mechanistic investigations.  Ki8751 efficiently disrupted the endocytic process of influenza virus in different cell lines carrying plateletderived growth factor receptor β (PDGFRb), an RTK that is known to act at GM3 ganglioside-pos. lipid rafts.  The more efficient virus entry in CHO-K1 cells compared to the wild-type ancestor (CHO-wt) cells indicated a pos. effect of GM3, which is abundant in CHO-K1 but not in CHO-wt cells.  Entering virus localized to GM3-pos. lipid rafts and the PDGFRbcontaining endosomal compartment.  PDGFRβ/GM3-dependent virus internalization involved PDGFRβ phosphorylation, which was potently inhibited by Ki8751, and desialylation of activated PDGFRb by the viral neuraminidase.  Virus uptake coincided with strong activation of the Raf/MEK/Erk cascade, but not of PI3K/Akt or phospholipase C-γ.  We conclude that influenza virus efficiently hijacks the GM3-enhanced PDGFR β signaling pathway for cell penetration, providing an opportunity for host cell-targeting antiviral intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq594Z0J-3b3rVg90H21EOLACvtfcHk0liOJ96Mw99sBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVWrtL%252FO&md5=a1ee95dd1afd4a15abd613de41e43358</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1099%2Fjgv.0.001235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fjgv.0.001235%26sid%3Dliteratum%253Aachs%26aulast%3DVrijens%26aufirst%3DP.%26aulast%3DNoppen%26aufirst%3DS.%26aulast%3DBoogaerts%26aufirst%3DT.%26aulast%3DVanstreels%26aufirst%3DE.%26aulast%3DRonca%26aufirst%3DR.%26aulast%3DChiodelli%26aufirst%3DP.%26aulast%3DLaporte%26aufirst%3DM.%26aulast%3DVanderlinden%26aufirst%3DE.%26aulast%3DLiekens%26aufirst%3DS.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DNaesens%26aufirst%3DL.%26atitle%3DInfluenza%2520virus%2520entry%2520via%2520the%2520GM3%2520ganglioside-mediated%2520platelet-derived%2520growth%2520factor%2520receptor%2520%25CE%25B2%2520signalling%2520pathway%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2019%26volume%3D100%26spage%3D583%26epage%3D601%26doi%3D10.1099%2Fjgv.0.001235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szűcs, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevaert, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostorházi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herczegh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borbás, A.</span></span> <span> </span><span class="NLM_article-title">Reprogramming of the antibacterial drug vancomycin results in potent antiviral agents devoid of antibacterial activity</span>. <i>Pharmaceuticals (Basel)</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">139</span>, <span class="refDoi"> DOI: 10.3390/ph13070139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;key=10.3390%2Fph13070139" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2020&pages=139&author=Z.+Sz%C5%B1csauthor=L.+Naesensauthor=A.+Stevaertauthor=E.+Ostorh%C3%A1ziauthor=G.+Battaauthor=P.+Herczeghauthor=A.+Borb%C3%A1s&title=Reprogramming+of+the+antibacterial+drug+vancomycin+results+in+potent+antiviral+agents+devoid+of+antibacterial+activity&doi=10.3390%2Fph13070139"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.3390%2Fph13070139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph13070139%26sid%3Dliteratum%253Aachs%26aulast%3DSz%25C5%25B1cs%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DStevaert%26aufirst%3DA.%26aulast%3DOstorh%25C3%25A1zi%26aufirst%3DE.%26aulast%3DBatta%26aufirst%3DG.%26aulast%3DHerczegh%26aufirst%3DP.%26aulast%3DBorb%25C3%25A1s%26aufirst%3DA.%26atitle%3DReprogramming%2520of%2520the%2520antibacterial%2520drug%2520vancomycin%2520results%2520in%2520potent%2520antiviral%2520agents%2520devoid%2520of%2520antibacterial%2520activity%26jtitle%3DPharmaceuticals%2520%2528Basel%2529%26date%3D2020%26volume%3D13%26spage%3D139%26doi%3D10.3390%2Fph13070139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RS4','PDB','4RS4'); return false;">PDB: 4RS4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7K1O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7K1O','PDB','7K1O'); return false;">PDB: 7K1O</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02124">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80063"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02124?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02124</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">(Figure S1) Alignment of nsp15 protein sequences, HPLC traces of key compounds, and <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of the synthesized compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_001.pdf">PDF</a>)</p></li><li><p class="inline">PDB coordinates for computational models: <b>5h</b> in complex with nsp15 from HCoV-229E (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_002.pdb">PDB</a>)</p></li><li><p class="inline">PDB coordinates for computational models: <b>5h</b> in complex with nsp15 from SARS-CoV-2 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_004.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_001.pdf">jm0c02124_si_001.pdf (8.54 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_002.pdb">jm0c02124_si_002.pdb (880.46 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_003.pdb">jm0c02124_si_003.pdb (889.72 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02124/suppl_file/jm0c02124_si_004.csv">jm0c02124_si_004.csv (3.84 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02124&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02124%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02124" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a4380488832d7","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
