
<html lang="en"     class="pb-page"  data-request-id="556b5845-3e82-49cd-b2cf-5d53b2803f8f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-22;article:article:10.1021/acs.jmedchem.9b00988"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Small Molecule Antagonists of the USP5 Zinc Finger Ubiquitin-Binding Domain" /></meta><meta name="dc.Creator" content="Mandeep K.  Mann" /></meta><meta name="dc.Creator" content="Ivan  Franzoni" /></meta><meta name="dc.Creator" content="Renato Ferreira  de Freitas" /></meta><meta name="dc.Creator" content="Wolfram  Tempel" /></meta><meta name="dc.Creator" content="Scott  Houliston" /></meta><meta name="dc.Creator" content="Leanna  Smith" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Rachel J.  Harding" /></meta><meta name="dc.Creator" content="Matthieu  Schapira" /></meta><meta name="dc.Description" content="USP5 disassembles unanchored polyubiquitin chains to recycle free monoubiquitin, and is one of the 12 ubiquitin specific proteases featuring a zinc finger ubiquitin-binding domain (ZnF-UBD). This d..." /></meta><meta name="Description" content="USP5 disassembles unanchored polyubiquitin chains to recycle free monoubiquitin, and is one of the 12 ubiquitin specific proteases featuring a zinc finger ubiquitin-binding domain (ZnF-UBD). This d..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 30, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00988" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00988" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00988" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00988" /></link>
        
    
    

<title>Discovery of Small Molecule Antagonists of the USP5 Zinc Finger Ubiquitin-Binding Domain | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00988" /></meta><meta property="og:title" content="Discovery of Small Molecule Antagonists of the USP5 Zinc Finger Ubiquitin-Binding Domain" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0009.jpeg" /></meta><meta property="og:description" content="USP5 disassembles unanchored polyubiquitin chains to recycle free monoubiquitin, and is one of the 12 ubiquitin specific proteases featuring a zinc finger ubiquitin-binding domain (ZnF-UBD). This distinct structural module has been associated with substrate positioning or allosteric modulation of catalytic activity, but its cellular function remains unclear. We screened a chemical library focused on the ZnF-UBD of USP5, crystallized hits in complex with the protein, and generated a preliminary structure–activity relationship, which enables the development of more potent and selective compounds. This work serves as a framework for the discovery of a chemical probe to delineate the function of USP5 ZnF-UBD in proteasomal degradation and other ubiquitin signaling pathways in health and disease." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00988"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00988">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00988&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00988&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00988&amp;href=/doi/10.1021/acs.jmedchem.9b00988" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10144-10155</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00952" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01000" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Small Molecule Antagonists of the USP5 Zinc Finger Ubiquitin-Binding Domain</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Mandeep K. Mann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mandeep K. Mann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, University of Toronto, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mandeep+K.++Mann">Mandeep K. Mann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8252-6107" title="Orcid link">http://orcid.org/0000-0002-8252-6107</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ivan Franzoni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ivan Franzoni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Toronto, 80 St. George St., Toronto, Ontario M5S 3H6, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ivan++Franzoni">Ivan Franzoni</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8110-6218" title="Orcid link">http://orcid.org/0000-0001-8110-6218</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Renato Ferreira de Freitas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renato Ferreira de Freitas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renato+Ferreira++de+Freitas">Renato Ferreira de Freitas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wolfram Tempel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wolfram Tempel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wolfram++Tempel">Wolfram Tempel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott Houliston</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott Houliston</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">University Health Network, 661 University Avenue, Toronto, Ontario M5G 2C4, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott++Houliston">Scott Houliston</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Leanna Smith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Leanna Smith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Leanna++Smith">Leanna Smith</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Masoud Vedadi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Masoud Vedadi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, University of Toronto, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheryl H. Arrowsmith</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheryl H. Arrowsmith</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">University Health Network, 661 University Avenue, Toronto, Ontario M5G 2C4, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rachel J. Harding</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachel J. Harding</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#750714161d10195b1d1407111c1b123500011a071a1b011a5b1614"><span class="__cf_email__" data-cfemail="99ebf8faf1fcf5b7f1f8ebfdf0f7fed9ecedf6ebf6f7edf6b7faf8">[email protected]</span></a> (R.J.H.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachel+J.++Harding">Rachel J. Harding</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1134-391X" title="Orcid link">http://orcid.org/0000-0002-1134-391X</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Matthieu Schapira</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthieu Schapira</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Structural Genomics Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite 700, Toronto, Ontario M5G 1L7, Canada</div><div class="loa-info-affiliations-info">Department of Pharmacology and Toxicology, University of Toronto, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#dfb2beababb7b6baaaf1acbcb7beafb6adbe9faaabb0adb0b1abb0f1bcbe"><span class="__cf_email__" data-cfemail="c4a9a5b0b0acada1b1eab7a7aca5b4adb6a584b1b0abb6abaab0abeaa7a5">[email protected]</span></a> (M.S.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthieu++Schapira">Matthieu Schapira</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link">http://orcid.org/0000-0002-1047-3309</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00988&amp;href=/doi/10.1021%2Facs.jmedchem.9b00988" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10144–10155</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 30, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 June 2019</li><li><span class="item_label"><b>Published</b> online</span>30 October 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00988" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00988</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10144%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMandeep%2BK.%2BMann%252C%2BIvan%2BFranzoni%252C%2BRenato%2BFerreira%2Bde%2BFreitas%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D22%26contentID%3Dacs.jmedchem.9b00988%26title%3DDiscovery%2Bof%2BSmall%2BMolecule%2BAntagonists%2Bof%2Bthe%2BUSP5%2BZinc%2BFinger%2BUbiquitin-Binding%2BDomain%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10155%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00988"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1295</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00988" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Small Molecule Antagonists of the USP5 Zinc Finger Ubiquitin-Binding Domain&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Mandeep&quot;,&quot;last_name&quot;:&quot;K. Mann&quot;},{&quot;first_name&quot;:&quot;Ivan&quot;,&quot;last_name&quot;:&quot;Franzoni&quot;},{&quot;first_name&quot;:&quot;Renato&quot;,&quot;last_name&quot;:&quot;Ferreira de Freitas&quot;},{&quot;first_name&quot;:&quot;Wolfram&quot;,&quot;last_name&quot;:&quot;Tempel&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;Houliston&quot;},{&quot;first_name&quot;:&quot;Leanna&quot;,&quot;last_name&quot;:&quot;Smith&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Rachel&quot;,&quot;last_name&quot;:&quot;J. Harding&quot;},{&quot;first_name&quot;:&quot;Matthieu&quot;,&quot;last_name&quot;:&quot;Schapira&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10144-10155&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00988&quot;},&quot;abstract&quot;:&quot;USP5 disassembles unanchored polyubiquitin chains to recycle free monoubiquitin, and is one of the 12 ubiquitin specific proteases featuring a zinc finger ubiquitin-binding domain (ZnF-UBD). This distinct structural module has been associated with substrate positioning or allosteric modulation of catalytic activity, but its cellular function remains unclear. We screened a chemical library focused on the ZnF-UBD of USP5, crystallized hits in complex with the protein, and generated a preliminary structure–activity relationship, which enables the development of more potent and selective compounds. This work serves as a framework for the discovery of a chemical probe to delineate the function of USP5 ZnF-UBD in proteasomal degradation and other ubiquitin signaling pathways in health and disease.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00988&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00988" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00988&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00988" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00988&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00988" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00988&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00988&amp;href=/doi/10.1021/acs.jmedchem.9b00988" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00988" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00988" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00988%26sid%3Dliteratum%253Aachs%26pmid%3D31663737%26genre%3Darticle%26aulast%3DMann%26date%3D2019%26atitle%3DDiscovery%2Bof%2BSmall%2BMolecule%2BAntagonists%2Bof%2Bthe%2BUSP5%2BZinc%2BFinger%2BUbiquitin-Binding%2BDomain%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D22%26spage%3D10144%26epage%3D10155%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (16)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291774" title="Nuclear magnetic resonance spectroscopy">Nuclear magnetic resonance spectroscopy</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/jmcmar.2019.62.issue-22/20191127/jmcmar.2019.62.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">USP5 disassembles unanchored polyubiquitin chains to recycle free monoubiquitin, and is one of the 12 ubiquitin specific proteases featuring a zinc finger ubiquitin-binding domain (ZnF-UBD). This distinct structural module has been associated with substrate positioning or allosteric modulation of catalytic activity, but its cellular function remains unclear. We screened a chemical library focused on the ZnF-UBD of USP5, crystallized hits in complex with the protein, and generated a preliminary structure–activity relationship, which enables the development of more potent and selective compounds. This work serves as a framework for the discovery of a chemical probe to delineate the function of USP5 ZnF-UBD in proteasomal degradation and other ubiquitin signaling pathways in health and disease.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53120" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53120" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Ubiquitination is a reversible, posttranslational modification involving the conjugation of a conserved 76 amino acid protein, ubiquitin (Ub), to substrate proteins for proteasomal targeting or the regulation of cell signaling.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Substrate proteins are generally ubiquitinated through an isopeptide bond between the C-terminal glycine residue of ubiquitin and the terminal nitrogen of lysine side chains.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> Substrate proteins can be mono- or poly-ubiquitinated, where ubiquitin chains are formed through isopeptide linkages with any one of the seven internal lysine surface residues or conjugation to the N-terminal amine group.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Deubiquitination, the removal of ubiquitin, is carried out by a family of deubiquitinase enzymes (DUBs).<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Ubiquitin specific proteases (USPs) are the largest subfamily of DUBs consisting of more than 50 cysteine proteases with diverse roles in ubiquitin homeostasis.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">USP5, also known as isopeptidase T or isoT, disassembles a variety of unanchored polyubiquitin chains (Lys6, Lys11, Lys27, Lys29, Lys33, Lys48, Lys63)<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> to recycle free monoubiquitin. USP5 prevents the accumulation of polyubiquitin chains, which would otherwise overwhelm the proteasome by competing with ubiquitinated proteins targeted for degradation.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The domain architecture of USP5 consists of a zinc finger ubiquitin-binding domain (ZnF-UBD) spanning residues 173–283, a functional catalytic domain, two ubiquitin associated domains (UBA), and a cryptic zinc finger ubiquitin-binding domain (nUBP).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The ZnF-UBD recognizes the C-terminal diglycine motif of ubiquitin,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and has been associated with allosteric modulation of USP5 activity,<a onclick="showRef(event, 'ref8 ref11 ref14'); return false;" href="javascript:void(0);" class="ref ref8 ref11 ref14">(8,11,14)</a> and alternatively, with substrate recognition and positioning.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p last">USP5 has been associated with tumorigenesis,<a onclick="showRef(event, 'ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18 ref19">(15−19)</a> neurodegeneration,<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a> and viral infections.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> In addition, USP5 plays a role in the regulation of ubiquitin levels in DNA damage repair<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and stress granules.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> According to CRISPR knockout screens against large panels of cancer cell lines, USP5 is essential to the survival of cells from most cancer types.<a onclick="showRef(event, 'ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref27 ref28 ref29">(27−29)</a> Inhibition of USP catalytic activity by small molecules has been reported for USP1, USP7, and USP14.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30−34)</a> Targeting the noncatalytic ZnF-UBD may be an alternative strategy to antagonize USP5 function, similar to the recent development of small molecule inhibitors against the noncatalytic domain of HDAC6.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> Here, we report the structure-based discovery of the first USP5 ZnF-UBD inhibitors and their characterization by <sup>19</sup>F NMR spectroscopy, surface plasmon resonance (SPR), and X-ray crystallography. Experimental validation of four low-affinity ligands followed by a hit expansion campaign defined a preliminary structure–activity relationship (SAR). Our work provides a framework for the development of potent USP5 ZnF-UBD inhibitors to investigate the function of this noncatalytic domain in cells.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05175" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05175" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Virtual Screen and Hit Identification</h3><div class="NLM_p">Following our observation that compounds featuring a short carboxylic acid chain can mimic the C-terminal diglycine carboxylate of ubiquitin and favorably exploit the ZnF-UBD of HDAC6,<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> we compiled a virtual library of 9480 in-house and commercially available compounds to screen against USP5 ZnF-UBD. The library was docked using ICM-Pro<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> (Molsoft, CA) and Glide<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (Schrodinger, NY) to the X-ray crystal structure conformation of USP5 ZnF-UBD (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45">2G45</a>) and an alternate conformational state modeled after the structure of HDAC6 ZnF-UBD in complex with inhibitors. In this alternate “stacked” conformation, the guanidinium plane of Arg221 is positioned parallel to the phenyl plane of Tyr259 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf" class="ext-link">Figure S1</a>), which would allow favorable π-stacking interaction with aromatic inhibitors. Ninety-six compounds were selected from ICM<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and Glide<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> by visual inspection of the docked binding pose and physical samples of 33 (mostly fragments) were tested experimentally. Since one of the two tryptophan side chains of USP5 ZnF-UBD is located at the targeted ubiquitin C-terminus binding pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), we decided to use a <sup>19</sup>F nuclear magnetic resonance (NMR) spectroscopy assay as a primary, qualitative screening method. We first verified that USP5 ZnF-UBD labeled with 5-fluorotryptophan (5FW) produced two distinct peaks in a <sup>19</sup>F spectrum. Titration of 5FW-USP5 ZnF-UBD with the C-terminal ubiquitin peptide LRLRGG resulted in the broadening of the downfield peak (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b), which we consequently assigned as Trp209 in the ubiquitin-binding site. Upon screening the 33 compounds, we found that 11 caused chemical shift perturbation and/or broadening of this <sup>19</sup>F resonance, suggesting binding at the ubiquitin-binding pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf" class="ext-link">Table S2</a>). We next used SPR as an orthogonal binding assay to validate primary hits (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). While the LRLRGG ubiquitin peptide bound USP5 ZnF-UBD with a <i>K</i><sub>D</sub> of 52 ± 2 μM, seven of the 11 NMR hits showed weak affinities in the high micromolar range (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Fragments <b>5</b> and <b>7</b> were the most potent ligands with a <i>K</i><sub>D</sub> of 220 ± 23, 170 ± 50 μM, and ligand efficiency<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (LE) of 0.33 and 0.30, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of USP5 ZnF-UBD interactions: (a) crystal structure of USP5 ZnF-UBD (grey) in complex with a C-terminal ubiquitin peptide (cyan) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45">2G45</a>). The Trp side chains of USP5 are highlighted. (b) <sup>19</sup>F NMR spectra of 5FW-USP5 ZnF-UBD before (blue) and after the addition of LRLRGG ubiquitin peptide (red) (at 6-fold excess) (c) <sup>19</sup>F NMR spectra of 5FW-USP5 ZnF-UBD before (blue) and after the addition of compound <b>5</b> (at 25-fold excess) (red) (d) SPR sensorgram and binding curve of LRLRGG ubiquitin peptide (left) and compound <b>5</b> (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SPR <i>K</i><sub>D</sub> and Ligand Efficiency of Preliminary Hit Compounds<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0003.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>D</sub> determination experiments, <i>n</i> ≥ 3, values are presented as mean ± SD; NB = no binding.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Ligand efficiency<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (in kcal mol<sup>–1</sup> atom<sup>–1</sup>) LE = (1.37 × p<i>K</i><sub>D</sub>)/HA; HA = heavy atom.</p></div></div><div></div></div><div class="NLM_p">To confirm that the screening hits bound at the C-terminal ubiquitin-binding pocket, and to guide future optimization efforts, we attempted soaking apo USP5 ZnF-UBD crystals with ligands but were unsuccessful. Cocrystallization attempts with the most potent compounds produced crystal structures with <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXT">6DXT</a>), <b>5</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT">6NFT</a>), and <b>7</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Compound <b>1</b> was not fully resolved by electron density and modeled for only one of the two USP5 ZnF-UBD molecules in the crystallographic asymmetric unit. The carboxylate group of the ligands recapitulates interactions observed with the ubiquitin C-terminal diglycine,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and is engaged in three direct or water-mediated hydrogen bonds with surrounding residues. An aromatic ring is engaged in π-stacking interactions with Tyr259, but we did not observe a conformational rearrangement of Arg221 that could have further increased stacking interactions, as seen with the corresponding HDAC6 ZnF-UBD arginine residue, Arg1155. The absence of π-stacking interactions may be due to the absence of an extended polyaromatic system in the ligand. The side chain of Arg221 is engaged in a hydrogen bond with the 1,3,4-oxadiazole ring of compound <b>1</b> and the carbonyl oxygen of compounds <b>5</b> and <b>7</b>. Interestingly, binding of compound <b>7</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>) induces the remodeling of a loop in the vicinity of the ligand (residues 221–230), which significantly opens up the binding pocket, and may offer a distinct vector for ligand optimization (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of USP5 ZnF-UBD in complex with (a) C-terminal ubiquitin peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45">2G45</a>) (b) <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXT">6DXT</a>), (c) <b>5</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT">6NFT</a>), and (d) <b>7</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>). Binding pocket residues (grey), ligands (cyan). Hydrogen bonds are shown as dotted green lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Superimposed crystal structure of USP5 ZnF-UBD (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G43">2G43</a>) (grey) in the apo state and bound to <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXT">6DXT</a>) (magenta), <b>5</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT">6NFT</a>) (green), and <b>7</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>) (blue) highlighting the shifted loop in the compound <b>7</b> complex structure. (b) Surface representation of USP5 ZnF-UBD (grey) in complex with <b>5</b> and <b>7</b> (purple) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT">6NFT</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Hit Expansion</h3><div class="NLM_p">A substructure search of commercial libraries using the chemical scaffolds of compounds <b>1</b>, <b>5</b>, and <b>7</b> found 399, 657 and 2450 analogues, respectively. The resulting libraries were docked to the USP5 ZnF-UBD crystallographic conformers corresponding to the respective ligand scaffold using Glide.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> After visual inspection 30, 14 and 39 analogues of <b>1</b>, <b>5</b>, and <b>7</b>, respectively, were selected and rescored using free energy perturbation (FEP) (Desmond Molecular Dynamics System; D.E. Shaw Research: New York, NY, 2018). Compounds were clustered based on their chemical similarity, and physical samples were obtained for a selection based on high docking score and low molecular weight. Twenty-two compounds were tested using an SPR assay (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Changes in the 1,3,4-oxadiazole dioxazole ring of <b>1</b>, which forms a critical hydrogen bond with the side chain of Arg221, were not tolerated (<b>12</b>, <b>13</b>, and <b>14</b>), and only analogs <b>18</b> and <b>19</b>, where a halogen atom decorates the phenyl ring, showed a minor improvement in potency, with <i>K</i><sub>D</sub> values of 280 and 270 μM, respectively. Among the eight analogues of <b>5</b> (LE = 0.33), two had improved potency; installing methoxy groups on the benzene ring (compound <b>22</b>) resulted in a <i>K</i><sub>D</sub> of 110 ± 10 μM, while adding a methyl group on the aliphatic carbon of the carboxylic tail (compound <b>21</b>) resulted in a <i>K</i><sub>D</sub> of 60 ± 8 μM and a good ligand efficiency of 0.36. Encouraged by this result, we subsequently ordered compounds <b>26</b>, <b>27</b>, <b>28</b>, and <b>29</b>. Comparing <b>21</b> with <b>28</b> and <b>29</b> reveals that the pocket where the methyl group in <b>21</b> binds is very tight and does not tolerate larger hydrophobic groups. We solved the crystal structure of <b>21</b> in complex with USP5 ZnF-UBD, revealing that the added methyl group optimally occupies a hydrophobic cavity lined by Trp209, and a crystal packing similar to <b>7</b> with a shift in the loop orientation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Finally, one analogue of compound <b>7</b> (<i>K</i><sub>D</sub> = 170 ± 50 μM) had an improved binding affinity over the parent molecule. Compound <b>37</b> resulted in a <i>K</i><sub>D</sub> of 80 ± 1 μM, suggesting an i-propyl at the para position of the benzene ring is preferred over a <i>t</i>-butyl moiety. The preliminary structure–activity relationship emerging from this work will serve as a framework for improvement of the current chemical series and exploration of novel ligand scaffolds.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of USP5 ZnF-UBD (grey) in complex with <b>21</b> (cyan). A methyl group optimally exploiting a buried hydrophobic cavity is highlighted (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P9G">6P9G</a>). The surface of the binding pocket is colored by binding properties: blue: hydrogen-bond donor, red: hydrogen-bond acceptor, grey: hydrophobic.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SPR <i>K</i><sub>D</sub> of Hit Expansion Compounds<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0004.gif" alt="" id="fx2" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0005.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0007.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>D</sub> determination experiments, <i>n</i> ≥ 2, values are presented as mean ± SD; NB = no binding.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Ligand efficiency<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (in kcal mol<sup>–1</sup> atom<sup>–1</sup>) LE = (1.37 × p<i>K</i><sub>D</sub>)/HA; HA = heavy atom.</p></div></div><div></div></div><div class="NLM_p">We used SPR to show that <b>21</b> binds to wild-type (WT) full-length (FL) USP5 with a <i>K</i><sub>D</sub> of 84 ± 37 μM, comparable to its affinity for the isolated ZnF-UBD (<i>K</i><sub>D</sub> = 60 ± 8 μM) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). A FL USP5 R221A<a onclick="showRef(event, 'ref11 ref40'); return false;" href="javascript:void(0);" class="ref ref11 ref40">(11,40)</a> mutant was used to attenuate binding at the ZnF-UBD site. <b>21</b> shows no detectable binding to the R221A FL USP5, indicating that it is indeed binding to the ZnF-UBD of FL USP5 (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Ub binds with a 100-fold stronger affinity for WT FL USP5 than R221A FL USP5 (<i>K</i><sub>D</sub> = 0.3 ± 0.1 versus 38 ± 3 μM respectively) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf" class="ext-link">Figure S2</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representative SPR sensorgram and binding curve of compound <b>21</b> with (a) USP5 ZnF-UBD (b) WT FL USP5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding Data to FL USP5<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">FL USP5<sup>WT</sup><i>K</i><sub>D</sub> (μM)</th><th class="colsep0 rowsep0" align="center">FL USP5<sup>R221A</sup><i>K</i><sub>D</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char="±">84 ± 37</td><td class="colsep0 rowsep0" align="left">NB</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>Ub</b></td><td class="colsep0 rowsep0" align="char" char="±">0.3 ± 0.1</td><td class="colsep0 rowsep0" align="left">38 ± 3</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i><sub>D</sub> determination experiments, <i>n</i> ≥ 2, values are presented as mean ± SD; NB = no binding.</p></div></div></div><div class="NLM_p last">Finally, we observed a dose-dependent inhibition of WT USP5 catalytic activity by compound <b>21</b> using a ubiquitin-rhodamine (Ub-Rho) DUB assay as described in Materials and Methods. The maximum inhibition achieved by <b>21</b> was ∼75% (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf" class="ext-link">Figure S3</a>). This is in agreement with our observation that catalytic activity of R221A ZnF-UBD USP5 mutant is only 40% of the WT and consistent with previous reports suggesting that binding of ubiquitin to the ZnF-UBD enhances USP5 catalytic activity from a basal to a fully active state.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Future USP5 characterization with native polyubiquitin substrates will clarify whether ZnF-UBD ligands antagonize the cleavage of some polyubiquitin substrates over others.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20274" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20274" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Here, we report a combination of computational screens, binding assays, and crystal structures for the discovery of ligands targeting the ZnF-UBD of USP5. This work will serve as a platform to optimize the chemical series presented here and potentially identify novel scaffolds. Our aim is to develop a chemical probe that engages USP5 in a cell-based assay at one micromolar or less, that is selective over ZnF-UBDs found in other USPs and HDAC6, and that can be used to investigate the contribution of this domain towards the substrate selectivity and function of USP5.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82084" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82084" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Primary Virtual Screening</h3><div class="NLM_p last">Ligprep (Schrödinger Release 2018-2: LigPrep, Schrödinger, LLC, New York, NY, 2018) and ICM<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> were used to prepare three-dimensional models of 7680 compounds from commercial vendors and 1800 “in-house” compounds. Protonation states at pH 7.4 ± 0.2 were generated with EpiK<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> (Schrödinger Release 2018-2: EpiK, Schrödinger, LLC, New York, NY, 2018). The library compounds were docked using either ICM-Pro<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> or Glide SP<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (Schrödinger Release 2018-2: Glide, Schrödinger, LLC, New York, NY, 2018) to each of three representations of the USP5 ZnF-UBD: (1) a crystal conformer based on PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45">2G45</a><a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> including water molecule W404, which forms hydrogen-bonds with residues Arg221 and Asn207, referred to as “+W404”; (2) the same crystal conformer without that water molecule, “–W404”; (3) a simulated stacked receptor, obtained by remodeling the side-chain conformations for Arg221 and Tyr223 based on the stacked conformation of HDAC6 ZnF-UBD in complex with various inhibitors. After ICM docking with +W404, −W404, and stacked receptor representations, the 138, 89, and 139 best-scoring compounds were visually inspected, respectively; 28 compounds occurred in all three of the ICM best-scoring selections. After docking with the +404, −404, and stacked receptor representations, the 97, 103, and 162 best-scoring compounds were visually inspected, respectively. Thirty-six compounds occurred in all three of the Glide best-scoring selections. Ninety-six compounds were selected during visual inspection based on the subjective plausibility of the predicted binding pose and properties of compound chemical scaffolds. Thirty-three compounds were ordered.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Cloning, Protein Expression, and Purification</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">DNA encoding Ub<sup>1–76</sup>, USP5<sup>171–290</sup>, FL USP5<sup>WT 1-835</sup>, and FL USP5<sup>R221A 1-835</sup> was subcloned into a modified pET28 vector encoding a TEV cleavable N-terminal His6-tag (pET28-MHL) and a modified vector with an N-terminal AviTag for biotinylation and C-terminal His6-tag (p28BIOH-LIC) using a ligation-independent InFusion cloning kit (ClonTech). Site-directed mutagenesis was used to clone FL USP5<sup>R221A 1-835</sup> and was verified by DNA sequencing. Proteins were expressed in BL21 (DE3) Codon Plus RIL <i>Escherichia coli</i>. Cultures were grown in M9 minimal media, supplemented with 50 μM ZnSO<sub>4</sub> for pET28-MHL and 10 μg/mL biotin for USP5<sup>171–290</sup> p28BIOH-LIC expression. Cultures were grown in TB media (Sigma-Aldrich) supplemented with 50 μM ZnSO<sub>4</sub> for pET28-MHL and 10 μg/mL biotin for Ub<sup>1–76</sup>, FL USP5<sup>WT 1-835</sup>, FL USP5<sup>R221A 1-835</sup>, and p28BIOH-LIC expression. For USP5<sup>171–290</sup> (pET28-MHL) with 5-fluorotryptophan (5FW) labels (Sigma-Aldrich), cultures were grown in defined media.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Expression cultures were induced at OD<sub>600</sub> ∼0.6 with 0.5 mM IPTG overnight at 15 °C. Proteins were purified using nickel–nitriloacetic acid (Ni–NTA) agarose resin, and the tag was removed by TEV for USP5<sup>171–290</sup> pET28-MHL. Uncleaved proteins and TEV were removed by a second pass with Ni–NTA resin (Qiagen). Proteins were concentrated and further purified using gel filtration (S75 16/60, S200 16/60, GE Healthcare) and ion-exchange (Hi-Load Q column, GE Healthcare) for the 5FW-labeled protein. The incorporation of the 5FW label was verified by mass spectrometry. The final concentration of purified proteins was 7–30 mg/mL as measured by UV absorbance at 280 nm.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Crystallization</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78823" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78823" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Crystallization solutions of 12 mg/mL of tag-free USP5<sup>171–290</sup> (pET28-MHL) and 1% (v/v) and 4.5% (v/v) of a 200 mM DMSO-solubilized stock of compounds were prepared. Sparse-matrix crystallization experiments yielded crystals in different conditions for each USP5 ZnF-UBD-compound complex. 1:2.5 USP5<sup>171–290</sup>: compound <b>1</b> was cocrystallized with 1:1 protein/compound: mother liquor comprising 1.5 M ammonium sulfate, 0.1 M bis-tris pH 7.0. 1:2.5 USP5<sup>171–290</sup>: compound <b>5</b> was cocrystallized with 1:1 protein/compound: mother liquor comprising 2 M sodium/potassium phosphate pH 7.0. 1:5 USP5<sup>171–290</sup>: compound <b>7</b> was cocrystallized with 1:1 protein/compound: mother liquor comprising 1.5 M ammonium sulfate, 0.1 M bis-tris pH 7.0. 1:2.5 USP5<sup>171–290</sup>: compound <b>21</b> was cocrystallized with 1:1 protein/compound: mother liquor comprising 1.75 M ammonium sulfate, 0.2 M sodium acetate, 0.1 M sodium cacodylate pH 6.8. All cocrystals were cryo-protected with mother liquor supplemented with 25% ethylene glycol (v/v) prior to mounting and cryo-cooling.</div><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Data Collection, Structure Determination, and Refinement</h3><div class="NLM_p last">X-ray diffraction data for USP5<sup>171–290</sup> cocrystals with <b>1</b>, <b>5</b>, <b>7</b>, and <b>21</b> were collected at 100 K at a Rigaku FR-E Superbright copper source. Diffraction data were processed with the XDS<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and AIMLESS,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> using the Xia2 interfaces<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> for complex <b>1</b>, <b>5</b>, and <b>7</b>. Structures for compounds <b>1</b> and <b>5</b> were solved by the direct refinement of protein chains A and B of the isomorphous PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G43">2G43</a>. The complex structure for compound <b>7</b> was solved by molecular replacement with Phaser<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and coordinates from chain A of PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G43">2G43</a>. The cocrystal structure for compound <b>21</b> was solved by the direct refinement of coordinates from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a> with the DIMPLE<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> script. Geometry restraints for <b>21</b> were prepared with ACEDRG.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Models were refined with cycles of COOT,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> for model building and visualization, REFMAC,<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> for restrained refinement, and model geometry analyzed with MolProbity.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> <sup>19</sup>F NMR Spectroscopy</h3><div class="NLM_p last">5FW-USP5 USP5<sup>171–290</sup> (at 25 μM) was buffered in 50 mM Tris pH 8, 150 mM NaCl, 1 mM TCEP, 0.005% Tween-20 (v/v), 5% D<sub>2</sub>O (v/v), and 0.5% DMSO (v/v). For the binding screen, each compound was assayed individually (at 25:1 compound: protein). <sup>19</sup>F spectra were acquired on a Bruker 600 MHz spectrometer equipped with a QCI cryoprobe. Each spectrum was acquired with 700–800 scans, an acquisition time of 90 ms, a recycle delay of 0.7 s, and processed by applying an exponential window function (LB = 10). The spectra were analyzed using software TopSpin (Bruker).</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Surface Plasmon Resonance</h3><div class="NLM_p last">Studies were performed using a Biacore T200 (GE Health Sciences). Approximately 4000–6000 response units (RU) of biotinylated USP5<sup>171–290</sup>, FL USP5<sup>WT 1-835</sup>, and FL USP5<sup>R221A 1-835</sup> was coupled to flow cells of a SA chip per manufacturer’s protocol, and an empty flow cell used for reference subtraction. Two-fold serial dilutions were prepared in 20 mM Hepes pH 7.4, 150 mM NaCl, 0.005% Tween-20 (v/v), and 1% DMSO (v/v). <i>K</i><sub>D</sub> determination experiments were performed using multicycle kinetics with 35–60 s contact time, and 30 μL/min flow rate at 20 °C. <i>K</i><sub>D</sub> values were calculated using steady-state affinity fitting with the Biacore T200 evaluation software.</div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Substructure Search</h3><div class="NLM_p last">Substructure search was run against ∼45 million commercial compounds from Molport (MolPort, version 2.92, 2019) using SMARTS strings generated in MarvinSketch (ChemAxon), with constraints on all aromatic rings and the carboxylate moiety. Similarly, search against ∼6 million commercial compounds from Emolecules (Emolecules Inc, 2019), was run using SMILES with a similarity index of 0.8.</div></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Virtual Hit Expansion</h3><div class="NLM_p last">LigPrep (Schrödinger Release 2018-2: LigPrep, Schrödinger, LLC, New York, NY, 2018) was used to prepare the ligands using default settings. The library designed by the substructure search was docked to the USP5 ZnF-UBD structure for each respective complex (<b>1</b>, <b>5</b>, <b>7</b>) using Glide<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (Schrödinger Release 2018-2: Glide, Schrödinger, LLC, New York, NY, 2018). Three hydrogen bond constraints between the carboxylate moiety of the ligand and NH of Arg221 backbone, NH of Arg221 side chain, and OH of Tyr261. Docked compounds were clustered in ICM<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and selected based on docking scores, complexity, and visual inspection of docked poses.</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Free Energy Perturbation</h3><div class="NLM_p last">All FEP calculations were conducted with the academic LigandFEP methodology of Desmond (Desmond Molecular Dynamics System; D. E. Shaw Research: New York, NY, 2017);<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> using the OPLS_2005 force field.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> The systems were solvated in an orthogonal box of SPC water molecules with a buffer width of 5 Å for the complex and 10 Å for the solvent simulations. The full systems were relaxed and equilibrated using the default Desmond protocol, consisting of: (i) a minimization using a Brownian dynamics NVT integrator for 100 ps with the solute molecules restrained (50 kcal/mol/Å<sup>2</sup>), (ii) 12 ps simulation in the NVT ensemble, keeping the restraints and temperature at 10 K, (iii) 12 ps simulation in the NPT ensemble, keeping restraints and temperature at 10 K, (iv) 24 ps simulation in the NPT simulation with solute heavy atom restraints at 300 K, and (v) 240 ps simulation in the NPT ensemble at 300 K without restraints. Production simulations in the NPT ensemble lasted 5 ns for both the complex and the solvent systems. A total of 12 λ windows were used for all calculations. The free energy differences between the initial and final states were calculated using the Bennett acceptance ratio (BAR) method,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and the errors estimated using bootstrapping.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a></div></div><div id="sec6_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Liquid Chromatography–Mass Spectrometry (LCMS)</h3><div class="NLM_p last">Chromatographic analyses were carried out on an ACQUITY UPLC BEH C18 (2.1 × 50 mm<sup>2</sup>, 1.7 μm) column. The mobile phase was 0.1% formic acid in water (solvent A) and 0.1% formic acid in acetonitrile (solvent B) at a flow rate of 0.4 mL/min. A gradient starting at 95% solvent A going to 5% in 4.5 min, holding for 0.5 min, going back to 95% in 0.5 min and equilibrating the column for 1 min was employed. A Waters Xevo QTof or a Waters SYNAPT G2-S MS equipped with an atmospheric pressure ionization source was used for MS analysis. MassLynx 4.1 was used for data analysis. All compounds have ≥95% purity, with exceptions for compounds <b>7</b> (92%), <b>9</b> (93%), and <b>12</b> (93%).</div></div><div id="sec6_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Ubiquitin-Rhodamine110 Deubiquitinase Assay</h3><div class="NLM_p last">Experiments were performed in a total volume of 60 μL in 384-well black polypropylene microplates (Grenier). Fluorescence was measured using a Biotek Synergy H1 microplate reader (Biotek) at excitation and emission wavelengths of 485 and 528 nm, respectively. Ligands were prepared in 20 mM Tris pH 7.5, 30 mM NaCl, 1 mM DTT, 0.01% Triton-X (v/v), and 1% DMSO (v/v) for 2-fold dilution titration series. 1 nM USP5<sup>1–835</sup> and 200 nM ubiquitin-rhodamine110<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> (UBPBio) were added to each well. Following 1 min centrifugation at 250 g, fluorescence readings were immediately taken for 10 min. The data were analyzed with GraphPad Prism 8.2.0.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00988" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00988" class="ext-link">10.1021/acs.jmedchem.9b00988</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">X-ray crystallography data collection and refinement statistics and NMR screening data and UPLC MS data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXT (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_003.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXT-sf (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_004.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6NFT (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_005.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6NFT-sf (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_006.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXH (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_007.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXH-sf (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_008.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6P9G (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_009.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6P9G-sf (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_010.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">3D PDB file used for preliminary docking studies “+W404” (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_011.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">3D PDB file used for preliminary docking studies “–W404” (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_012.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">3D PDB file used for preliminary docking studies “stacked” (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_013.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">3D PDB file used for virtual screen compound <b>1</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_014.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">3D PDB file used for virtual screen compound <b>5</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_015.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">3D PDB file used for virtual screen compound <b>7</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_016.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf">jm9b00988_si_001.pdf (2.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_002.csv">jm9b00988_si_002.csv (1.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_003.cif">jm9b00988_si_003.cif (140.07 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_004.cif">jm9b00988_si_004.cif (1.68 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_005.cif">jm9b00988_si_005.cif (237.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_006.cif">jm9b00988_si_006.cif (2.82 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_007.cif">jm9b00988_si_007.cif (263.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_008.cif">jm9b00988_si_008.cif (4.63 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_009.cif">jm9b00988_si_009.cif (181.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_010.cif">jm9b00988_si_010.cif (7.5 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_011.pdb">jm9b00988_si_011.pdb (237.07 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_012.pdb">jm9b00988_si_012.pdb (141.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_013.pdb">jm9b00988_si_013.pdb (236.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_014.pdb">jm9b00988_si_014.pdb (159.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_015.pdb">jm9b00988_si_015.pdb (167.39 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_016.pdb">jm9b00988_si_016.pdb (160.53 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p><b>1</b> (6DXT), <b>5</b> (6NFT), <b>7</b> (6DXH), <b>21</b> (6P9G)</p><p class="last">Authors will release atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00988" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17632" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17632" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachel J. Harding</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1134-391X" title="Orcid link">http://orcid.org/0000-0002-1134-391X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5f2d3e3c373a3371373e2d3b3631381f2a2b302d30312b30713c3e"><span class="__cf_email__" data-cfemail="5d2f3c3e35383173353c2f3934333a1d2829322f32332932733e3c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthieu Schapira</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1047-3309" title="Orcid link">http://orcid.org/0000-0002-1047-3309</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1d707c696975747868336e7e757c6d746f7c5d6869726f72736972337e7c"><span class="__cf_email__" data-cfemail="05686471716d6c60702b76666d64756c77644570716a776a6b716a2b6664">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mandeep K. Mann</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8252-6107" title="Orcid link">http://orcid.org/0000-0002-8252-6107</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ivan Franzoni</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, University of Toronto, 80 St. George St., Toronto, Ontario M5S 3H6, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8110-6218" title="Orcid link">http://orcid.org/0000-0001-8110-6218</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renato Ferreira de Freitas</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wolfram Tempel</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott Houliston</span> - <span class="hlFld-Affiliation affiliation">University Health Network, 661 University Avenue, Toronto, Ontario M5G 2C4, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Leanna Smith</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural Genomics
Consortium, University of Toronto, MaRS Centre, South Tower, 101 College St., Suite
700, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">University Health Network, 661 University Avenue, Toronto, Ontario M5G 2C4, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4971-3250" title="Orcid link">http://orcid.org/0000-0002-4971-3250</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.K.M. and M.S. wrote the manuscript. I.F. and R.F.d.F. completed preliminary docking studies. R.J.H. and M.K.M. performed the crystallographic calculations and model refinement for X-ray crystal structures with 1, 5, and 7. W.T. performed the crystallographic calculations and model refinement of the X-ray structure with compound 21. L.S. developed and optimized the Ub-Rho catalytic activity assay. M.K.M. and S.H. tested compounds. M.K.M. completed hit expansion. C.H.A, M.V., R.J.H., and M.S. advised throughout the project.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e1345-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35734" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35734" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The SGC is a registered charity (number 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada through Ontario Genomics Institute [OGI-055], Innovative Medicines Initiative (EU/EFPIA) [ULTRA-DD grant no. 115766], Janssen, Merck KGaA, Darmstadt, Germany, MSD, Novartis Pharma AG, Innovation and Science (MRIS), Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome. M.S. gratefully acknowledges support from NSERC [Grant RGPIN-2019-04416]. C.H.A. gratefully acknowledges support from NSERC [RGPIN-2015-05939]. M.M. is supported by an NSERC Collaborative Research and Training Experience grant [432008-2013] and a CQDM Quantum Leap Award. R.J.H. is the recipient of the Huntington’s Disease Society of America Berman Topper Career Development Fellowship. We are grateful to D.E. Shaw Research for providing us with an academic license for their Desmond. We thank Peter Loppnau for the cloning of USP5 protein constructs and Peter Brown and Carlos Zepeda-Velazquez for analytical LCMS of compounds.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BAR</td><td class="NLM_def"><p class="first last">Bennett acceptance ratio</p></td></tr><tr><td class="NLM_term">DUB</td><td class="NLM_def"><p class="first last">deubiquitinase</p></td></tr><tr><td class="NLM_term">FEP</td><td class="NLM_def"><p class="first last">free energy perturbation</p></td></tr><tr><td class="NLM_term">FL</td><td class="NLM_def"><p class="first last">full-length</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">dissociation constant</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure activity relationship</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">Ub</td><td class="NLM_def"><p class="first last">ubiquitin</p></td></tr><tr><td class="NLM_term">UBA</td><td class="NLM_def"><p class="first last">ubiquitin associated domain</p></td></tr><tr><td class="NLM_term">UBP</td><td class="NLM_def"><p class="first last">ubiquitin-binding domain</p></td></tr><tr><td class="NLM_term">Ub-Rho</td><td class="NLM_def"><p class="first last">ubiquitin-rhodamine</p></td></tr><tr><td class="NLM_term">USP</td><td class="NLM_def"><p class="first last">ubiquitin specific protease</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild-type</p></td></tr><tr><td class="NLM_term">ZnF-UBD</td><td class="NLM_def"><p class="first last">zinc finger ubiquitin-binding domain</p></td></tr><tr><td class="NLM_term">5FW</td><td class="NLM_def"><p class="first last">5-fluorotryptophan</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08450" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08450" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varshavsky, A.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin System</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1016/S0968-0004(97)01122-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2FS0968-0004%2897%2901122-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=9357313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK2sXntFeltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1997&pages=383-387&author=A.+Varshavsky&title=The+Ubiquitin+System&doi=10.1016%2FS0968-0004%2897%2901122-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin system</span></div><div class="casAuthors">Varshavsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">383-387</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 70 refs.  Eukaryotes contain a highly conserved multi-enzyme system that covalently links ubiquitin to a variety of intracellular proteins that bear degrdn. signals recognized by this system.  The resulting ubiquitin-protein conjugates are degraded by the 26S proteasome, a large ATP-dependent protease.  Pathways that involve ubiquitin underlie a multitude of processes, including cell differentiation, the cell cycle and responses to stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALosaVY0BRbVg90H21EOLACvtfcHk0linN63ZeMLtgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntFeltLY%253D&md5=61205ba96af5c0d89c028120d609cc52</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2897%2901122-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252897%252901122-5%26sid%3Dliteratum%253Aachs%26aulast%3DVarshavsky%26aufirst%3DA.%26atitle%3DThe%2520Ubiquitin%2520System%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1997%26volume%3D22%26spage%3D383%26epage%3D387%26doi%3D10.1016%2FS0968-0004%2897%2901122-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Ubiquitin-Specific Proteases as a Novel Anticancer Therapeutic Strategy</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1080</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.01080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.3389%2Ffphar.2018.01080" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=1080&author=T.+Yuanauthor=F.+Yanauthor=M.+Yingauthor=J.+Caoauthor=Q.+Heauthor=H.+Zhuauthor=B.+Yang&title=Inhibition+of+Ubiquitin-Specific+Proteases+as+a+Novel+Anticancer+Therapeutic+Strategy&doi=10.3389%2Ffphar.2018.01080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01080%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DYing%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520Ubiquitin-Specific%2520Proteases%2520as%2520a%2520Novel%2520Anticancer%2520Therapeutic%2520Strategy%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D27%26spage%3D1080%26doi%3D10.3389%2Ffphar.2018.01080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Turcu, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ventii, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span> <span> </span><span class="NLM_article-title">Regulation and Cellular Roles of Ubiquitin-Specific Deubiquitinating Enzymes</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">397</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.78.082307.091526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1146%2Fannurev.biochem.78.082307.091526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=19489724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1Ghtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2009&pages=363-397&author=F.+E.+Reyes-Turcuauthor=K.+H.+Ventiiauthor=K.+D.+Wilkinson&title=Regulation+and+Cellular+Roles+of+Ubiquitin-Specific+Deubiquitinating+Enzymes&doi=10.1146%2Fannurev.biochem.78.082307.091526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and cellular roles of ubiquitin-specific deubiquitinating enzymes</span></div><div class="casAuthors">Reyes-Turcu, Francisca E.; Ventii, Karen H.; Wilkinson, Keith D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">363-397</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Deubiquitinating enzymes (DUBs; deubiquitinases) are proteases that process ubiquitin or ubiquitin-like gene products, reverse the modification of proteins by a single ubiquitin(-like) protein, and remodel polyubiquitin(-like) chains on target proteins.  The human genome encodes nearly 100 DUBs with specificity for ubiquitin in 5 gene families.  Most DUB activity is cryptic, and conformational rearrangements often occur during the binding of ubiquitin and/or scaffold proteins.  DUBs with specificity for ubiquitin contain insertions and extensions modulating DUB substrate specificity, protein-protein interactions, and cellular localization.  Binding partners and multiprotein complexes with which DUBs assoc. modulate DUB activity and substrate specificity.  Quant. studies of activity and protein-protein interactions, together with genetic studies and the advent of RNAi, have led to new insights into the function of yeast and human DUBs.  Here, the authors discuss ubiquitin-specific DUBs, some of the generalizations emerging from recent studies of the regulation of DUB activity, and their roles in various cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzNm7SqfgcRLVg90H21EOLACvtfcHk0lh7PitjCZ2u0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1Ghtbc%253D&md5=7e7248a63b46c5dbc7cf333d8d406d90</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.78.082307.091526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.78.082307.091526%26sid%3Dliteratum%253Aachs%26aulast%3DReyes-Turcu%26aufirst%3DF.%2BE.%26aulast%3DVentii%26aufirst%3DK.%2BH.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26atitle%3DRegulation%2520and%2520Cellular%2520Roles%2520of%2520Ubiquitin-Specific%2520Deubiquitinating%2520Enzymes%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2009%26volume%3D78%26spage%3D363%26epage%3D397%26doi%3D10.1146%2Fannurev.biochem.78.082307.091526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clague, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heride, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbé, S.</span></span> <span> </span><span class="NLM_article-title">The Demographics of the Ubiquitin System</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2015.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.tcb.2015.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=25906909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVGqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=417-426&author=M.+J.+Clagueauthor=C.+Herideauthor=S.+Urb%C3%A9&title=The+Demographics+of+the+Ubiquitin+System&doi=10.1016%2Fj.tcb.2015.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The demographics of the ubiquitin system</span></div><div class="casAuthors">Clague, Michael J.; Heride, Claire; Urbe, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">417-426</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin system is a major coordinator of cellular physiol. through regulation of both protein degrdn. and signalling pathways.  A key building block of a systems-level understanding has been generated by global proteomic studies, which provide copy no. ests. for each component.  The aggregate of ubiquitin, conjugating enzymes (E1, E2, and E3s), and deubiquitylases (DUBs) represents ∼1.3% of total cellular protein.  Complementary approaches have generated quant. measurements of various ubiquitin pools and further subdivision into different ubiquitin chain topologies.  Systematic studies aimed at assocg. specific enzymes (E2s and DUBs) with the dynamics of these different pools have also made significant progress.  Here, we delineate the emerging picture of the most significant determinants of the cellular ubiquitin economy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQMwzCvXv5bVg90H21EOLACvtfcHk0lh7PitjCZ2u0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVGqtbw%253D&md5=38563c26b5d83eab2bb0dc193041d1e5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2015.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2015.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DHeride%26aufirst%3DC.%26aulast%3DUrb%25C3%25A9%26aufirst%3DS.%26atitle%3DThe%2520Demographics%2520of%2520the%2520Ubiquitin%2520System%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2015%26volume%3D25%26spage%3D417%26epage%3D426%26doi%3D10.1016%2Fj.tcb.2015.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swatek, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin Modifications</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">399</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1038/cr.2016.39</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fcr.2016.39" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=27012465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=399-422&author=K.+N.+Swatekauthor=D.+Komander&title=Ubiquitin+Modifications&doi=10.1038%2Fcr.2016.39"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin modifications</span></div><div class="casAuthors">Swatek, Kirby N.; Komander, David</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">399-422</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination is a dynamic multifaceted post-translational modification involved in nearly all aspects of eukaryotic biol.  Once attached to a substrate, the 76-amino acid protein ubiquitin is subjected to further modifications, creating a multitude of distinct signals with distinct cellular outcomes, referred to as the 'ubiquitin code'.  Ubiquitin can be ubiquitinated on seven lysine (Lys) residues or on the N-terminus, leading to polyubiquitin chains that can encompass complex topologies.  Alternatively or in addn., ubiquitin Lys residues can be modified by ubiquitin-like mols. (such as SUMO or NEDD8).  Finally, ubiquitin can also be acetylated on Lys, or phosphorylated on Ser, Thr or Tyr residues, and each modification has the potential to dramatically alter the signaling outcome.  While the no. of distinctly modified ubiquitin species in cells is mind-boggling, much progress has been made to characterize the roles of distinct ubiquitin modifications, and many enzymes and receptors have been identified that create, recognize or remove these ubiquitin modifications.  We here provide an overview of the various ubiquitin modifications present in cells, and highlight recent progress on ubiquitin chain biol.  We then discuss the recent findings in the field of ubiquitin acetylation and phosphorylation, with a focus on Ser65-phosphorylation and its role in mitophagy and Parkin activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSk-y3UHKix7Vg90H21EOLACvtfcHk0lh7PitjCZ2u0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGmsbg%253D&md5=0a4538fe56b01c700e8392f397084db8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcr.2016.39&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2016.39%26sid%3Dliteratum%253Aachs%26aulast%3DSwatek%26aufirst%3DK.%2BN.%26aulast%3DKomander%26aufirst%3DD.%26atitle%3DUbiquitin%2520Modifications%26jtitle%3DCell%2520Res.%26date%3D2016%26spage%3D399%26epage%3D422%26doi%3D10.1038%2Fcr.2016.39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rape, M.</span></span> <span> </span><span class="NLM_article-title">The Increasing Complexity of the Ubiquitin Code</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1038/ncb3358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fncb3358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=27230526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC28Xosl2it7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=579-586&author=R.+Yauauthor=M.+Rape&title=The+Increasing+Complexity+of+the+Ubiquitin+Code&doi=10.1038%2Fncb3358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The increasing complexity of the ubiquitin code</span></div><div class="casAuthors">Yau, Richard; Rape, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">579-586</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ubiquitylation is essential for signal transduction as well as cell division and differentiation in all eukaryotes.  Substrate modifications range from a single ubiquitin mol. to complex polymeric chains, with different types of ubiquitylation often eliciting distinct outcomes.  The recent identification of novel chain topologies has improved our understanding of how ubiquitylation establishes precise communication within cells.  Here, we discuss how the increasing complexity of ubiquitylation is employed to ensure robust and faithful signal transduction in eukaryotic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod2OUBSjtqwLVg90H21EOLACvtfcHk0lh7PitjCZ2u0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xosl2it7k%253D&md5=2223ea6ecf6d7fa869b9ee064444ff8b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncb3358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb3358%26sid%3Dliteratum%253Aachs%26aulast%3DYau%26aufirst%3DR.%26aulast%3DRape%26aufirst%3DM.%26atitle%3DThe%2520Increasing%2520Complexity%2520of%2520the%2520Ubiquitin%2520Code%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2016%26volume%3D18%26spage%3D579%26epage%3D586%26doi%3D10.1038%2Fncb3358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walczak, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">Generation and Physiological Roles of Linear Ubiquitin Chains</span>. <i>BMC Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">23</span>, <span class="refDoi"> DOI: 10.1186/1741-7007-10-23</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1186%2F1741-7007-10-23" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=22420778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlsl2rs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=23&author=H.+Walczakauthor=K.+Iwaiauthor=I.+Dikic&title=Generation+and+Physiological+Roles+of+Linear+Ubiquitin+Chains&doi=10.1186%2F1741-7007-10-23"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Generation and physiological roles of linear ubiquitin chains</span></div><div class="casAuthors">Walczak, Henning; Iwai, Kazuhiro; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">BMC Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23</span>CODEN:
                <span class="NLM_cas:coden">BBMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1741-7007</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Ubiquitination now ranks with phosphorylation as one of the best-studied post-translational modifications of proteins with broad regulatory roles across all of biol.  Ubiquitination usually involves the addn. of ubiquitin chains to target protein mols., and these may be of eight different types, seven of which involve the linkage of one of the seven internal lysine (K) residues in one ubiquitin mol. to the carboxy-terminal diglycine of the next.  In the eighth, the so-called linear ubiquitin chains, the linkage is between the amino-terminal amino group of methionine on a ubiquitin that is conjugated with a target protein and the carboxy-terminal carboxy group of the incoming ubiquitin.  Physiol. roles are well established for K48-linked chains, which are essential for signaling proteasomal degrdn. of proteins, and for K63-linked chains, which play a part in recruitment of DNA repair enzymes, cell signaling and endocytosis.  We focus here on linear ubiquitin chains, how they are assembled, and how three different avenues of research have indicated physiol. roles for linear ubiquitination in innate and adaptive immunity and suppression of inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUfVgvYr8WNrVg90H21EOLACvtfcHk0liuCgCoPjukTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlsl2rs7g%253D&md5=d6c6d5add1b55eeda621aff62f16c44d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1741-7007-10-23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1741-7007-10-23%26sid%3Dliteratum%253Aachs%26aulast%3DWalczak%26aufirst%3DH.%26aulast%3DIwai%26aufirst%3DK.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DGeneration%2520and%2520Physiological%2520Roles%2520of%2520Linear%2520Ubiquitin%2520Chains%26jtitle%3DBMC%2520Biol.%26date%3D2012%26volume%3D10%26spage%3D23%26doi%3D10.1186%2F1741-7007-10-23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tashayev, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickart, C. M.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the Polyubiquitin Degradation Signal: Structure, Mechanism, and Role of Isopeptidase T</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">14535</span>– <span class="NLM_lpage">14546</span>, <span class="refDoi"> DOI: 10.1021/bi00044a032</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00044a032" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK2MXoslGhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=14535-14546&author=K.+D.+Wilkinsonauthor=V.+L.+Tashayevauthor=L.+B.+O%E2%80%99Connorauthor=C.+N.+Larsenauthor=E.+Kasperekauthor=C.+M.+Pickart&title=Metabolism+of+the+Polyubiquitin+Degradation+Signal%3A+Structure%2C+Mechanism%2C+and+Role+of+Isopeptidase+T&doi=10.1021%2Fbi00044a032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the polyubiquitin degradation signal: structure, mechanism, and role of isopeptidase T</span></div><div class="casAuthors">Wilkinson, Keith D.; Tashayev, Valery L.; O'Connor, Lydia B.; Larsen, Christopher N.; Kasperek, Eileen; Pickart, Cecile M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14535-46</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A necessary step in ubiquitin-dependent proteolysis is the addn. of a polyubiquitin chain to the target protein.  This ubiquitinated protein is degraded by a multi-subunit complex known as the 26S proteasome.  The polyubiquitin chain is probably not released until a late stage in the proteolysis by the proteasome.  It is subsequently disassembled to yield functional ubiquitin monomers.  Here the authors present evidence that a 93 kDa protein, isopeptidase T, has the properties expected for the enzyme which disassembles these branched polyubiquitin chains.  Protein and cDNA sequencing revealed that isopeptidase T is a member of the ubiquitin specific protease family (UBP).  Isopeptidase T disassembles branched polyubiquitin chains (linked by the G76-K48 isopeptide bond) by a sequential exo mechanism, starting at the proximal end of the chain (the proximal ubiquitin contains a free carboxyl-terminus).  Isopeptidase T prefers to disassemble chains in which there is an intact and unblocked RGG sequence at the C-terminus of the proximal subunit.  Rates of disassembly are reduced when G76 of the proximal ubiquitin is modified, for example, by ligation to substrate protein, by esterification, by replacement of the proximal glycine with alanine (G76A), or by truncation.  Linear proubiquitin is only a poor substrate.  Obsd. rates and specificity are consistent with isopeptidase T playing a major role in disassembly of polyubiquitin chains.  The high discrimination against chains that are blocked or modified at the proximal end indicates that the enzyme acts after release of the chains from conjugated proteins or degrdn. intermediates.  Thus, the proteolytic degrdn. signal is not disassembled by isopeptidase T before the ubiquitinated protein is degraded.  These (and earlier) results suggest that UBP isoenzymes may exhibit significant substrate specificity, consistent with a role in the regulated catabolism of the polymeric ubiquitin, including the polyubiquitin protein degrdn. signal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWWP5m1sxSw7Vg90H21EOLACvtfcHk0liuCgCoPjukTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXoslGhs7s%253D&md5=cdad0da4f8b78836524cc8656a360378</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi00044a032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00044a032%26sid%3Dliteratum%253Aachs%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26aulast%3DTashayev%26aufirst%3DV.%2BL.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DL.%2BB.%26aulast%3DLarsen%26aufirst%3DC.%2BN.%26aulast%3DKasperek%26aufirst%3DE.%26aulast%3DPickart%26aufirst%3DC.%2BM.%26atitle%3DMetabolism%2520of%2520the%2520Polyubiquitin%2520Degradation%2520Signal%253A%2520Structure%252C%2520Mechanism%252C%2520and%2520Role%2520of%2520Isopeptidase%2520T%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D14535%26epage%3D14546%26doi%3D10.1021%2Fbi00044a032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welchman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, R. J.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin and Ubiquitin-like Proteins as Multifunctional Signals</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1038/nrm1700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnrm1700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=16064136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=599-609&author=R.+L.+Welchmanauthor=C.+Gordonauthor=R.+J.+Mayer&title=Ubiquitin+and+Ubiquitin-like+Proteins+as+Multifunctional+Signals&doi=10.1038%2Fnrm1700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin and ubiquitin-like proteins as multifunctional signals</span></div><div class="casAuthors">Welchman, Rebecca L.; Gordon, Colin; Mayer, R. John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">599-609</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination is a recognized signal for protein degrdn.  However, it is increasingly realized that ubiquitin conjugation to proteins can be used for many other purposes.  Furthermore, there are many ubiquitin-like proteins that control the activities of proteins.  The central structural element of these post-translational modifications is the ubiquitin superfold.  A common ancestor based on this superfold has evolved to give various proteins that are involved in diverse activities in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr72A1XAR84W7Vg90H21EOLACvtfcHk0liuCgCoPjukTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D&md5=25f0eb2aec7ffb99b78034da9f81f3f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrm1700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1700%26sid%3Dliteratum%253Aachs%26aulast%3DWelchman%26aufirst%3DR.%2BL.%26aulast%3DGordon%26aufirst%3DC.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26atitle%3DUbiquitin%2520and%2520Ubiquitin-like%2520Proteins%2520as%2520Multifunctional%2520Signals%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D599%26epage%3D609%26doi%3D10.1038%2Fnrm1700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clague, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbé, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span> <span> </span><span class="NLM_article-title">Breaking the Chains: Deubiquitylating Enzyme Specificity Begets Function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">338</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/s41580-019-0099-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fs41580-019-0099-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30733604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlWlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=338-352&author=M.+J.+Clagueauthor=S.+Urb%C3%A9author=D.+Komander&title=Breaking+the+Chains%3A+Deubiquitylating+Enzyme+Specificity+Begets+Function&doi=10.1038%2Fs41580-019-0099-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking the chains: deubiquitylating enzyme specificity begets function</span></div><div class="casAuthors">Clague, Michael J.; Urbe, Sylvie; Komander, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">338-352</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The deubiquitylating enzymes (DUBs, also known as deubiquitylases or deubiquitinases) maintain the dynamic state of the cellular ubiquitome by releasing conjugated ubiquitin from proteins.  In light of the many cellular functions of ubiquitin, DUBs occupy key roles in almost all aspects of cell behavior.  Many DUBs show selectivity for particular ubiquitin linkage types or positions within ubiquitin chains.  Others show chain-type promiscuity but can select a distinct palette of protein substrates via specific protein-protein interactions established through binding modules outside of the catalytic domain.  The ubiquitin chain cleavage mode or chain linkage specificity has been related directly to biol. functions.  Examples include regulation of protein degrdn. and ubiquitin recycling by the proteasome, DNA repair pathways and innate immune signalling.  DUB cleavage specificity is also being harnessed for anal. of ubiquitin chain architecture that is assembled on specific proteins.  The recent development of highly specific DUB inhibitors heralds their emergence as a new class of therapeutic targets for numerous diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZMzX1ZpHBPrVg90H21EOLACvtfcHk0li44J6vJ7yvsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlWlt74%253D&md5=616b04e41e4184608c22b86bbcd328b5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41580-019-0099-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41580-019-0099-1%26sid%3Dliteratum%253Aachs%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DUrb%25C3%25A9%26aufirst%3DS.%26aulast%3DKomander%26aufirst%3DD.%26atitle%3DBreaking%2520the%2520Chains%253A%2520Deubiquitylating%2520Enzyme%2520Specificity%2520Begets%2520Function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2019%26volume%3D20%26spage%3D338%26epage%3D352%26doi%3D10.1038%2Fs41580-019-0099-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Turcu, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullally, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heroux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin Binding Domain ZnF UBP Recognizes the C-Terminal Diglycine Motif of Unanchored Ubiquitin</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1197</span>– <span class="NLM_lpage">1208</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.02.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.cell.2006.02.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=16564012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt1KhsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=1197-1208&author=F.+E.+Reyes-Turcuauthor=J.+R.+Hortonauthor=J.+E.+Mullallyauthor=A.+Herouxauthor=X.+Chengauthor=K.+D.+Wilkinson&title=The+Ubiquitin+Binding+Domain+ZnF+UBP+Recognizes+the+C-Terminal+Diglycine+Motif+of+Unanchored+Ubiquitin&doi=10.1016%2Fj.cell.2006.02.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin binding domain ZnF UBP recognizes the C-terminal diglycine motif of unanchored ubiquitin</span></div><div class="casAuthors">Reyes-Turcu, Francisca E.; Horton, John R.; Mullally, James E.; Heroux, Annie; Cheng, Xiaodong; Wilkinson, Keith D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1197-1208</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ubiquitin binding proteins regulate the stability, function, and/or localization of ubiquitinated proteins.  Here we report the crystal structures of the zinc-finger ubiquitin binding domain (ZnF UBP) from the deubiquitinating enzyme isopeptidase T (IsoT, or USP5) alone and in complex with ubiquitin.  Unlike other ubiquitin binding domains, this domain contains a deep binding pocket where the C-terminal diglycine motif of ubiquitin is inserted, thus explaining the specificity of IsoT for an unmodified C terminus on the proximal subunit of polyubiquitin.  Mutations in the domain demonstrate that it is required for optimal catalytic activation of IsoT.  This domain is present in several other protein families, and the ZnF UBP domain from an E3 ligase also requires the C terminus of ubiquitin for binding.  These data suggest that binding the ubiquitin C terminus may be necessary for the function of other proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohW2SwRsFle7Vg90H21EOLACvtfcHk0li44J6vJ7yvsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt1KhsrY%253D&md5=1558ddae03f9884e8848e8ab449a1ae3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.02.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.02.038%26sid%3Dliteratum%253Aachs%26aulast%3DReyes-Turcu%26aufirst%3DF.%2BE.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DMullally%26aufirst%3DJ.%2BE.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26atitle%3DThe%2520Ubiquitin%2520Binding%2520Domain%2520ZnF%2520UBP%2520Recognizes%2520the%2520C-Terminal%2520Diglycine%2520Motif%2520of%2520Unanchored%2520Ubiquitin%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D1197%26epage%3D1208%26doi%3D10.1016%2Fj.cell.2006.02.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avvakumov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asinas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, U. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span> <span> </span><span class="NLM_article-title">Two ZnF-UBP Domains in Isopeptidase T (USP5)</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1188</span>– <span class="NLM_lpage">1198</span>, <span class="refDoi"> DOI: 10.1021/bi200854q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200854q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1188-1198&author=G.+V.+Avvakumovauthor=J.+R.+Walkerauthor=S.+Xueauthor=A.+Allali-Hassaniauthor=A.+Asinasauthor=U.+B.+Nairauthor=X.+Fangauthor=X.+Zuoauthor=Y.+X.+Wangauthor=K.+D.+Wilkinsonauthor=S.+Dhe-Paganon&title=Two+ZnF-UBP+Domains+in+Isopeptidase+T+%28USP5%29&doi=10.1021%2Fbi200854q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Two ZnF-UBP Domains in Isopeptidase T (USP5)</span></div><div class="casAuthors">Avvakumov, George V.; Walker, John R.; Xue, Sheng; Allali-Hassani, Abdellah; Asinas, Abdalin; Nair, Usha B.; Fang, Xianyang; Zuo, Xiaobing; Wang, Yun-Xing; Wilkinson, Keith D.; Dhe-Paganon, Sirano</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1188-1198</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human ubiquitin-specific cysteine protease 5 (USP5, also known as ISOT and isopeptidase T), an 835-residue multidomain enzyme, recycles ubiquitin by hydrolyzing isopeptide bonds in a variety of unanchored polyubiquitin substrates.  Activation of the enzyme's hydrolytic activity toward ubiquitin-AMC (7-amino-4-methylcoumarin), a fluorogenic substrate, by the addn. of free, unanchored monoubiquitin suggested an allosteric mechanism of activation by the ZnF-UBP domain (residues 163-291), which binds the substrate's unanchored diglycine carboxyl tail.  By detg. the structure of full-length USP5, we discovered the existence of a cryptic ZnF-UBP domain (residues 1-156), which was tightly bound to the catalytic core and was indispensable for catalytic activity.  In contrast, the previously characterized ZnF-UBP domain did not contribute directly to the active site; a paucity of interactions suggested flexibility between these two domains consistent with an ability of the enzyme to hydrolyze a variety of different polyubiquitin chain linkages.  Deletion of the known ZnF-UBP domain did not significantly affect rate of hydrolysis of ubiquitin-AMC and suggested that it is likely assocd. mainly with substrate targeting and specificity.  Together, our findings show that USP5 uses multiple ZnF-UBP domains for substrate targeting and core catalytic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALXcZB0G4X7Vg90H21EOLACvtfcHk0li44J6vJ7yvsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWhs7g%253D&md5=45159b86dcdd454a0ac3974a48a36ab3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi200854q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200854q%26sid%3Dliteratum%253Aachs%26aulast%3DAvvakumov%26aufirst%3DG.%2BV.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DXue%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAsinas%26aufirst%3DA.%26aulast%3DNair%26aufirst%3DU.%2BB.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%2BX.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26atitle%3DTwo%2520ZnF-UBP%2520Domains%2520in%2520Isopeptidase%2520T%2520%2528USP5%2529%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D1188%26epage%3D1198%26doi%3D10.1021%2Fbi200854q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amerik, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochstrasser, M.</span></span> <span> </span><span class="NLM_article-title">Mechanism and Function of Deubiquitinating Enzymes</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1695</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2004.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.bbamcr.2004.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=15571815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1695&publication_year=2004&pages=189-207&author=A.+Y.+Amerikauthor=M.+Hochstrasser&title=Mechanism+and+Function+of+Deubiquitinating+Enzymes&doi=10.1016%2Fj.bbamcr.2004.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and function of deubiquitinating enzymes</span></div><div class="casAuthors">Amerik, Alexander Y.; Hochstrasser, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1695</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">189-207</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The attachment of ubiquitin (I) to proteins is a crucial step in many cellular regulatory mechanisms and contributes to numerous biol. processes, including embryonic development, the cell cycle, growth control, and prevention of neurodegeneration.  In these diverse regulatory settings, the most widespread mechanism of I action is probably in the context of protein degrdn.  Polyubiquitin attachment targets many intracellular proteins for degrdn. by the proteasome, and (mono)ubiquitination is often required for down-regulating plasma membrane proteins by targeting them to the vacuole (lysosome).  Ubiquitin-protein conjugates are highly dynamic structures.  Whereas an array of enzymes directs the conjugation of I to substrates, there are also dozens of deubiquitinating enzymes (DUBs) that can reverse the process.  Several lines of evidence indicate that DUBs are important regulators of the I system.  These enzymes are responsible for processing inactive I precursors, proofreading I-protein conjugates, removing I from cellular adducts, and keeping the 26S proteasome free of inhibitory I chains.  The present review focuses on recent discoveries that have led to a better understanding the mechanisms and physiol. roles of this diverse and still poorly understood group of enzymes.  The authors also discuss briefly some of the proteases that act on ubiquitin-like protein (UBL) conjugates and compare them to DUBs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4r5Q5j5cVbrVg90H21EOLACvtfcHk0lhGNVbOXw37Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzF&md5=4be57f84af923c82da6b84c6da73a9d9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2004.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2004.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DAmerik%26aufirst%3DA.%2BY.%26aulast%3DHochstrasser%26aufirst%3DM.%26atitle%3DMechanism%2520and%2520Function%2520of%2520Deubiquitinating%2520Enzymes%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2004%26volume%3D1695%26spage%3D189%26epage%3D207%26doi%3D10.1016%2Fj.bbamcr.2004.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melandri, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, R. L.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Mechanistic Studies on the Hydrolysis of Ubiquitin C- Terminal 7-Amido-4-Methylcoumarin by Deubiquitinating Enzymes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1868</span>– <span class="NLM_lpage">1879</span>, <span class="refDoi"> DOI: 10.1021/bi9723360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9723360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK1cXms1OjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=1868-1879&author=L.+C.+Dangauthor=F.+D.+Melandriauthor=R.+L.+Stein&title=Kinetic+and+Mechanistic+Studies+on+the+Hydrolysis+of+Ubiquitin+C-+Terminal+7-Amido-4-Methylcoumarin+by+Deubiquitinating+Enzymes&doi=10.1021%2Fbi9723360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and Mechanistic Studies on the Hydrolysis of Ubiquitin C-Terminal 7-Amido-4-Methylcoumarin by Deubiquitinating Enzymes</span></div><div class="casAuthors">Dang, Luan C.; Melandri, Francesco D.; Stein, Ross L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1868-1879</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Deubiquitinating enzymes constitute a family of cysteine hydrolases that specifically cleave ubiquitin-derived substrates of general structure Ub-X, where X can be any no. of leaving groups ranging from small thiols and amines to Ub and other proteins (Ub, ubiquitin).  We have developed a general assay for deubiquitinating enzymes based on the substrate ubiquitin C-terminal 7-amido-4-methylcoumarin (Ub-AMC).  Ub-AMC is efficiently hydrolyzed with liberation of highly fluorescent AMC by two rabbit reticulocyte deubiquitinating enzymes: isopeptidase T (IPaseT), a member of the gene family of ubiquitin-specific processing enzymes, and UCH-L3, a member of the family of ubiquitin C-terminal hydrolases.  We used this new assay to probe kinetic and mechanistic aspects of catalysis by IPaseT and UCH-L3.  Results from four series of expts. are discussed: (1) For UCH-L3, we detd. steady-state kinetic parameters that suggest a diffusion-limited reaction of UCH-L3 with Ub-AMC.  To probe this, we detd. the viscosity dependence of kc/Km, as well as kc.  We found complex viscosity dependencies and interpreted these in the context of a model in which assocn. and acylation are viscosity-dependent but deacylation is viscosity-independent.  (2) The kinetics of inhibition of UCH-L3 by ubiquitin C-terminal aldehyde (Ub-H) were detd. and reveal a Ki that is less than 10-14 M.  Several mechanisms are considered to account for the extreme inhibition.  (3) The IPaseT-catalyzed hydrolysis of Ub-AMC is modulated by Ub with activation at low [Ub] and inhibition at high [Ub].  (4) Finally, we compare kc/Km values for deubiquitinating enzyme-catalyzed hydrolysis of Ub-AMC and Z-Leu-Arg-Gly-Gly-AMC.  For IPaseT, the ratio of rate consts. is 104, while for UCH-L3 this ratio is >107.  These results suggest the following: (i) Deubiquitinating enzymes are able to utilize the free energy that is released from remote interactions with Ub-contg. substrates for stabilization of catalytic transition states, and (ii) UCHs are more efficient at utilizing the energy from these interactions, presumably because they do not possess a binding domain for a Ub "leaving group".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOpMDRjOWV2LVg90H21EOLACvtfcHk0lhGNVbOXw37Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXms1OjsA%253D%253D&md5=078fe0b916c22ef4ea7d4fc77aca75c1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fbi9723360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9723360%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DL.%2BC.%26aulast%3DMelandri%26aufirst%3DF.%2BD.%26aulast%3DStein%26aufirst%3DR.%2BL.%26atitle%3DKinetic%2520and%2520Mechanistic%2520Studies%2520on%2520the%2520Hydrolysis%2520of%2520Ubiquitin%2520C-%2520Terminal%25207-Amido-4-Methylcoumarin%2520by%2520Deubiquitinating%2520Enzymes%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D1868%26epage%3D1879%26doi%3D10.1021%2Fbi9723360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dayal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allende-Vega, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saville, M. K.</span></span> <span> </span><span class="NLM_article-title">Suppression of the Deubiquitinating Enzyme USP5 Causes the Accumulation of Unanchored Polyubiquitin and the Activation of P53</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">5030</span>– <span class="NLM_lpage">5041</span>, <span class="refDoi"> DOI: 10.1074/jbc.M805871200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1074%2Fjbc.M805871200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=19098288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVSnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=5030-5041&author=S.+Dayalauthor=A.+Sparksauthor=J.+Jacobauthor=N.+Allende-Vegaauthor=D.+P.+Laneauthor=M.+K.+Saville&title=Suppression+of+the+Deubiquitinating+Enzyme+USP5+Causes+the+Accumulation+of+Unanchored+Polyubiquitin+and+the+Activation+of+P53&doi=10.1074%2Fjbc.M805871200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of the Deubiquitinating Enzyme USP5 Causes the Accumulation of Unanchored Polyubiquitin and the Activation of p53</span></div><div class="casAuthors">Dayal, Saurabh; Sparks, Alison; Jacob, Jimmy; Allende-Vega, Nerea; Lane, David P.; Saville, Mark K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5030-5041</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Both p53 and its repressor Mdm2 are subject to ubiquitination and proteasomal degrdn.  The authors show that knockdown of the deubiquitinating enzyme USP5 (isopeptidase T) results in an increase in the level and transcriptional activity of p53.  Suppression of USP5 stabilizes p53, whereas it has little or no effect on the stability of Mdm2.  This provides a mechanism for transcriptional activation of p53.  USP5 knockdown interferes with the degrdn. of ubiquitinated p53 rather than attenuating p53 ubiquitination.  In vitro studies have shown that a preferred substrate for USP5 is unanchored polyubiquitin.  Consistent with this, the authors obsd. for the first time in a mammalian system that USP5 makes a major contribution to Lys-48-linked polyubiquitin disassembly and that suppression of USP5 results in the accumulation of unanchored polyubiquitin chains.  Ectopic expression of a C-terminal mutant of ubiquitin (G75A/G76A), which also causes the accumulation of free polyubiquitin, recapitulates the effects of USP5 knockdown on the p53 pathway.  The authors propose a model in which p53 is selectively stabilized because the unanchored polyubiquitin that accumulates after USP5 knockdown is able to compete with ubiquitinated p53 but not with Mdm2 for proteasomal recognition.  This raises the possibility that there are significant differences in proteasomal recognition of p53 and Mdm2.  These differences could be exploited therapeutically.  This study reveals a novel mechanism for regulation of p53 and identifies USP5 as a potential target for p53 activating therapeutic agents for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi-dWbk_lGsrVg90H21EOLACvtfcHk0lgR6sq_MTH_ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVSnsL0%253D&md5=037f3359f4376b81a1726734f31acc18</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M805871200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M805871200%26sid%3Dliteratum%253Aachs%26aulast%3DDayal%26aufirst%3DS.%26aulast%3DSparks%26aufirst%3DA.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DAllende-Vega%26aufirst%3DN.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DSaville%26aufirst%3DM.%2BK.%26atitle%3DSuppression%2520of%2520the%2520Deubiquitinating%2520Enzyme%2520USP5%2520Causes%2520the%2520Accumulation%2520of%2520Unanchored%2520Polyubiquitin%2520and%2520the%2520Activation%2520of%2520P53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D5030%26epage%3D5041%26doi%3D10.1074%2Fjbc.M805871200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. F.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin Specific Peptidase 5 Mediates Histidine-Rich Protein Hpn Induced Cell Apoptosis in Hepatocellular Carcinoma through P14-P53 Signaling</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>, <span class="NLM_elocation-id">1600350</span> <span class="refDoi"> DOI: 10.1002/pmic.201600350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1002%2Fpmic.201600350" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&author=Y.+Liuauthor=W.+M.+Wangauthor=L.+Y.+Zouauthor=L.+Liauthor=L.+Fengauthor=M.+Z.+Panauthor=M.+Y.+Lvauthor=Y.+Caoauthor=H.+Wangauthor=H.+F.+Kungauthor=J.+X.+Pangauthor=W.+M.+Fuauthor=J.+F.+Zhang&title=Ubiquitin+Specific+Peptidase+5+Mediates+Histidine-Rich+Protein+Hpn+Induced+Cell+Apoptosis+in+Hepatocellular+Carcinoma+through+P14-P53+Signaling&doi=10.1002%2Fpmic.201600350"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201600350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201600350%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%2BM.%26aulast%3DZou%26aufirst%3DL.%2BY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DM.%2BZ.%26aulast%3DLv%26aufirst%3DM.%2BY.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DKung%26aufirst%3DH.%2BF.%26aulast%3DPang%26aufirst%3DJ.%2BX.%26aulast%3DFu%26aufirst%3DW.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%2BF.%26atitle%3DUbiquitin%2520Specific%2520Peptidase%25205%2520Mediates%2520Histidine-Rich%2520Protein%2520Hpn%2520Induced%2520Cell%2520Apoptosis%2520in%2520Hepatocellular%2520Carcinoma%2520through%2520P14-P53%2520Signaling%26jtitle%3DProteomics%26date%3D2017%26volume%3D17%26doi%3D10.1002%2Fpmic.201600350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaistha, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krattenmacher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredebohm, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrens, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strobel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoheisel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gress, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchholz, M.</span></span> <span> </span><span class="NLM_article-title">The Deubiquitinating Enzyme USP5 Promotes Pancreatic Cancer via Modulating Cell Cycle Regulators</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">66215</span>– <span class="NLM_lpage">66225</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.19882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.18632%2Foncotarget.19882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29029505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gsVCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=66215-66225&author=B.+P.+Kaisthaauthor=A.+Krattenmacherauthor=J.+Fredebohmauthor=H.+Schmidtauthor=D.+Behrensauthor=M.+Widderauthor=T.+Hackertauthor=O.+Strobelauthor=J.+D.+Hoheiselauthor=T.+M.+Gressauthor=M.+Buchholz&title=The+Deubiquitinating+Enzyme+USP5+Promotes+Pancreatic+Cancer+via+Modulating+Cell+Cycle+Regulators&doi=10.18632%2Foncotarget.19882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The deubiquitinating enzyme USP5 promotes pancreatic cancer via modulating cell cycle regulators</span></div><div class="casAuthors">Kaistha Brajesh P; Krattenmacher Anja; Schmidt Harald; Gress Thomas M; Buchholz Malte; Fredebohm Johannes; Hoheisel Jorg D; Behrens Diana; Widder Miriam; Hackert Thilo; Strobel Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">66215-66225</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal solid tumors.  With an overall five-year survival rate remaining below 6%, there is an explicit need to search for new molecular targets for therapeutic interventions.  We undertook a barcode labelled short-hairpin (shRNA) library screen in pancreatic cancer cells in order to identify novel genes promoting cancer survival and progression.  Among the candidate genes identified in this screen was the deubiquitinase USP5, which subsequent gene expression analyses demonstrated to be significantly upregulated in primary human pancreatic cancer tissues.  Using different knockdown approaches, we show that expression of USP5 is essential for the proliferation and survival of pancreatic cancer cells, tested under different 2D and 3D cell culture conditions as well as in in vivo experiments.  These growth inhibition effects upon knockdown of USP5 are mediated primarily by the attenuation of G1/S phase transition in the cells, which is accompanied by accumulation of DNA damage, upregulation of p27, and increased apoptosis rates.  Since USP5 is overexpressed in cancer tissues, it can thus potentially serve as a new target for therapeutic interventions, especially given the fact that deubiquitinases are currently emerging as new class of attractive drug targets in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJAEqHM0GGgT69zv1FsJFMfW6udTcc2eZIi_-VB10i5bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gsVCmsw%253D%253D&md5=d95f1f00cdc6a04e3fd9495252ceedb9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.19882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.19882%26sid%3Dliteratum%253Aachs%26aulast%3DKaistha%26aufirst%3DB.%2BP.%26aulast%3DKrattenmacher%26aufirst%3DA.%26aulast%3DFredebohm%26aufirst%3DJ.%26aulast%3DSchmidt%26aufirst%3DH.%26aulast%3DBehrens%26aufirst%3DD.%26aulast%3DWidder%26aufirst%3DM.%26aulast%3DHackert%26aufirst%3DT.%26aulast%3DStrobel%26aufirst%3DO.%26aulast%3DHoheisel%26aufirst%3DJ.%2BD.%26aulast%3DGress%26aufirst%3DT.%2BM.%26aulast%3DBuchholz%26aufirst%3DM.%26atitle%3DThe%2520Deubiquitinating%2520Enzyme%2520USP5%2520Promotes%2520Pancreatic%2520Cancer%2520via%2520Modulating%2520Cell%2520Cycle%2520Regulators%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D66215%26epage%3D66225%26doi%3D10.18632%2Foncotarget.19882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the Deubiquitinase USP5 Leads to C-Maf Protein Degradation and Myeloma Cell Apoptosis</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e3058</span> <span class="refDoi"> DOI: 10.1038/cddis.2017.450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fcddis.2017.450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=28933784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=S.+Wangauthor=J.+Juanauthor=Z.+Zhangauthor=Y.+Duauthor=Y.+Xuauthor=J.+Tongauthor=B.+Caoauthor=M.+F.+Moranauthor=Y.+Zengauthor=X.+Mao&title=Inhibition+of+the+Deubiquitinase+USP5+Leads+to+C-Maf+Protein+Degradation+and+Myeloma+Cell+Apoptosis&doi=10.1038%2Fcddis.2017.450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the deubiquitinase USP5 leads to c-Maf protein degradation and myeloma cell apoptosis</span></div><div class="casAuthors">Wang, Siyu; Juan, Jiaxiang; Zhang, Zubin; Du, Yanyun; Xu, Yujia; Tong, Jiefei; Cao, Biyin; Moran, Michael F.; Zeng, Yuanying; Mao, Xinliang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e3058</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The deubiquitinase USP5 stabilizes c-Maf, a key transcription factor in multiple myeloma (MM), but the mechanisms and significance are unclear.  In the present study, USP5 was found to interact with c-Maf and prevented it from degrdn. by decreasing its polyubiquitination level.  Specifically, the 308th and 347th lysine residues in c-Maf were crit. for USP5-mediated deubiquitination and stability.  There are five key domains in the USP5 protein and subsequent studies revealed that the cryptic ZnF domain and the C-box domain interacted with c-Maf but the UBA1/UBA2 domain partly increased its stability.  Notably, MafA and MafB are also members of the c-Maf family, however, USP5 failed to deubiquitinate MafA, suggesting its substrate specificity.  In the functional studies, USP5 was found to promoted the transcriptional activity of c-Maf.  Consistent with the high level of c-Maf protein in MM cells, USP5 was also highly expressed.  When USP5 was knocked down, c-Maf underwent degrdn.  Interestingly, USP5 silence led to apoptosis of MM cells expressing c-Maf but not MM cells lacking c-Maf, indicating c-Maf is a key factor in USP5-mediated MM cell proliferation and survival.  Consistent with this finding, WP1130, an inhibitor of several Dubs including USP5, suppressed the transcriptional activity of c-Maf and induced MM cell apoptosis.  When c-Maf was overexpressed, WP1130-induced MM cell apoptosis was abolished.  Taken together, these findings suggest that USP5 regulates c-Maf stability and MM cell survival.  Targeting the USP5/c-Maf axis could be a potential strategy for MM treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6AdLuHwkC3rVg90H21EOLACvtfcHk0lgR6sq_MTH_ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrvE&md5=a44f223532f555e4bfcab67f4a91a9b3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2017.450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2017.450%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DJuan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DB.%26aulast%3DMoran%26aufirst%3DM.%2BF.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DX.%26atitle%3DInhibition%2520of%2520the%2520Deubiquitinase%2520USP5%2520Leads%2520to%2520C-Maf%2520Protein%2520Degradation%2520and%2520Myeloma%2520Cell%2520Apoptosis%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fcddis.2017.450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span> <span> </span><span class="NLM_article-title">Overexpression of USP5 Contributes to Tumorigenesis in Non-Small Cell Lung Cancer via the Stabilization of β-Catenin Protein</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2284</span>– <span class="NLM_lpage">2295</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30555744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVylur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=2284-2295&author=X.+Maauthor=W.+Qiauthor=H.+Panauthor=F.+Yangauthor=J.+Deng&title=Overexpression+of+USP5+Contributes+to+Tumorigenesis+in+Non-Small+Cell+Lung+Cancer+via+the+Stabilization+of+%CE%B2-Catenin+Protein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of USP5 contributes to tumorigenesis in non-small cell lung cancer via the stabilization of β-catenin protein</span></div><div class="casAuthors">Ma, Xingjie; Qi, Weibo; Pan, Huan; Yang, Fan; Deng, Jiali</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2284-2295</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">The ubiquitin-specific protease 5 (USP5), a deubiquitinating enzyme, has been identified as a tumor promoter in several types of human cancer.  However, the role of USP5 in non-small lung cancer (NSCLC) has not yet been elucidated.  In this study, we found that USP5 was upregulated in NSCLC tissues compared with normal tissues.  High expression of USP5 was correlated with large primary tumor size, poor differentiation and advanced TNM stage, and led to a significantly shorter overall survival (OS).  USP5 overexpression enhanced, whereas USP5 silencing impaired the cell proliferation and colony formation of NSCLC cells in vitro.  Moreover, knockdown of USP5 in H1299 cells inhibited tumor growth in vivo.  Mechanistically, we found that USP5 deubiquitinated β-catenin, prevented ubiquitination mediated β-catenin degrdn. and promoted β-catenin nuclear accumulation, leading to the activation of Wnt/β-catenin signal pathway in NSCLC cells.  Taken together, these findings suggest that USP5 functions as an oncogene in NSCLC and its oncogenic activity involves in part through Wnt/β-catenin signal pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ1pGOpVPzmLVg90H21EOLACvtfcHk0lgjb5LE70GLqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVylur3E&md5=36aa00674ccda08ed764e24a9c579c81</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DX.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DDeng%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520USP5%2520Contributes%2520to%2520Tumorigenesis%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%2520via%2520the%2520Stabilization%2520of%2520%25CE%25B2-Catenin%2520Protein%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2018%26volume%3D8%26spage%3D2284%26epage%3D2295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">García-Caballero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadotti, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemkowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgkinson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bladen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizzoccaro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deage, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francois, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourinet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamponi, G. W.</span></span> <span> </span><span class="NLM_article-title">The Deubiquitinating Enzyme USP5 Modulates Neuropathic and Inflammatory Pain by Enhancing Cav3.2 Channel Activity</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">1144</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2014.07.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.neuron.2014.07.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=25189210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFagtr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=1144-1158&author=A.+Garc%C3%ADa-Caballeroauthor=V.+M.+Gadottiauthor=P.+Stemkowskiauthor=N.+Weissauthor=I.+A.+Souzaauthor=V.+Hodgkinsonauthor=C.+Bladenauthor=L.+Chenauthor=J.+Hamidauthor=A.+Pizzoccaroauthor=M.+Deageauthor=A.+Francoisauthor=E.+Bourinetauthor=G.+W.+Zamponi&title=The+Deubiquitinating+Enzyme+USP5+Modulates+Neuropathic+and+Inflammatory+Pain+by+Enhancing+Cav3.2+Channel+Activity&doi=10.1016%2Fj.neuron.2014.07.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Deubiquitinating Enzyme USP5 Modulates Neuropathic and Inflammatory Pain by Enhancing Cav3.2 Channel Activity</span></div><div class="casAuthors">Garcia-Caballero, Agustin; Gadotti, Vinicius M.; Stemkowski, Patrick; Weiss, Norbert; Souza, Ivana A.; Hodgkinson, Victoria; Bladen, Chris; Chen, Lina; Hamid, Jawed; Pizzoccaro, Anne; Deage, Mickael; Francois, Amaury; Bourinet, Emmanuel; Zamponi, Gerald W.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1144-1158</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">T-type calcium channels are essential contributors to the transmission of nociceptive signals in the primary afferent pain pathway.  Here, we show that T-type calcium channels are ubiquitinated by WWP1, a plasma-membrane-assocd. ubiquitin ligase that binds to the intracellular domain III-IV linker region of the Cav3.2 T-type channel and modifies specific lysine residues in this region.  A proteomic screen identified the deubiquitinating enzyme USP5 as a Cav3.2 III-IV linker interacting partner.  Knockdown of USP5 via shRNA increases Cav3.2 ubiquitination, decreases Cav3.2 protein levels, and reduces Cav3.2 whole-cell currents.  In vivo knockdown of USP5 or uncoupling USP5 from native Cav3.2 channels via intrathecal delivery of Tat peptides mediates analgesia in both inflammatory and neuropathic mouse models of mech. hypersensitivity.  Altogether, our expts. reveal a cell signaling pathway that regulates T-type channel activity and their role in nociceptive signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqppH6fdxwbPbVg90H21EOLACvtfcHk0lgjb5LE70GLqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFagtr7K&md5=31e1e09b05a169c67c87ad531d3c1885</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2014.07.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2014.07.036%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Caballero%26aufirst%3DA.%26aulast%3DGadotti%26aufirst%3DV.%2BM.%26aulast%3DStemkowski%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DN.%26aulast%3DSouza%26aufirst%3DI.%2BA.%26aulast%3DHodgkinson%26aufirst%3DV.%26aulast%3DBladen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHamid%26aufirst%3DJ.%26aulast%3DPizzoccaro%26aufirst%3DA.%26aulast%3DDeage%26aufirst%3DM.%26aulast%3DFrancois%26aufirst%3DA.%26aulast%3DBourinet%26aufirst%3DE.%26aulast%3DZamponi%26aufirst%3DG.%2BW.%26atitle%3DThe%2520Deubiquitinating%2520Enzyme%2520USP5%2520Modulates%2520Neuropathic%2520and%2520Inflammatory%2520Pain%2520by%2520Enhancing%2520Cav3.2%2520Channel%2520Activity%26jtitle%3DNeuron%26date%3D2014%26volume%3D83%26spage%3D1144%26epage%3D1158%26doi%3D10.1016%2Fj.neuron.2014.07.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Caballero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadotti, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamponi, G. W.</span></span> <span> </span><span class="NLM_article-title">A Cell-Permeant Peptide Corresponding to the CUBP Domain of USP5 Reverses Inflammatory and Neuropathic Pain</span>. <i>Mol. Pain</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1177/1744806916642444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1177%2F1744806916642444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVOjs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1-8&author=A.+Garcia-Caballeroauthor=V.+M.+Gadottiauthor=L.+Chenauthor=G.+W.+Zamponi&title=A+Cell-Permeant+Peptide+Corresponding+to+the+CUBP+Domain+of+USP5+Reverses+Inflammatory+and+Neuropathic+Pain&doi=10.1177%2F1744806916642444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A cell-permeant peptide corresponding to the cUBP domain of USP5 reverses inflammatory and neuropathic pain</span></div><div class="casAuthors">Garcia-Caballero, Agustin; Gadotti, Vinicius M.; Chen, Lina; Zamponi, Gerald W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pain</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">MPOAC5</span>;
        ISSN:<span class="NLM_cas:issn">1744-8069</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Background: Cav3.2 T-type calcium currents in primary afferents are enhanced in various painful pathol. conditions, whereas inhibiting Cav3.2 activity or expression offers a strategy for combating the development of pain hypersensitivity.  We have shown that Cav3.2 channel surface d. is strongly regulated by the ubiquitination machinery and we identified the deubiquitinase USP5 as a Cav3.2 channel interacting protein and regulator of its cell surface expression.  We also reported that USP5 is upregulated in chronic pain conditions.  Conversely, preventing its binding to the channel in vivo mediates analgesia in inflammatory and neuropathic pain models.  Results: To identify which USP5 domain is responsible for the interaction, we used a series of USP5-derived peptides corresponding to different regions in nUBP, cUBP, UBA1, and UBA2 domains to outcompete full length USP5.  We identified a stretch of amino acid residues within the cUBP domain of USP5 as responsible for binding to Cav3.2 calcium channels.  Based on this information, we generated a TAT-cUBP1-USP5 peptide that could disrupt the Cav3.2/USP5 interaction in vitro and tested its physiol. effect in well-established models of persistent inflammatory pain (CFA test) and chronic mononeuropathy and polyneuropathy in mice (partial sciatic nerve injury and the (ob/ob) diabetic spontaneous neuropathic mice).  Our results reveal that the TAT-cUBP1-USP5 peptide attenuated mech. hyperalgesia induced by both Complete Freund's Adjuvant and partial sciatic nerve injury, and thermal hyperalgesia in diabetic neuropathic animals.  In contrast, Cav3.2 null mice were not affected by the peptide in the partial sciatic nerve injury model.  Cav3.2 calcium channel levels in diabetic mice were reduced following the administration of the TAT-cUBP1-USP5 peptide.  Conclusions: Our findings reveal a crucial region in the cUBP domain of USP5 that is important for substrate recognition and binding to the III-IV linker of Cav3.2 channels.  Targeting the interaction of this region with the Cav3.2 channel can be exploited as the basis for therapeutic intervention into inflammatory and neuropathic pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyT-_P1ltHgLVg90H21EOLACvtfcHk0lgjb5LE70GLqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVOjs70%253D&md5=fa46923173ccfe6a9b2fd4848171f116</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1744806916642444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1744806916642444%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia-Caballero%26aufirst%3DA.%26aulast%3DGadotti%26aufirst%3DV.%2BM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZamponi%26aufirst%3DG.%2BW.%26atitle%3DA%2520Cell-Permeant%2520Peptide%2520Corresponding%2520to%2520the%2520CUBP%2520Domain%2520of%2520USP5%2520Reverses%2520Inflammatory%2520and%2520Neuropathic%2520Pain%26jtitle%3DMol.%2520Pain%26date%3D2016%26volume%3D12%26spage%3D1%26epage%3D8%26doi%3D10.1177%2F1744806916642444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stemkowski, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Caballero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadotti, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">M’Dahoma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamponi, G. W.</span></span> <span> </span><span class="NLM_article-title">Identification of Interleukin-1 Beta as a Key Mediator in the Upregulation of Cav3.2–USP5 Interactions in the Pain Pathway</span>. <i>Mol. Pain</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1177/1744806917724698</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1177%2F1744806917724698" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1-7&author=P.+L.+Stemkowskiauthor=A.+Garcia-Caballeroauthor=V.+M.+Gadottiauthor=S.+M%E2%80%99Dahomaauthor=L.+Chenauthor=I.+A.+Souzaauthor=G.+W.+Zamponi&title=Identification+of+Interleukin-1+Beta+as+a+Key+Mediator+in+the+Upregulation+of+Cav3.2%E2%80%93USP5+Interactions+in+the+Pain+Pathway&doi=10.1177%2F1744806917724698"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1177%2F1744806917724698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1744806917724698%26sid%3Dliteratum%253Aachs%26aulast%3DStemkowski%26aufirst%3DP.%2BL.%26aulast%3DGarcia-Caballero%26aufirst%3DA.%26aulast%3DGadotti%26aufirst%3DV.%2BM.%26aulast%3DM%25E2%2580%2599Dahoma%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSouza%26aufirst%3DI.%2BA.%26aulast%3DZamponi%26aufirst%3DG.%2BW.%26atitle%3DIdentification%2520of%2520Interleukin-1%2520Beta%2520as%2520a%2520Key%2520Mediator%2520in%2520the%2520Upregulation%2520of%2520Cav3.2%25E2%2580%2593USP5%2520Interactions%2520in%2520the%2520Pain%2520Pathway%26jtitle%3DMol.%2520Pain%26date%3D2017%26volume%3D13%26spage%3D1%26epage%3D7%26doi%3D10.1177%2F1744806917724698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao-Feng Qin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span> <span> </span><span class="NLM_article-title">Broad and Diverse Mechanisms Used by Deubiquitinase Family Members in Regulating the Type i Interferon Signaling Pathway during Antiviral Responses</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">eaar2824</span> <span class="refDoi"> DOI: 10.1126/sciadv.aar2824</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1126%2Fsciadv.aar2824" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29732405" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&author=Q.+Liuauthor=Y.+Wuauthor=Y.+Qinauthor=J.+Huauthor=W.+Xieauthor=F.+Xiao-Feng+Qinauthor=J.+Cui&title=Broad+and+Diverse+Mechanisms+Used+by+Deubiquitinase+Family+Members+in+Regulating+the+Type+i+Interferon+Signaling+Pathway+during+Antiviral+Responses&doi=10.1126%2Fsciadv.aar2824"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aar2824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aar2824%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DQin%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DXiao-Feng%2BQin%26aufirst%3DF.%26aulast%3DCui%26aufirst%3DJ.%26atitle%3DBroad%2520and%2520Diverse%2520Mechanisms%2520Used%2520by%2520Deubiquitinase%2520Family%2520Members%2520in%2520Regulating%2520the%2520Type%2520i%2520Interferon%2520Signaling%2520Pathway%2520during%2520Antiviral%2520Responses%26jtitle%3DSci.%2520Adv.%26date%3D2018%26volume%3D4%26doi%3D10.1126%2Fsciadv.aar2824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galardy, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploegh, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masucci, M. G.</span></span> <span> </span><span class="NLM_article-title">Activity-Based Ubiquitin-Specific Protease (USP) Profiling of Virus-Infected and Malignant Human Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">2253</span>– <span class="NLM_lpage">2258</span>, <span class="refDoi"> DOI: 10.1073/pnas.0308411100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1073%2Fpnas.0308411100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=14982996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvVOisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=2253-2258&author=H.+Ovaaauthor=B.+M.+Kesslerauthor=U.+Rolenauthor=P.+J.+Galardyauthor=H.+L.+Ploeghauthor=M.+G.+Masucci&title=Activity-Based+Ubiquitin-Specific+Protease+%28USP%29+Profiling+of+Virus-Infected+and+Malignant+Human+Cells&doi=10.1073%2Fpnas.0308411100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based ubiquitin-specific protease (USP) profiling of virus-infected and malignant human cells</span></div><div class="casAuthors">Ovaa, Huib; Kessler, Benedikt M.; Rolen, Ulrika; Galardy, Paul J.; Ploegh, Hidde L.; Masucci, Maria G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2253-2258</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The family of ubiquitin (Ub)-specific proteases (USP) removes Ub from Ub conjugates and regulates a variety of cellular processes.  The human genome contains many putative USP-encoding genes, but little is known about USP tissue distribution, pattern of expression, activity, and substrate specificity.  We have used a chem.-based functional proteomics approach to identify active USPs in normal, virus-infected, and tumor-derived human cells.  Depending on tissue origin and stage of activation/differentiation, different USP activity profiles were revealed.  The activity of specific USPs, including USP5, -7, -9, -13, -15, and -22, was up-regulated by mitogen activation or virus infection in normal T and B lymphocytes.  UCH-L1 was highly expressed in tumor cell lines of epithelial and hematopoietic cell origin but was not detected in freshly isolated and mitogen-activated cells.  Up-regulation of this USP was a late event in the establishment of Epstein-Barr virus-immortalized lymphoblastoid cell lines and correlated with enhanced proliferation, suggesting a possible role in growth transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyC75pjCOjm7Vg90H21EOLACvtfcHk0lhxEqolFZi4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvVOisrg%253D&md5=e7842bcd2842f37f4e1c9b19a8e55e8f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0308411100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0308411100%26sid%3Dliteratum%253Aachs%26aulast%3DOvaa%26aufirst%3DH.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DRolen%26aufirst%3DU.%26aulast%3DGalardy%26aufirst%3DP.%2BJ.%26aulast%3DPloegh%26aufirst%3DH.%2BL.%26aulast%3DMasucci%26aufirst%3DM.%2BG.%26atitle%3DActivity-Based%2520Ubiquitin-Specific%2520Protease%2520%2528USP%2529%2520Profiling%2520of%2520Virus-Infected%2520and%2520Malignant%2520Human%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D2253%26epage%3D2258%26doi%3D10.1073%2Fpnas.0308411100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapić-Otrin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, A. S.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin-Specific Protease 5 Is Required for the Efficient Repair of DNA Double-Strand Breaks</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e84899</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0084899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1371%2Fjournal.pone.0084899" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=24454762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFKgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=S.+Nakajimaauthor=L.+Lanauthor=L.+Weiauthor=C.+L.+Hsiehauthor=V.+Rapi%C4%87-Otrinauthor=A.+Yasuiauthor=A.+S.+Levine&title=Ubiquitin-Specific+Protease+5+Is+Required+for+the+Efficient+Repair+of+DNA+Double-Strand+Breaks&doi=10.1371%2Fjournal.pone.0084899"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-specific protease 5 is required for the efficient repair of DNA double-strand breaks</span></div><div class="casAuthors">Nakajima, Satoshi; Lan, Li; Wei, Leizhen; Hsieh, Ching-Lung; Rapic-Otrin, Vesna; Yasui, Akira; Levine, Arthur S.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e84899/1-e84899/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">During the DNA damage response (DDR), ubiquitination plays an important role in the recruitment and regulation of repair proteins.  However, little is known about elimination of the ubiquitination signal after repair is completed.  Here we show that the ubiquitin-specific protease 5 (USP5), a deubiquitinating enzyme, is involved in the elimination of the ubiquitin signal from damaged sites and is required for efficient DNA double-strand break (DSB) repair.  Depletion of USP5 sensitizes cells to DNA damaging agents, produces DSBs, causes delayed disappearance of γH2AX foci after Bleocin treatment, and influences DSB repair efficiency in the homologous recombination pathway but not in the non-homologous end joining pathway.  USP5 co-localizes to DSBs induced by laser micro-irradn. in a RAD18-dependent manner.  Importantly, polyubiquitin chains at sites of DNA damage remained for longer periods in USP5-depleted cells.  Our results show that disassembly of polyubiquitin chains by USP5 at sites of damage is important for efficient DSB repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOeYhezJqUerVg90H21EOLACvtfcHk0lhxEqolFZi4eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFKgt70%253D&md5=8887ac9cef564f4df2282f697585ab82</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084899%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DS.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DHsieh%26aufirst%3DC.%2BL.%26aulast%3DRapi%25C4%2587-Otrin%26aufirst%3DV.%26aulast%3DYasui%26aufirst%3DA.%26aulast%3DLevine%26aufirst%3DA.%2BS.%26atitle%3DUbiquitin-Specific%2520Protease%25205%2520Is%2520Required%2520for%2520the%2520Efficient%2520Repair%2520of%2520DNA%2520Double-Strand%2520Breaks%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0084899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawahara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saeki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komada, M.</span></span> <span> </span><span class="NLM_article-title">Deubiquitylases USP5 and USP13 Are Recruited to and Regulate Heat-Induced Stress Granules through Their Deubiquitylating Activities</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>131</i></span>, <span class="NLM_elocation-id">jcs210856</span> <span class="refDoi"> DOI: 10.1242/jcs.210856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1242%2Fjcs.210856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29567855" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2018&author=X.+Xieauthor=S.+Matsumotoauthor=A.+Endoauthor=T.+Fukushimaauthor=H.+Kawaharaauthor=Y.+Saekiauthor=M.+Komada&title=Deubiquitylases+USP5+and+USP13+Are+Recruited+to+and+Regulate+Heat-Induced+Stress+Granules+through+Their+Deubiquitylating+Activities&doi=10.1242%2Fjcs.210856"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1242%2Fjcs.210856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.210856%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DX.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DEndo%26aufirst%3DA.%26aulast%3DFukushima%26aufirst%3DT.%26aulast%3DKawahara%26aufirst%3DH.%26aulast%3DSaeki%26aufirst%3DY.%26aulast%3DKomada%26aufirst%3DM.%26atitle%3DDeubiquitylases%2520USP5%2520and%2520USP13%2520Are%2520Recruited%2520to%2520and%2520Regulate%2520Heat-Induced%2520Stress%2520Granules%2520through%2520Their%2520Deubiquitylating%2520Activities%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2018%26volume%3D131%26doi%3D10.1242%2Fjcs.210856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sizemore, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharia, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubonja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawes, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhivich, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span> <span> </span><span class="NLM_article-title">Computational Correction of Copy Number Effect Improves Specificity of CRISPR-Cas9 Essentiality Screens in Cancer Cells</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1779</span>– <span class="NLM_lpage">1784</span>, <span class="refDoi"> DOI: 10.1038/ng.3984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fng.3984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29083409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=1779-1784&author=R.+M.+Meyersauthor=J.+G.+Bryanauthor=J.+M.+McFarlandauthor=B.+A.+Weirauthor=A.+E.+Sizemoreauthor=H.+Xuauthor=N.+V.+Dhariaauthor=P.+G.+Montgomeryauthor=G.+S.+Cowleyauthor=S.+Pantelauthor=A.+Goodaleauthor=Y.+Leeauthor=L.+D.+Aliauthor=G.+Jiangauthor=R.+Lubonjaauthor=W.+F.+Harringtonauthor=M.+Stricklandauthor=T.+Wuauthor=D.+C.+Hawesauthor=V.+A.+Zhivichauthor=M.+R.+Wyattauthor=Z.+Kalaniauthor=J.+J.+Changauthor=M.+Okamotoauthor=K.+Stegmaierauthor=T.+R.+Golubauthor=J.+S.+Boehmauthor=F.+Vazquezauthor=D.+E.+Rootauthor=W.+C.+Hahnauthor=A.+Tsherniak&title=Computational+Correction+of+Copy+Number+Effect+Improves+Specificity+of+CRISPR-Cas9+Essentiality+Screens+in+Cancer+Cells&doi=10.1038%2Fng.3984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells</span></div><div class="casAuthors">Meyers, Robin M.; Bryan, Jordan G.; McFarland, James M.; Weir, Barbara A.; Sizemore, Ann E.; Xu, Han; Dharia, Neekesh V.; Montgomery, Phillip G.; Cowley, Glenn S.; Pantel, Sasha; Goodale, Amy; Ali, Levi D.; Jiang, Guozhi; Lubonja, Rakela; Harrington, William F.; Strickland, Matthew; Wu, Ting; Hawes, Derek C.; Zhivich, Victor A.; Wyatt, Meghan R.; Kalani, Zohra; Chang, Jaime J.; Okamoto, Michael; Stegmaier, Kimberly; Golub, Todd R.; Boehm, Jesse S.; Vazquez, Francisca; Root, David E.; Hahn, William C.; Tsherniak, Aviad</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1779-1784</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The CRISPR-Cas9 system has revolutionized gene editing both at single genes and in multiplexed loss-of-function screens, thus enabling precise genome-scale identification of genes essential for proliferation and survival of cancer cells.  However, previous studies have reported that a gene-independent antiproliferative effect of Cas9-mediated DNA cleavage confounds such measurement of genetic dependency, thereby leading to false-pos. results in copy no.-amplified regions.  We developed CERES, a computational method to est. gene-dependency levels from CRISPR-Cas9 essentiality screens while accounting for the copy no.-specific effect.  In our efforts to define a cancer dependency map, we performed genome-scale CRISPR-Cas9 essentiality screens across 342 cancer cell lines and applied CERES to this data set.  We found that CERES decreased false-pos. results and estd. sgRNA activity for both this data set and previously published screens performed with different sgRNA libraries.  We further demonstrate the utility of this collection of screens, after CERES correction, for identifying cancer-type-specific vulnerabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8iODR9NtDbVg90H21EOLACvtfcHk0lgCxLPsUv1ddQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI&md5=539e8fb16958480ce2d430a7f8e6cddf</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fng.3984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3984%26sid%3Dliteratum%253Aachs%26aulast%3DMeyers%26aufirst%3DR.%2BM.%26aulast%3DBryan%26aufirst%3DJ.%2BG.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DSizemore%26aufirst%3DA.%2BE.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDharia%26aufirst%3DN.%2BV.%26aulast%3DMontgomery%26aufirst%3DP.%2BG.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DPantel%26aufirst%3DS.%26aulast%3DGoodale%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAli%26aufirst%3DL.%2BD.%26aulast%3DJiang%26aufirst%3DG.%26aulast%3DLubonja%26aufirst%3DR.%26aulast%3DHarrington%26aufirst%3DW.%2BF.%26aulast%3DStrickland%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DHawes%26aufirst%3DD.%2BC.%26aulast%3DZhivich%26aufirst%3DV.%2BA.%26aulast%3DWyatt%26aufirst%3DM.%2BR.%26aulast%3DKalani%26aufirst%3DZ.%26aulast%3DChang%26aufirst%3DJ.%2BJ.%26aulast%3DOkamoto%26aufirst%3DM.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DTsherniak%26aufirst%3DA.%26atitle%3DComputational%2520Correction%2520of%2520Copy%2520Number%2520Effect%2520Improves%2520Specificity%2520of%2520CRISPR-Cas9%2520Essentiality%2520Screens%2520in%2520Cancer%2520Cells%26jtitle%3DNat.%2520Genet.%26date%3D2017%26volume%3D49%26spage%3D1779%26epage%3D1784%26doi%3D10.1038%2Fng.3984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Migliardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinnelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRae, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pooley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trusolino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stronach, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez-Rodiguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yusa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, M. J.</span></span> <span> </span><span class="NLM_article-title">Prioritization of Cancer Therapeutic Targets Using CRISPR–Cas9 Screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>568</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1103-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fs41586-019-1103-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30971826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=568&publication_year=2019&pages=511-516&author=F.+M.+Behanauthor=F.+Iorioauthor=G.+Piccoauthor=E.+Gon%C3%A7alvesauthor=C.+M.+Beaverauthor=G.+Migliardiauthor=R.+Santosauthor=Y.+Raoauthor=F.+Sassiauthor=M.+Pinnelliauthor=R.+Ansariauthor=S.+Harperauthor=D.+A.+Jacksonauthor=R.+McRaeauthor=R.+Pooleyauthor=P.+Wilkinsonauthor=D.+van+der+Meerauthor=D.+Dowauthor=C.+Buser-Doepnerauthor=A.+Bertottiauthor=L.+Trusolinoauthor=E.+A.+Stronachauthor=J.+Saez-Rodiguezauthor=K.+Yusaauthor=M.+J.+Garnett&title=Prioritization+of+Cancer+Therapeutic+Targets+Using+CRISPR%E2%80%93Cas9+Screens&doi=10.1038%2Fs41586-019-1103-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens</span></div><div class="casAuthors">Behan, Fiona M.; Iorio, Francesco; Picco, Gabriele; Goncalves, Emanuel; Beaver, Charlotte M.; Migliardi, Giorgia; Santos, Rita; Rao, Yanhua; Sassi, Francesco; Pinnelli, Marika; Ansari, Rizwan; Harper, Sarah; Jackson, David Adam; McRae, Rebecca; Pooley, Rachel; Wilkinson, Piers; van der Meer, Dieudonne; Dow, David; Buser-Doepner, Carolyn; Bertotti, Andrea; Trusolino, Livio; Stronach, Euan A.; Saez-Rodriguez, Julio; Yusa, Kosuke; Garnett, Mathew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">568</span>
        (<span class="NLM_cas:issue">7753</span>),
    <span class="NLM_cas:pages">511-516</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Functional genomics approaches can overcome limitations-such as the lack of identification of robust targets and poor clin. efficacy-that hamper cancer drug development.  Here we performed genome-scale CRISPR-Cas9 screens in 324 human cancer cell lines from 30 cancer types and developed a data-driven framework to prioritize candidates for cancer therapeutics.  We integrated cell fitness effects with genomic biomarkers and target tractability for drug development to systematically prioritize new targets in defined tissues and genotypes.  We verified one of our most promising dependencies, the Werner syndrome ATP-dependent helicase, as a synthetic lethal target in tumors from multiple cancer types with microsatellite instability.  Our anal. provides a resource of cancer dependencies, generates a framework to prioritize cancer drug targets and suggests specific new targets.  The principles described in this study can inform the initial stages of drug development by contributing to a new, diverse and more effective portfolio of cancer drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHnWQcCFsy-LVg90H21EOLACvtfcHk0lgCxLPsUv1ddQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D&md5=3cac7e1a9ee5c34c8599154e8fc30573</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1103-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1103-9%26sid%3Dliteratum%253Aachs%26aulast%3DBehan%26aufirst%3DF.%2BM.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DPicco%26aufirst%3DG.%26aulast%3DGon%25C3%25A7alves%26aufirst%3DE.%26aulast%3DBeaver%26aufirst%3DC.%2BM.%26aulast%3DMigliardi%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DY.%26aulast%3DSassi%26aufirst%3DF.%26aulast%3DPinnelli%26aufirst%3DM.%26aulast%3DAnsari%26aufirst%3DR.%26aulast%3DHarper%26aufirst%3DS.%26aulast%3DJackson%26aufirst%3DD.%2BA.%26aulast%3DMcRae%26aufirst%3DR.%26aulast%3DPooley%26aufirst%3DR.%26aulast%3DWilkinson%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DD.%26aulast%3DDow%26aufirst%3DD.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DBertotti%26aufirst%3DA.%26aulast%3DTrusolino%26aufirst%3DL.%26aulast%3DStronach%26aufirst%3DE.%2BA.%26aulast%3DSaez-Rodiguez%26aufirst%3DJ.%26aulast%3DYusa%26aufirst%3DK.%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26atitle%3DPrioritization%2520of%2520Cancer%2520Therapeutic%2520Targets%2520Using%2520CRISPR%25E2%2580%2593Cas9%2520Screens%26jtitle%3DNature%26date%3D2019%26volume%3D568%26spage%3D511%26epage%3D516%26doi%3D10.1038%2Fs41586-019-1103-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Depmap
Broad</span>. <span> </span><span class="NLM_article-title">DepMap Achilles 19Q1 Public</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Depmap%0ABroad&title=DepMap+Achilles+19Q1+Public"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DDepMap%2520Achilles%252019Q1%2520Public%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavory, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dowd, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flasz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkoudis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dossang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClelland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odrzywol, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, T.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Highly Potent and Selective Allosteric USP7 Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnchembio.2528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29200206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=118-125&author=G.+Gavoryauthor=C.+R.+O%E2%80%99Dowdauthor=M.+D.+Helmauthor=J.+Flaszauthor=E.+Arkoudisauthor=A.+Dossangauthor=C.+Hughesauthor=E.+Cassidyauthor=K.+McClellandauthor=E.+Odrzywolauthor=N.+Pageauthor=O.+Barkerauthor=H.+Mielauthor=T.+Harrison&title=Discovery+and+Characterization+of+Highly+Potent+and+Selective+Allosteric+USP7+Inhibitors&doi=10.1038%2Fnchembio.2528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of highly potent and selective allosteric USP7 inhibitors</span></div><div class="casAuthors">Gavory, Gerald; O'Dowd, Colin R.; Helm, Matthew D.; Flasz, Jakub; Arkoudis, Elias; Dossang, Anthony; Hughes, Caroline; Cassidy, Eamon; McClelland, Keeva; Odrzywol, Ewa; Page, Natalie; Barker, Oliver; Miel, Hugues; Harrison, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-125</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Given the importance of ubiquitin-specific protease 7 (USP7) in oncogenic pathways, identification of USP7 inhibitors has attracted considerable interest.  Despite substantial efforts, however, the development of validated deubiquitinase (DUB) inhibitors that exhibit drug-like properties and a well-defined mechanism of action has proven particularly challenging.  In this article, we describe the identification, optimization and detailed characterization of highly potent (IC50 < 10 nM), selective USP7 inhibitors together with their less active, enantiomeric counterparts.  We also disclose, for the first time, co-crystal structures of a human DUB enzyme complexed with small-mol. inhibitors, which reveal a previously undisclosed allosteric binding site.  Finally, we report the identification of cancer cell lines hypersensitive to USP7 inhibition (EC50 < 30 nM) and demonstrate equal or superior activity in these cell models compared to clin. relevant MDM2 antagonists.  Overall, these findings demonstrate the tractability and druggability of DUBs, and provide important tools for addnl. target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe5GeOEWbfz7Vg90H21EOLACvtfcHk0lgxy0F2DdENJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgs77F&md5=de6068d105b681c319e76e81ebfe9676</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2528%26sid%3Dliteratum%253Aachs%26aulast%3DGavory%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Dowd%26aufirst%3DC.%2BR.%26aulast%3DHelm%26aufirst%3DM.%2BD.%26aulast%3DFlasz%26aufirst%3DJ.%26aulast%3DArkoudis%26aufirst%3DE.%26aulast%3DDossang%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DC.%26aulast%3DCassidy%26aufirst%3DE.%26aulast%3DMcClelland%26aufirst%3DK.%26aulast%3DOdrzywol%26aufirst%3DE.%26aulast%3DPage%26aufirst%3DN.%26aulast%3DBarker%26aufirst%3DO.%26aulast%3DMiel%26aufirst%3DH.%26aulast%3DHarrison%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Highly%2520Potent%2520and%2520Selective%2520Allosteric%2520USP7%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D118%26epage%3D125%26doi%3D10.1038%2Fnchembio.2528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kategaya, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rougé, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drummond, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heideker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritorto, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trost, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bainbridge, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, M. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiffler, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaishankar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peale, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrest, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndubaku, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwerdtfeger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span> <span> </span><span class="NLM_article-title">USP7 Small-Molecule Inhibitors Interfere with Ubiquitin Binding</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/nature24006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnature24006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29045385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1OhtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=534-538&author=L.+Kategayaauthor=P.+Di+Lelloauthor=L.+Roug%C3%A9author=R.+Pastorauthor=K.+R.+Clarkauthor=J.+Drummondauthor=T.+Kleinheinzauthor=E.+Linauthor=J.+P.+Uptonauthor=S.+Prakashauthor=J.+Heidekerauthor=M.+McClelandauthor=M.+S.+Ritortoauthor=D.+R.+Alessiauthor=M.+Trostauthor=T.+W.+Bainbridgeauthor=M.+C.+M.+Kwokauthor=T.+P.+Maauthor=Z.+Stifflerauthor=B.+Brasherauthor=Y.+Tangauthor=P.+Jaishankarauthor=B.+R.+Hearnauthor=A.+R.+Rensloauthor=M.+R.+Arkinauthor=F.+Cohenauthor=K.+Yuauthor=F.+Pealeauthor=F.+Gnadauthor=M.+T.+Changauthor=C.+Klijnauthor=E.+Blackwoodauthor=S.+E.+Martinauthor=W.+F.+Forrestauthor=J.+A.+Ernstauthor=C.+Ndubakuauthor=X.+Wangauthor=M.+H.+Beresiniauthor=V.+Tsuiauthor=C.+Schwerdtfegerauthor=R.+A.+Blakeauthor=J.+Murrayauthor=T.+Maurerauthor=I.+E.+Wertz&title=USP7+Small-Molecule+Inhibitors+Interfere+with+Ubiquitin+Binding&doi=10.1038%2Fnature24006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">USP7 small-molecule inhibitors interfere with ubiquitin binding</span></div><div class="casAuthors">Kategaya, Lorna; Di Lello, Paola; Rouge, Lionel; Pastor, Richard; Clark, Kevin R.; Drummond, Jason; Kleinheinz, Tracy; Lin, Eva; Upton, John-Paul; Prakash, Sumit; Heideker, Johanna; McCleland, Mark; Ritorto, Maria Stella; Alessi, Dario R.; Trost, Matthias; Bainbridge, Travis W.; Kwok, Michael C. M.; Ma, Taylur P.; Stiffler, Zachary; Brasher, Bradley; Tang, Yinyan; Jaishankar, Priyadarshini; Hearn, Brian R.; Renslo, Adam R.; Arkin, Michelle R.; Cohen, Frederick; Yu, Kebing; Peale, Frank; Gnad, Florian; Chang, Matthew T.; Klijn, Christiaan; Blackwood, Elizabeth; Martin, Scott E.; Forrest, William F.; Ernst, James A.; Ndubaku, Chudi; Wang, Xiaojing; Beresini, Maureen H.; Tsui, Vickie; Schwerdtfeger, Carsten; Blake, Robert A.; Murray, Jeremy; Maurer, Till; Wertz, Ingrid E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7677</span>),
    <span class="NLM_cas:pages">534-538</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The ubiquitin system regulates essential cellular processes in eukaryotes.  Ubiquitin is ligated to substrate proteins as monomers or chains and the topol. of ubiquitin modifications regulates substrate interactions with specific proteins.  Thus, ubiquitination directs a variety of substrate fates including proteasomal degrdn.  Deubiquitinases cleave ubiquitin from substrates and are implicated in disease; e.g., ubiquitin-specific protease-7 (USP7) regulates the stability of the p53 tumor suppressor and other proteins crit. for tumor cell survival.  However, developing selective deubiquitinase inhibitors has been challenging and no co-crystal structures have been solved with small-mol. inhibitors.  Here, using NMR-based screening and structure-based design, we describe the development of the selective USP7 inhibitors, GNE-6640 and GNE-6776.  These compds. induce tumor cell death and enhance cytotoxicity with chemotherapeutic agents and targeted compds., including PIM kinase inhibitors.  Structural studies revealed that GNE-6640 and GNE-6776 non-covalently targeted USP7 12 Å distant from the catalytic Cys residue.  The compds. attenuated ubiquitin binding and thus inhibited USP7 deubiquitinase activity.  GNE-6640 and GNE-6776 interacted with acidic residues that mediate H-bond interactions with the ubiquitin Lys-48 side-chain, suggesting that USP7 preferentially interacts with and cleaves ubiquitin moieties that have free Lys-48 side-chains.  We investigated this idea by engineering di-ubiquitin chains contg. differential proximal and distal isotopic labels and measuring USP7 binding by NMR.  This preferential binding protracted the depolymn. kinetics of Lys-48-linked ubiquitin chains relative to Lys-63-linked chains.  In summary, engineering compds. that inhibit USP7 activity by attenuating ubiquitin binding suggests opportunities for developing other deubiquitinase inhibitors and may be a strategy more broadly applicable to inhibiting proteins that require ubiquitin binding for full functional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDUMQKwY0VGbVg90H21EOLACvtfcHk0lgxy0F2DdENJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1OhtrjL&md5=85af379796b4026105815b5ffdd01f8d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature24006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24006%26sid%3Dliteratum%253Aachs%26aulast%3DKategaya%26aufirst%3DL.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DRoug%25C3%25A9%26aufirst%3DL.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DK.%2BR.%26aulast%3DDrummond%26aufirst%3DJ.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DE.%26aulast%3DUpton%26aufirst%3DJ.%2BP.%26aulast%3DPrakash%26aufirst%3DS.%26aulast%3DHeideker%26aufirst%3DJ.%26aulast%3DMcCleland%26aufirst%3DM.%26aulast%3DRitorto%26aufirst%3DM.%2BS.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DTrost%26aufirst%3DM.%26aulast%3DBainbridge%26aufirst%3DT.%2BW.%26aulast%3DKwok%26aufirst%3DM.%2BC.%2BM.%26aulast%3DMa%26aufirst%3DT.%2BP.%26aulast%3DStiffler%26aufirst%3DZ.%26aulast%3DBrasher%26aufirst%3DB.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DJaishankar%26aufirst%3DP.%26aulast%3DHearn%26aufirst%3DB.%2BR.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DPeale%26aufirst%3DF.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DChang%26aufirst%3DM.%2BT.%26aulast%3DKlijn%26aufirst%3DC.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%2BE.%26aulast%3DForrest%26aufirst%3DW.%2BF.%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DNdubaku%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DSchwerdtfeger%26aufirst%3DC.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26atitle%3DUSP7%2520Small-Molecule%2520Inhibitors%2520Interfere%2520with%2520Ubiquitin%2520Binding%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D534%26epage%3D538%26doi%3D10.1038%2Fnature24006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villamil, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luci, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Z.</span></span> <span> </span><span class="NLM_article-title">A Selective USP1-UAF1 Inhibitor Links Deubiquitination to DNA Damage Responses</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnchembio.1455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=24531842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=298-304&author=Q.+Liangauthor=T.+S.+Dexheimerauthor=P.+Zhangauthor=A.+S.+Rosenthalauthor=M.+A.+Villamilauthor=C.+Youauthor=Q.+Zhangauthor=J.+Chenauthor=C.+A.+Ottauthor=H.+Sunauthor=D.+K.+Luciauthor=B.+Yuanauthor=A.+Simeonovauthor=A.+Jadhavauthor=H.+Xiaoauthor=Y.+Wangauthor=D.+J.+Maloneyauthor=Z.+Zhuang&title=A+Selective+USP1-UAF1+Inhibitor+Links+Deubiquitination+to+DNA+Damage+Responses&doi=10.1038%2Fnchembio.1455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A selective USP1-UAF1 inhibitor links deubiquitination to DNA damage responses</span></div><div class="casAuthors">Liang, Qin; Dexheimer, Thomas S.; Zhang, Ping; Rosenthal, Andrew S.; Villamil, Mark A.; You, Changjun; Zhang, Qiuting; Chen, Junjun; Ott, Christine A.; Sun, Hongmao; Luci, Diane K.; Yuan, Bifeng; Simeonov, Anton; Jadhav, Ajit; Xiao, Hui; Wang, Yinsheng; Maloney, David J.; Zhuang, Zhihao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">298-304</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein ubiquitination and deubiquitination are central to the control of a large no. of cellular pathways and signaling networks in eukaryotes.  Although the essential roles of ubiquitination have been established in the eukaryotic DNA damage response, the deubiquitination process remains poorly defined.  Chem. probes that perturb the activity of deubiquitinases (DUBs) are needed to characterize the cellular function of deubiquitination.  Here the authors report ML323 , a highly potent inhibitor of the USP1-UAF1 deubiquitinase complex with excellent selectivity against human DUBs, deSUMOylase, deneddylase and unrelated proteases.  Using ML323, the authors interrogated deubiquitination in the cellular response to UV- and cisplatin-induced DNA damage and revealed new insights into the requirement of deubiquitination in the DNA translesion synthesis and Fanconi anemia pathways.  Moreover, ML323 potentiates cisplatin cytotoxicity in non-small cell lung cancer and osteosarcoma cells.  The authors' findings point to USP1-UAF1 as a key regulator of the DNA damage response and a target for overcoming resistance to the platinum-based anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqra2Cr2SOx7bVg90H21EOLACvtfcHk0lgxy0F2DdENJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOgtrw%253D&md5=3486f59d79d15b197ea384e9df9291ee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1455%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DQ.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DRosenthal%26aufirst%3DA.%2BS.%26aulast%3DVillamil%26aufirst%3DM.%2BA.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DOtt%26aufirst%3DC.%2BA.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DLuci%26aufirst%3DD.%2BK.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DZhuang%26aufirst%3DZ.%26atitle%3DA%2520Selective%2520USP1-UAF1%2520Inhibitor%2520Links%2520Deubiquitination%2520to%2520DNA%2520Damage%2520Responses%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D298%26epage%3D304%26doi%3D10.1038%2Fnchembio.1455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turnbull, A. P.</span>; <span class="NLM_string-name">Ioannidis, S.</span>; <span class="NLM_string-name">Krajewski, W. W.</span>; <span class="NLM_string-name">Pinto-Fernandez, A.</span>; <span class="NLM_string-name">Heride, C.</span>; <span class="NLM_string-name">Martin, A. C. L.</span>; <span class="NLM_string-name">Tonkin, L. M.</span>; <span class="NLM_string-name">Townsend, E. C.</span>; <span class="NLM_string-name">Buker, S. M.</span>; <span class="NLM_string-name">Lancia, D. R.</span>; <span class="NLM_string-name">Caravella, J. A.</span>; <span class="NLM_string-name">Toms, A. V.</span>; <span class="NLM_string-name">Charlton, T. M.</span>; <span class="NLM_string-name">Lahdenranta, J.</span>; <span class="NLM_string-name">Wilker, E.</span>; <span class="NLM_string-name">Follows, B. C.</span>; <span class="NLM_string-name">Evans, N. J.</span>; <span class="NLM_string-name">Stead, L.</span>; <span class="NLM_string-name">Alli, C.</span>; <span class="NLM_string-name">Zarayskiy, V. V.</span>; <span class="NLM_string-name">Talbot, A. C.</span>; <span class="NLM_string-name">Buckmelter, A. J.</span>; <span class="NLM_string-name">Wang, M.</span>; <span class="NLM_string-name">McKinnon, C. L.</span>; <span class="NLM_string-name">Saab, F.</span>; <span class="NLM_string-name">McGouran, J. F.</span>; <span class="NLM_string-name">Century, H.</span>; <span class="NLM_string-name">Gersch, M.</span>; <span class="NLM_string-name">Pittman, M. S.</span>; <span class="NLM_string-name">Marshall, C. G.</span>; <span class="NLM_string-name">Raynham, T. M.</span>; <span class="NLM_string-name">Simcox, M.</span>; <span class="NLM_string-name">Stewart, L. M. D.</span>; <span class="NLM_string-name">McLoughlin, S. B.</span>; <span class="NLM_string-name">Escobedo, J. A.</span>; <span class="NLM_string-name">Bair, K. W.</span>; <span class="NLM_string-name">Dinsmore, C. J.</span>; <span class="NLM_string-name">Hammonds, T. R.</span>; <span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Urbe, S.</span>; <span class="NLM_string-name">Clague, M. J.</span>; <span class="NLM_string-name">Kessler, B. M.</span>; <span class="NLM_string-name">Komander, D.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis of USP7 Inhibition by Selective Small-Molecule Inhibitors</span>.  <i>Nature</i> <span class="NLM_year">2017</span>,  <span class="NLM_volume">550</span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">486</span>.<span class="refDoi"> DOI: 10.1038/nature24451</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnature24451" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=481-486&author=A.+P.+Turnbull&author=S.+Ioannidis&author=W.+W.+Krajewski&author=A.+Pinto-Fernandez&author=C.+Heride&author=A.+C.+L.+Martin&author=L.+M.+Tonkin&author=E.+C.+Townsend&author=S.+M.+Buker&author=D.+R.+Lancia&author=J.+A.+Caravella&author=A.+V.+Toms&author=T.+M.+Charlton&author=J.+Lahdenranta&author=E.+Wilker&author=B.+C.+Follows&author=N.+J.+Evans&author=L.+Stead&author=C.+Alli&author=V.+V.+Zarayskiy&author=A.+C.+Talbot&author=A.+J.+Buckmelter&author=M.+Wang&author=C.+L.+McKinnon&author=F.+Saab&author=J.+F.+McGouran&author=H.+Century&author=M.+Gersch&author=M.+S.+Pittman&author=C.+G.+Marshall&author=T.+M.+Raynham&author=M.+Simcox&author=L.+M.+D.+Stewart&author=S.+B.+McLoughlin&author=J.+A.+Escobedo&author=K.+W.+Bair&author=C.+J.+Dinsmore&author=T.+R.+Hammonds&author=S.+Kim&author=S.+Urbe&author=M.+J.+Clague&author=B.+M.+Kessler&author=D.+Komander&title=Nature"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature24451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1038%252Fnature24451%26sid%3Dliteratum%253Aachs%26aulast%3DTurnbull%26aufirst%3DA.%2BP.%26atitle%3DMolecular%2520Basis%2520of%2520USP7%2520Inhibition%2520by%2520Selective%2520Small-Molecule%2520Inhibitors%26btitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D481%26epage%3D486%26doi%3D10.1038%2Fnature24451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Z.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors Reveal Allosteric Regulation of USP14 via Steric Blockade</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1186</span>– <span class="NLM_lpage">1194</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0091-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fs41422-018-0091-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30254335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslymur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1186-1194&author=Y.+Wangauthor=Y.+Jiangauthor=S.+Dingauthor=J.+Liauthor=N.+Songauthor=Y.+Renauthor=D.+Hongauthor=C.+Wuauthor=B.+Liauthor=F.+Wangauthor=W.+Heauthor=J.+Wangauthor=Z.+Mei&title=Small+Molecule+Inhibitors+Reveal+Allosteric+Regulation+of+USP14+via+Steric+Blockade&doi=10.1038%2Fs41422-018-0091-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors reveal allosteric regulation of USP14 via steric blockade</span></div><div class="casAuthors">Wang, Yiwei; Jiang, Yuxuan; Ding, Shan; Li, Jiawang; Song, Ningjing; Ren, Yujing; Hong, Danning; Wu, Cai; Li, Bin; Wang, Feng; He, Wei; Wang, Jiawei; Mei, Ziqing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1186-1194</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The ubiquitin system is important for drug discovery, and the discovery of selective small-mol. inhibitors of deubiquitinating enzymes (DUBs) remains an active yet extremely challenging task.  With a few exceptions, previously developed inhibitors have been found to bind the evolutionarily conserved catalytic centers of DUBs, resulting in poor selectivity.  The small mol. IU1 was the first-ever specific inhibitor identified and exhibited surprisingly excellent selectivity for USP14 over other DUBs.  However, the mol. mechanism for this selectivity was elusive.  Herein, we report the high-resoln. co-crystal structures of the catalytic domain of USP14 bound to IU1 and three IU1 derivs.  All the structures of these complexes indicate that IU1 and its analogs bind to a previously unknown steric binding site in USP14, thus blocking the access of the C-terminus of ubiquitin to the active site of USP14 and abrogating USP14 activity.  Importantly, this steric site in USP14 is very unique, as suggested by structural alignments of USP14 with several known DUB X-ray structures.  These results, in conjunction with biochem. characterization, indicate a coherent steric blockade mechanism for USP14 inhibition by compds. of the IU series.  In light of the recent report of steric blockade of USP7 by FT671, this work suggests a potential generally applicable allosteric mechanism for the regulation of DUBs via steric blockade, as showcased by our discovery of IU1-248 which is 10-fold more potent than IU1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvvGynB0ABp7Vg90H21EOLACvtfcHk0litn0Oz800Yiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslymur7I&md5=21b4833424dea660f0ae4a38dcb49030</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0091-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0091-x%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DHong%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMei%26aufirst%3DZ.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520Reveal%2520Allosteric%2520Regulation%2520of%2520USP14%2520via%2520Steric%2520Blockade%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D1186%26epage%3D1194%26doi%3D10.1038%2Fs41422-018-0091-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira De Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lautens, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhakumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Identification and Structure-Activity Relationship of HDAC6 Zinc-Finger Ubiquitin Binding Domain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4517</span>– <span class="NLM_lpage">4527</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVKiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4517-4527&author=R.+Ferreira+De+Freitasauthor=R.+J.+Hardingauthor=I.+Franzoniauthor=M.+Ravichandranauthor=M.+K.+Mannauthor=H.+Ouyangauthor=M.+Lautensauthor=V.+Santhakumarauthor=C.+H.+Arrowsmithauthor=M.+Schapira&title=Identification+and+Structure-Activity+Relationship+of+HDAC6+Zinc-Finger+Ubiquitin+Binding+Domain+Inhibitors&doi=10.1021%2Facs.jmedchem.8b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Structure-Activity Relationship of HDAC6 Zinc-Finger Ubiquitin Binding Domain Inhibitors</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Harding, Rachel J.; Franzoni, Ivan; Ravichandran, Mani; Mann, Mandeep K.; Ouyang, Hui; Lautens, Mark; Santhakumar, Vijayaratnam; Arrowsmith, Cheryl H.; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4517-4527</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HDAC6 plays a central role in the recruitment of protein aggregates for lysosomal degrdn. and is a promising target for combination therapy with proteasome inhibitors in multiple myeloma.  Pharmacol. displacing ubiquitin from the zinc-finger ubiquitin-binding domain (ZnF-UBD) of HDAC6 is an underexplored alternative to catalytic inhibition.  Here, we present the discovery of an HDAC6 ZnF-UBD-focused chem. series and its progression from virtual screening hits to low micromolar inhibitors.  A carboxylate mimicking the C-terminal extremity of ubiquitin, and an extended arom. system stacking with W1182 and R1155, are necessary for activity.  One of the compds. induced a conformational remodeling of the binding site where the primary binding pocket opens up onto a ligand-able secondary pocket that may be exploited to increase potency.  The preliminary structure-activity relationship accompanied by nine crystal structures should enable further optimization into a chem. probe to investigate the merit of targeting the ZnF-UBD of HDAC6 in multiple myeloma and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6ugnzPt7hI7Vg90H21EOLACvtfcHk0litn0Oz800Yiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVKiurg%253D&md5=90a444d3428edbff5b6fe48798c58988</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00258%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2BDe%2BFreitas%26aufirst%3DR.%26aulast%3DHarding%26aufirst%3DR.%2BJ.%26aulast%3DFranzoni%26aufirst%3DI.%26aulast%3DRavichandran%26aufirst%3DM.%26aulast%3DMann%26aufirst%3DM.%2BK.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DLautens%26aufirst%3DM.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520Structure-Activity%2520Relationship%2520of%2520HDAC6%2520Zinc-Finger%2520Ubiquitin%2520Binding%2520Domain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4517%26epage%3D4527%26doi%3D10.1021%2Facs.jmedchem.8b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harding, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira De Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juarez-Ornelas, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lautens, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhakumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowmsith, C. H.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Antagonists of the Interaction between the Histone Deacetylase 6 Zinc-Finger Domain and Ubiquitin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9090</span>– <span class="NLM_lpage">9096</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00933</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00933" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1CitrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9090-9096&author=R.+J.+Hardingauthor=R.+Ferreira+De+Freitasauthor=P.+Collinsauthor=I.+Franzoniauthor=M.+Ravichandranauthor=H.+Ouyangauthor=K.+A.+Juarez-Ornelasauthor=M.+Lautensauthor=M.+Schapiraauthor=F.+Von+Delftauthor=V.+Santhakumarauthor=C.+H.+Arrowmsith&title=Small+Molecule+Antagonists+of+the+Interaction+between+the+Histone+Deacetylase+6+Zinc-Finger+Domain+and+Ubiquitin&doi=10.1021%2Facs.jmedchem.7b00933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Antagonists of the Interaction between the Histone Deacetylase 6 Zinc-Finger Domain and Ubiquitin</span></div><div class="casAuthors">Harding, Rachel J.; Ferreira de Freitas, Renato; Collins, Patrick; Franzoni, Ivan; Ravichandran, Mani; Ouyang, Hui; Juarez-Ornelas, Kevin A.; Lautens, Mark; Schapira, Matthieu; von Delft, Frank; Santhakumar, Vjayaratnam; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9090-9096</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of histone deacetylase 6 (HDAC6) have attractive therapeutic potential in numerous cancers.  HDAC6 inhibitors to date target the catalytic domains, but targeting the unique zinc-finger ubiquitin-binding domain (Zf-UBD) of HDAC6 may be an attractive alternative strategy.  Here, we developed x-ray crystallog. and biophys. assays to identify and characterize small mols. capable of binding to the Zf-UBD and compete with ubiquitin binding.  The results revealed 2 adjacent ligandable pockets of HDAC6 Zf-UBD and the 1st functional ligands for this domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-sTt_4DLZ6LVg90H21EOLACvtfcHk0lgJAn7sG9C1gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1CitrvM&md5=a65ee31124328d8f5e672cca0f11363b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00933%26sid%3Dliteratum%253Aachs%26aulast%3DHarding%26aufirst%3DR.%2BJ.%26aulast%3DFerreira%2BDe%2BFreitas%26aufirst%3DR.%26aulast%3DCollins%26aufirst%3DP.%26aulast%3DFranzoni%26aufirst%3DI.%26aulast%3DRavichandran%26aufirst%3DM.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DJuarez-Ornelas%26aufirst%3DK.%2BA.%26aulast%3DLautens%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DArrowmsith%26aufirst%3DC.%2BH.%26atitle%3DSmall%2520Molecule%2520Antagonists%2520of%2520the%2520Interaction%2520between%2520the%2520Histone%2520Deacetylase%25206%2520Zinc-Finger%2520Domain%2520and%2520Ubiquitin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9090%26epage%3D9096%26doi%3D10.1021%2Facs.jmedchem.7b00933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM—A New Method for Protein Modeling and Design: Applications to Docking and Structure Prediction from the Distorted Native Conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%E2%80%94A+New+Method+for+Protein+Modeling+and+Design%3A+Applications+to+Docking+and+Structure+Prediction+from+the+Distorted+Native+Conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0lgJAn7sG9C1gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%25E2%2580%2594A%2520New%2520Method%2520for%2520Protein%2520Modeling%2520and%2520Design%253A%2520Applications%2520to%2520Docking%2520and%2520Structure%2520Prediction%2520from%2520the%2520Distorted%2520Native%2520Conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span>; <span class="NLM_string-name">Murphy, R. B.</span>; <span class="NLM_string-name">Friesner, R. A.</span>; <span class="NLM_string-name">Beard, H. S.</span>; <span class="NLM_string-name">Frye, L. L.</span>; <span class="NLM_string-name">Pollard, W. T.</span>; <span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>.  <i>J. Med. Chem.</i> <span class="NLM_year">2004</span>,  <span class="NLM_volume">47</span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>.<span class="refDoi"> DOI: 10.1021/jm030644s</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=1750-1759&author=T.+A.+Halgren&author=R.+B.+Murphy&author=R.+A.+Friesner&author=H.+S.+Beard&author=L.+L.+Frye&author=W.+T.+Pollard&author=J.+L.+Banks&title=J.+Med.+Chem."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26btitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand Efficiency: A Useful Metric for Lead Selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+Efficiency%3A+A+Useful+Metric+for+Lead+Selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eYrkkpznPCjBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520Efficiency%253A%2520A%2520Useful%2520Metric%2520for%2520Lead%2520Selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Turcu, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span> <span> </span><span class="NLM_article-title">Recognition of Polyubiquitin Isoforms by the Multiple Ubiquitin Binding Modules of Isopeptidase T</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">19581</span>– <span class="NLM_lpage">19592</span>, <span class="refDoi"> DOI: 10.1074/jbc.M800947200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1074%2Fjbc.M800947200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=18482987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotF2ktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=19581-19592&author=F.+E.+Reyes-Turcuauthor=J.+R.+Shanksauthor=D.+Komanderauthor=K.+D.+Wilkinson&title=Recognition+of+Polyubiquitin+Isoforms+by+the+Multiple+Ubiquitin+Binding+Modules+of+Isopeptidase+T&doi=10.1074%2Fjbc.M800947200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Polyubiquitin Isoforms by the Multiple Ubiquitin Binding Modules of Isopeptidase T</span></div><div class="casAuthors">Reyes-Turcu, Francisca E.; Shanks, John R.; Komander, David; Wilkinson, Keith D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">19581-19592</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The conjugation of polyubiquitin to target proteins acts as a signal that regulates target stability, localization, and function.  Several ubiquitin binding domains have been described, and while much is known about ubiquitin binding to the isolated domains, little is known with regard to how the domains interact with polyubiquitin in the context of full-length proteins.  Isopeptidase T (IsoT/USP5) is a deubiquitinating enzyme that is largely responsible for the disassembly of unanchored polyubiquitin in the cell.  IsoT has four ubiquitin binding domains: a zinc finger domain (ZnF UBP), which binds the proximal ubiquitin, a UBP domain that forms the active site, and two ubiquitin-assocd. (UBA) domains whose roles are unknown.  Here, we show that the UBA domains are involved in binding two different polyubiquitin isoforms, linear and K48-linked.  Using isothermal titrn. calorimetry, we show that IsoT has at least four ubiquitin binding sites for both polyubiquitin isoforms.  The thermodn. of the interactions reveal that the binding is enthalpy-driven.  Mutation of the UBA domains suggests that UBA1 and UBA2 domains of IsoT interact with the third and fourth ubiquitins in both polyubiquitin isoforms, resp.  These data suggest that recognition of the polyubiquitin isoforms by IsoT involves considerable conformational mobility in the polyubiquitin ligand, in the enzyme, or in both.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouzvkXauRD4rVg90H21EOLACvtfcHk0litboDVGSf9tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotF2ktr8%253D&md5=e34bce5e2f2911f50631478d79a7b67d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M800947200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M800947200%26sid%3Dliteratum%253Aachs%26aulast%3DReyes-Turcu%26aufirst%3DF.%2BE.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26atitle%3DRecognition%2520of%2520Polyubiquitin%2520Isoforms%2520by%2520the%2520Multiple%2520Ubiquitin%2520Binding%2520Modules%2520of%2520Isopeptidase%2520T%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D19581%26epage%3D19592%26doi%3D10.1074%2Fjbc.M800947200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1007/s10822-010-9349-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1007%2Fs10822-010-9349-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20354892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=591-604&author=J.+R.+Greenwoodauthor=D.+Calkinsauthor=A.+P.+Sullivanauthor=J.+C.+Shelley&title=Towards+the+comprehensive%2C+rapid%2C+and+accurate+prediction+of+the+favorable+tautomeric+states+of+drug-like+molecules+in+aqueous+solution&doi=10.1007%2Fs10822-010-9349-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span></div><div class="casAuthors">Greenwood, Jeremy R.; Calkins, David; Sullivan, Arron P.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">591-604</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Generating the appropriate protonation states of drug-like mols. in soln. is important for success in both ligand- and structure-based virtual screening.  Screening collections of millions of compds. requires a method for detg. tautomers and their energies that is sufficiently rapid, accurate, and comprehensive.  To maximize enrichment, the lowest energy tautomers must be detd. from heterogeneous input, without over-enumerating unfavorable states.  While computationally expensive, the d. functional theory (DFT) method M06-2X/aug-cc-pVTZ(-f) [PB-SCRF] provides accurate energies for enumerated model tautomeric systems.  The empirical Hammett-Taft methodol. can very rapidly extrapolate substituent effects from model systems to drug-like mols. via the relationship between pKT and pKa.  Combining the 2 complementary approaches transforms the tautomer problem from a scientific challenge to one of engineering scale-up, and avoids issues that arise due to the very limited no. of measured pKT values, esp. for the complicated heterocycles often favored by medicinal chemists for their novelty and versatility.  Several hundreds of pre-calcd. tautomer energies and substituent pKa effects are tabulated in databases for use in structural adjustment by the program Epik, which treats tautomers as a subset of the larger problem of the protonation states in aq. ensembles and their energy penalties.  Accuracy and coverage is continually improved and expanded by parameterizing new systems of interest using DFT and exptl. data.  Recommendations are made for how to best incorporate tautomers in mol. design and virtual screening workflows.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigkzHeo8JY7Vg90H21EOLACvtfcHk0litboDVGSf9tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D&md5=1d7bc0f966ca793d6be80554868367b8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9349-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9349-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DCalkins%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DA.%2BP.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DTowards%2520the%2520comprehensive%252C%2520rapid%252C%2520and%2520accurate%2520prediction%2520of%2520the%2520favorable%2520tautomeric%2520states%2520of%2520drug-like%2520molecules%2520in%2520aqueous%2520solution%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D591%26epage%3D604%26doi%3D10.1007%2Fs10822-010-9349-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: A Software Program for PKa Prediction and Protonation State Generation for Drug-like Molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+A+Software+Program+for+PKa+Prediction+and+Protonation+State+Generation+for+Drug-like+Molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lg8LlzSQReLQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520A%2520Software%2520Program%2520for%2520PKa%2520Prediction%2520and%2520Protonation%2520State%2520Generation%2520for%2520Drug-like%2520Molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arntson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, L. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisniewski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Protein-Observed 19F-NMR for Fragment Screening, Affinity Quantification and Druggability Assessment</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1414</span>– <span class="NLM_lpage">1427</span>, <span class="refDoi"> DOI: 10.1038/nprot.2016.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnprot.2016.079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=27414758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A280%3ADC%252BC2s3gvVOmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1414-1427&author=C.+T.+Geeauthor=K.+E.+Arntsonauthor=A.+K.+Urickauthor=N.+K.+Mishraauthor=L.+M.+L.+Hawkauthor=A.+J.+Wisniewskiauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+19F-NMR+for+Fragment+Screening%2C+Affinity+Quantification+and+Druggability+Assessment&doi=10.1038%2Fnprot.2016.079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-observed (19)F-NMR for fragment screening, affinity quantification and druggability assessment</span></div><div class="casAuthors">Gee Clifford T; Arntson Keith E; Urick Andrew K; Mishra Neeraj K; Hawk Laura M L; Pomerantz William C K; Wisniewski Andrea J</div><div class="citationInfo"><span class="NLM_cas:title">Nature protocols</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1414-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">NMR spectroscopy can be used to quantify the binding affinity between proteins and low-complexity molecules, termed 'fragments'; this versatile screening approach allows researchers to assess the druggability of new protein targets.  Protein-observed (19)F-NMR (PrOF NMR) using (19)F-labeled amino acids generates relatively simple spectra that are able to provide dynamic structural information toward understanding protein folding and function.  Changes in these spectra upon the addition of fragment molecules can be observed and quantified.  This protocol describes the sequence-selective labeling of three proteins (the first bromodomains of Brd4 and BrdT, and the KIX domain of the CREB-binding protein) using commercially available fluorinated aromatic amino acids and fluorinated precursors as example applications of the method developed by our research group.  Fragment-screening approaches are discussed, as well as Kd determination, ligand-efficiency calculations and druggability assessment, i.e., the ability to target these proteins using small-molecule ligands.  Experiment times on the order of a few minutes and the simplicity of the NMR spectra obtained make this approach well-suited to the investigation of small- to medium-sized proteins, as well as the screening of multiple proteins in the same experiment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTi7QXqlmq0fkPI-fCgwosgfW6udTcc2eacss5LoSfIZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3gvVOmsw%253D%253D&md5=7fd74fe2067137a91782eab12138f1e7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.079%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%252019F-NMR%2520for%2520Fragment%2520Screening%252C%2520Affinity%2520Quantification%2520and%2520Druggability%2520Assessment%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1414%26epage%3D1427%26doi%3D10.1038%2Fnprot.2016.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lg8LlzSQReLQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lg8LlzSQReLQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">Xia2: An Expert System for Macromolecular Crystallography Data Reduction</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/S0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=Xia2%3A+An+Expert+System+for+Macromolecular+Crystallography+Data+Reduction&doi=10.1107%2FS0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0lgquS8XXqhZbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DXia2%253A%2520An%2520Expert%2520System%2520for%2520Macromolecular%2520Crystallography%2520Data%2520Reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2FS0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgquS8XXqhZbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wojdyr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, A.</span></span> <span> </span><span class="NLM_article-title">DIMPLE - a Pipeline for the Rapid Generation of Difference Maps from Protein Crystals with Putatively Bound Ligands</span>. <i>Acta Crystallogr., Sect. A: Found. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">s299</span>, <span class="refDoi"> DOI: 10.1107/S0108767313097419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0108767313097419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=23459360" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=s299&author=M.+Wojdyrauthor=R.+Keeganauthor=G.+Winterauthor=A.+Ashton&title=DIMPLE+-+a+Pipeline+for+the+Rapid+Generation+of+Difference+Maps+from+Protein+Crystals+with+Putatively+Bound+Ligands&doi=10.1107%2FS0108767313097419"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0108767313097419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767313097419%26sid%3Dliteratum%253Aachs%26aulast%3DWojdyr%26aufirst%3DM.%26aulast%3DKeegan%26aufirst%3DR.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DAshton%26aufirst%3DA.%26atitle%3DDIMPLE%2520-%2520a%2520Pipeline%2520for%2520the%2520Rapid%2520Generation%2520of%2520Difference%2520Maps%2520from%2520Protein%2520Crystals%2520with%2520Putatively%2520Bound%2520Ligands%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3Ds299%26doi%3D10.1107%2FS0108767313097419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gražulis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkys, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitkus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">AceDRG: A Stereochemical Description Generator for Ligands</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1107/S2059798317000067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS2059798317000067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=112-122&author=F.+Longauthor=R.+A.+Nichollsauthor=P.+Emsleyauthor=S.+Gra%C5%BEulisauthor=A.+Merkysauthor=A.+Vaitkusauthor=G.+N.+Murshudov&title=AceDRG%3A+A+Stereochemical+Description+Generator+for+Ligands&doi=10.1107%2FS2059798317000067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">AceDRG: a stereochemical description generator for ligands</span></div><div class="casAuthors">Long, Fei; Nicholls, Robert A.; Emsley, Paul; Grazulis, Saulius; Merkys, Andrius; Vaitkus, Antanas; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The program AceDRG is designed for the derivation of stereochem. information about small mols.  It uses local chem. and topol. environment-based atom typing to derive and organize bond lengths and angles from a small-mol. database: the Crystallog. Open Database (COD).  Information about the hybridization states of atoms, whether they belong to small rings (up to seven-membered rings), ring aromaticity and nearest-neighbor information is encoded in the atom types.  All atoms from the COD have been classified according to the generated atom types.  All bonds and angles have also been classified according to the atom types and, in a certain sense, bond types.  Derived data are tabulated in a machine-readable form that is freely available from CCP4.  AceDRG can also generate stereochem. information, provided that the basic bonding pattern of a ligand is known.  The basic bonding pattern is perceived from one of the computational chem. file formats, including SMILES, mmCIF, SDF MOL and SYBYL MOL2 files.  Using the bonding chem., atom types, and bond and angle tables generated from the COD, AceDRG derives the 'ideal' bond lengths, angles, plane groups, arom. rings and chirality information, and writes them to an mmCIF file that can be used by the refinement program REFMAC5 and the model-building program Coot.  Other refinement and model-building programs such as PHENIX and BUSTER can also use these files.  AceDRG also generates one or more coordinate sets corresponding to the most favorable conformation(s) of a given ligand.  AceDRG employs RDKit for chem. perception and for initial conformation generation, as well as for the interpretation of SMILES strings, SDF MOL and SYBYL MOL2 files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0LaF0rLfdLVg90H21EOLACvtfcHk0lgquS8XXqhZbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D&md5=0a581d62dd3b00004e37be490ee71fe8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS2059798317000067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317000067%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DGra%25C5%25BEulis%26aufirst%3DS.%26aulast%3DMerkys%26aufirst%3DA.%26aulast%3DVaitkus%26aufirst%3DA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAceDRG%253A%2520A%2520Stereochemical%2520Description%2520Generator%2520for%2520Ligands%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D112%26epage%3D122%26doi%3D10.1107%2FS2059798317000067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljQf48J36QTGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr., Sect D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0ljQf48J36QTGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr. Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljQf48J36QTGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, K. J.</span>; <span class="NLM_string-name">Chow, D. E.</span>; <span class="NLM_string-name">Xu, H.</span>; <span class="NLM_string-name">Dror, R. O.</span>; <span class="NLM_string-name">Eastwood, M. P.</span>; <span class="NLM_string-name">Gregersen, B. A.</span>; <span class="NLM_string-name">Klepeis, J. L.</span>; <span class="NLM_string-name">Kolossvary, I.</span>; <span class="NLM_string-name">Moraes, M. A.</span>; <span class="NLM_string-name">Sacerdoti, F. D.</span>; <span class="NLM_string-name">Salmon, J. K.</span>; <span class="NLM_string-name">Shan, Y.</span>; <span class="NLM_string-name">Shaw, D. E.</span></span> In  <i>Molecular Dynamics---Scalable Algorithms for Molecular Dynamics Simulations on Commodity Clusters</i>, Proceedings of the 2006 ACM/IEEE Conference on Supercomputing - SC ’06, <span class="NLM_year">2006</span>; p  <span class="NLM_fpage">84</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=84&author=K.+J.+Bowers&author=D.+E.+Chow&author=H.+Xu&author=R.+O.+Dror&author=M.+P.+Eastwood&author=B.+A.+Gregersen&author=J.+L.+Klepeis&author=I.+Kolossvary&author=M.+A.+Moraes&author=F.+D.+Sacerdoti&author=J.+K.+Salmon&author=Y.+Shan&author=D.+E.+Shaw&title=Molecular+Dynamics%2D%2D-Scalable+Algorithms+for+Molecular+Dynamics+Simulations+on+Commodity+Clusters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DK.%2BJ.%26jtitle%3DMolecular%2520Dynamics---Scalable%2520Algorithms%2520for%2520Molecular%2520Dynamics%2520Simulations%2520on%2520Commodity%2520Clusters%26date%3D2006%26spage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felts, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillip, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallicchio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. M.</span></span> <span> </span><span class="NLM_article-title">Integrated Modeling Program, Applied Chemical Theory (IMPACT)</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1780</span>, <span class="refDoi"> DOI: 10.1002/jcc.20292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1002%2Fjcc.20292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=16211539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1752-1780&author=J.+L.+Banksauthor=H.+S.+Beardauthor=Y.+Caoauthor=A.+E.+Choauthor=W.+Dammauthor=R.+Faridauthor=A.+K.+Feltsauthor=T.+A.+Halgrenauthor=D.+T.+Mainzauthor=J.+R.+Mapleauthor=R.+Murphyauthor=D.+M.+Phillipauthor=M.+P.+Repaskyauthor=L.+Y.+Zhangauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=E.+Gallicchioauthor=R.+M.+Levy&title=Integrated+Modeling+Program%2C+Applied+Chemical+Theory+%28IMPACT%29&doi=10.1002%2Fjcc.20292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated modeling program, applied chemical theory (IMPACT)</span></div><div class="casAuthors">Banks, Jay L.; Beard, Hege S.; Cao, Yixiang; Cho, Art E.; Damm, Wolfgang; Farid, Ramy; Felts, Anthony K.; Halgren, Thomas A.; Mainz, Daniel T.; Maple, Jon R.; Murphy, Robert; Philipp, Dean M.; Repasky, Matthew P.; Zhang, Linda Y.; Berne, Bruce J.; Friesner, Richard A.; Gallicchio, Emilio; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1752-1780</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  We provide an overview of the IMPACT mol. mechanics program with an emphasis on recent developments and a description of its current functionality.  With respect to core mol. mechanics technologies we include a status report for the fixed charge and polarizable force fields that can be used with the program and illustrate how the force fields, when used together with new atom typing and parameter assignment modules, have greatly expanded the coverage of org. compds. and medicinally relevant ligands.  As we discuss in this review, explicit solvent simulations have been used to guide our design of implicit solvent models based on the generalized Born framework and a novel nonpolar estimator that have recently been incorporated into the program.  With IMPACT it is possible to use several different advanced conformational sampling algorithms based on combining features of mol. dynamics and Monte Carlo simulations.  The program includes two specialized mol. mechanics modules: Glide, a high-throughput docking program, and QSite, a mixed quantum mechanics/mol. mechanics module.  These modules employ the IMPACT infrastructure as a starting point for the construction of the protein model and assignment of mol. mechanics parameters, but have then been developed to meet specialized objectives with respect to sampling and the energy function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZX2FzcGWGM7Vg90H21EOLACvtfcHk0lhFAIq04rfV-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI&md5=0094f15727c7fb77a0fae16ccf8292f0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20292%26sid%3Dliteratum%253Aachs%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DA.%2BE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFelts%26aufirst%3DA.%2BK.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DMaple%26aufirst%3DJ.%2BR.%26aulast%3DMurphy%26aufirst%3DR.%26aulast%3DPhillip%26aufirst%3DD.%2BM.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DZhang%26aufirst%3DL.%2BY.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DIntegrated%2520Modeling%2520Program%252C%2520Applied%2520Chemical%2520Theory%2520%2528IMPACT%2529%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1752%26epage%3D1780%26doi%3D10.1002%2Fjcc.20292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. H.</span></span> <span> </span><span class="NLM_article-title">Efficient Estimation of Free Energy Differences from Monte Carlo Data</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(76)90078-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2F0021-9991%2876%2990078-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1976&pages=245-268&author=C.+H.+Bennett&title=Efficient+Estimation+of+Free+Energy+Differences+from+Monte+Carlo+Data&doi=10.1016%2F0021-9991%2876%2990078-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2876%2990078-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252876%252990078-4%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DC.%2BH.%26atitle%3DEfficient%2520Estimation%2520of%2520Free%2520Energy%2520Differences%2520from%2520Monte%2520Carlo%2520Data%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1976%26volume%3D22%26spage%3D245%26epage%3D268%26doi%3D10.1016%2F0021-9991%2876%2990078-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paliwal, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span></span> <span> </span><span class="NLM_article-title">A Benchmark Test Set for Alchemical Free Energy Transformations and Its Use to Quantify Error in Common Free Energy Methods</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">4115</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1021/ct2003995</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct2003995" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=4115-4134&author=H.+Paliwalauthor=M.+R.+Shirts&title=A+Benchmark+Test+Set+for+Alchemical+Free+Energy+Transformations+and+Its+Use+to+Quantify+Error+in+Common+Free+Energy+Methods&doi=10.1021%2Fct2003995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Benchmark Test Set for Alchemical Free Energy Transformations and Its Use to Quantify Error in Common Free Energy Methods</span></div><div class="casAuthors">Paliwal, Himanshu; Shirts, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4115-4134</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a significant need for improved tools to validate thermophys. quantities computed via mol. simulation.  In this paper we present the initial version of a benchmark set of testing methods for calcg. free energies of mol. transformation in soln.  This set is based on mol. changes common to many mol. design problems, such as insertion and deletion of at. sites and changing at. partial charges.  We use this benchmark set to compare the statistical efficiency, reliability, and quality of uncertainty ests. for a no. of published free energy methods, including thermodn. integration, free energy perturbation, the Bennett acceptance ratio (BAR) and its multistate equiv. MBAR.  We identify MBAR as the consistently best performing method, though other methods are frequently comparable in reliability and accuracy in many cases.  We demonstrate that assumptions of Gaussian distributed errors in free energies are usually valid for most methods studied.  We demonstrate that bootstrap error estn. is a robust and useful technique for estg. statistical variance for all free energy methods studied.  This benchmark set is provided in a no. of different file formats with the hope of becoming a useful and general tool for method comparisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxxPO-qPI-gLVg90H21EOLACvtfcHk0lhFAIq04rfV-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM&md5=41383373300216165a7f1724ad535451</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fct2003995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct2003995%26sid%3Dliteratum%253Aachs%26aulast%3DPaliwal%26aufirst%3DH.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DA%2520Benchmark%2520Test%2520Set%2520for%2520Alchemical%2520Free%2520Energy%2520Transformations%2520and%2520Its%2520Use%2520to%2520Quantify%2520Error%2520in%2520Common%2520Free%2520Energy%2520Methods%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D4115%26epage%3D4134%26doi%3D10.1021%2Fct2003995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidhoff, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulber, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodendorf, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorthiois, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martoglio, B.</span></span> <span> </span><span class="NLM_article-title">A Sensitive Fluorescence Intensity Assay for Deubiquitinating Proteases Using Ubiquitin-Rhodamine110-Glycine as Substrate</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2007.07.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.ab.2007.07.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=17869210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1GlurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=201-207&author=U.+Hassiepenauthor=U.+Eidhoffauthor=G.+Mederauthor=J.+F.+Bulberauthor=A.+Heinauthor=U.+Bodendorfauthor=E.+Lorthioisauthor=B.+Martoglio&title=A+Sensitive+Fluorescence+Intensity+Assay+for+Deubiquitinating+Proteases+Using+Ubiquitin-Rhodamine110-Glycine+as+Substrate&doi=10.1016%2Fj.ab.2007.07.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A sensitive fluorescence intensity assay for deubiquitinating proteases using ubiquitin-rhodamine110-glycine as substrate</span></div><div class="casAuthors">Hassiepen, Ulrich; Eidhoff, Ulf; Meder, Gabriele; Bulber, Jean-Francois; Hein, Andreas; Bodendorf, Ursula; Lorthiois, Edwige; Martoglio, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-207</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The dynamic modification of proteins with ubiquitin is a key regulation paradigm in eukaryotic cells that controls stability, localization, and function of the vast majority of intracellular proteins.  Here we describe a robust fluorescence intensity assay for monitoring the enzymic activity of deubiquitinating proteases, which reverse ubiquitin modifications and comprise over 100 members in humans.  The assay was developed for the catalytic domain of human ubiquitin-specific protease 2 (USP2) and human ubiquitin carboxyterminal hydrolase L3 (UCH-L3), and makes use of the novel substrate ubiquitin-rhodamine110-glycine.  The latter combines the advantages of a high dynamic range and beneficial optical properties.  Its enzymic behavior is characterized by the kinetic consts. Km = 1.5 μM, kcat = 0.53 s-1 and kcat/Km = 3.5×105 M-1 s-1 for USP2 and Km = 34 nM, kcat = 4.72 s-1, and kcat/Km = 1.4×108 M-1 s-1 for UCH-L3.  This new assay is suitable for inhibitor screening and characterizations, and has been established for the 384-well plate format using protease concns. of 120 pM for USP2 and 1 pM for UCH-L3 and substrate concns. of 100 nM for both enzymes.  Due to the low protease concns. and high sensitivity, this assay would allow the detn. of inhibitory consts. in the subnanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM2_cU_l6AT7Vg90H21EOLACvtfcHk0lhFAIq04rfV-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1GlurjK&md5=e9a430d206d99d4b8bef17030139e61a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2007.07.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2007.07.034%26sid%3Dliteratum%253Aachs%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DEidhoff%26aufirst%3DU.%26aulast%3DMeder%26aufirst%3DG.%26aulast%3DBulber%26aufirst%3DJ.%2BF.%26aulast%3DHein%26aufirst%3DA.%26aulast%3DBodendorf%26aufirst%3DU.%26aulast%3DLorthiois%26aufirst%3DE.%26aulast%3DMartoglio%26aufirst%3DB.%26atitle%3DA%2520Sensitive%2520Fluorescence%2520Intensity%2520Assay%2520for%2520Deubiquitinating%2520Proteases%2520Using%2520Ubiquitin-Rhodamine110-Glycine%2520as%2520Substrate%26jtitle%3DAnal.%2520Biochem.%26date%3D2007%26volume%3D371%26spage%3D201%26epage%3D207%26doi%3D10.1016%2Fj.ab.2007.07.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Characterization of USP5 ZnF-UBD interactions: (a) crystal structure of USP5 ZnF-UBD (grey) in complex with a C-terminal ubiquitin peptide (cyan) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45">2G45</a>). The Trp side chains of USP5 are highlighted. (b) <sup>19</sup>F NMR spectra of 5FW-USP5 ZnF-UBD before (blue) and after the addition of LRLRGG ubiquitin peptide (red) (at 6-fold excess) (c) <sup>19</sup>F NMR spectra of 5FW-USP5 ZnF-UBD before (blue) and after the addition of compound <b>5</b> (at 25-fold excess) (red) (d) SPR sensorgram and binding curve of LRLRGG ubiquitin peptide (left) and compound <b>5</b> (right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of USP5 ZnF-UBD in complex with (a) C-terminal ubiquitin peptide (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45">2G45</a>) (b) <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXT">6DXT</a>), (c) <b>5</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT">6NFT</a>), and (d) <b>7</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>). Binding pocket residues (grey), ligands (cyan). Hydrogen bonds are shown as dotted green lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Superimposed crystal structure of USP5 ZnF-UBD (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G43">2G43</a>) (grey) in the apo state and bound to <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXT">6DXT</a>) (magenta), <b>5</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT">6NFT</a>) (green), and <b>7</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>) (blue) highlighting the shifted loop in the compound <b>7</b> complex structure. (b) Surface representation of USP5 ZnF-UBD (grey) in complex with <b>5</b> and <b>7</b> (purple) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT">6NFT</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH">6DXH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of USP5 ZnF-UBD (grey) in complex with <b>21</b> (cyan). A methyl group optimally exploiting a buried hydrophobic cavity is highlighted (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P9G">6P9G</a>). The surface of the binding pocket is colored by binding properties: blue: hydrogen-bond donor, red: hydrogen-bond acceptor, grey: hydrophobic.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/medium/jm9b00988_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representative SPR sensorgram and binding curve of compound <b>21</b> with (a) USP5 ZnF-UBD (b) WT FL USP5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b00988/20191122/images/large/jm9b00988_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00988&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65476" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65476" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Varshavsky, A.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin System</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">383</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1016/S0968-0004(97)01122-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2FS0968-0004%2897%2901122-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=9357313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK2sXntFeltLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1997&pages=383-387&author=A.+Varshavsky&title=The+Ubiquitin+System&doi=10.1016%2FS0968-0004%2897%2901122-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin system</span></div><div class="casAuthors">Varshavsky, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">383-387</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 70 refs.  Eukaryotes contain a highly conserved multi-enzyme system that covalently links ubiquitin to a variety of intracellular proteins that bear degrdn. signals recognized by this system.  The resulting ubiquitin-protein conjugates are degraded by the 26S proteasome, a large ATP-dependent protease.  Pathways that involve ubiquitin underlie a multitude of processes, including cell differentiation, the cell cycle and responses to stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALosaVY0BRbVg90H21EOLACvtfcHk0liBGHdlpWgx-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXntFeltLY%253D&md5=61205ba96af5c0d89c028120d609cc52</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0968-0004%2897%2901122-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0004%252897%252901122-5%26sid%3Dliteratum%253Aachs%26aulast%3DVarshavsky%26aufirst%3DA.%26atitle%3DThe%2520Ubiquitin%2520System%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1997%26volume%3D22%26spage%3D383%26epage%3D387%26doi%3D10.1016%2FS0968-0004%2897%2901122-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Ubiquitin-Specific Proteases as a Novel Anticancer Therapeutic Strategy</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1080</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.01080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.3389%2Ffphar.2018.01080" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2018&pages=1080&author=T.+Yuanauthor=F.+Yanauthor=M.+Yingauthor=J.+Caoauthor=Q.+Heauthor=H.+Zhuauthor=B.+Yang&title=Inhibition+of+Ubiquitin-Specific+Proteases+as+a+Novel+Anticancer+Therapeutic+Strategy&doi=10.3389%2Ffphar.2018.01080"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01080%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DYing%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DB.%26atitle%3DInhibition%2520of%2520Ubiquitin-Specific%2520Proteases%2520as%2520a%2520Novel%2520Anticancer%2520Therapeutic%2520Strategy%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D27%26spage%3D1080%26doi%3D10.3389%2Ffphar.2018.01080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Turcu, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ventii, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span> <span> </span><span class="NLM_article-title">Regulation and Cellular Roles of Ubiquitin-Specific Deubiquitinating Enzymes</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">397</span>, <span class="refDoi"> DOI: 10.1146/annurev.biochem.78.082307.091526</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1146%2Fannurev.biochem.78.082307.091526" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=19489724" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD1MXos1Ghtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2009&pages=363-397&author=F.+E.+Reyes-Turcuauthor=K.+H.+Ventiiauthor=K.+D.+Wilkinson&title=Regulation+and+Cellular+Roles+of+Ubiquitin-Specific+Deubiquitinating+Enzymes&doi=10.1146%2Fannurev.biochem.78.082307.091526"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and cellular roles of ubiquitin-specific deubiquitinating enzymes</span></div><div class="casAuthors">Reyes-Turcu, Francisca E.; Ventii, Karen H.; Wilkinson, Keith D.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">363-397</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Deubiquitinating enzymes (DUBs; deubiquitinases) are proteases that process ubiquitin or ubiquitin-like gene products, reverse the modification of proteins by a single ubiquitin(-like) protein, and remodel polyubiquitin(-like) chains on target proteins.  The human genome encodes nearly 100 DUBs with specificity for ubiquitin in 5 gene families.  Most DUB activity is cryptic, and conformational rearrangements often occur during the binding of ubiquitin and/or scaffold proteins.  DUBs with specificity for ubiquitin contain insertions and extensions modulating DUB substrate specificity, protein-protein interactions, and cellular localization.  Binding partners and multiprotein complexes with which DUBs assoc. modulate DUB activity and substrate specificity.  Quant. studies of activity and protein-protein interactions, together with genetic studies and the advent of RNAi, have led to new insights into the function of yeast and human DUBs.  Here, the authors discuss ubiquitin-specific DUBs, some of the generalizations emerging from recent studies of the regulation of DUB activity, and their roles in various cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzNm7SqfgcRLVg90H21EOLACvtfcHk0liBGHdlpWgx-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXos1Ghtbc%253D&md5=7e7248a63b46c5dbc7cf333d8d406d90</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.78.082307.091526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.78.082307.091526%26sid%3Dliteratum%253Aachs%26aulast%3DReyes-Turcu%26aufirst%3DF.%2BE.%26aulast%3DVentii%26aufirst%3DK.%2BH.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26atitle%3DRegulation%2520and%2520Cellular%2520Roles%2520of%2520Ubiquitin-Specific%2520Deubiquitinating%2520Enzymes%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2009%26volume%3D78%26spage%3D363%26epage%3D397%26doi%3D10.1146%2Fannurev.biochem.78.082307.091526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clague, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heride, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbé, S.</span></span> <span> </span><span class="NLM_article-title">The Demographics of the Ubiquitin System</span>. <i>Trends Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1016/j.tcb.2015.03.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.tcb.2015.03.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=25906909" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVGqtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=417-426&author=M.+J.+Clagueauthor=C.+Herideauthor=S.+Urb%C3%A9&title=The+Demographics+of+the+Ubiquitin+System&doi=10.1016%2Fj.tcb.2015.03.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The demographics of the ubiquitin system</span></div><div class="casAuthors">Clague, Michael J.; Heride, Claire; Urbe, Sylvie</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">417-426</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The ubiquitin system is a major coordinator of cellular physiol. through regulation of both protein degrdn. and signalling pathways.  A key building block of a systems-level understanding has been generated by global proteomic studies, which provide copy no. ests. for each component.  The aggregate of ubiquitin, conjugating enzymes (E1, E2, and E3s), and deubiquitylases (DUBs) represents ∼1.3% of total cellular protein.  Complementary approaches have generated quant. measurements of various ubiquitin pools and further subdivision into different ubiquitin chain topologies.  Systematic studies aimed at assocg. specific enzymes (E2s and DUBs) with the dynamics of these different pools have also made significant progress.  Here, we delineate the emerging picture of the most significant determinants of the cellular ubiquitin economy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZQMwzCvXv5bVg90H21EOLACvtfcHk0liUJoW7OOFH4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVGqtbw%253D&md5=38563c26b5d83eab2bb0dc193041d1e5</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2015.03.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2015.03.002%26sid%3Dliteratum%253Aachs%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DHeride%26aufirst%3DC.%26aulast%3DUrb%25C3%25A9%26aufirst%3DS.%26atitle%3DThe%2520Demographics%2520of%2520the%2520Ubiquitin%2520System%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2015%26volume%3D25%26spage%3D417%26epage%3D426%26doi%3D10.1016%2Fj.tcb.2015.03.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Swatek, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin Modifications</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">399</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1038/cr.2016.39</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fcr.2016.39" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=27012465" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC28XltFGmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=399-422&author=K.+N.+Swatekauthor=D.+Komander&title=Ubiquitin+Modifications&doi=10.1038%2Fcr.2016.39"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin modifications</span></div><div class="casAuthors">Swatek, Kirby N.; Komander, David</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">399-422</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination is a dynamic multifaceted post-translational modification involved in nearly all aspects of eukaryotic biol.  Once attached to a substrate, the 76-amino acid protein ubiquitin is subjected to further modifications, creating a multitude of distinct signals with distinct cellular outcomes, referred to as the 'ubiquitin code'.  Ubiquitin can be ubiquitinated on seven lysine (Lys) residues or on the N-terminus, leading to polyubiquitin chains that can encompass complex topologies.  Alternatively or in addn., ubiquitin Lys residues can be modified by ubiquitin-like mols. (such as SUMO or NEDD8).  Finally, ubiquitin can also be acetylated on Lys, or phosphorylated on Ser, Thr or Tyr residues, and each modification has the potential to dramatically alter the signaling outcome.  While the no. of distinctly modified ubiquitin species in cells is mind-boggling, much progress has been made to characterize the roles of distinct ubiquitin modifications, and many enzymes and receptors have been identified that create, recognize or remove these ubiquitin modifications.  We here provide an overview of the various ubiquitin modifications present in cells, and highlight recent progress on ubiquitin chain biol.  We then discuss the recent findings in the field of ubiquitin acetylation and phosphorylation, with a focus on Ser65-phosphorylation and its role in mitophagy and Parkin activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSk-y3UHKix7Vg90H21EOLACvtfcHk0liUJoW7OOFH4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XltFGmsbg%253D&md5=0a4538fe56b01c700e8392f397084db8</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcr.2016.39&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2016.39%26sid%3Dliteratum%253Aachs%26aulast%3DSwatek%26aufirst%3DK.%2BN.%26aulast%3DKomander%26aufirst%3DD.%26atitle%3DUbiquitin%2520Modifications%26jtitle%3DCell%2520Res.%26date%3D2016%26spage%3D399%26epage%3D422%26doi%3D10.1038%2Fcr.2016.39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rape, M.</span></span> <span> </span><span class="NLM_article-title">The Increasing Complexity of the Ubiquitin Code</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">579</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1038/ncb3358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fncb3358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=27230526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC28Xosl2it7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=579-586&author=R.+Yauauthor=M.+Rape&title=The+Increasing+Complexity+of+the+Ubiquitin+Code&doi=10.1038%2Fncb3358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The increasing complexity of the ubiquitin code</span></div><div class="casAuthors">Yau, Richard; Rape, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">579-586</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Ubiquitylation is essential for signal transduction as well as cell division and differentiation in all eukaryotes.  Substrate modifications range from a single ubiquitin mol. to complex polymeric chains, with different types of ubiquitylation often eliciting distinct outcomes.  The recent identification of novel chain topologies has improved our understanding of how ubiquitylation establishes precise communication within cells.  Here, we discuss how the increasing complexity of ubiquitylation is employed to ensure robust and faithful signal transduction in eukaryotic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod2OUBSjtqwLVg90H21EOLACvtfcHk0liUJoW7OOFH4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xosl2it7k%253D&md5=2223ea6ecf6d7fa869b9ee064444ff8b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncb3358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb3358%26sid%3Dliteratum%253Aachs%26aulast%3DYau%26aufirst%3DR.%26aulast%3DRape%26aufirst%3DM.%26atitle%3DThe%2520Increasing%2520Complexity%2520of%2520the%2520Ubiquitin%2520Code%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2016%26volume%3D18%26spage%3D579%26epage%3D586%26doi%3D10.1038%2Fncb3358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walczak, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dikic, I.</span></span> <span> </span><span class="NLM_article-title">Generation and Physiological Roles of Linear Ubiquitin Chains</span>. <i>BMC Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">23</span>, <span class="refDoi"> DOI: 10.1186/1741-7007-10-23</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1186%2F1741-7007-10-23" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=22420778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlsl2rs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=23&author=H.+Walczakauthor=K.+Iwaiauthor=I.+Dikic&title=Generation+and+Physiological+Roles+of+Linear+Ubiquitin+Chains&doi=10.1186%2F1741-7007-10-23"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Generation and physiological roles of linear ubiquitin chains</span></div><div class="casAuthors">Walczak, Henning; Iwai, Kazuhiro; Dikic, Ivan</div><div class="citationInfo"><span class="NLM_cas:title">BMC Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23</span>CODEN:
                <span class="NLM_cas:coden">BBMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1741-7007</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Ubiquitination now ranks with phosphorylation as one of the best-studied post-translational modifications of proteins with broad regulatory roles across all of biol.  Ubiquitination usually involves the addn. of ubiquitin chains to target protein mols., and these may be of eight different types, seven of which involve the linkage of one of the seven internal lysine (K) residues in one ubiquitin mol. to the carboxy-terminal diglycine of the next.  In the eighth, the so-called linear ubiquitin chains, the linkage is between the amino-terminal amino group of methionine on a ubiquitin that is conjugated with a target protein and the carboxy-terminal carboxy group of the incoming ubiquitin.  Physiol. roles are well established for K48-linked chains, which are essential for signaling proteasomal degrdn. of proteins, and for K63-linked chains, which play a part in recruitment of DNA repair enzymes, cell signaling and endocytosis.  We focus here on linear ubiquitin chains, how they are assembled, and how three different avenues of research have indicated physiol. roles for linear ubiquitination in innate and adaptive immunity and suppression of inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUfVgvYr8WNrVg90H21EOLACvtfcHk0liUJoW7OOFH4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlsl2rs7g%253D&md5=d6c6d5add1b55eeda621aff62f16c44d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1741-7007-10-23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1741-7007-10-23%26sid%3Dliteratum%253Aachs%26aulast%3DWalczak%26aufirst%3DH.%26aulast%3DIwai%26aufirst%3DK.%26aulast%3DDikic%26aufirst%3DI.%26atitle%3DGeneration%2520and%2520Physiological%2520Roles%2520of%2520Linear%2520Ubiquitin%2520Chains%26jtitle%3DBMC%2520Biol.%26date%3D2012%26volume%3D10%26spage%3D23%26doi%3D10.1186%2F1741-7007-10-23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tashayev, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickart, C. M.</span></span> <span> </span><span class="NLM_article-title">Metabolism of the Polyubiquitin Degradation Signal: Structure, Mechanism, and Role of Isopeptidase T</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">14535</span>– <span class="NLM_lpage">14546</span>, <span class="refDoi"> DOI: 10.1021/bi00044a032</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00044a032" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK2MXoslGhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=14535-14546&author=K.+D.+Wilkinsonauthor=V.+L.+Tashayevauthor=L.+B.+O%E2%80%99Connorauthor=C.+N.+Larsenauthor=E.+Kasperekauthor=C.+M.+Pickart&title=Metabolism+of+the+Polyubiquitin+Degradation+Signal%3A+Structure%2C+Mechanism%2C+and+Role+of+Isopeptidase+T&doi=10.1021%2Fbi00044a032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of the polyubiquitin degradation signal: structure, mechanism, and role of isopeptidase T</span></div><div class="casAuthors">Wilkinson, Keith D.; Tashayev, Valery L.; O'Connor, Lydia B.; Larsen, Christopher N.; Kasperek, Eileen; Pickart, Cecile M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14535-46</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A necessary step in ubiquitin-dependent proteolysis is the addn. of a polyubiquitin chain to the target protein.  This ubiquitinated protein is degraded by a multi-subunit complex known as the 26S proteasome.  The polyubiquitin chain is probably not released until a late stage in the proteolysis by the proteasome.  It is subsequently disassembled to yield functional ubiquitin monomers.  Here the authors present evidence that a 93 kDa protein, isopeptidase T, has the properties expected for the enzyme which disassembles these branched polyubiquitin chains.  Protein and cDNA sequencing revealed that isopeptidase T is a member of the ubiquitin specific protease family (UBP).  Isopeptidase T disassembles branched polyubiquitin chains (linked by the G76-K48 isopeptide bond) by a sequential exo mechanism, starting at the proximal end of the chain (the proximal ubiquitin contains a free carboxyl-terminus).  Isopeptidase T prefers to disassemble chains in which there is an intact and unblocked RGG sequence at the C-terminus of the proximal subunit.  Rates of disassembly are reduced when G76 of the proximal ubiquitin is modified, for example, by ligation to substrate protein, by esterification, by replacement of the proximal glycine with alanine (G76A), or by truncation.  Linear proubiquitin is only a poor substrate.  Obsd. rates and specificity are consistent with isopeptidase T playing a major role in disassembly of polyubiquitin chains.  The high discrimination against chains that are blocked or modified at the proximal end indicates that the enzyme acts after release of the chains from conjugated proteins or degrdn. intermediates.  Thus, the proteolytic degrdn. signal is not disassembled by isopeptidase T before the ubiquitinated protein is degraded.  These (and earlier) results suggest that UBP isoenzymes may exhibit significant substrate specificity, consistent with a role in the regulated catabolism of the polymeric ubiquitin, including the polyubiquitin protein degrdn. signal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWWP5m1sxSw7Vg90H21EOLACvtfcHk0ljy71w_OrVfOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXoslGhs7s%253D&md5=cdad0da4f8b78836524cc8656a360378</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi00044a032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00044a032%26sid%3Dliteratum%253Aachs%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26aulast%3DTashayev%26aufirst%3DV.%2BL.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DL.%2BB.%26aulast%3DLarsen%26aufirst%3DC.%2BN.%26aulast%3DKasperek%26aufirst%3DE.%26aulast%3DPickart%26aufirst%3DC.%2BM.%26atitle%3DMetabolism%2520of%2520the%2520Polyubiquitin%2520Degradation%2520Signal%253A%2520Structure%252C%2520Mechanism%252C%2520and%2520Role%2520of%2520Isopeptidase%2520T%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D14535%26epage%3D14546%26doi%3D10.1021%2Fbi00044a032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Welchman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, R. J.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin and Ubiquitin-like Proteins as Multifunctional Signals</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1038/nrm1700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnrm1700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=16064136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=599-609&author=R.+L.+Welchmanauthor=C.+Gordonauthor=R.+J.+Mayer&title=Ubiquitin+and+Ubiquitin-like+Proteins+as+Multifunctional+Signals&doi=10.1038%2Fnrm1700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin and ubiquitin-like proteins as multifunctional signals</span></div><div class="casAuthors">Welchman, Rebecca L.; Gordon, Colin; Mayer, R. John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">599-609</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination is a recognized signal for protein degrdn.  However, it is increasingly realized that ubiquitin conjugation to proteins can be used for many other purposes.  Furthermore, there are many ubiquitin-like proteins that control the activities of proteins.  The central structural element of these post-translational modifications is the ubiquitin superfold.  A common ancestor based on this superfold has evolved to give various proteins that are involved in diverse activities in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr72A1XAR84W7Vg90H21EOLACvtfcHk0ljy71w_OrVfOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D&md5=25f0eb2aec7ffb99b78034da9f81f3f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrm1700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1700%26sid%3Dliteratum%253Aachs%26aulast%3DWelchman%26aufirst%3DR.%2BL.%26aulast%3DGordon%26aufirst%3DC.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26atitle%3DUbiquitin%2520and%2520Ubiquitin-like%2520Proteins%2520as%2520Multifunctional%2520Signals%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D599%26epage%3D609%26doi%3D10.1038%2Fnrm1700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clague, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbé, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span> <span> </span><span class="NLM_article-title">Breaking the Chains: Deubiquitylating Enzyme Specificity Begets Function</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">338</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/s41580-019-0099-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fs41580-019-0099-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30733604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmtlWlt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=338-352&author=M.+J.+Clagueauthor=S.+Urb%C3%A9author=D.+Komander&title=Breaking+the+Chains%3A+Deubiquitylating+Enzyme+Specificity+Begets+Function&doi=10.1038%2Fs41580-019-0099-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking the chains: deubiquitylating enzyme specificity begets function</span></div><div class="casAuthors">Clague, Michael J.; Urbe, Sylvie; Komander, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">338-352</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The deubiquitylating enzymes (DUBs, also known as deubiquitylases or deubiquitinases) maintain the dynamic state of the cellular ubiquitome by releasing conjugated ubiquitin from proteins.  In light of the many cellular functions of ubiquitin, DUBs occupy key roles in almost all aspects of cell behavior.  Many DUBs show selectivity for particular ubiquitin linkage types or positions within ubiquitin chains.  Others show chain-type promiscuity but can select a distinct palette of protein substrates via specific protein-protein interactions established through binding modules outside of the catalytic domain.  The ubiquitin chain cleavage mode or chain linkage specificity has been related directly to biol. functions.  Examples include regulation of protein degrdn. and ubiquitin recycling by the proteasome, DNA repair pathways and innate immune signalling.  DUB cleavage specificity is also being harnessed for anal. of ubiquitin chain architecture that is assembled on specific proteins.  The recent development of highly specific DUB inhibitors heralds their emergence as a new class of therapeutic targets for numerous diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZMzX1ZpHBPrVg90H21EOLACvtfcHk0ljy71w_OrVfOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmtlWlt74%253D&md5=616b04e41e4184608c22b86bbcd328b5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fs41580-019-0099-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41580-019-0099-1%26sid%3Dliteratum%253Aachs%26aulast%3DClague%26aufirst%3DM.%2BJ.%26aulast%3DUrb%25C3%25A9%26aufirst%3DS.%26aulast%3DKomander%26aufirst%3DD.%26atitle%3DBreaking%2520the%2520Chains%253A%2520Deubiquitylating%2520Enzyme%2520Specificity%2520Begets%2520Function%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2019%26volume%3D20%26spage%3D338%26epage%3D352%26doi%3D10.1038%2Fs41580-019-0099-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Turcu, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullally, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heroux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span> <span> </span><span class="NLM_article-title">The Ubiquitin Binding Domain ZnF UBP Recognizes the C-Terminal Diglycine Motif of Unanchored Ubiquitin</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">1197</span>– <span class="NLM_lpage">1208</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2006.02.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.cell.2006.02.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=16564012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD28Xjt1KhsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=1197-1208&author=F.+E.+Reyes-Turcuauthor=J.+R.+Hortonauthor=J.+E.+Mullallyauthor=A.+Herouxauthor=X.+Chengauthor=K.+D.+Wilkinson&title=The+Ubiquitin+Binding+Domain+ZnF+UBP+Recognizes+the+C-Terminal+Diglycine+Motif+of+Unanchored+Ubiquitin&doi=10.1016%2Fj.cell.2006.02.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The ubiquitin binding domain ZnF UBP recognizes the C-terminal diglycine motif of unanchored ubiquitin</span></div><div class="casAuthors">Reyes-Turcu, Francisca E.; Horton, John R.; Mullally, James E.; Heroux, Annie; Cheng, Xiaodong; Wilkinson, Keith D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1197-1208</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ubiquitin binding proteins regulate the stability, function, and/or localization of ubiquitinated proteins.  Here we report the crystal structures of the zinc-finger ubiquitin binding domain (ZnF UBP) from the deubiquitinating enzyme isopeptidase T (IsoT, or USP5) alone and in complex with ubiquitin.  Unlike other ubiquitin binding domains, this domain contains a deep binding pocket where the C-terminal diglycine motif of ubiquitin is inserted, thus explaining the specificity of IsoT for an unmodified C terminus on the proximal subunit of polyubiquitin.  Mutations in the domain demonstrate that it is required for optimal catalytic activation of IsoT.  This domain is present in several other protein families, and the ZnF UBP domain from an E3 ligase also requires the C terminus of ubiquitin for binding.  These data suggest that binding the ubiquitin C terminus may be necessary for the function of other proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohW2SwRsFle7Vg90H21EOLACvtfcHk0lhuKD9WpB5Xbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xjt1KhsrY%253D&md5=1558ddae03f9884e8848e8ab449a1ae3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.02.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.02.038%26sid%3Dliteratum%253Aachs%26aulast%3DReyes-Turcu%26aufirst%3DF.%2BE.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DMullally%26aufirst%3DJ.%2BE.%26aulast%3DHeroux%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26atitle%3DThe%2520Ubiquitin%2520Binding%2520Domain%2520ZnF%2520UBP%2520Recognizes%2520the%2520C-Terminal%2520Diglycine%2520Motif%2520of%2520Unanchored%2520Ubiquitin%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D1197%26epage%3D1208%26doi%3D10.1016%2Fj.cell.2006.02.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avvakumov, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asinas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, U. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span> <span> </span><span class="NLM_article-title">Two ZnF-UBP Domains in Isopeptidase T (USP5)</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1188</span>– <span class="NLM_lpage">1198</span>, <span class="refDoi"> DOI: 10.1021/bi200854q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi200854q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWhs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1188-1198&author=G.+V.+Avvakumovauthor=J.+R.+Walkerauthor=S.+Xueauthor=A.+Allali-Hassaniauthor=A.+Asinasauthor=U.+B.+Nairauthor=X.+Fangauthor=X.+Zuoauthor=Y.+X.+Wangauthor=K.+D.+Wilkinsonauthor=S.+Dhe-Paganon&title=Two+ZnF-UBP+Domains+in+Isopeptidase+T+%28USP5%29&doi=10.1021%2Fbi200854q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Two ZnF-UBP Domains in Isopeptidase T (USP5)</span></div><div class="casAuthors">Avvakumov, George V.; Walker, John R.; Xue, Sheng; Allali-Hassani, Abdellah; Asinas, Abdalin; Nair, Usha B.; Fang, Xianyang; Zuo, Xiaobing; Wang, Yun-Xing; Wilkinson, Keith D.; Dhe-Paganon, Sirano</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1188-1198</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human ubiquitin-specific cysteine protease 5 (USP5, also known as ISOT and isopeptidase T), an 835-residue multidomain enzyme, recycles ubiquitin by hydrolyzing isopeptide bonds in a variety of unanchored polyubiquitin substrates.  Activation of the enzyme's hydrolytic activity toward ubiquitin-AMC (7-amino-4-methylcoumarin), a fluorogenic substrate, by the addn. of free, unanchored monoubiquitin suggested an allosteric mechanism of activation by the ZnF-UBP domain (residues 163-291), which binds the substrate's unanchored diglycine carboxyl tail.  By detg. the structure of full-length USP5, we discovered the existence of a cryptic ZnF-UBP domain (residues 1-156), which was tightly bound to the catalytic core and was indispensable for catalytic activity.  In contrast, the previously characterized ZnF-UBP domain did not contribute directly to the active site; a paucity of interactions suggested flexibility between these two domains consistent with an ability of the enzyme to hydrolyze a variety of different polyubiquitin chain linkages.  Deletion of the known ZnF-UBP domain did not significantly affect rate of hydrolysis of ubiquitin-AMC and suggested that it is likely assocd. mainly with substrate targeting and specificity.  Together, our findings show that USP5 uses multiple ZnF-UBP domains for substrate targeting and core catalytic function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpALXcZB0G4X7Vg90H21EOLACvtfcHk0lhuKD9WpB5Xbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWhs7g%253D&md5=45159b86dcdd454a0ac3974a48a36ab3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fbi200854q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi200854q%26sid%3Dliteratum%253Aachs%26aulast%3DAvvakumov%26aufirst%3DG.%2BV.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DXue%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DAsinas%26aufirst%3DA.%26aulast%3DNair%26aufirst%3DU.%2BB.%26aulast%3DFang%26aufirst%3DX.%26aulast%3DZuo%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%2BX.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26atitle%3DTwo%2520ZnF-UBP%2520Domains%2520in%2520Isopeptidase%2520T%2520%2528USP5%2529%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D1188%26epage%3D1198%26doi%3D10.1021%2Fbi200854q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amerik, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochstrasser, M.</span></span> <span> </span><span class="NLM_article-title">Mechanism and Function of Deubiquitinating Enzymes</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>1695</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1016/j.bbamcr.2004.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.bbamcr.2004.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=15571815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1695&publication_year=2004&pages=189-207&author=A.+Y.+Amerikauthor=M.+Hochstrasser&title=Mechanism+and+Function+of+Deubiquitinating+Enzymes&doi=10.1016%2Fj.bbamcr.2004.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism and function of deubiquitinating enzymes</span></div><div class="casAuthors">Amerik, Alexander Y.; Hochstrasser, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1695</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">189-207</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The attachment of ubiquitin (I) to proteins is a crucial step in many cellular regulatory mechanisms and contributes to numerous biol. processes, including embryonic development, the cell cycle, growth control, and prevention of neurodegeneration.  In these diverse regulatory settings, the most widespread mechanism of I action is probably in the context of protein degrdn.  Polyubiquitin attachment targets many intracellular proteins for degrdn. by the proteasome, and (mono)ubiquitination is often required for down-regulating plasma membrane proteins by targeting them to the vacuole (lysosome).  Ubiquitin-protein conjugates are highly dynamic structures.  Whereas an array of enzymes directs the conjugation of I to substrates, there are also dozens of deubiquitinating enzymes (DUBs) that can reverse the process.  Several lines of evidence indicate that DUBs are important regulators of the I system.  These enzymes are responsible for processing inactive I precursors, proofreading I-protein conjugates, removing I from cellular adducts, and keeping the 26S proteasome free of inhibitory I chains.  The present review focuses on recent discoveries that have led to a better understanding the mechanisms and physiol. roles of this diverse and still poorly understood group of enzymes.  The authors also discuss briefly some of the proteases that act on ubiquitin-like protein (UBL) conjugates and compare them to DUBs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4r5Q5j5cVbrVg90H21EOLACvtfcHk0lhuKD9WpB5Xbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzF&md5=4be57f84af923c82da6b84c6da73a9d9</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2004.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2004.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DAmerik%26aufirst%3DA.%2BY.%26aulast%3DHochstrasser%26aufirst%3DM.%26atitle%3DMechanism%2520and%2520Function%2520of%2520Deubiquitinating%2520Enzymes%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2004%26volume%3D1695%26spage%3D189%26epage%3D207%26doi%3D10.1016%2Fj.bbamcr.2004.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melandri, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, R. L.</span></span> <span> </span><span class="NLM_article-title">Kinetic and Mechanistic Studies on the Hydrolysis of Ubiquitin C- Terminal 7-Amido-4-Methylcoumarin by Deubiquitinating Enzymes</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1868</span>– <span class="NLM_lpage">1879</span>, <span class="refDoi"> DOI: 10.1021/bi9723360</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9723360" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK1cXms1OjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=1868-1879&author=L.+C.+Dangauthor=F.+D.+Melandriauthor=R.+L.+Stein&title=Kinetic+and+Mechanistic+Studies+on+the+Hydrolysis+of+Ubiquitin+C-+Terminal+7-Amido-4-Methylcoumarin+by+Deubiquitinating+Enzymes&doi=10.1021%2Fbi9723360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and Mechanistic Studies on the Hydrolysis of Ubiquitin C-Terminal 7-Amido-4-Methylcoumarin by Deubiquitinating Enzymes</span></div><div class="casAuthors">Dang, Luan C.; Melandri, Francesco D.; Stein, Ross L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1868-1879</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Deubiquitinating enzymes constitute a family of cysteine hydrolases that specifically cleave ubiquitin-derived substrates of general structure Ub-X, where X can be any no. of leaving groups ranging from small thiols and amines to Ub and other proteins (Ub, ubiquitin).  We have developed a general assay for deubiquitinating enzymes based on the substrate ubiquitin C-terminal 7-amido-4-methylcoumarin (Ub-AMC).  Ub-AMC is efficiently hydrolyzed with liberation of highly fluorescent AMC by two rabbit reticulocyte deubiquitinating enzymes: isopeptidase T (IPaseT), a member of the gene family of ubiquitin-specific processing enzymes, and UCH-L3, a member of the family of ubiquitin C-terminal hydrolases.  We used this new assay to probe kinetic and mechanistic aspects of catalysis by IPaseT and UCH-L3.  Results from four series of expts. are discussed: (1) For UCH-L3, we detd. steady-state kinetic parameters that suggest a diffusion-limited reaction of UCH-L3 with Ub-AMC.  To probe this, we detd. the viscosity dependence of kc/Km, as well as kc.  We found complex viscosity dependencies and interpreted these in the context of a model in which assocn. and acylation are viscosity-dependent but deacylation is viscosity-independent.  (2) The kinetics of inhibition of UCH-L3 by ubiquitin C-terminal aldehyde (Ub-H) were detd. and reveal a Ki that is less than 10-14 M.  Several mechanisms are considered to account for the extreme inhibition.  (3) The IPaseT-catalyzed hydrolysis of Ub-AMC is modulated by Ub with activation at low [Ub] and inhibition at high [Ub].  (4) Finally, we compare kc/Km values for deubiquitinating enzyme-catalyzed hydrolysis of Ub-AMC and Z-Leu-Arg-Gly-Gly-AMC.  For IPaseT, the ratio of rate consts. is 104, while for UCH-L3 this ratio is >107.  These results suggest the following: (i) Deubiquitinating enzymes are able to utilize the free energy that is released from remote interactions with Ub-contg. substrates for stabilization of catalytic transition states, and (ii) UCHs are more efficient at utilizing the energy from these interactions, presumably because they do not possess a binding domain for a Ub "leaving group".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOpMDRjOWV2LVg90H21EOLACvtfcHk0lhuKD9WpB5Xbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXms1OjsA%253D%253D&md5=078fe0b916c22ef4ea7d4fc77aca75c1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fbi9723360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9723360%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DL.%2BC.%26aulast%3DMelandri%26aufirst%3DF.%2BD.%26aulast%3DStein%26aufirst%3DR.%2BL.%26atitle%3DKinetic%2520and%2520Mechanistic%2520Studies%2520on%2520the%2520Hydrolysis%2520of%2520Ubiquitin%2520C-%2520Terminal%25207-Amido-4-Methylcoumarin%2520by%2520Deubiquitinating%2520Enzymes%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D1868%26epage%3D1879%26doi%3D10.1021%2Fbi9723360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dayal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allende-Vega, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saville, M. K.</span></span> <span> </span><span class="NLM_article-title">Suppression of the Deubiquitinating Enzyme USP5 Causes the Accumulation of Unanchored Polyubiquitin and the Activation of P53</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>284</i></span>,  <span class="NLM_fpage">5030</span>– <span class="NLM_lpage">5041</span>, <span class="refDoi"> DOI: 10.1074/jbc.M805871200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1074%2Fjbc.M805871200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=19098288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvVSnsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=5030-5041&author=S.+Dayalauthor=A.+Sparksauthor=J.+Jacobauthor=N.+Allende-Vegaauthor=D.+P.+Laneauthor=M.+K.+Saville&title=Suppression+of+the+Deubiquitinating+Enzyme+USP5+Causes+the+Accumulation+of+Unanchored+Polyubiquitin+and+the+Activation+of+P53&doi=10.1074%2Fjbc.M805871200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of the Deubiquitinating Enzyme USP5 Causes the Accumulation of Unanchored Polyubiquitin and the Activation of p53</span></div><div class="casAuthors">Dayal, Saurabh; Sparks, Alison; Jacob, Jimmy; Allende-Vega, Nerea; Lane, David P.; Saville, Mark K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">5030-5041</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Both p53 and its repressor Mdm2 are subject to ubiquitination and proteasomal degrdn.  The authors show that knockdown of the deubiquitinating enzyme USP5 (isopeptidase T) results in an increase in the level and transcriptional activity of p53.  Suppression of USP5 stabilizes p53, whereas it has little or no effect on the stability of Mdm2.  This provides a mechanism for transcriptional activation of p53.  USP5 knockdown interferes with the degrdn. of ubiquitinated p53 rather than attenuating p53 ubiquitination.  In vitro studies have shown that a preferred substrate for USP5 is unanchored polyubiquitin.  Consistent with this, the authors obsd. for the first time in a mammalian system that USP5 makes a major contribution to Lys-48-linked polyubiquitin disassembly and that suppression of USP5 results in the accumulation of unanchored polyubiquitin chains.  Ectopic expression of a C-terminal mutant of ubiquitin (G75A/G76A), which also causes the accumulation of free polyubiquitin, recapitulates the effects of USP5 knockdown on the p53 pathway.  The authors propose a model in which p53 is selectively stabilized because the unanchored polyubiquitin that accumulates after USP5 knockdown is able to compete with ubiquitinated p53 but not with Mdm2 for proteasomal recognition.  This raises the possibility that there are significant differences in proteasomal recognition of p53 and Mdm2.  These differences could be exploited therapeutically.  This study reveals a novel mechanism for regulation of p53 and identifies USP5 as a potential target for p53 activating therapeutic agents for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi-dWbk_lGsrVg90H21EOLACvtfcHk0ljthj4FKat6zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvVSnsL0%253D&md5=037f3359f4376b81a1726734f31acc18</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M805871200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M805871200%26sid%3Dliteratum%253Aachs%26aulast%3DDayal%26aufirst%3DS.%26aulast%3DSparks%26aufirst%3DA.%26aulast%3DJacob%26aufirst%3DJ.%26aulast%3DAllende-Vega%26aufirst%3DN.%26aulast%3DLane%26aufirst%3DD.%2BP.%26aulast%3DSaville%26aufirst%3DM.%2BK.%26atitle%3DSuppression%2520of%2520the%2520Deubiquitinating%2520Enzyme%2520USP5%2520Causes%2520the%2520Accumulation%2520of%2520Unanchored%2520Polyubiquitin%2520and%2520the%2520Activation%2520of%2520P53%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D5030%26epage%3D5041%26doi%3D10.1074%2Fjbc.M805871200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, M. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, J. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. F.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin Specific Peptidase 5 Mediates Histidine-Rich Protein Hpn Induced Cell Apoptosis in Hepatocellular Carcinoma through P14-P53 Signaling</span>. <i>Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>, <span class="NLM_elocation-id">1600350</span> <span class="refDoi"> DOI: 10.1002/pmic.201600350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1002%2Fpmic.201600350" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&author=Y.+Liuauthor=W.+M.+Wangauthor=L.+Y.+Zouauthor=L.+Liauthor=L.+Fengauthor=M.+Z.+Panauthor=M.+Y.+Lvauthor=Y.+Caoauthor=H.+Wangauthor=H.+F.+Kungauthor=J.+X.+Pangauthor=W.+M.+Fuauthor=J.+F.+Zhang&title=Ubiquitin+Specific+Peptidase+5+Mediates+Histidine-Rich+Protein+Hpn+Induced+Cell+Apoptosis+in+Hepatocellular+Carcinoma+through+P14-P53+Signaling&doi=10.1002%2Fpmic.201600350"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fpmic.201600350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpmic.201600350%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%2BM.%26aulast%3DZou%26aufirst%3DL.%2BY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DPan%26aufirst%3DM.%2BZ.%26aulast%3DLv%26aufirst%3DM.%2BY.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DKung%26aufirst%3DH.%2BF.%26aulast%3DPang%26aufirst%3DJ.%2BX.%26aulast%3DFu%26aufirst%3DW.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%2BF.%26atitle%3DUbiquitin%2520Specific%2520Peptidase%25205%2520Mediates%2520Histidine-Rich%2520Protein%2520Hpn%2520Induced%2520Cell%2520Apoptosis%2520in%2520Hepatocellular%2520Carcinoma%2520through%2520P14-P53%2520Signaling%26jtitle%3DProteomics%26date%3D2017%26volume%3D17%26doi%3D10.1002%2Fpmic.201600350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaistha, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krattenmacher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fredebohm, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrens, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hackert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strobel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoheisel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gress, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchholz, M.</span></span> <span> </span><span class="NLM_article-title">The Deubiquitinating Enzyme USP5 Promotes Pancreatic Cancer via Modulating Cell Cycle Regulators</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">66215</span>– <span class="NLM_lpage">66225</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.19882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.18632%2Foncotarget.19882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29029505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A280%3ADC%252BC1M7gsVCmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=66215-66225&author=B.+P.+Kaisthaauthor=A.+Krattenmacherauthor=J.+Fredebohmauthor=H.+Schmidtauthor=D.+Behrensauthor=M.+Widderauthor=T.+Hackertauthor=O.+Strobelauthor=J.+D.+Hoheiselauthor=T.+M.+Gressauthor=M.+Buchholz&title=The+Deubiquitinating+Enzyme+USP5+Promotes+Pancreatic+Cancer+via+Modulating+Cell+Cycle+Regulators&doi=10.18632%2Foncotarget.19882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The deubiquitinating enzyme USP5 promotes pancreatic cancer via modulating cell cycle regulators</span></div><div class="casAuthors">Kaistha Brajesh P; Krattenmacher Anja; Schmidt Harald; Gress Thomas M; Buchholz Malte; Fredebohm Johannes; Hoheisel Jorg D; Behrens Diana; Widder Miriam; Hackert Thilo; Strobel Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">66215-66225</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal solid tumors.  With an overall five-year survival rate remaining below 6%, there is an explicit need to search for new molecular targets for therapeutic interventions.  We undertook a barcode labelled short-hairpin (shRNA) library screen in pancreatic cancer cells in order to identify novel genes promoting cancer survival and progression.  Among the candidate genes identified in this screen was the deubiquitinase USP5, which subsequent gene expression analyses demonstrated to be significantly upregulated in primary human pancreatic cancer tissues.  Using different knockdown approaches, we show that expression of USP5 is essential for the proliferation and survival of pancreatic cancer cells, tested under different 2D and 3D cell culture conditions as well as in in vivo experiments.  These growth inhibition effects upon knockdown of USP5 are mediated primarily by the attenuation of G1/S phase transition in the cells, which is accompanied by accumulation of DNA damage, upregulation of p27, and increased apoptosis rates.  Since USP5 is overexpressed in cancer tissues, it can thus potentially serve as a new target for therapeutic interventions, especially given the fact that deubiquitinases are currently emerging as new class of attractive drug targets in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJAEqHM0GGgT69zv1FsJFMfW6udTcc2eYKoiflwDwRwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7gsVCmsw%253D%253D&md5=d95f1f00cdc6a04e3fd9495252ceedb9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.19882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.19882%26sid%3Dliteratum%253Aachs%26aulast%3DKaistha%26aufirst%3DB.%2BP.%26aulast%3DKrattenmacher%26aufirst%3DA.%26aulast%3DFredebohm%26aufirst%3DJ.%26aulast%3DSchmidt%26aufirst%3DH.%26aulast%3DBehrens%26aufirst%3DD.%26aulast%3DWidder%26aufirst%3DM.%26aulast%3DHackert%26aufirst%3DT.%26aulast%3DStrobel%26aufirst%3DO.%26aulast%3DHoheisel%26aufirst%3DJ.%2BD.%26aulast%3DGress%26aufirst%3DT.%2BM.%26aulast%3DBuchholz%26aufirst%3DM.%26atitle%3DThe%2520Deubiquitinating%2520Enzyme%2520USP5%2520Promotes%2520Pancreatic%2520Cancer%2520via%2520Modulating%2520Cell%2520Cycle%2520Regulators%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D66215%26epage%3D66225%26doi%3D10.18632%2Foncotarget.19882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the Deubiquitinase USP5 Leads to C-Maf Protein Degradation and Myeloma Cell Apoptosis</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e3058</span> <span class="refDoi"> DOI: 10.1038/cddis.2017.450</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fcddis.2017.450" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=28933784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=S.+Wangauthor=J.+Juanauthor=Z.+Zhangauthor=Y.+Duauthor=Y.+Xuauthor=J.+Tongauthor=B.+Caoauthor=M.+F.+Moranauthor=Y.+Zengauthor=X.+Mao&title=Inhibition+of+the+Deubiquitinase+USP5+Leads+to+C-Maf+Protein+Degradation+and+Myeloma+Cell+Apoptosis&doi=10.1038%2Fcddis.2017.450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the deubiquitinase USP5 leads to c-Maf protein degradation and myeloma cell apoptosis</span></div><div class="casAuthors">Wang, Siyu; Juan, Jiaxiang; Zhang, Zubin; Du, Yanyun; Xu, Yujia; Tong, Jiefei; Cao, Biyin; Moran, Michael F.; Zeng, Yuanying; Mao, Xinliang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e3058</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The deubiquitinase USP5 stabilizes c-Maf, a key transcription factor in multiple myeloma (MM), but the mechanisms and significance are unclear.  In the present study, USP5 was found to interact with c-Maf and prevented it from degrdn. by decreasing its polyubiquitination level.  Specifically, the 308th and 347th lysine residues in c-Maf were crit. for USP5-mediated deubiquitination and stability.  There are five key domains in the USP5 protein and subsequent studies revealed that the cryptic ZnF domain and the C-box domain interacted with c-Maf but the UBA1/UBA2 domain partly increased its stability.  Notably, MafA and MafB are also members of the c-Maf family, however, USP5 failed to deubiquitinate MafA, suggesting its substrate specificity.  In the functional studies, USP5 was found to promoted the transcriptional activity of c-Maf.  Consistent with the high level of c-Maf protein in MM cells, USP5 was also highly expressed.  When USP5 was knocked down, c-Maf underwent degrdn.  Interestingly, USP5 silence led to apoptosis of MM cells expressing c-Maf but not MM cells lacking c-Maf, indicating c-Maf is a key factor in USP5-mediated MM cell proliferation and survival.  Consistent with this finding, WP1130, an inhibitor of several Dubs including USP5, suppressed the transcriptional activity of c-Maf and induced MM cell apoptosis.  When c-Maf was overexpressed, WP1130-induced MM cell apoptosis was abolished.  Taken together, these findings suggest that USP5 regulates c-Maf stability and MM cell survival.  Targeting the USP5/c-Maf axis could be a potential strategy for MM treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6AdLuHwkC3rVg90H21EOLACvtfcHk0lj8j_ZtA3qmCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrvE&md5=a44f223532f555e4bfcab67f4a91a9b3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2017.450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2017.450%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DJuan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DB.%26aulast%3DMoran%26aufirst%3DM.%2BF.%26aulast%3DZeng%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DX.%26atitle%3DInhibition%2520of%2520the%2520Deubiquitinase%2520USP5%2520Leads%2520to%2520C-Maf%2520Protein%2520Degradation%2520and%2520Myeloma%2520Cell%2520Apoptosis%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fcddis.2017.450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span> <span> </span><span class="NLM_article-title">Overexpression of USP5 Contributes to Tumorigenesis in Non-Small Cell Lung Cancer via the Stabilization of β-Catenin Protein</span>. <i>Am. J. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2284</span>– <span class="NLM_lpage">2295</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30555744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVylur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=2284-2295&author=X.+Maauthor=W.+Qiauthor=H.+Panauthor=F.+Yangauthor=J.+Deng&title=Overexpression+of+USP5+Contributes+to+Tumorigenesis+in+Non-Small+Cell+Lung+Cancer+via+the+Stabilization+of+%CE%B2-Catenin+Protein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of USP5 contributes to tumorigenesis in non-small cell lung cancer via the stabilization of β-catenin protein</span></div><div class="casAuthors">Ma, Xingjie; Qi, Weibo; Pan, Huan; Yang, Fan; Deng, Jiali</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Cancer Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2284-2295</span>CODEN:
                <span class="NLM_cas:coden">AJCRFT</span>;
        ISSN:<span class="NLM_cas:issn">2156-6976</span>.
    
            (<span class="NLM_cas:orgname">e-Century Publishing Corp.</span>)
        </div><div class="casAbstract">The ubiquitin-specific protease 5 (USP5), a deubiquitinating enzyme, has been identified as a tumor promoter in several types of human cancer.  However, the role of USP5 in non-small lung cancer (NSCLC) has not yet been elucidated.  In this study, we found that USP5 was upregulated in NSCLC tissues compared with normal tissues.  High expression of USP5 was correlated with large primary tumor size, poor differentiation and advanced TNM stage, and led to a significantly shorter overall survival (OS).  USP5 overexpression enhanced, whereas USP5 silencing impaired the cell proliferation and colony formation of NSCLC cells in vitro.  Moreover, knockdown of USP5 in H1299 cells inhibited tumor growth in vivo.  Mechanistically, we found that USP5 deubiquitinated β-catenin, prevented ubiquitination mediated β-catenin degrdn. and promoted β-catenin nuclear accumulation, leading to the activation of Wnt/β-catenin signal pathway in NSCLC cells.  Taken together, these findings suggest that USP5 functions as an oncogene in NSCLC and its oncogenic activity involves in part through Wnt/β-catenin signal pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ1pGOpVPzmLVg90H21EOLACvtfcHk0lj8j_ZtA3qmCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVylur3E&md5=36aa00674ccda08ed764e24a9c579c81</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DX.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DPan%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DF.%26aulast%3DDeng%26aufirst%3DJ.%26atitle%3DOverexpression%2520of%2520USP5%2520Contributes%2520to%2520Tumorigenesis%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%2520via%2520the%2520Stabilization%2520of%2520%25CE%25B2-Catenin%2520Protein%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2018%26volume%3D8%26spage%3D2284%26epage%3D2295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">García-Caballero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadotti, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stemkowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgkinson, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bladen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizzoccaro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deage, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francois, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourinet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamponi, G. W.</span></span> <span> </span><span class="NLM_article-title">The Deubiquitinating Enzyme USP5 Modulates Neuropathic and Inflammatory Pain by Enhancing Cav3.2 Channel Activity</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">1144</span>– <span class="NLM_lpage">1158</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2014.07.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.neuron.2014.07.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=25189210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFagtr7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=1144-1158&author=A.+Garc%C3%ADa-Caballeroauthor=V.+M.+Gadottiauthor=P.+Stemkowskiauthor=N.+Weissauthor=I.+A.+Souzaauthor=V.+Hodgkinsonauthor=C.+Bladenauthor=L.+Chenauthor=J.+Hamidauthor=A.+Pizzoccaroauthor=M.+Deageauthor=A.+Francoisauthor=E.+Bourinetauthor=G.+W.+Zamponi&title=The+Deubiquitinating+Enzyme+USP5+Modulates+Neuropathic+and+Inflammatory+Pain+by+Enhancing+Cav3.2+Channel+Activity&doi=10.1016%2Fj.neuron.2014.07.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The Deubiquitinating Enzyme USP5 Modulates Neuropathic and Inflammatory Pain by Enhancing Cav3.2 Channel Activity</span></div><div class="casAuthors">Garcia-Caballero, Agustin; Gadotti, Vinicius M.; Stemkowski, Patrick; Weiss, Norbert; Souza, Ivana A.; Hodgkinson, Victoria; Bladen, Chris; Chen, Lina; Hamid, Jawed; Pizzoccaro, Anne; Deage, Mickael; Francois, Amaury; Bourinet, Emmanuel; Zamponi, Gerald W.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1144-1158</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">T-type calcium channels are essential contributors to the transmission of nociceptive signals in the primary afferent pain pathway.  Here, we show that T-type calcium channels are ubiquitinated by WWP1, a plasma-membrane-assocd. ubiquitin ligase that binds to the intracellular domain III-IV linker region of the Cav3.2 T-type channel and modifies specific lysine residues in this region.  A proteomic screen identified the deubiquitinating enzyme USP5 as a Cav3.2 III-IV linker interacting partner.  Knockdown of USP5 via shRNA increases Cav3.2 ubiquitination, decreases Cav3.2 protein levels, and reduces Cav3.2 whole-cell currents.  In vivo knockdown of USP5 or uncoupling USP5 from native Cav3.2 channels via intrathecal delivery of Tat peptides mediates analgesia in both inflammatory and neuropathic mouse models of mech. hypersensitivity.  Altogether, our expts. reveal a cell signaling pathway that regulates T-type channel activity and their role in nociceptive signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqppH6fdxwbPbVg90H21EOLACvtfcHk0lj8j_ZtA3qmCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFagtr7K&md5=31e1e09b05a169c67c87ad531d3c1885</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2014.07.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2014.07.036%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Caballero%26aufirst%3DA.%26aulast%3DGadotti%26aufirst%3DV.%2BM.%26aulast%3DStemkowski%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DN.%26aulast%3DSouza%26aufirst%3DI.%2BA.%26aulast%3DHodgkinson%26aufirst%3DV.%26aulast%3DBladen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHamid%26aufirst%3DJ.%26aulast%3DPizzoccaro%26aufirst%3DA.%26aulast%3DDeage%26aufirst%3DM.%26aulast%3DFrancois%26aufirst%3DA.%26aulast%3DBourinet%26aufirst%3DE.%26aulast%3DZamponi%26aufirst%3DG.%2BW.%26atitle%3DThe%2520Deubiquitinating%2520Enzyme%2520USP5%2520Modulates%2520Neuropathic%2520and%2520Inflammatory%2520Pain%2520by%2520Enhancing%2520Cav3.2%2520Channel%2520Activity%26jtitle%3DNeuron%26date%3D2014%26volume%3D83%26spage%3D1144%26epage%3D1158%26doi%3D10.1016%2Fj.neuron.2014.07.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Caballero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadotti, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamponi, G. W.</span></span> <span> </span><span class="NLM_article-title">A Cell-Permeant Peptide Corresponding to the CUBP Domain of USP5 Reverses Inflammatory and Neuropathic Pain</span>. <i>Mol. Pain</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1177/1744806916642444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1177%2F1744806916642444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtVOjs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=1-8&author=A.+Garcia-Caballeroauthor=V.+M.+Gadottiauthor=L.+Chenauthor=G.+W.+Zamponi&title=A+Cell-Permeant+Peptide+Corresponding+to+the+CUBP+Domain+of+USP5+Reverses+Inflammatory+and+Neuropathic+Pain&doi=10.1177%2F1744806916642444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A cell-permeant peptide corresponding to the cUBP domain of USP5 reverses inflammatory and neuropathic pain</span></div><div class="casAuthors">Garcia-Caballero, Agustin; Gadotti, Vinicius M.; Chen, Lina; Zamponi, Gerald W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pain</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">MPOAC5</span>;
        ISSN:<span class="NLM_cas:issn">1744-8069</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">Background: Cav3.2 T-type calcium currents in primary afferents are enhanced in various painful pathol. conditions, whereas inhibiting Cav3.2 activity or expression offers a strategy for combating the development of pain hypersensitivity.  We have shown that Cav3.2 channel surface d. is strongly regulated by the ubiquitination machinery and we identified the deubiquitinase USP5 as a Cav3.2 channel interacting protein and regulator of its cell surface expression.  We also reported that USP5 is upregulated in chronic pain conditions.  Conversely, preventing its binding to the channel in vivo mediates analgesia in inflammatory and neuropathic pain models.  Results: To identify which USP5 domain is responsible for the interaction, we used a series of USP5-derived peptides corresponding to different regions in nUBP, cUBP, UBA1, and UBA2 domains to outcompete full length USP5.  We identified a stretch of amino acid residues within the cUBP domain of USP5 as responsible for binding to Cav3.2 calcium channels.  Based on this information, we generated a TAT-cUBP1-USP5 peptide that could disrupt the Cav3.2/USP5 interaction in vitro and tested its physiol. effect in well-established models of persistent inflammatory pain (CFA test) and chronic mononeuropathy and polyneuropathy in mice (partial sciatic nerve injury and the (ob/ob) diabetic spontaneous neuropathic mice).  Our results reveal that the TAT-cUBP1-USP5 peptide attenuated mech. hyperalgesia induced by both Complete Freund's Adjuvant and partial sciatic nerve injury, and thermal hyperalgesia in diabetic neuropathic animals.  In contrast, Cav3.2 null mice were not affected by the peptide in the partial sciatic nerve injury model.  Cav3.2 calcium channel levels in diabetic mice were reduced following the administration of the TAT-cUBP1-USP5 peptide.  Conclusions: Our findings reveal a crucial region in the cUBP domain of USP5 that is important for substrate recognition and binding to the III-IV linker of Cav3.2 channels.  Targeting the interaction of this region with the Cav3.2 channel can be exploited as the basis for therapeutic intervention into inflammatory and neuropathic pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyT-_P1ltHgLVg90H21EOLACvtfcHk0lhvLFViV80a9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtVOjs70%253D&md5=fa46923173ccfe6a9b2fd4848171f116</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1177%2F1744806916642444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1744806916642444%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia-Caballero%26aufirst%3DA.%26aulast%3DGadotti%26aufirst%3DV.%2BM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DZamponi%26aufirst%3DG.%2BW.%26atitle%3DA%2520Cell-Permeant%2520Peptide%2520Corresponding%2520to%2520the%2520CUBP%2520Domain%2520of%2520USP5%2520Reverses%2520Inflammatory%2520and%2520Neuropathic%2520Pain%26jtitle%3DMol.%2520Pain%26date%3D2016%26volume%3D12%26spage%3D1%26epage%3D8%26doi%3D10.1177%2F1744806916642444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stemkowski, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Caballero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadotti, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">M’Dahoma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamponi, G. W.</span></span> <span> </span><span class="NLM_article-title">Identification of Interleukin-1 Beta as a Key Mediator in the Upregulation of Cav3.2–USP5 Interactions in the Pain Pathway</span>. <i>Mol. Pain</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">7</span>, <span class="refDoi"> DOI: 10.1177/1744806917724698</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1177%2F1744806917724698" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=1-7&author=P.+L.+Stemkowskiauthor=A.+Garcia-Caballeroauthor=V.+M.+Gadottiauthor=S.+M%E2%80%99Dahomaauthor=L.+Chenauthor=I.+A.+Souzaauthor=G.+W.+Zamponi&title=Identification+of+Interleukin-1+Beta+as+a+Key+Mediator+in+the+Upregulation+of+Cav3.2%E2%80%93USP5+Interactions+in+the+Pain+Pathway&doi=10.1177%2F1744806917724698"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1177%2F1744806917724698&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1744806917724698%26sid%3Dliteratum%253Aachs%26aulast%3DStemkowski%26aufirst%3DP.%2BL.%26aulast%3DGarcia-Caballero%26aufirst%3DA.%26aulast%3DGadotti%26aufirst%3DV.%2BM.%26aulast%3DM%25E2%2580%2599Dahoma%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSouza%26aufirst%3DI.%2BA.%26aulast%3DZamponi%26aufirst%3DG.%2BW.%26atitle%3DIdentification%2520of%2520Interleukin-1%2520Beta%2520as%2520a%2520Key%2520Mediator%2520in%2520the%2520Upregulation%2520of%2520Cav3.2%25E2%2580%2593USP5%2520Interactions%2520in%2520the%2520Pain%2520Pathway%26jtitle%3DMol.%2520Pain%26date%3D2017%26volume%3D13%26spage%3D1%26epage%3D7%26doi%3D10.1177%2F1744806917724698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao-Feng Qin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span> <span> </span><span class="NLM_article-title">Broad and Diverse Mechanisms Used by Deubiquitinase Family Members in Regulating the Type i Interferon Signaling Pathway during Antiviral Responses</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">eaar2824</span> <span class="refDoi"> DOI: 10.1126/sciadv.aar2824</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1126%2Fsciadv.aar2824" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29732405" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&author=Q.+Liuauthor=Y.+Wuauthor=Y.+Qinauthor=J.+Huauthor=W.+Xieauthor=F.+Xiao-Feng+Qinauthor=J.+Cui&title=Broad+and+Diverse+Mechanisms+Used+by+Deubiquitinase+Family+Members+in+Regulating+the+Type+i+Interferon+Signaling+Pathway+during+Antiviral+Responses&doi=10.1126%2Fsciadv.aar2824"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.aar2824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.aar2824%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DQin%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DXiao-Feng%2BQin%26aufirst%3DF.%26aulast%3DCui%26aufirst%3DJ.%26atitle%3DBroad%2520and%2520Diverse%2520Mechanisms%2520Used%2520by%2520Deubiquitinase%2520Family%2520Members%2520in%2520Regulating%2520the%2520Type%2520i%2520Interferon%2520Signaling%2520Pathway%2520during%2520Antiviral%2520Responses%26jtitle%3DSci.%2520Adv.%26date%3D2018%26volume%3D4%26doi%3D10.1126%2Fsciadv.aar2824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ovaa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galardy, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ploegh, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masucci, M. G.</span></span> <span> </span><span class="NLM_article-title">Activity-Based Ubiquitin-Specific Protease (USP) Profiling of Virus-Infected and Malignant Human Cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">2253</span>– <span class="NLM_lpage">2258</span>, <span class="refDoi"> DOI: 10.1073/pnas.0308411100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1073%2Fpnas.0308411100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=14982996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvVOisrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=2253-2258&author=H.+Ovaaauthor=B.+M.+Kesslerauthor=U.+Rolenauthor=P.+J.+Galardyauthor=H.+L.+Ploeghauthor=M.+G.+Masucci&title=Activity-Based+Ubiquitin-Specific+Protease+%28USP%29+Profiling+of+Virus-Infected+and+Malignant+Human+Cells&doi=10.1073%2Fpnas.0308411100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Activity-based ubiquitin-specific protease (USP) profiling of virus-infected and malignant human cells</span></div><div class="casAuthors">Ovaa, Huib; Kessler, Benedikt M.; Rolen, Ulrika; Galardy, Paul J.; Ploegh, Hidde L.; Masucci, Maria G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2253-2258</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The family of ubiquitin (Ub)-specific proteases (USP) removes Ub from Ub conjugates and regulates a variety of cellular processes.  The human genome contains many putative USP-encoding genes, but little is known about USP tissue distribution, pattern of expression, activity, and substrate specificity.  We have used a chem.-based functional proteomics approach to identify active USPs in normal, virus-infected, and tumor-derived human cells.  Depending on tissue origin and stage of activation/differentiation, different USP activity profiles were revealed.  The activity of specific USPs, including USP5, -7, -9, -13, -15, and -22, was up-regulated by mitogen activation or virus infection in normal T and B lymphocytes.  UCH-L1 was highly expressed in tumor cell lines of epithelial and hematopoietic cell origin but was not detected in freshly isolated and mitogen-activated cells.  Up-regulation of this USP was a late event in the establishment of Epstein-Barr virus-immortalized lymphoblastoid cell lines and correlated with enhanced proliferation, suggesting a possible role in growth transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyC75pjCOjm7Vg90H21EOLACvtfcHk0lhvLFViV80a9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhvVOisrg%253D&md5=e7842bcd2842f37f4e1c9b19a8e55e8f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0308411100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0308411100%26sid%3Dliteratum%253Aachs%26aulast%3DOvaa%26aufirst%3DH.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DRolen%26aufirst%3DU.%26aulast%3DGalardy%26aufirst%3DP.%2BJ.%26aulast%3DPloegh%26aufirst%3DH.%2BL.%26aulast%3DMasucci%26aufirst%3DM.%2BG.%26atitle%3DActivity-Based%2520Ubiquitin-Specific%2520Protease%2520%2528USP%2529%2520Profiling%2520of%2520Virus-Infected%2520and%2520Malignant%2520Human%2520Cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D2253%26epage%3D2258%26doi%3D10.1073%2Fpnas.0308411100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapić-Otrin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, A. S.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin-Specific Protease 5 Is Required for the Efficient Repair of DNA Double-Strand Breaks</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e84899</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0084899</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1371%2Fjournal.pone.0084899" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=24454762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFKgt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=S.+Nakajimaauthor=L.+Lanauthor=L.+Weiauthor=C.+L.+Hsiehauthor=V.+Rapi%C4%87-Otrinauthor=A.+Yasuiauthor=A.+S.+Levine&title=Ubiquitin-Specific+Protease+5+Is+Required+for+the+Efficient+Repair+of+DNA+Double-Strand+Breaks&doi=10.1371%2Fjournal.pone.0084899"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-specific protease 5 is required for the efficient repair of DNA double-strand breaks</span></div><div class="casAuthors">Nakajima, Satoshi; Lan, Li; Wei, Leizhen; Hsieh, Ching-Lung; Rapic-Otrin, Vesna; Yasui, Akira; Levine, Arthur S.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e84899/1-e84899/11, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">During the DNA damage response (DDR), ubiquitination plays an important role in the recruitment and regulation of repair proteins.  However, little is known about elimination of the ubiquitination signal after repair is completed.  Here we show that the ubiquitin-specific protease 5 (USP5), a deubiquitinating enzyme, is involved in the elimination of the ubiquitin signal from damaged sites and is required for efficient DNA double-strand break (DSB) repair.  Depletion of USP5 sensitizes cells to DNA damaging agents, produces DSBs, causes delayed disappearance of γH2AX foci after Bleocin treatment, and influences DSB repair efficiency in the homologous recombination pathway but not in the non-homologous end joining pathway.  USP5 co-localizes to DSBs induced by laser micro-irradn. in a RAD18-dependent manner.  Importantly, polyubiquitin chains at sites of DNA damage remained for longer periods in USP5-depleted cells.  Our results show that disassembly of polyubiquitin chains by USP5 at sites of damage is important for efficient DSB repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOeYhezJqUerVg90H21EOLACvtfcHk0lhvLFViV80a9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFKgt70%253D&md5=8887ac9cef564f4df2282f697585ab82</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084899%26sid%3Dliteratum%253Aachs%26aulast%3DNakajima%26aufirst%3DS.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DHsieh%26aufirst%3DC.%2BL.%26aulast%3DRapi%25C4%2587-Otrin%26aufirst%3DV.%26aulast%3DYasui%26aufirst%3DA.%26aulast%3DLevine%26aufirst%3DA.%2BS.%26atitle%3DUbiquitin-Specific%2520Protease%25205%2520Is%2520Required%2520for%2520the%2520Efficient%2520Repair%2520of%2520DNA%2520Double-Strand%2520Breaks%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0084899" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukushima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawahara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saeki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komada, M.</span></span> <span> </span><span class="NLM_article-title">Deubiquitylases USP5 and USP13 Are Recruited to and Regulate Heat-Induced Stress Granules through Their Deubiquitylating Activities</span>. <i>J. Cell Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>131</i></span>, <span class="NLM_elocation-id">jcs210856</span> <span class="refDoi"> DOI: 10.1242/jcs.210856</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1242%2Fjcs.210856" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29567855" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2018&author=X.+Xieauthor=S.+Matsumotoauthor=A.+Endoauthor=T.+Fukushimaauthor=H.+Kawaharaauthor=Y.+Saekiauthor=M.+Komada&title=Deubiquitylases+USP5+and+USP13+Are+Recruited+to+and+Regulate+Heat-Induced+Stress+Granules+through+Their+Deubiquitylating+Activities&doi=10.1242%2Fjcs.210856"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1242%2Fjcs.210856&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.210856%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DX.%26aulast%3DMatsumoto%26aufirst%3DS.%26aulast%3DEndo%26aufirst%3DA.%26aulast%3DFukushima%26aufirst%3DT.%26aulast%3DKawahara%26aufirst%3DH.%26aulast%3DSaeki%26aufirst%3DY.%26aulast%3DKomada%26aufirst%3DM.%26atitle%3DDeubiquitylases%2520USP5%2520and%2520USP13%2520Are%2520Recruited%2520to%2520and%2520Regulate%2520Heat-Induced%2520Stress%2520Granules%2520through%2520Their%2520Deubiquitylating%2520Activities%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2018%26volume%3D131%26doi%3D10.1242%2Fjcs.210856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sizemore, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dharia, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pantel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubonja, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawes, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhivich, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyatt, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalani, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span> <span> </span><span class="NLM_article-title">Computational Correction of Copy Number Effect Improves Specificity of CRISPR-Cas9 Essentiality Screens in Cancer Cells</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1779</span>– <span class="NLM_lpage">1784</span>, <span class="refDoi"> DOI: 10.1038/ng.3984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fng.3984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29083409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2017&pages=1779-1784&author=R.+M.+Meyersauthor=J.+G.+Bryanauthor=J.+M.+McFarlandauthor=B.+A.+Weirauthor=A.+E.+Sizemoreauthor=H.+Xuauthor=N.+V.+Dhariaauthor=P.+G.+Montgomeryauthor=G.+S.+Cowleyauthor=S.+Pantelauthor=A.+Goodaleauthor=Y.+Leeauthor=L.+D.+Aliauthor=G.+Jiangauthor=R.+Lubonjaauthor=W.+F.+Harringtonauthor=M.+Stricklandauthor=T.+Wuauthor=D.+C.+Hawesauthor=V.+A.+Zhivichauthor=M.+R.+Wyattauthor=Z.+Kalaniauthor=J.+J.+Changauthor=M.+Okamotoauthor=K.+Stegmaierauthor=T.+R.+Golubauthor=J.+S.+Boehmauthor=F.+Vazquezauthor=D.+E.+Rootauthor=W.+C.+Hahnauthor=A.+Tsherniak&title=Computational+Correction+of+Copy+Number+Effect+Improves+Specificity+of+CRISPR-Cas9+Essentiality+Screens+in+Cancer+Cells&doi=10.1038%2Fng.3984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Computational correction of copy number effect improves specificity of CRISPR-Cas9 essentiality screens in cancer cells</span></div><div class="casAuthors">Meyers, Robin M.; Bryan, Jordan G.; McFarland, James M.; Weir, Barbara A.; Sizemore, Ann E.; Xu, Han; Dharia, Neekesh V.; Montgomery, Phillip G.; Cowley, Glenn S.; Pantel, Sasha; Goodale, Amy; Ali, Levi D.; Jiang, Guozhi; Lubonja, Rakela; Harrington, William F.; Strickland, Matthew; Wu, Ting; Hawes, Derek C.; Zhivich, Victor A.; Wyatt, Meghan R.; Kalani, Zohra; Chang, Jaime J.; Okamoto, Michael; Stegmaier, Kimberly; Golub, Todd R.; Boehm, Jesse S.; Vazquez, Francisca; Root, David E.; Hahn, William C.; Tsherniak, Aviad</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1779-1784</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The CRISPR-Cas9 system has revolutionized gene editing both at single genes and in multiplexed loss-of-function screens, thus enabling precise genome-scale identification of genes essential for proliferation and survival of cancer cells.  However, previous studies have reported that a gene-independent antiproliferative effect of Cas9-mediated DNA cleavage confounds such measurement of genetic dependency, thereby leading to false-pos. results in copy no.-amplified regions.  We developed CERES, a computational method to est. gene-dependency levels from CRISPR-Cas9 essentiality screens while accounting for the copy no.-specific effect.  In our efforts to define a cancer dependency map, we performed genome-scale CRISPR-Cas9 essentiality screens across 342 cancer cell lines and applied CERES to this data set.  We found that CERES decreased false-pos. results and estd. sgRNA activity for both this data set and previously published screens performed with different sgRNA libraries.  We further demonstrate the utility of this collection of screens, after CERES correction, for identifying cancer-type-specific vulnerabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8iODR9NtDbVg90H21EOLACvtfcHk0liSyH0gbUsr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslehtrjI&md5=539e8fb16958480ce2d430a7f8e6cddf</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fng.3984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3984%26sid%3Dliteratum%253Aachs%26aulast%3DMeyers%26aufirst%3DR.%2BM.%26aulast%3DBryan%26aufirst%3DJ.%2BG.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DWeir%26aufirst%3DB.%2BA.%26aulast%3DSizemore%26aufirst%3DA.%2BE.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDharia%26aufirst%3DN.%2BV.%26aulast%3DMontgomery%26aufirst%3DP.%2BG.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DPantel%26aufirst%3DS.%26aulast%3DGoodale%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAli%26aufirst%3DL.%2BD.%26aulast%3DJiang%26aufirst%3DG.%26aulast%3DLubonja%26aufirst%3DR.%26aulast%3DHarrington%26aufirst%3DW.%2BF.%26aulast%3DStrickland%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DHawes%26aufirst%3DD.%2BC.%26aulast%3DZhivich%26aufirst%3DV.%2BA.%26aulast%3DWyatt%26aufirst%3DM.%2BR.%26aulast%3DKalani%26aufirst%3DZ.%26aulast%3DChang%26aufirst%3DJ.%2BJ.%26aulast%3DOkamoto%26aufirst%3DM.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DTsherniak%26aufirst%3DA.%26atitle%3DComputational%2520Correction%2520of%2520Copy%2520Number%2520Effect%2520Improves%2520Specificity%2520of%2520CRISPR-Cas9%2520Essentiality%2520Screens%2520in%2520Cancer%2520Cells%26jtitle%3DNat.%2520Genet.%26date%3D2017%26volume%3D49%26spage%3D1779%26epage%3D1784%26doi%3D10.1038%2Fng.3984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behan, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picco, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Migliardi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinnelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McRae, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pooley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser-Doepner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trusolino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stronach, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez-Rodiguez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yusa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, M. J.</span></span> <span> </span><span class="NLM_article-title">Prioritization of Cancer Therapeutic Targets Using CRISPR–Cas9 Screens</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>568</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1103-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fs41586-019-1103-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30971826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=568&publication_year=2019&pages=511-516&author=F.+M.+Behanauthor=F.+Iorioauthor=G.+Piccoauthor=E.+Gon%C3%A7alvesauthor=C.+M.+Beaverauthor=G.+Migliardiauthor=R.+Santosauthor=Y.+Raoauthor=F.+Sassiauthor=M.+Pinnelliauthor=R.+Ansariauthor=S.+Harperauthor=D.+A.+Jacksonauthor=R.+McRaeauthor=R.+Pooleyauthor=P.+Wilkinsonauthor=D.+van+der+Meerauthor=D.+Dowauthor=C.+Buser-Doepnerauthor=A.+Bertottiauthor=L.+Trusolinoauthor=E.+A.+Stronachauthor=J.+Saez-Rodiguezauthor=K.+Yusaauthor=M.+J.+Garnett&title=Prioritization+of+Cancer+Therapeutic+Targets+Using+CRISPR%E2%80%93Cas9+Screens&doi=10.1038%2Fs41586-019-1103-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Prioritization of cancer therapeutic targets using CRISPR-Cas9 screens</span></div><div class="casAuthors">Behan, Fiona M.; Iorio, Francesco; Picco, Gabriele; Goncalves, Emanuel; Beaver, Charlotte M.; Migliardi, Giorgia; Santos, Rita; Rao, Yanhua; Sassi, Francesco; Pinnelli, Marika; Ansari, Rizwan; Harper, Sarah; Jackson, David Adam; McRae, Rebecca; Pooley, Rachel; Wilkinson, Piers; van der Meer, Dieudonne; Dow, David; Buser-Doepner, Carolyn; Bertotti, Andrea; Trusolino, Livio; Stronach, Euan A.; Saez-Rodriguez, Julio; Yusa, Kosuke; Garnett, Mathew J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">568</span>
        (<span class="NLM_cas:issue">7753</span>),
    <span class="NLM_cas:pages">511-516</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Functional genomics approaches can overcome limitations-such as the lack of identification of robust targets and poor clin. efficacy-that hamper cancer drug development.  Here we performed genome-scale CRISPR-Cas9 screens in 324 human cancer cell lines from 30 cancer types and developed a data-driven framework to prioritize candidates for cancer therapeutics.  We integrated cell fitness effects with genomic biomarkers and target tractability for drug development to systematically prioritize new targets in defined tissues and genotypes.  We verified one of our most promising dependencies, the Werner syndrome ATP-dependent helicase, as a synthetic lethal target in tumors from multiple cancer types with microsatellite instability.  Our anal. provides a resource of cancer dependencies, generates a framework to prioritize cancer drug targets and suggests specific new targets.  The principles described in this study can inform the initial stages of drug development by contributing to a new, diverse and more effective portfolio of cancer drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHnWQcCFsy-LVg90H21EOLACvtfcHk0lj_TfS_z09DwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVSgtrw%253D&md5=3cac7e1a9ee5c34c8599154e8fc30573</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1103-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1103-9%26sid%3Dliteratum%253Aachs%26aulast%3DBehan%26aufirst%3DF.%2BM.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DPicco%26aufirst%3DG.%26aulast%3DGon%25C3%25A7alves%26aufirst%3DE.%26aulast%3DBeaver%26aufirst%3DC.%2BM.%26aulast%3DMigliardi%26aufirst%3DG.%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DY.%26aulast%3DSassi%26aufirst%3DF.%26aulast%3DPinnelli%26aufirst%3DM.%26aulast%3DAnsari%26aufirst%3DR.%26aulast%3DHarper%26aufirst%3DS.%26aulast%3DJackson%26aufirst%3DD.%2BA.%26aulast%3DMcRae%26aufirst%3DR.%26aulast%3DPooley%26aufirst%3DR.%26aulast%3DWilkinson%26aufirst%3DP.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DD.%26aulast%3DDow%26aufirst%3DD.%26aulast%3DBuser-Doepner%26aufirst%3DC.%26aulast%3DBertotti%26aufirst%3DA.%26aulast%3DTrusolino%26aufirst%3DL.%26aulast%3DStronach%26aufirst%3DE.%2BA.%26aulast%3DSaez-Rodiguez%26aufirst%3DJ.%26aulast%3DYusa%26aufirst%3DK.%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26atitle%3DPrioritization%2520of%2520Cancer%2520Therapeutic%2520Targets%2520Using%2520CRISPR%25E2%2580%2593Cas9%2520Screens%26jtitle%3DNature%26date%3D2019%26volume%3D568%26spage%3D511%26epage%3D516%26doi%3D10.1038%2Fs41586-019-1103-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Depmap
Broad</span>. <span> </span><span class="NLM_article-title">DepMap Achilles 19Q1 Public</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=Depmap%0ABroad&title=DepMap+Achilles+19Q1+Public"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DDepMap%2520Achilles%252019Q1%2520Public%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavory, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Dowd, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flasz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkoudis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dossang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClelland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odrzywol, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, T.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Highly Potent and Selective Allosteric USP7 Inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">118</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnchembio.2528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29200206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWgs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=118-125&author=G.+Gavoryauthor=C.+R.+O%E2%80%99Dowdauthor=M.+D.+Helmauthor=J.+Flaszauthor=E.+Arkoudisauthor=A.+Dossangauthor=C.+Hughesauthor=E.+Cassidyauthor=K.+McClellandauthor=E.+Odrzywolauthor=N.+Pageauthor=O.+Barkerauthor=H.+Mielauthor=T.+Harrison&title=Discovery+and+Characterization+of+Highly+Potent+and+Selective+Allosteric+USP7+Inhibitors&doi=10.1038%2Fnchembio.2528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of highly potent and selective allosteric USP7 inhibitors</span></div><div class="casAuthors">Gavory, Gerald; O'Dowd, Colin R.; Helm, Matthew D.; Flasz, Jakub; Arkoudis, Elias; Dossang, Anthony; Hughes, Caroline; Cassidy, Eamon; McClelland, Keeva; Odrzywol, Ewa; Page, Natalie; Barker, Oliver; Miel, Hugues; Harrison, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">118-125</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Given the importance of ubiquitin-specific protease 7 (USP7) in oncogenic pathways, identification of USP7 inhibitors has attracted considerable interest.  Despite substantial efforts, however, the development of validated deubiquitinase (DUB) inhibitors that exhibit drug-like properties and a well-defined mechanism of action has proven particularly challenging.  In this article, we describe the identification, optimization and detailed characterization of highly potent (IC50 < 10 nM), selective USP7 inhibitors together with their less active, enantiomeric counterparts.  We also disclose, for the first time, co-crystal structures of a human DUB enzyme complexed with small-mol. inhibitors, which reveal a previously undisclosed allosteric binding site.  Finally, we report the identification of cancer cell lines hypersensitive to USP7 inhibition (EC50 < 30 nM) and demonstrate equal or superior activity in these cell models compared to clin. relevant MDM2 antagonists.  Overall, these findings demonstrate the tractability and druggability of DUBs, and provide important tools for addnl. target validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe5GeOEWbfz7Vg90H21EOLACvtfcHk0lj_TfS_z09DwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWgs77F&md5=de6068d105b681c319e76e81ebfe9676</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2528%26sid%3Dliteratum%253Aachs%26aulast%3DGavory%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Dowd%26aufirst%3DC.%2BR.%26aulast%3DHelm%26aufirst%3DM.%2BD.%26aulast%3DFlasz%26aufirst%3DJ.%26aulast%3DArkoudis%26aufirst%3DE.%26aulast%3DDossang%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DC.%26aulast%3DCassidy%26aufirst%3DE.%26aulast%3DMcClelland%26aufirst%3DK.%26aulast%3DOdrzywol%26aufirst%3DE.%26aulast%3DPage%26aufirst%3DN.%26aulast%3DBarker%26aufirst%3DO.%26aulast%3DMiel%26aufirst%3DH.%26aulast%3DHarrison%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Highly%2520Potent%2520and%2520Selective%2520Allosteric%2520USP7%2520Inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D118%26epage%3D125%26doi%3D10.1038%2Fnchembio.2528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kategaya, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Lello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rougé, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pastor, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drummond, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleinheinz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heideker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCleland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritorto, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trost, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bainbridge, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, M. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiffler, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasher, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaishankar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peale, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gnad, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrest, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndubaku, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresini, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsui, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwerdtfeger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span> <span> </span><span class="NLM_article-title">USP7 Small-Molecule Inhibitors Interfere with Ubiquitin Binding</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">534</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/nature24006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnature24006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=29045385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1OhtrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=534-538&author=L.+Kategayaauthor=P.+Di+Lelloauthor=L.+Roug%C3%A9author=R.+Pastorauthor=K.+R.+Clarkauthor=J.+Drummondauthor=T.+Kleinheinzauthor=E.+Linauthor=J.+P.+Uptonauthor=S.+Prakashauthor=J.+Heidekerauthor=M.+McClelandauthor=M.+S.+Ritortoauthor=D.+R.+Alessiauthor=M.+Trostauthor=T.+W.+Bainbridgeauthor=M.+C.+M.+Kwokauthor=T.+P.+Maauthor=Z.+Stifflerauthor=B.+Brasherauthor=Y.+Tangauthor=P.+Jaishankarauthor=B.+R.+Hearnauthor=A.+R.+Rensloauthor=M.+R.+Arkinauthor=F.+Cohenauthor=K.+Yuauthor=F.+Pealeauthor=F.+Gnadauthor=M.+T.+Changauthor=C.+Klijnauthor=E.+Blackwoodauthor=S.+E.+Martinauthor=W.+F.+Forrestauthor=J.+A.+Ernstauthor=C.+Ndubakuauthor=X.+Wangauthor=M.+H.+Beresiniauthor=V.+Tsuiauthor=C.+Schwerdtfegerauthor=R.+A.+Blakeauthor=J.+Murrayauthor=T.+Maurerauthor=I.+E.+Wertz&title=USP7+Small-Molecule+Inhibitors+Interfere+with+Ubiquitin+Binding&doi=10.1038%2Fnature24006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">USP7 small-molecule inhibitors interfere with ubiquitin binding</span></div><div class="casAuthors">Kategaya, Lorna; Di Lello, Paola; Rouge, Lionel; Pastor, Richard; Clark, Kevin R.; Drummond, Jason; Kleinheinz, Tracy; Lin, Eva; Upton, John-Paul; Prakash, Sumit; Heideker, Johanna; McCleland, Mark; Ritorto, Maria Stella; Alessi, Dario R.; Trost, Matthias; Bainbridge, Travis W.; Kwok, Michael C. M.; Ma, Taylur P.; Stiffler, Zachary; Brasher, Bradley; Tang, Yinyan; Jaishankar, Priyadarshini; Hearn, Brian R.; Renslo, Adam R.; Arkin, Michelle R.; Cohen, Frederick; Yu, Kebing; Peale, Frank; Gnad, Florian; Chang, Matthew T.; Klijn, Christiaan; Blackwood, Elizabeth; Martin, Scott E.; Forrest, William F.; Ernst, James A.; Ndubaku, Chudi; Wang, Xiaojing; Beresini, Maureen H.; Tsui, Vickie; Schwerdtfeger, Carsten; Blake, Robert A.; Murray, Jeremy; Maurer, Till; Wertz, Ingrid E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7677</span>),
    <span class="NLM_cas:pages">534-538</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The ubiquitin system regulates essential cellular processes in eukaryotes.  Ubiquitin is ligated to substrate proteins as monomers or chains and the topol. of ubiquitin modifications regulates substrate interactions with specific proteins.  Thus, ubiquitination directs a variety of substrate fates including proteasomal degrdn.  Deubiquitinases cleave ubiquitin from substrates and are implicated in disease; e.g., ubiquitin-specific protease-7 (USP7) regulates the stability of the p53 tumor suppressor and other proteins crit. for tumor cell survival.  However, developing selective deubiquitinase inhibitors has been challenging and no co-crystal structures have been solved with small-mol. inhibitors.  Here, using NMR-based screening and structure-based design, we describe the development of the selective USP7 inhibitors, GNE-6640 and GNE-6776.  These compds. induce tumor cell death and enhance cytotoxicity with chemotherapeutic agents and targeted compds., including PIM kinase inhibitors.  Structural studies revealed that GNE-6640 and GNE-6776 non-covalently targeted USP7 12 Å distant from the catalytic Cys residue.  The compds. attenuated ubiquitin binding and thus inhibited USP7 deubiquitinase activity.  GNE-6640 and GNE-6776 interacted with acidic residues that mediate H-bond interactions with the ubiquitin Lys-48 side-chain, suggesting that USP7 preferentially interacts with and cleaves ubiquitin moieties that have free Lys-48 side-chains.  We investigated this idea by engineering di-ubiquitin chains contg. differential proximal and distal isotopic labels and measuring USP7 binding by NMR.  This preferential binding protracted the depolymn. kinetics of Lys-48-linked ubiquitin chains relative to Lys-63-linked chains.  In summary, engineering compds. that inhibit USP7 activity by attenuating ubiquitin binding suggests opportunities for developing other deubiquitinase inhibitors and may be a strategy more broadly applicable to inhibiting proteins that require ubiquitin binding for full functional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDUMQKwY0VGbVg90H21EOLACvtfcHk0lh2LS0m_e2tVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1OhtrjL&md5=85af379796b4026105815b5ffdd01f8d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnature24006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24006%26sid%3Dliteratum%253Aachs%26aulast%3DKategaya%26aufirst%3DL.%26aulast%3DDi%2BLello%26aufirst%3DP.%26aulast%3DRoug%25C3%25A9%26aufirst%3DL.%26aulast%3DPastor%26aufirst%3DR.%26aulast%3DClark%26aufirst%3DK.%2BR.%26aulast%3DDrummond%26aufirst%3DJ.%26aulast%3DKleinheinz%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DE.%26aulast%3DUpton%26aufirst%3DJ.%2BP.%26aulast%3DPrakash%26aufirst%3DS.%26aulast%3DHeideker%26aufirst%3DJ.%26aulast%3DMcCleland%26aufirst%3DM.%26aulast%3DRitorto%26aufirst%3DM.%2BS.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DTrost%26aufirst%3DM.%26aulast%3DBainbridge%26aufirst%3DT.%2BW.%26aulast%3DKwok%26aufirst%3DM.%2BC.%2BM.%26aulast%3DMa%26aufirst%3DT.%2BP.%26aulast%3DStiffler%26aufirst%3DZ.%26aulast%3DBrasher%26aufirst%3DB.%26aulast%3DTang%26aufirst%3DY.%26aulast%3DJaishankar%26aufirst%3DP.%26aulast%3DHearn%26aufirst%3DB.%2BR.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DCohen%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DPeale%26aufirst%3DF.%26aulast%3DGnad%26aufirst%3DF.%26aulast%3DChang%26aufirst%3DM.%2BT.%26aulast%3DKlijn%26aufirst%3DC.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DS.%2BE.%26aulast%3DForrest%26aufirst%3DW.%2BF.%26aulast%3DErnst%26aufirst%3DJ.%2BA.%26aulast%3DNdubaku%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DBeresini%26aufirst%3DM.%2BH.%26aulast%3DTsui%26aufirst%3DV.%26aulast%3DSchwerdtfeger%26aufirst%3DC.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DMurray%26aufirst%3DJ.%26aulast%3DMaurer%26aufirst%3DT.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26atitle%3DUSP7%2520Small-Molecule%2520Inhibitors%2520Interfere%2520with%2520Ubiquitin%2520Binding%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D534%26epage%3D538%26doi%3D10.1038%2Fnature24006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dexheimer, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villamil, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luci, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Z.</span></span> <span> </span><span class="NLM_article-title">A Selective USP1-UAF1 Inhibitor Links Deubiquitination to DNA Damage Responses</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnchembio.1455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=24531842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOgtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=298-304&author=Q.+Liangauthor=T.+S.+Dexheimerauthor=P.+Zhangauthor=A.+S.+Rosenthalauthor=M.+A.+Villamilauthor=C.+Youauthor=Q.+Zhangauthor=J.+Chenauthor=C.+A.+Ottauthor=H.+Sunauthor=D.+K.+Luciauthor=B.+Yuanauthor=A.+Simeonovauthor=A.+Jadhavauthor=H.+Xiaoauthor=Y.+Wangauthor=D.+J.+Maloneyauthor=Z.+Zhuang&title=A+Selective+USP1-UAF1+Inhibitor+Links+Deubiquitination+to+DNA+Damage+Responses&doi=10.1038%2Fnchembio.1455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A selective USP1-UAF1 inhibitor links deubiquitination to DNA damage responses</span></div><div class="casAuthors">Liang, Qin; Dexheimer, Thomas S.; Zhang, Ping; Rosenthal, Andrew S.; Villamil, Mark A.; You, Changjun; Zhang, Qiuting; Chen, Junjun; Ott, Christine A.; Sun, Hongmao; Luci, Diane K.; Yuan, Bifeng; Simeonov, Anton; Jadhav, Ajit; Xiao, Hui; Wang, Yinsheng; Maloney, David J.; Zhuang, Zhihao</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">298-304</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein ubiquitination and deubiquitination are central to the control of a large no. of cellular pathways and signaling networks in eukaryotes.  Although the essential roles of ubiquitination have been established in the eukaryotic DNA damage response, the deubiquitination process remains poorly defined.  Chem. probes that perturb the activity of deubiquitinases (DUBs) are needed to characterize the cellular function of deubiquitination.  Here the authors report ML323 , a highly potent inhibitor of the USP1-UAF1 deubiquitinase complex with excellent selectivity against human DUBs, deSUMOylase, deneddylase and unrelated proteases.  Using ML323, the authors interrogated deubiquitination in the cellular response to UV- and cisplatin-induced DNA damage and revealed new insights into the requirement of deubiquitination in the DNA translesion synthesis and Fanconi anemia pathways.  Moreover, ML323 potentiates cisplatin cytotoxicity in non-small cell lung cancer and osteosarcoma cells.  The authors' findings point to USP1-UAF1 as a key regulator of the DNA damage response and a target for overcoming resistance to the platinum-based anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqra2Cr2SOx7bVg90H21EOLACvtfcHk0lh2LS0m_e2tVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOgtrw%253D&md5=3486f59d79d15b197ea384e9df9291ee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1455%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DQ.%26aulast%3DDexheimer%26aufirst%3DT.%2BS.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DRosenthal%26aufirst%3DA.%2BS.%26aulast%3DVillamil%26aufirst%3DM.%2BA.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DOtt%26aufirst%3DC.%2BA.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DLuci%26aufirst%3DD.%2BK.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXiao%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DZhuang%26aufirst%3DZ.%26atitle%3DA%2520Selective%2520USP1-UAF1%2520Inhibitor%2520Links%2520Deubiquitination%2520to%2520DNA%2520Damage%2520Responses%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D298%26epage%3D304%26doi%3D10.1038%2Fnchembio.1455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Turnbull, A. P.</span>; <span class="NLM_string-name">Ioannidis, S.</span>; <span class="NLM_string-name">Krajewski, W. W.</span>; <span class="NLM_string-name">Pinto-Fernandez, A.</span>; <span class="NLM_string-name">Heride, C.</span>; <span class="NLM_string-name">Martin, A. C. L.</span>; <span class="NLM_string-name">Tonkin, L. M.</span>; <span class="NLM_string-name">Townsend, E. C.</span>; <span class="NLM_string-name">Buker, S. M.</span>; <span class="NLM_string-name">Lancia, D. R.</span>; <span class="NLM_string-name">Caravella, J. A.</span>; <span class="NLM_string-name">Toms, A. V.</span>; <span class="NLM_string-name">Charlton, T. M.</span>; <span class="NLM_string-name">Lahdenranta, J.</span>; <span class="NLM_string-name">Wilker, E.</span>; <span class="NLM_string-name">Follows, B. C.</span>; <span class="NLM_string-name">Evans, N. J.</span>; <span class="NLM_string-name">Stead, L.</span>; <span class="NLM_string-name">Alli, C.</span>; <span class="NLM_string-name">Zarayskiy, V. V.</span>; <span class="NLM_string-name">Talbot, A. C.</span>; <span class="NLM_string-name">Buckmelter, A. J.</span>; <span class="NLM_string-name">Wang, M.</span>; <span class="NLM_string-name">McKinnon, C. L.</span>; <span class="NLM_string-name">Saab, F.</span>; <span class="NLM_string-name">McGouran, J. F.</span>; <span class="NLM_string-name">Century, H.</span>; <span class="NLM_string-name">Gersch, M.</span>; <span class="NLM_string-name">Pittman, M. S.</span>; <span class="NLM_string-name">Marshall, C. G.</span>; <span class="NLM_string-name">Raynham, T. M.</span>; <span class="NLM_string-name">Simcox, M.</span>; <span class="NLM_string-name">Stewart, L. M. D.</span>; <span class="NLM_string-name">McLoughlin, S. B.</span>; <span class="NLM_string-name">Escobedo, J. A.</span>; <span class="NLM_string-name">Bair, K. W.</span>; <span class="NLM_string-name">Dinsmore, C. J.</span>; <span class="NLM_string-name">Hammonds, T. R.</span>; <span class="NLM_string-name">Kim, S.</span>; <span class="NLM_string-name">Urbe, S.</span>; <span class="NLM_string-name">Clague, M. J.</span>; <span class="NLM_string-name">Kessler, B. M.</span>; <span class="NLM_string-name">Komander, D.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis of USP7 Inhibition by Selective Small-Molecule Inhibitors</span>.  <i>Nature</i> <span class="NLM_year">2017</span>,  <span class="NLM_volume">550</span>,  <span class="NLM_fpage">481</span>– <span class="NLM_lpage">486</span>.<span class="refDoi"> DOI: 10.1038/nature24451</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnature24451" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=481-486&author=A.+P.+Turnbull&author=S.+Ioannidis&author=W.+W.+Krajewski&author=A.+Pinto-Fernandez&author=C.+Heride&author=A.+C.+L.+Martin&author=L.+M.+Tonkin&author=E.+C.+Townsend&author=S.+M.+Buker&author=D.+R.+Lancia&author=J.+A.+Caravella&author=A.+V.+Toms&author=T.+M.+Charlton&author=J.+Lahdenranta&author=E.+Wilker&author=B.+C.+Follows&author=N.+J.+Evans&author=L.+Stead&author=C.+Alli&author=V.+V.+Zarayskiy&author=A.+C.+Talbot&author=A.+J.+Buckmelter&author=M.+Wang&author=C.+L.+McKinnon&author=F.+Saab&author=J.+F.+McGouran&author=H.+Century&author=M.+Gersch&author=M.+S.+Pittman&author=C.+G.+Marshall&author=T.+M.+Raynham&author=M.+Simcox&author=L.+M.+D.+Stewart&author=S.+B.+McLoughlin&author=J.+A.+Escobedo&author=K.+W.+Bair&author=C.+J.+Dinsmore&author=T.+R.+Hammonds&author=S.+Kim&author=S.+Urbe&author=M.+J.+Clague&author=B.+M.+Kessler&author=D.+Komander&title=Nature"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature24451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1038%252Fnature24451%26sid%3Dliteratum%253Aachs%26aulast%3DTurnbull%26aufirst%3DA.%2BP.%26atitle%3DMolecular%2520Basis%2520of%2520USP7%2520Inhibition%2520by%2520Selective%2520Small-Molecule%2520Inhibitors%26btitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D481%26epage%3D486%26doi%3D10.1038%2Fnature24451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Z.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors Reveal Allosteric Regulation of USP14 via Steric Blockade</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1186</span>– <span class="NLM_lpage">1194</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0091-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fs41422-018-0091-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=30254335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslymur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=1186-1194&author=Y.+Wangauthor=Y.+Jiangauthor=S.+Dingauthor=J.+Liauthor=N.+Songauthor=Y.+Renauthor=D.+Hongauthor=C.+Wuauthor=B.+Liauthor=F.+Wangauthor=W.+Heauthor=J.+Wangauthor=Z.+Mei&title=Small+Molecule+Inhibitors+Reveal+Allosteric+Regulation+of+USP14+via+Steric+Blockade&doi=10.1038%2Fs41422-018-0091-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule inhibitors reveal allosteric regulation of USP14 via steric blockade</span></div><div class="casAuthors">Wang, Yiwei; Jiang, Yuxuan; Ding, Shan; Li, Jiawang; Song, Ningjing; Ren, Yujing; Hong, Danning; Wu, Cai; Li, Bin; Wang, Feng; He, Wei; Wang, Jiawei; Mei, Ziqing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1186-1194</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The ubiquitin system is important for drug discovery, and the discovery of selective small-mol. inhibitors of deubiquitinating enzymes (DUBs) remains an active yet extremely challenging task.  With a few exceptions, previously developed inhibitors have been found to bind the evolutionarily conserved catalytic centers of DUBs, resulting in poor selectivity.  The small mol. IU1 was the first-ever specific inhibitor identified and exhibited surprisingly excellent selectivity for USP14 over other DUBs.  However, the mol. mechanism for this selectivity was elusive.  Herein, we report the high-resoln. co-crystal structures of the catalytic domain of USP14 bound to IU1 and three IU1 derivs.  All the structures of these complexes indicate that IU1 and its analogs bind to a previously unknown steric binding site in USP14, thus blocking the access of the C-terminus of ubiquitin to the active site of USP14 and abrogating USP14 activity.  Importantly, this steric site in USP14 is very unique, as suggested by structural alignments of USP14 with several known DUB X-ray structures.  These results, in conjunction with biochem. characterization, indicate a coherent steric blockade mechanism for USP14 inhibition by compds. of the IU series.  In light of the recent report of steric blockade of USP7 by FT671, this work suggests a potential generally applicable allosteric mechanism for the regulation of DUBs via steric blockade, as showcased by our discovery of IU1-248 which is 10-fold more potent than IU1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvvGynB0ABp7Vg90H21EOLACvtfcHk0lhi8bRqtqv26w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslymur7I&md5=21b4833424dea660f0ae4a38dcb49030</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0091-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0091-x%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DHong%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMei%26aufirst%3DZ.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520Reveal%2520Allosteric%2520Regulation%2520of%2520USP14%2520via%2520Steric%2520Blockade%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D1186%26epage%3D1194%26doi%3D10.1038%2Fs41422-018-0091-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira De Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harding, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lautens, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhakumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Identification and Structure-Activity Relationship of HDAC6 Zinc-Finger Ubiquitin Binding Domain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4517</span>– <span class="NLM_lpage">4527</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVKiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4517-4527&author=R.+Ferreira+De+Freitasauthor=R.+J.+Hardingauthor=I.+Franzoniauthor=M.+Ravichandranauthor=M.+K.+Mannauthor=H.+Ouyangauthor=M.+Lautensauthor=V.+Santhakumarauthor=C.+H.+Arrowsmithauthor=M.+Schapira&title=Identification+and+Structure-Activity+Relationship+of+HDAC6+Zinc-Finger+Ubiquitin+Binding+Domain+Inhibitors&doi=10.1021%2Facs.jmedchem.8b00258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Structure-Activity Relationship of HDAC6 Zinc-Finger Ubiquitin Binding Domain Inhibitors</span></div><div class="casAuthors">Ferreira de Freitas, Renato; Harding, Rachel J.; Franzoni, Ivan; Ravichandran, Mani; Mann, Mandeep K.; Ouyang, Hui; Lautens, Mark; Santhakumar, Vijayaratnam; Arrowsmith, Cheryl H.; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4517-4527</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">HDAC6 plays a central role in the recruitment of protein aggregates for lysosomal degrdn. and is a promising target for combination therapy with proteasome inhibitors in multiple myeloma.  Pharmacol. displacing ubiquitin from the zinc-finger ubiquitin-binding domain (ZnF-UBD) of HDAC6 is an underexplored alternative to catalytic inhibition.  Here, we present the discovery of an HDAC6 ZnF-UBD-focused chem. series and its progression from virtual screening hits to low micromolar inhibitors.  A carboxylate mimicking the C-terminal extremity of ubiquitin, and an extended arom. system stacking with W1182 and R1155, are necessary for activity.  One of the compds. induced a conformational remodeling of the binding site where the primary binding pocket opens up onto a ligand-able secondary pocket that may be exploited to increase potency.  The preliminary structure-activity relationship accompanied by nine crystal structures should enable further optimization into a chem. probe to investigate the merit of targeting the ZnF-UBD of HDAC6 in multiple myeloma and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6ugnzPt7hI7Vg90H21EOLACvtfcHk0lhi8bRqtqv26w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVKiurg%253D&md5=90a444d3428edbff5b6fe48798c58988</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00258%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%2BDe%2BFreitas%26aufirst%3DR.%26aulast%3DHarding%26aufirst%3DR.%2BJ.%26aulast%3DFranzoni%26aufirst%3DI.%26aulast%3DRavichandran%26aufirst%3DM.%26aulast%3DMann%26aufirst%3DM.%2BK.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DLautens%26aufirst%3DM.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520Structure-Activity%2520Relationship%2520of%2520HDAC6%2520Zinc-Finger%2520Ubiquitin%2520Binding%2520Domain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4517%26epage%3D4527%26doi%3D10.1021%2Facs.jmedchem.8b00258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harding, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira De Freitas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franzoni, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravichandran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juarez-Ornelas, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lautens, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Von Delft, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santhakumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowmsith, C. H.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Antagonists of the Interaction between the Histone Deacetylase 6 Zinc-Finger Domain and Ubiquitin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9090</span>– <span class="NLM_lpage">9096</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00933</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00933" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1CitrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9090-9096&author=R.+J.+Hardingauthor=R.+Ferreira+De+Freitasauthor=P.+Collinsauthor=I.+Franzoniauthor=M.+Ravichandranauthor=H.+Ouyangauthor=K.+A.+Juarez-Ornelasauthor=M.+Lautensauthor=M.+Schapiraauthor=F.+Von+Delftauthor=V.+Santhakumarauthor=C.+H.+Arrowmsith&title=Small+Molecule+Antagonists+of+the+Interaction+between+the+Histone+Deacetylase+6+Zinc-Finger+Domain+and+Ubiquitin&doi=10.1021%2Facs.jmedchem.7b00933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Antagonists of the Interaction between the Histone Deacetylase 6 Zinc-Finger Domain and Ubiquitin</span></div><div class="casAuthors">Harding, Rachel J.; Ferreira de Freitas, Renato; Collins, Patrick; Franzoni, Ivan; Ravichandran, Mani; Ouyang, Hui; Juarez-Ornelas, Kevin A.; Lautens, Mark; Schapira, Matthieu; von Delft, Frank; Santhakumar, Vjayaratnam; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9090-9096</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of histone deacetylase 6 (HDAC6) have attractive therapeutic potential in numerous cancers.  HDAC6 inhibitors to date target the catalytic domains, but targeting the unique zinc-finger ubiquitin-binding domain (Zf-UBD) of HDAC6 may be an attractive alternative strategy.  Here, we developed x-ray crystallog. and biophys. assays to identify and characterize small mols. capable of binding to the Zf-UBD and compete with ubiquitin binding.  The results revealed 2 adjacent ligandable pockets of HDAC6 Zf-UBD and the 1st functional ligands for this domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-sTt_4DLZ6LVg90H21EOLACvtfcHk0lj5o5C4nWJzcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1CitrvM&md5=a65ee31124328d8f5e672cca0f11363b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00933%26sid%3Dliteratum%253Aachs%26aulast%3DHarding%26aufirst%3DR.%2BJ.%26aulast%3DFerreira%2BDe%2BFreitas%26aufirst%3DR.%26aulast%3DCollins%26aufirst%3DP.%26aulast%3DFranzoni%26aufirst%3DI.%26aulast%3DRavichandran%26aufirst%3DM.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DJuarez-Ornelas%26aufirst%3DK.%2BA.%26aulast%3DLautens%26aufirst%3DM.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DSanthakumar%26aufirst%3DV.%26aulast%3DArrowmsith%26aufirst%3DC.%2BH.%26atitle%3DSmall%2520Molecule%2520Antagonists%2520of%2520the%2520Interaction%2520between%2520the%2520Histone%2520Deacetylase%25206%2520Zinc-Finger%2520Domain%2520and%2520Ubiquitin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9090%26epage%3D9096%26doi%3D10.1021%2Facs.jmedchem.7b00933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM—A New Method for Protein Modeling and Design: Applications to Docking and Structure Prediction from the Distorted Native Conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%E2%80%94A+New+Method+for+Protein+Modeling+and+Design%3A+Applications+to+Docking+and+Structure+Prediction+from+the+Distorted+Native+Conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0lj5o5C4nWJzcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%25E2%2580%2594A%2520New%2520Method%2520for%2520Protein%2520Modeling%2520and%2520Design%253A%2520Applications%2520to%2520Docking%2520and%2520Structure%2520Prediction%2520from%2520the%2520Distorted%2520Native%2520Conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span>; <span class="NLM_string-name">Murphy, R. B.</span>; <span class="NLM_string-name">Friesner, R. A.</span>; <span class="NLM_string-name">Beard, H. S.</span>; <span class="NLM_string-name">Frye, L. L.</span>; <span class="NLM_string-name">Pollard, W. T.</span>; <span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>.  <i>J. Med. Chem.</i> <span class="NLM_year">2004</span>,  <span class="NLM_volume">47</span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>.<span class="refDoi"> DOI: 10.1021/jm030644s</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=1750-1759&author=T.+A.+Halgren&author=R.+B.+Murphy&author=R.+A.+Friesner&author=H.+S.+Beard&author=L.+L.+Frye&author=W.+T.+Pollard&author=J.+L.+Banks&title=J.+Med.+Chem."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26btitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span> <span> </span><span class="NLM_article-title">Ligand Efficiency: A Useful Metric for Lead Selection</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">430</span>– <span class="NLM_lpage">431</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(04)03069-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+Efficiency%3A+A+Useful+Metric+for+Lead+Selection&doi=10.1016%2FS1359-6446%2804%2903069-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eaxydhHnbGVHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520Efficiency%253A%2520A%2520Useful%2520Metric%2520for%2520Lead%2520Selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431%26doi%3D10.1016%2FS1359-6446%2804%2903069-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reyes-Turcu, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komander, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, K. D.</span></span> <span> </span><span class="NLM_article-title">Recognition of Polyubiquitin Isoforms by the Multiple Ubiquitin Binding Modules of Isopeptidase T</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>283</i></span>,  <span class="NLM_fpage">19581</span>– <span class="NLM_lpage">19592</span>, <span class="refDoi"> DOI: 10.1074/jbc.M800947200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1074%2Fjbc.M800947200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=18482987" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotF2ktr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=19581-19592&author=F.+E.+Reyes-Turcuauthor=J.+R.+Shanksauthor=D.+Komanderauthor=K.+D.+Wilkinson&title=Recognition+of+Polyubiquitin+Isoforms+by+the+Multiple+Ubiquitin+Binding+Modules+of+Isopeptidase+T&doi=10.1074%2Fjbc.M800947200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Polyubiquitin Isoforms by the Multiple Ubiquitin Binding Modules of Isopeptidase T</span></div><div class="casAuthors">Reyes-Turcu, Francisca E.; Shanks, John R.; Komander, David; Wilkinson, Keith D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">19581-19592</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The conjugation of polyubiquitin to target proteins acts as a signal that regulates target stability, localization, and function.  Several ubiquitin binding domains have been described, and while much is known about ubiquitin binding to the isolated domains, little is known with regard to how the domains interact with polyubiquitin in the context of full-length proteins.  Isopeptidase T (IsoT/USP5) is a deubiquitinating enzyme that is largely responsible for the disassembly of unanchored polyubiquitin in the cell.  IsoT has four ubiquitin binding domains: a zinc finger domain (ZnF UBP), which binds the proximal ubiquitin, a UBP domain that forms the active site, and two ubiquitin-assocd. (UBA) domains whose roles are unknown.  Here, we show that the UBA domains are involved in binding two different polyubiquitin isoforms, linear and K48-linked.  Using isothermal titrn. calorimetry, we show that IsoT has at least four ubiquitin binding sites for both polyubiquitin isoforms.  The thermodn. of the interactions reveal that the binding is enthalpy-driven.  Mutation of the UBA domains suggests that UBA1 and UBA2 domains of IsoT interact with the third and fourth ubiquitins in both polyubiquitin isoforms, resp.  These data suggest that recognition of the polyubiquitin isoforms by IsoT involves considerable conformational mobility in the polyubiquitin ligand, in the enzyme, or in both.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouzvkXauRD4rVg90H21EOLACvtfcHk0lgqsYMsXmoa2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotF2ktr8%253D&md5=e34bce5e2f2911f50631478d79a7b67d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M800947200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M800947200%26sid%3Dliteratum%253Aachs%26aulast%3DReyes-Turcu%26aufirst%3DF.%2BE.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DKomander%26aufirst%3DD.%26aulast%3DWilkinson%26aufirst%3DK.%2BD.%26atitle%3DRecognition%2520of%2520Polyubiquitin%2520Isoforms%2520by%2520the%2520Multiple%2520Ubiquitin%2520Binding%2520Modules%2520of%2520Isopeptidase%2520T%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D19581%26epage%3D19592%26doi%3D10.1074%2Fjbc.M800947200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1007/s10822-010-9349-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1007%2Fs10822-010-9349-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20354892" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=591-604&author=J.+R.+Greenwoodauthor=D.+Calkinsauthor=A.+P.+Sullivanauthor=J.+C.+Shelley&title=Towards+the+comprehensive%2C+rapid%2C+and+accurate+prediction+of+the+favorable+tautomeric+states+of+drug-like+molecules+in+aqueous+solution&doi=10.1007%2Fs10822-010-9349-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Towards the comprehensive, rapid, and accurate prediction of the favorable tautomeric states of drug-like molecules in aqueous solution</span></div><div class="casAuthors">Greenwood, Jeremy R.; Calkins, David; Sullivan, Arron P.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">591-604</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Generating the appropriate protonation states of drug-like mols. in soln. is important for success in both ligand- and structure-based virtual screening.  Screening collections of millions of compds. requires a method for detg. tautomers and their energies that is sufficiently rapid, accurate, and comprehensive.  To maximize enrichment, the lowest energy tautomers must be detd. from heterogeneous input, without over-enumerating unfavorable states.  While computationally expensive, the d. functional theory (DFT) method M06-2X/aug-cc-pVTZ(-f) [PB-SCRF] provides accurate energies for enumerated model tautomeric systems.  The empirical Hammett-Taft methodol. can very rapidly extrapolate substituent effects from model systems to drug-like mols. via the relationship between pKT and pKa.  Combining the 2 complementary approaches transforms the tautomer problem from a scientific challenge to one of engineering scale-up, and avoids issues that arise due to the very limited no. of measured pKT values, esp. for the complicated heterocycles often favored by medicinal chemists for their novelty and versatility.  Several hundreds of pre-calcd. tautomer energies and substituent pKa effects are tabulated in databases for use in structural adjustment by the program Epik, which treats tautomers as a subset of the larger problem of the protonation states in aq. ensembles and their energy penalties.  Accuracy and coverage is continually improved and expanded by parameterizing new systems of interest using DFT and exptl. data.  Recommendations are made for how to best incorporate tautomers in mol. design and virtual screening workflows.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigkzHeo8JY7Vg90H21EOLACvtfcHk0lgqsYMsXmoa2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnsFGqtbo%253D&md5=1d7bc0f966ca793d6be80554868367b8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2Fs10822-010-9349-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-010-9349-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DCalkins%26aufirst%3DD.%26aulast%3DSullivan%26aufirst%3DA.%2BP.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DTowards%2520the%2520comprehensive%252C%2520rapid%252C%2520and%2520accurate%2520prediction%2520of%2520the%2520favorable%2520tautomeric%2520states%2520of%2520drug-like%2520molecules%2520in%2520aqueous%2520solution%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2010%26volume%3D24%26spage%3D591%26epage%3D604%26doi%3D10.1007%2Fs10822-010-9349-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: A Software Program for PKa Prediction and Protonation State Generation for Drug-like Molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+A+Software+Program+for+PKa+Prediction+and+Protonation+State+Generation+for+Drug-like+Molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lgqsYMsXmoa2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520A%2520Software%2520Program%2520for%2520PKa%2520Prediction%2520and%2520Protonation%2520State%2520Generation%2520for%2520Drug-like%2520Molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arntson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, L. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisniewski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Protein-Observed 19F-NMR for Fragment Screening, Affinity Quantification and Druggability Assessment</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1414</span>– <span class="NLM_lpage">1427</span>, <span class="refDoi"> DOI: 10.1038/nprot.2016.079</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1038%2Fnprot.2016.079" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=27414758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A280%3ADC%252BC2s3gvVOmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1414-1427&author=C.+T.+Geeauthor=K.+E.+Arntsonauthor=A.+K.+Urickauthor=N.+K.+Mishraauthor=L.+M.+L.+Hawkauthor=A.+J.+Wisniewskiauthor=W.+C.+K.+Pomerantz&title=Protein-Observed+19F-NMR+for+Fragment+Screening%2C+Affinity+Quantification+and+Druggability+Assessment&doi=10.1038%2Fnprot.2016.079"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-observed (19)F-NMR for fragment screening, affinity quantification and druggability assessment</span></div><div class="casAuthors">Gee Clifford T; Arntson Keith E; Urick Andrew K; Mishra Neeraj K; Hawk Laura M L; Pomerantz William C K; Wisniewski Andrea J</div><div class="citationInfo"><span class="NLM_cas:title">Nature protocols</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1414-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">NMR spectroscopy can be used to quantify the binding affinity between proteins and low-complexity molecules, termed 'fragments'; this versatile screening approach allows researchers to assess the druggability of new protein targets.  Protein-observed (19)F-NMR (PrOF NMR) using (19)F-labeled amino acids generates relatively simple spectra that are able to provide dynamic structural information toward understanding protein folding and function.  Changes in these spectra upon the addition of fragment molecules can be observed and quantified.  This protocol describes the sequence-selective labeling of three proteins (the first bromodomains of Brd4 and BrdT, and the KIX domain of the CREB-binding protein) using commercially available fluorinated aromatic amino acids and fluorinated precursors as example applications of the method developed by our research group.  Fragment-screening approaches are discussed, as well as Kd determination, ligand-efficiency calculations and druggability assessment, i.e., the ability to target these proteins using small-molecule ligands.  Experiment times on the order of a few minutes and the simplicity of the NMR spectra obtained make this approach well-suited to the investigation of small- to medium-sized proteins, as well as the screening of multiple proteins in the same experiment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTi7QXqlmq0fkPI-fCgwosgfW6udTcc2ebHPYa2-X81lLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3gvVOmsw%253D%253D&md5=7fd74fe2067137a91782eab12138f1e7</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.079&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.079%26sid%3Dliteratum%253Aachs%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DArntson%26aufirst%3DK.%2BE.%26aulast%3DUrick%26aufirst%3DA.%2BK.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DProtein-Observed%252019F-NMR%2520for%2520Fragment%2520Screening%252C%2520Affinity%2520Quantification%2520and%2520Druggability%2520Assessment%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1414%26epage%3D1427%26doi%3D10.1038%2Fnprot.2016.079" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgwvqEv7vR4Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How Good Are My Data and What Is the Resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1214</span>, <span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+Good+Are+My+Data+and+What+Is+the+Resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgwvqEv7vR4Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520Good%2520Are%2520My%2520Data%2520and%2520What%2520Is%2520the%2520Resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">Xia2: An Expert System for Macromolecular Crystallography Data Reduction</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1107/S0021889809045701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0021889809045701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2010&pages=186-190&author=G.+Winter&title=Xia2%3A+An+Expert+System+for+Macromolecular+Crystallography+Data+Reduction&doi=10.1107%2FS0021889809045701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">xia2: An expert system for macromolecular crystallography data reduction</span></div><div class="casAuthors">Winter, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-190, S186/1-S186/5</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">An expert system for macromol. crystallog. data redn. is presented, which builds on existing software to automate the complete data redn. process from images to merged structure factor amplitudes.  This can automatically identify multi-wedge, multi-pass and multiwavelength data sets and includes explicit procedures to test for crystallog. special cases.  With the push towards high-throughput crystallog. at synchrotron beamlines and automation of structure soln., the ability to reduce data with no user input fills an important gap in the pipeline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOi3u0ydXbSrVg90H21EOLACvtfcHk0lgwvqEv7vR4Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFalsrk%253D&md5=27771caa813f3e828abdf2a331ae55e3</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0021889809045701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889809045701%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DXia2%253A%2520An%2520Expert%2520System%2520for%2520Macromolecular%2520Crystallography%2520Data%2520Reduction%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2010%26volume%3D43%26spage%3D186%26epage%3D190%26doi%3D10.1107%2FS0021889809045701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhzkV8gA2GutA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wojdyr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, A.</span></span> <span> </span><span class="NLM_article-title">DIMPLE - a Pipeline for the Rapid Generation of Difference Maps from Protein Crystals with Putatively Bound Ligands</span>. <i>Acta Crystallogr., Sect. A: Found. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">s299</span>, <span class="refDoi"> DOI: 10.1107/S0108767313097419</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0108767313097419" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=23459360" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=s299&author=M.+Wojdyrauthor=R.+Keeganauthor=G.+Winterauthor=A.+Ashton&title=DIMPLE+-+a+Pipeline+for+the+Rapid+Generation+of+Difference+Maps+from+Protein+Crystals+with+Putatively+Bound+Ligands&doi=10.1107%2FS0108767313097419"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0108767313097419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767313097419%26sid%3Dliteratum%253Aachs%26aulast%3DWojdyr%26aufirst%3DM.%26aulast%3DKeegan%26aufirst%3DR.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DAshton%26aufirst%3DA.%26atitle%3DDIMPLE%2520-%2520a%2520Pipeline%2520for%2520the%2520Rapid%2520Generation%2520of%2520Difference%2520Maps%2520from%2520Protein%2520Crystals%2520with%2520Putatively%2520Bound%2520Ligands%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3Ds299%26doi%3D10.1107%2FS0108767313097419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gražulis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merkys, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaitkus, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">AceDRG: A Stereochemical Description Generator for Ligands</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">122</span>, <span class="refDoi"> DOI: 10.1107/S2059798317000067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS2059798317000067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2017&pages=112-122&author=F.+Longauthor=R.+A.+Nichollsauthor=P.+Emsleyauthor=S.+Gra%C5%BEulisauthor=A.+Merkysauthor=A.+Vaitkusauthor=G.+N.+Murshudov&title=AceDRG%3A+A+Stereochemical+Description+Generator+for+Ligands&doi=10.1107%2FS2059798317000067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">AceDRG: a stereochemical description generator for ligands</span></div><div class="casAuthors">Long, Fei; Nicholls, Robert A.; Emsley, Paul; Grazulis, Saulius; Merkys, Andrius; Vaitkus, Antanas; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">112-122</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The program AceDRG is designed for the derivation of stereochem. information about small mols.  It uses local chem. and topol. environment-based atom typing to derive and organize bond lengths and angles from a small-mol. database: the Crystallog. Open Database (COD).  Information about the hybridization states of atoms, whether they belong to small rings (up to seven-membered rings), ring aromaticity and nearest-neighbor information is encoded in the atom types.  All atoms from the COD have been classified according to the generated atom types.  All bonds and angles have also been classified according to the atom types and, in a certain sense, bond types.  Derived data are tabulated in a machine-readable form that is freely available from CCP4.  AceDRG can also generate stereochem. information, provided that the basic bonding pattern of a ligand is known.  The basic bonding pattern is perceived from one of the computational chem. file formats, including SMILES, mmCIF, SDF MOL and SYBYL MOL2 files.  Using the bonding chem., atom types, and bond and angle tables generated from the COD, AceDRG derives the 'ideal' bond lengths, angles, plane groups, arom. rings and chirality information, and writes them to an mmCIF file that can be used by the refinement program REFMAC5 and the model-building program Coot.  Other refinement and model-building programs such as PHENIX and BUSTER can also use these files.  AceDRG also generates one or more coordinate sets corresponding to the most favorable conformation(s) of a given ligand.  AceDRG employs RDKit for chem. perception and for initial conformation generation, as well as for the interpretation of SMILES strings, SDF MOL and SYBYL MOL2 files.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow0LaF0rLfdLVg90H21EOLACvtfcHk0lhzkV8gA2GutA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitlGgu78%253D&md5=0a581d62dd3b00004e37be490ee71fe8</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS2059798317000067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798317000067%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DF.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DGra%25C5%25BEulis%26aufirst%3DS.%26aulast%3DMerkys%26aufirst%3DA.%26aulast%3DVaitkus%26aufirst%3DA.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DAceDRG%253A%2520A%2520Stereochemical%2520Description%2520Generator%2520for%2520Ligands%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2017%26volume%3D73%26spage%3D112%26epage%3D122%26doi%3D10.1107%2FS2059798317000067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhzkV8gA2GutA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the Refinement of Macromolecular Crystal Structures</span>. <i>Acta Crystallogr., Sect D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/S0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+Refinement+of+Macromolecular+Crystal+Structures&doi=10.1107%2FS0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0li1W1ic-ge1dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520Refinement%2520of%2520Macromolecular%2520Crystal%2520Structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2FS0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span>. <i>Acta Crystallogr. Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0li1W1ic-ge1dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, K. J.</span>; <span class="NLM_string-name">Chow, D. E.</span>; <span class="NLM_string-name">Xu, H.</span>; <span class="NLM_string-name">Dror, R. O.</span>; <span class="NLM_string-name">Eastwood, M. P.</span>; <span class="NLM_string-name">Gregersen, B. A.</span>; <span class="NLM_string-name">Klepeis, J. L.</span>; <span class="NLM_string-name">Kolossvary, I.</span>; <span class="NLM_string-name">Moraes, M. A.</span>; <span class="NLM_string-name">Sacerdoti, F. D.</span>; <span class="NLM_string-name">Salmon, J. K.</span>; <span class="NLM_string-name">Shan, Y.</span>; <span class="NLM_string-name">Shaw, D. E.</span></span> In  <i>Molecular Dynamics---Scalable Algorithms for Molecular Dynamics Simulations on Commodity Clusters</i>, Proceedings of the 2006 ACM/IEEE Conference on Supercomputing - SC ’06, <span class="NLM_year">2006</span>; p  <span class="NLM_fpage">84</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=84&author=K.+J.+Bowers&author=D.+E.+Chow&author=H.+Xu&author=R.+O.+Dror&author=M.+P.+Eastwood&author=B.+A.+Gregersen&author=J.+L.+Klepeis&author=I.+Kolossvary&author=M.+A.+Moraes&author=F.+D.+Sacerdoti&author=J.+K.+Salmon&author=Y.+Shan&author=D.+E.+Shaw&title=Molecular+Dynamics%2D%2D-Scalable+Algorithms+for+Molecular+Dynamics+Simulations+on+Commodity+Clusters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DK.%2BJ.%26jtitle%3DMolecular%2520Dynamics---Scalable%2520Algorithms%2520for%2520Molecular%2520Dynamics%2520Simulations%2520on%2520Commodity%2520Clusters%26date%3D2006%26spage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felts, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillip, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallicchio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, R. M.</span></span> <span> </span><span class="NLM_article-title">Integrated Modeling Program, Applied Chemical Theory (IMPACT)</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1752</span>– <span class="NLM_lpage">1780</span>, <span class="refDoi"> DOI: 10.1002/jcc.20292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1002%2Fjcc.20292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=16211539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1752-1780&author=J.+L.+Banksauthor=H.+S.+Beardauthor=Y.+Caoauthor=A.+E.+Choauthor=W.+Dammauthor=R.+Faridauthor=A.+K.+Feltsauthor=T.+A.+Halgrenauthor=D.+T.+Mainzauthor=J.+R.+Mapleauthor=R.+Murphyauthor=D.+M.+Phillipauthor=M.+P.+Repaskyauthor=L.+Y.+Zhangauthor=B.+J.+Berneauthor=R.+A.+Friesnerauthor=E.+Gallicchioauthor=R.+M.+Levy&title=Integrated+Modeling+Program%2C+Applied+Chemical+Theory+%28IMPACT%29&doi=10.1002%2Fjcc.20292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated modeling program, applied chemical theory (IMPACT)</span></div><div class="casAuthors">Banks, Jay L.; Beard, Hege S.; Cao, Yixiang; Cho, Art E.; Damm, Wolfgang; Farid, Ramy; Felts, Anthony K.; Halgren, Thomas A.; Mainz, Daniel T.; Maple, Jon R.; Murphy, Robert; Philipp, Dean M.; Repasky, Matthew P.; Zhang, Linda Y.; Berne, Bruce J.; Friesner, Richard A.; Gallicchio, Emilio; Levy, Ronald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1752-1780</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  We provide an overview of the IMPACT mol. mechanics program with an emphasis on recent developments and a description of its current functionality.  With respect to core mol. mechanics technologies we include a status report for the fixed charge and polarizable force fields that can be used with the program and illustrate how the force fields, when used together with new atom typing and parameter assignment modules, have greatly expanded the coverage of org. compds. and medicinally relevant ligands.  As we discuss in this review, explicit solvent simulations have been used to guide our design of implicit solvent models based on the generalized Born framework and a novel nonpolar estimator that have recently been incorporated into the program.  With IMPACT it is possible to use several different advanced conformational sampling algorithms based on combining features of mol. dynamics and Monte Carlo simulations.  The program includes two specialized mol. mechanics modules: Glide, a high-throughput docking program, and QSite, a mixed quantum mechanics/mol. mechanics module.  These modules employ the IMPACT infrastructure as a starting point for the construction of the protein model and assignment of mol. mechanics parameters, but have then been developed to meet specialized objectives with respect to sampling and the energy function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZX2FzcGWGM7Vg90H21EOLACvtfcHk0li1W1ic-ge1dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbI&md5=0094f15727c7fb77a0fae16ccf8292f0</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20292%26sid%3Dliteratum%253Aachs%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DA.%2BE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DFelts%26aufirst%3DA.%2BK.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DMaple%26aufirst%3DJ.%2BR.%26aulast%3DMurphy%26aufirst%3DR.%26aulast%3DPhillip%26aufirst%3DD.%2BM.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DZhang%26aufirst%3DL.%2BY.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DGallicchio%26aufirst%3DE.%26aulast%3DLevy%26aufirst%3DR.%2BM.%26atitle%3DIntegrated%2520Modeling%2520Program%252C%2520Applied%2520Chemical%2520Theory%2520%2528IMPACT%2529%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1752%26epage%3D1780%26doi%3D10.1002%2Fjcc.20292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. H.</span></span> <span> </span><span class="NLM_article-title">Efficient Estimation of Free Energy Differences from Monte Carlo Data</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(76)90078-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2F0021-9991%2876%2990078-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1976&pages=245-268&author=C.+H.+Bennett&title=Efficient+Estimation+of+Free+Energy+Differences+from+Monte+Carlo+Data&doi=10.1016%2F0021-9991%2876%2990078-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2876%2990078-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252876%252990078-4%26sid%3Dliteratum%253Aachs%26aulast%3DBennett%26aufirst%3DC.%2BH.%26atitle%3DEfficient%2520Estimation%2520of%2520Free%2520Energy%2520Differences%2520from%2520Monte%2520Carlo%2520Data%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1976%26volume%3D22%26spage%3D245%26epage%3D268%26doi%3D10.1016%2F0021-9991%2876%2990078-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paliwal, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirts, M. R.</span></span> <span> </span><span class="NLM_article-title">A Benchmark Test Set for Alchemical Free Energy Transformations and Its Use to Quantify Error in Common Free Energy Methods</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">4115</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1021/ct2003995</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct2003995" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=4115-4134&author=H.+Paliwalauthor=M.+R.+Shirts&title=A+Benchmark+Test+Set+for+Alchemical+Free+Energy+Transformations+and+Its+Use+to+Quantify+Error+in+Common+Free+Energy+Methods&doi=10.1021%2Fct2003995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A Benchmark Test Set for Alchemical Free Energy Transformations and Its Use to Quantify Error in Common Free Energy Methods</span></div><div class="casAuthors">Paliwal, Himanshu; Shirts, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4115-4134</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is a significant need for improved tools to validate thermophys. quantities computed via mol. simulation.  In this paper we present the initial version of a benchmark set of testing methods for calcg. free energies of mol. transformation in soln.  This set is based on mol. changes common to many mol. design problems, such as insertion and deletion of at. sites and changing at. partial charges.  We use this benchmark set to compare the statistical efficiency, reliability, and quality of uncertainty ests. for a no. of published free energy methods, including thermodn. integration, free energy perturbation, the Bennett acceptance ratio (BAR) and its multistate equiv. MBAR.  We identify MBAR as the consistently best performing method, though other methods are frequently comparable in reliability and accuracy in many cases.  We demonstrate that assumptions of Gaussian distributed errors in free energies are usually valid for most methods studied.  We demonstrate that bootstrap error estn. is a robust and useful technique for estg. statistical variance for all free energy methods studied.  This benchmark set is provided in a no. of different file formats with the hope of becoming a useful and general tool for method comparisons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxxPO-qPI-gLVg90H21EOLACvtfcHk0lgM_Wc4mb6GHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqtrvM&md5=41383373300216165a7f1724ad535451</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fct2003995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct2003995%26sid%3Dliteratum%253Aachs%26aulast%3DPaliwal%26aufirst%3DH.%26aulast%3DShirts%26aufirst%3DM.%2BR.%26atitle%3DA%2520Benchmark%2520Test%2520Set%2520for%2520Alchemical%2520Free%2520Energy%2520Transformations%2520and%2520Its%2520Use%2520to%2520Quantify%2520Error%2520in%2520Common%2520Free%2520Energy%2520Methods%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D4115%26epage%3D4134%26doi%3D10.1021%2Fct2003995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidhoff, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulber, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodendorf, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorthiois, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martoglio, B.</span></span> <span> </span><span class="NLM_article-title">A Sensitive Fluorescence Intensity Assay for Deubiquitinating Proteases Using Ubiquitin-Rhodamine110-Glycine as Substrate</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2007.07.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=10.1016%2Fj.ab.2007.07.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=17869210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1GlurjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2007&pages=201-207&author=U.+Hassiepenauthor=U.+Eidhoffauthor=G.+Mederauthor=J.+F.+Bulberauthor=A.+Heinauthor=U.+Bodendorfauthor=E.+Lorthioisauthor=B.+Martoglio&title=A+Sensitive+Fluorescence+Intensity+Assay+for+Deubiquitinating+Proteases+Using+Ubiquitin-Rhodamine110-Glycine+as+Substrate&doi=10.1016%2Fj.ab.2007.07.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">A sensitive fluorescence intensity assay for deubiquitinating proteases using ubiquitin-rhodamine110-glycine as substrate</span></div><div class="casAuthors">Hassiepen, Ulrich; Eidhoff, Ulf; Meder, Gabriele; Bulber, Jean-Francois; Hein, Andreas; Bodendorf, Ursula; Lorthiois, Edwige; Martoglio, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-207</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The dynamic modification of proteins with ubiquitin is a key regulation paradigm in eukaryotic cells that controls stability, localization, and function of the vast majority of intracellular proteins.  Here we describe a robust fluorescence intensity assay for monitoring the enzymic activity of deubiquitinating proteases, which reverse ubiquitin modifications and comprise over 100 members in humans.  The assay was developed for the catalytic domain of human ubiquitin-specific protease 2 (USP2) and human ubiquitin carboxyterminal hydrolase L3 (UCH-L3), and makes use of the novel substrate ubiquitin-rhodamine110-glycine.  The latter combines the advantages of a high dynamic range and beneficial optical properties.  Its enzymic behavior is characterized by the kinetic consts. Km = 1.5 μM, kcat = 0.53 s-1 and kcat/Km = 3.5×105 M-1 s-1 for USP2 and Km = 34 nM, kcat = 4.72 s-1, and kcat/Km = 1.4×108 M-1 s-1 for UCH-L3.  This new assay is suitable for inhibitor screening and characterizations, and has been established for the 384-well plate format using protease concns. of 120 pM for USP2 and 1 pM for UCH-L3 and substrate concns. of 100 nM for both enzymes.  Due to the low protease concns. and high sensitivity, this assay would allow the detn. of inhibitory consts. in the subnanomolar range.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM2_cU_l6AT7Vg90H21EOLACvtfcHk0lgM_Wc4mb6GHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1GlurjK&md5=e9a430d206d99d4b8bef17030139e61a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2007.07.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2007.07.034%26sid%3Dliteratum%253Aachs%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DEidhoff%26aufirst%3DU.%26aulast%3DMeder%26aufirst%3DG.%26aulast%3DBulber%26aufirst%3DJ.%2BF.%26aulast%3DHein%26aufirst%3DA.%26aulast%3DBodendorf%26aufirst%3DU.%26aulast%3DLorthiois%26aufirst%3DE.%26aulast%3DMartoglio%26aufirst%3DB.%26atitle%3DA%2520Sensitive%2520Fluorescence%2520Intensity%2520Assay%2520for%2520Deubiquitinating%2520Proteases%2520Using%2520Ubiquitin-Rhodamine110-Glycine%2520as%2520Substrate%26jtitle%3DAnal.%2520Biochem.%26date%3D2007%26volume%3D371%26spage%3D201%26epage%3D207%26doi%3D10.1016%2Fj.ab.2007.07.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G45','PDB','2G45'); return false;">PDB: 2G45</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXT','PDB','6DXT'); return false;">PDB: 6DXT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6NFT','PDB','6NFT'); return false;">PDB: 6NFT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DXH','PDB','6DXH'); return false;">PDB: 6DXH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G43" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G43','PDB','2G43'); return false;">PDB: 2G43</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P9G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P9G','PDB','6P9G'); return false;">PDB: 6P9G</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i22"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_61548"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00988">10.1021/acs.jmedchem.9b00988</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">X-ray crystallography data collection and refinement statistics and NMR screening data and UPLC MS data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_002.csv">CSV</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXT (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_003.cif">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXT-sf (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_004.cif">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6NFT (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_005.cif">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6NFT-sf (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_006.cif">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXH (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_007.cif">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6DXH-sf (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_008.cif">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6P9G (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_009.cif">CIF</a>)</p></li><li><p class="inline">USP5 co-crystal structure 6P9G-sf (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_010.cif">CIF</a>)</p></li><li><p class="inline">3D PDB file used for preliminary docking studies “+W404” (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_011.pdb">PDB</a>)</p></li><li><p class="inline">3D PDB file used for preliminary docking studies “–W404” (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_012.pdb">PDB</a>)</p></li><li><p class="inline">3D PDB file used for preliminary docking studies “stacked” (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_013.pdb">PDB</a>)</p></li><li><p class="inline">3D PDB file used for virtual screen compound <b>1</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_014.pdb">PDB</a>)</p></li><li><p class="inline">3D PDB file used for virtual screen compound <b>5</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_015.pdb">PDB</a>)</p></li><li><p class="inline">3D PDB file used for virtual screen compound <b>7</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_016.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_001.pdf">jm9b00988_si_001.pdf (2.24 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_002.csv">jm9b00988_si_002.csv (1.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_003.cif">jm9b00988_si_003.cif (140.07 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_004.cif">jm9b00988_si_004.cif (1.68 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_005.cif">jm9b00988_si_005.cif (237.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_006.cif">jm9b00988_si_006.cif (2.82 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_007.cif">jm9b00988_si_007.cif (263.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_008.cif">jm9b00988_si_008.cif (4.63 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_009.cif">jm9b00988_si_009.cif (181.32 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_010.cif">jm9b00988_si_010.cif (7.5 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_011.pdb">jm9b00988_si_011.pdb (237.07 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_012.pdb">jm9b00988_si_012.pdb (141.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_013.pdb">jm9b00988_si_013.pdb (236.68 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_014.pdb">jm9b00988_si_014.pdb (159.54 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_015.pdb">jm9b00988_si_015.pdb (167.39 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00988/suppl_file/jm9b00988_si_016.pdb">jm9b00988_si_016.pdb (160.53 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p><b>1</b> (6DXT), <b>5</b> (6NFT), <b>7</b> (6DXH), <b>21</b> (6P9G)</p><p class="last">Authors will release atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00988&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-22%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00988" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00988" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac75cb97d3d8e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
