
<html lang="en"     class="pb-page"  data-request-id="cf71bfe8-b22c-4d11-98f3-2da4fb3662be"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01403;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-5;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="DDX3X Helicase Inhibitors as a New Strategy To Fight the West Nile Virus Infection" /></meta><meta name="dc.Creator" content="Annalaura  Brai" /></meta><meta name="dc.Creator" content="Francesco  Martelli" /></meta><meta name="dc.Creator" content="Valentina  Riva" /></meta><meta name="dc.Creator" content="Anna  Garbelli" /></meta><meta name="dc.Creator" content="Roberta  Fazi" /></meta><meta name="dc.Creator" content="Claudio  Zamperini" /></meta><meta name="dc.Creator" content="Alessandro  Pollutri" /></meta><meta name="dc.Creator" content="Lucia  Falsitta" /></meta><meta name="dc.Creator" content="Stefania  Ronzini" /></meta><meta name="dc.Creator" content="Laura  Maccari" /></meta><meta name="dc.Creator" content="Giovanni  Maga" /></meta><meta name="dc.Creator" content="Simone  Giannecchini" /></meta><meta name="dc.Creator" content="Maurizio  Botta" /></meta><meta name="dc.Description" content="Increased frequency of arbovirus outbreaks in the last 10 years represents an important emergence for global health. Climate warming, extensive urbanization of tropical regions, and human migration..." /></meta><meta name="Description" content="Increased frequency of arbovirus outbreaks in the last 10 years represents an important emergence for global health. Climate warming, extensive urbanization of tropical regions, and human migration..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 5, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01403" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01403" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01403" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01403" /></link>
        
    
    

<title>DDX3X Helicase Inhibitors as a New Strategy To Fight the West Nile Virus Infection | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01403" /></meta><meta property="og:title" content="DDX3X Helicase Inhibitors as a New Strategy To Fight the West Nile Virus Infection" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0010.jpeg" /></meta><meta property="og:description" content="Increased frequency of arbovirus outbreaks in the last 10 years represents an important emergence for global health. Climate warming, extensive urbanization of tropical regions, and human migration flows facilitate the expansion of anthropophilic mosquitos and the emerging or re-emerging of new viral infections. Only recently the human adenosinetriphosphatase/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3X) emerged as a novel therapeutic target in the fight against infectious diseases. Herein, starting from our previous studies, a new family of DDX3X inhibitors was designed, synthesized, validated on the target enzyme, and evaluated against the West Nile virus (WNV) infection. Time of addition experiments after virus infection indicated that the compounds exerted their antiviral activities after the entry process, likely at the protein translation step of WNV replication. Finally, the most interesting compounds were then analyzed for their in vitro pharmacokinetic parameters, revealing favorable absorption, distribution, metabolism, and excretion values. The good safety profile together with a good activity against WNV for which no treatments are currently available, make this new class of molecules a good starting point for further in vivo studies." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01403"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01403">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01403&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01403&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01403&amp;href=/doi/10.1021/acs.jmedchem.8b01403" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2333-2347</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01353" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01491" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">DDX3X Helicase Inhibitors as a New Strategy To Fight the West Nile Virus Infection</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Annalaura Brai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Annalaura Brai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento Biotecnologie, Chimica e Farmacia, UniversitÃ  degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</div><div class="loa-info-affiliations-info">Lead Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Annalaura++Brai">Annalaura Brai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6395-9348" title="Orcid link">http://orcid.org/0000-0001-6395-9348</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Francesco Martelli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francesco Martelli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Experimental and Clinical Medicine, University of Florence, I-50134 Florence, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francesco++Martelli">Francesco Martelli</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Valentina Riva</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Valentina Riva</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Istituto di Genetica Molecolare, IGM-CNR, Via Abbiategrasso 207, I-27100 Pavia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Valentina++Riva">Valentina Riva</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Anna Garbelli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anna Garbelli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Istituto di Genetica Molecolare, IGM-CNR, Via Abbiategrasso 207, I-27100 Pavia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anna++Garbelli">Anna Garbelli</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Roberta Fazi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roberta Fazi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento Biotecnologie, Chimica e Farmacia, UniversitÃ  degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roberta++Fazi">Roberta Fazi</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Claudio Zamperini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Claudio Zamperini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento Biotecnologie, Chimica e Farmacia, UniversitÃ  degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</div><div class="loa-info-affiliations-info">Lead Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Claudio++Zamperini">Claudio Zamperini</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Alessandro Pollutri</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alessandro Pollutri</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento Biotecnologie, Chimica e Farmacia, UniversitÃ  degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alessandro++Pollutri">Alessandro Pollutri</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lucia Falsitta</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lucia Falsitta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento Biotecnologie, Chimica e Farmacia, UniversitÃ  degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lucia++Falsitta">Lucia Falsitta</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Stefania Ronzini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefania Ronzini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento Biotecnologie, Chimica e Farmacia, UniversitÃ  degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefania++Ronzini">Stefania Ronzini</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Laura Maccari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura Maccari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Maccari">Laura Maccari</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Giovanni Maga</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Giovanni Maga</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Istituto di Genetica Molecolare, IGM-CNR, Via Abbiategrasso 207, I-27100 Pavia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Giovanni++Maga">Giovanni Maga</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8092-1552" title="Orcid link">http://orcid.org/0000-0001-8092-1552</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Simone Giannecchini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simone Giannecchini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Experimental and Clinical Medicine, University of Florence, I-50134 Florence, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simone++Giannecchini">Simone Giannecchini</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Maurizio Botta</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maurizio Botta</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dipartimento Biotecnologie, Chimica e Farmacia, UniversitÃ  degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</div><div class="loa-info-affiliations-info">Lead Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</div><div class="loa-info-affiliations-info">Biotechnology College of Science and Technology, Temple University, BioLife Science Building, Suite 333, 1900 North 12th Street, Philadelphia, Pennsylvania 19122, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#5d3f3229293c73303c282f342734321d3a303c3431733e3230"><span class="__cf_email__" data-cfemail="b6d4d9c2c2d798dbd7c3c4dfccdfd9f6d1dbd7dfda98d5d9db">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maurizio++Botta">Maurizio Botta</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0456-6995" title="Orcid link">http://orcid.org/0000-0003-0456-6995</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01403&amp;href=/doi/10.1021%2Facs.jmedchem.8b01403" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2333â2347</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 5, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 September 2018</li><li><span class="item_label"><b>Published</b> online</span>5 February 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 March 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01403" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01403</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2333%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAnnalaura%2BBrai%252C%2BFrancesco%2BMartelli%252C%2BValentina%2BRiva%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D5%26contentID%3Dacs.jmedchem.8b01403%26title%3DDDX3X%2BHelicase%2BInhibitors%2Bas%2Ba%2BNew%2BStrategy%2BTo%2BFight%2Bthe%2BWest%2BNile%2BVirus%2BInfection%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2347%26publicationDate%3DMarch%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01403"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1864</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01403" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;DDX3X Helicase Inhibitors as a New Strategy To Fight the West Nile Virus Infection&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Annalaura&quot;,&quot;last_name&quot;:&quot;Brai&quot;},{&quot;first_name&quot;:&quot;Francesco&quot;,&quot;last_name&quot;:&quot;Martelli&quot;},{&quot;first_name&quot;:&quot;Valentina&quot;,&quot;last_name&quot;:&quot;Riva&quot;},{&quot;first_name&quot;:&quot;Anna&quot;,&quot;last_name&quot;:&quot;Garbelli&quot;},{&quot;first_name&quot;:&quot;Roberta&quot;,&quot;last_name&quot;:&quot;Fazi&quot;},{&quot;first_name&quot;:&quot;Claudio&quot;,&quot;last_name&quot;:&quot;Zamperini&quot;},{&quot;first_name&quot;:&quot;Alessandro&quot;,&quot;last_name&quot;:&quot;Pollutri&quot;},{&quot;first_name&quot;:&quot;Lucia&quot;,&quot;last_name&quot;:&quot;Falsitta&quot;},{&quot;first_name&quot;:&quot;Stefania&quot;,&quot;last_name&quot;:&quot;Ronzini&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Maccari&quot;},{&quot;first_name&quot;:&quot;Giovanni&quot;,&quot;last_name&quot;:&quot;Maga&quot;},{&quot;first_name&quot;:&quot;Simone&quot;,&quot;last_name&quot;:&quot;Giannecchini&quot;},{&quot;first_name&quot;:&quot;Maurizio&quot;,&quot;last_name&quot;:&quot;Botta&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;2333-2347&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01403&quot;},&quot;abstract&quot;:&quot;Increased frequency of arbovirus outbreaks in the last 10 years represents an important emergence for global health. Climate warming, extensive urbanization of tropical regions, and human migration flows facilitate the expansion of anthropophilic mosquitos and the emerging or re-emerging of new viral infections. Only recently the human adenosinetriphosphatase/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3X) emerged as a novel therapeutic target in the fight against infectious diseases. Herein, starting from our previous studies, a new family of DDX3X inhibitors was designed, synthesized, validated on the target enzyme, and evaluated against the West Nile virus (WNV) infection. Time of addition experiments after virus infection indicated that the compounds exerted their antiviral activities after the entry process, likely at the protein translation step of WNV replication. Finally, the most interesting compounds were then analyzed for their in vitro pharmacokinetic parameters, revealing favorable absorptio&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01403&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01403" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01403&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01403" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01403&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01403" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01403&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01403&amp;href=/doi/10.1021/acs.jmedchem.8b01403" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01403" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01403" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01403%26sid%3Dliteratum%253Aachs%26pmid%3D30721061%26genre%3Darticle%26aulast%3DBrai%26date%3D2019%26atitle%3DDDX3X%2BHelicase%2BInhibitors%2Bas%2Ba%2BNew%2BStrategy%2BTo%2BFight%2Bthe%2BWest%2BNile%2BVirus%2BInfection%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D5%26spage%3D2333%26epage%3D2347%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291430" title="Infectious diseases">Infectious diseases</a>,</li><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/jmcmar.2019.62.issue-5/20190314/jmcmar.2019.62.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Increased frequency of arbovirus outbreaks in the last 10 years represents an important emergence for global health. Climate warming, extensive urbanization of tropical regions, and human migration flows facilitate the expansion of anthropophilic mosquitos and the emerging or re-emerging of new viral infections. Only recently the human adenosinetriphosphatase/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3X) emerged as a novel therapeutic target in the fight against infectious diseases. Herein, starting from our previous studies, a new family of DDX3X inhibitors was designed, synthesized, validated on the target enzyme, and evaluated against the West Nile virus (WNV) infection. Time of addition experiments after virus infection indicated that the compounds exerted their antiviral activities after the entry process, likely at the protein translation step of WNV replication. Finally, the most interesting compounds were then analyzed for their in vitro pharmacokinetic parameters, revealing favorable absorption, distribution, metabolism, and excretion values. The good safety profile together with a good activity against WNV for which no treatments are currently available, make this new class of molecules a good starting point for further in vivo studies.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">West Nile virus (WNV) is a neurotropic arbovirus member of <i>Flaviviridae</i> family.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Since its introduction in Europe and America the number of novel infections is increasing, counting during 2017, 204 novel reported cases in Europe<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and 2002 in America.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The number is impressive considering that about 80% of WNV infections is usually asymptomatic and only the remaining 20% may cause meningitis, encephalitis, or rarely death in 1%.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In addition, probably due to climate warming, the number of novel human cases sharply increased in EU member states during the current year, reaching 710 infections and 63 deaths as of 30 August.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The rapid diffusion of WNV is dependent on its zoonotic life cycle, which involves migratory birds, Culex and Aedes mosquitos, and mammals including horses and humans. In this context, inadequate vector control, globalization, and human travel networks are only a few factors that are contributing to this and other arboviruses spreading. Despite continuous efforts being made to identify new drugs (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) or vaccines, no specific therapy or prophylaxis is actually available on the market and the treatment remains symptomatic.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In this context, a drug repurposing strategy provided new drug candidates to fight emerging and re-emerging viruses<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> but often the antiviral compounds are poorly active or are directed on viral enzymes and are thus generally characterized by a low genetic barrier to resistance (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Two-dimensional (2D) chemical structures of the known WNV inhibitors and their potential inhibitory target.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the last few years, the human adenosinetriphosphatase (ATPase)/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3X) protein has been identified as a component of critical host-cell pathways, hijacked by several pathogenic human viruses.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> DDX3X, an adenosine 5â²-triphosphate (ATP)-dependent RNA helicase, is involved in many aspects of RNA metabolism, such as transcription, translation, and RNA decay. It localizes to P-bodies, cytoplasmic foci related to mRNA turnover that are disrupted in response to the infection of viruses such as WNV or dengue virus (DENV).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Furthermore, after WNV infection, DDX3X was released from P-bodies and colocalized at a perinuclear region with viral NS3.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> It was found, moreover, that DDX3X knockdown strongly decreased WNV replication, suggesting an important implication not yet completely understood.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Although its role in the life cycle of different viruses was extensively studied,<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14 ref15">(10â15)</a> only few DDX3X inhibitors have been published as antiviral agents<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20">(16â20)</a> and the mechanism of action is clearly described only for HIV-1.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In a recent work,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> we identified the first inhibitor of the helicase binding site of DDX3X also active against WNV. Such a compound belonged to a family of DDX3X RNA-competitive molecules characterized by high selectivity and no cytotoxic effects in both in vitro and in vivo models. Among them, compound <b>16d</b> (compound <b>1</b> of the manuscript), belonging to the urea series, was the first DDX3X inhibitor endowed with a broad spectrum of antiviral activity against HIV-1 (both wild-type and drug-resistant strains), HCV, and new emerging viruses (WNV and DENV).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Despite <b>1</b> representing the proof of concept that DDX3X inhibitors could be used as novel antivirals drugs, its poor aqueous (aq) solubility limited its bioavailability in preclinical models, causing its accumulation in fat tissues (Logâ<i>S</i> = â7.05).</div><div class="NLM_p last">With the attempt to enlarge our library of DDX3X inhibitors, we performed a homology model-based virtual screening<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> that led us to identify another small-molecule inhibitor of DDX3X helicase activity (compound <b>2</b> of the present paper) characterized by a sulfonamide moiety. Despite its low passive permeability, compound <b>2</b> was characterized by a very good aqueous solubility and at the same time by a promising antienzymatic activity value of 0.36 Î¼M.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Pursuing our research of novel druglike DDX3X inhibitors, we performed a docking analysis of the two hit compounds <b>1</b> and <b>2</b>. On the basis of the three-dimensional (3D) overlap of the two molecules within the biding site, we designed a new series of hybrid compounds by merging the key structural moieties of <b>1</b> and <b>2</b> able to interact with the main residues involved in the binding of both of them (Arg276, Pro274, and Arg480). Novel compounds (<b>8</b>â<b>20</b> and <b>31</b>â<b>36</b>) were then synthesized and validated against DDX3X enzyme and evaluated for their ability to inhibit WNV replication, leading to a small library of derivatives endowed with antiviral activities comparable or higher than the broad spectrum antiviral compound 1-Î²-<span class="smallcaps smallerCapital">d</span>-ribofuranosyl-1,2,4-triazole-3-carboxamide (ribavirin). Moreover, the time of addition experiments highlighted that compounds likely act in the first phases of WNV replication. Finally, absorption, distribution, metabolism, and excretion (ADME) analysis of the most potent derivatives revealed 100 times improved aqueous solubility with respect to the hit <b>1</b>, thus allowing to overcome an important limitation of our compound.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08624" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08624" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Molecular Modeling</h3><div class="NLM_p">With the purpose of identifying novel small-molecule inhibitors of the DDX3X helicase binding site, we analyzed the docking poses of our hit compounds <b>1</b> and <b>2</b> within the DDX3X RNA binding pocket. The 3D structure of DDX3X in its RNA-bound closed conformation previously generated by homology modeling<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> was used to perform our studies. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, the docking analysis of compound <b>2</b> highlighted favorable hydrogen bond interactions between the nitrobenzene oxygen atom and Arg276 and Gly325, as well as between the sulfonamide group and Pro274 and Arg480 whereas the hydroxyl substituent is engaged in profitable H-bond interactions with the residues Pro274 and Arg503. Furthermore, hydrophobic interactions were found with Phe357, Val500, and Arg503. The predicted binding mode of compound <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) confirmed some key interactions, as seen for compound <b>2</b>; more in detail, the triazole ring interacted with the guanidine group of Arg276, whereas the urea NH groups and carbonyl moiety were involved in hydrogen bonds with Pro274 and Arg480, respectively. Moreover, the butyl terminus established beneficial interactions with Arg326 and Gly302. Finally, the tolyl ring established hydrophobic contacts with Arg503 and Pro274 side chains and the phenyl ring made hydrophobic interactions with Phe357.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Overlapped binding mode of compound <b>1</b> (pink) and <b>2</b> (blue marine) in DDX3X helicase pocket. Main interactions of both compounds with key residues of the active site are highlighted. Remarkably, Arg276, Arg480, and Gly302 make important contacts also with RNA phosphate backbone, whereas Gly325, Pro274, and Arg503 interact directly with the nucleobases. (B) Two-dimensional (2D) structure representation of the schematic idea behind the merged series design. (C) Proposed binding mode for merged derivative <b>13</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It is worthy to note that two compounds were both able to interact with Arg276 and Arg480 and also make profitable contacts with Gly325, Pro274, and Arg503, which our computational model<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> suggested to be important in RNA strand interactions into the helicase binding site (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Surface and cartoon representation of RNAâs main interactions within the helicase binding site of our homology model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Starting from this analysis, we merged the key structural units of <b>1</b><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and <b>2</b><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> to provide a new series of compounds able to interact at the same time with most of the residues involved in the binding of both of them, with particular attention to Arg276, Pro274, and Arg480, according to their suggested role in RNA strand binding. To this aim, as shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B, the <i>n</i>-alkyl substituted triazole group was selected from <b>1</b> structure whereas the phenyl-substituted sulfonamide moiety was chosen from the structure of compound <b>2</b>; the âhybridâ structure was finally composed by using the common aromatic ring (B) as a linker, and a small library of compounds were designed around this starting structure to explore the chemical space in relation to the activity against DDX3X.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The synthesis of compounds was accomplished following the synthetic route reported in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Aminothiophenol <b>3</b> was first converted into azide <b>4</b> by diazotization; a subsequent click reaction with the opportune terminal alkynes <b>5a</b>â<b>c</b> provides triazoles <b>6a</b>â<b>c</b>. Oxidative chlorination of thiols <b>6a</b>â<b>c</b> led to the corresponding sulfonyl chloride intermediates that were immediately converted into final compounds <b>8</b>â<b>20</b> by nucleophilic reaction with the opportune aromatic amines.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Sulfonamides <b>8</b>â<b>20</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (a) <i>t</i>-BuONO, acetonitrile (ACN), 20 min, 0 Â°C; (b) TMSN<sub>3</sub>, ACN, 2 h, room temperature (r.t.); (ii) CuSO<sub>4</sub>Â·5 H<sub>2</sub>O, sodium ascorbate, H<sub>2</sub>O/<i>t</i>-BuOH (1:1), microwave (MW), 10 min, 120Â°C; (iii) H<sub>2</sub>O<sub>2</sub>, SOCl<sub>2</sub>, ACN, from 0 Â°C to r.t., 15 min; (iv) opportune amine, pyridine (dry), 5 h, r.t.</p></p></figure><div class="NLM_p">Terminal alkynes <b>5a</b> and <b>5b</b> were, respectively, synthesized from propargyl alcohol and but-3-yn-1-ol by Haworth alkylation and purified by distillation (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). As shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, aniline <b>25</b> was synthesized by nucleophilic displacement of 1-fluoro-3-nitrobenzene <b>23</b> with morpholine in the presence of K<sub>2</sub>CO<sub>3</sub> and subsequent reduction catalyzed by Pd on charcoal.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Ethers <b>5a</b> and <b>5b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (a) NaOH(aq) (6 M), 20 min, r.t; (b) dimethylsulfate or diethylsulfate, 60 min, 50â55 Â°C.</p></p></figure><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Aniline <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (a) morpholine, K<sub>2</sub>CO<sub>3</sub>, dimethyl sulfoxide (DMSO) (dry), 3 h, 200 Â°C, (ii) H<sub>2</sub>, Pd/C, MeOH, 1 h.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, the synthetic route to the final compounds <b>31</b>â<b>34</b> first entailed the preparation of the common azide intermediate <b>30</b>, using synthetic procedures described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Derivatives <b>31</b>â<b>34</b> were then obtained by click reaction with the opportune terminal alkyne. Para-substituted compounds <b>35</b> and <b>36</b> were synthesized analogously.</div><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Sulfonamides <b>31</b>â<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Pyr, 5 h, r.t. (ii) H<sub>2</sub>, Pd/C, MeOH; (iii) (a) <i>t-</i>BuONO, CH<sub>3</sub>CN, 20 min, 0 Â°C; (b) TMSN<sub>3</sub>, CH<sub>3</sub>CN, 2 h, r.t.; (iv) opportune terminal alkyne, CuSO<sub>4</sub>Â·5H<sub>2</sub>O, sodium ascorbate, H<sub>2</sub>O/<i>t</i>-BuOH (1:1), MW, 10 min, 120 Â°C.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Biological Evaluation</h3><div class="NLM_p">Derivatives <b>8</b>â<b>20</b> and <b>31</b>â<b>36</b> were first evaluated for their ability to inhibit the helicase activity of DDX3X; results are reported in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Seven compounds showed promising inhibitory values higher than of 80% (compounds <b>8</b>â<b>10</b>, <b>14</b>, <b>15</b>, <b>19</b>, and <b>31</b>). The meta substitution of triazole on ring B was more a favorite with respect to the para position, <b>36</b> being about 26% less active than the corresponding meta derivative <b>8</b>. This is also confirmed by the docking studies, due to the higher shape complementarity of the binding site and <b>8</b> with respect to <b>36</b>. Three-dimensional conformations of the latter results into being much more constrained within the pocket, and this is likely responsible for the lower activity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. DDX3X Antienzymatic Activity<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0009.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data represent mean two values of the least two experiments.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">% inhibition at a fixed concentration of 20 Î¼M.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">na: not active. nd: not determined, compound precipitated from medium.</p></div></div><div></div></div><div class="NLM_p">Replacement of triazole butyl chain with isopentyl was well tolerated (compound <b>31</b>); when the length was reduced to two carbons, the activity decreased to about 25% in compound <b>33</b>, in line with a lower occupancy of the binding site and poorer complementarity between ligand and protein cavity, as also shown by the docking model. Yet, according to this observation, the introduction of bulkier substituents, such as cyclohexyl and phenyl, was detrimental for the activity, which decreased to 41 and 52% for compounds <b>34</b> and <b>32</b>, respectively.</div><div class="NLM_p">Alkyl chain of the triazole ring was also explored by checking the effect of introducing a polar atom. As a result, methyl-ethyl ether derivatives <b>10</b> and <b>15</b> retained their activities; in contrast, ethyl-methyl ether <b>11</b> was found 31% less active than the corresponding isomer <b>10</b>. According to our docking studies, the ether oxygen atom can establish a hydrogen bond interaction with Arg326. In fact, in the ethyl-methyl ether derivatives, the distance between the ether oxygen atom and its counterpart is less favorable to the hydrogen bond interaction. On the whole, pocket-hosting the substituted triazole moiety requires a precise spatial disposition (meta substitution preferred over the para one) while preferring more flexible, less constrained triazole substituents (butyl and isopentyl preferred over phenyl and cyclohexyl rings) occupying as much space as possible (lower activity in the presence of ethyl substituent). Polar atoms are tolerated, and a proper distance to efficiently interact with Arg326 appears to favor compound activity.</div><div class="NLM_p">Analogously, the pocket that houses the other terminal side of our compound series, i.e., ring A, also presents distinctive hallmarks. In particular, a proper combination of both polar and hydrophobic interactions appears to be beneficial to compoundsâ activity. Noteworthily, the presence of a hydrogen bond acceptor on ring A leads to a high percentage of DDX3X inhibition across this chemical series, hydrogen bond interactions being feasible with Pro274 and Arg503 of the binding pocket, as revealed by our docking studies. On confirmation of that, the introduction of an isoquinolyl-ring (compounds <b>14</b> and <b>15</b>) on ring A was well tolerated as well as the âOH and the âOCH<sub>3</sub> ortho substitution (compounds <b>8</b> and <b>19</b>), all bearing hydrogen-bonding moieties. Also, compounds <b>9</b> and <b>10</b>, bearing the âCF<sub>3</sub> moiety at the ortho position of ring A, satisfy such requirements, being the trifluoromethyl group reported to have both hydrophobic and polar (hydrogen-bonding) properties.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Substitution of the trifluoromethyl or hydroxyl groups of compounds <b>8</b>â<b>10</b> with a methyl group (compound <b>13</b>), which shows only hydrophobic properties, leads to a lower DDX3X % of inhibition in the sulfonamide series. Activity further decreases in the presence of fluorine or hydrogen substituents (compounds <b>12</b> and <b>20</b>), which have similar percent of inhibition values. Lower activity of compound <b>12</b> is likely due to the lower occupancy combined with a less effective hydrogen-bonding interaction, both exerted by the fluorine atom; the unsubstituted derivative <b>20</b> likewise showed a reduced activity value of 31%. The abolishment of the activity for compound <b>16</b>, where a morpholine moiety was introduced on ring A, is explained by docking studies to be due to the spatial constraints of the region of the binding site, being able to accommodate the planar isoquinoline system but unfavorable in hosting the morpholine ring.</div><div class="NLM_p">Also, the ortho position of the substituents on ring A results in being preferred although the para-methoxy substitution of derivative <b>18</b> is tolerated with a slight loss of activity. Unfortunately, the low solubility of <b>17</b> and <b>35</b> limited their enzymatic evaluation.</div><div class="NLM_p">The docking studies suggested that compounds could act as competitive inhibitors in the helicase binding site. To prove our hypothesis, we generated doseâresponse curves, titrating compound <b>8</b> in DDX3X helicase assays, in the presence of increasing fixed concentrations of the RNA substrate. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, the inhibitory potency (ID<sub>50</sub>) of the compound decreased (higher absolute values) as the RNA concentrations increased. This is consistent with the hypothesized competitive mechanism of inhibition, with respect to the RNA substrate.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Increasing concentrations of the inhibitor <b>8</b> were titrated in helicase assays, in the presence of 30 pmol of DDX3X and increasing fixed concentrations of the dsRNA substrate. Values are the mean of three independent measurements Â± standard deviation (S.D.). The residual enzymatic activity values were plotted as a function of the inhibitor concentrations, and the corresponding ID<sub>50</sub> values calculated for each dsRNA concentration are indicated on the right side of the panel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The compounds described here have been rationally designed to specifically target the helicase activity of DDX3X. To confirm the selectivity of our compounds, we tested the ability of <b>1</b>, <b>2</b>, <b>9</b>, and <b>14</b> to inhibit the ATPase activity of DDX3X and the helicase activity of the related human DEAD-box helicase DDX1. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, no inhibition was found with any compound concentration.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Enzymatic Data on Selected Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">cpd ID</th><th class="colsep0 rowsep0" align="center">ATPaseÂ DDX3,Â IC<sub>50</sub>Â (Î¼M)</th><th class="colsep0 rowsep0" align="center">DDX1,Â IC<sub>50</sub>Â (Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>200<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The value >200 indicates that less than 20% of inhibition was observed at 200 Î¼M, the highest concentration tested.</p></div></div></div><div class="NLM_p">Then, we quantified the cellular concentrations of the protein DDX3X, to confirm the presence of our target before carrying out anti-WNV and cytotoxicity assays. As reported in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>, African green monkey kidney epithelial (VERO) cells are endowed by a DDX3X concentration of 421 nM, higher than adenocarcinomic human alveolar basal epithelial cells (A549) but lower than human hepatocellular carcinoma cell line Huh7 (respectively valued 103 and 755 nM).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. DDX3X Cellular Expression in Huh7, A549, and VERO<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center" char="Â±">Huh7<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="Â±">VERO<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="Â±">A549<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DDX3X (nM)</td><td class="colsep0 rowsep0" align="char" char="Â±">755Â Â±Â 170</td><td class="colsep0 rowsep0" align="char" char="Â±">421Â Â±Â 140</td><td class="colsep0 rowsep0" align="char" char="Â±">103Â Â±Â 20</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Values represent mean Â± S.D. of three independent experiments. For details, see <a class="ref internalNav" href="#sec4" aria-label="Methods.">Methods.</a></p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Evaluated in Huh7: hepatocellular carcinoma cells.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Evaluated in VERO: African green monkey kidney cells.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Evaluated in A549: adenocarcinomic human alveolar basal epithelial cells.</p></div></div></div><div class="NLM_p">The three cell lines were infected with WNV at an multiplicity of infection (MOI) of 0.1, and the antiviral activity of compound <b>9</b> was assayed. As reported in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, its activities seemed proportional to the DDX3X concentrations. Probably due to the DDX3X implications in pulmonary cancer,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a><b>9</b> showed a cytotoxic activity of 21.3 Î¼M. For these reasons, to avoid interferences due to anticancer-related cytotoxic effects, the human Huh7 cells characterized by a higher DDX3X concentration of 755 nM were chosen to perform our studies.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Comparison of Antiviral Activities of <b>9</b> in Different Cell Lines Infected with WNV<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="2" align="center" char="Â±">Huh7<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char="Â±">VERO<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th><th class="rowsep1 colsep0" colspan="2" align="center" char="Â±">A549<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd ID</th><th class="colsep0 rowsep0" align="center" char="Â±">EC<sub>50</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center" char="Â±">EC<sub>50</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center" char="Â±">EC<sub>50</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a> (Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char="Â±">23.1Â Â±Â 0.4</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="char" char="Â±">12Â Â±Â 1.1</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="char" char="Â±">3.4Â Â±Â 0.8</td><td class="colsep0 rowsep0" align="left">21.3Â Â±Â 1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ribavirin</td><td class="colsep0 rowsep0" align="char" char="Â±">90Â Â±Â 2.5</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="char" char="Â±">95Â Â±Â 5.1</td><td class="colsep0 rowsep0" align="left">nt</td><td class="colsep0 rowsep0" align="char" char="Â±">46Â Â±Â 2.3</td><td class="colsep0 rowsep0" align="left">nt</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Data represent the mean of two values of the least two experiments.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">EC<sub>50</sub>: half-maximal effective concentration or the needed concentration to inhibit 50% viral-induced cell death.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">CC<sub>50</sub>: cytotoxic concentration 50 or the needed concentration to induce 50% death of noninfected cells.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Evaluated in Huh7: hepatocellular carcinoma cells.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Evaluated in VERO: African green monkey kidney cells.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">Evaluated in A549: adenocarcinomic human alveolar basal epithelial cells. Ribavirin was used as a reference compound. nt = not tested.</p></div></div></div><div class="NLM_p">Subsequently, Huh7 cells were infected with WNV and compounds endowed with the best IC<sub>50</sub>s against DDX3X were assayed for their antiviral properties. As reported in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, compounds showed interesting antiviral activities, ranging from 2.3 to 65.9 Î¼M, comparable or even higher than those of the broad spectrum antiviral ribavirin, which was used as a reference compound. Compound <b>8</b> showed the highest antiviral activity value of 2.3 Î¼M, followed by <b>15</b>, <b>9</b>, <b>13</b>, <b>14</b>, and <b>10</b>. All of these compounds had comparable antienzymatic profiles; the overall trend of their cellular activities could be ascribed to their different physicochemical parameters, which may influence the cellular penetration and consequently antiviral effects. Compound <b>15</b> resulted to be particularly promising, since it showed good antiviral activity together with favorable aqueous solubility, 100 times higher than that of hit compound <b>1</b>. Compound <b>2</b> showed the lowest activity, probably due to the limited cellular permeability (see <a class="ref internalNav" href="#sec5_2_4" aria-label="ADME Assay">ADME Assay</a> and <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). The mild antiviral activity of compound <b>13</b> is in line with its % inhibition of about 60% against DDX3X and was considered not worthy of further investigation. Finally, compound <b>12</b> endowed with the worst DDX3X inhibitory activity was found to be inactive in the cell-based assay.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Antiviral Activities and Cytotoxicity of Selected Compounds against WNV-Infected Huh7 Cells<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">WNV<a class="ref internalNav" href="#t5fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">Huh7</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cpd ID</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub><a class="ref internalNav" href="#t5fn2" aria-label="b">b</a> (Î¼M)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub><a class="ref internalNav" href="#t5fn3" aria-label="c">c</a> (Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">8.8Â Â±Â 0.2</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">120Â Â±Â 5.2</td><td class="colsep0 rowsep0" align="left">175Â Â±Â 11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">2.3Â Â±Â 0.5</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">23.1Â Â±Â 0.4</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">65.9Â Â±Â 2.3</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">39.9Â Â±Â 1.1</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">55Â Â±Â 1.1</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">13Â Â±Â 0.8</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ribavirin</td><td class="colsep0 rowsep0" align="left">95.5Â Â±Â 3.2</td><td class="colsep0 rowsep0" align="left">>200</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Data represent mean Â± standard deviation of three experiments.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">EC<sub>50</sub>: half-maximal effective concentration or needed concentration to inhibit 50% viral-induced cell death.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">CC<sub>50</sub>: cytotoxic concentration 50 or needed concentration to induce 50% death of noninfected cells.</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last">evaluated in Huh7 cells. nt = not tested. Ribavirin was used as a reference compound.</p></div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. In Vitro ADME Analysis of Selected Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="Ã" /></col><col align="char" char="â" /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">cpd ID</th><th class="colsep0 rowsep0" align="center" char="Ã"><i>P</i><sub>app</sub><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="â">Logââ<i>S</i><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="Â±">HLM stability<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>1</b></td><td class="colsep0 rowsep0" align="char" char="Ã">2.86Â ÃÂ 10<sup>â6</sup></td><td class="colsep0 rowsep0" align="char" char="â">â7.05</td><td class="colsep0 rowsep0" align="char" char="Â±">99.0Â Â±Â 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>2</b></td><td class="colsep0 rowsep0" align="char" char="Ã"><0.1Â ÃÂ 10<sup>â6</sup></td><td class="colsep0 rowsep0" align="char" char="â">â4.36</td><td class="colsep0 rowsep0" align="char" char="Â±">98.3Â Â±Â 1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>8</b></td><td class="colsep0 rowsep0" align="char" char="Ã">0.1Â ÃÂ 10<sup>â6</sup></td><td class="colsep0 rowsep0" align="char" char="â">â7.5</td><td class="colsep0 rowsep0" align="char" char="Â±">95.6Â Â±Â 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>9</b></td><td class="colsep0 rowsep0" align="char" char="Ã">0.21Â ÃÂ 10<sup>â6</sup></td><td class="colsep0 rowsep0" align="char" char="â">â7.5</td><td class="colsep0 rowsep0" align="char" char="Â±">99.0Â Â±Â 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>10</b></td><td class="colsep0 rowsep0" align="char" char="Ã">0.71Â ÃÂ 10<sup>â6</sup></td><td class="colsep0 rowsep0" align="char" char="â">â4.6</td><td class="colsep0 rowsep0" align="char" char="Â±">97.1Â Â±Â 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>14</b></td><td class="colsep0 rowsep0" align="char" char="Ã">0.68Â ÃÂ 10<sup>â6</sup></td><td class="colsep0 rowsep0" align="char" char="â">â5.4</td><td class="colsep0 rowsep0" align="char" char="Â±">98.0Â Â±Â 0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char="."><b>15</b></td><td class="colsep0 rowsep0" align="char" char="Ã">0.44Â ÃÂ 10<sup>â6</sup></td><td class="colsep0 rowsep0" align="char" char="â">â4.7</td><td class="colsep0 rowsep0" align="char" char="Â±">96.0Â Â±Â 0.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Apparent permeability reported in cm s<sup>â1</sup>.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Aqueous solubility expressed as Log of molar concentration.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Human liver microsomal (HLM) metabolic stability.</p></div></div></div><div class="NLM_p">To better evaluate the antiviral activities of our compounds, we selected <b>9</b> as a representative of the series, to investigate the mechanism of action (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Time of addition studies, performed at compound inhibitory concentration near and under the IC<sub>50</sub> values, clearly demonstrated that <b>9</b> acted in the first 2 h of the WNV life cycle, since the addition of <b>9</b> after 3 h post infection did not affect viral replication (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). In particular, although an antiviral effect was visible when added before viral infection and cell entry occurred, the substantial activity of compound <b>9</b> after 2 h of infection also could indicate that its main target activity was in subsequent steps of the viral replication. Consequently, we evaluated the effect of <b>9</b> in the production of both genomic and complementary viral RNA. Importantly, our inhibitor was able to interfere with the concentration of genomic RNA at an early time of the infection (from 2 to 12 h post infection) only when added 1 h post infection and caused no significant changes when added 3 h post infection. Moreover, although the complementary RNA was not revealed up to 12 h post infection, 24 h post infection the quantity of complementary RNA was slightly lower after the addition of compound <b>9</b> 1 h post infection. Moreover, our inhibitor was able to interfere marginally with the concentration of genomic RNA, without significant changes in the quantity of the complementary noninfective RNA. Finally, we evaluated its effect on the production of WNV nucleocapsid (NC) protein. As highlighted in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C, the addition of <b>9</b> 1 h post infection significantly reduced the concentration of NC whereas according to the results shown in panel C, its addition after 3 h did not change NC concentration, which was comparable to the control. Overall, these results suggested a possible implication of <b>9</b> in the first phases of viral replication, probably during protein translation, after the entry process.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of varying anti-DDX3X compound <b>9</b> at the time of administration on WNV replication. (A) WNV infection of Huh7 cells at MOI of 0.1 in presence of compound <b>9</b> at 1 and 10 Î¼M added at the indicated time (h) relative to the infection and assayed with the viral plaque reduction assay. Data represent mean Â± standard deviation. (B) WNV genomic and complementary viral RNA in Huh7 cells assayed after infection in absence or in presence of compound <b>9</b> at 1 Î¼M added at 1 and 3 h of the infection. Data represent mean Â± standard deviation. (C) WNV capsid protein electrophoretic mobility obtained from Huh7 cells after infection in absence or in presence of compound <b>9</b> at 1 Î¼M added at 1 and 3 h of the infection. No compound <b>9</b> addition performed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> In Vitro ADME Analysis</h3><div class="NLM_p last">Compounds <b>2</b>, <b>8</b>, <b>9</b>, <b>10</b>, <b>14</b>, and <b>15</b> were in vitro profiled for aqueous solubility, liver microsomal stability, and membrane permeability. As reported in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>, compounds were characterized by improved aqueous solubility values, in particular, compounds <b>14</b>, <b>15</b>, and <b>10</b>, even if less active against WNV than compound <b>1</b> and about 100 times more soluble than hit compound <b>1</b> with Logâ<i>S</i> comparable to that of <b>2</b>. Furthermore, passive membrane permeability assay (PAMPA) indicated nonlimiting values with the exception of hit compound <b>2</b>, that showed a minor value of 0.1 Ã 10<sup>â6</sup> cm s<sup>â1</sup> that is probably the cause of its inactivity in antiviral assay. Finally, stability tests in human liver microsomes (HLM) disclosed that all selected compounds showed a high metabolic stability major of 95.6%.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17671" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17671" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We identified a completely new family of selective inhibitors of the helicase activity of DDX3X. The compounds described here have been rationally designed to specifically target the helicase activity of DDX3X. Accordingly, they were inactive in the ATPase assay and in the helicase assay against the related human DEAD-box helicase DDX1. Future plans for improving the assessment of selectivity of our compounds include their evaluation on a broader panel of DDX-family helicases, as well as the exploitation of Huh7 cells with inducible silencing of DDX3X expression. Both approaches are currently under development in our laboratory and will be the subject of future communications. Among the 21 compounds reported, six of them (namely, compounds <b>8</b>, <b>9</b>, <b>10</b>, <b>13</b>, <b>14</b>, and <b>15</b>) are characterized by promising antiviral activities against WNV, lower than those of the broad spectrum antiviral ribavirin, and more importantly without signs of cellular toxicities. It is worth pointing out that the antiviral activities are DDX3X-dependant, since compound <b>12</b>, endowed with the worst inhibitory activity, did not show any antiviral effect up to 200 Î¼M. Furthermore, merged derivatives <b>10</b>, <b>14</b>, and <b>15</b>, even if less active against WNV than compound <b>1</b>, are endowed with very good aqueous solubility, with values 100 times higher than those of the hit <b>1</b>, thus overcoming an important limitation of the previously published compounds. Furthermore, the time of addition experiments is consistent with literature studies previously published on DDX3X recruitment on WNV replication site,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> highlighting an involvement of our compounds in the first phases of WNV replication and suggesting their possible role in the step of protein translation, after the entry process. Noteworthily, although tolerability and bioavailability need to be proved in vivo, the effectiveness of the compounds also when added before viral cell infection suggests their possible use as prophylactic antiviral drugs. Even if we are aware that deep investigations are needed to elucidate the mechanism of action of our compounds, to date, the exact implications of our own target on the WNV life cycle remain poorly understood.</div><div class="NLM_p last">In the absence of any available antiviral compound for the treatment of WNV infection,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> the present study confirms our previous findings, suggesting that DDX3X helicase inhibitors can represent the Achillesâ hell of viruses and that human proteins can be manipulated to fight novel emerging viral threats.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81456" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81456" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Docking Studies</h3><div class="NLM_p last">All compounds studied herein were docked within the RNA binding site of the modeled human DDX3X closed conformation previously published<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> using the software package GOLD 4.1.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> The pocket under investigation was inserted into a grid box centered on residue Phe357 and enclosing residues lying within 10 Ã from such amino acid. The genetic algorithm parameter settings were employed using the search efficiency set at 100%, and 100 runs were carried out for each ligand. Chemscore was chosen as the fitness function. Finally, results differing by less than 1.5 Ã in ligand-all atom root-mean-square deviation were clustered together. For each inhibitor, the first ranked solution was selected for further analysis. Pictures of the modeled ligandâenzyme complexes together with graphic manipulations were rendered using PyMOL molecular graphic system.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Chemistry</h3><div class="NLM_p">Reagents were obtained from commercial suppliers (for example, Sigma-Aldrich, Alfa Aesar). All commercially available chemicals were used as purchased without further purification. CH<sub>2</sub>Cl<sub>2</sub> and MeOH were dried prior to use by distilling from calcium hydride or magnesium methoxide. Anhydrous reactions were run under positive pressure of dry N<sub>2</sub> or argon. Thin-layer chromatography (TLC) was carried out using Merck TLC plate silica gel 60 F254. Chromatographic purifications were performed on columns packed with Merck 60 silica gel, 23â400 mesh, for the flash technique.</div><div class="NLM_p">All NMR spectra were recorded on a Bruker Avance DPX400 spectrometer at 400 MHz for <sup>1</sup>H NMR or 100 MHz for <sup>13</sup>C NMR. Chemical shifts are reported relative to tetramethylsilane at 0.00 ppm. <sup>1</sup>H patterns are described using the following abbreviations: s = singlet, d = doublet, t = triplet, q = quartet, quint = quintet, sx = sextet, sept = septet, m = multiplet, br = broad signal, and br s = broad singlet.</div><div class="NLM_p last">Mass spectra (MS) data were obtained using an Agilent 1100 LC/mass spectra detection (MSD) VL system (G1946C) with a 0.4 mL min<sup>â1</sup> flow rate using a binary solvent system 25 of 95:5 methyl alcohol/water ratio. UV detection was monitored at 254 nm. Mass spectra were acquired in positive and negative mode scanning over the mass range.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Microwave Irradiation Experiments</h3><div class="NLM_p last">These were conducted using CEM Discover Synthesis Unit (CEM Corp., Matthews, NC). The machine consists of a continuous focused microwave power delivery system with operator selectable power output from 0 to 300 W. The temperature of the content vessels was monitored using a calibrated infrared temperature control mounted under the reaction vessel. All experiments were performed using a stirring option whereby the contents of the vessels are stirred by means of a rotating magnetic plate located below the floor of the microwave cavity and a Teflon-coated magnetic stir bar in the vessel.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> UV/Layer Chromatography (LC)-MS Method</h3><div class="NLM_p last">For the quantitative analysis, a UV/LC-MS system was used. LC analysis was performed by Agilent 1100 LC/MSD VL system (G1946C) (Agilent Technologies, Palo Alto, CA) constituted by a vacuum solvent degassing unit, a binary high-pressure gradient pump, and a 1100 series UV detector, and a 1100 MSD model VL benchtop mass spectrometer was used. The Agilent 1100 series mass spectra detection (MSD) single-quadrupole instrument was equipped with the orthogonal spray atmospheric pressure ionization electrospray (API-ES, Agilent Technologies, Palo Alto, CA). Nitrogen was used as nebulizing and drying gas. The pressure of the nebulizing gas, the flow of the drying gas, the capillary voltage, the fragmentor voltage, and the vaporization temperature were set at 40 psi, 9 L min<sup>â1</sup>, 3000 V, 70 V, and 350 Â°C, respectively. UV detection was monitored at 280 nm. The LC-electrospray ionization (ESI)-MS determination was performed by operating the MSD in the positive-ion mode. Spectra were acquired over the scan range <i>m</i>/<i>z</i> 50â1500 using a step size of 0.1 u. Chromatographic analysis was performed using a Varian Polaris 5 C18-A column (150 Ã 4.6 mm<sup>2</sup>, 5 Î¼m particle size) at room temperature. Analysis was carried out using gradient elution of a binary solution; eluent A was ACN, whereas eluent B consisted of water. The analysis started at 0% A for 3 min, then rapidly increased up to 98% in 12 min, finally remaining at 98% A until 18 min. The analysis was performed at a flow rate of 0.8 mL min<sup>â1</sup>, and the injection volume was 20 Î¼L. LC retention times, molecular ion (<i>m</i>/<i>z</i>), and LC purity (by UV) were based on the method below. Purity of compounds (as measured by the peak area ratio) was >97%.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 3-Azidobenzenethiol (<b>4</b>)</h3><div class="NLM_p last">3-Aminothiol (2.0 g, 15.97 mmol) was dissolved in CH<sub>3</sub>CN and cooled to 0 Â°C in an iceâsalt bath. To this stirred solution was added <i>t</i>-BuONO (2.84 mL, 23.9 mmol), and the mixture was stirred for 10 min; after this time, TMSN<sub>3</sub> (2.53 mL, 19.17 mmol) was added dropwise, during 10 min, and the resulting brown solution was stirred at r.t. One hour later the solvent was removed at reduced pressure and the residue was purified by flash chromatography on silica gel. Purification eluent: EP/EA 8:2, yield 80%. Pale white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 7.3â7.2 (m, 2H), 7.1 (s, 1H) 6.9 (d, <i>J</i> = 6.4 Hz, 1H) ppm.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Procedure for the Preparation of Compounds <b>5a</b> and <b>5b</b></h3><div class="NLM_p">To a stirring solution of 200 g of NaOH in 300 mL of water (0.3 mol, 16.8 g) was added the opportune alcohol (2.5 mL, 33.02 mmol). To this was slowly added the corresponding sulfate (15 mmol, 2082 mg) in 2 h dropwise, and the mixture was heated at 50 Â°C. The final product was distilled off; the distillation was stopped at 95 Â°C; then, the content of the receiver was washed with cold NH<sub>4</sub>Cl(aq) solution and separated.</div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 3-Ethoxyprop-1-yne (<b>5a</b>)</h4><div class="NLM_p last">Yield 68% colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 4.13â4.13 (d, <i>J</i> = 2.4 Hz, 2H), 3.60â3.55 (q, <i>J</i> = 6.9 Hz, 2H), 2.41â2.40 (t, <i>J</i> = 4.8 Hz, 1H), 1.24â1.22 (t, <i>J</i> =6.9 Hz, 3H) ppm.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 4-Methoxybut-1-yne (<b>5b</b>)</h4><div class="NLM_p last">Yield 52% colorless oil. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 3.52â3.49 (m, 2H), 3.33 (s, 3H), 2.47â2.43 (m, 2H), 1.99â1.97 (m, 1H) ppm.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure for the Preparation of Triazoles (<b>6a</b>â<b>c</b>)</h3><div class="NLM_p">The appropriate alkyne (6.08 mmol) and the opportune azide (5.07 mmol) were suspended in a 1:1 mixture of water and <i>t-</i>BuOH (1.5 mL each) in a 10 mL glass vial equipped with a small magnetic stirring bar. To this was added sodium ascorbate (2.5 mmol) and copper(II) sulfate pentahydrate (2.50 mmol). The mixture was then heated for 10 min at 80 Â°C under microwave irradiation, using an irradiation power of 300 W. Thereafter, the solvent was removed at reduced pressure, water was added, and the mixture was extracted with EtOAc (3 Ã 20 mL). The organic layers were collected, washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. The crude was purified by flash chromatography on silica gel using the opportune eluent to give the desired triazole compounds.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 3-(4-(Ethoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzenethiol (<b>6a</b>)</h4><div class="NLM_p last">Purification eluent: EP/EA 7:3, white solid, yield 62%. <sup>1</sup>H NMR (400 MHz, MeOD): Î´ 7.97 (s, 1H), 7.94 (s, 1H), 7.60â7.57 (m, 2H), 7.49 (t, <i>J</i> = 8.0 Hz, 1H), 4.70 (s, 2H), 3.66 (q, <i>J</i> = 6.8 Hz, 2H), 0.89 (t, <i>J</i> = 6.8 Hz, 3H) ppm.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 3-(4-(2-Methoxyethyl)-1<i>H</i>-1,2,3-triazol-1-yl)benzenethiol (<b>6b</b>)</h3><div class="NLM_p">Purification eluent: EP/EA 7:3, white solid, yield 50%. <sup>1</sup>H NMR (400 MHz, MeOD): Î´ 8.02 (s, 1H), 7.45â7.29 (m, 2H), 7.28 (d, <i>J</i> = 7.4 Hz, 1H), 7.18 (t, <i>J</i> = 7.5, 1H), 3.75 (t, <i>J</i> = 6.6 Hz, 2H), 3.38 (s, 3H), 2.97 (t, <i>J</i> = 6.6 Hz, 2H).</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)benzenethiol (<b>6c</b>)</h4><div class="NLM_p last">Purification eluent: EP/EA 8:2, pale white solid, yield 78%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 7.95 (s, 1H), 7.72 (s, 1H), 7.6â7.53 (m, 2H), 7.47 (t, <i>J</i> = 8.0 Hz, 1H), 2.81 (t, <i>J</i> = 7.6 Hz, 2H), 1.75â1.67 (m, 2H), 1.47â1.39 (m, 2H), 0.97 (t, <i>J</i> = 7.6 Hz, 3H) ppm.</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> General Procedure for the Synthesis of Sulfonamides <b>8</b>â<b>20</b></h3><div class="NLM_p">The opportune benzenethiol (<b>6a</b>â<b>c</b>) (0.43 mmol) was dissolved in 2 mL of acetonitrile; to this, 30% H<sub>2</sub>O<sub>2</sub> (1.2 mmol, 133 Î¼L) and SOCl<sub>2</sub> (0.43 mmol, 31 Î¼L) were added and the corresponding solution was stirred at room temperature for 30 min. After this time, the reaction mixture was extracted with EtOAc (3 Ã 25 mL), washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed at reduced pressure, and the resulting residue was dissolved in 5 mL of dichloromethane (DCM). To this, a solution of the opportune aromatic amine in 1 mL of pyridine was added drop by drop. The resulting mixture was stirred at r.t. for 4 h. Thereafter, water was added and the mixture was extracted with DCM (6 Ã 30 mL), washed with brine, and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The corresponding residue was purified on silica with the opportune eluent.</div><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-hydroxyphenyl)benzenesulfonamide (<b>8</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1, yield 80%. <sup>1</sup>H NMR (400 MHz, acetone): Î´ 8.79â8.35 (m, 2H), 8.34â8.31 (m, 2H), 8.11â8.09 (d, <i>J</i> = 8.0 Hz, 1H), 7.81â7.79 (d, <i>J</i> = 8.0 Hz, 1H), 7.71â7.67 (t, <i>J</i> = 8.0 Hz, 1H), 7.37â7.35 (d, <i>J</i> = 8.0 Hz, 1H), 6.99â6.95, (t, <i>J</i> = 8.0 Hz, 1H), 6.81â6.77 (m, 2H), 2.84 (t, <i>J</i> = 7.6 Hz, 2H), 1.77â1.69 (m, 2H), 1.46â1.38 (m, 2H), 0.99 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (100 MHz, acetone): Î´ 149.88, 142.46, 131.77, 137.68, 130.44, 126.66, 126.50, 124.40, 124.19, 123.58, 119.90, 119.47, 118.32, 115.59, 31.30, 24.94, 21.95, 13.19. High-resolution mass spectrometry (HRMS) (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S [M â H]<sup>â</sup> 371.1178, found 371.1180.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>9</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1, yield 90%. <sup>1</sup>H NMR (acetone): Î´ 8.78 (s, 1H), 8.41â8.37 (m, 2H), 8.18â8.16 (d, <i>J</i> = 8.2 Hz, 1H), 7.89â7.87 (d, <i>J</i> = 7.6 Hz, 1H), 7.82â7.78 (t, <i>J</i> = 7.8 Hz, 1H), 7.69â7.61 (m 2H), 7.54â7.52 (d, <i>J</i> = 8 Hz, 1H), 7.45â7.44 (t, <i>J</i> = 7.4 Hz, 1H), 2.77â2.73 (t, <i>J</i> = 7.4 Hz, 2H), 1.71â1.67 (m, 2H), 1.45â1.35 (m, 2H), 0.94â0.89 (t, <i>J</i> = 8.0 Hz, 3H) ppm. <sup>13</sup>C NMR (acetone): 149.1, 142.66, 137.89, 134.15, 133.28, 130.93 (2C), 127.40 (q, <i>J</i><sub>(CâF)</sub> = 5 Hz), 126.90, 126.28, 124.80, 123.85, 122.29 (q, <i>J</i><sub>(CâF)</sub> = 273 Hz), 119.54, 118.03, 31.22, 24.95, 21.95, 13.18 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 423.1103, found 423.1107.</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 3-(4-(Ethoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl) benzenesulfonamide (<b>10</b>)</h4><div class="NLM_p last">Purification eluent: PE/EtOAc 7:2, yield 85%, white solid <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 8.09 (1H, s), 7.99â7.97 (m, 2H), 7.79â7.75 (m, 2H), 7.60â7.47 (m, 3H), 7.26â7.22 (t, <i>J</i> = 7.2 Hz, 1H), 7.14 (s, 1H), 4.66 (s, 2H), 3.63â3.58 (q, <i>J</i> = 6.8 Hz, 2H), 1.24â1.20 (t, <i>J</i> = 6.8 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub><i>-d</i>): 146.95, 144.13, 140.78, 137.44, 133.39, 130.80, 126.99, 126.07 (2C), 125.84, 124.87, 124.69, 122.09, 120.58 (q, <i>J</i><sub>(CâF)</sub> = 280 Hz), 118.71, 66.39, 63.99, 15.07 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub>S [M â H]<sup>â</sup> 425.0895, found 425.0890.</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 3-(4-(2-Methoxyethyl)-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>11</b>)</h4><div class="NLM_p last">Purification eluent: PE/EtOAc 7:2, yield 63%, white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub><i>-d</i>): Î´ 8.07 (1H, s), 8.02â8.00 (d, <i>J</i> = 8.0 Hz, 1H), 7.86â7.84 (d, <i>J</i> = 7.9 Hz, 2H), 7.77â7.75 (d, <i>J</i> = 7.6 Hz, 1H), 7.61â7.50 (m, 3H), 7.28â7.24 (m, 2H), 3.72 (m, 2H), 3.394 (s, 3H), 3.08â3.07 (m, 2H) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>): Î´ 145.32, 139.71, 133.11, 132.91, 132.43, 129.12, 128.81, 127.36, 126.04 (2C), 124.09, 119.21 (q, <i>J</i><sub>(CâF)</sub> = 268 Hz), 119.1, 116.66, 115.51, 77.81, 59.44, 30.91 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub>S [M â H]<sup>â</sup> 425.0895, found 425.0893.</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-fluorophenyl)benzenesulfonamide (<b>12</b>)</h4><div class="NLM_p last">Purification eluent: PE/EtOAc 7:2, yield 71%, white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub><i>-d</i>): 8.07 (s, 1H), 8.02â8.01 (d, <i>J</i> = 8.0 Hz, 1H), 7.77â7.76 (d, <i>J</i> = 8.0 Hz, 1H), 7.68 (s, 1H), 7.62â7.57 (m, 2H), 7.15â7.09 (m, 2H), 6.99â6.94 (m, 2H), 2.80â2.77 (t, <i>J</i> = 7.6 Hz, 2H), 1.74â1.66 (m, 2H), 1.46â1.37 (m, 2H), 0.97â0.93 (t, <i>J</i> = 7.6 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): 157.9 (d, <i>J</i><sub>(CâF)</sub> = 238 Hz), 149.77, 140.75, 137.65, 130.69, 127.13, 126.59, 124.96, 124.70, 124.59, 124.38, 118.44 (d, <i>J</i><sub>(CâF)</sub> = 20 Hz), 115.79, 115.58 (d, <i>J</i><sub>(CâF)</sub> = 9 Hz), 31.32, 25.23, 22.26, 13.78 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>FN<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 373.1134, found 373.1129.</div></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(<i>o</i>-tolyl)benzenesulfonamide (<b>13</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1, yield 90%. <sup>1</sup>H NMR (400 MHz, acetone): Î´ 8.56 (s, 1H), 8.35.8.33 (d, <i>J</i> = 8.0 Hz, 1H), 8.25â8.23 (d, <i>J</i> = 8.0 Hz, 1H), 8.14â8.11 (t, <i>J</i> = 7.2 Hz, 1H), 7.74â7.72 (m, 2H), 7.17â7.10 (m, 4H), 2.77â2.73 (t, <i>J</i> = 7.4 Hz, 2H), 2.72 (s, 3H), 1.73â1.65 (m, 2H), 1.43â1.37 (m, 2H), 0.95â0.91 (t, <i>J</i> = 7.8 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, acetone): 149.06, 142.56, 137.76, 134.78, 134.16, 130.91, 130.72, 126.78, 126.51, 126.37, 126.28, 123.51, 119.46, 118.11 31.22, 24.95, 21.95, 17.18, 13.18 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 369.1385, found 369.1387.</div></div><div id="sec4_9_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(isoquinolin-6-yl)benzenesulfonamide (<b>14</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1, yield 77%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 9.23 (s, 1H), 8.42 (s, 1H), 8.26 (s, 1H), 7.89â7.83 (m, 4H), 7.66â7.62 (m, 3H), 7.55â7.46 (m, 2H), 2.74â2.71 (t, <i>J</i> = 7.8 Hz, 2H), 1.66â1.59 (m, 2H), 1.39â1.25 (m, 2H), 0.90â0.86 (t, <i>J</i> = 8.0 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>-<i>d</i>): 152.43, 149.84, 142.93, 141.34, 137.68, 132.70, 131.10, 130.51, 129.38, 129.01, 128.51, 127.62, 127.38, 126.85, 124.13, 118.81, 115.75, 31.36, 25.25, 22.26, 13.62 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>22</sub>N<sub>5</sub>O<sub>2</sub>S [M + H]<sup>+</sup> 408.1494, found 408.1489.</div></div><div id="sec4_9_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 3-(4-(Ethoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(isoquinolin-5-yl)benzenesulfonamide (<b>15</b>)</h4><div class="NLM_p last">Purification eluent: CHCl<sub>3</sub>/acetone 98:2, white solid, yield 73%. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): Î´ 9.50 (s, 1H), 8.51 (s, 1H), 8.45â8.43 (m, 2H), 8.23â8.18 (m, 3H), 8.07â8.05 (d, <i>J</i> = 8.0 Hz, 1H), 7.80â7.66 (m, 4H), 4.63 (s, 2H), 3.61â3.56 (q, <i>J</i> = 7.2 Hz, 2H), 1.32 (t, <i>J</i> = 7.2, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>-<i>d</i>) Î´ 192.82, 174.69, 149.87, 146.32, 141.42, 137.79, 136.51, 134.59, 132.21, 131.44, 130.80, 129.22, 126.87, 124.19, 123.06, 121.88, 118.89, 65.82, 62.91, 14.31 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>5</sub>O<sub>3</sub>S [M + H]<sup>+</sup> 410.1287, found 410.1282.</div></div><div id="sec4_9_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 3-(4-(Ethoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(3-morpholinophenyl)benzenesulfonamide (<b>16</b>)</h4><div class="NLM_p last">Purification eluent: CHCl<sub>3</sub>/acetone 98:2, white solid, yield 64%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.17 (s, 1H), 7.99 (s, 1H), 7.92 (d, <i>J</i> = 7.4 Hz, 1H), 7.79 (d, <i>J</i> = 7.6 Hz, 1H), 7.72 (s, 1H), 7.59 (t, <i>J</i> = 8.0 Hz, 1H), 7.12 (t, <i>J</i> = 8.4 Hz, 1H), 6.78 (s, 1H), 6.68 (d, <i>J</i> = 8.4 Hz, 1H), 6.61 (d, <i>J</i> = 7.6 Hz, 1H), 4.71 (s, 2H), 3.81 (t, <i>J</i> = 4.4 Hz, 4H), 3.64 (q, <i>J</i> = 6.8 Hz, 2H), 3.1 (t, <i>J</i> = 4.4 Hz, 4H), 1.27 (t, <i>J</i> = 6.8 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>-<i>d</i>) Î´ 152.8, 147.23, 141.24, 137.34, 137.13, 130.56, 129.88, 127.23, 124.35, 120.75, 119.00, 113.20, 112.88, 109.30, 66.56, 63.92 (2C), 48.91 (2C), 29.70, 15.15 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>26</sub>N<sub>5</sub>O<sub>4</sub>S [M + H]<sup>+</sup> 444.1706, found 444.1701.</div></div><div id="sec4_9_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 3-(4-(Ethoxymethyl)-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(4-isopropylphenyl)benzenesulfonamide (<b>17</b>)</h4><div class="NLM_p last">Purification eluent: PE/EA 7:3, white solid, yield 71%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.07 (s, 1H), 7.98 (t, <i>J</i> = 8.4 Hz, 1H), 7.79 (d, <i>J</i> = 8.0 Hz, 1H), 7.61, (t, <i>J</i> = 8.0 Hz, 1H), 7.55 (s, 1H) 7.12 (d, <i>J</i> = 8.0 Hz, 2H), 7.25 (s, 1H), 7.05 (d, <i>J</i> = 8.4 Hz, 2H), 4.69 (s, 2H), 3.66â3.60 (m, 2H), 2.87â2.78 (m, 1H), 1.27 (t, <i>J</i> = 7.2 Hz, 3H), 1.19 (d, <i>J</i> = 6.8 Hz, 6H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>-<i>d</i>) Î´ 147.08, 141.22, 138.37, 137.35, 134.97, 133.40, 130.61, 129.60, 127.44, 127.14, 124.49, 122.99, 120.61, 118.86, 66.44, 64.06, 33.31, 24.10 (2C), 15.28 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>N<sub>4</sub>O<sub>3</sub>S [M â H]<sup>â</sup> 399.1491, found 399.1488.</div></div><div id="sec4_9_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(4-methoxyphenyl)benzenesulfonamide (<b>18</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1, yield 79%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 8.10 (s, 1H), 7.95â7.93 (d, <i>J</i> = 7.6 Hz, 1H), 7.71 (s, 1H), 7.68â7.66 (d, <i>J</i> = 7.7 Hz, 1H), 7.61 (s, 1H), 7.56â7.52 (t, <i>J</i> = 7.9 Hz, 1H), 7.07â7.04 (d, <i>J</i> = 8.2 Hz, 2H), 6.77â6.75 (d, <i>J</i> = 8.2 Hz, 2H), 3.73 (s, 3H), 2.78â2.75 (t, <i>J</i> = 7.7 Hz, 2H), 1.71â1.64 (m, 2H), 1.43â1.36 (m, 2H), 0.95â0.91 (t <i>J</i> = 7.6 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, MeOD-<i>d</i><sub>4</sub>): Î´ 158.45, 149.70, 141.20, 137.55, 130.75, 127.59, 125.89 (2C), 124.43, 119.27, 118.89, 118.68, 114.46 (2C), 55.90, 31.84, 25.52, 21.87, 13.85 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub>S [M â H]<sup>â</sup> 385.1334, found 385.1338.</div></div><div id="sec4_9_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-methoxyphenyl)benzenesulfonamide (<b>19</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 4:1, yield 64%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 8.28 (s, 1H), 8.08 (s, 1H), 7.97 (s, 1H), 7.62â7.60 (d, <i>J</i> = 8.0 Hz, 2H), 7.28â7.25 (t, <i>J</i> = 8.0 Hz, 1H), 6.68â6.65 (m, 4H), 3.80 (s, 3H), 2.78â2.73 (t, <i>J</i> = 8.0 Hz, 2H), 1.68â1.64 (m, 2H), 1.44â1.39 (m, 2H), 1.02â0.97 (m, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) Î´ 149.27, 149.11, 139.90, 139.74, 128.46, 127.64, 126.49, 124.93, 123.87, 123.11, 119.85, 119.72, 119.16, 114.93, 74.56, 31.00, 24.41, 21.46, 12.91 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>N<sub>4</sub>O<sub>3</sub>S [M â H]<sup>â</sup> 385.1334, found 385.1329.</div></div><div id="sec4_9_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 3-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-phenylbenzenesulfonamide (<b>20</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1, yield 89%. <sup>1</sup>H NMR (400 MHz, acetone): Î´ 8.51 (s, 1H), 8.08 (s, 1H), 7.88â7.74 (m, 2H), 7.69 (m, 1H), 7.45 (t, <i>J</i> = 7.9 Hz, 1H), 7.11 (t, <i>J</i> = 7.9 Hz, 2H), 6.90â6.79 (m, 1H), 6.60 (d, <i>J</i> = 7.5 Hz, 2H), 2.76 (t, <i>J</i> = 7.8 Hz, 2H), 1.70 (t, <i>J</i> = 7.9 Hz, 2H), 1.53â1.37 (m, 2H), 0.98 (t, <i>J</i> = 6.6 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, acetone): 153.56, 141.73, 137.53, 137.00, 129.89, 129.42 (2C), 128.59, 126.40, 125.09, 123.98, 121.69 (2C), 119.95, 29.99, 27.98, 22.18, 14.05 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>19</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 355.1229, found 355.1224.</div></div><div id="sec4_9_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 4-(3-Nitrophenyl)morpholine (<b>23</b>)</h4><div class="NLM_p last">To a solution of 1-fluoro-3-nitrobenzene (1.0 g, 7.1 mmol) in DMSO, K<sub>2</sub>CO<sub>3</sub> (2.930 g, 21.24 mmol) and morpholine (741 mg, 8.5 mmol) were added. The solution was stirred at 60 Â°C for 3 h. Thereafter, 5% LiCl(aq) was added and the reaction mixture was extracted with EtOAc (3 Ã 25 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed at reduced pressure and the residue purified by flash chromatography on silica gel. Purification eluent: PE/EA 8:2, yield 80%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) Î´ 7.72 (d, <i>J</i> = 8.0 Hz, 1H), 7.43 (t, <i>J</i> = 8.0 Hz, 1H), 7.26 (s, 1H), 7.22 (d, <i>J</i> = 6.4 Hz, 1H) 3.90 (t, <i>J</i> = 4.8 Hz, 4H) 3.27 (t, <i>J</i> = 5.2 Hz, 4H) ppm.</div></div><div id="sec4_9_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 3-Morpholinoaniline (<b>24</b>)</h4><div class="NLM_p"><b>23</b> (500 mg, 2.5 mmol) was solubilized in 30 mL of anhydrous MeOH; to this, palladium on charcoal (50 mg) was added. The reaction mixture was stirred under hydrogen atmosphere for 1 h; then, the mixture was filtered off on a celite pad, the solvent evaporated at reduced pressure, and the residue purified by flash chromatography on silica gel.</div><div class="NLM_p last">Purification eluent: PE/EA = 8:2, yield 99%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 7.05 (t, <i>J</i> = 7.2 Hz, 1H), 6.37 (d, <i>J</i> = 8.0 Hz, 1H), 6.25â6.20 (m, 2H), 4.24 (s, 2H), 3.19 (m, 4H), 2.6 (t, <i>J</i> = 4.4 Hz, 4H) ppm.</div></div><div id="sec4_9_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> <i>N</i>-(2-Trifluoromethyl)-3-nitro-phenylbenzenesulfonamide (<b>28</b>)</h4><div class="NLM_p last">To a stirred solution of 2-(trifluoromethyl)aniline (1.09 g, 6.76 mmol) in 5 mL of anhydrous pyridine was added the corresponding 3-nitrobenzene sulfonyl chloride (1.5 g, 6.76 mmol) at 0 Â°C. The corresponding solution was stirred at r.t. under nitrogen atmosphere, for 5 h. After completion of the reaction, the mixture was acidified with 20 mL of 2 N HCl, the aqueous phase was extracted several times, and the combined organic phases were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated. The residue was purified by flash chromatography on silica gel (hexane/AcOEt 3:1). Yield 84%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 8.56 (s, 1H), 8.40â8.38 (d, <i>J</i> = 8.4 Hz, 1H), 8.06â8.04 (d, <i>J</i> = 8 Hz, 1H), 7.86â7.84 (d, <i>J</i> = 8 Hz, 1H), 7.69â7.64 (t, <i>J</i> = 8 Hz, 1H), 7.61â7.57 (t, <i>J</i> = 7.8 Hz, 1H), 7.52â7.50 (d, <i>J</i> = 8.0 Hz, 1H), 7.31â7.27 (t, <i>J</i> = 8.1 Hz, 1H), 6.86 (s, 1H). MS (ESI): <i>m</i>/<i>z</i> 346.8 [M + H]<sup>+</sup>.</div></div><div id="sec4_9_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 3-Amino-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>29</b>)</h4><div class="NLM_p last">Sulfonamide <b>28</b> (400 mg, 1.35 mmol) was solubilized in 20 mL of anhydrous EtOH, and palladium on charcoal (60 mg) was added. The reaction mixture was stirred under hydrogen atmosphere for 1 h. Then, the mixture was filtered off on a celite pad and concentrated in vacuo and the crude product was purified by flash chromatography on silica gel (purification eluent: hexane/AcOEt 3:1). Yield 84%. <sup>1</sup>H NMR (acetone): Î´ 7.63â7.61 (d, <i>J</i> = 8 Hz, 1H), 7.51â7.58 (m, 2H), 7.24â7.20 (t, <i>J</i> = 8 Hz, 1H), 7.19â7.17 (m, 3H), 7.07â7.05 (d, <i>J</i> = 8 Hz, 1H), 6.90â6.88 (d, <i>J</i> = 8 Hz, 1H), 4.91 (s, 2H). MS (ESI): <i>m</i>/<i>z</i> 338.8 [M + Na]<sup>+</sup>.</div></div><div id="sec4_9_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 3-Azido-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>30</b>)</h4><div class="NLM_p last"><b>29</b> (100 mg, 0.41 mmol) was dissolved in CH<sub>3</sub>CN and cooled to 0 Â°C in an iceâsalt bath. To this stirred solution, <i>t</i>-BuONO (225 Î¼L, 1.89 mmol) was added and the mixture was stirred for 10 min; thereafter, TMSN<sub>3</sub> (200 Î¼L, 1.52 mmol) was added dropwise, for 10 min, and the resulting brown solution was stirred at r.t. One hour later the solvent was removed at reduced pressure and the residue was purified by flash chromatography on silica gel with the appropriate eluent. Purification eluent: hexane/AcOEt 4:1. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 7.84â7.81 (d, <i>J</i> = 8.4 Hz, 1H), 7.56â7.50 (m, 3H), 7.43â7.37 (m, 2H), 7.27â7.23 (t, <i>J</i> = 7.6 Hz, 1H), 7.18â7.16 (d, <i>J</i> = 8 Hz, 1H), 6.87 (s, 1H) ppm. MS (ESI): <i>m</i>/<i>z</i> 364.8 [M + Na]<sup>+</sup>.</div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> General Procedure for the Preparation of Compounds <b>31</b>â<b>34</b></h3><div class="NLM_p">The appropriate alkyne (6.08 mmol) and the opportune azide (5.07 mmol) were suspended in a 1:1 mixture of water and <i>t-</i>BuOH (1.5 mL each) in a 10 mL glass vial equipped with a small magnetic stirring bar. To this, sodium ascorbate (2.5 mmol) and copper(II) sulfate pentahydrate (2.50 mmol) were added. The mixture was then heated for 10 min at 125 Â°C under microwave irradiation, using an irradiation power of 300 W. Thereafter, the solvent was removed at reduced pressure, water was added, and the mixture was extracted with EtOAc (3 Ã 20 mL). The organic layers were collected, washed with brine, and dried over Na<sub>2</sub>SO<sub>4</sub>. The crude was purified by flash chromatography on silica gel using the opportune eluent: to give the desired triazole compounds <b>31</b>â<b>34</b>.</div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 3-(4-Isopentyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>31</b>)</h4><div class="NLM_p last">Purification eluent: PE/EtOAc 7:2, yield 69%, yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>): Î´ 8.06 (s, 1H), 8.01â8.00 (d, <i>J</i> = 7.6 Hz, 1H), 7.84â7.82 (d, <i>J</i> = 8.0 Hz, 1H), 7.76â7.74 (d, <i>J</i> = 7.6 Hz, 1H), 7.69 (1H, s), 7.60â7.56 (t, <i>J</i> = 8.4 Hz, 1H), 7.53â7.49 (m, 2H), 7.27â7.25 (d, <i>J</i> = 8.0 Hz, 1H), 6.98 (s, 1H), 2.80â2.78 (m, 2H), 1.62 (m, 3H), 0.99â0.94 (m, 6H) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>): Î´ 149.95, 143.95, 137.71, 136.51, 133.39, 130.71, 130.14, 128.54, 126.68, 126.01, 125.29, 124.91, 124.15, 121.56 (q, <i>J</i><sub>(CâF)</sub> = 270 Hz), 118.51, 38.27, 27.62, 23.51, 22.36 (2C) ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>20</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 437.1259, found 437.1255.</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 3-(4-Cyclohexyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>32</b>)</h4><div class="NLM_p last">Purification eluent: PE/EtOAc 7:3, yield 71%, pale white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>): Î´ 8.39 (s, 1H), 8.35 (s, 1H), 8.18â8.16 (d, <i>J</i> = 7.8 Hz, 1H), 7.87â7.85 (d, <i>J</i> = 7.8 Hz, 1H), 7.80â7.76 (t, <i>J</i> = 7.8 Hz, 1H), 7.68â7.66 (d, <i>J</i> = 7.8 Hz, 1H), 7.63â8.60 (m, 2H), 7.52â7.50 (d, <i>J</i> = 7.8 Hz, 1H), 7.44â7.40 (t, <i>J</i> = 8.0 Hz, 1H), 3.47â3.41 (m, 1H), 2.51â2.37 (m, 4H), 1.80â1.63 (m, 6H). <sup>13</sup>C NMR (CDCl<sub>3</sub>): Î´ 149.31, 141.73, 138.07 (q, <i>J</i><sub>(CâF)</sub> = 32 Hz), 137.00, 134.84, 123.37, 128.55, 128.22, 127.57, 127.48, 127.07, 126.24, 123.95, 123.67 (q, <i>J</i><sub>(CâF)</sub> = 270 Hz), 119.92, 37.20, 32.15 (2C), 25.92, 25.15 (2C) ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>20</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 449.1259, found 449.1262.</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 3-(4-Ethyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>33</b>)</h4><div class="NLM_p last">Purification eluent: PE/EtOAc 7:2, yield 78%, white solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>): Î´ 8.77 (s, 1H), 8.40 (s, 1H), 8.35 (s, 1H), 8.17â8.15 (d, <i>J</i> = 7.6 Hz, 1H), 7.88â7.86 (d, <i>J</i> = 7.6 Hz, 1H), 7.81â7.77 (t, <i>J</i> = 8.0 Hz, 1H), 7.69â7.67 (t, <i>J</i> = 8.0 Hz, 1H), 7.64â7.60 (t, <i>J</i> = 8.0 Hz, 1H), 7.52â7.50 (d, <i>J</i> = 8.0 Hz, 1H), 7.45â7.41 (t, <i>J</i> = 8.0 Hz, 1H), 2.76â2.78 (q, <i>J</i> = 7.6 Hz, 2H), 1.30â1.26 (t, <i>J</i> = 7.6 Hz, 3H) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>): Î´ 149.1, 142.76, 137.94, 134.22, 133.37, 130.97, 127.47, 126.97, 126.37, 125.31, 124.80, 123.87, 121.89 (q, <i>J</i><sub>(CâF)</sub> = 270 Hz), 119.26 (m), 118.29 (m), 18.75, 12.92 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>14</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 395.0790, found 395.0787.</div></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 3-(4-Phenyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>34</b>)</h4><div class="NLM_p last">Purification eluent: PE/EtOAc 7:2, yield 78%, yellow solid. <sup>1</sup>H NMR (CDCl<sub>3</sub>): Î´ 9.09 (s, 1H), 8.82 (s, 1H), 8.44 (s, 1H), 8.28â8.26 (d, <i>J</i> = 7.6 Hz, 1H), 7.99â7.97 (d, <i>J</i> = 8.0 Hz, 2H), 7.94â7.92 (d, <i>J</i> = 7.6 Hz, 1H), 7.86â7.82 (t, <i>J</i> = 7.6 Hz, 1H), 7.70â7.68 (d, <i>J</i> = 8.1 Hz, 1H), 7.65â7.61 (t, <i>J</i> = 7.6 Hz, 1H), 7.55â7.53 (t, <i>J</i> = 8.0 Hz, 1H), 7.48â7.42 (m, 3H), 7.36â7.34 (t, <i>J</i> = 8.0 Hz, 1H) ppm. <sup>13</sup>C NMR (CDCl<sub>3</sub>): Î´ 148.29, 142.80, 137.76, 134.10, 133.37, 131.09, 130.54, 129.00 (2C), 128.82, 128.45, 127.63, 127.40, 126.96, 126.74, 125.66 (2C), 124.94, 124.14 (m), 121.90, 118.84 (q, <i>J</i><sub>(CâF)</sub> = 30.3 Hz), ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>14</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 443.0790, found 443.0793.</div></div><div id="sec4_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide (<b>35</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1, yield 93%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 8.89â7.81 (m, 5H) 7.75 (s, 1H), 7.57â7.53 (t, <i>J</i> = 8.0 Hz, 1H), 7.51â7.49 (d, <i>J</i> = 8.0 Hz, 1H), 7.26â7.22 (m 1H), 6.94 (s, 1H), 2.79â2.75 (t, <i>J</i> = 7.4 Hz, 2H), 1.73â1.65 (m, 2H), 1.43â1.36 (m, 2H), 0.95â0.91 (t, <i>J</i> = 8.0 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>-<i>d</i>): 149.9, 140.8, 137.99, 133.88, 133.81, 133.56, 129.53 (2C), 126.71 (q, <i>J</i><sub>(CâF)</sub> = 4.4 Hz), 125.70, 123.89, 122.27 (q, <i>J</i><sub>(CâF)</sub> = 274 Hz), 120.88 (2C), 118.45, 31.32, 25.24, 22.24, 13.76 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>2</sub>S [M â H]<sup>â</sup> 423.1103, found 423.1107.</div></div><div id="sec4_10_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 4-Nitro-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide</h4><div class="NLM_p last">The residue was purified by flash chromatography on silica gel (hexane/AcOEt 3:1), yield 84%. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>): Î´ 8.82 (s, 1H), 8.27â8.25 (d, <i>J</i> = 8.0 Hz, 2H), 7.91â7.89 (d, <i>J</i> = 8.0 Hz, 1H), 7.87â7.85 (d, <i>J</i> = 8.0 Hz, 2H), 7.61â7.57 (t, <i>J</i> = 8.0 Hz, 1H), 7.53â7.51 (d, <i>J</i> = 8.0 Hz, 1H), 7.32â7.25 (t, <i>J</i> = 8 Hz, 1H) ppm. MS: <i>m</i>/<i>z</i> 345.8 [M â H]<sup>â</sup>.</div></div><div id="sec4_10_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 4-Amino-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 2:1, yield 97%. <sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>): Î´ 7.89 (s, 1H), 7.53â7.51 (d, <i>J</i>=8.0 Hz, 2H), 7.49â7.47 (d, <i>J</i> = 8.0 Hz, 1H), 7.45â7.43 (d, <i>J</i> = 8 Hz, 2H), 7.30â7.28 (t, <i>J</i> = 8 Hz, 1H), 6.62â6.60 (t, <i>J</i> = 7.8 Hz, 1H), 6.50 (d, <i>J</i> = 8.0 Hz, 1H), 4.62 (s, 2H). MS (ESI): <i>m</i>/<i>z</i> 338.8 [M + Na]<sup>+</sup>.</div></div><div id="sec4_10_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 4-Azido-<i>N</i>-(2-(trifluoromethyl)phenyl)benzenesulfonamide</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 4:1. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>): Î´ 7.81â7.79 (m, 2H), 7.74â7.71 (d, <i>J</i> = 7.4 Hz, 1H), 7.52â7.46 (m, 2H), 7.22â7.18 (t, <i>J</i> = 8 Hz, 1H), 7.02â6.98 (m, 3H) ppm. <i>m</i>/<i>z</i> 364.7 [M + Na]<sup>+</sup>.</div></div><div id="sec4_10_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-(2-Hydroxyphenyl)-4-nitrobenzenesulfonamide</h4><div class="NLM_p last"><sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>): Î´ 8.25â8.22 (d, <i>J</i> = 8.4 Hz, 2H), Î´ 7.93â7.91 (m, 3H), 7.33â7.31 (m, 2H), 6.97â6.94 (t, <i>J</i> = 7.6 Hz, 1H), 6.76â6.73 (t, <i>J</i> = 7.6 Hz, 1H), 6.66â6.64 (d, <i>J</i> = 8 Hz, 1H) ppm. MS: <i>m</i>/<i>z</i> 292.8 [M â H]<sup>â</sup>.</div></div><div id="sec4_10_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 4-Amino-<i>N</i>-(2-hydroxyphenyl)benzenesulfonamide</h4><div class="NLM_p last"><sup>1</sup>H NMR (acetone-<i>d</i><sub>6</sub>): Î´ 7.55â7.53 (d, <i>J</i> = 8.0 Hz, 2H), 7.42â7.40 (d, <i>J</i> = 8.0 Hz, 2H), 7.22â7.18 (m, 1H), 6.90â6.84 (m, 2H), 6.70â6.66 (m, 2H), 6.55â6.53 (d, <i>J</i> = 8.8 Hz, 1H) ppm. MS: <i>m</i>/<i>z</i> 265.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_10_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-Azido-<i>N</i>-(2-hydroxyphenyl)benzenesulfonamide</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 3:1. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>): Î´ 7.73â7.72 (d, <i>J</i> = 8.0 Hz, 2H), 7.36â7.35 (d, <i>J</i> = 6.4 Hz, 1H), 7.09â7.07 (d, <i>J</i> = 8.0 Hz, 2H), 6.94â6.90 (m, 2H), 6.73â6.66 (m, 2H) ppm. MS: <i>m</i>/<i>z</i> 312.8 [M + Na]<sup>+</sup>.</div></div><div id="sec4_10_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 4-(4-Butyl-1<i>H</i>-1,2,3-triazol-1-yl)-<i>N</i>-(2-hydroxyphenyl)benzenesulfonamide (<b>36</b>)</h4><div class="NLM_p last">Purification eluent: hexane/AcOEt 4:1, yield 80%. <sup>1</sup>H NMR (400 MHz, MeOD-<i>d</i><sub>4</sub>): Î´ 8.34 (s, 1H), 7.93â7.87 (m, 4H), 7.33â7.31 (d, <i>J</i> = 6.8 Hz, 1H), 6.96â6.92 (m, 1H), 6.76â7.73 (t, <i>J</i> = 7.6 Hz, 1H), 6.67â6.65 (d, <i>J</i> = 8.0 Hz, 1H), 2.78â2.74 (t, <i>J</i> = 7.6 Hz, 2H), 1.74â1.66 (m, 2H), 1.45â1.36 (m, 2H), 1.27â1.22 (t, <i>J</i> = 7.2 Hz, 3H) ppm. <sup>13</sup>C NMR (100 MHz, MeOD-<i>d</i><sub>4</sub>): Î´ 150.25, 149.21, 139.89, 139.78, 128.89 (2C), 126.49, 124.93, 123.87, 119.85, 119.55 (2C), 119.16, 114.93, 30.99, 24.39, 21.44, 12.89 ppm. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S [M â H]<sup>â</sup> 371.1178, found 371.1173.</div></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64">Assays</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74843" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74843" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Enzymatic Assays</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Protein Expression and Purification</h4><div class="NLM_p last">Recombinant his-tagged human full length DDX3X was cloned in the <i>Escherichia coli</i> expression vector pET-30a(+). ShuffleT7 <i>E. coli</i> cells were transformed with the plasmid and grown at 37 Â°C up to OD<sub>600</sub> = 0.7. DDX3X expression was induced with 0.5 mM isopropyl Î²-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside at 15 Â°C O/N. Cells were harvested by centrifugation and lysed and the crude extract centrifuged at 100.000xg for 60 min at 4 Â°C in a Beckman centrifuge before being loaded onto a FPLC Ni-NTA column (GE Healthcare). The column was equilibrated in buffer A (50 mM TrisâHCl pH 8.0, 250 mM NaCl, 25 mM imidazole and 20% glycerol). After extensive washing in buffer A, the column was eluted with a linear gradient in buffer A from 25 to 250 mM imidazole. Proteins in the eluted fractions were visualized on a sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE) and tested for the presence of DDX3X by Western blot with anti-DDX3X A300-A450 (BETHYL) at 1:2000 dilution in 5% milk. Fractions containing the purest DDX3X protein were pooled and stored at â80 Â°C.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Helicase Assay Based on Fluorescence Resonance Energy Transfer</h4><div class="NLM_p">The dsRNA substrate for the helicase assay was prepared by hybridizing two ssRNA oligonucleotides with the following sequences:</div><div class="NLM_p">Fluo-FAM 5â² UUUUUUUUUUUUU<u class="uu">UAGUACCGCCACCCUCAGAACC</u> 3â²</div><div class="NLM_p">Qu-BHQ1 5â² GGUUCUGAGGGUGGCGGUACUA 3â²</div><div class="NLM_p">DNA capture 5â² TAGTACCGCCACCCTCAGAACC 3â²</div><div class="NLM_p">The sequence in Fluo-FAM complementary to Qu-BHQ1 is underlined. Flu-FAM carries a 6-carboxyfluorescein fluorophore at its 3â² end, whereas Qu-BHQ1 carries a Black Hole Quencher group at its 5â² end. The DNA capture oligonucleotide is complementary to the Qu-BHQ1 oligonucleotide but bears no modifications.</div><div class="NLM_p last">Helicase assay using the dsRNA substrate was performed in 20 mM TrisâHCl (pH 8), 70 mM KCl, 2 mM MgCl<sub>2</sub>, 2 mM dithiothreitol, 12 units RNasin, 2 mM ATP, 50 nM dsRNA, and 100 nM capture strand in 20 Î¼L of reaction volume. The unwinding reaction was started by adding 60 pmol of DDX3X recombinant protein and carried out at 37 Â°C for 40 min using a LightCycler 480 (Roche). The fluorescence intensity was recorded every 30 s. Data of fluorescence signal were analyzed by linear interpolation, and the corresponding slope values were used to determine the apparent unwinding rate.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Kinetic Analysis</h4><div class="NLM_p last">The IC<sub>50</sub> values have been calculated from doseâresponse curves. Data (in triplicate) were plotted and analyzed by least-squares nonlinear regression, according to the method of MarquardtâLevenberg, with the computer program GraphPad Prism 6.0. Data were fitted to the following equation<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_m001.gif" alt="" id="_i69" /></img><span class="labelSpan">(1)</span></span>where <i>E</i><sub>(obs)</sub> is the observed enzymatic activity in the presence of each inhibitor dose [I], <i>E</i><sub>(max)</sub> is the maximal enzymatic activity in the absence of the inhibitor, and <i>n</i> is an exponential term to take into account sigmoidal doseâresponse curves.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> Cell Extracts (CEs) and DDX3X Quantification</h4><div class="NLM_p last">Ten million cells were ruptured with a Dounce homogenizer in lysis buffer (50 mM TrisâHCl pH 8.0, 0.1% SDS, 350 mM NaCl, 0.25% Triton X-100, protease inhibitor cocktail (Sigma-Aldrich)). The lysate was incubated for 30 min on ice, sonicated for 5 min (at 30 s intervals), and centrifuged at 15.000xg for 10 min at 4 Â°C. The protein concentration in the supernatant (crude extract) was quantified with Bradford. For DDX3X quantification, increasing concentrations of crude extract (5, 10, 20, and 40 Î¼g) were loaded on a SDS-PAGE alongside known concentrations (50, 100, 150, and 200 ng) of recombinant purified DDX3X. Separated proteins were subjected to Western blot analysis with anti-DDX3X A300-A450 (BETHYL) at 1:2000 dilution in 5% milk. The blot was next incubated with a horseradish peroxidase-conjugated secondary antibody, and the bands corresponding to DDX3X were visualized with enhanced chemiluminescent substrate (Westar Nova 2.0, Cyanagen) using a ChemiDoc XRS (Bio-Rad) apparatus. The intensity of the bands was measured by densitometry, and the values obtained for the purified DDX3X were plotted as a function of the protein concentration and analyzed by linear interpolation to derive a reference curve, whose slope corresponded to the estimated intensity (<i>I</i>) Ã ng<sup>â1</sup> (DDX3X) value. This parameter was used to calculate the <i>I</i> Ã ng<sup>â1</sup> values for the DDX3X in the cell extract, from the intensities of the DDX3X bands in the corresponding cell extract (CE) samples. The mean <i>I</i> Ã ng<sup>â1</sup> (CE) value was used to calculate the ng of DDX3X Ã Î¼L<sup>â1</sup> of extract and then the total ng of DDX3X present in the extract. This value was divided for the total number of cells used (10<sup>7</sup>), to derive the ng DDX3X/cell. On the basis of the known molecular weight of DDX3X, the Avogadro number, and assuming a mean cellular volume of 6.55 Ã 10<sup>â11</sup> L, the final molar concentration of DDX3X per cell was calculated. Each experiment was repeated three times, with each blot carrying a reference curve alongside the extract samples, to account for variations in loading/transfer efficiency. The reference <i>I</i> Ã ng<sup>â1</sup> (DDX3X) value obtained was anyway comparable across the different experiments (Â±20% variation).</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> ATPase Assay</h4><div class="NLM_p">The ATPase assay was carried out by using the commercial kit Promega ADP-Glo Kinase Assay. The reaction was performed in 30 mM TrisâHCl, 9 mM MgCl<sub>2</sub>, 0.05 mg mL<sup>â1</sup> bovine serum albumin, 50 Î¼M ATP, and 4 Î¼M DDX3. The reaction was performed following the ADP-Glo Kinase Assay Protocol, and luminescence was measured with MicroBeta TriLux PerkinElmer.</div><div class="NLM_p last">The anti-DDX1 activity was evaluated as previously reported.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Antiviral Assay</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> WNV Inhibitory Viral Plaque Reduction Assay</h4><div class="NLM_p last">Huh7 cells, derived from human hepatoma (kindly provided from Istituto Toscano Tumori, Core Research Laboratory, Siena, Italy) was used for the inhibitory viral plaque reduction assay. The cell propagation medium was Dulbeccoâs modified Eagleâs medium (DMEM; Sigma, Milano, Italy) supplemented with 10% fetal bovine serum (Sigma) and 1% penicillin/streptomicin (Sigma). WNV (strain of lineage 2) viral stock, consisting of cell-free supernatants of acutely infected Huh7 cells, was aliquoted and stored at â80 Â°C until used. Titration of the viral stocks as a plaque forming unit (PFU) was carried out in Huh7 cells. WNV was used to infect Huh7 cell line in duplicate, and viral plaques were visualized 4 days following infection. Briefly, six-well plates were seeded with 2.5 Ã 10<sup>5</sup> cells in 3 mL of growth medium and kept overnight at 37 Â°C with 5% CO<sub>2</sub>. On the day of infection, after removal of the growth medium, cell monolayers at 80â90% confluence were infected with WNV viral stock with a multiplicity of infection (MOI) of 0.1 in a final volume of 0.3 mL and incubated for 1 h at 37 Â°C with 5% CO<sub>2</sub>. Then, cells were washed with phosphate-buffered saline (PBS) 1Ã and 30 Î¼L of dimethyl sulfoxide (DMSO) alone (viral positive control) or with 10-fold serial dilutions of DDX3X inhibitory compounds immediately added in duplicates together with 300 Î¼L of fresh DMEM complete medium (compound final concentrations of 100, 10, 1, 0.1, and 0.01 Î¼M). 1-Î²-<span class="smallcaps smallerCapital">d</span>-Ribofuranosyl-1,2,4-triazole-3-carboxamide (ribavirin; Sigma) diluted in DMSO was used as inhibitory reference control. Then, the overlay medium composed by 0.5% SeaPlaque Agarose (Lonza, Basel, Switzerland) diluted in propagation medium was added to each well. After 4 days (Huh7) of incubation at 37 Â°C, the monolayers were fixed with methanol (Carlo Erba Chemicals, Milan, Italy) and stained with 0.1% crystal violet (Carlo Erba Chemicals) and the viral titers were calculated by plaque forming unit (PFU) counting. The percent of plaque reduction activity was calculated by dividing the average PFU of treated samples by the average of DMSO-treated samples (viral positive control). Fifty percent inhibitory concentrations (IC<sub>50</sub>) were calculated using the predicted exponential growth function in Microsoft Excel, which uses existing <i>x</i>â<i>y</i> data to estimate the corresponding anti-DDX3X compound concentration (<i>x</i>) from a known value (<i>y</i>), which in this case was 50% PFU. Mean IC<sub>50</sub> Â± standard deviations (S.D.) were calculated using all replicates. All experiments were repeated at least twice. All experimental procedures were conducted under biosafety level 3 containment.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Cytotoxicity Assay</h4><div class="NLM_p last">Monolayers of 2.5 Ã 10<sup>4</sup> Huh7 cells per well were kept in flat-bottom 96-well culture plates and allowed to adhere overnight. Then, when the cell layers were confluent, the medium was removed, the wells were washed twice with PBS, treated with 100 Î¼L of DMEM, with 10 Î¼L of DMSO alone (cell positive control), or with various concentrations of DDX3X inhibitory compounds under study (compound final concentrations of 100, 10, 1, 0.1, and 0.01 Î¼M) and incubated for 3 days at 37 Â°C in a CO<sub>2</sub> atmosphere. After treatment, an 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide kit (Roche, Milan, Italy) was used according to the supplierâs instructions and the absorbance of each well was determined using a microplate spectrophotometer at a wavelength of 570 nm. Cytotoxicity was calculated by dividing the average optical density of treated samples by the average of DMSO-treated samples (cell positive control).</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Virus RNA Quantification and Capsid Protein Detection Assay</h4><div class="NLM_p">Total RNA was extracted and purified from infected and control cells using an RNAeasy mini Kit (Qiagen). Two hundred nanograms of total RNA of each sample in a 20 Î¼L reaction mixture were reverse-transcribed by using the indicated primers (primer PROCf 5â² C C T g T g T g A g C T g A C A A A C T T A g T 3â² for the transcription of WNV complementary RNA and primer PROCr 5â² g C g T T T T A g C A T A T T g A C A g C C 3â² for the transcription of WNV genomic RNA) and amplified by real-time PCR in a Rotor-gene 3000 real-time thermal cycler (Corbett Research, Australia) with the following WNV specific primer and probe (primer PROCf 5â² C C T g T g T g A g C T g A C A A A C T T A g T 3â² and primer PROCr 5â² g C g T T T T A g C A T A T T g A C A g C C 3â², and probe PROC-TMD VIC-5â² CCTGGTTTCTTAGACATCGAGATCTTCGTGC 3â²).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Plasmid calibration standards were used for quantification of the template concentration. The sensibility of the assay was 10 copies.</div><div class="NLM_p last">The specific WNV capsid protein was investigated with Western blot analysis of Huh7-infected cells electrophoresed on 10% SDS-PAGE and then probed with anticapsid protein polyclonal antibody followed by peroxidase-conjugated antirabbit IgG polyclonal antibody.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> ADME Assay</h4><div id="sec5_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> Chemicals</h5><div class="NLM_p last">All solvents and reagents were from Sigma-Aldrich Srl (Milan,Italy), brain polar lipid extract (porcine) was from Avanti Polar Lipids, INC. (Alabama). Dodecane was purchased from Fluka (Milan, Italy). Pooled male donors 20 mg mL<sup>â1</sup> HLM were from BD Gentest-Biosciences (San Jose, California). Milli-Q quality water (Millipore, Milford, MA) was used. Hydrophobic filter plates (MultiScreen-IP, Clear Plates, 0.45 Î¼m diameter pore size), 96-well microplates, and 96-well UV-transparent microplates were obtained from Millipore (Bedford, MA).</div></div><div id="sec5_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> Parallel Artificial Membrane Permeability Assay (PAMPA and PAMPAâBloodâBrain Barrier (BBB))</h5><div class="NLM_p last">Donor solution (0.5 mM) was prepared by diluting 1 mM dimethyl sulfoxide (DMSO) compound stock solution using phosphate buffer (pH 7.4, 0.025 M). Filters were coated with 5 Î¼L of a 1% (w/v) dodecane solution of phosphatidylcholine or 4 Î¼L of brain polar lipid solution (20 mg mL<sup>â1</sup> 16% CHCl<sub>3</sub>, 84% dodecane) prepared from CHCl<sub>3</sub> solution 10% w/v, for intestinal permeability and BBB permeability, respectively. Donor solution (150 Î¼L) was added to each well of the filter plate. To each well of the acceptor plate was added 300 Î¼L of solution (50% DMSO in phosphate buffer). All compounds were tested in three different plates on different days. The sandwich was incubated for 5 h at room temperature under gentle shaking. After the incubation time, the plates were separated and samples were taken from both receiver and donor sides and analyzed using LC with UV detection at 280 nm. LC analysis was performed with a PerkinElmer (series 200) instrument equipped with an UV detector (PerkinElmer 785A, UV/vis detector). Chromatographic separation was conducted using a Polaris C18 column (150 Ã 4.6 mm<sup>2</sup>, 5 Î¼m particle size) at a flow rate of 0.8 mL min<sup>â1</sup> with a mobile phase composed of 60% ACN/40% H<sub>2</sub>Oâformic acid 0.1% for all compounds. Permeability (<i>P</i><sub>app</sub>) was calculated according to the following equation with some modification to obtain permeability values in cm s<sup>â1</sup><span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_m002.gif" alt="" id="_i79" /></img></span>where <i>V</i><sub>A</sub> is the volume in the acceptor well, <i>V</i><sub>D</sub> is the volume in the donor well (cm<sup>3</sup>), <i>A</i> is the âeffective areaâ of the membrane (cm<sup>2</sup>), <i>t</i> is the incubation time (s), and <i>r</i> is the ratio between drug concentration in the acceptor and equilibrium concentration of the drug in the total volume(<i>V</i><sub>D</sub> + <i>V</i><sub>A</sub>). Drug concentration is estimated by using the peak area integration. Membrane retentions (%) were calculated according to the following equation<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_m003.gif" alt="" id="_i80" /></img></span>where <i>r</i> is the ratio between drug concentration in the acceptor and equilibrium concentration and <i>D</i>, <i>A</i>, and Eq represented drug concentration in the donor, acceptor and equilibrium solution, respectively.</div></div><div id="sec5_2_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> Water Solubility Assay</h5><div class="NLM_p last">Compound <b>1</b> (1 mg) was added to 1 mL of water. The sample was shaked in a shaker bath at room temperature for 24â36 h. The suspensions were filtered through a 0.45 Î¼m nylon filter (Acrodisc) and the solubilized compound determined by LC-MS-MS assay. The determination was performed in triplicate. For the quantification was used an LC-MS system consisting of a Varian apparatus (Varian Inc.), including a vacuum solvent degassing unit, two pumps (212-LC), a Triple Quadrupole MSD (mod. 320-LC) mass spectrometer with ES interface and Varian MS Workstation System Control vers. 6.9 software. Chromatographic separation was obtained using a Pursuit C18 column (50 Ã 2.0 mm<sup>2</sup>) (Varian) with 3 Î¼m particle size and gradient elution: eluent A being ACN and eluent B consisting of an aqueous solution of formic acid (0.1%). The analysis started with 0% of eluent A, which was linearly increased up to 70% in 10 min, then slowly increased up to 98% up to 15 min. The flow rate was 0.3 mL min<sup>â1</sup>, and the injection volume was 5 Î¼L. The instrument operated in positive mode and parameters were: detector 1850 V, drying gas pressure 25.0 psi, desolvation temperature 300.0 Â°C, nebulizing gas 45.0 psi, needle 5000 V, and shield 600 V. Nitrogen was used as nebulizer gas and drying gas. Collision-induced dissociation was performed using argon as the collision gas at a pressure of 1.8 mTorr in the collision cell. Quantification of the single compound was made by comparison with apposite calibration curves realized with standard solutions in methanol.</div></div><div id="sec5_2_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> Microsomal Stability Assay</h5><div class="NLM_p last">Each compound in DMSO solution was incubated at 37 Â°C for 60 min in 125 mM phosphate buffer (pH 7.4) and 5 Î¼L of human liver microsomal protein (0.2 mg mL<sup>â1</sup>), in the presence of a NADPH-generating system at a final volume of 0.5 mL (compoundsâ final concentration, 50 Î¼M); DMSO did not exceed 2% (final solution). The reaction was stopped by cooling in ice and adding 1.0 mL of acetonitrile. The reaction mixtures were then centrifuged, and the parent drug and metabolites were subsequently determined by LC-UV-MS. Chromatographic analysis was performed with an Agilent 1100 LC/MSD VL system (G1946C) (Agilent Technologies, Palo Alto, CA) constituted by a vacuum solvent degassing unit, a binary high-pressure gradient pump, an 1100 series UV detector, and an 1100 MSD model VL benchtop mass spectrometer. Chromatographic separation was obtained using a Varian Polaris C18-A column (150 Ã 4.6 mm<sup>2</sup>, 5 Î¼m particle size) and gradient elution: eluent A being ACN and eluent B consisting of an aqueous solution of formic acid (0.1%). The analysis started with 2% of eluent A, which was rapidly increased up to 70% in 12 min, then slowly increased up to 98% in 20 min. The flow rate was 0.8 mL min<sup>â1</sup>, and the injection volume was 20 Î¼L. The Agilent 1100 series mass spectra detection (MSD) single-quadrupole instrument was equipped with the orthogonal spray API-ES (Agilent Technologies, Palo Alto, CA). Nitrogen was used as nebulizing and drying gas. The pressure of the nebulizing gas, the flow of the drying gas, the capillary voltage, the fragmentor voltage, and the vaporization temperature were set at 40 psi, 9 L min<sup>â1</sup>, 3000 V, 70 V, and 350 Â°C, respectively. UV detection was monitored at 280 nm. The LC-ESI-MS determination was performed by operating the MSD in the positive-ion mode. Spectra were acquired over the scan range <i>m</i>/<i>z</i> 100â1500 using a step size of 0.1 u. The percentage of a nonmetabolized compound was calculated by comparison with reference solutions.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01403" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41434" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41434" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01403" class="ext-link">10.1021/acs.jmedchem.8b01403</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Western blot experiments for the quantification of DDX3X in cell extracts, representative IC<sub>50</sub> curves, anti-WNV doseâresponse and cytotoxicity curves (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_001.csv">jm8b01403_si_001.csv (1.87 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_002.pdf">jm8b01403_si_002.pdf (328.1 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01403" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87562" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87562" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maurizio Botta</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
Biotecnologie, Chimica e Farmacia, UniversitÃ 
degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</span>;Â 
    <span class="hlFld-Affiliation affiliation">Lead
Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</span>;Â 
    <span class="hlFld-Affiliation affiliation">Biotechnology
College of Science and Technology, Temple
University, BioLife Science
Building, Suite 333, 1900 North 12th Street, Philadelphia, Pennsylvania 19122, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0456-6995" title="Orcid link">http://orcid.org/0000-0003-0456-6995</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#33515c4747521d5e5246415a495a5c73545e525a5f1d505c5e"><span class="__cf_email__" data-cfemail="8ceee3f8f8eda2e1edf9fee5f6e5e3ccebe1ede5e0a2efe3e1">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Annalaura Brai</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
Biotecnologie, Chimica e Farmacia, UniversitÃ 
degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</span>;Â 
    <span class="hlFld-Affiliation affiliation">Lead
Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6395-9348" title="Orcid link">http://orcid.org/0000-0001-6395-9348</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesco Martelli</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental and Clinical Medicine, University
of Florence, I-50134 Florence, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Valentina Riva</span> - <span class="hlFld-Affiliation affiliation">Istituto
di Genetica Molecolare, IGM-CNR, Via Abbiategrasso 207, I-27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anna Garbelli</span> - <span class="hlFld-Affiliation affiliation">Istituto
di Genetica Molecolare, IGM-CNR, Via Abbiategrasso 207, I-27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberta Fazi</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
Biotecnologie, Chimica e Farmacia, UniversitÃ 
degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claudio Zamperini</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
Biotecnologie, Chimica e Farmacia, UniversitÃ 
degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</span>;Â 
    <span class="hlFld-Affiliation affiliation">Lead
Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessandro Pollutri</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
Biotecnologie, Chimica e Farmacia, UniversitÃ 
degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lucia Falsitta</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
Biotecnologie, Chimica e Farmacia, UniversitÃ 
degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefania Ronzini</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
Biotecnologie, Chimica e Farmacia, UniversitÃ 
degli Studi di Siena, Via A. De Gasperi 2, I-53100 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Maccari</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery Siena S.r.l., Castelnuovo Berardenga, I-53019 Siena, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giovanni Maga</span> - <span class="hlFld-Affiliation affiliation">Istituto
di Genetica Molecolare, IGM-CNR, Via Abbiategrasso 207, I-27100 Pavia, Italy</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8092-1552" title="Orcid link">http://orcid.org/0000-0001-8092-1552</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simone Giannecchini</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental and Clinical Medicine, University
of Florence, I-50134 Florence, Italy</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4554-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i85">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41434" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41434" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by First Health Pharmaceuticals B.V. (Amsterdam) for M.B. Financial support has been also provided by Tuscany Region Grant FAS Salute 2014 (DD 4042/2014) to M.B. and S.G.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">WNV</td><td class="NLM_def"><p class="first last">West Nile virus</p></td></tr><tr><td class="NLM_term"><i>t</i>-BuONO</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyl nitrite</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, and excretion</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i88">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60834" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60834" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 28 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chancey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rios, M.</span></span> <span> </span><span class="NLM_article-title">The global ecology and epidemiology of West Nile Virus</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>, <span class="NLM_elocation-id">376230</span> <span class="refDoi">Â DOI: 10.1155/2015/376230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1155%2F2015%2F376230" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&author=C.+Chanceyauthor=A.+Grinevauthor=E.+Volkovaauthor=M.+Rios&title=The+global+ecology+and+epidemiology+of+West+Nile+Virus&doi=10.1155%2F2015%2F376230"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1155%2F2015%2F376230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F376230%26sid%3Dliteratum%253Aachs%26aulast%3DChancey%26aufirst%3DC.%26aulast%3DGrinev%26aufirst%3DA.%26aulast%3DVolkova%26aufirst%3DE.%26aulast%3DRios%26aufirst%3DM.%26atitle%3DThe%2520global%2520ecology%2520and%2520epidemiology%2520of%2520West%2520Nile%2520Virus%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2015%26volume%3D2015%26doi%3D10.1155%2F2015%2F376230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">European
Centre for Disease
Prevention and Control</span>. Epidemiological update:
West Nile virus transmission season in Europe, 2017 [online], <span class="NLM_year">2018</span>. Available at: <a href="https://ecdc.europa.eu/en/news-events/epidemiological-update-west-nile-virus-transmission-season-europe-2017" class="extLink">https://ecdc.europa.eu/en/news-events/epidemiological-update-west-nile-virus-transmission-season-europe-2017</a> (Accessed 22 Aug, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=European%0ACentre+for+Disease%0APrevention+and+Control.+Epidemiological+update%3A%0AWest+Nile+virus+transmission+season+in+Europe%2C+2017+%5Bonline%5D%2C+2018.+Available+at%3A+https%3A%2F%2Fecdc.europa.eu%2Fen%2Fnews-events%2Fepidemiological-update-west-nile-virus-transmission-season-europe-2017+%28Accessed+22+Aug%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Cdc.gov</span>. Preliminary maps & data for 2017 | West Nile Virus | CDC [online], <span class="NLM_year">2018</span>. Available at: <a href="https://www.cdc.gov/westnile/statsmaps/preliminarymapsdata2017/index.html" class="extLink">https://www.cdc.gov/westnile/statsmaps/preliminarymapsdata2017/index.html</a> (Accessed 22 Aug, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cdc.gov.+Preliminary+maps+%26+data+for+2017+%7C+West+Nile+Virus+%7C+CDC+%5Bonline%5D%2C+2018.+Available+at%3A+https%3A%2F%2Fwww.cdc.gov%2Fwestnile%2Fstatsmaps%2Fpreliminarymapsdata2017%2Findex.html+%28Accessed+22+Aug%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">European centre for disease
prevention and control</span>. Weekly Updates: 2018
West Nile Fever Transmission Season [online], <span class="NLM_year">2018</span>. Available
at: <a href="https://ecdc.europa.eu/en/west-nile-fever/surveillance-and-disease-data/disease-data-ecdc" class="extLink">https://ecdc.europa.eu/en/west-nile-fever/surveillance-and-disease-data/disease-data-ecdc</a> (Accessed 31 Aug, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=European+centre+for+disease%0Aprevention+and+control.+Weekly+Updates%3A+2018%0AWest+Nile+Fever+Transmission+Season+%5Bonline%5D%2C+2018.+Available%0Aat%3A+https%3A%2F%2Fecdc.europa.eu%2Fen%2Fwest-nile-fever%2Fsurveillance-and-disease-data%2Fdisease-data-ecdc+%28Accessed+31+Aug%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilder-Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monath, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymann, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. W.</span></span> <span> </span><span class="NLM_article-title">Epidemic arboviral diseases: priorities for research and public health</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">e101</span>â <span class="NLM_lpage">e106</span>, <span class="refDoi">Â DOI: 10.1016/S1473-3099(16)30518-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2FS1473-3099%2816%2930518-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=28011234" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=e101-e106&author=A.+Wilder-Smithauthor=D.+J.+Gublerauthor=S.+C.+Weaverauthor=T.+P.+Monathauthor=D.+L.+Heymannauthor=T.+W.+Scott&title=Epidemic+arboviral+diseases%3A+priorities+for+research+and+public+health&doi=10.1016%2FS1473-3099%2816%2930518-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2816%2930518-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252816%252930518-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilder-Smith%26aufirst%3DA.%26aulast%3DGubler%26aufirst%3DD.%2BJ.%26aulast%3DWeaver%26aufirst%3DS.%2BC.%26aulast%3DMonath%26aufirst%3DT.%2BP.%26aulast%3DHeymann%26aufirst%3DD.%2BL.%26aulast%3DScott%26aufirst%3DT.%2BW.%26atitle%3DEpidemic%2520arboviral%2520diseases%253A%2520priorities%2520for%2520research%2520and%2520public%2520health%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2017%26volume%3D17%26spage%3De101%26epage%3De106%26doi%3D10.1016%2FS1473-3099%2816%2930518-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercorelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PalÃ¹, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loregian, A.</span></span> <span> </span><span class="NLM_article-title">Drug repurposing for viral infectious diseases: how far are we?</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">865</span>â <span class="NLM_lpage">876</span>, <span class="refDoi">Â DOI: 10.1016/j.tim.2018.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.tim.2018.04.004" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=865-876&author=B.+Mercorelliauthor=G.+Pal%C3%B9author=A.+Loregian&title=Drug+repurposing+for+viral+infectious+diseases%3A+how+far+are+we%3F&doi=10.1016%2Fj.tim.2018.04.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2018.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2018.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DMercorelli%26aufirst%3DB.%26aulast%3DPal%25C3%25B9%26aufirst%3DG.%26aulast%3DLoregian%26aufirst%3DA.%26atitle%3DDrug%2520repurposing%2520for%2520viral%2520infectious%2520diseases%253A%2520how%2520far%2520are%2520we%253F%26jtitle%3DTrends%2520Microbiol.%26date%3D2018%26spage%3D865%26epage%3D876%26doi%3D10.1016%2Fj.tim.2018.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valiente-EcheverrÃ­a, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermoso, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto-Rifo, R.</span></span> <span> </span><span class="NLM_article-title">RNA helicase DDX3: at the crossroad of viral replication and antiviral immunity</span>. <i>Rev. Med. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">286</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1002/rmv.1845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1002%2Frmv.1845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=26174373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFejtb3F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=286-299&author=F.+Valiente-Echeverr%C3%ADaauthor=M.+A.+Hermosoauthor=R.+Soto-Rifo&title=RNA+helicase+DDX3%3A+at+the+crossroad+of+viral+replication+and+antiviral+immunity&doi=10.1002%2Frmv.1845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">RNA helicase DDX3: at the crossroad of viral replication and antiviral immunity</span></div><div class="casAuthors">Valiente-Echeverria, Fernando; Hermoso, Marcela A.; Soto-Rifo, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">Reviews in Medical Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">286-299</span>CODEN:
                <span class="NLM_cas:coden">RMVIEW</span>;
        ISSN:<span class="NLM_cas:issn">1052-9276</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Summary : Asp-Glu-Ala-Asp (DEAD)-box polypeptide 3, or DDX3, belongs to the DEAD-box family of ATP-dependent RNA helicases and is known to play different roles in RNA metab. ranging from transcription to nuclear export, translation, and assembly of stress granules.  In addn., there is growing evidence that DDX3 is a component of the innate immune response against viral infections.  As such, DDX3 has been shown to play roles both upstream and downstream of I-kappa beta kinase Îµ (IKKÎµ)/TANK-binding kinase 1, leading to IFN-Î² prodn.  Interestingly, several RNA viruses, including human threats such as HIV-1 and hepatitis C virus, hijack DDX3 to accomplish various steps of their replication cycles.  Thus, it seems that viruses have evolved to exploit DDX3's functions while threatening the innate immune response.  Understanding this interesting dichotomy in DDX3 function will help us not only to improve our knowledge of virus-host interactions but also to develop novel antiviral drugs targeting the multifaceted roles of DDX3 in viral replication.  Copyright Â© 2015 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreE9U2GQXnXbVg90H21EOLACvtfcHk0ljJVjqj0Cnn_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFejtb3F&md5=93ccf21610b21186c66cdad36f6c4d99</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Frmv.1845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frmv.1845%26sid%3Dliteratum%253Aachs%26aulast%3DValiente-Echeverr%25C3%25ADa%26aufirst%3DF.%26aulast%3DHermoso%26aufirst%3DM.%2BA.%26aulast%3DSoto-Rifo%26aufirst%3DR.%26atitle%3DRNA%2520helicase%2520DDX3%253A%2520at%2520the%2520crossroad%2520of%2520viral%2520replication%2520and%2520antiviral%2520immunity%26jtitle%3DRev.%2520Med.%2520Virol.%26date%3D2015%26volume%3D25%26spage%3D286%26epage%3D299%26doi%3D10.1002%2Frmv.1845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chahar, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierson, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manjunath, N.</span></span> <span> </span><span class="NLM_article-title">Ago-2-mediated slicer activity is essential for anti-flaviviral efficacy of RNAi</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e27551</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0027551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1371%2Fjournal.pone.0027551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=22102908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyntLbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=S.+Chenauthor=H.+S.+Chaharauthor=S.+Abrahamauthor=H.+Wuauthor=T.+C.+Piersonauthor=X.+A.+Wangauthor=N.+Manjunath&title=Ago-2-mediated+slicer+activity+is+essential+for+anti-flaviviral+efficacy+of+RNAi&doi=10.1371%2Fjournal.pone.0027551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Ago-2-mediated slicer activity is essential for anti-flaviviral efficacy of RNAi</span></div><div class="casAuthors">Chen, Shuiping; Chahar, Harendra S.; Abraham, Sojan; Wu, Haoquan; Pierson, Theodore C.; Wang, Xiaozhong A.; Manjunath, N.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e27551</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">RNA interference can be mediated by fully complementary siRNA or partially complementary miRNA.  SiRNAs are widely used to suppress viral replication and the fully complementary siRNA bound Ago-2 in the RISC is known to degrade the target RNA.  Although other argonaute proteins lacking slicer activity can also bind oligonucleotides with both si and miRNA structures, whether they can also contribute to antiviral effects is not entirely clear.  We tested si and miRNA structured oligos for target repression in dual luciferase assays as well as for inhibition of Dengue and West Nile virus replication in ES cells expressing individual Ago proteins.  In luciferase assays, both fully complementary and partially complementary oligos effectively repressed their targets in all individual Ago expressing cell lines, although the efficacy with fully complementary oligos was higher in Ago-2+ cells.  However, partially complementary oligos had no effect on virus replication in any cell line, while fully complementary siRNAs were highly effective in Ago-2 expressing, but not in cells expressing other Ago proteins.  This occurred irresp. of whether the target sequences were located in the coding region or 3'UTR of the virus.  We conclude that Ago-2 slicer activity is essential for anti-viral efficacy of siRNAs and miRNA-mediated translational repression/transcript destabilization is too weak to suppress the abundantly expressed flaviviral proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLSq2VQ419g7Vg90H21EOLACvtfcHk0ljJVjqj0Cnn_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyntLbM&md5=5315d8fd2f8f7d87295e0864fba723d8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027551%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DChahar%26aufirst%3DH.%2BS.%26aulast%3DAbraham%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DPierson%26aufirst%3DT.%2BC.%26aulast%3DWang%26aufirst%3DX.%2BA.%26aulast%3DManjunath%26aufirst%3DN.%26atitle%3DAgo-2-mediated%2520slicer%2520activity%2520is%2520essential%2520for%2520anti-flaviviral%2520efficacy%2520of%2520RNAi%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0027551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chahar, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manjunath, N.</span></span> <span> </span><span class="NLM_article-title">P-body components LSM1, GW182, DDX3, DDX6 and XRN1 are recruited to WNV replication sites and positively regulate viral replication</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">7</span>, <span class="refDoi">Â DOI: 10.1016/j.virol.2012.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.virol.2012.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=23102969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKns7zL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2013&pages=1-7&author=H.+S.+Chaharauthor=S.+Chenauthor=N.+Manjunath&title=P-body+components+LSM1%2C+GW182%2C+DDX3%2C+DDX6+and+XRN1+are+recruited+to+WNV+replication+sites+and+positively+regulate+viral+replication&doi=10.1016%2Fj.virol.2012.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">P-body components LSM1, GW182, DDX3, DDX6 and XRN1 are recruited to WNV replication sites and positively regulate viral replication</span></div><div class="casAuthors">Chahar, Harendra S.; Chen, Shuiping; Manjunath, N.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In mammalian cells, proteins involved in mRNA silencing and degrdn. localize to discrete cytoplasmic foci called processing or P-bodies.  Here we show that microscopically visible P-bodies are greatly diminished following West Nile viral infection, but the component proteins are not depleted.  On the other hand, many P-body components including LSM1, GW182, DDX3, DDX6 and XRN1, but not others like DCP1a and EDC4 are recruited to the viral replication sites, as evidenced by their colocalization at perinuclear region with viral NS3.  Kinetic studies suggest that the component proteins are first released from P-bodies in response to WNV infection within 12 h post-infection, followed by recruitment to the viral replication sites by 24-36 h post-infection.  Silencing of the recruited proteins individually with siRNA interfered with viral replication to varying extents suggesting that the recruited proteins are required for efficient viral replication.  Thus, the P-body proteins might provide novel drug targets for inhibiting viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBfz18OOluLVg90H21EOLACvtfcHk0ljJVjqj0Cnn_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKns7zL&md5=d0ed494c172da24e0a8943b7126667ca</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2012.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2012.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DChahar%26aufirst%3DH.%2BS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DManjunath%26aufirst%3DN.%26atitle%3DP-body%2520components%2520LSM1%252C%2520GW182%252C%2520DDX3%252C%2520DDX6%2520and%2520XRN1%2520are%2520recruited%2520to%2520WNV%2520replication%2520sites%2520and%2520positively%2520regulate%2520viral%2520replication%26jtitle%3DVirology%26date%3D2013%26volume%3D436%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.virol.2012.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owsianka, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A. H.</span></span> <span> </span><span class="NLM_article-title">Hepatitis C virus core protein interacts with a human DEAD box protein DDX3</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>257</i></span>,  <span class="NLM_fpage">330</span>â <span class="NLM_lpage">340</span>, <span class="refDoi">Â DOI: 10.1006/viro.1999.9659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1006%2Fviro.1999.9659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10329544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADyaK1MXislWrt78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1999&pages=330-340&author=A.+M.+Owsiankaauthor=A.+H.+Patel&title=Hepatitis+C+virus+core+protein+interacts+with+a+human+DEAD+box+protein+DDX3&doi=10.1006%2Fviro.1999.9659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis C Virus Core Protein Interacts with a Human DEAD Box Protein DDX3</span></div><div class="casAuthors">Owsianka, Ania M.; Patel, Arvind H.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">257</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">330-340</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Several studies have implicated hepatitis C virus (HCV) core in influencing the expression of host genes.  To identify cellular factors with a possible role in HCV replication and pathogenesis, we looked for cellular proteins that interact with the viral core protein.  A human liver cDNA library was screened in a yeast two-hybrid assay to identify cellular proteins that bind to core.  Several pos. clones were isolated, one of which encoded the C-terminal 253 amino acids of a putative RNA helicase, a DEAD box protein designated DDX3.  Bacterially expressed glutathione-S-transferase-DDX3 fusion protein specifically pulled down in vitro translated and radiolabeled HCV core, confirming a direct interaction.  Immunofluorescent staining of HeLa cells with a polyclonal antiserum showed that DDX3 is located predominantly in nuclear speckles and at low levels throughout the cytoplasm.  In cells infected with a recombinant vaccinia virus expressing HCV structural proteins (core, E1, and E2), DDX3 and core colocalized in distinct spots in the perinuclear region of the cytoplasm.  The regions of the proteins involved in binding were found by deletion anal. to be the N-terminal 59 amino acid residues of core and a C-terminal RS-like domain of DDX3.  The human DDX3 is a putative RNA helicase and a member of a highly conserved DEAD box subclass that includes murine PL10, Xenopus An3, and yeast Ded1 proteins.  Their role in RNA metab. or gene expression is unknown.  The significance of core-helicase interaction in HCV replication and pathogenesis is discussed.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaMXsdQhsa4bVg90H21EOLACvtfcHk0ljJVjqj0Cnn_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXislWrt78%253D&md5=f9f73f66232a90557c53e6f6e23f6aca</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1006%2Fviro.1999.9659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fviro.1999.9659%26sid%3Dliteratum%253Aachs%26aulast%3DOwsianka%26aufirst%3DA.%2BM.%26aulast%3DPatel%26aufirst%3DA.%2BH.%26atitle%3DHepatitis%2520C%2520virus%2520core%2520protein%2520interacts%2520with%2520a%2520human%2520DEAD%2520box%2520protein%2520DDX3%26jtitle%3DVirology%26date%3D1999%26volume%3D257%26spage%3D330%26epage%3D340%26doi%3D10.1006%2Fviro.1999.9659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mamiya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worman, H. J.</span></span> <span> </span><span class="NLM_article-title">Hepatitis C virus core protein binds to a DEAD box RNA helicase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">15751</span>â <span class="NLM_lpage">15756</span>, <span class="refDoi">Â DOI: 10.1074/jbc.274.22.15751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1074%2Fjbc.274.22.15751" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=15751-15756&author=N.+Mamiyaauthor=H.+J.+Worman&title=Hepatitis+C+virus+core+protein+binds+to+a+DEAD+box+RNA+helicase&doi=10.1074%2Fjbc.274.22.15751"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.22.15751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.22.15751%26sid%3Dliteratum%253Aachs%26aulast%3DMamiya%26aufirst%3DN.%26aulast%3DWorman%26aufirst%3DH.%2BJ.%26atitle%3DHepatitis%2520C%2520virus%2520core%2520protein%2520binds%2520to%2520a%2520DEAD%2520box%2520RNA%2520helicase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D15751%26epage%3D15756%26doi%3D10.1074%2Fjbc.274.22.15751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dansako, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span> <span> </span><span class="NLM_article-title">DDX3 DEAD-box RNA helicase is required for hepatitis C virus RNA replication</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">13922</span>â <span class="NLM_lpage">13926</span>, <span class="refDoi">Â DOI: 10.1128/JVI.01517-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1128%2FJVI.01517-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=17855521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsValsbfK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=13922-13926&author=Y.+Ariumiauthor=M.+Kurokiauthor=K.-i.+Abeauthor=H.+Dansakoauthor=M.+Ikedaauthor=T.+Wakitaauthor=N.+Kato&title=DDX3+DEAD-box+RNA+helicase+is+required+for+hepatitis+C+virus+RNA+replication&doi=10.1128%2FJVI.01517-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DDX3 DEAD-box RNA helicase is required for hepatitis C virus RNA replication</span></div><div class="casAuthors">Ariumi, Yasuo; Kuroki, Misao; Abe, Ken-ichi; Dansako, Hiromichi; Ikeda, Masanori; Wakita, Takaji; Kato, Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">13922-13926</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DDX3, a DEAD-box RNA helicase, binds to the hepatitis C virus (HCV) core protein.  However, the role(s) of DDX3 in HCV replication is still not understood.  Here the authors show that the accumulation of both genome-length HCV RNA (HCV-O, genotype 1b) and its replicon RNA were significantly suppressed in HuH-7-derived cells expressing short hairpin RNA targeted to DDX3 by lentivirus vector transduction.  As well, RNA replication of JFH1 (genotype 2a) and release of the core into the culture supernatants were suppressed in DDX3 knockdown cells after inoculation of the cell culture-generated HCVcc.  Thus, DDX3 is required for HCV RNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCK8FCrndFrrVg90H21EOLACvtfcHk0lg4PjUuYz903g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsValsbfK&md5=d3f569c960c7df4814b51c596b6b4b2c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FJVI.01517-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01517-07%26sid%3Dliteratum%253Aachs%26aulast%3DAriumi%26aufirst%3DY.%26aulast%3DKuroki%26aufirst%3DM.%26aulast%3DAbe%26aufirst%3DK.-i.%26aulast%3DDansako%26aufirst%3DH.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DWakita%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DN.%26atitle%3DDDX3%2520DEAD-box%2520RNA%2520helicase%2520is%2520required%2520for%2520hepatitis%2520C%2520virus%2520RNA%2520replication%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D13922%26epage%3D13926%26doi%3D10.1128%2FJVI.01517-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishag, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span> <span> </span><span class="NLM_article-title">Cellular DDX3 regulates Japanese encephalitis virus replication by interacting with viral un-translated regions</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">70</span>â <span class="NLM_lpage">81</span>, <span class="refDoi">Â DOI: 10.1016/j.virol.2013.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.virol.2013.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=24418539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1ehtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2014&pages=70-81&author=C.+Liauthor=L.+L.+Geauthor=P.+P.+Liauthor=Y.+Wangauthor=J.+J.+Daiauthor=M.+X.+Sunauthor=L.+Huangauthor=Z.+Q.+Shenauthor=X.+C.+Huauthor=H.+Ishagauthor=Z.+Mao&title=Cellular+DDX3+regulates+Japanese+encephalitis+virus+replication+by+interacting+with+viral+un-translated+regions&doi=10.1016%2Fj.virol.2013.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular DDX3 regulates Japanese encephalitis virus replication by interacting with viral un-translated regions</span></div><div class="casAuthors">Li, Chen; Ge, Ling-ling; Li, Peng-peng; Wang, Yue; Dai, Juan-juan; Sun, Ming-xia; Huang, Li; Shen, Zhi-qiang; Hu, Xiao-chun; Ishag, Hassan; Mao, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">70-81</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Japanese encephalitis virus is one of the most common causes for epidemic viral encephalitis in humans and animals.  Herein we demonstrated that cellular helicase DDX3 is involved in JEV replication.  DDX3 knockdown inhibits JEV replication.  The helicase activity of DDX3 is crucial for JEV replication.  GST-pulldown and co-immunopptn. expts. demonstrated that DDX3 could interact with JEV non-structural proteins 3 and 5.  Co-immunopptn. and confocal microscopy anal. confirmed that DDX3 interacts and colocalizes with these viral proteins and viral RNA during the infection.  We detd. that DDX3 binds to JEV 5' and 3' un-translated regions.  We used a JEV-replicon system to demonstrate that DDX3 pos. regulates viral RNA translation, which might affect viral RNA replication at the late stage of virus infection.  Collectively, we identified that DDX3 is necessary for JEV infection, suggesting that DDX3 might be a novel target to design new antiviral agents against JEV or other flavivirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWB6dSzUY6LVg90H21EOLACvtfcHk0lg4PjUuYz903g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1ehtQ%253D%253D&md5=5593f579761a879895279c4082160c6e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2013.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2013.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGe%26aufirst%3DL.%2BL.%26aulast%3DLi%26aufirst%3DP.%2BP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DJ.%2BJ.%26aulast%3DSun%26aufirst%3DM.%2BX.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DZ.%2BQ.%26aulast%3DHu%26aufirst%3DX.%2BC.%26aulast%3DIshag%26aufirst%3DH.%26aulast%3DMao%26aufirst%3DZ.%26atitle%3DCellular%2520DDX3%2520regulates%2520Japanese%2520encephalitis%2520virus%2520replication%2520by%2520interacting%2520with%2520viral%2520un-translated%2520regions%26jtitle%3DVirology%26date%3D2014%26volume%3D449%26spage%3D70%26epage%3D81%26doi%3D10.1016%2Fj.virol.2013.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schul, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P. Y.</span></span> <span> </span><span class="NLM_article-title">Strategies for development of Dengue virus inhibitors</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">450</span>â <span class="NLM_lpage">462</span>, <span class="refDoi">Â DOI: 10.1016/j.antiviral.2009.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.antiviral.2009.12.011" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2010&pages=450-462&author=C.+G.+Nobleauthor=Y.+L.+Chenauthor=H.+Dongauthor=F.+Guauthor=S.+P.+Limauthor=W.+Schulauthor=Q.+Y.+Wangauthor=P.+Y.+Shi&title=Strategies+for+development+of+Dengue+virus+inhibitors&doi=10.1016%2Fj.antiviral.2009.12.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2009.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2009.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DC.%2BG.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DF.%26aulast%3DLim%26aufirst%3DS.%2BP.%26aulast%3DSchul%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DQ.%2BY.%26aulast%3DShi%26aufirst%3DP.%2BY.%26atitle%3DStrategies%2520for%2520development%2520of%2520Dengue%2520virus%2520inhibitors%26jtitle%3DAntiviral%2520Res.%26date%3D2010%26volume%3D85%26spage%3D450%26epage%3D462%26doi%3D10.1016%2Fj.antiviral.2009.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benfield, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahsoun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. L.</span></span> <span> </span><span class="NLM_article-title">Vaccinia virus protein K7 is a virulence factor that alters the acute immune response to infection</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1647</span>â <span class="NLM_lpage">1657</span>, <span class="refDoi">Â DOI: 10.1099/vir.0.052670-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1099%2Fvir.0.052670-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=23580427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWltL3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2013&pages=1647-1657&author=C.+T.+Benfieldauthor=H.+Renauthor=S.+J.+Lucasauthor=B.+Bahsounauthor=G.+L.+Smith&title=Vaccinia+virus+protein+K7+is+a+virulence+factor+that+alters+the+acute+immune+response+to+infection&doi=10.1099%2Fvir.0.052670-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Vaccinia virus protein K7 is a virulence factor that alters the acute immune response to infection</span></div><div class="casAuthors">Benfield, Camilla T. O.; Ren, Hongwei; Lucas, Stuart J.; Bahsoun, Basma; Smith, Geoffrey L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1647-1657</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">Vaccinia virus (VACV) encodes many proteins that antagonize the innate immune system including a family of intracellular proteins with a B-cell lymphoma (Bcl)-2-like structure.  One of these Bcl-2 proteins called K7 binds Toll-like receptor-adaptor proteins and the DEAD-box RNA helicase DDX3 and thereby inhibits the activation of NF-ÎºB and interferon regulatory factor 3.  However, the contribution of K7 to virus virulence is not known.  Here a VACV lacking the K7R gene (vÎK7) was constructed and compared with control viruses that included a plaque purified wt (vK7), a revertant with the K7R gene reinserted (vK7-rev) and a frame-shifted virus in which the translational initiation codon was mutated to prevent K7 protein expression (vK7-fs).  Data presented show that loss of K7 does not affect virus replication in cell culture or in vivo; however, viruses lacking the K7 protein were less virulent than controls in murine intradermal (i.d.) and intranasal (i.n.) infection models and there was an altered acute immune response to infection.  In the i.d. model, vÎK7 induced smaller lesions than controls, and after i.n. infection vÎK7 induced a reduced wt. loss and signs of illness, and more rapid clearance of virus from infected tissue.  Concomitantly, the intrapulmonary innate immune response to infection with vÎK7 showed increased infiltration of NK cells and CD8+ T-cells, enhanced MHC class II expression by macrophages, and enhanced cytolysis of target cells by NK cells and VACV-specific CD8+ T-cells.  Thus protein K7 is a virulence factor that affects the acute immune response to infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNukdrK617NbVg90H21EOLACvtfcHk0liQEBAw7PEiww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWltL3P&md5=09b687b0b56e6059bcce7313a3f1bc47</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1099%2Fvir.0.052670-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fvir.0.052670-0%26sid%3Dliteratum%253Aachs%26aulast%3DBenfield%26aufirst%3DC.%2BT.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DLucas%26aufirst%3DS.%2BJ.%26aulast%3DBahsoun%26aufirst%3DB.%26aulast%3DSmith%26aufirst%3DG.%2BL.%26atitle%3DVaccinia%2520virus%2520protein%2520K7%2520is%2520a%2520virulence%2520factor%2520that%2520alters%2520the%2520acute%2520immune%2520response%2520to%2520infection%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2013%26volume%3D94%26spage%3D1647%26epage%3D1657%26doi%3D10.1099%2Fvir.0.052670-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallacara, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Protein-protein interactions and human cellular cofactors as new targets for HIV therapy</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">8</span>, <span class="refDoi">Â DOI: 10.1016/j.coph.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.coph.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=24993074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKmtL%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=1-8&author=C.+Tintoriauthor=A.+Braiauthor=A.+L.+Fallacaraauthor=R.+Faziauthor=S.+Schenoneauthor=M.+Botta&title=Protein-protein+interactions+and+human+cellular+cofactors+as+new+targets+for+HIV+therapy&doi=10.1016%2Fj.coph.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interactions and human cellular cofactors as new targets for HIV therapy</span></div><div class="casAuthors">Tintori, Cristina; Brai, Annalaura; Fallacara, Anna Lucia; Fazi, Roberta; Schenone, Silvia; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Two novel approaches for the development of new drugs against AIDS are summarized each leading to the achievement of important discoveries in anti-HIV therapy.  Despite the success of HAART in reducing mortality, resistant strains continue to emerge in the clinic, underscoring the importance of developing next-generation drugs.  Protein-protein interactions and human cellular cofactors represent the new targets of tomorrow in HIV research.  The most relevant results obtained in the last few years by the two new strategies are described herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfeMocO2GFdLVg90H21EOLACvtfcHk0liQEBAw7PEiww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKmtL%252FK&md5=c00f92556fbf3aa779219b5655ea5ceb</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DFallacara%26aufirst%3DA.%2BL.%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DProtein-protein%2520interactions%2520and%2520human%2520cellular%2520cofactors%2520as%2520new%2520targets%2520for%2520HIV%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2014%26volume%3D18%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.coph.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldanti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of the first small molecule inhibitor of human DDX3 specifically designed to target the RNA binding site: towards the next generation HIV-1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2094</span>â <span class="NLM_lpage">2098</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2011.12.135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.bmcl.2011.12.135" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2094-2098&author=M.+Radiauthor=F.+Falchiauthor=A.+Garbelliauthor=A.+Samueleauthor=V.+Bernardoauthor=S.+Paolucciauthor=F.+Baldantiauthor=S.+Schenoneauthor=F.+Manettiauthor=G.+Magaauthor=M.+Botta&title=Discovery+of+the+first+small+molecule+inhibitor+of+human+DDX3+specifically+designed+to+target+the+RNA+binding+site%3A+towards+the+next+generation+HIV-1+inhibitors&doi=10.1016%2Fj.bmcl.2011.12.135"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.12.135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.12.135%26sid%3Dliteratum%253Aachs%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DSamuele%26aufirst%3DA.%26aulast%3DBernardo%26aufirst%3DV.%26aulast%3DPaolucci%26aufirst%3DS.%26aulast%3DBaldanti%26aufirst%3DF.%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DManetti%26aufirst%3DF.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520the%2520first%2520small%2520molecule%2520inhibitor%2520of%2520human%2520DDX3%2520specifically%2520designed%2520to%2520target%2520the%2520RNA%2520binding%2520site%253A%2520towards%2520the%2520next%2520generation%2520HIV-1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2094%26epage%3D2098%26doi%3D10.1016%2Fj.bmcl.2011.12.135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamperini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stigliano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">EstÃ©, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerhans, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saladini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zazzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Human DDX3 protein is a valuable target to develop broad spectrum antiviral agents</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">5388</span>â <span class="NLM_lpage">5393</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1522987113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1073%2Fpnas.1522987113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=27118832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslKrtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=5388-5393&author=A.+Braiauthor=R.+Faziauthor=C.+Tintoriauthor=C.+Zamperiniauthor=F.+Bugliauthor=M.+Sanguinettiauthor=E.+Stiglianoauthor=J.+Est%C3%A9author=R.+Badiaauthor=S.+Francoauthor=M.+A.+Martinezauthor=J.+P.+Martinezauthor=A.+Meyerhansauthor=F.+Saladiniauthor=M.+Zazziauthor=A.+Garbelliauthor=G.+Magaauthor=M.+Botta&title=Human+DDX3+protein+is+a+valuable+target+to+develop+broad+spectrum+antiviral+agents&doi=10.1073%2Fpnas.1522987113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Human DDX3 protein is a valuable target to develop broad spectrum antiviral agents</span></div><div class="casAuthors">Brai, Annalaura; Fazi, Roberta; Tintori, Cristina; Zamperini, Claudio; Bugli, Francesca; Sanguinetti, Maurizio; Stigliano, Egidio; Este, Jose; Badia, Roger; Franco, Sandra; Martinez, Miguel A.; Martinez, Javier P.; Meyerhans, Andreas; Saladini, Francesco; Zazzi, Maurizio; Garbelli, Anna; Maga, Giovanni; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5388-5393</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Targeting a host factor essential for the replication of different viruses but not for the cells offers a higher genetic barrier to the development of resistance, may simplify therapy regimens for coinfections, and facilitates management of emerging viral diseases.  DEAD-box polypeptide 3 (DDX3) is a human host factor required for the replication of several DNA and RNA viruses, including some of the most challenging human pathogens currently circulating, such as HIV-1, Hepatitis C virus, Dengue virus, and West Nile virus.  Herein, the authors showed for the first time, to the authors' knowledge, that the inhibition of DDX3 by a small mol. could be successfully exploited for the development of a broad spectrum antiviral agent.  In addn. to the multiple antiviral activities, hit compd. (I) retained full activity against drug-resistant HIV-1 strains in the absence of cellular toxicity.  Pharmacokinetics and toxicity studies in rats confirmed a good safety profile and bioavailability of (I).  Thus, DDX3 is here validated as a valuable therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosRPcNHvwVBrVg90H21EOLACvtfcHk0liQEBAw7PEiww"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslKrtrg%253D&md5=d619c77e8edf9eaeb492993cb0964aab</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1522987113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1522987113%26sid%3Dliteratum%253Aachs%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DZamperini%26aufirst%3DC.%26aulast%3DBugli%26aufirst%3DF.%26aulast%3DSanguinetti%26aufirst%3DM.%26aulast%3DStigliano%26aufirst%3DE.%26aulast%3DEst%25C3%25A9%26aufirst%3DJ.%26aulast%3DBadia%26aufirst%3DR.%26aulast%3DFranco%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DM.%2BA.%26aulast%3DMartinez%26aufirst%3DJ.%2BP.%26aulast%3DMeyerhans%26aufirst%3DA.%26aulast%3DSaladini%26aufirst%3DF.%26aulast%3DZazzi%26aufirst%3DM.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DHuman%2520DDX3%2520protein%2520is%2520a%2520valuable%2520target%2520to%2520develop%2520broad%2520spectrum%2520antiviral%2520agents%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D5388%26epage%3D5393%26doi%3D10.1073%2Fpnas.1522987113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fazi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraj, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavagnoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Homology model-based virtual screening for the identification of helicase human DDX3X inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2443</span>â <span class="NLM_lpage">2454</span>, <span class="refDoi">Â DOI: 10.1021/acs.jcim.5b00419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2443-2454&author=R.+Faziauthor=C.+Tintoriauthor=A.+Braiauthor=M.+K.+Selvarajauthor=L.+Bavagnoliauthor=A.+Garbelliauthor=G.+Magaauthor=M.+Botta&title=Homology+model-based+virtual+screening+for+the+identification+of+helicase+human+DDX3X+inhibitors&doi=10.1021%2Facs.jcim.5b00419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Homology Model-Based Virtual Screening for the Identification of Human Helicase DDX3 Inhibitors</span></div><div class="casAuthors">Fazi, Roberta; Tintori, Cristina; Brai, Annalaura; Botta, Lorenzo; Selvaraj, Manikandan; Garbelli, Anna; Maga, Giovanni; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2443-2454</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting cellular cofactors instead of viral enzymes represents a new strategy to combat infectious diseases, which should help to overcome the problem of viral resistance.  Recently, it has been revealed that the cellular ATPase/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3) is an essential host factor for the replication of several viruses such as HIV, HCV, JEV, Dengue, and West Nile.  Accordingly, a drug targeting DDX3 could theor. inhibit all viruses that are dependent on this host factor.  Herein, for the first time, a model of hDDX3 in its closed conformation, which binds the viral RNA was developed by using the homol. module of Prime through the Maestro interface of Schrodinger.  Next, a structure-based virtual screening protocol was applied to identify DDX3 small mol. inhibitors targeting the RNA binding pocket.  As a result, an impressive hit rate of 40% was obtained with the identification of 10 active compds. out of the 25 tested small mols.  The best poses of the active ligands highlighted the crucial residues to be targeted for the inhibition of the helicase activity of DDX3.  The obtained results confirm the reliability of the constructed DDX3/RNA model and the proposed computational strategy for investigating novel DDX3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO1Sy7ZFr2kbVg90H21EOLACvtfcHk0liJtFxBrl5UEA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN&md5=0c739457f73969394c6a31c113617bb6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00419%26sid%3Dliteratum%253Aachs%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DSelvaraj%26aufirst%3DM.%2BK.%26aulast%3DBavagnoli%26aufirst%3DL.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DHomology%2520model-based%2520virtual%2520screening%2520for%2520the%2520identification%2520of%2520helicase%2520human%2520DDX3X%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2443%26epage%3D2454%26doi%3D10.1021%2Facs.jcim.5b00419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yedavalli, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starost, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosmane, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeang, K. T.</span></span> <span> </span><span class="NLM_article-title">Ring expanded nucleoside analogues inhibit RNA helicase and intracellular human immunodeficiency virus type 1 replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5043</span>â <span class="NLM_lpage">5051</span>, <span class="refDoi">Â DOI: 10.1021/jm800332m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800332m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5043-5051&author=V.+S.+Yedavalliauthor=N.+Zhangauthor=H.+Caiauthor=P.+Zhangauthor=M.+F.+Starostauthor=R.+S.+Hosmaneauthor=K.+T.+Jeang&title=Ring+expanded+nucleoside+analogues+inhibit+RNA+helicase+and+intracellular+human+immunodeficiency+virus+type+1+replication&doi=10.1021%2Fjm800332m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm800332m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800332m%26sid%3Dliteratum%253Aachs%26aulast%3DYedavalli%26aufirst%3DV.%2BS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DStarost%26aufirst%3DM.%2BF.%26aulast%3DHosmane%26aufirst%3DR.%2BS.%26aulast%3DJeang%26aufirst%3DK.%2BT.%26atitle%3DRing%2520expanded%2520nucleoside%2520analogues%2520inhibit%2520RNA%2520helicase%2520and%2520intracellular%2520human%2520immunodeficiency%2520virus%2520type%25201%2520replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5043%26epage%3D5051%26doi%3D10.1021%2Fjm800332m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yedavalli, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuveut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeang, K. T.</span></span> <span> </span><span class="NLM_article-title">Requirement of DDX3X DEAD box RNA helicase for HIV-1 Rev-RRE export function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">381</span>â <span class="NLM_lpage">392</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2004.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.cell.2004.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=15507209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvFOgtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=381-392&author=V.+S.+Yedavalliauthor=C.+Neuveutauthor=Y.+H.+Chiauthor=L.+Kleimanauthor=K.+T.+Jeang&title=Requirement+of+DDX3X+DEAD+box+RNA+helicase+for+HIV-1+Rev-RRE+export+function&doi=10.1016%2Fj.cell.2004.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement of DDX3 DEAD box RNA helicase for HIV-1 Rev-RRE export function</span></div><div class="casAuthors">Yedavalli, Venkat S. R. K.; Neuveut, Christine; Chi, Ya-hui; Kleiman, Lawrence; Jeang, Kuan-Teh</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-392</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A single transcript in its unspliced and spliced forms directs the synthesis of all HIV-1 proteins.  Although nuclear export of intron-contg. cellular transcripts is restricted in mammalian cells, HIV-1 has evolved the viral Rev protein to overcome this restriction for viral transcripts.  Previously, CRM1 was identified as a cellular cofactor for Rev-dependent export of intron-contg. HIV-1 RNA.  Here, we present evidence that Rev/CRM1 activity utilizes the ATP-dependent DEAD box RNA helicase, DDX3.  We show that DDX3 is a nucleo-cytoplasmic shuttling protein, which binds CRM1 and localizes to nuclear membrane pores.  Knockdown of DDX3 using either antisense vector or dominant-neg. mutants suppressed Rev-RRE-function in the export of incompletely spliced HIV-1 RNAs.  Plausibly, DDX3 is the human RNA helicase which functions in the CRM1 RNA export pathway analogously to the postulated role for Dbp5p in yeast mRNA export.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreblW84ZWZULVg90H21EOLACvtfcHk0liJtFxBrl5UEA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvFOgtbs%253D&md5=165d2abcfc244ec9dd51b09aa972b94a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DYedavalli%26aufirst%3DV.%2BS.%26aulast%3DNeuveut%26aufirst%3DC.%26aulast%3DChi%26aufirst%3DY.%2BH.%26aulast%3DKleiman%26aufirst%3DL.%26aulast%3DJeang%26aufirst%3DK.%2BT.%26atitle%3DRequirement%2520of%2520DDX3X%2520DEAD%2520box%2520RNA%2520helicase%2520for%2520HIV-1%2520Rev-RRE%2520export%2520function%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D381%26epage%3D392%26doi%3D10.1016%2Fj.cell.2004.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhari, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogurampelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suryaprakash, N.</span></span> <span> </span><span class="NLM_article-title">Engagement of CF<sub>3</sub> group in N-HÂ·Â·Â·F-C hydrogen bond in the solution state: NMR spectroscopy and MD simulation studies</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1123</span>â <span class="NLM_lpage">1129</span>, <span class="refDoi">Â DOI: 10.1021/jp310798d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp310798d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=1123-1129&author=S.+R.+Chaudhariauthor=S.+Mogurampellyauthor=N.+Suryaprakash&title=Engagement+of+CF3+group+in+N-H%C2%B7%C2%B7%C2%B7F-C+hydrogen+bond+in+the+solution+state%3A+NMR+spectroscopy+and+MD+simulation+studies&doi=10.1021%2Fjp310798d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjp310798d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp310798d%26sid%3Dliteratum%253Aachs%26aulast%3DChaudhari%26aufirst%3DS.%2BR.%26aulast%3DMogurampelly%26aufirst%3DS.%26aulast%3DSuryaprakash%26aufirst%3DN.%26atitle%3DEngagement%2520of%2520CF3%2520group%2520in%2520N-H%25C2%25B7%25C2%25B7%25C2%25B7F-C%2520hydrogen%2520bond%2520in%2520the%2520solution%2520state%253A%2520NMR%2520spectroscopy%2520and%2520MD%2520simulation%2520studies%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D1123%26epage%3D1129%26doi%3D10.1021%2Fjp310798d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bol, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vesuna, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aziz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantravedi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerma van Voss, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordt, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polster, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cope, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Groep, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondaskar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosmane, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wall, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Diest, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, V.</span></span> <span> </span><span class="NLM_article-title">Targeting DDX3 with a small molecule inhibitor for lung cancer therapy</span>. <i>EMBO Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">648</span>â <span class="NLM_lpage">669</span>, <span class="refDoi">Â DOI: 10.15252/emmm.201404368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.15252%2Femmm.201404368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=25820276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVWktLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=648-669&author=G.+M.+Bolauthor=F.+Vesunaauthor=M.+Xieauthor=J.+Zengauthor=K.+Azizauthor=N.+Gandhiauthor=A.+Levineauthor=A.+Irvingauthor=D.+Korzauthor=S.+Tantravediauthor=M.+R.+Heerma+van+Vossauthor=K.+Gabrielsonauthor=E.+A.+Bordtauthor=B.+M.+Polsterauthor=L.+Copeauthor=P.+van+der%0AGroepauthor=A.+Kondaskarauthor=M.+A.+Rudekauthor=R.+S.+Hosmaneauthor=E.+van+der+Wallauthor=P.+J.+van+Diestauthor=P.+T.+Tranauthor=V.+Raman&title=Targeting+DDX3+with+a+small+molecule+inhibitor+for+lung+cancer+therapy&doi=10.15252%2Femmm.201404368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DDX3 with a small molecule inhibitor for lung cancer therapy</span></div><div class="casAuthors">Bol, Guus M.; Vesuna, Farhad; Xie, Min; Zeng, Jing; Aziz, Khaled; Gandhi, Nishant; Levine, Anne; Irving, Ashley; Korz, Dorian; Tantravedi, Saritha; Heerma van Voss, Marise R.; Gabrielson, Kathleen; Bordt, Evan A.; Polster, Brian M.; Cope, Leslie; van der Groep, Petra; Kondaskar, Atul; Rudek, Michelle A.; Hosmane, Ramachandra S.; van der Wall, Elsken; van Diest, Paul J.; Tran, Phuoc T.; Raman, Venu</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">648-669</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Lung cancer is the most common malignancy worldwide and is a focus for developing targeted therapies due to its refractory nature to current treatment.  We identified a RNA helicase, DDX3, which is overexpressed in many cancer types including lung cancer and is assocd. with lower survival in lung cancer patients.  We designed a first-in-class small mol. inhibitor, RK-33, which binds to DDX3 and abrogates its activity.  Inhibition of DDX3 by RK-33 caused G1 cell cycle arrest, induced apoptosis, and promoted radiation sensitization in DDX3-overexpressing cells.  Importantly, RK-33 in combination with radiation induced tumor regression in multiple mouse models of lung cancer.  Mechanistically, loss of DDX3 function either by shRNA or by RK-33 impaired Wnt signaling through disruption of the DDX3-Î²-catenin axis and inhibited non-homologous end joining-the major DNA repair pathway in mammalian somatic cells.  Overall, inhibition of DDX3 by RK-33 promotes tumor regression, thus providing a compelling argument to develop DDX3 inhibitors for lung cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOehktsrE8jLVg90H21EOLACvtfcHk0lj4Rwl6FyW-AQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVWktLY%253D&md5=149e69c6868fe66c4ae40619b528dbd1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.15252%2Femmm.201404368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Femmm.201404368%26sid%3Dliteratum%253Aachs%26aulast%3DBol%26aufirst%3DG.%2BM.%26aulast%3DVesuna%26aufirst%3DF.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DAziz%26aufirst%3DK.%26aulast%3DGandhi%26aufirst%3DN.%26aulast%3DLevine%26aufirst%3DA.%26aulast%3DIrving%26aufirst%3DA.%26aulast%3DKorz%26aufirst%3DD.%26aulast%3DTantravedi%26aufirst%3DS.%26aulast%3DHeerma%2Bvan%2BVoss%26aufirst%3DM.%2BR.%26aulast%3DGabrielson%26aufirst%3DK.%26aulast%3DBordt%26aufirst%3DE.%2BA.%26aulast%3DPolster%26aufirst%3DB.%2BM.%26aulast%3DCope%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BGroep%26aufirst%3DP.%26aulast%3DKondaskar%26aufirst%3DA.%26aulast%3DRudek%26aufirst%3DM.%2BA.%26aulast%3DHosmane%26aufirst%3DR.%2BS.%26aulast%3Dvan%2Bder%2BWall%26aufirst%3DE.%26aulast%3Dvan%2BDiest%26aufirst%3DP.%2BJ.%26aulast%3DTran%26aufirst%3DP.%2BT.%26aulast%3DRaman%26aufirst%3DV.%26atitle%3DTargeting%2520DDX3%2520with%2520a%2520small%2520molecule%2520inhibitor%2520for%2520lung%2520cancer%2520therapy%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2015%26volume%3D7%26spage%3D648%26epage%3D669%26doi%3D10.15252%2Femmm.201404368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellerbrok, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedrig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauli, G.</span></span> <span> </span><span class="NLM_article-title">Detection of West Nile virus lineages 1 and 2 by real-time PCR</span>. <i>J. Virol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">355</span>â <span class="NLM_lpage">358</span>, <span class="refDoi">Â DOI: 10.1016/j.jviromet.2007.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.jviromet.2007.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=17604132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1KrsLrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2007&pages=355-358&author=S.+Linkeauthor=H.+Ellerbrokauthor=M.+Niedrigauthor=A.+Nitscheauthor=G.+Pauli&title=Detection+of+West+Nile+virus+lineages+1+and+2+by+real-time+PCR&doi=10.1016%2Fj.jviromet.2007.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of West Nile virus lineages 1 and 2 by real-time PCR</span></div><div class="casAuthors">Linke, Sonja; Ellerbrok, Heinz; Niedrig, Matthias; Nitsche, Andreas; Pauli, Georg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virological Methods</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">355-358</span>CODEN:
                <span class="NLM_cas:coden">JVMEDH</span>;
        ISSN:<span class="NLM_cas:issn">0166-0934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">West Nile virus (WNV) is a Flavivirus attracting worldwide attention because it has spread rapidly across the Americas since its first appearance in New York City in 1999.  Several PCR-based diagnostic methods have been developed for the detection of WNV.  The focus of these assays has been WNV lineage 1 which can be found worldwide, while lineage 2 viruses were thought to be endemic only in some regions of Africa.  However, both lineages may be imported from Africa to Europe by migrating birds.  In order to det. the incidence of WNV in Germany, a real-time-based PCR assay was developed, targeting a conserved region of WNV lineages 1 and 2.  This assay is a suitable tool for the diagnosis of WNV and for surveillance studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJNzshDNwNGrVg90H21EOLACvtfcHk0lj4Rwl6FyW-AQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1KrsLrL&md5=f90bcf395a82a8284ab79134549b219a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.jviromet.2007.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jviromet.2007.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DLinke%26aufirst%3DS.%26aulast%3DEllerbrok%26aufirst%3DH.%26aulast%3DNiedrig%26aufirst%3DM.%26aulast%3DNitsche%26aufirst%3DA.%26aulast%3DPauli%26aufirst%3DG.%26atitle%3DDetection%2520of%2520West%2520Nile%2520virus%2520lineages%25201%2520and%25202%2520by%2520real-time%2520PCR%26jtitle%3DJ.%2520Virol.%2520Methods%26date%3D2007%26volume%3D146%26spage%3D355%26epage%3D358%26doi%3D10.1016%2Fj.jviromet.2007.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved protein-ligand docking using GOLD</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>â <span class="NLM_lpage">623</span>, <span class="refDoi">Â DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+protein-ligand+docking+using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Ã of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of â¼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0lj17s7jz2j5hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520protein-ligand%2520docking%2520using%2520GOLD%26jtitle%3DProteins%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span>; <span class="NLM_string-name">Nissink, J. W. M.</span>; <span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">ProteinâLigand Docking and Virtual Screening with GOLD</span>. In  <i>Virtual Screening in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span>, <span class="NLM_string-name">Alvarez, J.</span></span>, Eds.; <span class="NLM_publisher-name">Taylor & Francis CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, Florida</span>, <span class="NLM_year">2005</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1201%2F9781420028775.ch15" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=J.+C.+Cole&author=J.+W.+M.+Nissink&author=R.+Taylorauthor=B.+Shoichet&author=J.+Alvarez&title=Virtual+Screening+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1201%2F9781420028775.ch15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1201%252F9781420028775.ch15%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DJ.%2BC.%26atitle%3DProtein%25E2%2580%2593Ligand%2520Docking%2520and%2520Virtual%2520Screening%2520with%2520GOLD%26btitle%3DVirtual%2520Screening%2520in%2520Drug%2520Discovery%26aulast%3DShoichet%26aufirst%3DB.%26pub%3DTaylor%2520%2526%2520Francis%2520CRC%2520Press%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span> <i>PyMOL Molecular Graphics System</i>, version 0.99; <span class="NLM_publisher-name">SchrÃ¶dinger, LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PyMOL+Molecular+Graphics+System%2C+version+0.99%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaluce, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irannejad, H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Toward the discovery of novel anti-HIV drugs. Second-generation inhibitors of the cellular ATPase DDX3 with improved anti-HIV activity: synthesis, structure-activity relationship analysis, cytotoxicity studies, and target validation</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1371</span>â <span class="NLM_lpage">1389</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201100166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1002%2Fcmdc.201100166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=21698775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsl2gtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1371-1389&author=G.+Magaauthor=F.+Falchiauthor=M.+Radiauthor=L.+Bottaauthor=G.+Casaluceauthor=M.+Bernardiniauthor=H.+Irannejad&title=Toward+the+discovery+of+novel+anti-HIV+drugs.+Second-generation+inhibitors+of+the+cellular+ATPase+DDX3+with+improved+anti-HIV+activity%3A+synthesis%2C+structure-activity+relationship+analysis%2C+cytotoxicity+studies%2C+and+target+validation&doi=10.1002%2Fcmdc.201100166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Discovery of Novel Anti-HIV Drugs. Second-Generation Inhibitors of the Cellular ATPase DDX3 with Improved Anti-HIV Activity: Synthesis, Structure-Activity Relationship Analysis, Cytotoxicity Studies, and Target Validation</span></div><div class="casAuthors">Maga, Giovanni; Falchi, Federico; Radi, Marco; Botta, Lorenzo; Casaluce, Gianni; Bernardini, Martina; Irannejad, Hamid; Manetti, Fabrizio; Garbelli, Anna; Samuele, Alberta; Zanoli, Samantha; Este, Jose A.; Gonzalez, Emmanuel; Zucca, Elisa; Paolucci, Stefania; Baldanti, Fausto; De Rijck, Jan; Debyser, Zeger; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1371-1389</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A hit optimization protocol applied to the first nonnucleoside inhibitor of the ATPase activity of human DEAD-box RNA helicase DDX3 led to the design and synthesis of second-generation rhodanine derivs. with better inhibitory activity toward cellular DDX3 and HIV-1 replication.  Addnl. DDX3 inhibitors were identified among triazine compds.  Biol. data were rationalized in terms of structure-activity relationships and docking simulations.  Antiviral activity and cytotoxicity of selected DDX3 inhibitors are reported and discussed.  A thorough anal. confirmed human DDX3 as a valid anti-HIV target.  The compds. described herein represent a significant advance in the pursuit of novel drugs that target HIV-1 host cofactors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAxJL-NB7IFbVg90H21EOLACvtfcHk0lj17s7jz2j5hw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsl2gtrg%253D&md5=6316aa57e7a2cb2ef5fdb676bc2ebdc7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100166%26sid%3Dliteratum%253Aachs%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DBotta%26aufirst%3DL.%26aulast%3DCasaluce%26aufirst%3DG.%26aulast%3DBernardini%26aufirst%3DM.%26aulast%3DIrannejad%26aufirst%3DH.%26atitle%3DToward%2520the%2520discovery%2520of%2520novel%2520anti-HIV%2520drugs.%2520Second-generation%2520inhibitors%2520of%2520the%2520cellular%2520ATPase%2520DDX3%2520with%2520improved%2520anti-HIV%2520activity%253A%2520synthesis%252C%2520structure-activity%2520relationship%2520analysis%252C%2520cytotoxicity%2520studies%252C%2520and%2520target%2520validation%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D1371%26epage%3D1389%26doi%3D10.1002%2Fcmdc.201100166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Valentina Riva, Anna Garbelli, Annalaura Brai, Federica Casiraghi, Roberta Fazi, Claudia I. Trivisani, Adele Boccuto, Francesco Saladini, Ilaria Vicenti, Francesco Martelli, Maurizio Zazzi, Simone Giannecchini, Elena Dreassi, Maurizio Botta, <span class="NLM_string-name hlFld-ContribAuthor">Giovanni Maga</span>. </span><span class="cited-content_cbyCitation_article-title">Unique Domain for a Unique Target: Selective Inhibitors of Host Cell DDX3X to Fight Emerging Viruses. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9876-9887. <a href="https://doi.org/10.1021/acs.jmedchem.0c01039" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01039%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DUnique%252BDomain%252Bfor%252Ba%252BUnique%252BTarget%25253A%252BSelective%252BInhibitors%252Bof%252BHost%252BCell%252BDDX3X%252Bto%252BFight%252BEmerging%252BViruses%26aulast%3DRiva%26aufirst%3DValentina%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17062020%26date%3D18082020%26date%3D30072020%26volume%3D63%26issue%3D17%26spage%3D9876%26epage%3D9887" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Annalaura Brai, Adele Boccuto, Martina Monti, Serena Marchi, Ilaria Vicenti, Francesco Saladini, Claudia Immacolata Trivisani, Alessandro Pollutri, Claudia Maria Trombetta, Emanuele Montomoli, Valentina Riva, Anna Garbelli, Emanuele Maria Nola, Maurizio Zazzi, Giovanni Maga, Elena Dreassi, <span class="NLM_string-name hlFld-ContribAuthor">Maurizio Botta</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the Implication of DDX3X in DENV Infection: Discovery of the First-in-Class DDX3X Fluorescent Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 956-962. <a href="https://doi.org/10.1021/acsmedchemlett.9b00681" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00681%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DExploring%252Bthe%252BImplication%252Bof%252BDDX3X%252Bin%252BDENV%252BInfection%25253A%252BDiscovery%252Bof%252Bthe%252BFirst-in-Class%252BDDX3X%252BFluorescent%252BInhibitor%26aulast%3DBrai%26aufirst%3DAnnalaura%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D31122019%26date%3D02032020%26date%3D09042020%26volume%3D11%26issue%3D5%26spage%3D956%26epage%3D962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabiola  Ciccosanti</span>, <span class="hlFld-ContribAuthor ">Martina  Di Rienzo</span>, <span class="hlFld-ContribAuthor ">Alessandra  Romagnoli</span>, <span class="hlFld-ContribAuthor ">Francesca  Colavita</span>, <span class="hlFld-ContribAuthor ">Giulia  Refolo</span>, <span class="hlFld-ContribAuthor ">Concetta  Castilletti</span>, <span class="hlFld-ContribAuthor ">Chiara  Agrati</span>, <span class="hlFld-ContribAuthor ">Annalaura  Brai</span>, <span class="hlFld-ContribAuthor ">Fabrizio  Manetti</span>, <span class="hlFld-ContribAuthor ">Lorenzo  Botta</span>, <span class="hlFld-ContribAuthor ">Maria Rosaria  Capobianchi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ippolito</span>, <span class="hlFld-ContribAuthor ">Mauro  Piacentini</span>, <span class="hlFld-ContribAuthor ">Gian Maria  Fimia</span>. </span><span class="cited-content_cbyCitation_article-title">Proteomic analysis identifies the RNA helicase DDX3X as a host target against SARS-CoV-2 infection. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2021,</strong> <em>190 </em>, 105064. <a href="https://doi.org/10.1016/j.antiviral.2021.105064" title="DOI URL">https://doi.org/10.1016/j.antiviral.2021.105064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2021.105064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2021.105064%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DProteomic%252Banalysis%252Bidentifies%252Bthe%252BRNA%252Bhelicase%252BDDX3X%252Bas%252Ba%252Bhost%252Btarget%252Bagainst%252BSARS-CoV-2%252Binfection%26aulast%3DCiccosanti%26aufirst%3DFabiola%26date%3D2021%26volume%3D190%26spage%3D105064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">TomÃ¡s  HernÃ¡ndez-DÃ­az</span>, <span class="hlFld-ContribAuthor ">Fernando  Valiente-EcheverrÃ­a</span>, <span class="hlFld-ContribAuthor ">Ricardo  Soto-Rifo</span>. </span><span class="cited-content_cbyCitation_article-title">RNA Helicase DDX3: A Double-Edged Sword for Viral Replication and Immune Signaling. </span><span class="cited-content_cbyCitation_journal-name">Microorganisms</span><span> <strong>2021,</strong> <em>9 </em>
                                    (6)
                                     , 1206. <a href="https://doi.org/10.3390/microorganisms9061206" title="DOI URL">https://doi.org/10.3390/microorganisms9061206</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/microorganisms9061206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmicroorganisms9061206%26sid%3Dliteratum%253Aachs%26jtitle%3DMicroorganisms%26atitle%3DRNA%252BHelicase%252BDDX3%25253A%252BA%252BDouble-Edged%252BSword%252Bfor%252BViral%252BReplication%252Band%252BImmune%252BSignaling%26aulast%3DHern%25C3%25A1ndez-D%25C3%25ADaz%26aufirst%3DTom%25C3%25A1s%26date%3D2021%26date%3D2021%26volume%3D9%26issue%3D6%26spage%3D1206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lukasz  Markiewicz</span>, <span class="hlFld-ContribAuthor ">Karolina  Drazkowska</span>, <span class="hlFld-ContribAuthor ">Pawel J.  Sikorski</span>. </span><span class="cited-content_cbyCitation_article-title">Tricks and threats of RNA viruses â towards understanding the fate of viral RNA. </span><span class="cited-content_cbyCitation_journal-name">RNA Biology</span><span> <strong>2021,</strong> <em>18 </em>
                                    (5)
                                     , 669-687. <a href="https://doi.org/10.1080/15476286.2021.1875680" title="DOI URL">https://doi.org/10.1080/15476286.2021.1875680</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15476286.2021.1875680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15476286.2021.1875680%26sid%3Dliteratum%253Aachs%26jtitle%3DRNA%2520Biology%26atitle%3DTricks%252Band%252Bthreats%252Bof%252BRNA%252Bviruses%252B%2525E2%252580%252593%252Btowards%252Bunderstanding%252Bthe%252Bfate%252Bof%252Bviral%252BRNA%26aulast%3DMarkiewicz%26aufirst%3DLukasz%26date%3D2021%26date%3D2021%26volume%3D18%26issue%3D5%26spage%3D669%26epage%3D687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yue  Ma</span>, <span class="hlFld-ContribAuthor ">Estrella  Frutos-BeltrÃ¡n</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Christophe  Pannecouque</span>, <span class="hlFld-ContribAuthor ">Erik  De Clercq</span>, <span class="hlFld-ContribAuthor ">Luis  MenÃ©ndez-Arias</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Medicinal chemistry strategies for discovering antivirals effective against drug-resistant viruses. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2021,</strong> <em>50 </em>
                                    (7)
                                     , 4514-4540. <a href="https://doi.org/10.1039/D0CS01084G" title="DOI URL">https://doi.org/10.1039/D0CS01084G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS01084G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS01084G%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DMedicinal%252Bchemistry%252Bstrategies%252Bfor%252Bdiscovering%252Bantivirals%252Beffective%252Bagainst%252Bdrug-resistant%252Bviruses%26aulast%3DMa%26aufirst%3DYue%26date%3D2021%26date%3D2021%26volume%3D50%26issue%3D7%26spage%3D4514%26epage%3D4540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paul T.  Winnard</span>, <span class="hlFld-ContribAuthor ">Farhad  Vesuna</span>, <span class="hlFld-ContribAuthor ">Venu  Raman</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting host DEAD-box RNA helicase DDX3X for treating viral infections. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2021,</strong> <em>185 </em>, 104994. <a href="https://doi.org/10.1016/j.antiviral.2020.104994" title="DOI URL">https://doi.org/10.1016/j.antiviral.2020.104994</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2020.104994&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2020.104994%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DTargeting%252Bhost%252BDEAD-box%252BRNA%252Bhelicase%252BDDX3X%252Bfor%252Btreating%252Bviral%252Binfections%26aulast%3DWinnard%26aufirst%3DPaul%2BT.%26date%3D2021%26volume%3D185%26spage%3D104994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Corey  Nelson</span>, <span class="hlFld-ContribAuthor ">Tyler  Mrozowich</span>, <span class="hlFld-ContribAuthor ">Darren L.  Gemmill</span>, <span class="hlFld-ContribAuthor ">Sean M.  Park</span>, <span class="hlFld-ContribAuthor ">Trushar R.  Patel</span>. </span><span class="cited-content_cbyCitation_article-title">Human DDX3X Unwinds Japanese Encephalitis and Zika Viral 5â² Terminal Regions. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     , 413. <a href="https://doi.org/10.3390/ijms22010413" title="DOI URL">https://doi.org/10.3390/ijms22010413</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22010413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22010413%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DHuman%252BDDX3X%252BUnwinds%252BJapanese%252BEncephalitis%252Band%252BZika%252BViral%252B5%2525E2%252580%2525B2%252BTerminal%252BRegions%26aulast%3DNelson%26aufirst%3DCorey%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D1%26spage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tommaso  Felicetti</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Manfroni</span>, <span class="hlFld-ContribAuthor ">Violetta  Cecchetti</span>, <span class="hlFld-ContribAuthor ">Rolando  Cannalire</span>. </span><span class="cited-content_cbyCitation_article-title">BroadâSpectrum Flavivirus Inhibitors: a Medicinal Chemistry Point of View. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (24)
                                     , 2391-2419. <a href="https://doi.org/10.1002/cmdc.202000464" title="DOI URL">https://doi.org/10.1002/cmdc.202000464</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000464&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000464%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DBroad%2525E2%252580%252590Spectrum%252BFlavivirus%252BInhibitors%25253A%252Ba%252BMedicinal%252BChemistry%252BPoint%252Bof%252BView%26aulast%3DFelicetti%26aufirst%3DTommaso%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D24%26spage%3D2391%26epage%3D2419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Flavia  Squeglia</span>, <span class="hlFld-ContribAuthor ">Maria  Romano</span>, <span class="hlFld-ContribAuthor ">Alessia  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Giovanni  Maga</span>, <span class="hlFld-ContribAuthor ">Rita  Berisio</span>. </span><span class="cited-content_cbyCitation_article-title">Host DDX Helicases as Possible SARS-CoV-2 Proviral Factors: A Structural Overview of Their Hijacking Through Multiple Viral Proteins. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.602162" title="DOI URL">https://doi.org/10.3389/fchem.2020.602162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.602162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.602162%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DHost%252BDDX%252BHelicases%252Bas%252BPossible%252BSARS-CoV-2%252BProviral%252BFactors%25253A%252BA%252BStructural%252BOverview%252Bof%252BTheir%252BHijacking%252BThrough%252BMultiple%252BViral%252BProteins%26aulast%3DSqueglia%26aufirst%3DFlavia%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentina  Riva</span>, <span class="hlFld-ContribAuthor ">Anna  Garbelli</span>, <span class="hlFld-ContribAuthor ">Federica  Casiraghi</span>, <span class="hlFld-ContribAuthor ">Francesca  Arena</span>, <span class="hlFld-ContribAuthor ">ClaudiaÂ Immacolata  Trivisani</span>, <span class="hlFld-ContribAuthor ">Assunta  Gagliardi</span>, <span class="hlFld-ContribAuthor ">Luca  Bini</span>, <span class="hlFld-ContribAuthor ">Martina  Schroeder</span>, <span class="hlFld-ContribAuthor ">Antonio  Maffia</span>, <span class="hlFld-ContribAuthor ">Simone  Sabbioneda</span>, <span class="hlFld-ContribAuthor ">Giovanni  Maga</span>. </span><span class="cited-content_cbyCitation_article-title">Novel alternative ribonucleotide excision repair pathways in human cells by DDX3X and specialized DNA polymerases. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2020,</strong> <em>48 </em>
                                    (20)
                                     , 11551-11565. <a href="https://doi.org/10.1093/nar/gkaa948" title="DOI URL">https://doi.org/10.1093/nar/gkaa948</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa948%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DNovel%252Balternative%252Bribonucleotide%252Bexcision%252Brepair%252Bpathways%252Bin%252Bhuman%252Bcells%252Bby%252BDDX3X%252Band%252Bspecialized%252BDNA%252Bpolymerases%26aulast%3DRiva%26aufirst%3DValentina%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D20%26spage%3D11551%26epage%3D11565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bartolo  Bono</span>, <span class="hlFld-ContribAuthor ">Giulia  Franco</span>, <span class="hlFld-ContribAuthor ">Valentina  Riva</span>, <span class="hlFld-ContribAuthor ">Anna  Garbelli</span>, <span class="hlFld-ContribAuthor ">Giovanni  Maga</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Insights into the Biochemical Mechanism of CK1Îµ and its Functional Interplay with DDX3X. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (17)
                                     , 6449. <a href="https://doi.org/10.3390/ijms21176449" title="DOI URL">https://doi.org/10.3390/ijms21176449</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21176449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21176449%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DNovel%252BInsights%252Binto%252Bthe%252BBiochemical%252BMechanism%252Bof%252BCK1%2525CE%2525B5%252Band%252Bits%252BFunctional%252BInterplay%252Bwith%252BDDX3X%26aulast%3DBono%26aufirst%3DBartolo%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D17%26spage%3D6449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deanna M.  Patmore</span>, <span class="hlFld-ContribAuthor ">Amir  Jassim</span>, <span class="hlFld-ContribAuthor ">Erica  Nathan</span>, <span class="hlFld-ContribAuthor ">Reuben J.  Gilbertson</span>, <span class="hlFld-ContribAuthor ">Daniel  Tahan</span>, <span class="hlFld-ContribAuthor ">Nadin  Hoffmann</span>, <span class="hlFld-ContribAuthor ">Yiai  Tong</span>, <span class="hlFld-ContribAuthor ">Kyle S.  Smith</span>, <span class="hlFld-ContribAuthor ">Thirumala-Devi  Kanneganti</span>, <span class="hlFld-ContribAuthor ">Hiromichi  Suzuki</span>, <span class="hlFld-ContribAuthor ">Michael D.  Taylor</span>, <span class="hlFld-ContribAuthor ">Paul  Northcott</span>, <span class="hlFld-ContribAuthor ">Richard J.  Gilbertson</span>. </span><span class="cited-content_cbyCitation_article-title">DDX3X Suppresses the Susceptibility of Hindbrain Lineages to Medulloblastoma. </span><span class="cited-content_cbyCitation_journal-name">Developmental Cell</span><span> <strong>2020,</strong> <em>54 </em>
                                    (4)
                                     , 455-470.e5. <a href="https://doi.org/10.1016/j.devcel.2020.05.027" title="DOI URL">https://doi.org/10.1016/j.devcel.2020.05.027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.devcel.2020.05.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.devcel.2020.05.027%26sid%3Dliteratum%253Aachs%26jtitle%3DDevelopmental%2520Cell%26atitle%3DDDX3X%252BSuppresses%252Bthe%252BSusceptibility%252Bof%252BHindbrain%252BLineages%252Bto%252BMedulloblastoma%26aulast%3DPatmore%26aufirst%3DDeanna%2BM.%26date%3D2020%26volume%3D54%26issue%3D4%26spage%3D455%26epage%3D470.e5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Annalaura  Brai</span>, <span class="hlFld-ContribAuthor ">Valentina  Riva</span>, <span class="hlFld-ContribAuthor ">Francesco  Saladini</span>, <span class="hlFld-ContribAuthor ">Claudio  Zamperini</span>, <span class="hlFld-ContribAuthor ">Claudia Immacolata  Trivisani</span>, <span class="hlFld-ContribAuthor ">Anna  Garbelli</span>, <span class="hlFld-ContribAuthor ">Carla  Pennisi</span>, <span class="hlFld-ContribAuthor ">Alessia  Giannini</span>, <span class="hlFld-ContribAuthor ">Adele  Boccuto</span>, <span class="hlFld-ContribAuthor ">Francesca  Bugli</span>, <span class="hlFld-ContribAuthor ">Maurizio  Martini</span>, <span class="hlFld-ContribAuthor ">Maurizio  Sanguinetti</span>, <span class="hlFld-ContribAuthor ">Maurizio  Zazzi</span>, <span class="hlFld-ContribAuthor ">Elena  Dreassi</span>, <span class="hlFld-ContribAuthor ">Maurizio  Botta</span>, <span class="hlFld-ContribAuthor ">Giovanni  Maga</span>. </span><span class="cited-content_cbyCitation_article-title">DDX3X inhibitors, an effective way to overcome HIV-1 resistance targeting host proteins. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112319. <a href="https://doi.org/10.1016/j.ejmech.2020.112319" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112319</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112319%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDDX3X%252Binhibitors%25252C%252Ban%252Beffective%252Bway%252Bto%252Bovercome%252BHIV-1%252Bresistance%252Btargeting%252Bhost%252Bproteins%26aulast%3DBrai%26aufirst%3DAnnalaura%26date%3D2020%26volume%3D200%26spage%3D112319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rudra Narayan  Dash</span>, <span class="hlFld-ContribAuthor ">Alok Kumar  Moharana</span>, <span class="hlFld-ContribAuthor ">Bharat Bhusan  Subudhi</span>. </span><span class="cited-content_cbyCitation_article-title">Sulfonamides: Antiviral Strategy for Neglected Tropical Disease Virus. </span><span class="cited-content_cbyCitation_journal-name">Current Organic Chemistry</span><span> <strong>2020,</strong> <em>24 </em>
                                    (9)
                                     , 1018-1041. <a href="https://doi.org/10.2174/1385272824999200515094100" title="DOI URL">https://doi.org/10.2174/1385272824999200515094100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1385272824999200515094100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1385272824999200515094100%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Organic%2520Chemistry%26atitle%3DSulfonamides%25253A%252BAntiviral%252BStrategy%252Bfor%252BNeglected%252BTropical%252BDisease%252BVirus%26aulast%3DDash%26aufirst%3DRudra%2BNarayan%26date%3D2020%26volume%3D24%26issue%3D9%26spage%3D1018%26epage%3D1041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paola  Quaranta</span>, <span class="hlFld-ContribAuthor ">Giulia  Lottini</span>, <span class="hlFld-ContribAuthor ">Giulia  Chesi</span>, <span class="hlFld-ContribAuthor ">Flavia  Contrafatto</span>, <span class="hlFld-ContribAuthor ">Roberta  Russotto</span>, <span class="hlFld-ContribAuthor ">Lisa  Macera</span>, <span class="hlFld-ContribAuthor ">Michele  Lai</span>, <span class="hlFld-ContribAuthor ">Pietro Giorgio  Spezia</span>, <span class="hlFld-ContribAuthor ">Annalaura  Brai</span>, <span class="hlFld-ContribAuthor ">Maurizio  Botta</span>, <span class="hlFld-ContribAuthor ">Giulia  Freer</span>, <span class="hlFld-ContribAuthor ">Mauro  Pistello</span>. </span><span class="cited-content_cbyCitation_article-title">DDX3 inhibitors show antiviral activity against positive-sense single-stranded RNA viruses but not against negative-sense single-stranded RNA viruses: The coxsackie B model. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2020,</strong> <em>178 </em>, 104750. <a href="https://doi.org/10.1016/j.antiviral.2020.104750" title="DOI URL">https://doi.org/10.1016/j.antiviral.2020.104750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2020.104750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2020.104750%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DDDX3%252Binhibitors%252Bshow%252Bantiviral%252Bactivity%252Bagainst%252Bpositive-sense%252Bsingle-stranded%252BRNA%252Bviruses%252Bbut%252Bnot%252Bagainst%252Bnegative-sense%252Bsingle-stranded%252BRNA%252Bviruses%25253A%252BThe%252Bcoxsackie%252BB%252Bmodel%26aulast%3DQuaranta%26aufirst%3DPaola%26date%3D2020%26volume%3D178%26spage%3D104750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria  Romano</span>, <span class="hlFld-ContribAuthor ">Alessia  Ruggiero</span>, <span class="hlFld-ContribAuthor ">Flavia  Squeglia</span>, <span class="hlFld-ContribAuthor ">Giovanni  Maga</span>, <span class="hlFld-ContribAuthor ">Rita  Berisio</span>. </span><span class="cited-content_cbyCitation_article-title">A Structural View of SARS-CoV-2 RNA Replication Machinery: RNA Synthesis, Proofreading and Final Capping. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2020,</strong> <em>9 </em>
                                    (5)
                                     , 1267. <a href="https://doi.org/10.3390/cells9051267" title="DOI URL">https://doi.org/10.3390/cells9051267</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells9051267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells9051267%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DA%252BStructural%252BView%252Bof%252BSARS-CoV-2%252BRNA%252BReplication%252BMachinery%25253A%252BRNA%252BSynthesis%25252C%252BProofreading%252Band%252BFinal%252BCapping%26aulast%3DRomano%26aufirst%3DMaria%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D5%26spage%3D1267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandro  Sinigaglia</span>, <span class="hlFld-ContribAuthor ">Elektra  Peta</span>, <span class="hlFld-ContribAuthor ">Silvia  Riccetti</span>, <span class="hlFld-ContribAuthor ">Luisa  Barzon</span>. </span><span class="cited-content_cbyCitation_article-title">New avenues for therapeutic discovery against West Nile virus. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2020,</strong> <em>15 </em>
                                    (3)
                                     , 333-348. <a href="https://doi.org/10.1080/17460441.2020.1714586" title="DOI URL">https://doi.org/10.1080/17460441.2020.1714586</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2020.1714586&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2020.1714586%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DNew%252Bavenues%252Bfor%252Btherapeutic%252Bdiscovery%252Bagainst%252BWest%252BNile%252Bvirus%26aulast%3DSinigaglia%26aufirst%3DAlessandro%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D3%26spage%3D333%26epage%3D348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marina K.  Kukhanova</span>, <span class="hlFld-ContribAuthor ">Inna L.  Karpenko</span>, <span class="hlFld-ContribAuthor ">Alexander V.  Ivanov</span>. </span><span class="cited-content_cbyCitation_article-title">DEAD-box RNA Helicase DDX3: Functional Properties and Development of DDX3 Inhibitors as Antiviral and Anticancer Drugs. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (4)
                                     , 1015. <a href="https://doi.org/10.3390/molecules25041015" title="DOI URL">https://doi.org/10.3390/molecules25041015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25041015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25041015%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDEAD-box%252BRNA%252BHelicase%252BDDX3%25253A%252BFunctional%252BProperties%252Band%252BDevelopment%252Bof%252BDDX3%252BInhibitors%252Bas%252BAntiviral%252Band%252BAnticancer%252BDrugs%26aulast%3DKukhanova%26aufirst%3DMarina%2BK.%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D4%26spage%3D1015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sundy N. Y.  Yang</span>, <span class="hlFld-ContribAuthor ">Sarah C.  Atkinson</span>, <span class="hlFld-ContribAuthor ">Michelle D.  Audsley</span>, <span class="hlFld-ContribAuthor ">Steven M.  Heaton</span>, <span class="hlFld-ContribAuthor ">David A.  Jans</span>, <span class="hlFld-ContribAuthor ">Natalie A.  Borg</span>. </span><span class="cited-content_cbyCitation_article-title">RK-33 Is a Broad-Spectrum Antiviral Agent That Targets DEAD-Box RNA Helicase DDX3X. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2020,</strong> <em>9 </em>
                                    (1)
                                     , 170. <a href="https://doi.org/10.3390/cells9010170" title="DOI URL">https://doi.org/10.3390/cells9010170</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells9010170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells9010170%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DRK-33%252BIs%252Ba%252BBroad-Spectrum%252BAntiviral%252BAgent%252BThat%252BTargets%252BDEAD-Box%252BRNA%252BHelicase%252BDDX3X%26aulast%3DYang%26aufirst%3DSundy%2BN.%2BY.%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D1%26spage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brai</span>, <span class="hlFld-ContribAuthor ">Ronzini</span>, <span class="hlFld-ContribAuthor ">Riva</span>, <span class="hlFld-ContribAuthor ">Botta</span>, <span class="hlFld-ContribAuthor ">Zamperini</span>, <span class="hlFld-ContribAuthor ">Borgini</span>, <span class="hlFld-ContribAuthor ">Trivisani</span>, <span class="hlFld-ContribAuthor ">Garbelli</span>, <span class="hlFld-ContribAuthor ">Pennisi</span>, <span class="hlFld-ContribAuthor ">Boccuto</span>, <span class="hlFld-ContribAuthor ">Saladini</span>, <span class="hlFld-ContribAuthor ">Zazzi</span>, <span class="hlFld-ContribAuthor ">Maga</span>, <span class="hlFld-ContribAuthor ">Botta</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Antiviral Activity of Novel 1,3,4-Thiadiazole Inhibitors of DDX3X. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (21)
                                     , 3988. <a href="https://doi.org/10.3390/molecules24213988" title="DOI URL">https://doi.org/10.3390/molecules24213988</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24213988&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24213988%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSynthesis%252Band%252BAntiviral%252BActivity%252Bof%252BNovel%252B1%25252C3%25252C4-Thiadiazole%252BInhibitors%252Bof%252BDDX3X%26aulast%3DBrai%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D21%26spage%3D3988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Two-dimensional (2D) chemical structures of the known WNV inhibitors and their potential inhibitory target.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Overlapped binding mode of compound <b>1</b> (pink) and <b>2</b> (blue marine) in DDX3X helicase pocket. Main interactions of both compounds with key residues of the active site are highlighted. Remarkably, Arg276, Arg480, and Gly302 make important contacts also with RNA phosphate backbone, whereas Gly325, Pro274, and Arg503 interact directly with the nucleobases. (B) Two-dimensional (2D) structure representation of the schematic idea behind the merged series design. (C) Proposed binding mode for merged derivative <b>13</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Surface and cartoon representation of RNAâs main interactions within the helicase binding site of our homology model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Sulfonamides <b>8</b>â<b>20</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (a) <i>t</i>-BuONO, acetonitrile (ACN), 20 min, 0 Â°C; (b) TMSN<sub>3</sub>, ACN, 2 h, room temperature (r.t.); (ii) CuSO<sub>4</sub>Â·5 H<sub>2</sub>O, sodium ascorbate, H<sub>2</sub>O/<i>t</i>-BuOH (1:1), microwave (MW), 10 min, 120Â°C; (iii) H<sub>2</sub>O<sub>2</sub>, SOCl<sub>2</sub>, ACN, from 0 Â°C to r.t., 15 min; (iv) opportune amine, pyridine (dry), 5 h, r.t.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Ethers <b>5a</b> and <b>5b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (a) NaOH(aq) (6 M), 20 min, r.t; (b) dimethylsulfate or diethylsulfate, 60 min, 50â55 Â°C.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Aniline <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) (a) morpholine, K<sub>2</sub>CO<sub>3</sub>, dimethyl sulfoxide (DMSO) (dry), 3 h, 200 Â°C, (ii) H<sub>2</sub>, Pd/C, MeOH, 1 h.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Sulfonamides <b>31</b>â<b>34</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) Pyr, 5 h, r.t. (ii) H<sub>2</sub>, Pd/C, MeOH; (iii) (a) <i>t-</i>BuONO, CH<sub>3</sub>CN, 20 min, 0 Â°C; (b) TMSN<sub>3</sub>, CH<sub>3</sub>CN, 2 h, r.t.; (iv) opportune terminal alkyne, CuSO<sub>4</sub>Â·5H<sub>2</sub>O, sodium ascorbate, H<sub>2</sub>O/<i>t</i>-BuOH (1:1), MW, 10 min, 120 Â°C.</p></p></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Increasing concentrations of the inhibitor <b>8</b> were titrated in helicase assays, in the presence of 30 pmol of DDX3X and increasing fixed concentrations of the dsRNA substrate. Values are the mean of three independent measurements Â± standard deviation (S.D.). The residual enzymatic activity values were plotted as a function of the inhibitor concentrations, and the corresponding ID<sub>50</sub> values calculated for each dsRNA concentration are indicated on the right side of the panel.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/medium/jm-2018-014038_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of varying anti-DDX3X compound <b>9</b> at the time of administration on WNV replication. (A) WNV infection of Huh7 cells at MOI of 0.1 in presence of compound <b>9</b> at 1 and 10 Î¼M added at the indicated time (h) relative to the infection and assayed with the viral plaque reduction assay. Data represent mean Â± standard deviation. (B) WNV genomic and complementary viral RNA in Huh7 cells assayed after infection in absence or in presence of compound <b>9</b> at 1 Î¼M added at 1 and 3 h of the infection. Data represent mean Â± standard deviation. (C) WNV capsid protein electrophoretic mobility obtained from Huh7 cells after infection in absence or in presence of compound <b>9</b> at 1 Î¼M added at 1 and 3 h of the infection. No compound <b>9</b> addition performed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-5/acs.jmedchem.8b01403/20190308/images/large/jm-2018-014038_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01403&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i88">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11228" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11228" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 28 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chancey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rios, M.</span></span> <span> </span><span class="NLM_article-title">The global ecology and epidemiology of West Nile Virus</span>. <i>BioMed Res. Int.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2015</i></span>, <span class="NLM_elocation-id">376230</span> <span class="refDoi">Â DOI: 10.1155/2015/376230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1155%2F2015%2F376230" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2015&publication_year=2015&author=C.+Chanceyauthor=A.+Grinevauthor=E.+Volkovaauthor=M.+Rios&title=The+global+ecology+and+epidemiology+of+West+Nile+Virus&doi=10.1155%2F2015%2F376230"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1155%2F2015%2F376230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2015%252F376230%26sid%3Dliteratum%253Aachs%26aulast%3DChancey%26aufirst%3DC.%26aulast%3DGrinev%26aufirst%3DA.%26aulast%3DVolkova%26aufirst%3DE.%26aulast%3DRios%26aufirst%3DM.%26atitle%3DThe%2520global%2520ecology%2520and%2520epidemiology%2520of%2520West%2520Nile%2520Virus%26jtitle%3DBioMed%2520Res.%2520Int.%26date%3D2015%26volume%3D2015%26doi%3D10.1155%2F2015%2F376230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">European
Centre for Disease
Prevention and Control</span>. Epidemiological update:
West Nile virus transmission season in Europe, 2017 [online], <span class="NLM_year">2018</span>. Available at: <a href="https://ecdc.europa.eu/en/news-events/epidemiological-update-west-nile-virus-transmission-season-europe-2017" class="extLink">https://ecdc.europa.eu/en/news-events/epidemiological-update-west-nile-virus-transmission-season-europe-2017</a> (Accessed 22 Aug, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=European%0ACentre+for+Disease%0APrevention+and+Control.+Epidemiological+update%3A%0AWest+Nile+virus+transmission+season+in+Europe%2C+2017+%5Bonline%5D%2C+2018.+Available+at%3A+https%3A%2F%2Fecdc.europa.eu%2Fen%2Fnews-events%2Fepidemiological-update-west-nile-virus-transmission-season-europe-2017+%28Accessed+22+Aug%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Cdc.gov</span>. Preliminary maps & data for 2017 | West Nile Virus | CDC [online], <span class="NLM_year">2018</span>. Available at: <a href="https://www.cdc.gov/westnile/statsmaps/preliminarymapsdata2017/index.html" class="extLink">https://www.cdc.gov/westnile/statsmaps/preliminarymapsdata2017/index.html</a> (Accessed 22 Aug, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cdc.gov.+Preliminary+maps+%26+data+for+2017+%7C+West+Nile+Virus+%7C+CDC+%5Bonline%5D%2C+2018.+Available+at%3A+https%3A%2F%2Fwww.cdc.gov%2Fwestnile%2Fstatsmaps%2Fpreliminarymapsdata2017%2Findex.html+%28Accessed+22+Aug%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">European centre for disease
prevention and control</span>. Weekly Updates: 2018
West Nile Fever Transmission Season [online], <span class="NLM_year">2018</span>. Available
at: <a href="https://ecdc.europa.eu/en/west-nile-fever/surveillance-and-disease-data/disease-data-ecdc" class="extLink">https://ecdc.europa.eu/en/west-nile-fever/surveillance-and-disease-data/disease-data-ecdc</a> (Accessed 31 Aug, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=European+centre+for+disease%0Aprevention+and+control.+Weekly+Updates%3A+2018%0AWest+Nile+Fever+Transmission+Season+%5Bonline%5D%2C+2018.+Available%0Aat%3A+https%3A%2F%2Fecdc.europa.eu%2Fen%2Fwest-nile-fever%2Fsurveillance-and-disease-data%2Fdisease-data-ecdc+%28Accessed+31+Aug%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilder-Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubler, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weaver, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monath, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymann, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. W.</span></span> <span> </span><span class="NLM_article-title">Epidemic arboviral diseases: priorities for research and public health</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">e101</span>â <span class="NLM_lpage">e106</span>, <span class="refDoi">Â DOI: 10.1016/S1473-3099(16)30518-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2FS1473-3099%2816%2930518-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=28011234" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=e101-e106&author=A.+Wilder-Smithauthor=D.+J.+Gublerauthor=S.+C.+Weaverauthor=T.+P.+Monathauthor=D.+L.+Heymannauthor=T.+W.+Scott&title=Epidemic+arboviral+diseases%3A+priorities+for+research+and+public+health&doi=10.1016%2FS1473-3099%2816%2930518-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2816%2930518-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252816%252930518-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilder-Smith%26aufirst%3DA.%26aulast%3DGubler%26aufirst%3DD.%2BJ.%26aulast%3DWeaver%26aufirst%3DS.%2BC.%26aulast%3DMonath%26aufirst%3DT.%2BP.%26aulast%3DHeymann%26aufirst%3DD.%2BL.%26aulast%3DScott%26aufirst%3DT.%2BW.%26atitle%3DEpidemic%2520arboviral%2520diseases%253A%2520priorities%2520for%2520research%2520and%2520public%2520health%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2017%26volume%3D17%26spage%3De101%26epage%3De106%26doi%3D10.1016%2FS1473-3099%2816%2930518-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mercorelli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PalÃ¹, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loregian, A.</span></span> <span> </span><span class="NLM_article-title">Drug repurposing for viral infectious diseases: how far are we?</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">865</span>â <span class="NLM_lpage">876</span>, <span class="refDoi">Â DOI: 10.1016/j.tim.2018.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.tim.2018.04.004" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=865-876&author=B.+Mercorelliauthor=G.+Pal%C3%B9author=A.+Loregian&title=Drug+repurposing+for+viral+infectious+diseases%3A+how+far+are+we%3F&doi=10.1016%2Fj.tim.2018.04.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2018.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2018.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DMercorelli%26aufirst%3DB.%26aulast%3DPal%25C3%25B9%26aufirst%3DG.%26aulast%3DLoregian%26aufirst%3DA.%26atitle%3DDrug%2520repurposing%2520for%2520viral%2520infectious%2520diseases%253A%2520how%2520far%2520are%2520we%253F%26jtitle%3DTrends%2520Microbiol.%26date%3D2018%26spage%3D865%26epage%3D876%26doi%3D10.1016%2Fj.tim.2018.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valiente-EcheverrÃ­a, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermoso, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soto-Rifo, R.</span></span> <span> </span><span class="NLM_article-title">RNA helicase DDX3: at the crossroad of viral replication and antiviral immunity</span>. <i>Rev. Med. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">286</span>â <span class="NLM_lpage">299</span>, <span class="refDoi">Â DOI: 10.1002/rmv.1845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1002%2Frmv.1845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=26174373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFejtb3F" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=286-299&author=F.+Valiente-Echeverr%C3%ADaauthor=M.+A.+Hermosoauthor=R.+Soto-Rifo&title=RNA+helicase+DDX3%3A+at+the+crossroad+of+viral+replication+and+antiviral+immunity&doi=10.1002%2Frmv.1845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">RNA helicase DDX3: at the crossroad of viral replication and antiviral immunity</span></div><div class="casAuthors">Valiente-Echeverria, Fernando; Hermoso, Marcela A.; Soto-Rifo, Ricardo</div><div class="citationInfo"><span class="NLM_cas:title">Reviews in Medical Virology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">286-299</span>CODEN:
                <span class="NLM_cas:coden">RMVIEW</span>;
        ISSN:<span class="NLM_cas:issn">1052-9276</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Summary : Asp-Glu-Ala-Asp (DEAD)-box polypeptide 3, or DDX3, belongs to the DEAD-box family of ATP-dependent RNA helicases and is known to play different roles in RNA metab. ranging from transcription to nuclear export, translation, and assembly of stress granules.  In addn., there is growing evidence that DDX3 is a component of the innate immune response against viral infections.  As such, DDX3 has been shown to play roles both upstream and downstream of I-kappa beta kinase Îµ (IKKÎµ)/TANK-binding kinase 1, leading to IFN-Î² prodn.  Interestingly, several RNA viruses, including human threats such as HIV-1 and hepatitis C virus, hijack DDX3 to accomplish various steps of their replication cycles.  Thus, it seems that viruses have evolved to exploit DDX3's functions while threatening the innate immune response.  Understanding this interesting dichotomy in DDX3 function will help us not only to improve our knowledge of virus-host interactions but also to develop novel antiviral drugs targeting the multifaceted roles of DDX3 in viral replication.  Copyright Â© 2015 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreE9U2GQXnXbVg90H21EOLACvtfcHk0lhjVrX32yTHbQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFejtb3F&md5=93ccf21610b21186c66cdad36f6c4d99</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Frmv.1845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Frmv.1845%26sid%3Dliteratum%253Aachs%26aulast%3DValiente-Echeverr%25C3%25ADa%26aufirst%3DF.%26aulast%3DHermoso%26aufirst%3DM.%2BA.%26aulast%3DSoto-Rifo%26aufirst%3DR.%26atitle%3DRNA%2520helicase%2520DDX3%253A%2520at%2520the%2520crossroad%2520of%2520viral%2520replication%2520and%2520antiviral%2520immunity%26jtitle%3DRev.%2520Med.%2520Virol.%26date%3D2015%26volume%3D25%26spage%3D286%26epage%3D299%26doi%3D10.1002%2Frmv.1845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chahar, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierson, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manjunath, N.</span></span> <span> </span><span class="NLM_article-title">Ago-2-mediated slicer activity is essential for anti-flaviviral efficacy of RNAi</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e27551</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0027551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1371%2Fjournal.pone.0027551" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=22102908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFyntLbM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=S.+Chenauthor=H.+S.+Chaharauthor=S.+Abrahamauthor=H.+Wuauthor=T.+C.+Piersonauthor=X.+A.+Wangauthor=N.+Manjunath&title=Ago-2-mediated+slicer+activity+is+essential+for+anti-flaviviral+efficacy+of+RNAi&doi=10.1371%2Fjournal.pone.0027551"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Ago-2-mediated slicer activity is essential for anti-flaviviral efficacy of RNAi</span></div><div class="casAuthors">Chen, Shuiping; Chahar, Harendra S.; Abraham, Sojan; Wu, Haoquan; Pierson, Theodore C.; Wang, Xiaozhong A.; Manjunath, N.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e27551</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">RNA interference can be mediated by fully complementary siRNA or partially complementary miRNA.  SiRNAs are widely used to suppress viral replication and the fully complementary siRNA bound Ago-2 in the RISC is known to degrade the target RNA.  Although other argonaute proteins lacking slicer activity can also bind oligonucleotides with both si and miRNA structures, whether they can also contribute to antiviral effects is not entirely clear.  We tested si and miRNA structured oligos for target repression in dual luciferase assays as well as for inhibition of Dengue and West Nile virus replication in ES cells expressing individual Ago proteins.  In luciferase assays, both fully complementary and partially complementary oligos effectively repressed their targets in all individual Ago expressing cell lines, although the efficacy with fully complementary oligos was higher in Ago-2+ cells.  However, partially complementary oligos had no effect on virus replication in any cell line, while fully complementary siRNAs were highly effective in Ago-2 expressing, but not in cells expressing other Ago proteins.  This occurred irresp. of whether the target sequences were located in the coding region or 3'UTR of the virus.  We conclude that Ago-2 slicer activity is essential for anti-viral efficacy of siRNAs and miRNA-mediated translational repression/transcript destabilization is too weak to suppress the abundantly expressed flaviviral proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLSq2VQ419g7Vg90H21EOLACvtfcHk0lh9YYsmBbH_Hg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFyntLbM&md5=5315d8fd2f8f7d87295e0864fba723d8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0027551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0027551%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DChahar%26aufirst%3DH.%2BS.%26aulast%3DAbraham%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DPierson%26aufirst%3DT.%2BC.%26aulast%3DWang%26aufirst%3DX.%2BA.%26aulast%3DManjunath%26aufirst%3DN.%26atitle%3DAgo-2-mediated%2520slicer%2520activity%2520is%2520essential%2520for%2520anti-flaviviral%2520efficacy%2520of%2520RNAi%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0027551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chahar, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manjunath, N.</span></span> <span> </span><span class="NLM_article-title">P-body components LSM1, GW182, DDX3, DDX6 and XRN1 are recruited to WNV replication sites and positively regulate viral replication</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>436</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">7</span>, <span class="refDoi">Â DOI: 10.1016/j.virol.2012.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.virol.2012.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=23102969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKns7zL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2013&pages=1-7&author=H.+S.+Chaharauthor=S.+Chenauthor=N.+Manjunath&title=P-body+components+LSM1%2C+GW182%2C+DDX3%2C+DDX6+and+XRN1+are+recruited+to+WNV+replication+sites+and+positively+regulate+viral+replication&doi=10.1016%2Fj.virol.2012.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">P-body components LSM1, GW182, DDX3, DDX6 and XRN1 are recruited to WNV replication sites and positively regulate viral replication</span></div><div class="casAuthors">Chahar, Harendra S.; Chen, Shuiping; Manjunath, N.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-7</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In mammalian cells, proteins involved in mRNA silencing and degrdn. localize to discrete cytoplasmic foci called processing or P-bodies.  Here we show that microscopically visible P-bodies are greatly diminished following West Nile viral infection, but the component proteins are not depleted.  On the other hand, many P-body components including LSM1, GW182, DDX3, DDX6 and XRN1, but not others like DCP1a and EDC4 are recruited to the viral replication sites, as evidenced by their colocalization at perinuclear region with viral NS3.  Kinetic studies suggest that the component proteins are first released from P-bodies in response to WNV infection within 12 h post-infection, followed by recruitment to the viral replication sites by 24-36 h post-infection.  Silencing of the recruited proteins individually with siRNA interfered with viral replication to varying extents suggesting that the recruited proteins are required for efficient viral replication.  Thus, the P-body proteins might provide novel drug targets for inhibiting viral infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBfz18OOluLVg90H21EOLACvtfcHk0lgeeI7FTC9DqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKns7zL&md5=d0ed494c172da24e0a8943b7126667ca</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2012.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2012.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DChahar%26aufirst%3DH.%2BS.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DManjunath%26aufirst%3DN.%26atitle%3DP-body%2520components%2520LSM1%252C%2520GW182%252C%2520DDX3%252C%2520DDX6%2520and%2520XRN1%2520are%2520recruited%2520to%2520WNV%2520replication%2520sites%2520and%2520positively%2520regulate%2520viral%2520replication%26jtitle%3DVirology%26date%3D2013%26volume%3D436%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.virol.2012.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Owsianka, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A. H.</span></span> <span> </span><span class="NLM_article-title">Hepatitis C virus core protein interacts with a human DEAD box protein DDX3</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>257</i></span>,  <span class="NLM_fpage">330</span>â <span class="NLM_lpage">340</span>, <span class="refDoi">Â DOI: 10.1006/viro.1999.9659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1006%2Fviro.1999.9659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10329544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADyaK1MXislWrt78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1999&pages=330-340&author=A.+M.+Owsiankaauthor=A.+H.+Patel&title=Hepatitis+C+virus+core+protein+interacts+with+a+human+DEAD+box+protein+DDX3&doi=10.1006%2Fviro.1999.9659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Hepatitis C Virus Core Protein Interacts with a Human DEAD Box Protein DDX3</span></div><div class="casAuthors">Owsianka, Ania M.; Patel, Arvind H.</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">257</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">330-340</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Several studies have implicated hepatitis C virus (HCV) core in influencing the expression of host genes.  To identify cellular factors with a possible role in HCV replication and pathogenesis, we looked for cellular proteins that interact with the viral core protein.  A human liver cDNA library was screened in a yeast two-hybrid assay to identify cellular proteins that bind to core.  Several pos. clones were isolated, one of which encoded the C-terminal 253 amino acids of a putative RNA helicase, a DEAD box protein designated DDX3.  Bacterially expressed glutathione-S-transferase-DDX3 fusion protein specifically pulled down in vitro translated and radiolabeled HCV core, confirming a direct interaction.  Immunofluorescent staining of HeLa cells with a polyclonal antiserum showed that DDX3 is located predominantly in nuclear speckles and at low levels throughout the cytoplasm.  In cells infected with a recombinant vaccinia virus expressing HCV structural proteins (core, E1, and E2), DDX3 and core colocalized in distinct spots in the perinuclear region of the cytoplasm.  The regions of the proteins involved in binding were found by deletion anal. to be the N-terminal 59 amino acid residues of core and a C-terminal RS-like domain of DDX3.  The human DDX3 is a putative RNA helicase and a member of a highly conserved DEAD box subclass that includes murine PL10, Xenopus An3, and yeast Ded1 proteins.  Their role in RNA metab. or gene expression is unknown.  The significance of core-helicase interaction in HCV replication and pathogenesis is discussed.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaMXsdQhsa4bVg90H21EOLACvtfcHk0lgeeI7FTC9DqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXislWrt78%253D&md5=f9f73f66232a90557c53e6f6e23f6aca</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1006%2Fviro.1999.9659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fviro.1999.9659%26sid%3Dliteratum%253Aachs%26aulast%3DOwsianka%26aufirst%3DA.%2BM.%26aulast%3DPatel%26aufirst%3DA.%2BH.%26atitle%3DHepatitis%2520C%2520virus%2520core%2520protein%2520interacts%2520with%2520a%2520human%2520DEAD%2520box%2520protein%2520DDX3%26jtitle%3DVirology%26date%3D1999%26volume%3D257%26spage%3D330%26epage%3D340%26doi%3D10.1006%2Fviro.1999.9659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mamiya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worman, H. J.</span></span> <span> </span><span class="NLM_article-title">Hepatitis C virus core protein binds to a DEAD box RNA helicase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>274</i></span>,  <span class="NLM_fpage">15751</span>â <span class="NLM_lpage">15756</span>, <span class="refDoi">Â DOI: 10.1074/jbc.274.22.15751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1074%2Fjbc.274.22.15751" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=15751-15756&author=N.+Mamiyaauthor=H.+J.+Worman&title=Hepatitis+C+virus+core+protein+binds+to+a+DEAD+box+RNA+helicase&doi=10.1074%2Fjbc.274.22.15751"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.22.15751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.22.15751%26sid%3Dliteratum%253Aachs%26aulast%3DMamiya%26aufirst%3DN.%26aulast%3DWorman%26aufirst%3DH.%2BJ.%26atitle%3DHepatitis%2520C%2520virus%2520core%2520protein%2520binds%2520to%2520a%2520DEAD%2520box%2520RNA%2520helicase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D15751%26epage%3D15756%26doi%3D10.1074%2Fjbc.274.22.15751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ariumi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, K.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dansako, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikeda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span> <span> </span><span class="NLM_article-title">DDX3 DEAD-box RNA helicase is required for hepatitis C virus RNA replication</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">13922</span>â <span class="NLM_lpage">13926</span>, <span class="refDoi">Â DOI: 10.1128/JVI.01517-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1128%2FJVI.01517-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=17855521" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsValsbfK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=13922-13926&author=Y.+Ariumiauthor=M.+Kurokiauthor=K.-i.+Abeauthor=H.+Dansakoauthor=M.+Ikedaauthor=T.+Wakitaauthor=N.+Kato&title=DDX3+DEAD-box+RNA+helicase+is+required+for+hepatitis+C+virus+RNA+replication&doi=10.1128%2FJVI.01517-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">DDX3 DEAD-box RNA helicase is required for hepatitis C virus RNA replication</span></div><div class="casAuthors">Ariumi, Yasuo; Kuroki, Misao; Abe, Ken-ichi; Dansako, Hiromichi; Ikeda, Masanori; Wakita, Takaji; Kato, Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">13922-13926</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DDX3, a DEAD-box RNA helicase, binds to the hepatitis C virus (HCV) core protein.  However, the role(s) of DDX3 in HCV replication is still not understood.  Here the authors show that the accumulation of both genome-length HCV RNA (HCV-O, genotype 1b) and its replicon RNA were significantly suppressed in HuH-7-derived cells expressing short hairpin RNA targeted to DDX3 by lentivirus vector transduction.  As well, RNA replication of JFH1 (genotype 2a) and release of the core into the culture supernatants were suppressed in DDX3 knockdown cells after inoculation of the cell culture-generated HCVcc.  Thus, DDX3 is required for HCV RNA replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCK8FCrndFrrVg90H21EOLACvtfcHk0ljIqkR-AIpNGQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsValsbfK&md5=d3f569c960c7df4814b51c596b6b4b2c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FJVI.01517-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01517-07%26sid%3Dliteratum%253Aachs%26aulast%3DAriumi%26aufirst%3DY.%26aulast%3DKuroki%26aufirst%3DM.%26aulast%3DAbe%26aufirst%3DK.-i.%26aulast%3DDansako%26aufirst%3DH.%26aulast%3DIkeda%26aufirst%3DM.%26aulast%3DWakita%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DN.%26atitle%3DDDX3%2520DEAD-box%2520RNA%2520helicase%2520is%2520required%2520for%2520hepatitis%2520C%2520virus%2520RNA%2520replication%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D13922%26epage%3D13926%26doi%3D10.1128%2FJVI.01517-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishag, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span> <span> </span><span class="NLM_article-title">Cellular DDX3 regulates Japanese encephalitis virus replication by interacting with viral un-translated regions</span>. <i>Virology</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>449</i></span>,  <span class="NLM_fpage">70</span>â <span class="NLM_lpage">81</span>, <span class="refDoi">Â DOI: 10.1016/j.virol.2013.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.virol.2013.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=24418539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2cXns1ehtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2014&pages=70-81&author=C.+Liauthor=L.+L.+Geauthor=P.+P.+Liauthor=Y.+Wangauthor=J.+J.+Daiauthor=M.+X.+Sunauthor=L.+Huangauthor=Z.+Q.+Shenauthor=X.+C.+Huauthor=H.+Ishagauthor=Z.+Mao&title=Cellular+DDX3+regulates+Japanese+encephalitis+virus+replication+by+interacting+with+viral+un-translated+regions&doi=10.1016%2Fj.virol.2013.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cellular DDX3 regulates Japanese encephalitis virus replication by interacting with viral un-translated regions</span></div><div class="casAuthors">Li, Chen; Ge, Ling-ling; Li, Peng-peng; Wang, Yue; Dai, Juan-juan; Sun, Ming-xia; Huang, Li; Shen, Zhi-qiang; Hu, Xiao-chun; Ishag, Hassan; Mao, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">70-81</span>CODEN:
                <span class="NLM_cas:coden">VIRLAX</span>;
        ISSN:<span class="NLM_cas:issn">0042-6822</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Japanese encephalitis virus is one of the most common causes for epidemic viral encephalitis in humans and animals.  Herein we demonstrated that cellular helicase DDX3 is involved in JEV replication.  DDX3 knockdown inhibits JEV replication.  The helicase activity of DDX3 is crucial for JEV replication.  GST-pulldown and co-immunopptn. expts. demonstrated that DDX3 could interact with JEV non-structural proteins 3 and 5.  Co-immunopptn. and confocal microscopy anal. confirmed that DDX3 interacts and colocalizes with these viral proteins and viral RNA during the infection.  We detd. that DDX3 binds to JEV 5' and 3' un-translated regions.  We used a JEV-replicon system to demonstrate that DDX3 pos. regulates viral RNA translation, which might affect viral RNA replication at the late stage of virus infection.  Collectively, we identified that DDX3 is necessary for JEV infection, suggesting that DDX3 might be a novel target to design new antiviral agents against JEV or other flavivirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxWB6dSzUY6LVg90H21EOLACvtfcHk0lgRwXe38XsKsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXns1ehtQ%253D%253D&md5=5593f579761a879895279c4082160c6e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.virol.2013.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virol.2013.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGe%26aufirst%3DL.%2BL.%26aulast%3DLi%26aufirst%3DP.%2BP.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DJ.%2BJ.%26aulast%3DSun%26aufirst%3DM.%2BX.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DZ.%2BQ.%26aulast%3DHu%26aufirst%3DX.%2BC.%26aulast%3DIshag%26aufirst%3DH.%26aulast%3DMao%26aufirst%3DZ.%26atitle%3DCellular%2520DDX3%2520regulates%2520Japanese%2520encephalitis%2520virus%2520replication%2520by%2520interacting%2520with%2520viral%2520un-translated%2520regions%26jtitle%3DVirology%26date%3D2014%26volume%3D449%26spage%3D70%26epage%3D81%26doi%3D10.1016%2Fj.virol.2013.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noble, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schul, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, P. Y.</span></span> <span> </span><span class="NLM_article-title">Strategies for development of Dengue virus inhibitors</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">450</span>â <span class="NLM_lpage">462</span>, <span class="refDoi">Â DOI: 10.1016/j.antiviral.2009.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.antiviral.2009.12.011" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2010&pages=450-462&author=C.+G.+Nobleauthor=Y.+L.+Chenauthor=H.+Dongauthor=F.+Guauthor=S.+P.+Limauthor=W.+Schulauthor=Q.+Y.+Wangauthor=P.+Y.+Shi&title=Strategies+for+development+of+Dengue+virus+inhibitors&doi=10.1016%2Fj.antiviral.2009.12.011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.antiviral.2009.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.antiviral.2009.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DC.%2BG.%26aulast%3DChen%26aufirst%3DY.%2BL.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DF.%26aulast%3DLim%26aufirst%3DS.%2BP.%26aulast%3DSchul%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DQ.%2BY.%26aulast%3DShi%26aufirst%3DP.%2BY.%26atitle%3DStrategies%2520for%2520development%2520of%2520Dengue%2520virus%2520inhibitors%26jtitle%3DAntiviral%2520Res.%26date%3D2010%26volume%3D85%26spage%3D450%26epage%3D462%26doi%3D10.1016%2Fj.antiviral.2009.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benfield, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahsoun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. L.</span></span> <span> </span><span class="NLM_article-title">Vaccinia virus protein K7 is a virulence factor that alters the acute immune response to infection</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1647</span>â <span class="NLM_lpage">1657</span>, <span class="refDoi">Â DOI: 10.1099/vir.0.052670-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1099%2Fvir.0.052670-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=23580427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWltL3P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2013&pages=1647-1657&author=C.+T.+Benfieldauthor=H.+Renauthor=S.+J.+Lucasauthor=B.+Bahsounauthor=G.+L.+Smith&title=Vaccinia+virus+protein+K7+is+a+virulence+factor+that+alters+the+acute+immune+response+to+infection&doi=10.1099%2Fvir.0.052670-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Vaccinia virus protein K7 is a virulence factor that alters the acute immune response to infection</span></div><div class="casAuthors">Benfield, Camilla T. O.; Ren, Hongwei; Lucas, Stuart J.; Bahsoun, Basma; Smith, Geoffrey L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1647-1657</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">Vaccinia virus (VACV) encodes many proteins that antagonize the innate immune system including a family of intracellular proteins with a B-cell lymphoma (Bcl)-2-like structure.  One of these Bcl-2 proteins called K7 binds Toll-like receptor-adaptor proteins and the DEAD-box RNA helicase DDX3 and thereby inhibits the activation of NF-ÎºB and interferon regulatory factor 3.  However, the contribution of K7 to virus virulence is not known.  Here a VACV lacking the K7R gene (vÎK7) was constructed and compared with control viruses that included a plaque purified wt (vK7), a revertant with the K7R gene reinserted (vK7-rev) and a frame-shifted virus in which the translational initiation codon was mutated to prevent K7 protein expression (vK7-fs).  Data presented show that loss of K7 does not affect virus replication in cell culture or in vivo; however, viruses lacking the K7 protein were less virulent than controls in murine intradermal (i.d.) and intranasal (i.n.) infection models and there was an altered acute immune response to infection.  In the i.d. model, vÎK7 induced smaller lesions than controls, and after i.n. infection vÎK7 induced a reduced wt. loss and signs of illness, and more rapid clearance of virus from infected tissue.  Concomitantly, the intrapulmonary innate immune response to infection with vÎK7 showed increased infiltration of NK cells and CD8+ T-cells, enhanced MHC class II expression by macrophages, and enhanced cytolysis of target cells by NK cells and VACV-specific CD8+ T-cells.  Thus protein K7 is a virulence factor that affects the acute immune response to infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNukdrK617NbVg90H21EOLACvtfcHk0ligyW6bpUG5aQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlWltL3P&md5=09b687b0b56e6059bcce7313a3f1bc47</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1099%2Fvir.0.052670-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fvir.0.052670-0%26sid%3Dliteratum%253Aachs%26aulast%3DBenfield%26aufirst%3DC.%2BT.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DLucas%26aufirst%3DS.%2BJ.%26aulast%3DBahsoun%26aufirst%3DB.%26aulast%3DSmith%26aufirst%3DG.%2BL.%26atitle%3DVaccinia%2520virus%2520protein%2520K7%2520is%2520a%2520virulence%2520factor%2520that%2520alters%2520the%2520acute%2520immune%2520response%2520to%2520infection%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2013%26volume%3D94%26spage%3D1647%26epage%3D1657%26doi%3D10.1099%2Fvir.0.052670-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallacara, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Protein-protein interactions and human cellular cofactors as new targets for HIV therapy</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">8</span>, <span class="refDoi">Â DOI: 10.1016/j.coph.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.coph.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=24993074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKmtL%252FK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2014&pages=1-8&author=C.+Tintoriauthor=A.+Braiauthor=A.+L.+Fallacaraauthor=R.+Faziauthor=S.+Schenoneauthor=M.+Botta&title=Protein-protein+interactions+and+human+cellular+cofactors+as+new+targets+for+HIV+therapy&doi=10.1016%2Fj.coph.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interactions and human cellular cofactors as new targets for HIV therapy</span></div><div class="casAuthors">Tintori, Cristina; Brai, Annalaura; Fallacara, Anna Lucia; Fazi, Roberta; Schenone, Silvia; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Two novel approaches for the development of new drugs against AIDS are summarized each leading to the achievement of important discoveries in anti-HIV therapy.  Despite the success of HAART in reducing mortality, resistant strains continue to emerge in the clinic, underscoring the importance of developing next-generation drugs.  Protein-protein interactions and human cellular cofactors represent the new targets of tomorrow in HIV research.  The most relevant results obtained in the last few years by the two new strategies are described herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfeMocO2GFdLVg90H21EOLACvtfcHk0ligyW6bpUG5aQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKmtL%252FK&md5=c00f92556fbf3aa779219b5655ea5ceb</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DFallacara%26aufirst%3DA.%2BL.%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DProtein-protein%2520interactions%2520and%2520human%2520cellular%2520cofactors%2520as%2520new%2520targets%2520for%2520HIV%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2014%26volume%3D18%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.coph.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Radi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuele, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldanti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schenone, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of the first small molecule inhibitor of human DDX3 specifically designed to target the RNA binding site: towards the next generation HIV-1 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2094</span>â <span class="NLM_lpage">2098</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2011.12.135</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.bmcl.2011.12.135" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=2094-2098&author=M.+Radiauthor=F.+Falchiauthor=A.+Garbelliauthor=A.+Samueleauthor=V.+Bernardoauthor=S.+Paolucciauthor=F.+Baldantiauthor=S.+Schenoneauthor=F.+Manettiauthor=G.+Magaauthor=M.+Botta&title=Discovery+of+the+first+small+molecule+inhibitor+of+human+DDX3+specifically+designed+to+target+the+RNA+binding+site%3A+towards+the+next+generation+HIV-1+inhibitors&doi=10.1016%2Fj.bmcl.2011.12.135"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.12.135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.12.135%26sid%3Dliteratum%253Aachs%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DSamuele%26aufirst%3DA.%26aulast%3DBernardo%26aufirst%3DV.%26aulast%3DPaolucci%26aufirst%3DS.%26aulast%3DBaldanti%26aufirst%3DF.%26aulast%3DSchenone%26aufirst%3DS.%26aulast%3DManetti%26aufirst%3DF.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520the%2520first%2520small%2520molecule%2520inhibitor%2520of%2520human%2520DDX3%2520specifically%2520designed%2520to%2520target%2520the%2520RNA%2520binding%2520site%253A%2520towards%2520the%2520next%2520generation%2520HIV-1%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D2094%26epage%3D2098%26doi%3D10.1016%2Fj.bmcl.2011.12.135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamperini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stigliano, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">EstÃ©, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerhans, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saladini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zazzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Human DDX3 protein is a valuable target to develop broad spectrum antiviral agents</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">5388</span>â <span class="NLM_lpage">5393</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1522987113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1073%2Fpnas.1522987113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=27118832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC28XmslKrtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=5388-5393&author=A.+Braiauthor=R.+Faziauthor=C.+Tintoriauthor=C.+Zamperiniauthor=F.+Bugliauthor=M.+Sanguinettiauthor=E.+Stiglianoauthor=J.+Est%C3%A9author=R.+Badiaauthor=S.+Francoauthor=M.+A.+Martinezauthor=J.+P.+Martinezauthor=A.+Meyerhansauthor=F.+Saladiniauthor=M.+Zazziauthor=A.+Garbelliauthor=G.+Magaauthor=M.+Botta&title=Human+DDX3+protein+is+a+valuable+target+to+develop+broad+spectrum+antiviral+agents&doi=10.1073%2Fpnas.1522987113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Human DDX3 protein is a valuable target to develop broad spectrum antiviral agents</span></div><div class="casAuthors">Brai, Annalaura; Fazi, Roberta; Tintori, Cristina; Zamperini, Claudio; Bugli, Francesca; Sanguinetti, Maurizio; Stigliano, Egidio; Este, Jose; Badia, Roger; Franco, Sandra; Martinez, Miguel A.; Martinez, Javier P.; Meyerhans, Andreas; Saladini, Francesco; Zazzi, Maurizio; Garbelli, Anna; Maga, Giovanni; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5388-5393</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Targeting a host factor essential for the replication of different viruses but not for the cells offers a higher genetic barrier to the development of resistance, may simplify therapy regimens for coinfections, and facilitates management of emerging viral diseases.  DEAD-box polypeptide 3 (DDX3) is a human host factor required for the replication of several DNA and RNA viruses, including some of the most challenging human pathogens currently circulating, such as HIV-1, Hepatitis C virus, Dengue virus, and West Nile virus.  Herein, the authors showed for the first time, to the authors' knowledge, that the inhibition of DDX3 by a small mol. could be successfully exploited for the development of a broad spectrum antiviral agent.  In addn. to the multiple antiviral activities, hit compd. (I) retained full activity against drug-resistant HIV-1 strains in the absence of cellular toxicity.  Pharmacokinetics and toxicity studies in rats confirmed a good safety profile and bioavailability of (I).  Thus, DDX3 is here validated as a valuable therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosRPcNHvwVBrVg90H21EOLACvtfcHk0ljQHnKrVrwV-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmslKrtrg%253D&md5=d619c77e8edf9eaeb492993cb0964aab</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1522987113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1522987113%26sid%3Dliteratum%253Aachs%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DZamperini%26aufirst%3DC.%26aulast%3DBugli%26aufirst%3DF.%26aulast%3DSanguinetti%26aufirst%3DM.%26aulast%3DStigliano%26aufirst%3DE.%26aulast%3DEst%25C3%25A9%26aufirst%3DJ.%26aulast%3DBadia%26aufirst%3DR.%26aulast%3DFranco%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DM.%2BA.%26aulast%3DMartinez%26aufirst%3DJ.%2BP.%26aulast%3DMeyerhans%26aufirst%3DA.%26aulast%3DSaladini%26aufirst%3DF.%26aulast%3DZazzi%26aufirst%3DM.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DHuman%2520DDX3%2520protein%2520is%2520a%2520valuable%2520target%2520to%2520develop%2520broad%2520spectrum%2520antiviral%2520agents%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D5388%26epage%3D5393%26doi%3D10.1073%2Fpnas.1522987113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fazi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tintori, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selvaraj, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bavagnoli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, M.</span></span> <span> </span><span class="NLM_article-title">Homology model-based virtual screening for the identification of helicase human DDX3X inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2443</span>â <span class="NLM_lpage">2454</span>, <span class="refDoi">Â DOI: 10.1021/acs.jcim.5b00419</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00419" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2443-2454&author=R.+Faziauthor=C.+Tintoriauthor=A.+Braiauthor=M.+K.+Selvarajauthor=L.+Bavagnoliauthor=A.+Garbelliauthor=G.+Magaauthor=M.+Botta&title=Homology+model-based+virtual+screening+for+the+identification+of+helicase+human+DDX3X+inhibitors&doi=10.1021%2Facs.jcim.5b00419"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Homology Model-Based Virtual Screening for the Identification of Human Helicase DDX3 Inhibitors</span></div><div class="casAuthors">Fazi, Roberta; Tintori, Cristina; Brai, Annalaura; Botta, Lorenzo; Selvaraj, Manikandan; Garbelli, Anna; Maga, Giovanni; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2443-2454</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting cellular cofactors instead of viral enzymes represents a new strategy to combat infectious diseases, which should help to overcome the problem of viral resistance.  Recently, it has been revealed that the cellular ATPase/RNA helicase X-linked DEAD-box polypeptide 3 (DDX3) is an essential host factor for the replication of several viruses such as HIV, HCV, JEV, Dengue, and West Nile.  Accordingly, a drug targeting DDX3 could theor. inhibit all viruses that are dependent on this host factor.  Herein, for the first time, a model of hDDX3 in its closed conformation, which binds the viral RNA was developed by using the homol. module of Prime through the Maestro interface of Schrodinger.  Next, a structure-based virtual screening protocol was applied to identify DDX3 small mol. inhibitors targeting the RNA binding pocket.  As a result, an impressive hit rate of 40% was obtained with the identification of 10 active compds. out of the 25 tested small mols.  The best poses of the active ligands highlighted the crucial residues to be targeted for the inhibition of the helicase activity of DDX3.  The obtained results confirm the reliability of the constructed DDX3/RNA model and the proposed computational strategy for investigating novel DDX3 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO1Sy7ZFr2kbVg90H21EOLACvtfcHk0lhWMPVQDlo66Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVent7fN&md5=0c739457f73969394c6a31c113617bb6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00419%26sid%3Dliteratum%253Aachs%26aulast%3DFazi%26aufirst%3DR.%26aulast%3DTintori%26aufirst%3DC.%26aulast%3DBrai%26aufirst%3DA.%26aulast%3DSelvaraj%26aufirst%3DM.%2BK.%26aulast%3DBavagnoli%26aufirst%3DL.%26aulast%3DGarbelli%26aufirst%3DA.%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DBotta%26aufirst%3DM.%26atitle%3DHomology%2520model-based%2520virtual%2520screening%2520for%2520the%2520identification%2520of%2520helicase%2520human%2520DDX3X%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2443%26epage%3D2454%26doi%3D10.1021%2Facs.jcim.5b00419" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yedavalli, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starost, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosmane, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeang, K. T.</span></span> <span> </span><span class="NLM_article-title">Ring expanded nucleoside analogues inhibit RNA helicase and intracellular human immunodeficiency virus type 1 replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">5043</span>â <span class="NLM_lpage">5051</span>, <span class="refDoi">Â DOI: 10.1021/jm800332m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800332m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5043-5051&author=V.+S.+Yedavalliauthor=N.+Zhangauthor=H.+Caiauthor=P.+Zhangauthor=M.+F.+Starostauthor=R.+S.+Hosmaneauthor=K.+T.+Jeang&title=Ring+expanded+nucleoside+analogues+inhibit+RNA+helicase+and+intracellular+human+immunodeficiency+virus+type+1+replication&doi=10.1021%2Fjm800332m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm800332m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800332m%26sid%3Dliteratum%253Aachs%26aulast%3DYedavalli%26aufirst%3DV.%2BS.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DCai%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DStarost%26aufirst%3DM.%2BF.%26aulast%3DHosmane%26aufirst%3DR.%2BS.%26aulast%3DJeang%26aufirst%3DK.%2BT.%26atitle%3DRing%2520expanded%2520nucleoside%2520analogues%2520inhibit%2520RNA%2520helicase%2520and%2520intracellular%2520human%2520immunodeficiency%2520virus%2520type%25201%2520replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5043%26epage%3D5051%26doi%3D10.1021%2Fjm800332m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yedavalli, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuveut, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeang, K. T.</span></span> <span> </span><span class="NLM_article-title">Requirement of DDX3X DEAD box RNA helicase for HIV-1 Rev-RRE export function</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">381</span>â <span class="NLM_lpage">392</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2004.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.cell.2004.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=15507209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvFOgtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=381-392&author=V.+S.+Yedavalliauthor=C.+Neuveutauthor=Y.+H.+Chiauthor=L.+Kleimanauthor=K.+T.+Jeang&title=Requirement+of+DDX3X+DEAD+box+RNA+helicase+for+HIV-1+Rev-RRE+export+function&doi=10.1016%2Fj.cell.2004.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Requirement of DDX3 DEAD box RNA helicase for HIV-1 Rev-RRE export function</span></div><div class="casAuthors">Yedavalli, Venkat S. R. K.; Neuveut, Christine; Chi, Ya-hui; Kleiman, Lawrence; Jeang, Kuan-Teh</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-392</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A single transcript in its unspliced and spliced forms directs the synthesis of all HIV-1 proteins.  Although nuclear export of intron-contg. cellular transcripts is restricted in mammalian cells, HIV-1 has evolved the viral Rev protein to overcome this restriction for viral transcripts.  Previously, CRM1 was identified as a cellular cofactor for Rev-dependent export of intron-contg. HIV-1 RNA.  Here, we present evidence that Rev/CRM1 activity utilizes the ATP-dependent DEAD box RNA helicase, DDX3.  We show that DDX3 is a nucleo-cytoplasmic shuttling protein, which binds CRM1 and localizes to nuclear membrane pores.  Knockdown of DDX3 using either antisense vector or dominant-neg. mutants suppressed Rev-RRE-function in the export of incompletely spliced HIV-1 RNAs.  Plausibly, DDX3 is the human RNA helicase which functions in the CRM1 RNA export pathway analogously to the postulated role for Dbp5p in yeast mRNA export.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreblW84ZWZULVg90H21EOLACvtfcHk0lgwfe7cdBwEgg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvFOgtbs%253D&md5=165d2abcfc244ec9dd51b09aa972b94a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DYedavalli%26aufirst%3DV.%2BS.%26aulast%3DNeuveut%26aufirst%3DC.%26aulast%3DChi%26aufirst%3DY.%2BH.%26aulast%3DKleiman%26aufirst%3DL.%26aulast%3DJeang%26aufirst%3DK.%2BT.%26atitle%3DRequirement%2520of%2520DDX3X%2520DEAD%2520box%2520RNA%2520helicase%2520for%2520HIV-1%2520Rev-RRE%2520export%2520function%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D381%26epage%3D392%26doi%3D10.1016%2Fj.cell.2004.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhari, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mogurampelly, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suryaprakash, N.</span></span> <span> </span><span class="NLM_article-title">Engagement of CF<sub>3</sub> group in N-HÂ·Â·Â·F-C hydrogen bond in the solution state: NMR spectroscopy and MD simulation studies</span>. <i>J. Phys. Chem. B</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1123</span>â <span class="NLM_lpage">1129</span>, <span class="refDoi">Â DOI: 10.1021/jp310798d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp310798d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2013&pages=1123-1129&author=S.+R.+Chaudhariauthor=S.+Mogurampellyauthor=N.+Suryaprakash&title=Engagement+of+CF3+group+in+N-H%C2%B7%C2%B7%C2%B7F-C+hydrogen+bond+in+the+solution+state%3A+NMR+spectroscopy+and+MD+simulation+studies&doi=10.1021%2Fjp310798d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjp310798d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp310798d%26sid%3Dliteratum%253Aachs%26aulast%3DChaudhari%26aufirst%3DS.%2BR.%26aulast%3DMogurampelly%26aufirst%3DS.%26aulast%3DSuryaprakash%26aufirst%3DN.%26atitle%3DEngagement%2520of%2520CF3%2520group%2520in%2520N-H%25C2%25B7%25C2%25B7%25C2%25B7F-C%2520hydrogen%2520bond%2520in%2520the%2520solution%2520state%253A%2520NMR%2520spectroscopy%2520and%2520MD%2520simulation%2520studies%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2013%26volume%3D117%26spage%3D1123%26epage%3D1129%26doi%3D10.1021%2Fjp310798d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bol, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vesuna, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aziz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irving, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantravedi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heerma van Voss, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bordt, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polster, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cope, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Groep, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondaskar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosmane, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Wall, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Diest, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, V.</span></span> <span> </span><span class="NLM_article-title">Targeting DDX3 with a small molecule inhibitor for lung cancer therapy</span>. <i>EMBO Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">648</span>â <span class="NLM_lpage">669</span>, <span class="refDoi">Â DOI: 10.15252/emmm.201404368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.15252%2Femmm.201404368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=25820276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsVWktLY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=648-669&author=G.+M.+Bolauthor=F.+Vesunaauthor=M.+Xieauthor=J.+Zengauthor=K.+Azizauthor=N.+Gandhiauthor=A.+Levineauthor=A.+Irvingauthor=D.+Korzauthor=S.+Tantravediauthor=M.+R.+Heerma+van+Vossauthor=K.+Gabrielsonauthor=E.+A.+Bordtauthor=B.+M.+Polsterauthor=L.+Copeauthor=P.+van+der%0AGroepauthor=A.+Kondaskarauthor=M.+A.+Rudekauthor=R.+S.+Hosmaneauthor=E.+van+der+Wallauthor=P.+J.+van+Diestauthor=P.+T.+Tranauthor=V.+Raman&title=Targeting+DDX3+with+a+small+molecule+inhibitor+for+lung+cancer+therapy&doi=10.15252%2Femmm.201404368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting DDX3 with a small molecule inhibitor for lung cancer therapy</span></div><div class="casAuthors">Bol, Guus M.; Vesuna, Farhad; Xie, Min; Zeng, Jing; Aziz, Khaled; Gandhi, Nishant; Levine, Anne; Irving, Ashley; Korz, Dorian; Tantravedi, Saritha; Heerma van Voss, Marise R.; Gabrielson, Kathleen; Bordt, Evan A.; Polster, Brian M.; Cope, Leslie; van der Groep, Petra; Kondaskar, Atul; Rudek, Michelle A.; Hosmane, Ramachandra S.; van der Wall, Elsken; van Diest, Paul J.; Tran, Phuoc T.; Raman, Venu</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">648-669</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Lung cancer is the most common malignancy worldwide and is a focus for developing targeted therapies due to its refractory nature to current treatment.  We identified a RNA helicase, DDX3, which is overexpressed in many cancer types including lung cancer and is assocd. with lower survival in lung cancer patients.  We designed a first-in-class small mol. inhibitor, RK-33, which binds to DDX3 and abrogates its activity.  Inhibition of DDX3 by RK-33 caused G1 cell cycle arrest, induced apoptosis, and promoted radiation sensitization in DDX3-overexpressing cells.  Importantly, RK-33 in combination with radiation induced tumor regression in multiple mouse models of lung cancer.  Mechanistically, loss of DDX3 function either by shRNA or by RK-33 impaired Wnt signaling through disruption of the DDX3-Î²-catenin axis and inhibited non-homologous end joining-the major DNA repair pathway in mammalian somatic cells.  Overall, inhibition of DDX3 by RK-33 promotes tumor regression, thus providing a compelling argument to develop DDX3 inhibitors for lung cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOehktsrE8jLVg90H21EOLACvtfcHk0lg5NGXAxrZ_4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsVWktLY%253D&md5=149e69c6868fe66c4ae40619b528dbd1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.15252%2Femmm.201404368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Femmm.201404368%26sid%3Dliteratum%253Aachs%26aulast%3DBol%26aufirst%3DG.%2BM.%26aulast%3DVesuna%26aufirst%3DF.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DAziz%26aufirst%3DK.%26aulast%3DGandhi%26aufirst%3DN.%26aulast%3DLevine%26aufirst%3DA.%26aulast%3DIrving%26aufirst%3DA.%26aulast%3DKorz%26aufirst%3DD.%26aulast%3DTantravedi%26aufirst%3DS.%26aulast%3DHeerma%2Bvan%2BVoss%26aufirst%3DM.%2BR.%26aulast%3DGabrielson%26aufirst%3DK.%26aulast%3DBordt%26aufirst%3DE.%2BA.%26aulast%3DPolster%26aufirst%3DB.%2BM.%26aulast%3DCope%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BGroep%26aufirst%3DP.%26aulast%3DKondaskar%26aufirst%3DA.%26aulast%3DRudek%26aufirst%3DM.%2BA.%26aulast%3DHosmane%26aufirst%3DR.%2BS.%26aulast%3Dvan%2Bder%2BWall%26aufirst%3DE.%26aulast%3Dvan%2BDiest%26aufirst%3DP.%2BJ.%26aulast%3DTran%26aufirst%3DP.%2BT.%26aulast%3DRaman%26aufirst%3DV.%26atitle%3DTargeting%2520DDX3%2520with%2520a%2520small%2520molecule%2520inhibitor%2520for%2520lung%2520cancer%2520therapy%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2015%26volume%3D7%26spage%3D648%26epage%3D669%26doi%3D10.15252%2Femmm.201404368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Linke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellerbrok, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedrig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauli, G.</span></span> <span> </span><span class="NLM_article-title">Detection of West Nile virus lineages 1 and 2 by real-time PCR</span>. <i>J. Virol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">355</span>â <span class="NLM_lpage">358</span>, <span class="refDoi">Â DOI: 10.1016/j.jviromet.2007.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1016%2Fj.jviromet.2007.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=17604132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1KrsLrL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2007&pages=355-358&author=S.+Linkeauthor=H.+Ellerbrokauthor=M.+Niedrigauthor=A.+Nitscheauthor=G.+Pauli&title=Detection+of+West+Nile+virus+lineages+1+and+2+by+real-time+PCR&doi=10.1016%2Fj.jviromet.2007.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of West Nile virus lineages 1 and 2 by real-time PCR</span></div><div class="casAuthors">Linke, Sonja; Ellerbrok, Heinz; Niedrig, Matthias; Nitsche, Andreas; Pauli, Georg</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virological Methods</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">355-358</span>CODEN:
                <span class="NLM_cas:coden">JVMEDH</span>;
        ISSN:<span class="NLM_cas:issn">0166-0934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">West Nile virus (WNV) is a Flavivirus attracting worldwide attention because it has spread rapidly across the Americas since its first appearance in New York City in 1999.  Several PCR-based diagnostic methods have been developed for the detection of WNV.  The focus of these assays has been WNV lineage 1 which can be found worldwide, while lineage 2 viruses were thought to be endemic only in some regions of Africa.  However, both lineages may be imported from Africa to Europe by migrating birds.  In order to det. the incidence of WNV in Germany, a real-time-based PCR assay was developed, targeting a conserved region of WNV lineages 1 and 2.  This assay is a suitable tool for the diagnosis of WNV and for surveillance studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJNzshDNwNGrVg90H21EOLACvtfcHk0lg5NGXAxrZ_4w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1KrsLrL&md5=f90bcf395a82a8284ab79134549b219a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.jviromet.2007.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jviromet.2007.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DLinke%26aufirst%3DS.%26aulast%3DEllerbrok%26aufirst%3DH.%26aulast%3DNiedrig%26aufirst%3DM.%26aulast%3DNitsche%26aufirst%3DA.%26aulast%3DPauli%26aufirst%3DG.%26atitle%3DDetection%2520of%2520West%2520Nile%2520virus%2520lineages%25201%2520and%25202%2520by%2520real-time%2520PCR%26jtitle%3DJ.%2520Virol.%2520Methods%26date%3D2007%26volume%3D146%26spage%3D355%26epage%3D358%26doi%3D10.1016%2Fj.jviromet.2007.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved protein-ligand docking using GOLD</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>â <span class="NLM_lpage">623</span>, <span class="refDoi">Â DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+protein-ligand+docking+using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Ã of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of â¼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0lhVseEuJPv1dQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520protein-ligand%2520docking%2520using%2520GOLD%26jtitle%3DProteins%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span>; <span class="NLM_string-name">Nissink, J. W. M.</span>; <span class="NLM_string-name">Taylor, R.</span></span> <span> </span><span class="NLM_article-title">ProteinâLigand Docking and Virtual Screening with GOLD</span>. In  <i>Virtual Screening in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B.</span>, <span class="NLM_string-name">Alvarez, J.</span></span>, Eds.; <span class="NLM_publisher-name">Taylor & Francis CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, Florida</span>, <span class="NLM_year">2005</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1201%2F9781420028775.ch15" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=J.+C.+Cole&author=J.+W.+M.+Nissink&author=R.+Taylorauthor=B.+Shoichet&author=J.+Alvarez&title=Virtual+Screening+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1201%2F9781420028775.ch15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1201%252F9781420028775.ch15%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DJ.%2BC.%26atitle%3DProtein%25E2%2580%2593Ligand%2520Docking%2520and%2520Virtual%2520Screening%2520with%2520GOLD%26btitle%3DVirtual%2520Screening%2520in%2520Drug%2520Discovery%26aulast%3DShoichet%26aufirst%3DB.%26pub%3DTaylor%2520%2526%2520Francis%2520CRC%2520Press%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span> <i>PyMOL Molecular Graphics System</i>, version 0.99; <span class="NLM_publisher-name">SchrÃ¶dinger, LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PyMOL+Molecular+Graphics+System%2C+version+0.99%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casaluce, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Irannejad, H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Toward the discovery of novel anti-HIV drugs. Second-generation inhibitors of the cellular ATPase DDX3 with improved anti-HIV activity: synthesis, structure-activity relationship analysis, cytotoxicity studies, and target validation</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1371</span>â <span class="NLM_lpage">1389</span>, <span class="refDoi">Â DOI: 10.1002/cmdc.201100166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=10.1002%2Fcmdc.201100166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=21698775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsl2gtrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1371-1389&author=G.+Magaauthor=F.+Falchiauthor=M.+Radiauthor=L.+Bottaauthor=G.+Casaluceauthor=M.+Bernardiniauthor=H.+Irannejad&title=Toward+the+discovery+of+novel+anti-HIV+drugs.+Second-generation+inhibitors+of+the+cellular+ATPase+DDX3+with+improved+anti-HIV+activity%3A+synthesis%2C+structure-activity+relationship+analysis%2C+cytotoxicity+studies%2C+and+target+validation&doi=10.1002%2Fcmdc.201100166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Discovery of Novel Anti-HIV Drugs. Second-Generation Inhibitors of the Cellular ATPase DDX3 with Improved Anti-HIV Activity: Synthesis, Structure-Activity Relationship Analysis, Cytotoxicity Studies, and Target Validation</span></div><div class="casAuthors">Maga, Giovanni; Falchi, Federico; Radi, Marco; Botta, Lorenzo; Casaluce, Gianni; Bernardini, Martina; Irannejad, Hamid; Manetti, Fabrizio; Garbelli, Anna; Samuele, Alberta; Zanoli, Samantha; Este, Jose A.; Gonzalez, Emmanuel; Zucca, Elisa; Paolucci, Stefania; Baldanti, Fausto; De Rijck, Jan; Debyser, Zeger; Botta, Maurizio</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1371-1389</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A hit optimization protocol applied to the first nonnucleoside inhibitor of the ATPase activity of human DEAD-box RNA helicase DDX3 led to the design and synthesis of second-generation rhodanine derivs. with better inhibitory activity toward cellular DDX3 and HIV-1 replication.  Addnl. DDX3 inhibitors were identified among triazine compds.  Biol. data were rationalized in terms of structure-activity relationships and docking simulations.  Antiviral activity and cytotoxicity of selected DDX3 inhibitors are reported and discussed.  A thorough anal. confirmed human DDX3 as a valid anti-HIV target.  The compds. described herein represent a significant advance in the pursuit of novel drugs that target HIV-1 host cofactors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAxJL-NB7IFbVg90H21EOLACvtfcHk0ljoJWh_-wWLRg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsl2gtrg%253D&md5=6316aa57e7a2cb2ef5fdb676bc2ebdc7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100166%26sid%3Dliteratum%253Aachs%26aulast%3DMaga%26aufirst%3DG.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DRadi%26aufirst%3DM.%26aulast%3DBotta%26aufirst%3DL.%26aulast%3DCasaluce%26aufirst%3DG.%26aulast%3DBernardini%26aufirst%3DM.%26aulast%3DIrannejad%26aufirst%3DH.%26atitle%3DToward%2520the%2520discovery%2520of%2520novel%2520anti-HIV%2520drugs.%2520Second-generation%2520inhibitors%2520of%2520the%2520cellular%2520ATPase%2520DDX3%2520with%2520improved%2520anti-HIV%2520activity%253A%2520synthesis%252C%2520structure-activity%2520relationship%2520analysis%252C%2520cytotoxicity%2520studies%252C%2520and%2520target%2520validation%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D1371%26epage%3D1389%26doi%3D10.1002%2Fcmdc.201100166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i83"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01403">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_64566"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01403">10.1021/acs.jmedchem.8b01403</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Western blot experiments for the quantification of DDX3X in cell extracts, representative IC<sub>50</sub> curves, anti-WNV doseâresponse and cytotoxicity curves (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_001.csv">jm8b01403_si_001.csv (1.87 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01403/suppl_file/jm8b01403_si_002.pdf">jm8b01403_si_002.pdf (328.1 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01403&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01403%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-5%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01403" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a212a1e193cd0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
