
<html lang="en"     class="pb-page"  data-request-id="ea9f2eb5-9d85-4223-9ffc-867308195cb4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm400870h;requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2013.56.issue-23;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="High-Throughput Virtual Screening Identifies Novel N′-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors" /></meta><meta name="dc.Creator" content="Venkataswamy  Sorna" /></meta><meta name="dc.Creator" content="Emily R.  Theisen" /></meta><meta name="dc.Creator" content="Bret  Stephens" /></meta><meta name="dc.Creator" content="Steven L.  Warner" /></meta><meta name="dc.Creator" content="David J.  Bearss" /></meta><meta name="dc.Creator" content="Hariprasad  Vankayalapati" /></meta><meta name="dc.Creator" content="Sunil  Sharma" /></meta><meta name="dc.Description" content="Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple maligna..." /></meta><meta name="Description" content="Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple maligna..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 23, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm400870h" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm400870h" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm400870h" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm400870h" /></link>
        
    
    

<title>High-Throughput Virtual Screening Identifies Novel N′-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm400870h" /></meta><meta property="og:title" content="High-Throughput Virtual Screening Identifies Novel N′-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0013.jpeg" /></meta><meta property="og:description" content="Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies. Here we report a structure-based virtual screen of a compound library containing ∼2 million small molecular entities. Computational docking and scoring followed by biochemical screening led to the identification of a novel N′-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochemical IC50s in the 200–400 nM range. Hit-to-lead optimization and structure–activity relationship studies aided in the discovery of compound 12, with a Ki of 31 nM. Compound 12 is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer. Compound 12 may be used to probe LSD1’s biological role in these cancers." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm400870h"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm400870h">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm400870h&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm400870h&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm400870h&amp;href=/doi/10.1021/jm400870h" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 9496-9508</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm401249c" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm400924p" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">High-Throughput Virtual Screening Identifies Novel <i>N</i>′-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Venkataswamy++Sorna">Venkataswamy Sorna</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Emily+R.++Theisen">Emily R. Theisen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bret++Stephens">Bret Stephens</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven+L.++Warner">Steven L. Warner</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Bearss">David J. Bearss</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hariprasad++Vankayalapati">Hariprasad Vankayalapati</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sunil++Sharma">Sunil Sharma</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Center for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Pharmaceutics and Pharmaceutical Chemistry, College of Pharmacy, University of Utah, 301 Skaggs Hall, Salt Lake City, Utah 84112, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">School of Medicine, University of Utah, 30 North 1900 East, Salt Lake City, Utah 84132, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Division of Medical Oncology, Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: +1-(801)587-5559. Fax: +1-(801)585-0101. E-mail: <a href="/cdn-cgi/l/email-protection#0576706b6c692b766d64776864456d666c2b7071646d2b606170"><span class="__cf_email__" data-cfemail="87f4f2e9eeeba9f4efe6f5eae6c7efe4eea9f2f3e6efa9e2e3f2">[email protected]</span></a>. Address: Huntsman Cancer Institute, Room 3360, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm400870h&amp;href=/doi/10.1021%2Fjm400870h" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 9496–9508</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 17, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 June 2013</li><li><span class="item_label"><b>Published</b> online</span>23 November 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 December 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm400870h" title="DOI URL">https://doi.org/10.1021/jm400870h</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9496%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DVenkataswamy%2BSorna%252C%2BEmily%2BR.%2BTheisen%252C%2BBret%2BStephens%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D23%26contentID%3Djm400870h%26title%3DHigh-Throughput%2BVirtual%2BScreening%2BIdentifies%2BNovel%2BN%25E2%2580%25B2-%25281-Phenylethylidene%2529-benzohydrazides%2Bas%2BPotent%252C%2BSpecific%252C%2Band%2BReversible%2BLSD1%2BInhibitors%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9508%26publicationDate%3DDecember%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm400870h"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3951</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">111</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm400870h" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;High-Throughput Virtual Screening Identifies Novel N′-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Venkataswamy&quot;,&quot;last_name&quot;:&quot;Sorna&quot;},{&quot;first_name&quot;:&quot;Emily&quot;,&quot;last_name&quot;:&quot;R. Theisen&quot;},{&quot;first_name&quot;:&quot;Bret&quot;,&quot;last_name&quot;:&quot;Stephens&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;L. Warner&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Bearss&quot;},{&quot;first_name&quot;:&quot;Hariprasad&quot;,&quot;last_name&quot;:&quot;Vankayalapati&quot;},{&quot;first_name&quot;:&quot;Sunil&quot;,&quot;last_name&quot;:&quot;Sharma&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;9496-9508&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm400870h&quot;},&quot;abstract&quot;:&quot;Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies. Here we report a structure-based virtual screen of a compound library containing ∼2 million small molecular entities. Computational docking and scoring followed by biochemical screening led to the identification of a novel N′-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochemical IC50s in the 200–400 nM range. Hit-to-lead optimization and structure–activity relationship studies aided in the discovery of compound 12, with a Ki of 31 nM. Compound 12 is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer. Compound 12 may be used to probe LSD1’s biological role in these cancers.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400870h&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400870h" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400870h&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400870h" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400870h&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400870h" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm400870h&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400870h&amp;href=/doi/10.1021/jm400870h" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm400870h" /></input><a href="/doi/pdf/10.1021/jm400870h" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26pmid%3D24237195%26genre%3Darticle%26aulast%3DSorna%26date%3D2013%26atitle%3DHigh-Throughput%2BVirtual%2BScreening%2BIdentifies%2BNovel%2BN%25E2%2580%25B2-%25281-Phenylethylidene%2529-benzohydrazides%2Bas%2BPotent%252C%2BSpecific%252C%2Band%2BReversible%2BLSD1%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D23%26spage%3D9496%26epage%3D9508%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jmcmar.2013.56.issue-23/production/jmcmar.2013.56.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies. Here we report a structure-based virtual screen of a compound library containing ∼2 million small molecular entities. Computational docking and scoring followed by biochemical screening led to the identification of a novel <i>N</i>′-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochemical IC<sub>50</sub>s in the 200–400 nM range. Hit-to-lead optimization and structure–activity relationship studies aided in the discovery of compound <b>12</b>, with a <i>K</i><sub>i</sub> of 31 nM. Compound <b>12</b> is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer. Compound <b>12</b> may be used to probe LSD1’s biological role in these cancers.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Epigenetic disregulation contributes to the aberrant gene expression programs characteristic of cancer.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Transcriptional regulation through chromatin modification is reversible and dynamic such that enzymes implicated in the disregulation of chromatin represent a new class of protein targets for drug development. Various chromatin modifications mediate changes in gene expression including DNA methylation, posttranslational histone modifications, and nucleosome remodeling. The N-terminal tails of histones are subject to a variety of posttranslational modifications such as phosphorylation, acetylation, methylation, and ubiquitination. Certain modifications, particularly lysine acetylation and methylation, are important for regulating the local chromatin state and are often disregulated in cancer.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Histone methylation was believed to be an irreversible mark until the discovery of lysine-specific demethylase 1 (LSD1) in 2004.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> LSD1 catalyzes the oxidative demethylation of mono- and dimethylated histone H3 at lysine 4 (H3K4me1 and H3K4me2) and lysine 9 (H3K9me1 and H3K9me2) through a flavin adenine dinucleotide (FAD)-dependent amine oxidase reaction.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> While histone acetylation is associated with open chromatin and gene activation, the impact of histone methylation on the local chromatin state is more nuanced.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Using the substrates for LSD1 as an example, H3K4 methylation is generally associated with gene activation, whereas methylation of H3K9 is associated with transcriptional repression.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> In this manner, LSD1 can act either as a corepressor or coactivator depending on its substrate.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> Thus, LSD1 is a component of both repressive and activating complexes and the substrate specificity of LSD1 depends on which complex LSD1 interacts with. Repressive complexes containing histone deacetylases (HDACs), CtBP, CoREST, and BHC80 target LSD1 toward H3K4.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9-12)</a> In these complexes, HDACs remove the H3K9 acetyl mark, allowing LSD1 access to H3K4 methylation.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> The unmethylated histone tail will then be bound by BCH80 to maintain the repressive state.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> However, in complex with the estrogen or androgen receptor, LSD1 is targeted toward H3K9 at the promoters of target genes.<a onclick="showRef(event, 'ref4 ref10 ref15'); return false;" href="javascript:void(0);" class="ref ref4 ref10 ref15">(4, 10, 15)</a> Subsequent demethylation of H3K9 results in the activation of estrogen and androgen receptor target genes.<a onclick="showRef(event, 'ref4 ref16'); return false;" href="javascript:void(0);" class="ref ref4 ref16">(4, 16)</a></div><div class="NLM_p">LSD1 is a therapeutic target in cancer with overexpression observed in a variety of solid tumors, including neuroblastoma, breast, prostate, bladder, lung, liver, and colorectal tumors.<a onclick="showRef(event, 'ref16 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18 ref19 ref20">(16-20)</a> In many of these cases, LSD1 is reported as a corepressor with specificity for H3K4. Increased methylation at the H3K4 mark through either LSD1 knockdown or inhibition was shown to reactivate expression of tumor suppressor genes in breast, bladder, lung, and colorectal cancers.<a onclick="showRef(event, 'ref18 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref21">(18, 21)</a> In hormone-responsive cancer, association of LSD1 with the estrogen and androgen receptors led to increased proliferation.<a onclick="showRef(event, 'ref4 ref15'); return false;" href="javascript:void(0);" class="ref ref4 ref15">(4, 15)</a> LSD1 inhibition decreased expression of target genes in these models. Thus, inhibition of LSD1 is an effective strategy to reexpress epigenetically silenced tumor suppressor genes as well as downregulate important proliferative pathways in multiple cancer types. However, because of the complexity of factors regulating LSD1 function, the precise role LSD1 plays in cancer and how that role differs between cancers is not fully understood.</div><div class="NLM_p">Several LSD1 inhibitors are reported (Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a>), but they show poor selectivity and pharmacological properties making further exploration of LSD1 biology difficult. Monoamine oxidase (MAO) inhibitors such as tranylcypromine (TCP) and pargyline are known irreversible LSD1 inhibitors (<b>A</b> and <b>B</b>), and several reported inhibitors (<b>C</b>–<b>L</b>)<a onclick="showRef(event, 'ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25 ref26 ref27 ref28 ref29">(22-29)</a> are derivatives of these scaffolds with increased selectivity for LSD1. Peptide derivatives of pargyline have also been investigated, but delivery of peptide therapeutics to the nucleus remains an unsolved issue.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> Polyamine derivatives were also evaluated as LSD1 inhibitors (<b>M</b> and <b>N</b>), with compounds showing biochemical activity in the low micromolar range.<a onclick="showRef(event, 'ref21 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref21 ref32 ref33">(21, 32, 33)</a> Other reversible LSD1 inhibitors are reported to show selective activity against stem-like cancer cells (<b>O</b>)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and castration-resistant prostate cancer (<b>P</b>).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> In vivo efficacy is reported for the reversible inhibitor, namoline (<b>P</b>), however this was accompanied by significant weight loss indicative of off-target toxicity.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> In general, currently available small molecule LSD1 inhibitors display poor selectivity, low potency, or in vivo toxicity, limiting further interrogation of LSD1’s contribution to cancer at the organismal level. Identification of novel potent, selective, and reversible LSD1 inhibitors is essential to further elucidate LSD1’s role in cancer and identify whether or not reversible inhibition targeting LSD1 is a viable therapeutic strategy.</div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0010.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Representative Structures of Reported Classes of LSD1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>Where appropriate compound IDs and IC<sub>50</sub> values are indicated.</p></p></figure><div class="NLM_p last">Here, we report a structure-based virtual screen (VS) of a diverse compound library utilizing docking with Glide, ICM, GOLD scoring, and GOLD consensus rescoring energy calculations, which led to the identification of a novel <i>N</i>′-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors. On the basis of the initial hits, we rationally designed a series of small molecule LSD1 inhibitors which resulted in a selective and potent lead compound <b>12</b> which is a reversible and noncompetitive LSD1 inhibitor.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2 Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07388" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07388" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1 Docking Studies</h3><div class="NLM_p">The docking protocols used by both ICM and Glide SP were run with the adenosine phospate fragment of FAD, riboflavin fragment of FAD, and known LSD1 ligands from Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a> as positive controls within the Glide dl-400 1000-compound decoy set to confirm the accuracy of the docking protocols. Specifically, known LSD1 ligands were identified in the top 2% of the total decoy set. The structure-based VS was performed using the Glide docking module within the Schrödinger Suite 2011. The small molecule ligand library of 13 million compounds was first docked using Glide High Throughput Virtual Screen (HTVS), a method specifically proven to discard noticeable nonbinders with minimal computational time and then filtered for standard rule-of-five (RO5) criteria, medchem tractability based on physicochemical parameters in predicted in QikProp, and undesireable chemical features. The top 15% of compounds (∼2 million) from HTVS were then redocked with the more computationally expensive Glide standard precision (SP) scoring. This led to the selection of 0.5% (∼10000) of the top-ranked compounds by SP for subsequent screening using Glide extra-precision (XP) and ICM docking and scoring methods. These methods from the Schrödinger and Molsoft suites, respectively, are more resource intensive and used in our workflow to minimize false positives.</div><div class="NLM_p last">While molecular docking has proven a useful tool to quickly identify bioactive compounds, there are still problems with the accuracy and consistency of scoring functions in VS methods. Hence, we identified 121 compounds which scored either < −5.0 kcal/mol using Glide SP/XP or < −15.0 kcal/mol using ICM in addition to meeting certain physicochemical criteria, including solubility >50 μg/mL, permeability >50 nmol/s, and polar surface area (PSA) <120 Å<sup>2</sup> as determined by QikProp. In addition to the algorithm rankings, redundant compounds were removed to improve chemical diversity of the final selections and visual inspection of the docking results was used to evaluate binding mode, position, and orientation. Taken together, this methodology identified a set of 121 hits for further analysis. GOLD was used to rescore these hits. The GOLD consensus scoring and fitness functions produced similar compound rankings within the 121 hits to that of ICM and Glide scores, further supporting our hit selection process. Interestingly, compounds with hydroxyl moieties, hydrophobic electron withdrawing groups, and heterocycloalkyl groups were well represented in the initial docking experiments from all VS programs used.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2 Initial Hits</h3><div class="NLM_p">On the basis of the selection criteria discussed above, 121 structurally distinct compounds were procured and screened in the LSD1 biochemical assay. This identified a series of <i>N</i>′-(1-phenylethylidene)-benzohydrazides, which showed potent activity against LSD1. Biochemical assay results and docking scores for the series are reported in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Tables S1 and S2">Supporting Information Tables S1 and S2</a>. Out of the 121 in silico hits tested for LSD1 activity, compounds <b>1</b>–<b>5</b> showed biochemical activity in the 200–400 nM range. Compound <b>6</b> showed an IC<sub>50</sub> of 19 nM against LSD1. The docking poses determined from all three programs were predicted to be similar and show the protonated morpholin ring nitrogen of compound <b>6</b>, forming a favorable ionic interaction. Interestingly, the 2-hydroxyphenyl moiety of compound <b>6</b> extended deeper into the pocket as compared to compounds <b>1</b>–<b>5</b>. A representative binding mode is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. These initial hits support the utility of our VS methodology. Compounds <b>7</b>–<b>10</b> had similar core structures to compounds <b>1</b>–<b>6</b> and represented some of the chemical diversity present in the 121 hits. These compounds included substitution of the critical 2-hydroxylphenyl groups with biaryl naphthalene (<b>7</b>), an electron donating methoxyl group (<b>8</b>), absence of the hydroxyl group (<b>9</b>), and the introduction of a small hydrophobic methyl group with a lack of 2-hydroxyl (<b>10</b>). The additional 111 negative hits are reported in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S2">Supporting Information Table S2</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Commercially Available Highly-Ranked Hits from 121 Screened Compounds<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0008.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>Table a</sup><p class="last">Included are biochemical IC<sub>50</sub>s for compound <b>1</b>–<b>10</b>.</p></div></div><div></div></div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Mode of binding of compound <b>12</b> in complex with LSD1. The H-bonding interactions of compound <b>12</b> with LSD1 are depicted in dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The biochemical data showed compounds having 4-OH, 4-Br, or 3-Cl aryl substitutions on the benzohydrazides (compounds <b>1</b>, <b>4</b>, and <b>5</b>, respectively) had similar activity against LSD1 with IC<sub>50</sub>s of 218, 196 nM, and 333 nM, respectively. The 5-chloro-2-hydroxyl substituted derivative (<b>2</b>) and <i>N</i>′-(1-phenylethylidene)-benzohydrazide core (<b>3</b>) are well tolerated, with IC<sub>50</sub>s of 275 and 291 nM, respectively. The 3-substituted sulfonyl functional group on the arylhydrazide moiety of compound <b>6</b> improved biochemical activity10-fold with an IC<sub>50</sub> of 19 nM. The exchange of the 2-hydroxyphenyl moiety with naphthalene in compound <b>7</b> impaired biochemical activity with an IC<sub>50</sub> > 10 μM. Compounds <b>8</b>–<b>10</b> were representative examples of the remaining negative hits and showed no biochemical activity in the LSD1 assay. These biochemical results suggested further optimization of compounds <b>1</b>–<b>6</b> to explore the structure–activity relationship of the <i>N</i>′-(1-phenylethylidene)-benzohydrazide series and identified potential lead compounds shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> for further screening against LSD1.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Synthesized Compounds and Their Biochemical Activity against LSD1</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0009.gif" alt="" id="fx2" /></img><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.3 Structure–Activity Relationship of Initial Hits and Compound Optimization</h3><div class="NLM_p last">Compounds <b>11</b>–<b>22</b> were subsequently synthesized in-house, and their chemical structures and LSD1 inhibition are reported in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. We utilized the <i>N</i>′-(1-phenylethylidene)benzohydrazide core scaffolds from hit compounds <b>3</b> and <b>6</b> for further optimization and SAR in order to increase metabolic stability over the benzylidenebenzohydrazide core of compounds <b>1</b>, <b>2</b>, <b>4</b>, and <b>5</b>. In compound <b>11</b>, removing the sulfonyl moiety and retaining the 5-chloro-2-hydroxyphenyl group from compound <b>2</b> maintained a biochemically active compound (IC<sub>50</sub> = 128 nM) with a 10-fold reduction in potency as compared to <b>6</b>. In the case of <b>12</b>, reintroduction of the sulfonyl functional group and inclusion of the 5-chloro-2-hydroxyphenyl moiety on the <i>N</i>′-(1-phenylethylidene)benzohydrazide inhibited LSD1 with an IC<sub>50</sub> of 13 nM, which is comparable to compound <b>6</b>. A representative binding mode of <b>12</b> generated from ICM is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>. Exchanging 2-hydroxyl with 2-chloro in compound <b>13</b> resulted in a complete loss of LSD1 activity, emphasizing the importance of the 2-hydroxyl group. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>, the 2-hydroxyl moiety participates in hydrogen bonding and loss of this interaction likely impairs ligand binding. Replacement of the morpholin with an <i>N</i>,<i>N</i>-dimethyl sulfonamide (<b>14</b>) maintained LSD1 potency with an IC<sub>50</sub> of 14 nM, highlighting the significance of the core sulfonamide. On the basis of Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>, we hypothesized both <i>N</i>′-(1-phenylethylidene)benzohydrazide and <i>N</i>′-(1-phenylpropylidene)benzohydrazide were likely accommodated, however the <i>N</i>′-(1-phenylpropylidene)benzohydrazide derivative (<b>15</b>) showed no LSD1 activity (IC<sub>50</sub> > 3 μM). Generally, compounds <b>15</b>–<b>22</b> showed no activity against LSD1 in the biochemical assay. An additional polar hydroxyl group did not improve LSD1 activity nor did disubstitution of the aryl ring with strong electron withdrawing groups, like −F. Further, introduction of various heterorings were not tolerated. Because of the favorable biochemical activity of compound <b>12</b>, we utilized it to further investigate the mechanism of inhibition of this series of LSD1 inhibitors.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">2.4 Scaffold Novelty</h3><div class="NLM_p last">Many different classes of LSD1 inhibitors are already reported (Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a>), with some compounds showing nanomolar potency. We wanted to evaluate the similarity of the <i>N</i>′-(1-phenylethylidene)-benzohydrazide scaffold to previously reported inhibitors. The calculated Tanimoto similarity coefficients are reported in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S3">Supporting Information Table S3</a>. Typically, 0.7 is used as a cutoff, with >0.7 indicating similar compounds. Generally, the similarity coefficients calculated are all <0.4, ranging from 0.11 to 0.39, demonstrating that compound <b>12</b> is structurally distinct from previously reported LSD1 inhibitors. The most dissimilar compounds were the polyamine derivatives (<b>M</b> and <b>N</b>) and the reversible inhibitor namoline (<b>P</b>), with similarity scores of 0.11. The most similar compounds were various derivatives of tranylcypromine (<b>F</b>, <b>J</b>, and <b>K</b>), with similarity scores of 0.35, 0.36, and 0.39, respectively. The similarity score for the other reversible inhibitor reported, <b>O</b>, was calculated to be 0.32. These results corroborated the novelty of the <i>N</i>′-(1-phenylethylidene)-benzohydrazide series, represented in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10">2.5 Lead Compound <b>12</b> Is Specific and Reversible</h3><div class="NLM_p">The specificity of the compounds <b>1</b> and <b>12</b> were tested in MAO A and MAO B biochemical assays (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). In this assay, the MAO inhibitor TCP exhibited activity against both MAO A and B, with an IC<sub>50</sub> of 2.1 and 3.6 μM, respectively. Compound <b>1</b> was active against MAO B, with an IC<sub>50</sub> of 1.3 μM, but showed no activity against MAO A (IC<sub>50</sub> > 300 μM). In contrast, compound <b>12</b> did not exhibit activity against either MAO enzyme up to 300 μM. We further screened <b>12</b> against <span class="smallcaps smallerCapital">d</span>-lactate dehydrogenase, glucose oxidase, a panel of cytochrome P450s (CYP), and human ether-à-go-go (hERG), with IC<sub>50</sub> values summarized in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. Using an IC<sub>50</sub> of 3 μM as a cutoff, <b>12</b> showed low activity against CYP3A4, with an IC<sub>50</sub> of 2.96 μM. The inhibition data for the off-target assays are reported in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information (Table S4)">Supporting Information (Table S4)</a>.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. MAO activity of select compounds. (A) The MAO inhibitor TCP inhibited MAO A with an IC<sub>50</sub> of 2.1 μM, while compounds <b>1</b> and <b>12</b> inhibited MAO A with IC<sub>50</sub>s of 88 and >300 μM, respectively. (B) Compound <b>12</b> did not inhibit MAO B at the concentrations tested (IC<sub>50</sub> > 300 μM). However, compound <b>1</b> exhibited significant activity against MAO B, with an IC<sub>50</sub> of 1.3 μM, similar 3.6 μM for TCP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Off-Target Panel for Compound <b>12</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">enzyme</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><span class="smallcaps smallerCapital">d</span>-lactate dehydrogenase</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">glucose oxidase</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP1A2</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP2C19</td><td class="colsep0 rowsep0" align="char" char=".">9.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP2C9</td><td class="colsep0 rowsep0" align="char" char=".">8.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP2D6</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP3A4</td><td class="colsep0 rowsep0" align="char" char=".">2.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hERG</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr></tbody></table></div></div><div class="NLM_p">We performed a jump dilution to assay the reversibility of compound <b>12</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). In this assay, LSD1 was incubated with 10× the biochemical IC<sub>50</sub> of <b>12</b> for 1 h and then diluted 100-fold into the assay. The irreversible inhibitor TCP was used as a positive control, and the chemically similar but inactive compound <b>13</b> was used as a negative control. TCP incubation resulted in complete inactivation of the enzyme, which was not recovered once diluted into the assay buffer. When LSD1 was diluted into assay buffer after incubation with compound <b>12</b>, its activity returned with only 14.4 ± 3% inhibition. When the drug concentration was held constant at 200 nM through the dilution, activity was inhibited 65.7 ± 5%, suggesting compound <b>12</b> is a reversible inhibitor. Compound <b>13</b> showed no activity in this assay.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>12</b> reversibly inhibits the activity of LSD1. Dilution of compound <b>12</b>, but not of the covalently binding inhibitor TCP, results in recovery of LSD1 activity. Compound <b>13</b> is inactive.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">2.6 Compound <b>12</b> Is a Noncompetitive Inhibitor</h3><div class="NLM_p">We further performed differential scanning fluorimetry (DSF) to compare the melting profile of LSD1 bound by compound <b>12</b> with the LSD1-TCP complex. DSF uses SYPRO Orange, which preferentially fluoresces in the amphiphilic molten globule state of an unfolding protein. LSD1 was incubated either with compound <b>12</b>, <b>13</b>, or TCP for 30 min, and then DSF was performed. The raw melt curves were smoothed and fit to a Boltzmann curve for unfolding using Applied Biosystem’s Protein Thermal Shift (PTS) software. Derivatives of the smoothed curves were plotted to generate the derivative plot shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>. Both Boltzmann and derivative melting temperatures (<i>T</i><sub>m</sub>) were determined for each condition and are shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. LSD1 alone showed a multiple-phase unfolding with a Boltzmann <i>T</i><sub>m</sub> of 43.28 ± 0.45 °C and derivative <i>T</i><sub>m</sub>s at 40.49 ± 0.81 °C and 46.31 ± 1.1 °C. Compound <b>13</b> exhibited no effect on LSD1.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Derivative melt curves of LSD1 in the presence of DMSO, compound <b>12</b>, compound <b>13</b>, and TCP. LSD1 has a complex multiphase melt. Compound <b>12</b> and TCP induce changes in LSD1’s melt profile in distinct manners. Compound <b>13</b> shows no difference from DMSO.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Melting Temperatures as Determined by DSF</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">treatment</th><th class="colsep0 rowsep0" align="center" char="±">Boltzmann <i>T</i><sub>m</sub> ± SD (°C)</th><th class="colsep0 rowsep0" align="center" char="±">first derivative <i>T</i><sub>m</sub> ± SD (°C)</th><th class="colsep0 rowsep0" align="center">second derivative <i>T</i><sub>m</sub> ± SD (°C)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DMSO</td><td class="colsep0 rowsep0" align="char" char="±">43.28 ± 0.45</td><td class="colsep0 rowsep0" align="char" char="±">40.49 ± 0.81</td><td class="colsep0 rowsep0" align="center">46.31 ± 1.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15 μM <b>12</b></td><td class="colsep0 rowsep0" align="char" char="±">44.60 ± 0.47</td><td class="colsep0 rowsep0" align="char" char="±">45.35 ± 1.27</td><td class="colsep0 rowsep0" align="center">N/A</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15 μM <b>13</b></td><td class="colsep0 rowsep0" align="char" char="±">43.29 ± 0.67</td><td class="colsep0 rowsep0" align="char" char="±">40.76 ± 0.58</td><td class="colsep0 rowsep0" align="center">46.69 ± 0.76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">220 μM TCP</td><td class="colsep0 rowsep0" align="char" char="±">51.98 ± 1.18</td><td class="colsep0 rowsep0" align="char" char="±">55.60 ± 0.32</td><td class="colsep0 rowsep0" align="center">N/A</td></tr></tbody></table></div></div><div class="NLM_p">TCP stabilized LSD1 with a clear rightward shift of the derivative curve. Additional inspection of the data showed a long period of slow melting, followed by a sharp transition between 50 and 60 °C. The Boltzmann <i>T</i><sub>m</sub> was determined as 51.98 ± 1.2 °C, and the derivative method placed the <i>T</i><sub>m</sub> during the rapid melt phase at 55.60 ± 0.32 °C. Compound <b>12</b> shifted the <i>T</i><sub>m</sub> in a subtle but statistically significant fashion and constrained the melting dynamics of LSD1 to classical two-state unfolding. The Boltzmann <i>T</i><sub>m</sub> for compound <b>12</b> was 44.60 ± 0.47 °C with a derivative <i>T</i><sub>m</sub> of 45.38 ± 1.3 °C. This suggests compound <b>12</b> binds LSD1, changes its solution dynamics in a manner distinct from TCP, and shows small change in <i>T</i><sub>m</sub>.</div><div class="NLM_p">We used enzyme kinetics to investigate the mechanism of action of compound <b>12</b>. For each drug concentration, the initial velocity of the biochemical assay was plotted with respect to enzyme substrate in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>. Our calculated <i>K</i><sub>m</sub> of 1.3 ± 0.2 μM is similar to previous literature reports for the dimethylated K4 N-terminal H3 peptide.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Curves were analyzed individually using the Michaelis–Menten suite or globally using the Enzyme Inhibition suite in GraphPad Prism 5, with the results of the analysis summarized in Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>. The global fit to a mixed model inhibition gave a <i>K</i><sub>i</sub> of 31 ± 12 nM and an α value of 1.3, which is most indicative of noncompetitive inhibition. This is consistent with the observed drop in <i>v</i><sub>max</sub> and little change in <i>K</i><sub>m</sub> with increasing concentration of <b>12</b>. (For a comparison of competitive, noncompetitive, and uncompetitive inhibition global curve fits, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S5">Supporting Information Table S5</a>.) Individually fitting each curve and using this data to determine a <i>K</i><sub>i</sub> for compound <b>12</b> produced a <i>K</i><sub>i</sub> of 34 ± 1.9 nM, which correlated well with the global fit.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. LSD1 kinetics with multiple concentrations of compound <b>12</b>. Compound <b>12</b> causes a decrease in <i>v</i><sub>max</sub> with no change in <i>K</i><sub>m</sub>, characteristic of noncompetitive inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Summary of Michaelis–Menten Curve Fits</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">fit</th><th class="colsep0 rowsep0" align="center"><i>v</i><sub>max</sub> (F/s) ± std error</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>m</sub> (μM) ± std error</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>i</sub> (nM) ± std error</th><th class="colsep0 rowsep0" align="center">α ± std error</th><th class="colsep0 rowsep0" align="center"><i>R</i><sup>2</sup></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">global: mixed </td><td class="colsep0 rowsep0" align="center">687 ± 11.43</td><td class="colsep0 rowsep0" align="center">1.284 ± 0.1048</td><td class="colsep0 rowsep0" align="center">30.69 ± 12.4</td><td class="colsep0 rowsep0" align="center">1.333 ± 0.64</td><td class="colsep0 rowsep0" align="center">0.9241</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="6" align="left">Model Inhibition</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DMSO</td><td class="colsep0 rowsep0" align="center">697.5 ± 26.32</td><td class="colsep0 rowsep0" align="center">1.405 ± 0.2514</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">0.9275</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1 nM</td><td class="colsep0 rowsep0" align="center">649.4 ± 30.09</td><td class="colsep0 rowsep0" align="center">1.140 ± 0.2662</td><td class="colsep0 rowsep0" align="center">13.50</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">0.8807</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3 nM</td><td class="colsep0 rowsep0" align="center">642.9 ± 27.07</td><td class="colsep0 rowsep0" align="center">1.393 ± 0.2787</td><td class="colsep0 rowsep0" align="center">35.32</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">0.9082</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10 nM</td><td class="colsep0 rowsep0" align="center">531.1 ± 19.53</td><td class="colsep0 rowsep0" align="center">1.508 ± 0.2573</td><td class="colsep0 rowsep0" align="center">31.92</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">0.9308</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">30 nM</td><td class="colsep0 rowsep0" align="center">376.4 ± 12.25</td><td class="colsep0 rowsep0" align="center">1.108 ± 0.1832</td><td class="colsep0 rowsep0" align="center">35.17</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">0.9326</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">100 nM</td><td class="colsep0 rowsep0" align="center">246.5 ± 19.57</td><td class="colsep0 rowsep0" align="center">2.267 ± 0.7461</td><td class="colsep0 rowsep0" align="center">54.66</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">0.7803</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>k</i><sub>i</sub></td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">34.11 ± 6.54</td><td class="colsep0 rowsep0" align="center">N/A</td><td class="colsep0 rowsep0" align="center">N/A</td></tr></tbody></table></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">2.7 Compound <b>12</b> Activity in in Vitro Assays</h3><div class="NLM_p">Compound <b>12</b> was used to evaluate sensitivity in a panel of cancer cell lines (Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>). Cell line sensitivity to compound <b>12</b> varied by one log in a cell viability assay, with EC<sub>50</sub> values ranging from 300 nM to 3 μM. Nine of the 17 cell lines tested were sensitive to compound <b>12</b>, with an EC<sub>50</sub> < 1 μM. Endometrial, breast, colorectal, and pancreatic cancers were represented in the sensitive lines, consistent with a role for LSD1 in multiple cancers. EC<sub>50</sub> values were determined in T-47D breast cancer cells to evaluate the correlation between cell sensitivity and biochemical activity against LSD1 (Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>). With few exceptions, it was observed that T-47D cells were sensitive to test compounds that were active in the LSD1 biochemical assay. For compounds inactive in the biochemical assay, cellular sensitivity was more variable (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). Compounds with low EC<sub>50</sub>s against T-47D cells but no biochemical activity may possess uncharacterized cytotoxic off-target activity. The only active biochemical compounds without T-47D activity were compounds <b>1</b> and <b>14</b>. These compounds may show decreased permeability or solubility in the cell-based assay format, although compound <b>14</b> still shows activity near 1 μM. Importantly, when the hydroxyl of compound <b>12</b> was substituted with a chlorine (compound <b>13</b>), in vitro activity was lost, confirming the importance of the hydroxyl group in that position for both biochemical and cellular activity.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Compound <b>12</b> Inhibits Proliferation in Several Cell Lines in Vitro</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">cancer type</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AN3 Ca</td><td class="colsep0 rowsep0" align="char" char=".">0.356</td><td class="colsep0 rowsep0" align="left">endometrial</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BT-20</td><td class="colsep0 rowsep0" align="char" char=".">0.489</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BT-549</td><td class="colsep0 rowsep0" align="char" char=".">1.010</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HCT 116</td><td class="colsep0 rowsep0" align="char" char=".">0.614</td><td class="colsep0 rowsep0" align="left">colorectal</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HER218</td><td class="colsep0 rowsep0" align="char" char=".">0.612</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Hs-578-T</td><td class="colsep0 rowsep0" align="char" char=".">1.700</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HT29</td><td class="colsep0 rowsep0" align="char" char=".">0.429</td><td class="colsep0 rowsep0" align="left">colorectal</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCF-7</td><td class="colsep0 rowsep0" align="char" char=".">0.637</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB-231</td><td class="colsep0 rowsep0" align="char" char=".">1.040</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB-435</td><td class="colsep0 rowsep0" align="char" char=".">1.440</td><td class="colsep0 rowsep0" align="left">melanoma</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB-468</td><td class="colsep0 rowsep0" align="char" char=".">2.730</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MIA PaCa-2</td><td class="colsep0 rowsep0" align="char" char=".">0.468</td><td class="colsep0 rowsep0" align="left">pancreatic</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PANC-1</td><td class="colsep0 rowsep0" align="char" char=".">1.104</td><td class="colsep0 rowsep0" align="left">pancreatic</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PC-3</td><td class="colsep0 rowsep0" align="char" char=".">2.160</td><td class="colsep0 rowsep0" align="left">prostate</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SK-N-MC</td><td class="colsep0 rowsep0" align="char" char=".">0.329</td><td class="colsep0 rowsep0" align="left">sarcoma</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">T-47D</td><td class="colsep0 rowsep0" align="char" char=".">0.649</td><td class="colsep0 rowsep0" align="left">breast</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">U87</td><td class="colsep0 rowsep0" align="char" char=".">1.160</td><td class="colsep0 rowsep0" align="left">glioblastoma</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. In Vitro Growth Inhibition of Compound Panel in T-47D Cells</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">2.700</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char=".">0.821</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">0.971</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">0.096</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">0.615</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">0.524</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="char" char=".">0.352</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="char" char=".">0.649</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="char" char=".">1.700</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">1.375</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="char" char=".">0.352</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char=".">0.565</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="char" char=".">0.270</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="char" char=".">0.616</td></tr></tbody></table></div></div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds with biochemical activity against LSD1 show in vitro EC<sub>50</sub>s clustered near 1 μM. Compounds without LSD1 activity show a wide range of in vitro efficacy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition to cell viability, we assayed compound <b>12</b> for its effect on histone methylation in an androgen-sensitive prostate cancer cell line, VCaP. VCaP cells were treated for 24 h with vehicle or 0.1, 1, or 10 μM of compound <b>12</b> for 24 h. We focused specifically on the H3K9me2 mark. H3K9me2 is a target for LSD1 in complex with the androgen receptor in prostate cancer.<a onclick="showRef(event, 'ref4 ref15'); return false;" href="javascript:void(0);" class="ref ref4 ref15">(4, 15)</a> Demethylation of this mark activates transcription of androgen receptor target genes.<a onclick="showRef(event, 'ref4 ref15'); return false;" href="javascript:void(0);" class="ref ref4 ref15">(4, 15)</a> An increase in H3K9me2 is observed at 24 h with both 1 and 10 μM of treatment with <b>12</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). This suggests the antiproliferative effects of compound <b>12</b> are on-target and related to changes in histone methylation mediated through reversible LSD1 inhibition.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. VCaP cells treated with compound <b>12</b> show a dose-dependent increase in H3K9 dimethylation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">3 Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We used a virtual screen (VS) methodology with a custom compound library to evaluate the chemical space outside of previously reported LSD1 inhibitors. All computational methods utilized the crystal structure of LSD1 in complex with an FAD-tranylcypromine adduct (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z5U">2Z5U</a>).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The virtual small molecule screening library was curated from publicly available vendor libraries, totaling ∼13 million compounds. We then performed structure-based VS using Glide in HTVS mode to weed out noticeable nonbinders along with custom filters to narrow the library to ∼2 million compounds having drug-like properties and chemical diversity. Filters included medchem tractability, based on physicochemical parameters calculated in QikProp, and undesirable chemical features in addition to rule-of-five-based parameters (<5 H-bond donors, <8 H-bond acceptors, CLogP < 5, and molecular weight <500). This custom protocol increases the probability that hits from the top 0.5% of compounds from subsequent Glide SP docking will display favorable properties for later lead optimization and lead compound development. By cross-comparing results from three different docking algorithms (ICM, Glide XP, and GOLD), this study identified 121 initial hits that were procured and screened against LSD1 in a biochemical assay. Twelve novel compounds were subsequently synthesized in-house based on the SAR of the initial positive hits (<b>1</b>–<b>6</b>) to further elucidate the SAR, optimize drug-like properties, and increase potency. Biochemical activity for compounds <b>1</b>–<b>21</b> correlated well with in vitro activity in a breast cancer cell line, T-47D. Generally, compounds which were active in the biochemical assay showed an EC<sub>50</sub> in T-47Ds near or below 1 μM, suggesting a similar and consistent mechanism of action. Compounds with poor biochemical activity showed a range of activity in the cell viability assay, which indicates some of the compounds inactive against LSD1 may have uncharacterized off-target toxicities in vitro.</div><div class="NLM_p">Ultimately, our lead optimization strategy successfully identified a series of compounds more potent and specific than other reported LSD1 inhibitors. Tanimoto similarity scores are reported (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S3">Supporting Information Table S3</a>) comparing compound <b>12</b> against previously reported LSD1 inhibitors. The low range of similarity scores (0.11–0.39) supports these compounds as a novel class of LSD1 inhibitors. The in-house-synthesized compounds <b>11</b>, <b>12</b>, and <b>14</b> showed biochemical IC<sub>50</sub>s between ∼10 and 300 nM against LSD1 compared to 39.7 μM for TCP, which forms a noncovalent adduct with the FAD. The novelty of the scaffold may contribute to the improved specificity profile against the MAOs after lead optimization as compared to those reported for other compounds.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> FAD is bound in LSD1 by a Rossman fold in a manner homologous to MAO A and B. Given the high structural homology of LSD1 to the monoamine oxidase family of enzymes (17.6% for both MAO A and B),<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> these are likely off-target hits for LSD1 inhibitors. TCP and pargyline derivatives targeting the H3-binding cavity of LSD1 often show some activity against either MAO A or MAO B, limiting their use in preclinical studies. Optimization from compound <b>1</b> to compound <b>12</b> improved the specificity profile where the initial hit compound <b>1</b> was more active against the MAO B and compound <b>12</b> displayed a favorable off-target profile with minimal activity against both MAO A and B. Compound <b>12</b> was also selective over other flavoenzymes D-LDH and GO as well as against cytochrome P450s and hERG.</div><div class="NLM_p">We predicted that our lead optimization strategy would select reversible inhibitors of LSD1. Subsequent jump dilution experiments confirmed the reversibility of compound <b>12</b>, inactivity of compound <b>13</b>, and the irreversible mechanism of action of TCP. DSF analysis was performed to further probe the physical effect of compound <b>12</b> as compared to irreversible inhibition. In these experiments, LSD1 alone and in the presence of <b>13</b> shows a multiphase melting curve (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). This may indicate either two domains which melt at different temperatures or two populations of conformers with different thermal stability. The addition of compound <b>12</b> shows subtle changes in the <i>T</i><sub>m</sub>, with an effect on the melt profile evident in the derivative curve. Here, compound <b>12</b> appears to constrain LSD1’s derivative profile to a Boltzmann distribution indicative of a classical two-state unfolding. Irreversible inhibition with TCP shows a long period of slow “melting” followed by a sharp melt between 50 and 60 °C. We conclude compound <b>12</b> alters the solution dynamics of LSD1 in a manner distinct from TCP.</div><div class="NLM_p">Kinetic analysis was used to elucidate the reversible mechanism of action of our inhibitor series and to determine a <i>K</i><sub>i</sub> for compound <b>12</b>. Michaelis–Menten plots were generated across five inhibitor concentrations and seven substrate concentrations. A global fit of the data showed that noncompetitive inhibition best described compound <b>12</b>’s mechanism of action. This is corroborated by the decreased <i>v</i><sub>max</sub> and unchanged <i>K</i><sub>m</sub> observed when the individual curves for each inhibitor are fit independently. The <i>K</i><sub>i</sub> of <b>12</b> was near 30 nM. Compound <b>12</b> does not interfere with the binding of the N-terminal H3 peptide so transcription factors with N-terminal sequences homologous to histone H3 that have been shown to recruit LSD1, such as SNAI1 and Gfi-1, may still be able to recruit an inactive LSD1 via their SNAG domains.<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39-41)</a> However, if the conformational dynamics of LSD1 are constrained by compound <b>12</b>, other key protein–protein interactions which regulate LSD1’s activity may be disrupted. Further studies are required to determine the effect of compound <b>12</b> on the protein–protein interactions which guide LSD1’s biological function.</div><div class="NLM_p">Compound <b>12</b> shows cellular activity against several cancer cell lines including endometrial, breast, colorectal, pancreatic, and prostate cancer. We further screened compound <b>12</b> for its effects on H3K9 dimethylation in the VCaP prostate cancer cell line. The H3K9me2 mark is a target for LSD1 in prostate cancer when in complex with the androgen receptor.<a onclick="showRef(event, 'ref4 ref15'); return false;" href="javascript:void(0);" class="ref ref4 ref15">(4, 15)</a> Compound <b>12</b> showed a dose-dependent increase in the H3K9me2 mark after 24 h of treatment at both 1 and 10 μM. This result suggests compound <b>12</b> shows on-target activity in cell lines as well. The fact that significant changes in histone methylation are observed at 1 μM may be the reason for the EC<sub>50</sub> cluster around 1 μM in the cell viability assay for compounds with potent LSD1 biochemical activity. Further studies are needed to determine the exact mechanism of action of compound <b>12</b> in different cell lines.</div><div class="NLM_p last">Additional studies with compound <b>12</b> are currently underway to better characterize the binding mode, physicochemical attributes, and mechanism of action of these compounds including crystallographic, pharmacokinetic, and pharmacological studies. Importantly, as LSD1 activity often requires complexation with other proteins, like Co-REST, knowing the effects of inhibition on both enzymatic activity as well as complex stability is necessary to understand the pharmacology of reversible, noncompetitive inhibition with our compound series. The results presented so far lead us to conclude that these compounds may be considered for future preclinical studies. Efforts are underway to develop additional analogues with ideal physicochemical properties for consideration in in vivo pharmacokinetic and pharmacodynamic studies.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">4 Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55485" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55485" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we performed a structure-based VS of ∼2 million diverse and preprocessed compounds from a library developed in-house with the goal of identifying a novel series of LSD1 inhibitors. A novel <i>N</i>′-(1-phenylethylidene)-benzohydrazide series was identified. Optimization and exploration of the SAR were performed using both virtual and biochemical techniques. Biochemical analysis shows a specific, potent, and reversible lead compound (<b>12</b>) to take into further crystallographic, mechanistic, and pharmacological studies. These results support structure-based approaches as valid starting points for lead optimization strategies.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">5 Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75541" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75541" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21">5.1 Computational Methods</h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22">5.1.1 Structure-Based Virtual Screen</h4><div class="NLM_p">All computational studies employed PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z5U">2Z5U</a> for the structural coordinates of LSD1.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Virtual screening methods from Glide, ICM, and GOLD programs were used. The protein structure was prepared by 3D protonation, deletion of water molecules, and energy minimization using the ICM force field and distance-dependent dielectric potential with an RMS gradient of 0.1; heavy atoms in the protein were kept fixed, and histidine residues were considered as neutral. VS scoring calculations utilized default parameters unless explicitly specified otherwise. PocketFinder (ICM) and SiteMap (Schroedinger) were used to define the ligand-binding site for docking studies. In both cases, the PocketFinder and SiteMap predicted ligand site is located near both the substrate and FAD-binding pockets. Default parameters in ICM include a docking site as a rectagular box with a grid spacing of 0.5 Å centered at the ligand binding site as defined by PocketFinder. A threshold set to retain 2% of the ligands along with a threshold scores of −32, maximum ligand size of 500 molecular weight, H-bond donors of 5, H-bond acceptors of 10, and torsions of 10 were used. For Glide, the default parameters similarly included the docking site as a 12 Å box centered on the geometrical center of the SiteMap-defined ligand binding site, with the ligand internal energy offset option turned on with the top 10 ranked poses for each ligand retained for scoring. Energy grids representing the active site (van der Waals, H-bonding, electrostatics, and hydrophobic interaction) were calculated with a 0.5 Å grid spacing. Confirmation of the accuracy and efficiency of the applied docking protocol used a decoy set with the adenosine phosphate fragment of FAD, the riboflavin fragment of FAD, and known LSD1 inhibitors (Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a>) as positive controls within the Glide dl-400 1000-compound decoy set provided in the Schroedinger Suite.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> We employed the structure-based VS to identify new small molecules which target this site on LSD1. Two separate docking runs were carried out with the ICM and Glide SP docking programs. The decoys with no valid poses after minimization were excluded from RMSD-score analysis but included in other evaluations as bad poses (GlideScore or Emodel = 10000).</div><div class="NLM_p last">The compound database was prepared using Ligprep 2.1.23 of the Schrodinger Suite and ICM’s inbuilt preparation of three-dimensional (3D) ligands such that physiologically relevant protonation states were used. Prepared ligands were then docked against LSD1. Two rounds of VS, including HTVS and standard precision (SP) docking, were adopted. The top 10000 compounds ranked by Glide HTVS followed by SP were stored and submitted for additional docking experiments using Glide XP and ICM. GOLD was used only for rescoring. Specifically, the final set of 121 hits was selected based on ICM and Glide SP/XP scores as well as pharmaceutical properties predicted in QikProp, and individual compounds were visually inspected to check the docking poses and interactions between ligands and LSD1. To filter out redundant compounds, we used ICM Molcart with search criteria including compound ID, structure, and SMILES string to identify and discard the duplicates. Rescoring was performed on a minimized 121 top-ranked poses (selected from Glide SP and ICM) in Glide XP and GOLD using the “refine and do not dock” option. Finally, 121 compounds were purchased and screened in an LSD1 biochemical assay. The details of comparative docking and hit selection from ICM, Glide, and GOLD are available in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information (Section S1)">Supporting Information (Section S1)</a>.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23">5.1.2 Tanimoto Similarity Coefficient Calculation</h4><div class="NLM_p last">Tanimoto similarity coefficients were calculated using the Molcart module of ICM.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24">5.2 Chemistry</h3><div class="NLM_p">All reagents and solvents were purchased from commercial sources and used without further purification. Solvents were of analytical or anhydrous grade (Sigma-Aldrich). Reactions were monitored by HPLC. Reverse phase preparative HPLC was performed using a preparative HPLC system. <sup>1</sup>H NMR spectra were recorded on a Varian Unity 400 instrument. Chemical shifts (δ) are reported in ppm downfield from solvent references. Mass spectra were obtained on a Finnegan LCQ Duo LCMS ion trap electrospray ionization (ESI) mass spectrometer. The general reaction scheme is depicted in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>, with specific reaction schemes for compounds <b>11</b>–<b>21</b> given in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information (Figure S3)">Supporting Information (Figure S3)</a>. The purity of the synthesized compounds was determined by LC-MS analysis and was confirmed to be >95% purity for all biologically tested compounds.</div><figure id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0011.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedure for the Synthesis of <i>N</i>′-(1-Phenylethylidene)-benzohydrazides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) THF, K<sub>2</sub>CO<sub>3</sub>, RT, 1 h; (b) conc H<sub>2</sub>SO<sub>4</sub>, CH<sub>3</sub>OH, 65 °C, 12 h; (c) hydrazine, CH<sub>3</sub>OH, reflux 12 h; (d) AcOH, CH<sub>3</sub>OH, MW, 120 °C, 30 min.</p></p></figure><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">5.2.1 (<i>E</i>)-<i>N</i>′-(1-(5-Chloro-2-hydroxyphenyl)ethylidene)benzohydrazide (<b>11</b>)</h4><div class="NLM_p last">1-(5-Chloro-2-hydroxyphenyl) ethanone <b>23</b> (100 mg, 0.586 mmol) and benzohydrazide (<b>24</b>) (80 mg, 0.586 mmol) was dissolved in methanol (4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf (2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>), affording the title compound <b>11</b> as a white solid (90 mg, 53%). <sup>1</sup>H NMR (400 MHz, DMSO): δ 7.95 (m, 2H), 7.67–7.62 (m, 2H), 7.56 (m, 2H), 7.35 (dd, 1H, <i>J</i> = 2.4 and 8.8 Hz), 6.95 (d, 1H, <i>J</i> = 8.4 Hz), 3.35 (s, 3H). ESI-MS: 289.0 [M + H]<sup>+</sup>. LC-MS purity of compound <b>11</b> was found to be >95%.</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27">5.2.2 (<i>E</i>)-<i>N</i>′-(1-(5-Chloro-2-hydroxyphenyl) ethylidene)-3-(morpholinosulfonyl) Benzohydrazide (<b>12</b>)</h4><div class="NLM_p">The methyl 3-(morpholinosulfonyl) benzoate <b>25</b> was prepared in two steps. The 3-(chlorosulfonyl) benzoic acid <b>26</b> (250 mg, 1.133 mmol) was added to morpholin (99 mg, 1.133 mmol) in THF (5 mL) in presence of K<sub>2</sub>CO<sub>3</sub> (313 mg, 2.266 mmol), and the resulting reaction mixture was stirred at room temperature for 1 h. After completion of the reaction from TLC, the solvents were removed and the crude material was purified by Companion Rf (3% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>) to give compound <b>27</b> (160 mg, 0.590 mmol) in 52% yield as white solid. In a subsequent step, compound <b>27</b> (100 mg, 0.369 mmol) dissolved in methanol (4 mL) and was added the catalytic amount of conc H<sub>2</sub>SO<sub>4</sub>. The resulting reaction mixture was heated to 65 °C for overnight. The solvents were removed and the crude material after purification gave <b>25</b> in 54% yields (60 mg, 0.200 mmol). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.38 (t, 1H, <i>J</i> = 1.6 Hz), 8.27 (m, 1H), 7.92 (m, 1H), 7.64 (t, 1H, <i>J</i> = 8.0 Hz), 3.95 (s, 3H), 3.73 (m, 4H), 3.00 (m, 4H). ESI-MS: 286.1 [M + H]<sup>+</sup>.</div><div class="NLM_p">The methyl 3-(morpholinosulfonyl) benzoate <b>25</b> (120 mg, 0.421 mmol) was reacted with hydrazine hydrate (17.53 mg, 0.547 mmol) in methanol (5 mL) and was refluxed for 12 h at 65 °C. After completion of the reaction, solvents were removed by vacuum and the obtained crude material was purified by Companion Rf to give <b>28</b> (90 mg, 0.315 mmol, 75%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.16 (m, 1H), 8.12 (m, 1H), 8.04 (m, 1H), 7.85 (m, 1H), 7.63 (t, 1H, <i>J</i> = 8.0 Hz), 4.19 (m, 2H), 3.71 (m, 4H), 2.97 (m,4H). ESI-MS: 286.1 [M + H]<sup>+</sup>.</div><div class="NLM_p">In the final step, either the ester compound <b>25</b> or 3-(morpholinosulfonyl) benzoic acid <b>27</b> was utilized for the preparation of series of benzohydrazide compounds. In one example starting with 1-(5-chloro-2-hydroxyphenyl) ethanone <b>23</b> (25.2 mg, 0.147 mmol) and hydrazine <b>28</b> (4.72 mg, 0.147 mmol) was refluxed in methanol (5 mL) in the presence of catalytic acetic acid for 1 h. In certain examples, the microwave heating to 120 °C for 30 min was performed. After completion of the reaction, the solvents was removed by vacuum and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub> affording the title, in this case the compound <b>12</b> as a white solid (85%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.31 (s, 1H), 11.69 (s, 1H), 8.27 (d, 1H, <i>J</i> = 7.6 Hz), 8.18 (s, 1H), 7.96 (d, 1H, <i>J</i> = 7.6 Hz), 7.83 (t, 1H, <i>J</i> = 7.6 Hz), 7.65 (d, 1H, <i>J</i> = 2.4 Hz), 7.33 (dd, 1H, <i>J</i> = 2.4 and 8.4 Hz), 6.94 (d, 1H, <i>J</i> = 9.2 Hz), 3.62 (m, 4H), 2.90 (m, 4H), 2.49 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.0, 158.4, 158.1, 135.5, 134.8, 133.7, 131.7, 131.5, 1.30.5, 128.6, 127.8, 122.8, 121.4, 119.8, 65.9, 46.6, 15.1. HRMS: expected 438.0885 [M + H], observed 438.0898 [M + H]. LC-MS purity of compound <b>12</b> was found to be >95%.</div><div class="NLM_p last">Similar experimental procedures were employed for the preparation of list of compounds given in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, and their NMR and mass spectral data confirms the title compounds.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28">5.2.3 (<i>E</i>)-<i>N</i>′-(1-(2,5-Dichlorophenyl)ethylidene)-3-(morpholinosulfonyl)benzohydrazide (<b>13</b>)</h4><div class="NLM_p last">1-(2,5-Dichlorophenyl)ethanone <b>29</b> (20 mg, 0.106 mmol) and 3-(morpholinosulfonyl) benzohydrazide <b>28</b> (30.2 mg, 0.106 mmol) was dissolved in methanol (volume: 4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>13</b> as a solid (10 mg, 21%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.29 (m, 1H), 8.09 (m, 1H), 7.81 (m, 1H), 7.57 (m, 1H), 7.40 (m, 1H), 7.26 (m, 2H), 3.52 (m, 4H), 2.91 (m, 4H), 2.28 (s, 3H). ESI-MS: 456.1 [M + H]<sup>+</sup>. LC-MS purity of compound <b>13</b> was found to be >95%.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29">5.2.4 (<i>Z</i>)-3-(2-(1-(5-Chloro-2-hydroxyphenyl)ethylidene)hydrazinecarbonyl)-<i>N</i>,<i>N</i>-dimethylbenzenesulfonamide (<b>14</b>)</h4><div class="NLM_p">3-(<i>N</i>,<i>N</i>-Dimethylsulfamoyl)benzoic acid (200 mg, 0.872 mmol) was refluxed in the presence of conc H<sub>2</sub>SO<sub>4</sub> (5.64 mg, 0.044 mmol) in methanol at 70 °C for overnight, and after completion of the reaction, solvent was removed by vacuum and then compound was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the methyl 3-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)benzoate <b>30</b> as a solid (125 mg, 58.9%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.42 (s, 1H), 8.27 (d, 1H, <i>J</i> = 8.0 Hz), 7.97 (d, 1H, <i>J</i> = 7.2 Hz), 7.65 (t, 1H, <i>J</i> = 8.0 Hz), 3.96 (s, 3H), 2.74 (s, 6H). ESI-MS: 244.0 [M + H]<sup>+</sup>.</div><div class="NLM_p">Methyl 3-(<i>N</i>,<i>N</i>-dimethylsulfamoyl)-benzoate <b>30</b> (150 mg, 0.617 mmol) was added to the hydrazine (29.6 mg, 0.925 mmol) in methanol and refluxed for 8 h at 65 °C. Following cooling, reaction was monitored by TLC. After completion of the reaction, solvent was removed by vacuum and then compound was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the 3-(hydrazinecarbonyl)-<i>N</i>,<i>N</i>-dimethylbenzenesulfonamide <b>31</b> as a solid (60 mg, 40%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.11 (s, 1H), 8.01 (d, 1H, <i>J</i> = 8.4 Hz), 7.92 (d, 1H, <i>J</i> = 8.0 Hz), 7.65 (t, 1H, <i>J</i> = 8.0 Hz), 2.73 (s, 6H). ESI-MS: 244.0 [M + H]<sup>+</sup>.</div><div class="NLM_p last">3-(Hydrazinecarbonyl)-<i>N</i>,<i>N</i>-dimethylbenzenesulfonamide <b>31</b> (50 mg, 0.206 mmol) and 1-(5-chloro-2-hydroxyphenyl)ethanone <b>23</b> (35.1 mg, 0.206 mmol) was dissolved in methanol (volume: 4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Reaction was monitored by TLC. After completion of the reaction, following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, afforded the title compound <b>14</b> as a solid (15 mg, 18%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.29 (m, 2H), 8.01 (d, 1H, <i>J</i> = 8.4 Hz), 7.83 (t, 1H, <i>J</i> = 8.4 Hz), 7.62 (d, 1H, <i>J</i> = 2.4 Hz), 7.32 (dd, 1H, <i>J</i> = 2.4 and 8.8 Hz), 6.96 (d, 1H, <i>J</i> = 8.8 Hz), 2.73 (s, 6H), 2.58 (s, 3H). ESI-MS: 396.0 [M + H]<sup>+</sup>. LC-MS purity of compound <b>14</b> was found to be >95%.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30">5.2.5 (<i>Z</i>)-<i>N</i>′-(1-(5-Chloro-2-hydroxyphenyl)propylidene)-3-(morpholinosulfonyl)benzohydrazide (<b>15</b>)</h4><div class="NLM_p last">3-(Morpholinosulfonyl)benzohydrazide <b>28</b> (40 mg, 0.140 mmol) and 1-(5-chloro-2-hydroxyphenyl)propan-1-one <b>32</b> (25.9 mg, 0.140 mmol) was dissolved in methanol (volume: 4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Reaction was monitored by TLC. After completion of the reaction, following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>15</b> as a solid (20 mg, 31.6%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.26 (m, 2H), 8.00 (d, 1H, <i>J</i> = 7.6 Hz), 7.84 (t, 1H, <i>J</i> = 8.0 Hz), 7.64 (d, 1H, <i>J</i> = 2.4 Hz), 7.33 (m, 1H), 6.98 (d, 1H, <i>J</i> = 9.2 Hz), 3.69 (m, 4H), 3.10 (q, 2H, <i>J</i> = 7.6 Hz), 2.99 (m, 4H), 1.26 (t, 3H, <i>J</i> = 7.6 Hz). ESI-MS: 452.1 [M + H]<sup>+</sup>. LC-MS purity of compound <b>15</b> was found to be >95%.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31">5.2.6 (<i>E</i>)-<i>N</i>′-(1-(3-Chloro-2-fluorophenyl)ethylidene)-3-(morpholinosulfonyl)benzohydrazide (<b>16</b>)</h4><div class="NLM_p last">1-(3-Chloro-2-fluorophenyl)ethanone <b>33</b> (20 mg, 0.116 mmol) and 3-(morpholinosulfonyl) benzohydrazide <b>28</b> (33.1 mg, 0.116 mmol) was dissolved in methanol (4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Following cooling, the solvent was removed by vacuum, and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>16</b> as a white solid (22 mg, 43.2%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.34 (s, 1H), 8.37 (m, 1H), 8.16 (m, 1H), 7.87 (d, 1H, <i>J</i> = 7.2 Hz), 7.65 (m, 1H), 7.41 (m, 1H), 7.10 (t, 1H, <i>J</i> = 8.0 Hz), 3.71 (m, 4H), 2.95 (m, 4H), 2.38 (s, 3H). ESI-MS: 440.1 [M + H]<sup>+</sup>. LC-MS purity of compound <b>16</b> was found to be >95%.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32">5.2.7 (<i>E</i>)-<i>N</i>′-(1-(2,6-Dihydroxyphenyl)ethylidene)benzohydrazide (<b>17</b>)</h4><div class="NLM_p last">1-(2,6-Dihydroxyphenyl)ethanone (100 mg, 0.657 mmol) <b>34</b> and benzohydrazide <b>24</b> (89 mg, 0.657 mmol) was dissolved in methanol (4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>17</b> as a white solid (100 mg, 56.3%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.59 (m, 2H), 7.49 (m, 1H), 7.39 (m, 2H), 7.11 (t, 1H, <i>J</i> = 8.0 Hz), 6.45 (m, 2H), 2.35 (s, 3H). ESI-MS: 271.1 [M + H]<sup>+</sup>. LC-MS purity of compound <b>17</b> was found to be >95%.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33">5.2.8 (<i>E</i>)-<i>N</i>′-(1-(2-Chloropyridin-4-yl)ethylidene)-3-(morpholinosulfonyl)benzohydrazide (<b>18</b>)</h4><div class="NLM_p last">1-(2-Chloropyridin-4-yl)ethanone <b>35</b> (20 mg, 0.129 mmol) and 3-(morpholinosulfonyl)benzohydrazide <b>28</b> (36.7 mg, 0.129 mmol) was dissolved in methanol (4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>18</b> as a solid (32.6 mg, 60%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.43 (m, 1H), 8.39 (m, 2H), 8.15 (d, 1H, <i>J</i> = 8.0 Hz), 7.93 (d, 1H, <i>J</i> = 7.6 Hz), 7.70 (t, 1H, J= 7.6 Hz), 7.52 (m, 1H), 3.73 (m, 4H), 3.02 (m, 4H), 2.35 (s, 3H). ESI-MS: 423.1 [M + H]<sup>+</sup>. LC-MS purity of compound 18 was found to be >95%.</div></div><div id="sec5_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34">5.2.9 (<i>Z</i>)-3-Bromo-4-chloro-<i>N</i>′-(1-(5-chloro-2-hydroxyphenyl)ethylidene)benzohydrazide (<b>19</b>)</h4><div class="NLM_p">3-Bromo-4-chlorobenzoic acid (200 mg, 0.849 mmol) was refluxed in the presence of conc H<sub>2</sub>SO<sub>4</sub> (5.49 mg, 0.042 mmol) in methanol at 70 °C for overnight, and after completion of the reaction, solvent was removed by vacuum and then compound was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the methyl 3-bromo-4-chlorobenzoate <b>36</b> as a white solid (130 mg, 61.3%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.29 (d, 1H, <i>J</i> = 2.0 Hz), 7.91 (dd, 1H, <i>J</i> = 2.0 and 8.4 Hz), 7.52 (d, 1H, <i>J</i> = 8.4 Hz), 3.92 (s, 3H). ESI-MS: 250.9 [M + H]<sup>+</sup>.</div><div class="NLM_p">Compound <b>36</b> (120 mg, 0.481 mmol) was added to the hydrazine (23.12 mg, 0.721 mmol) in methanol at 70 °C for overnight. Reaction was monitored by TLC. After completion of the reaction, solvent was removed by vacuum and then compound was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the intermediate 3-bromo-4-chlorobenzohydrazide <b>37</b> as a solid (30 mg, 25%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.02 (d, 1H, <i>J</i> = 1.6 Hz), 7.60 (dd, 1H, <i>J</i> = 2.0 and 8.0 Hz), 7.52 (d, 1H, <i>J</i> = 8.0 Hz). ESI-MS: 250.9 [M + H]<sup>+</sup>.</div><div class="NLM_p last">The compound <b>37</b> (30 mg, 0.120 mmol) and 1-(5-chloro-2-hydroxyphenyl)ethanone <b>23</b> (20.51 mg, 0.120 mmol) was dissolved in methanol (volume: 4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Reaction was monitored by TLC. After completion of the reaction, following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>19</b> as a solid (15 mg, 31%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>): δ 8.30 (s, 1H), 7.98 (d, 1H, <i>J</i> = 8.4 Hz), 7.73 (d, 1H, <i>J</i> = 8.4 Hz), 7.61 (d, 1H, <i>J</i> = 2.4 Hz), 7.29 (dd, 1H, <i>J</i> = 2.4 and 8.4 Hz), 6.93 (d, 1H, <i>J</i> = 8.8 Hz), 2.55 (s, 3H). ESI-MS: 402.9 [M + H]<sup>+</sup>. LC-MS purity of compound <b>19</b> was found to be >95%.</div></div><div id="sec5_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35">5.2.10 (<i>Z</i>)-5-Bromo-6-chloro-<i>N</i>′-(1-(5-chloro-2-hydroxyphenyl)ethylidene)nicotinohydrazide (<b>20</b>)</h4><div class="NLM_p">Methyl 5-bromo-6-chloronicotinate (100 mg, 0.399 mmol) was added to the hydrazine (19.19 mg, 0.599 mmol) in methanol at 70 °C for overnight. Reaction was monitored by TLC. After completion of the reaction, solvent was removed by vacuum and then compound was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the 5-bromo-6-chloronicotinohydrazide <b>38</b> as a solid (20 mg, 20%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.33 (d, 1H, <i>J</i> = 2.4 Hz), 8.01 (d, 1H, <i>J</i> = 2.4 Hz).</div><div class="NLM_p last">5-Bromo-6-chloronicotinohydrazide <b>38</b> (15 mg, 0.060 mmol) and 1-(5-chloro-2-hydroxyphenyl)ethanone <b>23</b> (10.22 mg, 0.060 mmol) was dissolved in methanol (volume: 4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Reaction was monitored by TLC. After completion of the reaction, following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>20</b> as a solid (8 mg, 33%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.39 (d, 1H, <i>J</i> = 2.4 Hz), 8.28 (s, 1H), 7.63 (d, 1H, <i>J</i> = 2.4 Hz), 7.32 (dd, 1H, <i>J</i> = 2.4 and 8.8 Hz), 7.06 (d, 1H, <i>J</i> = 6.8 Hz), 6.92 (d, 1H, <i>J</i> = 9.2 Hz), 6.81 (d, 1H, <i>J</i> = 6.8 Hz), 2.47 (s, 3H). ESI-MS: 404.0 [M + H]<sup>+</sup>. LC-MS purity of compound 20 was found to be >95%.</div></div><div id="sec5_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36">5.2.11 (<i>Z</i>)-5-Chloro-<i>N</i>′-(1-(5-chloro-2-hydroxyphenyl)ethylidene)nicotinohydrazide (<b>21</b>)</h4><div class="NLM_p">5-Chloronicotinic acid (200 mg, 1.269 mmol) was refluxed in the presence of conv H<sub>2</sub>SO<sub>4</sub> (8.20 mg, 0.063 mmol) in methanol at 70 °C for overnight, and after completion of the reaction, solvent was removed by vacuum and then compound was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the methyl 5-chloronicotinate <b>39</b> as a solid (120 mg, 55%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.07 (d, 1H, <i>J</i> = 1.6 Hz), 8.72 (d, 1H, <i>J</i> = 2.0 Hz), 8.26 (m, 1H), 3.95 (s, 1H).</div><div class="NLM_p">Hydrazine (17.93 mg, 0.560 mmol) was added to the methyl 5-chloronicotinate <b>39</b> (80 mg, 0.466 mmol) in methanol at 70 °C overnight. Reaction was monitored by TLC. After completion of the reaction, solvent was removed by vacuum and then compound was purified by Companion Rf with 1% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the 5-chloronicotinohydrazide <b>40</b> as a solid (40 mg, 50%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.85 (d, 1H, <i>J</i> = 2.0 Hz), 8.70 (d, 1H, <i>J</i> = 2.4 Hz), 8.22 (t, 1H, <i>J</i> = 2.0 Hz). ESI-MS: 172.0 [M + H]<sup>+</sup>.</div><div class="NLM_p last">5-chloronicotinohydrazide <b>40</b> (30 mg, 0.175 mmol) and 1-(5-chloro-2-hydroxyphenyl)ethanone <b>23</b> (29.8 mg, 0.175 mmol) was dissolved in methanol (volume: 4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Reaction was monitored by TLC. After completion of the reaction, following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>21</b> as a solid (20 mg, 35.3%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>): δ 9.06 (s, 1H), 8.77 (s, 1H), 8.37 (s, 1H), 7.62 (d, 1H, <i>J</i> = 2.8 Hz), 7.31 (dd, 1H, <i>J</i> = 2.0 and 8.4 Hz), 6.95 (d, 1H, <i>J</i> = 8.8 Hz), 2.58 (s, 3H). ESI-MS: 324.0 [M + H]<sup>+</sup>. LC-MS purity of compound <b>21</b> was found to be >95%.</div></div><div id="sec5_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37">5.2.12 (<i>Z</i>)-3-(Morpholinosulfonyl)-<i>N</i>′-(1-(pyridin-3-yl)ethylidene)benzohydrazide (<b>22</b>)</h4><div class="NLM_p last">3-(Morpholino sulfonyl)benzohydrazide <b>28</b> (40 mg, 0.140 mmol) and 1-(pyridin-3-yl)ethanone <b>41</b> (16.98 mg, 0.140 mmol) was dissolved in methanol (4 mL) in the presence of acetic acid as a catalyst, and then the reaction mixture was heated via microwave irradiation to 120 °C for 30 min. Reaction was monitored by TLC. After completion of the reaction, following cooling, the solvent was removed by vacuum and the resulting crude material was purified by Companion Rf with 2% CH<sub>3</sub>OH/CH<sub>2</sub>Cl<sub>2</sub>, affording the title compound <b>22</b> as a solid (15 mg, 27.5%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.53 (bs, 1H), 8.87 (s, 1H), 8.59 (m, 1H), 8.39 (m, 1H), 8.17 (m, 1H), 7.98 (m, 1H), 7.89 (d, 1H, <i>J</i> = 8.0 Hz), 7.67 (t, 1H, <i>J</i> = 8.0 Hz), 7.32 (m, 1H), 3.70 (m, 4H), 3.00 (m, 4H), 2.39 (s, 3H). ESI-MS: 389.0 [M + H]<sup>+</sup>. LC-MS purity of compound <b>22</b> was found to be >95%.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38">5.3 Biochemical Assays</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39">5.3.1 LSD1 Screening Assay</h4><div class="NLM_p last">The LSD1 screening biochemical assay kit was purchased from Cayman Chemical (Ann Arbor, MI). Test compounds were diluted to 20× the desired test concentration in 100% DMSO and 2.5 μL of the diluted drug sample was added to a black 384-well plate. The LSD1 enzyme stock was diluted 17-fold with assay buffer, and 40 μL of the diluted LSD1 enzyme was added to the appropriate wells. Substrate, consisting of horseradish peroxidase, dimethyl K4 peptide corresponding to the first 21 amino acids of the N-terminal tail of histone H3, and 10-acetyl-3,7-dihydroxyphenoxazine was then added to wells. Resorufin was analyzed on an Envision plate reader with an excitation wavelength of 530 nm and an emission wavelength of 595 nm.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40">5.3.2 Off-Target Assays</h4><div class="NLM_p last">Monoamine oxidase A and B enzymes were purchased from Sigma (catalogue nos. m7316, and m7441, respectively). Biochemical kits were purchased as follows: MAO-Glo – Promega Corp. (Fitchburg, WI); <span class="smallcaps smallerCapital">d</span>-lactate dehydrogenase, Cayman Chemical (Ann Arbor, MI); glucose oxidase, Life Technologies Corp. (Grand Island, NY). Inhibition assay were carried out according to the manufacturer’s suggested protocol. CYP and hERG evaluation utilized the SelectScreen Biochemical P450 Profiling and hERG Screening Services provided by Invitrogen Corp. (Madison, WI).</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41">5.3.3 Reversibility and Michaelis–Menten Analysis</h4><div class="NLM_p last">Biochemical characterization of reversibility and <i>K</i><sub>i</sub> was performed using the biochemical screen from Cayman Chemical with purified recombinant full-length his<sub>6</sub>-LSD1 substituted for the commercially provided protein mix. Reversibility was determined using jump dilution. LSD1 was incubated at 10× IC<sub>50</sub> of compound <b>12</b> or tranylcypromine for 1 h then diluted into the reaction 100-fold. The reaction buffer was either compound-free or contained 10× the IC<sub>50</sub>. Compound <b>13</b> was used as a negative control. The <i>K</i><sub>i</sub> of compound <b>12</b> was determined using a Michaelis–Menten kinetic analysis across multiple concentrations of compound <b>12</b>. The data was plotted in GraphPad. <i>K</i><sub>i</sub> was determined using the following equation: <i>K</i><sub>i</sub> = [Inhibitor]/((<i>v</i><sub>max</sub>/<i>v</i><sub>apparent-max</sub>) – 1)</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42">5.4 In Vitro Assays</h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43">5.4.1 ATPlite Cell Viability A</h4><div class="NLM_p last">ATPlite was purchased from PerkinElmer (Waltham, MA). Cancer cell lines were obtained from ATCC. Cells were cultured according to the procedures provided. Cells were seeded in 96-well plates and then treated with different concentrations of inhibitor (0.1% final DMSO concentration). After 96 h of incubation, ATPlite was added directly to the culture well. Luminescence was read 5 min later on an Envision plate reader.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44">5.4.2 Global H3K4 and H3K9 Methylation Analysis</h4><div class="NLM_p last">VCaP cells were maintained in RPMI media containing 10% FBS and no phenol red. The day before the experiment was started, we plated 500000 cells per well of a 12-well plate. The following day, the culture medium was replaced with fresh RPMI containing 10% FBS and compound <b>12</b>. Individual compound <b>12</b> solutions were prepared using DMSO; the final concentration of the solvent in the culture medium was 1%. The cells then were incubated for 24 h at 37 °C. Individual wells were washed with PBS, and lysates were obtained as described previously.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> We subjected 50 μg of total protein to electrophoresis and immunoblot analyses using anti-H3K9Me2 and anti-β-Actin antibodies, as described.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The extent of immunoreactivity then was assessed by chemiluminescence, following incubation with appropriate, horseradish peroxidase-labeled, secondary antibodies.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/jm400870h">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74228" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74228" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Detailed virtual screening methods; purchased hits, analytical data; docking scores of compounds <b>1</b>–<b>10</b>, commercially available LSD1 hits (111) from the list of 121 compounds selected, Tanimoto similarity coefficients comparing compound <b>12</b> and known LSD1 inhibitors from Chart <a class="ref internalNav" href="#cht1" aria-label="1">1</a>; off-target inhibition; comparison of different model fits for enzyme kinetics; binding site model and definition of active site of LSD1 structure; flow diagram for the virtual ligand screening (VLS) using ICM-VLS; Schrodinger workflow GOLD programs; complete reaction schemes for compounds <b>11</b>–<b>22</b>; LC-MS data for compound <b>12</b>. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400870h/suppl_file/jm400870h_si_001.pdf">jm400870h_si_001.pdf (1.82 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm400870h" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12577" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12577" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sunil Sharma</span> - <span class="hlFld-Affiliation affiliation">Center
for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Medicine, University of Utah, 30 North 1900 East, Salt Lake City, Utah 84132, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Division
of Medical Oncology, Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#cebdbba0a7a2e0bda6afbca3af8ea6ada7e0bbbaafa6e0abaabb"><span class="__cf_email__" data-cfemail="90e3e5fef9fcbee3f8f1e2fdf1d0f8f3f9bee5e4f1f8bef5f4e5">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Venkataswamy Sorna</span> - <span class="hlFld-Affiliation affiliation">Center
for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emily R. Theisen</span> - <span class="hlFld-Affiliation affiliation">Center
for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutics and Pharmaceutical Chemistry, College of Pharmacy, University of Utah, 301 Skaggs Hall, Salt Lake City, Utah 84112, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bret Stephens</span> - <span class="hlFld-Affiliation affiliation">Center
for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven L. Warner</span> - <span class="hlFld-Affiliation affiliation">Center
for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Bearss</span> - <span class="hlFld-Affiliation affiliation">Center
for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hariprasad Vankayalapati</span> - <span class="hlFld-Affiliation affiliation">Center
for Investigational Therapeutics (CIT), Huntsman Cancer Institute, University of Utah, 2000 Circle of Hope, Salt Lake City, Utah 84112, United States</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Medicine, University of Utah, 30 North 1900 East, Salt Lake City, Utah 84132, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>These authors contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Drs. Sunil Sharma, David Bearss, Steve Warner, and Hariprasad Vankayalapati have equity interest in Salarius Pharmaceuticals.<br /></br></div></li></ul></div><div class="ack" id="ACK-d3174e3367-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i47">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46134" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46134" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We gratefully acknowledge Dr. Yang Shi for the generous gift of the LSD1 expression construct, Dr. Diana Stafforini for her assistance with VCaP cell culture and blots, and Jedediah Doane, Simon Currie, Dr. Niraja Bhachech, and Dr. Barbara Graves for their protein purification guidance. Research reported in this article utilized the Experimental Therapeutics program and was supported by the Huntsman Cancer Foundation and the National Cancer Institute of the National Institutes of Health under award number P30CA042014. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i48" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i48"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i49" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i49"> Abbreviations Used</h2><tr><td class="NLM_term">BHC80</td><td class="NLM_def"><p class="first last">BRAF35-HDAC complex protein 80</p></td></tr><tr><td class="NLM_term">Co-REST</td><td class="NLM_def"><p class="first last">REST corepressor 1</p></td></tr><tr><td class="NLM_term">CYP</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">flavin adenine dinucleotide</p></td></tr><tr><td class="NLM_term">Gfi-1</td><td class="NLM_def"><p class="first last">growth factor independent 1</p></td></tr><tr><td class="NLM_term">H3</td><td class="NLM_def"><p class="first last">histone H3</p></td></tr><tr><td class="NLM_term">H3K4</td><td class="NLM_def"><p class="first last">histone H3 at lysine 4</p></td></tr><tr><td class="NLM_term">H3K4me1</td><td class="NLM_def"><p class="first last">monomethylation of histone H3 at lysine 4</p></td></tr><tr><td class="NLM_term">H3K4me2</td><td class="NLM_def"><p class="first last">dimethylation of histone H3 at lysine 4</p></td></tr><tr><td class="NLM_term">H3K4me3</td><td class="NLM_def"><p class="first last">trimethylation of histone H3 at lysine 4</p></td></tr><tr><td class="NLM_term">H3K9</td><td class="NLM_def"><p class="first last">histone H3 at lysine 9</p></td></tr><tr><td class="NLM_term">H3K9me1</td><td class="NLM_def"><p class="first last">monomethylation of histone H3 at lysine 9</p></td></tr><tr><td class="NLM_term">H3K9me2</td><td class="NLM_def"><p class="first last">dimethylation of histone H3 at lysine 9</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-à-go-go</p></td></tr><tr><td class="NLM_term">HTVS</td><td class="NLM_def"><p class="first last">high-throughput virtual screen</p></td></tr><tr><td class="NLM_term">K4</td><td class="NLM_def"><p class="first last">lysine 4</p></td></tr><tr><td class="NLM_term">K9</td><td class="NLM_def"><p class="first last">lysine 9</p></td></tr><tr><td class="NLM_term">LSD1</td><td class="NLM_def"><p class="first last">lysine-specific demethylase</p></td></tr><tr><td class="NLM_term">MAO</td><td class="NLM_def"><p class="first last">monoamine oxidase</p></td></tr><tr><td class="NLM_term">NuRD</td><td class="NLM_def"><p class="first last">nucleosome remodeling and histone deacetylase</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SNAI1</td><td class="NLM_def"><p class="first last">snail homologue 1 (drosophila)</p></td></tr><tr><td class="NLM_term">TCP</td><td class="NLM_def"><p class="first last">tranylcypromine</p></td></tr><tr><td class="NLM_term">VS</td><td class="NLM_def"><p class="first last">virtual screen</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58647" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58647" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 43 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Tsai, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Cancer epigenetics: linking basic biology to clinical medicine</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">502</span><span class="NLM_x">–</span> <span class="NLM_lpage">517</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1038%2Fcr.2011.24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=21321605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=502-517&author=H.-C.+Tsaiauthor=S.+B.+Baylin&title=Cancer+epigenetics%3A+linking+basic+biology+to+clinical+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics: linking basic biology to clinical medicine</span></div><div class="casAuthors">Tsai, Hsing-Chen; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">502-517</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Cancer evolution at all stages is driven by both epigenetic abnormalities as well as genetic alterations.  Dysregulation of epigenetic control events may lead to abnormal patterns of DNA methylation and chromatin configurations, both of which are crit. contributors to the pathogenesis of cancer.  These epigenetic abnormalities are set and maintained by multiple protein complexes and the interplay between their individual components including DNA methylation machinery, histone modifiers, particularly, polycomb (PcG) proteins, and chromatin remodeling proteins.  Recent advances in genome-wide technol. have revealed that the involvement of these dysregulated epigenetic components appears to be extensive.  Moreover, there is a growing connection between epigenetic abnormalities in cancer and concepts concerning stem-like cell subpopulations as a driving force for cancer.  Emerging data suggest that aspects of the epigenetic landscape inherent to normal embryonic and adult stem/progenitor cells may help foster, under the stress of chronic inflammation or accumulating reactive oxygen species, evolution of malignant subpopulations.  Finally, understanding mol. mechanisms involved in initiation and maintenance of epigenetic abnormalities in all types of cancer has great potential for translational purposes.  This is already evident for epigenetic biomarker development, and for pharmacol. targeting aimed at reversing cancer-specific epigenetic alterations.  Cell Research (2011) 21:502-517. doi:10.1038/cr.2011.24; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqt533Ud9VRxbVg90H21EOLACvtfcHk0litLRKqJf8--w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurk%253D&md5=f4834740b42458e1adf7c7c24df2589c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.24%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DH.-C.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DCancer%2520epigenetics%253A%2520linking%2520basic%2520biology%2520to%2520clinical%2520medicine%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D502%26epage%3D517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Füllgrabe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, B.</span><span> </span><span class="NLM_article-title">Histone onco-modifications</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3391</span><span class="NLM_x">–</span> <span class="NLM_lpage">3403</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=3391-3403&author=J.+F%C3%BCllgrabeauthor=E.+Kavanaghauthor=B.+Joseph&title=Histone+onco-modifications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DF%25C3%25BCllgrabe%26aufirst%3DJ.%26aulast%3DKavanagh%26aufirst%3DE.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DHistone%2520onco-modifications%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D3391%26epage%3D3403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Demethylation Mediated by the Nuclear Amine Oxidase Homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+Demethylation+Mediated+by+the+Nuclear+Amine+Oxidase+Homolog+LSD1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0litLRKqJf8--w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Demethylation%2520Mediated%2520by%2520the%2520Nuclear%2520Amine%2520Oxidase%2520Homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H. F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the Histone Code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+Histone+Code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lhJCuCwPcVglw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520Histone%2520Code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Lachner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">An epigenetic road map for histone lysine methylation</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">2117</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1242%2Fjcs.00493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=12730288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkslCltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2003&pages=2117-2124&author=M.+Lachnerauthor=R.+J.+O%E2%80%99Sullivanauthor=T.+Jenuwein&title=An+epigenetic+road+map+for+histone+lysine+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">An epigenetic road map for histone lysine methylation</span></div><div class="casAuthors">Lachner, Monika; O'Sullivan, Roderick J.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2117-2124</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review discusses the complexity and coding potential of histone lysine methylation in epigenetic control.  The dynamic cycle of histone lysine methylation in transcriptional stimulation or repression is illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjyGLMb0IvPLVg90H21EOLACvtfcHk0lhJCuCwPcVglw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkslCltLg%253D&md5=bde6700de17f4131185bc9439c3b29d7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1242%2Fjcs.00493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.00493%26sid%3Dliteratum%253Aachs%26aulast%3DLachner%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DAn%2520epigenetic%2520road%2520map%2520for%2520histone%2520lysine%2520methylation%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2003%26volume%3D116%26spage%3D2117%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanoni, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span> </span><span class="NLM_article-title">Histone demethylation catalysed by LSD1 is a flavin-dependent oxidative process</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">579</span><span class="NLM_x">, </span> <span class="NLM_fpage">2203</span><span class="NLM_x">–</span> <span class="NLM_lpage">2207</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.febslet.2005.03.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=15811342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlWitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2005&pages=2203-2207&author=F.+Fornerisauthor=C.+Bindaauthor=M.+A.+Vanoniauthor=A.+Matteviauthor=E.+Battaglioli&title=Histone+demethylation+catalysed+by+LSD1+is+a+flavin-dependent+oxidative+process"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation catalyzed by LSD1 is a flavin-dependent oxidative process</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Vanoni, Maria Antonietta; Mattevi, Andrea; Battaglioli, Elena</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2203-2207</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific histone demethylase 1 (LSD1) is a very recently discovered enzyme which specifically removes Me groups from Lys-4 of histone H3.  Here, the authors addressed the functional properties of the protein demonstrating that histone demethylation involves the FAD-catalyzed oxidn. of the methylated Lys residue.  The nature of the substrate that acts as the electron acceptor required to complete the catalytic cycle was investigated.  LSD1 converts O2 to H2O2 although this reactivity is not as pronounced as that of other flavin-dependent oxidases.  The findings raised the possibility that in vivo LSD1 might not necessarily function as an oxidase, but it might use alternative electron acceptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaI1pvSxtE-7Vg90H21EOLACvtfcHk0lhJCuCwPcVglw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlWitr4%253D&md5=d993a48b0a9a8d9b4b27d83308ac8c0a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2005.03.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2005.03.015%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DVanoni%26aufirst%3DM.%2BA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26atitle%3DHistone%2520demethylation%2520catalysed%2520by%2520LSD1%2520is%2520a%2520flavin-dependent%2520oxidative%2520process%26jtitle%3DFEBS%2520Lett.%26date%3D2005%26volume%3D579%26spage%3D2203%26epage%3D2207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">LSD1: oxidative chemistry for multifaceted functions in chromatin regulation</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">189</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=181-189&author=F.+Fornerisauthor=C.+Bindaauthor=E.+Battaglioliauthor=A.+Mattevi&title=LSD1%3A+oxidative+chemistry+for+multifaceted+functions+in+chromatin+regulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DLSD1%253A%2520oxidative%2520chemistry%2520for%2520multifaceted%2520functions%2520in%2520chromatin%2520regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D181%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gocke, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borek, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomchick, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span> </span><span class="NLM_article-title">Structural Basis for CoREST-Dependent Demethylation of Nucleosomes by the Human LSD1 Histone Demethylase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.molcel.2006.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=16885027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD28XovVWlu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=377-387&author=M.+Yangauthor=C.+B.+Gockeauthor=X.+Luoauthor=D.+Borekauthor=D.+R.+Tomchickauthor=M.+Machiusauthor=Z.+Otwinowskiauthor=H.+Yu&title=Structural+Basis+for+CoREST-Dependent+Demethylation+of+Nucleosomes+by+the+Human+LSD1+Histone+Demethylase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for CoREST-dependent demethylation of nucleosomes by the human LSD1 histone demethylase</span></div><div class="casAuthors">Yang, Maojun; Gocke, Christian B.; Luo, Xuelian; Borek, Dominika; Tomchick, Diana R.; Machius, Mischa; Otwinowski, Zbyszek; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">377-387</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone methylation regulates diverse chromatin-templated processes, including transcription.  Many transcriptional corepressor complexes contain lysine-specific demethylase 1 (LSD1) and CoREST that collaborate to demethylate mono- and dimethylated H3-K4 of nucleosomes.  Here, we report the crystal structure of the LSD1-CoREST complex.  LSD1-CoREST forms an elongated structure with a long stalk connecting the catalytic domain of LSD1 and the CoREST SANT2 domain.  LSD1 recognizes a large segment of the H3 tail through a deep, neg. charged pocket at the active site and possibly a shallow groove on its surface.  CoREST SANT2 interacts with DNA.  Disruption of the SANT2-DNA interaction diminishes CoREST-dependent demethylation of nucleosomes by LSD1.  The shape and dimension of LSD1-CoREST suggest its bivalent binding to nucleosomes, allowing efficient H3-K4 demethylation.  This spatially sepd., multivalent nucleosome binding mode may apply to other chromatin-modifying enzymes that generally contain multiple nucleosome binding modules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon5vOAvWkcXrVg90H21EOLACvtfcHk0ljcqFnkW0FryQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XovVWlu74%253D&md5=32aa853fd60a8a562900defa70d658c0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DGocke%26aufirst%3DC.%2BB.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DBorek%26aufirst%3DD.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520Basis%2520for%2520CoREST-Dependent%2520Demethylation%2520of%2520Nucleosomes%2520by%2520the%2520Human%2520LSD1%2520Histone%2520Demethylase%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D377%26epage%3D387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Shi, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwase, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Regulation of LSD1 Histone Demethylase Activity by Its Associated Factors</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">857</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.molcel.2005.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=16140033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=857-864&author=Y.-J.+Shiauthor=C.+Matsonauthor=F.+Lanauthor=S.+Iwaseauthor=T.+Babaauthor=Y.+Shi&title=Regulation+of+LSD1+Histone+Demethylase+Activity+by+Its+Associated+Factors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of LSD1 histone demethylase activity by its associated factors</span></div><div class="casAuthors">Shi, Yu-Jiang; Matson, Caitlin; Lan, Fei; Iwase, Shigeki; Baba, Tadashi; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">857-864</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a recently identified human lysine (K)-specific histone demethylase.  LSD1 is assocd. with HDAC1/2, with CoREST, a SANT domain-contg. corepressor, and with BHC80, a PHD domain-contg. protein, among others.  We show that CoREST confers on LSD1 the ability to demethylate nucleosomal substrates and that it protects LSD1 from proteasomal degrdn. in vivo.  We find hyperacetylated nucleosomes less susceptible to CoREST/LSD1-mediated demethylation, suggesting that hypoacetylated nucleosomes may be the preferred physiol. substrates.  This raises the possibility that histone deacetylases and LSD1 may collaborate to generate a repressive chromatin environment.  Consistent with this model, TSA treatment results in derepression of LSD1 target genes.  While CoREST pos. regulates LSD1 function, BHC80 inhibits CoREST/LSD1-mediated demethylation in vitro and may therefore confer neg. regulation.  Taken together, these findings suggest that LSD1-mediated histone demethylation is regulated dynamically in vivo.  This is expected to have profound effects on gene expression under both physiol. and pathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraharVnxcaMLVg90H21EOLACvtfcHk0ljcqFnkW0FryQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK&md5=34ba238112151c6596dd6f6c7caa043e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.-J.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DIwase%26aufirst%3DS.%26aulast%3DBaba%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DRegulation%2520of%2520LSD1%2520Histone%2520Demethylase%2520Activity%2520by%2520Its%2520Associated%2520Factors%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D857%26epage%3D864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Cegli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alpatov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Recognition of unmethylated histone H3 lysine 4 links BHC80 to LSD1-mediated gene repression</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">718</span><span class="NLM_x">–</span> <span class="NLM_lpage">722</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=718-722&author=F.+Lanauthor=R.+E.+Collinsauthor=R.+De+Cegliauthor=R.+Alpatovauthor=J.+R.+Hortonauthor=X.+Shiauthor=O.+Gozaniauthor=X.+Chengauthor=Y.+Shi&title=Recognition+of+unmethylated+histone+H3+lysine+4+links+BHC80+to+LSD1-mediated+gene+repression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLan%26aufirst%3DF.%26aulast%3DCollins%26aufirst%3DR.%2BE.%26aulast%3DDe%2BCegli%26aufirst%3DR.%26aulast%3DAlpatov%26aufirst%3DR.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DRecognition%2520of%2520unmethylated%2520histone%2520H3%2520lysine%25204%2520links%2520BHC80%2520to%2520LSD1-mediated%2520gene%2520repression%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D718%26epage%3D722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooch, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dall’Aglio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">A Highly Specific Mechanism of Histone H3-K4 Recognition by Histone Demethylase LSD1</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">35289</span><span class="NLM_x">–</span> <span class="NLM_lpage">35295</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=35289-35295&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Dall%E2%80%99Aglioauthor=M.+W.+Fraaijeauthor=E.+Battaglioliauthor=A.+Mattevi&title=A+Highly+Specific+Mechanism+of+Histone+H3-K4+Recognition+by+Histone+Demethylase+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DDall%25E2%2580%2599Aglio%26aufirst%3DA.%26aulast%3DFraaije%26aufirst%3DM.%2BW.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520Highly%2520Specific%2520Mechanism%2520of%2520Histone%2520H3-K4%2520Recognition%2520by%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D35289%26epage%3D35295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanoni, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Human Histone Demethylase LSD1 Reads the Histone Code</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">41360</span><span class="NLM_x">–</span> <span class="NLM_lpage">41365</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1074%2Fjbc.M509549200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=16223729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=41360-41365&author=F.+Fornerisauthor=C.+Bindaauthor=M.+A.+Vanoniauthor=E.+Battaglioliauthor=A.+Mattevi&title=Human+Histone+Demethylase+LSD1+Reads+the+Histone+Code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Human Histone Demethylase LSD1 Reads the Histone Code</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Vanoni, Maria Antonietta; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41360-41365</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human histone demethylase LSD1 is a flavin-dependent amine oxidase that catalyzes the specific removal of Me groups from mono- and dimethylated Lys4 of histone H3.  The N-terminal tail of H3 is subject to various covalent modifications, and a fundamental question in LSD1 biol. is how these epigenetic marks affect the demethylase activity.  The authors show that LSD1 does not have a strong preference for mono- or dimethylated Lys4 of H3.  Substrate recognition is not confined to the residues neighboring Lys4, but it requires a sufficiently long peptide segment consisting of the N-terminal 20 amino acids of H3.  Electrostatic interactions are an important factor in protein-substrate recognition, as indicated by the high sensitivity of Km to ionic strength.  The authors have probed LSD1 for its ability to demethylate Lys4 in presence of a second modification on the same peptide substrate.  Methylation of Lys9 does not affect enzyme catalysis.  Conversely, Lys9 acetylation causes an almost 6-fold increase in the Km value, whereas phosphorylation of Ser10 totally abolishes activity.  LSD1 is inhibited by a demethylated peptide with an inhibition const. of 1.8 μM, suggesting that LSD1 can bind to H3 independently of Lys4 methylation.  LSD1 is a chromatin-modifying enzyme, which is able to read different epigenetic marks on the histone N-terminal tail and can serve as a docking module for the stabilization of the assocd. corepressor complex(es) on chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxu9eUWof47Vg90H21EOLACvtfcHk0liniCigKu-osA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L&md5=f39f7c79f1f789fb06ca7dd8a98752fe</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M509549200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M509549200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DVanoni%26aufirst%3DM.%2BA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DHuman%2520Histone%2520Demethylase%2520LSD1%2520Reads%2520the%2520Histone%2520Code%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D41360%26epage%3D41365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Garcia-Bassets, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telese, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prefontaine, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutt, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, B.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohgi, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escoubet-Lozach, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glass, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, X.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, M. G.</span><span> </span><span class="NLM_article-title">Histone Methylation-Dependent Mechanisms Impose Ligand Dependency for Gene Activation by Nuclear Receptors</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">518</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=505-518&author=I.+Garcia-Bassetsauthor=Y.-S.+Kwonauthor=F.+Teleseauthor=G.+G.+Prefontaineauthor=K.+R.+Huttauthor=C.+S.+Chengauthor=B.-G.+Juauthor=K.+A.+Ohgiauthor=J.+Wangauthor=L.+Escoubet-Lozachauthor=D.+W.+Roseauthor=C.+K.+Glassauthor=X.-D.+Fuauthor=M.+G.+Rosenfeld&title=Histone+Methylation-Dependent+Mechanisms+Impose+Ligand+Dependency+for+Gene+Activation+by+Nuclear+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia-Bassets%26aufirst%3DI.%26aulast%3DKwon%26aufirst%3DY.-S.%26aulast%3DTelese%26aufirst%3DF.%26aulast%3DPrefontaine%26aufirst%3DG.%2BG.%26aulast%3DHutt%26aufirst%3DK.%2BR.%26aulast%3DCheng%26aufirst%3DC.%2BS.%26aulast%3DJu%26aufirst%3DB.-G.%26aulast%3DOhgi%26aufirst%3DK.%2BA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DEscoubet-Lozach%26aufirst%3DL.%26aulast%3DRose%26aufirst%3DD.%2BW.%26aulast%3DGlass%26aufirst%3DC.%2BK.%26aulast%3DFu%26aufirst%3DX.-D.%26aulast%3DRosenfeld%26aufirst%3DM.%2BG.%26atitle%3DHistone%2520Methylation-Dependent%2520Mechanisms%2520Impose%2520Ligand%2520Dependency%2520for%2520Gene%2520Activation%2520by%2520Nuclear%2520Receptors%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D505%26epage%3D518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Sch%C3%BCleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+%28LSD1%29+is+highly+expressed+in+ER-negative+breast+cancers+and+a+biomarker+predicting+aggressive+biology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0liniCigKu-osA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520is%2520highly%2520expressed%2520in%2520ER-negative%2520breast%2520cancers%2520and%2520a%2520biomarker%2520predicting%2520aggressive%2520biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 Is Strongly Expressed in Poorly Differentiated Neuroblastoma: Implications for Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+Is+Strongly+Expressed+in+Poorly+Differentiated+Neuroblastoma%3A+Implications+for+Therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520Is%2520Strongly%2520Expressed%2520in%2520Poorly%2520Differentiated%2520Neuroblastoma%253A%2520Implications%2520for%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1002%2Fijc.25349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=20333681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=574-586&author=S.+Hayamiauthor=J.+D.+Kellyauthor=H.-S.+Choauthor=M.+Yoshimatsuauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=D.+E.+Nealauthor=H.+Yamaueauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Overexpression+of+LSD1+contributes+to+human+carcinogenesis+through+chromatin+regulation+in+various+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span></div><div class="casAuthors">Hayami Shinya; Kelly John D; Cho Hyun-Soo; Yoshimatsu Masanori; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Neal David E; Yamaue Hiroki; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A number of histone demethylases have been identified and biochemically characterized, but the pathological roles of their dysfunction in human disease like cancer have not been well understood.  Here, we demonstrate important roles of lysine-specific demethylase 1 (LSD1) in human carcinogenesis.  Expression levels of LSD1 are significantly elevated in human bladder carcinomas compared with nonneoplastic bladder tissues (p < 0.0001). cDNA microarray analysis also revealed its transactivation in lung and colorectal carcinomas.  LSD1-specific small interfering RNAs significantly knocked down its expression and resulted in suppression of proliferation of various bladder and lung cancer cell lines.  Concordantly, introduction of exogenous LSD1 expression promoted cell cycle progression of human embryonic kidney fibroblast cells.  Expression profile analysis showed that LSD1 could affect the expression of genes involved in various chromatin-modifying pathways such as chromatin remodeling at centromere, centromeric heterochromatin formation and chromatin assembly, indicating its essential roles in carcinogenesis through chromatin modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WuIHSmpohpd0fW6udTcc2eaL6H4ghAMgv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D&md5=b15e7aa7b05c05dd8cfd781ed1a7e3bd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fijc.25349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25349%26sid%3Dliteratum%253Aachs%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DCho%26aufirst%3DH.-S.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520LSD1%2520contributes%2520to%2520human%2520carcinogenesis%2520through%2520chromatin%2520regulation%2520in%2520various%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D574%26epage%3D586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen Receptor Coactivators Lysine-Specific Histone Demethylase 1 and Four and a Half LIM Domain Protein 2 Predict Risk of Prostate Cancer Recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Androgen+Receptor+Coactivators+Lysine-Specific+Histone+Demethylase+1+and+Four+and+a+Half+LIM+Domain+Protein+2+Predict+Risk+of+Prostate+Cancer+Recurrence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520Receptor%2520Coactivators%2520Lysine-Specific%2520Histone%2520Demethylase%25201%2520and%2520Four%2520and%2520a%2520Half%2520LIM%2520Domain%2520Protein%25202%2520Predict%2520Risk%2520of%2520Prostate%2520Cancer%2520Recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Zhao, Z.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.-B.</span><span> </span><span class="NLM_article-title">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6651</span><span class="NLM_x">–</span> <span class="NLM_lpage">6656</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.3748%2Fwjg.v18.i45.6651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6651-6656&author=Z.-K.+Zhaoauthor=H.-F.+Yuauthor=D.-R.+Wangauthor=P.+Dongauthor=L.+Chenauthor=W.-G.+Wuauthor=W.-J.+Dingauthor=Y.-B.+Liu&title=Overexpression+of+lysine+specific+demethylase+1+predicts+worse+prognosis+in+primary+hepatocellular+carcinoma+patients"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span></div><div class="casAuthors">Zhao, Ze-Kun; Yu, Hai-Feng; Wang, Dao-Rong; Dong, Ping; Chen, Lei; Wu, Wen-Guang; Ding, Wen-Jun; Liu, Ying-Bin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">6651-6656</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Aim: To investigate the clinicopathol. features and prognostic value of lysine specific demethylase 1 (LSD1) in hepatocellular carcinoma (HCC).  Methods: We examd. LSD1 expression in 60 paired liver cancer tissues and adjacent noncancerous tissues by quant. real time polymerase chain reaction (qRT-PCR) and Western blotting.  In addn., we analyzed LSD1 expression in 198 HCC samples by immunohistochem.  The relationship between LSD1 expression, clinicopathol. features and patient survival was investigated.  Results: Immunohistochem., Western blotting, and qRT-PCR consistently confirmed LSD1 overexpression in HCC tissues compared to adjacent non-neoplastic tissues (P < 0.01).  Addnl., immunostaining showed more LSD1-pos. cells in the higher tumor stage (T3-4) and tumor grade (G3) than in the lower tumor stage (T1-2, P < 0.001) and tumor grade (G1-2, P < 0.001), resp.  Moreover, HCC patients with high LSD1 expression had significantly lower 5-yr overall survival rates (P < 0.001) and lower 5-yr disease-free survival rates (P < 0.001), resp.  A Cox proportional hazards model further demonstrated that LSD1 over-expression was an independent predictor of poor prognosis for both 5-yr disease-free survival [hazards ratio (HR) = 1.426, 95%CI: 0.672-2.146, P < 0.001] and 5-yr overall survival (HR = 2.456, 95%CI: 1.234-3.932, P < 0.001) in HCC.  Conclusion: Our data suggest for the first time that the overexpression of LSD1 protein in HCC tissues indicates tumor progression and predicts poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqX4-UuO3NnLVg90H21EOLACvtfcHk0ljOCq3k1ivnUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ&md5=c657f6b1e0b5e183c28fefbda4cc8745</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v18.i45.6651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v18.i45.6651%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.-K.%26aulast%3DYu%26aufirst%3DH.-F.%26aulast%3DWang%26aufirst%3DD.-R.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DW.-G.%26aulast%3DDing%26aufirst%3DW.-J.%26aulast%3DLiu%26aufirst%3DY.-B.%26atitle%3DOverexpression%2520of%2520lysine%2520specific%2520demethylase%25201%2520predicts%2520worse%2520prognosis%2520in%2520primary%2520hepatocellular%2520carcinoma%2520patients%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2012%26volume%3D18%26spage%3D6651%26epage%3D6656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Novel oligoamine analogues inhibit lysine-specific demethylase 1 (LSD1) and induce re-expression of epigenetically silenced genes</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">7217</span><span class="NLM_x">–</span> <span class="NLM_lpage">7228</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1158%2F1078-0432.CCR-09-1293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=19934284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=7217-7228&author=Y.+Huangauthor=T.+M.+Stewartauthor=Y.+Wuauthor=S.+B.+Baylinauthor=L.+J.+Martonauthor=B.+Perkinsauthor=R.+J.+Jonesauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Novel+oligoamine+analogues+inhibit+lysine-specific+demethylase+1+%28LSD1%29+and+induce+re-expression+of+epigenetically+silenced+genes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oligoamine Analogues Inhibit Lysine-Specific Demethylase 1 and Induce Reexpression of Epigenetically Silenced Genes</span></div><div class="casAuthors">Huang, Yi; Stewart, Tracy Murray; Wu, Yu; Baylin, Stephen B.; Marton, Laurence J.; Perkins, Brandy; Jones, Richard J.; Woster, Patrick M.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7217-7228</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Abnormal DNA CpG island hypermethylation and transcriptionally repressive histone modifications are assocd. with the aberrant silencing of tumor suppressor genes.  Lysine methylation is a dynamic, enzymically controlled process.  Lysine-specific demethylase 1 (LSD1) has recently been identified as a histone lysine demethylase.  LSD1 specifically catalyzes demethylation of mono- and dimethyl-lysine 4 of histone 3 (H3K4), key pos. chromatin marks assocd. with transcriptional activation.  We hypothesized that a novel class of oligoamine analogs would effectively inhibit LSD1 and thus cause the reexpression of aberrantly silenced genes.  Exptl. Design: Human colorectal cancer cells were treated with the oligoamines and changes in mono- and dimethyl-H3K4 and other chromatin marks were monitored.  In addn., treated cells were evaluated for the reexpression of the aberrantly silenced secreted frizzled-related proteins (SFRP) Wnt signaling pathway antagonist genes.  Finally, the effects of the LSD1 inhibitors were evaluated in an in vivo xenograft model.  Results: Treatment of HCT116 human colon adenocarcinoma cells in vitro resulted in increased H3K4 methylation and reexpression of silenced SFRP genes.  This reexpression is also accompanied by a decrease in H3K9me2 repressive mark.  Importantly, cotreatment with low doses of oligoamines and a DNA methyltransferase inhibitor highly induces the reexpression of the aberrantly silenced SFRP2 gene and results in significant inhibition of the growth of established tumors in a human colon tumor model in vivo.  Conclusions: The use of LSD1-inhibiting oligoamine analogs in combination with DNA methyltransferase inhibitors represents a highly promising and novel approach for epigenetic therapy of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk1naS7xW5GbVg90H21EOLACvtfcHk0ljOCq3k1ivnUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE&md5=0f86cd2d3482aa0a36adacc4f10329dd</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-1293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-1293%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DStewart%26aufirst%3DT.%2BM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DMarton%26aufirst%3DL.%2BJ.%26aulast%3DPerkins%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DR.%2BJ.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DNovel%2520oligoamine%2520analogues%2520inhibit%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520and%2520induce%2520re-expression%2520of%2520epigenetically%2520silenced%2520genes%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D7217%26epage%3D7228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quenelle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascaro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">A Novel Selective LSD1/KDM1A Inhibitor Epigenetically Blocks Herpes Simplex Virus Lytic Replication and Reactivation from Latency</span> <span class="citation_source-journal">mBio</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">00558-12</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=00558-12&author=Y.+Liangauthor=D.+Quenelleauthor=J.+L.+Vogelauthor=C.+Mascaroauthor=A.+Ortegaauthor=T.+M.+Kristie&title=A+Novel+Selective+LSD1%2FKDM1A+Inhibitor+Epigenetically+Blocks+Herpes+Simplex+Virus+Lytic+Replication+and+Reactivation+from+Latency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DQuenelle%26aufirst%3DD.%26aulast%3DVogel%26aufirst%3DJ.%2BL.%26aulast%3DMascaro%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DA%2520Novel%2520Selective%2520LSD1%252FKDM1A%2520Inhibitor%2520Epigenetically%2520Blocks%2520Herpes%2520Simplex%2520Virus%2520Lytic%2520Replication%2520and%2520Reactivation%2520from%2520Latency%26jtitle%3DmBio%26date%3D2013%26volume%3D4%26spage%3D00558-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally Designed <i>trans</i>-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+Designed+trans-2-Phenylcyclopropylamine+Derivatives+Potently+Inhibit+Histone+Demethylase+LSD1%2FKDM1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lgmz7m4VktjEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520Designed%2520trans-2-Phenylcyclopropylamine%2520Derivatives%2520Potently%2520Inhibit%2520Histone%2520Demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+Structural%2C+and+Biological+Evaluation+of+Tranylcypromine+Derivatives+as+Inhibitors+of+Histone+Demethylases+LSD1+and+LSD2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520Structural%252C%2520and%2520Biological%2520Evaluation%2520of%2520Tranylcypromine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Histone%2520Demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of Cell-Active Lysine Specific Demethylase 1-Selective Inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">17536</span><span class="NLM_x">–</span> <span class="NLM_lpage">17537</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja907055q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=17536-17537&author=R.+Uedaauthor=T.+Suzukiauthor=K.+Minoauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyata&title=Identification+of+Cell-Active+Lysine+Specific+Demethylase+1-Selective+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja907055q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja907055q%26sid%3Dliteratum%253Aachs%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520Cell-Active%2520Lysine%2520Specific%2520Demethylase%25201-Selective%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D17536%26epage%3D17537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Ortega, A. M.; Castro-Palomino, L. J.; Fyfe, M. C. T.</span><span> </span><span class="NLM_article-title">Lysine specific demethylase inhibitors and their use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div> WO 2011035941 A1<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=A.+M.+Ortega&author=L.+J.+Castro-Palomino&author=M.+C.+T.+Fyfe&title=Lysine+specific+demethylase+inhibitors+and+their+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%26aufirst%3DA.%2BM.%26atitle%3DLysine%2520specific%2520demethylase%2520inhibitors%2520and%2520their%2520use%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Guibourt, N.; Ortega, A. M.; Castro-Palomino, L. J.</span><span> </span><span class="NLM_article-title">Phenylcyclopropylamine derivatives and their medical use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div>, WO 2010084160 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=N.+Guibourt&author=A.+M.+Ortega&author=L.+J.+Castro-Palomino&title=Phenylcyclopropylamine+derivatives+and+their+medical+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGuibourt%26aufirst%3DN.%26atitle%3DPhenylcyclopropylamine%2520derivatives%2520and%2520their%2520medical%2520use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Guibourt, N.; Ortega, A. M.; Castro-Palomino, L. J.</span><span> </span><span class="NLM_article-title">Oxidase inhibitors and their use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div> WO 2010043721 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=N.+Guibourt&author=A.+M.+Ortega&author=L.+J.+Castro-Palomino&title=Oxidase+inhibitors+and+their+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGuibourt%26aufirst%3DN.%26atitle%3DOxidase%2520inhibitors%2520and%2520their%2520use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">McCafferty, D. G.; Pollock, J.</span><span> </span><span class="NLM_article-title">Arylcyclopropylamines and methods of use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div>  US 20100324147 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=D.+G.+McCafferty&author=J.+Pollock&title=Arylcyclopropylamines+and+methods+of+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DArylcyclopropylamines%2520and%2520methods%2520of%2520use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A Mechanism-Based Inactivator for Histone Demethylase LSD1</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">4536</span><span class="NLM_x">–</span> <span class="NLM_lpage">4537</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0602748" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=4536-4537&author=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=X.+Liuauthor=G.+Daauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=A+Mechanism-Based+Inactivator+for+Histone+Demethylase+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja0602748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0602748%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDa%26aufirst%3DG.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520Mechanism-Based%2520Inactivator%2520for%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D4536%26epage%3D4537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Comparative Analysis of Small Molecules and Histone Substrate Analogs as LSD1 Lysine Demethylase Inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">3164</span><span class="NLM_x">–</span> <span class="NLM_lpage">3176</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja909996p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=3164-3176&author=J.+C.+Culhaneauthor=D.+Wangauthor=P.+M.+Yenauthor=P.+A.+Cole&title=Comparative+Analysis+of+Small+Molecules+and+Histone+Substrate+Analogs+as+LSD1+Lysine+Demethylase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja909996p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909996p%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DComparative%2520Analysis%2520of%2520Small%2520Molecules%2520and%2520Histone%2520Substrate%2520Analogs%2520as%2520LSD1%2520Lysine%2520Demethylase%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D3164%26epage%3D3176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Sharma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbergs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowley, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span> </span><span class="NLM_article-title">(Bis)urea and (Bis)thiourea Inhibitors of Lysine-Specific Demethylase 1 as Epigenetic Modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5197</span><span class="NLM_x">–</span> <span class="NLM_lpage">5212</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100217a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5197-5212&author=S.+K.+Sharmaauthor=Y.+Wuauthor=N.+Steinbergsauthor=M.+L.+Crowleyauthor=A.+S.+Hansonauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=%28Bis%29urea+and+%28Bis%29thiourea+Inhibitors+of+Lysine-Specific+Demethylase+1+as+Epigenetic+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm100217a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100217a%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BK.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSteinbergs%26aufirst%3DN.%26aulast%3DCrowley%26aufirst%3DM.%2BL.%26aulast%3DHanson%26aufirst%3DA.%2BS.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D%2528Bis%2529urea%2520and%2520%2528Bis%2529thiourea%2520Inhibitors%2520of%2520Lysine-Specific%2520Demethylase%25201%2520as%2520Epigenetic%2520Modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5197%26epage%3D5212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray Stewart, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodwin, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8023</span><span class="NLM_x">–</span> <span class="NLM_lpage">8028</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+Murray+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMurray%2BStewart%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Hui.</span><span> </span><span class="NLM_article-title">Novel Histone Demethylase LSD1 Inhibitors Selectively Target Cancer Cells with Pluripotent Stem Cell Properties</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">7238</span><span class="NLM_x">–</span> <span class="NLM_lpage">7249</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=7238-7249&author=J.+Wangauthor=F.+Luauthor=Q.+Renauthor=H.+Sunauthor=Z.+Xuauthor=R.+Lanauthor=Y.+Liuauthor=D.+Wardauthor=J.+Quanauthor=T.+Yeauthor=Hui.+Zhang&title=Novel+Histone+Demethylase+LSD1+Inhibitors+Selectively+Target+Cancer+Cells+with+Pluripotent+Stem+Cell+Properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DF.%26aulast%3DRen%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLan%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWard%26aufirst%3DD.%26aulast%3DQuan%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DHui.%26atitle%3DNovel%2520Histone%2520Demethylase%2520LSD1%2520Inhibitors%2520Selectively%2520Target%2520Cancer%2520Cells%2520with%2520Pluripotent%2520Stem%2520Cell%2520Properties%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D7238%26epage%3D7249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jandausch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forne, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">2704</span><span class="NLM_x">–</span> <span class="NLM_lpage">2709</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1002%2Fijc.27555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=22447389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2704-2709&author=D.+Willmannauthor=S.+Limauthor=S.+Wetzelauthor=E.+Metzgerauthor=A.+Jandauschauthor=W.+Wilkauthor=M.+Jungauthor=I.+Forneauthor=A.+Imhofauthor=A.+Janzerauthor=J.+Kirfelauthor=H.+Waldmannauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Impairment+of+prostate+cancer+cell+growth+by+a+selective+and+reversible+lysine-specific+demethylase+1+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span></div><div class="casAuthors">Willmann, Dominica; Lim, Soyoung; Wetzel, Stefan; Metzger, Eric; Jandausch, Anett; Wilk, Wolfram; Jung, Manfred; Forne, Ignasi; Imhof, Axel; Janzer, Andreas; Kirfel, Jutta; Waldmann, Herbert; Schuele, Roland; Buettner, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2704-2709</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Post-translational modifications of histones by chromatin modifying enzymes regulate chromatin structure and gene expression.  As deregulation of histone modifications contributes to cancer progression, inhibition of chromatin modifying enzymes such as histone demethylases is an attractive therapeutic strategy to impair cancer growth.  Lysine-specific demethylase 1 (LSD1) removes mono- and di-Me marks from lysine 4 or 9 of histone H3.  LSD1 in assocn. with the androgen receptor (AR) controls androgen-dependent gene expression and prostate tumor cell proliferation, thus highlighting LSD1 as a drug target.  By combining protein structure similarity clustering and in vitro screening, we identified Namoline, a γ-pyrone, as a novel, selective and reversible LSD1 inhibitor.  Namoline blocks LSD1 demethylase activity in vitro and in vivo.  Inhibition of LSD1 by Namoline leads to silencing of AR-regulated gene expression and severely impairs androgen-dependent proliferation in vitro and in vivo.  Thus, Namoline is a novel promising starting compd. for the development of therapeutics to treat androgen-dependent prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKxYi0f1zoOrVg90H21EOLACvtfcHk0ljNv1zYGqVCCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D&md5=3104057823357b199b635e1a22071b2a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fijc.27555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27555%26sid%3Dliteratum%253Aachs%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DJandausch%26aufirst%3DA.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DForne%26aufirst%3DI.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DImpairment%2520of%2520prostate%2520cancer%2520cell%2520growth%2520by%2520a%2520selective%2520and%2520reversible%2520lysine-specific%2520demethylase%25201%2520inhibitor%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26spage%3D2704%26epage%3D2709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Vankayalapati, H.; Sorna, V.; Warner, S. L.; Bearss, D. J.; Sharma, S.; Stephens, B.</span><span> </span><span class="NLM_article-title">Substituted (<i>E</i>)-<i>N</i>′-(1-phenylethylidene) benzohydrazide analogs as histone demethylase inhibitors</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div> WO 2013025805 A1,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=H.+Vankayalapati&author=V.+Sorna&author=S.+L.+Warner&author=D.+J.+Bearss&author=S.+Sharma&author=B.+Stephens&title=Substituted+%28E%29-N%E2%80%B2-%281-phenylethylidene%29+benzohydrazide+analogs+as+histone+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVankayalapati%26aufirst%3DH.%26atitle%3DSubstituted%2520%2528E%2529-N%25E2%2580%25B2-%25281-phenylethylidene%2529%2520benzohydrazide%2520analogs%2520as%2520histone%2520demethylase%2520inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengoku, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuzawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Crystal structure of histone demethylase LSD1 and tranylcypromine at 2.25 Å</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">366</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2008&pages=15-22&author=S.+Mimasuauthor=T.+Sengokuauthor=S.+Fukuzawaauthor=T.+Umeharaauthor=S.+Yokoyama&title=Crystal+structure+of+histone+demethylase+LSD1+and+tranylcypromine+at+2.25+%C3%85"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DSengoku%26aufirst%3DT.%26aulast%3DFukuzawa%26aufirst%3DS.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520histone%2520demethylase%2520LSD1%2520and%2520tranylcypromine%2520at%25202.25%2520%25C3%2585%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2008%26volume%3D366%26spage%3D15%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Gooden, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabadi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Facile synthesis of substituted <i>trans</i>-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3047</span><span class="NLM_x">–</span> <span class="NLM_lpage">3051</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.bmcl.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=18242989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3047-3051&author=D.+M.+Goodenauthor=D.+M.+Z.+Schmidtauthor=J.+A.+Pollockauthor=A.+M.+Kabadiauthor=D.+G.+McCafferty&title=Facile+synthesis+of+substituted+trans-2-arylcyclopropylamine+inhibitors+of+the+human+histone+demethylase+LSD1+and+monoamine+oxidases+A+and+B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span></div><div class="casAuthors">Gooden, David M.; Schmidt, Dawn M. Z.; Pollock, Julie A.; Kabadi, Ami M.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3047-3051</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A facile synthetic route to substituted trans-2-arylcyclopropylamines was developed to provide access to mechanism-based inhibitors of the human flavoenzyme oxidase lysine-specific histone demethylase LSD1 and related enzyme family members such as monoamine oxidases A and B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt7YjHKC3yrVg90H21EOLACvtfcHk0ljeFWHZn9apgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D&md5=2459dcb4cde160186fc17c09258c5906</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGooden%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%2BZ.%26aulast%3DPollock%26aufirst%3DJ.%2BA.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DFacile%2520synthesis%2520of%2520substituted%2520trans-2-arylcyclopropylamine%2520inhibitors%2520of%2520the%2520human%2520histone%2520demethylase%2520LSD1%2520and%2520monoamine%2520oxidases%2520A%2520and%2520B%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3047%26epage%3D3051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, Y.-I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The SNAG domain of Snail1 functions as a molecular hook for recruiting lysine-specific demethylase 1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1803</span><span class="NLM_x">–</span> <span class="NLM_lpage">1816</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=1803-1816&author=Y.+Linauthor=Y.+Wuauthor=J.+Liauthor=C.+Dongauthor=X.+Yeauthor=Y.-I.+Chiauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+SNAG+domain+of+Snail1+functions+as+a+molecular+hook+for+recruiting+lysine-specific+demethylase+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DChi%26aufirst%3DY.-I.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520SNAG%2520domain%2520of%2520Snail1%2520functions%2520as%2520a%2520molecular%2520hook%2520for%2520recruiting%2520lysine-specific%2520demethylase%25201%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D1803%26epage%3D1816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Laurent, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randrianarison-Huetz, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frisan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrieu-Soler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontenay, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dusanter-Fourt, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duménil, D.</span><span> </span><span class="NLM_article-title">A short Gfi-1B isoform controls erythroid differentiation by recruiting the LSD1–CoREST complex through the dimethylation of its SNAG domain</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">1002</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1242%2Fjcs.095877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmsl2nu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2012&pages=993-1002&author=B.+Laurentauthor=V.+Randrianarison-Huetzauthor=E.+Frisanauthor=C.+Andrieu-Solerauthor=E.+Solerauthor=M.+Fontenayauthor=I.+Dusanter-Fourtauthor=D.+Dum%C3%A9nil&title=A+short+Gfi-1B+isoform+controls+erythroid+differentiation+by+recruiting+the+LSD1%E2%80%93CoREST+complex+through+the+dimethylation+of+its+SNAG+domain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A short Gfi-1B isoform controls erythroid differentiation by recruiting the LSD1-CoREST complex through the dimethylation of its SNAG domain</span></div><div class="casAuthors">Laurent, Benoit; Randrianarison-Huetz, Voahangy; Frisan, Emilie; Andrieu-Soler, Charlotte; Soler, Eric; Fontenay, Michaela; Dusanter-Fourt, Isabelle; Dumenil, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">993-1002</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Gfi-1B is a transcriptional repressor essential for the regulation of erythropoiesis and megakaryopoiesis.  Here we identify Gfi-1B p32, a Gfi-1B isoform, as essential for erythroid differentiation.  Gfi-1B p32 is generated by alternative splicing and lacks the two first zinc finger domains of the protein.  Selective knock down of Gfi-1B p32 compromises erythroid differentiation, whereas its ectopic expression induces erythropoiesis in the absence of erythropoietin.  Gfi-1B p32 isoform binds to Gfi-1B target gene promoters and assocs. with the LSD1-CoREST repressor complex more efficiently than the major Gfi-1B p37 isoform.  Furthermore, we show that Gfi-1B includes a KSKK motif in its SNAG domain, which recruits the repressor complex only when dimethylated on lysine 8.  Mutation of lysine 8 prevents Gfi-1B p32-induced erythroid development.  Our results thus highlight a key role for the alternatively spliced Gfi-1B p32 isoform in erythroid development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3kloXBVMss7Vg90H21EOLACvtfcHk0ljnCFZb2TsnQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmsl2nu7w%253D&md5=6a9672c52e60a5ececdfaccdd1606706</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1242%2Fjcs.095877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.095877%26sid%3Dliteratum%253Aachs%26aulast%3DLaurent%26aufirst%3DB.%26aulast%3DRandrianarison-Huetz%26aufirst%3DV.%26aulast%3DFrisan%26aufirst%3DE.%26aulast%3DAndrieu-Soler%26aufirst%3DC.%26aulast%3DSoler%26aufirst%3DE.%26aulast%3DFontenay%26aufirst%3DM.%26aulast%3DDusanter-Fourt%26aufirst%3DI.%26aulast%3DDum%25C3%25A9nil%26aufirst%3DD.%26atitle%3DA%2520short%2520Gfi-1B%2520isoform%2520controls%2520erythroid%2520differentiation%2520by%2520recruiting%2520the%2520LSD1%25E2%2580%2593CoREST%2520complex%2520through%2520the%2520dimethylation%2520of%2520its%2520SNAG%2520domain%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2012%26volume%3D125%26spage%3D993%26epage%3D1002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Saleque, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooke, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Epigenetic Regulation of Hematopoietic Differentiation by Gfi-1 and Gfi-1b Is Mediated by the Cofactors CoREST and LSD1</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=562-572&author=S.+Salequeauthor=J.+Kimauthor=H.+M.+Rookeauthor=S.+H.+Orkin&title=Epigenetic+Regulation+of+Hematopoietic+Differentiation+by+Gfi-1+and+Gfi-1b+Is+Mediated+by+the+Cofactors+CoREST+and+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaleque%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DRooke%26aufirst%3DH.%2BM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DEpigenetic%2520Regulation%2520of%2520Hematopoietic%2520Differentiation%2520by%2520Gfi-1%2520and%2520Gfi-1b%2520Is%2520Mediated%2520by%2520the%2520Cofactors%2520CoREST%2520and%2520LSD1%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D562%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljD276td8aKlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichert, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revelo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topham, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stafforini, D. M.</span><span> </span><span class="NLM_article-title">Deficiency of Phospholipase A2 Group 7 Decreases Intestinal Polyposis and Colon Tumorigenesis in ApcMin/+ Mice</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">2806</span><span class="NLM_x">–</span> <span class="NLM_lpage">2816</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=2806-2816&author=C.+Xuauthor=E.+C.+Reichertauthor=T.+Nakanoauthor=M.+Lohseauthor=A.+A.+Gardnerauthor=M.+P.+Reveloauthor=M.+K.+Tophamauthor=D.+M.+Stafforini&title=Deficiency+of+Phospholipase+A2+Group+7+Decreases+Intestinal+Polyposis+and+Colon+Tumorigenesis+in+ApcMin%2F%2B+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DC.%26aulast%3DReichert%26aufirst%3DE.%2BC.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DLohse%26aufirst%3DM.%26aulast%3DGardner%26aufirst%3DA.%2BA.%26aulast%3DRevelo%26aufirst%3DM.%2BP.%26aulast%3DTopham%26aufirst%3DM.%2BK.%26aulast%3DStafforini%26aufirst%3DD.%2BM.%26atitle%3DDeficiency%2520of%2520Phospholipase%2520A2%2520Group%25207%2520Decreases%2520Intestinal%2520Polyposis%2520and%2520Colon%2520Tumorigenesis%2520in%2520ApcMin%252F%252B%2520Mice%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D2806%26epage%3D2816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 111 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Lei-Peng Xue, Xiao-Peng Xiong, Ying Zhou, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2466-2488. <a href="https://doi.org/10.1021/acs.jmedchem.0c02176" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02176%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DReversible%252BLysine%252BSpecific%252BDemethylase%252B1%252B%252528LSD1%252529%252BInhibitors%25253A%252BA%252BPromising%252BWrench%252Bto%252BImpair%252BLSD1%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D23022021%26volume%3D64%26issue%3D5%26spage%3D2466%26epage%3D2488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Xiao-Peng Xiong, Lei-Peng Xue, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Tranylcypromine Based Lysine-Specific Demethylase 1 Inhibitor: Summary and Perspective. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14197-14215. <a href="https://doi.org/10.1021/acs.jmedchem.0c00919" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00919</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00919%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTranylcypromine%252BBased%252BLysine-Specific%252BDemethylase%252B1%252BInhibitor%25253A%252BSummary%252Band%252BPerspective%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30052020%26date%3D30092020%26date%3D15092020%26volume%3D63%26issue%3D23%26spage%3D14197%26epage%3D14215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Waleed A.  Zalloum</span> and <span class="hlFld-ContribAuthor ">Hiba M.  Zalloum</span>  . </span><span class="cited-content_cbyCitation_article-title">Exploring the Active Center of the LSD1/CoREST Complex by Molecular Dynamics Simulation Utilizing Its Co-crystallized Co-factor Tetrahydrofolate as a Probe. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (12)
                                     , 3022-3031. <a href="https://doi.org/10.1021/acs.jcim.7b00256" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00256%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DExploring%252Bthe%252BActive%252BCenter%252Bof%252Bthe%252BLSD1%25252FCoREST%252BComplex%252Bby%252BMolecular%252BDynamics%252BSimulation%252BUtilizing%252BIts%252BCo-crystallized%252BCo-factor%252BTetrahydrofolate%252Bas%252Ba%252BProbe%26aulast%3DZalloum%26aufirst%3DWaleed%2BA.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D08052017%26date%3D06122017%26date%3D26122017%26date%3D21112017%26volume%3D57%26issue%3D12%26spage%3D3022%26epage%3D3031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Xia</span>, <span class="hlFld-ContribAuthor ">Jui-Hua  Hsieh</span>, <span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Song  Wu</span>, and <span class="hlFld-ContribAuthor ">Xiang Simon  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">The Development of Target-Specific Pose Filter Ensembles To Boost Ligand Enrichment for Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (6)
                                     , 1414-1425. <a href="https://doi.org/10.1021/acs.jcim.6b00749" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00749%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BDevelopment%252Bof%252BTarget-Specific%252BPose%252BFilter%252BEnsembles%252BTo%252BBoost%252BLigand%252BEnrichment%252Bfor%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DXia%26aufirst%3DJie%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D12122016%26date%3D01062017%26date%3D26062017%26date%3D16052017%26volume%3D57%26issue%3D6%26spage%3D1414%26epage%3D1425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting-Ting  Yao</span>, <span class="hlFld-ContribAuthor ">Shao-Wei  Fang</span>, <span class="hlFld-ContribAuthor ">Zhong-Shan  Li</span>, <span class="hlFld-ContribAuthor ">Dou-Xin  Xiao</span>, <span class="hlFld-ContribAuthor ">Jing-Li  Cheng</span>, <span class="hlFld-ContribAuthor ">Hua-Zhou  Ying</span>, <span class="hlFld-ContribAuthor ">Yong-Jun  Du</span>, <span class="hlFld-ContribAuthor ">Jin-Hao  Zhao</span>, and <span class="hlFld-ContribAuthor ">Xiao-Wu  Dong</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Succinate Dehydrogenase Inhibitors by the Integration of in Silico Library Design and Pharmacophore Mapping. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2017,</strong> <em>65 </em>
                                    (15)
                                     , 3204-3211. <a href="https://doi.org/10.1021/acs.jafc.7b00249" title="DOI URL">https://doi.org/10.1021/acs.jafc.7b00249</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.7b00249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.7b00249%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BSuccinate%252BDehydrogenase%252BInhibitors%252Bby%252Bthe%252BIntegration%252Bof%252Bin%252BSilico%252BLibrary%252BDesign%252Band%252BPharmacophore%252BMapping%26aulast%3DYao%26aufirst%3DTing-Ting%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D17012017%26date%3D29032017%26date%3D30032017%26date%3D05042017%26date%3D19042017%26date%3D30032017%26volume%3D65%26issue%3D15%26spage%3D3204%26epage%3D3211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Xue-Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Peng-Fei  Geng</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Jin-Lian  Ma</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Tao-Qian  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhao-Qing  Zhou</span>, <span class="hlFld-ContribAuthor ">Chen-Xi  Huang</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, and <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of [1,2,3]Triazolo[4,5-d]pyrimidine Derivatives as Novel LSD1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (4)
                                     , 384-389. <a href="https://doi.org/10.1021/acsmedchemlett.6b00423" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.6b00423</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.6b00423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.6b00423%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252B%25255B1%25252C2%25252C3%25255DTriazolo%25255B4%25252C5-d%25255Dpyrimidine%252BDerivatives%252Bas%252BNovel%252BLSD1%252BInhibitors%26aulast%3DLi%26aufirst%3DZhong-Hua%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D25102016%26date%3D06032017%26date%3D08032017%26date%3D13042017%26date%3D06032017%26volume%3D8%26issue%3D4%26spage%3D384%26epage%3D389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Luca  Sartori</span>, <span class="hlFld-ContribAuthor ">Federica  Amigoni</span>, <span class="hlFld-ContribAuthor ">Anna  Cappa</span>, <span class="hlFld-ContribAuthor ">Giovanni  Fagá</span>, <span class="hlFld-ContribAuthor ">Raimondo  Fattori</span>, <span class="hlFld-ContribAuthor ">Elena  Legnaghi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Meroni</span>, <span class="hlFld-ContribAuthor ">Loris  Moretti</span>, <span class="hlFld-ContribAuthor ">Valentina  Cecatiello</span>, <span class="hlFld-ContribAuthor ">Sebastiano  Pasqualato</span>, <span class="hlFld-ContribAuthor ">Alessia  Romussi</span>, <span class="hlFld-ContribAuthor ">Florian  Thaler</span>, <span class="hlFld-ContribAuthor ">Paolo  Trifiró</span>, <span class="hlFld-ContribAuthor ">Manuela  Villa</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Stefania  Vultaggio</span>, <span class="hlFld-ContribAuthor ">Elisa  Zagarrí</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, and <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>  . </span><span class="cited-content_cbyCitation_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1693-1715. <a href="https://doi.org/10.1021/acs.jmedchem.6b01019" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxamides%252Bas%252BNew%252BReversible%252BInhibitors%252Bof%252BHistone%252BLysine%252BDemethylase%252BKDM1A%25252FLSD1.%252BPart%252B2%25253A%252BStructure-Based%252BDrug%252BDesign%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%26aulast%3DVianello%26aufirst%3DPaola%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D26072016%26date%3D27022017%26date%3D09032017%26date%3D10022017%26volume%3D60%26issue%3D5%26spage%3D1693%26epage%3D1715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Irwin</span> and <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking Screens for Novel Ligands Conferring New Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4103-4120. <a href="https://doi.org/10.1021/acs.jmedchem.5b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BScreens%252Bfor%252BNovel%252BLigands%252BConferring%252BNew%252BBiology%26aulast%3DIrwin%26aufirst%3DJohn%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D25122015%26date%3D15032016%26date%3D12052016%26date%3D25022016%26volume%3D59%26issue%3D9%26spage%3D4103%26epage%3D4120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fangrui  Wu</span>, <span class="hlFld-ContribAuthor ">Chao  Zhou</span>, <span class="hlFld-ContribAuthor ">Yuan  Yao</span>, <span class="hlFld-ContribAuthor ">Liping  Wei</span>, <span class="hlFld-ContribAuthor ">Zizhen  Feng</span>, <span class="hlFld-ContribAuthor ">Lisheng  Deng</span>, and <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>  . </span><span class="cited-content_cbyCitation_article-title">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (1)
                                     , 253-263. <a href="https://doi.org/10.1021/acs.jmedchem.5b01361" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01361</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D3-%252528Piperidin-4-ylmethoxy%252529pyridine%252BContaining%252BCompounds%252BAre%252BPotent%252BInhibitors%252Bof%252BLysine%252BSpecific%252BDemethylase%252B1%26aulast%3DWu%26aufirst%3DFangrui%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D03092015%26date%3D24122015%26date%3D14012016%26date%3D10122015%26volume%3D59%26issue%3D1%26spage%3D253%26epage%3D263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fanwang  Meng</span>, <span class="hlFld-ContribAuthor ">Sufang  Cheng</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Shien  Liu</span>, <span class="hlFld-ContribAuthor ">Yan  Liu</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Shijie  Chen</span>, <span class="hlFld-ContribAuthor ">Junyan  Lu</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Linjuan  Li</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Zhe  Shi</span>, <span class="hlFld-ContribAuthor ">Yu  Zhou</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Wencong  Lu</span>, <span class="hlFld-ContribAuthor ">Hong  Liu</span>, and <span class="hlFld-ContribAuthor ">Cheng  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (20)
                                     , 8166-8181. <a href="https://doi.org/10.1021/acs.jmedchem.5b01154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01154</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252BNovel%25252C%252BSelective%252BHistone%252BMethyltransferase%252BSET7%252BInhibitors%252Bby%252BPharmacophore-%252Band%252BDocking-Based%252BVirtual%252BScreening%26aulast%3DMeng%26aufirst%3DFanwang%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D06072015%26date%3D30092015%26date%3D22102015%26date%3D21092015%26volume%3D58%26issue%3D20%26spage%3D8166%26epage%3D8181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Sai-Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Bo  Wang</span>, <span class="hlFld-ContribAuthor ">Zhi-Ru  Wang</span>, <span class="hlFld-ContribAuthor ">Lu-Ping  Pang</span>, <span class="hlFld-ContribAuthor ">Miao  Zhang</span>, <span class="hlFld-ContribAuthor ">Jun-Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Juan  Li</span>, <span class="hlFld-ContribAuthor ">Cong  Wang</span>, <span class="hlFld-ContribAuthor ">Biao  Hu</span>, <span class="hlFld-ContribAuthor ">Ying  Liu</span>, <span class="hlFld-ContribAuthor ">Xiao-Dan  Zhang</span>, <span class="hlFld-ContribAuthor ">Jia-Jia  Wang</span>, <span class="hlFld-ContribAuthor ">Zhi-Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, and <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>  . </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Structure–Activity Relationship of Novel LSD1 Inhibitors Based on Pyrimidine–Thiourea Hybrids As Potent, Orally Active Antitumor Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (4)
                                     , 1705-1716. <a href="https://doi.org/10.1021/acs.jmedchem.5b00037" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b00037%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%252Bof%252BNovel%252BLSD1%252BInhibitors%252BBased%252Bon%252BPyrimidine%2525E2%252580%252593Thiourea%252BHybrids%252BAs%252BPotent%25252C%252BOrally%252BActive%252BAntitumor%252BAgents%26aulast%3DMa%26aufirst%3DLi-Ying%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D09092014%26date%3D06022015%26date%3D26022015%26date%3D22012015%26volume%3D58%26issue%3D4%26spage%3D1705%26epage%3D1716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Veronica  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Bruna  Saponara</span>, <span class="hlFld-ContribAuthor ">Roberto  Cirilli</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Donatella  Labella</span>, <span class="hlFld-ContribAuthor ">Biagina  Marrocco</span>, <span class="hlFld-ContribAuthor ">Giovanni  Ruoppolo</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, and <span class="hlFld-ContribAuthor ">Antonello  Mai</span>  . </span><span class="cited-content_cbyCitation_article-title">Pure Diastereomers of a Tranylcypromine-Based LSD1 Inhibitor: Enzyme Selectivity and In-Cell Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2015,</strong> <em>6 </em>
                                    (2)
                                     , 173-177. <a href="https://doi.org/10.1021/ml500424z" title="DOI URL">https://doi.org/10.1021/ml500424z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml500424z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml500424z%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPure%252BDiastereomers%252Bof%252Ba%252BTranylcypromine-Based%252BLSD1%252BInhibitor%25253A%252BEnzyme%252BSelectivity%252Band%252BIn-Cell%252BStudies%26aulast%3DValente%26aufirst%3DSergio%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D18102014%26date%3D08122014%26date%3D12122014%26date%3D12022015%26date%3D08122014%26volume%3D6%26issue%3D2%26spage%3D173%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adel  Hamza</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Wagner</span>, <span class="hlFld-ContribAuthor ">Timothy J.  Evans</span>, <span class="hlFld-ContribAuthor ">Mykhaylo S.  Frasinyuk</span>, <span class="hlFld-ContribAuthor ">Stefan  Kwiatkowski</span>, <span class="hlFld-ContribAuthor ">Chang-Guo  Zhan</span>, <span class="hlFld-ContribAuthor ">David S.  Watt</span>, and <span class="hlFld-ContribAuthor ">Konstantin V.  Korotkov</span>  . </span><span class="cited-content_cbyCitation_article-title">Novel Mycosin Protease MycP1 Inhibitors Identified by Virtual Screening and 4D Fingerprints. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (4)
                                     , 1166-1173. <a href="https://doi.org/10.1021/ci500025r" title="DOI URL">https://doi.org/10.1021/ci500025r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500025r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500025r%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNovel%252BMycosin%252BProtease%252BMycP1%252BInhibitors%252BIdentified%252Bby%252BVirtual%252BScreening%252Band%252B4D%252BFingerprints%26aulast%3DHamza%26aufirst%3DAdel%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D13012014%26date%3D27032014%26date%3D28042014%26date%3D16032014%26volume%3D54%26issue%3D4%26spage%3D1166%26epage%3D1173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Lin-Feng  Jin</span>, <span class="hlFld-ContribAuthor ">Hong-Mei  Ren</span>, <span class="hlFld-ContribAuthor ">Shao-Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Yue-Jiao  Liu</span>, <span class="hlFld-ContribAuthor ">Yong-Tao  Xu</span>, <span class="hlFld-ContribAuthor ">Zi-Hao  He</span>, <span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">Hang  Yuan</span>, <span class="hlFld-ContribAuthor ">Shu-Hui  Chen</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and biological evaluation of novel dual inhibitors targeting lysine specific demethylase 1 (LSD1) and histone deacetylases (HDAC) for treatment of gastric cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113453. <a href="https://doi.org/10.1016/j.ejmech.2021.113453" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113453</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113453%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bdual%252Binhibitors%252Btargeting%252Blysine%252Bspecific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Band%252Bhistone%252Bdeacetylases%252B%252528HDAC%252529%252Bfor%252Btreatment%252Bof%252Bgastric%252Bcancer%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2021%26volume%3D220%26spage%3D113453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Hailan  Huang</span>, <span class="hlFld-ContribAuthor ">Ziheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Yixiang  Sun</span>, <span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Yanting  Gu</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel benzofuran derivatives as potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113501. <a href="https://doi.org/10.1016/j.ejmech.2021.113501" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113501</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113501%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bbenzofuran%252Bderivatives%252Bas%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26volume%3D220%26spage%3D113501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Clement  Agboyibor</span>, <span class="hlFld-ContribAuthor ">Jianshu  Dong</span>, <span class="hlFld-ContribAuthor ">Clement Yaw  Effah</span>, <span class="hlFld-ContribAuthor ">Emmanuel Kwateng  Drokow</span>, <span class="hlFld-ContribAuthor ">Waqar  Pervaiz</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, . </span><span class="cited-content_cbyCitation_article-title">LSD1 as a Biomarker and the Outcome of Its Inhibitors in the Clinical Trial: The Therapy Opportunity in Tumor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Oncology</span><span> <strong>2021,</strong> <em>2021 </em>, 1-11. <a href="https://doi.org/10.1155/2021/5512524" title="DOI URL">https://doi.org/10.1155/2021/5512524</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2021/5512524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2021%2F5512524%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Oncology%26atitle%3DLSD1%252Bas%252Ba%252BBiomarker%252Band%252Bthe%252BOutcome%252Bof%252BIts%252BInhibitors%252Bin%252Bthe%252BClinical%252BTrial%25253A%252BThe%252BTherapy%252BOpportunity%252Bin%252BTumor%26aulast%3DAgboyibor%26aufirst%3DClement%26date%3D2021%26volume%3D2021%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking, 3D-QSAR, and molecular dynamics simulations of thieno[3,2-b]pyrrole derivatives against anticancer targets of KDM1A/LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>39 </em>
                                    (4)
                                     , 1189-1202. <a href="https://doi.org/10.1080/07391102.2020.1726819" title="DOI URL">https://doi.org/10.1080/07391102.2020.1726819</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1726819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1726819%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DMolecular%252Bdocking%25252C%252B3D-QSAR%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bof%252Bthieno%25255B3%25252C2-b%25255Dpyrrole%252Bderivatives%252Bagainst%252Banticancer%252Btargets%252Bof%252BKDM1A%25252FLSD1%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1189%26epage%3D1202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chenhao  Xu</span>, <span class="hlFld-ContribAuthor ">Wenjuan  Zhou</span>, <span class="hlFld-ContribAuthor ">Guanjun  Dong</span>, <span class="hlFld-ContribAuthor ">Hui  Qiao</span>, <span class="hlFld-ContribAuthor ">Jiadi  Peng</span>, <span class="hlFld-ContribAuthor ">Pengfei  Jia</span>, <span class="hlFld-ContribAuthor ">Yuhao  Li</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Kai  Sun</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Novel [1,2,3]triazolo[4,5-d]pyrimidine derivatives containing hydrazone fragment as potent and selective anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>105 </em>, 104424. <a href="https://doi.org/10.1016/j.bioorg.2020.104424" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104424%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNovel%252B%25255B1%25252C2%25252C3%25255Dtriazolo%25255B4%25252C5-d%25255Dpyrimidine%252Bderivatives%252Bcontaining%252Bhydrazone%252Bfragment%252Bas%252Bpotent%252Band%252Bselective%252Banticancer%252Bagents%26aulast%3DXu%26aufirst%3DChenhao%26date%3D2020%26volume%3D105%26spage%3D104424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Dong</span>, <span class="hlFld-ContribAuthor ">Zhaoping  Qiu</span>, <span class="hlFld-ContribAuthor ">Yadi  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Tackle Epithelial-Mesenchymal Transition With Epigenetic Drugs in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.596239" title="DOI URL">https://doi.org/10.3389/fphar.2020.596239</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.596239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.596239%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DTackle%252BEpithelial-Mesenchymal%252BTransition%252BWith%252BEpigenetic%252BDrugs%252Bin%252BCancer%26aulast%3DDong%26aufirst%3DBo%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gang  Jia</span>, <span class="hlFld-ContribAuthor ">Shundong  Cang</span>, <span class="hlFld-ContribAuthor ">Peizhi  Ma</span>, <span class="hlFld-ContribAuthor ">Zhiyu  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Capsaicin: A “hot” KDM1A/LSD1 inhibitor from peppers. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>103 </em>, 104161. <a href="https://doi.org/10.1016/j.bioorg.2020.104161" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104161</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104161%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DCapsaicin%25253A%252BA%252B%2525E2%252580%25259Chot%2525E2%252580%25259D%252BKDM1A%25252FLSD1%252Binhibitor%252Bfrom%252Bpeppers%26aulast%3DJia%26aufirst%3DGang%26date%3D2020%26volume%3D103%26spage%3D104161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarder  Arifuzzaman</span>, <span class="hlFld-ContribAuthor ">Mst Reshma  Khatun</span>, <span class="hlFld-ContribAuthor ">Rabeya  Khatun</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging of lysine demethylases (KDMs): From pathophysiological insights to novel therapeutic opportunities. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2020,</strong> <em>129 </em>, 110392. <a href="https://doi.org/10.1016/j.biopha.2020.110392" title="DOI URL">https://doi.org/10.1016/j.biopha.2020.110392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2020.110392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2020.110392%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DEmerging%252Bof%252Blysine%252Bdemethylases%252B%252528KDMs%252529%25253A%252BFrom%252Bpathophysiological%252Binsights%252Bto%252Bnovel%252Btherapeutic%252Bopportunities%26aulast%3DArifuzzaman%26aufirst%3DSarder%26date%3D2020%26volume%3D129%26spage%3D110392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raffaella  Soldi</span>, <span class="hlFld-ContribAuthor ">Tithi  Ghosh Halder</span>, <span class="hlFld-ContribAuthor ">Alexis  Weston</span>, <span class="hlFld-ContribAuthor ">Trason  Thode</span>, <span class="hlFld-ContribAuthor ">Kevin  Drenner</span>, <span class="hlFld-ContribAuthor ">Rhonda  Lewis</span>, <span class="hlFld-ContribAuthor ">Mohan R.  Kaadige</span>, <span class="hlFld-ContribAuthor ">Shreyesi  Srivastava</span>, <span class="hlFld-ContribAuthor ">Sherin  Daniel Ampanattu</span>, <span class="hlFld-ContribAuthor ">Ryan  Rodriguez del Villar</span>, <span class="hlFld-ContribAuthor ">Jessica  Lang</span>, <span class="hlFld-ContribAuthor ">Hariprasad  Vankayalapati</span>, <span class="hlFld-ContribAuthor ">Bernard  Weissman</span>, <span class="hlFld-ContribAuthor ">Jeffrey M.  Trent</span>, <span class="hlFld-ContribAuthor ">William P. D.  Hendricks</span>, <span class="hlFld-ContribAuthor ">Sunil  Sharma</span>, . </span><span class="cited-content_cbyCitation_article-title">The novel reversible LSD1 inhibitor SP-2577 promotes anti-tumor immunity in SWItch/Sucrose-NonFermentable (SWI/SNF) complex mutated ovarian cancer. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2020,</strong> <em>15 </em>
                                    (7)
                                     , e0235705. <a href="https://doi.org/10.1371/journal.pone.0235705" title="DOI URL">https://doi.org/10.1371/journal.pone.0235705</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0235705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0235705%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DThe%252Bnovel%252Breversible%252BLSD1%252Binhibitor%252BSP-2577%252Bpromotes%252Banti-tumor%252Bimmunity%252Bin%252BSWItch%25252FSucrose-NonFermentable%252B%252528SWI%25252FSNF%252529%252Bcomplex%252Bmutated%252Bovarian%252Bcancer%26aulast%3DSoldi%26aufirst%3DRaffaella%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D7%26spage%3De0235705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Brullo</span>, <span class="hlFld-ContribAuthor ">Federica  Rapetti</span>, <span class="hlFld-ContribAuthor ">Silvana  Alfei</span>, <span class="hlFld-ContribAuthor ">Irena  Maric</span>, <span class="hlFld-ContribAuthor ">Francesca  Rizzelli</span>, <span class="hlFld-ContribAuthor ">Marina  Mapelli</span>, <span class="hlFld-ContribAuthor ">Camillo  Rosano</span>, <span class="hlFld-ContribAuthor ">Maurizio  Viale</span>, <span class="hlFld-ContribAuthor ">Olga  Bruno</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of New Antiproliferative Imidazopyrazole Acylhydrazones Able To Interact with Microtubule Systems. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (11)
                                     , 961-969. <a href="https://doi.org/10.1002/cmdc.202000122" title="DOI URL">https://doi.org/10.1002/cmdc.202000122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000122%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDiscovery%252Bof%252BNew%252BAntiproliferative%252BImidazopyrazole%252BAcylhydrazones%252BAble%252BTo%252BInteract%252Bwith%252BMicrotubule%252BSystems%26aulast%3DBrullo%26aufirst%3DChiara%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D11%26spage%3D961%26epage%3D969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xingrui  He</span>, <span class="hlFld-ContribAuthor ">Yuan  Gao</span>, <span class="hlFld-ContribAuthor ">Zi  Hui</span>, <span class="hlFld-ContribAuthor ">Guodong  Shen</span>, <span class="hlFld-ContribAuthor ">Shuo  Wang</span>, <span class="hlFld-ContribAuthor ">Tian  Xie</span>, <span class="hlFld-ContribAuthor ">Xiang-Yang  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">4-Hydroxy-3-methylbenzofuran-2-carbohydrazones as novel LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (10)
                                     , 127109. <a href="https://doi.org/10.1016/j.bmcl.2020.127109" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127109%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3D4-Hydroxy-3-methylbenzofuran-2-carbohydrazones%252Bas%252Bnovel%252BLSD1%252Binhibitors%26aulast%3DHe%26aufirst%3DXingrui%26date%3D2020%26volume%3D30%26issue%3D10%26spage%3D127109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Cai  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Jiming  Wang</span>, <span class="hlFld-ContribAuthor ">Qinwen  Jiang</span>, <span class="hlFld-ContribAuthor ">Liyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>194 </em>, 112243. <a href="https://doi.org/10.1016/j.ejmech.2020.112243" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112243</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112243%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Btetrahydroquinoline-based%252Breversible%252BLSD1%252Binhibitors%26aulast%3DWang%26aufirst%3DXinran%26date%3D2020%26volume%3D194%26spage%3D112243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongtao  Xu</span>, <span class="hlFld-ContribAuthor ">Zihao  He</span>, <span class="hlFld-ContribAuthor ">Hongyi  Liu</span>, <span class="hlFld-ContribAuthor ">Yifan  Chen</span>, <span class="hlFld-ContribAuthor ">Yunlong  Gao</span>, <span class="hlFld-ContribAuthor ">Songjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Meiting  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Lu</span>, <span class="hlFld-ContribAuthor ">Chang  Wang</span>, <span class="hlFld-ContribAuthor ">Zongya  Zhao</span>, <span class="hlFld-ContribAuthor ">Yan  Liu</span>, <span class="hlFld-ContribAuthor ">Junqiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yi  Yu</span>, <span class="hlFld-ContribAuthor ">Min  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR, molecular docking, and molecular dynamics simulation study of thieno[3,2-
              b
              ]pyrrole-5-carboxamide derivatives as LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 6927-6943. <a href="https://doi.org/10.1039/C9RA10085G" title="DOI URL">https://doi.org/10.1039/C9RA10085G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA10085G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA10085G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3D3D-QSAR%25252C%252Bmolecular%252Bdocking%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulation%252Bstudy%252Bof%252Bthieno%25255B3%25252C2-%252Bb%252B%25255Dpyrrole-5-carboxamide%252Bderivatives%252Bas%252BLSD1%252Binhibitors%26aulast%3DXu%26aufirst%3DYongtao%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D6927%26epage%3D6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Jin-Lian  Ma</span>, <span class="hlFld-ContribAuthor ">Gai-Zhi  Liu</span>, <span class="hlFld-ContribAuthor ">Xin-Hui  Zhang</span>, <span class="hlFld-ContribAuthor ">Ting-Ting  Qin</span>, <span class="hlFld-ContribAuthor ">Wei-Hong  Ren</span>, <span class="hlFld-ContribAuthor ">Tao-Qian  Zhao</span>, <span class="hlFld-ContribAuthor ">Xiao-Hui  Chen</span>, <span class="hlFld-ContribAuthor ">Zhen-Qiang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">[1,2,3]Triazolo[4,5-d]pyrimidine derivatives incorporating (thio)urea moiety as a novel scaffold for LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111989. <a href="https://doi.org/10.1016/j.ejmech.2019.111989" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111989</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111989%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3D%25255B1%25252C2%25252C3%25255DTriazolo%25255B4%25252C5-d%25255Dpyrimidine%252Bderivatives%252Bincorporating%252B%252528thio%252529urea%252Bmoiety%252Bas%252Ba%252Bnovel%252Bscaffold%252Bfor%252BLSD1%252Binhibitors%26aulast%3DLi%26aufirst%3DZhong-Hua%26date%3D2020%26volume%3D187%26spage%3D111989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sumaira  Javaid</span>, <span class="hlFld-ContribAuthor ">Syed Muhammad  Saad</span>, <span class="hlFld-ContribAuthor ">Humaira  Zafar</span>, <span class="hlFld-ContribAuthor ">Rizwana  Malik</span>, <span class="hlFld-ContribAuthor ">Khalid Mohammed  Khan</span>, <span class="hlFld-ContribAuthor ">M. Iqbal  Choudhary</span>, <span class="hlFld-ContribAuthor ">Atta-ur  Rahman</span>, . </span><span class="cited-content_cbyCitation_article-title">Thymidine phosphorylase and prostrate cancer cell proliferation inhibitory activities of synthetic 4-hydroxybenzohydrazides: In vitro, kinetic, and in silico studies. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2020,</strong> <em>15 </em>
                                    (1)
                                     , e0227549. <a href="https://doi.org/10.1371/journal.pone.0227549" title="DOI URL">https://doi.org/10.1371/journal.pone.0227549</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0227549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0227549%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DThymidine%252Bphosphorylase%252Band%252Bprostrate%252Bcancer%252Bcell%252Bproliferation%252Binhibitory%252Bactivities%252Bof%252Bsynthetic%252B4-hydroxybenzohydrazides%25253A%252BIn%252Bvitro%25252C%252Bkinetic%25252C%252Band%252Bin%252Bsilico%252Bstudies%26aulast%3DJavaid%26aufirst%3DSumaira%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D1%26spage%3De0227549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Fang</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">LSD1/KDM1A inhibitors in clinical trials: advances and prospects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2019,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-019-0811-9" title="DOI URL">https://doi.org/10.1186/s13045-019-0811-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DLSD1%25252FKDM1A%252Binhibitors%252Bin%252Bclinical%252Btrials%25253A%252Badvances%252Band%252Bprospects%26aulast%3DFang%26aufirst%3DYuan%26date%3D2019%26date%3D2019%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Qi-Bing  Gao</span>, <span class="hlFld-ContribAuthor ">Ying  Yan</span>, <span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Li</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying the novel inhibitors of lysine-specific demethylase 1 (LSD1) combining pharmacophore-based and structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (16)
                                     , 4200-4214. <a href="https://doi.org/10.1080/07391102.2018.1538903" title="DOI URL">https://doi.org/10.1080/07391102.2018.1538903</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1538903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1538903%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DIdentifying%252Bthe%252Bnovel%252Binhibitors%252Bof%252Blysine-specific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Bcombining%252Bpharmacophore-based%252Band%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DSun%26aufirst%3DXu-Dong%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D16%26spage%3D4200%26epage%3D4214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pedro  de Sena M. Pinheiro</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Rodrigues</span>, <span class="hlFld-ContribAuthor ">Rodolfo  do Couto Maia</span>, <span class="hlFld-ContribAuthor ">Sreekanth  Thota</span>, <span class="hlFld-ContribAuthor ">Carlos A.M.  Fraga</span>. </span><span class="cited-content_cbyCitation_article-title">The Use of Conformational Restriction in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (19)
                                     , 1712-1733. <a href="https://doi.org/10.2174/1568026619666190712205025" title="DOI URL">https://doi.org/10.2174/1568026619666190712205025</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666190712205025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666190712205025%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DThe%252BUse%252Bof%252BConformational%252BRestriction%252Bin%252BMedicinal%252BChemistry%26aulast%3Dde%2BSena%2BM.%2BPinheiro%26aufirst%3DPedro%26date%3D2019%26volume%3D19%26issue%3D19%26spage%3D1712%26epage%3D1733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Qi-Bing  Gao</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the binding mechanism of substituted pyridine derivatives as effective and selective lysine-specific demethylase 1 inhibitors using 3D-QSAR, molecular docking and molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (13)
                                     , 3482-3495. <a href="https://doi.org/10.1080/07391102.2018.1518158" title="DOI URL">https://doi.org/10.1080/07391102.2018.1518158</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1518158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1518158%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DProbing%252Bthe%252Bbinding%252Bmechanism%252Bof%252Bsubstituted%252Bpyridine%252Bderivatives%252Bas%252Beffective%252Band%252Bselective%252Blysine-specific%252Bdemethylase%252B1%252Binhibitors%252Busing%252B3D-QSAR%25252C%252Bmolecular%252Bdocking%252Band%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DWang%26aufirst%3DZhi-Zheng%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D13%26spage%3D3482%26epage%3D3495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhonghua  Li</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Zhongrui  Li</span>, <span class="hlFld-ContribAuthor ">Zhizheng  Wang</span>, <span class="hlFld-ContribAuthor ">Fengzhi  Suo</span>, <span class="hlFld-ContribAuthor ">Dandan  Shen</span>, <span class="hlFld-ContribAuthor ">Taoqian  Zhao</span>, <span class="hlFld-ContribAuthor ">Xudong  Sun</span>, <span class="hlFld-ContribAuthor ">Junwei  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Liu</span>, <span class="hlFld-ContribAuthor ">Liying  Ma</span>, <span class="hlFld-ContribAuthor ">Bing  Zhao</span>, <span class="hlFld-ContribAuthor ">Pengfei  Geng</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Development of the triazole-fused pyrimidine derivatives as highly potent and reversible inhibitors of histone lysine specific demethylase 1 (LSD1/KDM1A). </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2019,</strong> <em>9 </em>
                                    (4)
                                     , 794-808. <a href="https://doi.org/10.1016/j.apsb.2019.01.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2019.01.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2019.01.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2019.01.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDevelopment%252Bof%252Bthe%252Btriazole-fused%252Bpyrimidine%252Bderivatives%252Bas%252Bhighly%252Bpotent%252Band%252Breversible%252Binhibitors%252Bof%252Bhistone%252Blysine%252Bspecific%252Bdemethylase%252B1%252B%252528LSD1%25252FKDM1A%252529%26aulast%3DLi%26aufirst%3DZhonghua%26date%3D2019%26volume%3D9%26issue%3D4%26spage%3D794%26epage%3D808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Priya S.  Dalvi</span>, <span class="hlFld-ContribAuthor ">Iris F.  Macheleidt</span>, <span class="hlFld-ContribAuthor ">So-Young  Lim</span>, <span class="hlFld-ContribAuthor ">Sonja  Meemboor</span>, <span class="hlFld-ContribAuthor ">Marion  Müller</span>, <span class="hlFld-ContribAuthor ">Hannah  Eischeid-Scholz</span>, <span class="hlFld-ContribAuthor ">Stephan C.  Schaefer</span>, <span class="hlFld-ContribAuthor ">Reinhard  Buettner</span>, <span class="hlFld-ContribAuthor ">Sebastian  Klein</span>, <span class="hlFld-ContribAuthor ">Margarete  Odenthal</span>. </span><span class="cited-content_cbyCitation_article-title">LSD1 Inhibition Attenuates Tumor Growth by Disrupting PLK1 Mitotic Pathway. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Research</span><span> <strong>2019,</strong> <em>17 </em>
                                    (6)
                                     , 1326-1337. <a href="https://doi.org/10.1158/1541-7786.MCR-18-0971" title="DOI URL">https://doi.org/10.1158/1541-7786.MCR-18-0971</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1541-7786.MCR-18-0971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1541-7786.MCR-18-0971%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Research%26atitle%3DLSD1%252BInhibition%252BAttenuates%252BTumor%252BGrowth%252Bby%252BDisrupting%252BPLK1%252BMitotic%252BPathway%26aulast%3DDalvi%26aufirst%3DPriya%2BS.%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D6%26spage%3D1326%26epage%3D1337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cristina  Mascaró</span>, <span class="hlFld-ContribAuthor ">Alberto  Ortega</span>, <span class="hlFld-ContribAuthor ">Elena  Carceller</span>, <span class="hlFld-ContribAuthor ">Raquel  Ruiz Rodriguez</span>, <span class="hlFld-ContribAuthor ">Filippo  Ciceri</span>, <span class="hlFld-ContribAuthor ">Serena  Lunardi</span>, <span class="hlFld-ContribAuthor ">Li  Yu</span>, <span class="hlFld-ContribAuthor ">Manuel  Hilbert</span>, <span class="hlFld-ContribAuthor ">Tamara  Maes</span>. </span><span class="cited-content_cbyCitation_article-title">Chemoprobe-based assays of histone lysine demethylase 1A target occupation enable in vivo pharmacokinetics and pharmacodynamics studies of KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2019,</strong> <em>294 </em>
                                    (20)
                                     , 8311-8322. <a href="https://doi.org/10.1074/jbc.RA118.006980" title="DOI URL">https://doi.org/10.1074/jbc.RA118.006980</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA118.006980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA118.006980%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DChemoprobe-based%252Bassays%252Bof%252Bhistone%252Blysine%252Bdemethylase%252B1A%252Btarget%252Boccupation%252Benable%252Bin%252Bvivo%252Bpharmacokinetics%252Band%252Bpharmacodynamics%252Bstudies%252Bof%252BKDM1A%252Binhibitors%26aulast%3DMascar%25C3%25B3%26aufirst%3DCristina%26date%3D2019%26volume%3D294%26issue%3D20%26spage%3D8311%26epage%3D8322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Zhong-Rui  Li</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 388-401. <a href="https://doi.org/10.1016/j.ejmech.2019.02.039" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.039%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bstructure-activity%252Brelationship%252Bstudies%252Band%252Bbiological%252Bcharacterization%252Bof%252Bnew%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine-based%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DWang%26aufirst%3DShuai%26date%3D2019%26volume%3D167%26spage%3D388%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Rui  Li</span>, <span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Linlin  Yang</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Experience-based discovery (EBD) of aryl hydrazines as new scaffolds for the development of LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>166 </em>, 432-444. <a href="https://doi.org/10.1016/j.ejmech.2019.01.075" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.075%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DExperience-based%252Bdiscovery%252B%252528EBD%252529%252Bof%252Baryl%252Bhydrazines%252Bas%252Bnew%252Bscaffolds%252Bfor%252Bthe%252Bdevelopment%252Bof%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DLi%26aufirst%3DZhong-Rui%26date%3D2019%26volume%3D166%26spage%3D432%26epage%3D444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Helai P.  Mohammad</span>, <span class="hlFld-ContribAuthor ">Olena  Barbash</span>, <span class="hlFld-ContribAuthor ">Caretha L.  Creasy</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic modifications in cancer therapy: erasing the roadmap to cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Medicine</span><span> <strong>2019,</strong> <em>25 </em>
                                    (3)
                                     , 403-418. <a href="https://doi.org/10.1038/s41591-019-0376-8" title="DOI URL">https://doi.org/10.1038/s41591-019-0376-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41591-019-0376-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41591-019-0376-8%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Medicine%26atitle%3DTargeting%252Bepigenetic%252Bmodifications%252Bin%252Bcancer%252Btherapy%25253A%252Berasing%252Bthe%252Broadmap%252Bto%252Bcancer%26aulast%3DMohammad%26aufirst%3DHelai%2BP.%26date%3D2019%26date%3D2019%26volume%3D25%26issue%3D3%26spage%3D403%26epage%3D418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">LSD
              (Lysine‐Specific Demethylase): A Decade‐Long Trip from Discovery to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-261. <a href="https://doi.org/10.1002/9783527809257.ch10" title="DOI URL">https://doi.org/10.1002/9783527809257.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DLSD%252B%252528Lysine%2525E2%252580%252590Specific%252BDemethylase%252529%25253A%252BA%252BDecade%2525E2%252580%252590Long%252BTrip%252Bfrom%252BDiscovery%252Bto%252BClinical%252BTrials%26aulast%3DLee%26aufirst%3DAdam%26date%3D2019%26date%3D2019%26spage%3D221%26epage%3D261%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Tino  Heimburg</span>, <span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Computer‐based Lead Identification for Epigenetic Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 45-77. <a href="https://doi.org/10.1002/9783527809257.ch3" title="DOI URL">https://doi.org/10.1002/9783527809257.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DComputer%2525E2%252580%252590based%252BLead%252BIdentification%252Bfor%252BEpigenetic%252BTargets%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2019%26date%3D2019%26spage%3D45%26epage%3D77%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine-Specific Histone Demethylases 1/2 (LSD1/2) and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 197-219. <a href="https://doi.org/10.1007/7355_2019_74" title="DOI URL">https://doi.org/10.1007/7355_2019_74</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_74%26sid%3Dliteratum%253Aachs%26atitle%3DLysine-Specific%252BHistone%252BDemethylases%252B1%25252F2%252B%252528LSD1%25252F2%252529%252Band%252BTheir%252BInhibitors%26aulast%3DSuzuki%26aufirst%3DTakayoshi%26date%3D2019%26date%3D2020%26spage%3D197%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacoepigenetics of LSD1 Inhibitors in Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 523-530. <a href="https://doi.org/10.1016/B978-0-12-813939-4.00014-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-813939-4.00014-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-813939-4.00014-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-813939-4.00014-0%26sid%3Dliteratum%253Aachs%26atitle%3DPharmacoepigenetics%252Bof%252BLSD1%252BInhibitors%252Bin%252BCancer%26aulast%3DYu%26aufirst%3DBin%26date%3D2019%26spage%3D523%26epage%3D530%26pub%3DElsevier%26atitle%3DPharmacoepigenetics%26date%3D2019%26volume%3D118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon L.  Currie</span>, <span class="hlFld-ContribAuthor ">Steven L.  Warner</span>, <span class="hlFld-ContribAuthor ">Hariprasad  Vankayalapati</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Liu</span>, <span class="hlFld-ContribAuthor ">Sunil  Sharma</span>, <span class="hlFld-ContribAuthor ">David J.  Bearss</span>, <span class="hlFld-ContribAuthor ">Barbara J.  Graves</span>. </span><span class="cited-content_cbyCitation_article-title">Development of High-Throughput Screening Assays for Inhibitors of ETS Transcription Factors. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2019,</strong> <em>24 </em>
                                    (1)
                                     , 77-85. <a href="https://doi.org/10.1177/2472555218798571" title="DOI URL">https://doi.org/10.1177/2472555218798571</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555218798571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555218798571%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DDevelopment%252Bof%252BHigh-Throughput%252BScreening%252BAssays%252Bfor%252BInhibitors%252Bof%252BETS%252BTranscription%252BFactors%26aulast%3DCurrie%26aufirst%3DSimon%2BL.%26date%3D2019%26date%3D2018%26volume%3D24%26issue%3D1%26spage%3D77%26epage%3D85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xudong  Sun</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the binding mode and unbinding mechanism of LSD1 inhibitors by combined computational methods. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (47)
                                     , 29833-29846. <a href="https://doi.org/10.1039/C8CP03090A" title="DOI URL">https://doi.org/10.1039/C8CP03090A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP03090A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP03090A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DProbing%252Bthe%252Bbinding%252Bmode%252Band%252Bunbinding%252Bmechanism%252Bof%252BLSD1%252Binhibitors%252Bby%252Bcombined%252Bcomputational%252Bmethods%26aulast%3DSun%26aufirst%3DXudong%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D47%26spage%3D29833%26epage%3D29846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yosuke  Ota</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Design Concepts for LSD1‐Selective Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1782-1791. <a href="https://doi.org/10.1002/tcr.201810031" title="DOI URL">https://doi.org/10.1002/tcr.201810031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201810031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201810031%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DDrug%252BDesign%252BConcepts%252Bfor%252BLSD1%2525E2%252580%252590Selective%252BInhibitors%26aulast%3DOta%26aufirst%3DYosuke%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1782%26epage%3D1791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingchao  Duan</span>, <span class="hlFld-ContribAuthor ">Wenping  Qin</span>, <span class="hlFld-ContribAuthor ">Fengzhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhai</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Guan</span>, <span class="hlFld-ContribAuthor ">Xiaojuan  Wang</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (23-24)
                                     , 6000-6014. <a href="https://doi.org/10.1016/j.bmc.2018.10.037" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.10.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.10.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.10.037%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bin%252Bvitro%252Bevaluation%252Bof%252Bstilbene%252Bderivatives%252Bas%252Bnovel%252BLSD1%252Binhibitors%252Bfor%252BAML%252Btherapy%26aulast%3DDuan%26aufirst%3DYingchao%26date%3D2018%26volume%3D26%26issue%3D23-24%26spage%3D6000%26epage%3D6014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Pui-Man  Lei</span>, <span class="hlFld-ContribAuthor ">Suk-Yu  Wong</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological Inhibition of LSD1 for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (12)
                                     , 3194. <a href="https://doi.org/10.3390/molecules23123194" title="DOI URL">https://doi.org/10.3390/molecules23123194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23123194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23123194%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPharmacological%252BInhibition%252Bof%252BLSD1%252Bfor%252BCancer%252BTreatment%26aulast%3DYang%26aufirst%3DGuan-Jun%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D12%26spage%3D3194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iris F.  Macheleidt</span>, <span class="hlFld-ContribAuthor ">Priya S.  Dalvi</span>, <span class="hlFld-ContribAuthor ">So‐Young  Lim</span>, <span class="hlFld-ContribAuthor ">Sonja  Meemboor</span>, <span class="hlFld-ContribAuthor ">Lydia  Meder</span>, <span class="hlFld-ContribAuthor ">Olivia  Käsgen</span>, <span class="hlFld-ContribAuthor ">Marion  Müller</span>, <span class="hlFld-ContribAuthor ">Karolin  Kleemann</span>, <span class="hlFld-ContribAuthor ">Lingyu  Wang</span>, <span class="hlFld-ContribAuthor ">Peter  Nürnberg</span>, <span class="hlFld-ContribAuthor ">Vanessa  Rüsseler</span>, <span class="hlFld-ContribAuthor ">Stephan C.  Schäfer</span>, <span class="hlFld-ContribAuthor ">Esther  Mahabir</span>, <span class="hlFld-ContribAuthor ">Reinhard  Büttner</span>, <span class="hlFld-ContribAuthor ">Margarete  Odenthal</span>. </span><span class="cited-content_cbyCitation_article-title">Preclinical studies reveal that
              LSD
              1 inhibition results in tumor growth arrest in lung adenocarcinoma independently of driver mutations. </span><span class="cited-content_cbyCitation_journal-name">Molecular Oncology</span><span> <strong>2018,</strong> <em>12 </em>
                                    (11)
                                     , 1965-1979. <a href="https://doi.org/10.1002/1878-0261.12382" title="DOI URL">https://doi.org/10.1002/1878-0261.12382</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/1878-0261.12382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F1878-0261.12382%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Oncology%26atitle%3DPreclinical%252Bstudies%252Breveal%252Bthat%252BLSD%252B1%252Binhibition%252Bresults%252Bin%252Btumor%252Bgrowth%252Barrest%252Bin%252Blung%252Badenocarcinoma%252Bindependently%252Bof%252Bdriver%252Bmutations%26aulast%3DMacheleidt%26aufirst%3DIris%2BF.%26date%3D2018%26date%3D2018%26volume%3D12%26issue%3D11%26spage%3D1965%26epage%3D1979" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jürgen  Sonnemann</span>, <span class="hlFld-ContribAuthor ">Miriam  Zimmermann</span>, <span class="hlFld-ContribAuthor ">Christian  Marx</span>, <span class="hlFld-ContribAuthor ">Franziska  Ebert</span>, <span class="hlFld-ContribAuthor ">Sabine  Becker</span>, <span class="hlFld-ContribAuthor ">Marie-Luise  Lauterjung</span>, <span class="hlFld-ContribAuthor ">James F.  Beck</span>. </span><span class="cited-content_cbyCitation_article-title">LSD1 (KDM1A)-independent effects of the LSD1 inhibitor SP2509 in cancer cells. </span><span class="cited-content_cbyCitation_journal-name">British Journal of Haematology</span><span> <strong>2018,</strong> <em>183 </em>
                                    (3)
                                     , 494-497. <a href="https://doi.org/10.1111/bjh.14983" title="DOI URL">https://doi.org/10.1111/bjh.14983</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/bjh.14983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbjh.14983%26sid%3Dliteratum%253Aachs%26jtitle%3DBritish%2520Journal%2520of%2520Haematology%26atitle%3DLSD1%252B%252528KDM1A%252529-independent%252Beffects%252Bof%252Bthe%252BLSD1%252Binhibitor%252BSP2509%252Bin%252Bcancer%252Bcells%26aulast%3DSonnemann%26aufirst%3DJ%25C3%25BCrgen%26date%3D2018%26date%3D2017%26volume%3D183%26issue%3D3%26spage%3D494%26epage%3D497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Federica  Sarno</span>, <span class="hlFld-ContribAuthor ">Chiara  Papulino</span>, <span class="hlFld-ContribAuthor ">Gianluigi  Franci</span>, <span class="hlFld-ContribAuthor ">Jeanette H.  Andersen</span>, <span class="hlFld-ContribAuthor ">Bastien  Cautain</span>, <span class="hlFld-ContribAuthor ">Colombina  Melardo</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>, <span class="hlFld-ContribAuthor ">Angela  Nebbioso</span>. </span><span class="cited-content_cbyCitation_article-title">3-Chloro-N′-(2-hydroxybenzylidene) benzohydrazide: An LSD1-Selective Inhibitor and Iron-Chelating Agent for Anticancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.01006" title="DOI URL">https://doi.org/10.3389/fphar.2018.01006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.01006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.01006%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3D3-Chloro-N%2525E2%252580%2525B2-%2525282-hydroxybenzylidene%252529%252Bbenzohydrazide%25253A%252BAn%252BLSD1-Selective%252BInhibitor%252Band%252BIron-Chelating%252BAgent%252Bfor%252BAnticancer%252BTherapy%26aulast%3DSarno%26aufirst%3DFederica%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathleen I.  Pishas</span>, <span class="hlFld-ContribAuthor ">Christina D.  Drenberg</span>, <span class="hlFld-ContribAuthor ">Cenny  Taslim</span>, <span class="hlFld-ContribAuthor ">Emily R.  Theisen</span>, <span class="hlFld-ContribAuthor ">Kirsten M.  Johnson</span>, <span class="hlFld-ContribAuthor ">Ranajeet S.  Saund</span>, <span class="hlFld-ContribAuthor ">Ioana L.  Pop</span>, <span class="hlFld-ContribAuthor ">Brian D.  Crompton</span>, <span class="hlFld-ContribAuthor ">Elizabeth R.  Lawlor</span>, <span class="hlFld-ContribAuthor ">Franck  Tirode</span>, <span class="hlFld-ContribAuthor ">Jaume  Mora</span>, <span class="hlFld-ContribAuthor ">Olivier  Delattre</span>, <span class="hlFld-ContribAuthor ">Mary C.  Beckerle</span>, <span class="hlFld-ContribAuthor ">David F.  Callen</span>, <span class="hlFld-ContribAuthor ">Sunil  Sharma</span>, <span class="hlFld-ContribAuthor ">Stephen L.  Lessnick</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic Targeting of KDM1A/LSD1 in Ewing Sarcoma with SP-2509 Engages the Endoplasmic Reticulum Stress Response. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2018,</strong> <em>17 </em>
                                    (9)
                                     , 1902-1916. <a href="https://doi.org/10.1158/1535-7163.MCT-18-0373" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-18-0373</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-18-0373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-18-0373%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DTherapeutic%252BTargeting%252Bof%252BKDM1A%25252FLSD1%252Bin%252BEwing%252BSarcoma%252Bwith%252BSP-2509%252BEngages%252Bthe%252BEndoplasmic%252BReticulum%252BStress%252BResponse%26aulast%3DPishas%26aufirst%3DKathleen%2BI.%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D9%26spage%3D1902%26epage%3D1916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Lulu  Zhang</span>, <span class="hlFld-ContribAuthor ">Xueyuan  Bi</span>, <span class="hlFld-ContribAuthor ">Yanshen  Ren</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Yueqing  Gu</span>, <span class="hlFld-ContribAuthor ">Yungen  Xu</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2018,</strong> <em>78 </em>, 7-16. <a href="https://doi.org/10.1016/j.bioorg.2018.02.016" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.02.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.02.016%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bactivity%252Bof%252B4-%2525284-benzyloxy%252529phenoxypiperidines%252Bas%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%26aulast%3DXi%26aufirst%3DJiayue%26date%3D2018%26volume%3D78%26spage%3D7%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yong  Li</span>, <span class="hlFld-ContribAuthor ">Wei  Yan</span>, <span class="hlFld-ContribAuthor ">Jianhong  Yang</span>, <span class="hlFld-ContribAuthor ">Zhuang  Yang</span>, <span class="hlFld-ContribAuthor ">Mengshi  Hu</span>, <span class="hlFld-ContribAuthor ">Peng  Bai</span>, <span class="hlFld-ContribAuthor ">Minghai  Tang</span>, <span class="hlFld-ContribAuthor ">Lijuan  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel β-carboline/acylhydrazone hybrids as potent antitumor agents and overcome drug resistance. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>152 </em>, 516-526. <a href="https://doi.org/10.1016/j.ejmech.2018.05.003" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.05.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.05.003%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252B%2525CE%2525B2-carboline%25252Facylhydrazone%252Bhybrids%252Bas%252Bpotent%252Bantitumor%252Bagents%252Band%252Bovercome%252Bdrug%252Bresistance%26aulast%3DLi%26aufirst%3DYong%26date%3D2018%26volume%3D152%26spage%3D516%26epage%3D526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Huimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Drug Design in Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00057" title="DOI URL">https://doi.org/10.3389/fchem.2018.00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00057%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DComputer-Aided%252BDrug%252BDesign%252Bin%252BEpigenetics%26aulast%3DLu%26aufirst%3DWenchao%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling-Yu  Li</span>, <span class="hlFld-ContribAuthor ">Jia-Di  Peng</span>, <span class="hlFld-ContribAuthor ">Wenjuan  Zhou</span>, <span class="hlFld-ContribAuthor ">Hui  Qiao</span>, <span class="hlFld-ContribAuthor ">Xin  Deng</span>, <span class="hlFld-ContribAuthor ">Zhou-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Ji-Deng  Li</span>, <span class="hlFld-ContribAuthor ">Yun-Dong  Fu</span>, <span class="hlFld-ContribAuthor ">Song  Li</span>, <span class="hlFld-ContribAuthor ">Kai  Sun</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Potent hydrazone derivatives targeting esophageal cancer cells. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>148 </em>, 359-371. <a href="https://doi.org/10.1016/j.ejmech.2018.02.033" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.02.033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.02.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.02.033%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPotent%252Bhydrazone%252Bderivatives%252Btargeting%252Besophageal%252Bcancer%252Bcells%26aulast%3DLi%26aufirst%3DLing-Yu%26date%3D2018%26volume%3D148%26spage%3D359%26epage%3D371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chia-Lung  Tsai</span>, <span class="hlFld-ContribAuthor ">An-Shine  Chao</span>, <span class="hlFld-ContribAuthor ">Shih-Ming  Jung</span>, <span class="hlFld-ContribAuthor ">Chiao-Yun  Lin</span>, <span class="hlFld-ContribAuthor ">Angel  Chao</span>, <span class="hlFld-ContribAuthor ">Tzu-Hao  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Stress-induced phosphoprotein 1 acts as a scaffold protein for glycogen synthase kinase-3 beta-mediated phosphorylation of lysine-specific demethylase 1. </span><span class="cited-content_cbyCitation_journal-name">Oncogenesis</span><span> <strong>2018,</strong> <em>7 </em>
                                    (3)
                                     <a href="https://doi.org/10.1038/s41389-018-0040-z" title="DOI URL">https://doi.org/10.1038/s41389-018-0040-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41389-018-0040-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41389-018-0040-z%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogenesis%26atitle%3DStress-induced%252Bphosphoprotein%252B1%252Bacts%252Bas%252Ba%252Bscaffold%252Bprotein%252Bfor%252Bglycogen%252Bsynthase%252Bkinase-3%252Bbeta-mediated%252Bphosphorylation%252Bof%252Blysine-specific%252Bdemethylase%252B1%26aulast%3DTsai%26aufirst%3DChia-Lung%26date%3D2018%26date%3D2018%26volume%3D7%26issue%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sumati  Gupta</span>, <span class="hlFld-ContribAuthor ">Kelly  Doyle</span>, <span class="hlFld-ContribAuthor ">Timothy L.  Mosbruger</span>, <span class="hlFld-ContribAuthor ">Andrew  Butterfield</span>, <span class="hlFld-ContribAuthor ">Alexis  Weston</span>, <span class="hlFld-ContribAuthor ">Allison  Ast</span>, <span class="hlFld-ContribAuthor ">Mohan  Kaadige</span>, <span class="hlFld-ContribAuthor ">Anupam  Verma</span>, <span class="hlFld-ContribAuthor ">Sunil  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible LSD1 inhibition with HCI-2509 induces the p53 gene expression signature and disrupts the MYCN signature in high-risk neuroblastoma cells. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2018,</strong> <em>9 </em>
                                    (11)
                                     , 9907-9924. <a href="https://doi.org/10.18632/oncotarget.24035" title="DOI URL">https://doi.org/10.18632/oncotarget.24035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.24035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.24035%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DReversible%252BLSD1%252Binhibition%252Bwith%252BHCI-2509%252Binduces%252Bthe%252Bp53%252Bgene%252Bexpression%252Bsignature%252Band%252Bdisrupts%252Bthe%252BMYCN%252Bsignature%252Bin%252Bhigh-risk%252Bneuroblastoma%252Bcells%26aulast%3DGupta%26aufirst%3DSumati%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D11%26spage%3D9907%26epage%3D9924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shijun  Wen</span>, <span class="hlFld-ContribAuthor ">Jiankang  Wang</span>, <span class="hlFld-ContribAuthor ">Panpan  Liu</span>, <span class="hlFld-ContribAuthor ">Yiqing  Li</span>, <span class="hlFld-ContribAuthor ">Wenhua  Lu</span>, <span class="hlFld-ContribAuthor ">Yumin  Hu</span>, <span class="hlFld-ContribAuthor ">Jinyun  Liu</span>, <span class="hlFld-ContribAuthor ">Zhiyuan  He</span>, <span class="hlFld-ContribAuthor ">Peng  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Novel combination of histone methylation modulators with therapeutic synergy against acute myeloid leukemia in vitro and in vivo. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2018,</strong> <em>413 </em>, 35-45. <a href="https://doi.org/10.1016/j.canlet.2017.10.015" title="DOI URL">https://doi.org/10.1016/j.canlet.2017.10.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2017.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2017.10.015%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DNovel%252Bcombination%252Bof%252Bhistone%252Bmethylation%252Bmodulators%252Bwith%252Btherapeutic%252Bsynergy%252Bagainst%252Bacute%252Bmyeloid%252Bleukemia%252Bin%2525C2%2525A0vitro%252Band%252Bin%2525C2%2525A0vivo%26aulast%3DWen%26aufirst%3DShijun%26date%3D2018%26volume%3D413%26spage%3D35%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tilak  Khanal</span>, <span class="hlFld-ContribAuthor ">Kwangmin  Choi</span>, <span class="hlFld-ContribAuthor ">Yuet-Kin  Leung</span>, <span class="hlFld-ContribAuthor ">Jiang  Wang</span>, <span class="hlFld-ContribAuthor ">Dasom  Kim</span>, <span class="hlFld-ContribAuthor ">Vinothini  Janakiram</span>, <span class="hlFld-ContribAuthor ">Sung-Gook  Cho</span>, <span class="hlFld-ContribAuthor ">Alvaro  Puga</span>, <span class="hlFld-ContribAuthor ">Shuk-Mei  Ho</span>, <span class="hlFld-ContribAuthor ">Kyounghyun  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Loss of NR2E3 represses AHR by LSD1 reprogramming, is associated with poor prognosis in liver cancer. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-11106-2" title="DOI URL">https://doi.org/10.1038/s41598-017-11106-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-11106-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-11106-2%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DLoss%252Bof%252BNR2E3%252Brepresses%252BAHR%252Bby%252BLSD1%252Breprogramming%25252C%252Bis%252Bassociated%252Bwith%252Bpoor%252Bprognosis%252Bin%252Bliver%252Bcancer%26aulast%3DKhanal%26aufirst%3DTilak%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Sun</span>, <span class="hlFld-ContribAuthor ">Jia-Di  Peng</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Ting  Zhang</span>, <span class="hlFld-ContribAuthor ">Yun-Dong  Fu</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of tranylcypromine analogs with an acylhydrazone substituent as LSD1 inactivators: Design, synthesis and their biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (22)
                                     , 5036-5039. <a href="https://doi.org/10.1016/j.bmcl.2017.10.003" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.10.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.10.003%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Btranylcypromine%252Banalogs%252Bwith%252Ban%252Bacylhydrazone%252Bsubstituent%252Bas%252BLSD1%252Binactivators%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Btheir%252Bbiological%252Bevaluation%26aulast%3DSun%26aufirst%3DKai%26date%3D2017%26volume%3D27%26issue%3D22%26spage%3D5036%26epage%3D5039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Yong-Cheng  Ma</span>, <span class="hlFld-ContribAuthor ">Wen-Ping  Qin</span>, <span class="hlFld-ContribAuthor ">Li-Na  Ding</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Ying-Li  Zhu</span>, <span class="hlFld-ContribAuthor ">Xiao-Yu  Zhai</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of tranylcypromine derivatives as novel LSD1/HDACs dual inhibitors for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>140 </em>, 392-402. <a href="https://doi.org/10.1016/j.ejmech.2017.09.038" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.09.038</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.09.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.09.038%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Btranylcypromine%252Bderivatives%252Bas%252Bnovel%252BLSD1%25252FHDACs%252Bdual%252Binhibitors%252Bfor%252Bcancer%252Btreatment%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2017%26volume%3D140%26spage%3D392%26epage%3D402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angel  Chao</span>, <span class="hlFld-ContribAuthor ">Chiao-Yun  Lin</span>, <span class="hlFld-ContribAuthor ">An-Ning  Chao</span>, <span class="hlFld-ContribAuthor ">Chia-Lung  Tsai</span>, <span class="hlFld-ContribAuthor ">Ming-Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Li-Yu  Lee</span>, <span class="hlFld-ContribAuthor ">Ting-Chang  Chang</span>, <span class="hlFld-ContribAuthor ">Tzu-Hao  Wang</span>, <span class="hlFld-ContribAuthor ">Chyong-Huey  Lai</span>, <span class="hlFld-ContribAuthor ">Hsin-Shih  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine-specific demethylase 1 (LSD1) destabilizes p62 and inhibits autophagy in gynecologic malignancies. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (43)
                                     , 74434-74450. <a href="https://doi.org/10.18632/oncotarget.20158" title="DOI URL">https://doi.org/10.18632/oncotarget.20158</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.20158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.20158%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DLysine-specific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Bdestabilizes%252Bp62%252Band%252Binhibits%252Bautophagy%252Bin%252Bgynecologic%252Bmalignancies%26aulast%3DChao%26aufirst%3DAngel%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D43%26spage%3D74434%26epage%3D74450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael T.  McCabe</span>, <span class="hlFld-ContribAuthor ">Helai P.  Mohammad</span>, <span class="hlFld-ContribAuthor ">Olena  Barbash</span>, <span class="hlFld-ContribAuthor ">Ryan G.  Kruger</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Histone Methylation in Cancer. </span><span class="cited-content_cbyCitation_journal-name">The Cancer Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (5)
                                     , 292-301. <a href="https://doi.org/10.1097/PPO.0000000000000283" title="DOI URL">https://doi.org/10.1097/PPO.0000000000000283</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1097/PPO.0000000000000283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1097%2FPPO.0000000000000283%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Cancer%2520Journal%26atitle%3DTargeting%252BHistone%252BMethylation%252Bin%252BCancer%26aulast%3DMcCabe%26aufirst%3DMichael%2BT.%26date%3D2017%26volume%3D23%26issue%3D5%26spage%3D292%26epage%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Li</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR (CoMFA, CoMSIA), molecular docking and molecular dynamics simulations study of 6-aryl-5-cyano-pyrimidine derivatives to explore the structure requirements of LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (15)
                                     , 3521-3528. <a href="https://doi.org/10.1016/j.bmcl.2017.05.065" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.05.065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.05.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.05.065%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3D3D-QSAR%252B%252528CoMFA%25252C%252BCoMSIA%252529%25252C%252Bmolecular%252Bdocking%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bstudy%252Bof%252B6-aryl-5-cyano-pyrimidine%252Bderivatives%252Bto%252Bexplore%252Bthe%252Bstructure%252Brequirements%252Bof%252BLSD1%252Binhibitors%26aulast%3DDing%26aufirst%3DLina%26date%3D2017%26volume%3D27%26issue%3D15%26spage%3D3521%26epage%3D3528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoli  Fu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Advances toward LSD1 inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (11)
                                     , 1227-1242. <a href="https://doi.org/10.4155/fmc-2017-0068" title="DOI URL">https://doi.org/10.4155/fmc-2017-0068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Btoward%252BLSD1%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DFu%26aufirst%3DXiaoli%26date%3D2017%26volume%3D9%26issue%3D11%26spage%3D1227%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Liming  Wu</span>, <span class="hlFld-ContribAuthor ">Tianfang  Ma</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Dawei  Deng</span>, <span class="hlFld-ContribAuthor ">Yueqing  Gu</span>, <span class="hlFld-ContribAuthor ">Jinpei  Zhou</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological activity of 3-oxoamino-benzenesulfonamides as selective and reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2017,</strong> <em>72 </em>, 182-189. <a href="https://doi.org/10.1016/j.bioorg.2017.04.006" title="DOI URL">https://doi.org/10.1016/j.bioorg.2017.04.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2017.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2017.04.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bactivity%252Bof%252B3-oxoamino-benzenesulfonamides%252Bas%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%26aulast%3DXi%26aufirst%3DJiayue%26date%3D2017%26volume%3D72%26spage%3D182%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 340-352. <a href="https://doi.org/10.1080/15592294.2017.1290032" title="DOI URL">https://doi.org/10.1080/15592294.2017.1290032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DStructural%252Binsight%252Binto%252Binhibitors%252Bof%252Bflavin%252Badenine%252Bdinucleotide-dependent%252Blysine%252Bdemethylases%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D340%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria E.  Ourailidou</span>, <span class="hlFld-ContribAuthor ">Alessia  Lenoci</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Frank J.  Dekker</span>. </span><span class="cited-content_cbyCitation_article-title">Towards the development of activity-based probes for detection of lysine-specific demethylase-1 activity. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (3)
                                     , 847-856. <a href="https://doi.org/10.1016/j.bmc.2016.11.043" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.11.043</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.11.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.11.043%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DTowards%252Bthe%252Bdevelopment%252Bof%252Bactivity-based%252Bprobes%252Bfor%252Bdetection%252Bof%252Blysine-specific%252Bdemethylase-1%252Bactivity%26aulast%3DOurailidou%26aufirst%3DMaria%2BE.%26date%3D2017%26volume%3D25%26issue%3D3%26spage%3D847%26epage%3D856" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kenneth W.  Duncan</span>, <span class="hlFld-ContribAuthor ">John E.  Campbell</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Modulators. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 227-227. <a href="https://doi.org/10.1007/7355_2017_30" title="DOI URL">https://doi.org/10.1007/7355_2017_30</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2017_30&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2017_30%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BModulators%26aulast%3DDuncan%26aufirst%3DKenneth%2BW.%26date%3D2017%26spage%3D227%26epage%3D227%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DCancer%252BII%26aulast%3DWaring%26aufirst%3DMichael%2BJ.%26date%3D2018%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kexin  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">DNA and Histone Methylation in Prostate Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 489-529. <a href="https://doi.org/10.1007/978-3-319-59786-7_18" title="DOI URL">https://doi.org/10.1007/978-3-319-59786-7_18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-59786-7_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-59786-7_18%26sid%3Dliteratum%253Aachs%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bin%252BProstate%252BCancer%26aulast%3DXu%26aufirst%3DKexin%26date%3D2017%26date%3D2017%26spage%3D489%26epage%3D529%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bas%252BCancer%252BTargets%26aulast%3DKaneda%26aufirst%3DAtsushi%26date%3D2017%26volume%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.M.  Woster</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Therapeutics Targeting Epigenetics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 297-328. <a href="https://doi.org/10.1016/B978-0-12-409547-2.12325-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-409547-2.12325-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-409547-2.12325-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-409547-2.12325-X%26sid%3Dliteratum%253Aachs%26atitle%3DNovel%252BTherapeutics%252BTargeting%252BEpigenetics%26aulast%3DWoster%26aufirst%3DP.M.%26date%3D2017%26spage%3D297%26epage%3D328%26pub%3DElsevier%26atitle%3DComprehensive%252BMedicinal%252BChemistry%252BIII%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>, <span class="hlFld-ContribAuthor ">Xiao-Yu  Zhai</span>, <span class="hlFld-ContribAuthor ">Li-Na  Ding</span>, <span class="hlFld-ContribAuthor ">Wen-Ping  Qin</span>, <span class="hlFld-ContribAuthor ">Dan-Dan  Shen</span>, <span class="hlFld-ContribAuthor ">Xue-Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of resveratrol derivatives as novel LSD1 inhibitors: Design, synthesis and their biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>126 </em>, 246-258. <a href="https://doi.org/10.1016/j.ejmech.2016.11.035" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.11.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.11.035%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bresveratrol%252Bderivatives%252Bas%252Bnovel%252BLSD1%252Binhibitors%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Btheir%252Bbiological%252Bevaluation%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2017%26volume%3D126%26spage%3D246%26epage%3D258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashwini  Jambhekar</span>, <span class="hlFld-ContribAuthor ">Jamie N.  Anastas</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Lysine Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cold Spring Harbor Perspectives in Medicine</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     , a026484. <a href="https://doi.org/10.1101/cshperspect.a026484" title="DOI URL">https://doi.org/10.1101/cshperspect.a026484</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/cshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspectives%2520in%2520Medicine%26atitle%3DHistone%252BLysine%252BDemethylase%252BInhibitors%26aulast%3DJambhekar%26aufirst%3DAshwini%26date%3D2017%26date%3D2016%26volume%3D7%26issue%3D1%26spage%3Da026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>, <span class="hlFld-ContribAuthor ">Liming  Wu</span>, <span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Fangxia  Zou</span>, <span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Tianfang  Ma</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Dawei  Deng</span>, <span class="hlFld-ContribAuthor ">Yueqing  Gu</span>, <span class="hlFld-ContribAuthor ">Jinpei  Zhou</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological activity of N-(3-substituted-phenyl)benzenesulfonamides as selective and reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2016,</strong> <em>25 </em>
                                    (12)
                                     , 2822-2831. <a href="https://doi.org/10.1007/s00044-016-1706-8" title="DOI URL">https://doi.org/10.1007/s00044-016-1706-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-016-1706-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-016-1706-8%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bactivity%252Bof%252BN-%2525283-substituted-phenyl%252529benzenesulfonamides%252Bas%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%26aulast%3DZha%26aufirst%3DXiaoming%26date%3D2016%26date%3D2016%26volume%3D25%26issue%3D12%26spage%3D2822%26epage%3D2831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S  Gupta</span>, <span class="hlFld-ContribAuthor ">A  Weston</span>, <span class="hlFld-ContribAuthor ">J  Bearrs</span>, <span class="hlFld-ContribAuthor ">T  Thode</span>, <span class="hlFld-ContribAuthor ">A  Neiss</span>, <span class="hlFld-ContribAuthor ">R  Soldi</span>, <span class="hlFld-ContribAuthor ">S  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible lysine-specific demethylase 1 antagonist HCI-2509 inhibits growth and decreases c-MYC in castration- and docetaxel-resistant prostate cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Prostate Cancer and Prostatic Diseases</span><span> <strong>2016,</strong> <em>19 </em>
                                    (4)
                                     , 349-357. <a href="https://doi.org/10.1038/pcan.2016.21" title="DOI URL">https://doi.org/10.1038/pcan.2016.21</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/pcan.2016.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fpcan.2016.21%26sid%3Dliteratum%253Aachs%26jtitle%3DProstate%2520Cancer%2520and%2520Prostatic%2520Diseases%26atitle%3DReversible%252Blysine-specific%252Bdemethylase%252B1%252Bantagonist%252BHCI-2509%252Binhibits%252Bgrowth%252Band%252Bdecreases%252Bc-MYC%252Bin%252Bcastration-%252Band%252Bdocetaxel-resistant%252Bprostate%252Bcancer%252Bcells%26aulast%3DGupta%26aufirst%3DS%26date%3D2016%26date%3D2016%26volume%3D19%26issue%3D4%26spage%3D349%26epage%3D357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zizhen  Feng</span>, <span class="hlFld-ContribAuthor ">Yuan  Yao</span>, <span class="hlFld-ContribAuthor ">Chao  Zhou</span>, <span class="hlFld-ContribAuthor ">Fengju  Chen</span>, <span class="hlFld-ContribAuthor ">Fangrui  Wu</span>, <span class="hlFld-ContribAuthor ">Liping  Wei</span>, <span class="hlFld-ContribAuthor ">Wei  Liu</span>, <span class="hlFld-ContribAuthor ">Shuo  Dong</span>, <span class="hlFld-ContribAuthor ">Michele  Redell</span>, <span class="hlFld-ContribAuthor ">Qianxing  Mo</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological inhibition of LSD1 for the treatment of MLL-rearranged leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2016,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-016-0252-7" title="DOI URL">https://doi.org/10.1186/s13045-016-0252-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-016-0252-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-016-0252-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DPharmacological%252Binhibition%252Bof%252BLSD1%252Bfor%252Bthe%252Btreatment%252Bof%252BMLL-rearranged%252Bleukemia%26aulast%3DFeng%26aufirst%3DZizhen%26date%3D2016%26date%3D2016%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongcheng  Song</span>, <span class="hlFld-ContribAuthor ">Fangrui  Wu</span>, <span class="hlFld-ContribAuthor ">Jingyu  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting histone methylation for cancer therapy: enzymes, inhibitors, biological activity and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2016,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-016-0279-9" title="DOI URL">https://doi.org/10.1186/s13045-016-0279-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-016-0279-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-016-0279-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DTargeting%252Bhistone%252Bmethylation%252Bfor%252Bcancer%252Btherapy%25253A%252Benzymes%25252C%252Binhibitors%25252C%252Bbiological%252Bactivity%252Band%252Bperspectives%26aulast%3DSong%26aufirst%3DYongcheng%26date%3D2016%26date%3D2016%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Dong-Xiao  Yang</span>, <span class="hlFld-ContribAuthor ">Peng-Fei  Geng</span>, <span class="hlFld-ContribAuthor ">Ji  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao-Ming  Wei</span>, <span class="hlFld-ContribAuthor ">Biao  Hu</span>, <span class="hlFld-ContribAuthor ">Qian  Guo</span>, <span class="hlFld-ContribAuthor ">Xin-Hui  Zhang</span>, <span class="hlFld-ContribAuthor ">Wen-Ge  Guo</span>, <span class="hlFld-ContribAuthor ">Bing  Zhao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of [1,2,3]triazolo[4,5- d ]pyrimidine derivatives possessing a hydrazone moiety as antiproliferative agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>124 </em>, 967-980. <a href="https://doi.org/10.1016/j.ejmech.2016.10.022" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.10.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.10.022%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252B%25255B1%25252C2%25252C3%25255Dtriazolo%25255B4%25252C5-%252Bd%252B%25255Dpyrimidine%252Bderivatives%252Bpossessing%252Ba%252Bhydrazone%252Bmoiety%252Bas%252Bantiproliferative%252Bagents%26aulast%3DLi%26aufirst%3DZhong-Hua%26date%3D2016%26volume%3D124%26spage%3D967%26epage%3D980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew  Velinder</span>, <span class="hlFld-ContribAuthor ">Jason  Singer</span>, <span class="hlFld-ContribAuthor ">Diana  Bareyan</span>, <span class="hlFld-ContribAuthor ">Jessica  Meznarich</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Tracy</span>, <span class="hlFld-ContribAuthor ">James M.  Fulcher</span>, <span class="hlFld-ContribAuthor ">David  McClellan</span>, <span class="hlFld-ContribAuthor ">Helena  Lucente</span>, <span class="hlFld-ContribAuthor ">Sarah  Franklin</span>, <span class="hlFld-ContribAuthor ">Sunil  Sharma</span>, <span class="hlFld-ContribAuthor ">Michael E.  Engel</span>. </span><span class="cited-content_cbyCitation_article-title">GFI1 functions in transcriptional control and cell fate determination require SNAG domain methylation to recruit LSD1. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2016,</strong> <em>473 </em>
                                    (19)
                                     , 3355-3369. <a href="https://doi.org/10.1042/BCJ20160558" title="DOI URL">https://doi.org/10.1042/BCJ20160558</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20160558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20160558%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DGFI1%252Bfunctions%252Bin%252Btranscriptional%252Bcontrol%252Band%252Bcell%252Bfate%252Bdetermination%252Brequire%252BSNAG%252Bdomain%252Bmethylation%252Bto%252Brecruit%252BLSD1%26aulast%3DVelinder%26aufirst%3DMatthew%26date%3D2016%26date%3D2016%26volume%3D473%26issue%3D19%26spage%3D3355%26epage%3D3369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yang  Zhou</span>, <span class="hlFld-ContribAuthor ">Yan  Li</span>, <span class="hlFld-ContribAuthor ">Wen-Jing  Wang</span>, <span class="hlFld-ContribAuthor ">Pu  Xiang</span>, <span class="hlFld-ContribAuthor ">Xin-Mei  Luo</span>, <span class="hlFld-ContribAuthor ">Li  Yang</span>, <span class="hlFld-ContribAuthor ">Sheng-Yong  Yang</span>, <span class="hlFld-ContribAuthor ">Ying-Lan  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of novel (E)-N′-(2,3-dihydro-1H-inden-1-ylidene) benzohydrazides as potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>26 </em>
                                    (18)
                                     , 4552-4557. <a href="https://doi.org/10.1016/j.bmcl.2015.06.054" title="DOI URL">https://doi.org/10.1016/j.bmcl.2015.06.054</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2015.06.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2015.06.054%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B%252528E%252529-N%2525E2%252580%2525B2-%2525282%25252C3-dihydro-1H-inden-1-ylidene%252529%252Bbenzohydrazides%252Bas%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DZhou%26aufirst%3DYang%26date%3D2016%26volume%3D26%26issue%3D18%26spage%3D4552%26epage%3D4557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Patcharapong  Thangsunan</span>, <span class="hlFld-ContribAuthor ">Sila  Kittiwachana</span>, <span class="hlFld-ContribAuthor ">Puttinan  Meepowpan</span>, <span class="hlFld-ContribAuthor ">Nawee  Kungwan</span>, <span class="hlFld-ContribAuthor ">Panchika  Prangkio</span>, <span class="hlFld-ContribAuthor ">Supa  Hannongbua</span>, <span class="hlFld-ContribAuthor ">Nuttee  Suree</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid activity prediction of HIV-1 integrase inhibitors: harnessing docking energetic components for empirical scoring by chemometric and artificial neural network approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2016,</strong> <em>30 </em>
                                    (6)
                                     , 471-488. <a href="https://doi.org/10.1007/s10822-016-9917-0" title="DOI URL">https://doi.org/10.1007/s10822-016-9917-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-016-9917-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-016-9917-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DRapid%252Bactivity%252Bprediction%252Bof%252BHIV-1%252Bintegrase%252Binhibitors%25253A%252Bharnessing%252Bdocking%252Benergetic%252Bcomponents%252Bfor%252Bempirical%252Bscoring%252Bby%252Bchemometric%252Band%252Bartificial%252Bneural%252Bnetwork%252Bapproaches%26aulast%3DThangsunan%26aufirst%3DPatcharapong%26date%3D2016%26date%3D2016%26volume%3D30%26issue%3D6%26spage%3D471%26epage%3D488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Andrade</span>, <span class="hlFld-ContribAuthor ">Matthew  Velinder</span>, <span class="hlFld-ContribAuthor ">Jason  Singer</span>, <span class="hlFld-ContribAuthor ">Luke  Maese</span>, <span class="hlFld-ContribAuthor ">Diana  Bareyan</span>, <span class="hlFld-ContribAuthor ">Hong  Nguyen</span>, <span class="hlFld-ContribAuthor ">Mahesh B.  Chandrasekharan</span>, <span class="hlFld-ContribAuthor ">Helena  Lucente</span>, <span class="hlFld-ContribAuthor ">David  McClellan</span>, <span class="hlFld-ContribAuthor ">David  Jones</span>, <span class="hlFld-ContribAuthor ">Sunil  Sharma</span>, <span class="hlFld-ContribAuthor ">Fang  Liu</span>, <span class="hlFld-ContribAuthor ">Michael E.  Engel</span>. </span><span class="cited-content_cbyCitation_article-title">SUMOylation Regulates Growth Factor Independence 1 in Transcriptional Control and Hematopoiesis. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Biology</span><span> <strong>2016,</strong> <em>36 </em>
                                    (10)
                                     , 1438-1450. <a href="https://doi.org/10.1128/MCB.01001-15" title="DOI URL">https://doi.org/10.1128/MCB.01001-15</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/MCB.01001-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FMCB.01001-15%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Biology%26atitle%3DSUMOylation%252BRegulates%252BGrowth%252BFactor%252BIndependence%252B1%252Bin%252BTranscriptional%252BControl%252Band%252BHematopoiesis%26aulast%3DAndrade%26aufirst%3DDaniel%26date%3D2016%26volume%3D36%26issue%3D10%26spage%3D1438%26epage%3D1450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Zhe-Sheng  Chen</span>, <span class="hlFld-ContribAuthor ">Ying  Liu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">TCPs: privileged scaffolds for identifying potent LSD1 inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (5)
                                     , 651-666. <a href="https://doi.org/10.2217/epi-2015-0002" title="DOI URL">https://doi.org/10.2217/epi-2015-0002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi-2015-0002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi-2015-0002%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DTCPs%25253A%252Bprivileged%252Bscaffolds%252Bfor%252Bidentifying%252Bpotent%252BLSD1%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DZheng%26aufirst%3DYi-Chao%26date%3D2016%26volume%3D8%26issue%3D5%26spage%3D651%26epage%3D666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emily R.  Theisen</span>, <span class="hlFld-ContribAuthor ">Kathleen I.  Pishas</span>, <span class="hlFld-ContribAuthor ">Ranajeet S.  Saund</span>, <span class="hlFld-ContribAuthor ">Stephen L.  Lessnick</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic opportunities in Ewing sarcoma: EWS-FLI inhibition
              via
              LSD1 targeting. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2016,</strong> <em>7 </em>
                                    (14)
                                     , 17616-17630. <a href="https://doi.org/10.18632/oncotarget.7124" title="DOI URL">https://doi.org/10.18632/oncotarget.7124</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.7124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.7124%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DTherapeutic%252Bopportunities%252Bin%252BEwing%252Bsarcoma%25253A%252BEWS-FLI%252Binhibition%252Bvia%252BLSD1%252Btargeting%26aulast%3DTheisen%26aufirst%3DEmily%2BR.%26date%3D2016%26date%3D2016%26volume%3D7%26issue%3D14%26spage%3D17616%26epage%3D17630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">LING-LING  KONG</span>, <span class="hlFld-ContribAuthor ">DONG-MEI  MAN</span>, <span class="hlFld-ContribAuthor ">TIAN  WANG</span>, <span class="hlFld-ContribAuthor ">GUO-AN  ZHANG</span>, <span class="hlFld-ContribAuthor ">WEN  CUI</span>. </span><span class="cited-content_cbyCitation_article-title">Downregulation of LSD1 suppresses the proliferation, tumorigenicity and invasion of papillary thyroid carcinoma K1 cells. </span><span class="cited-content_cbyCitation_journal-name">Oncology Letters</span><span> <strong>2016,</strong> <em>11 </em>
                                    (4)
                                     , 2475-2480. <a href="https://doi.org/10.3892/ol.2016.4244" title="DOI URL">https://doi.org/10.3892/ol.2016.4244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ol.2016.4244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fol.2016.4244%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Letters%26atitle%3DDownregulation%252Bof%252BLSD1%252Bsuppresses%252Bthe%252Bproliferation%25252C%252Btumorigenicity%252Band%252Binvasion%252Bof%252Bpapillary%252Bthyroid%252Bcarcinoma%252BK1%252Bcells%26aulast%3DKONG%26aufirst%3DLING-LING%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D4%26spage%3D2475%26epage%3D2480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. V.  Zerov</span>, <span class="hlFld-ContribAuthor ">T. S.  Krupenya</span>, <span class="hlFld-ContribAuthor ">A. A.  Petrov</span>, <span class="hlFld-ContribAuthor ">S. I.  Yakimovich</span>. </span><span class="cited-content_cbyCitation_article-title">Reaction of trifluoromethyl-containing 1,3-dicarbonyl compounds with bis-hydrazides. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2016,</strong> <em>52 </em>
                                    (3)
                                     , 312-318. <a href="https://doi.org/10.1134/S1070428016030039" title="DOI URL">https://doi.org/10.1134/S1070428016030039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428016030039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428016030039%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DReaction%252Bof%252Btrifluoromethyl-containing%252B1%25252C3-dicarbonyl%252Bcompounds%252Bwith%252Bbis-hydrazides%26aulast%3DZerov%26aufirst%3DA.%2BV.%26date%3D2016%26date%3D2016%26volume%3D52%26issue%3D3%26spage%3D312%26epage%3D318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing-Zhang  Wang</span>, <span class="hlFld-ContribAuthor ">Ning  Xiao</span>, <span class="hlFld-ContribAuthor ">Ying-Zhou  Zhang</span>, <span class="hlFld-ContribAuthor ">Chao-Xian  Zhao</span>, <span class="hlFld-ContribAuthor ">Xin-Hua  Guo</span>, <span class="hlFld-ContribAuthor ">Li-Min  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Mfsd2a-based pharmacological strategies for drug delivery across the blood–brain barrier. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2016,</strong> <em>104 </em>, 124-131. <a href="https://doi.org/10.1016/j.phrs.2015.12.024" title="DOI URL">https://doi.org/10.1016/j.phrs.2015.12.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2015.12.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2015.12.024%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DMfsd2a-based%252Bpharmacological%252Bstrategies%252Bfor%252Bdrug%252Bdelivery%252Bacross%252Bthe%252Bblood%2525E2%252580%252593brain%252Bbarrier%26aulast%3DWang%26aufirst%3DJing-Zhang%26date%3D2016%26volume%3D104%26spage%3D124%26epage%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marco P.  Licciardello</span>, <span class="hlFld-ContribAuthor ">Stefan  Kubicek</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Histone Methylation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 209-238. <a href="https://doi.org/10.1016/B978-0-12-802208-5.00009-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-802208-5.00009-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802208-5.00009-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802208-5.00009-6%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252BHistone%252BMethylation%26aulast%3DLicciardello%26aufirst%3DMarco%2BP.%26date%3D2016%26spage%3D209%26epage%3D238%26pub%3DElsevier%26atitle%3DDrug%252BDiscovery%252Bin%252BCancer%252BEpigenetics%26date%3D2016%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P.  Trojer</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Methylation Modifiers in Medical Therapeutics. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 705-729. <a href="https://doi.org/10.1016/B978-0-12-803239-8.00037-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-803239-8.00037-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-803239-8.00037-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-803239-8.00037-5%26sid%3Dliteratum%253Aachs%26atitle%3DHistone%252BMethylation%252BModifiers%252Bin%252BMedical%252BTherapeutics%26aulast%3DTrojer%26aufirst%3DP.%26date%3D2016%26spage%3D705%26epage%3D729%26pub%3DElsevier%26atitle%3DMedical%252BEpigenetics%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A. V.  Zerov</span>, <span class="hlFld-ContribAuthor ">T. S.  Krupenya</span>, <span class="hlFld-ContribAuthor ">A. A.  Petrov</span>, <span class="hlFld-ContribAuthor ">S. I.  Yakimovich</span>. </span><span class="cited-content_cbyCitation_article-title">Reactions of tri- and hexafluoroacetylacetones and 4-ethoxy-1,1,1-trifluorobut-3-en-2-one with 2-aminobenzohydrazide. </span><span class="cited-content_cbyCitation_journal-name">Russian Journal of Organic Chemistry</span><span> <strong>2015,</strong> <em>51 </em>
                                    (12)
                                     , 1758-1761. <a href="https://doi.org/10.1134/S1070428015120167" title="DOI URL">https://doi.org/10.1134/S1070428015120167</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1134/S1070428015120167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1134%2FS1070428015120167%26sid%3Dliteratum%253Aachs%26jtitle%3DRussian%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DReactions%252Bof%252Btri-%252Band%252Bhexafluoroacetylacetones%252Band%252B4-ethoxy-1%25252C1%25252C1-trifluorobut-3-en-2-one%252Bwith%252B2-aminobenzohydrazide%26aulast%3DZerov%26aufirst%3DA.%2BV.%26date%3D2015%26date%3D2016%26volume%3D51%26issue%3D12%26spage%3D1758%26epage%3D1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xueshun  Wang</span>, <span class="hlFld-ContribAuthor ">Boshi  Huang</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Medicinal chemistry insights in the discovery of novel LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (8)
                                     , 1379-1396. <a href="https://doi.org/10.2217/epi.15.86" title="DOI URL">https://doi.org/10.2217/epi.15.86</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi.15.86&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi.15.86%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DMedicinal%252Bchemistry%252Binsights%252Bin%252Bthe%252Bdiscovery%252Bof%252Bnovel%252BLSD1%252Binhibitors%26aulast%3DWang%26aufirst%3DXueshun%26date%3D2015%26volume%3D7%26issue%3D8%26spage%3D1379%26epage%3D1396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuefang  Shang</span>, <span class="hlFld-ContribAuthor ">Wanli  Li</span>, <span class="hlFld-ContribAuthor ">Yaqian  Feng</span>, <span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Xiufang  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Double Properties of Novel Acylhydrazone Nanomaterials Based on a Conjugated System: Anion Binding Ability and Antibacterial Activity. </span><span class="cited-content_cbyCitation_journal-name">Applied Sciences</span><span> <strong>2015,</strong> <em>5 </em>
                                    (4)
                                     , 910-925. <a href="https://doi.org/10.3390/app5040910" title="DOI URL">https://doi.org/10.3390/app5040910</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/app5040910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fapp5040910%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Sciences%26atitle%3DDouble%252BProperties%252Bof%252BNovel%252BAcylhydrazone%252BNanomaterials%252BBased%252Bon%252Ba%252BConjugated%252BSystem%25253A%252BAnion%252BBinding%252BAbility%252Band%252BAntibacterial%252BActivity%26aulast%3DShang%26aufirst%3DXuefang%26date%3D2015%26date%3D2015%26volume%3D5%26issue%3D4%26spage%3D910%26epage%3D925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edward O.  Pyzer-Knapp</span>, <span class="hlFld-ContribAuthor ">Kewei  Li</span>, <span class="hlFld-ContribAuthor ">Alan  Aspuru-Guzik</span>. </span><span class="cited-content_cbyCitation_article-title">Learning from the Harvard Clean Energy Project: The Use of Neural Networks to Accelerate Materials Discovery. </span><span class="cited-content_cbyCitation_journal-name">Advanced Functional Materials</span><span> <strong>2015,</strong> <em>25 </em>
                                    (41)
                                     , 6495-6502. <a href="https://doi.org/10.1002/adfm.201501919" title="DOI URL">https://doi.org/10.1002/adfm.201501919</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adfm.201501919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadfm.201501919%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Functional%2520Materials%26atitle%3DLearning%252Bfrom%252Bthe%252BHarvard%252BClean%252BEnergy%252BProject%25253A%252BThe%252BUse%252Bof%252BNeural%252BNetworks%252Bto%252BAccelerate%252BMaterials%252BDiscovery%26aulast%3DPyzer-Knapp%26aufirst%3DEdward%2BO.%26date%3D2015%26date%3D2015%26volume%3D25%26issue%3D41%26spage%3D6495%26epage%3D6502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Bayo</span>, <span class="hlFld-ContribAuthor ">Maithili P  Dalvi</span>, <span class="hlFld-ContribAuthor ">Elisabeth D  Martinez</span>. </span><span class="cited-content_cbyCitation_article-title">Successful strategies in the discovery of small-molecule epigenetic modulators with anticancer potential. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2015,</strong> <em>7 </em>
                                    (16)
                                     , 2243-2261. <a href="https://doi.org/10.4155/fmc.15.140" title="DOI URL">https://doi.org/10.4155/fmc.15.140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.140%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSuccessful%252Bstrategies%252Bin%252Bthe%252Bdiscovery%252Bof%252Bsmall-molecule%252Bepigenetic%252Bmodulators%252Bwith%252Banticancer%252Bpotential%26aulast%3DBayo%26aufirst%3DJuan%26date%3D2015%26volume%3D7%26issue%3D16%26spage%3D2243%26epage%3D2261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Jinlian  Ma</span>, <span class="hlFld-ContribAuthor ">Zhiru  Wang</span>, <span class="hlFld-ContribAuthor ">Jinfeng  Li</span>, <span class="hlFld-ContribAuthor ">Bailing  Jiang</span>, <span class="hlFld-ContribAuthor ">Wenjuan  Zhou</span>, <span class="hlFld-ContribAuthor ">Xiaojing  Shi</span>, <span class="hlFld-ContribAuthor ">Xixin  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2015,</strong> <em>35 </em>
                                    (5)
                                     , 1032-1071. <a href="https://doi.org/10.1002/med.21350" title="DOI URL">https://doi.org/10.1002/med.21350</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21350%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DA%252BSystematic%252BReview%252Bof%252BHistone%252BLysine-Specific%252BDemethylase%252B1%252Band%252BIts%252BInhibitors%26aulast%3DZheng%26aufirst%3DYi-Chao%26date%3D2015%26date%3D2015%26volume%3D35%26issue%3D5%26spage%3D1032%26epage%3D1071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan M.  Burg</span>, <span class="hlFld-ContribAuthor ">Jennifer E.  Link</span>, <span class="hlFld-ContribAuthor ">Brittany S.  Morgan</span>, <span class="hlFld-ContribAuthor ">Frederick J.  Heller</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>, <span class="hlFld-ContribAuthor ">Dewey G.  McCafferty</span>. </span><span class="cited-content_cbyCitation_article-title">KDM1 class flavin-dependent protein lysine demethylases. </span><span class="cited-content_cbyCitation_journal-name">Biopolymers</span><span> <strong>2015,</strong> <em>104 </em>
                                    (4)
                                     , 213-246. <a href="https://doi.org/10.1002/bip.22643" title="DOI URL">https://doi.org/10.1002/bip.22643</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bip.22643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbip.22643%26sid%3Dliteratum%253Aachs%26jtitle%3DBiopolymers%26atitle%3DKDM1%252Bclass%252Bflavin-dependent%252Bprotein%252Blysine%252Bdemethylases%26aulast%3DBurg%26aufirst%3DJonathan%2BM.%26date%3D2015%26date%3D2015%26volume%3D104%26issue%3D4%26spage%3D213%26epage%3D246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Yanli  Liu</span>, <span class="hlFld-ContribAuthor ">Johnathan L.  Lau</span>, <span class="hlFld-ContribAuthor ">Jinrong  Min</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic targets and drug discovery Part 2: Histone demethylation and DNA methylation. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2015,</strong> <em>151 </em>, 121-140. <a href="https://doi.org/10.1016/j.pharmthera.2015.04.001" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2015.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2015.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2015.04.001%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DEpigenetic%252Btargets%252Band%252Bdrug%252Bdiscovery%252BPart%252B2%25253A%252BHistone%252Bdemethylation%252Band%252BDNA%252Bmethylation%26aulast%3DLiu%26aufirst%3DKe%26date%3D2015%26volume%3D151%26spage%3D121%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John  McGrath</span>, <span class="hlFld-ContribAuthor ">Patrick  Trojer</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting histone lysine methylation in cancer. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2015,</strong> <em>150 </em>, 1-22. <a href="https://doi.org/10.1016/j.pharmthera.2015.01.002" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2015.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2015.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2015.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bhistone%252Blysine%252Bmethylation%252Bin%252Bcancer%26aulast%3DMcGrath%26aufirst%3DJohn%26date%3D2015%26volume%3D150%26spage%3D1%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Fjm400870h&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm400870h%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-23%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=111&amp;pagesCount=2&amp;pageNumber=-1" title="Load all citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load all citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0010.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Representative Structures of Reported Classes of LSD1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>Where appropriate compound IDs and IC<sub>50</sub> values are indicated.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Mode of binding of compound <b>12</b> in complex with LSD1. The H-bonding interactions of compound <b>12</b> with LSD1 are depicted in dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. MAO activity of select compounds. (A) The MAO inhibitor TCP inhibited MAO A with an IC<sub>50</sub> of 2.1 μM, while compounds <b>1</b> and <b>12</b> inhibited MAO A with IC<sub>50</sub>s of 88 and >300 μM, respectively. (B) Compound <b>12</b> did not inhibit MAO B at the concentrations tested (IC<sub>50</sub> > 300 μM). However, compound <b>1</b> exhibited significant activity against MAO B, with an IC<sub>50</sub> of 1.3 μM, similar 3.6 μM for TCP.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>12</b> reversibly inhibits the activity of LSD1. Dilution of compound <b>12</b>, but not of the covalently binding inhibitor TCP, results in recovery of LSD1 activity. Compound <b>13</b> is inactive.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Derivative melt curves of LSD1 in the presence of DMSO, compound <b>12</b>, compound <b>13</b>, and TCP. LSD1 has a complex multiphase melt. Compound <b>12</b> and TCP induce changes in LSD1’s melt profile in distinct manners. Compound <b>13</b> shows no difference from DMSO.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. LSD1 kinetics with multiple concentrations of compound <b>12</b>. Compound <b>12</b> causes a decrease in <i>v</i><sub>max</sub> with no change in <i>K</i><sub>m</sub>, characteristic of noncompetitive inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compounds with biochemical activity against LSD1 show in vitro EC<sub>50</sub>s clustered near 1 μM. Compounds without LSD1 activity show a wide range of in vitro efficacy.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. VCaP cells treated with compound <b>12</b> show a dose-dependent increase in H3K9 dimethylation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/medium/jm-2013-00870h_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0011.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Procedure for the Synthesis of <i>N</i>′-(1-Phenylethylidene)-benzohydrazides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-23/jm400870h/production/images/large/jm-2013-00870h_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400870h&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) THF, K<sub>2</sub>CO<sub>3</sub>, RT, 1 h; (b) conc H<sub>2</sub>SO<sub>4</sub>, CH<sub>3</sub>OH, 65 °C, 12 h; (c) hydrazine, CH<sub>3</sub>OH, reflux 12 h; (d) AcOH, CH<sub>3</sub>OH, MW, 120 °C, 30 min.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31380" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31380" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 43 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Tsai, H.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">Cancer epigenetics: linking basic biology to clinical medicine</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">502</span><span class="NLM_x">–</span> <span class="NLM_lpage">517</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1038%2Fcr.2011.24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=21321605" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVCgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=502-517&author=H.-C.+Tsaiauthor=S.+B.+Baylin&title=Cancer+epigenetics%3A+linking+basic+biology+to+clinical+medicine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics: linking basic biology to clinical medicine</span></div><div class="casAuthors">Tsai, Hsing-Chen; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">502-517</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Cancer evolution at all stages is driven by both epigenetic abnormalities as well as genetic alterations.  Dysregulation of epigenetic control events may lead to abnormal patterns of DNA methylation and chromatin configurations, both of which are crit. contributors to the pathogenesis of cancer.  These epigenetic abnormalities are set and maintained by multiple protein complexes and the interplay between their individual components including DNA methylation machinery, histone modifiers, particularly, polycomb (PcG) proteins, and chromatin remodeling proteins.  Recent advances in genome-wide technol. have revealed that the involvement of these dysregulated epigenetic components appears to be extensive.  Moreover, there is a growing connection between epigenetic abnormalities in cancer and concepts concerning stem-like cell subpopulations as a driving force for cancer.  Emerging data suggest that aspects of the epigenetic landscape inherent to normal embryonic and adult stem/progenitor cells may help foster, under the stress of chronic inflammation or accumulating reactive oxygen species, evolution of malignant subpopulations.  Finally, understanding mol. mechanisms involved in initiation and maintenance of epigenetic abnormalities in all types of cancer has great potential for translational purposes.  This is already evident for epigenetic biomarker development, and for pharmacol. targeting aimed at reversing cancer-specific epigenetic alterations.  Cell Research (2011) 21:502-517. doi:10.1038/cr.2011.24; published online 15 Feb. 2011.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqt533Ud9VRxbVg90H21EOLACvtfcHk0lhSY204Jz_zdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVCgurk%253D&md5=f4834740b42458e1adf7c7c24df2589c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fcr.2011.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2011.24%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DH.-C.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DCancer%2520epigenetics%253A%2520linking%2520basic%2520biology%2520to%2520clinical%2520medicine%26jtitle%3DCell%2520Res.%26date%3D2011%26volume%3D21%26spage%3D502%26epage%3D517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Füllgrabe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joseph, B.</span><span> </span><span class="NLM_article-title">Histone onco-modifications</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">3391</span><span class="NLM_x">–</span> <span class="NLM_lpage">3403</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=3391-3403&author=J.+F%C3%BCllgrabeauthor=E.+Kavanaghauthor=B.+Joseph&title=Histone+onco-modifications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DF%25C3%25BCllgrabe%26aufirst%3DJ.%26aulast%3DKavanagh%26aufirst%3DE.%26aulast%3DJoseph%26aufirst%3DB.%26atitle%3DHistone%2520onco-modifications%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D3391%26epage%3D3403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone Demethylation Mediated by the Nuclear Amine Oxidase Homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+Demethylation+Mediated+by+the+Nuclear+Amine+Oxidase+Homolog+LSD1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lh5qWQ7KEFntQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520Demethylation%2520Mediated%2520by%2520the%2520Nuclear%2520Amine%2520Oxidase%2520Homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H. F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+F.+M.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%2BF.%2BM.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the Histone Code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+Histone+Code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lh5qWQ7KEFntQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520Histone%2520Code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Lachner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">An epigenetic road map for histone lysine methylation</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">2117</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1242%2Fjcs.00493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=12730288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkslCltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2003&pages=2117-2124&author=M.+Lachnerauthor=R.+J.+O%E2%80%99Sullivanauthor=T.+Jenuwein&title=An+epigenetic+road+map+for+histone+lysine+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">An epigenetic road map for histone lysine methylation</span></div><div class="casAuthors">Lachner, Monika; O'Sullivan, Roderick J.; Jenuwein, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2117-2124</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review discusses the complexity and coding potential of histone lysine methylation in epigenetic control.  The dynamic cycle of histone lysine methylation in transcriptional stimulation or repression is illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjyGLMb0IvPLVg90H21EOLACvtfcHk0lh5qWQ7KEFntQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkslCltLg%253D&md5=bde6700de17f4131185bc9439c3b29d7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1242%2Fjcs.00493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.00493%26sid%3Dliteratum%253Aachs%26aulast%3DLachner%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DAn%2520epigenetic%2520road%2520map%2520for%2520histone%2520lysine%2520methylation%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2003%26volume%3D116%26spage%3D2117%26epage%3D2124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanoni, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span> </span><span class="NLM_article-title">Histone demethylation catalysed by LSD1 is a flavin-dependent oxidative process</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">579</span><span class="NLM_x">, </span> <span class="NLM_fpage">2203</span><span class="NLM_x">–</span> <span class="NLM_lpage">2207</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.febslet.2005.03.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=15811342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlWitr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2005&pages=2203-2207&author=F.+Fornerisauthor=C.+Bindaauthor=M.+A.+Vanoniauthor=A.+Matteviauthor=E.+Battaglioli&title=Histone+demethylation+catalysed+by+LSD1+is+a+flavin-dependent+oxidative+process"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation catalyzed by LSD1 is a flavin-dependent oxidative process</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Vanoni, Maria Antonietta; Mattevi, Andrea; Battaglioli, Elena</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2203-2207</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Lysine-specific histone demethylase 1 (LSD1) is a very recently discovered enzyme which specifically removes Me groups from Lys-4 of histone H3.  Here, the authors addressed the functional properties of the protein demonstrating that histone demethylation involves the FAD-catalyzed oxidn. of the methylated Lys residue.  The nature of the substrate that acts as the electron acceptor required to complete the catalytic cycle was investigated.  LSD1 converts O2 to H2O2 although this reactivity is not as pronounced as that of other flavin-dependent oxidases.  The findings raised the possibility that in vivo LSD1 might not necessarily function as an oxidase, but it might use alternative electron acceptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaI1pvSxtE-7Vg90H21EOLACvtfcHk0ljKAC6h64ePwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlWitr4%253D&md5=d993a48b0a9a8d9b4b27d83308ac8c0a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2005.03.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2005.03.015%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DVanoni%26aufirst%3DM.%2BA.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26atitle%3DHistone%2520demethylation%2520catalysed%2520by%2520LSD1%2520is%2520a%2520flavin-dependent%2520oxidative%2520process%26jtitle%3DFEBS%2520Lett.%26date%3D2005%26volume%3D579%26spage%3D2203%26epage%3D2207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">LSD1: oxidative chemistry for multifaceted functions in chromatin regulation</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">189</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=181-189&author=F.+Fornerisauthor=C.+Bindaauthor=E.+Battaglioliauthor=A.+Mattevi&title=LSD1%3A+oxidative+chemistry+for+multifaceted+functions+in+chromatin+regulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DLSD1%253A%2520oxidative%2520chemistry%2520for%2520multifaceted%2520functions%2520in%2520chromatin%2520regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D181%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gocke, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borek, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomchick, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machius, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span> </span><span class="NLM_article-title">Structural Basis for CoREST-Dependent Demethylation of Nucleosomes by the Human LSD1 Histone Demethylase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">387</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.molcel.2006.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=16885027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD28XovVWlu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=377-387&author=M.+Yangauthor=C.+B.+Gockeauthor=X.+Luoauthor=D.+Borekauthor=D.+R.+Tomchickauthor=M.+Machiusauthor=Z.+Otwinowskiauthor=H.+Yu&title=Structural+Basis+for+CoREST-Dependent+Demethylation+of+Nucleosomes+by+the+Human+LSD1+Histone+Demethylase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for CoREST-dependent demethylation of nucleosomes by the human LSD1 histone demethylase</span></div><div class="casAuthors">Yang, Maojun; Gocke, Christian B.; Luo, Xuelian; Borek, Dominika; Tomchick, Diana R.; Machius, Mischa; Otwinowski, Zbyszek; Yu, Hongtao</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">377-387</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone methylation regulates diverse chromatin-templated processes, including transcription.  Many transcriptional corepressor complexes contain lysine-specific demethylase 1 (LSD1) and CoREST that collaborate to demethylate mono- and dimethylated H3-K4 of nucleosomes.  Here, we report the crystal structure of the LSD1-CoREST complex.  LSD1-CoREST forms an elongated structure with a long stalk connecting the catalytic domain of LSD1 and the CoREST SANT2 domain.  LSD1 recognizes a large segment of the H3 tail through a deep, neg. charged pocket at the active site and possibly a shallow groove on its surface.  CoREST SANT2 interacts with DNA.  Disruption of the SANT2-DNA interaction diminishes CoREST-dependent demethylation of nucleosomes by LSD1.  The shape and dimension of LSD1-CoREST suggest its bivalent binding to nucleosomes, allowing efficient H3-K4 demethylation.  This spatially sepd., multivalent nucleosome binding mode may apply to other chromatin-modifying enzymes that generally contain multiple nucleosome binding modules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon5vOAvWkcXrVg90H21EOLACvtfcHk0ljKAC6h64ePwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XovVWlu74%253D&md5=32aa853fd60a8a562900defa70d658c0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DGocke%26aufirst%3DC.%2BB.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DBorek%26aufirst%3DD.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DMachius%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DH.%26atitle%3DStructural%2520Basis%2520for%2520CoREST-Dependent%2520Demethylation%2520of%2520Nucleosomes%2520by%2520the%2520Human%2520LSD1%2520Histone%2520Demethylase%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D377%26epage%3D387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Shi, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwase, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baba, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Regulation of LSD1 Histone Demethylase Activity by Its Associated Factors</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">857</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.molcel.2005.08.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=16140033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=857-864&author=Y.-J.+Shiauthor=C.+Matsonauthor=F.+Lanauthor=S.+Iwaseauthor=T.+Babaauthor=Y.+Shi&title=Regulation+of+LSD1+Histone+Demethylase+Activity+by+Its+Associated+Factors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of LSD1 histone demethylase activity by its associated factors</span></div><div class="casAuthors">Shi, Yu-Jiang; Matson, Caitlin; Lan, Fei; Iwase, Shigeki; Baba, Tadashi; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">857-864</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">LSD1 is a recently identified human lysine (K)-specific histone demethylase.  LSD1 is assocd. with HDAC1/2, with CoREST, a SANT domain-contg. corepressor, and with BHC80, a PHD domain-contg. protein, among others.  We show that CoREST confers on LSD1 the ability to demethylate nucleosomal substrates and that it protects LSD1 from proteasomal degrdn. in vivo.  We find hyperacetylated nucleosomes less susceptible to CoREST/LSD1-mediated demethylation, suggesting that hypoacetylated nucleosomes may be the preferred physiol. substrates.  This raises the possibility that histone deacetylases and LSD1 may collaborate to generate a repressive chromatin environment.  Consistent with this model, TSA treatment results in derepression of LSD1 target genes.  While CoREST pos. regulates LSD1 function, BHC80 inhibits CoREST/LSD1-mediated demethylation in vitro and may therefore confer neg. regulation.  Taken together, these findings suggest that LSD1-mediated histone demethylation is regulated dynamically in vivo.  This is expected to have profound effects on gene expression under both physiol. and pathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraharVnxcaMLVg90H21EOLACvtfcHk0ljKAC6h64ePwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVyhtbzK&md5=34ba238112151c6596dd6f6c7caa043e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.08.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.08.027%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.-J.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DIwase%26aufirst%3DS.%26aulast%3DBaba%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DRegulation%2520of%2520LSD1%2520Histone%2520Demethylase%2520Activity%2520by%2520Its%2520Associated%2520Factors%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D857%26epage%3D864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Cegli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alpatov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horton, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Recognition of unmethylated histone H3 lysine 4 links BHC80 to LSD1-mediated gene repression</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">718</span><span class="NLM_x">–</span> <span class="NLM_lpage">722</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=718-722&author=F.+Lanauthor=R.+E.+Collinsauthor=R.+De+Cegliauthor=R.+Alpatovauthor=J.+R.+Hortonauthor=X.+Shiauthor=O.+Gozaniauthor=X.+Chengauthor=Y.+Shi&title=Recognition+of+unmethylated+histone+H3+lysine+4+links+BHC80+to+LSD1-mediated+gene+repression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLan%26aufirst%3DF.%26aulast%3DCollins%26aufirst%3DR.%2BE.%26aulast%3DDe%2BCegli%26aufirst%3DR.%26aulast%3DAlpatov%26aufirst%3DR.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DRecognition%2520of%2520unmethylated%2520histone%2520H3%2520lysine%25204%2520links%2520BHC80%2520to%2520LSD1-mediated%2520gene%2520repression%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D718%26epage%3D722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wynder, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooch, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span> </span><span class="NLM_article-title">An essential role for CoREST in nucleosomal histone 3 lysine 4 demethylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=432-435&author=M.+G.+Leeauthor=C.+Wynderauthor=N.+Coochauthor=R.+Shiekhattar&title=An+essential+role+for+CoREST+in+nucleosomal+histone+3+lysine+4+demethylation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DWynder%26aufirst%3DC.%26aulast%3DCooch%26aufirst%3DN.%26aulast%3DShiekhattar%26aufirst%3DR.%26atitle%3DAn%2520essential%2520role%2520for%2520CoREST%2520in%2520nucleosomal%2520histone%25203%2520lysine%25204%2520demethylation%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D432%26epage%3D435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dall’Aglio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraaije, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">A Highly Specific Mechanism of Histone H3-K4 Recognition by Histone Demethylase LSD1</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">35289</span><span class="NLM_x">–</span> <span class="NLM_lpage">35295</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=35289-35295&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Dall%E2%80%99Aglioauthor=M.+W.+Fraaijeauthor=E.+Battaglioliauthor=A.+Mattevi&title=A+Highly+Specific+Mechanism+of+Histone+H3-K4+Recognition+by+Histone+Demethylase+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DDall%25E2%2580%2599Aglio%26aufirst%3DA.%26aulast%3DFraaije%26aufirst%3DM.%2BW.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DA%2520Highly%2520Specific%2520Mechanism%2520of%2520Histone%2520H3-K4%2520Recognition%2520by%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D35289%26epage%3D35295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vanoni, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Human Histone Demethylase LSD1 Reads the Histone Code</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">41360</span><span class="NLM_x">–</span> <span class="NLM_lpage">41365</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1074%2Fjbc.M509549200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=16223729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=41360-41365&author=F.+Fornerisauthor=C.+Bindaauthor=M.+A.+Vanoniauthor=E.+Battaglioliauthor=A.+Mattevi&title=Human+Histone+Demethylase+LSD1+Reads+the+Histone+Code"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Human Histone Demethylase LSD1 Reads the Histone Code</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Vanoni, Maria Antonietta; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41360-41365</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human histone demethylase LSD1 is a flavin-dependent amine oxidase that catalyzes the specific removal of Me groups from mono- and dimethylated Lys4 of histone H3.  The N-terminal tail of H3 is subject to various covalent modifications, and a fundamental question in LSD1 biol. is how these epigenetic marks affect the demethylase activity.  The authors show that LSD1 does not have a strong preference for mono- or dimethylated Lys4 of H3.  Substrate recognition is not confined to the residues neighboring Lys4, but it requires a sufficiently long peptide segment consisting of the N-terminal 20 amino acids of H3.  Electrostatic interactions are an important factor in protein-substrate recognition, as indicated by the high sensitivity of Km to ionic strength.  The authors have probed LSD1 for its ability to demethylate Lys4 in presence of a second modification on the same peptide substrate.  Methylation of Lys9 does not affect enzyme catalysis.  Conversely, Lys9 acetylation causes an almost 6-fold increase in the Km value, whereas phosphorylation of Ser10 totally abolishes activity.  LSD1 is inhibited by a demethylated peptide with an inhibition const. of 1.8 μM, suggesting that LSD1 can bind to H3 independently of Lys4 methylation.  LSD1 is a chromatin-modifying enzyme, which is able to read different epigenetic marks on the histone N-terminal tail and can serve as a docking module for the stabilization of the assocd. corepressor complex(es) on chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxu9eUWof47Vg90H21EOLACvtfcHk0lim7nVmaFoWtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShsb3L&md5=f39f7c79f1f789fb06ca7dd8a98752fe</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M509549200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M509549200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DVanoni%26aufirst%3DM.%2BA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DHuman%2520Histone%2520Demethylase%2520LSD1%2520Reads%2520the%2520Histone%2520Code%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D41360%26epage%3D41365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Garcia-Bassets, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telese, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prefontaine, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutt, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ju, B.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohgi, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escoubet-Lozach, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glass, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, X.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, M. G.</span><span> </span><span class="NLM_article-title">Histone Methylation-Dependent Mechanisms Impose Ligand Dependency for Gene Activation by Nuclear Receptors</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">518</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=505-518&author=I.+Garcia-Bassetsauthor=Y.-S.+Kwonauthor=F.+Teleseauthor=G.+G.+Prefontaineauthor=K.+R.+Huttauthor=C.+S.+Chengauthor=B.-G.+Juauthor=K.+A.+Ohgiauthor=J.+Wangauthor=L.+Escoubet-Lozachauthor=D.+W.+Roseauthor=C.+K.+Glassauthor=X.-D.+Fuauthor=M.+G.+Rosenfeld&title=Histone+Methylation-Dependent+Mechanisms+Impose+Ligand+Dependency+for+Gene+Activation+by+Nuclear+Receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGarcia-Bassets%26aufirst%3DI.%26aulast%3DKwon%26aufirst%3DY.-S.%26aulast%3DTelese%26aufirst%3DF.%26aulast%3DPrefontaine%26aufirst%3DG.%2BG.%26aulast%3DHutt%26aufirst%3DK.%2BR.%26aulast%3DCheng%26aufirst%3DC.%2BS.%26aulast%3DJu%26aufirst%3DB.-G.%26aulast%3DOhgi%26aufirst%3DK.%2BA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DEscoubet-Lozach%26aufirst%3DL.%26aulast%3DRose%26aufirst%3DD.%2BW.%26aulast%3DGlass%26aufirst%3DC.%2BK.%26aulast%3DFu%26aufirst%3DX.-D.%26aulast%3DRosenfeld%26aufirst%3DM.%2BG.%26atitle%3DHistone%2520Methylation-Dependent%2520Mechanisms%2520Impose%2520Ligand%2520Dependency%2520for%2520Gene%2520Activation%2520by%2520Nuclear%2520Receptors%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D505%26epage%3D518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Sch%C3%BCleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+%28LSD1%29+is+highly+expressed+in+ER-negative+breast+cancers+and+a+biomarker+predicting+aggressive+biology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0ljtF0arNRZ4Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520is%2520highly%2520expressed%2520in%2520ER-negative%2520breast%2520cancers%2520and%2520a%2520biomarker%2520predicting%2520aggressive%2520biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-Specific Demethylase 1 Is Strongly Expressed in Poorly Differentiated Neuroblastoma: Implications for Therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-Specific+Demethylase+1+Is+Strongly+Expressed+in+Poorly+Differentiated+Neuroblastoma%3A+Implications+for+Therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-Specific%2520Demethylase%25201%2520Is%2520Strongly%2520Expressed%2520in%2520Poorly%2520Differentiated%2520Neuroblastoma%253A%2520Implications%2520for%2520Therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsunoda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neal, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamaue, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">586</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1002%2Fijc.25349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=20333681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2011&pages=574-586&author=S.+Hayamiauthor=J.+D.+Kellyauthor=H.-S.+Choauthor=M.+Yoshimatsuauthor=M.+Unokiauthor=T.+Tsunodaauthor=H.+I.+Fieldauthor=D.+E.+Nealauthor=H.+Yamaueauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Overexpression+of+LSD1+contributes+to+human+carcinogenesis+through+chromatin+regulation+in+various+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of LSD1 contributes to human carcinogenesis through chromatin regulation in various cancers</span></div><div class="casAuthors">Hayami Shinya; Kelly John D; Cho Hyun-Soo; Yoshimatsu Masanori; Unoki Motoko; Tsunoda Tatsuhiko; Field Helen I; Neal David E; Yamaue Hiroki; Ponder Bruce A J; Nakamura Yusuke; Hamamoto Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">International journal of cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-86</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">A number of histone demethylases have been identified and biochemically characterized, but the pathological roles of their dysfunction in human disease like cancer have not been well understood.  Here, we demonstrate important roles of lysine-specific demethylase 1 (LSD1) in human carcinogenesis.  Expression levels of LSD1 are significantly elevated in human bladder carcinomas compared with nonneoplastic bladder tissues (p < 0.0001). cDNA microarray analysis also revealed its transactivation in lung and colorectal carcinomas.  LSD1-specific small interfering RNAs significantly knocked down its expression and resulted in suppression of proliferation of various bladder and lung cancer cell lines.  Concordantly, introduction of exogenous LSD1 expression promoted cell cycle progression of human embryonic kidney fibroblast cells.  Expression profile analysis showed that LSD1 could affect the expression of genes involved in various chromatin-modifying pathways such as chromatin remodeling at centromere, centromeric heterochromatin formation and chromatin assembly, indicating its essential roles in carcinogenesis through chromatin modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQAVHBzjMN_WuIHSmpohpd0fW6udTcc2eaxDSQkIKOk-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbpslentQ%253D%253D&md5=b15e7aa7b05c05dd8cfd781ed1a7e3bd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fijc.25349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25349%26sid%3Dliteratum%253Aachs%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DKelly%26aufirst%3DJ.%2BD.%26aulast%3DCho%26aufirst%3DH.-S.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DTsunoda%26aufirst%3DT.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DNeal%26aufirst%3DD.%2BE.%26aulast%3DYamaue%26aufirst%3DH.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520LSD1%2520contributes%2520to%2520human%2520carcinogenesis%2520through%2520chromatin%2520regulation%2520in%2520various%2520cancers%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D128%26spage%3D574%26epage%3D586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen Receptor Coactivators Lysine-Specific Histone Demethylase 1 and Four and a Half LIM Domain Protein 2 Predict Risk of Prostate Cancer Recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Androgen+Receptor+Coactivators+Lysine-Specific+Histone+Demethylase+1+and+Four+and+a+Half+LIM+Domain+Protein+2+Predict+Risk+of+Prostate+Cancer+Recurrence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520Receptor%2520Coactivators%2520Lysine-Specific%2520Histone%2520Demethylase%25201%2520and%2520Four%2520and%2520a%2520Half%2520LIM%2520Domain%2520Protein%25202%2520Predict%2520Risk%2520of%2520Prostate%2520Cancer%2520Recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Zhao, Z.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.-R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, W.-G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, W.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.-B.</span><span> </span><span class="NLM_article-title">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span> <span class="citation_source-journal">World J. Gastroenterol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">6651</span><span class="NLM_x">–</span> <span class="NLM_lpage">6656</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.3748%2Fwjg.v18.i45.6651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=6651-6656&author=Z.-K.+Zhaoauthor=H.-F.+Yuauthor=D.-R.+Wangauthor=P.+Dongauthor=L.+Chenauthor=W.-G.+Wuauthor=W.-J.+Dingauthor=Y.-B.+Liu&title=Overexpression+of+lysine+specific+demethylase+1+predicts+worse+prognosis+in+primary+hepatocellular+carcinoma+patients"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of lysine specific demethylase 1 predicts worse prognosis in primary hepatocellular carcinoma patients</span></div><div class="casAuthors">Zhao, Ze-Kun; Yu, Hai-Feng; Wang, Dao-Rong; Dong, Ping; Chen, Lei; Wu, Wen-Guang; Ding, Wen-Jun; Liu, Ying-Bin</div><div class="citationInfo"><span class="NLM_cas:title">World Journal of Gastroenterology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">6651-6656</span>CODEN:
                <span class="NLM_cas:coden">WJGAF2</span>;
        ISSN:<span class="NLM_cas:issn">1007-9327</span>.
    
            (<span class="NLM_cas:orgname">WJG Press and Beijing Baishideng BioMed Scientific Co., Ltd.</span>)
        </div><div class="casAbstract">Aim: To investigate the clinicopathol. features and prognostic value of lysine specific demethylase 1 (LSD1) in hepatocellular carcinoma (HCC).  Methods: We examd. LSD1 expression in 60 paired liver cancer tissues and adjacent noncancerous tissues by quant. real time polymerase chain reaction (qRT-PCR) and Western blotting.  In addn., we analyzed LSD1 expression in 198 HCC samples by immunohistochem.  The relationship between LSD1 expression, clinicopathol. features and patient survival was investigated.  Results: Immunohistochem., Western blotting, and qRT-PCR consistently confirmed LSD1 overexpression in HCC tissues compared to adjacent non-neoplastic tissues (P < 0.01).  Addnl., immunostaining showed more LSD1-pos. cells in the higher tumor stage (T3-4) and tumor grade (G3) than in the lower tumor stage (T1-2, P < 0.001) and tumor grade (G1-2, P < 0.001), resp.  Moreover, HCC patients with high LSD1 expression had significantly lower 5-yr overall survival rates (P < 0.001) and lower 5-yr disease-free survival rates (P < 0.001), resp.  A Cox proportional hazards model further demonstrated that LSD1 over-expression was an independent predictor of poor prognosis for both 5-yr disease-free survival [hazards ratio (HR) = 1.426, 95%CI: 0.672-2.146, P < 0.001] and 5-yr overall survival (HR = 2.456, 95%CI: 1.234-3.932, P < 0.001) in HCC.  Conclusion: Our data suggest for the first time that the overexpression of LSD1 protein in HCC tissues indicates tumor progression and predicts poor prognosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqX4-UuO3NnLVg90H21EOLACvtfcHk0ljOPOEAhW6zog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVyisLnJ&md5=c657f6b1e0b5e183c28fefbda4cc8745</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3748%2Fwjg.v18.i45.6651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3748%252Fwjg.v18.i45.6651%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.-K.%26aulast%3DYu%26aufirst%3DH.-F.%26aulast%3DWang%26aufirst%3DD.-R.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DW.-G.%26aulast%3DDing%26aufirst%3DW.-J.%26aulast%3DLiu%26aufirst%3DY.-B.%26atitle%3DOverexpression%2520of%2520lysine%2520specific%2520demethylase%25201%2520predicts%2520worse%2520prognosis%2520in%2520primary%2520hepatocellular%2520carcinoma%2520patients%26jtitle%3DWorld%2520J.%2520Gastroenterol.%26date%3D2012%26volume%3D18%26spage%3D6651%26epage%3D6656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Novel oligoamine analogues inhibit lysine-specific demethylase 1 (LSD1) and induce re-expression of epigenetically silenced genes</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">7217</span><span class="NLM_x">–</span> <span class="NLM_lpage">7228</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1158%2F1078-0432.CCR-09-1293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=19934284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=7217-7228&author=Y.+Huangauthor=T.+M.+Stewartauthor=Y.+Wuauthor=S.+B.+Baylinauthor=L.+J.+Martonauthor=B.+Perkinsauthor=R.+J.+Jonesauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Novel+oligoamine+analogues+inhibit+lysine-specific+demethylase+1+%28LSD1%29+and+induce+re-expression+of+epigenetically+silenced+genes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oligoamine Analogues Inhibit Lysine-Specific Demethylase 1 and Induce Reexpression of Epigenetically Silenced Genes</span></div><div class="casAuthors">Huang, Yi; Stewart, Tracy Murray; Wu, Yu; Baylin, Stephen B.; Marton, Laurence J.; Perkins, Brandy; Jones, Richard J.; Woster, Patrick M.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7217-7228</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Abnormal DNA CpG island hypermethylation and transcriptionally repressive histone modifications are assocd. with the aberrant silencing of tumor suppressor genes.  Lysine methylation is a dynamic, enzymically controlled process.  Lysine-specific demethylase 1 (LSD1) has recently been identified as a histone lysine demethylase.  LSD1 specifically catalyzes demethylation of mono- and dimethyl-lysine 4 of histone 3 (H3K4), key pos. chromatin marks assocd. with transcriptional activation.  We hypothesized that a novel class of oligoamine analogs would effectively inhibit LSD1 and thus cause the reexpression of aberrantly silenced genes.  Exptl. Design: Human colorectal cancer cells were treated with the oligoamines and changes in mono- and dimethyl-H3K4 and other chromatin marks were monitored.  In addn., treated cells were evaluated for the reexpression of the aberrantly silenced secreted frizzled-related proteins (SFRP) Wnt signaling pathway antagonist genes.  Finally, the effects of the LSD1 inhibitors were evaluated in an in vivo xenograft model.  Results: Treatment of HCT116 human colon adenocarcinoma cells in vitro resulted in increased H3K4 methylation and reexpression of silenced SFRP genes.  This reexpression is also accompanied by a decrease in H3K9me2 repressive mark.  Importantly, cotreatment with low doses of oligoamines and a DNA methyltransferase inhibitor highly induces the reexpression of the aberrantly silenced SFRP2 gene and results in significant inhibition of the growth of established tumors in a human colon tumor model in vivo.  Conclusions: The use of LSD1-inhibiting oligoamine analogs in combination with DNA methyltransferase inhibitors represents a highly promising and novel approach for epigenetic therapy of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk1naS7xW5GbVg90H21EOLACvtfcHk0lhO-lXLQA2lVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE&md5=0f86cd2d3482aa0a36adacc4f10329dd</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-1293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-1293%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DStewart%26aufirst%3DT.%2BM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DMarton%26aufirst%3DL.%2BJ.%26aulast%3DPerkins%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DR.%2BJ.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DNovel%2520oligoamine%2520analogues%2520inhibit%2520lysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520and%2520induce%2520re-expression%2520of%2520epigenetically%2520silenced%2520genes%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D7217%26epage%3D7228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quenelle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mascaro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortega, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">A Novel Selective LSD1/KDM1A Inhibitor Epigenetically Blocks Herpes Simplex Virus Lytic Replication and Reactivation from Latency</span> <span class="citation_source-journal">mBio</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">00558-12</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=00558-12&author=Y.+Liangauthor=D.+Quenelleauthor=J.+L.+Vogelauthor=C.+Mascaroauthor=A.+Ortegaauthor=T.+M.+Kristie&title=A+Novel+Selective+LSD1%2FKDM1A+Inhibitor+Epigenetically+Blocks+Herpes+Simplex+Virus+Lytic+Replication+and+Reactivation+from+Latency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DQuenelle%26aufirst%3DD.%26aulast%3DVogel%26aufirst%3DJ.%2BL.%26aulast%3DMascaro%26aufirst%3DC.%26aulast%3DOrtega%26aufirst%3DA.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DA%2520Novel%2520Selective%2520LSD1%252FKDM1A%2520Inhibitor%2520Epigenetically%2520Blocks%2520Herpes%2520Simplex%2520Virus%2520Lytic%2520Replication%2520and%2520Reactivation%2520from%2520Latency%26jtitle%3DmBio%26date%3D2013%26volume%3D4%26spage%3D00558-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally Designed <i>trans</i>-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+Designed+trans-2-Phenylcyclopropylamine+Derivatives+Potently+Inhibit+Histone+Demethylase+LSD1%2FKDM1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lhqrEJ_-ViXeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520Designed%2520trans-2-Phenylcyclopropylamine%2520Derivatives%2520Potently%2520Inhibit%2520Histone%2520Demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+Structural%2C+and+Biological+Evaluation+of+Tranylcypromine+Derivatives+as+Inhibitors+of+Histone+Demethylases+LSD1+and+LSD2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520Structural%252C%2520and%2520Biological%2520Evaluation%2520of%2520Tranylcypromine%2520Derivatives%2520as%2520Inhibitors%2520of%2520Histone%2520Demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Ueda, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Identification of Cell-Active Lysine Specific Demethylase 1-Selective Inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">17536</span><span class="NLM_x">–</span> <span class="NLM_lpage">17537</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja907055q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=17536-17537&author=R.+Uedaauthor=T.+Suzukiauthor=K.+Minoauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=M.+Hasegawaauthor=R.+Sasakiauthor=T.+Mizukamiauthor=N.+Miyata&title=Identification+of+Cell-Active+Lysine+Specific+Demethylase+1-Selective+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja907055q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja907055q%26sid%3Dliteratum%253Aachs%26aulast%3DUeda%26aufirst%3DR.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DM.%26aulast%3DSasaki%26aufirst%3DR.%26aulast%3DMizukami%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DIdentification%2520of%2520Cell-Active%2520Lysine%2520Specific%2520Demethylase%25201-Selective%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D17536%26epage%3D17537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Ortega, A. M.; Castro-Palomino, L. J.; Fyfe, M. C. T.</span><span> </span><span class="NLM_article-title">Lysine specific demethylase inhibitors and their use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div> WO 2011035941 A1<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=A.+M.+Ortega&author=L.+J.+Castro-Palomino&author=M.+C.+T.+Fyfe&title=Lysine+specific+demethylase+inhibitors+and+their+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DOrtega%26aufirst%3DA.%2BM.%26atitle%3DLysine%2520specific%2520demethylase%2520inhibitors%2520and%2520their%2520use%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Guibourt, N.; Ortega, A. M.; Castro-Palomino, L. J.</span><span> </span><span class="NLM_article-title">Phenylcyclopropylamine derivatives and their medical use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div>, WO 2010084160 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=N.+Guibourt&author=A.+M.+Ortega&author=L.+J.+Castro-Palomino&title=Phenylcyclopropylamine+derivatives+and+their+medical+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGuibourt%26aufirst%3DN.%26atitle%3DPhenylcyclopropylamine%2520derivatives%2520and%2520their%2520medical%2520use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Guibourt, N.; Ortega, A. M.; Castro-Palomino, L. J.</span><span> </span><span class="NLM_article-title">Oxidase inhibitors and their use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div> WO 2010043721 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=N.+Guibourt&author=A.+M.+Ortega&author=L.+J.+Castro-Palomino&title=Oxidase+inhibitors+and+their+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGuibourt%26aufirst%3DN.%26atitle%3DOxidase%2520inhibitors%2520and%2520their%2520use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">McCafferty, D. G.; Pollock, J.</span><span> </span><span class="NLM_article-title">Arylcyclopropylamines and methods of use</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div>  US 20100324147 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=D.+G.+McCafferty&author=J.+Pollock&title=Arylcyclopropylamines+and+methods+of+use"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DArylcyclopropylamines%2520and%2520methods%2520of%2520use%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szewczuk, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A Mechanism-Based Inactivator for Histone Demethylase LSD1</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">4536</span><span class="NLM_x">–</span> <span class="NLM_lpage">4537</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0602748" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=4536-4537&author=J.+C.+Culhaneauthor=L.+M.+Szewczukauthor=X.+Liuauthor=G.+Daauthor=R.+Marmorsteinauthor=P.+A.+Cole&title=A+Mechanism-Based+Inactivator+for+Histone+Demethylase+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja0602748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0602748%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DSzewczuk%26aufirst%3DL.%2BM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDa%26aufirst%3DG.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520Mechanism-Based%2520Inactivator%2520for%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D4536%26epage%3D4537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Comparative Analysis of Small Molecules and Histone Substrate Analogs as LSD1 Lysine Demethylase Inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">3164</span><span class="NLM_x">–</span> <span class="NLM_lpage">3176</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja909996p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=3164-3176&author=J.+C.+Culhaneauthor=D.+Wangauthor=P.+M.+Yenauthor=P.+A.+Cole&title=Comparative+Analysis+of+Small+Molecules+and+Histone+Substrate+Analogs+as+LSD1+Lysine+Demethylase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fja909996p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909996p%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DComparative%2520Analysis%2520of%2520Small%2520Molecules%2520and%2520Histone%2520Substrate%2520Analogs%2520as%2520LSD1%2520Lysine%2520Demethylase%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D3164%26epage%3D3176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Sharma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbergs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowley, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span> </span><span class="NLM_article-title">(Bis)urea and (Bis)thiourea Inhibitors of Lysine-Specific Demethylase 1 as Epigenetic Modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5197</span><span class="NLM_x">–</span> <span class="NLM_lpage">5212</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100217a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5197-5212&author=S.+K.+Sharmaauthor=Y.+Wuauthor=N.+Steinbergsauthor=M.+L.+Crowleyauthor=A.+S.+Hansonauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=%28Bis%29urea+and+%28Bis%29thiourea+Inhibitors+of+Lysine-Specific+Demethylase+1+as+Epigenetic+Modulators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm100217a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100217a%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BK.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSteinbergs%26aufirst%3DN.%26aulast%3DCrowley%26aufirst%3DM.%2BL.%26aulast%3DHanson%26aufirst%3DA.%2BS.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D%2528Bis%2529urea%2520and%2520%2528Bis%2529thiourea%2520Inhibitors%2520of%2520Lysine-Specific%2520Demethylase%25201%2520as%2520Epigenetic%2520Modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5197%26epage%3D5212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray Stewart, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodwin, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8023</span><span class="NLM_x">–</span> <span class="NLM_lpage">8028</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+Murray+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DMurray%2BStewart%26aufirst%3DT.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Hui.</span><span> </span><span class="NLM_article-title">Novel Histone Demethylase LSD1 Inhibitors Selectively Target Cancer Cells with Pluripotent Stem Cell Properties</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">7238</span><span class="NLM_x">–</span> <span class="NLM_lpage">7249</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=7238-7249&author=J.+Wangauthor=F.+Luauthor=Q.+Renauthor=H.+Sunauthor=Z.+Xuauthor=R.+Lanauthor=Y.+Liuauthor=D.+Wardauthor=J.+Quanauthor=T.+Yeauthor=Hui.+Zhang&title=Novel+Histone+Demethylase+LSD1+Inhibitors+Selectively+Target+Cancer+Cells+with+Pluripotent+Stem+Cell+Properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DF.%26aulast%3DRen%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLan%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWard%26aufirst%3DD.%26aulast%3DQuan%26aufirst%3DJ.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DHui.%26atitle%3DNovel%2520Histone%2520Demethylase%2520LSD1%2520Inhibitors%2520Selectively%2520Target%2520Cancer%2520Cells%2520with%2520Pluripotent%2520Stem%2520Cell%2520Properties%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D7238%26epage%3D7249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jandausch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forne, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imhof, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">2704</span><span class="NLM_x">–</span> <span class="NLM_lpage">2709</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1002%2Fijc.27555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=22447389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2012&pages=2704-2709&author=D.+Willmannauthor=S.+Limauthor=S.+Wetzelauthor=E.+Metzgerauthor=A.+Jandauschauthor=W.+Wilkauthor=M.+Jungauthor=I.+Forneauthor=A.+Imhofauthor=A.+Janzerauthor=J.+Kirfelauthor=H.+Waldmannauthor=R.+Sch%C3%BCleauthor=R.+Buettner&title=Impairment+of+prostate+cancer+cell+growth+by+a+selective+and+reversible+lysine-specific+demethylase+1+inhibitor"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Impairment of prostate cancer cell growth by a selective and reversible lysine-specific demethylase 1 inhibitor</span></div><div class="casAuthors">Willmann, Dominica; Lim, Soyoung; Wetzel, Stefan; Metzger, Eric; Jandausch, Anett; Wilk, Wolfram; Jung, Manfred; Forne, Ignasi; Imhof, Axel; Janzer, Andreas; Kirfel, Jutta; Waldmann, Herbert; Schuele, Roland; Buettner, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2704-2709</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Post-translational modifications of histones by chromatin modifying enzymes regulate chromatin structure and gene expression.  As deregulation of histone modifications contributes to cancer progression, inhibition of chromatin modifying enzymes such as histone demethylases is an attractive therapeutic strategy to impair cancer growth.  Lysine-specific demethylase 1 (LSD1) removes mono- and di-Me marks from lysine 4 or 9 of histone H3.  LSD1 in assocn. with the androgen receptor (AR) controls androgen-dependent gene expression and prostate tumor cell proliferation, thus highlighting LSD1 as a drug target.  By combining protein structure similarity clustering and in vitro screening, we identified Namoline, a γ-pyrone, as a novel, selective and reversible LSD1 inhibitor.  Namoline blocks LSD1 demethylase activity in vitro and in vivo.  Inhibition of LSD1 by Namoline leads to silencing of AR-regulated gene expression and severely impairs androgen-dependent proliferation in vitro and in vivo.  Thus, Namoline is a novel promising starting compd. for the development of therapeutics to treat androgen-dependent prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKxYi0f1zoOrVg90H21EOLACvtfcHk0ljAK1XNCtW4OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptlSis78%253D&md5=3104057823357b199b635e1a22071b2a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fijc.27555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27555%26sid%3Dliteratum%253Aachs%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DJandausch%26aufirst%3DA.%26aulast%3DWilk%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DForne%26aufirst%3DI.%26aulast%3DImhof%26aufirst%3DA.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DKirfel%26aufirst%3DJ.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DImpairment%2520of%2520prostate%2520cancer%2520cell%2520growth%2520by%2520a%2520selective%2520and%2520reversible%2520lysine-specific%2520demethylase%25201%2520inhibitor%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D131%26spage%3D2704%26epage%3D2709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Vankayalapati, H.; Sorna, V.; Warner, S. L.; Bearss, D. J.; Sharma, S.; Stephens, B.</span><span> </span><span class="NLM_article-title">Substituted (<i>E</i>)-<i>N</i>′-(1-phenylethylidene) benzohydrazide analogs as histone demethylase inhibitors</span>. <div class="note"><p class="first last">PCT Int. Appl.</p></div> WO 2013025805 A1,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=H.+Vankayalapati&author=V.+Sorna&author=S.+L.+Warner&author=D.+J.+Bearss&author=S.+Sharma&author=B.+Stephens&title=Substituted+%28E%29-N%E2%80%B2-%281-phenylethylidene%29+benzohydrazide+analogs+as+histone+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DVankayalapati%26aufirst%3DH.%26atitle%3DSubstituted%2520%2528E%2529-N%25E2%2580%25B2-%25281-phenylethylidene%2529%2520benzohydrazide%2520analogs%2520as%2520histone%2520demethylase%2520inhibitors%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengoku, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuzawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Crystal structure of histone demethylase LSD1 and tranylcypromine at 2.25 Å</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">366</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2008&pages=15-22&author=S.+Mimasuauthor=T.+Sengokuauthor=S.+Fukuzawaauthor=T.+Umeharaauthor=S.+Yokoyama&title=Crystal+structure+of+histone+demethylase+LSD1+and+tranylcypromine+at+2.25+%C3%85"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DSengoku%26aufirst%3DT.%26aulast%3DFukuzawa%26aufirst%3DS.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520histone%2520demethylase%2520LSD1%2520and%2520tranylcypromine%2520at%25202.25%2520%25C3%2585%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2008%26volume%3D366%26spage%3D15%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Gooden, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, D. M. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabadi, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, D. G.</span><span> </span><span class="NLM_article-title">Facile synthesis of substituted <i>trans</i>-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">3047</span><span class="NLM_x">–</span> <span class="NLM_lpage">3051</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1016%2Fj.bmcl.2008.01.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm400870h&amp;key=18242989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=3047-3051&author=D.+M.+Goodenauthor=D.+M.+Z.+Schmidtauthor=J.+A.+Pollockauthor=A.+M.+Kabadiauthor=D.+G.+McCafferty&title=Facile+synthesis+of+substituted+trans-2-arylcyclopropylamine+inhibitors+of+the+human+histone+demethylase+LSD1+and+monoamine+oxidases+A+and+B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of substituted trans-2-arylcyclopropylamine inhibitors of the human histone demethylase LSD1 and monoamine oxidases A and B</span></div><div class="casAuthors">Gooden, David M.; Schmidt, Dawn M. Z.; Pollock, Julie A.; Kabadi, Ami M.; McCafferty, Dewey G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3047-3051</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A facile synthetic route to substituted trans-2-arylcyclopropylamines was developed to provide access to mechanism-based inhibitors of the human flavoenzyme oxidase lysine-specific histone demethylase LSD1 and related enzyme family members such as monoamine oxidases A and B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojt7YjHKC3yrVg90H21EOLACvtfcHk0ljoKE6kNp-ffw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVagurs%253D&md5=2459dcb4cde160186fc17c09258c5906</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2008.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2008.01.003%26sid%3Dliteratum%253Aachs%26aulast%3DGooden%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DD.%2BM.%2BZ.%26aulast%3DPollock%26aufirst%3DJ.%2BA.%26aulast%3DKabadi%26aufirst%3DA.%2BM.%26aulast%3DMcCafferty%26aufirst%3DD.%2BG.%26atitle%3DFacile%2520synthesis%2520of%2520substituted%2520trans-2-arylcyclopropylamine%2520inhibitors%2520of%2520the%2520human%2520histone%2520demethylase%2520LSD1%2520and%2520monoamine%2520oxidases%2520A%2520and%2520B%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D3047%26epage%3D3051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, Y.-I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evers, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B. P.</span><span> </span><span class="NLM_article-title">The SNAG domain of Snail1 functions as a molecular hook for recruiting lysine-specific demethylase 1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1803</span><span class="NLM_x">–</span> <span class="NLM_lpage">1816</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=1803-1816&author=Y.+Linauthor=Y.+Wuauthor=J.+Liauthor=C.+Dongauthor=X.+Yeauthor=Y.-I.+Chiauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=The+SNAG+domain+of+Snail1+functions+as+a+molecular+hook+for+recruiting+lysine-specific+demethylase+1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DC.%26aulast%3DYe%26aufirst%3DX.%26aulast%3DChi%26aufirst%3DY.-I.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DThe%2520SNAG%2520domain%2520of%2520Snail1%2520functions%2520as%2520a%2520molecular%2520hook%2520for%2520recruiting%2520lysine-specific%2520demethylase%25201%26jtitle%3DEMBO%2520J.%26date%3D2010%26volume%3D29%26spage%3D1803%26epage%3D1816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Laurent, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randrianarison-Huetz, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frisan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrieu-Soler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fontenay, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dusanter-Fourt, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duménil, D.</span><span> </span><span class="NLM_article-title">A short Gfi-1B isoform controls erythroid differentiation by recruiting the LSD1–CoREST complex through the dimethylation of its SNAG domain</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">993</span><span class="NLM_x">–</span> <span class="NLM_lpage">1002</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm400870h&amp;key=10.1242%2Fjcs.095877" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmsl2nu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2012&pages=993-1002&author=B.+Laurentauthor=V.+Randrianarison-Huetzauthor=E.+Frisanauthor=C.+Andrieu-Solerauthor=E.+Solerauthor=M.+Fontenayauthor=I.+Dusanter-Fourtauthor=D.+Dum%C3%A9nil&title=A+short+Gfi-1B+isoform+controls+erythroid+differentiation+by+recruiting+the+LSD1%E2%80%93CoREST+complex+through+the+dimethylation+of+its+SNAG+domain"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A short Gfi-1B isoform controls erythroid differentiation by recruiting the LSD1-CoREST complex through the dimethylation of its SNAG domain</span></div><div class="casAuthors">Laurent, Benoit; Randrianarison-Huetz, Voahangy; Frisan, Emilie; Andrieu-Soler, Charlotte; Soler, Eric; Fontenay, Michaela; Dusanter-Fourt, Isabelle; Dumenil, Dominique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">993-1002</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Gfi-1B is a transcriptional repressor essential for the regulation of erythropoiesis and megakaryopoiesis.  Here we identify Gfi-1B p32, a Gfi-1B isoform, as essential for erythroid differentiation.  Gfi-1B p32 is generated by alternative splicing and lacks the two first zinc finger domains of the protein.  Selective knock down of Gfi-1B p32 compromises erythroid differentiation, whereas its ectopic expression induces erythropoiesis in the absence of erythropoietin.  Gfi-1B p32 isoform binds to Gfi-1B target gene promoters and assocs. with the LSD1-CoREST repressor complex more efficiently than the major Gfi-1B p37 isoform.  Furthermore, we show that Gfi-1B includes a KSKK motif in its SNAG domain, which recruits the repressor complex only when dimethylated on lysine 8.  Mutation of lysine 8 prevents Gfi-1B p32-induced erythroid development.  Our results thus highlight a key role for the alternatively spliced Gfi-1B p32 isoform in erythroid development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3kloXBVMss7Vg90H21EOLACvtfcHk0liCN7qWkOij1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmsl2nu7w%253D&md5=6a9672c52e60a5ececdfaccdd1606706</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1242%2Fjcs.095877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.095877%26sid%3Dliteratum%253Aachs%26aulast%3DLaurent%26aufirst%3DB.%26aulast%3DRandrianarison-Huetz%26aufirst%3DV.%26aulast%3DFrisan%26aufirst%3DE.%26aulast%3DAndrieu-Soler%26aufirst%3DC.%26aulast%3DSoler%26aufirst%3DE.%26aulast%3DFontenay%26aufirst%3DM.%26aulast%3DDusanter-Fourt%26aufirst%3DI.%26aulast%3DDum%25C3%25A9nil%26aufirst%3DD.%26atitle%3DA%2520short%2520Gfi-1B%2520isoform%2520controls%2520erythroid%2520differentiation%2520by%2520recruiting%2520the%2520LSD1%25E2%2580%2593CoREST%2520complex%2520through%2520the%2520dimethylation%2520of%2520its%2520SNAG%2520domain%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2012%26volume%3D125%26spage%3D993%26epage%3D1002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Saleque, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooke, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Epigenetic Regulation of Hematopoietic Differentiation by Gfi-1 and Gfi-1b Is Mediated by the Cofactors CoREST and LSD1</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">562</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=562-572&author=S.+Salequeauthor=J.+Kimauthor=H.+M.+Rookeauthor=S.+H.+Orkin&title=Epigenetic+Regulation+of+Hematopoietic+Differentiation+by+Gfi-1+and+Gfi-1b+Is+Mediated+by+the+Cofactors+CoREST+and+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSaleque%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DRooke%26aufirst%3DH.%2BM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DEpigenetic%2520Regulation%2520of%2520Hematopoietic%2520Differentiation%2520by%2520Gfi-1%2520and%2520Gfi-1b%2520Is%2520Mediated%2520by%2520the%2520Cofactors%2520CoREST%2520and%2520LSD1%26jtitle%3DMol.%2520Cell%26date%3D2007%26volume%3D27%26spage%3D562%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm400870h&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0liCN7qWkOij1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reichert, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohse, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Revelo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Topham, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stafforini, D. M.</span><span> </span><span class="NLM_article-title">Deficiency of Phospholipase A2 Group 7 Decreases Intestinal Polyposis and Colon Tumorigenesis in ApcMin/+ Mice</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">2806</span><span class="NLM_x">–</span> <span class="NLM_lpage">2816</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=2806-2816&author=C.+Xuauthor=E.+C.+Reichertauthor=T.+Nakanoauthor=M.+Lohseauthor=A.+A.+Gardnerauthor=M.+P.+Reveloauthor=M.+K.+Tophamauthor=D.+M.+Stafforini&title=Deficiency+of+Phospholipase+A2+Group+7+Decreases+Intestinal+Polyposis+and+Colon+Tumorigenesis+in+ApcMin%2F%2B+Mice"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DC.%26aulast%3DReichert%26aufirst%3DE.%2BC.%26aulast%3DNakano%26aufirst%3DT.%26aulast%3DLohse%26aufirst%3DM.%26aulast%3DGardner%26aufirst%3DA.%2BA.%26aulast%3DRevelo%26aufirst%3DM.%2BP.%26aulast%3DTopham%26aufirst%3DM.%2BK.%26aulast%3DStafforini%26aufirst%3DD.%2BM.%26atitle%3DDeficiency%2520of%2520Phospholipase%2520A2%2520Group%25207%2520Decreases%2520Intestinal%2520Polyposis%2520and%2520Colon%2520Tumorigenesis%2520in%2520ApcMin%252F%252B%2520Mice%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D2806%26epage%3D2816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z5U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Z5U','PDB','2Z5U'); return false;">PDB: 2Z5U</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a href="/doi/suppl/10.1021/jm400870h">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_12765"></div></div></div></div></div><hr /></hr><p class="last">Detailed virtual screening methods; purchased hits, analytical data; docking scores of compounds <b>1</b>–<b>10</b>, commercially available LSD1 hits (111) from the list of 121 compounds selected, Tanimoto similarity coefficients comparing compound <b>12</b> and known LSD1 inhibitors from Chart <a class="ref internalNav" href="#cht1">1</a>; off-target inhibition; comparison of different model fits for enzyme kinetics; binding site model and definition of active site of LSD1 structure; flow diagram for the virtual ligand screening (VLS) using ICM-VLS; Schrodinger workflow GOLD programs; complete reaction schemes for compounds <b>11</b>–<b>22</b>; LC-MS data for compound <b>12</b>. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400870h/suppl_file/jm400870h_si_001.pdf">jm400870h_si_001.pdf (1.82 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm400870h&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm400870h%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-23%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm400870h" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992695fd763da6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
