
<html lang="en"     class="pb-page"  data-request-id="6f01daa7-8ba5-4ec0-8b9c-6b87bad5845c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b00147;website:website:acspubs;issue:issue:10.1021/jmcmar.2017.60.issue-9;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Allosteric Targeting of the Fanconi Anemia Ubiquitin-Conjugating Enzyme Ube2T by Fragment Screening" /></meta><meta name="dc.Creator" content="Francesca E.  Morreale" /></meta><meta name="dc.Creator" content="Alessio  Bortoluzzi" /></meta><meta name="dc.Creator" content="Viduth K.  Chaugule" /></meta><meta name="dc.Creator" content="Connor  Arkinson" /></meta><meta name="dc.Creator" content="Helen  Walden" /></meta><meta name="dc.Creator" content="Alessio  Ciulli" /></meta><meta name="dc.Description" content="Ube2T is the E2 ubiquitin-conjugating enzyme of the Fanconi anemia DNA repair pathway and it is overexpressed in several cancers, representing an attractive target for the development of inhibitors..." /></meta><meta name="Description" content="Ube2T is the E2 ubiquitin-conjugating enzyme of the Fanconi anemia DNA repair pathway and it is overexpressed in several cancers, representing an attractive target for the development of inhibitors..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 2, 2017" /></meta><meta name="dc.Type" content="rapid-communication" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00147" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00147" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00147" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00147" /></link>
        
    
    

<title>Allosteric Targeting of the Fanconi Anemia Ubiquitin-Conjugating Enzyme Ube2T by Fragment Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00147" /></meta><meta property="og:title" content="Allosteric Targeting of the Fanconi Anemia Ubiquitin-Conjugating Enzyme Ube2T by Fragment Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0004.jpeg" /></meta><meta property="og:description" content="Ube2T is the E2 ubiquitin-conjugating enzyme of the Fanconi anemia DNA repair pathway and it is overexpressed in several cancers, representing an attractive target for the development of inhibitors. Despite the extensive efforts in targeting the ubiquitin system, very few E2 binders have currently been discovered. Herein we report the identification of a new allosteric pocket on Ube2T through a fragment screening using biophysical methods. Several fragments binding to this site inhibit ubiquitin conjugation in vitro." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00147"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00147">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00147&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00147&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00147&amp;href=/doi/10.1021/acs.jmedchem.7b00147" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4093-4098</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00062" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Brief Article</span><a href="/doi/10.1021/acs.jmedchem.7b00613" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Allosteric Targeting of the Fanconi Anemia Ubiquitin-Conjugating Enzyme Ube2T by Fragment Screening</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Francesca+E.++Morreale">Francesca E. Morreale</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-9416-0459" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alessio++Bortoluzzi">Alessio Bortoluzzi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Viduth+K.++Chaugule">Viduth K. Chaugule</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Connor++Arkinson">Connor Arkinson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Helen++Walden">Helen Walden</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alessio++Ciulli">Alessio Ciulli</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-8654-1670" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">MRC Protein Phosphorylation and Ubiquitylation Unit, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United Kingdom</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Division of Biological Chemistry and Drug Discovery, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United Kingdom</span></div><div class="corresp-info"><strong>*</strong>For A.C.: phone, +441382386230; fax, +441382386373; e-mail, <a href="/cdn-cgi/l/email-protection#e283cc818b978e8e8ba286978c868787cc8381cc9789"><span class="__cf_email__" data-cfemail="d5b4fbb6bca0b9b9bc95b1a0bbb1b0b0fbb4b6fba0be">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong> H.W.: e-mail: <a href="/cdn-cgi/l/email-protection#dfb7f1a8beb3bbbab19fbbaab1bbbabaf1bebcf1aab4"><span class="__cf_email__" data-cfemail="fd95d38a9c91999893bd998893999898d39c9ed38896">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00147&amp;href=/doi/10.1021%2Facs.jmedchem.7b00147" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4093–4098</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 24, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 January 2017</li><li><span class="item_label"><b>Published</b> online</span>2 May 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 May 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00147" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00147</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00147"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2418</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00147" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Allosteric Targeting of the Fanconi Anemia Ubiquitin-Conjugating Enzyme Ube2T by Fragment Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Francesca&quot;,&quot;last_name&quot;:&quot;E. Morreale&quot;},{&quot;first_name&quot;:&quot;Alessio&quot;,&quot;last_name&quot;:&quot;Bortoluzzi&quot;},{&quot;first_name&quot;:&quot;Viduth&quot;,&quot;last_name&quot;:&quot;K. Chaugule&quot;},{&quot;first_name&quot;:&quot;Connor&quot;,&quot;last_name&quot;:&quot;Arkinson&quot;},{&quot;first_name&quot;:&quot;Helen&quot;,&quot;last_name&quot;:&quot;Walden&quot;},{&quot;first_name&quot;:&quot;Alessio&quot;,&quot;last_name&quot;:&quot;Ciulli&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;4093-4098&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00147&quot;},&quot;abstract&quot;:&quot;Ube2T is the E2 ubiquitin-conjugating enzyme of the Fanconi anemia DNA repair pathway and it is overexpressed in several cancers, representing an attractive target for the development of inhibitors. Despite the extensive efforts in targeting the ubiquitin system, very few E2 binders have currently been discovered. Herein we report the identification of a new allosteric pocket on Ube2T through a fragment screening using biophysical methods. Several fragments binding to this site inhibit ubiquitin conjugation in vitro.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00147&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00147" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00147&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00147" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00147&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00147" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00147&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00147&amp;href=/doi/10.1021/acs.jmedchem.7b00147" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00147" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00147" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00147%26sid%3Dliteratum%253Aachs%26pmid%3D28437106%26genre%3Darticle%26aulast%3DMorreale%26date%3D2017%26atitle%3DAllosteric%2BTargeting%2Bof%2Bthe%2BFanconi%2BAnemia%2BUbiquitin-Conjugating%2BEnzyme%2BUbe2T%2Bby%2BFragment%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D9%26spage%3D4093%26epage%3D4098%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=290682" title="Post-translational modification">Post-translational modification</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/jmcmar.2017.60.issue-9/20170511/jmcmar.2017.60.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Ube2T is the E2 ubiquitin-conjugating enzyme of the Fanconi anemia DNA repair pathway and it is overexpressed in several cancers, representing an attractive target for the development of inhibitors. Despite the extensive efforts in targeting the ubiquitin system, very few E2 binders have currently been discovered. Herein we report the identification of a new allosteric pocket on Ube2T through a fragment screening using biophysical methods. Several fragments binding to this site inhibit ubiquitin conjugation in vitro.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48521" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48521" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Ubiquitination is a post-translational modification of proteins that regulates many cellular processes, from protein degradation to cell cycle progression and DNA repair.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Ubiquitin conjugation to substrate proteins is catalyzed by the sequential action of three enzymes: E1 (ubiquitin-activating enzyme), E2 (ubiquitin-conjugating enzyme), and E3 (ubiquitin ligase).<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> There are approximately 40 known E2s in humans that regulate diverse biological processes, making them attractive drug targets.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> Structural information is available for many E2s, but the lack of deep active site clefts and the need to target the protein surface have led to E2s being considered challenging targets to small molecules.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> Indeed, to date, the possibility of targeting E2s with small molecules has been largely unexplored and very few inhibitors are known.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10">(7-10)</a></div><div class="NLM_p">Ube2T is the E2 enzyme of the Fanconi anemia (FA) pathway,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> which is essential for the repair of DNA interstrand cross-links. Together with FANCL (the E3 ligase), Ube2T catalyzes the monoubiquitination of the heterodimeric FANCI/FANCD2 complex, which is the key signaling event to activate the FA pathway for DNA repair.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14">(11-14)</a> Modulation of DNA repair pathways is an emerging strategy for the development of inhibitors of tumor cell growth, as it can either potentiate the effects of radiotherapy and conventional genotoxins or exploit synthetic lethal interactions.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15-17)</a> The latter approach relies on genetic defects in DNA repair pathways in certain tumor cells, which lead to hypersensitivity toward inhibitors of compensatory pathways.</div><div class="NLM_p last">Ube2T has recently been found to be overexpressed in several tumors,<a onclick="showRef(event, 'ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21">(18-21)</a> including breast<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and prostate cancer,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and therefore it represents an attractive therapeutic target. Here we unveil an allosteric pocket on Ube2T that is ligandable to small molecules as identified via fragment screening using biophysical methods. Fragment-based drug discovery is now an established approach for the development of lead compounds.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25">(22-25)</a> “Fragments” are low molecular weight compounds (typically <250–300 Da), which bind to the target protein with weak affinities (high micromolar to low millimolar range). Fragments represent synthetically tractable starting points for medicinal chemistry to aid their elaboration into high affinity small molecules.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> The weak affinities typically observed for fragments make it challenging to reliably detect and validate their binding. However, direct binding methods can aid detection of novel, secondary, and potentially allosteric sites on protein surfaces.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> One way to enhance success rates is to deploy a combination or cascade of biophysical methods in order to complement the different detection and sensitivity capabilities of the single techniques.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29, 30)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64554" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64554" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our biophysical cascade consisted of a first screen of a library of approximately 1200 fragments using two orthogonal techniques: differential scanning fluorimetry (DSF) and biolayer interferometry (BLI). This was followed by a secondary screen using one-dimensional <sup>1</sup>H NMR spectroscopy, binding site identification through protein-observed NMR spectroscopy, and X-ray crystallography.</div><div class="NLM_p">DSF monitors the unfolding temperature of a protein using a fluorescent dye. We screened our fragment library measuring the effect of fragments as singletons on the melting temperature of Ube2T (Δ<i>T</i><sub>m</sub> = <i>T</i><sub>m Ube2T+fragment</sub> – <i>T</i><sub>m Ube2T</sub>; Supporting Information (SI) <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S1</a>). We selected as primary hits nine fragments that induced stabilization of the protein (Δ<i>T</i><sub>m</sub> ≥ + 0.5 °C) as well as 33 fragments that generated significant negative shifts (Δ<i>T</i><sub>m</sub> ≤ −2.5 °C). Previous work has indeed demonstrated that several destabilizers can be confirmed as real binders.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In parallel, we screened the fragment library using BLI, an optical technique that requires immobilization of the target protein on a biosensor surface. Biotin-labeled Ube2T was attached to streptavidin-coated biosensors and screened against the fragment library at a concentration of 200 μM. A total of 90 fragments gave a statistically significant response in BLI (above a threshold value calculated as the sum of the robust standard deviation multiplied by three and the median). Interestingly, most of these 90 BLI hits caused negative Δ<i>T</i><sub>m</sub> in the DSF screening (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S2</a>). We then repeated the BLI experiment with the 90 fragments using a six-point concentration series, and 37 hits were selected after visual inspection of the individual binding sensograms. As a result of this first screen, we obtained 42 DSF hits (nine stabilizers and 33 destabilizers) and 37 BLI hits, out of which 10 were in common between the two screens (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S3A</a>), making a total of 69 unique hits (5.5% overall hit rate).</div><div class="NLM_p">Of these initial hits, 34 fragments were chosen based on chemical diversity and commercial availability and submitted to a series of one-dimensional <sup>1</sup>H NMR spectroscopy binding experiments: water ligand observed gradient spectroscopy (WaterLOGSY),<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> saturation transfer difference (STD),<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and Carr–Purcell–Meiboom–Gill (CPMG)<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34, 35)</a> relaxation-edited sequences. Binding was assessed by comparing the proton signals of the fragments in the presence and in the absence of the protein. This secondary screen validated 13 of the 34 hits identified in the primary screen (61.8% attrition rate).</div><div class="NLM_p">The next goal was the identification of fragments’ binding sites on the Ube2T surface. X-ray crystallography is normally the method of choice for investigating the molecular details of protein–ligand interactions. Protein–fragment complexes are frequently obtained by soaking high-concentration solutions of fragments into apo protein crystals, however, this method has some limitations. Poor solubility and weak affinities of fragments can hamper the obtainment of cocrystal structures with high bound fragment occupancy. Furthermore, targeted pockets may be occluded by crystal packing contacts and limited structural rearrangement can occur without damaging the apo protein crystals, often preventing the identification of cryptic pockets. For the above reasons, we first used protein-observed NMR spectroscopy in solution to map the binding sites.</div><div class="NLM_p">We expressed a truncated form of Ube2T (Ube2T<sub>1–154</sub>, lacking the C-terminal flexible tail),<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> assigned the backbone amide protons of the HSQC spectrum (BMRB entry 27035), and performed chemical shift perturbation (CSP) analysis with each individual fragment. Out of the 13 fragments, six induced detectable shifts at 2 mM concentration after visual inspection of the spectra (the six validated binders were equally distributed between BLI and DSF hits; a summary of the confirmed hits is shown in SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S3</a>, together with some representative data). CSPs shown by the six fragments were concentration-dependent, as demonstrated by a four-point titration of fragment into protein (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S4</a>).</div><div class="NLM_p">Shifts mapped onto the X-ray structures of Ube2T (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YH2">1YH2</a>,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> 4CCG<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>) were primarily clustered around a pocket adjacent to the catalytic cysteine (Cys86), formed by the loop between strand β4 and the 3<sub>10</sub> helix of the E2 fold (<a class="ref internalNav" href="#fig1" aria-label="Figures 1">Figures 1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). A slightly different behavior was observed for EM02 and EM17 (to a lesser extent), which also shifted several amino acids located between strands β1 and β2 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A,E).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical shift perturbations observed upon addition of 3 mM fragment* to 80 μM Ube2T<sub>1–154</sub>. Weighted Δδs ≥ average + SD are indicated by a yellow line. Weighted Δδs ≥ average + 2 × SD are indicated by an orange line. Equivalent colors are used to highlight corresponding residues on the Ube2T surface. As a reference, the catalytic cysteine (C86) is colored in green. Chemical structures for individual fragments are shown. For HSQC spectra and weighted Δδs formula, see <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">SI</a>. *Concentration accuracy is limited by solubility issues for several fragments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of EM04 in complex with Ube2T. (A) EM04-bound Ube2T structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NGZ">5NGZ</a>) showing 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 0.9σ level for fragment EM04. Amino acid side chains in the EM04 binding site are shown as ball and sticks and colored in orange and yellow consistent with the results of the CSP experiments, as described in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. (B) Zoom-in of the fragment’s binding pocket. Hydrogen bonds are represented as dotted yellow lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To elucidate the fragments binding modes, we next obtained well-diffracting protein crystals (1.7 Å for the apo form) that we found to tolerate relatively high concentrations of organic solvents that are needed for fragment soaking. The binding site identified by protein NMR was found to be solvent-accessible in our crystal form, suggesting that it would be suitable for soaking experiments. We could confirm this by solving the crystal structure of Ube2T in complex with EM04 at 2.4 Å resolution (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NGZ">5NGZ</a>, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> and SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S5</a>). The newly discovered pocket is distinct from the small molecules binding sites identified on the E2s Cdc34 and Ubc9 (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S6</a>).</div><div class="NLM_p">EM04 binds Ube2T with a <i>K</i><sub>D</sub> = 1.3 mM (LE = 0.36 kcal mol<sup>–1</sup>), as measured by isothermal titration calorimetry (ITC) (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S7A</a>). As shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B, the binding mode is driven by several hydrophobic contacts and by a hydrogen bond between the amino group of EM04 and the carbonyl group of Phe70 on Ube2T. An −NH<sub>2</sub> (hydrogen bond donor) appears to be a common structural feature between the identified fragments, as it is also present in EM02, EM11, EM17, and EM29, potentially suggesting a conserved interaction. A second hydrogen bond is formed between the backbone NH of Ile74 and the sulfur atom of the benzothiazole ring. These data are in agreement with the CSP experiments, where major shifts were observed for Ile74 backbone NH upon fragment addition (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B and SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S4</a>).</div><div class="NLM_p">Fragments are typically thought as too weak binders to warrant testing in functional assays. However, fragments binding to secondary pockets have been found to exert allosteric effects on their targets.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37, 38)</a> To address this, we next investigated whether the validated fragments could impact Ube2T enzymatic activity using a biochemical assay, which measures the conjugation of fluorescently labeled ubiquitin to the substrate protein FANCD2.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> A significant reduction of substrate ubiquitination was observed upon addition of fragments EM02, EM04, and EM17 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A), suggesting that binding of small molecules to this pocket can lead to inhibition of Ube2T’s biochemical activity. The assay was repeated in the presence of detergent as a control for aggregation (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S8</a>). None of the fragments affected the formation of the Ube2T–ubiquitin thioester (as demonstrated by an E2-charging assay, SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S9</a>), suggesting that such inhibition involves the catalytic transfer of the thioester-linked ubiquitin molecule from Ube2T to the substrate protein FANCD2. The effect of EM02, EM04, and EM17 on FANCD2 ubiquitination was concentration-dependent (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B) as shown by a four-point titration series. Additionally, the three fragments did not show detectable inhibitory effect when tested in a similar biochemical assay set up using UbcH5c as ubiquitin-conjugating enzyme (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S10</a>), suggesting that selectivity toward Ube2T can be achieved.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Fragments inhibit substrate ubiquitination by Ube2T. (A) Effect of the six fragments at 2.5 mM on Ube2T-mediated FANCD2 ubiquitination. A representative fluorescence scan shows the levels of monoubiquitinated FANCD2 using fluorescently labeled ubiquitin. The additional assays components (E1, Ube2T, FANCL) are not shown. The bar chart shows the integrated intensities of monoubiquitinated FANCD2 from three independent experiments normalized to DMSO reactions and plotted as mean ± SEM. (B) Concentration-dependent effect of fragments EM02, EM04, and EM17 in a four-point titration series. Representative fluorescence scans are shown and integrated intensities from three independent experiments are normalized to DMSO reactions and plotted as mean ± range. (C) Effect of 2.5 mM EM02, EM04, and EM17 on FANCD2 ubiquitination using Ube2T P73K mutant in the assay. A representative fluorescence scan is shown, and integrated intensities from four independent experiments are normalized to wild type Ube2T-DMSO reactions and plotted as mean ± SEM. Statistical significance was determined by one-way analysis of variance with Dunnett’s multiple-comparison test. (<i>n</i> = 3 or 4, ****<i>P</i> ≤ 0.0001, ***<i>P</i> ≤ 0.001, **<i>P</i> ≤ 0.01, *<i>P</i> ≤ 0.05, <i>ns</i><i>P</i> > 0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To further confirm that the observed reduction in substrate ubiquitination is due to binding to the newly identified pocket, we introduced a point mutation on Ube2T (P73K), disrupting EM04 binding as measured by ITC (SI, <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">Figure S7B</a>). Such mutation abolished the effect of fragments EM04 and EM17, while EM02 still retained its inhibitory activity (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). These results validate EM04 binding site as responsible for Ube2T inhibition. It is worth noting that Ube2T P73K was less active compared to the wild-type protein (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C), further suggesting the importance of this site for Ube2T function. Our data suggests a different mechanism of action for EM02, which exhibited a different pattern in CSPs (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A) and is the subject of current investigation in our laboratories.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72652" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72652" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, using fragment screening, we discovered a new allosteric pocket on Ube2T, the E2 ubiquitin-conjugating enzyme of the FA pathway. Small molecules binding to this site can inhibit substrate ubiquitination in vitro, suggesting an allosteric modulation of Ube2T enzymatic activity, although we cannot rule out potential direct inhibition of specific protein–protein interactions. A cascade of biophysical methods was successfully implemented to screen and validate the binding of fragment-like small molecules and to elucidate their binding mode. Such fragments represent attractive starting points for further elaboration into high affinity inhibitors. A potent Ube2T inhibitor could find application in sensitizing cancer cells to cross-linking agents (such as cysplatin and mitomycin C) or be used to exploit specific synthetic lethal interactions. Very few E2 inhibitors are known to date, and targeting this newly discovered allosteric pocket could represent a successful strategy to develop high-affinity small molecules to effectively drug this class of enzymes.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00147">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43655" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43655" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00147" class="ext-link">10.1021/acs.jmedchem.7b00147</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental section: protein expression and purification, DSF, BLI, ligand-observed NMR spectroscopy, Ube2T backbone resonance assignment, CSP experiments, ITC, crystallization and structure determination, ubiquitination assays (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf">jm7b00147_si_001.pdf (8.96 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_002.csv">jm7b00147_si_002.csv (0.2 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates can be accessed using PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NGZ">5NGZ</a>. Authors will release the atomic coordinates and experimental data upon article publication. Backbone assignment of human Ube2T(1–154) can be accessed using BMRB accession number 27035.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b00147" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Helen Walden</span> - <span class="hlFld-Affiliation affiliation">MRC
Protein Phosphorylation and Ubiquitylation Unit, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United Kingdom</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bbd395ccdad7dfded5fbdfced5dfdede95dad895ced0"><span class="__cf_email__" data-cfemail="a4cc8ad3c5c8c0c1cae4c0d1cac0c1c18ac5c78ad1cf">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessio Ciulli</span> - <span class="hlFld-Affiliation affiliation">Division
of Biological Chemistry and Drug Discovery, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United
Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8654-1670" title="Orcid link">http://orcid.org/0000-0002-8654-1670</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f495da979d8198989db490819a909191da9597da819f"><span class="__cf_email__" data-cfemail="8beaa5e8e2fee7e7e2cbeffee5efeeeea5eae8a5fee0">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesca E. Morreale</span> - <span class="hlFld-Affiliation affiliation">MRC
Protein Phosphorylation and Ubiquitylation Unit, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9416-0459" title="Orcid link">http://orcid.org/0000-0001-9416-0459</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessio Bortoluzzi</span> - <span class="hlFld-Affiliation affiliation">Division
of Biological Chemistry and Drug Discovery, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United
Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Viduth K. Chaugule</span> - <span class="hlFld-Affiliation affiliation">MRC
Protein Phosphorylation and Ubiquitylation Unit, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Connor Arkinson</span> - <span class="hlFld-Affiliation affiliation">MRC
Protein Phosphorylation and Ubiquitylation Unit, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United Kingdom</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d46e566-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60200" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60200" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the European Research Council (ERC-2012-StG-311460 DrugE3CRLs starting grant to A.C.; ERC-2015-CoG-681582 ICLUb consolidator grant to H.W.); the UK Biotechnology and Biological Sciences Research Council (BBSRC BB/G023123/2 David Phillips Fellowship to A.C.); the Medical Research Council (MRC grant number MC_UU_12016/12); the EMBO Young Investigator Programme to H.W.; and the Wellcome Trust (strategic awards 100476/Z/12/Z for biophysics and drug discovery and 094090/Z/10/Z for structural biology and X-ray crystallography to BCDD). We thank D. Robinson (Drug Discovery Unit, University of Dundee) for providing access to fragment libraries and R. T. Hay (University of Dundee) for sharing the cDNA construct for 6xHis-MBP-RNF4 RING fusion protein. We thank the Diamond Light Source for beamtime (BAG proposal MX10071) and beamline support at beamline I04-1.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i7" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i7"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i8" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i8"> Abbreviations Used</h2><tr><td class="NLM_term">FA</td><td class="NLM_def"><p class="first last">Fanconi anemia</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">BLI</td><td class="NLM_def"><p class="first last">biolayer interferometry</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">WaterLOGSY</td><td class="NLM_def"><p class="first last">water ligand observed gradient spectroscopy</p></td></tr><tr><td class="NLM_term">STD</td><td class="NLM_def"><p class="first last">saturation transfer difference</p></td></tr><tr><td class="NLM_term">CPMG</td><td class="NLM_def"><p class="first last">Carr–Purcell–Meiboom–Gill</p></td></tr><tr><td class="NLM_term">CSP</td><td class="NLM_def"><p class="first last">chemical shift perturbation</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal titration calorimetry</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard errors of the mean</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Pickart, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eddins, M. J.</span><span> </span><span class="NLM_article-title">Ubiquitin: structures, functions, mechanisms</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1695</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2004.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.bbamcr.2004.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=15571809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1695&publication_year=2004&pages=55-72&author=C.+M.+Pickartauthor=M.+J.+Eddins&title=Ubiquitin%3A+structures%2C+functions%2C+mechanisms&doi=10.1016%2Fj.bbamcr.2004.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin: structures, functions, mechanisms</span></div><div class="casAuthors">Pickart, Cecile M.; Eddins, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1695</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">55-72</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Ubiquitin (Ub) is the founding member of a family of structurally conserved proteins that regulate a host of processes in eukaryotic cells.  Ub and its relatives carry out their functions through covalent attachment to other cellular proteins, thereby changing the stability, localization, or activity of the target protein.  Here, the authors review the basic biochem. of these protein conjugation reactions, focusing on Ub itself and emphasizing recent insights into mechanism and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZlvpyTnAFurVg90H21EOLACvtfcHk0ljMrfqgLVx0Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzO&md5=05bee85acf076fed3ade4b76109d8f7d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2004.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2004.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DPickart%26aufirst%3DC.%2BM.%26aulast%3DEddins%26aufirst%3DM.%2BJ.%26atitle%3DUbiquitin%253A%2520structures%252C%2520functions%252C%2520mechanisms%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2004%26volume%3D1695%26spage%3D55%26epage%3D72%26doi%3D10.1016%2Fj.bbamcr.2004.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Welchman, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, R. J.</span><span> </span><span class="NLM_article-title">Ubiquitin and ubiquitin-like proteins as multifunctional signals</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">599</span><span class="NLM_x">–</span> <span class="NLM_lpage">609</span><span class="refDoi"> DOI: 10.1038/nrm1700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrm1700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=16064136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=599-609&author=R.+L.+Welchmanauthor=C.+Gordonauthor=R.+J.+Mayer&title=Ubiquitin+and+ubiquitin-like+proteins+as+multifunctional+signals&doi=10.1038%2Fnrm1700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin and ubiquitin-like proteins as multifunctional signals</span></div><div class="casAuthors">Welchman, Rebecca L.; Gordon, Colin; Mayer, R. John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">599-609</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination is a recognized signal for protein degrdn.  However, it is increasingly realized that ubiquitin conjugation to proteins can be used for many other purposes.  Furthermore, there are many ubiquitin-like proteins that control the activities of proteins.  The central structural element of these post-translational modifications is the ubiquitin superfold.  A common ancestor based on this superfold has evolved to give various proteins that are involved in diverse activities in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr72A1XAR84W7Vg90H21EOLACvtfcHk0ljMrfqgLVx0Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D&md5=25f0eb2aec7ffb99b78034da9f81f3f3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrm1700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1700%26sid%3Dliteratum%253Aachs%26aulast%3DWelchman%26aufirst%3DR.%2BL.%26aulast%3DGordon%26aufirst%3DC.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26atitle%3DUbiquitin%2520and%2520ubiquitin-like%2520proteins%2520as%2520multifunctional%2520signals%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D599%26epage%3D609%26doi%3D10.1038%2Fnrm1700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Dye, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulman, B. A.</span><span> </span><span class="NLM_article-title">Structural mechanisms underlying posttranslational modification by ubiquitin-like proteins</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span><span class="refDoi"> DOI: 10.1146/annurev.biophys.36.040306.132820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1146%2Fannurev.biophys.36.040306.132820" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2007&pages=131-150&author=B.+T.+Dyeauthor=B.+A.+Schulman&title=Structural+mechanisms+underlying+posttranslational+modification+by+ubiquitin-like+proteins&doi=10.1146%2Fannurev.biophys.36.040306.132820"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.36.040306.132820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.36.040306.132820%26sid%3Dliteratum%253Aachs%26aulast%3DDye%26aufirst%3DB.%2BT.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructural%2520mechanisms%2520underlying%2520posttranslational%2520modification%2520by%2520ubiquitin-like%2520proteins%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2007%26volume%3D36%26spage%3D131%26epage%3D150%26doi%3D10.1146%2Fannurev.biophys.36.040306.132820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">van Wijk, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmers, H. T.</span><span> </span><span class="NLM_article-title">The family of ubiquitin-conjugating enzymes (E2s): deciding between life and death of proteins</span> <span class="citation_source-journal">FASEB J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">981</span><span class="NLM_x">–</span> <span class="NLM_lpage">993</span><span class="refDoi"> DOI: 10.1096/fj.09-136259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1096%2Ffj.09-136259" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=981-993&author=S.+J.+van+Wijkauthor=H.+T.+Timmers&title=The+family+of+ubiquitin-conjugating+enzymes+%28E2s%29%3A+deciding+between+life+and+death+of+proteins&doi=10.1096%2Ffj.09-136259"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1096%2Ffj.09-136259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.09-136259%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWijk%26aufirst%3DS.%2BJ.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%26atitle%3DThe%2520family%2520of%2520ubiquitin-conjugating%2520enzymes%2520%2528E2s%2529%253A%2520deciding%2520between%2520life%2520and%2520death%2520of%2520proteins%26jtitle%3DFASEB%2520J.%26date%3D2010%26volume%3D24%26spage%3D981%26epage%3D993%26doi%3D10.1096%2Ffj.09-136259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Stewart, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritterhoff, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klevit, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brzovic, P. S.</span><span> </span><span class="NLM_article-title">E2 enzymes: more than just middle men</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">440</span><span class="refDoi"> DOI: 10.1038/cr.2016.35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fcr.2016.35" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27002219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVyhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=423-440&author=M.+D.+Stewartauthor=T.+Ritterhoffauthor=R.+E.+Klevitauthor=P.+S.+Brzovic&title=E2+enzymes%3A+more+than+just+middle+men&doi=10.1038%2Fcr.2016.35"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">E2 enzymes: more than just middle men</span></div><div class="casAuthors">Stewart, Mikaela D.; Ritterhoff, Tobias; Klevit, Rachel E.; Brzovic, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">423-440</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Ubiquitin-conjugating enzymes (E2s) are the central players in the trio of enzymes responsible for the attachment of ubiquitin (Ub) to cellular proteins.  Humans have ∼40 E2s that are involved in the transfer of Ub or Ub-like (Ubl) proteins (e.g., SUMO and NEDD8).  Although the majority of E2s are only twice the size of Ub, this remarkable family of enzymes performs a variety of functional roles.  In this review, we summarize common functional and structural features that define unifying themes among E2s and highlight emerging concepts in the mechanism and regulation of E2s.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3nz4HNwXLKLVg90H21EOLACvtfcHk0lipEYO_EFgUeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvVyhs7s%253D&md5=de200ea92123a8aeabd2ecc3d0c3cf65</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcr.2016.35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2016.35%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DM.%2BD.%26aulast%3DRitterhoff%26aufirst%3DT.%26aulast%3DKlevit%26aufirst%3DR.%2BE.%26aulast%3DBrzovic%26aufirst%3DP.%2BS.%26atitle%3DE2%2520enzymes%253A%2520more%2520than%2520just%2520middle%2520men%26jtitle%3DCell%2520Res.%26date%3D2016%26volume%3D26%26spage%3D423%26epage%3D440%26doi%3D10.1038%2Fcr.2016.35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Sheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doherty, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srikumar, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shloush, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avvakumov, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neculai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raught, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span> </span><span class="NLM_article-title">A human ubiquitin conjugating enzyme (E2)-HECT E3 ligase structure-function screen</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1074/mcp.O111.013706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1074%2Fmcp.O111.013706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=22496338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A280%3ADC%252BC38rktlOjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=329-341&author=Y.+Shengauthor=J.+H.+Hongauthor=R.+Dohertyauthor=T.+Srikumarauthor=J.+Shloushauthor=G.+V.+Avvakumovauthor=J.+R.+Walkerauthor=S.+Xueauthor=D.+Neculaiauthor=J.+W.+Wanauthor=S.+K.+Kimauthor=C.+H.+Arrowsmithauthor=B.+Raughtauthor=S.+Dhe-Paganon&title=A+human+ubiquitin+conjugating+enzyme+%28E2%29-HECT+E3+ligase+structure-function+screen&doi=10.1074%2Fmcp.O111.013706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A human ubiquitin conjugating enzyme (E2)-HECT E3 ligase structure-function screen</span></div><div class="casAuthors">Sheng Yi; Hong Jenny H; Doherty Ryan; Srikumar Tharan; Shloush Jonathan; Avvakumov George V; Walker John R; Xue Sheng; Neculai Dante; Wan Janet W; Kim Sung K; Arrowsmith Cheryl H; Raught Brian; Dhe-Paganon Sirano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & cellular proteomics : MCP</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">329-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Here we describe a systematic structure-function analysis of the human ubiquitin (Ub) E2 conjugating proteins, consisting of the determination of 15 new high-resolution three-dimensional structures of E2 catalytic domains, and autoubiquitylation assays for 26 Ub-loading E2s screened against a panel of nine different HECT (homologous to E6-AP carboxyl terminus) E3 ligase domains.  Integration of our structural and biochemical data revealed several E2 surface properties associated with Ub chain building activity; (1) net positive or neutral E2 charge, (2) an "acidic trough" located near the catalytic Cys, surrounded by an extensive basic region, and (3) similarity to the previously described HECT binding signature in UBE2L3 (UbcH7).  Mass spectrometry was used to characterize the autoubiquitylation products of a number of functional E2-HECT pairs, and demonstrated that HECT domains from different subfamilies catalyze the formation of very different types of Ub chains, largely independent of the E2 in the reaction.  Our data set represents the first comprehensive analysis of E2-HECT E3 interactions, and thus provides a framework for better understanding the molecular mechanisms of ubiquitylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsQ0fBh6uWTUNHe_gFldTbfW6udTcc2eaSCez6Bqdwkbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rktlOjsw%253D%253D&md5=cd7b79319d270d7fbb1599557d274507</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fmcp.O111.013706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.O111.013706%26sid%3Dliteratum%253Aachs%26aulast%3DSheng%26aufirst%3DY.%26aulast%3DHong%26aufirst%3DJ.%2BH.%26aulast%3DDoherty%26aufirst%3DR.%26aulast%3DSrikumar%26aufirst%3DT.%26aulast%3DShloush%26aufirst%3DJ.%26aulast%3DAvvakumov%26aufirst%3DG.%2BV.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DXue%26aufirst%3DS.%26aulast%3DNeculai%26aufirst%3DD.%26aulast%3DWan%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DS.%2BK.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DRaught%26aufirst%3DB.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26atitle%3DA%2520human%2520ubiquitin%2520conjugating%2520enzyme%2520%2528E2%2529-HECT%2520E3%2520ligase%2520structure-function%2520screen%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2012%26volume%3D11%26spage%3D329%26epage%3D341%26doi%3D10.1074%2Fmcp.O111.013706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Ceccarelli, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlicky, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plantevin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neculai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogunjimi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Hakim, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varelas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koszela, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Girona, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrana, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durocher, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meloche, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicheri, F.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of the human Cdc34 ubiquitin-conjugating enzyme</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">145</span><span class="NLM_x">, </span> <span class="NLM_fpage">1075</span><span class="NLM_x">–</span> <span class="NLM_lpage">1087</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.05.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.cell.2011.05.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2011&pages=1075-1087&author=D.+F.+Ceccarelliauthor=X.+Tangauthor=B.+Pelletierauthor=S.+Orlickyauthor=W.+Xieauthor=V.+Plantevinauthor=D.+Neculaiauthor=Y.+C.+Chouauthor=A.+Ogunjimiauthor=A.+Al-Hakimauthor=X.+Varelasauthor=J.+Koszelaauthor=G.+A.+Wasneyauthor=M.+Vedadiauthor=S.+Dhe-Paganonauthor=S.+Coxauthor=S.+Xuauthor=A.+Lopez-Gironaauthor=F.+Mercurioauthor=J.+Wranaauthor=D.+Durocherauthor=S.+Melocheauthor=D.+R.+Webbauthor=M.+Tyersauthor=F.+Sicheri&title=An+allosteric+inhibitor+of+the+human+Cdc34+ubiquitin-conjugating+enzyme&doi=10.1016%2Fj.cell.2011.05.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.05.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.05.039%26sid%3Dliteratum%253Aachs%26aulast%3DCeccarelli%26aufirst%3DD.%2BF.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPelletier%26aufirst%3DB.%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DPlantevin%26aufirst%3DV.%26aulast%3DNeculai%26aufirst%3DD.%26aulast%3DChou%26aufirst%3DY.%2BC.%26aulast%3DOgunjimi%26aufirst%3DA.%26aulast%3DAl-Hakim%26aufirst%3DA.%26aulast%3DVarelas%26aufirst%3DX.%26aulast%3DKoszela%26aufirst%3DJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DCox%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DWrana%26aufirst%3DJ.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DMeloche%26aufirst%3DS.%26aulast%3DWebb%26aufirst%3DD.%2BR.%26aulast%3DTyers%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520the%2520human%2520Cdc34%2520ubiquitin-conjugating%2520enzyme%26jtitle%3DCell%26date%3D2011%26volume%3D145%26spage%3D1075%26epage%3D1087%26doi%3D10.1016%2Fj.cell.2011.05.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Hewitt, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lountos, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zlotkowski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlhauser, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saunders, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Needle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropea, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waugh, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneekloth, J. S.,  Jr.</span><span> </span><span class="NLM_article-title">Insights into the allosteric inhibition of the SUMO E2 enzyme Ubc9</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5703</span><span class="NLM_x">–</span> <span class="NLM_lpage">5707</span><span class="refDoi"> DOI: 10.1002/anie.201511351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1002%2Fanie.201511351" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5703-5707&author=W.+M.+Hewittauthor=G.+T.+Lountosauthor=K.+Zlotkowskiauthor=S.+D.+Dahlhauserauthor=L.+B.+Saundersauthor=D.+Needleauthor=J.+E.+Tropeaauthor=C.+Zhanauthor=G.+Weiauthor=B.+Maauthor=R.+Nussinovauthor=D.+S.+Waughauthor=J.+S.+Schneekloth&title=Insights+into+the+allosteric+inhibition+of+the+SUMO+E2+enzyme+Ubc9&doi=10.1002%2Fanie.201511351"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201511351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201511351%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLountos%26aufirst%3DG.%2BT.%26aulast%3DZlotkowski%26aufirst%3DK.%26aulast%3DDahlhauser%26aufirst%3DS.%2BD.%26aulast%3DSaunders%26aufirst%3DL.%2BB.%26aulast%3DNeedle%26aufirst%3DD.%26aulast%3DTropea%26aufirst%3DJ.%2BE.%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWaugh%26aufirst%3DD.%2BS.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DInsights%2520into%2520the%2520allosteric%2520inhibition%2520of%2520the%2520SUMO%2520E2%2520enzyme%2520Ubc9%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D5703%26epage%3D5707%26doi%3D10.1002%2Fanie.201511351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Hodge, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pulvino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spyracopoulos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendzel, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glover, J. N.</span><span> </span><span class="NLM_article-title">Covalent inhibition of Ubc13 affects ubiquitin signaling and reveals active site elements important for targeting</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1718</span><span class="NLM_x">–</span> <span class="NLM_lpage">1728</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1718-1728&author=C.+D.+Hodgeauthor=R.+A.+Edwardsauthor=C.+J.+Markinauthor=D.+McDonaldauthor=M.+Pulvinoauthor=M.+S.+Huenauthor=J.+Zhaoauthor=L.+Spyracopoulosauthor=M.+J.+Hendzelauthor=J.+N.+Glover&title=Covalent+inhibition+of+Ubc13+affects+ubiquitin+signaling+and+reveals+active+site+elements+important+for+targeting&doi=10.1021%2Facschembio.5b00222"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00222%26sid%3Dliteratum%253Aachs%26aulast%3DHodge%26aufirst%3DC.%2BD.%26aulast%3DEdwards%26aufirst%3DR.%2BA.%26aulast%3DMarkin%26aufirst%3DC.%2BJ.%26aulast%3DMcDonald%26aufirst%3DD.%26aulast%3DPulvino%26aufirst%3DM.%26aulast%3DHuen%26aufirst%3DM.%2BS.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DSpyracopoulos%26aufirst%3DL.%26aulast%3DHendzel%26aufirst%3DM.%2BJ.%26aulast%3DGlover%26aufirst%3DJ.%2BN.%26atitle%3DCovalent%2520inhibition%2520of%2520Ubc13%2520affects%2520ubiquitin%2520signaling%2520and%2520reveals%2520active%2520site%2520elements%2520important%2520for%2520targeting%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1718%26epage%3D1728%26doi%3D10.1021%2Facschembio.5b00222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Strickson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmerich, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knebel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plater, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritorto, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shpiro, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">The anti-inflammatory drug BAY 11–7082 suppresses the MyD88-dependent signalling network by targeting the ubiquitin system</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">451</span><span class="NLM_x">, </span> <span class="NLM_fpage">427</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1042/BJ20121651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1042%2FBJ20121651" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2013&pages=427-437&author=S.+Stricksonauthor=D.+G.+Campbellauthor=C.+H.+Emmerichauthor=A.+Knebelauthor=L.+Platerauthor=M.+S.+Ritortoauthor=N.+Shpiroauthor=P.+Cohen&title=The+anti-inflammatory+drug+BAY+11%E2%80%937082+suppresses+the+MyD88-dependent+signalling+network+by+targeting+the+ubiquitin+system&doi=10.1042%2FBJ20121651"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1042%2FBJ20121651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121651%26sid%3Dliteratum%253Aachs%26aulast%3DStrickson%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DD.%2BG.%26aulast%3DEmmerich%26aufirst%3DC.%2BH.%26aulast%3DKnebel%26aufirst%3DA.%26aulast%3DPlater%26aufirst%3DL.%26aulast%3DRitorto%26aufirst%3DM.%2BS.%26aulast%3DShpiro%26aufirst%3DN.%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DThe%2520anti-inflammatory%2520drug%2520BAY%252011%25E2%2580%25937082%2520suppresses%2520the%2520MyD88-dependent%2520signalling%2520network%2520by%2520targeting%2520the%2520ubiquitin%2520system%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D451%26spage%3D427%26epage%3D437%26doi%3D10.1042%2FBJ20121651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Machida, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurtan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupfer, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Andrea, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, A.</span><span> </span><span class="NLM_article-title">UBE2T is the E2 in the Fanconi anemia pathway and undergoes negative autoregulation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">596</span><span class="refDoi"> DOI: 10.1016/j.molcel.2006.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.molcel.2006.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=16916645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=589-596&author=Y.+J.+Machidaauthor=Y.+Machidaauthor=Y.+Chenauthor=A.+M.+Gurtanauthor=G.+M.+Kupferauthor=A.+D.+D%E2%80%99Andreaauthor=A.+Dutta&title=UBE2T+is+the+E2+in+the+Fanconi+anemia+pathway+and+undergoes+negative+autoregulation&doi=10.1016%2Fj.molcel.2006.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">UBE2T is the E2 in the Fanconi anemia pathway and undergoes negative autoregulation</span></div><div class="casAuthors">Machida, Yuichi J.; Machida, Yuka; Chen, Yuefeng; Gurtan, Allan M.; Kupfer, Gary M.; D'Andrea, Alan D.; Dutta, Anindya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">589-596</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Fanconi anemia pathway is required for the efficient repair of damaged DNA.  A key step in this pathway is the monoubiquitination of the FANCD2 protein by the ubiquitin ligase (E3) composed of Fanconi anemia core complex proteins.  Here, we show that UBE2T is the ubiquitin-conjugating enzyme (E2) essential for this pathway.  UBE2T binds to FANCL, the ubiquitin ligase subunit of the Fanconi anemia core complex, and is required for the monoubiquitination of FANCD2 in vivo.  DNA damage in UBE2T-depleted cells leads to the formation of abnormal chromosomes that are a hallmark of Fanconi anemia.  In addn., we show that UBE2T undergoes automonoubiquitination in vivo.  This monoubiquitination is stimulated by the presence of the FANCL protein and inactivates UBE2T.  Therefore, UBE2T is the E2 in the Fanconi anemia pathway and has a self-inactivation mechanism that could be important for neg. regulation of the Fanconi anemia pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8G6AIGwr7GrVg90H21EOLACvtfcHk0lgulghh_yYdvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurs%253D&md5=e891d40a0bee00d9939defb3ef05aeb6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DMachida%26aufirst%3DY.%2BJ.%26aulast%3DMachida%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGurtan%26aufirst%3DA.%2BM.%26aulast%3DKupfer%26aufirst%3DG.%2BM.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DA.%2BD.%26aulast%3DDutta%26aufirst%3DA.%26atitle%3DUBE2T%2520is%2520the%2520E2%2520in%2520the%2520Fanconi%2520anemia%2520pathway%2520and%2520undergoes%2520negative%2520autoregulation%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D589%26epage%3D596%26doi%3D10.1016%2Fj.molcel.2006.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Alpi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langevin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosedale, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machida, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, K. J.</span><span> </span><span class="NLM_article-title">UBE2T, the Fanconi anemia core complex, and FANCD2 are recruited independently to chromatin: a basis for the regulation of FANCD2 monoubiquitination</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">8421</span><span class="NLM_x">–</span> <span class="NLM_lpage">8430</span><span class="refDoi"> DOI: 10.1128/MCB.00504-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1128%2FMCB.00504-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=17938197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1eqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=8421-8430&author=A.+Alpiauthor=F.+Langevinauthor=G.+Mosedaleauthor=Y.+J.+Machidaauthor=A.+Duttaauthor=K.+J.+Patel&title=UBE2T%2C+the+Fanconi+anemia+core+complex%2C+and+FANCD2+are+recruited+independently+to+chromatin%3A+a+basis+for+the+regulation+of+FANCD2+monoubiquitination&doi=10.1128%2FMCB.00504-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">UBE2T, the Fanconi anemia core complex, and FANCD2 are recruited independently to chromatin: a basis for the regulation of FANCD2 monoubiquitination</span></div><div class="casAuthors">Alpi, Arno; Langevin, Frederic; Mosedale, Georgina; Machida, Yuichi J.; Dutta, Anindya; Patel, Ketan J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8421-8430</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Fanconi anemia (FA) nuclear core complex and the E2 ubiquitin-conjugating enzyme UBE2T are required for the S phase and DNA damage-restricted monoubiquitination of FANCD2.  This constitutes a key step in the FA tumor suppressor pathway, and much attention has been focused on the regulation at this point.  Here, we address the importance of the assembly of the FA core complex and the subcellular localization of UBE2T in the regulation of FANCD2 monoubiquitination.  We establish three points.  First, the stable assembly of the FA core complex can be dissocd. of its ability to function as an E3 ubiquitin ligase.  Second, the actual E3 ligase activity is not detd. by the assembly of the FA core complex but rather by its DNA damage-induced localization to chromatin.  Finally, UBE2T and FANCD2 access this subcellular fraction independently of the FA core complex.  FANCD2 monoubiquitination is therefore not regulated by multiprotein complex assembly but by the formation of an active E2/E3 holoenzyme on chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAOTpFekYGXbVg90H21EOLACvtfcHk0liv_1u0grtTig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1eqsg%253D%253D&md5=f8ad0db3046485f2a5d404a3b7356b23</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.00504-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00504-07%26sid%3Dliteratum%253Aachs%26aulast%3DAlpi%26aufirst%3DA.%26aulast%3DLangevin%26aufirst%3DF.%26aulast%3DMosedale%26aufirst%3DG.%26aulast%3DMachida%26aufirst%3DY.%2BJ.%26aulast%3DDutta%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DK.%2BJ.%26atitle%3DUBE2T%252C%2520the%2520Fanconi%2520anemia%2520core%2520complex%252C%2520and%2520FANCD2%2520are%2520recruited%2520independently%2520to%2520chromatin%253A%2520a%2520basis%2520for%2520the%2520regulation%2520of%2520FANCD2%2520monoubiquitination%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D8421%26epage%3D8430%26doi%3D10.1128%2FMCB.00504-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Alpi, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babu, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, K. J.</span><span> </span><span class="NLM_article-title">Mechanistic insight into site-restricted monoubiquitination of FANCD2 by Ube2t, FANCL, and FANCI</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">767</span><span class="NLM_x">–</span> <span class="NLM_lpage">777</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.molcel.2008.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=19111657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtFGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=767-777&author=A.+F.+Alpiauthor=P.+E.+Paceauthor=M.+M.+Babuauthor=K.+J.+Patel&title=Mechanistic+insight+into+site-restricted+monoubiquitination+of+FANCD2+by+Ube2t%2C+FANCL%2C+and+FANCI&doi=10.1016%2Fj.molcel.2008.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insight into site-restricted monoubiquitination of FANCD2 by Ube2t, FANCL, and FANCI</span></div><div class="casAuthors">Alpi, Arno F.; Pace, Paul E.; Babu, M. Madan; Patel, Ketan J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">767-777</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A key step in the Fanconi anemia (FA) tumor suppressor pathway is the site-specific monoubiquitination of the FANCD2 protein.  Genetic studies indicate that this crucial modification requires eight known FA gene products and the E2-conjugating enzyme Ube2t.  Here, the authors minimally reconstitute this monoubiquitination reaction with Ube2t and the FANCL protein, revealing that monoubiquitination is stimulated by a conserved RWD-like domain in FANCL.  Furthermore, addn. of the FANCI protein enhances monoubiquitination and also restricts it to the in vivo substrate lysine residue on FANCD2.  This work therefore establishes a system that provides mechanistic insight into the functions of FANCL and FANCI in the catalysis of FANCD2 monoubiquitination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ia-b-65807Vg90H21EOLACvtfcHk0liv_1u0grtTig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtFGmtQ%253D%253D&md5=562d6bfd13f96cb7ffadc4170a7097e7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DAlpi%26aufirst%3DA.%2BF.%26aulast%3DPace%26aufirst%3DP.%2BE.%26aulast%3DBabu%26aufirst%3DM.%2BM.%26aulast%3DPatel%26aufirst%3DK.%2BJ.%26atitle%3DMechanistic%2520insight%2520into%2520site-restricted%2520monoubiquitination%2520of%2520FANCD2%2520by%2520Ube2t%252C%2520FANCL%252C%2520and%2520FANCI%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D767%26epage%3D777%26doi%3D10.1016%2Fj.molcel.2008.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Walden, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deans, A. J.</span><span> </span><span class="NLM_article-title">The Fanconi anemia DNA repair pathway: structural and functional insights into a complex disorder</span> <span class="citation_source-journal">Annu. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">278</span><span class="refDoi"> DOI: 10.1146/annurev-biophys-051013-022737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1146%2Fannurev-biophys-051013-022737" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=257-278&author=H.+Waldenauthor=A.+J.+Deans&title=The+Fanconi+anemia+DNA+repair+pathway%3A+structural+and+functional+insights+into+a+complex+disorder&doi=10.1146%2Fannurev-biophys-051013-022737"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-051013-022737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-051013-022737%26sid%3Dliteratum%253Aachs%26aulast%3DWalden%26aufirst%3DH.%26aulast%3DDeans%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520Fanconi%2520anemia%2520DNA%2520repair%2520pathway%253A%2520structural%2520and%2520functional%2520insights%2520into%2520a%2520complex%2520disorder%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2014%26volume%3D43%26spage%3D257%26epage%3D278%26doi%3D10.1146%2Fannurev-biophys-051013-022737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Pearl, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierz, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearl, F. M.</span><span> </span><span class="NLM_article-title">Therapeutic opportunities within the DNA damage response</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">166</span><span class="NLM_x">–</span> <span class="NLM_lpage">180</span><span class="refDoi"> DOI: 10.1038/nrc3891</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrc3891" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=25709118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Snt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=166-180&author=L.+H.+Pearlauthor=A.+C.+Schierzauthor=S.+E.+Wardauthor=B.+Al-Lazikaniauthor=F.+M.+Pearl&title=Therapeutic+opportunities+within+the+DNA+damage+response&doi=10.1038%2Fnrc3891"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic opportunities within the DNA damage response</span></div><div class="casAuthors">Pearl, Laurence H.; Schierz, Amanda C.; Ward, Simon E.; Al-Lazikani, Bissan; Pearl, Frances M. G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-180</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The DNA damage response (DDR) is essential for maintaining the genomic integrity of the cell, and its disruption is one of the hallmarks of cancer.  Classically, defects in the DDR have been exploited therapeutically in the treatment of cancer with radiation therapies or genotoxic chemotherapies.  More recently, protein components of the DDR systems have been identified as promising avenues for targeted cancer therapeutics.  Here, we present an in-depth anal. of the function, role in cancer and therapeutic potential of 450 expert-curated human DDR genes.  We discuss the DDR drugs that have been approved by the US Food and Drug Administration (FDA) or that are under clin. investigation.  We examine large-scale genomic and expression data for 15 cancers to identify deregulated components of the DDR, and we apply systematic computational anal. to identify DDR proteins that are amenable to modulation by small mols., highlighting potential novel therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosptr9Y-2e6LVg90H21EOLACvtfcHk0ljzKOytXC6UzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Snt74%253D&md5=47ceba5c42cc7ebb59e18f3aa53e0f60</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc3891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3891%26sid%3Dliteratum%253Aachs%26aulast%3DPearl%26aufirst%3DL.%2BH.%26aulast%3DSchierz%26aufirst%3DA.%2BC.%26aulast%3DWard%26aufirst%3DS.%2BE.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DPearl%26aufirst%3DF.%2BM.%26atitle%3DTherapeutic%2520opportunities%2520within%2520the%2520DNA%2520damage%2520response%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D166%26epage%3D180%26doi%3D10.1038%2Fnrc3891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Curtin, N. J.</span><span> </span><span class="NLM_article-title">DNA repair dysregulation from cancer driver to therapeutic target</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">801</span><span class="NLM_x">–</span> <span class="NLM_lpage">817</span><span class="refDoi"> DOI: 10.1038/nrc3399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrc3399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23175119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=801-817&author=N.+J.+Curtin&title=DNA+repair+dysregulation+from+cancer+driver+to+therapeutic+target&doi=10.1038%2Fnrc3399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair dysregulation from cancer driver to therapeutic target</span></div><div class="casAuthors">Curtin, Nicola J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">801-817</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dysregulation of DNA damage repair and signalling to cell cycle checkpoints, known as the DNA damage response (DDR), is assocd. with a predisposition to cancer and affects responses to DNA-damaging anticancer therapy.  Dysfunction of one DNA repair pathway may be compensated for by the function of another compensatory DDR pathway, which may be increased and contribute to resistance to DNA-damaging chemotherapy and radiotherapy.  Therefore, DDR pathways make an ideal target for therapeutic intervention; first, to prevent or reverse therapy resistance; and second, using a synthetic lethal approach to specifically kill cancer cells that are dependent on a compensatory DNA repair pathway for survival in the context of cancer-assocd. oxidative and replicative stress.  These hypotheses are currently being tested in the lab. and are being translated into clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNDBV6krLGBbVg90H21EOLACvtfcHk0ljzKOytXC6UzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N&md5=1bcb90d4742e46696c8a30a4fce2400c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc3399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3399%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DDNA%2520repair%2520dysregulation%2520from%2520cancer%2520driver%2520to%2520therapeutic%2520target%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D801%26epage%3D817%26doi%3D10.1038%2Fnrc3399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petermann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, R. A.</span><span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0ljzKOytXC6UzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Ueki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishidate, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kijima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katagiri, T.</span><span> </span><span class="NLM_article-title">Ubiquitination and downregulation of BRCA1 by ubiquitin-conjugating enzyme E2T overexpression in human breast cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">8752</span><span class="NLM_x">–</span> <span class="NLM_lpage">8760</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-1809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1158%2F0008-5472.CAN-09-1809" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=8752-8760&author=T.+Uekiauthor=J.+H.+Parkauthor=T.+Nishidateauthor=K.+Kijimaauthor=K.+Hirataauthor=Y.+Nakamuraauthor=T.+Katagiri&title=Ubiquitination+and+downregulation+of+BRCA1+by+ubiquitin-conjugating+enzyme+E2T+overexpression+in+human+breast+cancer+cells&doi=10.1158%2F0008-5472.CAN-09-1809"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-1809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-1809%26sid%3Dliteratum%253Aachs%26aulast%3DUeki%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DJ.%2BH.%26aulast%3DNishidate%26aufirst%3DT.%26aulast%3DKijima%26aufirst%3DK.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DKatagiri%26aufirst%3DT.%26atitle%3DUbiquitination%2520and%2520downregulation%2520of%2520BRCA1%2520by%2520ubiquitin-conjugating%2520enzyme%2520E2T%2520overexpression%2520in%2520human%2520breast%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D8752%26epage%3D8760%26doi%3D10.1158%2F0008-5472.CAN-09-1809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span> </span><span class="NLM_article-title">Ubiquitin-conjugating enzyme E2T is an independent prognostic factor and promotes gastric cancer progression</span> <span class="citation_source-journal">Tumor Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">11723</span><span class="NLM_x">–</span> <span class="NLM_lpage">11732</span><span class="refDoi"> DOI: 10.1007/s13277-016-5020-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1007%2Fs13277-016-5020-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27020591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A280%3ADC%252BC28fksVWgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=11723-11732&author=H.+Yuauthor=P.+Xiangauthor=Q.+Panauthor=Y.+Huangauthor=N.+Xieauthor=W.+Zhu&title=Ubiquitin-conjugating+enzyme+E2T+is+an+independent+prognostic+factor+and+promotes+gastric+cancer+progression&doi=10.1007%2Fs13277-016-5020-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-Conjugating Enzyme E2T is an Independent Prognostic Factor and Promotes Gastric Cancer Progression</span></div><div class="casAuthors">Yu Hao; Xiang Pei; Pan Qi; Huang Yijiao; Xie Nanlan; Zhu Weimin</div><div class="citationInfo"><span class="NLM_cas:title">Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11723-11732</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ubiquitin-conjugating enzyme E2T (UBE2T) is a member of the E2 family that mediates the ubiquitin-proteasome system and regulates gene expression.  It is a major oncogene in several cancers such as lung cancer and breast cancer, while the potential functions of UBE2T in gastric cancer (GC) remains largely unknown.  Here, we identified the roles of UBE2T in GC progression and its potential to act as a prognostic marker of GC.  Our data demonstrated that UBE2T was significantly upregulated in gastric cancer tissues, and the high expression of UBE2T was significantly correlated with poor differentiation, high T classification, and poor prognosis.  In vitro experiments indicated that UBE2T promoted cell proliferation and inhibited cell cycle arrest.  In addition, we observed that UBE2T modulated cell mobility by inducing epithelial-mesenchymal transition.  Collectively, these findings suggest that UBE2T plays an important role in the tumorigenesis of gastric cancer and could act as a potential independent prognostic factor for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSL53q108tLqr9KldV63DuefW6udTcc2eaZD81LRJR2pbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fksVWgtg%253D%253D&md5=6460f6540875a39deb6bf05f585e3034</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs13277-016-5020-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-016-5020-3%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DXiang%26aufirst%3DP.%26aulast%3DPan%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DUbiquitin-conjugating%2520enzyme%2520E2T%2520is%2520an%2520independent%2520prognostic%2520factor%2520and%2520promotes%2520gastric%2520cancer%2520progression%26jtitle%3DTumor%2520Biol.%26date%3D2016%26volume%3D37%26spage%3D11723%26epage%3D11732%26doi%3D10.1007%2Fs13277-016-5020-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Wen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span> </span><span class="NLM_article-title">Elevated expression of UBE2T exhibits oncogenic properties in human prostate cancer</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">25226</span><span class="NLM_x">–</span> <span class="NLM_lpage">25239</span><span class="refDoi"> DOI: 10.18632/oncotarget.4712</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.18632%2Foncotarget.4712" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=25226-25239&author=M.+Wenauthor=Y.+Kwonauthor=Y.+Wangauthor=J.+H.+Maoauthor=G.+Wei&title=Elevated+expression+of+UBE2T+exhibits+oncogenic+properties+in+human+prostate+cancer&doi=10.18632%2Foncotarget.4712"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4712&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4712%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DM.%26aulast%3DKwon%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DJ.%2BH.%26aulast%3DWei%26aufirst%3DG.%26atitle%3DElevated%2520expression%2520of%2520UBE2T%2520exhibits%2520oncogenic%2520properties%2520in%2520human%2520prostate%2520cancer%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D25226%26epage%3D25239%26doi%3D10.18632%2Foncotarget.4712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Hu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.</span><span> </span><span class="NLM_article-title">UBE2T promotes nasopharyngeal carcinoma cell proliferation, invasion, and metastasis by activating the AKT/GSK3beta/beta-catenin pathway</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">15161</span><span class="NLM_x">–</span> <span class="NLM_lpage">15172</span><span class="refDoi"> DOI: 10.18632/oncotarget.7805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.18632%2Foncotarget.7805" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=15161-15172&author=W.+Huauthor=L.+Xiaoauthor=C.+Caoauthor=S.+Huaauthor=D.+Wu&title=UBE2T+promotes+nasopharyngeal+carcinoma+cell+proliferation%2C+invasion%2C+and+metastasis+by+activating+the+AKT%2FGSK3beta%2Fbeta-catenin+pathway&doi=10.18632%2Foncotarget.7805"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7805%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DD.%26atitle%3DUBE2T%2520promotes%2520nasopharyngeal%2520carcinoma%2520cell%2520proliferation%252C%2520invasion%252C%2520and%2520metastasis%2520by%2520activating%2520the%2520AKT%252FGSK3beta%252Fbeta-catenin%2520pathway%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D15161%26epage%3D15172%26doi%3D10.18632%2Foncotarget.7805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1038/nrd2220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned&doi=10.1038%2Fnrd2220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0lgdJGTK6Rwzig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D211%26epage%3D219%26doi%3D10.1038%2Fnrd2220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span><span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0lgdJGTK6Rwzig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Scott, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Fragment-based approaches in drug discovery and chemical biology</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4990</span><span class="NLM_x">–</span> <span class="NLM_lpage">5003</span><span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-based+approaches+in+drug+discovery+and+chemical+biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0liGDH-luNTWOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520in%2520drug%2520discovery%2520and%2520chemical%2520biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahnke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0liGDH-luNTWOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">From fragment to clinical candidate--a historical perspective</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">668</span><span class="NLM_x">–</span> <span class="NLM_lpage">675</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.drudis.2009.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=19427404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotFyktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=668-675&author=G.+Chessariauthor=A.+J.+Woodhead&title=From+fragment+to+clinical+candidate%2D%2Da+historical+perspective&doi=10.1016%2Fj.drudis.2009.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">From fragment to clinical candidate-a historical perspective</span></div><div class="casAuthors">Chessari, Gianni; Woodhead, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">13/14</span>),
    <span class="NLM_cas:pages">668-675</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  As recently as ten years ago few scientists had heard of fragment screening, let alone considered low mol. wt. fragments (MW <300) with weak binding affinities to be attractive start points for drug discovery programs.  Today, however, there is widespread acceptance that these fragments can be progressed into lead series and on to become clin. candidates.  Consequently, over the past three to four years, fragment-based drug discovery has become firmly established within the biotechnol. and pharmaceutical industries as a complimentary strategy to high-throughput screening.  In this review, we give a historical perspective of how rapidly fragment-based drug discovery has developed and describe a no. of clin. compds. discovered using this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIfibakR1lWLVg90H21EOLACvtfcHk0lhdVznGmwaLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotFyktLo%253D&md5=1bd47b7787b1b22681466ad220eb7cd1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DFrom%2520fragment%2520to%2520clinical%2520candidate--a%2520historical%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D668%26epage%3D675%26doi%3D10.1016%2Fj.drudis.2009.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/nrd3926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up&doi=10.1038%2Fnrd3926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0lhdVznGmwaLPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fnrd3926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saini, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Detection of secondary binding sites in proteins using fragment screening</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">15910</span><span class="NLM_x">–</span> <span class="NLM_lpage">15915</span><span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+secondary+binding+sites+in+proteins+using+fragment+screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0lhbrWwQBbbqSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520secondary%2520binding%2520sites%2520in%2520proteins%2520using%2520fragment%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Silvestre, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Integrated biophysical approach to fragment screening and validation for fragment-based lead discovery</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">12984</span><span class="NLM_x">–</span> <span class="NLM_lpage">12989</span><span class="refDoi"> DOI: 10.1073/pnas.1304045110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1073%2Fpnas.1304045110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23872845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVShurfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=12984-12989&author=H.+L.+Silvestreauthor=T.+L.+Blundellauthor=C.+Abellauthor=A.+Ciulli&title=Integrated+biophysical+approach+to+fragment+screening+and+validation+for+fragment-based+lead+discovery&doi=10.1073%2Fpnas.1304045110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated biophysical approach to fragment screening and validation for fragment-based lead discovery</span></div><div class="casAuthors">Silvestre, Hernani Leonardo; Blundell, Thomas L.; Abell, Chris; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12984-12989, S12984/1-S12984/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In fragment-based drug discovery, the weak affinities exhibited by fragments pose significant challenges for screening.  Biophys. techniques are used to address this challenge, but there is no clear consensus on which cascade of methods is best suited to identify fragment hits that ultimately translate into bound X-ray structures and provide bona fide starting points for synthesis.  We have benchmarked an integrated biophys. approach for fragment screening and validation against Mycobacterium tuberculosis pantothenate synthetase.  A primary screen of 1250 fragments library was performed by thermal shift, followed by secondary screen using one-dimensional NMR spectroscopy (water ligand obsd. gradient spectroscopy and satn. transfer difference binding expts.) and ultimate hit validation by isothermal titrn. calorimetry and X-ray crystallog.  Our multibiophys. approach identified three distinct binding sites for fragments and laid a solid foundation for successful structure-based elaboration into potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqOuAaB85047Vg90H21EOLACvtfcHk0lhbrWwQBbbqSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVShurfF&md5=7d24318292aa0f35b67a7b5e0ee1d3c2</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1304045110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1304045110%26sid%3Dliteratum%253Aachs%26aulast%3DSilvestre%26aufirst%3DH.%2BL.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIntegrated%2520biophysical%2520approach%2520to%2520fragment%2520screening%2520and%2520validation%2520for%2520fragment-based%2520lead%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D12984%26epage%3D12989%26doi%3D10.1073%2Fpnas.1304045110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Renaud, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danielson, U. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hennig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nar, H.</span><span> </span><span class="NLM_article-title">Biophysics in drug discovery: impact, challenges and opportunities</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1038/nrd.2016.123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd.2016.123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27516170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=679-698&author=J.+P.+Renaudauthor=C.+W.+Chungauthor=U.+H.+Danielsonauthor=U.+Egnerauthor=M.+Hennigauthor=R.+E.+Hubbardauthor=H.+Nar&title=Biophysics+in+drug+discovery%3A+impact%2C+challenges+and+opportunities&doi=10.1038%2Fnrd.2016.123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysics in drug discovery: impact, challenges and opportunities</span></div><div class="casAuthors">Renaud, Jean-Paul; Chung, Chun-wa; Danielson, U. Helena; Egner, Ursula; Hennig, Michael; Hubbard, Roderick E.; Nar, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">679-698</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past 25 years, biophys. technologies such as X-ray crystallog., NMR spectroscopy, surface plasmon resonance spectroscopy and isothermal titrn. calorimetry have become key components of drug discovery platforms in many pharmaceutical companies and academic labs.  There have been great improvements in the speed, sensitivity and range of possible measurements, providing high-resoln. mechanistic, kinetic, thermodn. and structural information on compd.-target interactions.  This Review provides a framework to understand this evolution by describing the key biophys. methods, the information they can provide and the ways in which they can be applied at different stages of the drug discovery process.  We also discuss the challenges for current technologies and future opportunities to use biophys. methods to solve drug discovery problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq91r76-q9prbVg90H21EOLACvtfcHk0lg0l1_MkkaP9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I&md5=1a73f14def6945e9806f57f1955f56d4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.123%26sid%3Dliteratum%253Aachs%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DHennig%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DBiophysics%2520in%2520drug%2520discovery%253A%2520impact%252C%2520challenges%2520and%2520opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D679%26epage%3D698%26doi%3D10.1038%2Fnrd.2016.123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Dai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geders, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnappinger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aldrich, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finzel, B. C.</span><span> </span><span class="NLM_article-title">Fragment-based exploration of binding site flexibility in Mycobacterium tuberculosis BioA</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5208</span><span class="NLM_x">–</span> <span class="NLM_lpage">5217</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5208-5217&author=R.+Daiauthor=T.+W.+Gedersauthor=F.+Liuauthor=S.+W.+Parkauthor=D.+Schnappingerauthor=C.+C.+Aldrichauthor=B.+C.+Finzel&title=Fragment-based+exploration+of+binding+site+flexibility+in+Mycobacterium+tuberculosis+BioA&doi=10.1021%2Facs.jmedchem.5b00092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00092%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DR.%26aulast%3DGeders%26aufirst%3DT.%2BW.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DPark%26aufirst%3DS.%2BW.%26aulast%3DSchnappinger%26aufirst%3DD.%26aulast%3DAldrich%26aufirst%3DC.%2BC.%26aulast%3DFinzel%26aufirst%3DB.%2BC.%26atitle%3DFragment-based%2520exploration%2520of%2520binding%2520site%2520flexibility%2520in%2520Mycobacterium%2520tuberculosis%2520BioA%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5208%26epage%3D5217%26doi%3D10.1021%2Facs.jmedchem.5b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogliatto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veronesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B.</span><span> </span><span class="NLM_article-title">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.1023/A:1013302231549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1023%2FA%3A1013302231549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=11824754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=349-359&author=C.+Dalvitauthor=G.+Fogliattoauthor=A.+Stewartauthor=M.+Veronesiauthor=B.+Stockman&title=WaterLOGSY+as+a+method+for+primary+NMR+screening%3A+practical+aspects+and+range+of+applicability&doi=10.1023%2FA%3A1013302231549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fogliatto, GianPaolo; Stewart, Albert; Veronesi, Marina; Stockman, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-359</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">WaterLOGSY represents a powerful method for primary NMR screening in the identification of compds. interacting with macromols., including proteins and DNA or RNA fragments.  Several relay pathways are used constructively in the expt. for transferring bulk water magnetization to the ligand.  The method is particularly useful for the identification of novel scaffolds of micromolar affinity that can be then optimized using directed screening, combinatorial chem., medicinal chem. and structure-based drug design.  The practical aspects and range of applicability of the WaterLOGSY expt. are analyzed in detail here.  Competition binding and titrn. WaterLOGSY permit, after proper correction, the evaluation of the dissocn. binding const.  The high sensitivity of the technique in combination with the easy deconvolution of the mixts. for the identification of the active components, significantly reduces the amt. of material and time needed for the NMR screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqqsXXXVX77Vg90H21EOLACvtfcHk0lg0l1_MkkaP9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D&md5=e7b431d72aa17c499c1e751956605dee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1023%2FA%3A1013302231549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1013302231549%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFogliatto%26aufirst%3DG.%26aulast%3DStewart%26aufirst%3DA.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DStockman%26aufirst%3DB.%26atitle%3DWaterLOGSY%2520as%2520a%2520method%2520for%2520primary%2520NMR%2520screening%253A%2520practical%2520aspects%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D21%26spage%3D349%26epage%3D359%26doi%3D10.1023%2FA%3A1013302231549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1784</span><span class="NLM_x">–</span> <span class="NLM_lpage">1788</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0lgiiUUFH10ojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Carr, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purcell, E. M.</span><span> </span><span class="NLM_article-title">Effects of diffusion on free precession in nuclear magnetic resonance experiments</span> <span class="citation_source-journal">Phys. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1954</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">630</span><span class="NLM_x">–</span> <span class="NLM_lpage">638</span><span class="refDoi"> DOI: 10.1103/PhysRev.94.630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1103%2FPhysRev.94.630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADyaG2cXksFKjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1954&pages=630-638&author=H.+Y.+Carrauthor=E.+M.+Purcell&title=Effects+of+diffusion+on+free+precession+in+nuclear+magnetic+resonance+experiments&doi=10.1103%2FPhysRev.94.630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of diffusion on free precession in nuclear magnetic resonance experiments</span></div><div class="casAuthors">Carr, H. Y.; Purcell, E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review</span>
        (<span class="NLM_cas:date">1954</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">630-8</span>CODEN:
                <span class="NLM_cas:coden">PHRVAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-899X</span>.
    </div><div class="casAbstract">Nuclear resonance techniques involving free precession are examd., and a convenient variation of Hahn's spin-echo method is described (ibid. 80, 580(1950)).  This variation employs a combination of pulses of different intensity or duration ("90°" and "180°" pulses).  Measurements of the transverse relaxation time T2 in fluids are often severely compromised by mol. diffusion.  Hahn's analysis of the effect of diffusion is reformulated and extended, and a new scheme for measuring T2 is described which, as predicted by the extended theory, largely circumvents the diffusion effect.  On the other hand, the free precession technique, applied in a different way, permits a direct measurement of the mol. self-diffusion const. in suitable fluids.  A measurement of the self-diffusion const. of H2O at 25° yields D = 2.5 ± 0.3 × 10-5 sq. cm./sec., in good agreement with previous detns.  The effect of convection on free precession is also analyzed.  A null method for measuring the longitudinal relaxation time T1, based on the unequal-pulse technique, is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtnBTee1_hPLVg90H21EOLACvtfcHk0lgiiUUFH10ojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2cXksFKjtA%253D%253D&md5=f729c8b5191575d29fdfd489021f05b7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1103%2FPhysRev.94.630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRev.94.630%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DH.%2BY.%26aulast%3DPurcell%26aufirst%3DE.%2BM.%26atitle%3DEffects%2520of%2520diffusion%2520on%2520free%2520precession%2520in%2520nuclear%2520magnetic%2520resonance%2520experiments%26jtitle%3DPhys.%2520Rev.%26date%3D1954%26volume%3D94%26spage%3D630%26epage%3D638%26doi%3D10.1103%2FPhysRev.94.630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Meiboom, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, D.</span><span> </span><span class="NLM_article-title">Modified spin-echo method for measuring nuclear relaxation times</span> <span class="citation_source-journal">Rev. Sci. Instrum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1958</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">688</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1063/1.1716296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1063%2F1.1716296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADyaF3cXosFWitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1958&pages=688-691&author=S.+Meiboomauthor=D.+Gill&title=Modified+spin-echo+method+for+measuring+nuclear+relaxation+times&doi=10.1063%2F1.1716296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Modified spin-echo method for measuring nuclear relaxation times</span></div><div class="casAuthors">Meiboom, S.; Gill, D.</div><div class="citationInfo"><span class="NLM_cas:title">Review of Scientific Instruments</span>
        (<span class="NLM_cas:date">1958</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">688-91</span>CODEN:
                <span class="NLM_cas:coden">RSINAK</span>;
        ISSN:<span class="NLM_cas:issn">0034-6748</span>.
    </div><div class="casAbstract">The Carr-Purcell method for the measurement of long nuclear relaxation times in liquids (cf. CA 48, 8077e) has been further modified.  While the pulse sequence of the modified technique was identical to the original scheme, the necessity of a very accurate adjustment of the 180° pulses was eliminated and the reproducibility was improved.  A schematic diagram of the transmitter gate used in the app. is discussed, and the derivation and principle of the method is explained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfTQfkWfruf7Vg90H21EOLACvtfcHk0lgiiUUFH10ojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3cXosFWitQ%253D%253D&md5=ecf0073dea1daa97157b5e2b6b62dc73</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1063%2F1.1716296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1716296%26sid%3Dliteratum%253Aachs%26aulast%3DMeiboom%26aufirst%3DS.%26aulast%3DGill%26aufirst%3DD.%26atitle%3DModified%2520spin-echo%2520method%2520for%2520measuring%2520nuclear%2520relaxation%2520times%26jtitle%3DRev.%2520Sci.%2520Instrum.%26date%3D1958%26volume%3D29%26spage%3D688%26epage%3D691%26doi%3D10.1063%2F1.1716296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Hodson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purkiss, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miles, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walden, H.</span><span> </span><span class="NLM_article-title">Structure of the human FANCL RING-Ube2T complex reveals determinants of cognate E3-E2 selection</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">344</span><span class="refDoi"> DOI: 10.1016/j.str.2013.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.str.2013.12.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=337-344&author=C.+Hodsonauthor=A.+Purkissauthor=J.+A.+Milesauthor=H.+Walden&title=Structure+of+the+human+FANCL+RING-Ube2T+complex+reveals+determinants+of+cognate+E3-E2+selection&doi=10.1016%2Fj.str.2013.12.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DHodson%26aufirst%3DC.%26aulast%3DPurkiss%26aufirst%3DA.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DWalden%26aufirst%3DH.%26atitle%3DStructure%2520of%2520the%2520human%2520FANCL%2520RING-Ube2T%2520complex%2520reveals%2520determinants%2520of%2520cognate%2520E3-E2%2520selection%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D337%26epage%3D344%26doi%3D10.1016%2Fj.str.2013.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Saalau-Bethell, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiscock, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pathuri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rich, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Discovery of an allosteric mechanism for the regulation of HCV NS3 protein function</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">920</span><span class="NLM_x">–</span> <span class="NLM_lpage">925</span><span class="refDoi"> DOI: 10.1038/nchembio.1081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnchembio.1081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23023261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=920-925&author=S.+M.+Saalau-Bethellauthor=A.+J.+Woodheadauthor=G.+Chessariauthor=M.+G.+Carrauthor=J.+Coyleauthor=B.+Grahamauthor=S.+D.+Hiscockauthor=C.+W.+Murrayauthor=P.+Pathuriauthor=S.+J.+Richauthor=C.+J.+Richardsonauthor=P.+A.+Williamsauthor=H.+Jhoti&title=Discovery+of+an+allosteric+mechanism+for+the+regulation+of+HCV+NS3+protein+function&doi=10.1038%2Fnchembio.1081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an allosteric mechanism for the regulation of HCV NS3 protein function</span></div><div class="casAuthors">Saalau-Bethell, Susanne M.; Woodhead, Andrew J.; Chessari, Gianni; Carr, Maria G.; Coyle, Joseph; Graham, Brent; Hiscock, Steven D.; Murray, Christopher W.; Pathuri, Puja; Rich, Sharna J.; Richardson, Caroline J.; Williams, Pamela A.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">920-925</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we report a highly conserved new binding site located at the interface between the protease and helicase domains of the hepatitis C virus (HCV) NS3 protein.  Using a chem. lead, identified by fragment screening and structure-guided design, we demonstrate that this site has a regulatory function on the protease activity via an allosteric mechanism.  We propose that compds. binding at this allosteric site inhibit the function of the NS3 protein by stabilizing an inactive conformation and thus represent a new class of direct-acting antiviral agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfidTQdFHIbrVg90H21EOLACvtfcHk0ljVMgVklBDD6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqt7rN&md5=d9b33f6109ae6c63935ecadf28f96345</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1081%26sid%3Dliteratum%253Aachs%26aulast%3DSaalau-Bethell%26aufirst%3DS.%2BM.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCoyle%26aufirst%3DJ.%26aulast%3DGraham%26aufirst%3DB.%26aulast%3DHiscock%26aufirst%3DS.%2BD.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DPathuri%26aufirst%3DP.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520an%2520allosteric%2520mechanism%2520for%2520the%2520regulation%2520of%2520HCV%2520NS3%2520protein%2520function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D920%26epage%3D925%26doi%3D10.1038%2Fnchembio.1081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Davies, T. G.; Jhoti, H.; Pathuri, P.; Williams, G.</span><span> </span><span class="NLM_article-title">Selecting the right targets for fragment-based drug discovery</span>. In  <span class="citation_source-book">Fragment-Based Drug Discovery Lessons and Outlook</span>, <span class="NLM_edition">1st</span> ed.; <span class="NLM_contrib-group">Erlanson, D. A.; Jahnke, W.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1002%2F9783527683604.ch02" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=T.+G.+Davies&author=H.+Jhoti&author=P.+Pathuri&author=G.+Williamsauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-Based+Drug+Discovery+Lessons+and+Outlook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2F9783527683604.ch02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527683604.ch02%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DSelecting%2520the%2520right%2520targets%2520for%2520fragment-based%2520drug%2520discovery%26btitle%3DFragment-Based%2520Drug%2520Discovery%2520Lessons%2520and%2520Outlook%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Miles, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frost, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sidhu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaugule, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alpi, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walden, H.</span><span> </span><span class="NLM_article-title">The Fanconi anemia DNA repair pathway is regulated by an interaction between ubiquitin and the E2-like fold domain of FANCL</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">20995</span><span class="NLM_x">–</span> <span class="NLM_lpage">21006</span><span class="refDoi"> DOI: 10.1074/jbc.M115.675835</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1074%2Fjbc.M115.675835" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=20995-21006&author=J.+A.+Milesauthor=M.+G.+Frostauthor=E.+Carrollauthor=M.+L.+Roweauthor=M.+J.+Howardauthor=A.+Sidhuauthor=V.+K.+Chauguleauthor=A.+F.+Alpiauthor=H.+Walden&title=The+Fanconi+anemia+DNA+repair+pathway+is+regulated+by+an+interaction+between+ubiquitin+and+the+E2-like+fold+domain+of+FANCL&doi=10.1074%2Fjbc.M115.675835"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.675835&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.675835%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DFrost%26aufirst%3DM.%2BG.%26aulast%3DCarroll%26aufirst%3DE.%26aulast%3DRowe%26aufirst%3DM.%2BL.%26aulast%3DHoward%26aufirst%3DM.%2BJ.%26aulast%3DSidhu%26aufirst%3DA.%26aulast%3DChaugule%26aufirst%3DV.%2BK.%26aulast%3DAlpi%26aufirst%3DA.%2BF.%26aulast%3DWalden%26aufirst%3DH.%26atitle%3DThe%2520Fanconi%2520anemia%2520DNA%2520repair%2520pathway%2520is%2520regulated%2520by%2520an%2520interaction%2520between%2520ubiquitin%2520and%2520the%2520E2-like%2520fold%2520domain%2520of%2520FANCL%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D20995%26epage%3D21006%26doi%3D10.1074%2Fjbc.M115.675835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca E.  Morreale</span>, <span class="hlFld-ContribAuthor ">Andrea  Testa</span>, <span class="hlFld-ContribAuthor ">Viduth K.  Chaugule</span>, <span class="hlFld-ContribAuthor ">Alessio  Bortoluzzi</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>, and <span class="hlFld-ContribAuthor ">Helen  Walden</span>  . </span><span class="cited-content_cbyCitation_article-title">Mind the Metal: A Fragment Library-Derived Zinc Impurity Binds the E2 Ubiquitin-Conjugating Enzyme Ube2T and Induces Structural Rearrangements. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (19)
                                     , 8183-8191. <a href="https://doi.org/10.1021/acs.jmedchem.7b01071" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01071</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01071%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMind%252Bthe%252BMetal%25253A%252BA%252BFragment%252BLibrary-Derived%252BZinc%252BImpurity%252BBinds%252Bthe%252BE2%252BUbiquitin-Conjugating%252BEnzyme%252BUbe2T%252Band%252BInduces%252BStructural%252BRearrangements%26aulast%3DMorreale%26aufirst%3DFrancesca%2BE.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D21072017%26date%3D03102017%26date%3D12102017%26date%3D21092017%26volume%3D60%26issue%3D19%26spage%3D8183%26epage%3D8191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael F.  Sharp</span>, <span class="hlFld-ContribAuthor ">Rohan  Bythell-Douglas</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Deans</span>, <span class="hlFld-ContribAuthor ">Wayne  Crismani</span>. </span><span class="cited-content_cbyCitation_article-title">The Fanconi anemia ubiquitin E3 ligase complex as an anti-cancer target. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2021,</strong> <em>81 </em>
                                    (11)
                                     , 2278-2289. <a href="https://doi.org/10.1016/j.molcel.2021.04.023" title="DOI URL">https://doi.org/10.1016/j.molcel.2021.04.023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2021.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2021.04.023%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DThe%252BFanconi%252Banemia%252Bubiquitin%252BE3%252Bligase%252Bcomplex%252Bas%252Ban%252Banti-cancer%252Btarget%26aulast%3DSharp%26aufirst%3DMichael%2BF.%26date%3D2021%26volume%3D81%26issue%3D11%26spage%3D2278%26epage%3D2289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qipeng  Shi</span>, <span class="hlFld-ContribAuthor ">Wei  Guo</span>, <span class="hlFld-ContribAuthor ">Qiancheng  Shen</span>, <span class="hlFld-ContribAuthor ">Jian  Han</span>, <span class="hlFld-ContribAuthor ">Lei  Lei</span>, <span class="hlFld-ContribAuthor ">Lianguo  Chen</span>, <span class="hlFld-ContribAuthor ">Lihua  Yang</span>, <span class="hlFld-ContribAuthor ">Chenglian  Feng</span>, <span class="hlFld-ContribAuthor ">Bingsheng  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">In vitro biolayer interferometry analysis of acetylcholinesterase as a potential target of aryl-organophosphorus flame-retardants. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hazardous Materials</span><span> <strong>2021,</strong> <em>409 </em>, 124999. <a href="https://doi.org/10.1016/j.jhazmat.2020.124999" title="DOI URL">https://doi.org/10.1016/j.jhazmat.2020.124999</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jhazmat.2020.124999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jhazmat.2020.124999%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hazardous%2520Materials%26atitle%3DIn%252Bvitro%252Bbiolayer%252Binterferometry%252Banalysis%252Bof%252Bacetylcholinesterase%252Bas%252Ba%252Bpotential%252Btarget%252Bof%252Baryl-organophosphorus%252Bflame-retardants%26aulast%3DShi%26aufirst%3DQipeng%26date%3D2021%26volume%3D409%26spage%3D124999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zeyuan  Yu</span>, <span class="hlFld-ContribAuthor ">Xiangyan  Jiang</span>, <span class="hlFld-ContribAuthor ">Long  Qin</span>, <span class="hlFld-ContribAuthor ">Haixiao  Deng</span>, <span class="hlFld-ContribAuthor ">Jianli  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Ren</span>, <span class="hlFld-ContribAuthor ">Hongbin  Li</span>, <span class="hlFld-ContribAuthor ">Lei  Zhao</span>, <span class="hlFld-ContribAuthor ">Huanxiang  Liu</span>, <span class="hlFld-ContribAuthor ">Hong  Yan</span>, <span class="hlFld-ContribAuthor ">Wengui  Shi</span>, <span class="hlFld-ContribAuthor ">Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Changjiang  Luo</span>, <span class="hlFld-ContribAuthor ">Bo  Long</span>, <span class="hlFld-ContribAuthor ">Huinian  Zhou</span>, <span class="hlFld-ContribAuthor ">Hui  Sun</span>, <span class="hlFld-ContribAuthor ">Zuoyi  Jiao</span>. </span><span class="cited-content_cbyCitation_article-title">A novel UBE2T inhibitor suppresses Wnt/β-catenin signaling hyperactivation and gastric cancer progression by blocking RACK1 ubiquitination. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2021,</strong> <em>40 </em>
                                    (5)
                                     , 1027-1042. <a href="https://doi.org/10.1038/s41388-020-01572-w" title="DOI URL">https://doi.org/10.1038/s41388-020-01572-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-020-01572-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-020-01572-w%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DA%252Bnovel%252BUBE2T%252Binhibitor%252Bsuppresses%252BWnt%25252F%2525CE%2525B2-catenin%252Bsignaling%252Bhyperactivation%252Band%252Bgastric%252Bcancer%252Bprogression%252Bby%252Bblocking%252BRACK1%252Bubiquitination%26aulast%3DYu%26aufirst%3DZeyuan%26date%3D2021%26date%3D2020%26volume%3D40%26issue%3D5%26spage%3D1027%26epage%3D1042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruoyao  Zou</span>, <span class="hlFld-ContribAuthor ">Haoya  Xu</span>, <span class="hlFld-ContribAuthor ">Feifei  Li</span>, <span class="hlFld-ContribAuthor ">Shengke  Wang</span>, <span class="hlFld-ContribAuthor ">Liancheng  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Increased Expression of
              UBE2T
              Predicting Poor Survival of Epithelial Ovarian Cancer: Based on Comprehensive Analysis of
              UBE2s
              , Clinical Samples, and the GEO Database. </span><span class="cited-content_cbyCitation_journal-name">DNA and Cell Biology</span><span> <strong>2021,</strong> <em>40 </em>
                                    (1)
                                     , 36-60. <a href="https://doi.org/10.1089/dna.2020.5823" title="DOI URL">https://doi.org/10.1089/dna.2020.5823</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/dna.2020.5823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fdna.2020.5823%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520and%2520Cell%2520Biology%26atitle%3DIncreased%252BExpression%252Bof%252BUBE2T%252BPredicting%252BPoor%252BSurvival%252Bof%252BEpithelial%252BOvarian%252BCancer%25253A%252BBased%252Bon%252BComprehensive%252BAnalysis%252Bof%252BUBE2s%252B%25252C%252BClinical%252BSamples%25252C%252Band%252Bthe%252BGEO%252BDatabase%26aulast%3DZou%26aufirst%3DRuoyao%26date%3D2021%26volume%3D40%26issue%3D1%26spage%3D36%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia-Xi  Mao</span>, <span class="hlFld-ContribAuthor ">Yuan-Yu  Zhao</span>, <span class="hlFld-ContribAuthor ">Jia-Yong  Dong</span>, <span class="hlFld-ContribAuthor ">Cong  Liu</span>, <span class="hlFld-ContribAuthor ">Qiang  Xue</span>, <span class="hlFld-ContribAuthor ">Guo-Shan  Ding</span>, <span class="hlFld-ContribAuthor ">Fei  Teng</span>, <span class="hlFld-ContribAuthor ">Wen-Yuan  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">UBE2T And CYP3A4: hub genes regulating the transformation of cirrhosis into hepatocellular carcinoma. </span><span class="cited-content_cbyCitation_journal-name">All Life</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     , 509-521. <a href="https://doi.org/10.1080/26895293.2021.1933208" title="DOI URL">https://doi.org/10.1080/26895293.2021.1933208</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/26895293.2021.1933208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F26895293.2021.1933208%26sid%3Dliteratum%253Aachs%26jtitle%3DAll%2520Life%26atitle%3DUBE2T%252BAnd%252BCYP3A4%25253A%252Bhub%252Bgenes%252Bregulating%252Bthe%252Btransformation%252Bof%252Bcirrhosis%252Binto%252Bhepatocellular%252Bcarcinoma%26aulast%3DMao%26aufirst%3DJia-Xi%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1%26spage%3D509%26epage%3D521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenjun  Liu</span>, <span class="hlFld-ContribAuthor ">Anna  Palovcak</span>, <span class="hlFld-ContribAuthor ">Fang  Li</span>, <span class="hlFld-ContribAuthor ">Alyan  Zafar</span>, <span class="hlFld-ContribAuthor ">Fenghua  Yuan</span>, <span class="hlFld-ContribAuthor ">Yanbin  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Fanconi anemia pathway as a prospective target for cancer intervention. </span><span class="cited-content_cbyCitation_journal-name">Cell & Bioscience</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13578-020-00401-7" title="DOI URL">https://doi.org/10.1186/s13578-020-00401-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13578-020-00401-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13578-020-00401-7%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520%2526%2520Bioscience%26atitle%3DFanconi%252Banemia%252Bpathway%252Bas%252Ba%252Bprospective%252Btarget%252Bfor%252Bcancer%252Bintervention%26aulast%3DLiu%26aufirst%3DWenjun%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael F.  Sharp</span>, <span class="hlFld-ContribAuthor ">Vince J.  Murphy</span>, <span class="hlFld-ContribAuthor ">Sylvie Van  Twest</span>, <span class="hlFld-ContribAuthor ">Winnie  Tan</span>, <span class="hlFld-ContribAuthor ">Jennii  Lui</span>, <span class="hlFld-ContribAuthor ">Kaylene J.  Simpson</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Deans</span>, <span class="hlFld-ContribAuthor ">Wayne  Crismani</span>. </span><span class="cited-content_cbyCitation_article-title">Methodology for the identification of small molecule inhibitors of the Fanconi Anaemia ubiquitin E3 ligase complex. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-64868-7" title="DOI URL">https://doi.org/10.1038/s41598-020-64868-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-64868-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-64868-7%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DMethodology%252Bfor%252Bthe%252Bidentification%252Bof%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252Bthe%252BFanconi%252BAnaemia%252Bubiquitin%252BE3%252Bligase%252Bcomplex%26aulast%3DSharp%26aufirst%3DMichael%2BF.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nibal  Betari</span>, <span class="hlFld-ContribAuthor ">Kristoffer  Sahlholm</span>, <span class="hlFld-ContribAuthor ">Yuta  Ishizuka</span>, <span class="hlFld-ContribAuthor ">Knut  Teigen</span>, <span class="hlFld-ContribAuthor ">Jan  Haavik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and biological characterization of a novel scaffold for potent inhibitors of peripheral serotonin synthesis. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (16)
                                     , 1461-1474. <a href="https://doi.org/10.4155/fmc-2020-0127" title="DOI URL">https://doi.org/10.4155/fmc-2020-0127</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0127%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252Bbiological%252Bcharacterization%252Bof%252Ba%252Bnovel%252Bscaffold%252Bfor%252Bpotent%252Binhibitors%252Bof%252Bperipheral%252Bserotonin%252Bsynthesis%26aulast%3DBetari%26aufirst%3DNibal%26date%3D2020%26volume%3D12%26issue%3D16%26spage%3D1461%26epage%3D1474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Viduth K.  Chaugule</span>, <span class="hlFld-ContribAuthor ">Connor  Arkinson</span>, <span class="hlFld-ContribAuthor ">Martin L.  Rennie</span>, <span class="hlFld-ContribAuthor ">Outi  Kämäräinen</span>, <span class="hlFld-ContribAuthor ">Rachel  Toth</span>, <span class="hlFld-ContribAuthor ">Helen  Walden</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric mechanism for site-specific ubiquitination of FANCD2. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2020,</strong> <em>16 </em>
                                    (3)
                                     , 291-301. <a href="https://doi.org/10.1038/s41589-019-0426-z" title="DOI URL">https://doi.org/10.1038/s41589-019-0426-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41589-019-0426-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41589-019-0426-z%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DAllosteric%252Bmechanism%252Bfor%252Bsite-specific%252Bubiquitination%252Bof%252BFANCD2%26aulast%3DChaugule%26aufirst%3DViduth%2BK.%26date%3D2020%26date%3D2019%26volume%3D16%26issue%3D3%26spage%3D291%26epage%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarah J  Taylor</span>, <span class="hlFld-ContribAuthor ">Mark J  Arends</span>, <span class="hlFld-ContribAuthor ">Simon P  Langdon</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of the Fanconi anaemia pathway as potential antitumour agents for ovarian cancer. </span><span class="cited-content_cbyCitation_journal-name">Exploration of Targeted Anti-tumor Therapy</span><span> <strong>2020,</strong> <em>1 </em>
                                    (1)
                                     , 26-52. <a href="https://doi.org/10.37349/etat.2020.00003" title="DOI URL">https://doi.org/10.37349/etat.2020.00003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.37349/etat.2020.00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.37349%2Fetat.2020.00003%26sid%3Dliteratum%253Aachs%26jtitle%3DExploration%2520of%2520Targeted%2520Anti-tumor%2520Therapy%26atitle%3DInhibitors%252Bof%252Bthe%252BFanconi%252Banaemia%252Bpathway%252Bas%252Bpotential%252Bantitumour%252Bagents%252Bfor%252Bovarian%252Bcancer%26aulast%3DTaylor%26aufirst%3DSarah%2BJ%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D1%26spage%3D26%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Ekinci  Elmira</span>, <span class="hlFld-ContribAuthor ">Sagar  Rohondia</span>, <span class="hlFld-ContribAuthor ">Jicang  Wang</span>, <span class="hlFld-ContribAuthor ">Jinbao  Liu</span>, <span class="hlFld-ContribAuthor ">Q. Ping  Dou</span>. </span><span class="cited-content_cbyCitation_article-title">A patent review of the ubiquitin ligase system: 2015–2018. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2018,</strong> <em>28 </em>
                                    (12)
                                     , 919-937. <a href="https://doi.org/10.1080/13543776.2018.1549229" title="DOI URL">https://doi.org/10.1080/13543776.2018.1549229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2018.1549229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2018.1549229%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DA%252Bpatent%252Breview%252Bof%252Bthe%252Bubiquitin%252Bligase%252Bsystem%25253A%252B2015%2525E2%252580%2525932018%26aulast%3DLi%26aufirst%3DXin%26date%3D2018%26date%3D2018%26volume%3D28%26issue%3D12%26spage%3D919%26epage%3D937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter J.  McHugh</span>, <span class="hlFld-ContribAuthor ">Caroline  Kisker</span>, <span class="hlFld-ContribAuthor ">Miroslav  Chovanec</span>, <span class="hlFld-ContribAuthor ">Bennett  Van Houten</span>. </span><span class="cited-content_cbyCitation_article-title">Third EU-US workshop on “Nucleotide excision repair and crosslink repair—From molecules to mankind”, Smolenice Castle, Slovak Republic, May 7th–11th 2017. </span><span class="cited-content_cbyCitation_journal-name">DNA Repair</span><span> <strong>2017,</strong> <em>58 </em>, 62-66. <a href="https://doi.org/10.1016/j.dnarep.2017.08.009" title="DOI URL">https://doi.org/10.1016/j.dnarep.2017.08.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dnarep.2017.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dnarep.2017.08.009%26sid%3Dliteratum%253Aachs%26jtitle%3DDNA%2520Repair%26atitle%3DThird%252BEU-US%252Bworkshop%252Bon%252B%2525E2%252580%25259CNucleotide%252Bexcision%252Brepair%252Band%252Bcrosslink%252Brepair%2525E2%252580%252594From%252Bmolecules%252Bto%252Bmankind%2525E2%252580%25259D%25252C%252BSmolenice%252BCastle%25252C%252BSlovak%252BRepublic%25252C%252BMay%252B7th%2525E2%252580%25259311th%252B2017%26aulast%3DMcHugh%26aufirst%3DPeter%2BJ.%26date%3D2017%26volume%3D58%26spage%3D62%26epage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical shift perturbations observed upon addition of 3 mM fragment* to 80 μM Ube2T<sub>1–154</sub>. Weighted Δδs ≥ average + SD are indicated by a yellow line. Weighted Δδs ≥ average + 2 × SD are indicated by an orange line. Equivalent colors are used to highlight corresponding residues on the Ube2T surface. As a reference, the catalytic cysteine (C86) is colored in green. Chemical structures for individual fragments are shown. For HSQC spectra and weighted Δδs formula, see <a href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf" class="ext-link">SI</a>. *Concentration accuracy is limited by solubility issues for several fragments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of EM04 in complex with Ube2T. (A) EM04-bound Ube2T structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NGZ">5NGZ</a>) showing 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured at 0.9σ level for fragment EM04. Amino acid side chains in the EM04 binding site are shown as ball and sticks and colored in orange and yellow consistent with the results of the CSP experiments, as described in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. (B) Zoom-in of the fragment’s binding pocket. Hydrogen bonds are represented as dotted yellow lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/medium/jm-2017-00147m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Fragments inhibit substrate ubiquitination by Ube2T. (A) Effect of the six fragments at 2.5 mM on Ube2T-mediated FANCD2 ubiquitination. A representative fluorescence scan shows the levels of monoubiquitinated FANCD2 using fluorescently labeled ubiquitin. The additional assays components (E1, Ube2T, FANCL) are not shown. The bar chart shows the integrated intensities of monoubiquitinated FANCD2 from three independent experiments normalized to DMSO reactions and plotted as mean ± SEM. (B) Concentration-dependent effect of fragments EM02, EM04, and EM17 in a four-point titration series. Representative fluorescence scans are shown and integrated intensities from three independent experiments are normalized to DMSO reactions and plotted as mean ± range. (C) Effect of 2.5 mM EM02, EM04, and EM17 on FANCD2 ubiquitination using Ube2T P73K mutant in the assay. A representative fluorescence scan is shown, and integrated intensities from four independent experiments are normalized to wild type Ube2T-DMSO reactions and plotted as mean ± SEM. Statistical significance was determined by one-way analysis of variance with Dunnett’s multiple-comparison test. (<i>n</i> = 3 or 4, ****<i>P</i> ≤ 0.0001, ***<i>P</i> ≤ 0.001, **<i>P</i> ≤ 0.01, *<i>P</i> ≤ 0.05, <i>ns</i><i>P</i> > 0.05).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-9/acs.jmedchem.7b00147/20170522/images/large/jm-2017-00147m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00147&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i9">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27763" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27763" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Pickart, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eddins, M. J.</span><span> </span><span class="NLM_article-title">Ubiquitin: structures, functions, mechanisms</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">1695</span><span class="NLM_x">, </span> <span class="NLM_fpage">55</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2004.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.bbamcr.2004.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=15571809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1695&publication_year=2004&pages=55-72&author=C.+M.+Pickartauthor=M.+J.+Eddins&title=Ubiquitin%3A+structures%2C+functions%2C+mechanisms&doi=10.1016%2Fj.bbamcr.2004.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin: structures, functions, mechanisms</span></div><div class="casAuthors">Pickart, Cecile M.; Eddins, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">1695</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">55-72</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Ubiquitin (Ub) is the founding member of a family of structurally conserved proteins that regulate a host of processes in eukaryotic cells.  Ub and its relatives carry out their functions through covalent attachment to other cellular proteins, thereby changing the stability, localization, or activity of the target protein.  Here, the authors review the basic biochem. of these protein conjugation reactions, focusing on Ub itself and emphasizing recent insights into mechanism and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZlvpyTnAFurVg90H21EOLACvtfcHk0lgKsajLStn87g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVantbzO&md5=05bee85acf076fed3ade4b76109d8f7d</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2004.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2004.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DPickart%26aufirst%3DC.%2BM.%26aulast%3DEddins%26aufirst%3DM.%2BJ.%26atitle%3DUbiquitin%253A%2520structures%252C%2520functions%252C%2520mechanisms%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2004%26volume%3D1695%26spage%3D55%26epage%3D72%26doi%3D10.1016%2Fj.bbamcr.2004.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Welchman, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, R. J.</span><span> </span><span class="NLM_article-title">Ubiquitin and ubiquitin-like proteins as multifunctional signals</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">599</span><span class="NLM_x">–</span> <span class="NLM_lpage">609</span><span class="refDoi"> DOI: 10.1038/nrm1700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrm1700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=16064136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=599-609&author=R.+L.+Welchmanauthor=C.+Gordonauthor=R.+J.+Mayer&title=Ubiquitin+and+ubiquitin-like+proteins+as+multifunctional+signals&doi=10.1038%2Fnrm1700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin and ubiquitin-like proteins as multifunctional signals</span></div><div class="casAuthors">Welchman, Rebecca L.; Gordon, Colin; Mayer, R. John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">599-609</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein ubiquitination is a recognized signal for protein degrdn.  However, it is increasingly realized that ubiquitin conjugation to proteins can be used for many other purposes.  Furthermore, there are many ubiquitin-like proteins that control the activities of proteins.  The central structural element of these post-translational modifications is the ubiquitin superfold.  A common ancestor based on this superfold has evolved to give various proteins that are involved in diverse activities in the cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr72A1XAR84W7Vg90H21EOLACvtfcHk0lgKsajLStn87g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmvVGhtrY%253D&md5=25f0eb2aec7ffb99b78034da9f81f3f3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrm1700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1700%26sid%3Dliteratum%253Aachs%26aulast%3DWelchman%26aufirst%3DR.%2BL.%26aulast%3DGordon%26aufirst%3DC.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26atitle%3DUbiquitin%2520and%2520ubiquitin-like%2520proteins%2520as%2520multifunctional%2520signals%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D599%26epage%3D609%26doi%3D10.1038%2Fnrm1700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Dye, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulman, B. A.</span><span> </span><span class="NLM_article-title">Structural mechanisms underlying posttranslational modification by ubiquitin-like proteins</span> <span class="citation_source-journal">Annu. Rev. Biophys. Biomol. Struct.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">131</span><span class="NLM_x">–</span> <span class="NLM_lpage">150</span><span class="refDoi"> DOI: 10.1146/annurev.biophys.36.040306.132820</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1146%2Fannurev.biophys.36.040306.132820" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2007&pages=131-150&author=B.+T.+Dyeauthor=B.+A.+Schulman&title=Structural+mechanisms+underlying+posttranslational+modification+by+ubiquitin-like+proteins&doi=10.1146%2Fannurev.biophys.36.040306.132820"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biophys.36.040306.132820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biophys.36.040306.132820%26sid%3Dliteratum%253Aachs%26aulast%3DDye%26aufirst%3DB.%2BT.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DStructural%2520mechanisms%2520underlying%2520posttranslational%2520modification%2520by%2520ubiquitin-like%2520proteins%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%2520Biomol.%2520Struct.%26date%3D2007%26volume%3D36%26spage%3D131%26epage%3D150%26doi%3D10.1146%2Fannurev.biophys.36.040306.132820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">van Wijk, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmers, H. T.</span><span> </span><span class="NLM_article-title">The family of ubiquitin-conjugating enzymes (E2s): deciding between life and death of proteins</span> <span class="citation_source-journal">FASEB J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">981</span><span class="NLM_x">–</span> <span class="NLM_lpage">993</span><span class="refDoi"> DOI: 10.1096/fj.09-136259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1096%2Ffj.09-136259" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=981-993&author=S.+J.+van+Wijkauthor=H.+T.+Timmers&title=The+family+of+ubiquitin-conjugating+enzymes+%28E2s%29%3A+deciding+between+life+and+death+of+proteins&doi=10.1096%2Ffj.09-136259"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1096%2Ffj.09-136259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.09-136259%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWijk%26aufirst%3DS.%2BJ.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%26atitle%3DThe%2520family%2520of%2520ubiquitin-conjugating%2520enzymes%2520%2528E2s%2529%253A%2520deciding%2520between%2520life%2520and%2520death%2520of%2520proteins%26jtitle%3DFASEB%2520J.%26date%3D2010%26volume%3D24%26spage%3D981%26epage%3D993%26doi%3D10.1096%2Ffj.09-136259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Stewart, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritterhoff, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klevit, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brzovic, P. S.</span><span> </span><span class="NLM_article-title">E2 enzymes: more than just middle men</span> <span class="citation_source-journal">Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">440</span><span class="refDoi"> DOI: 10.1038/cr.2016.35</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fcr.2016.35" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27002219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC28XkvVyhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=423-440&author=M.+D.+Stewartauthor=T.+Ritterhoffauthor=R.+E.+Klevitauthor=P.+S.+Brzovic&title=E2+enzymes%3A+more+than+just+middle+men&doi=10.1038%2Fcr.2016.35"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">E2 enzymes: more than just middle men</span></div><div class="casAuthors">Stewart, Mikaela D.; Ritterhoff, Tobias; Klevit, Rachel E.; Brzovic, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">423-440</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">A review.  Ubiquitin-conjugating enzymes (E2s) are the central players in the trio of enzymes responsible for the attachment of ubiquitin (Ub) to cellular proteins.  Humans have ∼40 E2s that are involved in the transfer of Ub or Ub-like (Ubl) proteins (e.g., SUMO and NEDD8).  Although the majority of E2s are only twice the size of Ub, this remarkable family of enzymes performs a variety of functional roles.  In this review, we summarize common functional and structural features that define unifying themes among E2s and highlight emerging concepts in the mechanism and regulation of E2s.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3nz4HNwXLKLVg90H21EOLACvtfcHk0lgYF7TooNofMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XkvVyhs7s%253D&md5=de200ea92123a8aeabd2ecc3d0c3cf65</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fcr.2016.35&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2016.35%26sid%3Dliteratum%253Aachs%26aulast%3DStewart%26aufirst%3DM.%2BD.%26aulast%3DRitterhoff%26aufirst%3DT.%26aulast%3DKlevit%26aufirst%3DR.%2BE.%26aulast%3DBrzovic%26aufirst%3DP.%2BS.%26atitle%3DE2%2520enzymes%253A%2520more%2520than%2520just%2520middle%2520men%26jtitle%3DCell%2520Res.%26date%3D2016%26volume%3D26%26spage%3D423%26epage%3D440%26doi%3D10.1038%2Fcr.2016.35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Sheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doherty, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Srikumar, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shloush, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avvakumov, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xue, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neculai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raught, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span> </span><span class="NLM_article-title">A human ubiquitin conjugating enzyme (E2)-HECT E3 ligase structure-function screen</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">329</span><span class="NLM_x">–</span> <span class="NLM_lpage">341</span><span class="refDoi"> DOI: 10.1074/mcp.O111.013706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1074%2Fmcp.O111.013706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=22496338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A280%3ADC%252BC38rktlOjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=329-341&author=Y.+Shengauthor=J.+H.+Hongauthor=R.+Dohertyauthor=T.+Srikumarauthor=J.+Shloushauthor=G.+V.+Avvakumovauthor=J.+R.+Walkerauthor=S.+Xueauthor=D.+Neculaiauthor=J.+W.+Wanauthor=S.+K.+Kimauthor=C.+H.+Arrowsmithauthor=B.+Raughtauthor=S.+Dhe-Paganon&title=A+human+ubiquitin+conjugating+enzyme+%28E2%29-HECT+E3+ligase+structure-function+screen&doi=10.1074%2Fmcp.O111.013706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A human ubiquitin conjugating enzyme (E2)-HECT E3 ligase structure-function screen</span></div><div class="casAuthors">Sheng Yi; Hong Jenny H; Doherty Ryan; Srikumar Tharan; Shloush Jonathan; Avvakumov George V; Walker John R; Xue Sheng; Neculai Dante; Wan Janet W; Kim Sung K; Arrowsmith Cheryl H; Raught Brian; Dhe-Paganon Sirano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & cellular proteomics : MCP</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">329-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Here we describe a systematic structure-function analysis of the human ubiquitin (Ub) E2 conjugating proteins, consisting of the determination of 15 new high-resolution three-dimensional structures of E2 catalytic domains, and autoubiquitylation assays for 26 Ub-loading E2s screened against a panel of nine different HECT (homologous to E6-AP carboxyl terminus) E3 ligase domains.  Integration of our structural and biochemical data revealed several E2 surface properties associated with Ub chain building activity; (1) net positive or neutral E2 charge, (2) an "acidic trough" located near the catalytic Cys, surrounded by an extensive basic region, and (3) similarity to the previously described HECT binding signature in UBE2L3 (UbcH7).  Mass spectrometry was used to characterize the autoubiquitylation products of a number of functional E2-HECT pairs, and demonstrated that HECT domains from different subfamilies catalyze the formation of very different types of Ub chains, largely independent of the E2 in the reaction.  Our data set represents the first comprehensive analysis of E2-HECT E3 interactions, and thus provides a framework for better understanding the molecular mechanisms of ubiquitylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsQ0fBh6uWTUNHe_gFldTbfW6udTcc2eY9wFcCjL4GM7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38rktlOjsw%253D%253D&md5=cd7b79319d270d7fbb1599557d274507</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fmcp.O111.013706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.O111.013706%26sid%3Dliteratum%253Aachs%26aulast%3DSheng%26aufirst%3DY.%26aulast%3DHong%26aufirst%3DJ.%2BH.%26aulast%3DDoherty%26aufirst%3DR.%26aulast%3DSrikumar%26aufirst%3DT.%26aulast%3DShloush%26aufirst%3DJ.%26aulast%3DAvvakumov%26aufirst%3DG.%2BV.%26aulast%3DWalker%26aufirst%3DJ.%2BR.%26aulast%3DXue%26aufirst%3DS.%26aulast%3DNeculai%26aufirst%3DD.%26aulast%3DWan%26aufirst%3DJ.%2BW.%26aulast%3DKim%26aufirst%3DS.%2BK.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DRaught%26aufirst%3DB.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26atitle%3DA%2520human%2520ubiquitin%2520conjugating%2520enzyme%2520%2528E2%2529-HECT%2520E3%2520ligase%2520structure-function%2520screen%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2012%26volume%3D11%26spage%3D329%26epage%3D341%26doi%3D10.1074%2Fmcp.O111.013706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Ceccarelli, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orlicky, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plantevin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neculai, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogunjimi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Hakim, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varelas, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koszela, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhe-Paganon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cox, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez-Girona, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercurio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wrana, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durocher, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meloche, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicheri, F.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of the human Cdc34 ubiquitin-conjugating enzyme</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">145</span><span class="NLM_x">, </span> <span class="NLM_fpage">1075</span><span class="NLM_x">–</span> <span class="NLM_lpage">1087</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.05.039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.cell.2011.05.039" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=145&publication_year=2011&pages=1075-1087&author=D.+F.+Ceccarelliauthor=X.+Tangauthor=B.+Pelletierauthor=S.+Orlickyauthor=W.+Xieauthor=V.+Plantevinauthor=D.+Neculaiauthor=Y.+C.+Chouauthor=A.+Ogunjimiauthor=A.+Al-Hakimauthor=X.+Varelasauthor=J.+Koszelaauthor=G.+A.+Wasneyauthor=M.+Vedadiauthor=S.+Dhe-Paganonauthor=S.+Coxauthor=S.+Xuauthor=A.+Lopez-Gironaauthor=F.+Mercurioauthor=J.+Wranaauthor=D.+Durocherauthor=S.+Melocheauthor=D.+R.+Webbauthor=M.+Tyersauthor=F.+Sicheri&title=An+allosteric+inhibitor+of+the+human+Cdc34+ubiquitin-conjugating+enzyme&doi=10.1016%2Fj.cell.2011.05.039"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.05.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.05.039%26sid%3Dliteratum%253Aachs%26aulast%3DCeccarelli%26aufirst%3DD.%2BF.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DPelletier%26aufirst%3DB.%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DW.%26aulast%3DPlantevin%26aufirst%3DV.%26aulast%3DNeculai%26aufirst%3DD.%26aulast%3DChou%26aufirst%3DY.%2BC.%26aulast%3DOgunjimi%26aufirst%3DA.%26aulast%3DAl-Hakim%26aufirst%3DA.%26aulast%3DVarelas%26aufirst%3DX.%26aulast%3DKoszela%26aufirst%3DJ.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DCox%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DWrana%26aufirst%3DJ.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DMeloche%26aufirst%3DS.%26aulast%3DWebb%26aufirst%3DD.%2BR.%26aulast%3DTyers%26aufirst%3DM.%26aulast%3DSicheri%26aufirst%3DF.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520the%2520human%2520Cdc34%2520ubiquitin-conjugating%2520enzyme%26jtitle%3DCell%26date%3D2011%26volume%3D145%26spage%3D1075%26epage%3D1087%26doi%3D10.1016%2Fj.cell.2011.05.039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Hewitt, W. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lountos, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zlotkowski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlhauser, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saunders, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Needle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropea, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nussinov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waugh, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneekloth, J. S.,  Jr.</span><span> </span><span class="NLM_article-title">Insights into the allosteric inhibition of the SUMO E2 enzyme Ubc9</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">5703</span><span class="NLM_x">–</span> <span class="NLM_lpage">5707</span><span class="refDoi"> DOI: 10.1002/anie.201511351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1002%2Fanie.201511351" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5703-5707&author=W.+M.+Hewittauthor=G.+T.+Lountosauthor=K.+Zlotkowskiauthor=S.+D.+Dahlhauserauthor=L.+B.+Saundersauthor=D.+Needleauthor=J.+E.+Tropeaauthor=C.+Zhanauthor=G.+Weiauthor=B.+Maauthor=R.+Nussinovauthor=D.+S.+Waughauthor=J.+S.+Schneekloth&title=Insights+into+the+allosteric+inhibition+of+the+SUMO+E2+enzyme+Ubc9&doi=10.1002%2Fanie.201511351"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fanie.201511351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201511351%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLountos%26aufirst%3DG.%2BT.%26aulast%3DZlotkowski%26aufirst%3DK.%26aulast%3DDahlhauser%26aufirst%3DS.%2BD.%26aulast%3DSaunders%26aufirst%3DL.%2BB.%26aulast%3DNeedle%26aufirst%3DD.%26aulast%3DTropea%26aufirst%3DJ.%2BE.%26aulast%3DZhan%26aufirst%3DC.%26aulast%3DWei%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DWaugh%26aufirst%3DD.%2BS.%26aulast%3DSchneekloth%26aufirst%3DJ.%2BS.%26atitle%3DInsights%2520into%2520the%2520allosteric%2520inhibition%2520of%2520the%2520SUMO%2520E2%2520enzyme%2520Ubc9%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D5703%26epage%3D5707%26doi%3D10.1002%2Fanie.201511351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Hodge, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pulvino, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spyracopoulos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendzel, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glover, J. N.</span><span> </span><span class="NLM_article-title">Covalent inhibition of Ubc13 affects ubiquitin signaling and reveals active site elements important for targeting</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1718</span><span class="NLM_x">–</span> <span class="NLM_lpage">1728</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00222</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00222" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1718-1728&author=C.+D.+Hodgeauthor=R.+A.+Edwardsauthor=C.+J.+Markinauthor=D.+McDonaldauthor=M.+Pulvinoauthor=M.+S.+Huenauthor=J.+Zhaoauthor=L.+Spyracopoulosauthor=M.+J.+Hendzelauthor=J.+N.+Glover&title=Covalent+inhibition+of+Ubc13+affects+ubiquitin+signaling+and+reveals+active+site+elements+important+for+targeting&doi=10.1021%2Facschembio.5b00222"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00222%26sid%3Dliteratum%253Aachs%26aulast%3DHodge%26aufirst%3DC.%2BD.%26aulast%3DEdwards%26aufirst%3DR.%2BA.%26aulast%3DMarkin%26aufirst%3DC.%2BJ.%26aulast%3DMcDonald%26aufirst%3DD.%26aulast%3DPulvino%26aufirst%3DM.%26aulast%3DHuen%26aufirst%3DM.%2BS.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DSpyracopoulos%26aufirst%3DL.%26aulast%3DHendzel%26aufirst%3DM.%2BJ.%26aulast%3DGlover%26aufirst%3DJ.%2BN.%26atitle%3DCovalent%2520inhibition%2520of%2520Ubc13%2520affects%2520ubiquitin%2520signaling%2520and%2520reveals%2520active%2520site%2520elements%2520important%2520for%2520targeting%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1718%26epage%3D1728%26doi%3D10.1021%2Facschembio.5b00222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Strickson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emmerich, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knebel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plater, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ritorto, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shpiro, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, P.</span><span> </span><span class="NLM_article-title">The anti-inflammatory drug BAY 11–7082 suppresses the MyD88-dependent signalling network by targeting the ubiquitin system</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">451</span><span class="NLM_x">, </span> <span class="NLM_fpage">427</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1042/BJ20121651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1042%2FBJ20121651" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=451&publication_year=2013&pages=427-437&author=S.+Stricksonauthor=D.+G.+Campbellauthor=C.+H.+Emmerichauthor=A.+Knebelauthor=L.+Platerauthor=M.+S.+Ritortoauthor=N.+Shpiroauthor=P.+Cohen&title=The+anti-inflammatory+drug+BAY+11%E2%80%937082+suppresses+the+MyD88-dependent+signalling+network+by+targeting+the+ubiquitin+system&doi=10.1042%2FBJ20121651"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1042%2FBJ20121651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121651%26sid%3Dliteratum%253Aachs%26aulast%3DStrickson%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DD.%2BG.%26aulast%3DEmmerich%26aufirst%3DC.%2BH.%26aulast%3DKnebel%26aufirst%3DA.%26aulast%3DPlater%26aufirst%3DL.%26aulast%3DRitorto%26aufirst%3DM.%2BS.%26aulast%3DShpiro%26aufirst%3DN.%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DThe%2520anti-inflammatory%2520drug%2520BAY%252011%25E2%2580%25937082%2520suppresses%2520the%2520MyD88-dependent%2520signalling%2520network%2520by%2520targeting%2520the%2520ubiquitin%2520system%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D451%26spage%3D427%26epage%3D437%26doi%3D10.1042%2FBJ20121651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Machida, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machida, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurtan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kupfer, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Andrea, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, A.</span><span> </span><span class="NLM_article-title">UBE2T is the E2 in the Fanconi anemia pathway and undergoes negative autoregulation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">596</span><span class="refDoi"> DOI: 10.1016/j.molcel.2006.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.molcel.2006.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=16916645" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=589-596&author=Y.+J.+Machidaauthor=Y.+Machidaauthor=Y.+Chenauthor=A.+M.+Gurtanauthor=G.+M.+Kupferauthor=A.+D.+D%E2%80%99Andreaauthor=A.+Dutta&title=UBE2T+is+the+E2+in+the+Fanconi+anemia+pathway+and+undergoes+negative+autoregulation&doi=10.1016%2Fj.molcel.2006.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">UBE2T is the E2 in the Fanconi anemia pathway and undergoes negative autoregulation</span></div><div class="casAuthors">Machida, Yuichi J.; Machida, Yuka; Chen, Yuefeng; Gurtan, Allan M.; Kupfer, Gary M.; D'Andrea, Alan D.; Dutta, Anindya</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">589-596</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Fanconi anemia pathway is required for the efficient repair of damaged DNA.  A key step in this pathway is the monoubiquitination of the FANCD2 protein by the ubiquitin ligase (E3) composed of Fanconi anemia core complex proteins.  Here, we show that UBE2T is the ubiquitin-conjugating enzyme (E2) essential for this pathway.  UBE2T binds to FANCL, the ubiquitin ligase subunit of the Fanconi anemia core complex, and is required for the monoubiquitination of FANCD2 in vivo.  DNA damage in UBE2T-depleted cells leads to the formation of abnormal chromosomes that are a hallmark of Fanconi anemia.  In addn., we show that UBE2T undergoes automonoubiquitination in vivo.  This monoubiquitination is stimulated by the presence of the FANCL protein and inactivates UBE2T.  Therefore, UBE2T is the E2 in the Fanconi anemia pathway and has a self-inactivation mechanism that could be important for neg. regulation of the Fanconi anemia pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8G6AIGwr7GrVg90H21EOLACvtfcHk0lgSTdGRU3GcAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xpt1Snurs%253D&md5=e891d40a0bee00d9939defb3ef05aeb6</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DMachida%26aufirst%3DY.%2BJ.%26aulast%3DMachida%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DGurtan%26aufirst%3DA.%2BM.%26aulast%3DKupfer%26aufirst%3DG.%2BM.%26aulast%3DD%25E2%2580%2599Andrea%26aufirst%3DA.%2BD.%26aulast%3DDutta%26aufirst%3DA.%26atitle%3DUBE2T%2520is%2520the%2520E2%2520in%2520the%2520Fanconi%2520anemia%2520pathway%2520and%2520undergoes%2520negative%2520autoregulation%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D589%26epage%3D596%26doi%3D10.1016%2Fj.molcel.2006.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Alpi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langevin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosedale, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machida, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dutta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, K. J.</span><span> </span><span class="NLM_article-title">UBE2T, the Fanconi anemia core complex, and FANCD2 are recruited independently to chromatin: a basis for the regulation of FANCD2 monoubiquitination</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">8421</span><span class="NLM_x">–</span> <span class="NLM_lpage">8430</span><span class="refDoi"> DOI: 10.1128/MCB.00504-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1128%2FMCB.00504-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=17938197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1eqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=8421-8430&author=A.+Alpiauthor=F.+Langevinauthor=G.+Mosedaleauthor=Y.+J.+Machidaauthor=A.+Duttaauthor=K.+J.+Patel&title=UBE2T%2C+the+Fanconi+anemia+core+complex%2C+and+FANCD2+are+recruited+independently+to+chromatin%3A+a+basis+for+the+regulation+of+FANCD2+monoubiquitination&doi=10.1128%2FMCB.00504-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">UBE2T, the Fanconi anemia core complex, and FANCD2 are recruited independently to chromatin: a basis for the regulation of FANCD2 monoubiquitination</span></div><div class="casAuthors">Alpi, Arno; Langevin, Frederic; Mosedale, Georgina; Machida, Yuichi J.; Dutta, Anindya; Patel, Ketan J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8421-8430</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The Fanconi anemia (FA) nuclear core complex and the E2 ubiquitin-conjugating enzyme UBE2T are required for the S phase and DNA damage-restricted monoubiquitination of FANCD2.  This constitutes a key step in the FA tumor suppressor pathway, and much attention has been focused on the regulation at this point.  Here, we address the importance of the assembly of the FA core complex and the subcellular localization of UBE2T in the regulation of FANCD2 monoubiquitination.  We establish three points.  First, the stable assembly of the FA core complex can be dissocd. of its ability to function as an E3 ubiquitin ligase.  Second, the actual E3 ligase activity is not detd. by the assembly of the FA core complex but rather by its DNA damage-induced localization to chromatin.  Finally, UBE2T and FANCD2 access this subcellular fraction independently of the FA core complex.  FANCD2 monoubiquitination is therefore not regulated by multiprotein complex assembly but by the formation of an active E2/E3 holoenzyme on chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAOTpFekYGXbVg90H21EOLACvtfcHk0liDtu3kkDFhIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1eqsg%253D%253D&md5=f8ad0db3046485f2a5d404a3b7356b23</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1128%2FMCB.00504-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00504-07%26sid%3Dliteratum%253Aachs%26aulast%3DAlpi%26aufirst%3DA.%26aulast%3DLangevin%26aufirst%3DF.%26aulast%3DMosedale%26aufirst%3DG.%26aulast%3DMachida%26aufirst%3DY.%2BJ.%26aulast%3DDutta%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DK.%2BJ.%26atitle%3DUBE2T%252C%2520the%2520Fanconi%2520anemia%2520core%2520complex%252C%2520and%2520FANCD2%2520are%2520recruited%2520independently%2520to%2520chromatin%253A%2520a%2520basis%2520for%2520the%2520regulation%2520of%2520FANCD2%2520monoubiquitination%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D8421%26epage%3D8430%26doi%3D10.1128%2FMCB.00504-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Alpi, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babu, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, K. J.</span><span> </span><span class="NLM_article-title">Mechanistic insight into site-restricted monoubiquitination of FANCD2 by Ube2t, FANCL, and FANCI</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">767</span><span class="NLM_x">–</span> <span class="NLM_lpage">777</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.molcel.2008.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=19111657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjtFGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=767-777&author=A.+F.+Alpiauthor=P.+E.+Paceauthor=M.+M.+Babuauthor=K.+J.+Patel&title=Mechanistic+insight+into+site-restricted+monoubiquitination+of+FANCD2+by+Ube2t%2C+FANCL%2C+and+FANCI&doi=10.1016%2Fj.molcel.2008.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanistic insight into site-restricted monoubiquitination of FANCD2 by Ube2t, FANCL, and FANCI</span></div><div class="casAuthors">Alpi, Arno F.; Pace, Paul E.; Babu, M. Madan; Patel, Ketan J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">767-777</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A key step in the Fanconi anemia (FA) tumor suppressor pathway is the site-specific monoubiquitination of the FANCD2 protein.  Genetic studies indicate that this crucial modification requires eight known FA gene products and the E2-conjugating enzyme Ube2t.  Here, the authors minimally reconstitute this monoubiquitination reaction with Ube2t and the FANCL protein, revealing that monoubiquitination is stimulated by a conserved RWD-like domain in FANCL.  Furthermore, addn. of the FANCI protein enhances monoubiquitination and also restricts it to the in vivo substrate lysine residue on FANCD2.  This work therefore establishes a system that provides mechanistic insight into the functions of FANCL and FANCI in the catalysis of FANCD2 monoubiquitination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ia-b-65807Vg90H21EOLACvtfcHk0liDtu3kkDFhIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjtFGmtQ%253D%253D&md5=562d6bfd13f96cb7ffadc4170a7097e7</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DAlpi%26aufirst%3DA.%2BF.%26aulast%3DPace%26aufirst%3DP.%2BE.%26aulast%3DBabu%26aufirst%3DM.%2BM.%26aulast%3DPatel%26aufirst%3DK.%2BJ.%26atitle%3DMechanistic%2520insight%2520into%2520site-restricted%2520monoubiquitination%2520of%2520FANCD2%2520by%2520Ube2t%252C%2520FANCL%252C%2520and%2520FANCI%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D767%26epage%3D777%26doi%3D10.1016%2Fj.molcel.2008.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Walden, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deans, A. J.</span><span> </span><span class="NLM_article-title">The Fanconi anemia DNA repair pathway: structural and functional insights into a complex disorder</span> <span class="citation_source-journal">Annu. Rev. Biophys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">278</span><span class="refDoi"> DOI: 10.1146/annurev-biophys-051013-022737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1146%2Fannurev-biophys-051013-022737" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=257-278&author=H.+Waldenauthor=A.+J.+Deans&title=The+Fanconi+anemia+DNA+repair+pathway%3A+structural+and+functional+insights+into+a+complex+disorder&doi=10.1146%2Fannurev-biophys-051013-022737"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-051013-022737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-051013-022737%26sid%3Dliteratum%253Aachs%26aulast%3DWalden%26aufirst%3DH.%26aulast%3DDeans%26aufirst%3DA.%2BJ.%26atitle%3DThe%2520Fanconi%2520anemia%2520DNA%2520repair%2520pathway%253A%2520structural%2520and%2520functional%2520insights%2520into%2520a%2520complex%2520disorder%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2014%26volume%3D43%26spage%3D257%26epage%3D278%26doi%3D10.1146%2Fannurev-biophys-051013-022737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Pearl, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schierz, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearl, F. M.</span><span> </span><span class="NLM_article-title">Therapeutic opportunities within the DNA damage response</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">166</span><span class="NLM_x">–</span> <span class="NLM_lpage">180</span><span class="refDoi"> DOI: 10.1038/nrc3891</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrc3891" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=25709118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjt1Snt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=166-180&author=L.+H.+Pearlauthor=A.+C.+Schierzauthor=S.+E.+Wardauthor=B.+Al-Lazikaniauthor=F.+M.+Pearl&title=Therapeutic+opportunities+within+the+DNA+damage+response&doi=10.1038%2Fnrc3891"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic opportunities within the DNA damage response</span></div><div class="casAuthors">Pearl, Laurence H.; Schierz, Amanda C.; Ward, Simon E.; Al-Lazikani, Bissan; Pearl, Frances M. G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-180</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The DNA damage response (DDR) is essential for maintaining the genomic integrity of the cell, and its disruption is one of the hallmarks of cancer.  Classically, defects in the DDR have been exploited therapeutically in the treatment of cancer with radiation therapies or genotoxic chemotherapies.  More recently, protein components of the DDR systems have been identified as promising avenues for targeted cancer therapeutics.  Here, we present an in-depth anal. of the function, role in cancer and therapeutic potential of 450 expert-curated human DDR genes.  We discuss the DDR drugs that have been approved by the US Food and Drug Administration (FDA) or that are under clin. investigation.  We examine large-scale genomic and expression data for 15 cancers to identify deregulated components of the DDR, and we apply systematic computational anal. to identify DDR proteins that are amenable to modulation by small mols., highlighting potential novel therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosptr9Y-2e6LVg90H21EOLACvtfcHk0lhJYtQlRcFY9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjt1Snt74%253D&md5=47ceba5c42cc7ebb59e18f3aa53e0f60</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc3891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3891%26sid%3Dliteratum%253Aachs%26aulast%3DPearl%26aufirst%3DL.%2BH.%26aulast%3DSchierz%26aufirst%3DA.%2BC.%26aulast%3DWard%26aufirst%3DS.%2BE.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DPearl%26aufirst%3DF.%2BM.%26atitle%3DTherapeutic%2520opportunities%2520within%2520the%2520DNA%2520damage%2520response%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D166%26epage%3D180%26doi%3D10.1038%2Fnrc3891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Curtin, N. J.</span><span> </span><span class="NLM_article-title">DNA repair dysregulation from cancer driver to therapeutic target</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">801</span><span class="NLM_x">–</span> <span class="NLM_lpage">817</span><span class="refDoi"> DOI: 10.1038/nrc3399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrc3399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23175119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=801-817&author=N.+J.+Curtin&title=DNA+repair+dysregulation+from+cancer+driver+to+therapeutic+target&doi=10.1038%2Fnrc3399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair dysregulation from cancer driver to therapeutic target</span></div><div class="casAuthors">Curtin, Nicola J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">801-817</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dysregulation of DNA damage repair and signalling to cell cycle checkpoints, known as the DNA damage response (DDR), is assocd. with a predisposition to cancer and affects responses to DNA-damaging anticancer therapy.  Dysfunction of one DNA repair pathway may be compensated for by the function of another compensatory DDR pathway, which may be increased and contribute to resistance to DNA-damaging chemotherapy and radiotherapy.  Therefore, DDR pathways make an ideal target for therapeutic intervention; first, to prevent or reverse therapy resistance; and second, using a synthetic lethal approach to specifically kill cancer cells that are dependent on a compensatory DNA repair pathway for survival in the context of cancer-assocd. oxidative and replicative stress.  These hypotheses are currently being tested in the lab. and are being translated into clin. studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNDBV6krLGBbVg90H21EOLACvtfcHk0lhJYtQlRcFY9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslagsb3N&md5=1bcb90d4742e46696c8a30a4fce2400c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc3399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3399%26sid%3Dliteratum%253Aachs%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DDNA%2520repair%2520dysregulation%2520from%2520cancer%2520driver%2520to%2520therapeutic%2520target%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D801%26epage%3D817%26doi%3D10.1038%2Fnrc3399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Helleday, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petermann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lundin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodgson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, R. A.</span><span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">204</span><span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0lhJYtQlRcFY9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Ueki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishidate, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kijima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirata, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katagiri, T.</span><span> </span><span class="NLM_article-title">Ubiquitination and downregulation of BRCA1 by ubiquitin-conjugating enzyme E2T overexpression in human breast cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">8752</span><span class="NLM_x">–</span> <span class="NLM_lpage">8760</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-09-1809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1158%2F0008-5472.CAN-09-1809" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=8752-8760&author=T.+Uekiauthor=J.+H.+Parkauthor=T.+Nishidateauthor=K.+Kijimaauthor=K.+Hirataauthor=Y.+Nakamuraauthor=T.+Katagiri&title=Ubiquitination+and+downregulation+of+BRCA1+by+ubiquitin-conjugating+enzyme+E2T+overexpression+in+human+breast+cancer+cells&doi=10.1158%2F0008-5472.CAN-09-1809"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-09-1809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-09-1809%26sid%3Dliteratum%253Aachs%26aulast%3DUeki%26aufirst%3DT.%26aulast%3DPark%26aufirst%3DJ.%2BH.%26aulast%3DNishidate%26aufirst%3DT.%26aulast%3DKijima%26aufirst%3DK.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DKatagiri%26aufirst%3DT.%26atitle%3DUbiquitination%2520and%2520downregulation%2520of%2520BRCA1%2520by%2520ubiquitin-conjugating%2520enzyme%2520E2T%2520overexpression%2520in%2520human%2520breast%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D8752%26epage%3D8760%26doi%3D10.1158%2F0008-5472.CAN-09-1809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.</span><span> </span><span class="NLM_article-title">Ubiquitin-conjugating enzyme E2T is an independent prognostic factor and promotes gastric cancer progression</span> <span class="citation_source-journal">Tumor Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">11723</span><span class="NLM_x">–</span> <span class="NLM_lpage">11732</span><span class="refDoi"> DOI: 10.1007/s13277-016-5020-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1007%2Fs13277-016-5020-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27020591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A280%3ADC%252BC28fksVWgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=11723-11732&author=H.+Yuauthor=P.+Xiangauthor=Q.+Panauthor=Y.+Huangauthor=N.+Xieauthor=W.+Zhu&title=Ubiquitin-conjugating+enzyme+E2T+is+an+independent+prognostic+factor+and+promotes+gastric+cancer+progression&doi=10.1007%2Fs13277-016-5020-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin-Conjugating Enzyme E2T is an Independent Prognostic Factor and Promotes Gastric Cancer Progression</span></div><div class="casAuthors">Yu Hao; Xiang Pei; Pan Qi; Huang Yijiao; Xie Nanlan; Zhu Weimin</div><div class="citationInfo"><span class="NLM_cas:title">Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11723-11732</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ubiquitin-conjugating enzyme E2T (UBE2T) is a member of the E2 family that mediates the ubiquitin-proteasome system and regulates gene expression.  It is a major oncogene in several cancers such as lung cancer and breast cancer, while the potential functions of UBE2T in gastric cancer (GC) remains largely unknown.  Here, we identified the roles of UBE2T in GC progression and its potential to act as a prognostic marker of GC.  Our data demonstrated that UBE2T was significantly upregulated in gastric cancer tissues, and the high expression of UBE2T was significantly correlated with poor differentiation, high T classification, and poor prognosis.  In vitro experiments indicated that UBE2T promoted cell proliferation and inhibited cell cycle arrest.  In addition, we observed that UBE2T modulated cell mobility by inducing epithelial-mesenchymal transition.  Collectively, these findings suggest that UBE2T plays an important role in the tumorigenesis of gastric cancer and could act as a potential independent prognostic factor for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSL53q108tLqr9KldV63DuefW6udTcc2eanURctRe-LYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28fksVWgtg%253D%253D&md5=6460f6540875a39deb6bf05f585e3034</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs13277-016-5020-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-016-5020-3%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%26aulast%3DXiang%26aufirst%3DP.%26aulast%3DPan%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DUbiquitin-conjugating%2520enzyme%2520E2T%2520is%2520an%2520independent%2520prognostic%2520factor%2520and%2520promotes%2520gastric%2520cancer%2520progression%26jtitle%3DTumor%2520Biol.%26date%3D2016%26volume%3D37%26spage%3D11723%26epage%3D11732%26doi%3D10.1007%2Fs13277-016-5020-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Wen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, G.</span><span> </span><span class="NLM_article-title">Elevated expression of UBE2T exhibits oncogenic properties in human prostate cancer</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">25226</span><span class="NLM_x">–</span> <span class="NLM_lpage">25239</span><span class="refDoi"> DOI: 10.18632/oncotarget.4712</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.18632%2Foncotarget.4712" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=25226-25239&author=M.+Wenauthor=Y.+Kwonauthor=Y.+Wangauthor=J.+H.+Maoauthor=G.+Wei&title=Elevated+expression+of+UBE2T+exhibits+oncogenic+properties+in+human+prostate+cancer&doi=10.18632%2Foncotarget.4712"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4712&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4712%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DM.%26aulast%3DKwon%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DMao%26aufirst%3DJ.%2BH.%26aulast%3DWei%26aufirst%3DG.%26atitle%3DElevated%2520expression%2520of%2520UBE2T%2520exhibits%2520oncogenic%2520properties%2520in%2520human%2520prostate%2520cancer%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D25226%26epage%3D25239%26doi%3D10.18632%2Foncotarget.4712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Hu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.</span><span> </span><span class="NLM_article-title">UBE2T promotes nasopharyngeal carcinoma cell proliferation, invasion, and metastasis by activating the AKT/GSK3beta/beta-catenin pathway</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">15161</span><span class="NLM_x">–</span> <span class="NLM_lpage">15172</span><span class="refDoi"> DOI: 10.18632/oncotarget.7805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.18632%2Foncotarget.7805" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=15161-15172&author=W.+Huauthor=L.+Xiaoauthor=C.+Caoauthor=S.+Huaauthor=D.+Wu&title=UBE2T+promotes+nasopharyngeal+carcinoma+cell+proliferation%2C+invasion%2C+and+metastasis+by+activating+the+AKT%2FGSK3beta%2Fbeta-catenin+pathway&doi=10.18632%2Foncotarget.7805"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.7805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.7805%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DC.%26aulast%3DHua%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DD.%26atitle%3DUBE2T%2520promotes%2520nasopharyngeal%2520carcinoma%2520cell%2520proliferation%252C%2520invasion%252C%2520and%2520metastasis%2520by%2520activating%2520the%2520AKT%252FGSK3beta%252Fbeta-catenin%2520pathway%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D15161%26epage%3D15172%26doi%3D10.18632%2Foncotarget.7805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span><span class="refDoi"> DOI: 10.1038/nrd2220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned&doi=10.1038%2Fnrd2220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0liGJZjbmd0Ypw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D211%26epage%3D219%26doi%3D10.1038%2Fnrd2220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span> </span><span class="NLM_article-title">The rise of fragment-based drug discovery</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">192</span><span class="refDoi"> DOI: 10.1038/nchem.217</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnchem.217" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=21378847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=187-192&author=C.+W.+Murrayauthor=D.+C.+Rees&title=The+rise+of+fragment-based+drug+discovery&doi=10.1038%2Fnchem.217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The rise of fragment-based drug discovery</span></div><div class="casAuthors">Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The search for new drugs is plagued by high attrition rates at all stages in research and development.  Chemists have an opportunity to tackle this problem because attrition can be traced back, in part, to the quality of the chem. leads.  Fragment-based drug discovery (FBDD) is a new approach, increasingly used in the pharmaceutical industry, for reducing attrition and providing leads for previously intractable biol. targets.  FBDD identifies low-mol.-wt. ligands (∼150 Da) that bind to biol. important macromols.  The three-dimensional exptl. binding mode of these fragments is detd. using X-ray crystallog. or NMR spectroscopy, and is used to facilitate their optimization into potent mols. with drug-like properties.  Compared with high-throughput-screening, the fragment approach requires fewer compds. to be screened, and, despite the lower initial potency of the screening hits, offers more efficient and fruitful optimization campaigns.  Here, we review the rise of FBDD, including its application to discovering clin. candidates against targets for which other chem. approaches have struggled.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg3ip2FKcYJbVg90H21EOLACvtfcHk0liGJZjbmd0Ypw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtF2rs78%253D&md5=0bd1a40fd927cce9200f0fa296a7462f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchem.217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.217%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DThe%2520rise%2520of%2520fragment-based%2520drug%2520discovery%26jtitle%3DNat.%2520Chem.%26date%3D2009%26volume%3D1%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnchem.217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Scott, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span> </span><span class="NLM_article-title">Fragment-based approaches in drug discovery and chemical biology</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">4990</span><span class="NLM_x">–</span> <span class="NLM_lpage">5003</span><span class="refDoi"> DOI: 10.1021/bi3005126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3005126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=4990-5003&author=D.+E.+Scottauthor=A.+G.+Coyneauthor=S.+A.+Hudsonauthor=C.+Abell&title=Fragment-based+approaches+in+drug+discovery+and+chemical+biology&doi=10.1021%2Fbi3005126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Approaches in Drug Discovery and Chemical Biology</span></div><div class="casAuthors">Scott, Duncan E.; Coyne, Anthony G.; Hudson, Sean A.; Abell, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4990-5003</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Fragment-based approaches to finding novel small mols. that bind to proteins are now firmly established in drug discovery and chem. biol.  Initially developed primarily in a few centers in the biotech and pharma industry, this methodol. has now been adopted widely in both the pharmaceutical industry and academia.  After the initial success with kinase targets, the versatility of this approach has now expanded to a broad range of different protein classes.  Herein we describe recent fragment-based approaches to a wide range of target types, including Hsp90, β-secretase, and allosteric sites in human immunodeficiency virus protease and fanesyl pyrophosphate synthase.  The role of fragment-based approaches in an academic research environment is also examd. with an emphasis on neglected diseases such as tuberculosis.  The development of a fragment library, the fragment screening process, and the subsequent fragment hit elaboration will be discussed using examples from the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE2ry6mWC83bVg90H21EOLACvtfcHk0lhEk-mzpu7B5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XosV2qsLY%253D&md5=39a6f0f741e711fd572e24c95142f55d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fbi3005126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3005126%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DCoyne%26aufirst%3DA.%2BG.%26aulast%3DHudson%26aufirst%3DS.%2BA.%26aulast%3DAbell%26aufirst%3DC.%26atitle%3DFragment-based%2520approaches%2520in%2520drug%2520discovery%2520and%2520chemical%2520biology%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D4990%26epage%3D5003%26doi%3D10.1021%2Fbi3005126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jahnke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Twenty years on: the impact of fragments on drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">619</span><span class="refDoi"> DOI: 10.1038/nrd.2016.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd.2016.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27417849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=605-619&author=D.+A.+Erlansonauthor=S.+W.+Fesikauthor=R.+E.+Hubbardauthor=W.+Jahnkeauthor=H.+Jhoti&title=Twenty+years+on%3A+the+impact+of+fragments+on+drug+discovery&doi=10.1038%2Fnrd.2016.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty years on: the impact of fragments on drug discovery</span></div><div class="casAuthors">Erlanson, Daniel A.; Fesik, Stephen W.; Hubbard, Roderick E.; Jahnke, Wolfgang; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">605-619</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">After 20 years of sometimes quiet growth, fragment-based drug discovery (FBDD) has become mainstream.  More than 30 drug candidates derived from fragments have entered the clinic, with two approved and several more in advanced trials.  FBDD has been widely applied in both academia and industry, as evidenced by the large no. of papers from universities, non-profit research institutions, biotechnol. companies and pharmaceutical companies.  Moreover, FBDD draws on a diverse range of disciplines, from biochem. and biophysics to computational and medicinal chem.  As the promise of FBDD strategies becomes increasingly realized, now is an opportune time to draw lessons and point the way to the future.  This Review briefly discusses how to design fragment libraries, how to select screening techniques and how to make the most of information gleaned from them.  It also shows how concepts from FBDD have permeated and enhanced drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrXQhFttydkrVg90H21EOLACvtfcHk0lhEk-mzpu7B5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Gisb%252FE&md5=fc2bcf0da6b7b1083a5ff263057cfa80</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DTwenty%2520years%2520on%253A%2520the%2520impact%2520of%2520fragments%2520on%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D605%26epage%3D619%26doi%3D10.1038%2Fnrd.2016.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">From fragment to clinical candidate--a historical perspective</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">668</span><span class="NLM_x">–</span> <span class="NLM_lpage">675</span><span class="refDoi"> DOI: 10.1016/j.drudis.2009.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.drudis.2009.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=19427404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotFyktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=668-675&author=G.+Chessariauthor=A.+J.+Woodhead&title=From+fragment+to+clinical+candidate%2D%2Da+historical+perspective&doi=10.1016%2Fj.drudis.2009.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">From fragment to clinical candidate-a historical perspective</span></div><div class="casAuthors">Chessari, Gianni; Woodhead, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">13/14</span>),
    <span class="NLM_cas:pages">668-675</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  As recently as ten years ago few scientists had heard of fragment screening, let alone considered low mol. wt. fragments (MW <300) with weak binding affinities to be attractive start points for drug discovery programs.  Today, however, there is widespread acceptance that these fragments can be progressed into lead series and on to become clin. candidates.  Consequently, over the past three to four years, fragment-based drug discovery has become firmly established within the biotechnol. and pharmaceutical industries as a complimentary strategy to high-throughput screening.  In this review, we give a historical perspective of how rapidly fragment-based drug discovery has developed and describe a no. of clin. compds. discovered using this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIfibakR1lWLVg90H21EOLACvtfcHk0lhEk-mzpu7B5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotFyktLo%253D&md5=1bd47b7787b1b22681466ad220eb7cd1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2009.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2009.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DFrom%2520fragment%2520to%2520clinical%2520candidate--a%2520historical%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D668%26epage%3D675%26doi%3D10.1016%2Fj.drudis.2009.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Baker, M.</span><span> </span><span class="NLM_article-title">Fragment-based lead discovery grows up</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1038/nrd3926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd3926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23274457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFOi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=5-7&author=M.+Baker&title=Fragment-based+lead+discovery+grows+up&doi=10.1038%2Fnrd3926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery grows up</span></div><div class="casAuthors">Baker, Monya</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-7</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With multiple drug candidates in the clinic that originated from fragment-based lead discovery, the approach of starting small has become big.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxKfzq4txN2bVg90H21EOLACvtfcHk0lgF1EsEultaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFOi&md5=854c670bd7388d04f76867d5818a5191</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd3926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3926%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DM.%26atitle%3DFragment-based%2520lead%2520discovery%2520grows%2520up%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D5%26epage%3D7%26doi%3D10.1038%2Fnrd3926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Ludlow, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saini, H. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tickle, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Detection of secondary binding sites in proteins using fragment screening</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">15910</span><span class="NLM_x">–</span> <span class="NLM_lpage">15915</span><span class="refDoi"> DOI: 10.1073/pnas.1518946112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1073%2Fpnas.1518946112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=26655740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=15910-15915&author=R.+F.+Ludlowauthor=M.+L.+Verdonkauthor=H.+K.+Sainiauthor=I.+J.+Tickleauthor=H.+Jhoti&title=Detection+of+secondary+binding+sites+in+proteins+using+fragment+screening&doi=10.1073%2Fpnas.1518946112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Detection of secondary binding sites in proteins using fragment screening</span></div><div class="casAuthors">Ludlow, R. Frederick; Verdonk, Marcel L.; Saini, Harpreet K.; Tickle, Ian J.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">15910-15915</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Proteins need to be tightly regulated as they control biol. processes in most normal cellular functions.  The precise mechanisms of regulation are rarely completely understood but can involve binding of endogenous ligands and/or partner proteins at specific locations on a protein that can modulate function.  Often, these addnl. secondary binding sites appear sep. to the primary binding site, which, e.g., for an enzyme may bind a substrate.  In previous work, the authors uncovered several examples in which secondary binding sites were discovered on proteins using fragment screening approaches.  In each case, the authors were able to establish that the newly identified secondary binding site was biol. relevant as it was able to modulate function by the binding of a small mol.  Here, the authors investigated how often secondary binding sites are located on proteins by analyzing 24 protein targets for which they performed a fragment screen using x-ray crystallog.  The anal. showed that, surprisingly, the majority of proteins contained secondary binding sites based on their ability to bind fragments.  Furthermore, sequence anal. of these previously unknown sites indicated high conservation, which suggested that they may have a biol. function, perhaps via an allosteric mechanism.  Comparing the physicochem. properties of the secondary sites with known primary ligand-binding sites also showed broad similarities indicating that many of the secondary sites may be druggable in nature with small mols. that could provide new opportunities to modulate potential therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ITF1w230fbVg90H21EOLACvtfcHk0lgF1EsEultaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvF2kt7zP&md5=9e5f15595263dc049c713d5a3f5912a1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1518946112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1518946112%26sid%3Dliteratum%253Aachs%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DSaini%26aufirst%3DH.%2BK.%26aulast%3DTickle%26aufirst%3DI.%2BJ.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDetection%2520of%2520secondary%2520binding%2520sites%2520in%2520proteins%2520using%2520fragment%2520screening%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D15910%26epage%3D15915%26doi%3D10.1073%2Fpnas.1518946112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Silvestre, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciulli, A.</span><span> </span><span class="NLM_article-title">Integrated biophysical approach to fragment screening and validation for fragment-based lead discovery</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">12984</span><span class="NLM_x">–</span> <span class="NLM_lpage">12989</span><span class="refDoi"> DOI: 10.1073/pnas.1304045110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1073%2Fpnas.1304045110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23872845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVShurfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=12984-12989&author=H.+L.+Silvestreauthor=T.+L.+Blundellauthor=C.+Abellauthor=A.+Ciulli&title=Integrated+biophysical+approach+to+fragment+screening+and+validation+for+fragment-based+lead+discovery&doi=10.1073%2Fpnas.1304045110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated biophysical approach to fragment screening and validation for fragment-based lead discovery</span></div><div class="casAuthors">Silvestre, Hernani Leonardo; Blundell, Thomas L.; Abell, Chris; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">12984-12989, S12984/1-S12984/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In fragment-based drug discovery, the weak affinities exhibited by fragments pose significant challenges for screening.  Biophys. techniques are used to address this challenge, but there is no clear consensus on which cascade of methods is best suited to identify fragment hits that ultimately translate into bound X-ray structures and provide bona fide starting points for synthesis.  We have benchmarked an integrated biophys. approach for fragment screening and validation against Mycobacterium tuberculosis pantothenate synthetase.  A primary screen of 1250 fragments library was performed by thermal shift, followed by secondary screen using one-dimensional NMR spectroscopy (water ligand obsd. gradient spectroscopy and satn. transfer difference binding expts.) and ultimate hit validation by isothermal titrn. calorimetry and X-ray crystallog.  Our multibiophys. approach identified three distinct binding sites for fragments and laid a solid foundation for successful structure-based elaboration into potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqOuAaB85047Vg90H21EOLACvtfcHk0lgF1EsEultaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVShurfF&md5=7d24318292aa0f35b67a7b5e0ee1d3c2</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1304045110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1304045110%26sid%3Dliteratum%253Aachs%26aulast%3DSilvestre%26aufirst%3DH.%2BL.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIntegrated%2520biophysical%2520approach%2520to%2520fragment%2520screening%2520and%2520validation%2520for%2520fragment-based%2520lead%2520discovery%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D12984%26epage%3D12989%26doi%3D10.1073%2Fpnas.1304045110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Renaud, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danielson, U. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egner, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hennig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nar, H.</span><span> </span><span class="NLM_article-title">Biophysics in drug discovery: impact, challenges and opportunities</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">679</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span><span class="refDoi"> DOI: 10.1038/nrd.2016.123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnrd.2016.123" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=27516170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=679-698&author=J.+P.+Renaudauthor=C.+W.+Chungauthor=U.+H.+Danielsonauthor=U.+Egnerauthor=M.+Hennigauthor=R.+E.+Hubbardauthor=H.+Nar&title=Biophysics+in+drug+discovery%3A+impact%2C+challenges+and+opportunities&doi=10.1038%2Fnrd.2016.123"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysics in drug discovery: impact, challenges and opportunities</span></div><div class="casAuthors">Renaud, Jean-Paul; Chung, Chun-wa; Danielson, U. Helena; Egner, Ursula; Hennig, Michael; Hubbard, Roderick E.; Nar, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">679-698</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Over the past 25 years, biophys. technologies such as X-ray crystallog., NMR spectroscopy, surface plasmon resonance spectroscopy and isothermal titrn. calorimetry have become key components of drug discovery platforms in many pharmaceutical companies and academic labs.  There have been great improvements in the speed, sensitivity and range of possible measurements, providing high-resoln. mechanistic, kinetic, thermodn. and structural information on compd.-target interactions.  This Review provides a framework to understand this evolution by describing the key biophys. methods, the information they can provide and the ways in which they can be applied at different stages of the drug discovery process.  We also discuss the challenges for current technologies and future opportunities to use biophys. methods to solve drug discovery problems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq91r76-q9prbVg90H21EOLACvtfcHk0lgF1EsEultaLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7I&md5=1a73f14def6945e9806f57f1955f56d4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.123%26sid%3Dliteratum%253Aachs%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDanielson%26aufirst%3DU.%2BH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DHennig%26aufirst%3DM.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DBiophysics%2520in%2520drug%2520discovery%253A%2520impact%252C%2520challenges%2520and%2520opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D679%26epage%3D698%26doi%3D10.1038%2Fnrd.2016.123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Dai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geders, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnappinger, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aldrich, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finzel, B. C.</span><span> </span><span class="NLM_article-title">Fragment-based exploration of binding site flexibility in Mycobacterium tuberculosis BioA</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">5208</span><span class="NLM_x">–</span> <span class="NLM_lpage">5217</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5208-5217&author=R.+Daiauthor=T.+W.+Gedersauthor=F.+Liuauthor=S.+W.+Parkauthor=D.+Schnappingerauthor=C.+C.+Aldrichauthor=B.+C.+Finzel&title=Fragment-based+exploration+of+binding+site+flexibility+in+Mycobacterium+tuberculosis+BioA&doi=10.1021%2Facs.jmedchem.5b00092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00092%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DR.%26aulast%3DGeders%26aufirst%3DT.%2BW.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DPark%26aufirst%3DS.%2BW.%26aulast%3DSchnappinger%26aufirst%3DD.%26aulast%3DAldrich%26aufirst%3DC.%2BC.%26aulast%3DFinzel%26aufirst%3DB.%2BC.%26atitle%3DFragment-based%2520exploration%2520of%2520binding%2520site%2520flexibility%2520in%2520Mycobacterium%2520tuberculosis%2520BioA%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D5208%26epage%3D5217%26doi%3D10.1021%2Facs.jmedchem.5b00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Dalvit, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogliatto, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veronesi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockman, B.</span><span> </span><span class="NLM_article-title">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">349</span><span class="NLM_x">–</span> <span class="NLM_lpage">359</span><span class="refDoi"> DOI: 10.1023/A:1013302231549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1023%2FA%3A1013302231549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=11824754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=349-359&author=C.+Dalvitauthor=G.+Fogliattoauthor=A.+Stewartauthor=M.+Veronesiauthor=B.+Stockman&title=WaterLOGSY+as+a+method+for+primary+NMR+screening%3A+practical+aspects+and+range+of+applicability&doi=10.1023%2FA%3A1013302231549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">WaterLOGSY as a method for primary NMR screening: practical aspects and range of applicability</span></div><div class="casAuthors">Dalvit, Claudio; Fogliatto, GianPaolo; Stewart, Albert; Veronesi, Marina; Stockman, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-359</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">WaterLOGSY represents a powerful method for primary NMR screening in the identification of compds. interacting with macromols., including proteins and DNA or RNA fragments.  Several relay pathways are used constructively in the expt. for transferring bulk water magnetization to the ligand.  The method is particularly useful for the identification of novel scaffolds of micromolar affinity that can be then optimized using directed screening, combinatorial chem., medicinal chem. and structure-based drug design.  The practical aspects and range of applicability of the WaterLOGSY expt. are analyzed in detail here.  Competition binding and titrn. WaterLOGSY permit, after proper correction, the evaluation of the dissocn. binding const.  The high sensitivity of the technique in combination with the easy deconvolution of the mixts. for the identification of the active components, significantly reduces the amt. of material and time needed for the NMR screening process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqqsXXXVX77Vg90H21EOLACvtfcHk0ljzW37rdlpSAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptVyksw%253D%253D&md5=e7b431d72aa17c499c1e751956605dee</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1023%2FA%3A1013302231549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1013302231549%26sid%3Dliteratum%253Aachs%26aulast%3DDalvit%26aufirst%3DC.%26aulast%3DFogliatto%26aufirst%3DG.%26aulast%3DStewart%26aufirst%3DA.%26aulast%3DVeronesi%26aufirst%3DM.%26aulast%3DStockman%26aufirst%3DB.%26atitle%3DWaterLOGSY%2520as%2520a%2520method%2520for%2520primary%2520NMR%2520screening%253A%2520practical%2520aspects%2520and%2520range%2520of%2520applicability%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D2001%26volume%3D21%26spage%3D349%26epage%3D359%26doi%3D10.1023%2FA%3A1013302231549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Mayer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, B.</span><span> </span><span class="NLM_article-title">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1784</span><span class="NLM_x">–</span> <span class="NLM_lpage">1788</span><span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+ligand+binding+by+saturation+transfer+difference+NMR+spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0ljzW37rdlpSAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520ligand%2520binding%2520by%2520saturation%2520transfer%2520difference%2520NMR%2520spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Carr, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purcell, E. M.</span><span> </span><span class="NLM_article-title">Effects of diffusion on free precession in nuclear magnetic resonance experiments</span> <span class="citation_source-journal">Phys. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1954</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">630</span><span class="NLM_x">–</span> <span class="NLM_lpage">638</span><span class="refDoi"> DOI: 10.1103/PhysRev.94.630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1103%2FPhysRev.94.630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADyaG2cXksFKjtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1954&pages=630-638&author=H.+Y.+Carrauthor=E.+M.+Purcell&title=Effects+of+diffusion+on+free+precession+in+nuclear+magnetic+resonance+experiments&doi=10.1103%2FPhysRev.94.630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of diffusion on free precession in nuclear magnetic resonance experiments</span></div><div class="casAuthors">Carr, H. Y.; Purcell, E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review</span>
        (<span class="NLM_cas:date">1954</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">630-8</span>CODEN:
                <span class="NLM_cas:coden">PHRVAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-899X</span>.
    </div><div class="casAbstract">Nuclear resonance techniques involving free precession are examd., and a convenient variation of Hahn's spin-echo method is described (ibid. 80, 580(1950)).  This variation employs a combination of pulses of different intensity or duration ("90°" and "180°" pulses).  Measurements of the transverse relaxation time T2 in fluids are often severely compromised by mol. diffusion.  Hahn's analysis of the effect of diffusion is reformulated and extended, and a new scheme for measuring T2 is described which, as predicted by the extended theory, largely circumvents the diffusion effect.  On the other hand, the free precession technique, applied in a different way, permits a direct measurement of the mol. self-diffusion const. in suitable fluids.  A measurement of the self-diffusion const. of H2O at 25° yields D = 2.5 ± 0.3 × 10-5 sq. cm./sec., in good agreement with previous detns.  The effect of convection on free precession is also analyzed.  A null method for measuring the longitudinal relaxation time T1, based on the unequal-pulse technique, is described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtnBTee1_hPLVg90H21EOLACvtfcHk0lir2eyPnUBn_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2cXksFKjtA%253D%253D&md5=f729c8b5191575d29fdfd489021f05b7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1103%2FPhysRev.94.630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRev.94.630%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DH.%2BY.%26aulast%3DPurcell%26aufirst%3DE.%2BM.%26atitle%3DEffects%2520of%2520diffusion%2520on%2520free%2520precession%2520in%2520nuclear%2520magnetic%2520resonance%2520experiments%26jtitle%3DPhys.%2520Rev.%26date%3D1954%26volume%3D94%26spage%3D630%26epage%3D638%26doi%3D10.1103%2FPhysRev.94.630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Meiboom, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gill, D.</span><span> </span><span class="NLM_article-title">Modified spin-echo method for measuring nuclear relaxation times</span> <span class="citation_source-journal">Rev. Sci. Instrum.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1958</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">688</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1063/1.1716296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1063%2F1.1716296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADyaF3cXosFWitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1958&pages=688-691&author=S.+Meiboomauthor=D.+Gill&title=Modified+spin-echo+method+for+measuring+nuclear+relaxation+times&doi=10.1063%2F1.1716296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Modified spin-echo method for measuring nuclear relaxation times</span></div><div class="casAuthors">Meiboom, S.; Gill, D.</div><div class="citationInfo"><span class="NLM_cas:title">Review of Scientific Instruments</span>
        (<span class="NLM_cas:date">1958</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">688-91</span>CODEN:
                <span class="NLM_cas:coden">RSINAK</span>;
        ISSN:<span class="NLM_cas:issn">0034-6748</span>.
    </div><div class="casAbstract">The Carr-Purcell method for the measurement of long nuclear relaxation times in liquids (cf. CA 48, 8077e) has been further modified.  While the pulse sequence of the modified technique was identical to the original scheme, the necessity of a very accurate adjustment of the 180° pulses was eliminated and the reproducibility was improved.  A schematic diagram of the transmitter gate used in the app. is discussed, and the derivation and principle of the method is explained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfTQfkWfruf7Vg90H21EOLACvtfcHk0lir2eyPnUBn_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3cXosFWitQ%253D%253D&md5=ecf0073dea1daa97157b5e2b6b62dc73</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1063%2F1.1716296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1716296%26sid%3Dliteratum%253Aachs%26aulast%3DMeiboom%26aufirst%3DS.%26aulast%3DGill%26aufirst%3DD.%26atitle%3DModified%2520spin-echo%2520method%2520for%2520measuring%2520nuclear%2520relaxation%2520times%26jtitle%3DRev.%2520Sci.%2520Instrum.%26date%3D1958%26volume%3D29%26spage%3D688%26epage%3D691%26doi%3D10.1063%2F1.1716296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Hodson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purkiss, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miles, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walden, H.</span><span> </span><span class="NLM_article-title">Structure of the human FANCL RING-Ube2T complex reveals determinants of cognate E3-E2 selection</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">344</span><span class="refDoi"> DOI: 10.1016/j.str.2013.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1016%2Fj.str.2013.12.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=337-344&author=C.+Hodsonauthor=A.+Purkissauthor=J.+A.+Milesauthor=H.+Walden&title=Structure+of+the+human+FANCL+RING-Ube2T+complex+reveals+determinants+of+cognate+E3-E2+selection&doi=10.1016%2Fj.str.2013.12.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DHodson%26aufirst%3DC.%26aulast%3DPurkiss%26aufirst%3DA.%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DWalden%26aufirst%3DH.%26atitle%3DStructure%2520of%2520the%2520human%2520FANCL%2520RING-Ube2T%2520complex%2520reveals%2520determinants%2520of%2520cognate%2520E3-E2%2520selection%26jtitle%3DStructure%26date%3D2014%26volume%3D22%26spage%3D337%26epage%3D344%26doi%3D10.1016%2Fj.str.2013.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Saalau-Bethell, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hiscock, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pathuri, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rich, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jhoti, H.</span><span> </span><span class="NLM_article-title">Discovery of an allosteric mechanism for the regulation of HCV NS3 protein function</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">920</span><span class="NLM_x">–</span> <span class="NLM_lpage">925</span><span class="refDoi"> DOI: 10.1038/nchembio.1081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1038%2Fnchembio.1081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=23023261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=920-925&author=S.+M.+Saalau-Bethellauthor=A.+J.+Woodheadauthor=G.+Chessariauthor=M.+G.+Carrauthor=J.+Coyleauthor=B.+Grahamauthor=S.+D.+Hiscockauthor=C.+W.+Murrayauthor=P.+Pathuriauthor=S.+J.+Richauthor=C.+J.+Richardsonauthor=P.+A.+Williamsauthor=H.+Jhoti&title=Discovery+of+an+allosteric+mechanism+for+the+regulation+of+HCV+NS3+protein+function&doi=10.1038%2Fnchembio.1081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an allosteric mechanism for the regulation of HCV NS3 protein function</span></div><div class="casAuthors">Saalau-Bethell, Susanne M.; Woodhead, Andrew J.; Chessari, Gianni; Carr, Maria G.; Coyle, Joseph; Graham, Brent; Hiscock, Steven D.; Murray, Christopher W.; Pathuri, Puja; Rich, Sharna J.; Richardson, Caroline J.; Williams, Pamela A.; Jhoti, Harren</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">920-925</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we report a highly conserved new binding site located at the interface between the protease and helicase domains of the hepatitis C virus (HCV) NS3 protein.  Using a chem. lead, identified by fragment screening and structure-guided design, we demonstrate that this site has a regulatory function on the protease activity via an allosteric mechanism.  We propose that compds. binding at this allosteric site inhibit the function of the NS3 protein by stabilizing an inactive conformation and thus represent a new class of direct-acting antiviral agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfidTQdFHIbrVg90H21EOLACvtfcHk0lir2eyPnUBn_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqt7rN&md5=d9b33f6109ae6c63935ecadf28f96345</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1081%26sid%3Dliteratum%253Aachs%26aulast%3DSaalau-Bethell%26aufirst%3DS.%2BM.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCoyle%26aufirst%3DJ.%26aulast%3DGraham%26aufirst%3DB.%26aulast%3DHiscock%26aufirst%3DS.%2BD.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DPathuri%26aufirst%3DP.%26aulast%3DRich%26aufirst%3DS.%2BJ.%26aulast%3DRichardson%26aufirst%3DC.%2BJ.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DDiscovery%2520of%2520an%2520allosteric%2520mechanism%2520for%2520the%2520regulation%2520of%2520HCV%2520NS3%2520protein%2520function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D920%26epage%3D925%26doi%3D10.1038%2Fnchembio.1081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Davies, T. G.; Jhoti, H.; Pathuri, P.; Williams, G.</span><span> </span><span class="NLM_article-title">Selecting the right targets for fragment-based drug discovery</span>. In  <span class="citation_source-book">Fragment-Based Drug Discovery Lessons and Outlook</span>, <span class="NLM_edition">1st</span> ed.; <span class="NLM_contrib-group">Erlanson, D. A.; Jahnke, W.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1002%2F9783527683604.ch02" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=T.+G.+Davies&author=H.+Jhoti&author=P.+Pathuri&author=G.+Williamsauthor=D.+A.+Erlanson&author=W.+Jahnke&title=Fragment-Based+Drug+Discovery+Lessons+and+Outlook"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2F9783527683604.ch02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527683604.ch02%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DT.%2BG.%26atitle%3DSelecting%2520the%2520right%2520targets%2520for%2520fragment-based%2520drug%2520discovery%26btitle%3DFragment-Based%2520Drug%2520Discovery%2520Lessons%2520and%2520Outlook%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Miles, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frost, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sidhu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaugule, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alpi, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walden, H.</span><span> </span><span class="NLM_article-title">The Fanconi anemia DNA repair pathway is regulated by an interaction between ubiquitin and the E2-like fold domain of FANCL</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">20995</span><span class="NLM_x">–</span> <span class="NLM_lpage">21006</span><span class="refDoi"> DOI: 10.1074/jbc.M115.675835</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;key=10.1074%2Fjbc.M115.675835" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=20995-21006&author=J.+A.+Milesauthor=M.+G.+Frostauthor=E.+Carrollauthor=M.+L.+Roweauthor=M.+J.+Howardauthor=A.+Sidhuauthor=V.+K.+Chauguleauthor=A.+F.+Alpiauthor=H.+Walden&title=The+Fanconi+anemia+DNA+repair+pathway+is+regulated+by+an+interaction+between+ubiquitin+and+the+E2-like+fold+domain+of+FANCL&doi=10.1074%2Fjbc.M115.675835"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.675835&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.675835%26sid%3Dliteratum%253Aachs%26aulast%3DMiles%26aufirst%3DJ.%2BA.%26aulast%3DFrost%26aufirst%3DM.%2BG.%26aulast%3DCarroll%26aufirst%3DE.%26aulast%3DRowe%26aufirst%3DM.%2BL.%26aulast%3DHoward%26aufirst%3DM.%2BJ.%26aulast%3DSidhu%26aufirst%3DA.%26aulast%3DChaugule%26aufirst%3DV.%2BK.%26aulast%3DAlpi%26aufirst%3DA.%2BF.%26aulast%3DWalden%26aufirst%3DH.%26atitle%3DThe%2520Fanconi%2520anemia%2520DNA%2520repair%2520pathway%2520is%2520regulated%2520by%2520an%2520interaction%2520between%2520ubiquitin%2520and%2520the%2520E2-like%2520fold%2520domain%2520of%2520FANCL%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D20995%26epage%3D21006%26doi%3D10.1074%2Fjbc.M115.675835" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YH2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1YH2','PDB','1YH2'); return false;">PDB: 1YH2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NGZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NGZ','PDB','5NGZ'); return false;">PDB: 5NGZ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i4"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00147">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_32068"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00147">10.1021/acs.jmedchem.7b00147</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental section: protein expression and purification, DSF, BLI, ligand-observed NMR spectroscopy, Ube2T backbone resonance assignment, CSP experiments, ITC, crystallization and structure determination, ubiquitination assays (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_001.pdf">jm7b00147_si_001.pdf (8.96 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00147/suppl_file/jm7b00147_si_002.csv">jm7b00147_si_002.csv (0.2 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates can be accessed using PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NGZ">5NGZ</a>. Authors will release the atomic coordinates and experimental data upon article publication. Backbone assignment of human Ube2T(1–154) can be accessed using BMRB accession number 27035.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00147&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00147%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-9%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00147" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a506b5fe63cf4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
