
<html lang="en"     class="pb-page"  data-request-id="42508a5a-0f41-4f4c-b171-c034ea247691"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-10;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.6b00052;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Toward the Validation of Maternal Embryonic Leucine Zipper Kinase: Discovery, Optimization of Highly Potent and Selective Inhibitors, and Preliminary Biology Insight" /></meta><meta name="dc.Creator" content="B. Barry  Touré" /></meta><meta name="dc.Creator" content="John  Giraldes" /></meta><meta name="dc.Creator" content="Troy  Smith" /></meta><meta name="dc.Creator" content="Elizabeth R.  Sprague" /></meta><meta name="dc.Creator" content="Yaping  Wang" /></meta><meta name="dc.Creator" content="Simon  Mathieu" /></meta><meta name="dc.Creator" content="Zhuoliang  Chen" /></meta><meta name="dc.Creator" content="Yuji  Mishina" /></meta><meta name="dc.Creator" content="Yun  Feng" /></meta><meta name="dc.Creator" content="Yan  Yan-Neale" /></meta><meta name="dc.Creator" content="Subarna  Shakya" /></meta><meta name="dc.Creator" content="Dongshu  Chen" /></meta><meta name="dc.Creator" content="Matthew  Meyer" /></meta><meta name="dc.Creator" content="David  Puleo" /></meta><meta name="dc.Creator" content="J. Tres  Brazell" /></meta><meta name="dc.Creator" content="Christopher  Straub" /></meta><meta name="dc.Creator" content="David  Sage" /></meta><meta name="dc.Creator" content="Kirk  Wright" /></meta><meta name="dc.Creator" content="Yanqiu  Yuan" /></meta><meta name="dc.Creator" content="Xin  Chen" /></meta><meta name="dc.Creator" content="Jose  Duca" /></meta><meta name="dc.Creator" content="Sean  Kim" /></meta><meta name="dc.Creator" content="Li  Tian" /></meta><meta name="dc.Creator" content="Eric  Martin" /></meta><meta name="dc.Creator" content="Kristen  Hurov" /></meta><meta name="dc.Creator" content="Wenlin  Shao" /></meta><meta name="dc.Description" content="MELK kinase has been implicated in playing an important role in tumorigenesis. Our previous studies suggested that MELK is involved in the regulation of cell cycle and its genetic depletion leads t..." /></meta><meta name="Description" content="MELK kinase has been implicated in playing an important role in tumorigenesis. Our previous studies suggested that MELK is involved in the regulation of cell cycle and its genetic depletion leads t..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 17, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00052" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00052" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00052" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00052" /></link>
        
    
    

<title>Toward the Validation of Maternal Embryonic Leucine Zipper Kinase: Discovery, Optimization of Highly Potent and Selective Inhibitors, and Preliminary Biology Insight | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00052" /></meta><meta property="og:title" content="Toward the Validation of Maternal Embryonic Leucine Zipper Kinase: Discovery, Optimization of Highly Potent and Selective Inhibitors, and Preliminary Biology Insight" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0013.jpeg" /></meta><meta property="og:description" content="MELK kinase has been implicated in playing an important role in tumorigenesis. Our previous studies suggested that MELK is involved in the regulation of cell cycle and its genetic depletion leads to growth inhibition in a subset of high MELK-expressing basal-like breast cancer cell lines. Herein we describe the discovery and optimization of novel MELK inhibitors 8a and 8b that recapitulate the cellular effects observed by short hairpin ribonucleic acid (shRNA)-mediated MELK knockdown in cellular models. We also discovered a novel fluorine-induced hydrophobic collapse that locked the ligand in its bioactive conformation and led to a 20-fold gain in potency. These novel pharmacological inhibitors achieved high exposure in vivo and were well tolerated, which may allow further in vivo evaluation." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00052"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00052">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00052&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00052&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00052&amp;href=/doi/10.1021/acs.jmedchem.6b00052" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 4711-4723</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00065" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00079" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Toward the Validation of Maternal Embryonic Leucine Zipper Kinase: Discovery, Optimization of Highly Potent and Selective Inhibitors, and Preliminary Biology Insight</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=B.+Barry++Tour%C3%A9">B. Barry Touré</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John++Giraldes">John Giraldes</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Troy++Smith">Troy Smith</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth+R.++Sprague">Elizabeth R. Sprague</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yaping++Wang">Yaping Wang</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Simon++Mathieu">Simon Mathieu</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhuoliang++Chen">Zhuoliang Chen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yuji++Mishina">Yuji Mishina</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yun++Feng">Yun Feng</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Yan-Neale">Yan Yan-Neale</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Subarna++Shakya">Subarna Shakya</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dongshu++Chen">Dongshu Chen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthew++Meyer">Matthew Meyer</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Puleo">David Puleo</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=J.+Tres++Brazell">J. Tres Brazell</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christopher++Straub">Christopher Straub</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Sage">David Sage</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kirk++Wright">Kirk Wright</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yanqiu++Yuan">Yanqiu Yuan</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Chen">Xin Chen</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jose++Duca">Jose Duca</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sean++Kim">Sean Kim</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Li++Tian">Li Tian</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eric++Martin">Eric Martin</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kristen++Hurov">Kristen Hurov</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wenlin++Shao">Wenlin Shao</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Novartis Institutes for Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 617-373-8818. E-mail: <a href="/cdn-cgi/l/email-protection#c6a4b2a9b3b4a386b4a3aaa7bfb2bee8a5a9ab"><span class="__cf_email__" data-cfemail="a2c0d6cdd7d0c7e2d0c7cec3dbd6da8cc1cdcf">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00052&amp;href=/doi/10.1021%2Facs.jmedchem.6b00052" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 4711–4723</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 17, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 January 2016</li><li><span class="item_label"><b>Published</b> online</span>17 May 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00052" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00052</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4711%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DB.%2BBarry%2BTour%25C3%25A9%252C%2BJohn%2BGiraldes%252C%2BTroy%2BSmith%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D10%26contentID%3Dacs.jmedchem.6b00052%26title%3DToward%2Bthe%2BValidation%2Bof%2BMaternal%2BEmbryonic%2BLeucine%2BZipper%2BKinase%253A%2BDiscovery%252C%2BOptimization%2Bof%2BHighly%2BPotent%2Band%2BSelective%2BInhibitors%252C%2Band%2BPreliminary%2BBiology%2BInsight%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4723%26publicationDate%3DMay%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00052"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1800</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">27</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00052" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Toward the Validation of Maternal Embryonic Leucine Zipper Kinase: Discovery, Optimization of Highly Potent and Selective Inhibitors, and Preliminary Biology Insight&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;B.&quot;,&quot;last_name&quot;:&quot;Barry Touré&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Giraldes&quot;},{&quot;first_name&quot;:&quot;Troy&quot;,&quot;last_name&quot;:&quot;Smith&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;R. Sprague&quot;},{&quot;first_name&quot;:&quot;Yaping&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;Mathieu&quot;},{&quot;first_name&quot;:&quot;Zhuoliang&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yuji&quot;,&quot;last_name&quot;:&quot;Mishina&quot;},{&quot;first_name&quot;:&quot;Yun&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Yan-Neale&quot;},{&quot;first_name&quot;:&quot;Subarna&quot;,&quot;last_name&quot;:&quot;Shakya&quot;},{&quot;first_name&quot;:&quot;Dongshu&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;Meyer&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Puleo&quot;},{&quot;first_name&quot;:&quot;J.&quot;,&quot;last_name&quot;:&quot;Tres Brazell&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;Straub&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Sage&quot;},{&quot;first_name&quot;:&quot;Kirk&quot;,&quot;last_name&quot;:&quot;Wright&quot;},{&quot;first_name&quot;:&quot;Yanqiu&quot;,&quot;last_name&quot;:&quot;Yuan&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Jose&quot;,&quot;last_name&quot;:&quot;Duca&quot;},{&quot;first_name&quot;:&quot;Sean&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Tian&quot;},{&quot;first_name&quot;:&quot;Eric&quot;,&quot;last_name&quot;:&quot;Martin&quot;},{&quot;first_name&quot;:&quot;Kristen&quot;,&quot;last_name&quot;:&quot;Hurov&quot;},{&quot;first_name&quot;:&quot;Wenlin&quot;,&quot;last_name&quot;:&quot;Shao&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;4711-4723&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00052&quot;},&quot;abstract&quot;:&quot;MELK kinase has been implicated in playing an important role in tumorigenesis. Our previous studies suggested that MELK is involved in the regulation of cell cycle and its genetic depletion leads to growth inhibition in a subset of high MELK-expressing basal-like breast cancer cell lines. Herein we describe the discovery and optimization of novel MELK inhibitors 8a and 8b that recapitulate the cellular effects observed by short hairpin ribonucleic acid (shRNA)-mediated MELK knockdown in cellular models. We also discovered a novel fluorine-induced hydrophobic collapse that locked the ligand in its bioactive conformation and led to a 20-fold gain in potency. These novel pharmacological inhibitors achieved high exposure in vivo and were well tolerated, which may allow further in vivo evaluation.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00052&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00052" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00052&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00052" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00052&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00052" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00052&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00052&amp;href=/doi/10.1021/acs.jmedchem.6b00052" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00052" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00052" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00052%26sid%3Dliteratum%253Aachs%26pmid%3D27187609%26genre%3Darticle%26aulast%3DTour%25C3%25A9%26date%3D2016%26atitle%3DToward%2Bthe%2BValidation%2Bof%2BMaternal%2BEmbryonic%2BLeucine%2BZipper%2BKinase%253A%2BDiscovery%252C%2BOptimization%2Bof%2BHighly%2BPotent%2Band%2BSelective%2BInhibitors%252C%2Band%2BPreliminary%2BBiology%2BInsight%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D10%26spage%3D4711%26epage%3D4723%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/jmcmar.2016.59.issue-10/20160526/jmcmar.2016.59.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">MELK kinase has been implicated in playing an important role in tumorigenesis. Our previous studies suggested that MELK is involved in the regulation of cell cycle and its genetic depletion leads to growth inhibition in a subset of high MELK-expressing basal-like breast cancer cell lines. Herein we describe the discovery and optimization of novel MELK inhibitors <b>8a</b> and <b>8b</b> that recapitulate the cellular effects observed by short hairpin ribonucleic acid (shRNA)-mediated MELK knockdown in cellular models. We also discovered a novel fluorine-induced hydrophobic collapse that locked the ligand in its bioactive conformation and led to a 20-fold gain in potency. These novel pharmacological inhibitors achieved high exposure in vivo and were well tolerated, which may allow further in vivo evaluation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Small molecule inhibition in conjunction with short hairpin ribonucleic acid (shRNA) knockdown can represent a very powerful target validation strategy, the desirable outcome being that the small molecule should match the shRNA-induced phenotype.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Herein, we discuss our reduction to practice of this drug target discovery paradigm to maternal embryonic leucine zipper kinase (MELK). MELK is a serine/threonine kinase and an atypical member of the sucrose nonfermenting 1/adenosine monophosphate-activated protein kinase (SNF1/AMPK) family.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Its expression has been found to be elevated in a broad spectrum of cancer types, including breast, ovarian, lung, pancreas, and colorectal cancers.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In glioma and breast cancer stem cells, overexpression of MELK was shown to be required for cancer cell proliferation.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> Correspondingly, MELK has been identified as part of a gene signature representing undifferentiated cancers that are thought to be more aggressive in behavior and linked to poor patient outcome,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and an increase in MELK expression correlated with more aggressive forms of astrocytoma, breast cancer, melanoma, and glioblastoma.<a onclick="showRef(event, 'ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref12">(9-12)</a> We have recently described the identification of MELK as a novel oncogenic kinase from an in vivo gain of function tumorigenesis screen using a kinome-wide open reading frames library.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Our studies demonstrated that MELK was highly expressed in the basal-like breast cancer cells and genetic depletion of MELK by shRNA led to growth inhibition in basal-like breast cancer cells but not in luminal breast cancer cells. Our data suggested that MELK could play an essential role in regulating cell mitosis in a subset of cancer cells; thus it presents an opportunity of a cancer-selective therapeutic target.</div><div class="NLM_p">Several MELK small molecule inhibitors have been reported in the literature. OTSSP167 was an inhibitor described to have subnanomolar potency against MELK in an in vitro biochemical assay and exhibited antiproliferative effects against a number of cancer cell lines.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The compound also demonstrated in vivo efficacy in tumor xenograft models. However, selectivity of this inhibitor and whether the antitumor effect was solely driven by MELK inhibition were not known. Studies led by a separate group reported the structural characterization of two type I compounds, which inhibited MELK with biochemical IC<sub>50</sub> of 27 nM and 12 nM, respectively. When tested against a panel of 50 kinases, the former also inhibited three other kinases (haspin, GSK3β, and CDK2/cylinA) with activities below 1 μM while the latter displayed inhibition against seven further kinases with a potency below 1 μM (AKT1, Flt3, PKCβ, PDK1, RET, PAK4, and PKAα).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The most potent compound displayed antiproliferative activity against A2789, LNCaP (<0.5 μM) and MDA-MB-468 (>2 μM) with concomitant increase in p53 or cleaved PARP cellular levels. Finally, Astex in collaboration with Janssen also reported type I and type II MELK inhibitors with low nanomolar potency in biochemical assays.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> In the type I series, the most potent compound (IC<sub>50</sub> = 37 nM against MELK) showed >50% inhibition at 1 μM against six additional kinases in a panel of 235 enzymes. Interestingly, the most potent compound in the type II series (19 nM against MELK) was less selective; it showed >50% inhibition at 1 μM for 31/243 kinases tested. These compounds were shown to be cell permeable, but no MELK cellular activity or PK data were reported. The advancement in developing pharmacological inhibitors of MELK is encouraging; at the same time, development of well-characterized and selective MELK inhibitors suitable as in vitro and in vivo probes is still needed to further explore the biological functions and therapeutic impact of this target.</div><div class="NLM_p last">On the basis of our key findings (discussed above)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and useful tools we have developed, we initiated medicinal chemistry efforts aimed at identifying highly potent and selective MELK inhibitors guided by structural biology. We then asked whether pharmacological inhibition of MELK could recapitulate the findings from shRNA-based genetic depletion.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58403" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58403" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Virtual Screening and Binding Mode of Pyrazole Hit</h3><div class="NLM_p">High-throughput screening (HTS) against a MELK kinase domain biochemical assay with a 250 K Chiron compound library yielded 1123 hits that showed MELK IC<sub>50</sub> values less than 10 μM. These hits were then used to build profile-QSAR (pQSAR) and Surrogate AutoShim models.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> pQSAR models achieve extremely high accuracy by combining the available MELK IC<sub>50</sub> values with all of the historical IC<sub>50</sub> data across the entire kinase family. 3D Surrogate AutoShim docking models create a customized MELK scoring function by training pharmacophore “shims” in the active site of an ensemble of 16 diverse kinase crystal structures to best reproduce the MELK IC<sub>50</sub> values. These models were subsequently used to screen 1.4 million compound library in silico. From this initial virtual screening campaign (VS1), 1700 compounds were selected for IC<sub>50</sub> measurement in the biochemical assay based primarily on sampling the chemical diversity of the Novartis archive not represented in the original data collection. The “novelty/hit-rate histogram” for VS1 in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>a shows that virtually all of the selected compounds were very unlike the training set, (Tanimoto coefficient of <0.5), yet the hit rates were still around 30%. New pQSAR and Surrogate AutoShim models were trained using the combined IC<sub>50</sub> values (VS2). The new model predicted 50K compounds would be active, so a sophisticated triaging scheme was employed to identify the most useful hits. Removing “exhausted” scaffolds that are well-known and widely patented, along with druglike filters and compounds very similar to known actives from VS1, reduced the number by half. The remaining compounds were divided into six overlapping bins: SAR sets around the attractive compounds from VS1, compounds that mapped to scaffolds in a kinase compendium but Tc < 0.8 to known actives, unique scaffolds not in that list, potent compounds with predicted IC<sub>50</sub> < 100 nM, efficient ligands with predicted LE > 0.34, and compounds with experimental activity on EGFR. Each of these bins was clustered and sampled at different prioritized densities to collect the total of 2K compounds. <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>b shows that about two-thirds of the compounds selected from VS2 were novel, and the hit-rate for the novel compounds was 40–50%. The combined hits led to several potent medicinal chemistry series with good physical properties and in vitro ADME. Among those, the <i>N</i>-arylpyrazoles (represented by <b>1a</b>, <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) emerged as a particularly promising series due to their kinase selectivity profile; a prototypical representative of this series (<b>1a</b>) only inhibited three kinases having IC<sub>50</sub> < 5 μM (Flt3, PDGFRα, and MAP4K4) besides MELK in a standard panel of 71 kinases.</div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Confirmed MELK Virtual Screen Hits<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>Kinase domain activity was measured at <sup>a</sup>[ATP] = 20 μM and <sup>b</sup>[ATP] = 2 mM.</p></p></figure><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Discovery of selective MELK inhibitors via virtual screening: (a) first iteration of MELK virtual screening, showing hit-rate versus novelty analysis; (b) novelty/hit-rate plot for second iteration of MELK virtual screening. The blue bars represent hit rates from the virtual screen, while the red bars plot the percentage of hits in each bin that share a Bemis and Murcko framework with the frameworks in the training set. The green circles show the hit counts for each similarity bin. For (b), while the majority of the compounds were still chosen for novelty, including novel scaffolds, a significant number were also chosen as SAR sets to further evaluate leads from the first iteration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The binding mode of <i>N</i>-arylpyrazole <b>1a</b> is shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. It is a type I inhibitor, binding MELK with the C-helix and DFG segment in the “in” conformations and utilizing a single hydrogen-bond interaction with the hinge (NH Cys89) combined with numerous van der Waals interactions. The nearly coplanar pyridine–pyrazole rings are sandwiched between Ala38 and Leu86 from the N-lobe and Leu139 and Ile149 from the C-lobe. The pyridine nitrogen forms a water-mediated hydrogen bond with Lys40, the conserved lysine in kinases that also forms a salt bridge with Glu57 of the C-helix in the active conformation. In general, MELK ATP-site binders sit higher in the pocket compared to other kinases due to MELK having a bulky Ile (Ile149) at the DFG-1 residue in contrast to the more frequently observed Ala, Gly, or Ser, thus likely increasing the chances of obtaining a MELK-specific inhibitor. The phenyl moiety is positioned perpendicular to Tyr88 forming edge-to-face π-interactions, a unique feature that was described in more detail elsewhere.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The Leu27, Ile17, Pro90, and Cys89 residues also contribute to the pocket. The remaining amide side chain is solvent exposed, providing a handle to impart better physical chemical properties on the series if needed. The P-loop is disordered as is often observed in kinase inhibitor cocrystal structures. Because this starting scaffold afforded excellent kinases selectivity, we elected to preserve the general binding features throughout the optimization process, specifically the single point interaction with the hinge, and edge-to-face interaction with Tyr88. Guided by the crystal structure, our medicinal chemistry optimization plan focused on decreasing the hydrophobicity in the solvent exposed region, probing the pyridine region and identifying a handle to reach the ribose pocket to further improve potency.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of <b>1a</b> bound to MELK. <b>1a</b> bound in the active site of MELK (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH8">5IH8</a>). Key interactions with catalytic Lys40, hinge Cys89, Tyr88, and Pro90 are highlighted. The ligand also makes several hydrophobic interactions with the protein. Red spheres represent water molecules with hydrogen bonds indicated by dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The compounds were prepared using three general routes. First, reaction of the commercially available 4-bromopyrazole with 1-fluoro-(4-methylsufonyl)benzene afforded the common intermediate sulfone <b>2</b>, which was coupled with the appropriate boronic acids to form 4-arylpyrazoles <b>3a</b>–<b>c</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> describes the synthesis of variants that replaced the sulfone moiety with <i>N</i>-methylpiperazine. The requisite bromopyrazole <b>6a</b> was synthesized via a two-step procedure that was initiated by lithiating commercially available bromoarene <b>4</b> and in situ trapping of the resulting organometallic intermediate with di-<i>tert</i>-butyl azodicarboxylate (DBAD). The product of this reaction (hydrazide <b>5</b>) was then deprotected in situ and condensed with 2-bromomalonaldehyde to form pyrazole <b>6a</b>. The coupling of pyridine derivatives with <b>6a</b> was achieved using either Negishi cross-coupling to afford 3-fluoro-4-pyrazolylpyridine <b>7a</b> or Suzuki–Miyaura cross-couplings to yield 3-methoxy-4-pyrazolylpyridine <b>7b</b>. The synthesis of unsubstituted 4-pyrazolylpyridine <b>7c</b> started with the Chan-Lam coupling of 4-(1<i>H</i>-pyrazol-4-yl)pyridine with 4-bromophenylboronic acid. The product of this reaction (<i>N</i>-halogenated arylpyrazole <b>6b</b>) was then carried through the Buchwald N-arylation condition to afford the title compound <b>7c</b>. <b>7a</b> was then converted into 3-alkoxy-4-pyrazolylpyridines <b>8a</b>–<b>g</b> by <i>ipso</i> displacement of the fluoride with alkoxides, which were formed in situ for the purpose. To probe the left side of the molecule (<i>N</i>-phenyl replacements), two orthogonal synthetic sequences were deployed. As shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, analogue alkoxypyridine <b>12a</b> was prepared in three sequential steps starting with the bromination of commercially available 1-phenylpyrazole (<b>13</b>) to form 4-bromopyrazole <b>14</b>. Suzuki–Miyaura coupling was then used to forge the pyrazole–pyridine bond (<b>14</b> into <b>15</b>), and finally, nucleophilic aromatic substitution yielded <b>12a</b>. The synthesis of related 3-arylpyrazole analogues <b>12b</b>–<b>d</b> commenced with commercially available building block <b>9</b>, which was submitted to conventional SN<sub>Ar</sub> reaction conditions to provide <b>10</b>. Coupling of <b>10</b> with 3-boronopyrazole then afforded the key NH-free pyrazole <b>11</b>; the latter was then N-arylated under Buchwald N-arylation reaction conditions to afford the target compounds <b>12b</b>–<b>d</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4-Arylpyzoles <b>3a</b>–<b>c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reaction conditions: (a) Cs<sub>2</sub>CO<sub>3</sub>, 95 °C, 1 h; (b) ArB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, P(Cy)<sub>3</sub>, K<sub>3</sub>PO<sub>4</sub>, microwave 130 °C, 20 min.</p></p></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0008.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>7a</b>–<b>c</b> and General Procedure for the Late Stage Introduction of Alkoxides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reaction conditions: (a) <i>n</i>-Bu-Li, −65 °C, 5 min and then DBAD, −60 °C to −10 °C, 30 min; (b) 2-bromomalonaldehyde, TFA, DCM, rt; (c) 2,2,6,6-tetramethylpiperidine, <i>n</i>-BuLi, THF, 0 °C, 20 min, add 3-fluoropyridine, −78 °C, 1 h; (d) ZnBr<sub>2</sub>, warm to rt, add <b>6a</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 60 °C, 1 h, then Pd(amphos)Cl<sub>2</sub>, 60 °C, 8 h; (e) 3-fluoro-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine, K<sub>3</sub>PO<sub>4</sub>, Pd(amphos)Cl<sub>2</sub>, dioxane/water, 100 °C, 2 h or 3-methoxy-4-pyridinylboronic acid, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, water, dioxane, microwave 140 °C, 30 min; (f) NaH, alcohols, DMF 25–85 °C; (g) 4-bromophenylboronic acid, Cu(OAc)<sub>2</sub>, 4 Å molecular sieves, DCM; (h) 1-methylpiperazine, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, <i>tert</i>-BuONa, dioxane, microwave 130 °C, 20 min.</p></p></figure><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0009.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Monocationic Compounds (<b>12a</b>–<b>d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reactions conditions: (a) NaH, <i>tert</i>-butyl 4-(hydroxymethyl)piperidine-1-carboxylate, 70 °C, 5 h; (b) 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-pyrazole, Pd(dppf)Cl<sub>2</sub>·DCM adduct, Na<sub>2</sub>CO<sub>3</sub>, dioxane, water, reflux, 16 h. (c) General procedure 3: ArI, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, <i>tert</i>-BuONa, toluene, 105 °C, 16 h, then deprotection using general procedure 2. (d) AcOH, Br<sub>2</sub>, 100 °C; (e) 3-fluoro-4-pyridineB(OH)<sub>2</sub>, Pd(amphos)Cl<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, dioxane, 80–95 °C, 2–16 h.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> MELK SAR, Structural Basis of Potency Improvement, and Rodent PK</h3><div class="NLM_p">Our SAR exploration efforts initially focused on improving the binding efficiency of <i>N</i>-arylpyrazole hit (<b>1a)</b> by targeting amide replacements. The synthesis of a focused set of compounds resulted in the identification of methylsulfone (<b>3a</b>) and <i>N</i>-methylpiperazine (<b>7a</b>) moieties as suitable amide surrogates (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Using 4-pyrazolylpyridine <b>3a</b> as a new starting point, we aimed our efforts at identifying potential pyridine replacements. Unlike compound <b>1b</b> (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>), deletion of the pyridine nitrogen (4-pyridine to phenyl) abrogated the activity of 4-phenylpyrazole <b>3b</b> in inhibiting MELK (IC<sub>50</sub> > 20 μM). Similarly, moving the pyridine nitrogen to the 3-position also had deleterious effect on potency (not shown). We hypothesized that this significant activity cliff<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> indicated that the water-mediated H-bonding with Lys40 was both optimal and critical; thus it should be preserved. In agreement, phenol <b>3c</b>, which is expected to establish a direct interaction with Lys40, was slightly less potent. Overall, these preliminary efforts fell short of our potency goal of achieving submicromolar IC<sub>50</sub> values at high ATP concentration (2 mM), which better mimics the cellular concentration.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR Summary</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0010.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0011.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Average of at least two separate runs (<b>8a</b> is an average of 26 runs). MELK kinase domain activity was measured at [ATP] = 20 μM.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">MELK kinase domain activity was measured at [ATP] = 2 mM.</p></div><div class="footnote" id="t1fn3"><sup>Table c</sup><p class="last">Full length MELK kinase activity was measured at [ATP] = 20 μM.</p></div><div class="footnote" id="t1fn4"><sup>Table d</sup><p class="last">SPR-derived <i>K</i><sub>D</sub>.</p></div></div><div></div></div><div class="NLM_p">Next, guided by X-ray crystal structure of <i>N</i>-arylpyrazole hit <b>1a</b> bound to MELK and modeling, we hypothesized that further extending the methoxy group in compound <b>7b</b> into the ribose pocket would improve potency. More specifically, we targeted piperidine containing 3-alkoxy-4-pyrazolylpyridine <b>8a</b>, reasoning that the piperidine amine would form a salt bridge with Glu93. The compound was duly synthesized, and gratifyingly this change afforded our first single-digit nanomolar MELK inhibitor (IC<sub>50</sub> = 0.002 μM). The X-ray crystal structure of Novartis MELK inhibitor <b>8a</b> (NVS-MELK8a) bound to MELK (shown in pink in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>) validated the design premise, and in addition, the P-loop was also ordered, which was not seen in initial <i>N</i>-arylpyrazole <b>1a</b> bound to MELK (vide supra). Compound <b>8a</b> remained very potent (IC<sub>50</sub> = 140 nM) when the ATP concentration in the biochemical assay was shifted from 20 μM to 2 mM. Its potency was well tracked between full-length MELK versus catalytic domain construct (5 nM versus 2 nM). Furthermore, the dissociation constant (<i>K</i><sub>D</sub>) for this compound determined by SPR binding was 2 nM; its binding profile was consistent with moderately slow dissociation kinetics (<i>t</i><sub>1/2</sub> = 60 s) (not shown). To assess its selectivity, the compound was profiled at a concentration of 1 μM against a panel of 456 kinases (KINOME<i>scan</i>); it only inhibited seven off-target kinases in addition to MELK with >85% inhibition of binding at 1 μM demonstrating great selectivity (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). To verify the panel test result, we determined the IC<sub>50</sub> values against four of these off-target kinases at ATP concentrations equal to the <i>K</i><sub>m</sub>. As shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, the compound was at least 90-fold more selective in targeting MELK in all cases. These results validated our medicinal chemistry strategy of prioritizing selective hits from the virtual screens at the outset using a small panel of 71 kinases.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Overlay of <b>8a</b> and <b>8f</b> bound in the kinase active site. For <b>8a</b> (pink sticks with pink ribbon), a salt bridge is observed between the piperidine basic amine and Glu93 as well as additional ordering of the P-loop. For <b>8f</b> (teal sticks with teal ribbon), asterisks denote the end points for the disordered P-loop. Water molecules are shown as spheres in the same color as the coordinating ligand with hydrogen bonds indicated by dashed lines (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH9">5IH9</a> for <b>8a</b> and PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHA">5IHA</a> for <b>8f</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Kinase Selectivity Profile of <b>8a</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0012.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">% inhibition at 1 μM<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Flt3 (ITD)</td><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left">0.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Haspin</td><td class="colsep0 rowsep0" align="left">93</td><td class="colsep0 rowsep0" align="left">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CSNK2A2</td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KIT (A829)</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KIT (D816)</td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MINK</td><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFRα</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">0.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDGFRβ</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TYK2</td><td class="colsep0 rowsep0" align="left">93</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MELK</td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left">0.002</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Screened against 456 kinases in DiscoveRx KINOMEscan panel.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">IC<sub>50</sub> values against off-target kinases were determined using our internal kinase assays.</p></div></div></div><div class="NLM_p">Compound <b>8a</b> was fairly soluble (0.22 g/L at pH 6.8) and showed a good permeability in the Caco-2 assay (<i>P</i><sub>app</sub>(A–B) = 14 × 10<sup>–6</sup> cm/s). Its extraction ratio in the mouse microsomal assay was moderate to high (ER = 0.75), and no chemical instability issues were detected upon stirring in mouse plasma for up to 4 h (not shown). Subcutaneous administration of compound <b>8a</b> at 30 mg/kg in C57BL/6 mice resulted in good plasma exposure (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The compound adsorption into the systemic circulation was rapid (<i>T</i><sub>max</sub> = 0.4 h) and peak plasma concentration reached 6.6 μM. An ascending dose PK study in female athymic nude mice showed that the rate of compound release was maximal at 120 mg/kg and all clearance mechanisms can be saturated at 240 mg/kg (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf" class="ext-link">Supporting Information Figure S4</a>). However, when administered orally at 10 mg/kg in C57BL/6 male mice, it showed very poor PK (3.6% oral bioavailability) consistent with very high in vivo clearance (113 mL min<sup>–1</sup> kg<sup>–1</sup>). We hypothesized that the poor oral PK properties were related to two basic amines embedded within the structure, making this compound subject to potential lysosomal trapping. Subsequently, we explored decreasing the number of cationic charges at neutral pH by targeting the left side <i>N</i>-methylpiperazine moiety. Potency dropped to 36 nM when this motif was deleted (<i>N</i>-phenylpyrazole <b>12a</b>); however, as hypothesized, this transformation improved considerably the oral PK profile, <i>F</i> = 43% (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The PK profile of other similar monocationic compounds was in agreement with this observation (data not shown). After identification of PK improvement handles, further structural changes were then undertaken to understand potency gain drivers in the ribose region. Guided by X-ray insight, <i>cis</i>-cyclohexylamine <b>8b</b> was designed to interact with both the carboxylic acid of Glu93 and the carbonyl of Glu136. This transformation further increased the potency by 3-fold under both low and high ATP biochemical conditions. The replacement of the piperidine with a pyridine ring (<b>8a</b> to <b>8c</b>) decreased the potency, and in general, saturated heterocycles were vastly superior to their corresponding aromatic cores. The deletion of the methylene linker had very little impact on the potency (compare compound <b>8a</b> with <b>8d)</b>, which hinted that the salt bridge interaction with Glu93 was not critical. It should be noted that a similar observation regarding amine-Glu93 interaction was published elsewhere.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In agreement, compound <b>8e</b>, featuring a simple bulky cyclohexane group (no amine), was similar to 3-alkoxy-4-pyrazolylpyridine <b>8a</b> in potency. Reduction of the size of the hydrophobic substituent to isobutyl (<b>8f</b>) resulted in 4.5-fold decrease in potency, while further truncation to methyl group (<b>7b</b>) resulted in 32-fold potency loss relative to compound <b>8a</b>. We obtained the X-ray cocrystal structure of MELK and compound isobutyl <b>8f</b> (see overlay in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>), which confirmed the position of the secondary butyl substituent in the ribose pocket (i.e., no change in binding mode), and in addition, the P-loop was disordered. It follows that the potency gain in the ribose pocket is driven primarily by hydrophobic contacts, and potent monoamine inhibitors can be obtained. However, the replacement of heterocycles in the ribose pocket with hydrophobic substituents resulted in poor physical chemical properties due to increased clogP.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Pharmacokinetics of <b>8a</b> and <b>12a</b> in C57BL/6 Male Mice</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>8a</b> (iv)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><b>8a</b> (po)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><b>8a</b> (sc)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center"><b>12a</b> (po)<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–7h</sub> (nM·h)</td><td class="colsep0 rowsep0" align="left">230</td><td class="colsep0 rowsep0" align="left">83.2</td><td class="colsep0 rowsep0" align="left">18863</td><td class="colsep0 rowsep0" align="left">3439</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clearance (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">113.8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>V</i><sub>ss</sub> (L/kg)</td><td class="colsep0 rowsep0" align="left">40.8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (nM)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left">6558</td><td class="colsep0 rowsep0" align="left">1032</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">3.6%</td><td class="colsep0 rowsep0" align="left">273%</td><td class="colsep0 rowsep0" align="left">43%</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Compound was dosed iv (1 mg/kg) and po (10 mg/kg) in C57BL/6 male mice using 5% ethanol, 10% PG, 25% (20%, v/v) Cremophor EL in PBS buffer as vehicle.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">Dosed subcutaneously at 30 mg/kg using 10% PG + 5% Solutol.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">Dosed orally at 5 mg/kg using 10% PG + 25% (20%, v/v) Solutol.</p></div></div></div><div class="NLM_p">Next, using <i>N</i>-phenylpyrazole <b>12a</b> as starting point, a modest gain in potency was realized upon ortho-fluorination of the phenyl ring (<b>12b</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). The cocrystal structure of this compound bound to MELK showed that the piperidine ring and the fluorine atom were within van der Waals interaction distance (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). It is worth mentioning that the hydrophobic collapse was not limited to piperidine; other simple hydrophobic groups gave a similar result (not shown). We hypothesized that this hydrophobic collapse<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> locked compound <b>12b</b> in its bioactive conformation, and as such, it can be reliably used for further potency gain. Combining all the above potency SAR learnings (understanding of the role of the piperazine moiety and the fluorine induced hydrophobic collapse) led to the synthesis of morpholine <b>12c</b> and its subsequent conversion into compound <b>12d</b> via fluorination of the aromatic ring. This time, the fluorine effect was even more pronounced and afforded 20-fold gain in IC<sub>50</sub>. As a result of these efforts, we obtained another distinct subclass of potent monocationic MELK inhibitors (<b>12a</b>–<b>d</b>), a representative example of which (<b>12a</b>) demonstrated superior PK profile when compared to the dicationic compound 3-alkoxy-4-pyrazolylpyridine <b>8a</b>, thus validating our design strategy.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0005.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Ortho-fluorine induced hydrophobic collapse in <b>12b</b>. Binding conformation of <b>12b</b> was determined by X-ray crystallography (the protein is excluded for clarity) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHC">5IHC</a>). Fluorine is within VDW interaction distance of the piperidine (space filling model shown on the right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cellular Profiling</h3><div class="NLM_p">To determine if compounds are able to inhibit MELK in a cellular context, we first evaluated the effects of compounds <b>8a</b> and <b>8b</b> on cell growth in a MELK-dependent cell line, MDA-MB-468, and a MELK-independent cell line, MCF-7. MDA-MB-468 is a triple negative basal breast cancer cell line that expresses MELK at a much higher level in comparison to MCF7, an ER-positive luminal breast cancer cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf" class="ext-link">Supporting Information Figure S2</a>). Corresponding to the higher expression of MELK in MDA-MB-468 compared to MCF-7, genetic knockdown of MELK by two different MELK-targeting shRNAs led to growth inhibition of MDA-MD-468, but not MCF-7, as determined by the colony formation assay (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf" class="ext-link">Supporting Information Figure S1A</a>). Knockdown efficiency of MELK was monitored by immunoblotting that showed greater than 90% of protein depletion (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf" class="ext-link">Supporting Information Figure S1B and Figure S1C</a>). These data are consistent with our previously reported findings.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Viability following compound treatment was measured using CellTiter-Glo, which monitors the level of ATP present in cells. Consistent with our data using shRNA-mediated depletion of MELK, both compounds <b>8a</b> and <b>8b</b> inhibited the growth of MDA-MB-468 cells with an IC<sub>50</sub> of approximately 0.06 and 0.1 μM, respectively (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf" class="ext-link">Supporting Information Figure S3</a>). The growth inhibitory effects of these compounds on MCF-7 cells were much less, resulting in IC<sub>50</sub> of 1.2 and 3.6 μM, respectively. These data suggest that NVS-MELK<b>8a</b> and NVS-MELK<b>8b</b> exhibit selective activity targeting MELK, leading to more effective proliferation inhibition in MELK-dependent cancer cells compared to MELK-independent cells.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Antiproliferative Activity of <b>8a</b> and <b>8b</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center" char="."><b>8a</b></th><th class="colsep0 rowsep0" align="center" char="."><b>8b</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB-468</td><td class="colsep0 rowsep0" align="char" char=".">0.11</td><td class="colsep0 rowsep0" align="char" char=".">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCF7</td><td class="colsep0 rowsep0" align="char" char=".">3.68</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Average of at least two experiments in MELK-dependent versus non-MELK dependent cell line.</p></div></div></div><div class="NLM_p">Next we examined the effect of compound <b>8b</b> on the cell cycle of cancer cells. Our previous report demonstrated that MELK knockdown led to G2/M arrest and cytokinesis failure in basal breast cancer cells.<a onclick="showRef(event, 'ref5 ref13 ref23'); return false;" href="javascript:void(0);" class="ref ref5 ref13 ref23">(5, 13, 23)</a> Our cell cycle analysis of MDA-MB-468 in comparison to MCF-7 showed that knockdown of MELK via two different shRNAs led to an increase of cells with 4N DNA, indicating a G2/M accumulation in MDA-MB-468 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf" class="ext-link">Supporting Information Figure S1D</a>). On the other hand, MELK knockdown showed no effect on the cell cycle profile of MCF-7 cells. We then tested the effect of compound <b>8b</b> on cell cycle in these two cell lines (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). We also included AZD1152, a potent Aurora B inhibitor, in the study for comparison. As shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a, compound <b>8b</b> treatment in MDA-MB-468 cells led to a time-dependent increase in cells harboring 4N DNA content, similar to what we have observed using MELK shRNAs by FACs analysis. Furthermore, the inhibitor treatment also resulted in an increase of cells with greater than 4N DNA content, suggesting impaired cell mitosis leading to polyploidy. A concomitant increase in cell death was also observed. Similar results were also observed with 3-alkoxy-4-pyrazolylpyridine <b>8a</b> (not shown). These findings are consistent with the notion that MELK plays an important role in regulating cell mitosis and division; its inhibition could lead to aberrant mitosis and cell death. In contrast to the observations in MDA-MB-468 cells, compound <b>8b</b> treatment did not show any cell cycle effect in MCF-7 cells (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b). This is in contrast to the cell cycle effect of AZD1152 in both cell lines. These data support our hypothesis that selective inhibition of MELK leads to specific growth inhibition of cancer cells that are dependent on MELK for proliferation while sparing cells that are independent.</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0006.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of <b>8b</b> on cell cycle. (a) Treatment with <b>8b</b> led to increased apoptosis and polyploidy in MDA-MB-468 in a time-dependent manner. (b) MELK inhibitor treatment did not lead to increased apoptosis and polyploidy in MCF-7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73338" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73338" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we utilized a virtual screening approach to rapidly discover novel and selective chemical inhibitor series of MELK kinase activity. On the basis of the <i>N</i>-arylpyrazole hit <b>1a</b>, rational structure-based drug design was utilized to optimize potency, yielding 3-alkoxy-4-pyrazolylpyridine <b>8a</b>, which potently and selectively inhibited MELK. Both compounds <b>8a</b> (NVS-MELK8a) and <b>8b</b> (NVS-MELK8b) retain biochemical potencies against MELK at physiologically relevant high ATP concentration and demonstrate selective antiproliferation and cell cycle effect in MELK-dependent cancer cells. These observations are consistent with those observed using MELK shRNA, suggesting inhibition of MELK using selective pharmacological inhibitors could recapitulate findings by genetic depletion. Moreover, when dosed subcutaneously, 3-alkoxy-4-pyrazolylpyridine <b>8a</b> was well tolerated and achieved very high plasma and tumor exposures (up to 96 μM). On the basis of the results discussed above, we believe that these compounds represent very valuable in vitro and in vivo probes for further studying MELK biology and its unique roles in tumorigenesis.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54568" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54568" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedures</h3><div class="NLM_p last">Unless otherwise noted, all materials were obtained from commercial suppliers and used without further purification. All reactions involving air or moisture-sensitive reagents were performed under nitrogen. Chromatographic purification of products was accomplished on a Teledyne Isco with RediSep normal-phase silica gel (35–60 μM) and UV detection at 254 nm. Preparative HPLC was performed on a Waters 2545 HPLC system equipped with Waters PDA 2998 and/or Waters 3100 mass spectrometer detection. Method A: Waters Sunfire 30 mm i.d. × 50 mm, 5 μm particle column, eluting with water/acetonitrile with 0.1% TFA modifier, flow rate 75 mL/min, 1.5 mL injection. Method B: Waters X-Bridge 30 mm i.d. × 50 mm, 5 μm particle column, water/acetonitrile with 5 mM NH<sub>4</sub>OH, flow rate 75 mL/min, 1.5 mL injection volume. All final compounds were purified to >95% (unless otherwise stated) as determined by Waters AcQuity UPLC–UV system equipped with a Waters LCT Premier mass spectrometer with UV detection at 254 nm. Low resolution mass spectra were acquired on LC/MS systems using electrospray ionization methods from a range of instruments of the following configurations: Agilent 1100 HPLC–UV system equipped with Waters ZQ mass spectrometer and Schimadzu ELSD detectors, Waters AcQuity UPLC–UV system equipped with Waters SQ mass spectrometer and Thermo CAD detectors with UV detection at 254 nm in both cases. Nuclear magnetic resonance (NMR) spectra were acquired on a Bruker Advance 400 spectrometer (400 MHz) unless stated otherwise. All <sup>1</sup>H NMR spectra are reported in parts per million (ppm) and were measured relative to the appropriate reference signals. Data for <sup>1</sup>H NMR are described as follows: chemical shift (δ in ppm), multiplicity (s, singlet; d, doublet; t, triplet; q, quartet; quint, quintuplet; m, multiplet; br, broad signal), integration, coupling constant <i>J</i> (Hz).</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> 4-Bromo-1-(4-(methylsulfonyl)phenyl)-1<i>H</i>-pyrazole (<b>2</b>)</h3><div class="NLM_p last">1-Fluoro-4-(methylsulfonyl)benzene (3.5 g, 20.2 mmol) and 4-bromo-1<i>H</i>-pyrazole (3.1 g, 21.1 mmol) with cesium carbonate (8 g, 24.55 mmol) were combined in DMF (30 mL) in a round-bottom flask. The resulting mixture was heated to 95 °C for 1 h, then quenched with water and extracted with ethyl acetate (3 × 150 mL). The combined organic phases was washed with 4% NaCl (aq), dried, filtered, and concentrated. The resulting white solid was triturated from acetonitrile to give the title compound as white crystals (5.91 g, 19.6 mmol). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 8.99 (s, 1H), 8.15–8.03 (m, 4H), 8.00 (s, 1H), 3.27 (s, 3H). MS calcd for C<sub>10</sub>H<sub>9</sub>BrN<sub>2</sub>O<sub>2</sub>S: [M]<sup>+</sup> = 302. Found: [M + H]<sup>+</sup> = 303.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 4-(1-(4-(Methylsulfonyl)phenyl)-1<i>H</i>-pyrazol-4-yl)pyridine (<b>3a</b>)</h3><div class="NLM_p last">In a 10 mL microwave vial, 4-bromo-1-(4-(methylsulfonyl)phenyl)-1<i>H</i>-pyrazole (<b>2</b>) (235 mg, 0.78 mmol) and 4-pyridineboronic acid (288 mg, 2.34 mmol) were dissolved in dioxane (5 mL). The vial was purged by bubbling N<sub>2</sub>, then Pd<sub>2</sub>(dba)<sub>3</sub> (21 mg, 0.02 mmol) and tricyclohexylphosphine (15 mg, 0.06 mmol) were added. Aqueous 1 M tripotassium phosphate (1.56 mL, 1.56 mmol) was added, and the reaction mixture was heated to 130 °C (2 bar) for 20 min. The reaction was cooled to ambient temperature, diluted with EtOAc (10 mL), and washed with water followed by brine. The organic layer was decanted and concentrated in vacuo. The crude mixture was purified by silica gel flash chromatography (50–100% EtOAc in heptane) followed by trituration from acetonitrile to obtain the title compound as a white powder (16 mg, 0.05 mmol). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 9.42 (s, 1H), 8.60 (d, <i>J</i> = 6.06 Hz, 2H), 8.51 (s, 1H), 8.18 (d, <i>J</i> = 8.80 Hz, 2H), 8.10 (d, <i>J</i> = 8.80 Hz, 2H), 7.71–7.76 (m, 2H), 3.28 (s, 3H). HRMS calcd for C<sub>15</sub>H<sub>13</sub>N<sub>3</sub>O<sub>2</sub>S [M + H]<sup>+</sup> = 300.0800. Found [M + H]<sup>+</sup> = 300.0804.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 1-(4-(Methylsulfonyl)phenyl)-4-phenyl-1<i>H</i>-pyrazole (<b>3b</b>)</h3><div class="NLM_p last">To a vial capped with a septum were added 4-bromo-1-(4-(methylsulfonyl)phenyl)-1<i>H</i>-pyrazole <b>(2)</b> (100 mg, 0.33 mmol), phenylboronic acid (50 mg, 0.41 mmol), bis(<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (25 mg, 0.04 mmol), tripotassium phosphate (105 mg, 0.50 mmol), and dioxane (3 mL). The suspension was heated to 95 °C for 2 h under conventional heating. The reaction mixture was allowed to cool, then diluted with EtOAc and filtered through Celite, washing with DCM (2 mL × 3), and MeOH (2 mL × 3). The filtrate was concentrated to a solid, then triturated from acetonitrile to obtain the title compound (85 mg, 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 9.20 (s, 1H), 8.35 (s, 1H), 8.21–8.14 (m, 2H), 8.11–8.04 (m, 2H), 7.80–7.71 (m, 2H), 7.48–7.39 (m, 2H), 7.34–7.24 (m, 1H), 3.27 (s, 3H). HRMS calcd for C<sub>16</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub>S [M + H]<sup>+</sup> = 298.0854. Found [M + H]<sup>+</sup> = 299.0847.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 4-(1-(4-(Methylsulfonyl)phenyl)-1<i>H</i>-pyrazol-4-yl)phenol (<b>3c</b>)</h3><div class="NLM_p last">In a vial capped with a septum were added 4-bromo-1-(4-(methylsulfonyl)phenyl)-1<i>H</i>-pyrazole (<b>2</b>) (100 mg, 0.33 mmol), 4-hydroxybenzene boronic acid (50.4 mg, 0.37 mmol), bis(<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (23.51 mg, 0.03 mmol), tripotassium phosphate (106 mg, 0.50 mmol), and dioxane (3 mL). The suspension was heated to 80 °C for 16 h under conventional heating. The reaction mixture was diluted with EtOAc and washed with water (10 mL × 3) followed by brine; the organic portions were combined and dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to a dark solid. The crude was triturated from acetonitrile to obtain the title compound (22 mg, 21%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ: 9.01 (s, 1H), 8.21 (s, 1H), 8.16–8.11 (m, 2H), 8.08–8.03 (m, 2H), 7.58–7.50 (m, 2H), 6.87–6.77 (m, 2H), 3.27 (s, 3H). HRMS calcd for C<sub>16</sub>H<sub>15</sub>N<sub>2</sub>O<sub>3</sub>S [M + H]<sup>+</sup> = 315.0803. Found [M + H<sup>+</sup>]<sup>+</sup> = 315.0791.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Di-<i>tert</i>-butyl 1-(4-(4-Methylpiperazin-1-yl)phenyl)hydrazine-1,2-dicarboxylate (<b>5</b>)</h3><div class="NLM_p last">To a stirred solution of 1-(4-bromophenyl)-4-methylpiperazine (<b>4</b>) (124.7 g, 490 mmol) in THF (1.9 L) was added 2.5 M <i>n</i>-BuLi solution (234.4 mL) at −78 °C under N<sub>2</sub>. The internal temperature was kept below −65 °C during the addition. After addition, the mixture was stirred at −78 °C for 5 min. A solution of DBAD (135 g) in THF (0.63 L) was added dropwise. The internal temperature was kept below −60 °C during the addition, then the reaction mixture was warmed to −10 °C in 30 min and quenched by slow addition of water (0.5 L). The resulting mixture was partitioned between EtOAc (2.0 L) and water (1.0 L). The aqueous layer was extracted with EtOAc (0.5 L × 2), and the combined organic layer was washed with brine (0.5 L) and dried (Na<sub>2</sub>SO<sub>4</sub>). The organic layer was concentrated under reduced pressure to yield a red oil and purified by silica gel chromatography giving the desired product (162 g, 82%). MS calcd for C<sub>21</sub>H<sub>35</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup> = 407.3. Found [M + H]<sup>+</sup> = 407.2.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 1-(4-(4-Bromo-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>6a</b>)</h3><div class="NLM_p last">To a stirred solution of di-<i>tert</i>-butyl 1-(4-(4-methylpiperazin-1-yl)phenyl)hydrazine-1,2-dicarboxylate (<b>5</b>) (120 g, 296 mmol) in DCM (2.40 L) was added 2-bromomalonaldehyde (53.6 g, 355 mL), followed by TFA (452 mL). After addition, the reaction was stirred at ambient temperature for 2 days. All the volatile solvent was removed under reduced pressure, and the residue was dissolved in EtOAc (2.5 L). Water (0.5 L) was added, followed by portionwise addition of solid NaHCO<sub>3</sub> until pH = 8 (∼250 g needed). The aqueous layer was extracted with EtOAc (0.5 L × 2) and DCM (0.5 L). The combined organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and filtered. The organic layer was concentrated under reduced pressure and the residue was purified by silica gel chromatography, then triturated using ACN (100 mL) and filtered to yield the title compound (40.0 g, 42% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.62 (s, 1H), 7.78 (s, 1H), 7.66–7.58 (m, 2H), 7.08–6.99 (m, 2H), 3.21–3.13 (m, 4H), 2.48–2.42 (m, 4H), 2.22 (s, 3H). MS calcd for C<sub>14</sub>H<sub>18</sub>BrN<sub>4</sub>: [M + H]<sup>+</sup> = 321.1. Found [M + H]<sup>+</sup> = 321.0.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 1-(4-(4-(3-Fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7a</b>)</h3><div class="NLM_p last">A solution of 2,2,6,6-tetramethylpiperidine (194 g, 1373 mmol) in THF (2.0 L) was cooled to 0 °C, and <i>n</i>-BuLi (550 mL, 1373 mmol, 2.5 M solution in hexane) was added slowly under N<sub>2</sub>. The mixture was stirred at 0 °C for 20 min. The mixture was then cooled to −78 °C (dry ice/acetone bath), and a solution of 3-fluoropyridine (121.4 g, 1250 mmol) in THF (0.2 L) was added slowly. The mixture was stirred at −78 °C for 1 h. Then a solution of ZnBr<sub>2</sub> (310 g, 1375 mmol) in THF (1.25 L) was added to this mixture and slowly warmed up to room temperature (∼1.5 h). To this suspension were added 1-(4-(4-bromo-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (40 g, 125 mmol) and Pd(PPh<sub>3</sub>)<sub>4</sub> (14.0 g, 12.5 mmol). The mixture was heated to 60 °C and stirred for 1 h, then bis(di-<i>tert</i>-butyl(4-dimethylaminophenyl)phosphine)dichloropalladium(II) (Pd(amphos)Cl<sub>2</sub>) (8.9 g, 12.5 mmol) was added to this mixture, which was then stirred at 60 °C for 18 h. The reaction was allowed to cool to ambient temperature, then quenched with saturated NaHCO<sub>3</sub> solution (1.8 L). The mixture was then filtered on a filter funnel. The filtered solid was washed with EtOAc (3 × 2 L). The combined filtrates were transferred to a separatory funnel, and the layers were separated. The organic layer was dried over MgSO<sub>4</sub> and filtered. The filtered organics were concentrated under reduced pressure to dryness. The dark residue was chromatographed on silica column (dry loading with silica) to give an orange solid contaminated with 2,2,6,6-tetramethylpiperidine (or its derivative) and other minor impurities. The orange solid was then triturated first with MTBE (500 mL) and then suspended in MeOH (250 mL) and rotated at 50 °C on a rotavap for 30 min. The mixture was diluted with water (750 mL) and rotated at 50 °C on a rotavap for 1 h. The suspension was cooled to room temp and filtered. The tan solid obtained was washed with water (2 × 125 mL), dried to furnish the desired product (17 g, 35%).</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Alternative Procedure for the Preparation of 1-(4-(4-(3-Fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7a</b>)</h3><div class="NLM_p last">A suspension of 1-(4-(4-bromo-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>6a</b>) (200 mg, 0.62 mmol), 3-fluoro-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine (167 mg, 0.747 mmol), K<sub>3</sub>PO<sub>4</sub> (264 mg, 1.25 mmol), and (Pd(amphos)Cl<sub>2</sub>) (27 mg, 0.04 mmol) in a mixture of dioxane (5 mL) and water (0.500 mL) was degassed with bubbling nitrogen for 3 min and then heated to 100 °C for 2 h. The resulting mixture was cooled and diluted with EtOAc, then washed with water followed by brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to an off-white solid. The crude mixture was purified using flash chromatography (12 g RediSep SiO<sub>2</sub> column, gradient EtOAc/heptane, 0–100% over 25 min, then 100% for 10 min, then eluted with MeOH/DCM 0–5%) to give the title compound (110 mg, 50%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.98 (d, <i>J</i> = 1.3 Hz, 1H), 8.61 (d, <i>J</i> = 2.8 Hz, 1H), 8.43 (dd, <i>J</i> = 5.1, 1.2 Hz, 1H), 8.28 (d, <i>J</i> = 1.9 Hz, 1H), 7.88 (dd, <i>J</i> = 6.8, 5.0 Hz, 1H), 7.79–7.66 (m, 2H), 7.15–6.99 (m, 2H), 3.24–3.14 (m, 4H), 2.48–2.44 (m, 4H), 2.23 (s, 3H). MS calcd for C<sub>19</sub>H<sub>21</sub>FN<sub>5</sub> [M]<sup>+</sup> = 338.2. Found [M + H]<sup>+</sup> = 338.2.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 1-(4-(4-(3-Methoxypyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7b</b>)</h3><div class="NLM_p last">To a 5 mL microwave vial containing a mixture of 3-methoxy-4-pyridinylboronic acid (304 mg, 2.0 mmol), 1-(4-(4-bromo-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>6a</b>) (213 mg, 0.66 mmol), and 2 M sodium carbonate solution (1.7 mL, 3.3 mmol) in dioxane (3 mL) was added <i>trans</i>-PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (46.5 mg, 0.066 mmol, 0.1 equiv). The vial was purged with nitrogen, and the mixture was heated to 140 °C in a microwave reactor for 30 min, then diluted with EtOAc and washed with water and then brine. The crude mixture was concentrated in vacuo, then purified by silica gel flash chromatography (100% EtOAc, then 0–20% MeOH in EtOAc) to obtain the title compound as an off-white powder (81 mg, 35%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.87 (s, 1H), 8.42 (s, 1H), 8.30 (s, 1H), 8.21 (d, <i>J</i> = 5.05 Hz, 1H), 7.69–7.74 (m, 3H), 7.07 (d, <i>J</i> = 9.35 Hz, 2H), 4.04 (s, 3H), 3.16–3.21 (m, 4H), 2.45–2.48 (m, 4H), 2.23 (s, 3H). HRMS calcd for C<sub>20</sub>H<sub>24</sub>N<sub>5</sub>O [M + H]<sup>+</sup> = 350.1981. Found [M + H]<sup>+</sup> = 350.1985.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 1-Methyl-4-(4-(4-(pyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)piperazine (<b>7c</b>)</h3><div class="NLM_p"><i>Step 1.</i> A combined suspension of 4-(1<i>H</i>-pyrazol-4-yl)pyridine (1.0 g, 13.8 mmol), (4-bromophenyl)boronic acid (2.77 g, 13.8 mmol), copper(II) acetate (3.75 g, 20.7 mmol), and 4 Å molecular sieves (7 g) in dichloromethane (14 mL) was stirred open to air for 2 days. The reaction mixture was vacuum filtered through Celite, then gravity filtered through glass wool. Then, it was purified by flash silica gel chromatography (EtOAc/heptane, 0–30%) to obtain 4-(1-(4-bromophenyl)-1<i>H</i>-pyrazol-4-yl)pyridine (<b>6b</b>) (50 mg, 2%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.28–9.27 (m, 1H), 8.60–8.55 (m, 2H), 8.43 (s, 1H), 7.88 (d, <i>J</i> = 9.0 Hz, 2H), 7.75 (d, <i>J</i> = 9.0 Hz, 2H), 7.73–7.70 (m, 2H).</div><div class="NLM_p last"><i>Step 2.</i> A combined mixture of 4-(1-(4-bromophenyl)-1<i>H</i>-pyrazol-4-yl)pyridine (50 mg, 0.17 mmol), 1-methylpiperazine (25 mg, 0.25 mmol), BINAP (1 mg, 1.6 μmol), tris(dibenzylideneacetone)dipalladium(0) (1 mg, 1.1 μmol), and sodium <i>tert</i>-butoxide (24 mg, 0.25 mmol) in dioxane (2 mL) was degassed under N<sub>2</sub>, then capped and heated by microwave to 130 °C for 20 min. The reaction mixture was cooled and diluted with ethyl acetate (20 mL), washed with water (2×) and then brine, concentrated, and purified by flash silica gel chromatography (7 mM ammonia in methanol/dichloromethane, 0–15%) to give the title compound as a pale yellow powder (18 mg, 34%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.07 (s, 1H), 8.54 (d, <i>J</i> = 6.1 Hz, 2H), 8.31 (s, 1H), 7.78–7.61 (m, 4H), 7.08 (d, <i>J</i> = 9.2 Hz, 2H), 3.25–3.13 (m, 4H), 2.48–2.44 (m, 4H), 2.23 (s, 3H). HRMS calcd for C<sub>19</sub>H<sub>22</sub>N<sub>5</sub> [M + H]<sup>+</sup> = 320.1875. Found [M + H]<sup>+</sup> = 320.1863.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure 1 for SN<sub>Ar</sub> Reactions with Alkoxides</h3><div class="NLM_p last">To a solution of alcohol (1.0–3.0 equiv) in DMF (0.02–0.2 M) at 0 °C was added NaH (60% oil immersion) (2–10 equiv) in one portion. The reaction was then warmed up to room temperature for 5–45 min before being cooled back to 0 °C where a solution 1-(4-(4-(3-fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7a</b>) (1.0 equiv) in DMF was added in a rapid, dropwise manner. The reaction mixture was then heated between 35 and 100 °C depending on alcohol while monitoring by LCMS (0.5–96 h). The resulting reaction mixture was diluted with EtOAc/water (∼10:1), and after effervescence ceased, the layers were partitioned. The organic layer was washed with water, 4% aqueous sodium chloride, brine, the organic layer dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and the organic filtrate concentrated in vacuo. The crude mixture was purified using either flash chromatography (SiO<sub>2</sub> and appropriate eluent) or preparative HPLC (acidic or basic methods found in the purification section) with concentration under reduced pressure or lyophilization until pure dried product was obtained.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Procedure 2 for Boc Removal</h3><div class="NLM_p last">To a solution of Boc-protected alkoxypyridine, obtained from the reaction according to general procedure 1 (1.0 equiv), in an appropriate solvent (typically DCM but in some cases specified as MeOH), was added a solution of hydrogen chloride (HCl) in dioxane (4 M in dioxane) (5.0–15 equiv), and the resulting mixture was stirred for 2–24 h at room temperature. The reaction mixture was concentrated under vacuum giving a white solid (as the HCl salt) which was either used as is or free-based using NH<sub>4</sub>OH and further purified by the following methods: (1) silica gel flash chromatography (eluting with 1–5% NH<sub>4</sub>OH in MeOH/DCM); (2) preparative HPLC followed by lyophilization.</div></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 4-((4-(1-(4-(4-Methylpiperazin-1-yl)phenyl)-1<i>H</i>-pyrazol-4-yl)pyridin-3-yloxy)methyl)piperidine (<b>8a</b>)</h3><div class="NLM_p last">Reaction was performed according to general procedure 1 using 1-(4-(4-(3-fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine <b>7a</b> (200 mg, 0.59 mmol), <i>tert</i>-butyl 4-(hydroxymethyl)piperidine-1-carboxylate (255 mg, 1.19 mmol), and 60% NaH (95 mg, 2.4 mmol) in DMF (5 mL) at 70 °C for 16 h. The product was purified using silica gel flash chromatography (12 g, 0–5% MeOH/DCM) to obtain the Boc-protected intermediate as a white solid (281 mg, 88%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.39 (d, <i>J</i> = 0.7 Hz, 1H), 8.34 (s, 1H), 8.28 (d, <i>J</i> = 4.9 Hz, 1H), 8.18 (d, <i>J</i> = 0.7 Hz, 1H), 7.63–7.56 (m, 2H), 7.49 (d, <i>J</i> = 4.9 Hz, 1H), 7.08–7.01 (m, 2H), 4.22 (s, 2H), 4.09 (d, <i>J</i> = 6.4 Hz, 2H), 3.30 (s, 4H), 2.81 (s, 2H), 2.63 (s, 4H), 2.40 (s, 3H), 2.13 (d, <i>J</i> = 12.8 Hz, 1H), 1.92 (d, <i>J</i> = 13.2 Hz, 2H), 1.48 (d, <i>J</i> = 3.2 Hz, 9H), 1.38 (d, <i>J</i> = 11.1 Hz, 2H). This intermediate was deprotected using general procedure 2 (280 mg, 0.526 mmol) to yield 300 mg (quantitative) of the de-Boc product as an HCl. An 80 mg portion of this material was purified by HPLC under basic conditions to obtain the title compound as a white solid (32 mg, 14%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.89 (s, 1H), 8.40 (s, 1H), 8.29 (s, 1H), 8.19 (d, <i>J</i> = 4.9 Hz, 1H), 7.74–7.67 (m, 3H), 7.08 (d, <i>J</i> = 9.1 Hz, 2H), 4.09 (d, <i>J</i> = 6.0 Hz, 2H), 3.24–3.15 (m, 4H), 3.09–3.03 (m, 2H), 2.71−2.64 (m, 2H), 2.48–2.43 (m, 4H), 2.23 (s, 3H), 2.05–1.90 (m, 1H), 1.81–1.68 (m, 2H), 1.29 (qd, <i>J</i> = 12.3, 4.0 Hz, 2H). HRMS calcd for C<sub>25</sub>H<sub>33</sub>N<sub>6</sub>O [M + H]<sup>+</sup> = 433.2716. Found [M + H]<sup>+</sup> = 433.2721.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (1<i>R</i>,4<i>R</i>)-4-(((4-(1-(4-(4-Methylpiperazin-1-yl)phenyl)-1<i>H</i>-pyrazol-4-yl)pyridin-3-yl)oxy)methyl)cyclohexanamine (<b>8b</b>)</h3><div class="NLM_p last"><b>8b</b> was prepared according to general procedure 1 using <b>7a</b> (231 mg, 0.685 mmol), ((1<i>R</i>,4<i>R</i>)-4-aminocyclohexyl)methanol (206 mg, 1.24 mmol), and 60% NaH (120 mg, 3.0 mmol) in DMF (5 mL) at 70 °C overnight. Purification using flash silica gel chromatography (4 g, 5% NH<sub>4</sub>OH in methanol/DCM) gave crude product as an off-white solid. HPLC purification furnished the title compound (80 mg, 26%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.06 (s, 1H), 8.61 (d, <i>J</i> = 0.8 Hz, 1H), 8.52 (s, 1H), 8.44 (dd, <i>J</i> = 6.0, 0.8 Hz, 1H), 8.34 (d, <i>J</i> = 6.1 Hz, 1H), 7.82–7.75 (m, 2H), 7.26–7.20 (m, 2H), 4.27 (d, <i>J</i> = 5.9 Hz, 2H), 4.03–3.92 (m, 2H), 3.67 (d, <i>J</i> = 12.2 Hz, 2H), 3.23–3.06 (m, 4H), 3.01 (d, <i>J</i> = 5.5 Hz, 3H), 2.26–2.07 (m, 6H), 1.64–1.48 (m, 2H), 1.46–1.34 (m, 2H). HRMS calcd for C<sub>26</sub>H<sub>35</sub>N<sub>6</sub>O [M + H]<sup>+</sup> = 447.2872. Found [M + H]<sup>+</sup> = 447.2847.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 1-Methyl-4-(4-(4-(3-(pyridin-4-ylmethoxy)pyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)piperazine (<b>8c</b>)</h3><div class="NLM_p last"><b>8c</b> was prepared according to general procedure 1 using 1-(4-(4-(3-fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7a</b>) (15 mg, 0.044 mmol), pyridin-4-ylmethanol (25 mg, 0.23 mmol), and 95% NaH (11 mg, 0.49 mmol) in DMF (2 mL) at 50 °C. Purification by HPLC yielded the title compound (4 mg, 21.1%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.89 (s, 1H), 8.63 (d, <i>J</i> = 5.31 Hz, 2H), 8.44 (s, 1H), 8.30 (s, 1H), 8.23 (d, <i>J</i> = 4.80 Hz, 1H), 7.76 (d, <i>J</i> = 5.05 Hz, 1H), 7.65 (d, <i>J</i> = 8.84 Hz, 2H), 7.54 (d, <i>J</i> = 5.31 Hz, 2H), 7.07 (d, <i>J</i> = 8.84 Hz, 2H), 5.48 (s, 2H), 3.12–3.24 (m, 4H), 2.43–2.49 (m, 4H), 2.23 (s, 3H). HRMS calcd for C<sub>25</sub>H<sub>27</sub>N<sub>6</sub>O [M + H]<sup>+</sup> = 427.2246. Found [M + H]<sup>+</sup> = 427.2228.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 1-Methyl-4-(4-(4-(3-(piperidin-4-yloxy)pyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)piperazine (<b>8d</b>)</h3><div class="NLM_p last"><b>8d</b> was prepared according to general procedure 1 using 1-(4-(4-(3-fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7a</b>) (15 mg, 0.044 mmol), <i>tert</i>-butyl-4-hydroxypiperidine (38 mg, 0.18 mmol), and 95% NaH (11 mg, 0.46 mmol) at 50 °C. The title compound was obtained as a TFA salt after Boc-removal using TFA in DCM and HPLC purification (16 mg, 47.3%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.06 (s, 1H), 8.67 (s, 1H), 8.33–8.44 (m, 2H), 8.04 (d, <i>J</i> = 5.56 Hz, 1H), 7.78 (d, <i>J</i> = 9.09 Hz, 2H), 7.18 (d, <i>J</i> = 9.35 Hz, 2H), 4.92–5.03 (m, 1H), 3.79–4.00 (m, 2H), 3.52–3.61 (m, 2H), 3.27–3.34 (m, 2H), 3.10–3.22 (m, 4H), 2.98–3.06 (m, 2H), 2.89 (s, 3H), 2.19–2.30 (m, 2H), 1.94–2.08 (m, 2H). HRMS calcd for C<sub>24</sub>H<sub>31</sub>N<sub>6</sub>O [M + H]<sup>+</sup> = 419.2559. Found [M + H]<sup>+</sup> = 419.2542.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 1-(4-(4-(3-(Cyclohexyloxy)pyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>8e</b>)</h3><div class="NLM_p last"><b>8e</b> was prepared according to general procedure 1 using 1-(4-(4-(3-fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7a</b>) (11 mg, 0.03 mmol), cyclohexanol (16 mg, 0.16 mmol), and 95% NaH (8 mg, 0.3 mmol) at 50 °C. Purification by HPLC yielded the title compound (6 mg, 44%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.85 (s, 1H), 8.44 (s, 1H), 8.30 (s, 1H), 8.16 (d, <i>J</i> = 4.9 Hz, 1H), 7.71 (d, <i>J</i> = 4.9 Hz, 1H), 7.68 (d, <i>J</i> = 9.0 Hz, 2H), 7.08 (d, <i>J</i> = 9.1 Hz, 2H), 4.70–4.62 (m, 1H), 3.22–3.16 (m, 4H), 2.49–2.44 (m, 4H), 2.23 (s, 3H), 2.07–1.98 (m, 2H), 1.77–1.66 (m, 2H), 1.66–1.50 (m, 3H), 1.49–1.26 (m, 3H). HRMS calcd for C<sub>25</sub>H<sub>32</sub>N<sub>5</sub>O [M + H]<sup>+</sup> = 418.2607. Found [M + H]<sup>+</sup> = 418.2591.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 1-(4-(4-(3-Isobutoxypyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>8f</b>)</h3><div class="NLM_p last"><b>8f</b> was prepared according to general procedure 1 using 1-(4-(4-(3-fluoropyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)-4-methylpiperazine (<b>7a</b>) (11 mg, 0.03 mmol), 2-methylpropan-1-ol (12 mg, 0.16 mmol), and 95% NaH (8 mg, 0.3 mmol) at 50 °C. Purification by HPLC yielded the title compound (9 mg, 71%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.87 (s, 1H), 8.40 (s, 1H), 8.30 (s, 1H), 8.20 (d, <i>J</i> = 5.05 Hz, 1H), 7.73 (d, <i>J</i> = 5.05 Hz, 1H), 7.67 (d, <i>J</i> = 9.09 Hz, 2H), 7.08 (d, <i>J</i> = 9.09 Hz, 2H), 4.04 (d, <i>J</i> = 6.32 Hz, 2H), 3.12–3.23 (m, 4H), 2.42–2.49 (m, 4H), 2.22 (s, 3H), 2.15–2.22 (m, 1H), 1.06 (d, <i>J</i> = 6.82 Hz, 6H). HRMS calcd for C<sub>23</sub>H<sub>30</sub>N<sub>5</sub>O [M + H]<sup>+</sup> = 391.2450. Found [M + H]<sup>+</sup> = 392.2432.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>tert</i>-Butyl 4-(((4-Bromopyridin-3-yl)oxy)methyl)piperidine-1-carboxylate (<b>10</b>)</h3><div class="NLM_p last"><b>10</b> was prepared using general procedure 1 from <i>tert</i>-butyl 4-(hydroxymethyl)piperidine-1-carboxylate (3.49 g, 16.19 mmol, (60% oil immersion, 0.78 g, 19.43 mmol) and 4-bromo-3-fluoropyridine (1.90 g, 10.80 mmol) at 70 °C for 5 h. The crude mixture was purified by silica gel flash chromatography to give an oil (1.8 g, 45%). HRMS calcd for C<sub>16</sub>H<sub>24</sub>BrN<sub>2</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 371.0970. Found [M + H]<sup>+</sup> = 371.0968.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>tert</i>-Butyl 4-(((4-(1<i>H</i>-Pyrazol-4-yl)pyridin-3-yl)oxy)methyl)piperidine-1-carboxylate (<b>11</b>)</h3><div class="NLM_p last">A combined suspension of <i>tert</i>-butyl 4-(((4-bromopyridin-3-yl)oxy)methyl)piperidine-1-carboxylate (<b>10</b>) (1.80 g, 4.85 mmol), 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-pyrazole (1.317 g, 6.79 mmol), dichloro-1,1′-bis(diphenylphosphino)ferrocenepalladium(II) DCM adduct (0.277 g, 0.34 mmol), and sodium carbonate (1.54 g, 14.54 mmol) in a mixture of dioxane (30 mL) and water (6 mL) was degassed for 5 min by a bubbling stream of N<sub>2</sub> and then heated at 110 °C for 16 h. The reaction mixture was diluted with EtOAc, washed with saturated sodium bicarbonate, then brine. The organic portion was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude mixture was purified by silica gel flash chromatography (80g, 0–2% MeOH/EtOAc) to give the title compound as an off-white solid (570 mg, 33%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.15 (s, 1H), 8.36 (s, 1H), 8.30 (s, 1H), 8.14 (d, <i>J</i> = 5.0 Hz, 1H), 8.11 (s, 1H), 7.64 (d, <i>J</i> = 4.9 Hz, 1H), 4.08 (d, <i>J</i> = 6.3 Hz, 2H), 4.01 (dd, <i>J</i> = 13.8, 8.9 Hz, 2H), 2.78 (s, 2H), 2.15–2.01 (m, 1H), 1.79 (dd, <i>J</i> = 13.1, 3.8 Hz, 2H), 1.40 (s, 9H), 1.30–1.12 (m, 2H). MS calcd for C<sub>19</sub>H<sub>25</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup> = 358.44. Found [M + H]<sup>+</sup> = 359.5.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> General Procedure 3 for N-Arylation</h3><div class="NLM_p last">A sealed vial was charged with <i>tert</i>-butyl 4-(((4-(1<i>H</i>-pyrazol-4-yl)pyridin-3-yl)oxy)methyl)piperidine-1-carboxylate (<b>11</b>) (1.0 equiv), aryl halide (1.1 equiv), X-Phos (0.2 equiv), Pd<sub>2</sub>(dba)<sub>3</sub> (0.1 equiv), sodium <i>tert</i>-butoxide (1.6 equiv), and toluene (0.08 M). The mixture was degassed for 2 min with N<sub>2</sub> and then heated to 105 °C for 16 h. The reaction mixture was cooled and filtered and then the filtrate concentrated in vacuo. The residue was dissolved in MeOH, a hydrogen chloride solution in dioxane (4 N in dioxane) added, and the reaction was stirred at room temperature until completion. The mixture was concentrated in vacuo, and the crude product was dissolved in 7 N NH<sub>3</sub>/MeOH, then reconcentrated in vacuo. Methanol was added and the crude solution was purified to afford the desired product.</div></div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 4-(1-Phenyl-1<i>H</i>-pyrazol-4-yl)-3-(piperidin-4-ylmethoxy)pyridine (<b>12a</b>)</h3><div class="NLM_p last"><b>12a</b> was prepared from 3-fluoro-4-(1-phenyl-1<i>H</i>-pyrazol-4-yl)pyridine (<b>15</b>) (100 mg, 0.48 mmol), <i>tert</i>-butyl 4-(hydroxymethyl)piperidine-1-carboxylate (157 mg, 0.73 mmol), and 60% NaH (50 mg, 1.25 mmol) in DMF (1.5 mL) heated to 65 °C for 3 h. Reaction mixture was deprotected according to general procedure 2 and purified by HPLC to obtain the title compound (127 mg, 68%) as a TFA salt. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.06 (s, 1H), 8.44 (s, 1H), 8.37 (s, 1H), 8.22 (d, <i>J</i> = 4.9 Hz, 1H), 7.94–7.86 (m, 2H), 7.76 (d, <i>J</i> = 4.9 Hz, 1H), 7.60–7.50 (m, 2H), 7.41–7.31 (m, 1H), 4.12 (d, <i>J</i> = 6.1 Hz, 2H), 3.05 (dt, <i>J</i> = 12.1, 3.2 Hz, 2H), 2.60 (td, <i>J</i> = 12.0, 2.6 Hz, 2H), 2.12–1.97 (m, 1H), 1.79 (dd, <i>J</i> = 13.4, 3.5 Hz, 2H), 1.33 (qd, <i>J</i> = 12.2, 4.0 Hz, 2H). HRMS calcd for C<sub>20</sub>H<sub>23</sub>N<sub>4</sub>O [M + H]<sup>+</sup> = 335.1872. Found [M + H]<sup>+</sup> = 335.1872.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 4-(1-(2-Fluorophenyl)-1<i>H</i>-pyrazol-4-yl)-3-(piperidin-4-ylmethoxy)pyridine (<b>12b</b>)</h3><div class="NLM_p last"><b>12b</b> was prepared according to general procedure 3 using <i>tert</i>-butyl 4-(((4-(1<i>H</i>-pyrazol-4-yl)pyridin-3-yl)oxy)methyl)piperidine-1-carboxylate (<b>11</b>) (50 mg, 0.139 mmol) and 1-bromo-2-fluorobenzene (30 mg, 0.17 mmol). HPLC purification using acidic method afforded the title compound (26 mg, 41%) as a TFA salt. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.95 (d, <i>J</i> = 2.5 Hz, 1H), 8.66 (bs, 1H), 8.61 (s, 1H), 8.58 (s, 1H), 8.43 (d, <i>J</i> = 5.4 Hz, 1H), 8.46–8.35 (bs, 1H), 8.10 (d, <i>J</i> = 5.5 Hz, 1H), 7.90 (td, <i>J</i> = 7.9, 1.6 Hz, 1H), 7.57–7.48 (m, 2H), 7.47–7.39 (m, 1H), 4.23 (d, <i>J</i> = 6.4 Hz, 2H), 3.41–3.31 (m, 2H), 2.97 (q, <i>J</i> = 11.3 Hz, 2H), 2.36–2.24 (m, 1H), 2.05–1.96 (m, 2H), 1.60–1.45 (m, 2H). HRMS calcd for C<sub>20</sub>H<sub>22</sub>FN<sub>4</sub>O [M + H]<sup>+</sup> = 353.1778. Found [M + H]<sup>+</sup> = 353.1240.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-(4-(4-(3-(Piperidin-4-ylmethoxy)pyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)morpholine (<b>12c</b>)</h3><div class="NLM_p last"><b>12c</b> was prepared according to general procedure 3 using <i>tert</i>-butyl 4-(((4-(1<i>H</i>-pyrazol-4-yl)pyridin-3-yl)oxy)methyl)piperidine-1-carboxylate (<b>11</b>) (50 mg, 0.139 mmol) and 4-(4-bromophenyl)morpholine (43.9 mg, 0.181 mmol, 1.3 equiv). HPLC purification using basic method afforded the title compound (23 mg, 40%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.90 (s, 1H), 8.41 (s, 1H), 8.30 (s, 1H), 8.20 (d, <i>J</i> = 5.0 Hz, 1H), 7.75–7.70 (m, 3H), 7.13–7.06 (m, 2H), 4.09 (d, <i>J</i> = 6.0 Hz, 2H), 3.84–3.70 (m, 4H), 3.22–3.14 (m, 4H), 3.00 (dt, <i>J</i> = 12.2, 3.3 Hz, 2H), 2.55 (dd, <i>J</i> = 11.9, 2.6 Hz, 2H), 1.99 (bs, 1H), 1.82–1.68 (m, 2H), 1.29 (qd, <i>J</i> = 12.1, 4.1 Hz, 2H). HRMS calcd for C<sub>24</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]+ = 420.2400. Found [M + H]<sup>+</sup> = 420.2387.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 4-(3-Fluoro-4-(4-(3-(piperidin-4-ylmethoxy)pyridin-4-yl)-1<i>H</i>-pyrazol-1-yl)phenyl)morpholine (<b>12d</b>)</h3><div class="NLM_p last"><b>12d</b> was prepared according to general procedure 3 using <i>tert</i>-butyl 4-(((4-(1<i>H</i>-pyrazol-4-yl)pyridin-3-yl)oxy)methyl)piperidine-1-carboxylate (<b>11</b>) (100 mg, 0.28 mmol) and 4-(4-bromo-3-fluorophenyl)morpholine (94 mg, 0.36 mmol). HPLC purification using basic method afforded the title compound (6 mg, 5%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.65 (d, <i>J</i> = 2.4 Hz, 1H), 8.41 (s, 1H), 8.38 (s, 1H), 8.19 (d, <i>J</i> = 4.9 Hz, 1H), 7.74 (d, <i>J</i> = 4.9 Hz, 1H), 7.65 (t, <i>J</i> = 9.1 Hz, 1H), 7.04 (dd, <i>J</i> = 15.2, 2.6 Hz, 1H), 6.93 (dd, <i>J</i> = 9.1, 2.6 Hz, 1H), 4.08 (d, <i>J</i> = 6.3 Hz, 2H), 3.77–3.71 (m, 4H), 3.24–3.19 (m, 4H), 3.02–2.95 (m, 2H), 2.6–2.4 (m, 2H) (partially overlaps with solvent signal), 2.04–1.91 (m, 1H), 1.81–1.72 (m, 2H), 1.30–1.17 (m, 2H). HRMS calcd for C<sub>24</sub>H<sub>29</sub>FN<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup> = 438.2305. Found [M + H]<sup>+</sup> = 438.2281.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 4-Bromo-1-phenyl-1<i>H</i>-pyrazole (<b>14</b>)</h3><div class="NLM_p last">To 1-phenylpyrazole (1.00 g, 6.94 mmol) in acetic acid (10 mL) was added bromine (0.360 mL, 6.99 mmol) in acetic acid (10.00 mL). This mixture was heated to 100 °C in a thick-walled screw cap reaction vessel for 16 h. The material was cooled to ambient temperature, poured into ice in a 500 mL beaker, and excess saturated aqueous NaHCO<sub>3</sub> was added until all the acetic acid had been quenched. EtOAc (100 mL) was added, and the layers were separated. The aqueous layer was extracted (50 mL × 2) with EtOAc, and the combined organic layers were washed with brine and dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure to give a crude solid. Purification via flash column chromatography (SiO<sub>2</sub>, 12g, 10–90% EtOAc/heptane over 25 min) gave the title compound as a white solid (373 mg, 24%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.80 (s, 1H), 7.88 (s, 1H), 7.86–7.79 (m, 2H), 7.56–7.46 (m, 2H), 7.40–7.29 (m, 1H). MS calcd for C<sub>9</sub>H<sub>8</sub>BrN<sub>2</sub> [M + H]<sup>+</sup> = 223.0. Found [M + H]<sup>+</sup> = 223.2.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 3-Fluoro-4-(1-phenyl-1<i>H</i>-pyrazol-4-yl)pyridine (<b>15</b>)</h3><div class="NLM_p last">A combined suspension of 4-bromo-1-phenyl-1<i>H</i>-pyrazole (<b>14</b>) (373 mg, 1.67 mmol), 3-fluoro-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine (411 mg, 1.84 mmol), tripotassium phosphate (781 mg, 3.68 mmol) and (Pd(amphos)Cl<sub>2</sub>) (68 mg, 0.096 mmol) in a mixture of dioxane (10 mL) and water (1.0 mL) was degassed under nitrogen, then capped and heated to 95 °C for 4 h. The reaction mixture was cooled and diluted with EtOAc, washed with water (2×), brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to an oil. Purification by flash silica gel chromatography (12 g, EtOAc/heptane) gave the title compound as a light yellow solid (173 mg, 43%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.14 (s, 1H), 8.64 (d, <i>J</i> = 2.8 Hz, 1H), 8.46 (dd, <i>J</i> = 5.0, 1.3 Hz, 1H), 8.37 (d, <i>J</i> = 1.9 Hz, 1H), 7.98–7.88 (m, 3H), 7.61–7.51 (m, 2H), 7.43–7.32 (m, 1H). HRMS calcd for C<sub>14</sub>H<sub>11</sub>FN<sub>3</sub> [M + H]<sup>+</sup> = 240.0937. Found [M + H]<sup>+</sup> = 240.0948.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Biochemical Activities</h3><div class="NLM_p last">MELK kinase activity was determined using MELK protein containing both kinase domain and UBA domain (or full-length MELK, as noted) and KinEASE STK S1 peptide (Cisbio HTRF) as substrate. Phosphorylation of the peptide was measured by homogeneous time-resolved fluorescence technology using XL665 labeled streptavidin and anti-phosphoserine antibody conjugated to Eu<sup>3+</sup> cryptate (Cisbio HTRF). For IC<sub>50</sub> measurement, the compounds were serially diluted and incubated with MELK protein for 20 min at room temperature before being added to a reaction mixture. The final reaction conditions were 3 pM MELK protein, 300 nM peptide substrate, 20 μM ATP, 50 mM Hepes (pH 7.5), 5 mM MgCl<sub>2</sub>, 2 mM DTT, and 1% DMSO. The reaction was stopped in 20 min by adding equal volume of streptavidin-XL665/antibody in detection buffer (CisBio HTRF). Following incubation at room temperature for 60 min, time-resolved fluorescence (Ex, 337; dual Em, 665/620) was measured by Wallac EnVision multilabel reader (PerkinElmer). IC<sub>50</sub> values were calculated by nonlinear four-parameter fitting equations.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Cell Lines and Culture Conditions</h3><div class="NLM_p last">MDA-MB-468 and MCF7 cell lines were obtained from the ATCC and cultured in RPMI and EMEM, respectively, supplemented with 10% FBS and <span class="smallcaps smallerCapital">l</span>-glutamine. MDA-MB-468 and MCF7 shRNA stable cell lines were generated following shRNA lentiviral infection and selection.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Proliferation Assay</h3><div class="NLM_p last">MDA-MB-468 and MCF7 cells were seeded in growth medium into 96-well plates at 1000 and 4000 cells/well, respectively. Sixteen hours after plating, compounds were added to obtain the indicated concentrations, in triplicate, and incubated for 7 days. For each well, ATPLite reagent (PerkinElmer) was added and incubated according to the manufacturer’s instructions. Luminescence was measured on an Envision 2104 multilabel plate reader (PerkinElmer).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> FACS Cell Cycle Analysis</h3><div class="NLM_p last">At harvest time point, growth media, PBS wash, and trypsinized cells from each condition were combined to one conical tube. 0.5 × 10<sup>6</sup> cells were transferred to 15 mL tube and pelleted at 1100 rpm for 5 min. Cells were washed twice with PBS containing 2% FBS and then resuspended in 0.3 mL of PBS containing 2% FBS. Cells were fixed by adding 3 mL of −20 °C prechilled 70% ethanol dropwise while vortexing and then stored at 4 °C overnight. Before staining, fixed cells were pelleted at 1100 rpm for 5 min at 4 °C and then washed twice with 0.5% BSA in PBS. The pellet was resuspended and stained with 200–500 μL of propidium iodide (50 μg/mL PI Biosure 1021) solution containing 10 μg/mL DNase-free RNase A (Roche 11579681001). After a 30 min incubation at room temperature, the samples were run though a BD FACSCanto analyzer with proper gating. At least 10 000 single cells were collected. The data were analyzed with the FlowJo software.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Crystallography</h3><div class="NLM_p last">MELK (3-330) with a carboxy-terminal six-histidine tag was expressed in <i>E. coli</i> and subsequently purified via Ni-affinity resin, anion exchange, and gel filtration. The pure protein was concentrated to approximately 5 mg/mL in 20 mM Tris, pH 8.0, 260 mM NaCl, 2 mM TCEP, and 5% (v/v) glycerol. Apo MELK crystallized from drops containing equal volume of protein and reservoir solution containing 100 mM Hepes, pH 7.6, 0.2 M NaCl, 4.5–16.5% PEG 3350 at 4 °C. For soaks, crystals were harvested and transferred to drops containing 0.25–1 mM compound in 100 mM Hepes, pH 7.6, 0.2 M NaCl, 15–25% PEG 3350 for 12–36 h. For cocrystallization experiments, the protein was incubated with compound for 30 min prior to mixing with the reservoir solution and then crystallized and harvested as described above. Crystals were transferred to cryoprotection solution containing an additional 15–25% glycerol and flash cooled in liquid nitrogen. Data were collected at 100 K using a Pilatus 6M detector and synchrotron radiation (λ = 1.0000 Å) at the IMCA-CAT beamline 17-ID of the Advanced Photon Source at Argonne National Laboratory or at the X10SA beamline of Swiss Light Source at Paul Scherrer Institut and processed using Autoproc (Global Phasing Ltd.). MELK/ligand cocrystals (obtained from soaking or cocrystallization) are in the space group <i>P</i>212121 with one molecule in the asymmetric unit and diffract to a typical resolution of 1.8–2 Å. Structures were solved by molecular replacement using an internal MELK apo structure originally determined by starting with a MARK3 homology model (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QNJ">2QNJ</a>). The model was built using iterative cycles of model building in COOT<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and refined in BUSTER (Global Phasing Ltd.). Individual <i>B</i>-factors were refined using an overall anisotropic <i>B</i>-factor refinement along with a bulk solvent correction. Data collection and refinement statistics are excellent. Structures obtained using cocrystallization and soaking methods with the same compound were identical. The structure factors and coordinates have been deposited in the RCSB PDB with accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH8">5IH8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH9">5IH9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHA">5IHA</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHC">5IHC</a>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Mouse Pharmacokinetic Studies</h3><div class="NLM_p last">All animal related procedures were conducted under a Novartis IACUC approved protocol in compliance with Animal Welfare Act regulations and the Guide for the Care and Use of Laboratory Animals. Mice were housed in a temperature- and humidity-controlled animal facility with ad libitum access to food and water and acclimated for at least 3 days before experimental manipulation and with a 12 h light/12 h dark cycle. For pharmacokinetic studies described in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, the intravenous and oral dose was prepared in a solution containing 5% ethanol, 100% PG, 5% CremophorEL, and 80% PBS. The subcutaneous dose was formulated in 10% PG and 25% (20%, v/v) Solutol. Plasma samples were collected at specified time points and stored frozen (−20 °C) until compound analysis. An LC–MS/MS method was used to quantitate MELK8a and <b>12a</b> drug levels in plasma. All pharmacokinetic (PK) parameters were derived from concentration–time data by noncompartmental analyses using WinNonlin Phoenix, version 6.4 (Certara, St. Louis, MO).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i51"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00052">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78138" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78138" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00052" class="ext-link">10.1021/acs.jmedchem.6b00052</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Genetic knockdown of MELK by two different MELK-targeting shRNAs in MDA-MB-468, MCF-7 (colony formation assay, immunoblotting, quantitative RT-PCR, cell cycle effect), quantitation of MELK expression levels in MDA-MB-468 and MCF-7, IC<sub>50</sub> curves for <b>8a</b> and <b>8b</b>, mouse maximum-tolerated dose (MTD) and PK studies (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_001.csv">jm6b00052_si_001.csv (1.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf">jm6b00052_si_002.pdf (1.46 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH8">5IH8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH9">5IH9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHA">5IHA</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHC">5IHC</a> correspond respectively to <b>1a</b>, <b>8a</b>, <b>8f</b>, and <b>12b</b>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b00052" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44384" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44384" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">B. Barry Touré</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dcbea8b3a9aeb99caeb9b0bda5a8a4f2bfb3b1"><span class="__cf_email__" data-cfemail="95f7e1fae0e7f0d5e7f0f9f4ece1edbbf6faf8">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Giraldes</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Troy Smith</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth R. Sprague</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yaping Wang</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon Mathieu</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhuoliang Chen</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuji Mishina</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yun Feng</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Yan-Neale</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Subarna Shakya</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongshu Chen</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew Meyer</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Puleo</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">J. Tres Brazell</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher Straub</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Sage</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kirk Wright</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanqiu Yuan</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Chen</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jose Duca</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sean Kim</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Tian</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eric Martin</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kristen Hurov</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenlin Shao</span> - <span class="hlFld-Affiliation affiliation">Novartis
Institutes for
Biomedical Research, 250 Massachusetts Avenue, Cambridge, Massachusetts 02139, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The authors declare financial interests as current or former Novartis employees and shareholders.<br /></br></div></li></ul></div><div class="ack" id="ACK-d198e3333-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i58">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65023" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65023" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Drs. Dallas Bednarczyk and Sebastien Ronseaux for mechanistic PK studies and Caco-2 permeability data. Use of the IMCA-CAT beamline 17-ID at the Advanced Photon Source was supported by the companies of the Industrial Macromolecular Crystallography Association through a contract with Hauptman-Woodward Medical Research Institute. Use of the Advanced Photon Source was supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences, under Contract DE-AC02-06CH11357. We acknowledge the Paul Scherrer Institut, Villigen, Switzerland, for provision of synchrotron radiation beamtime at beamline X10SA of the SLS. The synthetic route development and scale-up of <b>5</b> were done at Pharmacore.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i59" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i59"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i60" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i60"> Abbreviations Used</h2><tr><td class="NLM_term">MELK</td><td class="NLM_def"><p class="first last">maternal embryonic leucine zipper kinase</p></td></tr><tr><td class="NLM_term">shRNA</td><td class="NLM_def"><p class="first last">short hairpin ribonucleic acid</p></td></tr><tr><td class="NLM_term">SNF1</td><td class="NLM_def"><p class="first last">sucrose nonfermenting 1</p></td></tr><tr><td class="NLM_term">AMPK</td><td class="NLM_def"><p class="first last">adenosine monophosphate-activated protein kinase</p></td></tr><tr><td class="NLM_term">PARP</td><td class="NLM_def"><p class="first last">poly ADP ribose polymerase</p></td></tr><tr><td class="NLM_term">VS</td><td class="NLM_def"><p class="first last">virtual screening</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">DBAD</td><td class="NLM_def"><p class="first last">di-<i>tert</i>-butyl azodicarboxylate</p></td></tr><tr><td class="NLM_term"><i>P</i><sub>app</sub></td><td class="NLM_def"><p class="first last">apparent permeability</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">extraction ratio</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">MBTE</td><td class="NLM_def"><p class="first last">methyl <i>tert</i>-butyl ether</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78547" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78547" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 24 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Weiss, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span><span class="refDoi"> DOI: 10.1038/nchembio1207-739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1038%2Fnchembio1207-739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=18007642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=739-744&author=W.+A.+Weissauthor=S.+S.+Taylorauthor=K.+M.+Shokat&title=Recognizing+and+exploiting+differences+between+RNAi+and+small-molecule+inhibitors&doi=10.1038%2Fnchembio1207-739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span></div><div class="casAuthors">Weiss, William A.; Taylor, Stephen S.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The biol. of RNA interference has greatly facilitated anal. of loss-of-function phenotypes, but correlating these phenotypes with small-mol. inhibition profiles is not always straightforward.  The authors examine the rationale of comparing RNA interference to pharmacol. intervention in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JPzqVyshS7Vg90H21EOLACvtfcHk0ljNcgnSCRY_RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO&md5=9a26c990b358a6b2c717d1e361cac2da</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1207-739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1207-739%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DRecognizing%2520and%2520exploiting%2520differences%2520between%2520RNAi%2520and%2520small-molecule%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnchembio1207-739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span> </span><span class="NLM_article-title">Target (In)Validation: A Critical, Sometimes Unheralded, Role of Modern Medicinal Chemistry</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">618</span><span class="NLM_x">–</span> <span class="NLM_lpage">621</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=618-621&author=R.+F.+Sweis&title=Target+%28In%29Validation%3A+A+Critical%2C+Sometimes+Unheralded%2C+Role+of+Modern+Medicinal+Chemistry&doi=10.1021%2Facsmedchemlett.5b00183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00183%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26atitle%3DTarget%2520%2528In%2529Validation%253A%2520A%2520Critical%252C%2520Sometimes%2520Unheralded%252C%2520Role%2520of%2520Modern%2520Medicinal%2520Chemistry%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D618%26epage%3D621%26doi%3D10.1021%2Facsmedchemlett.5b00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Lizcano, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goransson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boudeau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawley, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Udd, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makela, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardie, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alessi, D. R.</span><span> </span><span class="NLM_article-title">LKB1 is a master kinase that activates 13 kinases of the AMPK subfamily, including MARK/PAR-1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">833</span><span class="NLM_x">–</span> <span class="NLM_lpage">843</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7600110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1038%2Fsj.emboj.7600110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=14976552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFCjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=833-843&author=J.+M.+Lizcanoauthor=O.+Goranssonauthor=R.+Tothauthor=M.+Deakauthor=N.+A.+Morriceauthor=J.+Boudeauauthor=S.+A.+Hawleyauthor=L.+Uddauthor=T.+P.+Makelaauthor=D.+G.+Hardieauthor=D.+R.+Alessi&title=LKB1+is+a+master+kinase+that+activates+13+kinases+of+the+AMPK+subfamily%2C+including+MARK%2FPAR-1&doi=10.1038%2Fsj.emboj.7600110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">LKB1 is a master kinase that activates 13 kinases of the AMPK subfamily, including MARK/PAR-1</span></div><div class="casAuthors">Lizcano, Jose M.; Goeransson, Olga; Toth, Rachel; Deak, Maria; Morrice, Nick A.; Boudeau, Jerome; Hawley, Simon A.; Udd, Lina; Maekelae, Tomi P.; Hardie, D. Grahame; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">833-843</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We recently demonstrated that the LKB1 tumor suppressor kinase, in complex with the pseudokinase STRAD and the scaffolding protein MO25, phosphorylates and activates AMP-activated protein kinase (AMPK).  A total of 12 human kinases (NUAK1, NUAK2, BRSK1, BRSK2, QIK, QSK, SIK, MARK1, MARK2, MARK3, MARK4 and MELK) are related to AMPK.  Here we demonstrate that LKB1 can phosphorylate the T-loop of all the members of this subfamily, apart from MELK, increasing their activity >50-fold.  LKB1 catalytic activity and the presence of MO25 and STRAD are required for activation.  Mutation of the T-loop Thr phosphorylated by LKB1 to Ala prevented activation, while mutation to glutamate produced active forms of many of the AMPK-related kinases.  Activities of endogenous NUAK2, QIK, QSK, SIK, MARK1, MARK2/3 and MARK4 were markedly reduced in LKB1-deficient cells.  Neither LKB1 activity nor that of AMPK-related kinases was stimulated by phenformin or AICAR, which activate AMPK.  Our results show that LKB1 functions as a master upstream protein kinase, regulating AMPK-related kinases as well as AMPK.  Between them, these kinases may mediate the physiol. effects of LKB1, including its tumor suppressor function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzr93XYu3yFbVg90H21EOLACvtfcHk0ljmV4MpM-4N0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFCjtbs%253D&md5=1aab7e4182fae41b3597b73baa5f0b7d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600110%26sid%3Dliteratum%253Aachs%26aulast%3DLizcano%26aufirst%3DJ.%2BM.%26aulast%3DGoransson%26aufirst%3DO.%26aulast%3DToth%26aufirst%3DR.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DBoudeau%26aufirst%3DJ.%26aulast%3DHawley%26aufirst%3DS.%2BA.%26aulast%3DUdd%26aufirst%3DL.%26aulast%3DMakela%26aufirst%3DT.%2BP.%26aulast%3DHardie%26aufirst%3DD.%2BG.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DLKB1%2520is%2520a%2520master%2520kinase%2520that%2520activates%252013%2520kinases%2520of%2520the%2520AMPK%2520subfamily%252C%2520including%2520MARK%252FPAR-1%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D833%26epage%3D843%26doi%3D10.1038%2Fsj.emboj.7600110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Beullens, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vancauwenbergh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Derua, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceulemans, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waelkens, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollen, M.</span><span> </span><span class="NLM_article-title">Substrate specificity and activity regulation of protein kinase MELK</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">40003</span><span class="NLM_x">–</span> <span class="NLM_lpage">40011</span><span class="refDoi"> DOI: 10.1074/jbc.M507274200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1074%2Fjbc.M507274200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=40003-40011&author=M.+Beullensauthor=S.+Vancauwenberghauthor=N.+Morriceauthor=R.+Deruaauthor=H.+Ceulemansauthor=E.+Waelkensauthor=M.+Bollen&title=Substrate+specificity+and+activity+regulation+of+protein+kinase+MELK&doi=10.1074%2Fjbc.M507274200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M507274200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M507274200%26sid%3Dliteratum%253Aachs%26aulast%3DBeullens%26aufirst%3DM.%26aulast%3DVancauwenbergh%26aufirst%3DS.%26aulast%3DMorrice%26aufirst%3DN.%26aulast%3DDerua%26aufirst%3DR.%26aulast%3DCeulemans%26aufirst%3DH.%26aulast%3DWaelkens%26aufirst%3DE.%26aulast%3DBollen%26aufirst%3DM.%26atitle%3DSubstrate%2520specificity%2520and%2520activity%2520regulation%2520of%2520protein%2520kinase%2520MELK%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D40003%26epage%3D40011%26doi%3D10.1074%2Fjbc.M507274200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Gray, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jubb, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hogue, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowd, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kljavin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frantz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Sauvage, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, D. P.</span><span> </span><span class="NLM_article-title">Maternal embryonic leucine zipper kinase/murine protein serine-threonine kinase 38 is a promising therapeutic target for multiple cancers</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">9751</span><span class="NLM_x">–</span> <span class="NLM_lpage">9761</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-04-4531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1158%2F0008-5472.CAN-04-4531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=16266996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKktL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=9751-9761&author=D.+Grayauthor=A.+M.+Jubbauthor=D.+Hogueauthor=P.+Dowdauthor=N.+Kljavinauthor=S.+Yiauthor=W.+Baiauthor=G.+Frantzauthor=Z.+Zhangauthor=H.+Koeppenauthor=F.+J.+de+Sauvageauthor=D.+P.+Davis&title=Maternal+embryonic+leucine+zipper+kinase%2Fmurine+protein+serine-threonine+kinase+38+is+a+promising+therapeutic+target+for+multiple+cancers&doi=10.1158%2F0008-5472.CAN-04-4531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Maternal embryonic leucine zipper kinase/murine protein serine-threonine kinase 38 is a promising therapeutic target for multiple cancers</span></div><div class="casAuthors">Gray, Daniel; Jubb, Adrian M.; Hogue, Deborah; Dowd, Patrick; Kljavin, Noelyn; Yi, Sothy; Bai, Wei; Frantz, Gretchen; Zhang, Zemin; Koeppen, Hartmut; de Sauvage, Frederic J.; Davis, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9751-9761</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">To identify genes that could serve as targets for novel cancer therapeutics, we used a bioinformatic anal. of microarray data comparing gene expression between normal and tumor-derived primary human tissues.  From this approach, we have found that maternal embryonic leucine zipper kinase (Melk), a member of the AMP serine/threonine kinase family, exhibits multiple features consistent with the potential utility of this gene as an anticancer target.  An oligonucleotide microarray anal. of multiple human tumor samples and cell lines suggests that Melk expression is frequently elevated in cancer relative to normal tissues, a pattern confirmed by quant. reverse transcription-PCR and Western blotting of selected primary tumor samples.  In situ hybridization localized Melk expression to malignant epithelial cells in 96%, 23%, and 13% of colorectal, lung, and ovarian tissue tumor samples, resp.  Expression of this gene is also elevated in spontaneous tumors derived from the ApcMin and Apc1638N murine models of intestinal tumorigenesis.  To begin addressing whether Melk is relevant for tumorigenesis, RNA interference-mediated silencing within human and murine tumor cell lines was done.  We show that Melk knockdown decreases proliferation and anchorage-independent growth in vitro as well as tumor growth in a xenograft model.  Together, these results suggest that Melk may provide a growth advantage for neoplastic cells and, therefore, inactivation may be therapeutically beneficial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_9K-J3ofrbrVg90H21EOLACvtfcHk0ljBevBEuwBXUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKktL7F&md5=9d923595fbdf3b522d22c1329ba634c9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-04-4531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-04-4531%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DD.%26aulast%3DJubb%26aufirst%3DA.%2BM.%26aulast%3DHogue%26aufirst%3DD.%26aulast%3DDowd%26aufirst%3DP.%26aulast%3DKljavin%26aufirst%3DN.%26aulast%3DYi%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DW.%26aulast%3DFrantz%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3Dde%2BSauvage%26aufirst%3DF.%2BJ.%26aulast%3DDavis%26aufirst%3DD.%2BP.%26atitle%3DMaternal%2520embryonic%2520leucine%2520zipper%2520kinase%252Fmurine%2520protein%2520serine-threonine%2520kinase%252038%2520is%2520a%2520promising%2520therapeutic%2520target%2520for%2520multiple%2520cancers%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D9751%26epage%3D9761%26doi%3D10.1158%2F0008-5472.CAN-04-4531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Gu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banasavadi-Siddegowda, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurt, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, I.</span><span> </span><span class="NLM_article-title">Tumor-specific activation of the C-JUN/MELK pathway regulates glioma stem cell growth in a p53-dependent manner</span> <span class="citation_source-journal">Stem Cells</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">870</span><span class="NLM_x">–</span> <span class="NLM_lpage">881</span><span class="refDoi"> DOI: 10.1002/stem.1322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1002%2Fstem.1322" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=870-881&author=C.+Guauthor=Y.+K.+Banasavadi-Siddegowdaauthor=K.+Joshiauthor=Y.+Nakamuraauthor=H.+Kurtauthor=S.+Guptaauthor=I.+Nakano&title=Tumor-specific+activation+of+the+C-JUN%2FMELK+pathway+regulates+glioma+stem+cell+growth+in+a+p53-dependent+manner&doi=10.1002%2Fstem.1322"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fstem.1322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1322%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DC.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%2BK.%26aulast%3DJoshi%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DKurt%26aufirst%3DH.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DTumor-specific%2520activation%2520of%2520the%2520C-JUN%252FMELK%2520pathway%2520regulates%2520glioma%2520stem%2520cell%2520growth%2520in%2520a%2520p53-dependent%2520manner%26jtitle%3DStem%2520Cells%26date%3D2013%26volume%3D31%26spage%3D870%26epage%3D881%26doi%3D10.1002%2Fstem.1322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Joshi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banasavadi-Siddegowda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kig, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nardini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobol, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornblum, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waclaw, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beullens, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, I.</span><span> </span><span class="NLM_article-title">MELK-dependent FOXM1 phosphorylation is essential for proliferation of glioma stem cells</span> <span class="citation_source-journal">Stem Cells</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1051</span><span class="NLM_x">–</span> <span class="NLM_lpage">1063</span><span class="refDoi"> DOI: 10.1002/stem.1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1002%2Fstem.1358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=1051-1063&author=K.+Joshiauthor=Y.+Banasavadi-Siddegowdaauthor=X.+Moauthor=S.+H.+Kimauthor=P.+Maoauthor=C.+Kigauthor=D.+Nardiniauthor=R.+W.+Sobolauthor=L.+M.+Chowauthor=H.+I.+Kornblumauthor=R.+Waclawauthor=M.+Beullensauthor=I.+Nakano&title=MELK-dependent+FOXM1+phosphorylation+is+essential+for+proliferation+of+glioma+stem+cells&doi=10.1002%2Fstem.1358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fstem.1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1358%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DK.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DMo%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DMao%26aufirst%3DP.%26aulast%3DKig%26aufirst%3DC.%26aulast%3DNardini%26aufirst%3DD.%26aulast%3DSobol%26aufirst%3DR.%2BW.%26aulast%3DChow%26aufirst%3DL.%2BM.%26aulast%3DKornblum%26aufirst%3DH.%2BI.%26aulast%3DWaclaw%26aufirst%3DR.%26aulast%3DBeullens%26aufirst%3DM.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DMELK-dependent%2520FOXM1%2520phosphorylation%2520is%2520essential%2520for%2520proliferation%2520of%2520glioma%2520stem%2520cells%26jtitle%3DStem%2520Cells%26date%3D2013%26volume%3D31%26spage%3D1051%26epage%3D1063%26doi%3D10.1002%2Fstem.1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Rhodes, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshpande, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varambally, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrette, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span> </span><span class="NLM_article-title">Large-scale meta-analysis of cancer microarray data identifies common transcriptional profiles of neoplastic transformation and progression</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">9309</span><span class="NLM_x">–</span> <span class="NLM_lpage">9314</span><span class="refDoi"> DOI: 10.1073/pnas.0401994101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1073%2Fpnas.0401994101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=9309-9314&author=D.+R.+Rhodesauthor=J.+Yuauthor=K.+Shankerauthor=N.+Deshpandeauthor=R.+Varamballyauthor=D.+Ghoshauthor=T.+Barretteauthor=A.+Pandeyauthor=A.+M.+Chinnaiyan&title=Large-scale+meta-analysis+of+cancer+microarray+data+identifies+common+transcriptional+profiles+of+neoplastic+transformation+and+progression&doi=10.1073%2Fpnas.0401994101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0401994101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0401994101%26sid%3Dliteratum%253Aachs%26aulast%3DRhodes%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DShanker%26aufirst%3DK.%26aulast%3DDeshpande%26aufirst%3DN.%26aulast%3DVarambally%26aufirst%3DR.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DBarrette%26aufirst%3DT.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DLarge-scale%2520meta-analysis%2520of%2520cancer%2520microarray%2520data%2520identifies%2520common%2520transcriptional%2520profiles%2520of%2520neoplastic%2520transformation%2520and%2520progression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D9309%26epage%3D9314%26doi%3D10.1073%2Fpnas.0401994101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Pickard, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, I. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedge, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourtada-Maarabouni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. T.</span><span> </span><span class="NLM_article-title">Dysregulated expression of Fau and MELK is associated with poor prognosis in breast cancer</span> <span class="citation_source-journal">Breast Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">R60</span><span class="refDoi"> DOI: 10.1186/bcr2350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1186%2Fbcr2350" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=R60&author=M.+R.+Pickardauthor=A.+R.+Greenauthor=I.+O.+Ellisauthor=C.+Caldasauthor=V.+L.+Hedgeauthor=M.+Mourtada-Maarabouniauthor=G.+T.+Williams&title=Dysregulated+expression+of+Fau+and+MELK+is+associated+with+poor+prognosis+in+breast+cancer&doi=10.1186%2Fbcr2350"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fbcr2350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2350%26sid%3Dliteratum%253Aachs%26aulast%3DPickard%26aufirst%3DM.%2BR.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DCaldas%26aufirst%3DC.%26aulast%3DHedge%26aufirst%3DV.%2BL.%26aulast%3DMourtada-Maarabouni%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DG.%2BT.%26atitle%3DDysregulated%2520expression%2520of%2520Fau%2520and%2520MELK%2520is%2520associated%2520with%2520poor%2520prognosis%2520in%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2009%26volume%3D11%26spage%3DR60%26doi%3D10.1186%2Fbcr2350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunici, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulkadir, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irvin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J. S.</span><span> </span><span class="NLM_article-title">Analysis of gene expression and chemoresistance of CD133+ cancer stem cells in glioblastoma</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="refDoi"> DOI: 10.1186/1476-4598-5-67</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1186%2F1476-4598-5-67" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=17140455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A280%3ADC%252BD28jit1CntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=67&author=G.+Liuauthor=X.+Yuanauthor=Z.+Zengauthor=P.+Tuniciauthor=H.+Ngauthor=I.+R.+Abdulkadirauthor=L.+Luauthor=D.+Irvinauthor=K.+L.+Blackauthor=J.+S.+Yu&title=Analysis+of+gene+expression+and+chemoresistance+of+CD133%2B+cancer+stem+cells+in+glioblastoma&doi=10.1186%2F1476-4598-5-67"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of gene expression and chemoresistance of CD133+ cancer stem cells in glioblastoma</span></div><div class="casAuthors">Liu Gentao; Yuan Xiangpeng; Zeng Zhaohui; Tunici Patrizia; Ng Hiushan; Abdulkadir Iman R; Lu Lizhi; Irvin Dwain; Black Keith L; Yu John S</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Recently, a small population of cancer stem cells in adult and pediatric brain tumors has been identified.  Some evidence has suggested that CD133 is a marker for a subset of leukemia and glioblastoma cancer stem cells.  Especially, CD133 positive cells isolated from human glioblastoma may initiate tumors and represent novel targets for therapeutics.  The gene expression and the drug resistance property of CD133 positive cancer stem cells, however, are still unknown.  RESULTS:  In this study, by FACS analysis we determined the percentage of CD133 positive cells in three primary cultured cell lines established from glioblastoma patients 10.2%, 69.7% and 27.5%, respectively.  We also determined the average mRNA levels of markers associated with neural precursors.  For example, CD90, CD44, CXCR4, Nestin, Msi1 and MELK mRNA on CD133 positive cells increased to 15.6, 5.7, 337.8, 21.4, 84 and 1351 times, respectively, compared to autologous CD133 negative cells derived from cell line No. 66.  Additionally, CD133 positive cells express higher levels of BCRP1 and MGMT mRNA, as well as higher mRNA levels of genes that inhibit apoptosis.  Furthermore, CD133 positive cells were significantly resistant to chemotherapeutic agents including temozolomide, carboplatin, paclitaxel (Taxol) and etoposide (VP16) compared to autologous CD133 negative cells.  Finally, CD133 expression was significantly higher in recurrent GBM tissue obtained from five patients as compared to their respective newly diagnosed tumors.  CONCLUSION:  Our study for the first time provided evidence that CD133 positive cancer stem cells display strong capability on tumor's resistance to chemotherapy.  This resistance is probably contributed by the CD133 positive cell with higher expression of on BCRP1 and MGMT, as well as the anti-apoptosis protein and inhibitors of apoptosis protein families.  Future treatment should target this small population of CD133 positive cancer stem cells in tumors to improve the survival of brain tumor patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSrHS0LIOVgyO98rul4nhSdfW6udTcc2eZsiBwBQnFGY7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28jit1CntA%253D%253D&md5=47d6847d5afff4bc3e663b83f1154e52</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-67&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-67%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DTunici%26aufirst%3DP.%26aulast%3DNg%26aufirst%3DH.%26aulast%3DAbdulkadir%26aufirst%3DI.%2BR.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DIrvin%26aufirst%3DD.%26aulast%3DBlack%26aufirst%3DK.%2BL.%26aulast%3DYu%26aufirst%3DJ.%2BS.%26atitle%3DAnalysis%2520of%2520gene%2520expression%2520and%2520chemoresistance%2520of%2520CD133%252B%2520cancer%2520stem%2520cells%2520in%2520glioblastoma%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D67%26doi%3D10.1186%2F1476-4598-5-67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Ryu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deluca, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alani, R. M.</span><span> </span><span class="NLM_article-title">Comprehensive expression profiling of tumor cell lines identifies molecular signatures of melanoma progression</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e594</span><span class="refDoi"> DOI: 10.1371/journal.pone.0000594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1371%2Fjournal.pone.0000594" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=e594&author=B.+Ryuauthor=D.+S.+Kimauthor=A.+M.+Delucaauthor=R.+M.+Alani&title=Comprehensive+expression+profiling+of+tumor+cell+lines+identifies+molecular+signatures+of+melanoma+progression&doi=10.1371%2Fjournal.pone.0000594"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0000594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0000594%26sid%3Dliteratum%253Aachs%26aulast%3DRyu%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DD.%2BS.%26aulast%3DDeluca%26aufirst%3DA.%2BM.%26aulast%3DAlani%26aufirst%3DR.%2BM.%26atitle%3DComprehensive%2520expression%2520profiling%2520of%2520tumor%2520cell%2520lines%2520identifies%2520molecular%2520signatures%2520of%2520melanoma%2520progression%26jtitle%3DPLoS%2520One%26date%3D2007%26volume%3D2%26spage%3De594%26doi%3D10.1371%2Fjournal.pone.0000594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Marie, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, O. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oba-Shinjo, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camargo, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlotti, C. G.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreira-Filho, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zago, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caballero, O. L.</span><span> </span><span class="NLM_article-title">Maternal embryonic leucine zipper kinase transcript abundance correlates with malignancy grade in human astrocytomas</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">807</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1002/ijc.23189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1002%2Fijc.23189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=17960622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFShs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2008&pages=807-815&author=S.+K.+Marieauthor=O.+K.+Okamotoauthor=M.+Unoauthor=A.+P.+Hasegawaauthor=S.+M.+Oba-Shinjoauthor=T.+Cohenauthor=A.+A.+Camargoauthor=A.+Kosoyauthor=C.+G.+Carlottiauthor=S.+Toledoauthor=C.+A.+Moreira-Filhoauthor=M.+A.+Zagoauthor=A.+J.+Simpsonauthor=O.+L.+Caballero&title=Maternal+embryonic+leucine+zipper+kinase+transcript+abundance+correlates+with+malignancy+grade+in+human+astrocytomas&doi=10.1002%2Fijc.23189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Maternal embryonic leucine zipper kinase transcript abundance correlates with malignancy grade in human astrocytomas</span></div><div class="casAuthors">Marie, Suely K. N.; Okamoto, Oswaldo K.; Uno, Miyuki; Paula, Ana; Hasegawa, G.; Oba-Shinjo, Sueli M.; Cohen, Tzeela; Camargo, Anamaria A.; Kosoy, Ana; Carlotti, Carlos G., Jr.; Toledo, Silvia; Moreira-Filho, Carlos A.; Zago, Marco A.; Simpson, Andrew J.; Caballero, Otavia L.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">807-815</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">We have performed cDNA microarray analyses to identify gene expression differences between highly invasive glioblastoma multiforme (GBM) and typically benign pilocytic astrocytomas (PA).  Despite the significant clin. and pathol. differences between the 2 tumor types, only 63 genes were found to exhibit 2-fold or greater overexpression in GBM as compared to PA.  Forty percent of these genes are related to the regulation of the cell cycle and mitosis.  QT-PCR validation of 6 overexpressed genes: MELK, AUKB, ASPM, PRC1, IL13RA2 and KIAA0101 confirmed at least a 5-fold increase in the av. expression levels in GBM.  Maternal embryonic leucine zipper kinase (MELK) exhibited the most statistically significant difference.  A more detailed investigation of MELK expression was undertaken to study its oncogenic relevance.  In the examn. of more than 100 tumors of the central nervous system, we found progressively higher expression of MELK with astrocytoma grade and a noteworthy uniformity of high level expression in GBM.  Similar level of overexpression was also obsd. in medulloblastoma.  We found neither gene promoter hypomethylation nor amplification to be a factor in MELK expression, but were able to demonstrate that MELK knockdown in malignant astrocytoma cell lines caused a redn. in proliferation and anchorage-independent growth in in vitro assays.  Our results indicate that GBM and PA differ by the expression of surprisingly few genes.  Among them, MELK correlated with malignancy grade in astrocytomas and represents a therapeutic target for the management of the most frequent brain tumors in adult and children.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonxXHYLQpPabVg90H21EOLACvtfcHk0lhXgbPJS3-Q2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFShs70%253D&md5=0682ef28bdcfadc200fecda059046afe</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fijc.23189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.23189%26sid%3Dliteratum%253Aachs%26aulast%3DMarie%26aufirst%3DS.%2BK.%26aulast%3DOkamoto%26aufirst%3DO.%2BK.%26aulast%3DUno%26aufirst%3DM.%26aulast%3DHasegawa%26aufirst%3DA.%2BP.%26aulast%3DOba-Shinjo%26aufirst%3DS.%2BM.%26aulast%3DCohen%26aufirst%3DT.%26aulast%3DCamargo%26aufirst%3DA.%2BA.%26aulast%3DKosoy%26aufirst%3DA.%26aulast%3DCarlotti%26aufirst%3DC.%2BG.%26aulast%3DToledo%26aufirst%3DS.%26aulast%3DMoreira-Filho%26aufirst%3DC.%2BA.%26aulast%3DZago%26aufirst%3DM.%2BA.%26aulast%3DSimpson%26aufirst%3DA.%2BJ.%26aulast%3DCaballero%26aufirst%3DO.%2BL.%26atitle%3DMaternal%2520embryonic%2520leucine%2520zipper%2520kinase%2520transcript%2520abundance%2520correlates%2520with%2520malignancy%2520grade%2520in%2520human%2520astrocytomas%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2008%26volume%3D122%26spage%3D807%26epage%3D815%26doi%3D10.1002%2Fijc.23189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baitsch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lako, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Von, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stegmeier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchison, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, J. J.</span><span> </span><span class="NLM_article-title">MELK is an oncogenic kinase essential for mitotic progression in basal-like breast cancer cells</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e01763</span><span class="refDoi"> DOI: 10.7554/eLife.01763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.7554%2FeLife.01763" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=e01763&author=Y.+Wangauthor=Y.+M.+Leeauthor=L.+Baitschauthor=A.+Huangauthor=Y.+Xiangauthor=H.+Tongauthor=A.+Lakoauthor=T.+Vonauthor=C.+Choiauthor=E.+Limauthor=J.+Minauthor=L.+Liauthor=F.+Stegmeierauthor=R.+Schlegelauthor=M.+J.+Eckauthor=N.+S.+Grayauthor=T.+J.+Mitchisonauthor=J.+J.+Zhao&title=MELK+is+an+oncogenic+kinase+essential+for+mitotic+progression+in+basal-like+breast+cancer+cells&doi=10.7554%2FeLife.01763"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.7554%2FeLife.01763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01763%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DY.%2BM.%26aulast%3DBaitsch%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DA.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DH.%26aulast%3DLako%26aufirst%3DA.%26aulast%3DVon%26aufirst%3DT.%26aulast%3DChoi%26aufirst%3DC.%26aulast%3DLim%26aufirst%3DE.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26atitle%3DMELK%2520is%2520an%2520oncogenic%2520kinase%2520essential%2520for%2520mitotic%2520progression%2520in%2520basal-like%2520breast%2520cancer%2520cells%26jtitle%3DeLife%26date%3D2014%26volume%3D3%26spage%3De01763%26doi%3D10.7554%2FeLife.01763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Chung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takamatsu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatsuguchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kijima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuo, Y.</span><span> </span><span class="NLM_article-title">Development of an orally-administrative MELK-targeting inhibitor that suppresses the growth of various types of human cancer</span> <span class="citation_source-journal">Oncotarget.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1629</span><span class="NLM_x">–</span> <span class="NLM_lpage">1640</span><span class="refDoi"> DOI: 10.18632/oncotarget.790</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.18632%2Foncotarget.790" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1629-1640&author=S.+Chungauthor=H.+Suzukiauthor=T.+Miyamotoauthor=N.+Takamatsuauthor=A.+Tatsuguchiauthor=K.+Uedaauthor=K.+Kijimaauthor=Y.+Nakamuraauthor=Y.+Matsuo&title=Development+of+an+orally-administrative+MELK-targeting+inhibitor+that+suppresses+the+growth+of+various+types+of+human+cancer&doi=10.18632%2Foncotarget.790"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.790%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DMiyamoto%26aufirst%3DT.%26aulast%3DTakamatsu%26aufirst%3DN.%26aulast%3DTatsuguchi%26aufirst%3DA.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DKijima%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DMatsuo%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520an%2520orally-administrative%2520MELK-targeting%2520inhibitor%2520that%2520suppresses%2520the%2520growth%2520of%2520various%2520types%2520of%2520human%2520cancer%26jtitle%3DOncotarget.%26date%3D2012%26volume%3D3%26spage%3D1629%26epage%3D1640%26doi%3D10.18632%2Foncotarget.790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Canevari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Re Depaolini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cucchi, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casale, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrera, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forte, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpinelli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felder, E. R.</span><span> </span><span class="NLM_article-title">Structural insight into maternal embryonic leucine zipper kinase (MELK) conformation and inhibition toward structure-based drug design</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6380</span><span class="NLM_x">–</span> <span class="NLM_lpage">6387</span><span class="refDoi"> DOI: 10.1021/bi4005864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi4005864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6380-6387&author=G.+Canevariauthor=S.+Re+Depaoliniauthor=U.+Cucchiauthor=J.+A.+Bertrandauthor=E.+Casaleauthor=C.+Perreraauthor=B.+Forteauthor=P.+Carpinelliauthor=E.+R.+Felder&title=Structural+insight+into+maternal+embryonic+leucine+zipper+kinase+%28MELK%29+conformation+and+inhibition+toward+structure-based+drug+design&doi=10.1021%2Fbi4005864"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fbi4005864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi4005864%26sid%3Dliteratum%253Aachs%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DRe%2BDepaolini%26aufirst%3DS.%26aulast%3DCucchi%26aufirst%3DU.%26aulast%3DBertrand%26aufirst%3DJ.%2BA.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DPerrera%26aufirst%3DC.%26aulast%3DForte%26aufirst%3DB.%26aulast%3DCarpinelli%26aufirst%3DP.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26atitle%3DStructural%2520insight%2520into%2520maternal%2520embryonic%2520leucine%2520zipper%2520kinase%2520%2528MELK%2529%2520conformation%2520and%2520inhibition%2520toward%2520structure-based%2520drug%2520design%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D6380%26epage%3D6387%26doi%3D10.1021%2Fbi4005864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brehmer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyne, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilissen, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlett, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meerpoel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sommen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linders, J. T.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of type I inhibitors of maternal embryonic leucine zipper kinase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1021/ml5001245</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001245" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=25-30&author=C.+N.+Johnsonauthor=V.+Berdiniauthor=L.+Bekeauthor=P.+Bonnetauthor=D.+Brehmerauthor=J.+E.+Coyleauthor=P.+J.+Dayauthor=M.+Fredericksonauthor=E.+J.+Freyneauthor=R.+A.+Gilissenauthor=C.+C.+Hamlettauthor=S.+Howardauthor=L.+Meerpoelauthor=R.+McMenaminauthor=S.+Patelauthor=D.+C.+Reesauthor=A.+Sharffauthor=F.+Sommenauthor=T.+Wuauthor=J.+T.+Linders&title=Fragment-based+discovery+of+type+I+inhibitors+of+maternal+embryonic+leucine+zipper+kinase&doi=10.1021%2Fml5001245"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fml5001245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001245%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DFreyne%26aufirst%3DE.%2BJ.%26aulast%3DGilissen%26aufirst%3DR.%2BA.%26aulast%3DHamlett%26aufirst%3DC.%2BC.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DMeerpoel%26aufirst%3DL.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSommen%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26atitle%3DFragment-based%2520discovery%2520of%2520type%2520I%2520inhibitors%2520of%2520maternal%2520embryonic%2520leucine%2520zipper%2520kinase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D25%26epage%3D30%26doi%3D10.1021%2Fml5001245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adelinet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brehmer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyne, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilissen, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlett, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meerpoel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mevellec, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasquier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linders, J. T.</span><span> </span><span class="NLM_article-title">Structure-Based Design of Type II Inhibitors Applied to Maternal Embryonic Leucine Zipper Kinase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.1021/ml5001273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1WltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=31-36&author=C.+N.+Johnsonauthor=C.+Adelinetauthor=V.+Berdiniauthor=L.+Bekeauthor=P.+Bonnetauthor=D.+Brehmerauthor=F.+Caloauthor=J.+E.+Coyleauthor=P.+J.+Dayauthor=M.+Fredericksonauthor=E.+J.+Freyneauthor=R.+A.+Gilissenauthor=C.+C.+Hamlettauthor=S.+Howardauthor=L.+Meerpoelauthor=L.+Mevellecauthor=R.+McMenaminauthor=E.+Pasquierauthor=S.+Patelauthor=D.+C.+Reesauthor=J.+T.+Linders&title=Structure-Based+Design+of+Type+II+Inhibitors+Applied+to+Maternal+Embryonic+Leucine+Zipper+Kinase&doi=10.1021%2Fml5001273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Type II Inhibitors Applied to Maternal Embryonic Leucine Zipper Kinase</span></div><div class="casAuthors">Johnson, Christopher N.; Adelinet, Christophe; Berdini, Valerio; Beke, Lijs; Bonnet, Pascal; Brehmer, Dirk; Calo, Frederick; Coyle, Joseph E.; Day, Phillip J.; Frederickson, Martyn; Freyne, Eddy J. E.; Gilissen, Ron A. H. J.; Hamlett, Christopher C. F.; Howard, Steven; Meerpoel, Lieven; Mevellec, Laurence; McMenamin, Rachel; Pasquier, Elisabeth; Patel, Sahil; Rees, David C.; Linders, Joannes T. M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-36</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel Type II kinase inhibitor chemotype has been identified for maternal embryonic leucine zipper kinase (MELK) using structure-based ligand design.  The strategy involved structural characterization of an induced DFG-out pocket by protein-ligand X-ray crystallog. and incorporation of a slender linkage capable of bypassing a large gate-keeper residue, thus enabling design of mols. accessing both hinge and induced pocket regions.  Optimization of an initial hit led to the identification of a low-nanomolar, cell-penetrant Type II inhibitor suitable for use as a chem. probe for MELK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWmY300FJpQ7Vg90H21EOLACvtfcHk0li6XNNE6EO9aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1WltL8%253D&md5=35e87bcfe134c8373da3dc57c3c4e6c5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fml5001273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001273%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DAdelinet%26aufirst%3DC.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DCalo%26aufirst%3DF.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DFreyne%26aufirst%3DE.%2BJ.%26aulast%3DGilissen%26aufirst%3DR.%2BA.%26aulast%3DHamlett%26aufirst%3DC.%2BC.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DMeerpoel%26aufirst%3DL.%26aulast%3DMevellec%26aufirst%3DL.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DPasquier%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26atitle%3DStructure-Based%2520Design%2520of%2520Type%2520II%2520Inhibitors%2520Applied%2520to%2520Maternal%2520Embryonic%2520Leucine%2520Zipper%2520Kinase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D31%26epage%3D36%26doi%3D10.1021%2Fml5001273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Martin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansen, J.</span><span> </span><span class="NLM_article-title">Profile-QSAR: a novel meta-QSAR method that combines activities across the kinase family to accurately predict affinity, selectivity, and cellular activity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1942</span><span class="NLM_x">–</span> <span class="NLM_lpage">1956</span><span class="refDoi"> DOI: 10.1021/ci1005004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+a+novel+meta-QSAR+method+that+combines+activities+across+the+kinase+family+to+accurately+predict+affinity%2C+selectivity%2C+and+cellular+activity&doi=10.1021%2Fci1005004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0ljLiXe2G3I4FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520a%2520novel%2520meta-QSAR%2520method%2520that%2520combines%2520activities%2520across%2520the%2520kinase%2520family%2520to%2520accurately%2520predict%2520affinity%252C%2520selectivity%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956%26doi%3D10.1021%2Fci1005004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Martin, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D. C.</span><span> </span><span class="NLM_article-title">Surrogate AutoShim: predocking into a universal ensemble kinase receptor for three dimensional activity prediction, very quickly, without a crystal structure</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">881</span><span class="refDoi"> DOI: 10.1021/ci700455u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700455u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=873-881&author=E.+J.+Martinauthor=D.+C.+Sullivan&title=Surrogate+AutoShim%3A+predocking+into+a+universal+ensemble+kinase+receptor+for+three+dimensional+activity+prediction%2C+very+quickly%2C+without+a+crystal+structure&doi=10.1021%2Fci700455u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Surrogate AutoShim: Predocking into a Universal Ensemble Kinase Receptor for Three Dimensional Activity Prediction, Very Quickly, without a Crystal Structure</span></div><div class="casAuthors">Martin, Eric J.; Sullivan, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">"Ensemble surrogate AutoShim" is a kinase specific extension of the AutoShim docking method that solves the three traditional limitations of conventional docking: (1) it gives good correlations with affinity, (2) does not require a target protein structure, and (3) for a preprocessed company archive of 1.5 million compds., is as fast as traditional 2D QSAR.  It does require several hundred exptl. IC50 values for each new target.  Original AutoShim adds pharmacophore "shims" to a crystal structure binding site.  An iterative partial least squares (PLS) procedure selects the best pose, while adjusting the shim wts. to reproduce IC50 data.  Surrogate AutoShim adjusts shims in one crystal structure to reproduce IC50 data for a different kinase target.  Ensemble surrogate AutoShim uses 16 structurally diverse kinase crystal structures as a "universal ensemble kinase receptor", suitable for any kinase target.  The 1.5 million member Novartis screening collection has been predocked into the shimmed ensemble, so new kinase models can be built, and the entire corporate archive virtually screened, in hours rather than weeks.  A kinase-biased set of 10 000 compds., that samples the entire corporate archive, has been designed for lead discovery by iterative kinase screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5HSUkdEqTcLVg90H21EOLACvtfcHk0ljLiXe2G3I4FQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D&md5=0fe14c274192ec38b65224cae779cf35</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci700455u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700455u%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DSullivan%26aufirst%3DD.%2BC.%26atitle%3DSurrogate%2520AutoShim%253A%2520predocking%2520into%2520a%2520universal%2520ensemble%2520kinase%2520receptor%2520for%2520three%2520dimensional%2520activity%2520prediction%252C%2520very%2520quickly%252C%2520without%2520a%2520crystal%2520structure%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D873%26epage%3D881%26doi%3D10.1021%2Fci700455u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Furtmann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of three-dimensional activity cliffs formed by kinase inhibitors with different binding modes and cliff mapping of structural analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1021/jm5009264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5009264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=252-264&author=N.+Furtmannauthor=Y.+Huauthor=J.+Bajorath&title=Comprehensive+analysis+of+three-dimensional+activity+cliffs+formed+by+kinase+inhibitors+with+different+binding+modes+and+cliff+mapping+of+structural+analogues&doi=10.1021%2Fjm5009264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm5009264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5009264%26sid%3Dliteratum%253Aachs%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComprehensive%2520analysis%2520of%2520three-dimensional%2520activity%2520cliffs%2520formed%2520by%2520kinase%2520inhibitors%2520with%2520different%2520binding%2520modes%2520and%2520cliff%2520mapping%2520of%2520structural%2520analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D252%26epage%3D264%26doi%3D10.1021%2Fjm5009264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Desai, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzi, J. P.</span><span> </span><span class="NLM_article-title">Importance of parallel vectors and "hydrophobic collapse" of the aligned aromatic rings: discovery of a potent substance P antagonist</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">4263</span><span class="NLM_x">–</span> <span class="NLM_lpage">4266</span><span class="refDoi"> DOI: 10.1021/jm00051a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00051a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=4263-4266&author=M.+C.+Desaiauthor=L.+A.+Vincentauthor=J.+P.+Rizzi&title=Importance+of+parallel+vectors+and+%22hydrophobic+collapse%22+of+the+aligned+aromatic+rings%3A+discovery+of+a+potent+substance+P+antagonist&doi=10.1021%2Fjm00051a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm00051a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00051a001%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DM.%2BC.%26aulast%3DVincent%26aufirst%3DL.%2BA.%26aulast%3DRizzi%26aufirst%3DJ.%2BP.%26atitle%3DImportance%2520of%2520parallel%2520vectors%2520and%2520%2522hydrophobic%2520collapse%2522%2520of%2520the%2520aligned%2520aromatic%2520rings%253A%2520discovery%2520of%2520a%2520potent%2520substance%2520P%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D4263%26epage%3D4266%26doi%3D10.1021%2Fjm00051a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Faller, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spanka, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sergejew, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tschinke, V.</span><span> </span><span class="NLM_article-title">Improving the oral bioavailability of the iron chelator HBED by breaking the symmetry of the intramolecular H-bond network</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1467</span><span class="NLM_x">–</span> <span class="NLM_lpage">1475</span><span class="refDoi"> DOI: 10.1021/jm990261n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990261n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvFKnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1467-1475&author=B.+Fallerauthor=C.+Spankaauthor=T.+Sergejewauthor=V.+Tschinke&title=Improving+the+oral+bioavailability+of+the+iron+chelator+HBED+by+breaking+the+symmetry+of+the+intramolecular+H-bond+network&doi=10.1021%2Fjm990261n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Oral Bioavailability of the Iron Chelator HBED by Breaking the Symmetry of the Intramolecular H-Bond Network</span></div><div class="casAuthors">Faller, Bernard; Spanka, Carsten; Sergejew, Thomas; Tschinke, Vincenzo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1467-1475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Physicochem. anal. and Monte Carlo simulations were used to identify structural features which prevent oral absorption of HBED, a potent iron chelator.  In water the dominant conformations of HBED involve the hydrophobic collapse of the two arom. rings.  These conformations are favored in polar media because they expose the polar phenolic hydroxy groups to the solvent and partially shield the nonpolar arom. rings.  In a less polar solvent such as chloroform, a sym. H-bond network between the carboxylates and the amines dominates the conformational space.  This leads to the exposure of the phenolic hydroxy groups to the solvent, which is unfavorable for solvation.  The low soly. of HBED in nonpolar solvents was confirmed exptl. by detn. of the partition coeffs. in octanol, chloroform, and cyclohexane and may explain the poor membrane permeability of this compd.  The high conformational stability which disfavors partitioning into phospholipids is mainly due to the sym. H-bond network.  Potentiometric titrns. of a monoester of HBED in MeOH/water indicate that the protonation sequence was changed compared to that of the parent compd., suggesting that the sym. H-bond network was disrupted.  Conformational anal. in chloroform confirmed that, in contrast to HBED, no sym. interaction between the carboxylate and the nitrogen amines is possible in the half-ester and a variety of conformations which allow partial shielding of the polar phenolic OH groups are energetically possible.  This theor. model predicting a better soly. of the half-esters in nonpolar solvents was supported by the large increase in the partition coeffs. in octanol, chloroform, and cyclohexane measured exptl.  The high absorbability predicted by physicochem. and computer simulation methods was corroborated by in vivo expts. in marmoset monkeys where the monoethyl ester deriv. of HBED was well-absorbed orally while the parent compd. was nearly ineffective in the same model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPO8b-CoWmebVg90H21EOLACvtfcHk0ljOb27OufiYfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvFKnur8%253D&md5=8ee74f968fdb30328a99a413b3ddabff</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm990261n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990261n%26sid%3Dliteratum%253Aachs%26aulast%3DFaller%26aufirst%3DB.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DSergejew%26aufirst%3DT.%26aulast%3DTschinke%26aufirst%3DV.%26atitle%3DImproving%2520the%2520oral%2520bioavailability%2520of%2520the%2520iron%2520chelator%2520HBED%2520by%2520breaking%2520the%2520symmetry%2520of%2520the%2520intramolecular%2520H-bond%2520network%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1467%26epage%3D1475%26doi%3D10.1021%2Fjm990261n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Davezac, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blot, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ducommun, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tassan, J. P.</span><span> </span><span class="NLM_article-title">Human pEg3 kinase associates with and phosphorylates CDC25B phosphatase: a potential role for pEg3 in cell cycle regulation</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">7630</span><span class="NLM_x">–</span> <span class="NLM_lpage">7641</span><span class="refDoi"> DOI: 10.1038/sj.onc.1205870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1038%2Fsj.onc.1205870" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=7630-7641&author=N.+Davezacauthor=V.+Baldinauthor=J.+Blotauthor=B.+Ducommunauthor=J.+P.+Tassan&title=Human+pEg3+kinase+associates+with+and+phosphorylates+CDC25B+phosphatase%3A+a+potential+role+for+pEg3+in+cell+cycle+regulation&doi=10.1038%2Fsj.onc.1205870"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1205870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1205870%26sid%3Dliteratum%253Aachs%26aulast%3DDavezac%26aufirst%3DN.%26aulast%3DBaldin%26aufirst%3DV.%26aulast%3DBlot%26aufirst%3DJ.%26aulast%3DDucommun%26aufirst%3DB.%26aulast%3DTassan%26aufirst%3DJ.%2BP.%26atitle%3DHuman%2520pEg3%2520kinase%2520associates%2520with%2520and%2520phosphorylates%2520CDC25B%2520phosphatase%253A%2520a%2520potential%2520role%2520for%2520pEg3%2520in%2520cell%2520cycle%2520regulation%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D7630%26epage%3D7641%26doi%3D10.1038%2Fsj.onc.1205870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0li0MJgyJaIk9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 27 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Shunsuke Onodera, Takuya Kochi, <span class="NLM_string-name hlFld-ContribAuthor">Fumitoshi Kakiuchi</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of N-Arylpyrazoles by Palladium-Catalyzed Coupling of Aryl Triflates with Pyrazole Derivatives. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (10)
                                     , 6508-6515. <a href="https://doi.org/10.1021/acs.joc.9b00673" title="DOI URL">https://doi.org/10.1021/acs.joc.9b00673</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b00673%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Bof%252BN-Arylpyrazoles%252Bby%252BPalladium-Catalyzed%252BCoupling%252Bof%252BAryl%252BTriflates%252Bwith%252BPyrazole%252BDerivatives%26aulast%3DOnodera%26aufirst%3DShunsuke%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08032019%26date%3D01052019%26date%3D26042019%26volume%3D84%26issue%3D10%26spage%3D6508%26epage%3D6515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">John  Giraldes</span>, <span class="hlFld-ContribAuthor ">Elizabeth R.  Sprague</span>, <span class="hlFld-ContribAuthor ">Subarna  Shakya</span>, <span class="hlFld-ContribAuthor ">Zhuoliang  Chen</span>, <span class="hlFld-ContribAuthor ">Yaping  Wang</span>, <span class="hlFld-ContribAuthor ">Carol  Joud</span>, <span class="hlFld-ContribAuthor ">Simon  Mathieu</span>, <span class="hlFld-ContribAuthor ">Christine Hiu-Tung  Chen</span>, <span class="hlFld-ContribAuthor ">Christopher  Straub</span>, <span class="hlFld-ContribAuthor ">Jose  Duca</span>, <span class="hlFld-ContribAuthor ">Kristen  Hurov</span>, <span class="hlFld-ContribAuthor ">Yanqiu  Yuan</span>, <span class="hlFld-ContribAuthor ">Wenlin  Shao</span>, and <span class="hlFld-ContribAuthor ">B. Barry  Touré</span>  . </span><span class="cited-content_cbyCitation_article-title">“Addition” and “Subtraction”: Selectivity Design for Type II Maternal Embryonic Leucine Zipper Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 2155-2161. <a href="https://doi.org/10.1021/acs.jmedchem.7b00033" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00033%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D%2525E2%252580%25259CAddition%2525E2%252580%25259D%252Band%252B%2525E2%252580%25259CSubtraction%2525E2%252580%25259D%25253A%252BSelectivity%252BDesign%252Bfor%252BType%252BII%252BMaternal%252BEmbryonic%252BLeucine%252BZipper%252BKinase%252BInhibitors%26aulast%3DChen%26aufirst%3DXin%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D09012017%26date%3D24022017%26date%3D09032017%26date%3D10022017%26volume%3D60%26issue%3D5%26spage%3D2155%26epage%3D2161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tetsuro  Yoshimaru</span>, <span class="hlFld-ContribAuthor ">Yusuke  Nakamura</span>, <span class="hlFld-ContribAuthor ">Toyomasa  Katagiri</span>. </span><span class="cited-content_cbyCitation_article-title">Functional genomics for breast cancer drug target discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Human Genetics</span><span> <strong>2021,</strong> <em>57 </em><a href="https://doi.org/10.1038/s10038-021-00962-6" title="DOI URL">https://doi.org/10.1038/s10038-021-00962-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s10038-021-00962-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs10038-021-00962-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Human%2520Genetics%26atitle%3DFunctional%252Bgenomics%252Bfor%252Bbreast%252Bcancer%252Bdrug%252Btarget%252Bdiscovery%26aulast%3DYoshimaru%26aufirst%3DTetsuro%26date%3D2021%26date%3D2021%26volume%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Karthik  Thangaraj</span>, <span class="hlFld-ContribAuthor ">Lavanya  Ponnusamy</span>, <span class="hlFld-ContribAuthor ">Sathan Raj  Natarajan</span>, <span class="hlFld-ContribAuthor ">Ravi  Manoharan</span>. </span><span class="cited-content_cbyCitation_article-title">MELK/MPK38 in cancer: from mechanistic aspects to therapeutic strategies. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (12)
                                     , 2161-2173. <a href="https://doi.org/10.1016/j.drudis.2020.09.029" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.09.029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.09.029%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DMELK%25252FMPK38%252Bin%252Bcancer%25253A%252Bfrom%252Bmechanistic%252Baspects%252Bto%252Btherapeutic%252Bstrategies%26aulast%3DThangaraj%26aufirst%3DKarthik%26date%3D2020%26volume%3D25%26issue%3D12%26spage%3D2161%26epage%3D2173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">László  Petri</span>, <span class="hlFld-ContribAuthor ">Attila  Egyed</span>, <span class="hlFld-ContribAuthor ">Dávid  Bajusz</span>, <span class="hlFld-ContribAuthor ">Tímea  Imre</span>, <span class="hlFld-ContribAuthor ">Anasztázia  Hetényi</span>, <span class="hlFld-ContribAuthor ">Tamás  Martinek</span>, <span class="hlFld-ContribAuthor ">Péter  Ábrányi-Balogh</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">An electrophilic warhead library for mapping the reactivity and accessibility of tractable cysteines in protein kinases. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112836. <a href="https://doi.org/10.1016/j.ejmech.2020.112836" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112836</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112836%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAn%252Belectrophilic%252Bwarhead%252Blibrary%252Bfor%252Bmapping%252Bthe%252Breactivity%252Band%252Baccessibility%252Bof%252Btractable%252Bcysteines%252Bin%252Bprotein%252Bkinases%26aulast%3DPetri%26aufirst%3DL%25C3%25A1szl%25C3%25B3%26date%3D2020%26volume%3D207%26spage%3D112836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lakshmi Reddy  Bollu</span>, <span class="hlFld-ContribAuthor ">Jonathan  Shepherd</span>, <span class="hlFld-ContribAuthor ">Dekuang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yanxia  Ma</span>, <span class="hlFld-ContribAuthor ">William  Tahaney</span>, <span class="hlFld-ContribAuthor ">Corey  Speers</span>, <span class="hlFld-ContribAuthor ">Abhijit  Mazumdar</span>, <span class="hlFld-ContribAuthor ">Gordon B.  Mills</span>, <span class="hlFld-ContribAuthor ">Powel H.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Mutant P53 induces MELK expression by release of wild-type P53-dependent suppression of FOXM1. </span><span class="cited-content_cbyCitation_journal-name">npj Breast Cancer</span><span> <strong>2020,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41523-019-0143-5" title="DOI URL">https://doi.org/10.1038/s41523-019-0143-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41523-019-0143-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41523-019-0143-5%26sid%3Dliteratum%253Aachs%26jtitle%3Dnpj%2520Breast%2520Cancer%26atitle%3DMutant%252BP53%252Binduces%252BMELK%252Bexpression%252Bby%252Brelease%252Bof%252Bwild-type%252BP53-dependent%252Bsuppression%252Bof%252BFOXM1%26aulast%3DBollu%26aufirst%3DLakshmi%2BReddy%26date%3D2020%26date%3D2020%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Navnath S  Gavande</span>, <span class="hlFld-ContribAuthor ">Pamela S  VanderVere-Carozza</span>, <span class="hlFld-ContribAuthor ">Katherine S  Pawelczak</span>, <span class="hlFld-ContribAuthor ">Pamela  Mendoza-Munoz</span>, <span class="hlFld-ContribAuthor ">Tyler L  Vernon</span>, <span class="hlFld-ContribAuthor ">Leslyn A  Hanakahi</span>, <span class="hlFld-ContribAuthor ">Matthew  Summerlin</span>, <span class="hlFld-ContribAuthor ">Joseph R  Dynlacht</span>, <span class="hlFld-ContribAuthor ">Annabelle H  Farmer</span>, <span class="hlFld-ContribAuthor ">Catherine R  Sears</span>, <span class="hlFld-ContribAuthor ">Nawar Al  Nasrallah</span>, <span class="hlFld-ContribAuthor ">Joy  Garrett</span>, <span class="hlFld-ContribAuthor ">John J  Turchi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and development of novel DNA-PK inhibitors by targeting the unique Ku–DNA interaction. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2020,</strong> <em>48 </em>
                                    (20)
                                     , 11536-11550. <a href="https://doi.org/10.1093/nar/gkaa934" title="DOI URL">https://doi.org/10.1093/nar/gkaa934</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa934%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DDiscovery%252Band%252Bdevelopment%252Bof%252Bnovel%252BDNA-PK%252Binhibitors%252Bby%252Btargeting%252Bthe%252Bunique%252BKu%2525E2%252580%252593DNA%252Binteraction%26aulast%3DGavande%26aufirst%3DNavnath%2BS%26date%3D2020%26date%3D2020%26volume%3D48%26issue%3D20%26spage%3D11536%26epage%3D11550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hai  Wei</span>, <span class="hlFld-ContribAuthor ">Eric J.  Lambie</span>, <span class="hlFld-ContribAuthor ">Daniel S.  Osório</span>, <span class="hlFld-ContribAuthor ">Ana X.  Carvalho</span>, <span class="hlFld-ContribAuthor ">Barbara  Conradt</span>, . </span><span class="cited-content_cbyCitation_article-title">PIG-1 MELK-dependent phosphorylation of nonmuscle myosin II promotes apoptosis through CES-1 Snail partitioning. </span><span class="cited-content_cbyCitation_journal-name">PLOS Genetics</span><span> <strong>2020,</strong> <em>16 </em>
                                    (9)
                                     , e1008912. <a href="https://doi.org/10.1371/journal.pgen.1008912" title="DOI URL">https://doi.org/10.1371/journal.pgen.1008912</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pgen.1008912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pgen.1008912%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Genetics%26atitle%3DPIG-1%252BMELK-dependent%252Bphosphorylation%252Bof%252Bnonmuscle%252Bmyosin%252BII%252Bpromotes%252Bapoptosis%252Bthrough%252BCES-1%252BSnail%252Bpartitioning%26aulast%3DWei%26aufirst%3DHai%26date%3D2020%26date%3D2020%26volume%3D16%26issue%3D9%26spage%3De1008912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ian M.  McDonald</span>, <span class="hlFld-ContribAuthor ">Lee M.  Graves</span>. </span><span class="cited-content_cbyCitation_article-title">Enigmatic MELK: The controversy surrounding its complex role in cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2020,</strong> <em>295 </em>
                                    (24)
                                     , 8195-8203. <a href="https://doi.org/10.1074/jbc.REV120.013433" title="DOI URL">https://doi.org/10.1074/jbc.REV120.013433</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.REV120.013433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.REV120.013433%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DEnigmatic%252BMELK%25253A%252BThe%252Bcontroversy%252Bsurrounding%252Bits%252Bcomplex%252Brole%252Bin%252Bcancer%26aulast%3DMcDonald%26aufirst%3DIan%2BM.%26date%3D2020%26volume%3D295%26issue%3D24%26spage%3D8195%26epage%3D8203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George M.  Burslem</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery. </span><span class="cited-content_cbyCitation_journal-name">Cell</span><span> <strong>2020,</strong> <em>181 </em>
                                    (1)
                                     , 102-114. <a href="https://doi.org/10.1016/j.cell.2019.11.031" title="DOI URL">https://doi.org/10.1016/j.cell.2019.11.031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%26atitle%3DProteolysis-Targeting%252BChimeras%252Bas%252BTherapeutics%252Band%252BTools%252Bfor%252BBiological%252BDiscovery%26aulast%3DBurslem%26aufirst%3DGeorge%2BM.%26date%3D2020%26volume%3D181%26issue%3D1%26spage%3D102%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ian M.  McDonald</span>, <span class="hlFld-ContribAuthor ">Gavin D.  Grant</span>, <span class="hlFld-ContribAuthor ">Michael P.  East</span>, <span class="hlFld-ContribAuthor ">Thomas S.K.  Gilbert</span>, <span class="hlFld-ContribAuthor ">Emily M.  Wilkerson</span>, <span class="hlFld-ContribAuthor ">Dennis  Goldfarb</span>, <span class="hlFld-ContribAuthor ">Joshua  Beri</span>, <span class="hlFld-ContribAuthor ">Laura E.  Herring</span>, <span class="hlFld-ContribAuthor ">Cyrus  Vaziri</span>, <span class="hlFld-ContribAuthor ">Jeanette Gowen  Cook</span>, <span class="hlFld-ContribAuthor ">Michael J.  Emanuele</span>, <span class="hlFld-ContribAuthor ">Lee M.  Graves</span>. </span><span class="cited-content_cbyCitation_article-title">Mass spectrometry–based selectivity profiling identifies a highly selective inhibitor of the kinase MELK that delays mitotic entry in cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2020,</strong> <em>295 </em>
                                    (8)
                                     , 2359-2374. <a href="https://doi.org/10.1074/jbc.RA119.011083" title="DOI URL">https://doi.org/10.1074/jbc.RA119.011083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA119.011083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA119.011083%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DMass%252Bspectrometry%2525E2%252580%252593based%252Bselectivity%252Bprofiling%252Bidentifies%252Ba%252Bhighly%252Bselective%252Binhibitor%252Bof%252Bthe%252Bkinase%252BMELK%252Bthat%252Bdelays%252Bmitotic%252Bentry%252Bin%252Bcancer%252Bcells%26aulast%3DMcDonald%26aufirst%3DIan%2BM.%26date%3D2020%26volume%3D295%26issue%3D8%26spage%3D2359%26epage%3D2374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anke  Maes</span>, <span class="hlFld-ContribAuthor ">Ken  Maes</span>, <span class="hlFld-ContribAuthor ">Philip  Vlummens</span>, <span class="hlFld-ContribAuthor ">Hendrik  De Raeve</span>, <span class="hlFld-ContribAuthor ">Julie  Devin</span>, <span class="hlFld-ContribAuthor ">Vanessa  Szablewski</span>, <span class="hlFld-ContribAuthor ">Kim  De Veirman</span>, <span class="hlFld-ContribAuthor ">Eline  Menu</span>, <span class="hlFld-ContribAuthor ">Jerome  Moreaux</span>, <span class="hlFld-ContribAuthor ">Karin  Vanderkerken</span>, <span class="hlFld-ContribAuthor ">Elke  De Bruyne</span>. </span><span class="cited-content_cbyCitation_article-title">Maternal embryonic leucine zipper kinase is a novel target for diffuse large B cell lymphoma and mantle cell lymphoma. </span><span class="cited-content_cbyCitation_journal-name">Blood Cancer Journal</span><span> <strong>2019,</strong> <em>9 </em>
                                    (12)
                                     <a href="https://doi.org/10.1038/s41408-019-0249-x" title="DOI URL">https://doi.org/10.1038/s41408-019-0249-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41408-019-0249-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41408-019-0249-x%26sid%3Dliteratum%253Aachs%26jtitle%3DBlood%2520Cancer%2520Journal%26atitle%3DMaternal%252Bembryonic%252Bleucine%252Bzipper%252Bkinase%252Bis%252Ba%252Bnovel%252Btarget%252Bfor%252Bdiffuse%252Blarge%252BB%252Bcell%252Blymphoma%252Band%252Bmantle%252Bcell%252Blymphoma%26aulast%3DMaes%26aufirst%3DAnke%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicole A.  Perry</span>, <span class="hlFld-ContribAuthor ">Kevin P.  Fialkowski</span>, <span class="hlFld-ContribAuthor ">Tamer S.  Kaoud</span>, <span class="hlFld-ContribAuthor ">Ali I.  Kaya</span>, <span class="hlFld-ContribAuthor ">Andrew L.  Chen</span>, <span class="hlFld-ContribAuthor ">Juliana M.  Taliaferro</span>, <span class="hlFld-ContribAuthor ">Vsevolod V.  Gurevich</span>, <span class="hlFld-ContribAuthor ">Kevin N.  Dalby</span>, <span class="hlFld-ContribAuthor ">T.M.  Iverson</span>. </span><span class="cited-content_cbyCitation_article-title">Arrestin-3 interaction with maternal embryonic leucine-zipper kinase. </span><span class="cited-content_cbyCitation_journal-name">Cellular Signalling</span><span> <strong>2019,</strong> <em>63 </em>, 109366. <a href="https://doi.org/10.1016/j.cellsig.2019.109366" title="DOI URL">https://doi.org/10.1016/j.cellsig.2019.109366</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cellsig.2019.109366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cellsig.2019.109366%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520Signalling%26atitle%3DArrestin-3%252Binteraction%252Bwith%252Bmaternal%252Bembryonic%252Bleucine-zipper%252Bkinase%26aulast%3DPerry%26aufirst%3DNicole%2BA.%26date%3D2019%26volume%3D63%26spage%3D109366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krzysztof P.  Rembacz</span>, <span class="hlFld-ContribAuthor ">Karol M.  Zrubek</span>, <span class="hlFld-ContribAuthor ">Przemyslaw  Golik</span>, <span class="hlFld-ContribAuthor ">Kinga  Michalik</span>, <span class="hlFld-ContribAuthor ">Jozefina  Bogusz</span>, <span class="hlFld-ContribAuthor ">Benedykt  Wladyka</span>, <span class="hlFld-ContribAuthor ">Malgorzata  Romanowska</span>, <span class="hlFld-ContribAuthor ">Grzegorz  Dubin</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal structure of Maternal Embryonic Leucine Zipper Kinase (MELK) in complex with dorsomorphin (Compound C). </span><span class="cited-content_cbyCitation_journal-name">Archives of Biochemistry and Biophysics</span><span> <strong>2019,</strong> <em>671 </em>, 1-7. <a href="https://doi.org/10.1016/j.abb.2019.05.014" title="DOI URL">https://doi.org/10.1016/j.abb.2019.05.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.abb.2019.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.abb.2019.05.014%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Biochemistry%2520and%2520Biophysics%26atitle%3DCrystal%252Bstructure%252Bof%252BMaternal%252BEmbryonic%252BLeucine%252BZipper%252BKinase%252B%252528MELK%252529%252Bin%252Bcomplex%252Bwith%252Bdorsomorphin%252B%252528Compound%252BC%252529%26aulast%3DRembacz%26aufirst%3DKrzysztof%2BP.%26date%3D2019%26volume%3D671%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicolas  Boutard</span>, <span class="hlFld-ContribAuthor ">Aleksandra  Sabiniarz</span>, <span class="hlFld-ContribAuthor ">Klaudia  Czerwińska</span>, <span class="hlFld-ContribAuthor ">Małgorzata  Jarosz</span>, <span class="hlFld-ContribAuthor ">Anna  Cierpich</span>, <span class="hlFld-ContribAuthor ">Ewa  Kolasińska</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Wiklik</span>, <span class="hlFld-ContribAuthor ">Karolina  Gluza</span>, <span class="hlFld-ContribAuthor ">Claude  Commandeur</span>, <span class="hlFld-ContribAuthor ">Anna  Buda</span>, <span class="hlFld-ContribAuthor ">Agata  Stasiowska</span>, <span class="hlFld-ContribAuthor ">Aneta  Bobowska</span>, <span class="hlFld-ContribAuthor ">Mariusz  Galek</span>, <span class="hlFld-ContribAuthor ">Charles-Henry  Fabritius</span>, <span class="hlFld-ContribAuthor ">Marta  Bugaj</span>, <span class="hlFld-ContribAuthor ">Edyta  Palacz</span>, <span class="hlFld-ContribAuthor ">Andrzej  Mazan</span>, <span class="hlFld-ContribAuthor ">Adrian  Zarębski</span>, <span class="hlFld-ContribAuthor ">Karolina  Krawczyńska</span>, <span class="hlFld-ContribAuthor ">Małgorzata  Żurawska</span>, <span class="hlFld-ContribAuthor ">Przemysław  Zawadzki</span>, <span class="hlFld-ContribAuthor ">Mariusz  Milik</span>, <span class="hlFld-ContribAuthor ">Paulina  Węgrzyn</span>, <span class="hlFld-ContribAuthor ">Monika  Dobrzańska</span>, <span class="hlFld-ContribAuthor ">Krzysztof  Brzózka</span>, <span class="hlFld-ContribAuthor ">Piotr  Kowalczyk</span>. </span><span class="cited-content_cbyCitation_article-title">5-Keto-3-cyano-2,4-diaminothiophenes as selective maternal embryonic leucine zipper kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (4)
                                     , 607-613. <a href="https://doi.org/10.1016/j.bmcl.2018.12.051" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.12.051</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.12.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.12.051%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3D5-Keto-3-cyano-2%25252C4-diaminothiophenes%252Bas%252Bselective%252Bmaternal%252Bembryonic%252Bleucine%252Bzipper%252Bkinase%252Binhibitors%26aulast%3DBoutard%26aufirst%3DNicolas%26date%3D2019%26volume%3D29%26issue%3D4%26spage%3D607%26epage%3D613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rachel L.  Mintz</span>, <span class="hlFld-ContribAuthor ">Madeleine A.  Gao</span>, <span class="hlFld-ContribAuthor ">Kahmun  Lo</span>, <span class="hlFld-ContribAuthor ">Yeh-Hsing  Lao</span>, <span class="hlFld-ContribAuthor ">Mingqiang  Li</span>, <span class="hlFld-ContribAuthor ">Kam W.  Leong</span>. </span><span class="cited-content_cbyCitation_article-title">CRISPR Technology for Breast Cancer: Diagnostics, Modeling, and Therapy. </span><span class="cited-content_cbyCitation_journal-name">Advanced Biosystems</span><span> <strong>2018,</strong> <em>2 </em>
                                    (11)
                                     , 1800132. <a href="https://doi.org/10.1002/adbi.201800132" title="DOI URL">https://doi.org/10.1002/adbi.201800132</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adbi.201800132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadbi.201800132%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Biosystems%26atitle%3DCRISPR%252BTechnology%252Bfor%252BBreast%252BCancer%25253A%252BDiagnostics%25252C%252BModeling%25252C%252Band%252BTherapy%26aulast%3DMintz%26aufirst%3DRachel%2BL.%26date%3D2018%26date%3D2018%26volume%3D2%26issue%3D11%26spage%3D1800132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yubao  Wang</span>, <span class="hlFld-ContribAuthor ">Ben B.  Li</span>, <span class="hlFld-ContribAuthor ">Jing  Li</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Roberts</span>, <span class="hlFld-ContribAuthor ">Jean J.  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">A Conditional Dependency on MELK for the Proliferation of Triple-Negative Breast Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">iScience</span><span> <strong>2018,</strong> <em>9 </em>, 149-160. <a href="https://doi.org/10.1016/j.isci.2018.10.015" title="DOI URL">https://doi.org/10.1016/j.isci.2018.10.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.isci.2018.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.isci.2018.10.015%26sid%3Dliteratum%253Aachs%26jtitle%3DiScience%26atitle%3DA%252BConditional%252BDependency%252Bon%252BMELK%252Bfor%252Bthe%252BProliferation%252Bof%252BTriple-Negative%252BBreast%252BCancer%252BCells%26aulast%3DWang%26aufirst%3DYubao%26date%3D2018%26volume%3D9%26spage%3D149%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yubao  Wang</span>, <span class="hlFld-ContribAuthor ">Young-Mi  Li</span>, <span class="hlFld-ContribAuthor ">Lukas  Baitsch</span>, <span class="hlFld-ContribAuthor ">Alan  Huang</span>, <span class="hlFld-ContribAuthor ">Yi  Xiang</span>, <span class="hlFld-ContribAuthor ">Haoxuan  Tong</span>, <span class="hlFld-ContribAuthor ">Ana  Lako</span>, <span class="hlFld-ContribAuthor ">Thanh  Von</span>, <span class="hlFld-ContribAuthor ">Christine  Choi</span>, <span class="hlFld-ContribAuthor ">Elgene  Lim</span>, <span class="hlFld-ContribAuthor ">Junxia  Min</span>, <span class="hlFld-ContribAuthor ">Li  Li</span>, <span class="hlFld-ContribAuthor ">Frank  Stegmeier</span>, <span class="hlFld-ContribAuthor ">Robert  Schlegel</span>, <span class="hlFld-ContribAuthor ">Michael J  Eck</span>, <span class="hlFld-ContribAuthor ">Nathanael S  Gray</span>, <span class="hlFld-ContribAuthor ">Timothy J  Mitchison</span>, <span class="hlFld-ContribAuthor ">Jean J  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Correction: MELK is an oncogenic kinase essential for mitotic progression in basal-like breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">eLife</span><span> <strong>2018,</strong> <em>7 </em><a href="https://doi.org/10.7554/eLife.36414" title="DOI URL">https://doi.org/10.7554/eLife.36414</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7554/eLife.36414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7554%2FeLife.36414%26sid%3Dliteratum%253Aachs%26jtitle%3DeLife%26atitle%3DCorrection%25253A%252BMELK%252Bis%252Ban%252Boncogenic%252Bkinase%252Bessential%252Bfor%252Bmitotic%252Bprogression%252Bin%252Bbasal-like%252Bbreast%252Bcancer%252Bcells%26aulast%3DWang%26aufirst%3DYubao%26date%3D2018%26date%3D2018%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeffrey  Settleman</span>, <span class="hlFld-ContribAuthor ">Charles L  Sawyers</span>, <span class="hlFld-ContribAuthor ">Tony  Hunter</span>. </span><span class="cited-content_cbyCitation_article-title">Challenges in validating candidate therapeutic targets in cancer. </span><span class="cited-content_cbyCitation_journal-name">eLife</span><span> <strong>2018,</strong> <em>7 </em><a href="https://doi.org/10.7554/eLife.32402" title="DOI URL">https://doi.org/10.7554/eLife.32402</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7554/eLife.32402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7554%2FeLife.32402%26sid%3Dliteratum%253Aachs%26jtitle%3DeLife%26atitle%3DChallenges%252Bin%252Bvalidating%252Bcandidate%252Btherapeutic%252Btargets%252Bin%252Bcancer%26aulast%3DSettleman%26aufirst%3DJeffrey%26date%3D2018%26date%3D2018%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shu  Zhou</span>, <span class="hlFld-ContribAuthor ">Guo-Bo  Li</span>, <span class="hlFld-ContribAuthor ">Lin  Luo</span>, <span class="hlFld-ContribAuthor ">Lei  Zhong</span>, <span class="hlFld-ContribAuthor ">Kai  Chen</span>, <span class="hlFld-ContribAuthor ">Hui  Li</span>, <span class="hlFld-ContribAuthor ">Xiao-Juan  Jiang</span>, <span class="hlFld-ContribAuthor ">Qi  Fu</span>, <span class="hlFld-ContribAuthor ">Xin  Long</span>, <span class="hlFld-ContribAuthor ">Jin-ku  Bao</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based discovery of new maternal embryonic leucine zipper kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2018,</strong> <em>16 </em>
                                    (9)
                                     , 1489-1495. <a href="https://doi.org/10.1039/C7OB02344H" title="DOI URL">https://doi.org/10.1039/C7OB02344H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB02344H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB02344H%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DStructure-based%252Bdiscovery%252Bof%252Bnew%252Bmaternal%252Bembryonic%252Bleucine%252Bzipper%252Bkinase%252Binhibitors%26aulast%3DZhou%26aufirst%3DShu%26date%3D2018%26date%3D2018%26volume%3D16%26issue%3D9%26spage%3D1489%26epage%3D1495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Radoslav  Janostiak</span>, <span class="hlFld-ContribAuthor ">Navin  Rauniyar</span>, <span class="hlFld-ContribAuthor ">TuKiet T.  Lam</span>, <span class="hlFld-ContribAuthor ">Jianhong  Ou</span>, <span class="hlFld-ContribAuthor ">Lihua J.  Zhu</span>, <span class="hlFld-ContribAuthor ">Michael R.  Green</span>, <span class="hlFld-ContribAuthor ">Narendra  Wajapeyee</span>. </span><span class="cited-content_cbyCitation_article-title">MELK Promotes Melanoma Growth by Stimulating the NF-κB Pathway. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2017,</strong> <em>21 </em>
                                    (10)
                                     , 2829-2841. <a href="https://doi.org/10.1016/j.celrep.2017.11.033" title="DOI URL">https://doi.org/10.1016/j.celrep.2017.11.033</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2017.11.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2017.11.033%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DMELK%252BPromotes%252BMelanoma%252BGrowth%252Bby%252BStimulating%252Bthe%252BNF-%2525CE%2525BAB%252BPathway%26aulast%3DJanostiak%26aufirst%3DRadoslav%26date%3D2017%26volume%3D21%26issue%3D10%26spage%3D2829%26epage%3D2841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan  Klaeger</span>, <span class="hlFld-ContribAuthor ">Stephanie  Heinzlmeir</span>, <span class="hlFld-ContribAuthor ">Mathias  Wilhelm</span>, <span class="hlFld-ContribAuthor ">Harald  Polzer</span>, <span class="hlFld-ContribAuthor ">Binje  Vick</span>, <span class="hlFld-ContribAuthor ">Paul-Albert  Koenig</span>, <span class="hlFld-ContribAuthor ">Maria  Reinecke</span>, <span class="hlFld-ContribAuthor ">Benjamin  Ruprecht</span>, <span class="hlFld-ContribAuthor ">Svenja  Petzoldt</span>, <span class="hlFld-ContribAuthor ">Chen  Meng</span>, <span class="hlFld-ContribAuthor ">Jana  Zecha</span>, <span class="hlFld-ContribAuthor ">Katrin  Reiter</span>, <span class="hlFld-ContribAuthor ">Huichao  Qiao</span>, <span class="hlFld-ContribAuthor ">Dominic  Helm</span>, <span class="hlFld-ContribAuthor ">Heiner  Koch</span>, <span class="hlFld-ContribAuthor ">Melanie  Schoof</span>, <span class="hlFld-ContribAuthor ">Giulia  Canevari</span>, <span class="hlFld-ContribAuthor ">Elena  Casale</span>, <span class="hlFld-ContribAuthor ">Stefania Re  Depaolini</span>, <span class="hlFld-ContribAuthor ">Annette  Feuchtinger</span>, <span class="hlFld-ContribAuthor ">Zhixiang  Wu</span>, <span class="hlFld-ContribAuthor ">Tobias  Schmidt</span>, <span class="hlFld-ContribAuthor ">Lars  Rueckert</span>, <span class="hlFld-ContribAuthor ">Wilhelm  Becker</span>, <span class="hlFld-ContribAuthor ">Jan  Huenges</span>, <span class="hlFld-ContribAuthor ">Anne-Kathrin  Garz</span>, <span class="hlFld-ContribAuthor ">Bjoern-Oliver  Gohlke</span>, <span class="hlFld-ContribAuthor ">Daniel Paul  Zolg</span>, <span class="hlFld-ContribAuthor ">Gian  Kayser</span>, <span class="hlFld-ContribAuthor ">Tonu  Vooder</span>, <span class="hlFld-ContribAuthor ">Robert  Preissner</span>, <span class="hlFld-ContribAuthor ">Hannes  Hahne</span>, <span class="hlFld-ContribAuthor ">Neeme  Tõnisson</span>, <span class="hlFld-ContribAuthor ">Karl  Kramer</span>, <span class="hlFld-ContribAuthor ">Katharina  Götze</span>, <span class="hlFld-ContribAuthor ">Florian  Bassermann</span>, <span class="hlFld-ContribAuthor ">Judith  Schlegl</span>, <span class="hlFld-ContribAuthor ">Hans-Christian  Ehrlich</span>, <span class="hlFld-ContribAuthor ">Stephan  Aiche</span>, <span class="hlFld-ContribAuthor ">Axel  Walch</span>, <span class="hlFld-ContribAuthor ">Philipp A.  Greif</span>, <span class="hlFld-ContribAuthor ">Sabine  Schneider</span>, <span class="hlFld-ContribAuthor ">Eduard Rudolf  Felder</span>, <span class="hlFld-ContribAuthor ">Juergen  Ruland</span>, <span class="hlFld-ContribAuthor ">Guillaume  Médard</span>, <span class="hlFld-ContribAuthor ">Irmela  Jeremias</span>, <span class="hlFld-ContribAuthor ">Karsten  Spiekermann</span>, <span class="hlFld-ContribAuthor ">Bernhard  Kuster</span>. </span><span class="cited-content_cbyCitation_article-title">The target landscape of clinical kinase drugs. </span><span class="cited-content_cbyCitation_journal-name">Science</span><span> <strong>2017,</strong> <em>358 </em>
                                    (6367)
                                     , eaan4368. <a href="https://doi.org/10.1126/science.aan4368" title="DOI URL">https://doi.org/10.1126/science.aan4368</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/science.aan4368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscience.aan4368%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%26atitle%3DThe%252Btarget%252Blandscape%252Bof%252Bclinical%252Bkinase%252Bdrugs%26aulast%3DKlaeger%26aufirst%3DSusan%26date%3D2017%26date%3D2017%26volume%3D358%26issue%3D6367%26spage%3Deaan4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hai-Tsang  Huang</span>, <span class="hlFld-ContribAuthor ">Hyuk-Soo  Seo</span>, <span class="hlFld-ContribAuthor ">Tinghu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yubao  Wang</span>, <span class="hlFld-ContribAuthor ">Baishan  Jiang</span>, <span class="hlFld-ContribAuthor ">Qing  Li</span>, <span class="hlFld-ContribAuthor ">Dennis L  Buckley</span>, <span class="hlFld-ContribAuthor ">Behnam  Nabet</span>, <span class="hlFld-ContribAuthor ">Justin M  Roberts</span>, <span class="hlFld-ContribAuthor ">Joshiawa  Paulk</span>, <span class="hlFld-ContribAuthor ">Shiva  Dastjerdi</span>, <span class="hlFld-ContribAuthor ">Georg E  Winter</span>, <span class="hlFld-ContribAuthor ">Hilary  McLauchlan</span>, <span class="hlFld-ContribAuthor ">Jennifer  Moran</span>, <span class="hlFld-ContribAuthor ">James E  Bradner</span>, <span class="hlFld-ContribAuthor ">Michael J  Eck</span>, <span class="hlFld-ContribAuthor ">Sirano  Dhe-Paganon</span>, <span class="hlFld-ContribAuthor ">Jean J  Zhao</span>, <span class="hlFld-ContribAuthor ">Nathanael S  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">MELK is not necessary for the proliferation of basal-like breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">eLife</span><span> <strong>2017,</strong> <em>6 </em><a href="https://doi.org/10.7554/eLife.26693" title="DOI URL">https://doi.org/10.7554/eLife.26693</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7554/eLife.26693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7554%2FeLife.26693%26sid%3Dliteratum%253Aachs%26jtitle%3DeLife%26atitle%3DMELK%252Bis%252Bnot%252Bnecessary%252Bfor%252Bthe%252Bproliferation%252Bof%252Bbasal-like%252Bbreast%252Bcancer%252Bcells%26aulast%3DHuang%26aufirst%3DHai-Tsang%26date%3D2017%26date%3D2017%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mary Kathryn  Pitner</span>, <span class="hlFld-ContribAuthor ">Juliana M.  Taliaferro</span>, <span class="hlFld-ContribAuthor ">Kevin N.  Dalby</span>, <span class="hlFld-ContribAuthor ">Chandra  Bartholomeusz</span>. </span><span class="cited-content_cbyCitation_article-title">MELK: a potential novel therapeutic target for TNBC and other aggressive malignancies. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2017,</strong> <em>21 </em>
                                    (9)
                                     , 849-859. <a href="https://doi.org/10.1080/14728222.2017.1363183" title="DOI URL">https://doi.org/10.1080/14728222.2017.1363183</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14728222.2017.1363183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14728222.2017.1363183%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DMELK%25253A%252Ba%252Bpotential%252Bnovel%252Btherapeutic%252Btarget%252Bfor%252BTNBC%252Band%252Bother%252Baggressive%252Bmalignancies%26aulast%3DPitner%26aufirst%3DMary%2BKathryn%26date%3D2017%26date%3D2017%26volume%3D21%26issue%3D9%26spage%3D849%26epage%3D859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chul Min  Park</span>, <span class="hlFld-ContribAuthor ">Vithal B.  Jadhav</span>, <span class="hlFld-ContribAuthor ">Jong-Hwan  Song</span>, <span class="hlFld-ContribAuthor ">Sunkyung  Lee</span>, <span class="hlFld-ContribAuthor ">Hee Young  Won</span>, <span class="hlFld-ContribAuthor ">Sang Un  Choi</span>, <span class="hlFld-ContribAuthor ">You Hwa  Son</span>. </span><span class="cited-content_cbyCitation_article-title">3-Amino-1
              H
              -pyrazolopyridine Derivatives as a Maternal Embryonic Leucine Zipper Kinase Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Bulletin of the Korean Chemical Society</span><span> <strong>2017,</strong> <em>38 </em>
                                    (6)
                                     , 595-602. <a href="https://doi.org/10.1002/bkcs.11129" title="DOI URL">https://doi.org/10.1002/bkcs.11129</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bkcs.11129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbkcs.11129%26sid%3Dliteratum%253Aachs%26jtitle%3DBulletin%2520of%2520the%2520Korean%2520Chemical%2520Society%26atitle%3D3-Amino-1%252BH%252B-pyrazolopyridine%252BDerivatives%252Bas%252Ba%252BMaternal%252BEmbryonic%252BLeucine%252BZipper%252BKinase%252BInhibitor%26aulast%3DPark%26aufirst%3DChul%2BMin%26date%3D2017%26date%3D2017%26volume%3D38%26issue%3D6%26spage%3D595%26epage%3D602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ramakrishna  Edupuganti</span>, <span class="hlFld-ContribAuthor ">Juliana M.  Taliaferro</span>, <span class="hlFld-ContribAuthor ">Qiantao  Wang</span>, <span class="hlFld-ContribAuthor ">Xuemei  Xie</span>, <span class="hlFld-ContribAuthor ">Eun Jeong  Cho</span>, <span class="hlFld-ContribAuthor ">Fnu  Vidhu</span>, <span class="hlFld-ContribAuthor ">Pengyu  Ren</span>, <span class="hlFld-ContribAuthor ">Eric V.  Anslyn</span>, <span class="hlFld-ContribAuthor ">Chandra  Bartholomeusz</span>, <span class="hlFld-ContribAuthor ">Kevin N.  Dalby</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent inhibitor of MELK that inhibits expression of the anti-apoptotic protein Mcl-1 and TNBC cell growth. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (9)
                                     , 2609-2616. <a href="https://doi.org/10.1016/j.bmc.2017.03.018" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.03.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.03.018%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252Binhibitor%252Bof%252BMELK%252Bthat%252Binhibits%252Bexpression%252Bof%252Bthe%252Banti-apoptotic%252Bprotein%252BMcl-1%252Band%252BTNBC%252Bcell%252Bgrowth%26aulast%3DEdupuganti%26aufirst%3DRamakrishna%26date%3D2017%26volume%3D25%26issue%3D9%26spage%3D2609%26epage%3D2616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ann  Lin</span>, <span class="hlFld-ContribAuthor ">Christopher J  Giuliano</span>, <span class="hlFld-ContribAuthor ">Nicole M  Sayles</span>, <span class="hlFld-ContribAuthor ">Jason M  Sheltzer</span>. </span><span class="cited-content_cbyCitation_article-title">CRISPR/Cas9 mutagenesis invalidates a putative cancer dependency targeted in on-going clinical trials. </span><span class="cited-content_cbyCitation_journal-name">eLife</span><span> <strong>2017,</strong> <em>6 </em><a href="https://doi.org/10.7554/eLife.24179" title="DOI URL">https://doi.org/10.7554/eLife.24179</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7554/eLife.24179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7554%2FeLife.24179%26sid%3Dliteratum%253Aachs%26jtitle%3DeLife%26atitle%3DCRISPR%25252FCas9%252Bmutagenesis%252Binvalidates%252Ba%252Bputative%252Bcancer%252Bdependency%252Btargeted%252Bin%252Bon-going%252Bclinical%252Btrials%26aulast%3DLin%26aufirst%3DAnn%26date%3D2017%26date%3D2017%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Confirmed MELK Virtual Screen Hits<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>Kinase domain activity was measured at <sup>a</sup>[ATP] = 20 μM and <sup>b</sup>[ATP] = 2 mM.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Discovery of selective MELK inhibitors via virtual screening: (a) first iteration of MELK virtual screening, showing hit-rate versus novelty analysis; (b) novelty/hit-rate plot for second iteration of MELK virtual screening. The blue bars represent hit rates from the virtual screen, while the red bars plot the percentage of hits in each bin that share a Bemis and Murcko framework with the frameworks in the training set. The green circles show the hit counts for each similarity bin. For (b), while the majority of the compounds were still chosen for novelty, including novel scaffolds, a significant number were also chosen as SAR sets to further evaluate leads from the first iteration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cocrystal structure of <b>1a</b> bound to MELK. <b>1a</b> bound in the active site of MELK (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH8">5IH8</a>). Key interactions with catalytic Lys40, hinge Cys89, Tyr88, and Pro90 are highlighted. The ligand also makes several hydrophobic interactions with the protein. Red spheres represent water molecules with hydrogen bonds indicated by dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4-Arylpyzoles <b>3a</b>–<b>c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reaction conditions: (a) Cs<sub>2</sub>CO<sub>3</sub>, 95 °C, 1 h; (b) ArB(OH)<sub>2</sub>, Pd<sub>2</sub>(dba)<sub>3</sub>, P(Cy)<sub>3</sub>, K<sub>3</sub>PO<sub>4</sub>, microwave 130 °C, 20 min.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0008.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>7a</b>–<b>c</b> and General Procedure for the Late Stage Introduction of Alkoxides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reaction conditions: (a) <i>n</i>-Bu-Li, −65 °C, 5 min and then DBAD, −60 °C to −10 °C, 30 min; (b) 2-bromomalonaldehyde, TFA, DCM, rt; (c) 2,2,6,6-tetramethylpiperidine, <i>n</i>-BuLi, THF, 0 °C, 20 min, add 3-fluoropyridine, −78 °C, 1 h; (d) ZnBr<sub>2</sub>, warm to rt, add <b>6a</b>, Pd(PPh<sub>3</sub>)<sub>4</sub>, 60 °C, 1 h, then Pd(amphos)Cl<sub>2</sub>, 60 °C, 8 h; (e) 3-fluoro-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)pyridine, K<sub>3</sub>PO<sub>4</sub>, Pd(amphos)Cl<sub>2</sub>, dioxane/water, 100 °C, 2 h or 3-methoxy-4-pyridinylboronic acid, PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, water, dioxane, microwave 140 °C, 30 min; (f) NaH, alcohols, DMF 25–85 °C; (g) 4-bromophenylboronic acid, Cu(OAc)<sub>2</sub>, 4 Å molecular sieves, DCM; (h) 1-methylpiperazine, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, <i>tert</i>-BuONa, dioxane, microwave 130 °C, 20 min.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0009.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Monocationic Compounds (<b>12a</b>–<b>d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reactions conditions: (a) NaH, <i>tert</i>-butyl 4-(hydroxymethyl)piperidine-1-carboxylate, 70 °C, 5 h; (b) 4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-pyrazole, Pd(dppf)Cl<sub>2</sub>·DCM adduct, Na<sub>2</sub>CO<sub>3</sub>, dioxane, water, reflux, 16 h. (c) General procedure 3: ArI, Pd<sub>2</sub>(dba)<sub>3</sub>, XPhos, <i>tert</i>-BuONa, toluene, 105 °C, 16 h, then deprotection using general procedure 2. (d) AcOH, Br<sub>2</sub>, 100 °C; (e) 3-fluoro-4-pyridineB(OH)<sub>2</sub>, Pd(amphos)Cl<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, dioxane, 80–95 °C, 2–16 h.</p></p></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Overlay of <b>8a</b> and <b>8f</b> bound in the kinase active site. For <b>8a</b> (pink sticks with pink ribbon), a salt bridge is observed between the piperidine basic amine and Glu93 as well as additional ordering of the P-loop. For <b>8f</b> (teal sticks with teal ribbon), asterisks denote the end points for the disordered P-loop. Water molecules are shown as spheres in the same color as the coordinating ligand with hydrogen bonds indicated by dashed lines (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH9">5IH9</a> for <b>8a</b> and PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHA">5IHA</a> for <b>8f</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0005.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Ortho-fluorine induced hydrophobic collapse in <b>12b</b>. Binding conformation of <b>12b</b> was determined by X-ray crystallography (the protein is excluded for clarity) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHC">5IHC</a>). Fluorine is within VDW interaction distance of the piperidine (space filling model shown on the right).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/medium/jm-2016-00052e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0006.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of <b>8b</b> on cell cycle. (a) Treatment with <b>8b</b> led to increased apoptosis and polyploidy in MDA-MB-468 in a time-dependent manner. (b) MELK inhibitor treatment did not lead to increased apoptosis and polyploidy in MCF-7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00052/20160520/images/large/jm-2016-00052e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00052&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i61">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25756" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25756" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 24 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Weiss, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shokat, K. M.</span><span> </span><span class="NLM_article-title">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">739</span><span class="NLM_x">–</span> <span class="NLM_lpage">744</span><span class="refDoi"> DOI: 10.1038/nchembio1207-739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1038%2Fnchembio1207-739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=18007642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=739-744&author=W.+A.+Weissauthor=S.+S.+Taylorauthor=K.+M.+Shokat&title=Recognizing+and+exploiting+differences+between+RNAi+and+small-molecule+inhibitors&doi=10.1038%2Fnchembio1207-739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Recognizing and exploiting differences between RNAi and small-molecule inhibitors</span></div><div class="casAuthors">Weiss, William A.; Taylor, Stephen S.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">739-744</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The biol. of RNA interference has greatly facilitated anal. of loss-of-function phenotypes, but correlating these phenotypes with small-mol. inhibition profiles is not always straightforward.  The authors examine the rationale of comparing RNA interference to pharmacol. intervention in chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9JPzqVyshS7Vg90H21EOLACvtfcHk0lhXj14zo-9rSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlWqurjO&md5=9a26c990b358a6b2c717d1e361cac2da</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1207-739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1207-739%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DRecognizing%2520and%2520exploiting%2520differences%2520between%2520RNAi%2520and%2520small-molecule%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26spage%3D739%26epage%3D744%26doi%3D10.1038%2Fnchembio1207-739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Sweis, R. F.</span><span> </span><span class="NLM_article-title">Target (In)Validation: A Critical, Sometimes Unheralded, Role of Modern Medicinal Chemistry</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">618</span><span class="NLM_x">–</span> <span class="NLM_lpage">621</span><span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00183</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00183" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=618-621&author=R.+F.+Sweis&title=Target+%28In%29Validation%3A+A+Critical%2C+Sometimes+Unheralded%2C+Role+of+Modern+Medicinal+Chemistry&doi=10.1021%2Facsmedchemlett.5b00183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00183%26sid%3Dliteratum%253Aachs%26aulast%3DSweis%26aufirst%3DR.%2BF.%26atitle%3DTarget%2520%2528In%2529Validation%253A%2520A%2520Critical%252C%2520Sometimes%2520Unheralded%252C%2520Role%2520of%2520Modern%2520Medicinal%2520Chemistry%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D618%26epage%3D621%26doi%3D10.1021%2Facsmedchemlett.5b00183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Lizcano, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goransson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boudeau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawley, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Udd, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makela, T. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardie, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alessi, D. R.</span><span> </span><span class="NLM_article-title">LKB1 is a master kinase that activates 13 kinases of the AMPK subfamily, including MARK/PAR-1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">833</span><span class="NLM_x">–</span> <span class="NLM_lpage">843</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7600110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1038%2Fsj.emboj.7600110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=14976552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFCjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=833-843&author=J.+M.+Lizcanoauthor=O.+Goranssonauthor=R.+Tothauthor=M.+Deakauthor=N.+A.+Morriceauthor=J.+Boudeauauthor=S.+A.+Hawleyauthor=L.+Uddauthor=T.+P.+Makelaauthor=D.+G.+Hardieauthor=D.+R.+Alessi&title=LKB1+is+a+master+kinase+that+activates+13+kinases+of+the+AMPK+subfamily%2C+including+MARK%2FPAR-1&doi=10.1038%2Fsj.emboj.7600110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">LKB1 is a master kinase that activates 13 kinases of the AMPK subfamily, including MARK/PAR-1</span></div><div class="casAuthors">Lizcano, Jose M.; Goeransson, Olga; Toth, Rachel; Deak, Maria; Morrice, Nick A.; Boudeau, Jerome; Hawley, Simon A.; Udd, Lina; Maekelae, Tomi P.; Hardie, D. Grahame; Alessi, Dario R.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">833-843</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We recently demonstrated that the LKB1 tumor suppressor kinase, in complex with the pseudokinase STRAD and the scaffolding protein MO25, phosphorylates and activates AMP-activated protein kinase (AMPK).  A total of 12 human kinases (NUAK1, NUAK2, BRSK1, BRSK2, QIK, QSK, SIK, MARK1, MARK2, MARK3, MARK4 and MELK) are related to AMPK.  Here we demonstrate that LKB1 can phosphorylate the T-loop of all the members of this subfamily, apart from MELK, increasing their activity >50-fold.  LKB1 catalytic activity and the presence of MO25 and STRAD are required for activation.  Mutation of the T-loop Thr phosphorylated by LKB1 to Ala prevented activation, while mutation to glutamate produced active forms of many of the AMPK-related kinases.  Activities of endogenous NUAK2, QIK, QSK, SIK, MARK1, MARK2/3 and MARK4 were markedly reduced in LKB1-deficient cells.  Neither LKB1 activity nor that of AMPK-related kinases was stimulated by phenformin or AICAR, which activate AMPK.  Our results show that LKB1 functions as a master upstream protein kinase, regulating AMPK-related kinases as well as AMPK.  Between them, these kinases may mediate the physiol. effects of LKB1, including its tumor suppressor function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzr93XYu3yFbVg90H21EOLACvtfcHk0ljEKJ5BS1py-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFCjtbs%253D&md5=1aab7e4182fae41b3597b73baa5f0b7d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600110%26sid%3Dliteratum%253Aachs%26aulast%3DLizcano%26aufirst%3DJ.%2BM.%26aulast%3DGoransson%26aufirst%3DO.%26aulast%3DToth%26aufirst%3DR.%26aulast%3DDeak%26aufirst%3DM.%26aulast%3DMorrice%26aufirst%3DN.%2BA.%26aulast%3DBoudeau%26aufirst%3DJ.%26aulast%3DHawley%26aufirst%3DS.%2BA.%26aulast%3DUdd%26aufirst%3DL.%26aulast%3DMakela%26aufirst%3DT.%2BP.%26aulast%3DHardie%26aufirst%3DD.%2BG.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26atitle%3DLKB1%2520is%2520a%2520master%2520kinase%2520that%2520activates%252013%2520kinases%2520of%2520the%2520AMPK%2520subfamily%252C%2520including%2520MARK%252FPAR-1%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D833%26epage%3D843%26doi%3D10.1038%2Fsj.emboj.7600110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Beullens, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vancauwenbergh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrice, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Derua, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceulemans, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waelkens, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bollen, M.</span><span> </span><span class="NLM_article-title">Substrate specificity and activity regulation of protein kinase MELK</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">40003</span><span class="NLM_x">–</span> <span class="NLM_lpage">40011</span><span class="refDoi"> DOI: 10.1074/jbc.M507274200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1074%2Fjbc.M507274200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=40003-40011&author=M.+Beullensauthor=S.+Vancauwenberghauthor=N.+Morriceauthor=R.+Deruaauthor=H.+Ceulemansauthor=E.+Waelkensauthor=M.+Bollen&title=Substrate+specificity+and+activity+regulation+of+protein+kinase+MELK&doi=10.1074%2Fjbc.M507274200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M507274200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M507274200%26sid%3Dliteratum%253Aachs%26aulast%3DBeullens%26aufirst%3DM.%26aulast%3DVancauwenbergh%26aufirst%3DS.%26aulast%3DMorrice%26aufirst%3DN.%26aulast%3DDerua%26aufirst%3DR.%26aulast%3DCeulemans%26aufirst%3DH.%26aulast%3DWaelkens%26aufirst%3DE.%26aulast%3DBollen%26aufirst%3DM.%26atitle%3DSubstrate%2520specificity%2520and%2520activity%2520regulation%2520of%2520protein%2520kinase%2520MELK%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D40003%26epage%3D40011%26doi%3D10.1074%2Fjbc.M507274200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Gray, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jubb, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hogue, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dowd, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kljavin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frantz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koeppen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Sauvage, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, D. P.</span><span> </span><span class="NLM_article-title">Maternal embryonic leucine zipper kinase/murine protein serine-threonine kinase 38 is a promising therapeutic target for multiple cancers</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">9751</span><span class="NLM_x">–</span> <span class="NLM_lpage">9761</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-04-4531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1158%2F0008-5472.CAN-04-4531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=16266996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKktL7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=9751-9761&author=D.+Grayauthor=A.+M.+Jubbauthor=D.+Hogueauthor=P.+Dowdauthor=N.+Kljavinauthor=S.+Yiauthor=W.+Baiauthor=G.+Frantzauthor=Z.+Zhangauthor=H.+Koeppenauthor=F.+J.+de+Sauvageauthor=D.+P.+Davis&title=Maternal+embryonic+leucine+zipper+kinase%2Fmurine+protein+serine-threonine+kinase+38+is+a+promising+therapeutic+target+for+multiple+cancers&doi=10.1158%2F0008-5472.CAN-04-4531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Maternal embryonic leucine zipper kinase/murine protein serine-threonine kinase 38 is a promising therapeutic target for multiple cancers</span></div><div class="casAuthors">Gray, Daniel; Jubb, Adrian M.; Hogue, Deborah; Dowd, Patrick; Kljavin, Noelyn; Yi, Sothy; Bai, Wei; Frantz, Gretchen; Zhang, Zemin; Koeppen, Hartmut; de Sauvage, Frederic J.; Davis, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9751-9761</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">To identify genes that could serve as targets for novel cancer therapeutics, we used a bioinformatic anal. of microarray data comparing gene expression between normal and tumor-derived primary human tissues.  From this approach, we have found that maternal embryonic leucine zipper kinase (Melk), a member of the AMP serine/threonine kinase family, exhibits multiple features consistent with the potential utility of this gene as an anticancer target.  An oligonucleotide microarray anal. of multiple human tumor samples and cell lines suggests that Melk expression is frequently elevated in cancer relative to normal tissues, a pattern confirmed by quant. reverse transcription-PCR and Western blotting of selected primary tumor samples.  In situ hybridization localized Melk expression to malignant epithelial cells in 96%, 23%, and 13% of colorectal, lung, and ovarian tissue tumor samples, resp.  Expression of this gene is also elevated in spontaneous tumors derived from the ApcMin and Apc1638N murine models of intestinal tumorigenesis.  To begin addressing whether Melk is relevant for tumorigenesis, RNA interference-mediated silencing within human and murine tumor cell lines was done.  We show that Melk knockdown decreases proliferation and anchorage-independent growth in vitro as well as tumor growth in a xenograft model.  Together, these results suggest that Melk may provide a growth advantage for neoplastic cells and, therefore, inactivation may be therapeutically beneficial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_9K-J3ofrbrVg90H21EOLACvtfcHk0lglx1Vt6rUVDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKktL7F&md5=9d923595fbdf3b522d22c1329ba634c9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-04-4531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-04-4531%26sid%3Dliteratum%253Aachs%26aulast%3DGray%26aufirst%3DD.%26aulast%3DJubb%26aufirst%3DA.%2BM.%26aulast%3DHogue%26aufirst%3DD.%26aulast%3DDowd%26aufirst%3DP.%26aulast%3DKljavin%26aufirst%3DN.%26aulast%3DYi%26aufirst%3DS.%26aulast%3DBai%26aufirst%3DW.%26aulast%3DFrantz%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKoeppen%26aufirst%3DH.%26aulast%3Dde%2BSauvage%26aufirst%3DF.%2BJ.%26aulast%3DDavis%26aufirst%3DD.%2BP.%26atitle%3DMaternal%2520embryonic%2520leucine%2520zipper%2520kinase%252Fmurine%2520protein%2520serine-threonine%2520kinase%252038%2520is%2520a%2520promising%2520therapeutic%2520target%2520for%2520multiple%2520cancers%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D9751%26epage%3D9761%26doi%3D10.1158%2F0008-5472.CAN-04-4531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Gu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banasavadi-Siddegowda, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joshi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurt, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, I.</span><span> </span><span class="NLM_article-title">Tumor-specific activation of the C-JUN/MELK pathway regulates glioma stem cell growth in a p53-dependent manner</span> <span class="citation_source-journal">Stem Cells</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">870</span><span class="NLM_x">–</span> <span class="NLM_lpage">881</span><span class="refDoi"> DOI: 10.1002/stem.1322</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1002%2Fstem.1322" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=870-881&author=C.+Guauthor=Y.+K.+Banasavadi-Siddegowdaauthor=K.+Joshiauthor=Y.+Nakamuraauthor=H.+Kurtauthor=S.+Guptaauthor=I.+Nakano&title=Tumor-specific+activation+of+the+C-JUN%2FMELK+pathway+regulates+glioma+stem+cell+growth+in+a+p53-dependent+manner&doi=10.1002%2Fstem.1322"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fstem.1322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1322%26sid%3Dliteratum%253Aachs%26aulast%3DGu%26aufirst%3DC.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%2BK.%26aulast%3DJoshi%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DKurt%26aufirst%3DH.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DTumor-specific%2520activation%2520of%2520the%2520C-JUN%252FMELK%2520pathway%2520regulates%2520glioma%2520stem%2520cell%2520growth%2520in%2520a%2520p53-dependent%2520manner%26jtitle%3DStem%2520Cells%26date%3D2013%26volume%3D31%26spage%3D870%26epage%3D881%26doi%3D10.1002%2Fstem.1322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Joshi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banasavadi-Siddegowda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kig, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nardini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobol, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chow, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornblum, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waclaw, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beullens, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakano, I.</span><span> </span><span class="NLM_article-title">MELK-dependent FOXM1 phosphorylation is essential for proliferation of glioma stem cells</span> <span class="citation_source-journal">Stem Cells</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1051</span><span class="NLM_x">–</span> <span class="NLM_lpage">1063</span><span class="refDoi"> DOI: 10.1002/stem.1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1002%2Fstem.1358" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=1051-1063&author=K.+Joshiauthor=Y.+Banasavadi-Siddegowdaauthor=X.+Moauthor=S.+H.+Kimauthor=P.+Maoauthor=C.+Kigauthor=D.+Nardiniauthor=R.+W.+Sobolauthor=L.+M.+Chowauthor=H.+I.+Kornblumauthor=R.+Waclawauthor=M.+Beullensauthor=I.+Nakano&title=MELK-dependent+FOXM1+phosphorylation+is+essential+for+proliferation+of+glioma+stem+cells&doi=10.1002%2Fstem.1358"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fstem.1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fstem.1358%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DK.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DMo%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DMao%26aufirst%3DP.%26aulast%3DKig%26aufirst%3DC.%26aulast%3DNardini%26aufirst%3DD.%26aulast%3DSobol%26aufirst%3DR.%2BW.%26aulast%3DChow%26aufirst%3DL.%2BM.%26aulast%3DKornblum%26aufirst%3DH.%2BI.%26aulast%3DWaclaw%26aufirst%3DR.%26aulast%3DBeullens%26aufirst%3DM.%26aulast%3DNakano%26aufirst%3DI.%26atitle%3DMELK-dependent%2520FOXM1%2520phosphorylation%2520is%2520essential%2520for%2520proliferation%2520of%2520glioma%2520stem%2520cells%26jtitle%3DStem%2520Cells%26date%3D2013%26volume%3D31%26spage%3D1051%26epage%3D1063%26doi%3D10.1002%2Fstem.1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Rhodes, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deshpande, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varambally, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrette, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span> </span><span class="NLM_article-title">Large-scale meta-analysis of cancer microarray data identifies common transcriptional profiles of neoplastic transformation and progression</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">9309</span><span class="NLM_x">–</span> <span class="NLM_lpage">9314</span><span class="refDoi"> DOI: 10.1073/pnas.0401994101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1073%2Fpnas.0401994101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=9309-9314&author=D.+R.+Rhodesauthor=J.+Yuauthor=K.+Shankerauthor=N.+Deshpandeauthor=R.+Varamballyauthor=D.+Ghoshauthor=T.+Barretteauthor=A.+Pandeyauthor=A.+M.+Chinnaiyan&title=Large-scale+meta-analysis+of+cancer+microarray+data+identifies+common+transcriptional+profiles+of+neoplastic+transformation+and+progression&doi=10.1073%2Fpnas.0401994101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0401994101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0401994101%26sid%3Dliteratum%253Aachs%26aulast%3DRhodes%26aufirst%3DD.%2BR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DShanker%26aufirst%3DK.%26aulast%3DDeshpande%26aufirst%3DN.%26aulast%3DVarambally%26aufirst%3DR.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DBarrette%26aufirst%3DT.%26aulast%3DPandey%26aufirst%3DA.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DLarge-scale%2520meta-analysis%2520of%2520cancer%2520microarray%2520data%2520identifies%2520common%2520transcriptional%2520profiles%2520of%2520neoplastic%2520transformation%2520and%2520progression%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D9309%26epage%3D9314%26doi%3D10.1073%2Fpnas.0401994101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Pickard, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, I. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hedge, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mourtada-Maarabouni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. T.</span><span> </span><span class="NLM_article-title">Dysregulated expression of Fau and MELK is associated with poor prognosis in breast cancer</span> <span class="citation_source-journal">Breast Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">R60</span><span class="refDoi"> DOI: 10.1186/bcr2350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1186%2Fbcr2350" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=R60&author=M.+R.+Pickardauthor=A.+R.+Greenauthor=I.+O.+Ellisauthor=C.+Caldasauthor=V.+L.+Hedgeauthor=M.+Mourtada-Maarabouniauthor=G.+T.+Williams&title=Dysregulated+expression+of+Fau+and+MELK+is+associated+with+poor+prognosis+in+breast+cancer&doi=10.1186%2Fbcr2350"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fbcr2350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2350%26sid%3Dliteratum%253Aachs%26aulast%3DPickard%26aufirst%3DM.%2BR.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DCaldas%26aufirst%3DC.%26aulast%3DHedge%26aufirst%3DV.%2BL.%26aulast%3DMourtada-Maarabouni%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DG.%2BT.%26atitle%3DDysregulated%2520expression%2520of%2520Fau%2520and%2520MELK%2520is%2520associated%2520with%2520poor%2520prognosis%2520in%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2009%26volume%3D11%26spage%3DR60%26doi%3D10.1186%2Fbcr2350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tunici, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abdulkadir, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irvin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Black, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J. S.</span><span> </span><span class="NLM_article-title">Analysis of gene expression and chemoresistance of CD133+ cancer stem cells in glioblastoma</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="refDoi"> DOI: 10.1186/1476-4598-5-67</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1186%2F1476-4598-5-67" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=17140455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A280%3ADC%252BD28jit1CntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=67&author=G.+Liuauthor=X.+Yuanauthor=Z.+Zengauthor=P.+Tuniciauthor=H.+Ngauthor=I.+R.+Abdulkadirauthor=L.+Luauthor=D.+Irvinauthor=K.+L.+Blackauthor=J.+S.+Yu&title=Analysis+of+gene+expression+and+chemoresistance+of+CD133%2B+cancer+stem+cells+in+glioblastoma&doi=10.1186%2F1476-4598-5-67"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of gene expression and chemoresistance of CD133+ cancer stem cells in glioblastoma</span></div><div class="casAuthors">Liu Gentao; Yuan Xiangpeng; Zeng Zhaohui; Tunici Patrizia; Ng Hiushan; Abdulkadir Iman R; Lu Lizhi; Irvin Dwain; Black Keith L; Yu John S</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Recently, a small population of cancer stem cells in adult and pediatric brain tumors has been identified.  Some evidence has suggested that CD133 is a marker for a subset of leukemia and glioblastoma cancer stem cells.  Especially, CD133 positive cells isolated from human glioblastoma may initiate tumors and represent novel targets for therapeutics.  The gene expression and the drug resistance property of CD133 positive cancer stem cells, however, are still unknown.  RESULTS:  In this study, by FACS analysis we determined the percentage of CD133 positive cells in three primary cultured cell lines established from glioblastoma patients 10.2%, 69.7% and 27.5%, respectively.  We also determined the average mRNA levels of markers associated with neural precursors.  For example, CD90, CD44, CXCR4, Nestin, Msi1 and MELK mRNA on CD133 positive cells increased to 15.6, 5.7, 337.8, 21.4, 84 and 1351 times, respectively, compared to autologous CD133 negative cells derived from cell line No. 66.  Additionally, CD133 positive cells express higher levels of BCRP1 and MGMT mRNA, as well as higher mRNA levels of genes that inhibit apoptosis.  Furthermore, CD133 positive cells were significantly resistant to chemotherapeutic agents including temozolomide, carboplatin, paclitaxel (Taxol) and etoposide (VP16) compared to autologous CD133 negative cells.  Finally, CD133 expression was significantly higher in recurrent GBM tissue obtained from five patients as compared to their respective newly diagnosed tumors.  CONCLUSION:  Our study for the first time provided evidence that CD133 positive cancer stem cells display strong capability on tumor's resistance to chemotherapy.  This resistance is probably contributed by the CD133 positive cell with higher expression of on BCRP1 and MGMT, as well as the anti-apoptosis protein and inhibitors of apoptosis protein families.  Future treatment should target this small population of CD133 positive cancer stem cells in tumors to improve the survival of brain tumor patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSrHS0LIOVgyO98rul4nhSdfW6udTcc2eaNSvGR6h5kqrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28jit1CntA%253D%253D&md5=47d6847d5afff4bc3e663b83f1154e52</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-5-67&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-5-67%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DTunici%26aufirst%3DP.%26aulast%3DNg%26aufirst%3DH.%26aulast%3DAbdulkadir%26aufirst%3DI.%2BR.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DIrvin%26aufirst%3DD.%26aulast%3DBlack%26aufirst%3DK.%2BL.%26aulast%3DYu%26aufirst%3DJ.%2BS.%26atitle%3DAnalysis%2520of%2520gene%2520expression%2520and%2520chemoresistance%2520of%2520CD133%252B%2520cancer%2520stem%2520cells%2520in%2520glioblastoma%26jtitle%3DMol.%2520Cancer%26date%3D2006%26volume%3D5%26spage%3D67%26doi%3D10.1186%2F1476-4598-5-67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Ryu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deluca, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alani, R. M.</span><span> </span><span class="NLM_article-title">Comprehensive expression profiling of tumor cell lines identifies molecular signatures of melanoma progression</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e594</span><span class="refDoi"> DOI: 10.1371/journal.pone.0000594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1371%2Fjournal.pone.0000594" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=e594&author=B.+Ryuauthor=D.+S.+Kimauthor=A.+M.+Delucaauthor=R.+M.+Alani&title=Comprehensive+expression+profiling+of+tumor+cell+lines+identifies+molecular+signatures+of+melanoma+progression&doi=10.1371%2Fjournal.pone.0000594"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0000594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0000594%26sid%3Dliteratum%253Aachs%26aulast%3DRyu%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DD.%2BS.%26aulast%3DDeluca%26aufirst%3DA.%2BM.%26aulast%3DAlani%26aufirst%3DR.%2BM.%26atitle%3DComprehensive%2520expression%2520profiling%2520of%2520tumor%2520cell%2520lines%2520identifies%2520molecular%2520signatures%2520of%2520melanoma%2520progression%26jtitle%3DPLoS%2520One%26date%3D2007%26volume%3D2%26spage%3De594%26doi%3D10.1371%2Fjournal.pone.0000594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Marie, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okamoto, O. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oba-Shinjo, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camargo, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kosoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlotti, C. G.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toledo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreira-Filho, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zago, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caballero, O. L.</span><span> </span><span class="NLM_article-title">Maternal embryonic leucine zipper kinase transcript abundance correlates with malignancy grade in human astrocytomas</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">807</span><span class="NLM_x">–</span> <span class="NLM_lpage">815</span><span class="refDoi"> DOI: 10.1002/ijc.23189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1002%2Fijc.23189" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=17960622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFShs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2008&pages=807-815&author=S.+K.+Marieauthor=O.+K.+Okamotoauthor=M.+Unoauthor=A.+P.+Hasegawaauthor=S.+M.+Oba-Shinjoauthor=T.+Cohenauthor=A.+A.+Camargoauthor=A.+Kosoyauthor=C.+G.+Carlottiauthor=S.+Toledoauthor=C.+A.+Moreira-Filhoauthor=M.+A.+Zagoauthor=A.+J.+Simpsonauthor=O.+L.+Caballero&title=Maternal+embryonic+leucine+zipper+kinase+transcript+abundance+correlates+with+malignancy+grade+in+human+astrocytomas&doi=10.1002%2Fijc.23189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Maternal embryonic leucine zipper kinase transcript abundance correlates with malignancy grade in human astrocytomas</span></div><div class="casAuthors">Marie, Suely K. N.; Okamoto, Oswaldo K.; Uno, Miyuki; Paula, Ana; Hasegawa, G.; Oba-Shinjo, Sueli M.; Cohen, Tzeela; Camargo, Anamaria A.; Kosoy, Ana; Carlotti, Carlos G., Jr.; Toledo, Silvia; Moreira-Filho, Carlos A.; Zago, Marco A.; Simpson, Andrew J.; Caballero, Otavia L.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">807-815</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">We have performed cDNA microarray analyses to identify gene expression differences between highly invasive glioblastoma multiforme (GBM) and typically benign pilocytic astrocytomas (PA).  Despite the significant clin. and pathol. differences between the 2 tumor types, only 63 genes were found to exhibit 2-fold or greater overexpression in GBM as compared to PA.  Forty percent of these genes are related to the regulation of the cell cycle and mitosis.  QT-PCR validation of 6 overexpressed genes: MELK, AUKB, ASPM, PRC1, IL13RA2 and KIAA0101 confirmed at least a 5-fold increase in the av. expression levels in GBM.  Maternal embryonic leucine zipper kinase (MELK) exhibited the most statistically significant difference.  A more detailed investigation of MELK expression was undertaken to study its oncogenic relevance.  In the examn. of more than 100 tumors of the central nervous system, we found progressively higher expression of MELK with astrocytoma grade and a noteworthy uniformity of high level expression in GBM.  Similar level of overexpression was also obsd. in medulloblastoma.  We found neither gene promoter hypomethylation nor amplification to be a factor in MELK expression, but were able to demonstrate that MELK knockdown in malignant astrocytoma cell lines caused a redn. in proliferation and anchorage-independent growth in in vitro assays.  Our results indicate that GBM and PA differ by the expression of surprisingly few genes.  Among them, MELK correlated with malignancy grade in astrocytomas and represents a therapeutic target for the management of the most frequent brain tumors in adult and children.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonxXHYLQpPabVg90H21EOLACvtfcHk0libpxxG9KEUEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFShs70%253D&md5=0682ef28bdcfadc200fecda059046afe</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1002%2Fijc.23189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.23189%26sid%3Dliteratum%253Aachs%26aulast%3DMarie%26aufirst%3DS.%2BK.%26aulast%3DOkamoto%26aufirst%3DO.%2BK.%26aulast%3DUno%26aufirst%3DM.%26aulast%3DHasegawa%26aufirst%3DA.%2BP.%26aulast%3DOba-Shinjo%26aufirst%3DS.%2BM.%26aulast%3DCohen%26aufirst%3DT.%26aulast%3DCamargo%26aufirst%3DA.%2BA.%26aulast%3DKosoy%26aufirst%3DA.%26aulast%3DCarlotti%26aufirst%3DC.%2BG.%26aulast%3DToledo%26aufirst%3DS.%26aulast%3DMoreira-Filho%26aufirst%3DC.%2BA.%26aulast%3DZago%26aufirst%3DM.%2BA.%26aulast%3DSimpson%26aufirst%3DA.%2BJ.%26aulast%3DCaballero%26aufirst%3DO.%2BL.%26atitle%3DMaternal%2520embryonic%2520leucine%2520zipper%2520kinase%2520transcript%2520abundance%2520correlates%2520with%2520malignancy%2520grade%2520in%2520human%2520astrocytomas%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2008%26volume%3D122%26spage%3D807%26epage%3D815%26doi%3D10.1002%2Fijc.23189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baitsch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lako, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Von, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stegmeier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitchison, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, J. J.</span><span> </span><span class="NLM_article-title">MELK is an oncogenic kinase essential for mitotic progression in basal-like breast cancer cells</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e01763</span><span class="refDoi"> DOI: 10.7554/eLife.01763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.7554%2FeLife.01763" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=e01763&author=Y.+Wangauthor=Y.+M.+Leeauthor=L.+Baitschauthor=A.+Huangauthor=Y.+Xiangauthor=H.+Tongauthor=A.+Lakoauthor=T.+Vonauthor=C.+Choiauthor=E.+Limauthor=J.+Minauthor=L.+Liauthor=F.+Stegmeierauthor=R.+Schlegelauthor=M.+J.+Eckauthor=N.+S.+Grayauthor=T.+J.+Mitchisonauthor=J.+J.+Zhao&title=MELK+is+an+oncogenic+kinase+essential+for+mitotic+progression+in+basal-like+breast+cancer+cells&doi=10.7554%2FeLife.01763"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.7554%2FeLife.01763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01763%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DY.%2BM.%26aulast%3DBaitsch%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DA.%26aulast%3DXiang%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DH.%26aulast%3DLako%26aufirst%3DA.%26aulast%3DVon%26aufirst%3DT.%26aulast%3DChoi%26aufirst%3DC.%26aulast%3DLim%26aufirst%3DE.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DMitchison%26aufirst%3DT.%2BJ.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26atitle%3DMELK%2520is%2520an%2520oncogenic%2520kinase%2520essential%2520for%2520mitotic%2520progression%2520in%2520basal-like%2520breast%2520cancer%2520cells%26jtitle%3DeLife%26date%3D2014%26volume%3D3%26spage%3De01763%26doi%3D10.7554%2FeLife.01763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Chung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takamatsu, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatsuguchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kijima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuo, Y.</span><span> </span><span class="NLM_article-title">Development of an orally-administrative MELK-targeting inhibitor that suppresses the growth of various types of human cancer</span> <span class="citation_source-journal">Oncotarget.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1629</span><span class="NLM_x">–</span> <span class="NLM_lpage">1640</span><span class="refDoi"> DOI: 10.18632/oncotarget.790</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.18632%2Foncotarget.790" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1629-1640&author=S.+Chungauthor=H.+Suzukiauthor=T.+Miyamotoauthor=N.+Takamatsuauthor=A.+Tatsuguchiauthor=K.+Uedaauthor=K.+Kijimaauthor=Y.+Nakamuraauthor=Y.+Matsuo&title=Development+of+an+orally-administrative+MELK-targeting+inhibitor+that+suppresses+the+growth+of+various+types+of+human+cancer&doi=10.18632%2Foncotarget.790"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.790%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DH.%26aulast%3DMiyamoto%26aufirst%3DT.%26aulast%3DTakamatsu%26aufirst%3DN.%26aulast%3DTatsuguchi%26aufirst%3DA.%26aulast%3DUeda%26aufirst%3DK.%26aulast%3DKijima%26aufirst%3DK.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DMatsuo%26aufirst%3DY.%26atitle%3DDevelopment%2520of%2520an%2520orally-administrative%2520MELK-targeting%2520inhibitor%2520that%2520suppresses%2520the%2520growth%2520of%2520various%2520types%2520of%2520human%2520cancer%26jtitle%3DOncotarget.%26date%3D2012%26volume%3D3%26spage%3D1629%26epage%3D1640%26doi%3D10.18632%2Foncotarget.790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Canevari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Re Depaolini, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cucchi, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casale, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrera, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forte, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carpinelli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felder, E. R.</span><span> </span><span class="NLM_article-title">Structural insight into maternal embryonic leucine zipper kinase (MELK) conformation and inhibition toward structure-based drug design</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6380</span><span class="NLM_x">–</span> <span class="NLM_lpage">6387</span><span class="refDoi"> DOI: 10.1021/bi4005864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi4005864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=6380-6387&author=G.+Canevariauthor=S.+Re+Depaoliniauthor=U.+Cucchiauthor=J.+A.+Bertrandauthor=E.+Casaleauthor=C.+Perreraauthor=B.+Forteauthor=P.+Carpinelliauthor=E.+R.+Felder&title=Structural+insight+into+maternal+embryonic+leucine+zipper+kinase+%28MELK%29+conformation+and+inhibition+toward+structure-based+drug+design&doi=10.1021%2Fbi4005864"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fbi4005864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi4005864%26sid%3Dliteratum%253Aachs%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DRe%2BDepaolini%26aufirst%3DS.%26aulast%3DCucchi%26aufirst%3DU.%26aulast%3DBertrand%26aufirst%3DJ.%2BA.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DPerrera%26aufirst%3DC.%26aulast%3DForte%26aufirst%3DB.%26aulast%3DCarpinelli%26aufirst%3DP.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26atitle%3DStructural%2520insight%2520into%2520maternal%2520embryonic%2520leucine%2520zipper%2520kinase%2520%2528MELK%2529%2520conformation%2520and%2520inhibition%2520toward%2520structure-based%2520drug%2520design%26jtitle%3DBiochemistry%26date%3D2013%26volume%3D52%26spage%3D6380%26epage%3D6387%26doi%3D10.1021%2Fbi4005864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brehmer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyne, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilissen, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlett, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meerpoel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sommen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linders, J. T.</span><span> </span><span class="NLM_article-title">Fragment-based discovery of type I inhibitors of maternal embryonic leucine zipper kinase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1021/ml5001245</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001245" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=25-30&author=C.+N.+Johnsonauthor=V.+Berdiniauthor=L.+Bekeauthor=P.+Bonnetauthor=D.+Brehmerauthor=J.+E.+Coyleauthor=P.+J.+Dayauthor=M.+Fredericksonauthor=E.+J.+Freyneauthor=R.+A.+Gilissenauthor=C.+C.+Hamlettauthor=S.+Howardauthor=L.+Meerpoelauthor=R.+McMenaminauthor=S.+Patelauthor=D.+C.+Reesauthor=A.+Sharffauthor=F.+Sommenauthor=T.+Wuauthor=J.+T.+Linders&title=Fragment-based+discovery+of+type+I+inhibitors+of+maternal+embryonic+leucine+zipper+kinase&doi=10.1021%2Fml5001245"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fml5001245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001245%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DFreyne%26aufirst%3DE.%2BJ.%26aulast%3DGilissen%26aufirst%3DR.%2BA.%26aulast%3DHamlett%26aufirst%3DC.%2BC.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DMeerpoel%26aufirst%3DL.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSommen%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26atitle%3DFragment-based%2520discovery%2520of%2520type%2520I%2520inhibitors%2520of%2520maternal%2520embryonic%2520leucine%2520zipper%2520kinase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D25%26epage%3D30%26doi%3D10.1021%2Fml5001245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Johnson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adelinet, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beke, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brehmer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyne, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilissen, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlett, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meerpoel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mevellec, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasquier, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linders, J. T.</span><span> </span><span class="NLM_article-title">Structure-Based Design of Type II Inhibitors Applied to Maternal Embryonic Leucine Zipper Kinase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.1021/ml5001273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5001273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1WltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=31-36&author=C.+N.+Johnsonauthor=C.+Adelinetauthor=V.+Berdiniauthor=L.+Bekeauthor=P.+Bonnetauthor=D.+Brehmerauthor=F.+Caloauthor=J.+E.+Coyleauthor=P.+J.+Dayauthor=M.+Fredericksonauthor=E.+J.+Freyneauthor=R.+A.+Gilissenauthor=C.+C.+Hamlettauthor=S.+Howardauthor=L.+Meerpoelauthor=L.+Mevellecauthor=R.+McMenaminauthor=E.+Pasquierauthor=S.+Patelauthor=D.+C.+Reesauthor=J.+T.+Linders&title=Structure-Based+Design+of+Type+II+Inhibitors+Applied+to+Maternal+Embryonic+Leucine+Zipper+Kinase&doi=10.1021%2Fml5001273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Type II Inhibitors Applied to Maternal Embryonic Leucine Zipper Kinase</span></div><div class="casAuthors">Johnson, Christopher N.; Adelinet, Christophe; Berdini, Valerio; Beke, Lijs; Bonnet, Pascal; Brehmer, Dirk; Calo, Frederick; Coyle, Joseph E.; Day, Phillip J.; Frederickson, Martyn; Freyne, Eddy J. E.; Gilissen, Ron A. H. J.; Hamlett, Christopher C. F.; Howard, Steven; Meerpoel, Lieven; Mevellec, Laurence; McMenamin, Rachel; Pasquier, Elisabeth; Patel, Sahil; Rees, David C.; Linders, Joannes T. M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-36</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel Type II kinase inhibitor chemotype has been identified for maternal embryonic leucine zipper kinase (MELK) using structure-based ligand design.  The strategy involved structural characterization of an induced DFG-out pocket by protein-ligand X-ray crystallog. and incorporation of a slender linkage capable of bypassing a large gate-keeper residue, thus enabling design of mols. accessing both hinge and induced pocket regions.  Optimization of an initial hit led to the identification of a low-nanomolar, cell-penetrant Type II inhibitor suitable for use as a chem. probe for MELK.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWmY300FJpQ7Vg90H21EOLACvtfcHk0lgnMgQlYsjNqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1WltL8%253D&md5=35e87bcfe134c8373da3dc57c3c4e6c5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fml5001273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5001273%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DAdelinet%26aufirst%3DC.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DBeke%26aufirst%3DL.%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DCalo%26aufirst%3DF.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDay%26aufirst%3DP.%2BJ.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DFreyne%26aufirst%3DE.%2BJ.%26aulast%3DGilissen%26aufirst%3DR.%2BA.%26aulast%3DHamlett%26aufirst%3DC.%2BC.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DMeerpoel%26aufirst%3DL.%26aulast%3DMevellec%26aufirst%3DL.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DPasquier%26aufirst%3DE.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DLinders%26aufirst%3DJ.%2BT.%26atitle%3DStructure-Based%2520Design%2520of%2520Type%2520II%2520Inhibitors%2520Applied%2520to%2520Maternal%2520Embryonic%2520Leucine%2520Zipper%2520Kinase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D31%26epage%3D36%26doi%3D10.1021%2Fml5001273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Martin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jansen, J.</span><span> </span><span class="NLM_article-title">Profile-QSAR: a novel meta-QSAR method that combines activities across the kinase family to accurately predict affinity, selectivity, and cellular activity</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1942</span><span class="NLM_x">–</span> <span class="NLM_lpage">1956</span><span class="refDoi"> DOI: 10.1021/ci1005004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci1005004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1942-1956&author=E.+Martinauthor=P.+Mukherjeeauthor=D.+Sullivanauthor=J.+Jansen&title=Profile-QSAR%3A+a+novel+meta-QSAR+method+that+combines+activities+across+the+kinase+family+to+accurately+predict+affinity%2C+selectivity%2C+and+cellular+activity&doi=10.1021%2Fci1005004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR: A Novel meta-QSAR Method that Combines Activities across the Kinase Family To Accurately Predict Affinity, Selectivity, and Cellular Activity</span></div><div class="casAuthors">Martin, Eric; Mukherjee, Prasenjit; Sullivan, David; Jansen, Johanna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1942-1956</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Profile-QSAR is a novel 2D predictive model building method for kinases.  This "meta-QSAR" method models the activity of each compd. against a new kinase target as a linear combination of its predicted activities against a large panel of 92 previously studied kinases comprised from 115 assays.  Profile-QSAR starts with a sparse incomplete kinase by compd. (KxC) activity matrix, used to generate Bayesian QSAR models for the 92 "basis-set" kinases.  These Bayesian QSARs generate a complete "synthetic" KxC activity matrix of predictions.  These synthetic activities are used as "chem. descriptors" to train partial-least squares (PLS) models, from modest amts. of medium-throughput screening data, for predicting activity against new kinases.  The Profile-QSAR predictions for the 92 kinases (115 assays) gave a median external R2ext = 0.59 on 25% held-out test sets.  The method has proven accurate enough to predict pairwise kinase selectivities with a median correlation of R2ext = 0.61 for 958 kinase pairs with at least 600 common compds.  It has been further expanded by adding a "CkXC" cellular activity matrix to the KxC matrix to predict cellular activity for 42 kinase driven cellular assays with median R2ext = 0.58 for 24 target modulation assays and R2ext = 0.41 for 18 cell proliferation assays.  The 2D Profile-QSAR, along with the 3D Surrogate AutoShim, are the foundations of an internally developed iterative medium-throughput screening (IMTS) methodol. for virtual screening (VS) of compd. archives as an alternative to exptl. high-throughput screening (HTS).  The method has been applied to 20 actual prospective kinase projects.  Biol. results have so far been obtained in eight of them.  Q2 values ranged from 0.3 to 0.7.  Hit-rates at 10 uM for exptl. tested compds. varied from 25% to 80%, except in K5, which was a special case aimed specifically at finding "type II" binders, where none of the compds. were predicted to be active at 10 μM.  These overall results are particularly striking as chem. novelty was an important criterion in selecting compds. for testing.  The method is completely automated.  Predicted activities for nearly 4 million internal and com. compds. across 115 kinase assays and 42 cellular assays are stored in the corporate database.  Like computed phys. properties, this predicted kinase activity profile can be computed and stored as each compd. is registered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjTg6KqlgEyLVg90H21EOLACvtfcHk0liaSxQgsilH6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVSrsrw%253D&md5=49422d01fd686a4befc9db7c123cfcce</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci1005004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci1005004%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DMukherjee%26aufirst%3DP.%26aulast%3DSullivan%26aufirst%3DD.%26aulast%3DJansen%26aufirst%3DJ.%26atitle%3DProfile-QSAR%253A%2520a%2520novel%2520meta-QSAR%2520method%2520that%2520combines%2520activities%2520across%2520the%2520kinase%2520family%2520to%2520accurately%2520predict%2520affinity%252C%2520selectivity%252C%2520and%2520cellular%2520activity%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1942%26epage%3D1956%26doi%3D10.1021%2Fci1005004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Martin, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sullivan, D. C.</span><span> </span><span class="NLM_article-title">Surrogate AutoShim: predocking into a universal ensemble kinase receptor for three dimensional activity prediction, very quickly, without a crystal structure</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">881</span><span class="refDoi"> DOI: 10.1021/ci700455u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700455u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=873-881&author=E.+J.+Martinauthor=D.+C.+Sullivan&title=Surrogate+AutoShim%3A+predocking+into+a+universal+ensemble+kinase+receptor+for+three+dimensional+activity+prediction%2C+very+quickly%2C+without+a+crystal+structure&doi=10.1021%2Fci700455u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Surrogate AutoShim: Predocking into a Universal Ensemble Kinase Receptor for Three Dimensional Activity Prediction, Very Quickly, without a Crystal Structure</span></div><div class="casAuthors">Martin, Eric J.; Sullivan, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-881</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">"Ensemble surrogate AutoShim" is a kinase specific extension of the AutoShim docking method that solves the three traditional limitations of conventional docking: (1) it gives good correlations with affinity, (2) does not require a target protein structure, and (3) for a preprocessed company archive of 1.5 million compds., is as fast as traditional 2D QSAR.  It does require several hundred exptl. IC50 values for each new target.  Original AutoShim adds pharmacophore "shims" to a crystal structure binding site.  An iterative partial least squares (PLS) procedure selects the best pose, while adjusting the shim wts. to reproduce IC50 data.  Surrogate AutoShim adjusts shims in one crystal structure to reproduce IC50 data for a different kinase target.  Ensemble surrogate AutoShim uses 16 structurally diverse kinase crystal structures as a "universal ensemble kinase receptor", suitable for any kinase target.  The 1.5 million member Novartis screening collection has been predocked into the shimmed ensemble, so new kinase models can be built, and the entire corporate archive virtually screened, in hours rather than weeks.  A kinase-biased set of 10 000 compds., that samples the entire corporate archive, has been designed for lead discovery by iterative kinase screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5HSUkdEqTcLVg90H21EOLACvtfcHk0liaSxQgsilH6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktVOkurg%253D&md5=0fe14c274192ec38b65224cae779cf35</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci700455u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700455u%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DSullivan%26aufirst%3DD.%2BC.%26atitle%3DSurrogate%2520AutoShim%253A%2520predocking%2520into%2520a%2520universal%2520ensemble%2520kinase%2520receptor%2520for%2520three%2520dimensional%2520activity%2520prediction%252C%2520very%2520quickly%252C%2520without%2520a%2520crystal%2520structure%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D873%26epage%3D881%26doi%3D10.1021%2Fci700455u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Furtmann, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Comprehensive analysis of three-dimensional activity cliffs formed by kinase inhibitors with different binding modes and cliff mapping of structural analogues</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">264</span><span class="refDoi"> DOI: 10.1021/jm5009264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5009264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=252-264&author=N.+Furtmannauthor=Y.+Huauthor=J.+Bajorath&title=Comprehensive+analysis+of+three-dimensional+activity+cliffs+formed+by+kinase+inhibitors+with+different+binding+modes+and+cliff+mapping+of+structural+analogues&doi=10.1021%2Fjm5009264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm5009264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5009264%26sid%3Dliteratum%253Aachs%26aulast%3DFurtmann%26aufirst%3DN.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComprehensive%2520analysis%2520of%2520three-dimensional%2520activity%2520cliffs%2520formed%2520by%2520kinase%2520inhibitors%2520with%2520different%2520binding%2520modes%2520and%2520cliff%2520mapping%2520of%2520structural%2520analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D252%26epage%3D264%26doi%3D10.1021%2Fjm5009264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Desai, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzi, J. P.</span><span> </span><span class="NLM_article-title">Importance of parallel vectors and "hydrophobic collapse" of the aligned aromatic rings: discovery of a potent substance P antagonist</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">4263</span><span class="NLM_x">–</span> <span class="NLM_lpage">4266</span><span class="refDoi"> DOI: 10.1021/jm00051a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00051a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=4263-4266&author=M.+C.+Desaiauthor=L.+A.+Vincentauthor=J.+P.+Rizzi&title=Importance+of+parallel+vectors+and+%22hydrophobic+collapse%22+of+the+aligned+aromatic+rings%3A+discovery+of+a+potent+substance+P+antagonist&doi=10.1021%2Fjm00051a001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm00051a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00051a001%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DM.%2BC.%26aulast%3DVincent%26aufirst%3DL.%2BA.%26aulast%3DRizzi%26aufirst%3DJ.%2BP.%26atitle%3DImportance%2520of%2520parallel%2520vectors%2520and%2520%2522hydrophobic%2520collapse%2522%2520of%2520the%2520aligned%2520aromatic%2520rings%253A%2520discovery%2520of%2520a%2520potent%2520substance%2520P%2520antagonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D4263%26epage%3D4266%26doi%3D10.1021%2Fjm00051a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Faller, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spanka, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sergejew, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tschinke, V.</span><span> </span><span class="NLM_article-title">Improving the oral bioavailability of the iron chelator HBED by breaking the symmetry of the intramolecular H-bond network</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">1467</span><span class="NLM_x">–</span> <span class="NLM_lpage">1475</span><span class="refDoi"> DOI: 10.1021/jm990261n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm990261n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvFKnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=1467-1475&author=B.+Fallerauthor=C.+Spankaauthor=T.+Sergejewauthor=V.+Tschinke&title=Improving+the+oral+bioavailability+of+the+iron+chelator+HBED+by+breaking+the+symmetry+of+the+intramolecular+H-bond+network&doi=10.1021%2Fjm990261n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Oral Bioavailability of the Iron Chelator HBED by Breaking the Symmetry of the Intramolecular H-Bond Network</span></div><div class="casAuthors">Faller, Bernard; Spanka, Carsten; Sergejew, Thomas; Tschinke, Vincenzo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1467-1475</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Physicochem. anal. and Monte Carlo simulations were used to identify structural features which prevent oral absorption of HBED, a potent iron chelator.  In water the dominant conformations of HBED involve the hydrophobic collapse of the two arom. rings.  These conformations are favored in polar media because they expose the polar phenolic hydroxy groups to the solvent and partially shield the nonpolar arom. rings.  In a less polar solvent such as chloroform, a sym. H-bond network between the carboxylates and the amines dominates the conformational space.  This leads to the exposure of the phenolic hydroxy groups to the solvent, which is unfavorable for solvation.  The low soly. of HBED in nonpolar solvents was confirmed exptl. by detn. of the partition coeffs. in octanol, chloroform, and cyclohexane and may explain the poor membrane permeability of this compd.  The high conformational stability which disfavors partitioning into phospholipids is mainly due to the sym. H-bond network.  Potentiometric titrns. of a monoester of HBED in MeOH/water indicate that the protonation sequence was changed compared to that of the parent compd., suggesting that the sym. H-bond network was disrupted.  Conformational anal. in chloroform confirmed that, in contrast to HBED, no sym. interaction between the carboxylate and the nitrogen amines is possible in the half-ester and a variety of conformations which allow partial shielding of the polar phenolic OH groups are energetically possible.  This theor. model predicting a better soly. of the half-esters in nonpolar solvents was supported by the large increase in the partition coeffs. in octanol, chloroform, and cyclohexane measured exptl.  The high absorbability predicted by physicochem. and computer simulation methods was corroborated by in vivo expts. in marmoset monkeys where the monoethyl ester deriv. of HBED was well-absorbed orally while the parent compd. was nearly ineffective in the same model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPO8b-CoWmebVg90H21EOLACvtfcHk0lgEGW2uUc21eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvFKnur8%253D&md5=8ee74f968fdb30328a99a413b3ddabff</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm990261n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm990261n%26sid%3Dliteratum%253Aachs%26aulast%3DFaller%26aufirst%3DB.%26aulast%3DSpanka%26aufirst%3DC.%26aulast%3DSergejew%26aufirst%3DT.%26aulast%3DTschinke%26aufirst%3DV.%26atitle%3DImproving%2520the%2520oral%2520bioavailability%2520of%2520the%2520iron%2520chelator%2520HBED%2520by%2520breaking%2520the%2520symmetry%2520of%2520the%2520intramolecular%2520H-bond%2520network%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D1467%26epage%3D1475%26doi%3D10.1021%2Fjm990261n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Davezac, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blot, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ducommun, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tassan, J. P.</span><span> </span><span class="NLM_article-title">Human pEg3 kinase associates with and phosphorylates CDC25B phosphatase: a potential role for pEg3 in cell cycle regulation</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">7630</span><span class="NLM_x">–</span> <span class="NLM_lpage">7641</span><span class="refDoi"> DOI: 10.1038/sj.onc.1205870</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1038%2Fsj.onc.1205870" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=7630-7641&author=N.+Davezacauthor=V.+Baldinauthor=J.+Blotauthor=B.+Ducommunauthor=J.+P.+Tassan&title=Human+pEg3+kinase+associates+with+and+phosphorylates+CDC25B+phosphatase%3A+a+potential+role+for+pEg3+in+cell+cycle+regulation&doi=10.1038%2Fsj.onc.1205870"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1205870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1205870%26sid%3Dliteratum%253Aachs%26aulast%3DDavezac%26aufirst%3DN.%26aulast%3DBaldin%26aufirst%3DV.%26aulast%3DBlot%26aufirst%3DJ.%26aulast%3DDucommun%26aufirst%3DB.%26aulast%3DTassan%26aufirst%3DJ.%2BP.%26atitle%3DHuman%2520pEg3%2520kinase%2520associates%2520with%2520and%2520phosphorylates%2520CDC25B%2520phosphatase%253A%2520a%2520potential%2520role%2520for%2520pEg3%2520in%2520cell%2520cycle%2520regulation%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D7630%26epage%3D7641%26doi%3D10.1038%2Fsj.onc.1205870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhNbRKvNbsg1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH8','PDB','5IH8'); return false;">PDB: 5IH8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH9','PDB','5IH9'); return false;">PDB: 5IH9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHA','PDB','5IHA'); return false;">PDB: 5IHA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHC','PDB','5IHC'); return false;">PDB: 5IHC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QNJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QNJ','PDB','2QNJ'); return false;">PDB: 2QNJ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i51"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00052">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_50025"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00052">10.1021/acs.jmedchem.6b00052</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_001.csv">CSV</a>)</p></li><li><p class="inline">Genetic knockdown of MELK by two different MELK-targeting shRNAs in MDA-MB-468, MCF-7 (colony formation assay, immunoblotting, quantitative RT-PCR, cell cycle effect), quantitation of MELK expression levels in MDA-MB-468 and MCF-7, IC<sub>50</sub> curves for <b>8a</b> and <b>8b</b>, mouse maximum-tolerated dose (MTD) and PK studies (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_001.csv">jm6b00052_si_001.csv (1.11 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00052/suppl_file/jm6b00052_si_002.pdf">jm6b00052_si_002.pdf (1.46 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH8">5IH8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IH9">5IH9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHA">5IHA</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5IHC">5IHC</a> correspond respectively to <b>1a</b>, <b>8a</b>, <b>8f</b>, and <b>12b</b>. The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00052&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-10%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00052%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00052" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a24e5bd1c3dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
